
<html lang="en"     class="pb-page"  data-request-id="57a31911-0deb-47f0-9ded-b258803b7710"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c01202;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Peptide Inhibitors of the α-Cobratoxin–Nicotinic Acetylcholine Receptor Interaction" /></meta><meta name="dc.Creator" content="Timothy  Lynagh" /></meta><meta name="dc.Creator" content="Stephan  Kiontke" /></meta><meta name="dc.Creator" content="Maria  Meyhoff-Madsen" /></meta><meta name="dc.Creator" content="Bengt H.  Gless" /></meta><meta name="dc.Creator" content="Jónas  Johannesen" /></meta><meta name="dc.Creator" content="Sabrina  Kattelmann" /></meta><meta name="dc.Creator" content="Anders  Christiansen" /></meta><meta name="dc.Creator" content="Martin  Dufva" /></meta><meta name="dc.Creator" content="Andreas H.  Laustsen" /></meta><meta name="dc.Creator" content="Kanchan  Devkota" /></meta><meta name="dc.Creator" content="Christian A.  Olsen" /></meta><meta name="dc.Creator" content="Daniel  Kümmel" /></meta><meta name="dc.Creator" content="Stephan Alexander  Pless" /></meta><meta name="dc.Creator" content="Brian  Lohse" /></meta><meta name="dc.Description" content="Venomous snakebites cause &gt;100 000 deaths every year, in many cases via potent depression of human neuromuscular signaling by snake α-neurotoxins. Emergency therapy still relies on antibody-based a..." /></meta><meta name="Description" content="Venomous snakebites cause &gt;100 000 deaths every year, in many cases via potent depression of human neuromuscular signaling by snake α-neurotoxins. Emergency therapy still relies on antibody-based a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 4, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01202" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01202" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01202" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01202" /></link>
        
    
    

<title>Peptide Inhibitors of the α-Cobratoxin–Nicotinic Acetylcholine Receptor Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01202" /></meta><meta property="og:title" content="Peptide Inhibitors of the α-Cobratoxin–Nicotinic Acetylcholine Receptor Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0004.jpeg" /></meta><meta property="og:description" content="Venomous snakebites cause &gt;100 000 deaths every year, in many cases via potent depression of human neuromuscular signaling by snake α-neurotoxins. Emergency therapy still relies on antibody-based antivenom, hampered by poor access, frequent adverse reactions, and cumbersome production/purification. Combining high-throughput discovery and subsequent structure–function characterization, we present simple peptides that bind α-cobratoxin (α-Cbtx) and prevent its inhibition of nicotinic acetylcholine receptors (nAChRs) as a lead for the development of alternative antivenoms. Candidate peptides were identified by phage display and deep sequencing, and hits were characterized by electrophysiological recordings, leading to an 8-mer peptide that prevented α-Cbtx inhibition of nAChRs. We also solved the peptide:α-Cbtx cocrystal structure, revealing that the peptide, although of unique primary sequence, binds to α-Cbtx by mimicking structural features of the nAChR binding pocket. This demonstrates the potential of small peptides to neutralize lethal snake toxins in vitro, establishing a potential route to simple, synthetic, low-cost antivenoms." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01202"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01202">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01202&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01202&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01202&amp;href=/doi/10.1021/acs.jmedchem.0c01202" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 13709-13718</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01257" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01277" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Peptide Inhibitors of the α-Cobratoxin–Nicotinic Acetylcholine Receptor Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Timothy Lynagh</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Timothy Lynagh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sars International Centre for Marine Molecular Biology, University of Bergen, Thormøhlensgate 55, 5008 Bergen, Norway</div><div class="loa-info-affiliations-info">Center for Biopharmaceuticals & Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><span class="conrtib-corresp"><strong>*</strong>T.L.: email, <a href="/cdn-cgi/l/email-protection#1b6f7276357762757a7c735b6e7279357574"><span class="__cf_email__" data-cfemail="f98d9094d7958097989e91b98c909bd79796">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Timothy++Lynagh">Timothy Lynagh</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4888-4098" title="Orcid link">http://orcid.org/0000-0003-4888-4098</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephan Kiontke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephan Kiontke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Department of Biology/Chemistry, University of Osnabrück, Barbarastraße 13, Osnabrück 49076, Germany</div><div class="loa-info-affiliations-info">Faculty of Biology, Department of Plant Physiology and Photobiology, Philipps-Universität Marburg, Karl-von-Frisch-Straße 8, 35032 Marburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephan++Kiontke">Stephan Kiontke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maria Meyhoff-Madsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maria Meyhoff-Madsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maria++Meyhoff-Madsen">Maria Meyhoff-Madsen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bengt H. Gless</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bengt H. Gless</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Biopharmaceuticals & Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bengt+H.++Gless">Bengt H. Gless</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jónas Johannesen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jónas Johannesen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%B3nas++Johannesen">Jónas Johannesen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sabrina Kattelmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sabrina Kattelmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, University of Münster, Corrensstraße 36, 48149 Münster, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sabrina++Kattelmann">Sabrina Kattelmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anders Christiansen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anders Christiansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fluid Array Systems and Technology, Nano and Bio-physical Systems, Department of Health Technology, Technical University of Denmark, Building 423 Produktionstorvet, DK-2800 Kongens Lyngby, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anders++Christiansen">Anders Christiansen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Dufva</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Dufva</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fluid Array Systems and Technology, Nano and Bio-physical Systems, Department of Health Technology, Technical University of Denmark, Building 423 Produktionstorvet, DK-2800 Kongens Lyngby, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Dufva">Martin Dufva</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas H. Laustsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas H. Laustsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas+H.++Laustsen">Andreas H. Laustsen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kanchan Devkota</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kanchan Devkota</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kanchan++Devkota">Kanchan Devkota</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian A. Olsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian A. Olsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Biopharmaceuticals & Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian+A.++Olsen">Christian A. Olsen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2953-8942" title="Orcid link">http://orcid.org/0000-0002-2953-8942</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel Kümmel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Kümmel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Department of Biology/Chemistry, University of Osnabrück, Barbarastraße 13, Osnabrück 49076, Germany</div><div class="loa-info-affiliations-info">Institute of Biochemistry, University of Münster, Corrensstraße 36, 48149 Münster, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++K%C3%BCmmel">Daniel Kümmel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephan Alexander Pless</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephan Alexander Pless</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Biopharmaceuticals & Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephan+Alexander++Pless">Stephan Alexander Pless</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6654-114X" title="Orcid link">http://orcid.org/0000-0001-6654-114X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Brian Lohse</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brian Lohse</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><span class="conrtib-corresp"><strong>*</strong>B.L.: email, <a href="/cdn-cgi/l/email-protection#1577677c795566607b713b7e603b717e"><span class="__cf_email__" data-cfemail="95f7e7fcf9d5e6e0fbf1bbfee0bbf1fe">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brian++Lohse">Brian Lohse</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01202&amp;href=/doi/10.1021%2Facs.jmedchem.0c01202" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 13709–13718</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 3, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 July 2020</li><li><span class="item_label"><b>Published</b> online</span>4 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01202" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01202</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01202"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2596</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01202" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Peptide Inhibitors of the α-Cobratoxin–Nicotinic Acetylcholine Receptor Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;Lynagh&quot;},{&quot;first_name&quot;:&quot;Stephan&quot;,&quot;last_name&quot;:&quot;Kiontke&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Meyhoff-Madsen&quot;},{&quot;first_name&quot;:&quot;Bengt&quot;,&quot;last_name&quot;:&quot;H. Gless&quot;},{&quot;first_name&quot;:&quot;Jónas&quot;,&quot;last_name&quot;:&quot;Johannesen&quot;},{&quot;first_name&quot;:&quot;Sabrina&quot;,&quot;last_name&quot;:&quot;Kattelmann&quot;},{&quot;first_name&quot;:&quot;Anders&quot;,&quot;last_name&quot;:&quot;Christiansen&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Dufva&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;H. Laustsen&quot;},{&quot;first_name&quot;:&quot;Kanchan&quot;,&quot;last_name&quot;:&quot;Devkota&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;A. Olsen&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Kümmel&quot;},{&quot;first_name&quot;:&quot;Stephan&quot;,&quot;last_name&quot;:&quot;Alexander Pless&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;Lohse&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;13709-13718&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01202&quot;},&quot;abstract&quot;:&quot;Venomous snakebites cause &gt;100 000 deaths every year, in many cases via potent depression of human neuromuscular signaling by snake α-neurotoxins. Emergency therapy still relies on antibody-based antivenom, hampered by poor access, frequent adverse reactions, and cumbersome production/purification. Combining high-throughput discovery and subsequent structure–function characterization, we present simple peptides that bind α-cobratoxin (α-Cbtx) and prevent its inhibition of nicotinic acetylcholine receptors (nAChRs) as a lead for the development of alternative antivenoms. Candidate peptides were identified by phage display and deep sequencing, and hits were characterized by electrophysiological recordings, leading to an 8-mer peptide that prevented α-Cbtx inhibition of nAChRs. We also solved the peptide:α-Cbtx cocrystal structure, revealing that the peptide, although of unique primary sequence, binds to α-Cbtx by mimicking structural features of the nAChR binding pocket. This demonstrates the potential of small&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01202&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01202" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01202&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01202" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01202&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01202" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01202&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01202&amp;href=/doi/10.1021/acs.jmedchem.0c01202" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01202" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01202" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01202%26sid%3Dliteratum%253Aachs%26pmid%3D33143415%26genre%3Darticle%26aulast%3DLynagh%26date%3D2020%26atitle%3DPeptide%2BInhibitors%2Bof%2Bthe%2B%25CE%25B1-Cobratoxin%25E2%2580%2593Nicotinic%2BAcetylcholine%2BReceptor%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D22%26spage%3D13709%26epage%3D13718%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=290979" title="Peptide identification">Peptide identification</a>,</li><li><a href="/action/doSearch?ConceptID=292035" title="Viruses">Viruses</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/jmcmar.2020.63.issue-22/20201125/jmcmar.2020.63.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Venomous snakebites cause >100 000 deaths every year, in many cases via potent depression of human neuromuscular signaling by snake α-neurotoxins. Emergency therapy still relies on antibody-based antivenom, hampered by poor access, frequent adverse reactions, and cumbersome production/purification. Combining high-throughput discovery and subsequent structure–function characterization, we present simple peptides that bind α-cobratoxin (α-Cbtx) and prevent its inhibition of nicotinic acetylcholine receptors (nAChRs) as a lead for the development of alternative antivenoms. Candidate peptides were identified by phage display and deep sequencing, and hits were characterized by electrophysiological recordings, leading to an 8-mer peptide that prevented α-Cbtx inhibition of nAChRs. We also solved the peptide:α-Cbtx cocrystal structure, revealing that the peptide, although of unique primary sequence, binds to α-Cbtx by mimicking structural features of the nAChR binding pocket. This demonstrates the potential of small peptides to neutralize lethal snake toxins in vitro, establishing a potential route to simple, synthetic, low-cost antivenoms.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Note</h4><p class="last">Supporting Information Figure S2 was misformatted in the version published on November 4, 2020 and was correctly restored on November 5, 2020.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14174" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14174" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Snakebite envenoming kills or maims hundreds of thousands of people every year,<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> and loss of livestock is also emerging as a substantial problem.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Treatment requires rapid transport to a medical facility and intravenous administration of antivenom, a mix of immunoglobulin G antibodies isolated from, e.g., horse or sheep immunized with snake venom.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> As the only effective treatment, antivenoms save numerous lives but have the disadvantages of time-consuming and expensive production, poor access for those in need, and alarming numbers of adverse reactions.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> There is therefore an urgent need for novel treatments and new strategies, reflected in growing attention from the World Health Organization.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">In the case of elapid snakes, such as cobras and kraits, three-finger α-neurotoxins such as α-cobratoxin (α-Cbtx) and α-bungarotoxin (α-Bgtx) are the most prominent and lethal venom components.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> α-Cbtx and α-Bgtx potently inhibit postsynaptic nicotinic acetylcholine receptors (nAChRs), which depresses neuromuscular signaling, causing paralysis and loss of respiration.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The effectiveness of conventional antivenoms is limited by immunogenicity, abundance of α-neurotoxin antibodies in the immunized animal at the time of production,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and which type of snake inflicts the bite. Newer antibody-based strategies have emerged, with a focus on smaller, less immunogenic, and venom-specific antivenoms. Smaller camelid antibody fragments raised against cobra venom were shown to neutralize the effects of α-Cbtx in mice and may allow for greater tissue penetration than conventional antivenoms.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Phage display of a library of human-derived antibody single-chain variable fragments against α-Bgtx from krait venom led to one fragment that neutralized toxin injected into mice.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In the future, combinations of such lead entities could enable development of safe and effective treatments, as reflected in the recent application of cocktails of human monoclonal antibodies targeting mamba toxins.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> These showed promising neutralization of dendrotoxins in mice, although some improvement is needed in neutralizing α-neurotoxins.</div><div class="NLM_p last">As an alternative to antibody-based strategies, we sought to discover short peptide-based neutralizers of α-Cbtx by screening phage display peptide libraries for α-Cbtx binders. Hits were then tested for neutralization of α-Cbtx inhibition of nicotinic nAChRs using electrophysiology. This led to the identification of an effective 8-mer peptide, for which we solved the peptide:α-Cbtx X-ray cocrystal structure, revealing the detailed binding mode. The identification and characterization of short peptide neutralizers of α-Cbtx could lead to the development of targeted, fully synthetic, next-generation antivenoms for snakebite envenoming.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87785" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87785" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> High-Throughput Identification of Peptide Binders of α-Cbtx</h3><div class="NLM_p">Seeking novel binders of α-Cbtx, we screened naïve phage display libraries containing ≥1 × 10<sup>9</sup> disulfide-constrained 7-mers and linear 7-mers, 12-mers, 16-mers, or 20-mers against immobilized α-Cbtx from <i>Naja kaouthia</i> (monocled cobra). Isolation and sequencing of the most prevalent phages after five rounds of biopanning identified two 16-mers, peptides <b>1</b> and <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Additionally, deep sequencing of phages from all rounds identified six additional prevalent peptides, <b>3</b>–<b>8</b>, that were also considered further (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Phage Display Hits, Derivatives Thereof, and Their Prevention of α-Cbtx Inhibition of nAChRs</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center">sequence</th><th class="colsep0 rowsep0" align="center">fold prevention of α-Cbtx inhibition<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">affinity<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">identified by</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">no peptide (α-cbtx alone)</td><td class="colsep0 rowsep0" align="center">1.0</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">Ac-HAKTYMWDGWYMPTSH-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">7.4</td><td class="colsep0 rowsep0" align="center">2 ± 2 μM</td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">Ac-HVNTYMWDGRYMRTSH-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">1.0</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">H-NWAPAQHGQKWYMQDYTSLE-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">3.8</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">H-TSNTTPWQTSWELMYAQQNY-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">H-KPHTYINDHNFIQYDNQQWL-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">not soluble</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">H-YVTHPWIDWWDNEPYM-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">not soluble</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H-GHHHRVYMTDVTEHFRWMGK-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.6</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">H-GHHHRVYMTDVTEHFRWWVN-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">phage display<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">TAMRA-HAKTYMWDGWYMPTSH-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">not tested</td><td class="colsep0 rowsep0" align="center">52 ± 3 μM</td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">Ac-TYMWDGWYMPT-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">4.5</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">Ac-YMWDGWYM-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="center">6.9</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">[PEG]<sub>6</sub>-YMWDGWYM-NH<sub>2</sub><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">6.2</td><td class="colsep0 rowsep0" align="center">26 ± 4 μM</td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">[PEG]<sub>6</sub>-YMWDGW-OH<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">[PEG]<sub>6</sub>-YMWDGW-NH<sub>2</sub><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">H-WDGWYM-[PEG]<sub>6</sub><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">Ac-WDGWYM-[PEG]<sub>6</sub><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">[PEG]<sub>6</sub>-YMWEGWYM-OH<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">4.8</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">[PEG]<sub>6</sub>-YM(Nal)DGWYM-OH<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a>,<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="center">2.8</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">optimization</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">α-Cbtx (40 nM) inhibited ACh (100 μM) gated currents through nAChRs to 8 ± 2% their control level, which is here normalized to 1.0. In the presence of, e.g., peptide <b>1</b> (100 μM), α-Cbtx (40 nM) only inhibited ACh (100 μM) gated currents to 60 ± 18% of control, here normalized to 7.4-<i>fold prevention of inhibition</i>.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last"><i>K</i><sub>d</sub> value of <b>9</b> determined in fluorescence polarization (FP) saturation experiment (<i>n</i> = 2). <i>K</i><sub>i</sub> values for peptides <b>1</b> and <b>12</b> determined by inhibition of <b>9</b> binding to α-Cbtx in FP competition experiments (<i>n</i> = 6), only measured for two lead peptides.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">After five rounds of selection by biopanning.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">From deep sequencing of all phages.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">PEG, polyethylene glycol.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Nal, 3-(2-naphthyl)-<span class="smallcaps smallerCapital">l</span>-alanine.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Characterization of Peptides with YM Motifs That Modulate α-Cbtx Inhibition of Nicotinic Acetylcholine Receptors</h3><div class="NLM_p">We next tested these binders for functional modulation of α-Cbtx activity in electrophysiological assays. We heterologously expressed nicotinic α7-homomeric nAChRs in <i>Xenopus laevis</i> oocytes and measured acetylcholine-gated currents in response to acetylcholine alone, in the presence of α-Cbtx, or in the presence of α-Cbtx preincubated with the identified peptides. As expected, α-Cbtx alone (40 nM) inhibited nAChR responses to ∼10% compared to the control level, and washout of this effect was minimal, with subsequent responses to ACh in the absence of α-Cbtx recovering to only 20% (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B). In the presence of peptide <b>1</b> (100 μM), α-Cbtx (40 nM) only inhibited responses to ∼50% of the control level, with responses after washout reaching 70% (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B). Peptide <b>1</b> thus appears to prevent the effects of α-Cbtx. Peptide <b>2</b> had no effect on α-Cbtx inhibiting nAChRs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Among the six additional binders, peptide <b>3</b> prevented α-Cbtx inhibition of nAChRs comparably to <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). However, peptide <b>4</b> showed no effect (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), and peptides <b>5</b> and <b>6</b> were poorly soluble and could not be tested. Curiously, peptides <b>7</b> and <b>8</b><i>enhanced</i> both the onset and the washout of α-Cbtx inhibition of nAChRs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Thus, for two of the eight α-Cbtx binders identified by phage display, the desired effect of preventing α-Cbtx inhibition of nAChRs could be demonstrated, and the shorter peptide <b>1</b> was chosen for the further investigations.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Peptide <b>1</b> prevents α-Cbtx inhibition of nAChRs. (A) Example two electrode voltage clamp recordings of ACh-gated currents in <i>Xenopus laevis</i> oocytes expressing L247V mutant α7 nAChRs and RIC-3 (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>, <a class="ref internalNav" href="#sec5_3" aria-label="Electrophysiological Assays">Electrophysiological Assays</a>). In testing peptides, ACh was applied for 10 s in regular conditions (“control”), three times after the application of either α-Cbtx (blue bars) or α-Cbtx with peptide (blue and magenta bars; “App 1–App 3”), and then three more times to observe washout of any effects (“Wash 1–Wash 3”). (B) Mean ± SEM ACh-gated current amplitude, normalized to control (“<i>I</i>/<i>I</i><sub>control</sub>”). Each peptide was tested with α-Cbtx on at least three oocytes: <i>n</i> (magenta columns) = 3–4. α-Cbtx alone was tested on the same day on at least two oocytes: <i>n</i> (blue columns) = 2 (peptides <b>1</b> and <b>2</b>) – 4 (all others).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next established the dose-dependency of the effects of <b>1</b> on α-Cbtx in our electrophysiological assay. The initial protection from α-Cbtx inhibition (“<i>I</i><sub>App3</sub>/<i>I</i><sub>Control</sub>”) occurred at concentrations of <b>1</b> in the range 30–100 μM, whereas apparent enhancement of α-Cbtx washout (“<i>I</i><sub>Wash3</sub>/<i>I</i><sub>Control</sub>”) occurred at concentrations of <b>1</b> between 3–30 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). To better estimate binding affinity, we turned to a fluorescence polarization (FP) assay, for which we generated peptide <b>9</b>, a rhodamine dye-labeled derivative of peptide <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). From incubating <b>9</b> in increasing concentrations of α-Cbtx, we determined the dissociation constant for the α-Cbtx–<b>9</b> interaction (<i>K</i><sub>d</sub> 52 ± 3 μM, <i>n</i> = 2, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). By incubating premixed α-Cbtx (40 μM) and <b>9</b> (50 nM) with increasing concentrations of <b>1</b>, we determined an IC<sub>50</sub> value of 29 ± 6 μM for compound <b>1</b> (<i>n</i> = 6, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E), corresponding to a <i>K</i><sub>i</sub> value of 1.9 μM (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>, <a class="ref internalNav" href="#sec5_5" aria-label="Fluorescence Polarization Assay">Fluorescence Polarization Assay</a>). Regarding concentrations of <b>1</b> that elicited functional effects, this <i>K</i><sub>i</sub> is more consistent with the enhanced washout of α-Cbtx than with the prevention of α-Cbtx inhibition.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Concentration dependence of peptide activity. (A,B) Electrophysiological experiments as per <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, but here only initial ACh application (“Control”), the third ACh application after peptide + α-Cbtx (40 nM) (“App 3”), and the third ACh application after washout (“Wash 3”) are shown. Mean ± SEM (<i>n</i> = 3) are shown to the right. (C) Chemical structure of <b>9</b>. (D) Fluorescence polarization of <b>9</b> (50 nM) by increasing concentrations of α-Cbtx. Mean ± SEM, <i>n</i> = 2. (E) Concentration-dependent displacement of <b>9</b> (50 nM) from α-Cbtx (40 μM) by compound <b>1</b> and <b>12</b>. Mean ± SEM, <i>n</i> = 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Peptide Aromatic Side Chains Bind to α-Cbtx Key Residues R33 and D27</h3><div class="NLM_p">Seeking insight into the peptide−α-Cbtx interaction, we turned to structure determination by X-ray crystallography. This required improved peptide solubility, and we therefore designed and tested truncated (peptides <b>10</b> and <b>11</b>) and truncated, polyethylene glycolated (PEG<sub>6</sub>-ylated, peptides <b>12</b>–<b>16</b>) derivatives of <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S1</a>). The PEG<sub>6</sub>-ylated 8-mer, peptide <b>12</b>, exhibited similar potency to <b>1</b> in electrophysiological assays, with <b>12</b> (100 μM) restoring α-Cbtx (40 nM) inhibited ACh-gated currents from ∼10% to ∼50% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). In the FP assay, <b>12</b> outcompeted <b>9</b> from α-Cbtx similarly to <b>1</b>, despite slightly reduced affinity compared to <b>1</b> (<b>12</b><i>K</i><sub>i</sub> = 26 ± 4 μM, <i>n</i> = 6, <i>P</i> < 0.003, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E). We also observed increased stability of <b>12</b> compared to <b>1</b> at both room temperature and at 37 °C (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S2</a>). Given the similar functional properties but improved solubility of <b>12</b>, it was used for X-ray crystallography.</div><div class="NLM_p">We obtained diffraction-quality cocrystals and solved the X-ray crystal structure of the <b>12</b>:α-Cbtx complex at 1.9 Å resolution (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Table S1</a>). The asymmetric unit contains four copies of α-Cbtx and two copies of <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). One molecule of <b>12</b> binds to two toxin molecules (designated α-Cbtx-A and α-Cbtx-B), forming a 1:2 complex (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S3</a>). The two α-Cbtx-A:<b>12</b>:α-Cbtx-B complexes present in the asymmetric unit are related by noncrystallographic symmetry and highly similar to a root mean square deviation (rmsd) of 0.2 Å. Thus, we observed two different binding modes of <b>12</b> with α-Cbtx (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,C). Whereas the structure of α-Cbtx-B is very similar to the conformation of α-Cbtx observed in previous crystal structures, <b>12</b> binding induces substantial structural rearrangements of loop II and indirectly also of loop III in α-Cbtx-A (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D,E). The formation of a 1:2 <b>12</b>:α-Cbtx complex was surprising considering that <b>12</b> was added in 1.2-fold molar excess in the crystallization set-ups. This raises the possibility that the mechanism of α-Cbtx inhibition by <b>12</b> involves inducing toxin dimerization. There is very little contact between the two α-Cbtx molecules themselves in the complex (52 Å<sup>2</sup>) as determined by computational analysis (PISA),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> suggesting that these interactions do not contribute to the stabilization of the complex. We reasoned that if both motifs A and B were acting independently as inhibitors, they should prevent α-Cbtx inhibition of nAChRs on their own. However, shorter peptides designed to include only binding motif A (peptides <b>13</b> and <b>14</b>) or motif B (peptides <b>15</b> and <b>16</b>) did not have any effect on α-Cbtx activity in electrophysiological assays (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S1</a>). We therefore tentatively conclude that the simultaneous binding of <b>12</b> to two α-Cbtx molecules leads to a mutual stabilization of both interfaces or that peptide <b>12</b> blocks α-Cbtx by linking two α-Cbtx molecules and preventing them from further binding to nAChRs. However, analysis of <b>12</b>-induced dimerization of α-Cbtx in solution by size exclusion chromatography was not possible due to the high hydrophobicity of the peptide, which led to aggregation on the gel filtration column.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of the <b>12</b>:α-Cbtx complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a>). (A) Asymmetric unit of the <b>12</b>:α-Cbtx crystal structure. Each molecule of <b>12</b> (magenta) binds two toxin molecules A and B (light and dark cyan, respectively). A second α-Cbtx-A:<b>12</b>:α-Cbtx-B complex in the asymmetric unit is related by noncrystallographic symmetry. (B) Magnified view showing interaction of α-Cbtx loop II (light cyan) with <b>12</b> (magenta) in binding mode A. Two alternative conformations for the side chain of <sup><b>12</b></sup>M2 were modeled. Key interacting amino acid residues shown in stick representation and labeled. Cation−π-stacking interactions and hydrogen bonds are highlighted by dashed and dotted lines, respectively. (C) Binding mode B, with second α-Cbtx molecule (dark cyan), from same structure with interacting residues drawn as in (B). (D) Binding mode A leads to conformational changes in α-Cbtx (light cyan) compared to the structure observed of α-Cbtx from <i>Naja naja siamensis</i> (dark gray, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CTX">2CTX</a>, root-mean-square deviation (rmsd) 1,31 Å), <i>Naja naja oxania</i> (light gray, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NTN">1NTN</a>, rmsd 1.7 Å), or <i>Naja naja kaouthia</i> in complex with AChBP (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YI5">1YI5</a>, rmsd 1.47 Å). The greatest displacement is observed in loop II (11 Å) and loop III (4.7 Å). (E) Binding mode B (cyan) resembles more closely previously reported structures of α-Cbtx from <i>Naja naja siamensis</i> (dark gray, rmsd 1,07 Å), <i>Naja naja oxania</i> (light gray, rmsd 0.67 Å), or <i>Naja naja kaouthia</i> in complex with AChBP (green, rmsd 0.87 Å). (F) Interaction of α-Bgtx (blue) loop II with the principal (α subunit) and complementary (δ subunit) faces of the muscle nAChR (pink, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UWZ">6UWZ</a>). Key interacting residues drawn as in (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although binding modes A and B are distinct, they share common features (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,C, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). (i) Two different YM motifs bind at equivalent positions in binding mode A and B, forming a hydrogen bond between <sup><b>12</b></sup>Y7 or <sup><b>12</b></sup>Y1 and <sup>α-Cbtx</sup>D27 and interacting with the surface of α-Cbtx at the beginning of loop II. (ii) Using different aromatic residues, modes A and B both create, together with <sup>α-Cbtx</sup>F29, an aromatic pocket for <sup>α-Cbtx</sup>R33 through extensive cation−π stacking interactions of parallel and T-shaped geometry (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,C). Thus, the key residue <sup>α-Cbtx</sup>R33 that has been implicated in penetrating the ACh binding pocket of nAChRs<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> is tightly blocked by <b>12</b> in both binding modes. In binding mode A, a tryptophan residue that precedes the YM motif also interacts with <sup>α-Cbtx</sup>F65 at the C-terminus of α-Cbtx and forms a hydrogen bond to the backbone carbonyl of <sup>α-Cbtx</sup>P7 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Exchange of this tryptophan for arginine (along with three other substitutions) in peptide <b>2</b> led to a loss of the preventive activity of the peptide (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), underscoring the functional importance of this tryptophan residue.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Key Residues of the <b>12</b>–α-Cbtx Interaction (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a>) in Comparison to the nAChR−α-Bgtx Complex</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">α-Cbtx</th><th class="colsep0 rowsep0" align="center"><b>12</b>-A</th><th class="colsep0 rowsep0" align="center"><b>12</b>-B</th><th class="colsep0 rowsep0" align="center">α-Bgtx</th><th class="colsep0 rowsep0" align="center">nAChR (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UWZ">6UWZ</a>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">D27</td><td class="colsep0 rowsep0" align="left">Y7 (H bond)</td><td class="colsep0 rowsep0" align="left">Y1 (H bond)</td><td class="colsep0 rowsep0" align="left">D30</td><td class="colsep0 rowsep0" align="left">αY190 (H bond)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">F29</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">W6 (π–π)</td><td class="colsep0 rowsep0" align="left">F32</td><td class="colsep0 rowsep0" align="left">γW55/δW57, αW149 (hydrophobic)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R33</td><td class="colsep0 rowsep0" align="left">W3, Y7 (cation−π)</td><td class="colsep0 rowsep0" align="left">Y1, W6 (cation−π)</td><td class="colsep0 rowsep0" align="left">R36</td><td class="colsep0 rowsep0" align="left">Y190α, Y198α (cation−π); γW55/δW57, αW149 (hydrophobic)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R36</td><td class="colsep0 rowsep0" align="left">D4 (salt bridge)</td><td class="colsep0 rowsep0" align="left">Y1 (hydrophobic)</td><td class="colsep0 rowsep0" align="left">V39</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">F65</td><td class="colsep0 rowsep0" align="left">W6 (hydrophobic)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">H68</td><td class="colsep0 rowsep0" align="left">backbone (H-bond)</td></tr></tbody></table></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> The Inhibitory Peptide Mimics the α-Cbtx Interaction Site and Binding Mode of the nAChR</h3><div class="NLM_p last">The structure of the nAChR:α-Cbtx complex has not been determined, but the structure of the muscle-type (heteromeric α–γ–α–δ−β) nAChR:α-Bgtx complex was recently reported.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Because α-Cbtx and α-Bgtx share high sequence and structural homology,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> we can use the nAChR:α-Bgtx complex to compare α-Cbtx interactions with the peptide inhibitor to those with its native target (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The nAChR:α-Bgtx interaction is much more extensive (∼1100 Å<sup>2</sup>) compared to the <b>12</b>:α-Cbtx complex (∼440 Å<sup>2</sup> in mode A and ∼510 Å<sup>2</sup> in mode B), but interestingly, the peptide inhibitor <b>12</b> partially mimics the binding site of the natural substrate. Most strikingly, <sup><b>12</b></sup>Y1 (in binding mode B) and <sup><b>12</b></sup>Y7 (in binding mode A) adopt the same position as <sup>nAChRα</sup>Y190, which forms a conserved hydrogen bond with <sup>α-Cbtx</sup>D27 and <sup>α-Bgtx</sup>D30, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This interaction is also conserved in the complex of α-Cbtx with the acetylcholine binding protein (AChBP),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> where <sup>AChBP</sup>Y185 binds <sup>α-Cbtx</sup>D27 (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S4A</a>). The similarity between <b>12</b> and nAChR is also apparent in the pocket for <sup>α-Cbtx</sup>R33/<sup>α-Bgtx</sup>R36 and <sup>α-Cbtx</sup>F29/<sup>α-Bgtx</sup>F32 in binding mode A, which is formed by aromatic side chains <sup><b>12</b></sup>W3/<sup><b>12</b></sup>Y7 in <b>12</b>-A and <sup>nAChRα</sup>Y190/<sup>nAChRα</sup>Y198 in the nAChR. In contrast, <sup><b>12</b></sup>Y1/<sup><b>12</b></sup>W6 in <b>12</b>-B are also engaged in cation−π interactions with <sup>α-Cbtx</sup>R33, but the orientations of side chains differ from nAChR:α-Bgtx. The aromatic pocket for <sup>α-Bgtx</sup>R36 is closed by y<sup>nAChRγ</sup>W55 /<sup>nAChRγ</sup>δW57 and <sup>nAChRα</sup>W149 through hydrophobic interaction, which lacks a counterpart in the <b>12</b>:α-Cbtx complex. It is also interesting to note that aspartate <sup>nAChRα</sup>D200 superposes with <sup><b>12</b></sup>D4 of binding mode A. <sup>nAChRα</sup>D200 does not play a role in α-Bgtx binding to the receptor, but the substitution of <sup>α-Bgtx</sup>V39 to <sup>α-Cbtx</sup>R36 likely leads to the formation of a salt bridge upon binding of α-Cbtx to the nAChR. Thus, <sup><b>12</b></sup>D4 would further imitate the target binding site. Taken together, the structural comparison reveals that the phage display selection provided a peptide that blocks the binding site of α-Cbtx by imitating the cognate interface through an unrelated sequence.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26027" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26027" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The urgent need for alternatives to conventional antivenom for snakebite has given rise to a handful of novel approaches, including camelid- and human-derived antibodies that target particular venom components,<a onclick="showRef(event, 'ref14 ref15 ref16 ref21'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref21">(14−16,21)</a> nanoparticles to adsorb plasma proteins,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and small molecules or peptides to inhibit key components of venom.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> Smaller peptides are likely cheaper than traditional antivenom and may lead to more convenient and effective drugs that can be injected subcutaneously.<a onclick="showRef(event, 'ref7 ref25'); return false;" href="javascript:void(0);" class="ref ref7 ref25">(7,25)</a> Consequently, we pursued the idea that short, synthetic peptides inhibiting critical toxin–receptor interactions could serve as lead compounds for the development of novel peptide-based antivenoms. Furthermore, our results suggest that such small peptides are conducive to crystallization and rational design and could be useful for the academic community as pharmacological tools. Starting from some five billion 7-mers, 12-mers, 16-mers, and 20-mers, we arrived at an 8-mer that prevents α-Cbtx from inhibiting nicotinic nAChRs.</div><div class="NLM_p">Our initial search identified eight peptides that bound α-Cbtx. Despite the absence of primary sequence homology with nAChRs, each peptide contained at least one YM motif. The tyrosine forms a hydrogen bond to <sup>α-Cbtx</sup>D27, and this Y–D bond is conserved in both peptide binding modes that we observed in AChBP:α-Cbtx and nAChR:α-Bgtx structures<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> and also in a nAChR-like 13-mer binding to α-Bgtx<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S4B</a>), suggesting that this tyrosine residue represents an essential feature for α-Cbtx-binding peptides. The methionine occupies a unique binding site at the base of α-Cbtx loop II that might be exploited in future optimizations. Remarkably, the YM motif is flanked by a tryptophan in the two peptides (and derivatives thereof) that had the desired effect of preventing nAChR inhibition by α-Cbtx. The YM sequence in combination with an upstream or downstream tryptophan residue thus represents a novel motif that might serve as a scaffold for designing future peptide inhibitors or peptide-derived small molecule inhibitors of α-Cbtx. Simultaneous binding of our inhibitor to two α-Cbtx molecules seems to be required for functionality. This is evident in the inhibition of α-Cbtx by peptides <b>1</b>, <b>10</b>, <b>11</b>, and <b>12</b>, which contain both motifs A and B of <b>12</b>, while peptides <b>13</b>–<b>16</b>, which contain either motif A or motif B of <b>12</b>, fail to inhibit α-Cbtx. We suspect that binding of two α-Cbtx molecules results in stabilization of the peptide conformation. Introducing dimerization of α-Cbtx might also be part of the inhibitory mechanism of the identified peptides. Thus, strategies to rigidify the two identified epitopes might be employed for improved activity.</div><div class="NLM_p">Optimization will also require improved affinity for α-Cbtx, as half-maximal effective concentrations and <i>K</i><sub>i</sub> values for our 8-mer, 16-mer, and 20-mer inhibitors were in the low micromolar range. Much greater affinity for α-Bgtx was seen with previously studied 13-mers,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> possibly due to greater stability through the additional intramolecular hydrogen bonds afforded by the β-hairpin structure of those peptides. Although also identified through phage display, those peptides bear striking similarity to the principal face of nAChRs.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Our identification of a nAChR-unrelated sequence with a mimicking effect opens up entirely new avenues for future work. It is also noteworthy that, in addition to inhibitors of α-Cbtx, we also identified enhancers of α-Cbtx activity, tentatively suggesting that toxin activity can be manipulated by peptides via distinct mechanisms. The desired improvement of affinity of <b>1</b>, <b>12</b>, and similar molecules from the present study may be possible through subtle modifications. In limited structure–activity relationship experiments, we observed that single substitutions of D4 and W3 to glutamic acid and naphthylalanine, respectively, yielded two peptides, <b>17</b> and <b>18</b>, that retained the inhibitory effect on α-Cbtx (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S1B</a>). Tolerance of isofunctional substitutions at W3 and D4, which indeed interact with <sup>α-Cbtx</sup>R33 and <sup>α-Cbtx</sup>R36 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), suggests some scope for optimization at these positions.</div><div class="NLM_p last">That PEGylation was also well tolerated could prove useful in future antivenom drug development. Weaknesses of peptide-based drugs include short in vivo half-life/rapid elimination and poor membrane permeability.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> However, these can sometimes be overcome by increasing molecular mass via PEGylation and conjugation to larger peptides or other moieties.<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> The fact that the YMWDGWYM motif was effective on its own (as <b>11</b>), PEGylated (as <b>12</b>), straddled by other residues (in <b>1</b>), or even as a divergent, longer peptide (as <b>3</b>) is also promising when considering typical approaches to improve stability and membrane permeation, such as cyclization, stapling, and conjugation.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Our measurements of peptides <b>1</b> (free peptide) and <b>12</b> (PEGylated) in phosphate-buffered saline show that both are reasonably stable over the course of 1 day, and notably, the purity of <b>12</b> was decreased by just 7% after incubation for 9 days at room temperature or 2 days at 37 °C (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S2</a>).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Current therapies for snakebite envenoming are difficult to produce, far too inaccessible for those in need, and often cause adverse reactions.<a onclick="showRef(event, 'ref1 ref32'); return false;" href="javascript:void(0);" class="ref ref1 ref32">(1,32)</a> We speculate that a combination of several peptides targeting the key venom components will provide simpler, more distributable, safer, and subcutaneous- and/or intramuscular-capable alternatives to conventional antivenom, or at least a feasible short-term solution that prevents excess toxin from reaching nAChRs immediately after snakebite. Our report on the identification of such snake toxin-binding peptides, which we term “serpentides”,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> the characterization of their effects on the physiological action of the toxin, and the high-resolution structural characterization of the peptide–toxin interaction should facilitate future endeavors in developing such molecules into therapies.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Phage Display Screening</h3><div class="NLM_p">Maleic anhydride activated 96-well plates (ThermoFisher Scientific) wells were incubated with 100 μL of 10.0 μM α-cobratoxin (≥99%, from <i>Naja kaouthia</i>, Latoxan) in phosphate buffered saline (PBS) or PBS alone (as control), capped with 200 μL 10% (v/v) ethanolamine, for 1 h at room temperature (RT, ∼21 °C) and then overnight at 4 °C. Subsequently, the wells were washed 3× with 200 μL of 0.05% Tween-20 (PBST) and 2× with 200 μL of PBS. Wells were then blocked by incubation in 100 μL blocking buffer (0.1 M NaHCO<sub>3</sub> pH 8.6, 0.02% NaN<sub>3</sub>, 0.1 μg/mL streptavidin (incidental, as the selection process did not utilize biotinylation), 5 mg/mL bovine serum albumin at RT on a plate shaker for 2 h. Wells were then washed 10× with PBST and 2× with PBS to remove excess blocking agent.</div><div class="NLM_p">For the first biopanning round, 2.5 μL of both TriCo-16 and TriCo-20 libraries (Creative Biolabs) were combined in 105 μL of blocking buffer and incubated for 30 min at RT. This phage solution (100 μL) was transferred to a blocked but not α-cobratoxin-coated MaxiSorp plate and incubated for 1 h at RT to remove unspecific binding phages. Unbound phage (100 μL) was then transferred to the α-cobratoxin-coated wells and incubated for 1 h at RT. The solution was then removed, and wells were washed 10x PBST and 2× with PBS. Bound phages were eluted by incubating for 10 min with triethylamine (200 μL, 100 mM) followed by neutralization with a solution of Tris-HCl (400 μL, 1 M, pH 7.4). Before each of four subsequent rounds of selection, the eluted phages were amplified in 2xYT medium (8–10 mL) overnight at 37 °C after infecting TG1 <i>Escherichia coli</i>. Amplified phages were precipitated using PEG<sub>6000</sub>/NaCl and concentrated in PBS (300 μL). Phages (115 μL) mixed with blocking buffer were incubated for 1 h in a MaxiSorp well before the next round of panning. The specific binder/control ratios (based on numbers of plaque forming units) were 24, 203, 755, and 175 for rounds two through five, indicating that specific α-Cbtx binders had been amplified (except in the final round).</div><div class="NLM_p last">Twenty-four single plaques were picked and amplified overnight, centrifuged for 10 min at 8000 g, and 125 μL supernatant was added to α-Cbtx-coated wells for overnight incubation. The solution was then removed, and wells washed, blocked, and washed again, and each eluate (100 μL in PBS) was mixed 1:1 with blocking buffer. 100 μL was added to α-Cbtx-coated (or control) wells, incubated for 1 h at RT on a plate shaker, and then washed 10× with 100 μL PBST and 2× with PBS. Bound phages were then incubated in 100 μL of a solution of monoclonal horseradish peroxidase (HRP) conjugated anti-M13 phage antibody (GE Healthcare) diluted 1:100 in blocking buffer, for 1 h on a plate shaker, washed again (10× with PBST, 2× with PBS), and each well was incubated in 100 μL HRP substrate (<i>ortho</i>-phenylenediamine (OPD, Dako, 2 mg in 3 mL water) with 5 μL H<sub>2</sub>O<sub>2</sub>) for ∼30 min. Reactions were stopped by the addition of 100 μL of 0.5 M H<sub>2</sub>SO<sub>4</sub>, and absorbance (490 nm) was measured (VersaMax Tunable microplate reader, Molecular Devices). Empty wells were used as blank controls for subtraction of signal. The 12 clones that gave the highest responses in the ELISA were selected for further processing. They were amplified overnight and PEG<sub>6000</sub>/NaCl precipitated. ssDNA was isolated and Sanger sequenced (Eurofins Genomics). Eleven encoded the same peptide, <b>1</b>, and the remaining encoded <b>2</b>, which is 75% identical to <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Deep Sequencing of Phage Libraries and Selection of an Additional Six Peptides for Testing</h3><div class="NLM_p">Amplified phage eluates from biopanning rounds one, two, three, and five were used for deep sequencing of variable regions along with rounds two, three, four and five for the control selections. The libraries after those rounds were PCR amplified with barcoded primers (based on the above phage libraries and according to suppliers’ instructions), gel purified, and sequencing adapters were ligated onto the DNA using the Ion Plus fragment library kit (ThermoFisher Scientific). Finally, the DNA was ligated to Ion Sphere Particles, subjected to emulsion PCR according to the Ion OneTouch 200 protocol. The Ion Sphere Particles were loaded onto an Ion 318 chip and sequenced in an Ion Torrent Personal Genome Machine system. The phage sequences were extracted, split according to barcodes and converted to peptide reads. For unknown reasons, the sequencing failed for the fifth biopanning round against α-Cbtx as well as the second biopanning round for the control. The number of peptides allocated to these samples were <400 compared to >100 000 for the remaining six samples. Accordingly, these samples were disregarded in further analyses. A total of 1,469,811 sequences were obtained representing an average of 224,969 per sample (range 110 503–499 660). The overall quality of the sequencing run was relatively low: if the quality (Phred) score cutoff was increased from 15 to 20, the approved number of sequences dropped to 674,989. Regarding the phage selection process, the number of unique peptides dropped with each subsequent round. Specifically, from 4938 to 410 for round one to three for the toxin-selected samples and similarly from 413 to 108 from rounds three to five in the control samples.</div><div class="NLM_p last">The most frequent peptide sequence identified in deep sequencing of the third selection round was identical to peptide <b>1</b>, a known binder from phage display/ELISA, suggesting that frequently occurring sequences in this round could be potential binders. Accordingly, three additional peptides (peptides <b>7</b>, <b>3</b>, and <b>5</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) ranking second through fourth in frequency in round three were selected for further studies. Another peptide, <b>8</b>, was included because it ranked seventh in round three and was consistently prevalent in the later rounds. The most frequent peptide in the first selection round, <b>6</b>, was also selected. Finally, a peptide ranking sixth in the third round, <b>4</b>, was included because it was highly similar to <b>8</b> differing only in 3 C-terminal amino acids out of the 20-mer peptide.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Electrophysiological Assays</h3><div class="NLM_p last">Stage V/VI <i>Xenopus laevis</i> oocytes were prepared as described previously<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and under license 2014-15-0201-00031 from the Danish Veterinary and Food Administration. Oocytes were injected with a volume of 40 nL containing 4 ng of cRNA for α7 nicotinic acetylcholine receptor (rat) carrying the L247V mutation to reduce desensitization<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and 4 ng of cRNA for the chaperone RIC-3 (human), which enhances acetylcholine receptor function.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> cRNAs were prepared and oocytes were maintained as described previously.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> One to 3 days after injecting, two electrode voltage clamp recordings were made as follows: A single oocyte was transferred to a custom recording chamber<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and continuously perfused with bath solution (in mM; 96 NaCl, 2 KCl, 1.8 CaCl<sub>2</sub>, 1 MgCl<sub>2</sub>, 5 HEPES, pH 7.4, with NaOH). The oocyte was impaled with borosillicate glass micropipettes containing 3 M KCl (resistance ∼0.5–1.5 MΩ), membrane voltage was clamped at −40 mV, and current recorded with OC-725C amplifier (Warner Instruments), Digidata 1550 digitizer, and pClamp 10 Software (Molecular Devices) at 1 kHz with 200 Hz filtering. Replicate experiments were generally performed on four individual oocytes (see <a class="ref internalNav" href="#sec5_7" aria-label="Quantification and Statistical Analysis">Quantification and Statistical Analysis</a> below). Solution exchange was performed with a ValveBank 8 perfusion system (AutoMate Scientific). ACh and α-Cbtx were prepared from stocks dissolved in water (100 mM and 10 μM, respectively, stored at −20 °C). Peptide (<b>1</b>–<b>18</b>) stocks of 10 mM were prepared in bath solution and typically used within 1 week (if not used in 1 day, these were stored at −20 °C). Peptides were mixed with α-Cbtx 20 min to 1.5 h before application to oocytes. Current amplitude was measured in Clampfit 10 (pClamp, Molecular Devices) and current amplitude was plotted and analyzed in GraphPad Prism 7.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Peptides</h3><div class="NLM_p last">All peptides used for subsequent assays, except <b>9</b>, were purchased from Schafer-N (Copenhagen, Denmark) or GenScript, ≥95% purity (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figures S5–S22</a>). The 5(6)-carboxytetramethylrhodamine (TAMRA)-labeled peptide <b>9</b> was synthesized using an automated peptide synthesizer and standard Fmoc (fluorenylmethoxycarbonyl)-based solid-phase peptide synthesis (SPPS) on Rink amide TentaGel resin (0.23 mmol/g) at a 0.02 mmol scale. Fmoc deprotections were performed in two steps: (1) piperidine in DMF (2:3, v/v) for 3 min and (2) piperidine in DMF (1:4, v/v) for 12 min. Deprotection steps were followed by washing with DMF (2 × 45 s), CH<sub>2</sub>Cl<sub>2</sub> (1 × 45 s), and DMF (2 × 45 s). Coupling steps were performed as double couplings with Fmoc-Xaa-OH (5.00 equiv to the resin loading), 2-(1<i>H</i>-benzotriazol-1-yl)-1,1,3,3-tetramethyluronoium hexafluorophosphate (HBTU) (4.90 equiv), and <i>i</i>-Pr<sub>2</sub>NEt in NMP (10.0 equiv, 2.0 M) in DMF (final concentration = 0.2 M for Fmoc-Xaa-OH) for 40 min for each coupling. Glycine was incorporated as <i>N</i>-α-Fmoc-<i>N</i>-α-(2,4-dimethoxybenzyl)-glycine (Fmoc-(Dmb)Gly-OH). TAMRA-acid (1.50 equiv) was coupled manually using 2-(1<i>H</i>-7-azabenzotriazol-1-yl)-1,1,3,3-tetramethyluronoium hexafluorphosphate (HATU) (1.50 equiv) and <i>i</i>-Pr<sub>2</sub>NEt (3.00 equiv) in DMF (final concentration = 0.06 M) for 16 h. Global deprotection and cleavage was conducted in the cleavage cocktail (TFA–<i>i-</i>Pr<sub>3</sub>SiH–water, 95:2.5:2.5, v/v/v) at room temperature (RT) for 2 h, followed by TFA evaporation and ether precipitation. The crude peptide was purified as single isomer by preparative reverse-phase high performance liquid chromatography (RP-HPLC) on a C8 Phenomenex Luna column (5 μm, 100 Å, 250 mm × 20 mm) using an Agilent 1260 LC system. Fractions were analyzed by matrix assisted laser desorption ionization–time-of-flight mass spectrometry (MALDI-TOF MS), and pure fractions were pooled and lyophilized. Purity was determined using an Agilent 1100 system equipped with C18 Phenomenex Luna column (2.6 μm, 100 Å, 150 mm × 4.60 mm). Analytical HPLC purity 95% (λ = 215 nm; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S13</a>). UPLC-MS <i>m</i>/<i>z</i> calcd for C<sub>124</sub>H<sub>157</sub>N<sub>27</sub>O<sub>30</sub>S<sub>2</sub><sup>2+</sup> [M + 2H]<sup>2+</sup>: 1284.55, found 1284.38; C<sub>124</sub>H<sub>158</sub>N<sub>27</sub>O<sub>30</sub>S<sub>2</sub><sup>3+</sup> [M + 3H]<sup>3+</sup>: 856.70, found 856.82; C<sub>124</sub>H<sub>159</sub>N<sub>27</sub>O<sub>30</sub>S<sub>2</sub><sup>4+</sup> [M + 4H]<sup>4+</sup>: 642.78, found 642.91 (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S13</a>).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Fluorescence Polarization (FP) Assays</h3><div class="NLM_p last">Protein concentrations and DMSO stock concentrations of unlabeled peptides were determined by UV absorbance (NanoDrop One<sup>C</sup>, ThermoFisher Scientific) at 280 nm. The DMSO stock of TAMRA ligand (<b>9</b>) was determined by weight of the corresponding TFA salt (MW + 5 × TFA = 3138 g/mol). Binding affinities were determined in a 384-well plate format (Corning Life Science) using a Safire 2 plate reader (Tecan). The instrument <i>G</i>-factor was calibrated to give an initial millipolarization at 20 (excitation at 530 nm; emission at 580 nm) and the instrumental <i>Z</i>-factor was adjusted to maximum fluorescence. All measurements were conducted in phosphate-buffered saline (PBS, pH = 7.4) at 25 °C. In a FP saturation assay, 50 nM of <b>9</b> was mixed with increasing concentrations (0.125–213 μM) of α-Cbtx. The resulting polarization was plotted as a function of the protein concentration and fitted to a one-site binding model GraphPad Prism 7.0 software to yield the <i>K</i><sub>d</sub> at 50% saturation. The binding experiment was performed as a single experiment with two replicates. FP competition experiments were conducted by mixing a preformed α-Cbtx:<b>9</b> complex at fixed concentration (40 μM/50 nM) with varying unlabeled peptide concentrations ranging from 0.2–1024 μM. The polarization (mP) was plotted as a function of peptide concentration and fitted to a sigmoidal dose response curve using GraphPad Prism 7 software. The <i>K</i><sub>i</sub> values were calculated according to the equations by Nikolovska-Coleska et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The <i>K</i><sub>i</sub> values were determined form three individual experiments performed with two replicates.<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m001.gif" alt="" id="_i19" /></img><span class="labelSpan">(1)</span></span><span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m002.gif" alt="" id="_i20" /></img><span class="labelSpan">(2)</span></span><span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m003.gif" alt="" id="_i21" /></img><span class="labelSpan">(3)</span></span><span class="NLM_disp-formula" id="eq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m004.gif" alt="" id="_i22" /></img><span class="labelSpan">(4)</span></span><span class="NLM_disp-formula" id="eq5"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m005.gif" alt="" id="_i23" /></img><span class="labelSpan">(5)</span></span><span class="NLM_disp-formula" id="eq5a"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m006.gif" alt="" id="_i24" /></img><span class="labelSpan">(6)</span></span><span class="NLM_disp-formula" id="eq6"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_m007.gif" alt="" id="_i25" /></img><span class="labelSpan">(7)</span></span><i>P</i>, protein (α-Cbtx); <i>L</i>, ligand (TAMRA-peptide <b>9</b>); <i>I</i>, inhibitor (peptide <b>1</b> or <b>12</b>); <i>K</i><sub>d</sub>, dissociation constant of the protein–ligand complex; [<i>P</i>]<sub>T</sub>, total protein concentration; [<i>L</i>]<sub>T</sub>, total ligand concentration; [<i>P</i>]<sub>0</sub>, unbound protein concentration at 0% inhibition; [<i>L</i>]<sub>0</sub>, unbound ligand concentration at 0% inhibition; [<i>PL</i>]<sub>0</sub>, protein–ligand complex concentration at 0% inhibition; [<i>P</i>]<sub>50</sub>, unbound protein concentration at 50% inhibition; [<i>L</i>]<sub>50</sub>, unbound ligand concentration at 50% inhibition; [<i>PL</i>]<sub>50</sub>, protein–ligand complex concentration at 50% inhibition; [<i>I</i>]<sub>50</sub>, unbound inhibitor concentration at 50% inhibition; IC<sub>50</sub>, half maximal inhibitory concentration; <i>K</i><sub>i</sub>, dissociation constant of the protein–inhibitor complex.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Crystallization and Structure Determination</h3><div class="NLM_p last">α-Cbtx (lyophilized powder, ≥99%, Latoxan) and <b>12</b> (≥95%, GenScript) were solubilized in 20 mM HEPES, pH 7.2, 100 mM NaCl at concentrations of 14.4 mg mL<sup>–1</sup> (1.87 mM) and 8.2 mg mL<sup>–1</sup> (5.5 mM), respectively, and mixed at 1:1.2 molar ratio. Crystallization trays were set up with a semiautomated pipetting robot (Gryphon, Art Robbins Instruments). Best crystals grew in 0.1 M carboxylic acids (sodium formate/ammonium acetate/sodium citrate/sodium potassium tartrate/sodium oxamate), 0.1 M HEPES/MOPS, pH 7.5, 30.0% v/v PEG 500 MME/PEG 20000 (Morpheus Crystallization Screen, Molecular Dimensions) and were directly flash frozen in liquid nitrogen. Data sets were collected at beamline MX14.2, BESSY Berlin<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and beamline P13, EMBL Hamburg, respectively. Diffraction data was processed with XDSAPP<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> and initial phases were obtained by molecular replacement using phenix.phaser<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> with the neurotoxin-1 from <i>Naja naja oxiana</i> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NTN">1NTN</a>,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> as a search model. A molecular model for the peptide portion of <b>12</b> could be built unambiguously into the remaining difference density. The final model was obtained through iterative cycles of model building in Coot<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and refinement with refmac/phenix.refine<a onclick="showRef(event, 'ref42 ref45'); return false;" href="javascript:void(0);" class="ref ref42 ref45">(42,45)</a> and is deposited at the protein database (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a>).</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Quantification and Statistical Analysis</h3><div class="NLM_p">In <i>Phage display screening</i>, sequencing identified only two peptide binders of α-Cbtx, and both were characterized further. In <i>Deep sequencing of phage libraries and selection of an additional six peptides for testing</i>, several more potential peptide binders were identified, but only six were selected for further characterization based on their amino acid sequence identity with proven binders from <i>Phage Display Screening</i>.</div><div class="NLM_p">Data from <i>Electrophysiological assays</i> are mean ± SEM (<i>n</i> = 2–4 oocytes in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B; <i>n</i> = 3 oocytes in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B; and <i>n</i> = 2–5 in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">Figure S1B</a>, as indicated in figure legends). We have chosen to show SEM error bars for the few groups where <i>n</i> = 2, as this shows the range of these data. To establish that, e.g., peptide <b>1</b> prevented α-Cbtx from inhibiting acetylcholine-gated currents through acetylcholine receptors, we compared the mean current amplitude (relative to control) in the presence of α-Cbtx to the mean current amplitude (relative to control) in the presence of both α-Cbtx and peptide <b>1</b>, using a two-tailed unpaired <i>t</i> test (GraphPad Prism).</div><div class="NLM_p last">In FP assays, the binding saturation with <b>9</b> and α-Cbtx was performed as a single experiment with two replicates. For competition with <b>1</b> and <b>12</b>, <i>K</i><sub>i</sub> values were determined from three individual experiments performed with two replicates. Data are given as mean ± SEM <i>K</i><sub>i</sub> values were calculated according to Nikolovska-Coleska et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and compared using a two-tailed unpaired <i>t</i> test (GraphPad Prism).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01202" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01202?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01202</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Effects of additional peptides on α-cobratoxin inhibition of nAChRs; chemical stability of peptides <b>1</b> and <b>12</b>; architecture of the <b>12</b>:α-Cbtx complex observed in the crystal structure; structurally homologous α-neurotoxin binding interactions with other proteins; HPLC chromatograms and MS spectra; PDB file <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a> (<b>12</b>:α-Cbtx X-ray cocrystal structure) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf">jm0c01202_si_001.pdf (2.73 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the 12:α-Cbtx X-ray cocrystal structure have been deposited in the RCSB Protein Data Bank with accession number PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01202" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy Lynagh</span> - <span class="hlFld-Affiliation affiliation">Sars
International Centre for Marine Molecular Biology, University of Bergen, Thormøhlensgate 55, 5008 Bergen, Norway</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Biopharmaceuticals & Department of Drug Design and Pharmacology,
Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4888-4098" title="Orcid link">http://orcid.org/0000-0003-4888-4098</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#88fce1e5a6e4f1e6e9efe0c8fde1eaa6e6e7"><span class="__cf_email__" data-cfemail="35415c581b594c5b54525d75405c571b5b5a">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian Lohse</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9efcecf7f2deedebf0fab0f5ebb0faf5"><span class="__cf_email__" data-cfemail="0466766d684477716a602a6f712a606f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephan Kiontke</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Department of Biology/Chemistry, University of Osnabrück, Barbarastraße 13, Osnabrück 49076, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Faculty
of Biology, Department of Plant Physiology and Photobiology, Philipps-Universität Marburg, Karl-von-Frisch-Straße 8, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Meyhoff-Madsen</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bengt H. Gless</span> - <span class="hlFld-Affiliation affiliation">Center
for Biopharmaceuticals & Department of Drug Design and Pharmacology,
Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jónas Johannesen</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sabrina Kattelmann</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, University of Münster, Corrensstraße 36, 48149 Münster, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anders Christiansen</span> - <span class="hlFld-Affiliation affiliation">Fluid
Array Systems and Technology, Nano and Bio-physical Systems, Department
of Health Technology, Technical University
of Denmark, Building 423 Produktionstorvet, DK-2800 Kongens Lyngby, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Dufva</span> - <span class="hlFld-Affiliation affiliation">Fluid
Array Systems and Technology, Nano and Bio-physical Systems, Department
of Health Technology, Technical University
of Denmark, Building 423 Produktionstorvet, DK-2800 Kongens Lyngby, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas H. Laustsen</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span>Present Address:
                        A.H.L.: Department of Biotechnology and Biomedicine, Building 224 Søltofts Plads, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kanchan Devkota</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian A. Olsen</span> - <span class="hlFld-Affiliation affiliation">Center
for Biopharmaceuticals & Department of Drug Design and Pharmacology,
Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2953-8942" title="Orcid link">http://orcid.org/0000-0002-2953-8942</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Kümmel</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Department of Biology/Chemistry, University of Osnabrück, Barbarastraße 13, Osnabrück 49076, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, University of Münster, Corrensstraße 36, 48149 Münster, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephan Alexander Pless</span> - <span class="hlFld-Affiliation affiliation">Center
for Biopharmaceuticals & Department of Drug Design and Pharmacology,
Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6654-114X" title="Orcid link">http://orcid.org/0000-0001-6654-114X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>B.L. conceived the study; T.L., B.H.G., A.C., M.D., K.D., A.H.L., C.A.O., D.K., S.A.P. and B.L. designed experiments; B.L. and B.H.G. performed peptide synthesis; B.L. and J.J. performed phage display; B.H.G. and M.M.-M. performed fluorescence polarization assays; T.L. performed electrophysiology; A.C. performed deep sequencing; S. Kiontke, S. Kattelmann, and D.K. performed X-ray crystallography; T.L., B.H.G., A.H.L., K.D., D.K., and B.L. analyzed the data. M.D., A.H.L., C.A.O., D.K., S.A.P. and B.L. provided supervision. T.L., D.K., and B.L. wrote the manuscript with input from all other authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Brian Lohse is owner of Serpentides and therefore declares a commercial interest in the development of peptide-based antivenoms.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16825" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16825" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Carlsberg Foundation grant 2013_01_0333 and Lundbeck Foundation grant R289-2018-2074 to C.A.O., Lundbeck Foundation Fellowship R139-2012-12390 to S.A.P., and the Tech Transfer Office, University of Copenhagen (to B.L.). We thank Morten Jørgensen, Brian Bentzen, and Jonas Caspersen, at Terrariet, Vissenbjerg, Denmark, for <i>Naja kaouthia</i> venom used in preliminary exploratory studies.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AChBP</td><td class="NLM_def"><p class="first last">acetylcholine binding protein</p></td></tr><tr><td class="NLM_term">nAChRs</td><td class="NLM_def"><p class="first last">acetylcholine receptors</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">polyethelene glycol</p></td></tr><tr><td class="NLM_term">TAMRA</td><td class="NLM_def"><p class="first last">5(6)-carboxytetramethylrhodamine</p></td></tr><tr><td class="NLM_term">α-Bgtx</td><td class="NLM_def"><p class="first last">α bungarotoxin</p></td></tr><tr><td class="NLM_term">α-Cbtx</td><td class="NLM_def"><p class="first last">α cobratoxin</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 45 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvete, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habib, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrell, D. A.</span></span> <span> </span><span class="NLM_article-title">Snakebite envenoming</span>. <i>Nat. Rev. Dis. Primers</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">17063</span>, <span class="refDoi"> DOI: 10.1038/nrdp.2017.63</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fnrdp.2017.63" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28905944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BC1cbotFSmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=17063&author=J.+M.+Guti%C3%A9rrezauthor=J.+J.+Calveteauthor=A.+G.+Habibauthor=R.+A.+Harrisonauthor=D.+J.+Williamsauthor=D.+A.+Warrell&title=Snakebite+envenoming&doi=10.1038%2Fnrdp.2017.63"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Snakebite envenoming</span></div><div class="casAuthors">Gutierrez Jose Maria; Calvete Juan J; Habib Abdulrazaq G; Harrison Robert A; Williams David J; Williams David J; Warrell David A</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Disease primers</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17063</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Snakebite envenoming is a neglected tropical disease that kills >100,000 people and maims >400,000 people every year.  Impoverished populations living in the rural tropics are particularly vulnerable; snakebite envenoming perpetuates the cycle of poverty.  Snake venoms are complex mixtures of proteins that exert a wide range of toxic actions.  The high variability in snake venom composition is responsible for the various clinical manifestations in envenomings, ranging from local tissue damage to potentially life-threatening systemic effects.  Intravenous administration of antivenom is the only specific treatment to counteract envenoming.  Analgesics, ventilator support, fluid therapy, haemodialysis and antibiotic therapy are also used.  Novel therapeutic alternatives based on recombinant antibody technologies and new toxin inhibitors are being explored.  Confronting snakebite envenoming at a global level demands the implementation of an integrated intervention strategy involving the WHO, the research community, antivenom manufacturers, regulatory agencies, national and regional health authorities, professional health organizations, international funding agencies, advocacy groups and civil society institutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0ERa1BdrKexBrI9DTGjC0fW6udTcc2ebMOeQKaENfRrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbotFSmtA%253D%253D&md5=954c284c2d68cb247bac05af03df727a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrdp.2017.63&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrdp.2017.63%26sid%3Dliteratum%253Aachs%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DCalvete%26aufirst%3DJ.%2BJ.%26aulast%3DHabib%26aufirst%3DA.%2BG.%26aulast%3DHarrison%26aufirst%3DR.%2BA.%26aulast%3DWilliams%26aufirst%3DD.%2BJ.%26aulast%3DWarrell%26aufirst%3DD.%2BA.%26atitle%3DSnakebite%2520envenoming%26jtitle%3DNat.%2520Rev.%2520Dis.%2520Primers%26date%3D2017%26volume%3D3%26spage%3D17063%26doi%3D10.1038%2Fnrdp.2017.63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohapatra, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suraweera, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhingra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jotkar, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitaker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, P.</span></span> <span> </span><span class="NLM_article-title">Million Death Study, C., Snakebite mortality in India: a nationally representative mortality survey</span>. <i>PLoS Neglected Trop. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">4</span>), <span class="NLM_elocation-id">e1018</span> <span class="refDoi"> DOI: 10.1371/journal.pntd.0001018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1371%2Fjournal.pntd.0001018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=21532748" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&issue=4&author=B.+Mohapatraauthor=D.+A.+Warrellauthor=W.+Suraweeraauthor=P.+Bhatiaauthor=N.+Dhingraauthor=R.+M.+Jotkarauthor=P.+S.+Rodriguezauthor=K.+Mishraauthor=R.+Whitakerauthor=P.+Jha&title=Million+Death+Study%2C+C.%2C+Snakebite+mortality+in+India%3A+a+nationally+representative+mortality+survey&doi=10.1371%2Fjournal.pntd.0001018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pntd.0001018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pntd.0001018%26sid%3Dliteratum%253Aachs%26aulast%3DMohapatra%26aufirst%3DB.%26aulast%3DWarrell%26aufirst%3DD.%2BA.%26aulast%3DSuraweera%26aufirst%3DW.%26aulast%3DBhatia%26aufirst%3DP.%26aulast%3DDhingra%26aufirst%3DN.%26aulast%3DJotkar%26aufirst%3DR.%2BM.%26aulast%3DRodriguez%26aufirst%3DP.%2BS.%26aulast%3DMishra%26aufirst%3DK.%26aulast%3DWhitaker%26aufirst%3DR.%26aulast%3DJha%26aufirst%3DP.%26atitle%3DMillion%2520Death%2520Study%252C%2520C.%252C%2520Snakebite%2520mortality%2520in%2520India%253A%2520a%2520nationally%2520representative%2520mortality%2520survey%26jtitle%3DPLoS%2520Neglected%2520Trop.%2520Dis.%26date%3D2011%26volume%3D5%26issue%3D4%26doi%3D10.1371%2Fjournal.pntd.0001018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grace, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schütte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babo Martins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz de Castañeda, R.</span></span> <span> </span><span class="NLM_article-title">Snakebite in domestic animals: first global scoping review</span>. <i>Preventive Veterinary Medicine</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">104729</span>, <span class="refDoi"> DOI: 10.1016/j.prevetmed.2019.104729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.prevetmed.2019.104729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=31421490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BB3MvovVKhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2019&pages=104729&author=I.+Bolonauthor=M.+Finatauthor=M.+Herreraauthor=A.+Nickersonauthor=D.+Graceauthor=S.+Sch%C3%BCtteauthor=S.+Babo+Martinsauthor=R.+Ruiz+de+Casta%C3%B1eda&title=Snakebite+in+domestic+animals%3A+first+global+scoping+review&doi=10.1016%2Fj.prevetmed.2019.104729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Snakebite in domestic animals: First global scoping review</span></div><div class="casAuthors">Bolon Isabelle; Finat Matias; Babo Martins Sara; Ruiz de Castaneda Rafael; Herrera Maria; Nickerson Andrea; Grace Delia; Schutte Stephanie</div><div class="citationInfo"><span class="NLM_cas:title">Preventive veterinary medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104729</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Snakebite is a Neglected Tropical Disease estimated to cause more than 100,000 human deaths and disable more than 400,000 victims each year.  It primarily affects poor agricultural workers, farmers, and cattle herders living in rural areas of developing countries.  It is thus an occupational disease.  However, the impact of snakebite on these rural communities could be even higher if a One Health approach is taken to consider the direct impact on domestic animals and indirect impact on the livelihood of affected communities.  To explore this hypothesis we developed the first scoping review to identify and characterize the global literature on snakebite in domestic animals.  Three bibliographic databases (PubMed, Web of Science and Agricola) were searched using terms related to snake, snakebite and domestic animals for publications up to December 31(st), 2016.  Two independent reviewers screened publications applying inclusion/exclusion criteria to select relevant material.  Relevant information was then extracted from the selected literature.  The global literature on snakebite in domestic animals (n = 143 observational studies, reviews and letters) mainly represented North America, Europe and Australia (69%) with less information from Central and South America, Asia and Africa (31%).  Observational studies (n = 119) mostly concerned pets (78%) and to a lesser extent livestock (22%).  Thirty-four snake species were reported as responsible for bites in domestic animals.  WHO's Medically Important Venomous Snakes were more frequently involved.  The social-ecological determinants of snakebite were poorly documented but the reviewed literature suggested a strong seasonality and a diversity of habitat.  Snakebite in animals caused neurotoxic, cytotoxic and hemotoxic envenomation syndromes similar to humans and death.  Half of publications on envenomed livestock reported a fatality rate above 47%.  There was no literature on the indirect impact of snakebite on livelihood caused by animal morbidity and mortality.  The results of this scoping review suggest a high and under-reported burden in terms of mortality in animals and a potential economic impact of snakebite in terms of losses in livelihood of affected communities.  However, major knowledge gaps with respect to impact of snakebite on livestock and livelihood were identified.  Filling these gaps is necessary for a full understanding of the impact of snakebite and to raise scientific, political and public awareness on this neglected issue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfSxzB4H-OUb5XHngOXiV_fW6udTcc2ebMOeQKaENfRrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvovVKhsQ%253D%253D&md5=bf7ede624000ddffe870f0acba39ed45</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.prevetmed.2019.104729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.prevetmed.2019.104729%26sid%3Dliteratum%253Aachs%26aulast%3DBolon%26aufirst%3DI.%26aulast%3DFinat%26aufirst%3DM.%26aulast%3DHerrera%26aufirst%3DM.%26aulast%3DNickerson%26aufirst%3DA.%26aulast%3DGrace%26aufirst%3DD.%26aulast%3DSch%25C3%25BCtte%26aufirst%3DS.%26aulast%3DBabo%2BMartins%26aufirst%3DS.%26aulast%3DRuiz%2Bde%2BCasta%25C3%25B1eda%26aufirst%3DR.%26atitle%3DSnakebite%2520in%2520domestic%2520animals%253A%2520first%2520global%2520scoping%2520review%26jtitle%3DPreventive%2520Veterinary%2520Medicine%26date%3D2019%26volume%3D170%26spage%3D104729%26doi%3D10.1016%2Fj.prevetmed.2019.104729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Silva, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, H. J.</span></span> <span> </span><span class="NLM_article-title">Adverse reactions to snake antivenom, and their prevention and treatment</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>81</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1111/bcp.12739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1111%2Fbcp.12739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=26256124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BC287htFGgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2016&pages=446-452&issue=3&author=H.+A.+de%0ASilvaauthor=N.+M.+Ryanauthor=H.+J.+de+Silva&title=Adverse+reactions+to+snake+antivenom%2C+and+their+prevention+and+treatment&doi=10.1111%2Fbcp.12739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Adverse reactions to snake antivenom, and their prevention and treatment</span></div><div class="casAuthors">de Silva H Asita; de Silva H Janaka; Ryan Nicole M</div><div class="citationInfo"><span class="NLM_cas:title">British journal of clinical pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-52</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Antivenom is the mainstay of treatment of snakebite envenoming.  However, adverse reactions to snake antivenom that is available are common in many parts of the world where snakebite is prevalent.  Both acute (anaphylactic or pyrogenic) and delayed (serum sickness type) reactions occur.  Acute reactions are usually mild but severe systemic anaphylaxis may develop, often within an hour or so of exposure to antivenom.  Serum sickness after antivenom has a delayed onset between 5 and 14 days after its administration.  Ultimately, the prevention reactions will depend mainly on improving the quality of antivenom.  Until these overdue improvements take place, doctors will have to depend on pharmacological prophylaxis, where the search for the best prophylactic agent is still on-going, as well as careful observation of patients receiving antivenom in preparation for prompt management of acute as well as delayed reactions when they occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTuLdwobba-MjAEfvC7EH1LfW6udTcc2eaY0MH_Quwun7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287htFGgug%253D%253D&md5=b0d685d2386759e707520aeda7eae050</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fbcp.12739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcp.12739%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSilva%26aufirst%3DH.%2BA.%26aulast%3DRyan%26aufirst%3DN.%2BM.%26aulast%3Dde%2BSilva%26aufirst%3DH.%2BJ.%26atitle%3DAdverse%2520reactions%2520to%2520snake%2520antivenom%252C%2520and%2520their%2520prevention%2520and%2520treatment%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2016%26volume%3D81%26issue%3D3%26spage%3D446%26epage%3D452%26doi%3D10.1111%2Fbcp.12739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">León, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span> <span> </span><span class="NLM_article-title">Current technology for the industrial manufacture of snake antivenoms</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2018.06.084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2018.06.084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=29959968" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1CnurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=63-73&author=G.+Le%C3%B3nauthor=M.+Vargasauthor=%C3%81.+Seguraauthor=M.+Herreraauthor=M.+Villaltaauthor=A.+S%C3%A1nchezauthor=G.+Solanoauthor=A.+G%C3%B3mezauthor=M.+S%C3%A1nchezauthor=R.+Estradaauthor=J.+M.+Guti%C3%A9rrez&title=Current+technology+for+the+industrial+manufacture+of+snake+antivenoms&doi=10.1016%2Fj.toxicon.2018.06.084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Current technology for the industrial manufacture of snake antivenoms</span></div><div class="casAuthors">Leon, Guillermo; Vargas, Mariangela; Segura, Alvaro; Herrera, Maria; Villalta, Mauren; Sanchez, Andres; Solano, Gabriela; Gomez, Aaron; Sanchez, Melvin; Estrada, Ricardo; Gutierrez, Jose Maria</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-73</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Snake antivenoms are formulations of animal Igs used in the treatment of snakebite envenomation.  The general scheme for producing snake antivenoms has undergone few changes since its development more than a century ago; however, technol. innovations have been introduced in the manufg. process.  These medicines must comply with identity, purity, safety and efficacy profiles, as requested by the current Good Manufg. Practices (GMPs) applied to modern biopharmaceutical drugs.  Industrial prodn. of snake antivenoms comprises several stages, such as: 1. prodn. of ref. venom pools, 2. prodn. of hyperimmune plasma, 3. purifn. of the antivenom Igs, 4. formulation of the antivenom, 5. stabilization of the formulation, and 6. quality control of in-process and final products.  In this work, a general review of the existing technol. used for the industrial manuf. of snake antivenoms is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzTJNIEQUD1rVg90H21EOLACvtfcHk0lhCYmMp4g8PHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1CnurrM&md5=a651ddb9a1051cafef83dd1b180fa1bc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2018.06.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2018.06.084%26sid%3Dliteratum%253Aachs%26aulast%3DLe%25C3%25B3n%26aufirst%3DG.%26aulast%3DVargas%26aufirst%3DM.%26aulast%3DSegura%26aufirst%3D%25C3%2581.%26aulast%3DHerrera%26aufirst%3DM.%26aulast%3DVillalta%26aufirst%3DM.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DA.%26aulast%3DSolano%26aufirst%3DG.%26aulast%3DG%25C3%25B3mez%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DM.%26aulast%3DEstrada%26aufirst%3DR.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26atitle%3DCurrent%2520technology%2520for%2520the%2520industrial%2520manufacture%2520of%2520snake%2520antivenoms%26jtitle%3DToxicon%26date%3D2018%26volume%3D151%26spage%3D63%26epage%3D73%26doi%3D10.1016%2Fj.toxicon.2018.06.084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span> </span><span class="NLM_article-title">Snake bite—the neglected tropical disease</span>.  <i>Lancet</i>, <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">386</span> (<span class="NLM_issue">9999</span>)  <span class="NLM_fpage">1110</span>.<span class="refDoi"> DOI: 10.1016/S0140-6736(15)00247-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2FS0140-6736%2815%2900247-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Snake+bite%E2%80%94the+neglected+tropical+disease.+Lancet%2C+2015%2C+386+%289999%29+1110.+10.1016%2FS0140-6736%2815%2900247-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2900247-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252900247-0%26sid%3Dliteratum%253Aachs%26atitle%3DSnake%2520bite%25E2%2580%2594the%2520neglected%2520tropical%2520disease%26jtitle%3DLancet%26date%3D2015%26volume%3D386%26issue%3D9999%26spage%3D1110%26doi%3D10.1016%2FS0140-6736%2815%2900247-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laustsen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engmark, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milbo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannesen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomonte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, B.</span></span> <span> </span><span class="NLM_article-title">From fangs to pharmacology: The future of snakebite envenoming therapy</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">34</span>),  <span class="NLM_fpage">5270</span>– <span class="NLM_lpage">5293</span>, <span class="refDoi"> DOI: 10.2174/1381612822666160623073438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.2174%2F1381612822666160623073438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=27339430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOhu7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5270-5293&issue=34&author=A.+H.+Laustsenauthor=M.+Engmarkauthor=C.+Milboauthor=J.+Johannesenauthor=B.+Lomonteauthor=J.+M.+Guti%C3%A9rrezauthor=B.+Lohse&title=From+fangs+to+pharmacology%3A+The+future+of+snakebite+envenoming+therapy&doi=10.2174%2F1381612822666160623073438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">From Fangs to Pharmacology: The Future of Snakebite Envenoming Therapy</span></div><div class="casAuthors">Laustsen, Andreas H.; Engmark, Mikael; Milbo, Christina; Johannesen, Jonas; Lomonte, Bruno; Gutierrez, Jose Maria; Lohse, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">5270-5293</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The snake is the symbol of medicine due to its assocn. with Asclepius, the Greek god of medicine, and so with good reasons.  More than 725 species of venomous snakes have toxins specifically evolved to exert potent bioactivity in prey or victims, and snakebites constitute a public health hazard of high impact in Asia, Africa, Latin America, and parts of Oceania.  Parenteral administration of antivenoms is the mainstay in snakebite envenoming therapy.  However, despite well-demonstrated efficacy and safety of many antivenoms worldwide, they are still being produced by traditional animal immunization procedures, and therefore present a no. of drawbacks.  Technol. advances within biopharmaceutical development and medicinal chem. could pave the way for rational drug design approaches against snake toxins.  This could minimize the use of animals and bring forward more effective therapies for snakebite envenomings.  In this review, current state-of-the-art in biopharmaceutical antitoxin development is presented together with an overview of available bioinformatics and structural data on snake venom toxins.  This growing body of scientific and technol. tools could define the basis for introducing a rational drug design approach into the field of snakebite envenoming therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozSZqbnE51C7Vg90H21EOLACvtfcHk0lgSNgC6U8fQQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOhu7%252FP&md5=c8815e3930a53d46e60e2cd1b0287de3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1381612822666160623073438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612822666160623073438%26sid%3Dliteratum%253Aachs%26aulast%3DLaustsen%26aufirst%3DA.%2BH.%26aulast%3DEngmark%26aufirst%3DM.%26aulast%3DMilbo%26aufirst%3DC.%26aulast%3DJohannesen%26aufirst%3DJ.%26aulast%3DLomonte%26aufirst%3DB.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DLohse%26aufirst%3DB.%26atitle%3DFrom%2520fangs%2520to%2520pharmacology%253A%2520The%2520future%2520of%2520snakebite%2520envenoming%2520therapy%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2016%26volume%3D22%26issue%3D34%26spage%3D5270%26epage%3D5293%26doi%3D10.2174%2F1381612822666160623073438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layfield, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallance, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicknell, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trim, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaiyapuri, S.</span></span> <span> </span><span class="NLM_article-title">The urgent need to develop novel strategies for the diagnosis and treatment of snakebites</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">363</span>, <span class="refDoi"> DOI: 10.3390/toxins11060363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.3390%2Ftoxins11060363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlGn" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=363&issue=6&author=H.+F.+Williamsauthor=H.+J.+Layfieldauthor=T.+Vallanceauthor=K.+Patelauthor=A.+B.+Bicknellauthor=S.+A.+Trimauthor=S.+Vaiyapuri&title=The+urgent+need+to+develop+novel+strategies+for+the+diagnosis+and+treatment+of+snakebites&doi=10.3390%2Ftoxins11060363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The urgent need to develop novel strategies for the diagnosis and treatment of snakebites</span></div><div class="casAuthors">Williams, Harry F.; Layfield, Harry J.; Vallance, Thomas; Patel, Ketan; Bicknell, Andrew B.; Trim, Steven A.; Vaiyapuri, Sakthivel</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">363</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Snakebite envenoming (SBE) is a priority neglected tropical disease, which kills in excess of 100,000 people per yr.  Addnl., many millions of survivors also suffer through disabilities and long-term health consequences.  The only treatment for SBE, antivenom, has a no. of major assocd. problems, not least, adverse reactions and limited availability.  This emphasizes the necessity for urgent improvements to the management of this disease.  Administration of antivenom is too frequently based on symptomatol., which results in wasting crucial time.  The majority of SBE-affected regions rely on broad-spectrum polyvalent antivenoms that have a low content of case-specific efficacious Igs.  Research into small mol. therapeutics such as varespladib/methyl-varespladib (PLA2 inhibitors) and batimastat/marimastat (metalloprotease inhibitors) suggest that such adjunctive treatments could be hugely beneficial to victims.  Progress into toxin-specific monoclonal antibodies as well as alternative binding scaffolds such as aptamers hold much promise for future treatment strategies.  SBE is not implicit during snakebite, due to venom metering.  Thus, the delay between bite and symptom presentation is crit. and when symptoms appear it may often already be too late to effectively treat SBE.  The development of reliable diagnostical tools could therefore initiate a paradigm shift in the treatment of SBE.  While the complete eradication of SBE is an impossibility, mitigation is in the pipeline, with new treatments and diagnostics rapidly emerging.  Here we critically review the urgent necessity for the development of diagnostic tools and improved therapeutics to mitigate the deaths and disabilities caused by SBE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov52jwUZ73CLVg90H21EOLACvtfcHk0lgSNgC6U8fQQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlGn&md5=07ebd90157a62fc89624d7e8b3450fbb</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3390%2Ftoxins11060363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins11060363%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DH.%2BF.%26aulast%3DLayfield%26aufirst%3DH.%2BJ.%26aulast%3DVallance%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DBicknell%26aufirst%3DA.%2BB.%26aulast%3DTrim%26aufirst%3DS.%2BA.%26aulast%3DVaiyapuri%26aufirst%3DS.%26atitle%3DThe%2520urgent%2520need%2520to%2520develop%2520novel%2520strategies%2520for%2520the%2520diagnosis%2520and%2520treatment%2520of%2520snakebites%26jtitle%3DToxins%26date%3D2019%26volume%3D11%26issue%3D6%26spage%3D363%26doi%3D10.3390%2Ftoxins11060363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Modahl, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackessy, S. P.</span></span> <span> </span><span class="NLM_article-title">An analysis of venom ontogeny and prey-specific toxicity in the Monocled Cobra (Naja kaouthia)</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2016.04.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2016.04.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=27163885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFegt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2016&pages=8-20&author=C.+M.+Modahlauthor=A.+K.+Mukherjeeauthor=S.+P.+Mackessy&title=An+analysis+of+venom+ontogeny+and+prey-specific+toxicity+in+the+Monocled+Cobra+%28Naja+kaouthia%29&doi=10.1016%2Fj.toxicon.2016.04.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of venom ontogeny and prey-specific toxicity in the Monocled Cobra (Naja kaouthia)</span></div><div class="casAuthors">Modahl, Cassandra M.; Mukherjee, Ashis K.; Mackessy, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8-20</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Venoms of snakes of the family Elapidae (cobras, kraits, mambas, and relatives) are predominantly composed of numerous phospholipases A2 (PLA2s) and three-finger toxins (3FTxs), some of which are lethal while others are not significantly toxic.  Currently, the only identified prey-specific toxins are several nonconventional 3FTxs, and given the large diversity of 3FTxs within Monocled Cobra (Naja kaouthia) venom, it was hypothesized that several 3FTxs, previously found to be non-toxic or weakly toxic 3FTxs in murine models, could potentially be toxic towards non-murine prey.  Addnl., it was hypothesized that ontogenetic dietary shifts will be correlated with observable changes in specific 3FTx isoform abundance.  Adult and juvenile N. kaouthia venom compn. was investigated using ion-exchange FPLC, 1D and 2D SDS-PAGE, mass spectrometry, and various enzymic and LD50 assays.  Alpha-cobratoxin (α-elapitoxin) was the only significantly toxic (LD50 < 1 μg/g) 3FTx found in N. kaouthia venom and was equally toxic toward both lizard and mouse models.  The abundance and diversity of 3FTxs and most enzyme activities did not vary between adult and juvenile cobra venoms; however, total venom PLA2 activity and specific PLA2 isoforms did vary, with juveniles lacking several of the least acidic PLA2s, and these differences could have both biol. (related to predation) and clin. (antivenom efficacy) implications.  Nevertheless, the ubiquitous presence of α-cobratoxin in both adult and juvenile cobra venoms, with high toxicity toward both reptiles and mammals, represents a venom compositional strategy wherein a single potent toxin effectively immobilizes a variety of prey types encountered across life history stages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdhLcg3rSqyrVg90H21EOLACvtfcHk0lgSNgC6U8fQQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFegt7o%253D&md5=06e89a0f850d772f1352b30a71bf71ea</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2016.04.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2016.04.049%26sid%3Dliteratum%253Aachs%26aulast%3DModahl%26aufirst%3DC.%2BM.%26aulast%3DMukherjee%26aufirst%3DA.%2BK.%26aulast%3DMackessy%26aufirst%3DS.%2BP.%26atitle%3DAn%2520analysis%2520of%2520venom%2520ontogeny%2520and%2520prey-specific%2520toxicity%2520in%2520the%2520Monocled%2520Cobra%2520%2528Naja%2520kaouthia%2529%26jtitle%3DToxicon%26date%3D2016%26volume%3D119%26spage%3D8%26epage%3D20%26doi%3D10.1016%2Fj.toxicon.2016.04.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suryamohan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnankutty, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillory, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jevit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriakose, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, O. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perumal, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koludarov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velayutham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muraleedharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faherty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Rosa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagolla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vucic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannoush, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durinck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrusan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stawiski, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raudsepp, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kini, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zachariah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seshagiri, S.</span></span> <span> </span><span class="NLM_article-title">The Indian cobra reference genome and transcriptome enables comprehensive identification of venom toxins</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1038/s41588-019-0559-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fs41588-019-0559-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=31907489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjvF2ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2020&pages=106-117&issue=1&author=K.+Suryamohanauthor=S.+P.+Krishnankuttyauthor=J.+Guilloryauthor=M.+Jevitauthor=M.+S.+Schr%C3%B6derauthor=M.+Wuauthor=B.+Kuriakoseauthor=O.+K.+Mathewauthor=R.+C.+Perumalauthor=I.+Koludarovauthor=L.+D.+Goldsteinauthor=K.+Sengerauthor=M.+D.+Dixonauthor=D.+Velayuthamauthor=D.+Vargasauthor=S.+Chaudhuriauthor=M.+Muraleedharanauthor=R.+Goelauthor=Y.-J.+J.+Chenauthor=A.+Ratanauthor=P.+Liuauthor=B.+Fahertyauthor=G.+de+la+Rosaauthor=H.+Shibataauthor=M.+Bacaauthor=M.+Sagollaauthor=J.+Ziaiauthor=G.+A.+Wrightauthor=D.+Vucicauthor=S.+Mohanauthor=A.+Antonyauthor=J.+Stinsonauthor=D.+S.+Kirkpatrickauthor=R.+N.+Hannoushauthor=S.+Durinckauthor=Z.+Modrusanauthor=E.+W.+Stawiskiauthor=K.+Wileyauthor=T.+Raudseppauthor=R.+M.+Kiniauthor=A.+Zachariahauthor=S.+Seshagiri&title=The+Indian+cobra+reference+genome+and+transcriptome+enables+comprehensive+identification+of+venom+toxins&doi=10.1038%2Fs41588-019-0559-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Indian cobra reference genome and transcriptome enables comprehensive identification of venom toxins</span></div><div class="casAuthors">Suryamohan, Kushal; Krishnankutty, Sajesh P.; Guillory, Joseph; Jevit, Matthew; Schroder, Markus S.; Wu, Meng; Kuriakose, Boney; Mathew, Oommen K.; Perumal, Rajadurai C.; Koludarov, Ivan; Goldstein, Leonard D.; Senger, Kate; Dixon, Mandumpala Davis; Velayutham, Dinesh; Vargas, Derek; Chaudhuri, Subhra; Muraleedharan, Megha; Goel, Ridhi; Chen, Ying-Jiun J.; Ratan, Aakrosh; Liu, Peter; Faherty, Brendan; de la Rosa, Guillermo; Shibata, Hiroki; Baca, Miriam; Sagolla, Meredith; Ziai, James; Wright, Gus A.; Vucic, Domagoj; Mohan, Sangeetha; Antony, Aju; Stinson, Jeremy; Kirkpatrick, Donald S.; Hannoush, Rami N.; Durinck, Steffen; Modrusan, Zora; Stawiski, Eric W.; Wiley, Kristen; Raudsepp, Terje; Kini, R. Manjunatha; Zachariah, Arun; Seshagiri, Somasekar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106-117</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Snakebite envenoming is a serious and neglected tropical disease that kills ∼100,000 people annually.  High-quality, genome-enabled comprehensive characterization of toxin genes will facilitate development of effective humanized recombinant antivenom.  We report a de novo near-chromosomal genome assembly of Naja naja, the Indian cobra, a highly venomous, medically important snake.  Our assembly has a scaffold N50 of 223.35 Mb, with 19 scaffolds contg. 95% of the genome.  Of the 23,248 predicted protein-coding genes, 12,346 venom-gland-expressed genes constitute the 'venom-ome' and this included 139 genes from 33 toxin families.  Among the 139 toxin genes were 19 'venom-ome-specific toxins' (VSTs) that showed venom-gland-specific expression, and these probably encode the minimal core venom effector proteins.  Synthetic venom reconstituted through recombinant VST expression will aid in the rapid development of safe and effective synthetic antivenom.  Addnl., our genome could serve as a ref. for snake genomes, support evolutionary studies and enable venom-driven drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVc9xuao9CibVg90H21EOLACvtfcHk0lj9PLY9HmoYSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjvF2ksQ%253D%253D&md5=8f5d4f08e49153d68bdebfb675832b00</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41588-019-0559-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-019-0559-8%26sid%3Dliteratum%253Aachs%26aulast%3DSuryamohan%26aufirst%3DK.%26aulast%3DKrishnankutty%26aufirst%3DS.%2BP.%26aulast%3DGuillory%26aufirst%3DJ.%26aulast%3DJevit%26aufirst%3DM.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%2BS.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DKuriakose%26aufirst%3DB.%26aulast%3DMathew%26aufirst%3DO.%2BK.%26aulast%3DPerumal%26aufirst%3DR.%2BC.%26aulast%3DKoludarov%26aufirst%3DI.%26aulast%3DGoldstein%26aufirst%3DL.%2BD.%26aulast%3DSenger%26aufirst%3DK.%26aulast%3DDixon%26aufirst%3DM.%2BD.%26aulast%3DVelayutham%26aufirst%3DD.%26aulast%3DVargas%26aufirst%3DD.%26aulast%3DChaudhuri%26aufirst%3DS.%26aulast%3DMuraleedharan%26aufirst%3DM.%26aulast%3DGoel%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.-J.%2BJ.%26aulast%3DRatan%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DFaherty%26aufirst%3DB.%26aulast%3Dde%2Bla%2BRosa%26aufirst%3DG.%26aulast%3DShibata%26aufirst%3DH.%26aulast%3DBaca%26aufirst%3DM.%26aulast%3DSagolla%26aufirst%3DM.%26aulast%3DZiai%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DG.%2BA.%26aulast%3DVucic%26aufirst%3DD.%26aulast%3DMohan%26aufirst%3DS.%26aulast%3DAntony%26aufirst%3DA.%26aulast%3DStinson%26aufirst%3DJ.%26aulast%3DKirkpatrick%26aufirst%3DD.%2BS.%26aulast%3DHannoush%26aufirst%3DR.%2BN.%26aulast%3DDurinck%26aufirst%3DS.%26aulast%3DModrusan%26aufirst%3DZ.%26aulast%3DStawiski%26aufirst%3DE.%2BW.%26aulast%3DWiley%26aufirst%3DK.%26aulast%3DRaudsepp%26aufirst%3DT.%26aulast%3DKini%26aufirst%3DR.%2BM.%26aulast%3DZachariah%26aufirst%3DA.%26aulast%3DSeshagiri%26aufirst%3DS.%26atitle%3DThe%2520Indian%2520cobra%2520reference%2520genome%2520and%2520transcriptome%2520enables%2520comprehensive%2520identification%2520of%2520venom%2520toxins%26jtitle%3DNat.%2520Genet.%26date%3D2020%26volume%3D52%26issue%3D1%26spage%3D106%26epage%3D117%26doi%3D10.1038%2Fs41588-019-0559-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, N. H.</span></span> <span> </span><span class="NLM_article-title">Venom and purified toxins of the spectacled cobra (Naja naja) from Pakistan: insights into toxicity and antivenom neutralization</span>. <i>Am. J. Trop. Med. Hyg.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>94</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1392</span>– <span class="NLM_lpage">1399</span>, <span class="refDoi"> DOI: 10.4269/ajtmh.15-0871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.4269%2Fajtmh.15-0871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=27022154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFWltLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2016&pages=1392-1399&issue=6&author=K.+Y.+Wongauthor=C.+H.+Tanauthor=N.+H.+Tan&title=Venom+and+purified+toxins+of+the+spectacled+cobra+%28Naja+naja%29+from+Pakistan%3A+insights+into+toxicity+and+antivenom+neutralization&doi=10.4269%2Fajtmh.15-0871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Venom and purified toxins of the spectacled cobra (Naja naja) from Pakistan: insights into toxicity and antivenom neutralization</span></div><div class="casAuthors">Wong, Kin Ying; Tan, Choo Hock; Tan, Nget Hong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Tropical Medicine and Hygiene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1392-1399</span>CODEN:
                <span class="NLM_cas:coden">AJTHAB</span>;
        ISSN:<span class="NLM_cas:issn">1476-1645</span>.
    
            (<span class="NLM_cas:orgname">American Society of Tropical Medicine and Hygiene</span>)
        </div><div class="casAbstract">Geog. variations of snake venoms can result in suboptimal effectiveness of Indian antivenoms that are currently used in most South Asian countries.  This study investigated the toxicity and neutralization profile of the venom and toxins from Pakistani spectacled cobra, Naja naja, using VINS polyvalent antivenom (VPAV, India), Naja kaouthia monovalent antivenom (NKMAV, Thailand), and neuro bivalent antivenom (NBAV, Taiwan).  Cationexchange and reverse-phase high-performance liq. chromatog. fractionations followed by toxin identification through liq. chromatog.-mass spectrometry (MS)/MS indicated that the venom comprised mainly of postsynaptic neurotoxins (NTXs) (long neurotoxins [LNTXs], 28.3%; short neurotoxins [SNTXs], 8%), cytotoxins (CTXs) (31.2%), and acidic phospholipases A2 (12.3%).  NKMAV is the most effective in neutralizing the lethal effect of the venom (potency = 1.1 mg venom/mL) and its LNTX (potency = 0.5 mg toxin/mL), consistent with the high content of LNTX in N. kaouthia venom.  VPAV was effective in neutralizing the CTX (potency = 0.4 mg toxin/mL), in agreement with the higher CTX abundance in Indian cobra venom.  All the three antivenoms were weak in neutralizing the SNTX (potency = 0.03-0.04 mg toxin/mL), including NBAV that was raised from the SNTX-rich Taiwanese cobra venom.  In a challenge-rescue expt., envenomed mice were prevented from death by a maximal dose of VPAV (i.v. 200 μL) but the recovery from paralysis was slow, indicating the need for higher or repeated doses of VPAV.  Our results suggest that optimal neutralization for Pakistani N. naja venom may be achieved by improving the formulation of antivenom prodn. to enhance antivenom immunoreactivity against long and SNTXs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuU_ljPBcbm7Vg90H21EOLACvtfcHk0lj9PLY9HmoYSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFWltLvJ&md5=375955bb588047f56674298ba57dd792</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4269%2Fajtmh.15-0871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4269%252Fajtmh.15-0871%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DK.%2BY.%26aulast%3DTan%26aufirst%3DC.%2BH.%26aulast%3DTan%26aufirst%3DN.%2BH.%26atitle%3DVenom%2520and%2520purified%2520toxins%2520of%2520the%2520spectacled%2520cobra%2520%2528Naja%2520naja%2529%2520from%2520Pakistan%253A%2520insights%2520into%2520toxicity%2520and%2520antivenom%2520neutralization%26jtitle%3DAm.%2520J.%2520Trop.%2520Med.%2520Hyg.%26date%3D2016%26volume%3D94%26issue%3D6%26spage%3D1392%26epage%3D1399%26doi%3D10.4269%2Fajtmh.15-0871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barber, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isbister, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, W. C.</span></span> <span> </span><span class="NLM_article-title">Alpha neurotoxins</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2013.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2013.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=23416229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFOqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2013&pages=47-58&author=C.+M.+Barberauthor=G.+K.+Isbisterauthor=W.+C.+Hodgson&title=Alpha+neurotoxins&doi=10.1016%2Fj.toxicon.2013.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha neurotoxins</span></div><div class="casAuthors">Barber, Carmel M.; Isbister, Geoffrey K.; Hodgson, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  α-Neurotoxins have been isolated from hydrophid, elapid and, more recently, colubrid snake venoms.  Also referred to as postsynaptic neurotoxins or curare mimetic' neurotoxins, they play an important role in the capture and/or killing of prey by binding to the nicotinic acetylcholine receptor on the skeletal muscle disrupting neurotransmission.  They are also thought to cause respiratory paralysis in envenomed humans.  This review will discuss the historical background into the discovery, isolation, structure and mechanism of action of the α-neurotoxins, including targets and cellular outcomes, and then will examine the potential uses of α-neurotoxins as pharmacol. tools and/or as drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4R4vR45UvEbVg90H21EOLACvtfcHk0lj9PLY9HmoYSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFOqsrk%253D&md5=094db1cf51390a01d15c4f2883752d26</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2013.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2013.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DBarber%26aufirst%3DC.%2BM.%26aulast%3DIsbister%26aufirst%3DG.%2BK.%26aulast%3DHodgson%26aufirst%3DW.%2BC.%26atitle%3DAlpha%2520neurotoxins%26jtitle%3DToxicon%26date%3D2013%26volume%3D66%26spage%3D47%26epage%3D58%26doi%3D10.1016%2Fj.toxicon.2013.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomonte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvete, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">León, G.</span></span> <span> </span><span class="NLM_article-title">Assessing the preclinical efficacy of antivenoms: from the lethality neutralization assay to antivenomics</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">168</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2012.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2012.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=23201503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ntLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=168-179&author=J.+M.+Guti%C3%A9rrezauthor=G.+Solanoauthor=D.+Plaauthor=M.+Herreraauthor=%C3%81.+Seguraauthor=M.+Villaltaauthor=M.+Vargasauthor=L.+Sanzauthor=B.+Lomonteauthor=J.+J.+Calveteauthor=G.+Le%C3%B3n&title=Assessing+the+preclinical+efficacy+of+antivenoms%3A+from+the+lethality+neutralization+assay+to+antivenomics&doi=10.1016%2Fj.toxicon.2012.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the preclinical efficacy of antivenoms: From the lethality neutralization assay to antivenomics</span></div><div class="casAuthors">Gutierrez, Jose Maria; Solano, Gabriela; Pla, Davinia; Herrera, Maria; Segura, Alvaro; Villalta, Mauren; Vargas, Mariangela; Sanz, Libia; Lomonte, Bruno; Calvete, Juan J.; Leon, Guillermo</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">168-179</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The assessment of the capacity of antivenoms to neutralize the lethal activity of snake venoms is the gold std. in the preclin. anal. of antivenom efficacy, and is routinely performed by manufacturers and quality control labs.  However, the complexity of snake venom compn. and toxicol. profile demands that, for many venoms, such as those of viperid snakes and some elapids, the neutralization of lethality be complemented with the anal. of the neutralization of other relevant toxic activities, such as hemorrhagic, myotoxic, necrotizing, procoagulant and defibrinogenating effects.  This expanded protocol for preclin. testing of antivenoms should be used when a new antivenom is developed or when an existing antivenom is introduced in a new geog. setting for the neutralization of either homologous or heterologous venoms.  In recent years, the assessment of the immunol. reactivity of antivenoms has been enriched by the use of proteomic tools, with a methodol. named 'antivenomics'.  This allows the identification of venom components to which antivenoms have, or lack, antibodies, and thus complements the data gathered in neutralization tests, paving the way for a knowledge-based improvement of antivenom design and efficacy.  International projects involving participants of manufg., quality control and academic research groups should be promoted in order to gain a deeper understanding on the preclin. neutralizing spectrum of antivenoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh35k3CR3hT7Vg90H21EOLACvtfcHk0liHGW07iNJgkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ntLvF&md5=d17b834cf8056a5b904676a9206fde5f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2012.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2012.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DSolano%26aufirst%3DG.%26aulast%3DPla%26aufirst%3DD.%26aulast%3DHerrera%26aufirst%3DM.%26aulast%3DSegura%26aufirst%3D%25C3%2581.%26aulast%3DVillalta%26aufirst%3DM.%26aulast%3DVargas%26aufirst%3DM.%26aulast%3DSanz%26aufirst%3DL.%26aulast%3DLomonte%26aufirst%3DB.%26aulast%3DCalvete%26aufirst%3DJ.%2BJ.%26aulast%3DLe%25C3%25B3n%26aufirst%3DG.%26atitle%3DAssessing%2520the%2520preclinical%2520efficacy%2520of%2520antivenoms%253A%2520from%2520the%2520lethality%2520neutralization%2520assay%2520to%2520antivenomics%26jtitle%3DToxicon%26date%3D2013%26volume%3D69%26spage%3D168%26epage%3D179%26doi%3D10.1016%2Fj.toxicon.2012.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arbabi-Ghahroudi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, J. C.</span></span> <span> </span><span class="NLM_article-title">In vivo neutralization of α-cobratoxin with high-affinity llama single-domain antibodies (VHHs) and a VHH-Fc antibody</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>), <span class="NLM_elocation-id">e69495</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0069495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1371%2Fjournal.pone.0069495" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=23894495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ejtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&issue=7&author=G.+Richardauthor=A.+J.+Meyersauthor=M.+D.+McLeanauthor=M.+Arbabi-Ghahroudiauthor=R.+MacKenzieauthor=J.+C.+Hall&title=In+vivo+neutralization+of+%CE%B1-cobratoxin+with+high-affinity+llama+single-domain+antibodies+%28VHHs%29+and+a+VHH-Fc+antibody&doi=10.1371%2Fjournal.pone.0069495"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo neutralization of α-cobratoxin with high-affinity llama single-domain antibodies (VHHs) and a VHH-Fc antibody</span></div><div class="casAuthors">Richard, Gabrielle; Meyers, Ashley J.; McLean, Michael D.; Arbabi-Ghahroudi, Mehdi; MacKenzie, Roger; Hall, J. Christopher</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e69495</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Small recombinant antibody fragments (e.g. scFvs and VHHs), which are highly tissue permeable, are being investigated for antivenom prodn. as conventional antivenoms consisting of IgG or F(ab')2 antibody fragments do not effectively neutralize venom toxins located in deep tissues.  However, antivenoms composed entirely of small antibody fragments may have poor therapeutic efficacy due to their short serum half-lives.  To increase serum persistence and maintain tissue penetration, we prepd. low and high mol. mass antivenom antibodies.  Four llama VHHs were isolated from an immune VHH-displayed phage library and were shown to have high affinity, in the low nM range, for α-cobratoxin (α-Cbtx), the most lethal component of Naja kaouthia venom.  Subsequently, our highest affinity VHH (C2) was fused to a human Fc fragment to create a VHH2-Fc antibody that would offer prolonged serum persistence.  After in planta (Nicotiana benthamiana) expression and purifn., we show that our VHH2-Fc antibody retained high affinity binding to α-Cbtx.  Mouse α-Cbtx challenge studies showed that our highest affinity VHHs (C2 and C20) and the VHH2-Fc antibody effectively neutralized lethality induced by α-Cbtx at an antibody:toxin molar ratio as low as ca. 0.75x:1.  Further research towards the development of an antivenom therapeutic involving these anti-α-Cbtx VHHs and VHH2-Fc antibody mols. should involve testing them as a combination, to det. whether they maintain tissue penetration capability and low immunogenicity, and whether they exhibit improved serum persistence and therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4hiykuDKo3LVg90H21EOLACvtfcHk0liHGW07iNJgkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1ejtbvP&md5=d7d8d4bd6987d18cda0fa7ca0be6b9d6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0069495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0069495%26sid%3Dliteratum%253Aachs%26aulast%3DRichard%26aufirst%3DG.%26aulast%3DMeyers%26aufirst%3DA.%2BJ.%26aulast%3DMcLean%26aufirst%3DM.%2BD.%26aulast%3DArbabi-Ghahroudi%26aufirst%3DM.%26aulast%3DMacKenzie%26aufirst%3DR.%26aulast%3DHall%26aufirst%3DJ.%2BC.%26atitle%3DIn%2520vivo%2520neutralization%2520of%2520%25CE%25B1-cobratoxin%2520with%2520high-affinity%2520llama%2520single-domain%2520antibodies%2520%2528VHHs%2529%2520and%2520a%2520VHH-Fc%2520antibody%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26issue%3D7%26doi%3D10.1371%2Fjournal.pone.0069495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulkeaw, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakolvaree, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srimanote, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tongtawe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneewatch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sookrung, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tungtrongchitr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tapchaisri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurazono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaicumpa, W.</span></span> <span> </span><span class="NLM_article-title">Human monoclonal ScFv neutralize lethal Thai cobra, Naja kaouthia, neurotoxin</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2008.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.jprot.2008.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=19162253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Oqurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2009&pages=270-282&issue=2&author=K.+Kulkeawauthor=Y.+Sakolvareeauthor=P.+Srimanoteauthor=P.+Tongtaweauthor=S.+Maneewatchauthor=N.+Sookrungauthor=A.+Tungtrongchitrauthor=P.+Tapchaisriauthor=H.+Kurazonoauthor=W.+Chaicumpa&title=Human+monoclonal+ScFv+neutralize+lethal+Thai+cobra%2C+Naja+kaouthia%2C+neurotoxin&doi=10.1016%2Fj.jprot.2008.12.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Human monoclonal ScFv neutralize lethal Thai cobra, Naja kaouthia, neurotoxin</span></div><div class="casAuthors">Kulkeaw, Kasem; Sakolvaree, Yuwaporn; Srimanote, Potjanee; Tongtawe, Pongsri; Maneewatch, Santi; Sookrung, Nitat; Tungtrongchitr, Anchalee; Tapchaisri, Pramuan; Kurazono, Hisao; Chaicumpa, Wanpen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">270-282</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Animal derived anti-Naja. kaouthia (Thai cobra) venom is used for specific treatment of the snake bitten victims.  Many recipients develop allergic reaction or anti-isotype response which causes serum sickness.  A better therapeutic antibody is needed.  In this study, long α-neurotoxin was purified from the N. kaouthia holovenom and verified by 2D-LC/MS-MS.  The toxin was used as antigen in a phage biopanning to select phage clones displaying human single chain variable antibody fragments (HuScFv) from a phage display antibody library constructed from Ig genes of nonimmunized Thai blood donors.  HuScFv that specifically bound to the neurotoxin were produced from huscfv-phagemid transformed E. coli clones and affinity purified.  The HuScFv could neutralize toxicity of the N. kaouthia neurotoxin and rescued the envenomized mice from the neurotoxin mediated lethality.  Peptide mimotope of the neutralizing HuScFv matched with an amino acid sequence (epitope) located in the loop-3 of the N. kaouthia long α-neurotoxin which functions in acetylcholine receptor binding.  The mimotope is also similar to peptide sequences found on other snake venom neurotoxins implying a possibility of the HuScFv to exert pan-neutralizing activity against multiple snake neurotoxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHcs6Zu1Jc3LVg90H21EOLACvtfcHk0liHGW07iNJgkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Oqurs%253D&md5=ce5e963b123f89101e3df77169836cbd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2008.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2008.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DKulkeaw%26aufirst%3DK.%26aulast%3DSakolvaree%26aufirst%3DY.%26aulast%3DSrimanote%26aufirst%3DP.%26aulast%3DTongtawe%26aufirst%3DP.%26aulast%3DManeewatch%26aufirst%3DS.%26aulast%3DSookrung%26aufirst%3DN.%26aulast%3DTungtrongchitr%26aufirst%3DA.%26aulast%3DTapchaisri%26aufirst%3DP.%26aulast%3DKurazono%26aufirst%3DH.%26aulast%3DChaicumpa%26aufirst%3DW.%26atitle%3DHuman%2520monoclonal%2520ScFv%2520neutralize%2520lethal%2520Thai%2520cobra%252C%2520Naja%2520kaouthia%252C%2520neurotoxin%26jtitle%3DJ.%2520Proteomics%26date%3D2009%26volume%3D72%26issue%3D2%26spage%3D270%26epage%3D282%26doi%3D10.1016%2Fj.jprot.2008.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laustsen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatt-Vellatt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arias, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pus, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oscoz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavny, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leah, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindholm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomonte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, J.</span></span> <span> </span><span class="NLM_article-title">In vivo neutralization of dendrotoxin-mediated neurotoxicity of black mamba venom by oligoclonal human IgG antibodies</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3928</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-06086-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fs41467-018-06086-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=30279409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BB3czltFymtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3928&author=A.+H.+Laustsenauthor=A.+Karatt-Vellattauthor=E.+W.+Mastersauthor=A.+S.+Ariasauthor=U.+Pusauthor=C.+Knudsenauthor=S.+Oscozauthor=P.+Slavnyauthor=D.+T.+Griffithsauthor=A.+M.+Lutherauthor=R.+A.+Leahauthor=M.+Lindholmauthor=B.+Lomonteauthor=J.+M.+Guti%C3%A9rrezauthor=J.+McCafferty&title=In+vivo+neutralization+of+dendrotoxin-mediated+neurotoxicity+of+black+mamba+venom+by+oligoclonal+human+IgG+antibodies&doi=10.1038%2Fs41467-018-06086-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo neutralization of dendrotoxin-mediated neurotoxicity of black mamba venom by oligoclonal human IgG antibodies</span></div><div class="casAuthors">Laustsen Andreas H; Pus Urska; Knudsen Cecilie; Karatt-Vellatt Aneesh; Masters Edward W; Slavny Peter; Griffiths Daniel T; Luther Alice M; Leah Rachael A; Lindholm Majken; McCafferty John; Arias Ana Silvia; Oscoz Saioa; Lomonte Bruno; Gutierrez Jose Maria</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3928</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The black mamba (Dendroaspis polylepis) is one of the most feared snake species of the African savanna.  It has a potent, fast-acting neurotoxic venom comprised of dendrotoxins and α-neurotoxins associated with high fatality in untreated victims.  Current antivenoms are both scarce on the African continent and present a number of drawbacks as they are derived from the plasma of hyper-immunized large mammals.  Here, we describe the development of an experimental recombinant antivenom by a combined toxicovenomics and phage display approach.  The recombinant antivenom is based on a cocktail of fully human immunoglobulin G (IgG) monoclonal antibodies capable of neutralizing dendrotoxin-mediated neurotoxicity of black mamba whole venom in a rodent model.  Our results show the potential use of fully human monoclonal IgGs against animal toxins and the first use of oligoclonal human IgG mixtures against experimental snakebite envenoming.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnUvQzyIEmj4qdXsLcGTEUfW6udTcc2eaMgIil5CHxmbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czltFymtg%253D%253D&md5=c1a6ba8b1e8531eab98a6d614af80e26</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-06086-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-06086-4%26sid%3Dliteratum%253Aachs%26aulast%3DLaustsen%26aufirst%3DA.%2BH.%26aulast%3DKaratt-Vellatt%26aufirst%3DA.%26aulast%3DMasters%26aufirst%3DE.%2BW.%26aulast%3DArias%26aufirst%3DA.%2BS.%26aulast%3DPus%26aufirst%3DU.%26aulast%3DKnudsen%26aufirst%3DC.%26aulast%3DOscoz%26aufirst%3DS.%26aulast%3DSlavny%26aufirst%3DP.%26aulast%3DGriffiths%26aufirst%3DD.%2BT.%26aulast%3DLuther%26aufirst%3DA.%2BM.%26aulast%3DLeah%26aufirst%3DR.%2BA.%26aulast%3DLindholm%26aufirst%3DM.%26aulast%3DLomonte%26aufirst%3DB.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DMcCafferty%26aufirst%3DJ.%26atitle%3DIn%2520vivo%2520neutralization%2520of%2520dendrotoxin-mediated%2520neurotoxicity%2520of%2520black%2520mamba%2520venom%2520by%2520oligoclonal%2520human%2520IgG%2520antibodies%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D3928%26doi%3D10.1038%2Fs41467-018-06086-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrick, K.</span></span> <span> </span><span class="NLM_article-title">Inference of macromolecular assemblies from crystalline state</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>372</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">797</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.05.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.jmb.2007.05.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=17681537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2007&pages=774-797&issue=3&author=E.+Krissinelauthor=K.+Henrick&title=Inference+of+macromolecular+assemblies+from+crystalline+state&doi=10.1016%2Fj.jmb.2007.05.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Inference of Macromolecular Assemblies from Crystalline State</span></div><div class="casAuthors">Krissinel, Evgeny; Henrick, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-797</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors discuss basic phys.-chem. principles underlying the formation of stable macromol. complexes, which in many cases are likely to be the biol. units performing a certain physiol. function.  The authors also consider available theor. approaches to the calcn. of macromol. affinity and entropy of complexation.  The latter is shown to play an important role and make a major effect on complex size and symmetry.  The authors develop a new method, based on chem. thermodn., for automatic detection of macromol. assemblies in the Protein Data Bank (PDB) entries that are the results of x-ray diffraction expts.  As found, biol. units may be recovered at 80-90% success rate, which makes x-ray crystallog. an important source of exptl. data on macromol. complexes and protein-protein interactions.  The method is implemented as a public WWW service (http://www.ebi.ac.uk/msd-srv/prot_int/pistart.html).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3cilBvVoBS7Vg90H21EOLACvtfcHk0lhk5K-r4K2GFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D&md5=a5c764cfc7dc129f53ddc31ef9d475fa</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.05.022%26sid%3Dliteratum%253Aachs%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DInference%2520of%2520macromolecular%2520assemblies%2520from%2520crystalline%2520state%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D372%26issue%3D3%26spage%3D774%26epage%3D797%26doi%3D10.1016%2Fj.jmb.2007.05.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nirthanan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwee, M. C. E.</span></span> <span> </span><span class="NLM_article-title">Three-finger alpha-neurotoxins and the nicotinic acetylcholine receptor, forty years on</span>. <i>J. Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>94</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1254/jphs.94.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1254%2Fjphs.94.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=14745112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXosVamsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2004&pages=1-17&issue=1&author=S.+Nirthananauthor=M.+C.+E.+Gwee&title=Three-finger+alpha-neurotoxins+and+the+nicotinic+acetylcholine+receptor%2C+forty+years+on&doi=10.1254%2Fjphs.94.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Three-finger α-neurotoxins and the nicotinic acetylcholine receptor, forty years on</span></div><div class="casAuthors">Nirthanan, Selvanayagam; Gwee, Matthew C. E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological Sciences (Tokyo, Japan)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-17</span>CODEN:
                <span class="NLM_cas:coden">JPSTGJ</span>;
        ISSN:<span class="NLM_cas:issn">1347-8613</span>.
    
            (<span class="NLM_cas:orgname">Japanese Pharmacological Society</span>)
        </div><div class="casAbstract">A review.  The discovery, about forty years ago, of α-bungarotoxin, a three-finger α-neurotoxin from Bungarus multicinctus venom, enabled the isolation of the nicotinic acetylcholine receptor (nAChR), making it one of the most thoroughly characterized receptors today.  Since then, the sites of interaction between α-neurotoxins and nAChRs have largely been delineated, revealing the remarkable plasticity of the three-finger toxin fold that has optimally evolved to utilize different combinations of functional groups to generate a panoply of target specificities to discern subtle differences between nAChR subtypes.  New facets in toxinol. have now broadened the scope for the use of α-neurotoxins in scientific discovery.  For instance, the development of short, combinatorial library-derived, synthetic peptides that bind with subnanomolar affinity to α-bungarotoxin and prevent its interaction with muscle nAChRs has led to the in vivo neutralization of exptl. α-bungarotoxin envenomation, while the successful introduction of pharmatopes bearing "α-bungarotoxin-sensitive sites" into toxin-insensitive nAChRs has permitted the use of various α-neurotoxin tags to localize and characterize new receptor subtypes.  More ambitious strategies can now be envisaged for engineering rationally designed novel activities on three-finger toxin scaffolds to generate lead peptides of therapeutic value that target the nicotinic pharmacopoeia.  This review details the progress made towards achieving this goal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv04nhW9gh2bVg90H21EOLACvtfcHk0lhk5K-r4K2GFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXosVamsg%253D%253D&md5=de858ec5f133be902b11f34675f48c34</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1254%2Fjphs.94.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjphs.94.1%26sid%3Dliteratum%253Aachs%26aulast%3DNirthanan%26aufirst%3DS.%26aulast%3DGwee%26aufirst%3DM.%2BC.%2BE.%26atitle%3DThree-finger%2520alpha-neurotoxins%2520and%2520the%2520nicotinic%2520acetylcholine%2520receptor%252C%2520forty%2520years%2520on%26jtitle%3DJ.%2520Pharmacol.%2520Sci.%26date%3D2004%26volume%3D94%26issue%3D1%26spage%3D1%26epage%3D17%26doi%3D10.1254%2Fjphs.94.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worrell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noviello, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stowell, M. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibbs, R. E.</span></span> <span> </span><span class="NLM_article-title">Structure of the native muscle-type nicotinic receptor and inhibition by snake venom toxins</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2020.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.neuron.2020.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=32275860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmvFWhur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2020&pages=952-962&issue=6&author=M.+M.+Rahmanauthor=J.+Tengauthor=B.+T.+Worrellauthor=C.+M.+Novielloauthor=M.+Leeauthor=A.+Karlinauthor=M.+H.+B.+Stowellauthor=R.+E.+Hibbs&title=Structure+of+the+native+muscle-type+nicotinic+receptor+and+inhibition+by+snake+venom+toxins&doi=10.1016%2Fj.neuron.2020.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Native Muscle-type Nicotinic Receptor and Inhibition by Snake Venom Toxins.</span></div><div class="casAuthors">Rahman, Md. Mahfuzur; Teng, Jinfeng; Worrell, Brady T.; Noviello, Colleen M.; Lee, Myeongseon; Karlin, Arthur; Stowell, Michael H. B.; Hibbs, Ryan E.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">952-962.e5</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The nicotinic acetylcholine receptor, a pentameric ligand-gated ion channel, converts the free energy of binding of the neurotransmitter acetylcholine into opening of its central pore.  Here we present the first high-resoln. structure of the receptor type found in muscle-endplate membrane and in the muscle-derived elec. tissues of fish.  The native receptor was purified from Torpedo elec. tissue and functionally reconstituted in lipids optimal for cryo-electron microscopy.  The receptor was stabilized in a closed state by the binding of α-bungarotoxin.  The structure reveals the binding of a toxin mol. at each of two subunit interfaces in a manner that would block the binding of acetylcholine.  It also reveals a closed gate in the ion-conducting pore, formed by hydrophobic amino acid side chains, located ∼60 Å from the toxin binding sites.  The structure provides a framework for understanding gating in ligand-gated channels and how mutations in the acetylcholine receptor cause congenital myasthenic syndromes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW_NVMcdEgUbVg90H21EOLACvtfcHk0lg5YVzkKbLeHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmvFWhur0%253D&md5=52d7f7343f1bfca9e2f6a044f8d3015b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2020.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DRahman%26aufirst%3DM.%2BM.%26aulast%3DTeng%26aufirst%3DJ.%26aulast%3DWorrell%26aufirst%3DB.%2BT.%26aulast%3DNoviello%26aufirst%3DC.%2BM.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DKarlin%26aufirst%3DA.%26aulast%3DStowell%26aufirst%3DM.%2BH.%2BB.%26aulast%3DHibbs%26aufirst%3DR.%2BE.%26atitle%3DStructure%2520of%2520the%2520native%2520muscle-type%2520nicotinic%2520receptor%2520and%2520inhibition%2520by%2520snake%2520venom%2520toxins%26jtitle%3DNeuron%26date%3D2020%26volume%3D106%26issue%3D6%26spage%3D952%26epage%3D962%26doi%3D10.1016%2Fj.neuron.2020.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talley, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchot, P.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of a Cbtx-AChBP complex reveals essential interactions between snake alpha-neurotoxins and nicotinic receptors</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1512</span>– <span class="NLM_lpage">1522</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fsj.emboj.7600620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15791209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFequro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=1512-1522&issue=8&author=Y.+Bourneauthor=T.+T.+Talleyauthor=S.+B.+Hansenauthor=P.+Taylorauthor=P.+Marchot&title=Crystal+structure+of+a+Cbtx-AChBP+complex+reveals+essential+interactions+between+snake+alpha-neurotoxins+and+nicotinic+receptors&doi=10.1038%2Fsj.emboj.7600620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a Cbtx-AChBP complex reveals essential interactions between snake α-neurotoxins and nicotinic receptors</span></div><div class="casAuthors">Bourne, Yves; Talley, Todd T.; Hansen, Scott B.; Taylor, Palmer; Marchot, Pascale</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1512-1522</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The crystal structure of the snake long α-neurotoxin, α-cobratoxin, bound to the pentameric acetylcholine-binding protein (AChBP) from Lymnaea stagnalis, was solved from good quality d. maps despite a 4.2 Å overall resoln.  The structure unambiguously reveals the positions and orientations of all five three-fingered toxin mols. inserted at the AChBP subunit interfaces and the conformational changes assocd. with toxin binding.  AChBP loops C and F that border the ligandbinding pocket move markedly from their original positions to wrap around the tips of the toxin first and second fingers and part of its C-terminus, while rearrangements also occur in the toxin fingers.  At the interface of the complex, major interactions involve arom. and aliph. side chains within the AChBP binding pocket and, at the buried tip of the toxin second finger, conserved Phe and Arg residues that partially mimic a bound agonist mol.  Hence this structure, in revealing a distinctive and unpredicted conformation of the toxin-bound AChBP mol., provides a lead template resembling a resting state conformation of the nicotinic receptor and for understanding selectivity of curaremimetic α-neurotoxins for the various receptor species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ieEvb-on7rVg90H21EOLACvtfcHk0lg5YVzkKbLeHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFequro%253D&md5=3d3c40b85aa80586e9d94d30d704e715</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600620%26sid%3Dliteratum%253Aachs%26aulast%3DBourne%26aufirst%3DY.%26aulast%3DTalley%26aufirst%3DT.%2BT.%26aulast%3DHansen%26aufirst%3DS.%2BB.%26aulast%3DTaylor%26aufirst%3DP.%26aulast%3DMarchot%26aufirst%3DP.%26atitle%3DCrystal%2520structure%2520of%2520a%2520Cbtx-AChBP%2520complex%2520reveals%2520essential%2520interactions%2520between%2520snake%2520alpha-neurotoxins%2520and%2520nicotinic%2520receptors%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26issue%3D8%26spage%3D1512%26epage%3D1522%26doi%3D10.1038%2Fsj.emboj.7600620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jurgilas, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves-Ferreira, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domont, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perales, J.</span></span> <span> </span><span class="NLM_article-title">PO41, a snake venom metalloproteinase inhibitor isolated from Philander opossum serum</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2003.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2003.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=14602117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosFOrsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=621-628&issue=6&author=P.+B.+Jurgilasauthor=A.+G.+Neves-Ferreiraauthor=G.+B.+Domontauthor=J.+Perales&title=PO41%2C+a+snake+venom+metalloproteinase+inhibitor+isolated+from+Philander+opossum+serum&doi=10.1016%2Fj.toxicon.2003.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PO41, a snake venom metalloproteinase inhibitor isolated from Philander opossum serum</span></div><div class="casAuthors">Jurgilas, Patricia B.; Neves-Ferreira, Ana G. C.; Domont, Gilberto B.; Perales, Jonas</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">621-628</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">PO41 was isolated from Philander opossum serum by DEAE-Sephacel, Ph Superose and Superdex 200 chromatogs. and showed a mol. mass of 41,330 Da by MALDI-TOF MS.  Mol. masses of 81.5 and 84.5 kDa were obtained by size exclusion chromatog. and dynamic laser light scattering, resp., suggesting that PO41 is dimeric.  Its isoelec. point was estd. to be lower than 3.5.  PO41 presented similar amino terminal sequence to those of DM40 and DM43, two antihaemorrhagins previously isolated from Didelphis marsupialis serum and was recognized by polyclonal antibodies raised against D. marsupialis antibothropic fraction.  To study the inhibitory properties of this protein, the metalloproteinases bothrolysin and jararhagin were isolated from Bothrops jararaca venom by chromatogs. on Superdex 200 and Ph Superose.  Jararhagin was further submitted to a Mono Q column.  The proteolytic and haemorrhagic effects of these haemorrhagins were neutralized by PO41.  Both snake venom metalloproteinases formed stable complexes with PO41.  The stoichiometry of the complex PO41-jararhagin was one inhibitor subunit to one mol. of the enzyme.  These results show that PO41 has physicochem., structural, immunoreactive and biol. properties similar to other metalloproteinase inhibitors belonging to the supergene family of Igs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAId4XtPyRLVg90H21EOLACvtfcHk0lg5YVzkKbLeHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosFOrsrg%253D&md5=4aa34ae231a50d007df5a46732091c15</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2003.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2003.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DJurgilas%26aufirst%3DP.%2BB.%26aulast%3DNeves-Ferreira%26aufirst%3DA.%2BG.%26aulast%3DDomont%26aufirst%3DG.%2BB.%26aulast%3DPerales%26aufirst%3DJ.%26atitle%3DPO41%252C%2520a%2520snake%2520venom%2520metalloproteinase%2520inhibitor%2520isolated%2520from%2520Philander%2520opossum%2520serum%26jtitle%3DToxicon%26date%3D2003%26volume%3D42%26issue%3D6%26spage%3D621%26epage%3D628%26doi%3D10.1016%2Fj.toxicon.2003.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karain, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, W. K.</span></span> <span> </span><span class="NLM_article-title">C60 fullerenes as a novel treatment for poisoning and envenomation: a proof-of-concept study for snakebite</span>. <i>J. Nanosci. Nanotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">7764</span>– <span class="NLM_lpage">7771</span>, <span class="refDoi"> DOI: 10.1166/jnn.2016.12851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1166%2Fjnn.2016.12851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWks7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=7764-7771&issue=7&author=B.+D.+Karainauthor=M.+K.+H.+Leeauthor=W.+K.+Hayes&title=C60+fullerenes+as+a+novel+treatment+for+poisoning+and+envenomation%3A+a+proof-of-concept+study+for+snakebite&doi=10.1166%2Fjnn.2016.12851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">C60 fullerenes as a novel treatment for poisoning and envenomation: a proof-of-concept study for snakebite</span></div><div class="casAuthors">Karain, Bradley Darryl; Lee, Maximus Kyung Hyun; Hayes, William K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Nanoscience and Nanotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">7764-7771</span>CODEN:
                <span class="NLM_cas:coden">JNNOAR</span>;
        ISSN:<span class="NLM_cas:issn">1533-4880</span>.
    
            (<span class="NLM_cas:orgname">American Scientific Publishers</span>)
        </div><div class="casAbstract">Millions of poisonings and envenomations occur annually worldwide.  When treatments for these conditions exist, they are often highly specific and may be costly or unavailable, esp. in remote regions and developing countries.  In cases of snake envenomation, mortality and severe morbidity occur in hundreds of thousands of cases annually.  These tragic physiol. sequelae result from the complex mixts. of protein toxins comprising snake venom.  The current gold std. for snakebite treatment is to use a mixt. of polyclonal antibodies, referred to as antivenom.  While effective, antivenoms have severe limitations: they are costly to produce and distribute; they must be delivered in an acute care facility; they have a relatively short shelf life; and they often induce side effects that are generally mild but occasionally serious.  Certain nanoparticles, in contrast, possess protein-binding and pharmacokinetic properties that make them a potentially low-cost and readily available adjunct or alternative to antivenom for treatment of systemically distributed toxins.  We therefore conducted a proof-of-concept study to test the ability of C60 fullerene to neutralize rattlesnake venom in a cricket model.  We injected approx. LD50 doses of southern Pacific rattlesnake (Crotalus oreganus helleri) venom into crickets (Acheta domesticus).  In the first of two expts., crickets injected with C60 fullerene prior to envenomation experienced 15.7% and 25.0% higher rates of survival at 24 and 48 h, resp., compared to control crickets.  In the second expt., survival anal. confirmed that crickets protected with C60, regardless of C60 dose (1, 2, or 4 μg/g cricket mass), experienced greater survival up to 60 h and survived significantly longer than controls.  C60 fullerene conferred significant protection against rattlesnake envenomation in a cricket model.  Accordingly, C60 fullerene merits further consideration as a simple, low-cost treatment for snakebite and potentially many other forms of poisoning and envenomation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphOH2Y_cCIx7Vg90H21EOLACvtfcHk0lj6opDnSfFb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWks7vI&md5=fc4b74c66eb2942a909c0235e3750664</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1166%2Fjnn.2016.12851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1166%252Fjnn.2016.12851%26sid%3Dliteratum%253Aachs%26aulast%3DKarain%26aufirst%3DB.%2BD.%26aulast%3DLee%26aufirst%3DM.%2BK.%2BH.%26aulast%3DHayes%26aufirst%3DW.%2BK.%26atitle%3DC60%2520fullerenes%2520as%2520a%2520novel%2520treatment%2520for%2520poisoning%2520and%2520envenomation%253A%2520a%2520proof-of-concept%2520study%2520for%2520snakebite%26jtitle%3DJ.%2520Nanosci.%2520Nanotechnol.%26date%3D2016%26volume%3D16%26issue%3D7%26spage%3D7764%26epage%3D7771%26doi%3D10.1166%2Fjnn.2016.12851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickler, P.</span></span> <span> </span><span class="NLM_article-title">Varespladib (LY315920) appears to be a potent, broad-spectrum, inhibitor of snake venom phospholipase A2 and a possible pre-referral treatment for envenomation</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">248</span>, <span class="refDoi"> DOI: 10.3390/toxins8090248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.3390%2Ftoxins8090248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ClsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=248&issue=9&author=M.+Lewinauthor=S.+Samuelauthor=J.+Merkelauthor=P.+Bickler&title=Varespladib+%28LY315920%29+appears+to+be+a+potent%2C+broad-spectrum%2C+inhibitor+of+snake+venom+phospholipase+A2+and+a+possible+pre-referral+treatment+for+envenomation&doi=10.3390%2Ftoxins8090248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Varespladib (LY315920) appears to be a potent, broad-spectrum, inhibitor of snake venom phospholipase A2 and a possible pre-referral treatment for envenomation</span></div><div class="casAuthors">Lewin, Matthew; Samuel, Stephen; Merkel, Janie; Bickler, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">248/1-248/16</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Snakebite remains a neglected medical problem of the developing world with up to 125,000 deaths each year despite more than a century of calls to improve snakebite prevention and care.  An estd. 75% of fatalities from snakebite occur outside the hospital setting.  Because phospholipase A2 (PLA2) activity is an important component of venom toxicity, we sought candidate PLA2 inhibitors by directly testing drugs.  Surprisingly, varespladib and its orally bioavailable prodrug, methyl-varespladib showed high-level secretory PLA2 (sPLA2) inhibition at nanomolar and picomolar concns. against 28 medically important snake venoms from six continents.  In vivo proof-of-concept studies with varespladib had striking survival benefit against LDs of Micrurus fulvius and Vipera berus venom, and suppressed venom-induced sPLA2 activity in rats challenged with 100% LDs of M. fulvius venom.  Rapid development and deployment of a broad-spectrum PLA2 inhibitor alone or in combination with other small mol. inhibitors of snake toxins (e.g., metalloproteases) could fill the crit. therapeutic gap spanning pre-referral and hospital setting.  Lower barriers for clin. testing of safety tested, repurposed small mol. therapeutics are a potentially economical and effective path forward to fill the pre-referral gap in the setting of snakebite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgyWpYG3kB9LVg90H21EOLACvtfcHk0lj6opDnSfFb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ClsL%252FO&md5=7342f424a5f3375ed75ff45d84f7b74b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Ftoxins8090248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins8090248%26sid%3Dliteratum%253Aachs%26aulast%3DLewin%26aufirst%3DM.%26aulast%3DSamuel%26aufirst%3DS.%26aulast%3DMerkel%26aufirst%3DJ.%26aulast%3DBickler%26aufirst%3DP.%26atitle%3DVarespladib%2520%2528LY315920%2529%2520appears%2520to%2520be%2520a%2520potent%252C%2520broad-spectrum%252C%2520inhibitor%2520of%2520snake%2520venom%2520phospholipase%2520A2%2520and%2520a%2520possible%2520pre-referral%2520treatment%2520for%2520envenomation%26jtitle%3DToxins%26date%3D2016%26volume%3D8%26issue%3D9%26spage%3D248%26doi%3D10.3390%2Ftoxins8090248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scherf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katchalski-Katzir, E.</span></span> <span> </span><span class="NLM_article-title">A β-hairpin structure in a 13-mer peptide that binds α-bungarotoxin with high affinity and neutralizes its toxicity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">6629</span>– <span class="NLM_lpage">6634</span>, <span class="refDoi"> DOI: 10.1073/pnas.111164298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1073%2Fpnas.111164298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=11381118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksVOktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=6629-6634&issue=12&author=T.+Scherfauthor=R.+Kasherauthor=M.+Balassauthor=M.+Fridkinauthor=S.+Fuchsauthor=E.+Katchalski-Katzir&title=A+%CE%B2-hairpin+structure+in+a+13-mer+peptide+that+binds+%CE%B1-bungarotoxin+with+high+affinity+and+neutralizes+its+toxicity&doi=10.1073%2Fpnas.111164298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A β-hairpin structure in a 13-mer peptide that binds α-bungarotoxin with high affinity and neutralizes its toxicity</span></div><div class="casAuthors">Scherf, Tali; Kasher, Roni; Balass, Moshe; Fridkin, Mati; Fuchs, Sara; Katchalski-Katzir, Ephraim</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6629-6634</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Snake-venom α-bungarotoxin is a member of the α-neurotoxin family that binds with very high affinity to the nicotinic acetylcholine receptor (AChR) at the neuromuscular junction.  The structure of the complex between α-bungarotoxin and a 13-mer peptide (WRYYESSLEPYPD) that binds the toxin with high affinity, thus inhibiting its interactions with AChR with an IC50 of 2 nM, has been solved by 1H-NMR spectroscopy.  The bound peptide folds into a β-hairpin structure created by two antiparallel β-strands, which combine with the already existing triple-stranded β-sheet of the toxin to form a five-stranded intermol., antiparallel β-sheet.  Peptide residues Y3P, E5P, and L8P have the highest intermol. contact area, indicating their importance in the binding of α-bungarotoxin; W1P, R2P, and Y4P also contribute significantly to the binding.  A large no. of characteristic hydrogen bonds and electrostatic and hydrophobic interactions are obsd. in the complex.  The high-affinity peptide exhibits inhibitory potency that is better than any known peptide derived from AChR, and is equal to that of the whole α-subunit of AChR.  The high degree of sequence similarity between the peptide and various types of AChRs implies that the binding mode found within the complex might possibly mimic the receptor binding to the toxin.  The design of the high-affinity peptide was based on our previous findings: (i) the detection of a lead peptide (MRYYESSLKSYPD) that binds α-bungarotoxin, using a phage-display peptide library, (ii) the information about the three-dimensional structure of α-bungarotoxin/lead-peptide complex, and (iii) the amino acid sequence anal. of different AChRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1LI6CqiC7AbVg90H21EOLACvtfcHk0lj6opDnSfFb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksVOktb4%253D&md5=0e2764b893941637227f790f3fe6984b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.111164298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.111164298%26sid%3Dliteratum%253Aachs%26aulast%3DScherf%26aufirst%3DT.%26aulast%3DKasher%26aufirst%3DR.%26aulast%3DBalass%26aufirst%3DM.%26aulast%3DFridkin%26aufirst%3DM.%26aulast%3DFuchs%26aufirst%3DS.%26aulast%3DKatchalski-Katzir%26aufirst%3DE.%26atitle%3DA%2520%25CE%25B2-hairpin%2520structure%2520in%2520a%252013-mer%2520peptide%2520that%2520binds%2520%25CE%25B1-bungarotoxin%2520with%2520high%2520affinity%2520and%2520neutralizes%2520its%2520toxicity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26issue%3D12%26spage%3D6629%26epage%3D6634%26doi%3D10.1073%2Fpnas.111164298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fosgerau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span> <span> </span><span class="NLM_article-title">Peptide therapeutics: current status and future directions</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.drudis.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=25450771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=122-128&issue=1&author=K.+Fosgerauauthor=T.+Hoffmann&title=Peptide+therapeutics%3A+current+status+and+future+directions&doi=10.1016%2Fj.drudis.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics: current status and future directions</span></div><div class="casAuthors">Fosgerau, Keld; Hoffmann, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized for being highly selective and efficacious and, at the same time, relatively safe and well tolerated.  Consequently, there is an increased interest in peptides in pharmaceutical research and development (R&D), and approx. 140 peptide therapeutics are currently being evaluated in clin. trials.  Given that the low-hanging fruits in the form of obvious peptide targets have already been picked, it has now become necessary to explore new routes beyond traditional peptide design.  Examples of such approaches are multifunctional and cell penetrating peptides, as well as peptide drug conjugates.  Here, we discuss the current status, strengths, and weaknesses of peptides as medicines and the emerging new opportunities in peptide drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmhpp8GnJ6JLVg90H21EOLACvtfcHk0lgLC1vOVAl2Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM&md5=f6b74aaa4e3fdb317b091b0450f3618a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DFosgerau%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26atitle%3DPeptide%2520therapeutics%253A%2520current%2520status%2520and%2520future%2520directions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26issue%3D1%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.drudis.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guss, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brejc, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sixma, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katchalski-Katzir, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sussman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, S.</span></span> <span> </span><span class="NLM_article-title">The binding site of acetylcholine receptor as visualized in the X-ray structure of a complex between α-bungarotoxin and a mimotope peptide</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/S0896-6273(01)00461-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2FS0896-6273%2801%2900461-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=11683996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3MXosVKmtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2001&pages=265-275&issue=2&author=M.+Harelauthor=R.+Kasherauthor=A.+Nicolasauthor=J.+M.+Gussauthor=M.+Balassauthor=M.+Fridkinauthor=A.+B.+Smitauthor=K.+Brejcauthor=T.+K.+Sixmaauthor=E.+Katchalski-Katzirauthor=J.+L.+Sussmanauthor=S.+Fuchs&title=The+binding+site+of+acetylcholine+receptor+as+visualized+in+the+X-ray+structure+of+a+complex+between+%CE%B1-bungarotoxin+and+a+mimotope+peptide&doi=10.1016%2FS0896-6273%2801%2900461-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The binding site of acetylcholine receptor as visualized in the X-ray structure of a complex between α-bungarotoxin and a mimotope peptide</span></div><div class="casAuthors">Harel, Michal; Kasher, Roni; Nicolas, Anne; Guss, J. Mitchell; Balass, Moshe; Fridkin, Mati; Smit, August B.; Brejc, Katjusa; Sixma, Titia K.; Katchalski-Katzir, Ephraim; Sussman, Joel L.; Fuchs, Sara</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">265-275</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We have detd. the crystal structure at 1.8 Å resoln. of a complex of α-bungarotoxin with a high affinity 13-residue peptide that is homologous to the binding region of the α subunit of acetylcholine receptor.  The peptide fits snugly to the toxin and adopts a β hairpin conformation.  The structures of the bound peptide and the homologous loop of acetylcholine binding protein, a sol. analog of the extracellular domain of acetylcholine receptor, are remarkably similar.  Their superposition indicates that the toxin wraps around the receptor binding site loop, and in addn., binds tightly at the interface of two of the receptor subunits where it inserts a finger into the ligand binding site, thus blocking access to the acetylcholine binding site and explaining its strong antagonistic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1Q9LO6C59FrVg90H21EOLACvtfcHk0lgLC1vOVAl2Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXosVKmtbo%253D&md5=017ecc2988e6dea2d943e7b83e0bb185</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0896-6273%2801%2900461-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0896-6273%252801%252900461-5%26sid%3Dliteratum%253Aachs%26aulast%3DHarel%26aufirst%3DM.%26aulast%3DKasher%26aufirst%3DR.%26aulast%3DNicolas%26aufirst%3DA.%26aulast%3DGuss%26aufirst%3DJ.%2BM.%26aulast%3DBalass%26aufirst%3DM.%26aulast%3DFridkin%26aufirst%3DM.%26aulast%3DSmit%26aufirst%3DA.%2BB.%26aulast%3DBrejc%26aufirst%3DK.%26aulast%3DSixma%26aufirst%3DT.%2BK.%26aulast%3DKatchalski-Katzir%26aufirst%3DE.%26aulast%3DSussman%26aufirst%3DJ.%2BL.%26aulast%3DFuchs%26aufirst%3DS.%26atitle%3DThe%2520binding%2520site%2520of%2520acetylcholine%2520receptor%2520as%2520visualized%2520in%2520the%2520X-ray%2520structure%2520of%2520a%2520complex%2520between%2520%25CE%25B1-bungarotoxin%2520and%2520a%2520mimotope%2520peptide%26jtitle%3DNeuron%26date%3D2001%26volume%3D32%26issue%3D2%26spage%3D265%26epage%3D275%26doi%3D10.1016%2FS0896-6273%2801%2900461-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chandy, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide blockers of Kv1.3 channels in T cells as therapeutics for autoimmune disease</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.cbpa.2017.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28412597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=97-107&author=K.+G.+Chandyauthor=R.+S.+Norton&title=Peptide+blockers+of+Kv1.3+channels+in+T+cells+as+therapeutics+for+autoimmune+disease&doi=10.1016%2Fj.cbpa.2017.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide blockers of Kv1.3 channels in T cells as therapeutics for autoimmune disease</span></div><div class="casAuthors">Chandy, K. George; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97-107</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The voltage-gated Kv1.3 channel in T lymphocytes is a validated therapeutic target for diverse autoimmune diseases.  Here we review the discovery of Kv1.3, its physiol. role in T cells, and why it is an attractive target for modulating autoimmune responses.  We focus on peptide inhibitors because the first Kv1.3-selective inhibitor in human trials is a peptide derived from a marine organism.  Two broad classes of peptides block Kv1.3, the first from scorpions and the second from sea anemones.  We describe their structures, their binding site in the external vestibule of Kv1.3, how they have been engineered to improve Kv1.3-specificity, and their pharmacokinetic and pharmacodynamic properties.  Finally, we highlight the therapeutic potential of Kv1.3 peptide inhibitors to treat autoimmune diseases without compromising protective immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkIKfZApsJ-7Vg90H21EOLACvtfcHk0lgLC1vOVAl2Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFOhtL8%253D&md5=d8a6ae633f17def39ce5b90b954a823d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DChandy%26aufirst%3DK.%2BG.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520blockers%2520of%2520Kv1.3%2520channels%2520in%2520T%2520cells%2520as%2520therapeutics%2520for%2520autoimmune%2520disease%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D97%26epage%3D107%26doi%3D10.1016%2Fj.cbpa.2017.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guidotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span> <span> </span><span class="NLM_article-title">Cell-penetrating peptides: from basic research to clinics</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">406</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2017.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.tips.2017.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28209404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=406-424&issue=4&author=G.+Guidottiauthor=L.+Brambillaauthor=D.+Rossi&title=Cell-penetrating+peptides%3A+from+basic+research+to+clinics&doi=10.1016%2Fj.tips.2017.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Penetrating Peptides: From Basic Research to Clinics</span></div><div class="casAuthors">Guidotti, Giulia; Brambilla, Liliana; Rossi, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">406-424</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The presence of cell and tissue barriers together with the low biomembrane permeability of various therapeutics often hampers systemic drug distribution; thus, most of the available mols. are of limited therapeutic value.  Opportunities to increase medicament concns. in areas that are difficult to access now exist with the advent of cell-penetrating peptides (CPPs), which can transport into the cell a wide variety of biol. active conjugates (cargoes).  Numerous preclin. evaluations with CPP-derived therapeutics have provided promising results in various disease models that, in some cases, prompted clin. trials.  The outcome of these investigations has thus opened new perspectives for CPP application in the development of unprecedented human therapies that are well tolerated and directed to intracellular targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG2xC3lO7EbrVg90H21EOLACvtfcHk0lhcNoJvEZqyuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeitr8%253D&md5=58d735c7bea04d19417baa4e2cb49102</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2017.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2017.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGuidotti%26aufirst%3DG.%26aulast%3DBrambilla%26aufirst%3DL.%26aulast%3DRossi%26aufirst%3DD.%26atitle%3DCell-penetrating%2520peptides%253A%2520from%2520basic%2520research%2520to%2520clinics%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2017%26volume%3D38%26issue%3D4%26spage%3D406%26epage%3D424%26doi%3D10.1016%2Fj.tips.2017.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strohl, W. R.</span></span> <span> </span><span class="NLM_article-title">Fusion proteins for half-life extension of biologics as a strategy to make biobetters</span>. <i>BioDrugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1007/s40259-015-0133-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1007%2Fs40259-015-0133-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=26177629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WrtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=215-239&author=W.+R.+Strohl&title=Fusion+proteins+for+half-life+extension+of+biologics+as+a+strategy+to+make+biobetters&doi=10.1007%2Fs40259-015-0133-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fusion Proteins for Half-Life Extension of Biologics as a Strategy to Make Biobetters</span></div><div class="casAuthors">Strohl, William R.</div><div class="citationInfo"><span class="NLM_cas:title">BioDrugs</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215-239</span>CODEN:
                <span class="NLM_cas:coden">BIDRF4</span>;
        ISSN:<span class="NLM_cas:issn">1173-8804</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">The purpose of making a "biobetter" biol. is to improve on the salient characteristics of a known biol. for which there is, minimally, clin. proof of concept or, maximally, marketed product data.  There already are several examples in which second-generation or biobetter biologics have been generated by improving the pharmacokinetic properties of an innovative drug, including Neulasta [a PEGylated, longer-half-life version of Neupogen (filgrastim)] and Aranesp [a longer-half-life version of Epogen (epoetin-α)].  This review describes the use of protein fusion technologies such as Fc fusion proteins, fusion to human serum albumin, fusion to carboxy-terminal peptide, and other polypeptide fusion approaches to make biobetter drugs with more desirable pharmacokinetic profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqorgYj1NyuaLVg90H21EOLACvtfcHk0lhcNoJvEZqyuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WrtLjN&md5=328f936b91e29c83e069db83a4d05cab</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs40259-015-0133-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40259-015-0133-6%26sid%3Dliteratum%253Aachs%26aulast%3DStrohl%26aufirst%3DW.%2BR.%26atitle%3DFusion%2520proteins%2520for%2520half-life%2520extension%2520of%2520biologics%2520as%2520a%2520strategy%2520to%2520make%2520biobetters%26jtitle%3DBioDrugs%26date%3D2015%26volume%3D29%26spage%3D215%26epage%3D239%26doi%3D10.1007%2Fs40259-015-0133-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span> <span> </span><span class="NLM_article-title">Strategic approaches to optimizing peptide ADME properties</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1208/s12248-014-9687-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1208%2Fs12248-014-9687-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=25366889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=134-143&issue=1&author=L.+Di&title=Strategic+approaches+to+optimizing+peptide+ADME+properties&doi=10.1208%2Fs12248-014-9687-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Strategic Approaches to Optimizing Peptide ADME Properties</span></div><div class="casAuthors">Di, Li</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">134-143</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Development of peptide drugs is challenging but also quite rewarding.  Five blockbuster peptide drugs are currently on the market, and six new peptides received first marketing approval as new mol. entities in 2012.  Although peptides only represent 2% of the drug market, the market is growing twice as quickly and might soon occupy a larger niche.  Natural peptides typically have poor absorption, distribution, metab., and excretion (ADME) properties with rapid clearance, short half-life, low permeability, and sometimes low soly.  Strategies have been developed to improve peptide drugability through enhancing permeability, reducing proteolysis and renal clearance, and prolonging half-life.  In vivo, in vitro, and in silico tools are available to evaluate ADME properties of peptides, and structural modification strategies are in place to improve peptide developability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDLDsR-Zk0OLVg90H21EOLACvtfcHk0ljW3C62Fwln4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ&md5=530597d3719d65509105096f433fee0a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1208%2Fs12248-014-9687-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-014-9687-3%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26atitle%3DStrategic%2520approaches%2520to%2520optimizing%2520peptide%2520ADME%2520properties%26jtitle%3DAAPS%2520J.%26date%3D2015%26volume%3D17%26issue%3D1%26spage%3D134%26epage%3D143%26doi%3D10.1208%2Fs12248-014-9687-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalafatovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giralt, E.</span></span> <span> </span><span class="NLM_article-title">Cell-penetrating peptides: design strategies beyond primary structure and amphipathicity</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1929</span>, <span class="refDoi"> DOI: 10.3390/molecules22111929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.3390%2Fmolecules22111929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVGjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1929&issue=11&author=D.+Kalafatovicauthor=E.+Giralt&title=Cell-penetrating+peptides%3A+design+strategies+beyond+primary+structure+and+amphipathicity&doi=10.3390%2Fmolecules22111929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-penetrating peptides: design strategies beyond primary structure and amphipathicity</span></div><div class="casAuthors">Kalafatovic, Daniela; Giralt, Ernest</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1929/1-1929/38</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Efficient intracellular drug delivery and target specificity are often hampered by the presence of biol. barriers.  Thus, compds. that efficiently cross cell membranes are the key to improving the therapeutic value and on-target specificity of non-permeable drugs.  The discovery of cell-penetrating peptides (CPPs) and the early design approaches through mimicking the natural penetration domains used by viruses have led to greater efficiency of intracellular delivery.  Following these nature-inspired examples, a no. of rationally designed CPPs has been developed.  In this review, a variety of CPP designs will be described, including linear and flexible, pos. charged and often amphipathic CPPs, and more rigid versions comprising cyclic, stapled, or dimeric and/or multivalent, self-assembled peptides or peptido-mimetics.  The application of distinct design strategies to known physico-chem. properties of CPPs offers the opportunity to improve their penetration efficiency and/or internalization kinetics.  This led to increased design complexity of new CPPs that does not always result in greater CPP activity.  Therefore, the transition of CPPs to a clin. setting remains a challenge also due to the concomitant involvement of various internalization routes and heterogeneity of cells used in the in vitro studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXVMiu0WvRi7Vg90H21EOLACvtfcHk0ljW3C62Fwln4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVGjtrY%253D&md5=cde796080669da1abb1bd06fd1ed57a5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.3390%2Fmolecules22111929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules22111929%26sid%3Dliteratum%253Aachs%26aulast%3DKalafatovic%26aufirst%3DD.%26aulast%3DGiralt%26aufirst%3DE.%26atitle%3DCell-penetrating%2520peptides%253A%2520design%2520strategies%2520beyond%2520primary%2520structure%2520and%2520amphipathicity%26jtitle%3DMolecules%26date%3D2017%26volume%3D22%26issue%3D11%26spage%3D1929%26doi%3D10.3390%2Fmolecules22111929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Longbottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearer, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devine, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcoba, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappuis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz de Castañeda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pigott, D. M.</span></span> <span> </span><span class="NLM_article-title">Vulnerability to snakebite envenoming: a global mapping of hotspots</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>392</i></span> (<span class="NLM_issue">10148</span>),  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(18)31224-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2FS0140-6736%2818%2931224-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=30017551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FptFeksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2018&pages=673-684&issue=10148&author=J.+Longbottomauthor=F.+M.+Shearerauthor=M.+Devineauthor=G.+Alcobaauthor=F.+Chappuisauthor=D.+J.+Weissauthor=S.+E.+Rayauthor=N.+Rayauthor=D.+A.+Warrellauthor=R.+Ruiz+de+Casta%C3%B1edaauthor=D.+J.+Williamsauthor=S.+I.+Hayauthor=D.+M.+Pigott&title=Vulnerability+to+snakebite+envenoming%3A+a+global+mapping+of+hotspots&doi=10.1016%2FS0140-6736%2818%2931224-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Vulnerability to snakebite envenoming: a global mapping of hotspots</span></div><div class="casAuthors">Longbottom Joshua; Shearer Freya M; Devine Maria; Weiss Daniel J; Alcoba Gabriel; Chappuis Francois; Ray Sarah E; Pigott David M; Ray Nicolas; Warrell David A; Ruiz de Castaneda Rafael; Williams David J; Hay Simon I</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">392</span>
        (<span class="NLM_cas:issue">10148</span>),
    <span class="NLM_cas:pages">673-684</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Snakebite envenoming is a frequently overlooked cause of mortality and morbidity.  Data for snake ecology and existing snakebite interventions are scarce, limiting accurate burden estimation initiatives.  Low global awareness stunts new interventions, adequate health resources, and available health care.  Therefore, we aimed to synthesise currently available data to identify the most vulnerable populations at risk of snakebite, and where additional data to manage this global problem are needed.  METHODS:  We assembled a list of snake species using WHO guidelines.  Where relevant, we obtained expert opinion range (EOR) maps from WHO or the Clinical Toxinology Resources.  We also obtained occurrence data for each snake species from a variety of websites, such as VertNet and iNaturalist, using the spocc R package (version 0.7.0).  We removed duplicate occurrence data and categorised snakes into three groups: group A (no available EOR map or species occurrence records), group B (EOR map but <5 species occurrence records), and group C (EOR map and ≥5 species occurrence records).  For group C species, we did a multivariate environmental similarity analysis using the 2008 WHO EOR maps and newly available evidence.  Using these data and the EOR maps, we produced contemporary range maps for medically important venomous snake species at a 5 × 5 km resolution.  We subsequently triangulated these data with three health system metrics (antivenom availability, accessibility to urban centres, and the Healthcare Access and Quality [HAQ] Index) to identify the populations most vulnerable to snakebite morbidity and mortality.  FINDINGS:  We provide a map showing the ranges of 278 snake species globally.  Although about 6·85 billion people worldwide live within range of areas inhabited by snakes, about 146·70 million live within remote areas lacking quality health-care provisioning.  Comparing opposite ends of the HAQ Index, 272·91 million individuals (65·25%) of the population within the lowest decile are at risk of exposure to any snake for which no effective therapy exists compared with 519·46 million individuals (27·79%) within the highest HAQ Index decile, showing a disproportionate coverage in reported antivenom availability.  Antivenoms were available for 119 (43%) of 278 snake species evaluated by WHO, while globally 750·19 million (10·95%) of those living within snake ranges live more than 1 h from population centres.  In total, we identify about 92·66 million people living within these vulnerable geographies, including many sub-Saharan countries, Indonesia, and other parts of southeast Asia.  INTERPRETATION:  Identifying exact populations vulnerable to the most severe outcomes of snakebite envenoming at a subnational level is important for prioritising new data collection and collation, reinforcing envenoming treatment, existing health-care systems, and deploying currently available and future interventions.  These maps can guide future research efforts on snakebite envenoming from both ecological and public health perspectives and better target future estimates of the burden of this neglected tropical disease.  FUNDING:  Bill & Melinda Gates Foundation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6cc4xiLoYKv6BottYdIAWfW6udTcc2ebNxHtg1LdYRLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FptFeksw%253D%253D&md5=630c65ee36b0a83ae04585d3dbdaaf87</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2818%2931224-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252818%252931224-8%26sid%3Dliteratum%253Aachs%26aulast%3DLongbottom%26aufirst%3DJ.%26aulast%3DShearer%26aufirst%3DF.%2BM.%26aulast%3DDevine%26aufirst%3DM.%26aulast%3DAlcoba%26aufirst%3DG.%26aulast%3DChappuis%26aufirst%3DF.%26aulast%3DWeiss%26aufirst%3DD.%2BJ.%26aulast%3DRay%26aufirst%3DS.%2BE.%26aulast%3DRay%26aufirst%3DN.%26aulast%3DWarrell%26aufirst%3DD.%2BA.%26aulast%3DRuiz%2Bde%2BCasta%25C3%25B1eda%26aufirst%3DR.%26aulast%3DWilliams%26aufirst%3DD.%2BJ.%26aulast%3DHay%26aufirst%3DS.%2BI.%26aulast%3DPigott%26aufirst%3DD.%2BM.%26atitle%3DVulnerability%2520to%2520snakebite%2520envenoming%253A%2520a%2520global%2520mapping%2520of%2520hotspots%26jtitle%3DLancet%26date%3D2018%26volume%3D392%26issue%3D10148%26spage%3D673%26epage%3D684%26doi%3D10.1016%2FS0140-6736%2818%2931224-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, B.</span></span> <span> </span><span class="NLM_article-title">Synthetic antivenoms for snakebites: a slithering road!</span>. <i>Biochemist</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">6</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1042/BIO04106006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1042%2FBIO04106006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=6-9&issue=6&author=B.+Lohse&title=Synthetic+antivenoms+for+snakebites%3A+a+slithering+road%21&doi=10.1042%2FBIO04106006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1042%2FBIO04106006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBIO04106006%26sid%3Dliteratum%253Aachs%26aulast%3DLohse%26aufirst%3DB.%26atitle%3DSynthetic%2520antivenoms%2520for%2520snakebites%253A%2520a%2520slithering%2520road%2521%26jtitle%3DBiochemist%26date%3D2019%26volume%3D41%26issue%3D6%26spage%3D6%26epage%3D9%26doi%3D10.1042%2FBIO04106006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lynagh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komnatnyy, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pless, S. A.</span></span> <span> </span><span class="NLM_article-title">Unique contributions of an arginine side chain to ligand recognition in a glutamate-gated chloride channel</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3940</span>– <span class="NLM_lpage">3946</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.772939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1074%2Fjbc.M116.772939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28096462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjslSju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=3940-3946&issue=9&author=T.+Lynaghauthor=V.+V.+Komnatnyyauthor=S.+A.+Pless&title=Unique+contributions+of+an+arginine+side+chain+to+ligand+recognition+in+a+glutamate-gated+chloride+channel&doi=10.1074%2Fjbc.M116.772939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Unique Contributions of an Arginine Side Chain to Ligand Recognition in a Glutamate-gated Chloride Channel</span></div><div class="casAuthors">Lynagh, Timothy; Komnatnyy, Vitaly V.; Pless, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3940-3946</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Glutamate recognition by neurotransmitter receptors often relies on Arg residues in the binding site, leading to the assumption that charge-charge interactions underlie ligand recognition.  However, assessing the precise chem. contribution of Arg side chains to protein function and pharmacol. has proven to be exceedingly difficult in such large and complex proteins.  Using the in vivo nonsense suppression approach, we report the first successful incorporation of the isosteric, titratable Arg analog, canavanine, into a neurotransmitter receptor in a living cell, utilizing a glutamate-gated chloride channel from the nematode Haemonchus contortus.  Our data unveil a surprisingly small contribution of charge at a conserved arginine side chain previously suggested to form a salt bridge with the ligand, glutamate.  Instead, our data show that Arg contributes crucially to ligand sensitivity via a hydrogen bond network, where Arg interacts both with agonist and with a conserved Thr side chain within the receptor.  Together, the data provide a new explanation for the reliance of neurotransmitter receptors on Arg side chains and highlight the exceptional capacity of unnatural amino acid incorporation for increasing our understanding of ligand recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2BltxAbOrdrVg90H21EOLACvtfcHk0li-8qQilp95gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjslSju7k%253D&md5=c5bddfb68e38f3b286c43e4c453b6e29</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.772939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.772939%26sid%3Dliteratum%253Aachs%26aulast%3DLynagh%26aufirst%3DT.%26aulast%3DKomnatnyy%26aufirst%3DV.%2BV.%26aulast%3DPless%26aufirst%3DS.%2BA.%26atitle%3DUnique%2520contributions%2520of%2520an%2520arginine%2520side%2520chain%2520to%2520ligand%2520recognition%2520in%2520a%2520glutamate-gated%2520chloride%2520channel%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26issue%3D9%26spage%3D3940%26epage%3D3946%26doi%3D10.1074%2Fjbc.M116.772939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Revah, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galzi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devillers-Thiéry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballivet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J. P.</span></span> <span> </span><span class="NLM_article-title">Mutations in the channel domain alter desensitization of a neuronal nicotinic receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>353</i></span> (<span class="NLM_issue">6347</span>),  <span class="NLM_fpage">846</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.1038/353846a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2F353846a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1719423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADyaK38XjvFKk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=1991&pages=846-849&issue=6347&author=F.+Revahauthor=D.+Bertrandauthor=J.+L.+Galziauthor=A.+Devillers-Thi%C3%A9ryauthor=C.+Mulleauthor=N.+Hussyauthor=S.+Bertrandauthor=M.+Ballivetauthor=J.+P.+Changeux&title=Mutations+in+the+channel+domain+alter+desensitization+of+a+neuronal+nicotinic+receptor&doi=10.1038%2F353846a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the channel domain alter desensitization of a neuronal nicotinic receptor</span></div><div class="casAuthors">Revah, F.; Bertrand, D.; Galzi, J. L.; Devillers-Thiery, A.; Mulle, C.; Hussy, N.; Bertrand, S.; Ballivet, M.; Changeux, J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6347</span>),
    <span class="NLM_cas:pages">846-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The authors investigated how structural elements could be involved in the desensitization of the acetylcholine-gated ion channel from the chick brain α-bungarotoxin-sensitive homo-oligomeric α7 receptor, using site-directed mutagenesis and expression in Xenopus oocytes.  Mutations of the highly conserved leucine 247 residue from the uncharged MII segment of α7 suppress inhibition by the open-channel blocker QX-222, indicating that this residue, like others from MII, faces the lumen of the channel.  But, unexpectedly, the same mutations decrease the rate of desensitization of the response, increase the apparent affinity for acetylcholine and abolish current rectification.  Moreover, unlike wild-type α7, which has channels with a single conductance level, the leucine-to-threonine mutant has an addnl. conducting state active at low acetylcholine concns.  It is possible that mutation of Leu247 renders conductive 1 of the high-affinity desensitized states of the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwHhZKCopf5LVg90H21EOLACvtfcHk0li-8qQilp95gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XjvFKk&md5=41c3680c40d1fe9fc28f9bf1cbee46c6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2F353846a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F353846a0%26sid%3Dliteratum%253Aachs%26aulast%3DRevah%26aufirst%3DF.%26aulast%3DBertrand%26aufirst%3DD.%26aulast%3DGalzi%26aufirst%3DJ.%2BL.%26aulast%3DDevillers-Thi%25C3%25A9ry%26aufirst%3DA.%26aulast%3DMulle%26aufirst%3DC.%26aulast%3DHussy%26aufirst%3DN.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DBallivet%26aufirst%3DM.%26aulast%3DChangeux%26aufirst%3DJ.%2BP.%26atitle%3DMutations%2520in%2520the%2520channel%2520domain%2520alter%2520desensitization%2520of%2520a%2520neuronal%2520nicotinic%2520receptor%26jtitle%3DNature%26date%3D1991%26volume%3D353%26issue%3D6347%26spage%3D846%26epage%3D849%26doi%3D10.1038%2F353846a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halevi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yassin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treinin, M.</span></span> <span> </span><span class="NLM_article-title">Conservation within the RIC-3 Gene Family: effectors of mammalian nicotinic acetylcholine receptor expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">34411</span>– <span class="NLM_lpage">34417</span>, <span class="refDoi"> DOI: 10.1074/jbc.M300170200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1074%2Fjbc.M300170200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=12821669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvVCqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=34411-34417&issue=36&author=S.+Haleviauthor=L.+Yassinauthor=M.+Eshelauthor=F.+Salaauthor=S.+Salaauthor=M.+Criadoauthor=M.+Treinin&title=Conservation+within+the+RIC-3+Gene+Family%3A+effectors+of+mammalian+nicotinic+acetylcholine+receptor+expression&doi=10.1074%2Fjbc.M300170200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation within the RIC-3 gene family: effectors of mammalian nicotinic acetylcholine receptor expression</span></div><div class="casAuthors">Halevi, Sarah; Yassin, Lina; Eshel, Margalit; Sala, Francisco; Sala, Salvador; Criado, Manuel; Treinin, Millet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">34411-34417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In Caenorhabditis elegans, the ric-3 gene is required for the maturation of multiple nicotinic acetylcholine receptors (nAChRs), whereas other neurotransmitter gated channels expressed within the same cells are unaffected by the presence of RIC-3.  RIC-3 is a member of a conserved gene family with representatives in both vertebrates and invertebrates.  All members of this family have two transmembrane domains followed by a coiled-coil domain.  Expression of the human ric-3 homolog, hric3, like the C. elegans ric-3, enhances C. elegans DEG-3/DES-2, rat α7, and human α7 nAChR-dependent whole-cell current amplitudes in Xenopus laevis oocytes, thus demonstrating functional conservation.  However, hric3 also reduces human α4β2 and α3β4 nAChR-dependent whole-cell current amplitudes.  Thus, hric3 shows differential effects on human nAChRs unlike the obsd. uniform effect of ric-3 on C. elegans nAChRs.  Moreover, hric3 totally abolished currents evoked by 5-HT3 serotonin receptors, whereas it barely modified α1 glycine receptor currents.  With this caveat, RIC-3 belongs to a conserved family of genes likely to regulate nAChR-mediated transmission throughout evolution.  Anal. of transcripts encoded by the hric3 locus shows that it encodes for multiple transcripts, likely to produce multiple hric3 isoforms, and that hric3 is expressed in neurons and muscles, thus enabling its interactions with nAChRs in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiT7p8XOtg3bVg90H21EOLACvtfcHk0li-8qQilp95gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvVCqsbc%253D&md5=acb379bab49af26982974eec4cb1f180</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M300170200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M300170200%26sid%3Dliteratum%253Aachs%26aulast%3DHalevi%26aufirst%3DS.%26aulast%3DYassin%26aufirst%3DL.%26aulast%3DEshel%26aufirst%3DM.%26aulast%3DSala%26aufirst%3DF.%26aulast%3DSala%26aufirst%3DS.%26aulast%3DCriado%26aufirst%3DM.%26aulast%3DTreinin%26aufirst%3DM.%26atitle%3DConservation%2520within%2520the%2520RIC-3%2520Gene%2520Family%253A%2520effectors%2520of%2520mammalian%2520nicotinic%2520acetylcholine%2520receptor%2520expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26issue%3D36%26spage%3D34411%26epage%3D34417%26doi%3D10.1074%2Fjbc.M300170200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dibas, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auyeung, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezanilla, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lester, H. A.</span></span> <span> </span><span class="NLM_article-title">A fluorophore attached to nicotinic acetylcholine receptor βM2 detects productive binding of agonist to the αδ site</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">10195</span>– <span class="NLM_lpage">10200</span>, <span class="refDoi"> DOI: 10.1073/pnas.0301885101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1073%2Fpnas.0301885101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15218096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFOhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=10195-10200&issue=27&author=D.+S.+Dahanauthor=M.+I.+Dibasauthor=E.+J.+Peterssonauthor=V.+C.+Auyeungauthor=B.+Chandaauthor=F.+Bezanillaauthor=D.+A.+Doughertyauthor=H.+A.+Lester&title=A+fluorophore+attached+to+nicotinic+acetylcholine+receptor+%CE%B2M2+detects+productive+binding+of+agonist+to+the+%CE%B1%CE%B4+site&doi=10.1073%2Fpnas.0301885101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorophore attached to nicotinic acetylcholine receptor βM2 detects productive binding of agonist to the αδ site</span></div><div class="casAuthors">Dahan, David S.; Dibas, Mohammed I.; Petersson, E. James; Auyeung, Vincent C.; Chanda, Baron; Bezanilla, Francisco; Dougherty, Dennis A.; Lester, Henry A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">10195-10200</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">To study conformational transitions at the muscle nicotinic acetylcholine (ACh) receptor (nAChR), a rhodamine fluorophore was tethered to a Cys side chain introduced at the β19' position in the M2 region of the nAChR expressed in Xenopus oocytes.  This procedure led to only minor changes in receptor function.  During agonist application, fluorescence increased by (ΔF/F) ≈10%, and the emission peak shifted to lower wavelengths, indicating a more hydrophobic environment for the fluorophore.  The dose-response relations for ΔF agreed well with those for epibatidine-induced currents, but were shifted ≈100-fold to the left of those for ACh-induced currents.  Because (i) epibatidine binds more tightly to the αγ-binding site than to the αδ site and (ii) ACh binds with reverse-site selectivity, these data suggest that ΔF monitors an event linked to binding specifically at the αδ-subunit interface.  In expts. with flash-applied agonists, the earliest detectable ΔF occurs within milliseconds, i.e., during activation.  At low [ACh] (≤ 10 μM), a phase of ΔF occurs with the same time const. as desensitization, presumably monitoring an increased population of agonist-bound receptors.  However, recovery from ΔF is complete before the slowest phase of recovery from desensitization (time const. ≈250 s), showing that one or more desensitized states have fluorescence like that of the resting channel.  That conformational transitions at the αδ-binding site are not tightly coupled to channel activation suggests that sequential rather than fully concerted transitions occur during receptor gating.  Thus, time-resolved fluorescence changes provide a powerful probe of nAChR conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGQiNXhLapRbVg90H21EOLACvtfcHk0lgZNfnge6l7GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFOhsb4%253D&md5=834c616a35697f1e131bfd5701595afe</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0301885101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0301885101%26sid%3Dliteratum%253Aachs%26aulast%3DDahan%26aufirst%3DD.%2BS.%26aulast%3DDibas%26aufirst%3DM.%2BI.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DAuyeung%26aufirst%3DV.%2BC.%26aulast%3DChanda%26aufirst%3DB.%26aulast%3DBezanilla%26aufirst%3DF.%26aulast%3DDougherty%26aufirst%3DD.%2BA.%26aulast%3DLester%26aufirst%3DH.%2BA.%26atitle%3DA%2520fluorophore%2520attached%2520to%2520nicotinic%2520acetylcholine%2520receptor%2520%25CE%25B2M2%2520detects%2520productive%2520binding%2520of%2520agonist%2520to%2520the%2520%25CE%25B1%25CE%25B4%2520site%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D27%26spage%3D10195%26epage%3D10200%26doi%3D10.1073%2Fpnas.0301885101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&issue=2&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization&doi=10.1016%2Fj.ab.2004.05.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0lgZNfnge6l7GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26issue%3D2%26spage%3D261%26epage%3D273%26doi%3D10.1016%2Fj.ab.2004.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellmig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paithankar, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puhringer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zocher, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">Facilities for macromolecular crystallography at the Helmholtz-Zentrum Berlin</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1107/S0909049512006395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0909049512006395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=22514183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFGhu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=442-449&issue=3&author=U.+Muellerauthor=N.+Darowskiauthor=M.+R.+Fuchsauthor=R.+Forsterauthor=M.+Hellmigauthor=K.+S.+Paithankarauthor=S.+Puhringerauthor=M.+Steffienauthor=G.+Zocherauthor=M.+S.+Weiss&title=Facilities+for+macromolecular+crystallography+at+the+Helmholtz-Zentrum+Berlin&doi=10.1107%2FS0909049512006395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Facilities for macromolecular crystallography at the Helmholtz-Zentrum Berlin</span></div><div class="casAuthors">Mueller, Uwe; Darowski, Nora; Fuchs, Martin R.; Foerster, Ronald; Hellmig, Michael; Paithankar, Karthik S.; Puehringer, Sandra; Steffien, Michael; Zocher, Georg; Weiss, Manfred S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">442-449</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Three macromol. crystallog. (MX) beamlines at the Helmholtz-Zentrum Berlin (HZB) are available for the regional, national and international structural biol. user community.  The state-of-the-art synchrotron beamlines for MX BL14.1, BL14.2 and BL14.3 are located within the low-β section of the BESSY II electron storage ring.  All beamlines are fed from a superconducting 7 T wavelength-shifter insertion device.  BL14.1 and BL14.2 are energy tunable in the range 5-16 keV, while BL14.3 is a fixed-energy side station operated at 13.8 keV.  All three beamlines are equipped with CCD detectors.  BL14.1 and BL14.2 are in regular user operation providing about 200 beam days per yr and about 600 user shifts to approx. 50 research groups across Europe.  BL14.3 has initially been used as a test facility and was brought into regular user mode operation during the year 2010.  BL14.1 has recently been upgraded with a microdiffractometer including a mini-κ goniometer and an automated sample changer.  Addnl. user facilities include office space adjacent to the beamlines, a sample prepn. lab., a biol. lab. (safety level 1) and high-end computing resources.  In this article the instrumentation of the beamlines is described, and a summary of the exptl. possibilities of the beamlines and the provided ancillary equipment for the user community is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEtRGj9KmXzLVg90H21EOLACvtfcHk0lgZNfnge6l7GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFGhu7o%253D&md5=f065b38da0a77d30031a9db5a67eb1cc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0909049512006395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049512006395%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DDarowski%26aufirst%3DN.%26aulast%3DFuchs%26aufirst%3DM.%2BR.%26aulast%3DForster%26aufirst%3DR.%26aulast%3DHellmig%26aufirst%3DM.%26aulast%3DPaithankar%26aufirst%3DK.%2BS.%26aulast%3DPuhringer%26aufirst%3DS.%26aulast%3DSteffien%26aufirst%3DM.%26aulast%3DZocher%26aufirst%3DG.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DFacilities%2520for%2520macromolecular%2520crystallography%2520at%2520the%2520Helmholtz-Zentrum%2520Berlin%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2012%26volume%3D19%26issue%3D3%26spage%3D442%26epage%3D449%26doi%3D10.1107%2FS0909049512006395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&issue=2&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhS5v2KzQonaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krug, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span> <span> </span><span class="NLM_article-title">XDSAPP: a graphical user interface for the convenient processing of diffraction data using XDS</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">572</span>, <span class="refDoi"> DOI: 10.1107/S0021889812011715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0021889812011715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFGmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=568-572&issue=3&author=M.+Krugauthor=M.+S.+Weissauthor=U.+Heinemannauthor=U.+Mueller&title=XDSAPP%3A+a+graphical+user+interface+for+the+convenient+processing+of+diffraction+data+using+XDS&doi=10.1107%2FS0021889812011715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">XDSAPP: a graphical user interface for the convenient processing of diffraction data using XDS</span></div><div class="casAuthors">Krug, Michael; Weiss, Manfred S.; Heinemann, Udo; Mueller, Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">568-572</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">XDSAPP is a Tcl/Tk-based graphical user interface for the easy and convenient processing of diffraction data sets using XDS.  It provides easy access to all XDS functionalities, automates the data processing and generates graphical plots of various data set statistics provided by XDS.  By incorporating addnl. software, further information on certain features of the data set, such as radiation decay during data collection or the presence of pseudo-translational symmetry and/or twinning, can be obtained.  Intensity files suitable for CCP4, CNS and SHELX are generated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqFLQ2J-k07Vg90H21EOLACvtfcHk0lhS5v2KzQonaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFGmtLo%253D&md5=8f0b78fdea7ed6cc580de9eb561e8b9e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0021889812011715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889812011715%26sid%3Dliteratum%253Aachs%26aulast%3DKrug%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DHeinemann%26aufirst%3DU.%26aulast%3DMueller%26aufirst%3DU.%26atitle%3DXDSAPP%253A%2520a%2520graphical%2520user%2520interface%2520for%2520the%2520convenient%2520processing%2520of%2520diffraction%2520data%2520using%2520XDS%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2012%26volume%3D45%26issue%3D3%26spage%3D568%26epage%3D572%26doi%3D10.1107%2FS0021889812011715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhS5v2KzQonaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickitenko, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michailov, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Three-dimensional structure of neurotoxin-1 from Naja naja oxiana venom at 1.9 Å resolution</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>320</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(93)80073-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2F0014-5793%2893%2980073-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=8458425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1amt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=1993&pages=111-117&issue=2&author=A.+V.+Nickitenkoauthor=A.+M.+Michailovauthor=C.+Betzelauthor=K.+S.+Wilson&title=Three-dimensional+structure+of+neurotoxin-1+from+Naja+naja+oxiana+venom+at+1.9+%C3%85+resolution&doi=10.1016%2F0014-5793%2893%2980073-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Three-dimensional structure of neurotoxin-1 from Naja naja oxiana venom at 1.9 Å resolution</span></div><div class="casAuthors">Nickitenko, A. V.; Michailov, A. M.; Betzel, Ch.; Wilson, K. S.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-17</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    </div><div class="casAbstract">Neurotoxin-1 from N. naja oxiana venom (NTX-1) was crystd. by vapor diffusion in sitting drops.  The crystals have cell dimensions of a = 25.2 Å, b = 75.6 Å, c = 35.9 Å, and are in space group P212121.  Three-dimensional data to 1.9 Å were recorded by a Syntex P21 automatic diffractometer.  The at. structure of the toxin was detd. by mol. replacement using the α-cobratoxin (α-CTX) as the search model.  The position of 534 nonhydrogen protein atoms were detd.  The model contains 65 water mols.  Refinement has led to an R-factor of 19.3% at 1.9 Å resoln.  The secondary and tertiary structures of NTX-1 were analyzed and a comparison with structure of the α-CTX was made.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1EFZ5A5GNr7Vg90H21EOLACvtfcHk0ljPIqXZA8L7HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1amt7s%253D&md5=81d7257e408b21e60b8569571e6923d7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2893%2980073-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252893%252980073-4%26sid%3Dliteratum%253Aachs%26aulast%3DNickitenko%26aufirst%3DA.%2BV.%26aulast%3DMichailov%26aufirst%3DA.%2BM.%26aulast%3DBetzel%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DThree-dimensional%2520structure%2520of%2520neurotoxin-1%2520from%2520Naja%2520naja%2520oxiana%2520venom%2520at%25201.9%2520%25C3%2585%2520resolution%26jtitle%3DFEBS%2520Lett.%26date%3D1993%26volume%3D320%26issue%3D2%26spage%3D111%26epage%3D117%26doi%3D10.1016%2F0014-5793%2893%2980073-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12 Part 1</span>),  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&issue=12+Part+1&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljPIqXZA8L7HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D12%2520Part%25201%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&issue=3&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljPIqXZA8L7HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26issue%3D3%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José María  Gutiérrez</span>, <span class="hlFld-ContribAuthor ">Laura-Oana  Albulescu</span>, <span class="hlFld-ContribAuthor ">Rachel H.  Clare</span>, <span class="hlFld-ContribAuthor ">Nicholas R.  Casewell</span>, <span class="hlFld-ContribAuthor ">Tarek Mohamed Abd  Abd El-Aziz</span>, <span class="hlFld-ContribAuthor ">Teresa  Escalante</span>, <span class="hlFld-ContribAuthor ">Alexandra  Rucavado</span>. </span><span class="cited-content_cbyCitation_article-title">The Search for Natural and Synthetic Inhibitors That Would Complement Antivenoms as Therapeutics for Snakebite Envenoming. </span><span class="cited-content_cbyCitation_journal-name">Toxins</span><span> <strong>2021,</strong> <em>13 </em>
                                    (7)
                                     , 451. <a href="https://doi.org/10.3390/toxins13070451" title="DOI URL">https://doi.org/10.3390/toxins13070451</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/toxins13070451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Ftoxins13070451%26sid%3Dliteratum%253Aachs%26jtitle%3DToxins%26atitle%3DThe%252BSearch%252Bfor%252BNatural%252Band%252BSynthetic%252BInhibitors%252BThat%252BWould%252BComplement%252BAntivenoms%252Bas%252BTherapeutics%252Bfor%252BSnakebite%252BEnvenoming%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJos%25C3%25A9%2BMar%25C3%25ADa%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D7%26spage%3D451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel H.  Clare</span>, <span class="hlFld-ContribAuthor ">Steven R.  Hall</span>, <span class="hlFld-ContribAuthor ">Rohit N.  Patel</span>, <span class="hlFld-ContribAuthor ">Nicholas R.  Casewell</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Drug Discovery for Neglected Tropical Snakebite. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2021,</strong> <em>42 </em>
                                    (5)
                                     , 340-353. <a href="https://doi.org/10.1016/j.tips.2021.02.005" title="DOI URL">https://doi.org/10.1016/j.tips.2021.02.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2021.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2021.02.005%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DSmall%252BMolecule%252BDrug%252BDiscovery%252Bfor%252BNeglected%252BTropical%252BSnakebite%26aulast%3DClare%26aufirst%3DRachel%2BH.%26date%3D2021%26volume%3D42%26issue%3D5%26spage%3D340%26epage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Denis  Kudryavtsev</span>, <span class="hlFld-ContribAuthor ">Anastasia  Isaeva</span>, <span class="hlFld-ContribAuthor ">Daria  Barkova</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Spirova</span>, <span class="hlFld-ContribAuthor ">Renata  Mukhutdinova</span>, <span class="hlFld-ContribAuthor ">Igor  Kasheverov</span>, <span class="hlFld-ContribAuthor ">Victor  Tsetlin</span>. </span><span class="cited-content_cbyCitation_article-title">Point Mutations of Nicotinic Receptor α1 Subunit Reveal New Molecular Features of G153S Slow-Channel Myasthenia. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (5)
                                     , 1278. <a href="https://doi.org/10.3390/molecules26051278" title="DOI URL">https://doi.org/10.3390/molecules26051278</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26051278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26051278%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPoint%252BMutations%252Bof%252BNicotinic%252BReceptor%252B%2525CE%2525B11%252BSubunit%252BReveal%252BNew%252BMolecular%252BFeatures%252Bof%252BG153S%252BSlow-Channel%252BMyasthenia%26aulast%3DKudryavtsev%26aufirst%3DDenis%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D5%26spage%3D1278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Peptide <b>1</b> prevents α-Cbtx inhibition of nAChRs. (A) Example two electrode voltage clamp recordings of ACh-gated currents in <i>Xenopus laevis</i> oocytes expressing L247V mutant α7 nAChRs and RIC-3 (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>, <a class="ref internalNav" href="#sec5_3" aria-label="Electrophysiological Assays">Electrophysiological Assays</a>). In testing peptides, ACh was applied for 10 s in regular conditions (“control”), three times after the application of either α-Cbtx (blue bars) or α-Cbtx with peptide (blue and magenta bars; “App 1–App 3”), and then three more times to observe washout of any effects (“Wash 1–Wash 3”). (B) Mean ± SEM ACh-gated current amplitude, normalized to control (“<i>I</i>/<i>I</i><sub>control</sub>”). Each peptide was tested with α-Cbtx on at least three oocytes: <i>n</i> (magenta columns) = 3–4. α-Cbtx alone was tested on the same day on at least two oocytes: <i>n</i> (blue columns) = 2 (peptides <b>1</b> and <b>2</b>) – 4 (all others).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Concentration dependence of peptide activity. (A,B) Electrophysiological experiments as per <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, but here only initial ACh application (“Control”), the third ACh application after peptide + α-Cbtx (40 nM) (“App 3”), and the third ACh application after washout (“Wash 3”) are shown. Mean ± SEM (<i>n</i> = 3) are shown to the right. (C) Chemical structure of <b>9</b>. (D) Fluorescence polarization of <b>9</b> (50 nM) by increasing concentrations of α-Cbtx. Mean ± SEM, <i>n</i> = 2. (E) Concentration-dependent displacement of <b>9</b> (50 nM) from α-Cbtx (40 μM) by compound <b>1</b> and <b>12</b>. Mean ± SEM, <i>n</i> = 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/medium/jm0c01202_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of the <b>12</b>:α-Cbtx complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a>). (A) Asymmetric unit of the <b>12</b>:α-Cbtx crystal structure. Each molecule of <b>12</b> (magenta) binds two toxin molecules A and B (light and dark cyan, respectively). A second α-Cbtx-A:<b>12</b>:α-Cbtx-B complex in the asymmetric unit is related by noncrystallographic symmetry. (B) Magnified view showing interaction of α-Cbtx loop II (light cyan) with <b>12</b> (magenta) in binding mode A. Two alternative conformations for the side chain of <sup><b>12</b></sup>M2 were modeled. Key interacting amino acid residues shown in stick representation and labeled. Cation−π-stacking interactions and hydrogen bonds are highlighted by dashed and dotted lines, respectively. (C) Binding mode B, with second α-Cbtx molecule (dark cyan), from same structure with interacting residues drawn as in (B). (D) Binding mode A leads to conformational changes in α-Cbtx (light cyan) compared to the structure observed of α-Cbtx from <i>Naja naja siamensis</i> (dark gray, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CTX">2CTX</a>, root-mean-square deviation (rmsd) 1,31 Å), <i>Naja naja oxania</i> (light gray, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NTN">1NTN</a>, rmsd 1.7 Å), or <i>Naja naja kaouthia</i> in complex with AChBP (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YI5">1YI5</a>, rmsd 1.47 Å). The greatest displacement is observed in loop II (11 Å) and loop III (4.7 Å). (E) Binding mode B (cyan) resembles more closely previously reported structures of α-Cbtx from <i>Naja naja siamensis</i> (dark gray, rmsd 1,07 Å), <i>Naja naja oxania</i> (light gray, rmsd 0.67 Å), or <i>Naja naja kaouthia</i> in complex with AChBP (green, rmsd 0.87 Å). (F) Interaction of α-Bgtx (blue) loop II with the principal (α subunit) and complementary (δ subunit) faces of the muscle nAChR (pink, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UWZ">6UWZ</a>). Key interacting residues drawn as in (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01202/20201120/images/large/jm0c01202_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01202&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38125" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38125" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 45 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvete, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habib, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrell, D. A.</span></span> <span> </span><span class="NLM_article-title">Snakebite envenoming</span>. <i>Nat. Rev. Dis. Primers</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">17063</span>, <span class="refDoi"> DOI: 10.1038/nrdp.2017.63</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fnrdp.2017.63" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28905944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BC1cbotFSmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=17063&author=J.+M.+Guti%C3%A9rrezauthor=J.+J.+Calveteauthor=A.+G.+Habibauthor=R.+A.+Harrisonauthor=D.+J.+Williamsauthor=D.+A.+Warrell&title=Snakebite+envenoming&doi=10.1038%2Fnrdp.2017.63"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Snakebite envenoming</span></div><div class="casAuthors">Gutierrez Jose Maria; Calvete Juan J; Habib Abdulrazaq G; Harrison Robert A; Williams David J; Williams David J; Warrell David A</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Disease primers</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17063</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Snakebite envenoming is a neglected tropical disease that kills >100,000 people and maims >400,000 people every year.  Impoverished populations living in the rural tropics are particularly vulnerable; snakebite envenoming perpetuates the cycle of poverty.  Snake venoms are complex mixtures of proteins that exert a wide range of toxic actions.  The high variability in snake venom composition is responsible for the various clinical manifestations in envenomings, ranging from local tissue damage to potentially life-threatening systemic effects.  Intravenous administration of antivenom is the only specific treatment to counteract envenoming.  Analgesics, ventilator support, fluid therapy, haemodialysis and antibiotic therapy are also used.  Novel therapeutic alternatives based on recombinant antibody technologies and new toxin inhibitors are being explored.  Confronting snakebite envenoming at a global level demands the implementation of an integrated intervention strategy involving the WHO, the research community, antivenom manufacturers, regulatory agencies, national and regional health authorities, professional health organizations, international funding agencies, advocacy groups and civil society institutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0ERa1BdrKexBrI9DTGjC0fW6udTcc2eYOaqlP8p1Debntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbotFSmtA%253D%253D&md5=954c284c2d68cb247bac05af03df727a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrdp.2017.63&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrdp.2017.63%26sid%3Dliteratum%253Aachs%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DCalvete%26aufirst%3DJ.%2BJ.%26aulast%3DHabib%26aufirst%3DA.%2BG.%26aulast%3DHarrison%26aufirst%3DR.%2BA.%26aulast%3DWilliams%26aufirst%3DD.%2BJ.%26aulast%3DWarrell%26aufirst%3DD.%2BA.%26atitle%3DSnakebite%2520envenoming%26jtitle%3DNat.%2520Rev.%2520Dis.%2520Primers%26date%3D2017%26volume%3D3%26spage%3D17063%26doi%3D10.1038%2Fnrdp.2017.63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohapatra, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suraweera, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhingra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jotkar, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitaker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, P.</span></span> <span> </span><span class="NLM_article-title">Million Death Study, C., Snakebite mortality in India: a nationally representative mortality survey</span>. <i>PLoS Neglected Trop. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">4</span>), <span class="NLM_elocation-id">e1018</span> <span class="refDoi"> DOI: 10.1371/journal.pntd.0001018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1371%2Fjournal.pntd.0001018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=21532748" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&issue=4&author=B.+Mohapatraauthor=D.+A.+Warrellauthor=W.+Suraweeraauthor=P.+Bhatiaauthor=N.+Dhingraauthor=R.+M.+Jotkarauthor=P.+S.+Rodriguezauthor=K.+Mishraauthor=R.+Whitakerauthor=P.+Jha&title=Million+Death+Study%2C+C.%2C+Snakebite+mortality+in+India%3A+a+nationally+representative+mortality+survey&doi=10.1371%2Fjournal.pntd.0001018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pntd.0001018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pntd.0001018%26sid%3Dliteratum%253Aachs%26aulast%3DMohapatra%26aufirst%3DB.%26aulast%3DWarrell%26aufirst%3DD.%2BA.%26aulast%3DSuraweera%26aufirst%3DW.%26aulast%3DBhatia%26aufirst%3DP.%26aulast%3DDhingra%26aufirst%3DN.%26aulast%3DJotkar%26aufirst%3DR.%2BM.%26aulast%3DRodriguez%26aufirst%3DP.%2BS.%26aulast%3DMishra%26aufirst%3DK.%26aulast%3DWhitaker%26aufirst%3DR.%26aulast%3DJha%26aufirst%3DP.%26atitle%3DMillion%2520Death%2520Study%252C%2520C.%252C%2520Snakebite%2520mortality%2520in%2520India%253A%2520a%2520nationally%2520representative%2520mortality%2520survey%26jtitle%3DPLoS%2520Neglected%2520Trop.%2520Dis.%26date%3D2011%26volume%3D5%26issue%3D4%26doi%3D10.1371%2Fjournal.pntd.0001018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grace, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schütte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babo Martins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz de Castañeda, R.</span></span> <span> </span><span class="NLM_article-title">Snakebite in domestic animals: first global scoping review</span>. <i>Preventive Veterinary Medicine</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">104729</span>, <span class="refDoi"> DOI: 10.1016/j.prevetmed.2019.104729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.prevetmed.2019.104729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=31421490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BB3MvovVKhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2019&pages=104729&author=I.+Bolonauthor=M.+Finatauthor=M.+Herreraauthor=A.+Nickersonauthor=D.+Graceauthor=S.+Sch%C3%BCtteauthor=S.+Babo+Martinsauthor=R.+Ruiz+de+Casta%C3%B1eda&title=Snakebite+in+domestic+animals%3A+first+global+scoping+review&doi=10.1016%2Fj.prevetmed.2019.104729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Snakebite in domestic animals: First global scoping review</span></div><div class="casAuthors">Bolon Isabelle; Finat Matias; Babo Martins Sara; Ruiz de Castaneda Rafael; Herrera Maria; Nickerson Andrea; Grace Delia; Schutte Stephanie</div><div class="citationInfo"><span class="NLM_cas:title">Preventive veterinary medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104729</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Snakebite is a Neglected Tropical Disease estimated to cause more than 100,000 human deaths and disable more than 400,000 victims each year.  It primarily affects poor agricultural workers, farmers, and cattle herders living in rural areas of developing countries.  It is thus an occupational disease.  However, the impact of snakebite on these rural communities could be even higher if a One Health approach is taken to consider the direct impact on domestic animals and indirect impact on the livelihood of affected communities.  To explore this hypothesis we developed the first scoping review to identify and characterize the global literature on snakebite in domestic animals.  Three bibliographic databases (PubMed, Web of Science and Agricola) were searched using terms related to snake, snakebite and domestic animals for publications up to December 31(st), 2016.  Two independent reviewers screened publications applying inclusion/exclusion criteria to select relevant material.  Relevant information was then extracted from the selected literature.  The global literature on snakebite in domestic animals (n = 143 observational studies, reviews and letters) mainly represented North America, Europe and Australia (69%) with less information from Central and South America, Asia and Africa (31%).  Observational studies (n = 119) mostly concerned pets (78%) and to a lesser extent livestock (22%).  Thirty-four snake species were reported as responsible for bites in domestic animals.  WHO's Medically Important Venomous Snakes were more frequently involved.  The social-ecological determinants of snakebite were poorly documented but the reviewed literature suggested a strong seasonality and a diversity of habitat.  Snakebite in animals caused neurotoxic, cytotoxic and hemotoxic envenomation syndromes similar to humans and death.  Half of publications on envenomed livestock reported a fatality rate above 47%.  There was no literature on the indirect impact of snakebite on livelihood caused by animal morbidity and mortality.  The results of this scoping review suggest a high and under-reported burden in terms of mortality in animals and a potential economic impact of snakebite in terms of losses in livelihood of affected communities.  However, major knowledge gaps with respect to impact of snakebite on livestock and livelihood were identified.  Filling these gaps is necessary for a full understanding of the impact of snakebite and to raise scientific, political and public awareness on this neglected issue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfSxzB4H-OUb5XHngOXiV_fW6udTcc2eYOaqlP8p1Debntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvovVKhsQ%253D%253D&md5=bf7ede624000ddffe870f0acba39ed45</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.prevetmed.2019.104729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.prevetmed.2019.104729%26sid%3Dliteratum%253Aachs%26aulast%3DBolon%26aufirst%3DI.%26aulast%3DFinat%26aufirst%3DM.%26aulast%3DHerrera%26aufirst%3DM.%26aulast%3DNickerson%26aufirst%3DA.%26aulast%3DGrace%26aufirst%3DD.%26aulast%3DSch%25C3%25BCtte%26aufirst%3DS.%26aulast%3DBabo%2BMartins%26aufirst%3DS.%26aulast%3DRuiz%2Bde%2BCasta%25C3%25B1eda%26aufirst%3DR.%26atitle%3DSnakebite%2520in%2520domestic%2520animals%253A%2520first%2520global%2520scoping%2520review%26jtitle%3DPreventive%2520Veterinary%2520Medicine%26date%3D2019%26volume%3D170%26spage%3D104729%26doi%3D10.1016%2Fj.prevetmed.2019.104729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Silva, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, H. J.</span></span> <span> </span><span class="NLM_article-title">Adverse reactions to snake antivenom, and their prevention and treatment</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>81</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1111/bcp.12739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1111%2Fbcp.12739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=26256124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BC287htFGgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2016&pages=446-452&issue=3&author=H.+A.+de%0ASilvaauthor=N.+M.+Ryanauthor=H.+J.+de+Silva&title=Adverse+reactions+to+snake+antivenom%2C+and+their+prevention+and+treatment&doi=10.1111%2Fbcp.12739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Adverse reactions to snake antivenom, and their prevention and treatment</span></div><div class="casAuthors">de Silva H Asita; de Silva H Janaka; Ryan Nicole M</div><div class="citationInfo"><span class="NLM_cas:title">British journal of clinical pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-52</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Antivenom is the mainstay of treatment of snakebite envenoming.  However, adverse reactions to snake antivenom that is available are common in many parts of the world where snakebite is prevalent.  Both acute (anaphylactic or pyrogenic) and delayed (serum sickness type) reactions occur.  Acute reactions are usually mild but severe systemic anaphylaxis may develop, often within an hour or so of exposure to antivenom.  Serum sickness after antivenom has a delayed onset between 5 and 14 days after its administration.  Ultimately, the prevention reactions will depend mainly on improving the quality of antivenom.  Until these overdue improvements take place, doctors will have to depend on pharmacological prophylaxis, where the search for the best prophylactic agent is still on-going, as well as careful observation of patients receiving antivenom in preparation for prompt management of acute as well as delayed reactions when they occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTuLdwobba-MjAEfvC7EH1LfW6udTcc2eYJK6CeXA0ZJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287htFGgug%253D%253D&md5=b0d685d2386759e707520aeda7eae050</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fbcp.12739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcp.12739%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSilva%26aufirst%3DH.%2BA.%26aulast%3DRyan%26aufirst%3DN.%2BM.%26aulast%3Dde%2BSilva%26aufirst%3DH.%2BJ.%26atitle%3DAdverse%2520reactions%2520to%2520snake%2520antivenom%252C%2520and%2520their%2520prevention%2520and%2520treatment%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2016%26volume%3D81%26issue%3D3%26spage%3D446%26epage%3D452%26doi%3D10.1111%2Fbcp.12739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">León, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span> <span> </span><span class="NLM_article-title">Current technology for the industrial manufacture of snake antivenoms</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2018.06.084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2018.06.084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=29959968" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1CnurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=63-73&author=G.+Le%C3%B3nauthor=M.+Vargasauthor=%C3%81.+Seguraauthor=M.+Herreraauthor=M.+Villaltaauthor=A.+S%C3%A1nchezauthor=G.+Solanoauthor=A.+G%C3%B3mezauthor=M.+S%C3%A1nchezauthor=R.+Estradaauthor=J.+M.+Guti%C3%A9rrez&title=Current+technology+for+the+industrial+manufacture+of+snake+antivenoms&doi=10.1016%2Fj.toxicon.2018.06.084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Current technology for the industrial manufacture of snake antivenoms</span></div><div class="casAuthors">Leon, Guillermo; Vargas, Mariangela; Segura, Alvaro; Herrera, Maria; Villalta, Mauren; Sanchez, Andres; Solano, Gabriela; Gomez, Aaron; Sanchez, Melvin; Estrada, Ricardo; Gutierrez, Jose Maria</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-73</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Snake antivenoms are formulations of animal Igs used in the treatment of snakebite envenomation.  The general scheme for producing snake antivenoms has undergone few changes since its development more than a century ago; however, technol. innovations have been introduced in the manufg. process.  These medicines must comply with identity, purity, safety and efficacy profiles, as requested by the current Good Manufg. Practices (GMPs) applied to modern biopharmaceutical drugs.  Industrial prodn. of snake antivenoms comprises several stages, such as: 1. prodn. of ref. venom pools, 2. prodn. of hyperimmune plasma, 3. purifn. of the antivenom Igs, 4. formulation of the antivenom, 5. stabilization of the formulation, and 6. quality control of in-process and final products.  In this work, a general review of the existing technol. used for the industrial manuf. of snake antivenoms is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzTJNIEQUD1rVg90H21EOLACvtfcHk0lieo71vXm-s8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1CnurrM&md5=a651ddb9a1051cafef83dd1b180fa1bc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2018.06.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2018.06.084%26sid%3Dliteratum%253Aachs%26aulast%3DLe%25C3%25B3n%26aufirst%3DG.%26aulast%3DVargas%26aufirst%3DM.%26aulast%3DSegura%26aufirst%3D%25C3%2581.%26aulast%3DHerrera%26aufirst%3DM.%26aulast%3DVillalta%26aufirst%3DM.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DA.%26aulast%3DSolano%26aufirst%3DG.%26aulast%3DG%25C3%25B3mez%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez%26aufirst%3DM.%26aulast%3DEstrada%26aufirst%3DR.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26atitle%3DCurrent%2520technology%2520for%2520the%2520industrial%2520manufacture%2520of%2520snake%2520antivenoms%26jtitle%3DToxicon%26date%3D2018%26volume%3D151%26spage%3D63%26epage%3D73%26doi%3D10.1016%2Fj.toxicon.2018.06.084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span> </span><span class="NLM_article-title">Snake bite—the neglected tropical disease</span>.  <i>Lancet</i>, <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">386</span> (<span class="NLM_issue">9999</span>)  <span class="NLM_fpage">1110</span>.<span class="refDoi"> DOI: 10.1016/S0140-6736(15)00247-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2FS0140-6736%2815%2900247-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Snake+bite%E2%80%94the+neglected+tropical+disease.+Lancet%2C+2015%2C+386+%289999%29+1110.+10.1016%2FS0140-6736%2815%2900247-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2900247-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252900247-0%26sid%3Dliteratum%253Aachs%26atitle%3DSnake%2520bite%25E2%2580%2594the%2520neglected%2520tropical%2520disease%26jtitle%3DLancet%26date%3D2015%26volume%3D386%26issue%3D9999%26spage%3D1110%26doi%3D10.1016%2FS0140-6736%2815%2900247-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laustsen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engmark, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milbo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannesen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomonte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, B.</span></span> <span> </span><span class="NLM_article-title">From fangs to pharmacology: The future of snakebite envenoming therapy</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">34</span>),  <span class="NLM_fpage">5270</span>– <span class="NLM_lpage">5293</span>, <span class="refDoi"> DOI: 10.2174/1381612822666160623073438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.2174%2F1381612822666160623073438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=27339430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOhu7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5270-5293&issue=34&author=A.+H.+Laustsenauthor=M.+Engmarkauthor=C.+Milboauthor=J.+Johannesenauthor=B.+Lomonteauthor=J.+M.+Guti%C3%A9rrezauthor=B.+Lohse&title=From+fangs+to+pharmacology%3A+The+future+of+snakebite+envenoming+therapy&doi=10.2174%2F1381612822666160623073438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">From Fangs to Pharmacology: The Future of Snakebite Envenoming Therapy</span></div><div class="casAuthors">Laustsen, Andreas H.; Engmark, Mikael; Milbo, Christina; Johannesen, Jonas; Lomonte, Bruno; Gutierrez, Jose Maria; Lohse, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">5270-5293</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The snake is the symbol of medicine due to its assocn. with Asclepius, the Greek god of medicine, and so with good reasons.  More than 725 species of venomous snakes have toxins specifically evolved to exert potent bioactivity in prey or victims, and snakebites constitute a public health hazard of high impact in Asia, Africa, Latin America, and parts of Oceania.  Parenteral administration of antivenoms is the mainstay in snakebite envenoming therapy.  However, despite well-demonstrated efficacy and safety of many antivenoms worldwide, they are still being produced by traditional animal immunization procedures, and therefore present a no. of drawbacks.  Technol. advances within biopharmaceutical development and medicinal chem. could pave the way for rational drug design approaches against snake toxins.  This could minimize the use of animals and bring forward more effective therapies for snakebite envenomings.  In this review, current state-of-the-art in biopharmaceutical antitoxin development is presented together with an overview of available bioinformatics and structural data on snake venom toxins.  This growing body of scientific and technol. tools could define the basis for introducing a rational drug design approach into the field of snakebite envenoming therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozSZqbnE51C7Vg90H21EOLACvtfcHk0lg_hUVTFBJTsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOhu7%252FP&md5=c8815e3930a53d46e60e2cd1b0287de3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1381612822666160623073438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612822666160623073438%26sid%3Dliteratum%253Aachs%26aulast%3DLaustsen%26aufirst%3DA.%2BH.%26aulast%3DEngmark%26aufirst%3DM.%26aulast%3DMilbo%26aufirst%3DC.%26aulast%3DJohannesen%26aufirst%3DJ.%26aulast%3DLomonte%26aufirst%3DB.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DLohse%26aufirst%3DB.%26atitle%3DFrom%2520fangs%2520to%2520pharmacology%253A%2520The%2520future%2520of%2520snakebite%2520envenoming%2520therapy%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2016%26volume%3D22%26issue%3D34%26spage%3D5270%26epage%3D5293%26doi%3D10.2174%2F1381612822666160623073438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layfield, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallance, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicknell, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trim, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaiyapuri, S.</span></span> <span> </span><span class="NLM_article-title">The urgent need to develop novel strategies for the diagnosis and treatment of snakebites</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">363</span>, <span class="refDoi"> DOI: 10.3390/toxins11060363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.3390%2Ftoxins11060363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlGn" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=363&issue=6&author=H.+F.+Williamsauthor=H.+J.+Layfieldauthor=T.+Vallanceauthor=K.+Patelauthor=A.+B.+Bicknellauthor=S.+A.+Trimauthor=S.+Vaiyapuri&title=The+urgent+need+to+develop+novel+strategies+for+the+diagnosis+and+treatment+of+snakebites&doi=10.3390%2Ftoxins11060363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The urgent need to develop novel strategies for the diagnosis and treatment of snakebites</span></div><div class="casAuthors">Williams, Harry F.; Layfield, Harry J.; Vallance, Thomas; Patel, Ketan; Bicknell, Andrew B.; Trim, Steven A.; Vaiyapuri, Sakthivel</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">363</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Snakebite envenoming (SBE) is a priority neglected tropical disease, which kills in excess of 100,000 people per yr.  Addnl., many millions of survivors also suffer through disabilities and long-term health consequences.  The only treatment for SBE, antivenom, has a no. of major assocd. problems, not least, adverse reactions and limited availability.  This emphasizes the necessity for urgent improvements to the management of this disease.  Administration of antivenom is too frequently based on symptomatol., which results in wasting crucial time.  The majority of SBE-affected regions rely on broad-spectrum polyvalent antivenoms that have a low content of case-specific efficacious Igs.  Research into small mol. therapeutics such as varespladib/methyl-varespladib (PLA2 inhibitors) and batimastat/marimastat (metalloprotease inhibitors) suggest that such adjunctive treatments could be hugely beneficial to victims.  Progress into toxin-specific monoclonal antibodies as well as alternative binding scaffolds such as aptamers hold much promise for future treatment strategies.  SBE is not implicit during snakebite, due to venom metering.  Thus, the delay between bite and symptom presentation is crit. and when symptoms appear it may often already be too late to effectively treat SBE.  The development of reliable diagnostical tools could therefore initiate a paradigm shift in the treatment of SBE.  While the complete eradication of SBE is an impossibility, mitigation is in the pipeline, with new treatments and diagnostics rapidly emerging.  Here we critically review the urgent necessity for the development of diagnostic tools and improved therapeutics to mitigate the deaths and disabilities caused by SBE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov52jwUZ73CLVg90H21EOLACvtfcHk0lg_hUVTFBJTsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlGn&md5=07ebd90157a62fc89624d7e8b3450fbb</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3390%2Ftoxins11060363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins11060363%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DH.%2BF.%26aulast%3DLayfield%26aufirst%3DH.%2BJ.%26aulast%3DVallance%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DBicknell%26aufirst%3DA.%2BB.%26aulast%3DTrim%26aufirst%3DS.%2BA.%26aulast%3DVaiyapuri%26aufirst%3DS.%26atitle%3DThe%2520urgent%2520need%2520to%2520develop%2520novel%2520strategies%2520for%2520the%2520diagnosis%2520and%2520treatment%2520of%2520snakebites%26jtitle%3DToxins%26date%3D2019%26volume%3D11%26issue%3D6%26spage%3D363%26doi%3D10.3390%2Ftoxins11060363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Modahl, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackessy, S. P.</span></span> <span> </span><span class="NLM_article-title">An analysis of venom ontogeny and prey-specific toxicity in the Monocled Cobra (Naja kaouthia)</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2016.04.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2016.04.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=27163885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFegt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2016&pages=8-20&author=C.+M.+Modahlauthor=A.+K.+Mukherjeeauthor=S.+P.+Mackessy&title=An+analysis+of+venom+ontogeny+and+prey-specific+toxicity+in+the+Monocled+Cobra+%28Naja+kaouthia%29&doi=10.1016%2Fj.toxicon.2016.04.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of venom ontogeny and prey-specific toxicity in the Monocled Cobra (Naja kaouthia)</span></div><div class="casAuthors">Modahl, Cassandra M.; Mukherjee, Ashis K.; Mackessy, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8-20</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Venoms of snakes of the family Elapidae (cobras, kraits, mambas, and relatives) are predominantly composed of numerous phospholipases A2 (PLA2s) and three-finger toxins (3FTxs), some of which are lethal while others are not significantly toxic.  Currently, the only identified prey-specific toxins are several nonconventional 3FTxs, and given the large diversity of 3FTxs within Monocled Cobra (Naja kaouthia) venom, it was hypothesized that several 3FTxs, previously found to be non-toxic or weakly toxic 3FTxs in murine models, could potentially be toxic towards non-murine prey.  Addnl., it was hypothesized that ontogenetic dietary shifts will be correlated with observable changes in specific 3FTx isoform abundance.  Adult and juvenile N. kaouthia venom compn. was investigated using ion-exchange FPLC, 1D and 2D SDS-PAGE, mass spectrometry, and various enzymic and LD50 assays.  Alpha-cobratoxin (α-elapitoxin) was the only significantly toxic (LD50 < 1 μg/g) 3FTx found in N. kaouthia venom and was equally toxic toward both lizard and mouse models.  The abundance and diversity of 3FTxs and most enzyme activities did not vary between adult and juvenile cobra venoms; however, total venom PLA2 activity and specific PLA2 isoforms did vary, with juveniles lacking several of the least acidic PLA2s, and these differences could have both biol. (related to predation) and clin. (antivenom efficacy) implications.  Nevertheless, the ubiquitous presence of α-cobratoxin in both adult and juvenile cobra venoms, with high toxicity toward both reptiles and mammals, represents a venom compositional strategy wherein a single potent toxin effectively immobilizes a variety of prey types encountered across life history stages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdhLcg3rSqyrVg90H21EOLACvtfcHk0lg_hUVTFBJTsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFegt7o%253D&md5=06e89a0f850d772f1352b30a71bf71ea</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2016.04.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2016.04.049%26sid%3Dliteratum%253Aachs%26aulast%3DModahl%26aufirst%3DC.%2BM.%26aulast%3DMukherjee%26aufirst%3DA.%2BK.%26aulast%3DMackessy%26aufirst%3DS.%2BP.%26atitle%3DAn%2520analysis%2520of%2520venom%2520ontogeny%2520and%2520prey-specific%2520toxicity%2520in%2520the%2520Monocled%2520Cobra%2520%2528Naja%2520kaouthia%2529%26jtitle%3DToxicon%26date%3D2016%26volume%3D119%26spage%3D8%26epage%3D20%26doi%3D10.1016%2Fj.toxicon.2016.04.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suryamohan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnankutty, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillory, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jevit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriakose, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, O. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perumal, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koludarov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velayutham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muraleedharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faherty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Rosa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagolla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vucic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannoush, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durinck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrusan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stawiski, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raudsepp, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kini, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zachariah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seshagiri, S.</span></span> <span> </span><span class="NLM_article-title">The Indian cobra reference genome and transcriptome enables comprehensive identification of venom toxins</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1038/s41588-019-0559-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fs41588-019-0559-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=31907489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjvF2ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2020&pages=106-117&issue=1&author=K.+Suryamohanauthor=S.+P.+Krishnankuttyauthor=J.+Guilloryauthor=M.+Jevitauthor=M.+S.+Schr%C3%B6derauthor=M.+Wuauthor=B.+Kuriakoseauthor=O.+K.+Mathewauthor=R.+C.+Perumalauthor=I.+Koludarovauthor=L.+D.+Goldsteinauthor=K.+Sengerauthor=M.+D.+Dixonauthor=D.+Velayuthamauthor=D.+Vargasauthor=S.+Chaudhuriauthor=M.+Muraleedharanauthor=R.+Goelauthor=Y.-J.+J.+Chenauthor=A.+Ratanauthor=P.+Liuauthor=B.+Fahertyauthor=G.+de+la+Rosaauthor=H.+Shibataauthor=M.+Bacaauthor=M.+Sagollaauthor=J.+Ziaiauthor=G.+A.+Wrightauthor=D.+Vucicauthor=S.+Mohanauthor=A.+Antonyauthor=J.+Stinsonauthor=D.+S.+Kirkpatrickauthor=R.+N.+Hannoushauthor=S.+Durinckauthor=Z.+Modrusanauthor=E.+W.+Stawiskiauthor=K.+Wileyauthor=T.+Raudseppauthor=R.+M.+Kiniauthor=A.+Zachariahauthor=S.+Seshagiri&title=The+Indian+cobra+reference+genome+and+transcriptome+enables+comprehensive+identification+of+venom+toxins&doi=10.1038%2Fs41588-019-0559-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Indian cobra reference genome and transcriptome enables comprehensive identification of venom toxins</span></div><div class="casAuthors">Suryamohan, Kushal; Krishnankutty, Sajesh P.; Guillory, Joseph; Jevit, Matthew; Schroder, Markus S.; Wu, Meng; Kuriakose, Boney; Mathew, Oommen K.; Perumal, Rajadurai C.; Koludarov, Ivan; Goldstein, Leonard D.; Senger, Kate; Dixon, Mandumpala Davis; Velayutham, Dinesh; Vargas, Derek; Chaudhuri, Subhra; Muraleedharan, Megha; Goel, Ridhi; Chen, Ying-Jiun J.; Ratan, Aakrosh; Liu, Peter; Faherty, Brendan; de la Rosa, Guillermo; Shibata, Hiroki; Baca, Miriam; Sagolla, Meredith; Ziai, James; Wright, Gus A.; Vucic, Domagoj; Mohan, Sangeetha; Antony, Aju; Stinson, Jeremy; Kirkpatrick, Donald S.; Hannoush, Rami N.; Durinck, Steffen; Modrusan, Zora; Stawiski, Eric W.; Wiley, Kristen; Raudsepp, Terje; Kini, R. Manjunatha; Zachariah, Arun; Seshagiri, Somasekar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106-117</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Snakebite envenoming is a serious and neglected tropical disease that kills ∼100,000 people annually.  High-quality, genome-enabled comprehensive characterization of toxin genes will facilitate development of effective humanized recombinant antivenom.  We report a de novo near-chromosomal genome assembly of Naja naja, the Indian cobra, a highly venomous, medically important snake.  Our assembly has a scaffold N50 of 223.35 Mb, with 19 scaffolds contg. 95% of the genome.  Of the 23,248 predicted protein-coding genes, 12,346 venom-gland-expressed genes constitute the 'venom-ome' and this included 139 genes from 33 toxin families.  Among the 139 toxin genes were 19 'venom-ome-specific toxins' (VSTs) that showed venom-gland-specific expression, and these probably encode the minimal core venom effector proteins.  Synthetic venom reconstituted through recombinant VST expression will aid in the rapid development of safe and effective synthetic antivenom.  Addnl., our genome could serve as a ref. for snake genomes, support evolutionary studies and enable venom-driven drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVc9xuao9CibVg90H21EOLACvtfcHk0ljBz-MJABlTVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjvF2ksQ%253D%253D&md5=8f5d4f08e49153d68bdebfb675832b00</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41588-019-0559-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-019-0559-8%26sid%3Dliteratum%253Aachs%26aulast%3DSuryamohan%26aufirst%3DK.%26aulast%3DKrishnankutty%26aufirst%3DS.%2BP.%26aulast%3DGuillory%26aufirst%3DJ.%26aulast%3DJevit%26aufirst%3DM.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%2BS.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DKuriakose%26aufirst%3DB.%26aulast%3DMathew%26aufirst%3DO.%2BK.%26aulast%3DPerumal%26aufirst%3DR.%2BC.%26aulast%3DKoludarov%26aufirst%3DI.%26aulast%3DGoldstein%26aufirst%3DL.%2BD.%26aulast%3DSenger%26aufirst%3DK.%26aulast%3DDixon%26aufirst%3DM.%2BD.%26aulast%3DVelayutham%26aufirst%3DD.%26aulast%3DVargas%26aufirst%3DD.%26aulast%3DChaudhuri%26aufirst%3DS.%26aulast%3DMuraleedharan%26aufirst%3DM.%26aulast%3DGoel%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.-J.%2BJ.%26aulast%3DRatan%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DFaherty%26aufirst%3DB.%26aulast%3Dde%2Bla%2BRosa%26aufirst%3DG.%26aulast%3DShibata%26aufirst%3DH.%26aulast%3DBaca%26aufirst%3DM.%26aulast%3DSagolla%26aufirst%3DM.%26aulast%3DZiai%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DG.%2BA.%26aulast%3DVucic%26aufirst%3DD.%26aulast%3DMohan%26aufirst%3DS.%26aulast%3DAntony%26aufirst%3DA.%26aulast%3DStinson%26aufirst%3DJ.%26aulast%3DKirkpatrick%26aufirst%3DD.%2BS.%26aulast%3DHannoush%26aufirst%3DR.%2BN.%26aulast%3DDurinck%26aufirst%3DS.%26aulast%3DModrusan%26aufirst%3DZ.%26aulast%3DStawiski%26aufirst%3DE.%2BW.%26aulast%3DWiley%26aufirst%3DK.%26aulast%3DRaudsepp%26aufirst%3DT.%26aulast%3DKini%26aufirst%3DR.%2BM.%26aulast%3DZachariah%26aufirst%3DA.%26aulast%3DSeshagiri%26aufirst%3DS.%26atitle%3DThe%2520Indian%2520cobra%2520reference%2520genome%2520and%2520transcriptome%2520enables%2520comprehensive%2520identification%2520of%2520venom%2520toxins%26jtitle%3DNat.%2520Genet.%26date%3D2020%26volume%3D52%26issue%3D1%26spage%3D106%26epage%3D117%26doi%3D10.1038%2Fs41588-019-0559-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, N. H.</span></span> <span> </span><span class="NLM_article-title">Venom and purified toxins of the spectacled cobra (Naja naja) from Pakistan: insights into toxicity and antivenom neutralization</span>. <i>Am. J. Trop. Med. Hyg.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>94</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1392</span>– <span class="NLM_lpage">1399</span>, <span class="refDoi"> DOI: 10.4269/ajtmh.15-0871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.4269%2Fajtmh.15-0871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=27022154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFWltLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2016&pages=1392-1399&issue=6&author=K.+Y.+Wongauthor=C.+H.+Tanauthor=N.+H.+Tan&title=Venom+and+purified+toxins+of+the+spectacled+cobra+%28Naja+naja%29+from+Pakistan%3A+insights+into+toxicity+and+antivenom+neutralization&doi=10.4269%2Fajtmh.15-0871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Venom and purified toxins of the spectacled cobra (Naja naja) from Pakistan: insights into toxicity and antivenom neutralization</span></div><div class="casAuthors">Wong, Kin Ying; Tan, Choo Hock; Tan, Nget Hong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Tropical Medicine and Hygiene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1392-1399</span>CODEN:
                <span class="NLM_cas:coden">AJTHAB</span>;
        ISSN:<span class="NLM_cas:issn">1476-1645</span>.
    
            (<span class="NLM_cas:orgname">American Society of Tropical Medicine and Hygiene</span>)
        </div><div class="casAbstract">Geog. variations of snake venoms can result in suboptimal effectiveness of Indian antivenoms that are currently used in most South Asian countries.  This study investigated the toxicity and neutralization profile of the venom and toxins from Pakistani spectacled cobra, Naja naja, using VINS polyvalent antivenom (VPAV, India), Naja kaouthia monovalent antivenom (NKMAV, Thailand), and neuro bivalent antivenom (NBAV, Taiwan).  Cationexchange and reverse-phase high-performance liq. chromatog. fractionations followed by toxin identification through liq. chromatog.-mass spectrometry (MS)/MS indicated that the venom comprised mainly of postsynaptic neurotoxins (NTXs) (long neurotoxins [LNTXs], 28.3%; short neurotoxins [SNTXs], 8%), cytotoxins (CTXs) (31.2%), and acidic phospholipases A2 (12.3%).  NKMAV is the most effective in neutralizing the lethal effect of the venom (potency = 1.1 mg venom/mL) and its LNTX (potency = 0.5 mg toxin/mL), consistent with the high content of LNTX in N. kaouthia venom.  VPAV was effective in neutralizing the CTX (potency = 0.4 mg toxin/mL), in agreement with the higher CTX abundance in Indian cobra venom.  All the three antivenoms were weak in neutralizing the SNTX (potency = 0.03-0.04 mg toxin/mL), including NBAV that was raised from the SNTX-rich Taiwanese cobra venom.  In a challenge-rescue expt., envenomed mice were prevented from death by a maximal dose of VPAV (i.v. 200 μL) but the recovery from paralysis was slow, indicating the need for higher or repeated doses of VPAV.  Our results suggest that optimal neutralization for Pakistani N. naja venom may be achieved by improving the formulation of antivenom prodn. to enhance antivenom immunoreactivity against long and SNTXs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuU_ljPBcbm7Vg90H21EOLACvtfcHk0ljBz-MJABlTVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFWltLvJ&md5=375955bb588047f56674298ba57dd792</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4269%2Fajtmh.15-0871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4269%252Fajtmh.15-0871%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DK.%2BY.%26aulast%3DTan%26aufirst%3DC.%2BH.%26aulast%3DTan%26aufirst%3DN.%2BH.%26atitle%3DVenom%2520and%2520purified%2520toxins%2520of%2520the%2520spectacled%2520cobra%2520%2528Naja%2520naja%2529%2520from%2520Pakistan%253A%2520insights%2520into%2520toxicity%2520and%2520antivenom%2520neutralization%26jtitle%3DAm.%2520J.%2520Trop.%2520Med.%2520Hyg.%26date%3D2016%26volume%3D94%26issue%3D6%26spage%3D1392%26epage%3D1399%26doi%3D10.4269%2Fajtmh.15-0871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barber, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isbister, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, W. C.</span></span> <span> </span><span class="NLM_article-title">Alpha neurotoxins</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2013.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2013.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=23416229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFOqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2013&pages=47-58&author=C.+M.+Barberauthor=G.+K.+Isbisterauthor=W.+C.+Hodgson&title=Alpha+neurotoxins&doi=10.1016%2Fj.toxicon.2013.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha neurotoxins</span></div><div class="casAuthors">Barber, Carmel M.; Isbister, Geoffrey K.; Hodgson, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  α-Neurotoxins have been isolated from hydrophid, elapid and, more recently, colubrid snake venoms.  Also referred to as postsynaptic neurotoxins or curare mimetic' neurotoxins, they play an important role in the capture and/or killing of prey by binding to the nicotinic acetylcholine receptor on the skeletal muscle disrupting neurotransmission.  They are also thought to cause respiratory paralysis in envenomed humans.  This review will discuss the historical background into the discovery, isolation, structure and mechanism of action of the α-neurotoxins, including targets and cellular outcomes, and then will examine the potential uses of α-neurotoxins as pharmacol. tools and/or as drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4R4vR45UvEbVg90H21EOLACvtfcHk0ljBz-MJABlTVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFOqsrk%253D&md5=094db1cf51390a01d15c4f2883752d26</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2013.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2013.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DBarber%26aufirst%3DC.%2BM.%26aulast%3DIsbister%26aufirst%3DG.%2BK.%26aulast%3DHodgson%26aufirst%3DW.%2BC.%26atitle%3DAlpha%2520neurotoxins%26jtitle%3DToxicon%26date%3D2013%26volume%3D66%26spage%3D47%26epage%3D58%26doi%3D10.1016%2Fj.toxicon.2013.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomonte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvete, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">León, G.</span></span> <span> </span><span class="NLM_article-title">Assessing the preclinical efficacy of antivenoms: from the lethality neutralization assay to antivenomics</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">168</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2012.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2012.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=23201503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ntLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=168-179&author=J.+M.+Guti%C3%A9rrezauthor=G.+Solanoauthor=D.+Plaauthor=M.+Herreraauthor=%C3%81.+Seguraauthor=M.+Villaltaauthor=M.+Vargasauthor=L.+Sanzauthor=B.+Lomonteauthor=J.+J.+Calveteauthor=G.+Le%C3%B3n&title=Assessing+the+preclinical+efficacy+of+antivenoms%3A+from+the+lethality+neutralization+assay+to+antivenomics&doi=10.1016%2Fj.toxicon.2012.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the preclinical efficacy of antivenoms: From the lethality neutralization assay to antivenomics</span></div><div class="casAuthors">Gutierrez, Jose Maria; Solano, Gabriela; Pla, Davinia; Herrera, Maria; Segura, Alvaro; Villalta, Mauren; Vargas, Mariangela; Sanz, Libia; Lomonte, Bruno; Calvete, Juan J.; Leon, Guillermo</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">168-179</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The assessment of the capacity of antivenoms to neutralize the lethal activity of snake venoms is the gold std. in the preclin. anal. of antivenom efficacy, and is routinely performed by manufacturers and quality control labs.  However, the complexity of snake venom compn. and toxicol. profile demands that, for many venoms, such as those of viperid snakes and some elapids, the neutralization of lethality be complemented with the anal. of the neutralization of other relevant toxic activities, such as hemorrhagic, myotoxic, necrotizing, procoagulant and defibrinogenating effects.  This expanded protocol for preclin. testing of antivenoms should be used when a new antivenom is developed or when an existing antivenom is introduced in a new geog. setting for the neutralization of either homologous or heterologous venoms.  In recent years, the assessment of the immunol. reactivity of antivenoms has been enriched by the use of proteomic tools, with a methodol. named 'antivenomics'.  This allows the identification of venom components to which antivenoms have, or lack, antibodies, and thus complements the data gathered in neutralization tests, paving the way for a knowledge-based improvement of antivenom design and efficacy.  International projects involving participants of manufg., quality control and academic research groups should be promoted in order to gain a deeper understanding on the preclin. neutralizing spectrum of antivenoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh35k3CR3hT7Vg90H21EOLACvtfcHk0liZ0ZFtZ2sAlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ntLvF&md5=d17b834cf8056a5b904676a9206fde5f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2012.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2012.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DSolano%26aufirst%3DG.%26aulast%3DPla%26aufirst%3DD.%26aulast%3DHerrera%26aufirst%3DM.%26aulast%3DSegura%26aufirst%3D%25C3%2581.%26aulast%3DVillalta%26aufirst%3DM.%26aulast%3DVargas%26aufirst%3DM.%26aulast%3DSanz%26aufirst%3DL.%26aulast%3DLomonte%26aufirst%3DB.%26aulast%3DCalvete%26aufirst%3DJ.%2BJ.%26aulast%3DLe%25C3%25B3n%26aufirst%3DG.%26atitle%3DAssessing%2520the%2520preclinical%2520efficacy%2520of%2520antivenoms%253A%2520from%2520the%2520lethality%2520neutralization%2520assay%2520to%2520antivenomics%26jtitle%3DToxicon%26date%3D2013%26volume%3D69%26spage%3D168%26epage%3D179%26doi%3D10.1016%2Fj.toxicon.2012.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arbabi-Ghahroudi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, J. C.</span></span> <span> </span><span class="NLM_article-title">In vivo neutralization of α-cobratoxin with high-affinity llama single-domain antibodies (VHHs) and a VHH-Fc antibody</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>), <span class="NLM_elocation-id">e69495</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0069495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1371%2Fjournal.pone.0069495" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=23894495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ejtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&issue=7&author=G.+Richardauthor=A.+J.+Meyersauthor=M.+D.+McLeanauthor=M.+Arbabi-Ghahroudiauthor=R.+MacKenzieauthor=J.+C.+Hall&title=In+vivo+neutralization+of+%CE%B1-cobratoxin+with+high-affinity+llama+single-domain+antibodies+%28VHHs%29+and+a+VHH-Fc+antibody&doi=10.1371%2Fjournal.pone.0069495"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo neutralization of α-cobratoxin with high-affinity llama single-domain antibodies (VHHs) and a VHH-Fc antibody</span></div><div class="casAuthors">Richard, Gabrielle; Meyers, Ashley J.; McLean, Michael D.; Arbabi-Ghahroudi, Mehdi; MacKenzie, Roger; Hall, J. Christopher</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e69495</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Small recombinant antibody fragments (e.g. scFvs and VHHs), which are highly tissue permeable, are being investigated for antivenom prodn. as conventional antivenoms consisting of IgG or F(ab')2 antibody fragments do not effectively neutralize venom toxins located in deep tissues.  However, antivenoms composed entirely of small antibody fragments may have poor therapeutic efficacy due to their short serum half-lives.  To increase serum persistence and maintain tissue penetration, we prepd. low and high mol. mass antivenom antibodies.  Four llama VHHs were isolated from an immune VHH-displayed phage library and were shown to have high affinity, in the low nM range, for α-cobratoxin (α-Cbtx), the most lethal component of Naja kaouthia venom.  Subsequently, our highest affinity VHH (C2) was fused to a human Fc fragment to create a VHH2-Fc antibody that would offer prolonged serum persistence.  After in planta (Nicotiana benthamiana) expression and purifn., we show that our VHH2-Fc antibody retained high affinity binding to α-Cbtx.  Mouse α-Cbtx challenge studies showed that our highest affinity VHHs (C2 and C20) and the VHH2-Fc antibody effectively neutralized lethality induced by α-Cbtx at an antibody:toxin molar ratio as low as ca. 0.75x:1.  Further research towards the development of an antivenom therapeutic involving these anti-α-Cbtx VHHs and VHH2-Fc antibody mols. should involve testing them as a combination, to det. whether they maintain tissue penetration capability and low immunogenicity, and whether they exhibit improved serum persistence and therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4hiykuDKo3LVg90H21EOLACvtfcHk0liZ0ZFtZ2sAlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1ejtbvP&md5=d7d8d4bd6987d18cda0fa7ca0be6b9d6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0069495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0069495%26sid%3Dliteratum%253Aachs%26aulast%3DRichard%26aufirst%3DG.%26aulast%3DMeyers%26aufirst%3DA.%2BJ.%26aulast%3DMcLean%26aufirst%3DM.%2BD.%26aulast%3DArbabi-Ghahroudi%26aufirst%3DM.%26aulast%3DMacKenzie%26aufirst%3DR.%26aulast%3DHall%26aufirst%3DJ.%2BC.%26atitle%3DIn%2520vivo%2520neutralization%2520of%2520%25CE%25B1-cobratoxin%2520with%2520high-affinity%2520llama%2520single-domain%2520antibodies%2520%2528VHHs%2529%2520and%2520a%2520VHH-Fc%2520antibody%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26issue%3D7%26doi%3D10.1371%2Fjournal.pone.0069495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulkeaw, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakolvaree, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srimanote, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tongtawe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneewatch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sookrung, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tungtrongchitr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tapchaisri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurazono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaicumpa, W.</span></span> <span> </span><span class="NLM_article-title">Human monoclonal ScFv neutralize lethal Thai cobra, Naja kaouthia, neurotoxin</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2008.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.jprot.2008.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=19162253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Oqurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2009&pages=270-282&issue=2&author=K.+Kulkeawauthor=Y.+Sakolvareeauthor=P.+Srimanoteauthor=P.+Tongtaweauthor=S.+Maneewatchauthor=N.+Sookrungauthor=A.+Tungtrongchitrauthor=P.+Tapchaisriauthor=H.+Kurazonoauthor=W.+Chaicumpa&title=Human+monoclonal+ScFv+neutralize+lethal+Thai+cobra%2C+Naja+kaouthia%2C+neurotoxin&doi=10.1016%2Fj.jprot.2008.12.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Human monoclonal ScFv neutralize lethal Thai cobra, Naja kaouthia, neurotoxin</span></div><div class="casAuthors">Kulkeaw, Kasem; Sakolvaree, Yuwaporn; Srimanote, Potjanee; Tongtawe, Pongsri; Maneewatch, Santi; Sookrung, Nitat; Tungtrongchitr, Anchalee; Tapchaisri, Pramuan; Kurazono, Hisao; Chaicumpa, Wanpen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">270-282</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Animal derived anti-Naja. kaouthia (Thai cobra) venom is used for specific treatment of the snake bitten victims.  Many recipients develop allergic reaction or anti-isotype response which causes serum sickness.  A better therapeutic antibody is needed.  In this study, long α-neurotoxin was purified from the N. kaouthia holovenom and verified by 2D-LC/MS-MS.  The toxin was used as antigen in a phage biopanning to select phage clones displaying human single chain variable antibody fragments (HuScFv) from a phage display antibody library constructed from Ig genes of nonimmunized Thai blood donors.  HuScFv that specifically bound to the neurotoxin were produced from huscfv-phagemid transformed E. coli clones and affinity purified.  The HuScFv could neutralize toxicity of the N. kaouthia neurotoxin and rescued the envenomized mice from the neurotoxin mediated lethality.  Peptide mimotope of the neutralizing HuScFv matched with an amino acid sequence (epitope) located in the loop-3 of the N. kaouthia long α-neurotoxin which functions in acetylcholine receptor binding.  The mimotope is also similar to peptide sequences found on other snake venom neurotoxins implying a possibility of the HuScFv to exert pan-neutralizing activity against multiple snake neurotoxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHcs6Zu1Jc3LVg90H21EOLACvtfcHk0liZ0ZFtZ2sAlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Oqurs%253D&md5=ce5e963b123f89101e3df77169836cbd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2008.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2008.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DKulkeaw%26aufirst%3DK.%26aulast%3DSakolvaree%26aufirst%3DY.%26aulast%3DSrimanote%26aufirst%3DP.%26aulast%3DTongtawe%26aufirst%3DP.%26aulast%3DManeewatch%26aufirst%3DS.%26aulast%3DSookrung%26aufirst%3DN.%26aulast%3DTungtrongchitr%26aufirst%3DA.%26aulast%3DTapchaisri%26aufirst%3DP.%26aulast%3DKurazono%26aufirst%3DH.%26aulast%3DChaicumpa%26aufirst%3DW.%26atitle%3DHuman%2520monoclonal%2520ScFv%2520neutralize%2520lethal%2520Thai%2520cobra%252C%2520Naja%2520kaouthia%252C%2520neurotoxin%26jtitle%3DJ.%2520Proteomics%26date%3D2009%26volume%3D72%26issue%3D2%26spage%3D270%26epage%3D282%26doi%3D10.1016%2Fj.jprot.2008.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laustsen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatt-Vellatt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arias, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pus, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oscoz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavny, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leah, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindholm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomonte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, J.</span></span> <span> </span><span class="NLM_article-title">In vivo neutralization of dendrotoxin-mediated neurotoxicity of black mamba venom by oligoclonal human IgG antibodies</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3928</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-06086-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fs41467-018-06086-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=30279409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BB3czltFymtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3928&author=A.+H.+Laustsenauthor=A.+Karatt-Vellattauthor=E.+W.+Mastersauthor=A.+S.+Ariasauthor=U.+Pusauthor=C.+Knudsenauthor=S.+Oscozauthor=P.+Slavnyauthor=D.+T.+Griffithsauthor=A.+M.+Lutherauthor=R.+A.+Leahauthor=M.+Lindholmauthor=B.+Lomonteauthor=J.+M.+Guti%C3%A9rrezauthor=J.+McCafferty&title=In+vivo+neutralization+of+dendrotoxin-mediated+neurotoxicity+of+black+mamba+venom+by+oligoclonal+human+IgG+antibodies&doi=10.1038%2Fs41467-018-06086-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo neutralization of dendrotoxin-mediated neurotoxicity of black mamba venom by oligoclonal human IgG antibodies</span></div><div class="casAuthors">Laustsen Andreas H; Pus Urska; Knudsen Cecilie; Karatt-Vellatt Aneesh; Masters Edward W; Slavny Peter; Griffiths Daniel T; Luther Alice M; Leah Rachael A; Lindholm Majken; McCafferty John; Arias Ana Silvia; Oscoz Saioa; Lomonte Bruno; Gutierrez Jose Maria</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3928</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The black mamba (Dendroaspis polylepis) is one of the most feared snake species of the African savanna.  It has a potent, fast-acting neurotoxic venom comprised of dendrotoxins and α-neurotoxins associated with high fatality in untreated victims.  Current antivenoms are both scarce on the African continent and present a number of drawbacks as they are derived from the plasma of hyper-immunized large mammals.  Here, we describe the development of an experimental recombinant antivenom by a combined toxicovenomics and phage display approach.  The recombinant antivenom is based on a cocktail of fully human immunoglobulin G (IgG) monoclonal antibodies capable of neutralizing dendrotoxin-mediated neurotoxicity of black mamba whole venom in a rodent model.  Our results show the potential use of fully human monoclonal IgGs against animal toxins and the first use of oligoclonal human IgG mixtures against experimental snakebite envenoming.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnUvQzyIEmj4qdXsLcGTEUfW6udTcc2eZgmaSe_9ansrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czltFymtg%253D%253D&md5=c1a6ba8b1e8531eab98a6d614af80e26</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-06086-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-06086-4%26sid%3Dliteratum%253Aachs%26aulast%3DLaustsen%26aufirst%3DA.%2BH.%26aulast%3DKaratt-Vellatt%26aufirst%3DA.%26aulast%3DMasters%26aufirst%3DE.%2BW.%26aulast%3DArias%26aufirst%3DA.%2BS.%26aulast%3DPus%26aufirst%3DU.%26aulast%3DKnudsen%26aufirst%3DC.%26aulast%3DOscoz%26aufirst%3DS.%26aulast%3DSlavny%26aufirst%3DP.%26aulast%3DGriffiths%26aufirst%3DD.%2BT.%26aulast%3DLuther%26aufirst%3DA.%2BM.%26aulast%3DLeah%26aufirst%3DR.%2BA.%26aulast%3DLindholm%26aufirst%3DM.%26aulast%3DLomonte%26aufirst%3DB.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DJ.%2BM.%26aulast%3DMcCafferty%26aufirst%3DJ.%26atitle%3DIn%2520vivo%2520neutralization%2520of%2520dendrotoxin-mediated%2520neurotoxicity%2520of%2520black%2520mamba%2520venom%2520by%2520oligoclonal%2520human%2520IgG%2520antibodies%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D3928%26doi%3D10.1038%2Fs41467-018-06086-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrick, K.</span></span> <span> </span><span class="NLM_article-title">Inference of macromolecular assemblies from crystalline state</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>372</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">797</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.05.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.jmb.2007.05.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=17681537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2007&pages=774-797&issue=3&author=E.+Krissinelauthor=K.+Henrick&title=Inference+of+macromolecular+assemblies+from+crystalline+state&doi=10.1016%2Fj.jmb.2007.05.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Inference of Macromolecular Assemblies from Crystalline State</span></div><div class="casAuthors">Krissinel, Evgeny; Henrick, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-797</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors discuss basic phys.-chem. principles underlying the formation of stable macromol. complexes, which in many cases are likely to be the biol. units performing a certain physiol. function.  The authors also consider available theor. approaches to the calcn. of macromol. affinity and entropy of complexation.  The latter is shown to play an important role and make a major effect on complex size and symmetry.  The authors develop a new method, based on chem. thermodn., for automatic detection of macromol. assemblies in the Protein Data Bank (PDB) entries that are the results of x-ray diffraction expts.  As found, biol. units may be recovered at 80-90% success rate, which makes x-ray crystallog. an important source of exptl. data on macromol. complexes and protein-protein interactions.  The method is implemented as a public WWW service (http://www.ebi.ac.uk/msd-srv/prot_int/pistart.html).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3cilBvVoBS7Vg90H21EOLACvtfcHk0liTUT1nWfxx9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D&md5=a5c764cfc7dc129f53ddc31ef9d475fa</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.05.022%26sid%3Dliteratum%253Aachs%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DInference%2520of%2520macromolecular%2520assemblies%2520from%2520crystalline%2520state%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D372%26issue%3D3%26spage%3D774%26epage%3D797%26doi%3D10.1016%2Fj.jmb.2007.05.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nirthanan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwee, M. C. E.</span></span> <span> </span><span class="NLM_article-title">Three-finger alpha-neurotoxins and the nicotinic acetylcholine receptor, forty years on</span>. <i>J. Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>94</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1254/jphs.94.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1254%2Fjphs.94.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=14745112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXosVamsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2004&pages=1-17&issue=1&author=S.+Nirthananauthor=M.+C.+E.+Gwee&title=Three-finger+alpha-neurotoxins+and+the+nicotinic+acetylcholine+receptor%2C+forty+years+on&doi=10.1254%2Fjphs.94.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Three-finger α-neurotoxins and the nicotinic acetylcholine receptor, forty years on</span></div><div class="casAuthors">Nirthanan, Selvanayagam; Gwee, Matthew C. E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological Sciences (Tokyo, Japan)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-17</span>CODEN:
                <span class="NLM_cas:coden">JPSTGJ</span>;
        ISSN:<span class="NLM_cas:issn">1347-8613</span>.
    
            (<span class="NLM_cas:orgname">Japanese Pharmacological Society</span>)
        </div><div class="casAbstract">A review.  The discovery, about forty years ago, of α-bungarotoxin, a three-finger α-neurotoxin from Bungarus multicinctus venom, enabled the isolation of the nicotinic acetylcholine receptor (nAChR), making it one of the most thoroughly characterized receptors today.  Since then, the sites of interaction between α-neurotoxins and nAChRs have largely been delineated, revealing the remarkable plasticity of the three-finger toxin fold that has optimally evolved to utilize different combinations of functional groups to generate a panoply of target specificities to discern subtle differences between nAChR subtypes.  New facets in toxinol. have now broadened the scope for the use of α-neurotoxins in scientific discovery.  For instance, the development of short, combinatorial library-derived, synthetic peptides that bind with subnanomolar affinity to α-bungarotoxin and prevent its interaction with muscle nAChRs has led to the in vivo neutralization of exptl. α-bungarotoxin envenomation, while the successful introduction of pharmatopes bearing "α-bungarotoxin-sensitive sites" into toxin-insensitive nAChRs has permitted the use of various α-neurotoxin tags to localize and characterize new receptor subtypes.  More ambitious strategies can now be envisaged for engineering rationally designed novel activities on three-finger toxin scaffolds to generate lead peptides of therapeutic value that target the nicotinic pharmacopoeia.  This review details the progress made towards achieving this goal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv04nhW9gh2bVg90H21EOLACvtfcHk0liTUT1nWfxx9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXosVamsg%253D%253D&md5=de858ec5f133be902b11f34675f48c34</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1254%2Fjphs.94.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjphs.94.1%26sid%3Dliteratum%253Aachs%26aulast%3DNirthanan%26aufirst%3DS.%26aulast%3DGwee%26aufirst%3DM.%2BC.%2BE.%26atitle%3DThree-finger%2520alpha-neurotoxins%2520and%2520the%2520nicotinic%2520acetylcholine%2520receptor%252C%2520forty%2520years%2520on%26jtitle%3DJ.%2520Pharmacol.%2520Sci.%26date%3D2004%26volume%3D94%26issue%3D1%26spage%3D1%26epage%3D17%26doi%3D10.1254%2Fjphs.94.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worrell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noviello, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stowell, M. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibbs, R. E.</span></span> <span> </span><span class="NLM_article-title">Structure of the native muscle-type nicotinic receptor and inhibition by snake venom toxins</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2020.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.neuron.2020.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=32275860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmvFWhur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2020&pages=952-962&issue=6&author=M.+M.+Rahmanauthor=J.+Tengauthor=B.+T.+Worrellauthor=C.+M.+Novielloauthor=M.+Leeauthor=A.+Karlinauthor=M.+H.+B.+Stowellauthor=R.+E.+Hibbs&title=Structure+of+the+native+muscle-type+nicotinic+receptor+and+inhibition+by+snake+venom+toxins&doi=10.1016%2Fj.neuron.2020.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Native Muscle-type Nicotinic Receptor and Inhibition by Snake Venom Toxins.</span></div><div class="casAuthors">Rahman, Md. Mahfuzur; Teng, Jinfeng; Worrell, Brady T.; Noviello, Colleen M.; Lee, Myeongseon; Karlin, Arthur; Stowell, Michael H. B.; Hibbs, Ryan E.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">952-962.e5</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The nicotinic acetylcholine receptor, a pentameric ligand-gated ion channel, converts the free energy of binding of the neurotransmitter acetylcholine into opening of its central pore.  Here we present the first high-resoln. structure of the receptor type found in muscle-endplate membrane and in the muscle-derived elec. tissues of fish.  The native receptor was purified from Torpedo elec. tissue and functionally reconstituted in lipids optimal for cryo-electron microscopy.  The receptor was stabilized in a closed state by the binding of α-bungarotoxin.  The structure reveals the binding of a toxin mol. at each of two subunit interfaces in a manner that would block the binding of acetylcholine.  It also reveals a closed gate in the ion-conducting pore, formed by hydrophobic amino acid side chains, located ∼60 Å from the toxin binding sites.  The structure provides a framework for understanding gating in ligand-gated channels and how mutations in the acetylcholine receptor cause congenital myasthenic syndromes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW_NVMcdEgUbVg90H21EOLACvtfcHk0lg43Nt-U8ZnQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmvFWhur0%253D&md5=52d7f7343f1bfca9e2f6a044f8d3015b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2020.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DRahman%26aufirst%3DM.%2BM.%26aulast%3DTeng%26aufirst%3DJ.%26aulast%3DWorrell%26aufirst%3DB.%2BT.%26aulast%3DNoviello%26aufirst%3DC.%2BM.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DKarlin%26aufirst%3DA.%26aulast%3DStowell%26aufirst%3DM.%2BH.%2BB.%26aulast%3DHibbs%26aufirst%3DR.%2BE.%26atitle%3DStructure%2520of%2520the%2520native%2520muscle-type%2520nicotinic%2520receptor%2520and%2520inhibition%2520by%2520snake%2520venom%2520toxins%26jtitle%3DNeuron%26date%3D2020%26volume%3D106%26issue%3D6%26spage%3D952%26epage%3D962%26doi%3D10.1016%2Fj.neuron.2020.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talley, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchot, P.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of a Cbtx-AChBP complex reveals essential interactions between snake alpha-neurotoxins and nicotinic receptors</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1512</span>– <span class="NLM_lpage">1522</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2Fsj.emboj.7600620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15791209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFequro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=1512-1522&issue=8&author=Y.+Bourneauthor=T.+T.+Talleyauthor=S.+B.+Hansenauthor=P.+Taylorauthor=P.+Marchot&title=Crystal+structure+of+a+Cbtx-AChBP+complex+reveals+essential+interactions+between+snake+alpha-neurotoxins+and+nicotinic+receptors&doi=10.1038%2Fsj.emboj.7600620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a Cbtx-AChBP complex reveals essential interactions between snake α-neurotoxins and nicotinic receptors</span></div><div class="casAuthors">Bourne, Yves; Talley, Todd T.; Hansen, Scott B.; Taylor, Palmer; Marchot, Pascale</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1512-1522</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The crystal structure of the snake long α-neurotoxin, α-cobratoxin, bound to the pentameric acetylcholine-binding protein (AChBP) from Lymnaea stagnalis, was solved from good quality d. maps despite a 4.2 Å overall resoln.  The structure unambiguously reveals the positions and orientations of all five three-fingered toxin mols. inserted at the AChBP subunit interfaces and the conformational changes assocd. with toxin binding.  AChBP loops C and F that border the ligandbinding pocket move markedly from their original positions to wrap around the tips of the toxin first and second fingers and part of its C-terminus, while rearrangements also occur in the toxin fingers.  At the interface of the complex, major interactions involve arom. and aliph. side chains within the AChBP binding pocket and, at the buried tip of the toxin second finger, conserved Phe and Arg residues that partially mimic a bound agonist mol.  Hence this structure, in revealing a distinctive and unpredicted conformation of the toxin-bound AChBP mol., provides a lead template resembling a resting state conformation of the nicotinic receptor and for understanding selectivity of curaremimetic α-neurotoxins for the various receptor species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ieEvb-on7rVg90H21EOLACvtfcHk0lg43Nt-U8ZnQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFequro%253D&md5=3d3c40b85aa80586e9d94d30d704e715</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600620%26sid%3Dliteratum%253Aachs%26aulast%3DBourne%26aufirst%3DY.%26aulast%3DTalley%26aufirst%3DT.%2BT.%26aulast%3DHansen%26aufirst%3DS.%2BB.%26aulast%3DTaylor%26aufirst%3DP.%26aulast%3DMarchot%26aufirst%3DP.%26atitle%3DCrystal%2520structure%2520of%2520a%2520Cbtx-AChBP%2520complex%2520reveals%2520essential%2520interactions%2520between%2520snake%2520alpha-neurotoxins%2520and%2520nicotinic%2520receptors%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26issue%3D8%26spage%3D1512%26epage%3D1522%26doi%3D10.1038%2Fsj.emboj.7600620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jurgilas, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves-Ferreira, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domont, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perales, J.</span></span> <span> </span><span class="NLM_article-title">PO41, a snake venom metalloproteinase inhibitor isolated from Philander opossum serum</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2003.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.toxicon.2003.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=14602117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosFOrsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=621-628&issue=6&author=P.+B.+Jurgilasauthor=A.+G.+Neves-Ferreiraauthor=G.+B.+Domontauthor=J.+Perales&title=PO41%2C+a+snake+venom+metalloproteinase+inhibitor+isolated+from+Philander+opossum+serum&doi=10.1016%2Fj.toxicon.2003.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PO41, a snake venom metalloproteinase inhibitor isolated from Philander opossum serum</span></div><div class="casAuthors">Jurgilas, Patricia B.; Neves-Ferreira, Ana G. C.; Domont, Gilberto B.; Perales, Jonas</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">621-628</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">PO41 was isolated from Philander opossum serum by DEAE-Sephacel, Ph Superose and Superdex 200 chromatogs. and showed a mol. mass of 41,330 Da by MALDI-TOF MS.  Mol. masses of 81.5 and 84.5 kDa were obtained by size exclusion chromatog. and dynamic laser light scattering, resp., suggesting that PO41 is dimeric.  Its isoelec. point was estd. to be lower than 3.5.  PO41 presented similar amino terminal sequence to those of DM40 and DM43, two antihaemorrhagins previously isolated from Didelphis marsupialis serum and was recognized by polyclonal antibodies raised against D. marsupialis antibothropic fraction.  To study the inhibitory properties of this protein, the metalloproteinases bothrolysin and jararhagin were isolated from Bothrops jararaca venom by chromatogs. on Superdex 200 and Ph Superose.  Jararhagin was further submitted to a Mono Q column.  The proteolytic and haemorrhagic effects of these haemorrhagins were neutralized by PO41.  Both snake venom metalloproteinases formed stable complexes with PO41.  The stoichiometry of the complex PO41-jararhagin was one inhibitor subunit to one mol. of the enzyme.  These results show that PO41 has physicochem., structural, immunoreactive and biol. properties similar to other metalloproteinase inhibitors belonging to the supergene family of Igs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAId4XtPyRLVg90H21EOLACvtfcHk0lg43Nt-U8ZnQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosFOrsrg%253D&md5=4aa34ae231a50d007df5a46732091c15</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2003.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2003.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DJurgilas%26aufirst%3DP.%2BB.%26aulast%3DNeves-Ferreira%26aufirst%3DA.%2BG.%26aulast%3DDomont%26aufirst%3DG.%2BB.%26aulast%3DPerales%26aufirst%3DJ.%26atitle%3DPO41%252C%2520a%2520snake%2520venom%2520metalloproteinase%2520inhibitor%2520isolated%2520from%2520Philander%2520opossum%2520serum%26jtitle%3DToxicon%26date%3D2003%26volume%3D42%26issue%3D6%26spage%3D621%26epage%3D628%26doi%3D10.1016%2Fj.toxicon.2003.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karain, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, W. K.</span></span> <span> </span><span class="NLM_article-title">C60 fullerenes as a novel treatment for poisoning and envenomation: a proof-of-concept study for snakebite</span>. <i>J. Nanosci. Nanotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">7764</span>– <span class="NLM_lpage">7771</span>, <span class="refDoi"> DOI: 10.1166/jnn.2016.12851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1166%2Fjnn.2016.12851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWks7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=7764-7771&issue=7&author=B.+D.+Karainauthor=M.+K.+H.+Leeauthor=W.+K.+Hayes&title=C60+fullerenes+as+a+novel+treatment+for+poisoning+and+envenomation%3A+a+proof-of-concept+study+for+snakebite&doi=10.1166%2Fjnn.2016.12851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">C60 fullerenes as a novel treatment for poisoning and envenomation: a proof-of-concept study for snakebite</span></div><div class="casAuthors">Karain, Bradley Darryl; Lee, Maximus Kyung Hyun; Hayes, William K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Nanoscience and Nanotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">7764-7771</span>CODEN:
                <span class="NLM_cas:coden">JNNOAR</span>;
        ISSN:<span class="NLM_cas:issn">1533-4880</span>.
    
            (<span class="NLM_cas:orgname">American Scientific Publishers</span>)
        </div><div class="casAbstract">Millions of poisonings and envenomations occur annually worldwide.  When treatments for these conditions exist, they are often highly specific and may be costly or unavailable, esp. in remote regions and developing countries.  In cases of snake envenomation, mortality and severe morbidity occur in hundreds of thousands of cases annually.  These tragic physiol. sequelae result from the complex mixts. of protein toxins comprising snake venom.  The current gold std. for snakebite treatment is to use a mixt. of polyclonal antibodies, referred to as antivenom.  While effective, antivenoms have severe limitations: they are costly to produce and distribute; they must be delivered in an acute care facility; they have a relatively short shelf life; and they often induce side effects that are generally mild but occasionally serious.  Certain nanoparticles, in contrast, possess protein-binding and pharmacokinetic properties that make them a potentially low-cost and readily available adjunct or alternative to antivenom for treatment of systemically distributed toxins.  We therefore conducted a proof-of-concept study to test the ability of C60 fullerene to neutralize rattlesnake venom in a cricket model.  We injected approx. LD50 doses of southern Pacific rattlesnake (Crotalus oreganus helleri) venom into crickets (Acheta domesticus).  In the first of two expts., crickets injected with C60 fullerene prior to envenomation experienced 15.7% and 25.0% higher rates of survival at 24 and 48 h, resp., compared to control crickets.  In the second expt., survival anal. confirmed that crickets protected with C60, regardless of C60 dose (1, 2, or 4 μg/g cricket mass), experienced greater survival up to 60 h and survived significantly longer than controls.  C60 fullerene conferred significant protection against rattlesnake envenomation in a cricket model.  Accordingly, C60 fullerene merits further consideration as a simple, low-cost treatment for snakebite and potentially many other forms of poisoning and envenomation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphOH2Y_cCIx7Vg90H21EOLACvtfcHk0liBEjdomm7X8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWks7vI&md5=fc4b74c66eb2942a909c0235e3750664</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1166%2Fjnn.2016.12851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1166%252Fjnn.2016.12851%26sid%3Dliteratum%253Aachs%26aulast%3DKarain%26aufirst%3DB.%2BD.%26aulast%3DLee%26aufirst%3DM.%2BK.%2BH.%26aulast%3DHayes%26aufirst%3DW.%2BK.%26atitle%3DC60%2520fullerenes%2520as%2520a%2520novel%2520treatment%2520for%2520poisoning%2520and%2520envenomation%253A%2520a%2520proof-of-concept%2520study%2520for%2520snakebite%26jtitle%3DJ.%2520Nanosci.%2520Nanotechnol.%26date%3D2016%26volume%3D16%26issue%3D7%26spage%3D7764%26epage%3D7771%26doi%3D10.1166%2Fjnn.2016.12851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickler, P.</span></span> <span> </span><span class="NLM_article-title">Varespladib (LY315920) appears to be a potent, broad-spectrum, inhibitor of snake venom phospholipase A2 and a possible pre-referral treatment for envenomation</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">248</span>, <span class="refDoi"> DOI: 10.3390/toxins8090248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.3390%2Ftoxins8090248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ClsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=248&issue=9&author=M.+Lewinauthor=S.+Samuelauthor=J.+Merkelauthor=P.+Bickler&title=Varespladib+%28LY315920%29+appears+to+be+a+potent%2C+broad-spectrum%2C+inhibitor+of+snake+venom+phospholipase+A2+and+a+possible+pre-referral+treatment+for+envenomation&doi=10.3390%2Ftoxins8090248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Varespladib (LY315920) appears to be a potent, broad-spectrum, inhibitor of snake venom phospholipase A2 and a possible pre-referral treatment for envenomation</span></div><div class="casAuthors">Lewin, Matthew; Samuel, Stephen; Merkel, Janie; Bickler, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">248/1-248/16</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Snakebite remains a neglected medical problem of the developing world with up to 125,000 deaths each year despite more than a century of calls to improve snakebite prevention and care.  An estd. 75% of fatalities from snakebite occur outside the hospital setting.  Because phospholipase A2 (PLA2) activity is an important component of venom toxicity, we sought candidate PLA2 inhibitors by directly testing drugs.  Surprisingly, varespladib and its orally bioavailable prodrug, methyl-varespladib showed high-level secretory PLA2 (sPLA2) inhibition at nanomolar and picomolar concns. against 28 medically important snake venoms from six continents.  In vivo proof-of-concept studies with varespladib had striking survival benefit against LDs of Micrurus fulvius and Vipera berus venom, and suppressed venom-induced sPLA2 activity in rats challenged with 100% LDs of M. fulvius venom.  Rapid development and deployment of a broad-spectrum PLA2 inhibitor alone or in combination with other small mol. inhibitors of snake toxins (e.g., metalloproteases) could fill the crit. therapeutic gap spanning pre-referral and hospital setting.  Lower barriers for clin. testing of safety tested, repurposed small mol. therapeutics are a potentially economical and effective path forward to fill the pre-referral gap in the setting of snakebite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgyWpYG3kB9LVg90H21EOLACvtfcHk0liBEjdomm7X8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ClsL%252FO&md5=7342f424a5f3375ed75ff45d84f7b74b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Ftoxins8090248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins8090248%26sid%3Dliteratum%253Aachs%26aulast%3DLewin%26aufirst%3DM.%26aulast%3DSamuel%26aufirst%3DS.%26aulast%3DMerkel%26aufirst%3DJ.%26aulast%3DBickler%26aufirst%3DP.%26atitle%3DVarespladib%2520%2528LY315920%2529%2520appears%2520to%2520be%2520a%2520potent%252C%2520broad-spectrum%252C%2520inhibitor%2520of%2520snake%2520venom%2520phospholipase%2520A2%2520and%2520a%2520possible%2520pre-referral%2520treatment%2520for%2520envenomation%26jtitle%3DToxins%26date%3D2016%26volume%3D8%26issue%3D9%26spage%3D248%26doi%3D10.3390%2Ftoxins8090248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scherf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katchalski-Katzir, E.</span></span> <span> </span><span class="NLM_article-title">A β-hairpin structure in a 13-mer peptide that binds α-bungarotoxin with high affinity and neutralizes its toxicity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">6629</span>– <span class="NLM_lpage">6634</span>, <span class="refDoi"> DOI: 10.1073/pnas.111164298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1073%2Fpnas.111164298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=11381118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksVOktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=6629-6634&issue=12&author=T.+Scherfauthor=R.+Kasherauthor=M.+Balassauthor=M.+Fridkinauthor=S.+Fuchsauthor=E.+Katchalski-Katzir&title=A+%CE%B2-hairpin+structure+in+a+13-mer+peptide+that+binds+%CE%B1-bungarotoxin+with+high+affinity+and+neutralizes+its+toxicity&doi=10.1073%2Fpnas.111164298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A β-hairpin structure in a 13-mer peptide that binds α-bungarotoxin with high affinity and neutralizes its toxicity</span></div><div class="casAuthors">Scherf, Tali; Kasher, Roni; Balass, Moshe; Fridkin, Mati; Fuchs, Sara; Katchalski-Katzir, Ephraim</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6629-6634</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Snake-venom α-bungarotoxin is a member of the α-neurotoxin family that binds with very high affinity to the nicotinic acetylcholine receptor (AChR) at the neuromuscular junction.  The structure of the complex between α-bungarotoxin and a 13-mer peptide (WRYYESSLEPYPD) that binds the toxin with high affinity, thus inhibiting its interactions with AChR with an IC50 of 2 nM, has been solved by 1H-NMR spectroscopy.  The bound peptide folds into a β-hairpin structure created by two antiparallel β-strands, which combine with the already existing triple-stranded β-sheet of the toxin to form a five-stranded intermol., antiparallel β-sheet.  Peptide residues Y3P, E5P, and L8P have the highest intermol. contact area, indicating their importance in the binding of α-bungarotoxin; W1P, R2P, and Y4P also contribute significantly to the binding.  A large no. of characteristic hydrogen bonds and electrostatic and hydrophobic interactions are obsd. in the complex.  The high-affinity peptide exhibits inhibitory potency that is better than any known peptide derived from AChR, and is equal to that of the whole α-subunit of AChR.  The high degree of sequence similarity between the peptide and various types of AChRs implies that the binding mode found within the complex might possibly mimic the receptor binding to the toxin.  The design of the high-affinity peptide was based on our previous findings: (i) the detection of a lead peptide (MRYYESSLKSYPD) that binds α-bungarotoxin, using a phage-display peptide library, (ii) the information about the three-dimensional structure of α-bungarotoxin/lead-peptide complex, and (iii) the amino acid sequence anal. of different AChRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1LI6CqiC7AbVg90H21EOLACvtfcHk0liBEjdomm7X8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksVOktb4%253D&md5=0e2764b893941637227f790f3fe6984b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.111164298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.111164298%26sid%3Dliteratum%253Aachs%26aulast%3DScherf%26aufirst%3DT.%26aulast%3DKasher%26aufirst%3DR.%26aulast%3DBalass%26aufirst%3DM.%26aulast%3DFridkin%26aufirst%3DM.%26aulast%3DFuchs%26aufirst%3DS.%26aulast%3DKatchalski-Katzir%26aufirst%3DE.%26atitle%3DA%2520%25CE%25B2-hairpin%2520structure%2520in%2520a%252013-mer%2520peptide%2520that%2520binds%2520%25CE%25B1-bungarotoxin%2520with%2520high%2520affinity%2520and%2520neutralizes%2520its%2520toxicity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26issue%3D12%26spage%3D6629%26epage%3D6634%26doi%3D10.1073%2Fpnas.111164298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fosgerau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span> <span> </span><span class="NLM_article-title">Peptide therapeutics: current status and future directions</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.drudis.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=25450771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=122-128&issue=1&author=K.+Fosgerauauthor=T.+Hoffmann&title=Peptide+therapeutics%3A+current+status+and+future+directions&doi=10.1016%2Fj.drudis.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics: current status and future directions</span></div><div class="casAuthors">Fosgerau, Keld; Hoffmann, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized for being highly selective and efficacious and, at the same time, relatively safe and well tolerated.  Consequently, there is an increased interest in peptides in pharmaceutical research and development (R&D), and approx. 140 peptide therapeutics are currently being evaluated in clin. trials.  Given that the low-hanging fruits in the form of obvious peptide targets have already been picked, it has now become necessary to explore new routes beyond traditional peptide design.  Examples of such approaches are multifunctional and cell penetrating peptides, as well as peptide drug conjugates.  Here, we discuss the current status, strengths, and weaknesses of peptides as medicines and the emerging new opportunities in peptide drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmhpp8GnJ6JLVg90H21EOLACvtfcHk0lieU6mOYeYNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM&md5=f6b74aaa4e3fdb317b091b0450f3618a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DFosgerau%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26atitle%3DPeptide%2520therapeutics%253A%2520current%2520status%2520and%2520future%2520directions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26issue%3D1%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.drudis.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guss, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brejc, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sixma, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katchalski-Katzir, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sussman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, S.</span></span> <span> </span><span class="NLM_article-title">The binding site of acetylcholine receptor as visualized in the X-ray structure of a complex between α-bungarotoxin and a mimotope peptide</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/S0896-6273(01)00461-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2FS0896-6273%2801%2900461-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=11683996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3MXosVKmtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2001&pages=265-275&issue=2&author=M.+Harelauthor=R.+Kasherauthor=A.+Nicolasauthor=J.+M.+Gussauthor=M.+Balassauthor=M.+Fridkinauthor=A.+B.+Smitauthor=K.+Brejcauthor=T.+K.+Sixmaauthor=E.+Katchalski-Katzirauthor=J.+L.+Sussmanauthor=S.+Fuchs&title=The+binding+site+of+acetylcholine+receptor+as+visualized+in+the+X-ray+structure+of+a+complex+between+%CE%B1-bungarotoxin+and+a+mimotope+peptide&doi=10.1016%2FS0896-6273%2801%2900461-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The binding site of acetylcholine receptor as visualized in the X-ray structure of a complex between α-bungarotoxin and a mimotope peptide</span></div><div class="casAuthors">Harel, Michal; Kasher, Roni; Nicolas, Anne; Guss, J. Mitchell; Balass, Moshe; Fridkin, Mati; Smit, August B.; Brejc, Katjusa; Sixma, Titia K.; Katchalski-Katzir, Ephraim; Sussman, Joel L.; Fuchs, Sara</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">265-275</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We have detd. the crystal structure at 1.8 Å resoln. of a complex of α-bungarotoxin with a high affinity 13-residue peptide that is homologous to the binding region of the α subunit of acetylcholine receptor.  The peptide fits snugly to the toxin and adopts a β hairpin conformation.  The structures of the bound peptide and the homologous loop of acetylcholine binding protein, a sol. analog of the extracellular domain of acetylcholine receptor, are remarkably similar.  Their superposition indicates that the toxin wraps around the receptor binding site loop, and in addn., binds tightly at the interface of two of the receptor subunits where it inserts a finger into the ligand binding site, thus blocking access to the acetylcholine binding site and explaining its strong antagonistic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1Q9LO6C59FrVg90H21EOLACvtfcHk0lieU6mOYeYNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXosVKmtbo%253D&md5=017ecc2988e6dea2d943e7b83e0bb185</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0896-6273%2801%2900461-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0896-6273%252801%252900461-5%26sid%3Dliteratum%253Aachs%26aulast%3DHarel%26aufirst%3DM.%26aulast%3DKasher%26aufirst%3DR.%26aulast%3DNicolas%26aufirst%3DA.%26aulast%3DGuss%26aufirst%3DJ.%2BM.%26aulast%3DBalass%26aufirst%3DM.%26aulast%3DFridkin%26aufirst%3DM.%26aulast%3DSmit%26aufirst%3DA.%2BB.%26aulast%3DBrejc%26aufirst%3DK.%26aulast%3DSixma%26aufirst%3DT.%2BK.%26aulast%3DKatchalski-Katzir%26aufirst%3DE.%26aulast%3DSussman%26aufirst%3DJ.%2BL.%26aulast%3DFuchs%26aufirst%3DS.%26atitle%3DThe%2520binding%2520site%2520of%2520acetylcholine%2520receptor%2520as%2520visualized%2520in%2520the%2520X-ray%2520structure%2520of%2520a%2520complex%2520between%2520%25CE%25B1-bungarotoxin%2520and%2520a%2520mimotope%2520peptide%26jtitle%3DNeuron%26date%3D2001%26volume%3D32%26issue%3D2%26spage%3D265%26epage%3D275%26doi%3D10.1016%2FS0896-6273%2801%2900461-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chandy, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide blockers of Kv1.3 channels in T cells as therapeutics for autoimmune disease</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.cbpa.2017.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28412597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=97-107&author=K.+G.+Chandyauthor=R.+S.+Norton&title=Peptide+blockers+of+Kv1.3+channels+in+T+cells+as+therapeutics+for+autoimmune+disease&doi=10.1016%2Fj.cbpa.2017.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide blockers of Kv1.3 channels in T cells as therapeutics for autoimmune disease</span></div><div class="casAuthors">Chandy, K. George; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97-107</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The voltage-gated Kv1.3 channel in T lymphocytes is a validated therapeutic target for diverse autoimmune diseases.  Here we review the discovery of Kv1.3, its physiol. role in T cells, and why it is an attractive target for modulating autoimmune responses.  We focus on peptide inhibitors because the first Kv1.3-selective inhibitor in human trials is a peptide derived from a marine organism.  Two broad classes of peptides block Kv1.3, the first from scorpions and the second from sea anemones.  We describe their structures, their binding site in the external vestibule of Kv1.3, how they have been engineered to improve Kv1.3-specificity, and their pharmacokinetic and pharmacodynamic properties.  Finally, we highlight the therapeutic potential of Kv1.3 peptide inhibitors to treat autoimmune diseases without compromising protective immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkIKfZApsJ-7Vg90H21EOLACvtfcHk0lieU6mOYeYNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFOhtL8%253D&md5=d8a6ae633f17def39ce5b90b954a823d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DChandy%26aufirst%3DK.%2BG.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520blockers%2520of%2520Kv1.3%2520channels%2520in%2520T%2520cells%2520as%2520therapeutics%2520for%2520autoimmune%2520disease%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D97%26epage%3D107%26doi%3D10.1016%2Fj.cbpa.2017.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guidotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span> <span> </span><span class="NLM_article-title">Cell-penetrating peptides: from basic research to clinics</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">406</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2017.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.tips.2017.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28209404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=406-424&issue=4&author=G.+Guidottiauthor=L.+Brambillaauthor=D.+Rossi&title=Cell-penetrating+peptides%3A+from+basic+research+to+clinics&doi=10.1016%2Fj.tips.2017.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Penetrating Peptides: From Basic Research to Clinics</span></div><div class="casAuthors">Guidotti, Giulia; Brambilla, Liliana; Rossi, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">406-424</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The presence of cell and tissue barriers together with the low biomembrane permeability of various therapeutics often hampers systemic drug distribution; thus, most of the available mols. are of limited therapeutic value.  Opportunities to increase medicament concns. in areas that are difficult to access now exist with the advent of cell-penetrating peptides (CPPs), which can transport into the cell a wide variety of biol. active conjugates (cargoes).  Numerous preclin. evaluations with CPP-derived therapeutics have provided promising results in various disease models that, in some cases, prompted clin. trials.  The outcome of these investigations has thus opened new perspectives for CPP application in the development of unprecedented human therapies that are well tolerated and directed to intracellular targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG2xC3lO7EbrVg90H21EOLACvtfcHk0lg0ahNf9yktrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeitr8%253D&md5=58d735c7bea04d19417baa4e2cb49102</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2017.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2017.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGuidotti%26aufirst%3DG.%26aulast%3DBrambilla%26aufirst%3DL.%26aulast%3DRossi%26aufirst%3DD.%26atitle%3DCell-penetrating%2520peptides%253A%2520from%2520basic%2520research%2520to%2520clinics%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2017%26volume%3D38%26issue%3D4%26spage%3D406%26epage%3D424%26doi%3D10.1016%2Fj.tips.2017.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strohl, W. R.</span></span> <span> </span><span class="NLM_article-title">Fusion proteins for half-life extension of biologics as a strategy to make biobetters</span>. <i>BioDrugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1007/s40259-015-0133-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1007%2Fs40259-015-0133-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=26177629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WrtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=215-239&author=W.+R.+Strohl&title=Fusion+proteins+for+half-life+extension+of+biologics+as+a+strategy+to+make+biobetters&doi=10.1007%2Fs40259-015-0133-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fusion Proteins for Half-Life Extension of Biologics as a Strategy to Make Biobetters</span></div><div class="casAuthors">Strohl, William R.</div><div class="citationInfo"><span class="NLM_cas:title">BioDrugs</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215-239</span>CODEN:
                <span class="NLM_cas:coden">BIDRF4</span>;
        ISSN:<span class="NLM_cas:issn">1173-8804</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">The purpose of making a "biobetter" biol. is to improve on the salient characteristics of a known biol. for which there is, minimally, clin. proof of concept or, maximally, marketed product data.  There already are several examples in which second-generation or biobetter biologics have been generated by improving the pharmacokinetic properties of an innovative drug, including Neulasta [a PEGylated, longer-half-life version of Neupogen (filgrastim)] and Aranesp [a longer-half-life version of Epogen (epoetin-α)].  This review describes the use of protein fusion technologies such as Fc fusion proteins, fusion to human serum albumin, fusion to carboxy-terminal peptide, and other polypeptide fusion approaches to make biobetter drugs with more desirable pharmacokinetic profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqorgYj1NyuaLVg90H21EOLACvtfcHk0lg0ahNf9yktrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WrtLjN&md5=328f936b91e29c83e069db83a4d05cab</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs40259-015-0133-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40259-015-0133-6%26sid%3Dliteratum%253Aachs%26aulast%3DStrohl%26aufirst%3DW.%2BR.%26atitle%3DFusion%2520proteins%2520for%2520half-life%2520extension%2520of%2520biologics%2520as%2520a%2520strategy%2520to%2520make%2520biobetters%26jtitle%3DBioDrugs%26date%3D2015%26volume%3D29%26spage%3D215%26epage%3D239%26doi%3D10.1007%2Fs40259-015-0133-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span> <span> </span><span class="NLM_article-title">Strategic approaches to optimizing peptide ADME properties</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1208/s12248-014-9687-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1208%2Fs12248-014-9687-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=25366889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=134-143&issue=1&author=L.+Di&title=Strategic+approaches+to+optimizing+peptide+ADME+properties&doi=10.1208%2Fs12248-014-9687-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Strategic Approaches to Optimizing Peptide ADME Properties</span></div><div class="casAuthors">Di, Li</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">134-143</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Development of peptide drugs is challenging but also quite rewarding.  Five blockbuster peptide drugs are currently on the market, and six new peptides received first marketing approval as new mol. entities in 2012.  Although peptides only represent 2% of the drug market, the market is growing twice as quickly and might soon occupy a larger niche.  Natural peptides typically have poor absorption, distribution, metab., and excretion (ADME) properties with rapid clearance, short half-life, low permeability, and sometimes low soly.  Strategies have been developed to improve peptide drugability through enhancing permeability, reducing proteolysis and renal clearance, and prolonging half-life.  In vivo, in vitro, and in silico tools are available to evaluate ADME properties of peptides, and structural modification strategies are in place to improve peptide developability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDLDsR-Zk0OLVg90H21EOLACvtfcHk0lg0ahNf9yktrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ&md5=530597d3719d65509105096f433fee0a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1208%2Fs12248-014-9687-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-014-9687-3%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26atitle%3DStrategic%2520approaches%2520to%2520optimizing%2520peptide%2520ADME%2520properties%26jtitle%3DAAPS%2520J.%26date%3D2015%26volume%3D17%26issue%3D1%26spage%3D134%26epage%3D143%26doi%3D10.1208%2Fs12248-014-9687-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalafatovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giralt, E.</span></span> <span> </span><span class="NLM_article-title">Cell-penetrating peptides: design strategies beyond primary structure and amphipathicity</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1929</span>, <span class="refDoi"> DOI: 10.3390/molecules22111929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.3390%2Fmolecules22111929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVGjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1929&issue=11&author=D.+Kalafatovicauthor=E.+Giralt&title=Cell-penetrating+peptides%3A+design+strategies+beyond+primary+structure+and+amphipathicity&doi=10.3390%2Fmolecules22111929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-penetrating peptides: design strategies beyond primary structure and amphipathicity</span></div><div class="casAuthors">Kalafatovic, Daniela; Giralt, Ernest</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1929/1-1929/38</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Efficient intracellular drug delivery and target specificity are often hampered by the presence of biol. barriers.  Thus, compds. that efficiently cross cell membranes are the key to improving the therapeutic value and on-target specificity of non-permeable drugs.  The discovery of cell-penetrating peptides (CPPs) and the early design approaches through mimicking the natural penetration domains used by viruses have led to greater efficiency of intracellular delivery.  Following these nature-inspired examples, a no. of rationally designed CPPs has been developed.  In this review, a variety of CPP designs will be described, including linear and flexible, pos. charged and often amphipathic CPPs, and more rigid versions comprising cyclic, stapled, or dimeric and/or multivalent, self-assembled peptides or peptido-mimetics.  The application of distinct design strategies to known physico-chem. properties of CPPs offers the opportunity to improve their penetration efficiency and/or internalization kinetics.  This led to increased design complexity of new CPPs that does not always result in greater CPP activity.  Therefore, the transition of CPPs to a clin. setting remains a challenge also due to the concomitant involvement of various internalization routes and heterogeneity of cells used in the in vitro studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXVMiu0WvRi7Vg90H21EOLACvtfcHk0liYaqA7TlWwiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVGjtrY%253D&md5=cde796080669da1abb1bd06fd1ed57a5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.3390%2Fmolecules22111929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules22111929%26sid%3Dliteratum%253Aachs%26aulast%3DKalafatovic%26aufirst%3DD.%26aulast%3DGiralt%26aufirst%3DE.%26atitle%3DCell-penetrating%2520peptides%253A%2520design%2520strategies%2520beyond%2520primary%2520structure%2520and%2520amphipathicity%26jtitle%3DMolecules%26date%3D2017%26volume%3D22%26issue%3D11%26spage%3D1929%26doi%3D10.3390%2Fmolecules22111929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Longbottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearer, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devine, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcoba, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappuis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz de Castañeda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pigott, D. M.</span></span> <span> </span><span class="NLM_article-title">Vulnerability to snakebite envenoming: a global mapping of hotspots</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>392</i></span> (<span class="NLM_issue">10148</span>),  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(18)31224-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2FS0140-6736%2818%2931224-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=30017551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FptFeksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2018&pages=673-684&issue=10148&author=J.+Longbottomauthor=F.+M.+Shearerauthor=M.+Devineauthor=G.+Alcobaauthor=F.+Chappuisauthor=D.+J.+Weissauthor=S.+E.+Rayauthor=N.+Rayauthor=D.+A.+Warrellauthor=R.+Ruiz+de+Casta%C3%B1edaauthor=D.+J.+Williamsauthor=S.+I.+Hayauthor=D.+M.+Pigott&title=Vulnerability+to+snakebite+envenoming%3A+a+global+mapping+of+hotspots&doi=10.1016%2FS0140-6736%2818%2931224-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Vulnerability to snakebite envenoming: a global mapping of hotspots</span></div><div class="casAuthors">Longbottom Joshua; Shearer Freya M; Devine Maria; Weiss Daniel J; Alcoba Gabriel; Chappuis Francois; Ray Sarah E; Pigott David M; Ray Nicolas; Warrell David A; Ruiz de Castaneda Rafael; Williams David J; Hay Simon I</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">392</span>
        (<span class="NLM_cas:issue">10148</span>),
    <span class="NLM_cas:pages">673-684</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Snakebite envenoming is a frequently overlooked cause of mortality and morbidity.  Data for snake ecology and existing snakebite interventions are scarce, limiting accurate burden estimation initiatives.  Low global awareness stunts new interventions, adequate health resources, and available health care.  Therefore, we aimed to synthesise currently available data to identify the most vulnerable populations at risk of snakebite, and where additional data to manage this global problem are needed.  METHODS:  We assembled a list of snake species using WHO guidelines.  Where relevant, we obtained expert opinion range (EOR) maps from WHO or the Clinical Toxinology Resources.  We also obtained occurrence data for each snake species from a variety of websites, such as VertNet and iNaturalist, using the spocc R package (version 0.7.0).  We removed duplicate occurrence data and categorised snakes into three groups: group A (no available EOR map or species occurrence records), group B (EOR map but <5 species occurrence records), and group C (EOR map and ≥5 species occurrence records).  For group C species, we did a multivariate environmental similarity analysis using the 2008 WHO EOR maps and newly available evidence.  Using these data and the EOR maps, we produced contemporary range maps for medically important venomous snake species at a 5 × 5 km resolution.  We subsequently triangulated these data with three health system metrics (antivenom availability, accessibility to urban centres, and the Healthcare Access and Quality [HAQ] Index) to identify the populations most vulnerable to snakebite morbidity and mortality.  FINDINGS:  We provide a map showing the ranges of 278 snake species globally.  Although about 6·85 billion people worldwide live within range of areas inhabited by snakes, about 146·70 million live within remote areas lacking quality health-care provisioning.  Comparing opposite ends of the HAQ Index, 272·91 million individuals (65·25%) of the population within the lowest decile are at risk of exposure to any snake for which no effective therapy exists compared with 519·46 million individuals (27·79%) within the highest HAQ Index decile, showing a disproportionate coverage in reported antivenom availability.  Antivenoms were available for 119 (43%) of 278 snake species evaluated by WHO, while globally 750·19 million (10·95%) of those living within snake ranges live more than 1 h from population centres.  In total, we identify about 92·66 million people living within these vulnerable geographies, including many sub-Saharan countries, Indonesia, and other parts of southeast Asia.  INTERPRETATION:  Identifying exact populations vulnerable to the most severe outcomes of snakebite envenoming at a subnational level is important for prioritising new data collection and collation, reinforcing envenoming treatment, existing health-care systems, and deploying currently available and future interventions.  These maps can guide future research efforts on snakebite envenoming from both ecological and public health perspectives and better target future estimates of the burden of this neglected tropical disease.  FUNDING:  Bill & Melinda Gates Foundation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6cc4xiLoYKv6BottYdIAWfW6udTcc2eZSwYXmG3IqvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FptFeksw%253D%253D&md5=630c65ee36b0a83ae04585d3dbdaaf87</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2818%2931224-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252818%252931224-8%26sid%3Dliteratum%253Aachs%26aulast%3DLongbottom%26aufirst%3DJ.%26aulast%3DShearer%26aufirst%3DF.%2BM.%26aulast%3DDevine%26aufirst%3DM.%26aulast%3DAlcoba%26aufirst%3DG.%26aulast%3DChappuis%26aufirst%3DF.%26aulast%3DWeiss%26aufirst%3DD.%2BJ.%26aulast%3DRay%26aufirst%3DS.%2BE.%26aulast%3DRay%26aufirst%3DN.%26aulast%3DWarrell%26aufirst%3DD.%2BA.%26aulast%3DRuiz%2Bde%2BCasta%25C3%25B1eda%26aufirst%3DR.%26aulast%3DWilliams%26aufirst%3DD.%2BJ.%26aulast%3DHay%26aufirst%3DS.%2BI.%26aulast%3DPigott%26aufirst%3DD.%2BM.%26atitle%3DVulnerability%2520to%2520snakebite%2520envenoming%253A%2520a%2520global%2520mapping%2520of%2520hotspots%26jtitle%3DLancet%26date%3D2018%26volume%3D392%26issue%3D10148%26spage%3D673%26epage%3D684%26doi%3D10.1016%2FS0140-6736%2818%2931224-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, B.</span></span> <span> </span><span class="NLM_article-title">Synthetic antivenoms for snakebites: a slithering road!</span>. <i>Biochemist</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">6</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1042/BIO04106006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1042%2FBIO04106006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=6-9&issue=6&author=B.+Lohse&title=Synthetic+antivenoms+for+snakebites%3A+a+slithering+road%21&doi=10.1042%2FBIO04106006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1042%2FBIO04106006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBIO04106006%26sid%3Dliteratum%253Aachs%26aulast%3DLohse%26aufirst%3DB.%26atitle%3DSynthetic%2520antivenoms%2520for%2520snakebites%253A%2520a%2520slithering%2520road%2521%26jtitle%3DBiochemist%26date%3D2019%26volume%3D41%26issue%3D6%26spage%3D6%26epage%3D9%26doi%3D10.1042%2FBIO04106006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lynagh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komnatnyy, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pless, S. A.</span></span> <span> </span><span class="NLM_article-title">Unique contributions of an arginine side chain to ligand recognition in a glutamate-gated chloride channel</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3940</span>– <span class="NLM_lpage">3946</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.772939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1074%2Fjbc.M116.772939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=28096462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjslSju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=3940-3946&issue=9&author=T.+Lynaghauthor=V.+V.+Komnatnyyauthor=S.+A.+Pless&title=Unique+contributions+of+an+arginine+side+chain+to+ligand+recognition+in+a+glutamate-gated+chloride+channel&doi=10.1074%2Fjbc.M116.772939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Unique Contributions of an Arginine Side Chain to Ligand Recognition in a Glutamate-gated Chloride Channel</span></div><div class="casAuthors">Lynagh, Timothy; Komnatnyy, Vitaly V.; Pless, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3940-3946</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Glutamate recognition by neurotransmitter receptors often relies on Arg residues in the binding site, leading to the assumption that charge-charge interactions underlie ligand recognition.  However, assessing the precise chem. contribution of Arg side chains to protein function and pharmacol. has proven to be exceedingly difficult in such large and complex proteins.  Using the in vivo nonsense suppression approach, we report the first successful incorporation of the isosteric, titratable Arg analog, canavanine, into a neurotransmitter receptor in a living cell, utilizing a glutamate-gated chloride channel from the nematode Haemonchus contortus.  Our data unveil a surprisingly small contribution of charge at a conserved arginine side chain previously suggested to form a salt bridge with the ligand, glutamate.  Instead, our data show that Arg contributes crucially to ligand sensitivity via a hydrogen bond network, where Arg interacts both with agonist and with a conserved Thr side chain within the receptor.  Together, the data provide a new explanation for the reliance of neurotransmitter receptors on Arg side chains and highlight the exceptional capacity of unnatural amino acid incorporation for increasing our understanding of ligand recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2BltxAbOrdrVg90H21EOLACvtfcHk0lhLmjmMOGI-Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjslSju7k%253D&md5=c5bddfb68e38f3b286c43e4c453b6e29</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.772939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.772939%26sid%3Dliteratum%253Aachs%26aulast%3DLynagh%26aufirst%3DT.%26aulast%3DKomnatnyy%26aufirst%3DV.%2BV.%26aulast%3DPless%26aufirst%3DS.%2BA.%26atitle%3DUnique%2520contributions%2520of%2520an%2520arginine%2520side%2520chain%2520to%2520ligand%2520recognition%2520in%2520a%2520glutamate-gated%2520chloride%2520channel%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26issue%3D9%26spage%3D3940%26epage%3D3946%26doi%3D10.1074%2Fjbc.M116.772939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Revah, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galzi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devillers-Thiéry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballivet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J. P.</span></span> <span> </span><span class="NLM_article-title">Mutations in the channel domain alter desensitization of a neuronal nicotinic receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>353</i></span> (<span class="NLM_issue">6347</span>),  <span class="NLM_fpage">846</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.1038/353846a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1038%2F353846a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1719423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADyaK38XjvFKk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=1991&pages=846-849&issue=6347&author=F.+Revahauthor=D.+Bertrandauthor=J.+L.+Galziauthor=A.+Devillers-Thi%C3%A9ryauthor=C.+Mulleauthor=N.+Hussyauthor=S.+Bertrandauthor=M.+Ballivetauthor=J.+P.+Changeux&title=Mutations+in+the+channel+domain+alter+desensitization+of+a+neuronal+nicotinic+receptor&doi=10.1038%2F353846a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the channel domain alter desensitization of a neuronal nicotinic receptor</span></div><div class="casAuthors">Revah, F.; Bertrand, D.; Galzi, J. L.; Devillers-Thiery, A.; Mulle, C.; Hussy, N.; Bertrand, S.; Ballivet, M.; Changeux, J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6347</span>),
    <span class="NLM_cas:pages">846-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The authors investigated how structural elements could be involved in the desensitization of the acetylcholine-gated ion channel from the chick brain α-bungarotoxin-sensitive homo-oligomeric α7 receptor, using site-directed mutagenesis and expression in Xenopus oocytes.  Mutations of the highly conserved leucine 247 residue from the uncharged MII segment of α7 suppress inhibition by the open-channel blocker QX-222, indicating that this residue, like others from MII, faces the lumen of the channel.  But, unexpectedly, the same mutations decrease the rate of desensitization of the response, increase the apparent affinity for acetylcholine and abolish current rectification.  Moreover, unlike wild-type α7, which has channels with a single conductance level, the leucine-to-threonine mutant has an addnl. conducting state active at low acetylcholine concns.  It is possible that mutation of Leu247 renders conductive 1 of the high-affinity desensitized states of the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwHhZKCopf5LVg90H21EOLACvtfcHk0lhLmjmMOGI-Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XjvFKk&md5=41c3680c40d1fe9fc28f9bf1cbee46c6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2F353846a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F353846a0%26sid%3Dliteratum%253Aachs%26aulast%3DRevah%26aufirst%3DF.%26aulast%3DBertrand%26aufirst%3DD.%26aulast%3DGalzi%26aufirst%3DJ.%2BL.%26aulast%3DDevillers-Thi%25C3%25A9ry%26aufirst%3DA.%26aulast%3DMulle%26aufirst%3DC.%26aulast%3DHussy%26aufirst%3DN.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DBallivet%26aufirst%3DM.%26aulast%3DChangeux%26aufirst%3DJ.%2BP.%26atitle%3DMutations%2520in%2520the%2520channel%2520domain%2520alter%2520desensitization%2520of%2520a%2520neuronal%2520nicotinic%2520receptor%26jtitle%3DNature%26date%3D1991%26volume%3D353%26issue%3D6347%26spage%3D846%26epage%3D849%26doi%3D10.1038%2F353846a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halevi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yassin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treinin, M.</span></span> <span> </span><span class="NLM_article-title">Conservation within the RIC-3 Gene Family: effectors of mammalian nicotinic acetylcholine receptor expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">34411</span>– <span class="NLM_lpage">34417</span>, <span class="refDoi"> DOI: 10.1074/jbc.M300170200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1074%2Fjbc.M300170200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=12821669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvVCqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=34411-34417&issue=36&author=S.+Haleviauthor=L.+Yassinauthor=M.+Eshelauthor=F.+Salaauthor=S.+Salaauthor=M.+Criadoauthor=M.+Treinin&title=Conservation+within+the+RIC-3+Gene+Family%3A+effectors+of+mammalian+nicotinic+acetylcholine+receptor+expression&doi=10.1074%2Fjbc.M300170200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation within the RIC-3 gene family: effectors of mammalian nicotinic acetylcholine receptor expression</span></div><div class="casAuthors">Halevi, Sarah; Yassin, Lina; Eshel, Margalit; Sala, Francisco; Sala, Salvador; Criado, Manuel; Treinin, Millet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">34411-34417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In Caenorhabditis elegans, the ric-3 gene is required for the maturation of multiple nicotinic acetylcholine receptors (nAChRs), whereas other neurotransmitter gated channels expressed within the same cells are unaffected by the presence of RIC-3.  RIC-3 is a member of a conserved gene family with representatives in both vertebrates and invertebrates.  All members of this family have two transmembrane domains followed by a coiled-coil domain.  Expression of the human ric-3 homolog, hric3, like the C. elegans ric-3, enhances C. elegans DEG-3/DES-2, rat α7, and human α7 nAChR-dependent whole-cell current amplitudes in Xenopus laevis oocytes, thus demonstrating functional conservation.  However, hric3 also reduces human α4β2 and α3β4 nAChR-dependent whole-cell current amplitudes.  Thus, hric3 shows differential effects on human nAChRs unlike the obsd. uniform effect of ric-3 on C. elegans nAChRs.  Moreover, hric3 totally abolished currents evoked by 5-HT3 serotonin receptors, whereas it barely modified α1 glycine receptor currents.  With this caveat, RIC-3 belongs to a conserved family of genes likely to regulate nAChR-mediated transmission throughout evolution.  Anal. of transcripts encoded by the hric3 locus shows that it encodes for multiple transcripts, likely to produce multiple hric3 isoforms, and that hric3 is expressed in neurons and muscles, thus enabling its interactions with nAChRs in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiT7p8XOtg3bVg90H21EOLACvtfcHk0lhWYZnO_gIWTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvVCqsbc%253D&md5=acb379bab49af26982974eec4cb1f180</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M300170200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M300170200%26sid%3Dliteratum%253Aachs%26aulast%3DHalevi%26aufirst%3DS.%26aulast%3DYassin%26aufirst%3DL.%26aulast%3DEshel%26aufirst%3DM.%26aulast%3DSala%26aufirst%3DF.%26aulast%3DSala%26aufirst%3DS.%26aulast%3DCriado%26aufirst%3DM.%26aulast%3DTreinin%26aufirst%3DM.%26atitle%3DConservation%2520within%2520the%2520RIC-3%2520Gene%2520Family%253A%2520effectors%2520of%2520mammalian%2520nicotinic%2520acetylcholine%2520receptor%2520expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26issue%3D36%26spage%3D34411%26epage%3D34417%26doi%3D10.1074%2Fjbc.M300170200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dibas, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auyeung, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezanilla, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lester, H. A.</span></span> <span> </span><span class="NLM_article-title">A fluorophore attached to nicotinic acetylcholine receptor βM2 detects productive binding of agonist to the αδ site</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">10195</span>– <span class="NLM_lpage">10200</span>, <span class="refDoi"> DOI: 10.1073/pnas.0301885101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1073%2Fpnas.0301885101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15218096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFOhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=10195-10200&issue=27&author=D.+S.+Dahanauthor=M.+I.+Dibasauthor=E.+J.+Peterssonauthor=V.+C.+Auyeungauthor=B.+Chandaauthor=F.+Bezanillaauthor=D.+A.+Doughertyauthor=H.+A.+Lester&title=A+fluorophore+attached+to+nicotinic+acetylcholine+receptor+%CE%B2M2+detects+productive+binding+of+agonist+to+the+%CE%B1%CE%B4+site&doi=10.1073%2Fpnas.0301885101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorophore attached to nicotinic acetylcholine receptor βM2 detects productive binding of agonist to the αδ site</span></div><div class="casAuthors">Dahan, David S.; Dibas, Mohammed I.; Petersson, E. James; Auyeung, Vincent C.; Chanda, Baron; Bezanilla, Francisco; Dougherty, Dennis A.; Lester, Henry A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">10195-10200</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">To study conformational transitions at the muscle nicotinic acetylcholine (ACh) receptor (nAChR), a rhodamine fluorophore was tethered to a Cys side chain introduced at the β19' position in the M2 region of the nAChR expressed in Xenopus oocytes.  This procedure led to only minor changes in receptor function.  During agonist application, fluorescence increased by (ΔF/F) ≈10%, and the emission peak shifted to lower wavelengths, indicating a more hydrophobic environment for the fluorophore.  The dose-response relations for ΔF agreed well with those for epibatidine-induced currents, but were shifted ≈100-fold to the left of those for ACh-induced currents.  Because (i) epibatidine binds more tightly to the αγ-binding site than to the αδ site and (ii) ACh binds with reverse-site selectivity, these data suggest that ΔF monitors an event linked to binding specifically at the αδ-subunit interface.  In expts. with flash-applied agonists, the earliest detectable ΔF occurs within milliseconds, i.e., during activation.  At low [ACh] (≤ 10 μM), a phase of ΔF occurs with the same time const. as desensitization, presumably monitoring an increased population of agonist-bound receptors.  However, recovery from ΔF is complete before the slowest phase of recovery from desensitization (time const. ≈250 s), showing that one or more desensitized states have fluorescence like that of the resting channel.  That conformational transitions at the αδ-binding site are not tightly coupled to channel activation suggests that sequential rather than fully concerted transitions occur during receptor gating.  Thus, time-resolved fluorescence changes provide a powerful probe of nAChR conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGQiNXhLapRbVg90H21EOLACvtfcHk0lhWYZnO_gIWTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFOhsb4%253D&md5=834c616a35697f1e131bfd5701595afe</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0301885101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0301885101%26sid%3Dliteratum%253Aachs%26aulast%3DDahan%26aufirst%3DD.%2BS.%26aulast%3DDibas%26aufirst%3DM.%2BI.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DAuyeung%26aufirst%3DV.%2BC.%26aulast%3DChanda%26aufirst%3DB.%26aulast%3DBezanilla%26aufirst%3DF.%26aulast%3DDougherty%26aufirst%3DD.%2BA.%26aulast%3DLester%26aufirst%3DH.%2BA.%26atitle%3DA%2520fluorophore%2520attached%2520to%2520nicotinic%2520acetylcholine%2520receptor%2520%25CE%25B2M2%2520detects%2520productive%2520binding%2520of%2520agonist%2520to%2520the%2520%25CE%25B1%25CE%25B4%2520site%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D27%26spage%3D10195%26epage%3D10200%26doi%3D10.1073%2Fpnas.0301885101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&issue=2&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization&doi=10.1016%2Fj.ab.2004.05.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0lhWYZnO_gIWTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26issue%3D2%26spage%3D261%26epage%3D273%26doi%3D10.1016%2Fj.ab.2004.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellmig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paithankar, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puhringer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zocher, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">Facilities for macromolecular crystallography at the Helmholtz-Zentrum Berlin</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1107/S0909049512006395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0909049512006395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=22514183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFGhu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=442-449&issue=3&author=U.+Muellerauthor=N.+Darowskiauthor=M.+R.+Fuchsauthor=R.+Forsterauthor=M.+Hellmigauthor=K.+S.+Paithankarauthor=S.+Puhringerauthor=M.+Steffienauthor=G.+Zocherauthor=M.+S.+Weiss&title=Facilities+for+macromolecular+crystallography+at+the+Helmholtz-Zentrum+Berlin&doi=10.1107%2FS0909049512006395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Facilities for macromolecular crystallography at the Helmholtz-Zentrum Berlin</span></div><div class="casAuthors">Mueller, Uwe; Darowski, Nora; Fuchs, Martin R.; Foerster, Ronald; Hellmig, Michael; Paithankar, Karthik S.; Puehringer, Sandra; Steffien, Michael; Zocher, Georg; Weiss, Manfred S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">442-449</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Three macromol. crystallog. (MX) beamlines at the Helmholtz-Zentrum Berlin (HZB) are available for the regional, national and international structural biol. user community.  The state-of-the-art synchrotron beamlines for MX BL14.1, BL14.2 and BL14.3 are located within the low-β section of the BESSY II electron storage ring.  All beamlines are fed from a superconducting 7 T wavelength-shifter insertion device.  BL14.1 and BL14.2 are energy tunable in the range 5-16 keV, while BL14.3 is a fixed-energy side station operated at 13.8 keV.  All three beamlines are equipped with CCD detectors.  BL14.1 and BL14.2 are in regular user operation providing about 200 beam days per yr and about 600 user shifts to approx. 50 research groups across Europe.  BL14.3 has initially been used as a test facility and was brought into regular user mode operation during the year 2010.  BL14.1 has recently been upgraded with a microdiffractometer including a mini-κ goniometer and an automated sample changer.  Addnl. user facilities include office space adjacent to the beamlines, a sample prepn. lab., a biol. lab. (safety level 1) and high-end computing resources.  In this article the instrumentation of the beamlines is described, and a summary of the exptl. possibilities of the beamlines and the provided ancillary equipment for the user community is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEtRGj9KmXzLVg90H21EOLACvtfcHk0lhjIFhuz4ywiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFGhu7o%253D&md5=f065b38da0a77d30031a9db5a67eb1cc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0909049512006395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049512006395%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DDarowski%26aufirst%3DN.%26aulast%3DFuchs%26aufirst%3DM.%2BR.%26aulast%3DForster%26aufirst%3DR.%26aulast%3DHellmig%26aufirst%3DM.%26aulast%3DPaithankar%26aufirst%3DK.%2BS.%26aulast%3DPuhringer%26aufirst%3DS.%26aulast%3DSteffien%26aufirst%3DM.%26aulast%3DZocher%26aufirst%3DG.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DFacilities%2520for%2520macromolecular%2520crystallography%2520at%2520the%2520Helmholtz-Zentrum%2520Berlin%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2012%26volume%3D19%26issue%3D3%26spage%3D442%26epage%3D449%26doi%3D10.1107%2FS0909049512006395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&issue=2&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhjIFhuz4ywiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krug, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span> <span> </span><span class="NLM_article-title">XDSAPP: a graphical user interface for the convenient processing of diffraction data using XDS</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">572</span>, <span class="refDoi"> DOI: 10.1107/S0021889812011715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0021889812011715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFGmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=568-572&issue=3&author=M.+Krugauthor=M.+S.+Weissauthor=U.+Heinemannauthor=U.+Mueller&title=XDSAPP%3A+a+graphical+user+interface+for+the+convenient+processing+of+diffraction+data+using+XDS&doi=10.1107%2FS0021889812011715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">XDSAPP: a graphical user interface for the convenient processing of diffraction data using XDS</span></div><div class="casAuthors">Krug, Michael; Weiss, Manfred S.; Heinemann, Udo; Mueller, Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">568-572</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">XDSAPP is a Tcl/Tk-based graphical user interface for the easy and convenient processing of diffraction data sets using XDS.  It provides easy access to all XDS functionalities, automates the data processing and generates graphical plots of various data set statistics provided by XDS.  By incorporating addnl. software, further information on certain features of the data set, such as radiation decay during data collection or the presence of pseudo-translational symmetry and/or twinning, can be obtained.  Intensity files suitable for CCP4, CNS and SHELX are generated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqFLQ2J-k07Vg90H21EOLACvtfcHk0lhjIFhuz4ywiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFGmtLo%253D&md5=8f0b78fdea7ed6cc580de9eb561e8b9e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0021889812011715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889812011715%26sid%3Dliteratum%253Aachs%26aulast%3DKrug%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DHeinemann%26aufirst%3DU.%26aulast%3DMueller%26aufirst%3DU.%26atitle%3DXDSAPP%253A%2520a%2520graphical%2520user%2520interface%2520for%2520the%2520convenient%2520processing%2520of%2520diffraction%2520data%2520using%2520XDS%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2012%26volume%3D45%26issue%3D3%26spage%3D568%26epage%3D572%26doi%3D10.1107%2FS0021889812011715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgWdw26DrcTzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickitenko, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michailov, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Three-dimensional structure of neurotoxin-1 from Naja naja oxiana venom at 1.9 Å resolution</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>320</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(93)80073-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1016%2F0014-5793%2893%2980073-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=8458425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1amt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=1993&pages=111-117&issue=2&author=A.+V.+Nickitenkoauthor=A.+M.+Michailovauthor=C.+Betzelauthor=K.+S.+Wilson&title=Three-dimensional+structure+of+neurotoxin-1+from+Naja+naja+oxiana+venom+at+1.9+%C3%85+resolution&doi=10.1016%2F0014-5793%2893%2980073-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Three-dimensional structure of neurotoxin-1 from Naja naja oxiana venom at 1.9 Å resolution</span></div><div class="casAuthors">Nickitenko, A. V.; Michailov, A. M.; Betzel, Ch.; Wilson, K. S.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-17</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    </div><div class="casAbstract">Neurotoxin-1 from N. naja oxiana venom (NTX-1) was crystd. by vapor diffusion in sitting drops.  The crystals have cell dimensions of a = 25.2 Å, b = 75.6 Å, c = 35.9 Å, and are in space group P212121.  Three-dimensional data to 1.9 Å were recorded by a Syntex P21 automatic diffractometer.  The at. structure of the toxin was detd. by mol. replacement using the α-cobratoxin (α-CTX) as the search model.  The position of 534 nonhydrogen protein atoms were detd.  The model contains 65 water mols.  Refinement has led to an R-factor of 19.3% at 1.9 Å resoln.  The secondary and tertiary structures of NTX-1 were analyzed and a comparison with structure of the α-CTX was made.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1EFZ5A5GNr7Vg90H21EOLACvtfcHk0lgWdw26DrcTzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1amt7s%253D&md5=81d7257e408b21e60b8569571e6923d7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2893%2980073-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252893%252980073-4%26sid%3Dliteratum%253Aachs%26aulast%3DNickitenko%26aufirst%3DA.%2BV.%26aulast%3DMichailov%26aufirst%3DA.%2BM.%26aulast%3DBetzel%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DThree-dimensional%2520structure%2520of%2520neurotoxin-1%2520from%2520Naja%2520naja%2520oxiana%2520venom%2520at%25201.9%2520%25C3%2585%2520resolution%26jtitle%3DFEBS%2520Lett.%26date%3D1993%26volume%3D320%26issue%3D2%26spage%3D111%26epage%3D117%26doi%3D10.1016%2F0014-5793%2893%2980073-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12 Part 1</span>),  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&issue=12+Part+1&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgWdw26DrcTzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D12%2520Part%25201%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&issue=3&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgWdw26DrcTzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26issue%3D3%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM','PDB','6ZFM'); return false;">PDB: 6ZFM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CTX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CTX','PDB','2CTX'); return false;">PDB: 2CTX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NTN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NTN','PDB','1NTN'); return false;">PDB: 1NTN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YI5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YI5','PDB','1YI5'); return false;">PDB: 1YI5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UWZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UWZ','PDB','6UWZ'); return false;">PDB: 6UWZ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01202">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_70327"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01202?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01202</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Effects of additional peptides on α-cobratoxin inhibition of nAChRs; chemical stability of peptides <b>1</b> and <b>12</b>; architecture of the <b>12</b>:α-Cbtx complex observed in the crystal structure; structurally homologous α-neurotoxin binding interactions with other proteins; HPLC chromatograms and MS spectra; PDB file <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a> (<b>12</b>:α-Cbtx X-ray cocrystal structure) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01202/suppl_file/jm0c01202_si_001.pdf">jm0c01202_si_001.pdf (2.73 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the 12:α-Cbtx X-ray cocrystal structure have been deposited in the RCSB Protein Data Bank with accession number PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZFM">6ZFM</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01202&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01202%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01202" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799142a8bfc1923","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
