
<html lang="en"     class="pb-page"  data-request-id="c4e5f939-64bf-4400-9ca8-664e0dd506dd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2014.57.issue-14;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm5007098;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Flavin Monooxygenase Metabolism: Why Medicinal Chemists Should Matter" /></meta><meta name="dc.Creator" content="Gabriele  Cruciani" /></meta><meta name="dc.Creator" content="Aurora  Valeri" /></meta><meta name="dc.Creator" content="Laura  Goracci" /></meta><meta name="dc.Creator" content="Roberto Maria  Pellegrino" /></meta><meta name="dc.Creator" content="Federica  Buonerba" /></meta><meta name="dc.Creator" content="Massimo  Baroni" /></meta><meta name="dc.Description" content="FMO enzymes (FMOs) play a key role in the processes of detoxification and/or bioactivation of specific pharmaceuticals and xenobiotics bearing nucleophilic centers. The N-oxide and S-oxide metaboli..." /></meta><meta name="Description" content="FMO enzymes (FMOs) play a key role in the processes of detoxification and/or bioactivation of specific pharmaceuticals and xenobiotics bearing nucleophilic centers. The N-oxide and S-oxide metaboli..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 15, 2014" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm5007098" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2014 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm5007098" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm5007098" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm5007098" /></link>
        
    
    

<title>Flavin Monooxygenase Metabolism: Why Medicinal Chemists Should Matter | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm5007098" /></meta><meta property="og:title" content="Flavin Monooxygenase Metabolism: Why Medicinal Chemists Should Matter" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0017.jpeg" /></meta><meta property="og:description" content="FMO enzymes (FMOs) play a key role in the processes of detoxification and/or bioactivation of specific pharmaceuticals and xenobiotics bearing nucleophilic centers. The N-oxide and S-oxide metabolites produced by FMOs are often active metabolites. The FMOs are more active than cytochromes in the brain and work in tandem with CYP3A4 in the liver. FMOs might reduce the risk of phospholipidosis of CAD-like drugs, although some FMOs metabolites seem to be neurotoxic and hepatotoxic. However, in silico methods for FMO metabolism prediction are not yet available. This paper reports, for the first time, a substrate-specificity and catalytic-activity model for FMO3, the most relevant isoform of the FMOs in humans. The application of this model to a series of compounds with unknown FMO metabolism is also reported. The model has also been very useful to design compounds with optimal clearance and in finding erroneous literature data, particularly cases in which substances have been reported to be FMO3 substrates when, in reality, the experimentally validated in silico model correctly predicts that they are not." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm5007098"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm5007098">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm5007098&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm5007098&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm5007098&amp;href=/doi/10.1021/jm5007098" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2014</span><span class="cit-fg-volume">, 57</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6183-6196</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/57/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm500707r" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm500716s" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Flavin Monooxygenase Metabolism: Why Medicinal Chemists Should Matter</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gabriele++Cruciani">Gabriele Cruciani</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aurora++Valeri">Aurora Valeri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Goracci">Laura Goracci</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Roberto+Maria++Pellegrino">Roberto Maria Pellegrino</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Federica++Buonerba">Federica Buonerba</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Massimo++Baroni">Massimo Baroni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">Laboratory for Chemoinformatics and Molecular Modelling, Department of Chemistry, Biology and Biotechnology, University of Perugia, Via Elce di Sotto 8, 06123 Perugia, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¡</span> <span class="aff-text">Department of Chemistry, Biology and Biotechnology, University of Perugia, Via Elce di Sotto 8, 06123 Perugia, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">Â§</span> <span class="aff-text">Molecular Discovery, 215 Marsh Road, Pinner, HA55NE, London, U.K.</span></div><div class="corresp-info"><strong>*</strong>G.C.: phone, +39 075 585 5629; fax, +39 075 45646; e-mail, <a href="/cdn-cgi/l/email-protection#0a6d6b6878636f666f2469787f69636b64634a7f64637a6d24637e"><span class="__cf_email__" data-cfemail="7e191f1c0c171b121b501d0c0b1d171f10173e0b10170e1950170a">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm5007098&amp;href=/doi/10.1021%2Fjm5007098" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2014</span></span><span class="cit-volume">, 57</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6183â6196</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 8, 2014</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 May 2014</li><li><span class="item_label"><b>Published</b> online</span>15 July 2014</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 July 2014</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm5007098" title="DOI URL">https://doi.org/10.1021/jm5007098</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2014 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6183%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGabriele%2BCruciani%252C%2BAurora%2BValeri%252C%2BLaura%2BGoracci%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D57%26issueNum%3D14%26contentID%3Djm5007098%26title%3DFlavin%2BMonooxygenase%2BMetabolism%253A%2BWhy%2BMedicinal%2BChemists%2BShould%2BMatter%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6196%26publicationDate%3DJuly%2B2014">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm5007098"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2339</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">31</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm5007098" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Flavin Monooxygenase Metabolism: Why Medicinal Chemists Should Matter&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Gabriele&quot;,&quot;last_name&quot;:&quot;Cruciani&quot;},{&quot;first_name&quot;:&quot;Aurora&quot;,&quot;last_name&quot;:&quot;Valeri&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Goracci&quot;},{&quot;first_name&quot;:&quot;Roberto&quot;,&quot;last_name&quot;:&quot;Maria Pellegrino&quot;},{&quot;first_name&quot;:&quot;Federica&quot;,&quot;last_name&quot;:&quot;Buonerba&quot;},{&quot;first_name&quot;:&quot;Massimo&quot;,&quot;last_name&quot;:&quot;Baroni&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2014&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;57&quot;,&quot;pages&quot;:&quot;6183-6196&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm5007098&quot;},&quot;abstract&quot;:&quot;FMO enzymes (FMOs) play a key role in the processes of detoxification and/or bioactivation of specific pharmaceuticals and xenobiotics bearing nucleophilic centers. The N-oxide and S-oxide metabolites produced by FMOs are often active metabolites. The FMOs are more active than cytochromes in the brain and work in tandem with CYP3A4 in the liver. FMOs might reduce the risk of phospholipidosis of CAD-like drugs, although some FMOs metabolites seem to be neurotoxic and hepatotoxic. However, in silico methods for FMO metabolism prediction are not yet available. This paper reports, for the first time, a substrate-specificity and catalytic-activity model for FMO3, the most relevant isoform of the FMOs in humans. The application of this model to a series of compounds with unknown FMO metabolism is also reported. The model has also been very useful to design compounds with optimal clearance and in finding erroneous literature data, particularly cases in which substances have been reported to be FMO3 substrates when, &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5007098&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5007098" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5007098&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5007098" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5007098&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5007098" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm5007098&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5007098&amp;href=/doi/10.1021/jm5007098" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm5007098" /></input><a href="/doi/pdf/10.1021/jm5007098" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm5007098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm5007098%26sid%3Dliteratum%253Aachs%26pmid%3D25003501%26genre%3Darticle%26aulast%3DCruciani%26date%3D2014%26atitle%3DFlavin%2BMonooxygenase%2BMetabolism%253A%2BWhy%2BMedicinal%2BChemists%2BShould%2BMatter%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D57%26issue%3D14%26spage%3D6183%26epage%3D6196%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292143" title="Metabolism">Metabolism</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292541" title="Cavities">Cavities</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/57/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jmcmar.2014.57.issue-14/production/jmcmar.2014.57.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">FMO enzymes (FMOs) play a key role in the processes of detoxification and/or bioactivation of specific pharmaceuticals and xenobiotics bearing nucleophilic centers. The <i>N</i>-oxide and <i>S</i>-oxide metabolites produced by FMOs are often active metabolites. The FMOs are more active than cytochromes in the brain and work in tandem with CYP3A4 in the liver. FMOs might reduce the risk of phospholipidosis of CAD-like drugs, although some FMOs metabolites seem to be neurotoxic and hepatotoxic. However, in silico methods for FMO metabolism prediction are not yet available. This paper reports, for the first time, a substrate-specificity and catalytic-activity model for FMO3, the most relevant isoform of the FMOs in humans. The application of this model to a series of compounds with unknown FMO metabolism is also reported. The model has also been very useful to design compounds with optimal clearance and in finding erroneous literature data, particularly cases in which substances have been reported to be FMO3 substrates when, in reality, the experimentally validated in silico model correctly predicts that they are not.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24472" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24472" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Flavin containing monooxygenases (FMOs) are an important component in the arsenal of enzymes that metabolize drugs, as they complement the work performed on xenobiotics by the P450 enzymes. While the P450 enzymes tend to primarily oxidize molecules through electrophilic reactions, principally through radical intermediates, the FMO enzymes tend to oxidize the same molecules through nucleophilic addition reactions.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Discovered in the 1960s,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> this system of hepatic microsomal enzymes uses NADPH and oxygen to convert molecules containing primarily nucleophilic centers, in particular nitrogen, sulfur, phosphorus, and selenium atoms, all of which have a free lone pair of electrons, into the corresponding oxides. Besides oxidizing the same atoms, the cytochromes P450 directly oxidize carbon atoms readily. This implies that the FMO enzymes metabolize a more restricted range of compounds than the cytochromes P450. Recently, Testa et al.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> have shown that the FMOs are responsible for about 2.5% of all metabolic reactions and therefore about 6% of all the phase I metabolic reactions. Since FMO3 is the isoform of FMO that is most dominant in the liver of human adults, almost all the reactions of pharmaceutical interest are mediated by this enzyme. FMO3 is the major representative of the FMO family, just as CYP3A4 is the major representative of the cytochrome family.</div><div class="NLM_p">The experimental work being presented here aimed at proving that the importance of the FMOs, and in particular that of FMO3, remains underestimated. Many oxidation reactions that in the past have been attributed to CYP3A4 are in reality performed solely by, or in part by, FMO3. We believe that the FMOs are more frequently responsible for a number of these reactions than is currently attributed to them and that the role played by the principal isoform FMO3 is second only to the few cytochrome P450s considered to be the most important. As an example, the metabolism of a few selected compounds will be discussed. Arbidol, an antiviral agent, is a CYP3A4 substrate. It has been shown recently that this drug first reacts with FMO3 (S-oxidation) to give the sulfoxide metabolite and that the resulting metabolite undergoes subsequent N-demethylation by CYP3A4.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Benzydamine is an anti-inflammatory agent whose elimination pathway in the human body can be primarily attributed to FMO3.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Moclobemide is a reversible MAO inhibitor of which the <i>N</i>-oxide, formed through FMO3 catalysis, is the primary metabolite.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Itopride, a gastroprokinetic agent, has a metabolic clearance that is practically all caused by FMO3.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The <i>N</i>-oxide ranitidine metabolite, which blocks histamine H<sub>2</sub> receptors, is formed exclusively by FMO3.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Other examples in which the catalysis of FMO enzymes (including FMO3) is important, though not necessarily dominant in the overall clearance, are olanzapine,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> pargyline,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> xanomeline,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> zimelidine,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and many others. However, limited information about other drugs is provided in the literature. Imipramine has not been considered to be a substrate of FMO3,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> even though other sources classify it as an optimal substrate.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Preliminary studies indicated that diphenhydramine, a known substrate of CYP2D6,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> is actually a better substrate of FMO3 for N-oxidation, while CYP2D6 (and in a minor extent CYP3A4) mainly generates the N-dealkylated metabolite (see Figure S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). In the past, diphenhydramine was only reported to be a substrate for porcine FMO1 to give the <i>N</i>-oxide metabolite.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> In addition to diphenhydramine, ziprasidone, besides being metabolized by CYP3A4, also proved to be an optimal substrate for FMO3 (Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). It must be noticed that among the drugs reported above, ranitidine, zimelidine, olanzapine, imipramine, and ziprasidone are phospholipidosis (PLD) inducers,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> although for ziprasidone and olanzapine medium confidence data are available.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Recently the importance of metabolism in PLD induction has been reported,<a onclick="showRef(event, 'ref19 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref21 ref22">(19, 21, 22)</a> although metabolism by FMO3 has not been related to phospholipidosis effect so far.</div><div class="NLM_p">As the next paragraphs will demonstrate, it is not simple to understand whether a xenobiotic is a substrate of the FMOs or not, especially for FMO3. It is also difficult to predict the site of oxidation and any potential competition with CYP3A4. In silico methods can be extremely useful in helping the researcher formulate hypothesis and explain/interpret experiments of metabolite identification.</div><div class="NLM_p last">While in silico models that reproduce in part the behavior of human cytochromes are currently available, nothing has been developed so far for substrate specificity prediction and site of oxidation prediction for xenobiotics that interact with FMO. This work reports, for the first time, a simulation model that is capable of predicting the substrate specificity and the site of metabolism for the isoform that is most relevant to medicinal chemistry: the FMO3 isoform. Some examples of the applications of this simulation model and tests of its predictive ability in compounds of which the FMO metabolism has never been reported are also presented.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> FMO3 Substrate Selectivity and SoM</h3><div class="NLM_p">To a casual observer the task of deciding which metabolic reactions the FMO enzymes can perform on a substrate might seem trivial. The real situation is obviously much more complex than simple appearances. Zotepine (<b>1</b>) (see Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) is known to be an FMO3 substrate.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Molecules <b>2</b>â<b>4</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A are structurally similar to zotepine, as shown from alignment in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B. However, metabolism studies performed using recombinant human FMO3 (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details, and see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for LC/MS data) showed that diphenhydramine (<b>2</b>), like zotepine (<b>1</b>), proved to be an optimal FMO3 substrate, reacting to form the <i>N</i>-oxide, while the N-dealkylated metabolite was formed upon incubation with recombinant CYP2D6 isoform (Figure S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). On the other hand, cyamemazine (<b>3</b>) and bepridil (<b>4</b>) did not produce any metabolite at the experimental conditions used, with cyamemazine being a weak substrate only when tested at higher concentration. Since the same reactive group is present in all the molecules, the reason for the observed difference should not be found in the reactivity of these compounds but rather in their diverse exposition of the same functional group or complete lack of interaction with the enzyme. The fact that these molecules are very similar to each other even from a three-dimensional point of view demonstrates that the FMO3âsubstrate interaction is very specific and very sensitive to small changes in the three-dimensional structure of the substrate.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structures of zotepine (<b>1</b>), diphenidramine (<b>2</b>), cyamemazine (<b>3</b>), and bepridil (<b>4</b>). Compound <b>1</b> is a known FMO3 substrate.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compound <b>2</b>, incubated in recombinant FMO3, is an FMO3 substrate, while <b>3</b> and <b>4</b> proved to be nonsubstrates. (B) Superimposition of the chemical structures of <b>1</b>â<b>4</b>. Their 3D structures are very similar, with a good alignment of the tertiary amine nitrogen atoms and of the aromatic moieties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The same situation arises in small molecules, such as arecoline (<b>5</b>) and alvameline (<b>6</b>), with the latter developed as a bioisoster of the former muscarinic cholinergic compound (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Arecoline is an optimal substrate of FMO3,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> while the alvameline is not.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Arecoline (<b>5</b>) and its bioisoster alvameline (<b>6</b>). The former is an optimal substrate of FMO3, while the latter is not.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another case is presented by phenethylamine (<b>7</b>), which is a good FMO3 substrate,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> while the very similar tetrahydroisoquinoline (<b>8</b>) was only a very weak substrate (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), according to the experimental findings (LC/MS metabolism data for all compounds proved to be substrates for FMO3 in this section are reported in Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). The GRID force field<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> shows that the hydroperoxyflavin group in phenethylamine can produce a favorable electrophilic reaction in a much larger volume than tetrahydroisoquinoline, and therefore the geometric control is less tight.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Phenethylamine (A), a FMO3 substrate, compared with tetrahydroisoquinoline (B), which is not a FMO3 substrate. The GRID MIF shows favorable energetic interactions with hydroperoxyflavin group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">These last two cases show that FMO3, unlike CYP3A4 that performs the same reactions, is selective even with small molecules. It is evident that the productive spatial interaction with the enzyme (favorable enzyme recognition) plays a key role and is probably more important than chemical reactivity. From all these examples it emerges that it is not at all simple to predict whether a compound is a substrate of FMO3 or not. Obviously, if this information was available in literature, predicting the site of metabolism (SoM) might seem a simple task. Since the FMO reactions are more limited than those of P450, it is definitely easier to predict the SoM for the FMOs than for the P450s. Even so, a superficial treatment will not suffice, since the correct prediction might be more difficult than previously thought. On the basis of these considerations, a number of compounds possessing two or three potential sites of oxidation were selected for testing. Compound <b>9</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> has three potential centers for N-oxidation. The most nucleophilic center, which is also the most potentially reactive one, is represented by the imine nitrogen. However, we found that the reaction happens exclusively in the nitrogen atom of the terminal tertiary amine, even though this center is less nucleophilic, as demonstrated by the presence of the fragment with <i>m</i>/<i>z</i> 187.12 in the MSMS spectrum of the metabolite. Compound <b>10</b> has two potential centers for N-oxidation. The most nucleophilic center is the quinuclidine group, which is exactly where the only N-oxidation reaction happens, as the fragment with <i>m</i>/<i>z</i> 154.0 proves. Cimetidine (<b>11</b>) has two centers of similar nucleophilicity. The oxidation reaction happens at the sulfur atom, according to the MSMS fragments with <i>m</i>/<i>z</i> 95.06 and 99.06, among the others. As with the cytochrome P450s, even in this case the oxidation reaction happens as a combination of substrate reactivity (caused in this case by its nucleophilicity) and by the productive spatial interaction with the enzyme (favorable enzyme recognition). Evaluating these two factors separately is not simple. Furthermore, it is even more complicated to evaluate the importance that the enzyme attributes to either component.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Experimental sites of metabolism (oxidation) for compounds <b>9</b>â<b>11</b> are marked with asterisks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The importance of pure chemical reactivity can be described using two very similar compounds which will probably be exposed in the FMO3 catalytic site in a very similar manner. Compounds <b>12</b> and <b>13</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> were acquired and tested, both being FMO3 substrates. The former reacts at the sulfur atom of the methylsulfide and at the pyrrolidine nitrogen, while the latter reacts only at the methylsulfide moiety. This difference can be explained by the different nucleophilicity of the pyrrolidine and thiophene moieties. As reported later in <a class="ref internalNav" href="#sec4_7" aria-label="Computational Section">Computational Section</a>, the pyrrolidine moiety is a strong nucleophilic group, while the thiophene moiety is only very weakly nucleophilic. We may therefore conclude that compounds will be hardly metabolized by FMO3 enzyme in thiophene groups, especially when better nucleophilic moieties are present in the molecules.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds <b>12</b> (A) and <b>13</b> (B) are similar FMO3 substrates that react in a different manner. The former reacts at the sulfur atom of the methylsulfide and at the pyrrolidine nitrogen, while the latter reacts only at the methylsulfide moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, as reported above, cases in which FMO3 generates more than one reaction are not rare. In such cases, the prediction of the fastest, or most important reaction, is made even more difficult. Would the reader be able to predict the site of oxidation responsible for the fastest generation of the metabolite by FMO3 for the molecules in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>?</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Predicting the correct rank for sites of metabolism (SoM) in compounds <b>14</b>â<b>16</b> by FMO3 is not a simple task.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> FMO3: 3D Structure</h3><div class="NLM_p">FMO3 is the dominant form of FMO in the liver of adult humans. It possesses ample substrate specificity, and it is also the most selective FMO enzyme. Unfortunately, no structural information at the atomic resolution is available for any microsomal form of FMO enzymes. Nevertheless, high resolution crystalline structures have been published for yeast and for some bacterial isoforms of this enzyme, as shown in the Protein Data Bank (PDB). These isoforms are normally found to be soluble in the cytosol, utilize NADP(H) as a cofactor, and contain conserved sequences for cofactorâprotein binding. These soluble enzymes provide a reasonable base from which homology models for the human microsomal FMO enzymes can be produced.</div><div class="NLM_p">The model presented in the <a class="ref internalNav" href="#sec4_7" aria-label="Computational Section">Computational Section</a> of this work suggests that FMOs possess two distinctive domains with FAD and NADP(H) bound close to the interface. From their position in the structure, the nicotinamide ring and the ring adjacent to the ribose in NADP(+) are an integral part of the catalytic site, being actively employed in the stabilization of the oxygenated intermediate. This characteristic suggests that NADP(H) has a fundamental role because of the two binding modes, which allow it to function in the modulation of both reduction and oxidation.</div><div class="NLM_p">The catalytic cycle demonstrates that the flavoneâhydroperoxide (FAD-OOH) species transfers an oxygen atom to the substrate in the first step of the process. The reaction mechanism is concerted; that is, NADP(H), oxygen, and the oxidable substrate are added to the enzyme before any of the products are released. The release of water and/or NADP(+) is considered to be the limiting step. The flavoneâhydroperoxide (FAD-OOH) is a relatively stable species, though with a low oxidative potential. For the oxidation of the FMO substrates to happen, the substrates themselves must be sufficiently nucleophilic and must be able to interact with the enzyme with the right geometry. As with the cytochromes,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> even in this case the oxidation reaction happens by a combination of substrate reactivity (described in this case by its nucleophilicity) and by the productive spatial interaction with the enzyme (favorable enzyme recognition).</div><div class="NLM_p">Unlike the cytochromes, which exhibit a notable preference in the selection of their substrates (positively charged in the case of CYP2D6, negatively charged in the case of CYP2C9, neutral in the case of CYP3A4), the FMOs do not demonstrate a charge preference in their substrates, as shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, which lists some noteworthy substrates of FMO3. It can be noted that neutral, positively charged, and negatively charged substrates are reported.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chemical Structures of FMO3 Substrates<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0016.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">It can be noted that the substrates are very diverse, and they show different charge (positive (A), negative (B), zwitterionic (C), neutral (D)), size, shape, hydrophobicity, and polarity. The structures correspond to the predominant species at pH 8.0 (FMO3 works in basic conditions). The computation of the predominant species was carried out using MoKa.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a></p></div></div><div></div></div><div class="NLM_p">Even though little is known on how the enzyme controls the access of the substrate to the active site, some hints have emerged from the X-ray structure of the bacterial and yeast forms concerning possible mechanisms for stabilization of the hydroperoxide group in the active site. The FAD group interacts with an asparagine residue (Asn78 in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7">2vq7</a>, conserved in all FMOs). It has been suggested that Asn78 stabilizes C4Î±-hydroperoxide through polar interactions.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> It is interesting to note that a polymorphic mutation to this residue in the human form of FMO3 (N to S/K) causes complete loss of function.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Another contribution to the stability of the hydroperoxide could be the NADP(+) bound in the active site. It has been hypothesized that this group shields the hydroperoxide group from interacting with the solvent.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Molecular dynamics simulations on a prokaryotic FMO protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7">2vq7</a>; see computational details) show that the active complex proteinâNADâFAD is stable enough and that the active site of the enzyme is not very flexible. The volume of the active site varies from a minimum of 1000 Ã to a maximum of 1200 Ã, with an entrance of relatively modest size. Similar findings were obtained when a model for the human protein was used (see computational details in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). This implies that only small to medium-sized substrates may enter. The catalytic site of the enzyme and the amino acids that play an important role in substrate recognition are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Bacterial (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7">2vq7</a> structure, A) and human (calculated structure, B) catalytic sites of FMO3. Amino acids that play an important role in substrate orientation are reported. Cavities are computed by FLAP software.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Calculation of the FMO3âSubstrate Interaction</h3><div class="NLM_p">The function describing the recognition between the FMO3 enzyme and the substrate is based on the FLAP algorithm,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> which has been designed to anchor a flexible substrate in a flexible protein in an extremely rapid manner. The function does not calculate the thermodynamics of the process but rather considers the three-dimensional compatibility between the molecular interaction fields (MIFs) of the protein and those created by the substrate.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Since the flexibility of both the substrate and FMO3 is of fundamental importance, the method used has been designed in such a way to reproduce poses that are similar to experimentally determined poses in a precise and rapid manner. In fact, FLAP is capable of evaluating tens of thousands of poses in a fraction of a second. Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>A shows the flow diagram of the algorithm. The shape of the protein cavity is precisely evaluated through the use of the GRID force field<a onclick="showRef(event, 'ref45 ref27'); return false;" href="javascript:void(0);" class="ref ref45 ref27">(45, 27)</a> in a dynamic manner, which is considering the enzyme as a flexible object in such a way that the shape of the cavity can be modified or chosen by the substrate itself. This is similar in concept to having more than one cavity being made available by the single FMO3 enzyme.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Metasite flowchart for FMO3 SoM calculation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Simultaneously, each substrate is modeled in different conformations, all of which are used to generate all the possible energetically favorable poses of the substrate inside the cavity. An energetically favorable enzymeâsubstrate interaction is obtained when a molecule in a defined conformation possesses many atoms that can be superimposed onto the MIFs of the enzyme (refer to Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>B). The function evaluates the degree of superposition of all the poses. At the end of the process, the most probable pose (the one showing the best superposition) will be evident from the comparison of all the poses inside all the possible cavities. Even so, it is not only the single best pose in terms of enzymeâsubstrate fields similarity that is chosen, but a given number of optimal poses are memorized and analyzed. Indeed, it might often happen that the best exposition of the reactive center is not for a very reactive atom, which could lead to a slow reaction. Better kinetics could be obtained for a pose with a lower similarity score, which however exposes a more reactive atom in the active site.</div><div class="NLM_p last">The study of all the optimal poses (refer to the computational details) is also necessary in order to predict whether a compound can be considered to be a substrate of FMO3 or not. From the analysis of the data obtained from a collection of 95 FMO3 substrates and 50 nonsubstrates, two important facts have emerged. First, in order for a xenobiotic compound to be considered a substrate of FMO3, it must possess at least one group with a nucleophilic value that is higher than a defined threshold, as reported in the next paragraph. Second, the substrate must also be able to form at least one geometrically efficient interaction with the enzyme. This interaction is evaluated through the similarity of the interaction fields and must be assigned a value that is equal to or higher than a defined threshold. Obviously if a large number of virtually reactive poses are found, that compound will have a higher probability of being a substrate of FMO3.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Calculation of the Nucleophilicity of the Substrate</h3><div class="NLM_p">A nucleophile is an atom, ion, or molecule that has an electron pair that may be donated in forming a covalent bond to an electrophile (or Lewis acid). Polar and protic solvents, such as water, solvate anions by hydrogen bonding interactions. The solvated species are more stable and less reactive than the unsolvated ânakedâ anions. Therefore, nucleophilicity in water depends on the charge, polarizability, solvation, and HOMO energy of a chemical moiety. A set of 50 chemical compounds for which the experimentally measured nucleophilicity value (in water) is available<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> was used to correlate the experimental nucleophilicity value with a number of in silico calculated parameters, such as the nucleophile-atom charge, solvation, polarizability, and HOMO energy. PLS<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> was used to produce a chemometric model linking the in silico parameters to the experimental nucleophilicity values. The experimental nucleophile-reactivity parameters in water solvent are listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, together with the recalculated parameters obtained from the PLS model. The same model was later used to calculate the nucleophilicity in water of many different nucleophilic groups that are frequently used in medicinal chemistry for which the experimental values are not yet available. In this way it was possible to construct a nucleophile-reactivity scale for more than 115 small synthesizable compounds, all of which are potential substrates of FMO. When other factors are constant, this scale can be used to estimate the likelihood that a chemical group suffers an electrophilic attack by FMO enzymes. Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a> shows the model that relates the experimental and the recalculated nucleophile-reactivity values for all the data points used in the training set and for the most diverse data points predicted in the test set.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Experimental vs recalculated nucleophilicity (in water) for 50 chemical entities from Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> (in blue rhombus). The remaining 65 compounds (red triangles) lack experimental value and are predicted from nucleophilicity model (see methods section). The nucleophilicity range is arbitrarily divided into four regions (thresholds), from not reactive (when it is very low) to low, medium, and high nucleophilicity values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Nucleophile-Reactivity Parameter in Water Solvent<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">exptl<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">calcd</th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">exptl<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">calcd</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1,2-ethanediamine</td><td class="colsep0 rowsep0" align="char" char=".">13.3</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td><td class="colsep0 rowsep0" align="left">glycineamide</td><td class="colsep0 rowsep0" align="char" char=".">12.3</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1,3-diaminopropane</td><td class="colsep0 rowsep0" align="char" char=".">14.0</td><td class="colsep0 rowsep0" align="char" char=".">14.4</td><td class="colsep0 rowsep0" align="left">glycinenitrile</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="char" char=".">10.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1-methyluracil (anion)</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">9.8</td><td class="colsep0 rowsep0" align="left">hydroxylamine</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="char" char=".">11.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2-formylimidazole (anion)</td><td class="colsep0 rowsep0" align="char" char=".">11.1</td><td class="colsep0 rowsep0" align="char" char=".">13.1</td><td class="colsep0 rowsep0" align="left">isopropylamine</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td><td class="colsep0 rowsep0" align="char" char=".">13.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2-pyridone (anion)</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="char" char=".">11.9</td><td class="colsep0 rowsep0" align="left">methyl-2-aminoacetate</td><td class="colsep0 rowsep0" align="char" char=".">12.1</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-methylpyridine</td><td class="colsep0 rowsep0" align="char" char=".">11.1</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="left">methylamine</td><td class="colsep0 rowsep0" align="char" char=".">13.8</td><td class="colsep0 rowsep0" align="char" char=".">13.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-(dimethylamino)pyridine</td><td class="colsep0 rowsep0" align="char" char=".">13.2</td><td class="colsep0 rowsep0" align="char" char=".">14.3</td><td class="colsep0 rowsep0" align="left">methylhydrazine</td><td class="colsep0 rowsep0" align="char" char=".">17.2</td><td class="colsep0 rowsep0" align="char" char=".">16.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-aminopyridine</td><td class="colsep0 rowsep0" align="char" char=".">12.2</td><td class="colsep0 rowsep0" align="char" char=".">12.2</td><td class="colsep0 rowsep0" align="left">morpholine</td><td class="colsep0 rowsep0" align="char" char=".">15.6</td><td class="colsep0 rowsep0" align="char" char=".">15.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-chloropyridine</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="left"><i>n</i>-buthylamine</td><td class="colsep0 rowsep0" align="char" char=".">13.1</td><td class="colsep0 rowsep0" align="char" char=".">13.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-methoxyaniline</td><td class="colsep0 rowsep0" align="char" char=".">16.5</td><td class="colsep0 rowsep0" align="char" char=".">15.8</td><td class="colsep0 rowsep0" align="left"><i>N</i>-methylglycinenitrile</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td><td class="colsep0 rowsep0" align="char" char=".">13.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-methoxypyridine</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="char" char=".">12.2</td><td class="colsep0 rowsep0" align="left"><i>N</i>-methylmorpholine</td><td class="colsep0 rowsep0" align="char" char=".">16.4</td><td class="colsep0 rowsep0" align="char" char=".">16.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-nitroimidazole (anion)</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="char" char=".">11.5</td><td class="colsep0 rowsep0" align="left"><i>N</i>-methylpropargylamine</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-pyrrolidinopiridine</td><td class="colsep0 rowsep0" align="char" char=".">13.4</td><td class="colsep0 rowsep0" align="char" char=".">15.5</td><td class="colsep0 rowsep0" align="left"><i>n</i>-propylamine</td><td class="colsep0 rowsep0" align="char" char=".">13.3</td><td class="colsep0 rowsep0" align="char" char=".">13.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4-pyridone (anion)</td><td class="colsep0 rowsep0" align="char" char=".">14.8</td><td class="colsep0 rowsep0" align="char" char=".">15.1</td><td class="colsep0 rowsep0" align="left">perhydroazepine</td><td class="colsep0 rowsep0" align="char" char=".">18.3</td><td class="colsep0 rowsep0" align="char" char=".">16.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9-methylguanine (anion)</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="char" char=".">10.2</td><td class="colsep0 rowsep0" align="left">piperazine</td><td class="colsep0 rowsep0" align="char" char=".">17.2</td><td class="colsep0 rowsep0" align="char" char=".">17.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">adenine (anion)</td><td class="colsep0 rowsep0" align="char" char=".">10.4</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="left">piperidine</td><td class="colsep0 rowsep0" align="char" char=".">18.1</td><td class="colsep0 rowsep0" align="char" char=".">16.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allylamine</td><td class="colsep0 rowsep0" align="char" char=".">13.2</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="left">propargylamine</td><td class="colsep0 rowsep0" align="char" char=".">12.3</td><td class="colsep0 rowsep0" align="char" char=".">11.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ammonia</td><td class="colsep0 rowsep0" align="char" char=".">9.5</td><td class="colsep0 rowsep0" align="char" char=".">9.7</td><td class="colsep0 rowsep0" align="left"><i>p</i>-toluidine</td><td class="colsep0 rowsep0" align="char" char=".">13.0</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aniline</td><td class="colsep0 rowsep0" align="char" char=".">13.0</td><td class="colsep0 rowsep0" align="char" char=".">12.2</td><td class="colsep0 rowsep0" align="left">purine (anion)</td><td class="colsep0 rowsep0" align="char" char=".">11.0</td><td class="colsep0 rowsep0" align="char" char=".">9.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">benzotriazole (anion)</td><td class="colsep0 rowsep0" align="char" char=".">11.5</td><td class="colsep0 rowsep0" align="char" char=".">11.7</td><td class="colsep0 rowsep0" align="left">pyridine</td><td class="colsep0 rowsep0" align="char" char=".">11.1</td><td class="colsep0 rowsep0" align="char" char=".">11.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">benzylamine</td><td class="colsep0 rowsep0" align="char" char=".">13.4</td><td class="colsep0 rowsep0" align="char" char=".">12.3</td><td class="colsep0 rowsep0" align="left">pyrrolidine</td><td class="colsep0 rowsep0" align="char" char=".">17.2</td><td class="colsep0 rowsep0" align="char" char=".">15.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">diethylamine</td><td class="colsep0 rowsep0" align="char" char=".">14.7</td><td class="colsep0 rowsep0" align="char" char=".">15.7</td><td class="colsep0 rowsep0" align="left">semicarbazide</td><td class="colsep0 rowsep0" align="char" char=".">11.0</td><td class="colsep0 rowsep0" align="char" char=".">10.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dimethylamine</td><td class="colsep0 rowsep0" align="char" char=".">17.1</td><td class="colsep0 rowsep0" align="char" char=".">15.6</td><td class="colsep0 rowsep0" align="left"><i>tert</i>-butylamine</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="char" char=".">13.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ethanolamine</td><td class="colsep0 rowsep0" align="char" char=".">12.6</td><td class="colsep0 rowsep0" align="char" char=".">13.3</td><td class="colsep0 rowsep0" align="left">theophylline (anion)</td><td class="colsep0 rowsep0" align="char" char=".">10.1</td><td class="colsep0 rowsep0" align="char" char=".">10.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ethylamine</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td><td class="colsep0 rowsep0" align="left">thymine (anion)</td><td class="colsep0 rowsep0" align="char" char=".">11.7</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">Experimental (exptl) and calculated (calcd) values are reported.</p></div><div class="footnote" id="t2fn1"><sup>b</sup><p class="last">From ref <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">46</a>.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Calculation of the SoM for FMO3</h3><div class="NLM_p">As with the cytochromes,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> even in this case the oxidation reaction happens by a combination of substrate reactivity (described in this case by its nucleophilicity) and by the productive spatial interaction with the enzyme (favorable enzyme recognition).</div><div class="NLM_p">Once the enzyme-accessibility and nucleophilic-reactivity components are calculated, the site of metabolism is described by a probability function <i>P</i><sub>SoM</sub> (probability for the site of metabolism), which is correlated to and can be considered to be an approximation of the free energy of the overall process.<a onclick="showRef(event, 'ref48 ref28'); return false;" href="javascript:void(0);" class="ref ref48 ref28">(48, 28)</a> The two factors that contribute to the creation of the <i>P</i><sub>SoM</sub> function in the original equation (recognition and reactivity) were equally weighed, though in reality their respective contributions very much depend on both the cytochrome/FMO being used and the substrate molecule.</div><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a> illustrates an example of how this may happen when using different substrates with the same FMO enzyme. In âcase Aâ, a substrate with a low molecular weight (methimazole) interacts with the FMO3 catalytic site. The substrate is smaller than the enzyme cavity and can interact with the reactive oxygen atom in many different ways. In this case, recognition is less important than the reactivity of the various fragments, and so the most reactive fragment is the most likely to be metabolized. âCase Bâ (with compound darexaban from Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) is an opposite example.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0010.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Reactivity equalization. (A) Small substrates can interact with FAD in different orientations, exposure constraints being practically absent. When two or more oxidation sites are possible, their nucleophilicity values will determine the oxidation rate and the metaboliteâs abundance. (B) Large substrates can be strongly constrained because of several interactions with the cavity. In this case exposure, rather than nucleophilicity, plays a relevant role in determining the site of metabolism.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Here, the volume of the substrate is almost as large as that of the cavity itself. This is not so frequent for FMO but is common for P450 enzymes. Recognition is very important here because of the many different interactions that the substrate might have with the cavity. The reactivity of the fragments in this case is of minor importance when computing the SoM and only serves as a âcorrectiveâ role.</div><div class="NLM_p last">This example demonstrates that calculating the SoM using fixed weights for recognition and reactivity is conceptually wrong. It would be more correct to use weight coefficients (<i>w</i><sub>e</sub> and <i>w</i><sub>r</sub> in eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a>) that are based on the type of the enzyme cavity and the structure of the substrate. This feature in MetaSite5 is called reactivity equalization.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The use of this function allows the automatic recalculation of the weights in eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a> to consider the enzyme cavity (which is determined by the type of CYP/FMO being used) and the interactions of the substrate with the amino acids within this cavity. One result of the reactivity-equalization function is that the weight coefficient of the reactivity component of the equation will be increased for small to medium substrates in large enzymes (CYP3A4) and decreased for medium to large molecules in enzymes with small cavities (i.e., CYP2E1, FMO3).<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/jm-2014-007098_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> MetaSite Predictions</h3><div class="NLM_p">Once the reactivity and recognition models were introduced into MetaSite, this procedure was used to predict the isoform selectivity and the site of metabolism for a number of known and unknown xenobiotics, the experimental data of which have been produced in house. It was noticed that the prediction of isoform selectivity for voriconazole (<b>17</b>), sulfamethoxazole (<b>18</b>), dapsone (<b>19</b>), and imipramine (<b>20</b>) (Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a>) did not coincide with literature data.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0011.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structures of voriconazole (<b>17</b>), sulfamethoxazole (<b>18</b>), dapsone (<b>19</b>), and imipramine (<b>20</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The first three of these compounds are given as FMO3 substrates,<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49, 50)</a> while the last compound is an uncertain case because it has been reported to be both a substrate and a nonsubstrate.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Nevertheless, experiments performed using recombinant human FMO3 proved without a doubt that the predictions are correct, with the first three compounds being nonsubstrates of FMO3 and with imipramine being a good substrate. Regarding the compounds listed in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, MetaSite correctly predicts them as FMO3 substrates. However, while the first one is a good substrate (Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a>), compounds <b>15</b> and <b>16</b> are metabolized in less amount with the experimental conditions used (Figure S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). The site of metabolism prediction for the three substrates in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a> is optimal. For these compounds, not only does MetaSite correctly predict the oxidized groups but it also correctly predicts their ranking (see Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a> and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0012.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Comparison between predicted and experimental SoM for compounds <b>14</b>. (A) MetaSite SoM prediction for compound <b>14</b>. The highest oxidation probability (marked with 1) corresponds to the fastest N-oxidation reaction. S-oxidation is predicted to be the second most probable reaction (marked with 2). (B) In vitro metabolism study for compound <b>14</b>. The N-oxidized metabolite at the benzylamine position and the S-oxidized metabolites are formed. (C) Clearance determination for compound <b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An even more interesting situation was the prediction of bepridil, of which no information concerning FMO3 metabolism was available. This drug was predicted to be a nonsubstrate, and the subsequent incubation with FMO3 verified the correct nature of the prediction. Nevertheless, it is still possible to use MetaSite to obtain information on which groups need to be modified/eliminated/added to enhance or reduce the interaction of a substrate with FMO3. In practice, by forcing the exposition of the potentially reactive group to the C4Î±-hydroperoxide center, it is possible to obtain information on which part of the molecule needs to be modified to favor or disfavor a specific interaction. In the case of bepridil, the model proposed that if the oxidation of the pyrridoline nitrogen were required, the lateral isobutoxy chain would need to be eliminated to favor the correct interaction, since this chain interacts repulsively with the Phe461 amino acid residue of FMO3 cavity and does not allow the correct orientation of the molecule for the oxidation reaction to occur. (Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a>A).</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0013.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Our experimental data show that bepridil is not an FMO3 substrate. MetaSite reports that this may be due to steric clash (A) between the isopropyloxy chain with the enzyme cavity. When the chain was synthetically removed, the new compound in (B) was a good FMO3 substrate with SoM in the predicted pyrrolidil nitrogen.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The bepridil derivative called BPEA (benzyl[2-(1-pyrrolidinyl)ethyl]aniline) âsuggestedâ by the computational procedure was synthesized and tested. In agreement with the MetaSite prediction, this derivative was found to be a good substrate of FMO3, and it also reacted in the predicted position, that is, oxidation of the pyrridoline nitrogen (Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a>B). Kinetic behavior for compound BPEA is reported in Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a>.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0014.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. (A) BPEA depletion and BPEA <i>N</i>-oxide formation in human FMO3 enzyme. (B) Clearance data for BPEA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">BPEA (benzyl[2-(1-pyrrolidinyl)ethyl]aniline, <b>c</b>) was synthesized according to Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>. Nucleophilic replacement of bromide from 2-bromoethanol by pyrrolidine in refluxing acetonitrile, in the presence of potassium carbonate, afforded the amino alcohol <b>a</b> which was converted into the corresponding triflate <b>b</b> by reaction with triflic anhydride in dichloromethane. Finally, the target product <b>c</b> was obtained by reacting the alkyl triflate <b>b</b> with the sodium amide of <i>N</i>-benzylaniline in refluxing toluene.</div><figure id="sch1" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0015.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 82 Â°C; (ii) Tf<sub>2</sub>O, DCM, rt; (iii) <i>N</i>-benzylaniline, NaH, 60% dispersion in mineral oil, toluene, 110 Â°C.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Human FMO enzymes are a small family of highly active microsomal monooxygenases that primarily detoxify, and to a lesser extent bioactivate, xenobiotics containing nucleophilic centers. The literature search discussed in this paper revealed that the role of FMO enyzmes has been underestimated, especially in the case of FMO3, which is the most dominant isoform in the liver. The LC/MS studies performed on a selected number of compounds proved useful for an improved understanding of the nature of FMO3 substrates and of the enzyme selectivity. This study therefore suggests that medicinal chemists should consider FMO metabolism during pharmaceutical development as they currently do for the cytochrome P450 (CYP) enzymes. To help the medicinal chemists in this task, it follows that in silico tools are required to aid design that also allow the prediction of FMO metabolism. Therefore, the most important of these enzymes, FMO3, was modeled starting from a bacterial analogue, with the aim of obtaining predictions on substrate selectivity and site of metabolism for all potential xenobiotics that interact with human FMO3. The procedure, once developed and inserted into the MetaSite software, correctly predicted more than 90% of the substrates known today for FMO3 (86 out of 95) and about 85% of the known nonsubstrates (42 out of 50). A high fidelity was also observed when predicting the site of metabolism, when used in the presence of more than one potentially reactive groups. The SoM prediction for FMO3 in MetaSite has a 92% success rate (using top1 ranking scheme). It is interesting to note that in most cases in which the MetaSite prediction did not coincide with literature data (voriconazole, sulfamethoxazole, dapsone, imipramine) experiments demonstrated the correctness of the prediction. Moreover, MetaSite was able to guide the design of one FMO3 substrate, thus demonstrating that the procedure can be used to transform an FMO3 substrate into a nonsubstrate or vice versa and to eliminate or improve the clearance of potential substrates. We believe that computer aided drug metabolism elucidation (CADME), when applied to the isoform FMO3, can sensibly improve and hasten our understanding of the role of this enzyme in the clearance and eventual toxicity of the drug/xenobiotic compounds.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Materials</h3><div class="NLM_p last">Diphenydramine hydrochloride (<b>2</b>), cyamemazine (<b>3</b>), bepridil hydrochloride (<b>4</b>), tetrahydroisoquinoline (<b>8</b>), cimetidine (<b>11</b>), voriconazole (<b>17</b>), sulfamethoxazole (<b>18</b>), dapsone (<b>19</b>), and imipramine hydrochloride (<b>20</b>) were purchased from Sigma-Aldrich (St. Louis, MO). Compounds <b>9</b>, <b>10</b>, <b>12</b>â<b>17</b> were acquired from SPECS (<a href="http://www.specs.net" class="extLink">www.specs.net</a>). Human FMO3 supersomes (catalog number 456233), expressed in the baculovirus system, were purchased from BD Gentest (Woburn, MA).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Purities of the Acquired Compounds</h3><div class="NLM_p last">Purity of the acquired compounds was determined by UHPLC on Agilent Technologies 6540 UHD Accurate Mass Q-TOF LC/MS, 1290 Infinity Series with DAD detector and evaluated to be higher than 95%. Chromatographic conditions to assess the purity of acquired or final compounds were as follows: column, Phenomenex Aeris peptide 2.1 mm Ã 100 mm (1.7 Î¼m); flow rate, 0.3 mL/min; acquisition time, 20 min; DAD 190â650 nm; oven temperature, 45 Â°C; linear gradient of acetonitrile in water both containing 0.1% of formic acid (0â100% in 20 min).</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry. Materials and Instrumentation</h3><div class="NLM_p"><sup>1</sup>H and <sup>13</sup>C NMR nuclear magnetic resonance (NMR) spectra were recorded at 400 and 100.6 MHz, respectively, on Bruker Avance II 400 MHz spectrometer at room temperature. Samples have been dissolved in deuterochloroform. Chemical shifts (Î´) are given in parts per million (ppm) relative to the internal standard tetramethylsilane. Peak multiplicities are reported as s (singlet), d (doublet), dd (double doublet), t (triplet),dt (double triplet), q (quartet), p (pentet), sept (septet), hept (heptet), m (multiplet), or br s (broad singlet). Coupling constants (<i>J</i>) are given in Hz. HRMS spectra were registered on Agilent Technologies 6540 UHD Accurate Mass Q-TOF LC/MS system. Purity of the final compound was determined by UHPLC and was â¥98% pure.</div><div class="NLM_p last">2-Bromoethanol and pyrrolidine were commercial products (Aldrich) of the highest purity; they were used without further purification. Aniline (Aldrich) was freshly distilled at reduced pressure before use. <i>N</i>-Benzylaniline was prepared from benzaldehyde and aniline according to literature method.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Acetonitrile and toluene were distilled from P<sub>2</sub>O<sub>5</sub>. Air and moisture sensitive compounds were stored in Schlenk tubes or Schlenk burets. They were handled under an atmosphere of 99.995% pure nitrogen, using appropriate glassware.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthetic Procedures</h3><div class="NLM_p last">Synthesis of benzyl[2-(1-pyrrolidinyl)ethyl]aniline (BPEA) was performed according to Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>.</div></div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 2-(Pyrrolidin-1-yl)ethanol (<b>a</b>)<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></h3><div class="NLM_p last">2-Bromoethanol (3.5 mL, 50 mmol) in dry acetonitrile (25 mL) was added dropwise, under nitrogen atmosphere, to a refluxing mixture of pyrrolidine (4.4 mL, 53 mmol) and anhydrous powdered potassium carbonate (6.2 g, 45 mmol) in dry acetonitrile (75 mL). After 15 h, the mixture was allowed to cool to 25 Â°C, the solid was filtered off, washed with AcOEt, and the solvent was evaporated at reduced pressure. Flash chromatography on silica gel (eluent, 8:2 dichloromethane/methanol mixture) allowed collection of the expected product as a yellow oil (3.6 g, 31 mmol, 62% yield) exhibiting the following characteristics: <sup>1</sup>H NMR (CDCl<sub>3</sub>) Î´ 4.22 (br s, 1H), 3.66 (t, <i>J</i> = 1.6 Hz, 2H), 2.70 (t, <i>J</i> = 1.6 Hz, 2H), 2.64 (br s, 4H), 1.80 (br s, 4H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) Î´ 59.6, 57.8, 54.0 (2C), 23.4 (2C); HRMS calcd for C<sub>6</sub>H<sub>13</sub>NO 116,1075 (M + H<sup>+</sup>), found 116.1069 (M + H<sup>+</sup>).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 2-(Pyrrolidin-1-yl)ethyl Trifluoromethanesulfonate (<b>b</b>)</h3><div class="NLM_p last">Trifluoromethanesulfonic anhydride (0.16 mL, 0.96 mmol) was added to a solution of <b>a</b> (100 mg, 0.87 mmol) in dry DCM (1 mL) at â20 Â°C, under nitrogen atmosphere. After the cooling bath was removed, the mixture was stirred for 30 min at 25 Â°C. After solvent evaporation at reduced pressure, the residual brown oil was pure enough to be used in the following step without further purification.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>N</i>-Benzyl-<i>N</i>-(2-(pyrrolidin-1-yl)ethyl)aniline (BPEA, <b>c</b>)</h3><div class="NLM_p last"><i>N</i>-Benzylaniline (0.3 mL, 1.74 mmol) was added to a stirred suspension of NaH (83 mg, 3.48 mmol) in dry toluene (1 mL) under nitrogen atmosphere. After hydrogen generation ceased, a solution of aminotriflate <b>b</b> (0.87 mmol) in dry toluene (1 mL) was added and the reaction mixture was refluxed overnight. Upon cooling at room temperature, the reaction was quenched with MeOH and H<sub>2</sub>O, extracted with AcOEt, and the organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After the solvent was evaporated at reduced pressure, chromatography of the residue on silica gel (eluent 95:5 DCM/MeOH mixture) allowed recovery of an orange oil (50 mg, 21% yield) exhibiting the following spectroscopic characteristics: <sup>1</sup>H NMR (CDCl<sub>3</sub>) Î´ 7.24â7.21 (m, 2H), 7.18â7.09 (m, 5H), 6.67â6.60 (m, 3H), 4.49 (s, 2H), 3.56 (t, <i>J</i> = 8.0 Hz, 2H), 2.74 (t, <i>J</i> = 8.0 Hz, 2H), 2.60 (br s, 4H), 1.76 (br s, 4H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) Î´ 148.2, 138.7, 129.3 (2C), 128.6 (2C), 126.9, 126.6 (2C), 116.6, 112.4 (2C), 54.9, 54.5 (2C), 52.9, 49.7, 23.4 (2C); HRMS calcd for C<sub>19</sub>H<sub>24</sub>N<sub>2</sub> 281,2018 (M + H<sup>+</sup>), found 281.2017 (M + H<sup>+</sup>).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Drug Metabolism with Recombinant Human Flavin-Containing Monooxygenase FMO3</h3><div class="NLM_p last">Substrates were preincubated for 5 min at 37 Â°C in a 50 mM glycine buffer (pH 9.5) with NADPH (1 mM) in a total volume of 250 Î¼L. The reactions were initiated by addition of human FMO3 supersomes (100 Î¼g) in a shaking water bath at 37 Â°C. Preliminary studies were performed incubating each compound at 0 and 30 min. A volume of 250 Î¼L of cold acetonitrile (containing 0.6 Î¼M labetalol as an internal standard) was added to the mix to terminate the reaction. Proteins were precipitated by centrifugation at 12â000<i>g</i> for 5 min at 4 Â°C, and aliquots of the supernatants were analyzed by LCâMS/MS as described in the next paragraph. Compounds proved to be substrates for FMO3 were further investigated, repeating the assay with more incubation times (0, 5, 10, 20, and 30 min). Structural elucidation of the metabolites by MSMS data and Cl<sub>int</sub> evaluation were automatically performed by Mass-MetaSite software (Molecular Discovery).<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53, 54)</a> Theoretical and experimental molecular ion mass, elemental composition, mass error, characteristic fragments, and retention times <i>t</i><sub>R</sub> for substrates and metabolites are reported as <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. To determine the potential metabolic products formed by the FMO3 enzyme for voriconazole, sulfamethoxazole, and dapsone, which were previously reported to be FMO3 substrates, FMO3 in different concentrations (100, 500 Î¼g) was incubated with the individual substrates (5, 50, 100 Î¼M concentrations) in the presence of NADPH (2 mM) for 5, 10, 20, 30, and 60 min. No metabolites were formed in all of the analyses. The same conditions were used for all the other nonsubstrates tested in this paper.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Analytical Equipment and Methods</h3><div class="NLM_p">The LC/MS analyses were run on a Agilent 6540 UHD accurate mass Q-TOF LC/MSMS system governed by Agilent MassHunter software (B.05.00 version). The system consists of a binary pump, autosampler, thermostated column compartment, DAD detector, source, and Q-TOF spectrometer. A volume of 2 Î¼L of each sample was injected by the autosampler. Chromatographic separation of the metabolites was performed on the two chromatographic conditions indicated in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. In both cases the mobile phases consisted of A (H<sub>2</sub>O/0.1% formic acid) and B (acetonitrile/0.1% formic acid) and the columns were operating at a constant temperature of 40 Â°C. Compounds <b>3</b>, <b>8</b>â<b>10</b>, <b>12</b>â<b>16</b> were acquired with method a, and all the other compounds were acquired with method b.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. LC Methods Used for Metabolism Studies</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">column</th><th class="colsep0 rowsep0" align="center">gradient</th><th class="colsep0 rowsep0" align="center">flow rate (mL/min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">MethodÂ a</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AerisÂ wideporeÂ 3.6Â Î¼mÂ C4, 100Â mmÂ ÃÂ 4.6Â mm (Phenomenex, USA)</td><td class="colsep0 rowsep0" align="left">time 0 min, B 0%</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">time 15 min, B 15%</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">time 18 min, B 100%</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">timeÂ 20Â min,Â stopÂ run</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">MethodÂ b</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AerisÂ peptideÂ 1.7Â Î¼mÂ XB-C18, 100Â mmÂ ÃÂ 2.1Â mm (Phenomenex,Â USA)</td><td class="colsep0 rowsep0" align="left">time 0 min, B 0%</td><td class="colsep0 rowsep0" align="left">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">timeÂ 20Â min,Â BÂ 100%</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">time 20 min, stop run</td><td class="colsep0 rowsep0" align="left">Â </td></tr></tbody></table></div></div><div class="NLM_p last">The DAD detector stored all the acquired spectra in the 190â640 nm range (2 nm spectrum step). The ion source was an Agilent Dual JetStream operating under positive ionization mode (4000 V), with nitrogen as desolvating gas (320 Â°C, 10 L/min, 35 psig). The fragmentor was set to 110 V, the skimmer to 65 V, and the octapole rf to 750 V. The spectrometric data were collected in AutoMSMS mode in the 100â1000 mass range, with 3 scans/s both in MS and in MS/MS scans. The TOF operated at 2 GHz.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Computational Section</h3><div class="NLM_p">The starting structure of human FMO3 was built with the homology modeling web server I-TASSER<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55, 56)</a> which used as templates different bacterial FMO3 structures (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7">2vq7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gwd">3gwd</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ucl">3ucl</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xve">2xve</a>).</div><div class="NLM_p">The model was then subjected to minimization, equilibration, and a 50 ns long MD simulation. The standard protonation state at physiological pH was assigned to ionizable residue. Parameters for the FADH2 and NADP+ cofactors were obtained with antechamber using the bcc charging methodology. Waters and ions were added with tleap.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> In particular, the structure was immersed in a pre-equilibrated cubic box of TIP3P<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> water molecules, and the final system contained around 23â100 waters.</div><div class="NLM_p last">MD simulations were run using Gromacs 4.6.1.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The LINCS algorithm<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> was used for constraining all bonds. Trajectories were collected in the <i>NVT</i> ensemble using periodic boundary conditions and Ewald sums for long-range electrostatic interactions. The system was first minimized with the steepest descent algorithm and then equilibrated by heating from 100 to 300 K in three 100 ps steps at 100, 200, and 300 K. Finally, a 50 ns production trajectory was run, collecting frames at 2 ps intervals. Each of the MD snapshots extracted using normal-mode analysis was submitted to FLAP analysis to find the surface binding pockets and the corresponding protein-snapshot pharmacophoric features. The procedure was repeated for all of the protein snapshots and the single pharmacophoric features were collected in a unique global âdynamic pharmacophoreâ model. Finally, the global pharmacophore was used as a template in a structure based approach inside the MetaSite algorithm.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> MIF and Similarity Calculation</h3><div class="NLM_p">The GRID force field<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> was used to produce molecular interaction fields (MIFs) inside the FMO active site. The program GRID is calibrated in a water environment to obtain chemically specific information about a macromolecule (in this case the human FMO3). An electrostatic potential does not normally allow favorable binding sites to be differentiated for a primary, secondary, or tertiary amine cation, for pyridinium, or for a carboxy anion, so the GRID method is an attempt to compute analogous potentials that do have chemical specificity. The object used to measure the potential at each point is given the generic name âProbeâ. Many different Probes can be used on the same macromolecule one after another, and each represents a specific chemical group. A great deal of chemically specific information can therefore be accumulated concerning the way in which the macromolecule might interact favorably with other ligand molecules.</div><div class="NLM_p last">The MIFs in the binding sites of the human FMO3 were obtained using the flexible mode in GRID. With the flexible option, the amino acid side chains can automatically move in response to attractive or repulsive interactions with the chemical probe. The side chain flexibility in GRID can mimic the amino acid movements that occur in the CYP active site to accommodate different substrates according to their sizes, shapes, and interaction patterns.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Nucleophilicity Calculation</h3><div class="NLM_p last">A set of more than <b>50</b> molecules for which the experimentally measured nucleophilicity value (in water) is available was used to create a model correlating these experimentally derived values to the charge, solvation, polarizability, and HOMO energy of the same molecules. The structures have been preliminary relaxed at a semiempirical level using the Austin model 1 (AM1) theory. The resulting geometries have been further optimized with density functional theory (DFT) method at the B3LYP/6-31+G* level of theory in the presence of water. The solvation effect of water has been treated by self-consistent reaction field (SCRF) and the CPCM polarizable conductor calculation model. HOMO, MEP potential, frequencies, and exact polarizability have been computed at the same level of theory using the software Spartan.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Once computed, the charge, solvation, polarizability, and HOMO energy were correlated with the experimental nucleophilicity using the PLS chemometric tool. The maximum prediction power, estimated by the LOO technique, was obtained using one latent variable. This variable explained 90% of the <i>Y</i> variance. The same procedure was then repeated from the beginning for new chemical groups for which the nucleophilicity values in water are not available. The test set molecules were projected into the previously developed PLS model in order to obtain the predictions of the nucleophilicity values.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/jm5007098">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Metabolism of diphenydramine in FMO3, CYP2D6 and CYP3A4; metabolism of compounds <b>15</b> and <b>16</b> in FMO3; main metabolites identified in FMO3 by HRMS; molecular formula strings (in SMILES) for all the molecules investigated in csv format. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5007098/suppl_file/jm5007098_si_001.pdf">jm5007098_si_001.pdf (407.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5007098/suppl_file/jm5007098_si_002.csv">jm5007098_si_002.csv (1.08 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm5007098" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gabriele Cruciani</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Chemoinformatics and Molecular Modelling, Department of Chemistry,
Biology and Biotechnology, University of
Perugia, Via Elce di
Sotto 8, 06123 Perugia, Italy</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e3848281918a868f86cd809196808a828d8aa3968d8a9384cd8a97"><span class="__cf_email__" data-cfemail="b3d4d2d1c1dad6dfd69dd0c1c6d0dad2dddaf3c6dddac3d49ddac7">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aurora Valeri</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Biology and Biotechnology, University of Perugia, Via Elce di Sotto 8, 06123 Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Goracci</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Chemoinformatics and Molecular Modelling, Department of Chemistry,
Biology and Biotechnology, University of
Perugia, Via Elce di
Sotto 8, 06123 Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberto Maria Pellegrino</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Biology and Biotechnology, University of Perugia, Via Elce di Sotto 8, 06123 Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federica Buonerba</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Biology and Biotechnology, University of Perugia, Via Elce di Sotto 8, 06123 Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Massimo Baroni</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery, 215 Marsh
Road, Pinner, HA55NE, London, U.K.</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d529e1905-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47460" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47460" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Prof. Andrea Mattevi (Pavia University, Italy) for the assistance provided during the work on the crystalline structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7">2vq7</a> and the Fondazione Cassa Risparmio Perugia (Project Code No. 2012.0138.021) and âFIRB-Futuro in Ricerca 2010â Program (Project RBFR10X5OO) for their financial support.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i29" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i29"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i30" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i30"> Abbreviations Used</h2><tr><td class="NLM_term">CADME</td><td class="NLM_def"><p class="first last">computer aided drug metabolism elucidation</p></td></tr><tr><td class="NLM_term">FMO</td><td class="NLM_def"><p class="first last">flavin containing monooxygenase</p></td></tr><tr><td class="NLM_term">SoM</td><td class="NLM_def"><p class="first last">site of metabolism</p></td></tr><tr><td class="NLM_term">CYP450</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">MIF</td><td class="NLM_def"><p class="first last">molecular interaction field</p></td></tr><tr><td class="NLM_term">FLAP</td><td class="NLM_def"><p class="first last">fingerprint for ligands and proteins</p></td></tr><tr><td class="NLM_term">MetaSite</td><td class="NLM_def"><p class="first last">site of metabolism calculation</p></td></tr><tr><td class="NLM_term">MetID</td><td class="NLM_def"><p class="first last">metabolite identification</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoaminooxidase</p></td></tr><tr><td class="NLM_term">NADPH</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide phosphate</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">PLD</td><td class="NLM_def"><p class="first last">phospholipidosis</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PLS</td><td class="NLM_def"><p class="first last">partial least squares</p></td></tr><tr><td class="NLM_term">AM1</td><td class="NLM_def"><p class="first last">Austin model 1</p></td></tr><tr><td class="NLM_term">DFT</td><td class="NLM_def"><p class="first last">density functional theory</p></td></tr><tr><td class="NLM_term">SCRF</td><td class="NLM_def"><p class="first last">self-consistent reaction field</p></td></tr><tr><td class="NLM_term">LOO</td><td class="NLM_def"><p class="first last">leave one out</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">BPEA</td><td class="NLM_def"><p class="first last">benzyl[2-(1-pyrrolidinyl)ethyl]aniline</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54185" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54185" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 61 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Cashman, J. R.</span><span> </span><span class="NLM_article-title">Human flavin-containing monooxygenase: substrate specificity and role in drug metabolism</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">â</span> <span class="NLM_lpage">191</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.2174%2F1389200003339135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=11465082" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntlOrs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2000&pages=181-191&author=J.+R.+Cashman&title=Human+flavin-containing+monooxygenase%3A+substrate+specificity+and+role+in+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Human flavin-containing monooxygenase: substrate specificity and role in drug metabolism</span></div><div class="casAuthors">Cashman, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review with 57 refs.  The human flavin-contg. monooxygenase (FMO3) is a prominent enzyme system that converts nucleophilic heteroatom-contg. chems., drugs and xenobiotics to more polar materials that are more efficiently excreted in the urine.  The substrate specificity for FMO3 is distinct from that of FMO1.  Human FMO3 N-oxygenates primary, secondary and tertiary amines whereas human FMO1 is only highly efficient at N-oxygenating tertiary amines.  Both human FMO1 and FMO3 S-oxygenate a no. of nucleophilic sulfur-contg. substrates and in some cases, does so with great stereoselectivity.  Human FMO3 is sensitive to steric features of the substrate and aliph. amines with linkages between the nitrogen atom and a large arom. group such as a phenothiazine of at least five carbons are N-oxygenated significantly more efficiently than those substrates with two or three carbons.  For amines with smaller arom. substituents such as phenethylamines, often these compds. are efficiently N-oxygenated by human FMO3.  Currently, the most promising non-invasive probe of in vivo human FMO3 functional activity is the formation of trimethylamine N-oxide from trimethylamine that comes from dietary choline.  (S)-Nicotine N-1'-oxide formation can also be used as a highly stereoselective probe of human FMO3 function for adult humans that smoke cigarettes.  Finally, cimetidine S-oxygenation or ranitidine N-oxidn. can also be used as a functional probe of human FMO3.  With the recent observation of human FMO3 genetic polymorphism and poor metab. phenotype in certain human populations, variant human FMO3 may contribute to adverse drug reactions or exaggerated clin. response to certain medications.  Knowledge of the substrate specificity for human FMO3 may aid in the future design of more efficacious and less toxic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3KYB29CjVo7Vg90H21EOLACvtfcHk0lhmWVmcUUithw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntlOrs7s%253D&md5=484ccf04bf0c5751015c2b9eb251954d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F1389200003339135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389200003339135%26sid%3Dliteratum%253Aachs%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26atitle%3DHuman%2520flavin-containing%2520monooxygenase%253A%2520substrate%2520specificity%2520and%2520role%2520in%2520drug%2520metabolism%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2000%26volume%3D1%26spage%3D181%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Cashman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span> </span><span class="NLM_article-title">Human flavin-containing monooxygenases</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">â</span> <span class="NLM_lpage">100</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=65-100&author=J.+R.+Cashmanauthor=J.+Zhang&title=Human+flavin-containing+monooxygenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DHuman%2520flavin-containing%2520monooxygenases%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2006%26volume%3D46%26spage%3D65%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Ziegler, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettit, F. H.</span><span> </span><span class="NLM_article-title">Microsomal oxidases. I. The isolation and dialkylarylamine oxygenase activity of pork liver microsomes</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1966</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2932</span><span class="NLM_x">â</span> <span class="NLM_lpage">2938</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi00873a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1966&pages=2932-2938&author=D.+M.+Zieglerauthor=F.+H.+Pettit&title=Microsomal+oxidases.+I.+The+isolation+and+dialkylarylamine+oxygenase+activity+of+pork+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fbi00873a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00873a024%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DD.%2BM.%26aulast%3DPettit%26aufirst%3DF.%2BH.%26atitle%3DMicrosomal%2520oxidases.%2520I.%2520The%2520isolation%2520and%2520dialkylarylamine%2520oxygenase%2520activity%2520of%2520pork%2520liver%2520microsomes%26jtitle%3DBiochemistry%26date%3D1966%26volume%3D5%26spage%3D2932%26epage%3D2938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Testa, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedretti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span> </span><span class="NLM_article-title">Reactions and enzymes in the metabolism of drugs and other xenobiotics</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">â</span> <span class="NLM_lpage">560</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1016%2Fj.drudis.2012.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=22305937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=549-560&author=B.+Testaauthor=A.+Pedrettiauthor=G.+Vistoli&title=Reactions+and+enzymes+in+the+metabolism+of+drugs+and+other+xenobiotics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Reactions and enzymes in the metabolism of drugs and other xenobiotics</span></div><div class="casAuthors">Testa, Bernard; Pedretti, Alessandro; Vistoli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In this article, we offer an overview of the compared quant. importance of biotransformation reactions in the metab. of drugs and other xenobiotics, based on a meta-anal. of current research interests.  Also, we assess the relative significance the enzyme (super)families or categories catalyzing these reactions.  We put the facts unveiled by the anal. into a drug discovery context and draw some implications.  The results confirm the primary role of cytochrome P 450-catalyzed oxidns. and UDP-glucuronosyl-catalyzed glucuronidations, but they also document the marked significance of several other reactions.  Thus, there is a need for several drug discovery scientists to better grasp the variety of drug metab. reactions and enzymes and their consequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbRR72SBUfcLVg90H21EOLACvtfcHk0lhr9Zbf48VM9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D&md5=e11b5b1b7ea18e8bc817bff83efad456</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DPedretti%26aufirst%3DA.%26aulast%3DVistoli%26aufirst%3DG.%26atitle%3DReactions%2520and%2520enzymes%2520in%2520the%2520metabolism%2520of%2520drugs%2520and%2520other%2520xenobiotics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D549%26epage%3D560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Deng, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span> </span><span class="NLM_article-title">Pharmacokinetics, metabolism, and excretion of the antiviral drug arbidol in humans</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1743</span><span class="NLM_x">â</span> <span class="NLM_lpage">1755</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=1743-1755&author=P.+Dengauthor=D.+Zhongauthor=K.+Yuauthor=Y.+Zhangauthor=T.+Wangauthor=X.+Chen&title=Pharmacokinetics%2C+metabolism%2C+and+excretion+of+the+antiviral+drug+arbidol+in+humans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DPharmacokinetics%252C%2520metabolism%252C%2520and%2520excretion%2520of%2520the%2520antiviral%2520drug%2520arbidol%2520in%2520humans%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D1743%26epage%3D1755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Capolongo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anfossi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montesissa, C.</span><span> </span><span class="NLM_article-title">Benzydamine as a useful substrate of hepatic flavin-containing monooxygenase activity in veterinary species</span> <span class="citation_source-journal">J. Vet. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">â</span> <span class="NLM_lpage">346</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=341-346&author=F.+Capolongoauthor=A.+Santiauthor=P.+Anfossiauthor=C.+Montesissa&title=Benzydamine+as+a+useful+substrate+of+hepatic+flavin-containing+monooxygenase+activity+in+veterinary+species"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCapolongo%26aufirst%3DF.%26aulast%3DSanti%26aufirst%3DA.%26aulast%3DAnfossi%26aufirst%3DP.%26aulast%3DMontesissa%26aufirst%3DC.%26atitle%3DBenzydamine%2520as%2520a%2520useful%2520substrate%2520of%2520hepatic%2520flavin-containing%2520monooxygenase%2520activity%2520in%2520veterinary%2520species%26jtitle%3DJ.%2520Vet.%2520Pharmacol.%2520Ther.%26date%3D2009%26volume%3D33%26spage%3D341%26epage%3D346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Hanlon, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camattari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glieder, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kittelmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LÃ¼tz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wirz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, M.</span><span> </span><span class="NLM_article-title">Expression of recombinant human flavin monooxygenase and moclobemide-<i>N</i>-oxide synthesis on multi-mg scale</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6001</span><span class="NLM_x">â</span> <span class="NLM_lpage">6003</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=6001-6003&author=S.+P.+Hanlonauthor=A.+Camattariauthor=S.+Abadauthor=A.+Gliederauthor=M.+Kittelmannauthor=S.+L%C3%BCtzauthor=B.+Wirzauthor=M.+Winkler&title=Expression+of+recombinant+human+flavin+monooxygenase+and+moclobemide-N-oxide+synthesis+on+multi-mg+scale"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanlon%26aufirst%3DS.%2BP.%26aulast%3DCamattari%26aufirst%3DA.%26aulast%3DAbad%26aufirst%3DS.%26aulast%3DGlieder%26aufirst%3DA.%26aulast%3DKittelmann%26aufirst%3DM.%26aulast%3DL%25C3%25BCtz%26aufirst%3DS.%26aulast%3DWirz%26aufirst%3DB.%26aulast%3DWinkler%26aufirst%3DM.%26atitle%3DExpression%2520of%2520recombinant%2520human%2520flavin%2520monooxygenase%2520and%2520moclobemide-N-oxide%2520synthesis%2520on%2520multi-mg%2520scale%26jtitle%3DChem.%2520Commun.%26date%3D2012%26volume%3D48%26spage%3D6001%26epage%3D6003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Mushiroda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douya, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahara, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, O.</span><span> </span><span class="NLM_article-title">The involvement of flavin-containing monooxygenase but not CYP3A4 in metabolism of itopride hydrochloride, a gastroprokinetic agent: comparison with cisapride and mosapride citrate</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">â</span> <span class="NLM_lpage">1237</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=10997945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntVSlt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1231-1237&author=T.+Mushirodaauthor=R.+Douyaauthor=E.+Takaharaauthor=O.+Nagata&title=The+involvement+of+flavin-containing+monooxygenase+but+not+CYP3A4+in+metabolism+of+itopride+hydrochloride%2C+a+gastroprokinetic+agent%3A+comparison+with+cisapride+and+mosapride+citrate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The involvement of flavin-containing monooxygenase but not CYP3A4 in metabolism of itopride hydrochloride, a gastroprokinetic agent: comparison with cisapride and mosapride citrate</span></div><div class="casAuthors">Mushiroda, Taisei; Douya, Rika; Takahara, Eiji; Nagata, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1231-1237</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The goals of the present study were to identify the enzyme responsible for metab. of itopride hydrochloride (itopride) and to evaluate the likelihood of drug interaction involving itopride.  In human liver microsomes, the involvement of flavin-contg. monooxygenase in N-oxygenation, the major metabolic pathway of itopride, was indicated by the following results: inhibition by methimazole and thiourea, heat inactivation, and protection against heat inactivation by NADPH.  When the effects of ketoconazole on the metab. of itopride, cisapride, and mosapride citrate (mosapride) were examd. using human liver microsomes, ketoconazole strongly inhibited the formation of the primary metabolites of cisapride and mosapride, but not itopride.  Other cytochrome P 450 (CYP) 3A4 inhibitors, cimetidine, erythromycin, and clarithromycin, also inhibited the metab. of cisapride and mosapride.  In an in vivo study, itopride (30 mg/kg), cisapride (1.5 mg/kg), or mosapride (3 mg/kg) was orally administered to male rats with or without oral pretreatment with ketoconazole (120 mg/kg) twice daily for 2 days.  The ketoconazole pretreatment significantly increased the area under the serum concn. curve and the max. serum concn. of cisapride and mosapride but had no significant effect on the pharmacokinetics of itopride.  In addn., itopride did not inhibit five specific CYP-mediated reactions of human liver microsomes.  These results suggest that itopride is unlikely to alter the pharmacokinetics of other concomitantly administered drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPynvFuLvYVLVg90H21EOLACvtfcHk0lhNyUDHnCXDFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntVSlt7o%253D&md5=c141c96269304302f08d27e2d7d08371</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMushiroda%26aufirst%3DT.%26aulast%3DDouya%26aufirst%3DR.%26aulast%3DTakahara%26aufirst%3DE.%26aulast%3DNagata%26aufirst%3DO.%26atitle%3DThe%2520involvement%2520of%2520flavin-containing%2520monooxygenase%2520but%2520not%2520CYP3A4%2520in%2520metabolism%2520of%2520itopride%2520hydrochloride%252C%2520a%2520gastroprokinetic%2520agent%253A%2520comparison%2520with%2520cisapride%2520and%2520mosapride%2520citrate%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2000%26volume%3D28%26spage%3D1231%26epage%3D1237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Chung, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cha, Y. N.</span><span> </span><span class="NLM_article-title">Oxidation of ranitidine by isozymes of flavin-containing monooxygenase and cytochrome P450</span> <span class="citation_source-journal">Jpn. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">220</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1254%2Fjjp.84.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=11128045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGgt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2000&pages=213-220&author=W.+G.+Chungauthor=C.+S.+Parkauthor=H.+K.+Rohauthor=W.+K.+Leeauthor=Y.+N.+Cha&title=Oxidation+of+ranitidine+by+isozymes+of+flavin-containing+monooxygenase+and+cytochrome+P450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidation of ranitidine by isozymes of flavin-containing monooxygenase and cytochrome P450</span></div><div class="casAuthors">Chung, Woon-Gye; Park, Chang-Shin; Roh, Hyung-Keun; Lee, Woon-Kee; Cha, Young-Nam</div><div class="citationInfo"><span class="NLM_cas:title">Japanese Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-220</span>CODEN:
                <span class="NLM_cas:coden">JJPAAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-5198</span>.
    
            (<span class="NLM_cas:orgname">Japanese Pharmacological Society</span>)
        </div><div class="casAbstract">Rat and human liver microsomes oxidized ranitidine to its N-oxide (66-76%) and S-oxide (13-18%) and desmethylranitidine (12-16%).  N- and S-oxidns. of ranitidine were inhibited by metimazole [flavin-contg. monooxygenase (FMO) inhibitor] to 96-97% and 71-85%, resp., and desmethylation of ranitidine was inhibited by SKF525A [cytochrome P 450 (CYP) inhibitor] by 71-95%.  Recombinant FMO isoenzymes like FMO1, FMO2, FMO3 and FMO5 produced 39, 79, 2180 and 4 ranitidine N-oxide and 45, 0, 580 and 280 ranitidine S-oxide pmolÂ·min-1Â·nmol-1 FMO, resp.  Desmethylranitidine was not produced by recombinant FMOs.  Prodn. of desmethylranitidine by rat and human liver microsomes was inhibited by tranylcypromine, Î±-naphthoflavon and quinidine, which are known to inhibit CYP2C19, 1A2 and 2D6, resp.  FMO3, the major form in adult liver, produced both ranitidine N- and S-oxides at a 4 to 1 ratio.  FMO1, expressed primarily in human kidney, was 55- and 13-fold less efficient than the hepatic FMO3 in producing ranitidine N- and S-oxides, resp.  FMO2 and FMO5, although expressed slightly in human liver, kidney and lung, were not efficient producers of ranitidine N- and S-oxides.  Thus, urinary contents of ranitidine N-oxide can be used as the in vivo probe to det. the hepatic FMO3 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF4YBYHNhk17Vg90H21EOLACvtfcHk0lhNyUDHnCXDFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGgt7w%253D&md5=48e92a754083d443e376b27f5425c630</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1254%2Fjjp.84.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjjp.84.213%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DW.%2BG.%26aulast%3DPark%26aufirst%3DC.%2BS.%26aulast%3DRoh%26aufirst%3DH.%2BK.%26aulast%3DLee%26aufirst%3DW.%2BK.%26aulast%3DCha%26aufirst%3DY.%2BN.%26atitle%3DOxidation%2520of%2520ranitidine%2520by%2520isozymes%2520of%2520flavin-containing%2520monooxygenase%2520and%2520cytochrome%2520P450%26jtitle%3DJpn.%2520J.%2520Pharmacol.%26date%3D2000%26volume%3D84%26spage%3D213%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">SÃ¶derberg, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahl, M. L.</span><span> </span><span class="NLM_article-title">Pharmacogenetics of olanzapine metabolism</span> <span class="citation_source-journal">Pharmacogenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1319</span><span class="NLM_x">â</span> <span class="NLM_lpage">1336</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=1319-1336&author=M.+M.+S%C3%B6derbergauthor=M.+L.+Dahl&title=Pharmacogenetics+of+olanzapine+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25B6derberg%26aufirst%3DM.%2BM.%26aulast%3DDahl%26aufirst%3DM.%2BL.%26atitle%3DPharmacogenetics%2520of%2520olanzapine%2520metabolism%26jtitle%3DPharmacogenomics%26date%3D2013%26volume%3D14%26spage%3D1319%26epage%3D1336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lohr, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willsky, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acara, M. A.</span><span> </span><span class="NLM_article-title">Renal drug metabolism</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">107</span><span class="NLM_x">â</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1998&pages=107-141&author=J.+W.+Lohrauthor=G.+R.+Willskyauthor=M.+A.+Acara&title=Renal+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLohr%26aufirst%3DJ.%2BW.%26aulast%3DWillsky%26aufirst%3DG.%2BR.%26aulast%3DAcara%26aufirst%3DM.%2BA.%26atitle%3DRenal%2520drug%2520metabolism%26jtitle%3DPharmacol.%2520Rev.%26date%3D1998%26volume%3D50%26spage%3D107%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Ring, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrighton, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aldridge, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haehner, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipley, L. A.</span><span> </span><span class="NLM_article-title">Flavin-containing monooxygenase-mediated N-oxidation of the M(1)-muscarinic agonist xanomeline</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1099</span><span class="NLM_x">â</span> <span class="NLM_lpage">1103</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1099-1103&author=B.+J.+Ringauthor=S.+A.+Wrightonauthor=S.+L.+Aldridgeauthor=K.+Hansenauthor=B.+Haehnerauthor=L.+A.+Shipley&title=Flavin-containing+monooxygenase-mediated+N-oxidation+of+the+M%281%29-muscarinic+agonist+xanomeline"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRing%26aufirst%3DB.%2BJ.%26aulast%3DWrighton%26aufirst%3DS.%2BA.%26aulast%3DAldridge%26aufirst%3DS.%2BL.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DHaehner%26aufirst%3DB.%26aulast%3DShipley%26aufirst%3DL.%2BA.%26atitle%3DFlavin-containing%2520monooxygenase-mediated%2520N-oxidation%2520of%2520the%2520M%25281%2529-muscarinic%2520agonist%2520xanomeline%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1099%26epage%3D1103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Cashman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Celestial, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newdoll, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. B.</span><span> </span><span class="NLM_article-title">Tertiary amines related to brompheniramine: preferred conformations for N-oxygenation by the hog liver flavin-containing monooxygenase</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1097</span><span class="NLM_x">â</span> <span class="NLM_lpage">1105</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1993&pages=1097-1105&author=J.+R.+Cashmanauthor=J.+R.+Celestialauthor=A.+Leachauthor=J.+Newdollauthor=S.+B.+Park&title=Tertiary+amines+related+to+brompheniramine%3A+preferred+conformations+for+N-oxygenation+by+the+hog+liver+flavin-containing+monooxygenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26aulast%3DCelestial%26aufirst%3DJ.%2BR.%26aulast%3DLeach%26aufirst%3DA.%26aulast%3DNewdoll%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DS.%2BB.%26atitle%3DTertiary%2520amines%2520related%2520to%2520brompheniramine%253A%2520preferred%2520conformations%2520for%2520N-oxygenation%2520by%2520the%2520hog%2520liver%2520flavin-containing%2520monooxygenase%26jtitle%3DPharm.%2520Res.%26date%3D1993%26volume%3D10%26spage%3D1097%26epage%3D1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hernandez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janmohamed, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omar, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shephard, E. A.</span><span> </span><span class="NLM_article-title">Deletion of the mouse Fmo1 gene results in enhanced pharmacological behavioural responses to imipramine</span> <span class="citation_source-journal">Pharmacogenet. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">289</span><span class="NLM_x">â</span> <span class="NLM_lpage">299</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1097%2FFPC.0b013e328328d507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=19262426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtF2isLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=289-299&author=D.+Hernandezauthor=A.+Janmohamedauthor=P.+Chandanauthor=B.+A.+Omarauthor=I.+R.+Phillipsauthor=E.+A.+Shephard&title=Deletion+of+the+mouse+Fmo1+gene+results+in+enhanced+pharmacological+behavioural+responses+to+imipramine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of the mouse Fmo1 gene results in enhanced pharmacological behavioural responses to imipramine</span></div><div class="casAuthors">Hernandez, Diana; Janmohamed, Azara; Chandan, Pritpal; Omar, Bilal A.; Phillips, Ian R.; Shephard, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacogenetics and Genomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">289-299</span>CODEN:
                <span class="NLM_cas:coden">PGHEAI</span>;
        ISSN:<span class="NLM_cas:issn">1744-6872</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Many drugs are the subject of multipathway oxidative metab. catalyzed by one or more cytochromes P 450 or flavin-contg. monooxygenases (FMOs).  This complicates assessment of the role of individual enzymes in metabolizing the drug and, hence, in understanding its pharmacogenetics.  To define the role of FMOs in drug metab., we produced FMO-deficient mice.  An Fmo1(-/-), Fmo2(-/-), Fmo4(-/-) mouse line was produced by using chromosomal engineering and Cre-loxP technol.  To assess the utility of the mutant mouse line, it was used to investigate the role of FMO in the metab. of and response to the antidepressant imipramine, which has four major metabolites, three produced by cytochromes P 450 and one, imipramine N-oxide, solely by FMO1.  On treatment with imipramine, wild-type mice became sedated and produced imipramine N-oxide in the brain and other tissues.  In contrast, knockout mice did not produce imipramine N-oxide, but showed exaggerated pharmacol. behavioral responses, such as tremor and body spasm, and had a higher concn. of the parent compd. imipramine in the serum and kidney and there was an increase in desipramine in the brain.  The absence of FMO1-mediated N-oxidn. of imipramine results in enhanced central nervous system effects of the drug.  The results provide insights into the metab. of imipramine in the brain and may explain the basis of the adverse reactions to the drug seen in some patients.  The knockout mouse line will provide a valuable resource for defining the role of FMO1 in the metab. of drugs and other foreign chems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowFKYAzxxWGbVg90H21EOLACvtfcHk0lgiJTUDJMhtvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtF2isLw%253D&md5=99fcc555eb163b0c27b834fec6ec042c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1097%2FFPC.0b013e328328d507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FFPC.0b013e328328d507%26sid%3Dliteratum%253Aachs%26aulast%3DHernandez%26aufirst%3DD.%26aulast%3DJanmohamed%26aufirst%3DA.%26aulast%3DChandan%26aufirst%3DP.%26aulast%3DOmar%26aufirst%3DB.%2BA.%26aulast%3DPhillips%26aufirst%3DI.%2BR.%26aulast%3DShephard%26aufirst%3DE.%2BA.%26atitle%3DDeletion%2520of%2520the%2520mouse%2520Fmo1%2520gene%2520results%2520in%2520enhanced%2520pharmacological%2520behavioural%2520responses%2520to%2520imipramine%26jtitle%3DPharmacogenet.%2520Genomics%26date%3D2009%26volume%3D19%26spage%3D289%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Adali, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpot, R. M.</span><span> </span><span class="NLM_article-title">Modulation of human flavin-containing monooxygenase 3 activity by tricyclic antidepressants and other agents: importance of residue 428</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">358</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">â</span> <span class="NLM_lpage">97</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=1998&pages=92-97&author=O.+Adaliauthor=G.+C.+Carverauthor=R.+M.+Philpot&title=Modulation+of+human+flavin-containing+monooxygenase+3+activity+by+tricyclic+antidepressants+and+other+agents%3A+importance+of+residue+428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAdali%26aufirst%3DO.%26aulast%3DCarver%26aufirst%3DG.%2BC.%26aulast%3DPhilpot%26aufirst%3DR.%2BM.%26atitle%3DModulation%2520of%2520human%2520flavin-containing%2520monooxygenase%25203%2520activity%2520by%2520tricyclic%2520antidepressants%2520and%2520other%2520agents%253A%2520importance%2520of%2520residue%2520428%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1998%26volume%3D358%26spage%3D92%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Akutsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurada, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikegaya, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furihata, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiba, K.</span><span> </span><span class="NLM_article-title">Identification of human cytochrome p450 isozymes involved in diphenhydramine N-demethylation</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">â</span> <span class="NLM_lpage">78</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=72-78&author=T.+Akutsuauthor=K.+Kobayashiauthor=K.+Sakuradaauthor=H.+Ikegayaauthor=T.+Furihataauthor=K.+Chiba&title=Identification+of+human+cytochrome+p450+isozymes+involved+in+diphenhydramine+N-demethylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAkutsu%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DK.%26aulast%3DSakurada%26aufirst%3DK.%26aulast%3DIkegaya%26aufirst%3DH.%26aulast%3DFurihata%26aufirst%3DT.%26aulast%3DChiba%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520human%2520cytochrome%2520p450%2520isozymes%2520involved%2520in%2520diphenhydramine%2520N-demethylation%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2007%26volume%3D35%26spage%3D72%26epage%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ziegler, D. M.</span><span> </span><span class="NLM_article-title">Flavin-containing monooxygenases: catalytic mechanism and substrate specificities</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">â</span> <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.3109%2F03602538809049617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=3293953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaL1cXkvVyntL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1988&pages=1-32&author=D.+M.+Ziegler&title=Flavin-containing+monooxygenases%3A+catalytic+mechanism+and+substrate+specificities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Flavin-containing monooxygenases:  catalytic mechanism and substrate specificities</span></div><div class="casAuthors">Ziegler, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-32</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    </div><div class="casAbstract">A review, with 104 refs., on flavin-contg. monooxygenases with emphasis on the species and tissue enzyme variants, the catalytic mechanism, substrate specificity, assay methods, and the monooxygenase's role in metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1v1Qqf-cp2bVg90H21EOLACvtfcHk0lgiJTUDJMhtvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXkvVyntL4%253D&md5=fad68367091f3761706798f3247dc623</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3109%2F03602538809049617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602538809049617%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DD.%2BM.%26atitle%3DFlavin-containing%2520monooxygenases%253A%2520catalytic%2520mechanism%2520and%2520substrate%2520specificities%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D1988%26volume%3D19%26spage%3D1%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Krueger, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. E.</span><span> </span><span class="NLM_article-title">Mammalian flavin-containing monooxygenases: structure/function, genetic polymorphisms and role in drug metabolism</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">â</span> <span class="NLM_lpage">387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1016%2Fj.pharmthera.2005.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=15922018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1SltLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2005&pages=357-387&author=S.+K.+Kruegerauthor=D.+E.+Williams&title=Mammalian+flavin-containing+monooxygenases%3A+structure%2Ffunction%2C+genetic+polymorphisms+and+role+in+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian flavin-containing monooxygenases: structure/function, genetic polymorphisms and role in drug metabolism</span></div><div class="casAuthors">Krueger, Sharon K.; Williams, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">357-387</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Flavin-contg. monooxygenase (FMO) oxygenates drugs and xenobiotics contg. a "soft nucleophile", usually N or S.  FMO, like cytochrome P 450 (CYP), is a monooxygenase, utilizing the reducing equiv. of NADPH to reduce 1 atom of O2 to H2O, while the other atom is used to oxidize the substrate.  FMO and CYP also exhibit similar tissue and cellular location, mol. wt., substrate specificity, and exist as multiple enzymes under developmental control.  The human FMO functional gene family is much smaller (5 families each with a single member) than CYP.  FMO does not require a reductase to transfer electrons from NADPH and the catalytic cycle of the 2 monooxygenases is strikingly different.  Another distinction is the lack of induction of FMOs by xenobiotics.  In general, CYP is the major contributor to oxidative xenobiotic metab.  However, FMO activity may be of significance in a no. of cases and should not be overlooked.  FMO and CYP have overlapping substrate specificities, but often yield distinct metabolites with potentially significant toxicol./pharmacol. consequences.  The physiol. function(s) of FMOs are poorly understood.  Three of the 5 expressed human FMO genes, FMO1, FMO2, and FMO3, exhibit genetic polymorphisms.  The most studied of these is FMO3 (adult human liver) in which mutant alleles contribute to the disease known as trimethylaminuria.  The consequences of these FMO genetic polymorphisms in drug metab. and human health are areas of research requiring further exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfwDBzxhgaYbVg90H21EOLACvtfcHk0ljK5dc6rsNfvw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1SltLc%253D&md5=2f0b76495aa8a53069bacc9b4ae8de43</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2005.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2005.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DKrueger%26aufirst%3DS.%2BK.%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26atitle%3DMammalian%2520flavin-containing%2520monooxygenases%253A%2520structure%252Ffunction%252C%2520genetic%2520polymorphisms%2520and%2520role%2520in%2520drug%2520metabolism%26jtitle%3DPharmacol.%2520Ther.%26date%3D2005%26volume%3D106%26spage%3D357%26epage%3D387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccarelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonelli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">Modeling phospholipidosis induction: reliability and warnings</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1436</span><span class="NLM_x">â</span> <span class="NLM_lpage">1446</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci400113t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVKmu7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1436-1446&author=L.+Goracciauthor=M.+Ceccarelliauthor=D.+Bonelliauthor=G.+Cruciani&title=Modeling+phospholipidosis+induction%3A+reliability+and+warnings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Phospholipidosis Induction: Reliability and Warnings</span></div><div class="casAuthors">Goracci, Laura; Ceccarelli, Martina; Bonelli, Daniela; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1436-1446</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-induced phospholipidosis (PLD) is characterized by accumulation of phospholipids, the inducing drugs and lamellar inclusion bodies in the lysosomes of affected tissues.  These side effects must be considered as early as possible during drug discovery, and, in fact, numerous in silico models designed to predict PLD have been published.  However, the quality of any in silico model cannot be better than the quality of the exptl. data set used to build it.  The present paper reports an overview of the difficulties and errors encountered in the generation of databases used for the published PLD models.  A new database of 466 compds. was constructed from seven literature sources, contg. only publicly available compds.  A comparison of the PLD assignations in selected databases proved useful in revealing some inconsistencies and raised doubts about the previously assigned PLD+ and PLD- classifications for some chems.  Finally, a Partial Least Squares Discriminant Anal. (PLS-DA) approach was also applied, revealing further anomalies and clearly showing that metab. as well as data quality must be taken into account when generating accurate methods for predicting the likelihood that a compd. will induce PLD.  A new curated database of 331 compds. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruk5nZE3GTi7Vg90H21EOLACvtfcHk0ljK5dc6rsNfvw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVKmu7w%253D&md5=b4134aa063aa8a4abddb525840faba4b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci400113t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400113t%26sid%3Dliteratum%253Aachs%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DCeccarelli%26aufirst%3DM.%26aulast%3DBonelli%26aufirst%3DD.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DModeling%2520phospholipidosis%2520induction%253A%2520reliability%2520and%2520warnings%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1436%26epage%3D1446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Orogo, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minnier, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruhlak, N. L.</span><span> </span><span class="NLM_article-title">Construction and consensus performance of (Q)SAR models for predicting phospholipidosis using a dataset of 743 compounds</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">â</span> <span class="NLM_lpage">739</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2Fminf.201200048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlajsbvI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=725-739&author=A.+M.+Orogoauthor=S.+S.+Choiauthor=B.+L.+Minnierauthor=N.+L.+Kruhlak&title=Construction+and+consensus+performance+of+%28Q%29SAR+models+for+predicting+phospholipidosis+using+a+dataset+of+743+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Construction and Consensus Performance of (Q)SAR Models for Predicting Phospholipidosis Using a Dataset of 743 Compounds</span></div><div class="casAuthors">Orogo, Amabel M.; Choi, Sydney S.; Minnier, Barbara L.; Kruhlak, Naomi L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">725-739</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Drug-induced phospholipidosis (PLD) continues to be a safety concern for pharmaceutical companies and regulatory agencies, prompting the FDA/CDER Phospholipidosis Working Group to develop a database of PLD findings that was recently expanded to contain a total of 743 compds. (385 pos. and 358 neg.).  Three com. (quant.) structure-activity relationship [(Q)SAR] software platforms [MC4PC, Leadscope Predictive Data Miner (LPDM), and Derek for Windows (DfW)] were used to build and/or test models with the database and evaluated individually and together for their ability to predict PLD induction.  Models constructed with MC4PC showed improved sensitivity over previous models constructed with an earlier version of the database and software (61.2 % vs. 50.0 %), but lower specificity in cross-validation expts. (58.2 % vs. 91.9 %) due in part to the more balanced ratio of positives to negatives in the training set.  A new model created with LPDM gave good cross-validation statistics (79.0 % sensitivity, 78.0 % specificity) and the single DfW structural alert for PLD was found to have high pos. predictivity (83.3 %) but low sensitivity (10.4 %) when tested with the entire PLD database.  Combining the predictions of MC4PC, LPDM and/or DfW resulted in increased sensitivity and coverage over using one software platform alone, although it did not enhance the overall prediction accuracy beyond that of the best performing individual software platform.  The comparison across software platforms, however, facilitated the identification and anal. of chems. that were consistently predicted incorrectly by all platforms.  The prevalence of cationic amphiphilic drug (CAD) structural motifs in the database contributed heavily to many of the structural alerts and discriminating features in the models, but the subset of incorrectly predicted structures across all models underscores the need to account for mitigating features and/or addnl. filtering criteria to assess PLD, in particular for PLD-inducing non-CADs and non-PLD-inducing CADs.  (Q)SAR tools may be used as part of an early screening battery or regulatory risk assessment approach to identify those compds. with the greatest chance of inducing PLD and potentially toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmr8bi8YXcbLVg90H21EOLACvtfcHk0ljK5dc6rsNfvw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlajsbvI&md5=2889c571bbbf0d04193690a481cbc3e8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fminf.201200048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201200048%26sid%3Dliteratum%253Aachs%26aulast%3DOrogo%26aufirst%3DA.%2BM.%26aulast%3DChoi%26aufirst%3DS.%2BS.%26aulast%3DMinnier%26aufirst%3DB.%2BL.%26aulast%3DKruhlak%26aufirst%3DN.%2BL.%26atitle%3DConstruction%2520and%2520consensus%2520performance%2520of%2520%2528Q%2529SAR%2520models%2520for%2520predicting%2520phospholipidosis%2520using%2520a%2520dataset%2520of%2520743%2520compounds%26jtitle%3DMol.%2520Inf.%26date%3D2012%26volume%3D31%26spage%3D725%26epage%3D739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geraci, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argikar, U.</span><span> </span><span class="NLM_article-title">Predicting phospholipidosis: a fluorescence non cell based in vitro assay for the determination of drug phospholipid complex formation in early drug discovery</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">6980</span><span class="NLM_x">â</span> <span class="NLM_lpage">6987</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac200683k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2011&pages=6980-6987&author=L.+Zhouauthor=G.+Geraciauthor=S.+Hessauthor=L.+Yangauthor=J.+Wangauthor=U.+Argikar&title=Predicting+phospholipidosis%3A+a+fluorescence+non+cell+based+in+vitro+assay+for+the+determination+of+drug+phospholipid+complex+formation+in+early+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fac200683k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac200683k%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DGeraci%26aufirst%3DG.%26aulast%3DHess%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DArgikar%26aufirst%3DU.%26atitle%3DPredicting%2520phospholipidosis%253A%2520a%2520fluorescence%2520non%2520cell%2520based%2520in%2520vitro%2520assay%2520for%2520the%2520determination%2520of%2520drug%2520phospholipid%2520complex%2520formation%2520in%2520early%2520drug%2520discovery%26jtitle%3DAnal.%2520Chem.%26date%3D2011%26volume%3D83%26spage%3D6980%26epage%3D6987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Quaglino, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruttomesso, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bigler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Follath, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Realdi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettenazzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baritussio, A.</span><span> </span><span class="NLM_article-title">Effects of metabolites and analogs of amiodarone on alveolar macrophages: structureâactivity relationship</span> <span class="citation_source-journal">Am. J. Physiol.: Lung Cell. Mol. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="NLM_x">â</span> <span class="NLM_lpage">447</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1152%2Fajplung.00434.2003" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2004&pages=438-447&author=D.+Quaglinoauthor=H.+R.+Haauthor=E.+Dunerauthor=D.+Bruttomessoauthor=L.+Biglerauthor=F.+Follathauthor=G.+Realdiauthor=A.+Pettenazzoauthor=A.+Baritussio&title=Effects+of+metabolites+and+analogs+of+amiodarone+on+alveolar+macrophages%3A+structure%E2%80%93activity+relationship"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1152%2Fajplung.00434.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajplung.00434.2003%26sid%3Dliteratum%253Aachs%26aulast%3DQuaglino%26aufirst%3DD.%26aulast%3DHa%26aufirst%3DH.%2BR.%26aulast%3DDuner%26aufirst%3DE.%26aulast%3DBruttomesso%26aufirst%3DD.%26aulast%3DBigler%26aufirst%3DL.%26aulast%3DFollath%26aufirst%3DF.%26aulast%3DRealdi%26aufirst%3DG.%26aulast%3DPettenazzo%26aufirst%3DA.%26aulast%3DBaritussio%26aufirst%3DA.%26atitle%3DEffects%2520of%2520metabolites%2520and%2520analogs%2520of%2520amiodarone%2520on%2520alveolar%2520macrophages%253A%2520structure%25E2%2580%2593activity%2520relationship%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Lung%2520Cell.%2520Mol.%2520Physiol.%26date%3D2004%26volume%3D287%26spage%3D438%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Shiraga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaneko, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tozuka, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hata, T.</span><span> </span><span class="NLM_article-title">Identification of cytochrome P450 enzymes involved in the metabolism of zotepine, an antipsychotic drug, in human liver microsomes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">â</span> <span class="NLM_lpage">229</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1080%2F004982599238623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=10219963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaK1MXitleitLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1999&pages=217-229&author=T.+Shiragaauthor=H.+Kanekoauthor=K.+Iwasakiauthor=Z.+Tozukaauthor=A.+Suzukiauthor=T.+Hata&title=Identification+of+cytochrome+P450+enzymes+involved+in+the+metabolism+of+zotepine%2C+an+antipsychotic+drug%2C+in+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of cytochrome P450 enzymes involved in the metabolism of zotepine, an antipsychotic drug, in human liver microsomes</span></div><div class="casAuthors">Shiraga, T.; Kaneko, H.; Iwasaki, K.; Tozuka, Z.; Suzuki, A.; Hata, T.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">217-229</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">1. Studies using human liver microsomes and recombinant human cytochrome P 450 (P 450) enzymes and flavin-contg. monooxygenase (FMO) were performed to identify the enzymes responsible for the formation of zotepine metabolites in man.  2. Human liver microsomes produced four metabolites and a tentative order of importance was: norzotepine, 3-hydroxyzotepine, zotepine S-oxide and 2-hydroxyzotepine.  Zotepine N-oxide was also detected, but it could not be quantified.  3. The rates of formation of the major metabolite, norzotepine, and zotepine S-oxide (at a substrate concn. of 20 Î¼M) were significantly correlated with the testosterone 6Î²-hydroxylase activities and CYP3A4 contents of the 12 different human liver microsomal samples.  Inhibition studies with P 450 enzyme selective inhibitors and anti-rat CYP3A2 antibodies also indicated a predominant role of CYP3A4 in the formation of norzotepine and zotepine S-oxide.  Furafylline and sulphaphenazole inhibited the N-demethylation of zotepine by up to â¼ 30%.  4. Correlation and inhibition data for the 2- and 3-hydroxylation of zotepine were consistent with the predominant role of CYP1A2 and 2D6 in the formation of these metabolites, resp.  5. Recombinant CYP1A1, 1A2, 2B6, 2C19, 3A4 and 3A5 efficiently catalyzed N-demethylation of zotepine.  CYP1A1, 1A2, 2B6 and 3A4 were also active for S-oxidn.  CYP1A2 and 2D6*1-Val374 efficiently produced 2-hydroxyzotepine and 3-hyroxyzotepine, resp.  Recombinant human FMO3 did not catalyze zotepine S-oxidn.  6. These results suggest that both the N-demethylation and S-oxidn. of zotepine are mediated mainly by CYP3A4, and that CYP1A2 and 2D6 play an important role in the 2- and 3-hydroxylation of zotepine, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3CrJhdh2-cLVg90H21EOLACvtfcHk0lhtXXo_tXgvVA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitleitLw%253D&md5=bac2a7fbba2981b64e5b0ab5295377f6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F004982599238623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F004982599238623%26sid%3Dliteratum%253Aachs%26aulast%3DShiraga%26aufirst%3DT.%26aulast%3DKaneko%26aufirst%3DH.%26aulast%3DIwasaki%26aufirst%3DK.%26aulast%3DTozuka%26aufirst%3DZ.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DHata%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520cytochrome%2520P450%2520enzymes%2520involved%2520in%2520the%2520metabolism%2520of%2520zotepine%252C%2520an%2520antipsychotic%2520drug%252C%2520in%2520human%2520liver%2520microsomes%26jtitle%3DXenobiotica%26date%3D1999%26volume%3D29%26spage%3D217%26epage%3D229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Giri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krausz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Idle, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, F. J.</span><span> </span><span class="NLM_article-title">The metabolomics of (+/â)-arecoline 1-oxide in the mouse and its formation by human flavin-containing monooxygenases</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">â</span> <span class="NLM_lpage">573</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2007&pages=561-573&author=S.+Giriauthor=K.+W.+Krauszauthor=J.+R.+Idleauthor=F.+J.+Gonzalez&title=The+metabolomics+of+%28%2B%2F%E2%88%92%29-arecoline+1-oxide+in+the+mouse+and+its+formation+by+human+flavin-containing+monooxygenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGiri%26aufirst%3DS.%26aulast%3DKrausz%26aufirst%3DK.%2BW.%26aulast%3DIdle%26aufirst%3DJ.%2BR.%26aulast%3DGonzalez%26aufirst%3DF.%2BJ.%26atitle%3DThe%2520metabolomics%2520of%2520%2528%252B%252F%25E2%2588%2592%2529-arecoline%25201-oxide%2520in%2520the%2520mouse%2520and%2520its%2520formation%2520by%2520human%2520flavin-containing%2520monooxygenases%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2007%26volume%3D73%26spage%3D561%26epage%3D573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Jensen, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgaard, L.</span><span> </span><span class="NLM_article-title">In vitro metabolism of the m1-muscarinic agonist 5-(2-ethyl-2<i>H</i>-tetrazol-5-yl)-1-methyl-1,2,3,6-tetrahydropyridine by human hepatic cytochromes p-450 determined at pH 7.4 and 8.5</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=125-132&author=K.+G.+Jensenauthor=L.+Dalgaard&title=In+vitro+metabolism+of+the+m1-muscarinic+agonist+5-%282-ethyl-2H-tetrazol-5-yl%29-1-methyl-1%2C2%2C3%2C6-tetrahydropyridine+by+human+hepatic+cytochromes+p-450+determined+at+pH+7.4+and+8.5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DK.%2BG.%26aulast%3DDalgaard%26aufirst%3DL.%26atitle%3DIn%2520vitro%2520metabolism%2520of%2520the%2520m1-muscarinic%2520agonist%25205-%25282-ethyl-2H-tetrazol-5-yl%2529-1-methyl-1%252C2%252C3%252C6-tetrahydropyridine%2520by%2520human%2520hepatic%2520cytochromes%2520p-450%2520determined%2520at%2520pH%25207.4%2520and%25208.5%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D125%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cashman, J. R.</span><span> </span><span class="NLM_article-title">N-oxygenation of phenethylamine to the <i>trans</i>-oxime by adult human liver flavin-containing monooxygenase and retroreduction of phenethylamine hydroxylamine by human liver microsomes</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">1269</span><span class="NLM_x">â</span> <span class="NLM_lpage">1279</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=9316835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaK2sXmsVGlt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1997&pages=1269-1279&author=J.+Linauthor=J.+R.+Cashman&title=N-oxygenation+of+phenethylamine+to+the+trans-oxime+by+adult+human+liver+flavin-containing+monooxygenase+and+retroreduction+of+phenethylamine+hydroxylamine+by+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">N-oxygenation of phenethylamine to the trans-oxime by adult human liver flavin-containing monooxygenase and retroreduction of phenethylamine hydroxylamine by human liver microsomes</span></div><div class="casAuthors">Lin, Jing; Cashman, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1269-1279</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">The biogenic amine phenethylamine has been shown to be N-oxygenated by human flavin-contg. monooxygenase (FMO) (form 3) and human liver microsomes and, to a much lesser extent, N-oxygenated by porcine liver FMO1 and porcine liver microsomes but not by rabbit FMO2.  Adult human liver microsomes catalyze the NADPH-dependent N-oxygenation of phenethylamine to the corresponding trans-oxime through the intermediacy of phenethyl hydroxylamine.  In addn. to trans-oxime formation, phenethyl hydroxylamine is retroreduced to phenethylamine in the presence of human or porcine liver microsomes.  Studies on the biochem. mechanism of N-oxygenation suggested that trans-oxime formation was dependent on the human FMO (form 3) and that retroredn. was stimulated by superoxide and dependent on a cytochrome P 450 system.  These conclusions are based on studies examg. the effects of incubation conditions on phenethylamine N-oxygenation and the effect of reactive oxygen species on phenethyl hydroxylamine retroredn., resp.  The pharmacol. activity of synthetic phenethyl hydroxylamine and phenethyl oxime with a no. of biogenic amine receptors and transporters was examd. in vitro.  In all cases examd., the affinity of phenethyl hydroxylamine and the corresponding oxime for a biogenic transporter or receptors was very poor.  The results suggest that the biogenic amine phenethylamine is efficiently sequentially N-oxygenated in the presence of human liver microsomes or cDNA-expressed FMO (form 3) to phenethyl hydroxylamine and then to oximes that are pharmacol. inactive and serve to terminate biol. activity.  N-oxygenation of phenethylamine to the corresponding trans-oxime is a detoxication process that abrogates pharmacol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwngd9RyWOLVg90H21EOLACvtfcHk0lhtXXo_tXgvVA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsVGlt7w%253D&md5=3f35e2087db2d748ca7b49ffb3f42040</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26atitle%3DN-oxygenation%2520of%2520phenethylamine%2520to%2520the%2520trans-oxime%2520by%2520adult%2520human%2520liver%2520flavin-containing%2520monooxygenase%2520and%2520retroreduction%2520of%2520phenethylamine%2520hydroxylamine%2520by%2520human%2520liver%2520microsomes%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1997%26volume%3D282%26spage%3D1269%26epage%3D1279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">Hydrogen bonding interactions of covalently bonded fluorine atoms: from crystallographic data to a new angular function in the GRID force field</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5114</span><span class="NLM_x">â</span> <span class="NLM_lpage">5125</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0498349" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFGhs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5114-5125&author=E.+Carosatiauthor=S.+Sciabolaauthor=G.+Cruciani&title=Hydrogen+bonding+interactions+of+covalently+bonded+fluorine+atoms%3A+from+crystallographic+data+to+a+new+angular+function+in+the+GRID+force+field"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Bonding Interactions of Covalently Bonded Fluorine Atoms: From Crystallographic Data to a New Angular Function in the GRID Force Field</span></div><div class="casAuthors">Carosati, Emanuele; Sciabola, Simone; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5114-5125</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through the years the GRID force field has been tuned to fit exptl. observations in crystal structures.  This paper describes the detn. of the hydrogen bonding pattern for org. fluorines based on an exhaustive inspection of the Protein Data Bank.  All the PDB complexes, whose protein structures have cocrystd. fluorine-contg. ligands, were examd. and geometrically inspected.  By applying statistics, the hydrogen bonding geometry was described as a distribution function of the angle at the fluorine: a new specific angular function was consequently defined and inserted in the program GRID to est. the effect of fluorine hydrogen bonds on the ligand-protein binding.  All the fluorine-contg. ligands collected from the PDB were docked within their corresponding protein binding sites: introducing the fluorine hydrogen bonding contribution improves the results of the docking expts. in terms of accuracy and ranking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP__-TFksNRbVg90H21EOLACvtfcHk0lhtXXo_tXgvVA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFGhs7s%253D&md5=f119b9adae5296499d467f45fb30c5cf</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm0498349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0498349%26sid%3Dliteratum%253Aachs%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DHydrogen%2520bonding%2520interactions%2520of%2520covalently%2520bonded%2520fluorine%2520atoms%253A%2520from%2520crystallographic%2520data%2520to%2520a%2520new%2520angular%2520function%2520in%2520the%2520GRID%2520force%2520field%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5114%26epage%3D5125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fortuna, C. G.</span><span> </span><span class="NLM_article-title">Exposition and reactivity optimization to predict sites of metabolism in chemicals</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e155</span><span class="NLM_x">â</span> <span class="NLM_lpage">e165</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1016%2Fj.ddtec.2012.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=24050245" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e155-e165&author=G.+Crucianiauthor=M.+Baroniauthor=P.+Benedettiauthor=L.+Goracciauthor=C.+G.+Fortuna&title=Exposition+and+reactivity+optimization+to+predict+sites+of+metabolism+in+chemicals"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DBenedetti%26aufirst%3DP.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DFortuna%26aufirst%3DC.%2BG.%26atitle%3DExposition%2520and%2520reactivity%2520optimization%2520to%2520predict%2520sites%2520of%2520metabolism%2520in%2520chemicals%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De155%26epage%3De165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Shiraga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yajima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teragaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatsubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyashita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Usui, T.</span><span> </span><span class="NLM_article-title">Identification of enzymes responsible for the N-oxidation of darexaban glucuronide, the pharmacologically active metabolite of darexaban, and the glucuronidation of darexaban N-oxides in human liver microsomes</span> <span class="citation_source-journal">Biol. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">413</span><span class="NLM_x">â</span> <span class="NLM_lpage">421</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1248%2Fbpb.35.413" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=22382330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsVejtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2012&pages=413-421&author=T.+Shiragaauthor=K.+Yajimaauthor=T.+Teragakiauthor=K.+Suzukiauthor=T.+Hashimotoauthor=T.+Iwatsuboauthor=A.+Miyashitaauthor=T.+Usui&title=Identification+of+enzymes+responsible+for+the+N-oxidation+of+darexaban+glucuronide%2C+the+pharmacologically+active+metabolite+of+darexaban%2C+and+the+glucuronidation+of+darexaban+N-oxides+in+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of enzymes responsible for the N-oxidation of darexaban glucuronide, the pharmacologically active metabolite of darexaban, and the glucuronidation of darexaban N-oxides in human liver microsomes</span></div><div class="casAuthors">Shiraga, Toshifumi; Yajima, Kanako; Teragaki, Takuya; Suzuki, Katsuhiro; Hashimoto, Tadashi; Iwatsubo, Takafumi; Miyashita, Aiji; Usui, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">413-421</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Darexaban maleate is a novel oral direct factor Xa inhibitor.  Darexaban glucuronide (YM-222714) was the major component in plasma after oral administration of darexaban to humans and is the pharmacol. active metabolite.  Addnl., YM-222714 N-oxides were detected as minor metabolites in human plasma and urine.  It is possible that YM-222714 N-oxides are formed by the N-oxidn. of YM-222714 and/or the glucuronidation of darexaban N-oxides (YM-542845) in vivo.  The former reaction is the pharmacol. inactivation process.  In this study, we identified the human enzymes responsible for YM-222714 N-oxidn. and the UDP (UDP)-glucuronosyltransferase (UGT) isoforms involved in YM-542845 glucuronidation in vitro.  YM-222714 N-oxidn. activity was detected in human liver microsomes (HLM), but not in human intestinal microsomes.  In HLM, YM-222714 N-oxidn. activities were significantly correlated with flavin-contg. monooxygenase (FMO) marker enzyme activities (p<0.001) and inhibited by methimazole, a typical inhibitor of FMOs.  Recombinant human FMO3 and FMO1 were capable of efficiently catalyzing YM-222714 N-oxidn., but not FMO5 or any recombinant human cytochrome P 450 (CYP) isoforms.  Considering the mRNA expression levels of FMO isoforms in human liver, these results strongly suggest that YM-222714 N-oxidn. in HLM is mainly catalyzed by FMO3.  In HLM, YM-542845 glucuronidation was strongly inhibited by typical substrates for UGT1A8, UGT1A9, and UGT1A10.  Recombinant human UGT1A7, UGT1A8, UGT1A9, and UGT1A10 were capable of catalyzing YM-542845 glucuronidation, and UGT1A9 exhibited the highest intrinsic clearance.  Considered together with the expression levels of UGT isoforms in human liver, these results strongly suggest that YM-542845 glucuronidation in HLM is mainly catalyzed by UGT1A9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvizFY_DogYbVg90H21EOLACvtfcHk0lgySCXZD8WIGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsVejtb0%253D&md5=1d9823b3eb6239f63ce48d517e98aa2b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1248%2Fbpb.35.413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.35.413%26sid%3Dliteratum%253Aachs%26aulast%3DShiraga%26aufirst%3DT.%26aulast%3DYajima%26aufirst%3DK.%26aulast%3DTeragaki%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DMiyashita%26aufirst%3DA.%26aulast%3DUsui%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520enzymes%2520responsible%2520for%2520the%2520N-oxidation%2520of%2520darexaban%2520glucuronide%252C%2520the%2520pharmacologically%2520active%2520metabolite%2520of%2520darexaban%252C%2520and%2520the%2520glucuronidation%2520of%2520darexaban%2520N-oxides%2520in%2520human%2520liver%2520microsomes%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D2012%26volume%3D35%26spage%3D413%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Jacobsen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christians, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benet, L. Z.</span><span> </span><span class="NLM_article-title">In vitro evaluation of the disposition of A novel cysteine protease inhibitor</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1343</span><span class="NLM_x">â</span> <span class="NLM_lpage">1351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=11038163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnsFOlsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1343-1351&author=W.+Jacobsenauthor=U.+Christiansauthor=L.+Z.+Benet&title=In+vitro+evaluation+of+the+disposition+of+A+novel+cysteine+protease+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro evaluation of the disposition of a novel cysteine protease inhibitor</span></div><div class="casAuthors">Jacobsen, Wolfgang; Christians, Uwe; Benet, Leslie Z.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1343-1351</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">K11777 (N-methyl-piperazine-Phe-homoPhe-vinylsulfone-phenyl) is a potent, irreversible cysteine protease inhibitor.  Its therapeutic targets are cruzain, a cysteine protease of the protozoan parasite Trypanosoma cruzi, and cathepsins B and L, which are assocd. with cancer progression.  The authors evaluated the metab. of K11777 by human liver microsomes, isolated cytochrome P 450 (CYP) enzymes, and flavin-contg. monooxygenase 3 (FMO3) in vitro.  K11777 was metabolized by human liver microsomes to three major metabolites: N-oxide K11777 (apparent Km = 14.0 Î¼M and apparent Vmax = 3460 pmol Â· mg-1 Â· min-1), Î²-hydroxy-homoPhe K11777 (Km = 16.8 Î¼M and Vmax = 1260 pmol Â· mg-1 Â· min-1), and N-desmethyl K11777 (Km = 18.3 Î¼M and Vmax = 2070 pmol Â· mg-1 Â· min-1).  All three K11777 metabolites were formed by isolated CYP3A and their formation by human liver microsomes was inhibited by the CYP3A inhibitor cyclosporine (50 Î¼M, 54-62% inhibition) and antibodies against human CYP3A4/5 (100 Î¼g of antibodies/100 Î¼g microsomal protein, 55-68% inhibition).  CYP2D6 metabolized K11777 to its N-desmethyl metabolite with an apparent Km (9.2 Î¼M) lower than for CYP3A4 (25.0 Î¼M) and human liver microsomes.  The apparent Km for N-oxide K11777 formation by cDNA-expressed FMO3 was 109 Î¼M.  Based on the intrinsic formation clearances and the results of inhibition expts. (CYP2D6, 50 Î¼M bufuralol; FMO3 mediated, 100 mM methionine) using human liver microsomes, it was estd. that CYP3A contributes to >80% of K11777 metabolite formation.  K11777 was a potent (IC50 = 0.06 Î¼M) and efficacious (max. inhibition 85%) NADPH-dependent inhibitor of human CYP3A4 mediated 6'Î²-hydroxy lovastatin formation, suggesting that K11777 is not only a substrate but also a mechanism-based inhibitor of CYP3A4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnuA6vCCgc0bVg90H21EOLACvtfcHk0lgySCXZD8WIGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnsFOlsbw%253D&md5=d4becc1389b076d394ff43b1d7a093f9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJacobsen%26aufirst%3DW.%26aulast%3DChristians%26aufirst%3DU.%26aulast%3DBenet%26aufirst%3DL.%2BZ.%26atitle%3DIn%2520vitro%2520evaluation%2520of%2520the%2520disposition%2520of%2520A%2520novel%2520cysteine%2520protease%2520inhibitor%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2000%26volume%3D28%26spage%3D1343%26epage%3D1351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Ohmi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akimoto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, S.</span><span> </span><span class="NLM_article-title">S-oxidation of S-methyl-esonarimod by flavin-containing monooxygenases in human liver microsomes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1221</span><span class="NLM_x">â</span> <span class="NLM_lpage">1231</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2003&pages=1221-1231&author=N.+Ohmiauthor=H.+Yoshidaauthor=H.+Endoauthor=M.+Hasegawaauthor=M.+Akimotoauthor=S.+Higuchi&title=S-oxidation+of+S-methyl-esonarimod+by+flavin-containing+monooxygenases+in+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOhmi%26aufirst%3DN.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DEndo%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DAkimoto%26aufirst%3DM.%26aulast%3DHiguchi%26aufirst%3DS.%26atitle%3DS-oxidation%2520of%2520S-methyl-esonarimod%2520by%2520flavin-containing%2520monooxygenases%2520in%2520human%2520liver%2520microsomes%26jtitle%3DXenobiotica%26date%3D2003%26volume%3D33%26spage%3D1221%26epage%3D1231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Xie, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constantinides, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rigas, B.</span><span> </span><span class="NLM_article-title">Regioselective oxidation of phospho-NSAIDs by human cytochrome P450 and flavin monooxygenase isoforms: implications for their pharmacokinetic properties and safety</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">222</span><span class="NLM_x">â</span> <span class="NLM_lpage">232</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1111%2Fj.1476-5381.2012.01982.x" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=222-232&author=G.+Xieauthor=C.+C.+Wongauthor=K.+W.+Chengauthor=L.+Huangauthor=P.+P.+Constantinidesauthor=B.+Rigas&title=Regioselective+oxidation+of+phospho-NSAIDs+by+human+cytochrome+P450+and+flavin+monooxygenase+isoforms%3A+implications+for+their+pharmacokinetic+properties+and+safety"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2012.01982.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2012.01982.x%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DC.%2BC.%26aulast%3DCheng%26aufirst%3DK.%2BW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DConstantinides%26aufirst%3DP.%2BP.%26aulast%3DRigas%26aufirst%3DB.%26atitle%3DRegioselective%2520oxidation%2520of%2520phospho-NSAIDs%2520by%2520human%2520cytochrome%2520P450%2520and%2520flavin%2520monooxygenase%2520isoforms%253A%2520implications%2520for%2520their%2520pharmacokinetic%2520properties%2520and%2520safety%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2012%26volume%3D16%26spage%3D222%26epage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Attar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang-Liu, D. D.</span><span> </span><span class="NLM_article-title">Cytochrome P450 2C8 and flavin-containing monooxygenases are involved in the metabolism of tazarotenic acid in humans</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">476</span><span class="NLM_x">â</span> <span class="NLM_lpage">481</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=476-481&author=M.+Attarauthor=D.+Dongauthor=K.+H.+Lingauthor=D.+D.+Tang-Liu&title=Cytochrome+P450+2C8+and+flavin-containing+monooxygenases+are+involved+in+the+metabolism+of+tazarotenic+acid+in+humans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAttar%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DD.%26aulast%3DLing%26aufirst%3DK.%2BH.%26aulast%3DTang-Liu%26aufirst%3DD.%2BD.%26atitle%3DCytochrome%2520P450%25202C8%2520and%2520flavin-containing%2520monooxygenases%2520are%2520involved%2520in%2520the%2520metabolism%2520of%2520tazarotenic%2520acid%2520in%2520humans%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2003%26volume%3D31%26spage%3D476%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kajita, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inano, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuse, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuwabara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, H.</span><span> </span><span class="NLM_article-title">Effects of olopatadine, a new antiallergic agent, on human liver microsomal cytochrome P450 activities</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">â</span> <span class="NLM_lpage">1511</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=1504-1511&author=J.+Kajitaauthor=K.+Inanoauthor=E.+Fuseauthor=T.+Kuwabaraauthor=H.+Kobayashi&title=Effects+of+olopatadine%2C+a+new+antiallergic+agent%2C+on+human+liver+microsomal+cytochrome+P450+activities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKajita%26aufirst%3DJ.%26aulast%3DInano%26aufirst%3DK.%26aulast%3DFuse%26aufirst%3DE.%26aulast%3DKuwabara%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DH.%26atitle%3DEffects%2520of%2520olopatadine%252C%2520a%2520new%2520antiallergic%2520agent%252C%2520on%2520human%2520liver%2520microsomal%2520cytochrome%2520P450%2520activities%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2002%26volume%3D30%26spage%3D1504%26epage%3D1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Ripp, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overby, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpot, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elfarra, A. A.</span><span> </span><span class="NLM_article-title">Oxidation of cysteine S-conjugates by rabbit liver microsomes and cDNA-expressed flavin-containing mono-oxygenases: studies with S-(1,2-dichlorovinyl)-L-cysteine, S-(1,2,2-trichlorovinyl)-L-cysteine, S-allyl-L-cysteine, and S-benzyl-L-cysteine</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">507</span><span class="NLM_x">â</span> <span class="NLM_lpage">515</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1997&pages=507-515&author=S.+L.+Rippauthor=L.+H.+Overbyauthor=R.+M.+Philpotauthor=A.+A.+Elfarra&title=Oxidation+of+cysteine+S-conjugates+by+rabbit+liver+microsomes+and+cDNA-expressed+flavin-containing+mono-oxygenases%3A+studies+with+S-%281%2C2-dichlorovinyl%29-L-cysteine%2C+S-%281%2C2%2C2-trichlorovinyl%29-L-cysteine%2C+S-allyl-L-cysteine%2C+and+S-benzyl-L-cysteine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRipp%26aufirst%3DS.%2BL.%26aulast%3DOverby%26aufirst%3DL.%2BH.%26aulast%3DPhilpot%26aufirst%3DR.%2BM.%26aulast%3DElfarra%26aufirst%3DA.%2BA.%26atitle%3DOxidation%2520of%2520cysteine%2520S-conjugates%2520by%2520rabbit%2520liver%2520microsomes%2520and%2520cDNA-expressed%2520flavin-containing%2520mono-oxygenases%253A%2520studies%2520with%2520S-%25281%252C2-dichlorovinyl%2529-L-cysteine%252C%2520S-%25281%252C2%252C2-trichlorovinyl%2529-L-cysteine%252C%2520S-allyl-L-cysteine%252C%2520and%2520S-benzyl-L-cysteine%26jtitle%3DMol.%2520Pharmacol.%26date%3D1997%26volume%3D51%26spage%3D507%26epage%3D515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Dever, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elfarra, A. A.</span><span> </span><span class="NLM_article-title">In vivo metabolism of L-methionine in mice: evidence for stereoselective formation of methionine-d-sulfoxide and quantitation of other major metabolites</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2036</span><span class="NLM_x">â</span> <span class="NLM_lpage">2043</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2036-2043&author=J.+T.+Deverauthor=A.+A.+Elfarra&title=In+vivo+metabolism+of+L-methionine+in+mice%3A+evidence+for+stereoselective+formation+of+methionine-d-sulfoxide+and+quantitation+of+other+major+metabolites"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDever%26aufirst%3DJ.%2BT.%26aulast%3DElfarra%26aufirst%3DA.%2BA.%26atitle%3DIn%2520vivo%2520metabolism%2520of%2520L-methionine%2520in%2520mice%253A%2520evidence%2520for%2520stereoselective%2520formation%2520of%2520methionine-d-sulfoxide%2520and%2520quantitation%2520of%2520other%2520major%2520metabolites%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2006%26volume%3D34%26spage%3D2036%26epage%3D2043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">el Amri, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargetton, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delatour, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batt, A. M.</span><span> </span><span class="NLM_article-title">Sulphoxidation of albendazole by the FAD-containing and cytochrome P-450 dependent mono-oxygenases from pig liver microsomes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1159</span><span class="NLM_x">â</span> <span class="NLM_lpage">1168</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1987&pages=1159-1168&author=H.+S.+el+Amriauthor=X.+Fargettonauthor=P.+Delatourauthor=A.+M.+Batt&title=Sulphoxidation+of+albendazole+by+the+FAD-containing+and+cytochrome+P-450+dependent+mono-oxygenases+from+pig+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Del%2BAmri%26aufirst%3DH.%2BS.%26aulast%3DFargetton%26aufirst%3DX.%26aulast%3DDelatour%26aufirst%3DP.%26aulast%3DBatt%26aufirst%3DA.%2BM.%26atitle%3DSulphoxidation%2520of%2520albendazole%2520by%2520the%2520FAD-containing%2520and%2520cytochrome%2520P-450%2520dependent%2520mono-oxygenases%2520from%2520pig%2520liver%2520microsomes%26jtitle%3DXenobiotica%26date%3D1987%26volume%3D17%26spage%3D1159%26epage%3D1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Furnes, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlenk, D.</span><span> </span><span class="NLM_article-title">Extrahepatic metabolism of carbamate and organophosphate thioether compounds by the flavin-containing monooxygenase and cytochrome P450 systems</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">â</span> <span class="NLM_lpage">218</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=214-218&author=B.+Furnesauthor=D.+Schlenk&title=Extrahepatic+metabolism+of+carbamate+and+organophosphate+thioether+compounds+by+the+flavin-containing+monooxygenase+and+cytochrome+P450+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFurnes%26aufirst%3DB.%26aulast%3DSchlenk%26aufirst%3DD.%26atitle%3DExtrahepatic%2520metabolism%2520of%2520carbamate%2520and%2520organophosphate%2520thioether%2520compounds%2520by%2520the%2520flavin-containing%2520monooxygenase%2520and%2520cytochrome%2520P450%2520systems%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D214%26epage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">New and original pK<sub>a</sub> prediction method using grid molecular interaction fields</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2172</span><span class="NLM_x">â</span> <span class="NLM_lpage">2181</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700018y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2172-2181&author=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=G.+Cruciani&title=New+and+original+pKa+prediction+method+using+grid+molecular+interaction+fields"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New and Original pKa Prediction Method Using Grid Molecular Interaction Fields</span></div><div class="casAuthors">Milletti, Francesca; Storchi, Loriano; Sforna, Gianluca; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2172-2181</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the most important physicochem. properties of a mol. is pKa.  It is known that two parameters imperative in ADME profiling, soly., and lipophilicity are governed by pKa, and receptor binding can be influenced by pKa.  Because most drugs are ionized in physiol. conditions, pKa is particularly relevant to medicinal chem.  Despite the numerous advances in high-throughput measurements, in silico detn. is still the fastest and cheapest way of obtaining pKa.  This paper presents a new original computational method for pKa prediction of org. compds.  Descriptors were generated using the program GRID, and these descriptors are based on mol. interaction fields precomputed on a set of mol. fragments.  The new method was developed, trained, and cross-validated by using a large and diverse data set of 24 617 pKa values.  This paper presents the results for a class of 421 acidic nitrogen compds. (RMSE = 0.41, r2 = 0.97, q2 = 0.87) and for a class of 947 six-membered N-heterocyclic bases (RMSE = 0.60, r2 = 0.93, q2 = 0.85).  For external validation 28 novel compds. were selected that covered nine different ionizable groups, and 39 pKa values could be exptl. detd. by spectral gradient anal. (SGA).  Comparison of exptl. pKa with calcd. pKa demonstrated that the predictive ability of the method is good (external set, r2 = 0.85, RMSE = 0.90).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzEk0uiS1E6rVg90H21EOLACvtfcHk0lh1UNK8MGb0fg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF&md5=f2b1dec853ccb0ef1a40540d234232fb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci700018y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700018y%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DNew%2520and%2520original%2520pKa%2520prediction%2520method%2520using%2520grid%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2172%26epage%3D2181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span> </span><span class="NLM_article-title">In silico pK<sub>a</sub> prediction and ADME profiling</span> <span class="citation_source-journal">Chem. Biodiversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1812</span><span class="NLM_x">â</span> <span class="NLM_lpage">1821</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=1812-1821&author=G.+Crucianiauthor=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=L.+Goracci&title=In+silico+pKa+prediction+and+ADME+profiling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DGoracci%26aufirst%3DL.%26atitle%3DIn%2520silico%2520pKa%2520prediction%2520and%2520ADME%2520profiling%26jtitle%3DChem.%2520Biodiversity%26date%3D2009%26volume%3D6%26spage%3D1812%26epage%3D1821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Alfieri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malito, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orru, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Revealing the moonlighting role of NADP in the structure of a flavin-containing monooxygenase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6572</span><span class="NLM_x">â</span> <span class="NLM_lpage">6577</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6572-6577&author=A.+Alfieriauthor=E.+Malitoauthor=R.+Orruauthor=M.+W.+Fraaijeauthor=A.+Mattevi&title=Revealing+the+moonlighting+role+of+NADP+in+the+structure+of+a+flavin-containing+monooxygenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAlfieri%26aufirst%3DA.%26aulast%3DMalito%26aufirst%3DE.%26aulast%3DOrru%26aufirst%3DR.%26aulast%3DFraaije%26aufirst%3DM.%2BW.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DRevealing%2520the%2520moonlighting%2520role%2520of%2520NADP%2520in%2520the%2520structure%2520of%2520a%2520flavin-containing%2520monooxygenase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D6572%26epage%3D6577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Koukouritaki, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddens, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krueger, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VanDyke, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajewski, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, R. N.</span><span> </span><span class="NLM_article-title">Identification and functional analysis of common human flavin-containing monooxygenase 3 genetic variants</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">â</span> <span class="NLM_lpage">273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1124%2Fjpet.106.112268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=17050781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsFCisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2007&pages=266-273&author=S.+B.+Koukouritakiauthor=M.+T.+Pochauthor=M.+C.+Hendersonauthor=L.+K.+Siddensauthor=S.+K.+Kruegerauthor=J.+E.+VanDykeauthor=D.+E.+Williamsauthor=N.+M.+Pajewskiauthor=T.+Wangauthor=R.+N.+Hines&title=Identification+and+functional+analysis+of+common+human+flavin-containing+monooxygenase+3+genetic+variants"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and functional analysis of common human flavin-containing monooxygenase 3 genetic variants</span></div><div class="casAuthors">Koukouritaki, Sevasti B.; Poch, Mark T.; Henderson, Marilyn C.; Siddens, Lisbeth K.; Krueger, Sharon K.; VanDyke, Jonathan E.; Williams, David E.; Pajewski, Nicholas M.; Wang, Tao; Hines, Ronald N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">266-273</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Flavin-contg. monooxygenases (FMOs) are important for the disposition of many therapeutics, environmental toxicants, and nutrients.  FMO3, the major adult hepatic FMO enzyme, exhibits significant interindividual variation.  Eighteen FMO3 single-nucleotide polymorphism (SNP) frequencies were detd. in 202 Hispanics (Mexican descent), 201 African Americans, and 200 non-Latino whites.  Using expressed recombinant enzyme with methimazole, trimethylamine, sulindac, and ethylenethiourea, the novel structural variants FMO3 E24D and K416N were shown to cause modest changes in catalytic efficiency, whereas a third novel variant, FMO3 N61K, was essentially devoid of activity.  The latter variant was present at an allelic frequency of 5.2% in non-Latino whites and 3.5% in African Americans, but it was absent in Hispanics.  Inferring haplotypes using PHASE, version 2.1, the greatest haplotype diversity was obsd. in African Americans followed by non-Latino whites and Hispanics.  Haplotype 2A and 2B, consisting of a hypermorphic promoter SNP cluster (-2650C>G, -2543T>A, and -2177G>C) in linkage with synonymous structural variants was inferred at a frequency of 27% in the Hispanic population, but only 5% in non-Latino whites and African Americans.  This same promoter SNP cluster in linkage with one or more hypomorphic structural variant also was inferred in multiple haplotypes at a total frequency of 5.6% in the African-American study group but less than 1% in the other two groups.  The sum frequencies of the hypomorphic haplotypes H3 [15,167G>A (E158K)], H5B [-2650C>G, 15,167G>A (E158K), 21,375C>T (N285N), 21,443A>G (E308G)], and H6 [15,167G>A (E158K), 21,375C>T (N285N)] was 28% in Hispanics, 23% in non-Latino whites, and 24% in African Americans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyD6Jw2FEfcrVg90H21EOLACvtfcHk0lhDw5UvZo67FQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsFCisw%253D%253D&md5=5af13bff757eed636ca6834e795332cb</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.112268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.112268%26sid%3Dliteratum%253Aachs%26aulast%3DKoukouritaki%26aufirst%3DS.%2BB.%26aulast%3DPoch%26aufirst%3DM.%2BT.%26aulast%3DHenderson%26aufirst%3DM.%2BC.%26aulast%3DSiddens%26aufirst%3DL.%2BK.%26aulast%3DKrueger%26aufirst%3DS.%2BK.%26aulast%3DVanDyke%26aufirst%3DJ.%2BE.%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DPajewski%26aufirst%3DN.%2BM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DHines%26aufirst%3DR.%2BN.%26atitle%3DIdentification%2520and%2520functional%2520analysis%2520of%2520common%2520human%2520flavin-containing%2520monooxygenase%25203%2520genetic%2520variants%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2007%26volume%3D320%26spage%3D266%26epage%3D273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perruccio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span> </span><span class="NLM_article-title">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for ligands and proteins (FLAP): theory and application</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">279</span><span class="NLM_x">â</span> <span class="NLM_lpage">294</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600253e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=279-294&author=M.+Baroniauthor=G.+Crucianiauthor=S.+Sciabolaauthor=F.+Perruccioauthor=J.+S.+Mason&title=A+common+reference+framework+for+analyzing%2Fcomparing+proteins+and+ligands.+Fingerprints+for+ligands+and+proteins+%28FLAP%29%3A+theory+and+application"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for ligands and proteins (FLAP): theory and application</span></div><div class="casAuthors">Baroni, Massimo; Cruciani, Gabriele; Sciabola, Simone; Perruccio, Francesca; Mason, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-294</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast new algorithm (fingerprints for ligands and proteins or FLAP) able to describe small mols. and protein structures using a common ref. framework of four-point pharmacophore fingerprints and a mol.-cavity shape is described in detail.  The procedure starts by using the GRID force field to calc. mol. interaction fields, which are then used to identify particular target locations where an energetic interaction with small mol. features would be very favorable.  The target points thus calcd. are then used by FLAP to build all possible four-point pharmacophores present in the given target site.  A related approach can be applied to small mols., using directly the GRID atom types to identify pharmacophoric features, and this complementary description of the target and ligand then leads to several novel applications.  FLAP can be used for selectivity studies or similarity analyses in order to compare macromols. without superposing them.  Protein families can be compared and clustered into target classes, without bias from previous knowledge and without requiring protein superposition, alignment, or knowledge-based comparison.  FLAP can be used effectively for ligand-based virtual screening and structure-based virtual screening, with the pharmacophore mol. recognition.  Finally, the new method can calc. descriptors for chemometric anal. and can initiate a docking procedure.  This paper presents the background to the new procedure and includes case studies illustrating several relevant applications of the new approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppMc0fYDNn7LVg90H21EOLACvtfcHk0lgVy4yDFAPeNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D&md5=9facb99c5741c7158afab7a3ac387ec3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fci600253e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600253e%26sid%3Dliteratum%253Aachs%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DPerruccio%26aufirst%3DF.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DA%2520common%2520reference%2520framework%2520for%2520analyzing%252Fcomparing%2520proteins%2520and%2520ligands.%2520Fingerprints%2520for%2520ligands%2520and%2520proteins%2520%2528FLAP%2529%253A%2520theory%2520and%2520application%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D279%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Sirci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">RodrÃ­guez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Muijlwijk-Koezen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">GutiÃ©rrez-de-TerÃ¡n, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mannhold, R.</span><span> </span><span class="NLM_article-title">Ligand-, structure- and pharmacophore-based molecular fingerprints: a case study on adenosine A(1), A (2A), A (2B), and A (3) receptor antagonists</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1247</span><span class="NLM_x">â</span> <span class="NLM_lpage">1266</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1247-1266&author=F.+Sirciauthor=L.+Goracciauthor=D.+Rodr%C3%ADguezauthor=J.+van+Muijlwijk-Koezenauthor=H.+Guti%C3%A9rrez-de-Ter%C3%A1nauthor=R.+Mannhold&title=Ligand-%2C+structure-+and+pharmacophore-based+molecular+fingerprints%3A+a+case+study+on+adenosine+A%281%29%2C+A+%282A%29%2C+A+%282B%29%2C+and+A+%283%29+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSirci%26aufirst%3DF.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%26aulast%3DGuti%25C3%25A9rrez-de-Ter%25C3%25A1n%26aufirst%3DH.%26aulast%3DMannhold%26aufirst%3DR.%26atitle%3DLigand-%252C%2520structure-%2520and%2520pharmacophore-based%2520molecular%2520fingerprints%253A%2520a%2520case%2520study%2520on%2520adenosine%2520A%25281%2529%252C%2520A%2520%25282A%2529%252C%2520A%2520%25282B%2529%252C%2520and%2520A%2520%25283%2529%2520receptor%2520antagonists%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D1247%26epage%3D1266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">The basic principles of GRID</span>. In  <span class="citation_source-book">Molecular Interaction Fields: Applications in Drug Discovery and ADME Prediction</span>; <span class="NLM_contrib-group">Cruciani, G.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">â</span> <span class="NLM_lpage">25</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2F3527607676.ch1" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=1-25&author=P.+J.+Goodfordauthor=G.+Cruciani&title=Molecular+Interaction+Fields%3A+Applications+in+Drug+Discovery+and+ADME+Prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2F3527607676.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527607676.ch1%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520basic%2520principles%2520of%2520GRID%26btitle%3DMolecular%2520Interaction%2520Fields%253A%2520Applications%2520in%2520Drug%2520Discovery%2520and%2520ADME%2520Prediction%26aulast%3DCruciani%26aufirst%3DG.%26pub%3DWiley-VCH%26date%3D2006%26spage%3D1%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span>Mayrâs Database of Reactivity Parameters. <a href="http://www.cup.lmu.de/oc/mayr/reaktionsdatenbank/fe/showclass/40" class="extLink">http://www.cup.lmu.de/oc/mayr/reaktionsdatenbank/fe/showclass/40</a> (accessed February 1,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mayr%E2%80%99s+Database+of+Reactivity+Parameters.+http%3A%2F%2Fwww.cup.lmu.de%2Foc%2Fmayr%2Freaktionsdatenbank%2Ffe%2Fshowclass%2F40+%28accessed+February+1%2C+2014%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Rosipal, R.; Kramer, N.</span><span> </span><span class="NLM_article-title">Overview and recent advances in partial least squares</span>. In  <span class="citation_source-book">Subspace, Latent Structure and Feature Selection: Statistical and Optimization Perspectives Workshop (SLSFS 2005)</span>; <span class="NLM_contrib-group">Saunders, C.</span>, <span class="NLM_etal"></span>, Eds.; <span class="NLM_publisher-name">Springer-Verlag</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">34</span><span class="NLM_x">â</span> <span class="NLM_lpage">51</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1007%2F11752790_2" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=34-51&author=R.+Rosipal&author=N.+Kramerauthor=C.+Saunders&title=Subspace%2C+Latent+Structure+and+Feature+Selection%3A+Statistical+and+Optimization+Perspectives+Workshop+%28SLSFS+2005%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2F11752790_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F11752790_2%26sid%3Dliteratum%253Aachs%26aulast%3DRosipal%26aufirst%3DR.%26atitle%3DOverview%2520and%2520recent%2520advances%2520in%2520partial%2520least%2520squares%26btitle%3DSubspace%252C%2520Latent%2520Structure%2520and%2520Feature%2520Selection%253A%2520Statistical%2520and%2520Optimization%2520Perspectives%2520Workshop%2520%2528SLSFS%25202005%2529%26aulast%3DSaunders%26aufirst%3DC.%26pub%3DSpringer-Verlag%26date%3D2006%26spage%3D34%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Boeck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ethirajulu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, R.</span><span> </span><span class="NLM_article-title">MetaSite: understanding metabolism in human cytochromes from the perspective of the chemist</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6970</span><span class="NLM_x">â</span> <span class="NLM_lpage">6979</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050529c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6970-6979&author=G.+Crucianiauthor=E.+Carosatiauthor=B.+De+Boeckauthor=K.+Ethirajuluauthor=C.+Mackieauthor=T.+Howeauthor=R.+Vianello&title=MetaSite%3A+understanding+metabolism+in+human+cytochromes+from+the+perspective+of+the+chemist"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm050529c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050529c%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DDe%2BBoeck%26aufirst%3DB.%26aulast%3DEthirajulu%26aufirst%3DK.%26aulast%3DMackie%26aufirst%3DC.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DVianello%26aufirst%3DR.%26atitle%3DMetaSite%253A%2520understanding%2520metabolism%2520in%2520human%2520cytochromes%2520from%2520the%2520perspective%2520of%2520the%2520chemist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6970%26epage%3D6979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Yanni, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annaert, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustijns, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benjamin, D. K.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakker, D. R.</span><span> </span><span class="NLM_article-title">In vitro hepatic metabolism explains higher clearance of voriconazole in children versus adults: role of CYP2C19 and flavin-containing monooxygenase 3</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">â</span> <span class="NLM_lpage">31</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=25-31&author=S.+B.+Yanniauthor=P.+P.+Annaertauthor=P.+Augustijnsauthor=J.+G.+Ibrahimauthor=D.+K.+Benjaminauthor=D.+R.+Thakker&title=In+vitro+hepatic+metabolism+explains+higher+clearance+of+voriconazole+in+children+versus+adults%3A+role+of+CYP2C19+and+flavin-containing+monooxygenase+3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYanni%26aufirst%3DS.%2BB.%26aulast%3DAnnaert%26aufirst%3DP.%2BP.%26aulast%3DAugustijns%26aufirst%3DP.%26aulast%3DIbrahim%26aufirst%3DJ.%2BG.%26aulast%3DBenjamin%26aufirst%3DD.%2BK.%26aulast%3DThakker%26aufirst%3DD.%2BR.%26atitle%3DIn%2520vitro%2520hepatic%2520metabolism%2520explains%2520higher%2520clearance%2520of%2520voriconazole%2520in%2520children%2520versus%2520adults%253A%2520role%2520of%2520CYP2C19%2520and%2520flavin-containing%2520monooxygenase%25203%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2010%26volume%3D38%26spage%3D25%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Vyas, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roychowdhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koukouritaki, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krueger, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nauseef, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, C. K.</span><span> </span><span class="NLM_article-title">Enzyme-mediated protein haptenation of dapsone and sulfamethoxazole in human keratinocytes: II. Expression and role of flavin-containing monooxygenases and peroxidases</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">319</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">â</span> <span class="NLM_lpage">505</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1124%2Fjpet.106.105874" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=16857727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCksLjJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2006&pages=497-505&author=P.+M.+Vyasauthor=S.+Roychowdhuryauthor=S.+B.+Koukouritakiauthor=R.+N.+Hinesauthor=S.+K.+Kruegerauthor=D.+E.+Williamsauthor=W.+M.+Nauseefauthor=C.+K.+Svensson&title=Enzyme-mediated+protein+haptenation+of+dapsone+and+sulfamethoxazole+in+human+keratinocytes%3A+II.+Expression+and+role+of+flavin-containing+monooxygenases+and+peroxidases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme-mediated protein haptenation of dapsone and sulfamethoxazole in human keratinocytes: II. Expression and role of flavin-containing monooxygenases and peroxidases</span></div><div class="casAuthors">Vyas, Piyush M.; Roychowdhury, Sanjoy; Koukouritaki, Sevasti B.; Hines, Ronald N.; Krueger, Sharon K.; Williams, David E.; Nauseef, William M.; Svensson, Craig K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">497-505</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Arylamine compds., such as sulfamethoxazole (SMX) and dapsone (DDS), are metabolized in epidermal keratinocytes to arylhydroxylamine metabolites that auto-oxidize to arylnitroso derivs., which in turn bind to cellular proteins and can act as antigens/immunogens.  Previous studies have demonstrated that neither cytochromes P 450 nor cyclooxygenases mediate this bioactivation in normal human epidermal keratinocytes (NHEKs).  In this investigation, we demonstrated that methimazole (MMZ), a prototypical substrate of the flavin-contg. monooxygenases (FMOs), attenuated the protein haptenation obsd. in NHEKs exposed to SMX or DDS.  In addn., recombinant FMO1 and FMO3 were able to bioactivate both SMX and DDS, resulting in covalent adduct formation.  Western blot anal. confirmed the presence of FMO3 in NHEKs, whereas FMO1 was not detectable.  In addn. to MMZ, 4-aminobenzoic acid hydrazide (ABH) also attenuated SMX- and DDS-dependent protein haptenation in NHEKs.  ABH did not alter the bioactivation of these drugs by recombinant FMO3, suggesting its inhibitory effect in NHEKs was due to its known ability to inhibit peroxidases.  Studies confirmed the presence of peroxidase activity in NHEKs; however, immunoblot anal. and reverse transcription-polymerase chain reaction indicated that myeloperoxidase, lactoperoxidase, and thyroid peroxidase were absent.  Thus, our results suggest an important role for FMO3 and yet-to-be identified peroxidases in the bioactivation of sulfonamides in NHEKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozWEsbvLmO3LVg90H21EOLACvtfcHk0lhQBI81cuyPMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCksLjJ&md5=87c2f80e0c8517f6976ab4bd0791dd68</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.105874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.105874%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DP.%2BM.%26aulast%3DRoychowdhury%26aufirst%3DS.%26aulast%3DKoukouritaki%26aufirst%3DS.%2BB.%26aulast%3DHines%26aufirst%3DR.%2BN.%26aulast%3DKrueger%26aufirst%3DS.%2BK.%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DNauseef%26aufirst%3DW.%2BM.%26aulast%3DSvensson%26aufirst%3DC.%2BK.%26atitle%3DEnzyme-mediated%2520protein%2520haptenation%2520of%2520dapsone%2520and%2520sulfamethoxazole%2520in%2520human%2520keratinocytes%253A%2520II.%2520Expression%2520and%2520role%2520of%2520flavin-containing%2520monooxygenases%2520and%2520peroxidases%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2006%26volume%3D319%26spage%3D497%26epage%3D505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Alinezhad, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tajbakhsh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahdavi, N.</span><span> </span><span class="NLM_article-title">One-pot reductive amination of carbonyl compounds using sodium borohydride-amberlist 15</span> <span class="citation_source-journal">Synth. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">951</span><span class="NLM_x">â</span> <span class="NLM_lpage">956</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1080%2F00397910903026731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC3cXislKksrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2010&pages=951-956&author=H.+Alinezhadauthor=M.+Tajbakhshauthor=N.+Mahdavi&title=One-pot+reductive+amination+of+carbonyl+compounds+using+sodium+borohydride-amberlist+15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">One-pot reductive amination of carbonyl compounds using sodium borohydride-Amberlyst 15</span></div><div class="casAuthors">Alinezhad, Heshmatollah; Tajbakhsh, Mahmood; Mahdavi, Nastaran</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">951-956</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A fast, efficient, and high-yielding method for the prepn. of amines by reductive amination of aldehydes and ketones using sodium borohydride in the presence of Amberlyst 15 in THF and under solvent-free conditions at room temp. was described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGH1A62i0Jk7Vg90H21EOLACvtfcHk0lhQBI81cuyPMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXislKksrs%253D&md5=93942f8c5572fef66b9d03d14a0bc0bf</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1080%2F00397910903026731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00397910903026731%26sid%3Dliteratum%253Aachs%26aulast%3DAlinezhad%26aufirst%3DH.%26aulast%3DTajbakhsh%26aufirst%3DM.%26aulast%3DMahdavi%26aufirst%3DN.%26atitle%3DOne-pot%2520reductive%2520amination%2520of%2520carbonyl%2520compounds%2520using%2520sodium%2520borohydride-amberlist%252015%26jtitle%3DSynth.%2520Commun.%26date%3D2010%26volume%3D40%26spage%3D951%26epage%3D956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">OâConnor, S.; Dumas, J.; Lee, W.; Dixon, J.; Cantin, D.; Gunn, D.; Burke, J.; Phillips, B.; Lowe, D.; Shelekhin, T.; Wang, G.; Ma, X.; Ying, S.; McClure, A.; Achebe, F.; Lobell, M.; Ehrgott, F.; Iwuagwu, C.; Parcella, K.</span><span> </span><span class="NLM_article-title">Pirrolo[2,1-F][1,2,4]triazin-4-ylamines IGF-1R kinase inhibitors for the treatment of cancer and other hyperproliferative diseases</span>. U.S. Patent US20110294776 A1, Dec 1,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=S.+O%E2%80%99Connor&author=J.+Dumas&author=W.+Lee&author=J.+Dixon&author=D.+Cantin&author=D.+Gunn&author=J.+Burke&author=B.+Phillips&author=D.+Lowe&author=T.+Shelekhin&author=G.+Wang&author=X.+Ma&author=S.+Ying&author=A.+McClure&author=F.+Achebe&author=M.+Lobell&author=F.+Ehrgott&author=C.+Iwuagwu&author=K.+Parcella&title=Pirrolo%5B2%2C1-F%5D%5B1%2C2%2C4%5Dtriazin-4-ylamines+IGF-1R+kinase+inhibitors+for+the+treatment+of+cancer+and+other+hyperproliferative+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DS.%26atitle%3DPirrolo%255B2%252C1-F%255D%255B1%252C2%252C4%255Dtriazin-4-ylamines%2520IGF-1R%2520kinase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520cancer%2520and%2520other%2520hyperproliferative%2520diseases%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Bonn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leandersson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontaine, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamora, I.</span><span> </span><span class="NLM_article-title">Enhanced metabolite identification with MS(E) and a semi-automated software for structural elucidation</span> <span class="citation_source-journal">Rapid Commun. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3127</span><span class="NLM_x">â</span> <span class="NLM_lpage">3138</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2Frcm.4753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=20941759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVKktr3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=3127-3138&author=B.+Bonnauthor=C.+Leanderssonauthor=F.+Fontaineauthor=I.+Zamora&title=Enhanced+metabolite+identification+with+MS%28E%29+and+a+semi-automated+software+for+structural+elucidation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced metabolite identification with MSE and a semi-automated software for structural elucidation</span></div><div class="casAuthors">Bonn, Britta; Leandersson, Carina; Fontaine, Fabien; Zamora, Ismael</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3127-3138</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The identification of metabolites is almost exclusively done with liq. chromatog./tandem mass spectrometry (LC/MSMS) and despite the enormous progress in the development of these techniques and software for handling of data this is a time-consuming task.  In this study the use of quadrupole time-of-flight (QTOF)-generated MSE and MS/MS data were compared with respect to rationalization of metabolites.  In addn. Mass-MetaSite, a semi-automated software for metabolite identification, was evaluated.  The program combines the information from MS raw data, in the form of collision-induced dissocn. spectra, with a prediction of the site of metab. in order to assign the structure of a metabolite.  The aim of the software is to mimic the rationalization of fragment ions performed by a biotransformation scientist in the process of structural elucidation.  For this evaluation, metabolite identification in human liver microsomes was accomplished for 19 com. available compds. and 15 inhouse compds.  The results were very encouraging and for 96% of the metabolites the same structures were assigned using MSE compared with MSMS acquired data.  The possibility of using MSE could considerably reduce the anal. time.  Moreover, Mass-MetaSite performed well and the correct assigned structure, compared to manual inspection of the data, was picked in the first rank in â¼80% of the cases.  In conclusion MSE could be successfully used for metabolite identification in order to reduce time of anal. and Mass-MetaSite could alleviate the work of a biotransformation scientist and decrease the workload by assigning the structure for a majority of the metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PuqO3V_E0bVg90H21EOLACvtfcHk0lhQBI81cuyPMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVKktr3O&md5=a5d1b3b95ce164cb9b7b068a3852c66a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Frcm.4753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.4753%26sid%3Dliteratum%253Aachs%26aulast%3DBonn%26aufirst%3DB.%26aulast%3DLeandersson%26aufirst%3DC.%26aulast%3DFontaine%26aufirst%3DF.%26aulast%3DZamora%26aufirst%3DI.%26atitle%3DEnhanced%2520metabolite%2520identification%2520with%2520MS%2528E%2529%2520and%2520a%2520semi-automated%2520software%2520for%2520structural%2520elucidation%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2010%26volume%3D24%26spage%3D3127%26epage%3D3138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Zamora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontaine, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serra, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plasencia, G.</span><span> </span><span class="NLM_article-title">High-throughput, fully automated, specific MetID. A revolution for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e199</span><span class="NLM_x">â</span> <span class="NLM_lpage">205</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e199-205&author=I.+Zamoraauthor=F.+Fontaineauthor=B.+Serraauthor=G.+Plasencia&title=High-throughput%2C+fully+automated%2C+specific+MetID.+A+revolution+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZamora%26aufirst%3DI.%26aulast%3DFontaine%26aufirst%3DF.%26aulast%3DSerra%26aufirst%3DB.%26aulast%3DPlasencia%26aufirst%3DG.%26atitle%3DHigh-throughput%252C%2520fully%2520automated%252C%2520specific%2520MetID.%2520A%2520revolution%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De199%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">I-TASSER server for protein 3D structure prediction</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1186%2F1471-2105-9-40" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=18215316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=40&author=Y.+Zhang&title=I-TASSER+server+for+protein+3D+structure+prediction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">I-TASSER server for protein 3D structure prediction</span></div><div class="casAuthors">Zhang Yang</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Prediction of 3-dimensional protein structures from amino acid sequences represents one of the most important problems in computational structural biology.  The community-wide Critical Assessment of Structure Prediction (CASP) experiments have been designed to obtain an objective assessment of the state-of-the-art of the field, where I-TASSER was ranked as the best method in the server section of the recent 7th CASP experiment.  Our laboratory has since then received numerous requests about the public availability of the I-TASSER algorithm and the usage of the I-TASSER predictions.  RESULTS:  An on-line version of I-TASSER is developed at the KU Center for Bioinformatics which has generated protein structure predictions for thousands of modeling requests from more than 35 countries.  A scoring function (C-score) based on the relative clustering structural density and the consensus significance score of multiple threading templates is introduced to estimate the accuracy of the I-TASSER predictions.  A large-scale benchmark test demonstrates a strong correlation between the C-score and the TM-score (a structural similarity measurement with values in [0, 1]) of the first models with a correlation coefficient of 0.91.  Using a C-score cutoff > -1.5 for the models of correct topology, both false positive and false negative rates are below 0.1.  Combining C-score and protein length, the accuracy of the I-TASSER models can be predicted with an average error of 0.08 for TM-score and 2 A for RMSD.  CONCLUSION:  The I-TASSER server has been developed to generate automated full-length 3D protein structural predictions where the benchmarked scoring system helps users to obtain quantitative assessments of the I-TASSER models.  The output of the I-TASSER server for each query includes up to five full-length models, the confidence score, the estimated TM-score and RMSD, and the standard deviation of the estimations.  The I-TASSER server is freely available to the academic community at http://zhang.bioinformatics.ku.edu/I-TASSER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUJ3biR9wC1IvKgEt20y1XfW6udTcc2ea9LJ0AXZth-7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D&md5=48ded5b2e40af27acf1e5753f827080c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-40&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-40%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DI-TASSER%2520server%2520for%2520protein%25203D%2520structure%2520prediction%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span>I-TASSER Online. <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/" class="extLink">http://zhanglab.ccmb.med.umich.edu/I-TASSER/</a> (accessed February 12,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=I-TASSER+Online.+http%3A%2F%2Fzhanglab.ccmb.med.umich.edu%2FI-TASSER%2F+%28accessed+February+12%2C+2014%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Case, D. A.; Darden, T. A.; Cheatham, T. E.,  III; Simmerling, C. L.; Wang, J.; Duke, R. E.; Luo, R.; Walker, R. C.; Zhang, W.; Merz, K. M.; Roberts, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Swails, J.; Goetz, A. W.; KolossvÃ¡ry, I.; Wong, K. F.; Paesani, F.; Vanicek, J.; Wolf, R. M.; Liu, J.; Wu, X.; Brozell, S. R.; Steinbrecher, T.; Gohlke, H.; Cai, Q.; Ye, X.; Wang, J.; Hsieh, M.-J.; Cui, G.; Roe, D. R.; Mathews, D. H.; Seetin, M. G.; Salomon-Ferrer, R.; Sagui, C.; Babin, V.; Luchko, T.; Gusarov, S.; Kovalenko, A.; Kollman, P. A.</span>AMBER 12; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>.<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Roberts&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=J.+Swails&author=A.+W.+Goetz&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=R.+M.+Wolf&author=J.+Liu&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=Q.+Cai&author=X.+Ye&author=J.+Wang&author=M.-J.+Hsieh&author=G.+Cui&author=D.+R.+Roe&author=D.+H.+Mathews&author=M.+G.+Seetin&author=R.+Salomon-Ferrer&author=C.+Sagui&author=V.+Babin&author=T.+Luchko&author=S.+Gusarov&author=A.+Kovalenko&author=P.+A.+Kollman&title=AMBER+12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252012%26pub%3DUniversity%2520of%2520California%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">â</span> <span class="NLM_lpage">935</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25Â° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0li2hU4LKzWgiw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutzner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span> </span><span class="NLM_article-title">GROMACS 4: algorithms for highly efficient, load-balanced, and scalable molecular simulation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">â</span> <span class="NLM_lpage">447</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct700301q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=435-447&author=B.+Hessauthor=C.+Kutznerauthor=D.+van+der+Spoelauthor=E.+Lindahl&title=GROMACS+4%3A+algorithms+for+highly+efficient%2C+load-balanced%2C+and+scalable+molecular+simulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span></div><div class="casAuthors">Hess, Berk; Kutzner, Carsten; van der Spoel, David; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. simulation is an extremely useful, but computationally very expensive tool for studies of chem. and biomol. systems.  Here, we present a new implementation of our mol. simulation toolkit GROMACS which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines.  The code encompasses a minimal-communication domain decompn. algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel.  To improve the scaling properties of the common particle mesh Ewald electrostatics algorithms, we have in addn. used a Multiple-Program, Multiple-Data approach, with sep. node domains responsible for direct and reciprocal space interactions.  Not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest nos. of std. cluster nodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajjWdk6mUFrVg90H21EOLACvtfcHk0lgAfdwOOvjDjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D&md5=d53c94901386260221792ea30f151c5f</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fct700301q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700301q%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DKutzner%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204%253A%2520algorithms%2520for%2520highly%2520efficient%252C%2520load-balanced%252C%2520and%2520scalable%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D435%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bekker, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, J. G. E. M.</span><span> </span><span class="NLM_article-title">Lincs: a linear constraint solver for molecular simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1463</span><span class="NLM_x">â</span> <span class="NLM_lpage">1472</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+G.+E.+M.+Fraaije&title=Lincs%3A+a+linear+constraint+solver+for+molecular+simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS: a linear constraint solver for molecular simulations</span></div><div class="casAuthors">Hess, Berk; Bekker, Henk; Berendsen, Herman J. C.; Fraaije, Johannes G. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1472</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">We present a new LINear Constraint Solver (LINCS) for mol. simulations with bond constraints using the enzyme lysozyme and a 32-residue peptide as test systems.  The algorithm is inherently stable, as the constraints themselves are reset instead of derivs. of the constraints, thereby eliminating drift.  Although the derivation of the algorithm is presented in terms of matrixes, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large mols.  At the same accuracy, the LINCS algorithm is 3-4 times faster than the SHAKE algorithm.  Parallelization of the algorithm is straightforward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk6h4pdV0Nx7Vg90H21EOLACvtfcHk0lgAfdwOOvjDjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D&md5=890f8af0d2ca1f65aa93db5a3a0bacf2</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.0.CO%253B2-H%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%2BG.%2BE.%2BM.%26atitle%3DLincs%253A%2520a%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26spage%3D1463%26epage%3D1472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Shao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molnar, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kussmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ochsenfeld, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A. T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slipchenko, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levchenko, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâNeill, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiStasio, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lochan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beran, G. J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besley, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herbert, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Voorhis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steele, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rassolov, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maslen, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korambath, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamson, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, E. F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dachsel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doerksen, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreuw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunietz, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutoi, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furlani, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gwaltney, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, C.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kedziora, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khalliulin, R. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klunzinger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, W. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lotan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Proynov, E. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieniazek, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhee, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritchie, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherrill, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonett, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subotnik, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodcock, H. L.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakraborty, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipman, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keil, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warshel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hehre, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krylov, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, P. M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head-Gordon, M.</span><span> </span><span class="NLM_article-title">Advances in methods and algorithms in a modern quantum chemistry program package</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3172</span><span class="NLM_x">â</span> <span class="NLM_lpage">3191</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1039%2FB517914A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=16902710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsValsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=3172-3191&author=Y.+Shaoauthor=L.+F.+Molnarauthor=Y.+Jungauthor=J.+Kussmannauthor=C.+Ochsenfeldauthor=S.+T.+Brownauthor=A.+T.+B.+Gilbertauthor=L.+V.+Slipchenkoauthor=S.+V.+Levchenkoauthor=D.+P.+O%E2%80%99Neillauthor=R.+A.+DiStasioauthor=R.+C.+Lochanauthor=T.+Wangauthor=G.+J.+O.+Beranauthor=N.+A.+Besleyauthor=J.+M.+Herbertauthor=C.+Y.+Linauthor=T.+Van+Voorhisauthor=S.+H.+Chienauthor=A.+Sodtauthor=R.+P.+Steeleauthor=V.+A.+Rassolovauthor=P.+E.+Maslenauthor=P.+P.+Korambathauthor=R.+D.+Adamsonauthor=B.+Austinauthor=J.+Bakerauthor=E.+F.+C.+Byrdauthor=H.+Dachselauthor=R.+J.+Doerksenauthor=A.+Dreuwauthor=B.+D.+Dunietzauthor=A.+D.+Dutoiauthor=T.+R.+Furlaniauthor=S.+R.+Gwaltneyauthor=A.+Heydenauthor=S.+Hirataauthor=C.-P.+Hsuauthor=G.+Kedzioraauthor=R.+Z.+Khalliulinauthor=P.+Klunzingerauthor=A.+M.+Leeauthor=M.+S.+Leeauthor=W.+Z.+Liangauthor=I.+Lotanauthor=N.+Nairauthor=B.+Petersauthor=E.+I.+Proynovauthor=P.+A.+Pieniazekauthor=Y.+M.+Rheeauthor=J.+Ritchieauthor=E.+Rostaauthor=C.+D.+Sherrillauthor=A.+C.+Simmonettauthor=J.+E.+Subotnikauthor=H.+L.+Woodcockauthor=W.+Zhangauthor=A.+T.+Bellauthor=A.+K.+Chakrabortyauthor=D.+M.+Chipmanauthor=F.+J.+Keilauthor=A.+Warshelauthor=W.+J.+Hehreauthor=H.+F.+Schaeferauthor=J.+Kongauthor=A.+I.+Krylovauthor=P.+M.+W.+Gillauthor=M.+Head-Gordon&title=Advances+in+methods+and+algorithms+in+a+modern+quantum+chemistry+program+package"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in methods and algorithms in a modern quantum chemistry program package</span></div><div class="casAuthors">Shao, Yihan; Molnar, Laszlo Fusti; Jung, Yousung; Kussmann, Jorg; Ochsenfeld, Christian; Brown, Shawn T.; Gilbert, Andrew T. B.; Slipchenko, Lyudmila V.; Levchenko, Sergey V.; O'Neill, Darragh P.; DiStasio, Robert A., Jr.; Lochan, Rohini C.; Wang, Tao; Beran, Gregory J. O.; Besley, Nicholas A.; Herbert, John M.; Lin, Ching Yeh; Van Voorhis, Troy; Chien, Siu Hung; Sodt, Alex; Steele, Ryan P.; Rassolov, Vitaly A.; Maslen, Paul E.; Korambath, Prakashan P.; Adamson, Ross D.; Austin, Brian; Baker, Jon; Byrd, Edward F. C.; Dachsel, Holger; Doerksen, Robert J.; Dreuw, Andreas; Dunietz, Barry D.; Dutoi, Anthony D.; Furlani, Thomas R.; Gwaltney, Steven R.; Heyden, Andreas; Hirata, So; Hsu, Chao-Ping; Kedziora, Gary; Khalliulin, Rustam Z.; Klunzinger, Phil; Lee, Aaron M.; Lee, Michael S.; Liang, WanZhen; Lotan, Itay; Nair, Nikhil; Peters, Baron; Proynov, Emil I.; Pieniazek, Piotr A.; Rhee, Young Min; Ritchie, Jim; Rosta, Edina; Sherrill, C. David; Simmonett, Andrew C.; Subotnik, Joseph E.; Woodcock, H. Lee, III; Zhang, Weimin; Bell, Alexis T.; Chakraborty, Arup K.; Chipman, Daniel M.; Keil, Frerich J.; Warshel, Arieh; Hehre, Warren J.; Schaefer, Henry F., III; Kong, Jing; Krylov, Anna I.; Gill, Peter M. W.; Head-Gordon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3172-3191</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Advances in theory and algorithms for electronic structure calcns. must be incorporated into program packages to enable them to become routinely used by the broader chem. community.  This work reviews advances made over the past five years or so that constitute the major improvements contained in a new release of the Q-Chem quantum chem. package, together with illustrative timings and applications.  Specific developments discussed include fast methods for d. functional theory calcns., linear scaling evaluation of energies, NMR chem. shifts and elec. properties, fast auxiliary basis function methods for correlated energies and gradients, equation-of-motion coupled cluster methods for ground and excited states, geminal wavefunctions, embedding methods and techniques for exploring potential energy surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNN2YG1uDDs7Vg90H21EOLACvtfcHk0lgAfdwOOvjDjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsValsbs%253D&md5=0ba1d2a350afc7922d2f9ca0c6e47d34</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1039%2FB517914A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB517914A%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DY.%26aulast%3DMolnar%26aufirst%3DL.%2BF.%26aulast%3DJung%26aufirst%3DY.%26aulast%3DKussmann%26aufirst%3DJ.%26aulast%3DOchsenfeld%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DS.%2BT.%26aulast%3DGilbert%26aufirst%3DA.%2BT.%2BB.%26aulast%3DSlipchenko%26aufirst%3DL.%2BV.%26aulast%3DLevchenko%26aufirst%3DS.%2BV.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DD.%2BP.%26aulast%3DDiStasio%26aufirst%3DR.%2BA.%26aulast%3DLochan%26aufirst%3DR.%2BC.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DBeran%26aufirst%3DG.%2BJ.%2BO.%26aulast%3DBesley%26aufirst%3DN.%2BA.%26aulast%3DHerbert%26aufirst%3DJ.%2BM.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DVan%2BVoorhis%26aufirst%3DT.%26aulast%3DChien%26aufirst%3DS.%2BH.%26aulast%3DSodt%26aufirst%3DA.%26aulast%3DSteele%26aufirst%3DR.%2BP.%26aulast%3DRassolov%26aufirst%3DV.%2BA.%26aulast%3DMaslen%26aufirst%3DP.%2BE.%26aulast%3DKorambath%26aufirst%3DP.%2BP.%26aulast%3DAdamson%26aufirst%3DR.%2BD.%26aulast%3DAustin%26aufirst%3DB.%26aulast%3DBaker%26aufirst%3DJ.%26aulast%3DByrd%26aufirst%3DE.%2BF.%2BC.%26aulast%3DDachsel%26aufirst%3DH.%26aulast%3DDoerksen%26aufirst%3DR.%2BJ.%26aulast%3DDreuw%26aufirst%3DA.%26aulast%3DDunietz%26aufirst%3DB.%2BD.%26aulast%3DDutoi%26aufirst%3DA.%2BD.%26aulast%3DFurlani%26aufirst%3DT.%2BR.%26aulast%3DGwaltney%26aufirst%3DS.%2BR.%26aulast%3DHeyden%26aufirst%3DA.%26aulast%3DHirata%26aufirst%3DS.%26aulast%3DHsu%26aufirst%3DC.-P.%26aulast%3DKedziora%26aufirst%3DG.%26aulast%3DKhalliulin%26aufirst%3DR.%2BZ.%26aulast%3DKlunzinger%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DA.%2BM.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DLiang%26aufirst%3DW.%2BZ.%26aulast%3DLotan%26aufirst%3DI.%26aulast%3DNair%26aufirst%3DN.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DProynov%26aufirst%3DE.%2BI.%26aulast%3DPieniazek%26aufirst%3DP.%2BA.%26aulast%3DRhee%26aufirst%3DY.%2BM.%26aulast%3DRitchie%26aufirst%3DJ.%26aulast%3DRosta%26aufirst%3DE.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26aulast%3DSimmonett%26aufirst%3DA.%2BC.%26aulast%3DSubotnik%26aufirst%3DJ.%2BE.%26aulast%3DWoodcock%26aufirst%3DH.%2BL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBell%26aufirst%3DA.%2BT.%26aulast%3DChakraborty%26aufirst%3DA.%2BK.%26aulast%3DChipman%26aufirst%3DD.%2BM.%26aulast%3DKeil%26aufirst%3DF.%2BJ.%26aulast%3DWarshel%26aufirst%3DA.%26aulast%3DHehre%26aufirst%3DW.%2BJ.%26aulast%3DSchaefer%26aufirst%3DH.%2BF.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DKrylov%26aufirst%3DA.%2BI.%26aulast%3DGill%26aufirst%3DP.%2BM.%2BW.%26aulast%3DHead-Gordon%26aufirst%3DM.%26atitle%3DAdvances%2520in%2520methods%2520and%2520algorithms%2520in%2520a%2520modern%2520quantum%2520chemistry%2520program%2520package%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D8%26spage%3D3172%26epage%3D3191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 31 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Julia Shanu-Wilson, Liam Evans, Stephen Wrigley, Jonathan Steele, James Atherton, <span class="NLM_string-name hlFld-ContribAuthor">Jason Boer</span>. </span><span class="cited-content_cbyCitation_article-title">Biotransformation: Impact and Application of Metabolism in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (11)
                                     , 2087-2107. <a href="https://doi.org/10.1021/acsmedchemlett.0c00202" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00202</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00202%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DBiotransformation%25253A%252BImpact%252Band%252BApplication%252Bof%252BMetabolism%252Bin%252BDrug%252BDiscovery%26aulast%3DShanu-Wilson%26aufirst%3DJulia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21042020%26date%3D13082020%26date%3D28082020%26volume%3D11%26issue%3D11%26spage%3D2087%26epage%3D2107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xin Yang, Yifei Wang, Ryan Byrne, Gisbert Schneider, <span class="NLM_string-name hlFld-ContribAuthor">Shengyong Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (18)
                                     , 10520-10594. <a href="https://doi.org/10.1021/acs.chemrev.8b00728" title="DOI URL">https://doi.org/10.1021/acs.chemrev.8b00728</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.8b00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.8b00728%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DConcepts%252Bof%252BArtificial%252BIntelligence%252Bfor%252BComputer-Assisted%252BDrug%252BDiscovery%26aulast%3DYang%26aufirst%3DXin%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D29112018%26date%3D11072019%26volume%3D119%26issue%3D18%26spage%3D10520%26epage%3D10594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yun Ma, Russell F. Algera, Ryan A. Woltornist, <span class="NLM_string-name hlFld-ContribAuthor">David B. Collum</span>. </span><span class="cited-content_cbyCitation_article-title">Sodium Diisopropylamide-Mediated Dehydrohalogenations: Influence of Primary- and Secondary-Shell Solvation. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (17)
                                     , 10860-10869. <a href="https://doi.org/10.1021/acs.joc.9b01428" title="DOI URL">https://doi.org/10.1021/acs.joc.9b01428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b01428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b01428%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSodium%252BDiisopropylamide-Mediated%252BDehydrohalogenations%25253A%252BInfluence%252Bof%252BPrimary-%252Band%252BSecondary-Shell%252BSolvation%26aulast%3DMa%26aufirst%3DYun%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D30052019%26date%3D22082019%26volume%3D84%26issue%3D17%26spage%3D10860%26epage%3D10869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriele  Cruciani</span>, <span class="hlFld-ContribAuthor ">NicolÃ²  Milani</span>, <span class="hlFld-ContribAuthor ">Paolo  Benedetti</span>, <span class="hlFld-ContribAuthor ">Susan  Lepri</span>, <span class="hlFld-ContribAuthor ">Lucia  Cesarini</span>, <span class="hlFld-ContribAuthor ">Massimo  Baroni</span>, <span class="hlFld-ContribAuthor ">Francesca  Spyrakis</span>, <span class="hlFld-ContribAuthor ">Sara  Tortorella</span>, <span class="hlFld-ContribAuthor ">Edoardo  Mosconi</span>, and <span class="hlFld-ContribAuthor ">Laura  Goracci</span>  . </span><span class="cited-content_cbyCitation_article-title">From Experiments to a Fast Easy-to-Use Computational Methodology to Predict Human Aldehyde Oxidase Selectivity and Metabolic Reactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (1)
                                     , 360-371. <a href="https://doi.org/10.1021/acs.jmedchem.7b01552" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01552</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01552%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFrom%252BExperiments%252Bto%252Ba%252BFast%252BEasy-to-Use%252BComputational%252BMethodology%252Bto%252BPredict%252BHuman%252BAldehyde%252BOxidase%252BSelectivity%252Band%252BMetabolic%252BReactions%26aulast%3DCruciani%26aufirst%3DGabriele%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D23102017%26date%3D29122017%26date%3D11012018%26date%3D14122017%26volume%3D61%26issue%3D1%26spage%3D360%26epage%3D371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Filippo  Fiorentini</span>, <span class="hlFld-ContribAuthor ">Elvira  Romero</span>, <span class="hlFld-ContribAuthor ">Marco W.  Fraaije</span>, <span class="hlFld-ContribAuthor ">Kurt  Faber</span>, <span class="hlFld-ContribAuthor ">MÃ©lanie  Hall</span>, and <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>  . </span><span class="cited-content_cbyCitation_article-title">BaeyerâVilliger Monooxygenase FMO5 as Entry Point in Drug Metabolism. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (9)
                                     , 2379-2387. <a href="https://doi.org/10.1021/acschembio.7b00470" title="DOI URL">https://doi.org/10.1021/acschembio.7b00470</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00470%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DBaeyer%2525E2%252580%252593Villiger%252BMonooxygenase%252BFMO5%252Bas%252BEntry%252BPoint%252Bin%252BDrug%252BMetabolism%26aulast%3DFiorentini%26aufirst%3DFilippo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D06062017%26date%3D07082017%26date%3D21082017%26date%3D15092017%26date%3D07082017%26volume%3D12%26issue%3D9%26spage%3D2379%26epage%3D2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Filippo  Fiorentini</span>, <span class="hlFld-ContribAuthor ">Martina  Geier</span>, <span class="hlFld-ContribAuthor ">Claudia  Binda</span>, <span class="hlFld-ContribAuthor ">Margit  Winkler</span>, <span class="hlFld-ContribAuthor ">Kurt  Faber</span>, <span class="hlFld-ContribAuthor ">MÃ©lanie  Hall</span>, and <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>  . </span><span class="cited-content_cbyCitation_article-title">Biocatalytic Characterization of Human FMO5: Unearthing BaeyerâVilliger Reactions in Humans. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (4)
                                     , 1039-1048. <a href="https://doi.org/10.1021/acschembio.5b01016" title="DOI URL">https://doi.org/10.1021/acschembio.5b01016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.5b01016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.5b01016%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DBiocatalytic%252BCharacterization%252Bof%252BHuman%252BFMO5%25253A%252BUnearthing%252BBaeyer%2525E2%252580%252593Villiger%252BReactions%252Bin%252BHumans%26aulast%3DFiorentini%26aufirst%3DFilippo%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D10122015%26date%3D15012016%26date%3D01022016%26date%3D15042016%26date%3D15012016%26volume%3D11%26issue%3D4%26spage%3D1039%26epage%3D1048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luigi  Donato</span>, <span class="hlFld-ContribAuthor ">Simona  Alibrandi</span>, <span class="hlFld-ContribAuthor ">Concetta  Scimone</span>, <span class="hlFld-ContribAuthor ">Andrea  Castagnetti</span>, <span class="hlFld-ContribAuthor ">Giacomo  Rao</span>, <span class="hlFld-ContribAuthor ">Antonina  Sidoti</span>, <span class="hlFld-ContribAuthor ">Rosalia  DâAngelo</span>. </span><span class="cited-content_cbyCitation_article-title">Gut-Brain Axis Cross-Talk and Limbic Disorders as Biological Basis of Secondary TMAU. </span><span class="cited-content_cbyCitation_journal-name">Journal of Personalized Medicine</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 87. <a href="https://doi.org/10.3390/jpm11020087" title="DOI URL">https://doi.org/10.3390/jpm11020087</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/jpm11020087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fjpm11020087%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Personalized%2520Medicine%26atitle%3DGut-Brain%252BAxis%252BCross-Talk%252Band%252BLimbic%252BDisorders%252Bas%252BBiological%252BBasis%252Bof%252BSecondary%252BTMAU%26aulast%3DDonato%26aufirst%3DLuigi%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gautier  Bailleul</span>, <span class="hlFld-ContribAuthor ">Callum R.  Nicoll</span>, <span class="hlFld-ContribAuthor ">MarÃ­a Laura  Mascotti</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Marco W.  Fraaije</span>. </span><span class="cited-content_cbyCitation_article-title">Ancestral reconstruction of mammalian FMO1 enables structural determination, revealing unique features that explain its catalytic properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2021,</strong> <em>296 </em>, 100221. <a href="https://doi.org/10.1074/jbc.RA120.016297" title="DOI URL">https://doi.org/10.1074/jbc.RA120.016297</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA120.016297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA120.016297%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DAncestral%252Breconstruction%252Bof%252Bmammalian%252BFMO1%252Benables%252Bstructural%252Bdetermination%25252C%252Brevealing%252Bunique%252Bfeatures%252Bthat%252Bexplain%252Bits%252Bcatalytic%252Bproperties%26aulast%3DBailleul%26aufirst%3DGautier%26date%3D2021%26volume%3D296%26spage%3D100221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huirong  Fan</span>, <span class="hlFld-ContribAuthor ">Aijie  Zhang</span>, <span class="hlFld-ContribAuthor ">Cuiping  Liao</span>, <span class="hlFld-ContribAuthor ">Yuanhui  Yang</span>, <span class="hlFld-ContribAuthor ">Lihua  Zhang</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Xia</span>, <span class="hlFld-ContribAuthor ">Duanyun  Si</span>, <span class="hlFld-ContribAuthor ">Shiqi  Dong</span>, <span class="hlFld-ContribAuthor ">Changxiao  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro
              metabolism and
              in vivo
              pharmacokinetics of bentysrepinine (Y101), an investigational new drug for anti-HBV-infected hepatitis: focus on interspecies comparison. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2020,</strong> <em>50 </em>
                                    (4)
                                     , 468-478. <a href="https://doi.org/10.1080/00498254.2019.1646946" title="DOI URL">https://doi.org/10.1080/00498254.2019.1646946</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00498254.2019.1646946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00498254.2019.1646946%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DIn%252Bvitro%252Bmetabolism%252Band%252Bin%252Bvivo%252Bpharmacokinetics%252Bof%252Bbentysrepinine%252B%252528Y101%252529%25252C%252Ban%252Binvestigational%252Bnew%252Bdrug%252Bfor%252Banti-HBV-infected%252Bhepatitis%25253A%252Bfocus%252Bon%252Binterspecies%252Bcomparison%26aulast%3DFan%26aufirst%3DHuirong%26date%3D2020%26date%3D2019%26volume%3D50%26issue%3D4%26spage%3D468%26epage%3D478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuta  Tsunematsu</span>, <span class="hlFld-ContribAuthor ">Yuichiro  Hirayama</span>, <span class="hlFld-ContribAuthor ">Takahiro  Masuya</span>, <span class="hlFld-ContribAuthor ">Kenji  Watanabe</span>. </span><span class="cited-content_cbyCitation_article-title">Oxidative Modification Enzymes in Polyketide Biosynthetic Pathways. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 479-505. <a href="https://doi.org/10.1016/B978-0-12-409547-2.14637-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.14637-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.14637-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.14637-2%26sid%3Dliteratum%253Aachs%26atitle%3DOxidative%252BModification%252BEnzymes%252Bin%252BPolyketide%252BBiosynthetic%252BPathways%26aulast%3DTsunematsu%26aufirst%3DYuta%26date%3D2020%26spage%3D479%26epage%3D505%26pub%3DElsevier%26atitle%3DComprehensive%252BNatural%252BProducts%252BIII%26date%3D2020%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adrian J.  Fretland</span>, <span class="hlFld-ContribAuthor ">Tashinga E.  Bapiro</span>, <span class="hlFld-ContribAuthor ">Barry  Jones</span>, <span class="hlFld-ContribAuthor ">Roshini  Markandu</span>, <span class="hlFld-ContribAuthor ">Alexandra L.  Orton</span>, <span class="hlFld-ContribAuthor ">Venkatesh  Pilla Reddy</span>. </span><span class="cited-content_cbyCitation_article-title">Using in vitro methods to determine P450s responsible for metabolism and discrimination from other oxidative pathways. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 213-236. <a href="https://doi.org/10.1016/B978-0-12-820018-6.00007-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-820018-6.00007-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820018-6.00007-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820018-6.00007-7%26sid%3Dliteratum%253Aachs%26atitle%3DUsing%252Bin%252Bvitro%252Bmethods%252Bto%252Bdetermine%252BP450s%252Bresponsible%252Bfor%252Bmetabolism%252Band%252Bdiscrimination%252Bfrom%252Bother%252Boxidative%252Bpathways%26aulast%3DFretland%26aufirst%3DAdrian%2BJ.%26date%3D2020%26spage%3D213%26epage%3D236%26pub%3DElsevier%26atitle%3DIdentification%252Band%252BQuantification%252Bof%252BDrugs%25252C%252BMetabolites%25252C%252BDrug%252BMetabolizing%252BEnzymes%25252C%252Band%252BTransporters%26date%3D2020%26volume%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Callum R.  Nicoll</span>, <span class="hlFld-ContribAuthor ">Gautier  Bailleul</span>, <span class="hlFld-ContribAuthor ">Filippo  Fiorentini</span>, <span class="hlFld-ContribAuthor ">MarÃ­a Laura  Mascotti</span>, <span class="hlFld-ContribAuthor ">Marco W.  Fraaije</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>. </span><span class="cited-content_cbyCitation_article-title">Ancestral-sequence reconstruction unveils the structural basis of function in mammalian FMOs. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (1)
                                     , 14-24. <a href="https://doi.org/10.1038/s41594-019-0347-2" title="DOI URL">https://doi.org/10.1038/s41594-019-0347-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41594-019-0347-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41594-019-0347-2%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DAncestral-sequence%252Breconstruction%252Bunveils%252Bthe%252Bstructural%252Bbasis%252Bof%252Bfunction%252Bin%252Bmammalian%252BFMOs%26aulast%3DNicoll%26aufirst%3DCallum%2BR.%26date%3D2020%26date%3D2019%26volume%3D27%26issue%3D1%26spage%3D14%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando Postalli  Rodrigues</span>, <span class="hlFld-ContribAuthor ">Simone  da Costa e Silva Carvalho</span>, <span class="hlFld-ContribAuthor ">Claudia Bueno dos Reis  Martinez</span>, <span class="hlFld-ContribAuthor ">Guilherme  Malafaia</span>, <span class="hlFld-ContribAuthor ">Carmen LuÃ­sa Barbosa  Guedes</span>, <span class="hlFld-ContribAuthor ">Berenice Quinzani  JordÃ£o</span>. </span><span class="cited-content_cbyCitation_article-title">Are the damaging effects of oil refinery effluents on Corbicula fluminea (mollusca) reversible after its transfer to clean water?. </span><span class="cited-content_cbyCitation_journal-name">Ecological Indicators</span><span> <strong>2019,</strong> <em>101 </em>, 1045-1054. <a href="https://doi.org/10.1016/j.ecolind.2019.02.025" title="DOI URL">https://doi.org/10.1016/j.ecolind.2019.02.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ecolind.2019.02.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ecolind.2019.02.025%26sid%3Dliteratum%253Aachs%26jtitle%3DEcological%2520Indicators%26atitle%3DAre%252Bthe%252Bdamaging%252Beffects%252Bof%252Boil%252Brefinery%252Beffluents%252Bon%252BCorbicula%252Bfluminea%252B%252528mollusca%252529%252Breversible%252Bafter%252Bits%252Btransfer%252Bto%252Bclean%252Bwater%25253F%26aulast%3DRodrigues%26aufirst%3DFernando%2BPostalli%26date%3D2019%26volume%3D101%26spage%3D1045%26epage%3D1054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sanja  Jeremic</span>, <span class="hlFld-ContribAuthor ">Lidija  Djokic</span>, <span class="hlFld-ContribAuthor ">Vladimir  AjdaÄiÄ</span>, <span class="hlFld-ContribAuthor ">Nina  BoÅ¾inoviÄ</span>, <span class="hlFld-ContribAuthor ">Vladimir  Pavlovic</span>, <span class="hlFld-ContribAuthor ">Dragan D.  ManojloviÄ</span>, <span class="hlFld-ContribAuthor ">Ramesh  Babu</span>, <span class="hlFld-ContribAuthor ">Ramsankar  Senthamaraikannan</span>, <span class="hlFld-ContribAuthor ">Orlando  Rojas</span>, <span class="hlFld-ContribAuthor ">Igor  Opsenica</span>, <span class="hlFld-ContribAuthor ">Jasmina  Nikodinovic-Runic</span>. </span><span class="cited-content_cbyCitation_article-title">Production of bacterial nanocellulose (BNC) and its application as a solid support in transition metal catalysed cross-coupling reactions. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2019,</strong> <em>129 </em>, 351-360. <a href="https://doi.org/10.1016/j.ijbiomac.2019.01.154" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2019.01.154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2019.01.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2019.01.154%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DProduction%252Bof%252Bbacterial%252Bnanocellulose%252B%252528BNC%252529%252Band%252Bits%252Bapplication%252Bas%252Ba%252Bsolid%252Bsupport%252Bin%252Btransition%252Bmetal%252Bcatalysed%252Bcross-coupling%252Breactions%26aulast%3DJeremic%26aufirst%3DSanja%26date%3D2019%26volume%3D129%26spage%3D351%26epage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">June M.  Chan</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Metabolism and Pharmacogenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 70-90. <a href="https://doi.org/10.1016/B978-0-323-48110-6.00004-1" title="DOI URL">https://doi.org/10.1016/B978-0-323-48110-6.00004-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-48110-6.00004-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-48110-6.00004-1%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BMetabolism%252Band%252BPharmacogenetics%26aulast%3DChan%26aufirst%3DJune%2BM.%26date%3D2019%26spage%3D70%26epage%3D90%26pub%3DElsevier%26atitle%3DPharmacology%252Band%252BPhysiology%252Bfor%252BAnesthesia%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Cini</span>, <span class="hlFld-ContribAuthor ">Valentina  Faltoni</span>, <span class="hlFld-ContribAuthor ">Elena  Petricci</span>, <span class="hlFld-ContribAuthor ">Maurizio  Taddei</span>, <span class="hlFld-ContribAuthor ">Laura  Salvini</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Giannini</span>, <span class="hlFld-ContribAuthor ">Loredana  Vesci</span>, <span class="hlFld-ContribAuthor ">Ferdinando Maria  Milazzo</span>, <span class="hlFld-ContribAuthor ">Anna Maria  Anastasi</span>, <span class="hlFld-ContribAuthor ">Gianfranco  Battistuzzi</span>, <span class="hlFld-ContribAuthor ">Rita  De Santis</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody drug conjugates (ADCs) charged with HDAC inhibitor for targeted epigenetic modulation. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2018,</strong> <em>9 </em>
                                    (31)
                                     , 6490-6496. <a href="https://doi.org/10.1039/C7SC05266A" title="DOI URL">https://doi.org/10.1039/C7SC05266A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7SC05266A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7SC05266A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DAntibody%252Bdrug%252Bconjugates%252B%252528ADCs%252529%252Bcharged%252Bwith%252BHDAC%252Binhibitor%252Bfor%252Btargeted%252Bepigenetic%252Bmodulation%26aulast%3DCini%26aufirst%3DElena%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D31%26spage%3D6490%26epage%3D6496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irina N.  Krasnova</span>, <span class="hlFld-ContribAuthor ">Maria Carla  Gerra</span>, <span class="hlFld-ContribAuthor ">Donna  Walther</span>, <span class="hlFld-ContribAuthor ">Subramaniam  Jayanthi</span>, <span class="hlFld-ContribAuthor ">Bruce  Ladenheim</span>, <span class="hlFld-ContribAuthor ">Michael T.  McCoy</span>, <span class="hlFld-ContribAuthor ">Christie  Brannock</span>, <span class="hlFld-ContribAuthor ">Jean Lud  Cadet</span>. </span><span class="cited-content_cbyCitation_article-title">Compulsive methamphetamine taking in the presence of punishment is associated with increased oxytocin expression in the nucleus accumbens of rats. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-08898-8" title="DOI URL">https://doi.org/10.1038/s41598-017-08898-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-08898-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-08898-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCompulsive%252Bmethamphetamine%252Btaking%252Bin%252Bthe%252Bpresence%252Bof%252Bpunishment%252Bis%252Bassociated%252Bwith%252Bincreased%252Boxytocin%252Bexpression%252Bin%252Bthe%252Bnucleus%252Baccumbens%252Bof%252Brats%26aulast%3DKrasnova%26aufirst%3DIrina%2BN.%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meijuan  Xu</span>, <span class="hlFld-ContribAuthor ">Deepak Kumar  Bhatt</span>, <span class="hlFld-ContribAuthor ">Catherine K.  Yeung</span>, <span class="hlFld-ContribAuthor ">Katrina G.  Claw</span>, <span class="hlFld-ContribAuthor ">Amarjit S.  Chaudhry</span>, <span class="hlFld-ContribAuthor ">Andrea  Gaedigk</span>, <span class="hlFld-ContribAuthor ">Robin E.  Pearce</span>, <span class="hlFld-ContribAuthor ">Ulrich  Broeckel</span>, <span class="hlFld-ContribAuthor ">Roger  Gaedigk</span>, <span class="hlFld-ContribAuthor ">Deborah A.  Nickerson</span>, <span class="hlFld-ContribAuthor ">Erin  Schuetz</span>, <span class="hlFld-ContribAuthor ">Allan E.  Rettie</span>, <span class="hlFld-ContribAuthor ">J. Steven  Leeder</span>, <span class="hlFld-ContribAuthor ">Kenneth E.  Thummel</span>, <span class="hlFld-ContribAuthor ">Bhagwat  Prasad</span>. </span><span class="cited-content_cbyCitation_article-title">Genetic and Nongenetic Factors Associated with Protein Abundance of Flavin-Containing Monooxygenase 3 in Human Liver. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmacology and Experimental Therapeutics</span><span> <strong>2017,</strong> <em>363 </em>
                                    (2)
                                     , 265-274. <a href="https://doi.org/10.1124/jpet.117.243113" title="DOI URL">https://doi.org/10.1124/jpet.117.243113</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/jpet.117.243113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fjpet.117.243113%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmacology%2520and%2520Experimental%2520Therapeutics%26atitle%3DGenetic%252Band%252BNongenetic%252BFactors%252BAssociated%252Bwith%252BProtein%252BAbundance%252Bof%252BFlavin-Containing%252BMonooxygenase%252B3%252Bin%252BHuman%252BLiver%26aulast%3DXu%26aufirst%3DMeijuan%26date%3D2017%26date%3D2017%26volume%3D363%26issue%3D2%26spage%3D265%26epage%3D274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vaibhav A.  Dixit</span>, <span class="hlFld-ContribAuthor ">L. Arun  Lal</span>, <span class="hlFld-ContribAuthor ">Simran R.  Agrawal</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in the prediction of nonâ
              CYP450
              âmediated drug metabolism. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2017,</strong> <em>7 </em>
                                    (6)
                                     <a href="https://doi.org/10.1002/wcms.1323" title="DOI URL">https://doi.org/10.1002/wcms.1323</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1323%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DRecent%252Badvances%252Bin%252Bthe%252Bprediction%252Bof%252Bnon%2525E2%252580%252590%252BCYP450%252B%2525E2%252580%252590mediated%252Bdrug%252Bmetabolism%26aulast%3DDixit%26aufirst%3DVaibhav%2BA.%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan  Lepri</span>, <span class="hlFld-ContribAuthor ">Martina  Ceccarelli</span>, <span class="hlFld-ContribAuthor ">NicolÃ²  Milani</span>, <span class="hlFld-ContribAuthor ">Sara  Tortorella</span>, <span class="hlFld-ContribAuthor ">Andrea  Cucco</span>, <span class="hlFld-ContribAuthor ">Aurora  Valeri</span>, <span class="hlFld-ContribAuthor ">Laura  Goracci</span>, <span class="hlFld-ContribAuthor ">Andreas  Brink</span>, <span class="hlFld-ContribAuthor ">Gabriele  Cruciani</span>. </span><span class="cited-content_cbyCitation_article-title">Structureâmetabolism relationships in
              human-
              AOX: Chemical insights from a large database of aza-aromatic and amide compounds. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2017,</strong> <em>114 </em>
                                    (16)
                                     , E3178-E3187. <a href="https://doi.org/10.1073/pnas.1618881114" title="DOI URL">https://doi.org/10.1073/pnas.1618881114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1618881114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1618881114%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DStructure%2525E2%252580%252593metabolism%252Brelationships%252Bin%252Bhuman-%252BAOX%25253A%252BChemical%252Binsights%252Bfrom%252Ba%252Blarge%252Bdatabase%252Bof%252Baza-aromatic%252Band%252Bamide%252Bcompounds%26aulast%3DLepri%26aufirst%3DSusan%26date%3D2017%26date%3D2017%26volume%3D114%26issue%3D16%26spage%3DE3178%26epage%3DE3187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chien-wei  Fu</span>, <span class="hlFld-ContribAuthor ">Thy-Hou  Lin</span>, . </span><span class="cited-content_cbyCitation_article-title">Predicting the Metabolic Sites by Flavin-Containing Monooxygenase on Drug Molecules Using SVM Classification on Computed Quantum Mechanics and Circular Fingerprints Molecular Descriptors. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (1)
                                     , e0169910. <a href="https://doi.org/10.1371/journal.pone.0169910" title="DOI URL">https://doi.org/10.1371/journal.pone.0169910</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0169910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0169910%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DPredicting%252Bthe%252BMetabolic%252BSites%252Bby%252BFlavin-Containing%252BMonooxygenase%252Bon%252BDrug%252BMolecules%252BUsing%252BSVM%252BClassification%252Bon%252BComputed%252BQuantum%252BMechanics%252Band%252BCircular%252BFingerprints%252BMolecular%252BDescriptors%26aulast%3DFu%26aufirst%3DChien-wei%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D1%26spage%3De0169910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.  Tosco</span>, <span class="hlFld-ContribAuthor ">M.  Mackey</span>. </span><span class="cited-content_cbyCitation_article-title">Lessons and Successes in the Use of Molecular Fields. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 253-296. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12353-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12353-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12353-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12353-4%26sid%3Dliteratum%253Aachs%26atitle%3DLessons%252Band%252BSuccesses%252Bin%252Bthe%252BUse%252Bof%252BMolecular%252BFields%26aulast%3DTosco%26aufirst%3DP.%26date%3D2017%26spage%3D253%26epage%3D296%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huirong  Fan</span>, <span class="hlFld-ContribAuthor ">Zhanxing  Hu</span>, <span class="hlFld-ContribAuthor ">Ruixing  Li</span>, <span class="hlFld-ContribAuthor ">Shiqi  Dong</span>, <span class="hlFld-ContribAuthor ">Yuan  Gu</span>, <span class="hlFld-ContribAuthor ">Ting  Liu</span>, <span class="hlFld-ContribAuthor ">Duanyun  Si</span>, <span class="hlFld-ContribAuthor ">Guangyi  Liang</span>, <span class="hlFld-ContribAuthor ">Changxiao  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Metabolite identification of bentysrepinine (Y101), a novel anti-HBV agent in rats using a five-step strategy based on a combined workflow with two different platforms of liquid chromatographyâtandem mass spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chromatography B</span><span> <strong>2017,</strong> <em>1040 </em>, 118-128. <a href="https://doi.org/10.1016/j.jchromb.2016.12.008" title="DOI URL">https://doi.org/10.1016/j.jchromb.2016.12.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jchromb.2016.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jchromb.2016.12.008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chromatography%2520B%26atitle%3DMetabolite%252Bidentification%252Bof%252Bbentysrepinine%252B%252528Y101%252529%25252C%252Ba%252Bnovel%252Banti-HBV%252Bagent%252Bin%252Brats%252Busing%252Ba%252Bfive-step%252Bstrategy%252Bbased%252Bon%252Ba%252Bcombined%252Bworkflow%252Bwith%252Btwo%252Bdifferent%252Bplatforms%252Bof%252Bliquid%252Bchromatography%2525E2%252580%252593tandem%252Bmass%252Bspectrometry%26aulast%3DFan%26aufirst%3DHuirong%26date%3D2017%26volume%3D1040%26spage%3D118%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.  Fennema</span>, <span class="hlFld-ContribAuthor ">I. R.  Phillips</span>, <span class="hlFld-ContribAuthor ">E. A.  Shephard</span>. </span><span class="cited-content_cbyCitation_article-title">Trimethylamine and Trimethylamine N-Oxide, a Flavin-Containing Monooxygenase 3 (FMO3)-Mediated Host-Microbiome Metabolic Axis Implicated in Health and Disease. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2016,</strong> <em>44 </em>
                                    (11)
                                     , 1839-1850. <a href="https://doi.org/10.1124/dmd.116.070615" title="DOI URL">https://doi.org/10.1124/dmd.116.070615</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.116.070615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.116.070615%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DTrimethylamine%252Band%252BTrimethylamine%252BN-Oxide%25252C%252Ba%252BFlavin-Containing%252BMonooxygenase%252B3%252B%252528FMO3%252529-Mediated%252BHost-Microbiome%252BMetabolic%252BAxis%252BImplicated%252Bin%252BHealth%252Band%252BDisease%26aulast%3DFennema%26aufirst%3DD.%26date%3D2016%26date%3D2016%26volume%3D44%26issue%3D11%26spage%3D1839%26epage%3D1850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  De Vita</span>, <span class="hlFld-ContribAuthor ">Fabiana  Pandolfi</span>, <span class="hlFld-ContribAuthor ">Roberto  Cirilli</span>, <span class="hlFld-ContribAuthor ">Luigi  Scipione</span>, <span class="hlFld-ContribAuthor ">Roberto  Di Santo</span>, <span class="hlFld-ContribAuthor ">Laura  Friggeri</span>, <span class="hlFld-ContribAuthor ">Mattia  Mori</span>, <span class="hlFld-ContribAuthor ">Diego  Fiorucci</span>, <span class="hlFld-ContribAuthor ">Giorgio  Maccari</span>, <span class="hlFld-ContribAuthor ">Robert Selwyne  Arul Christopher</span>, <span class="hlFld-ContribAuthor ">Claudio  Zamperini</span>, <span class="hlFld-ContribAuthor ">Valentina  Pau</span>, <span class="hlFld-ContribAuthor ">Alessandro  De Logu</span>, <span class="hlFld-ContribAuthor ">Silvano  Tortorella</span>, <span class="hlFld-ContribAuthor ">Maurizio  Botta</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of inÂ vitro antitubercular agents through in silico ligand-based approaches. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>121 </em>, 169-180. <a href="https://doi.org/10.1016/j.ejmech.2016.05.032" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.05.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.05.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.05.032%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bin%2525C2%2525A0vitro%252Bantitubercular%252Bagents%252Bthrough%252Bin%252Bsilico%252Bligand-based%252Bapproaches%26aulast%3DDe%2BVita%26aufirst%3DDaniela%26date%3D2016%26volume%3D121%26spage%3D169%26epage%3D180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan  Lepri</span>, <span class="hlFld-ContribAuthor ">Laura  Goracci</span>, <span class="hlFld-ContribAuthor ">Aurora  Valeri</span>, <span class="hlFld-ContribAuthor ">Gabriele  Cruciani</span>. </span><span class="cited-content_cbyCitation_article-title">Metabolism study and biological evaluation of bosentan derivatives. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>121 </em>, 658-670. <a href="https://doi.org/10.1016/j.ejmech.2016.06.006" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.06.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.06.006%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMetabolism%252Bstudy%252Band%252Bbiological%252Bevaluation%252Bof%252Bbosentan%252Bderivatives%26aulast%3DLepri%26aufirst%3DSusan%26date%3D2016%26volume%3D121%26spage%3D658%26epage%3D670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lea  Wagmann</span>, <span class="hlFld-ContribAuthor ">Markus R.  Meyer</span>, <span class="hlFld-ContribAuthor ">Hans H.  Maurer</span>. </span><span class="cited-content_cbyCitation_article-title">What is the contribution of human FMO3 in the N -oxygenation of selected therapeutic drugs and drugs of abuse?. </span><span class="cited-content_cbyCitation_journal-name">Toxicology Letters</span><span> <strong>2016,</strong> <em>258 </em>, 55-70. <a href="https://doi.org/10.1016/j.toxlet.2016.06.013" title="DOI URL">https://doi.org/10.1016/j.toxlet.2016.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.toxlet.2016.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.toxlet.2016.06.013%26sid%3Dliteratum%253Aachs%26jtitle%3DToxicology%2520Letters%26atitle%3DWhat%252Bis%252Bthe%252Bcontribution%252Bof%252Bhuman%252BFMO3%252Bin%252Bthe%252BN%252B-oxygenation%252Bof%252Bselected%252Btherapeutic%252Bdrugs%252Band%252Bdrugs%252Bof%252Babuse%25253F%26aulast%3DWagmann%26aufirst%3DLea%26date%3D2016%26volume%3D258%26spage%3D55%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Pasqualini</span>, <span class="hlFld-ContribAuthor ">E.  Paoletti</span>, <span class="hlFld-ContribAuthor ">G.  Cruciani</span>, <span class="hlFld-ContribAuthor ">R.  Pellegrino</span>, <span class="hlFld-ContribAuthor ">L.  Ederli</span>. </span><span class="cited-content_cbyCitation_article-title">Effects of different routes of application on ethylenediurea persistence in tobacco leaves. </span><span class="cited-content_cbyCitation_journal-name">Environmental Pollution</span><span> <strong>2016,</strong> <em>212 </em>, 559-564. <a href="https://doi.org/10.1016/j.envpol.2016.03.017" title="DOI URL">https://doi.org/10.1016/j.envpol.2016.03.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.envpol.2016.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.envpol.2016.03.017%26sid%3Dliteratum%253Aachs%26jtitle%3DEnvironmental%2520Pollution%26atitle%3DEffects%252Bof%252Bdifferent%252Broutes%252Bof%252Bapplication%252Bon%252Bethylenediurea%252Bpersistence%252Bin%252Btobacco%252Bleaves%26aulast%3DPasqualini%26aufirst%3DS.%26date%3D2016%26volume%3D212%26spage%3D559%26epage%3D564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamil  Kuder</span>, <span class="hlFld-ContribAuthor ">Dorota  ÅaÅ¼ewska</span>, <span class="hlFld-ContribAuthor ">Gniewomir  Latacz</span>, <span class="hlFld-ContribAuthor ">Johannes Stephan  Schwed</span>, <span class="hlFld-ContribAuthor ">Tadeusz  Karcz</span>, <span class="hlFld-ContribAuthor ">Holger  Stark</span>, <span class="hlFld-ContribAuthor ">Janina  Karolak-Wojciechowska</span>, <span class="hlFld-ContribAuthor ">Katarzyna  KieÄ-Kononowicz</span>. </span><span class="cited-content_cbyCitation_article-title">Chlorophenoxy aminoalkyl derivatives as histamine H3R ligands and antiseizure agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (2)
                                     , 53-72. <a href="https://doi.org/10.1016/j.bmc.2015.11.021" title="DOI URL">https://doi.org/10.1016/j.bmc.2015.11.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2015.11.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2015.11.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DChlorophenoxy%252Baminoalkyl%252Bderivatives%252Bas%252Bhistamine%252BH3R%252Bligands%252Band%252Bantiseizure%252Bagents%26aulast%3DKuder%26aufirst%3DKamil%26date%3D2016%26volume%3D24%26issue%3D2%26spage%3D53%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Geier</span>, <span class="hlFld-ContribAuthor ">Thorsten  Bachler</span>, <span class="hlFld-ContribAuthor ">Steven P  Hanlon</span>, <span class="hlFld-ContribAuthor ">Fabian K  Eggimann</span>, <span class="hlFld-ContribAuthor ">Matthias  Kittelmann</span>, <span class="hlFld-ContribAuthor ">HansjÃ¶rg  Weber</span>, <span class="hlFld-ContribAuthor ">Stephan  LÃ¼tz</span>, <span class="hlFld-ContribAuthor ">Beat  Wirz</span>, <span class="hlFld-ContribAuthor ">Margit  Winkler</span>. </span><span class="cited-content_cbyCitation_article-title">Human FMO2-based microbial whole-cell catalysts for drug metabolite synthesis. </span><span class="cited-content_cbyCitation_journal-name">Microbial Cell Factories</span><span> <strong>2015,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12934-015-0262-0" title="DOI URL">https://doi.org/10.1186/s12934-015-0262-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12934-015-0262-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12934-015-0262-0%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrobial%2520Cell%2520Factories%26atitle%3DHuman%252BFMO2-based%252Bmicrobial%252Bwhole-cell%252Bcatalysts%252Bfor%252Bdrug%252Bmetabolite%252Bsynthesis%26aulast%3DGeier%26aufirst%3DMartina%26date%3D2015%26date%3D2015%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hannu  Raunio</span>, <span class="hlFld-ContribAuthor ">Mira  Kuusisto</span>, <span class="hlFld-ContribAuthor ">Risto O.  Juvonen</span>, <span class="hlFld-ContribAuthor ">Olli T.  PentikÃÂ¤inen</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling of interactions between xenobiotics and cytochrome P450 (CYP) enzymes. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2015,</strong> <em>6 </em><a href="https://doi.org/10.3389/fphar.2015.00123" title="DOI URL">https://doi.org/10.3389/fphar.2015.00123</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2015.00123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2015.00123%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DModeling%252Bof%252Binteractions%252Bbetween%252Bxenobiotics%252Band%252Bcytochrome%252BP450%252B%252528CYP%252529%252Benzymes%26aulast%3DRaunio%26aufirst%3DHannu%26date%3D2015%26date%3D2015%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structures of zotepine (<b>1</b>), diphenidramine (<b>2</b>), cyamemazine (<b>3</b>), and bepridil (<b>4</b>). Compound <b>1</b> is a known FMO3 substrate.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compound <b>2</b>, incubated in recombinant FMO3, is an FMO3 substrate, while <b>3</b> and <b>4</b> proved to be nonsubstrates. (B) Superimposition of the chemical structures of <b>1</b>â<b>4</b>. Their 3D structures are very similar, with a good alignment of the tertiary amine nitrogen atoms and of the aromatic moieties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Arecoline (<b>5</b>) and its bioisoster alvameline (<b>6</b>). The former is an optimal substrate of FMO3, while the latter is not.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Phenethylamine (A), a FMO3 substrate, compared with tetrahydroisoquinoline (B), which is not a FMO3 substrate. The GRID MIF shows favorable energetic interactions with hydroperoxyflavin group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Experimental sites of metabolism (oxidation) for compounds <b>9</b>â<b>11</b> are marked with asterisks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compounds <b>12</b> (A) and <b>13</b> (B) are similar FMO3 substrates that react in a different manner. The former reacts at the sulfur atom of the methylsulfide and at the pyrrolidine nitrogen, while the latter reacts only at the methylsulfide moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Predicting the correct rank for sites of metabolism (SoM) in compounds <b>14</b>â<b>16</b> by FMO3 is not a simple task.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Bacterial (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7">2vq7</a> structure, A) and human (calculated structure, B) catalytic sites of FMO3. Amino acids that play an important role in substrate orientation are reported. Cavities are computed by FLAP software.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Metasite flowchart for FMO3 SoM calculation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Experimental vs recalculated nucleophilicity (in water) for 50 chemical entities from Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> (in blue rhombus). The remaining 65 compounds (red triangles) lack experimental value and are predicted from nucleophilicity model (see methods section). The nucleophilicity range is arbitrarily divided into four regions (thresholds), from not reactive (when it is very low) to low, medium, and high nucleophilicity values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0010.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Reactivity equalization. (A) Small substrates can interact with FAD in different orientations, exposure constraints being practically absent. When two or more oxidation sites are possible, their nucleophilicity values will determine the oxidation rate and the metaboliteâs abundance. (B) Large substrates can be strongly constrained because of several interactions with the cavity. In this case exposure, rather than nucleophilicity, plays a relevant role in determining the site of metabolism.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0011.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structures of voriconazole (<b>17</b>), sulfamethoxazole (<b>18</b>), dapsone (<b>19</b>), and imipramine (<b>20</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0012.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Comparison between predicted and experimental SoM for compounds <b>14</b>. (A) MetaSite SoM prediction for compound <b>14</b>. The highest oxidation probability (marked with 1) corresponds to the fastest N-oxidation reaction. S-oxidation is predicted to be the second most probable reaction (marked with 2). (B) In vitro metabolism study for compound <b>14</b>. The N-oxidized metabolite at the benzylamine position and the S-oxidized metabolites are formed. (C) Clearance determination for compound <b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0013.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Our experimental data show that bepridil is not an FMO3 substrate. MetaSite reports that this may be due to steric clash (A) between the isopropyloxy chain with the enzyme cavity. When the chain was synthetically removed, the new compound in (B) was a good FMO3 substrate with SoM in the predicted pyrrolidil nitrogen.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0014.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. (A) BPEA depletion and BPEA <i>N</i>-oxide formation in human FMO3 enzyme. (B) Clearance data for BPEA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/medium/jm-2014-007098_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0015.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm5007098/production/images/large/jm-2014-007098_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5007098&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 82 Â°C; (ii) Tf<sub>2</sub>O, DCM, rt; (iii) <i>N</i>-benzylaniline, NaH, 60% dispersion in mineral oil, toluene, 110 Â°C.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73465" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73465" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 61 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Cashman, J. R.</span><span> </span><span class="NLM_article-title">Human flavin-containing monooxygenase: substrate specificity and role in drug metabolism</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">â</span> <span class="NLM_lpage">191</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.2174%2F1389200003339135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=11465082" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntlOrs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2000&pages=181-191&author=J.+R.+Cashman&title=Human+flavin-containing+monooxygenase%3A+substrate+specificity+and+role+in+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Human flavin-containing monooxygenase: substrate specificity and role in drug metabolism</span></div><div class="casAuthors">Cashman, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review with 57 refs.  The human flavin-contg. monooxygenase (FMO3) is a prominent enzyme system that converts nucleophilic heteroatom-contg. chems., drugs and xenobiotics to more polar materials that are more efficiently excreted in the urine.  The substrate specificity for FMO3 is distinct from that of FMO1.  Human FMO3 N-oxygenates primary, secondary and tertiary amines whereas human FMO1 is only highly efficient at N-oxygenating tertiary amines.  Both human FMO1 and FMO3 S-oxygenate a no. of nucleophilic sulfur-contg. substrates and in some cases, does so with great stereoselectivity.  Human FMO3 is sensitive to steric features of the substrate and aliph. amines with linkages between the nitrogen atom and a large arom. group such as a phenothiazine of at least five carbons are N-oxygenated significantly more efficiently than those substrates with two or three carbons.  For amines with smaller arom. substituents such as phenethylamines, often these compds. are efficiently N-oxygenated by human FMO3.  Currently, the most promising non-invasive probe of in vivo human FMO3 functional activity is the formation of trimethylamine N-oxide from trimethylamine that comes from dietary choline.  (S)-Nicotine N-1'-oxide formation can also be used as a highly stereoselective probe of human FMO3 function for adult humans that smoke cigarettes.  Finally, cimetidine S-oxygenation or ranitidine N-oxidn. can also be used as a functional probe of human FMO3.  With the recent observation of human FMO3 genetic polymorphism and poor metab. phenotype in certain human populations, variant human FMO3 may contribute to adverse drug reactions or exaggerated clin. response to certain medications.  Knowledge of the substrate specificity for human FMO3 may aid in the future design of more efficacious and less toxic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3KYB29CjVo7Vg90H21EOLACvtfcHk0ljqhDh85rDs9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntlOrs7s%253D&md5=484ccf04bf0c5751015c2b9eb251954d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F1389200003339135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389200003339135%26sid%3Dliteratum%253Aachs%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26atitle%3DHuman%2520flavin-containing%2520monooxygenase%253A%2520substrate%2520specificity%2520and%2520role%2520in%2520drug%2520metabolism%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2000%26volume%3D1%26spage%3D181%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Cashman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span> </span><span class="NLM_article-title">Human flavin-containing monooxygenases</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">â</span> <span class="NLM_lpage">100</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=65-100&author=J.+R.+Cashmanauthor=J.+Zhang&title=Human+flavin-containing+monooxygenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DHuman%2520flavin-containing%2520monooxygenases%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2006%26volume%3D46%26spage%3D65%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Ziegler, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettit, F. H.</span><span> </span><span class="NLM_article-title">Microsomal oxidases. I. The isolation and dialkylarylamine oxygenase activity of pork liver microsomes</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1966</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2932</span><span class="NLM_x">â</span> <span class="NLM_lpage">2938</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi00873a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1966&pages=2932-2938&author=D.+M.+Zieglerauthor=F.+H.+Pettit&title=Microsomal+oxidases.+I.+The+isolation+and+dialkylarylamine+oxygenase+activity+of+pork+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fbi00873a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00873a024%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DD.%2BM.%26aulast%3DPettit%26aufirst%3DF.%2BH.%26atitle%3DMicrosomal%2520oxidases.%2520I.%2520The%2520isolation%2520and%2520dialkylarylamine%2520oxygenase%2520activity%2520of%2520pork%2520liver%2520microsomes%26jtitle%3DBiochemistry%26date%3D1966%26volume%3D5%26spage%3D2932%26epage%3D2938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Testa, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedretti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span> </span><span class="NLM_article-title">Reactions and enzymes in the metabolism of drugs and other xenobiotics</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">â</span> <span class="NLM_lpage">560</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1016%2Fj.drudis.2012.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=22305937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=549-560&author=B.+Testaauthor=A.+Pedrettiauthor=G.+Vistoli&title=Reactions+and+enzymes+in+the+metabolism+of+drugs+and+other+xenobiotics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Reactions and enzymes in the metabolism of drugs and other xenobiotics</span></div><div class="casAuthors">Testa, Bernard; Pedretti, Alessandro; Vistoli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In this article, we offer an overview of the compared quant. importance of biotransformation reactions in the metab. of drugs and other xenobiotics, based on a meta-anal. of current research interests.  Also, we assess the relative significance the enzyme (super)families or categories catalyzing these reactions.  We put the facts unveiled by the anal. into a drug discovery context and draw some implications.  The results confirm the primary role of cytochrome P 450-catalyzed oxidns. and UDP-glucuronosyl-catalyzed glucuronidations, but they also document the marked significance of several other reactions.  Thus, there is a need for several drug discovery scientists to better grasp the variety of drug metab. reactions and enzymes and their consequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbRR72SBUfcLVg90H21EOLACvtfcHk0ljqhDh85rDs9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D&md5=e11b5b1b7ea18e8bc817bff83efad456</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DPedretti%26aufirst%3DA.%26aulast%3DVistoli%26aufirst%3DG.%26atitle%3DReactions%2520and%2520enzymes%2520in%2520the%2520metabolism%2520of%2520drugs%2520and%2520other%2520xenobiotics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D549%26epage%3D560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Deng, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span> </span><span class="NLM_article-title">Pharmacokinetics, metabolism, and excretion of the antiviral drug arbidol in humans</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1743</span><span class="NLM_x">â</span> <span class="NLM_lpage">1755</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=1743-1755&author=P.+Dengauthor=D.+Zhongauthor=K.+Yuauthor=Y.+Zhangauthor=T.+Wangauthor=X.+Chen&title=Pharmacokinetics%2C+metabolism%2C+and+excretion+of+the+antiviral+drug+arbidol+in+humans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DPharmacokinetics%252C%2520metabolism%252C%2520and%2520excretion%2520of%2520the%2520antiviral%2520drug%2520arbidol%2520in%2520humans%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D1743%26epage%3D1755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Capolongo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anfossi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montesissa, C.</span><span> </span><span class="NLM_article-title">Benzydamine as a useful substrate of hepatic flavin-containing monooxygenase activity in veterinary species</span> <span class="citation_source-journal">J. Vet. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">â</span> <span class="NLM_lpage">346</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=341-346&author=F.+Capolongoauthor=A.+Santiauthor=P.+Anfossiauthor=C.+Montesissa&title=Benzydamine+as+a+useful+substrate+of+hepatic+flavin-containing+monooxygenase+activity+in+veterinary+species"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCapolongo%26aufirst%3DF.%26aulast%3DSanti%26aufirst%3DA.%26aulast%3DAnfossi%26aufirst%3DP.%26aulast%3DMontesissa%26aufirst%3DC.%26atitle%3DBenzydamine%2520as%2520a%2520useful%2520substrate%2520of%2520hepatic%2520flavin-containing%2520monooxygenase%2520activity%2520in%2520veterinary%2520species%26jtitle%3DJ.%2520Vet.%2520Pharmacol.%2520Ther.%26date%3D2009%26volume%3D33%26spage%3D341%26epage%3D346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Hanlon, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camattari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glieder, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kittelmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LÃ¼tz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wirz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, M.</span><span> </span><span class="NLM_article-title">Expression of recombinant human flavin monooxygenase and moclobemide-<i>N</i>-oxide synthesis on multi-mg scale</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6001</span><span class="NLM_x">â</span> <span class="NLM_lpage">6003</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=6001-6003&author=S.+P.+Hanlonauthor=A.+Camattariauthor=S.+Abadauthor=A.+Gliederauthor=M.+Kittelmannauthor=S.+L%C3%BCtzauthor=B.+Wirzauthor=M.+Winkler&title=Expression+of+recombinant+human+flavin+monooxygenase+and+moclobemide-N-oxide+synthesis+on+multi-mg+scale"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanlon%26aufirst%3DS.%2BP.%26aulast%3DCamattari%26aufirst%3DA.%26aulast%3DAbad%26aufirst%3DS.%26aulast%3DGlieder%26aufirst%3DA.%26aulast%3DKittelmann%26aufirst%3DM.%26aulast%3DL%25C3%25BCtz%26aufirst%3DS.%26aulast%3DWirz%26aufirst%3DB.%26aulast%3DWinkler%26aufirst%3DM.%26atitle%3DExpression%2520of%2520recombinant%2520human%2520flavin%2520monooxygenase%2520and%2520moclobemide-N-oxide%2520synthesis%2520on%2520multi-mg%2520scale%26jtitle%3DChem.%2520Commun.%26date%3D2012%26volume%3D48%26spage%3D6001%26epage%3D6003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Mushiroda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douya, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahara, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, O.</span><span> </span><span class="NLM_article-title">The involvement of flavin-containing monooxygenase but not CYP3A4 in metabolism of itopride hydrochloride, a gastroprokinetic agent: comparison with cisapride and mosapride citrate</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">â</span> <span class="NLM_lpage">1237</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=10997945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntVSlt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1231-1237&author=T.+Mushirodaauthor=R.+Douyaauthor=E.+Takaharaauthor=O.+Nagata&title=The+involvement+of+flavin-containing+monooxygenase+but+not+CYP3A4+in+metabolism+of+itopride+hydrochloride%2C+a+gastroprokinetic+agent%3A+comparison+with+cisapride+and+mosapride+citrate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The involvement of flavin-containing monooxygenase but not CYP3A4 in metabolism of itopride hydrochloride, a gastroprokinetic agent: comparison with cisapride and mosapride citrate</span></div><div class="casAuthors">Mushiroda, Taisei; Douya, Rika; Takahara, Eiji; Nagata, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1231-1237</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The goals of the present study were to identify the enzyme responsible for metab. of itopride hydrochloride (itopride) and to evaluate the likelihood of drug interaction involving itopride.  In human liver microsomes, the involvement of flavin-contg. monooxygenase in N-oxygenation, the major metabolic pathway of itopride, was indicated by the following results: inhibition by methimazole and thiourea, heat inactivation, and protection against heat inactivation by NADPH.  When the effects of ketoconazole on the metab. of itopride, cisapride, and mosapride citrate (mosapride) were examd. using human liver microsomes, ketoconazole strongly inhibited the formation of the primary metabolites of cisapride and mosapride, but not itopride.  Other cytochrome P 450 (CYP) 3A4 inhibitors, cimetidine, erythromycin, and clarithromycin, also inhibited the metab. of cisapride and mosapride.  In an in vivo study, itopride (30 mg/kg), cisapride (1.5 mg/kg), or mosapride (3 mg/kg) was orally administered to male rats with or without oral pretreatment with ketoconazole (120 mg/kg) twice daily for 2 days.  The ketoconazole pretreatment significantly increased the area under the serum concn. curve and the max. serum concn. of cisapride and mosapride but had no significant effect on the pharmacokinetics of itopride.  In addn., itopride did not inhibit five specific CYP-mediated reactions of human liver microsomes.  These results suggest that itopride is unlikely to alter the pharmacokinetics of other concomitantly administered drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPynvFuLvYVLVg90H21EOLACvtfcHk0lhhKpxvShy5Ug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntVSlt7o%253D&md5=c141c96269304302f08d27e2d7d08371</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMushiroda%26aufirst%3DT.%26aulast%3DDouya%26aufirst%3DR.%26aulast%3DTakahara%26aufirst%3DE.%26aulast%3DNagata%26aufirst%3DO.%26atitle%3DThe%2520involvement%2520of%2520flavin-containing%2520monooxygenase%2520but%2520not%2520CYP3A4%2520in%2520metabolism%2520of%2520itopride%2520hydrochloride%252C%2520a%2520gastroprokinetic%2520agent%253A%2520comparison%2520with%2520cisapride%2520and%2520mosapride%2520citrate%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2000%26volume%3D28%26spage%3D1231%26epage%3D1237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Chung, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cha, Y. N.</span><span> </span><span class="NLM_article-title">Oxidation of ranitidine by isozymes of flavin-containing monooxygenase and cytochrome P450</span> <span class="citation_source-journal">Jpn. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">220</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1254%2Fjjp.84.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=11128045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGgt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2000&pages=213-220&author=W.+G.+Chungauthor=C.+S.+Parkauthor=H.+K.+Rohauthor=W.+K.+Leeauthor=Y.+N.+Cha&title=Oxidation+of+ranitidine+by+isozymes+of+flavin-containing+monooxygenase+and+cytochrome+P450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidation of ranitidine by isozymes of flavin-containing monooxygenase and cytochrome P450</span></div><div class="casAuthors">Chung, Woon-Gye; Park, Chang-Shin; Roh, Hyung-Keun; Lee, Woon-Kee; Cha, Young-Nam</div><div class="citationInfo"><span class="NLM_cas:title">Japanese Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-220</span>CODEN:
                <span class="NLM_cas:coden">JJPAAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-5198</span>.
    
            (<span class="NLM_cas:orgname">Japanese Pharmacological Society</span>)
        </div><div class="casAbstract">Rat and human liver microsomes oxidized ranitidine to its N-oxide (66-76%) and S-oxide (13-18%) and desmethylranitidine (12-16%).  N- and S-oxidns. of ranitidine were inhibited by metimazole [flavin-contg. monooxygenase (FMO) inhibitor] to 96-97% and 71-85%, resp., and desmethylation of ranitidine was inhibited by SKF525A [cytochrome P 450 (CYP) inhibitor] by 71-95%.  Recombinant FMO isoenzymes like FMO1, FMO2, FMO3 and FMO5 produced 39, 79, 2180 and 4 ranitidine N-oxide and 45, 0, 580 and 280 ranitidine S-oxide pmolÂ·min-1Â·nmol-1 FMO, resp.  Desmethylranitidine was not produced by recombinant FMOs.  Prodn. of desmethylranitidine by rat and human liver microsomes was inhibited by tranylcypromine, Î±-naphthoflavon and quinidine, which are known to inhibit CYP2C19, 1A2 and 2D6, resp.  FMO3, the major form in adult liver, produced both ranitidine N- and S-oxides at a 4 to 1 ratio.  FMO1, expressed primarily in human kidney, was 55- and 13-fold less efficient than the hepatic FMO3 in producing ranitidine N- and S-oxides, resp.  FMO2 and FMO5, although expressed slightly in human liver, kidney and lung, were not efficient producers of ranitidine N- and S-oxides.  Thus, urinary contents of ranitidine N-oxide can be used as the in vivo probe to det. the hepatic FMO3 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF4YBYHNhk17Vg90H21EOLACvtfcHk0lhhKpxvShy5Ug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGgt7w%253D&md5=48e92a754083d443e376b27f5425c630</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1254%2Fjjp.84.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjjp.84.213%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DW.%2BG.%26aulast%3DPark%26aufirst%3DC.%2BS.%26aulast%3DRoh%26aufirst%3DH.%2BK.%26aulast%3DLee%26aufirst%3DW.%2BK.%26aulast%3DCha%26aufirst%3DY.%2BN.%26atitle%3DOxidation%2520of%2520ranitidine%2520by%2520isozymes%2520of%2520flavin-containing%2520monooxygenase%2520and%2520cytochrome%2520P450%26jtitle%3DJpn.%2520J.%2520Pharmacol.%26date%3D2000%26volume%3D84%26spage%3D213%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">SÃ¶derberg, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahl, M. L.</span><span> </span><span class="NLM_article-title">Pharmacogenetics of olanzapine metabolism</span> <span class="citation_source-journal">Pharmacogenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1319</span><span class="NLM_x">â</span> <span class="NLM_lpage">1336</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=1319-1336&author=M.+M.+S%C3%B6derbergauthor=M.+L.+Dahl&title=Pharmacogenetics+of+olanzapine+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25B6derberg%26aufirst%3DM.%2BM.%26aulast%3DDahl%26aufirst%3DM.%2BL.%26atitle%3DPharmacogenetics%2520of%2520olanzapine%2520metabolism%26jtitle%3DPharmacogenomics%26date%3D2013%26volume%3D14%26spage%3D1319%26epage%3D1336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lohr, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willsky, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acara, M. A.</span><span> </span><span class="NLM_article-title">Renal drug metabolism</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">107</span><span class="NLM_x">â</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1998&pages=107-141&author=J.+W.+Lohrauthor=G.+R.+Willskyauthor=M.+A.+Acara&title=Renal+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLohr%26aufirst%3DJ.%2BW.%26aulast%3DWillsky%26aufirst%3DG.%2BR.%26aulast%3DAcara%26aufirst%3DM.%2BA.%26atitle%3DRenal%2520drug%2520metabolism%26jtitle%3DPharmacol.%2520Rev.%26date%3D1998%26volume%3D50%26spage%3D107%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Ring, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrighton, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aldridge, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haehner, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipley, L. A.</span><span> </span><span class="NLM_article-title">Flavin-containing monooxygenase-mediated N-oxidation of the M(1)-muscarinic agonist xanomeline</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1099</span><span class="NLM_x">â</span> <span class="NLM_lpage">1103</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1099-1103&author=B.+J.+Ringauthor=S.+A.+Wrightonauthor=S.+L.+Aldridgeauthor=K.+Hansenauthor=B.+Haehnerauthor=L.+A.+Shipley&title=Flavin-containing+monooxygenase-mediated+N-oxidation+of+the+M%281%29-muscarinic+agonist+xanomeline"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRing%26aufirst%3DB.%2BJ.%26aulast%3DWrighton%26aufirst%3DS.%2BA.%26aulast%3DAldridge%26aufirst%3DS.%2BL.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DHaehner%26aufirst%3DB.%26aulast%3DShipley%26aufirst%3DL.%2BA.%26atitle%3DFlavin-containing%2520monooxygenase-mediated%2520N-oxidation%2520of%2520the%2520M%25281%2529-muscarinic%2520agonist%2520xanomeline%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1099%26epage%3D1103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Cashman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Celestial, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newdoll, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. B.</span><span> </span><span class="NLM_article-title">Tertiary amines related to brompheniramine: preferred conformations for N-oxygenation by the hog liver flavin-containing monooxygenase</span> <span class="citation_source-journal">Pharm. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1097</span><span class="NLM_x">â</span> <span class="NLM_lpage">1105</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1993&pages=1097-1105&author=J.+R.+Cashmanauthor=J.+R.+Celestialauthor=A.+Leachauthor=J.+Newdollauthor=S.+B.+Park&title=Tertiary+amines+related+to+brompheniramine%3A+preferred+conformations+for+N-oxygenation+by+the+hog+liver+flavin-containing+monooxygenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26aulast%3DCelestial%26aufirst%3DJ.%2BR.%26aulast%3DLeach%26aufirst%3DA.%26aulast%3DNewdoll%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DS.%2BB.%26atitle%3DTertiary%2520amines%2520related%2520to%2520brompheniramine%253A%2520preferred%2520conformations%2520for%2520N-oxygenation%2520by%2520the%2520hog%2520liver%2520flavin-containing%2520monooxygenase%26jtitle%3DPharm.%2520Res.%26date%3D1993%26volume%3D10%26spage%3D1097%26epage%3D1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hernandez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janmohamed, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omar, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shephard, E. A.</span><span> </span><span class="NLM_article-title">Deletion of the mouse Fmo1 gene results in enhanced pharmacological behavioural responses to imipramine</span> <span class="citation_source-journal">Pharmacogenet. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">289</span><span class="NLM_x">â</span> <span class="NLM_lpage">299</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1097%2FFPC.0b013e328328d507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=19262426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtF2isLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=289-299&author=D.+Hernandezauthor=A.+Janmohamedauthor=P.+Chandanauthor=B.+A.+Omarauthor=I.+R.+Phillipsauthor=E.+A.+Shephard&title=Deletion+of+the+mouse+Fmo1+gene+results+in+enhanced+pharmacological+behavioural+responses+to+imipramine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of the mouse Fmo1 gene results in enhanced pharmacological behavioural responses to imipramine</span></div><div class="casAuthors">Hernandez, Diana; Janmohamed, Azara; Chandan, Pritpal; Omar, Bilal A.; Phillips, Ian R.; Shephard, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacogenetics and Genomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">289-299</span>CODEN:
                <span class="NLM_cas:coden">PGHEAI</span>;
        ISSN:<span class="NLM_cas:issn">1744-6872</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Many drugs are the subject of multipathway oxidative metab. catalyzed by one or more cytochromes P 450 or flavin-contg. monooxygenases (FMOs).  This complicates assessment of the role of individual enzymes in metabolizing the drug and, hence, in understanding its pharmacogenetics.  To define the role of FMOs in drug metab., we produced FMO-deficient mice.  An Fmo1(-/-), Fmo2(-/-), Fmo4(-/-) mouse line was produced by using chromosomal engineering and Cre-loxP technol.  To assess the utility of the mutant mouse line, it was used to investigate the role of FMO in the metab. of and response to the antidepressant imipramine, which has four major metabolites, three produced by cytochromes P 450 and one, imipramine N-oxide, solely by FMO1.  On treatment with imipramine, wild-type mice became sedated and produced imipramine N-oxide in the brain and other tissues.  In contrast, knockout mice did not produce imipramine N-oxide, but showed exaggerated pharmacol. behavioral responses, such as tremor and body spasm, and had a higher concn. of the parent compd. imipramine in the serum and kidney and there was an increase in desipramine in the brain.  The absence of FMO1-mediated N-oxidn. of imipramine results in enhanced central nervous system effects of the drug.  The results provide insights into the metab. of imipramine in the brain and may explain the basis of the adverse reactions to the drug seen in some patients.  The knockout mouse line will provide a valuable resource for defining the role of FMO1 in the metab. of drugs and other foreign chems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowFKYAzxxWGbVg90H21EOLACvtfcHk0lgNg1yM9Sb4Dg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtF2isLw%253D&md5=99fcc555eb163b0c27b834fec6ec042c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1097%2FFPC.0b013e328328d507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FFPC.0b013e328328d507%26sid%3Dliteratum%253Aachs%26aulast%3DHernandez%26aufirst%3DD.%26aulast%3DJanmohamed%26aufirst%3DA.%26aulast%3DChandan%26aufirst%3DP.%26aulast%3DOmar%26aufirst%3DB.%2BA.%26aulast%3DPhillips%26aufirst%3DI.%2BR.%26aulast%3DShephard%26aufirst%3DE.%2BA.%26atitle%3DDeletion%2520of%2520the%2520mouse%2520Fmo1%2520gene%2520results%2520in%2520enhanced%2520pharmacological%2520behavioural%2520responses%2520to%2520imipramine%26jtitle%3DPharmacogenet.%2520Genomics%26date%3D2009%26volume%3D19%26spage%3D289%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Adali, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpot, R. M.</span><span> </span><span class="NLM_article-title">Modulation of human flavin-containing monooxygenase 3 activity by tricyclic antidepressants and other agents: importance of residue 428</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">358</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">â</span> <span class="NLM_lpage">97</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=1998&pages=92-97&author=O.+Adaliauthor=G.+C.+Carverauthor=R.+M.+Philpot&title=Modulation+of+human+flavin-containing+monooxygenase+3+activity+by+tricyclic+antidepressants+and+other+agents%3A+importance+of+residue+428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAdali%26aufirst%3DO.%26aulast%3DCarver%26aufirst%3DG.%2BC.%26aulast%3DPhilpot%26aufirst%3DR.%2BM.%26atitle%3DModulation%2520of%2520human%2520flavin-containing%2520monooxygenase%25203%2520activity%2520by%2520tricyclic%2520antidepressants%2520and%2520other%2520agents%253A%2520importance%2520of%2520residue%2520428%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1998%26volume%3D358%26spage%3D92%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Akutsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurada, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikegaya, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furihata, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiba, K.</span><span> </span><span class="NLM_article-title">Identification of human cytochrome p450 isozymes involved in diphenhydramine N-demethylation</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">â</span> <span class="NLM_lpage">78</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=72-78&author=T.+Akutsuauthor=K.+Kobayashiauthor=K.+Sakuradaauthor=H.+Ikegayaauthor=T.+Furihataauthor=K.+Chiba&title=Identification+of+human+cytochrome+p450+isozymes+involved+in+diphenhydramine+N-demethylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAkutsu%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DK.%26aulast%3DSakurada%26aufirst%3DK.%26aulast%3DIkegaya%26aufirst%3DH.%26aulast%3DFurihata%26aufirst%3DT.%26aulast%3DChiba%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520human%2520cytochrome%2520p450%2520isozymes%2520involved%2520in%2520diphenhydramine%2520N-demethylation%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2007%26volume%3D35%26spage%3D72%26epage%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ziegler, D. M.</span><span> </span><span class="NLM_article-title">Flavin-containing monooxygenases: catalytic mechanism and substrate specificities</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">â</span> <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.3109%2F03602538809049617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=3293953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaL1cXkvVyntL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1988&pages=1-32&author=D.+M.+Ziegler&title=Flavin-containing+monooxygenases%3A+catalytic+mechanism+and+substrate+specificities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Flavin-containing monooxygenases:  catalytic mechanism and substrate specificities</span></div><div class="casAuthors">Ziegler, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-32</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    </div><div class="casAbstract">A review, with 104 refs., on flavin-contg. monooxygenases with emphasis on the species and tissue enzyme variants, the catalytic mechanism, substrate specificity, assay methods, and the monooxygenase's role in metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1v1Qqf-cp2bVg90H21EOLACvtfcHk0lgNg1yM9Sb4Dg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXkvVyntL4%253D&md5=fad68367091f3761706798f3247dc623</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3109%2F03602538809049617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602538809049617%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DD.%2BM.%26atitle%3DFlavin-containing%2520monooxygenases%253A%2520catalytic%2520mechanism%2520and%2520substrate%2520specificities%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D1988%26volume%3D19%26spage%3D1%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Krueger, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. E.</span><span> </span><span class="NLM_article-title">Mammalian flavin-containing monooxygenases: structure/function, genetic polymorphisms and role in drug metabolism</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">â</span> <span class="NLM_lpage">387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1016%2Fj.pharmthera.2005.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=15922018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1SltLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2005&pages=357-387&author=S.+K.+Kruegerauthor=D.+E.+Williams&title=Mammalian+flavin-containing+monooxygenases%3A+structure%2Ffunction%2C+genetic+polymorphisms+and+role+in+drug+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian flavin-containing monooxygenases: structure/function, genetic polymorphisms and role in drug metabolism</span></div><div class="casAuthors">Krueger, Sharon K.; Williams, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">357-387</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Flavin-contg. monooxygenase (FMO) oxygenates drugs and xenobiotics contg. a "soft nucleophile", usually N or S.  FMO, like cytochrome P 450 (CYP), is a monooxygenase, utilizing the reducing equiv. of NADPH to reduce 1 atom of O2 to H2O, while the other atom is used to oxidize the substrate.  FMO and CYP also exhibit similar tissue and cellular location, mol. wt., substrate specificity, and exist as multiple enzymes under developmental control.  The human FMO functional gene family is much smaller (5 families each with a single member) than CYP.  FMO does not require a reductase to transfer electrons from NADPH and the catalytic cycle of the 2 monooxygenases is strikingly different.  Another distinction is the lack of induction of FMOs by xenobiotics.  In general, CYP is the major contributor to oxidative xenobiotic metab.  However, FMO activity may be of significance in a no. of cases and should not be overlooked.  FMO and CYP have overlapping substrate specificities, but often yield distinct metabolites with potentially significant toxicol./pharmacol. consequences.  The physiol. function(s) of FMOs are poorly understood.  Three of the 5 expressed human FMO genes, FMO1, FMO2, and FMO3, exhibit genetic polymorphisms.  The most studied of these is FMO3 (adult human liver) in which mutant alleles contribute to the disease known as trimethylaminuria.  The consequences of these FMO genetic polymorphisms in drug metab. and human health are areas of research requiring further exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfwDBzxhgaYbVg90H21EOLACvtfcHk0ljtkmcOKU-XKQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1SltLc%253D&md5=2f0b76495aa8a53069bacc9b4ae8de43</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2005.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2005.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DKrueger%26aufirst%3DS.%2BK.%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26atitle%3DMammalian%2520flavin-containing%2520monooxygenases%253A%2520structure%252Ffunction%252C%2520genetic%2520polymorphisms%2520and%2520role%2520in%2520drug%2520metabolism%26jtitle%3DPharmacol.%2520Ther.%26date%3D2005%26volume%3D106%26spage%3D357%26epage%3D387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccarelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonelli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">Modeling phospholipidosis induction: reliability and warnings</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1436</span><span class="NLM_x">â</span> <span class="NLM_lpage">1446</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci400113t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVKmu7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1436-1446&author=L.+Goracciauthor=M.+Ceccarelliauthor=D.+Bonelliauthor=G.+Cruciani&title=Modeling+phospholipidosis+induction%3A+reliability+and+warnings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Phospholipidosis Induction: Reliability and Warnings</span></div><div class="casAuthors">Goracci, Laura; Ceccarelli, Martina; Bonelli, Daniela; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1436-1446</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-induced phospholipidosis (PLD) is characterized by accumulation of phospholipids, the inducing drugs and lamellar inclusion bodies in the lysosomes of affected tissues.  These side effects must be considered as early as possible during drug discovery, and, in fact, numerous in silico models designed to predict PLD have been published.  However, the quality of any in silico model cannot be better than the quality of the exptl. data set used to build it.  The present paper reports an overview of the difficulties and errors encountered in the generation of databases used for the published PLD models.  A new database of 466 compds. was constructed from seven literature sources, contg. only publicly available compds.  A comparison of the PLD assignations in selected databases proved useful in revealing some inconsistencies and raised doubts about the previously assigned PLD+ and PLD- classifications for some chems.  Finally, a Partial Least Squares Discriminant Anal. (PLS-DA) approach was also applied, revealing further anomalies and clearly showing that metab. as well as data quality must be taken into account when generating accurate methods for predicting the likelihood that a compd. will induce PLD.  A new curated database of 331 compds. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruk5nZE3GTi7Vg90H21EOLACvtfcHk0ljtkmcOKU-XKQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVKmu7w%253D&md5=b4134aa063aa8a4abddb525840faba4b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci400113t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400113t%26sid%3Dliteratum%253Aachs%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DCeccarelli%26aufirst%3DM.%26aulast%3DBonelli%26aufirst%3DD.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DModeling%2520phospholipidosis%2520induction%253A%2520reliability%2520and%2520warnings%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1436%26epage%3D1446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Orogo, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minnier, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruhlak, N. L.</span><span> </span><span class="NLM_article-title">Construction and consensus performance of (Q)SAR models for predicting phospholipidosis using a dataset of 743 compounds</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">â</span> <span class="NLM_lpage">739</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2Fminf.201200048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlajsbvI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=725-739&author=A.+M.+Orogoauthor=S.+S.+Choiauthor=B.+L.+Minnierauthor=N.+L.+Kruhlak&title=Construction+and+consensus+performance+of+%28Q%29SAR+models+for+predicting+phospholipidosis+using+a+dataset+of+743+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Construction and Consensus Performance of (Q)SAR Models for Predicting Phospholipidosis Using a Dataset of 743 Compounds</span></div><div class="casAuthors">Orogo, Amabel M.; Choi, Sydney S.; Minnier, Barbara L.; Kruhlak, Naomi L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">725-739</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Drug-induced phospholipidosis (PLD) continues to be a safety concern for pharmaceutical companies and regulatory agencies, prompting the FDA/CDER Phospholipidosis Working Group to develop a database of PLD findings that was recently expanded to contain a total of 743 compds. (385 pos. and 358 neg.).  Three com. (quant.) structure-activity relationship [(Q)SAR] software platforms [MC4PC, Leadscope Predictive Data Miner (LPDM), and Derek for Windows (DfW)] were used to build and/or test models with the database and evaluated individually and together for their ability to predict PLD induction.  Models constructed with MC4PC showed improved sensitivity over previous models constructed with an earlier version of the database and software (61.2 % vs. 50.0 %), but lower specificity in cross-validation expts. (58.2 % vs. 91.9 %) due in part to the more balanced ratio of positives to negatives in the training set.  A new model created with LPDM gave good cross-validation statistics (79.0 % sensitivity, 78.0 % specificity) and the single DfW structural alert for PLD was found to have high pos. predictivity (83.3 %) but low sensitivity (10.4 %) when tested with the entire PLD database.  Combining the predictions of MC4PC, LPDM and/or DfW resulted in increased sensitivity and coverage over using one software platform alone, although it did not enhance the overall prediction accuracy beyond that of the best performing individual software platform.  The comparison across software platforms, however, facilitated the identification and anal. of chems. that were consistently predicted incorrectly by all platforms.  The prevalence of cationic amphiphilic drug (CAD) structural motifs in the database contributed heavily to many of the structural alerts and discriminating features in the models, but the subset of incorrectly predicted structures across all models underscores the need to account for mitigating features and/or addnl. filtering criteria to assess PLD, in particular for PLD-inducing non-CADs and non-PLD-inducing CADs.  (Q)SAR tools may be used as part of an early screening battery or regulatory risk assessment approach to identify those compds. with the greatest chance of inducing PLD and potentially toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmr8bi8YXcbLVg90H21EOLACvtfcHk0ljtkmcOKU-XKQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlajsbvI&md5=2889c571bbbf0d04193690a481cbc3e8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fminf.201200048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201200048%26sid%3Dliteratum%253Aachs%26aulast%3DOrogo%26aufirst%3DA.%2BM.%26aulast%3DChoi%26aufirst%3DS.%2BS.%26aulast%3DMinnier%26aufirst%3DB.%2BL.%26aulast%3DKruhlak%26aufirst%3DN.%2BL.%26atitle%3DConstruction%2520and%2520consensus%2520performance%2520of%2520%2528Q%2529SAR%2520models%2520for%2520predicting%2520phospholipidosis%2520using%2520a%2520dataset%2520of%2520743%2520compounds%26jtitle%3DMol.%2520Inf.%26date%3D2012%26volume%3D31%26spage%3D725%26epage%3D739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geraci, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argikar, U.</span><span> </span><span class="NLM_article-title">Predicting phospholipidosis: a fluorescence non cell based in vitro assay for the determination of drug phospholipid complex formation in early drug discovery</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">6980</span><span class="NLM_x">â</span> <span class="NLM_lpage">6987</span></span><div class="citationLinks">[<a href="/doi/10.1021/ac200683k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2011&pages=6980-6987&author=L.+Zhouauthor=G.+Geraciauthor=S.+Hessauthor=L.+Yangauthor=J.+Wangauthor=U.+Argikar&title=Predicting+phospholipidosis%3A+a+fluorescence+non+cell+based+in+vitro+assay+for+the+determination+of+drug+phospholipid+complex+formation+in+early+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fac200683k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac200683k%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DGeraci%26aufirst%3DG.%26aulast%3DHess%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DArgikar%26aufirst%3DU.%26atitle%3DPredicting%2520phospholipidosis%253A%2520a%2520fluorescence%2520non%2520cell%2520based%2520in%2520vitro%2520assay%2520for%2520the%2520determination%2520of%2520drug%2520phospholipid%2520complex%2520formation%2520in%2520early%2520drug%2520discovery%26jtitle%3DAnal.%2520Chem.%26date%3D2011%26volume%3D83%26spage%3D6980%26epage%3D6987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Quaglino, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruttomesso, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bigler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Follath, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Realdi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettenazzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baritussio, A.</span><span> </span><span class="NLM_article-title">Effects of metabolites and analogs of amiodarone on alveolar macrophages: structureâactivity relationship</span> <span class="citation_source-journal">Am. J. Physiol.: Lung Cell. Mol. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="NLM_x">â</span> <span class="NLM_lpage">447</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1152%2Fajplung.00434.2003" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2004&pages=438-447&author=D.+Quaglinoauthor=H.+R.+Haauthor=E.+Dunerauthor=D.+Bruttomessoauthor=L.+Biglerauthor=F.+Follathauthor=G.+Realdiauthor=A.+Pettenazzoauthor=A.+Baritussio&title=Effects+of+metabolites+and+analogs+of+amiodarone+on+alveolar+macrophages%3A+structure%E2%80%93activity+relationship"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1152%2Fajplung.00434.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajplung.00434.2003%26sid%3Dliteratum%253Aachs%26aulast%3DQuaglino%26aufirst%3DD.%26aulast%3DHa%26aufirst%3DH.%2BR.%26aulast%3DDuner%26aufirst%3DE.%26aulast%3DBruttomesso%26aufirst%3DD.%26aulast%3DBigler%26aufirst%3DL.%26aulast%3DFollath%26aufirst%3DF.%26aulast%3DRealdi%26aufirst%3DG.%26aulast%3DPettenazzo%26aufirst%3DA.%26aulast%3DBaritussio%26aufirst%3DA.%26atitle%3DEffects%2520of%2520metabolites%2520and%2520analogs%2520of%2520amiodarone%2520on%2520alveolar%2520macrophages%253A%2520structure%25E2%2580%2593activity%2520relationship%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Lung%2520Cell.%2520Mol.%2520Physiol.%26date%3D2004%26volume%3D287%26spage%3D438%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Shiraga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaneko, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tozuka, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hata, T.</span><span> </span><span class="NLM_article-title">Identification of cytochrome P450 enzymes involved in the metabolism of zotepine, an antipsychotic drug, in human liver microsomes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">â</span> <span class="NLM_lpage">229</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1080%2F004982599238623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=10219963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaK1MXitleitLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1999&pages=217-229&author=T.+Shiragaauthor=H.+Kanekoauthor=K.+Iwasakiauthor=Z.+Tozukaauthor=A.+Suzukiauthor=T.+Hata&title=Identification+of+cytochrome+P450+enzymes+involved+in+the+metabolism+of+zotepine%2C+an+antipsychotic+drug%2C+in+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of cytochrome P450 enzymes involved in the metabolism of zotepine, an antipsychotic drug, in human liver microsomes</span></div><div class="casAuthors">Shiraga, T.; Kaneko, H.; Iwasaki, K.; Tozuka, Z.; Suzuki, A.; Hata, T.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">217-229</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">1. Studies using human liver microsomes and recombinant human cytochrome P 450 (P 450) enzymes and flavin-contg. monooxygenase (FMO) were performed to identify the enzymes responsible for the formation of zotepine metabolites in man.  2. Human liver microsomes produced four metabolites and a tentative order of importance was: norzotepine, 3-hydroxyzotepine, zotepine S-oxide and 2-hydroxyzotepine.  Zotepine N-oxide was also detected, but it could not be quantified.  3. The rates of formation of the major metabolite, norzotepine, and zotepine S-oxide (at a substrate concn. of 20 Î¼M) were significantly correlated with the testosterone 6Î²-hydroxylase activities and CYP3A4 contents of the 12 different human liver microsomal samples.  Inhibition studies with P 450 enzyme selective inhibitors and anti-rat CYP3A2 antibodies also indicated a predominant role of CYP3A4 in the formation of norzotepine and zotepine S-oxide.  Furafylline and sulphaphenazole inhibited the N-demethylation of zotepine by up to â¼ 30%.  4. Correlation and inhibition data for the 2- and 3-hydroxylation of zotepine were consistent with the predominant role of CYP1A2 and 2D6 in the formation of these metabolites, resp.  5. Recombinant CYP1A1, 1A2, 2B6, 2C19, 3A4 and 3A5 efficiently catalyzed N-demethylation of zotepine.  CYP1A1, 1A2, 2B6 and 3A4 were also active for S-oxidn.  CYP1A2 and 2D6*1-Val374 efficiently produced 2-hydroxyzotepine and 3-hyroxyzotepine, resp.  Recombinant human FMO3 did not catalyze zotepine S-oxidn.  6. These results suggest that both the N-demethylation and S-oxidn. of zotepine are mediated mainly by CYP3A4, and that CYP1A2 and 2D6 play an important role in the 2- and 3-hydroxylation of zotepine, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3CrJhdh2-cLVg90H21EOLACvtfcHk0limAgi-o6LUnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitleitLw%253D&md5=bac2a7fbba2981b64e5b0ab5295377f6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F004982599238623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F004982599238623%26sid%3Dliteratum%253Aachs%26aulast%3DShiraga%26aufirst%3DT.%26aulast%3DKaneko%26aufirst%3DH.%26aulast%3DIwasaki%26aufirst%3DK.%26aulast%3DTozuka%26aufirst%3DZ.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DHata%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520cytochrome%2520P450%2520enzymes%2520involved%2520in%2520the%2520metabolism%2520of%2520zotepine%252C%2520an%2520antipsychotic%2520drug%252C%2520in%2520human%2520liver%2520microsomes%26jtitle%3DXenobiotica%26date%3D1999%26volume%3D29%26spage%3D217%26epage%3D229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Giri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krausz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Idle, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, F. J.</span><span> </span><span class="NLM_article-title">The metabolomics of (+/â)-arecoline 1-oxide in the mouse and its formation by human flavin-containing monooxygenases</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">â</span> <span class="NLM_lpage">573</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2007&pages=561-573&author=S.+Giriauthor=K.+W.+Krauszauthor=J.+R.+Idleauthor=F.+J.+Gonzalez&title=The+metabolomics+of+%28%2B%2F%E2%88%92%29-arecoline+1-oxide+in+the+mouse+and+its+formation+by+human+flavin-containing+monooxygenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGiri%26aufirst%3DS.%26aulast%3DKrausz%26aufirst%3DK.%2BW.%26aulast%3DIdle%26aufirst%3DJ.%2BR.%26aulast%3DGonzalez%26aufirst%3DF.%2BJ.%26atitle%3DThe%2520metabolomics%2520of%2520%2528%252B%252F%25E2%2588%2592%2529-arecoline%25201-oxide%2520in%2520the%2520mouse%2520and%2520its%2520formation%2520by%2520human%2520flavin-containing%2520monooxygenases%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2007%26volume%3D73%26spage%3D561%26epage%3D573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Jensen, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgaard, L.</span><span> </span><span class="NLM_article-title">In vitro metabolism of the m1-muscarinic agonist 5-(2-ethyl-2<i>H</i>-tetrazol-5-yl)-1-methyl-1,2,3,6-tetrahydropyridine by human hepatic cytochromes p-450 determined at pH 7.4 and 8.5</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=125-132&author=K.+G.+Jensenauthor=L.+Dalgaard&title=In+vitro+metabolism+of+the+m1-muscarinic+agonist+5-%282-ethyl-2H-tetrazol-5-yl%29-1-methyl-1%2C2%2C3%2C6-tetrahydropyridine+by+human+hepatic+cytochromes+p-450+determined+at+pH+7.4+and+8.5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DK.%2BG.%26aulast%3DDalgaard%26aufirst%3DL.%26atitle%3DIn%2520vitro%2520metabolism%2520of%2520the%2520m1-muscarinic%2520agonist%25205-%25282-ethyl-2H-tetrazol-5-yl%2529-1-methyl-1%252C2%252C3%252C6-tetrahydropyridine%2520by%2520human%2520hepatic%2520cytochromes%2520p-450%2520determined%2520at%2520pH%25207.4%2520and%25208.5%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D125%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cashman, J. R.</span><span> </span><span class="NLM_article-title">N-oxygenation of phenethylamine to the <i>trans</i>-oxime by adult human liver flavin-containing monooxygenase and retroreduction of phenethylamine hydroxylamine by human liver microsomes</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">1269</span><span class="NLM_x">â</span> <span class="NLM_lpage">1279</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=9316835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaK2sXmsVGlt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1997&pages=1269-1279&author=J.+Linauthor=J.+R.+Cashman&title=N-oxygenation+of+phenethylamine+to+the+trans-oxime+by+adult+human+liver+flavin-containing+monooxygenase+and+retroreduction+of+phenethylamine+hydroxylamine+by+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">N-oxygenation of phenethylamine to the trans-oxime by adult human liver flavin-containing monooxygenase and retroreduction of phenethylamine hydroxylamine by human liver microsomes</span></div><div class="casAuthors">Lin, Jing; Cashman, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1269-1279</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">The biogenic amine phenethylamine has been shown to be N-oxygenated by human flavin-contg. monooxygenase (FMO) (form 3) and human liver microsomes and, to a much lesser extent, N-oxygenated by porcine liver FMO1 and porcine liver microsomes but not by rabbit FMO2.  Adult human liver microsomes catalyze the NADPH-dependent N-oxygenation of phenethylamine to the corresponding trans-oxime through the intermediacy of phenethyl hydroxylamine.  In addn. to trans-oxime formation, phenethyl hydroxylamine is retroreduced to phenethylamine in the presence of human or porcine liver microsomes.  Studies on the biochem. mechanism of N-oxygenation suggested that trans-oxime formation was dependent on the human FMO (form 3) and that retroredn. was stimulated by superoxide and dependent on a cytochrome P 450 system.  These conclusions are based on studies examg. the effects of incubation conditions on phenethylamine N-oxygenation and the effect of reactive oxygen species on phenethyl hydroxylamine retroredn., resp.  The pharmacol. activity of synthetic phenethyl hydroxylamine and phenethyl oxime with a no. of biogenic amine receptors and transporters was examd. in vitro.  In all cases examd., the affinity of phenethyl hydroxylamine and the corresponding oxime for a biogenic transporter or receptors was very poor.  The results suggest that the biogenic amine phenethylamine is efficiently sequentially N-oxygenated in the presence of human liver microsomes or cDNA-expressed FMO (form 3) to phenethyl hydroxylamine and then to oximes that are pharmacol. inactive and serve to terminate biol. activity.  N-oxygenation of phenethylamine to the corresponding trans-oxime is a detoxication process that abrogates pharmacol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwngd9RyWOLVg90H21EOLACvtfcHk0lghkIR0r-Qk2A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsVGlt7w%253D&md5=3f35e2087db2d748ca7b49ffb3f42040</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26atitle%3DN-oxygenation%2520of%2520phenethylamine%2520to%2520the%2520trans-oxime%2520by%2520adult%2520human%2520liver%2520flavin-containing%2520monooxygenase%2520and%2520retroreduction%2520of%2520phenethylamine%2520hydroxylamine%2520by%2520human%2520liver%2520microsomes%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1997%26volume%3D282%26spage%3D1269%26epage%3D1279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">Hydrogen bonding interactions of covalently bonded fluorine atoms: from crystallographic data to a new angular function in the GRID force field</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5114</span><span class="NLM_x">â</span> <span class="NLM_lpage">5125</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0498349" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFGhs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5114-5125&author=E.+Carosatiauthor=S.+Sciabolaauthor=G.+Cruciani&title=Hydrogen+bonding+interactions+of+covalently+bonded+fluorine+atoms%3A+from+crystallographic+data+to+a+new+angular+function+in+the+GRID+force+field"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Bonding Interactions of Covalently Bonded Fluorine Atoms: From Crystallographic Data to a New Angular Function in the GRID Force Field</span></div><div class="casAuthors">Carosati, Emanuele; Sciabola, Simone; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5114-5125</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through the years the GRID force field has been tuned to fit exptl. observations in crystal structures.  This paper describes the detn. of the hydrogen bonding pattern for org. fluorines based on an exhaustive inspection of the Protein Data Bank.  All the PDB complexes, whose protein structures have cocrystd. fluorine-contg. ligands, were examd. and geometrically inspected.  By applying statistics, the hydrogen bonding geometry was described as a distribution function of the angle at the fluorine: a new specific angular function was consequently defined and inserted in the program GRID to est. the effect of fluorine hydrogen bonds on the ligand-protein binding.  All the fluorine-contg. ligands collected from the PDB were docked within their corresponding protein binding sites: introducing the fluorine hydrogen bonding contribution improves the results of the docking expts. in terms of accuracy and ranking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP__-TFksNRbVg90H21EOLACvtfcHk0lghkIR0r-Qk2A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFGhs7s%253D&md5=f119b9adae5296499d467f45fb30c5cf</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm0498349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0498349%26sid%3Dliteratum%253Aachs%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DHydrogen%2520bonding%2520interactions%2520of%2520covalently%2520bonded%2520fluorine%2520atoms%253A%2520from%2520crystallographic%2520data%2520to%2520a%2520new%2520angular%2520function%2520in%2520the%2520GRID%2520force%2520field%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5114%26epage%3D5125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fortuna, C. G.</span><span> </span><span class="NLM_article-title">Exposition and reactivity optimization to predict sites of metabolism in chemicals</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e155</span><span class="NLM_x">â</span> <span class="NLM_lpage">e165</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1016%2Fj.ddtec.2012.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=24050245" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e155-e165&author=G.+Crucianiauthor=M.+Baroniauthor=P.+Benedettiauthor=L.+Goracciauthor=C.+G.+Fortuna&title=Exposition+and+reactivity+optimization+to+predict+sites+of+metabolism+in+chemicals"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DBenedetti%26aufirst%3DP.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DFortuna%26aufirst%3DC.%2BG.%26atitle%3DExposition%2520and%2520reactivity%2520optimization%2520to%2520predict%2520sites%2520of%2520metabolism%2520in%2520chemicals%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De155%26epage%3De165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Shiraga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yajima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teragaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatsubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyashita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Usui, T.</span><span> </span><span class="NLM_article-title">Identification of enzymes responsible for the N-oxidation of darexaban glucuronide, the pharmacologically active metabolite of darexaban, and the glucuronidation of darexaban N-oxides in human liver microsomes</span> <span class="citation_source-journal">Biol. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">413</span><span class="NLM_x">â</span> <span class="NLM_lpage">421</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1248%2Fbpb.35.413" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=22382330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsVejtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2012&pages=413-421&author=T.+Shiragaauthor=K.+Yajimaauthor=T.+Teragakiauthor=K.+Suzukiauthor=T.+Hashimotoauthor=T.+Iwatsuboauthor=A.+Miyashitaauthor=T.+Usui&title=Identification+of+enzymes+responsible+for+the+N-oxidation+of+darexaban+glucuronide%2C+the+pharmacologically+active+metabolite+of+darexaban%2C+and+the+glucuronidation+of+darexaban+N-oxides+in+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of enzymes responsible for the N-oxidation of darexaban glucuronide, the pharmacologically active metabolite of darexaban, and the glucuronidation of darexaban N-oxides in human liver microsomes</span></div><div class="casAuthors">Shiraga, Toshifumi; Yajima, Kanako; Teragaki, Takuya; Suzuki, Katsuhiro; Hashimoto, Tadashi; Iwatsubo, Takafumi; Miyashita, Aiji; Usui, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">413-421</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Darexaban maleate is a novel oral direct factor Xa inhibitor.  Darexaban glucuronide (YM-222714) was the major component in plasma after oral administration of darexaban to humans and is the pharmacol. active metabolite.  Addnl., YM-222714 N-oxides were detected as minor metabolites in human plasma and urine.  It is possible that YM-222714 N-oxides are formed by the N-oxidn. of YM-222714 and/or the glucuronidation of darexaban N-oxides (YM-542845) in vivo.  The former reaction is the pharmacol. inactivation process.  In this study, we identified the human enzymes responsible for YM-222714 N-oxidn. and the UDP (UDP)-glucuronosyltransferase (UGT) isoforms involved in YM-542845 glucuronidation in vitro.  YM-222714 N-oxidn. activity was detected in human liver microsomes (HLM), but not in human intestinal microsomes.  In HLM, YM-222714 N-oxidn. activities were significantly correlated with flavin-contg. monooxygenase (FMO) marker enzyme activities (p<0.001) and inhibited by methimazole, a typical inhibitor of FMOs.  Recombinant human FMO3 and FMO1 were capable of efficiently catalyzing YM-222714 N-oxidn., but not FMO5 or any recombinant human cytochrome P 450 (CYP) isoforms.  Considering the mRNA expression levels of FMO isoforms in human liver, these results strongly suggest that YM-222714 N-oxidn. in HLM is mainly catalyzed by FMO3.  In HLM, YM-542845 glucuronidation was strongly inhibited by typical substrates for UGT1A8, UGT1A9, and UGT1A10.  Recombinant human UGT1A7, UGT1A8, UGT1A9, and UGT1A10 were capable of catalyzing YM-542845 glucuronidation, and UGT1A9 exhibited the highest intrinsic clearance.  Considered together with the expression levels of UGT isoforms in human liver, these results strongly suggest that YM-542845 glucuronidation in HLM is mainly catalyzed by UGT1A9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvizFY_DogYbVg90H21EOLACvtfcHk0ljHqsRADUjBeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsVejtb0%253D&md5=1d9823b3eb6239f63ce48d517e98aa2b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1248%2Fbpb.35.413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.35.413%26sid%3Dliteratum%253Aachs%26aulast%3DShiraga%26aufirst%3DT.%26aulast%3DYajima%26aufirst%3DK.%26aulast%3DTeragaki%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DMiyashita%26aufirst%3DA.%26aulast%3DUsui%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520enzymes%2520responsible%2520for%2520the%2520N-oxidation%2520of%2520darexaban%2520glucuronide%252C%2520the%2520pharmacologically%2520active%2520metabolite%2520of%2520darexaban%252C%2520and%2520the%2520glucuronidation%2520of%2520darexaban%2520N-oxides%2520in%2520human%2520liver%2520microsomes%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D2012%26volume%3D35%26spage%3D413%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Jacobsen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christians, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benet, L. Z.</span><span> </span><span class="NLM_article-title">In vitro evaluation of the disposition of A novel cysteine protease inhibitor</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1343</span><span class="NLM_x">â</span> <span class="NLM_lpage">1351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=11038163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnsFOlsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1343-1351&author=W.+Jacobsenauthor=U.+Christiansauthor=L.+Z.+Benet&title=In+vitro+evaluation+of+the+disposition+of+A+novel+cysteine+protease+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro evaluation of the disposition of a novel cysteine protease inhibitor</span></div><div class="casAuthors">Jacobsen, Wolfgang; Christians, Uwe; Benet, Leslie Z.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1343-1351</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">K11777 (N-methyl-piperazine-Phe-homoPhe-vinylsulfone-phenyl) is a potent, irreversible cysteine protease inhibitor.  Its therapeutic targets are cruzain, a cysteine protease of the protozoan parasite Trypanosoma cruzi, and cathepsins B and L, which are assocd. with cancer progression.  The authors evaluated the metab. of K11777 by human liver microsomes, isolated cytochrome P 450 (CYP) enzymes, and flavin-contg. monooxygenase 3 (FMO3) in vitro.  K11777 was metabolized by human liver microsomes to three major metabolites: N-oxide K11777 (apparent Km = 14.0 Î¼M and apparent Vmax = 3460 pmol Â· mg-1 Â· min-1), Î²-hydroxy-homoPhe K11777 (Km = 16.8 Î¼M and Vmax = 1260 pmol Â· mg-1 Â· min-1), and N-desmethyl K11777 (Km = 18.3 Î¼M and Vmax = 2070 pmol Â· mg-1 Â· min-1).  All three K11777 metabolites were formed by isolated CYP3A and their formation by human liver microsomes was inhibited by the CYP3A inhibitor cyclosporine (50 Î¼M, 54-62% inhibition) and antibodies against human CYP3A4/5 (100 Î¼g of antibodies/100 Î¼g microsomal protein, 55-68% inhibition).  CYP2D6 metabolized K11777 to its N-desmethyl metabolite with an apparent Km (9.2 Î¼M) lower than for CYP3A4 (25.0 Î¼M) and human liver microsomes.  The apparent Km for N-oxide K11777 formation by cDNA-expressed FMO3 was 109 Î¼M.  Based on the intrinsic formation clearances and the results of inhibition expts. (CYP2D6, 50 Î¼M bufuralol; FMO3 mediated, 100 mM methionine) using human liver microsomes, it was estd. that CYP3A contributes to >80% of K11777 metabolite formation.  K11777 was a potent (IC50 = 0.06 Î¼M) and efficacious (max. inhibition 85%) NADPH-dependent inhibitor of human CYP3A4 mediated 6'Î²-hydroxy lovastatin formation, suggesting that K11777 is not only a substrate but also a mechanism-based inhibitor of CYP3A4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnuA6vCCgc0bVg90H21EOLACvtfcHk0lgGoRq1V1doJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnsFOlsbw%253D&md5=d4becc1389b076d394ff43b1d7a093f9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJacobsen%26aufirst%3DW.%26aulast%3DChristians%26aufirst%3DU.%26aulast%3DBenet%26aufirst%3DL.%2BZ.%26atitle%3DIn%2520vitro%2520evaluation%2520of%2520the%2520disposition%2520of%2520A%2520novel%2520cysteine%2520protease%2520inhibitor%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2000%26volume%3D28%26spage%3D1343%26epage%3D1351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Ohmi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akimoto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, S.</span><span> </span><span class="NLM_article-title">S-oxidation of S-methyl-esonarimod by flavin-containing monooxygenases in human liver microsomes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1221</span><span class="NLM_x">â</span> <span class="NLM_lpage">1231</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2003&pages=1221-1231&author=N.+Ohmiauthor=H.+Yoshidaauthor=H.+Endoauthor=M.+Hasegawaauthor=M.+Akimotoauthor=S.+Higuchi&title=S-oxidation+of+S-methyl-esonarimod+by+flavin-containing+monooxygenases+in+human+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOhmi%26aufirst%3DN.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DEndo%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DAkimoto%26aufirst%3DM.%26aulast%3DHiguchi%26aufirst%3DS.%26atitle%3DS-oxidation%2520of%2520S-methyl-esonarimod%2520by%2520flavin-containing%2520monooxygenases%2520in%2520human%2520liver%2520microsomes%26jtitle%3DXenobiotica%26date%3D2003%26volume%3D33%26spage%3D1221%26epage%3D1231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Xie, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constantinides, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rigas, B.</span><span> </span><span class="NLM_article-title">Regioselective oxidation of phospho-NSAIDs by human cytochrome P450 and flavin monooxygenase isoforms: implications for their pharmacokinetic properties and safety</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">222</span><span class="NLM_x">â</span> <span class="NLM_lpage">232</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1111%2Fj.1476-5381.2012.01982.x" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=222-232&author=G.+Xieauthor=C.+C.+Wongauthor=K.+W.+Chengauthor=L.+Huangauthor=P.+P.+Constantinidesauthor=B.+Rigas&title=Regioselective+oxidation+of+phospho-NSAIDs+by+human+cytochrome+P450+and+flavin+monooxygenase+isoforms%3A+implications+for+their+pharmacokinetic+properties+and+safety"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2012.01982.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2012.01982.x%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DC.%2BC.%26aulast%3DCheng%26aufirst%3DK.%2BW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DConstantinides%26aufirst%3DP.%2BP.%26aulast%3DRigas%26aufirst%3DB.%26atitle%3DRegioselective%2520oxidation%2520of%2520phospho-NSAIDs%2520by%2520human%2520cytochrome%2520P450%2520and%2520flavin%2520monooxygenase%2520isoforms%253A%2520implications%2520for%2520their%2520pharmacokinetic%2520properties%2520and%2520safety%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2012%26volume%3D16%26spage%3D222%26epage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Attar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang-Liu, D. D.</span><span> </span><span class="NLM_article-title">Cytochrome P450 2C8 and flavin-containing monooxygenases are involved in the metabolism of tazarotenic acid in humans</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">476</span><span class="NLM_x">â</span> <span class="NLM_lpage">481</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=476-481&author=M.+Attarauthor=D.+Dongauthor=K.+H.+Lingauthor=D.+D.+Tang-Liu&title=Cytochrome+P450+2C8+and+flavin-containing+monooxygenases+are+involved+in+the+metabolism+of+tazarotenic+acid+in+humans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAttar%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DD.%26aulast%3DLing%26aufirst%3DK.%2BH.%26aulast%3DTang-Liu%26aufirst%3DD.%2BD.%26atitle%3DCytochrome%2520P450%25202C8%2520and%2520flavin-containing%2520monooxygenases%2520are%2520involved%2520in%2520the%2520metabolism%2520of%2520tazarotenic%2520acid%2520in%2520humans%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2003%26volume%3D31%26spage%3D476%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kajita, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inano, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuse, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuwabara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, H.</span><span> </span><span class="NLM_article-title">Effects of olopatadine, a new antiallergic agent, on human liver microsomal cytochrome P450 activities</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">â</span> <span class="NLM_lpage">1511</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=1504-1511&author=J.+Kajitaauthor=K.+Inanoauthor=E.+Fuseauthor=T.+Kuwabaraauthor=H.+Kobayashi&title=Effects+of+olopatadine%2C+a+new+antiallergic+agent%2C+on+human+liver+microsomal+cytochrome+P450+activities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKajita%26aufirst%3DJ.%26aulast%3DInano%26aufirst%3DK.%26aulast%3DFuse%26aufirst%3DE.%26aulast%3DKuwabara%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DH.%26atitle%3DEffects%2520of%2520olopatadine%252C%2520a%2520new%2520antiallergic%2520agent%252C%2520on%2520human%2520liver%2520microsomal%2520cytochrome%2520P450%2520activities%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2002%26volume%3D30%26spage%3D1504%26epage%3D1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Ripp, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overby, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpot, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elfarra, A. A.</span><span> </span><span class="NLM_article-title">Oxidation of cysteine S-conjugates by rabbit liver microsomes and cDNA-expressed flavin-containing mono-oxygenases: studies with S-(1,2-dichlorovinyl)-L-cysteine, S-(1,2,2-trichlorovinyl)-L-cysteine, S-allyl-L-cysteine, and S-benzyl-L-cysteine</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">507</span><span class="NLM_x">â</span> <span class="NLM_lpage">515</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1997&pages=507-515&author=S.+L.+Rippauthor=L.+H.+Overbyauthor=R.+M.+Philpotauthor=A.+A.+Elfarra&title=Oxidation+of+cysteine+S-conjugates+by+rabbit+liver+microsomes+and+cDNA-expressed+flavin-containing+mono-oxygenases%3A+studies+with+S-%281%2C2-dichlorovinyl%29-L-cysteine%2C+S-%281%2C2%2C2-trichlorovinyl%29-L-cysteine%2C+S-allyl-L-cysteine%2C+and+S-benzyl-L-cysteine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRipp%26aufirst%3DS.%2BL.%26aulast%3DOverby%26aufirst%3DL.%2BH.%26aulast%3DPhilpot%26aufirst%3DR.%2BM.%26aulast%3DElfarra%26aufirst%3DA.%2BA.%26atitle%3DOxidation%2520of%2520cysteine%2520S-conjugates%2520by%2520rabbit%2520liver%2520microsomes%2520and%2520cDNA-expressed%2520flavin-containing%2520mono-oxygenases%253A%2520studies%2520with%2520S-%25281%252C2-dichlorovinyl%2529-L-cysteine%252C%2520S-%25281%252C2%252C2-trichlorovinyl%2529-L-cysteine%252C%2520S-allyl-L-cysteine%252C%2520and%2520S-benzyl-L-cysteine%26jtitle%3DMol.%2520Pharmacol.%26date%3D1997%26volume%3D51%26spage%3D507%26epage%3D515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Dever, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elfarra, A. A.</span><span> </span><span class="NLM_article-title">In vivo metabolism of L-methionine in mice: evidence for stereoselective formation of methionine-d-sulfoxide and quantitation of other major metabolites</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2036</span><span class="NLM_x">â</span> <span class="NLM_lpage">2043</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2036-2043&author=J.+T.+Deverauthor=A.+A.+Elfarra&title=In+vivo+metabolism+of+L-methionine+in+mice%3A+evidence+for+stereoselective+formation+of+methionine-d-sulfoxide+and+quantitation+of+other+major+metabolites"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDever%26aufirst%3DJ.%2BT.%26aulast%3DElfarra%26aufirst%3DA.%2BA.%26atitle%3DIn%2520vivo%2520metabolism%2520of%2520L-methionine%2520in%2520mice%253A%2520evidence%2520for%2520stereoselective%2520formation%2520of%2520methionine-d-sulfoxide%2520and%2520quantitation%2520of%2520other%2520major%2520metabolites%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2006%26volume%3D34%26spage%3D2036%26epage%3D2043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">el Amri, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargetton, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delatour, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batt, A. M.</span><span> </span><span class="NLM_article-title">Sulphoxidation of albendazole by the FAD-containing and cytochrome P-450 dependent mono-oxygenases from pig liver microsomes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1159</span><span class="NLM_x">â</span> <span class="NLM_lpage">1168</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1987&pages=1159-1168&author=H.+S.+el+Amriauthor=X.+Fargettonauthor=P.+Delatourauthor=A.+M.+Batt&title=Sulphoxidation+of+albendazole+by+the+FAD-containing+and+cytochrome+P-450+dependent+mono-oxygenases+from+pig+liver+microsomes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Del%2BAmri%26aufirst%3DH.%2BS.%26aulast%3DFargetton%26aufirst%3DX.%26aulast%3DDelatour%26aufirst%3DP.%26aulast%3DBatt%26aufirst%3DA.%2BM.%26atitle%3DSulphoxidation%2520of%2520albendazole%2520by%2520the%2520FAD-containing%2520and%2520cytochrome%2520P-450%2520dependent%2520mono-oxygenases%2520from%2520pig%2520liver%2520microsomes%26jtitle%3DXenobiotica%26date%3D1987%26volume%3D17%26spage%3D1159%26epage%3D1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Furnes, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlenk, D.</span><span> </span><span class="NLM_article-title">Extrahepatic metabolism of carbamate and organophosphate thioether compounds by the flavin-containing monooxygenase and cytochrome P450 systems</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">â</span> <span class="NLM_lpage">218</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=214-218&author=B.+Furnesauthor=D.+Schlenk&title=Extrahepatic+metabolism+of+carbamate+and+organophosphate+thioether+compounds+by+the+flavin-containing+monooxygenase+and+cytochrome+P450+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFurnes%26aufirst%3DB.%26aulast%3DSchlenk%26aufirst%3DD.%26atitle%3DExtrahepatic%2520metabolism%2520of%2520carbamate%2520and%2520organophosphate%2520thioether%2520compounds%2520by%2520the%2520flavin-containing%2520monooxygenase%2520and%2520cytochrome%2520P450%2520systems%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D214%26epage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">New and original pK<sub>a</sub> prediction method using grid molecular interaction fields</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2172</span><span class="NLM_x">â</span> <span class="NLM_lpage">2181</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700018y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2172-2181&author=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=G.+Cruciani&title=New+and+original+pKa+prediction+method+using+grid+molecular+interaction+fields"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New and Original pKa Prediction Method Using Grid Molecular Interaction Fields</span></div><div class="casAuthors">Milletti, Francesca; Storchi, Loriano; Sforna, Gianluca; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2172-2181</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the most important physicochem. properties of a mol. is pKa.  It is known that two parameters imperative in ADME profiling, soly., and lipophilicity are governed by pKa, and receptor binding can be influenced by pKa.  Because most drugs are ionized in physiol. conditions, pKa is particularly relevant to medicinal chem.  Despite the numerous advances in high-throughput measurements, in silico detn. is still the fastest and cheapest way of obtaining pKa.  This paper presents a new original computational method for pKa prediction of org. compds.  Descriptors were generated using the program GRID, and these descriptors are based on mol. interaction fields precomputed on a set of mol. fragments.  The new method was developed, trained, and cross-validated by using a large and diverse data set of 24 617 pKa values.  This paper presents the results for a class of 421 acidic nitrogen compds. (RMSE = 0.41, r2 = 0.97, q2 = 0.87) and for a class of 947 six-membered N-heterocyclic bases (RMSE = 0.60, r2 = 0.93, q2 = 0.85).  For external validation 28 novel compds. were selected that covered nine different ionizable groups, and 39 pKa values could be exptl. detd. by spectral gradient anal. (SGA).  Comparison of exptl. pKa with calcd. pKa demonstrated that the predictive ability of the method is good (external set, r2 = 0.85, RMSE = 0.90).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzEk0uiS1E6rVg90H21EOLACvtfcHk0lhE7uvkYEkB7A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF&md5=f2b1dec853ccb0ef1a40540d234232fb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci700018y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700018y%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DNew%2520and%2520original%2520pKa%2520prediction%2520method%2520using%2520grid%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2172%26epage%3D2181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span> </span><span class="NLM_article-title">In silico pK<sub>a</sub> prediction and ADME profiling</span> <span class="citation_source-journal">Chem. Biodiversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1812</span><span class="NLM_x">â</span> <span class="NLM_lpage">1821</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=1812-1821&author=G.+Crucianiauthor=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=L.+Goracci&title=In+silico+pKa+prediction+and+ADME+profiling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DGoracci%26aufirst%3DL.%26atitle%3DIn%2520silico%2520pKa%2520prediction%2520and%2520ADME%2520profiling%26jtitle%3DChem.%2520Biodiversity%26date%3D2009%26volume%3D6%26spage%3D1812%26epage%3D1821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Alfieri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malito, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orru, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Revealing the moonlighting role of NADP in the structure of a flavin-containing monooxygenase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6572</span><span class="NLM_x">â</span> <span class="NLM_lpage">6577</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=6572-6577&author=A.+Alfieriauthor=E.+Malitoauthor=R.+Orruauthor=M.+W.+Fraaijeauthor=A.+Mattevi&title=Revealing+the+moonlighting+role+of+NADP+in+the+structure+of+a+flavin-containing+monooxygenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAlfieri%26aufirst%3DA.%26aulast%3DMalito%26aufirst%3DE.%26aulast%3DOrru%26aufirst%3DR.%26aulast%3DFraaije%26aufirst%3DM.%2BW.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DRevealing%2520the%2520moonlighting%2520role%2520of%2520NADP%2520in%2520the%2520structure%2520of%2520a%2520flavin-containing%2520monooxygenase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D6572%26epage%3D6577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Koukouritaki, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddens, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krueger, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VanDyke, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajewski, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, R. N.</span><span> </span><span class="NLM_article-title">Identification and functional analysis of common human flavin-containing monooxygenase 3 genetic variants</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">â</span> <span class="NLM_lpage">273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1124%2Fjpet.106.112268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=17050781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsFCisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2007&pages=266-273&author=S.+B.+Koukouritakiauthor=M.+T.+Pochauthor=M.+C.+Hendersonauthor=L.+K.+Siddensauthor=S.+K.+Kruegerauthor=J.+E.+VanDykeauthor=D.+E.+Williamsauthor=N.+M.+Pajewskiauthor=T.+Wangauthor=R.+N.+Hines&title=Identification+and+functional+analysis+of+common+human+flavin-containing+monooxygenase+3+genetic+variants"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and functional analysis of common human flavin-containing monooxygenase 3 genetic variants</span></div><div class="casAuthors">Koukouritaki, Sevasti B.; Poch, Mark T.; Henderson, Marilyn C.; Siddens, Lisbeth K.; Krueger, Sharon K.; VanDyke, Jonathan E.; Williams, David E.; Pajewski, Nicholas M.; Wang, Tao; Hines, Ronald N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">266-273</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Flavin-contg. monooxygenases (FMOs) are important for the disposition of many therapeutics, environmental toxicants, and nutrients.  FMO3, the major adult hepatic FMO enzyme, exhibits significant interindividual variation.  Eighteen FMO3 single-nucleotide polymorphism (SNP) frequencies were detd. in 202 Hispanics (Mexican descent), 201 African Americans, and 200 non-Latino whites.  Using expressed recombinant enzyme with methimazole, trimethylamine, sulindac, and ethylenethiourea, the novel structural variants FMO3 E24D and K416N were shown to cause modest changes in catalytic efficiency, whereas a third novel variant, FMO3 N61K, was essentially devoid of activity.  The latter variant was present at an allelic frequency of 5.2% in non-Latino whites and 3.5% in African Americans, but it was absent in Hispanics.  Inferring haplotypes using PHASE, version 2.1, the greatest haplotype diversity was obsd. in African Americans followed by non-Latino whites and Hispanics.  Haplotype 2A and 2B, consisting of a hypermorphic promoter SNP cluster (-2650C>G, -2543T>A, and -2177G>C) in linkage with synonymous structural variants was inferred at a frequency of 27% in the Hispanic population, but only 5% in non-Latino whites and African Americans.  This same promoter SNP cluster in linkage with one or more hypomorphic structural variant also was inferred in multiple haplotypes at a total frequency of 5.6% in the African-American study group but less than 1% in the other two groups.  The sum frequencies of the hypomorphic haplotypes H3 [15,167G>A (E158K)], H5B [-2650C>G, 15,167G>A (E158K), 21,375C>T (N285N), 21,443A>G (E308G)], and H6 [15,167G>A (E158K), 21,375C>T (N285N)] was 28% in Hispanics, 23% in non-Latino whites, and 24% in African Americans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyD6Jw2FEfcrVg90H21EOLACvtfcHk0lhvE_Uoy7QchA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsFCisw%253D%253D&md5=5af13bff757eed636ca6834e795332cb</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.112268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.112268%26sid%3Dliteratum%253Aachs%26aulast%3DKoukouritaki%26aufirst%3DS.%2BB.%26aulast%3DPoch%26aufirst%3DM.%2BT.%26aulast%3DHenderson%26aufirst%3DM.%2BC.%26aulast%3DSiddens%26aufirst%3DL.%2BK.%26aulast%3DKrueger%26aufirst%3DS.%2BK.%26aulast%3DVanDyke%26aufirst%3DJ.%2BE.%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DPajewski%26aufirst%3DN.%2BM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DHines%26aufirst%3DR.%2BN.%26atitle%3DIdentification%2520and%2520functional%2520analysis%2520of%2520common%2520human%2520flavin-containing%2520monooxygenase%25203%2520genetic%2520variants%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2007%26volume%3D320%26spage%3D266%26epage%3D273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perruccio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span> </span><span class="NLM_article-title">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for ligands and proteins (FLAP): theory and application</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">279</span><span class="NLM_x">â</span> <span class="NLM_lpage">294</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600253e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=279-294&author=M.+Baroniauthor=G.+Crucianiauthor=S.+Sciabolaauthor=F.+Perruccioauthor=J.+S.+Mason&title=A+common+reference+framework+for+analyzing%2Fcomparing+proteins+and+ligands.+Fingerprints+for+ligands+and+proteins+%28FLAP%29%3A+theory+and+application"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for ligands and proteins (FLAP): theory and application</span></div><div class="casAuthors">Baroni, Massimo; Cruciani, Gabriele; Sciabola, Simone; Perruccio, Francesca; Mason, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-294</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast new algorithm (fingerprints for ligands and proteins or FLAP) able to describe small mols. and protein structures using a common ref. framework of four-point pharmacophore fingerprints and a mol.-cavity shape is described in detail.  The procedure starts by using the GRID force field to calc. mol. interaction fields, which are then used to identify particular target locations where an energetic interaction with small mol. features would be very favorable.  The target points thus calcd. are then used by FLAP to build all possible four-point pharmacophores present in the given target site.  A related approach can be applied to small mols., using directly the GRID atom types to identify pharmacophoric features, and this complementary description of the target and ligand then leads to several novel applications.  FLAP can be used for selectivity studies or similarity analyses in order to compare macromols. without superposing them.  Protein families can be compared and clustered into target classes, without bias from previous knowledge and without requiring protein superposition, alignment, or knowledge-based comparison.  FLAP can be used effectively for ligand-based virtual screening and structure-based virtual screening, with the pharmacophore mol. recognition.  Finally, the new method can calc. descriptors for chemometric anal. and can initiate a docking procedure.  This paper presents the background to the new procedure and includes case studies illustrating several relevant applications of the new approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppMc0fYDNn7LVg90H21EOLACvtfcHk0lhvE_Uoy7QchA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D&md5=9facb99c5741c7158afab7a3ac387ec3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fci600253e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600253e%26sid%3Dliteratum%253Aachs%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DPerruccio%26aufirst%3DF.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DA%2520common%2520reference%2520framework%2520for%2520analyzing%252Fcomparing%2520proteins%2520and%2520ligands.%2520Fingerprints%2520for%2520ligands%2520and%2520proteins%2520%2528FLAP%2529%253A%2520theory%2520and%2520application%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D279%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Sirci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">RodrÃ­guez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Muijlwijk-Koezen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">GutiÃ©rrez-de-TerÃ¡n, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mannhold, R.</span><span> </span><span class="NLM_article-title">Ligand-, structure- and pharmacophore-based molecular fingerprints: a case study on adenosine A(1), A (2A), A (2B), and A (3) receptor antagonists</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1247</span><span class="NLM_x">â</span> <span class="NLM_lpage">1266</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1247-1266&author=F.+Sirciauthor=L.+Goracciauthor=D.+Rodr%C3%ADguezauthor=J.+van+Muijlwijk-Koezenauthor=H.+Guti%C3%A9rrez-de-Ter%C3%A1nauthor=R.+Mannhold&title=Ligand-%2C+structure-+and+pharmacophore-based+molecular+fingerprints%3A+a+case+study+on+adenosine+A%281%29%2C+A+%282A%29%2C+A+%282B%29%2C+and+A+%283%29+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSirci%26aufirst%3DF.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%26aulast%3DGuti%25C3%25A9rrez-de-Ter%25C3%25A1n%26aufirst%3DH.%26aulast%3DMannhold%26aufirst%3DR.%26atitle%3DLigand-%252C%2520structure-%2520and%2520pharmacophore-based%2520molecular%2520fingerprints%253A%2520a%2520case%2520study%2520on%2520adenosine%2520A%25281%2529%252C%2520A%2520%25282A%2529%252C%2520A%2520%25282B%2529%252C%2520and%2520A%2520%25283%2529%2520receptor%2520antagonists%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D1247%26epage%3D1266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">The basic principles of GRID</span>. In  <span class="citation_source-book">Molecular Interaction Fields: Applications in Drug Discovery and ADME Prediction</span>; <span class="NLM_contrib-group">Cruciani, G.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">â</span> <span class="NLM_lpage">25</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2F3527607676.ch1" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=1-25&author=P.+J.+Goodfordauthor=G.+Cruciani&title=Molecular+Interaction+Fields%3A+Applications+in+Drug+Discovery+and+ADME+Prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2F3527607676.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527607676.ch1%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520basic%2520principles%2520of%2520GRID%26btitle%3DMolecular%2520Interaction%2520Fields%253A%2520Applications%2520in%2520Drug%2520Discovery%2520and%2520ADME%2520Prediction%26aulast%3DCruciani%26aufirst%3DG.%26pub%3DWiley-VCH%26date%3D2006%26spage%3D1%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span>Mayrâs Database of Reactivity Parameters. <a href="http://www.cup.lmu.de/oc/mayr/reaktionsdatenbank/fe/showclass/40" class="extLink">http://www.cup.lmu.de/oc/mayr/reaktionsdatenbank/fe/showclass/40</a> (accessed February 1,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mayr%E2%80%99s+Database+of+Reactivity+Parameters.+http%3A%2F%2Fwww.cup.lmu.de%2Foc%2Fmayr%2Freaktionsdatenbank%2Ffe%2Fshowclass%2F40+%28accessed+February+1%2C+2014%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Rosipal, R.; Kramer, N.</span><span> </span><span class="NLM_article-title">Overview and recent advances in partial least squares</span>. In  <span class="citation_source-book">Subspace, Latent Structure and Feature Selection: Statistical and Optimization Perspectives Workshop (SLSFS 2005)</span>; <span class="NLM_contrib-group">Saunders, C.</span>, <span class="NLM_etal"></span>, Eds.; <span class="NLM_publisher-name">Springer-Verlag</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">34</span><span class="NLM_x">â</span> <span class="NLM_lpage">51</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1007%2F11752790_2" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=34-51&author=R.+Rosipal&author=N.+Kramerauthor=C.+Saunders&title=Subspace%2C+Latent+Structure+and+Feature+Selection%3A+Statistical+and+Optimization+Perspectives+Workshop+%28SLSFS+2005%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2F11752790_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F11752790_2%26sid%3Dliteratum%253Aachs%26aulast%3DRosipal%26aufirst%3DR.%26atitle%3DOverview%2520and%2520recent%2520advances%2520in%2520partial%2520least%2520squares%26btitle%3DSubspace%252C%2520Latent%2520Structure%2520and%2520Feature%2520Selection%253A%2520Statistical%2520and%2520Optimization%2520Perspectives%2520Workshop%2520%2528SLSFS%25202005%2529%26aulast%3DSaunders%26aufirst%3DC.%26pub%3DSpringer-Verlag%26date%3D2006%26spage%3D34%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Boeck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ethirajulu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, R.</span><span> </span><span class="NLM_article-title">MetaSite: understanding metabolism in human cytochromes from the perspective of the chemist</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6970</span><span class="NLM_x">â</span> <span class="NLM_lpage">6979</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050529c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6970-6979&author=G.+Crucianiauthor=E.+Carosatiauthor=B.+De+Boeckauthor=K.+Ethirajuluauthor=C.+Mackieauthor=T.+Howeauthor=R.+Vianello&title=MetaSite%3A+understanding+metabolism+in+human+cytochromes+from+the+perspective+of+the+chemist"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm050529c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050529c%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DDe%2BBoeck%26aufirst%3DB.%26aulast%3DEthirajulu%26aufirst%3DK.%26aulast%3DMackie%26aufirst%3DC.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DVianello%26aufirst%3DR.%26atitle%3DMetaSite%253A%2520understanding%2520metabolism%2520in%2520human%2520cytochromes%2520from%2520the%2520perspective%2520of%2520the%2520chemist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6970%26epage%3D6979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Yanni, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annaert, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustijns, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benjamin, D. K.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakker, D. R.</span><span> </span><span class="NLM_article-title">In vitro hepatic metabolism explains higher clearance of voriconazole in children versus adults: role of CYP2C19 and flavin-containing monooxygenase 3</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">â</span> <span class="NLM_lpage">31</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=25-31&author=S.+B.+Yanniauthor=P.+P.+Annaertauthor=P.+Augustijnsauthor=J.+G.+Ibrahimauthor=D.+K.+Benjaminauthor=D.+R.+Thakker&title=In+vitro+hepatic+metabolism+explains+higher+clearance+of+voriconazole+in+children+versus+adults%3A+role+of+CYP2C19+and+flavin-containing+monooxygenase+3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYanni%26aufirst%3DS.%2BB.%26aulast%3DAnnaert%26aufirst%3DP.%2BP.%26aulast%3DAugustijns%26aufirst%3DP.%26aulast%3DIbrahim%26aufirst%3DJ.%2BG.%26aulast%3DBenjamin%26aufirst%3DD.%2BK.%26aulast%3DThakker%26aufirst%3DD.%2BR.%26atitle%3DIn%2520vitro%2520hepatic%2520metabolism%2520explains%2520higher%2520clearance%2520of%2520voriconazole%2520in%2520children%2520versus%2520adults%253A%2520role%2520of%2520CYP2C19%2520and%2520flavin-containing%2520monooxygenase%25203%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2010%26volume%3D38%26spage%3D25%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Vyas, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roychowdhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koukouritaki, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krueger, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nauseef, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, C. K.</span><span> </span><span class="NLM_article-title">Enzyme-mediated protein haptenation of dapsone and sulfamethoxazole in human keratinocytes: II. Expression and role of flavin-containing monooxygenases and peroxidases</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">319</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">â</span> <span class="NLM_lpage">505</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1124%2Fjpet.106.105874" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=16857727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCksLjJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2006&pages=497-505&author=P.+M.+Vyasauthor=S.+Roychowdhuryauthor=S.+B.+Koukouritakiauthor=R.+N.+Hinesauthor=S.+K.+Kruegerauthor=D.+E.+Williamsauthor=W.+M.+Nauseefauthor=C.+K.+Svensson&title=Enzyme-mediated+protein+haptenation+of+dapsone+and+sulfamethoxazole+in+human+keratinocytes%3A+II.+Expression+and+role+of+flavin-containing+monooxygenases+and+peroxidases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme-mediated protein haptenation of dapsone and sulfamethoxazole in human keratinocytes: II. Expression and role of flavin-containing monooxygenases and peroxidases</span></div><div class="casAuthors">Vyas, Piyush M.; Roychowdhury, Sanjoy; Koukouritaki, Sevasti B.; Hines, Ronald N.; Krueger, Sharon K.; Williams, David E.; Nauseef, William M.; Svensson, Craig K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">497-505</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Arylamine compds., such as sulfamethoxazole (SMX) and dapsone (DDS), are metabolized in epidermal keratinocytes to arylhydroxylamine metabolites that auto-oxidize to arylnitroso derivs., which in turn bind to cellular proteins and can act as antigens/immunogens.  Previous studies have demonstrated that neither cytochromes P 450 nor cyclooxygenases mediate this bioactivation in normal human epidermal keratinocytes (NHEKs).  In this investigation, we demonstrated that methimazole (MMZ), a prototypical substrate of the flavin-contg. monooxygenases (FMOs), attenuated the protein haptenation obsd. in NHEKs exposed to SMX or DDS.  In addn., recombinant FMO1 and FMO3 were able to bioactivate both SMX and DDS, resulting in covalent adduct formation.  Western blot anal. confirmed the presence of FMO3 in NHEKs, whereas FMO1 was not detectable.  In addn. to MMZ, 4-aminobenzoic acid hydrazide (ABH) also attenuated SMX- and DDS-dependent protein haptenation in NHEKs.  ABH did not alter the bioactivation of these drugs by recombinant FMO3, suggesting its inhibitory effect in NHEKs was due to its known ability to inhibit peroxidases.  Studies confirmed the presence of peroxidase activity in NHEKs; however, immunoblot anal. and reverse transcription-polymerase chain reaction indicated that myeloperoxidase, lactoperoxidase, and thyroid peroxidase were absent.  Thus, our results suggest an important role for FMO3 and yet-to-be identified peroxidases in the bioactivation of sulfonamides in NHEKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozWEsbvLmO3LVg90H21EOLACvtfcHk0lhve-8nvk8BSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCksLjJ&md5=87c2f80e0c8517f6976ab4bd0791dd68</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.105874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.105874%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DP.%2BM.%26aulast%3DRoychowdhury%26aufirst%3DS.%26aulast%3DKoukouritaki%26aufirst%3DS.%2BB.%26aulast%3DHines%26aufirst%3DR.%2BN.%26aulast%3DKrueger%26aufirst%3DS.%2BK.%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DNauseef%26aufirst%3DW.%2BM.%26aulast%3DSvensson%26aufirst%3DC.%2BK.%26atitle%3DEnzyme-mediated%2520protein%2520haptenation%2520of%2520dapsone%2520and%2520sulfamethoxazole%2520in%2520human%2520keratinocytes%253A%2520II.%2520Expression%2520and%2520role%2520of%2520flavin-containing%2520monooxygenases%2520and%2520peroxidases%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2006%26volume%3D319%26spage%3D497%26epage%3D505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Alinezhad, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tajbakhsh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahdavi, N.</span><span> </span><span class="NLM_article-title">One-pot reductive amination of carbonyl compounds using sodium borohydride-amberlist 15</span> <span class="citation_source-journal">Synth. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">951</span><span class="NLM_x">â</span> <span class="NLM_lpage">956</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1080%2F00397910903026731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC3cXislKksrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2010&pages=951-956&author=H.+Alinezhadauthor=M.+Tajbakhshauthor=N.+Mahdavi&title=One-pot+reductive+amination+of+carbonyl+compounds+using+sodium+borohydride-amberlist+15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">One-pot reductive amination of carbonyl compounds using sodium borohydride-Amberlyst 15</span></div><div class="casAuthors">Alinezhad, Heshmatollah; Tajbakhsh, Mahmood; Mahdavi, Nastaran</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">951-956</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A fast, efficient, and high-yielding method for the prepn. of amines by reductive amination of aldehydes and ketones using sodium borohydride in the presence of Amberlyst 15 in THF and under solvent-free conditions at room temp. was described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGH1A62i0Jk7Vg90H21EOLACvtfcHk0lj_MgCtOuoRhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXislKksrs%253D&md5=93942f8c5572fef66b9d03d14a0bc0bf</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1080%2F00397910903026731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00397910903026731%26sid%3Dliteratum%253Aachs%26aulast%3DAlinezhad%26aufirst%3DH.%26aulast%3DTajbakhsh%26aufirst%3DM.%26aulast%3DMahdavi%26aufirst%3DN.%26atitle%3DOne-pot%2520reductive%2520amination%2520of%2520carbonyl%2520compounds%2520using%2520sodium%2520borohydride-amberlist%252015%26jtitle%3DSynth.%2520Commun.%26date%3D2010%26volume%3D40%26spage%3D951%26epage%3D956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">OâConnor, S.; Dumas, J.; Lee, W.; Dixon, J.; Cantin, D.; Gunn, D.; Burke, J.; Phillips, B.; Lowe, D.; Shelekhin, T.; Wang, G.; Ma, X.; Ying, S.; McClure, A.; Achebe, F.; Lobell, M.; Ehrgott, F.; Iwuagwu, C.; Parcella, K.</span><span> </span><span class="NLM_article-title">Pirrolo[2,1-F][1,2,4]triazin-4-ylamines IGF-1R kinase inhibitors for the treatment of cancer and other hyperproliferative diseases</span>. U.S. Patent US20110294776 A1, Dec 1,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=S.+O%E2%80%99Connor&author=J.+Dumas&author=W.+Lee&author=J.+Dixon&author=D.+Cantin&author=D.+Gunn&author=J.+Burke&author=B.+Phillips&author=D.+Lowe&author=T.+Shelekhin&author=G.+Wang&author=X.+Ma&author=S.+Ying&author=A.+McClure&author=F.+Achebe&author=M.+Lobell&author=F.+Ehrgott&author=C.+Iwuagwu&author=K.+Parcella&title=Pirrolo%5B2%2C1-F%5D%5B1%2C2%2C4%5Dtriazin-4-ylamines+IGF-1R+kinase+inhibitors+for+the+treatment+of+cancer+and+other+hyperproliferative+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DS.%26atitle%3DPirrolo%255B2%252C1-F%255D%255B1%252C2%252C4%255Dtriazin-4-ylamines%2520IGF-1R%2520kinase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520cancer%2520and%2520other%2520hyperproliferative%2520diseases%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Bonn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leandersson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontaine, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamora, I.</span><span> </span><span class="NLM_article-title">Enhanced metabolite identification with MS(E) and a semi-automated software for structural elucidation</span> <span class="citation_source-journal">Rapid Commun. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3127</span><span class="NLM_x">â</span> <span class="NLM_lpage">3138</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2Frcm.4753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=20941759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVKktr3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=3127-3138&author=B.+Bonnauthor=C.+Leanderssonauthor=F.+Fontaineauthor=I.+Zamora&title=Enhanced+metabolite+identification+with+MS%28E%29+and+a+semi-automated+software+for+structural+elucidation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced metabolite identification with MSE and a semi-automated software for structural elucidation</span></div><div class="casAuthors">Bonn, Britta; Leandersson, Carina; Fontaine, Fabien; Zamora, Ismael</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3127-3138</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The identification of metabolites is almost exclusively done with liq. chromatog./tandem mass spectrometry (LC/MSMS) and despite the enormous progress in the development of these techniques and software for handling of data this is a time-consuming task.  In this study the use of quadrupole time-of-flight (QTOF)-generated MSE and MS/MS data were compared with respect to rationalization of metabolites.  In addn. Mass-MetaSite, a semi-automated software for metabolite identification, was evaluated.  The program combines the information from MS raw data, in the form of collision-induced dissocn. spectra, with a prediction of the site of metab. in order to assign the structure of a metabolite.  The aim of the software is to mimic the rationalization of fragment ions performed by a biotransformation scientist in the process of structural elucidation.  For this evaluation, metabolite identification in human liver microsomes was accomplished for 19 com. available compds. and 15 inhouse compds.  The results were very encouraging and for 96% of the metabolites the same structures were assigned using MSE compared with MSMS acquired data.  The possibility of using MSE could considerably reduce the anal. time.  Moreover, Mass-MetaSite performed well and the correct assigned structure, compared to manual inspection of the data, was picked in the first rank in â¼80% of the cases.  In conclusion MSE could be successfully used for metabolite identification in order to reduce time of anal. and Mass-MetaSite could alleviate the work of a biotransformation scientist and decrease the workload by assigning the structure for a majority of the metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PuqO3V_E0bVg90H21EOLACvtfcHk0lj_MgCtOuoRhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVKktr3O&md5=a5d1b3b95ce164cb9b7b068a3852c66a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Frcm.4753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.4753%26sid%3Dliteratum%253Aachs%26aulast%3DBonn%26aufirst%3DB.%26aulast%3DLeandersson%26aufirst%3DC.%26aulast%3DFontaine%26aufirst%3DF.%26aulast%3DZamora%26aufirst%3DI.%26atitle%3DEnhanced%2520metabolite%2520identification%2520with%2520MS%2528E%2529%2520and%2520a%2520semi-automated%2520software%2520for%2520structural%2520elucidation%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2010%26volume%3D24%26spage%3D3127%26epage%3D3138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Zamora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontaine, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serra, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plasencia, G.</span><span> </span><span class="NLM_article-title">High-throughput, fully automated, specific MetID. A revolution for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e199</span><span class="NLM_x">â</span> <span class="NLM_lpage">205</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e199-205&author=I.+Zamoraauthor=F.+Fontaineauthor=B.+Serraauthor=G.+Plasencia&title=High-throughput%2C+fully+automated%2C+specific+MetID.+A+revolution+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZamora%26aufirst%3DI.%26aulast%3DFontaine%26aufirst%3DF.%26aulast%3DSerra%26aufirst%3DB.%26aulast%3DPlasencia%26aufirst%3DG.%26atitle%3DHigh-throughput%252C%2520fully%2520automated%252C%2520specific%2520MetID.%2520A%2520revolution%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De199%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">I-TASSER server for protein 3D structure prediction</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1186%2F1471-2105-9-40" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=18215316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=40&author=Y.+Zhang&title=I-TASSER+server+for+protein+3D+structure+prediction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">I-TASSER server for protein 3D structure prediction</span></div><div class="casAuthors">Zhang Yang</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Prediction of 3-dimensional protein structures from amino acid sequences represents one of the most important problems in computational structural biology.  The community-wide Critical Assessment of Structure Prediction (CASP) experiments have been designed to obtain an objective assessment of the state-of-the-art of the field, where I-TASSER was ranked as the best method in the server section of the recent 7th CASP experiment.  Our laboratory has since then received numerous requests about the public availability of the I-TASSER algorithm and the usage of the I-TASSER predictions.  RESULTS:  An on-line version of I-TASSER is developed at the KU Center for Bioinformatics which has generated protein structure predictions for thousands of modeling requests from more than 35 countries.  A scoring function (C-score) based on the relative clustering structural density and the consensus significance score of multiple threading templates is introduced to estimate the accuracy of the I-TASSER predictions.  A large-scale benchmark test demonstrates a strong correlation between the C-score and the TM-score (a structural similarity measurement with values in [0, 1]) of the first models with a correlation coefficient of 0.91.  Using a C-score cutoff > -1.5 for the models of correct topology, both false positive and false negative rates are below 0.1.  Combining C-score and protein length, the accuracy of the I-TASSER models can be predicted with an average error of 0.08 for TM-score and 2 A for RMSD.  CONCLUSION:  The I-TASSER server has been developed to generate automated full-length 3D protein structural predictions where the benchmarked scoring system helps users to obtain quantitative assessments of the I-TASSER models.  The output of the I-TASSER server for each query includes up to five full-length models, the confidence score, the estimated TM-score and RMSD, and the standard deviation of the estimations.  The I-TASSER server is freely available to the academic community at http://zhang.bioinformatics.ku.edu/I-TASSER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUJ3biR9wC1IvKgEt20y1XfW6udTcc2eZvGQb2PVMbqrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D&md5=48ded5b2e40af27acf1e5753f827080c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-40&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-40%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DI-TASSER%2520server%2520for%2520protein%25203D%2520structure%2520prediction%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span>I-TASSER Online. <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/" class="extLink">http://zhanglab.ccmb.med.umich.edu/I-TASSER/</a> (accessed February 12,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=I-TASSER+Online.+http%3A%2F%2Fzhanglab.ccmb.med.umich.edu%2FI-TASSER%2F+%28accessed+February+12%2C+2014%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Case, D. A.; Darden, T. A.; Cheatham, T. E.,  III; Simmerling, C. L.; Wang, J.; Duke, R. E.; Luo, R.; Walker, R. C.; Zhang, W.; Merz, K. M.; Roberts, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Swails, J.; Goetz, A. W.; KolossvÃ¡ry, I.; Wong, K. F.; Paesani, F.; Vanicek, J.; Wolf, R. M.; Liu, J.; Wu, X.; Brozell, S. R.; Steinbrecher, T.; Gohlke, H.; Cai, Q.; Ye, X.; Wang, J.; Hsieh, M.-J.; Cui, G.; Roe, D. R.; Mathews, D. H.; Seetin, M. G.; Salomon-Ferrer, R.; Sagui, C.; Babin, V.; Luchko, T.; Gusarov, S.; Kovalenko, A.; Kollman, P. A.</span>AMBER 12; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>.<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Roberts&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=J.+Swails&author=A.+W.+Goetz&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=R.+M.+Wolf&author=J.+Liu&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=Q.+Cai&author=X.+Ye&author=J.+Wang&author=M.-J.+Hsieh&author=G.+Cui&author=D.+R.+Roe&author=D.+H.+Mathews&author=M.+G.+Seetin&author=R.+Salomon-Ferrer&author=C.+Sagui&author=V.+Babin&author=T.+Luchko&author=S.+Gusarov&author=A.+Kovalenko&author=P.+A.+Kollman&title=AMBER+12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252012%26pub%3DUniversity%2520of%2520California%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">â</span> <span class="NLM_lpage">935</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25Â° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0liTnOHsj8edvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutzner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span> </span><span class="NLM_article-title">GROMACS 4: algorithms for highly efficient, load-balanced, and scalable molecular simulation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">â</span> <span class="NLM_lpage">447</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct700301q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=435-447&author=B.+Hessauthor=C.+Kutznerauthor=D.+van+der+Spoelauthor=E.+Lindahl&title=GROMACS+4%3A+algorithms+for+highly+efficient%2C+load-balanced%2C+and+scalable+molecular+simulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span></div><div class="casAuthors">Hess, Berk; Kutzner, Carsten; van der Spoel, David; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. simulation is an extremely useful, but computationally very expensive tool for studies of chem. and biomol. systems.  Here, we present a new implementation of our mol. simulation toolkit GROMACS which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines.  The code encompasses a minimal-communication domain decompn. algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel.  To improve the scaling properties of the common particle mesh Ewald electrostatics algorithms, we have in addn. used a Multiple-Program, Multiple-Data approach, with sep. node domains responsible for direct and reciprocal space interactions.  Not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest nos. of std. cluster nodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajjWdk6mUFrVg90H21EOLACvtfcHk0liTnOHsj8edvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D&md5=d53c94901386260221792ea30f151c5f</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fct700301q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700301q%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DKutzner%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204%253A%2520algorithms%2520for%2520highly%2520efficient%252C%2520load-balanced%252C%2520and%2520scalable%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D435%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bekker, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, J. G. E. M.</span><span> </span><span class="NLM_article-title">Lincs: a linear constraint solver for molecular simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1463</span><span class="NLM_x">â</span> <span class="NLM_lpage">1472</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+G.+E.+M.+Fraaije&title=Lincs%3A+a+linear+constraint+solver+for+molecular+simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS: a linear constraint solver for molecular simulations</span></div><div class="casAuthors">Hess, Berk; Bekker, Henk; Berendsen, Herman J. C.; Fraaije, Johannes G. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1472</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">We present a new LINear Constraint Solver (LINCS) for mol. simulations with bond constraints using the enzyme lysozyme and a 32-residue peptide as test systems.  The algorithm is inherently stable, as the constraints themselves are reset instead of derivs. of the constraints, thereby eliminating drift.  Although the derivation of the algorithm is presented in terms of matrixes, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large mols.  At the same accuracy, the LINCS algorithm is 3-4 times faster than the SHAKE algorithm.  Parallelization of the algorithm is straightforward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk6h4pdV0Nx7Vg90H21EOLACvtfcHk0liTnOHsj8edvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D&md5=890f8af0d2ca1f65aa93db5a3a0bacf2</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.0.CO%253B2-H%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%2BG.%2BE.%2BM.%26atitle%3DLincs%253A%2520a%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26spage%3D1463%26epage%3D1472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Shao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molnar, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kussmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ochsenfeld, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A. T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slipchenko, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levchenko, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâNeill, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiStasio, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lochan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beran, G. J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besley, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herbert, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Voorhis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steele, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rassolov, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maslen, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korambath, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamson, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, E. F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dachsel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doerksen, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreuw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunietz, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutoi, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furlani, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gwaltney, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, C.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kedziora, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khalliulin, R. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klunzinger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, W. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lotan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Proynov, E. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieniazek, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhee, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritchie, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherrill, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonett, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subotnik, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodcock, H. L.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakraborty, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipman, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keil, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warshel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hehre, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krylov, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, P. M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head-Gordon, M.</span><span> </span><span class="NLM_article-title">Advances in methods and algorithms in a modern quantum chemistry program package</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3172</span><span class="NLM_x">â</span> <span class="NLM_lpage">3191</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm5007098&amp;key=10.1039%2FB517914A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm5007098&amp;key=16902710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm5007098&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsValsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=3172-3191&author=Y.+Shaoauthor=L.+F.+Molnarauthor=Y.+Jungauthor=J.+Kussmannauthor=C.+Ochsenfeldauthor=S.+T.+Brownauthor=A.+T.+B.+Gilbertauthor=L.+V.+Slipchenkoauthor=S.+V.+Levchenkoauthor=D.+P.+O%E2%80%99Neillauthor=R.+A.+DiStasioauthor=R.+C.+Lochanauthor=T.+Wangauthor=G.+J.+O.+Beranauthor=N.+A.+Besleyauthor=J.+M.+Herbertauthor=C.+Y.+Linauthor=T.+Van+Voorhisauthor=S.+H.+Chienauthor=A.+Sodtauthor=R.+P.+Steeleauthor=V.+A.+Rassolovauthor=P.+E.+Maslenauthor=P.+P.+Korambathauthor=R.+D.+Adamsonauthor=B.+Austinauthor=J.+Bakerauthor=E.+F.+C.+Byrdauthor=H.+Dachselauthor=R.+J.+Doerksenauthor=A.+Dreuwauthor=B.+D.+Dunietzauthor=A.+D.+Dutoiauthor=T.+R.+Furlaniauthor=S.+R.+Gwaltneyauthor=A.+Heydenauthor=S.+Hirataauthor=C.-P.+Hsuauthor=G.+Kedzioraauthor=R.+Z.+Khalliulinauthor=P.+Klunzingerauthor=A.+M.+Leeauthor=M.+S.+Leeauthor=W.+Z.+Liangauthor=I.+Lotanauthor=N.+Nairauthor=B.+Petersauthor=E.+I.+Proynovauthor=P.+A.+Pieniazekauthor=Y.+M.+Rheeauthor=J.+Ritchieauthor=E.+Rostaauthor=C.+D.+Sherrillauthor=A.+C.+Simmonettauthor=J.+E.+Subotnikauthor=H.+L.+Woodcockauthor=W.+Zhangauthor=A.+T.+Bellauthor=A.+K.+Chakrabortyauthor=D.+M.+Chipmanauthor=F.+J.+Keilauthor=A.+Warshelauthor=W.+J.+Hehreauthor=H.+F.+Schaeferauthor=J.+Kongauthor=A.+I.+Krylovauthor=P.+M.+W.+Gillauthor=M.+Head-Gordon&title=Advances+in+methods+and+algorithms+in+a+modern+quantum+chemistry+program+package"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in methods and algorithms in a modern quantum chemistry program package</span></div><div class="casAuthors">Shao, Yihan; Molnar, Laszlo Fusti; Jung, Yousung; Kussmann, Jorg; Ochsenfeld, Christian; Brown, Shawn T.; Gilbert, Andrew T. B.; Slipchenko, Lyudmila V.; Levchenko, Sergey V.; O'Neill, Darragh P.; DiStasio, Robert A., Jr.; Lochan, Rohini C.; Wang, Tao; Beran, Gregory J. O.; Besley, Nicholas A.; Herbert, John M.; Lin, Ching Yeh; Van Voorhis, Troy; Chien, Siu Hung; Sodt, Alex; Steele, Ryan P.; Rassolov, Vitaly A.; Maslen, Paul E.; Korambath, Prakashan P.; Adamson, Ross D.; Austin, Brian; Baker, Jon; Byrd, Edward F. C.; Dachsel, Holger; Doerksen, Robert J.; Dreuw, Andreas; Dunietz, Barry D.; Dutoi, Anthony D.; Furlani, Thomas R.; Gwaltney, Steven R.; Heyden, Andreas; Hirata, So; Hsu, Chao-Ping; Kedziora, Gary; Khalliulin, Rustam Z.; Klunzinger, Phil; Lee, Aaron M.; Lee, Michael S.; Liang, WanZhen; Lotan, Itay; Nair, Nikhil; Peters, Baron; Proynov, Emil I.; Pieniazek, Piotr A.; Rhee, Young Min; Ritchie, Jim; Rosta, Edina; Sherrill, C. David; Simmonett, Andrew C.; Subotnik, Joseph E.; Woodcock, H. Lee, III; Zhang, Weimin; Bell, Alexis T.; Chakraborty, Arup K.; Chipman, Daniel M.; Keil, Frerich J.; Warshel, Arieh; Hehre, Warren J.; Schaefer, Henry F., III; Kong, Jing; Krylov, Anna I.; Gill, Peter M. W.; Head-Gordon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3172-3191</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Advances in theory and algorithms for electronic structure calcns. must be incorporated into program packages to enable them to become routinely used by the broader chem. community.  This work reviews advances made over the past five years or so that constitute the major improvements contained in a new release of the Q-Chem quantum chem. package, together with illustrative timings and applications.  Specific developments discussed include fast methods for d. functional theory calcns., linear scaling evaluation of energies, NMR chem. shifts and elec. properties, fast auxiliary basis function methods for correlated energies and gradients, equation-of-motion coupled cluster methods for ground and excited states, geminal wavefunctions, embedding methods and techniques for exploring potential energy surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNN2YG1uDDs7Vg90H21EOLACvtfcHk0lg9CNfc_SP3-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsValsbs%253D&md5=0ba1d2a350afc7922d2f9ca0c6e47d34</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1039%2FB517914A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB517914A%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DY.%26aulast%3DMolnar%26aufirst%3DL.%2BF.%26aulast%3DJung%26aufirst%3DY.%26aulast%3DKussmann%26aufirst%3DJ.%26aulast%3DOchsenfeld%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DS.%2BT.%26aulast%3DGilbert%26aufirst%3DA.%2BT.%2BB.%26aulast%3DSlipchenko%26aufirst%3DL.%2BV.%26aulast%3DLevchenko%26aufirst%3DS.%2BV.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DD.%2BP.%26aulast%3DDiStasio%26aufirst%3DR.%2BA.%26aulast%3DLochan%26aufirst%3DR.%2BC.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DBeran%26aufirst%3DG.%2BJ.%2BO.%26aulast%3DBesley%26aufirst%3DN.%2BA.%26aulast%3DHerbert%26aufirst%3DJ.%2BM.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DVan%2BVoorhis%26aufirst%3DT.%26aulast%3DChien%26aufirst%3DS.%2BH.%26aulast%3DSodt%26aufirst%3DA.%26aulast%3DSteele%26aufirst%3DR.%2BP.%26aulast%3DRassolov%26aufirst%3DV.%2BA.%26aulast%3DMaslen%26aufirst%3DP.%2BE.%26aulast%3DKorambath%26aufirst%3DP.%2BP.%26aulast%3DAdamson%26aufirst%3DR.%2BD.%26aulast%3DAustin%26aufirst%3DB.%26aulast%3DBaker%26aufirst%3DJ.%26aulast%3DByrd%26aufirst%3DE.%2BF.%2BC.%26aulast%3DDachsel%26aufirst%3DH.%26aulast%3DDoerksen%26aufirst%3DR.%2BJ.%26aulast%3DDreuw%26aufirst%3DA.%26aulast%3DDunietz%26aufirst%3DB.%2BD.%26aulast%3DDutoi%26aufirst%3DA.%2BD.%26aulast%3DFurlani%26aufirst%3DT.%2BR.%26aulast%3DGwaltney%26aufirst%3DS.%2BR.%26aulast%3DHeyden%26aufirst%3DA.%26aulast%3DHirata%26aufirst%3DS.%26aulast%3DHsu%26aufirst%3DC.-P.%26aulast%3DKedziora%26aufirst%3DG.%26aulast%3DKhalliulin%26aufirst%3DR.%2BZ.%26aulast%3DKlunzinger%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DA.%2BM.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DLiang%26aufirst%3DW.%2BZ.%26aulast%3DLotan%26aufirst%3DI.%26aulast%3DNair%26aufirst%3DN.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DProynov%26aufirst%3DE.%2BI.%26aulast%3DPieniazek%26aufirst%3DP.%2BA.%26aulast%3DRhee%26aufirst%3DY.%2BM.%26aulast%3DRitchie%26aufirst%3DJ.%26aulast%3DRosta%26aufirst%3DE.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26aulast%3DSimmonett%26aufirst%3DA.%2BC.%26aulast%3DSubotnik%26aufirst%3DJ.%2BE.%26aulast%3DWoodcock%26aufirst%3DH.%2BL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBell%26aufirst%3DA.%2BT.%26aulast%3DChakraborty%26aufirst%3DA.%2BK.%26aulast%3DChipman%26aufirst%3DD.%2BM.%26aulast%3DKeil%26aufirst%3DF.%2BJ.%26aulast%3DWarshel%26aufirst%3DA.%26aulast%3DHehre%26aufirst%3DW.%2BJ.%26aulast%3DSchaefer%26aufirst%3DH.%2BF.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DKrylov%26aufirst%3DA.%2BI.%26aulast%3DGill%26aufirst%3DP.%2BM.%2BW.%26aulast%3DHead-Gordon%26aufirst%3DM.%26atitle%3DAdvances%2520in%2520methods%2520and%2520algorithms%2520in%2520a%2520modern%2520quantum%2520chemistry%2520program%2520package%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D8%26spage%3D3172%26epage%3D3191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vq7','PDB','2vq7'); return false;">PDB: 2vq7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gwd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gwd','PDB','3gwd'); return false;">PDB: 3gwd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ucl" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ucl','PDB','3ucl'); return false;">PDB: 3ucl</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xve" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xve','PDB','2xve'); return false;">PDB: 2xve</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a href="/doi/suppl/10.1021/jm5007098">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28137"></div></div></div></div></div><hr /></hr><p class="last">Metabolism of diphenydramine in FMO3, CYP2D6 and CYP3A4; metabolism of compounds <b>15</b> and <b>16</b> in FMO3; main metabolites identified in FMO3 by HRMS; molecular formula strings (in SMILES) for all the molecules investigated in csv format. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5007098/suppl_file/jm5007098_si_001.pdf">jm5007098_si_001.pdf (407.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5007098/suppl_file/jm5007098_si_002.csv">jm5007098_si_002.csv (1.08 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm5007098&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2014.57.issue-14%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm5007098%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm5007098" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799ef6499d9198b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
