
<html lang="en"     class="pb-page"  data-request-id="0809e6e0-f0ee-4a01-b219-7176fecfbba5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-3;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.7b01406;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766)" /></meta><meta name="dc.Creator" content="Nicola E. A.  Chessum" /></meta><meta name="dc.Creator" content="Swee Y.  Sharp" /></meta><meta name="dc.Creator" content="John J.  Caldwell" /></meta><meta name="dc.Creator" content="A. Elisa  Pasqua" /></meta><meta name="dc.Creator" content="Birgit  Wilding" /></meta><meta name="dc.Creator" content="Giampiero  Colombano" /></meta><meta name="dc.Creator" content="Ian  Collins" /></meta><meta name="dc.Creator" content="Bugra  Ozer" /></meta><meta name="dc.Creator" content="Meirion  Richards" /></meta><meta name="dc.Creator" content="Martin  Rowlands" /></meta><meta name="dc.Creator" content="Mark  Stubbs" /></meta><meta name="dc.Creator" content="Rosemary  Burke" /></meta><meta name="dc.Creator" content="P. Craig  McAndrew" /></meta><meta name="dc.Creator" content="Paul A.  Clarke" /></meta><meta name="dc.Creator" content="Paul  Workman" /></meta><meta name="dc.Creator" content="Matthew D.  Cheeseman" /></meta><meta name="dc.Creator" content="Keith  Jones" /></meta><meta name="dc.Description" content="Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chemical probes and potential small molecule therapeutics. However, this can be ..." /></meta><meta name="Description" content="Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chemical probes and potential small molecule therapeutics. However, this can be ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 5, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01406" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01406" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01406" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01406" /></link>
        
    
    

<title>Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01406" /></meta><meta property="og:title" content="Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0009.jpeg" /></meta><meta property="og:description" content="Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chemical probes and potential small molecule therapeutics. However, this can be particularly challenging for poorly studied and noncatalytic proteins, as robust proximal biomarkers are rarely known. To confirm that our recently discovered chemical probe 1 (CCT251236) binds the putative transcription factor regulator pirin in living cells, we developed a heterobifunctional protein degradation probe. Focusing on linker design and physicochemical properties, we generated a highly active probe 16 (CCT367766) in only three iterations, validating our efficient strategy for degradation probe design against nonvalidated protein targets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01406"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01406">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01406&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01406&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01406&amp;href=/doi/10.1021/acs.jmedchem.7b01406" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 918-933</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01404" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01411" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicola+E.+A.++Chessum">Nicola E. A. Chessum</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-4125-320X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Swee+Y.++Sharp">Swee Y. Sharp</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+J.++Caldwell">John J. Caldwell</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=A.+Elisa++Pasqua">A. Elisa Pasqua</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-7966-4672" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Birgit++Wilding">Birgit Wilding</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-1896-3708" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giampiero++Colombano">Giampiero Colombano</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ian++Collins">Ian Collins</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8143-8498" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bugra++Ozer">Bugra Ozer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-1441-4162" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Meirion++Richards">Meirion Richards</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Rowlands">Martin Rowlands</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mark++Stubbs">Mark Stubbs</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-7855-9435" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rosemary++Burke">Rosemary Burke</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-0011-9701" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=P.+Craig++McAndrew">P. Craig McAndrew</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+A.++Clarke">Paul A. Clarke</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9342-1290" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Workman">Paul Workman</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-1659-3034" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Cheeseman">Matthew D. Cheeseman</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-1121-6985" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Keith++Jones">Keith Jones</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9440-4094" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">Cancer Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>For M.D.C.: phone, (+44) 208 722 4168; E-mail, <a href="/cdn-cgi/l/email-protection#f09d918484989587de9398959583959d919eb0999382de9193de859b"><span class="__cf_email__" data-cfemail="e5888491918d8092cb868d8080968088848ba58c8697cb8486cb908e">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>For P.A.C.: E-mail, <a href="/cdn-cgi/l/email-protection#c3b3a2b6afeda0afa2b1a8a683aaa0b1eda2a0edb6a8"><span class="__cf_email__" data-cfemail="661607130a48050a07140d03260f051448070548130d">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>For P.W.: E-mail, <a href="/cdn-cgi/l/email-protection#55253420397b223a273e38343b153c36277b34367b203e"><span class="__cf_email__" data-cfemail="35455440591b425a475e58545b755c56471b54561b405e">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>For K.J.: E-mail, <a href="/cdn-cgi/l/email-protection#d9b2bcb0adb1f7b3b6b7bcaa99b0baabf7b8baf7acb2"><span class="__cf_email__" data-cfemail="0368666a776b2d696c6d6670436a60712d62602d7668">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01406&amp;href=/doi/10.1021%2Facs.jmedchem.7b01406" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 918â933</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 14, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 September 2017</li><li><span class="item_label"><b>Published</b> online</span>5 January 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01406" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01406</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01406"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6911</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">51</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01406" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Nicola&quot;,&quot;last_name&quot;:&quot;E. A. Chessum&quot;},{&quot;first_name&quot;:&quot;Swee&quot;,&quot;last_name&quot;:&quot;Y. Sharp&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;J. Caldwell&quot;},{&quot;first_name&quot;:&quot;A.&quot;,&quot;last_name&quot;:&quot;Elisa Pasqua&quot;},{&quot;first_name&quot;:&quot;Birgit&quot;,&quot;last_name&quot;:&quot;Wilding&quot;},{&quot;first_name&quot;:&quot;Giampiero&quot;,&quot;last_name&quot;:&quot;Colombano&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;Collins&quot;},{&quot;first_name&quot;:&quot;Bugra&quot;,&quot;last_name&quot;:&quot;Ozer&quot;},{&quot;first_name&quot;:&quot;Meirion&quot;,&quot;last_name&quot;:&quot;Richards&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Rowlands&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;Stubbs&quot;},{&quot;first_name&quot;:&quot;Rosemary&quot;,&quot;last_name&quot;:&quot;Burke&quot;},{&quot;first_name&quot;:&quot;P.&quot;,&quot;last_name&quot;:&quot;Craig McAndrew&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;A. Clarke&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Workman&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Cheeseman&quot;},{&quot;first_name&quot;:&quot;Keith&quot;,&quot;last_name&quot;:&quot;Jones&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;918-933&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01406&quot;},&quot;abstract&quot;:&quot;Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chemical probes and potential small molecule therapeutics. However, this can be particularly challenging for poorly studied and noncatalytic proteins, as robust proximal biomarkers are rarely known. To confirm that our recently discovered chemical probe 1 (CCT251236) binds the putative transcription factor regulator pirin in living cells, we developed a heterobifunctional protein degradation probe. Focusing on linker design and physicochemical properties, we generated a highly active probe 16 (CCT367766) in only three iterations, validating our efficient strategy for degradation probe design against nonvalidated protein targets.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01406&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01406" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01406&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01406" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01406&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01406" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01406&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01406&amp;href=/doi/10.1021/acs.jmedchem.7b01406" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01406" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01406" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01406%26sid%3Dliteratum%253Aachs%26pmid%3D29240418%26genre%3Darticle%26aulast%3DChessum%26date%3D2018%26atitle%3DDemonstrating%2BIn-Cell%2BTarget%2BEngagement%2BUsing%2Ba%2BPirin%2BProtein%2BDegradation%2BProbe%2B%2528CCT367766%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D3%26spage%3D918%26epage%3D933%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292452" title="Chemical probes">Chemical probes</a>,</li><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/jmcmar.2018.61.issue-3/20180208/jmcmar.2018.61.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Demonstrating intracellular protein target engagement is an essential step in the development and progression of new chemical probes and potential small molecule therapeutics. However, this can be particularly challenging for poorly studied and noncatalytic proteins, as robust proximal biomarkers are rarely known. To confirm that our recently discovered chemical probe <b>1</b> (CCT251236) binds the putative transcription factor regulator pirin in living cells, we developed a heterobifunctional protein degradation probe. Focusing on linker design and physicochemical properties, we generated a highly active probe <b>16</b> (CCT367766) in only three iterations, validating our efficient strategy for degradation probe design against nonvalidated protein targets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Drug discovery is reliant on recombinant proteins and biochemical screens to develop structureâactivity relationships (SAR) and progress compounds.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> However, the conditions in biochemical assays often display little relevance to the intracellular environment, which can result in a failure to translate high target affinity to activity within a living cell.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Target-proximal biomarker modulation is the most important confirmation of intracellular target engagement.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Unfortunately, this is often not possible in early stage chemical probe or drug discovery projects, especially against novel biological targets, where poorly understood biology and pharmacology make it difficult to discover and robustly validate biomarkers, a process that is particularly challenging for noncatalytic proteins.</div><div class="NLM_p">Owing to the importance of confirming intracellular target engagement, several techniques have been developed. The overexpression of fusion proteins allows for a direct readout of target occupancy.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, the engineered cell lines are often difficult to generate and the protein label can impact compound binding. The cellular thermal shift assay (CETSA) is a label-free technique for intracellular target engagement.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> It exploits compound-induced stabilization of protein melting temperatures, but CETSA cannot be applied to all targets.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Activity-based protein profiling (ABPP) methods utilize nonselective irreversible covalent ligands, or intracellular fluorescence polarization probes, combined with intracellular reversible ligand competition, to study target engagement.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Proteolysis targeting chimeras (PROTACs)<a onclick="showRef(event, 'ref8 ref15'); return false;" href="javascript:void(0);" class="ref ref8 ref15">(8, 15)</a> and specific and nongenetic IAP-dependent protein erasers (SNIPERs)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> are heterobifunctional molecules that induce rapid and selective protein degradation, via the proteasome, within living cells. One portion of the molecule engages the target protein, while the other, attached via a flexible linker, recruits an E3 ligase to ubiquitinate the target, marking it for degradation as part of the cullinâRING finger machinery.<a onclick="showRef(event, 'ref10 ref15'); return false;" href="javascript:void(0);" class="ref ref10 ref15">(10, 15)</a></div><div class="NLM_p">We recently reported the development of a high affinity pirin chemical probe <b>1</b> (CCT251236, SPR <i>K</i><sub>D</sub> = 44 nM) discovered through a cell-based phenotypic screen for inhibitors of the heat shock transcription factor 1 (HSF1) stress pathway.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Pirin is an iron-binding member of the cupin super family of proteins and has been reported as a putative transcription factor regulator.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> It has no known enzymatic function in mammalian cells, no endogenous ligands have been reported, and no validated proximal biomarkers have been described.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> This makes demonstrating intracellular target engagement in living cells very challenging.</div><div class="NLM_p last">We hypothesized that we could demonstrate chemical probe <b>1</b> binding to pirin within living cells by developing a pirin-targeting protein degradation probe (PDP).</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75862" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75862" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Protein Degradation Probe Design</h3><div class="NLM_p">PDPs have been described against various proteins,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> with the bromodomain epigenetic target, BRD4, the most extensively studied.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> These heterobifunctional PDP molecules have utilized several ligands that bind to the E3 ligases: VHL,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> the IAP proteins,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and CRBN.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Despite the rapid expansion in PDP research, there remains no clear methodology to determine which proteins are amenable to PDP-mediated degradation,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> which E3 ligase ligand should be exploited or the optimal features of probe design. The structure of the linker, its length, and physicochemical properties have all been demonstrated to be important for PDP activity.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The linker controls the formation of the essential ternary complex,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> with evidence that it may stabilize the proteinâprotein interaction (PPI) between the target and the E3 ligase rather than forming a detached linear ternary complex (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), although it is unclear whether the ternary complex is part of the multiprotein cullinâRING finger complex.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23-25)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (top) Pirin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>)/CRBN (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CI1">5CI1</a>)/PDP ternary complex design model. The PDP can either stabilize a PPI or simply bring the proteins in close proximity depending on the role of the linker. (bottom left) Cartoon representation of the chemical probe <b>1</b> (yellow) bound to recombinant pirin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>). The cloud represents the shape of the binding pocket with key residues shown in black and the metal in orange. Red = oxygen, blue = nitrogen. Hydrogens and solvent are omitted for clarity except the water coordinated to the metal, shown as a red sphere. Both representations were generated using the PyMOL Molecular Graphics System, version 1.8; SchrÃ¶dinger, LLC. (bottom right) Key residues in the binding site and the clear solvent exposed vector for the chemical probe <b>1</b> binding to pirin are shown adapted from an analysis using MOE 2014.09. The ethyl pyrrolidine solubilizing group of chemical probe <b>1</b> was not resolved in the crystal structure and therefore is not drawn in the analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although we had discovered a high affinity ligand for pirin, there was no evidence that this protein would be a suitable substrate for ubiquitination by a PDP-recruited E3 ligase, without which extensive linker optimization could be futile. Therefore, we initially designed a synthetically tractable 15-atom linker that we predicted would not affect the affinity of the PDP for the isolated target proteins.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Analysis of the crystal structure of the chemical probe <b>1</b> bound to pirin (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>)<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> suggested that the solvent-exposed solubilizing group vector should be amenable to linker attachment. We selected a CRBN-targeting thalidomide ligand as the basis of our E3 ligase binding motif due to its low molecular weight. The CRBN-targeting ligand would be attached via the solvent exposed hydroxyl group of the 4-hydroxythalidomide analogue <b>2</b>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The linker would then attach the pirin- and CRBN-targeting specific binding groups via two amide moieties to give our first generation pirin-targeting PDP <b>3</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Design and Synthesis of the First Generation PDP <b>3</b> Based on the Chemical Probe <b>1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) PPh<sub>3</sub>, <sup>t</sup>butyl 2-hydroxyacetate, DTBAD, THF, 0 Â°C â RT, 16 h, 75%, (ii) HCO<sub>2</sub>H, DCM, 40 Â°C, 16 h, 54%, (iii) HATU, DIPEA, DMF, <sup>t</sup>butyl (2-(2-aminoethoxy)ethyl)carbamate, RT, 16 h, 81%, (iv) 4 M HCl in dioxane, 0.5 h, 100%; (b) ethyl 4-bromobutanoate, K<sub>2</sub>CO<sub>3</sub>, DMF, RT, 16 h, 37%; (c) Herrmannâs palladacycle,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a><sup><i>t</i></sup>Bu<sub>3</sub>PHBF<sub>4</sub>, MoCO<sub>6</sub>, DBU, 2-(trimethylsilyl)ethan-1-ol, 130 Â°C, 62%; (d) TBAF, THF, RT, 16 h; (e) (i) HATU, DIPEA, DMF, RT, 38% (over 2 steps), (ii) LiOHÂ·H<sub>2</sub>O, MeOH/THF/H<sub>2</sub>O, RT, 48 h, 18%; (f) HATU, DIPEA, DMF, 74%. For the synthesis of hydroxythalidomide <b>2</b>, see ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a>.</p></p></figure><div class="NLM_p last">The CRBN-targeting motif <b>4</b> of PDP <b>3</b> was synthesized from 4-hydroxythalidomide <b>2</b> in two steps and 84% yield, in a similar manner to that previously described.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The pirin-binding motif of the PDP <b>3</b> was synthesized from 6-bromoquinoline <b>5</b> via a palladium-mediated carbonylation reaction on the ether derivative <b>6</b>. Trapping the carbonylation intermediate with 2-(trimethylsilyl)ethan-1-ol gave the (trimethylsilyl)ethyl ester <b>7</b>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> which facilitated TBAF-selective hydrolysis. Amide coupling to the previously described bis-aniline derivative <b>8</b><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> was followed by hydrolysis of the aliphatic linker ester to give acid <b>9</b>. Final amide coupling to the CRBN-targeting derivative <b>4</b> gave the first-generation pirin-targeting PDP <b>3</b> in 10 steps and 0.4% overall yield.</div></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> The First Generation PDP</h3><div class="NLM_p">Analysis of the heterobifunctional PDP <b>3</b> revealed that it possessed good affinity for recombinant pirin (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 1) when measured using SPR, confirming the success of the rationally designed attachment vector.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The affinity of PDP <b>3</b> was then assessed against the CRBN-DDB1 complex, with DDB1 acting as a scaffolding protein, using an FP-assay similar to that previously described (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S6</a>).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> PDP <b>3</b> displayed moderate affinity for CRBN-DDB1, with <i>K</i><sub>i</sub> = 230 nM,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> comparable to the affinities of the parent CRBN ligands, thalidomide and lenalidomide (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S7</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Physicochemical Properties and Affinities for Recombinant Protein Targets of the Three Generations of PDPs<a class="ref internalNav" href="#t1fn1" aria-label="I">I</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0007.gif" alt="" id="GRAPHIC-d7e599-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0008.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn2"><div class="footnote" id="t1fn2"><sup>Table a</sup><p class="last">HBD = hydrogen bond donor count.</p></div><div class="footnote" id="t1fn3"><sup>Table b</sup><p class="last">ALogP was calculated using Biovia Pipeline Pilot, version 9.5, 2 SF.</p></div><div class="footnote" id="t1fn4"><sup>Table c</sup><p class="last">Log <i>D</i><sub>7.4</sub> measured using a HPLC-based method, <i>n</i> = 1, 2 SF.</p></div><div class="footnote" id="t1fn5"><sup>Table d</sup><p class="last">tPSA was calculated using ChemDraw (16.0.1.4) based on the O- and N-count, 3 SF.</p></div><div class="footnote" id="t1fn6"><sup>Table e</sup><p class="last">KS = kinetic solubility in pH 7.4 phosphate buffer at room temperature, <i>n</i> = 1, 1 SF.</p></div><div class="footnote" id="t1fn7"><sup>Table f</sup><p class="last"><i>K</i><sub>D</sub> values are reported to 2 SF and are calculated by equilibrium analysis using a one site specific binding model from SPR sensorgrams at equilibrium where possible, p<i>K</i><sub>D</sub> = âlog(<i>K</i><sub>D</sub> (M) Ã 10<sup>â9</sup>) and represents the geometric mean of <i>n</i> = 3 independent biological repeats.</p></div><div class="footnote" id="t1fn8"><sup>Table g</sup><p class="last">IC<sub>50</sub> values are reported to 2 SF and are calculated from an FP-assay doseâresponse curve to displace a thalidomide derived fluorescent probe using a log[inhibitor] vs response â variable slope (four parameters) model, pIC<sub>50</sub> = âlog(IC<sub>50</sub> (M) Ã 10<sup>â9</sup>) and represents the geometric mean of <i>n</i> = 3 independent biological repeats, also see ref <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">31</a>.</p></div><div class="footnote" id="t1fn9"><sup>Table h</sup><p class="last"><i>K</i><sub>i</sub> values are calculated from the geometric mean CRBN-DDB1 complex IC<sub>50</sub> and the FP-probe <i>K</i><sub>D</sub> using methods described in ref <a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">32</a>.</p></div><div class="footnote" id="t1fn1"><sup>Table I</sup><p class="last">SF = significant figure. All data was reprocessed using GraphPad Prism 7.01. See Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figures S8âS16</a>. SEM = standard error of the mean.</p></div></div><div></div></div><div class="NLM_p last">Following confirmation that both binding motifs of the PDP <b>3</b> retained high affinity for their respective targets, we then investigated its activity against pirin in human cancer cell lines. Several cell lines were assessed for CRBN expression by quantitative capillary electrophoresis (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S17</a>).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The SK-OV-3 ovarian carcinoma cell line<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> displayed good basal CRBN and also pirin expression, and there was no observable depletion of pirin in these cells when treated with chemical probe <b>1</b> (1 Î¼M, data not shown), so this line was selected for further study. Unfortunately, treatment of SK-OV-3 cells with PDP <b>3</b>, at high concentrations (>1 Î¼M) and for extended time periods (>48 h), resulted in no measurable effects on the cancer cells (data not shown).</div></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> The Second Generation PDP</h3><div class="NLM_p">We speculated several causes for the failure of the first generation PDP <b>3</b>. Pirin may simply be incompatible with this methodology, and if so, no further improvements could be made.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Alternatively, CRBN could be the wrong E3 ligase target to deplete pirin,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> or the linker length could be inconsistent with formation of the ternary complex.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Finally, the physicochemical properties of PDP <b>3</b> could be limiting its intracellular free concentration.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Given these variables, designing a PDP against a protein target that has not previously been validated as being susceptible to E3 ligase-directed degradation is challenging, as optimization cycles can appear lengthy and the difficult and low yielding synthesis discourages the generation of multiple analogues. It was also unclear how strict and narrow the requirements for optimal PDP design would be. We hypothesized that the physicochemical properties of the PDP were the primary cause of the failure of the first generation probe, and so began a redesign process that should increase cell membrane flux while maintaining the same linker length and CRBN-targeting ligand (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). By carrying out multiple changes to the PDP, we aimed to minimize the number of iterative design cycles, so we could more rapidly validate this approach.</div><div class="NLM_p">The design of heterobifunctional molecules unavoidably results in high molecular weight compounds, making it difficult to balance their physicochemical properties in a manner consistent with acceptable permeability and solubility.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In the case of the first generation pirin-targeting PDP <b>3</b>, although the Log <i>D</i><sub>7.4</sub> was acceptable (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 1),<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> the linker had introduced two new hydrogen bond donors (HBD), which can have a negative impact on permeability. The calculated tPSA was very high (258 Ã<sup>2</sup>), although it is inevitable that tPSA will be high for large molecules and outside the standard cut-offs for cell permeability unless the compounds are highly lipophilic.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> In designing a second generation PDP, we followed standard medicinal chemistry principles to reduce the tPSA and HBD count while maintaining an acceptable Log <i>D</i><sub>7.4</sub>. To achieve this, we redesigned the ether linker in the first generation PDP <b>3</b> to include a methylene piperazine that would project into solvent based on analysis of the crystal structure of the chemical probe <b>1</b> bound to pirin. The resulting tertiary amide would remove one HBD. We also sought to mask the quinoline amide HBD, using a dipoleâdipole interaction, via a bioisosteric replacement with fluorine.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> These changes resulted in the design of the second generation pirin-targeting PDP <b>10</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) with a modestly reduced tPSA (244 Ã<sup>2</sup>).</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Second (<b>10</b>) and Third (<b>16</b>) Generation Probes and Control Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) X = F <b>12</b>, 2-methylquinoline carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine, 18 h quant, (ii) Fe(0), NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O, 90 Â°C, 1 h, quant; (b) 2,3-dihydrobenzo-[<i>b</i>][1,4]-dioxine-6-carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine, RT, 2 h, 85%; (c) (i) SeO<sub>2</sub>, 1,4-dioxane/DMF, reflux, 1 h, (ii) <i>N</i>-Boc-piperazine, DCM, RT, 12 h then NaBH(OAc)<sub>3</sub>, DCM, RT, 2 h, 94% (over 2 steps), (iii) TFA, DCM, RT, 2 h, 69%; (d) (i) <b>2</b>, PPh<sub>3</sub>, <sup>t</sup>butyl 2-hydroxyacetate, DTBAD, THF, 0 Â°C â RT, 16 h, 75%, (ii) HCO<sub>2</sub>H, DCM, 40 Â°C, 16 h, 54%, (iii) HATU, DIPEA, DMF, <sup>t</sup>butyl 3-(2-aminoethoxy)propanoate, RT, 16 h, 72%, (iv) HCO<sub>2</sub>H, DCM, 40 Â°C, 6 h, 93%; (e) HATU, DIPEA, DMF, RT, 16 h, 52%; (f) (i) X = Cl <b>17</b>, 2-methylquinoline carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine 2 h, 88%, (ii) Fe(0), NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O, 90 Â°C, 1 h, quant; (g) 2,3-dihydrobenzo-[<i>b</i>][1,4]-dioxine-6-carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine 2 h, 61%; (h) (i) SeO<sub>2</sub>, DMF, 1,4-dioxane, 50 Â°C, 16 h, (ii) <i>N</i>-Boc-piperazine, NaBH<sub>3</sub>CN, AcOH, DMF, 0 Â°C â RT, 16 h, (iii) 4 M HCl in dioxane, MeOH 0 Â°C â RT, 16 h, 32% over 3 steps; (i) 2-[2-(2-hydroxyethoxy)ethoxy]ethyl 4-methylbenzenesulfonate, K<sub>2</sub>CO<sub>3</sub>, DMF, RT, 16 h, 48%; (j) PPh<sub>3</sub>, DTBAD, THF, RT, 2 h, 27%; (k) (i) <b>23</b>, 2,3-dihydro-1,4-benzodioxine-5-carboxylic acid, oxalyl chloride, DMF, DCM, RT, 2 h, then pyridine 48 h, 84%, (ii) Pd/C, EtOH/DCM, H<sub>2</sub> (1 atm), 77%, (iii) 2-methylquinoline carboxylic acid, oxalyl chloride, DMF, DCM, RT, 2 h, then pyridine 16 h, 58%, then same procedure as from <b>18</b>, 6% yield over 5 steps. (l) (i) SeO<sub>2</sub>, 1,4-dioxane/DMF, 50 Â°C, 5.5 h, (ii) <i>N</i>-ethylpiperazine, DCM, RT, 20 h, then NaBH(OAc)<sub>3</sub>, DCM, RT, 2 h, 35% (over 2 steps).</p></p></figure><div class="NLM_p">The synthesis of PDP <b>10</b> began from the fluoroaniline carboxamide <b>11</b>, synthesized in a similar manner to that previously described from 2-fluoro-5-nitroaniline <b>12</b> in three steps and 85% yield. Amide coupling to give bisamide <b>13</b> and benzylic oxidation using selenium dioxide<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was followed by reductive amination of the resulting aldehyde with <i>N</i>-Boc-piperazine, and subsequent deprotection, to give <b>14</b>. The CRBN-targeting thalidomide derivative precursor <b>15</b> was prepared in four steps and 27% yield from 4-hydroxythalidomide <b>2</b>. Final amide coupling with monosubstituted piperazine <b>14</b> gave the second generation pirin-targeting PDP <b>10</b> in 11 steps and 8% overall yield (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><div class="NLM_p">PDP <b>10</b> displayed a similar affinity for recombinant pirin and CRBN-DDB1 to our first generation probe <b>3</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 2), so it was progressed to cellular assessment of pirin degradation. Pleasingly, treatment of SK-OV-3 ovarian cancer cells with <b>10</b> at 3.0 Î¼M total concentration for up to 48 h revealed a clear and time-dependent depletion of intracellular pirin expression (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S18</a>).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> This confirmed, for the first time, that pirin is amenable to modulation using a PDP and that the bisamide chemotype not only binds recombinant pirin with high affinity but also binds pirin within living cells.</div><div class="NLM_p last">Although demonstrating pirin depletion with the second generation probe <b>10</b> was a very encouraging result, we found the effects were poorly reproducible. The concentrations of PDP <b>10</b> needed to observe pirin degradation were high and close to its kinetic solubility (KS). Furthermore, at least 24 h of compound exposure was needed before pirin degradation was observed (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S18</a>). To explore the cause of the variable results obtained with PDP <b>10</b>, we assessed its chemical stability. At room temperature, PDP <b>10</b> was stable as a solid and in DMSO stock solution (>1 month, data not shown), but at 37 Â°C in pH 7.4 phosphate buffer, consistent with the cell assay conditions, it underwent rapid decomposition (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Table S1</a>), displaying a half-life of only â¼4 h. This poor chemical stability was consistent with the known decomposition of the parent CRBN-targeting thalidomide ligand under these conditions, where multiple hydrolysis products of the imide and glutaramide moieties are observed.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The chemical probe <b>1</b> displayed no instability under these conditions; therefore, we concluded that the facile hydrolysis of the CRBN-targeting motif limited the reproducibility of our slow-acting second generation pirin-targeting PDP <b>10</b>.</div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> The Third Generation PDP</h3><div class="NLM_p">In the design of a third generation probe, we aimed to increase permeability further. This should result in higher intracellular free concentrations more quickly, mitigating the poor stability of the CRBN-targeting motif. We decided to carry out a bioisosteric replacement of the central ring fluorine for the larger and more sterically hindering chlorine substituent.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> However, we were concerned that the increase in lipophilicity from this exchange would negatively impact both the solubility and permeability of the probe. Large lipophilic heterobifunctional molecules are particularly susceptible to aggregation,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and this decreases the free concentration that drives cell membrane flux.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> To balance the lipophilicity, we removed the tertiary amide bond to the piperazine, introducing a cationic amine, which would be substantially charged at pH 7.4 (MoKa, version 2.5.2, p<i>K</i><sub>a</sub> = 8.0).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The second amide in the linker was also removed, reducing the HBD count further and increasing the overall flexibility of the ligand, consistent with the formation of the crucial PDP ternary protein complex. The linker length was reduced by one atom to accommodate these changes and resulted in the design of the third generation pirin-targeting PDP <b>16</b> (CCT367766), which now displayed a notably reduced, but still high, tPSA (207 Ã<sup>2</sup>).</div><div class="NLM_p">The synthesis of the pirin-targeting motif of the third generation PDP <b>16</b> was carried out in a similar manner to that previously described starting from 2-chloro-5-nitroaniline <b>17</b> to give chlorobisamide <b>18</b> in three steps and 54% yield (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Oxidation of the methylquinoline moiety with SeO<sub>2</sub> and reductive amination of the resulting aldehyde with <i>N</i>-Boc-piperazine and <i>N</i>-Boc deprotection gave <b>19</b> in 32% yield. Following S<sub>N</sub>2 alkylation with the ether linker to give <b>20</b>, selective Mitsunobu alkylation with 4-hydroxythalidomide <b>2</b> gave the third generation PDP <b>16</b> in eight steps and 2% overall yield. A nonpirin-binding negative control matched pair compound, PDP <b>21</b>, based on our negative control pirin chemical probe,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> was synthesized from regioisomer <b>23</b> in a similar manner in 2% overall yield. A non-CRBN binding control <b>22</b><a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> was also synthesized from <b>18</b>, utilizing reductive amination with <i>N</i>-ethylpiperazine in 35% yield (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><div class="NLM_p">Analysis of the third generation PDP <b>16</b> confirmed that it retained acceptable lipophilicity (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 3) but also displayed a 4.2-fold increase in affinity for recombinant pirin compared to the second generation PDP <b>10</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A) and comparable affinity for CRBN (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). SK-OV-3 cells were then treated with PDP <b>16</b> at concentrations from 50 to 1500 nM for up to 24 h (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Representative SPR sensorgram of the third generation PDP <b>16</b> and recombinant pirin. (B) Representative binding curve of PDP <b>16</b> in the CRBN-DDB1 FP-assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Immunoblot of SK-OV-3 human ovarian cancer cells demonstrating the depletion of pirin protein using the third generation PDP <b>16</b> and the time-dependent hook-effect. (B) Immunoblot demonstrating the concentration-dependent depletion of pirin protein after 2 h exposure in SK-OV-3 cells. (C) Capillary electrophoresis and immunoassay were used to quantify the pirin protein expression after 2 h exposure with PDP <b>16</b>, and all values are normalized to vinculin loading control and relative to the measured basal pirin protein expression, all bars represent the arithmetic mean of <i>n</i> = 3 independent biological repeats, error bars are SEM. (D) Proteomics analysis of the third generation PDP <b>16</b> (50 nM) exposure (4 h) in SK-OV-3 cells compared to vehicle control, using a tandem mass tagging (TMT) MS2 protocol on the cell lysate, 8547 quantifiable proteins were identified; each blue dot represents a single quantifiable protein, pirin is marked in red (adjusted <i>p</i> value = 1.4 Ã 10<sup>â4</sup>), <i>p</i> values were calculated using a linear modeling based <i>t</i> test and corrected for multiple comparisons using the BenjaminiâHochberg method to give the <i>p</i>(adj) values shown, dotted lines represent 2-fold depletion of the protein and a <i>p</i>(adj) = 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to our earlier generation PDPs, near complete pirin degradation was now observed with just 50 nM treatment and only 2 h exposure (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). Increasing the total initial concentration of pirin-targeting PDP <b>16</b> resulted in a clear hook effect. This bell-shaped concentrationâresponse is consistent with the formation of a ternary complex.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Interestingly the hook effect was seen to decrease over time (24 h), possibly either due to the slower degradation at high concentration or from the depletion of PDP <b>16</b> due to thalidomide hydrolysis (half-life = â¼3 h, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S1</a>), which would reduce the effective free concentration below the negative cooperativity threshold of the ternary complex.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Pirin degradation was subsequently shown to be concentration-responsive with activity at concentrations as low as 0.5 nM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B), which was confirmed with a quantitative capillary electrophoresis-based immunoassay (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). The negative control benzodioxane regioisomer PDP <b>21</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 4) displayed no pirin depletion at equimolar concentrations (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S19</a>),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and degradation was also confirmed to be proteasome-dependent by rescue following preincubation with the proteosome inhibitor, MG132 (500 nM, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S19</a>).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> We then carried out whole proteome mass spectrometry to estimate the cellular selectivity of the pirin-targeting PDP <b>16</b> in an unbiased manner, quantified using tandem mass tagging (TMT) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D, <a href="http://www.proteomics.com" class="extLink">www.proteomics.com</a>). After treating SK-OV-3 cells with 50 nM PDP <b>16</b> for 4 h, and comparing to vehicle treated cells using BenjaminiâHochberg corrected <i>p</i> values, we found that from 8547 quantifiable proteins identified, only pirin (2.3-fold reduction, <i>p</i>(adj) = 1.4 Ã 10<sup>â4</sup>) displayed a statistically significant (<i>p</i>(adj) < 0.05)<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> difference in protein expression.</div><div class="NLM_p">To confirm that our chemical probe <b>1</b> also bound pirin within SK-OV-3 cancer cells, we carried out competition experiments designed to rescue pirin depletion by PDP <b>16</b>. The concentrations required for a mutually exclusive binding ligand to displace a probe molecule are dependent on the affinity of the ligand relative to the ratio of the free concentration of the probe and its affinity for the protein target complex.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Because the depletion of pirin is a nonequilibrium event that does not necessarily require complete target occupancy, it is more difficult to observe competition at later time points owing to continued protein turnover.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Pretreating SK-OV-3 cells with 10 Î¼M thalidomide, as a CRBN-binding competitive ligand, demonstrated that after 2 h treatment with PDP <b>16</b> (5 nM), we successfully rescued pirin depletion (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Figure S19</a>). The non-CRBN-binding control chlorobisamide <b>22</b> displayed high affinity for recombinant pirin (SPR, <i>K</i><sub>D</sub> = 21 nM, p<i>K</i><sub>D</sub> = 7.68 Â± 0.03, <i>n</i> = 3). Pretreatment of SK-OV-3 cells with chlorobisamide <b>22</b> displayed concentration-dependent rescue of pirin expression following treatment with the PDP <b>16</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, top), with complete rescue observed at 1 Î¼M. Finally, pretreatment with the chemical probe <b>1</b> demonstrated clear rescue of pirin depletion, confirming intracellular target engagement (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, bottom).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Intracellular competition studies with PDP <b>16</b> and the chemical probes. SK-OV-3 cells were pretreated with increasing concentrations of chemical probe for 4 h before exposing to PDP <b>16</b> for 2 h at the concentrations shown. Cells were then lysed and protein expression analyzed using immunoblot. For clarity, gel images have been cropped where appropriate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Exploiting cell-based phenotypic screens to identify new disease-associated therapeutic targets is an increasingly frequent strategy in drug discovery. While this approach can identify novel targets with unique mechanisms of action, these proteins are often poorly characterized and can lack identifiable enzymatic activity, ligands, and biomarkers of target engagement. The main focus of research into PDPs has been as potential therapeutics with novel mechanisms of action. We designed a PDP as an intracellular probe against the poorly understood and noncatalytic molecular target, pirin. Developing PDPs to confirm intracellular target engagement, and potentially develop intracellular SAR, against challenging proteins, is an important addition to the current methods for compound profiling.</div><div class="NLM_p last">For PDPs to be used as target engagement probes, their rapid development and validation is crucial. The ideal strategies for efficient and successful PDP design are still under investigation and will clearly improve as more protein targets are modulated and additional crystallographic evidence of the target protein/E3 ligase/PDP ternary complexes are discovered. The number of variables involved in PDP design against nonvalidated target proteins can make the process daunting. By focusing on the physicochemical properties of our probe molecules, in only three iterations, we developed a selective degradation probe that eliminates pirin at low concentration and in a short time period. This confirmed our chemical probe <b>1</b> does bind pirin in an intracellular environment, and PDP <b>16</b> provides another chemical tool to study a largely unexplored protein.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52270" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52270" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General Experimental</h3><div class="NLM_p">Unless otherwise stated, reactions were conducted in oven-dried glassware under an atmosphere of nitrogen or argon using anhydrous solvents. All commercially obtained reagents and solvents were used as received. Thin layer chromatography (TLC) was performed on precoated aluminum sheets of silica (60 F254 nm, Merck) and visualized using short-wave UV light. Flash column chromatography was carried out on Merck silica gel 60 (partial size 40â65 Î¼m). Column chromatography was also performed on a Biotage SP1 or Biotage Isolera Four purification system using Biotage Flash silica cartridges (SNAP KP-Sil) or for reverse phase purifications SNAP Ultra C18 cartridges. Ion exchange chromatography was performed using acidic Isolute Flash SCX-II columns. Semipreparative HPLC was performed on an Agilent 6120 system, flow 20 mL/min, eluents 0.1% acetic acid in water and 0.1% acetic acid in methanol, gradient of 10â100% organic phase. Lipophilic method: Chromatographic separation at room temperature was carried out using a 1200 series preparative HPLC (Agilent, Santa Clara, USA) over a 15 min gradient elution (gradient 15 min, 20 mL) from 60:40 to 0:100 water:methanol (both modified with 0.1% formic acid) at a flow rate of 20 mL/min. <sup>1</sup>H NMR spectra were recorded on Bruker AMX500 (500 MHz) spectrometers using an internal deuterium lock. Chemical shifts are quoted in parts per million (ppm) using the following internal references: CDCl<sub>3</sub> (Î´H 7.26), MeOD (Î´H 3.31), and DMSO-<i>d</i><sub>6</sub> (Î´H 2.50). Signal multiplicities are recorded as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), doublet of doublets (dd), doublet of doublet of doublets (ddd), broad (br), or obscured (obs). Coupling constants, <i>J</i>, are measured to the nearest 0.1 Hz. <sup>13</sup>C NMR spectra were recorded on Bruker AMX500 spectrometers at 126 MHz using an internal deuterium lock. Chemical shifts are quoted to 0.01 ppm, unless greater accuracy was required, using the following internal references: CDCl<sub>3</sub> (Î´C 77.0), MeOD (Î´C 49.0), and DMSO-<i>d</i><sub>6</sub> (Î´C 39.5). High resolution mass spectra were recorded on an Agilent 1200 series HPLC and diode array detector coupled to a 6210 time-of-flight mass spectrometer with dual multimode APCI/ESI source or on a Waters Acquity UPLC and diode array detector coupled to a Waters G2 QToF mass spectrometer fitted with a multimode ESI/APCI source. All compounds were >95% purity by LCMS analysis unless otherwise stated.</div><div class="NLM_p">Thalidomide (<a href="http://www.sigmaaldrich.com/catalog/product/sigma/t144?lang=en&amp;region=GB" class="extLink">http://www.sigmaaldrich.com/catalog/product/sigma/t144?lang=en&region=GB</a>, accessed August 29, 2017), Lenalidomide (<a href="http://www.sigmaaldrich.com/catalog/product/aldrich/cds022536?lang=en&amp;region=GB" class="extLink">http://www.sigmaaldrich.com/catalog/product/aldrich/cds022536?lang=en&region=GB</a>, accessed August 29, 2017) and MG132 (<a href="http://www.sigmaaldrich.com/catalog/product/sigma/m8699?lang=en&amp;region=GB" class="extLink">http://www.sigmaaldrich.com/catalog/product/sigma/m8699?lang=en&region=GB</a>, accessed August 29, 2017) were purchased from Sigma-Aldrich and used without further purification.</div><div class="NLM_p">The pirin chemical probe <b>1</b> CCT251236 was synthesized using the previously described procedure.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div id="sec4-1-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> 2-(2,6-Dioxopiperidin-3-yl)-4-hydroxyisoindoline-1,3-dione <b>2</b></h4><div class="NLM_p last">3-Aminopiperidine-2,6-dione hydrochloride (300 mg, 1.82 mmol) was added to a solution of 4-hydroxyisobenzofuran-1,3-dione (299 mg, 1.82 mmol) and triethylamine (0.38 mL, 2.73 mmol) in anhydrous THF (36.5 mL) and the reaction heated to reflux for 24 h under inert atmosphere. The reaction mixture was allowed to cool to room temperature, then 1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide (384 mg, 2.00 mmol) and DMAP (22 mg, 0.18 mmol) were added. The reaction was heated to reflux for 24 h. Then, the reaction mixture was allowed to cool, and the solvents were removed under reduced pressure. The resulting residue was taken up in methanol and passed sequentially through two ion exchange columns (Isolute SCX-II), eluting with methanol to afford the title compound as an amorphous off-white solid (422 mg, 84%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.18 (s, 1H), 11.09 (s, 1H), 7.65 (dd, <i>J</i> = 8.4, 7.2 Hz, 1H), 7.32 (dd, <i>J</i> = 7.1, 0.7 Hz, 1H), 7.25 (dd, <i>J</i> = 8.4, 0.7 Hz, 1H), 5.07 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 2.88 (ddd, <i>J</i> = 17.0, 13.9, 5.5 Hz, 1H), 2.64â2.46 (m, 2H), 2.02 (dtd, <i>J</i> = 13.0, 5.3, 2.2 Hz, 1H). LCMS (ESI<sup>+</sup>) RT = 0.77 min, 100%, M + H<sup>+</sup> 275, M + Na<sup>+</sup> 297.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> This compound is also commercially available from several suppliers.</div></div><div id="sec4-1-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> <i>tert</i>-Butyl 2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetate</h4><div class="NLM_p last">Triphenylphosphine (0.261 g, 0.996 mmol) and <i>tert</i>-butyl 2-hydroxyacetate (101 mg, 0.77 mmol) were dissolved in anhydrous THF (3 mL) while stirring at 0 Â°C. Then a solution of DTBAD (229 mg, 0.10 mmol) in anhydrous THF (2 mL) was added dropwise. Finally, a solution of (2-(2,6-dioxopiperidin-3-yl)-4-hydroxyisoindoline-1,3-dione) <b>2</b> in anhydrous THF (3 mL) was added. The mixture was stirred for 1 h at 0 Â°C then allowed to warm to room temperature and stirred overnight. Then the solvent was removed under reduced pressure. The crude product was purified by Biotage chromatography using a gradient of 0â50% EtOAc in cyclohexane to afford the title compound as an amorphous white solid (224 mg, 0.577 mmol, 75% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.11 (s, 1H), 7.80 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.48 (d, <i>J</i> = 7.2 Hz, 1H), 7.38 (d, <i>J</i> = 8.5 Hz, 1H), 5.10 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.97 (s, 2H), 2.89 (ddd, <i>J</i> = 17.0, 13.9, 5.4 Hz, 1H), 2.64â2.52 (m, 2H), 2.07â2.00 (m, 1H), 1.43 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 172.78, 169.89, 167.14, 166.71, 165.12, 155.03, 136.76, 133.25, 119.96, 116.44, 115.90, 81.92, 65.50, 48.80, 30.95, 27.68, 21.97. HRMS (ESI<sup>+</sup>): calcd for C<sub>15</sub>H<sub>13</sub>N<sub>2</sub>O<sub>7</sub> (M + H, â<i><sup>t</sup></i>Bu)<sup>+</sup> 333.0717, found 333.0722.</div></div><div id="sec4-1-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic Acid</h4><div class="NLM_p last"><i>tert</i>-Butyl 2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetate (1.30 g, 3.35 mmol) was dissolved in anhydrous DCM, then formic acid (12.8 mL, 335 mmol) was added while stirring at room temperature. The reaction was stirred overnight at 40 Â°C. The solvents were removed under reduced pressure and the resulting residue purified by reverse phase Biotage chromatography using a gradient of 0â50% MeOH in water + 0.1% formic acid to afford the title compound as an amorphous white solid (600 mg, 1.81 mmol, 54% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 13.29 (s, 1H), 11.11 (s, 1H), 7.79 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.47 (d, <i>J</i> = 7.2 Hz, 1H), 7.39 (d, <i>J</i> = 8.6 Hz, 1H), 5.10 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.98 (s, 2H), 2.89 (ddd, <i>J</i> = 16.9, 13.9, 5.4 Hz, 1H), 2.64â2.52 (m, 2H), 2.04 (dtd, <i>J</i> = 13.0, 5.4, 2.3 Hz, 1H). LCMS (ESI<sup>+</sup>): RT = 0.71 min, 100%, M + Na<sup>+</sup> 355.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div><div id="sec4-1-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>tert</i>-Butyl 3-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)propanoate</h4><div class="NLM_p last">2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic acid (217 mg, 0.65 mmol), <i>tert</i>-butyl 3-(2-aminoethoxy)propanoate (124 mg, 0.65 mmol), and DIPEA (0.34 mL, 1.96 mmol) were dissolved in anhydrous DMF (3.27 mL) at room temperature under inert atmosphere. HATU (230 mg, 0.98 mmol) was added and the reaction stirred overnight. The solvents were removed under reduced pressure, and the resulting residue was purified by reverse phase Biotage chromatography using a gradient of 10â90% MeOH in water + 0.1% formic acid to afford the title compound as an amorphous white solid (236 mg, 72%). <sup>1</sup>H NMR (500 MHz, MeOD) Î´ 7.82 (dd, <i>J</i> = 8.4, 7.4 Hz, 1H), 7.55 (dd, <i>J</i> = 7.3, 0.5 Hz, 1H), 7.46â7.43 (m, 1H), 5.16 (dd, <i>J</i> = 12.6, 5.5 Hz, 1H), 4.78 (s, 2H), 3.71 (t, <i>J</i> = 6.2 Hz, 2H), 3.62â3.56 (m, 2H), 3.52â3.48 (m, 2H), 2.90 (ddd, <i>J</i> = 18.0, 14.4, 5.2 Hz, 1H), 2.82â2.70 (m, 2H), 2.50 (td, <i>J</i> = 6.2, 3.8 Hz, 2H), 2.16 (dddd, <i>J</i> = 10.8, 7.9, 5.1, 2.4 Hz, 1H), 1.42 (s, 9H). <sup>13</sup>C NMR (126 MHz, MeOD) Î´ 174.55, 172.81, 171.33, 170.00, 168.31, 167.59, 156.21, 138.16, 134.96, 121.60, 119.30, 117.89, 81.75, 70.01, 69.26, 67.75, 50.56, 40.15, 37.22, 32.20, 28.34, 23.66. HRMS (ESI<sup>+</sup>): calcd for C<sub>24</sub>H<sub>30</sub>N<sub>3</sub>O<sub>9</sub> (M + H)<sup>+</sup> 504.1982, found 504.1981.</div></div><div id="sec4-1-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>tert</i>-Butyl (2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethyl)carbamate</h4><div class="NLM_p last">2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic acid (0.19 g, 0.56 mmol), <i>tert</i>-butyl (2-(2-aminoethoxy)ethyl)carbamate (0.11 g, 0.56 mmol), and DIPEA (0.29 mL, 1.68 mmol) were dissolved in anhydrous DMF (2.80 mL) under inert atmosphere at room temperature. HATU (198 mg, 0.84 mmol) was added and the reaction stirred for 16 h. The reaction mixture was concentrated in vacuo and the resulting residue purified by Biotage chromatography using a gradient of 0â2% methanol in ethyl acetate. However, this material was not pure, therefore the material was purified by further Biotage chromatography using a gradient of 1â10% EtOH in DCM to afford the title compound as an amorphous white solid (243 mg, 81%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.11 (s, 1H), 8.02 (t, <i>J</i> = 5.4 Hz, 1H), 7.81 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.49 (d, <i>J</i> = 7.2 Hz, 1H), 7.39 (d, <i>J</i> = 8.5 Hz, 1H), 6.76 (t, <i>J</i> = 4.7 Hz, 1H), 5.12 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.79 (s, 2H), 3.44 (t, <i>J</i> = 5.6 Hz, 2H), 3.38 (t, <i>J</i> = 6.1 Hz, 2H), 3.31 (d, <i>J</i> = 5.5 Hz, 2H), 3.08 (q, <i>J</i> = 5.6 Hz, 2H), 2.89 (ddd, <i>J</i> = 16.6, 13.8, 5.3 Hz, 1H), 2.65â2.51 (m, 2H), 2.08â1.98 (m, 1H), 1.36 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 172.73, 169.85, 166.88, 166.80, 166.72, 165.45, 155.59, 155.00, 136.92, 133.03, 120.34, 116.75, 116.02, 77.63, 69.02, 68.55, 67.48, 48.80, 38.38, 30.95, 28.22, 21.99. HRMS (ESI<sup>+</sup>): calcd for C<sub>24</sub>H<sub>30</sub>N<sub>4</sub>O<sub>9</sub>Na (M + Na)<sup>+</sup> 541.1911, found 541.1915.</div></div><div id="sec4-1-6" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(4-((2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethyl)amino)-4-oxobutoxy)quinoline-6-carboxamide <b>3</b></h4><div class="NLM_p last"><i>N</i>-(2-(2-Aminoethoxy)ethyl)-2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamide hydrochloride <b>4</b> (21.0 mg, 0.046 mmol), 4-((6-((5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)carbamoyl)quinolin-2-yl)oxy)butanoic acid (25.0 mg, 0.046 mmol), and DIPEA (0.032 mL, 0.184 mmol) were dissolved in anhydrous DMF (0.23 mL) under inert atmosphere. HATU (16.0 mg, 0.069 mmol) was added and the reaction left to stir at room temperature for 2 h. The reaction was concentrated in vacuo, then purified by reverse phase Biotage chromatography using a gradient of 20â100% MeOH in water + 0.1% formic acid to afford the title compound as a beige solid (32 mg, 74%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.09 (s, 1H), 10.07 (s, 1H), 10.06 (s, 1H), 8.56 (d, <i>J</i> = 1.9 Hz, 1H), 8.37 (d, <i>J</i> = 8.8 Hz, 1H), 8.21 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 8.01 (t, <i>J</i> = 5.5 Hz, 1H), 7.91 (t, <i>J</i> = 5.5 Hz, 1H), 7.87â7.83 (m, 2H), 7.80 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.58 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.48 (d, <i>J</i> = 7.2 Hz, 1H), 7.39 (d, <i>J</i> = 8.5 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 1H), 7.08 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 5.12 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.79 (s, 2H), 4.42 (t, <i>J</i> = 6.5 Hz, 2H), 4.33â4.26 (m, 4H), 3.45 (t, <i>J</i> = 5.7 Hz, 2H), 3.42 (t, <i>J</i> = 5.9 Hz, 2H), 3.36â3.28 (m, <i>J</i> = 5.5 Hz, 2H, obscured by water peak), 3.24 (q, <i>J</i> = 5.8 Hz, 2H), 2.89 (ddd, <i>J</i> = 16.3, 13.6, 5.1 Hz, 1H), 2.64â2.52 (m, 2H), 2.29 (t, <i>J</i> = 7.3 Hz, 2H), 2.23 (s, 3H), 2.11â1.93 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 172.74, 171.66, 169.87, 166.90, 166.71, 165.44, 164.91, 164.34, 162.79, 154.96, 147.58, 146.33, 142.92, 140.16, 137.28, 136.90, 136.33, 133.01, 130.10, 130.00, 128.75, 128.37, 128.10, 127.68, 126.72, 123.99, 121.19, 120.32, 118.61, 118.11, 116.83, 116.74, 116.65, 116.02, 113.94, 68.91, 68.58, 67.48, 65.38, 64.39, 64.02, 48.80, 38.44, 38.33, 31.78, 30.94, 24.62, 21.99, 17.48. HRMS: calcd for C<sub>49</sub>H<sub>48</sub>N<sub>7</sub>O<sub>13</sub> (M + H)<sup>+</sup> 942.3305, found 942.3322.</div></div><div id="sec4-1-7" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>N</i>-(2-(2-Aminoethoxy)ethyl)-2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamide <b>4</b></h4><div class="NLM_p last"><i>tert</i>-Butyl (2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethyl)carbamate (24 mg, 0.05 mmol) was stirred in 4 M HCl in dioxane (1.16 mL, 4.63 mmol) at room temperature. After 30 min, the solution was concentrated and used as the hydrochloride salt without further purification. LCMS (ESI<sup>+</sup>): RT = 0.63 min, 100%, M + H<sup>+</sup> 419.</div></div><div id="sec4-1-8" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Ethyl 4-((6-Bromoquinolin-2-yl)oxy)butanoate <b>6</b></h4><div class="NLM_p last">6-Bromoquinolin-2(1<i>H</i>)-one <b>5</b> (200 mg, 0.89 mmol) was dissolved in anhydrous DMF (4.5 mL) at room temperature under nitrogen, and potassium carbonate (185 mg, 1.34 mmol) was added. The reaction mixture was allowed to stir for 1.5 h, then ethyl 4-bromobutanoate (0.26 mL, 1.79 mmol) was added dropwise and the reaction stirred overnight. The reaction mixture was poured into water (50 mL) and the aqueous layer extracted with DCM (3 Ã 15 mL). The combined organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo to afford the crude product as an orange oil. This material was purified by Biotage chromatography using a gradient of 0â100% EtOAc in cyclohexane to afford the required O-linked isomer as a white solid (111 mg, 37%) and the undesired N-linked isomer as a yellow oil (164 mg, 55%). The regioisomers were distinguishable by NOE spectroscopy. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 7.91 (d, <i>J</i> = 8.9 Hz, 1H), 7.88â7.87 (br m, 1H), 7.71â7.69 (m, 2H), 6.92 (d, <i>J</i> = 8.8 Hz, 1H), 4.52 (t, <i>J</i> = 6.3 Hz, 2H), 4.18 (q, <i>J</i> = 7.1 Hz, 2H), 2.55 (t, <i>J</i> = 7.5 Hz, 2H), 2.19 (ddd, <i>J</i> = 13.7, 7.4, 6.3 Hz, 2H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 173.39, 162.35, 145.41, 137.80, 132.82, 129.61, 129.11, 126.42, 117.29, 114.30, 65.17, 60.59, 31.27, 24.60, 14.39. HRMS (ESI<sup>+</sup>): calcd for C<sub>15</sub>H<sub>17</sub><sup>79</sup>BrNO<sub>3</sub> (M + H)<sup>+</sup> 339.0419, found 339.0403.</div></div><div id="sec4-1-9" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 2-(Trimethylsilyl)ethyl 2-(4-ethoxy-4-oxobutoxy)quinoline-6-carboxylate <b>7</b></h4><div class="NLM_p last">Ethyl 4-((6-bromoquinolin-2-yl)oxy)butanoate <b>6</b> (554 mg, 1.64 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (95 mg, 0.33 mmol), and Herrmannâs palladacyle (77 mg, 0.08 mmol) were added to a microwave vial and suspended in 2-(trimethylsilyl)ethanol (15 mL). Molybdenum hexacarbonyl (865 mg, 3.28 mmol) followed by DBU 1.0 M in THF (4.91 mL, 4.91 mmol) were then added and the vial promptly sealed. The reaction was heated to 130 Â°C for 1 h in a microwave. The reaction mixture was diluted with DCM and filtered to remove solids. The filtrate was concentrated in vacuo. This residue was diluted with water (20 mL) and extracted with DCM (3 Ã 20 mL). The combined organic layer was washed with brine and concentrated under reduced pressure. The resulting crude was purified by Biotage chromatography using a gradient of 0â100% EtOAc in cyclohexane to give 472 mg of product (88% purity by LCMS, 62%) as a brown oil. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.47 (d, <i>J</i> = 1.9 Hz, 1H), 8.23 (dd, <i>J</i> = 8.7, 1.9 Hz, 1H), 8.06 (d, <i>J</i> = 8.8 Hz, 1H), 7.84 (d, <i>J</i> = 8.7 Hz, 1H), 6.94 (d, <i>J</i> = 8.8 Hz, 1H), 4.56 (t, <i>J</i> = 6.3 Hz, 2H), 4.51â4.43 (m, 2H), 4.17 (q, <i>J</i> = 7.1 Hz, 2H), 2.55 (t, <i>J</i> = 7.4 Hz, 2H), 2.19 (p, <i>J</i> = 7.0, 6.5 Hz, 2H), 1.27 (t, <i>J</i> = 7.1 Hz, 3H), 1.23â1.15 (m, 2H), 0.12 (s, 9H). LCMS (ESI<sup>+</sup>): RT = 1.79 min, 88%, (M + H)<sup>+</sup> 404.</div></div><div id="sec4-1-10" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Ethyl 4-((6-((5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)carbamoyl)quinolin-2-yl)oxy)butanoate</h4><div class="NLM_p">To a stirring solution of 2-(trimethylsilyl)ethyl 2-(4-ethoxy-4-oxobutoxy)quinoline-6-carboxylate <b>7</b> (463 mg, 1.15 mmol) in anhydrous THF (12 mL) at room temperature under nitrogen was dropwise added TBAF 1.0 M in THF (1.72 mL, 1.72 mmol). The reaction was allowed to stir at RT overnight, after which time the starting material had been consumed as indicated by LCMS. The reaction mixture was diluted with water and concentrated in vacuo to remove the THF. The aqueous layer was acidified to â¼ pH 2 with 1 M HCl aq and extracted with DCM (3 Ã 15 mL). The combined organic layer was washed with brine (20 mL) and dried (Na<sub>2</sub>SO<sub>4</sub>) to afford the crude product as a yellow oil. This material was used directly in the next step without further purification. LCMS (ESI<sup>+</sup>): RT = 1.57 min, 84%, (M + H)<sup>+</sup> 304.</div><div class="NLM_p last">2-(4-Ethoxy-4-oxobutoxy)quinoline-6-carboxylic acid (74 mg, 0.24 mmol) was dissolved in anhydrous DMF (2 mL), and DIPEA (0.12 mL, 0.67 mmol) was added, followed by HATU (105 mg, 0.28 mmol). The reaction was allowed to stir for 5 min, and then <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (63 mg, 0.22 mmol) was added. The reaction was stirred at room temperature overnight. The reaction mixture was poured into water and the resulting precipitate collected by filtration. The precipitate was dissolved in DCM/MeOH and preabsorbed onto silica. The crude material was purified by Biotage chromatography using a gradient of 0â5% MeOH in DCM and then by preparative HPLC (20 mL, lipophilic method) to afford the title compound as an amorphous beige solid (48 mg, 38% over two steps). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.30 (d, <i>J</i> = 1.9 Hz, 1H), 8.13â8.05 (m, 3H), 7.93â7.87 (m, 3H),7.68 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.44 (d, <i>J</i> = 2.1 Hz, 1H), 7.38 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.23 (d, <i>J</i> = 8.4 Hz, 1H), 6.97 (d, <i>J</i> = 8.8 Hz, 1H), 6.94 (d, <i>J</i> = 8.4 Hz, 1H), 4.57 (t, <i>J</i> = 6.3 Hz, 2H), 4.31 (ddd, <i>J</i> = 11.1, 3.7, 1.8 Hz, 4H), 4.18 (q, <i>J</i> = 7.1 Hz, 2H), 2.56 (t, <i>J</i> = 7.4 Hz, 2H), 2.35 (s, 3H), 2.21 (p, <i>J</i> = 6.7 Hz, 2H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 173.37, 165.49, 165.09, 163.48, 148.69, 146.89, 143.67, 139.54, 136.92, 136.11, 131.17, 130.22, 128.23, 128.05, 127.62, 127.30, 125.17, 124.61, 120.59, 117.63, 117.52, 116.89, 114.86, 114.58, 65.40, 64.71, 64.34, 60.62, 31.26, 24.58, 17.52, 14.39. HRMS (ESI<sup>+</sup>): calcd for C<sub>32</sub>H<sub>32</sub>N<sub>3</sub>O<sub>7</sub> (M + H)<sup>+</sup> 570.2253, found 570.2222.</div></div><div id="sec4-1-11" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-((6-((5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)carbamoyl)quinolin-2-yl)oxy)butanoic Acid <b>9</b></h4><div class="NLM_p last">To a stirring solution of ethyl 4-((6-((5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)carbamoyl)quinolin-2-yl)oxy)butanoate (37 mg, 0.065 mmol) in THF (0.5 mL) and MeOH (0.25 mL) was added a solution of LiOH.H<sub>2</sub>O (13.6 mg, 0.325 mmol) in water (0.25 mL). After 2 days, the reaction mixture was diluted with water (30 mL) and the aqueous layer acidified to  pH â¼2 with 1 M aq HCl. The aqueous layer was extracted with DCM (3 Ã 15 mL). The combined organic layer was washed with 1 M aq HCl (20 mL) and brine (20 mL), and dried (Na<sub>2</sub>SO<sub>4</sub>) then concentrated in vacuo to afford the product as an amorphous pale-yellow solid (6.5 mg, 18%). This material was used directly in the next reaction without further purification. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.16 (br s, 1H), 10.07 (s, 2H), 8.56 (d, <i>J</i> = 1.9 Hz, 1H), 8.38 (d, <i>J</i> = 8.8 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.1 Hz, 1H), 7.89â7.83 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.6 Hz, 1H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 6.99 (s, 1H), 4.47 (t, <i>J</i> = 6.5 Hz, 2H), 4.34â4.21 (m, 4H), 2.43 (t, <i>J</i> = 7.4 Hz, 2H), 2.24 (s, 3H), 2.03 (p, <i>J</i> = 7.0 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 174.11, 164.94, 164.35, 162.78, 147.57, 146.34, 142.93, 140.23, 137.29, 136.33, 130.11, 130.05, 128.76, 128.40, 128.12, 127.68, 126.75, 124.03, 121.20, 118.61, 118.11, 116.84, 116.66, 113.94, 65.06, 64.39, 64.02, 30.30, 24.05, 17.49. HRMS (ESI<sup>+</sup>): calcd for C<sub>30</sub>H<sub>28</sub>N<sub>3</sub>O<sub>7</sub> (M + H)<sup>+</sup> 542.1922, found 542.1905.</div></div><div id="sec4-1-12" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-(2-Fluoro-5-nitrophenyl)-2-methylquinoline-6-carboxamide</h4><div class="NLM_p last">Oxalyl chloride (3.25 mL, 38.4 mmol) was added dropwise to a solution of 2-methylquinoline-6-carboxylic acid (6.59 g, 35.2 mmol) and DMF (0.0062 mL, 0.080 mmol) in anhydrous DCM (80 mL). The reaction mixture was stirred at room temperature for 3 h and then concentrated under reduced pressure. The residue was dissolved in DCM (30 mL) and concentrated again under reduced pressure. The resulting dry residue was dissolved in pyridine (80 mL), and 2-fluoro-5-nitroaniline <b>12</b> (5.00 g, 32.0 mmol) was added in one portion. The reaction mixture was stirred at room temperature for 18 h and then poured onto water (100 mL). The green precipitate was filtered and washed with water (3 Ã 20 mL), diethyl ether (3 Ã 20 mL), and DCM (10 mL), to afford the title compound as a light-green solid, which was carried onto the next step without further purification (10.4 g, quant). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.71 (s, 1H), 8.72 (dd, <i>J</i> = 6.5, 2.9 Hz, 1H), 8.63 (d, <i>J</i> = 2.0 Hz, 1H), 8.43 (d, <i>J</i> = 8.5 Hz, 1H), 8.23 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.21â8.16 (m, 1H), 8.05 (d, <i>J</i> = 8.8 Hz, 1H), 7.65 (app t, <i>J</i> = 9.5 Hz, 1H), 7.55 (d, <i>J</i> = 8.4 Hz, 1H), 2.71 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.53, 161.22, 158.67 (d, <i>J</i> = 258.2 Hz), 148.65, 143.72, 137.32, 130.36, 128.88, 128.48, 128.00, 127.08 (d, <i>J</i> = 13.9 Hz), 125.33, 123.18, 122.14 (d, <i>J</i> = 9.6 Hz), 121.25 (d, <i>J</i> = 3.8 Hz), 117.19 (d, <i>J</i> = 22.8 Hz), 25.07. <sup>19</sup>F NMR (470 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ â110.20. HRMS (ESI<sup>+</sup>): calcd for C<sub>17</sub>H<sub>13</sub>FN<sub>3</sub>O<sub>3</sub> (M + H)<sup>+</sup> 326.0935, found 326.0931.</div></div><div id="sec4-1-99" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)-2-((4-(3-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)propanoyl)piperazin-1-yl)methyl)quinoline-6-carboxamide <b>10</b></h4><div class="NLM_p last">3-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)propanoic acid <b>15</b> (15 mg, 0.033 mmol), <i>N</i>,<i>N</i>-(5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)-2-(piperazin-1-ylmethyl)quinoline-6-carboxamide <b>14</b> (18 mg, 0.033 mmol), and HATU (19 mg, 0.050 mmol) were dissolved in anhydrous DMF (0.34 mL) at room temperature under inert atmosphere. DIPEA (23.4 Î¼L, 0.134 mmol) was added and the reaction stirred for 16 h. The reaction mixture was concentrated in vacuo and purified by reverse phase Biotage chromatography (10â80% MeOH in water + 0.1% formic acid), however, the material obtained was not pure. Further purification by Biotage chromatography using a gradient of 2â50% EtOH in DCM afforded the product as an off-white amorphous solid (17 mg, 52%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.12 (s, 1H), 10.39 (s, 1H), 10.18 (s, 1H), 8.65 (d, <i>J</i> = 2.0 Hz, 1H), 8.50 (d, <i>J</i> = 8.4 Hz, 1H), 8.25 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.14 (dd, <i>J</i> = 7.1, 2.6 Hz, 1H), 8.08 (d, <i>J</i> = 8.8 Hz, 1H), 7.99 (t, <i>J</i> = 5.6 Hz, 1H), 7.80 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.74 (d, <i>J</i> = 8.5 Hz, 1H), 7.65 (ddd, <i>J</i> = 9.0, 4.4, 2.7 Hz, 1H), 7.56â7.46 (m, 3H), 7.39 (d, <i>J</i> = 8.5 Hz, 1H), 7.30 (dd, <i>J</i> = 10.1, 9.0 Hz, 1H), 6.99 (d, <i>J</i> = 8.4 Hz, 1H), 5.12 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.79 (s, 2H), 4.31 (td, <i>J</i> = 5.3, 3.6 Hz, 4H), 3.82 (s, 2H), 3.62 (t, <i>J</i> = 6.6 Hz, 2H), 3.52â3.41 (m, 6H), 3.33 (s, 2H), 2.89 (ddd, <i>J</i> = 16.7, 13.7, 5.4 Hz, 1H), 2.62â2.52 (m, 4H), 2.43 (d, <i>J</i> = 21.7 Hz, 4H), 2.09â1.99 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 171.71, 169.89, 168.79, 167.12, 166.76, 166.06, 165.15, 165.08, 161.23, 154.62, 149.43 (d, <i>J</i> = 240.9 Hz), 149.10, 146.95, 143.59, 137.71, 137.06, 134.97, 133.71, 131.94, 129.99, 127.99, 127.94 (d, <i>J</i> = 9.5 Hz), 127.28, 126.81, 126.42 (d, <i>J</i> = 11.2 Hz), 122.32, 120.74, 119.68, 118.21, 117.48, 117.46 (d, <i>J</i> = 6.5 Hz), 117.05 (d, <i>J</i> = 7.0 Hz), 116.93, 115.30 (d, <i>J</i> = 20.0 Hz), 113.74, 69.20, 68.22, 67.52, 64.74, 64.70, 64.32, 53.39, 53.15, 49.42, 45.80, 41.68, 39.47, 33.60, 31.52, 23.00. HRMS (ESI<sup>+</sup>): calcd for C<sub>50</sub>H<sub>48</sub>FN<sub>8</sub>O<sub>12</sub> (M + H)<sup>+</sup> 971.3370, found 971.3343.</div></div><div id="sec4-1-13" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(5-Amino-2-fluorophenyl)-2-methylquinoline-6-carboxamide <b>11</b></h4><div class="NLM_p last">To a solution of <i>N</i>-(2-fluoro-5-nitrophenyl)-2-methylquinoline-6-carboxamide (10.4 g, 32.0 mmol) in ethanol (120 mL) and water (40 mL), ammonium chloride (12.0 g, 224 mmol) and iron powder (12.5 g, 224 mmol) were added in one portion, and the resulting suspension was allowed to stir at 90 Â°C for 1 h. The reaction mixture was allowed to cool to room temperature, diluted with MeOH (20 mL) and DCM (20 mL), and filtered through a pad of Celite. The resulting filtrate was concentrated under vacuum to afford a light-brown solid which was redissolved in a mixture of DCM:MeOH (9:1, 150 mL) and washed with saturated aqueous NaHCO<sub>3</sub> (150 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford a yellow solid as crude product, which was taken directly onto the next step without further purification (9.46 g, quant). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.05 (s, 1H), 8.57 (d, <i>J</i> = 1.8 Hz, 1H), 8.39 (d, <i>J</i> = 8.5 Hz, 1H), 8.19 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.01 (d, <i>J</i> = 8.8 Hz, 1H), 7.52 (d, <i>J</i> = 8.4 Hz, 1H), 6.94 (dd, <i>J</i> = 10.3, 8.8 Hz, 1H), 6.89 (dd, <i>J</i> = 6.6, 2.7 Hz, 1H), 6.46â6.39 (m, 1H), 5.05 (br s, 2H), 2.70 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 164.93, 160.84, 148.49, 147.72 (d, <i>J</i> = 233.9 Hz), 145.08 (d, <i>J</i> = 1.9 Hz), 137.19, 131.19, 128.44, 128.33, 127.95, 125.50 (d, <i>J</i> = 13.1 Hz), 125.34, 122.99, 115.54 (d, <i>J</i> = 20.6 Hz), 111.52, 111.39 (d, <i>J</i> = 6.6 Hz), 25.05. <sup>19</sup>F NMR (470 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ â138.12. HRMS (ESI<sup>+</sup>): calcd for C<sub>17</sub>H<sub>15</sub>FN<sub>3</sub>O (M + H)<sup>+</sup> 296.1194, found 296.1191.</div></div><div id="sec4-1-14" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)-2-methylquinoline-6-carboxamide <b>13</b></h4><div class="NLM_p last">To a suspension of 2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxylic acid (12.7 g, 70.5 mmol) in anhydrous DCM (100 mL), a catalytic amount of anhydrous DMF (6.16 Î¼L, 0.080 mmol) and oxalyl chloride (6.51 mL, 77.0 mmol) were added dropwise, and the resulting green solution was allowed to stir at room temperature for 3 h, after which time, the reaction mixture was concentrated in vacuo to afford a dry pale-green solid. The solid was dissolved in pyridine (100 mL), and <i>N</i>-(5-amino-2-fluorophenyl)-2-methylquinoline-6-carboxamide (9.46 g, 32.0 mmol) was added in one portion. The resulting dark-yellow suspension was allowed to stir for 2 h and was then poured onto water (100 mL). The yellow precipitate was filtered and washed with water (3 Ã 20 mL), diethyl ether (3 Ã 20 mL), and DCM (10 mL) to afford the crude product as a pale-yellow solid, which was taken directly onto the next step without purification (12.5 g, 85%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 10.38 (s, 1H), 10.19 (s, 1H), 8.62 (d, <i>J</i> = 1.7 Hz, 1H), 8.41 (d, <i>J</i> = 8.8 Hz, 1H), 8.23 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 8.13 (dd, <i>J</i> = 7.0, 2.5 Hz, 1H), 8.03 (d, <i>J</i> = 8.8 Hz, 1H), 7.68â7.62 (m, 1H), 7.57â7.46 (m, 3H), 7.29 (app t, <i>J</i> = 9.5, 1H), 6.99 (d, <i>J</i> = 8.4 Hz, 1H), 4.34â4.28 (m, 4H), 2.71 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.09, 164.48, 160.95, 151.83 (d, <i>J</i> = 243.4 Hz), 148.57, 146.45, 142.95, 137.22, 135.42 (d, <i>J</i> = 2.0 Hz), 130.87, 128.50, 128.41, 127.94, 127.47, 125.39 (d, <i>J</i> = 9.5 Hz), 125.35, 123.05, 121.25, 118.81, 118.75 (d, <i>J</i> = 13.0 Hz), 116.89, 116.69, 115.56 (d, <i>J</i> = 21.2 Hz), 64.41, 64.03, 25.06. <sup>19</sup>F NMR (470 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ â126.65. HRMS (ESI+): calcd for C<sub>26</sub>H<sub>21</sub>FN<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup> 458.1511, found 458.1499.</div></div><div id="sec4-1-15" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)-2-(piperazin-1-ylmethyl)quinoline-6-carboxamide <b>14</b></h4><div class="NLM_p">A solution of <i>N</i>-(5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)-2-methylquinoline-6-carboxamide <b>13</b> (5.00 g, 10.9 mmol) and selenium dioxide (1.33 g, 12.0 mmol) in anhydrous DMF (40 mL) and 1,4-dioxane (120 mL) was heated at reflux for 1 h. Then the reaction mixture was allowed to cool to room temperature, diluted with DCM (20 mL), and filtered through a pad of Celite. The filtrate was concentrated under vacuum (using a heptane/EtOAc azeotrope to remove DMF) to afford the crude product as a yellow solid, which was carried onto the next step without further purification (5.15 g).</div><div class="NLM_p">A solution of <i>N</i>-(5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)-2-formylquinoline-6-carboxamide (255 mg, 0.541 mmol) and <i>tert</i>-butyl piperazine-1-carboxylate (302 mg, 1.62 mmol) in anhydrous DCM (5 mL) was allowed to stir at 20 Â°C for 12 h, after which time sodium triacetoxyborohydride (344 mg, 1.62 mmol) was added in one portion and the resulting mixture was allowed to stir at 20 Â°C for 2 h. The reaction was quenched with NaHCO<sub>3</sub> saturated aqueous solution (5 mL) and extracted with a DCM:MeOH, 9:1 mixture (3 Ã 5 mL). The crude product (pale-yellow solid) was purified by column chromatography on silica gel using a gradient of 0â6% MeOH in DCM, followed by trituration in diethyl ether to afford the desired product as a pale-beige solid (325 mg, 94%).</div><div class="NLM_p last">To a suspension of <i>tert</i>-butyl 4-((6-((5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-fluorophenyl)carbamoyl)quinolin-2-yl)methyl)piperazine-1-carboxylate (300 mg, 0.468 mmol) in anhydrous DCM (5 mL), TFA (0.179 mL, 2.33 mmol) was added dropwise, and the resulting mixture was allowed to stir at 20 Â°C for 3 h. The reaction mixture was concentrated under reduced pressure to afford the crude product as a light-brown oil. The crude was purified by column chromatography on silica gel using a gradient of 0â15% MeOH in DCM, followed by trituration in diethyl ether to afford the title compound as a white solid (174 mg, 69%). <sup>1</sup>H NMR (500 MHz, DMSO<i>-d</i><sub>6</sub>): Î´ 10.42 (s, 1H), 10.20 (s, 1H), 8.73 (br s, 1H), 8.67 (d, <i>J</i> = 1.66 Hz, 1H), 8.52 (d, <i>J</i> = 8.71 Hz, 1H), 8.27 (dd, <i>J</i> = 8.71, 1.66 Hz, 1H), 8.15 (dd, <i>J</i> = 7.08, 2.72 Hz, 1H), 8.09 (d, <i>J</i> = 8.71 Hz, 1H), 7.74 (d, <i>J</i> = 8.42 Hz, 1H), 7.66â7.61 (m, 1H), 7.54 (d, <i>J</i> = 2.11 Hz, 1H), 7.52 (dd, <i>J</i> = 8.42, 2.11 Hz, 1H), 7.30 (app t, <i>J</i> = 10.10 Hz, 1H), 6.99 (d, <i>J</i> = 8.42 Hz, 1H), 4.35â4.27 (m, 4H), 3.90 (s, 2H), 3.19â3.09 (m, 4H), 2.76â2.66 (m, 4H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 164.98, 164.50, 160.59, 151.87 (d, <i>J</i> = 243.8 Hz), 148.27, 146.47, 142.96, 137.76, 135.45 (d, <i>J</i> = 2.7 Hz), 131.45, 128.79, 128.57, 128.13, 127.46, 126.25, 125.30 (d, <i>J</i> = 13.2 Hz), 121.95, 121.26, 118.86, 118.78 (d, <i>J</i> = 7.8 Hz), 116.90, 116.70, 115.59 (d, <i>J</i> = 20.9 Hz), 64.42, 64.04, 63.67, 49.45, 43.03. HRMS (ESI<sup>+</sup>): calcd for C<sub>30</sub>H<sub>29</sub>FN<sub>5</sub>O<sub>4</sub> (M + H)<sup>+</sup> 542.2198, found 542.2190.</div></div><div id="sec4-1-16" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 3-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)propanoic Acid <b>15</b></h4><div class="NLM_p last"><i>tert</i>-Butyl 3-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)propanoate (29 mg, 0.06 mmol) and formic acid (0.22 mL, 5.76 mmol) were dissolved in anhydrous DCM (0.29 mL) at 40 Â°C for 6 h. The reaction mixture was concentrated in vacuo and the resulting residue purified by reverse phase Biotage chromatography using a gradient of 10â80% MeOH in water + 0.1% formic acid to afford the title compound as an amorphous white solid (24 mg, 93%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 12.21 (br s, 1H), 11.12 (s, 1H), 8.01 (t, <i>J</i> = 5.6 Hz, 1H), 7.81 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.50 (d, <i>J</i> = 7.2 Hz, 1H), 7.39 (d, <i>J</i> = 8.4 Hz, 1H), 5.11 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 4.79 (s, 2H), 3.60 (t, <i>J</i> = 6.4 Hz, 2H), 3.44 (t, <i>J</i> = 5.7 Hz, 2H), 3.30 (q, <i>J</i> = 5.8 Hz, 2H, partially obscured by water peak), 2.94â2.82 (m, 1H), 2.64â2.51 (m, 2H), 2.48â2.41 (m, 2H), 2.08â1.99 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 172.80, 172.69, 169.91, 166.92, 166.77, 165.48, 155.00, 136.98, 133.06, 120.36, 116.78, 116.07, 68.58, 67.49, 66.06, 48.84, 38.32, 34.67, 30.98, 22.01. HRMS (ESI<sup>+</sup>): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O<sub>9</sub> (M + H)<sup>+</sup> 448.1356, found 448.1351.</div></div><div id="sec4-1-17" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i>-(2-Chloro-5-nitrophenyl)-2-methylquinoline-6-carboxamide</h4><div class="NLM_p last">2-Methylquinoline-6-carboxylic acid (1.50 g, 8.01 mmol) was suspended in anhydrous DCM (40 mL) under inert atmosphere. DMF (1.40 Î¼L, 0.018 mmol) and oxalyl chloride (0.74 mL, 8.74 mmol) were added dropwise, and the resulting green solution was allowed to stir at 20 Â°C for 3 h, after which time it was concentrated in vacuo to afford a dry pale-green solid. The solid was dissolved in pyridine (40.0 mL) and 2-chloro-5-nitroaniline <b>17</b> (1.26 g, 7.28 mmol) was added in one portion. The resulting dark-yellow suspension was allowed to stir for 2 h, then it was poured onto water and the yellow precipitate was filtered and washed several times with water, Et<sub>2</sub>O, and finally with a minimum amount of DCM to afford the crude product as a yellow amorphous solid which was used without further purification (2.20 g, 88%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 10.59 (s, 1H), 8.65 (d, <i>J</i> = 2.0 Hz, 1H), 8.60 (d, <i>J</i> = 2.7 Hz, 1H), 8.44 (d, <i>J</i> = 8.6 Hz, 1H), 8.25 (dd, <i>J</i> = 8.6, 2.0 Hz, 1H), 8.15 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 8.06 (d, <i>J</i> = 8.4 Hz, 1H), 7.91 (d, <i>J</i> = 8.6 Hz, 1H), 7.55 (d, <i>J</i> = 8.0 Hz, 1H), 2.71 (s, 3H). HRMS (ESI<sup>+</sup>): found [M + H]<sup>+</sup> 342.0646, C<sub>17</sub>H<sub>13</sub><sup>35</sup>ClN<sub>3</sub>O<sub>3</sub> requires 342.0640.</div></div><div id="sec4-1-98" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-((4-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)ethyl)piperazin-1-yl)methyl)quinoline-6-carboxamide <b>16</b> (CCT367766)</h4><div class="NLM_p last">2-(2,6-Dioxo-3-piperidyl)-4-hydroxy-isoindoline-1,3-dione <b>2</b> (29 mg, 0.100 mmol) was dissolved in anhydrous THF (1 mL) under argon and then triphenylphosphine (29 mg, 0.110 mmol) was added and <i>N</i>-(2-chloro-5-(2,3-dihydro-1,4-benzodioxine-6-carbonylamino)phenyl)-2-((4-(2-(2-(2-hydroxyethoxy)ethoxy)ethyl)piperazin-1-yl)methyl)quinoline-6-carboxamide <b>20</b> (72.00 mg, 0.100 mmol) were added, followed by <i>tert</i>-butyl (NE)-<i>N</i>-<i>tert</i>-butoxycarbonyliminocarbamate (25 mg, 0.110 mmol). The reaction was stirred at room temperature for 2 h. The crude product was purified by Biotage chromatography using a gradient of 0â30% MeOH in DCM to afford the product as a pale-yellow amorphous solid (27 mg, 27%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.13 (s, 1H), 10.34 (s, 1H), 10.28 (s, 1H), 8.67 (d, <i>J</i> = 2.1 Hz, 1H), 8.50 (d, <i>J</i> = 8.5 Hz, 1H), 8.27 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 8.16 (d, <i>J</i> = 2.5 Hz, 1H), 8.10 (d, <i>J</i> = 8.8 Hz, 1H), 7.80 (dd, <i>J</i> = 8.5, 7.3 Hz, 1H), 7.75 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.72 (d, <i>J</i> = 8.5 Hz, 1H), 7.57â7.52 (m, 4H), 7.45 (d, <i>J</i> = 7.2 Hz, 1H), 7.01 (d, <i>J</i> = 8.4 Hz, 1H), 5.10 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.38â4.28 (m, 6H), 3.87â3.76 (m, 4H), 3.65 (d, <i>J</i> = 5.0 Hz, 2H), 3.54 (s, 4H), 2.89 (ddd, <i>J</i> = 16.9, 13.8, 5.4 Hz, 1H), 2.69â2.32 (m, 12H), 2.08â1.99 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 172.78, 169.93, 166.79, 165.27, 164.98, 164.65, 155.83, 148.27, 146.58, 142.97, 138.59, 137.56, 136.97, 134.88, 133.23, 131.37, 129.37, 128.83, 128.42, 127.90, 127.30, 126.23, 123.55, 121.82, 121.33, 119.99, 119.82, 119.24, 116.93, 116.75, 116.29, 115.39, 79.18, 70.10, 69.70, 68.90, 68.66, 64.42, 64.03, 57.22, 53.03, 48.75, 30.96, 22.02 (1 signal not observed/overlapping signals). HRMS (ESI<sup>+</sup>): calcd for C<sub>49</sub>H<sub>49</sub><sup>35</sup>ClN<sub>7</sub>O<sub>11</sub> (M + H)<sup>+</sup> 947.3173, found 947.3173.</div></div><div id="sec4-1-18" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-methylquinoline-6-carboxamide <b>18</b></h4><div class="NLM_p last">2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxylic acid (1.27 g, 7.06 mmol) was suspended in anhydrous DCM (20 mL) under inert atmosphere. DMF (1.23 Î¼L, 0.016 mmol) and oxalyl chloride (0.65 mL, 7.70 mmol) were added dropwise, and the resulting green solution was allowed to stir at 20 Â°C for 3 h, after which time it was concentrated under vacuum to afford a dry pale-green solid. The solid was dissolved in pyridine (20 mL), and <i>N</i>-(5-amino-2-chlorophenyl)-2-methylquinoline-6-carboxamide <b>24</b> (2.00 g, 6.42 mmol) was added in one portion. The resulting dark-yellow suspension was allowed to stir for 2 h, then it was poured onto water and the yellow precipitate was filtered and washed several times with water, Et<sub>2</sub>O, and finally with a minimum amount of DCM to afford the crude product as a pale-yellow amorphous solid, which was carried onto the next step without purification (1.86 g, 61%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 10.31 (s, 1H), 10.27 (s, 1H), 8.63 (d, <i>J</i> = 1.5 Hz, 1H), 8.43 (d, <i>J</i> = 8.8 Hz, 1H), 8.25 (dd, <i>J</i> = 8.8, 2.2 Hz, 1H), 8.14 (d, <i>J</i> = 2.2 Hz, 1H), 8.04 (d, <i>J</i> = 8.8 Hz, 1H), 7.75 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 7.58â7.49 (m, 4H), 7.00 (d, <i>J</i> = 8.8 Hz, 1H), 4.37â4.26 (m, 4H), 2.71 (s, 3H). HRMS (ESI<sup>+</sup>): found [M + H]<sup>+</sup> 474.1210, C<sub>26</sub>H<sub>21</sub><sup>35</sup>ClN<sub>3</sub>O<sub>4</sub> requires 474.1215.</div></div><div id="sec4-1-19" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-(piperazin-1-ylmethyl)quinoline-6-carboxamide <b>19</b></h4><div class="NLM_p"><i>N</i>-[2-Chloro-5-(2,3-dihydro-1,4-benzodioxine-6-carbonylamino)phenyl)-2-methyl-quinoline-6-carboxamide <b>18</b> (4.00 g, 8.44 mmol) was taken up in anhydrous DMF (20.00 mL) and anhydrous 1,4-dioxane (20 mL). Selenium dioxide (1.03 g, 9.28 mmol) was added, and the reaction was degassed via 3Ã vacuum/nitrogen cycles. The reaction mixture was heated to 50 Â°C and stirred for 16 h. Then, the reaction was filtered through Celite, eluting with 1:1 DCM/EtOAc and concentrated in vacuo. Remaining DMF was removed by azeotrope with EtOAc/heptane to afford <i>N</i>-(2-chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-formylquinoline-6-carboxamide as a brown solid. The crude material was used directly in the next step, assuming 100% yield, without further purification. Sodium cyanoborohydride (2.12 g, 33.8 mmol) was added to a stirring suspension of <i>N</i>-(2-chloro-5-(2,3-dihydro-1,4-benzodioxine-6-carbonylamino)phenyl)-2-formyl-quinoline-6-carboxamide (4.12 g, 8.44 mmol) and 1-Boc-piperazine (3.15 g, 16.89 mmol) in anhydrous DMF (60 mL) at 0 Â°C under nitrogen. Then acetic acid (0.53 mL, 9.29 mmol) was added. The reaction mixture was allowed to warm to room temperature and stirred overnight. A bleach bubbler was used to vent the reaction. The reaction was quenched by slow addition of an aqueous solution of 1 M NaOH (50 mL), and the reaction mixture concentrated in vacuo by using heptane for azeotropic removal of DMF. The resulting residue was taken up in a small amount of MeOH and poured into a large volume of water (300â400 mL). The precipitate formed was collected by filtration, washed with water, then Et<sub>2</sub>O, and dried under vacuum overnight to afford the crude <i>tert</i>-butyl 4-((6-((2-chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)carbamoyl)quinolin-2-yl)methyl)piperazine-1-carboxylate as a pale-brown solid (4.57 g). This material was used directly in the next reaction without further purification.</div><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-((6-((2-chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)carbamoyl)quinolin-2-yl)methyl)piperazine-1-carboxylate (4.57 g, 6.94 mmol) in anhydrous MeOH (60 mL) at 0 Â°C under argon was added 4 M HCl in dioxane (26.1 mL, 104.4 mmol). Upon addition of the acid, the reaction mixture became darker red/brown in color. The reaction was allowed to warm to room temperature and stirred overnight. Then the solvents were removed in vacuo. The resulting residue was suspended in a small amount of MeOH. The suspension was poured into a large volume of water (with stirring), and the precipitate formed was collected by filtration, washed well with water, then Et<sub>2</sub>O, and dried under vacuum to afford the crude product as a brown solid (3.42 g). Purification by Biotage column chromatography using a gradient of 0â10% MeOH in DCM + 1% 7N NH<sub>3</sub> in MeOH afforded the title compound as a dark-yellow amorphous solid (1.53 g, 32% over 3 steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.32 (s, 1H), 10.27 (s, 1H), 8.65 (d, <i>J</i> = 2.0 Hz, 1H), 8.49 (d, <i>J</i> = 8.4 Hz, 1H), 8.26 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.15 (d, <i>J</i> = 2.5 Hz, 1H), 8.09 (d, <i>J</i> = 8.8 Hz, 1H), 7.76â7.71 (m, 2H), 7.58â7.47 (m, 3H), 7.00 (d, <i>J</i> = 8.4 Hz, 1H), 4.40â4.23 (m, 4H), 3.76 (s, 2H), 2.74 (t, <i>J</i> = 4.8 Hz, 4H), 2.47â2.42 (br m, 4H) (1 proton missing). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.00, 164.64, 161.79, 148.29, 146.57, 142.97, 138.58, 137.45, 134.89, 131.32, 129.36, 128.81, 128.39, 127.85, 127.31, 126.21, 123.54, 121.85, 121.32, 119.81, 119.23, 116.92, 116.74, 65.11, 64.42, 64.03, 54.26, 45.54. HRMS (ESI<sup>+</sup>): calcd for C<sub>30</sub>H<sub>29</sub><sup>35</sup>ClN<sub>5</sub>O<sub>4</sub> (M + H)<sup>+</sup> 558.1903, found 558.1885.</div></div><div id="sec4-1-20" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-(2-(2-(4-((6-((2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)carbamoyl)quinolin-2-yl)methyl)piperazin-1-yl)ethoxy)ethoxy)ethyl 4-Methylbenzenesulfonate <b>20</b></h4><div class="NLM_p last"><i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-(piperazin-1-ylmethyl)quinoline-6-carboxamide <b>19</b> (500 mg, 0.90 mmol) was dissolved in anhydrous DMF (8 mL) under nitrogen at room temperature. K<sub>2</sub>CO<sub>3</sub> (372 mg, 2.69 mmol) was added, followed by a solution of 2-(2-(2-hydroxyethoxy)ethoxy)ethyl 4-methylbenzenesulfonate (573 mg, 1.88 mmol) in anhydrous DMF (2 mL), and the reaction stirred at room temperature overnight. Further 2-(2-(2-hydroxyethoxy)ethoxy)ethyl 4-methylbenzenesulfonate (286 mg, 0.94 mmol) in anhydrous DMF (1 mL) was added and the reaction stirred overnight. The reaction mixture was poured into water and the aqueous layer extracted three times with 10% MeOH in DCM. The combined organic layer was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. EtOAc/heptane was added to remove remaining traces of DMF by azeotrope. The crude oil was purified by Biotage chromatography using a gradient of 0â10% MeOH in DCM + 1% 7N NH<sub>3</sub> in MeOH to afford the title compound as a yellow amorphous solid (297 mg, 48%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 8.63 (s, 1H), 8.59 (d, <i>J</i> = 2.5 Hz, 1H), 8.42 (d, <i>J</i> = 1.9 Hz, 1H), 8.28 (d, <i>J</i> = 8.5 Hz, 1H), 8.21 (d, <i>J</i> = 8.8 Hz, 1H), 8.17 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 7.98 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.95 (s, 1H), 7.76 (d, <i>J</i> = 8.5 Hz, 1H), 7.47 (s, 1H), 7.45 (d, <i>J</i> = 6.6 Hz, 1H), 7.40 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.97 (d, <i>J</i> = 8.4 Hz, 1H), 4.42â4.43 (m, 4H), 3.90 (s, 2H), 3.78â3.71 (m, 2H), 3.71â3.66 (m, 2H), 3.66â3.59 (m, 6H), 2.74â2.56 (m, 10H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 165.20, 165.02, 162.36, 149.26, 147.08, 143.70, 138.10, 137.49, 134.77, 131.89, 130.26, 129.63, 127.91, 127.88, 126.89, 122.42, 120.64, 117.96, 117.57, 117.05, 116.93, 112.63, 72.76, 70.49, 70.44, 68.81, 65.16, 64.70, 64.32, 61.83, 57.90, 53.69, 53.39 (1 signal not observed/overlapping signals). HRMS (ESI<sup>+</sup>): calcd for C<sub>36</sub>H<sub>41</sub><sup>35</sup>ClN<sub>5</sub>O<sub>7</sub> (M + H)<sup>+</sup> 690.2689, found 690.2692.</div></div><div id="sec4-1-21" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-(4-Chloro-3-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide</h4><div class="NLM_p">2,3-Dihydro-1,4-benzodioxine-5-carboxylic acid (1.73 g, 9.61 mmol) was suspended in anhydrous DCM (45 mL), and 3 drops of anhydrous DMF were added. Oxalyl chloride (0.81 mL, 9.61 mmol) was added dropwise (effervescence observed) and the reaction stirred at room temperature under nitrogen for 2 h. After this time, the effervescence had ceased and the reaction mixture had become fully dissolved. Then the solvents were removed in vacuo. Anhydrous DCM (10 mL) was added and the reaction concentrated again. The residue was taken up in anhydrous DCM (5 mL, then 1 mL to rinse flask) and added slowly to a solution of 4-chloro-3-nitro-aniline <b>23</b> (1.11 g, 6.41 mmol) and pyridine (1.55 mL, 19.22 mmol) in anhydrous DCM (45 mL). The reaction was stirred at room temperature under nitrogen for 48 h. The solvent was removed in vacuo and the resulting residue partitioned between saturated aqueous NaHCO<sub>3</sub> solution and 10% MeOH in DCM (30 mL). The aqueous layer was extracted with two further portions of 10% MeOH in DCM (30 mL), the combined organic layer was washed with water, brine, dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo, and then dried under high vacuum to afford the crude product as a pale-brown solid (2.24 g).</div><div class="NLM_p last">However, this material was contaminated with 2,3-dihydro-1,4-benzodioxine-5-carboxylic acid, therefore the solid was suspended in a small volume of MeOH and diluted with saturated aqueous NaHCO<sub>3</sub> solution. The suspension was vigorously stirred for 1 h, after which time the precipitate was collected by filtration. The precipitate was washed with copious amounts of water, followed by a small portion of Et<sub>2</sub>O, then dried under vacuum to afford the product as a beige amorphous solid (1.82 g, 84%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.62 (s, 1H), 8.55 (d, <i>J</i> = 2.5 Hz, 1H), 7.95 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.75 (d, <i>J</i> = 8.8 Hz, 1H), 7.15 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.94 (t, <i>J</i> = 7.8 Hz, 1H), 4.39â4.26 (m, 4H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 164.71, 147.29, 143.70, 141.31, 138.82, 132.01, 124.79, 124.51, 121.25, 120.84, 119.59, 118.73, 115.87, 64.52, 63.77. HRMS (ESI<sup>+</sup>): calcd for C<sub>15</sub>H<sub>12</sub><sup>35</sup>ClN<sub>2</sub>O<sub>5</sub> (M + H)<sup>+</sup> 335.0429, found 335.0413.</div></div><div id="sec4-1-22" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-(3-Amino-4-chlorophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide</h4><div class="NLM_p last"><i>N</i>-(4-Chloro-3-nitro-phenyl)-2,3-dihydro-1,4-benzodioxine-5-carboxamide (1.50 g, 4.48 mmol) was suspended in a mixture of DCM (18 mL) and EtOH (9 mL). Then 10% Pd/C (200 mg) was added and the reaction mixture stirred for 3 days under 1 atm H<sub>2</sub>. The reaction mixture was filtered through Celite, eluting with 10% MeOH in DCM. The filtrate was concentrated in vacuo and dried under high vacuum to afford the crude product as a pale-brown solid (1.13 g, 93% purity, 77%), which was used directly in the next reaction. A portion of this material (320 mg) was purified by Biotage chromatography using a gradient of 0â5% EtOAc in DCM to afford the title compound as a pale-yellow amorphous solid (198 mg, 52% based on starting material). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.93 (s, 1H), 7.35 (d, <i>J</i> = 2.2 Hz, 1H), 7.10 (d, <i>J</i> = 8.8 Hz, 2H), 6.99 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 6.91 (t, <i>J</i> = 7.8 Hz, 1H), 6.81 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 5.37 (s, 2H), 4.39â4.32 (m, 2H), 4.31â4.27 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 163.77, 144.72, 143.63, 141.13, 138.48, 128.82, 125.80, 121.21, 120.70, 118.98, 111.77, 108.78, 106.29, 64.43, 63.74. HRMS (ESI<sup>+</sup>): calcd for C<sub>15</sub>H<sub>14</sub><sup>35</sup>ClN<sub>2</sub>O<sub>3</sub> (M + H)<sup>+</sup> 305.0687, found 305.0688.</div></div><div id="sec4-1-23" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamido)phenyl)-2-methylquinoline-6-carboxamide</h4><div class="NLM_p last">2-Methylquinoline-6-carboxylic acid (685 mg, 3.66 mmol) was dissolved in anhydrous DCM (24 mL) at room temperature under inert atmosphere and 1 drop anhydrous DMF added. Then oxalyl chloride (0.31 mL, 3.66 mmol) was added dropwise (effervescence observed) and the reaction stirred at room temperature for 2 h, over which time effervescence ceased and the solution became dark green. The solvent was removed in vacuo. The resulting residue was redissolved in anhydrous DCM (2 mL) and concentrated in vacuo (Ã2). The solid was taken up in anhydrous DCM (1 mL, then 1 mL to rinse flask) and slowly added to a stirring solution of <i>N</i>-(3-amino-4-chloro-phenyl)-2,3-dihydro-1,4-benzodioxine-5-carboxamide (800.00 mg, 2.44 mmol) in anhydrous pyridine (18 mL) at room temperature under inert atmosphere. The reaction was stirred overnight. A precipitate formed which was isolated by filtration, washed with water (Ã2) and ether, and then dried under vacuum to afford the product as a beige amorphous solid (672 mg, 58%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.33 (s, 1H), 10.32 (s, 1H), 8.63 (d, <i>J</i> = 1.8 Hz, 1H), 8.42 (d, <i>J</i> = 8.5 Hz, 1H), 8.25 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.07 (d, <i>J</i> = 2.3 Hz, 1H), 8.04 (d, <i>J</i> = 8.8 Hz, 1H), 7.68 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.54 (dd, <i>J</i> = 8.6, 2.8 Hz, 2H), 7.15 (dd, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.02 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 6.93 (t, <i>J</i> = 7.8 Hz, 1H), 4.37 (dd, <i>J</i> = 5.1, 2.4 Hz, 2H), 4.31 (dd, <i>J</i> = 5.0, 2.7 Hz, 2H), 2.71 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.07, 164.24, 160.97, 148.57, 143.67, 141.20, 138.27, 137.25, 135.12, 130.87, 129.53, 128.46, 128.44, 127.85, 125.51, 125.39, 123.79, 123.07, 121.19, 120.76, 119.31, 119.19, 118.67, 64.47, 63.77, 25.05. HRMS (ESI<sup>+</sup>): calcd for C<sub>26</sub>H<sub>21</sub><sup>35</sup>ClN<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup> 474.1215, found 474.1192.</div></div><div id="sec4-1-24" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamido)phenyl)-2-(piperazin-1-ylmethyl)quinoline-6-carboxamide</h4><div class="NLM_p"><i>N</i>-(2-Chloro-5-(2,3-dihydro-1,4-benzodioxine-5-carbonylamino)phenyl)-2-methyl-quinoline-6-carboxamide (500 mg, 1.06 mmol) was taken up in anhydrous DMF (5 mL) and anhydrous 1,4-dioxane (5 mL). Selenium dioxide (128 mg, 1.16 mmol) was added, the reaction was degassed via 3Ã vacuum/nitrogen cycles, and stirred at 50 Â°C for 5 h. The reaction mixture was filtered through Celite (eluting with 1:1 DCM/EtOAc), then concentrated in vacuo, using EtOAc/heptane to remove traces of DMF. This material was used directly in the next reaction without further purification, assuming 100% yield.</div><div class="NLM_p"><i>N</i>-[2-Chloro-5-(2,3-dihydro-1,4-benzodioxine-5-carbonylamino)phenyl]-2-formyl-quinoline-6-carboxamide (0.51 g, 1.06 mmol) and 1-Boc-piperazine (392 mg, 2.11 mmol) were dissolved in anhydrous DMF (10 mL) at 0 Â°C under nitrogen. Then sodium cyanoborohydride (265 mg, 4.22 mmol) was added in one portion, followed by acetic acid (0.07 mL, 1.16 mmol). A bleach bubbler was used to vent the reaction. The reaction was allowed to warm to room temperature and stirred overnight. The reaction was quenched by addition of 1 M NaOH (aq) and then concentrated in vacuo. The resulting residue was suspended in MeOH and diluted with water to afford a brown precipitate. The precipitate was collected by filtration and washed well with water, followed by Et<sub>2</sub>O, and dried under vacuum to afford the product as a pale-brown solid (520 mg). This material was used directly in the next reaction without further purification.</div><div class="NLM_p last"><i>tert</i>-Butyl 4-((6-((2-chloro-5-(2,3-dihydro-1,4-benzodioxine-5 carbonylamino)phenyl)carbamoyl)-2-quinolyl)methyl)piperazine-1-carboxylate (508 mg, 0.770 mmol) was dissolved in anhydrous MeOH (7 mL) at 0 Â°C under inert atmosphere. Then 4 M HCl in dioxane (2.90 mL, 11.61 mmol) was added and the reaction warmed to room temperature. After 1.5 h, a further 4 M HCl in dioxane (2.90 mL, 11.61 mmol) was added and the reaction stirred overnight. Then, the reaction mixture was concentrated in vacuo. The resulting residue was taken up in 1:1 MeOH/DCM and subjected to Isolute Flash SCX-II chromatography to afford the free salt, eluting first with MeOH followed by 10% 7N NH<sub>3</sub> in MeOH. This material was further purified by Biotage chromatography using a gradient of 0â10% MeOH in DCM + 1% 7N NH<sub>3</sub> in MeOH to afford the title compound as a pale-brown amorphous solid (253 mg, 43% over 3 steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.35 (s, 1H), 10.32 (s, 1H), 8.71â8.54 (m, 1H), 8.49 (d, <i>J</i> = 8.5 Hz, 1H), 8.26 (dd, <i>J</i> = 8.8, 1.8 Hz, 1H), 8.09 (app. d, <i>J</i> = 8.6 Hz, 2H), 7.74 (d, <i>J</i> = 8.5 Hz, 1H), 7.68 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.54 (d, <i>J</i> = 8.8 Hz, 1H), 7.14 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.02 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 6.93 (t, <i>J</i> = 7.8 Hz, 1H), 4.42â4.34 (m, 2H), 4.33â4.29 (m, 2H), 3.77 (s, 2H), 2.76 (t, <i>J</i> = 4.5 Hz, 4H), 2.42 (br s, 4H) (1 proton missing). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.00, 164.24, 161.72, 148.30, 143.67, 141.20, 138.27, 137.48, 135.08, 131.28, 129.54, 128.81, 128.44, 127.88, 126.22, 125.50, 123.78, 121.86, 121.19, 120.76, 119.32, 119.19, 118.70, 65.02, 64.47, 63.77, 53.94, 48.60, 45.37 (1 missing/overlapping signal). HRMS (ESI<sup>+</sup>): calcd for C<sub>30</sub>H<sub>29</sub><sup>35</sup>ClN<sub>5</sub>O<sub>4</sub> (M + H)<sup>+</sup> 558.1903, found 558.1890.</div></div><div id="sec4-1-25" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamido)phenyl)-2-((4-(2-(2-(2-hydroxyethoxy)ethoxy)ethyl)piperazin-1-yl)methyl)quinoline-6-carboxamide</h4><div class="NLM_p last"><i>N</i>-(2-Chloro-5-(2,3-dihydro-1,4-benzodioxine-5-carbonylamino)phenyl)-2-(piperazin-1-ylmethyl)quinoline-6-carboxamide (248 mg, 0.440 mmol) was dissolved in anhydrous DMF (3 mL) at room temperature under inert atmosphere, and K<sub>2</sub>CO<sub>3</sub> (371 mg, 2.69 mmol) was added. Then a solution of 2-(2-(2 hydroxyethoxy)ethoxy)ethyl 4-methylbenzenesulfonate (573 mg, 1.88 mmol) in anhydrous DMF (1 mL) was added and reaction stirred overnight. The reaction was diluted with water and the aqueous layer extracted (3 Ã 10% MeOH in DCM). The combined organic layer was washed with brine and dried (Na<sub>2</sub>SO<sub>4</sub>). Purification by Biotage chromatography using a gradient of 0â10% MeOH in DCM + 1% 7N NH<sub>3</sub> in MeOH gave the title compound as a pale-yellow amorphous solid (108 mg, 35%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´ 9.59 (s, 1H), 8.63 (s, 1H), 8.54 (d, <i>J</i> = 2.5 Hz, 1H), 8.42 (d, <i>J</i> = 1.6 Hz, 1H), 8.27 (d, <i>J</i> = 8.5 Hz, 1H), 8.21 (d, <i>J</i> = 8.8 Hz, 1H), 8.17 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 8.00 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.81 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.75 (d, <i>J</i> = 8.5 Hz, 1H), 7.43 (d, <i>J</i> = 8.8 Hz, 1H), 7.07 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 6.99 (t, <i>J</i> = 7.9 Hz, 1H), 4.56 (dd, <i>J</i> = 5.0, 3.1 Hz, 2H), 4.38 (dd, <i>J</i> = 5.0, 3.1 Hz, 2H), 3.89 (s, 2H), 3.77â3.71 (m, 2H), 3.70â3.58 (m, 8H), 2.73â2.53 (m, 10H) (1 proton missing). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´ 164.98, 163.11, 162.34, 149.27, 143.81, 141.99, 138.12, 137.44, 134.75, 131.96, 130.31, 129.55, 127.80, 126.89, 124.52, 122.41, 122.16, 121.76, 121.56, 118.01, 117.66, 113.13, 72.74, 70.48, 70.44, 68.82, 65.46, 65.14, 63.75, 61.82, 57.87, 53.67, 53.36 (1 overlapping/missing signal). HRMS (ESI<sup>+</sup>): calcd for C<sub>36</sub>H<sub>41</sub><sup>35</sup>ClN<sub>5</sub>O<sub>7</sub> (M + H)<sup>+</sup> 690.2689, found 690.2696.</div></div><div id="sec4-1-26" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamido)phenyl)-2-((4-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)ethyl)piperazin-1-yl)methyl)quinoline-6-carboxamide <b>21</b></h4><div class="NLM_p last">2-(2,6-Dioxo-3-piperidyl)-4-hydroxy-isoindoline-1,3-dione (19.87 mg, 0.070 mmol) was dissolved in anhydrous THF (0.72 mL) under inert atmosphere, and then triphenylphosphine (19.95 mg, 0.080 mmol) and <i>tert</i>-butyl (NE)-<i>N</i>-<i>tert</i>-butoxycarbonyliminocarbamate (17.51 mg, 0.080 mmol) were added, followed by <i>N</i>-(2-chloro-5-(2,3-dihydro-1,4-benzodioxine-5-carbonylamino)phenyl)-2-((4-(2-(2-(2-hydroxyethoxy)ethoxy]ethyl)piperazin-1-yl)methyl)quinoline-6-carboxamide (50.00 mg, 0.070 mmol). The reaction was stirred at room temperature for 2 h. The reaction mixture was preabsorbed onto silica and purified by Biotage chromatography using a gradient of 0â30% MeOH in DCM to afford the product as a pale-yellow amorphous solid (28 mg, 41%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 11.12 (s, 1H), 10.35 (s, 1H), 10.32 (s, 1H), 8.65 (d, <i>J</i> = 2.0 Hz, 1H), 8.49 (d, <i>J</i> = 8.6 Hz, 1H), 8.26 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.13â8.06 (m, 2H), 7.79 (dd, <i>J</i> = 8.6, 7.2 Hz, 1H), 7.75â7.65 (m, 2H), 7.53 (t, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 7.2 Hz, 1H), 7.14 (dd, <i>J</i> = 7.6, 1.7 Hz, 1H), 7.02 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 6.93 (t, <i>J</i> = 7.8 Hz, 1H), 5.09 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.42â4.26 (m, 6H), 3.85â3.73 (m, 4H), 3.64 (dd, <i>J</i> = 5.9, 3.7 Hz, 2H), 3.54â3.46 (dt, <i>J</i> = 11.8, 5.7 Hz, 4H), 2.88 (ddd, <i>J</i> = 16.9, 13.8, 5.4 Hz, 1H), 2.65â2.23 (m, 12H), 2.09â1.96 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 172.76, 169.90, 166.79, 165.24, 164.98, 164.23, 161.71, 155.84, 148.28, 143.66, 141.19, 138.26, 137.51, 136.95, 135.07, 133.23, 131.27, 129.53, 128.81, 128.43, 127.87, 126.21, 125.50, 123.76, 121.79, 121.18, 120.76, 120.01, 119.30, 119.18, 118.68, 116.31, 115.37, 70.12, 69.72, 68.90, 68.67, 68.21, 64.46, 64.35, 63.76, 57.22, 54.91, 53.13, 53.01, 48.74, 40.02, 30.95, 22.01. HRMS (ESI<sup>+</sup>): calcd for C<sub>49</sub>H<sub>49</sub><sup>35</sup>ClN<sub>7</sub>O<sub>11</sub> (M + H)<sup>+</sup> 946.3173, found 946.3183.</div></div><div id="sec4-1-27" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-((4-ethylpiperazin-1-yl)methyl)quinoline-6-carboxamide <b>22</b></h4><div class="NLM_p"><i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-methylquinoline-6-carboxamide <b>18</b> (1.00 g, 2.11 mmol) was taken up in anhydrous DMF (10 mL) and anhydrous 1,4-dioxane (10 mL). Selenium dioxide (280 mg, 2.53 mmol) was added, and the reaction was degassed via 3Ã vacuum/nitrogen cycles and stirred at 50 Â°C under nitrogen for 3.5 h. Further selenium dioxide (47 mg, 0.42 mmol) was added and the reaction stirred at 50 Â°C for an additional 2 h. The reaction mixture was cooled, filtered through Celite (eluting with DCM/EtOAc), and concentrated in vacuo, using EtOAc/heptane azeotrope to remove DMF. This material was used directly in the next reaction without further purification, assuming 100% yield.</div><div class="NLM_p last"><i>N</i>-(2-Chloro-5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)-2-formylquinoline-6-carboxamide (1.03 g, 2.11 mmol) was suspended in anhydrous DCM (20 mL) at room temperature under nitrogen, and 1-ethylpiperazine (0.80 mL, 6.33 mmol) was added. The reaction was allowed to stir overnight, and then NaBH(OAc)<sub>3</sub> (1.34 g, 6.33 mmol) was added and stirred for 3 h. The reaction mixture was quenched with a saturated aqueous solution of NaHCO<sub>3</sub> and the aqueous layer extracted with 10% MeOH in DCM (Ã3). The combined organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. The residue was purified by Biotage chromatography using a gradient of 0â10% MeOH in DCM. This material was further purified by Isolute Flash SCX-II chromatography, eluting with MeOH, followed by 10% 7N NH<sub>3</sub> in MeOH to afford the product as an amorphous yellow solid (431 mg, 35%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.32 (s, 1H), 10.27 (s, 1H), 8.65 (d, <i>J</i> = 1.9 Hz, 1H), 8.50 (d, <i>J</i> = 8.5 Hz, 1H), 8.26 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.15 (d, <i>J</i> = 2.5 Hz, 1H), 8.09 (d, <i>J</i> = 8.8 Hz, 1H), 7.77â7.74 (m, 1H), 7.73 (d, <i>J</i> = 8.7 Hz, 1H), 7.56â7.50 (m, 3H), 7.00 (d, <i>J</i> = 8.4 Hz, 1H), 4.34â4.26 (m, 4H), 3.80 (s, 2H), 2.60â2.25 (m, 10H), 0.99 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 165.00, 164.64, 161.70, 148.28, 146.57, 142.97, 138.58, 137.52, 134.89, 131.35, 129.35, 128.82, 128.39, 127.87, 127.31, 126.23, 123.53, 121.82, 121.32, 119.80, 119.23, 116.92, 116.74, 64.42, 64.35, 64.03, 52.94, 52.34, 51.59, 11.92. HRMS (ESI<sup>+</sup>): calcd for C<sub>32</sub>H<sub>33</sub><sup>35</sup>ClN<sub>5</sub>O<sub>4</sub> (M + H)<sup>+</sup> 586.2216, found 586.2239.</div></div><div id="sec4-1-28" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(5-Amino-2-chlorophenyl)-2-methylquinoline-6-carboxamide <b>24</b></h4><div class="NLM_p last"><i>N</i>-(2-Chloro-5-nitrophenyl)-2-methylquinoline-6-carboxamide was suspended in water (7 mL) and EtOH (21 mL). Ammonium chloride (2.41 g, 45.1 mmol) and iron powder (2.52 g, 45.1 mmol) were added, and the resulting suspension was allowed to stir at 90 Â°C for 1 h. The reaction mixture was allowed to cool to room temperature, diluted with MeOH and DCM, and filtered through a pad of Celite. The resulting filtrate was concentrated under vacuum to afford a light-brown amorphous solid as crude product, which was used directly in the next step without any further purification (2.00 g, 100%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 9.96 (s, 1H), 8.58 (d, <i>J</i> = 2.2 Hz, 1H), 8.41 (d, <i>J</i> = 8.7 Hz, 1H), 8.21 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 8.02 (d, <i>J</i> = 8.7 Hz, 1H), 7.53 (d, <i>J</i> = 7.6 Hz, 1H), 7.15 (d, <i>J</i> = 8.7 Hz, 1H), 6.87 (d, <i>J</i> = 2.2 Hz, 1H), 6.50 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 5.41 (bs, 2H), 2.70 (s, 3H). HRMS (ESI<sup>+</sup>): found [M + H]<sup>+</sup> 312.0902, C<sub>17</sub>H<sub>15</sub><sup>35</sup>ClN<sub>3</sub>O requires 312.0898.</div></div><div id="sec4-1-97" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 1-(6-((2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethyl)amino)-6-oxohexyl)-3,3-dimethyl-2-((1<i>E</i>,3<i>E</i>)-5-((<i>E</i>)-1,3,3-trimethyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-3<i>H</i>-indol-1-ium-5-sulfonate</h4><div class="NLM_p last">In the absence of light, <i>N</i>-(2-(2-aminoethoxy)ethyl)-2-(2-(2,6-dioxo-3-piperidyl)-1,3-dioxo-isoindolin-4-yl)oxy-acetamide hydrochloride <b>4</b> (0.60 mg, 0.0013 mmol) was added to a solution of sodium (2<i>E</i>)-2-((2<i>E</i>,4<i>E</i>)-5-(1-(6-(2,5-dioxopyrrolidin-1-yl)oxy-6-oxo-hexyl)-3,3-dimethyl-5-sulfonato-indol-1-ium-2-yl)penta-2,4-dienylidene)-1,3,3-trimethyl-indoline-5-sulfonate (1.00 mg, 0.0013 mmol) dissolved in 20 Î¼L of triethylamine and 300 Î¼L of DMF, and the deep-blue solution was left to stand for for 16 h. The crude product was then purified by semipreparative HPLC to give the desired product. LCMS (ESI<sup>+</sup>) RT = 2.59 min, 77%, M + H<sup>+</sup> 1043.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01406">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20055" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20055" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01406" class="ext-link">10.1021/acs.jmedchem.7b01406</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full chemistry experimental, NMR spectra of final compounds, physicochemical properties and PDP stability experimental, PDP design pictures, biochemical experimental, biology experimental, proteomics experimental, additional information, proteomics data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf">jm7b01406_si_001.pdf (17.36 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_002.csv">jm7b01406_si_002.csv (0.71 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01406" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul A. Clarke</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9342-1290" title="Orcid link">http://orcid.org/0000-0001-9342-1290</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#136372667f3d707f72617876537a70613d72703d6678"><span class="__cf_email__" data-cfemail="9eeeffebf2b0fdf2ffecf5fbdef7fdecb0fffdb0ebf5">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Workman</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1659-3034" title="Orcid link">http://orcid.org/0000-0003-1659-3034</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ddadbca8b1f3aab2afb6b0bcb39db4beaff3bcbef3a8b6"><span class="__cf_email__" data-cfemail="324253475e1c455d40595f535c725b51401c53511c4759">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Cheeseman</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1121-6985" title="Orcid link">http://orcid.org/0000-0003-1121-6985</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#117c7065657974663f7279747462747c707f517872633f70723f647a"><span class="__cf_email__" data-cfemail="b0ddd1c4c4d8d5c79ed3d8d5d5c3d5ddd1def0d9d3c29ed1d39ec5db">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keith Jones</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9440-4094" title="Orcid link">http://orcid.org/0000-0002-9440-4094</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#78131d110c10561217161d0b38111b0a56191b560d13"><span class="__cf_email__" data-cfemail="b9d2dcd0cdd197d3d6d7dccaf9d0dacb97d8da97ccd2">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicola E. A. Chessum</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4125-320X" title="Orcid link">http://orcid.org/0000-0003-4125-320X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Swee Y. Sharp</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John J. Caldwell</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">A. Elisa Pasqua</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7966-4672" title="Orcid link">http://orcid.org/0000-0002-7966-4672</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Birgit Wilding</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1896-3708" title="Orcid link">http://orcid.org/0000-0002-1896-3708</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giampiero Colombano</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian Collins</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8143-8498" title="Orcid link">http://orcid.org/0000-0002-8143-8498</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bugra Ozer</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1441-4162" title="Orcid link">http://orcid.org/0000-0002-1441-4162</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meirion Richards</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Rowlands</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark Stubbs</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7855-9435" title="Orcid link">http://orcid.org/0000-0001-7855-9435</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosemary Burke</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0011-9701" title="Orcid link">http://orcid.org/0000-0002-0011-9701</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">P. Craig McAndrew</span> - <span class="hlFld-Affiliation affiliation">Cancer Research
UK Cancer Therapeutics Unit at The Institute
of Cancer Research, London SW7 3RP, United Kingdom</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>N.E.A.C. and S.Y.S. contributed equally. S.Y.S. carried out in vitro cellular biology experiments. N.E.A.C and J.J.C. synthesized compounds. N.E.A.C., J.J.C., A.E.P., B.W., G.C., I.C., M.D.C., and K.J. contributed to the design of compounds. M.R. ran and designed the physicochemical and stability analysis of compounds. P.C.M. expressed and purified the recombinant proteins. B.O. analyzed proteomics data. M.R., M.S., and R.B. designed and carried out biochemical experiments. N.E.A.C., S.Y.S., P.C., P.W., M.D.C., and K.J. designed studies and interpreted results. N.E.A.C. and M.D.C. wrote the manuscript. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The Institute of Cancer Research has a potential financial interest in ligands of pirin and operates a Rewards to Discoverers scheme.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66884" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66884" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Cancer Research UK grant numbers C309/A8274 and C309/A11566. We acknowledge CRUK Centre funding and NHS funding to the NIHR Biomedical Research Centre at The Institute of Cancer Research, the Royal Marsden Hospital, and funding from Cancer Research Technology Pioneer Fund and Battle Against Cancer Investment Trust. Paul Workman is Cancer Research UK Life Fellow. We would like to thank Dr. Bissan Al-Lazikani for her helpful discussions and Nicky Evans for editorial assistance.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i47" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i47"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i48" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i48"> Abbreviations Used</h2><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationships</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">ABPP</td><td class="NLM_def"><p class="first last">activity-based protein profiling</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SNIPER</td><td class="NLM_def"><p class="first last">specific and nongenetic IAP-dependent protein eraser</p></td></tr><tr><td class="NLM_term">PDP</td><td class="NLM_def"><p class="first last">protein degradation probe</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance spectroscopy</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">KS</td><td class="NLM_def"><p class="first last">kinetic solubility</p></td></tr><tr><td class="NLM_term">tPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HSF1</td><td class="NLM_def"><p class="first last">heat shock transcription factor 1</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">Von HippelâLindau disease tumor suppressor</p></td></tr><tr><td class="NLM_term">IAP</td><td class="NLM_def"><p class="first last">inhibitors of apoptosis proteins</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">protein cereblon</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">proteinâprotein interaction</p></td></tr><tr><td class="NLM_term">TBAF</td><td class="NLM_def"><p class="first last">tetra-<i>n</i>-butylammonium fluoride</p></td></tr><tr><td class="NLM_term">HBD</td><td class="NLM_def"><p class="first last">hydrogen bond donor</p></td></tr><tr><td class="NLM_term">TMT</td><td class="NLM_def"><p class="first last">tandem mass tagging</p></td></tr><tr><td class="NLM_term">MS2</td><td class="NLM_def"><p class="first last">MS/MS mass spectrometry</p></td></tr><tr><td class="NLM_term">DTBAD</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butylazocarboxylate</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DBU</td><td class="NLM_def"><p class="first last">1,8-diazabicyclo(5.4.0)undec-7-ene</p></td></tr><tr><td class="NLM_term">SF</td><td class="NLM_def"><p class="first last">significant figures.</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Swinney, D. C.</span><span> </span><span class="NLM_article-title">Biochemical mechanisms of drug action: What does it take for success?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">801</span><span class="NLM_x">â</span> <span class="NLM_lpage">808</span><span class="refDoi">Â DOI: 10.1038/nrd1500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnrd1500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=15340390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=801-808&author=D.+C.+Swinney&title=Biochemical+mechanisms+of+drug+action%3A+What+does+it+take+for+success%3F&doi=10.1038%2Fnrd1500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Biochemical mechanisms of drug action: what does it take for success?</span></div><div class="casAuthors">Swinney, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">801-808</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Drug discovery is extremely difficult.  There are many unanticipated scientific, medical and business challenges to every drug discovery program.  It is important to increase our understanding of the fundamental properties of effective drugs so that we can anticipate potential problems in developing new agents.  This article addresses potential drug discovery and development risks assocd. with the biochem. mechanism of drug action, and proposes simple rules to minimize these risks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXhyDgozFUdrVg90H21EOLACvtfcHk0lj9vegweMaTEA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D&md5=b06bdc6ace3168dcb011679b43b72aed</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1500%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DBiochemical%2520mechanisms%2520of%2520drug%2520action%253A%2520What%2520does%2520it%2520take%2520for%2520success%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D801%26epage%3D808%26doi%3D10.1038%2Fnrd1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">SchÃ¼rmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janning, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziegler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Small-molecule target engagement in cells</span> <span class="citation_source-journal">Cell Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">â</span> <span class="NLM_lpage">441</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2016.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.chembiol.2016.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27049669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqtb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=435-441&author=M.+Sch%C3%BCrmannauthor=P.+Janningauthor=S.+Zieglerauthor=H.+Waldmann&title=Small-molecule+target+engagement+in+cells&doi=10.1016%2Fj.chembiol.2016.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Target Engagement in Cells</span></div><div class="casAuthors">Schuermann, Marc; Janning, Petra; Ziegler, Slava; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-441</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Monitoring how, when, and where small mols. engage their targets inside living cells is a crit. step in chem. biol. and pharmacol. research, because it enables compd. efficacy and confirmation of mode of action to be assessed.  In this mini-review we summarize the currently available methodologies to detect and prove direct target engagement in cells and offer a crit. view of their key advantages and disadvantages.  As the interest of the field shifts toward discovery and validation of high-quality agents, we expect that efforts to develop and refine these types of methodologies will also intensify in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFG52Nxfgik7Vg90H21EOLACvtfcHk0lgKa0nupX03iw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlSqtb4%253D&md5=b81e50faa188d916de61b5dc01294ea8</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCrmann%26aufirst%3DM.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall-molecule%2520target%2520engagement%2520in%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D435%26epage%3D441%26doi%3D10.1016%2Fj.chembiol.2016.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Simon, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niphakis, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">Determining target engagement in living systems</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">â</span> <span class="NLM_lpage">205</span><span class="refDoi">Â DOI: 10.1038/nchembio.1211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.1211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=23508173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWku7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=200-205&author=G.+M.+Simonauthor=M.+J.+Niphakisauthor=B.+F.+Cravatt&title=Determining+target+engagement+in+living+systems&doi=10.1038%2Fnchembio.1211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Determining target engagement in living systems</span></div><div class="casAuthors">Simon, Gabriel M.; Niphakis, Micah J.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-205</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. probes are crit. tools for elucidating the biol. functions of proteins and can lead to new medicines for treating disease.  The pharmacol. validation of protein function requires verification that chem. probes engage their intended targets in vivo.  Here we discuss technologies, both established and emergent, for measuring target engagement in living systems and propose that detg. this parameter should become std. practice for chem. probe and drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5eudh3YJSOLVg90H21EOLACvtfcHk0lgKa0nupX03iw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFWku7g%253D&md5=48a97d0d8ff7a7de4f9de0e9c54a1c5c</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1211%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DNiphakis%26aufirst%3DM.%2BJ.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDetermining%2520target%2520engagement%2520in%2520living%2520systems%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D200%26epage%3D205%26doi%3D10.1038%2Fnchembio.1211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Durham, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanco, M.</span><span> </span><span class="NLM_article-title">Target engagement in lead generation</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">998</span><span class="NLM_x">â</span> <span class="NLM_lpage">1008</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2014.12.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2014.12.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25630223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BC2MrgsFejug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=998-1008&author=T.+B.+Durhamauthor=M.+Blanco&title=Target+engagement+in+lead+generation&doi=10.1016%2Fj.bmcl.2014.12.076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement in lead generation</span></div><div class="casAuthors">Durham Timothy B; Blanco Maria-Jesus</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">998-1008</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmaceutical industry is currently facing multiple challenges, in particular the low number of new drug approvals in spite of the high level of R&D investment.  In order to improve target selection and assess properly the clinical hypothesis, it is important to start building an integrated drug discovery approach during Lead Generation.  This should include special emphasis on evaluating target engagement in the target tissue and linking preclinical to clinical readouts.  In this review, we would like to illustrate several strategies and technologies for assessing target engagement and the value of its application to medicinal chemistry efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTY-Uy19yFbSukdrW1FkXG-fW6udTcc2eaYDcwzH05qerntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrgsFejug%253D%253D&md5=ecafc397f5418f21d83f7881548ae490</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.12.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.12.076%26sid%3Dliteratum%253Aachs%26aulast%3DDurham%26aufirst%3DT.%2BB.%26aulast%3DBlanco%26aufirst%3DM.%26atitle%3DTarget%2520engagement%2520in%2520lead%2520generation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D998%26epage%3D1008%26doi%3D10.1016%2Fj.bmcl.2014.12.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Sun, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hays, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Periasamy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, R. N.</span><span> </span><span class="NLM_article-title">Monitoring protein interactions in living cells with fluorescence lifetime imaging microscopy</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">371</span><span class="NLM_x">â</span> <span class="NLM_lpage">391</span><span class="refDoi">Â DOI: 10.1016/B978-0-12-391857-4.00019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2FB978-0-12-391857-4.00019-7" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2012&pages=371-391&author=Y.+S.+Sunauthor=N.+M.+Haysauthor=A.+Periasamyauthor=M.+W.+Davidsonauthor=R.+N.+Day&title=Monitoring+protein+interactions+in+living+cells+with+fluorescence+lifetime+imaging+microscopy&doi=10.1016%2FB978-0-12-391857-4.00019-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-391857-4.00019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-391857-4.00019-7%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%2BS.%26aulast%3DHays%26aufirst%3DN.%2BM.%26aulast%3DPeriasamy%26aufirst%3DA.%26aulast%3DDavidson%26aufirst%3DM.%2BW.%26aulast%3DDay%26aufirst%3DR.%2BN.%26atitle%3DMonitoring%2520protein%2520interactions%2520in%2520living%2520cells%2520with%2520fluorescence%2520lifetime%2520imaging%2520microscopy%26jtitle%3DMethods%2520Enzymol.%26date%3D2012%26volume%3D504%26spage%3D371%26epage%3D391%26doi%3D10.1016%2FB978-0-12-391857-4.00019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Robers, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dart, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodroofe, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimprich, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirkland, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machleidt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupcho, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartnett, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmerman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niles, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohana, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slater, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, K. V.</span><span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">10091</span><span class="refDoi">Â DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=10091&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0lhr3E7eV9P2Jw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D10091%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Franken, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Childs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G. M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togel, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doce, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhard, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span> </span><span class="NLM_article-title">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1567</span><span class="NLM_x">â</span> <span class="NLM_lpage">1593</span><span class="refDoi">Â DOI: 10.1038/nprot.2015.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnprot.2015.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26379230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1567-1593&author=H.+Frankenauthor=T.+Mathiesonauthor=D.+Childsauthor=G.+M.+A.+Sweetmanauthor=T.+Wernerauthor=I.+Togelauthor=C.+Doceauthor=S.+Gadeauthor=M.+Bantscheffauthor=G.+Drewesauthor=F.+B.+Reinhardauthor=W.+Huberauthor=M.+M.+Savitski&title=Thermal+proteome+profiling+for+unbiased+identification+of+direct+and+indirect+drug+targets+using+multiplexed+quantitative+mass+spectrometry&doi=10.1038%2Fnprot.2015.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span></div><div class="casAuthors">Franken, Holger; Mathieson, Toby; Childs, Dorothee; Sweetman, Gavain M. A.; Werner, Thilo; Toegel, Ina; Doce, Carola; Gade, Stephan; Bantscheff, Marcus; Drewes, Gerard; Reinhard, Friedrich B. M.; Huber, Wolfgang; Savitski, Mikhail M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1567-1593</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The direct detection of drug-protein interactions in living cells is a major challenge in drug discovery research.  Recently, the authors introduced an approach termed thermal proteome profiling (TPP), which enables the monitoring of changes in protein thermal stability across the proteome using quant. mass spectrometry.  The authors detd. the intracellular thermal profiles for up to 7000 proteins, and by comparing profiles derived from cultured mammalian cells in the presence or absence of a drug it was possible to identify direct and indirect targets of drugs in living cells in an unbiased manner.  Here the authors demonstrate the complete workflow using the histone deacetylase inhibitor panobinostat.  The key to this approach is the use of isobaric tandem mass tag 10-plex (TMT10) reagents to label digested protein samples corresponding to each temp. point in the melting curve so that the samples can be analyzed by multiplexed quant. mass spectrometry.  Important steps in the bioinformatic anal. include data normalization, melting curve fitting and statistical significance detn. of compd. concn.-dependent changes in protein stability.  All anal. tools are made freely available as R and Python packages.  The workflow can be completed in 2 wk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS4LU9YprZqLVg90H21EOLACvtfcHk0lhr3E7eV9P2Jw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN&md5=d9c225616a35fc5dcc97db234452d7e1</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.101%26sid%3Dliteratum%253Aachs%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DChilds%26aufirst%3DD.%26aulast%3DSweetman%26aufirst%3DG.%2BM.%2BA.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DTogel%26aufirst%3DI.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DGade%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520proteome%2520profiling%2520for%2520unbiased%2520identification%2520of%2520direct%2520and%2520indirect%2520drug%2520targets%2520using%2520multiplexed%2520quantitative%2520mass%2520spectrometry%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D1567%26epage%3D1593%26doi%3D10.1038%2Fnprot.2015.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhard, F. B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franken, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jafari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dovega, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaeger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Tracking cancer drugs in living cells by thermal profiling of the proteome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255784</span><span class="refDoi">Â DOI: 10.1126/science.1255784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1126%2Fscience.1255784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25278616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=1255784&author=M.+M.+Savitskiauthor=F.+B.+M.+Reinhardauthor=H.+Frankenauthor=T.+Wernerauthor=M.+F.+Savitskiauthor=D.+Eberhardauthor=D.+M.+Molinaauthor=R.+Jafariauthor=R.+B.+Dovegaauthor=S.+Klaegerauthor=B.+Kusterauthor=P.+Nordlundauthor=M.+Bantscheffauthor=G.+Drewes&title=Tracking+cancer+drugs+in+living+cells+by+thermal+profiling+of+the+proteome&doi=10.1126%2Fscience.1255784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Tracking cancer drugs in living cells by thermal profiling of the proteome</span></div><div class="casAuthors">Savitski Mikhail M; Reinhard Friedrich B M; Franken Holger; Werner Thilo; Savitski Maria Falth; Eberhard Dirk; Bantscheff Marcus; Drewes Gerard; Martinez Molina Daniel; Jafari Rozbeh; Dovega Rebecca Bakszt; Klaeger Susan; Kuster Bernhard; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6205</span>),
    <span class="NLM_cas:pages">1255784</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The thermal stability of proteins can be used to assess ligand binding in living cells.  We have generalized this concept by determining the thermal profiles of more than 7000 proteins in human cells by means of mass spectrometry.  Monitoring the effects of small-molecule ligands on the profiles delineated more than 50 targets for the kinase inhibitor staurosporine.  We identified the heme biosynthesis enzyme ferrochelatase as a target of kinase inhibitors and suggest that its inhibition causes the phototoxicity observed with vemurafenib and alectinib.  Thermal shifts were also observed for downstream effectors of drug treatment.  In live cells, dasatinib induced shifts in BCR-ABL pathway proteins, including CRK/CRKL.  Thermal proteome profiling provides an unbiased measure of drug-target engagement and facilitates identification of markers for drug efficacy and toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0ur3d8ba-Bu2mnZh2OgS0fW6udTcc2ebNWZeu-OaVD7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D&md5=493a89ad107e656e0f0786ff5c8e5f63</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1126%2Fscience.1255784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1255784%26sid%3Dliteratum%253Aachs%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%2BM.%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DSavitski%26aufirst%3DM.%2BF.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DDovega%26aufirst%3DR.%2BB.%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DTracking%2520cancer%2520drugs%2520in%2520living%2520cells%2520by%2520thermal%2520profiling%2520of%2520the%2520proteome%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D1255784%26doi%3D10.1126%2Fscience.1255784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Huber, K. V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olek, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¼ller, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, C. S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colinge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1055</span><span class="NLM_x">â</span> <span class="NLM_lpage">1057</span><span class="refDoi">Â DOI: 10.1038/nmeth.3590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnmeth.3590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26389571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1055-1057&author=K.+V.+M.+Huberauthor=K.+M.+Olekauthor=A.+C.+M%C3%BCllerauthor=C.+S.+H.+Tanauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furga&title=Proteome-wide+drug+and+metabolite+interaction+mapping+by+thermal-stability+profiling&doi=10.1038%2Fnmeth.3590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span></div><div class="casAuthors">Huber, Kilian V. M.; Olek, Karin M.; Mueller, Andre C.; Tan, Chris Soon Heng; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1055-1057</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal stabilization of proteins after ligand binding provides an efficient means to assess the binding of small mols. to proteins.  We show here that in combination with quant. mass spectrometry, the approach allows for the systematic survey of protein engagement by cellular metabolites and drugs.  We profiled the targets of the drugs methotrexate and (S)-crizotinib and the metabolite 2'3'-cGAMP in intact cells and identified the 2'3'-cGAMP cognate transmembrane receptor STING, involved in immune signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrow66ugdg1oLVg90H21EOLACvtfcHk0lg3W6GJ4MSYnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI&md5=619a2c08c4f151f4e3ab8bd8fc45c2b6</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3590%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DOlek%26aufirst%3DK.%2BM.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DTan%26aufirst%3DC.%2BS.%2BH.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProteome-wide%2520drug%2520and%2520metabolite%2520interaction%2520mapping%2520by%2520thermal-stability%2520profiling%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1055%26epage%3D1057%26doi%3D10.1038%2Fnmeth.3590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Mateus, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¤Ã¤ttÃ¤, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span> </span><span class="NLM_article-title">Thermal proteome profiling: unbiased assessment of protein state through heat-induced stability changes</span> <span class="citation_source-journal">Proteome Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="refDoi">Â DOI: 10.1186/s12953-017-0122-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1186%2Fs12953-017-0122-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28652855" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=13&author=A.+Mateusauthor=T.+A.+M%C3%A4%C3%A4tt%C3%A4author=M.+M.+Savitski&title=Thermal+proteome+profiling%3A+unbiased+assessment+of+protein+state+through+heat-induced+stability+changes&doi=10.1186%2Fs12953-017-0122-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1186%2Fs12953-017-0122-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12953-017-0122-4%26sid%3Dliteratum%253Aachs%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DM%25C3%25A4%25C3%25A4tt%25C3%25A4%26aufirst%3DT.%2BA.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520proteome%2520profiling%253A%2520unbiased%2520assessment%2520of%2520protein%2520state%2520through%2520heat-induced%2520stability%2520changes%26jtitle%3DProteome%2520Sci.%26date%3D2017%26volume%3D15%26spage%3D13%26doi%3D10.1186%2Fs12953-017-0122-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozarich, J. W.</span><span> </span><span class="NLM_article-title">Activity-based protein profiling: from enzyme chemistry to proteomic chemistry</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">â</span> <span class="NLM_lpage">414</span><span class="refDoi">Â DOI: 10.1146/annurev.biochem.75.101304.124125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1146%2Fannurev.biochem.75.101304.124125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=18366325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2008&pages=383-414&author=B.+F.+Cravattauthor=A.+T.+Wrightauthor=J.+W.+Kozarich&title=Activity-based+protein+profiling%3A+from+enzyme+chemistry+to+proteomic+chemistry&doi=10.1146%2Fannurev.biochem.75.101304.124125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: From enzyme chemistry to proteomic chemistry</span></div><div class="casAuthors">Cravatt, Benjamin F.; Wright, Aaron T.; Kozarich, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">383-414</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Genome sequencing projects have provided researchers with a complete inventory of the predicted proteins produced by eukaryotic and prokaryotic organisms.  Assignment of functions to these proteins represents one of the principal challenges for the field of proteomics.  Activity-based protein profiling (ABPP) has emerged as a powerful chem. proteomic strategy to characterize enzyme function directly in native biol. systems on a global scale.  Here, the authors review the basic technol. of ABPP, the enzyme classes addressable by this method, and the biol. discoveries attributable to its application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQFnUWDsqzOLVg90H21EOLACvtfcHk0lg3W6GJ4MSYnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D&md5=2f2c471169d2473f7024eeb41d4eaecd</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.75.101304.124125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.75.101304.124125%26sid%3Dliteratum%253Aachs%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DWright%26aufirst%3DA.%2BT.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520from%2520enzyme%2520chemistry%2520to%2520proteomic%2520chemistry%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2008%26volume%3D77%26spage%3D383%26epage%3D414%26doi%3D10.1146%2Fannurev.biochem.75.101304.124125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lanning, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitby, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dix, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douhan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hett, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joslyn, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niessen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnute, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hulce, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whiteley, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayward, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">â</span> <span class="NLM_lpage">769</span><span class="refDoi">Â DOI: 10.1038/nchembio.1582</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.1582" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25038787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=760-769&author=B.+R.+Lanningauthor=L.+R.+Whitbyauthor=M.+M.+Dixauthor=J.+Douhanauthor=A.+M.+Gilbertauthor=E.+C.+Hettauthor=T.+O.+Johnsonauthor=C.+Joslynauthor=J.+C.+Kathauthor=S.+Niessenauthor=L.+R.+Robertsauthor=M.+E.+Schnuteauthor=C.+Wangauthor=J.+J.+Hulceauthor=B.+Weiauthor=L.+O.+Whiteleyauthor=M.+M.+Haywardauthor=B.+F.+Cravatt&title=A+road+map+to+evaluate+the+proteome-wide+selectivity+of+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1582"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span></div><div class="casAuthors">Lanning, Bryan R.; Whitby, Landon R.; Dix, Melissa M.; Douhan, John; Gilbert, Adam M.; Hett, Erik C.; Johnson, Theodore O.; Joslyn, Chris; Kath, John C.; Niessen, Sherry; Roberts, Lee R.; Schnute, Mark E.; Wang, Chu; Hulce, Jonathan J.; Wei, Baoxian; Whiteley, Laurence O.; Hayward, Matthew M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">760-767</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinases are principal components of signal transduction pathways and the focus of intense basic and drug discovery research.  Irreversible inhibitors that covalently modify non-catalytic cysteines in kinase active sites have emerged as valuable probes and approved drugs.  Many protein classes, however, have functional cysteines, and therefore understanding the proteome-wide selectivity of covalent kinase inhibitors is imperative.  Here, we accomplish this objective using activity-based protein profiling coupled with quant. MS to globally map the targets, both specific and nonspecific, of covalent kinase inhibitors in human cells.  Many of the specific off-targets represent nonkinase proteins that, notably, have conserved active site cysteines.  We define windows of selectivity for covalent kinase inhibitors and show that, when these windows are exceeded, rampant proteome-wide reactivity and kinase target-independent cell death conjointly occur.  Our findings, taken together, provide an exptl. road map to illuminate opportunities and surmount challenges for the development of covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMAiUlEg9R5rVg90H21EOLACvtfcHk0lg3W6GJ4MSYnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO&md5=6f17b0d1b00e088f37c511eea577d290</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1582%26sid%3Dliteratum%253Aachs%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DWhitby%26aufirst%3DL.%2BR.%26aulast%3DDix%26aufirst%3DM.%2BM.%26aulast%3DDouhan%26aufirst%3DJ.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DHett%26aufirst%3DE.%2BC.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DJoslyn%26aufirst%3DC.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DSchnute%26aufirst%3DM.%2BE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHulce%26aufirst%3DJ.%2BJ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWhiteley%26aufirst%3DL.%2BO.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DA%2520road%2520map%2520to%2520evaluate%2520the%2520proteome-wide%2520selectivity%2520of%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D760%26epage%3D769%26doi%3D10.1038%2Fnchembio.1582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit7c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Dubach, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuccarese, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giedt, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meimetis, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinegoni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissleder, R.</span><span> </span><span class="NLM_article-title">Quantitating drug-target engagement in single cells in vitro and in vivo</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">168</span><span class="NLM_x">â</span> <span class="NLM_lpage">173</span><span class="refDoi">Â DOI: 10.1038/nchembio.2248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.2248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27918558" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=168-173&author=J.+M.+Dubachauthor=E.+Kimauthor=K.+Yangauthor=M.+Cuccareseauthor=R.+J.+Giedtauthor=L.+G.+Meimetisauthor=C.+Vinegoniauthor=R.+Weissleder&title=Quantitating+drug-target+engagement+in+single+cells+in+vitro+and+in+vivo&doi=10.1038%2Fnchembio.2248"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7c&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2248%26sid%3Dliteratum%253Aachs%26aulast%3DDubach%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DCuccarese%26aufirst%3DM.%26aulast%3DGiedt%26aufirst%3DR.%2BJ.%26aulast%3DMeimetis%26aufirst%3DL.%2BG.%26aulast%3DVinegoni%26aufirst%3DC.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DQuantitating%2520drug-target%2520engagement%2520in%2520single%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D13%26spage%3D168%26epage%3D173%26doi%3D10.1038%2Fnchembio.2248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bondeson, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mares, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, I. E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routly, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faelth-Savitski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casillas, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votta, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Besten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harling, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">611</span><span class="NLM_x">â</span> <span class="NLM_lpage">617</span><span class="refDoi">Â DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lgoFIh2A9bxJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuhira, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ujikawa, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimokawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sano, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koyama, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teratani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span> </span><span class="NLM_article-title">In vivo knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">292</span><span class="NLM_x">, </span> <span class="NLM_fpage">4556</span><span class="NLM_x">â</span> <span class="NLM_lpage">4570</span><span class="refDoi">Â DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor Î± (ERÎ±), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERÎ± levels in tumor xenografts and suppressed the growth of ERÎ±-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERÎ± via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0lgoFIh2A9bxJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, P.</span><span> </span><span class="NLM_article-title">DCAFs, the missing link of the CUL4-DDB1 ubiquitin ligase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">775</span><span class="NLM_x">â</span> <span class="NLM_lpage">780</span><span class="refDoi">Â DOI: 10.1016/j.molcel.2007.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.molcel.2007.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=17588513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFSru78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=775-780&author=J.+Leeauthor=P.+Zhou&title=DCAFs%2C+the+missing+link+of+the+CUL4-DDB1+ubiquitin+ligase&doi=10.1016%2Fj.molcel.2007.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">DCAFs, the missing link of the CUL4-DDB1 ubiquitin ligase</span></div><div class="casAuthors">Lee, Jennifer; Zhou, Pengbo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">775-780</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  CUL4-DDB1 ubiquitin ligase regulates cell proliferation, survival, DNA repair, and genomic integrity through targeted ubiquitination of key regulators; yet, the substrate receptors that dictate the specificity of this ubiquitination machinery have been largely unknown.  Recent work has identified a family of DDB1 and CUL4-assocd. factors (DCAFs) as substrate receptors, implicating a broad spectrum of cellular processes regulated by CUL4-DDB1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWyUZvvs-r7Vg90H21EOLACvtfcHk0lhRMdpcluuDmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFSru78%253D&md5=c18a4a72c16a452add79520a554acdc8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DDCAFs%252C%2520the%2520missing%2520link%2520of%2520the%2520CUL4-DDB1%2520ubiquitin%2520ligase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D26%26spage%3D775%26epage%3D780%26doi%3D10.1016%2Fj.molcel.2007.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessum, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rye, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasqua, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilding, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlands, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâFee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henley, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Te Poele, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellegrino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raynaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Montfort, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eccles, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span> </span><span class="NLM_article-title">Discovery of a chemical probe bisamide (CCT251236): an orally bioavailable efficacious pirin ligand from a heat shock transcription factor 1 (HSF1) phenotypic screen</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">â</span> <span class="NLM_lpage">201</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=180-201&author=M.+D.+Cheesemanauthor=N.+E.+Chessumauthor=C.+S.+Ryeauthor=A.+E.+Pasquaauthor=M.+J.+Tuckerauthor=B.+Wildingauthor=L.+E.+Evansauthor=S.+Lepriauthor=M.+Richardsauthor=S.+Y.+Sharpauthor=S.+Aliauthor=M.+Rowlandsauthor=L.+O%E2%80%99Feeauthor=A.+Miahauthor=A.+Hayesauthor=A.+T.+Henleyauthor=M.+Powersauthor=R.+Te+Poeleauthor=E.+De+Billyauthor=L.+Pellegrinoauthor=F.+Raynaudauthor=R.+Burkeauthor=R.+L.+van+Montfortauthor=S.+A.+Ecclesauthor=P.+Workmanauthor=K.+Jones&title=Discovery+of+a+chemical+probe+bisamide+%28CCT251236%29%3A+an+orally+bioavailable+efficacious+pirin+ligand+from+a+heat+shock+transcription+factor+1+%28HSF1%29+phenotypic+screen&doi=10.1021%2Facs.jmedchem.6b01055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01055%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DChessum%26aufirst%3DN.%2BE.%26aulast%3DRye%26aufirst%3DC.%2BS.%26aulast%3DPasqua%26aufirst%3DA.%2BE.%26aulast%3DTucker%26aufirst%3DM.%2BJ.%26aulast%3DWilding%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Fee%26aufirst%3DL.%26aulast%3DMiah%26aufirst%3DA.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHenley%26aufirst%3DA.%2BT.%26aulast%3DPowers%26aufirst%3DM.%26aulast%3DTe%2BPoele%26aufirst%3DR.%26aulast%3DDe%2BBilly%26aufirst%3DE.%26aulast%3DPellegrino%26aufirst%3DL.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520a%2520chemical%2520probe%2520bisamide%2520%2528CCT251236%2529%253A%2520an%2520orally%2520bioavailable%2520efficacious%2520pirin%2520ligand%2520from%2520a%2520heat%2520shock%2520transcription%2520factor%25201%2520%2528HSF1%2529%2520phenotypic%2520screen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D180%26epage%3D201%26doi%3D10.1021%2Facs.jmedchem.6b01055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehmani, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Serrano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span> </span><span class="NLM_article-title">Pirin is an iron-dependent redox regulator of NF-ÎºB</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">9722</span><span class="NLM_x">â</span> <span class="NLM_lpage">9727</span><span class="refDoi">Â DOI: 10.1073/pnas.1221743110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1073%2Fpnas.1221743110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=23716661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=9722-9727&author=F.+Liuauthor=I.+Rehmaniauthor=S.+Esakiauthor=R.+Fuauthor=L.+Chenauthor=V.+de+Serranoauthor=A.+Liu&title=Pirin+is+an+iron-dependent+redox+regulator+of+NF-%CE%BAB&doi=10.1073%2Fpnas.1221743110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin is an iron-dependent redox regulator of NF-ÎºB</span></div><div class="casAuthors">Liu, Fange; Rehmani, Imran; Esaki, Shingo; Fu, Rong; Chen, Lirong; de Serrano, Vesna; Liu, Aimin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9722-9727, S9722/1-S9722/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pirin is a nuclear nonheme Fe protein of unknown function present in all human tissues.  Here the authors describe that pirin may act as a redox sensor for the nuclear factor ÎºB (NF-ÎºB) transcription factor, a crit. mediator of intracellular signaling that has been linked to cellular responses to proinflammatory signals and controls the expression of a vast array of genes involved in immune and stress responses.  Pirin's regulatory effect was tested with several metals and at different oxidns. states, and the authors' spectroscopic results show that only the ferric form of pirin substantially facilitates binding of NF-ÎºB proteins to target ÎºB genes, a finding that suggests that pirin performs a redox-sensing role in NF-ÎºB regulation.  The mol. mechanism of such a metal identity- and redox state-dependent regulation is revealed by the authors' structural studies of pirin.  The ferrous and ferric pirin proteins differ only by one electron, yet they have distinct conformations.  The Fe center is shown to play an allosteric role on an Î²-shaped surface area that has two distinct conformations based on the identity and the formal redox state of the metal.  The authors show that the R-shaped area composes the interface for pirin-NF-ÎºB binding that is responsible for modulation of NF-ÎºB's DNA-binding properties.  The nonheme Fe protein pirin is proposed to serve as a reversible functional switch that enables NF-ÎºB to respond to changes in the redox levels of the cell nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Tm-I0Q4mD7Vg90H21EOLACvtfcHk0lhRMdpcluuDmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF&md5=d81627e3a8aa09ee6a46a8557719394d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221743110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221743110%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRehmani%26aufirst%3DI.%26aulast%3DEsaki%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3Dde%2BSerrano%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DA.%26atitle%3DPirin%2520is%2520an%2520iron-dependent%2520redox%2520regulator%2520of%2520NF-%25CE%25BAB%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D9722%26epage%3D9727%26doi%3D10.1073%2Fpnas.1221743110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Dunwell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purvis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, S.</span><span> </span><span class="NLM_article-title">Cupins: the most functionally diverse protein superfamily?</span> <span class="citation_source-journal">Phytochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">â</span> <span class="NLM_lpage">17</span><span class="refDoi">Â DOI: 10.1016/j.phytochem.2003.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.phytochem.2003.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=14697267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2004&pages=7-17&author=J.+M.+Dunwellauthor=A.+Purvisauthor=S.+Khuri&title=Cupins%3A+the+most+functionally+diverse+protein+superfamily%3F&doi=10.1016%2Fj.phytochem.2003.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cupins: the most functionally diverse protein superfamily?</span></div><div class="casAuthors">Dunwell, Jim M.; Purvis, Alan; Khuri, Sawsan</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry (Elsevier)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-17</span>CODEN:
                <span class="NLM_cas:coden">PYTCAS</span>;
        ISSN:<span class="NLM_cas:issn">0031-9422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  The cupin superfamily of proteins, named on the basis of a conserved Î²-barrel fold ('cupa' is the Latin term for a small barrel), was originally discovered using a conserved motif found within germin and germin-like proteins from higher plants.  Previous anal. of cupins had identified some 18 different functional classes that range from single-domain bacterial enzymes such as isomerases and epimerases involved in the modification of cell wall carbohydrates, through to two-domain bicupins such as the desiccation-tolerant seed storage globulins, and multidomain transcription factors including one linked to the nodulation response in legumes.  Recent advances in comparative genomics, and the resoln. of many more 3-D structures have now revealed that the largest subset of the cupin superfamily is the 2-oxyglutarate-Fe2+ dependent dioxygenases.  The substrates for this subclass of enzyme are many and varied and in total amt. to probably 50-100 different biochem. reactions, including several involved in plant growth and development.  Although the majority of enzymic cupins contain iron as an active site metal, other members contain either copper, zinc, cobalt, nickel or manganese ions as a cofactor, with each cofactor allowing a different type of chem. to occur within the conserved tertiary structure.  This review discusses the range of structures and functions found in this most diverse of superfamilies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGLBE2G_DAE7Vg90H21EOLACvtfcHk0lhuDUTbop8x3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D&md5=8af457482acd815ffc0d8ff05bf247c7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2003.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2003.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26aulast%3DPurvis%26aufirst%3DA.%26aulast%3DKhuri%26aufirst%3DS.%26atitle%3DCupins%253A%2520the%2520most%2520functionally%2520diverse%2520protein%2520superfamily%253F%26jtitle%3DPhytochemistry%26date%3D2004%26volume%3D65%26spage%3D7%26epage%3D17%26doi%3D10.1016%2Fj.phytochem.2003.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Komai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niwa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasazawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simizu, S.</span><span> </span><span class="NLM_article-title">Pirin regulates epithelial to mesenchymal transition independently of BCL3-SLUG signaling</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">589</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">â</span> <span class="NLM_lpage">743</span><span class="refDoi">Â DOI: 10.1016/j.febslet.2015.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.febslet.2015.01.040" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=738-743&author=K.+Komaiauthor=Y.+Niwaauthor=Y.+Sasazawaauthor=S.+Simizu&title=Pirin+regulates+epithelial+to+mesenchymal+transition+independently+of+BCL3-SLUG+signaling&doi=10.1016%2Fj.febslet.2015.01.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DKomai%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DY.%26aulast%3DSasazawa%26aufirst%3DY.%26aulast%3DSimizu%26aufirst%3DS.%26atitle%3DPirin%2520regulates%2520epithelial%2520to%2520mesenchymal%2520transition%2520independently%2520of%2520BCL3-SLUG%2520signaling%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D738%26epage%3D743%26doi%3D10.1016%2Fj.febslet.2015.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span> </span><span class="NLM_article-title">Protein knockdown technology: Application of ubiquitin ligase to cancer therapy</span> <span class="citation_source-journal">Curr. Cancer Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">136</span><span class="NLM_x">â</span> <span class="NLM_lpage">146</span><span class="refDoi">Â DOI: 10.2174/1568009616666151112122502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.2174%2F1568009616666151112122502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26560118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=136-146&author=N.+Ohokaauthor=N.+Shibataauthor=T.+Hattoriauthor=M.+Naito&title=Protein+knockdown+technology%3A+Application+of+ubiquitin+ligase+to+cancer+therapy&doi=10.2174%2F1568009616666151112122502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Knockdown Technology: Application of Ubiquitin Ligase to Cancer Therapy</span></div><div class="casAuthors">Ohoka, Nobumichi; Shibata, Norihito; Hattori, Takayuki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-146</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Selective degrdn. of pathogenic proteins by small mols. in cells is a novel approach for development of therapeutic agents against various diseases, including cancer.  We and others have developed a protein knockdown technol. with a series of hybrid small compds., called SNIPERs (Specific and Nongenetic IAP-dependent Protein ERasers); and peptidic chimeric mols., called PROTACs (proteolysis-targeting chimeric mols.), which induce selective degrdn. of target proteins via the ubiquitin-proteasome pathway.  These compds. include two different ligands connected by a linker; one is a ligand for a ubiquitin ligase and the other is a ligand for the target protein, which are expected to crosslink these proteins in cells.  Theor., any cytosolic protein can be targeted for degrdn. by this technol.  To date, several SNIPERs and PROTACs against various oncogenic proteins have been developed, which specifically induce polyubiquitylation and proteasomal degrdn. of the oncogenic proteins, resulting in cell death, growth arrest, or impaired migration of cancer cells.  Thus, this protein knockdown technol. has a great potential for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmS_kGOWfeuLVg90H21EOLACvtfcHk0lhuDUTbop8x3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D&md5=d78f1ae70ea89843d9f946454cc95058</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.2174%2F1568009616666151112122502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009616666151112122502%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DProtein%2520knockdown%2520technology%253A%2520Application%2520of%2520ubiquitin%2520ligase%2520to%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2016%26volume%3D16%26spage%3D136%26epage%3D146%26doi%3D10.2174%2F1568009616666151112122502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Collins, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopra, R.</span><span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">474</span><span class="NLM_x">, </span> <span class="NLM_fpage">1127</span><span class="NLM_x">â</span> <span class="NLM_lpage">1147</span><span class="refDoi">Â DOI: 10.1042/BCJ20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1042%2FBCJ20160762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28298557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin-proteasome+pathway&doi=10.1042%2FBCJ20160762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Collins, Ian; Wang, Hannah; Caldwell, John J.; Chopra, Raj</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1127-1147</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Manipulation of the ubiquitin-proteasome system to achieve targeted degrdn. of proteins within cells using chem. tools and drugs has the potential to transform pharmacol. and therapeutic approaches in cancer and other diseases.  An increased understanding of the mol. mechanism of thalidomide and its analogs following their clin. use has unlocked small-mol. modulation of the substrate specificity of the E3 ligase cereblon (CRBN), which in turn has resulted in the advancement of new immunomodulatory drugs (IMiDs) into the clinic.  The degrdn. of multiple context-specific proteins by these pleiotropic small mols. provides a means to uncover new cell biol. and to generate future drug mols. against currently undruggable targets.  In parallel, the development of larger bifunctional mols. that bring together highly specific protein targets in complexes with CRBN, von Hippel-Lindau, or other E3 ligases to promote ubiquitin-dependent degrdn. has progressed to generate selective chem. compds. with potent effects in cells and in vivo models, providing valuable tools for biol. target validation and with future potential for therapeutic use.  In this review, we survey recent breakthroughs achieved in these two complementary methods and the discovery of new modes of direct and indirect engagement of target proteins with the proteasome.  We discuss the exptl. characterization that validates the use of mols. that promote protein degrdn. as chem. tools, the preclin. and clin. examples disclosed to date, and the future prospects for this exciting area of chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkoPeMHzjTbVg90H21EOLACvtfcHk0lhuDUTbop8x3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D&md5=8c9ccac22f5088d776b81203dc8ebf2b</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin-proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2FBCJ20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Small-molecule PROTACS: New approaches to protein degradation</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1966</span><span class="NLM_x">â</span> <span class="NLM_lpage">1973</span><span class="refDoi">Â DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+New+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljs5lZId0yqpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520New%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Induced protein degradation: An emerging drug discovery paradigm</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">â</span> <span class="NLM_lpage">114</span><span class="refDoi">Â DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+An+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15dR"><div class="casContent"><span class="casTitleNuber">15d</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljs5lZId0yqpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit15d&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520An%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="note"><p class="first last">For example,</p></div><div class="NLM_citation" id="cit16a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1770</span><span class="NLM_x">â</span> <span class="NLM_lpage">1777</span><span class="refDoi">Â DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16aR"><div class="casContent"><span class="casTitleNuber">16a</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1Î±.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0ljs5lZId0yqpg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit16a&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit16b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altieri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crew, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">â</span> <span class="NLM_lpage">763</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16bR"><div class="casContent"><span class="casTitleNuber">16b</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lgdhv1YfNrelA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit16b&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Galdeano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadd, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scaffidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Molle, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birced, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Structure guided design and optimization of small molecules targeting the protein-protein interaction between the von hippel-lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8657</span><span class="NLM_x">â</span> <span class="NLM_lpage">8663</span><span class="refDoi">Â DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure+guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure%2520guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Demizu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, M.</span><span> </span><span class="NLM_article-title">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">4865</span><span class="NLM_x">â</span> <span class="NLM_lpage">4869</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2016.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27666635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4865-4869&author=Y.+Demizuauthor=N.+Shibataauthor=T.+Hattoriauthor=N.+Ohokaauthor=H.+Motoiauthor=T.+Misawaauthor=T.+Shodaauthor=M.+Naitoauthor=M.+Kurihara&title=Development+of+BCR-ABL+degradation+inducers+via+the+conjugation+of+an+imatinib+derivative+and+a+cIAP1+ligand&doi=10.1016%2Fj.bmcl.2016.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span></div><div class="casAuthors">Demizu, Yosuke; Shibata, Norihito; Hattori, Takayuki; Ohoka, Nobumichi; Motoi, Hiromi; Misawa, Takashi; Shoda, Takuji; Naito, Mikihiko; Kurihara, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4865-4869</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The manipulation of protein stability with small mols. has great potential as a technique for aiding the development of clin. therapies, including treatments for cancer.  In this study, BCR-ABL protein degrdn. inducers called SNIPER(ABL) (Specific and Nongenetic inhibitors of apoptosis protein [IAP]-dependent Protein Erasers) were developed.  The designed mols. contained two biol. active scaffolds: one was an imatinib deriv. that binds to BCL-ABL and the other was a Me bestatin that binds to cellular IAP 1 (cIAP1).  The hybrid mols., SNIPER(ABL), were expected to recruit BCR-ABL to cIAP1 for removal by proteasomes.  In fact, SNIPER(ABL) induced the degrdn. of BCR-ABL protein and a subsequent redn. in cell growth.  Thus, the degrdn. of BCR-ABL by SNIPER(ABL) is one potential strategy for treating BCR-ABL driven chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLT6dtmVPL47Vg90H21EOLACvtfcHk0lgdhv1YfNrelA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK&md5=31c045087293acf60932a9d6c4f38504</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMotoi%26aufirst%3DH.%26aulast%3DMisawa%26aufirst%3DT.%26aulast%3DShoda%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DKurihara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520BCR-ABL%2520degradation%2520inducers%2520via%2520the%2520conjugation%2520of%2520an%2520imatinib%2520derivative%2520and%2520a%2520cIAP1%2520ligand%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4865%26epage%3D4869%26doi%3D10.1016%2Fj.bmcl.2016.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Winter, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">1376</span><span class="NLM_x">â</span> <span class="NLM_lpage">1381</span><span class="refDoi">Â DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0ljFzuXQt5Ox-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><div class="note"><p class="first last">A recent study demonstrated a method to determine which E3 ligase can be manipulated for protein degradation:</p></div><span class="NLM_contrib-group">Ottis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cromm, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Assessing different E3 ligases for small molecule induced protein ubiquitination and degradation</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2570</span><span class="NLM_x">â</span> <span class="NLM_lpage">2578</span><span class="refDoi">Â DOI: 10.1021/acschembio.7b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2570-2578&author=P.+Ottisauthor=M.+Toureauthor=P.+M.+Crommauthor=E.+Koauthor=J.+L.+Gustafsonauthor=C.+M.+Crews&title=Assessing+different+E3+ligases+for+small+molecule+induced+protein+ubiquitination+and+degradation&doi=10.1021%2Facschembio.7b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing Different E3 Ligases for Small Molecule Induced Protein Ubiquitination and Degradation</span></div><div class="casAuthors">Ottis, Philipp; Toure, Momar; Cromm, Philipp M.; Ko, Eunhwa; Gustafson, Jeffrey L.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2570-2578</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol., the recruitment of E3 ubiquitin ligases to induce the degrdn. of a protein target, is rapidly impacting chem. biol., as well as modern drug development.  Here, we explore the breadth of this approach by evaluating different E3 ubiquitin ligases engineered in their substrate binding domains to accept a recruiting ligand.  Five out of six E3 ligases were found to be amenable to recruitment for target degrdn.  Taking advantage of the tight spatio-temporal control of inducing ubiquitination on a pre-selected target in living cells, we focused on two of the engineered E3 ligases, Î²TRCP and parkin, to characterize their ability to induce ubiquitination in comparison with the PROTAC-recruited endogenous E3 ligases VHL and cereblon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhBU1NFnjNbVg90H21EOLACvtfcHk0ljFzuXQt5Ox-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O&md5=b643d34cb33b330a7658b8b7feb2c6cc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00485%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAssessing%2520different%2520E3%2520ligases%2520for%2520small%2520molecule%2520induced%2520protein%2520ubiquitination%2520and%2520degradation%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2570%26epage%3D2578%26doi%3D10.1021%2Facschembio.7b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Cyrus, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehenkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swanson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K.</span><span> </span><span class="NLM_article-title">Impact of linker length on the activity of PROTACs</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">â</span> <span class="NLM_lpage">364</span><span class="refDoi">Â DOI: 10.1039/C0MB00074D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1039%2FC0MB00074D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=20922213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslGhsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=359-364&author=K.+Cyrusauthor=M.+Wehenkelauthor=E.+Choiauthor=H.+Hanauthor=H.+Leeauthor=H.+Swansonauthor=K.+Kim&title=Impact+of+linker+length+on+the+activity+of+PROTACs&doi=10.1039%2FC0MB00074D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of linker length on the activity of PROTACs</span></div><div class="casAuthors">Cyrus, Kedra; Wehenkel, Marie; Choi, Eun-Young; Han, Hyeong-Jun; Lee, Hyosung; Swanson, Hollie; Kim, Kyung-Bo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">359-364</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Conventional genetic approaches have provided a powerful tool in the study of proteins.  However, these techniques often preclude selective manipulation of temporal and spatial protein functions, which is crucial for the investigation of dynamic cellular processes.  To overcome these limitations, a small mol.-based novel technol. termed "PROteolysis TArgeting ChimeraS (PROTACs)" has been developed, targeting proteins for degrdn. at the post-translational level.  Despite the promising potential of PROTACs to serve as mol. probes of complex signaling pathways, their design has not been generalized for broad application.  Here, we present the first generalized approach for PROTAC design by fine-tuning the distance between the two participating partner proteins, the E3 ubiquitin ligase and the target protein.  As such, we took a chem. approach to create estrogen receptor (ER)-Î± targeting PROTACs with varying linker lengths and the loss of the ER in cultured cells was monitored via western blot and fluorometric analyses.  We found a significant effect of chain length on PROTAC efficacy, and, in this case, the optimum distance between the E3 recognition motif and the ligand was a 16 atom chain length.  The information gathered from this expt. may offer a generalizable PROTAC design strategy to further the expansion of the PROTAC toolbox, opening new possibilities for the broad application of the PROTAC strategy in the study of multiple signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP7hxQSvFw57Vg90H21EOLACvtfcHk0ljFzuXQt5Ox-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslGhsw%253D%253D&md5=56dc56cb3a73d17bc8903c4962d7648c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC0MB00074D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0MB00074D%26sid%3Dliteratum%253Aachs%26aulast%3DCyrus%26aufirst%3DK.%26aulast%3DWehenkel%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DE.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DSwanson%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DK.%26atitle%3DImpact%2520of%2520linker%2520length%2520on%2520the%2520activity%2520of%2520PROTACs%26jtitle%3DMol.%2520BioSyst.%26date%3D2011%26volume%3D7%26spage%3D359%26epage%3D364%26doi%3D10.1039%2FC0MB00074D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Douglass, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegel, D. A.</span><span> </span><span class="NLM_article-title">A comprehensive mathematical model for three-body binding equilibria</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">6092</span><span class="NLM_x">â</span> <span class="NLM_lpage">6099</span><span class="refDoi">Â DOI: 10.1021/ja311795d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311795d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6092-6099&author=E.+F.+Douglassauthor=C.+J.+Millerauthor=G.+Sparerauthor=H.+Shapiroauthor=D.+A.+Spiegel&title=A+comprehensive+mathematical+model+for+three-body+binding+equilibria&doi=10.1021%2Fja311795d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Mathematical Model for Three-Body Binding Equilibria</span></div><div class="casAuthors">Douglass, Eugene F.; Miller, Chad J.; Sparer, Gerson; Shapiro, Harold; Spiegel, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6092-6099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-component systems are often more complex than their two-component counterparts.  Although the reversible assocn. of three components in soln. is crit. for a vast array of chem. and biol. processes, no general phys. picture of such systems has emerged.  Here the authors have developed a general, comprehensive framework for understanding ternary complex equil., which relates directly to familiar concepts such as EC50 and IC50 from simpler (binary complex) equil.  Importantly, application of the authors' model to data from the published literature has enabled the authors to achieve new insights into complex systems ranging from coagulation to therapeutic dosing regimens for monoclonal antibodies.  The authors also provide an Excel spreadsheet to assist readers in both conceptualizing and applying the authors' models.  Overall, the anal. has the potential to render complex three-component systems-which have previously been characterized as "anal. intractable"-readily comprehensible to theoreticians and experimentalists alike.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4T36t0kgZ3LVg90H21EOLACvtfcHk0ljFzuXQt5Ox-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D&md5=44a6da2858f9813d7f1329c96a7085ac</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja311795d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311795d%26sid%3Dliteratum%253Aachs%26aulast%3DDouglass%26aufirst%3DE.%2BF.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DSparer%26aufirst%3DG.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DSpiegel%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520mathematical%2520model%2520for%2520three-body%2520binding%2520equilibria%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6092%26epage%3D6099%26doi%3D10.1021%2Fja311795d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Gadd, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">514</span><span class="NLM_x">â</span> <span class="NLM_lpage">521</span><span class="refDoi">Â DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lht6flSaTmwUg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Petzold, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ThomÃ¤, N. H.</span><span> </span><span class="NLM_article-title">Structural basis of lenalidomide-induced CK1Î± degradation by the CRL4(CRBN) ubiquitin ligase</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">532</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">â</span> <span class="NLM_lpage">130</span><span class="refDoi">Â DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature16979" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+basis+of+lenalidomide-induced+CK1%CE%B1+degradation+by+the+CRL4%28CRBN%29+ubiquitin+ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520basis%2520of%2520lenalidomide-induced%2520CK1%25CE%25B1%2520degradation%2520by%2520the%2520CRL4%2528CRBN%2529%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Angers, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacCoss, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moon, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, N.</span><span> </span><span class="NLM_article-title">Molecular architecture and assembly of the DDB1âCUL4A ubiquitin ligase machinery</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">443</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">â</span> <span class="NLM_lpage">593</span><span class="refDoi">Â DOI: 10.1038/nature05175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature05175" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=443&publication_year=2006&pages=590-593&author=S.+Angersauthor=T.+Liauthor=X.+Yiauthor=M.+J.+MacCossauthor=R.+T.+Moonauthor=N.+Zheng&title=Molecular+architecture+and+assembly+of+the+DDB1%E2%80%93CUL4A+ubiquitin+ligase+machinery&doi=10.1038%2Fnature05175"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature05175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05175%26sid%3Dliteratum%253Aachs%26aulast%3DAngers%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DYi%26aufirst%3DX.%26aulast%3DMacCoss%26aufirst%3DM.%2BJ.%26aulast%3DMoon%26aufirst%3DR.%2BT.%26aulast%3DZheng%26aufirst%3DN.%26atitle%3DMolecular%2520architecture%2520and%2520assembly%2520of%2520the%2520DDB1%25E2%2580%2593CUL4A%2520ubiquitin%2520ligase%2520machinery%26jtitle%3DNature%26date%3D2006%26volume%3D443%26spage%3D590%26epage%3D593%26doi%3D10.1038%2Fnature05175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="note"><p class="first last">A recent publication demonstrated for a VHL-PROTAC targeting a kinase that a wide range of linker lengths were tolerated but with an apparent lower limit cutoff of 12 linear non-H linker atoms, see:</p></div><div class="NLM_citation" id="cit26a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Crew, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serebrenik, Y. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamman, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferraro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neklesa, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+Von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit26b"><span><span class="NLM_label">(b) </span><div class="note"><p class="first last">While shorter linker lengths were not investigated during the course of this work, shorter linker lengths in degradation probes have been successfully previously employed against other protein targets, see:</p></div><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Salas, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McEachern, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Przybranowski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit26c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Robb, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contreras, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kour, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonawane, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murry, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natarajan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rana, S.</span><span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">7577</span><span class="NLM_x">â</span> <span class="NLM_lpage">7580</span><span class="refDoi">Â DOI: 10.1039/C7CC03879H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1039%2FC7CC03879H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2FC7CC03879H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26cR"><div class="casContent"><span class="casTitleNuber">26c</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0lhlgJrjvnlROw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit26c&amp;dbid=16384&amp;doi=10.1039%2FC7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC03879H%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2FC7CC03879H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Ruchelman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Man, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corral, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babusis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghaddam, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirley, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, G. W.</span><span> </span><span class="NLM_article-title">Isosteric analogs of lenalidomide and pomalidomide: Synthesis and biological activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">360</span><span class="NLM_x">â</span> <span class="NLM_lpage">365</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2012.10.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2012.10.071" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=360-365&author=A.+L.+Ruchelmanauthor=H.+Manauthor=W.+Zhangauthor=R.+Chenauthor=L.+Caponeauthor=J.+Kangauthor=A.+Partonauthor=L.+Corralauthor=P.+H.+Schaferauthor=D.+Babusisauthor=M.+F.+Moghaddamauthor=Y.+Tangauthor=M.+A.+Shirleyauthor=G.+W.+Muller&title=Isosteric+analogs+of+lenalidomide+and+pomalidomide%3A+Synthesis+and+biological+activity&doi=10.1016%2Fj.bmcl.2012.10.071"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.071%26sid%3Dliteratum%253Aachs%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DMan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DCapone%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DParton%26aufirst%3DA.%26aulast%3DCorral%26aufirst%3DL.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DBabusis%26aufirst%3DD.%26aulast%3DMoghaddam%26aufirst%3DM.%2BF.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DShirley%26aufirst%3DM.%2BA.%26aulast%3DMuller%26aufirst%3DG.%2BW.%26atitle%3DIsosteric%2520analogs%2520of%2520lenalidomide%2520and%2520pomalidomide%253A%2520Synthesis%2520and%2520biological%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D360%26epage%3D365%26doi%3D10.1016%2Fj.bmcl.2012.10.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Lohbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. K.</span><span> </span><span class="NLM_article-title">Practical synthesis of a phthalimide-based cereblon ligand to enable PROTAC development</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">5260</span><span class="NLM_x">â</span> <span class="NLM_lpage">5262</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2016.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27687673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5260-5262&author=J.+Lohbeckauthor=A.+K.+Miller&title=Practical+synthesis+of+a+phthalimide-based+cereblon+ligand+to+enable+PROTAC+development&doi=10.1016%2Fj.bmcl.2016.09.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Practical synthesis of a phthalimide-based Cereblon ligand to enable PROTAC development</span></div><div class="casAuthors">Lohbeck, Jasmin; Miller, Aubry K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5260-5262</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The use of small mols. to regulate cellular levels of specific proteins is poised to become a powerful technique in the coming years.  Crit. to the success of any project using such an approach will be the ability to synthesize libraries of candidate small mols. for testing in cellular systems.  Herein, the authors describe a practical synthesis of a phthalimide-based scaffold, which can be easily diversified to make Cereblon-targeting PROTACs.  The authors demonstrate the effectiveness of this approach by synthesizing a 'PROTAC toolbox' of four amines which can be coupled to inhibitors in a straightforward manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo01AK2Qjj97Vg90H21EOLACvtfcHk0lhlgJrjvnlROw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI&md5=864f4c671ef2c8d43191fc6a6e22b3b2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DLohbeck%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DA.%2BK.%26atitle%3DPractical%2520synthesis%2520of%2520a%2520phthalimide-based%2520cereblon%2520ligand%2520to%2520enable%2520PROTAC%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5260%26epage%3D5262%26doi%3D10.1016%2Fj.bmcl.2016.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Georgsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larhed, M.</span><span> </span><span class="NLM_article-title">Rapid palladium-catalyzed synthesis of esters from aryl halides utilizing Mo(CO)<sub>6</sub> as a solid carbon monoxide source</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">350</span><span class="NLM_x">â</span> <span class="NLM_lpage">352</span><span class="refDoi">Â DOI: 10.1021/cc0201086</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc0201086" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktlaiurk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=350-352&author=J.+Georgssonauthor=A.+Hallbergauthor=M.+Larhed&title=Rapid+palladium-catalyzed+synthesis+of+esters+from+aryl+halides+utilizing+Mo%28CO%296+as+a+solid+carbon+monoxide+source&doi=10.1021%2Fcc0201086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29aR"><div class="casContent"><span class="casTitleNuber">29a</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Palladium-Catalyzed Synthesis of Esters from Aryl Halides Utilizing Mo(CO)6 as a Solid Carbon Monoxide Source</span></div><div class="casAuthors">Georgsson, Jennie; Hallberg, Anders; Larhed, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">350-352</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of arom. esters R1CO2R2 (R1 = 3-BrC6H4, 2-MeC6H4, 4-MeOC6H4, 4-F3CC6H4, 1-naphthyl; R2 = Bu, Me3SiCH2CH2, PhCH2) was synthesized in moderate to high yields (52-89%) via palladium-catalyzed carbonylation of aryl halides R1X (X = Br, iodo) in the presence of excess of a primary alc. R2OH under microwave irradn. using Mo(CO)6 as a solid carbon monoxide source.  The reactions with sterically hindered tert-butanol as an alc. failed or gave low yields of the desired products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS1k1kMsuaA7Vg90H21EOLACvtfcHk0lhlgJrjvnlROw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktlaiurk%253D&md5=7ee6bd650c0c7cfd98b9fc83045e70d2</span></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=10.1021%2Fcc0201086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc0201086%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgsson%26aufirst%3DJ.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DLarhed%26aufirst%3DM.%26atitle%3DRapid%2520palladium-catalyzed%2520synthesis%2520of%2520esters%2520from%2520aryl%2520halides%2520utilizing%2520Mo%2528CO%25296%2520as%2520a%2520solid%2520carbon%2520monoxide%2520source%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2003%26volume%3D5%26spage%3D350%26epage%3D352%26doi%3D10.1021%2Fcc0201086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit29b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Georgsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SkÃ¶ld, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plouffe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindeberg, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botros, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larhed, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nyberg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallo-Payet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gogoll, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KarlÃ©n, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span> </span><span class="NLM_article-title">Angiotensin II pseudopeptides containing 1,3,5-trisubstituted benzene scaffolds with high AT2 receptor affinity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6620</span><span class="NLM_x">â</span> <span class="NLM_lpage">6631</span><span class="refDoi">Â DOI: 10.1021/jm050280z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050280z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6620-6631&author=J.+Georgssonauthor=C.+Sk%C3%B6ldauthor=B.+Plouffeauthor=G.+Lindebergauthor=M.+Botrosauthor=M.+Larhedauthor=F.+Nybergauthor=N.+Gallo-Payetauthor=A.+Gogollauthor=A.+Karl%C3%A9nauthor=A.+Hallberg&title=Angiotensin+II+pseudopeptides+containing+1%2C3%2C5-trisubstituted+benzene+scaffolds+with+high+AT2+receptor+affinity&doi=10.1021%2Fjm050280z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=10.1021%2Fjm050280z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050280z%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgsson%26aufirst%3DJ.%26aulast%3DSk%25C3%25B6ld%26aufirst%3DC.%26aulast%3DPlouffe%26aufirst%3DB.%26aulast%3DLindeberg%26aufirst%3DG.%26aulast%3DBotros%26aufirst%3DM.%26aulast%3DLarhed%26aufirst%3DM.%26aulast%3DNyberg%26aufirst%3DF.%26aulast%3DGallo-Payet%26aufirst%3DN.%26aulast%3DGogoll%26aufirst%3DA.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26aulast%3DHallberg%26aufirst%3DA.%26atitle%3DAngiotensin%2520II%2520pseudopeptides%2520containing%25201%252C3%252C5-trisubstituted%2520benzene%2520scaffolds%2520with%2520high%2520AT2%2520receptor%2520affinity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6620%26epage%3D6631%26doi%3D10.1021%2Fjm050280z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: Comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=K.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+Comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520Comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><div class="note"><p class="first last">Our FP-probe was generated using a Sulfo-Cy5 fluorophore rather than Cy5 but displayed similar <i>K</i><sub>i</sub> values for thalidomide and lenalidomide as:</p></div><span class="NLM_contrib-group">Fischer, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">BÃ¶hm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lydeard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadler, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavadini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serluca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acker, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraju, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tichkule, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schebesta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forrester, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassiepen, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hild, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckwith, R. E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ThomÃ¤, N. H.</span><span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">512</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="NLM_x">â</span> <span class="NLM_lpage">53</span><span class="refDoi">Â DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0li6bt_QFXXnag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="note"><p class="first last"><i>K</i><sub>i</sub> values were calculated from the geometric mean IC<sub>50</sub> and the FP-assay probe affinity using the methods described by:</p></div><div class="NLM_citation" id="cit32a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Huang, X.</span><span> </span><span class="NLM_article-title">Fluorescence polarization competition assay: The range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">â</span> <span class="NLM_lpage">38</span><span class="refDoi">Â DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=12854996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&author=X.+Huang&title=Fluorescence+polarization+competition+assay%3A+The+range+of+resolvable+inhibitor+potency+is+limited+by+the+affinity+of+the+fluorescent+ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32aR"><div class="casContent"><span class="casTitleNuber">32a</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span></div><div class="casAuthors">Huang, Xinyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">For the development of fluorescence polarization (FP) competition assays, there is a widespread belief that tight-binding fluorescent ligands should be avoided to identify inhibitors of low or intermediate potency in the screening of small-mol. compd. libraries.  It is demonstrated herein that this statement is a misconception; in fact, the higher the affinity of the fluorescent ligand, the wider the range of inhibitor potency that can be resolved.  An approx. est. for the low end of inhibitor Ki values that can be resolved is the Kd value of the fluorescent ligand.  Because FP competition assays are typically conducted under nonstoichiometric titrn. conditions, it is suggested that a fluorescent ligand of highest affinity that also has an adequate quantum yield to satisfy such conditions be selected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUYlJzQaNj7Vg90H21EOLACvtfcHk0li6bt_QFXXnag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D&md5=7c706c96b6e534486f6780c93a0d441b</span></div><a href="/servlet/linkout?suffix=cit32a&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DFluorescence%2520polarization%2520competition%2520assay%253A%2520The%2520range%2520of%2520resolvable%2520inhibitor%2520potency%2520is%2520limited%2520by%2520the%2520affinity%2520of%2520the%2520fluorescent%2520ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit32b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Munson, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodbard, D.</span><span> </span><span class="NLM_article-title">An exact correction to the â³Cheng-Prusoffâ³ correction</span> <span class="citation_source-journal">J. Recept. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">â</span> <span class="NLM_lpage">456</span><span class="refDoi">Â DOI: 10.3109/10799898809049010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.3109%2F10799898809049010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=3385692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADyaL1cXitFGks7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1988&pages=533-456&author=P.+J.+Munsonauthor=D.+Rodbard&title=An+exact+correction+to+the+%E2%80%B3Cheng-Prusoff%E2%80%B3+correction&doi=10.3109%2F10799898809049010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32bR"><div class="casContent"><span class="casTitleNuber">32b</span><div class="casTitle"><span class="NLM_cas:atitle">An exact correction to the "Cheng-Prusoff" correction</span></div><div class="casAuthors">Munson, Peter J.; Rodbard, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptor Research</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1-4</span>),
    <span class="NLM_cas:pages">533-46</span>CODEN:
                <span class="NLM_cas:coden">JRERDM</span>;
        ISSN:<span class="NLM_cas:issn">0197-5110</span>.
    </div><div class="casAbstract">The Cheng-Prusoff correction for obtaining the inhibition const. Ki from the graphical midpoint or 50% ED (ED50) of a displacement or inhibition curve may lead to 3-6-fold or larger errors, esp. when applied to drug-receptor binding studies.  A simple exact correction rule based on total concns. for the reactants was derived to relate the Ki to the ED50 concn. of the inhibitor.  The exact correction is needed esp. when the ratio of bound to free ligand (y0) or the total concn. of labeled and unlabeled ligands was large or when the ED50 value was low.  The use of the Cheng-Prusoff correction is appropriate only when exptl. errors are extremely small and when y0 â¤0.1 and the ED50 of the inhibitor is greater than the ED50 for the self-competition of the labeled ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsuJtjVMXcI7Vg90H21EOLACvtfcHk0lh72uLv9sfAUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXitFGks7g%253D&md5=59b3521f1967f6d7ac0655772c7660ec</span></div><a href="/servlet/linkout?suffix=cit32b&amp;dbid=16384&amp;doi=10.3109%2F10799898809049010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10799898809049010%26sid%3Dliteratum%253Aachs%26aulast%3DMunson%26aufirst%3DP.%2BJ.%26aulast%3DRodbard%26aufirst%3DD.%26atitle%3DAn%2520exact%2520correction%2520to%2520the%2520%25E2%2580%25B3Cheng-Prusoff%25E2%2580%25B3%2520correction%26jtitle%3DJ.%2520Recept.%2520Res.%26date%3D1988%26volume%3D8%26spage%3D533%26epage%3D456%26doi%3D10.3109%2F10799898809049010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span>Wes: Simple Westerns Just Got Even Simpler; <span class="NLM_publisher-name">Proteinsimple</span>: <span class="NLM_publisher-loc">San Jose, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">; </span><a href="http://www.proteinsimple.com/wes.html" class="extLink">https://www.proteinsimple.com/wes.html</a> (accessed August 28, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Wes%3A+Simple+Westerns+Just+Got+Even+Simpler%3B+Proteinsimple%3A+San+Jose%2C+CA%2C+2017%3B+https%3A%2F%2Fwww.proteinsimple.com%2Fwes.html+%28accessed+August+28%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DWes%253A%2520Simple%2520Westerns%2520Just%2520Got%2520Even%2520Simpler%26pub%3DProteinsimple%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="note"><p class="first last">For the origin of our SK-OV-3 ovarian carcinoma cell line, see the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Erales, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coffino, P.</span><span> </span><span class="NLM_article-title">Ubiquitin-independent proteasomal degradation</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1843</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">â</span> <span class="NLM_lpage">221</span><span class="refDoi">Â DOI: 10.1016/j.bbamcr.2013.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bbamcr.2013.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=23684952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12qsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=216-221&author=J.+Eralesauthor=P.+Coffino&title=Ubiquitin-independent+proteasomal+degradation&doi=10.1016%2Fj.bbamcr.2013.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-independent proteasomal degradation</span></div><div class="casAuthors">Erales, Jenny; Coffino, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">216-221</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Most proteasome substrates are marked for degrdn. by ubiquitin conjugation, but some are targeted by other means.  The properties of these exceptional cases provide insights into the general requirements for proteasomal degrdn.  Here the focus is on three ubiquitin-independent substrates that have been the subject of detailed study.  These are Rpn4, a transcriptional regulator of proteasome homeostasis, thymidylate synthase, an enzyme required for prodn. of DNA precursors and ornithine decarboxylase, the initial enzyme committed to polyamine biosynthesis.  It can be inferred from these cases that proteasome assocn. and the presence of an unstructured region are the sole prerequisites for degrdn.  Based on that inference, artificial substrates have been designed to test the proteasome's capacity for substrate processing and its limitations.  Ubiquitin-independent substrates may in some cases be a remnant of the pre-ubiquitome world, but in other cases could provide optimized regulatory solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9wxiAlcl4CbVg90H21EOLACvtfcHk0lh72uLv9sfAUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12qsb4%253D&md5=605cbe83ddb7be7bb9cbeae02d1f57dd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2013.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2013.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DErales%26aufirst%3DJ.%26aulast%3DCoffino%26aufirst%3DP.%26atitle%3DUbiquitin-independent%2520proteasomal%2520degradation%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D216%26epage%3D221%26doi%3D10.1016%2Fj.bbamcr.2013.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellerschmied, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salami, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaime-Figueroa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">807</span><span class="NLM_x">â</span> <span class="NLM_lpage">810</span><span class="refDoi">Â DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0lh72uLv9sfAUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Mateus, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wayne, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almqvist, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Axelsson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seashore-Ludlow, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treyer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LundbÃ¤ck, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Prediction of intracellular exposure bridges the gap between target- and cell-based drug discovery</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">E6231</span><span class="NLM_x">â</span> <span class="NLM_lpage">E6239</span><span class="refDoi">Â DOI: 10.1073/pnas.1701848114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1073%2Fpnas.1701848114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28701380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E6231-E6239&author=A.+Mateusauthor=L.+J.+Gordonauthor=G.+J.+Wayneauthor=H.+Almqvistauthor=H.+Axelssonauthor=B.+Seashore-Ludlowauthor=A.+Treyerauthor=P.+Matssonauthor=T.+Lundb%C3%A4ckauthor=A.+Westauthor=M.+M.+Hannauthor=P.+Artursson&title=Prediction+of+intracellular+exposure+bridges+the+gap+between+target-+and+cell-based+drug+discovery&doi=10.1073%2Fpnas.1701848114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of intracellular exposure bridges the gap between target- and cell-based drug discovery</span></div><div class="casAuthors">Mateus, Andre; Gordon, Laurie J.; Wayne, Gareth J.; Almqvist, Helena; Axelsson, Hanna; Seashore-Ludlow, Brinton; Treyer, Andrea; Matsson, Paer; Lundbaeck, Thomas; West, Andy; Hann, Michael M.; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">E6231-E6239</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inadequate target exposure is a major cause of high attrition in drug discovery.  Here, we show that a label-free method for quantifying the intracellular bioavailability (Fic) of drug mols. predicts drug access to intracellular targets and hence, pharmacol. effect.  We detd. Fic in multiple cellular assays and cell types representing different targets from a no. of therapeutic areas, including cancer, inflammation, and dementia.  Both cytosolic targets and targets localized in subcellular compartments were investigated.  Fic gives insights on membrane-permeable compds. in terms of cellular potency and intracellular target engagement, compared with biochem. potency measurements alone.  Knowledge of the amt. of drug that is locally available to bind intracellular targets provides a powerful tool for compd. selection in early drug discovery. localization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo1IEI-YIPNbVg90H21EOLACvtfcHk0lgA0OCnCsvjBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK&md5=ace62bf5bda6cfd214bac4b7c5bf9cf1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1701848114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1701848114%26sid%3Dliteratum%253Aachs%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26aulast%3DTreyer%26aufirst%3DA.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DWest%26aufirst%3DA.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DPrediction%2520of%2520intracellular%2520exposure%2520bridges%2520the%2520gap%2520between%2520target-%2520and%2520cell-based%2520drug%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE6231%26epage%3DE6239%26doi%3D10.1073%2Fpnas.1701848114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span> </span><span class="NLM_article-title">How big is too big for cell permeability?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1662</span><span class="NLM_x">â</span> <span class="NLM_lpage">1664</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00237</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00237" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCjtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1662-1664&author=P.+Matssonauthor=J.+Kihlberg&title=How+big+is+too+big+for+cell+permeability%3F&doi=10.1021%2Facs.jmedchem.7b00237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38aR"><div class="casContent"><span class="casTitleNuber">38a</span><div class="casTitle"><span class="NLM_cas:atitle">How Big Is Too Big for Cell Permeability?</span></div><div class="casAuthors">Matsson, Paer; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1662-1664</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Understanding how to design cell permeable ligands for intracellular targets that have difficult binding sites, such as protein-protein interactions, would open vast opportunities for drug discovery.  Interestingly, libraries of cyclic peptides displayed a steep drop-off in membrane permeability at mol. wts. above 1000 Da and it appears likely that this cutoff constitutes an upper size limit also for more drug-like compds.  However, chem. space from 500 to 1000 Da remains virtually unexplored and represents a vast opportunity for those prepd. to venture into new territories of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXxiuvHQSnJLVg90H21EOLACvtfcHk0lgA0OCnCsvjBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCjtL8%253D&md5=563efb3b4603c33c4a976e88b1c7b514</span></div><a href="/servlet/linkout?suffix=cit38a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00237%26sid%3Dliteratum%253Aachs%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520big%2520is%2520too%2520big%2520for%2520cell%2520permeability%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1662%26epage%3D1664%26doi%3D10.1021%2Facs.jmedchem.7b00237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit38b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Pye, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwochert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etienne, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limberakis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furukawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathiowetz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liras, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lokey, R. S.</span><span> </span><span class="NLM_article-title">Nonclassical size dependence of permeation defines bounds for passive adsorption of large drug molecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1665</span><span class="NLM_x">â</span> <span class="NLM_lpage">1672</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslajtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1665-1672&author=C.+R.+Pyeauthor=W.+M.+Hewittauthor=J.+Schwochertauthor=T.+D.+Haddadauthor=C.+E.+Townsendauthor=L.+Etienneauthor=Y.+Laoauthor=C.+Limberakisauthor=A.+Furukawaauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=R.+S.+Lokey&title=Nonclassical+size+dependence+of+permeation+defines+bounds+for+passive+adsorption+of+large+drug+molecules&doi=10.1021%2Facs.jmedchem.6b01483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38bR"><div class="casContent"><span class="casTitleNuber">38b</span><div class="casTitle"><span class="NLM_cas:atitle">Nonclassical Size Dependence of Permeation Defines Bounds for Passive Adsorption of Large Drug Molecules</span></div><div class="casAuthors">Pye, Cameron R.; Hewitt, William M.; Schwochert, Joshua; Haddad, Terra D.; Townsend, Chad E.; Etienne, Lyns; Lao, Yongtong; Limberakis, Chris; Furukawa, Akihiro; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1665-1672</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a growing no. of intracellular protein-protein interactions (PPI's) with potential therapeutic importance that demand larger ligands than those that neatly fit into most common criteria for "drug-likeness".  Macrocyclic peptides pro-vide the large surface area required to inhibit PPIs and, in principle, avoid many of the pitfalls assocd. with peptides, including low stability and membrane permeability.  However, the design of passively cell-permeable mols. in this space remains a great challenge due to the poorly understood roles of mol. size and lipophilicity in detg. passive membrane permeability above MW â¼500.  Using split-pool combinatorial synthesis, we constructed a library of cyclic, per-N-methlyated peptides spanning a wide range of calcd. lipohilicities (0 < AlogP < 8) and mol. wts. (â¼800 Da < MW < â¼1200 Da).  Assaying mixts. of compds. using the Parallel Artificial Membrane Assay (PAMPA) revealed a steep drop-off in apparent passive permeability with increasing size and a bilinear relationship with lipophilicity.  We obsd. a size dependence of permeability that is in stark disagreement with current permeation models.  The lipophilicities required to overcome this size penalty led to poor aq. soly., effectively bounding the chem. space of likely passively permeable mols.  This understanding provides a new set of design criteria for achieving permeability in larger mol. size regimes and suggests an operational cut-off beyond which passive permeability is constrained by a sharply increasing penalty on permeation through the system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXD-PeEQUtu7Vg90H21EOLACvtfcHk0lhxe6acHYvOcw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslajtw%253D%253D&md5=7b51a3c8fe24560c9445d8d32bf3b70f</span></div><a href="/servlet/linkout?suffix=cit38b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01483%26sid%3Dliteratum%253Aachs%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DHaddad%26aufirst%3DT.%2BD.%26aulast%3DTownsend%26aufirst%3DC.%2BE.%26aulast%3DEtienne%26aufirst%3DL.%26aulast%3DLao%26aufirst%3DY.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DFurukawa%26aufirst%3DA.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DNonclassical%2520size%2520dependence%2520of%2520permeation%2520defines%2520bounds%2520for%2520passive%2520adsorption%2520of%2520large%2520drug%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1665%26epage%3D1672%26doi%3D10.1021%2Facs.jmedchem.6b01483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KeserÃ¼, G. M.</span><span> </span><span class="NLM_article-title">Finding the sweet spot: The role of nature and nurture in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">â</span> <span class="NLM_lpage">365</span><span class="refDoi">Â DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&author=M.+M.+Hannauthor=G.+M.+Keser%C3%BC&title=Finding+the+sweet+spot%3A+The+role+of+nature+and+nurture+in+medicinal+chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0lhxe6acHYvOcw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520sweet%2520spot%253A%2520The%2520role%2520of%2520nature%2520and%2520nurture%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Palm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ungell, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strandlund, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beigi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Evaluation of dynamic polar molecular surface area as predictor of drug absorption: Comparison with other computational and experimental predictors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">5382</span><span class="NLM_x">â</span> <span class="NLM_lpage">5392</span><span class="refDoi">Â DOI: 10.1021/jm980313t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980313t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADyaK1cXns1yltrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=5382-5392&author=K.+Palmauthor=K.+Luthmanauthor=A.+L.+Ungellauthor=G.+Strandlundauthor=F.+Beigiauthor=P.+Lundahlauthor=P.+Artursson&title=Evaluation+of+dynamic+polar+molecular+surface+area+as+predictor+of+drug+absorption%3A+Comparison+with+other+computational+and+experimental+predictors&doi=10.1021%2Fjm980313t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Dynamic Polar Molecular Surface Area as Predictor of Drug Absorption: Comparison with Other Computational and Experimental Predictors</span></div><div class="casAuthors">Palm, Katrin; Luthman, Kristina; Ungell, Anna-Lena; Strandlund, Gert; Beigi, Farideh; Lundahl, Per; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">5382-5392</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The relationship between various mol. descriptors and transport of drugs across the intestinal epithelium was evaluated.  The monolayer permeability (Pc) of human intestinal Caco-2 cells to a series of nine Î²-receptor-blocking agents was investigated in vitro.  The dynamic polar mol. surface area (PSAd) of the compds. was calcd. from all low-energy conformations identified in mol. mechanics calcns. in vacuum and in simulated chloroform and water environments.  For most of the investigated drugs, the effects of the different environments on PSAd were small.  The exception was H 216/44, which is a large flexible compd. contg. several functional groups capable of hydrogen bonding (PSAd,chloroform = 70.8 Ã2 and PSAd,water = 116.6 Ã2).  The relationship between Pc and PSAd was stronger than those between Pc and the calcd. octanol/water distribution coeffs. (log Dcalc) or the exptl. detd. immobilized liposome chromatog. (ILC) retention.  Pc values for two new practolol analogs and H 216/44 were predicted from the structure-permeability relationships of a subset of the nine compds. and compared with exptl. values.  The Pc values of the two practolol analogs were predicted well from both PSAd calcns. and ILC retention studies.  The Pc value of H 216/44 was reasonably well-predicted only from the PSAd of conformations preferred in vacuum and in water.  The other descriptors overestimated the Pc of H 216/44 100-500-fold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolKDCQqqUS9rVg90H21EOLACvtfcHk0lhxe6acHYvOcw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1yltrc%253D&md5=83820600b8fe6ffde348c272d45c628a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm980313t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980313t%26sid%3Dliteratum%253Aachs%26aulast%3DPalm%26aufirst%3DK.%26aulast%3DLuthman%26aufirst%3DK.%26aulast%3DUngell%26aufirst%3DA.%2BL.%26aulast%3DStrandlund%26aufirst%3DG.%26aulast%3DBeigi%26aufirst%3DF.%26aulast%3DLundahl%26aufirst%3DP.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DEvaluation%2520of%2520dynamic%2520polar%2520molecular%2520surface%2520area%2520as%2520predictor%2520of%2520drug%2520absorption%253A%2520Comparison%2520with%2520other%2520computational%2520and%2520experimental%2520predictors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D5382%26epage%3D5392%26doi%3D10.1021%2Fjm980313t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Gillis, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastman, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donnelly, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meanwell, N. A.</span><span> </span><span class="NLM_article-title">Applications of fluorine in medicinal chemistry</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8315</span><span class="NLM_x">â</span> <span class="NLM_lpage">8359</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8315-8359&author=E.+P.+Gillisauthor=K.+J.+Eastmanauthor=M.+D.+Hillauthor=D.+J.+Donnellyauthor=N.+A.+Meanwell&title=Applications+of+fluorine+in+medicinal+chemistry&doi=10.1021%2Facs.jmedchem.5b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of Fluorine in Medicinal Chemistry</span></div><div class="casAuthors">Gillis, Eric P.; Eastman, Kyle J.; Hill, Matthew D.; Donnelly, David J.; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8315-8359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with meta-anal. The role of fluorine in drug design and development is expanding rapidly as we learn more about the unique properties assocd. with this unusual element and how to deploy it with greater sophistication.  The judicious introduction of fluorine into a mol. can productively influence conformation, pKa, intrinsic potency, membrane permeability, metabolic pathways, and pharmacokinetic properties.  In addn., 18F has been established as a useful positron emitting isotope for use with in vivo imaging technol. that potentially has extensive application in drug discovery and development, often limited only by convenient synthetic accessibility to labeled compds.  The wide ranging applications of fluorine in drug design are providing a strong stimulus for the development of new synthetic methodologies that allow more facile access to a wide range of fluorinated compds.  In this review, we provide an update on the effects of the strategic incorporation of fluorine in drug mols. and applications in positron emission tomog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8fUUtkPB4L7Vg90H21EOLACvtfcHk0lg46yMVrZOUkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK&md5=9995829a94a8c0b8d9fb0d21bdfd5a1d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00258%26sid%3Dliteratum%253Aachs%26aulast%3DGillis%26aufirst%3DE.%2BP.%26aulast%3DEastman%26aufirst%3DK.%2BJ.%26aulast%3DHill%26aufirst%3DM.%2BD.%26aulast%3DDonnelly%26aufirst%3DD.%2BJ.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DApplications%2520of%2520fluorine%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8315%26epage%3D8359%26doi%3D10.1021%2Facs.jmedchem.5b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">George, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moody, C. J.</span><span> </span><span class="NLM_article-title">Synthesis of iodopyridone</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">8209</span><span class="NLM_x">â</span> <span class="NLM_lpage">8215</span><span class="refDoi">Â DOI: 10.1016/j.tet.2013.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.tet.2013.07.033" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=8209-8215&author=I.+R.+Georgeauthor=W.+Lewisauthor=C.+J.+Moody&title=Synthesis+of+iodopyridone&doi=10.1016%2Fj.tet.2013.07.033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2013.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2013.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DGeorge%26aufirst%3DI.%2BR.%26aulast%3DLewis%26aufirst%3DW.%26aulast%3DMoody%26aufirst%3DC.%2BJ.%26atitle%3DSynthesis%2520of%2520iodopyridone%26jtitle%3DTetrahedron%26date%3D2013%26volume%3D69%26spage%3D8209%26epage%3D8215%26doi%3D10.1016%2Fj.tet.2013.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><div class="note"><p class="first last">In mouse fibroblasts, pirin has a measured inherent protein half-life of 64 h:</p></div><span class="NLM_contrib-group">SchwanhÃ¤usser, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Busse, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmar, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuchhardt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selbach, M.</span><span> </span><span class="NLM_article-title">Global quantification of mammalian gene expression control</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">473</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">â</span> <span class="NLM_lpage">342</span><span class="refDoi">Â DOI: 10.1038/nature10098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature10098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=21593866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BC3Mrjtlaguw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2011&pages=337-342&author=B.+Schwanh%C3%A4usserauthor=D.+Busseauthor=N.+Liauthor=G.+Dittmarauthor=J.+Schuchhardtauthor=J.+Wolfauthor=W.+Chenauthor=M.+Selbach&title=Global+quantification+of+mammalian+gene+expression+control&doi=10.1038%2Fnature10098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Global quantification of mammalian gene expression control</span></div><div class="casAuthors">Schwanhausser Bjorn; Busse Dorothea; Li Na; Dittmar Gunnar; Schuchhardt Johannes; Wolf Jana; Chen Wei; Selbach Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">7347</span>),
    <span class="NLM_cas:pages">337-42</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Gene expression is a multistep process that involves the transcription, translation and turnover of messenger RNAs and proteins.  Although it is one of the most fundamental processes of life, the entire cascade has never been quantified on a genome-wide scale.  Here we simultaneously measured absolute mRNA and protein abundance and turnover by parallel metabolic pulse labelling for more than 5,000 genes in mammalian cells.  Whereas mRNA and protein levels correlated better than previously thought, corresponding half-lives showed no correlation.  Using a quantitative model we have obtained the first genome-scale prediction of synthesis rates of mRNAs and proteins.  We find that the cellular abundance of proteins is predominantly controlled at the level of translation.  Genes with similar combinations of mRNA and protein stability shared functional properties, indicating that half-lives evolved under energetic and dynamic constraints.  Quantitative information about all stages of gene expression provides a rich resource and helps to provide a greater understanding of the underlying design principles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWkDdhCnw1Wty3lHz94U-GfW6udTcc2eYVE7jURpUZFLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mrjtlaguw%253D%253D&md5=aa6204aff902c51bb1323966d0090c64</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnature10098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10098%26sid%3Dliteratum%253Aachs%26aulast%3DSchwanh%25C3%25A4usser%26aufirst%3DB.%26aulast%3DBusse%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DDittmar%26aufirst%3DG.%26aulast%3DSchuchhardt%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DSelbach%26aufirst%3DM.%26atitle%3DGlobal%2520quantification%2520of%2520mammalian%2520gene%2520expression%2520control%26jtitle%3DNature%26date%3D2011%26volume%3D473%26spage%3D337%26epage%3D342%26doi%3D10.1038%2Fnature10098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="note"><p class="first last">The stability of the thalidomide derived probes in DMSO stock appears dependent on the water content of the DMSO. We recommend fresh DMSO stocks are prepared from solid prior to each use. For a discussion of thalidomide stability, see:</p></div><div class="NLM_citation" id="cit44a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Lepper, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scripture, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figg, W. D.</span><span> </span><span class="NLM_article-title">Thalidomide metabolism and hydrolysis: mechanisms and implications</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">â</span> <span class="NLM_lpage">685</span><span class="refDoi">Â DOI: 10.2174/138920006778017777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.2174%2F138920006778017777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=16918319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvV2mtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=677-685&author=E.+R.+Lepperauthor=N.+F.+Smithauthor=M.+C.+Coxauthor=C.+D.+Scriptureauthor=W.+D.+Figg&title=Thalidomide+metabolism+and+hydrolysis%3A+mechanisms+and+implications&doi=10.2174%2F138920006778017777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44aR"><div class="casContent"><span class="casTitleNuber">44a</span><div class="casTitle"><span class="NLM_cas:atitle">Thalidomide metabolism and hydrolysis: mechanisms and implications</span></div><div class="casAuthors">Lepper, Erin R.; Smith, Nicola F.; Cox, Michael C.; Scripture, Charity D.; Figg, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">677-685</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite its controversial past, thalidomide is currently under investigation for the treatment of several disease types, ranging from inflammatory conditions to cancer.  The mechanism of action of thalidomide is complex and not yet fully understood, but there is some evidence to suggest that metab. may play a role.  Consequently, there has been a considerable effort to characterize the metab. of thalidomide in recent years.  Thalidomide undergoes biotransformation by non-enzymic hydrolysis and enzyme-mediated hydroxylation to form a multitude of metabolites.  Metabolite identification and reaction phenotyping studies have been performed and will be discussed in this review in addn. to interspecies differences in thalidomide metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-PXbzjhKAALVg90H21EOLACvtfcHk0lg46yMVrZOUkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvV2mtLc%253D&md5=09e2dcde069bccdadbe4a9707818e4e0</span></div><a href="/servlet/linkout?suffix=cit44a&amp;dbid=16384&amp;doi=10.2174%2F138920006778017777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920006778017777%26sid%3Dliteratum%253Aachs%26aulast%3DLepper%26aufirst%3DE.%2BR.%26aulast%3DSmith%26aufirst%3DN.%2BF.%26aulast%3DCox%26aufirst%3DM.%2BC.%26aulast%3DScripture%26aufirst%3DC.%2BD.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26atitle%3DThalidomide%2520metabolism%2520and%2520hydrolysis%253A%2520mechanisms%2520and%2520implications%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2006%26volume%3D7%26spage%3D677%26epage%3D685%26doi%3D10.2174%2F138920006778017777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit44b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Reist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrupt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francotte, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, B.</span><span> </span><span class="NLM_article-title">Chiral inversion and hydrolysis of thalidomide: mechanisms and catalysis by bases and serum albumin, and chiral stability of teratogenic metabolites</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1521</span><span class="NLM_x">â</span> <span class="NLM_lpage">1528</span><span class="refDoi">Â DOI: 10.1021/tx9801817</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx9801817" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADyaK1cXntFens7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=1521-1528&author=M.+Reistauthor=P.+Carruptauthor=E.+Francotteauthor=B.+Testa&title=Chiral+inversion+and+hydrolysis+of+thalidomide%3A+mechanisms+and+catalysis+by+bases+and+serum+albumin%2C+and+chiral+stability+of+teratogenic+metabolites&doi=10.1021%2Ftx9801817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44bR"><div class="casContent"><span class="casTitleNuber">44b</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral Inversion and Hydrolysis of Thalidomide: Mechanisms and Catalysis by Bases and Serum Albumin, and Chiral Stability of Teratogenic Metabolites</span></div><div class="casAuthors">Reist, Marianne; Carrupt, Pierre-Alain; Francotte, Eric; Testa, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1521-1528</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The chiral inversion and hydrolysis of thalidomide and the catalysis by bases and human serum albumin were investigated by using a stereoselective HPLC assay.  Chiral inversion was catalyzed by albumin, hydroxyl ions, phosphate, and amino acids.  Basic amino acids (Arg and Lys) had a superior potency in catalyzing chiral inversion compared to acid and neutral ones.  The chiral inversion of thalidomide is thus subject to specific and general base catalysis, and it is suggested that the ability of HSA to catalyze the reaction is due to the basic groups of the amino acids Arg and Lys and not to a single catalytic site on the macromol.  The hydrolysis of thalidomide was also base-catalyzed.  However, albumin had no effect on hydrolysis, and there was no difference between the catalytic potencies of acidic, neutral, and basic amino acids.  This may be explained by different reaction mechanisms of the chiral inversion and hydrolysis of thalidomide.  Chiral inversion is deduced to occur by electrophilic substitution involving specific and general base catalysis, whereas hydrolysis is thought to occur by nucleophilic substitution involving specific and general base as well as nucleophilic catalysis.  As nucleophilic attack is sensitive to steric properties of the catalyst, steric hindrance might be the reason albumin is not able to catalyze hydrolysis.  1H NMR expts. revealed that the three teratogenic metabolites of thalidomide, in sharp contrast to the drug itself, had complete chiral stability.  This leads to the speculation that, were some enantioselectivity to exist in the teratogenicity of thalidomide, it could result from fast hydrolysis to chirally stable teratogenic metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfmXgvPgsp8rVg90H21EOLACvtfcHk0lg46yMVrZOUkg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFens7s%253D&md5=1952fa00825e6791eeb8388419f3cc09</span></div><a href="/servlet/linkout?suffix=cit44b&amp;dbid=16384&amp;doi=10.1021%2Ftx9801817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx9801817%26sid%3Dliteratum%253Aachs%26aulast%3DReist%26aufirst%3DM.%26aulast%3DCarrupt%26aufirst%3DP.%26aulast%3DFrancotte%26aufirst%3DE.%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DChiral%2520inversion%2520and%2520hydrolysis%2520of%2520thalidomide%253A%2520mechanisms%2520and%2520catalysis%2520by%2520bases%2520and%2520serum%2520albumin%252C%2520and%2520chiral%2520stability%2520of%2520teratogenic%2520metabolites%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1998%26volume%3D11%26spage%3D1521%26epage%3D1528%26doi%3D10.1021%2Ftx9801817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Goodwin, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conradi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, N. F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, P. S.</span><span> </span><span class="NLM_article-title">Physicochemical determinants of passive membrane permeability: Role of solute hydrogen-bonding potential and volume</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">3721</span><span class="NLM_x">â</span> <span class="NLM_lpage">3729</span><span class="refDoi">Â DOI: 10.1021/jm010253i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010253i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFOmtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3721-3729&author=J.+T.+Goodwinauthor=R.+A.+Conradiauthor=N.+F.+H.+Hoauthor=P.+S.+Burton&title=Physicochemical+determinants+of+passive+membrane+permeability%3A+Role+of+solute+hydrogen-bonding+potential+and+volume&doi=10.1021%2Fjm010253i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical determinants of passive membrane permeability: role of solute hydrogen-bonding potential and volume</span></div><div class="casAuthors">Goodwin, Jay T.; Conradi, Robert A.; Ho, Norman F. H.; Burton, Philip S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3721-3729</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The relationship of solute structure with cellular permeability was probed.  Two series of dipeptide mimetics consisting of glycine, alanine, valine, leucine, phenylalanine, and cyclohexylalanine with amino acids in the D-configuration were prepd.  Partition coeffs. for the peptide mimetics were obtained in the octanol/water (log Poctanol/water), hydrocarbon/octanol (Î log P), and heptane/ethylene glycol (log Pheptane/glycol) systems in order to explore the contributions of solute vol., or surface area, and hydrogen-bond potential to the permeability of the solutes.  Permeability coeffs. were obtained in Caco-2 cell monolayers as a model of the human intestinal mucosa.  The results were interpreted in terms of a partition/diffusion model for solute transport where membrane partitioning into the permeability-limiting membrane microdomain is estd. from the solvent partition coeffs.  Neither log Poctanol/water nor Î log P alone correlated with cellular permeability for all the solutes.  In contrast, log Pheptane/glycol gave a qual. better correlation.  With regard to solute properties, log Poctanol/water is predominantly a measure of solute vol., or surface area, and hydrogen-bond acceptor potential, while Î log P is principally a measure of hydrogen-bond donor strength.  Log Pheptane/glycol contains contributions from all these solute properties.  The results demonstrate that both hydrogen-bond potential and vol. of the solutes contribute to permeability and suggests that the nature of the permeability-limiting microenvironment within the cell depends on the properties of a specific solute.  Collectively, these findings support the conclusion that a general model of permeability will require consideration of a no. of different solute structural properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGociCiZQGkcmLVg90H21EOLACvtfcHk0lhtEs86TZExlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFOmtrs%253D&md5=b05b1ee611d18cd72ce5652d7f8aa350</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm010253i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010253i%26sid%3Dliteratum%253Aachs%26aulast%3DGoodwin%26aufirst%3DJ.%2BT.%26aulast%3DConradi%26aufirst%3DR.%2BA.%26aulast%3DHo%26aufirst%3DN.%2BF.%2BH.%26aulast%3DBurton%26aufirst%3DP.%2BS.%26atitle%3DPhysicochemical%2520determinants%2520of%2520passive%2520membrane%2520permeability%253A%2520Role%2520of%2520solute%2520hydrogen-bonding%2520potential%2520and%2520volume%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3721%26epage%3D3729%26doi%3D10.1021%2Fjm010253i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Evans, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span> </span><span class="NLM_article-title">Targeting secondary protein complexes in drug discovery: studying the druggability and chemical biology of the HSP70/BAG1 complex</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5167</span><span class="NLM_x">â</span> <span class="NLM_lpage">5170</span><span class="refDoi">Â DOI: 10.1039/C7CC01376K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1039%2FC7CC01376K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28439591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=5167-5170&author=L.+E.+Evansauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Targeting+secondary+protein+complexes+in+drug+discovery%3A+studying+the+druggability+and+chemical+biology+of+the+HSP70%2FBAG1+complex&doi=10.1039%2FC7CC01376K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting secondary protein complexes in drug discovery: studying the druggability and chemical biology of the HSP70/BAG1 complex</span></div><div class="casAuthors">Evans, Lindsay E.; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5167-5170</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Proteins typically carry out their biol. functions as multi-protein complexes, which can significantly affect the affinity of small-mol. inhibitors.  HSP70 is an important target in oncol., so to study its chem. biol. and the drug discovery potential of the HSP70/BAG1 complex, we designed a high-affinity non-nucleotide fluorescence polarization probe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwTl7NEh65w7Vg90H21EOLACvtfcHk0lhtEs86TZExlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D&md5=f72aa2f8aba028100f2eb1fd45bf336e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2FC7CC01376K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC01376K%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DTargeting%2520secondary%2520protein%2520complexes%2520in%2520drug%2520discovery%253A%2520studying%2520the%2520druggability%2520and%2520chemical%2520biology%2520of%2520the%2520HSP70%252FBAG1%2520complex%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D5167%26epage%3D5170%26doi%3D10.1039%2FC7CC01376K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Youdim, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avdeef, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbott, N. J.</span><span> </span><span class="NLM_article-title">In vitro trans-monolayer permeability calculations: Often forgotten assumptions</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">997</span><span class="NLM_x">â</span> <span class="NLM_lpage">1003</span><span class="refDoi">Â DOI: 10.1016/S1359-6446(03)02873-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2FS1359-6446%2803%2902873-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=14643163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlCltbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=997-1003&author=K.+A.+Youdimauthor=A.+Avdeefauthor=N.+J.+Abbott&title=In+vitro+trans-monolayer+permeability+calculations%3A+Often+forgotten+assumptions&doi=10.1016%2FS1359-6446%2803%2902873-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47aR"><div class="casContent"><span class="casTitleNuber">47a</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro trans-monolayer permeability calculations: often forgotten assumptions</span></div><div class="casAuthors">Youdim, Kuresh A.; Avdeef, Alex; Abbott, N. Joan</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">997-1003</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  In designing effective therapeutic strategies, novel drugs must exhibit favorable pharmacokinetic properties.  The physicochem. characteristics of a drug, such as pKa, mol. wt., soly. and lipophilicity, will influence the way the drug partitions from the aq. phase into membranes, and thus, will influence its ability to cross cellular barriers, such as the lining of the gastrointestinal tract and the blood-brain barrier.  Physicochem. characteristics also influence the degree to which a drug is able to cross a barrier layer, and the route by which it does this; whether transcellular (across the cells) - by diffusion, carrier-mediated transport or transcytosis - or paracellular - by diffusing through the tight junctions between the cells.  The in vitro model systems that are currently employed to screen the permeation characteristics of a drug often represent a compromise between high throughput with low predictive potential and low throughput with high predictive potential.  Here, we will examine the way in which in vitro cellular permeability assays are often performed and the assumptions that are implied but sometimes forgotten, and we will make simple suggestions for improving the methodol. techniques and math. equations used to det. drug permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr74qqjSfpJsrVg90H21EOLACvtfcHk0lhtEs86TZExlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlCltbk%253D&md5=fb9e071aba01fc7a0b70a72511385dc8</span></div><a href="/servlet/linkout?suffix=cit47a&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902873-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902873-3%26sid%3Dliteratum%253Aachs%26aulast%3DYoudim%26aufirst%3DK.%2BA.%26aulast%3DAvdeef%26aufirst%3DA.%26aulast%3DAbbott%26aufirst%3DN.%2BJ.%26atitle%3DIn%2520vitro%2520trans-monolayer%2520permeability%2520calculations%253A%2520Often%2520forgotten%2520assumptions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D997%26epage%3D1003%26doi%3D10.1016%2FS1359-6446%2803%2902873-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit47b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Filipe, H. A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salvador, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silvestre, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaz, W. L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, M. J.</span><span> </span><span class="NLM_article-title">Beyond Overtonâs rule: Quantitative modeling of passive permeation through tight cell monolayers</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3696</span><span class="NLM_x">â</span> <span class="NLM_lpage">3706</span><span class="refDoi">Â DOI: 10.1021/mp500437e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp500437e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyntrrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=3696-3706&author=H.+A.+L.+Filipeauthor=A.+Salvadorauthor=J.+M.+Silvestreauthor=W.+L.+C.+Vazauthor=M.+J.+Moreno&title=Beyond+Overton%E2%80%99s+rule%3A+Quantitative+modeling+of+passive+permeation+through+tight+cell+monolayers&doi=10.1021%2Fmp500437e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47bR"><div class="casContent"><span class="casTitleNuber">47b</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond Overton's Rule: Quantitative Modeling of Passive Permeation through Tight Cell Monolayers</span></div><div class="casAuthors">Filipe, H. A. L.; Salvador, A.; Silvestre, J. M.; Vaz, W. L. C.; Moreno, M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3696-3706</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the great challenges in pharmacokinetics is to find a means to optimize the transport across cell barriers.  In this work, permeation across a cell monolayer, such as the tight endothelia in the blood-brain barrier, was modeled using a homologous series of amphipatic mols., 7-nitrobenz-2-oxa-1,3-diazol-4-yl (NBD)-labeled alkyl chain amphiphiles (NBD-Cn, n = 2 to 16), to obtain rules that relate permeant structure to permeability.  The amphiphile enters the system from the serum, equilibrated with serum albumin and lipoproteins, and its sequestration by serum components, interaction with the endothelium, and accumulation in the tissue is followed over time.  The dependence of the permeability coeff. on the no. of carbons of the amphiphile's alkyl chain has a parabolic-like shape.  After a threshold value, an increase in the hydrophobicity of the amphiphile, along the homologous series, results in a decrease in the characteristic rate of permeation to the tissue.  A sensitivity anal. was performed, and the rate limiting steps for permeation of each amphiphile were identified.  Sequestration in the serum and rate of interaction with the endothelium, particularly the rate of desorption, were found to be the determinant processes for some amphiphiles, while for others translocation was the rate limiting step.  Addnl., for some amphiphiles a single rate limiting step could not be identified, with several steps contributing significantly to the overall permeation.  Finally, we derived anal. equations that adequately describe the rate of amphiphile accumulation in the tissue for the cases where permeation is controlled by a single rate limiting step.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrDofWmSchFrVg90H21EOLACvtfcHk0lgVystTMedB0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyntrrI&md5=41e911bbe66703e5d3c899bc14a9b505</span></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1021%2Fmp500437e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp500437e%26sid%3Dliteratum%253Aachs%26aulast%3DFilipe%26aufirst%3DH.%2BA.%2BL.%26aulast%3DSalvador%26aufirst%3DA.%26aulast%3DSilvestre%26aufirst%3DJ.%2BM.%26aulast%3DVaz%26aufirst%3DW.%2BL.%2BC.%26aulast%3DMoreno%26aufirst%3DM.%2BJ.%26atitle%3DBeyond%2520Overton%25E2%2580%2599s%2520rule%253A%2520Quantitative%2520modeling%2520of%2520passive%2520permeation%2520through%2520tight%2520cell%2520monolayers%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2014%26volume%3D11%26spage%3D3696%26epage%3D3706%26doi%3D10.1021%2Fmp500437e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit47c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Owen, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nedyalkova, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLaughlin, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, M. S.</span><span> </span><span class="NLM_article-title">Colloidal drug formulations can explain âbell-shapedâ concentration-response curves</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">â</span> <span class="NLM_lpage">784</span><span class="refDoi">Â DOI: 10.1021/cb4007584</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4007584" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=777-784&author=S.+C.+Owenauthor=A.+K.+Doakauthor=A.+N.+Ganeshauthor=L.+Nedyalkovaauthor=C.+K.+McLaughlinauthor=B.+K.+Shoichetauthor=M.+S.+Shoichet&title=Colloidal+drug+formulations+can+explain+%E2%80%9Cbell-shaped%E2%80%9D+concentration-response+curves&doi=10.1021%2Fcb4007584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47cR"><div class="casContent"><span class="casTitleNuber">47c</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal Drug Formulations Can Explain "Bell-Shaped" Concentration-Response Curves</span></div><div class="casAuthors">Owen, Shawn C.; Doak, Allison K.; Ganesh, Ahil N.; Nedyalkova, Lyudmila; McLaughlin, Christopher K.; Shoichet, Brian K.; Shoichet, Molly S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">777-784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug efficacy does not always increase sigmoidally with concn., which has puzzled the community for decades.  Unlike std. sigmoidal curves, bell-shaped concn.-response curves suggest more complex biol. effects, such as multiple-binding sites or multiple targets.  Here, we investigate a phys. property-based mechanism for bell-shaped curves.  Beginning with the observation that some drugs form colloidal aggregates at relevant concns., we detd. concn.-response curves for three aggregating anticancer drugs, formulated both as colloids and as free monomer.  Colloidal formulations exhibited bell-shaped curves, losing activity at higher concns., while monomeric formulations gave typical sigmoidal curves, sustaining a plateau of max. activity.  Inverting the question, we next asked if mols. with bell-shaped curves, reported in the literature, form colloidal aggregates at relevant concns.  We selected 12 mols. reported to have bell-shaped concn.-response curves and found that five of these formed colloids.  To understand the mechanism behind the loss of activity at concns. where colloids are present, we investigated the diffusion of colloid-forming dye Evans blue into cells.  We found that colloidal species are excluded from cells, which may explain the mechanism behind toxicol. screens that use Evans blue, Trypan blue, and related dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg5AlTuU9YnrVg90H21EOLACvtfcHk0lgVystTMedB0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D&md5=8cb90ab5f6348326d08ccfcf5377b1f3</span></div><a href="/servlet/linkout?suffix=cit47c&amp;dbid=16384&amp;doi=10.1021%2Fcb4007584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4007584%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DGanesh%26aufirst%3DA.%2BN.%26aulast%3DNedyalkova%26aufirst%3DL.%26aulast%3DMcLaughlin%26aufirst%3DC.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26atitle%3DColloidal%2520drug%2520formulations%2520can%2520explain%2520%25E2%2580%259Cbell-shaped%25E2%2580%259D%2520concentration-response%2520curves%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D777%26epage%3D784%26doi%3D10.1021%2Fcb4007584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span>MoKA; <span class="NLM_publisher-name">Molecular Discovery</span>: <span class="NLM_publisher-loc">Borehamwood, Hertfordshire, UK</span>,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">; </span><a href="http://moldiscovery.com/software/moka" class="extLink">http://www.moldiscovery.com/software/moka</a> (accessed August 28, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MoKA%3B+Molecular+Discovery%3A+Borehamwood%2C+Hertfordshire%2C+UK%2C+2017%3B+http%3A%2F%2Fwww.moldiscovery.com%2Fsoftware%2Fmoka+%28accessed+August+28%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMoKA%26pub%3DMolecular%2520Discovery%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Jones, K.; Rye, C.; Chessum, N.; Cheeseman, M.; Pasqua, A. E.; Pike, K. G.; Faulder, P. F.</span><span> </span><span class="NLM_article-title">Fused 1,4-dihydrodioxin derivatives as inhibitors of heat shock transcription factor 1</span>. WO 2015/049535 A1, April 9,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Jones&author=C.+Rye&author=N.+Chessum&author=M.+Cheeseman&author=A.+E.+Pasqua&author=K.+G.+Pike&author=P.+F.+Faulder&title=Fused+1%2C4-dihydrodioxin+derivatives+as+inhibitors+of+heat+shock+transcription+factor+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DK.%26atitle%3DFused%25201%252C4-dihydrodioxin%2520derivatives%2520as%2520inhibitors%2520of%2520heat%2520shock%2520transcription%2520factor%25201%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="note"><p class="first last">The poor aqueous stability of the thalidomide derived PDPs made measurement of their assay free fractions by dialysis, as well as standard permeability assessments using PAMPA and CACO-2, unreliable so it was not possible to compare PDP probes at their free concentrations or to determine permeability values.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Kisselev, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldberg, A. L.</span><span> </span><span class="NLM_article-title">Proteasome inhibitors: From research tools to drug candidates</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">â</span> <span class="NLM_lpage">758</span><span class="refDoi">Â DOI: 10.1016/S1074-5521(01)00056-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2FS1074-5521%2801%2900056-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=11514224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtV2ku7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=739-758&author=A.+F.+Kisselevauthor=A.+L.+Goldberg&title=Proteasome+inhibitors%3A+From+research+tools+to+drug+candidates&doi=10.1016%2FS1074-5521%2801%2900056-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome inhibitors: from research tools to drug candidates</span></div><div class="casAuthors">Kisselev, A. F.; Goldberg, A. L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">739-758</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The 26S proteasome is a 2.4 MDa multifunctional ATP-dependent proteolytic complex, which degrades the majority of cellular polypeptides by an unusual enzyme mechanism.  Several groups of proteasome inhibitors have been developed and are now widely used as research tools to study the role of the ubiquitin-proteasome pathway in various cellular processes, and two inhibitors are now in clin. trials for treatment of multiple cancers and stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTXBSmAzp9vLVg90H21EOLACvtfcHk0lgVystTMedB0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtV2ku7g%253D&md5=b729e5f56c402172aca5b139df61c09e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2801%2900056-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252801%252900056-4%26sid%3Dliteratum%253Aachs%26aulast%3DKisselev%26aufirst%3DA.%2BF.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DProteasome%2520inhibitors%253A%2520From%2520research%2520tools%2520to%2520drug%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D739%26epage%3D758%26doi%3D10.1016%2FS1074-5521%2801%2900056-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Diz, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvajal-Rodriguez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skibinski, D. O. F.</span><span> </span><span class="NLM_article-title">Multiple hypothesis testing in proteomics: A strategy for experimental work</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">M110.004374</span><span class="refDoi">Â DOI: 10.1074/mcp.M110.004374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1074%2Fmcp.M110.004374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=21364085" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M110.004374&author=A.+P.+Dizauthor=A.+Carvajal-Rodriguezauthor=D.+O.+F.+Skibinski&title=Multiple+hypothesis+testing+in+proteomics%3A+A+strategy+for+experimental+work&doi=10.1074%2Fmcp.M110.004374"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52a&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M110.004374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M110.004374%26sid%3Dliteratum%253Aachs%26aulast%3DDiz%26aufirst%3DA.%2BP.%26aulast%3DCarvajal-Rodriguez%26aufirst%3DA.%26aulast%3DSkibinski%26aufirst%3DD.%2BO.%2BF.%26atitle%3DMultiple%2520hypothesis%2520testing%2520in%2520proteomics%253A%2520A%2520strategy%2520for%2520experimental%2520work%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DM110.004374%26doi%3D10.1074%2Fmcp.M110.004374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit52b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Noble, W. S.</span><span> </span><span class="NLM_article-title">How does multiple testing correction work?</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1135</span><span class="NLM_x">â</span> <span class="NLM_lpage">1137</span><span class="refDoi">Â DOI: 10.1038/nbt1209-1135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnbt1209-1135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=20010596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFemtr3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=1135-1137&author=W.+S.+Noble&title=How+does+multiple+testing+correction+work%3F&doi=10.1038%2Fnbt1209-1135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52bR"><div class="casContent"><span class="casTitleNuber">52b</span><div class="casTitle"><span class="NLM_cas:atitle">How does multiple testing correction work?</span></div><div class="casAuthors">Noble, William S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1135-1137</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  When prioritizing hits from a high-throughput expt., it is important to correct for random events that falsely appear significant.  How is this done and what methods should be used.  Techniques for doing this are discussed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6wxnVhdu_rVg90H21EOLACvtfcHk0lgC2ftVEkv-oA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFemtr3N&md5=7376f0b000468adffe6e6c681ef84c95</span></div><a href="/servlet/linkout?suffix=cit52b&amp;dbid=16384&amp;doi=10.1038%2Fnbt1209-1135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1209-1135%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DW.%2BS.%26atitle%3DHow%2520does%2520multiple%2520testing%2520correction%2520work%253F%26jtitle%3DNat.%2520Biotechnol.%26date%3D2009%26volume%3D27%26spage%3D1135%26epage%3D1137%26doi%3D10.1038%2Fnbt1209-1135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Thorarensen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banker, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fensome, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telliez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juba, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czerwinski, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casimiro-Garcia, A.</span><span> </span><span class="NLM_article-title">ATP-mediated kinome selectivity: The missing link in understanding the contribution of individual JAK kinase isoforms to cellular signaling</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1552</span><span class="NLM_x">â</span> <span class="NLM_lpage">1558</span><span class="refDoi">Â DOI: 10.1021/cb5002125</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5002125" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslart7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1552-1558&author=A.+Thorarensenauthor=M.+E.+Bankerauthor=A.+Fensomeauthor=J.+Telliezauthor=B.+Jubaauthor=F.+Vincentauthor=R.+M.+Czerwinskiauthor=A.+Casimiro-Garcia&title=ATP-mediated+kinome+selectivity%3A+The+missing+link+in+understanding+the+contribution+of+individual+JAK+kinase+isoforms+to+cellular+signaling&doi=10.1021%2Fcb5002125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53aR"><div class="casContent"><span class="casTitleNuber">53a</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-Mediated Kinome Selectivity: The Missing Link in Understanding the Contribution of Individual JAK Kinase Isoforms to Cellular Signaling</span></div><div class="casAuthors">Thorarensen, Atli; Banker, Mary Ellen; Fensome, Andrew; Telliez, Jean-Baptiste; Juba, Brian; Vincent, Fabien; Czerwinski, Robert M.; Casimiro-Garcia, Agustin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1552-1558</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases constitute an important class of therapeutic targets being explored both by academia and the pharmaceutical industry.  The major focus of this effort was directed toward the identification of ATP competitive inhibitors.  Although it has long been recognized that the intracellular concn. of ATP is very different from the concns. utilized in biochem. enzyme assays, little thought has been devoted to incorporating this discrepancy into our understanding of translation from enzyme inhibition to cellular function.  Significant work has been dedicated to the discovery of JAK kinase inhibitors; however, a disconnect between enzyme and cellular function is prominently displayed in the literature for this class of inhibitors.  Herein, we demonstrate utilizing the four JAK family members (JAK1, JAK2, JAK3, and TYK2) that the difference in the ATP Km of each individual kinase has a significant impact on the enzyme to cell inhibition translation.  We evaluated a large no. of JAK inhibitors in enzymic assays utilizing either 1 mM ATP or Km ATP for the four isoforms as well as in primary cell assays.  This data set provided the opportunity to examine individual kinase contributions to the heterodimeric kinase complexes mediating cellular signaling.  In contrast to a recent study, we demonstrate that for IL-15 cytokine signaling it is sufficient to inhibit either JAK1 or JAK3 to fully inhibit downstream STAT5 phosphorylation.  This addnl. data thus provides a crit. piece of information explaining why JAK1 has incorrectly been thought to have a dominant role over JAK3.  Beyond enabling a deeper understanding of JAK signaling, conducting similar analyses for other kinases by taking into account potency at high ATP rather than Km ATP may provide crucial insights into a compd.'s activity and selectivity in cellular contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYiMVl7OpPFLVg90H21EOLACvtfcHk0lgC2ftVEkv-oA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslart7s%253D&md5=c87874cb1748e13eef2aa241ecef3e79</span></div><a href="/servlet/linkout?suffix=cit53a&amp;dbid=16384&amp;doi=10.1021%2Fcb5002125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5002125%26sid%3Dliteratum%253Aachs%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DFensome%26aufirst%3DA.%26aulast%3DTelliez%26aufirst%3DJ.%26aulast%3DJuba%26aufirst%3DB.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DCzerwinski%26aufirst%3DR.%2BM.%26aulast%3DCasimiro-Garcia%26aufirst%3DA.%26atitle%3DATP-mediated%2520kinome%2520selectivity%253A%2520The%2520missing%2520link%2520in%2520understanding%2520the%2520contribution%2520of%2520individual%2520JAK%2520kinase%2520isoforms%2520to%2520cellular%2520signaling%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1552%26epage%3D1558%26doi%3D10.1021%2Fcb5002125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit53b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Smyth, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Measuring and interpreting the selectivity of protein kinase inhibitors</span> <span class="citation_source-journal">J. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">â</span> <span class="NLM_lpage">151</span><span class="refDoi">Â DOI: 10.1007/s12154-009-0023-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1007%2Fs12154-009-0023-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=19568781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=131-151&author=L.+A.+Smythauthor=I.+Collins&title=Measuring+and+interpreting+the+selectivity+of+protein+kinase+inhibitors&doi=10.1007%2Fs12154-009-0023-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53bR"><div class="casContent"><span class="casTitleNuber">53b</span><div class="casTitle"><span class="NLM_cas:atitle">Measuring and interpreting the selectivity of protein kinase inhibitors</span></div><div class="casAuthors">Smyth Lynette A; Collins Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">131-51</span>
        ISSN:<span class="NLM_cas:issn">1864-6158</span>.
    </div><div class="casAbstract">Protein kinase inhibitors are a well-established class of clinically useful drugs, particularly for the treatment of cancer.  Achieving inhibitor selectivity for particular protein kinases often remains a significant challenge in the development of new small molecules as drugs or as tools for chemical biology research.  This review summarises the methodologies available for measuring kinase inhibitor selectivity, both in vitro and in cells.  The interpretation of kinase inhibitor selectivity data is discussed, particularly with reference to the structural biology of the protein targets.  Measurement and prediction of kinase inhibitor selectivity will be important for the development of new multi-targeted kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzU9rA5WZcz9FxYDaSSP8XfW6udTcc2eYbz1IaZWPQ_rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D&md5=f78ff1202a949a71ffd65e804298c37f</span></div><a href="/servlet/linkout?suffix=cit53b&amp;dbid=16384&amp;doi=10.1007%2Fs12154-009-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12154-009-0023-9%26sid%3Dliteratum%253Aachs%26aulast%3DSmyth%26aufirst%3DL.%2BA.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMeasuring%2520and%2520interpreting%2520the%2520selectivity%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D2%26spage%3D131%26epage%3D151%26doi%3D10.1007%2Fs12154-009-0023-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit53c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Knight, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Features of selective kinase inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">621</span><span class="NLM_x">â</span> <span class="NLM_lpage">637</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2005.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.chembiol.2005.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=15975507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFGhtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=621-637&author=Z.+A.+Knightauthor=K.+M.+Shokat&title=Features+of+selective+kinase+inhibitors&doi=10.1016%2Fj.chembiol.2005.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53cR"><div class="casContent"><span class="casTitleNuber">53c</span><div class="casTitle"><span class="NLM_cas:atitle">Features of Selective Kinase Inhibitors</span></div><div class="casAuthors">Knight, Zachary A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">621-637</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Summary: Small-mol. inhibitors of protein and lipid kinases have emerged as indispensable tools for studying signal transduction.  Despite the widespread use of these reagents, there is little consensus about the biochem. criteria that define their potency and selectivity in cells.  We discuss some of the features that det. the cellular activity of kinase inhibitors and propose a framework for interpreting inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAJWhVxtSrCLVg90H21EOLACvtfcHk0lgKs_kBbsZbrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsFGhtrc%253D&md5=ce5b3505a0326b04479172ec05573152</span></div><a href="/servlet/linkout?suffix=cit53c&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DFeatures%2520of%2520selective%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D621%26epage%3D637%26doi%3D10.1016%2Fj.chembiol.2005.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Strelow, J. M.</span><span> </span><span class="NLM_article-title">A perspective on the kinetics of covalent and irreversible inhibition</span> <span class="citation_source-journal">SLAS Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">â</span> <span class="NLM_lpage">20</span><span class="refDoi">Â DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&author=J.+M.+Strelow&title=A+perspective+on+the+kinetics+of+covalent+and+irreversible+inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54aR"><div class="casContent"><span class="casTitleNuber">54a</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0lhuwY4jDrs5mg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit54a&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520perspective%2520on%2520the%2520kinetics%2520of%2520covalent%2520and%2520irreversible%2520inhibition%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note">b<p class="last">Competition of PDP 16 at 6 h exposure with control compounds <b>1</b> and <b>22</b> was unsuccessful at concentration > 1 Î¼M (data not shown).</p></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2a','cit2b'],'ref3':['cit3'],'ref4':['cit4a','cit4b'],'ref5':['cit5a','cit5b'],'ref6':['cit6a','cit6b'],'ref7':['cit7a','cit7b','cit7c'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15a','cit15b','cit15c','cit15d'],'ref16':['cit16a','cit16b'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26a','cit26b','cit26c'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29a','cit29b'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32a','cit32b'],'ref33':['cit33'],'ref34':[],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38a','cit38b'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44a','cit44b'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47a','cit47b','cit47c'],'ref48':['cit48'],'ref49':['cit49'],'ref50':[],'ref51':['cit51'],'ref52':['cit52a','cit52b'],'ref53':['cit53a','cit53b','cit53c'],'ref54':['cit54a']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 51 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Shuhui Lim, Regina Khoo, Yu-Chi Juang, Pooja Gopal, Huibin Zhang, Constance Yeo, Khong Ming Peh, Jinkai Teo, Simon Ng, Brian Henry, <span class="NLM_string-name hlFld-ContribAuthor">Anthony W. Partridge</span>. </span><span class="cited-content_cbyCitation_article-title">Exquisitely Specific anti-KRAS Biodegraders Inform on the Cellular Prevalence of Nucleotide-Loaded States. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2021,</strong> <em>7 </em>
                                    (2)
                                     , 274-291. <a href="https://doi.org/10.1021/acscentsci.0c01337" title="DOI URL">https://doi.org/10.1021/acscentsci.0c01337</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.0c01337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.0c01337%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DExquisitely%252BSpecific%252Banti-KRAS%252BBiodegraders%252BInform%252Bon%252Bthe%252BCellular%252BPrevalence%252Bof%252BNucleotide-Loaded%252BStates%26aulast%3DLim%26aufirst%3DShuhui%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D05102020%26date%3D28122020%26volume%3D7%26issue%3D2%26spage%3D274%26epage%3D291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael L. Drummond, Andrew Henry, Huifang Li, <span class="NLM_string-name hlFld-ContribAuthor">Christopher I. Williams</span>. </span><span class="cited-content_cbyCitation_article-title">Improved Accuracy for Modeling PROTAC-Mediated Ternary Complex Formation and Targeted Protein Degradation via New In Silico Methodologies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 5234-5254. <a href="https://doi.org/10.1021/acs.jcim.0c00897" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00897</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00897%26sid%3Dliteratum%253Aachs%26aulast%3DDrummond%26aufirst%3DM.%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D1634%26epage%3D1644%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproved%252BAccuracy%252Bfor%252BModeling%252BPROTAC-Mediated%252BTernary%252BComplex%252BFormation%252Band%252BTargeted%252BProtein%252BDegradation%252Bvia%252BNew%252BIn%252BSilico%252BMethodologies%26aulast%3DDrummond%26aufirst%3DMichael%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06082020%26date%3D12102020%26date%3D24092020%26volume%3D60%26issue%3D10%26spage%3D5234%26epage%3D5254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Duncan E. Scott, Timothy P. C. Rooney, Elliott D. Bayle, Tashfina Mirza, Henriette M. G. Willems, Jonathan H. Clarke, Stephen P. Andrews, <span class="NLM_string-name hlFld-ContribAuthor">John Skidmore</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Investigation of the Permeability of Androgen Receptor PROTACs. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1539-1547. <a href="https://doi.org/10.1021/acsmedchemlett.0c00194" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00194%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSystematic%252BInvestigation%252Bof%252Bthe%252BPermeability%252Bof%252BAndrogen%252BReceptor%252BPROTACs%26aulast%3DScott%26aufirst%3DDuncan%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15042020%26date%3D01062020%26date%3D08062020%26volume%3D11%26issue%3D8%26spage%3D1539%26epage%3D1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jakub Stefaniak, <span class="NLM_string-name hlFld-ContribAuthor">Kilian V. M. Huber</span>. </span><span class="cited-content_cbyCitation_article-title">Importance of Quantifying Drug-Target Engagement in Cells. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (4)
                                     , 403-406. <a href="https://doi.org/10.1021/acsmedchemlett.9b00570" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00570</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00570%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DImportance%252Bof%252BQuantifying%252BDrug-Target%252BEngagement%252Bin%252BCells%26aulast%3DStefaniak%26aufirst%3DJakub%26date%3D2020%26date%3D2020%26date%3D06032020%26volume%3D11%26issue%3D4%26spage%3D403%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Caroline A. Foley, Frances Potjewyd, Kelsey N. Lamb, Lindsey I. James, <span class="NLM_string-name hlFld-ContribAuthor">Stephen V. Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Assessing the Cell Permeability of Bivalent Chemical Degraders Using the Chloroalkane Penetration Assay. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (1)
                                     , 290-295. <a href="https://doi.org/10.1021/acschembio.9b00972" title="DOI URL">https://doi.org/10.1021/acschembio.9b00972</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00972%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAssessing%252Bthe%252BCell%252BPermeability%252Bof%252BBivalent%252BChemical%252BDegraders%252BUsing%252Bthe%252BChloroalkane%252BPenetration%252BAssay%26aulast%3DFoley%26aufirst%3DCaroline%2BA.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04122019%26date%3D17122019%26date%3D27122019%26date%3D17122019%26volume%3D15%26issue%3D1%26spage%3D290%26epage%3D295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Johannes Popow, Heribert Arnhof, Gerd Bader, Helmut Berger, Alessio Ciulli, David Covini, Christian Dank, Teresa Gmaschitz, Peter Greb, Jale Karolyi-Ãzguer, Manfred Koegl, Darryl B. McConnell, Mark Pearson, Maria Rieger, Joerg Rinnenthal, Vanessa Roessler, Andreas Schrenk, Markus Spina, Steffen Steurer, Nicole Trainor, Elisabeth Traxler, Corinna Wieshofer, Andreas Zoephel, <span class="NLM_string-name hlFld-ContribAuthor">Peter Ettmayer</span>. </span><span class="cited-content_cbyCitation_article-title">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2508-2520. <a href="https://doi.org/10.1021/acs.jmedchem.8b01826" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01826</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHighly%252BSelective%252BPTK2%252BProteolysis%252BTargeting%252BChimeras%252Bto%252BProbe%252BFocal%252BAdhesion%252BKinase%252BScaffolding%252BFunctions%26aulast%3DPopow%26aufirst%3DJohannes%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D22112018%26date%3D22022019%26date%3D09022019%26volume%3D62%26issue%3D5%26spage%3D2508%26epage%3D2520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joshua Meyers, Nicola E. A. Chessum, Salyha Ali, N. Yi Mok, Birgit Wilding, A. Elisa Pasqua, Martin Rowlands, Michael J. Tucker, Lindsay E. Evans, Carl S. Rye, Lisa OâFee, Yann-VaÃ¯ Le Bihan, Rosemary Burke, Michael Carter, Paul Workman, Julian Blagg, Nathan Brown, Rob L. M. van Montfort, Keith Jones, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Cheeseman</span>. </span><span class="cited-content_cbyCitation_article-title">Privileged Structures and Polypharmacology within and between Protein Families. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (12)
                                     , 1199-1204. <a href="https://doi.org/10.1021/acsmedchemlett.8b00364" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00364</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00364%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPrivileged%252BStructures%252Band%252BPolypharmacology%252Bwithin%252Band%252Bbetween%252BProtein%252BFamilies%26aulast%3DMeyers%26aufirst%3DJoshua%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D09082018%26date%3D16112018%26date%3D26112018%26date%3D16112018%26volume%3D9%26issue%3D12%26spage%3D1199%26epage%3D1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christian Steinebach, Stefanie Lindner, Namrata D. Udeshi, Deepak C. Mani, Hannes Kehm, Simon KÃ¶pff, Steven A. Carr, Michael GÃ¼tschow, <span class="NLM_string-name hlFld-ContribAuthor">Jan KrÃ¶nke</span>. </span><span class="cited-content_cbyCitation_article-title">Homo-PROTACs for the Chemical Knockdown of Cereblon. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (9)
                                     , 2771-2782. <a href="https://doi.org/10.1021/acschembio.8b00693" title="DOI URL">https://doi.org/10.1021/acschembio.8b00693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00693%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DHomo-PROTACs%252Bfor%252Bthe%252BChemical%252BKnockdown%252Bof%252BCereblon%26aulast%3DSteinebach%26aufirst%3DChristian%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D25072018%26date%3D17082018%26date%3D05092018%26date%3D17082018%26volume%3D13%26issue%3D9%26spage%3D2771%26epage%3D2782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joni V.  Lindbohm</span>, <span class="hlFld-ContribAuthor ">Nina  Mars</span>, <span class="hlFld-ContribAuthor ">Keenan A.  Walker</span>, <span class="hlFld-ContribAuthor ">Archana  SinghâManoux</span>, <span class="hlFld-ContribAuthor ">Gill  Livingston</span>, <span class="hlFld-ContribAuthor ">Eric J.  Brunner</span>, <span class="hlFld-ContribAuthor ">Pyry N.  SipilÃ¤</span>, <span class="hlFld-ContribAuthor ">Kalle  Saksela</span>, <span class="hlFld-ContribAuthor ">Jane E.  Ferrie</span>, <span class="hlFld-ContribAuthor ">Ruth C.  Lovering</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Williams</span>, <span class="hlFld-ContribAuthor ">Aroon D.  Hingorani</span>, <span class="hlFld-ContribAuthor ">Rebecca F.  Gottesman</span>, <span class="hlFld-ContribAuthor ">Henrik  Zetterberg</span>, <span class="hlFld-ContribAuthor ">Mika  KivimÃ¤ki</span>. </span><span class="cited-content_cbyCitation_article-title">Plasma proteins, cognitive decline, and 20âyear risk of dementia in the Whitehall II and Atherosclerosis Risk in Communities studies. </span><span class="cited-content_cbyCitation_journal-name">Alzheimer's & Dementia</span><span> <strong>2021,</strong> <em>380 </em><a href="https://doi.org/10.1002/alz.12419" title="DOI URL">https://doi.org/10.1002/alz.12419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/alz.12419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Falz.12419%26sid%3Dliteratum%253Aachs%26jtitle%3DAlzheimer%2527s%2520%2526%2520Dementia%26atitle%3DPlasma%252Bproteins%25252C%252Bcognitive%252Bdecline%25252C%252Band%252B20%2525E2%252580%252590year%252Brisk%252Bof%252Bdementia%252Bin%252Bthe%252BWhitehall%252BII%252Band%252BAtherosclerosis%252BRisk%252Bin%252BCommunities%252Bstudies%26aulast%3DLindbohm%26aufirst%3DJoni%2BV.%26date%3D2021%26date%3D2021%26volume%3D380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">AleÅ¡a  Bricelj</span>, <span class="hlFld-ContribAuthor ">Christian  Steinebach</span>, <span class="hlFld-ContribAuthor ">Robert  Kuchta</span>, <span class="hlFld-ContribAuthor ">Michael  GÃ¼tschow</span>, <span class="hlFld-ContribAuthor ">Izidor  SosiÄ</span>. </span><span class="cited-content_cbyCitation_article-title">E3 Ligase Ligands in Successful PROTACs: An Overview of Syntheses and Linker Attachment Points. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.707317" title="DOI URL">https://doi.org/10.3389/fchem.2021.707317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.707317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.707317%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DE3%252BLigase%252BLigands%252Bin%252BSuccessful%252BPROTACs%25253A%252BAn%252BOverview%252Bof%252BSyntheses%252Band%252BLinker%252BAttachment%252BPoints%26aulast%3DBricelj%26aufirst%3DAle%25C5%25A1a%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hannah  Kiely-Collins</span>, <span class="hlFld-ContribAuthor ">Georg E.  Winter</span>, <span class="hlFld-ContribAuthor ">GonÃ§alo J.L.  Bernardes</span>. </span><span class="cited-content_cbyCitation_article-title">The role of reversible and irreversible covalent chemistry in targeted protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 952-968. <a href="https://doi.org/10.1016/j.chembiol.2021.03.005" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.03.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.03.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DThe%252Brole%252Bof%252Breversible%252Band%252Birreversible%252Bcovalent%252Bchemistry%252Bin%252Btargeted%252Bprotein%252Bdegradation%26aulast%3DKiely-Collins%26aufirst%3DHannah%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D952%26epage%3D968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-37. <a href="https://doi.org/10.1002/0471266949.bmc256" title="DOI URL">https://doi.org/10.1002/0471266949.bmc256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc256%26sid%3Dliteratum%253Aachs%26atitle%3DMedicinal%252BChemical%252BBiology%26aulast%3DJones%26aufirst%3DLyn%2BH.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D37%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlotta  Cecchini</span>, <span class="hlFld-ContribAuthor ">Sara  Pannilunghi</span>, <span class="hlFld-ContribAuthor ">SÃ©bastien  Tardy</span>, <span class="hlFld-ContribAuthor ">Leonardo  Scapozza</span>. </span><span class="cited-content_cbyCitation_article-title">From Conception to Development: Investigating PROTACs Features for Improved Cell Permeability and Successful Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.672267" title="DOI URL">https://doi.org/10.3389/fchem.2021.672267</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.672267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.672267%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DFrom%252BConception%252Bto%252BDevelopment%25253A%252BInvestigating%252BPROTACs%252BFeatures%252Bfor%252BImproved%252BCell%252BPermeability%252Band%252BSuccessful%252BProtein%252BDegradation%26aulast%3DCecchini%26aufirst%3DCarlotta%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">RadosÅaw P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Target Validation Using PROTACs: Applying the Four Pillars Framework. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 474-483. <a href="https://doi.org/10.1177/2472555220979584" title="DOI URL">https://doi.org/10.1177/2472555220979584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220979584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220979584%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DTarget%252BValidation%252BUsing%252BPROTACs%25253A%252BApplying%252Bthe%252BFour%252BPillars%252BFramework%26aulast%3DNowak%26aufirst%3DRados%25C5%2582aw%2BP.%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D4%26spage%3D474%26epage%3D483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinrui  Yang</span>, <span class="hlFld-ContribAuthor ">He  Yin</span>, <span class="hlFld-ContribAuthor ">Richard D.  Kim</span>, <span class="hlFld-ContribAuthor ">Jason B.  Fleming</span>, <span class="hlFld-ContribAuthor ">Hao  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical and Clinical Advances of Targeted Protein Degradation as a Novel Cancer Therapeutic Strategy: An Oncologist Perspective. </span><span class="cited-content_cbyCitation_journal-name">Targeted Oncology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 1-12. <a href="https://doi.org/10.1007/s11523-020-00782-2" title="DOI URL">https://doi.org/10.1007/s11523-020-00782-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11523-020-00782-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11523-020-00782-2%26sid%3Dliteratum%253Aachs%26jtitle%3DTargeted%2520Oncology%26atitle%3DPreclinical%252Band%252BClinical%252BAdvances%252Bof%252BTargeted%252BProtein%252BDegradation%252Bas%252Ba%252BNovel%252BCancer%252BTherapeutic%252BStrategy%25253A%252BAn%252BOncologist%252BPerspective%26aulast%3DYang%26aufirst%3DXinrui%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Target ValidationâProsecuting the Target. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-820472-6.00014-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-820472-6.00014-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820472-6.00014-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820472-6.00014-1%26sid%3Dliteratum%253Aachs%26atitle%3DTarget%252BValidation%2525E2%252580%252594Prosecuting%252Bthe%252BTarget%26aulast%3DJones%26aufirst%3DLyn%2BH.%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BBiomedical%252BSciences%26date%3D2021%26volume%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Nicola  Mautone</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112750. <a href="https://doi.org/10.1016/j.ejmech.2020.112750" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bepigenetic%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528Epi-PROTACs%252529%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26volume%3D207%26spage%3D112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Jieqing  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC: A Novel Technology for Drug Development**. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (42)
                                     , 13232-13247. <a href="https://doi.org/10.1002/slct.202003162" title="DOI URL">https://doi.org/10.1002/slct.202003162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202003162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202003162%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DPROTAC%25253A%252BA%252BNovel%252BTechnology%252Bfor%252BDrug%252BDevelopment%252A%252A%26aulast%3DLi%26aufirst%3DJie%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D42%26spage%3D13232%26epage%3D13247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert I.  Troup</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>, <span class="hlFld-ContribAuthor ">Matthias G. J.  Baud</span>. </span><span class="cited-content_cbyCitation_article-title">Current strategies for the design of PROTAC linkers: a critical review. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (5)
                                     <a href="https://doi.org/10.37349/etat.2020.00018" title="DOI URL">https://doi.org/10.37349/etat.2020.00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00018%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DCurrent%252Bstrategies%252Bfor%252Bthe%252Bdesign%252Bof%252BPROTAC%252Blinkers%25253A%252Ba%252Bcritical%252Breview%26aulast%3DTroup%26aufirst%3DRobert%2BI.%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Ermondi</span>, <span class="hlFld-ContribAuthor ">Maura  Vallaro</span>, <span class="hlFld-ContribAuthor ">Giulia  Caron</span>. </span><span class="cited-content_cbyCitation_article-title">Degraders early developability assessment: face-to-face with molecular properties. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1585-1591. <a href="https://doi.org/10.1016/j.drudis.2020.06.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.06.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.06.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DDegraders%252Bearly%252Bdevelopability%252Bassessment%25253A%252Bface-to-face%252Bwith%252Bmolecular%252Bproperties%26aulast%3DErmondi%26aufirst%3DGiuseppe%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1585%26epage%3D1591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yilin  Chen</span>, <span class="hlFld-ContribAuthor ">Jianping  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">The application of ubiquitin ligases in the PROTAC drug design. </span><span class="cited-content_cbyCitation_journal-name">Acta Biochimica et Biophysica Sinica</span><span> <strong>2020,</strong> <em>52 </em>
                                    (7)
                                     , 776-790. <a href="https://doi.org/10.1093/abbs/gmaa053" title="DOI URL">https://doi.org/10.1093/abbs/gmaa053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abbs/gmaa053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabbs%2Fgmaa053%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Biochimica%2520et%2520Biophysica%2520Sinica%26atitle%3DThe%252Bapplication%252Bof%252Bubiquitin%252Bligases%252Bin%252Bthe%252BPROTAC%252Bdrug%252Bdesign%26aulast%3DChen%26aufirst%3DYilin%26date%3D2020%26date%3D2020%26volume%3D52%26issue%3D7%26spage%3D776%26epage%3D790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tao  Wu</span>, <span class="hlFld-ContribAuthor ">Hojong  Yoon</span>, <span class="hlFld-ContribAuthor ">Yuan  Xiong</span>, <span class="hlFld-ContribAuthor ">Sarah E.  Dixon-Clarke</span>, <span class="hlFld-ContribAuthor ">RadosÅaw P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted protein degradation as a powerful research tool in basic biology and drug target discovery. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (7)
                                     , 605-614. <a href="https://doi.org/10.1038/s41594-020-0438-0" title="DOI URL">https://doi.org/10.1038/s41594-020-0438-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41594-020-0438-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41594-020-0438-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DTargeted%252Bprotein%252Bdegradation%252Bas%252Ba%252Bpowerful%252Bresearch%252Btool%252Bin%252Bbasic%252Bbiology%252Band%252Bdrug%252Btarget%252Bdiscovery%26aulast%3DWu%26aufirst%3DTao%26date%3D2020%26date%3D2020%26volume%3D27%26issue%3D7%26spage%3D605%26epage%3D614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ka  Yang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhao</span>, <span class="hlFld-ContribAuthor ">Xueqing  Nie</span>, <span class="hlFld-ContribAuthor ">Hao  Wu</span>, <span class="hlFld-ContribAuthor ">Bo  Wang</span>, <span class="hlFld-ContribAuthor ">Chelsi M.  Almodovar-Rivera</span>, <span class="hlFld-ContribAuthor ">Haibo  Xie</span>, <span class="hlFld-ContribAuthor ">Weiping  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">A Cell-Based Target Engagement Assay for the Identification of Cereblon E3Â Ubiquitin Ligase Ligands and Their Application in HDAC6 Degraders. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (7)
                                     , 866-876.e8. <a href="https://doi.org/10.1016/j.chembiol.2020.04.008" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.04.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.04.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DA%252BCell-Based%252BTarget%252BEngagement%252BAssay%252Bfor%252Bthe%252BIdentification%252Bof%252BCereblon%252BE3%2525C2%2525A0Ubiquitin%252BLigase%252BLigands%252Band%252BTheir%252BApplication%252Bin%252BHDAC6%252BDegraders%26aulast%3DYang%26aufirst%3DKa%26date%3D2020%26volume%3D27%26issue%3D7%26spage%3D866%26epage%3D876.e8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachael  Arthur</span>, <span class="hlFld-ContribAuthor ">Beatriz  Beatriz Valle-Argos</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Steele</span>, <span class="hlFld-ContribAuthor ">Graham  Packham</span>. </span><span class="cited-content_cbyCitation_article-title">Development of PROTACs to address clinical limitations associated with BTK-targeted kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (3)
                                     , 131-152. <a href="https://doi.org/10.37349/etat.2020.00009" title="DOI URL">https://doi.org/10.37349/etat.2020.00009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00009%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DDevelopment%252Bof%252BPROTACs%252Bto%252Baddress%252Bclinical%252Blimitations%252Bassociated%252Bwith%252BBTK-targeted%252Bkinase%252Binhibitors%26aulast%3DArthur%26aufirst%3DRachael%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D3%26spage%3D131%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carina  Cantrill</span>, <span class="hlFld-ContribAuthor ">Prasoon  Chaturvedi</span>, <span class="hlFld-ContribAuthor ">Caroline  Rynn</span>, <span class="hlFld-ContribAuthor ">Jeannine  Petrig Schaffland</span>, <span class="hlFld-ContribAuthor ">Isabelle  Walter</span>, <span class="hlFld-ContribAuthor ">Matthias B.  Wittwer</span>. </span><span class="cited-content_cbyCitation_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (6)
                                     , 969-982. <a href="https://doi.org/10.1016/j.drudis.2020.03.012" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.03.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DFundamental%252Baspects%252Bof%252BDMPK%252Boptimization%252Bof%252Btargeted%252Bprotein%252Bdegraders%26aulast%3DCantrill%26aufirst%3DCarina%26date%3D2020%26volume%3D25%26issue%3D6%26spage%3D969%26epage%3D982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexa C.  Cannon</span>, <span class="hlFld-ContribAuthor ">Cristina  Uribe-Alvarez</span>, <span class="hlFld-ContribAuthor ">Jonathan  Chernoff</span>. </span><span class="cited-content_cbyCitation_article-title">RAC1 as a Therapeutic Target in Malignant Melanoma. </span><span class="cited-content_cbyCitation_journal-name">Trends in Cancer</span><span> <strong>2020,</strong> <em>6 </em>
                                    (6)
                                     , 478-488. <a href="https://doi.org/10.1016/j.trecan.2020.02.021" title="DOI URL">https://doi.org/10.1016/j.trecan.2020.02.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.trecan.2020.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.trecan.2020.02.021%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Cancer%26atitle%3DRAC1%252Bas%252Ba%252BTherapeutic%252BTarget%252Bin%252BMalignant%252BMelanoma%26aulast%3DCannon%26aufirst%3DAlexa%2BC.%26date%3D2020%26volume%3D6%26issue%3D6%26spage%3D478%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Dongwen  Lv</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Assays and technologies for developing proteolysis targeting chimera degraders. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>18 </em><a href="https://doi.org/10.4155/fmc-2020-0073" title="DOI URL">https://doi.org/10.4155/fmc-2020-0073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0073%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAssays%252Band%252Btechnologies%252Bfor%252Bdeveloping%252Bproteolysis%252Btargeting%252Bchimera%252Bdegraders%26aulast%3DLiu%26aufirst%3DXingui%26date%3D2020%26date%3D2020%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Milka  Kostic</span>, <span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Critical Assessment of Targeted Protein Degradation as a Research Tool and Pharmacological Modality. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (5)
                                     , 305-317. <a href="https://doi.org/10.1016/j.tips.2020.02.006" title="DOI URL">https://doi.org/10.1016/j.tips.2020.02.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.02.006%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DCritical%252BAssessment%252Bof%252BTargeted%252BProtein%252BDegradation%252Bas%252Ba%252BResearch%252BTool%252Band%252BPharmacological%252BModality%26aulast%3DKostic%26aufirst%3DMilka%26date%3D2020%26volume%3D41%26issue%3D5%26spage%3D305%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samuel E.  Dalton</span>, <span class="hlFld-ContribAuthor ">Sebastien  Campos</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Small Molecules as Enabling Platforms for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2020,</strong> <em>21 </em>
                                    (8)
                                     , 1080-1100. <a href="https://doi.org/10.1002/cbic.201900674" title="DOI URL">https://doi.org/10.1002/cbic.201900674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201900674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201900674%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DCovalent%252BSmall%252BMolecules%252Bas%252BEnabling%252BPlatforms%252Bfor%252BDrug%252BDiscovery%26aulast%3DDalton%26aufirst%3DSamuel%2BE.%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D8%26spage%3D1080%26epage%3D1100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Steinebach</span>, <span class="hlFld-ContribAuthor ">Yuen Lam Dora  Ng</span>, <span class="hlFld-ContribAuthor ">Izidor  SosiÄ</span>, <span class="hlFld-ContribAuthor ">Chih-Shia  Lee</span>, <span class="hlFld-ContribAuthor ">Sirui  Chen</span>, <span class="hlFld-ContribAuthor ">Stefanie  Lindner</span>, <span class="hlFld-ContribAuthor ">Lan Phuong  Vu</span>, <span class="hlFld-ContribAuthor ">AleÅ¡a  Bricelj</span>, <span class="hlFld-ContribAuthor ">Reza  Haschemi</span>, <span class="hlFld-ContribAuthor ">Marius  Monschke</span>, <span class="hlFld-ContribAuthor ">Elisabeth  Steinwarz</span>, <span class="hlFld-ContribAuthor ">Karl G.  Wagner</span>, <span class="hlFld-ContribAuthor ">Gerd  Bendas</span>, <span class="hlFld-ContribAuthor ">Ji  Luo</span>, <span class="hlFld-ContribAuthor ">Michael  GÃ¼tschow</span>, <span class="hlFld-ContribAuthor ">Jan  KrÃ¶nke</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic exploration of different E3 ubiquitin ligases: an approach towards potent and selective CDK6 degraders. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2020,</strong> <em>11 </em>
                                    (13)
                                     , 3474-3486. <a href="https://doi.org/10.1039/D0SC00167H" title="DOI URL">https://doi.org/10.1039/D0SC00167H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC00167H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC00167H%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DSystematic%252Bexploration%252Bof%252Bdifferent%252BE3%252Bubiquitin%252Bligases%25253A%252Ban%252Bapproach%252Btowards%252Bpotent%252Band%252Bselective%252BCDK6%252Bdegraders%26aulast%3DSteinebach%26aufirst%3DChristian%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D13%26spage%3D3474%26epage%3D3486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George  Karageorgis</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Foley</span>, <span class="hlFld-ContribAuthor ">Luca  Laraia</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>. </span><span class="cited-content_cbyCitation_article-title">Principle and design of pseudo-natural products. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (3)
                                     , 227-235. <a href="https://doi.org/10.1038/s41557-019-0411-x" title="DOI URL">https://doi.org/10.1038/s41557-019-0411-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41557-019-0411-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41557-019-0411-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemistry%26atitle%3DPrinciple%252Band%252Bdesign%252Bof%252Bpseudo-natural%252Bproducts%26aulast%3DKarageorgis%26aufirst%3DGeorge%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D3%26spage%3D227%26epage%3D235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rati  Verma</span>, <span class="hlFld-ContribAuthor ">Dane  Mohl</span>, <span class="hlFld-ContribAuthor ">Raymond J.  Deshaies</span>. </span><span class="cited-content_cbyCitation_article-title">Harnessing the Power of Proteolysis for Targeted Protein Inactivation. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2020,</strong> <em>77 </em>
                                    (3)
                                     , 446-460. <a href="https://doi.org/10.1016/j.molcel.2020.01.010" title="DOI URL">https://doi.org/10.1016/j.molcel.2020.01.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2020.01.010%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DHarnessing%252Bthe%252BPower%252Bof%252BProteolysis%252Bfor%252BTargeted%252BProtein%252BInactivation%26aulast%3DVerma%26aufirst%3DRati%26date%3D2020%26volume%3D77%26issue%3D3%26spage%3D446%26epage%3D460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Zhang</span>, <span class="hlFld-ContribAuthor ">Dorothy  Wang</span>, <span class="hlFld-ContribAuthor ">Bin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors Targeting Heat Shock Response Pathways: Lessons from Clinical and Preclinical Studies in Cancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 79-99. <a href="https://doi.org/10.1007/7515_2020_2" title="DOI URL">https://doi.org/10.1007/7515_2020_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7515_2020_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7515_2020_2%26sid%3Dliteratum%253Aachs%26atitle%3DSmall%252BMolecule%252BInhibitors%252BTargeting%252BHeat%252BShock%252BResponse%252BPathways%25253A%252BLessons%252Bfrom%252BClinical%252Band%252BPreclinical%252BStudies%252Bin%252BCancer%252BTherapeutics%26aulast%3DZhang%26aufirst%3DDaniel%26date%3D2020%26date%3D2020%26spage%3D79%26epage%3D99%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHeat%252BShock%252BProteins%252Bin%252BHuman%252BDiseases%26aulast%3DAsea%26aufirst%3DAlexzander%2BA.%2BA.%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Markella  Konstantinidou</span>, <span class="hlFld-ContribAuthor ">Jingyao  Li</span>, <span class="hlFld-ContribAuthor ">Bidong  Zhang</span>, <span class="hlFld-ContribAuthor ">Zefeng  Wang</span>, <span class="hlFld-ContribAuthor ">Shabnam  Shaabani</span>, <span class="hlFld-ContribAuthor ">Frans  Ter Brake</span>, <span class="hlFld-ContribAuthor ">Khaled  Essa</span>, <span class="hlFld-ContribAuthor ">Alexander  DÃ¶mling</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACsâ a game-changing technology. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 1255-1268. <a href="https://doi.org/10.1080/17460441.2019.1659242" title="DOI URL">https://doi.org/10.1080/17460441.2019.1659242</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1659242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1659242%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DPROTACs%2525E2%252580%252593%252Ba%252Bgame-changing%252Btechnology%26aulast%3DKonstantinidou%26aufirst%3DMarkella%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D12%26spage%3D1255%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuyun  Sun</span>, <span class="hlFld-ContribAuthor ">Hongying  Gao</span>, <span class="hlFld-ContribAuthor ">Yiqing  Yang</span>, <span class="hlFld-ContribAuthor ">Ming  He</span>, <span class="hlFld-ContribAuthor ">Yue  Wu</span>, <span class="hlFld-ContribAuthor ">Yugang  Song</span>, <span class="hlFld-ContribAuthor ">Yan  Tong</span>, <span class="hlFld-ContribAuthor ">Yu  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: great opportunities for academia and industry. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-019-0101-6" title="DOI URL">https://doi.org/10.1038/s41392-019-0101-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DPROTACs%25253A%252Bgreat%252Bopportunities%252Bfor%252Bacademia%252Band%252Bindustry%26aulast%3DSun%26aufirst%3DXiuyun%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ji  Yu</span>, <span class="hlFld-ContribAuthor ">Zhongxia  Zhou</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design opportunities presented by solventâexposed regions of target proteins. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2194-2238. <a href="https://doi.org/10.1002/med.21581" title="DOI URL">https://doi.org/10.1002/med.21581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21581%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMolecular%252Bdesign%252Bopportunities%252Bpresented%252Bby%252Bsolvent%2525E2%252580%252590exposed%252Bregions%252Bof%252Btarget%252Bproteins%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2194%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hannah J.  Maple</span>, <span class="hlFld-ContribAuthor ">Nat  Clayden</span>, <span class="hlFld-ContribAuthor ">Anne  Baron</span>, <span class="hlFld-ContribAuthor ">Callum  Stacey</span>, <span class="hlFld-ContribAuthor ">Robert  Felix</span>. </span><span class="cited-content_cbyCitation_article-title">Developing degraders: principles and perspectives on design and chemical space. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 1755-1764. <a href="https://doi.org/10.1039/C9MD00272C" title="DOI URL">https://doi.org/10.1039/C9MD00272C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00272C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00272C%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DDeveloping%252Bdegraders%25253A%252Bprinciples%252Band%252Bperspectives%252Bon%252Bdesign%252Band%252Bchemical%252Bspace%26aulast%3DMaple%26aufirst%3DHannah%2BJ.%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D1755%26epage%3D1764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed  Benchekroun</span>. </span><span class="cited-content_cbyCitation_article-title">The advent of directed protein degraders in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Future Drug Discovery</span><span> <strong>2019,</strong> <em>1 </em>
                                    (2)
                                     , FDD16. <a href="https://doi.org/10.4155/fdd-2019-0019" title="DOI URL">https://doi.org/10.4155/fdd-2019-0019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fdd-2019-0019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffdd-2019-0019%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Drug%2520Discovery%26atitle%3DThe%252Badvent%252Bof%252Bdirected%252Bprotein%252Bdegraders%252Bin%252Bdrug%252Bdiscovery%26aulast%3DBenchekroun%26aufirst%3DMohamed%26date%3D2019%26volume%3D1%26issue%3D2%26spage%3DFDD16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Valeur</span>, <span class="hlFld-ContribAuthor ">Frank  Narjes</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging modes-of-action in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1550-1568. <a href="https://doi.org/10.1039/C9MD00263D" title="DOI URL">https://doi.org/10.1039/C9MD00263D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00263D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DEmerging%252Bmodes-of-action%252Bin%252Bdrug%252Bdiscovery%26aulast%3DValeur%26aufirst%3DEric%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D9%26spage%3D1550%26epage%3D1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">AlexandrÃ©  Delport</span>, <span class="hlFld-ContribAuthor ">Raymond  Hewer</span>. </span><span class="cited-content_cbyCitation_article-title">Inducing the Degradation of Disease-Related Proteins Using Heterobifunctional Molecules. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (18)
                                     , 3272. <a href="https://doi.org/10.3390/molecules24183272" title="DOI URL">https://doi.org/10.3390/molecules24183272</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24183272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24183272%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DInducing%252Bthe%252BDegradation%252Bof%252BDisease-Related%252BProteins%252BUsing%252BHeterobifunctional%252BMolecules%26aulast%3DDelport%26aufirst%3DAlexandr%25C3%25A9%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D18%26spage%3D3272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James E.  Kath</span>, <span class="hlFld-ContribAuthor ">Aleksandra  Baranczak</span>. </span><span class="cited-content_cbyCitation_article-title">Target engagement approaches for pharmacological evaluation in animal models. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (63)
                                     , 9241-9250. <a href="https://doi.org/10.1039/C9CC02824B" title="DOI URL">https://doi.org/10.1039/C9CC02824B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CC02824B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CC02824B%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DTarget%252Bengagement%252Bapproaches%252Bfor%252Bpharmacological%252Bevaluation%252Bin%252Banimal%252Bmodels%26aulast%3DKath%26aufirst%3DJames%2BE.%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D63%26spage%3D9241%26epage%3D9250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Scott D.  Edmondson</span>, <span class="hlFld-ContribAuthor ">Bin  Yang</span>, <span class="hlFld-ContribAuthor ">Charlene  Fallan</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimeras (PROTACs) in âbeyond rule-of-fiveâ chemical space: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (13)
                                     , 1555-1564. <a href="https://doi.org/10.1016/j.bmcl.2019.04.030" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.04.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DProteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bin%252B%2525E2%252580%252598beyond%252Brule-of-five%2525E2%252580%252599%252Bchemical%252Bspace%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DEdmondson%26aufirst%3DScott%2BD.%26date%3D2019%26volume%3D29%26issue%3D13%26spage%3D1555%26epage%3D1564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hugues  Prevet</span>, <span class="hlFld-ContribAuthor ">Ian  Collins</span>. </span><span class="cited-content_cbyCitation_article-title">Labelled chemical probes for demonstrating direct target engagement in living systems. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (10)
                                     , 1195-1224. <a href="https://doi.org/10.4155/fmc-2018-0370" title="DOI URL">https://doi.org/10.4155/fmc-2018-0370</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0370%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DLabelled%252Bchemical%252Bprobes%252Bfor%252Bdemonstrating%252Bdirect%252Btarget%252Bengagement%252Bin%252Bliving%252Bsystems%26aulast%3DPrevet%26aufirst%3DHugues%26date%3D2019%26volume%3D11%26issue%3D10%26spage%3D1195%26epage%3D1224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariell  Pettersson</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">PROteolysis TArgeting Chimeras (PROTACs) â Past, present and future. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2019,</strong> <em>31 </em>, 15-27. <a href="https://doi.org/10.1016/j.ddtec.2019.01.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2019.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DPROteolysis%252BTArgeting%252BChimeras%252B%252528PROTACs%252529%252B%2525E2%252580%252594%252BPast%25252C%252Bpresent%252Band%252Bfuture%26aulast%3DPettersson%26aufirst%3DMariell%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajesh  Chopra</span>, <span class="hlFld-ContribAuthor ">Amine  Sadok</span>, <span class="hlFld-ContribAuthor ">Ian  Collins</span>. </span><span class="cited-content_cbyCitation_article-title">A critical evaluation of the approaches to targeted protein degradation for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2019,</strong> <em>31 </em>, 5-13. <a href="https://doi.org/10.1016/j.ddtec.2019.02.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2019.02.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2019.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2019.02.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DA%252Bcritical%252Bevaluation%252Bof%252Bthe%252Bapproaches%252Bto%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DChopra%26aufirst%3DRajesh%26date%3D2019%26volume%3D31%26spage%3D5%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan  Scudellari</span>. </span><span class="cited-content_cbyCitation_article-title">Protein-slaying drugs could be the next blockbuster therapies. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2019,</strong> <em>567 </em>
                                    (7748)
                                     , 298-300. <a href="https://doi.org/10.1038/d41586-019-00879-3" title="DOI URL">https://doi.org/10.1038/d41586-019-00879-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/d41586-019-00879-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fd41586-019-00879-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DProtein-slaying%252Bdrugs%252Bcould%252Bbe%252Bthe%252Bnext%252Bblockbuster%252Btherapies%26aulast%3DScudellari%26aufirst%3DMegan%26date%3D2019%26date%3D2019%26volume%3D567%26issue%3D7748%26spage%3D298%26epage%3D300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Steinebach</span>, <span class="hlFld-ContribAuthor ">Hannes  Kehm</span>, <span class="hlFld-ContribAuthor ">Stefanie  Lindner</span>, <span class="hlFld-ContribAuthor ">Lan Phuong  Vu</span>, <span class="hlFld-ContribAuthor ">Simon  KÃ¶pff</span>, <span class="hlFld-ContribAuthor ">Ãlvaro  LÃ³pez MÃ¡rmol</span>, <span class="hlFld-ContribAuthor ">Corinna  Weiler</span>, <span class="hlFld-ContribAuthor ">Karl G.  Wagner</span>, <span class="hlFld-ContribAuthor ">Michaela  Reichenzeller</span>, <span class="hlFld-ContribAuthor ">Jan  KrÃ¶nke</span>, <span class="hlFld-ContribAuthor ">Michael  GÃ¼tschow</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC-mediated crosstalk between E3 ligases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (12)
                                     , 1821-1824. <a href="https://doi.org/10.1039/C8CC09541H" title="DOI URL">https://doi.org/10.1039/C8CC09541H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CC09541H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CC09541H%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DPROTAC-mediated%252Bcrosstalk%252Bbetween%252BE3%252Bligases%26aulast%3DSteinebach%26aufirst%3DChristian%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D12%26spage%3D1821%26epage%3D1824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura  PÃ©rez-Benito</span>, <span class="hlFld-ContribAuthor ">Andrew  Henry</span>, <span class="hlFld-ContribAuthor ">Minos-Timotheos  Matsoukas</span>, <span class="hlFld-ContribAuthor ">Laura  Lopez</span>, <span class="hlFld-ContribAuthor ">Daniel  Pulido</span>, <span class="hlFld-ContribAuthor ">Miriam  Royo</span>, <span class="hlFld-ContribAuthor ">Arnau  CordomÃ­</span>, <span class="hlFld-ContribAuthor ">Gary  Tresadern</span>, <span class="hlFld-ContribAuthor ">Leonardo  Pardo</span>, . </span><span class="cited-content_cbyCitation_article-title">The size matters? A computational tool to design bivalent ligands. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2018,</strong> <em>34 </em>
                                    (22)
                                     , 3857-3863. <a href="https://doi.org/10.1093/bioinformatics/bty422" title="DOI URL">https://doi.org/10.1093/bioinformatics/bty422</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/bty422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbty422%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DThe%252Bsize%252Bmatters%25253F%252BA%252Bcomputational%252Btool%252Bto%252Bdesign%252Bbivalent%252Bligands%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DLaura%26date%3D2018%26date%3D2018%26volume%3D34%26issue%3D22%26spage%3D3857%26epage%3D3863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George M.  Burslem</span>, <span class="hlFld-ContribAuthor ">Philipp  Ottis</span>, <span class="hlFld-ContribAuthor ">Saul  Jaime-Figueroa</span>, <span class="hlFld-ContribAuthor ">Alicia  Morgan</span>, <span class="hlFld-ContribAuthor ">Philipp M.  Cromm</span>, <span class="hlFld-ContribAuthor ">Momar  Toure</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient Synthesis of Immunomodulatory Drug Analogues Enables Exploration of Structure-Degradation Relationships. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (15)
                                     , 1508-1512. <a href="https://doi.org/10.1002/cmdc.201800271" title="DOI URL">https://doi.org/10.1002/cmdc.201800271</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800271%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DEfficient%252BSynthesis%252Bof%252BImmunomodulatory%252BDrug%252BAnalogues%252BEnables%252BExploration%252Bof%252BStructure-Degradation%252BRelationships%26aulast%3DBurslem%26aufirst%3DGeorge%2BM.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D15%26spage%3D1508%26epage%3D1512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (top) Pirin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>)/CRBN (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CI1">5CI1</a>)/PDP ternary complex design model. The PDP can either stabilize a PPI or simply bring the proteins in close proximity depending on the role of the linker. (bottom left) Cartoon representation of the chemical probe <b>1</b> (yellow) bound to recombinant pirin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>). The cloud represents the shape of the binding pocket with key residues shown in black and the metal in orange. Red = oxygen, blue = nitrogen. Hydrogens and solvent are omitted for clarity except the water coordinated to the metal, shown as a red sphere. Both representations were generated using the PyMOL Molecular Graphics System, version 1.8; SchrÃ¶dinger, LLC. (bottom right) Key residues in the binding site and the clear solvent exposed vector for the chemical probe <b>1</b> binding to pirin are shown adapted from an analysis using MOE 2014.09. The ethyl pyrrolidine solubilizing group of chemical probe <b>1</b> was not resolved in the crystal structure and therefore is not drawn in the analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Design and Synthesis of the First Generation PDP <b>3</b> Based on the Chemical Probe <b>1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) PPh<sub>3</sub>, <sup>t</sup>butyl 2-hydroxyacetate, DTBAD, THF, 0 Â°C â RT, 16 h, 75%, (ii) HCO<sub>2</sub>H, DCM, 40 Â°C, 16 h, 54%, (iii) HATU, DIPEA, DMF, <sup>t</sup>butyl (2-(2-aminoethoxy)ethyl)carbamate, RT, 16 h, 81%, (iv) 4 M HCl in dioxane, 0.5 h, 100%; (b) ethyl 4-bromobutanoate, K<sub>2</sub>CO<sub>3</sub>, DMF, RT, 16 h, 37%; (c) Herrmannâs palladacycle,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a><sup><i>t</i></sup>Bu<sub>3</sub>PHBF<sub>4</sub>, MoCO<sub>6</sub>, DBU, 2-(trimethylsilyl)ethan-1-ol, 130 Â°C, 62%; (d) TBAF, THF, RT, 16 h; (e) (i) HATU, DIPEA, DMF, RT, 38% (over 2 steps), (ii) LiOHÂ·H<sub>2</sub>O, MeOH/THF/H<sub>2</sub>O, RT, 48 h, 18%; (f) HATU, DIPEA, DMF, 74%. For the synthesis of hydroxythalidomide <b>2</b>, see ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a>.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Second (<b>10</b>) and Third (<b>16</b>) Generation Probes and Control Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) X = F <b>12</b>, 2-methylquinoline carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine, 18 h quant, (ii) Fe(0), NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O, 90 Â°C, 1 h, quant; (b) 2,3-dihydrobenzo-[<i>b</i>][1,4]-dioxine-6-carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine, RT, 2 h, 85%; (c) (i) SeO<sub>2</sub>, 1,4-dioxane/DMF, reflux, 1 h, (ii) <i>N</i>-Boc-piperazine, DCM, RT, 12 h then NaBH(OAc)<sub>3</sub>, DCM, RT, 2 h, 94% (over 2 steps), (iii) TFA, DCM, RT, 2 h, 69%; (d) (i) <b>2</b>, PPh<sub>3</sub>, <sup>t</sup>butyl 2-hydroxyacetate, DTBAD, THF, 0 Â°C â RT, 16 h, 75%, (ii) HCO<sub>2</sub>H, DCM, 40 Â°C, 16 h, 54%, (iii) HATU, DIPEA, DMF, <sup>t</sup>butyl 3-(2-aminoethoxy)propanoate, RT, 16 h, 72%, (iv) HCO<sub>2</sub>H, DCM, 40 Â°C, 6 h, 93%; (e) HATU, DIPEA, DMF, RT, 16 h, 52%; (f) (i) X = Cl <b>17</b>, 2-methylquinoline carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine 2 h, 88%, (ii) Fe(0), NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O, 90 Â°C, 1 h, quant; (g) 2,3-dihydrobenzo-[<i>b</i>][1,4]-dioxine-6-carboxylic acid, oxalyl chloride, DMF, DCM, RT, 3 h, then pyridine 2 h, 61%; (h) (i) SeO<sub>2</sub>, DMF, 1,4-dioxane, 50 Â°C, 16 h, (ii) <i>N</i>-Boc-piperazine, NaBH<sub>3</sub>CN, AcOH, DMF, 0 Â°C â RT, 16 h, (iii) 4 M HCl in dioxane, MeOH 0 Â°C â RT, 16 h, 32% over 3 steps; (i) 2-[2-(2-hydroxyethoxy)ethoxy]ethyl 4-methylbenzenesulfonate, K<sub>2</sub>CO<sub>3</sub>, DMF, RT, 16 h, 48%; (j) PPh<sub>3</sub>, DTBAD, THF, RT, 2 h, 27%; (k) (i) <b>23</b>, 2,3-dihydro-1,4-benzodioxine-5-carboxylic acid, oxalyl chloride, DMF, DCM, RT, 2 h, then pyridine 48 h, 84%, (ii) Pd/C, EtOH/DCM, H<sub>2</sub> (1 atm), 77%, (iii) 2-methylquinoline carboxylic acid, oxalyl chloride, DMF, DCM, RT, 2 h, then pyridine 16 h, 58%, then same procedure as from <b>18</b>, 6% yield over 5 steps. (l) (i) SeO<sub>2</sub>, 1,4-dioxane/DMF, 50 Â°C, 5.5 h, (ii) <i>N</i>-ethylpiperazine, DCM, RT, 20 h, then NaBH(OAc)<sub>3</sub>, DCM, RT, 2 h, 35% (over 2 steps).</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Representative SPR sensorgram of the third generation PDP <b>16</b> and recombinant pirin. (B) Representative binding curve of PDP <b>16</b> in the CRBN-DDB1 FP-assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Immunoblot of SK-OV-3 human ovarian cancer cells demonstrating the depletion of pirin protein using the third generation PDP <b>16</b> and the time-dependent hook-effect. (B) Immunoblot demonstrating the concentration-dependent depletion of pirin protein after 2 h exposure in SK-OV-3 cells. (C) Capillary electrophoresis and immunoassay were used to quantify the pirin protein expression after 2 h exposure with PDP <b>16</b>, and all values are normalized to vinculin loading control and relative to the measured basal pirin protein expression, all bars represent the arithmetic mean of <i>n</i> = 3 independent biological repeats, error bars are SEM. (D) Proteomics analysis of the third generation PDP <b>16</b> (50 nM) exposure (4 h) in SK-OV-3 cells compared to vehicle control, using a tandem mass tagging (TMT) MS2 protocol on the cell lysate, 8547 quantifiable proteins were identified; each blue dot represents a single quantifiable protein, pirin is marked in red (adjusted <i>p</i> value = 1.4 Ã 10<sup>â4</sup>), <i>p</i> values were calculated using a linear modeling based <i>t</i> test and corrected for multiple comparisons using the BenjaminiâHochberg method to give the <i>p</i>(adj) values shown, dotted lines represent 2-fold depletion of the protein and a <i>p</i>(adj) = 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/medium/jm-2017-01406d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Intracellular competition studies with PDP <b>16</b> and the chemical probes. SK-OV-3 cells were pretreated with increasing concentrations of chemical probe for 4 h before exposing to PDP <b>16</b> for 2 h at the concentrations shown. Cells were then lysed and protein expression analyzed using immunoblot. For clarity, gel images have been cropped where appropriate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01406/20180216/images/large/jm-2017-01406d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01406&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Swinney, D. C.</span><span> </span><span class="NLM_article-title">Biochemical mechanisms of drug action: What does it take for success?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">801</span><span class="NLM_x">â</span> <span class="NLM_lpage">808</span><span class="refDoi">Â DOI: 10.1038/nrd1500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnrd1500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=15340390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=801-808&author=D.+C.+Swinney&title=Biochemical+mechanisms+of+drug+action%3A+What+does+it+take+for+success%3F&doi=10.1038%2Fnrd1500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Biochemical mechanisms of drug action: what does it take for success?</span></div><div class="casAuthors">Swinney, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">801-808</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Drug discovery is extremely difficult.  There are many unanticipated scientific, medical and business challenges to every drug discovery program.  It is important to increase our understanding of the fundamental properties of effective drugs so that we can anticipate potential problems in developing new agents.  This article addresses potential drug discovery and development risks assocd. with the biochem. mechanism of drug action, and proposes simple rules to minimize these risks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXhyDgozFUdrVg90H21EOLACvtfcHk0lj1mixYCaWRXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D&md5=b06bdc6ace3168dcb011679b43b72aed</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1500%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DBiochemical%2520mechanisms%2520of%2520drug%2520action%253A%2520What%2520does%2520it%2520take%2520for%2520success%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D801%26epage%3D808%26doi%3D10.1038%2Fnrd1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">SchÃ¼rmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janning, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziegler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Small-molecule target engagement in cells</span> <span class="citation_source-journal">Cell Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">â</span> <span class="NLM_lpage">441</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2016.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.chembiol.2016.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27049669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqtb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=435-441&author=M.+Sch%C3%BCrmannauthor=P.+Janningauthor=S.+Zieglerauthor=H.+Waldmann&title=Small-molecule+target+engagement+in+cells&doi=10.1016%2Fj.chembiol.2016.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Target Engagement in Cells</span></div><div class="casAuthors">Schuermann, Marc; Janning, Petra; Ziegler, Slava; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-441</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Monitoring how, when, and where small mols. engage their targets inside living cells is a crit. step in chem. biol. and pharmacol. research, because it enables compd. efficacy and confirmation of mode of action to be assessed.  In this mini-review we summarize the currently available methodologies to detect and prove direct target engagement in cells and offer a crit. view of their key advantages and disadvantages.  As the interest of the field shifts toward discovery and validation of high-quality agents, we expect that efforts to develop and refine these types of methodologies will also intensify in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFG52Nxfgik7Vg90H21EOLACvtfcHk0lj1mixYCaWRXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlSqtb4%253D&md5=b81e50faa188d916de61b5dc01294ea8</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCrmann%26aufirst%3DM.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall-molecule%2520target%2520engagement%2520in%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D435%26epage%3D441%26doi%3D10.1016%2Fj.chembiol.2016.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Simon, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niphakis, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">Determining target engagement in living systems</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">â</span> <span class="NLM_lpage">205</span><span class="refDoi">Â DOI: 10.1038/nchembio.1211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.1211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=23508173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWku7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=200-205&author=G.+M.+Simonauthor=M.+J.+Niphakisauthor=B.+F.+Cravatt&title=Determining+target+engagement+in+living+systems&doi=10.1038%2Fnchembio.1211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Determining target engagement in living systems</span></div><div class="casAuthors">Simon, Gabriel M.; Niphakis, Micah J.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-205</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. probes are crit. tools for elucidating the biol. functions of proteins and can lead to new medicines for treating disease.  The pharmacol. validation of protein function requires verification that chem. probes engage their intended targets in vivo.  Here we discuss technologies, both established and emergent, for measuring target engagement in living systems and propose that detg. this parameter should become std. practice for chem. probe and drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5eudh3YJSOLVg90H21EOLACvtfcHk0liTblC6dvHhiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFWku7g%253D&md5=48a97d0d8ff7a7de4f9de0e9c54a1c5c</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1211%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DNiphakis%26aufirst%3DM.%2BJ.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DDetermining%2520target%2520engagement%2520in%2520living%2520systems%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D200%26epage%3D205%26doi%3D10.1038%2Fnchembio.1211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Durham, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanco, M.</span><span> </span><span class="NLM_article-title">Target engagement in lead generation</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">998</span><span class="NLM_x">â</span> <span class="NLM_lpage">1008</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2014.12.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2014.12.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25630223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BC2MrgsFejug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=998-1008&author=T.+B.+Durhamauthor=M.+Blanco&title=Target+engagement+in+lead+generation&doi=10.1016%2Fj.bmcl.2014.12.076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement in lead generation</span></div><div class="casAuthors">Durham Timothy B; Blanco Maria-Jesus</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">998-1008</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmaceutical industry is currently facing multiple challenges, in particular the low number of new drug approvals in spite of the high level of R&D investment.  In order to improve target selection and assess properly the clinical hypothesis, it is important to start building an integrated drug discovery approach during Lead Generation.  This should include special emphasis on evaluating target engagement in the target tissue and linking preclinical to clinical readouts.  In this review, we would like to illustrate several strategies and technologies for assessing target engagement and the value of its application to medicinal chemistry efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTY-Uy19yFbSukdrW1FkXG-fW6udTcc2ebhJrGc_VSbkrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrgsFejug%253D%253D&md5=ecafc397f5418f21d83f7881548ae490</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.12.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.12.076%26sid%3Dliteratum%253Aachs%26aulast%3DDurham%26aufirst%3DT.%2BB.%26aulast%3DBlanco%26aufirst%3DM.%26atitle%3DTarget%2520engagement%2520in%2520lead%2520generation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D998%26epage%3D1008%26doi%3D10.1016%2Fj.bmcl.2014.12.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Sun, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hays, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Periasamy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, R. N.</span><span> </span><span class="NLM_article-title">Monitoring protein interactions in living cells with fluorescence lifetime imaging microscopy</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">371</span><span class="NLM_x">â</span> <span class="NLM_lpage">391</span><span class="refDoi">Â DOI: 10.1016/B978-0-12-391857-4.00019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2FB978-0-12-391857-4.00019-7" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2012&pages=371-391&author=Y.+S.+Sunauthor=N.+M.+Haysauthor=A.+Periasamyauthor=M.+W.+Davidsonauthor=R.+N.+Day&title=Monitoring+protein+interactions+in+living+cells+with+fluorescence+lifetime+imaging+microscopy&doi=10.1016%2FB978-0-12-391857-4.00019-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-391857-4.00019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-391857-4.00019-7%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%2BS.%26aulast%3DHays%26aufirst%3DN.%2BM.%26aulast%3DPeriasamy%26aufirst%3DA.%26aulast%3DDavidson%26aufirst%3DM.%2BW.%26aulast%3DDay%26aufirst%3DR.%2BN.%26atitle%3DMonitoring%2520protein%2520interactions%2520in%2520living%2520cells%2520with%2520fluorescence%2520lifetime%2520imaging%2520microscopy%26jtitle%3DMethods%2520Enzymol.%26date%3D2012%26volume%3D504%26spage%3D371%26epage%3D391%26doi%3D10.1016%2FB978-0-12-391857-4.00019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Robers, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dart, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodroofe, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimprich, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirkland, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machleidt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupcho, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartnett, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmerman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niles, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohana, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slater, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, K. V.</span><span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">10091</span><span class="refDoi">Â DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=10091&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0ligQFPcMABAZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D10091%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Franken, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Childs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G. M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togel, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doce, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhard, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span> </span><span class="NLM_article-title">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1567</span><span class="NLM_x">â</span> <span class="NLM_lpage">1593</span><span class="refDoi">Â DOI: 10.1038/nprot.2015.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnprot.2015.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26379230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1567-1593&author=H.+Frankenauthor=T.+Mathiesonauthor=D.+Childsauthor=G.+M.+A.+Sweetmanauthor=T.+Wernerauthor=I.+Togelauthor=C.+Doceauthor=S.+Gadeauthor=M.+Bantscheffauthor=G.+Drewesauthor=F.+B.+Reinhardauthor=W.+Huberauthor=M.+M.+Savitski&title=Thermal+proteome+profiling+for+unbiased+identification+of+direct+and+indirect+drug+targets+using+multiplexed+quantitative+mass+spectrometry&doi=10.1038%2Fnprot.2015.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry</span></div><div class="casAuthors">Franken, Holger; Mathieson, Toby; Childs, Dorothee; Sweetman, Gavain M. A.; Werner, Thilo; Toegel, Ina; Doce, Carola; Gade, Stephan; Bantscheff, Marcus; Drewes, Gerard; Reinhard, Friedrich B. M.; Huber, Wolfgang; Savitski, Mikhail M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1567-1593</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The direct detection of drug-protein interactions in living cells is a major challenge in drug discovery research.  Recently, the authors introduced an approach termed thermal proteome profiling (TPP), which enables the monitoring of changes in protein thermal stability across the proteome using quant. mass spectrometry.  The authors detd. the intracellular thermal profiles for up to 7000 proteins, and by comparing profiles derived from cultured mammalian cells in the presence or absence of a drug it was possible to identify direct and indirect targets of drugs in living cells in an unbiased manner.  Here the authors demonstrate the complete workflow using the histone deacetylase inhibitor panobinostat.  The key to this approach is the use of isobaric tandem mass tag 10-plex (TMT10) reagents to label digested protein samples corresponding to each temp. point in the melting curve so that the samples can be analyzed by multiplexed quant. mass spectrometry.  Important steps in the bioinformatic anal. include data normalization, melting curve fitting and statistical significance detn. of compd. concn.-dependent changes in protein stability.  All anal. tools are made freely available as R and Python packages.  The workflow can be completed in 2 wk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS4LU9YprZqLVg90H21EOLACvtfcHk0ligQFPcMABAZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzN&md5=d9c225616a35fc5dcc97db234452d7e1</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.101%26sid%3Dliteratum%253Aachs%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DChilds%26aufirst%3DD.%26aulast%3DSweetman%26aufirst%3DG.%2BM.%2BA.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DTogel%26aufirst%3DI.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DGade%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520proteome%2520profiling%2520for%2520unbiased%2520identification%2520of%2520direct%2520and%2520indirect%2520drug%2520targets%2520using%2520multiplexed%2520quantitative%2520mass%2520spectrometry%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D1567%26epage%3D1593%26doi%3D10.1038%2Fnprot.2015.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhard, F. B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franken, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jafari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dovega, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaeger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Tracking cancer drugs in living cells by thermal profiling of the proteome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255784</span><span class="refDoi">Â DOI: 10.1126/science.1255784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1126%2Fscience.1255784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25278616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=1255784&author=M.+M.+Savitskiauthor=F.+B.+M.+Reinhardauthor=H.+Frankenauthor=T.+Wernerauthor=M.+F.+Savitskiauthor=D.+Eberhardauthor=D.+M.+Molinaauthor=R.+Jafariauthor=R.+B.+Dovegaauthor=S.+Klaegerauthor=B.+Kusterauthor=P.+Nordlundauthor=M.+Bantscheffauthor=G.+Drewes&title=Tracking+cancer+drugs+in+living+cells+by+thermal+profiling+of+the+proteome&doi=10.1126%2Fscience.1255784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Tracking cancer drugs in living cells by thermal profiling of the proteome</span></div><div class="casAuthors">Savitski Mikhail M; Reinhard Friedrich B M; Franken Holger; Werner Thilo; Savitski Maria Falth; Eberhard Dirk; Bantscheff Marcus; Drewes Gerard; Martinez Molina Daniel; Jafari Rozbeh; Dovega Rebecca Bakszt; Klaeger Susan; Kuster Bernhard; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6205</span>),
    <span class="NLM_cas:pages">1255784</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The thermal stability of proteins can be used to assess ligand binding in living cells.  We have generalized this concept by determining the thermal profiles of more than 7000 proteins in human cells by means of mass spectrometry.  Monitoring the effects of small-molecule ligands on the profiles delineated more than 50 targets for the kinase inhibitor staurosporine.  We identified the heme biosynthesis enzyme ferrochelatase as a target of kinase inhibitors and suggest that its inhibition causes the phototoxicity observed with vemurafenib and alectinib.  Thermal shifts were also observed for downstream effectors of drug treatment.  In live cells, dasatinib induced shifts in BCR-ABL pathway proteins, including CRK/CRKL.  Thermal proteome profiling provides an unbiased measure of drug-target engagement and facilitates identification of markers for drug efficacy and toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS0ur3d8ba-Bu2mnZh2OgS0fW6udTcc2eYFEQHINw5PS7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7ntVKgsg%253D%253D&md5=493a89ad107e656e0f0786ff5c8e5f63</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1126%2Fscience.1255784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1255784%26sid%3Dliteratum%253Aachs%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DReinhard%26aufirst%3DF.%2BB.%2BM.%26aulast%3DFranken%26aufirst%3DH.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DSavitski%26aufirst%3DM.%2BF.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DDovega%26aufirst%3DR.%2BB.%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DTracking%2520cancer%2520drugs%2520in%2520living%2520cells%2520by%2520thermal%2520profiling%2520of%2520the%2520proteome%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D1255784%26doi%3D10.1126%2Fscience.1255784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Huber, K. V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olek, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¼ller, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, C. S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colinge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1055</span><span class="NLM_x">â</span> <span class="NLM_lpage">1057</span><span class="refDoi">Â DOI: 10.1038/nmeth.3590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnmeth.3590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26389571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1055-1057&author=K.+V.+M.+Huberauthor=K.+M.+Olekauthor=A.+C.+M%C3%BCllerauthor=C.+S.+H.+Tanauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furga&title=Proteome-wide+drug+and+metabolite+interaction+mapping+by+thermal-stability+profiling&doi=10.1038%2Fnmeth.3590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span></div><div class="casAuthors">Huber, Kilian V. M.; Olek, Karin M.; Mueller, Andre C.; Tan, Chris Soon Heng; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1055-1057</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal stabilization of proteins after ligand binding provides an efficient means to assess the binding of small mols. to proteins.  We show here that in combination with quant. mass spectrometry, the approach allows for the systematic survey of protein engagement by cellular metabolites and drugs.  We profiled the targets of the drugs methotrexate and (S)-crizotinib and the metabolite 2'3'-cGAMP in intact cells and identified the 2'3'-cGAMP cognate transmembrane receptor STING, involved in immune signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrow66ugdg1oLVg90H21EOLACvtfcHk0ljHUVVdxgutmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI&md5=619a2c08c4f151f4e3ab8bd8fc45c2b6</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3590%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DOlek%26aufirst%3DK.%2BM.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DTan%26aufirst%3DC.%2BS.%2BH.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProteome-wide%2520drug%2520and%2520metabolite%2520interaction%2520mapping%2520by%2520thermal-stability%2520profiling%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1055%26epage%3D1057%26doi%3D10.1038%2Fnmeth.3590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Mateus, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MÃ¤Ã¤ttÃ¤, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span> </span><span class="NLM_article-title">Thermal proteome profiling: unbiased assessment of protein state through heat-induced stability changes</span> <span class="citation_source-journal">Proteome Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="refDoi">Â DOI: 10.1186/s12953-017-0122-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1186%2Fs12953-017-0122-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28652855" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=13&author=A.+Mateusauthor=T.+A.+M%C3%A4%C3%A4tt%C3%A4author=M.+M.+Savitski&title=Thermal+proteome+profiling%3A+unbiased+assessment+of+protein+state+through+heat-induced+stability+changes&doi=10.1186%2Fs12953-017-0122-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1186%2Fs12953-017-0122-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12953-017-0122-4%26sid%3Dliteratum%253Aachs%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DM%25C3%25A4%25C3%25A4tt%25C3%25A4%26aufirst%3DT.%2BA.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26atitle%3DThermal%2520proteome%2520profiling%253A%2520unbiased%2520assessment%2520of%2520protein%2520state%2520through%2520heat-induced%2520stability%2520changes%26jtitle%3DProteome%2520Sci.%26date%3D2017%26volume%3D15%26spage%3D13%26doi%3D10.1186%2Fs12953-017-0122-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozarich, J. W.</span><span> </span><span class="NLM_article-title">Activity-based protein profiling: from enzyme chemistry to proteomic chemistry</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">â</span> <span class="NLM_lpage">414</span><span class="refDoi">Â DOI: 10.1146/annurev.biochem.75.101304.124125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1146%2Fannurev.biochem.75.101304.124125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=18366325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2008&pages=383-414&author=B.+F.+Cravattauthor=A.+T.+Wrightauthor=J.+W.+Kozarich&title=Activity-based+protein+profiling%3A+from+enzyme+chemistry+to+proteomic+chemistry&doi=10.1146%2Fannurev.biochem.75.101304.124125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: From enzyme chemistry to proteomic chemistry</span></div><div class="casAuthors">Cravatt, Benjamin F.; Wright, Aaron T.; Kozarich, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">383-414</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Genome sequencing projects have provided researchers with a complete inventory of the predicted proteins produced by eukaryotic and prokaryotic organisms.  Assignment of functions to these proteins represents one of the principal challenges for the field of proteomics.  Activity-based protein profiling (ABPP) has emerged as a powerful chem. proteomic strategy to characterize enzyme function directly in native biol. systems on a global scale.  Here, the authors review the basic technol. of ABPP, the enzyme classes addressable by this method, and the biol. discoveries attributable to its application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQFnUWDsqzOLVg90H21EOLACvtfcHk0ljHUVVdxgutmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D&md5=2f2c471169d2473f7024eeb41d4eaecd</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.75.101304.124125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.75.101304.124125%26sid%3Dliteratum%253Aachs%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DWright%26aufirst%3DA.%2BT.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520from%2520enzyme%2520chemistry%2520to%2520proteomic%2520chemistry%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2008%26volume%3D77%26spage%3D383%26epage%3D414%26doi%3D10.1146%2Fannurev.biochem.75.101304.124125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lanning, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitby, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dix, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douhan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hett, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joslyn, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niessen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnute, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hulce, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whiteley, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayward, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">â</span> <span class="NLM_lpage">769</span><span class="refDoi">Â DOI: 10.1038/nchembio.1582</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.1582" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25038787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=760-769&author=B.+R.+Lanningauthor=L.+R.+Whitbyauthor=M.+M.+Dixauthor=J.+Douhanauthor=A.+M.+Gilbertauthor=E.+C.+Hettauthor=T.+O.+Johnsonauthor=C.+Joslynauthor=J.+C.+Kathauthor=S.+Niessenauthor=L.+R.+Robertsauthor=M.+E.+Schnuteauthor=C.+Wangauthor=J.+J.+Hulceauthor=B.+Weiauthor=L.+O.+Whiteleyauthor=M.+M.+Haywardauthor=B.+F.+Cravatt&title=A+road+map+to+evaluate+the+proteome-wide+selectivity+of+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1582"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span></div><div class="casAuthors">Lanning, Bryan R.; Whitby, Landon R.; Dix, Melissa M.; Douhan, John; Gilbert, Adam M.; Hett, Erik C.; Johnson, Theodore O.; Joslyn, Chris; Kath, John C.; Niessen, Sherry; Roberts, Lee R.; Schnute, Mark E.; Wang, Chu; Hulce, Jonathan J.; Wei, Baoxian; Whiteley, Laurence O.; Hayward, Matthew M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">760-767</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinases are principal components of signal transduction pathways and the focus of intense basic and drug discovery research.  Irreversible inhibitors that covalently modify non-catalytic cysteines in kinase active sites have emerged as valuable probes and approved drugs.  Many protein classes, however, have functional cysteines, and therefore understanding the proteome-wide selectivity of covalent kinase inhibitors is imperative.  Here, we accomplish this objective using activity-based protein profiling coupled with quant. MS to globally map the targets, both specific and nonspecific, of covalent kinase inhibitors in human cells.  Many of the specific off-targets represent nonkinase proteins that, notably, have conserved active site cysteines.  We define windows of selectivity for covalent kinase inhibitors and show that, when these windows are exceeded, rampant proteome-wide reactivity and kinase target-independent cell death conjointly occur.  Our findings, taken together, provide an exptl. road map to illuminate opportunities and surmount challenges for the development of covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMAiUlEg9R5rVg90H21EOLACvtfcHk0ljHUVVdxgutmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO&md5=6f17b0d1b00e088f37c511eea577d290</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1582%26sid%3Dliteratum%253Aachs%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DWhitby%26aufirst%3DL.%2BR.%26aulast%3DDix%26aufirst%3DM.%2BM.%26aulast%3DDouhan%26aufirst%3DJ.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DHett%26aufirst%3DE.%2BC.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DJoslyn%26aufirst%3DC.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DSchnute%26aufirst%3DM.%2BE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHulce%26aufirst%3DJ.%2BJ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWhiteley%26aufirst%3DL.%2BO.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DA%2520road%2520map%2520to%2520evaluate%2520the%2520proteome-wide%2520selectivity%2520of%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D760%26epage%3D769%26doi%3D10.1038%2Fnchembio.1582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit7c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Dubach, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuccarese, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giedt, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meimetis, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinegoni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissleder, R.</span><span> </span><span class="NLM_article-title">Quantitating drug-target engagement in single cells in vitro and in vivo</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">168</span><span class="NLM_x">â</span> <span class="NLM_lpage">173</span><span class="refDoi">Â DOI: 10.1038/nchembio.2248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.2248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27918558" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=168-173&author=J.+M.+Dubachauthor=E.+Kimauthor=K.+Yangauthor=M.+Cuccareseauthor=R.+J.+Giedtauthor=L.+G.+Meimetisauthor=C.+Vinegoniauthor=R.+Weissleder&title=Quantitating+drug-target+engagement+in+single+cells+in+vitro+and+in+vivo&doi=10.1038%2Fnchembio.2248"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7c&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2248%26sid%3Dliteratum%253Aachs%26aulast%3DDubach%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DCuccarese%26aufirst%3DM.%26aulast%3DGiedt%26aufirst%3DR.%2BJ.%26aulast%3DMeimetis%26aufirst%3DL.%2BG.%26aulast%3DVinegoni%26aufirst%3DC.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DQuantitating%2520drug-target%2520engagement%2520in%2520single%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D13%26spage%3D168%26epage%3D173%26doi%3D10.1038%2Fnchembio.2248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bondeson, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mares, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, I. E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routly, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faelth-Savitski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casillas, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votta, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Besten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harling, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">611</span><span class="NLM_x">â</span> <span class="NLM_lpage">617</span><span class="refDoi">Â DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lgYOIye0_n90w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuhira, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ujikawa, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimokawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sano, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koyama, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teratani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span> </span><span class="NLM_article-title">In vivo knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">292</span><span class="NLM_x">, </span> <span class="NLM_fpage">4556</span><span class="NLM_x">â</span> <span class="NLM_lpage">4570</span><span class="refDoi">Â DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor Î± (ERÎ±), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERÎ± levels in tumor xenografts and suppressed the growth of ERÎ±-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERÎ± via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0lgYOIye0_n90w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, P.</span><span> </span><span class="NLM_article-title">DCAFs, the missing link of the CUL4-DDB1 ubiquitin ligase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">775</span><span class="NLM_x">â</span> <span class="NLM_lpage">780</span><span class="refDoi">Â DOI: 10.1016/j.molcel.2007.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.molcel.2007.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=17588513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFSru78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=775-780&author=J.+Leeauthor=P.+Zhou&title=DCAFs%2C+the+missing+link+of+the+CUL4-DDB1+ubiquitin+ligase&doi=10.1016%2Fj.molcel.2007.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">DCAFs, the missing link of the CUL4-DDB1 ubiquitin ligase</span></div><div class="casAuthors">Lee, Jennifer; Zhou, Pengbo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">775-780</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  CUL4-DDB1 ubiquitin ligase regulates cell proliferation, survival, DNA repair, and genomic integrity through targeted ubiquitination of key regulators; yet, the substrate receptors that dictate the specificity of this ubiquitination machinery have been largely unknown.  Recent work has identified a family of DDB1 and CUL4-assocd. factors (DCAFs) as substrate receptors, implicating a broad spectrum of cellular processes regulated by CUL4-DDB1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWyUZvvs-r7Vg90H21EOLACvtfcHk0lj9X-qeyQkDyg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFSru78%253D&md5=c18a4a72c16a452add79520a554acdc8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DDCAFs%252C%2520the%2520missing%2520link%2520of%2520the%2520CUL4-DDB1%2520ubiquitin%2520ligase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D26%26spage%3D775%26epage%3D780%26doi%3D10.1016%2Fj.molcel.2007.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessum, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rye, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasqua, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilding, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlands, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâFee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henley, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Te Poele, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellegrino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raynaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Montfort, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eccles, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span> </span><span class="NLM_article-title">Discovery of a chemical probe bisamide (CCT251236): an orally bioavailable efficacious pirin ligand from a heat shock transcription factor 1 (HSF1) phenotypic screen</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">â</span> <span class="NLM_lpage">201</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=180-201&author=M.+D.+Cheesemanauthor=N.+E.+Chessumauthor=C.+S.+Ryeauthor=A.+E.+Pasquaauthor=M.+J.+Tuckerauthor=B.+Wildingauthor=L.+E.+Evansauthor=S.+Lepriauthor=M.+Richardsauthor=S.+Y.+Sharpauthor=S.+Aliauthor=M.+Rowlandsauthor=L.+O%E2%80%99Feeauthor=A.+Miahauthor=A.+Hayesauthor=A.+T.+Henleyauthor=M.+Powersauthor=R.+Te+Poeleauthor=E.+De+Billyauthor=L.+Pellegrinoauthor=F.+Raynaudauthor=R.+Burkeauthor=R.+L.+van+Montfortauthor=S.+A.+Ecclesauthor=P.+Workmanauthor=K.+Jones&title=Discovery+of+a+chemical+probe+bisamide+%28CCT251236%29%3A+an+orally+bioavailable+efficacious+pirin+ligand+from+a+heat+shock+transcription+factor+1+%28HSF1%29+phenotypic+screen&doi=10.1021%2Facs.jmedchem.6b01055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01055%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DChessum%26aufirst%3DN.%2BE.%26aulast%3DRye%26aufirst%3DC.%2BS.%26aulast%3DPasqua%26aufirst%3DA.%2BE.%26aulast%3DTucker%26aufirst%3DM.%2BJ.%26aulast%3DWilding%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Fee%26aufirst%3DL.%26aulast%3DMiah%26aufirst%3DA.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHenley%26aufirst%3DA.%2BT.%26aulast%3DPowers%26aufirst%3DM.%26aulast%3DTe%2BPoele%26aufirst%3DR.%26aulast%3DDe%2BBilly%26aufirst%3DE.%26aulast%3DPellegrino%26aufirst%3DL.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520a%2520chemical%2520probe%2520bisamide%2520%2528CCT251236%2529%253A%2520an%2520orally%2520bioavailable%2520efficacious%2520pirin%2520ligand%2520from%2520a%2520heat%2520shock%2520transcription%2520factor%25201%2520%2528HSF1%2529%2520phenotypic%2520screen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D180%26epage%3D201%26doi%3D10.1021%2Facs.jmedchem.6b01055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehmani, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Serrano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span> </span><span class="NLM_article-title">Pirin is an iron-dependent redox regulator of NF-ÎºB</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">9722</span><span class="NLM_x">â</span> <span class="NLM_lpage">9727</span><span class="refDoi">Â DOI: 10.1073/pnas.1221743110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1073%2Fpnas.1221743110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=23716661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=9722-9727&author=F.+Liuauthor=I.+Rehmaniauthor=S.+Esakiauthor=R.+Fuauthor=L.+Chenauthor=V.+de+Serranoauthor=A.+Liu&title=Pirin+is+an+iron-dependent+redox+regulator+of+NF-%CE%BAB&doi=10.1073%2Fpnas.1221743110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin is an iron-dependent redox regulator of NF-ÎºB</span></div><div class="casAuthors">Liu, Fange; Rehmani, Imran; Esaki, Shingo; Fu, Rong; Chen, Lirong; de Serrano, Vesna; Liu, Aimin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9722-9727, S9722/1-S9722/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pirin is a nuclear nonheme Fe protein of unknown function present in all human tissues.  Here the authors describe that pirin may act as a redox sensor for the nuclear factor ÎºB (NF-ÎºB) transcription factor, a crit. mediator of intracellular signaling that has been linked to cellular responses to proinflammatory signals and controls the expression of a vast array of genes involved in immune and stress responses.  Pirin's regulatory effect was tested with several metals and at different oxidns. states, and the authors' spectroscopic results show that only the ferric form of pirin substantially facilitates binding of NF-ÎºB proteins to target ÎºB genes, a finding that suggests that pirin performs a redox-sensing role in NF-ÎºB regulation.  The mol. mechanism of such a metal identity- and redox state-dependent regulation is revealed by the authors' structural studies of pirin.  The ferrous and ferric pirin proteins differ only by one electron, yet they have distinct conformations.  The Fe center is shown to play an allosteric role on an Î²-shaped surface area that has two distinct conformations based on the identity and the formal redox state of the metal.  The authors show that the R-shaped area composes the interface for pirin-NF-ÎºB binding that is responsible for modulation of NF-ÎºB's DNA-binding properties.  The nonheme Fe protein pirin is proposed to serve as a reversible functional switch that enables NF-ÎºB to respond to changes in the redox levels of the cell nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Tm-I0Q4mD7Vg90H21EOLACvtfcHk0lj9X-qeyQkDyg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF&md5=d81627e3a8aa09ee6a46a8557719394d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221743110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221743110%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRehmani%26aufirst%3DI.%26aulast%3DEsaki%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3Dde%2BSerrano%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DA.%26atitle%3DPirin%2520is%2520an%2520iron-dependent%2520redox%2520regulator%2520of%2520NF-%25CE%25BAB%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D9722%26epage%3D9727%26doi%3D10.1073%2Fpnas.1221743110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Dunwell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purvis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, S.</span><span> </span><span class="NLM_article-title">Cupins: the most functionally diverse protein superfamily?</span> <span class="citation_source-journal">Phytochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">â</span> <span class="NLM_lpage">17</span><span class="refDoi">Â DOI: 10.1016/j.phytochem.2003.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.phytochem.2003.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=14697267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2004&pages=7-17&author=J.+M.+Dunwellauthor=A.+Purvisauthor=S.+Khuri&title=Cupins%3A+the+most+functionally+diverse+protein+superfamily%3F&doi=10.1016%2Fj.phytochem.2003.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cupins: the most functionally diverse protein superfamily?</span></div><div class="casAuthors">Dunwell, Jim M.; Purvis, Alan; Khuri, Sawsan</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry (Elsevier)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-17</span>CODEN:
                <span class="NLM_cas:coden">PYTCAS</span>;
        ISSN:<span class="NLM_cas:issn">0031-9422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  The cupin superfamily of proteins, named on the basis of a conserved Î²-barrel fold ('cupa' is the Latin term for a small barrel), was originally discovered using a conserved motif found within germin and germin-like proteins from higher plants.  Previous anal. of cupins had identified some 18 different functional classes that range from single-domain bacterial enzymes such as isomerases and epimerases involved in the modification of cell wall carbohydrates, through to two-domain bicupins such as the desiccation-tolerant seed storage globulins, and multidomain transcription factors including one linked to the nodulation response in legumes.  Recent advances in comparative genomics, and the resoln. of many more 3-D structures have now revealed that the largest subset of the cupin superfamily is the 2-oxyglutarate-Fe2+ dependent dioxygenases.  The substrates for this subclass of enzyme are many and varied and in total amt. to probably 50-100 different biochem. reactions, including several involved in plant growth and development.  Although the majority of enzymic cupins contain iron as an active site metal, other members contain either copper, zinc, cobalt, nickel or manganese ions as a cofactor, with each cofactor allowing a different type of chem. to occur within the conserved tertiary structure.  This review discusses the range of structures and functions found in this most diverse of superfamilies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGLBE2G_DAE7Vg90H21EOLACvtfcHk0lgs3GrOE0zX7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D&md5=8af457482acd815ffc0d8ff05bf247c7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2003.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2003.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26aulast%3DPurvis%26aufirst%3DA.%26aulast%3DKhuri%26aufirst%3DS.%26atitle%3DCupins%253A%2520the%2520most%2520functionally%2520diverse%2520protein%2520superfamily%253F%26jtitle%3DPhytochemistry%26date%3D2004%26volume%3D65%26spage%3D7%26epage%3D17%26doi%3D10.1016%2Fj.phytochem.2003.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Komai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niwa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasazawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simizu, S.</span><span> </span><span class="NLM_article-title">Pirin regulates epithelial to mesenchymal transition independently of BCL3-SLUG signaling</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">589</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">â</span> <span class="NLM_lpage">743</span><span class="refDoi">Â DOI: 10.1016/j.febslet.2015.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.febslet.2015.01.040" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=738-743&author=K.+Komaiauthor=Y.+Niwaauthor=Y.+Sasazawaauthor=S.+Simizu&title=Pirin+regulates+epithelial+to+mesenchymal+transition+independently+of+BCL3-SLUG+signaling&doi=10.1016%2Fj.febslet.2015.01.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DKomai%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DY.%26aulast%3DSasazawa%26aufirst%3DY.%26aulast%3DSimizu%26aufirst%3DS.%26atitle%3DPirin%2520regulates%2520epithelial%2520to%2520mesenchymal%2520transition%2520independently%2520of%2520BCL3-SLUG%2520signaling%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D738%26epage%3D743%26doi%3D10.1016%2Fj.febslet.2015.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span> </span><span class="NLM_article-title">Protein knockdown technology: Application of ubiquitin ligase to cancer therapy</span> <span class="citation_source-journal">Curr. Cancer Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">136</span><span class="NLM_x">â</span> <span class="NLM_lpage">146</span><span class="refDoi">Â DOI: 10.2174/1568009616666151112122502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.2174%2F1568009616666151112122502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26560118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=136-146&author=N.+Ohokaauthor=N.+Shibataauthor=T.+Hattoriauthor=M.+Naito&title=Protein+knockdown+technology%3A+Application+of+ubiquitin+ligase+to+cancer+therapy&doi=10.2174%2F1568009616666151112122502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Knockdown Technology: Application of Ubiquitin Ligase to Cancer Therapy</span></div><div class="casAuthors">Ohoka, Nobumichi; Shibata, Norihito; Hattori, Takayuki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-146</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Selective degrdn. of pathogenic proteins by small mols. in cells is a novel approach for development of therapeutic agents against various diseases, including cancer.  We and others have developed a protein knockdown technol. with a series of hybrid small compds., called SNIPERs (Specific and Nongenetic IAP-dependent Protein ERasers); and peptidic chimeric mols., called PROTACs (proteolysis-targeting chimeric mols.), which induce selective degrdn. of target proteins via the ubiquitin-proteasome pathway.  These compds. include two different ligands connected by a linker; one is a ligand for a ubiquitin ligase and the other is a ligand for the target protein, which are expected to crosslink these proteins in cells.  Theor., any cytosolic protein can be targeted for degrdn. by this technol.  To date, several SNIPERs and PROTACs against various oncogenic proteins have been developed, which specifically induce polyubiquitylation and proteasomal degrdn. of the oncogenic proteins, resulting in cell death, growth arrest, or impaired migration of cancer cells.  Thus, this protein knockdown technol. has a great potential for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmS_kGOWfeuLVg90H21EOLACvtfcHk0lgs3GrOE0zX7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D&md5=d78f1ae70ea89843d9f946454cc95058</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.2174%2F1568009616666151112122502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009616666151112122502%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DProtein%2520knockdown%2520technology%253A%2520Application%2520of%2520ubiquitin%2520ligase%2520to%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2016%26volume%3D16%26spage%3D136%26epage%3D146%26doi%3D10.2174%2F1568009616666151112122502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Collins, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopra, R.</span><span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">474</span><span class="NLM_x">, </span> <span class="NLM_fpage">1127</span><span class="NLM_x">â</span> <span class="NLM_lpage">1147</span><span class="refDoi">Â DOI: 10.1042/BCJ20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1042%2FBCJ20160762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28298557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin-proteasome+pathway&doi=10.1042%2FBCJ20160762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Collins, Ian; Wang, Hannah; Caldwell, John J.; Chopra, Raj</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1127-1147</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Manipulation of the ubiquitin-proteasome system to achieve targeted degrdn. of proteins within cells using chem. tools and drugs has the potential to transform pharmacol. and therapeutic approaches in cancer and other diseases.  An increased understanding of the mol. mechanism of thalidomide and its analogs following their clin. use has unlocked small-mol. modulation of the substrate specificity of the E3 ligase cereblon (CRBN), which in turn has resulted in the advancement of new immunomodulatory drugs (IMiDs) into the clinic.  The degrdn. of multiple context-specific proteins by these pleiotropic small mols. provides a means to uncover new cell biol. and to generate future drug mols. against currently undruggable targets.  In parallel, the development of larger bifunctional mols. that bring together highly specific protein targets in complexes with CRBN, von Hippel-Lindau, or other E3 ligases to promote ubiquitin-dependent degrdn. has progressed to generate selective chem. compds. with potent effects in cells and in vivo models, providing valuable tools for biol. target validation and with future potential for therapeutic use.  In this review, we survey recent breakthroughs achieved in these two complementary methods and the discovery of new modes of direct and indirect engagement of target proteins with the proteasome.  We discuss the exptl. characterization that validates the use of mols. that promote protein degrdn. as chem. tools, the preclin. and clin. examples disclosed to date, and the future prospects for this exciting area of chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkoPeMHzjTbVg90H21EOLACvtfcHk0lgs3GrOE0zX7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D&md5=8c9ccac22f5088d776b81203dc8ebf2b</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin-proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2FBCJ20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Small-molecule PROTACS: New approaches to protein degradation</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1966</span><span class="NLM_x">â</span> <span class="NLM_lpage">1973</span><span class="refDoi">Â DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+New+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lh3tmsbqwPBxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520New%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Induced protein degradation: An emerging drug discovery paradigm</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">â</span> <span class="NLM_lpage">114</span><span class="refDoi">Â DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+An+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15dR"><div class="casContent"><span class="casTitleNuber">15d</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lh3tmsbqwPBxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit15d&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520An%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="note"><p class="first last">For example,</p></div><div class="NLM_citation" id="rightTab-cit16a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1770</span><span class="NLM_x">â</span> <span class="NLM_lpage">1777</span><span class="refDoi">Â DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16aR"><div class="casContent"><span class="casTitleNuber">16a</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1Î±.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lh3tmsbqwPBxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit16a&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit16b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altieri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crew, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">â</span> <span class="NLM_lpage">763</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16bR"><div class="casContent"><span class="casTitleNuber">16b</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhOBS6AyAFyTg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit16b&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Galdeano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadd, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scaffidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Molle, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birced, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Structure guided design and optimization of small molecules targeting the protein-protein interaction between the von hippel-lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8657</span><span class="NLM_x">â</span> <span class="NLM_lpage">8663</span><span class="refDoi">Â DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure+guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure%2520guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Demizu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hattori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohoka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, M.</span><span> </span><span class="NLM_article-title">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">4865</span><span class="NLM_x">â</span> <span class="NLM_lpage">4869</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2016.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27666635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4865-4869&author=Y.+Demizuauthor=N.+Shibataauthor=T.+Hattoriauthor=N.+Ohokaauthor=H.+Motoiauthor=T.+Misawaauthor=T.+Shodaauthor=M.+Naitoauthor=M.+Kurihara&title=Development+of+BCR-ABL+degradation+inducers+via+the+conjugation+of+an+imatinib+derivative+and+a+cIAP1+ligand&doi=10.1016%2Fj.bmcl.2016.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span></div><div class="casAuthors">Demizu, Yosuke; Shibata, Norihito; Hattori, Takayuki; Ohoka, Nobumichi; Motoi, Hiromi; Misawa, Takashi; Shoda, Takuji; Naito, Mikihiko; Kurihara, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4865-4869</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The manipulation of protein stability with small mols. has great potential as a technique for aiding the development of clin. therapies, including treatments for cancer.  In this study, BCR-ABL protein degrdn. inducers called SNIPER(ABL) (Specific and Nongenetic inhibitors of apoptosis protein [IAP]-dependent Protein Erasers) were developed.  The designed mols. contained two biol. active scaffolds: one was an imatinib deriv. that binds to BCL-ABL and the other was a Me bestatin that binds to cellular IAP 1 (cIAP1).  The hybrid mols., SNIPER(ABL), were expected to recruit BCR-ABL to cIAP1 for removal by proteasomes.  In fact, SNIPER(ABL) induced the degrdn. of BCR-ABL protein and a subsequent redn. in cell growth.  Thus, the degrdn. of BCR-ABL by SNIPER(ABL) is one potential strategy for treating BCR-ABL driven chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLT6dtmVPL47Vg90H21EOLACvtfcHk0lhOBS6AyAFyTg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK&md5=31c045087293acf60932a9d6c4f38504</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMotoi%26aufirst%3DH.%26aulast%3DMisawa%26aufirst%3DT.%26aulast%3DShoda%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DKurihara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520BCR-ABL%2520degradation%2520inducers%2520via%2520the%2520conjugation%2520of%2520an%2520imatinib%2520derivative%2520and%2520a%2520cIAP1%2520ligand%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4865%26epage%3D4869%26doi%3D10.1016%2Fj.bmcl.2016.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Winter, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">348</span><span class="NLM_x">, </span> <span class="NLM_fpage">1376</span><span class="NLM_x">â</span> <span class="NLM_lpage">1381</span><span class="refDoi">Â DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lityAXcl9gOPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><div class="note"><p class="first last">A recent study demonstrated a method to determine which E3 ligase can be manipulated for protein degradation:</p></div><span class="NLM_contrib-group">Ottis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cromm, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Assessing different E3 ligases for small molecule induced protein ubiquitination and degradation</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2570</span><span class="NLM_x">â</span> <span class="NLM_lpage">2578</span><span class="refDoi">Â DOI: 10.1021/acschembio.7b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2570-2578&author=P.+Ottisauthor=M.+Toureauthor=P.+M.+Crommauthor=E.+Koauthor=J.+L.+Gustafsonauthor=C.+M.+Crews&title=Assessing+different+E3+ligases+for+small+molecule+induced+protein+ubiquitination+and+degradation&doi=10.1021%2Facschembio.7b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing Different E3 Ligases for Small Molecule Induced Protein Ubiquitination and Degradation</span></div><div class="casAuthors">Ottis, Philipp; Toure, Momar; Cromm, Philipp M.; Ko, Eunhwa; Gustafson, Jeffrey L.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2570-2578</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol., the recruitment of E3 ubiquitin ligases to induce the degrdn. of a protein target, is rapidly impacting chem. biol., as well as modern drug development.  Here, we explore the breadth of this approach by evaluating different E3 ubiquitin ligases engineered in their substrate binding domains to accept a recruiting ligand.  Five out of six E3 ligases were found to be amenable to recruitment for target degrdn.  Taking advantage of the tight spatio-temporal control of inducing ubiquitination on a pre-selected target in living cells, we focused on two of the engineered E3 ligases, Î²TRCP and parkin, to characterize their ability to induce ubiquitination in comparison with the PROTAC-recruited endogenous E3 ligases VHL and cereblon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhBU1NFnjNbVg90H21EOLACvtfcHk0lityAXcl9gOPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O&md5=b643d34cb33b330a7658b8b7feb2c6cc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00485%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAssessing%2520different%2520E3%2520ligases%2520for%2520small%2520molecule%2520induced%2520protein%2520ubiquitination%2520and%2520degradation%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2570%26epage%3D2578%26doi%3D10.1021%2Facschembio.7b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Cyrus, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehenkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swanson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K.</span><span> </span><span class="NLM_article-title">Impact of linker length on the activity of PROTACs</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">â</span> <span class="NLM_lpage">364</span><span class="refDoi">Â DOI: 10.1039/C0MB00074D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1039%2FC0MB00074D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=20922213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslGhsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=359-364&author=K.+Cyrusauthor=M.+Wehenkelauthor=E.+Choiauthor=H.+Hanauthor=H.+Leeauthor=H.+Swansonauthor=K.+Kim&title=Impact+of+linker+length+on+the+activity+of+PROTACs&doi=10.1039%2FC0MB00074D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of linker length on the activity of PROTACs</span></div><div class="casAuthors">Cyrus, Kedra; Wehenkel, Marie; Choi, Eun-Young; Han, Hyeong-Jun; Lee, Hyosung; Swanson, Hollie; Kim, Kyung-Bo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">359-364</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Conventional genetic approaches have provided a powerful tool in the study of proteins.  However, these techniques often preclude selective manipulation of temporal and spatial protein functions, which is crucial for the investigation of dynamic cellular processes.  To overcome these limitations, a small mol.-based novel technol. termed "PROteolysis TArgeting ChimeraS (PROTACs)" has been developed, targeting proteins for degrdn. at the post-translational level.  Despite the promising potential of PROTACs to serve as mol. probes of complex signaling pathways, their design has not been generalized for broad application.  Here, we present the first generalized approach for PROTAC design by fine-tuning the distance between the two participating partner proteins, the E3 ubiquitin ligase and the target protein.  As such, we took a chem. approach to create estrogen receptor (ER)-Î± targeting PROTACs with varying linker lengths and the loss of the ER in cultured cells was monitored via western blot and fluorometric analyses.  We found a significant effect of chain length on PROTAC efficacy, and, in this case, the optimum distance between the E3 recognition motif and the ligand was a 16 atom chain length.  The information gathered from this expt. may offer a generalizable PROTAC design strategy to further the expansion of the PROTAC toolbox, opening new possibilities for the broad application of the PROTAC strategy in the study of multiple signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP7hxQSvFw57Vg90H21EOLACvtfcHk0lityAXcl9gOPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslGhsw%253D%253D&md5=56dc56cb3a73d17bc8903c4962d7648c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC0MB00074D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0MB00074D%26sid%3Dliteratum%253Aachs%26aulast%3DCyrus%26aufirst%3DK.%26aulast%3DWehenkel%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DE.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DSwanson%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DK.%26atitle%3DImpact%2520of%2520linker%2520length%2520on%2520the%2520activity%2520of%2520PROTACs%26jtitle%3DMol.%2520BioSyst.%26date%3D2011%26volume%3D7%26spage%3D359%26epage%3D364%26doi%3D10.1039%2FC0MB00074D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Douglass, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegel, D. A.</span><span> </span><span class="NLM_article-title">A comprehensive mathematical model for three-body binding equilibria</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">6092</span><span class="NLM_x">â</span> <span class="NLM_lpage">6099</span><span class="refDoi">Â DOI: 10.1021/ja311795d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311795d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6092-6099&author=E.+F.+Douglassauthor=C.+J.+Millerauthor=G.+Sparerauthor=H.+Shapiroauthor=D.+A.+Spiegel&title=A+comprehensive+mathematical+model+for+three-body+binding+equilibria&doi=10.1021%2Fja311795d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Mathematical Model for Three-Body Binding Equilibria</span></div><div class="casAuthors">Douglass, Eugene F.; Miller, Chad J.; Sparer, Gerson; Shapiro, Harold; Spiegel, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6092-6099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-component systems are often more complex than their two-component counterparts.  Although the reversible assocn. of three components in soln. is crit. for a vast array of chem. and biol. processes, no general phys. picture of such systems has emerged.  Here the authors have developed a general, comprehensive framework for understanding ternary complex equil., which relates directly to familiar concepts such as EC50 and IC50 from simpler (binary complex) equil.  Importantly, application of the authors' model to data from the published literature has enabled the authors to achieve new insights into complex systems ranging from coagulation to therapeutic dosing regimens for monoclonal antibodies.  The authors also provide an Excel spreadsheet to assist readers in both conceptualizing and applying the authors' models.  Overall, the anal. has the potential to render complex three-component systems-which have previously been characterized as "anal. intractable"-readily comprehensible to theoreticians and experimentalists alike.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4T36t0kgZ3LVg90H21EOLACvtfcHk0lityAXcl9gOPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCktr4%253D&md5=44a6da2858f9813d7f1329c96a7085ac</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja311795d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311795d%26sid%3Dliteratum%253Aachs%26aulast%3DDouglass%26aufirst%3DE.%2BF.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DSparer%26aufirst%3DG.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DSpiegel%26aufirst%3DD.%2BA.%26atitle%3DA%2520comprehensive%2520mathematical%2520model%2520for%2520three-body%2520binding%2520equilibria%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6092%26epage%3D6099%26doi%3D10.1021%2Fja311795d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Gadd, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">514</span><span class="NLM_x">â</span> <span class="NLM_lpage">521</span><span class="refDoi">Â DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lhO-ZVt2fXjTA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Petzold, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ThomÃ¤, N. H.</span><span> </span><span class="NLM_article-title">Structural basis of lenalidomide-induced CK1Î± degradation by the CRL4(CRBN) ubiquitin ligase</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">532</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">â</span> <span class="NLM_lpage">130</span><span class="refDoi">Â DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature16979" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+basis+of+lenalidomide-induced+CK1%CE%B1+degradation+by+the+CRL4%28CRBN%29+ubiquitin+ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520basis%2520of%2520lenalidomide-induced%2520CK1%25CE%25B1%2520degradation%2520by%2520the%2520CRL4%2528CRBN%2529%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Angers, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacCoss, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moon, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, N.</span><span> </span><span class="NLM_article-title">Molecular architecture and assembly of the DDB1âCUL4A ubiquitin ligase machinery</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">443</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">â</span> <span class="NLM_lpage">593</span><span class="refDoi">Â DOI: 10.1038/nature05175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature05175" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=443&publication_year=2006&pages=590-593&author=S.+Angersauthor=T.+Liauthor=X.+Yiauthor=M.+J.+MacCossauthor=R.+T.+Moonauthor=N.+Zheng&title=Molecular+architecture+and+assembly+of+the+DDB1%E2%80%93CUL4A+ubiquitin+ligase+machinery&doi=10.1038%2Fnature05175"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature05175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05175%26sid%3Dliteratum%253Aachs%26aulast%3DAngers%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DYi%26aufirst%3DX.%26aulast%3DMacCoss%26aufirst%3DM.%2BJ.%26aulast%3DMoon%26aufirst%3DR.%2BT.%26aulast%3DZheng%26aufirst%3DN.%26atitle%3DMolecular%2520architecture%2520and%2520assembly%2520of%2520the%2520DDB1%25E2%2580%2593CUL4A%2520ubiquitin%2520ligase%2520machinery%26jtitle%3DNature%26date%3D2006%26volume%3D443%26spage%3D590%26epage%3D593%26doi%3D10.1038%2Fnature05175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="note"><p class="first last">A recent publication demonstrated for a VHL-PROTAC targeting a kinase that a wide range of linker lengths were tolerated but with an apparent lower limit cutoff of 12 linear non-H linker atoms, see:</p></div><div class="NLM_citation" id="rightTab-cit26a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Crew, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serebrenik, Y. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamman, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferraro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neklesa, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+Von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit26b"><span><span class="NLM_label">(b) </span><div class="note"><p class="first last">While shorter linker lengths were not investigated during the course of this work, shorter linker lengths in degradation probes have been successfully previously employed against other protein targets, see:</p></div><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Salas, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McEachern, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Przybranowski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit26c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Robb, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contreras, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kour, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonawane, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murry, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natarajan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rana, S.</span><span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">7577</span><span class="NLM_x">â</span> <span class="NLM_lpage">7580</span><span class="refDoi">Â DOI: 10.1039/C7CC03879H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1039%2FC7CC03879H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2FC7CC03879H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26cR"><div class="casContent"><span class="casTitleNuber">26c</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0lh6gTTKQe_r-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit26c&amp;dbid=16384&amp;doi=10.1039%2FC7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC03879H%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2FC7CC03879H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Ruchelman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Man, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corral, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babusis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moghaddam, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirley, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, G. W.</span><span> </span><span class="NLM_article-title">Isosteric analogs of lenalidomide and pomalidomide: Synthesis and biological activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">360</span><span class="NLM_x">â</span> <span class="NLM_lpage">365</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2012.10.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2012.10.071" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=360-365&author=A.+L.+Ruchelmanauthor=H.+Manauthor=W.+Zhangauthor=R.+Chenauthor=L.+Caponeauthor=J.+Kangauthor=A.+Partonauthor=L.+Corralauthor=P.+H.+Schaferauthor=D.+Babusisauthor=M.+F.+Moghaddamauthor=Y.+Tangauthor=M.+A.+Shirleyauthor=G.+W.+Muller&title=Isosteric+analogs+of+lenalidomide+and+pomalidomide%3A+Synthesis+and+biological+activity&doi=10.1016%2Fj.bmcl.2012.10.071"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.071%26sid%3Dliteratum%253Aachs%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DMan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DCapone%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DParton%26aufirst%3DA.%26aulast%3DCorral%26aufirst%3DL.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DBabusis%26aufirst%3DD.%26aulast%3DMoghaddam%26aufirst%3DM.%2BF.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DShirley%26aufirst%3DM.%2BA.%26aulast%3DMuller%26aufirst%3DG.%2BW.%26atitle%3DIsosteric%2520analogs%2520of%2520lenalidomide%2520and%2520pomalidomide%253A%2520Synthesis%2520and%2520biological%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D360%26epage%3D365%26doi%3D10.1016%2Fj.bmcl.2012.10.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Lohbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. K.</span><span> </span><span class="NLM_article-title">Practical synthesis of a phthalimide-based cereblon ligand to enable PROTAC development</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">5260</span><span class="NLM_x">â</span> <span class="NLM_lpage">5262</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bmcl.2016.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27687673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5260-5262&author=J.+Lohbeckauthor=A.+K.+Miller&title=Practical+synthesis+of+a+phthalimide-based+cereblon+ligand+to+enable+PROTAC+development&doi=10.1016%2Fj.bmcl.2016.09.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Practical synthesis of a phthalimide-based Cereblon ligand to enable PROTAC development</span></div><div class="casAuthors">Lohbeck, Jasmin; Miller, Aubry K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5260-5262</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The use of small mols. to regulate cellular levels of specific proteins is poised to become a powerful technique in the coming years.  Crit. to the success of any project using such an approach will be the ability to synthesize libraries of candidate small mols. for testing in cellular systems.  Herein, the authors describe a practical synthesis of a phthalimide-based scaffold, which can be easily diversified to make Cereblon-targeting PROTACs.  The authors demonstrate the effectiveness of this approach by synthesizing a 'PROTAC toolbox' of four amines which can be coupled to inhibitors in a straightforward manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo01AK2Qjj97Vg90H21EOLACvtfcHk0lh6gTTKQe_r-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymsLbI&md5=864f4c671ef2c8d43191fc6a6e22b3b2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DLohbeck%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DA.%2BK.%26atitle%3DPractical%2520synthesis%2520of%2520a%2520phthalimide-based%2520cereblon%2520ligand%2520to%2520enable%2520PROTAC%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5260%26epage%3D5262%26doi%3D10.1016%2Fj.bmcl.2016.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Georgsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larhed, M.</span><span> </span><span class="NLM_article-title">Rapid palladium-catalyzed synthesis of esters from aryl halides utilizing Mo(CO)<sub>6</sub> as a solid carbon monoxide source</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">350</span><span class="NLM_x">â</span> <span class="NLM_lpage">352</span><span class="refDoi">Â DOI: 10.1021/cc0201086</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc0201086" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktlaiurk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=350-352&author=J.+Georgssonauthor=A.+Hallbergauthor=M.+Larhed&title=Rapid+palladium-catalyzed+synthesis+of+esters+from+aryl+halides+utilizing+Mo%28CO%296+as+a+solid+carbon+monoxide+source&doi=10.1021%2Fcc0201086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29aR"><div class="casContent"><span class="casTitleNuber">29a</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Palladium-Catalyzed Synthesis of Esters from Aryl Halides Utilizing Mo(CO)6 as a Solid Carbon Monoxide Source</span></div><div class="casAuthors">Georgsson, Jennie; Hallberg, Anders; Larhed, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">350-352</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of arom. esters R1CO2R2 (R1 = 3-BrC6H4, 2-MeC6H4, 4-MeOC6H4, 4-F3CC6H4, 1-naphthyl; R2 = Bu, Me3SiCH2CH2, PhCH2) was synthesized in moderate to high yields (52-89%) via palladium-catalyzed carbonylation of aryl halides R1X (X = Br, iodo) in the presence of excess of a primary alc. R2OH under microwave irradn. using Mo(CO)6 as a solid carbon monoxide source.  The reactions with sterically hindered tert-butanol as an alc. failed or gave low yields of the desired products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS1k1kMsuaA7Vg90H21EOLACvtfcHk0li-Qqk1IkflDw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktlaiurk%253D&md5=7ee6bd650c0c7cfd98b9fc83045e70d2</span></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=10.1021%2Fcc0201086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc0201086%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgsson%26aufirst%3DJ.%26aulast%3DHallberg%26aufirst%3DA.%26aulast%3DLarhed%26aufirst%3DM.%26atitle%3DRapid%2520palladium-catalyzed%2520synthesis%2520of%2520esters%2520from%2520aryl%2520halides%2520utilizing%2520Mo%2528CO%25296%2520as%2520a%2520solid%2520carbon%2520monoxide%2520source%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2003%26volume%3D5%26spage%3D350%26epage%3D352%26doi%3D10.1021%2Fcc0201086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit29b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Georgsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SkÃ¶ld, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plouffe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindeberg, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botros, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larhed, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nyberg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallo-Payet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gogoll, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KarlÃ©n, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span> </span><span class="NLM_article-title">Angiotensin II pseudopeptides containing 1,3,5-trisubstituted benzene scaffolds with high AT2 receptor affinity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6620</span><span class="NLM_x">â</span> <span class="NLM_lpage">6631</span><span class="refDoi">Â DOI: 10.1021/jm050280z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050280z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6620-6631&author=J.+Georgssonauthor=C.+Sk%C3%B6ldauthor=B.+Plouffeauthor=G.+Lindebergauthor=M.+Botrosauthor=M.+Larhedauthor=F.+Nybergauthor=N.+Gallo-Payetauthor=A.+Gogollauthor=A.+Karl%C3%A9nauthor=A.+Hallberg&title=Angiotensin+II+pseudopeptides+containing+1%2C3%2C5-trisubstituted+benzene+scaffolds+with+high+AT2+receptor+affinity&doi=10.1021%2Fjm050280z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=10.1021%2Fjm050280z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050280z%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgsson%26aufirst%3DJ.%26aulast%3DSk%25C3%25B6ld%26aufirst%3DC.%26aulast%3DPlouffe%26aufirst%3DB.%26aulast%3DLindeberg%26aufirst%3DG.%26aulast%3DBotros%26aufirst%3DM.%26aulast%3DLarhed%26aufirst%3DM.%26aulast%3DNyberg%26aufirst%3DF.%26aulast%3DGallo-Payet%26aufirst%3DN.%26aulast%3DGogoll%26aufirst%3DA.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26aulast%3DHallberg%26aufirst%3DA.%26atitle%3DAngiotensin%2520II%2520pseudopeptides%2520containing%25201%252C3%252C5-trisubstituted%2520benzene%2520scaffolds%2520with%2520high%2520AT2%2520receptor%2520affinity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6620%26epage%3D6631%26doi%3D10.1021%2Fjm050280z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: Comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=K.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+Comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520Comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><div class="note"><p class="first last">Our FP-probe was generated using a Sulfo-Cy5 fluorophore rather than Cy5 but displayed similar <i>K</i><sub>i</sub> values for thalidomide and lenalidomide as:</p></div><span class="NLM_contrib-group">Fischer, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">BÃ¶hm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lydeard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadler, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavadini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serluca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acker, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraju, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tichkule, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schebesta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forrester, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassiepen, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hild, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckwith, R. E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ThomÃ¤, N. H.</span><span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">512</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="NLM_x">â</span> <span class="NLM_lpage">53</span><span class="refDoi">Â DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0li-Qqk1IkflDw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="note"><p class="first last"><i>K</i><sub>i</sub> values were calculated from the geometric mean IC<sub>50</sub> and the FP-assay probe affinity using the methods described by:</p></div><div class="NLM_citation" id="rightTab-cit32a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Huang, X.</span><span> </span><span class="NLM_article-title">Fluorescence polarization competition assay: The range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">â</span> <span class="NLM_lpage">38</span><span class="refDoi">Â DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=12854996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&author=X.+Huang&title=Fluorescence+polarization+competition+assay%3A+The+range+of+resolvable+inhibitor+potency+is+limited+by+the+affinity+of+the+fluorescent+ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32aR"><div class="casContent"><span class="casTitleNuber">32a</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span></div><div class="casAuthors">Huang, Xinyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">For the development of fluorescence polarization (FP) competition assays, there is a widespread belief that tight-binding fluorescent ligands should be avoided to identify inhibitors of low or intermediate potency in the screening of small-mol. compd. libraries.  It is demonstrated herein that this statement is a misconception; in fact, the higher the affinity of the fluorescent ligand, the wider the range of inhibitor potency that can be resolved.  An approx. est. for the low end of inhibitor Ki values that can be resolved is the Kd value of the fluorescent ligand.  Because FP competition assays are typically conducted under nonstoichiometric titrn. conditions, it is suggested that a fluorescent ligand of highest affinity that also has an adequate quantum yield to satisfy such conditions be selected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUYlJzQaNj7Vg90H21EOLACvtfcHk0li-Qqk1IkflDw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D&md5=7c706c96b6e534486f6780c93a0d441b</span></div><a href="/servlet/linkout?suffix=cit32a&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DFluorescence%2520polarization%2520competition%2520assay%253A%2520The%2520range%2520of%2520resolvable%2520inhibitor%2520potency%2520is%2520limited%2520by%2520the%2520affinity%2520of%2520the%2520fluorescent%2520ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit32b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Munson, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodbard, D.</span><span> </span><span class="NLM_article-title">An exact correction to the â³Cheng-Prusoffâ³ correction</span> <span class="citation_source-journal">J. Recept. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">â</span> <span class="NLM_lpage">456</span><span class="refDoi">Â DOI: 10.3109/10799898809049010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.3109%2F10799898809049010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=3385692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADyaL1cXitFGks7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1988&pages=533-456&author=P.+J.+Munsonauthor=D.+Rodbard&title=An+exact+correction+to+the+%E2%80%B3Cheng-Prusoff%E2%80%B3+correction&doi=10.3109%2F10799898809049010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32bR"><div class="casContent"><span class="casTitleNuber">32b</span><div class="casTitle"><span class="NLM_cas:atitle">An exact correction to the "Cheng-Prusoff" correction</span></div><div class="casAuthors">Munson, Peter J.; Rodbard, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptor Research</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1-4</span>),
    <span class="NLM_cas:pages">533-46</span>CODEN:
                <span class="NLM_cas:coden">JRERDM</span>;
        ISSN:<span class="NLM_cas:issn">0197-5110</span>.
    </div><div class="casAbstract">The Cheng-Prusoff correction for obtaining the inhibition const. Ki from the graphical midpoint or 50% ED (ED50) of a displacement or inhibition curve may lead to 3-6-fold or larger errors, esp. when applied to drug-receptor binding studies.  A simple exact correction rule based on total concns. for the reactants was derived to relate the Ki to the ED50 concn. of the inhibitor.  The exact correction is needed esp. when the ratio of bound to free ligand (y0) or the total concn. of labeled and unlabeled ligands was large or when the ED50 value was low.  The use of the Cheng-Prusoff correction is appropriate only when exptl. errors are extremely small and when y0 â¤0.1 and the ED50 of the inhibitor is greater than the ED50 for the self-competition of the labeled ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsuJtjVMXcI7Vg90H21EOLACvtfcHk0ljvu-UKf8UwKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXitFGks7g%253D&md5=59b3521f1967f6d7ac0655772c7660ec</span></div><a href="/servlet/linkout?suffix=cit32b&amp;dbid=16384&amp;doi=10.3109%2F10799898809049010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10799898809049010%26sid%3Dliteratum%253Aachs%26aulast%3DMunson%26aufirst%3DP.%2BJ.%26aulast%3DRodbard%26aufirst%3DD.%26atitle%3DAn%2520exact%2520correction%2520to%2520the%2520%25E2%2580%25B3Cheng-Prusoff%25E2%2580%25B3%2520correction%26jtitle%3DJ.%2520Recept.%2520Res.%26date%3D1988%26volume%3D8%26spage%3D533%26epage%3D456%26doi%3D10.3109%2F10799898809049010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span>Wes: Simple Westerns Just Got Even Simpler; <span class="NLM_publisher-name">Proteinsimple</span>: <span class="NLM_publisher-loc">San Jose, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">; </span><a href="http://www.proteinsimple.com/wes.html" class="extLink">https://www.proteinsimple.com/wes.html</a> (accessed August 28, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Wes%3A+Simple+Westerns+Just+Got+Even+Simpler%3B+Proteinsimple%3A+San+Jose%2C+CA%2C+2017%3B+https%3A%2F%2Fwww.proteinsimple.com%2Fwes.html+%28accessed+August+28%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DWes%253A%2520Simple%2520Westerns%2520Just%2520Got%2520Even%2520Simpler%26pub%3DProteinsimple%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="note"><p class="first last">For the origin of our SK-OV-3 ovarian carcinoma cell line, see the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Erales, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coffino, P.</span><span> </span><span class="NLM_article-title">Ubiquitin-independent proteasomal degradation</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1843</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">â</span> <span class="NLM_lpage">221</span><span class="refDoi">Â DOI: 10.1016/j.bbamcr.2013.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.bbamcr.2013.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=23684952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12qsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=216-221&author=J.+Eralesauthor=P.+Coffino&title=Ubiquitin-independent+proteasomal+degradation&doi=10.1016%2Fj.bbamcr.2013.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-independent proteasomal degradation</span></div><div class="casAuthors">Erales, Jenny; Coffino, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">216-221</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Most proteasome substrates are marked for degrdn. by ubiquitin conjugation, but some are targeted by other means.  The properties of these exceptional cases provide insights into the general requirements for proteasomal degrdn.  Here the focus is on three ubiquitin-independent substrates that have been the subject of detailed study.  These are Rpn4, a transcriptional regulator of proteasome homeostasis, thymidylate synthase, an enzyme required for prodn. of DNA precursors and ornithine decarboxylase, the initial enzyme committed to polyamine biosynthesis.  It can be inferred from these cases that proteasome assocn. and the presence of an unstructured region are the sole prerequisites for degrdn.  Based on that inference, artificial substrates have been designed to test the proteasome's capacity for substrate processing and its limitations.  Ubiquitin-independent substrates may in some cases be a remnant of the pre-ubiquitome world, but in other cases could provide optimized regulatory solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9wxiAlcl4CbVg90H21EOLACvtfcHk0ljvu-UKf8UwKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12qsb4%253D&md5=605cbe83ddb7be7bb9cbeae02d1f57dd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2013.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2013.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DErales%26aufirst%3DJ.%26aulast%3DCoffino%26aufirst%3DP.%26atitle%3DUbiquitin-independent%2520proteasomal%2520degradation%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D216%26epage%3D221%26doi%3D10.1016%2Fj.bbamcr.2013.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Lai, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellerschmied, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salami, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaime-Figueroa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">807</span><span class="NLM_x">â</span> <span class="NLM_lpage">810</span><span class="refDoi">Â DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0ljvu-UKf8UwKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Mateus, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wayne, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almqvist, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Axelsson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seashore-Ludlow, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treyer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LundbÃ¤ck, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Prediction of intracellular exposure bridges the gap between target- and cell-based drug discovery</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">E6231</span><span class="NLM_x">â</span> <span class="NLM_lpage">E6239</span><span class="refDoi">Â DOI: 10.1073/pnas.1701848114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1073%2Fpnas.1701848114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28701380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E6231-E6239&author=A.+Mateusauthor=L.+J.+Gordonauthor=G.+J.+Wayneauthor=H.+Almqvistauthor=H.+Axelssonauthor=B.+Seashore-Ludlowauthor=A.+Treyerauthor=P.+Matssonauthor=T.+Lundb%C3%A4ckauthor=A.+Westauthor=M.+M.+Hannauthor=P.+Artursson&title=Prediction+of+intracellular+exposure+bridges+the+gap+between+target-+and+cell-based+drug+discovery&doi=10.1073%2Fpnas.1701848114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of intracellular exposure bridges the gap between target- and cell-based drug discovery</span></div><div class="casAuthors">Mateus, Andre; Gordon, Laurie J.; Wayne, Gareth J.; Almqvist, Helena; Axelsson, Hanna; Seashore-Ludlow, Brinton; Treyer, Andrea; Matsson, Paer; Lundbaeck, Thomas; West, Andy; Hann, Michael M.; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">E6231-E6239</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inadequate target exposure is a major cause of high attrition in drug discovery.  Here, we show that a label-free method for quantifying the intracellular bioavailability (Fic) of drug mols. predicts drug access to intracellular targets and hence, pharmacol. effect.  We detd. Fic in multiple cellular assays and cell types representing different targets from a no. of therapeutic areas, including cancer, inflammation, and dementia.  Both cytosolic targets and targets localized in subcellular compartments were investigated.  Fic gives insights on membrane-permeable compds. in terms of cellular potency and intracellular target engagement, compared with biochem. potency measurements alone.  Knowledge of the amt. of drug that is locally available to bind intracellular targets provides a powerful tool for compd. selection in early drug discovery. localization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo1IEI-YIPNbVg90H21EOLACvtfcHk0lglWRi5wTalhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKmsr%252FK&md5=ace62bf5bda6cfd214bac4b7c5bf9cf1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1701848114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1701848114%26sid%3Dliteratum%253Aachs%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DSeashore-Ludlow%26aufirst%3DB.%26aulast%3DTreyer%26aufirst%3DA.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DWest%26aufirst%3DA.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DPrediction%2520of%2520intracellular%2520exposure%2520bridges%2520the%2520gap%2520between%2520target-%2520and%2520cell-based%2520drug%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE6231%26epage%3DE6239%26doi%3D10.1073%2Fpnas.1701848114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span> </span><span class="NLM_article-title">How big is too big for cell permeability?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1662</span><span class="NLM_x">â</span> <span class="NLM_lpage">1664</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00237</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00237" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCjtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1662-1664&author=P.+Matssonauthor=J.+Kihlberg&title=How+big+is+too+big+for+cell+permeability%3F&doi=10.1021%2Facs.jmedchem.7b00237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38aR"><div class="casContent"><span class="casTitleNuber">38a</span><div class="casTitle"><span class="NLM_cas:atitle">How Big Is Too Big for Cell Permeability?</span></div><div class="casAuthors">Matsson, Paer; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1662-1664</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Understanding how to design cell permeable ligands for intracellular targets that have difficult binding sites, such as protein-protein interactions, would open vast opportunities for drug discovery.  Interestingly, libraries of cyclic peptides displayed a steep drop-off in membrane permeability at mol. wts. above 1000 Da and it appears likely that this cutoff constitutes an upper size limit also for more drug-like compds.  However, chem. space from 500 to 1000 Da remains virtually unexplored and represents a vast opportunity for those prepd. to venture into new territories of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXxiuvHQSnJLVg90H21EOLACvtfcHk0lglWRi5wTalhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCjtL8%253D&md5=563efb3b4603c33c4a976e88b1c7b514</span></div><a href="/servlet/linkout?suffix=cit38a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00237%26sid%3Dliteratum%253Aachs%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520big%2520is%2520too%2520big%2520for%2520cell%2520permeability%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1662%26epage%3D1664%26doi%3D10.1021%2Facs.jmedchem.7b00237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit38b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Pye, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwochert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etienne, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limberakis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furukawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathiowetz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liras, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lokey, R. S.</span><span> </span><span class="NLM_article-title">Nonclassical size dependence of permeation defines bounds for passive adsorption of large drug molecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1665</span><span class="NLM_x">â</span> <span class="NLM_lpage">1672</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslajtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1665-1672&author=C.+R.+Pyeauthor=W.+M.+Hewittauthor=J.+Schwochertauthor=T.+D.+Haddadauthor=C.+E.+Townsendauthor=L.+Etienneauthor=Y.+Laoauthor=C.+Limberakisauthor=A.+Furukawaauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=R.+S.+Lokey&title=Nonclassical+size+dependence+of+permeation+defines+bounds+for+passive+adsorption+of+large+drug+molecules&doi=10.1021%2Facs.jmedchem.6b01483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38bR"><div class="casContent"><span class="casTitleNuber">38b</span><div class="casTitle"><span class="NLM_cas:atitle">Nonclassical Size Dependence of Permeation Defines Bounds for Passive Adsorption of Large Drug Molecules</span></div><div class="casAuthors">Pye, Cameron R.; Hewitt, William M.; Schwochert, Joshua; Haddad, Terra D.; Townsend, Chad E.; Etienne, Lyns; Lao, Yongtong; Limberakis, Chris; Furukawa, Akihiro; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1665-1672</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a growing no. of intracellular protein-protein interactions (PPI's) with potential therapeutic importance that demand larger ligands than those that neatly fit into most common criteria for "drug-likeness".  Macrocyclic peptides pro-vide the large surface area required to inhibit PPIs and, in principle, avoid many of the pitfalls assocd. with peptides, including low stability and membrane permeability.  However, the design of passively cell-permeable mols. in this space remains a great challenge due to the poorly understood roles of mol. size and lipophilicity in detg. passive membrane permeability above MW â¼500.  Using split-pool combinatorial synthesis, we constructed a library of cyclic, per-N-methlyated peptides spanning a wide range of calcd. lipohilicities (0 < AlogP < 8) and mol. wts. (â¼800 Da < MW < â¼1200 Da).  Assaying mixts. of compds. using the Parallel Artificial Membrane Assay (PAMPA) revealed a steep drop-off in apparent passive permeability with increasing size and a bilinear relationship with lipophilicity.  We obsd. a size dependence of permeability that is in stark disagreement with current permeation models.  The lipophilicities required to overcome this size penalty led to poor aq. soly., effectively bounding the chem. space of likely passively permeable mols.  This understanding provides a new set of design criteria for achieving permeability in larger mol. size regimes and suggests an operational cut-off beyond which passive permeability is constrained by a sharply increasing penalty on permeation through the system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXD-PeEQUtu7Vg90H21EOLACvtfcHk0lglWRi5wTalhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslajtw%253D%253D&md5=7b51a3c8fe24560c9445d8d32bf3b70f</span></div><a href="/servlet/linkout?suffix=cit38b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01483%26sid%3Dliteratum%253Aachs%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DHaddad%26aufirst%3DT.%2BD.%26aulast%3DTownsend%26aufirst%3DC.%2BE.%26aulast%3DEtienne%26aufirst%3DL.%26aulast%3DLao%26aufirst%3DY.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DFurukawa%26aufirst%3DA.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DNonclassical%2520size%2520dependence%2520of%2520permeation%2520defines%2520bounds%2520for%2520passive%2520adsorption%2520of%2520large%2520drug%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1665%26epage%3D1672%26doi%3D10.1021%2Facs.jmedchem.6b01483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KeserÃ¼, G. M.</span><span> </span><span class="NLM_article-title">Finding the sweet spot: The role of nature and nurture in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">â</span> <span class="NLM_lpage">365</span><span class="refDoi">Â DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&author=M.+M.+Hannauthor=G.+M.+Keser%C3%BC&title=Finding+the+sweet+spot%3A+The+role+of+nature+and+nurture+in+medicinal+chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0ljt-lgjhfSuSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520sweet%2520spot%253A%2520The%2520role%2520of%2520nature%2520and%2520nurture%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Palm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ungell, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strandlund, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beigi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Evaluation of dynamic polar molecular surface area as predictor of drug absorption: Comparison with other computational and experimental predictors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">5382</span><span class="NLM_x">â</span> <span class="NLM_lpage">5392</span><span class="refDoi">Â DOI: 10.1021/jm980313t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980313t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADyaK1cXns1yltrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=5382-5392&author=K.+Palmauthor=K.+Luthmanauthor=A.+L.+Ungellauthor=G.+Strandlundauthor=F.+Beigiauthor=P.+Lundahlauthor=P.+Artursson&title=Evaluation+of+dynamic+polar+molecular+surface+area+as+predictor+of+drug+absorption%3A+Comparison+with+other+computational+and+experimental+predictors&doi=10.1021%2Fjm980313t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Dynamic Polar Molecular Surface Area as Predictor of Drug Absorption: Comparison with Other Computational and Experimental Predictors</span></div><div class="casAuthors">Palm, Katrin; Luthman, Kristina; Ungell, Anna-Lena; Strandlund, Gert; Beigi, Farideh; Lundahl, Per; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">5382-5392</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The relationship between various mol. descriptors and transport of drugs across the intestinal epithelium was evaluated.  The monolayer permeability (Pc) of human intestinal Caco-2 cells to a series of nine Î²-receptor-blocking agents was investigated in vitro.  The dynamic polar mol. surface area (PSAd) of the compds. was calcd. from all low-energy conformations identified in mol. mechanics calcns. in vacuum and in simulated chloroform and water environments.  For most of the investigated drugs, the effects of the different environments on PSAd were small.  The exception was H 216/44, which is a large flexible compd. contg. several functional groups capable of hydrogen bonding (PSAd,chloroform = 70.8 Ã2 and PSAd,water = 116.6 Ã2).  The relationship between Pc and PSAd was stronger than those between Pc and the calcd. octanol/water distribution coeffs. (log Dcalc) or the exptl. detd. immobilized liposome chromatog. (ILC) retention.  Pc values for two new practolol analogs and H 216/44 were predicted from the structure-permeability relationships of a subset of the nine compds. and compared with exptl. values.  The Pc values of the two practolol analogs were predicted well from both PSAd calcns. and ILC retention studies.  The Pc value of H 216/44 was reasonably well-predicted only from the PSAd of conformations preferred in vacuum and in water.  The other descriptors overestimated the Pc of H 216/44 100-500-fold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolKDCQqqUS9rVg90H21EOLACvtfcHk0ljt-lgjhfSuSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1yltrc%253D&md5=83820600b8fe6ffde348c272d45c628a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm980313t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980313t%26sid%3Dliteratum%253Aachs%26aulast%3DPalm%26aufirst%3DK.%26aulast%3DLuthman%26aufirst%3DK.%26aulast%3DUngell%26aufirst%3DA.%2BL.%26aulast%3DStrandlund%26aufirst%3DG.%26aulast%3DBeigi%26aufirst%3DF.%26aulast%3DLundahl%26aufirst%3DP.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DEvaluation%2520of%2520dynamic%2520polar%2520molecular%2520surface%2520area%2520as%2520predictor%2520of%2520drug%2520absorption%253A%2520Comparison%2520with%2520other%2520computational%2520and%2520experimental%2520predictors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D5382%26epage%3D5392%26doi%3D10.1021%2Fjm980313t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Gillis, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastman, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donnelly, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meanwell, N. A.</span><span> </span><span class="NLM_article-title">Applications of fluorine in medicinal chemistry</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8315</span><span class="NLM_x">â</span> <span class="NLM_lpage">8359</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8315-8359&author=E.+P.+Gillisauthor=K.+J.+Eastmanauthor=M.+D.+Hillauthor=D.+J.+Donnellyauthor=N.+A.+Meanwell&title=Applications+of+fluorine+in+medicinal+chemistry&doi=10.1021%2Facs.jmedchem.5b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of Fluorine in Medicinal Chemistry</span></div><div class="casAuthors">Gillis, Eric P.; Eastman, Kyle J.; Hill, Matthew D.; Donnelly, David J.; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8315-8359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with meta-anal. The role of fluorine in drug design and development is expanding rapidly as we learn more about the unique properties assocd. with this unusual element and how to deploy it with greater sophistication.  The judicious introduction of fluorine into a mol. can productively influence conformation, pKa, intrinsic potency, membrane permeability, metabolic pathways, and pharmacokinetic properties.  In addn., 18F has been established as a useful positron emitting isotope for use with in vivo imaging technol. that potentially has extensive application in drug discovery and development, often limited only by convenient synthetic accessibility to labeled compds.  The wide ranging applications of fluorine in drug design are providing a strong stimulus for the development of new synthetic methodologies that allow more facile access to a wide range of fluorinated compds.  In this review, we provide an update on the effects of the strategic incorporation of fluorine in drug mols. and applications in positron emission tomog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8fUUtkPB4L7Vg90H21EOLACvtfcHk0ljheCvFuHeyqQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK&md5=9995829a94a8c0b8d9fb0d21bdfd5a1d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00258%26sid%3Dliteratum%253Aachs%26aulast%3DGillis%26aufirst%3DE.%2BP.%26aulast%3DEastman%26aufirst%3DK.%2BJ.%26aulast%3DHill%26aufirst%3DM.%2BD.%26aulast%3DDonnelly%26aufirst%3DD.%2BJ.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DApplications%2520of%2520fluorine%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8315%26epage%3D8359%26doi%3D10.1021%2Facs.jmedchem.5b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">George, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moody, C. J.</span><span> </span><span class="NLM_article-title">Synthesis of iodopyridone</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">8209</span><span class="NLM_x">â</span> <span class="NLM_lpage">8215</span><span class="refDoi">Â DOI: 10.1016/j.tet.2013.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.tet.2013.07.033" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=8209-8215&author=I.+R.+Georgeauthor=W.+Lewisauthor=C.+J.+Moody&title=Synthesis+of+iodopyridone&doi=10.1016%2Fj.tet.2013.07.033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2013.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2013.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DGeorge%26aufirst%3DI.%2BR.%26aulast%3DLewis%26aufirst%3DW.%26aulast%3DMoody%26aufirst%3DC.%2BJ.%26atitle%3DSynthesis%2520of%2520iodopyridone%26jtitle%3DTetrahedron%26date%3D2013%26volume%3D69%26spage%3D8209%26epage%3D8215%26doi%3D10.1016%2Fj.tet.2013.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><div class="note"><p class="first last">In mouse fibroblasts, pirin has a measured inherent protein half-life of 64 h:</p></div><span class="NLM_contrib-group">SchwanhÃ¤usser, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Busse, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmar, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuchhardt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selbach, M.</span><span> </span><span class="NLM_article-title">Global quantification of mammalian gene expression control</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">473</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">â</span> <span class="NLM_lpage">342</span><span class="refDoi">Â DOI: 10.1038/nature10098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnature10098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=21593866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BC3Mrjtlaguw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2011&pages=337-342&author=B.+Schwanh%C3%A4usserauthor=D.+Busseauthor=N.+Liauthor=G.+Dittmarauthor=J.+Schuchhardtauthor=J.+Wolfauthor=W.+Chenauthor=M.+Selbach&title=Global+quantification+of+mammalian+gene+expression+control&doi=10.1038%2Fnature10098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Global quantification of mammalian gene expression control</span></div><div class="casAuthors">Schwanhausser Bjorn; Busse Dorothea; Li Na; Dittmar Gunnar; Schuchhardt Johannes; Wolf Jana; Chen Wei; Selbach Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">7347</span>),
    <span class="NLM_cas:pages">337-42</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Gene expression is a multistep process that involves the transcription, translation and turnover of messenger RNAs and proteins.  Although it is one of the most fundamental processes of life, the entire cascade has never been quantified on a genome-wide scale.  Here we simultaneously measured absolute mRNA and protein abundance and turnover by parallel metabolic pulse labelling for more than 5,000 genes in mammalian cells.  Whereas mRNA and protein levels correlated better than previously thought, corresponding half-lives showed no correlation.  Using a quantitative model we have obtained the first genome-scale prediction of synthesis rates of mRNAs and proteins.  We find that the cellular abundance of proteins is predominantly controlled at the level of translation.  Genes with similar combinations of mRNA and protein stability shared functional properties, indicating that half-lives evolved under energetic and dynamic constraints.  Quantitative information about all stages of gene expression provides a rich resource and helps to provide a greater understanding of the underlying design principles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWkDdhCnw1Wty3lHz94U-GfW6udTcc2eZv7i6Kny0Qprntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mrjtlaguw%253D%253D&md5=aa6204aff902c51bb1323966d0090c64</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnature10098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10098%26sid%3Dliteratum%253Aachs%26aulast%3DSchwanh%25C3%25A4usser%26aufirst%3DB.%26aulast%3DBusse%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DDittmar%26aufirst%3DG.%26aulast%3DSchuchhardt%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DSelbach%26aufirst%3DM.%26atitle%3DGlobal%2520quantification%2520of%2520mammalian%2520gene%2520expression%2520control%26jtitle%3DNature%26date%3D2011%26volume%3D473%26spage%3D337%26epage%3D342%26doi%3D10.1038%2Fnature10098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="note"><p class="first last">The stability of the thalidomide derived probes in DMSO stock appears dependent on the water content of the DMSO. We recommend fresh DMSO stocks are prepared from solid prior to each use. For a discussion of thalidomide stability, see:</p></div><div class="NLM_citation" id="rightTab-cit44a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Lepper, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scripture, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figg, W. D.</span><span> </span><span class="NLM_article-title">Thalidomide metabolism and hydrolysis: mechanisms and implications</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">â</span> <span class="NLM_lpage">685</span><span class="refDoi">Â DOI: 10.2174/138920006778017777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.2174%2F138920006778017777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=16918319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvV2mtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=677-685&author=E.+R.+Lepperauthor=N.+F.+Smithauthor=M.+C.+Coxauthor=C.+D.+Scriptureauthor=W.+D.+Figg&title=Thalidomide+metabolism+and+hydrolysis%3A+mechanisms+and+implications&doi=10.2174%2F138920006778017777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44aR"><div class="casContent"><span class="casTitleNuber">44a</span><div class="casTitle"><span class="NLM_cas:atitle">Thalidomide metabolism and hydrolysis: mechanisms and implications</span></div><div class="casAuthors">Lepper, Erin R.; Smith, Nicola F.; Cox, Michael C.; Scripture, Charity D.; Figg, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">677-685</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite its controversial past, thalidomide is currently under investigation for the treatment of several disease types, ranging from inflammatory conditions to cancer.  The mechanism of action of thalidomide is complex and not yet fully understood, but there is some evidence to suggest that metab. may play a role.  Consequently, there has been a considerable effort to characterize the metab. of thalidomide in recent years.  Thalidomide undergoes biotransformation by non-enzymic hydrolysis and enzyme-mediated hydroxylation to form a multitude of metabolites.  Metabolite identification and reaction phenotyping studies have been performed and will be discussed in this review in addn. to interspecies differences in thalidomide metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-PXbzjhKAALVg90H21EOLACvtfcHk0ljheCvFuHeyqQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvV2mtLc%253D&md5=09e2dcde069bccdadbe4a9707818e4e0</span></div><a href="/servlet/linkout?suffix=cit44a&amp;dbid=16384&amp;doi=10.2174%2F138920006778017777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920006778017777%26sid%3Dliteratum%253Aachs%26aulast%3DLepper%26aufirst%3DE.%2BR.%26aulast%3DSmith%26aufirst%3DN.%2BF.%26aulast%3DCox%26aufirst%3DM.%2BC.%26aulast%3DScripture%26aufirst%3DC.%2BD.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26atitle%3DThalidomide%2520metabolism%2520and%2520hydrolysis%253A%2520mechanisms%2520and%2520implications%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2006%26volume%3D7%26spage%3D677%26epage%3D685%26doi%3D10.2174%2F138920006778017777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit44b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Reist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrupt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francotte, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, B.</span><span> </span><span class="NLM_article-title">Chiral inversion and hydrolysis of thalidomide: mechanisms and catalysis by bases and serum albumin, and chiral stability of teratogenic metabolites</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1521</span><span class="NLM_x">â</span> <span class="NLM_lpage">1528</span><span class="refDoi">Â DOI: 10.1021/tx9801817</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx9801817" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADyaK1cXntFens7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=1521-1528&author=M.+Reistauthor=P.+Carruptauthor=E.+Francotteauthor=B.+Testa&title=Chiral+inversion+and+hydrolysis+of+thalidomide%3A+mechanisms+and+catalysis+by+bases+and+serum+albumin%2C+and+chiral+stability+of+teratogenic+metabolites&doi=10.1021%2Ftx9801817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44bR"><div class="casContent"><span class="casTitleNuber">44b</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral Inversion and Hydrolysis of Thalidomide: Mechanisms and Catalysis by Bases and Serum Albumin, and Chiral Stability of Teratogenic Metabolites</span></div><div class="casAuthors">Reist, Marianne; Carrupt, Pierre-Alain; Francotte, Eric; Testa, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1521-1528</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The chiral inversion and hydrolysis of thalidomide and the catalysis by bases and human serum albumin were investigated by using a stereoselective HPLC assay.  Chiral inversion was catalyzed by albumin, hydroxyl ions, phosphate, and amino acids.  Basic amino acids (Arg and Lys) had a superior potency in catalyzing chiral inversion compared to acid and neutral ones.  The chiral inversion of thalidomide is thus subject to specific and general base catalysis, and it is suggested that the ability of HSA to catalyze the reaction is due to the basic groups of the amino acids Arg and Lys and not to a single catalytic site on the macromol.  The hydrolysis of thalidomide was also base-catalyzed.  However, albumin had no effect on hydrolysis, and there was no difference between the catalytic potencies of acidic, neutral, and basic amino acids.  This may be explained by different reaction mechanisms of the chiral inversion and hydrolysis of thalidomide.  Chiral inversion is deduced to occur by electrophilic substitution involving specific and general base catalysis, whereas hydrolysis is thought to occur by nucleophilic substitution involving specific and general base as well as nucleophilic catalysis.  As nucleophilic attack is sensitive to steric properties of the catalyst, steric hindrance might be the reason albumin is not able to catalyze hydrolysis.  1H NMR expts. revealed that the three teratogenic metabolites of thalidomide, in sharp contrast to the drug itself, had complete chiral stability.  This leads to the speculation that, were some enantioselectivity to exist in the teratogenicity of thalidomide, it could result from fast hydrolysis to chirally stable teratogenic metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfmXgvPgsp8rVg90H21EOLACvtfcHk0lgq_S_MisY9IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFens7s%253D&md5=1952fa00825e6791eeb8388419f3cc09</span></div><a href="/servlet/linkout?suffix=cit44b&amp;dbid=16384&amp;doi=10.1021%2Ftx9801817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx9801817%26sid%3Dliteratum%253Aachs%26aulast%3DReist%26aufirst%3DM.%26aulast%3DCarrupt%26aufirst%3DP.%26aulast%3DFrancotte%26aufirst%3DE.%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DChiral%2520inversion%2520and%2520hydrolysis%2520of%2520thalidomide%253A%2520mechanisms%2520and%2520catalysis%2520by%2520bases%2520and%2520serum%2520albumin%252C%2520and%2520chiral%2520stability%2520of%2520teratogenic%2520metabolites%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1998%26volume%3D11%26spage%3D1521%26epage%3D1528%26doi%3D10.1021%2Ftx9801817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Goodwin, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conradi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, N. F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, P. S.</span><span> </span><span class="NLM_article-title">Physicochemical determinants of passive membrane permeability: Role of solute hydrogen-bonding potential and volume</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">3721</span><span class="NLM_x">â</span> <span class="NLM_lpage">3729</span><span class="refDoi">Â DOI: 10.1021/jm010253i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010253i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFOmtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3721-3729&author=J.+T.+Goodwinauthor=R.+A.+Conradiauthor=N.+F.+H.+Hoauthor=P.+S.+Burton&title=Physicochemical+determinants+of+passive+membrane+permeability%3A+Role+of+solute+hydrogen-bonding+potential+and+volume&doi=10.1021%2Fjm010253i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical determinants of passive membrane permeability: role of solute hydrogen-bonding potential and volume</span></div><div class="casAuthors">Goodwin, Jay T.; Conradi, Robert A.; Ho, Norman F. H.; Burton, Philip S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3721-3729</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The relationship of solute structure with cellular permeability was probed.  Two series of dipeptide mimetics consisting of glycine, alanine, valine, leucine, phenylalanine, and cyclohexylalanine with amino acids in the D-configuration were prepd.  Partition coeffs. for the peptide mimetics were obtained in the octanol/water (log Poctanol/water), hydrocarbon/octanol (Î log P), and heptane/ethylene glycol (log Pheptane/glycol) systems in order to explore the contributions of solute vol., or surface area, and hydrogen-bond potential to the permeability of the solutes.  Permeability coeffs. were obtained in Caco-2 cell monolayers as a model of the human intestinal mucosa.  The results were interpreted in terms of a partition/diffusion model for solute transport where membrane partitioning into the permeability-limiting membrane microdomain is estd. from the solvent partition coeffs.  Neither log Poctanol/water nor Î log P alone correlated with cellular permeability for all the solutes.  In contrast, log Pheptane/glycol gave a qual. better correlation.  With regard to solute properties, log Poctanol/water is predominantly a measure of solute vol., or surface area, and hydrogen-bond acceptor potential, while Î log P is principally a measure of hydrogen-bond donor strength.  Log Pheptane/glycol contains contributions from all these solute properties.  The results demonstrate that both hydrogen-bond potential and vol. of the solutes contribute to permeability and suggests that the nature of the permeability-limiting microenvironment within the cell depends on the properties of a specific solute.  Collectively, these findings support the conclusion that a general model of permeability will require consideration of a no. of different solute structural properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGociCiZQGkcmLVg90H21EOLACvtfcHk0lgq_S_MisY9IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFOmtrs%253D&md5=b05b1ee611d18cd72ce5652d7f8aa350</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm010253i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010253i%26sid%3Dliteratum%253Aachs%26aulast%3DGoodwin%26aufirst%3DJ.%2BT.%26aulast%3DConradi%26aufirst%3DR.%2BA.%26aulast%3DHo%26aufirst%3DN.%2BF.%2BH.%26aulast%3DBurton%26aufirst%3DP.%2BS.%26atitle%3DPhysicochemical%2520determinants%2520of%2520passive%2520membrane%2520permeability%253A%2520Role%2520of%2520solute%2520hydrogen-bonding%2520potential%2520and%2520volume%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3721%26epage%3D3729%26doi%3D10.1021%2Fjm010253i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Evans, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span> </span><span class="NLM_article-title">Targeting secondary protein complexes in drug discovery: studying the druggability and chemical biology of the HSP70/BAG1 complex</span> <span class="citation_source-journal">Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5167</span><span class="NLM_x">â</span> <span class="NLM_lpage">5170</span><span class="refDoi">Â DOI: 10.1039/C7CC01376K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1039%2FC7CC01376K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=28439591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=5167-5170&author=L.+E.+Evansauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Targeting+secondary+protein+complexes+in+drug+discovery%3A+studying+the+druggability+and+chemical+biology+of+the+HSP70%2FBAG1+complex&doi=10.1039%2FC7CC01376K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting secondary protein complexes in drug discovery: studying the druggability and chemical biology of the HSP70/BAG1 complex</span></div><div class="casAuthors">Evans, Lindsay E.; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5167-5170</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Proteins typically carry out their biol. functions as multi-protein complexes, which can significantly affect the affinity of small-mol. inhibitors.  HSP70 is an important target in oncol., so to study its chem. biol. and the drug discovery potential of the HSP70/BAG1 complex, we designed a high-affinity non-nucleotide fluorescence polarization probe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwTl7NEh65w7Vg90H21EOLACvtfcHk0lhkJmAn0znc9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D&md5=f72aa2f8aba028100f2eb1fd45bf336e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2FC7CC01376K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC01376K%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DTargeting%2520secondary%2520protein%2520complexes%2520in%2520drug%2520discovery%253A%2520studying%2520the%2520druggability%2520and%2520chemical%2520biology%2520of%2520the%2520HSP70%252FBAG1%2520complex%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D5167%26epage%3D5170%26doi%3D10.1039%2FC7CC01376K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Youdim, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avdeef, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbott, N. J.</span><span> </span><span class="NLM_article-title">In vitro trans-monolayer permeability calculations: Often forgotten assumptions</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">997</span><span class="NLM_x">â</span> <span class="NLM_lpage">1003</span><span class="refDoi">Â DOI: 10.1016/S1359-6446(03)02873-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2FS1359-6446%2803%2902873-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=14643163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlCltbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=997-1003&author=K.+A.+Youdimauthor=A.+Avdeefauthor=N.+J.+Abbott&title=In+vitro+trans-monolayer+permeability+calculations%3A+Often+forgotten+assumptions&doi=10.1016%2FS1359-6446%2803%2902873-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47aR"><div class="casContent"><span class="casTitleNuber">47a</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro trans-monolayer permeability calculations: often forgotten assumptions</span></div><div class="casAuthors">Youdim, Kuresh A.; Avdeef, Alex; Abbott, N. Joan</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">997-1003</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  In designing effective therapeutic strategies, novel drugs must exhibit favorable pharmacokinetic properties.  The physicochem. characteristics of a drug, such as pKa, mol. wt., soly. and lipophilicity, will influence the way the drug partitions from the aq. phase into membranes, and thus, will influence its ability to cross cellular barriers, such as the lining of the gastrointestinal tract and the blood-brain barrier.  Physicochem. characteristics also influence the degree to which a drug is able to cross a barrier layer, and the route by which it does this; whether transcellular (across the cells) - by diffusion, carrier-mediated transport or transcytosis - or paracellular - by diffusing through the tight junctions between the cells.  The in vitro model systems that are currently employed to screen the permeation characteristics of a drug often represent a compromise between high throughput with low predictive potential and low throughput with high predictive potential.  Here, we will examine the way in which in vitro cellular permeability assays are often performed and the assumptions that are implied but sometimes forgotten, and we will make simple suggestions for improving the methodol. techniques and math. equations used to det. drug permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr74qqjSfpJsrVg90H21EOLACvtfcHk0liLN74IXeJDyg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlCltbk%253D&md5=fb9e071aba01fc7a0b70a72511385dc8</span></div><a href="/servlet/linkout?suffix=cit47a&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902873-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902873-3%26sid%3Dliteratum%253Aachs%26aulast%3DYoudim%26aufirst%3DK.%2BA.%26aulast%3DAvdeef%26aufirst%3DA.%26aulast%3DAbbott%26aufirst%3DN.%2BJ.%26atitle%3DIn%2520vitro%2520trans-monolayer%2520permeability%2520calculations%253A%2520Often%2520forgotten%2520assumptions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D997%26epage%3D1003%26doi%3D10.1016%2FS1359-6446%2803%2902873-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit47b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Filipe, H. A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salvador, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silvestre, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaz, W. L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, M. J.</span><span> </span><span class="NLM_article-title">Beyond Overtonâs rule: Quantitative modeling of passive permeation through tight cell monolayers</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3696</span><span class="NLM_x">â</span> <span class="NLM_lpage">3706</span><span class="refDoi">Â DOI: 10.1021/mp500437e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp500437e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyntrrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=3696-3706&author=H.+A.+L.+Filipeauthor=A.+Salvadorauthor=J.+M.+Silvestreauthor=W.+L.+C.+Vazauthor=M.+J.+Moreno&title=Beyond+Overton%E2%80%99s+rule%3A+Quantitative+modeling+of+passive+permeation+through+tight+cell+monolayers&doi=10.1021%2Fmp500437e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47bR"><div class="casContent"><span class="casTitleNuber">47b</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond Overton's Rule: Quantitative Modeling of Passive Permeation through Tight Cell Monolayers</span></div><div class="casAuthors">Filipe, H. A. L.; Salvador, A.; Silvestre, J. M.; Vaz, W. L. C.; Moreno, M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3696-3706</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the great challenges in pharmacokinetics is to find a means to optimize the transport across cell barriers.  In this work, permeation across a cell monolayer, such as the tight endothelia in the blood-brain barrier, was modeled using a homologous series of amphipatic mols., 7-nitrobenz-2-oxa-1,3-diazol-4-yl (NBD)-labeled alkyl chain amphiphiles (NBD-Cn, n = 2 to 16), to obtain rules that relate permeant structure to permeability.  The amphiphile enters the system from the serum, equilibrated with serum albumin and lipoproteins, and its sequestration by serum components, interaction with the endothelium, and accumulation in the tissue is followed over time.  The dependence of the permeability coeff. on the no. of carbons of the amphiphile's alkyl chain has a parabolic-like shape.  After a threshold value, an increase in the hydrophobicity of the amphiphile, along the homologous series, results in a decrease in the characteristic rate of permeation to the tissue.  A sensitivity anal. was performed, and the rate limiting steps for permeation of each amphiphile were identified.  Sequestration in the serum and rate of interaction with the endothelium, particularly the rate of desorption, were found to be the determinant processes for some amphiphiles, while for others translocation was the rate limiting step.  Addnl., for some amphiphiles a single rate limiting step could not be identified, with several steps contributing significantly to the overall permeation.  Finally, we derived anal. equations that adequately describe the rate of amphiphile accumulation in the tissue for the cases where permeation is controlled by a single rate limiting step.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrDofWmSchFrVg90H21EOLACvtfcHk0lir4iLoxh1cYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyntrrI&md5=41e911bbe66703e5d3c899bc14a9b505</span></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1021%2Fmp500437e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp500437e%26sid%3Dliteratum%253Aachs%26aulast%3DFilipe%26aufirst%3DH.%2BA.%2BL.%26aulast%3DSalvador%26aufirst%3DA.%26aulast%3DSilvestre%26aufirst%3DJ.%2BM.%26aulast%3DVaz%26aufirst%3DW.%2BL.%2BC.%26aulast%3DMoreno%26aufirst%3DM.%2BJ.%26atitle%3DBeyond%2520Overton%25E2%2580%2599s%2520rule%253A%2520Quantitative%2520modeling%2520of%2520passive%2520permeation%2520through%2520tight%2520cell%2520monolayers%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2014%26volume%3D11%26spage%3D3696%26epage%3D3706%26doi%3D10.1021%2Fmp500437e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit47c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Owen, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nedyalkova, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLaughlin, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, M. S.</span><span> </span><span class="NLM_article-title">Colloidal drug formulations can explain âbell-shapedâ concentration-response curves</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">â</span> <span class="NLM_lpage">784</span><span class="refDoi">Â DOI: 10.1021/cb4007584</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4007584" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=777-784&author=S.+C.+Owenauthor=A.+K.+Doakauthor=A.+N.+Ganeshauthor=L.+Nedyalkovaauthor=C.+K.+McLaughlinauthor=B.+K.+Shoichetauthor=M.+S.+Shoichet&title=Colloidal+drug+formulations+can+explain+%E2%80%9Cbell-shaped%E2%80%9D+concentration-response+curves&doi=10.1021%2Fcb4007584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47cR"><div class="casContent"><span class="casTitleNuber">47c</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal Drug Formulations Can Explain "Bell-Shaped" Concentration-Response Curves</span></div><div class="casAuthors">Owen, Shawn C.; Doak, Allison K.; Ganesh, Ahil N.; Nedyalkova, Lyudmila; McLaughlin, Christopher K.; Shoichet, Brian K.; Shoichet, Molly S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">777-784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug efficacy does not always increase sigmoidally with concn., which has puzzled the community for decades.  Unlike std. sigmoidal curves, bell-shaped concn.-response curves suggest more complex biol. effects, such as multiple-binding sites or multiple targets.  Here, we investigate a phys. property-based mechanism for bell-shaped curves.  Beginning with the observation that some drugs form colloidal aggregates at relevant concns., we detd. concn.-response curves for three aggregating anticancer drugs, formulated both as colloids and as free monomer.  Colloidal formulations exhibited bell-shaped curves, losing activity at higher concns., while monomeric formulations gave typical sigmoidal curves, sustaining a plateau of max. activity.  Inverting the question, we next asked if mols. with bell-shaped curves, reported in the literature, form colloidal aggregates at relevant concns.  We selected 12 mols. reported to have bell-shaped concn.-response curves and found that five of these formed colloids.  To understand the mechanism behind the loss of activity at concns. where colloids are present, we investigated the diffusion of colloid-forming dye Evans blue into cells.  We found that colloidal species are excluded from cells, which may explain the mechanism behind toxicol. screens that use Evans blue, Trypan blue, and related dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg5AlTuU9YnrVg90H21EOLACvtfcHk0lir4iLoxh1cYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D&md5=8cb90ab5f6348326d08ccfcf5377b1f3</span></div><a href="/servlet/linkout?suffix=cit47c&amp;dbid=16384&amp;doi=10.1021%2Fcb4007584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4007584%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DGanesh%26aufirst%3DA.%2BN.%26aulast%3DNedyalkova%26aufirst%3DL.%26aulast%3DMcLaughlin%26aufirst%3DC.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26atitle%3DColloidal%2520drug%2520formulations%2520can%2520explain%2520%25E2%2580%259Cbell-shaped%25E2%2580%259D%2520concentration-response%2520curves%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D777%26epage%3D784%26doi%3D10.1021%2Fcb4007584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span>MoKA; <span class="NLM_publisher-name">Molecular Discovery</span>: <span class="NLM_publisher-loc">Borehamwood, Hertfordshire, UK</span>,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">; </span><a href="http://moldiscovery.com/software/moka" class="extLink">http://www.moldiscovery.com/software/moka</a> (accessed August 28, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MoKA%3B+Molecular+Discovery%3A+Borehamwood%2C+Hertfordshire%2C+UK%2C+2017%3B+http%3A%2F%2Fwww.moldiscovery.com%2Fsoftware%2Fmoka+%28accessed+August+28%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMoKA%26pub%3DMolecular%2520Discovery%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Jones, K.; Rye, C.; Chessum, N.; Cheeseman, M.; Pasqua, A. E.; Pike, K. G.; Faulder, P. F.</span><span> </span><span class="NLM_article-title">Fused 1,4-dihydrodioxin derivatives as inhibitors of heat shock transcription factor 1</span>. WO 2015/049535 A1, April 9,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Jones&author=C.+Rye&author=N.+Chessum&author=M.+Cheeseman&author=A.+E.+Pasqua&author=K.+G.+Pike&author=P.+F.+Faulder&title=Fused+1%2C4-dihydrodioxin+derivatives+as+inhibitors+of+heat+shock+transcription+factor+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DK.%26atitle%3DFused%25201%252C4-dihydrodioxin%2520derivatives%2520as%2520inhibitors%2520of%2520heat%2520shock%2520transcription%2520factor%25201%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="note"><p class="first last">The poor aqueous stability of the thalidomide derived PDPs made measurement of their assay free fractions by dialysis, as well as standard permeability assessments using PAMPA and CACO-2, unreliable so it was not possible to compare PDP probes at their free concentrations or to determine permeability values.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Kisselev, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldberg, A. L.</span><span> </span><span class="NLM_article-title">Proteasome inhibitors: From research tools to drug candidates</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">â</span> <span class="NLM_lpage">758</span><span class="refDoi">Â DOI: 10.1016/S1074-5521(01)00056-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2FS1074-5521%2801%2900056-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=11514224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtV2ku7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=739-758&author=A.+F.+Kisselevauthor=A.+L.+Goldberg&title=Proteasome+inhibitors%3A+From+research+tools+to+drug+candidates&doi=10.1016%2FS1074-5521%2801%2900056-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome inhibitors: from research tools to drug candidates</span></div><div class="casAuthors">Kisselev, A. F.; Goldberg, A. L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">739-758</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The 26S proteasome is a 2.4 MDa multifunctional ATP-dependent proteolytic complex, which degrades the majority of cellular polypeptides by an unusual enzyme mechanism.  Several groups of proteasome inhibitors have been developed and are now widely used as research tools to study the role of the ubiquitin-proteasome pathway in various cellular processes, and two inhibitors are now in clin. trials for treatment of multiple cancers and stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTXBSmAzp9vLVg90H21EOLACvtfcHk0ljJliKzf6HuBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtV2ku7g%253D&md5=b729e5f56c402172aca5b139df61c09e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2801%2900056-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252801%252900056-4%26sid%3Dliteratum%253Aachs%26aulast%3DKisselev%26aufirst%3DA.%2BF.%26aulast%3DGoldberg%26aufirst%3DA.%2BL.%26atitle%3DProteasome%2520inhibitors%253A%2520From%2520research%2520tools%2520to%2520drug%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D739%26epage%3D758%26doi%3D10.1016%2FS1074-5521%2801%2900056-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Diz, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvajal-Rodriguez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skibinski, D. O. F.</span><span> </span><span class="NLM_article-title">Multiple hypothesis testing in proteomics: A strategy for experimental work</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">M110.004374</span><span class="refDoi">Â DOI: 10.1074/mcp.M110.004374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1074%2Fmcp.M110.004374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=21364085" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M110.004374&author=A.+P.+Dizauthor=A.+Carvajal-Rodriguezauthor=D.+O.+F.+Skibinski&title=Multiple+hypothesis+testing+in+proteomics%3A+A+strategy+for+experimental+work&doi=10.1074%2Fmcp.M110.004374"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52a&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M110.004374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M110.004374%26sid%3Dliteratum%253Aachs%26aulast%3DDiz%26aufirst%3DA.%2BP.%26aulast%3DCarvajal-Rodriguez%26aufirst%3DA.%26aulast%3DSkibinski%26aufirst%3DD.%2BO.%2BF.%26atitle%3DMultiple%2520hypothesis%2520testing%2520in%2520proteomics%253A%2520A%2520strategy%2520for%2520experimental%2520work%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DM110.004374%26doi%3D10.1074%2Fmcp.M110.004374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit52b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Noble, W. S.</span><span> </span><span class="NLM_article-title">How does multiple testing correction work?</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1135</span><span class="NLM_x">â</span> <span class="NLM_lpage">1137</span><span class="refDoi">Â DOI: 10.1038/nbt1209-1135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1038%2Fnbt1209-1135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=20010596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFemtr3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=1135-1137&author=W.+S.+Noble&title=How+does+multiple+testing+correction+work%3F&doi=10.1038%2Fnbt1209-1135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52bR"><div class="casContent"><span class="casTitleNuber">52b</span><div class="casTitle"><span class="NLM_cas:atitle">How does multiple testing correction work?</span></div><div class="casAuthors">Noble, William S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1135-1137</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  When prioritizing hits from a high-throughput expt., it is important to correct for random events that falsely appear significant.  How is this done and what methods should be used.  Techniques for doing this are discussed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6wxnVhdu_rVg90H21EOLACvtfcHk0ljWzPcztJ8BaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFemtr3N&md5=7376f0b000468adffe6e6c681ef84c95</span></div><a href="/servlet/linkout?suffix=cit52b&amp;dbid=16384&amp;doi=10.1038%2Fnbt1209-1135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1209-1135%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DW.%2BS.%26atitle%3DHow%2520does%2520multiple%2520testing%2520correction%2520work%253F%26jtitle%3DNat.%2520Biotechnol.%26date%3D2009%26volume%3D27%26spage%3D1135%26epage%3D1137%26doi%3D10.1038%2Fnbt1209-1135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Thorarensen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banker, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fensome, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telliez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juba, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czerwinski, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casimiro-Garcia, A.</span><span> </span><span class="NLM_article-title">ATP-mediated kinome selectivity: The missing link in understanding the contribution of individual JAK kinase isoforms to cellular signaling</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1552</span><span class="NLM_x">â</span> <span class="NLM_lpage">1558</span><span class="refDoi">Â DOI: 10.1021/cb5002125</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5002125" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslart7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1552-1558&author=A.+Thorarensenauthor=M.+E.+Bankerauthor=A.+Fensomeauthor=J.+Telliezauthor=B.+Jubaauthor=F.+Vincentauthor=R.+M.+Czerwinskiauthor=A.+Casimiro-Garcia&title=ATP-mediated+kinome+selectivity%3A+The+missing+link+in+understanding+the+contribution+of+individual+JAK+kinase+isoforms+to+cellular+signaling&doi=10.1021%2Fcb5002125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53aR"><div class="casContent"><span class="casTitleNuber">53a</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-Mediated Kinome Selectivity: The Missing Link in Understanding the Contribution of Individual JAK Kinase Isoforms to Cellular Signaling</span></div><div class="casAuthors">Thorarensen, Atli; Banker, Mary Ellen; Fensome, Andrew; Telliez, Jean-Baptiste; Juba, Brian; Vincent, Fabien; Czerwinski, Robert M.; Casimiro-Garcia, Agustin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1552-1558</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases constitute an important class of therapeutic targets being explored both by academia and the pharmaceutical industry.  The major focus of this effort was directed toward the identification of ATP competitive inhibitors.  Although it has long been recognized that the intracellular concn. of ATP is very different from the concns. utilized in biochem. enzyme assays, little thought has been devoted to incorporating this discrepancy into our understanding of translation from enzyme inhibition to cellular function.  Significant work has been dedicated to the discovery of JAK kinase inhibitors; however, a disconnect between enzyme and cellular function is prominently displayed in the literature for this class of inhibitors.  Herein, we demonstrate utilizing the four JAK family members (JAK1, JAK2, JAK3, and TYK2) that the difference in the ATP Km of each individual kinase has a significant impact on the enzyme to cell inhibition translation.  We evaluated a large no. of JAK inhibitors in enzymic assays utilizing either 1 mM ATP or Km ATP for the four isoforms as well as in primary cell assays.  This data set provided the opportunity to examine individual kinase contributions to the heterodimeric kinase complexes mediating cellular signaling.  In contrast to a recent study, we demonstrate that for IL-15 cytokine signaling it is sufficient to inhibit either JAK1 or JAK3 to fully inhibit downstream STAT5 phosphorylation.  This addnl. data thus provides a crit. piece of information explaining why JAK1 has incorrectly been thought to have a dominant role over JAK3.  Beyond enabling a deeper understanding of JAK signaling, conducting similar analyses for other kinases by taking into account potency at high ATP rather than Km ATP may provide crucial insights into a compd.'s activity and selectivity in cellular contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYiMVl7OpPFLVg90H21EOLACvtfcHk0ljWzPcztJ8BaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslart7s%253D&md5=c87874cb1748e13eef2aa241ecef3e79</span></div><a href="/servlet/linkout?suffix=cit53a&amp;dbid=16384&amp;doi=10.1021%2Fcb5002125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5002125%26sid%3Dliteratum%253Aachs%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DFensome%26aufirst%3DA.%26aulast%3DTelliez%26aufirst%3DJ.%26aulast%3DJuba%26aufirst%3DB.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DCzerwinski%26aufirst%3DR.%2BM.%26aulast%3DCasimiro-Garcia%26aufirst%3DA.%26atitle%3DATP-mediated%2520kinome%2520selectivity%253A%2520The%2520missing%2520link%2520in%2520understanding%2520the%2520contribution%2520of%2520individual%2520JAK%2520kinase%2520isoforms%2520to%2520cellular%2520signaling%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1552%26epage%3D1558%26doi%3D10.1021%2Fcb5002125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit53b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Smyth, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Measuring and interpreting the selectivity of protein kinase inhibitors</span> <span class="citation_source-journal">J. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">â</span> <span class="NLM_lpage">151</span><span class="refDoi">Â DOI: 10.1007/s12154-009-0023-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1007%2Fs12154-009-0023-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=19568781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=131-151&author=L.+A.+Smythauthor=I.+Collins&title=Measuring+and+interpreting+the+selectivity+of+protein+kinase+inhibitors&doi=10.1007%2Fs12154-009-0023-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53bR"><div class="casContent"><span class="casTitleNuber">53b</span><div class="casTitle"><span class="NLM_cas:atitle">Measuring and interpreting the selectivity of protein kinase inhibitors</span></div><div class="casAuthors">Smyth Lynette A; Collins Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">131-51</span>
        ISSN:<span class="NLM_cas:issn">1864-6158</span>.
    </div><div class="casAbstract">Protein kinase inhibitors are a well-established class of clinically useful drugs, particularly for the treatment of cancer.  Achieving inhibitor selectivity for particular protein kinases often remains a significant challenge in the development of new small molecules as drugs or as tools for chemical biology research.  This review summarises the methodologies available for measuring kinase inhibitor selectivity, both in vitro and in cells.  The interpretation of kinase inhibitor selectivity data is discussed, particularly with reference to the structural biology of the protein targets.  Measurement and prediction of kinase inhibitor selectivity will be important for the development of new multi-targeted kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzU9rA5WZcz9FxYDaSSP8XfW6udTcc2eZTYQM4StPjM7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D&md5=f78ff1202a949a71ffd65e804298c37f</span></div><a href="/servlet/linkout?suffix=cit53b&amp;dbid=16384&amp;doi=10.1007%2Fs12154-009-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12154-009-0023-9%26sid%3Dliteratum%253Aachs%26aulast%3DSmyth%26aufirst%3DL.%2BA.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMeasuring%2520and%2520interpreting%2520the%2520selectivity%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D2%26spage%3D131%26epage%3D151%26doi%3D10.1007%2Fs12154-009-0023-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit53c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Knight, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Features of selective kinase inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">621</span><span class="NLM_x">â</span> <span class="NLM_lpage">637</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2005.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1016%2Fj.chembiol.2005.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=15975507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFGhtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=621-637&author=Z.+A.+Knightauthor=K.+M.+Shokat&title=Features+of+selective+kinase+inhibitors&doi=10.1016%2Fj.chembiol.2005.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53cR"><div class="casContent"><span class="casTitleNuber">53c</span><div class="casTitle"><span class="NLM_cas:atitle">Features of Selective Kinase Inhibitors</span></div><div class="casAuthors">Knight, Zachary A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">621-637</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Summary: Small-mol. inhibitors of protein and lipid kinases have emerged as indispensable tools for studying signal transduction.  Despite the widespread use of these reagents, there is little consensus about the biochem. criteria that define their potency and selectivity in cells.  We discuss some of the features that det. the cellular activity of kinase inhibitors and propose a framework for interpreting inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAJWhVxtSrCLVg90H21EOLACvtfcHk0lhqxoaryv-MSA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsFGhtrc%253D&md5=ce5b3505a0326b04479172ec05573152</span></div><a href="/servlet/linkout?suffix=cit53c&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DFeatures%2520of%2520selective%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D621%26epage%3D637%26doi%3D10.1016%2Fj.chembiol.2005.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Strelow, J. M.</span><span> </span><span class="NLM_article-title">A perspective on the kinetics of covalent and irreversible inhibition</span> <span class="citation_source-journal">SLAS Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">â</span> <span class="NLM_lpage">20</span><span class="refDoi">Â DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&author=J.+M.+Strelow&title=A+perspective+on+the+kinetics+of+covalent+and+irreversible+inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54aR"><div class="casContent"><span class="casTitleNuber">54a</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0ljI9ZlJ2LEbew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit54a&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520perspective%2520on%2520the%2520kinetics%2520of%2520covalent%2520and%2520irreversible%2520inhibition%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note">b<p class="last">Competition of PDP 16 at 6 h exposure with control compounds <b>1</b> and <b>22</b> was unsuccessful at concentration > 1 Î¼M (data not shown).</p></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT','PDB','5JCT'); return false;">PDB: 5JCT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CI1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CI1','PDB','5CI1'); return false;">PDB: 5CI1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i44"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01406">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04854"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01406">10.1021/acs.jmedchem.7b01406</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full chemistry experimental, NMR spectra of final compounds, physicochemical properties and PDP stability experimental, PDP design pictures, biochemical experimental, biology experimental, proteomics experimental, additional information, proteomics data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_001.pdf">jm7b01406_si_001.pdf (17.36 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01406/suppl_file/jm7b01406_si_002.csv">jm7b01406_si_002.csv (0.71 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01406&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-3%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01406%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01406" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997076be40d1a7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
