
<html lang="en"     class="pb-page"  data-request-id="e851264d-2c23-42cc-8854-86ed09b75ec6"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01709;issue:issue:10.1021/jmcmar.2019.62.issue-24;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72" /></meta><meta name="dc.Creator" content="Jonathan  Pettinger" /></meta><meta name="dc.Creator" content="Michael  Carter" /></meta><meta name="dc.Creator" content="Keith  Jones" /></meta><meta name="dc.Creator" content="Matthew D.  Cheeseman" /></meta><meta name="dc.Description" content="The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chemical probes and drug..." /></meta><meta name="Description" content="The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chemical probes and drug..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 14, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01709" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01709" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01709" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01709" /></link>
        
    
    

<title>Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01709" /></meta><meta property="og:title" content="Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0004.jpeg" /></meta><meta property="og:description" content="The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chemical probes and drugs. The success of this strategy has centered on exposed cysteine residues as nucleophiles but the low abundance of cysteine in the proteome has limited its application. We have recently reported our discovery that lysine-56 in the difficult-to-drug target HSP72 could form a covalent bond with a small-molecule inhibitor. We now disclose the optimization of these targeted covalent inhibitors using rational design. Essential to our optimization was the development of a new covalent fluorescence polarization assay, which allows for the direct measurement of the key kinetic parameter in covalent inhibitor design, kinact/KI, extrapolation of the underlying parameters, kinact and Ki, and direct comparison to reversible analogues. Using our approach, we demonstrate a &gt;100-fold enhancement in covalent efficiency and key learnings in lysine-selective electrophile optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01709"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01709">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01709&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01709&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01709&amp;href=/doi/10.1021/acs.jmedchem.9b01709" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 11383-11398</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01697" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01767" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jonathan Pettinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan Pettinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan++Pettinger">Jonathan Pettinger</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Michael Carter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Carter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Carter">Michael Carter</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Keith Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Keith Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Keith++Jones">Keith Jones</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9440-4094" title="Orcid link">http://orcid.org/0000-0002-9440-4094</a></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Cheeseman</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Cheeseman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4c212d383824293b622f2429293f29212d220c252f3e622d2f623927"><span class="__cf_email__" data-cfemail="731e1207071b16045d101b161600161e121d331a10015d12105d0618">[emailÂ protected]</span></a>. Phone: (+44) 208 722 4168.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Cheeseman">Matthew D. Cheeseman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1121-6985" title="Orcid link">http://orcid.org/0000-0003-1121-6985</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01709&amp;href=/doi/10.1021%2Facs.jmedchem.9b01709" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 11383â11398</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 14, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 October 2019</li><li><span class="item_label"><b>Published</b> online</span>14 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01709" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01709</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01709"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4184</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01709" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;Pettinger&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Carter&quot;},{&quot;first_name&quot;:&quot;Keith&quot;,&quot;last_name&quot;:&quot;Jones&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Cheeseman&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;11383-11398&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01709&quot;},&quot;abstract&quot;:&quot;The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chemical probes and drugs. The success of this strategy has centered on exposed cysteine residues as nucleophiles but the low abundance of cysteine in the proteome has limited its application. We have recently reported our discovery that lysine-56 in the difficult-to-drug target HSP72 could form a covalent bond with a small-molecule inhibitor. We now disclose the optimization of these targeted covalent inhibitors using rational design. Essential to our optimization was the development of a new covalent fluorescence polarization assay, which allows for the direct measurement of the key kinetic parameter in covalent inhibitor design, kinact/KI, extrapolation of the underlying parameters, kinact and Ki, and direct comparison to reversible analogues. Using our approach, we demonstrate a &gt;100-fold enhancement in covalent efficiency and key lear&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01709&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01709" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01709&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01709" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01709&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01709" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01709&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01709&amp;href=/doi/10.1021/acs.jmedchem.9b01709" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01709" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01709" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01709%26sid%3Dliteratum%253Aachs%26pmid%3D31725295%26genre%3Darticle%26aulast%3DPettinger%26date%3D2019%26atitle%3DKinetic%2BOptimization%2Bof%2BLysine-Targeting%2BCovalent%2BInhibitors%2Bof%2BHSP72%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D24%26spage%3D11383%26epage%3D11398%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291922" title="Electrophiles">Electrophiles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/jmcmar.2019.62.issue-24/20191226/jmcmar.2019.62.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chemical probes and drugs. The success of this strategy has centered on exposed cysteine residues as nucleophiles but the low abundance of cysteine in the proteome has limited its application. We have recently reported our discovery that lysine-56 in the difficult-to-drug target HSP72 could form a covalent bond with a small-molecule inhibitor. We now disclose the optimization of these targeted covalent inhibitors using rational design. Essential to our optimization was the development of a new covalent fluorescence polarization assay, which allows for the direct measurement of the key kinetic parameter in covalent inhibitor design, <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>, extrapolation of the underlying parameters, <i>k</i><sub>inact</sub> and <i>K</i><sub><i>i</i></sub>, and direct comparison to reversible analogues. Using our approach, we demonstrate a >100-fold enhancement in covalent efficiency and key learnings in lysine-selective electrophile optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64254" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64254" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Despite many of our most important drugs utilizing irreversible covalent inhibition of an enzyme,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> concerns relating to idiosyncratic toxicity led to the near-exclusion of this mechanism of action (MOA) from drug discovery programs.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The recent renaissance in covalent inhibitors is in large part due to their inherent advantage over reversible counterparts for antagonising proteins that have high-affinity, high-abundance natural substrates.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> While the previous generation of irreversible inhibitor drugs were discovered by serendipity or were natural products, the rational design strategy for modern targeted covalent inhibitors (TCIs) focuses on exploiting high-resolution small-molecule/protein X-ray crystal structures of high-affinity reversible ligands to target active site, solvent-exposed cysteine residues with sparingly reactive electrophiles.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Unfortunately, the rarity of cysteine in the proteome has limited its application,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> leading to an increased interest in targeting other potentially nucleophilic residues.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6â8)</a></div><div class="NLM_p">Depending on the length of exposure and the concentration, TCIs utilize both reversible and irreversible occupancy of a protein (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Defining a TCI only via an IC<sub>50</sub> value can be limiting for rational design, as the inhibitor (I) will inevitably appear more potent with increasing preincubation time. TCIs typically react via a two-step MOA (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), initially binding to the protein (E) in a reversible manner to generate a noncovalent complex (EI). The occupancy of the reversible complex is determined by the free concentration of the TCI and the equilibrium constant <i>K</i><sub><i>i</i></sub>. The reversible complex can then undergo covalent bond formation as determined by the first-order rate constant <i>k</i><sub>inact</sub>, to give the covalent complex (E-I). These fundamental parameters describe the efficiency of the TCI but cannot be determined directly from assay data.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Simulated data describing the MOA and kinetic parameters used to quantify the activity of TCIs. (A) TCI two-step binding mechanism. (B) The two components of TCI MOA, both reversible and covalent occupancy, contribute to the total target occupancy at a given concentration and time. Left = covalent occupancy (%CO) and right = total occupancy (%TO), both simulated using <i>K</i><sub><i>I</i></sub> = 1 Î¼M and <i>k</i><sub>inact</sub> = 0.069 min<sup>â1</sup>. FO = fraction reversible occupancy (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a> for derivation). (C) Determination of the essential second-order rate constant <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> from the concentration-dependent <i>k</i><sub>obs</sub> pseudo-first-order rate constant. Left = example where <i>K</i><sub><i>I</i></sub> < [I] so <i>k</i><sub>inact</sub> and <i>K</i><sub><i>I</i></sub> can be deconvoluted. Right = example where <i>K</i><sub><i>I</i></sub> â« [I] so the individual kinetic parameters cannot be distinguished. (D) Nucleoside-derived reversible <b>1</b> and covalent inhibitor <b>2</b> of HSP72.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">TCI activity is described by an equation analogous to the MichaelisâMenten equation. When normalized for protein concentration, the rate of covalent bond formation can be quantified by the pseudo-first-order rate constant, <i>k</i><sub>obs</sub> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). The reversible binding event, <i>k</i><sub>obs</sub> is not a true constant, as its value is dependent on the concentration of the TCI. At TCI concentrations approaching binding-site saturation, <i>k</i><sub>obs</sub> tends to the constant, <i>k</i><sub>inact</sub>, equivalent to the half-life of the reaction at a theoretical infinite concentration (<i>t</i><sub>1/2</sub><sup>inf</sup>).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> [I] at <i>k</i><sub>inact</sub>/2 determines the pseudo-equilibrium constant <i>K</i><sub><i>I</i></sub>, equivalent to <i>K</i><sub>m</sub> when describing enzyme substrates. The true reversible equilibrium constant for the process, <i>K</i><sub><i>i</i></sub>, is often used interchangeably with <i>K</i><sub><i>I</i></sub>, but this is only a fair assumption when <i>k</i><sub>inact</sub> âª <i>k</i><sub>off</sub><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and may not be true for tight reversible binding TCIs. At concentrations much lower than <i>K</i><sub><i>I</i></sub>, the response of <i>k</i><sub>obs</sub> to changing TCI concentration becomes linear. The gradient of this relationship gives the second-order rate constant <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>, which is the key kinetic parameter that describes the efficiency of the reaction, and its optimization is the primary goal of any TCI discovery effort toward a chemical probe (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).</div><div class="NLM_p">Analysis of the <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> parameter clearly demonstrates that there are two strategies for TCI optimization: first, through the reduction of <i>K</i><sub><i>I</i></sub> by increasing the reversible affinity of the ligand for the target protein, and second by increasing <i>k</i><sub>inact</sub>. The optimization of <i>k</i><sub>inact</sub> differs from simply increasing the intrinsic reactivity of the electrophile, as this would likely lead to a greater off-target activity; instead, <i>k</i><sub>inact</sub> optimization focuses on the particular environment within the protein-binding site compared to bulk aqueous solvent.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The effect of solvent dielectric constant, proximal residues, perturbed <i>pK</i><sub>a</sub>, and the tightly controlled bond angles and distances resulting from the reversible binding of the ligand can lead to a dramatically enhanced rate of covalent bond formation and high selectivity. This effect of binding-site rate enhancement has led to effective, highly selective, and successful TCI design of KRAS G12C inhibitors, which display a very weak reversible affinity but exploit extremely high <i>k</i><sub>inact</sub> values with sparingly low intrinsic reactivity electrophiles, resulting in <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> values suitable for in vivo efficacy.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a></div><div class="NLM_p">Heat shock 70 kDa protein 1 (HSP72) is a member of the HSP70 family of molecular chaperones. It is an ATPase that binds misfolded proteins, stabilizing the cellular environment and allowing the cell to return to homeostasis.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> HSP72 is induced in an HSF1-dependent manner when the cell is undergoing stress and is overexpressed in several cancer cell types.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This overexpression is correlated with metastasis, poor prognosis, and resistance to chemotherapy in patients.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Because of the clear role of HSP72 in cancer, it has become an important target in drug discovery, but despite considerable research effort, there is currently no potent, selective, cellularly active chemical probe to study the function of HSP72 in cancer cells.</div><div class="NLM_p">The nucleotide-binding domain (NBD) of HSP72 (HSP72-NBD) can be reversibly targeted with a series of <i>bis</i>-aryl nucleoside-derived inhibitors (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D), which display a very high apparent affinity in biochemical assays but suffer a steep dropoff in activity in cancer cells.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> We hypothesized that the poor cellular activity of this chemotype was due to competition with the high-affinity (<i>K</i><sub>M</sub> for ATP = â¼1 Î¼M), high-abundance (â¼5 mM)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> substrate of HSP72, ATP. We have recently reported the discovery that lysine-56 can be selectivity targeted with an acrylate-derived TCI <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Targeting lysine residues with TCIs presents a number of unique challenges and is still in its infancy,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> but the greater prevalence of lysine in the proteome<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> could result in more wide-ranging applications of the irreversible inhibitor paradigm than has so far been possible through the rational targeting of cysteine.</div><div class="NLM_p last">As part of our continuing effort to develop strategies and techniques to discover lysine-targeting covalent ligands, we now report the disclosure of an HSP72 TCI, where our approach led to a 108-fold improvement in <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>. Critical to the optimization was the development of a covalent fluorescence polarization (covalent FP) assay that can distinguish between the reversible and covalent components of target occupancy, allowing for a direct comparison between covalent and reversible analogues, which is essential for correct structure activity relationship (SAR) interpretation. The covalent FP-assay will be widely applicable to the optimization of TCIs of other nonenzymatic or poorly catalytic proteins.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Analysis of the First-Generation Lysine-56 TCI Binding Mode and Optimization of <i>K</i><sub><i>i</i></sub></h3><div class="NLM_p">Exploiting extensive mass spectrometry studies, single-point mutant proteins, and through the design of several key control compounds, we had previously demonstrated that our first-generation acrylate TCI <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D) was both highly selective for lysine-56 and that an initial reversible binding step was critical to its MOA. However, the rate of reaction of <b>2</b> was very slow, requiring >24 h exposure, even at high concentrations, before the intact-protein mass spectrometry (<b>MS</b>) indicated complete formation of the covalent complex.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Therefore, our aim was to develop a strategy to optimize lysine-56-targeting TCIs of HSP72 toward a potentially cell active chemical probe.</div><div class="NLM_p">To redesign the second-generation TCI with increased potency, we analyzed our two previously reported co-crystal structures of HSP72-NBD with acrylate TCI <b>2</b> reversibly bound.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These structures revealed two distinct binding modes and protein-residue conformations, which we hypothesized were both essential for the efficiency of the covalent inhibition MOA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of Lys56-targeting TCI binding modes. The tertiary conformation of the protein is maintained in the previously described semiopen form in all structures and modeling. The polar interactions of the adenine-type base and ribose are maintained in each binding mode. All diagrams were adapted from analysis using MOE (2014.09) and PyMOL Molecular Graphics System, Version 2.0 SchrÃ¶dinger, LLC. 2.2.3 A: Tyr15 up-conformation. The acrylate electrophile of <b>2</b> forms a Ï-stack interaction with Tyr15 and a hydrogen bond with Thr-37. In this reversible complex, the electrophile is too far from Lys56 to form a covalent bond (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKR">5MKR</a>). B: Tyr15 downconformation. Following rotation of Tyr15, rotation around the linker of <b>2</b> now positions the acrylate electrophile in close proximity to Lys56, suitable for covalent-bond formation in this precovalent complex. The acrylate moiety was not observed in the electron density due to flexibility, so was modeled to estimate distances (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKS">5MKS</a>). C: Binding mode of the high-affinity reversible HSP72 inhibitor <b>1</b> in the Tyr15 upconformation (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IO8">4IO8</a>). D: Model of a putative binding mode for a high-affinity Lys56-targeting TCI. With Tyr15 in a downconformation, rotation of the benzylic ether positions a potential <i>para</i>-electrophile at an appropriate distance from Lys56 in a precovalent complex. E: Three-step MOA, TCI binds HSP72 to form the reversible complex (EI) before transition to the precovalent complex (EI*) that can result in the formation of the covalent bond and the irreversible complex (E-I).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the putative reversible complex of acrylate TCI <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKR">5MKR</a>),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Tyr15 was in an upconformation, blocking any direct vector from the ligand to the nucleophilic Lys56. The electrophilic acrylate moiety formed an eclipsed conformation, resulting in a Ï-stack interaction with Tyr15 and a hydrogen bond to Thr37. In the putative precovalent complex (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKS">5MKS</a>),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> the electrophile was observed in an elongated conformation and with Tyr15 in a downconformation.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> The conformational flip of Tyr15 allowed the acrylate moiety to position proximally to the Lys56 nucleophile, which is essential for covalent-bond formation. The hydrogen-bonding array of the 8-aminoadenosine-motif and the position of the lipophilic <i>p</i>-chlorophenyl substituent were maintained in both binding modes. From this analysis, we proposed that the MOA of an HSP72 TCI proceeds via a three-step mechanism (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E). First, the TCI binds HSP72 to give a reversible complex (EI) that would contribute to the reversible occupancy but could not lead directly to covalent occupancy. Transition of the reversible complex (EI) to the precovalent complex (EI*) would be essential for the formation of the covalent bond and the resulting irreversible complex (E-I). Optimization of the affinity for the reversible complex would still result in increased potency of the covalent bond formation, although <i>K<sub>i</sub></i> would now describe a pseudo-equilibrium constant as a combination of the microscopic rate constants for the formation of the encounter complex and conformational switch, in a manner consistent with an induced fit binding MOA.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This would also assume that the conformational flexibility of Tyr15 is sufficient to allow transition to the precovalent complex and access the Lys56 nucleophile so that covalent bond formation is rate determining in <i>k</i><sub>inact</sub>.</div><div class="NLM_p">To optimize the reversible affinity of our TCI, we aimed to exploit the high-affinity 5â²-<i>O</i>-benzyl class of HSP72 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Analysis of the reversible complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IO8">4IO8</a>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> of nitrile <b>1</b> (FP-Assay <i>pK</i><sub><i>i</i></sub> > 6.70, <i>K</i><sub><i>i</i></sub> = <0.20 Î¼M, <i>N</i> = 3)<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> revealed that this ligand forms similar polar and lipophilic interactions in the 8-aminoadenosine- and <i>p</i>-chlorophenyl-regions of HSP72-NBD as our acrylate TCI <b>2</b>. Importantly, the 5â²-<i>O-</i>benzyl substituent occupies the same pocket as the acrylate moiety, so we hypothesized that a benzyl group with an appropriately positioned electrophile could exploit both the high-affinity reversible complex and could transition to the precovalent complex (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D) in a similar putative three-step mechanism to our acrylate TCI <b>2</b>. To assess whether the proposed transition of the reversible to the precovalent complex was viable (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), we carried out a rapid overlay of chemical structures (ROCS)<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> ligand-based analysis on the favorability of the linker torsional angles in this conformationally restricted structure, as this would be critical if we were to incorporate a 5â²-benzylic substituent in our TCI design. Through alignment of the 5â²-<i>O</i>-benzyl motif of <b>1</b> to the precovalent complex acrylate conformation of <b>2</b> and by comparison with known conformations of this chemotype in the Cambridge Structural Database (CSD)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C), it was demonstrated that a second-generation inhibitor could adopt an acceptable benzylic torsional angle, resulting in a viable conformation for covalent-bond formation. Finally, in this conformation, the <i>para</i>-position gave the shortest distance to Lys56 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), so an electrophile at this position was incorporated to complete our rational design hypothesis for the second-generation HSP72 TCI.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Conformation analysis of the proposed second-generation HSP72 TCI. A: Overlay of the acrylate TCI <b>2</b> (green) and the high-affinity reversible inhibitor <b>1</b> (cyan). B: Overlay of the acrylate TCI <b>2</b> (green) and the proposed conformation of the second-generation TCI (cyan) in the precovalent complex. The generic electrophile is represented as a purple sphere. C: Frequency distribution of torsional angles for 5â²-<i>O</i>-adenosine derivatives observed in the CSD; the green line represents the desired torsional angle. D: At the desired torsional angle, the modeled distances to Lys56 from different ring positions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Selection and Synthesis of a Lysine-Targeting Warhead and Optimization of <i>k</i><sub>inact</sub></h3><div class="NLM_p">Our first-generation TCI <b>2</b> utilized an acrylate warhead to form the covalent bond, which we hypothesized was suboptimal for targeting lysine residues in proteins. Lysine is a hard nucleophile and hence should display an enhanced rate of reaction with hard electrophiles, but few lysine-selective electrophiles have so far been described in the literature.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> A recent study by Campos et al. successfully exploited activated phenolic esters as hard electrophiles to target the catalytic lysine of PI3KÎ´.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> To incorporate this concept into the design of our second-generation TCI, we developed a synthetic strategy that added an activated ester with a <i>p</i>-fluorophenol leaving group into the 5â²-<i>para</i>-benzylic vector we had identified from our TCI MOA analysis.</div><div class="NLM_p">The synthesis of the second-generation TCI began with benzylation of the 5â²-hydroxyl of 2â²,3â²-acetonide-protected 6-chlororiboside <b>3</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The order of addition is essential in this transformation to avoid oligomerization at the 6-chloro position; <b>3</b> was first treated with 4-(bromomethyl) benzonitrile, followed by exposure to NaH at 0 Â°C,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> which gave the 5â²-ether <b>4</b> in 56% yield. S<sub>N</sub>Ar displacement with ammonia at the 6-position before selective oxidation with bromine at the 8-position gave <b>5</b> in 59% yield over two steps as single regioisomer. A second S<sub>N</sub>Ar displacement with 4-chlorobenzylamine gave the key covalent precursor <b>6</b> in 66% yield. To synthesize the lysine-targeting warhead, the nitrile moiety of <b>6</b> was hydrolyzed under basic conditions to give benzoic acid <b>7</b>, which then underwent coupling with 4-fluorophenol using standard HATU conditions, and following acetonide deprotection, gave the second-generation TCI <b>8</b> in seven steps and 5% overall yield. Deprotection of the intermediate <b>6</b> gave the reversible molecular matched pair (MMP) <b>9</b> in 47% yield.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0003.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Second-Generation Lysine-56-Targeting TCI<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 4-(bromomethyl) benzonitrile, NaH (60% in mineral oil), dimethylformamide (DMF), room temperature (RT), 1 h, 56%; (ii) 7 M NH<sub>3</sub>/MeOH, 110 Â°C, MW, 2 h, 82%; (iii) Br<sub>2</sub>, K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O, H<sub>2</sub>O, 1,4-dioxane, RT, 1 h, 72%; (iv) 4-chlorobenzylamine, EtOH, 160 Â°C, MW, 1 h, 66%; (v) 2 M NaOH, EtOH, 100 Â°C 3 h, 58%; (vi) 4-fluorophenol, HATU, DIPEA, DMF, RT, 18 h, 47%; (vii) TFA/H<sub>2</sub>O (5:2), RT, 0.5 h, 85%; (viii) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 47%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Characterization of the Second-Generation TCI <b>8</b></h3><div class="NLM_p">To investigate the reversible affinity and the potential to form a covalent bond with Lys56 in HSP72-NBD with our second-generation TCI <b>8</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 3; <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 2), we repeated our previously described analysis using the nucleotide-derived FP-assay, comparing the data to the first-generation TCI <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 1; <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 1). Briefly, displacement of the nucleotide-derived FP probe by the ligand was used to determine an apparent (App.) <i>K</i><sub><i>i</i></sub>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Because the bound fraction of the probe is dependent on the effective concentration of the protein, a time-dependent decrease following covalent bond formation should result in a shift in the binding curve for the TCI. The time-dependent FP-assay data revealed that the initial reversible binding affinity of activated ester <b>8</b> displayed a 7-fold improvement over our first-generation TCI <b>2</b> but was >13-fold weaker than the tight-binding reversible nitrile MMP <b>9</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 2; <i>pK</i><sub><i>i</i></sub> > 6.70, <i>K</i><sub><i>i</i></sub> = <0.20 Î¼M, <i>N</i> = 3). Disappointingly, there was no clear time dependence in the App. <i>K<sub>i</sub></i>, and consistent with this result, analysis of the intact-protein MS data for <b>8</b> also revealed no evidence of specific and selective covalent-bond formation.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Kinetic and Affinity Analysis of Covalent and Noncovalent Inhibitors of HSP72<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>â</sup><a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0009.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All data were processed and analyzed using GraphPad Prism 7.04. All values are quoted to two significant figures. NA = not applicable, ND = not determined.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">App. <i>K</i><sub><i>i</i></sub> = Apparent <i>K</i><sub><i>i</i></sub>. Each concentration represents <i>n</i> = 3 statistical repeats, arithmetic mean Â± standard error of the mean (SEM). Each time course was generated from continuous measurements of each assay and assumes no significant TCI depletion. App. <i>K</i><sub><i>i</i></sub> values were calculated from the fitted IC<sub>50</sub> curve using nonlinear regression (four parameters) using the method in ref <a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a>).</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Calculated using the method described in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. Each value represents the arithmetic mean Â± SEM of <i>n</i> = 3 biological repeats.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Calculated from the respective <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> and initial <i>K</i><sub><i>i</i></sub> values using the method described in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last"><i>t</i><sub>1/2</sub><sup>inf</sup> = lnâ¯2/<i>k</i><sub>inact</sub>.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Calculated from the IC<sub>50</sub> curves using nonlinear regression (four parameters) using the method described in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. HSP72 TCI Analysis<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><sup>â</sup><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0001.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All data were processed and analyzed using GraphPad Prism 7.04.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">App. <i>K</i><sub><i>i</i></sub> = Apparent <i>K</i><sub><i>i</i></sub>. Each concentration represents <i>n</i> = 3 statistical repeats, arithmetic mean Â± SEM. Each time course was generated from continuous measurements of each assay and assumes no significant TCI depletion. App. <i>K</i><sub><i>i</i></sub> values were calculated from the fitted IC<sub>50</sub> curve using nonlinear regression (four parameters) using the method in ref <a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a>).</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Intact protein mass spectrometry. Entry 1: HSP72-NBD [2.3 Î¼M] and TCI [200 Î¼M] incubated for the time indicated. Entries 2â4: HSP72-NBD [2.0 Î¼M] and TCI [20 Î¼M] incubated for the time indicated. The MS of the resulting protein/TCI adducts were analyzed using Agilent MassHunter Qualitative B.06.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last"><i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> values calculated from the covalent FP-assay. The gradient of each slope was calculated from the linear regression, representative example of <i>N</i> = 3 independent biological repeats (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a> for details).</p></div></div><div></div></div><div class="NLM_p last">From these data, we concluded that although our TCI design was successful in predicting that the binding site could accommodate the activated ester and maintain reversible affinity, we had failed to account for the stereoelectronic requirements of the electrophile. Efficient nucleophilic addition to the carbonyl must satisfy the correct BÃ¼rgiâDunitz<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and FlippinâLodge angles<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> at appropriate reaction distances. This could not be achieved with <i>p</i>-fluorophenolate leaving group adopting the necessary vector-to-solvent in the conformationally restrictive TCI reversible binding mode, thus blocking covalent-bond formation and the E-I complex.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Design and Synthesis of the Third-Generation HSP72 TCI</h3><div class="NLM_p">The aryl sulfonyl fluoride electrophile has recently become popular in both synthetic chemistry and chemical biology.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> Sulfonyl fluorides are stable in water under physiologically relevant conditions and have previously been shown to react readily with lysine residues in proteins.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> A recent study by Grimster et al. demonstrated that the electrophilicity of the moiety displays a strong dependence on the electronics of the attached aromatic ring and can be modulated to give an intrinsic reactivity against glutathione, comparable to chemical probe-relevant <i>N</i>-arylacrylamide electrophiles.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The solvation-dependent fluoride leaving group is less likely to form a steric clash, and the proposed S<sub>A</sub>N associative mechanism<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> should allow for a less restrictive stereoelectronic requirement for the reaction in the conformationally rigid protein-binding site. We therefore hypothesized that incorporation of an aryl sulfonyl fluoride electrophile would be effective in our Lys56-targeting HSP72 third-generation TCI. Unfortunately, our current methodology for the synthesis of 5â²-benzylriboside ethers proved incompatible with the incorporation of the sulfonyl fluoride electrophile, so we adapted our TCI design to include an ester linker.</div><div class="NLM_p">2â²,3â²-Acetonide-protected adenosine <b>10</b> was selectively oxidized with bromine to give <b>11</b> in moderate yield. S<sub>N</sub>Ar displacement with 4-chlorobenzylamine gave <b>12</b>, which then underwent selective esterification with 4-(fluorosulfonyl)benzoyl chloride to give <b>13</b> in 56% yield. The sulfonyl fluoride electrophile proved stable to the acetonide-deprotection conditions and gave the third-generation TCI <b>14</b>, following treatment with TFA/H<sub>2</sub>O, in four steps and 13% overall yield. To determine the effect of the 5â²-ester linker on the reversible affinity, we synthesized the noncovalent ester MMP <b>15</b> of ether <b>9</b> in two steps from the primary alcohol intermediate <b>12</b> in 16% yield using 4-cyanobenzoyl chloride (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Third-Generation Aryl Sulfonyl Fluoride HSP72 TCIs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Br<sub>2</sub>, K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O, 1,4-dioxane, H<sub>2</sub>O, RT, 0.5 h, 74%; (ii) 4-chlorobenzylamine, EtOH, 160 Â°C MW, 1 h, 97%; (iii) 4-(fluorosulfonyl)benzoyl chloride, Et<sub>3</sub>N, DMF, 0 Â°C, 4 h; 56% (iv) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 32%; (v) 4-cyanobenzoyl chloride, Et<sub>3</sub>N, DMF, 0 Â°C, 3 h; (vi) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 16% over two steps; (vii) 4-(fluorosulfonyl)benzoic acid, HBTU, Et<sub>3</sub>N, DMF, RT, 2 h; 66%; (viii) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 74%; (ix) 33% MeNH<sub>2</sub> in EtOH, 160 Â°C, MW, 1 h, 77%; (x) 4-(fluorosulfonyl)benzoic acid, HBTU, Et<sub>3</sub>N, DMF, RT, 2 h; 35%; (xi) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 40%; (xii) quinoline-6-yl methanamine, EtOH, 160 Â°C, MW, 2 h, 8%; (xiii) 4-(fluorosulfonyl)benzoic acid, HBTU, Et<sub>3</sub>N, DMF, RT, 2 h; 40%; (xiv) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 11%.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Characterization of the Third-Generation TCI</h3><div class="NLM_p">The reversible 5â²-ester analogue <b>15</b> displayed a binding affinity of <i>K</i><sub><i>i</i></sub> = 3.5 Î¼M (<i>pK</i><sub><i>i</i></sub> = 5.45 Â± 0.01, <i>N</i> = 3), >18-fold less potent than the tight-binding ether MMP <b>9</b> but sufficiently potent to investigate the role of the electrophile in HSP72 TCI design. Therefore, the third-generation ester sulfonyl fluoride TCI <b>14</b> was analyzed in the HSP72-NBD FP-assay. Pleasingly, <b>14</b> displayed a clear time-dependent shift in the probe displacement curve, consistent with covalent bond formation. The App. <i>K</i><sub><i>i</i></sub> = 17 Î¼M observed after 5 min exposure of TCI <b>14</b> was comparable to the reversible ester analogue <b>15</b>. The App. <i>K</i><sub><i>i</i></sub> appeared to increase in activity 24-fold over 2 h. The MMP irreversible control <b>13</b> showed no reversible binding affinity and no time-dependent displacement of the FP-probe. The analysis was repeated using the HSP72-NBD K56A mutant (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a>):<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> no time-dependent shift in the probe displacement curve was observed, suggesting no significant formation of the covalent adduct with TCI <b>14</b> under the same conditions as the WT-HSP72-NBD, confirming the reaction specificity and requirement for an initial reversible binding event.</div><div class="NLM_p last">To confirm these results were due to covalent-bond formation, we then analyzed the reaction by intact-protein MS. A solution of HSP72-NBD and <i>p</i>-sulfonyl fluoride (SF) TCI <b>14</b> (20 Î¼M <b>14</b> and 2.0 Î¼M HSP72-NBD) was incubated at 21 Â°C (room temperature) for 2 h. The experiment was repeated with irreversible control <b>13</b> under the same conditions. These data revealed that SF TCI <b>14</b> formed a covalent bond with HSP72-NBD, with the reaction going apparently to completion within 3 h of exposure using this semiquantitative assessment. The irreversible control MMP <b>13</b> gave no reaction under these conditions.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Covalent FP-Assay</h3><div class="NLM_p">The timeframe of the App. <i>K</i><sub><i>i</i></sub> shift and intact-protein MS with the third-generation SF TCI <b>14</b> strongly indicated that it was far more efficient than the first-generation acrylate TCI <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 1 vs <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 4), as the formation of the covalent adduct was reduced from days to hours. However, using these data alone, it was not possible to quantify this optimization or to determine whether the increased activity was due to an increase in the reversible binding affinity <i>K</i><sub><i>i</i></sub>, an increase in efficiency of the covalent reaction <i>k</i><sub>inact</sub>, or a mixture of the two, although analysis of the early time point App. <i>K<sub>i</sub></i> values did suggest the two analogues might possess comparable reversible affinity. To deconvolute the TCI optimization, it would be necessary to develop a new method to determine the kinetic parameters involved in the irreversible inhibition of HSP72-NBD.</div><div class="NLM_p">Determining the kinetic parameters involved in covalent bond formation with proteins can be challenging.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Kinetic data often relies upon reaction rate changes evaluated from secondary readouts, such as substrate to product formation. While this analysis can be accurate in determining the key second-order rate constant for the process, <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>, it can be difficult to accurately distinguish whether the retardation of the substrate to product reaction rate is due to reversible target occupancy or irreversible covalent bond formation. When attempting to determine <i>k</i><sub>inact</sub>, the reversible target occupancy is very high and approaches saturation, which significantly slows the substrate to product reaction separately from the covalent occupancy. Under these conditions, the time-dependent change in the rate of substrate to product reaction, necessary to determine <i>k</i><sub>inact</sub>, is unavoidably very small and difficult to quantify accurately.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> This can lead to a significant underestimation of <i>k</i><sub>inact</sub> and a resulting overestimation of the binding affinity, <i>K</i><sub><i>I</i></sub>. While a direct measurement of the rate of proteinâTCI covalent adduct formation, the actual product of interest, using quantified mass spectrometry would circumvent many of these challenges, though determining kinetic parameters for tight-binding and high-<i>k</i><sub>inact</sub> TCIs would still be difficult, this method is typically low-throughput and cannot observe noncovalent adducts due to the denaturing conditions of the assay. Therefore, TCI reversible affinities cannot be simply compared to their reversible noncovalent MMPs, a crucial requirement for efficient optimization.</div><div class="NLM_p">Following analysis of the nucleotide-derived HSP72-NBD FP-assay, we hypothesized that it could be adapted to determine the kinetic parameters of covalent bond formation and would allow us to directly compare TCIs with reversible analogues. The probe bound fraction (<i>F</i><sub>b</sub>) is determined by the affinity of the probe and the apparent concentration of the protein (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a> for details).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Changes in the bound fraction of the probe are observed through changes in the polarization of light emitted. Crucially, changes in the bound fraction are proportional to the effective concentration of protein. Displacement of the FP-probe by an inhibitor essentially decreases the effective concentration of protein, resulting in a decrease in the bound fraction. For a reversible inhibitor at equilibrium, the bound fraction remains constant at a given concentration. For an irreversible covalent inhibitor, the effective protein concentration decreases with time, which must result in a decrease in bound fraction. The time-dependent change in probe bound fraction could therefore be used to quantify the covalent-bond formation without secondary product formation or MS analysis.</div><div class="NLM_p">To quantify the covalent bond formation for our TCIs with HSP72-NBD, we would need to interpret the time-dependent change in polarization of light from the probe. The polarization must first be converted into the anisotropy, as the bound fraction of the fluorescent probe is directly proportional to the anisotropy (A). However, F<sub>b</sub> displays a nonlinear relationship with the effective protein concentration (E) such that at high bound fractions (<i>F</i><sub>b</sub> > 0.8), small changes in anisotropy would equate to very large changes in effective protein concentration, resulting in low accuracy. At low bound fractions (<i>F</i><sub>b</sub> < 0.4), large changes in anisotropy would be needed to observe a small change in effective protein concentration, which would result in low sensitivity (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a>). Between these two extremes, the response of bound fraction to changes in the effective protein concentration are linear to an acceptable approximation.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p">To develop the HSP72 covalent FP-assay, we selected an initial protein concentration that would lead to a high bound fraction (<i>F</i><sub>b</sub> = 0.8), as the reversible occupancy of the protein would rapidly displace the probe and reduce <i>F</i><sub>b</sub>. If the initial effective protein concentration was too low, this reversible displacement would move our analysis outside of the linear quantification window of the assay. As the concentration of the TCI is increased, the rate of change of <i>F</i><sub>b</sub> will increase. Following extrapolation of the linear regression to <i>t</i> = 0, the initial <i>F</i><sub>b</sub> values are then plotted against the TCI concentration to generate a displacement curve and calculate the initial <i>K</i><sub><i>i</i></sub>. This quantifies the reversible affinity of the ligand, prior to the formation of the covalent bond and depletion of the effective protein concentration. The initial <i>K<sub>i</sub></i> value is then used to focus a second FP titration at TCI concentrations below initial <i>K</i><sub><i>i</i></sub> to estimate the crucial second-order rate constant <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>. Linear regression on the <i>F</i><sub>b</sub> versus time graph will give the rate of change of <i>F</i><sub>b</sub>, which is converted into the pseudo-first-order rate constant <i>k</i><sub>obs</sub>, using the <i>F</i><sub>b</sub> values extrapolated from <i>t</i> = 0. Finally, the plot of <i>k</i><sub>obs</sub> against the TCI concentration would give <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> from the gradient of the linear region of the graph, and by assuming <i>K</i><sub><i>I</i></sub><i>= K</i><sub><i>i</i></sub>, we can estimate <i>k</i><sub>inact</sub> from this relationship and the initial <i>K</i><sub><i>i</i></sub> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Covalent FP-assay to determine the efficiency of covalent-bond formation. (1) Initial titration across a wide range of TCI concentrations and timepoints. (2) Extrapolation of the time-dependent change in <i>F</i><sub>b</sub> to t = 0. (3) Estimation of initial <i>K</i><sub><i>i</i></sub> from extrapolated <i>t</i> = 0 <i>F</i><sub>b</sub> values; for an example of how to determine <i>K<sub>i</sub></i> from an IC<sub>50</sub> in the FP-assay, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (4) Second focused titration on concentrations of TCI < <i>K</i><sub><i>i</i></sub>. (5) Gradient of time-dependent change in <i>F</i><sub>b</sub> used to calculate <i>k</i><sub>obs</sub> at a given concentration of TCI. (6) The gradient of the rate of change of <i>k</i><sub>obs</sub> with [TCI] determines the second-order rate constant <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>. (7) By assuming <i>K</i><sub><i>i</i></sub> = <i>K</i><sub><i>I</i></sub>, <i>k</i><sub>inact</sub> can be calculated from <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> and converted to <i>t</i><sub>1/2</sub><sup>inf</sup>. See the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a> for details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Kinetic Characterization of the Lysine-Targeting TCIs</h3><div class="NLM_p">Using our analysis from the covalent FP-assay, third-generation aryl-SF TCI <b>14</b> displayed a second-order rate constant for the efficiency of the covalent-bond formation with HSP72-NBD<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> of <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> = 35 Â± 1.7 M<sup>â1</sup> s<sup>â1</sup> and <i>k</i><sub>inact</sub> calculated as 3.6 Ã 10<sup>â4</sup> s<sup>â1</sup>, equivalent to <i>t</i><sub>1/2</sub><sup>inf</sup> = 32 min (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 6). The half-life from the covalent FP-assay was consistent with the data from our intact-protein MS assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 4) that showed complete modification of HSP72-NBD by SF TCI <b>14</b> (20 Î¼M, 2 Ã initial <i>K<sub>i</sub></i>) within 3 h (5.6 half-lives).</div><div class="NLM_p last">Comparing to the first-generation acrylate TCI <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 1; <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 1): the aryl-SF-TCI <b>14</b> displayed a 41-fold increase in <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>, consistent with the time-dependent shift observed in the FP-assay. However, this improvement in covalent efficiency was not due to an increase in reversible affinity, as initial <i>K</i><sub><i>i</i></sub> only increased by 1.8-fold. The optimization of <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> was derived largely by an improved <i>k</i><sub>inact</sub> (23-fold increase). The negative-control MMP of aryl-SF <b>14</b>, acetonide <b>13</b>, displayed no reactivity with accessible nucleophilic residues on HSP72-NBD to form a covalent adduct when assessed by intact protein MS, which suggested that the compatibility of the harder lysine electrophile was the driver of <i>k</i><sub>inact</sub> rather than intrinsic reactivity.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> <i>K</i><sub><i>I</i></sub> Optimization of the Third-Generation Aryl-SF TCI</h3><div class="NLM_p">Exploiting the versatility of the covalent FP-assay to directly compare covalent and noncovalent ligands, we designed a series of 5â²-aryl-SF TCI analogues based on the known affinities of their reversible MMPs. Substitution at the 8-position of the adenine ring is essential for the affinity of this class of inhibitors, and these diverse structures are synthetically tractable (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).<a onclick="showRef(event, 'ref18 ref23 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref18 ref23 ref41 ref42">(18,23,41,42)</a></div><div class="NLM_p last">Analysis of the kinetics of the Lys56-targeting HSP72-NBD TCI series revealed that <i>k</i><sub>inact</sub> was comparable across the three aryl-SF analogues where a value could be determined (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries 6 and 8â9). The 8-nonsubstituted analogue <b>16</b> displayed an activity 70-fold weaker than that of <i>N</i>-4-chlorobenzyl TCI <b>14</b>, demonstrating the importance of reversible affinity for TCI efficiency against this target. The 8-<i>N</i>-methyl-substituted analogue <b>17</b> decreased the covalent efficiency of the TCI 11-fold, which was predominately due to a drop in initial <i>K</i><sub><i>i</i></sub> consistent with the reported activity of its reversible MMP.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Finally, substitution with the 8-<i>N</i>-quinoline moiety, an analogue previously demonstrated to display the highest affinity as a reversible MMP,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> to give <b>18</b>, enhanced <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> 2.7-fold compared to the <i>p</i>-chloro aryl-SF TCI <b>14</b>, consistent with the 2.1-fold improvement in reversible affinity, and representing a 108-fold enhancement in covalent efficiency over our first-generation acrylate TCI <b>2</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The design, application, and analysis of TCIs in a rational and quantitative manner remain a critical challenge in covalent inhibitor drug discovery. The covalent FP-assay we developed utilized the time-dependent change in FP-probe bound fraction to determine the fundamental parameters of covalent-bond formation. For proteins like HSP72 with poor catalytic turnover in biochemical assays or nonenzymatic receptors and scaffolding proteins, the covalent FP-assay will be an important addition to the available methods to quantify and deconvolute the activity of TCIs, particularly as noncovalent reversible MMPs can be directly compared without the need to change assay formats.</div><div class="NLM_p last">We exploited our novel covalent FP-assay to continue our development of methods and strategies to discover lysine-targeting covalent inhibitors. Through our exhaustive understanding of the SAR and binding mode of nucleoside-derived reversible ligands of HSP72, we designed a next-generation sulfonyl fluoride TCI <b>18</b>, which displayed a 108-fold enhancement in the critical second-order rate constant, <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>. Further analysis revealed that the rate enhancement was due to both optimization of <i>K</i><sub><i>I</i></sub>, in a manner consistent with their MMP reversible analogues, and through a significant increase in <i>k</i><sub>inact</sub>. The failure of our activated ester second-generation TCI <b>8</b> demonstrates the importance of electrophile design when targeting the harder nucleophile in lysine residues. The angles of attack in a conformationally restrictive environment, while maintaining the vectors and steric requirements to accommodate a leaving group, makes the design of lysine-selective electrophiles challenging. The sulfonyl fluoride electrophile was able to circumvent many of these difficulties, with its small fluoride leaving group and accommodating sulfur electrophilic center, and this represents a key learning in lysine electrophile design. As we continue to progress toward a cell active chemical probe for HSP72 and as we improve our understanding and design strategy toward lysine-targeting covalent inhibitors of other challenging targets, the nature of the electrophile will prove crucial if we are to be successful.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Experimental</h3><div class="NLM_p">Unless otherwise stated, reactions were conducted in oven-dried glassware under an atmosphere of nitrogen or argon using anhydrous solvents. All commercially obtained reagents and solvents were used as received. Thin-layer chromatography (TLC) was performed on precoated aluminum sheets of silica (60 F254 nm, Merck) and visualized using short-wave UV light. Flash column chromatography was carried out on Merck silica gel 60 (partial size, 40â65 Î¼m). Column chromatography was also performed on a Biotage SP1 or Biotage Isolera Four purification system using Biotage Flash silica cartridges (SNAP KP-Sil) or for reverse-phase purifications SNAP Ultra C18 cartridges. Ion-exchange chromatography was performed using acidic Isolute Flash SCX-II columns. <sup>1</sup>H NMR spectra were recorded on Bruker AMX500 (500 MHz) spectrometers using an internal deuterium lock. Chemical shifts are quoted in parts per million (ppm) using the following internal references: CDCl<sub>3</sub> (Î´H 7.26), MeOD (Î´H 3.31), and dimethyl sulfoxide (DMSO)-<i>d</i><sub>6</sub> (Î´H 2.50). Signal multiplicities are recorded as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), doublet of doublets (dd), doublet of doublet of doublets (ddd), broad (br), apparent (app), or obscured (obs). Coupling constants, J, are measured to the nearest 0.1 Hz. <sup>13</sup>C NMR spectra were recorded on Bruker AMX500 spectrometers at 126 MHz using an internal deuterium lock. Chemical shifts are quoted to 0.01 ppm, unless greater accuracy was required, using the following internal references: CDCl<sub>3</sub> (Î´C 77.0), MeOD (Î´C 49.0), and DMSO-<i>d</i><sub>6</sub> (Î´C 39.5). High-resolution mass spectra were recorded on an Agilent 1200 series HPLC and diode array detector coupled to a 6210 time-of-flight mass spectrometer with dual multimode APCI/ESI source or on a Waters Acquity UPLC and diode array detector coupled to a Waters G2 QToF mass spectrometer fitted with a multimode ESI/APCI source. For HRMS and liquid chromatography-mass spectrometry (LCMS) extended mass (100â1000 AMU), analytical separation was carried out at 30 Â°C on a Merck Chromolith Flash column (RP-18e, 25 mm Ã 2 mm) using a flow rate of 0.75 mL/min in a 4 min gradient elution with detection at 254 nm. The mobile phase was a mixture of methanol (solvent A) and water (solvent B), both containing formic acid at 0.1%. Gradient elution was as follows: 5:95 (A/B) to 100:0 (A/B) over 2.5 min, 100:0 (A/B) for 1 min, and then reversion back to 5:95 (A/B) over 0.1 min, finally 5:95 (A/B) for 0.4 min. HRMS references: caffeine [M + H]<sup>+</sup> 195.087652; hexakis (2,2-difluroethoxy)phosphazene [M + H]<sup>+</sup> 622.02896; and hexakis(1<i>H</i>,1<i>H</i>,3<i>H</i>-tetrafluoropentoxy)phosphazene [M + H]<sup>+</sup> 922.009798. For standard LCMS, analytical separation was carried out at 40 Â°C on a Merck Chromolith Flash column (RP-18e, 25 mm Ã 2 mm) using a flow rate of 1.5 mL/min in a 2 min gradient elution with detection at 254 nm. The mobile phase was a mixture of methanol (solvent A) and water (solvent B), both containing formic acid at 0.1%. Gradient elution was as follows: 5:95 (A/B) to 100:0 (A/B) over 1.25 min, 100:0 (A/B) for 0.5 min, and then reversion back to 5:95 (A/B) over 0.05 min, finally 5:95 (A/B) for 0.2 min. Infrared spectra were recorded on a Bruker Î±-p Fourier transform infrared (FT-IR) spectrometer. Absorption maxima (Î½<sub>max</sub>) are quoted in wavenumbers (cm<sup>â1</sup>). All compounds were found to be >95% pure by HPLC analysis unless otherwise stated. The standard adenine and adenosine numbering has been used throughout. All compounds were found to be >95% pure by LCMS analysis unless otherwise stated.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((3,4-dichlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methoxy)methyl)benzonitrile <b>1</b></h4><div class="NLM_p last">Synthesized via the method described by Williamson et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 3-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)propyl acrylate <b>2</b></h4><div class="NLM_p last">Synthesized via the method described by Pettinger et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-chloro-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>4</b></h4><div class="NLM_p last">6-Chloro-9-[2,3-<i>O</i>-(1-methylethylidene)-<i>Î²</i>-<span class="smallcaps smallerCapital">d</span>-ribofuranosyl]-9<i>H</i>-Purine <b>3</b> (3.05 g, 9.34 mmol) and 4-(bromomethyl)benzonitrile (7.33 g, 37.4 mmol) were dissolved in DMF (80 mL) and stirred at room temperature for 5 min. Sodium hydride (60% in mineral oil, 0.41 g, 10.3 mmol) was then added, and the reaction was stirred at room temperature for a further 45 min. The reaction was quenched with 1% AcOH (20 mL), then taken up in EtOAc (60 mL) and water (60 mL). The organic extracts were washed with sat. NaCl (3 Ã 50 mL) and dried over MgSO<sub>4</sub>. The solvent was then removed under reduced pressure to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (Cyc/EtOAc 100:0 to 70:30) to give the title compound <b>4</b> as a colorless foam (2.3 g, 56%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.73 (s, 1H), 8.30 (s, 1H), 7.55 (d, <i>J</i> = 8.3 Hz, 2H), 7.23 (d, <i>J</i> = 8.2 Hz, 2H), 6.23 (d, <i>J</i> = 2.3 Hz, 1H), 5.38 (dd, <i>J</i> = 6.1, 2.3 Hz, 1H), 5.00 (dd, <i>J</i> = 6.1, 2.5 Hz, 1H), 4.59 (app. dt, <i>J</i> = 4.1, 2.9 Hz, 1H), 4.50 (d, <i>J</i> = 12.7 Hz, 1H), 4.47 (d, <i>J</i> = 12.7 Hz, 1H), 3.75 (dd, <i>J</i> = 10.5, 3.1 Hz, 1H), 3.68 (dd, <i>J</i> = 10.6, 4.2 Hz, 1H), 1.64 (s, 3H), 1.42 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl3) Î´C 152.21, 151.36, 151.05, 143.87, 142.34, 132.39, 127.82, 118.83, 118.61, 114.68, 112.02, 92.38, 86.40, 84.92, 81.79, 72.78, 71.01, 27.29, 25.50; HRMS (ESI) C<sub>21</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub><sup>35</sup>Cl (M + H<sup>+</sup>) requires 442.1277, found 442.1254; <i>t</i><sub>R</sub> (LCMS) = 1.39 min.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-amino-8-bromo-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>5</b></h4><div class="NLM_p last">4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-chloro-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>4</b> (2.3 g, 5.21 mmol) was dissolved in ammonia (7 N in MeOH, 20 mL, 140 mmol) and heated in microwave at 110 Â°C for 2 h. The solvent was removed under reduced pressure, and the subsequent residue was partitioned between EtOAc (50 mL) and water (50 mL). The combined organic extracts were washed with sat. NaCl (3 Ã 30 mL) and dried over MgSO<sub>4</sub>. The solvent was then removed under reduced pressure to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 60:40) to give 4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile as a white amorphous solid (1.81 g, 82%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.30 (s, 1H), 7.92 (s, 1H), 7.54 (d, <i>J</i> = 8.3 Hz, 2H), 7.26 (obs. d, <i>J</i> = 8.0 Hz, 2H), 6.13 (d, <i>J</i> = 2.1 Hz, 1H), 5.58 (br. s, 2H), 5.44 (dd, <i>J</i> = 6.2, 2.2 Hz, 1H), 5.04 (dd, <i>J</i> = 6.2, 3.0 Hz, 1H), 4.55â4.48 (m, 3H), 3.73 (dd, <i>J</i> = 10.4, 3.8 Hz, 1H), 3.67 (dd, <i>J</i> = 10.4, 5.5 Hz, 1H), 1.63 (s, 3H), 1.41 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 155.45, 153.25, 149.46, 142.99, 139.74, 132.30, 127.74, 120.33, 118.85, 114.50, 111.69, 91.56, 86.36, 84.60, 81.91, 72.60, 70.96, 27.32, 25.56; HRMS (ESI) C<sub>21</sub>H<sub>23</sub>N<sub>6</sub>O<sub>4</sub> (M + H<sup>+</sup>) requires 423.1775, found 423.1745; <i>t</i><sub>R</sub> (LCMS) = 1.24 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3313, 3141, 2987, 2938, 2230, 1641, 1600, 1470, 1417, 1373. 4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile (1.62 g, 3.84 mmol) was dissolved in 1,4-dioxane (35.0 mL). To a solution of K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O (2.63 g, 11.53 mmol) and bromine (1.54 g, 9.60 mmol) in water (35.0 mL) was then added dropwise to the stirred solution of 100 at room temperature. After 1 h, the reaction was quenched with sat. aq. Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> solution (40 mL) and stirred for a further 2 min. The resulting mixture was extracted with EtOAc (3 Ã 50 mL), then the combined organic layers were washed with sat. NaCl (3 Ã 50 mL) and dried over MgSO<sub>4</sub>. The solvent was then removed under reduced pressure to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (Cyc/EtOAc 50:50 to 0:100) to give the title compound <b>5</b> as a yellow oil (1.39 g, 72%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.16 (s, 1H), 7.54 (d, <i>J</i> = 8.3 Hz, 2H), 7.30 (d, <i>J</i> = 8.4 Hz, 2H), 6.21 (d, <i>J</i> = 1.7 Hz, 1H), 5.67 (dd, <i>J</i> = 6.4, 1.8 Hz, 1H), 5.61 (br. s, 2H), 5.16 (dd, <i>J</i> = 6.3, 3.8 Hz, 1H), 4.54 (d, <i>J</i> = 13.3 Hz, 1H), 4.48 (d, <i>J</i> = 13.3 Hz, 1H), 4.44 (app. dt, <i>J</i> = 7.5, 4.5 Hz, 1H), 3.67 (dd, <i>J</i> = 10.3, 4.8 Hz, 1H), 3.60 (dd, <i>J</i> = 10.3, 7.4 Hz, 1H), 1.63 (s, 3H), 1.40 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 154.24, 152.97, 150.51, 143.49, 132.22, 127.88, 127.73, 120.30, 118.96, 114.57, 111.47, 91.37, 86.98, 83.47, 82.21, 72.37, 70.64, 27.34, 25.60; HRMS (ESI) C<sub>21</sub>H<sub>22</sub>N<sub>6</sub>O<sub>4</sub><sup>79</sup>Br (M + H<sup>+</sup>) requires 501.0880, found 501.0871; <i>t</i><sub>R</sub> (LCMS) = 1.46 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3322, 3172, 2987, 2228, 1639, 1597, 1577, 1454, 1373, 1290, 1205.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>6</b></h4><div class="NLM_p last">4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-bromo-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>5</b> (0.98 g, 1.95 mmol) was dissolved in EtOH (16 mL) and added to 4-chlorobenzylamine (2.77 g, 19.5 mmol), and the reaction was heated in microwave for 1 h at 160 Â°C. The solvent was removed under reduced pressure, and the resulting residue was taken up in EtOAc (50 mL) and 1% aq. AcOH (50 mL). The product was extracted with EtOAc (3 Ã 40 mL); washed with 1% aq. AcOH (3 Ã 40 mL), sat. NaHCO<sub>3</sub> (3 Ã 40 mL) and sat. NaCl (40 mL); and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 80:20) to give the title compound <b>6</b> as an orange foam (0.72 g, 66%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.17 (s, 1H), 7.55 (d, <i>J</i> = 8.3 Hz, 2H), 7.25 (d, <i>J</i> = 8.4 Hz, 2H), 7.19â7.15 (m, 4H), 6.23 (d, <i>J</i> = 3.3 Hz, 1H), 5.90 (app. t, <i>J</i> = 5.9 Hz, 1H), 5.33 (dd, <i>J</i> = 6.7, 3.3 Hz, 1H), 5.15 (br. s, 2H), 4.97 (dd, <i>J</i> = 6.7, 3.9 Hz, 1H), 4.45 (dd, <i>J</i> = 15.6, 6.0 Hz, 1H), 4.42 (dd, <i>J</i> = 15.5, 5.6 Hz, 1H), 4.30â4.26 (m, 3H), 3.73 (dd, <i>J</i> = 10.5, 3.0 Hz, 1H), 3.57 (dd, <i>J</i> = 10.5, 2.9 Hz, 1H), 1.62 (s, 3H), 1.38 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 152.18, 151.61, 150.34, 150.28, 142.30, 137.35, 133.37, 132.52, 128.87, 128.69, 127.74, 118.60, 117.58, 115.34, 112.08, 88.63, 83.86, 82.82, 80.14, 72.60, 69.50, 46.06, 27.41, 25.58; HRMS (ESI) C<sub>28</sub>H<sub>29</sub>N<sub>7</sub>O<sub>4</sub><sup>35</sup>Cl (M + H<sup>+</sup>) requires 562.1964, found 562.1922; <i>t</i><sub>R</sub> (LCMS) = 1.48 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3325, 2934, 2229, 1633, 1606, 1571, 1491, 1469, 1434, 1361, 1333, 1285, 1210.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzoic acid <b>7</b></h4><div class="NLM_p last">4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>6</b> (91 mg, 0.16 mmol) was dissolved in aq. NaOH (2 M, 0.8 mL, 1.62 mmol) and ethanol (0.9 mL) and heated in a sealed microwave vial at 110 Â°C for 3 h. The reaction was cooled to room temperature and neutralized to pH 7 with 1 M HCl. The mixture was extracted with EtOAc (3 Ã 20 mL), and the combined organic extracts were washed with sat. aq. NaHCO<sub>3</sub> (3 Ã 20 mL) and sat. NaCl (3 Ã 20 mL) and dried over MgSO<sub>4</sub>. The solvent was then removed under reduced pressure to give the crude product, which was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN 70:30 to 0:100) to give the title compound <b>7</b> as a white amorphous solid (55 mg, 58%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.03 (s, 1H), 7.94 (d, <i>J</i> = 8.2 Hz, 2H), 7.24 (d, <i>J</i> = 8.4 Hz, 2H), 7.13 (d, <i>J</i> = 8.2 Hz, 2H), 7.10 (d, <i>J</i> = 7.9 Hz, 2H), 6.49 (br. s, 2H), 6.16 (d, <i>J</i> = 2.8 Hz, 1H), 6.05 (app. t, <i>J</i> = 6.0 Hz, 1H), 5.59 (dd, <i>J</i> = 6.4, 2.9 Hz, 1H), 5.01 (dd, <i>J</i> = 6.5, 3.2 Hz, 1H), 4.42â4.24 (m, 5H), 3.73 (dd, <i>J</i> = 10.6, 2.8 Hz, 1H), 3.64 (dd, <i>J</i> = 10.5, 3.0 Hz, 1H), 1.62 (s, 3H), 1.40 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 170.16, 152.14, 151.89, 149.52, 148.83, 141.29, 137.12, 133.33, 131.51, 130.14, 128.83, 128.71, 127.41, 116.69, 114.82, 89.87, 84.99, 82.80, 80.67, 73.51, 69.79, 45.80, 27.33, 25.55; <i>t</i><sub>R</sub> (LCMS) = 1.47 min.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-fluorophenyl 4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methoxy)methyl)benzoate <b>8</b></h4><div class="NLM_p last">4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzoic acid <b>7</b> (20 mg, 34 Î¼mol) was dissolved in DMF (0.4 mL). HATU (16 mg, 41 Î¼mol) and DIPEA (9 mg, 69 Î¼mol) were added, and the reaction was stirred at room temperature for 1 min. 4-Fluorophenol (12 mg, 0.10 mmol) was added, and the reaction was stirred at room temperature for 18 h. The solvent was then removed under reduced pressure to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 80:20) to give 4-fluorophenyl 4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzoate as a yellow oil (11 mg, 47%); <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 8.03 (d, <i>J</i> = 8.3 Hz, 2H), 7.89 (s, 1H), 7.49 (app. t, <i>J</i> = 6.0 Hz, 1H), 7.41 (d, <i>J</i> = 8.1 Hz, 2H), 7.40â7.28 (m, 8H), 6.52 (br. s, 2H), 6.13 (d, <i>J</i> = 2.4 Hz, 1H), 5.70 (dd, <i>J</i> = 6.3, 2.4 Hz, 1H), 5.07 (dd, <i>J</i> = 6.3, 3.3 Hz, 1H), 4.58â4.47 (m, 4H), 4.29 (app. td, <i>J</i> = 5.5, 3.3 Hz, 1H), 3.65 (dd, <i>J</i> = 10.5, 5.1 Hz, 1H), 3.53 (dd, <i>J</i> = 10.5, 6.2 Hz, 1H), 1.54 (s, 3H), 1.34 (s, 3H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 164.41, 152.56, 151.11, 149.20, 148.92, 146.69, 144.46, 138.71, 131.28, 129.81, 129.10, 128.10, 127.77, 127.38, 123.77 (d, <i>J</i> = 8.8 Hz), 117.06, 116.23, 116.08, 113.25, 87.51, 84.55, 81.89, 81.42, 71.54, 69.94, 44.72, 27.06, 25.29; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) Î´F â117.07; HRMS (ESI) C<sub>34</sub>H<sub>33</sub>N<sub>6</sub>O<sub>6</sub><sup>35</sup>ClF (M + H<sup>+</sup>) requires 675.2129, found 675.2037; <i>t</i><sub>R</sub> (LCMS_extended) = 3.23 min. 4-fluorophenyl 4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzoate (5 mg, 7 Î¼mol) was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (0.7 mL) and stirred at room temperature for 30 min. The solvent was then removed under reduced pressure to give the crude product, which was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN 80:20 to 0:100) to give the title compound <b>8</b> as a white amorphous solid (4 mg, 85%); <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 8.02 (d, <i>J</i> = 8.3 Hz, 2H), 7.90 (s, 1H), 7.43 (d, <i>J</i> = 8.1 Hz, 2H), 7.36â7.29 (m, 8H), 7.13 (app. t, <i>J</i> = 6.0 Hz, 1H), 6.46 (br. s, 2H), 5.88 (d, <i>J</i> = 5.8 Hz, 1H), 5.47 (br. app. s, 1H), 5.29 (br. app. s, 1H), 4.94â4.88 (m, 1H), 4.59â4.53 (m, 2H), 4.52â4.44 (m, 2H), 4.33â4.28 (m, 1H), 4.07â4.02 (m, 1H), 3.78 (dd, <i>J</i> = 10.7, 3.1 Hz, 1H), 3.65 (dd, <i>J</i> = 10.8, 4.5 Hz, 1H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 164.35, 152.47, 151.27, 149.92, 148.92, 146.67, 144.36, 138.79, 131.20, 129.82, 128.83, 128.07, 127.81, 127.41, 123.75 (d, <i>J</i> = 8.7 Hz), 116.91, 116.23, 116.07, 86.92, 82.86, 71.69, 70.49, 70.39, 44.61, 40.06; HRMS (ESI) C<sub>31</sub>H<sub>29</sub>N<sub>6</sub>O<sub>6</sub><sup>35</sup>ClF (M + H<sup>+</sup>) requires 635.1816, found 635.1823; <i>t</i><sub>R</sub> (LCMS) = 1.64 min.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methoxy)methyl)benzonitrile <b>9</b></h4><div class="NLM_p last">4-((((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methoxy)methyl)benzonitrile <b>6</b> (25 mg, 45 Î¼mol) was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (1.4 mL) and stirred at room temperature for 30 min. The solvent was then removed under reduced pressure to give the crude product, which was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN + 0.1% formic acid, 70:30 to 0:100) to give the title compound <b>9</b> as a colorless oil (11 mg, 47%); <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 7.89 (s, 1H), 7.72 (d, <i>J</i> = 7.8 Hz, 2H), 7.38 (d, <i>J</i> = 7.9 Hz, 2H), 7.32 (d, <i>J</i> = 8.3 Hz, 2H), 7.29 (d, <i>J</i> = 8.3 Hz, 2H), 7.11 (app. t, <i>J</i> = 6.1 Hz, 1H), 6.47 (br. s, 2H), 5.86 (d, <i>J</i> = 5.6 Hz, 1H), 5.38 (d, <i>J</i> = 5.8, 1H), 5.21 (d, <i>J</i> = 5.2 Hz, 1H), 4.93 (app. q, <i>J</i> = 5.6 Hz, 1H), 4.55â4.49 (m, 2H), 4.49â4.41 (m, 2H), 4.31 (app. q, <i>J</i> = 5.0 Hz, 1H), 4.03 (app. q, <i>J</i> = 4.0 Hz, 1H), 3.76 (dd, J = 10.9, 3.1 Hz, 1H), 3.64 (dd, <i>J</i> = 10.8, 4.5 Hz, 1H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 152.48, 151.30, 149.88, 148.91, 143.79, 138.75, 132.13, 131.22, 128.84, 128.07, 127.70, 118.75, 116.92, 110.11, 86.95, 82.78, 71.43, 70.46, 70.39, 70.28, 44.60; HRMS (ESI) C<sub>25</sub>H<sub>25</sub>N<sub>7</sub>O<sub>4</sub><sup>35</sup>Cl (M + H<sup>+</sup>) requires 522.1651, found 522.1570; <i>t</i><sub>R</sub> (LCMS) = 1.32 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3432, 3325, 3232, 2925, 2230, 1651, 1612, 1595, 1573, 1531, 1490, 1443, 1333, 1299.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> ((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-amino-8-bromo-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>11</b></h4><div class="NLM_p last">2â²,3â²-<i>O</i>-isopropylideneadenosine <b>10</b> (1.17 g, 3.81 mmol) was dissolved in 1,4-dioxane (16 mL) and stirred to dissolution. K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O (2.61 g, 11.4 mmol) was dissolved in water (16 mL) and then added to bromine (1.52 g, 9.52 mmol). The bromine solution was added dropwise to the stirred adenosine solution at room temperature. After 30 min, the reaction was quenched with sat. aq. Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> solution (30 mL) and stirred for a further 2 min. The resulting mixture was extracted with EtOAc (3 Ã 100 mL), then the combined organic layers washed with sat. NaCl and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 80:20) to give the title compound <b>11</b> as an orange solid (1.1 g, 74%); <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 8.15 (s, 1H), 7.56 (s, 2H), 6.02 (d, <i>J</i> = 2.7 Hz, 1H), 5.66 (dd, <i>J</i> = 6.2, 2.7 Hz, 1H), 5.12 (dd, <i>J</i> = 6.3, 5.5 Hz, 1H), 5.03 (dd, <i>J</i> = 6.2, 3.0 Hz, 1H), 4.16 (td, <i>J</i> = 5.8, 3.0 Hz, 1H), 3.52 (dt, <i>J</i> = 11.5, 5.5 Hz, 1H), 3.43 (dt, <i>J</i> = 11.5, 6.3 Hz, 1H), 1.55 (s, 3H), 1.33 (s, 3H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 154.1, 151.6, 149.6, 126.9, 119.3, 113.3, 91.0, 87.3, 82.0, 81.6, 61.4, 27.1, 25.3; HRMS (ESI) C<sub>13</sub>H<sub>17</sub>N<sub>5</sub>O<sub>4</sub><sup>79</sup>Br (M + H<sup>+</sup>) requires 386.0458, found 386.0456; <i>t</i><sub>R</sub> (LCMS) = 1.22 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3321, 3172, 2953, 2851, 1657, 1596, 1575, 1497, 1461.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> ((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>12</b></h4><div class="NLM_p last">((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-bromo-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>11</b> (1.71 g, 4.44 mmol) and 4-chlorobenzylamine (5.03 g, 35.5 mmol) were dissolved in EtOH (15 mL), and the reaction was heated in microwave for 1 h at 160 Â°C. The solvent was removed under reduced pressure, and the resulting residue was taken up in EtOAc (50 mL), washed with 1% AcOH solution (3 Ã 50 mL) and sat. aq. NaHCO<sub>3</sub> solution (3 Ã 50 mL), dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 70:30) to give the title compound <b>12</b> as an orange solid (1.93 g, 97%); <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 7.93 (s, 1H), 7.61 (t, <i>J</i> = 6.0 Hz, 1H), 7.40 (d, <i>J</i> = 8.8 Hz, 2H), 7.38 (d, <i>J</i> = 8.8 Hz, 2H), 6.57 (s, 2H), 6.09 (d, <i>J</i> = 3.5 Hz, 1H), 5.47 (dd, <i>J</i> = 6.2, 3.5 Hz, 1H), 5.42 (app. t, <i>J</i> = 5.2 Hz, 1H), 4.97 (dd, <i>J</i> = 6.3, 2.8 Hz, 1H), 4.58 (dd, <i>J</i> = 15.6, 5.8 Hz, 1H), 4.53 (dd, <i>J</i> = 15.6, 6.0 Hz, 1H), 4.17 (app. td, <i>J</i> = 4.4, 2.8 Hz, 1H), 3.59â3.49 (m, 2H), 1.55 (s, 3H), 1.32 (s, 3H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 153.04, 151.44, 149.69, 149.39, 139.23, 131.75, 129.63, 128.62, 117.45, 113.62, 88.37, 85.86, 81.85, 81.63, 61.82, 45.20, 27.61, 25.73; HRMS (ESI) C<sub>20</sub>H<sub>24</sub>N<sub>6</sub>O<sub>4</sub><sup>35</sup>Cl (M + H<sup>+</sup>) requires 447.1542, found 447.1525; <i>t</i><sub>R</sub> (LCMS) = 1.36 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3190, 2934, 1645, 1607, 1573, 1491, 1457, 1371, 1326, 1213.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> ((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate <b>13</b></h4><div class="NLM_p last">((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>12</b> (0.37 g, 0.82 mmol) was dissolved in DMF (8.2 mL) and cooled to 0 Â°C. Triethylamine (0.12 g, 1.24 mmol) and 4-(fluorosulfonyl)benzoyl chloride (0.22 g, 0.99 mmol) were added, and the reaction stirred at 0 Â°C for 4 h. The solvent was removed under reduced pressure, and the subsequent oil was taken up in EtOAc (30 mL), washed with sat. NaCl (3 Ã 30 mL), and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressured to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 80:20) to give the title compound <b>13</b> as an orange solid (0.29 g, 56%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.11 (s, 1H), 8.05 (d, <i>J</i> = 8.5 Hz, 2H), 7.99 (d, <i>J</i> = 8.6 Hz, 2H), 7.30 (app. s, 4H), 6.13 (dd, <i>J</i> = 6.1, 1.6 Hz, 1H), 5.90 (d, <i>J</i> = 1.6 Hz, 1H), 5.18 (dd, <i>J</i> = 6.1, 3.1 Hz, 1H), 5.14 (s, 2H), 5.02 (app. t, <i>J</i> = 5.7 Hz, 1H), 4.62 (dd, <i>J</i> = 14.5, 6.1 Hz, 1H), 4.56â4.49 (m, 3H), 4.39 (dd, <i>J</i> = 11.8, 5.8 Hz, 1H), 1.60 (s, 3H), 1.44 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 164.02, 152.36, 151.76, 150.14, 149.84, 137.04 (d, <i>J</i> = 25.6 Hz), 136.69, 135.61, 133.73, 130.70, 129.24, 129.05, 128.54, 117.85, 114.44, 89.61, 85.43, 82.52, 81.71, 64.87, 46.59, 27.25, 25.55; <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>) Î´F 65.68; HRMS (ESI) C<sub>27</sub>H<sub>27</sub>N<sub>6</sub>O<sub>7</sub>SF<sup>35</sup>Cl (M + H<sup>+</sup>) requires 633.1329, found 633.1360; <i>t</i><sub>R</sub> (LCMS) = 1.57 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3192, 1675, 1600, 1492, 1410, 1270, 1211.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> ((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl 4-(fluorosulfonyl)benzoate <b>14</b></h4><div class="NLM_p last">((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate <b>13</b> (77 mg, 0.12 mmol) was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (1.4 mL) and stirred at room temperature for 30 min. The solvent was then removed under reduced pressure to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 80:20) to give the title compound <b>14</b> as an off-white solid (24 mg, 32%); <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 8.20 (d, <i>J</i> = 8.6 Hz, 2H), 8.12 (d, <i>J</i> = 8.4 Hz, 2H), 7.81 (s, 1H), 7.49 (app. t, <i>J</i> = 6.0 Hz, 1H), 7.38 (d, <i>J</i> = 8.5 Hz, 2H), 7.33 (d, <i>J</i> = 8.5 Hz, 2H), 6.46 (s, 2H), 5.81 (d, <i>J</i> = 4.0 Hz, 1H), 5.46 (d, <i>J</i> = 5.2 Hz, 1H), 5.33 (d, <i>J</i> = 5.9 Hz, 1H), 5.19 (app. q, <i>J</i> = 5.2 Hz, 1H), 4.70â4.63 (m, 2H), 4.53 (app. d, <i>J</i> = 5.9 Hz, 2H), 4.46 (dd, <i>J</i> = 12.0, 5.0 Hz, 1H), 4.16 (app. td, <i>J</i> = 5.3, 3.2 Hz, 1H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 163.82, 152.50, 151.67, 149.52, 148.78, 138.87, 136.18, 135.21 (d, <i>J</i> = 24.0 Hz), 131.22, 130.75, 129.21, 128.77, 128.04, 117.29, 87.67, 80.52, 70.75, 69.65, 64.78, 44.74; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) Î´F 66.03; HRMS (ESI) C<sub>24</sub>H<sub>23</sub>N<sub>6</sub>O<sub>7</sub>SF<sup>35</sup>Cl (M + H<sup>+</sup>) requires 593.1016, found 593.1008; <i>t</i><sub>R</sub> (LCMS) = 1.41 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3346, 1726, 1637, 1608, 1573, 1409, 1270, 1210.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> ((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl 4-cyanobenzoate <b>15</b></h4><div class="NLM_p last">((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>12</b> (0.11 g, 0.24 mmol) was dissolved in DMF (2.4 mL) and cooled to 0 Â°C. Triethylamine (36 mg, 0.36 mmol) and 4-cyanobenzoyl chloride (47 mg, 0.28 mmol) were added, and the reaction was stirred at 0 Â°C for 3 h. The solvent was removed under reduced pressured to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 95:5 to 70:30) to give ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-cyanobenzoate as a yellow oil (45 mg, 33%); <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´H 8.11 (s, 1H), 7.93 (d, <i>J</i> = 8.7 Hz, 2H), 7.65 (d, <i>J</i> = 8.8 Hz, 2H), 7.29 (s, 4H), 6.09 (dd, <i>J</i> = 6.2, 1.7 Hz, 1H), 5.91 (d, <i>J</i> = 1.7 Hz, 1H), 5.25 (s, 2H), 5.15 (dd, <i>J</i> = 6.2, 3.2 Hz, 1H), 5.07 (app. t, <i>J</i> = 5.7, 1H), 4.61 (dd, <i>J</i> = 14.6, 6.1 Hz, 1H), 4.56â4.51 (m, 2H), 4.46 (dd, <i>J</i> = 11.9, 4.7 Hz, 1H), 4.36 (dd, <i>J</i> = 11.9, 6.0 Hz, 1H), 1.60 (s, 3H), 1.43 (s, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´C 164.51, 152.35, 151.79, 150.01, 149.86, 136.69, 133.70, 133.09, 132.28, 130.15, 129.20, 129.03, 117.97, 117.82, 116.84, 114.50, 89.62, 85.22, 82.50, 81.65, 64.61, 46.56, 27.24, 25.55; HRMS (ESI) C<sub>28</sub>H<sub>27</sub>N<sub>7</sub>O<sub>5</sub><sup>35</sup>Cl (M + H<sup>+</sup>) requires 576.1757, found 576.1746; <i>t</i><sub>R</sub> (LCMS) = 1.51 min. ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((4-chlorobenzyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-cyanobenzoate was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (1.4 mL) and stirred at room temperature for 45 min. The solvent was then removed under reduced pressure to give the crude product, which was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 95:5 to 65:35) to give the title compound <b>15</b> as a yellow oil (16 mg, 48%); <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 7.93 (d, <i>J</i> = 8.9 Hz, 2H), 7.91 (d, <i>J</i> = 8.9 Hz, 2H), 7.83 (s, 1H), 7.48 (app. t, <i>J</i> = 6.0 Hz, 1H), 7.37 (d, <i>J</i> = 8.6 Hz, 2H), 7.33 (d, <i>J</i> = 8.5 Hz, 2H), 6.49 (br. s, 2H), 5.79 (d, <i>J</i> = 3.9 Hz, 1H), 5.45 (d, <i>J</i> = 5.2 Hz, 1H), 5.31 (d, <i>J</i> = 5.9 Hz, 1H), 5.22 (app. td, <i>J</i> = 5.3, 4.0 Hz, 1H), 4.65 (app. q, <i>J</i> = 5.8 Hz, 1H), 4.61 (dd, <i>J</i> = 12.4, 3.5 Hz, 1H), 4.53 (app. d, <i>J</i> = 5.9 Hz, 2H), 4.42 (dd, <i>J</i> = 12.0, 4.7 Hz, 1H), 4.14 (ddd, <i>J</i> = 6.2, 4.6, 3.2 Hz, 1H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 164.28, 152.50, 151.71, 149.51, 148.78, 138.87, 133.30, 132.64, 131.21, 129.74, 129.19, 128.03, 118.04, 117.29, 115.45, 87.65, 80.46, 70.72, 69.56, 64.32, 44.73; HRMS (ESI) C<sub>25</sub>H<sub>23</sub>N<sub>7</sub>O<sub>5</sub><sup>35</sup>Cl (M + H<sup>+</sup>) requires 536.1444, found 536.1418; <i>t</i><sub>R</sub> (LCMS) = 1.32 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3364, 6101, 2233, 1720, 1611, 1574, 1480, 1435, 1270.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> ((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl 4-(fluorosulfonyl)benzoate <b>16</b></h4><div class="NLM_p last">2â²,3â²-<i>O</i>-isopropylideneadenosine <b>10</b> (0.15 g, 0.49 mmol) was dissolved in DMF (2.5 mL). 4-(Fluorosulfonyl)benzoic acid (0.12 g, 0.59 mmol), triethylamine (99 mg, 0.98 mmol), and HBTU (0.22 g, 0.59 mmol) were added, and the reaction was stirred at room temperature for 2 h. The solvent was then removed under reduced pressure to give an orange oil that was taken up in EtOAc (20 mL), washed with sat. aq. NaHCO<sub>3</sub> (2 Ã 20 mL) and sat. NaCl (2 Ã 20 mL), and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 70:30) to give ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate as an orange foam (0.16 g, 66%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.30 (s, 1H), 8.17 (d, <i>J</i> = 8.3 Hz, 2H), 8.04 (d, <i>J</i> = 8.6 Hz, 2H), 7.86 (s, 1H), 6.10 (d, <i>J</i> = 1.9 Hz, 1H), 5.70 (br. s, 2H), 5.60 (dd, <i>J</i> = 6.3, 1.9 Hz, 1H), 5.23 (dd, <i>J</i> = 6.3, 3.6 Hz, 1H), 4.69 (dd, <i>J</i> = 11.5, 4.1 Hz, 1H), 4.60 (app. dt, <i>J</i> = 6.3, 3.9 Hz, 1H), 4.55 (dd, <i>J</i> = 11.5, 6.3 Hz, 1H), 1.64 (s, 3H), 1.42 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 164.11, 155.63, 153.31, 149.28, 140.21, 137.03 (d, <i>J</i> = 25.4 Hz), 135.98, 130.89, 128.59, 120.57, 114.93, 91.24, 85.19, 84.17, 81.80, 65.42, 27.34, 25.57; <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>) Î´F 65.78; HRMS (ESI) C<sub>20</sub>H<sub>21</sub>N<sub>5</sub>O<sub>7</sub>FS (M + H<sup>+</sup>) requires 494.1140, found 494.1152; <i>t</i><sub>R</sub> (LCMS) = 1.40 min. ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate (60 mg, 0.12 mmol) was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (1.4 mL) and stirred at room temperature for 1 h. The solvent was then removed under reduced pressure to give the crude product, which was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN + 1% formic acid, 90:10 to 40:60) to give the title compound <b>16</b> as an orange oil (41 mg, 74%); <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 8.31 (s, 1H), 8.27 (d, <i>J</i> = 8.7 Hz, 2H), 8.25 (d, <i>J</i> = 8.7 Hz, 2H), 8.06 (s, 1H), 7.28 (br. s, 2H), 5.93 (d, <i>J</i> = 4.7 Hz, 1H), 5.59 (d, <i>J</i> = 5.7 Hz, 1H), 5.43 (d, <i>J</i> = 5.5 Hz, 1H), 4.77 (app. q, <i>J</i> = 5.2 Hz, 1H), 4.68 (dd, <i>J</i> = 12.0, 3.6 Hz, 1H), 4.52 (dd, <i>J</i> = 12.0, 5.9 Hz, 1H), 4.47 (app. q, <i>J</i> = 5.3 Hz, 1H), 4.24 (app. td, <i>J</i> = 5.5, 3.6 Hz, 1H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 163.88, 156.07, 152.58, 149.20, 140.12, 136.26, 135.28 (d, <i>J</i> = 24.0), 130.92, 128.89, 119.25, 88.18, 81.18, 72.65, 70.10, 65.27; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) Î´F 65.99; HRMS (ESI) C<sub>17</sub>H<sub>17</sub>N<sub>5</sub>O<sub>7</sub>FS (M + H<sup>+</sup>) requires 454.0827, found 454.0804; <i>t</i><sub>R</sub> (LCMS) = 1.15 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3378, 3095, 1738, 1682, 1608, 1576, 1483, 1407, 1378.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> ((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-(methylamino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl 4-(fluorosulfonyl)benzoate <b>17</b></h4><div class="NLM_p last">((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-bromo-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>11</b> (0.40 g, 1.04 mmol) was dissolved in 33% methylamine in ethanol (5.2 mL) and heated in microwave for 1 h at 160 Â°C. The solvent was removed under reduced pressure, and the crude product was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN + 1% formic acid, 90:10 to 40:60) to give ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-(methylamino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol as an orange oil (0.27 g, 77%); <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) Î´H 8.06 (s, 1H), 6.41 (s, 2H), 6.00 (d, <i>J</i> = 4.2 Hz, 1H), 5.76 (app. s, 1H), 5.14 (dd, <i>J</i> = 6.5, 4.2 Hz, 1H), 5.03 (dd, <i>J</i> = 6.4, 2.7 Hz, 1H), 4.36 (app. q, <i>J</i> = 2.2 Hz, 1H), 3.98 (dd, <i>J</i> = 12.1, 2.2 Hz, 1H), 3.85 (dd, <i>J</i> = 12.1, 2.1 Hz, 1H), 2.95 (app. s, 3H), 1.62 (s, 3H), 1.36 (s, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) Î´C 152.52, 151.40, 149.30, 147.87, 117.23, 114.85, 90.03, 85.38, 82.67, 80.74, 62.16, 29.72, 27.54, 25.38; HRMS (ESI) C<sub>14</sub>H<sub>21</sub>N<sub>6</sub>O<sub>4</sub> (M + H<sup>+</sup>) requires 337.1619, found 337.1608; <i>t</i><sub>R</sub> (LCMS) = 0.93 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3184, 1612, 1581, 1434, 1472, 1376, 1337, 1285, 1211. ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-(methylamino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol (65 mg, 0.19 mmol) was dissolved in DMF (1.9 mL). 4-(Fluorosulfonyl)benzoic acid (47 mg, 0.23 mmol), triethylamine (39 mg, 0.39 mmol), and HBTU (88 mg, 0.23 mmol) were added, and the reaction was stirred at room temperature for 5 h. The solvent was then removed under reduced pressure to give an orange oil that was taken up in EtOAc (20 mL), washed with sat. aq. NaHCO<sub>3</sub> (2 Ã 20 mL) and sat. NaCl (2 Ã 20 mL), and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 60:40) to give ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-(methylamino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate as an orange oil (70% pure by LCMS) that was used without further purification; <i>t</i><sub>R</sub> (LCMS) = 1.32 min. ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-(methylamino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (1.4 mL) and stirred at room temperature for 1 h. The solvent was then removed under reduced pressure to give the crude product, which was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN + 1% formic acid, 90:10 to 40:60) to give the title compound <b>17</b> as an orange oil (11 mg, 40%); <sup>1</sup>H NMR (600 MHz, MeOD) Î´H 8.07 (app. s, 4H), 7.85 (s, 1H), 5.70 (d, <i>J</i> = 3.4 Hz, 1H), 5.38 (dd, <i>J</i> = 5.4, 3.4 Hz, 1H), 4.86â4.83 (obs. m, 1H), 4.80 (dd, <i>J</i> = 12.2, 3.1 Hz, 1H), 4.53 (dd, <i>J</i> = 12.2, 4.0 Hz, 1H), 4.29 (app. dt, <i>J</i> = 6.7, 3.5 Hz, 1H), 2.66 (s, 3H); <sup>13</sup>C NMR (151 MHz, MeOD) Î´C 165.46, 154.87, 153.26, 150.78, 150.18, 137.49, 131.71, 129.62, 90.04, 82.91, 72.77, 71.45, 65.03, 40.43, 29.60; <sup>19</sup>F NMR (471 MHz, MeOD) Î´F 63.73; <i>t</i><sub>R</sub> (LCMS) = 1.07 min.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> ((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-8-((quinolin-6-ylmethyl)amino)-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl 4-(fluorosulfonyl)benzoate <b>18</b></h4><div class="NLM_p last">((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-bromo-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>11</b> (0.62 g, 1.61 mmol) was dissolved in EtOH (8.0 mL) and added to quinolin-6-yl methanamine (1.11 g, 7.2 mmol), and the reaction was heated in microwave for 2 h at 160 Â°C. The solvent was removed under reduced pressure, and the resulting residue was taken up in EtOAc (50 mL) and 1% aq. AcOH (50 mL). The product was extracted with EtOAc (3 Ã 40 mL); washed with 1% aq. AcOH (3 Ã 40 mL), sat. aq. NaHCO3 (3 Ã 40 mL), and sat. NaCl (40 mL); and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 95:5 to 50:50), followed by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN + 1% formic acid, 80:20 to 0:100) to give ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((quinolin-6-ylmethyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol as a yellow oil (60 mg, 8%); <sup>1</sup>H NMR (600 MHz, MeOD) Î´H 8.80 (dd, <i>J</i> = 4.5, 1.7 Hz, 1H), 8.33 (d, <i>J</i> = 8.2 Hz, 1H), 8.12 (app. s, 1H), 8.01 (s, 1H), 7.91 (app. s, 1H), 7.83 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 7.51 (dd, <i>J</i> = 8.3, 4.3 Hz, 1H), 6.23 (d, <i>J</i> = 4.3 Hz, 1H), 5.34 (dd, <i>J</i> = 6.4, 4.3 Hz, 1H), 5.03 (dd, <i>J</i> = 6.3, 2.6 Hz, 1H), 4.86â4.79 (obs. m, 2H), 4.34 (app. q, <i>J</i> = 2.8 Hz, 1H), 3.82 (dd, <i>J</i> = 11.7, 3.0 Hz, 1H), 3.76 (dd, <i>J</i> = 11.7, 2.7 Hz, 1H), 1.63 (s, 3H), 1.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, MeOD) Î´C 153.20, 150.93, 150.69, 149.99, 148.07, 139.18, 138.36, 130.76, 129.30, 126.62, 122.75, 117.92, 115.63, 90.58, 86.56, 83.26, 82.43, 62.80, 46.85, 44.55, 40.42, 27.62, 25.59; HRMS (ESI) C<sub>23</sub>H<sub>26</sub>N<sub>7</sub>O<sub>4</sub> (M + H<sup>+</sup>) requires 464.2041, found 464.2023; <i>t</i><sub>R</sub> (LCMS) = 0.97 min. ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((quinolin-6-ylmethyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol (0.14 g, 0.30 mmol) was dissolved in DMF (2.9 mL). 4-(Fluorosulfonyl)benzoic acid (72 mg, 0.35 mmol), triethylamine (60 mg, 0.59 mmol), and HBTU (0.13 g, 0.35 mmol) were added, and the reaction was stirred at room temperature for 2 h. The solvent was then removed under reduced pressure to give an orange oil that was taken up in EtOAc (20 mL), washed with sat. aq. NaHCO<sub>3</sub> (2 Ã 20 mL) and sat. NaCl (2 Ã 20 mL), and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude product was purified by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 100:0 to 60:40) to give ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((quinolin-6-ylmethyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate as an orange oil (77 mg, 40%); <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) Î´H 8.90 (dd, <i>J</i> = 4.2, 1.7 Hz, 1H), 8.11â8.08 (m, 2H), 8.06 (d, <i>J</i> = 8.6 Hz, 1H), 8.00 (d, <i>J</i> = 8.3 Hz, 2H), 7.93 (d, <i>J</i> = 8.6 Hz, 2H), 7.74 (s, 1H), 7.71 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.40 (dd, <i>J</i> = 8.3, 4.2 Hz, 1H), 6.13 (dd, <i>J</i> = 6.1, 1.6 Hz, 1H), 5.97 (d, <i>J</i> = 1.6 Hz, 1H), 5.28 (app. t, <i>J</i> = 5.7 Hz, 1H), 5.21 (br. s, 2H), 5.18 (dd, <i>J</i> = 6.2, 2.8 Hz, 1H), 4.85 (dd, <i>J</i> = 14.7, 6.1 Hz, 1H), 4.75 (dd, <i>J</i> = 14.7, 5.2 Hz, 1H), 4.55â4.50 (m, 2H), 4.43â4.37 (m, 1H), 1.58 (s, 3H), 1.43 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) Î´C 164.00, 152.38, 151.86, 150.71, 150.10, 149.86, 147.92, 136.95 (d, <i>J</i> = 25.6 Hz), 136.56, 135.99, 135.55, 130.64, 130.23, 129.44, 128.46, 128.25, 126.21, 121.67, 117.89, 114.46, 89.60, 85.36, 82.53, 81.68, 64.87, 47.08, 27.23, 25.55; <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>) Î´F 65.69; HRMS (ESI) C<sub>30</sub>H<sub>29</sub>N<sub>7</sub>O<sub>7</sub>FS (M + H<sup>+</sup>) requires 650.1828, found 650.1810; <i>t</i><sub>R</sub> (LCMS_extended) = 2.55 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3328, 1725, 1635, 1606, 1572, 1503, 1410, 1374, 1329, 1268, 1210. ((3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-8-((quinolin-6-ylmethyl)amino)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl 4-(fluorosulfonyl)benzoate (68 mg, 0.10 mmol) was dissolved in a 5:2 mixture of TFA/H<sub>2</sub>O (1.4 mL) and stirred at room temperature for 30 min. The solvent was then removed under reduced pressure to give the crude product, which was purified by reverse-phase C18 chromatography with the Biotage SP1 purification system (water/MeCN + 1% formic acid, 90:10 to 60:40), followed by silica gel chromatography with the Biotage SP1 purification system (EtOAc/EtOH 95:5 to 40:60) to give the title compound <b>18</b> as an amorphous yellow solid (7 mg, 11%); <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´H 8.84 (dd, <i>J</i> = 4.2, 1.7 Hz, 1H), 8.28 (dd, <i>J</i> = 8.5, 0.9 Hz, 1H), 8.18 (d, <i>J</i> = 8.6 Hz, 2H), 8.10 (d, <i>J</i> = 8.5 Hz, 2H), 7.95 (d, <i>J</i> = 8.6 Hz, 1H), 7.90 (d, <i>J</i> = 0.7 Hz, 1H), 7.82 (s, 1H), 7.78 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.59 (app. t, <i>J</i> = 6.0 Hz, 1H), 7.48 (dd, <i>J</i> = 8.3, 4.2 Hz, 1H), 6.46 (br. s, 2H), 5.86 (d, <i>J</i> = 4.0 Hz, 1H), 5.47 (d, <i>J</i> = 5.2 Hz, 1H), 5.33 (d, <i>J</i> = 5.8 Hz, 1H), 5.22 (app. q, <i>J</i> = 5.0 Hz, 1H), 4.76 (app. d, <i>J</i> = 5.8 Hz, 2H), 4.70 (dd, <i>J</i> = 12.0, 3.2 Hz, 1H), 4.66 (app. q, <i>J</i> = 5.8 Hz, 1H), 4.47 (dd, <i>J</i> = 12.0, 5.0 Hz, 1H), 4.17 (app. td, <i>J</i> = 5.3, 3.2 Hz, 1H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´C 163.81, 152.52, 151.78, 150.07, 149.54, 148.78, 147.02, 138.09, 136.13, 135.66, 135.17 (d, <i>J</i> = 24.3 Hz), 130.70, 129.55, 128.79, 128.72, 127.57, 125.60, 121.49, 117.32, 87.72, 80.55, 70.78, 69.65, 64.78, 45.42; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) Î´F 66.03; HRMS (ESI) C<sub>27</sub>H<sub>25</sub>N<sub>7</sub>O<sub>7</sub>FS (M + H<sup>+</sup>) requires 610.1515, found 610.1499; <i>t</i><sub>R</sub> (LCMS_extended) = 2.04 min; IR (FTIR-ATR)/cm<sup>â1</sup> = 3324, 1724, 1635, 1608, 1573, 1505, 1409, 1330, 1270, 1210.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01709" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00745" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00745" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01709?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01709</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Covalent FP worked example (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Computational chemistry methods, worked example of the covalent FP-assay, covalent FP and intact-protein MS figures, and NMR spectra of final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_001.xlsx">jm9b01709_si_001.xlsx (30.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_002.csv">jm9b01709_si_002.csv (0.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf">jm9b01709_si_003.pdf (3.3 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01709" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Cheeseman</span> - <span class="hlFld-Affiliation affiliation">Cancer Research UK Cancer
Therapeutics Unit, The Institute of Cancer
Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1121-6985" title="Orcid link">http://orcid.org/0000-0003-1121-6985</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0a676b7e7e626f7d2469626f6f796f676b644a636978246b69247f61"><span class="__cf_email__" data-cfemail="b5d8d4c1c1ddd0c29bd6ddd0d0c6d0d8d4dbf5dcd6c79bd4d69bc0de">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan Pettinger</span> - <span class="hlFld-Affiliation affiliation">Cancer Research UK Cancer
Therapeutics Unit, The Institute of Cancer
Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Carter</span> - <span class="hlFld-Affiliation affiliation">Cancer Research UK Cancer
Therapeutics Unit, The Institute of Cancer
Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keith Jones</span> - <span class="hlFld-Affiliation affiliation">Cancer Research UK Cancer
Therapeutics Unit, The Institute of Cancer
Research, London SW7 3RP, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9440-4094" title="Orcid link">http://orcid.org/0000-0002-9440-4094</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.P. synthesized the compounds and carried out the biochemical analysis. M.C. carried out the ROCS conformational analysis. J.P., K.J., and M.D.C. designed the compounds and interpreted the data. J.P. and M.D.C. designed the covalent FP-assay. J.P. and M.D.C. wrote the manuscript. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was funded by an ICR Chairmanâs Studentship Award (JP) and Cancer Research UK grants (C309/A8274, C309/A11566).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4633-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank Katia Grira, Meirion Richards, Maggie Liu, and Amin Mirza of the Structural Chemistry team for their expertise and assistance.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">TCI</td><td class="NLM_def"><p class="first last">targeted covalent inhibitor</p></td></tr><tr><td class="NLM_term">E</td><td class="NLM_def"><p class="first last">enzyme/protein</p></td></tr><tr><td class="NLM_term">EI</td><td class="NLM_def"><p class="first last">reversible enzyme/protein-inhibitor complex</p></td></tr><tr><td class="NLM_term">E-I</td><td class="NLM_def"><p class="first last">irreversible enzyme/protein-inhibitor covalent complex</p></td></tr><tr><td class="NLM_term">I</td><td class="NLM_def"><p class="first last">inhibitor/ligand</p></td></tr><tr><td class="NLM_term">EI*</td><td class="NLM_def"><p class="first last">precovalent complex</p></td></tr><tr><td class="NLM_term"><i>K<sub>i</sub></i></td><td class="NLM_def"><p class="first last">equilibrium constant</p></td></tr><tr><td class="NLM_term"><i>k</i><sub>inact</sub></td><td class="NLM_def"><p class="first last">first-order rate constant for covalent inhibition</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub><sup>inf</sup></td><td class="NLM_def"><p class="first last">theoretical half-life at infinite concentration</p></td></tr><tr><td class="NLM_term"><i>K<sub>I</sub></i></td><td class="NLM_def"><p class="first last">concentration at <i>k</i><sub>inact</sub>/2</p></td></tr><tr><td class="NLM_term"><i>k</i><sub>obs</sub></td><td class="NLM_def"><p class="first last">observed first-order rate constant</p></td></tr><tr><td class="NLM_term"><i>k</i><sub>off</sub></td><td class="NLM_def"><p class="first last">dissociation rate constant</p></td></tr><tr><td class="NLM_term"><i>k</i><sub>on</sub></td><td class="NLM_def"><p class="first last">association rate constant</p></td></tr><tr><td class="NLM_term">%CO</td><td class="NLM_def"><p class="first last">percentage covalent occupancy; percentage total occupancy</p></td></tr><tr><td class="NLM_term">HSP72</td><td class="NLM_def"><p class="first last">Heat shock 70 kDa protein 1</p></td></tr><tr><td class="NLM_term">KRAS</td><td class="NLM_def"><p class="first last">GTPase KRas</p></td></tr><tr><td class="NLM_term">HSF1</td><td class="NLM_def"><p class="first last">Heat shock factor protein 1</p></td></tr><tr><td class="NLM_term">NBD</td><td class="NLM_def"><p class="first last">nucleotide-binding domain</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>m</sub></td><td class="NLM_def"><p class="first last">MichaelisâMenten constant</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">SF</td><td class="NLM_def"><p class="first last"><i>p</i>-sulfonyl fluoride</p></td></tr><tr><td class="NLM_term">ROCS</td><td class="NLM_def"><p class="first last">rapid overlay of chemical structures</p></td></tr><tr><td class="NLM_term">CSD</td><td class="NLM_def"><p class="first last">Cambridge Structural Database</p></td></tr><tr><td class="NLM_term">PI3KÎ´</td><td class="NLM_def"><p class="first last">Phosphoinositide 3-kinase delta</p></td></tr><tr><td class="NLM_term">S<sub>N</sub>Ar</td><td class="NLM_def"><p class="first last">substitution nucleophilic aromatic</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">molecular matched pair</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">NA</td><td class="NLM_def"><p class="first last">not applicable</p></td></tr><tr><td class="NLM_term">ND</td><td class="NLM_def"><p class="first last">not determined</p></td></tr><tr><td class="NLM_term">App.</td><td class="NLM_def"><p class="first last">apparent</p></td></tr><tr><td class="NLM_term">S<sub>A</sub>N</td><td class="NLM_def"><p class="first last">substitution associative nucleophilic</p></td></tr><tr><td class="NLM_term"><i>F</i><sub>b</sub></td><td class="NLM_def"><p class="first last">fraction bound</p></td></tr><tr><td class="NLM_term">A</td><td class="NLM_def"><p class="first last">anisotropy</p></td></tr><tr><td class="NLM_term">mP</td><td class="NLM_def"><p class="first last">millipolarization</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr><tr><td class="NLM_term">FO</td><td class="NLM_def"><p class="first last">fractional reversible occupancy</p></td></tr><tr><td class="NLM_term">sat.</td><td class="NLM_def"><p class="first last">saturated</p></td></tr><tr><td class="NLM_term">cyc</td><td class="NLM_def"><p class="first last">cyclohexane</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">aq.</td><td class="NLM_def"><p class="first last">aqueous</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HBTU</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>,<i>N</i>â²,<i>N</i>â²-Tetramethyl-<i>O</i>-(1<i>H</i>-benzotriazol-1-yl)uronium hexafluorophosphate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span> <span> </span><span class="NLM_article-title">Targeted covalent inhibitors for drug design</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">13408</span>â <span class="NLM_lpage">13421</span>, <span class="refDoi">Â DOI: 10.1002/anie.201601091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1002%2Fanie.201601091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=27539547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A280%3ADC%252BC2szitVegtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=13408-13421&author=T.+A.+Baillie&title=Targeted+covalent+inhibitors+for+drug+design&doi=10.1002%2Fanie.201601091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Covalent Inhibitors for Drug Design</span></div><div class="casAuthors">Baillie Thomas A</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">13408-13421</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In contrast to the traditional mechanism of drug action that relies on the reversible, noncovalent interaction of a ligand with its biological target, a targeted covalent inhibitor (TCI) is designed such that the initial, reversible association is followed by the formation of a covalent bond between an electrophile on the ligand and a nucleophilic center in the protein.  Although this approach offers a variety of potential benefits (high potency and extended duration of action), concerns over the possible toxicological consequences of protein haptenization have hindered the development of the TCI concept.  Recently, approaches to mitigate the risk of serious adverse reactions to this new class of agent have emerged, thus stimulating interest in the field and leading to authorization of the first cadre of TCIs to be marketed.  The covalent inhibitor approach is rapidly gaining acceptance as a valuable tool in drug discovery, and is poised to make a major impact on the design of enzyme inhibitors and receptor modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQYqmMVNSmxle98rul4nhSdfW6udTcc2eZDgpYG7jJ-3Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szitVegtg%253D%253D&md5=c4f6a83580f47f4609541f121d2c63db</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fanie.201601091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201601091%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DTargeted%2520covalent%2520inhibitors%2520for%2520drug%2520design%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D13408%26epage%3D13421%26doi%3D10.1002%2Fanie.201601091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">949</span>â <span class="NLM_lpage">964</span>, <span class="refDoi">Â DOI: 10.4155/fmc.10.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.4155%2Ffmc.10.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20640225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=949-964&author=D.+S.+Johnsonauthor=E.+Weerapanaauthor=B.+F.+Cravatt&title=Strategies+for+discovering+and+derisking+covalent%2C+irreversible+enzyme+inhibitors&doi=10.4155%2Ffmc.10.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span></div><div class="casAuthors">Johnson, Douglas S.; Weerapana, Eranthie; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-964</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article presents several covalent inhibitors, including examples of successful drugs, as well as highly selective, irreversible inhibitors of emerging therapeutic targets, such as fatty acid amide hydolase.  Covalent inhibitors have many desirable features, including increased biochem. efficiency of target disruption, less sensitivity toward pharmacokinetic parameters and increased duration of action that outlasts the pharmacokinetics of the compd.  Safety concerns that must be mitigated include lack of specificity and the potential immunogenicity of protein-inhibitor adduct(s).  Particular attention will be given to recent technologies, such as activity-based protein profiling, which allow one to define the proteome-wide selectivity patterns for covalent inhibitors in vitro and in vivo.  For instance, any covalent inhibitor can, in principle, be modified with a 'clickable' tag to generate an activity probe that is almost indistinguishable from the original agent.  These probes can be applied to any living system across a broad dose range to fully inventory their on and off targets.  The substantial no. of drugs on the market today that act by a covalent mechanism belies historical prejudices against the development of irreversibly acting therapeutic small mols.  Emerging proteomic technologies offer a means to systematically discriminate safe (selective) vs. deleterious (nonselective) covalent inhibitors and thus should inspire their future design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqm0whEybB-LVg90H21EOLACvtfcHk0ljWOsCVnGTx8w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D&md5=07e1f89418b168798ae5c1a9fa43e9c1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.21%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DStrategies%2520for%2520discovering%2520and%2520derisking%2520covalent%252C%2520irreversible%2520enzyme%2520inhibitors%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D949%26epage%3D964%26doi%3D10.4155%2Ffmc.10.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, R. A.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors in drug discovery: From accidental discoveries to avoided liabilities and designed therapies</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1061</span>â <span class="NLM_lpage">1073</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2015.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.drudis.2015.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=26002380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1061-1073&author=R.+A.+Bauer&title=Covalent+inhibitors+in+drug+discovery%3A+From+accidental+discoveries+to+avoided+liabilities+and+designed+therapies&doi=10.1016%2Fj.drudis.2015.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors in drug discovery: from accidental discoveries to avoided liabilities and designed therapies</span></div><div class="casAuthors">Bauer, Renato A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1061-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Drugs that covalently bond to their biol. targets have a long history in drug discovery.  A look at drug approvals in recent years suggests that covalent drugs will continue to make impacts on human health for years to come.  Although fraught with concerns about toxicity, the high potencies and prolonged effects achievable with covalent drugs may result in less-frequent drug dosing and in wide therapeutic margins for patients.  Covalent inhibition can also dissoc. drug pharmacodynamics (PD) from pharmacokinetics (PK), which can result in desired drug efficacy for inhibitors that have short systemic exposure.  Evidence suggests that there is a reduced risk for the development of resistance against covalent drugs, which is a major challenge in areas such as oncol. and infectious disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMkBDsUEu4arVg90H21EOLACvtfcHk0ljWOsCVnGTx8w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D&md5=6ba5b016d2804c7d56838a2724b9a0b9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DR.%2BA.%26atitle%3DCovalent%2520inhibitors%2520in%2520drug%2520discovery%253A%2520From%2520accidental%2520discoveries%2520to%2520avoided%2520liabilities%2520and%2520designed%2520therapies%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D1061%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2015.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomassi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Termathe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer-Wrangowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GrÃ¼tter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlenbrock, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KrÃ¼ll, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaumann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eppmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kibies, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffgaard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heil, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Cuaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuckmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tinnefeld, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahedi, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kast, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Targeting drug resistance in egfr with covalent inhibitors: A structure-based design approach</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6844</span>â <span class="NLM_lpage">6863</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01082</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01082" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6844-6863&author=J.+Engelauthor=A.+Richtersauthor=M.+Getlikauthor=S.+Tomassiauthor=M.+Keulauthor=M.+Termatheauthor=J.+Lategahnauthor=C.+Beckerauthor=S.+Mayer-Wrangowskiauthor=C.+Gr%C3%BCtterauthor=N.+Uhlenbrockauthor=J.+Kr%C3%BCllauthor=N.+Schaumannauthor=S.+Eppmannauthor=P.+Kibiesauthor=F.+Hoffgaardauthor=J.+Heilauthor=S.+Menningerauthor=S.+Ortiz-Cuaranauthor=J.+M.+Heuckmannauthor=V.+Tinnefeldauthor=R.+P.+Zahediauthor=M.+L.+Sosauthor=C.+Schultz-Fademrechtauthor=R.+K.+Thomasauthor=S.+M.+Kastauthor=D.+Rauh&title=Targeting+drug+resistance+in+egfr+with+covalent+inhibitors%3A+A+structure-based+design+approach&doi=10.1021%2Facs.jmedchem.5b01082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach</span></div><div class="casAuthors">Engel, Julian; Richters, Andre; Getlik, Matthaeus; Tomassi, Stefano; Keul, Marina; Termathe, Martin; Lategahn, Jonas; Becker, Christian; Mayer-Wrangowski, Svenja; Gruetter, Christian; Uhlenbrock, Niklas; Kruell, Jasmin; Schaumann, Niklas; Eppmann, Simone; Kibies, Patrick; Hoffgaard, Franziska; Heil, Jochen; Menninger, Sascha; Ortiz-Cuaran, Sandra; Heuckmann, Johannes M.; Tinnefeld, Verena; Zahedi, Rene P.; Sos, Martin L.; Schultz-Fademrecht, Carsten; Thomas, Roman K.; Kast, Stefan M.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6844-6863</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC).  Strong efforts were directed to the development of irreversible inhibitors and led to compd. CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects.  Here, the authors present a structure-based approach, rationalized by subsequent computational anal. of conformational ligand ensembles in soln., to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approx. 1500 compds.  Using protein x-ray crystallog., the authors deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational design approaches.  Chem. synthesis led to further compd. collections that revealed increased biochem. potency and, in part, selectivity toward mutated (L858R and L858R/T790M) vs. nonmutated EGFR.  Further cell-based and kinetic studies were performed to substantiate the authors initial findings.  Utilizing proteolytic digestion and nano-LC-MS/MS anal., the authors confirmed the alkylation of Cys797.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNGJPIuqoVObVg90H21EOLACvtfcHk0lhwTX5rScmltA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM&md5=d5264a2a99d7752af0e587f23aeb7723</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01082%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DTomassi%26aufirst%3DS.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DTermathe%26aufirst%3DM.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DMayer-Wrangowski%26aufirst%3DS.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DUhlenbrock%26aufirst%3DN.%26aulast%3DKr%25C3%25BCll%26aufirst%3DJ.%26aulast%3DSchaumann%26aufirst%3DN.%26aulast%3DEppmann%26aufirst%3DS.%26aulast%3DKibies%26aufirst%3DP.%26aulast%3DHoffgaard%26aufirst%3DF.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DMenninger%26aufirst%3DS.%26aulast%3DOrtiz-Cuaran%26aufirst%3DS.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DTinnefeld%26aufirst%3DV.%26aulast%3DZahedi%26aufirst%3DR.%2BP.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DTargeting%2520drug%2520resistance%2520in%2520egfr%2520with%2520covalent%2520inhibitors%253A%2520A%2520structure-based%2520design%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6844%26epage%3D6863%26doi%3D10.1021%2Facs.jmedchem.5b01082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marino, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladyshev, V. N.</span></span> <span> </span><span class="NLM_article-title">Cysteine function governs its conservation and degeneration and restricts its utilization on protein surfaces</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>404</i></span>,  <span class="NLM_fpage">902</span>â <span class="NLM_lpage">916</span>, <span class="refDoi">Â DOI: 10.1016/j.jmb.2010.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.jmb.2010.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20950627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGiur%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2010&pages=902-916&author=S.+M.+Marinoauthor=V.+N.+Gladyshev&title=Cysteine+function+governs+its+conservation+and+degeneration+and+restricts+its+utilization+on+protein+surfaces&doi=10.1016%2Fj.jmb.2010.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine Function Governs Its Conservation and Degeneration and Restricts Its Utilization on Protein Surfaces</span></div><div class="casAuthors">Marino, Stefano M.; Gladyshev, Vadim N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">404</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">902-916</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cysteine (Cys) is an enigmatic amino acid residue.  Although one of the least abundant amino acids, Cys often occurs in the functional sites of proteins.  Whereas free Cys is a polar amino acid, Cys in proteins is often buried, and its classification on the hydrophobicity scale is ambiguous.  We hypothesized that the deviation of Cys residues from the properties of a free amino acid is due to their reactivity and addressed this possibility by examg. Cys in large protein structure data sets.  Compared to other amino acids, Cys was characterized by the most extreme conservation pattern, with the majority of Cys being either highly conserved or poorly conserved.  In addn., clustering of Cys with another Cys residue was assocd. with high conservation, whereas exposure of Cys on protein surfaces was assocd. with low conservation.  Moreover, although clustered Cys behaved as polar residues, isolated Cys was the most buried residue of all, in disagreement with known chem. properties of Cys.  Thus, the anomalous hydrophobic behavior and conservation pattern of Cys can be explained by elimination of isolated Cys from protein surfaces during evolution and by clustering of other Cys residues.  These findings indicate that Cys abundance is governed by Cys function in proteins rather than by the sheer chem.-phys. properties of free amino acids, and suggest that a high tendency of Cys to be functionally active can considerably limit its abundance on protein surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpePL0RDIZDlrVg90H21EOLACvtfcHk0lhwTX5rScmltA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGiur%252FF&md5=c2de18b2dcc4ac2b431e7f80173d5af3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DMarino%26aufirst%3DS.%2BM.%26aulast%3DGladyshev%26aufirst%3DV.%2BN.%26atitle%3DCysteine%2520function%2520governs%2520its%2520conservation%2520and%2520degeneration%2520and%2520restricts%2520its%2520utilization%2520on%2520protein%2520surfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D404%26spage%3D902%26epage%3D916%26doi%3D10.1016%2Fj.jmb.2010.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donner, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addo, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cepa, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woller, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shotwell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutti, J. E.</span></span> <span> </span><span class="NLM_article-title">Novel modes of inhibition of wild-type isocitrate dehydrogenase 1 (IDH1): Direct covalent modification of his315</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6647</span>â <span class="NLM_lpage">6657</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtrbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6647-6657&author=C.+G.+Jakobauthor=A.+K.+Upadhyayauthor=P.+L.+Donnerauthor=E.+Nichollauthor=S.+N.+Addoauthor=W.+Qiuauthor=C.+Lingauthor=S.+M.+Gopalakrishnanauthor=M.+Torrentauthor=S.+P.+Cepaauthor=J.+Shanleyauthor=A.+R.+Shoemakerauthor=C.+C.+Sunauthor=A.+Vasudevanauthor=K.+R.+Wollerauthor=J.+B.+Shotwellauthor=B.+Shawauthor=Z.+Bianauthor=J.+E.+Hutti&title=Novel+modes+of+inhibition+of+wild-type+isocitrate+dehydrogenase+1+%28IDH1%29%3A+Direct+covalent+modification+of+his315&doi=10.1021%2Facs.jmedchem.8b00305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Modes of Inhibition of Wild-Type Isocitrate Dehydrogenase 1 (IDH1): Direct Covalent Modification of His315</span></div><div class="casAuthors">Jakob, Clarissa G.; Upadhyay, Anup K.; Donner, Pamela L.; Nicholl, Emily; Addo, Sadiya N.; Qiu, Wei; Ling, Christopher; Gopalakrishnan, Sujatha M.; Torrent, Maricel; Cepa, Steven P.; Shanley, Jason; Shoemaker, Alexander R.; Sun, Chaohong C.; Vasudevan, Anil; Woller, Kevin R.; Shotwell, J. Brad; Shaw, Bailin; Bian, Zhiguo; Hutti, Jessica E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6647-6657</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">IDH1 plays a crit. role in a no. of metabolic processes and serves as a key source of cytosolic NADPH under conditions of cellular stress.  However, few inhibitors of wild-type IDH1 have been reported.  Here we present the discovery and biochem. characterization of two novel inhibitors of wild-type IDH1.  In addn., we present the first ligand-bound crystallog. characterization of these novel small mol. IDH1 binding pockets.  Importantly, the NADPH competitive Î±,Î²-unsatd. enone 1 makes a unique covalent linkage through active site H315.  As few small mols. have been shown to covalently react with histidine residues, these data support the potential utility of an underutilized strategy for reversible covalent small mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Rg5CN6JOyLVg90H21EOLACvtfcHk0li6wNbtut1WNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtrbF&md5=15c54988cd3080d5a675badbb62b6fec</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00305%26sid%3Dliteratum%253Aachs%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DDonner%26aufirst%3DP.%2BL.%26aulast%3DNicholl%26aufirst%3DE.%26aulast%3DAddo%26aufirst%3DS.%2BN.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DLing%26aufirst%3DC.%26aulast%3DGopalakrishnan%26aufirst%3DS.%2BM.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DCepa%26aufirst%3DS.%2BP.%26aulast%3DShanley%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DWoller%26aufirst%3DK.%2BR.%26aulast%3DShotwell%26aufirst%3DJ.%2BB.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DBian%26aufirst%3DZ.%26aulast%3DHutti%26aufirst%3DJ.%2BE.%26atitle%3DNovel%2520modes%2520of%2520inhibition%2520of%2520wild-type%2520isocitrate%2520dehydrogenase%25201%2520%2528IDH1%2529%253A%2520Direct%2520covalent%2520modification%2520of%2520his315%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6647%26epage%3D6657%26doi%3D10.1021%2Facs.jmedchem.8b00305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Horst, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of novel covalent bromodomain and extra-terminal domain (BET) inhibitors targeting a methionine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8202</span>â <span class="NLM_lpage">8211</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8202-8211&author=O.+A.+Kharenkoauthor=R.+G.+Patelauthor=S.+D.+Brownauthor=C.+Calosingauthor=A.+Whiteauthor=D.+Lakshminarasimhanauthor=R.+K.+Sutoauthor=B.+C.+Duffyauthor=D.+B.+Kitchenauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=E.+H.+van+der+Horstauthor=P.+R.+Young&title=Design+and+characterization+of+novel+covalent+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+targeting+a+methionine&doi=10.1021%2Facs.jmedchem.8b00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Novel Covalent Bromodomain and Extra-Terminal Domain (BET) Inhibitors Targeting a Methionine</span></div><div class="casAuthors">Kharenko, Olesya A.; Patel, Reena G.; Brown, S. David; Calosing, Cyrus; White, Andre; Lakshminarasimhan, Damodharan; Suto, Robert K.; Duffy, Bryan C.; Kitchen, Douglas B.; McLure, Kevin G.; Hansen, Henrik C.; van der Horst, Edward H.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8202-8211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET proteins are key epigenetic regulators that regulate transcription through binding to acetylated lysine (AcLys) residues of histones and transcription factors through bromodomains (BDs).  The disruption of this interaction with small mol. bromodomain inhibitors is a promising approach to treat various diseases including cancer, autoimmune and cardiovascular diseases.  Covalent inhibitors can potentially offer a more durable target inhibition leading to improved in vivo pharmacol.  Here the authors describe the design of covalent inhibitors of BRD4(BD1) that target a methionine in the binding pocket by attaching an epoxide warhead to a suitably oriented noncovalent inhibitor.  Using thermal denaturation, MALDI-TOF mass spectrometry, and an x-ray crystal structure, the authors demonstrate that these inhibitors selectively form a covalent bond with Met 149 in BRD4(BD1) but not other bromodomains and provide durable transcriptional and antiproliferative activity in cell based assays.  Covalent targeting of methionine offers a novel approach to drug discovery for BET proteins and other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHccGRxf5ShrVg90H21EOLACvtfcHk0li6wNbtut1WNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO&md5=1c3ae393a5f8a8c12ffb16e6f3798118</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00666%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520novel%2520covalent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520targeting%2520a%2520methionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8202%26epage%3D8211%26doi%3D10.1021%2Facs.jmedchem.8b00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimster, N. P.</span></span> <span> </span><span class="NLM_article-title">Beyond cysteine: Recent developments in the area of targeted covalent inhibition</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">30</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1016/j.cbpa.2018.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.cbpa.2018.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29857316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVelt7zL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2018&pages=30-38&author=H.+Mukherjeeauthor=N.+P.+Grimster&title=Beyond+cysteine%3A+Recent+developments+in+the+area+of+targeted+covalent+inhibition&doi=10.1016%2Fj.cbpa.2018.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond cysteine: recent developments in the area of targeted covalent inhibition</span></div><div class="casAuthors">Mukherjee, Herschel; Grimster, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30-38</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Over the past decade targeted covalent inhibitors have undergone a renaissance due to the clin. validation and regulatory approval of several small mol. therapeutics that are designed to irreversibly modify their target protein.  Invariably, these compds. rely on the serendipitous placement of a cysteine residue proximal to the small mol. binding site; while this strategy has afforded numerous successes, it necessarily limits the no. of proteins that can be targeted by this approach.  This drawback has led several research groups to develop novel methodologies that target non-cysteine residues for covalent modification.  Herein, we survey the current literature of warheads that covalently modify non-cysteine amino acids in proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy0WIngEAtJLVg90H21EOLACvtfcHk0liEfC7H2NFX-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVelt7zL&md5=2f576af4dbd3046b3ac7486d2dfc04d9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DH.%26aulast%3DGrimster%26aufirst%3DN.%2BP.%26atitle%3DBeyond%2520cysteine%253A%2520Recent%2520developments%2520in%2520the%2520area%2520of%2520targeted%2520covalent%2520inhibition%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D44%26spage%3D30%26epage%3D38%26doi%3D10.1016%2Fj.cbpa.2018.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J. M.</span></span> <span> </span><span class="NLM_article-title">A perspective on the kinetics of covalent and irreversible inhibition</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3</span>â <span class="NLM_lpage">20</span>, <span class="refDoi">Â DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&author=J.+M.+Strelow&title=A+perspective+on+the+kinetics+of+covalent+and+irreversible+inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0liEfC7H2NFX-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520perspective%2520on%2520the%2520kinetics%2520of%2520covalent%2520and%2520irreversible%2520inhibition%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="note"><p class="first last"><i>t</i><sub>1/2</sub><sup>inf</sup> = lnâ¯2/<i>k</i><sub>inact</sub></p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="note"><p class="first last"><i>K<sub>I</sub></i> = (<i>k</i><sub>off</sub> + <i>k</i><sub>inact</sub>)/<i>k</i><sub>on</sub>, <i>K<sub>i</sub></i> = <i>k</i><sub>off</sub>/<i>k</i><sub>on</sub> When <i>k</i><sub>off</sub> â« <i>k</i><sub>inact</sub> then <i>k</i><sub>off</sub> + <i>k</i><sub>inact</sub> = â¼<i>k</i><sub>off</sub> so <i>K<sub>I</sub></i> = â¼<i>K<sub>i</sub></i><i>k</i><sub>off</sub> = reversible dissociation first order rate constant of the non-covalent complex EI, <i>k</i><sub>on</sub> = reversible association second order rate constant of the non-covalent complex EI, <i>k</i><sub>inact</sub> = first order rate constant of the covalent complex E-I, <i>K<sub>i</sub></i> = true reversible equilibrium constant of the non-covalent complex EI, <i>K<sub>I</sub></i> = pseudo reversible equilibrium constant of the non-covalent complex EI, determined from <i>k</i><sub>obs</sub> = <i>k</i><sub>inact</sub>/2, <i>k</i><sub>obs</sub> = concentration dependent pseudo first order rate constant</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Cesco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufresne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittermaier, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors design and discovery</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">96</span>â <span class="NLM_lpage">114</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2017.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.ejmech.2017.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28651155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=96-114&author=S.+De%0ACescoauthor=J.+Kurianauthor=C.+Dufresneauthor=A.+K.+Mittermaierauthor=N.+Moitessier&title=Covalent+inhibitors+design+and+discovery&doi=10.1016%2Fj.ejmech.2017.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors design and discovery</span></div><div class="casAuthors">De Cesco, Stephane; Kurian, Jerry; Dufresne, Caroline; Mittermaier, Anthony K.; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-114</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In the history of therapeutics, covalent drugs occupy a very distinct category.  While representing a significant fraction of the drugs on the market, very few have been deliberately designed to interact covalently with their biol. target.  In this review, the prevalence of covalent drugs will first be briefly covered, followed by an introduction to their mechanisms of action and more detailed discussions of their discovery and the development of safe and efficient covalent enzyme inhibitors.  All stages of a drug discovery program will be covered, from target considerations to lead optimization, strategies to tune reactivity and computational methods.  The goal of this article is to provide an overview of the field and to outline good practices that are needed for the proper assessment and development of covalent inhibitors as well as a good understanding of the potential and limitations of current computational methods for the design of covalent drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeLNDmLtBdB7Vg90H21EOLACvtfcHk0liEfC7H2NFX-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI&md5=551c9a9c4bd69d4ad4d9156c315d5a94</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BCesco%26aufirst%3DS.%26aulast%3DKurian%26aufirst%3DJ.%26aulast%3DDufresne%26aufirst%3DC.%26aulast%3DMittermaier%26aufirst%3DA.%2BK.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DCovalent%2520inhibitors%2520design%2520and%2520discovery%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D96%26epage%3D114%26doi%3D10.1016%2Fj.ejmech.2017.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">The reactivity-driven biochemical mechanism of covalent KRASG12C inhibitors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">454</span>â <span class="NLM_lpage">462</span>, <span class="refDoi">Â DOI: 10.1038/s41594-018-0061-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1038%2Fs41594-018-0061-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29760531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=454-462&author=R.+Hansenauthor=U.+Petersauthor=A.+Babbarauthor=Y.+Chenauthor=J.+Fengauthor=M.+R.+Janesauthor=L.-S.+Liauthor=P.+Renauthor=Y.+Liuauthor=P.+P.+Zarrinkar&title=The+reactivity-driven+biochemical+mechanism+of+covalent+KRASG12C+inhibitors&doi=10.1038%2Fs41594-018-0061-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The reactivity-driven biochemical mechanism of covalent KRASG12C inhibitors</span></div><div class="casAuthors">Hansen, Rasmus; Peters, Ulf; Babbar, Anjali; Chen, Yuching; Feng, Jun; Janes, Matthew R.; Li, Lian-Sheng; Ren, Pingda; Liu, Yi; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">454-462</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Activating mutations in KRAS are among the most common tumor driver mutations.  Until recently, KRAS had been considered undruggable with small mols.; the discovery of the covalent KRASG12C inhibitors ARS-853 and ARS-1620 has demonstrated that it is feasible to inhibit KRAS with high potency in cells and animals.  Although the biol. activity of these inhibitors has been described, the biochem. mechanism of how the compds. achieve potent inhibition remained incompletely understood.  We now show that the activity of ARS-853 and ARS-1620 is primarily driven by KRAS-mediated catalysis of the chem. reaction with Cys12 in human KRASG12C, while the reversible binding affinity is weak, in the hundreds of micromolar or higher range.  The mechanism resolves how an induced, shallow and dynamic pocket not expected to support high-affinity binding of small mols. can nevertheless be targeted with potent inhibitors and may be applicable to other targets conventionally considered undruggable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrya_A4ZYnOu7Vg90H21EOLACvtfcHk0ljXwixELC--Ug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL&md5=b3f9198f55998306f3b881c256be22a9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0061-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0061-5%26sid%3Dliteratum%253Aachs%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DThe%2520reactivity-driven%2520biochemical%2520mechanism%2520of%2520covalent%2520KRASG12C%2520inhibitors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D454%26epage%3D462%26doi%3D10.1038%2Fs41594-018-0061-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darjania, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarieh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu-Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS mutant cancers with a covalent G12C-specific inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">578</span>â <span class="NLM_lpage">589.e17</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2018.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.cell.2018.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29373830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=578-589.e17&author=M.+R.+Janesauthor=J.+Zhangauthor=L.-S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=X.+Guoauthor=Y.+Chenauthor=A.+Babbarauthor=S.+J.+Firdausauthor=L.+Darjaniaauthor=J.+Fengauthor=J.+H.+Chenauthor=S.+Liauthor=S.+Liauthor=Y.+O.+Longauthor=C.+Thachauthor=Y.+Liuauthor=A.+Zariehauthor=T.+Elyauthor=J.+M.+Kucharskiauthor=L.+V.+Kesslerauthor=T.+Wuauthor=K.+Yuauthor=Y.+Wangauthor=Y.+Yaoauthor=X.+Dengauthor=P.+P.+Zarrinkarauthor=D.+Brehmerauthor=D.+Dhanakauthor=M.+V.+Lorenziauthor=D.+Hu-Loweauthor=M.+P.+Patricelliauthor=P.+Renauthor=Y.+Liu&title=Targeting+KRAS+mutant+cancers+with+a+covalent+G12C-specific+inhibitor&doi=10.1016%2Fj.cell.2018.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span></div><div class="casAuthors">Janes, Matthew R.; Zhang, Jingchuan; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Guo, Xin; Chen, Yuching; Babbar, Anjali; Firdaus, Sarah J.; Darjania, Levan; Feng, Jun; Chen, Jeffrey H.; Li, Shuangwei; Li, Shisheng; Long, Yun O.; Thach, Carol; Liu, Yuan; Zarieh, Ata; Ely, Tess; Kucharski, Jeff M.; Kessler, Linda V.; Wu, Tao; Yu, Ke; Wang, Yi; Yao, Yvonne; Deng, Xiaohu; Zarrinkar, Patrick P.; Brehmer, Dirk; Dhanak, Dashyant; Lorenzi, Matthew V.; Hu-Lowe, Dana; Patricelli, Matthew P.; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-589.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRASG12C was recently identified to be potentially druggable by allele-specific covalent targeting of Cys 12 in vicinity to an inducible allosteric switch II pocket (S-IIP).  Success of this approach requires active cycling of KRASG12C between its active-GTP and inactive-GDP conformations as accessibility of the S-IIP is restricted only to the GDP-bound state.  This strategy proved feasible for inhibiting mutant KRAS in vitro; however, it is uncertain whether this approach would translate to in vivo.  Here, the authors describe structure-based design and identification of ARS-1620, a covalent compd. with high potency and selectivity for KRASG12C.  ARS-1620 achieves rapid and sustained in vivo target occupancy to induce tumor regression.  The authors use ARS-1620 to dissect oncogenic KRAS dependency and demonstrate that monolayer culture formats significantly underestimate KRAS dependency in vivo.  This study provides in vivo evidence that mutant KRAS can be selectively targeted and reveals ARS-1620 as representing a new generation of KRASG12C-specific inhibitors with promising therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVmv4KAl4_dLVg90H21EOLACvtfcHk0ljXwixELC--Ug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D&md5=d0883bde981b51e41a794179ffae87be</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DDarjania%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DY.%2BO.%26aulast%3DThach%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarieh%26aufirst%3DA.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DHu-Lowe%26aufirst%3DD.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DTargeting%2520KRAS%2520mutant%2520cancers%2520with%2520a%2520covalent%2520G12C-specific%2520inhibitor%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D578%26epage%3D589.e17%26doi%3D10.1016%2Fj.cell.2018.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">FernÃ¡ndez-FernÃ¡ndez, M. R.</span>; <span class="NLM_string-name">Valpuesta, J.</span></span> <span> </span><span class="NLM_article-title">HSP70 chaperone: A master player in protein homeostasis [version 1; peer review: 3 approved]</span>.  <i>F1000Research</i> <span class="NLM_year">2018</span>,  <span class="NLM_volume">7</span>.<span class="refDoi">Â DOI: 10.12688/f1000research.15528.1</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.12688%2Ff1000research.15528.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=30338057" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+R.+Fern%C3%A1ndez-Fern%C3%A1ndez&author=J.+Valpuesta&title=HSP70+chaperone%3A+A+master+player+in+protein+homeostasis+%5Bversion+1%3B+peer+review%3A+3+approved%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.15528.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.12688%252Ff1000research.15528.1%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Fern%25C3%25A1ndez%26aufirst%3DM.%2BR.%26atitle%3DHSP70%2520chaperone%253A%2520A%2520master%2520player%2520in%2520protein%2520homeostasis%2520%255Bversion%25201%253B%2520peer%2520review%253A%25203%2520approved%255D%26jtitle%3DF1000Research%26date%3D2018%26volume%3D7%26doi%3D10.12688%2Ff1000research.15528.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scissum Gunn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manne, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, M.</span></span> <span> </span><span class="NLM_article-title">Targeting HSP70: A possible therapy for cancer</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">156</span>â <span class="NLM_lpage">166</span>, <span class="refDoi">Â DOI: 10.1016/j.canlet.2016.01.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.canlet.2016.01.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=26898980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVGktrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=156-166&author=S.+Kumarauthor=J.+Stokesauthor=U.+P.+Singhauthor=K.+Scissum+Gunnauthor=A.+Acharyaauthor=U.+Manneauthor=M.+Mishra&title=Targeting+HSP70%3A+A+possible+therapy+for+cancer&doi=10.1016%2Fj.canlet.2016.01.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Hsp70: A possible therapy for cancer</span></div><div class="casAuthors">Kumar, Sanjay; Stokes, James; Singh, Udai P.; Scissum Gunn, Karyn; Acharya, Arbind; Manne, Upender; Mishra, Manoj</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">156-166</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">In all organisms, heat-shock proteins (HSPs) provide an ancient defense system.  These proteins act as mol. chaperones by assisting proper folding and refolding of misfolded proteins and aid in the elimination of old and damaged cells.  HSPs include Hsp100, Hsp90, Hsp70, Hsp40, and small HSPs.  Through its substrate-binding domains, Hsp70 interacts with wide spectrum of mols., ranging from unfolded to natively folded and aggregated proteins, and provides cytoprotective role against various cellular stresses.  Under pathophysiol. conditions, the high expression of Hsp70 allows cells to survive with lethal injuries.  Increased Hsp70, by interacting at several points on apoptotic signaling pathways, leads to inhibition of apoptosis.  Elevated expression of Hsp70 in cancer cells may be responsible for tumorigenesis and for tumor progression by providing resistance to chemotherapy.  In contrast, inhibition or knockdown of Hsp70 reduces the size of tumors and can cause their complete regression.  Moreover, extracellular Hsp70 acts as an immunogen that participates in cross presentation of MHC-I mols.  The goals of this review are to examine the roles of Hsp70 in cancer and to present strategies targeting Hsp70 in the development of cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9lrUPziWvP7Vg90H21EOLACvtfcHk0lj8ggYWBOLedg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVGktrk%253D&md5=6295bea5fef02a457a9fdbed9350f850</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.01.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.01.056%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DStokes%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DU.%2BP.%26aulast%3DScissum%2BGunn%26aufirst%3DK.%26aulast%3DAcharya%26aufirst%3DA.%26aulast%3DManne%26aufirst%3DU.%26aulast%3DMishra%26aufirst%3DM.%26atitle%3DTargeting%2520HSP70%253A%2520A%2520possible%2520therapy%2520for%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2016%26volume%3D374%26spage%3D156%26epage%3D166%26doi%3D10.1016%2Fj.canlet.2016.01.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powers, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barillari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montfort, R. L. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span> <span> </span><span class="NLM_article-title">Targeting HSP70: The second potentially druggable heat shock protein and molecular chaperone?</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1542</span>â <span class="NLM_lpage">1550</span>, <span class="refDoi">Â DOI: 10.4161/cc.9.8.11204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.4161%2Fcc.9.8.11204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20372081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12qsL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1542-1550&author=M.+V.+Powersauthor=K.+Jonesauthor=C.+Barillariauthor=I.+Westwoodauthor=R.+L.+M.+V.+Montfortauthor=P.+Workman&title=Targeting+HSP70%3A+The+second+potentially+druggable+heat+shock+protein+and+molecular+chaperone%3F&doi=10.4161%2Fcc.9.8.11204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting HSP70: the second potentially druggable heat shock protein and molecular chaperone?</span></div><div class="casAuthors">Powers, Marissa V.; Jones, Keith; Barillari, Caterina; Westwood, Isaac; van Montfort, Rob L. M.; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1542-1550</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  The HSF1-mediated stress response pathway is steadily gaining momentum as a crit. source of targets for cancer therapy.  Key mediators of this pathway include mol. chaperones such as heat shock protein (HSP) 90.  There was considerable progress in targeting HSP90 and the preclin. efficacy and signs of early clin. activity of HSP90 inhibitors have provided proof-of-concept for targeting this group of proteins.  The HSP70 family of mol. chaperones are also key mediators of the HSF-1-stress response pathway and have multiple addnl. roles in protein folding, trafficking, and degrdn., as well as regulating apoptosis.  Genetic and biochem. studies have supported the discovery of HSP70 inhibitors which have the potential for use as single agents or in combination to enhance the effects of classical chemotherapeutic or molecularly targeted drugs including HSP90 inhibitors.  Here we provide a perspective on the progress made so far in discovering small mols. which target the HSP70 family, in the context of the available structural data and potential issues in drugging this key chaperone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpODWFSuNNDKbVg90H21EOLACvtfcHk0lj8ggYWBOLedg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12qsL%252FO&md5=12ad24d78fe0c9d538d889ecde1daa18</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.8.11204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.8.11204%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DM.%2BV.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DWestwood%26aufirst%3DI.%26aulast%3DMontfort%26aufirst%3DR.%2BL.%2BM.%2BV.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DTargeting%2520HSP70%253A%2520The%2520second%2520potentially%2520druggable%2520heat%2520shock%2520protein%2520and%2520molecular%2520chaperone%253F%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D1542%26epage%3D1550%26doi%3D10.4161%2Fcc.9.8.11204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgognoni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokurno, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macias, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surgenor, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A. J.</span></span> <span> </span><span class="NLM_article-title">Novel adenosine-derived inhibitors of 70 kDa heat shock protein, discovered through structure-based design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1510</span>â <span class="NLM_lpage">1513</span>, <span class="refDoi">Â DOI: 10.1021/jm801627a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801627a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BD1MXis1arsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1510-1513&author=D.+S.+Williamsonauthor=J.+Borgognoniauthor=A.+Clayauthor=Z.+Danielsauthor=P.+Dokurnoauthor=M.+J.+Drysdaleauthor=N.+Foloppeauthor=G.+L.+Francisauthor=C.+J.+Grahamauthor=R.+Howesauthor=A.+T.+Maciasauthor=J.+B.+Murrayauthor=R.+Parsonsauthor=T.+Shawauthor=A.+E.+Surgenorauthor=L.+Terryauthor=Y.+Wangauthor=M.+Woodauthor=A.+J.+Massey&title=Novel+adenosine-derived+inhibitors+of+70+kDa+heat+shock+protein%2C+discovered+through+structure-based+design&doi=10.1021%2Fjm801627a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Adenosine-Derived Inhibitors of 70 kDa Heat Shock Protein, Discovered Through Structure-Based Design</span></div><div class="casAuthors">Williamson, Douglas S.; Borgognoni, Jenifer; Clay, Alexandra; Daniels, Zoe; Dokurno, Pawel; Drysdale, Martin J.; Foloppe, Nicolas; Francis, Geraint L.; Graham, Christopher J.; Howes, Rob; Macias, Alba T.; Murray, James B.; Parsons, Rachel; Shaw, Terry; Surgenor, Allan E.; Terry, Lindsey; Wang, Yikang; Wood, Mike; Massey, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1510-1513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design and synthesis of novel adenosine-derived inhibitors of HSP70, guided by modeling and X-ray crystallog. structures of these compds. in complex with HSC70/BAG-1, is described.  Examples exhibited submicromolar affinity for HSP70, were highly selective over HSP90, and some displayed potency against HCT116 cells.  Exposure of I to HCT116 cells caused significant redn. in cellular levels of Raf-1 and Her2 at concns. similar to that which caused cell growth arrest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7sPdbJciUk7Vg90H21EOLACvtfcHk0lj8ggYWBOLedg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXis1arsbg%253D&md5=714b981545d1aa8db6627a252b3da8cc</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm801627a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801627a%26sid%3Dliteratum%253Aachs%26aulast%3DWilliamson%26aufirst%3DD.%2BS.%26aulast%3DBorgognoni%26aufirst%3DJ.%26aulast%3DClay%26aufirst%3DA.%26aulast%3DDaniels%26aufirst%3DZ.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DFrancis%26aufirst%3DG.%2BL.%26aulast%3DGraham%26aufirst%3DC.%2BJ.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DMacias%26aufirst%3DA.%2BT.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DShaw%26aufirst%3DT.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26aulast%3DTerry%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DMassey%26aufirst%3DA.%2BJ.%26atitle%3DNovel%2520adenosine-derived%2520inhibitors%2520of%252070%2520kDa%2520heat%2520shock%2520protein%252C%2520discovered%2520through%2520structure-based%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1510%26epage%3D1513%26doi%3D10.1021%2Fjm801627a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prodromou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalusa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aherne, W.</span></span> <span> </span><span class="NLM_article-title">Detection of the ATPase activity of the molecular chaperones HSP90 and HSP72 using the transcreener ADP assay kit</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">279</span>â <span class="NLM_lpage">286</span>, <span class="refDoi">Â DOI: 10.1177/1087057109360253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1177%2F1087057109360253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20147598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFWjsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=279-286&author=M.+Rowlandsauthor=C.+McAndrewauthor=C.+Prodromouauthor=L.+Pearlauthor=A.+Kalusaauthor=K.+Jonesauthor=P.+Workmanauthor=W.+Aherne&title=Detection+of+the+ATPase+activity+of+the+molecular+chaperones+HSP90+and+HSP72+using+the+transcreener+ADP+assay+kit&doi=10.1177%2F1087057109360253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of the ATPase activity of the molecular chaperones Hsp90 and Hsp72 using the transcreener ADP assay kit</span></div><div class="casAuthors">Rowlands, Martin; McAndrew, Craig; Prodromou, Chrisostomos; Pearl, Laurence; Kalusa, Andrew; Jones, Keith; Workman, Paul; Aherne, Wynne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">279-286</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The mol. chaperone heat shock protein 90 (Hsp90) is required for the correct folding and stability of a no. of client proteins that are important for the growth and maintenance of cancer cells.  Heat shock protein 72 (Hsp72), a co-chaperone of Hsp90, is also emerging as an attractive cancer drug target.  Both proteins bind and hydrolyze ATP, and ATPase activity is essential for their function.  Inhibition of Hsp90 ATPase activity leads to the degrdn. of client proteins, resulting in cell growth inhibition and apoptosis.  Several small-mol. inhibitors of the ATPase activity of Hsp90 have been described and are currently being evaluated clin. for the treatment of cancer.  A no. of methods for the measurement of ATPase activity have been previously used, but not all of these are ideally suited to screening cascades in drug discovery projects.  The authors have evaluated the use of com. reagents (Transcreener ADP) for the measurement of ATPase activity of both yeast and human Hsp90 (ATP Km â¼500 Î¼M) and human Hsp72 (ATP Km â¼1 Î¼M).  The low ATPase activity of human Hsp90 and its stimulation by the co-chaperone Aha1 was measured with ease using reduced incubation times, generating robust data (Z' = 0.75).  The potency of several small-mol. inhibitors of both Hsp90 and Hsp72 was detd. using the Transcreener reagents and compared well to that detd. using other assay formats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhgslDxE_GR7Vg90H21EOLACvtfcHk0ljHZG0hHh5B5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFWjsrk%253D&md5=c5d113efdc01600390b365edd88f2525</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1087057109360253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109360253%26sid%3Dliteratum%253Aachs%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DProdromou%26aufirst%3DC.%26aulast%3DPearl%26aufirst%3DL.%26aulast%3DKalusa%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DAherne%26aufirst%3DW.%26atitle%3DDetection%2520of%2520the%2520ATPase%2520activity%2520of%2520the%2520molecular%2520chaperones%2520HSP90%2520and%2520HSP72%2520using%2520the%2520transcreener%2520ADP%2520assay%2520kit%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2010%26volume%3D15%26spage%3D279%26epage%3D286%26doi%3D10.1177%2F1087057109360253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bihan, Y.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">An irreversible inhibitor of HSP72 that unexpectedly targets lysine-56</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3536</span>â <span class="NLM_lpage">3540</span>, <span class="refDoi">Â DOI: 10.1002/anie.201611907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1002%2Fanie.201611907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28225177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A280%3ADC%252BC1c3ptlyjtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3536-3540&author=J.+Pettingerauthor=Y.-V.+Le+Bihanauthor=M.+Widyaauthor=R.+L.+M.+van+Montfortauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=An+irreversible+inhibitor+of+HSP72+that+unexpectedly+targets+lysine-56&doi=10.1002%2Fanie.201611907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">An Irreversible Inhibitor of HSP72 that Unexpectedly Targets Lysine-56</span></div><div class="casAuthors">Pettinger Jonathan; Le Bihan Yann-Vai; van Montfort Rob L M; Jones Keith; Cheeseman Matthew D; Le Bihan Yann-Vai; van Montfort Rob L M; Widya Marcella</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3536-3540</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The stress-inducible molecular chaperone, HSP72, is an important therapeutic target in oncology, but inhibiting this protein with small molecules has proven particularly challenging.  Validating HSP72 inhibitors in cells is difficult owing to competition with the high affinity and abundance of its endogenous nucleotide substrates.  We hypothesized this could be overcome using a cysteine-targeted irreversible inhibitor.  Using rational design, we adapted a validated 8-N-benzyladenosine ligand for covalent bond formation and confirmed targeted irreversible inhibition.  However, no cysteine in the protein was modified; instead, we demonstrate that lysine-56 is the key nucleophilic residue.  Targeting this lysine could lead to a new design paradigm for HSP72 chemical probes and drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2q3QuKVDECGBAiYWy6n3SfW6udTcc2eY6MZJ5JPeiZLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3ptlyjtg%253D%253D&md5=30645b206963d2f48add1a4d1de5376c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611907%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DLe%2BBihan%26aufirst%3DY.-V.%26aulast%3DWidya%26aufirst%3DM.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DAn%2520irreversible%2520inhibitor%2520of%2520HSP72%2520that%2520unexpectedly%2520targets%2520lysine-56%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2017%26volume%3D56%26spage%3D3536%26epage%3D3540%26doi%3D10.1002%2Fanie.201611907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Lysine-targeting covalent inhibitors</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">15200</span>â <span class="NLM_lpage">15209</span>, <span class="refDoi">Â DOI: 10.1002/anie.201707630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1002%2Fanie.201707630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28853194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A280%3ADC%252BC1cbivVGmsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=15200-15209&author=J.+Pettingerauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Lysine-targeting+covalent+inhibitors&doi=10.1002%2Fanie.201707630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Targeting Covalent Inhibitors</span></div><div class="casAuthors">Pettinger Jonathan; Jones Keith; Cheeseman Matthew D</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">15200-15209</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted covalent inhibitors have gained widespread attention in drug discovery as a validated method to circumvent acquired resistance in oncology.  This strategy exploits small-molecule/protein crystal structures to design tightly binding ligands with appropriately positioned electrophilic warheads.  Whilst most focus has been on targeting binding-site cysteine residues, targeting nucleophilic lysine residues can also represent a viable approach to irreversible inhibition.  However, owing to the basicity of the  -amino group in lysine, this strategy generates a number of specific challenges.  Herein, we review the key principles for inhibitor design, give historical examples, and present recent developments that demonstrate the potential of lysine targeting for future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQEe5pLdCmOT2mbXvSEQle5fW6udTcc2eY6MZJ5JPeiZLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbivVGmsQ%253D%253D&md5=bebb910023202e54213095ea353ea9a8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707630%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DLysine-targeting%2520covalent%2520inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2017%26volume%3D56%26spage%3D15200%26epage%3D15209%26doi%3D10.1002%2Fanie.201707630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazear, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Global profiling of lysine reactivity and ligandability in the human proteome</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1181</span>, <span class="refDoi">Â DOI: 10.1038/nchem.2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1038%2Fnchem.2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29168484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1181&author=S.+M.+Hackerauthor=K.+M.+Backusauthor=M.+R.+Lazearauthor=S.+Forliauthor=B.+E.+Correiaauthor=B.+F.+Cravatt&title=Global+profiling+of+lysine+reactivity+and+ligandability+in+the+human+proteome&doi=10.1038%2Fnchem.2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Global profiling of lysine reactivity and ligandability in the human proteome</span></div><div class="casAuthors">Hacker, Stephan M.; Backus, Keriann M.; Lazear, Michael R.; Forli, Stefano; Correia, Bruno E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nucleophilic amino acids make important contributions to protein function, including performing key roles in catalysis and serving as sites for post-translational modification.  Electrophilic groups that target amino-acid nucleophiles have been used to create covalent ligands and drugs, but have, so far, been mainly limited to cysteine and serine.  Here, the authors report a chem. proteomic platform for the global and quant. anal. of lysine residues in native biol. systems.  The authors have quantified, in total, >9000 lysines in human cell proteomes and have identified several hundred residues with heightened reactivity that are enriched at protein functional sites and can frequently be targeted by electrophilic small mols.  The authors also discovered lysine-reactive fragment electrophiles that inhibit enzymes by active site and allosteric mechanisms, as well as disrupt protein-protein interactions in transcriptional regulatory complexes, emphasizing the broad potential and diverse functional consequences of liganding lysine residues throughout the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKvUVcT5YRrVg90H21EOLACvtfcHk0lhNZVVLKVix1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN&md5=1dac8f079c1caffc8ae584291480fc61</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2826%26sid%3Dliteratum%253Aachs%26aulast%3DHacker%26aufirst%3DS.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DLazear%26aufirst%3DM.%2BR.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DGlobal%2520profiling%2520of%2520lysine%2520reactivity%2520and%2520ligandability%2520in%2520the%2520human%2520proteome%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1181%26doi%3D10.1038%2Fnchem.2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span> <span> </span><span class="NLM_article-title">Exploiting protein conformational change to optimize adenosine-derived inhibitors of HSP70</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4625</span>â <span class="NLM_lpage">4636</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b02001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b02001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4625-4636&author=M.+D.+Cheesemanauthor=I.+M.+Westwoodauthor=O.+Barbeauauthor=M.+Rowlandsauthor=S.+Dobsonauthor=A.+M.+Jonesauthor=F.+Jeganathanauthor=R.+Burkeauthor=N.+Kadiauthor=P.+Workmanauthor=I.+Collinsauthor=R.+L.+M.+van+Montfortauthor=K.+Jones&title=Exploiting+protein+conformational+change+to+optimize+adenosine-derived+inhibitors+of+HSP70&doi=10.1021%2Facs.jmedchem.5b02001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Protein Conformational Change to Optimize Adenosine-Derived Inhibitors of HSP70</span></div><div class="casAuthors">Cheeseman, Matthew D.; Westwood, Isaac M.; Barbeau, Olivier; Rowlands, Martin; Dobson, Sarah; Jones, Alan M.; Jeganathan, Fiona; Burke, Rosemary; Kadi, Nadia; Workman, Paul; Collins, Ian; van Montfort, Rob L. M.; Jones, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4625-4636</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HSP70 is a mol. chaperone and a key component of the heat-shock response.  Because of its proposed importance in oncol., this protein has become a popular target for drug discovery, efforts which have as yet brought little success.  This study demonstrates that adenosine-derived HSP70 inhibitors potentially bind to the protein with a novel mechanism of action, the stabilization by desolvation of an intramol. salt-bridge which induces a conformational change in the protein, leading to high affinity ligands.  We also demonstrate that through the application of this mechanism, adenosine-derived HSP70 inhibitors can be optimized in a rational manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-rro5uIFqZbVg90H21EOLACvtfcHk0lhNZVVLKVix1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7k%253D&md5=772967b1f315ca9c0e9bbbb115493754</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b02001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b02001%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DWestwood%26aufirst%3DI.%2BM.%26aulast%3DBarbeau%26aufirst%3DO.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DDobson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DA.%2BM.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DKadi%26aufirst%3DN.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DExploiting%2520protein%2520conformational%2520change%2520to%2520optimize%2520adenosine-derived%2520inhibitors%2520of%2520HSP70%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4625%26epage%3D4636%26doi%3D10.1021%2Facs.jmedchem.5b02001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yahya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span> <span> </span><span class="NLM_article-title">A fragment-based approach applied to a highly flexible target: Insights and challenges towards the inhibition of HSP70 isoforms</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">34701</span> <span class="refDoi">Â DOI: 10.1038/srep34701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1038%2Fsrep34701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=27708405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Cgt7bM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=A.+M.+Jonesauthor=I.+M.+Westwoodauthor=J.+D.+Osborneauthor=T.+P.+Matthewsauthor=M.+D.+Cheesemanauthor=M.+G.+Rowlandsauthor=F.+Jeganathanauthor=R.+Burkeauthor=D.+Leeauthor=N.+Kadiauthor=M.+Liuauthor=M.+Richardsauthor=C.+McAndrewauthor=N.+Yahyaauthor=S.+E.+Dobsonauthor=K.+Jonesauthor=P.+Workmanauthor=I.+Collinsauthor=R.+L.+M.+van+Montfort&title=A+fragment-based+approach+applied+to+a+highly+flexible+target%3A+Insights+and+challenges+towards+the+inhibition+of+HSP70+isoforms&doi=10.1038%2Fsrep34701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A fragment-based approach applied to a highly flexible target: Insights and challenges towards the inhibition of HSP70 isoforms</span></div><div class="casAuthors">Jones, Alan M.; Westwood, Isaac M.; Osborne, James D.; Matthews, Thomas P.; Cheeseman, Matthew D.; Rowlands, Martin G.; Jeganathan, Fiona; Burke, Rosemary; Lee, Diane; Kadi, Nadia; Liu, Manjuan; Richards, Meirion; McAndrew, Craig; Yahya, Norhakim; Dobson, Sarah E.; Jones, Keith; Workman, Paul; Collins, Ian; van Montfort, Rob L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34701</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Heat-shock protein 70s (HSP70s) are mol. chaperones implicated in many cancers and of significant interest as targets for novel cancer therapies.  Several HSP70 inhibitors have been reported, but because the majority have poor physicochem. properties and for many the exact mode of action is poorly understood, more detailed mechanistic and structural insight into ligand-binding to HSP70s is urgently needed.  Here, the authors describe the 1st comprehensive fragment-based inhibitor exploration of an HSP70 enzyme, which yielded an aminoquinazoline fragment that was elaborated to a novel ATP binding site ligand with different physicochem. properties to known adenosine-based HSP70 inhibitors.  Crystal structures of aminoquinazoline ligands bound to the different conformational states of the HSP70 nucleotide binding domain highlighted the challenges of a fragment-based approach when applied to this particular flexible enzyme class with an ATP-binding site that changes shape and size during its catalytic cycle.  In these studies, the authors showed that Ser-275 is a key residue in the selective binding of ATP.  Addnl., the structural data revealed a potential functional role for the ATP ribose moiety in priming the protein for the formation of the ATP-bound pre-hydrolysis complex by influencing the conformation of one of the phosphate binding loops.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOPetdvzOGKrVg90H21EOLACvtfcHk0lgRfPAlcFnFrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Cgt7bM&md5=6172a09b29ab4c7e284c33e7a463106d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsrep34701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep34701%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DA.%2BM.%26aulast%3DWestwood%26aufirst%3DI.%2BM.%26aulast%3DOsborne%26aufirst%3DJ.%2BD.%26aulast%3DMatthews%26aufirst%3DT.%2BP.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DRowlands%26aufirst%3DM.%2BG.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DKadi%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DYahya%26aufirst%3DN.%26aulast%3DDobson%26aufirst%3DS.%2BE.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26atitle%3DA%2520fragment-based%2520approach%2520applied%2520to%2520a%2520highly%2520flexible%2520target%253A%2520Insights%2520and%2520challenges%2520towards%2520the%2520inhibition%2520of%2520HSP70%2520isoforms%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep34701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Cera, E.</span></span> <span> </span><span class="NLM_article-title">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5894</span>â <span class="NLM_lpage">5902</span>, <span class="refDoi">Â DOI: 10.1021/bi3006913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3006913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5894-5902&author=A.+D.+Vogtauthor=E.+Di+Cera&title=Conformational+selection+or+induced+fit%3F+A+critical+appraisal+of+the+kinetic+mechanism&doi=10.1021%2Fbi3006913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span></div><div class="casAuthors">Vogt, Austin D.; Di Cera, Enrico</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5894-5902</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">For almost 5 decades, 2 competing mechanisms of ligand recognition, conformational selection and induced fit, have dominated the interpretation of ligand binding in biol. macromols.  When binding-dissocn. events are fast compared to conformational transitions, the rate of approach to equil., kobs, becomes diagnostic of conformational selection or induced fit based on whether it decreases or increases, resp., with the ligand concn. [L].  However, this simple conclusion based on the rapid equil. approxn. is not valid in general.  Here, the authors show that conformational selection is assocd. with a rich repertoire of kinetic properties, with kobs decreasing or increasing with [L] depending on the relative magnitude of the rate of ligand dissocn., koff, and the rate of conformational isomerization, kr.  The authors prove that, even for the simplest 2-step mechanism of ligand binding, a decrease in kobs with [L] is unequivocal evidence of conformational selection, but an increase in kobs with [L] is not unequivocal evidence of induced fit.  Ligand binding to glucokinase, thrombin, and its precursor, prethrombin-2, are used as relevant examples.  It is concluded that conformational selection as a mechanism for ligand binding to its target may be far more common than currently believed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdButaqMoNzLVg90H21EOLACvtfcHk0lgRfPAlcFnFrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D&md5=0f7c66027467c390967075c3be8e5254</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi3006913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3006913%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DA.%2BD.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DConformational%2520selection%2520or%2520induced%2520fit%253F%2520A%2520critical%2520appraisal%2520of%2520the%2520kinetic%2520mechanism%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5894%26epage%3D5902%26doi%3D10.1021%2Fbi3006913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirrenberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of HSP70 inhibitors</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e78443</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0078443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1371%2Fjournal.pone.0078443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=24265689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslylsLzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=R.+Schlechtauthor=S.+R.+Scholzauthor=H.+Dahmenauthor=A.+Wegenerauthor=C.+Sirrenbergauthor=D.+Musilauthor=J.+Bomkeauthor=H.-M.+Eggenweilerauthor=M.+P.+Mayerauthor=B.+Bukau&title=Functional+analysis+of+HSP70+inhibitors&doi=10.1371%2Fjournal.pone.0078443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Functional analysis of Hsp70 inhibitors</span></div><div class="casAuthors">Schlecht, Rainer; Scholz, Sebastian R.; Dahmen, Heike; Wegener, Ansgar; Sirrenberg, Christian; Musil, Djordje; Bomke, Joerg; Eggenweiler, Hans-Michael; Mayer, Matthias P.; Bukau, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e78443</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The mol. chaperones of the Hsp70 family have been recognized as targets for anti-cancer therapy.  Since several paralogs of Hsp70 proteins exist in cytosol, endoplasmic reticulum and mitochondria, we investigated which isoform needs to be down-regulated for reducing viability of cancer cells.  For two recently identified small mol. inhibitors, VER-155008 and 2-phenylethynesulfonamide (PES), which are proposed to target different sites in Hsp70s, we analyzed the mol. mode of action in vitro.  We found that for significant redn. of viability of cancer cells simultaneous knockdown of heat-inducible Hsp70 (HSPA1) and constitutive Hsc70 (HSPA8) is necessary.  The compd. VER-155008, which binds to the nucleotide binding site of Hsp70, arrests the nucleotide binding domain (NBD) in a half-open conformation and thereby acts as ATP-competitive inhibitor that prevents allosteric control between NBD and substrate binding domain (SBD).  Compd. PES interacts with the SBD of Hsp70 in an unspecific, detergent-like fashion, under the conditions tested.  None of the two inhibitors investigated was isoform-specific.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk_9mIXQAAX7Vg90H21EOLACvtfcHk0lgRfPAlcFnFrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslylsLzJ&md5=776f32214099c99744f6e32828b420ad</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0078443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0078443%26sid%3Dliteratum%253Aachs%26aulast%3DSchlecht%26aufirst%3DR.%26aulast%3DScholz%26aufirst%3DS.%2BR.%26aulast%3DDahmen%26aufirst%3DH.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DSirrenberg%26aufirst%3DC.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DEggenweiler%26aufirst%3DH.-M.%26aulast%3DMayer%26aufirst%3DM.%2BP.%26aulast%3DBukau%26aufirst%3DB.%26atitle%3DFunctional%2520analysis%2520of%2520HSP70%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0078443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xinyi, H.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization competition assay: The range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=12854996" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&author=H.+Xinyi&title=Fluorescence+polarization+competition+assay%3A+The+range+of+resolvable+inhibitor+potency+is+limited+by+the+affinity+of+the+fluorescent+ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DXinyi%26aufirst%3DH.%26atitle%3DFluorescence%2520polarization%2520competition%2520assay%253A%2520The%2520range%2520of%2520resolvable%2520inhibitor%2520potency%2520is%2520limited%2520by%2520the%2520affinity%2520of%2520the%2520fluorescent%2520ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="note"><p class="first last"><i>pK<sub>i</sub></i> = âlogâ¯<i>K<sub>i</sub></i> (<i>M</i>).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span> <span> </span><span class="NLM_article-title">Advances in the development of shape similarity methods and their application in drug discovery</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">315</span>, <span class="refDoi">Â DOI: 10.3389/fchem.2018.00315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.3389%2Ffchem.2018.00315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=30090808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaitrvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=315&author=A.+Kumarauthor=K.+Y.+J.+Zhang&title=Advances+in+the+development+of+shape+similarity+methods+and+their+application+in+drug+discovery&doi=10.3389%2Ffchem.2018.00315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of shape similarity methods and their application in drug discovery</span></div><div class="casAuthors">Kumar, Ashutosh; Zhang, Kam Y. J.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">315/1-315/21</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Mol. similarity is a key concept in drug discovery.  It is based on the assumption that structurally similar mols. frequently have similar properties.  Assessment of similarity between small mols. has been highly effective in the discovery and development of various drugs.  Esp., two-dimensional (2D) similarity approaches have been quite popular due to their simplicity, accuracy and efficiency.  Recently, the focus has been shifted toward the development of methods involving the representation and comparison of three-dimensional (3D) conformation of small mols.  Among the 3D similarity methods, evaluation of shape similarity is now gaining attention for its application not only in virtual screening but also in mol. target prediction, drug repurposing and scaffold hopping.  A wide range of methods have been developed to describe mol. shape and to det. the shape similarity between small mols.  The most widely used methods include atom distance-based methods, surface-based approaches such as spherical harmonics and 3D Zernike descriptors, atom-centered Gaussian overlay based representations.  Several of these methods demonstrated excellent virtual screening performance not only retrospectively but also prospectively.  In addn. to methods assessing the similarity between small mols., shape similarity approaches have been developed to compare shapes of protein structures and binding pockets.  Addnl., shape comparisons between at. models and 3D d. maps allowed the fitting of at. models into cryo-electron microscopy maps.  This review aims to summarize the methodol. advances in shape similarity assessment highlighting advantages, disadvantages and their application in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTIDZv0Wh2Q7Vg90H21EOLACvtfcHk0liRB3PrCWWn3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaitrvF&md5=9c7690b5caf58f26be5920318779deba</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2018.00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2018.00315%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520shape%2520similarity%2520methods%2520and%2520their%2520application%2520in%2520drug%2520discovery%26jtitle%3DFront.%2520Chem.%26date%3D2018%26volume%3D6%26spage%3D315%26doi%3D10.3389%2Ffchem.2018.00315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span> </span><span class="NLM_article-title">The Cambridge Structural Database (CSD)</span>. <a href="https://www.ccdc.cam.ac.uk/solutions/csd-system/components/csd/" class="extLink">https://www.ccdc.cam.ac.uk/solutions/csd-system/components/csd/</a> (accessed July, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+Cambridge+Structural+Database+%28CSD%29.+https%3A%2F%2Fwww.ccdc.cam.ac.uk%2Fsolutions%2Fcsd-system%2Fcomponents%2Fcsd%2F+%28accessed+July%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520Cambridge%2520Structural%2520Database%2520%2528CSD%2529%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span> <span> </span><span class="NLM_article-title">Characterising covalent warhead reactivity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2066</span>â <span class="NLM_lpage">2074</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2019.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.bmc.2019.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=30975501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntVGgsbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2066-2074&author=J.+S.+Martinauthor=C.+J.+MacKenzieauthor=D.+Fletcherauthor=I.+H.+Gilbert&title=Characterising+covalent+warhead+reactivity&doi=10.1016%2Fj.bmc.2019.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Characterising covalent warhead reactivity</span></div><div class="casAuthors">Martin, James S.; MacKenzie, Claire J.; Fletcher, Daniel; Gilbert, Ian H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2066-2074</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Many drugs currently used are covalent inhibitors and irreversibly inhibit their targets.  Most of these were discovered through serendipity.  Covalent inhibitions can have many advantages from a pharmacokinetic perspective.  However, until recently most organizations have shied away from covalent compd. design due to fears of non-specific inhibition of off-target proteins leading to toxicity risks.  However, there has been a renewed interest in covalent modifiers as potential drugs, as it possible to get highly selective compds.  It is therefore important to know how reactive a warhead is and to be able to select the least reactive warhead possible to avoid toxicity.  A robust NMR based assay was developed and used to measure the reactivity of a variety of covalent warheads against serine and cysteine - the two most common targets for covalent drugs.  A selection of these warheads also had their reactivity measured against threonine, tyrosine, lysine, histidine and arginine to better understand our ability to target non-traditional residues.  The reactivity was also measured at various pHs to assess what effect the environment in the active site would have on these reactions.  The reactivity of a covalent modifier was found to be very dependent on the amino acid residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVaz2cnHGpo7Vg90H21EOLACvtfcHk0liRB3PrCWWn3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntVGgsbo%253D&md5=487334d8d0a8b03d0a25d6d87c1b4329</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DJ.%2BS.%26aulast%3DMacKenzie%26aufirst%3DC.%2BJ.%26aulast%3DFletcher%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DCharacterising%2520covalent%2520warhead%2520reactivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2066%26epage%3D2074%26doi%3D10.1016%2Fj.bmc.2019.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Convery, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span> <span> </span><span class="NLM_article-title">Selectively targeting the kinome-conserved lysine of PI3KÎ´ as a general approach to covalent kinase inhibition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">932</span>â <span class="NLM_lpage">939</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b08979</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08979" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=932-939&author=S.+E.+Daltonauthor=L.+Dittusauthor=D.+A.+Thomasauthor=M.+A.+Converyauthor=J.+Nunesauthor=J.+T.+Bushauthor=J.+P.+Evansauthor=T.+Wernerauthor=M.+Bantscheffauthor=J.+A.+Murphyauthor=S.+Campos&title=Selectively+targeting+the+kinome-conserved+lysine+of+PI3K%CE%B4+as+a+general+approach+to+covalent+kinase+inhibition&doi=10.1021%2Fjacs.7b08979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Selectively Targeting the Kinome-Conserved Lysine of PI3KÎ´ as a General Approach to Covalent Kinase Inhibition</span></div><div class="casAuthors">Dalton, Samuel E.; Dittus, Lars; Thomas, Daniel A.; Convery, Maire A.; Nunes, Joao; Bush, Jacob T.; Evans, John P.; Werner, Thilo; Bantscheff, Marcus; Murphy, John A.; Campos, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">932-939</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective covalent inhibition of kinases by targeting poorly conserved cysteines has proven highly fruitful to date in the development of chem. probes and approved drugs.  However, this approach is limited to â¼200 kinases possessing such a cysteine near the ATP-binding pocket.  Herein, we report a novel approach to achieve selective, irreversible kinase inhibition, by targeting the conserved catalytic lysine residue.  We have illustrated our approach by developing selective, covalent PI3KÎ´ inhibitors that exhibit nanomolar potency in cellular assays, and a duration of action >48 h in CD4+ T cells.  Despite conservation of the lysine residue throughout the kinome, the lead compd. shows high levels of selectivity over a selection of lipid and protein kinases in biochem. assays, as well as covalent binding to very few off-target proteins in live-cell proteomic studies.  We anticipate this approach could offer a general strategy, as an alternative to targeting non-conserved cysteines, for the development of selective covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGvT7h2D9zf7Vg90H21EOLACvtfcHk0liRB3PrCWWn3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK&md5=90bb399303a942fb87f1b9096eb8bdc6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08979%26sid%3Dliteratum%253Aachs%26aulast%3DDalton%26aufirst%3DS.%2BE.%26aulast%3DDittus%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DD.%2BA.%26aulast%3DConvery%26aufirst%3DM.%2BA.%26aulast%3DNunes%26aufirst%3DJ.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DJ.%2BA.%26aulast%3DCampos%26aufirst%3DS.%26atitle%3DSelectively%2520targeting%2520the%2520kinome-conserved%2520lysine%2520of%2520PI3K%25CE%25B4%2520as%2520a%2520general%2520approach%2520to%2520covalent%2520kinase%2520inhibition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D932%26epage%3D939%26doi%3D10.1021%2Fjacs.7b08979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Light, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minasov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duban, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, W. F.</span></span> <span> </span><span class="NLM_article-title">Adherence to Burgi-Dunitz stereochemical principles requires significant structural rearrangements in Schiff-base formation: Insights from transaldolase complexes</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">544</span>â <span class="NLM_lpage">552</span>, <span class="refDoi">Â DOI: 10.1107/S1399004713030666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1107%2FS1399004713030666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=24531488" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2014&pages=544-552&author=S.+H.+Lightauthor=G.+Minasovauthor=M.-E.+Dubanauthor=W.+F.+Anderson&title=Adherence+to+Burgi-Dunitz+stereochemical+principles+requires+significant+structural+rearrangements+in+Schiff-base+formation%3A+Insights+from+transaldolase+complexes&doi=10.1107%2FS1399004713030666"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS1399004713030666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1399004713030666%26sid%3Dliteratum%253Aachs%26aulast%3DLight%26aufirst%3DS.%2BH.%26aulast%3DMinasov%26aufirst%3DG.%26aulast%3DDuban%26aufirst%3DM.-E.%26aulast%3DAnderson%26aufirst%3DW.%2BF.%26atitle%3DAdherence%2520to%2520Burgi-Dunitz%2520stereochemical%2520principles%2520requires%2520significant%2520structural%2520rearrangements%2520in%2520Schiff-base%2520formation%253A%2520Insights%2520from%2520transaldolase%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2014%26volume%3D70%26spage%3D544%26epage%3D552%26doi%3D10.1107%2FS1399004713030666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devaraj, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, C. L.</span></span> <span> </span><span class="NLM_article-title">Approach control. Stereoelectronic origin of geometric constraints on n-to-s and n-to-o acyl shifts in peptides</span>. <i>Chemical Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1789</span>â <span class="NLM_lpage">1794</span>, <span class="refDoi">Â DOI: 10.1039/C7SC04046F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC7SC04046F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29675223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFOqsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1789-1794&author=N.+K.+Devarajauthor=C.+L.+Perrin&title=Approach+control.+Stereoelectronic+origin+of+geometric+constraints+on+n-to-s+and+n-to-o+acyl+shifts+in+peptides&doi=10.1039%2FC7SC04046F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Approach control. Stereoelectronic origin of geometric constraints on N-to-S and N-to-O acyl shifts in peptides</span></div><div class="casAuthors">Devaraj, Neal K.; Perrin, Charles L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1789-1794</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Intramol. N-to-S or N-to-O acyl shifts in peptides are of fundamental and practical importance, as they constitute the first step in protein splicing and can be used for the synthesis of thioester-modified peptides required for native chem. ligation.  It has been stated that the nucleophile must be positioned anti to the carbonyl oxygen, as in a cis amide.  Despite the importance of such reactions, an understanding of this geometric restriction remains obscure.  Here we argue that the empirical requirement for positioning the nucleophile is a stereoelectronic effect arising from the ease of approach of the nucleophile to a carbonyl group, not ground-state destabilization.  DFT calcns. on model amides support our explanation and indicate a significant decrease in both the transition-state energy and the activation energy for a cis amide.  However, the approach of the nucleophile must be anti not only to the carbonyl oxygen but also to the nitrogen.  The direction of approach is expressed by a new, modified B.ovrddot.urgi-Dunitz angle.  Our data shed light on the mechanisms of acyl shifts in peptides, and they explain why a cis peptide might be required for protein splicing.  The further implications for acyl shifts in homoserine and homocysteine peptides and for aldol condensations are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFaMEQrfJxDLVg90H21EOLACvtfcHk0lidbavV-xmzxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFOqsg%253D%253D&md5=8de52bee047d5f195307b2211c9fa595</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FC7SC04046F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC04046F%26sid%3Dliteratum%253Aachs%26aulast%3DDevaraj%26aufirst%3DN.%2BK.%26aulast%3DPerrin%26aufirst%3DC.%2BL.%26atitle%3DApproach%2520control.%2520Stereoelectronic%2520origin%2520of%2520geometric%2520constraints%2520on%2520n-to-s%2520and%2520n-to-o%2520acyl%2520shifts%2520in%2520peptides%26jtitle%3DChemical%2520Science%26date%3D2018%26volume%3D9%26spage%3D1789%26epage%3D1794%26doi%3D10.1039%2FC7SC04046F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdul
Fattah, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saeed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albericio, F.</span></span> <span> </span><span class="NLM_article-title">Recent advances towards sulfur (vi) fluoride exchange (SUFEX) click chemistry</span>. <i>J. Fluorine Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>213</i></span>,  <span class="NLM_fpage">87</span>â <span class="NLM_lpage">112</span>, <span class="refDoi">Â DOI: 10.1016/j.jfluchem.2018.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.jfluchem.2018.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSrtb7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=213&publication_year=2018&pages=87-112&author=T.+Abdul%0AFattahauthor=A.+Saeedauthor=F.+Albericio&title=Recent+advances+towards+sulfur+%28vi%29+fluoride+exchange+%28SUFEX%29+click+chemistry&doi=10.1016%2Fj.jfluchem.2018.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances towards sulfur (VI) fluoride exchange (SuFEx) click chemistry</span></div><div class="casAuthors">Abdul Fattah, Tanzeela; Saeed, Aamer; Albericio, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Fluorine Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">213</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87-112</span>CODEN:
                <span class="NLM_cas:coden">JFLCAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-1139</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This review comprehensively focused on the sole reactivity of SuFEx click reaction and its role in the building of novel sterically hindered amides, fluorosulfates, diverse compds. from SOF4 hub, high-mol. wt. polymer materials (polysulfates and polysulfonates) and surface coatings.  The SuFEx part in post-polymn. reactions, late-stage drug functionalization (LSF) and oil/water sepns. was also highlighted.  Finally, the potential application of the SuFEx for bioconjugation is also analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRiA6hdvWJmbVg90H21EOLACvtfcHk0lidbavV-xmzxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSrtb7J&md5=9b151ab663fcdf5227068debf1c2d8e0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jfluchem.2018.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jfluchem.2018.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DAbdul%2BFattah%26aufirst%3DT.%26aulast%3DSaeed%26aufirst%3DA.%26aulast%3DAlbericio%26aufirst%3DF.%26atitle%3DRecent%2520advances%2520towards%2520sulfur%2520%2528vi%2529%2520fluoride%2520exchange%2520%2528SUFEX%2529%2520click%2520chemistry%26jtitle%3DJ.%2520Fluorine%2520Chem.%26date%3D2018%26volume%3D213%26spage%3D87%26epage%3D112%26doi%3D10.1016%2Fj.jfluchem.2018.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Sulfonyl fluorides as privileged warheads in chemical biology</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2650</span>â <span class="NLM_lpage">2659</span>, <span class="refDoi">Â DOI: 10.1039/C5SC00408J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC5SC00408J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28706662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVamurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2650-2659&author=A.+Narayananauthor=L.+H.+Jones&title=Sulfonyl+fluorides+as+privileged+warheads+in+chemical+biology&doi=10.1039%2FC5SC00408J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfonyl fluorides as privileged warheads in chemical biology</span></div><div class="casAuthors">Narayanan, Arjun; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2650-2659</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Sulfonyl fluoride electrophiles have found significant utility as reactive probes in chem. biol. and mol. pharmacol.  As warheads they possess the right balance of biocompatibility (including aq. stability) and protein reactivity.  Their functionality is privileged in this regard as they are known to modify not only reactive serines (resulting in their common use as protease inhibitors), but also context-specific threonine, lysine, tyrosine, cysteine and histidine residues.  This review describes the application of sulfonyl fluoride probes across various areas of research and explores new approaches that could further enhance the chem. biol. toolkit.  We believe that sulfonyl fluoride probes will find greater utility in areas such as covalent enzyme inhibition, target identification and validation, and the mapping of enzyme binding sites, substrates and protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAuadIMxOl2bVg90H21EOLACvtfcHk0lidbavV-xmzxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVamurc%253D&md5=0009feb4068e912b4784fb8411cdb702</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC5SC00408J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC00408J%26sid%3Dliteratum%253Aachs%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DSulfonyl%2520fluorides%2520as%2520privileged%2520warheads%2520in%2520chemical%2520biology%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D2650%26epage%3D2659%26doi%3D10.1039%2FC5SC00408J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debreczeni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tentarelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aquila, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimster, N. P.</span></span> <span> </span><span class="NLM_article-title">A study of the reactivity of S(VI)âF containing warheads with nucleophilic amino-acid side chains under physiological conditions</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">9685</span>â <span class="NLM_lpage">9695</span>, <span class="refDoi">Â DOI: 10.1039/C7OB02028G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC7OB02028G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29119993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSgsbnI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=9685-9695&author=H.+Mukherjeeauthor=J.+Debreczeniauthor=J.+Breedauthor=S.+Tentarelliauthor=B.+Aquilaauthor=J.+E.+Dowlingauthor=A.+Whittyauthor=N.+P.+Grimster&title=A+study+of+the+reactivity+of+S%28VI%29%E2%80%93F+containing+warheads+with+nucleophilic+amino-acid+side+chains+under+physiological+conditions&doi=10.1039%2FC7OB02028G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A study of the reactivity of S(VI)-F containing warheads with nucleophilic amino-acid side chains under physiological conditions</span></div><div class="casAuthors">Mukherjee, H.; Debreczeni, J.; Breed, J.; Tentarelli, S.; Aquila, B.; Dowling, J. E.; Whitty, A.; Grimster, N. P.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">9685-9695</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Sulfonyl fluorides (SFs) have recently emerged as a promising warhead for the targeted covalent modification of proteins.  Despite numerous examples of the successful deployment of SFs as covalent probe compds., a detailed exploration of the factors influencing the stability and reactivity of SFs has not yet appeared.  In this work we present an extensive study on the influence of steric and electronic factors on the reactivity and stability of the SF and related SVI-F groups.  While SFs react rapidly with N-acetylcysteine, the resulting adducts were found to be unstable, rendering SFs inappropriate for the durable covalent inhibition of cysteine residues.  In contrast, SFs afforded stable adducts with both N-acetyltyrosine and N-acetyllysine; furthermore, we show that the reactivity of arylsulfonyl fluorides towards these nucleophilic amino acids can be predictably modulated by adjusting the electronic properties of the warhead.  These trends were largely conserved when the covalent reaction occurred within a protein binding pocket.  We have also obtained a crystal structure depicting covalent modification of the catalytic lysine of a tyrosine kinase (FGFR1) by the ATP analog 5'-O-3-((fluorosulfonyl)benzoyl)adenosine (m-FSBA).  Highly reactive warheads were demonstrated to be unstable with respect to hydrolysis in buffered aq. solns., indicating that warhead reactivity must be carefully tuned to provide optimal rates of protein modification.  Our results demonstrate that the reactivity of SFs complements that of more commonly studied acrylamides, and we hope that this work spurs the rational design of novel SF-contg. covalent probe compds. and inhibitors, particularly in cases where a suitably positioned cysteine residue is not present.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIyTBiXHscB7Vg90H21EOLACvtfcHk0ljQmxCqTaNylA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSgsbnI&md5=cf00897a4b0b184e2d68df498ec8cbb7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC7OB02028G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB02028G%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DH.%26aulast%3DDebreczeni%26aufirst%3DJ.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DTentarelli%26aufirst%3DS.%26aulast%3DAquila%26aufirst%3DB.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DGrimster%26aufirst%3DN.%2BP.%26atitle%3DA%2520study%2520of%2520the%2520reactivity%2520of%2520S%2528VI%2529%25E2%2580%2593F%2520containing%2520warheads%2520with%2520nucleophilic%2520amino-acid%2520side%2520chains%2520under%2520physiological%2520conditions%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D9685%26epage%3D9695%26doi%3D10.1039%2FC7OB02028G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciuffarin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senatore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isola, M.</span></span> <span> </span><span class="NLM_article-title">Nucleophilic substitution at four-co-ordinate sulphur. Mobility of the leaving group</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_fpage">468</span>â <span class="NLM_lpage">471</span>, <span class="refDoi">Â DOI: 10.1039/p29720000468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2Fp29720000468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADyaE38XpvVyrtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1972&pages=468-471&author=E.+Ciuffarinauthor=L.+Senatoreauthor=M.+Isola&title=Nucleophilic+substitution+at+four-co-ordinate+sulphur.+Mobility+of+the+leaving+group&doi=10.1039%2Fp29720000468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleophilic substitution at four-coordinate sulfur.  Mobility of the leaving group</span></div><div class="casAuthors">Ciuffarin, Ennio; Senatore, Lucio; Isola, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 2:  Physical Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">468-71</span>CODEN:
                <span class="NLM_cas:coden">JCPKBH</span>;
        ISSN:<span class="NLM_cas:issn">0300-9580</span>.
    </div><div class="casAbstract">The kinetics of the reactions of PhNH2, BuNH2, and OH- with PhSO2X (X = I, Br, Cl, F) were detd. and the rate consts. and activation parameters of the displacement of X at 25Â° calcd.  The relative leaving group mobilities of I, Br, and Cl with each nucleophile were the same but the relative mobility of F varied greatly.  The mechanism may involve nucleophile-S bond formation (X = I, Br, Cl) as the rate-detg. step; but S-F bond cleavage greatly influenced the rate of reaction of PhSO2F with PhNH2 or BuNH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1ZZFqEY_XmrVg90H21EOLACvtfcHk0ljQmxCqTaNylA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE38XpvVyrtw%253D%253D&md5=a1753e416b258cea78f6850fda2d2bf7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2Fp29720000468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29720000468%26sid%3Dliteratum%253Aachs%26aulast%3DCiuffarin%26aufirst%3DE.%26aulast%3DSenatore%26aufirst%3DL.%26aulast%3DIsola%26aufirst%3DM.%26atitle%3DNucleophilic%2520substitution%2520at%2520four-co-ordinate%2520sulphur.%2520Mobility%2520of%2520the%2520leaving%2520group%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1972%26spage%3D468%26epage%3D471%26doi%3D10.1039%2Fp29720000468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolanos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almaden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span> <span> </span><span class="NLM_article-title">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+EGFR+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact â¤ 2.1 Ã 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0ljQmxCqTaNylA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520EGFR%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D173%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="note"><p class="first last">Arithmetic mean Â± SEM measured from 3 independent biological repeats.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Targeting secondary protein complexes in drug discovery: Studying the druggability and chemical biology of the HSP70/BAG1 complex</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5167</span>â <span class="NLM_lpage">5170</span>, <span class="refDoi">Â DOI: 10.1039/C7CC01376K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC7CC01376K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28439591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=5167-5170&author=L.+E.+Evansauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Targeting+secondary+protein+complexes+in+drug+discovery%3A+Studying+the+druggability+and+chemical+biology+of+the+HSP70%2FBAG1+complex&doi=10.1039%2FC7CC01376K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting secondary protein complexes in drug discovery: studying the druggability and chemical biology of the HSP70/BAG1 complex</span></div><div class="casAuthors">Evans, Lindsay E.; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5167-5170</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Proteins typically carry out their biol. functions as multi-protein complexes, which can significantly affect the affinity of small-mol. inhibitors.  HSP70 is an important target in oncol., so to study its chem. biol. and the drug discovery potential of the HSP70/BAG1 complex, we designed a high-affinity non-nucleotide fluorescence polarization probe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwTl7NEh65w7Vg90H21EOLACvtfcHk0liuAQHeiqXp-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D&md5=f72aa2f8aba028100f2eb1fd45bf336e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2FC7CC01376K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC01376K%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DTargeting%2520secondary%2520protein%2520complexes%2520in%2520drug%2520discovery%253A%2520Studying%2520the%2520druggability%2520and%2520chemical%2520biology%2520of%2520the%2520HSP70%252FBAG1%2520complex%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D5167%26epage%3D5170%26doi%3D10.1039%2FC7CC01376K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macias, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgognoni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokurno, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matassova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surgenor, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A. J.</span></span> <span> </span><span class="NLM_article-title">Adenosine-derived inhibitors of 78 kda glucose regulated protein (GRP78) ATPase: Insights into isoform selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4034</span>â <span class="NLM_lpage">4041</span>, <span class="refDoi">Â DOI: 10.1021/jm101625x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101625x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlWrtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4034-4041&author=A.+T.+Maciasauthor=D.+S.+Williamsonauthor=N.+Allenauthor=J.+Borgognoniauthor=A.+Clayauthor=Z.+Danielsauthor=P.+Dokurnoauthor=M.+J.+Drysdaleauthor=G.+L.+Francisauthor=C.+J.+Grahamauthor=R.+Howesauthor=N.+Matassovaauthor=J.+B.+Murrayauthor=R.+Parsonsauthor=T.+Shawauthor=A.+E.+Surgenorauthor=L.+Terryauthor=Y.+Wangauthor=M.+Woodauthor=A.+J.+Massey&title=Adenosine-derived+inhibitors+of+78+kda+glucose+regulated+protein+%28GRP78%29+ATPase%3A+Insights+into+isoform+selectivity&doi=10.1021%2Fjm101625x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine-Derived Inhibitors of 78 kDa Glucose Regulated Protein (Grp78) ATPase: Insights into Isoform Selectivity</span></div><div class="casAuthors">Macias, Alba T.; Williamson, Douglas S.; Allen, Nicola; Borgognoni, Jenifer; Clay, Alexandra; Daniels, Zoe; Dokurno, Pawel; Drysdale, Martin J.; Francis, Geraint L.; Graham, Christopher J.; Howes, Rob; Matassova, Natalia; Murray, James B.; Parsons, Rachel; Shaw, Terry; Surgenor, Allan E.; Terry, Lindsey; Wang, Yikang; Wood, Mike; Massey, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4034-4041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">78 KDa glucose-regulated protein (Grp78) is a heat shock protein (HSP) involved in protein folding that plays a role in cancer cell proliferation.  Binding of adenosine-derived inhibitors to Grp78 was characterized by surface plasmon resonance and isothermal titrn. calorimetry.  The most potent compds. were 13 (VER-155008) with KD = 80 nM and 14 with KD = 60 nM.  X-ray crystal structures of Grp78 bound to ATP, ADPnP, and adenosine deriv. 10 revealed differences in the binding site between Grp78 and homologous proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIYGKX4tuuUrVg90H21EOLACvtfcHk0liuAQHeiqXp-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlWrtr0%253D&md5=b382510c6bd1e382c9a42e81e0ed3362</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm101625x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101625x%26sid%3Dliteratum%253Aachs%26aulast%3DMacias%26aufirst%3DA.%2BT.%26aulast%3DWilliamson%26aufirst%3DD.%2BS.%26aulast%3DAllen%26aufirst%3DN.%26aulast%3DBorgognoni%26aufirst%3DJ.%26aulast%3DClay%26aufirst%3DA.%26aulast%3DDaniels%26aufirst%3DZ.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFrancis%26aufirst%3DG.%2BL.%26aulast%3DGraham%26aufirst%3DC.%2BJ.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DShaw%26aufirst%3DT.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26aulast%3DTerry%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DMassey%26aufirst%3DA.%2BJ.%26atitle%3DAdenosine-derived%2520inhibitors%2520of%252078%2520kda%2520glucose%2520regulated%2520protein%2520%2528GRP78%2529%2520ATPase%253A%2520Insights%2520into%2520isoform%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4034%26epage%3D4041%26doi%3D10.1021%2Fjm101625x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':[],'ref11':[],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':[],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':[],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Shuheng Xu, <span class="NLM_string-name hlFld-ContribAuthor">Sunliang Cui</span>. </span><span class="cited-content_cbyCitation_article-title">SuFExable Isocyanides for Ugi Reaction: Synthesis of Sulfonyl Fluoro Peptides. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2021,</strong> <em>23 </em>
                                    (13)
                                     , 5197-5202. <a href="https://doi.org/10.1021/acs.orglett.1c01734" title="DOI URL">https://doi.org/10.1021/acs.orglett.1c01734</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.1c01734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.1c01734%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSuFExable%252BIsocyanides%252Bfor%252BUgi%252BReaction%25253A%252BSynthesis%252Bof%252BSulfonyl%252BFluoro%252BPeptides%26aulast%3DXu%26aufirst%3DShuheng%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D23052021%26date%3D22062021%26volume%3D23%26issue%3D13%26spage%3D5197%26epage%3D5202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Madita Wolter, Dario Valenti, Peter J. Cossar, Stanimira Hristeva, Laura M. Levy, Thorsten Genski, Torsten Hoffmann, Luc Brunsveld, Dimitrios Tzalis, <span class="NLM_string-name hlFld-ContribAuthor">Christian Ottmann</span>. </span><span class="cited-content_cbyCitation_article-title">An Exploration of Chemical Properties Required for Cooperative Stabilization of the 14-3-3 Interaction with NF-ÎºBâUtilizing a Reversible Covalent Tethering Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8423-8436. <a href="https://doi.org/10.1021/acs.jmedchem.1c00401" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00401</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00401%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAn%252BExploration%252Bof%252BChemical%252BProperties%252BRequired%252Bfor%252BCooperative%252BStabilization%252Bof%252Bthe%252B14-3-3%252BInteraction%252Bwith%252BNF-%2525CE%2525BAB%2525E2%252580%252594Utilizing%252Ba%252BReversible%252BCovalent%252BTethering%252BApproach%26aulast%3DWolter%26aufirst%3DMadita%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D05032021%26date%3D02062021%26volume%3D64%26issue%3D12%26spage%3D8423%26epage%3D8436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrew J. Ambrose, <span class="NLM_string-name hlFld-ContribAuthor">Eli Chapman</span>. </span><span class="cited-content_cbyCitation_article-title">Function, Therapeutic Potential, and Inhibition of Hsp70 Chaperones. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7060-7082. <a href="https://doi.org/10.1021/acs.jmedchem.0c02091" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02091%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFunction%25252C%252BTherapeutic%252BPotential%25252C%252Band%252BInhibition%252Bof%252BHsp70%252BChaperones%26aulast%3DAmbrose%26aufirst%3DAndrew%2BJ.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D02122020%26date%3D19052021%26volume%3D64%26issue%3D11%26spage%3D7060%26epage%3D7082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter J. Cossar, Madita Wolter, Lars van Dijck, Dario Valenti, Laura M. Levy, Christian Ottmann, <span class="NLM_string-name hlFld-ContribAuthor">Luc Brunsveld</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Covalent Imine-Tethering for Selective Stabilization of 14-3-3 Hub Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (22)
                                     , 8454-8464. <a href="https://doi.org/10.1021/jacs.1c03035" title="DOI URL">https://doi.org/10.1021/jacs.1c03035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c03035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c03035%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DReversible%252BCovalent%252BImine-Tethering%252Bfor%252BSelective%252BStabilization%252Bof%252B14-3-3%252BHub%252BProtein%252BInteractions%26aulast%3DCossar%26aufirst%3DPeter%2BJ.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D21032021%26date%3D28052021%26volume%3D143%26issue%3D22%26spage%3D8454%26epage%3D8464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Luca Gambini, Parima Udompholkul, Carlo Baggio, Aruljothi Muralidharan, Nikola KenjiÄ, Zahra Assar, J. Jefferson P. Perry, <span class="NLM_string-name hlFld-ContribAuthor">Maurizio Pellecchia</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Structural Characterization of Lysine Covalent BH3 Peptides Targeting Mcl-1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4903-4912. <a href="https://doi.org/10.1021/acs.jmedchem.1c00005" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00005%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BStructural%252BCharacterization%252Bof%252BLysine%252BCovalent%252BBH3%252BPeptides%252BTargeting%252BMcl-1%26aulast%3DGambini%26aufirst%3DLuca%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D03012021%26date%3D02042021%26volume%3D64%26issue%3D8%26spage%3D4903%26epage%3D4912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue  Liu</span>, <span class="hlFld-ContribAuthor ">Shumin  Lv</span>, <span class="hlFld-ContribAuthor ">Lijie  Peng</span>, <span class="hlFld-ContribAuthor ">Chengliang  Xie</span>, <span class="hlFld-ContribAuthor ">Liqian  Gao</span>, <span class="hlFld-ContribAuthor ">Hongyan  Sun</span>, <span class="hlFld-ContribAuthor ">Ligen  Lin</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>, <span class="hlFld-ContribAuthor ">Zhengqiu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Development and application of novel electrophilic warheads in target identification and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2021,</strong> <em>190 </em>, 114636. <a href="https://doi.org/10.1016/j.bcp.2021.114636" title="DOI URL">https://doi.org/10.1016/j.bcp.2021.114636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2021.114636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2021.114636%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DDevelopment%252Band%252Bapplication%252Bof%252Bnovel%252Belectrophilic%252Bwarheads%252Bin%252Btarget%252Bidentification%252Band%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DYue%26date%3D2021%26volume%3D190%26spage%3D114636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimen  Aljoundi</span>, <span class="hlFld-ContribAuthor ">Ahmed  El Rashedy</span>, <span class="hlFld-ContribAuthor ">Mahmoud E. S.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of irreversible inhibition targeting HSP72 protein: the resurgence of covalent drug developments. </span><span class="cited-content_cbyCitation_journal-name">Molecular Simulation</span><span> <strong>2021,</strong> <em>61 </em>, 1-11. <a href="https://doi.org/10.1080/08927022.2021.1949457" title="DOI URL">https://doi.org/10.1080/08927022.2021.1949457</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08927022.2021.1949457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08927022.2021.1949457%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Simulation%26atitle%3DComparison%252Bof%252Birreversible%252Binhibition%252Btargeting%252BHSP72%252Bprotein%25253A%252Bthe%252Bresurgence%252Bof%252Bcovalent%252Bdrug%252Bdevelopments%26aulast%3DAljoundi%26aufirst%3DAimen%26date%3D2021%26date%3D2021%26volume%3D61%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangyan  Zhou</span>, <span class="hlFld-ContribAuthor ">Li  He</span>, <span class="hlFld-ContribAuthor ">Kathy H.  Li</span>, <span class="hlFld-ContribAuthor ">CÃ¡ssio C. S.  Pedroso</span>, <span class="hlFld-ContribAuthor ">Miriam  Gochin</span>. </span><span class="cited-content_cbyCitation_article-title">A targeted covalent small molecule inhibitor of HIV-1 fusion. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2021,</strong> <em>57 </em>
                                    (37)
                                     , 4528-4531. <a href="https://doi.org/10.1039/D1CC01013A" title="DOI URL">https://doi.org/10.1039/D1CC01013A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CC01013A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CC01013A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DA%252Btargeted%252Bcovalent%252Bsmall%252Bmolecule%252Binhibitor%252Bof%252BHIV-1%252Bfusion%26aulast%3DZhou%26aufirst%3DGuangyan%26date%3D2021%26date%3D2021%26volume%3D57%26issue%3D37%26spage%3D4528%26epage%3D4531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas A.  Baillie</span>. </span><span class="cited-content_cbyCitation_article-title">Approaches to mitigate the risk of serious adverse reactions in covalent drug design. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>16 </em>
                                    (3)
                                     , 275-287. <a href="https://doi.org/10.1080/17460441.2021.1832079" title="DOI URL">https://doi.org/10.1080/17460441.2021.1832079</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1832079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1832079%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DApproaches%252Bto%252Bmitigate%252Bthe%252Brisk%252Bof%252Bserious%252Badverse%252Breactions%252Bin%252Bcovalent%252Bdrug%252Bdesign%26aulast%3DBaillie%26aufirst%3DThomas%2BA.%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D3%26spage%3D275%26epage%3D287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernest  Awoonor-Williams</span>, <span class="hlFld-ContribAuthor ">Jacob  Kennedy</span>, <span class="hlFld-ContribAuthor ">Christopher N.  Rowley</span>. </span><span class="cited-content_cbyCitation_article-title">Measuring and predicting warhead and residue reactivity. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 203-227. <a href="https://doi.org/10.1016/bs.armc.2020.09.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.09.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.09.001%26sid%3Dliteratum%253Aachs%26atitle%3DMeasuring%252Band%252Bpredicting%252Bwarhead%252Band%252Bresidue%252Breactivity%26aulast%3DAwoonor-Williams%26aufirst%3DErnest%26date%3D2021%26spage%3D203%26epage%3D227%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Design of next-generation covalent inhibitors: Targeting residues beyond cysteine. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 95-134. <a href="https://doi.org/10.1016/bs.armc.2020.10.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.10.001%26sid%3Dliteratum%253Aachs%26atitle%3DDesign%252Bof%252Bnext-generation%252Bcovalent%252Binhibitors%25253A%252BTargeting%252Bresidues%252Bbeyond%252Bcysteine%26aulast%3DJones%26aufirst%3DLyn%2BH.%26date%3D2021%26spage%3D95%26epage%3D134%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Shao</span>, <span class="hlFld-ContribAuthor ">Jason E.  Gestwicki</span>. </span><span class="cited-content_cbyCitation_article-title">Neutral analogs of the heat shock protein 70 (Hsp70) inhibitor, JG-98. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (5)
                                     , 126954. <a href="https://doi.org/10.1016/j.bmcl.2020.126954" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.126954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.126954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.126954%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNeutral%252Banalogs%252Bof%252Bthe%252Bheat%252Bshock%252Bprotein%252B70%252B%252528Hsp70%252529%252Binhibitor%25252C%252BJG-98%26aulast%3DShao%26aufirst%3DHao%26date%3D2020%26volume%3D30%26issue%3D5%26spage%3D126954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Simulated data describing the MOA and kinetic parameters used to quantify the activity of TCIs. (A) TCI two-step binding mechanism. (B) The two components of TCI MOA, both reversible and covalent occupancy, contribute to the total target occupancy at a given concentration and time. Left = covalent occupancy (%CO) and right = total occupancy (%TO), both simulated using <i>K</i><sub><i>I</i></sub> = 1 Î¼M and <i>k</i><sub>inact</sub> = 0.069 min<sup>â1</sup>. FO = fraction reversible occupancy (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a> for derivation). (C) Determination of the essential second-order rate constant <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> from the concentration-dependent <i>k</i><sub>obs</sub> pseudo-first-order rate constant. Left = example where <i>K</i><sub><i>I</i></sub> < [I] so <i>k</i><sub>inact</sub> and <i>K</i><sub><i>I</i></sub> can be deconvoluted. Right = example where <i>K</i><sub><i>I</i></sub> â« [I] so the individual kinetic parameters cannot be distinguished. (D) Nucleoside-derived reversible <b>1</b> and covalent inhibitor <b>2</b> of HSP72.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of Lys56-targeting TCI binding modes. The tertiary conformation of the protein is maintained in the previously described semiopen form in all structures and modeling. The polar interactions of the adenine-type base and ribose are maintained in each binding mode. All diagrams were adapted from analysis using MOE (2014.09) and PyMOL Molecular Graphics System, Version 2.0 SchrÃ¶dinger, LLC. 2.2.3 A: Tyr15 up-conformation. The acrylate electrophile of <b>2</b> forms a Ï-stack interaction with Tyr15 and a hydrogen bond with Thr-37. In this reversible complex, the electrophile is too far from Lys56 to form a covalent bond (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKR">5MKR</a>). B: Tyr15 downconformation. Following rotation of Tyr15, rotation around the linker of <b>2</b> now positions the acrylate electrophile in close proximity to Lys56, suitable for covalent-bond formation in this precovalent complex. The acrylate moiety was not observed in the electron density due to flexibility, so was modeled to estimate distances (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKS">5MKS</a>). C: Binding mode of the high-affinity reversible HSP72 inhibitor <b>1</b> in the Tyr15 upconformation (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IO8">4IO8</a>). D: Model of a putative binding mode for a high-affinity Lys56-targeting TCI. With Tyr15 in a downconformation, rotation of the benzylic ether positions a potential <i>para</i>-electrophile at an appropriate distance from Lys56 in a precovalent complex. E: Three-step MOA, TCI binds HSP72 to form the reversible complex (EI) before transition to the precovalent complex (EI*) that can result in the formation of the covalent bond and the irreversible complex (E-I).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Conformation analysis of the proposed second-generation HSP72 TCI. A: Overlay of the acrylate TCI <b>2</b> (green) and the high-affinity reversible inhibitor <b>1</b> (cyan). B: Overlay of the acrylate TCI <b>2</b> (green) and the proposed conformation of the second-generation TCI (cyan) in the precovalent complex. The generic electrophile is represented as a purple sphere. C: Frequency distribution of torsional angles for 5â²-<i>O</i>-adenosine derivatives observed in the CSD; the green line represents the desired torsional angle. D: At the desired torsional angle, the modeled distances to Lys56 from different ring positions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0003.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Second-Generation Lysine-56-Targeting TCI<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 4-(bromomethyl) benzonitrile, NaH (60% in mineral oil), dimethylformamide (DMF), room temperature (RT), 1 h, 56%; (ii) 7 M NH<sub>3</sub>/MeOH, 110 Â°C, MW, 2 h, 82%; (iii) Br<sub>2</sub>, K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O, H<sub>2</sub>O, 1,4-dioxane, RT, 1 h, 72%; (iv) 4-chlorobenzylamine, EtOH, 160 Â°C, MW, 1 h, 66%; (v) 2 M NaOH, EtOH, 100 Â°C 3 h, 58%; (vi) 4-fluorophenol, HATU, DIPEA, DMF, RT, 18 h, 47%; (vii) TFA/H<sub>2</sub>O (5:2), RT, 0.5 h, 85%; (viii) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 47%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Third-Generation Aryl Sulfonyl Fluoride HSP72 TCIs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Br<sub>2</sub>, K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O, 1,4-dioxane, H<sub>2</sub>O, RT, 0.5 h, 74%; (ii) 4-chlorobenzylamine, EtOH, 160 Â°C MW, 1 h, 97%; (iii) 4-(fluorosulfonyl)benzoyl chloride, Et<sub>3</sub>N, DMF, 0 Â°C, 4 h; 56% (iv) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 32%; (v) 4-cyanobenzoyl chloride, Et<sub>3</sub>N, DMF, 0 Â°C, 3 h; (vi) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 16% over two steps; (vii) 4-(fluorosulfonyl)benzoic acid, HBTU, Et<sub>3</sub>N, DMF, RT, 2 h; 66%; (viii) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 74%; (ix) 33% MeNH<sub>2</sub> in EtOH, 160 Â°C, MW, 1 h, 77%; (x) 4-(fluorosulfonyl)benzoic acid, HBTU, Et<sub>3</sub>N, DMF, RT, 2 h; 35%; (xi) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 40%; (xii) quinoline-6-yl methanamine, EtOH, 160 Â°C, MW, 2 h, 8%; (xiii) 4-(fluorosulfonyl)benzoic acid, HBTU, Et<sub>3</sub>N, DMF, RT, 2 h; 40%; (xiv) 5:2 TFA/H<sub>2</sub>O, RT, 0.5 h, 11%.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/medium/jm9b01709_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Covalent FP-assay to determine the efficiency of covalent-bond formation. (1) Initial titration across a wide range of TCI concentrations and timepoints. (2) Extrapolation of the time-dependent change in <i>F</i><sub>b</sub> to t = 0. (3) Estimation of initial <i>K</i><sub><i>i</i></sub> from extrapolated <i>t</i> = 0 <i>F</i><sub>b</sub> values; for an example of how to determine <i>K<sub>i</sub></i> from an IC<sub>50</sub> in the FP-assay, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (4) Second focused titration on concentrations of TCI < <i>K</i><sub><i>i</i></sub>. (5) Gradient of time-dependent change in <i>F</i><sub>b</sub> used to calculate <i>k</i><sub>obs</sub> at a given concentration of TCI. (6) The gradient of the rate of change of <i>k</i><sub>obs</sub> with [TCI] determines the second-order rate constant <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub>. (7) By assuming <i>K</i><sub><i>i</i></sub> = <i>K</i><sub><i>I</i></sub>, <i>k</i><sub>inact</sub> can be calculated from <i>k</i><sub>inact</sub><i>/K</i><sub><i>I</i></sub> and converted to <i>t</i><sub>1/2</sub><sup>inf</sup>. See the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf" class="ext-link">Supporting Information</a> for details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01709/20200113/images/large/jm9b01709_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01709&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48144" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48144" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span> <span> </span><span class="NLM_article-title">Targeted covalent inhibitors for drug design</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">13408</span>â <span class="NLM_lpage">13421</span>, <span class="refDoi">Â DOI: 10.1002/anie.201601091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1002%2Fanie.201601091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=27539547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A280%3ADC%252BC2szitVegtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=13408-13421&author=T.+A.+Baillie&title=Targeted+covalent+inhibitors+for+drug+design&doi=10.1002%2Fanie.201601091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Covalent Inhibitors for Drug Design</span></div><div class="casAuthors">Baillie Thomas A</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">13408-13421</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In contrast to the traditional mechanism of drug action that relies on the reversible, noncovalent interaction of a ligand with its biological target, a targeted covalent inhibitor (TCI) is designed such that the initial, reversible association is followed by the formation of a covalent bond between an electrophile on the ligand and a nucleophilic center in the protein.  Although this approach offers a variety of potential benefits (high potency and extended duration of action), concerns over the possible toxicological consequences of protein haptenization have hindered the development of the TCI concept.  Recently, approaches to mitigate the risk of serious adverse reactions to this new class of agent have emerged, thus stimulating interest in the field and leading to authorization of the first cadre of TCIs to be marketed.  The covalent inhibitor approach is rapidly gaining acceptance as a valuable tool in drug discovery, and is poised to make a major impact on the design of enzyme inhibitors and receptor modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQYqmMVNSmxle98rul4nhSdfW6udTcc2ea77y0DVuh1rLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szitVegtg%253D%253D&md5=c4f6a83580f47f4609541f121d2c63db</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fanie.201601091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201601091%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DTargeted%2520covalent%2520inhibitors%2520for%2520drug%2520design%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D13408%26epage%3D13421%26doi%3D10.1002%2Fanie.201601091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">949</span>â <span class="NLM_lpage">964</span>, <span class="refDoi">Â DOI: 10.4155/fmc.10.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.4155%2Ffmc.10.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20640225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=949-964&author=D.+S.+Johnsonauthor=E.+Weerapanaauthor=B.+F.+Cravatt&title=Strategies+for+discovering+and+derisking+covalent%2C+irreversible+enzyme+inhibitors&doi=10.4155%2Ffmc.10.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for discovering and derisking covalent, irreversible enzyme inhibitors</span></div><div class="casAuthors">Johnson, Douglas S.; Weerapana, Eranthie; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-964</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article presents several covalent inhibitors, including examples of successful drugs, as well as highly selective, irreversible inhibitors of emerging therapeutic targets, such as fatty acid amide hydolase.  Covalent inhibitors have many desirable features, including increased biochem. efficiency of target disruption, less sensitivity toward pharmacokinetic parameters and increased duration of action that outlasts the pharmacokinetics of the compd.  Safety concerns that must be mitigated include lack of specificity and the potential immunogenicity of protein-inhibitor adduct(s).  Particular attention will be given to recent technologies, such as activity-based protein profiling, which allow one to define the proteome-wide selectivity patterns for covalent inhibitors in vitro and in vivo.  For instance, any covalent inhibitor can, in principle, be modified with a 'clickable' tag to generate an activity probe that is almost indistinguishable from the original agent.  These probes can be applied to any living system across a broad dose range to fully inventory their on and off targets.  The substantial no. of drugs on the market today that act by a covalent mechanism belies historical prejudices against the development of irreversibly acting therapeutic small mols.  Emerging proteomic technologies offer a means to systematically discriminate safe (selective) vs. deleterious (nonselective) covalent inhibitors and thus should inspire their future design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqm0whEybB-LVg90H21EOLACvtfcHk0lgLguDcsidP4Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFakur4%253D&md5=07e1f89418b168798ae5c1a9fa43e9c1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.21%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BS.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DStrategies%2520for%2520discovering%2520and%2520derisking%2520covalent%252C%2520irreversible%2520enzyme%2520inhibitors%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D949%26epage%3D964%26doi%3D10.4155%2Ffmc.10.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, R. A.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors in drug discovery: From accidental discoveries to avoided liabilities and designed therapies</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1061</span>â <span class="NLM_lpage">1073</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2015.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.drudis.2015.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=26002380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1061-1073&author=R.+A.+Bauer&title=Covalent+inhibitors+in+drug+discovery%3A+From+accidental+discoveries+to+avoided+liabilities+and+designed+therapies&doi=10.1016%2Fj.drudis.2015.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors in drug discovery: from accidental discoveries to avoided liabilities and designed therapies</span></div><div class="casAuthors">Bauer, Renato A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1061-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Drugs that covalently bond to their biol. targets have a long history in drug discovery.  A look at drug approvals in recent years suggests that covalent drugs will continue to make impacts on human health for years to come.  Although fraught with concerns about toxicity, the high potencies and prolonged effects achievable with covalent drugs may result in less-frequent drug dosing and in wide therapeutic margins for patients.  Covalent inhibition can also dissoc. drug pharmacodynamics (PD) from pharmacokinetics (PK), which can result in desired drug efficacy for inhibitors that have short systemic exposure.  Evidence suggests that there is a reduced risk for the development of resistance against covalent drugs, which is a major challenge in areas such as oncol. and infectious disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMkBDsUEu4arVg90H21EOLACvtfcHk0lhCtWXl2ydS5A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D&md5=6ba5b016d2804c7d56838a2724b9a0b9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DR.%2BA.%26atitle%3DCovalent%2520inhibitors%2520in%2520drug%2520discovery%253A%2520From%2520accidental%2520discoveries%2520to%2520avoided%2520liabilities%2520and%2520designed%2520therapies%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D1061%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2015.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomassi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Termathe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer-Wrangowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GrÃ¼tter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlenbrock, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KrÃ¼ll, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaumann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eppmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kibies, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffgaard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heil, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Cuaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuckmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tinnefeld, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahedi, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kast, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Targeting drug resistance in egfr with covalent inhibitors: A structure-based design approach</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6844</span>â <span class="NLM_lpage">6863</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01082</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01082" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6844-6863&author=J.+Engelauthor=A.+Richtersauthor=M.+Getlikauthor=S.+Tomassiauthor=M.+Keulauthor=M.+Termatheauthor=J.+Lategahnauthor=C.+Beckerauthor=S.+Mayer-Wrangowskiauthor=C.+Gr%C3%BCtterauthor=N.+Uhlenbrockauthor=J.+Kr%C3%BCllauthor=N.+Schaumannauthor=S.+Eppmannauthor=P.+Kibiesauthor=F.+Hoffgaardauthor=J.+Heilauthor=S.+Menningerauthor=S.+Ortiz-Cuaranauthor=J.+M.+Heuckmannauthor=V.+Tinnefeldauthor=R.+P.+Zahediauthor=M.+L.+Sosauthor=C.+Schultz-Fademrechtauthor=R.+K.+Thomasauthor=S.+M.+Kastauthor=D.+Rauh&title=Targeting+drug+resistance+in+egfr+with+covalent+inhibitors%3A+A+structure-based+design+approach&doi=10.1021%2Facs.jmedchem.5b01082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach</span></div><div class="casAuthors">Engel, Julian; Richters, Andre; Getlik, Matthaeus; Tomassi, Stefano; Keul, Marina; Termathe, Martin; Lategahn, Jonas; Becker, Christian; Mayer-Wrangowski, Svenja; Gruetter, Christian; Uhlenbrock, Niklas; Kruell, Jasmin; Schaumann, Niklas; Eppmann, Simone; Kibies, Patrick; Hoffgaard, Franziska; Heil, Jochen; Menninger, Sascha; Ortiz-Cuaran, Sandra; Heuckmann, Johannes M.; Tinnefeld, Verena; Zahedi, Rene P.; Sos, Martin L.; Schultz-Fademrecht, Carsten; Thomas, Roman K.; Kast, Stefan M.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6844-6863</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC).  Strong efforts were directed to the development of irreversible inhibitors and led to compd. CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects.  Here, the authors present a structure-based approach, rationalized by subsequent computational anal. of conformational ligand ensembles in soln., to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approx. 1500 compds.  Using protein x-ray crystallog., the authors deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational design approaches.  Chem. synthesis led to further compd. collections that revealed increased biochem. potency and, in part, selectivity toward mutated (L858R and L858R/T790M) vs. nonmutated EGFR.  Further cell-based and kinetic studies were performed to substantiate the authors initial findings.  Utilizing proteolytic digestion and nano-LC-MS/MS anal., the authors confirmed the alkylation of Cys797.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNGJPIuqoVObVg90H21EOLACvtfcHk0lhCtWXl2ydS5A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCjsrvM&md5=d5264a2a99d7752af0e587f23aeb7723</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01082%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DTomassi%26aufirst%3DS.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DTermathe%26aufirst%3DM.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DMayer-Wrangowski%26aufirst%3DS.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DUhlenbrock%26aufirst%3DN.%26aulast%3DKr%25C3%25BCll%26aufirst%3DJ.%26aulast%3DSchaumann%26aufirst%3DN.%26aulast%3DEppmann%26aufirst%3DS.%26aulast%3DKibies%26aufirst%3DP.%26aulast%3DHoffgaard%26aufirst%3DF.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DMenninger%26aufirst%3DS.%26aulast%3DOrtiz-Cuaran%26aufirst%3DS.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DTinnefeld%26aufirst%3DV.%26aulast%3DZahedi%26aufirst%3DR.%2BP.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DTargeting%2520drug%2520resistance%2520in%2520egfr%2520with%2520covalent%2520inhibitors%253A%2520A%2520structure-based%2520design%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6844%26epage%3D6863%26doi%3D10.1021%2Facs.jmedchem.5b01082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marino, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gladyshev, V. N.</span></span> <span> </span><span class="NLM_article-title">Cysteine function governs its conservation and degeneration and restricts its utilization on protein surfaces</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>404</i></span>,  <span class="NLM_fpage">902</span>â <span class="NLM_lpage">916</span>, <span class="refDoi">Â DOI: 10.1016/j.jmb.2010.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.jmb.2010.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20950627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGiur%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2010&pages=902-916&author=S.+M.+Marinoauthor=V.+N.+Gladyshev&title=Cysteine+function+governs+its+conservation+and+degeneration+and+restricts+its+utilization+on+protein+surfaces&doi=10.1016%2Fj.jmb.2010.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine Function Governs Its Conservation and Degeneration and Restricts Its Utilization on Protein Surfaces</span></div><div class="casAuthors">Marino, Stefano M.; Gladyshev, Vadim N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">404</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">902-916</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cysteine (Cys) is an enigmatic amino acid residue.  Although one of the least abundant amino acids, Cys often occurs in the functional sites of proteins.  Whereas free Cys is a polar amino acid, Cys in proteins is often buried, and its classification on the hydrophobicity scale is ambiguous.  We hypothesized that the deviation of Cys residues from the properties of a free amino acid is due to their reactivity and addressed this possibility by examg. Cys in large protein structure data sets.  Compared to other amino acids, Cys was characterized by the most extreme conservation pattern, with the majority of Cys being either highly conserved or poorly conserved.  In addn., clustering of Cys with another Cys residue was assocd. with high conservation, whereas exposure of Cys on protein surfaces was assocd. with low conservation.  Moreover, although clustered Cys behaved as polar residues, isolated Cys was the most buried residue of all, in disagreement with known chem. properties of Cys.  Thus, the anomalous hydrophobic behavior and conservation pattern of Cys can be explained by elimination of isolated Cys from protein surfaces during evolution and by clustering of other Cys residues.  These findings indicate that Cys abundance is governed by Cys function in proteins rather than by the sheer chem.-phys. properties of free amino acids, and suggest that a high tendency of Cys to be functionally active can considerably limit its abundance on protein surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpePL0RDIZDlrVg90H21EOLACvtfcHk0liF38c7JNeffA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGiur%252FF&md5=c2de18b2dcc4ac2b431e7f80173d5af3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DMarino%26aufirst%3DS.%2BM.%26aulast%3DGladyshev%26aufirst%3DV.%2BN.%26atitle%3DCysteine%2520function%2520governs%2520its%2520conservation%2520and%2520degeneration%2520and%2520restricts%2520its%2520utilization%2520on%2520protein%2520surfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D404%26spage%3D902%26epage%3D916%26doi%3D10.1016%2Fj.jmb.2010.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donner, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addo, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cepa, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woller, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shotwell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutti, J. E.</span></span> <span> </span><span class="NLM_article-title">Novel modes of inhibition of wild-type isocitrate dehydrogenase 1 (IDH1): Direct covalent modification of his315</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6647</span>â <span class="NLM_lpage">6657</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtrbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6647-6657&author=C.+G.+Jakobauthor=A.+K.+Upadhyayauthor=P.+L.+Donnerauthor=E.+Nichollauthor=S.+N.+Addoauthor=W.+Qiuauthor=C.+Lingauthor=S.+M.+Gopalakrishnanauthor=M.+Torrentauthor=S.+P.+Cepaauthor=J.+Shanleyauthor=A.+R.+Shoemakerauthor=C.+C.+Sunauthor=A.+Vasudevanauthor=K.+R.+Wollerauthor=J.+B.+Shotwellauthor=B.+Shawauthor=Z.+Bianauthor=J.+E.+Hutti&title=Novel+modes+of+inhibition+of+wild-type+isocitrate+dehydrogenase+1+%28IDH1%29%3A+Direct+covalent+modification+of+his315&doi=10.1021%2Facs.jmedchem.8b00305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Modes of Inhibition of Wild-Type Isocitrate Dehydrogenase 1 (IDH1): Direct Covalent Modification of His315</span></div><div class="casAuthors">Jakob, Clarissa G.; Upadhyay, Anup K.; Donner, Pamela L.; Nicholl, Emily; Addo, Sadiya N.; Qiu, Wei; Ling, Christopher; Gopalakrishnan, Sujatha M.; Torrent, Maricel; Cepa, Steven P.; Shanley, Jason; Shoemaker, Alexander R.; Sun, Chaohong C.; Vasudevan, Anil; Woller, Kevin R.; Shotwell, J. Brad; Shaw, Bailin; Bian, Zhiguo; Hutti, Jessica E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6647-6657</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">IDH1 plays a crit. role in a no. of metabolic processes and serves as a key source of cytosolic NADPH under conditions of cellular stress.  However, few inhibitors of wild-type IDH1 have been reported.  Here we present the discovery and biochem. characterization of two novel inhibitors of wild-type IDH1.  In addn., we present the first ligand-bound crystallog. characterization of these novel small mol. IDH1 binding pockets.  Importantly, the NADPH competitive Î±,Î²-unsatd. enone 1 makes a unique covalent linkage through active site H315.  As few small mols. have been shown to covalently react with histidine residues, these data support the potential utility of an underutilized strategy for reversible covalent small mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Rg5CN6JOyLVg90H21EOLACvtfcHk0lj3HM8PSLN5Zw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtrbF&md5=15c54988cd3080d5a675badbb62b6fec</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00305%26sid%3Dliteratum%253Aachs%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DDonner%26aufirst%3DP.%2BL.%26aulast%3DNicholl%26aufirst%3DE.%26aulast%3DAddo%26aufirst%3DS.%2BN.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DLing%26aufirst%3DC.%26aulast%3DGopalakrishnan%26aufirst%3DS.%2BM.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DCepa%26aufirst%3DS.%2BP.%26aulast%3DShanley%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DWoller%26aufirst%3DK.%2BR.%26aulast%3DShotwell%26aufirst%3DJ.%2BB.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DBian%26aufirst%3DZ.%26aulast%3DHutti%26aufirst%3DJ.%2BE.%26atitle%3DNovel%2520modes%2520of%2520inhibition%2520of%2520wild-type%2520isocitrate%2520dehydrogenase%25201%2520%2528IDH1%2529%253A%2520Direct%2520covalent%2520modification%2520of%2520his315%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6647%26epage%3D6657%26doi%3D10.1021%2Facs.jmedchem.8b00305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Horst, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of novel covalent bromodomain and extra-terminal domain (BET) inhibitors targeting a methionine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8202</span>â <span class="NLM_lpage">8211</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8202-8211&author=O.+A.+Kharenkoauthor=R.+G.+Patelauthor=S.+D.+Brownauthor=C.+Calosingauthor=A.+Whiteauthor=D.+Lakshminarasimhanauthor=R.+K.+Sutoauthor=B.+C.+Duffyauthor=D.+B.+Kitchenauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=E.+H.+van+der+Horstauthor=P.+R.+Young&title=Design+and+characterization+of+novel+covalent+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+targeting+a+methionine&doi=10.1021%2Facs.jmedchem.8b00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Novel Covalent Bromodomain and Extra-Terminal Domain (BET) Inhibitors Targeting a Methionine</span></div><div class="casAuthors">Kharenko, Olesya A.; Patel, Reena G.; Brown, S. David; Calosing, Cyrus; White, Andre; Lakshminarasimhan, Damodharan; Suto, Robert K.; Duffy, Bryan C.; Kitchen, Douglas B.; McLure, Kevin G.; Hansen, Henrik C.; van der Horst, Edward H.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8202-8211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET proteins are key epigenetic regulators that regulate transcription through binding to acetylated lysine (AcLys) residues of histones and transcription factors through bromodomains (BDs).  The disruption of this interaction with small mol. bromodomain inhibitors is a promising approach to treat various diseases including cancer, autoimmune and cardiovascular diseases.  Covalent inhibitors can potentially offer a more durable target inhibition leading to improved in vivo pharmacol.  Here the authors describe the design of covalent inhibitors of BRD4(BD1) that target a methionine in the binding pocket by attaching an epoxide warhead to a suitably oriented noncovalent inhibitor.  Using thermal denaturation, MALDI-TOF mass spectrometry, and an x-ray crystal structure, the authors demonstrate that these inhibitors selectively form a covalent bond with Met 149 in BRD4(BD1) but not other bromodomains and provide durable transcriptional and antiproliferative activity in cell based assays.  Covalent targeting of methionine offers a novel approach to drug discovery for BET proteins and other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHccGRxf5ShrVg90H21EOLACvtfcHk0lgcOYllCS2UWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO&md5=1c3ae393a5f8a8c12ffb16e6f3798118</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00666%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520novel%2520covalent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520targeting%2520a%2520methionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8202%26epage%3D8211%26doi%3D10.1021%2Facs.jmedchem.8b00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimster, N. P.</span></span> <span> </span><span class="NLM_article-title">Beyond cysteine: Recent developments in the area of targeted covalent inhibition</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">30</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1016/j.cbpa.2018.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.cbpa.2018.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29857316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVelt7zL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2018&pages=30-38&author=H.+Mukherjeeauthor=N.+P.+Grimster&title=Beyond+cysteine%3A+Recent+developments+in+the+area+of+targeted+covalent+inhibition&doi=10.1016%2Fj.cbpa.2018.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond cysteine: recent developments in the area of targeted covalent inhibition</span></div><div class="casAuthors">Mukherjee, Herschel; Grimster, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30-38</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Over the past decade targeted covalent inhibitors have undergone a renaissance due to the clin. validation and regulatory approval of several small mol. therapeutics that are designed to irreversibly modify their target protein.  Invariably, these compds. rely on the serendipitous placement of a cysteine residue proximal to the small mol. binding site; while this strategy has afforded numerous successes, it necessarily limits the no. of proteins that can be targeted by this approach.  This drawback has led several research groups to develop novel methodologies that target non-cysteine residues for covalent modification.  Herein, we survey the current literature of warheads that covalently modify non-cysteine amino acids in proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy0WIngEAtJLVg90H21EOLACvtfcHk0lgcOYllCS2UWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVelt7zL&md5=2f576af4dbd3046b3ac7486d2dfc04d9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DH.%26aulast%3DGrimster%26aufirst%3DN.%2BP.%26atitle%3DBeyond%2520cysteine%253A%2520Recent%2520developments%2520in%2520the%2520area%2520of%2520targeted%2520covalent%2520inhibition%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D44%26spage%3D30%26epage%3D38%26doi%3D10.1016%2Fj.cbpa.2018.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J. M.</span></span> <span> </span><span class="NLM_article-title">A perspective on the kinetics of covalent and irreversible inhibition</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3</span>â <span class="NLM_lpage">20</span>, <span class="refDoi">Â DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&author=J.+M.+Strelow&title=A+perspective+on+the+kinetics+of+covalent+and+irreversible+inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0lg2OJgiX_7F4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520perspective%2520on%2520the%2520kinetics%2520of%2520covalent%2520and%2520irreversible%2520inhibition%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="note"><p class="first last"><i>t</i><sub>1/2</sub><sup>inf</sup> = lnâ¯2/<i>k</i><sub>inact</sub></p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="note"><p class="first last"><i>K<sub>I</sub></i> = (<i>k</i><sub>off</sub> + <i>k</i><sub>inact</sub>)/<i>k</i><sub>on</sub>, <i>K<sub>i</sub></i> = <i>k</i><sub>off</sub>/<i>k</i><sub>on</sub> When <i>k</i><sub>off</sub> â« <i>k</i><sub>inact</sub> then <i>k</i><sub>off</sub> + <i>k</i><sub>inact</sub> = â¼<i>k</i><sub>off</sub> so <i>K<sub>I</sub></i> = â¼<i>K<sub>i</sub></i><i>k</i><sub>off</sub> = reversible dissociation first order rate constant of the non-covalent complex EI, <i>k</i><sub>on</sub> = reversible association second order rate constant of the non-covalent complex EI, <i>k</i><sub>inact</sub> = first order rate constant of the covalent complex E-I, <i>K<sub>i</sub></i> = true reversible equilibrium constant of the non-covalent complex EI, <i>K<sub>I</sub></i> = pseudo reversible equilibrium constant of the non-covalent complex EI, determined from <i>k</i><sub>obs</sub> = <i>k</i><sub>inact</sub>/2, <i>k</i><sub>obs</sub> = concentration dependent pseudo first order rate constant</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Cesco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufresne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittermaier, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors design and discovery</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">96</span>â <span class="NLM_lpage">114</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2017.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.ejmech.2017.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28651155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=96-114&author=S.+De%0ACescoauthor=J.+Kurianauthor=C.+Dufresneauthor=A.+K.+Mittermaierauthor=N.+Moitessier&title=Covalent+inhibitors+design+and+discovery&doi=10.1016%2Fj.ejmech.2017.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors design and discovery</span></div><div class="casAuthors">De Cesco, Stephane; Kurian, Jerry; Dufresne, Caroline; Mittermaier, Anthony K.; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-114</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In the history of therapeutics, covalent drugs occupy a very distinct category.  While representing a significant fraction of the drugs on the market, very few have been deliberately designed to interact covalently with their biol. target.  In this review, the prevalence of covalent drugs will first be briefly covered, followed by an introduction to their mechanisms of action and more detailed discussions of their discovery and the development of safe and efficient covalent enzyme inhibitors.  All stages of a drug discovery program will be covered, from target considerations to lead optimization, strategies to tune reactivity and computational methods.  The goal of this article is to provide an overview of the field and to outline good practices that are needed for the proper assessment and development of covalent inhibitors as well as a good understanding of the potential and limitations of current computational methods for the design of covalent drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeLNDmLtBdB7Vg90H21EOLACvtfcHk0lhqBSVGtJJ4wA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsLrI&md5=551c9a9c4bd69d4ad4d9156c315d5a94</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BCesco%26aufirst%3DS.%26aulast%3DKurian%26aufirst%3DJ.%26aulast%3DDufresne%26aufirst%3DC.%26aulast%3DMittermaier%26aufirst%3DA.%2BK.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DCovalent%2520inhibitors%2520design%2520and%2520discovery%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D96%26epage%3D114%26doi%3D10.1016%2Fj.ejmech.2017.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">The reactivity-driven biochemical mechanism of covalent KRASG12C inhibitors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">454</span>â <span class="NLM_lpage">462</span>, <span class="refDoi">Â DOI: 10.1038/s41594-018-0061-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1038%2Fs41594-018-0061-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29760531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=454-462&author=R.+Hansenauthor=U.+Petersauthor=A.+Babbarauthor=Y.+Chenauthor=J.+Fengauthor=M.+R.+Janesauthor=L.-S.+Liauthor=P.+Renauthor=Y.+Liuauthor=P.+P.+Zarrinkar&title=The+reactivity-driven+biochemical+mechanism+of+covalent+KRASG12C+inhibitors&doi=10.1038%2Fs41594-018-0061-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The reactivity-driven biochemical mechanism of covalent KRASG12C inhibitors</span></div><div class="casAuthors">Hansen, Rasmus; Peters, Ulf; Babbar, Anjali; Chen, Yuching; Feng, Jun; Janes, Matthew R.; Li, Lian-Sheng; Ren, Pingda; Liu, Yi; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">454-462</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Activating mutations in KRAS are among the most common tumor driver mutations.  Until recently, KRAS had been considered undruggable with small mols.; the discovery of the covalent KRASG12C inhibitors ARS-853 and ARS-1620 has demonstrated that it is feasible to inhibit KRAS with high potency in cells and animals.  Although the biol. activity of these inhibitors has been described, the biochem. mechanism of how the compds. achieve potent inhibition remained incompletely understood.  We now show that the activity of ARS-853 and ARS-1620 is primarily driven by KRAS-mediated catalysis of the chem. reaction with Cys12 in human KRASG12C, while the reversible binding affinity is weak, in the hundreds of micromolar or higher range.  The mechanism resolves how an induced, shallow and dynamic pocket not expected to support high-affinity binding of small mols. can nevertheless be targeted with potent inhibitors and may be applicable to other targets conventionally considered undruggable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrya_A4ZYnOu7Vg90H21EOLACvtfcHk0lhqBSVGtJJ4wA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL&md5=b3f9198f55998306f3b881c256be22a9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0061-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0061-5%26sid%3Dliteratum%253Aachs%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DThe%2520reactivity-driven%2520biochemical%2520mechanism%2520of%2520covalent%2520KRASG12C%2520inhibitors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D454%26epage%3D462%26doi%3D10.1038%2Fs41594-018-0061-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darjania, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarieh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu-Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS mutant cancers with a covalent G12C-specific inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">578</span>â <span class="NLM_lpage">589.e17</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2018.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.cell.2018.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29373830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=578-589.e17&author=M.+R.+Janesauthor=J.+Zhangauthor=L.-S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=X.+Guoauthor=Y.+Chenauthor=A.+Babbarauthor=S.+J.+Firdausauthor=L.+Darjaniaauthor=J.+Fengauthor=J.+H.+Chenauthor=S.+Liauthor=S.+Liauthor=Y.+O.+Longauthor=C.+Thachauthor=Y.+Liuauthor=A.+Zariehauthor=T.+Elyauthor=J.+M.+Kucharskiauthor=L.+V.+Kesslerauthor=T.+Wuauthor=K.+Yuauthor=Y.+Wangauthor=Y.+Yaoauthor=X.+Dengauthor=P.+P.+Zarrinkarauthor=D.+Brehmerauthor=D.+Dhanakauthor=M.+V.+Lorenziauthor=D.+Hu-Loweauthor=M.+P.+Patricelliauthor=P.+Renauthor=Y.+Liu&title=Targeting+KRAS+mutant+cancers+with+a+covalent+G12C-specific+inhibitor&doi=10.1016%2Fj.cell.2018.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span></div><div class="casAuthors">Janes, Matthew R.; Zhang, Jingchuan; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Guo, Xin; Chen, Yuching; Babbar, Anjali; Firdaus, Sarah J.; Darjania, Levan; Feng, Jun; Chen, Jeffrey H.; Li, Shuangwei; Li, Shisheng; Long, Yun O.; Thach, Carol; Liu, Yuan; Zarieh, Ata; Ely, Tess; Kucharski, Jeff M.; Kessler, Linda V.; Wu, Tao; Yu, Ke; Wang, Yi; Yao, Yvonne; Deng, Xiaohu; Zarrinkar, Patrick P.; Brehmer, Dirk; Dhanak, Dashyant; Lorenzi, Matthew V.; Hu-Lowe, Dana; Patricelli, Matthew P.; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-589.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRASG12C was recently identified to be potentially druggable by allele-specific covalent targeting of Cys 12 in vicinity to an inducible allosteric switch II pocket (S-IIP).  Success of this approach requires active cycling of KRASG12C between its active-GTP and inactive-GDP conformations as accessibility of the S-IIP is restricted only to the GDP-bound state.  This strategy proved feasible for inhibiting mutant KRAS in vitro; however, it is uncertain whether this approach would translate to in vivo.  Here, the authors describe structure-based design and identification of ARS-1620, a covalent compd. with high potency and selectivity for KRASG12C.  ARS-1620 achieves rapid and sustained in vivo target occupancy to induce tumor regression.  The authors use ARS-1620 to dissect oncogenic KRAS dependency and demonstrate that monolayer culture formats significantly underestimate KRAS dependency in vivo.  This study provides in vivo evidence that mutant KRAS can be selectively targeted and reveals ARS-1620 as representing a new generation of KRASG12C-specific inhibitors with promising therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVmv4KAl4_dLVg90H21EOLACvtfcHk0li9uikgL0_x-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D&md5=d0883bde981b51e41a794179ffae87be</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DDarjania%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DY.%2BO.%26aulast%3DThach%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarieh%26aufirst%3DA.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DHu-Lowe%26aufirst%3DD.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DTargeting%2520KRAS%2520mutant%2520cancers%2520with%2520a%2520covalent%2520G12C-specific%2520inhibitor%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D578%26epage%3D589.e17%26doi%3D10.1016%2Fj.cell.2018.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">FernÃ¡ndez-FernÃ¡ndez, M. R.</span>; <span class="NLM_string-name">Valpuesta, J.</span></span> <span> </span><span class="NLM_article-title">HSP70 chaperone: A master player in protein homeostasis [version 1; peer review: 3 approved]</span>.  <i>F1000Research</i> <span class="NLM_year">2018</span>,  <span class="NLM_volume">7</span>.<span class="refDoi">Â DOI: 10.12688/f1000research.15528.1</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.12688%2Ff1000research.15528.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=30338057" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+R.+Fern%C3%A1ndez-Fern%C3%A1ndez&author=J.+Valpuesta&title=HSP70+chaperone%3A+A+master+player+in+protein+homeostasis+%5Bversion+1%3B+peer+review%3A+3+approved%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.15528.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.12688%252Ff1000research.15528.1%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Fern%25C3%25A1ndez%26aufirst%3DM.%2BR.%26atitle%3DHSP70%2520chaperone%253A%2520A%2520master%2520player%2520in%2520protein%2520homeostasis%2520%255Bversion%25201%253B%2520peer%2520review%253A%25203%2520approved%255D%26jtitle%3DF1000Research%26date%3D2018%26volume%3D7%26doi%3D10.12688%2Ff1000research.15528.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scissum Gunn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manne, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, M.</span></span> <span> </span><span class="NLM_article-title">Targeting HSP70: A possible therapy for cancer</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">156</span>â <span class="NLM_lpage">166</span>, <span class="refDoi">Â DOI: 10.1016/j.canlet.2016.01.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.canlet.2016.01.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=26898980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVGktrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=156-166&author=S.+Kumarauthor=J.+Stokesauthor=U.+P.+Singhauthor=K.+Scissum+Gunnauthor=A.+Acharyaauthor=U.+Manneauthor=M.+Mishra&title=Targeting+HSP70%3A+A+possible+therapy+for+cancer&doi=10.1016%2Fj.canlet.2016.01.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Hsp70: A possible therapy for cancer</span></div><div class="casAuthors">Kumar, Sanjay; Stokes, James; Singh, Udai P.; Scissum Gunn, Karyn; Acharya, Arbind; Manne, Upender; Mishra, Manoj</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">156-166</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">In all organisms, heat-shock proteins (HSPs) provide an ancient defense system.  These proteins act as mol. chaperones by assisting proper folding and refolding of misfolded proteins and aid in the elimination of old and damaged cells.  HSPs include Hsp100, Hsp90, Hsp70, Hsp40, and small HSPs.  Through its substrate-binding domains, Hsp70 interacts with wide spectrum of mols., ranging from unfolded to natively folded and aggregated proteins, and provides cytoprotective role against various cellular stresses.  Under pathophysiol. conditions, the high expression of Hsp70 allows cells to survive with lethal injuries.  Increased Hsp70, by interacting at several points on apoptotic signaling pathways, leads to inhibition of apoptosis.  Elevated expression of Hsp70 in cancer cells may be responsible for tumorigenesis and for tumor progression by providing resistance to chemotherapy.  In contrast, inhibition or knockdown of Hsp70 reduces the size of tumors and can cause their complete regression.  Moreover, extracellular Hsp70 acts as an immunogen that participates in cross presentation of MHC-I mols.  The goals of this review are to examine the roles of Hsp70 in cancer and to present strategies targeting Hsp70 in the development of cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9lrUPziWvP7Vg90H21EOLACvtfcHk0li9uikgL0_x-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVGktrk%253D&md5=6295bea5fef02a457a9fdbed9350f850</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.01.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.01.056%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DStokes%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DU.%2BP.%26aulast%3DScissum%2BGunn%26aufirst%3DK.%26aulast%3DAcharya%26aufirst%3DA.%26aulast%3DManne%26aufirst%3DU.%26aulast%3DMishra%26aufirst%3DM.%26atitle%3DTargeting%2520HSP70%253A%2520A%2520possible%2520therapy%2520for%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2016%26volume%3D374%26spage%3D156%26epage%3D166%26doi%3D10.1016%2Fj.canlet.2016.01.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powers, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barillari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montfort, R. L. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span> <span> </span><span class="NLM_article-title">Targeting HSP70: The second potentially druggable heat shock protein and molecular chaperone?</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1542</span>â <span class="NLM_lpage">1550</span>, <span class="refDoi">Â DOI: 10.4161/cc.9.8.11204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.4161%2Fcc.9.8.11204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20372081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12qsL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1542-1550&author=M.+V.+Powersauthor=K.+Jonesauthor=C.+Barillariauthor=I.+Westwoodauthor=R.+L.+M.+V.+Montfortauthor=P.+Workman&title=Targeting+HSP70%3A+The+second+potentially+druggable+heat+shock+protein+and+molecular+chaperone%3F&doi=10.4161%2Fcc.9.8.11204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting HSP70: the second potentially druggable heat shock protein and molecular chaperone?</span></div><div class="casAuthors">Powers, Marissa V.; Jones, Keith; Barillari, Caterina; Westwood, Isaac; van Montfort, Rob L. M.; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1542-1550</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  The HSF1-mediated stress response pathway is steadily gaining momentum as a crit. source of targets for cancer therapy.  Key mediators of this pathway include mol. chaperones such as heat shock protein (HSP) 90.  There was considerable progress in targeting HSP90 and the preclin. efficacy and signs of early clin. activity of HSP90 inhibitors have provided proof-of-concept for targeting this group of proteins.  The HSP70 family of mol. chaperones are also key mediators of the HSF-1-stress response pathway and have multiple addnl. roles in protein folding, trafficking, and degrdn., as well as regulating apoptosis.  Genetic and biochem. studies have supported the discovery of HSP70 inhibitors which have the potential for use as single agents or in combination to enhance the effects of classical chemotherapeutic or molecularly targeted drugs including HSP90 inhibitors.  Here we provide a perspective on the progress made so far in discovering small mols. which target the HSP70 family, in the context of the available structural data and potential issues in drugging this key chaperone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpODWFSuNNDKbVg90H21EOLACvtfcHk0li9uikgL0_x-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12qsL%252FO&md5=12ad24d78fe0c9d538d889ecde1daa18</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.8.11204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.8.11204%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DM.%2BV.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DWestwood%26aufirst%3DI.%26aulast%3DMontfort%26aufirst%3DR.%2BL.%2BM.%2BV.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DTargeting%2520HSP70%253A%2520The%2520second%2520potentially%2520druggable%2520heat%2520shock%2520protein%2520and%2520molecular%2520chaperone%253F%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D1542%26epage%3D1550%26doi%3D10.4161%2Fcc.9.8.11204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgognoni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokurno, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macias, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surgenor, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A. J.</span></span> <span> </span><span class="NLM_article-title">Novel adenosine-derived inhibitors of 70 kDa heat shock protein, discovered through structure-based design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1510</span>â <span class="NLM_lpage">1513</span>, <span class="refDoi">Â DOI: 10.1021/jm801627a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801627a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BD1MXis1arsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1510-1513&author=D.+S.+Williamsonauthor=J.+Borgognoniauthor=A.+Clayauthor=Z.+Danielsauthor=P.+Dokurnoauthor=M.+J.+Drysdaleauthor=N.+Foloppeauthor=G.+L.+Francisauthor=C.+J.+Grahamauthor=R.+Howesauthor=A.+T.+Maciasauthor=J.+B.+Murrayauthor=R.+Parsonsauthor=T.+Shawauthor=A.+E.+Surgenorauthor=L.+Terryauthor=Y.+Wangauthor=M.+Woodauthor=A.+J.+Massey&title=Novel+adenosine-derived+inhibitors+of+70+kDa+heat+shock+protein%2C+discovered+through+structure-based+design&doi=10.1021%2Fjm801627a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Adenosine-Derived Inhibitors of 70 kDa Heat Shock Protein, Discovered Through Structure-Based Design</span></div><div class="casAuthors">Williamson, Douglas S.; Borgognoni, Jenifer; Clay, Alexandra; Daniels, Zoe; Dokurno, Pawel; Drysdale, Martin J.; Foloppe, Nicolas; Francis, Geraint L.; Graham, Christopher J.; Howes, Rob; Macias, Alba T.; Murray, James B.; Parsons, Rachel; Shaw, Terry; Surgenor, Allan E.; Terry, Lindsey; Wang, Yikang; Wood, Mike; Massey, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1510-1513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design and synthesis of novel adenosine-derived inhibitors of HSP70, guided by modeling and X-ray crystallog. structures of these compds. in complex with HSC70/BAG-1, is described.  Examples exhibited submicromolar affinity for HSP70, were highly selective over HSP90, and some displayed potency against HCT116 cells.  Exposure of I to HCT116 cells caused significant redn. in cellular levels of Raf-1 and Her2 at concns. similar to that which caused cell growth arrest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7sPdbJciUk7Vg90H21EOLACvtfcHk0lje1cjL_5ugoQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXis1arsbg%253D&md5=714b981545d1aa8db6627a252b3da8cc</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm801627a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801627a%26sid%3Dliteratum%253Aachs%26aulast%3DWilliamson%26aufirst%3DD.%2BS.%26aulast%3DBorgognoni%26aufirst%3DJ.%26aulast%3DClay%26aufirst%3DA.%26aulast%3DDaniels%26aufirst%3DZ.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DFrancis%26aufirst%3DG.%2BL.%26aulast%3DGraham%26aufirst%3DC.%2BJ.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DMacias%26aufirst%3DA.%2BT.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DShaw%26aufirst%3DT.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26aulast%3DTerry%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DMassey%26aufirst%3DA.%2BJ.%26atitle%3DNovel%2520adenosine-derived%2520inhibitors%2520of%252070%2520kDa%2520heat%2520shock%2520protein%252C%2520discovered%2520through%2520structure-based%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1510%26epage%3D1513%26doi%3D10.1021%2Fjm801627a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prodromou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalusa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aherne, W.</span></span> <span> </span><span class="NLM_article-title">Detection of the ATPase activity of the molecular chaperones HSP90 and HSP72 using the transcreener ADP assay kit</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">279</span>â <span class="NLM_lpage">286</span>, <span class="refDoi">Â DOI: 10.1177/1087057109360253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1177%2F1087057109360253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=20147598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFWjsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=279-286&author=M.+Rowlandsauthor=C.+McAndrewauthor=C.+Prodromouauthor=L.+Pearlauthor=A.+Kalusaauthor=K.+Jonesauthor=P.+Workmanauthor=W.+Aherne&title=Detection+of+the+ATPase+activity+of+the+molecular+chaperones+HSP90+and+HSP72+using+the+transcreener+ADP+assay+kit&doi=10.1177%2F1087057109360253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of the ATPase activity of the molecular chaperones Hsp90 and Hsp72 using the transcreener ADP assay kit</span></div><div class="casAuthors">Rowlands, Martin; McAndrew, Craig; Prodromou, Chrisostomos; Pearl, Laurence; Kalusa, Andrew; Jones, Keith; Workman, Paul; Aherne, Wynne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">279-286</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The mol. chaperone heat shock protein 90 (Hsp90) is required for the correct folding and stability of a no. of client proteins that are important for the growth and maintenance of cancer cells.  Heat shock protein 72 (Hsp72), a co-chaperone of Hsp90, is also emerging as an attractive cancer drug target.  Both proteins bind and hydrolyze ATP, and ATPase activity is essential for their function.  Inhibition of Hsp90 ATPase activity leads to the degrdn. of client proteins, resulting in cell growth inhibition and apoptosis.  Several small-mol. inhibitors of the ATPase activity of Hsp90 have been described and are currently being evaluated clin. for the treatment of cancer.  A no. of methods for the measurement of ATPase activity have been previously used, but not all of these are ideally suited to screening cascades in drug discovery projects.  The authors have evaluated the use of com. reagents (Transcreener ADP) for the measurement of ATPase activity of both yeast and human Hsp90 (ATP Km â¼500 Î¼M) and human Hsp72 (ATP Km â¼1 Î¼M).  The low ATPase activity of human Hsp90 and its stimulation by the co-chaperone Aha1 was measured with ease using reduced incubation times, generating robust data (Z' = 0.75).  The potency of several small-mol. inhibitors of both Hsp90 and Hsp72 was detd. using the Transcreener reagents and compared well to that detd. using other assay formats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhgslDxE_GR7Vg90H21EOLACvtfcHk0lje1cjL_5ugoQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFWjsrk%253D&md5=c5d113efdc01600390b365edd88f2525</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1087057109360253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109360253%26sid%3Dliteratum%253Aachs%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DProdromou%26aufirst%3DC.%26aulast%3DPearl%26aufirst%3DL.%26aulast%3DKalusa%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DAherne%26aufirst%3DW.%26atitle%3DDetection%2520of%2520the%2520ATPase%2520activity%2520of%2520the%2520molecular%2520chaperones%2520HSP90%2520and%2520HSP72%2520using%2520the%2520transcreener%2520ADP%2520assay%2520kit%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2010%26volume%3D15%26spage%3D279%26epage%3D286%26doi%3D10.1177%2F1087057109360253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bihan, Y.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">An irreversible inhibitor of HSP72 that unexpectedly targets lysine-56</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3536</span>â <span class="NLM_lpage">3540</span>, <span class="refDoi">Â DOI: 10.1002/anie.201611907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1002%2Fanie.201611907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28225177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A280%3ADC%252BC1c3ptlyjtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3536-3540&author=J.+Pettingerauthor=Y.-V.+Le+Bihanauthor=M.+Widyaauthor=R.+L.+M.+van+Montfortauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=An+irreversible+inhibitor+of+HSP72+that+unexpectedly+targets+lysine-56&doi=10.1002%2Fanie.201611907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">An Irreversible Inhibitor of HSP72 that Unexpectedly Targets Lysine-56</span></div><div class="casAuthors">Pettinger Jonathan; Le Bihan Yann-Vai; van Montfort Rob L M; Jones Keith; Cheeseman Matthew D; Le Bihan Yann-Vai; van Montfort Rob L M; Widya Marcella</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3536-3540</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The stress-inducible molecular chaperone, HSP72, is an important therapeutic target in oncology, but inhibiting this protein with small molecules has proven particularly challenging.  Validating HSP72 inhibitors in cells is difficult owing to competition with the high affinity and abundance of its endogenous nucleotide substrates.  We hypothesized this could be overcome using a cysteine-targeted irreversible inhibitor.  Using rational design, we adapted a validated 8-N-benzyladenosine ligand for covalent bond formation and confirmed targeted irreversible inhibition.  However, no cysteine in the protein was modified; instead, we demonstrate that lysine-56 is the key nucleophilic residue.  Targeting this lysine could lead to a new design paradigm for HSP72 chemical probes and drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2q3QuKVDECGBAiYWy6n3SfW6udTcc2eabhbdSoXXFybntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3ptlyjtg%253D%253D&md5=30645b206963d2f48add1a4d1de5376c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611907%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DLe%2BBihan%26aufirst%3DY.-V.%26aulast%3DWidya%26aufirst%3DM.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DAn%2520irreversible%2520inhibitor%2520of%2520HSP72%2520that%2520unexpectedly%2520targets%2520lysine-56%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2017%26volume%3D56%26spage%3D3536%26epage%3D3540%26doi%3D10.1002%2Fanie.201611907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Lysine-targeting covalent inhibitors</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">15200</span>â <span class="NLM_lpage">15209</span>, <span class="refDoi">Â DOI: 10.1002/anie.201707630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1002%2Fanie.201707630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28853194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A280%3ADC%252BC1cbivVGmsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=15200-15209&author=J.+Pettingerauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Lysine-targeting+covalent+inhibitors&doi=10.1002%2Fanie.201707630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Targeting Covalent Inhibitors</span></div><div class="casAuthors">Pettinger Jonathan; Jones Keith; Cheeseman Matthew D</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">15200-15209</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted covalent inhibitors have gained widespread attention in drug discovery as a validated method to circumvent acquired resistance in oncology.  This strategy exploits small-molecule/protein crystal structures to design tightly binding ligands with appropriately positioned electrophilic warheads.  Whilst most focus has been on targeting binding-site cysteine residues, targeting nucleophilic lysine residues can also represent a viable approach to irreversible inhibition.  However, owing to the basicity of the  -amino group in lysine, this strategy generates a number of specific challenges.  Herein, we review the key principles for inhibitor design, give historical examples, and present recent developments that demonstrate the potential of lysine targeting for future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQEe5pLdCmOT2mbXvSEQle5fW6udTcc2eabhbdSoXXFybntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbivVGmsQ%253D%253D&md5=bebb910023202e54213095ea353ea9a8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707630%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DLysine-targeting%2520covalent%2520inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2017%26volume%3D56%26spage%3D15200%26epage%3D15209%26doi%3D10.1002%2Fanie.201707630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazear, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Global profiling of lysine reactivity and ligandability in the human proteome</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1181</span>, <span class="refDoi">Â DOI: 10.1038/nchem.2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1038%2Fnchem.2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29168484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1181&author=S.+M.+Hackerauthor=K.+M.+Backusauthor=M.+R.+Lazearauthor=S.+Forliauthor=B.+E.+Correiaauthor=B.+F.+Cravatt&title=Global+profiling+of+lysine+reactivity+and+ligandability+in+the+human+proteome&doi=10.1038%2Fnchem.2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Global profiling of lysine reactivity and ligandability in the human proteome</span></div><div class="casAuthors">Hacker, Stephan M.; Backus, Keriann M.; Lazear, Michael R.; Forli, Stefano; Correia, Bruno E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nucleophilic amino acids make important contributions to protein function, including performing key roles in catalysis and serving as sites for post-translational modification.  Electrophilic groups that target amino-acid nucleophiles have been used to create covalent ligands and drugs, but have, so far, been mainly limited to cysteine and serine.  Here, the authors report a chem. proteomic platform for the global and quant. anal. of lysine residues in native biol. systems.  The authors have quantified, in total, >9000 lysines in human cell proteomes and have identified several hundred residues with heightened reactivity that are enriched at protein functional sites and can frequently be targeted by electrophilic small mols.  The authors also discovered lysine-reactive fragment electrophiles that inhibit enzymes by active site and allosteric mechanisms, as well as disrupt protein-protein interactions in transcriptional regulatory complexes, emphasizing the broad potential and diverse functional consequences of liganding lysine residues throughout the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKvUVcT5YRrVg90H21EOLACvtfcHk0lhJNJudYedOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN&md5=1dac8f079c1caffc8ae584291480fc61</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2826%26sid%3Dliteratum%253Aachs%26aulast%3DHacker%26aufirst%3DS.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DLazear%26aufirst%3DM.%2BR.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DGlobal%2520profiling%2520of%2520lysine%2520reactivity%2520and%2520ligandability%2520in%2520the%2520human%2520proteome%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1181%26doi%3D10.1038%2Fnchem.2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span> <span> </span><span class="NLM_article-title">Exploiting protein conformational change to optimize adenosine-derived inhibitors of HSP70</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4625</span>â <span class="NLM_lpage">4636</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b02001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b02001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4625-4636&author=M.+D.+Cheesemanauthor=I.+M.+Westwoodauthor=O.+Barbeauauthor=M.+Rowlandsauthor=S.+Dobsonauthor=A.+M.+Jonesauthor=F.+Jeganathanauthor=R.+Burkeauthor=N.+Kadiauthor=P.+Workmanauthor=I.+Collinsauthor=R.+L.+M.+van+Montfortauthor=K.+Jones&title=Exploiting+protein+conformational+change+to+optimize+adenosine-derived+inhibitors+of+HSP70&doi=10.1021%2Facs.jmedchem.5b02001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Protein Conformational Change to Optimize Adenosine-Derived Inhibitors of HSP70</span></div><div class="casAuthors">Cheeseman, Matthew D.; Westwood, Isaac M.; Barbeau, Olivier; Rowlands, Martin; Dobson, Sarah; Jones, Alan M.; Jeganathan, Fiona; Burke, Rosemary; Kadi, Nadia; Workman, Paul; Collins, Ian; van Montfort, Rob L. M.; Jones, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4625-4636</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HSP70 is a mol. chaperone and a key component of the heat-shock response.  Because of its proposed importance in oncol., this protein has become a popular target for drug discovery, efforts which have as yet brought little success.  This study demonstrates that adenosine-derived HSP70 inhibitors potentially bind to the protein with a novel mechanism of action, the stabilization by desolvation of an intramol. salt-bridge which induces a conformational change in the protein, leading to high affinity ligands.  We also demonstrate that through the application of this mechanism, adenosine-derived HSP70 inhibitors can be optimized in a rational manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-rro5uIFqZbVg90H21EOLACvtfcHk0liJrSAk52WZqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7k%253D&md5=772967b1f315ca9c0e9bbbb115493754</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b02001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b02001%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DWestwood%26aufirst%3DI.%2BM.%26aulast%3DBarbeau%26aufirst%3DO.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DDobson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DA.%2BM.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DKadi%26aufirst%3DN.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DExploiting%2520protein%2520conformational%2520change%2520to%2520optimize%2520adenosine-derived%2520inhibitors%2520of%2520HSP70%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4625%26epage%3D4636%26doi%3D10.1021%2Facs.jmedchem.5b02001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yahya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span> <span> </span><span class="NLM_article-title">A fragment-based approach applied to a highly flexible target: Insights and challenges towards the inhibition of HSP70 isoforms</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">34701</span> <span class="refDoi">Â DOI: 10.1038/srep34701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1038%2Fsrep34701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=27708405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Cgt7bM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=A.+M.+Jonesauthor=I.+M.+Westwoodauthor=J.+D.+Osborneauthor=T.+P.+Matthewsauthor=M.+D.+Cheesemanauthor=M.+G.+Rowlandsauthor=F.+Jeganathanauthor=R.+Burkeauthor=D.+Leeauthor=N.+Kadiauthor=M.+Liuauthor=M.+Richardsauthor=C.+McAndrewauthor=N.+Yahyaauthor=S.+E.+Dobsonauthor=K.+Jonesauthor=P.+Workmanauthor=I.+Collinsauthor=R.+L.+M.+van+Montfort&title=A+fragment-based+approach+applied+to+a+highly+flexible+target%3A+Insights+and+challenges+towards+the+inhibition+of+HSP70+isoforms&doi=10.1038%2Fsrep34701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A fragment-based approach applied to a highly flexible target: Insights and challenges towards the inhibition of HSP70 isoforms</span></div><div class="casAuthors">Jones, Alan M.; Westwood, Isaac M.; Osborne, James D.; Matthews, Thomas P.; Cheeseman, Matthew D.; Rowlands, Martin G.; Jeganathan, Fiona; Burke, Rosemary; Lee, Diane; Kadi, Nadia; Liu, Manjuan; Richards, Meirion; McAndrew, Craig; Yahya, Norhakim; Dobson, Sarah E.; Jones, Keith; Workman, Paul; Collins, Ian; van Montfort, Rob L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34701</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Heat-shock protein 70s (HSP70s) are mol. chaperones implicated in many cancers and of significant interest as targets for novel cancer therapies.  Several HSP70 inhibitors have been reported, but because the majority have poor physicochem. properties and for many the exact mode of action is poorly understood, more detailed mechanistic and structural insight into ligand-binding to HSP70s is urgently needed.  Here, the authors describe the 1st comprehensive fragment-based inhibitor exploration of an HSP70 enzyme, which yielded an aminoquinazoline fragment that was elaborated to a novel ATP binding site ligand with different physicochem. properties to known adenosine-based HSP70 inhibitors.  Crystal structures of aminoquinazoline ligands bound to the different conformational states of the HSP70 nucleotide binding domain highlighted the challenges of a fragment-based approach when applied to this particular flexible enzyme class with an ATP-binding site that changes shape and size during its catalytic cycle.  In these studies, the authors showed that Ser-275 is a key residue in the selective binding of ATP.  Addnl., the structural data revealed a potential functional role for the ATP ribose moiety in priming the protein for the formation of the ATP-bound pre-hydrolysis complex by influencing the conformation of one of the phosphate binding loops.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOPetdvzOGKrVg90H21EOLACvtfcHk0liJrSAk52WZqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Cgt7bM&md5=6172a09b29ab4c7e284c33e7a463106d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsrep34701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep34701%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DA.%2BM.%26aulast%3DWestwood%26aufirst%3DI.%2BM.%26aulast%3DOsborne%26aufirst%3DJ.%2BD.%26aulast%3DMatthews%26aufirst%3DT.%2BP.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DRowlands%26aufirst%3DM.%2BG.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DKadi%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DYahya%26aufirst%3DN.%26aulast%3DDobson%26aufirst%3DS.%2BE.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26atitle%3DA%2520fragment-based%2520approach%2520applied%2520to%2520a%2520highly%2520flexible%2520target%253A%2520Insights%2520and%2520challenges%2520towards%2520the%2520inhibition%2520of%2520HSP70%2520isoforms%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep34701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Cera, E.</span></span> <span> </span><span class="NLM_article-title">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5894</span>â <span class="NLM_lpage">5902</span>, <span class="refDoi">Â DOI: 10.1021/bi3006913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3006913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5894-5902&author=A.+D.+Vogtauthor=E.+Di+Cera&title=Conformational+selection+or+induced+fit%3F+A+critical+appraisal+of+the+kinetic+mechanism&doi=10.1021%2Fbi3006913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational selection or induced fit? A critical appraisal of the kinetic mechanism</span></div><div class="casAuthors">Vogt, Austin D.; Di Cera, Enrico</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5894-5902</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">For almost 5 decades, 2 competing mechanisms of ligand recognition, conformational selection and induced fit, have dominated the interpretation of ligand binding in biol. macromols.  When binding-dissocn. events are fast compared to conformational transitions, the rate of approach to equil., kobs, becomes diagnostic of conformational selection or induced fit based on whether it decreases or increases, resp., with the ligand concn. [L].  However, this simple conclusion based on the rapid equil. approxn. is not valid in general.  Here, the authors show that conformational selection is assocd. with a rich repertoire of kinetic properties, with kobs decreasing or increasing with [L] depending on the relative magnitude of the rate of ligand dissocn., koff, and the rate of conformational isomerization, kr.  The authors prove that, even for the simplest 2-step mechanism of ligand binding, a decrease in kobs with [L] is unequivocal evidence of conformational selection, but an increase in kobs with [L] is not unequivocal evidence of induced fit.  Ligand binding to glucokinase, thrombin, and its precursor, prethrombin-2, are used as relevant examples.  It is concluded that conformational selection as a mechanism for ligand binding to its target may be far more common than currently believed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdButaqMoNzLVg90H21EOLACvtfcHk0liJrSAk52WZqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVKltbo%253D&md5=0f7c66027467c390967075c3be8e5254</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi3006913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3006913%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DA.%2BD.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DConformational%2520selection%2520or%2520induced%2520fit%253F%2520A%2520critical%2520appraisal%2520of%2520the%2520kinetic%2520mechanism%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5894%26epage%3D5902%26doi%3D10.1021%2Fbi3006913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirrenberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of HSP70 inhibitors</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e78443</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0078443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1371%2Fjournal.pone.0078443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=24265689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslylsLzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=R.+Schlechtauthor=S.+R.+Scholzauthor=H.+Dahmenauthor=A.+Wegenerauthor=C.+Sirrenbergauthor=D.+Musilauthor=J.+Bomkeauthor=H.-M.+Eggenweilerauthor=M.+P.+Mayerauthor=B.+Bukau&title=Functional+analysis+of+HSP70+inhibitors&doi=10.1371%2Fjournal.pone.0078443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Functional analysis of Hsp70 inhibitors</span></div><div class="casAuthors">Schlecht, Rainer; Scholz, Sebastian R.; Dahmen, Heike; Wegener, Ansgar; Sirrenberg, Christian; Musil, Djordje; Bomke, Joerg; Eggenweiler, Hans-Michael; Mayer, Matthias P.; Bukau, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e78443</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The mol. chaperones of the Hsp70 family have been recognized as targets for anti-cancer therapy.  Since several paralogs of Hsp70 proteins exist in cytosol, endoplasmic reticulum and mitochondria, we investigated which isoform needs to be down-regulated for reducing viability of cancer cells.  For two recently identified small mol. inhibitors, VER-155008 and 2-phenylethynesulfonamide (PES), which are proposed to target different sites in Hsp70s, we analyzed the mol. mode of action in vitro.  We found that for significant redn. of viability of cancer cells simultaneous knockdown of heat-inducible Hsp70 (HSPA1) and constitutive Hsc70 (HSPA8) is necessary.  The compd. VER-155008, which binds to the nucleotide binding site of Hsp70, arrests the nucleotide binding domain (NBD) in a half-open conformation and thereby acts as ATP-competitive inhibitor that prevents allosteric control between NBD and substrate binding domain (SBD).  Compd. PES interacts with the SBD of Hsp70 in an unspecific, detergent-like fashion, under the conditions tested.  None of the two inhibitors investigated was isoform-specific.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk_9mIXQAAX7Vg90H21EOLACvtfcHk0ljLIG57cNvvjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslylsLzJ&md5=776f32214099c99744f6e32828b420ad</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0078443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0078443%26sid%3Dliteratum%253Aachs%26aulast%3DSchlecht%26aufirst%3DR.%26aulast%3DScholz%26aufirst%3DS.%2BR.%26aulast%3DDahmen%26aufirst%3DH.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DSirrenberg%26aufirst%3DC.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DEggenweiler%26aufirst%3DH.-M.%26aulast%3DMayer%26aufirst%3DM.%2BP.%26aulast%3DBukau%26aufirst%3DB.%26atitle%3DFunctional%2520analysis%2520of%2520HSP70%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0078443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xinyi, H.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization competition assay: The range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">34</span>â <span class="NLM_lpage">38</span>, <span class="refDoi">Â DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=12854996" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&author=H.+Xinyi&title=Fluorescence+polarization+competition+assay%3A+The+range+of+resolvable+inhibitor+potency+is+limited+by+the+affinity+of+the+fluorescent+ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DXinyi%26aufirst%3DH.%26atitle%3DFluorescence%2520polarization%2520competition%2520assay%253A%2520The%2520range%2520of%2520resolvable%2520inhibitor%2520potency%2520is%2520limited%2520by%2520the%2520affinity%2520of%2520the%2520fluorescent%2520ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="note"><p class="first last"><i>pK<sub>i</sub></i> = âlogâ¯<i>K<sub>i</sub></i> (<i>M</i>).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span> <span> </span><span class="NLM_article-title">Advances in the development of shape similarity methods and their application in drug discovery</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">315</span>, <span class="refDoi">Â DOI: 10.3389/fchem.2018.00315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.3389%2Ffchem.2018.00315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=30090808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaitrvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=315&author=A.+Kumarauthor=K.+Y.+J.+Zhang&title=Advances+in+the+development+of+shape+similarity+methods+and+their+application+in+drug+discovery&doi=10.3389%2Ffchem.2018.00315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of shape similarity methods and their application in drug discovery</span></div><div class="casAuthors">Kumar, Ashutosh; Zhang, Kam Y. J.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">315/1-315/21</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Mol. similarity is a key concept in drug discovery.  It is based on the assumption that structurally similar mols. frequently have similar properties.  Assessment of similarity between small mols. has been highly effective in the discovery and development of various drugs.  Esp., two-dimensional (2D) similarity approaches have been quite popular due to their simplicity, accuracy and efficiency.  Recently, the focus has been shifted toward the development of methods involving the representation and comparison of three-dimensional (3D) conformation of small mols.  Among the 3D similarity methods, evaluation of shape similarity is now gaining attention for its application not only in virtual screening but also in mol. target prediction, drug repurposing and scaffold hopping.  A wide range of methods have been developed to describe mol. shape and to det. the shape similarity between small mols.  The most widely used methods include atom distance-based methods, surface-based approaches such as spherical harmonics and 3D Zernike descriptors, atom-centered Gaussian overlay based representations.  Several of these methods demonstrated excellent virtual screening performance not only retrospectively but also prospectively.  In addn. to methods assessing the similarity between small mols., shape similarity approaches have been developed to compare shapes of protein structures and binding pockets.  Addnl., shape comparisons between at. models and 3D d. maps allowed the fitting of at. models into cryo-electron microscopy maps.  This review aims to summarize the methodol. advances in shape similarity assessment highlighting advantages, disadvantages and their application in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTIDZv0Wh2Q7Vg90H21EOLACvtfcHk0ljLIG57cNvvjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaitrvF&md5=9c7690b5caf58f26be5920318779deba</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2018.00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2018.00315%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520shape%2520similarity%2520methods%2520and%2520their%2520application%2520in%2520drug%2520discovery%26jtitle%3DFront.%2520Chem.%26date%3D2018%26volume%3D6%26spage%3D315%26doi%3D10.3389%2Ffchem.2018.00315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span> </span><span class="NLM_article-title">The Cambridge Structural Database (CSD)</span>. <a href="https://www.ccdc.cam.ac.uk/solutions/csd-system/components/csd/" class="extLink">https://www.ccdc.cam.ac.uk/solutions/csd-system/components/csd/</a> (accessed July, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+Cambridge+Structural+Database+%28CSD%29.+https%3A%2F%2Fwww.ccdc.cam.ac.uk%2Fsolutions%2Fcsd-system%2Fcomponents%2Fcsd%2F+%28accessed+July%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520Cambridge%2520Structural%2520Database%2520%2528CSD%2529%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span> <span> </span><span class="NLM_article-title">Characterising covalent warhead reactivity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2066</span>â <span class="NLM_lpage">2074</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2019.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.bmc.2019.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=30975501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntVGgsbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2066-2074&author=J.+S.+Martinauthor=C.+J.+MacKenzieauthor=D.+Fletcherauthor=I.+H.+Gilbert&title=Characterising+covalent+warhead+reactivity&doi=10.1016%2Fj.bmc.2019.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Characterising covalent warhead reactivity</span></div><div class="casAuthors">Martin, James S.; MacKenzie, Claire J.; Fletcher, Daniel; Gilbert, Ian H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2066-2074</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Many drugs currently used are covalent inhibitors and irreversibly inhibit their targets.  Most of these were discovered through serendipity.  Covalent inhibitions can have many advantages from a pharmacokinetic perspective.  However, until recently most organizations have shied away from covalent compd. design due to fears of non-specific inhibition of off-target proteins leading to toxicity risks.  However, there has been a renewed interest in covalent modifiers as potential drugs, as it possible to get highly selective compds.  It is therefore important to know how reactive a warhead is and to be able to select the least reactive warhead possible to avoid toxicity.  A robust NMR based assay was developed and used to measure the reactivity of a variety of covalent warheads against serine and cysteine - the two most common targets for covalent drugs.  A selection of these warheads also had their reactivity measured against threonine, tyrosine, lysine, histidine and arginine to better understand our ability to target non-traditional residues.  The reactivity was also measured at various pHs to assess what effect the environment in the active site would have on these reactions.  The reactivity of a covalent modifier was found to be very dependent on the amino acid residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVaz2cnHGpo7Vg90H21EOLACvtfcHk0lhgwTB6Fvbm-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntVGgsbo%253D&md5=487334d8d0a8b03d0a25d6d87c1b4329</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DJ.%2BS.%26aulast%3DMacKenzie%26aufirst%3DC.%2BJ.%26aulast%3DFletcher%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DCharacterising%2520covalent%2520warhead%2520reactivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2066%26epage%3D2074%26doi%3D10.1016%2Fj.bmc.2019.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Convery, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span> <span> </span><span class="NLM_article-title">Selectively targeting the kinome-conserved lysine of PI3KÎ´ as a general approach to covalent kinase inhibition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">932</span>â <span class="NLM_lpage">939</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b08979</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08979" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=932-939&author=S.+E.+Daltonauthor=L.+Dittusauthor=D.+A.+Thomasauthor=M.+A.+Converyauthor=J.+Nunesauthor=J.+T.+Bushauthor=J.+P.+Evansauthor=T.+Wernerauthor=M.+Bantscheffauthor=J.+A.+Murphyauthor=S.+Campos&title=Selectively+targeting+the+kinome-conserved+lysine+of+PI3K%CE%B4+as+a+general+approach+to+covalent+kinase+inhibition&doi=10.1021%2Fjacs.7b08979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Selectively Targeting the Kinome-Conserved Lysine of PI3KÎ´ as a General Approach to Covalent Kinase Inhibition</span></div><div class="casAuthors">Dalton, Samuel E.; Dittus, Lars; Thomas, Daniel A.; Convery, Maire A.; Nunes, Joao; Bush, Jacob T.; Evans, John P.; Werner, Thilo; Bantscheff, Marcus; Murphy, John A.; Campos, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">932-939</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective covalent inhibition of kinases by targeting poorly conserved cysteines has proven highly fruitful to date in the development of chem. probes and approved drugs.  However, this approach is limited to â¼200 kinases possessing such a cysteine near the ATP-binding pocket.  Herein, we report a novel approach to achieve selective, irreversible kinase inhibition, by targeting the conserved catalytic lysine residue.  We have illustrated our approach by developing selective, covalent PI3KÎ´ inhibitors that exhibit nanomolar potency in cellular assays, and a duration of action >48 h in CD4+ T cells.  Despite conservation of the lysine residue throughout the kinome, the lead compd. shows high levels of selectivity over a selection of lipid and protein kinases in biochem. assays, as well as covalent binding to very few off-target proteins in live-cell proteomic studies.  We anticipate this approach could offer a general strategy, as an alternative to targeting non-conserved cysteines, for the development of selective covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGvT7h2D9zf7Vg90H21EOLACvtfcHk0lhgwTB6Fvbm-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK&md5=90bb399303a942fb87f1b9096eb8bdc6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08979%26sid%3Dliteratum%253Aachs%26aulast%3DDalton%26aufirst%3DS.%2BE.%26aulast%3DDittus%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DD.%2BA.%26aulast%3DConvery%26aufirst%3DM.%2BA.%26aulast%3DNunes%26aufirst%3DJ.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DJ.%2BA.%26aulast%3DCampos%26aufirst%3DS.%26atitle%3DSelectively%2520targeting%2520the%2520kinome-conserved%2520lysine%2520of%2520PI3K%25CE%25B4%2520as%2520a%2520general%2520approach%2520to%2520covalent%2520kinase%2520inhibition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D932%26epage%3D939%26doi%3D10.1021%2Fjacs.7b08979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Light, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minasov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duban, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, W. F.</span></span> <span> </span><span class="NLM_article-title">Adherence to Burgi-Dunitz stereochemical principles requires significant structural rearrangements in Schiff-base formation: Insights from transaldolase complexes</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">544</span>â <span class="NLM_lpage">552</span>, <span class="refDoi">Â DOI: 10.1107/S1399004713030666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1107%2FS1399004713030666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=24531488" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2014&pages=544-552&author=S.+H.+Lightauthor=G.+Minasovauthor=M.-E.+Dubanauthor=W.+F.+Anderson&title=Adherence+to+Burgi-Dunitz+stereochemical+principles+requires+significant+structural+rearrangements+in+Schiff-base+formation%3A+Insights+from+transaldolase+complexes&doi=10.1107%2FS1399004713030666"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS1399004713030666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1399004713030666%26sid%3Dliteratum%253Aachs%26aulast%3DLight%26aufirst%3DS.%2BH.%26aulast%3DMinasov%26aufirst%3DG.%26aulast%3DDuban%26aufirst%3DM.-E.%26aulast%3DAnderson%26aufirst%3DW.%2BF.%26atitle%3DAdherence%2520to%2520Burgi-Dunitz%2520stereochemical%2520principles%2520requires%2520significant%2520structural%2520rearrangements%2520in%2520Schiff-base%2520formation%253A%2520Insights%2520from%2520transaldolase%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2014%26volume%3D70%26spage%3D544%26epage%3D552%26doi%3D10.1107%2FS1399004713030666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devaraj, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, C. L.</span></span> <span> </span><span class="NLM_article-title">Approach control. Stereoelectronic origin of geometric constraints on n-to-s and n-to-o acyl shifts in peptides</span>. <i>Chemical Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1789</span>â <span class="NLM_lpage">1794</span>, <span class="refDoi">Â DOI: 10.1039/C7SC04046F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC7SC04046F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29675223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFOqsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1789-1794&author=N.+K.+Devarajauthor=C.+L.+Perrin&title=Approach+control.+Stereoelectronic+origin+of+geometric+constraints+on+n-to-s+and+n-to-o+acyl+shifts+in+peptides&doi=10.1039%2FC7SC04046F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Approach control. Stereoelectronic origin of geometric constraints on N-to-S and N-to-O acyl shifts in peptides</span></div><div class="casAuthors">Devaraj, Neal K.; Perrin, Charles L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1789-1794</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Intramol. N-to-S or N-to-O acyl shifts in peptides are of fundamental and practical importance, as they constitute the first step in protein splicing and can be used for the synthesis of thioester-modified peptides required for native chem. ligation.  It has been stated that the nucleophile must be positioned anti to the carbonyl oxygen, as in a cis amide.  Despite the importance of such reactions, an understanding of this geometric restriction remains obscure.  Here we argue that the empirical requirement for positioning the nucleophile is a stereoelectronic effect arising from the ease of approach of the nucleophile to a carbonyl group, not ground-state destabilization.  DFT calcns. on model amides support our explanation and indicate a significant decrease in both the transition-state energy and the activation energy for a cis amide.  However, the approach of the nucleophile must be anti not only to the carbonyl oxygen but also to the nitrogen.  The direction of approach is expressed by a new, modified B.ovrddot.urgi-Dunitz angle.  Our data shed light on the mechanisms of acyl shifts in peptides, and they explain why a cis peptide might be required for protein splicing.  The further implications for acyl shifts in homoserine and homocysteine peptides and for aldol condensations are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFaMEQrfJxDLVg90H21EOLACvtfcHk0lj5llAkvmliAw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFOqsg%253D%253D&md5=8de52bee047d5f195307b2211c9fa595</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FC7SC04046F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC04046F%26sid%3Dliteratum%253Aachs%26aulast%3DDevaraj%26aufirst%3DN.%2BK.%26aulast%3DPerrin%26aufirst%3DC.%2BL.%26atitle%3DApproach%2520control.%2520Stereoelectronic%2520origin%2520of%2520geometric%2520constraints%2520on%2520n-to-s%2520and%2520n-to-o%2520acyl%2520shifts%2520in%2520peptides%26jtitle%3DChemical%2520Science%26date%3D2018%26volume%3D9%26spage%3D1789%26epage%3D1794%26doi%3D10.1039%2FC7SC04046F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdul
Fattah, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saeed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albericio, F.</span></span> <span> </span><span class="NLM_article-title">Recent advances towards sulfur (vi) fluoride exchange (SUFEX) click chemistry</span>. <i>J. Fluorine Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>213</i></span>,  <span class="NLM_fpage">87</span>â <span class="NLM_lpage">112</span>, <span class="refDoi">Â DOI: 10.1016/j.jfluchem.2018.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1016%2Fj.jfluchem.2018.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSrtb7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=213&publication_year=2018&pages=87-112&author=T.+Abdul%0AFattahauthor=A.+Saeedauthor=F.+Albericio&title=Recent+advances+towards+sulfur+%28vi%29+fluoride+exchange+%28SUFEX%29+click+chemistry&doi=10.1016%2Fj.jfluchem.2018.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances towards sulfur (VI) fluoride exchange (SuFEx) click chemistry</span></div><div class="casAuthors">Abdul Fattah, Tanzeela; Saeed, Aamer; Albericio, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Fluorine Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">213</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87-112</span>CODEN:
                <span class="NLM_cas:coden">JFLCAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-1139</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This review comprehensively focused on the sole reactivity of SuFEx click reaction and its role in the building of novel sterically hindered amides, fluorosulfates, diverse compds. from SOF4 hub, high-mol. wt. polymer materials (polysulfates and polysulfonates) and surface coatings.  The SuFEx part in post-polymn. reactions, late-stage drug functionalization (LSF) and oil/water sepns. was also highlighted.  Finally, the potential application of the SuFEx for bioconjugation is also analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRiA6hdvWJmbVg90H21EOLACvtfcHk0lj5llAkvmliAw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSrtb7J&md5=9b151ab663fcdf5227068debf1c2d8e0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jfluchem.2018.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jfluchem.2018.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DAbdul%2BFattah%26aufirst%3DT.%26aulast%3DSaeed%26aufirst%3DA.%26aulast%3DAlbericio%26aufirst%3DF.%26atitle%3DRecent%2520advances%2520towards%2520sulfur%2520%2528vi%2529%2520fluoride%2520exchange%2520%2528SUFEX%2529%2520click%2520chemistry%26jtitle%3DJ.%2520Fluorine%2520Chem.%26date%3D2018%26volume%3D213%26spage%3D87%26epage%3D112%26doi%3D10.1016%2Fj.jfluchem.2018.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Sulfonyl fluorides as privileged warheads in chemical biology</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2650</span>â <span class="NLM_lpage">2659</span>, <span class="refDoi">Â DOI: 10.1039/C5SC00408J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC5SC00408J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28706662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVamurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2650-2659&author=A.+Narayananauthor=L.+H.+Jones&title=Sulfonyl+fluorides+as+privileged+warheads+in+chemical+biology&doi=10.1039%2FC5SC00408J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfonyl fluorides as privileged warheads in chemical biology</span></div><div class="casAuthors">Narayanan, Arjun; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2650-2659</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Sulfonyl fluoride electrophiles have found significant utility as reactive probes in chem. biol. and mol. pharmacol.  As warheads they possess the right balance of biocompatibility (including aq. stability) and protein reactivity.  Their functionality is privileged in this regard as they are known to modify not only reactive serines (resulting in their common use as protease inhibitors), but also context-specific threonine, lysine, tyrosine, cysteine and histidine residues.  This review describes the application of sulfonyl fluoride probes across various areas of research and explores new approaches that could further enhance the chem. biol. toolkit.  We believe that sulfonyl fluoride probes will find greater utility in areas such as covalent enzyme inhibition, target identification and validation, and the mapping of enzyme binding sites, substrates and protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAuadIMxOl2bVg90H21EOLACvtfcHk0lj5llAkvmliAw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVamurc%253D&md5=0009feb4068e912b4784fb8411cdb702</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC5SC00408J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC00408J%26sid%3Dliteratum%253Aachs%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DSulfonyl%2520fluorides%2520as%2520privileged%2520warheads%2520in%2520chemical%2520biology%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D2650%26epage%3D2659%26doi%3D10.1039%2FC5SC00408J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debreczeni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tentarelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aquila, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimster, N. P.</span></span> <span> </span><span class="NLM_article-title">A study of the reactivity of S(VI)âF containing warheads with nucleophilic amino-acid side chains under physiological conditions</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">9685</span>â <span class="NLM_lpage">9695</span>, <span class="refDoi">Â DOI: 10.1039/C7OB02028G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC7OB02028G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=29119993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSgsbnI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=9685-9695&author=H.+Mukherjeeauthor=J.+Debreczeniauthor=J.+Breedauthor=S.+Tentarelliauthor=B.+Aquilaauthor=J.+E.+Dowlingauthor=A.+Whittyauthor=N.+P.+Grimster&title=A+study+of+the+reactivity+of+S%28VI%29%E2%80%93F+containing+warheads+with+nucleophilic+amino-acid+side+chains+under+physiological+conditions&doi=10.1039%2FC7OB02028G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A study of the reactivity of S(VI)-F containing warheads with nucleophilic amino-acid side chains under physiological conditions</span></div><div class="casAuthors">Mukherjee, H.; Debreczeni, J.; Breed, J.; Tentarelli, S.; Aquila, B.; Dowling, J. E.; Whitty, A.; Grimster, N. P.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">9685-9695</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Sulfonyl fluorides (SFs) have recently emerged as a promising warhead for the targeted covalent modification of proteins.  Despite numerous examples of the successful deployment of SFs as covalent probe compds., a detailed exploration of the factors influencing the stability and reactivity of SFs has not yet appeared.  In this work we present an extensive study on the influence of steric and electronic factors on the reactivity and stability of the SF and related SVI-F groups.  While SFs react rapidly with N-acetylcysteine, the resulting adducts were found to be unstable, rendering SFs inappropriate for the durable covalent inhibition of cysteine residues.  In contrast, SFs afforded stable adducts with both N-acetyltyrosine and N-acetyllysine; furthermore, we show that the reactivity of arylsulfonyl fluorides towards these nucleophilic amino acids can be predictably modulated by adjusting the electronic properties of the warhead.  These trends were largely conserved when the covalent reaction occurred within a protein binding pocket.  We have also obtained a crystal structure depicting covalent modification of the catalytic lysine of a tyrosine kinase (FGFR1) by the ATP analog 5'-O-3-((fluorosulfonyl)benzoyl)adenosine (m-FSBA).  Highly reactive warheads were demonstrated to be unstable with respect to hydrolysis in buffered aq. solns., indicating that warhead reactivity must be carefully tuned to provide optimal rates of protein modification.  Our results demonstrate that the reactivity of SFs complements that of more commonly studied acrylamides, and we hope that this work spurs the rational design of novel SF-contg. covalent probe compds. and inhibitors, particularly in cases where a suitably positioned cysteine residue is not present.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIyTBiXHscB7Vg90H21EOLACvtfcHk0ljgumDEuJ3Mmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSgsbnI&md5=cf00897a4b0b184e2d68df498ec8cbb7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC7OB02028G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB02028G%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DH.%26aulast%3DDebreczeni%26aufirst%3DJ.%26aulast%3DBreed%26aufirst%3DJ.%26aulast%3DTentarelli%26aufirst%3DS.%26aulast%3DAquila%26aufirst%3DB.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DGrimster%26aufirst%3DN.%2BP.%26atitle%3DA%2520study%2520of%2520the%2520reactivity%2520of%2520S%2528VI%2529%25E2%2580%2593F%2520containing%2520warheads%2520with%2520nucleophilic%2520amino-acid%2520side%2520chains%2520under%2520physiological%2520conditions%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D9685%26epage%3D9695%26doi%3D10.1039%2FC7OB02028G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciuffarin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senatore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isola, M.</span></span> <span> </span><span class="NLM_article-title">Nucleophilic substitution at four-co-ordinate sulphur. Mobility of the leaving group</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_fpage">468</span>â <span class="NLM_lpage">471</span>, <span class="refDoi">Â DOI: 10.1039/p29720000468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2Fp29720000468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADyaE38XpvVyrtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1972&pages=468-471&author=E.+Ciuffarinauthor=L.+Senatoreauthor=M.+Isola&title=Nucleophilic+substitution+at+four-co-ordinate+sulphur.+Mobility+of+the+leaving+group&doi=10.1039%2Fp29720000468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleophilic substitution at four-coordinate sulfur.  Mobility of the leaving group</span></div><div class="casAuthors">Ciuffarin, Ennio; Senatore, Lucio; Isola, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 2:  Physical Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">468-71</span>CODEN:
                <span class="NLM_cas:coden">JCPKBH</span>;
        ISSN:<span class="NLM_cas:issn">0300-9580</span>.
    </div><div class="casAbstract">The kinetics of the reactions of PhNH2, BuNH2, and OH- with PhSO2X (X = I, Br, Cl, F) were detd. and the rate consts. and activation parameters of the displacement of X at 25Â° calcd.  The relative leaving group mobilities of I, Br, and Cl with each nucleophile were the same but the relative mobility of F varied greatly.  The mechanism may involve nucleophile-S bond formation (X = I, Br, Cl) as the rate-detg. step; but S-F bond cleavage greatly influenced the rate of reaction of PhSO2F with PhNH2 or BuNH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1ZZFqEY_XmrVg90H21EOLACvtfcHk0ljgumDEuJ3Mmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE38XpvVyrtw%253D%253D&md5=a1753e416b258cea78f6850fda2d2bf7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2Fp29720000468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29720000468%26sid%3Dliteratum%253Aachs%26aulast%3DCiuffarin%26aufirst%3DE.%26aulast%3DSenatore%26aufirst%3DL.%26aulast%3DIsola%26aufirst%3DM.%26atitle%3DNucleophilic%2520substitution%2520at%2520four-co-ordinate%2520sulphur.%2520Mobility%2520of%2520the%2520leaving%2520group%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1972%26spage%3D468%26epage%3D471%26doi%3D10.1039%2Fp29720000468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolanos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almaden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span> <span> </span><span class="NLM_article-title">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+EGFR+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact â¤ 2.1 Ã 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0ljgumDEuJ3Mmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520EGFR%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D173%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="note"><p class="first last">Arithmetic mean Â± SEM measured from 3 independent biological repeats.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Targeting secondary protein complexes in drug discovery: Studying the druggability and chemical biology of the HSP70/BAG1 complex</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5167</span>â <span class="NLM_lpage">5170</span>, <span class="refDoi">Â DOI: 10.1039/C7CC01376K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=10.1039%2FC7CC01376K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=28439591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=5167-5170&author=L.+E.+Evansauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Targeting+secondary+protein+complexes+in+drug+discovery%3A+Studying+the+druggability+and+chemical+biology+of+the+HSP70%2FBAG1+complex&doi=10.1039%2FC7CC01376K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting secondary protein complexes in drug discovery: studying the druggability and chemical biology of the HSP70/BAG1 complex</span></div><div class="casAuthors">Evans, Lindsay E.; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5167-5170</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Proteins typically carry out their biol. functions as multi-protein complexes, which can significantly affect the affinity of small-mol. inhibitors.  HSP70 is an important target in oncol., so to study its chem. biol. and the drug discovery potential of the HSP70/BAG1 complex, we designed a high-affinity non-nucleotide fluorescence polarization probe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwTl7NEh65w7Vg90H21EOLACvtfcHk0ljftYI6Fwgc0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlOrtrs%253D&md5=f72aa2f8aba028100f2eb1fd45bf336e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2FC7CC01376K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC01376K%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DTargeting%2520secondary%2520protein%2520complexes%2520in%2520drug%2520discovery%253A%2520Studying%2520the%2520druggability%2520and%2520chemical%2520biology%2520of%2520the%2520HSP70%252FBAG1%2520complex%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D5167%26epage%3D5170%26doi%3D10.1039%2FC7CC01376K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macias, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgognoni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokurno, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matassova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surgenor, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A. J.</span></span> <span> </span><span class="NLM_article-title">Adenosine-derived inhibitors of 78 kda glucose regulated protein (GRP78) ATPase: Insights into isoform selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4034</span>â <span class="NLM_lpage">4041</span>, <span class="refDoi">Â DOI: 10.1021/jm101625x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101625x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlWrtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4034-4041&author=A.+T.+Maciasauthor=D.+S.+Williamsonauthor=N.+Allenauthor=J.+Borgognoniauthor=A.+Clayauthor=Z.+Danielsauthor=P.+Dokurnoauthor=M.+J.+Drysdaleauthor=G.+L.+Francisauthor=C.+J.+Grahamauthor=R.+Howesauthor=N.+Matassovaauthor=J.+B.+Murrayauthor=R.+Parsonsauthor=T.+Shawauthor=A.+E.+Surgenorauthor=L.+Terryauthor=Y.+Wangauthor=M.+Woodauthor=A.+J.+Massey&title=Adenosine-derived+inhibitors+of+78+kda+glucose+regulated+protein+%28GRP78%29+ATPase%3A+Insights+into+isoform+selectivity&doi=10.1021%2Fjm101625x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine-Derived Inhibitors of 78 kDa Glucose Regulated Protein (Grp78) ATPase: Insights into Isoform Selectivity</span></div><div class="casAuthors">Macias, Alba T.; Williamson, Douglas S.; Allen, Nicola; Borgognoni, Jenifer; Clay, Alexandra; Daniels, Zoe; Dokurno, Pawel; Drysdale, Martin J.; Francis, Geraint L.; Graham, Christopher J.; Howes, Rob; Matassova, Natalia; Murray, James B.; Parsons, Rachel; Shaw, Terry; Surgenor, Allan E.; Terry, Lindsey; Wang, Yikang; Wood, Mike; Massey, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4034-4041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">78 KDa glucose-regulated protein (Grp78) is a heat shock protein (HSP) involved in protein folding that plays a role in cancer cell proliferation.  Binding of adenosine-derived inhibitors to Grp78 was characterized by surface plasmon resonance and isothermal titrn. calorimetry.  The most potent compds. were 13 (VER-155008) with KD = 80 nM and 14 with KD = 60 nM.  X-ray crystal structures of Grp78 bound to ATP, ADPnP, and adenosine deriv. 10 revealed differences in the binding site between Grp78 and homologous proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIYGKX4tuuUrVg90H21EOLACvtfcHk0ljftYI6Fwgc0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlWrtr0%253D&md5=b382510c6bd1e382c9a42e81e0ed3362</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm101625x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101625x%26sid%3Dliteratum%253Aachs%26aulast%3DMacias%26aufirst%3DA.%2BT.%26aulast%3DWilliamson%26aufirst%3DD.%2BS.%26aulast%3DAllen%26aufirst%3DN.%26aulast%3DBorgognoni%26aufirst%3DJ.%26aulast%3DClay%26aufirst%3DA.%26aulast%3DDaniels%26aufirst%3DZ.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFrancis%26aufirst%3DG.%2BL.%26aulast%3DGraham%26aufirst%3DC.%2BJ.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DShaw%26aufirst%3DT.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26aulast%3DTerry%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DMassey%26aufirst%3DA.%2BJ.%26atitle%3DAdenosine-derived%2520inhibitors%2520of%252078%2520kda%2520glucose%2520regulated%2520protein%2520%2528GRP78%2529%2520ATPase%253A%2520Insights%2520into%2520isoform%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4034%26epage%3D4041%26doi%3D10.1021%2Fjm101625x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKR','PDB','5MKR'); return false;">PDB: 5MKR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MKS','PDB','5MKS'); return false;">PDB: 5MKS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IO8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IO8','PDB','4IO8'); return false;">PDB: 4IO8</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i35"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01709">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_24450"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01709?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01709</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Covalent FP worked example (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_002.csv">CSV</a>)</p></li><li><p class="inline">Computational chemistry methods, worked example of the covalent FP-assay, covalent FP and intact-protein MS figures, and NMR spectra of final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_001.xlsx">jm9b01709_si_001.xlsx (30.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_002.csv">jm9b01709_si_002.csv (0.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01709/suppl_file/jm9b01709_si_003.pdf">jm9b01709_si_003.pdf (3.3 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01709%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01709" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991fca09b33c76","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
