
<html lang="en"     class="pb-page"  data-request-id="424a8d15-a96a-4770-a940-c3ff900a3bf2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-3;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b01674;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)" /></meta><meta name="dc.Creator" content="H. Ümit  Kaniskan" /></meta><meta name="dc.Creator" content="Mohammad S.  Eram" /></meta><meta name="dc.Creator" content="Kehao  Zhao" /></meta><meta name="dc.Creator" content="Magdalena M.  Szewczyk" /></meta><meta name="dc.Creator" content="Xiaobao  Yang" /></meta><meta name="dc.Creator" content="Keith  Schmidt" /></meta><meta name="dc.Creator" content="Xiao  Luo" /></meta><meta name="dc.Creator" content="Sean  Xiao" /></meta><meta name="dc.Creator" content="Miao  Dai" /></meta><meta name="dc.Creator" content="Feng  He" /></meta><meta name="dc.Creator" content="Irene  Zang" /></meta><meta name="dc.Creator" content="Ying  Lin" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Elena  Dobrovetsky" /></meta><meta name="dc.Creator" content="David  Smil" /></meta><meta name="dc.Creator" content="Sun-Joon  Min" /></meta><meta name="dc.Creator" content="Jennifer  Lin-Jones" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Creator" content="Peter  Atadja" /></meta><meta name="dc.Creator" content="En  Li" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Feng  Liu" /></meta><meta name="dc.Creator" content="Zhengtian  Yu" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins. It is crucial for maturation of ribosomes and has been implicated in several diseases. We recently disclosed a..." /></meta><meta name="Description" content="PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins. It is crucial for maturation of ribosomes and has been implicated in several diseases. We recently disclosed a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 5, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01674" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01674" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01674" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01674" /></link>
        
    
    

<title>Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01674" /></meta><meta property="og:title" content="Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0019.jpeg" /></meta><meta property="og:description" content="PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins. It is crucial for maturation of ribosomes and has been implicated in several diseases. We recently disclosed a highly potent, selective, and cell-active allosteric inhibitor of PRMT3, compound 4. Here, we report comprehensive structure–activity relationship studies that target the allosteric binding site of PRMT3. We conducted design, synthesis, and evaluation of novel compounds in biochemical, selectivity, and cellular assays that culminated in the discovery of 4 and other highly potent (IC50 values: ∼10–36 nM), selective, and cell-active allosteric inhibitors of PRMT3 (compounds 29, 30, 36, and 37). In addition, we generated compounds that are very close analogs of these potent inhibitors but displayed drastically reduced potency as negative controls (compounds 49–51). These inhibitors and negative controls are valuable chemical tools for the biomedical community to further investigate biological functions and disease associations of PRMT3." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01674"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01674">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01674&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01674&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01674&amp;href=/doi/10.1021/acs.jmedchem.7b01674" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1204-1217</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01654" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01440" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=H.+%C3%9Cmit++Kaniskan">H. Ümit Kaniskan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>¶</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad+S.++Eram">Mohammad S. Eram</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>¶</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kehao++Zhao">Kehao Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>□</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena+M.++Szewczyk">Magdalena M. Szewczyk</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaobao++Yang">Xiaobao Yang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Keith++Schmidt">Keith Schmidt</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiao++Luo">Xiao Luo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sean++Xiao">Sean Xiao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Miao++Dai">Miao Dai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++He">Feng He</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Irene++Zang">Irene Zang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Lin">Ying Lin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Dobrovetsky">Elena Dobrovetsky</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Smil">David Smil</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sun-Joon++Min">Sun-Joon Min</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-0867-4416" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer++Lin-Jones">Jennifer Lin-Jones</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Atadja">Peter Atadja</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=En++Li">En Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Liu">Feng Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-2669-5448" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhengtian++Yu">Zhengtian Yu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Center for Chemical Biology and Drug Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Novartis Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">DiscoveRx Corporation, Fremont, California 94538, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Toronto, ON M5S 1A8, Canada</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Department of Medical Biophysics, University of Toronto and Princess Margaret Cancer Centre, 101 College Street, MaRS South Tower, Suite 707, Toronto, ON M5G 1L7, Canada</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">∇</span> <span class="aff-text">Jiangsu Key Laboratory of Translational Research and Therapy for Neuro-Psycho-Diseases and Department of Medicinal Chemistry, College of Pharmaceutical Sciences, Soochow University, Suzhou, Jiangsu 215123, China</span></div><div class="corresp-info"><strong>*</strong>(F.L.) E-mail: <a href="/cdn-cgi/l/email-protection#294f45405c1b695a5c4d48074c4d5c074a47"><span class="__cf_email__" data-cfemail="27414b4e5215675452434609424352094449">[email protected]</span></a>. Phone: +86 (512) 65882569.</div><div class="corresp-info"><strong>*</strong>(Z.Y.) E-mail: <a href="/cdn-cgi/l/email-protection#d5afbdb0bbb2a1bcb4bbfbaca095bbbaa3b4a7a1bca6fbb6bab8"><span class="__cf_email__" data-cfemail="403a28252e273429212e6e3935002e2f3621323429336e232f2d">[email protected]</span></a>. Phone: 18621082583.</div><div class="corresp-info"><strong>*</strong>(M.V.) E-mail: <a href="/cdn-cgi/l/email-protection#9ff2b1e9fafbfefbf6dfeaebf0edf0f1ebf0b1fcfe"><span class="__cf_email__" data-cfemail="35581b43505154515c7540415a475a5b415a1b5654">[email protected]</span></a>. Phone: (416) 976-0897.</div><div class="corresp-info"><strong>*</strong>(J.J.) E-mail: <a href="/cdn-cgi/l/email-protection#7e14171f10501417103e130d0d13501b1a0b"><span class="__cf_email__" data-cfemail="7a10131b14541013143a17090917541f1e0f">[email protected]</span></a>. Phone: (212) 659-8699.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01674&amp;href=/doi/10.1021%2Facs.jmedchem.7b01674" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1204–1217</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 15, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 November 2017</li><li><span class="item_label"><b>Published</b> online</span>5 January 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01674" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01674</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01674"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3588</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01674" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;H.&quot;,&quot;last_name&quot;:&quot;Ümit Kaniskan&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;S. Eram&quot;},{&quot;first_name&quot;:&quot;Kehao&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;M. Szewczyk&quot;},{&quot;first_name&quot;:&quot;Xiaobao&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Keith&quot;,&quot;last_name&quot;:&quot;Schmidt&quot;},{&quot;first_name&quot;:&quot;Xiao&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Sean&quot;,&quot;last_name&quot;:&quot;Xiao&quot;},{&quot;first_name&quot;:&quot;Miao&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;Zang&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Dobrovetsky&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Smil&quot;},{&quot;first_name&quot;:&quot;Sun-Joon&quot;,&quot;last_name&quot;:&quot;Min&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;Lin-Jones&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Atadja&quot;},{&quot;first_name&quot;:&quot;En&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Zhengtian&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;1204-1217&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01674&quot;},&quot;abstract&quot;:&quot;PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins. It is crucial for maturation of ribosomes and has been implicated in several diseases. We recently disclosed a highly potent, selective, and cell-active allosteric inhibitor of PRMT3, compound 4. Here, we report comprehensive structure–activity relationship studies that target the allosteric binding site of PRMT3. We conducted design, synthesis, and evaluation of novel compounds in biochemical, selectivity, and cellular assays that culminated in the discovery of 4 and other highly potent (IC50 values: ∼10–36 nM), selective, and cell-active allosteric inhibitors of PRMT3 (compounds 29, 30, 36, and 37). In addition, we generated compounds that are very close analogs of these potent inhibitors but displayed drastically reduced potency as negative controls (compounds 49–51). These inhibitors and negative controls are valuable chemical tools for the biomedical community to further investigate biological functions and disease ass&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01674&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01674" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01674&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01674" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01674&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01674" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01674&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01674&amp;href=/doi/10.1021/acs.jmedchem.7b01674" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01674" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01674" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01674%26sid%3Dliteratum%253Aachs%26pmid%3D29244490%26genre%3Darticle%26aulast%3DKaniskan%26date%3D2018%26atitle%3DDiscovery%2Bof%2BPotent%2Band%2BSelective%2BAllosteric%2BInhibitors%2Bof%2BProtein%2BArginine%2BMethyltransferase%2B3%2B%2528PRMT3%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D3%26spage%3D1204%26epage%3D1217%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292164" title="Pyrrolidines">Pyrrolidines</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/jmcmar.2018.61.issue-3/20180208/jmcmar.2018.61.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins. It is crucial for maturation of ribosomes and has been implicated in several diseases. We recently disclosed a highly potent, selective, and cell-active allosteric inhibitor of PRMT3, compound <b>4</b>. Here, we report comprehensive structure–activity relationship studies that target the allosteric binding site of PRMT3. We conducted design, synthesis, and evaluation of novel compounds in biochemical, selectivity, and cellular assays that culminated in the discovery of <b>4</b> and other highly potent (IC<sub>50</sub> values: ∼10–36 nM), selective, and cell-active allosteric inhibitors of PRMT3 (compounds <b>29</b>, <b>30</b>, <b>36</b>, and <b>37</b>). In addition, we generated compounds that are very close analogs of these potent inhibitors but displayed drastically reduced potency as negative controls (compounds <b>49</b>–<b>51</b>). These inhibitors and negative controls are valuable chemical tools for the biomedical community to further investigate biological functions and disease associations of PRMT3.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein arginine methyltransferase 3 (PRMT3) is a type I PRMT that catalyzes mono- and asymmetric dimethylation of arginine residues.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Ribosomal protein S2 (rpS2) was identified as the major substrate of PRMT3 via its interaction with PRMT3 zinc finger domain in mammalian cells.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> PRMT3 plays a role in ribosome biosynthesis. However, the molecular mechanism by which PRMT3 influences ribosomal biosynthesis remains unclear.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Very recently, an extraribosomal complex comprising PRMT3, rpS2, and human programmed cell-death 2-like (PDCD2L) protein was identified.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> While PRMT3 is localized exclusively in the cytoplasm,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> it has been shown that in cells treated with palmitic acid or T0901317 (a liver X receptor α (LXRα) agonist), PRMT3 colocalizes with LXRα in the cell nucleus, regulating hepatic lipogenesis.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> However, this effect appears to be independent of the PRMT3 methyltransferase activity. While rpS2 is the primary substrate of PRMT3, it is not the sole substrate. PRMT3 along with PRMT1 methylates the recombinant mammalian nuclear poly(A)-binding protein (PABPN1) and has been implicated in oculopharyngeal muscular dystrophy, which is caused by polyalanine expansion in PABPN1.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> A protein complex comprising the von Hippel–Lindau (VHL) tumor suppressor protein, PRMT3, and ARF (alternative reading frame) methylates p53.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Importantly, the tumor suppressor DAL-1 (differentially expressed in adenocarcinoma of the lung, also known as 4.1B) interacts with PRMT3 and consequently inhibits its methyltransferase activity, suggesting a possible role of PRMT3 regulation in tumor growth.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The interaction between DAL-1 and PRMT3 in the induction of apoptosis in MCF-7 cells suggests that this interaction is likely to be an important modulator of the apoptotic pathway and can be critical to controlling tumorigenesis in breast cancer cells.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> It has also been shown that PRMT3 methylates a histone peptide (H4 1–24) <i>in vitro</i>.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Modified histone H4R3 is associated with increased transcription of a number of genes, including those under control of estrogen and androgen receptors.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14-16)</a> Furthermore, PRMT3 expression levels are elevated in myocardial tissues from patients with atherosclerosis, potentially implicating the involvement of PRMT3.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Additionally, PRMT3 function has been reported to be essential for dendritic spine maturation in rats.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> A recent study suggests that PRMT3 mediates the preventive effects of irisin against lipogenesis and oxidative stress.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">Our group embarked on research efforts to discover potent, selective, and cell-active inhibitors of PRMT3 as chemical tools for better understanding of biology and function of this understudied protein methyltransferase. We previously reported the discovery of a novel allosteric binding site of PRMT3 and the first selective, allosteric inhibitors of PRMT3 (compounds <b>1</b>–<b>3</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) and subsequently disclosed the discovery of SGC707 (<b>4</b>) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), a highly potent, selective, and cell-active allosteric inhibitor of PRMT3.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20-22)</a> Here, we describe the design and synthesis of a large set of novel analogs and evaluation of these compounds in biochemical, selectivity, and cellular assays. This comprehensive structure–activity relationship (SAR) study resulted in the identification of multiple highly potent, selective, and cell-active allosteric inhibitors of PRMT3 including compound <b>4</b>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design and synthesis of highly potent inhibitors of PRMT3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40820" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40820" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our earlier efforts resulted in the identification of selective small-molecule inhibitors of PRMT3 (compounds <b>2</b> and <b>3</b>) starting from a hit, compound <b>1</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> X-ray crystal structures of <b>1</b> and <b>2</b> in complex with PRMT3 were obtained and showed that these inhibitors occupied a novel allosteric binding site (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG">4HSG</a>). These cocrystal structures revealed that the left-hand side (LHS) bicyclic benzothiadiazole moiety fits tightly in the allosteric pocket and the middle nitrogen atom forms a hydrogen bond with T466 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The urea linker is located at the entrance of the cavity and forms hydrogen bonds with the guanidine of R396 and the carboxylate of E422 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). In addition, the right-hand side (RHS) moiety extends out of the allosteric binding pocket and makes hydrophobic interactions with a surface composed of the side chains from two different subunits of the PRMT3 homodimer. Both structural and SAR data clearly indicate that the middle urea region is essential for the affinity, and replacement of this group with its bioisosteres and more rigid analogs did not yield any inhibitors with improved potency.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Therefore, in the current study the middle urea region of this scaffold was kept unmodified. However, further optimization of the RHS moiety led to the discovery of <b>3</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Herein, we further optimized both the LHS and RHS moieties of this scaffold to achieve improved potency for inhibiting PRMT3. First, we conducted a scaffold hopping exercise by using the benzothiadiazole ring of <b>3</b> as a query for the allosteric pocket. A hydrogen-bond constraint was imposed in this scaffold hopping study to preserve the important hydrogen-bond interaction between the inhibitors and the hydroxyl group of T466. As a result of this exercise,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> isoquinoline (compound <b>5</b>), isobenzofuran-1-one (compound <b>6</b>), and quinazoline (compound <b>7</b>) groups were selected for experimental validation (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Reported cocrystal structure of compound <b>1</b> in complex with PRMT3 indicating key hydrogen bonding interactions (magenta dotted lines) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitors with Different LHS Bicyclic Ring Systems</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0009.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_p">The RHS of compound <b>3</b>, piperidineamide, was kept exactly the same in these newly synthesized analogs to accurately compare the effect of the different LHS bicyclic heteroaromatic rings on potency. The isoquinoline containing analog (compound <b>5</b>) with IC<sub>50</sub> of 84 ± 5 nM showed a small improvement over the parent compound, <b>3</b> (IC<sub>50</sub> = 134 ± 5 nM). However, compounds <b>6</b> and <b>7</b> were around 6-fold less potent as compared to <b>3</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Therefore, the isoquinoline bicyclic ring system was taken forward as the LHS moiety for further optimization. Compound <b>8</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) was also prepared to confirm the importance of the positioning and hydrogen bonding of the isoquinoline nitrogen with T466. This bicyclic ring is still an isoquinoline but substituted at the 7-position instead of the 6-position (isoquinoline numbering) in effect walking the nitrogen to the adjacent position of compound <b>5</b>. As expected, this modification resulted in ablation of inhibitory activity.</div><div class="NLM_p">As the 6-substituted isoquinoline is an optimal LHS bicycle, we then turned our attention to optimizing the RHS moiety of the scaffold. We first revisited saturated aliphatic groups as the RHS functionality inspired from our initial studies (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> Compound <b>9</b>, a cyclohexenylethyl group containing analog, corresponding to the RHS region of the original hit (compound <b>1</b>) was 5-fold less potent than <b>5</b>, displaying IC<sub>50</sub> of 421 ± 29 nM. As a logical extension of this compound, we also synthesized the fully saturated, cyclohexylethyl analog (<b>10</b>) as well as derivatives containing different ring sizes. Compound <b>10</b> (IC<sub>50</sub> = 540 ± 54 nM) was very similar to <b>9</b> in potency. The cyclopentane-bearing analog (<b>11</b>) showed slight improvement in potency with IC<sub>50</sub> of 295 ± 43 nM, while the cyclopropane-containing analog <b>12</b> was virtually inactive (IC<sub>50</sub> > 8000 nM). These results indicated that hydrophobicity of the RHS moiety plays an important role in potency, and cyclohexyl and cylopentyl rings are preferred compared to the cyclopropyl group. Replacing the cyclohexyl group with a phenyl ring (compound <b>13</b>) resulted in a decreased potency (IC<sub>50</sub> = 833 ± 66 nM), underlying the importance of changes to the hydrophobicity of the RHS moiety. On the basis of these results, compound <b>10</b> was further investigated to improve the potency. A docking study with this compound hinted that a substituent at the C1 position of the cyclohexyl group would be a favorable position from which E422 might be possibly reached for further interactions. Therefore, compound <b>14</b>, featuring a methyleneamine substituted cyclohexyl group, was synthesized. Interestingly, compound <b>14</b> displayed a modest improvement in potency with IC<sub>50</sub> of 291 ± 36 nM, compared to compound <b>10</b>. These compounds were synthesized following the previously published synthetic route.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitors with Saturated Aliphatic Groups as the RHS Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0010.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0011.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">Since no significant improvement in potency was achieved with compounds <b>9</b>–<b>14</b> featuring aliphatic groups as the RHS moiety, we decided to keep the amide functionality as in compound <b>5</b> and further investigate the substituents on the nitrogen atom (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) as the amide group could form direct or water-mediated contacts with the side chain of K392.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> We first synthesized new amide derivatives of compound <b>5</b>, by replacing the six-membered piperidine ring with the four-membered azetidine ring (compound <b>15</b>), five-membered pyrrolidine ring (compound <b>4</b>), and seven-membered azepane ring (compound <b>16</b>). While the azetidine amide analog <b>15</b> displayed only a slight improvement in potency (IC<sub>50</sub> = 61 ± 8 nM), the pyrrolidine amide <b>4</b> (IC<sub>50</sub> = 19 ± 1 nM) and azepane amide <b>16</b> (IC<sub>50</sub> = 17 ± 2 nM) were ∼5-fold more potent than <b>5</b> in inhibiting PRMT3. The six-membered ring analogs of <b>5</b> such as 4,4-difluoro piperidine amide <b>17</b> (IC<sub>50</sub> = 35 ± 1 nM) showed a more than 2-fold improvement over the unsubstituted piperidine amide <b>5</b>. However, exchanging the piperidine ring with the 4-methylpiperizine ring (compound <b>18</b>), which alters the electronic nature and polarity of the ring system, resulted in the loss of the inhibitory activity. We also designed and synthesized noncyclic di- and monosubstituted amide derivatives (compounds <b>19</b>–<b>26</b>). The dimethyl and diethyl amide derivatives <b>19</b> and <b>20</b> displayed reduced potency with IC<sub>50</sub> of 150 ± 11 and 114 ± 3 nM, respectively. The <i>N</i>-cyclopentyl, <i>N</i>-methyl amide <b>21</b> (IC<sub>50</sub> = 87 ± 5 nM) was as potent as compound <b>5</b>. The Weinreb amide derivative <b>22</b>, however, was around 6-fold less potent than compound <b>5</b>. The monosubstituted amides were also investigated (compounds <b>23</b>–<b>26</b>). While <i>N</i>-methyl amide <b>23</b> displayed significantly weaker inhibitory effect, the cyclopropyl (<b>24</b>), cyclopentyl (<b>25</b>), and cyclohexyl (<b>26</b>) amide derivatives showed reduced potency as the ring size increased. Taken together, these results suggest that cyclic amides are preferred RHS moieties, which possess balanced steric and hydrophobic properties interacting with PRMT3. We, therefore, designated the pyrrolidine amide as the RHS moiety for the rest of the SAR study.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Inhibitors Containing Different RHS Amide Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0012.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0013.gif" alt="" id="fx6" /></img><div></div></div><div class="NLM_p">The compounds in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> were synthesized via the general route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The synthesis started with reacting commercially available 6-aminoisoquinoline and ethyl isocyanatoacetate to obtain the desired ethyl ester <b>27</b>. The hydrolysis of the ethyl ester <b>27</b> resulted in the carboxylic acid <b>28</b>, which was used as the key intermediate to perform amide coupling reactions with various amines to yield the desired amide analogs (<b>4</b>, <b>5</b>, and <b>15</b>–<b>26</b>). Detailed reaction conditions, yields, and characterization data for final compounds are reported in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route for the Preparation of Compounds Listed in <a class="ref internalNav" href="#tbl3" aria-label="Tables 3">Tables 3</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we focused our attention on analogs of compound <b>4</b> (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>), containing substituted pyrrolidine (compounds <b>29</b>–<b>31</b>) and fused bicyclic (compounds <b>32</b> and <b>33</b>) and bridged bicyclic moieties (compounds <b>34</b> and <b>35</b>). These compounds were again prepared according to the synthetic route outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The 3,3,4,4-tetrafluoro pyrrolidine derivative (compound <b>29</b>) as well as 2,5-dimethylpyrrolidine (a mixture of cis and trans isomers) (compound <b>30</b>) analog showed very similar potency compared to compound <b>4</b>. However, replacing pyrrolidine with <span class="smallcaps smallerCapital">l</span>-proline (compound <b>31</b>) resulted in around 9-fold drop in potency. The fused 5,5-bicyclic ring system (compound <b>32</b>) was also tolerated, albeit with reduced potency (IC<sub>50</sub> = 55 ± 5 nM). Interestingly, the benzene-fused pyrrolidine ring (isoindoline amide, compound <b>33</b>) led to a complete loss of the inhibitory effect. Finally, 7-azabicyclo[2.2.1]heptane (compound <b>34</b>) and 8-azabicyclo[3.2.1]octane (compound <b>35</b>), two bridged bicyclic pyrrolidine containing moieties, did not result in any improvement of potency compared to compound <b>4</b>, with IC<sub>50</sub> of 46 ± 5 and 22 ± 4 nM, respectively. These results indicate that small substituents such as fluoro and methyl groups on the pyrrolidine ring as well as relatively flexible hydrophobic bicyclic ring systems are tolerated.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Inhibitors with Modified Pyrrolidine Amide Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0014.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0015.gif" alt="" id="fx8" /></img><div></div></div><div class="NLM_p">The results summarized in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and discussed above have shown that analogs of <b>4</b>, namely, compounds <b>29</b>, <b>30</b>, <b>32</b>, <b>34</b>, and <b>35</b>, potently inhibited PRMT3 with IC<sub>50</sub> values of around 20–50 nM. Although these analogs did not display improved potency compared to compound <b>4</b>, these substituted pyrrolidine groups are valuable alternatives to the unsubstituted pyrrolidine group (compound <b>4</b>). After completing optimization of the RHS moiety, we further investigated the LHS isoquinoline ring. Analysis of the crystal structure of PRMT3 in complex with compound <b>4</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RYL">4RYL</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> suggested that there is room in the binding pocket to tolerate a relatively small substituent at the 1-, 3-, 7-, and 8-positions of the isoquinoline ring system. Our structural analysis also suggested that a substituent at the 4- and 5-positions of the isoquinoline ring would not be tolerated. Therefore, we designed and synthesized the corresponding substituted isoquinoline analogs (compounds <b>36</b>–<b>40</b>) to determine whether potency could further be enhanced (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). For example, compounds <b>36</b> and <b>37</b> featuring small 7-fluoro and 7-methyl substituents were prepared. Compound <b>36</b> showed similar potency as <b>4</b>, while <b>37</b>, which has a slightly larger methyl substituent, was around 2-fold less potent. Compound <b>38</b>, however, displayed significant potency loss (about 10-fold), indicating that the methyl group at the 1-position of the isoquinoline ring is not preferred. Interestingly, the 3-fluoro substituted analog <b>39</b> displayed almost 2-fold higher potency with IC<sub>50</sub> of 10 ± 1 nM. This result suggests that the electronic modulation of the isoquinoline ring by a fluoro group does not have significant impact on the hydrogen bonding ability of the isoquinoline with T466 and a small substituent such as the fluoro group at the 3-position enhances potency, consistent with our structural analysis. In addition, we synthesized the 8-chloroisoquinoline derivative <b>40</b>, which exhibited similar potency as compound <b>4</b>. While the result obtained for compound <b>39</b> was consistent with our predication based on the analysis of the crystal structure of the PRMT3–compound <b>4</b> complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RYL">4RYL</a>), it was surprising that a small substituent such as the fluoro group at the 7-position (compound <b>36</b>) or the chloro group at the 8-position (compound <b>40</b>) did not increase potency and a slightly larger substituent such as the methyl group at the 1- and 7-positions (compounds <b>37</b> and <b>38</b>) reduced potency. More comprehensive structural analyses such as molecular dynamics simulation are needed to explain the SAR results. Nevertheless, the fluoro or chloro substituent could potentially improve metabolic stability of these compounds (<b>36</b>, <b>39</b>, and <b>40</b>), which are interesting alternative PRMT3 inhibitors to compound <b>4</b>. Overall, these results have demonstrated that a small substituent at the isoquinoline ring can be tolerated but has limited impact on enhancing potency.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Inhibitors with Substituted Isoquinolines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0016.gif" alt="" id="fx9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0017.gif" alt="" id="fx10" /></img><div></div></div><div class="NLM_p">The substituted 6-amino isoquinoline derivatives used for the synthesis of compounds <b>36</b>–<b>40</b> (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) were not commercially available. Therefore, we devised synthetic routes and prepared these substituted 6-amino isoquinolines as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The synthesis of 6-amino-7-fluoroisoquinoline (<b>43</b>), 6-amino-7-methylisoquinoline (<b>44</b>), and 6-amino-1-methylisoquinoline (<b>45</b>) started with reductive amination reactions of amino acetaldehyde dimethyl acetal with the corresponding 4-bromo aryl aldehyde or methyl aryl ketone to give amino dimethyl acetals <b>41</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A). The intermediates <b>41</b> were converted to the sulfonamides <b>42</b>, via tosylation, which were then treated with aluminum chloride to yield the desired 6-bromoisoquinolines (<b>43</b>). These substituted 6-bromoisoquinolines were then converted to the desired 6-aminoisoquinolines (<b>44</b>–<b>46</b>) via aryl amination reactions (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A). The 3-fluoro-6-aminoisoquinoline (<b>47</b>) was synthesized starting from the commercially available 3-amino-6-bromoisoquinoline in two steps via the Balz–Schiemann reaction<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> followed by an aryl amination (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>B). As shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>C, 6-amino-8-chloroisoquinoline (<b>48</b>) was prepared in six steps. The commercially available 5-aminoisoquinoline was first acetylated and then chlorinated to install a chloro group at the 8-position. Bromination at the 6-position was achieved by using dibromoisocyanuric acid. Deacetylation followed by reductive diazotization resulted in 6-amino-8-chloroisoquinoline (<b>48</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>C). Intermediates <b>44</b>–<b>48</b> were then used to prepare compounds <b>36</b>–<b>40</b> according to the synthetic route outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes for Preparing Intermediates <b>44</b>–<b>48</b> for Synthesis of Compounds <b>36</b>–<b>40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, we designed and synthesized several close analogs of compound <b>4</b> to serve as negative controls for chemical biology studies. As described earlier, the middle urea region of these PRMT3 inhibitors forms the key hydrogen-bonding interactions with E422 of PRMT3. We therefore predicted that taking either of these hydrogen-bonding interactions away by methylating either nitrogen atom of the urea would drastically decrease PRMT3 inhibition. Indeed, as shown in <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>, compound <b>49</b> displayed markedly diminished inhibitory activity (IC<sub>50</sub> = 2594 ± 129 nM), while compound <b>50</b> was completely inactive (IC<sub>50</sub> > 50 000 nM). Furthermore, the nitrogen atom in the isoquinoline ring of compound <b>4</b> forms a key hydrogen bond with T466 of PRMT3 in the crystal structure of the PRMT3–compound <b>4</b> complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RYL">4RYL</a>). Thus, we replaced the isoquinoline ring of <b>4</b> with the naphthalene ring (compound <b>51</b> (<b>XY1</b>)),<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> effectively removing the critical hydrogen bond with T466. As we reported previously,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> compound <b>51</b> displayed no inhibition of the PRMT3 catalytic activity in biochemical assays.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Compounds Prepared as Negative Controls</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0018.gif" alt="" id="fx11" /></img><div></div></div><div class="NLM_p">We previously reported that compound <b>4</b> was more than 200-fold selective for PRMT3 over 31 other methyltransferases and more than 250 kinases, GPCRs, ion channels, and transporters.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Similarly, inhibitors <b>29</b>, <b>30</b>, and <b>36</b> were selective for PRMT3 over 31 other lysine methyltransferases, arginine methyltransferases, and DNA and RNA methyltransferases (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> and <a class="ref internalNav" href="#notes-5" aria-label="Supporting Information">Supporting Information</a>). In addition, <b>29</b>, <b>30</b>, and <b>36</b> were tested in a CEREP selectivity panel consisting of 55 protein targets (47 GPCRs, five ion channels, and three transporters) and did not show any significant off-target activities (% of inhibition <50% at 10 μM). It is of note that the cocrystal structure of compound <b>4</b> in complex with PRMT3 reported recently (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RYL">4RYL</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> clearly shows that this inhibitor binds the same allosteric pocket as earlier inhibitors (compounds <b>1</b> and <b>2</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG">4HSG</a>)).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibitor <b>36</b> is highly selective for PRMT3 over 31 other methyltransferases. The selectivity data for compounds <b>29</b> and <b>30</b> are shown in the <a class="ref internalNav" href="#notes-5" aria-label="Supporting Information">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To establish the target engagement of PRMT3 inhibitors in cells (namely, inhibitors <b>4</b>, <b>29</b>, <b>36</b>, and <b>37</b>), we used an InCELL Hunter Assay, which measures intracellular binding of inhibitors to the methyltransferase domain of PRMT3 in cell lines expressing the methyltransferase domain of PRMT3 tagged with a short fragment of β-galactosidase (ePL). Binding of a compound to ePL–PRMT3 increases the fusion protein half-life. Inhibitors <b>4</b>, <b>29</b>, <b>36</b>, and <b>37</b> stabilized PRMT3 in A549 cells, a human lung carcinoma cell line, with EC<sub>50</sub> values of 2.0, 2.7, 1.6, and 4.9 μM, respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, top). The same assay was performed in HEK293 cells and these compounds displayed EC<sub>50</sub> values of 1.8, 3.1, 2.7, and 5.2 μM, respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, bottom). Compound <b>51</b> was used as a negative control in these assays. As expected, no stabilization was observed with this compound.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. InCELL Hunter Assay results of compounds <b>4</b>, <b>29</b>, <b>36</b>, <b>37</b>, and <b>51</b> in A549 and HEK cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, to establish whether these PRMT3 inhibitors can inhibit the PRMT3 catalytic activity in cells, we examined their effects on H4R3 asymmetric dimethylation. Since methylated arginine residues have relatively slow turnover, we overexpressed human Flag-tagged PRMT3 and followed the methylation of both endogenous H4 and exogenously introduced GFP-tagged H4. As we previously reported, overexpressed PRMT3 increased the endogenous H4R3me2a from the baseline levels, and compound <b>4</b> effectively inhibited this increase with an IC<sub>50</sub> of 225 nM.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The asymmetric dimethylation of exogenous H4R3 was also inhibited by compound <b>4</b> (IC<sub>50</sub> = 91 nM), indicating that this inhibitor has robust cellular effect.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Similarly, as shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, compounds <b>29</b>, <b>30</b>, and <b>36</b> inhibited the exogenous asymmetric dimethylation of H4R3 (IC<sub>50</sub> = 240, 184, and 134 nM, respectively). The dependency on the transfected PRMT3 catalytic activity was determined by using the catalytically dead PRMT3 mutant (E335Q) that did not affect endogenous or exogenous H4R3me2a levels and therefore was used to establish the baseline levels of the mark. It is of note that effects of 1 μM compounds <b>29</b>, <b>30</b>, and <b>36</b> matched with that of the catalytically dead PRMT3 mutant E335Q.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular inhibitory activity of compounds <b>29</b>, <b>30</b>, and <b>36</b>. HEK293 cells were cotransfected with FLAG tagged PRMT3 (wt) or its catalytic mutant (mt) and GFP-tagged histone H4 and treated with different concentrations of compounds, as indicated. Total cell lysates were collected 24 h post inhibitor treatment and analyzed for H4R3me2a, GFP, and FLAG levels by Western blotting. The graphs represent nonlinear fits of H4R3me2a fluorescence intensities normalized to GFP. The results are the averages of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40842" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40842" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we conducted comprehensive SAR studies, starting from early inhibitors <b>1</b>–<b>3</b> and culminating in the discovery of highly potent (IC<sub>50</sub> values = ∼10–36 nM), selective, and cell-active allosteric inhibitors of PRMT3 (inhibitors <b>4</b>, <b>29</b>, <b>30</b>, <b>36</b>, and <b>37</b>). In addition, we generated compounds that are very close analogs of these potent inhibitors but that displayed drastically diminished potency as negative controls (compounds <b>49</b>–<b>51</b>). The new inhibitors (compounds <b>29</b>, <b>30</b>, and <b>36</b>) were highly selective for PRMT3 over 31 other methyltransferases and 55 other protein targets. In cell-based assays, compounds <b>29</b>, <b>30</b>, and <b>36</b> engaged PRMT3 and potently inhibited its methyltransferase activity (<a class="ref internalNav" href="#fig4" aria-label="Figures 4">Figures 4</a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). These inhibitors and negative controls are excellent chemical tools for the biomedical community to further investigate biological functions and disease associations of PRMT3.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62835" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62835" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry General Procedures</h3><div class="NLM_p">Analytical thin-layer chromatography (TLC) was performed employing EMD Milipore 210–270 μm 60-F254 silica gel plates. The plates were visualized by exposure to UV light. Flash column chromatography was performed on a Teledyne ISCO Combi<i>Flash</i> Rf<sup>+</sup> system equipped with a variable wavelength UV detector and a fraction collector using Redi<i>Sep</i> Rf normal phase silica columns. Nuclear magnetic resonance (NMR) spectra were acquired on a Bruker DRX-600 spectrometer or on a Varian Mercury spectrometer at 400 MHz. Chemical shifts are reported in parts per million (ppm, δ) scale relative to solvent residual peak (chloroform-<i>d</i>: <sup>1</sup>H, 7.26 ppm; <sup>13</sup>C, 77.16 ppm; methanol-<i>d</i><sub>4</sub>: <sup>1</sup>H, 3.31 ppm; <sup>13</sup>C, 49.0 ppm). <sup>1</sup>H NMR data are reported as follows: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, p = pentet, m = multiplet, app = apparent), coupling constant, and integration. HPLC spectra for all compounds were acquired using an Agilent 6110 series system with a UV detector set to 254 nm. Samples were injected (5 μL) onto an Agilent Eclipse Plus, 4.6 Å, ∼50 mm, 1.8 μM, C18 column at room temperature, either with a linear gradient from 50% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid, or by a linear gradient from 10% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid. The flow rate was 1.0 mL/min. Mass spectrometry (MS) data were acquired in positive ion mode using an Agilent 6110 single-quadrupole mass spectrometer with an electrospray ionization (ESI) source. HRMS analysis was conducted on an Agilent Technologies G1969A high-resolution API-TOF mass spectrometer attached to an Agilent Technologies 1200 HPLC system. Samples were ionized by ESI in positive mode. All biologically evaluated compounds had >95% purity using the HPLC methods described above.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-oxo-2-(piperidin-1-yl)ethyl)urea (<b>3</b>)</h4><div class="NLM_p last">Compound <b>3</b> was prepared according to previously published procedures.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 1-(Isoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>4</b>)</h4><div class="NLM_p last">Compound <b>4</b> was prepared according to previously published procedures.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 1-(Isoquinolin-6-yl)-3-(2-oxo-2-(piperidin-1-yl)ethyl)urea (<b>5</b>)</h4><div class="NLM_p last">To a solution of isoquinolin-6-amine (50 mg, 0.347 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (DMF) (1.6 mL) at room temperature was added <i>N</i>,<i>N</i>′-carbonyldiimidazole (CDI) (84 mg, 0.520 mmol). The resulting solution was stirred for 12 h prior to the addition of 2-amino-1-(piperidin-1-yl)ethan-1-one (99 mg, 0.694 mmol) and stirred for a further 6 h. Following dilution with water (20 mL), the aqueous layer was extracted with ethyl acetate (EtOAc) (3 × 20 mL), and the combined organic extracts were dried with anhydrous sodium sulfate. After filtration, all solvents were removed under reduced pressure, and the residue was purified by column chromatography on silica gel to afford title compound (<b>5</b>) (31 mg, 29% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.08 (s, 1H), 8.35 (d, <i>J</i> = 5.7 Hz, 1H), 8.08 (br s, 1H), 7.98 (d, <i>J</i> = 8.9 Hz, 1H), 7.63 (d, <i>J</i> = 5.8 Hz, 1H), 7.02 (br s, 2H), 6.57 (t, <i>J</i> = 4.6 Hz, 1H), 4.02 (d, <i>J</i> = 4.7 Hz, 2H), 3.50–3.44 (m, 2H), 3.38–3.33 (m, 2H), 1.65–1.57 (m, 2H), 1.57–1.50 (m, 2H), 1.50–1.40 (m, 2H). <i>m</i>/<i>z</i> (HRMS) [M + H]<sup><b>+</b></sup> for C<sub>17</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 313.1659, found 313.1662.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 1-(1-Oxo-1,3-dihydroisobenzofuran-5-yl)-3-(2-oxo-2-(piperidin-1-yl)ethyl)urea (<b>6</b>)</h4><div class="NLM_p last">To a solution of 5-amino-3<i>H</i>-benzofuran-1-one (75 mg, 0.5 mmol, 1.0 equiv) in DMF (1.5 mL) was added CDI (90 mg, 0.55 mmol, 1.1 equiv), and the resulting mixture was stirred for 8 h at rt. 2-Amino-1-piperidin-1-ylethanone hydrochloride salt (134 mg, 0.75 mmol, 1.5 equiv) was then added followed by Hunig’s base (131 μL, 0.75 mmol, 1.5 equiv). After being stirred for 18 h at rt, the resulting mixture was diluted with water (25 mL) and extracted with EtOAc (3 × 25 mL). Combined organic layers were dried over sodium sulfate and concentrated under reduced pressure to give crude product, which was then purified by flash column chromatography to yield desired compound as white solid (39 mg, 25%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.85 (s, 1H), 7.73 (d, <i>J</i> = 8.5 Hz, 1H), 7.42 (dd, <i>J</i> = 8.5, 1.8 Hz, 1H), 5.31 (s, 2H), 4.10 (s, 2H), 3.57 (t, <i>J</i> = 5.6 Hz, 2H), 3.45 (t, <i>J</i> = 5.5 Hz, 2H), 1.74–1.50 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>20</sub>N<sub>3</sub>O<sub>4</sub><sup>+</sup>: calculated 318.1, found 318.1.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 1-(2-Oxo-2-(piperidin-1-yl)ethyl)-3-(quinazolin-7-yl)urea (<b>7</b>)</h4><div class="NLM_p last">To a solution of quinazolin-7-amine (73 mg, 0.5 mmol, 1.0 equiv) in DMF (1.5 mL) was added CDI (90 mg, 0.55 mmol, 1.1 equiv), and the resulting mixture was stirred for 8 h at rt. 2-Amino-1-piperidin-1-ylethanone hydrochloride salt (134 mg, 0.75 mmol, 1.5 equiv) was then added followed by Hunig’s base (131 μL, 0.75 mmol, 1.5 equiv). After being stirred for 18 h at rt, the resulting mixture was diluted with water (25 mL) and extracted with EtOAc (3 × 25 mL). Combined organic layers were dried over sodium sulfate and concentrated under reduced pressure to give crude product, which was then purified by flash column chromatography to yield desired compound (10 mg, 6%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.29 (s, 1H), 9.06 (s, 1H), 8.22 (d, <i>J</i> = 2.1 Hz, 1H), 8.00 (d, <i>J</i> = 8.9 Hz, 1H), 7.71 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.14 (s, 2H), 3.58 (t, <i>J</i> = 5.6 Hz, 2H), 3.47 (t, <i>J</i> = 5.5 Hz, 2H), 1.75–1.53 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup>: calculated 314.2, found 314.2.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1-(Isoquinolin-7-yl)-3-(2-oxo-2-(piperidin-1-yl)ethyl)urea (<b>8</b>)</h4><div class="NLM_p last">To a solution of isoquinolin-7-amine (50 mg, 0.347 mmol) in DMF (1.6 mL) at room temperature was added CDI (84 mg, 0.520 mmol). The resulting solution was stirred for 12 h prior to the addition of 2-amino-1-(piperidin-1-yl)ethan-1-one (99 mg, 0.694 mmol) and stirred for a further 6 h. Following dilution with water (20 mL), the aqueous layer was extracted with EtOAc (3 × 20 mL), and the combined organic extracts were dried with anhydrous sodium sulfate. After filtration, all solvents were removed under reduced pressure, and the residue was purified by column chromatography on silica gel to afford title compound (<b>8</b>) (45 mg, 42% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.13 (s, 1H), 8.33 (d, <i>J</i> = 5.6 Hz, 1H), 8.22 (d, <i>J</i> = 1.9 Hz, 1H), 7.85 (d, <i>J</i> = 8.9 Hz, 1H), 7.69 (d, <i>J</i> = 5.6 Hz, 1H), 7.11 (br s, 1H), 6.92 (br s, 1H), 6.50 (t, <i>J</i> = 4.7 Hz, 1H), 4.03 (dd, <i>J</i> = 14.2, 5.9 Hz, 2H), 3.47–3.44 (m, 2H), 3.38–3.34 (m, 2H), 1.63–1.56 (m, 2H), 1.56–1.50 (m, 2H), 1.49–1.41 (m, 2H). <i>m</i>/<i>z</i> (HRMS) [M + H]<sup><b>+</b></sup> for C<sub>17</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 313.1659, found 313.1664.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedures for the Preparation of Compounds <b>9</b>–<b>14</b> in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a></h3><div class="NLM_p">Compounds <b>9</b>–<b>14</b> shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> were prepared according general procedures described below. To a solution of isoquinolin-6-amine (1.0 equiv) and triethylamine (TEA) (2 equiv) in DMF (1 mL/0.347 mmol) was added CDI (1.5 equiv), and the reaction mixture was allowed to stir at 25 °C for 4 h. To the reaction mixture was then added the corresponding amine (2 equiv), and the mixture was allowed to stir for additional 1 h. Then 50 mL of water and 50 mL of EtOAc were added to the reaction mixture. After extraction, the organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by column chromatography on silica gel eluting with 0–5% MeOH in DCM to give the product.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 1-(2-(Cyclohex-1-en-1-yl)ethyl)-3-(isoquinolin-6-yl)urea (<b>9</b>)</h4><div class="NLM_p last">Yellow oil (67 mg, 62% yield). <sup>1</sup>H NMR (chloroform-<i>d</i>) δ: 9.07 (s, 1H), 8.40 (d, <i>J</i> = 5.8 Hz, 1H), 8.04(s, 1H), 7.81 (d, J = 8.8 Hz, 1H), 7.66 (s, 1H), 7.53 (d, <i>J</i> = 5.8 Hz, 1H), 7.44 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 7.27 (s, 1H), 5.45 (s, 1H), 5.27–5.30 (m, 1H), 3.42–3.77 (m, 2H), 2.20–2.17 (m, 6H), 1.97–1.92 (m, 4H), 1.627–1.47 (m, 4H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>22</sub>N<sub>3</sub>O<sup>+</sup>: calculated 296.2, found 296.1.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1-(2-Cyclohexylethyl)-3-(isoquinolin-6-yl)urea (<b>10</b>)</h4><div class="NLM_p last">Yellow oil (69 mg, 64% yield). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 9.07 (s, 1H), 8.87 (s, 1H), 8.33 (d, <i>J</i> = 5.8 Hz, 1H), 8.06 (d, <i>J</i> = 1.8 Hz, 1H), 7.95 (d, <i>J</i> = 9.0 Hz, 1H), 7.60 (d, <i>J</i> = 6.0 Hz, 1H), 7.52 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 6.28 (br t, <i>J</i> = 5.5 Hz, 1H), 3.11–3.24 (m, 2H), 1.58–1.76 (m, 5H), 1.11–1.39 (m, 6H), 0.83–0.98 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>24</sub>N<sub>3</sub>O<sup>+</sup>: calculated 298.2, found 298.1.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 1-(2-Cyclopentylethyl)-3-(isoquinolin-6-yl)urea (<b>11</b>)</h4><div class="NLM_p last">Light yellow oil (30 mg, 29% yield). <sup>1</sup>H NMR (chloroform-<i>d</i>) δ: 9.04 (s, 1H), 8.36 (d, <i>J</i> = 5.8 Hz, 1H), 8.03 (s, 2H), 7.76 (d, <i>J</i> = 8.8 Hz, 1H), 7.34–7.56 (m, 2H), 7.27 (s, 1H), 3.19–3.40 (m, 2H), 1.64–1.82 (m, 3H), 1.40–1.64 (m, 6H), 0.95–1.16 (m, 2H), 0.01 (s, 1H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>17</sub>H<sub>22</sub>N<sub>3</sub>O<sup>+</sup>: calculated 284.2, found 284.1.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1-(2-Cyclopropylethyl)-3-(isoquinolin-6-yl)urea (<b>12</b>)</h4><div class="NLM_p last">White solid (27 mg, 29% yield). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 9.59 (s, 1H), 9.46 (s, 1H), 8.34–8.51 (m, 2H), 8.28 (d, <i>J</i> = 9.0 Hz, 1H), 8.12 (d, <i>J</i> = 6.5 Hz, 1H), 7.77 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 6.71 (br t, <i>J</i> = 5.4 Hz, 1H), 3.11–3.27 (m, 3H), 1.38 (q, <i>J</i> = 7.0 Hz, 2H), 0.61–0.80 (m, 1H), 0.35–0.49 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>15</sub>H<sub>18</sub>N<sub>3</sub>O<sup>+</sup>: calculated 256.1, found 256.1.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-(Isoquinolin-6-yl)-3-phenethylurea (<b>13</b>)</h4><div class="NLM_p last">Light yellow oil (23 mg, 22% yield). <sup>1</sup>H NMR (chloroform-<i>d</i>) δ: 8.95 (s, 1H), 7.89–8.47 (m, 3H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 7.46 (d, <i>J</i> = 5.8 Hz, 1H), 7.37 (dd, <i>J</i> = 8.9, 1.9 Hz, 1H), 7.20–7.30 (m, 2H), 7.09–7.20 (m, 3H), 5.63 (br s, 1H), 3.55 (q, <i>J</i> = 6.8 Hz, 2H), 2.84 (t, <i>J</i> = 6.9 Hz, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>18</sub>N<sub>3</sub>O<sup>+</sup>: calculated 292.1, found 292.1.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1-(2-(1-(Aminomethyl)cyclohexyl)ethyl)-3-(isoquinolin-6-yl)urea (<b>14</b>)</h4><div class="NLM_p last">The general procedure was applied using <i>tert</i>-butyl ((1-(2-aminoethyl)cyclohexyl)methyl)carbamate as the amine (285 mg, 1.11 mmol) to give <i>tert</i>-butyl ((1-(2-(3-(isoquinolin-6-yl)ureido)ethyl)cyclohexyl)methyl)carbamate as a white solid. To the solution of <i>tert</i>-butyl ((1-(2-(3-(isoquinolin-6-yl)ureido)ethyl)cyclohexyl)methyl)carbamate (130 mg, 0.305 mmol) in DCM (1 mL) was added TFA (1.000 mL, 12.98 mmol). Then the reaction mixture was stirred at 25 °C for 0.5 h. To the mixture was added 5 mL of toluene, and it was then concentrated in vacuo. The residue was purified by preparative-HPLC to give the product as white solid (23 mg, 22% yield). <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>) δ 8.94–9.07 (m, 1H), 8.28 (br d, <i>J</i> = 5.8 Hz, 1H), 8.07 (br s, 1H), 7.88–7.99 (m, 1H), 7.49–7.69 (m, 2H), 3.26–3.44 (m, 2H), 3.02–3.26 (m, 2H), 2.65 (s, 2H), 1.14–1.78 (m, 13H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>19</sub>H<sub>27</sub>N<sub>4</sub>O<sup>+</sup>: calculated 327.2, found 327.2.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedures for the Synthesis of Amides <b>15</b>–<b>35</b> in <a class="ref internalNav" href="#tbl3" aria-label="Tables 3">Tables 3</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Synthesis of (Isoquinolin-6-ylcarbamoyl)glycine (<b>28</b>) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h4><div class="NLM_p">To a stirring solution of 6-aminoisoquinoline (1.2 g, 8.32 mmol, 1 equiv) in a mixture of dichloromethane and DMF (30 and 10 mL) was added ethyl isocyanatoacetate (2.80 mL, 25 mmol, 3.0 equiv), and the resulting mixture was stirred overnight at room temperature. After removal of volatiles, the crude mixture was purified by flash column chromatography (gradient from 100% dichloromethane to 10% methanol in dichloromethane) to yield the desired ethyl ester (<b>27</b>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> as a pale yellow solid, which was resuspended in methanol (48 mL) and water (16 mL) followed by the addition of 1 N solution of NaOH (24 mL). The resulting clear mixture was then stirred at room temperature overnight. After concentration of the mixture under reduced pressure, the crude mixture was purified by reverse phase flash column chromatography (gradient from 100% water to 10% methanol in dichloromethane) to yield the desired acid <b>28</b> as a TFA salt (2.04 g, 57% over two steps).</div><div class="NLM_p last">To a stirring mixture of the above acid <b>28</b> (1.0 equiv) in DMF (0.8 mL/0.1 mmol) was added <i>N</i>-(3-(dimethylamino)propyl)-<i>N</i>′-ethylcarbodiimide hydrochloride (EDC·HCl) (1.5 equiv), 1-hydroxy-7-azabenzotriazole (HOAt) (1.5 equiv), and the corresponding amine (1.5 equiv) followed by <i>N</i>-methylmorpholine (NMM) (2 equiv), and the resulting mixture was stirred for 18 h at room temperature. The reaction was purified by either flash column chromatography or HPLC to give pure products.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 1-(2-(Azetidin-1-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>15</b>)</h4><div class="NLM_p last">The reaction mixture was purified by HPLC to give pure product as a white solid (mono-TFA salt, 8 mg, 10%). <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>) δ 9.02 (d, <i>J</i> = 0.9 Hz, 1H), 8.29 (d, <i>J</i> = 5.9 Hz, 1H), 8.08 (d, <i>J</i> = 2.1 Hz, 1H), 7.97 (d, <i>J</i> = 9.0 Hz, 1H), 7.66–7.61 (m, 1H), 7.57 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.35–4.27 (m, 2H), 4.07 (t, <i>J</i> = 7.8 Hz, 2H), 3.87 (s, 2H), 2.43–2.32 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>15</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 285.1, found 285.2.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-(2-(Azepan-1-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>16</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 21 mg, 40%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.37 (s, 1H), 8.37 (d, <i>J</i> = 2.0 Hz, 1H), 8.31 (d, <i>J</i> = 6.6 Hz, 1H), 8.27 (d, <i>J</i> = 9.0 Hz, 1H), 8.07 (d, <i>J</i> = 6.6 Hz, 1H), 7.81 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 4.17 (s, 2H), 3.61–3.56 (m, 2H), 3.56–3.51 (m, 2H), 1.86–1.82(m, 2H), 1.78–1.71 (m, 2H), 1.69–1.58 (m, 4H). <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 170.5, 156.7, 149.3, 146.9, 145.6, 142.1, 133.3, 132.1, 125.4, 124.2, 112.4, 48.1, 47.4, 42.5, 29.7, 28.5, 28.3, 27.8. MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 327.2, found 327.2.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 1-(2-(4,4-Difluoropiperidin-1-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>17</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 51 mg, 66%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.39 (d, <i>J</i> = 0.9 Hz, 1H), 8.45 (d, <i>J</i> = 2.1 Hz, 1H), 8.37–8.27 (m, 2H), 8.14 (d, <i>J</i> = 6.7 Hz, 1H), 7.86 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 4.21 (s, 2H), 3.75 (t, <i>J</i> = 6.1 Hz, 2H), 3.65 (t, <i>J</i> = 6.0 Hz, 2H), 2.14–1.96 (m, 4H). <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 169.5, 156.8, 149.3, 147.0, 145.7, 142.1, 132.2, 125.3, 124.3, 122.9, 112.5, 111.2, 42.6, 40.3, 35.1, 34.6. MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>17</sub>H<sub>19</sub>F<sub>2</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 349.2, found 349.2.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-(Isoquinolin-6-yl)-3-(2-(4-methylpiperazin-1-yl)-2-oxoethyl)urea (<b>18</b>)</h4><div class="NLM_p last">The title compound was obtained as a light yellow solid (bis-TFA salt, 48 mg, 52%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.96 (br s, 1H), 9.47 (s, 1H), 8.44 (d, <i>J</i> = 6.5 Hz, 1H), 8.32–8.28 (m, 2H), 8.10 (d, <i>J</i> = 6.5 Hz, 1H), 7.78 (dd, <i>J</i> = 9.0, 2.1 Hz, 1H), 6.90 (br s, 1H), 4.12 (br s, 2H), 3.40 (br s, 8H; these protons are obscured by residual water in DMSO), 2.83 (br s, 3H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>17</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup>: calculated 328.2, found 328.2.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-(3-(Isoquinolin-6-yl)ureido)-<i>N</i>,<i>N</i>-dimethylacetamide (<b>19</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 14.6 mg, 18%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.03 (s, 1H), 8.29 (d, <i>J</i> = 5.9 Hz, 1H), 8.10 (d, <i>J</i> = 2.1 Hz, 1H), 7.98 (d, <i>J</i> = 8.9 Hz, 1H), 7.65 (d, <i>J</i> = 5.9 Hz, 1H), 7.59 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.13 (s, 2H), 3.08 (s, 3H), 2.99 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>14</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 273.1, found 273.2.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>,<i>N</i>-Diethyl-2-(3-(isoquinolin-6-yl)ureido)acetamide (<b>20</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (34.1 mg, 81%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.05 (s, 1H), 8.30 (d, <i>J</i> = 5.9 Hz, 1H), 8.11 (s, 1H), 8.00 (d, <i>J</i> = 9.0 Hz, 1H), 7.67 (d, <i>J</i> = 6.0 Hz, 1H), 7.60 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.14 (s, 2H), 3.42 (apparent p, <i>J</i> = 7.0 Hz, 4H), 1.27 (t, <i>J</i> = 7.2 Hz, 3H), 1.16 (t, <i>J</i> = 7.1 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 301.2, found 301.2.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-Cyclopentyl-2-(3-(isoquinolin-6-yl)ureido)-<i>N</i>-methylacetamide (<b>21</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (18.2 mg, 40%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.03 (s, 1H), 8.29 (d, <i>J</i> = 5.9 Hz, 1H), 8.10 (d, <i>J</i> = 2.1 Hz, 1H), 7.98 (d, <i>J</i> = 8.9 Hz, 1H), 7.65 (d, <i>J</i> = 5.9 Hz, 1H), 7.59 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.92–4.90 (m, 1H), 4.27–4.25 (m, 1H), 4.21 (s, 1H), 4.11 (s, 1H), 2.93 (s, 1H), 2.86 (s, 1H), 1.94 (s, 1H), 1.86–1.55 (m, 7H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 327.2, found 327.2.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 2-(3-(Isoquinolin-6-yl)ureido)-<i>N</i>-methoxy-<i>N</i>-methylacetamide (<b>22</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 34.9 mg, 63%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.37 (d, <i>J</i> = 1.1 Hz, 1H), 8.33–8.28 (m, 2H), 8.25 (d, <i>J</i> = 9.1 Hz, 1H), 8.05 (d, <i>J</i> = 6.7 Hz, 1H), 7.79 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 4.24 (s, 2H), 3.82 (s, 3H), 3.25 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>14</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub><sup>+</sup>: calculated 289.1, found 289.1.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 2-(3-(Isoquinolin-6-yl)ureido)-<i>N</i>-methylacetamide (<b>23</b>)</h4><div class="NLM_p last">The reaction mixture was then purified by HPLC to give pure product as a white mono-TFA salt (4 mg, 8%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.41 (d, <i>J</i> = 0.9 Hz, 1H), 8.46 (d, <i>J</i> = 2.1 Hz, 1H), 8.36–8.27 (m, 2H), 8.16 (d, <i>J</i> = 6.7 Hz, 1H), 7.87 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 3.92(s, 2H), 2.78 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>13</sub>H<sub>15</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 259.1, found 259.1.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-Cyclopropyl-2-(3-(isoquinolin-6-yl)ureido)acetamide (<b>24</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (58 mg, 78%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.91 (s, 1H), 9.52 (s, 1H), 8.45 (d, <i>J</i> = 6.6 Hz, 1H), 8.44–8.28 (m, 2H), 8.18 (d, <i>J</i> = 6.6 Hz, 1H), 8.10 (d, <i>J</i> = 4.1 Hz, 1H), 7.79 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 6.83 (t, <i>J</i> = 5.3 Hz, 1H), 3.74 (d, <i>J</i> = 5.2 Hz, 2H), 2.69–262 (m, 1H), 0.65–0.61 (m, 2H), 0.48–0.37 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>15</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 285.1, found 285.1.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-Cyclopentyl-2-(3-(isoquinolin-6-yl)ureido)acetamide (<b>25</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (15 mg, 38%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.04 (s, 1H), 8.30 (d, <i>J</i> = 5.9 Hz, 1H), 8.09 (d, <i>J</i> = 2.1 Hz, 1H), 7.99 (d, <i>J</i> = 8.9 Hz, 1H), 7.66 (d, <i>J</i> = 6.0 Hz, 1H), 7.59 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.15 (p, <i>J</i> = 6.7 Hz, 1H), 3.88 (s, 2H), 2.00–1.89 (m, 2H), 1.73 (s, 2H), 1.68–1.56 (m, 2H), 1.49 (dq, <i>J</i> = 14.4, 8.3, 7.5 Hz, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>17</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 313.2, found 313.2.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-Cyclohexyl-2-(3-(isoquinolin-6-yl)ureido)acetamide (<b>26</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 48.3 mg, 53%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.40 (s, 1H), 8.45 (d, <i>J</i> = 2.1 Hz, 1H), 8.34–8.31 (m, 2H), 8.14 (d, <i>J</i> = 6.7 Hz, 1H), 7.86 (dd, <i>J</i> = 9.0, 2.1 Hz, 1H), 3.91 (s, 2H), 3.73–3.69 (m, 1H), 1.90–1.87 (m, 2H), 1.78–1.75 (m, 2H), 1.66–1.63 (m, 1H), 1.42–1.19 (m, 5H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>15</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 327.2, found 327.2.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 1-(Isoquinolin-6-yl)-3-(2-oxo-2-(3,3,4,4-tetrafluoropyrrolidin-1-yl)ethyl)urea (<b>29</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (48.3 mg, 53%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.04 (s, 1H), 8.30 (d, <i>J</i> = 5.9 Hz, 1H), 8.11–8.07 (m, 1H), 7.98 (d, <i>J</i> = 8.9 Hz, 1H), 7.65 (d, <i>J</i> = 5.9 Hz, 1H), 7.59 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 4.28 (t, <i>J</i> = 13.5 Hz, 2H), 4.08–4.01 (m, 4H). MS <i>m</i>/<i>z</i> (HRMS) [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>15</sub>F<sub>4</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 371.1126, found 371.1153.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-(2-(2,5-Dimethylpyrrolidin-1-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>30</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (48.2 mg, 88%). The amine, 2,5-dimethylpyrrolidine, is used as mixture of cis and trans for the coupling reaction, and only the major product is reported below. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.03 (br s, 1H), 8.29 (d, <i>J</i> = 5.9 Hz, 1H), 8.09 (d, <i>J</i> = 2.1 Hz, 1H), 7.98 (d, <i>J</i> = 8.9 Hz, 1H), 7.65 (d, <i>J</i> = 5.9 Hz, 1H), 7.59 (dd, <i>J</i> = 8.9, 2.2 Hz, 1H), 4.19–4.01 (m, 4H), 2.17–1.97 (m, 2H), 1.79–1.71 (m, 2H), 1.34 (d, <i>J</i> = 6.4 Hz, 3H), 1.33 (d, <i>J</i> = 6.4 Hz, 3H). MS <i>m</i>/<i>z</i> (HRMS) [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 327.1816, found 371.1819.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Methyl(isoquinolin-6-ylcarbamoyl)glycyl-<span class="smallcaps smallerCapital">l</span>-prolinate (<b>31</b>)</h4><div class="NLM_p last">The title compound was obtained as a yellow solid (mono-TFA salt, 51 mg, 65%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.40 (s, 1H), 8. 39 (s, 1H), 8.32 (br s, 2H), 8.13 (s, 1H), 7.84 (s, 1H), 4.52 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.23–4.08 (m, 2H), 3.81 (s, 1H), 3.75–3.57 (m, 4H), 2.32- 2.23 (m, 1H), 2.14–1.97 (m, 3H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub><sup>+</sup>: calculated 357.2, found 357.2. [α]<sup>20</sup><sub>D</sub> = −70 (<i>c</i> 1.6, CH<sub>3</sub>OH).</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 1-(2-(Hexahydrocyclopenta[<i>c</i>]pyrrol-2(1<i>H</i>)-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>32</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 33 mg, 44%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.41 (s, 1H), 8.46 (s, 1H), 8.33 (d, <i>J</i> = 7.4 Hz, 2H), 8.15 (s, 1H), 7.87 (d, <i>J</i> = 9.1 Hz, 1H), 4.07 (s, 2H), 3.73–3.65 (m, 2H), 3.31–3.30 (m, 2H), 2.87–279 (m, 1H), 2.75–2.66 (m, 1H), 1.97–1.77 (m, 3H), 1.73–1.63 (m, 1H), 1.59–1.46 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 339.2, found 339.2.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 1-(2-(Isoindolin-2-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>33</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (42 mg, 73%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.04 (s, 1H), 8.30 (d, <i>J</i> = 5.9 Hz, 1H), 8.12 (d, <i>J</i> = 2.1 Hz, 1H), 7.99 (d, <i>J</i> = 8.9 Hz, 1H), 7.63 (d, <i>J</i> = 5.8 Hz, 1H), 7.56 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 7.36–7.31 (m, 3H), 6.65 (t, <i>J</i> = 4.8 Hz, 1H), 4.88 (s, 2H), 4.71 (s, 2H), 4.09 (d, <i>J</i> = 4.7 Hz, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>20</sub>H<sub>19</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 347.2, found 347.2.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-(2-(7-Azabicyclo[2.2.1]heptan-7-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>34</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (mono-TFA salt, 33 mg, 45%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.40 (br s, 1H), 8.43 (d, <i>J</i> = 2.1 Hz, 1H), 8.35–8.28 (m, 2H), 8.13 (d, <i>J</i> = 6.7 Hz, 1H), 7.85 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 4.60 (t, <i>J</i> = 4.8 Hz, 1H), 4.41 (t, <i>J</i> = 4.8 Hz, 1H), 4.10 (s, 2H), 1.90 (q, <i>J</i> = 9.1, 7.0 Hz, 2H), 1.77 (s, 2H), 1.68–1.59 (m, 2H), 1.58–1.50 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>18</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 325.2, found 325.2.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 1-(2-(8-Azabicyclo[3.2.1]octan-8-yl)-2-oxoethyl)-3-(isoquinolin-6-yl)urea (<b>35</b>)</h4><div class="NLM_p last">The title compound was obtained as a white solid (22 mg, 38%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.99 (s, 1H), 8.26 (d, <i>J</i> = 5.9 Hz, 1H), 8.05 (s, 1H), 7.93 (d, <i>J</i> = 9.0 Hz, 1H), 7.62–7.52 (m, 2H), 4.62–4.55 (m, 1H), 4.27 (d, <i>J</i> = 6.7 Hz, 1H), 4.18–4.02 (m, 2H), 2.15–2.03 (m, 1H), 1.97–1.70 (m, 6H), 1.62 (dd, <i>J</i> = 13.2, 5.4 Hz, 2H), 1.53 (d, <i>J</i> = 12.8 Hz, 1H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 339.2, found 339.2.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-(7-Fluoroisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>36</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A; <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>)</h4><div class="NLM_p">A mixture of aminoacetaldehyde dimethyl acetal (0.5 g, 2.46 mmol, 1.0 equiv) and 4-bromo-3-fluorobenzaldehyde (0.4 mL, 3.70 mmol, 1.5 equiv) in toluene (40 mL) in a round-bottom flask equipped with a reflux condenser and Dean–Stark trap was heated to reflux for 6 h. Then the reaction mixture was quenched with water (20 mL) and extracted with DCM (40 mL). After concentration of volatiles, crude mixture was dissolved in EtOH (20 mL), and NaBH<sub>4</sub> (0.19 g, 4.92 mmol, 2 equiv) was added at room temperature and stirred overnight. The reaction was then quenched with water (50 mL) and extracted with EtOAc (50 mL × 3). Combined organic layers were washed with brine (50 mL × 2) and dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure to give the desired amino acetal (<b>41</b> (R<sub>1</sub> = F, R<sub>2</sub> = H) in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A) (0.65 g, 90%). The amino acetal (0.35 g, 1.19 mmol) was resuspended in DCM (20 mL), and TEA (0.5 mL, 3.57 mmol, 3 equiv), TsCl (0.27 g, 1.43 mmol, 1.2 equiv), and DMAP (15 mg, 10 mol %) were added to give a clear solution, which was stirred at room temperature overnight. The reaction mixture was then suspended in water (20 mL) and extracted with DCM (50 mL × 3). Combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated down. The crude mixture was purified by flash column chromatography to give the desired tosyl amine (0.51 g, 90%) (<b>42</b> (R<sub>1</sub> = F, R<sub>2</sub> = H) in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A).</div><div class="NLM_p">To a flame-dried flask equipped with Teflon stir bar was added AlCl<sub>3</sub> (0.27 g, 2.02 mmol, 4.5 equiv) under nitrogen atmosphere followed by the addition of the above solution of tosyl amine (0.2 g, 0.45 mmol, 1.0 equiv) in DCM (6 mL). The resulting solution was stirred under nitrogen at room temperature overnight. The reaction mixture was then cooled down to 0 °C, quenched with NaHCO<sub>3</sub> (10 mL), and extracted with DCM (20 mL × 3). Combined organic layers were washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The crude oil was then purified by flash column chromatography to yield desired 6-bromo-7-fluoroisouinoline (58 mg, 58%) (<b>43</b> (R<sub>1</sub> = F, R<sub>2</sub> = H) in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A).</div><div class="NLM_p last">To a flame-dried pressure vessel equipped with Teflon stirring bar was added CuI (10 mg, 0.05 mmol, 20 mol %), <span class="smallcaps smallerCapital">l</span>-proline (12 mg, 0.10 mmol, 40 mol %), and K<sub>2</sub>CO<sub>3</sub> (104 mg, 0.75 mmol, 3 equiv) as solid, and the vessel was flame-dried again under vacuum. Then DMSO (1 mL) solution of 6-bromo-7-fluoroisouinoline (58 mg, 0.25 mmol, 1 equiv) was added to the pressure vessel under nitrogen atmosphere followed by the addition of NH<sub>4</sub>OH (0.5 mL). The resulting suspension in a sealed vessel was then heated to 70 °C overnight. The reaction mixture was then cooled down to rt, suspended in water (5 mL) and EtOAc (10 mL), and further extracted with EtOAc (10 mL × 3). Combined organic layers were washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The crude oil was then purified by flash column chromatography to yield desired 7-fluoroisouinoline-6-amine (<b>44</b> (R<sub>1</sub> = F, R<sub>2</sub> = H) in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A). This amine was then immediately dissolved in DCM/DMF (1 mL/0.3 mL), and ethyl isocyanato acetate (96 mg, 0.75 mmol, 3 equiv) was added. The resulting mixture was stirred at room temperature overnight, and after removal of volatiles, it was purified by flash column chromatography and immediately hydrolyzed to acid with 1 N NaOH (1 mL) in MeOH (1.5 mL) and water (0.5 mL) overnight. After purification by reverse phase chromatography, the desired acid was obtained as a white solid. To a stirring mixture of the acid (27 mg, 0.072 mmol) in THF (1.0 mL) was added pyrrolidine (10 μL, 0.122 mmol, 1.70 equiv) followed by EDC·HCl (23.4 mg, 0.122 mmol, 1.70 equiv), and the resulting mixture was stirred overnight at room temperature. Flash column chromatography yielded 1-(7-fluoroisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>36</b>) as a white solid (16.2 mg, 20% yield over four steps, starting from 6-bromo-7-fluoroisouinoline <b>43</b> (R<sub>1</sub> = F, R<sub>2</sub> = H)). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.03 (s, 1H), 8.65 (d, <i>J</i> = 7.8 Hz, 1H), 8.31 (d, <i>J</i> = 5.8 Hz, 1H), 7.80 (d, <i>J</i> = 11.5 Hz, 1H), 7.67 (d, <i>J</i> = 5.9 Hz, 1H), 4.08 (br s, 2H), 3.53 (t, <i>J</i> = 6.8 Hz, 2H), 3.48 (t, <i>J</i> = 6.9 Hz, 2H), 2.07–2.00 (m, 2H), 1.95–1.88 (m, 2H). MS <i>m</i>/<i>z</i> (HRMS) [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 317.1408, found 317.1418.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-(7-Methylisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>37</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A; <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>)</h4><div class="NLM_p last">The same procedures as above were used for the synthesis of 1-(7-methylisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea starting from 4-bromo-3-methylbenzaldehyde. 6-Bromo-7-methylisouinoline (43 (R<sub>1</sub> = Me, R<sub>2</sub> = H); 50%, over the first four steps). Compound <b>37</b> was obtained in 20% (24 mg) over the last four steps as described above. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.98 (s, 1H), 8.41 (s, 1H), 8.26 (d, <i>J</i> = 5.9 Hz, 1H), 7.86 (s, 1H), 7.63 (d, <i>J</i> = 5.9 Hz, 1H), 4.07 (s, 2H), 3.53 (t, <i>J</i> = 6.8 Hz, 2H), 3.48 (t, <i>J</i> = 6.9 Hz, 2H), 2.50 (s, 3H), 2.03 (p, <i>J</i> = 6.7 Hz, 2H), 1.91 (p, <i>J</i> = 6.6 Hz, 2H). MS <i>m</i>/<i>z</i> (HRMS) [M + H]<sup>+</sup> for C<sub>17</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 313.1659, found 313.1662.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 1-(1-Methylisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>38</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A; <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>)</h4><div class="NLM_p last">The same procedures as above were used for the synthesis of 1-(7-methylisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>36</b>) starting from 4′-bromoacetophenone. 6-Bromo-7-methylisouinoline (<b>43</b> (R<sub>1</sub> = Me, R<sub>2</sub> = H)) was obtained in 27%, over the first four steps. Compound <b>38</b> was then obtained (2 mg, 2%) over the last four steps as described above. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.17–8.09 (m, 2H), 8.05 (d, <i>J</i> = 2.2 Hz, 1H), 7.59 (dd, <i>J</i> = 9.1, 2.2 Hz, 1H), 7.50 (d, <i>J</i> = 6.0 Hz, 1H), 4.05 (s, 2H), 3.52–3.45 (m, 4H), 2.87 (s, 3H), 2.03 (p, <i>J</i> = 6.8 Hz, 2H), 1.91 (p, <i>J</i> = 7.1 Hz, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>17</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 313.2, found 313.2.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 1-(3-Fluoroisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>39</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>B; <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>)</h4><div class="NLM_p last">To the mixture of 6-bromoisoquinolin-3-amine (158 mg, 0.71 mmol, 1.0 equiv) and HF·pyridine (10 mL) at 0 °C was added dropwise a solution of NaNO<sub>2</sub> (243 mg, 3.54 mmol, 5 equiv) in water (5 mL). Upon completion of the addition, the cold bath was removed and the reaction was allowed to warm up to room temperature for 1.5 h at which point saturated NaHCO<sub>3</sub> solution (45 mL) was added. The reaction mixture was diluted with DCM (20 mL) and saturated NH<sub>4</sub>Cl (10 mL) solution, and phases were separated and aqueous phase further extracted with DCM (20 mL × 3). Combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. Crude material was purified by flash column chromatography to yield 6-bromo-3-fluoroisoquinoline as a white solid (128 mg, 80%). The same procedure that was used for the synthesis of <b>36</b> (last four steps) was repeated to obtain the title compound <b>39</b> (6 mg, 5%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.79 (s, 1H), 8.10 (s, 1H), 7.99 (d, <i>J</i> = 8.9 Hz, 1H), 7.50 (dt, <i>J</i> = 8.9, 2.3 Hz, 1H), 7.23 (s, 1H), 4.05 (s, 2H), 3.53 (t, <i>J</i> = 6.8 Hz, 2H), 3.48 (t, <i>J</i> = 6.9 Hz, 2H), 2.02 (q, <i>J</i> = 6.8 Hz, 2H), 1.91 (p, <i>J</i> = 6.8 Hz, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 317.1, found 317.2.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 1-(8-Chloroisoquinolin-6-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>40</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>C; <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>)</h4><div class="NLM_p">To a stirring mixture of 5-aminoisoquinoline (5.76 g, 40 mmol, 1 equiv) in pyridine (40 mL) was added acetic anhydride (5.7 mL, 60 mmol, 1.5 equiv), and the resulting reaction mixture was stirred at room temperature overnight. The precipitate formed was filtered and washed with cold hexanes (400 mL) in portions to give acetylated product (6.98 g, 94%). To a stirring mixture of 5-acetyl aminoisoquinoline (3.36 g, 18 mmol, 1 equiv) in DMF (30 mL) was added NCS (2.40 mL, 18 mmol, 1.0 equiv), and the resulting mixture was heated to 65 °C for 3 days. The reaction mixture was then diluted with EtOAc (50 mL), and water (50 mL) was added. After separation, the aqueous phase was extracted with EtOAc (50 mL × 3), and combined organic layers were then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. Crude material was purified by flash column chromatography to yield desired compound (3.53 g, 88%).</div><div class="NLM_p last">To a solution of 5-acetylamino-8-chloroisoquinoline (1.0 g, 4.53 mmol, 1.0 equiv) in DMF (10 mL) was added dibromocyanuric acid (1.3 g, 4.53 mmol, 1 equiv), and the reaction mixture was heated to 65 °C for 3 h. After concentration, the obtained crude product resuspended in EtOH (45 mL), conc. HCl (9 mL) was added, and reaction mixture was heated to reflux for 3 h. The reaction mixture was then cooled down to rt and neutralized with 1 N NaOH to pH 7 and extracted with EtOAc (100 mL × 3). Combined organic layers were then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. Crude material (300 mg, 26% over two steps) was used for the next step without further purification. Crude material (300 mg, 1.2 mmol, 1.0 equiv) was dissolved in EtOH (45 mL) followed by the addition of Ac<sub>2</sub>O (7.5 mL) and a solution of NaNO<sub>2</sub> (in 15 mL water) and NaHSO<sub>3</sub> (in 18 mL water). The resulting mixture was stirred at rt for 10 min, AcOH (7.5 mL) was added, and the reaction was allowed to stir at rt overnight. The reaction mixture was neutralized with 1 N NaOH (∼350 mL) to pH 8–9 and extracted with EtOAc (250 mL × 3). Combined organic layers were then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. Crude material was purified by flash column chromatography to yield the desired compound (100 mg, 34%). The same procedures that were used for the synthesis of <b>36</b> (last four steps) were repeated to obtain the title compound <b>40</b> (51 mg, 30%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.60 (br s, 1H), 8.40 (d, <i>J</i> = 6.7 Hz, 1H), 8.29 (d, <i>J</i> = 2.0 Hz, 1H), 8.21 (d, <i>J</i> = 6.7 Hz, 1H), 8.12 (d, <i>J</i> = 1.9 Hz, 1H), 4.08 (s, 2H), 3.54–3.46 (m, 4H), 2.07–2.00 (m, 2H), 1.95–1.89 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>2</sub><sup>+</sup>: calculated 333.1 and 335.1; found 333.1 and 335.1.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 1-(Naphthalen-2-yl)-3-(2-oxo-2-(pyrrolidin-1-yl)ethyl)urea (<b>51</b>)</h4><div class="NLM_p">Compound <b>51</b> was prepared according to a previously published procedure.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p last">Synthesis and characterization data for compounds <b>49</b> and <b>50</b> are detailed in the <a class="ref internalNav" href="#notes-5" aria-label="Supporting Information">Supporting Information</a>.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> PRMT3 Biochemical Assay</h3><div class="NLM_p last">The radiometric scintillation proximity assays to evaluate the potency of the compounds were performed as described previously.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The reactions were done under balanced conditions using the biotinylated histone H4 peptide (Tufts University Peptide Synthesis Core Facility, Boston, Ma) with the sequence of SGRGKGGKGLGKGGAKRHRKVLRDK-biotin) as substrate and [<sup>3</sup>H]S-adenosylmethionine (Waltham, MA, Cat# NET155 V001MC, specific activity range 12–18 Ci/mmol) as the methyl donor.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Selectivity Assays</h3><div class="NLM_p last">The methyltransferase selectivity of <b>29</b>, <b>30</b>, and <b>36</b> was assessed at compound concentrations of 1, 5, and 20 μM as described previously.<a onclick="showRef(event, 'ref22 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref22 ref25 ref26 ref27">(22, 25-27)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Cellular PRMT3 Assay</h3><div class="NLM_p last">Compound effects in cells were determined as described previously (PMID: 27423858, 25728001). Briefly, HEK293 cells were grown in DMEM supplemented with 10% FBS, penicillin (100 units/mL), and streptomycin (100 μg/mL). The cells were cotransfected with FLAG-tagged PRMT3/mutantPRMT3 and GFP-tagged histone H4 (constructs described in PMID 25728001) using 293fectin Transfection Reagent (Invitrogen), following manufacturer instructions. Cells were lysed in lysis buffer (in mM: 20 Tris-HCl pH = 8, 150 NaCl, 1 EDTA, 10 MgCl<sub>2</sub>, 0.5% Triton-X100, 12.5 U/mL benzonase (Sigma), complete EDTA-free protease inhibitor cocktail (Roche)). After 3 min incubation at rt, SDS was added to a final 1% concentration. Lysates were separated on SDS PAGE, blotted, and probed with indicated antibodies: mouse anti-GFP (1:5000, Clontech #632381), mouse anti-H4 (1:1000, Abcam #174628), rabbit anti-H4R3me2a (1:1000 Active Motif #39705), and mouse anti-FLAG (1:5000, Sigma #F1804). The signal was read on an Odyssey scanner (LiCor) at 800 and 700 nm. Fluorescence intensity of H4R3me2a was quantified and normalized to GFP and H4 signals for exogenous and endogenous histones, respectively.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> PRMT3 In-Cell Hunter Assay</h3><div class="NLM_p last">This cellular assay was performed as described previously.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i58"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01674">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01674" class="ext-link">10.1021/acs.jmedchem.7b01674</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis and characterization data for compounds <b>49</b> and <b>50</b>, <sup>1</sup>NMR spectra of compounds <b>29</b>, <b>30</b>, <b>36</b>, and <b>37</b>, and selectivity data for compounds <b>29</b> and <b>30</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_001.pdf">jm7b01674_si_001.pdf (1.23 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_002.csv">jm7b01674_si_002.csv (2.2 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01674" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55733" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55733" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Liu</span> - <span class="hlFld-Affiliation affiliation">Jiangsu
Key Laboratory of Translational Research and Therapy for Neuro-Psycho-Diseases
and Department of Medicinal Chemistry, College of Pharmaceutical Sciences, Soochow University, Suzhou, Jiangsu 215123, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2669-5448" title="Orcid link">http://orcid.org/0000-0003-2669-5448</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#81e7ede8f4b3c1f2f4e5e0afe4e5f4afe2ef"><span class="__cf_email__" data-cfemail="9cfaf0f5e9aedcefe9f8fdb2f9f8e9b2fff2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhengtian Yu</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#245e4c414a43504d454a0a5d51644a4b524556504d570a474b49"><span class="__cf_email__" data-cfemail="e19b89848f869588808fcf9894a18f8e978093958892cf828e8c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, ON M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3d50134b58595c59547d4849524f52534952135e5c"><span class="__cf_email__" data-cfemail="2b46055d4e4f4a4f426b5e5f445944455f4405484a">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a7cdcec6c989cdcec9e7cad4d4ca89c2c3d2"><span class="__cf_email__" data-cfemail="1f75767e71317576715f726c6c72317a7b6a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">H. Ümit Kaniskan</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad S. Eram</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kehao Zhao</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena M. Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaobao Yang</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keith Schmidt</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao Luo</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sean Xiao</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miao Dai</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng He</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene Zang</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Lin</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Dobrovetsky</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Smil</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sun-Joon Min</span> - <span class="hlFld-Affiliation affiliation">Center
for Chemical Biology and Drug Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0867-4416" title="Orcid link">http://orcid.org/0000-0003-0867-4416</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer Lin-Jones</span> - <span class="hlFld-Affiliation affiliation">DiscoveRx
Corporation, Fremont, California 94538, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, ON M5S 1A8, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Atadja</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">En Li</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research (China), Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Biophysics, University of Toronto
and Princess Margaret Cancer Centre, 101 College Street, MaRS South Tower, Suite 707, Toronto, ON M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): K.Z., X.L., S.X., M.D., F.H., I.Z., Y.L., P.A., E.L., and Z.Y. are/were employees of Novartis. J.L. is an employee of DiscoveRx Corporation.<br /></br></div></li></ul></div><div class="ack" id="ACK-d215e4214-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80648" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80648" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants R01GM122749, R01HD088626 and R01CA218600 (to J.J.) from the U.S. National Institutes of Health. F.L. was supported by the grant 21302134 (to F.L.) from the National Natural Science Foundation of China. The SGC is a registered charity (No. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD Grant 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, Saõ Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust. GPCR, kinase, and ion channel off-target selectivity screening was kindly supplied by Eurofins-Cerep.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i64" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i64"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i65" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i65"> ABBREVIATIONS</h2><tr><td class="NLM_term">PRMT3</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 3</p></td></tr><tr><td class="NLM_term">rpS2</td><td class="NLM_def"><p class="first last">ribosomal protein S2</p></td></tr><tr><td class="NLM_term">PABPN1</td><td class="NLM_def"><p class="first last">recombinant mammalian nuclear poly(A)-binding protein</p></td></tr><tr><td class="NLM_term">LXRα</td><td class="NLM_def"><p class="first last">liver X receptor α</p></td></tr><tr><td class="NLM_term">DAL-1</td><td class="NLM_def"><p class="first last">differentially expressed in adenocarcinoma of the lung, also known as 4.1B</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">LHS</td><td class="NLM_def"><p class="first last">left-hand side</p></td></tr><tr><td class="NLM_term">RHS</td><td class="NLM_def"><p class="first last">right-hand side</p></td></tr><tr><td class="NLM_term">GPCRs</td><td class="NLM_def"><p class="first last">G-protein coupled receptors</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60535" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60535" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein Arginine Methylation in Mammals: Who, What, and Why</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+Arginine+Methylation+in+Mammals%3A+Who%2C+What%2C+and+Why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0liMvF6VUr2S9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520Arginine%2520Methylation%2520in%2520Mammals%253A%2520Who%252C%2520What%252C%2520and%2520Why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Person, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal Protein S2 is A Substrate for Mammalian PRMT3 (Protein Arginine Methyltransferase 3)</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">386</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1042/BJ20041466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1042%2FBJ20041466" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2005&pages=85-91&author=R.+Swierczauthor=M.+D.+Personauthor=M.+T.+Bedford&title=Ribosomal+Protein+S2+is+A+Substrate+for+Mammalian+PRMT3+%28Protein+Arginine+Methyltransferase+3%29&doi=10.1042%2FBJ20041466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1042%2FBJ20041466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20041466%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DPerson%26aufirst%3DM.%2BD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520Protein%2520S2%2520is%2520A%2520Substrate%2520for%2520Mammalian%2520PRMT3%2520%2528Protein%2520Arginine%2520Methyltransferase%25203%2529%26jtitle%3DBiochem.%2520J.%26date%3D2005%26volume%3D386%26spage%3D85%26epage%3D91%26doi%3D10.1042%2FBJ20041466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal Protein rpS2 is Hypomethylated in PRMT3-Deficient Mice</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">16917</span><span class="NLM_x">–</span> <span class="NLM_lpage">16923</span><span class="refDoi"> DOI: 10.1074/jbc.M609778200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M609778200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=17439947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=16917-16923&author=R.+Swierczauthor=D.+Chengauthor=D.+Kimauthor=M.+T.+Bedford&title=Ribosomal+Protein+rpS2+is+Hypomethylated+in+PRMT3-Deficient+Mice&doi=10.1074%2Fjbc.M609778200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ribosomal Protein rpS2 Is Hypomethylated in PRMT3-deficient Mice</span></div><div class="casAuthors">Swiercz, Rafal; Cheng, Donghang; Kim, Daehoon; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16917-16923</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PRMT3 is a type I arginine methyltransferase that resides in the cytoplasm.  A large proportion of this cytosolic PRMT3 is found assocd. with ribosomes.  It is tethered to the ribosomes through its interaction with rpS2, which is also its substrate.  Here we show that mouse embryos with a targeted disruption of PRMT3 are small in size but survive after birth and attain a normal size in adulthood, thus displaying Minute-like characteristics.  The ribosome protein rpS2 is hypomethylated in the absence of PRMT3, demonstrating that it is a bona fide, in vivo PRMT3 substrate that cannot be modified by other PRMTs.  Finally, the levels 40 S, 60 S, and 80 S monosomes and polyribosomes are unaffected by the loss of PRMT3, but there are addnl. as yet unidentified proteins that co-fractionate with ribosomes that are also dedicated PRMT3 substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-lEvztlUF4rVg90H21EOLACvtfcHk0liWbyu8fI5LYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D&md5=4d97b44b6706559086aa2554df786b31</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M609778200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M609778200%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520Protein%2520rpS2%2520is%2520Hypomethylated%2520in%2520PRMT3-Deficient%2520Mice%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D16917%26epage%3D16923%26doi%3D10.1074%2Fjbc.M609778200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Bachand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silver, P. A.</span><span> </span><span class="NLM_article-title">PRMT3 is A Ribosomal Protein Methyltransferase that Affects the Cellular Levels of Ribosomal Subunits</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">2641</span><span class="NLM_x">–</span> <span class="NLM_lpage">2650</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7600265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1038%2Fsj.emboj.7600265" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2641-2650&author=F.+Bachandauthor=P.+A.+Silver&title=PRMT3+is+A+Ribosomal+Protein+Methyltransferase+that+Affects+the+Cellular+Levels+of+Ribosomal+Subunits&doi=10.1038%2Fsj.emboj.7600265"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600265%26sid%3Dliteratum%253Aachs%26aulast%3DBachand%26aufirst%3DF.%26aulast%3DSilver%26aufirst%3DP.%2BA.%26atitle%3DPRMT3%2520is%2520A%2520Ribosomal%2520Protein%2520Methyltransferase%2520that%2520Affects%2520the%2520Cellular%2520Levels%2520of%2520Ribosomal%2520Subunits%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2641%26epage%3D2650%26doi%3D10.1038%2Fsj.emboj.7600265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Landry-Voyer, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilodeau, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dionne, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Port, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouleau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kehlenbach, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachand, F.</span><span> </span><span class="NLM_article-title">Human PDCD2L Is an Export Substrate of CRM1 That Associates with 40S Ribosomal Subunit Precursors</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3019</span><span class="NLM_x">–</span> <span class="NLM_lpage">3032</span><span class="refDoi"> DOI: 10.1128/MCB.00303-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1128%2FMCB.00303-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=27697862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitl2mtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=3019-3032&author=A.+M.+Landry-Voyerauthor=S.+Bilodeauauthor=D.+Bergeronauthor=K.+L.+Dionneauthor=S.+A.+Portauthor=C.+Rouleauauthor=F.+M.+Boisvertauthor=R.+H.+Kehlenbachauthor=F.+Bachand&title=Human+PDCD2L+Is+an+Export+Substrate+of+CRM1+That+Associates+with+40S+Ribosomal+Subunit+Precursors&doi=10.1128%2FMCB.00303-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Human PDCD2L is an export substrate of CRM1 that associates with 40S ribosomal subunit precursors</span></div><div class="casAuthors">Landry-Voyer, Anne-Marie; Bilodeau, Sarah; Bergeron, Danny; Dionne, Kiersten L.; Port, Sarah A.; Rouleau, Caroline; Boisvert, Francois-Michel; Kehlenbach, Ralph H.; Bachand, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3019-3032</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 3 (PRMT3) forms a stable complex with 40S ribosomal protein S2 (RPS2) and contributes to ribosome biogenesis.  However, the mol. mechanism by which PRMT3 influences ribosome biogenesis and/or function still remains unclear.  Using quant. proteomics, we identified human programmed cell death 2-like (PDCD2L) as a novel PRMT3-assocd. protein.  Our data suggest that RPS2 promotes the formation of a conserved extraribosomal complex with PRMT3 and PDCD2L.  We also show that PDCD2L assocs. with 40S subunit precursors that contain a 3'-extended form of the 18S rRNA (18S-E pre-rRNA) and several pre-40S maturation factors.  PDCD2L shuttles between the nucleus and the cytoplasm in a CRM1-dependent manner using a leucine-rich nuclear export signal that is sufficient to direct the export of a reporter protein.  Although PDCD2L is not required for the biogenesis and export of 40S ribosomal subunits, we found that PDCD2L-null cells accumulate free 60S ribosomal subunits, which is indicative of a deficiency in 40S subunit availability.  Our data also indicate that PDCD2L and its paralog, PDCD2, function redundantly in 40S ribosomal subunit prodn.  Our findings uncover the existence of an extraribosomal complex consisting of PDCD2L, RPS2, and PRMT3 and support a role for PDCD2L in the late maturation of 40S ribosomal subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbG98djoI-UrVg90H21EOLACvtfcHk0liWbyu8fI5LYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitl2mtbs%253D&md5=e5b0325e82737f37a4c8a4bcbeabb158</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.00303-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00303-16%26sid%3Dliteratum%253Aachs%26aulast%3DLandry-Voyer%26aufirst%3DA.%2BM.%26aulast%3DBilodeau%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DD.%26aulast%3DDionne%26aufirst%3DK.%2BL.%26aulast%3DPort%26aufirst%3DS.%2BA.%26aulast%3DRouleau%26aufirst%3DC.%26aulast%3DBoisvert%26aufirst%3DF.%2BM.%26aulast%3DKehlenbach%26aufirst%3DR.%2BH.%26aulast%3DBachand%26aufirst%3DF.%26atitle%3DHuman%2520PDCD2L%2520Is%2520an%2520Export%2520Substrate%2520of%2520CRM1%2520That%2520Associates%2520with%252040S%2520Ribosomal%2520Subunit%2520Precursors%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2016%26volume%3D36%26spage%3D3019%26epage%3D3032%26doi%3D10.1128%2FMCB.00303-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gary, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span> </span><span class="NLM_article-title">Prmt 3, A Type I Protein Arginine N-Methyltransferase that Differs From PRMT1 in its Oligomerization, Subcellular Localization, Substrate Specificity, and Regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">16935</span><span class="NLM_x">–</span> <span class="NLM_lpage">16945</span><span class="refDoi"> DOI: 10.1074/jbc.273.27.16935</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.273.27.16935" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=16935-16945&author=J.+Tangauthor=J.+D.+Garyauthor=S.+Clarkeauthor=H.+R.+Herschman&title=Prmt+3%2C+A+Type+I+Protein+Arginine+N-Methyltransferase+that+Differs+From+PRMT1+in+its+Oligomerization%2C+Subcellular+Localization%2C+Substrate+Specificity%2C+and+Regulation&doi=10.1074%2Fjbc.273.27.16935"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.27.16935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.27.16935%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DGary%26aufirst%3DJ.%2BD.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26atitle%3DPrmt%25203%252C%2520A%2520Type%2520I%2520Protein%2520Arginine%2520N-Methyltransferase%2520that%2520Differs%2520From%2520PRMT1%2520in%2520its%2520Oligomerization%252C%2520Subcellular%2520Localization%252C%2520Substrate%2520Specificity%252C%2520and%2520Regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D16935%26epage%3D16945%26doi%3D10.1074%2Fjbc.273.27.16935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Kim, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafsson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. H.</span><span> </span><span class="NLM_article-title">PRMT3 Regulates Hepatic Lipogenesis Through Direct Interaction with LXRalpha</span> <span class="citation_source-journal">Diabetes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.2337/db13-1394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.2337%2Fdb13-1394" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2015&pages=60-71&author=D.+I.+Kimauthor=M.+J.+Parkauthor=S.+K.+Limauthor=J.+I.+Parkauthor=K.+C.+Yoonauthor=H.+J.+Hanauthor=J.+A.+Gustafssonauthor=J.+H.+Limauthor=S.+H.+Park&title=PRMT3+Regulates+Hepatic+Lipogenesis+Through+Direct+Interaction+with+LXRalpha&doi=10.2337%2Fdb13-1394"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2337%2Fdb13-1394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb13-1394%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%2BI.%26aulast%3DPark%26aufirst%3DM.%2BJ.%26aulast%3DLim%26aufirst%3DS.%2BK.%26aulast%3DPark%26aufirst%3DJ.%2BI.%26aulast%3DYoon%26aufirst%3DK.%2BC.%26aulast%3DHan%26aufirst%3DH.%2BJ.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26aulast%3DLim%26aufirst%3DJ.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BH.%26atitle%3DPRMT3%2520Regulates%2520Hepatic%2520Lipogenesis%2520Through%2520Direct%2520Interaction%2520with%2520LXRalpha%26jtitle%3DDiabetes%26date%3D2015%26volume%3D64%26spage%3D60%26epage%3D71%26doi%3D10.2337%2Fdb13-1394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Fronz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolbel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierhorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck-Sickinger, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostareck-Lederer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahle, E.</span><span> </span><span class="NLM_article-title">Promiscuous Modification of the Nuclear Poly(A)-Binding Protein by Multiple Protein-Arginine Methyltransferases Does Not Affect The Aggregation Behavior</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">20408</span><span class="NLM_x">–</span> <span class="NLM_lpage">20420</span><span class="refDoi"> DOI: 10.1074/jbc.M802329200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M802329200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=20408-20420&author=K.+Fronzauthor=S.+Ottoauthor=K.+Kolbelauthor=U.+Kuhnauthor=H.+Friedrichauthor=A.+Schierhornauthor=A.+G.+Beck-Sickingerauthor=A.+Ostareck-Ledererauthor=E.+Wahle&title=Promiscuous+Modification+of+the+Nuclear+Poly%28A%29-Binding+Protein+by+Multiple+Protein-Arginine+Methyltransferases+Does+Not+Affect+The+Aggregation+Behavior&doi=10.1074%2Fjbc.M802329200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802329200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802329200%26sid%3Dliteratum%253Aachs%26aulast%3DFronz%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DS.%26aulast%3DKolbel%26aufirst%3DK.%26aulast%3DKuhn%26aufirst%3DU.%26aulast%3DFriedrich%26aufirst%3DH.%26aulast%3DSchierhorn%26aufirst%3DA.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DOstareck-Lederer%26aufirst%3DA.%26aulast%3DWahle%26aufirst%3DE.%26atitle%3DPromiscuous%2520Modification%2520of%2520the%2520Nuclear%2520Poly%2528A%2529-Binding%2520Protein%2520by%2520Multiple%2520Protein-Arginine%2520Methyltransferases%2520Does%2520Not%2520Affect%2520The%2520Aggregation%2520Behavior%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D20408%26epage%3D20420%26doi%3D10.1074%2Fjbc.M802329200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Tavanez, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bengoechea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berciano, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafarga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmo-Fonseca, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enguita, F. J.</span><span> </span><span class="NLM_article-title">Hsp70 Chaperones and Type I PRMTs are Sequestered at Intranuclear Inclusions Caused by Polyalanine Expansions in PABPN1</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e6418</span><span class="refDoi"> DOI: 10.1371/journal.pone.0006418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1371%2Fjournal.pone.0006418" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e6418&author=J.+P.+Tavanezauthor=R.+Bengoecheaauthor=M.+T.+Bercianoauthor=M.+Lafargaauthor=M.+Carmo-Fonsecaauthor=F.+J.+Enguita&title=Hsp70+Chaperones+and+Type+I+PRMTs+are+Sequestered+at+Intranuclear+Inclusions+Caused+by+Polyalanine+Expansions+in+PABPN1&doi=10.1371%2Fjournal.pone.0006418"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0006418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0006418%26sid%3Dliteratum%253Aachs%26aulast%3DTavanez%26aufirst%3DJ.%2BP.%26aulast%3DBengoechea%26aufirst%3DR.%26aulast%3DBerciano%26aufirst%3DM.%2BT.%26aulast%3DLafarga%26aufirst%3DM.%26aulast%3DCarmo-Fonseca%26aufirst%3DM.%26aulast%3DEnguita%26aufirst%3DF.%2BJ.%26atitle%3DHsp70%2520Chaperones%2520and%2520Type%2520I%2520PRMTs%2520are%2520Sequestered%2520at%2520Intranuclear%2520Inclusions%2520Caused%2520by%2520Polyalanine%2520Expansions%2520in%2520PABPN1%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De6418%26doi%3D10.1371%2Fjournal.pone.0006418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Lai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hakala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weintraub, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiio, Y.</span><span> </span><span class="NLM_article-title">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">5175</span><span class="NLM_x">–</span> <span class="NLM_lpage">5182</span><span class="refDoi"> DOI: 10.1021/pr200642c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr200642c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5175-5182&author=Y.+Laiauthor=M.+Songauthor=K.+Hakalaauthor=S.+T.+Weintraubauthor=Y.+Shiio&title=Proteomic+Dissection+of+the+von+Hippel-Lindau+%28VHL%29+Interactome&doi=10.1021%2Fpr200642c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span></div><div class="casAuthors">Lai, Yanlai; Song, Meihua; Hakala, Kevin; Weintraub, Susan T.; Shiio, Yuzuru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5175-5182</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) tumor suppressor gene encodes a component of a ubiquitin ligase complex contg. elongin B, elongin C, cullin 2, and Rbx1, which acts as a neg. regulator of hypoxia inducible factor (HIF).  VHL ubiquitinates and degrades the alpha subunits of HIF, and this is proposed to suppress tumorigenesis and tumor angiogenesis.  Several lines of evidence also suggest important roles for HIF-independent VHL functions in the maintenance of primary cilium, extracellular matrix formation, and tumor suppression.  We undertook a series of proteomic analyses to gain a comprehensive picture of the VHL-interacting proteins.  We found that the ARF tumor suppressor interacts with VHL30, a longer VHL isoform, but not with VHL19, a shorter VHL isoform.  ARF was found to release VHL30 from the E3 ligase complex, promoting the binding of VHL30 to a protein arginine methyltransferase, PRMT3.  Our anal. of the VHL19 interactome also uncovered that VHL19 displays an affinity to collagens and their biosynthesis enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqTYZ6RvZ7erVg90H21EOLACvtfcHk0ljr68KTpEKQ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL&md5=51500af3ec4256311dbd835c47a90200</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fpr200642c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200642c%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHakala%26aufirst%3DK.%26aulast%3DWeintraub%26aufirst%3DS.%2BT.%26aulast%3DShiio%26aufirst%3DY.%26atitle%3DProteomic%2520Dissection%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520Interactome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5175%26epage%3D5182%26doi%3D10.1021%2Fpr200642c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Singh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranda, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roemer, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutmann, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newsham, I. F.</span><span> </span><span class="NLM_article-title">DAL-1/4.1B Tumor Suppressor Interacts with Protein Arginine N-Methyltransferase 3 (PRMT3) and Inhibits its Ability to Methylate Substrates In Vitro and In Vivo</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">7761</span><span class="NLM_x">–</span> <span class="NLM_lpage">7771</span><span class="refDoi"> DOI: 10.1038/sj.onc.1208057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1038%2Fsj.onc.1208057" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=7761-7771&author=V.+Singhauthor=T.+B.+Mirandaauthor=W.+Jiangauthor=A.+Frankelauthor=M.+E.+Roemerauthor=V.+A.+Robbauthor=D.+H.+Gutmannauthor=H.+R.+Herschmanauthor=S.+Clarkeauthor=I.+F.+Newsham&title=DAL-1%2F4.1B+Tumor+Suppressor+Interacts+with+Protein+Arginine+N-Methyltransferase+3+%28PRMT3%29+and+Inhibits+its+Ability+to+Methylate+Substrates+In+Vitro+and+In+Vivo&doi=10.1038%2Fsj.onc.1208057"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1208057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1208057%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DMiranda%26aufirst%3DT.%2BB.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DRoemer%26aufirst%3DM.%2BE.%26aulast%3DRobb%26aufirst%3DV.%2BA.%26aulast%3DGutmann%26aufirst%3DD.%2BH.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DNewsham%26aufirst%3DI.%2BF.%26atitle%3DDAL-1%252F4.1B%2520Tumor%2520Suppressor%2520Interacts%2520with%2520Protein%2520Arginine%2520N-Methyltransferase%25203%2520%2528PRMT3%2529%2520and%2520Inhibits%2520its%2520Ability%2520to%2520Methylate%2520Substrates%2520In%2520Vitro%2520and%2520In%2520Vivo%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D7761%26epage%3D7771%26doi%3D10.1038%2Fsj.onc.1208057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newsham, I. F.</span><span> </span><span class="NLM_article-title">The Tumor Suppressor DAL-1/4.1B and Protein Methylation Cooperate in Inducing Apoptosis in MCF-7 Breast Cancer Cells</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="refDoi"> DOI: 10.1186/1476-4598-5-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1186%2F1476-4598-5-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=16420693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A280%3ADC%252BD1c7ntFSltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=4&author=W.+Jiangauthor=I.+F.+Newsham&title=The+Tumor+Suppressor+DAL-1%2F4.1B+and+Protein+Methylation+Cooperate+in+Inducing+Apoptosis+in+MCF-7+Breast+Cancer+Cells&doi=10.1186%2F1476-4598-5-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The tumor suppressor DAL-1/4.1B and protein methylation cooperate in inducing apoptosis in MCF-7 breast cancer cells</span></div><div class="casAuthors">Jiang Wei; Newsham Irene F</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  DAL-1 (Differentially Expressed in Adenocarcinoma of the Lung)/4.1B is a member of the protein 4.1 superfamily that has been shown to suppress growth in lung, breast and brain tumor cells.  In the case of the caspase-3 deficient MCF-7 breast cancer cells, this growth suppression has been shown to be partially mediated by the induction of apoptosis.  However the exact mechanism of action of DAL-1/4.1B is unknown.  Recently, protein arginine N-methyltransferase 3 (PRMT3) was identified as a DAL-1/4.1B interacting protein.  Protein arginine methyltransferases (PRMTs) posttranslationally methylate the arginine residues of proteins, a modification which has been implicated in the regulation of multiple cellular processes including nuclear-cytoplasmic transport, signal transduction, and transcription.  RESULTS:  To investigate the role of protein methylation in cell death induced by DAL-1/4.1B, DAL-1/4.1B-inducible MCF-7 cells were examined for apoptosis and caspase activation in the absence and presence of the protein methylation inhibitor adenosine dialdehyde (AdOX).  Flow cytometry analysis revealed that apoptosis was primarily associated with the activation of caspase 8, and inhibition of this activation blocked the ability of DAL-1/4.1B to induce cell death.  CONCLUSION:  These results suggest that protein methylation cooperates with DAL-1/4.1B-associated caspase 8-specific activation to induce apoptosis in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZnGCV1W8gGgpJitSxOdIlfW6udTcc2eYXpBlnATrcyLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7ntFSltw%253D%253D&md5=d2cf59aba875c4f913920cbb6e3cc4f2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-4%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DNewsham%26aufirst%3DI.%2BF.%26atitle%3DThe%2520Tumor%2520Suppressor%2520DAL-1%252F4.1B%2520and%2520Protein%2520Methylation%2520Cooperate%2520in%2520Inducing%2520Apoptosis%2520in%2520MCF-7%2520Breast%2520Cancer%2520Cells%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D4%26doi%3D10.1186%2F1476-4598-5-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Fluorescence-Based Methods For Screening Writers And Readers Of Histone Methyl Marks</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1177/1087057111422256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Fluorescence-Based+Methods+For+Screening+Writers+And+Readers+Of+Histone+Methyl+Marks&doi=10.1177%2F1087057111422256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0lhv432CBgTBUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DFluorescence-Based%2520Methods%2520For%2520Screening%2520Writers%2520And%2520Readers%2520Of%2520Histone%2520Methyl%2520Marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84%26doi%3D10.1177%2F1087057111422256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Herrmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pably, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckerich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fackelmayer, F. O.</span><span> </span><span class="NLM_article-title">Human Protein Arginine Methyltransferases in Vivo-Distinct Properties of Eight Canonical Members of the PRMT Family</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">677</span><span class="refDoi"> DOI: 10.1242/jcs.039933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1242%2Fjcs.039933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=19208762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=667-677&author=F.+Herrmannauthor=P.+Pablyauthor=C.+Eckerichauthor=M.+T.+Bedfordauthor=F.+O.+Fackelmayer&title=Human+Protein+Arginine+Methyltransferases+in+Vivo-Distinct+Properties+of+Eight+Canonical+Members+of+the+PRMT+Family&doi=10.1242%2Fjcs.039933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Human protein arginine methyltransferases in vivo - distinct properties of eight canonical members of the PRMT family</span></div><div class="casAuthors">Herrmann, Frank; Pably, Peter; Eckerich, Carmen; Bedford, Mark T.; Fackelmayer, Frank O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-677</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Methylation of arginine residues is a widespread post-translational modification of proteins catalyzed by a small family of protein arginine methyltransferases (PRMTs).  Functionally, the modification appears to regulate protein functions and interactions that affect gene regulation, signalling and subcellular localization of proteins and nucleic acids.  All members have been, to different degrees, characterized individually and their implication in cellular processes has been inferred from characterizing substrates and interactions.  Here, we report the first comprehensive comparison of all eight canonical members of the human PRMT family with respect to subcellular localization and dynamics in living cells.  We show that the individual family members differ significantly in their properties, as well as in their substrate specificities, suggesting that they fulfil distinctive, non-redundant functions in vivo.  In addn., certain PRMTs display different subcellular localization in different cell types, implicating cell- and tissue-specific mechanisms for regulating PRMT functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4G4_wbI1I-rVg90H21EOLACvtfcHk0lhv432CBgTBUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D&md5=939c3e7c7c0030afbe7e4c5eaa1852ed</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fjcs.039933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.039933%26sid%3Dliteratum%253Aachs%26aulast%3DHerrmann%26aufirst%3DF.%26aulast%3DPably%26aufirst%3DP.%26aulast%3DEckerich%26aufirst%3DC.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DFackelmayer%26aufirst%3DF.%2BO.%26atitle%3DHuman%2520Protein%2520Arginine%2520Methyltransferases%2520in%2520Vivo-Distinct%2520Properties%2520of%2520Eight%2520Canonical%2520Members%2520of%2520the%2520PRMT%2520Family%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2009%26volume%3D122%26spage%3D667%26epage%3D677%26doi%3D10.1242%2Fjcs.039933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Wagner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinschmidt, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, U. M.</span><span> </span><span class="NLM_article-title">SET-Mediated Promoter Hypoacetylation is A Prerequisite for Coactivation of the Estrogen-Responsive pS2 Gene by PRMT1</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">27242</span><span class="NLM_x">–</span> <span class="NLM_lpage">27250</span><span class="refDoi"> DOI: 10.1074/jbc.M605172200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M605172200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=27242-27250&author=S.+Wagnerauthor=S.+Weberauthor=M.+A.+Kleinschmidtauthor=K.+Nagataauthor=U.+M.+Bauer&title=SET-Mediated+Promoter+Hypoacetylation+is+A+Prerequisite+for+Coactivation+of+the+Estrogen-Responsive+pS2+Gene+by+PRMT1&doi=10.1074%2Fjbc.M605172200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605172200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605172200%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DS.%26aulast%3DWeber%26aufirst%3DS.%26aulast%3DKleinschmidt%26aufirst%3DM.%2BA.%26aulast%3DNagata%26aufirst%3DK.%26aulast%3DBauer%26aufirst%3DU.%2BM.%26atitle%3DSET-Mediated%2520Promoter%2520Hypoacetylation%2520is%2520A%2520Prerequisite%2520for%2520Coactivation%2520of%2520the%2520Estrogen-Responsive%2520pS2%2520Gene%2520by%2520PRMT1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D27242%26epage%3D27250%26doi%3D10.1074%2Fjbc.M605172200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Obianyo, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Causey, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, P. R.</span><span> </span><span class="NLM_article-title">Activity-Based Protein Profiling of Protein Arginine Methyltransferase 1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1127</span><span class="NLM_x">–</span> <span class="NLM_lpage">1135</span><span class="refDoi"> DOI: 10.1021/cb2001473</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb2001473" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1127-1135&author=O.+Obianyoauthor=C.+P.+Causeyauthor=J.+E.+Jonesauthor=P.+R.+Thompson&title=Activity-Based+Protein+Profiling+of+Protein+Arginine+Methyltransferase+1&doi=10.1021%2Fcb2001473"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fcb2001473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb2001473%26sid%3Dliteratum%253Aachs%26aulast%3DObianyo%26aufirst%3DO.%26aulast%3DCausey%26aufirst%3DC.%2BP.%26aulast%3DJones%26aufirst%3DJ.%2BE.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DActivity-Based%2520Protein%2520Profiling%2520of%2520Protein%2520Arginine%2520Methyltransferase%25201%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D1127%26epage%3D1135%26doi%3D10.1021%2Fcb2001473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niroomand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zankl, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katus, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiefenbacher, C. P.</span><span> </span><span class="NLM_article-title">Expression of Nitric Oxide Related Enzymes in Coronary Heart Disease</span> <span class="citation_source-journal">Basic Res. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">346</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1007/s00395-006-0592-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1007%2Fs00395-006-0592-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=16705470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2006&pages=346-353&author=X.+Chenauthor=F.+Niroomandauthor=Z.+Liuauthor=A.+Zanklauthor=H.+A.+Katusauthor=L.+Jahnauthor=C.+P.+Tiefenbacher&title=Expression+of+Nitric+Oxide+Related+Enzymes+in+Coronary+Heart+Disease&doi=10.1007%2Fs00395-006-0592-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nitric oxide related enzymes in coronary heart disease</span></div><div class="casAuthors">Chen, X.; Niroomand, F.; Liu, Z.; Zankl, A.; Katus, H. A.; Jahn, L.; Tiefenbacher, C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Basic Research in Cardiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">346-353</span>CODEN:
                <span class="NLM_cas:coden">BRCAB7</span>;
        ISSN:<span class="NLM_cas:issn">0300-8428</span>.
    
            (<span class="NLM_cas:orgname">Steinkopff Verlag</span>)
        </div><div class="casAbstract">Enzymes involved in the metab. nitric oxide (NO) and reactive oxygen species (ROS) may play a role for the decreased availability of NO in atherosclerosis.  We, therefore, hypothesized that the pattern of gene expression of these enzymes is altered in atherosclerosis.  Myocardial tissue from patients with coronary heart disease (CHD) or without CHD (control group) was investigated.  The level of enzymes related to NO/ROS metab. was detd. both at mRNA level and protein level by rt-PCR, real-time PCR, and western blot.  The expression of NOS1-3 (synthesis of NO), arginasel (redn. of L-arginine), p22phox (active subunit of NADPH oxidase), GTPCH (rate limiting enzyme for tetrahydrobiopterin), SOD1-3 (scavengers of superoxide anions), PRTMT1-3, and DDAH2 (involved in the metab. of ADMA) was detd.  All enzymes were found to be expressed in human myocardium.  NOS isoforms were decreased in CHD in protein level, but only the down regulation of NOS3 expression reached statistical significance.  The expression of PRMT1 and PRMT3 was increased.  In addn., the expression of DDAH2 was reduced, both theor. leading to an increase of ADMA concn.  SOD3 was down regulated in tissue from patients with CHD.  Taken together, in myocardial tissue from patients with atherosclerosis, the expression of genes increasing ADMA levels is enhanced in contrast to a reduced expression of genes promoting NO synthesis.  These results may contribute to the explanation of increased oxidative stress in atherosclerosis on the level of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZhed3XxK3bVg90H21EOLACvtfcHk0lhu0sDDOevriQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D&md5=e40c38e03c6dc050a5a894bd0be20155</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs00395-006-0592-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00395-006-0592-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DNiroomand%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZankl%26aufirst%3DA.%26aulast%3DKatus%26aufirst%3DH.%2BA.%26aulast%3DJahn%26aufirst%3DL.%26aulast%3DTiefenbacher%26aufirst%3DC.%2BP.%26atitle%3DExpression%2520of%2520Nitric%2520Oxide%2520Related%2520Enzymes%2520in%2520Coronary%2520Heart%2520Disease%26jtitle%3DBasic%2520Res.%2520Cardiol.%26date%3D2006%26volume%3D101%26spage%3D346%26epage%3D353%26doi%3D10.1007%2Fs00395-006-0592-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Miyata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohyama, M.</span><span> </span><span class="NLM_article-title">PRMT3 is Essential for Dendritic Spine Maturation in Rat Hippocampal Neurons</span> <span class="citation_source-journal">Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1352</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/j.brainres.2010.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.brainres.2010.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=20647003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1352&publication_year=2010&pages=11-20&author=S.+Miyataauthor=Y.+Moriauthor=M.+Tohyama&title=PRMT3+is+Essential+for+Dendritic+Spine+Maturation+in+Rat+Hippocampal+Neurons&doi=10.1016%2Fj.brainres.2010.07.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT3 is essential for dendritic spine maturation in rat hippocampal neurons</span></div><div class="casAuthors">Miyata, Shingo; Mori, Yasutake; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1352</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-20</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine N-methyltransferase 3 (PRMT3) is a cytoplasmic enzyme that utilizes S-adenosyl-L-methionine (AdoMet) to methylate specific proteins, most of which contain GAR (glycine-arginine rich) motifs.  PRMT3 has been shown to play a role in the proper maturation of the 80S ribosome by binding to and catalyzing the methylation of rpS2, a component of the 40S ribosomal subunit.  However, the other roles of PRMT3 are fairly unclear, particularly in the brain, which is abundant in methylated proteins.  In this study, we perturbed PRMT3 expression in cultured rat hippocampal neurons by transiently introducing siRNA oligonucleotides that were designed to hybridize with PRMT3 mRNA and then we examd. the morphol. and functional effects of neuronal PRMT3 depletion.  PRMT3-defective neurons showed deformed spines without any change in spine no.; less BDNF-induced protein translation of αCaMKII; and diminished rpS2 protein stability.  Furthermore, overexpression of a methylation-resistant rpS2, whose methylated arginine residues were deleted, produced phenotypes that were similar to those assocd. with PRMT3 down regulation.  These findings demonstrated that PRMT3 possibly plays a pivotal role in neuronal translation by interaction with rpS2 and that it contributes to activity-dependent changes in the dendritic spines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZvFaoRRY5lbVg90H21EOLACvtfcHk0lhu0sDDOevriQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L&md5=ccac2a5a931c483e516003342916bb17</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.brainres.2010.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainres.2010.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DPRMT3%2520is%2520Essential%2520for%2520Dendritic%2520Spine%2520Maturation%2520in%2520Rat%2520Hippocampal%2520Neurons%26jtitle%3DBrain%2520Res.%26date%3D2010%26volume%3D1352%26spage%3D11%26epage%3D20%26doi%3D10.1016%2Fj.brainres.2010.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Park, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heo, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. H.</span><span> </span><span class="NLM_article-title">New Role of Irisin in Hepatocytes: The Protective Effect of Hepatic Steatosis In Vitro</span> <span class="citation_source-journal">Cell. Signalling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1831</span><span class="NLM_x">–</span> <span class="NLM_lpage">1839</span><span class="refDoi"> DOI: 10.1016/j.cellsig.2015.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.cellsig.2015.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=25917316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFWgsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=1831-1839&author=M.+J.+Parkauthor=D.+I.+Kimauthor=J.+H.+Choiauthor=Y.+R.+Heoauthor=S.+H.+Park&title=New+Role+of+Irisin+in+Hepatocytes%3A+The+Protective+Effect+of+Hepatic+Steatosis+In+Vitro&doi=10.1016%2Fj.cellsig.2015.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">New role of irisin in hepatocytes: The protective effect of hepatic steatosis in vitro</span></div><div class="casAuthors">Park, Min-Jung; Kim, Dong-Il; Choi, Joo-Hee; Heo, Young-Ran; Park, Soo-Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1831-1839</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Irisin is a newly identified myokine related to exercise and the browning of white fat.  Recently, it was reported that irisin serum levels are assocd. with intrahepatic triglyceride content, suggesting that it might have an important role in the liver.  The aim of this study was to det. the role of irisin in hepatocytes.  Specifically, the effect of recombinant irisin on palmitic acid (PA)-induced lipogenesis and its related signal pathways were examd. in AML12 cells and mouse primary hepatocytes.  In the present study, we obsd. the presence of irisin inside the cells in response to the treatment of recombinant irisin by flow cytometry and cell imaging technique.  Recombinant irisin significantly inhibited the PA-induced increase in lipogenic markers ACC and FAS at the mRNA and protein levels, and prevented the PA-induced lipid accumulation in hepatocytes.  Addnl., irisin inhibited the PA-induced increase in the expression, nuclear localization, and transcriptional activities of the master regulators of lipogenesis (LXRα and SREBP-1c).  Moreover, irisin attenuated PA-induced oxidative stress, which was confirmed by measuring the expression of inflammatory markers (NFκB, COX-2, p38 MAPK, TNF, IL-6) and superoxide indicator (dihydroethidium).  The preventive effects of irisin against lipogenesis and oxidative stress were mediated by the inhibition of protein arginine methyltransferase-3 (PRMT3).  These findings suggested that irisin might have a beneficial role in the prevention of hepatic steatosis by altering the expression of lipogenic genes and attenuating oxidative stress in a PRMT3 dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom0-JmfttH1rVg90H21EOLACvtfcHk0lhu0sDDOevriQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWgsbfO&md5=79d7e48751ea13de650d183d7e81dbbf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2015.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2015.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DM.%2BJ.%26aulast%3DKim%26aufirst%3DD.%2BI.%26aulast%3DChoi%26aufirst%3DJ.%2BH.%26aulast%3DHeo%26aufirst%3DY.%2BR.%26aulast%3DPark%26aufirst%3DS.%2BH.%26atitle%3DNew%2520Role%2520of%2520Irisin%2520in%2520Hepatocytes%253A%2520The%2520Protective%2520Effect%2520of%2520Hepatic%2520Steatosis%2520In%2520Vitro%26jtitle%3DCell.%2520Signalling%26date%3D2015%26volume%3D27%26spage%3D1831%26epage%3D1839%26doi%3D10.1016%2Fj.cellsig.2015.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span><span class="refDoi"> DOI: 10.1016/j.str.2012.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=P.+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+Allosteric+Inhibitor+of+Protein+Arginine+Methyltransferase+3&doi=10.1016%2Fj.str.2012.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0ljkCFbI5obSDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520Allosteric%2520Inhibitor%2520of%2520Protein%2520Arginine%2520Methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435%26doi%3D10.1016%2Fj.str.2012.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting An Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+An+Allosteric+Binding+Site+of+PRMT3+Yields+Potent+and+Selective+Inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0ljkCFbI5obSDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520An%2520Allosteric%2520Binding%2520Site%2520of%2520PRMT3%2520Yields%2520Potent%2520and%2520Selective%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kaniskan, H. Ü.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin-Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5166</span><span class="NLM_x">–</span> <span class="NLM_lpage">5170</span><span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+%C3%9C.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+Yuauthor=M.+S.+Eramauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+Liauthor=E.+Dobrovetskyauthor=A.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+Zhaoauthor=J.+Jinauthor=M.+Vedadi&title=A+Potent%2C+Selective+and+Cell-Active+Allosteric+Inhibitor+of+Protein+Arginine+Methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Uemit; Szewczyk, Magdalena M.; Yu, Zhengtian; Eram, Mohammad S.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Kennedy, Steven; Li, Fengling; Dobrovetsky, Elena; Dong, Aiping; Smil, David; Min, Sun-Joon; Landon, Melissa; Lin-Jones, Jennifer; Huang, Xi-Ping; Roth, Bryan L.; Schapira, Matthieu; Atadja, Peter; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Zhao, Kehao; Jin, Jian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-5170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biol.  Here we report the discovery of the first PRMT3 chem. probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochem., biophys., and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50=31±2 nM, KD=53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chem. probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTk4dFfkDLm7Vg90H21EOLACvtfcHk0ljkCFbI5obSDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D&md5=283b0875813af0f7631235e8ba7811c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520Potent%252C%2520Selective%2520and%2520Cell-Active%2520Allosteric%2520Inhibitor%2520of%2520Protein%2520Arginine%2520Methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Balz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiemann, G.</span><span> </span><span class="NLM_article-title">On Aromatic Fluoric Compounds, I.: A New Method for its Representation</span> <span class="citation_source-journal">Ber. Dtsch. Chem. Ges. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1927</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1186</span><span class="NLM_x">–</span> <span class="NLM_lpage">1190</span><span class="refDoi"> DOI: 10.1002/cber.19270600539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1002%2Fcber.19270600539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADyaB2sXitFyhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1927&pages=1186-1190&author=G.+Balzauthor=G.+Schiemann&title=On+Aromatic+Fluoric+Compounds%2C+I.%3A+A+New+Method+for+its+Representation&doi=10.1002%2Fcber.19270600539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatic fluorine compounds. I. A new method for their preparation</span></div><div class="casAuthors">Balz, Gunther; Schiemann, Gunther</div><div class="citationInfo"><span class="NLM_cas:title">Berichte der Deutschen Chemischen Gesellschaft [Abteilung] B: Abhandlungen</span>
        (<span class="NLM_cas:date">1927</span>),
    <span class="NLM_cas:volume">60B</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1186-90</span>CODEN:
                <span class="NLM_cas:coden">BDCBAD</span>;
        ISSN:<span class="NLM_cas:issn">0365-9488</span>.
    </div><div class="casAbstract">Contrary to the earlier belief, the diazonium fluoborates (cf. C. A. 21, 1070) decomp. smoothly and quietly on warming: ArN2BF4 → ArF + N2 + BF3.  There are no side reactions.  Because of their slight soly. they can be obtained in dry form readily and in good yield and they are very stable; they can even be recrystd. from hot H2O and are entirely insensitive to shock.  On decompn. they yield the fluoride in many cases almost quant. and in pure form even when the diazonium fluoborate has not previously been especially purified.  In general the appropriate amine is diazotized in as concd. soln. in HCl as possible, pptd. with excess of HBF4, (about 40%), washed with HBF, alc. and Et2O, dried in vacuo and heated in a fractionating flask (the glass is practically not attacked at all) connected with a condenser, the gaseous BF3 being absorbed in H2O or NaOH.  Phenyldiazonium fluoborate (yield 63 %), becomes slightly pink 80°, decomps. 121-2°, d425 1.53, yields almost 100% PhF, b. 86°.  p-Tolyldiazonium fluoborate (67%), decomps. 110°, d425 1.48; MeC6H4F (97%), b756 116°.  m-Xylyldiazonium fluoborate (31%; the yield Can easily be increased), decomps. 108°, d425 1.50; 2,4-e2C6H2F (almost 100%, b749 143-40°.  α.Naphthyldiazonium fluoborate (62%), decomps. 113°; α-C10H7F(98%),b17110°, b758 215°.  Diphenylene-4,4'-bisdiazonium fluoborate (64%),turns pink about 130°, foams 157°, d425 1.73; (P-FC6H4)2, m. 94-5°, d425 1.328.  o-,m-and p-Nitrophenyl-diazonium fluoborates (yields, 74, 80, 100%, resp.), decomp. 135°, 178°, 156°, d425 1.69, 1.66, 1.66.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCcxfRtVim6LVg90H21EOLACvtfcHk0lhhifv5LAovAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaB2sXitFyhsw%253D%253D&md5=d87f8541798092e16fb21534d69dca75</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fcber.19270600539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.19270600539%26sid%3Dliteratum%253Aachs%26aulast%3DBalz%26aufirst%3DG.%26aulast%3DSchiemann%26aufirst%3DG.%26atitle%3DOn%2520Aromatic%2520Fluoric%2520Compounds%252C%2520I.%253A%2520A%2520New%2520Method%2520for%2520its%2520Representation%26jtitle%3DBer.%2520Dtsch.%2520Chem.%2520Ges.%2520B%26date%3D1927%26volume%3D60%26spage%3D1186%26epage%3D1190%26doi%3D10.1002%2Fcber.19270600539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Zhu, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claiborne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnone, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinghofer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksijew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoll, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Jong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oltersdorf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giranda, V. L.</span><span> </span><span class="NLM_article-title">Isoquinoline-Pyridine-Based Protein Kinase B/Akt Antagonists: SAR and In Vivo Antitumor Activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3150</span><span class="NLM_x">–</span> <span class="NLM_lpage">3155</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2006.03.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.bmcl.2006.03.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=3150-3155&author=G.+D.+Zhuauthor=J.+Gongauthor=A.+Claiborneauthor=K.+W.+Woodsauthor=V.+B.+Gandhiauthor=S.+Thomasauthor=Y.+Luoauthor=X.+Liuauthor=Y.+Shiauthor=R.+Guanauthor=S.+R.+Magnoneauthor=V.+Klinghoferauthor=E.+F.+Johnsonauthor=J.+Bouskaauthor=A.+Shoemakerauthor=A.+Oleksijewauthor=V.+S.+Stollauthor=R.+De+Jongauthor=T.+Oltersdorfauthor=Q.+Liauthor=S.+H.+Rosenbergauthor=V.+L.+Giranda&title=Isoquinoline-Pyridine-Based+Protein+Kinase+B%2FAkt+Antagonists%3A+SAR+and+In+Vivo+Antitumor+Activity&doi=10.1016%2Fj.bmcl.2006.03.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.03.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.03.041%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DG.%2BD.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DClaiborne%26aufirst%3DA.%26aulast%3DWoods%26aufirst%3DK.%2BW.%26aulast%3DGandhi%26aufirst%3DV.%2BB.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DR.%26aulast%3DMagnone%26aufirst%3DS.%2BR.%26aulast%3DKlinghofer%26aufirst%3DV.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DBouska%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DA.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DStoll%26aufirst%3DV.%2BS.%26aulast%3DDe%2BJong%26aufirst%3DR.%26aulast%3DOltersdorf%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DGiranda%26aufirst%3DV.%2BL.%26atitle%3DIsoquinoline-Pyridine-Based%2520Protein%2520Kinase%2520B%252FAkt%2520Antagonists%253A%2520SAR%2520and%2520In%2520Vivo%2520Antitumor%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D3150%26epage%3D3155%26doi%3D10.1016%2Fj.bmcl.2006.03.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">(R)-PFI-2 is a Potent and Selective Inhibitor of SETD7Methyltransferase Activity in Cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.+P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.+C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+Braamauthor=F.+M.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+is+a+Potent+and+Selective+Inhibitor+of+SETD7Methyltransferase+Activity+in+Cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520is%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520SETD7Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Kinetic Characterization of Human Histone H3 Lysine 36 Methyltransferases, ASH1L and SETD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">1842</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2015.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.bbagen.2015.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=26002201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=1842-1848&author=M.+S.+Eramauthor=E.+Kuznetsovaauthor=F.+Liauthor=E.+Lima-Fernandesauthor=S.+Kennedyauthor=I.+Chauauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=Kinetic+Characterization+of+Human+Histone+H3+Lysine+36+Methyltransferases%2C+ASH1L+and+SETD2&doi=10.1016%2Fj.bbagen.2015.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span></div><div class="casAuthors">Eram, Mohammad S.; Kuznetsova, Ekaterina; Li, Fengling; Lima-Fernandes, Evelyne; Kennedy, Steven; Chau, Irene; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dysregulation of methylation of lysine 36 on histone H3 (H3K36) have been implicated in a variety of diseases including cancers.  ASH1L and SETD2 are two enzymes among others that catalyze H3K36 methylation.  H3K4 methylation has also been reported for ASH1L.  Radioactivity-based enzyme assays, Western and immunoblotting using specific antibodies and mol. modeling were used to characterize substrate specificity of ASH1L and SETD2.  Here we report on the assay development and kinetic characterization of ASH1L and SETD2 and their substrate specificities in vitro.  Both enzymes were active with recombinant nucleosome as substrate.  However, SETD2 but not ASH1L methylated histone peptides as well, indicating that the interaction of the basic post-SET extension with substrate may not be crit. for SETD2 activity.  Neither enzyme was active with nucleosome contg. a H3K36A mutation, indicating their specificity for H3K36.  Analyzing the methylation state of the products of ASH1L and SETD2 reactions also confirmed that both enzymes mono- and dimethylate H3K36 and are inactive with H3K4 as substrate, and that only SETD2 is able to trimethylate H3K36 in vitro.  We detd. the kinetic parameters for ASH1L and SETD2 activity, enabling screening for inhibitors that can be used to further investigate the roles of these two proteins in health and disease.  Both ASH1L and SETD2 are H3K36-specific methyltransferases but only SETD2 can trimethylate this mark.  The basic post-SET extension is crit. for ASH1L but not SETD2 activity.  We provide full kinetic characterization of ASH1L and SETD2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproBY7GkmOLrVg90H21EOLACvtfcHk0lih7C2cfLXEZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D&md5=64839475d5ee9cc2f20e6e71f8e8a6ea</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DKinetic%2520Characterization%2520of%2520Human%2520Histone%2520H3%2520Lysine%252036%2520Methyltransferases%252C%2520ASH1L%2520and%2520SETD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bbagen.2015.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustos, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Trimethylation of Histone H3 Lysine 36 by Human Methyltransferase PRDM9 Protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">12177</span><span class="NLM_x">–</span> <span class="NLM_lpage">12188</span><span class="refDoi"> DOI: 10.1074/jbc.M113.523183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M113.523183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=12177-12188&author=M.+S.+Eramauthor=S.+P.+Bustosauthor=E.+Lima-Fernandesauthor=A.+Siarheyevaauthor=G.+Senisterraauthor=T.+Hajianauthor=I.+Chauauthor=S.+Duanauthor=H.+Wuauthor=L.+Dombrovskiauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Trimethylation+of+Histone+H3+Lysine+36+by+Human+Methyltransferase+PRDM9+Protein&doi=10.1074%2Fjbc.M113.523183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523183%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DBustos%26aufirst%3DS.%2BP.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DTrimethylation%2520of%2520Histone%2520H3%2520Lysine%252036%2520by%2520Human%2520Methyltransferase%2520PRDM9%2520Protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D12177%26epage%3D12188%26doi%3D10.1074%2Fjbc.M113.523183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Rachel L. Palte, Sebastian E. Schneider, Michael D. Altman, Robert P. Hayes, Shuhei Kawamura, Brian M. Lacey, My Sam Mansueto, Michael Reutershan, Phieng Siliphaivanh, Christopher Sondey, Haiyan Xu, Zangwei Xu, Yingchun Ye, <span class="NLM_string-name hlFld-ContribAuthor">Michelle R. Machacek</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modulation of Protein Arginine Methyltransferase 5 (PRMT5). </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 1688-1693. <a href="https://doi.org/10.1021/acsmedchemlett.9b00525" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00525%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DAllosteric%252BModulation%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%26aulast%3DPalte%26aufirst%3DRachel%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D14112019%26date%3D30072020%26date%3D07082020%26volume%3D11%26issue%3D9%26spage%3D1688%26epage%3D1693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Fengling Li, Magdalena M. Szewczyk, Levon Halabelian, Kwang-su Park, Irene Chau, Aiping Dong, Hong Zeng, He Chen, Fanye Meng, Dalia Barsyte-Lovejoy, Cheryl H. Arrowsmith, Peter J. Brown, Jing Liu, Masoud Vedadi, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5477-5487. <a href="https://doi.org/10.1021/acs.jmedchem.0c00406" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B6%252B%252528PRMT6%252529%252BCovalent%252BInhibitor%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09032020%26date%3D14052020%26date%3D05052020%26volume%3D63%26issue%3D10%26spage%3D5477%26epage%3D5487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shaoyong Lu, Xinheng He, Duan Ni, <span class="NLM_string-name hlFld-ContribAuthor">Jian Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modulator Discovery: From Serendipity to Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6405-6421. <a href="https://doi.org/10.1021/acs.jmedchem.8b01749" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01749%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252BModulator%252BDiscovery%25253A%252BFrom%252BSerendipity%252Bto%252BStructure-Based%252BDesign%26aulast%3DLu%26aufirst%3DShaoyong%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D11112018%26date%3D07032019%26date%3D28022019%26volume%3D62%26issue%3D14%26spage%3D6405%26epage%3D6421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">A Pan-Inhibitor for Protein Arginine Methyltransferase Family Enzymes. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2021,</strong> <em>11 </em>
                                    (6)
                                     , 854. <a href="https://doi.org/10.3390/biom11060854" title="DOI URL">https://doi.org/10.3390/biom11060854</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom11060854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom11060854%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DA%252BPan-Inhibitor%252Bfor%252BProtein%252BArginine%252BMethyltransferase%252BFamily%252BEnzymes%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D6%26spage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Logan  Tillery</span>, <span class="hlFld-ContribAuthor ">Kayleigh  Barrett</span>, <span class="hlFld-ContribAuthor ">Jenna  Goldstein</span>, <span class="hlFld-ContribAuthor ">Jared W.  Lassner</span>, <span class="hlFld-ContribAuthor ">Bram  Osterhout</span>, <span class="hlFld-ContribAuthor ">Nathan L.  Tran</span>, <span class="hlFld-ContribAuthor ">Lily  Xu</span>, <span class="hlFld-ContribAuthor ">Ryan M.  Young</span>, <span class="hlFld-ContribAuthor ">Justin  Craig</span>, <span class="hlFld-ContribAuthor ">Ian  Chun</span>, <span class="hlFld-ContribAuthor ">David M.  Dranow</span>, <span class="hlFld-ContribAuthor ">Jan  Abendroth</span>, <span class="hlFld-ContribAuthor ">Silvia L.  Delker</span>, <span class="hlFld-ContribAuthor ">Douglas R.  Davies</span>, <span class="hlFld-ContribAuthor ">Stephen J.  Mayclin</span>, <span class="hlFld-ContribAuthor ">Brandy  Calhoun</span>, <span class="hlFld-ContribAuthor ">Madison J.  Bolejack</span>, <span class="hlFld-ContribAuthor ">Bart  Staker</span>, <span class="hlFld-ContribAuthor ">Sandhya  Subramanian</span>, <span class="hlFld-ContribAuthor ">Isabelle  Phan</span>, <span class="hlFld-ContribAuthor ">Donald D.  Lorimer</span>, <span class="hlFld-ContribAuthor ">Peter J.  Myler</span>, <span class="hlFld-ContribAuthor ">Thomas E.  Edwards</span>, <span class="hlFld-ContribAuthor ">Dennis E.  Kyle</span>, <span class="hlFld-ContribAuthor ">Christopher A.  Rice</span>, <span class="hlFld-ContribAuthor ">James C.  Morris</span>, <span class="hlFld-ContribAuthor ">James W.  Leahy</span>, <span class="hlFld-ContribAuthor ">Roman  Manetsch</span>, <span class="hlFld-ContribAuthor ">Lynn K.  Barrett</span>, <span class="hlFld-ContribAuthor ">Craig L.  Smith</span>, <span class="hlFld-ContribAuthor ">Wesley C.  Van Voorhis</span>, . </span><span class="cited-content_cbyCitation_article-title">Naegleria fowleri: Protein structures to facilitate drug discovery for the deadly, pathogenic free-living amoeba. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2021,</strong> <em>16 </em>
                                    (3)
                                     , e0241738. <a href="https://doi.org/10.1371/journal.pone.0241738" title="DOI URL">https://doi.org/10.1371/journal.pone.0241738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0241738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0241738%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DNaegleria%252Bfowleri%25253A%252BProtein%252Bstructures%252Bto%252Bfacilitate%252Bdrug%252Bdiscovery%252Bfor%252Bthe%252Bdeadly%25252C%252Bpathogenic%252Bfree-living%252Bamoeba%26aulast%3DTillery%26aufirst%3DLogan%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D3%26spage%3De0241738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Paul  Bryant</span>, <span class="hlFld-ContribAuthor ">John  Heiss</span>, <span class="hlFld-ContribAuthor ">Yeshavanth Kumar  Banasavadi-Siddegowda</span>. </span><span class="cited-content_cbyCitation_article-title">Arginine Methylation in Brain Tumors: Tumor Biology and Therapeutic Strategies. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2021,</strong> <em>10 </em>
                                    (1)
                                     , 124. <a href="https://doi.org/10.3390/cells10010124" title="DOI URL">https://doi.org/10.3390/cells10010124</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells10010124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells10010124%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DArginine%252BMethylation%252Bin%252BBrain%252BTumors%25253A%252BTumor%252BBiology%252Band%252BTherapeutic%252BStrategies%26aulast%3DBryant%26aufirst%3DJean-Paul%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D1%26spage%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Dongxing  Chen</span>, <span class="hlFld-ContribAuthor ">Youchao  Deng</span>, <span class="hlFld-ContribAuthor ">Guangping  Dong</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent and dual-selective bisubstrate inhibitor for protein arginine methyltransferase 4/5. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>57 </em><a href="https://doi.org/10.1016/j.apsb.2020.10.013" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.10.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.10.013%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252Band%252Bdual-selective%252Bbisubstrate%252Binhibitor%252Bfor%252Bprotein%252Barginine%252Bmethyltransferase%252B4%25252F5%26aulast%3DAl-Hamashi%26aufirst%3DAyad%2BA.%26date%3D2020%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Krystal  Diaz</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Non-Histone Arginine Methylation by Protein Arginine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Current Protein & Peptide Science</span><span> <strong>2020,</strong> <em>21 </em>
                                    (7)
                                     , 699-712. <a href="https://doi.org/10.2174/1389203721666200507091952" title="DOI URL">https://doi.org/10.2174/1389203721666200507091952</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1389203721666200507091952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1389203721666200507091952%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Protein%2520%2526%2520Peptide%2520Science%26atitle%3DNon-Histone%252BArginine%252BMethylation%252Bby%252BProtein%252BArginine%252BMethyltransferases%26aulast%3DAl-Hamashi%26aufirst%3DAyad%2BA.%26date%3D2020%26volume%3D21%26issue%3D7%26spage%3D699%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renato  Ferreira de Freitas</span>, <span class="hlFld-ContribAuthor ">Danton  Ivanochko</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>. </span><span class="cited-content_cbyCitation_article-title">Methyltransferase Inhibitors: Competing with, or Exploiting the Bound Cofactor. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (24)
                                     , 4492. <a href="https://doi.org/10.3390/molecules24244492" title="DOI URL">https://doi.org/10.3390/molecules24244492</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24244492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24244492%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMethyltransferase%252BInhibitors%25253A%252BCompeting%252Bwith%25252C%252Bor%252BExploiting%252Bthe%252BBound%252BCofactor%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DRenato%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D24%26spage%3D4492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Audrey  Vincent</span>, <span class="hlFld-ContribAuthor ">Aïcha  Ouelkdite-Oumouchal</span>, <span class="hlFld-ContribAuthor ">Mouloud  Souidi</span>, <span class="hlFld-ContribAuthor ">Julie  Leclerc</span>, <span class="hlFld-ContribAuthor ">Bernadette  Neve</span>, <span class="hlFld-ContribAuthor ">Isabelle Van  Seuningen</span>. </span><span class="cited-content_cbyCitation_article-title">Colon cancer stemness as a reversible epigenetic state: Implications for anticancer therapies. </span><span class="cited-content_cbyCitation_journal-name">World Journal of Stem Cells</span><span> <strong>2019,</strong> <em>11 </em>
                                    (11)
                                     , 920-936. <a href="https://doi.org/10.4252/wjsc.v11.i11.920" title="DOI URL">https://doi.org/10.4252/wjsc.v11.i11.920</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4252/wjsc.v11.i11.920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4252%2Fwjsc.v11.i11.920%26sid%3Dliteratum%253Aachs%26jtitle%3DWorld%2520Journal%2520of%2520Stem%2520Cells%26atitle%3DColon%252Bcancer%252Bstemness%252Bas%252Ba%252Breversible%252Bepigenetic%252Bstate%25253A%252BImplications%252Bfor%252Banticancer%252Btherapies%26aulast%3DVincent%26aufirst%3DAudrey%26date%3D2019%26volume%3D11%26issue%3D11%26spage%3D920%26epage%3D936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sunil Kumar  Tewary</span>, <span class="hlFld-ContribAuthor ">Y. George  Zheng</span>, <span class="hlFld-ContribAuthor ">Meng-Chiao  Ho</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methyltransferases: insights into the enzyme structure and mechanism at the atomic level. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2917-2932. <a href="https://doi.org/10.1007/s00018-019-03145-x" title="DOI URL">https://doi.org/10.1007/s00018-019-03145-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03145-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03145-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DProtein%252Barginine%252Bmethyltransferases%25253A%252Binsights%252Binto%252Bthe%252Benzyme%252Bstructure%252Band%252Bmechanism%252Bat%252Bthe%252Batomic%252Blevel%26aulast%3DTewary%26aufirst%3DSunil%2BKumar%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2917%26epage%3D2932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Dilworth</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein methylation: from chemical tools to precision medicines. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2967-2985. <a href="https://doi.org/10.1007/s00018-019-03147-9" title="DOI URL">https://doi.org/10.1007/s00018-019-03147-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03147-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03147-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DTargeting%252Bprotein%252Bmethylation%25253A%252Bfrom%252Bchemical%252Btools%252Bto%252Bprecision%252Bmedicines%26aulast%3DDilworth%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2967%26epage%3D2985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luciano  Pirola</span>, <span class="hlFld-ContribAuthor ">Oskar  Ciesielski</span>, <span class="hlFld-ContribAuthor ">Aneta  Balcerczyk</span>. </span><span class="cited-content_cbyCitation_article-title">The Methylation Status of the Epigenome: Its Emerging Role in the Regulation of Tumor Angiogenesis and Tumor Growth, and Potential for Drug Targeting. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2018,</strong> <em>10 </em>
                                    (8)
                                     , 268. <a href="https://doi.org/10.3390/cancers10080268" title="DOI URL">https://doi.org/10.3390/cancers10080268</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers10080268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers10080268%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DThe%252BMethylation%252BStatus%252Bof%252Bthe%252BEpigenome%25253A%252BIts%252BEmerging%252BRole%252Bin%252Bthe%252BRegulation%252Bof%252BTumor%252BAngiogenesis%252Band%252BTumor%252BGrowth%25252C%252Band%252BPotential%252Bfor%252BDrug%252BTargeting%26aulast%3DPirola%26aufirst%3DLuciano%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D8%26spage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design and synthesis of highly potent inhibitors of PRMT3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Reported cocrystal structure of compound <b>1</b> in complex with PRMT3 indicating key hydrogen bonding interactions (magenta dotted lines) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route for the Preparation of Compounds Listed in <a class="ref internalNav" href="#tbl3" aria-label="Tables 3">Tables 3</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes for Preparing Intermediates <b>44</b>–<b>48</b> for Synthesis of Compounds <b>36</b>–<b>40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibitor <b>36</b> is highly selective for PRMT3 over 31 other methyltransferases. The selectivity data for compounds <b>29</b> and <b>30</b> are shown in the <a class="ref internalNav" href="#notes-5" aria-label="Supporting Information">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. InCELL Hunter Assay results of compounds <b>4</b>, <b>29</b>, <b>36</b>, <b>37</b>, and <b>51</b> in A549 and HEK cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/medium/jm-2017-01674p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular inhibitory activity of compounds <b>29</b>, <b>30</b>, and <b>36</b>. HEK293 cells were cotransfected with FLAG tagged PRMT3 (wt) or its catalytic mutant (mt) and GFP-tagged histone H4 and treated with different concentrations of compounds, as indicated. Total cell lysates were collected 24 h post inhibitor treatment and analyzed for H4R3me2a, GFP, and FLAG levels by Western blotting. The graphs represent nonlinear fits of H4R3me2a fluorescence intensities normalized to GFP. The results are the averages of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01674/20180220/images/large/jm-2017-01674p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01674&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein Arginine Methylation in Mammals: Who, What, and Why</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+Arginine+Methylation+in+Mammals%3A+Who%2C+What%2C+and+Why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0liRq4J2xtPhCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520Arginine%2520Methylation%2520in%2520Mammals%253A%2520Who%252C%2520What%252C%2520and%2520Why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Person, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal Protein S2 is A Substrate for Mammalian PRMT3 (Protein Arginine Methyltransferase 3)</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">386</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1042/BJ20041466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1042%2FBJ20041466" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2005&pages=85-91&author=R.+Swierczauthor=M.+D.+Personauthor=M.+T.+Bedford&title=Ribosomal+Protein+S2+is+A+Substrate+for+Mammalian+PRMT3+%28Protein+Arginine+Methyltransferase+3%29&doi=10.1042%2FBJ20041466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1042%2FBJ20041466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20041466%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DPerson%26aufirst%3DM.%2BD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520Protein%2520S2%2520is%2520A%2520Substrate%2520for%2520Mammalian%2520PRMT3%2520%2528Protein%2520Arginine%2520Methyltransferase%25203%2529%26jtitle%3DBiochem.%2520J.%26date%3D2005%26volume%3D386%26spage%3D85%26epage%3D91%26doi%3D10.1042%2FBJ20041466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal Protein rpS2 is Hypomethylated in PRMT3-Deficient Mice</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">16917</span><span class="NLM_x">–</span> <span class="NLM_lpage">16923</span><span class="refDoi"> DOI: 10.1074/jbc.M609778200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M609778200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=17439947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=16917-16923&author=R.+Swierczauthor=D.+Chengauthor=D.+Kimauthor=M.+T.+Bedford&title=Ribosomal+Protein+rpS2+is+Hypomethylated+in+PRMT3-Deficient+Mice&doi=10.1074%2Fjbc.M609778200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ribosomal Protein rpS2 Is Hypomethylated in PRMT3-deficient Mice</span></div><div class="casAuthors">Swiercz, Rafal; Cheng, Donghang; Kim, Daehoon; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16917-16923</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PRMT3 is a type I arginine methyltransferase that resides in the cytoplasm.  A large proportion of this cytosolic PRMT3 is found assocd. with ribosomes.  It is tethered to the ribosomes through its interaction with rpS2, which is also its substrate.  Here we show that mouse embryos with a targeted disruption of PRMT3 are small in size but survive after birth and attain a normal size in adulthood, thus displaying Minute-like characteristics.  The ribosome protein rpS2 is hypomethylated in the absence of PRMT3, demonstrating that it is a bona fide, in vivo PRMT3 substrate that cannot be modified by other PRMTs.  Finally, the levels 40 S, 60 S, and 80 S monosomes and polyribosomes are unaffected by the loss of PRMT3, but there are addnl. as yet unidentified proteins that co-fractionate with ribosomes that are also dedicated PRMT3 substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-lEvztlUF4rVg90H21EOLACvtfcHk0liRq4J2xtPhCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D&md5=4d97b44b6706559086aa2554df786b31</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M609778200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M609778200%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520Protein%2520rpS2%2520is%2520Hypomethylated%2520in%2520PRMT3-Deficient%2520Mice%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D16917%26epage%3D16923%26doi%3D10.1074%2Fjbc.M609778200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Bachand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silver, P. A.</span><span> </span><span class="NLM_article-title">PRMT3 is A Ribosomal Protein Methyltransferase that Affects the Cellular Levels of Ribosomal Subunits</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">2641</span><span class="NLM_x">–</span> <span class="NLM_lpage">2650</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7600265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1038%2Fsj.emboj.7600265" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2641-2650&author=F.+Bachandauthor=P.+A.+Silver&title=PRMT3+is+A+Ribosomal+Protein+Methyltransferase+that+Affects+the+Cellular+Levels+of+Ribosomal+Subunits&doi=10.1038%2Fsj.emboj.7600265"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600265%26sid%3Dliteratum%253Aachs%26aulast%3DBachand%26aufirst%3DF.%26aulast%3DSilver%26aufirst%3DP.%2BA.%26atitle%3DPRMT3%2520is%2520A%2520Ribosomal%2520Protein%2520Methyltransferase%2520that%2520Affects%2520the%2520Cellular%2520Levels%2520of%2520Ribosomal%2520Subunits%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2641%26epage%3D2650%26doi%3D10.1038%2Fsj.emboj.7600265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Landry-Voyer, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilodeau, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dionne, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Port, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouleau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kehlenbach, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachand, F.</span><span> </span><span class="NLM_article-title">Human PDCD2L Is an Export Substrate of CRM1 That Associates with 40S Ribosomal Subunit Precursors</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3019</span><span class="NLM_x">–</span> <span class="NLM_lpage">3032</span><span class="refDoi"> DOI: 10.1128/MCB.00303-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1128%2FMCB.00303-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=27697862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitl2mtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=3019-3032&author=A.+M.+Landry-Voyerauthor=S.+Bilodeauauthor=D.+Bergeronauthor=K.+L.+Dionneauthor=S.+A.+Portauthor=C.+Rouleauauthor=F.+M.+Boisvertauthor=R.+H.+Kehlenbachauthor=F.+Bachand&title=Human+PDCD2L+Is+an+Export+Substrate+of+CRM1+That+Associates+with+40S+Ribosomal+Subunit+Precursors&doi=10.1128%2FMCB.00303-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Human PDCD2L is an export substrate of CRM1 that associates with 40S ribosomal subunit precursors</span></div><div class="casAuthors">Landry-Voyer, Anne-Marie; Bilodeau, Sarah; Bergeron, Danny; Dionne, Kiersten L.; Port, Sarah A.; Rouleau, Caroline; Boisvert, Francois-Michel; Kehlenbach, Ralph H.; Bachand, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3019-3032</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 3 (PRMT3) forms a stable complex with 40S ribosomal protein S2 (RPS2) and contributes to ribosome biogenesis.  However, the mol. mechanism by which PRMT3 influences ribosome biogenesis and/or function still remains unclear.  Using quant. proteomics, we identified human programmed cell death 2-like (PDCD2L) as a novel PRMT3-assocd. protein.  Our data suggest that RPS2 promotes the formation of a conserved extraribosomal complex with PRMT3 and PDCD2L.  We also show that PDCD2L assocs. with 40S subunit precursors that contain a 3'-extended form of the 18S rRNA (18S-E pre-rRNA) and several pre-40S maturation factors.  PDCD2L shuttles between the nucleus and the cytoplasm in a CRM1-dependent manner using a leucine-rich nuclear export signal that is sufficient to direct the export of a reporter protein.  Although PDCD2L is not required for the biogenesis and export of 40S ribosomal subunits, we found that PDCD2L-null cells accumulate free 60S ribosomal subunits, which is indicative of a deficiency in 40S subunit availability.  Our data also indicate that PDCD2L and its paralog, PDCD2, function redundantly in 40S ribosomal subunit prodn.  Our findings uncover the existence of an extraribosomal complex consisting of PDCD2L, RPS2, and PRMT3 and support a role for PDCD2L in the late maturation of 40S ribosomal subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbG98djoI-UrVg90H21EOLACvtfcHk0lieYKClCjq1Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitl2mtbs%253D&md5=e5b0325e82737f37a4c8a4bcbeabb158</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.00303-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00303-16%26sid%3Dliteratum%253Aachs%26aulast%3DLandry-Voyer%26aufirst%3DA.%2BM.%26aulast%3DBilodeau%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DD.%26aulast%3DDionne%26aufirst%3DK.%2BL.%26aulast%3DPort%26aufirst%3DS.%2BA.%26aulast%3DRouleau%26aufirst%3DC.%26aulast%3DBoisvert%26aufirst%3DF.%2BM.%26aulast%3DKehlenbach%26aufirst%3DR.%2BH.%26aulast%3DBachand%26aufirst%3DF.%26atitle%3DHuman%2520PDCD2L%2520Is%2520an%2520Export%2520Substrate%2520of%2520CRM1%2520That%2520Associates%2520with%252040S%2520Ribosomal%2520Subunit%2520Precursors%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2016%26volume%3D36%26spage%3D3019%26epage%3D3032%26doi%3D10.1128%2FMCB.00303-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gary, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span> </span><span class="NLM_article-title">Prmt 3, A Type I Protein Arginine N-Methyltransferase that Differs From PRMT1 in its Oligomerization, Subcellular Localization, Substrate Specificity, and Regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">16935</span><span class="NLM_x">–</span> <span class="NLM_lpage">16945</span><span class="refDoi"> DOI: 10.1074/jbc.273.27.16935</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.273.27.16935" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=16935-16945&author=J.+Tangauthor=J.+D.+Garyauthor=S.+Clarkeauthor=H.+R.+Herschman&title=Prmt+3%2C+A+Type+I+Protein+Arginine+N-Methyltransferase+that+Differs+From+PRMT1+in+its+Oligomerization%2C+Subcellular+Localization%2C+Substrate+Specificity%2C+and+Regulation&doi=10.1074%2Fjbc.273.27.16935"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.27.16935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.27.16935%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DGary%26aufirst%3DJ.%2BD.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26atitle%3DPrmt%25203%252C%2520A%2520Type%2520I%2520Protein%2520Arginine%2520N-Methyltransferase%2520that%2520Differs%2520From%2520PRMT1%2520in%2520its%2520Oligomerization%252C%2520Subcellular%2520Localization%252C%2520Substrate%2520Specificity%252C%2520and%2520Regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D16935%26epage%3D16945%26doi%3D10.1074%2Fjbc.273.27.16935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Kim, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafsson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. H.</span><span> </span><span class="NLM_article-title">PRMT3 Regulates Hepatic Lipogenesis Through Direct Interaction with LXRalpha</span> <span class="citation_source-journal">Diabetes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.2337/db13-1394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.2337%2Fdb13-1394" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2015&pages=60-71&author=D.+I.+Kimauthor=M.+J.+Parkauthor=S.+K.+Limauthor=J.+I.+Parkauthor=K.+C.+Yoonauthor=H.+J.+Hanauthor=J.+A.+Gustafssonauthor=J.+H.+Limauthor=S.+H.+Park&title=PRMT3+Regulates+Hepatic+Lipogenesis+Through+Direct+Interaction+with+LXRalpha&doi=10.2337%2Fdb13-1394"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2337%2Fdb13-1394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb13-1394%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%2BI.%26aulast%3DPark%26aufirst%3DM.%2BJ.%26aulast%3DLim%26aufirst%3DS.%2BK.%26aulast%3DPark%26aufirst%3DJ.%2BI.%26aulast%3DYoon%26aufirst%3DK.%2BC.%26aulast%3DHan%26aufirst%3DH.%2BJ.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26aulast%3DLim%26aufirst%3DJ.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BH.%26atitle%3DPRMT3%2520Regulates%2520Hepatic%2520Lipogenesis%2520Through%2520Direct%2520Interaction%2520with%2520LXRalpha%26jtitle%3DDiabetes%26date%3D2015%26volume%3D64%26spage%3D60%26epage%3D71%26doi%3D10.2337%2Fdb13-1394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Fronz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolbel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierhorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck-Sickinger, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostareck-Lederer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahle, E.</span><span> </span><span class="NLM_article-title">Promiscuous Modification of the Nuclear Poly(A)-Binding Protein by Multiple Protein-Arginine Methyltransferases Does Not Affect The Aggregation Behavior</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">20408</span><span class="NLM_x">–</span> <span class="NLM_lpage">20420</span><span class="refDoi"> DOI: 10.1074/jbc.M802329200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M802329200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=20408-20420&author=K.+Fronzauthor=S.+Ottoauthor=K.+Kolbelauthor=U.+Kuhnauthor=H.+Friedrichauthor=A.+Schierhornauthor=A.+G.+Beck-Sickingerauthor=A.+Ostareck-Ledererauthor=E.+Wahle&title=Promiscuous+Modification+of+the+Nuclear+Poly%28A%29-Binding+Protein+by+Multiple+Protein-Arginine+Methyltransferases+Does+Not+Affect+The+Aggregation+Behavior&doi=10.1074%2Fjbc.M802329200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802329200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802329200%26sid%3Dliteratum%253Aachs%26aulast%3DFronz%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DS.%26aulast%3DKolbel%26aufirst%3DK.%26aulast%3DKuhn%26aufirst%3DU.%26aulast%3DFriedrich%26aufirst%3DH.%26aulast%3DSchierhorn%26aufirst%3DA.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DOstareck-Lederer%26aufirst%3DA.%26aulast%3DWahle%26aufirst%3DE.%26atitle%3DPromiscuous%2520Modification%2520of%2520the%2520Nuclear%2520Poly%2528A%2529-Binding%2520Protein%2520by%2520Multiple%2520Protein-Arginine%2520Methyltransferases%2520Does%2520Not%2520Affect%2520The%2520Aggregation%2520Behavior%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D20408%26epage%3D20420%26doi%3D10.1074%2Fjbc.M802329200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Tavanez, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bengoechea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berciano, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafarga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmo-Fonseca, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enguita, F. J.</span><span> </span><span class="NLM_article-title">Hsp70 Chaperones and Type I PRMTs are Sequestered at Intranuclear Inclusions Caused by Polyalanine Expansions in PABPN1</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e6418</span><span class="refDoi"> DOI: 10.1371/journal.pone.0006418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1371%2Fjournal.pone.0006418" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e6418&author=J.+P.+Tavanezauthor=R.+Bengoecheaauthor=M.+T.+Bercianoauthor=M.+Lafargaauthor=M.+Carmo-Fonsecaauthor=F.+J.+Enguita&title=Hsp70+Chaperones+and+Type+I+PRMTs+are+Sequestered+at+Intranuclear+Inclusions+Caused+by+Polyalanine+Expansions+in+PABPN1&doi=10.1371%2Fjournal.pone.0006418"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0006418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0006418%26sid%3Dliteratum%253Aachs%26aulast%3DTavanez%26aufirst%3DJ.%2BP.%26aulast%3DBengoechea%26aufirst%3DR.%26aulast%3DBerciano%26aufirst%3DM.%2BT.%26aulast%3DLafarga%26aufirst%3DM.%26aulast%3DCarmo-Fonseca%26aufirst%3DM.%26aulast%3DEnguita%26aufirst%3DF.%2BJ.%26atitle%3DHsp70%2520Chaperones%2520and%2520Type%2520I%2520PRMTs%2520are%2520Sequestered%2520at%2520Intranuclear%2520Inclusions%2520Caused%2520by%2520Polyalanine%2520Expansions%2520in%2520PABPN1%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De6418%26doi%3D10.1371%2Fjournal.pone.0006418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Lai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hakala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weintraub, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiio, Y.</span><span> </span><span class="NLM_article-title">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">5175</span><span class="NLM_x">–</span> <span class="NLM_lpage">5182</span><span class="refDoi"> DOI: 10.1021/pr200642c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr200642c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5175-5182&author=Y.+Laiauthor=M.+Songauthor=K.+Hakalaauthor=S.+T.+Weintraubauthor=Y.+Shiio&title=Proteomic+Dissection+of+the+von+Hippel-Lindau+%28VHL%29+Interactome&doi=10.1021%2Fpr200642c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span></div><div class="casAuthors">Lai, Yanlai; Song, Meihua; Hakala, Kevin; Weintraub, Susan T.; Shiio, Yuzuru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5175-5182</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) tumor suppressor gene encodes a component of a ubiquitin ligase complex contg. elongin B, elongin C, cullin 2, and Rbx1, which acts as a neg. regulator of hypoxia inducible factor (HIF).  VHL ubiquitinates and degrades the alpha subunits of HIF, and this is proposed to suppress tumorigenesis and tumor angiogenesis.  Several lines of evidence also suggest important roles for HIF-independent VHL functions in the maintenance of primary cilium, extracellular matrix formation, and tumor suppression.  We undertook a series of proteomic analyses to gain a comprehensive picture of the VHL-interacting proteins.  We found that the ARF tumor suppressor interacts with VHL30, a longer VHL isoform, but not with VHL19, a shorter VHL isoform.  ARF was found to release VHL30 from the E3 ligase complex, promoting the binding of VHL30 to a protein arginine methyltransferase, PRMT3.  Our anal. of the VHL19 interactome also uncovered that VHL19 displays an affinity to collagens and their biosynthesis enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqTYZ6RvZ7erVg90H21EOLACvtfcHk0lieYKClCjq1Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL&md5=51500af3ec4256311dbd835c47a90200</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fpr200642c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200642c%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHakala%26aufirst%3DK.%26aulast%3DWeintraub%26aufirst%3DS.%2BT.%26aulast%3DShiio%26aufirst%3DY.%26atitle%3DProteomic%2520Dissection%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520Interactome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5175%26epage%3D5182%26doi%3D10.1021%2Fpr200642c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Singh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranda, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roemer, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutmann, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newsham, I. F.</span><span> </span><span class="NLM_article-title">DAL-1/4.1B Tumor Suppressor Interacts with Protein Arginine N-Methyltransferase 3 (PRMT3) and Inhibits its Ability to Methylate Substrates In Vitro and In Vivo</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">7761</span><span class="NLM_x">–</span> <span class="NLM_lpage">7771</span><span class="refDoi"> DOI: 10.1038/sj.onc.1208057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1038%2Fsj.onc.1208057" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=7761-7771&author=V.+Singhauthor=T.+B.+Mirandaauthor=W.+Jiangauthor=A.+Frankelauthor=M.+E.+Roemerauthor=V.+A.+Robbauthor=D.+H.+Gutmannauthor=H.+R.+Herschmanauthor=S.+Clarkeauthor=I.+F.+Newsham&title=DAL-1%2F4.1B+Tumor+Suppressor+Interacts+with+Protein+Arginine+N-Methyltransferase+3+%28PRMT3%29+and+Inhibits+its+Ability+to+Methylate+Substrates+In+Vitro+and+In+Vivo&doi=10.1038%2Fsj.onc.1208057"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1208057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1208057%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DMiranda%26aufirst%3DT.%2BB.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DRoemer%26aufirst%3DM.%2BE.%26aulast%3DRobb%26aufirst%3DV.%2BA.%26aulast%3DGutmann%26aufirst%3DD.%2BH.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DNewsham%26aufirst%3DI.%2BF.%26atitle%3DDAL-1%252F4.1B%2520Tumor%2520Suppressor%2520Interacts%2520with%2520Protein%2520Arginine%2520N-Methyltransferase%25203%2520%2528PRMT3%2529%2520and%2520Inhibits%2520its%2520Ability%2520to%2520Methylate%2520Substrates%2520In%2520Vitro%2520and%2520In%2520Vivo%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D7761%26epage%3D7771%26doi%3D10.1038%2Fsj.onc.1208057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newsham, I. F.</span><span> </span><span class="NLM_article-title">The Tumor Suppressor DAL-1/4.1B and Protein Methylation Cooperate in Inducing Apoptosis in MCF-7 Breast Cancer Cells</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="refDoi"> DOI: 10.1186/1476-4598-5-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1186%2F1476-4598-5-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=16420693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A280%3ADC%252BD1c7ntFSltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=4&author=W.+Jiangauthor=I.+F.+Newsham&title=The+Tumor+Suppressor+DAL-1%2F4.1B+and+Protein+Methylation+Cooperate+in+Inducing+Apoptosis+in+MCF-7+Breast+Cancer+Cells&doi=10.1186%2F1476-4598-5-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The tumor suppressor DAL-1/4.1B and protein methylation cooperate in inducing apoptosis in MCF-7 breast cancer cells</span></div><div class="casAuthors">Jiang Wei; Newsham Irene F</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  DAL-1 (Differentially Expressed in Adenocarcinoma of the Lung)/4.1B is a member of the protein 4.1 superfamily that has been shown to suppress growth in lung, breast and brain tumor cells.  In the case of the caspase-3 deficient MCF-7 breast cancer cells, this growth suppression has been shown to be partially mediated by the induction of apoptosis.  However the exact mechanism of action of DAL-1/4.1B is unknown.  Recently, protein arginine N-methyltransferase 3 (PRMT3) was identified as a DAL-1/4.1B interacting protein.  Protein arginine methyltransferases (PRMTs) posttranslationally methylate the arginine residues of proteins, a modification which has been implicated in the regulation of multiple cellular processes including nuclear-cytoplasmic transport, signal transduction, and transcription.  RESULTS:  To investigate the role of protein methylation in cell death induced by DAL-1/4.1B, DAL-1/4.1B-inducible MCF-7 cells were examined for apoptosis and caspase activation in the absence and presence of the protein methylation inhibitor adenosine dialdehyde (AdOX).  Flow cytometry analysis revealed that apoptosis was primarily associated with the activation of caspase 8, and inhibition of this activation blocked the ability of DAL-1/4.1B to induce cell death.  CONCLUSION:  These results suggest that protein methylation cooperates with DAL-1/4.1B-associated caspase 8-specific activation to induce apoptosis in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZnGCV1W8gGgpJitSxOdIlfW6udTcc2eZs78vCzA-2sbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7ntFSltw%253D%253D&md5=d2cf59aba875c4f913920cbb6e3cc4f2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-4%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DNewsham%26aufirst%3DI.%2BF.%26atitle%3DThe%2520Tumor%2520Suppressor%2520DAL-1%252F4.1B%2520and%2520Protein%2520Methylation%2520Cooperate%2520in%2520Inducing%2520Apoptosis%2520in%2520MCF-7%2520Breast%2520Cancer%2520Cells%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D4%26doi%3D10.1186%2F1476-4598-5-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Fluorescence-Based Methods For Screening Writers And Readers Of Histone Methyl Marks</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1177/1087057111422256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Fluorescence-Based+Methods+For+Screening+Writers+And+Readers+Of+Histone+Methyl+Marks&doi=10.1177%2F1087057111422256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0ljXgPuaIdpPTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DFluorescence-Based%2520Methods%2520For%2520Screening%2520Writers%2520And%2520Readers%2520Of%2520Histone%2520Methyl%2520Marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84%26doi%3D10.1177%2F1087057111422256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Herrmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pably, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckerich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fackelmayer, F. O.</span><span> </span><span class="NLM_article-title">Human Protein Arginine Methyltransferases in Vivo-Distinct Properties of Eight Canonical Members of the PRMT Family</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">677</span><span class="refDoi"> DOI: 10.1242/jcs.039933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1242%2Fjcs.039933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=19208762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=667-677&author=F.+Herrmannauthor=P.+Pablyauthor=C.+Eckerichauthor=M.+T.+Bedfordauthor=F.+O.+Fackelmayer&title=Human+Protein+Arginine+Methyltransferases+in+Vivo-Distinct+Properties+of+Eight+Canonical+Members+of+the+PRMT+Family&doi=10.1242%2Fjcs.039933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Human protein arginine methyltransferases in vivo - distinct properties of eight canonical members of the PRMT family</span></div><div class="casAuthors">Herrmann, Frank; Pably, Peter; Eckerich, Carmen; Bedford, Mark T.; Fackelmayer, Frank O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-677</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Methylation of arginine residues is a widespread post-translational modification of proteins catalyzed by a small family of protein arginine methyltransferases (PRMTs).  Functionally, the modification appears to regulate protein functions and interactions that affect gene regulation, signalling and subcellular localization of proteins and nucleic acids.  All members have been, to different degrees, characterized individually and their implication in cellular processes has been inferred from characterizing substrates and interactions.  Here, we report the first comprehensive comparison of all eight canonical members of the human PRMT family with respect to subcellular localization and dynamics in living cells.  We show that the individual family members differ significantly in their properties, as well as in their substrate specificities, suggesting that they fulfil distinctive, non-redundant functions in vivo.  In addn., certain PRMTs display different subcellular localization in different cell types, implicating cell- and tissue-specific mechanisms for regulating PRMT functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4G4_wbI1I-rVg90H21EOLACvtfcHk0ljXgPuaIdpPTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D&md5=939c3e7c7c0030afbe7e4c5eaa1852ed</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fjcs.039933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.039933%26sid%3Dliteratum%253Aachs%26aulast%3DHerrmann%26aufirst%3DF.%26aulast%3DPably%26aufirst%3DP.%26aulast%3DEckerich%26aufirst%3DC.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DFackelmayer%26aufirst%3DF.%2BO.%26atitle%3DHuman%2520Protein%2520Arginine%2520Methyltransferases%2520in%2520Vivo-Distinct%2520Properties%2520of%2520Eight%2520Canonical%2520Members%2520of%2520the%2520PRMT%2520Family%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2009%26volume%3D122%26spage%3D667%26epage%3D677%26doi%3D10.1242%2Fjcs.039933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Wagner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinschmidt, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, U. M.</span><span> </span><span class="NLM_article-title">SET-Mediated Promoter Hypoacetylation is A Prerequisite for Coactivation of the Estrogen-Responsive pS2 Gene by PRMT1</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">27242</span><span class="NLM_x">–</span> <span class="NLM_lpage">27250</span><span class="refDoi"> DOI: 10.1074/jbc.M605172200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M605172200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=27242-27250&author=S.+Wagnerauthor=S.+Weberauthor=M.+A.+Kleinschmidtauthor=K.+Nagataauthor=U.+M.+Bauer&title=SET-Mediated+Promoter+Hypoacetylation+is+A+Prerequisite+for+Coactivation+of+the+Estrogen-Responsive+pS2+Gene+by+PRMT1&doi=10.1074%2Fjbc.M605172200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605172200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605172200%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DS.%26aulast%3DWeber%26aufirst%3DS.%26aulast%3DKleinschmidt%26aufirst%3DM.%2BA.%26aulast%3DNagata%26aufirst%3DK.%26aulast%3DBauer%26aufirst%3DU.%2BM.%26atitle%3DSET-Mediated%2520Promoter%2520Hypoacetylation%2520is%2520A%2520Prerequisite%2520for%2520Coactivation%2520of%2520the%2520Estrogen-Responsive%2520pS2%2520Gene%2520by%2520PRMT1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D27242%26epage%3D27250%26doi%3D10.1074%2Fjbc.M605172200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Obianyo, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Causey, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, P. R.</span><span> </span><span class="NLM_article-title">Activity-Based Protein Profiling of Protein Arginine Methyltransferase 1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1127</span><span class="NLM_x">–</span> <span class="NLM_lpage">1135</span><span class="refDoi"> DOI: 10.1021/cb2001473</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb2001473" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1127-1135&author=O.+Obianyoauthor=C.+P.+Causeyauthor=J.+E.+Jonesauthor=P.+R.+Thompson&title=Activity-Based+Protein+Profiling+of+Protein+Arginine+Methyltransferase+1&doi=10.1021%2Fcb2001473"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fcb2001473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb2001473%26sid%3Dliteratum%253Aachs%26aulast%3DObianyo%26aufirst%3DO.%26aulast%3DCausey%26aufirst%3DC.%2BP.%26aulast%3DJones%26aufirst%3DJ.%2BE.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DActivity-Based%2520Protein%2520Profiling%2520of%2520Protein%2520Arginine%2520Methyltransferase%25201%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D1127%26epage%3D1135%26doi%3D10.1021%2Fcb2001473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niroomand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zankl, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katus, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiefenbacher, C. P.</span><span> </span><span class="NLM_article-title">Expression of Nitric Oxide Related Enzymes in Coronary Heart Disease</span> <span class="citation_source-journal">Basic Res. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">346</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1007/s00395-006-0592-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1007%2Fs00395-006-0592-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=16705470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2006&pages=346-353&author=X.+Chenauthor=F.+Niroomandauthor=Z.+Liuauthor=A.+Zanklauthor=H.+A.+Katusauthor=L.+Jahnauthor=C.+P.+Tiefenbacher&title=Expression+of+Nitric+Oxide+Related+Enzymes+in+Coronary+Heart+Disease&doi=10.1007%2Fs00395-006-0592-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nitric oxide related enzymes in coronary heart disease</span></div><div class="casAuthors">Chen, X.; Niroomand, F.; Liu, Z.; Zankl, A.; Katus, H. A.; Jahn, L.; Tiefenbacher, C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Basic Research in Cardiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">346-353</span>CODEN:
                <span class="NLM_cas:coden">BRCAB7</span>;
        ISSN:<span class="NLM_cas:issn">0300-8428</span>.
    
            (<span class="NLM_cas:orgname">Steinkopff Verlag</span>)
        </div><div class="casAbstract">Enzymes involved in the metab. nitric oxide (NO) and reactive oxygen species (ROS) may play a role for the decreased availability of NO in atherosclerosis.  We, therefore, hypothesized that the pattern of gene expression of these enzymes is altered in atherosclerosis.  Myocardial tissue from patients with coronary heart disease (CHD) or without CHD (control group) was investigated.  The level of enzymes related to NO/ROS metab. was detd. both at mRNA level and protein level by rt-PCR, real-time PCR, and western blot.  The expression of NOS1-3 (synthesis of NO), arginasel (redn. of L-arginine), p22phox (active subunit of NADPH oxidase), GTPCH (rate limiting enzyme for tetrahydrobiopterin), SOD1-3 (scavengers of superoxide anions), PRTMT1-3, and DDAH2 (involved in the metab. of ADMA) was detd.  All enzymes were found to be expressed in human myocardium.  NOS isoforms were decreased in CHD in protein level, but only the down regulation of NOS3 expression reached statistical significance.  The expression of PRMT1 and PRMT3 was increased.  In addn., the expression of DDAH2 was reduced, both theor. leading to an increase of ADMA concn.  SOD3 was down regulated in tissue from patients with CHD.  Taken together, in myocardial tissue from patients with atherosclerosis, the expression of genes increasing ADMA levels is enhanced in contrast to a reduced expression of genes promoting NO synthesis.  These results may contribute to the explanation of increased oxidative stress in atherosclerosis on the level of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZhed3XxK3bVg90H21EOLACvtfcHk0lh9M8KU6FQirw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D&md5=e40c38e03c6dc050a5a894bd0be20155</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs00395-006-0592-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00395-006-0592-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DNiroomand%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZankl%26aufirst%3DA.%26aulast%3DKatus%26aufirst%3DH.%2BA.%26aulast%3DJahn%26aufirst%3DL.%26aulast%3DTiefenbacher%26aufirst%3DC.%2BP.%26atitle%3DExpression%2520of%2520Nitric%2520Oxide%2520Related%2520Enzymes%2520in%2520Coronary%2520Heart%2520Disease%26jtitle%3DBasic%2520Res.%2520Cardiol.%26date%3D2006%26volume%3D101%26spage%3D346%26epage%3D353%26doi%3D10.1007%2Fs00395-006-0592-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Miyata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohyama, M.</span><span> </span><span class="NLM_article-title">PRMT3 is Essential for Dendritic Spine Maturation in Rat Hippocampal Neurons</span> <span class="citation_source-journal">Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1352</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/j.brainres.2010.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.brainres.2010.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=20647003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1352&publication_year=2010&pages=11-20&author=S.+Miyataauthor=Y.+Moriauthor=M.+Tohyama&title=PRMT3+is+Essential+for+Dendritic+Spine+Maturation+in+Rat+Hippocampal+Neurons&doi=10.1016%2Fj.brainres.2010.07.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT3 is essential for dendritic spine maturation in rat hippocampal neurons</span></div><div class="casAuthors">Miyata, Shingo; Mori, Yasutake; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1352</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-20</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine N-methyltransferase 3 (PRMT3) is a cytoplasmic enzyme that utilizes S-adenosyl-L-methionine (AdoMet) to methylate specific proteins, most of which contain GAR (glycine-arginine rich) motifs.  PRMT3 has been shown to play a role in the proper maturation of the 80S ribosome by binding to and catalyzing the methylation of rpS2, a component of the 40S ribosomal subunit.  However, the other roles of PRMT3 are fairly unclear, particularly in the brain, which is abundant in methylated proteins.  In this study, we perturbed PRMT3 expression in cultured rat hippocampal neurons by transiently introducing siRNA oligonucleotides that were designed to hybridize with PRMT3 mRNA and then we examd. the morphol. and functional effects of neuronal PRMT3 depletion.  PRMT3-defective neurons showed deformed spines without any change in spine no.; less BDNF-induced protein translation of αCaMKII; and diminished rpS2 protein stability.  Furthermore, overexpression of a methylation-resistant rpS2, whose methylated arginine residues were deleted, produced phenotypes that were similar to those assocd. with PRMT3 down regulation.  These findings demonstrated that PRMT3 possibly plays a pivotal role in neuronal translation by interaction with rpS2 and that it contributes to activity-dependent changes in the dendritic spines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZvFaoRRY5lbVg90H21EOLACvtfcHk0lh9M8KU6FQirw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L&md5=ccac2a5a931c483e516003342916bb17</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.brainres.2010.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainres.2010.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DPRMT3%2520is%2520Essential%2520for%2520Dendritic%2520Spine%2520Maturation%2520in%2520Rat%2520Hippocampal%2520Neurons%26jtitle%3DBrain%2520Res.%26date%3D2010%26volume%3D1352%26spage%3D11%26epage%3D20%26doi%3D10.1016%2Fj.brainres.2010.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Park, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heo, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. H.</span><span> </span><span class="NLM_article-title">New Role of Irisin in Hepatocytes: The Protective Effect of Hepatic Steatosis In Vitro</span> <span class="citation_source-journal">Cell. Signalling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1831</span><span class="NLM_x">–</span> <span class="NLM_lpage">1839</span><span class="refDoi"> DOI: 10.1016/j.cellsig.2015.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.cellsig.2015.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=25917316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFWgsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=1831-1839&author=M.+J.+Parkauthor=D.+I.+Kimauthor=J.+H.+Choiauthor=Y.+R.+Heoauthor=S.+H.+Park&title=New+Role+of+Irisin+in+Hepatocytes%3A+The+Protective+Effect+of+Hepatic+Steatosis+In+Vitro&doi=10.1016%2Fj.cellsig.2015.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">New role of irisin in hepatocytes: The protective effect of hepatic steatosis in vitro</span></div><div class="casAuthors">Park, Min-Jung; Kim, Dong-Il; Choi, Joo-Hee; Heo, Young-Ran; Park, Soo-Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1831-1839</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Irisin is a newly identified myokine related to exercise and the browning of white fat.  Recently, it was reported that irisin serum levels are assocd. with intrahepatic triglyceride content, suggesting that it might have an important role in the liver.  The aim of this study was to det. the role of irisin in hepatocytes.  Specifically, the effect of recombinant irisin on palmitic acid (PA)-induced lipogenesis and its related signal pathways were examd. in AML12 cells and mouse primary hepatocytes.  In the present study, we obsd. the presence of irisin inside the cells in response to the treatment of recombinant irisin by flow cytometry and cell imaging technique.  Recombinant irisin significantly inhibited the PA-induced increase in lipogenic markers ACC and FAS at the mRNA and protein levels, and prevented the PA-induced lipid accumulation in hepatocytes.  Addnl., irisin inhibited the PA-induced increase in the expression, nuclear localization, and transcriptional activities of the master regulators of lipogenesis (LXRα and SREBP-1c).  Moreover, irisin attenuated PA-induced oxidative stress, which was confirmed by measuring the expression of inflammatory markers (NFκB, COX-2, p38 MAPK, TNF, IL-6) and superoxide indicator (dihydroethidium).  The preventive effects of irisin against lipogenesis and oxidative stress were mediated by the inhibition of protein arginine methyltransferase-3 (PRMT3).  These findings suggested that irisin might have a beneficial role in the prevention of hepatic steatosis by altering the expression of lipogenic genes and attenuating oxidative stress in a PRMT3 dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom0-JmfttH1rVg90H21EOLACvtfcHk0ljdqVWDE_IoHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWgsbfO&md5=79d7e48751ea13de650d183d7e81dbbf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2015.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2015.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DM.%2BJ.%26aulast%3DKim%26aufirst%3DD.%2BI.%26aulast%3DChoi%26aufirst%3DJ.%2BH.%26aulast%3DHeo%26aufirst%3DY.%2BR.%26aulast%3DPark%26aufirst%3DS.%2BH.%26atitle%3DNew%2520Role%2520of%2520Irisin%2520in%2520Hepatocytes%253A%2520The%2520Protective%2520Effect%2520of%2520Hepatic%2520Steatosis%2520In%2520Vitro%26jtitle%3DCell.%2520Signalling%26date%3D2015%26volume%3D27%26spage%3D1831%26epage%3D1839%26doi%3D10.1016%2Fj.cellsig.2015.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span><span class="refDoi"> DOI: 10.1016/j.str.2012.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=P.+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+Allosteric+Inhibitor+of+Protein+Arginine+Methyltransferase+3&doi=10.1016%2Fj.str.2012.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0ljdqVWDE_IoHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520Allosteric%2520Inhibitor%2520of%2520Protein%2520Arginine%2520Methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435%26doi%3D10.1016%2Fj.str.2012.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting An Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+An+Allosteric+Binding+Site+of+PRMT3+Yields+Potent+and+Selective+Inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0ljdqVWDE_IoHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520An%2520Allosteric%2520Binding%2520Site%2520of%2520PRMT3%2520Yields%2520Potent%2520and%2520Selective%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kaniskan, H. Ü.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin-Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5166</span><span class="NLM_x">–</span> <span class="NLM_lpage">5170</span><span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+%C3%9C.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+Yuauthor=M.+S.+Eramauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+Liauthor=E.+Dobrovetskyauthor=A.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+Zhaoauthor=J.+Jinauthor=M.+Vedadi&title=A+Potent%2C+Selective+and+Cell-Active+Allosteric+Inhibitor+of+Protein+Arginine+Methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Uemit; Szewczyk, Magdalena M.; Yu, Zhengtian; Eram, Mohammad S.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Kennedy, Steven; Li, Fengling; Dobrovetsky, Elena; Dong, Aiping; Smil, David; Min, Sun-Joon; Landon, Melissa; Lin-Jones, Jennifer; Huang, Xi-Ping; Roth, Bryan L.; Schapira, Matthieu; Atadja, Peter; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Zhao, Kehao; Jin, Jian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-5170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biol.  Here we report the discovery of the first PRMT3 chem. probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochem., biophys., and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50=31±2 nM, KD=53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chem. probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTk4dFfkDLm7Vg90H21EOLACvtfcHk0ligxg5K1_dxRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D&md5=283b0875813af0f7631235e8ba7811c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520Potent%252C%2520Selective%2520and%2520Cell-Active%2520Allosteric%2520Inhibitor%2520of%2520Protein%2520Arginine%2520Methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Balz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiemann, G.</span><span> </span><span class="NLM_article-title">On Aromatic Fluoric Compounds, I.: A New Method for its Representation</span> <span class="citation_source-journal">Ber. Dtsch. Chem. Ges. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1927</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1186</span><span class="NLM_x">–</span> <span class="NLM_lpage">1190</span><span class="refDoi"> DOI: 10.1002/cber.19270600539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1002%2Fcber.19270600539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADyaB2sXitFyhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1927&pages=1186-1190&author=G.+Balzauthor=G.+Schiemann&title=On+Aromatic+Fluoric+Compounds%2C+I.%3A+A+New+Method+for+its+Representation&doi=10.1002%2Fcber.19270600539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatic fluorine compounds. I. A new method for their preparation</span></div><div class="casAuthors">Balz, Gunther; Schiemann, Gunther</div><div class="citationInfo"><span class="NLM_cas:title">Berichte der Deutschen Chemischen Gesellschaft [Abteilung] B: Abhandlungen</span>
        (<span class="NLM_cas:date">1927</span>),
    <span class="NLM_cas:volume">60B</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1186-90</span>CODEN:
                <span class="NLM_cas:coden">BDCBAD</span>;
        ISSN:<span class="NLM_cas:issn">0365-9488</span>.
    </div><div class="casAbstract">Contrary to the earlier belief, the diazonium fluoborates (cf. C. A. 21, 1070) decomp. smoothly and quietly on warming: ArN2BF4 → ArF + N2 + BF3.  There are no side reactions.  Because of their slight soly. they can be obtained in dry form readily and in good yield and they are very stable; they can even be recrystd. from hot H2O and are entirely insensitive to shock.  On decompn. they yield the fluoride in many cases almost quant. and in pure form even when the diazonium fluoborate has not previously been especially purified.  In general the appropriate amine is diazotized in as concd. soln. in HCl as possible, pptd. with excess of HBF4, (about 40%), washed with HBF, alc. and Et2O, dried in vacuo and heated in a fractionating flask (the glass is practically not attacked at all) connected with a condenser, the gaseous BF3 being absorbed in H2O or NaOH.  Phenyldiazonium fluoborate (yield 63 %), becomes slightly pink 80°, decomps. 121-2°, d425 1.53, yields almost 100% PhF, b. 86°.  p-Tolyldiazonium fluoborate (67%), decomps. 110°, d425 1.48; MeC6H4F (97%), b756 116°.  m-Xylyldiazonium fluoborate (31%; the yield Can easily be increased), decomps. 108°, d425 1.50; 2,4-e2C6H2F (almost 100%, b749 143-40°.  α.Naphthyldiazonium fluoborate (62%), decomps. 113°; α-C10H7F(98%),b17110°, b758 215°.  Diphenylene-4,4'-bisdiazonium fluoborate (64%),turns pink about 130°, foams 157°, d425 1.73; (P-FC6H4)2, m. 94-5°, d425 1.328.  o-,m-and p-Nitrophenyl-diazonium fluoborates (yields, 74, 80, 100%, resp.), decomp. 135°, 178°, 156°, d425 1.69, 1.66, 1.66.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCcxfRtVim6LVg90H21EOLACvtfcHk0ligxg5K1_dxRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaB2sXitFyhsw%253D%253D&md5=d87f8541798092e16fb21534d69dca75</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fcber.19270600539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.19270600539%26sid%3Dliteratum%253Aachs%26aulast%3DBalz%26aufirst%3DG.%26aulast%3DSchiemann%26aufirst%3DG.%26atitle%3DOn%2520Aromatic%2520Fluoric%2520Compounds%252C%2520I.%253A%2520A%2520New%2520Method%2520for%2520its%2520Representation%26jtitle%3DBer.%2520Dtsch.%2520Chem.%2520Ges.%2520B%26date%3D1927%26volume%3D60%26spage%3D1186%26epage%3D1190%26doi%3D10.1002%2Fcber.19270600539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Zhu, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claiborne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnone, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinghofer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oleksijew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoll, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Jong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oltersdorf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giranda, V. L.</span><span> </span><span class="NLM_article-title">Isoquinoline-Pyridine-Based Protein Kinase B/Akt Antagonists: SAR and In Vivo Antitumor Activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3150</span><span class="NLM_x">–</span> <span class="NLM_lpage">3155</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2006.03.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.bmcl.2006.03.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=3150-3155&author=G.+D.+Zhuauthor=J.+Gongauthor=A.+Claiborneauthor=K.+W.+Woodsauthor=V.+B.+Gandhiauthor=S.+Thomasauthor=Y.+Luoauthor=X.+Liuauthor=Y.+Shiauthor=R.+Guanauthor=S.+R.+Magnoneauthor=V.+Klinghoferauthor=E.+F.+Johnsonauthor=J.+Bouskaauthor=A.+Shoemakerauthor=A.+Oleksijewauthor=V.+S.+Stollauthor=R.+De+Jongauthor=T.+Oltersdorfauthor=Q.+Liauthor=S.+H.+Rosenbergauthor=V.+L.+Giranda&title=Isoquinoline-Pyridine-Based+Protein+Kinase+B%2FAkt+Antagonists%3A+SAR+and+In+Vivo+Antitumor+Activity&doi=10.1016%2Fj.bmcl.2006.03.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.03.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.03.041%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DG.%2BD.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DClaiborne%26aufirst%3DA.%26aulast%3DWoods%26aufirst%3DK.%2BW.%26aulast%3DGandhi%26aufirst%3DV.%2BB.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DR.%26aulast%3DMagnone%26aufirst%3DS.%2BR.%26aulast%3DKlinghofer%26aufirst%3DV.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DBouska%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DA.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DStoll%26aufirst%3DV.%2BS.%26aulast%3DDe%2BJong%26aufirst%3DR.%26aulast%3DOltersdorf%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DGiranda%26aufirst%3DV.%2BL.%26atitle%3DIsoquinoline-Pyridine-Based%2520Protein%2520Kinase%2520B%252FAkt%2520Antagonists%253A%2520SAR%2520and%2520In%2520Vivo%2520Antitumor%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D3150%26epage%3D3155%26doi%3D10.1016%2Fj.bmcl.2006.03.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">(R)-PFI-2 is a Potent and Selective Inhibitor of SETD7Methyltransferase Activity in Cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.+P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.+C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+Braamauthor=F.+M.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+is+a+Potent+and+Selective+Inhibitor+of+SETD7Methyltransferase+Activity+in+Cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520is%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520SETD7Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Kinetic Characterization of Human Histone H3 Lysine 36 Methyltransferases, ASH1L and SETD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">1842</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2015.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1016%2Fj.bbagen.2015.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=26002201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=1842-1848&author=M.+S.+Eramauthor=E.+Kuznetsovaauthor=F.+Liauthor=E.+Lima-Fernandesauthor=S.+Kennedyauthor=I.+Chauauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=Kinetic+Characterization+of+Human+Histone+H3+Lysine+36+Methyltransferases%2C+ASH1L+and+SETD2&doi=10.1016%2Fj.bbagen.2015.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span></div><div class="casAuthors">Eram, Mohammad S.; Kuznetsova, Ekaterina; Li, Fengling; Lima-Fernandes, Evelyne; Kennedy, Steven; Chau, Irene; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dysregulation of methylation of lysine 36 on histone H3 (H3K36) have been implicated in a variety of diseases including cancers.  ASH1L and SETD2 are two enzymes among others that catalyze H3K36 methylation.  H3K4 methylation has also been reported for ASH1L.  Radioactivity-based enzyme assays, Western and immunoblotting using specific antibodies and mol. modeling were used to characterize substrate specificity of ASH1L and SETD2.  Here we report on the assay development and kinetic characterization of ASH1L and SETD2 and their substrate specificities in vitro.  Both enzymes were active with recombinant nucleosome as substrate.  However, SETD2 but not ASH1L methylated histone peptides as well, indicating that the interaction of the basic post-SET extension with substrate may not be crit. for SETD2 activity.  Neither enzyme was active with nucleosome contg. a H3K36A mutation, indicating their specificity for H3K36.  Analyzing the methylation state of the products of ASH1L and SETD2 reactions also confirmed that both enzymes mono- and dimethylate H3K36 and are inactive with H3K4 as substrate, and that only SETD2 is able to trimethylate H3K36 in vitro.  We detd. the kinetic parameters for ASH1L and SETD2 activity, enabling screening for inhibitors that can be used to further investigate the roles of these two proteins in health and disease.  Both ASH1L and SETD2 are H3K36-specific methyltransferases but only SETD2 can trimethylate this mark.  The basic post-SET extension is crit. for ASH1L but not SETD2 activity.  We provide full kinetic characterization of ASH1L and SETD2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproBY7GkmOLrVg90H21EOLACvtfcHk0lgQZo5rxzwJqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D&md5=64839475d5ee9cc2f20e6e71f8e8a6ea</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DKinetic%2520Characterization%2520of%2520Human%2520Histone%2520H3%2520Lysine%252036%2520Methyltransferases%252C%2520ASH1L%2520and%2520SETD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bbagen.2015.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustos, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Trimethylation of Histone H3 Lysine 36 by Human Methyltransferase PRDM9 Protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">12177</span><span class="NLM_x">–</span> <span class="NLM_lpage">12188</span><span class="refDoi"> DOI: 10.1074/jbc.M113.523183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;key=10.1074%2Fjbc.M113.523183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=12177-12188&author=M.+S.+Eramauthor=S.+P.+Bustosauthor=E.+Lima-Fernandesauthor=A.+Siarheyevaauthor=G.+Senisterraauthor=T.+Hajianauthor=I.+Chauauthor=S.+Duanauthor=H.+Wuauthor=L.+Dombrovskiauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Trimethylation+of+Histone+H3+Lysine+36+by+Human+Methyltransferase+PRDM9+Protein&doi=10.1074%2Fjbc.M113.523183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523183%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DBustos%26aufirst%3DS.%2BP.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DTrimethylation%2520of%2520Histone%2520H3%2520Lysine%252036%2520by%2520Human%2520Methyltransferase%2520PRDM9%2520Protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D12177%26epage%3D12188%26doi%3D10.1074%2Fjbc.M113.523183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ','PDB','3SMQ'); return false;">PDB: 3SMQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG','PDB','4HSG'); return false;">PDB: 4HSG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RYL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RYL','PDB','4RYL'); return false;">PDB: 4RYL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i58"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01674">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65877"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01674">10.1021/acs.jmedchem.7b01674</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis and characterization data for compounds <b>49</b> and <b>50</b>, <sup>1</sup>NMR spectra of compounds <b>29</b>, <b>30</b>, <b>36</b>, and <b>37</b>, and selectivity data for compounds <b>29</b> and <b>30</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_001.pdf">jm7b01674_si_001.pdf (1.23 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01674/suppl_file/jm7b01674_si_002.csv">jm7b01674_si_002.csv (2.2 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-3%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01674%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01674" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799aec8dfb93db2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
