
<html lang="en"     class="pb-page"  data-request-id="db8f8ff1-c498-4655-a673-ecc340c1d596"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.9b01030;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-21;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structural Basis for Achieving GSK-3β Inhibition with High Potency, Selectivity, and Brain Exposure for Positron Emission Tomography Imaging and Drug Discovery" /></meta><meta name="dc.Creator" content="Vadim  Bernard-Gauthier" /></meta><meta name="dc.Creator" content="Andrew V.  Mossine" /></meta><meta name="dc.Creator" content="Ashley  Knight" /></meta><meta name="dc.Creator" content="Debasis  Patnaik" /></meta><meta name="dc.Creator" content="Wen-Ning  Zhao" /></meta><meta name="dc.Creator" content="Chialin  Cheng" /></meta><meta name="dc.Creator" content="Hema S.  Krishnan" /></meta><meta name="dc.Creator" content="Lucius L.  Xuan" /></meta><meta name="dc.Creator" content="Peter S.  Chindavong" /></meta><meta name="dc.Creator" content="Surya A.  Reis" /></meta><meta name="dc.Creator" content="Jinshan Michael  Chen" /></meta><meta name="dc.Creator" content="Xia  Shao" /></meta><meta name="dc.Creator" content="Jenelle  Stauff" /></meta><meta name="dc.Creator" content="Janna  Arteaga" /></meta><meta name="dc.Creator" content="Phillip  Sherman" /></meta><meta name="dc.Creator" content="Nicolas  Salem" /></meta><meta name="dc.Creator" content="David  Bonsall" /></meta><meta name="dc.Creator" content="Brenda  Amaral" /></meta><meta name="dc.Creator" content="Cassis  Varlow" /></meta><meta name="dc.Creator" content="Lisa  Wells" /></meta><meta name="dc.Creator" content="Laurent  Martarello" /></meta><meta name="dc.Creator" content="Shil  Patel" /></meta><meta name="dc.Creator" content="Steven H.  Liang" /></meta><meta name="dc.Creator" content="Ravi G.  Kurumbail" /></meta><meta name="dc.Creator" content="Stephen J.  Haggarty" /></meta><meta name="dc.Creator" content="Peter J. H.  Scott" /></meta><meta name="dc.Creator" content="Neil  Vasdev" /></meta><meta name="dc.Description" content="Using structure-guided design, several cell based assays, and microdosed positron emission tomography (PET) imaging, we identified a series of highly potent, selective, and brain-penetrant oxazole-..." /></meta><meta name="Description" content="Using structure-guided design, several cell based assays, and microdosed positron emission tomography (PET) imaging, we identified a series of highly potent, selective, and brain-penetrant oxazole-..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 19, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01030" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01030" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01030" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01030" /></link>
        
    
    

<title>Structural Basis for Achieving GSK-3β Inhibition with High Potency, Selectivity, and Brain Exposure for Positron Emission Tomography Imaging and Drug Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01030" /></meta><meta property="og:title" content="Structural Basis for Achieving GSK-3β Inhibition with High Potency, Selectivity, and Brain Exposure for Positron Emission Tomography Imaging and Drug Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0007.jpeg" /></meta><meta property="og:description" content="Using structure-guided design, several cell based assays, and microdosed positron emission tomography (PET) imaging, we identified a series of highly potent, selective, and brain-penetrant oxazole-4-carboxamide-based inhibitors of glycogen synthase kinase-3 (GSK-3). An isotopologue of our first-generation lead, [3H]PF-367, demonstrates selective and specific target engagement in vitro, irrespective of the activation state. We discovered substantial ubiquitous GSK-3-specific radioligand binding in Tg2576 Alzheimer’s disease (AD), suggesting application for these compounds in AD diagnosis and identified [11C]OCM-44 as our lead GSK-3 radiotracer, with optimized brain uptake by PET imaging in nonhuman primates. GSK-3β-isozyme selectivity was assessed to reveal OCM-51, the most potent (IC50 = 0.030 nM) and selective (&gt;10-fold GSK-3β/GSK-3α) GSK-3β inhibitor known to date. Inhibition of CRMP2T514 and tau phosphorylation, as well as favorable therapeutic window against WNT/β-catenin signaling activation, was observed in cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01030"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01030">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01030&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01030&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01030&amp;href=/doi/10.1021/acs.jmedchem.9b01030" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9600-9617</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01019" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01032" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structural Basis for Achieving GSK-3β Inhibition with High Potency, Selectivity, and Brain Exposure for Positron Emission Tomography Imaging and Drug Discovery</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Vadim Bernard-Gauthier</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vadim Bernard-Gauthier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Azrieli Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</div><div class="loa-info-affiliations-info">Department of Psychiatry/Institute of Medical Science, University of Toronto, Toronto, Ontario M5T 1R8, Canada</div><div class="loa-info-affiliations-info">Division of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital and Department of Radiology, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#5e283f3a37333c393f2b2a36373b2c1e39333f3732703d3133"><span class="__cf_email__" data-cfemail="f68097929f9b94919783829e9f9384b6919b979f9ad895999b">[email protected]</span></a> (V.B.-G.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vadim++Bernard-Gauthier">Vadim Bernard-Gauthier</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew V. Mossine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew V. Mossine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+V.++Mossine">Andrew V. Mossine</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ashley Knight</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ashley Knight</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Azrieli Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</div><div class="loa-info-affiliations-info">Department of Psychiatry/Institute of Medical Science, University of Toronto, Toronto, Ontario M5T 1R8, Canada</div><div class="loa-info-affiliations-info">Eisai AiM Institute, Boston, Massachusetts 01810, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ashley++Knight">Ashley Knight</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2225-360X" title="Orcid link">http://orcid.org/0000-0003-2225-360X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Debasis Patnaik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Debasis Patnaik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Debasis++Patnaik">Debasis Patnaik</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wen-Ning Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wen-Ning Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wen-Ning++Zhao">Wen-Ning Zhao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5824-2032" title="Orcid link">http://orcid.org/0000-0002-5824-2032</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chialin Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chialin Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chialin++Cheng">Chialin Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hema S. Krishnan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hema S. Krishnan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital and Department of Radiology, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hema+S.++Krishnan">Hema S. Krishnan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lucius L. Xuan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lucius L. Xuan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lucius+L.++Xuan">Lucius L. Xuan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter S. Chindavong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter S. Chindavong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+S.++Chindavong">Peter S. Chindavong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Surya A. Reis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Surya A. Reis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Surya+A.++Reis">Surya A. Reis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jinshan Michael Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinshan Michael Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, Groton Laboratories, Eastern Point Road, Groton, Connecticut 06340, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinshan+Michael++Chen">Jinshan Michael Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xia Shao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xia Shao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xia++Shao">Xia Shao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jenelle Stauff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jenelle Stauff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jenelle++Stauff">Jenelle Stauff</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Janna Arteaga</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janna Arteaga</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janna++Arteaga">Janna Arteaga</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Phillip Sherman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Phillip Sherman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Phillip++Sherman">Phillip Sherman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicolas Salem</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Salem</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biogen, Research and Early Development Imaging, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Salem">Nicolas Salem</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Bonsall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Bonsall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro, London W12 0NN, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Bonsall">David Bonsall</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brenda Amaral</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brenda Amaral</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biogen, Research and Early Development Imaging, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brenda++Amaral">Brenda Amaral</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cassis Varlow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cassis Varlow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Azrieli Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cassis++Varlow">Cassis Varlow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lisa Wells</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lisa Wells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro, London W12 0NN, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lisa++Wells">Lisa Wells</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laurent Martarello</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurent Martarello</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biogen, Research and Early Development Imaging, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurent++Martarello">Laurent Martarello</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shil Patel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shil Patel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Eisai AiM Institute, Boston, Massachusetts 01810, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shil++Patel">Shil Patel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steven H. Liang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven H. Liang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital and Department of Radiology, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven+H.++Liang">Steven H. Liang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1413-6315" title="Orcid link">http://orcid.org/0000-0003-1413-6315</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ravi G. Kurumbail</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ravi G. Kurumbail</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, Groton Laboratories, Eastern Point Road, Groton, Connecticut 06340, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ravi+G.++Kurumbail">Ravi G. Kurumbail</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen J. Haggarty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen J. Haggarty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Neurobiology Laboratory, Massachusetts General Hospital, Center for Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+J.++Haggarty">Stephen J. Haggarty</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7872-168X" title="Orcid link">http://orcid.org/0000-0002-7872-168X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter J. H. Scott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. H. Scott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">The Interdepartmental Program in Medicinal Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.+H.++Scott">Peter J. H. Scott</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Neil Vasdev</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Neil Vasdev</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Azrieli Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</div><div class="loa-info-affiliations-info">Department of Psychiatry/Institute of Medical Science, University of Toronto, Toronto, Ontario M5T 1R8, Canada</div><div class="loa-info-affiliations-info">Division of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital and Department of Radiology, Harvard Medical School, Boston, Massachusetts 02114, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#d6b8b3bfbaf8a0b7a5b2b3a096a3a2b9a4b9b8a2b9f8b5b7"><span class="__cf_email__" data-cfemail="bfd1dad6d391c9deccdbdac9ffcacbd0cdd0d1cbd091dcde">[email protected]</span></a> (N.V.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Neil++Vasdev">Neil Vasdev</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2087-5125" title="Orcid link">http://orcid.org/0000-0002-2087-5125</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01030&amp;href=/doi/10.1021%2Facs.jmedchem.9b01030" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9600–9617</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 19, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 June 2019</li><li><span class="item_label"><b>Published</b> online</span>19 September 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01030" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01030</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9600%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DVadim%2BBernard-Gauthier%252C%2BAndrew%2BV.%2BMossine%252C%2BAshley%2BKnight%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D21%26contentID%3Dacs.jmedchem.9b01030%26title%3DStructural%2BBasis%2Bfor%2BAchieving%2BGSK-3%25CE%25B2%2BInhibition%2Bwith%2BHigh%2BPotency%252C%2BSelectivity%252C%2Band%2BBrain%2BExposure%2Bfor%2BPositron%2BEmission%2BTomography%2BImaging%2Band%2BDrug%2BDiscovery%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9617%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01030"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1637</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01030" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structural Basis for Achieving GSK-3β Inhibition with High Potency, Selectivity, and Brain Exposure for Positron Emission Tomography Imaging and Drug Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Vadim&quot;,&quot;last_name&quot;:&quot;Bernard-Gauthier&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;V. Mossine&quot;},{&quot;first_name&quot;:&quot;Ashley&quot;,&quot;last_name&quot;:&quot;Knight&quot;},{&quot;first_name&quot;:&quot;Debasis&quot;,&quot;last_name&quot;:&quot;Patnaik&quot;},{&quot;first_name&quot;:&quot;Wen-Ning&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Chialin&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Hema&quot;,&quot;last_name&quot;:&quot;S. Krishnan&quot;},{&quot;first_name&quot;:&quot;Lucius&quot;,&quot;last_name&quot;:&quot;L. Xuan&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;S. Chindavong&quot;},{&quot;first_name&quot;:&quot;Surya&quot;,&quot;last_name&quot;:&quot;A. Reis&quot;},{&quot;first_name&quot;:&quot;Jinshan&quot;,&quot;last_name&quot;:&quot;Michael Chen&quot;},{&quot;first_name&quot;:&quot;Xia&quot;,&quot;last_name&quot;:&quot;Shao&quot;},{&quot;first_name&quot;:&quot;Jenelle&quot;,&quot;last_name&quot;:&quot;Stauff&quot;},{&quot;first_name&quot;:&quot;Janna&quot;,&quot;last_name&quot;:&quot;Arteaga&quot;},{&quot;first_name&quot;:&quot;Phillip&quot;,&quot;last_name&quot;:&quot;Sherman&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Salem&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Bonsall&quot;},{&quot;first_name&quot;:&quot;Brenda&quot;,&quot;last_name&quot;:&quot;Amaral&quot;},{&quot;first_name&quot;:&quot;Cassis&quot;,&quot;last_name&quot;:&quot;Varlow&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;Wells&quot;},{&quot;first_name&quot;:&quot;Laurent&quot;,&quot;last_name&quot;:&quot;Martarello&quot;},{&quot;first_name&quot;:&quot;Shil&quot;,&quot;last_name&quot;:&quot;Patel&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;H. Liang&quot;},{&quot;first_name&quot;:&quot;Ravi&quot;,&quot;last_name&quot;:&quot;G. Kurumbail&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;J. Haggarty&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. H. Scott&quot;},{&quot;first_name&quot;:&quot;Neil&quot;,&quot;last_name&quot;:&quot;Vasdev&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9600-9617&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01030&quot;},&quot;abstract&quot;:&quot;Using structure-guided design, several cell based assays, and microdosed positron emission tomography (PET) imaging, we identified a series of highly potent, selective, and brain-penetrant oxazole-4-carboxamide-based inhibitors of glycogen synthase kinase-3 (GSK-3). An isotopologue of our first-generation lead, [3H]PF-367, demonstrates selective and specific target engagement in vitro, irrespective of the activation state. We discovered substantial ubiquitous GSK-3-specific radioligand binding in Tg2576 Alzheimer’s disease (AD), suggesting application for these compounds in AD diagnosis and identified [11C]OCM-44 as our lead GSK-3 radiotracer, with optimized brain uptake by PET imaging in nonhuman primates. GSK-3β-isozyme selectivity was assessed to reveal OCM-51, the most potent (IC50 = 0.030 nM) and selective (&gt;10-fold GSK-3β/GSK-3α) GSK-3β inhibitor known to date. Inhibition of CRMP2T514 and tau phosphorylation, as well as favorable therapeutic window against WNT/β-catenin signaling activation, was observe&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01030&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01030" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01030&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01030" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01030&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01030" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01030&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01030&amp;href=/doi/10.1021/acs.jmedchem.9b01030" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01030" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01030" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01030%26sid%3Dliteratum%253Aachs%26pmid%3D31535859%26genre%3Darticle%26aulast%3DBernard-Gauthier%26date%3D2019%26atitle%3DStructural%2BBasis%2Bfor%2BAchieving%2BGSK-3%25CE%25B2%2BInhibition%2Bwith%2BHigh%2BPotency%252C%2BSelectivity%252C%2Band%2BBrain%2BExposure%2Bfor%2BPositron%2BEmission%2BTomography%2BImaging%2Band%2BDrug%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D21%26spage%3D9600%26epage%3D9617%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292462" title="Central nervous system">Central nervous system</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/jmcmar.2019.62.issue-21/20191114/jmcmar.2019.62.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Using structure-guided design, several cell based assays, and microdosed positron emission tomography (PET) imaging, we identified a series of highly potent, selective, and brain-penetrant oxazole-4-carboxamide-based inhibitors of glycogen synthase kinase-3 (GSK-3). An isotopologue of our first-generation lead, [<sup>3</sup>H]PF-367, demonstrates selective and specific target engagement in vitro, irrespective of the activation state. We discovered substantial ubiquitous GSK-3-specific radioligand binding in Tg2576 Alzheimer’s disease (AD), suggesting application for these compounds in AD diagnosis and identified [<sup>11</sup>C]OCM-44 as our lead GSK-3 radiotracer, with optimized brain uptake by PET imaging in nonhuman primates. GSK-3β-isozyme selectivity was assessed to reveal OCM-51, the most potent (IC<sub>50</sub> = 0.030 nM) and selective (>10-fold GSK-3β/GSK-3α) GSK-3β inhibitor known to date. Inhibition of CRMP2<sup>T514</sup> and tau phosphorylation, as well as favorable therapeutic window against WNT/β-catenin signaling activation, was observed in cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81645" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81645" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Glycogen synthase kinase-3α and β (GSK-3α/β) are ubiquitous multifunctional serine/threonine kinases, which regulate diverse physiological functions in peripheral organs and the central nervous system (CNS), ranging from metabolism and cell cycle regulation to brain development. Appropriately, balanced GSK-3 regulation and signaling is therefore central to human physiology but often compromised in diseases of diverse etiologies such as diabetes, cancers, and neurological disorders.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> GSK-3α/β are remarkably networked enzymes, with involvement in numerous pathways and over one hundred distinct putative substrates.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Also, unusual among kinases are the fact that GSK-3α/β regulation occurs via inactivation while resting cells show high levels of catalytic activity. The inactivation of GSK-3 has been well-characterized and revealed primarily to proceed through N-terminus serine phosphorylation catalyzed by kinases such as AKT/PKB, p70RSK, and p90RSK (at serine 21 and serine 9 for GSK-3α and GSK-3β, respectively).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Of note, GSK-3α and GSK-3β present a significant kinase domain homology with one another, especially within the ATP-binding pocket (97% identity).<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Congruently, GSK-3α/β also exhibits substantial functional overlap, although some elements of nonredundancy such as GSK-3β-specific perinatal lethality have been characterized from paralogue-specific <i>Gsk3</i> knockout model studies, which together support the notion that individual isozyme contributions should be defined carefully.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a></div><div class="NLM_p">Several immunohistochemical, cellular, transgenic, and preclinical studies have implicated GSK-3 deregulation as a contributive factor in neurological diseases including Alzheimer’s disease (AD) and non-AD tauopathies.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a> While in most cases, GSK-3β has been the central focus in these studies, both GSK-3α/β overexpression and inhibition/knockdown experiments have demonstrated roles in APP processing and correlated with increased or decreased amyloid-β production, respectively.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> In turn, amyloid-β injections in rat trigger APP-processing changes and pathological tau phosphorylation linked to GSK-3β overexpression and activation.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The hyperphosphorylation of tau by GSK-3α/β also suggests a central implication in neurofibrillary tangle formation, however whether an increase in GSK-3α/β levels or activities alone, rather than parallel changes in phosphatase activities, constitute the causative pathogenic factor in tau hyperphosphorylation and neurofibrillary tangle accumulation remains to be clarified.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> Both inhibition of GSK-3 in transgenic mice-overexpressing human tau and overexpressing conditional <i>Gsk3b</i> mice were associated with changes in hyperphosphorylated tau protein (pTau) levels tightly linked to GSK-3 levels and activity in vivo.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Current evidence using human tissue suggests, as with most immunohistochemical cellular and transgenic rodent analyses, increases in total and active GSK-3β levels in AD compared to healthy controls.<a onclick="showRef(event, 'ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23">(21−23)</a> However, data gathered in human brains have been largely insufficient to substantiate in vitro preclinical findings while in vivo pharmacological intervention studies in transgenic mice as well as clinical trials aimed at validating GSK-3 in humans have been obscured by the systematic use of promiscuous inhibitors.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> Indeed, much of the inability to define whether GSK-3 is an actionable target in neurodegenerative diseases in vivo ensue from the lack of suitable molecular tools. To address this, highly potent, selective, and brain penetrant inhibitors of GSK-3 are needed. However, apart from rare exceptions where some of these criteria have been met, so far clinical and preclinical GSK-3 inhibitors have been both sub-optimal regarding potency or brain exposure and have displayed, despite sporadic claims of selectivity in restricted kinase subsets, poor overall kinome selectivity when systematically evaluated in large-scale screening.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27−31)</a> Moreover, only one recent report has described kinome-selective GSK-3 inhibitors also achieving α/β-isozyme selectivity, however, none of these isozyme-selective compounds exhibit single-digit nanomolar potencies or have been characterized in the CNS tissue.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p last">Herein, we used structure-guided design and microdosed positron emission tomography (PET) imaging to develop novel small-molecule inhibitors of GSK-3 achieving high potency, selectivity, and brain exposure. We describe the rational development of the novel GSK-3β-isozyme selective inhibitor, named herein as OCM-51 (<b>51</b>), which to the best of our knowledge, constitutes the most potent and selective GSK-3 inhibitor identified thus far. Biochemical, biophysical, cellular, and brain tissue-based assays collectively were used to confirm highly selective target engagement. Further analysis also demonstrates differential binding profiles in AD transgenic mice brains versus wild-type. Together, our results describe finely characterized candidate inhibitors which can lead to the development of GSK-3 specific therapeutics as well as diagnostic GSK-3 PET molecular probes for neuroimaging.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23378" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23378" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> [<sup>3</sup>H]PF-367 Engages GSK-3 with High Specificity in Cells and Human Postmortem Brain</h3><div class="NLM_p">Previously, we reported the discovery of PF-04802367 (PF-367; <b>1</b>), a potent type-I oxazole-4-carboxamide dual GSK-3α/β inhibitor, showing promising ADME properties combined with robust CNS/peripheral pTau and muscle pGS inhibition in vivo.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Here, we first tested whether the lead compound, PF-367, displayed specific GSK-3α/β binding in the mammalian brain tissue. We developed [<sup>3</sup>H]<b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) and observed, in rat, marmoset and human brains (frontal cortex and hippocampus), high and nearly ubiquitous total binding in the gray matter, consistent with known GSK-3α/β mRNA and protein levels (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B–F and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S1B,C</a>). Highest radioligand binding was measured in the cerebral cortex and consistently more pronounced within gray matter regions compared to the white matter in all species (50–60% difference). To confirm that our GSK-3α/β marker provided saturable signal, homologous competition was performed on adjacent sections using unlabeled <b>1</b> at 10 μM and resulted in significantly reduced binding compared to baseline experiments and overall low levels of nonsaturable binding (>87, 66, and 85% reductions in rat, marmoset, and human cerebral cortices, respectively, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,D,E). We then used two structurally unrelated and well-characterized heterologous pan-GSK-3 competitive inhibitors to determine on-target binding selectivity. Single concentration cotreatments with the advanced preclinical inhibitor CHIR99021 and the clinical lead LY2090314 both resulted in substantial decreases in tracer binding similar to unlabeled <b>1</b> incubations (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B–F and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S1A</a>). In all species and brain regions, there were no significant discrepancies in blocking effects between <b>1</b> and the heterologous probes, indicating that GSK-3α/β specificity could be determined using any of these inhibitors interchangeably.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. [<sup>3</sup>H]PF-04802367 ([<sup>3</sup>H]<b>1</b>) displays high specific and selective binding to GSK-3α/β in cells and mammalian brains in vitro. (A) Chemical structure of [<sup>3</sup>H]<b>1</b>. (B) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in the normal rat brain (20 μm sections, <i>n</i> = 3) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (C) In vitro autoradiography images of [<sup>3</sup>H]<b>1</b> normal rat brain (coronal sections) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (D) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in the normal marmoset brain (20 μm sections, <i>n</i> = 3) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (E) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in the normal human brain (20 μm sections, frontal cortex, <i>n</i> = 6) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (F) In vitro autoradiography images of [<sup>3</sup>H]<b>1</b> normal human brain under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (G) Saturation [<sup>3</sup>H]<b>1</b> binding curves, Scatchart plot, and <i>B</i><sub>max</sub> determination using the homogenate mouse brain tissue. (H,I) Cell binding assays with [<sup>3</sup>H]<b>1</b> in HEK-293T P301S hTau cells, overexpressing GSK-3α/β and GSK-3α/β knockdown (shRNA). [<sup>3</sup>H]<b>1</b> binding under 2h baseline and following timecourse chase experiments. (J) Western blot analysis of control, GSK-3α/β-overexpressing, and GSK-3α/β shRNA cells used for [<sup>3</sup>H]<b>1</b>-binding studies. Data show mean values ± standard deviation (SD). Statistical significance was calculated one-way ANOVA followed by Dunnett’s test (B,D,E,H).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further assess our probe in a cellular context, [<sup>3</sup>H]<b>1</b> binding in HEK293T P301S hTau cells overexpressing GSK-3α/β, shRNA-induced GSK-3α/β knockdown, and nonoverexpressing HEK293T P301S hTau cells was compared (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H–J and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S1F</a>). Significantly higher binding was detected in overexpressing cells compared to GSK-3α/β knockdown (three-fold) and control (six-fold) while cold chase studies (where nonradioactive agent is administered following the radiotracer) resulted in consistent time-dependent binding reductions in all cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H,I). Strong, dose-dependent, GSK-3 specific, and reversible binding was observed with overexpressing HEK293T P301S hTau cells when incubated with [<sup>3</sup>H]<b>1</b> in competition (co-incubation, 85% displacement at 5 μM) and pulse chase (96% displacement at 5 μM for 5 min) studies with a third heterologous inhibitor (e.g., AR79, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S1A,G</a>). Further evidence of GSK-3α/β target engagement and selectivity on brain tissue was found when comparing tumor to normal CNS contralateral tissue binding of [<sup>3</sup>H]<b>1</b> in rat brain sections with intracranial GS9L gliomas in vitro (>40% binding differences, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S1D,E</a>). We next performed binding saturation experiments using mouse brain homogenates which determined a GSK-3α/β binding site density (<i>B</i><sub>max</sub>) of 960.33 ± 234 fmol/mg and an affinity in line with biochemical data (<i>K</i><sub>D</sub> = 13.02 ± 3.3 nM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G). Consistently throughout these cellular and whole tissue assays, the observed signals are predictably derived from GSK-3α and GSK-3β—as large-scale proteomic analyses have indicated only about two- to fourfold higher CNS B<sub>max</sub> for GSK-3β when compared to GSK-3α.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> Together, these results also indicated that the <b>1</b> scaffold aligns suitable target engagement and selectivity properties to serve as a starting point in our lead-to-candidate optimization efforts.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Development and Characterization of Selective Oxazole-4-carboxamide GSK-3 Inhibitors</h3><div class="NLM_p">The hit-to-lead screening campaign which led to <b>1</b> had pointed out the importance of three key fragments with regard to GSK-3 potency and selectivity: (1) the 1,2,4-triazol-1-yl group involved in a cation−π interaction with Arg141; (2) a critical core oxazole-4-carboxamide moiety providing hinge domain–ligand hydrogen bond-pairing interactions around Val135/Asp133; and (3) the 3-chloro-4-methoxyphenyl ring mainly involved in hydrophobic contacts within the deep pocket formed between the N- and C-lobes in the vicinity of Phe67, Lys85, and Leu132 (GSK-3β residues) and corresponding GSK-3α residues.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> On the basis of the aforementioned in vitro cell and tissue studies, and guided by structural data gained previously, we undertook structural refinement efforts to improve blood–brain barrier (BBB) permeation, which is suboptimal for <b>1</b>,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and in vivo properties of our lead while retaining or improving physical and biochemical properties.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> To identify next-generation inhibitors, >50 optimized compounds were synthesized and underwent biochemical screening (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods and Tables S1–S4). In designing derivatives, emphasis was placed on the exploration of the effects of halogenation patterns, primarily fluorinations, and the modulation of the cation−π interacting group while keeping the oxazole-4-carboxamide pharmacophore intact and minimizing changes detrimental to calculated Fu_p (unbound brain fraction) and CNS multiparameter optimization (MPO) scores (properties which are suboptimal for most known GSK-3 inhibitors, including PET probes evaluated to date, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods and Tables S1–S4). Our structure–activity relationship (SAR) study around the cation−π interacting side chain amide substituents showed that 6-membered nitrogen-containing rings (pyrazine, pyridines) combined with 3-“chlorine-for-fluorine” substitutions at the 4-methoxyphenyl fragment were well tolerated. We identified three inhibitors with such structural patterns displaying single-digit nanomolar potencies toward GSK-3β (namely, PF-618 (<b>28</b>), OCM-37 (<b>37</b>), and OCM-44 (<b>44</b>); IC<sub>50</sub> < 2.5 nM) in our ADPGlo screening assay (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Gratifyingly, these structural modifications were associated with profound CNS-penetration enhancements in vivo (vide infra). The high binding affinity and dual GSK-3α/β inhibitory nature of these analogues were confirmed in a second biochemical assay format, utilizing radiometric filtration (γ-[<sup>33</sup>P]-ATP kinase assay, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods) (GSK-3α/β <i>K</i><sub>i</sub> = 3.3/4.2 nM, <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D). When assayed against a comprehensive panel of 371 kinase targets, these inhibitors displayed distinctive selectivity, inhibiting only two targets (DYRK1A and DYRK1B) beyond IC<sub>50</sub> concentrations at 1.0 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Tables S5 and S6</a>). While activity overlap between GSK-3 and the Down syndrome kinases DYRK1s is common across inhibitors aimed at both targets, we observed that in this case, it may be exacerbated by ligand-receptor π–π interactions from a conserved DYRK1A/B Asp247/198, mimicking the guanidinium Arg141 cation−π interaction (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E,G–I).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> However, we found 40–170-fold selectivity for GSK-3α/β when measuring specific inhibitory constants even against the closest off-targets (DYRK1A/DYRK1B <i>K</i><sub>i</sub> = 325 nM/171 nM; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D), well within the requirements expected for therapeutics and PET radioligand development.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Evolution and in vitro pharmacological characterization of a highly potent and selective oxazole-4-carboxamide-based GSK-3 inhibitor series. (A) Generalized strategy to synthesize arrays of carboxamide-based GSK-3 inhibitors, as well as structures and properties of selected GSK-3 inhibitors (for details on chemical synthesis, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Schemes S1–S4</a>). IC<sub>50</sub> values determined using an ADP-Glo kinase assay (<i>n</i> = 2). (B) Heat map representing key properties [IC<sub>50</sub>, central nervous system multiparameter optimization (CNS MPO), and Fu_p] of oxazole-4-carboxamide-based GSK-3 inhibitors including a comparison with previously described GSK-3 PET radiotracers (for structures and detailed parameter values, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Tables S1–S4</a>). IC<sub>50</sub> values determined using an ADP-GloTM kinase assay (<i>n</i> = 2). (C,D) The dose–response curves and binding affinities of the representative inhibitor <b>28</b> versus GSK-3α, GSK-3β (<i>n</i> = 3), and main kinases off targets DYRK1A, DRYK1B, and CLK4 (<i>n</i> = 1) demonstrate high selectivity for GSK-3α/β (radiometric filtration binding assay, [ATP] = <i>K</i><sub>m ATP</sub>, data show mean values ± SD). (E) Hinge/α2 region sequence alignment of GSK-3β and the main kinase secondary target DYRK1A. (F) Comprehensive kinase selectivity profile of representative inhibitor <b>44</b> and <b>28</b> against 371 kinases (kinases are ordered alphabetically, and the activity is relative to control at 1.0 μM inhibitor concentration, <i>n</i> = 2, [γ-<sup>33</sup>P]ATP-based enzymatic assay performed by Reaction Biology, for detailed data see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S5</a>). (G) Overlay of the hinge/α2 region of GSK-3β (gray, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>) and DYRK1A (yellow, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR">3ANR</a>). (H) Predicted detailed interaction of <b>44</b> (yellow sticks) with GSK-3β (gray cartoon, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>). The π–cation interaction is depicted as a black dash line. (I) Alternate view of the binding mode of <b>44</b> (yellow, space fill model) with GSK-3β (surface rendering, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>). CNS MPO; Fu_p, in silico calculated unbound fraction in plasma.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Oxazole-4-carboxamide Probes Engage npS9-GSK-3β and pS9-GSK-3β Irrespective of Activation States</h3><div class="NLM_p">Pathological changes associated with GSK-3β may be overexpression- or overactivation-driven, therefore it is crucial to address whether molecular probes engaging GSK-3β do so dependently or independently of the S9 phosphorylation status in order to extrapolate valid imaging applications (S21 in GSK-3α isozyme). To elucidate possible target engagement discrepancies between npS9-GSK-3β (active nonphospho-S9) and pS9-GSK-3β (inactive phospho-S9), we developed a high-sensitivity microscale thermophoresis (MST)-based binding assay. Use of MST allows for high-precision determination of equilibrium dissociation constants by measuring temperature-induced changes in a fluorescently labeled target protein as a function of small amounts of nonfluorescent small molecules with minimal influence by ATP concentration and other such factors that impact functional cell-base assays.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> We generated catalytically inactive pS9-GSK-3β by treating nonphosphorylation active npS9-GSK-3β (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) with AKT and ATP. When npS9-GSK-3β was treated with AKT and high ATP concentration for 3 h, complete loss of catalytic activity was observed, and high level of pS9-GSK-3β was produced as determined by ADPGlo kinase and western blot analyses (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). Under control conditions using ATP treatments alone, GSK-3β catalytic activity remained unaffected (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). Characterization by MST of the npS9-GSK-3β and pS9-GSK-3β proteins first confirmed the high binding affinities of oxazole-4-carboxamide inhibitors toward npS9-GSK-3β (<i>K</i><sub>d</sub> = 0.8 and 4.5 nM for <b>1</b> and <b>44</b> respectively) and revealed only marginal differences versus inactive pS9-GSK-3β with regard to target engagement (<two-fold, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E–I). Under the same conditions, CHIR99021 GSK-3β binding was also largely independent of the S9 status (<i>K</i><sub>d</sub> = 48.8 and 28.3 nM for pS9-GSK-3β and npS9-GSK-3β, respectively, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S2A</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Oxazole-4-carboxamide-based probes engage both active npS9 and catalytically inactive pS9 GSK-3β. (A) Structural basis for the recognition of inhibitors by pS9-GSK-3β (<b>44</b> is depicted as a representative inhibitor, yellow, space fill model). The predicted binding mode illustrates the distal positioning of <b>44</b> from the activation loop and pS9 auto-inhibitory N-terminal peptide (blue surface) and the negligible ATP-binding site rearrangement upon binding of the pS9 auto-inhibitory N-terminal peptide of GSK-3β (gray surface, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NM3">4NM3</a>). (B) Timecourse analysis showing GSK-3β activity in the presence of low ATP content and GSP2 (ADP-Glo-based kinase assay, <i>n</i> = 3). (C) Time-course analysis showing GSK-3β activity in the presence of high ATP content and high ATP content combined with AKT (ADP-Glo-based kinase assay, <i>n</i> = 3). (D) Western blot analysis of enzymatic reaction contents (control, ATP treated and ATP + AKT treated). (E) Binding affinities of <b>1</b> and <b>44</b> measured by MST. (F,G) MST dose–response curves for the binding interaction of between <b>1</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). (H,I) MST dose–response curves for the binding interaction of <b>44</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). Data show mean values ± SD. Statistical significance was calculated with paired <i>t</i>-test (B,C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Consistent with these MST results, in silico experiments and X-ray crystal structure analyses indicated that conformational changes occurring upon N-terminal inhibitory peptide sequence binding do so remotely from the inhibitor-binding site (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). GSK-3β structural remodeling following pS9 autoinhibition primarily affects the C-loop and activation segment, which are positioned distantly from the bound inhibitor and have negligible impact on the ATP-binding site conformations, thus having limited impact on the target engagement of type-I ATP competitive inhibitors as observed here (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> On the basis of these results, we conclude that oxazole-4-carboxamide-based probes, such as [<sup>3</sup>H]<b>1</b> and positron emitter-labeled derivatives thereof, provide a genuine assessment of total GSK-3 density irrespective of the S9 phosphorylation.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Discovery of Structural Determinants To Generate Picomolar GSK-3β-Isozyme Selective Inhibitors</h3><div class="NLM_p">In the course of the initial SAR study, we observed that the orientation of the 3-halogen phenyl substituents found in this series was optimal with respect to the deep hydrophobic cleft found in GSK-3β (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,H and C,E–G). Close inspection of GSK-3β-<b>1</b> and other GSK-3β cocrystal structures indicated that, as recently noted by Wagner et al. 2018, the topology of this pocket is dependent on a complex backend hydrogen-bonding network involving the side chain of Asp133, the only nonconserved residue from the ATP-binding site between GSK-3α and GSK-3β (Glu196 in GSK-3α).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> We observed that the hydrogen bond interactions in GSK-3β which bring the side chains of Asp133, Arg113, and Glu80 in close contact also directly impact proximal hydrophobic residues from β sheets 5 and 6 (gatekeeper Leu132, Leu130 and Leu112), pulling these residues away from the ATP-binding site and forcing the opening a deep yet narrow hydrophobic pocket, wherein the paralogue GSK-3α residue Glu196 could lack the ability to secure such interactions in the first place. Finer tuning of the occupation of this pocket using subtle variation in halogen patterning, we reasoned, would allow for the possibility of achieving β-isozyme selectivity at high potency.</div><div class="NLM_p">To assess this, we developed a subseries of derivatives leading to inhibitor <b>51</b>, which gratifyingly provided not only 65-fold improvement in potency compared to <b>44</b> (GSK-3β IC<sub>50</sub> = 0.031 nM) but also >10-fold selectivity for GSK-3β over GSK-3α (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,C,E–G and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S2B</a>). Similar results were obtained using our confirmation [<sup>33</sup>P]-radiometric binding assays (<i>K</i><sub>i</sub> = 0.10 nM, >10-fold GSK-3β selectivity over GSK-3α, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B,D). Increases in potency moving from <b>44</b> to <b>1</b> (via inhibitor OCM-47 (<b>47</b>)/49 (<b>49</b>)) could be attributed to both the incorporation of gradually bulky aryl halides substituents (F < Cl < I) and the use of an optimized 5-fluoropyridin-3-yl cation−π interacting moiety (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,D). However, the β-isozyme selectivity occurred specifically with the iodoaryl substitution (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E–G). We confirmed this effect in a similar subseries wherein the cation−π interacting fragment was kept untouched ([PF-618 (<b>28</b>), OCM-45 (<b>45</b>) and OCM-32 (<b>32</b>) <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,D, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Tables S3 and S4</a>]. As for previous compounds of this series, these inhibitors displayed exquisite selectivity in broad kinome coverages (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>I). Inhibitor <b>51</b> demonstrated improved selectivity even compared to previous inhibitors from this series (vide supra), with ≥10 000-fold selectivity against all 371 kinases tested including the DYRK family. Further analysis of the potent inhibitor <b>49</b> as a representative example of this series in Kinativ proteomic screening using mouse brain lysate revealed extensive GSK-3α/β kinase engagement without any detectable off-target interactions (>215 kinases, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>H). Notably, the Kinativ profiling used here included >40 CNS-enriched kinases not initially covered in recombinant enzyme panels. We obtained dissociation constants from MST analysis in line with biochemical data and showed, as previously noted, no quantifiable affinity differences between npS9-GSK-3β and pS9-GSK-3β enzymes (<b>51</b><i>K</i><sub>d</sub> = 0.39 nM and <i>K</i><sub>d</sub> = 0.49 nM for npS9-GSK-3β and pS9-GSK-3β, respectively; <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>L,M and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S3A</a>). Employing a high concentration cutoff of 10 μM, we also found negligible off-target binding for the representative inhibitor <b>51</b> in a CNS-related panel, with only 5 of 80 proteins (including receptors, ion channels and transporters), exhibiting >70% inhibition at this concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>J). Follow-up inhibitory constant measurements revealed >1000–20 000-fold gaps in favor of GSK-3β in all cases providing further proof of selectivity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>K). In addition, all representative compounds tested (<b>28</b>, <b>44</b>, and <b>51</b>) display low hERG inhibition (EC<sub>50</sub> = 1.7–62.6 μM) and correspondingly significant in vitro margins with regard to GSK-3β inhibition (8320-56667-fold, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S2C</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure-guided design and characterization of the GSK-3β isozyme-selective inhibitors <b>51</b>. (A) Structures and properties of selected GSK-3 inhibitors leading to <b>51</b> (for details on chemical synthesis, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Schemes S1–S4</a>). IC<sub>50</sub> values determined using an ADP-Glo kinase assay (<i>n</i> = 2). (B) Dose–response curves of <b>51</b> versus GSK-3α and GSK-3β (<i>n</i> = 3) (compared with <b>28</b>, radiometric filtration binding assay, [ATP] = <i>K</i><sub>m ATP</sub>, data show mean values ± SD). (C) Predicted binding mode of <b>51</b> (green sticks) with GSK-3β (gray cartoon, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK">5KPK</a>). Highlight on the interaction network associated with Asp133. Hydrogen bonds are depicted as a yellow dash lines. (D) Binding affinity measurements of inhibitors versus GSK-3α and GSK-3β (<i>n</i> = 3). (E–G) Alternate views of the predicted contact between <b>51</b> and the selectivity pocket of GSK-3β. (H) Kinativ data from mice brain lysates showing the selective engagement of the representative inhibitor <b>49</b> (1.0 μM) with GSK-3 (<i>n</i> = 2, for detailed data, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S5</a>). (I) Comprehensive kinase selectivity profile of representative inhibitor <b>51</b> against 371 kinases (kinases are ordered alphabetically, and the activity is relative to control at 1.0 μM inhibitor concentration, <i>n</i> = 2, [γ-<sup>33</sup>P]ATP-based enzymatic assay performed by Reaction Biology, for detailed data see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S9</a>). (J) Selectivity profile of <b>51</b> (10.0 μM) against a CNS target panel (<i>n</i> = 2, 80 targets). (K) Binding affinity of <b>51</b> against the five most inhibited CNS targets at a concentration 100 000-fold over the <i>K</i><sub>i</sub> of <b>51</b> for GSK-3β. (L) MST dose–response curve and binding affinity for the interaction between <b>51</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). (M) MST dose–response curve and binding affinity for the interaction between <b>51</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). Data show mean values ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Bioactivities of the Oxazole-4-carboxamide Inhibitors in Cells</h3><div class="NLM_p">We next explored the effect of oxazole-4-carboxamide inhibitors on endogenous cellular GSK-3β signaling. To define the cellular effect on pTau, we used high-content imaging of human cortical-like glutamatergic neurons differentiated from induced pluripotent stem cells through induced expression of Ngn2 transgene.<a onclick="showRef(event, 'ref18 ref38'); return false;" href="javascript:void(0);" class="ref ref18 ref38">(18,38)</a> In agreement with high biochemical potency and cellular target engagement, we found that the two novel representative compounds tested (<b>28</b> and <b>51</b>) caused dose-dependent decreases in pTau levels with efficacy similar to <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–D and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S3C</a>). We employed as the negative control the low potency, trifluoromethoxy-bearing structural analogue OCM-29 (<b>29</b>; IC<sub>50</sub> = 4.1 μM, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S3A</a>) and observed no appreciable decrease in pTau levels even with doses up to 30 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). We further evaluated the oxazole-4-carboxamide inhibitors in another neuronal assay, in which the phosphorylation of collapsin response mediator protein 2 (CRMP2) at Thr-514 (pCRMP2<sup>T514</sup>) is measured using immunostaining, following 24 h treatment with test compounds.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> CRMP2 is a direct protein substrate of GSK-3β, not GSK-3α, and important for diverse neuronal functions.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> Compounds were tested at six doses from 20 μM with 2-fold dilution. Robust dose-dependent effect with EC<sub>50</sub> ranging from 0.5 to 1.5 μM was detected for all active compounds, as well as the CHIR99021 control (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E–I and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S3E,H</a>). In contrast, the low potency, negative control <b>29</b> failed to elicit detectable pCRMP2<sup>T514</sup> reductions even at the highest concentration tested (20 μM, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selective oxazole-4-carboxamide GSK-3 inhibitors modulate both pTau and pCRMP2<sup>T514</sup> and display favorable therapeutic index against Wnt/β-catenin signaling in human neuronal cells. (A–D) Dose-dependent effect of selected compounds on decreasing phosphorylated tau in human neurons. 20 h treatment with <b>1</b>, <b>28</b>, <b>51</b>, but not <b>29</b> dose-dependently decreased mean pTau (PHF-1) intensity in neuronal cell bodies. Each bar is the mean values ± SEM of 14–21 fields/well. Each dose of each compound was tested in 1 well per plate. Data from 3 plates in total are shown in the figure. (E–H) High-content imaging assays using human neurons showed that representative inhibitors (<b>1</b>, <b>28</b>, <b>51</b>) reduced CRMP2 phosphorylation in a dose-dependent manner whereas <b>29</b> did not. CRMP2 is a direct substrate of GSK-3β. The relative level of pCRMP2<sup>T514</sup> intensity was reported as the percentage of the intensity for dimethylsulfoxide (DMSO)-treated (100%) (<i>n</i> = 3 biological replicates, <i>n</i> = 18 images for analysis. Data show mean values ± SEM). Relative levels of dendritic marker MAP2 were not affected by the compound treatments, indicating the health of neurons. (I) Representative images of <b>51</b> treatment on pCRMP2<sup>T514</sup> intensity (green) versus MAP2 intensity (red) illustrate the dose-dependent pCRMP2<sup>T514</sup> intensity reduction. (J–M) Pro-neurogenic potentials measured as NPC proliferation were detected for active oxazole-4-carboxamide GSK-3 inhibitors. Graphs show dose-dependent increases from PF-367, PF-618, and <b>51</b> but not <b>29</b>. NPC proliferation is expressed as the percentage of DMSO-treated (100%). (<i>n</i> = 4. Error bars display SEM. Unpaired <i>t</i>-test. *0.01 ≤ <i>p</i> < 0.05, **0.001 ≤ <i>p</i> < 0.01, ***0.0001 ≤ <i>p</i> < 0.001, ****<i>p</i> < 0.0001.) (N) Active oxazole-4-carboxamide GSK-3 inhibitors enhanced Wnt/β-catenin signaling in human NPCs. Dose–response curves of representative inhibitors (<b>1</b>, <b>28</b>, <b>51,</b> and inactive <b>29</b>) with calculated EC<sub>50</sub>s are shown. <i>n</i> = 4. Data show mean values ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Given the role of GSK-3 in WNT/β-catenin signaling, we also conducted Wnt-signaling gene transcription luciferase reporter assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>N).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Relative to pTau and pCRMP2<sup>T514</sup> inhibition, significantly higher concentrations of active oxazole-4-carboxamide inhibitors were required to elicit gene transcription through the WNT/β-catenin pathway (>10-25-fold selectivity for pCRMP2<sup>T514</sup> and 3-8-fold selectivity for pTau). Although 10 μM CHIR99021 was sufficient to induce 10-fold changes in reporter induction (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S3D</a>), oxazole-4-carboxamide inhibitor treatment only triggered two- to three-fold changes under the same conditions. GSK-3 signaling is also essential for coordinating the proliferation and differentiation of progenitor cells during brain development.<a onclick="showRef(event, 'ref2 ref42'); return false;" href="javascript:void(0);" class="ref ref2 ref42">(2,42)</a> We used human neural progenitor cells (NPCs) and a proliferation assay to evaluate the proneurogenic potential of these GSK-3 inhibitors. In line with previous results, effect on cell proliferation was, however, minor and observed only at micromolar concentrations for all active inhibitors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>K,N and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S3I,L</a>). Together, these data indicate that these oxazole-4-carboxamide inhibitors can access endogenous GSK-3 in human neuronal cells and modulate functions of various GSK-3 complexes with differential efficacies, delineating a potential therapeutic window wherein the inhibition of GSK-3β leads to the modulation of key neuronal molecular targets while sparing putative mechanism-based β-catenin-driven effects.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Oxazole-4-carboxamide Probes Display Increased Specific Binding in Tg2576 Mice Brains in Vitro and Robust BBB Permeation in Vivo</h3><div class="NLM_p">Given the implication of GSK-3α/β in APP processing and AD pathophysiology, we sought to determine in vitro GSK-3α/β-targeted radioligand binding in the brain tissue from transgenic mice with significant amyloid plaque burden. Remarkably, comparison between Tg2576 mice (15 month) and old wild-type animals (21 and 28 months) revealed significant differences in [<sup>3</sup>H]<b>1</b> total and specific binding in the brain tissue (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). As previously, we carried out triple-blocking experiments (10 μM of <b>1</b>, CHIR99021, or LY2090314) to confirm specificity and selectivity (80–91% blocking) in these tissues (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Although we observed a suitable total and specific binding level correlation between old control mice and other normal brain tissue from rats, marmosets, and humans (vide supra), we discovered important increases in total/specific binding in Tg2576 mice compared to 21-month controls. Binding changes were drastic and most pronounced in regions known to exhibit high amyloid plaque deposition such as the cortex (17-fold increase) while binding in regions which present with lower or negligible plaque burden such as the hippocampus (six-fold increase) or the caudate putamen (four-fold increase), respectively, were significantly less affected (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> In addition, regional distribution aligned well with previously characterized ubiquitous tracer binding in the brains of all other species (Ctx ≥ HP ≥ CB ≥ CPu > WM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Oxazole-4-carboxamide-based probes display robust brain exposure properties in vivo and differential target engagement in a mouse model of AD in vitro. (A) In vitro autoradiography images of [<sup>3</sup>H]<b>1</b> Tg2576 and C57BL/6 mice under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (B) Summed (5 to 30 min) PET images showing the distribution of [<sup>11</sup>C]<b>28</b>, [<sup>11</sup>C]<b>37</b>, and [<sup>11</sup>C]<b>44</b> in comparison with [<sup>11</sup>C]<b>1</b> in the rhesus monkey brain, following microdose injections. Image intensity displayed as 0–2 SUV scale. (C) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in Tg2576 (20 μm sections, <i>n</i> = 3) and C57BL/6 (20 μm sections, <i>n</i> = 2) mice brains under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (D) Representative rhesus monkey regional brain time–activity curves (TACs) after the intravenous injection of [<sup>11</sup>C]<b>44</b>. (E) Representative rhesus monkey regional brain time–activity curves (TACs) after intravenous injection of [<sup>11</sup>C]<b>1</b>. (F) Mice exposure of <b>44</b> (C57BL/6 mice, 50 mg/kg, single dose PO, <i>n</i> = 3). Data show mean values ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">With the dual goals of identifying putative GSK-3-targeted therapeutics for CNS diseases and defining structural basis to achieve PET neuroimaging for this target, we coupled the initial SAR component of this study with iterative micro-dosing rodent PET imaging experiments to gain insight into structural factors associated with BBB permeation. As referred to previously (vide supra), this led to the early realization that six-membered nitrogen-containing replacements of the triazole ring found in <b>1</b>, while upholding or improving other properties from our lead, triggered significant increases in brain exposure. Among selected inhibitors ([<sup>11</sup>C]PF-627 (<b>20</b>), [<sup>11</sup>C]<b>28</b>, [<sup>11</sup>C]<b>37</b>, [<sup>11</sup>C]<b>44</b>, and [<sup>11</sup>C]<b>51</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods Scheme S5 and Figures S5–S21), uptake of radioactivity in the rat brain following microdosed peripheral injection (total dose = 0.19–1.75 μg i.v.) was variable but in all cases non-negligible with whole brain SUV<sub>peak</sub> ranging from 0.84 to 3.08 followed by rapid washout (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S4A</a>). We observed the highest BBB permeation with [<sup>11</sup>C]<b>28</b> and [<sup>11</sup>C]<b>44</b> while the lowest radioactivity brain uptake was seen with the most potent inhibitor, [<sup>11</sup>C]<b>51</b>, which may indicate substantial GSK-3 binding in the periphery prior to reaching the CNS. Alternatively, this may be because of a faster clearance from the blood. Consistently, regional brain radioactivity following administration was largely homogeneous as observed in vitro. Next, to measure brain kinetics and determine brain exposure in higher species as a mean to assess the potential of our inhibitors as therapeutics and radioligands, we performed micro-dosed PET imaging experiments in nonhuman primates (one female rhesus macaque, total dose ≤ 1.1 μg i.v.). The selected radiolabeled inhibitors, [<sup>11</sup>C]<b>28</b>, [<sup>11</sup>C]<b>37</b>, and [<sup>11</sup>C]<b>44</b>, displayed in rhesus monkey both significant brain exposure with a SUV<sub>peak</sub> of 3.0–4.0 SUV and favorable brain kinetic for imaging and CNS-targeted therapy following injection (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,D and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">S4B,C</a>). Radioactivity uptake was equally high in all gray matter regions but lower in the white matter. These results are in contrast with the in vivo profile of [<sup>11</sup>C]<b>1</b>, previously reported and retested here, which demonstrate only low brain uptake in rhesus macaques (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B,E).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Finally, we conducted neuropharmacokinetic studies with our lead <b>44</b> to assess brain exposure over time. Conforming with microdosing data, we noted rapid (<i>T</i><sub>max</sub> = 1.0 h) excellent brain exposure (unbound <i>C</i><sub>max</sub> = 941 nM), following single oral administration in mice (50 mg/kg p.o.; <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). The unbound brain-to-plasma ratio (<i>K</i><sub>p,uu</sub>, unbound AUC<sub>last</sub> ratios) was 1.06 and largely consistent between animals, indicating continued distribution equilibrium between brain and blood compartments and no efflux liability (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). No clinical sign following oral administration of <b>44</b> at therapeutic dose in mice or micro-dosing in nonhuman primates for imaging was observed, and both procedures were well tolerated.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13666" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13666" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Modulating molecular targets efficiently and with high accuracy, particularly in the brain, where the BBB severely limits the capacity of peripherally administered small molecules to achieve target engagement, is a significant challenge in CNS drug discovery. To date, the roles of GSK-3α/β in neurodevelopmental and neurodegenerative diseases have been difficult to probe pharmacologically, owing to the complex nature of GSK-3α/β-mediated cellular pathways and the lack of suitable inhibitors. The data presented here describe brain-penetrant selective inhibitors, which potently engage and inhibit GSK-3 in the mammalian brain tissue, including human brains, and modulate GSK-3-regulated neuronal pathways in cells. We also define the structural underpinning for achieving picomolar-selective GSK-3β inhibition as well as β-isozyme-specificity, exploiting discrete topological variations from the deep ATP-binding site of GSK-3 isozymes as previously noted by Wagner et al. 2018.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> While exhibiting GSK-3β selectivity in ranges described for the few previously known GSK-3 isozyme selective inhibitors, the compounds described here do so at over two orders of magnitude higher potencies—which may be crucial in achieving a favorable therapeutic window for this target.<a onclick="showRef(event, 'ref26 ref32'); return false;" href="javascript:void(0);" class="ref ref26 ref32">(26,32)</a> Whether the maintenance of about the same fold-differences in isozyme selectivity observed here with double-digit picomolar versus previously described double-digit nanomolar potency inhibitors represents an intrinsic limitation ensuing from GSK-3α/β binding sites dissimilarities, or isozyme selectivity can be improved further, will require further detailed SAR work based on the results presented here. Previous studies have shown that deletion in embryonic stem cells of GSK-3α or GSK-3β do not individually alter β-catenin/TCF-mediated transcription, and that isoform-selective pharmacological inhibition of GSK-3α in acute myeloid leukemia cells prevents Wnt/β-catenin changes at specific concentrations where dual inhibition lacks such ability—thus providing strong rationale for isozyme-selective inhibition in mitigating putative mechanism-based toxicity driven by β-catenin translocation.<a onclick="showRef(event, 'ref9 ref32'); return false;" href="javascript:void(0);" class="ref ref9 ref32">(9,32)</a> In the current study, the lack of dual inhibitors equipotent to <b>51</b> (or <b>32</b>) has impeded the determination of such effect directly. It is important to note however, that inhibitors <b>51</b> and <b>32</b> only trigger Wnt/β-catenin transcriptional changes at higher concentrations than dual and drastically lower potency controls (<b>1</b> and CHIR99021). This suggests that the GSK-3β-isozyme selective inhibitors identified here may in fact cause similar shifts wherein GSK-3β inhibition is achieved while sparing Wnt/β-catenin. Irrespective of this effect, we note for all inhibitors identified here, a significant and favorable gap between the doses required to trigger Wnt/β-catenin signaling changes and modulate p-CRMP2<sup>T514</sup> and pTau levels.</div><div class="NLM_p last">Kinase-targeted in vivo noninvasive neuroimaging to explore intractable neurodegenerative and psychiatric conditions constitutes an unmet objective despite tremendous progress in the therapeutic and imaging use of kinase inhibitors outside the CNS.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> Our preclinical data demonstrate that radiolabeled oxazole-4-carboxamide inhibitors such as [<sup>11</sup>C]<b>28</b> and [<sup>11</sup>C]<b>44</b> align suitable physicochemical parameters, kinetics, and in vitro/in vivo properties to serve as leads to achieve GSK-3 PET imaging.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Our analysis of GSK-3α/β brain density further supports the view that this target can be imaged using PET. Findings indicating that oxazole-4-carboxamide probes provide total GSK-3 density readouts and display severe binding increases in AD mice compared to wild-type animals also provide a basis for studying the potential of these probes to measure changes in GSK-3 expression in neurodegeneration. A possible caveat pertaining to the characterization of the activation-independent binding mode of the (radio)ligands described here is the report of an alternative p38 MAPK-mediated GSK-3β inactivation phosphorylation at the S389 C terminus residue.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The relevance of this inactivation pathway has, however, only been characterized in mice and may not be relevant in human. Based on the results presented here, detailed PET quantification studies are currently underway. The major finding of this work is the discovery of orally available inhibitors with unique pharmacological profiles showing robust GSK-3 target engagement in the cell and tissue as well as high brain exposure, which warrants future intervention and imaging investigations.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75057" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75057" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemical Synthesis</h3><div class="NLM_p">Compound <b>PF-04802367</b> (≥98%) (hereafter called <b>1</b>), CHIR99021 (≥98%), and LY2090314 (≥98%) were purchased from Millipore Sigma. Alternatively, <b>1</b> was synthesized as described previously.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Detailed synthetic procedures for all inhibitors are provided in Schemes S1–S3 and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods. Briefly, the lead inhibitor (target compound), 5-(3-fluoro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-3-yl)propyl)oxazole-4-carboxamide (<b>44</b>, OCM-44), was synthesized as follows: the title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.137 g, 1.00 mmol) to give <b>44</b> (0.145 g, 48%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 356.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>FN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 356.1410; found, 356.1402. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.55 (s, 1H), 8.49 (t, <i>J</i> = 5.60 Hz, 1H), 8.45 (d, <i>J</i> = 2.00 Hz, 1H), 8.39 (dd, <i>J</i> = 1.40, 4.80 Hz, 1H), 8.35 (dd, <i>J</i> = 2.20, 13.20 Hz, 1H), 7.97 (d, <i>J</i> = 8.40 Hz, 1H), 7.66–7.64 (m, 1H), 7.32–7.28 (m, 2H), 3.91 (s, 3H), 3.34–3.26 (m, 2H), 2.63 (t, <i>J</i> = 7.80 Hz, 2H), 1.84 (quin, <i>J</i> = 7.34 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.09, 160.65, 150.78 (d, <i>J</i> = 241.3 Hz), 149.80, 149.74, 148.92, 148.30 (d, <i>J</i> = 10.6 Hz), 136.38, 128.31, 124.44 (d, <i>J</i> = 3 Hz), 122.67, 121.15, 119.68 (d, <i>J</i> = 8.1 Hz), 115.49 (d, <i>J</i> = 21.5 Hz), 113.64, 56.08, 38.20, 30.38, 29.58. Purity: HPLC <i>t</i><sub>R</sub> = 5.55 min, HPLC purity > 95.4%.</div><div class="NLM_p last">All final compounds were determined to have purity > 95% by HPLC. A X-select CSH C18 (4.6 × 150 mm 3.5 μm) HPLC column was used to determine the chemical purity. The radiosynthesis quality control was performed on a Phenomenex Luna C18, 5 mm, 4.6 × 150 mm column. Liquid chromatography mass spectroscopy utilized a X-select CSH C18 (3.0 × 50mm 2.5 μm) column.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Procedure for Synthesis of Inhibitors <b>6–37</b>, <b>39–42</b>, and <b>44–51</b> (OCM-6–OCM-37, OCM-39–OCM-42, and OCM-44–OCM-51)</h3><div class="NLM_p">To a mixture of an appropriate carboxylic acid (<b>74–84</b>, 1 equiv), the corresponding primary amine (1.2 equiv) and 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (1.2 equiv) in dimethylformamide (DMF; 10 vol) were added triethylamine (5 equiv) in dropwise manner. The reaction mixture was stirred at room temperature for 2 h. The progress of the reaction was monitored by thin-layer chromatography (50% EtOAc in hexane). After completion, the reaction mass was taken into water and extracted with EtOAc, combined organic extracts were dried over anhydrous sodium sulfate, filtered, and concentrated in vacuum to get crude mass. Unless otherwise noted, purification by reverse phase column chromatography using acetonitrile–water as the mobile phase was used to obtain purified final products, which were then triturated with pentane, filtered, and dried under vacuum to afford <b>6–37</b>, <b>39–42</b>, and <b>44–51</b> (OCM-6–OCM-37, OCM-39–OCM-42 and OCM-44–OCM-51).</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(3,4-difluorophenyl)oxazole-4-carboxamide (<b>6</b>, OCM-6)</h4><div class="NLM_p">The title compound was synthesized from 5-(3,4-difluorophenyl)oxazole-4-carboxylic acid (0.15 g, 0.66 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.10 g, 0.80 mmol) to give <b>6</b> (0.12 g, 51%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 334.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>F<sub>2</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 334.1116; found, 334.1112. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.66–8.64 (m, 2H), 8.56–8.50 (m, 2H), 8.04–8.02 (m, 1H), 7.96 (s, 1H), 7.63–7.56 (m, 1H), 4.22 (t, <i>J</i> = 6.80 Hz, 2H), 3.27 (q, <i>J</i> = 6.40 Hz, 2H), 2.04 (quin, <i>J</i> = 6.80 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.62, 151.37, 150.43 (m), 148.96 (m), 148.76 (m), 147.83 (m), 144.05, 129.36, 124.93 (m), 124.3 (dd, <i>J</i> = 11 and 3.48 Hz), 117.83 (d, <i>J</i> = 17.3 Hz), 117.27 (d, <i>J</i> = 20.6 Hz), 46.52, 36.00, 29.39. Purity: HPLC <i>t</i><sub>R</sub> = 7.17 min, HPLC purity > 99.7%.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(perfluorophenyl)oxazole-4-carboxamide (<b>7</b>, OCM-7)</h4><div class="NLM_p">The title compound was synthesized from 5-(perfluorophenyl)oxazole-4-carboxylic acid (0.4 g, 1.43 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.108 g, 0.85 mmol) to give <b>7</b> (0.06 g, 11%) as an pink-colored solid after SFC purification. LCMS (ESI): <i>m</i>/<i>z</i> = 388.4 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>11</sub>F<sub>5</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 388.0833; found, 388.0825. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.14 (s, 1H), 8.05 (s, 1H), 7.96 (s, 1H), 7.34 (br s, 1H), 4.28 (t, <i>J</i> = 6.40 Hz, 2H), 3.47 (q, <i>J</i> = 6.8 Hz, 2H), 2.20 (quin, <i>J</i> = 6.80 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 7.21 min, HPLC purity > 96.5%.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(3,4,5-trifluorophenyl)oxazole-4-carboxamide (<b>8</b>, OCM-8)</h4><div class="NLM_p">The title compound was synthesized from 5-(3,4,5-trifluorophenyl)oxazole-4-carboxylic acid (0.1 g, 0.41 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.06 g, 0.48 mmol) to give <b>8</b> (0.058 g, 41%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 352.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>13</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 352.1021; found, 352.1020. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.19–8.14 (m, 3H), 7.95 (s, 1H), 7.85 (s, 1H), 7.53 (br s, 1H), 4.28 (t, <i>J</i> = 6.40 Hz, 2H), 3.49 (q, <i>J</i> = 6.40 Hz, 2H), 2.21 (quin, <i>J</i> = 6.80 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 160.96, 152.18 (m), 149.75 (m), 148.23 (m), 143.29, 141.96, 139.48 (d, <i>J</i> = 15.7 Hz), 129.65, 122.62 (m), 112.69 (m), 47.06, 36.35, 29.92. Purity: HPLC <i>t</i><sub>R</sub> = 7.55 min, HPLC purity > 99.6%.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(3-fluoro-4-(trifluoromethoxy)phenyl)oxazole-4-carboxamide (<b>9</b>, OCM-9)</h4><div class="NLM_p">The title compound was synthesized from 5-(3-fluoro-4-(trifluoromethoxy)phenyl)oxazole-4-carboxylic acid (0.15 g, 0.51 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.09 g, 0.61 mmol) to give <b>9</b> (0.090 g, 44%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 400.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>F<sub>4</sub>N<sub>5</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 400.1033; found, 400.1026. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.69 (br s, 2H), 8.54 (br s, 2H), 8.11 (br s, 1H), 7.97 (br s, 1H), 7.74 (br s, 1H), 4.23 (br s, 2H), 3.30–3.21 (m, 2H), 2.05 (br s, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 8.05 min, HPLC purity > 98.2%.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(3-fluoro-4-(trifluoromethyl)phenyl)oxazole-4-carboxamide (<b>10</b>, OCM-10)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-(trifluoromethyl)phenyl)oxazole-4-carboxylic acid (0.2 g, 0.73 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.11 g, 0.87 mmol) to give <b>10</b> (0.09 g, 33%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 384.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>F<sub>4</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 384.1084; found, 384.1068. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.37 (d, <i>J</i> = 11.6 Hz, 1H), 8.26 (d, <i>J</i> = 8.0 Hz, 1H), 8.18 (s, 1H), 7.99 (s, 1H), 7.94 (s, 1H), 7.73–7.69 (m, 1H), 7.59 (br s, 1H), 4.32 (t, <i>J</i> = 6.80 Hz, 2H), 3.53 (q, <i>J</i> = 6.40 Hz, 2H), 2.25 (quin, <i>J</i> = 6.40 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 160.85, 158.26, 152.20, 149.96, 148.75 (d, <i>J</i> = 2.9 Hz), 143.30, 132.32 (d, <i>J</i> = 9.3 Hz), 130.68, 127.29, 123.54 (m), 120.92, 119.30 (dd, <i>J</i> = 45.5 and 12.6 Hz), 116.60 (d, <i>J</i> = 24.5 Hz), 47.07, 36.39, 29.93. Purity: HPLC <i>t</i><sub>R</sub> = 7.90 min, HPLC purity > 99.4%.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(3,5-difluoro-4-methoxyphenyl)oxazole-4-carboxamide (<b>11</b>, OCM-11)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,5-difluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.78 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.118 g, 0.93 mmol) to give <b>11</b> (0.09 g, 32%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 386.25 [M + Na]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>16</sub>F<sub>2</sub>N<sub>5</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 364.1221; found, 364.1214. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.65–8.69 (m, 1H), 8.64 (s, 1H), 8.53 (s, 1H), 8.19–8.10 (m, 2H), 7.96 (s, 1H), 4.21 (t, <i>J</i> = 6.80 Hz, 2H), 4.01 (s, 3H), 3.27 (q, <i>J</i> = 6.40 Hz, 2H), 1.98–2.09 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.58, 154.39 (dd, <i>J</i> = 251 and 6.8 Hz), 151.39, 150.67, 150.40, 148.64, 144.07, 129.52, 121.59 (m), 112.08 (m), 61.76, 46.52, 36.03, 29.38. Purity: HPLC <i>t</i><sub>R</sub> = 7.38 min, HPLC purity > 96.6%.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(3-(1<i>H</i>-1,2,4-Triazol-1-yl)propyl)-5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxamide (<b>12</b>, OCM-12)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and 3-(1<i>H</i>-1,2,4-triazol-1-yl)propan-1-amine (0.127 g, 1.00 mmol) to give <b>12</b> (0.09 g, 31%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 346.25 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m/z</i> calcd for C<sub>16</sub>H<sub>17</sub>FN<sub>5</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 346.1315; found, 346.1302. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.59–8.51 (m, 3H), 8.36–8.33 (m, 1H), 7.98 (s, 1H), 7.96 (s, 1H), 7.30 (t, <i>J</i> = 8.80 Hz, 1H), 4.22 (t, <i>J</i> = 6.80 Hz, 2H), 3.91 (s, 3H), 3.22–3.31 (m, 2H), 2.04 (quin, <i>J</i> = 6.80 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.87, 151.39, 150.78 (d, <i>J</i> = 241.3 Hz), 149.99, 149.80, 148.36 (d, <i>J</i> = 10.6 Hz), 144.04, 128.15, 124.51, 119.63 (d, <i>J</i> = 7.6 Hz), 115.53 (d, <i>J</i> = 21.5 Hz), 113.65, 56.13, 46.54, 35.96, 29.48. Purity: HPLC <i>t</i><sub>R</sub> = 6.95 min, HPLC purity > 96.7%.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (<i>Z</i>)-5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-phenylallyl)oxazole-4-carboxamide (<b>13</b>, OCM-13)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and (<i>Z</i>)-3-phenylprop-2-en-1-amine (0.134 g, 1.01 mmol) to get <b>13</b> (0.040 g, 13%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 353.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 353.1301; found, 353.1294. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.21–8.18 (m, 2H), 7.81 (s, 1H), 7.40–7.28 (m, 5H), 7.04 (t, <i>J</i> = 8.8 Hz, 1H), 6.65 (d, <i>J</i> = 11.6 Hz, 1H), 5.83–5.77 (m, 1H), 4.38 (t, <i>J</i> = 6 Hz, 2H), 3.96 (s, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 160.95, 151.84 (d, <i>J</i> = 243.8 Hz), 151.73, 149.12 (d, <i>J</i> = 10.6 Hz), 147.48 (d, <i>J</i> = 4.6 Hz), 136.29, 131.85, 128.76, 128.36, 128.03, 127.75, 127.29, 124.91, 119.90 (d, <i>J</i> = 7.6 Hz), 116.04 (d, <i>J</i> = 22 Hz), 112.83, 56.22, 37.68. Purity: HPLC <i>t</i><sub>R</sub> = 8.99 min, HPLC purity > 97.0%.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i>-Cinnamyl-5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxamide (<b>14</b>, OCM-14)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and (<i>E</i>)-3-phenylprop-2-en-1-amine (0.134 g, 1.01 mmol) to get <b>14</b> (0.065 g, 21%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 353.20 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 353.1301; found, 353.1287. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.22–8.18 (m, 2H), 7.82 (s, 1H), 7.48–7.23 (m, 6H), 7.05 (t, <i>J</i> = 8.4 Hz, 1H), 6.63 (d, <i>J</i> = 16 Hz, 1H), 6.34–6.27 (m, 1H), 4.25 (t, <i>J</i> = 6 Hz, 2H), 3.96 (s, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 160.92, 151.87 (d, <i>J</i> = 243.9 Hz), 151.81, 149.15 (d, <i>J</i> = 10.5 Hz), 147.50 (m), 136.55, 132.26, 128.56, 128.09, 127.70, 126.41, 125.32, 124.97 (d, <i>J</i> = 3.4 Hz), 119.93 (d, <i>J</i> = 7.6 Hz), 116.08 (d, <i>J</i> = 19.5 Hz), 112.86, 56.24, 41.26. Purity: HPLC <i>t</i><sub>R</sub> = 9.31 min, HPLC purity > 99.4%.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-phenethyloxazole-4-carboxamide (<b>15</b>, OCM-15)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.1 g, 0.42 mmol) and (2-phenylethan-1-amine (0.061 g, 0.504 mmol) to get <b>15</b> (0.110 g, 76%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 341.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 341.1301; found, 341.1295.<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.53 (s, 1H), 8.44–8.41 (m, 1H), 8.33 (dd, <i>J</i> = 1.6 & 13.2 Hz, 1H), 7.95 (d, <i>J</i> = 8.8 Hz, 1H), 7.32–7.18 (m, 6H), 3.91 (s, 3H), 3.53–3.48 (m, 2H), 2.84 (t, <i>J</i> = 7.2 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.57, 150.77 (d, <i>J</i> = 241.3 Hz), 149.88, 149.76, 148.33 (d, <i>J</i> = 10.5 Hz), 139.33, 128.59, 128.32, 128.18, 126.08, 124.44, 119.60 (d, <i>J</i> = 8 Hz), 115.47 (d, <i>J</i> = 21.5 Hz), 113.64, 56.09, 35.08. Purity: HPLC <i>t</i><sub>R</sub> = 9.26 min, HPLC purity > 99.6%.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-((1<i>R</i>,2<i>R</i>)-2-phenylcyclopropyl)oxazole-4-carboxamide and 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-((1<i>S</i>,2<i>S</i>)-2-phenylcyclopropyl)oxazole-4-carboxamide (<b>16</b>, OCM-16)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and <i>rac</i>-cis-2-phenylcyclopropan-1-amine (0.134 g, 1.01 mmol) to get <b>16</b> (0.110 g, 36%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 353.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 353.1301; found, 353.1290. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.16–8.13 (m, 2H), 7.65 (s, 1H), 7.35–7.22 (m, 5H), 7.04–6.97 (m, 2H), 3.95 (s, 3H), 3.34–3.28 (m, 1H), 2.47–2.41 (m, 1H), 1.53–1.47 (m, 1H), 1.20–1.16 (m, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 162.39, 151.81 (<i>J</i> = 242 Hz), 151.38, 149.05 (d, <i>J</i> = 11 Hz), 147.33 (d, <i>J</i> = 5 Hz), 136.24, 128.69, 128.36, 127.86, 126.58, 124.80, 119.91 (d, <i>J</i> = 8 Hz), 116.01 (d, <i>J</i> = 20 Hz), 112.79, 56.19, 28.57, 21.85, 12.71. Purity: HPLC <i>t</i><sub>R</sub> = 8.96 min, HPLC purity > 99.8%.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-((1<i>S</i>,2<i>R</i>)-2-phenylcyclopropyl)oxazole-4-carboxamide and 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-((1<i>R</i>,2<i>S</i>)-2-phenylcyclopropyl)oxazole-4-carboxamide (<b>17</b>, OCM-17)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and <i>rac</i>-trans-2-phenylcyclopropan-1-amine (0.134 g, 1 mmol) to get <b>17</b> (0.100 g, 33%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 353.00 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 353.1301; found, 353.1297. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.25–8.17 (m, 2H), 7.81 (s, 1H), 7.47 (s, 1H), 7.33–7.29 (m, 2H), 7.23–7.20 (m, 3H), 7.05 (t, <i>J</i> = 2.2 Hz, 1H), 3.96 (s, 3H), 3.15–3.10 (m, 1H), 2.24–2.19 (m, 1H), 1.40–1.31 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 162.16, 151.86 (d, <i>J</i> = 244.2 Hz), 151.77, 149.18 (d, <i>J</i> = 10.6 Hz), 147.43 (d, <i>J</i> = 4.3 Hz), 140.41, 128.41, 127.91, 126.42, 126.16, 125.01, 119.83 (d, <i>J</i> = 8.1 Hz), 116.02 (d, <i>J</i> = 21.5 Hz), 112.86, 56.23, 31.88, 24.70, 16.38. Purity: HPLC <i>t</i><sub>R</sub> = 9.31 min, HPLC purity > 99.9%.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-phenylpropyl)oxazole-4-carboxamide (<b>18</b>, OCM-18)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) using commercial available amine 3-phenylpropan-1-amine (0.136 g, 1.01 mmol) to give <b>18</b> (0.09 g, 31%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 355.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>FN<sub>2</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 355.1458; found, 355.1453. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.54 (s, 1H), 8.47–8.44 (m, 1H), 8.36–8.32 (m, 1H), 7.96 (d, <i>J</i> = 8.8 Hz, 1H), 7.31–7.25 (m, 3H), 7.20–7.14 (m, 3H), 3.89 (s, 3H), 3.32–3.26 (m, 2H), 2.6 (t, <i>J</i> = 7.20 Hz, 2H), 1.86–1.78 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 161.09, 151.84 (d, <i>J</i> = 243.6 Hz), 151.58, 149.08 (d, <i>J</i> = 10.9 Hz), 147.45, 141.35, 128.44, 128.37, 128.21, 125.96, 124.93 (d, <i>J</i> = 3.4 Hz), 119.98 (d, <i>J</i> = 7.5 Hz), 116.06 (d, <i>J</i> = 21.5 Hz), 112.83, 56.21, 38.87, 33.27, 31.21. Purity: HPLC <i>t</i><sub>R</sub> = 8.83 min, HPLC purity > 99.4%.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(2-methyl-1<i>H</i>-imidazole-1-yl)propyl)oxazole-4-carboxamide (<b>19</b>, PF-981)</h4><div class="NLM_p last">PF-981 was graciously provided by Pfizer (HPLC purity > 95%). Physical state: white solid. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>FN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 359.1519; found, 359.1514. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.14 (dt, <i>J</i> = 2.1, 10.8Hz, 2H), 7.82 (s, 1H), 7.43 (t, <i>J</i> = 5.8 Hz, 1H), 7.12 (d, <i>J</i> = 6.4 Hz, 2H), 7.05 (t, <i>J</i> = 8.5 Hz, 1H), 4.05 (t, <i>J</i> = 7.1 Hz, 2H), 3.95 (s, 3H), 3.51 (m, 2H), 2.62 (s, 3H), 2.12 (t, <i>J</i> = 6.8 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 161.8, 153.1–150.6 (d, <i>J</i> = 245.5 Hz), 152.1 (d, <i>J</i> = 2.7 Hz), 149.3 (d, <i>J</i> = 10.8 Hz), 147.7, 144.1, 127.5, 124.9 (d, <i>J</i> = 3.6 Hz), 122.3, 119.9, 119.5 (d, <i>J</i> = 7.9 Hz), 116.0 (d, <i>J</i> = 21.6 Hz), 112.9, 56.3, 44.5, 36.1, 30.7, 11.7.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(2-oxopyridin-1(2<i>H</i>)-yl)propyl)oxazole-4-carboxamide (<b>20</b>, PF-627)</h4><div class="NLM_p last">PF-627 was graciously provided by Pfizer (HPLC purity > 95%). Physical state: white solid. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>FN<sub>3</sub>O<sub>4</sub><sup>+</sup> (M + H)<sup>+</sup>, 372.1360; found, 372.1367. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.21 (dd, <i>J</i> = 12.8, 2.1 Hz, 1H), 8.12 (d, <i>J</i> = 8.8 Hz, 1H), 8.1 (s, 1H), 7.84 (s, 1H), 7.55 (d, <i>J</i> = 7.6 Hz, 1H), 7.49 (d, <i>J</i> = 7.7 Hz, 1H), 7.09–6.99 (m, 2H), 6.45 (t, <i>J</i> = 6.3 Hz, 1H), 4.17 (t, <i>J</i> = 6.2 Hz, 2H), 3.94 (s, 3H), 3.47 (m, 2H), 2.08 (t, <i>J</i> = 6.1 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 163.1, 161.5, 153.1–150.6 (d, <i>J</i> = 245.0 Hz), 151.5 (d, <i>J</i> = 2.4 Hz), 149.0 (d, <i>J</i> = 10.8 Hz), 147.8, 140.4, 137.7, 128.3, 124.8 (d, <i>J</i> = 3.6 Hz), 120.56, 120.1 (d, <i>J</i> = 7.9 Hz), 116.2 (d, <i>J</i> = 21.7 Hz), 112.8, 107.9, 56.2, 47.3, 35.4, 29.8.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(1-methyl-1<i>H</i>-pyrazol-4-yl)propyl)oxazole-4-carboxamide (<b>21</b>, PF-024)</h4><div class="NLM_p last">PF-024 was graciously provided by Pfizer (HPLC purity > 95%). Physical state: white solid. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>FN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 359.1519; found, 359.1530. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.20–8.13 (m, 2H), 7.79 (s, 1H), 7.40 (s, 1H), 7.31 (br s, 1H), 7.02 (t, <i>J</i> = 8.5 Hz, 1H) 3.94 (s, 3H), 3.91 (s, 3H), 3.46 (m, 2H), 2.57 (t, <i>J</i> = 7.5 Hz, 2H), 1.89 (t, <i>J</i> = 7.3 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 161.1, 153.1–150.6 (d, <i>J</i> = 245.1 Hz), 151.6 (d, <i>J</i> = 2.4 Hz), 149.1 (d, <i>J</i> = 10.8 Hz), 147.5, 137.5, 129.0, 128.2, 124.9 (d, <i>J</i> = 3.6 Hz), 121.0, 119.9 (d, <i>J</i> = 7.9 Hz), 116.1 (d, <i>J</i> = 21.7 Hz), 112.8 (d, <i>J</i> = 2.1 Hz), 56.2, 38.7, 38.6, 30.8, 21.5.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(2-methyl-1<i>H</i>-imidazol-4-yl)propyl)oxazole-4-carboxamide (<b>22</b>, PF-760)</h4><div class="NLM_p last">PF-760 was graciously provided by Pfizer (HPLC purity > 95%). Physical state: White solid. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>FN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 359.1519; found, 359.1514. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.20 (dd, <i>J</i> = 12.9, 2.0 Hz, 1H), 8.01 (d, <i>J</i> = 8.6 Hz, 1H), 7.84 (s, 1H), 7.62 (t, <i>J</i> = 6.3 Hz, 1H), 7.02 (t, <i>J</i> = 8.6 Hz, 1H), 6.87 (s, 1H), 3.94 (s, 3H), 3.50 (m, 2H), 2.71 (s, 3H), 1.98 (t, <i>J</i> = 6.5 Hz, 2H), 1.25 (s, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 162.3, 153.0–150.5 (d, <i>J</i> = 245.1 Hz), 152.0 (d, <i>J</i> = 2.4 Hz), 149.3 (d, <i>J</i> = 10.7 Hz), 147.8, 143.3, 132.2, 127.5, 124.8 (d, <i>J</i> = 3.4 Hz), 119.6 (d, <i>J</i> = 8.0 Hz), 116.3 (d, <i>J</i> = 21.8 Hz), 155.0, 112.9, 56.3, 38.0, 29.0, 21.5, 11.2.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 5-(3-Fluoro-4-(trifluoromethyl)phenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>23</b>, OCM-23)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-(trifluoromethyl)phenyl)oxazole-4-carboxylic acid (0.2 g, 0.72 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.12 g, 0.87 mmol) to give <b>23</b> (0.085 g, 30%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 395.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>F<sub>4</sub>N<sub>4</sub>O<sub>4</sub><sup>+</sup> (M + H)<sup>+</sup>, 395.1131; found, 395.1138. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.52–8.50 (m, 2H), 8.42 (s, 1H), 8.38 (d, <i>J</i> = 12.00 Hz, 1H), 8.25 (d, <i>J</i> = 8.00 Hz, 1H), 7.91 (s, 1H), 7.68 (t, <i>J</i> = 7.60 Hz, 1H), 7.59 (br s, 1H), 3.55 (q, <i>J</i> = 6.80 Hz, 2H), 2.95 (t, <i>J</i> = 7.60 Hz, 2H), 2.13 (quin, <i>J</i> = 7.20 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 160.52, 159.52 (d, <i>J</i> = 253 Hz), 156.61, 149.80, 148.65, 144.61, 144.04, 142.43, 132.46 (d, <i>J</i> = 9.3 Hz), 130.68, 127.19, 123.54 (m), 120.92, 119.30 (dd, <i>J</i> = 45.5 and 12.6 Hz), 116.60 (d, <i>J</i> = 24.4 Hz), 38.83, 32.63, 28.67. Purity: HPLC <i>t</i><sub>R</sub> = 8.52 min, HPLC purity > 95.3%.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(3-(Pyrazin-2-yl)propyl)-5-(3,4,5-trifluorophenyl)oxazole-4-carboxamide (<b>24</b>, OCM-24)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,4,5-trifluorophenyl)oxazole-4-carboxylic acid (0.1 g, 0.41 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.068 g, 0.49 mmol) to give <b>24</b> (0.042 g, 29%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 363.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>14</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 369.1069; found, 369.1075. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.52–8.46 (m, 2H), 8.41 (br s, 1H), 8.20–8.15 (m, 2H), 7.84 (s, 1H), 7.55 (s, 1H), 3.53 (q, <i>J</i> = 6.8 Hz, 2H), 2.92 (t, <i>J</i> = 7.6 Hz, 2H), 2.14–2.07 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 160.66, 156.61, 152.25 (m), 149.74 (m), 148.15 (m), 144.62, 144.04, 142.44, 134.64 (dt, <i>J</i> = 252.5 and 16.0 Hz), 129.94, 122.78 (m), 112.71 (m), 38.81, 32.62, 28.70. Purity: HPLC <i>t</i><sub>R</sub> = 8.20 min, HPLC purity > 99.7%.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 5-(3,4-Difluorophenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>25</b>, OCM-25)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,4-difluorophenyl)oxazole-4-carboxylic acid (0.15 g, 0.66 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.11 g, 0.80 mmol) to give <b>25</b> was obtained (0.1 g, 43%), as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 345.30 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>F<sub>2</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 345.1163; found, 345.1161. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.52–8.50 (m, 2H), 8.42 (d, <i>J</i> = 2.4 Hz, 1H), 8.39–8.34 (m, 1H), 8.16–8.13 (m, 1H), 7.85 (s, 1H), 7.60–7.50 (m, 1H), 7.30–7.20 (m, 1H), 3.58–3.53 (m, 2H), 2.95 (t, <i>J</i> = 7.60 Hz, 2H), 2.13 (quin, <i>J</i> = 7.20 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.46, 156.78, 150.42, 150.20 (m), 148.96 (m), 147.83, 144.46, 143.98 (d, <i>J</i> = 13.1 Hz), 142.28, 129.47, 124.88 (m), 124.3 (dd, <i>J</i> = 11 and 3.48 Hz), 117.83 (d, <i>J</i> = 17.7 Hz), 117.24 (d, <i>J</i> = 20.3 Hz), 38.32, 32.05, 28.46. Purity: HPLC <i>t</i><sub>R</sub> = 7.79 min, HPLC purity > 99.0%.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(3-(Pyrazin-2-yl)propyl)-5-(3,4,5-trifluorophenyl)oxazole-4-carboxamide (<b>26</b>, OCM-26)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,5-difluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.78 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.128 g, 0.93 mmol) to give <b>26</b> (0.070 g, 24%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 375.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>17</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 375.1269; found, 375.1267. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.60 (s, 1H), 8.58–8.56 (m, 1H), 8.54–8.50 (m, 1H), 8.44–8.42 (m, 1H), 8.15–8.08 (m, 2H), 4.00 (s, 3H), 3.36–3.31 (m, 2H), 2.80 (t, <i>J</i> = 7.60 Hz, 2H), 1.98–1,91 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.41, 156.77, 154.38 (dd, <i>J</i> = 244.9 and 6.4 Hz), 150.34, 148.51, 144.45, 143.95, 142.27, 129.62, 121.66 (m), 112.02 (m), 61.78, 38.33, 32.04, 28.43. Purity: HPLC <i>t</i><sub>R</sub> = 7.98 min, HPLC purity > 95.3%.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 5-(3-Fluoro-4-hydroxyphenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>27</b>, PF-796)</h4><div class="NLM_p last">PF-796 was graciously provided by Pfizer (HPLC purity > 95%). Physical state: white solid. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>FN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 343.1206; found, 343.1206. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.74–8.66 (m, 1H), 8.55 (d, <i>J</i> = 2.7 Hz, 1H), 8.23–8.07 (m, 2H), 7.83 (d, <i>J</i> = 6.6 Hz, 1H), 6.99 (t, <i>J</i> = 8.8 Hz, 1H), 3.47 (t, <i>J</i> = 6.8 Hz, 2H), 3.01 (t, <i>J</i> = 7.5 Hz, 2H), 2.11 (t, <i>J</i> = 7.6 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD): δ 162.2, 157.6, 151.8–149.9 (d, <i>J</i> = 239.8 Hz), 151.6 (d, <i>J</i> = 2.8 Hz), 148.7, 146.9, 143.5, 143.3, 140.7, 127.2, 124.4, 118.7 (d, <i>J</i> = 7.7 Hz), 117.2, 115.9 (d, <i>J</i> = 22.1 Hz), 38.2, 31.8, 28.4.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>28</b>, OCM-28)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.138 g, 1.00 mmol) to give <b>28</b> (0.060 g, 20%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 357.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 357.1363; found, 357.1369. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (s, 1H), 8.54 (s, 2H), 8.50 (t, <i>J</i> = 6.00 Hz, 1H), 8.45 (d, <i>J</i> = 2.40 Hz, 1H), 8.35 (dd, <i>J</i> = 2.00, 13.60 Hz, 1H), 7.96 (d, <i>J</i> = 8.80 Hz, 1H), 7.29 (t, <i>J</i> = 8.80 Hz, 1H), 3.91 (s, 3H), 3.35–3.30 (m, 2H), 2.81 (t, <i>J</i> = 7.6 Hz, 2H), 1.95 (quin, <i>J</i> = 7.20 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.69, 156.86, 150.77 (d, <i>J</i> = 241.3 Hz), 149.84, 149.75, 148.31 (d, <i>J</i> = 10.9 Hz), 144.45, 143.96, 142.27, 128.25, 124.45, 119.66 (d, <i>J</i> = 8 Hz), 115.48 (d, <i>J</i> = 21.5 Hz), 113.65, 56.18, 38.26, 32.06, 28.51. Purity: HPLC <i>t</i><sub>R</sub> = 7.53 min, HPLC purity > 99.7%.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 5-(3-Fluoro-4-(trifluoromethoxy)phenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>29</b>, OCM-29)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-(trifluoromethoxy)phenyl)oxazole-4-carboxylic acid (0.15 g, 0.51 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.08 g, 0.61 mmol) to give <b>29</b> (0.11 g, 52%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 411.20 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>F<sub>4</sub>N<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 411.1080; found, 411.1077. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.53–8.51 (m, 2H), 8.43 (d, <i>J</i> = 2.40 Hz, 1H), 8.40 (dd, <i>J</i> = 1.60, 11.60 Hz, 1H), 8.19 (d, <i>J</i> = 8.80 Hz, 1H), 7.89 (s, 1H), 7.58 (br s, 1H), 7.40 (t, <i>J</i> = 8.00 Hz, 1H), 3.56 (q, <i>J</i> = 7.2 Hz, 2H), 2.96 (t, <i>J</i> = 7.60 Hz, 2H), 2.17–2.10 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.35, 156.78 (d, <i>J</i> = 246.8 Hz), 150.88, 148.40, 144.47, 143.98, 142.30, 135.76 (d, <i>J</i> = 14.4 Hz), 130.33, 128.01, 127.92, 124.81 (m), 124.09, 120.15 (d, <i>J</i> = 210 Hz), 117.05 (d, <i>J</i> = 22.3 Hz), 38.35, 32.06, 28.44. Purity: HPLC <i>t</i><sub>R</sub> = 8.66 min, HPLC purity > 98.9%.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 5-(4-Cyano-3-fluorophenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>30</b>, OCM-30)</h4><div class="NLM_p last">The title compound was synthesized from 5-(4-cyano-3-fluorophenyl)oxazole-4-carboxylic acid (0.15 g, 0.65 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.11 g, 0.78 mmol) to give <b>30</b> (0.07 g, 31%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 352.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>FN<sub>5</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 352.1210; found, 352.1206. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.53–8.43 (m, 4H), 8.32 (dd, <i>J</i> = 1.47, 8.31 Hz, 1H), 7.93 (s, 1H), 7.70 (dd, <i>J</i> = 6.85, 8.31 Hz, 1H), 7.63 (br s, 1H), 3.55 (q, <i>J</i> = 6.36 Hz, 2H), 2.94 (t, <i>J</i> = 7.58 Hz, 2H), 2.17–2.10 (m, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 162.9 (d, <i>J</i> = 257 Hz), 160.4, 156.5, 149.0, 148.9 (d, <i>J</i> = 3 Hz), 144.6, 144.1, 142.5, 133.4, 133.26, 131.7, 124.0 (d, <i>J</i> = 4 Hz), 116.0 (d, <i>J</i> = 24 Hz), 113.7, 102.1 (d, <i>J</i> = 15.6 Hz), 38.9, 32.6, 28.6. Purity: HPLC <i>t</i><sub>R</sub> = 7.53 min, HPLC purity > 99.4%.</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 5-(3-Cyano-4-fluorophenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>31</b>, OCM-31)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-cyano-4-fluorophenyl)oxazole-4-carboxylic acid (0.15 g, 0.65 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.11 g, 0.78 mmol)to give <b>31</b> (0.045 g, 20%) as an off-white solid. LCMS (ESI) = 352.05 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>FN<sub>5</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 352.1210; found, 352.1216. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.77–8.73 (m, 1H), 8.65 (dd, <i>J</i> = 2.20, 6.11 Hz, 1H), 8.53–8.48 (m, 2H), 8.42 (d, <i>J</i> = 2.45 Hz, 1H), 7.89 (s, 1H), 7.60 (br s, 1H), 7.32 (t, <i>J</i> = 8.56 Hz, 1H), 3.55 (q, <i>J</i> = 6.85 Hz, 2H), 2.94 (t, <i>J</i> = 7.58 Hz, 2H), 2.17–2.09 (m, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 163.4 (d, <i>J</i> = 262.4 Hz), 160.6, 156.6, 149.5, 148.4 (d, <i>J</i> = 4 Hz), 144.6, 144.0, 142.5, 135.1 (d, <i>J</i> = 9 Hz), 133.4, 129.9, 124.4 (d, <i>J</i> = 3 Hz), 116.7 (d, <i>J</i> = 20 Hz), 113.4, 102.0 (d, <i>J</i> = 16 Hz), 38.8, 32.6, 28.7. Purity: HPLC <i>t</i><sub>R</sub> = 7.45 min, HPLC Purity > 99.7%.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 5-(3-Iodo-4-methoxyphenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>32</b>, OCM-32)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-iodo-4-methoxyphenyl)oxazole-4-carboxylic acid (0.35 g, 1.00 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.16 g, 1.20 mmol) to give <b>32</b> (0.18 g, 38%) as a pale yellow solid. LCMS (ESI) = 465.05 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>IN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 465.0424; found, 465.0410. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.64 (d, <i>J</i> = 1.96 Hz, 1H), 8.53–8.48 (m, 3H), 8.41 (d, <i>J</i> = 2.45 Hz, 1H), 7.80 (s, 1H), 7.48 (br s, 1H), 6.89 (d, <i>J</i> = 8.80 Hz, 1H), 3.94 (s, 3H), 3.54 (q, <i>J</i> = 6.85 Hz, 2H), 2.93 (t, <i>J</i> = 7.58 Hz, 2H), 2.15–2.06 (m, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 160.5, 158.6, 156.1, 150.6, 146.9, 143.9, 143.4, 141.7, 138.2, 129.7, 127.3, 120.8, 109.6, 84.8, 55.8, 38.0, 32.0, 28.2. Purity: HPLC <i>t</i><sub>R</sub> = 8.24 min, HPLC Purity > 98.8%.</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 5-(3-Fluoro-4-(trifluoromethyl)phenyl)-<i>N</i>-(3-(pyridin-2-yl)propyl)oxazole-4-carboxamide (<b>33</b>, OCM-33)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-(trifluoromethyl)phenyl)oxazole-4-carboxylic acid (0.2 g, 0.72 mmol) and 3-(pyridin-2-yl)propan-1-amine (0.12 g, 0.87 mmol) to give <b>33</b> (0.1 g, 35%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 394.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>F<sub>4</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 394.1179; found, 394.1166. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.56 (d, <i>J</i> = 4.40 Hz, 1H), 8.39 (d, <i>J</i> = 12.00 Hz, 1H), 8.26 (d, <i>J</i> = 8.00 Hz, 1H), 7.92 (s, 1H), 7.76 (br s, 1H), 7.71 (t, <i>J</i> = 14.2Hz, 1H), 7.60 (dt, <i>J</i> = 1.60, 7.60 Hz, 1H), 7.19 (d, <i>J</i> = 8.00 Hz, 1H), 7.15–7.12 (m, 1H), 3.53 (q, <i>J</i> = 6.50 Hz, 2H), 2.92 (t, <i>J</i> = 7.40 Hz, 2H), 2.11 (quin, <i>J</i> = 7.10 Hz, 2H). <sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 6.27 min, HPLC purity > 96.5%.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(3-(Pyridin-2-yl)propyl)-5-(3,4,5-trifluorophenyl)oxazole-4-carboxamide (<b>34</b>, OCM-34)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,4,5-trifluorophenyl)oxazole-4-carboxylic acid (0.2 g, 0.82 mmol) and 3-(pyridin-2-yl)propan-1-amine (0.135 g, 1.00 mmol) to give <b>34</b> (0.030 g, 10%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 362.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>F<sub>3</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 362.1116; found, 362.1109. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.56 (d, <i>J</i> = 4.4 Hz, 1H), 8.30–8.15 (m, 2H), 7.86 (s, 1H), 7.73 (br s, 1H), 7.64–7.60 (m, 1H), 7.21–7.19 (m, 1H), 7.15–7.12 (m, 1H), 3.53 (q, <i>J</i> = 6.50, 2H), 2.93 (t, <i>J</i> = 7.6 Hz, 2H), 2.11 (quin, <i>J</i> = 10.6 Hz, 2H) (one CH<sub>2</sub> merged in solvent peak). <sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 5.94 min, HPLC purity > 99.0%.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 5-(3,4-Difluorophenyl)-<i>N</i>-(3-(pyridin-2-yl)propyl)oxazole-4-carboxamide (<b>35</b>, OCM-35)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,4-difluorophenyl)oxazole-4-carboxylic acid (0.12 g, 0.53 mmol) and 3-(pyridin-2-yl)propan-1-amine (0.087 g, 0.64 mmol) to give <b>35</b> (0.09 g, 49%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 344.05 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>F<sub>2</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 344.1211; found, 344.1214. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.56 (d, <i>J</i> = 3.2 Hz, 1H), 8.40–8.34 (m, 1H), 8.17–8.13 (m, 1H), 7.85 (s, 1H), 7.67 (br s, 1H), 7.60 (dt, <i>J</i> = 2.00, 7.80 Hz, 1H), 7.26–7.11 (m, 3H), 3.52 (q, <i>J</i> = 6.50 Hz, 2H), 2.93 (t, <i>J</i> = 7.40 Hz, 2H), 2.18–2.06 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.06, 160.42, 150.77 (m), 150.41, 148.91, 148.29 (m), 136.42, 129.53, 124.92 (m), 124.42 (m), 122.69, 121.17, 117.84 (d, <i>J</i> = 17.7 Hz), 117.24 (d, <i>J</i> = 20.7 Hz), 38.54, 35.01, 28.94. Purity: HPLC <i>t</i><sub>R</sub> = 5.69 min, HPLC purity > 96.6%.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 5-(3,5-Difluoro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-2-yl)propyl)oxazole-4-carboxamide (<b>36</b>, OCM-36)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,5-difluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.140 g, 0.55 mmol) and 3-(pyridin-2-yl)propan-1-amine (0.07 g, 0.55 mmol) to give <b>36</b> (0.060 g, 29%) as an off-white solid after silica gel column purification. LCMS (ESI): <i>m</i>/<i>z</i> = 374.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>2</sub>N<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 374.1316; found, 374.1309. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.55 (d, <i>J</i> = 5.2 Hz, 1H), 8.12–8.02 (m, 2H), 7.82 (s, 1H), 7.66 (br s, 1H), 7.62–7.56 (m, 1H), 7.18 (d, <i>J</i> = 7.60 Hz, 1H), 7.13–7.10 (m, 1H), 4.07 (s, 3H), 3.51 (q, <i>J</i> = 6.50 Hz, 2H), 2.91 (t, <i>J</i> = 7.40 Hz, 2H), 2.14–2.05 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 161.02, 160.80, 155.04 (dd, <i>J</i> = 246 and 6.7 Hz), 150.27, 149.28, 147.84, 137.74 (m), 136.49, 129.44, 122.92, 121.41 (m), 121.19, 112.38 (m), 61.73, 38.94, 35.57, 29.13. Purity: HPLC <i>t</i><sub>R</sub> = 5.81 min, HPLC purity > 99.0%.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-2-yl)propyl)oxazole-4-carboxamide (<b>37</b>, OCM-37)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and 3-(pyridin-2-yl)propan-1-amine (0.37 g, 1.00 mmol) to give <b>37</b> (0.130 g, 44%), as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 356.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>FN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 356.1410; found, 356.1404. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.54 (s, 1H), 8.52–8.46 (m, 2H), 8.35 (dd, <i>J</i> = 2.00, 13.60 Hz, 1H), 7.96 (d, <i>J</i> = 8.40 Hz, 1H), 7.68 (dt, <i>J</i> = 1.60, 7.60 Hz, 1H), 7.32–7.26 (m, 2H), 7.20–7.17 (m, 1H), 3.90 (s, 3H), 3.34–3.29 (m, 2H), 2.78–2.74 (m, 2H), 1.92 (quin, <i>J</i> = 7.40 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.09, 160.65, 150.78 (d, <i>J</i> = 241.3 Hz), 149.80, 149.74, 148.92, 148.30 (d, <i>J</i> = 10.6 Hz), 136.38, 128.31, 124.44 (d, <i>J</i> = 3 Hz), 122.67, 121.15, 119.68 (d, <i>J</i> = 8.1 Hz), 115.49 (d, <i>J</i> = 21.5 Hz), 113.63, 56.10, 38.48, 35.03, 29.01. Purity: HPLC <i>t</i><sub>R</sub> = 5.51 min, HPLC purity > 99.2%.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 5-(3-Fluoro-4-(trifluoromethyl)phenyl)-<i>N</i>-(3-(pyridin-3-yl)propyl)oxazole-4-carboxamide (<b>39</b>, OCM-39)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-(trifluoromethyl)phenyl)oxazole-4-carboxylic acid (0.2 g, 0.72 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.12 g, 0.87 mmol) to give <b>39</b> (0.045 g, 15%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 394.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>F<sub>4</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 394.1179; found, 394.1172. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.60 (br s, 1H), 8.39 (d, <i>J</i> = 11.6 Hz, 1H), 8.26 (d, <i>J</i> = 8 Hz, 1H), 7.92 (s, 1H), 7.70 (t, <i>J</i> = 8.00 Hz, 1H), 7.57 (d, <i>J</i> = 8.00 Hz, 1H), 7.42 (m, 1H), 3.52 (q, <i>J</i> = 6.8 Hz, 2H), 2.76 (t, <i>J</i> = 7.8 Hz, 2H), 2.00 (quin, <i>J</i> = 7.40 Hz, 2H) (<i>1H merged in the solvent peak</i>). <sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 6.34 min, HPLC purity > 97.7%.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-(3-(Pyridin-3-yl)propyl)-5-(3,4,5-trifluorophenyl)oxazole-4-carboxamide (<b>40</b>, OCM-40)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,4,5-trifluorophenyl)oxazole-4-carboxylic acid (0.2 g, 0.82 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.135 g, 0.98 mmol) to give <b>40</b> (0.070 g, 23%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 362.65 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>F<sub>3</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 362.1116; found, 362.1116. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.50 (s, 1H), 8.47 (d, <i>J</i> = 4.40 Hz, 1H), 8.25–8.17 (m, 2H), 7.87 (s, 1H), 7.56 (d, <i>J</i> = 8.00 Hz, 1H), 7.40 (m, 1H), 7.26–7.23 (m, 1H), 3.52 (q, <i>J</i> = 6.90 Hz, 2H), 2.76–2.73 (m, 2H), 2.00 (quin, <i>J</i> = 7.40 Hz, 2H). <sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 6.01 min, HPLC purity > 99.6%.</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 5-(3,4-Difluorophenyl)-<i>N</i>-(3-(pyridin-3-yl)propyl)oxazole-4-carboxamide (<b>41</b>, OCM-41)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,4-difluorophenyl)oxazole-4-carboxylic acid (0.150 g, 0.66 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.11 g, 0.80 mmol) to give <b>41</b> (0.10 g, 43.5%) as an off white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 344.05 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>F<sub>2</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> (M + H)<sup>+</sup>, 344.1211; found, 344.1201. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.49–8.46 (m, 2H), 8.39–8.34 (m, 1H), 8.16–8.13 (m, 1H), 7.85 (s, 1H), 7.54 (d, <i>J</i> = 8.00 Hz, 1H), 7.38 (br s, 1H), 7.30–7.20 (m, 2H), 3.51 (q, <i>J</i> = 6.70 Hz, 2H), 2.74 (t, <i>J</i> = 7.80 Hz, 2H), 2.03–1.95 (m, 2H). <sup>13</sup>C NMR not assigned. Purity: HPLC <i>t</i><sub>R</sub> = 5.72 min, HPLC purity > 97.3%.</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 5-(3,5-Difluoro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-3-yl)propyl)oxazole-4-carboxamide (<b>42</b>, OCM-42)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3,5-difluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.78 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.127 g, 0.93 mmol) to give <b>42</b> (0.1 g, 34.6%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 374.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>2</sub>N<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 374.1316; found, 374.1321. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.61 (s, 1H), 8.60–8.56 (m, 1H), 8.50–8.35 (m, 2H), 8.18–8.06 (m, 2H), 7.64 (d, <i>J</i> = 7.60 Hz, 1H), 7.29 (m, 1H), 4.00 (s, 3H), 3.31–3.26 (m, 2H), 2.62 (t, <i>J</i> = 7.60 Hz, 2H), 1.90–1.80 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.42, 154.39 (dd, <i>J</i> = 244 and 6.7 Hz), 150.37, 149.57, 148.50, 147.10, 136.96 (m), 136.79 (d, <i>J</i> = 13.9 Hz), 135.75, 129.66, 123.45, 121.64 (m), 112.09 (m), 61.75, 38.29, 30.35, 29.60. Purity: HPLC <i>t</i><sub>R</sub> = 5.90 min, HPLC purity > 99.4%.</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 5-(3-Fluoro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-3-yl)propyl)oxazole-4-carboxamide (<b>44</b>, OCM-44)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-fluoro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.84 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.137 g, 1.00 mmol) to give <b>44</b> (0.145 g, 48%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 356.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>FN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 356.1410; found, 356.1402. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.55 (s, 1H), 8.49 (t, <i>J</i> = 5.60 Hz, 1H), 8.45 (d, <i>J</i> = 2.00 Hz, 1H), 8.39 (dd, <i>J</i> = 1.40, 4.80 Hz, 1H), 8.35 (dd, <i>J</i> = 2.20, 13.20 Hz, 1H), 7.97 (d, <i>J</i> = 8.40 Hz, 1H), 7.66–7.64 (m, 1H), 7.32–7.28 (m, 2H), 3.91 (s, 3H), 3.34–3.26 (m, 2H), 2.63 (t, <i>J</i> = 7.80 Hz, 2H), 1.84 (quin, <i>J</i> = 7.34 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.09, 160.65, 150.78 (d, <i>J</i> = 241.3 Hz), 149.80, 149.74, 148.92, 148.30 (d, <i>J</i> = 10.6 Hz), 136.38, 128.31, 124.44 (d, <i>J</i> = 3 Hz), 122.67, 121.15, 119.68 (d, <i>J</i> = 8.1 Hz), 115.49 (d, <i>J</i> = 21.5 Hz), 113.64, 56.08, 38.20, 30.38, 29.58. Purity: HPLC <i>t</i><sub>R</sub> = 5.55 min, HPLC purity > 95.4%.</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 5-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(3-(pyrazin-2-yl)propyl)oxazole-4-carboxamide (<b>45</b>, OCM-45)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.12 g, 0.5 mmol) and 3-(pyrazin-2-yl)propan-1-amine (0.084 g, 0.61 mmol) to give <b>45</b> (0.060 g, 31%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 373.15 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 373.1067; found, 373.1064. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.51–8.50 (m, 2H), 8.41 (d, <i>J</i> = 2.00 Hz, 1H), 8.38 (dd, <i>J</i> = 2.0, 8.80 Hz, 1H), 8.33 (d, <i>J</i> = 2.00 Hz, 1H), 7.81 (s, 1H), 7.48 (br s, 1H), 7.01 (d, <i>J</i> = 8.80 Hz, 1H), 3.96 (s, 3H), 3.54 (q, <i>J</i> = 6.80 Hz, 2H), 2.94 (t, <i>J</i> = 7.60 Hz, 2H), 2.16–2.09 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 161.12, 156.71, 156.16, 151.45, 147.52, 144.60, 144.01, 142.36, 129.82, 128.44, 128.11, 122.45, 120.36, 111.53, 56.21, 38.66, 32.65, 28.84. Purity: HPLC <i>t</i><sub>R</sub> = 7.84 min, HPLC purity > 99.8%.</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 5-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-2-yl)propyl)oxazole-4-carboxamide (<b>46</b>, OCM-46)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.79 mmol) and 3-(pyridin-2-yl)propan-1-amine (0.13 g, 0.95 mmol) to give <b>46</b> (0.090 g, 30%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 372.25 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>ClN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 372.1115; found, 372.1116. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.53–8.45 (m, 4H), 8.10 (dd, <i>J</i> = 2.00, 8.60 Hz, 1H), 7.66 (dt, <i>J</i> = 2.00, 7.60 Hz, 1H), 7.28–7.25 (m, 2H), 7.18–7.15 (m, 1H), 3.91 (s, 3H), 3.33–3.28 (m, 2H), 2.74 (t, <i>J</i> = 7.40 Hz, 2H), 1.91 (quin, <i>J</i> = 7.30 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.10, 160.63, 155.47, 149.83, 149.65, 148.94, 136.40, 129.31, 128.33, 127.91, 122.69, 121.17, 120.93, 120.30, 112.64, 56.31, 38.48, 35.04, 29.05. Purity: HPLC <i>t</i><sub>R</sub> = 5.82 min, HPLC purity > 98.8%.</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 5-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(3-(pyridin-3-yl)propyl)oxazole-4-carboxamide (<b>47</b>, OCM-47)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.79 mmol) and 3-(pyridin-3-yl)propan-1-amine (0.13 g, 0.95 mmol) to give <b>47</b> (0.12 g, 40.5%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 372.20 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>ClN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 372.1115; found, 372.1107. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.55 (s, 1H), 8.53 (d, <i>J</i> = 2.00 Hz, 1H), 8.50–8.45 (m, 2H), 8.39 (d, <i>J</i> = 3.60 Hz, 1H), 8.11 (dd, <i>J</i> = 2.00, 8.80 Hz, 1H), 7.65 (d, <i>J</i> = 7.60 Hz, 1H), 7.31–7.28 (m, 2H), 3.92 (s, 3H), 3.26–3.27 (m, 2H), 2.63 (t, <i>J</i> = 7.80 Hz, 2H), 1.84 (quin, <i>J</i> = 7.30 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 160.67, 155.47, 149.80, 149.67, 149.58, 147.09, 137.03, 135.75, 129.29, 128.30, 127.90, 123.36, 120.93, 120.28, 112.63, 56.30, 38.22, 30.44, 29.61. Purity: HPLC <i>t</i><sub>R</sub> = 5.88 min, HPLC purity > 98.8%.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 5-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(2-(5-fluoro-1<i>H</i>-indol-1-yl)ethyl)oxazole-4-carboxamide (<b>48</b>, OCM-48)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.2 g, 0.79 mmol) and 2-(5-fluoro-1<i>H</i>-indol-1-yl)ethan-1-amine (0.17 g, 0.95 mmol), and the crude residue obtained was purified by the Grace reverse phase system (40 g C-18 cartridge, gradient 0–70% ACN in H<sub>2</sub>O) and lyophilized to give <b>48</b> (0.09 g, 28%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 414.10 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>18</sub>ClFN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 414.1021; found, 414.1009. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.37–8.32 (m, 2H), 7.75 (s, 1H), 7.34–7.28 (m, 2H), 7.15 (d, <i>J</i> = 3.42 Hz, 1H), 7.04 (d, <i>J</i> = 9.29 Hz, 1H), 6.97–6.91 (m, 1H), 6.47 (d, <i>J</i> = 2.93 Hz, 1H), 4.37 (t, <i>J</i> = 6.11 Hz, 2H), 3.98 (s, 3H), 3.81 (q, <i>J</i> = 6.20 Hz, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 161.5, 157.9 (d, <i>J</i> = 234 Hz), 156.3, 151.8, 147.6 (d, <i>J</i> = 3.82 Hz), 132.6, 129.9, 129.5, 128.9 (d, <i>J</i> = 10.60 Hz), 128.4, 127.7, 122.6, 120.2, 111.6, 110.1 (d, <i>J</i> = 26 Hz), 109.9 (d, <i>J</i> = 26 Hz), 105.8 (d, <i>J</i> = 9 Hz), 101.7, 56.3, 46.0, 39.6. Purity: HPLC <i>t</i><sub>R</sub> = 8.99 min, HPLC purity > 98.8%.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 5-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(3-(5-fluoropyridin-3-yl)propyl)oxazole-4-carboxamide (<b>49</b>, OCM-49)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.1 g, 0.39 mmol) and 3-(5-fluoropyridin-3-yl)propan-1-amine (0.07 g, 0.47 mmol), and the crude residue obtained was purified by column chromatography (230–400 mesh silica gel gradient 1–50% EtOAc in <i>n</i>-hexane) to give <b>49</b> (0.077 g, 50%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 390.05 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>ClFN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 390.1021; found, 390.1026. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.37 (dd, <i>J</i> = 2.20, 8.56 Hz, 1H), 8.35–8.28 (m, 3H), 7.80 (s, 1H), 7.37–7.27 (m, 2H), 7.01 (d, <i>J</i> = 8.80 Hz, 1H), 3.96 (s, 3H), 3.51 (q, <i>J</i> = 6.68 Hz, 2H), 2.75 (t, <i>J</i> = 7.83 Hz, 2H), 2.04–1.96 (m, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 161.1, 159.46 (d, <i>J</i> = 255 Hz), 156.2, 151.5, 147.5 (d, <i>J</i> = 4 Hz), 145.6 (d, <i>J</i> = 3 Hz), 138.5 (d, <i>J</i> = 4 Hz), 135.9 (d, <i>J</i> = 23 Hz), 129.8, 128.4, 128.0, 122.5 (d, <i>J</i> = 17 Hz), 122.4, 120.3, 111.5, 56.2, 38.6, 30.8, 29.9. Purity: HPLC <i>t</i><sub>R</sub> = 8.61 min, HPLC purity > 99.3%.</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 5-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(3-(6-fluoropyridin-3-yl)propyl)oxazole-4-carboxamide (<b>50</b>, OCM-50)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.1 g, 0.39 mmol) and 3-(6-fluoropyridin-3-yl)propan-1-amine (0.07 g, 0.47 mmol), and the crude residue obtained was purified by column chromatography (gradient 0–70% EtOAc in <i>n</i>-hexane) to give <b>50</b> (0.05 g, 33%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 390.05 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>ClFN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 390.1021; found, 390.1015. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.38 (dd, <i>J</i> = 1.96, 8.80 Hz, 1H), 8.35 (d, <i>J</i> = 1.96 Hz, 1H), 8.07 (s, 1H), 7.82 (s, 1H), 7.67–7.62 (m, 1H), 7.34 (br s, 1H), 7.02 (d, <i>J</i> = 8.80 Hz, 1H), 6.87 (dd, <i>J</i> = 2.45, 8.31 Hz, 1H), 3.97 (s, 3H), 3.51 (q, <i>J</i> = 6.85 Hz, 2H), 2.73 (t, <i>J</i> = 7.83 Hz, 2H), 2.01–1.93 (m, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 162.5 (d, <i>J</i> = 237.6 Hz), 161.2, 156.3, 151.6, 147.6 (d, <i>J</i> = 4.24 Hz), 147.0 (d, <i>J</i> = 12.29 Hz), 141.0 (d, <i>J</i> = 7.21 Hz), 134.2 (d, <i>J</i> = 4.24 Hz), 129.9, 128.5, 128.0, 122.5, 120.3, 111.6, 109.3 (d, <i>J</i> = 37.1 Hz), 56.2 (d, <i>J</i> = 3.39 Hz), 38.6, 31.1, 29.5. Purity: HPLC <i>t</i><sub>R</sub> = 8.91 min, HPLC purity > 99.8%.</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(3-(5-Fluoropyridin-3-yl)propyl)-5-(3-iodo-4-methoxyphenyl)oxazole-4-carboxamide (<b>51</b>, OCM-51)</h4><div class="NLM_p last">The title compound was synthesized from 5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxylic acid (0.10 g, 0.29 mmol), and the crude residue obtained was purified by column chromatography (gradient 0–70% EtOAc in <i>n</i>-hexane) to give <b>51</b> (0.041 g, 29%) as an off-white solid. LCMS (ESI): <i>m</i>/<i>z</i> = 482.00 [M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>FIN<sub>3</sub>O<sub>3</sub><sup>+</sup> (M + H)<sup>+</sup>, 482.0377; found, 482.0370. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.64 (d, <i>J</i> = 1.96 Hz, 1H), 8.50 (dd, <i>J</i> = 1.96, 8.80 Hz, 1H), 8.33–8.29 (m, 2H), 7.80 (s, 1H), 7.33 (br s, 1H), 7.28 (br s, 1H), 6.90 (d, <i>J</i> = 8.80 Hz, 1H), 3.94 (s, 3H), 3.51 (q, <i>J</i> = 6.85 Hz, 2H), 2.75 (t, <i>J</i> = 7.83 Hz, 2H), 2.02–1.93 (m, 2H). <sup>13</sup>C NMR (100 MHz, chloroform-<i>d</i>): δ 160.7, 159.5 (d, <i>J</i> = 255.0 Hz), 159.3, 150.8, 147.1 (d, <i>J</i> = 4.66 Hz), 145.2, 138.4, 138.1 (d, <i>J</i> = 3.39 Hz), 135.8 (d, <i>J</i> = 23.2 Hz), 129.9, 127.4, 122.5 (d, <i>J</i> = 17.70 Hz), 120.9, 109.8, 85.1, 56.0, 38.2, 30.4, 29.5. Purity: HPLC <i>t</i><sub>R</sub> = 8.94 min, HPLC purity > 99.8%.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Radioligand Saturation Binding Assay</h3><div class="NLM_p last">[<sup>3</sup>H]<b>1</b> (81 Ci/mmol (3.0 TBq/mmol) molar activity, 99.9% radiochemical purity) was provided by MedChem Imaging, Inc. For additional details, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> HEK-293T P301S hTau cellular Assay</h3><div class="NLM_p last">HEK-293T P301S hTau cellular assay was performed as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> In Vitro Autoradiography</h3><div class="NLM_p last">For autoradiography experiments, fresh-frozen postmortem human brain tissue blocks were acquired from Folio-Conversant and the University of California at San Francisco NDBB. In vitro autoradiography studies were performed as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods. Phosphorimaging of tissue sections was performed using Amersham Typhoon (GE Healthcare, MA).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Docking Studies</h3><div class="NLM_p last">Molecular docking simulations of <b>44</b> and <b>51</b> were performed using the X-ray cocrystal structure of GSK-3β complexed with <b>1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>) and GSK-3β complexed with BRD0209 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK">5KPK</a>) using the FITTED 3.5 program (FORECASTER platform).<a onclick="showRef(event, 'ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51">(49−51)</a> Docking structures and figures were prepared using Pymol.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Kinase Assays</h3><div class="NLM_p last">ADPGlo binding studies were performed as previously described.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In vitro radiometric kinase assay including <i>K</i><sub>i</sub> determination and kinome selectivity profiles for selected inhibitors was obtained through [γ-<sup>33</sup>P]ATP-based enzymatic assays (Reaction Biology Corporation, Malvern, PA).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> For additional details, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods. Detailed results for kinome selectivity can also be found in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Tables S5–S7</a>. Chemical proteomic (KiNativ) profiling using brain lysate was performed by ActivX Bioscience (La Jolla, CA). Detailed results for the kinativ profiling can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S8</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> CNS Target-Binding Assays</h3><div class="NLM_p last">Inhibitor <b>51</b> was tested against a panel of 80 binding assays encompassing common targets found in the CNS (LeadProfilingScreen2 Panel, Eurofins Scientific). Detailed results for the CNS target-binding assay profiling are found in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S9</a>. The IC<sub>50</sub>s for the five main off-target of <b>51</b> (5-HT<sub>2B</sub>, BZD, Cl<sup>–</sup> channel (GABA-gated), KOP, NK1) were determined using radioligand-binding assays (Eurofins Scientific). <i>K</i><sub>i</sub> values were calculated using the Cheng–Prusoff equation.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> MST Assay</h3><div class="NLM_p last">The MST assay was performed at the Center for Macromolecule Interaction, Harvard University. For additional details, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> hERG Assay</h3><div class="NLM_p last">hERG assays were performed at Reaction Biology Corporation (Malvern, PA) as previously described.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> WNT Reporter Assay</h3><div class="NLM_p last">The WNT reporter assay was performed as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods and as previously described.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> NPC Proliferation/Viability Assay</h3><div class="NLM_p last">The NPC proliferation/viability assay was performed as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> CRMP2 and pTau Assays</h3><div class="NLM_p last">The CRMP2 and pTau assays were performed as described in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods and as previously described.<a onclick="showRef(event, 'ref38 ref40'); return false;" href="javascript:void(0);" class="ref ref38 ref40">(38,40)</a></div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Western Blot Analysis</h3><div class="NLM_p last">The Western blot analysis was performed as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Radiochemical Syntheses (<b>44</b>)</h3><div class="NLM_p last">Detailed synthetic procedures for all radiotracers are provided in Scheme S4 and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods. Briefly, for the radiosynthesis of [<sup>11</sup>C]<b>44</b>, [<sup>11</sup>C]CH<sub>3</sub>OTf (obtained from ∼1000 mCi of [<sup>11</sup>C]CH<sub>3</sub>I) was bubbled through a reaction solution containing 1.0 mg of the corresponding desmethyl precursor, 5.0 μL of 1.0 M methanolic TBAOH, and 100 μL of DMF at room temperature for 3 min at 15 mL min<sup>–1</sup> with Ar carrier gas. The reaction was quenched with 1000 μL 50/50 MeCN/10mM NH<sub>4</sub>OAc (pH: 7.3) and loaded onto HPLC (50/50 MeCN/20 mM NH<sub>4</sub>OAc (pH: 7.3) (v/v) isocratic; flow rate: 5.0 mL/min; semipreparative column: Phenomenex Luna C18, 10 μm, 10 × 250 mm, 5 mL/min). The peak between 7.5 and 8.1 min corresponding to [<sup>11</sup>C]<b>44</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figure S16</a>) was collected and diluted with 50 mL H<sub>2</sub>O, and the solution was transferred through preconditioned C18 1cc Vac Sep Pak. The radiotracer was eluted with 500 μL EtOH. The C18 cartridge was then washed with 4.5 mL buffered saline. Then, the product was passed through a sterile filter and into the product vial. Pure [<sup>11</sup>C]<b>44</b> was obtained in ∼1% nondecay-corrected radiochemical yields at end-of-synthesis based upon [<sup>11</sup>C]CH<sub>3</sub>I, radiochemical purity > 99% determined by radio-HPLC, <i>A</i><sub>m</sub> = 264 GBq/μmol (NHP experiment), and <i>A</i><sub>m</sub> = 426 GBq/μmol (rodent experiment). Quality control mobile phase: 50/50 MeCN/10mM NH<sub>4</sub>OAc (pH: 7.3) (v/v) isocratic; flow rate: 1.0 mL/min; QC column: Phenomenex Luna C18, 5 μm, 4.6 × 150 mm (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Figures S17 and S18</a>).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Rodents and Nonhuman Primate PET Imaging Studies (<b>1</b>, <b>28</b>, <b>37</b>, <b>44</b>)</h3><div class="NLM_p last">All imaging studies were performed in accordance with the standards set by the University Committee on Use and Care of Animals (UCUCA) at the University of Michigan. Rat imaging was done using Sprague Dawley (body weight ≈ 297–339 g). NHP imaging was done using a mature female rhesus monkey (<i>Macaca mulatta</i>) (body weight ≈ 8.8 kg with negligible variation throughout the duration of the study, 15 years of age). Imaging procedure and analysis is described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Neuropharmacokinetic Analysis</h3><div class="NLM_p last">NeuroPK was performed at Sai Life Sciences as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Supporting Information</a> Materials and Methods.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Statistical Analysis</h3><div class="NLM_p last">Data were expressed as mean ± SD. Statistical analysis was performed using GraphPad Prism 7.0c. <i>p</i> values less than 0.05 were considered statistically significant.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01030" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60240" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60240" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01030" class="ext-link">10.1021/acs.jmedchem.9b01030</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures illustrating chemistry and radiochemistry methods, HEK-293T P301S hTau cellular assay, in vitro autoradiography, kinase assays, MST assays, culturing and neural differentiation of human iPSC-derived NPCs, WNT signaling reporter assays, NPC proliferation assay with CellTiter-Glo, high-content neuronal assay for p-CRMP2<sup>T514</sup>, pTau assay, western blot analysis, rodents and nonhuman primate PET imaging studied, neuropharmacokinetic analysis, and NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_002.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf">jm9b01030_si_001.pdf (14.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_002.xlsx">jm9b01030_si_002.xlsx (13.72 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Existing PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK">5KPK</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a> were used for in silico docking studies.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01030" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vadim Bernard-Gauthier</span> - <span class="hlFld-Affiliation affiliation">Azrieli
Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Psychiatry/Institute of Medical Science, University of Toronto, Toronto, Ontario M5T 1R8, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital
and Department of Radiology, Harvard Medical
School, Boston, Massachusetts 02114, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#671106030e0a05000612130f0e021527000a060e0b4904080a"><span class="__cf_email__" data-cfemail="0e786f6a67636c696f7b7a66676b7c4e69636f6762206d6163">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Neil Vasdev</span> - <span class="hlFld-Affiliation affiliation">Azrieli
Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Psychiatry/Institute of Medical Science, University of Toronto, Toronto, Ontario M5T 1R8, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital
and Department of Radiology, Harvard Medical
School, Boston, Massachusetts 02114, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2087-5125" title="Orcid link">http://orcid.org/0000-0002-2087-5125</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#254b404c490b5344564140536550514a574a4b514a0b4644"><span class="__cf_email__" data-cfemail="dbb5beb2b7f5adbaa8bfbead9baeafb4a9b4b5afb4f5b8ba">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew V. Mossine</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ashley Knight</span> - <span class="hlFld-Affiliation affiliation">Azrieli
Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Psychiatry/Institute of Medical Science, University of Toronto, Toronto, Ontario M5T 1R8, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Eisai
AiM Institute, Boston, Massachusetts 01810, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2225-360X" title="Orcid link">http://orcid.org/0000-0003-2225-360X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Debasis Patnaik</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen-Ning Zhao</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5824-2032" title="Orcid link">http://orcid.org/0000-0002-5824-2032</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chialin Cheng</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hema S. Krishnan</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital
and Department of Radiology, Harvard Medical
School, Boston, Massachusetts 02114, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lucius L. Xuan</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter S. Chindavong</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Surya A. Reis</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinshan Michael Chen</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, Groton
Laboratories, Eastern
Point Road, Groton, Connecticut 06340, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xia Shao</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jenelle Stauff</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janna Arteaga</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Phillip Sherman</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Salem</span> - <span class="hlFld-Affiliation affiliation">Biogen,
Research and Early Development Imaging, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Bonsall</span> - <span class="hlFld-Affiliation affiliation">Invicro, London W12 0NN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brenda Amaral</span> - <span class="hlFld-Affiliation affiliation">Biogen,
Research and Early Development Imaging, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cassis Varlow</span> - <span class="hlFld-Affiliation affiliation">Azrieli
Centre for Neuro-Radiochemistry, Research Imaging Centre, Centre for Addiction and Mental Health, Toronto, Ontario M5T 1R8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa Wells</span> - <span class="hlFld-Affiliation affiliation">Invicro, London W12 0NN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurent Martarello</span> - <span class="hlFld-Affiliation affiliation">Biogen,
Research and Early Development Imaging, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shil Patel</span> - <span class="hlFld-Affiliation affiliation">Eisai
AiM Institute, Boston, Massachusetts 01810, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven H. Liang</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine and Molecular Imaging, Massachusetts General Hospital
and Department of Radiology, Harvard Medical
School, Boston, Massachusetts 02114, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1413-6315" title="Orcid link">http://orcid.org/0000-0003-1413-6315</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ravi G. Kurumbail</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, Groton
Laboratories, Eastern
Point Road, Groton, Connecticut 06340, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen J. Haggarty</span> - <span class="hlFld-Affiliation affiliation">Chemical
Neurobiology Laboratory, Massachusetts General Hospital, Center for
Genomic Medicine, Departments of Neurology & Psychiatry, Harvard Medical School, Boston, Massachusetts 02114, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7872-168X" title="Orcid link">http://orcid.org/0000-0002-7872-168X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. H. Scott</span> - <span class="hlFld-Affiliation affiliation">Division
of Nuclear Medicine, Department of Radiology, The University of Michigan Medical School, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="hlFld-Affiliation affiliation">The
Interdepartmental Program in Medicinal Chemistry, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>V.B.-G. and A.V.M. contributed equally to this work. V.B.-G., N.V., A.K., D.P., W.-N.Z., P.S.C., J.M.C., N.S., L.W., L.M., R.G.K., and S.J.H. designed research; V.B.-G., A.V.M., N.V., A.K., D.P., W.-N.Z., C.C., L.L.X, P.S.C., S.A.R., J.M.C., X.S., J.S., J.A., P.S., H.S.K., S.H.L., S.P.; N.S., L.W., and L.M. performed research; V.B.-G., N.V., N.S., L.M., R.G.K., S.J.H., and P.J.H.S. contributed new reagents/analytical tools; V.B.-G., A.V.M., N.V., A.K., D.P., W.-N.Z., C.C., P.S.C., J.M.C., S.P., N.S., C.V., L.W., L.M., R.G.K., and S.J.H. analyzed data; V.B.-G. wrote the paper.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Kelly L. Arnett and Harvard University’s Center for Macromolecular Interaction for advice regarding MST and Dr. Poguang Wang for HRMS analyses. Funding to S.J.H. provided by the Tau Consortium and MGH Research Scholars Program. We thank Smita Jacob for graphic design. N.V. thanks National Institute on Ageing of the NIH (R01AG054473), the Azrieli Foundation and the Canada Research Chairs Program for support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AD</td><td class="NLM_def"><p class="first last">Alzheimer’s disease</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption</p></td></tr><tr><td class="NLM_term">distribution</td><td class="NLM_def"><p class="first last">metabolism and elimination</p></td></tr><tr><td class="NLM_term">ADP</td><td class="NLM_def"><p class="first last">adenosine 5′-diphosphate</p></td></tr><tr><td class="NLM_term">AKT</td><td class="NLM_def"><p class="first last">protein kinase B</p></td></tr><tr><td class="NLM_term"><i>A</i><sub>m</sub></td><td class="NLM_def"><p class="first last">molar activity</p></td></tr><tr><td class="NLM_term">APP</td><td class="NLM_def"><p class="first last">amyloid precursor protein</p></td></tr><tr><td class="NLM_term">Ar</td><td class="NLM_def"><p class="first last">argon</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine 5′-triphosphate</p></td></tr><tr><td class="NLM_term">AUClast</td><td class="NLM_def"><p class="first last">area under time–concentration curve to last measurable concentration</p></td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">blood–brain barrier</p></td></tr><tr><td class="NLM_term"><i>B</i><sub>max</sub></td><td class="NLM_def"><p class="first last">binding site density</p></td></tr><tr><td class="NLM_term">BZD</td><td class="NLM_def"><p class="first last">benzodiazepine</p></td></tr><tr><td class="NLM_term">CB</td><td class="NLM_def"><p class="first last">cerebellum</p></td></tr><tr><td class="NLM_term">CC</td><td class="NLM_def"><p class="first last">corpus callosum</p></td></tr><tr><td class="NLM_term">Cl</td><td class="NLM_def"><p class="first last">chloro</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum serum concentration</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">CPu</td><td class="NLM_def"><p class="first last">caudate putamen</p></td></tr><tr><td class="NLM_term">CRMP</td><td class="NLM_def"><p class="first last">collapsing response mediator protein</p></td></tr><tr><td class="NLM_term">Ctx</td><td class="NLM_def"><p class="first last">cortex</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">Fu_p</td><td class="NLM_def"><p class="first last">unbound brain fraction</p></td></tr><tr><td class="NLM_term">GABA</td><td class="NLM_def"><p class="first last">γ-aminobutyric acid</p></td></tr><tr><td class="NLM_term">GBq</td><td class="NLM_def"><p class="first last">gigabecquerel</p></td></tr><tr><td class="NLM_term">GM</td><td class="NLM_def"><p class="first last">grey matter</p></td></tr><tr><td class="NLM_term">GSK-3</td><td class="NLM_def"><p class="first last">glycogen synthase kinase-3</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go related gene</p></td></tr><tr><td class="NLM_term">HP</td><td class="NLM_def"><p class="first last">hippocampus</p></td></tr><tr><td class="NLM_term">HT</td><td class="NLM_def"><p class="first last">hydroxytryptamine</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximum inhibitory concentration</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>i</sub></td><td class="NLM_def"><p class="first last">inhibition constant</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>M</sub></td><td class="NLM_def"><p class="first last">Michaelis constant</p></td></tr><tr><td class="NLM_term">KOP</td><td class="NLM_def"><p class="first last">kappa opioid receptor</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>p,uu</sub></td><td class="NLM_def"><p class="first last">unbound partition coefficient</p></td></tr><tr><td class="NLM_term">LCMS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">mCi</td><td class="NLM_def"><p class="first last">millicuries</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MPO</td><td class="NLM_def"><p class="first last">multiparameter optimization</p></td></tr><tr><td class="NLM_term">MST</td><td class="NLM_def"><p class="first last">microscale thermophoresis</p></td></tr><tr><td class="NLM_term">NH<sub>4</sub>OAc</td><td class="NLM_def"><p class="first last">ammonium acetate</p></td></tr><tr><td class="NLM_term">NHP</td><td class="NLM_def"><p class="first last">nonhuman primate</p></td></tr><tr><td class="NLM_term">NIH</td><td class="NLM_def"><p class="first last">National Institute of Health</p></td></tr><tr><td class="NLM_term">NK1</td><td class="NLM_def"><p class="first last">neurokinin 1</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">NPC</td><td class="NLM_def"><p class="first last">neural progenitor cell</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PET</td><td class="NLM_def"><p class="first last">positron emission tomography</p></td></tr><tr><td class="NLM_term">PF-367</td><td class="NLM_def"><p class="first last">PF-04802367</p></td></tr><tr><td class="NLM_term">pGS</td><td class="NLM_def"><p class="first last">phosphorylated glycogen synthase</p></td></tr><tr><td class="NLM_term">PO</td><td class="NLM_def"><p class="first last">oral gavage</p></td></tr><tr><td class="NLM_term">pTau</td><td class="NLM_def"><p class="first last">hyperphosphorylated tau protein</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">SUV</td><td class="NLM_def"><p class="first last">standard uptake value</p></td></tr><tr><td class="NLM_term">TAC</td><td class="NLM_def"><p class="first last">time activity curves</p></td></tr><tr><td class="NLM_term">TBAOH</td><td class="NLM_def"><p class="first last">tetrabutylammonium hydroxide</p></td></tr><tr><td class="NLM_term">Tg</td><td class="NLM_def"><p class="first last">transgenic</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>max</sub></td><td class="NLM_def"><p class="first last">time that maximum serum concentration is observed</p></td></tr><tr><td class="NLM_term">UCUCA</td><td class="NLM_def"><p class="first last">University Committee on Use and Care of Animals</p></td></tr><tr><td class="NLM_term">WM</td><td class="NLM_def"><p class="first last">white matter</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48305" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48305" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frame, S.</span></span> <span> </span><span class="NLM_article-title">The renaissance of GSK3</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">776</span>, <span class="refDoi"> DOI: 10.1038/35096075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2F35096075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=11584304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvVajsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=769-776&author=P.+Cohenauthor=S.+Frame&title=The+renaissance+of+GSK3&doi=10.1038%2F35096075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: The renaissance of GSK3</span></div><div class="casAuthors">Cohen, Philip; Frame, Sheelagh</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">769-776</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 67 refs.  Glycogen synthase kinase 3 (GSK3) was initially described as a key enzyme involved in glycogen metab., but is now known to regulate a diverse array of cell functions.  The study of the substrate specificity and regulation of GSK3 activity has been important in the quest for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_fen3f-nOX7Vg90H21EOLACvtfcHk0lht6k1yNeYE_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvVajsLc%253D&md5=00db2e5c9abcece9317bee545c1a1eb9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F35096075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35096075%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DFrame%26aufirst%3DS.%26atitle%3DThe%2520renaissance%2520of%2520GSK3%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2001%26volume%3D2%26spage%3D769%26epage%3D776%26doi%3D10.1038%2F35096075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hur, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.-Q.</span></span> <span> </span><span class="NLM_article-title">GSK3 signalling in neural development</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nrn2870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrn2870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVGisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=539-551&author=E.-M.+Hurauthor=F.-Q.+Zhou&title=GSK3+signalling+in+neural+development&doi=10.1038%2Fnrn2870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">GSK3 signaling in neural development</span></div><div class="casAuthors">Hur, Eun-Mi; Zhou, Feng-Quan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">539-551</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent evidence suggests that glycogen synthase kinase-3 (GSK3) proteins and their upstream and downstream regulators have key roles in many fundamental processes during neurodevelopment.  Disruption of GSK3 signaling adversely affects brain development and is assocd. with several neurodevelopmental disorders.  Here, the authors discuss the mechanisms by which GSK3 activity is regulated in the nervous system and provide an overview of the recent advances in the understanding of how GSK3 signaling controls neurogenesis, neuronal polarization, and axon growth during brain development.  These recent advances suggest that GSK3 is a crucial node that mediates various cellular processes that are controlled by multiple signaling mols., e.g., disrupted in schizophrenia 1 (DISC1), partitioning defective homolog 3 (PAR3), PAR6, and Wnt proteins, that regulate neurodevelopment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiOzmiQawg1rVg90H21EOLACvtfcHk0lgkZuY6oMq0rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptVGisro%253D&md5=7ad4651b8fd5a97028df1a57be2c4b38</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrn2870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn2870%26sid%3Dliteratum%253Aachs%26aulast%3DHur%26aufirst%3DE.-M.%26aulast%3DZhou%26aufirst%3DF.-Q.%26atitle%3DGSK3%2520signalling%2520in%2520neural%2520development%26jtitle%3DNat.%2520Neurosci.%26date%3D2010%26volume%3D11%26spage%3D539%26epage%3D551%26doi%3D10.1038%2Fnrn2870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, C.</span></span> <span> </span><span class="NLM_article-title">What Are thebona fideGSK3 Substrates?</span>. <i>Int. J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2011</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.4061/2011/505607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.4061%2F2011%2F505607" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2011&publication_year=2011&pages=1-23&author=C.+Sutherland&title=What+Are+thebona+fideGSK3+Substrates%3F&doi=10.4061%2F2011%2F505607"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4061%2F2011%2F505607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4061%252F2011%252F505607%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DC.%26atitle%3DWhat%2520Are%2520thebona%2520fideGSK3%2520Substrates%253F%26jtitle%3DInt.%2520J.%2520Alzheimers%2520Dis.%26date%3D2011%26volume%3D2011%26spage%3D1%26epage%3D23%26doi%3D10.4061%2F2011%2F505607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cross, D. A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andjelkovich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmings, B. A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of glycogen synthase kinase-3 by insulin mediated by protein kinase B</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1038/378785a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2F378785a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=8524413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK28XltFWj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=1995&pages=785-789&author=D.+A.+E.+Crossauthor=D.+R.+Alessiauthor=P.+Cohenauthor=M.+Andjelkovichauthor=B.+A.+Hemmings&title=Inhibition+of+glycogen+synthase+kinase-3+by+insulin+mediated+by+protein+kinase+B&doi=10.1038%2F378785a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of glycogen synthase kinase-3 by insulin mediated by protein kinase B</span></div><div class="casAuthors">Cross, Darren A. E.; Alessi, Dario R.; Cohen, Philip; Andjelkovich, Mirjana; Hemmings, Brian A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">6559</span>),
    <span class="NLM_cas:pages">785-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Agents which prevent the activation of both MAP kinase-activated protein kinase-1 and p70S6k by insulin in vivo do not block the phosphorylation and inhibition of glycogen synthase kinase-3 (GSK3) in L6 myotubes.  Another insulin-stimulated protein kinase inactivates GSK3 under these conditions, and the authors demonstrate that it is the product of the proto-oncogene protein kinase B (PKB, also known as Akt/RAC).  Like the inhibition of GSK3, the activation of PKB is prevented by inhibitors of phosphatidylinositol 3-kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKh6GNVgGhFbVg90H21EOLACvtfcHk0lgkZuY6oMq0rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltFWj&md5=56723cab1f5412b4e55aca875989d1fa</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F378785a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F378785a0%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DD.%2BA.%2BE.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DAndjelkovich%26aufirst%3DM.%26aulast%3DHemmings%26aufirst%3DB.%2BA.%26atitle%3DInhibition%2520of%2520glycogen%2520synthase%2520kinase-3%2520by%2520insulin%2520mediated%2520by%2520protein%2520kinase%2520B%26jtitle%3DNature%26date%3D1995%26volume%3D378%26spage%3D785%26epage%3D789%26doi%3D10.1038%2F378785a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eldar-Finkelman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenheede, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, E. G.</span></span> <span> </span><span class="NLM_article-title">Inactivation of glycogen synthase kinase-3 by epidermal growth factor is mediated by mitogen-activated protein kinase/p90 ribosomal protein S6 kinase signaling pathway in NIH/3T3 cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">987</span>– <span class="NLM_lpage">990</span>, <span class="refDoi"> DOI: 10.1074/jbc.270.3.987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1074%2Fjbc.270.3.987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=7836418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK2MXjtFOksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=987-990&author=H.+Eldar-Finkelmanauthor=R.+Segerauthor=J.+R.+Vandenheedeauthor=E.+G.+Krebs&title=Inactivation+of+glycogen+synthase+kinase-3+by+epidermal+growth+factor+is+mediated+by+mitogen-activated+protein+kinase%2Fp90+ribosomal+protein+S6+kinase+signaling+pathway+in+NIH%2F3T3+cells&doi=10.1074%2Fjbc.270.3.987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of glycogen synthase kinase-3 by epidermal growth factor is mediated by mitogen-activated protein kinase/p90 ribosomal protein S6 kinase signaling pathway in NIH/3T3 cells</span></div><div class="casAuthors">Eldar-Finkelman, Hagit; Seger, Rony; Vandenheede, Jackie R.; Krebs, Edwin G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">987-90</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The role of the p90 ribosomal protein S6 kinase/mitogen-activated protein kinase (RSK/MAPK) signaling pathway in regulating glycogen synthase kinase-3 (GSK-3) activity was investigated.  In vitro studies showed that GSK-3 was inactivated by 50% upon incubation with RSK purified from epidermal growth factor (EGF)-stimulated NIH/3T3 cells.  Subsequently, the effect of EGF on GSK-3 activity was measured in NIH/3T3 cells that stably overexpressed mutated forms of MAPK kinase (MAPKK).  The activation of RSK by EGF was markedly decreased in cell lines expressing the dominant neg. MAPKK mutants S222A and K97A and was increased in cells expressing the S222E mutant as compared with control cell lines.  EGF induced a rapid decrease in GSK-3β activity (50%) in control and S222E cells; however, only 25 and 10% inhibition in GSK-3β activity was obsd. in cell lines expressing the dominant neg. mutants K97A and S222A, resp., suggesting that inhibition of GSK-3 was partially blocked in these cells.  Apparently, the action of EGF on GSK-3 inactivation is mediated by the RSK/MAPK signaling pathway in NIH/3T3 cells and provide evidence for a mechanism regulating GSK-3 activity in intact cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMwJxIcPF8vLVg90H21EOLACvtfcHk0ljSIYF10JR3zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjtFOksr0%253D&md5=410b5eeb1f9b0ebeb4a7412775afd351</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.3.987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.3.987%26sid%3Dliteratum%253Aachs%26aulast%3DEldar-Finkelman%26aufirst%3DH.%26aulast%3DSeger%26aufirst%3DR.%26aulast%3DVandenheede%26aufirst%3DJ.%2BR.%26aulast%3DKrebs%26aufirst%3DE.%2BG.%26atitle%3DInactivation%2520of%2520glycogen%2520synthase%2520kinase-3%2520by%2520epidermal%2520growth%2520factor%2520is%2520mediated%2520by%2520mitogen-activated%2520protein%2520kinase%252Fp90%2520ribosomal%2520protein%2520S6%2520kinase%2520signaling%2520pathway%2520in%2520NIH%252F3T3%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D987%26epage%3D990%26doi%3D10.1074%2Fjbc.270.3.987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular cloning and expression of glycogen synthase kinase-3/factor A</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2431</span>– <span class="NLM_lpage">2438</span>, <span class="refDoi"> DOI: 10.1002/j.1460-2075.1990.tb07419.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2Fj.1460-2075.1990.tb07419.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=2164470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK3MXis1Gr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1990&pages=2431-2438&author=J.+R.+Woodgett&title=Molecular+cloning+and+expression+of+glycogen+synthase+kinase-3%2Ffactor+A&doi=10.1002%2Fj.1460-2075.1990.tb07419.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular cloning and expression of glycogen synthase kinase-3/factor A</span></div><div class="casAuthors">Woodgett, James Robert</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2431-8</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) is a protein-serine kinase implicated in the hormonal control of several regulatory proteins, including glycogen synthase and transcription factor c-jun.  Two classes of rat brain cDNA for this enzyme were isolated (GSK-3α and GSK-3β).  The α-type encoded a 51-kDa polypeptide, the sequence of which included all of the tryptic peptides detd. by protein sequence anal. of purified skeletal muscle GSK-3.  The novel β-type cDNA had the potential to encode a 47-kDA protein with 85% amino acid identity to GSK-3α.  The 2 types of cDNA were the products of distinct genes as detd. by genomic organization and nucleic acid sequence anal.  Both α and β clones exhibited kinase activity when expressed in COS-1 cells and type-specific antibodies to GSK-3 α and β detect proteins of 51 and 47 kDa, resp., in a variety of rat tissue exts., with highest levels of both in brain.  Partial purifn. of GSK-3 activity from bovine resulted in the isolation of active α and β proteins.  The physiol. importance of these 2 proteins in cellular signal transduction is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm736N83asgLVg90H21EOLACvtfcHk0liR__ou6hRqBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXis1Gr&md5=3bacbf348b8ba673831fe9b9ca5cfde6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1990.tb07419.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1990.tb07419.x%26sid%3Dliteratum%253Aachs%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520cloning%2520and%2520expression%2520of%2520glycogen%2520synthase%2520kinase-3%252Ffactor%2520A%26jtitle%3DEMBO%2520J.%26date%3D1990%26volume%3D9%26spage%3D2431%26epage%3D2438%26doi%3D10.1002%2Fj.1460-2075.1990.tb07419.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Glycogen Synthase Kinase 3: A Kinase for All Pathways?</span>. <i>Curr. Top. Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1016/bs.ctdb.2016.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fbs.ctdb.2016.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=28236969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKjs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2017&pages=277-302&author=P.+Patelauthor=J.+R.+Woodgett&title=Glycogen+Synthase+Kinase+3%3A+A+Kinase+for+All+Pathways%3F&doi=10.1016%2Fbs.ctdb.2016.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Glycogen synthase kinase 3: a kinase for all pathways?</span></div><div class="casAuthors">Patel, Prital; Woodgett, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Developmental Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">Protein Kinases in Development and Disease</span>),
    <span class="NLM_cas:pages">277-302</span>CODEN:
                <span class="NLM_cas:coden">CTDBA5</span>;
        ISSN:<span class="NLM_cas:issn">0070-2153</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) is an unusual protein-serine kinase in that it is primarily regulated by inhibition and lies downstream of multiple cell signaling pathways.  This raises a variety of questions in terms of its physiol. role(s), how signaling specificity is maintained and why so many eggs have been placed into one basket.  There are actually two baskets, as there are two isoforms, GSK-3α and β, that are highly related and largely redundant.  Their many substrates range from regulators of cellular metab. to mols. that control growth and differentiation.  In this chapter, we review the characteristics of GSK-3, update progress in understanding the kinase, and try to answer some of the questions raised by its unusual properties.  Indeed, the kinase may trigger transformation in our thinking of how cellular signals are organized and controlled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolxBj01kLDaLVg90H21EOLACvtfcHk0liR__ou6hRqBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKjs70%253D&md5=4c219601701e017e9101b3e7a15b301c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fbs.ctdb.2016.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.ctdb.2016.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DGlycogen%2520Synthase%2520Kinase%25203%253A%2520A%2520Kinase%2520for%2520All%2520Pathways%253F%26jtitle%3DCurr.%2520Top.%2520Dev.%2520Biol.%26date%3D2017%26volume%3D123%26spage%3D277%26epage%3D302%26doi%3D10.1016%2Fbs.ctdb.2016.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubie, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, M.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Requirement for glycogen synthase kinase-3β in cell survival and NF-κB activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1038/35017574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2F35017574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10894547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt12rtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=86-90&author=K.+P.+Hoeflichauthor=J.+Luoauthor=E.+A.+Rubieauthor=M.-S.+Tsaoauthor=O.+Jinauthor=J.+R.+Woodgett&title=Requirement+for+glycogen+synthase+kinase-3%CE%B2+in+cell+survival+and+NF-%CE%BAB+activation&doi=10.1038%2F35017574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement for glycogen synthase kinase-3β in cell survival and NF-κB activation</span></div><div class="casAuthors">Hoeflich, Klaus P.; Luo, Juan; Ruble, Elizabeth A.; Tsao, Ming-Sound; Jin, Ou; Woodgett, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6791</span>),
    <span class="NLM_cas:pages">86-90</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3)-α and -β are dosely related protein-serine kinases, which act as inhibitory components of Wnt signalling during embryonic development and cell proliferation in adult tissues.  Insight into the physiol. function of GSK-3 has emerged from genetic anal. in Drosophila, Dictyostelium and yeast.  Here, we show that disruption of the murine GSK-3β gene results in embryonic lethality caused by severe liver degeneration during mid-gestation, a phenotype consistent with excessive tumor necrosis factor (TNF) toxicity, as obsd. in mice lacking genes involved in the activation of the transcription factor activation NF-κB.  GSK-3β-deficient embryos were rescued by inhibition of TNF using an anti-TNF-α antibody.  Fibroblasts from GSK-3β-deficient embryos were hypersensitive to TNF-α and showed reduced NF-κB function.  Lithium treatment (which inhibits GSK-3) sensitized wild-type fibroblasts to TNF and inhibited transactivation of NF-κB.  The early steps leading to NF-κB activation (degrdn. of I-κB and translocation of NF-κB to the nucleus) were unaffected by the loss of GSK-3β, indicating that NF-κB is regulated by GSK-3β at the level of the transcriptional complex.  Thus, GSK-3β facilitates NF-κB function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN2-FF0neYjLVg90H21EOLACvtfcHk0lgCnJqXZ3tUVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt12rtbY%253D&md5=a3f2389ac771cbf1eb830171214c79c4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F35017574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35017574%26sid%3Dliteratum%253Aachs%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DRubie%26aufirst%3DE.%2BA.%26aulast%3DTsao%26aufirst%3DM.-S.%26aulast%3DJin%26aufirst%3DO.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DRequirement%2520for%2520glycogen%2520synthase%2520kinase-3%25CE%25B2%2520in%2520cell%2520survival%2520and%2520NF-%25CE%25BAB%2520activation%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D86%26epage%3D90%26doi%3D10.1038%2F35017574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doble, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kockeritz, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Functional Redundancy of GSK-3α and GSK-3β in Wnt/β-Catenin Signaling Shown by Using an Allelic Series of Embryonic Stem Cell Lines</span>. <i>Dev. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">957</span>– <span class="NLM_lpage">971</span>, <span class="refDoi"> DOI: 10.1016/j.devcel.2007.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.devcel.2007.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17543867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFentrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=957-971&author=B.+W.+Dobleauthor=S.+Patelauthor=G.+A.+Woodauthor=L.+K.+Kockeritzauthor=J.+R.+Woodgett&title=Functional+Redundancy+of+GSK-3%CE%B1+and+GSK-3%CE%B2+in+Wnt%2F%CE%B2-Catenin+Signaling+Shown+by+Using+an+Allelic+Series+of+Embryonic+Stem+Cell+Lines&doi=10.1016%2Fj.devcel.2007.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional redundancy of GSK-3α and GSK-3β in Wnt/β-catenin signaling shown by using an allelic series of embryonic stem cell lines</span></div><div class="casAuthors">Doble, Bradley W.; Patel, Satish; Wood, Geoffrey A.; Kockeritz, Lisa K.; Woodgett, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">957-971</span>CODEN:
                <span class="NLM_cas:coden">DCEEBE</span>;
        ISSN:<span class="NLM_cas:issn">1534-5807</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In mammalian cells, glycogen synthase kinase-3 (GSK-3) exists as two homologs, GSK-3α and GSK-3β, encoded by independent genes, which share similar kinase domains but differ substantially in their termini.  Here, we describe the generation of an allelic series of mouse embryonic stem cell (ESC) lines with 0-4 functional GSK-3 alleles and examine GSK-3-isoform function in Wnt/β-catenin signaling.  No compensatory upregulation in GSK-3 protein levels or activity was detected in cells lacking either GSK-3α or GSK-3β, and Wnt/β-catenin signaling was normal.  Only in cells lacking three or all four of the alleles was a gene-dosage effect on β-catenin/TCF-mediated transcription obsd.  Indeed, GSK-3α/β double-knockout ESCs displayed hyperactivated Wnt/β-catenin signaling and were severely compromised in their ability to differentiate, but could be rescued to normality by re-expression of functional GSK-3.  The rheostatic regulation of GSK-3 highlights the importance of considering the contributions of both homologs when studying GSK-3 functions in mammalian systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJw04huRADqLVg90H21EOLACvtfcHk0liRJG0PJbO0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsFentrg%253D&md5=8fae370755d55ce24159061f8436c265</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2007.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2007.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DDoble%26aufirst%3DB.%2BW.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DG.%2BA.%26aulast%3DKockeritz%26aufirst%3DL.%2BK.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DFunctional%2520Redundancy%2520of%2520GSK-3%25CE%25B1%2520and%2520GSK-3%25CE%25B2%2520in%2520Wnt%252F%25CE%25B2-Catenin%2520Signaling%2520Shown%2520by%2520Using%2520an%2520Allelic%2520Series%2520of%2520Embryonic%2520Stem%2520Cell%2520Lines%26jtitle%3DDev.%2520Cell%26date%3D2007%26volume%3D12%26spage%3D957%26epage%3D971%26doi%3D10.1016%2Fj.devcel.2007.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Killick, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovestone, S.</span></span> <span> </span><span class="NLM_article-title">The GSK3 hypothesis of Alzheimer’s disease</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">1433</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2007.05194.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1111%2Fj.1471-4159.2007.05194.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=18088381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkt1Citbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2008&pages=1433-1439&author=C.+Hooperauthor=R.+Killickauthor=S.+Lovestone&title=The+GSK3+hypothesis+of+Alzheimer%E2%80%99s+disease&doi=10.1111%2Fj.1471-4159.2007.05194.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The GSK3 hypothesis of Alzheimer's disease</span></div><div class="casAuthors">Hooper, Claudie; Killick, Richard; Lovestone, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1433-1439</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Glycogen synthase kinase 3 (GSK3) is a constitutively active, proline-directed serine/threonine kinase that plays a part in a no. of physiol. processes ranging from glycogen metab. to gene transcription.  GSK3 also plays a pivotal and central role in the pathogenesis of both sporadic and familial forms of Alzheimer's disease (AD), an observation that has led us to coin the 'GSK3 hypothesis of AD'.  According to this hypothesis, over-activity of GSK3 accounts for memory impairment, tau hyper-phosphorylation, increased β-amyloid prodn. and local plaque-assocd. microglial-mediated inflammatory responses; all of which are hallmark characteristics of AD.  If our 'GSK3 hypothesis of AD' is substantiated and GSK3 is indeed a causal mediator of AD then inhibitors of GSK3 would provide a novel avenue for therapeutic intervention in this devastating disorder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9lxWCe90horVg90H21EOLACvtfcHk0liRJG0PJbO0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkt1Citbw%253D&md5=6e5f9918668aac2d3127bacad12a8340</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2007.05194.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2007.05194.x%26sid%3Dliteratum%253Aachs%26aulast%3DHooper%26aufirst%3DC.%26aulast%3DKillick%26aufirst%3DR.%26aulast%3DLovestone%26aufirst%3DS.%26atitle%3DThe%2520GSK3%2520hypothesis%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Neurochem.%26date%3D2008%26volume%3D104%26spage%3D1433%26epage%3D1439%26doi%3D10.1111%2Fj.1471-4159.2007.05194.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballatore, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojanowski, J. Q.</span></span> <span> </span><span class="NLM_article-title">Tau-mediated neurodegeneration in Alzheimer’s disease and related disorders</span>. <i>Nat. Rev. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1038/nrn2194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrn2194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17684513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpt1Crt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=663-672&author=C.+Ballatoreauthor=V.+M.-Y.+Leeauthor=J.+Q.+Trojanowski&title=Tau-mediated+neurodegeneration+in+Alzheimer%E2%80%99s+disease+and+related+disorders&doi=10.1038%2Fnrn2194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Tau-mediated neurodegeneration in Alzheimer's disease and related disorders</span></div><div class="casAuthors">Ballatore, Carlo; Lee, Virginia M.-Y.; Trojanowski, John Q.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">663-672</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Advances in the understanding of the mechanisms of tau-mediated neurodegeneration in Alzheimer's disease (AD) and related tauopathies, which are characterized by prominent CNS accumulations of fibrillar tau inclusions, are rapidly moving this previously underexplored disease pathway to center stage for disease-modifying drug discovery efforts.  However, controversies abound concerning whether or not the deleterious effects of tau pathologies result from toxic gains-of-function by pathol. tau or from crit. losses of normal tau function in the disease state.  This review summarizes the most recent advances in the authors' knowledge of the mechanisms of tau-mediated neurodegeneration to forge an integrated concept of those tau-linked disease processes that drive the onset and progression of AD and related tauopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8-k_a3SnV7bVg90H21EOLACvtfcHk0lj3lpc_bBBszA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpt1Crt78%253D&md5=eef3cc99033075b3215b739521d672f6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrn2194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn2194%26sid%3Dliteratum%253Aachs%26aulast%3DBallatore%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DV.%2BM.-Y.%26aulast%3DTrojanowski%26aufirst%3DJ.%2BQ.%26atitle%3DTau-mediated%2520neurodegeneration%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%2520and%2520related%2520disorders%26jtitle%3DNat.%2520Rev.%2520Neurosci.%26date%3D2007%26volume%3D8%26spage%3D663%26epage%3D672%26doi%3D10.1038%2Fnrn2194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adlard, P. A.</span></span> <span> </span><span class="NLM_article-title">GSK-3 in neurodegenerative diseases</span>. <i>Int. J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2011</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.4061/2011/189246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.4061%2F2011%2F189246" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2011&publication_year=2011&pages=1-9&author=P.+Leiauthor=S.+Aytonauthor=A.+I.+Bushauthor=P.+A.+Adlard&title=GSK-3+in+neurodegenerative+diseases&doi=10.4061%2F2011%2F189246"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4061%2F2011%2F189246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4061%252F2011%252F189246%26sid%3Dliteratum%253Aachs%26aulast%3DLei%26aufirst%3DP.%26aulast%3DAyton%26aufirst%3DS.%26aulast%3DBush%26aufirst%3DA.%2BI.%26aulast%3DAdlard%26aufirst%3DP.%2BA.%26atitle%3DGSK-3%2520in%2520neurodegenerative%2520diseases%26jtitle%3DInt.%2520J.%2520Alzheimers%2520Dis.%26date%3D2011%26volume%3D2011%26spage%3D1%26epage%3D9%26doi%3D10.4061%2F2011%2F189246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phiel, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, P. S.</span></span> <span> </span><span class="NLM_article-title">GSK-3α regulates production of Alzheimer’s disease amyloid-β peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>423</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1038/nature01640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnature01640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=12761548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvFaltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2003&pages=435-439&author=C.+J.+Phielauthor=C.+A.+Wilsonauthor=V.+M.-Y.+Leeauthor=P.+S.+Klein&title=GSK-3%CE%B1+regulates+production+of+Alzheimer%E2%80%99s+disease+amyloid-%CE%B2+peptides&doi=10.1038%2Fnature01640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">GSK-3α regulates production of Alzheimer's disease amyloid-β peptides</span></div><div class="casAuthors">Phiel, Christopher J.; Wilson, Christina A.; Lee, Virginia M.-Y.; Klein, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">6938</span>),
    <span class="NLM_cas:pages">435-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Alzheimer's disease is assocd. with increased prodn. and aggregation of amyloid-β (Aβ) peptides.  Aβ peptides are derived from the amyloid precursor protein (APP) by sequential proteolysis, catalyzed by the aspartyl protease BACE, followed by presenilin-dependent γ-secretase cleavage.  Presenilin interacts with nicastrin, APH-1 and PEN-2, all of which are required for γ-secretase function.  Presenilins also interact with α-catenin, β-catenin and glycogen synthase kinase-3β (GSK-3β), but a functional role for these proteins in γ-secretase activity has not been established.  Here the authors show that therapeutic concns. of lithium, a GSK-3 inhibitor, block the prodn. of Aβ peptides by interfering with APP cleavage at the γ-secretase step, but do not inhibit Notch processing.  Importantly, lithium also blocks the accumulation of Aβ peptides in the brains of mice that overproduce APP.  The target of lithium in this setting is GSK-3α, which is required for maximal processing of APP.  Since GSK-3 also phosphorylates tau protein, the principal component of neurofibrillary tangles, inhibition of GSK-3α offers a new approach to reduce the formation of both amyloid plaques and neurofibrillary tangles, two pathol. hallmarks of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQu3H2UA-dbbVg90H21EOLACvtfcHk0listeQwQsX8DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvFaltLY%253D&md5=4f2db433242c3ad2f510a05dffe6f5d4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature01640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01640%26sid%3Dliteratum%253Aachs%26aulast%3DPhiel%26aufirst%3DC.%2BJ.%26aulast%3DWilson%26aufirst%3DC.%2BA.%26aulast%3DLee%26aufirst%3DV.%2BM.-Y.%26aulast%3DKlein%26aufirst%3DP.%2BS.%26atitle%3DGSK-3%25CE%25B1%2520regulates%2520production%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%2520amyloid-%25CE%25B2%2520peptides%26jtitle%3DNature%26date%3D2003%26volume%3D423%26spage%3D435%26epage%3D439%26doi%3D10.1038%2Fnature01640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, B.</span></span> <span> </span><span class="NLM_article-title">Divergent roles of GSK3 and CDK5 in APP processing</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>312</i></span>,  <span class="NLM_fpage">922</span>– <span class="NLM_lpage">929</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2003.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.bbrc.2003.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=14651959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptl2qu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=312&publication_year=2003&pages=922-929&author=J.+Ryderauthor=Y.+Suauthor=F.+Liuauthor=B.+Liauthor=Y.+Zhouauthor=B.+Ni&title=Divergent+roles+of+GSK3+and+CDK5+in+APP+processing&doi=10.1016%2Fj.bbrc.2003.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent roles of GSK3 and CDK5 in APP processing</span></div><div class="casAuthors">Ryder, John; Su, Yuan; Liu, Feng; Li, Baolin; Zhou, Yan; Ni, Binhui</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">312</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">922-929</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK3) and cyclin-dependent kinase 5 (CDK5) are related serine/threonine kinases that have been well studied for their role in tau hyperphosphorylation, however, little is known about their significance in APP processing.  Here we report that GSK3 and CDK5 are involved in APP processing in a divergent manner.  Specific inhibition of cellular GSK3 by lithium or GSK3β antisense elicits a redn. in Aβ.  Conversely, neg. modulation of cellular CDK5 activity by CDK5 inhibitor, roscovitine, or CDK5 antisense stimulates Aβ prodn.  Neither GSK3 nor CDK5 inhibition by these means significantly affected cellular APP levels or APP maturation.  Moreover, oral administration of lithium significantly reduces Aβ prodn. whereas direct ICV administration of roscovitine augmented Aβ prodn. in the brains of PDAPP (APPV717F) mice.  Our data support a function for both GSK3 and CDK5 in APP processing, further implicating these two kinases in the pathogenesis of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHF5xGtT6VGLVg90H21EOLACvtfcHk0lj2CpsNSDJ1JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptl2qu7o%253D&md5=cfa42655074ec97afdd76b5109844562</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2003.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2003.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DRyder%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DNi%26aufirst%3DB.%26atitle%3DDivergent%2520roles%2520of%2520GSK3%2520and%2520CDK5%2520in%2520APP%2520processing%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2003%26volume%3D312%26spage%3D922%26epage%3D929%26doi%3D10.1016%2Fj.bbrc.2003.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klementiev, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikova, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novitskaya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walmod, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dmytriyeva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pakkenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, E.</span></span> <span> </span><span class="NLM_article-title">A neural cell adhesion molecule-derived peptide reduces neuropathological signs and cognitive impairment induced by Aβ25-35</span>. <i>Neuroscience</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1016/j.neuroscience.2006.11.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.neuroscience.2006.11.060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17223274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVGms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2007&pages=209-224&author=B.+Klementievauthor=T.+Novikovaauthor=V.+Novitskayaauthor=P.+S.+Walmodauthor=O.+Dmytriyevaauthor=B.+Pakkenbergauthor=V.+Berezinauthor=E.+Bock&title=A+neural+cell+adhesion+molecule-derived+peptide+reduces+neuropathological+signs+and+cognitive+impairment+induced+by+A%CE%B225-35&doi=10.1016%2Fj.neuroscience.2006.11.060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A neural cell adhesion molecule-derived peptide reduces neuropathological signs and cognitive impairment induced by Aβ25-35</span></div><div class="casAuthors">Klementiev, B.; Novikova, T.; Novitskaya, V.; Walmod, P. S.; Dmytriyeva, O.; Pakkenberg, B.; Berezin, V.; Bock, E.</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">209-224</span>CODEN:
                <span class="NLM_cas:coden">NRSCDN</span>;
        ISSN:<span class="NLM_cas:issn">0306-4522</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">By means of i.c.v. administration of pre-aggregated oligomeric β-amyloid (Aβ)25-35 peptide it was possible in rats to generate neuropathol. signs related to those of early stages of Alzheimer's disease (AD).  Aβ25-35-administration induced the deposition of endogenously produced amyloid protein.  Furthermore, quant. immunohistochem. demonstrated time-related statistically significant increases in amyloid immunoreactivity, tau phosphorylation, microglial activation, and astrocytosis, and stereol. investigations demonstrated statistically significant increased neuronal cell death and brain atrophy in response to Aβ25-35.  Finally, the Aβ25-35-administration led to a reduced short-term memory as detd. by the social recognition test.  A synthetic peptide termed FGL derived from the neural cell adhesion mol. (NCAM) was able to prevent or, if already manifest, strongly reduce all investigated signs of Aβ25-35-induced neuropathol. and cognitive impairment.  The FGL peptide was recently demonstrated to be able to cross the blood-brain-barrier.  Accordingly, we found that the beneficial effects of FGL were achieved not only by intracisternal, but also by intranasal and s.c. administration of the peptide.  Furthermore, FGL-treatment was shown to inhibit the activity of GSK3β, a kinase implicated in signaling regulating cell survival, tau phosphorylation and the processing of the amyloid precursor protein (APP).  Thus, the peptide induced a statistically significant increase in the fraction of GSK3β phosphorylated on the Ser9-position, a posttranslational modification known to inhibit the activity of the kinase.  Hence, the mode of action of FGL with respect to the preventive and curative effects on Aβ25-35-induced neuropathol. manifestations and cognitive impairment involves the modulation of intracellular signal-transduction mediated through GSK3β.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogrbF8iKc417Vg90H21EOLACvtfcHk0lj2CpsNSDJ1JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVGms7w%253D&md5=3439510cfedc40d220b8dcda52511e57</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2006.11.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2006.11.060%26sid%3Dliteratum%253Aachs%26aulast%3DKlementiev%26aufirst%3DB.%26aulast%3DNovikova%26aufirst%3DT.%26aulast%3DNovitskaya%26aufirst%3DV.%26aulast%3DWalmod%26aufirst%3DP.%2BS.%26aulast%3DDmytriyeva%26aufirst%3DO.%26aulast%3DPakkenberg%26aufirst%3DB.%26aulast%3DBerezin%26aufirst%3DV.%26aulast%3DBock%26aufirst%3DE.%26atitle%3DA%2520neural%2520cell%2520adhesion%2520molecule-derived%2520peptide%2520reduces%2520neuropathological%2520signs%2520and%2520cognitive%2520impairment%2520induced%2520by%2520A%25CE%25B225-35%26jtitle%3DNeuroscience%26date%3D2007%26volume%3D145%26spage%3D209%26epage%3D224%26doi%3D10.1016%2Fj.neuroscience.2006.11.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanger, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brion, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, B. H.</span></span> <span> </span><span class="NLM_article-title">Glycogen synthase kinase-3 induces Alzheimer’s disease-like phosphorylation of tau: generation of paired helical filament epitopes and neuronal localisation of the kinase</span>. <i>Neurosci. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/0304-3940(92)90774-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2F0304-3940%2892%2990774-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1336152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK3sXpvFGksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=1992&pages=58-62&author=D.+P.+Hangerauthor=K.+Hughesauthor=J.+R.+Woodgettauthor=J.-P.+Brionauthor=B.+H.+Anderton&title=Glycogen+synthase+kinase-3+induces+Alzheimer%E2%80%99s+disease-like+phosphorylation+of+tau%3A+generation+of+paired+helical+filament+epitopes+and+neuronal+localisation+of+the+kinase&doi=10.1016%2F0304-3940%2892%2990774-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Glycogen synthase kinase-3 induces Alzheimer's disease-like phosphorylation of tau:  generation of paired helical filament epitopes and neuronal localization of the kinase</span></div><div class="casAuthors">Hanger, Diane P.; Hughes, Kenneth; Woodgett, James R.; Brion, Jean Pierre; Anderton, Brian H.</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-62</span>CODEN:
                <span class="NLM_cas:coden">NELED5</span>;
        ISSN:<span class="NLM_cas:issn">0304-3940</span>.
    </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) reduced the mobility of human tau on SDS-PAGE, prevented binding of the monoclonal antibody (mAb) Tau. 1, and induced binding of the mAb 8D8.  Recombinant tau phosphorylated by GSK-3 aligned on SDS-PAGE with the abnormally phosphorylated tau (PHF-tau) assocd. with the paired helical filaments in Alzheimer's disease brain.  Phosphorylated serine396 (numbering of the largest human brain tau isoform) was identified as a binding site on tau for mAb 8D8.  The localization of GSK-3 within granular structures in pyramidal cells indicates that GSK-3α and GSK-3β may have a role in the prodn. of PHF-tau in Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOM1_KaxlV9LVg90H21EOLACvtfcHk0ljA5JkpICpPPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXpvFGksg%253D%253D&md5=88d0a3271ce167362ce8b363bae5dbf4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2F0304-3940%2892%2990774-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3940%252892%252990774-2%26sid%3Dliteratum%253Aachs%26aulast%3DHanger%26aufirst%3DD.%2BP.%26aulast%3DHughes%26aufirst%3DK.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26aulast%3DBrion%26aufirst%3DJ.-P.%26aulast%3DAnderton%26aufirst%3DB.%2BH.%26atitle%3DGlycogen%2520synthase%2520kinase-3%2520induces%2520Alzheimer%25E2%2580%2599s%2520disease-like%2520phosphorylation%2520of%2520tau%253A%2520generation%2520of%2520paired%2520helical%2520filament%2520epitopes%2520and%2520neuronal%2520localisation%2520of%2520the%2520kinase%26jtitle%3DNeurosci.%2520Lett.%26date%3D1992%26volume%3D147%26spage%3D58%26epage%3D62%26doi%3D10.1016%2F0304-3940%2892%2990774-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishiguro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imahori, K.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation sites on tau by tau protein kinase I, a bovine derived kinase generating an epitope of paired helical filaments</span>. <i>Neurosci. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1016/0304-3940(92)90839-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2F0304-3940%2892%2990839-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1284442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK3sXhsVCqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=1992&pages=202-206&author=K.+Ishiguroauthor=A.+Omoriauthor=M.+Takamatsuauthor=K.+Satoauthor=M.+Ariokaauthor=T.+Uchidaauthor=K.+Imahori&title=Phosphorylation+sites+on+tau+by+tau+protein+kinase+I%2C+a+bovine+derived+kinase+generating+an+epitope+of+paired+helical+filaments&doi=10.1016%2F0304-3940%2892%2990839-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation sites on tau by tau protein kinase I, a bovine derived kinase generating an epitope of paired helical filaments</span></div><div class="casAuthors">Ishiguro, Koichi; Omori, Akira; Takamatsu, Masako; Sato, Kazuki; Arioka, Manabu; Uchida, Tsuneko; Imahori, Kazutomo</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">202-6</span>CODEN:
                <span class="NLM_cas:coden">NELED5</span>;
        ISSN:<span class="NLM_cas:issn">0304-3940</span>.
    </div><div class="casAbstract">Tau protein kinase I (TPKI) isolated from bovine brain has been detd. to phosphorylate tau at four distinct sites by detecting modified Ser and Thr residues with protein sequencer.  Ser199, Thr231, Ser396, and Ser413 were phosphorylated by TPKI (numbering of amino acids was done in relation to the longest human tau).  These phosphorylations generate an epitope of PHF (paired helical filaments) and eliminate the recognition of tau by the monoclonal antibody, tau-1.  Thus, TPKI might be responsible for at least some of the phosphorylation of tau to induce PHF formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfG_9kTJOSILVg90H21EOLACvtfcHk0lhduisH0KWf1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhsVCqurw%253D&md5=0fb06cefdd89a7bce3449e61ec9f95f6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2F0304-3940%2892%2990839-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3940%252892%252990839-y%26sid%3Dliteratum%253Aachs%26aulast%3DIshiguro%26aufirst%3DK.%26aulast%3DOmori%26aufirst%3DA.%26aulast%3DTakamatsu%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DArioka%26aufirst%3DM.%26aulast%3DUchida%26aufirst%3DT.%26aulast%3DImahori%26aufirst%3DK.%26atitle%3DPhosphorylation%2520sites%2520on%2520tau%2520by%2520tau%2520protein%2520kinase%2520I%252C%2520a%2520bovine%2520derived%2520kinase%2520generating%2520an%2520epitope%2520of%2520paired%2520helical%2520filaments%26jtitle%3DNeurosci.%2520Lett.%26date%3D1992%26volume%3D148%26spage%3D202%26epage%3D206%26doi%3D10.1016%2F0304-3940%2892%2990839-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, J.</span></span> <span> </span><span class="NLM_article-title">Tau-based therapies in neurodegeneration: opportunities and challenges</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">883</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrd.2017.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=28983098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=863-883&author=C.+Liauthor=J.+G%C3%B6tz&title=Tau-based+therapies+in+neurodegeneration%3A+opportunities+and+challenges&doi=10.1038%2Fnrd.2017.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Tau-based therapies in neurodegeneration: opportunities and challenges</span></div><div class="casAuthors">Li, Chuanzhou; Gotz, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">863-883</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Aggregates of the microtubule-assocd. protein tau are a defining feature of several neurodegenerative diseases that are collectively known as tauopathies, and constitute one of the hallmark lesions of Alzheimer disease (AD).  Given the lack of efficacy to date of amyloid-β-targeted therapies for AD, interest is growing in tau as a potential alternative target.  Several drug candidates, which are now in clin. trials, aim to reduce tau levels or to prevent the aggregation or pathol. post-translation modifications of this protein.  In this Review, we discuss preclin. and clin. studies in light of an increased understanding of the physiol. and pathol. roles of tau, advances in animal models of tauopathy, the identification of novel targets and the availability of novel tracers to track tau.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSC7Md8i-637Vg90H21EOLACvtfcHk0lhduisH0KWf1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7jF&md5=389b4be6419498344b5e53ca1be9c22e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.155%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DG%25C3%25B6tz%26aufirst%3DJ.%26atitle%3DTau-based%2520therapies%2520in%2520neurodegeneration%253A%2520opportunities%2520and%2520challenges%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D863%26epage%3D883%26doi%3D10.1038%2Fnrd.2017.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zehr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olm, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suleman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaynor, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFrancois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duff, K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of glycogen synthase kinase-3 by lithium correlates with reduced tauopathy and degeneration in vivo</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">6990</span>– <span class="NLM_lpage">6995</span>, <span class="refDoi"> DOI: 10.1073/pnas.0500466102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1073%2Fpnas.0500466102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=15867159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6990-6995&author=W.+Nobleauthor=E.+Planelauthor=C.+Zehrauthor=V.+Olmauthor=J.+Meyersonauthor=F.+Sulemanauthor=K.+Gaynorauthor=L.+Wangauthor=J.+LaFrancoisauthor=B.+Feinsteinauthor=M.+Burnsauthor=P.+Krishnamurthyauthor=Y.+Wenauthor=R.+Bhatauthor=J.+Lewisauthor=D.+Dicksonauthor=K.+Duff&title=Inhibition+of+glycogen+synthase+kinase-3+by+lithium+correlates+with+reduced+tauopathy+and+degeneration+in+vivo&doi=10.1073%2Fpnas.0500466102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of glycogen synthase kinase-3 by lithium correlates with reduced tauopathy and degeneration in vivo</span></div><div class="casAuthors">Noble, Wendy; Planel, Emmanuel; Zehr, Cindy; Olm, Vicki; Meyerson, Jordana; Suleman, Farhana; Gaynor, Kate; Wang, Lili; LaFrancois, John; Feinstein, Boris; Burns, Mark; Krishnamurthy, Pavan; Wen, Yi; Bhat, Ratan; Lewis, Jada; Dickson, Dennis; Duff, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6990-6995</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neurofibrillary tangles composed of hyperphosphorylated, aggregated tau are a common pathol. feature of tauopathies, including Alzheimer's disease.  Abnormal phosphorylation of tau by kinases or phosphatases has been proposed as a pathogenic mechanism in tangle formation.  To investigate whether kinase inhibition can reduce tauopathy and the degeneration assocd. with it in vivo, transgenic mice overexpressing mutant human tau were treated with the glycogen synthase kinase-3 (GSK-3) inhibitor lithium chloride.  Treatment resulted in significant inhibition of GSK-3 activity.  Lithium administration also resulted in significantly lower levels of phosphorylation at several epitopes of tau known to be hyperphosphorylated in Alzheimer's disease and significantly reduced levels of aggregated, insol. tau.  Administration of a second GSK-3 inhibitor also correlated with reduced insol. tau levels, supporting the idea that lithium exerts its effect through GSK-3 inhibition.  Levels of aggregated tau correlated strongly with degree of axonal degeneration, and lithium-chloride-treated mice showed less degeneration if administration was started during early stages of tangle development.  These results support the idea that kinases are involved in tauopathy progression and that kinase inhibitors may be effective therapeutically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE2qlkQMP4FLVg90H21EOLACvtfcHk0lgbLR6zxWn4VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKns78%253D&md5=b3f258b86536144dc5d2e483abca4f36</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0500466102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0500466102%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DW.%26aulast%3DPlanel%26aufirst%3DE.%26aulast%3DZehr%26aufirst%3DC.%26aulast%3DOlm%26aufirst%3DV.%26aulast%3DMeyerson%26aufirst%3DJ.%26aulast%3DSuleman%26aufirst%3DF.%26aulast%3DGaynor%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLaFrancois%26aufirst%3DJ.%26aulast%3DFeinstein%26aufirst%3DB.%26aulast%3DBurns%26aufirst%3DM.%26aulast%3DKrishnamurthy%26aufirst%3DP.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DBhat%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DJ.%26aulast%3DDickson%26aufirst%3DD.%26aulast%3DDuff%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520glycogen%2520synthase%2520kinase-3%2520by%2520lithium%2520correlates%2520with%2520reduced%2520tauopathy%2520and%2520degeneration%2520in%2520vivo%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D6990%26epage%3D6995%26doi%3D10.1073%2Fpnas.0500466102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Ramos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morán, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, J.</span></span> <span> </span><span class="NLM_article-title">Decreased nuclear beta-catenin, tau hyperphosphorylation and neurodegeneration in GSK-3beta conditional transgenic mice</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1093/emboj/20.1.27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1093%2Femboj%2F20.1.27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=11226152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvVCmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=27-39&author=J.+J.+Lucasauthor=F.+Hern%C3%A1ndezauthor=P.+G%C3%B3mez-Ramosauthor=M.+A.+Mor%C3%A1nauthor=R.+Henauthor=J.+Avila&title=Decreased+nuclear+beta-catenin%2C+tau+hyperphosphorylation+and+neurodegeneration+in+GSK-3beta+conditional+transgenic+mice&doi=10.1093%2Femboj%2F20.1.27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Decreased nuclear β-catenin, tau hyperphosphorylation and neurodegeneration in GSK-3β conditional transgenic mice</span></div><div class="casAuthors">Lucas, Jose J.; Hernandez, Felix; Gomez-Ramos, Pilar; Moran, Maria A.; Hen, Rene; Avila, Jesus</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1 & 2</span>),
    <span class="NLM_cas:pages">27-39</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3β (GSK-3β) has been postulated to mediate Alzheimer's disease tau hyperphosphorylation, β-amyloid-induced neurotoxicity and presenilin-1 mutation pathogenic effects.  By using the tet-regulated system we have produced conditional transgenic mice overexpressing GSK-3β in the brain during adulthood while avoiding perinatal lethality due to embryonic transgene expression.  These mice show decreased levels of nuclear β-catenin and hyperphosphorylation of tau in hippocampal neurons, the latter resulting in pretangle-like somatodendritic localization of tau.  Neurons displaying somatodendritic localization of tau often show abnormal morphologies and detachment from the surrounding neuropil.  Reactive astrocytosis and microgliosis were also indicative of neuronal stress and death.  This was further confirmed by TUNEL and cleaved caspase-3 immunostaining of dentate gyrus granule cells.  Our results demonstrate that in vivo overexpression of GSK-3β results in neurodegeneration and suggest that these mice can be used as an animal model to study the relevance of GSK-3β deregulation to the pathogenesis of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9riwM95AslLVg90H21EOLACvtfcHk0ljgKfWJ7uFwmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvVCmt78%253D&md5=70a7fa8aa64bb07c4b2cea1802f63e10</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F20.1.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F20.1.27%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DJ.%2BJ.%26aulast%3DHern%25C3%25A1ndez%26aufirst%3DF.%26aulast%3DG%25C3%25B3mez-Ramos%26aufirst%3DP.%26aulast%3DMor%25C3%25A1n%26aufirst%3DM.%2BA.%26aulast%3DHen%26aufirst%3DR.%26aulast%3DAvila%26aufirst%3DJ.%26atitle%3DDecreased%2520nuclear%2520beta-catenin%252C%2520tau%2520hyperphosphorylation%2520and%2520neurodegeneration%2520in%2520GSK-3beta%2520conditional%2520transgenic%2520mice%26jtitle%3DEMBO%2520J.%26date%3D2001%26volume%3D20%26spage%3D27%26epage%3D39%26doi%3D10.1093%2Femboj%2F20.1.27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundke-iqbal, I.</span></span> <span> </span><span class="NLM_article-title">Distribution, levels, and activity of glycogen synthase kinase-3 in the Alzheimer disease brain</span>. <i>J. Neuropathol. Exp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1097/00005072-199701000-00007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1097%2F00005072-199701000-00007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=8990130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A280%3ADyaK2s7ksVelsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1997&pages=70-78&author=J.-J.+Peiauthor=T.+Tanakaauthor=Y.-C.+Tungauthor=E.+Braakauthor=K.+Iqbalauthor=I.+Grundke-iqbal&title=Distribution%2C+levels%2C+and+activity+of+glycogen+synthase+kinase-3+in+the+Alzheimer+disease+brain&doi=10.1097%2F00005072-199701000-00007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Distribution, levels, and activity of glycogen synthase kinase-3 in the Alzheimer disease brain</span></div><div class="casAuthors">Pei J J; Tanaka T; Tung Y C; Braak E; Iqbal K; Grundke-Iqbal I</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuropathology and experimental neurology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-8</span>
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    </div><div class="casAbstract">A number of studies have implicated a proline-directed protein kinase, glycogen synthase kinase-3 (GSK-3) in the hyperphosphorylation of tau in Alzheimer's disease (AD).  Toward understanding the role of GSK-3 in the abnormal hyperphosphorylation of tau in AD we have found that GSK-3 is prominently present in neuronal cell bodies and their processes and co-localizes with neurofibrillary changes in AD brain.  Furthermore, the levels of GSK-3 as determined by indirect ELISA are approximately 50% increased in the postsynaptosomal supernatant from AD brains as compared to the controls.  However, no increase in GSK-3 enzyme activity was detected.  In AD brain, with its reduced phosphatase activity, even normal levels of GSK-3 activity might be sufficient for the hyperphosphorylation of tau.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQmu8m1y1XLJ0EgZCMqtRBifW6udTcc2eadl1FzWgUD_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2s7ksVelsQ%253D%253D&md5=18b0957c0a36ecb25875aaa4e8c6bc91</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1097%2F00005072-199701000-00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00005072-199701000-00007%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DJ.-J.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DTung%26aufirst%3DY.-C.%26aulast%3DBraak%26aufirst%3DE.%26aulast%3DIqbal%26aufirst%3DK.%26aulast%3DGrundke-iqbal%26aufirst%3DI.%26atitle%3DDistribution%252C%2520levels%252C%2520and%2520activity%2520of%2520glycogen%2520synthase%2520kinase-3%2520in%2520the%2520Alzheimer%2520disease%2520brain%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D1997%26volume%3D56%26spage%3D70%26epage%3D78%26doi%3D10.1097%2F00005072-199701000-00007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blalock, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geddes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markesbery, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landfield, P. W.</span></span> <span> </span><span class="NLM_article-title">Incipient Alzheimer’s disease: Microarray correlation analyses reveal major transcriptional and tumor suppressor responses</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">2173</span>– <span class="NLM_lpage">2178</span>, <span class="refDoi"> DOI: 10.1073/pnas.0308512100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1073%2Fpnas.0308512100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=14769913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1SrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=2173-2178&author=E.+M.+Blalockauthor=J.+W.+Geddesauthor=K.+C.+Chenauthor=N.+M.+Porterauthor=W.+R.+Markesberyauthor=P.+W.+Landfield&title=Incipient+Alzheimer%E2%80%99s+disease%3A+Microarray+correlation+analyses+reveal+major+transcriptional+and+tumor+suppressor+responses&doi=10.1073%2Fpnas.0308512100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Incipient Alzheimer's disease: Microarray correlation analyses reveal major transcriptional and tumor suppressor responses</span></div><div class="casAuthors">Blalock, Eric M.; Geddes, James W.; Chen, Kuey Chu; Porter, Nada M.; Markesbery, William R.; Landfield, Philip W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2173-2178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The pathogenesis of incipient Alzheimer's disease (AD) has been resistant to anal. because of the complexity of AD and the overlap of its early-stage markers with normal aging.  Gene microarrays provide new tools for addressing complexity because they allow overviews of the simultaneous activity of multiple cellular pathways.  However, microarray data interpretation is often hindered by low statistical power, high false pos. or false negatives, and by uncertain relevance to functional end-points.  Here, the authors analyzed hippocampal gene expression of nine control and 22 AD subjects of varying severity on 31 sep. microarrays.  The authors then tested the correlation of each gene's expression with MiniMental Status Examn. (MMSE) and neurofibrillary tangle (NFT) scores across all 31 subjects regardless of diagnosis.  These well powered tests revealed a major transcriptional response comprising thousands of genes significantly correlated with AD markers.  Several hundred of these genes were also correlated with AD markers across only control and incipient AD subjects (MMSE > 20).  Biol. process categories assocd. with incipient AD-correlated genes were identified statistically (EASE program) and revealed up-regulation of many transcription factor/signaling genes regulating proliferation and differentiation, including tumor suppressors, oligodendrocyte growth factors, and protein kinase A modulators.  In addn., up-regulation of adhesion, apoptosis, lipid metab., and initial inflammation processes occurred, and down-regulation of protein folding/metab./transport and some energy metab. and signaling pathways took place.  These findings suggest a new model of AD pathogenesis in which a genomically orchestrated up-regulation of tumor suppressor-mediated differentiation and involution processes induces the spread of pathol. along myelinated axons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM0MvvQu295bVg90H21EOLACvtfcHk0lhcWTH5iarCkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1SrsL4%253D&md5=59ecee7b27eb4ceeb906a2e605fac284</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0308512100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0308512100%26sid%3Dliteratum%253Aachs%26aulast%3DBlalock%26aufirst%3DE.%2BM.%26aulast%3DGeddes%26aufirst%3DJ.%2BW.%26aulast%3DChen%26aufirst%3DK.%2BC.%26aulast%3DPorter%26aufirst%3DN.%2BM.%26aulast%3DMarkesbery%26aufirst%3DW.%2BR.%26aulast%3DLandfield%26aufirst%3DP.%2BW.%26atitle%3DIncipient%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520Microarray%2520correlation%2520analyses%2520reveal%2520major%2520transcriptional%2520and%2520tumor%2520suppressor%2520responses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D2173%26epage%3D2178%26doi%3D10.1073%2Fpnas.0308512100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brion, J. P.</span></span> <span> </span><span class="NLM_article-title">Increased level of active GSK-3beta in Alzheimer’s disease and accumulation in argyrophilic grains and in neurones at different stages of neurofibrillary degeneration</span>. <i>Neuropathol. Appl. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2990.2006.00795.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1111%2Fj.1365-2990.2006.00795.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17239007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlemuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2007&pages=43-55&author=K.+Leroyauthor=Z.+Yilmazauthor=J.+P.+Brion&title=Increased+level+of+active+GSK-3beta+in+Alzheimer%E2%80%99s+disease+and+accumulation+in+argyrophilic+grains+and+in+neurones+at+different+stages+of+neurofibrillary+degeneration&doi=10.1111%2Fj.1365-2990.2006.00795.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Increased level of active GSK-3β in Alzheimer's disease and accumulation in argyrophilic grains and in neurones at different stages of neurofibrillary degeneration</span></div><div class="casAuthors">Leroy, K.; Yilmaz, Z.; Brion, J.-P.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropathology and Applied Neurobiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-55</span>CODEN:
                <span class="NLM_cas:coden">NANEDL</span>;
        ISSN:<span class="NLM_cas:issn">0305-1846</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The somatodendritic accumulation of hyperphosphorylated tau proteins is an early event preceding the appearance of neurofibrillary tangles (NFT) in Alzheimer's disease (AD) and might be necessary for their formation.  Glycogen synthase kinase-3β (GSK-3β) is a physiol. kinase for tau that generates many tau phosphorylation sites identified in NFT and in other tau-pos. inclusions.  We have studied the cellular distribution and the expression of the active form of GSK-3β (GSK-3 pTyr216) in AD patients, in argyrophilic grain disease and in diffuse Lewy body disease.  By Western blotting anal., a significant increase in the level of GSK-3 (pTyr216) was obsd. in the frontal cortex of AD patients.  A population of neurons showed a somatodendritic accumulation of GSK-3 (pTyr216) but not of the inactive form of GSK-3β (GSK-3 pSer9).  Most of these GSK-3 (pTyr216)-pos. cells were pos. for six different phosphotau epitopes known to be generated by GSK-3β.  By using a quadruple labeling method using GSK-3 (pTyr216) and phosphotau immunolabelling combined with Gallyas and DAPI staining, we examd. neurons contg. a somatodendritic GSK-3 (pTyr216) immunoreactivity at different stages of neurodegeneration.  A majority of neurons at the pretangle stage without Gallyas-pos. inclusions were GSK-3 (pTyr216) pos. and this GSK-3 (pTyr216) immunoreactivity remained in most cells contg. Gallyas and phosphotau-pos. inclusions excepted in extracellular NFT.  A GSK-3 (pTyr216) immunoreactivity was present in argyrophilic grains but not in cortical Lewy bodies.  These results directly suggest that the activity of GSK-3β is increased in AD and that somatodendritic accumulation and activation of GSK-3β is an early event preceding and accompanying the formation of NFT and of other tau-pos. inclusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-g_D-oz0erLVg90H21EOLACvtfcHk0lg7I0qh6t5wJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlemuro%253D&md5=fe58bb24f90616c61baf51d0c2a05149</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2990.2006.00795.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2990.2006.00795.x%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DK.%26aulast%3DYilmaz%26aufirst%3DZ.%26aulast%3DBrion%26aufirst%3DJ.%2BP.%26atitle%3DIncreased%2520level%2520of%2520active%2520GSK-3beta%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%2520and%2520accumulation%2520in%2520argyrophilic%2520grains%2520and%2520in%2520neurones%2520at%2520different%2520stages%2520of%2520neurofibrillary%2520degeneration%26jtitle%3DNeuropathol.%2520Appl.%2520Neurobiol.%26date%3D2007%26volume%3D33%26spage%3D43%26epage%3D55%26doi%3D10.1111%2Fj.1365-2990.2006.00795.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kremer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaworski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Leuven, F.</span></span> <span> </span><span class="NLM_article-title">GSK3 and Alzheimer’s disease: Facts and fiction...</span>. <i>Front. Mol. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.3389/fnmol.2011.00017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.3389%2Ffnmol.2011.00017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=21904524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC38XovVKr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=17&author=A.+Kremerauthor=J.+V.+Louisauthor=T.+Jaworskiauthor=F.+Van+Leuven&title=GSK3+and+Alzheimer%E2%80%99s+disease%3A+Facts+and+fiction...&doi=10.3389%2Ffnmol.2011.00017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">GSK3 and Alzheimer's disease: facts and fiction</span></div><div class="casAuthors">Kremer, Anna; Louis, Justin V.; Jaworski, Tomasz; Van Leuven, Fred</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Molecular Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Aug.</span>),
    <span class="NLM_cas:pages">17</span>CODEN:
                <span class="NLM_cas:coden">FMNRAJ</span>;
        ISSN:<span class="NLM_cas:issn">1662-5099</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  The physiol. functions and pathol. roles of the Glycogen synthase kinase-type 3 (GSK3) kinases in peripheral and central systems are diverse and complex, and therefore hard to unravel in mol. detail in vivo.  Our assignment to review and discuss available data to clarify the actual position of these kinases in the pathol. of Alzheimer's dementia (AD) was both ambitious and easy.  On the one hand, numerous studies are available in isolated, recombinant, or cell-based systems, which have resulted in very diverse data-sets that are hardly informative for the brain in vivo.  At the other extreme, reliable, and relevant models for the role of GSK3 in CNS are rare, if not lacking.  Moreover, (too) many in vivo studies used Li+ as "specific" inhibitor of GSK3, which is factually not valid because lithium ions are neither specific nor potent inhibitors of GSK3 in vivo.  More specific pharmacol. inhibitors of GSK3 have met with considerable problems, and are reviewed by others in this issue or elsewhere.  We conc. here on AD-related aspects of GSK3 in brain in vivo, mainly studied in transgenic mice and highlight some of the more important issues, among many remaining: activation of GSK3 by amyloid, phosphorylation of protein tau, effects on or interference with synaptic activity, differentiation between both GSK3 isoforms.  These relate directly to brain function, and brain dysfunction in AD, and are to be resolved if we want to understand the mol. pathol. of this dreadful disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkyOq4ZInd37Vg90H21EOLACvtfcHk0lhb5-xSQht-Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovVKr&md5=ca9888444963da083e23df3c17a51521</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3389%2Ffnmol.2011.00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnmol.2011.00017%26sid%3Dliteratum%253Aachs%26aulast%3DKremer%26aufirst%3DA.%26aulast%3DLouis%26aufirst%3DJ.%2BV.%26aulast%3DJaworski%26aufirst%3DT.%26aulast%3DVan%2BLeuven%26aufirst%3DF.%26atitle%3DGSK3%2520and%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520Facts%2520and%2520fiction...%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2011%26volume%3D4%26spage%3D17%26doi%3D10.3389%2Ffnmol.2011.00017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, T. R.</span></span> <span> </span><span class="NLM_article-title">Glycogen synthase kinase-3 (GSK-3)-targeted therapy and imaging</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.7150/thno.14334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.7150%2Fthno.14334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26941849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC28XpvVertbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=571-593&author=M.+K.+Pandeyauthor=T.+R.+DeGrado&title=Glycogen+synthase+kinase-3+%28GSK-3%29-targeted+therapy+and+imaging&doi=10.7150%2Fthno.14334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Glycogen synthase kinase-3 (GSK-3)-targeted therapy and imaging</span></div><div class="casAuthors">Pandey, Mukesh K.; DeGrado, Timothy R.</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">571-593</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) is assocd. with various key biol. processes, including glucose regulation, apoptosis, protein synthesis, cell signaling, cellular transport, gene transcription, proliferation, and intracellular communication.  Accordingly, GSK-3 has been implicated in a wide variety of diseases and specifically targeted for both therapeutic and imaging applications by a large no. of academic labs. and pharmaceutical companies.  Here, we review the structure, function, expression levels, and ligand-binding properties of GSK-3 and its connection to various diseases.  A selected list of highly potent GSK-3 inhibitors, with IC50 <20 nM for ATP (ATP)-competitive inhibitors and IC50 <5 μM for non-ATP-competitive inhibitors, were analyzed for structure activity relationships.  Furthermore, ubiquitous expression of GSK-3 and its possible impact on therapy and imaging are also highlighted.  Finally, a rational perspective and possible route to selective and effective GSK-3 inhibitors is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg397DfBVXrrVg90H21EOLACvtfcHk0lhb5-xSQht-Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpvVertbs%253D&md5=e66feec1b1b2f460000116b722678478</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.7150%2Fthno.14334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.14334%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DM.%2BK.%26aulast%3DDeGrado%26aufirst%3DT.%2BR.%26atitle%3DGlycogen%2520synthase%2520kinase-3%2520%2528GSK-3%2529-targeted%2520therapy%2520and%2520imaging%26jtitle%3DTheranostics%26date%3D2016%26volume%3D6%26spage%3D571%26epage%3D593%26doi%3D10.7150%2Fthno.14334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goedert, M.</span></span> <span> </span><span class="NLM_article-title">GSK3 inhibitors: development and therapeutic potential</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1038/nrd1415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrd1415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=15173837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVaqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=479-487&author=P.+Cohenauthor=M.+Goedert&title=GSK3+inhibitors%3A+development+and+therapeutic+potential&doi=10.1038%2Fnrd1415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">GSK3 inhibitors: development and therapeutic potential</span></div><div class="casAuthors">Cohen, Philip; Goedert, Michel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">479-487</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Glycogen synthase kinase-3 (GSK3) was initially identified more than two decades ago as an enzyme involved in the control of glycogen metab.  In recent years it has been shown to have key roles in regulating a diverse range of cellular functions, which have prompted efforts to develop GSK3 inhibitors as therapeutics.  Here, we describe the biol. of GSK3 relevant to its potential as a target for diabetes and neurodegenerative diseases, and discuss progress in the development of GSK3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL3ua5xi0_qrVg90H21EOLACvtfcHk0lgWgG1zfG__UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVaqsrs%253D&md5=165d1354bf11faa6021ac4dba797fda2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd1415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1415%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DGoedert%26aufirst%3DM.%26atitle%3DGSK3%2520inhibitors%253A%2520development%2520and%2520therapeutic%2520potential%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D479%26epage%3D487%26doi%3D10.1038%2Fnrd1415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketterman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weïwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germain, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nag, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbe, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzaro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacher, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheah, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holson, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J. Q.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Glycogen Synthase Kinase 3 with Exquisite Kinome-Wide Selectivity and Their Functional Effects</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1952</span>– <span class="NLM_lpage">1963</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00306</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00306" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFemtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1952-1963&author=F.+F.+Wagnerauthor=J.+A.+Bishopauthor=J.+P.+Galeauthor=X.+Shiauthor=M.+Walkauthor=J.+Kettermanauthor=D.+Patnaikauthor=D.+Barkerauthor=D.+Walpitaauthor=A.+J.+Campbellauthor=S.+Nguyenauthor=M.+Lewisauthor=L.+Rossauthor=M.+We%C3%AFwerauthor=W.+F.+Anauthor=A.+R.+Germainauthor=P.+P.+Nagauthor=S.+Metkarauthor=T.+Kayaauthor=S.+Dandapaniauthor=D.+E.+Olsonauthor=A.-L.+Barbeauthor=F.+Lazzaroauthor=J.+R.+Sacherauthor=J.+H.+Cheahauthor=D.+Feiauthor=J.+Perezauthor=B.+Munozauthor=M.+Palmerauthor=K.+Stegmaierauthor=S.+L.+Schreiberauthor=E.+Scolnickauthor=Y.-L.+Zhangauthor=S.+J.+Haggartyauthor=E.+B.+Holsonauthor=J.+Q.+Pan&title=Inhibitors+of+Glycogen+Synthase+Kinase+3+with+Exquisite+Kinome-Wide+Selectivity+and+Their+Functional+Effects&doi=10.1021%2Facschembio.6b00306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Glycogen Synthase Kinase 3 with Exquisite Kinome-Wide Selectivity and Their Functional Effects</span></div><div class="casAuthors">Wagner, Florence F.; Bishop, Joshua A.; Gale, Jennifer P.; Shi, Xi; Walk, Michelle; Ketterman, Joshua; Patnaik, Debasis; Barker, Doug; Walpita, Deepika; Campbell, Arthur J.; Nguyen, Shannon; Lewis, Michael; Ross, Linda; Weiwer, Michel; An, W. Frank; Germain, Andrew R.; Nag, Partha P.; Metkar, Shailesh; Kaya, Taner; Dandapani, Sivaraman; Olson, David E.; Barbe, Anne-Laure; Lazzaro, Fanny; Sacher, Joshua R.; Cheah, Jaime H.; Fei, David; Perez, Jose; Munoz, Benito; Palmer, Michelle; Stegmaier, Kimberly; Schreiber, Stuart L.; Scolnick, Edward; Zhang, Yan-Ling; Haggarty, Stephen J.; Holson, Edward B.; Pan, Jen Q.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1952-1963</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mood stabilizer lithium, the first-line treatment for bipolar disorder, is hypothesized to exert its effects through direct inhibition of glycogen synthase kinase 3 (GSK3) and indirectly by increasing GSK3's inhibitory serine phosphorylation.  GSK3 comprises two highly similar paralogs, GSK3α and GSK3β, which are key regulatory kinases in the canonical Wnt pathway.  GSK3 stands as a nodal target within this pathway and is an attractive therapeutic target for multiple indications.  Despite being an active field of research for the past 20 years, many GSK3 inhibitors demonstrate either poor to moderate selectivity vs. the broader human kinome or physicochem. properties unsuitable for use in in vitro systems or in vivo models.  A nonconventional anal. of data from a GSK3β inhibitor high-throughput screening campaign, which excluded known GSK3 inhibitor chemotypes, led to the discovery of a novel pyrazolo-tetrahydroquinolinone scaffold with unparalleled kinome-wide selectivity for the GSK3 kinases.  Taking advantage of an uncommon tridentate interaction with the hinge region of GSK3, we developed highly selective and potent GSK3 inhibitors, BRD1652 and BRD0209, which demonstrated in vivo efficacy in a dopaminergic signaling paradigm modeling mood-related disorders.  These new chem. probes open the way for exclusive analyses of the function of GSK3 kinases in multiple signaling pathways involved in many prevalent disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt6ZO9St9MGLVg90H21EOLACvtfcHk0lhT7C8Fvbg6Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFemtr7I&md5=2cfb60c9c1936e5ef9986842aedf473f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00306%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DBishop%26aufirst%3DJ.%2BA.%26aulast%3DGale%26aufirst%3DJ.%2BP.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWalk%26aufirst%3DM.%26aulast%3DKetterman%26aufirst%3DJ.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DBarker%26aufirst%3DD.%26aulast%3DWalpita%26aufirst%3DD.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DNguyen%26aufirst%3DS.%26aulast%3DLewis%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DL.%26aulast%3DWe%25C3%25AFwer%26aufirst%3DM.%26aulast%3DAn%26aufirst%3DW.%2BF.%26aulast%3DGermain%26aufirst%3DA.%2BR.%26aulast%3DNag%26aufirst%3DP.%2BP.%26aulast%3DMetkar%26aufirst%3DS.%26aulast%3DKaya%26aufirst%3DT.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DOlson%26aufirst%3DD.%2BE.%26aulast%3DBarbe%26aufirst%3DA.-L.%26aulast%3DLazzaro%26aufirst%3DF.%26aulast%3DSacher%26aufirst%3DJ.%2BR.%26aulast%3DCheah%26aufirst%3DJ.%2BH.%26aulast%3DFei%26aufirst%3DD.%26aulast%3DPerez%26aufirst%3DJ.%26aulast%3DMunoz%26aufirst%3DB.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DScolnick%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DHolson%26aufirst%3DE.%2BB.%26aulast%3DPan%26aufirst%3DJ.%2BQ.%26atitle%3DInhibitors%2520of%2520Glycogen%2520Synthase%2520Kinase%25203%2520with%2520Exquisite%2520Kinome-Wide%2520Selectivity%2520and%2520Their%2520Functional%2520Effects%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1952%26epage%3D1963%26doi%3D10.1021%2Facschembio.6b00306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normandin, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Para, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopresti-Morrow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanyon, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, K. E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schachter, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enerson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guehl, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holson, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Fakhri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumbail, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasdev, N.</span></span> <span> </span><span class="NLM_article-title">Discovery of a highly selective glycogen synthase kinase-3 inhibitor (PF-04802367) that modulates tau phosphorylation in the brain: translation for PET neuroimaging</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9601</span>– <span class="NLM_lpage">9605</span>, <span class="refDoi"> DOI: 10.1002/anie.201603797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2Fanie.201603797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=27355874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A280%3ADC%252BC2s7kslKhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=9601-9605&author=S.+H.+Liangauthor=J.+M.+Chenauthor=M.+D.+Normandinauthor=J.+S.+Changauthor=G.+C.+Changauthor=C.+K.+Taylorauthor=P.+Trapaauthor=M.+S.+Plummerauthor=K.+S.+Paraauthor=E.+L.+Connauthor=L.+Lopresti-Morrowauthor=L.+F.+Lanyonauthor=J.+M.+Cookauthor=K.+E.+G.+Richterauthor=C.+E.+Nolanauthor=J.+B.+Schachterauthor=F.+Janatauthor=Y.+Cheauthor=V.+Shanmugasundaramauthor=B.+A.+Lefkerauthor=B.+E.+Enersonauthor=E.+Livniauthor=L.+Wangauthor=N.+J.+Guehlauthor=D.+Patnaikauthor=F.+F.+Wagnerauthor=R.+Perlisauthor=E.+B.+Holsonauthor=S.+J.+Haggartyauthor=G.+El+Fakhriauthor=R.+G.+Kurumbailauthor=N.+Vasdev&title=Discovery+of+a+highly+selective+glycogen+synthase+kinase-3+inhibitor+%28PF-04802367%29+that+modulates+tau+phosphorylation+in+the+brain%3A+translation+for+PET+neuroimaging&doi=10.1002%2Fanie.201603797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Highly Selective Glycogen Synthase Kinase-3 Inhibitor (PF-04802367) That Modulates Tau Phosphorylation in the Brain: Translation for PET Neuroimaging</span></div><div class="casAuthors">Liang Steven H; Normandin Marc D; Livni Elijahu; Wang Lu; Guehl Nicolas J; El Fakhri Georges; Vasdev Neil; Chen Jinshan Michael; Chang Jeanne S; Chang George C; Taylor Christine K; Plummer Mark S; Para Kimberly S; Conn Edward L; Lopresti-Morrow Lori; Lanyon Lorraine F; Cook James M; Richter Karl E G; Nolan Charlie E; Schachter Joel B; Janat Fouad; Che Ye; Shanmugasundaram Veerabahu; Enerson Bradley E; Kurumbail Ravi G; Trapa Patrick; Lefker Bruce A; Patnaik Debasis; Perlis Roy; Haggarty Stephen J; Wagner Florence F; Perlis Roy; Holson Edward B</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">9601-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) regulates multiple cellular processes in diabetes, oncology, and neurology.  N-(3-(1H-1,2,4-triazol-1-yl)propyl)-5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxamide (PF-04802367 or PF-367) has been identified as a highly potent inhibitor, which is among the most selective antagonists of GSK-3 to date.  Its efficacy was demonstrated in modulation of tau phosphorylation in vitro and in vivo.  Whereas the kinetics of PF-367 binding in brain tissues are too fast for an effective therapeutic agent, the pharmacokinetic profile of PF-367 is ideal for discovery of radiopharmaceuticals for GSK-3 in the central nervous system.  A (11) C-isotopologue of PF-367 was synthesized and preliminary PET imaging studies in non-human primates confirmed that we have overcome the two major obstacles for imaging GSK-3, namely, reasonable brain permeability and displaceable binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBs_RxQsDg6XELvajU9pjPfW6udTcc2eb7NYAu2veiNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7kslKhtA%253D%253D&md5=ef5bafe40489bc7ab34c29988b38c9ec</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603797%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DS.%2BH.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DNormandin%26aufirst%3DM.%2BD.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DChang%26aufirst%3DG.%2BC.%26aulast%3DTaylor%26aufirst%3DC.%2BK.%26aulast%3DTrapa%26aufirst%3DP.%26aulast%3DPlummer%26aufirst%3DM.%2BS.%26aulast%3DPara%26aufirst%3DK.%2BS.%26aulast%3DConn%26aufirst%3DE.%2BL.%26aulast%3DLopresti-Morrow%26aufirst%3DL.%26aulast%3DLanyon%26aufirst%3DL.%2BF.%26aulast%3DCook%26aufirst%3DJ.%2BM.%26aulast%3DRichter%26aufirst%3DK.%2BE.%2BG.%26aulast%3DNolan%26aufirst%3DC.%2BE.%26aulast%3DSchachter%26aufirst%3DJ.%2BB.%26aulast%3DJanat%26aufirst%3DF.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DLefker%26aufirst%3DB.%2BA.%26aulast%3DEnerson%26aufirst%3DB.%2BE.%26aulast%3DLivni%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DGuehl%26aufirst%3DN.%2BJ.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DPerlis%26aufirst%3DR.%26aulast%3DHolson%26aufirst%3DE.%2BB.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DEl%2BFakhri%26aufirst%3DG.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DVasdev%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520a%2520highly%2520selective%2520glycogen%2520synthase%2520kinase-3%2520inhibitor%2520%2528PF-04802367%2529%2520that%2520modulates%2520tau%2520phosphorylation%2520in%2520the%2520brain%253A%2520translation%2520for%2520PET%2520neuroimaging%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D9601%26epage%3D9605%26doi%3D10.1002%2Fanie.201603797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snow, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raybon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pokross, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kish, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubowchik, G. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of isonicotinamides as highly selective, brain penetrable, and orally active glycogen synthase kinase-3 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1041</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01550</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01550" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmslertg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1041-1051&author=G.+Luoauthor=L.+Chenauthor=C.+R.+Burtonauthor=H.+Xiaoauthor=P.+Sivaprakasamauthor=C.+M.+Krauseauthor=Y.+Caoauthor=N.+Liuauthor=J.+Lippyauthor=W.+J.+Clarkeauthor=K.+Snowauthor=J.+Raybonauthor=V.+Aroraauthor=M.+Pokrossauthor=K.+Kishauthor=H.+A.+Lewisauthor=D.+R.+Langleyauthor=J.+E.+Macorauthor=G.+M.+Dubowchik&title=Discovery+of+isonicotinamides+as+highly+selective%2C+brain+penetrable%2C+and+orally+active+glycogen+synthase+kinase-3+inhibitors&doi=10.1021%2Facs.jmedchem.5b01550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Isonicotinamides as Highly Selective, Brain Penetrable, and Orally Active Glycogen Synthase Kinase-3 Inhibitors</span></div><div class="casAuthors">Luo, Guanglin; Chen, Ling; Burton, Catherine R.; Xiao, Hong; Sivaprakasam, Prasanna; Krause, Carol M.; Cao, Yang; Liu, Nengyin; Lippy, Jonathan; Clarke, Wendy J.; Snow, Kimberly; Raybon, Joseph; Arora, Vinod; Pokross, Matt; Kish, Kevin; Lewis, Hal A.; Langley, David R.; Macor, John E.; Dubowchik, Gene M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1041-1051</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GSK-3 is a serine/threonine kinase that has numerous substrates.  Many of these proteins are involved in the regulation of diverse cellular functions, including metab., differentiation, proliferation, and apoptosis.  Inhibition of GSK-3 may be useful in treating a no. of diseases including Alzheimer's disease (AD), type II diabetes, mood disorders, and some cancers, but the approach poses significant challenges.  Here, the authors present a class of isonicotinamides, e.g. I [R = F, Cl] that are potent, highly kinase-selective GSK-3 inhibitors, the members of which demonstrated oral activity in a triple-transgenic mouse model of AD.  The remarkably high kinase selectivity and straightforward synthesis of these compds. bode well for their further exploration as tool compds. and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaIEil5d6i8rVg90H21EOLACvtfcHk0lg79KL1G-Gi-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmslertg%253D%253D&md5=d0412c3fe9b430c59f861fa10d3eff79</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01550%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DBurton%26aufirst%3DC.%2BR.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DSivaprakasam%26aufirst%3DP.%26aulast%3DKrause%26aufirst%3DC.%2BM.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DLippy%26aufirst%3DJ.%26aulast%3DClarke%26aufirst%3DW.%2BJ.%26aulast%3DSnow%26aufirst%3DK.%26aulast%3DRaybon%26aufirst%3DJ.%26aulast%3DArora%26aufirst%3DV.%26aulast%3DPokross%26aufirst%3DM.%26aulast%3DKish%26aufirst%3DK.%26aulast%3DLewis%26aufirst%3DH.%2BA.%26aulast%3DLangley%26aufirst%3DD.%2BR.%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26aulast%3DDubowchik%26aufirst%3DG.%2BM.%26atitle%3DDiscovery%2520of%2520isonicotinamides%2520as%2520highly%2520selective%252C%2520brain%2520penetrable%252C%2520and%2520orally%2520active%2520glycogen%2520synthase%2520kinase-3%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1041%26epage%3D1051%26doi%3D10.1021%2Facs.jmedchem.5b01550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastassiadis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span> <span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0ljP0I96yg5L4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljP0I96yg5L4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benajiba, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weïwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacher, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puissant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Back, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galinsky, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermine, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemann, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holson, E. B.</span></span> <span> </span><span class="NLM_article-title">Exploiting an Asp-Glu “switch” in glycogen synthase kinase 3 to design paralog-selective inhibitors for use in acute myeloid leukemia</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">eaam8460</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aam8460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1126%2Fscitranslmed.aam8460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=29515000" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&author=F.+F.+Wagnerauthor=L.+Benajibaauthor=A.+J.+Campbellauthor=M.+We%C3%AFwerauthor=J.+R.+Sacherauthor=J.+P.+Galeauthor=L.+Rossauthor=A.+Puissantauthor=G.+Alexeauthor=A.+Conwayauthor=M.+Backauthor=Y.+Pikmanauthor=I.+Galinskyauthor=D.+J.+DeAngeloauthor=R.+M.+Stoneauthor=T.+Kayaauthor=X.+Shiauthor=M.+B.+Robersauthor=T.+Machleidtauthor=J.+Wilkinsonauthor=O.+Hermineauthor=A.+Kungauthor=A.+J.+Steinauthor=D.+Lakshminarasimhanauthor=M.+T.+Hemannauthor=E.+Scolnickauthor=Y.-L.+Zhangauthor=J.+Q.+Panauthor=K.+Stegmaierauthor=E.+B.+Holson&title=Exploiting+an+Asp-Glu+%E2%80%9Cswitch%E2%80%9D+in+glycogen+synthase+kinase+3+to+design+paralog-selective+inhibitors+for+use+in+acute+myeloid+leukemia&doi=10.1126%2Fscitranslmed.aam8460"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aam8460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aam8460%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DBenajiba%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DWe%25C3%25AFwer%26aufirst%3DM.%26aulast%3DSacher%26aufirst%3DJ.%2BR.%26aulast%3DGale%26aufirst%3DJ.%2BP.%26aulast%3DRoss%26aufirst%3DL.%26aulast%3DPuissant%26aufirst%3DA.%26aulast%3DAlexe%26aufirst%3DG.%26aulast%3DConway%26aufirst%3DA.%26aulast%3DBack%26aufirst%3DM.%26aulast%3DPikman%26aufirst%3DY.%26aulast%3DGalinsky%26aufirst%3DI.%26aulast%3DDeAngelo%26aufirst%3DD.%2BJ.%26aulast%3DStone%26aufirst%3DR.%2BM.%26aulast%3DKaya%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DHermine%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DA.%2BJ.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DHemann%26aufirst%3DM.%2BT.%26aulast%3DScolnick%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DPan%26aufirst%3DJ.%2BQ.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DHolson%26aufirst%3DE.%2BB.%26atitle%3DExploiting%2520an%2520Asp-Glu%2520%25E2%2580%259Cswitch%25E2%2580%259D%2520in%2520glycogen%2520synthase%2520kinase%25203%2520to%2520design%2520paralog-selective%2520inhibitors%2520for%2520use%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2018%26volume%3D10%26doi%3D10.1126%2Fscitranslmed.aam8460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergner, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyanova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannaiyan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manrique-Hoyos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kongi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantuti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanisch, U.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">Cell type- and brain region-resolved mouse brain proteome</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1819</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1038/nn.4160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnn.4160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26523646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1819-1831&author=K.+Sharmaauthor=S.+Schmittauthor=C.+G.+Bergnerauthor=S.+Tyanovaauthor=N.+Kannaiyanauthor=N.+Manrique-Hoyosauthor=K.+Kongiauthor=L.+Cantutiauthor=U.-K.+Hanischauthor=M.-A.+Philipsauthor=M.+J.+Rossnerauthor=M.+Mannauthor=M.+Simons&title=Cell+type-+and+brain+region-resolved+mouse+brain+proteome&doi=10.1038%2Fnn.4160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cell type- and brain region-resolved mouse brain proteome</span></div><div class="casAuthors">Sharma, Kirti; Schmitt, Sebastian; Bergner, Caroline G.; Tyanova, Stefka; Kannaiyan, Nirmal; Manrique-Hoyos, Natalia; Kongi, Karina; Cantuti, Ludovico; Hanisch, Uwe-Karsten; Philips, Mari-Anne; Rossner, Moritz J.; Mann, Matthias; Simons, Mikael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1819-1831</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Brain transcriptome and connectome maps are being generated, but an equiv. effort on the proteome is currently lacking.  We performed high-resoln. mass spectrometry-based proteomics for in-depth anal. of the mouse brain and its major brain regions and cell types.  Comparisons of the 12,934 identified proteins in oligodendrocytes, astrocytes, microglia and cortical neurons with deep sequencing data of the transcriptome indicated deep coverage of the proteome.  Cell type-specific proteins defined as tenfold more abundant than av. expression represented about a tenth of the proteome, with an overrepresentation of cell surface proteins.  To demonstrate the utility of our resource, we focused on this class of proteins and identified Lsamp, an adhesion mol. of the IgLON family, as a neg. regulator of myelination.  Our findings provide a framework for a system-level understanding of cell-type diversity in the CNS and serves as a rich resource for analyses of brain development and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHhWGHaIaSYrVg90H21EOLACvtfcHk0lh8gd3zIEliaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7bI&md5=30caea646f82e42968f84275ac17c106</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnn.4160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4160%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DK.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DBergner%26aufirst%3DC.%2BG.%26aulast%3DTyanova%26aufirst%3DS.%26aulast%3DKannaiyan%26aufirst%3DN.%26aulast%3DManrique-Hoyos%26aufirst%3DN.%26aulast%3DKongi%26aufirst%3DK.%26aulast%3DCantuti%26aufirst%3DL.%26aulast%3DHanisch%26aufirst%3DU.-K.%26aulast%3DPhilips%26aufirst%3DM.-A.%26aulast%3DRossner%26aufirst%3DM.%2BJ.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DCell%2520type-%2520and%2520brain%2520region-resolved%2520mouse%2520brain%2520proteome%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D1819%26epage%3D1831%26doi%3D10.1038%2Fnn.4160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorokina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S. G. N.</span></span> <span> </span><span class="NLM_article-title">Proteomic analysis of postsynaptic proteins in regions of the human neocortex</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1038/s41593-017-0025-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fs41593-017-0025-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=29203896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVyitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2018&pages=130-138&author=M.+Royauthor=O.+Sorokinaauthor=N.+Skeneauthor=C.+Simonnetauthor=F.+Mazzoauthor=R.+Zwartauthor=E.+Sherauthor=C.+Smithauthor=J.+D.+Armstrongauthor=S.+G.+N.+Grant&title=Proteomic+analysis+of+postsynaptic+proteins+in+regions+of+the+human+neocortex&doi=10.1038%2Fs41593-017-0025-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analysis of postsynaptic proteins in regions of the human neocortex</span></div><div class="casAuthors">Roy, Marcia; Sorokina, Oksana; Skene, Nathan; Simonnet, Clemence; Mazzo, Francesca; Zwart, Ruud; Sher, Emanuele; Smith, Colin; Armstrong, J. Douglas; Grant, Seth G. N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">130-138</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The postsynaptic proteome of excitatory synapses comprises ∼1,000 highly conserved proteins that control the behavioral repertoire, and mutations disrupting their function cause >130 brain diseases.  In this document the compn. of postsynaptic proteomes in human neocortical regions and integrate it with genetic, functional and structural magnetic resonance imaging, positron emission tomog. imaging, and behavioral data.  Neocortical regions show signatures of expression of individual proteins, protein complexes, biochem. and metabolic pathways.  This characterized the compositional signatures in brain regions involved with language, emotion and memory functions.  Integrating large-scale GWAS with regional proteome data identifies the same cortical region for smoking behavior as found with fMRI data.  The neocortical postsynaptic proteome data resource can be used to link genetics to brain imaging and behavior, and to study the role of postsynaptic proteins in localization of brain functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE7Af1gt2EVLVg90H21EOLACvtfcHk0lh8gd3zIEliaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVyitb4%253D&md5=065cb3a291a830866065e63addceddc5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41593-017-0025-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-017-0025-9%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%26aulast%3DSorokina%26aufirst%3DO.%26aulast%3DSkene%26aufirst%3DN.%26aulast%3DSimonnet%26aufirst%3DC.%26aulast%3DMazzo%26aufirst%3DF.%26aulast%3DZwart%26aufirst%3DR.%26aulast%3DSher%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DC.%26aulast%3DArmstrong%26aufirst%3DJ.%2BD.%26aulast%3DGrant%26aufirst%3DS.%2BG.%2BN.%26atitle%3DProteomic%2520analysis%2520of%2520postsynaptic%2520proteins%2520in%2520regions%2520of%2520the%2520human%2520neocortex%26jtitle%3DNat.%2520Neurosci.%26date%3D2018%26volume%3D21%26spage%3D130%26epage%3D138%26doi%3D10.1038%2Fs41593-017-0025-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeusen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swafford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursalaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deaton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Pinzon, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hering, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin Seidel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glynne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of DYRK1A and GSK3B induces human β-cell proliferation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8372</span>, <span class="refDoi"> DOI: 10.1038/ncomms9372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fncomms9372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26496802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8372&author=W.+Shenauthor=B.+Taylorauthor=Q.+Jinauthor=V.+Nguyen-Tranauthor=S.+Meeusenauthor=Y.-Q.+Zhangauthor=A.+Kamireddyauthor=A.+Swaffordauthor=A.+F.+Powersauthor=J.+Walkerauthor=J.+Lambauthor=B.+Bursalayaauthor=M.+DiDonatoauthor=G.+Harbauthor=M.+Qiuauthor=C.+M.+Filippiauthor=L.+Deatonauthor=C.+N.+Turkauthor=W.+L.+Suarez-Pinzonauthor=Y.+Liuauthor=X.+Haoauthor=T.+Moauthor=S.+Yanauthor=J.+Liauthor=A.+E.+Hermanauthor=B.+J.+Heringauthor=T.+Wuauthor=H.+Martin+Seidelauthor=P.+McNamaraauthor=R.+Glynneauthor=B.+Laffitte&title=Inhibition+of+DYRK1A+and+GSK3B+induces+human+%CE%B2-cell+proliferation&doi=10.1038%2Fncomms9372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A and GSK3B induces human β-cell proliferation</span></div><div class="casAuthors">Shen, Weijun; Taylor, Brandon; Jin, Qihui; Nguyen-Tran, Van; Meeusen, Shelly; Zhang, You-Qing; Kamireddy, Anwesh; Swafford, Austin; Powers, Andrew F.; Walker, John; Lamb, John; Bursalaya, Badry; DiDonato, Michael; Harb, George; Qiu, Minhua; Filippi, Christophe M.; Deaton, Lisa; Turk, Carolina N.; Suarez-Pinzon, Wilma L.; Liu, Yahu; Hao, Xueshi; Mo, Tingting; Yan, Shanshan; Li, Jing; Herman, Ann E.; Hering, Bernhard J.; Wu, Tom; Seidel, H. Martin; McNamara, Peter; Glynne, Richard; Laffitte, Bryan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8372</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insufficient pancreatic β-cell mass or function results in diabetes mellitus.  While significant progress has been made in regulating insulin secretion from β-cells in diabetic patients, no pharmacol. agents have been described that increase β-cell replication in humans.  Here we report aminopyrazine compds. that stimulate robust β-cell proliferation in adult primary islets, most likely as a result of combined inhibition of DYRK1A and GSK3B.  Aminopyrazine-treated human islets retain functionality in vitro and after transplantation into diabetic mice.  Oral dosing of these compds. in diabetic mice induces β-cell proliferation, increases β-cell mass and insulin content, and improves glycemic control.  Biochem., genetic and cell biol. data point to Dyrk1a as the key mol. target.  This study supports the feasibility of treating diabetes with an oral therapy to restore β-cell mass, and highlights a tractable pathway for future drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVHzP-VkH5nrVg90H21EOLACvtfcHk0ljmFayS2smVKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK&md5=5ef960af3bfd2795b9fdce52876e5d59</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fncomms9372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9372%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTaylor%26aufirst%3DB.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DNguyen-Tran%26aufirst%3DV.%26aulast%3DMeeusen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.-Q.%26aulast%3DKamireddy%26aufirst%3DA.%26aulast%3DSwafford%26aufirst%3DA.%26aulast%3DPowers%26aufirst%3DA.%2BF.%26aulast%3DWalker%26aufirst%3DJ.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DBursalaya%26aufirst%3DB.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DHarb%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DM.%26aulast%3DFilippi%26aufirst%3DC.%2BM.%26aulast%3DDeaton%26aufirst%3DL.%26aulast%3DTurk%26aufirst%3DC.%2BN.%26aulast%3DSuarez-Pinzon%26aufirst%3DW.%2BL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHerman%26aufirst%3DA.%2BE.%26aulast%3DHering%26aufirst%3DB.%2BJ.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DMartin%2BSeidel%26aufirst%3DH.%26aulast%3DMcNamara%26aufirst%3DP.%26aulast%3DGlynne%26aufirst%3DR.%26aulast%3DLaffitte%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520DYRK1A%2520and%2520GSK3B%2520induces%2520human%2520%25CE%25B2-cell%2520proliferation%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8372%26doi%3D10.1038%2Fncomms9372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartoschik, T.</span>; <span class="NLM_string-name">Maschberger, M.</span>; <span class="NLM_string-name">Feoli, A.</span>; <span class="NLM_string-name">André, T.</span>; <span class="NLM_string-name">Baaske, P.</span>; <span class="NLM_string-name">Duhr, S.</span>; <span class="NLM_string-name">Breitsprecher, D.</span></span> <span> </span><span class="NLM_article-title">Microscale Thermophoresis in Drug Discovery</span>.  <i>Applied Biophysics for Drug Discovery</i>; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">92</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2F9781119099512.ch5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=73-92&author=T.+Bartoschik&author=M.+Maschberger&author=A.+Feoli&author=T.+Andr%C3%A9&author=P.+Baaske&author=S.+Duhr&author=D.+Breitsprecher&title=Applied+Biophysics+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2F9781119099512.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781119099512.ch5%26sid%3Dliteratum%253Aachs%26aulast%3DBartoschik%26aufirst%3DT.%26atitle%3DMicroscale%2520Thermophoresis%2520in%2520Drug%2520Discovery%26btitle%3DApplied%2520Biophysics%2520for%2520Drug%2520Discovery%26pub%3DWiley%26date%3D2017%26spage%3D73%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stamos, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enos, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span> <span> </span><span class="NLM_article-title">Structural basis of GSK-3 inhibition by N-terminal phosphorylation and by the Wnt receptor LRP6</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>, <span class="NLM_elocation-id">e01998</span> <span class="refDoi"> DOI: 10.7554/eLife.01998</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.7554%2FeLife.01998" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&author=J.+L.+Stamosauthor=M.+L.+Chuauthor=M.+D.+Enosauthor=N.+Shahauthor=W.+I.+Weis&title=Structural+basis+of+GSK-3+inhibition+by+N-terminal+phosphorylation+and+by+the+Wnt+receptor+LRP6&doi=10.7554%2FeLife.01998"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7554%2FeLife.01998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01998%26sid%3Dliteratum%253Aachs%26aulast%3DStamos%26aufirst%3DJ.%2BL.%26aulast%3DChu%26aufirst%3DM.%2BL.%26aulast%3DEnos%26aufirst%3DM.%2BD.%26aulast%3DShah%26aufirst%3DN.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26atitle%3DStructural%2520basis%2520of%2520GSK-3%2520inhibition%2520by%2520N-terminal%2520phosphorylation%2520and%2520by%2520the%2520Wnt%2520receptor%2520LRP6%26jtitle%3DeLife%26date%3D2014%26volume%3D18%26doi%3D10.7554%2FeLife.01998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C.</span>; <span class="NLM_string-name">Fass, D.
M.</span>; <span class="NLM_string-name">Folz-Donahue, K.</span>; <span class="NLM_string-name">MacDonald, M. E.</span>; <span class="NLM_string-name">Haggarty, S. J.</span></span> <span> </span><span class="NLM_article-title">Highly Expandable Human iPS Cell-Derived Neural Progenitor Cells (NPC) and Neurons for Central Nervous System Disease Modeling and High-Throughput Screening</span>.  <i>Curr. Protoc. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume">92</span>,  <span class="NLM_lpage">21.8.21</span>.<span class="refDoi"> DOI: 10.1002/cphg.33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2Fcphg.33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=28075486" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=-21.8.21&author=C.+Cheng&author=D.%0AM.+Fass&author=K.+Folz-Donahue&author=M.+E.+MacDonald&author=S.+J.+Haggarty&title=Highly+Expandable+Human+iPS+Cell-Derived+Neural+Progenitor+Cells+%28NPC%29+and+Neurons+for+Central+Nervous+System+Disease+Modeling+and+High-Throughput+Screening&doi=10.1002%2Fcphg.33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fcphg.33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcphg.33%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%26atitle%3DHighly%2520Expandable%2520Human%2520iPS%2520Cell-Derived%2520Neural%2520Progenitor%2520Cells%2520%2528NPC%2529%2520and%2520Neurons%2520for%2520Central%2520Nervous%2520System%2520Disease%2520Modeling%2520and%2520High-Throughput%2520Screening%26jtitle%3DCurr.%2520Protoc.%2520Hum.%2520Genet.%26date%3D2017%26volume%3D92%26epage%3D21.8.21%26doi%3D10.1002%2Fcphg.33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span>; <span class="NLM_string-name">Haggerty, S. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Functional Lithium Mimetics Affecting CRMP2 Phosphorylation in Human iPSC-Derived Neurons</span>. Unpublished results, <span class="NLM_year" style="font-weight: bold;">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=W.+N.+Zhao&author=S.+J.+Haggerty&title=Discovery+of+Functional+Lithium+Mimetics+Affecting+CRMP2+Phosphorylation+in+Human+iPSC-Derived+Neurons"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520of%2520Functional%2520Lithium%2520Mimetics%2520Affecting%2520CRMP2%2520Phosphorylation%2520in%2520Human%2520iPSC-Derived%2520Neurons%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikuchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaibuchi, K.</span></span> <span> </span><span class="NLM_article-title">GSK-3β Regulates Phosphorylation of CRMP-2 and Neuronal Polarity</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.cell.2004.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=15652488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2MXot1CgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2005&pages=137-149&author=T.+Yoshimuraauthor=Y.+Kawanoauthor=N.+Arimuraauthor=S.+Kawabataauthor=A.+Kikuchiauthor=K.+Kaibuchi&title=GSK-3%CE%B2+Regulates+Phosphorylation+of+CRMP-2+and+Neuronal+Polarity&doi=10.1016%2Fj.cell.2004.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">GSK-3β regulates phosphorylation of CRMP-2 and neuronal polarity</span></div><div class="casAuthors">Yoshimura, Takeshi; Kawano, Yoji; Arimura, Nariko; Kawabata, Saeko; Kikuchi, Akira; Kaibuchi, Kozo</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">137-149</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Neurons are highly polarized and comprised of two structurally and functionally distinct parts, an axon and dendrites.  We previously showed that collapsin response mediator protein-2 (CRMP-2) is crit. for specifying axon/dendrite fate, possibly by promoting neurite elongation via microtubule assembly.  Here, we showed that glycogen synthase kinase-3β (GSK-3β) phosphorylated CRMP-2 at Thr-514 and inactivated it.  The expression of the nonphosphorylated form of CRMP-2 or inhibition of GSK-3β induced the formation of multiple axon-like neurites in hippocampal neurons.  The expression of constitutively active GSK-3β impaired neuronal polarization, whereas the nonphosphorylated form of CRMP-2 counteracted the inhibitory effects of GSK-3β, indicating that GSK-3β regulates neuronal polarity through the phosphorylation of CRMP-2.  Treatment of hippocampal neurons with neurotrophin-3 (NT-3) induced inactivation of GSK-3β and dephosphorylation of CRMP-2.  Knockdown of CRMP-2 inhibited NT-3-induced axon outgrowth.  These results suggest that NT-3 decreases phosphorylated CRMP-2 and increases nonphosphorylated active CRMP-2, thereby promoting axon outgrowth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSzNEmMU885bVg90H21EOLACvtfcHk0lj4mU55ROosQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXot1CgsQ%253D%253D&md5=8e54b169fadf81e88f3b93d26c4add95</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DT.%26aulast%3DKawano%26aufirst%3DY.%26aulast%3DArimura%26aufirst%3DN.%26aulast%3DKawabata%26aufirst%3DS.%26aulast%3DKikuchi%26aufirst%3DA.%26aulast%3DKaibuchi%26aufirst%3DK.%26atitle%3DGSK-3%25CE%25B2%2520Regulates%2520Phosphorylation%2520of%2520CRMP-2%2520and%2520Neuronal%2520Polarity%26jtitle%3DCell%26date%3D2005%26volume%3D120%26spage%3D137%26epage%3D149%26doi%3D10.1016%2Fj.cell.2004.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolattukudy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honnorat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikoshiba, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goshima, Y.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of CRMP2 (collapsin response mediator protein 2) is involved in proper dendritic field organization</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1360</span>– <span class="NLM_lpage">1365</span>, <span class="refDoi"> DOI: 10.1523/jneurosci.5563-11.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1523%2Fjneurosci.5563-11.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22279220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1Cmt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=1360-1365&author=N.+Yamashitaauthor=T.+Ohshimaauthor=F.+Nakamuraauthor=P.+Kolattukudyauthor=J.+Honnoratauthor=K.+Mikoshibaauthor=Y.+Goshima&title=Phosphorylation+of+CRMP2+%28collapsin+response+mediator+protein+2%29+is+involved+in+proper+dendritic+field+organization&doi=10.1523%2Fjneurosci.5563-11.2012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of CRMP2 (collapsin response mediator protein 2) is involved in proper dendritic field organization</span></div><div class="casAuthors">Yamashita, Naoya; Ohshima, Toshio; Nakamura, Fumio; Kolattukudy, Papachan; Honnorat, Jerome; Mikoshiba, Katsuhiko; Goshima, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1360-1365</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Collapsin response mediator proteins (CRMPs) are intracellular proteins that mediate signals for several extracellular mols., such as Semaphorin3A and neurotrophins.  The phosphorylation of CRMP1 and CRMP2 by Cdk5 at Ser522 is involved in axonal guidance and spine development.  Here, we found that the Ser522-phosphorylated CRMP1 and/or CRMP2 are enriched in the dendrites of cultured cortical neurons and P7 cortical section.  To det. the physiol. role of CRMPs in dendritic development, we generated CRMP2 knock-in mutant mice (crmp2ki/ki) in which the Ser residue at 522 was replaced with Ala.  Strikingly, the cortical basal dendrites of double mutant crmp2ki/ki and crmp1-/- mice exhibited severe abnormal dendritic patterning, which we defined as "curling phenotype.".  These findings demonstrate that the function of CRMP1 and CRMP2 synergistically control dendritic projection, and the phosphorylation of CRMP2 at Ser522 is essential for proper dendritic field organization in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr35ooJgSzUN7Vg90H21EOLACvtfcHk0lj4mU55ROosQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1Cmt7Y%253D&md5=8e52693649d0273a8758034f27a0017f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1523%2Fjneurosci.5563-11.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252Fjneurosci.5563-11.2012%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DN.%26aulast%3DOhshima%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DF.%26aulast%3DKolattukudy%26aufirst%3DP.%26aulast%3DHonnorat%26aufirst%3DJ.%26aulast%3DMikoshiba%26aufirst%3DK.%26aulast%3DGoshima%26aufirst%3DY.%26atitle%3DPhosphorylation%2520of%2520CRMP2%2520%2528collapsin%2520response%2520mediator%2520protein%25202%2529%2520is%2520involved%2520in%2520proper%2520dendritic%2520field%2520organization%26jtitle%3DJ.%2520Neurosci.%26date%3D2012%26volume%3D32%26spage%3D1360%26epage%3D1365%26doi%3D10.1523%2Fjneurosci.5563-11.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doble, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snider, W. D.</span></span> <span> </span><span class="NLM_article-title">GSK-3 is a master regulator of neural progenitor homeostasis</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1390</span>– <span class="NLM_lpage">1397</span>, <span class="refDoi"> DOI: 10.1038/nn.2408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnn.2408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=19801986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WgsbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=1390-1397&author=W.-Y.+Kimauthor=X.+Wangauthor=Y.+Wuauthor=B.+W.+Dobleauthor=S.+Patelauthor=J.+R.+Woodgettauthor=W.+D.+Snider&title=GSK-3+is+a+master+regulator+of+neural+progenitor+homeostasis&doi=10.1038%2Fnn.2408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">GSK-3 is a master regulator of neural progenitor homeostasis</span></div><div class="casAuthors">Kim, Woo-Yang; Wang, Xinshuo; Wu, Yaohong; Doble, Bradley W.; Patel, Satish; Woodgett, James R.; Snider, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1390-1397</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The development of the brain requires the exquisite coordination of progenitor proliferation and differentiation to achieve complex circuit assembly.  It has been suggested that glycogen synthase kinase 3 (GSK-3) acts as an integrating mol. for multiple proliferation and differentiation signals because of its essential role in the RTK, Wnt and Shh signaling pathways.  We created conditional mutations that deleted both the α and β forms of GSK-3 in mouse neural progenitors.  GSK-3 deletion resulted in massive hyperproliferation of neural progenitors along the entire neuraxis.  Generation of both intermediate neural progenitors and postmitotic neurons was markedly suppressed.  These effects were assocd. with the dysregulation of β-catenin, Sonic Hedgehog, Notch and fibroblast growth factor signaling.  Our results indicate that GSK-3 signaling is an essential mediator of homeostatic controls that regulate neural progenitors during mammalian brain development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovTk0M-V9NYrVg90H21EOLACvtfcHk0lj4mU55ROosQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WgsbzJ&md5=d7ff5a16e07b70e8b4338316c5d70b7e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnn.2408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2408%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.-Y.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDoble%26aufirst%3DB.%2BW.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26aulast%3DSnider%26aufirst%3DW.%2BD.%26atitle%3DGSK-3%2520is%2520a%2520master%2520regulator%2520of%2520neural%2520progenitor%2520homeostasis%26jtitle%3DNat.%2520Neurosci.%26date%3D2009%26volume%3D12%26spage%3D1390%26epage%3D1397%26doi%3D10.1038%2Fnn.2408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theriault, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span> <span> </span><span class="NLM_article-title">A High-Throughput Screen for Wnt/β-Catenin Signaling Pathway Modulators in Human iPSC-Derived Neural Progenitors</span>. <i>J. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1263</span>, <span class="refDoi"> DOI: 10.1177/1087057112456876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1177%2F1087057112456876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22923789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A280%3ADC%252BC38bhsVegtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1252-1263&author=W.-N.+Zhaoauthor=C.+Chengauthor=K.+M.+Theriaultauthor=S.+D.+Sheridanauthor=L.-H.+Tsaiauthor=S.+J.+Haggarty&title=A+High-Throughput+Screen+for+Wnt%2F%CE%B2-Catenin+Signaling+Pathway+Modulators+in+Human+iPSC-Derived+Neural+Progenitors&doi=10.1177%2F1087057112456876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A high-throughput screen for Wnt/β-catenin signaling pathway modulators in human iPSC-derived neural progenitors</span></div><div class="casAuthors">Zhao Wen-Ning; Cheng Chialin; Theriault Kraig M; Sheridan Steven D; Tsai Li-Huei; Haggarty Stephen J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1252-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Wnt/β-catenin signaling has emerged as a central player in pathways implicated in the pathophysiology and treatment of neuropsychiatric disorders.  To identify potential novel therapeutics for these disorders, high-throughput screening (HTS) assays reporting on Wnt/β-catenin signaling in disease-relevant contexts are needed.  The use of human patient-derived induced pluripotent stem cell (iPSC) models provides ideal disease-relevant context if these stem cell cultures can be adapted for HTS-compatible formats.  Here, we describe a sensitive, HTS-compatible Wnt/β-catenin signaling reporter system generated in homogeneous, expandable neural progenitor cells (NPCs) derived from human iPSCs.  We validated this system by demonstrating dose-responsive stimulation by several known Wnt/β-catenin signaling pathway modulators, including Wnt3a, a glycogen synthase kinase-3 (GSK3) inhibitor, and the bipolar disorder therapeutic lithium.  These responses were robust and reproducible over time across many repeated assays.  We then conducted a screen of ~1500 compounds from a library of Food and Drug Administration-approved drugs and known bioactives and confirmed the HTS hits, revealing multiple chemical and biological classes of novel small-molecule probes of Wnt/β-catenin signaling.  Generating these type of pathway-selective, cell-based phenotypic assays in human iPSC-derived neural cells will advance the field of human experimental neurobiology toward the goal of identifying and validating targets for neuropsychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTNG8hBgzHx2K_SFaaT9INfW6udTcc2eYQWFvtfXVg87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38bhsVegtw%253D%253D&md5=ad3b356f5acafe845e06450a6ac39f1e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1177%2F1087057112456876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112456876%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DW.-N.%26aulast%3DCheng%26aufirst%3DC.%26aulast%3DTheriault%26aufirst%3DK.%2BM.%26aulast%3DSheridan%26aufirst%3DS.%2BD.%26aulast%3DTsai%26aufirst%3DL.-H.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26atitle%3DA%2520High-Throughput%2520Screen%2520for%2520Wnt%252F%25CE%25B2-Catenin%2520Signaling%2520Pathway%2520Modulators%2520in%2520Human%2520iPSC-Derived%2520Neural%2520Progenitors%26jtitle%3DJ.%2520Biomol.%2520Screen.%26date%3D2012%26volume%3D17%26spage%3D1252%26epage%3D1263%26doi%3D10.1177%2F1087057112456876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harigaya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawarabayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsubara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakazato, Y.</span></span> <span> </span><span class="NLM_article-title">Amyloid cored plaques in Tg2576 transgenic mice are characterized by giant plaques, slightly activated microglia, and the lack of paired helical filament-typed, dystrophic neurites</span>. <i>Virchows Arch.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>441</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1007/s00428-002-0643-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1007%2Fs00428-002-0643-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=12404061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkslSrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=441&publication_year=2002&pages=358-367&author=A.+Sasakiauthor=M.+Shojiauthor=Y.+Harigayaauthor=T.+Kawarabayashiauthor=M.+Ikedaauthor=M.+Naitoauthor=E.+Matsubaraauthor=K.+Abeauthor=Y.+Nakazato&title=Amyloid+cored+plaques+in+Tg2576+transgenic+mice+are+characterized+by+giant+plaques%2C+slightly+activated+microglia%2C+and+the+lack+of+paired+helical+filament-typed%2C+dystrophic+neurites&doi=10.1007%2Fs00428-002-0643-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Amyloid cored plaques in Tg2576 transgenic mice are characterized by giant plaques, slightly activated microglia, and the lack of paired helical filament-typed, dystrophic neurites</span></div><div class="casAuthors">Sasaki, Atsushi; Shoji, Mikio; Harigaya, Yasuo; Kawarabayashi, Takeshi; Ikeda, Masaki; Naito, Makoto; Matsubara, Etsuro; Abe, Koji; Nakazato, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Virchows Archiv</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">441</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-367</span>CODEN:
                <span class="NLM_cas:coden">VARCEM</span>;
        ISSN:<span class="NLM_cas:issn">0945-6317</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">We examd. the brains of Tg2576 transgenic mice carrying human amyloid precursor protein with the Swedish mutation and Alzheimer's disease (AD) by means of immunohistochem. and electron microscopy to clarify the characteristics of amyloid-assocd. pathol. in the transgenic mice.  In 12- to 29-mo-old Tg2576 mice, congophilic cored plaques in the neocortex and hippocampus were labeled by all of the Aβ1-, Aβ40- and 42-specific antibodies, as seen in the classical plaques in AD.  However, large-sized (>50 μm in core diam.) plaques were seen more frequently in the older mice (18-29 mo) than in those with AD (approx. 20% vs 2% in total cored plaques), and Tg2576 mice contained giant plaques (>75 μm in core diam.), which were almost never seen in the brain of those with AD.  Neither thread-like structures nor peripheral coronas were obsd. in the cored plaques of the transgenic mice in the silver impregnations.  Immunohistochem., plaque-accompanied microglia showed a slight enlargement of the cytoplasm with consistent labeling of Mac-1 and macrosialin (murine CD68), and with partial labeling of Ia antigen and macrophage-colony stimulating factor receptor.  Ultrastructurally, the microglia surrounding the extracellular amyloid fibrils in the large, cored plaques showed some organella with phagocytic activity, such as secondary lysosomal, dense bodies, but intracellular amyloid fibrils were not evident.  Dystrophic neurites in the plaques of the transgenic mice contained many dense multilaminar bodies, but no paired helical filaments.  Our results suggest that giant cored plaques without coronas or paired helical filament-typed, dystrophic neurites are characteristic in Tg2576 mice, and that plaque-assocd. microglia in transgenic mice are activated to be in phagocytic function but not sufficient enough to digest extracellularly deposited amyloid fibrils.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbPbvLPXqSzLVg90H21EOLACvtfcHk0lgueL2g5kioPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkslSrtg%253D%253D&md5=9fac7e33706e4a04ee95d99f55e1942c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs00428-002-0643-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00428-002-0643-8%26sid%3Dliteratum%253Aachs%26aulast%3DSasaki%26aufirst%3DA.%26aulast%3DShoji%26aufirst%3DM.%26aulast%3DHarigaya%26aufirst%3DY.%26aulast%3DKawarabayashi%26aufirst%3DT.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DMatsubara%26aufirst%3DE.%26aulast%3DAbe%26aufirst%3DK.%26aulast%3DNakazato%26aufirst%3DY.%26atitle%3DAmyloid%2520cored%2520plaques%2520in%2520Tg2576%2520transgenic%2520mice%2520are%2520characterized%2520by%2520giant%2520plaques%252C%2520slightly%2520activated%2520microglia%252C%2520and%2520the%2520lack%2520of%2520paired%2520helical%2520filament-typed%252C%2520dystrophic%2520neurites%26jtitle%3DVirchows%2520Arch.%26date%3D2002%26volume%3D441%26spage%3D358%26epage%3D367%26doi%3D10.1007%2Fs00428-002-0643-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bero, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirrito, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raichle, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzman, D. M.</span></span> <span> </span><span class="NLM_article-title">Neuronal activity regulates the regional vulnerability to amyloid-β deposition</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nn.2801</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnn.2801" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=21532579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVGltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=750-756&author=A.+W.+Beroauthor=P.+Yanauthor=J.+H.+Rohauthor=J.+R.+Cirritoauthor=F.+R.+Stewartauthor=M.+E.+Raichleauthor=J.-M.+Leeauthor=D.+M.+Holtzman&title=Neuronal+activity+regulates+the+regional+vulnerability+to+amyloid-%CE%B2+deposition&doi=10.1038%2Fnn.2801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Neuronal activity regulates the regional vulnerability to amyloid-β deposition</span></div><div class="casAuthors">Bero, Adam W.; Yan, Ping; Roh, Jee Hoon; Cirrito, John R.; Stewart, Floy R.; Raichle, Marcus E.; Lee, Jin-Moo; Holtzman, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Amyloid-β (Aβ) plaque deposition in specific brain regions is a pathol. hallmark of Alzheimer's disease.  However, the mechanism underlying the regional vulnerability to Aβ deposition in Alzheimer's disease is unknown.  Herein, we provide evidence that endogenous neuronal activity regulates the regional concn. of interstitial fluid (ISF) Aβ, which drives local Aβ aggregation.  Using in vivo microdialysis, we show that ISF Aβ concns. in several brain regions of APP transgenic mice before plaque deposition were commensurate with the degree of subsequent plaque deposition and with the concn. of lactate, a marker of neuronal activity.  Furthermore, unilateral vibrissal stimulation increased ISF Aβ, and unilateral vibrissal deprivation decreased ISF Aβ and lactate, in contralateral barrel cortex.  Long-term unilateral vibrissal deprivation decreased amyloid plaque formation and growth.  Our results suggest a mechanism to account for the vulnerability of specific brain regions to Aβ deposition in Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvnBN2KKH8WbVg90H21EOLACvtfcHk0lhuCWwQkgv9VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVGltro%253D&md5=d1adad353ca3c996d554bb7112b9859a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnn.2801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2801%26sid%3Dliteratum%253Aachs%26aulast%3DBero%26aufirst%3DA.%2BW.%26aulast%3DYan%26aufirst%3DP.%26aulast%3DRoh%26aufirst%3DJ.%2BH.%26aulast%3DCirrito%26aufirst%3DJ.%2BR.%26aulast%3DStewart%26aufirst%3DF.%2BR.%26aulast%3DRaichle%26aufirst%3DM.%2BE.%26aulast%3DLee%26aufirst%3DJ.-M.%26aulast%3DHoltzman%26aufirst%3DD.%2BM.%26atitle%3DNeuronal%2520activity%2520regulates%2520the%2520regional%2520vulnerability%2520to%2520amyloid-%25CE%25B2%2520deposition%26jtitle%3DNat.%2520Neurosci.%26date%3D2011%26volume%3D14%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnn.2801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernard-Gauthier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirrmacher, R.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the development and application of radiolabeled kinase inhibitors for PET imaging</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">22000</span>– <span class="NLM_lpage">22027</span>, <span class="refDoi"> DOI: 10.3390/molecules201219816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.3390%2Fmolecules201219816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26690113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVKgsL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=22000-22027&author=V.+Bernard-Gauthierauthor=J.+Baileyauthor=S.+Berkeauthor=R.+Schirrmacher&title=Recent+advances+in+the+development+and+application+of+radiolabeled+kinase+inhibitors+for+PET+imaging&doi=10.3390%2Fmolecules201219816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the development and application of radiolabeled kinase inhibitors for PET imaging</span></div><div class="casAuthors">Bernard-Gauthier, Vadim; Bailey, Justin J.; Berke, Sheldon; Schirrmacher, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">22000-22027</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Over the last 20 years, intensive investigation and multiple clin. successes targeting protein kinases, mostly for cancer treatment, have identified small mol. kinase inhibitors as a prominent therapeutic class.  In the course of those investigations, radiolabeled kinase inhibitors for positron emission tomog. (PET) imaging have been synthesized and evaluated as diagnostic imaging probes for cancer characterization.  Given that inhibitor coverage of the kinome is continuously expanding, in vivo PET imaging will likely find increasing applications for therapy monitoring and receptor d. studies both in- and outside of oncol. conditions.  Early investigated radiolabeled inhibitors, which are mostly based on clin. approved tyrosine kinase inhibitor (TKI) isotopologues, have now entered clin. trials.  Novel radioligands for cancer and PET neuroimaging originating from novel but relevant target kinases are currently being explored in preclin. studies.  This article reviews the literature involving radiotracer design, radiochem. approaches, biol. tracer evaluation and nuclear imaging results of radiolabeled kinase inhibitors for PET reported between 2010 and mid-2015.  Aspects regarding the usefulness of pursuing selective vs. promiscuous inhibitor scaffolds and the inherent challenges assocd. with intracellular enzyme imaging will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoK7e4uBNDBLVg90H21EOLACvtfcHk0lhuCWwQkgv9VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVKgsL%252FI&md5=be84e68fe296d62fdcfd15857dda05f0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3390%2Fmolecules201219816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules201219816%26sid%3Dliteratum%253Aachs%26aulast%3DBernard-Gauthier%26aufirst%3DV.%26aulast%3DBailey%26aufirst%3DJ.%26aulast%3DBerke%26aufirst%3DS.%26aulast%3DSchirrmacher%26aufirst%3DR.%26atitle%3DRecent%2520advances%2520in%2520the%2520development%2520and%2520application%2520of%2520radiolabeled%2520kinase%2520inhibitors%2520for%2520PET%2520imaging%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D22000%26epage%3D22027%26doi%3D10.3390%2Fmolecules201219816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: the road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+the+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lhuCWwQkgv9VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520the%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D377%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimwood, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaddan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zasadny, K.</span></span> <span> </span><span class="NLM_article-title">Design and selection parameters to accelerate the discovery of novel central nervous system positron emission tomography (PET) ligands and their application in the development of a novel phosphodiesterase 2A PET ligand</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4568</span>– <span class="NLM_lpage">4579</span>, <span class="refDoi"> DOI: 10.1021/jm400312y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400312y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4568-4579&author=L.+Zhangauthor=A.+Villalobosauthor=E.+M.+Beckauthor=T.+Bocanauthor=T.+A.+Chappieauthor=L.+Chenauthor=S.+Grimwoodauthor=S.+D.+Heckauthor=C.+J.+Helalauthor=X.+Houauthor=J.+M.+Humphreyauthor=J.+Luauthor=M.+B.+Skaddanauthor=T.+J.+McCarthyauthor=P.+R.+Verhoestauthor=T.+T.+Wagerauthor=K.+Zasadny&title=Design+and+selection+parameters+to+accelerate+the+discovery+of+novel+central+nervous+system+positron+emission+tomography+%28PET%29+ligands+and+their+application+in+the+development+of+a+novel+phosphodiesterase+2A+PET+ligand&doi=10.1021%2Fjm400312y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Selection Parameters to Accelerate the Discovery of Novel Central Nervous System Positron Emission Tomography (PET) Ligands and Their Application in the Development of a Novel Phosphodiesterase 2A PET Ligand</span></div><div class="casAuthors">Zhang, Lei; Villalobos, Anabella; Beck, Elizabeth M.; Bocan, Thomas; Chappie, Thomas A.; Chen, Laigao; Grimwood, Sarah; Heck, Steven D.; Helal, Christopher J.; Hou, Xinjun; Humphrey, John M.; Lu, Jiemin; Skaddan, Marc B.; McCarthy, Timothy J.; Verhoest, Patrick R.; Wager, Travis T.; Zasadny, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4568-4579</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To accelerate the discovery of novel small mol. central nervous system (CNS) positron emission tomog. (PET) ligands, we aimed to define a property space that would facilitate ligand design and prioritization, thereby providing a higher probability of success for novel PET ligand development.  Toward this end, we built a database consisting of 62 PET ligands that have successfully reached the clinic and 15 radioligands that failed in late-stage development as neg. controls.  A systematic anal. of these ligands identified a set of preferred parameters for physicochem. properties, brain permeability, and nonspecific binding (NSB).  These preferred parameters have subsequently been applied to several programs and have led to the successful development of novel PET ligands with reduced resources and timelines.  This strategy is illustrated here by the discovery of the novel phosphodiesterase 2A (PDE2A) PET ligand 4-(3-[18F]fluoroazetidin-1-yl)-7-methyl-5-{1-methyl-5-[4-(trifluoromethyl)phenyl]-1H-pyrazol-4-yl}imidazo[5,1-f][1,2,4]triazine, [18F]PF-05270430 (5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTCG9TZE0X-bVg90H21EOLACvtfcHk0lhk5vHoEsELzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D&md5=ee7ce04829d7456a632c02c331c4edfa</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm400312y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400312y%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DVillalobos%26aufirst%3DA.%26aulast%3DBeck%26aufirst%3DE.%2BM.%26aulast%3DBocan%26aufirst%3DT.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DGrimwood%26aufirst%3DS.%26aulast%3DHeck%26aufirst%3DS.%2BD.%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DSkaddan%26aufirst%3DM.%2BB.%26aulast%3DMcCarthy%26aufirst%3DT.%2BJ.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DZasadny%26aufirst%3DK.%26atitle%3DDesign%2520and%2520selection%2520parameters%2520to%2520accelerate%2520the%2520discovery%2520of%2520novel%2520central%2520nervous%2520system%2520positron%2520emission%2520tomography%2520%2528PET%2529%2520ligands%2520and%2520their%2520application%2520in%2520the%2520development%2520of%2520a%2520novel%2520phosphodiesterase%25202A%2520PET%2520ligand%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4568%26epage%3D4579%26doi%3D10.1021%2Fjm400312y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Englebienne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 5. Force-field-based prediction of binding affinities of ligands to proteins</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2564</span>– <span class="NLM_lpage">2571</span>, <span class="refDoi"> DOI: 10.1021/ci900251k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900251k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtleqtLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2564-2571&author=P.+Englebienneauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+5.+Force-field-based+prediction+of+binding+affinities+of+ligands+to+proteins&doi=10.1021%2Fci900251k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. 5. Force-Field-Based Prediction of Binding Affinities of Ligands to Proteins</span></div><div class="casAuthors">Englebienne, Pablo; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2564-2571</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein our efforts in the development of three empirical scoring functions with application in protein-ligand docking.  A first scoring function was developed from 209 crystal structures of protein-ligand complexes and a second one from 946 cross-docked complexes.  Tuning of the coeffs. for the different terms making up these functions was performed by an iterative approach to optimize the correlations between obsd. activities and calcd. scores.  A third scoring function was developed from libraries of known actives and decoys docked to six different protein conformational ensembles.  In the latter case, the tuning of the coeffs. was performed so as to optimize the area under the curve of a receiver operating characteristic (ROC) for the discrimination of actives and inactives.  The newly developed scoring functions were next assessed on independent sets of protein-ligand complexes for their ability to predict binding affinities and to discriminate actives from inactives.  In the first validation the first function, which was trained on active compds. only, performed as well as other commonly used ones.  On a high-throughput virtual screening validation on five protein conformational ensembles, the third scoring function that included data from inactive compds. performed significantly better.  This validation showed that the inclusion of data from inactive compds. is crit. for performance in virtual high-throughput screening applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJkz6UVDkQ8LVg90H21EOLACvtfcHk0lhk5vHoEsELzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtleqtLrJ&md5=25916732a94e7cb8b4db2cdb5c913828</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci900251k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900251k%26sid%3Dliteratum%253Aachs%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25205.%2520Force-field-based%2520prediction%2520of%2520binding%2520affinities%2520of%2520ligands%2520to%2520proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2564%26epage%3D2571%26doi%3D10.1021%2Fci900251k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englebienne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza-Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbeil, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagna-Slater, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Integrating Medicinal Chemistry, Organic/Combinatorial Chemistry, and Computational Chemistry for the Discovery of Selective Estrogen Receptor Modulators with Forecaster, a Novel Platform for Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1021/ci2004779</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2004779" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=210-224&author=E.+Therrienauthor=P.+Englebienneauthor=A.+G.+Arrowsmithauthor=R.+Mendoza-Sanchezauthor=C.+R.+Corbeilauthor=N.+Weillauthor=V.+Campagna-Slaterauthor=N.+Moitessier&title=Integrating+Medicinal+Chemistry%2C+Organic%2FCombinatorial+Chemistry%2C+and+Computational+Chemistry+for+the+Discovery+of+Selective+Estrogen+Receptor+Modulators+with+Forecaster%2C+a+Novel+Platform+for+Drug+Discovery&doi=10.1021%2Fci2004779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating Medicinal Chemistry, Organic/Combinatorial Chemistry, and Computational Chemistry for the Discovery of Selective Estrogen Receptor Modulators with Forecaster, a Novel Platform for Drug Discovery</span></div><div class="casAuthors">Therrien, Eric; Englebienne, Pablo; Arrowsmith, Andrew G.; Mendoza-Sanchez, Rodrigo; Corbeil, Christopher R.; Weill, Nathanael; Campagna-Slater, Valerie; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-224</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of a large medicinal chem. program, we wish to develop novel selective estrogen receptor modulators (SERMs) as potential breast cancer treatments using a combination of exptl. and computational approaches.  However, one of the remaining difficulties nowadays is to fully integrate computational (i.e., virtual, theor.) and medicinal (i.e., exptl., intuitive) chem. to take advantage of the full potential of both.  For this purpose, we have developed a Web-based platform, Forecaster, and a no. of programs (e.g., Prep., React, Select) with the aim of combining computational chem. and medicinal chem. expertise to facilitate drug discovery and development and more specifically to integrate synthesis into computer-aided drug design.  In our quest for potent SERMs, this platform was used to build virtual combinatorial libraries, filter and ext. a highly diverse library from the NCI database, and dock them to the estrogen receptor (ER), with all of these steps being fully automated by computational chemists for use by medicinal chemists.  As a result, virtual screening of a diverse library seeded with active compds. followed by a search for analogs yielded an enrichment factor of 129, with 98% of the seeded active compds. recovered, while the screening of a designed virtual combinatorial library including known actives yielded an area under the receiver operating characteristic (AU-ROC) of 0.78.  The lead optimization proved less successful, further demonstrating the challenge to simulate structure activity relationship studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt6YsO45gkpLVg90H21EOLACvtfcHk0lhl7gux0ulDvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCru7vE&md5=a7b3218263c14ae60a38962eed567e87</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci2004779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2004779%26sid%3Dliteratum%253Aachs%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DArrowsmith%26aufirst%3DA.%2BG.%26aulast%3DMendoza-Sanchez%26aufirst%3DR.%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DIntegrating%2520Medicinal%2520Chemistry%252C%2520Organic%252FCombinatorial%2520Chemistry%252C%2520and%2520Computational%2520Chemistry%2520for%2520the%2520Discovery%2520of%2520Selective%2520Estrogen%2520Receptor%2520Modulators%2520with%2520Forecaster%252C%2520a%2520Novel%2520Platform%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D210%26epage%3D224%26doi%3D10.1021%2Fci2004779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corbeil, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englebienne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yannopoulos, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilimoria, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">L’Heureux, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 2. Development and application of FITTED 1.5 to the virtual screening of potential HCV polymerase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">902</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1021/ci700398h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700398h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVWgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=902-909&author=C.+R.+Corbeilauthor=P.+Englebienneauthor=C.+G.+Yannopoulosauthor=L.+Chanauthor=S.+K.+Dasauthor=D.+Bilimoriaauthor=L.+L%E2%80%99Heureuxauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+2.+Development+and+application+of+FITTED+1.5+to+the+virtual+screening+of+potential+HCV+polymerase+inhibitors&doi=10.1021%2Fci700398h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. Development and Application of FITTED 1.5 to the Virtual Screening of Potential HCV Polymerase Inhibitors</span></div><div class="casAuthors">Corbeil, Christopher R.; Englebienne, Pablo; Yannopoulos, Constantin G.; Chan, Laval; Das, Sanjoy K.; Bilimoria, Darius; L'Heureux, Lucille; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">902-909</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HCV NS5B polymerase is a validated target for the treatment of hepatitis C, known to be one of the most challenging enzymes for docking programs.  In order to improve the low accuracy of existing docking methods obsd. with this challenging enzyme, the authors have significantly modified and updated Fitted 1.0, a recently reported docking program, into FITTED 1.5.  This enhanced version is now applicable to the virtual screening of compd. libraries and includes new features such as filters and pharmacophore- or interaction-site-oriented docking.  As a first validation, FITTED 1.5 was applied to the testing set previously developed for FITTED 1.0 and extended to include hepatitis C virus (HCV) polymerase inhibitors.  This first validation showed an increased accuracy as well as an increase in speed.  It also shows that the accuracy toward HCV polymerase is better than previously obsd. with other programs.  Next, application of FITTED 1.5 to the virtual screening of the Maybridge library seeded with known HCV polymerase inhibitors revealed its ability to recover most of these actives in the top 5% of the hit list.  As a third validation, further biol. assays uncovered HCV polymerase inhibition for selected Maybridge compds. ranked in the top of the hit list.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXlpXNyj_8m7Vg90H21EOLACvtfcHk0lhl7gux0ulDvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVWgtr4%253D&md5=6a661760aea54b56b878b7f64c9e3ca7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci700398h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700398h%26sid%3Dliteratum%253Aachs%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DYannopoulos%26aufirst%3DC.%2BG.%26aulast%3DChan%26aufirst%3DL.%26aulast%3DDas%26aufirst%3DS.%2BK.%26aulast%3DBilimoria%26aufirst%3DD.%26aulast%3DL%25E2%2580%2599Heureux%26aufirst%3DL.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25202.%2520Development%2520and%2520application%2520of%2520FITTED%25201.5%2520to%2520the%2520virtual%2520screening%2520of%2520potential%2520HCV%2520polymerase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D902%26epage%3D909%26doi%3D10.1021%2Fci700398h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yuhua Zhong, Shaoxi Yang, Jianyu Cui, Jie Wang, Lin Li, Yilin Chen, Junjie Chen, Pengju Feng, Shun Huang, Hongsheng Li, Yanjian Han, Ganghua Tang, <span class="NLM_string-name hlFld-ContribAuthor">Kongzhen Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Novel 18F-Labeled Isonicotinamide-Based Radioligands for Positron Emission Tomography Imaging of Glycogen Synthase Kinase-3β. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2021,</strong> <em>18 </em>
                                    (3)
                                     , 1277-1284. <a href="https://doi.org/10.1021/acs.molpharmaceut.0c01133" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.0c01133</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.0c01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.0c01133%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DNovel%252B18F-Labeled%252BIsonicotinamide-Based%252BRadioligands%252Bfor%252BPositron%252BEmission%252BTomography%252BImaging%252Bof%252BGlycogen%252BSynthase%252BKinase-3%2525CE%2525B2%26aulast%3DZhong%26aufirst%3DYuhua%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D18112020%26date%3D11012021%26date%3D09012021%26date%3D25012021%26volume%3D18%26issue%3D3%26spage%3D1277%26epage%3D1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stephanie L. Saundh, Debasis Patnaik, Steve Gagné, Joshua A. Bishop, Sean Lipsit, Samat Amat, Narsimha Pujari, Anand Krishnan Nambisan, Robert Bigsby, Mary Murphy, Li-Huei Tsai, Stephen J. Haggarty, <span class="NLM_string-name hlFld-ContribAuthor">Adelaine Kwun-Wai Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Mechanistic Characterization of a Peptide Inhibitor of Glycogen Synthase Kinase (GSK3β) Derived from the Disrupted in Schizophrenia 1 (DISC1) Protein. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2020,</strong> <em>11 </em>
                                    (24)
                                     , 4128-4138. <a href="https://doi.org/10.1021/acschemneuro.0c00380" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00380%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DIdentification%252Band%252BMechanistic%252BCharacterization%252Bof%252Ba%252BPeptide%252BInhibitor%252Bof%252BGlycogen%252BSynthase%252BKinase%252B%252528GSK3%2525CE%2525B2%252529%252BDerived%252Bfrom%252Bthe%252BDisrupted%252Bin%252BSchizophrenia%252B1%252B%252528DISC1%252529%252BProtein%26aulast%3DSaundh%26aufirst%3DStephanie%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18062020%26date%3D09112020%26date%3D30112020%26volume%3D11%26issue%3D24%26spage%3D4128%26epage%3D4138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cassis  Varlow</span>, <span class="hlFld-ContribAuthor ">Andrew V.  Mossine</span>, <span class="hlFld-ContribAuthor ">Vadim  Bernard-Gauthier</span>, <span class="hlFld-ContribAuthor ">Peter J.H.  Scott</span>, <span class="hlFld-ContribAuthor ">Neil  Vasdev</span>. </span><span class="cited-content_cbyCitation_article-title">Radiofluorination of oxazole-carboxamides for preclinical PET neuroimaging of GSK-3. </span><span class="cited-content_cbyCitation_journal-name">Journal of Fluorine Chemistry</span><span> <strong>2021,</strong> <em>245 </em>, 109760. <a href="https://doi.org/10.1016/j.jfluchem.2021.109760" title="DOI URL">https://doi.org/10.1016/j.jfluchem.2021.109760</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jfluchem.2021.109760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jfluchem.2021.109760%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Fluorine%2520Chemistry%26atitle%3DRadiofluorination%252Bof%252Boxazole-carboxamides%252Bfor%252Bpreclinical%252BPET%252Bneuroimaging%252Bof%252BGSK-3%26aulast%3DVarlow%26aufirst%3DCassis%26date%3D2021%26volume%3D245%26spage%3D109760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen J.  Haggarty</span>, <span class="hlFld-ContribAuthor ">Rakesh  Karmacharya</span>, <span class="hlFld-ContribAuthor ">Roy H.  Perlis</span>. </span><span class="cited-content_cbyCitation_article-title">Advances toward precision medicine for bipolar disorder: mechanisms & molecules. </span><span class="cited-content_cbyCitation_journal-name">Molecular Psychiatry</span><span> <strong>2021,</strong> <em>26 </em>
                                    (1)
                                     , 168-185. <a href="https://doi.org/10.1038/s41380-020-0831-4" title="DOI URL">https://doi.org/10.1038/s41380-020-0831-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41380-020-0831-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41380-020-0831-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Psychiatry%26atitle%3DAdvances%252Btoward%252Bprecision%252Bmedicine%252Bfor%252Bbipolar%252Bdisorder%25253A%252Bmechanisms%252B%252526%252Bmolecules%26aulast%3DHaggarty%26aufirst%3DStephen%2BJ.%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D1%26spage%3D168%26epage%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlos  Roca</span>, <span class="hlFld-ContribAuthor ">Nuria E.  Campillo</span>. </span><span class="cited-content_cbyCitation_article-title">Glycogen synthase kinase 3 (GSK-3) inhibitors: a patent update (2016–2019). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2020,</strong> <em>30 </em>
                                    (11)
                                     , 863-872. <a href="https://doi.org/10.1080/13543776.2020.1815706" title="DOI URL">https://doi.org/10.1080/13543776.2020.1815706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2020.1815706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2020.1815706%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DGlycogen%252Bsynthase%252Bkinase%252B3%252B%252528GSK-3%252529%252Binhibitors%25253A%252Ba%252Bpatent%252Bupdate%252B%2525282016%2525E2%252580%2525932019%252529%26aulast%3DRoca%26aufirst%3DCarlos%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D11%26spage%3D863%26epage%3D872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. [<sup>3</sup>H]PF-04802367 ([<sup>3</sup>H]<b>1</b>) displays high specific and selective binding to GSK-3α/β in cells and mammalian brains in vitro. (A) Chemical structure of [<sup>3</sup>H]<b>1</b>. (B) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in the normal rat brain (20 μm sections, <i>n</i> = 3) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (C) In vitro autoradiography images of [<sup>3</sup>H]<b>1</b> normal rat brain (coronal sections) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (D) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in the normal marmoset brain (20 μm sections, <i>n</i> = 3) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (E) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in the normal human brain (20 μm sections, frontal cortex, <i>n</i> = 6) under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (F) In vitro autoradiography images of [<sup>3</sup>H]<b>1</b> normal human brain under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (G) Saturation [<sup>3</sup>H]<b>1</b> binding curves, Scatchart plot, and <i>B</i><sub>max</sub> determination using the homogenate mouse brain tissue. (H,I) Cell binding assays with [<sup>3</sup>H]<b>1</b> in HEK-293T P301S hTau cells, overexpressing GSK-3α/β and GSK-3α/β knockdown (shRNA). [<sup>3</sup>H]<b>1</b> binding under 2h baseline and following timecourse chase experiments. (J) Western blot analysis of control, GSK-3α/β-overexpressing, and GSK-3α/β shRNA cells used for [<sup>3</sup>H]<b>1</b>-binding studies. Data show mean values ± standard deviation (SD). Statistical significance was calculated one-way ANOVA followed by Dunnett’s test (B,D,E,H).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Evolution and in vitro pharmacological characterization of a highly potent and selective oxazole-4-carboxamide-based GSK-3 inhibitor series. (A) Generalized strategy to synthesize arrays of carboxamide-based GSK-3 inhibitors, as well as structures and properties of selected GSK-3 inhibitors (for details on chemical synthesis, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Schemes S1–S4</a>). IC<sub>50</sub> values determined using an ADP-Glo kinase assay (<i>n</i> = 2). (B) Heat map representing key properties [IC<sub>50</sub>, central nervous system multiparameter optimization (CNS MPO), and Fu_p] of oxazole-4-carboxamide-based GSK-3 inhibitors including a comparison with previously described GSK-3 PET radiotracers (for structures and detailed parameter values, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Tables S1–S4</a>). IC<sub>50</sub> values determined using an ADP-GloTM kinase assay (<i>n</i> = 2). (C,D) The dose–response curves and binding affinities of the representative inhibitor <b>28</b> versus GSK-3α, GSK-3β (<i>n</i> = 3), and main kinases off targets DYRK1A, DRYK1B, and CLK4 (<i>n</i> = 1) demonstrate high selectivity for GSK-3α/β (radiometric filtration binding assay, [ATP] = <i>K</i><sub>m ATP</sub>, data show mean values ± SD). (E) Hinge/α2 region sequence alignment of GSK-3β and the main kinase secondary target DYRK1A. (F) Comprehensive kinase selectivity profile of representative inhibitor <b>44</b> and <b>28</b> against 371 kinases (kinases are ordered alphabetically, and the activity is relative to control at 1.0 μM inhibitor concentration, <i>n</i> = 2, [γ-<sup>33</sup>P]ATP-based enzymatic assay performed by Reaction Biology, for detailed data see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S5</a>). (G) Overlay of the hinge/α2 region of GSK-3β (gray, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>) and DYRK1A (yellow, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR">3ANR</a>). (H) Predicted detailed interaction of <b>44</b> (yellow sticks) with GSK-3β (gray cartoon, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>). The π–cation interaction is depicted as a black dash line. (I) Alternate view of the binding mode of <b>44</b> (yellow, space fill model) with GSK-3β (surface rendering, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a>). CNS MPO; Fu_p, in silico calculated unbound fraction in plasma.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Oxazole-4-carboxamide-based probes engage both active npS9 and catalytically inactive pS9 GSK-3β. (A) Structural basis for the recognition of inhibitors by pS9-GSK-3β (<b>44</b> is depicted as a representative inhibitor, yellow, space fill model). The predicted binding mode illustrates the distal positioning of <b>44</b> from the activation loop and pS9 auto-inhibitory N-terminal peptide (blue surface) and the negligible ATP-binding site rearrangement upon binding of the pS9 auto-inhibitory N-terminal peptide of GSK-3β (gray surface, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NM3">4NM3</a>). (B) Timecourse analysis showing GSK-3β activity in the presence of low ATP content and GSP2 (ADP-Glo-based kinase assay, <i>n</i> = 3). (C) Time-course analysis showing GSK-3β activity in the presence of high ATP content and high ATP content combined with AKT (ADP-Glo-based kinase assay, <i>n</i> = 3). (D) Western blot analysis of enzymatic reaction contents (control, ATP treated and ATP + AKT treated). (E) Binding affinities of <b>1</b> and <b>44</b> measured by MST. (F,G) MST dose–response curves for the binding interaction of between <b>1</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). (H,I) MST dose–response curves for the binding interaction of <b>44</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). Data show mean values ± SD. Statistical significance was calculated with paired <i>t</i>-test (B,C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure-guided design and characterization of the GSK-3β isozyme-selective inhibitors <b>51</b>. (A) Structures and properties of selected GSK-3 inhibitors leading to <b>51</b> (for details on chemical synthesis, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Schemes S1–S4</a>). IC<sub>50</sub> values determined using an ADP-Glo kinase assay (<i>n</i> = 2). (B) Dose–response curves of <b>51</b> versus GSK-3α and GSK-3β (<i>n</i> = 3) (compared with <b>28</b>, radiometric filtration binding assay, [ATP] = <i>K</i><sub>m ATP</sub>, data show mean values ± SD). (C) Predicted binding mode of <b>51</b> (green sticks) with GSK-3β (gray cartoon, PDB ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK">5KPK</a>). Highlight on the interaction network associated with Asp133. Hydrogen bonds are depicted as a yellow dash lines. (D) Binding affinity measurements of inhibitors versus GSK-3α and GSK-3β (<i>n</i> = 3). (E–G) Alternate views of the predicted contact between <b>51</b> and the selectivity pocket of GSK-3β. (H) Kinativ data from mice brain lysates showing the selective engagement of the representative inhibitor <b>49</b> (1.0 μM) with GSK-3 (<i>n</i> = 2, for detailed data, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S5</a>). (I) Comprehensive kinase selectivity profile of representative inhibitor <b>51</b> against 371 kinases (kinases are ordered alphabetically, and the activity is relative to control at 1.0 μM inhibitor concentration, <i>n</i> = 2, [γ-<sup>33</sup>P]ATP-based enzymatic assay performed by Reaction Biology, for detailed data see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf" class="ext-link">Table S9</a>). (J) Selectivity profile of <b>51</b> (10.0 μM) against a CNS target panel (<i>n</i> = 2, 80 targets). (K) Binding affinity of <b>51</b> against the five most inhibited CNS targets at a concentration 100 000-fold over the <i>K</i><sub>i</sub> of <b>51</b> for GSK-3β. (L) MST dose–response curve and binding affinity for the interaction between <b>51</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). (M) MST dose–response curve and binding affinity for the interaction between <b>51</b> with active npS9-GSK-3β and pS9-GSK-3β (<i>n</i> = 2). Data show mean values ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selective oxazole-4-carboxamide GSK-3 inhibitors modulate both pTau and pCRMP2<sup>T514</sup> and display favorable therapeutic index against Wnt/β-catenin signaling in human neuronal cells. (A–D) Dose-dependent effect of selected compounds on decreasing phosphorylated tau in human neurons. 20 h treatment with <b>1</b>, <b>28</b>, <b>51</b>, but not <b>29</b> dose-dependently decreased mean pTau (PHF-1) intensity in neuronal cell bodies. Each bar is the mean values ± SEM of 14–21 fields/well. Each dose of each compound was tested in 1 well per plate. Data from 3 plates in total are shown in the figure. (E–H) High-content imaging assays using human neurons showed that representative inhibitors (<b>1</b>, <b>28</b>, <b>51</b>) reduced CRMP2 phosphorylation in a dose-dependent manner whereas <b>29</b> did not. CRMP2 is a direct substrate of GSK-3β. The relative level of pCRMP2<sup>T514</sup> intensity was reported as the percentage of the intensity for dimethylsulfoxide (DMSO)-treated (100%) (<i>n</i> = 3 biological replicates, <i>n</i> = 18 images for analysis. Data show mean values ± SEM). Relative levels of dendritic marker MAP2 were not affected by the compound treatments, indicating the health of neurons. (I) Representative images of <b>51</b> treatment on pCRMP2<sup>T514</sup> intensity (green) versus MAP2 intensity (red) illustrate the dose-dependent pCRMP2<sup>T514</sup> intensity reduction. (J–M) Pro-neurogenic potentials measured as NPC proliferation were detected for active oxazole-4-carboxamide GSK-3 inhibitors. Graphs show dose-dependent increases from PF-367, PF-618, and <b>51</b> but not <b>29</b>. NPC proliferation is expressed as the percentage of DMSO-treated (100%). (<i>n</i> = 4. Error bars display SEM. Unpaired <i>t</i>-test. *0.01 ≤ <i>p</i> < 0.05, **0.001 ≤ <i>p</i> < 0.01, ***0.0001 ≤ <i>p</i> < 0.001, ****<i>p</i> < 0.0001.) (N) Active oxazole-4-carboxamide GSK-3 inhibitors enhanced Wnt/β-catenin signaling in human NPCs. Dose–response curves of representative inhibitors (<b>1</b>, <b>28</b>, <b>51,</b> and inactive <b>29</b>) with calculated EC<sub>50</sub>s are shown. <i>n</i> = 4. Data show mean values ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/medium/jm9b01030_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Oxazole-4-carboxamide-based probes display robust brain exposure properties in vivo and differential target engagement in a mouse model of AD in vitro. (A) In vitro autoradiography images of [<sup>3</sup>H]<b>1</b> Tg2576 and C57BL/6 mice under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (B) Summed (5 to 30 min) PET images showing the distribution of [<sup>11</sup>C]<b>28</b>, [<sup>11</sup>C]<b>37</b>, and [<sup>11</sup>C]<b>44</b> in comparison with [<sup>11</sup>C]<b>1</b> in the rhesus monkey brain, following microdose injections. Image intensity displayed as 0–2 SUV scale. (C) Quantification for the in vitro autoradiography of [<sup>3</sup>H]<b>1</b> in Tg2576 (20 μm sections, <i>n</i> = 3) and C57BL/6 (20 μm sections, <i>n</i> = 2) mice brains under baseline, homologous blocking (nonradioactive <b>1</b>, 10 μM), and heterologous (LY2090314, 10 μM or CHIR99021, 10 μM) conditions. (D) Representative rhesus monkey regional brain time–activity curves (TACs) after the intravenous injection of [<sup>11</sup>C]<b>44</b>. (E) Representative rhesus monkey regional brain time–activity curves (TACs) after intravenous injection of [<sup>11</sup>C]<b>1</b>. (F) Mice exposure of <b>44</b> (C57BL/6 mice, 50 mg/kg, single dose PO, <i>n</i> = 3). Data show mean values ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01030/20191108/images/large/jm9b01030_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01030&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46777" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46777" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frame, S.</span></span> <span> </span><span class="NLM_article-title">The renaissance of GSK3</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">776</span>, <span class="refDoi"> DOI: 10.1038/35096075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2F35096075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=11584304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvVajsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=769-776&author=P.+Cohenauthor=S.+Frame&title=The+renaissance+of+GSK3&doi=10.1038%2F35096075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: The renaissance of GSK3</span></div><div class="casAuthors">Cohen, Philip; Frame, Sheelagh</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">769-776</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 67 refs.  Glycogen synthase kinase 3 (GSK3) was initially described as a key enzyme involved in glycogen metab., but is now known to regulate a diverse array of cell functions.  The study of the substrate specificity and regulation of GSK3 activity has been important in the quest for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_fen3f-nOX7Vg90H21EOLACvtfcHk0lh5nur75Lo0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvVajsLc%253D&md5=00db2e5c9abcece9317bee545c1a1eb9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F35096075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35096075%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DFrame%26aufirst%3DS.%26atitle%3DThe%2520renaissance%2520of%2520GSK3%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2001%26volume%3D2%26spage%3D769%26epage%3D776%26doi%3D10.1038%2F35096075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hur, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.-Q.</span></span> <span> </span><span class="NLM_article-title">GSK3 signalling in neural development</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nrn2870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrn2870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVGisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=539-551&author=E.-M.+Hurauthor=F.-Q.+Zhou&title=GSK3+signalling+in+neural+development&doi=10.1038%2Fnrn2870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">GSK3 signaling in neural development</span></div><div class="casAuthors">Hur, Eun-Mi; Zhou, Feng-Quan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">539-551</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent evidence suggests that glycogen synthase kinase-3 (GSK3) proteins and their upstream and downstream regulators have key roles in many fundamental processes during neurodevelopment.  Disruption of GSK3 signaling adversely affects brain development and is assocd. with several neurodevelopmental disorders.  Here, the authors discuss the mechanisms by which GSK3 activity is regulated in the nervous system and provide an overview of the recent advances in the understanding of how GSK3 signaling controls neurogenesis, neuronal polarization, and axon growth during brain development.  These recent advances suggest that GSK3 is a crucial node that mediates various cellular processes that are controlled by multiple signaling mols., e.g., disrupted in schizophrenia 1 (DISC1), partitioning defective homolog 3 (PAR3), PAR6, and Wnt proteins, that regulate neurodevelopment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiOzmiQawg1rVg90H21EOLACvtfcHk0lh5nur75Lo0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptVGisro%253D&md5=7ad4651b8fd5a97028df1a57be2c4b38</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrn2870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn2870%26sid%3Dliteratum%253Aachs%26aulast%3DHur%26aufirst%3DE.-M.%26aulast%3DZhou%26aufirst%3DF.-Q.%26atitle%3DGSK3%2520signalling%2520in%2520neural%2520development%26jtitle%3DNat.%2520Neurosci.%26date%3D2010%26volume%3D11%26spage%3D539%26epage%3D551%26doi%3D10.1038%2Fnrn2870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, C.</span></span> <span> </span><span class="NLM_article-title">What Are thebona fideGSK3 Substrates?</span>. <i>Int. J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2011</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.4061/2011/505607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.4061%2F2011%2F505607" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2011&publication_year=2011&pages=1-23&author=C.+Sutherland&title=What+Are+thebona+fideGSK3+Substrates%3F&doi=10.4061%2F2011%2F505607"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4061%2F2011%2F505607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4061%252F2011%252F505607%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DC.%26atitle%3DWhat%2520Are%2520thebona%2520fideGSK3%2520Substrates%253F%26jtitle%3DInt.%2520J.%2520Alzheimers%2520Dis.%26date%3D2011%26volume%3D2011%26spage%3D1%26epage%3D23%26doi%3D10.4061%2F2011%2F505607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cross, D. A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andjelkovich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmings, B. A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of glycogen synthase kinase-3 by insulin mediated by protein kinase B</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1038/378785a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2F378785a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=8524413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK28XltFWj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=1995&pages=785-789&author=D.+A.+E.+Crossauthor=D.+R.+Alessiauthor=P.+Cohenauthor=M.+Andjelkovichauthor=B.+A.+Hemmings&title=Inhibition+of+glycogen+synthase+kinase-3+by+insulin+mediated+by+protein+kinase+B&doi=10.1038%2F378785a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of glycogen synthase kinase-3 by insulin mediated by protein kinase B</span></div><div class="casAuthors">Cross, Darren A. E.; Alessi, Dario R.; Cohen, Philip; Andjelkovich, Mirjana; Hemmings, Brian A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">6559</span>),
    <span class="NLM_cas:pages">785-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Agents which prevent the activation of both MAP kinase-activated protein kinase-1 and p70S6k by insulin in vivo do not block the phosphorylation and inhibition of glycogen synthase kinase-3 (GSK3) in L6 myotubes.  Another insulin-stimulated protein kinase inactivates GSK3 under these conditions, and the authors demonstrate that it is the product of the proto-oncogene protein kinase B (PKB, also known as Akt/RAC).  Like the inhibition of GSK3, the activation of PKB is prevented by inhibitors of phosphatidylinositol 3-kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKh6GNVgGhFbVg90H21EOLACvtfcHk0lh5nur75Lo0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltFWj&md5=56723cab1f5412b4e55aca875989d1fa</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F378785a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F378785a0%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DD.%2BA.%2BE.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DAndjelkovich%26aufirst%3DM.%26aulast%3DHemmings%26aufirst%3DB.%2BA.%26atitle%3DInhibition%2520of%2520glycogen%2520synthase%2520kinase-3%2520by%2520insulin%2520mediated%2520by%2520protein%2520kinase%2520B%26jtitle%3DNature%26date%3D1995%26volume%3D378%26spage%3D785%26epage%3D789%26doi%3D10.1038%2F378785a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eldar-Finkelman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenheede, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, E. G.</span></span> <span> </span><span class="NLM_article-title">Inactivation of glycogen synthase kinase-3 by epidermal growth factor is mediated by mitogen-activated protein kinase/p90 ribosomal protein S6 kinase signaling pathway in NIH/3T3 cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">987</span>– <span class="NLM_lpage">990</span>, <span class="refDoi"> DOI: 10.1074/jbc.270.3.987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1074%2Fjbc.270.3.987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=7836418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK2MXjtFOksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=987-990&author=H.+Eldar-Finkelmanauthor=R.+Segerauthor=J.+R.+Vandenheedeauthor=E.+G.+Krebs&title=Inactivation+of+glycogen+synthase+kinase-3+by+epidermal+growth+factor+is+mediated+by+mitogen-activated+protein+kinase%2Fp90+ribosomal+protein+S6+kinase+signaling+pathway+in+NIH%2F3T3+cells&doi=10.1074%2Fjbc.270.3.987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of glycogen synthase kinase-3 by epidermal growth factor is mediated by mitogen-activated protein kinase/p90 ribosomal protein S6 kinase signaling pathway in NIH/3T3 cells</span></div><div class="casAuthors">Eldar-Finkelman, Hagit; Seger, Rony; Vandenheede, Jackie R.; Krebs, Edwin G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">987-90</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The role of the p90 ribosomal protein S6 kinase/mitogen-activated protein kinase (RSK/MAPK) signaling pathway in regulating glycogen synthase kinase-3 (GSK-3) activity was investigated.  In vitro studies showed that GSK-3 was inactivated by 50% upon incubation with RSK purified from epidermal growth factor (EGF)-stimulated NIH/3T3 cells.  Subsequently, the effect of EGF on GSK-3 activity was measured in NIH/3T3 cells that stably overexpressed mutated forms of MAPK kinase (MAPKK).  The activation of RSK by EGF was markedly decreased in cell lines expressing the dominant neg. MAPKK mutants S222A and K97A and was increased in cells expressing the S222E mutant as compared with control cell lines.  EGF induced a rapid decrease in GSK-3β activity (50%) in control and S222E cells; however, only 25 and 10% inhibition in GSK-3β activity was obsd. in cell lines expressing the dominant neg. mutants K97A and S222A, resp., suggesting that inhibition of GSK-3 was partially blocked in these cells.  Apparently, the action of EGF on GSK-3 inactivation is mediated by the RSK/MAPK signaling pathway in NIH/3T3 cells and provide evidence for a mechanism regulating GSK-3 activity in intact cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMwJxIcPF8vLVg90H21EOLACvtfcHk0lje87r5gFPnhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjtFOksr0%253D&md5=410b5eeb1f9b0ebeb4a7412775afd351</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.3.987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.3.987%26sid%3Dliteratum%253Aachs%26aulast%3DEldar-Finkelman%26aufirst%3DH.%26aulast%3DSeger%26aufirst%3DR.%26aulast%3DVandenheede%26aufirst%3DJ.%2BR.%26aulast%3DKrebs%26aufirst%3DE.%2BG.%26atitle%3DInactivation%2520of%2520glycogen%2520synthase%2520kinase-3%2520by%2520epidermal%2520growth%2520factor%2520is%2520mediated%2520by%2520mitogen-activated%2520protein%2520kinase%252Fp90%2520ribosomal%2520protein%2520S6%2520kinase%2520signaling%2520pathway%2520in%2520NIH%252F3T3%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D987%26epage%3D990%26doi%3D10.1074%2Fjbc.270.3.987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular cloning and expression of glycogen synthase kinase-3/factor A</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2431</span>– <span class="NLM_lpage">2438</span>, <span class="refDoi"> DOI: 10.1002/j.1460-2075.1990.tb07419.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2Fj.1460-2075.1990.tb07419.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=2164470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK3MXis1Gr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1990&pages=2431-2438&author=J.+R.+Woodgett&title=Molecular+cloning+and+expression+of+glycogen+synthase+kinase-3%2Ffactor+A&doi=10.1002%2Fj.1460-2075.1990.tb07419.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular cloning and expression of glycogen synthase kinase-3/factor A</span></div><div class="casAuthors">Woodgett, James Robert</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2431-8</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) is a protein-serine kinase implicated in the hormonal control of several regulatory proteins, including glycogen synthase and transcription factor c-jun.  Two classes of rat brain cDNA for this enzyme were isolated (GSK-3α and GSK-3β).  The α-type encoded a 51-kDa polypeptide, the sequence of which included all of the tryptic peptides detd. by protein sequence anal. of purified skeletal muscle GSK-3.  The novel β-type cDNA had the potential to encode a 47-kDA protein with 85% amino acid identity to GSK-3α.  The 2 types of cDNA were the products of distinct genes as detd. by genomic organization and nucleic acid sequence anal.  Both α and β clones exhibited kinase activity when expressed in COS-1 cells and type-specific antibodies to GSK-3 α and β detect proteins of 51 and 47 kDa, resp., in a variety of rat tissue exts., with highest levels of both in brain.  Partial purifn. of GSK-3 activity from bovine resulted in the isolation of active α and β proteins.  The physiol. importance of these 2 proteins in cellular signal transduction is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm736N83asgLVg90H21EOLACvtfcHk0lje87r5gFPnhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXis1Gr&md5=3bacbf348b8ba673831fe9b9ca5cfde6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1990.tb07419.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1990.tb07419.x%26sid%3Dliteratum%253Aachs%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520cloning%2520and%2520expression%2520of%2520glycogen%2520synthase%2520kinase-3%252Ffactor%2520A%26jtitle%3DEMBO%2520J.%26date%3D1990%26volume%3D9%26spage%3D2431%26epage%3D2438%26doi%3D10.1002%2Fj.1460-2075.1990.tb07419.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Glycogen Synthase Kinase 3: A Kinase for All Pathways?</span>. <i>Curr. Top. Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1016/bs.ctdb.2016.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fbs.ctdb.2016.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=28236969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKjs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2017&pages=277-302&author=P.+Patelauthor=J.+R.+Woodgett&title=Glycogen+Synthase+Kinase+3%3A+A+Kinase+for+All+Pathways%3F&doi=10.1016%2Fbs.ctdb.2016.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Glycogen synthase kinase 3: a kinase for all pathways?</span></div><div class="casAuthors">Patel, Prital; Woodgett, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Developmental Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">Protein Kinases in Development and Disease</span>),
    <span class="NLM_cas:pages">277-302</span>CODEN:
                <span class="NLM_cas:coden">CTDBA5</span>;
        ISSN:<span class="NLM_cas:issn">0070-2153</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) is an unusual protein-serine kinase in that it is primarily regulated by inhibition and lies downstream of multiple cell signaling pathways.  This raises a variety of questions in terms of its physiol. role(s), how signaling specificity is maintained and why so many eggs have been placed into one basket.  There are actually two baskets, as there are two isoforms, GSK-3α and β, that are highly related and largely redundant.  Their many substrates range from regulators of cellular metab. to mols. that control growth and differentiation.  In this chapter, we review the characteristics of GSK-3, update progress in understanding the kinase, and try to answer some of the questions raised by its unusual properties.  Indeed, the kinase may trigger transformation in our thinking of how cellular signals are organized and controlled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolxBj01kLDaLVg90H21EOLACvtfcHk0lgFU3c6DCVqZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKjs70%253D&md5=4c219601701e017e9101b3e7a15b301c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fbs.ctdb.2016.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.ctdb.2016.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DGlycogen%2520Synthase%2520Kinase%25203%253A%2520A%2520Kinase%2520for%2520All%2520Pathways%253F%26jtitle%3DCurr.%2520Top.%2520Dev.%2520Biol.%26date%3D2017%26volume%3D123%26spage%3D277%26epage%3D302%26doi%3D10.1016%2Fbs.ctdb.2016.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubie, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, M.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Requirement for glycogen synthase kinase-3β in cell survival and NF-κB activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1038/35017574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2F35017574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10894547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt12rtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=86-90&author=K.+P.+Hoeflichauthor=J.+Luoauthor=E.+A.+Rubieauthor=M.-S.+Tsaoauthor=O.+Jinauthor=J.+R.+Woodgett&title=Requirement+for+glycogen+synthase+kinase-3%CE%B2+in+cell+survival+and+NF-%CE%BAB+activation&doi=10.1038%2F35017574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement for glycogen synthase kinase-3β in cell survival and NF-κB activation</span></div><div class="casAuthors">Hoeflich, Klaus P.; Luo, Juan; Ruble, Elizabeth A.; Tsao, Ming-Sound; Jin, Ou; Woodgett, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6791</span>),
    <span class="NLM_cas:pages">86-90</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3)-α and -β are dosely related protein-serine kinases, which act as inhibitory components of Wnt signalling during embryonic development and cell proliferation in adult tissues.  Insight into the physiol. function of GSK-3 has emerged from genetic anal. in Drosophila, Dictyostelium and yeast.  Here, we show that disruption of the murine GSK-3β gene results in embryonic lethality caused by severe liver degeneration during mid-gestation, a phenotype consistent with excessive tumor necrosis factor (TNF) toxicity, as obsd. in mice lacking genes involved in the activation of the transcription factor activation NF-κB.  GSK-3β-deficient embryos were rescued by inhibition of TNF using an anti-TNF-α antibody.  Fibroblasts from GSK-3β-deficient embryos were hypersensitive to TNF-α and showed reduced NF-κB function.  Lithium treatment (which inhibits GSK-3) sensitized wild-type fibroblasts to TNF and inhibited transactivation of NF-κB.  The early steps leading to NF-κB activation (degrdn. of I-κB and translocation of NF-κB to the nucleus) were unaffected by the loss of GSK-3β, indicating that NF-κB is regulated by GSK-3β at the level of the transcriptional complex.  Thus, GSK-3β facilitates NF-κB function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN2-FF0neYjLVg90H21EOLACvtfcHk0lgFU3c6DCVqZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt12rtbY%253D&md5=a3f2389ac771cbf1eb830171214c79c4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F35017574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35017574%26sid%3Dliteratum%253Aachs%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DRubie%26aufirst%3DE.%2BA.%26aulast%3DTsao%26aufirst%3DM.-S.%26aulast%3DJin%26aufirst%3DO.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DRequirement%2520for%2520glycogen%2520synthase%2520kinase-3%25CE%25B2%2520in%2520cell%2520survival%2520and%2520NF-%25CE%25BAB%2520activation%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D86%26epage%3D90%26doi%3D10.1038%2F35017574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doble, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kockeritz, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span> <span> </span><span class="NLM_article-title">Functional Redundancy of GSK-3α and GSK-3β in Wnt/β-Catenin Signaling Shown by Using an Allelic Series of Embryonic Stem Cell Lines</span>. <i>Dev. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">957</span>– <span class="NLM_lpage">971</span>, <span class="refDoi"> DOI: 10.1016/j.devcel.2007.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.devcel.2007.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17543867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFentrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=957-971&author=B.+W.+Dobleauthor=S.+Patelauthor=G.+A.+Woodauthor=L.+K.+Kockeritzauthor=J.+R.+Woodgett&title=Functional+Redundancy+of+GSK-3%CE%B1+and+GSK-3%CE%B2+in+Wnt%2F%CE%B2-Catenin+Signaling+Shown+by+Using+an+Allelic+Series+of+Embryonic+Stem+Cell+Lines&doi=10.1016%2Fj.devcel.2007.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional redundancy of GSK-3α and GSK-3β in Wnt/β-catenin signaling shown by using an allelic series of embryonic stem cell lines</span></div><div class="casAuthors">Doble, Bradley W.; Patel, Satish; Wood, Geoffrey A.; Kockeritz, Lisa K.; Woodgett, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">957-971</span>CODEN:
                <span class="NLM_cas:coden">DCEEBE</span>;
        ISSN:<span class="NLM_cas:issn">1534-5807</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In mammalian cells, glycogen synthase kinase-3 (GSK-3) exists as two homologs, GSK-3α and GSK-3β, encoded by independent genes, which share similar kinase domains but differ substantially in their termini.  Here, we describe the generation of an allelic series of mouse embryonic stem cell (ESC) lines with 0-4 functional GSK-3 alleles and examine GSK-3-isoform function in Wnt/β-catenin signaling.  No compensatory upregulation in GSK-3 protein levels or activity was detected in cells lacking either GSK-3α or GSK-3β, and Wnt/β-catenin signaling was normal.  Only in cells lacking three or all four of the alleles was a gene-dosage effect on β-catenin/TCF-mediated transcription obsd.  Indeed, GSK-3α/β double-knockout ESCs displayed hyperactivated Wnt/β-catenin signaling and were severely compromised in their ability to differentiate, but could be rescued to normality by re-expression of functional GSK-3.  The rheostatic regulation of GSK-3 highlights the importance of considering the contributions of both homologs when studying GSK-3 functions in mammalian systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJw04huRADqLVg90H21EOLACvtfcHk0lgFU3c6DCVqZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsFentrg%253D&md5=8fae370755d55ce24159061f8436c265</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2007.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2007.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DDoble%26aufirst%3DB.%2BW.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DG.%2BA.%26aulast%3DKockeritz%26aufirst%3DL.%2BK.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26atitle%3DFunctional%2520Redundancy%2520of%2520GSK-3%25CE%25B1%2520and%2520GSK-3%25CE%25B2%2520in%2520Wnt%252F%25CE%25B2-Catenin%2520Signaling%2520Shown%2520by%2520Using%2520an%2520Allelic%2520Series%2520of%2520Embryonic%2520Stem%2520Cell%2520Lines%26jtitle%3DDev.%2520Cell%26date%3D2007%26volume%3D12%26spage%3D957%26epage%3D971%26doi%3D10.1016%2Fj.devcel.2007.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Killick, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovestone, S.</span></span> <span> </span><span class="NLM_article-title">The GSK3 hypothesis of Alzheimer’s disease</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">1433</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2007.05194.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1111%2Fj.1471-4159.2007.05194.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=18088381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkt1Citbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2008&pages=1433-1439&author=C.+Hooperauthor=R.+Killickauthor=S.+Lovestone&title=The+GSK3+hypothesis+of+Alzheimer%E2%80%99s+disease&doi=10.1111%2Fj.1471-4159.2007.05194.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The GSK3 hypothesis of Alzheimer's disease</span></div><div class="casAuthors">Hooper, Claudie; Killick, Richard; Lovestone, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1433-1439</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Glycogen synthase kinase 3 (GSK3) is a constitutively active, proline-directed serine/threonine kinase that plays a part in a no. of physiol. processes ranging from glycogen metab. to gene transcription.  GSK3 also plays a pivotal and central role in the pathogenesis of both sporadic and familial forms of Alzheimer's disease (AD), an observation that has led us to coin the 'GSK3 hypothesis of AD'.  According to this hypothesis, over-activity of GSK3 accounts for memory impairment, tau hyper-phosphorylation, increased β-amyloid prodn. and local plaque-assocd. microglial-mediated inflammatory responses; all of which are hallmark characteristics of AD.  If our 'GSK3 hypothesis of AD' is substantiated and GSK3 is indeed a causal mediator of AD then inhibitors of GSK3 would provide a novel avenue for therapeutic intervention in this devastating disorder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9lxWCe90horVg90H21EOLACvtfcHk0liDl6rBBOD9Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkt1Citbw%253D&md5=6e5f9918668aac2d3127bacad12a8340</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2007.05194.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2007.05194.x%26sid%3Dliteratum%253Aachs%26aulast%3DHooper%26aufirst%3DC.%26aulast%3DKillick%26aufirst%3DR.%26aulast%3DLovestone%26aufirst%3DS.%26atitle%3DThe%2520GSK3%2520hypothesis%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Neurochem.%26date%3D2008%26volume%3D104%26spage%3D1433%26epage%3D1439%26doi%3D10.1111%2Fj.1471-4159.2007.05194.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballatore, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojanowski, J. Q.</span></span> <span> </span><span class="NLM_article-title">Tau-mediated neurodegeneration in Alzheimer’s disease and related disorders</span>. <i>Nat. Rev. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1038/nrn2194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrn2194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17684513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpt1Crt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=663-672&author=C.+Ballatoreauthor=V.+M.-Y.+Leeauthor=J.+Q.+Trojanowski&title=Tau-mediated+neurodegeneration+in+Alzheimer%E2%80%99s+disease+and+related+disorders&doi=10.1038%2Fnrn2194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Tau-mediated neurodegeneration in Alzheimer's disease and related disorders</span></div><div class="casAuthors">Ballatore, Carlo; Lee, Virginia M.-Y.; Trojanowski, John Q.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">663-672</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Advances in the understanding of the mechanisms of tau-mediated neurodegeneration in Alzheimer's disease (AD) and related tauopathies, which are characterized by prominent CNS accumulations of fibrillar tau inclusions, are rapidly moving this previously underexplored disease pathway to center stage for disease-modifying drug discovery efforts.  However, controversies abound concerning whether or not the deleterious effects of tau pathologies result from toxic gains-of-function by pathol. tau or from crit. losses of normal tau function in the disease state.  This review summarizes the most recent advances in the authors' knowledge of the mechanisms of tau-mediated neurodegeneration to forge an integrated concept of those tau-linked disease processes that drive the onset and progression of AD and related tauopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8-k_a3SnV7bVg90H21EOLACvtfcHk0liDl6rBBOD9Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpt1Crt78%253D&md5=eef3cc99033075b3215b739521d672f6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrn2194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn2194%26sid%3Dliteratum%253Aachs%26aulast%3DBallatore%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DV.%2BM.-Y.%26aulast%3DTrojanowski%26aufirst%3DJ.%2BQ.%26atitle%3DTau-mediated%2520neurodegeneration%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%2520and%2520related%2520disorders%26jtitle%3DNat.%2520Rev.%2520Neurosci.%26date%3D2007%26volume%3D8%26spage%3D663%26epage%3D672%26doi%3D10.1038%2Fnrn2194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adlard, P. A.</span></span> <span> </span><span class="NLM_article-title">GSK-3 in neurodegenerative diseases</span>. <i>Int. J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2011</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.4061/2011/189246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.4061%2F2011%2F189246" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2011&publication_year=2011&pages=1-9&author=P.+Leiauthor=S.+Aytonauthor=A.+I.+Bushauthor=P.+A.+Adlard&title=GSK-3+in+neurodegenerative+diseases&doi=10.4061%2F2011%2F189246"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4061%2F2011%2F189246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4061%252F2011%252F189246%26sid%3Dliteratum%253Aachs%26aulast%3DLei%26aufirst%3DP.%26aulast%3DAyton%26aufirst%3DS.%26aulast%3DBush%26aufirst%3DA.%2BI.%26aulast%3DAdlard%26aufirst%3DP.%2BA.%26atitle%3DGSK-3%2520in%2520neurodegenerative%2520diseases%26jtitle%3DInt.%2520J.%2520Alzheimers%2520Dis.%26date%3D2011%26volume%3D2011%26spage%3D1%26epage%3D9%26doi%3D10.4061%2F2011%2F189246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phiel, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, P. S.</span></span> <span> </span><span class="NLM_article-title">GSK-3α regulates production of Alzheimer’s disease amyloid-β peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>423</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1038/nature01640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnature01640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=12761548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvFaltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2003&pages=435-439&author=C.+J.+Phielauthor=C.+A.+Wilsonauthor=V.+M.-Y.+Leeauthor=P.+S.+Klein&title=GSK-3%CE%B1+regulates+production+of+Alzheimer%E2%80%99s+disease+amyloid-%CE%B2+peptides&doi=10.1038%2Fnature01640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">GSK-3α regulates production of Alzheimer's disease amyloid-β peptides</span></div><div class="casAuthors">Phiel, Christopher J.; Wilson, Christina A.; Lee, Virginia M.-Y.; Klein, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">6938</span>),
    <span class="NLM_cas:pages">435-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Alzheimer's disease is assocd. with increased prodn. and aggregation of amyloid-β (Aβ) peptides.  Aβ peptides are derived from the amyloid precursor protein (APP) by sequential proteolysis, catalyzed by the aspartyl protease BACE, followed by presenilin-dependent γ-secretase cleavage.  Presenilin interacts with nicastrin, APH-1 and PEN-2, all of which are required for γ-secretase function.  Presenilins also interact with α-catenin, β-catenin and glycogen synthase kinase-3β (GSK-3β), but a functional role for these proteins in γ-secretase activity has not been established.  Here the authors show that therapeutic concns. of lithium, a GSK-3 inhibitor, block the prodn. of Aβ peptides by interfering with APP cleavage at the γ-secretase step, but do not inhibit Notch processing.  Importantly, lithium also blocks the accumulation of Aβ peptides in the brains of mice that overproduce APP.  The target of lithium in this setting is GSK-3α, which is required for maximal processing of APP.  Since GSK-3 also phosphorylates tau protein, the principal component of neurofibrillary tangles, inhibition of GSK-3α offers a new approach to reduce the formation of both amyloid plaques and neurofibrillary tangles, two pathol. hallmarks of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQu3H2UA-dbbVg90H21EOLACvtfcHk0lgN5_Wechbkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvFaltLY%253D&md5=4f2db433242c3ad2f510a05dffe6f5d4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature01640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01640%26sid%3Dliteratum%253Aachs%26aulast%3DPhiel%26aufirst%3DC.%2BJ.%26aulast%3DWilson%26aufirst%3DC.%2BA.%26aulast%3DLee%26aufirst%3DV.%2BM.-Y.%26aulast%3DKlein%26aufirst%3DP.%2BS.%26atitle%3DGSK-3%25CE%25B1%2520regulates%2520production%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%2520amyloid-%25CE%25B2%2520peptides%26jtitle%3DNature%26date%3D2003%26volume%3D423%26spage%3D435%26epage%3D439%26doi%3D10.1038%2Fnature01640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, B.</span></span> <span> </span><span class="NLM_article-title">Divergent roles of GSK3 and CDK5 in APP processing</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>312</i></span>,  <span class="NLM_fpage">922</span>– <span class="NLM_lpage">929</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2003.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.bbrc.2003.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=14651959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptl2qu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=312&publication_year=2003&pages=922-929&author=J.+Ryderauthor=Y.+Suauthor=F.+Liuauthor=B.+Liauthor=Y.+Zhouauthor=B.+Ni&title=Divergent+roles+of+GSK3+and+CDK5+in+APP+processing&doi=10.1016%2Fj.bbrc.2003.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent roles of GSK3 and CDK5 in APP processing</span></div><div class="casAuthors">Ryder, John; Su, Yuan; Liu, Feng; Li, Baolin; Zhou, Yan; Ni, Binhui</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">312</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">922-929</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK3) and cyclin-dependent kinase 5 (CDK5) are related serine/threonine kinases that have been well studied for their role in tau hyperphosphorylation, however, little is known about their significance in APP processing.  Here we report that GSK3 and CDK5 are involved in APP processing in a divergent manner.  Specific inhibition of cellular GSK3 by lithium or GSK3β antisense elicits a redn. in Aβ.  Conversely, neg. modulation of cellular CDK5 activity by CDK5 inhibitor, roscovitine, or CDK5 antisense stimulates Aβ prodn.  Neither GSK3 nor CDK5 inhibition by these means significantly affected cellular APP levels or APP maturation.  Moreover, oral administration of lithium significantly reduces Aβ prodn. whereas direct ICV administration of roscovitine augmented Aβ prodn. in the brains of PDAPP (APPV717F) mice.  Our data support a function for both GSK3 and CDK5 in APP processing, further implicating these two kinases in the pathogenesis of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHF5xGtT6VGLVg90H21EOLACvtfcHk0livuxhnfIal_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptl2qu7o%253D&md5=cfa42655074ec97afdd76b5109844562</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2003.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2003.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DRyder%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DNi%26aufirst%3DB.%26atitle%3DDivergent%2520roles%2520of%2520GSK3%2520and%2520CDK5%2520in%2520APP%2520processing%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2003%26volume%3D312%26spage%3D922%26epage%3D929%26doi%3D10.1016%2Fj.bbrc.2003.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klementiev, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikova, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novitskaya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walmod, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dmytriyeva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pakkenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, E.</span></span> <span> </span><span class="NLM_article-title">A neural cell adhesion molecule-derived peptide reduces neuropathological signs and cognitive impairment induced by Aβ25-35</span>. <i>Neuroscience</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1016/j.neuroscience.2006.11.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.neuroscience.2006.11.060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17223274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVGms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2007&pages=209-224&author=B.+Klementievauthor=T.+Novikovaauthor=V.+Novitskayaauthor=P.+S.+Walmodauthor=O.+Dmytriyevaauthor=B.+Pakkenbergauthor=V.+Berezinauthor=E.+Bock&title=A+neural+cell+adhesion+molecule-derived+peptide+reduces+neuropathological+signs+and+cognitive+impairment+induced+by+A%CE%B225-35&doi=10.1016%2Fj.neuroscience.2006.11.060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A neural cell adhesion molecule-derived peptide reduces neuropathological signs and cognitive impairment induced by Aβ25-35</span></div><div class="casAuthors">Klementiev, B.; Novikova, T.; Novitskaya, V.; Walmod, P. S.; Dmytriyeva, O.; Pakkenberg, B.; Berezin, V.; Bock, E.</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">209-224</span>CODEN:
                <span class="NLM_cas:coden">NRSCDN</span>;
        ISSN:<span class="NLM_cas:issn">0306-4522</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">By means of i.c.v. administration of pre-aggregated oligomeric β-amyloid (Aβ)25-35 peptide it was possible in rats to generate neuropathol. signs related to those of early stages of Alzheimer's disease (AD).  Aβ25-35-administration induced the deposition of endogenously produced amyloid protein.  Furthermore, quant. immunohistochem. demonstrated time-related statistically significant increases in amyloid immunoreactivity, tau phosphorylation, microglial activation, and astrocytosis, and stereol. investigations demonstrated statistically significant increased neuronal cell death and brain atrophy in response to Aβ25-35.  Finally, the Aβ25-35-administration led to a reduced short-term memory as detd. by the social recognition test.  A synthetic peptide termed FGL derived from the neural cell adhesion mol. (NCAM) was able to prevent or, if already manifest, strongly reduce all investigated signs of Aβ25-35-induced neuropathol. and cognitive impairment.  The FGL peptide was recently demonstrated to be able to cross the blood-brain-barrier.  Accordingly, we found that the beneficial effects of FGL were achieved not only by intracisternal, but also by intranasal and s.c. administration of the peptide.  Furthermore, FGL-treatment was shown to inhibit the activity of GSK3β, a kinase implicated in signaling regulating cell survival, tau phosphorylation and the processing of the amyloid precursor protein (APP).  Thus, the peptide induced a statistically significant increase in the fraction of GSK3β phosphorylated on the Ser9-position, a posttranslational modification known to inhibit the activity of the kinase.  Hence, the mode of action of FGL with respect to the preventive and curative effects on Aβ25-35-induced neuropathol. manifestations and cognitive impairment involves the modulation of intracellular signal-transduction mediated through GSK3β.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogrbF8iKc417Vg90H21EOLACvtfcHk0livuxhnfIal_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVGms7w%253D&md5=3439510cfedc40d220b8dcda52511e57</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2006.11.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2006.11.060%26sid%3Dliteratum%253Aachs%26aulast%3DKlementiev%26aufirst%3DB.%26aulast%3DNovikova%26aufirst%3DT.%26aulast%3DNovitskaya%26aufirst%3DV.%26aulast%3DWalmod%26aufirst%3DP.%2BS.%26aulast%3DDmytriyeva%26aufirst%3DO.%26aulast%3DPakkenberg%26aufirst%3DB.%26aulast%3DBerezin%26aufirst%3DV.%26aulast%3DBock%26aufirst%3DE.%26atitle%3DA%2520neural%2520cell%2520adhesion%2520molecule-derived%2520peptide%2520reduces%2520neuropathological%2520signs%2520and%2520cognitive%2520impairment%2520induced%2520by%2520A%25CE%25B225-35%26jtitle%3DNeuroscience%26date%3D2007%26volume%3D145%26spage%3D209%26epage%3D224%26doi%3D10.1016%2Fj.neuroscience.2006.11.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanger, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brion, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, B. H.</span></span> <span> </span><span class="NLM_article-title">Glycogen synthase kinase-3 induces Alzheimer’s disease-like phosphorylation of tau: generation of paired helical filament epitopes and neuronal localisation of the kinase</span>. <i>Neurosci. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/0304-3940(92)90774-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2F0304-3940%2892%2990774-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1336152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK3sXpvFGksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=1992&pages=58-62&author=D.+P.+Hangerauthor=K.+Hughesauthor=J.+R.+Woodgettauthor=J.-P.+Brionauthor=B.+H.+Anderton&title=Glycogen+synthase+kinase-3+induces+Alzheimer%E2%80%99s+disease-like+phosphorylation+of+tau%3A+generation+of+paired+helical+filament+epitopes+and+neuronal+localisation+of+the+kinase&doi=10.1016%2F0304-3940%2892%2990774-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Glycogen synthase kinase-3 induces Alzheimer's disease-like phosphorylation of tau:  generation of paired helical filament epitopes and neuronal localization of the kinase</span></div><div class="casAuthors">Hanger, Diane P.; Hughes, Kenneth; Woodgett, James R.; Brion, Jean Pierre; Anderton, Brian H.</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-62</span>CODEN:
                <span class="NLM_cas:coden">NELED5</span>;
        ISSN:<span class="NLM_cas:issn">0304-3940</span>.
    </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) reduced the mobility of human tau on SDS-PAGE, prevented binding of the monoclonal antibody (mAb) Tau. 1, and induced binding of the mAb 8D8.  Recombinant tau phosphorylated by GSK-3 aligned on SDS-PAGE with the abnormally phosphorylated tau (PHF-tau) assocd. with the paired helical filaments in Alzheimer's disease brain.  Phosphorylated serine396 (numbering of the largest human brain tau isoform) was identified as a binding site on tau for mAb 8D8.  The localization of GSK-3 within granular structures in pyramidal cells indicates that GSK-3α and GSK-3β may have a role in the prodn. of PHF-tau in Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOM1_KaxlV9LVg90H21EOLACvtfcHk0livuxhnfIal_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXpvFGksg%253D%253D&md5=88d0a3271ce167362ce8b363bae5dbf4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2F0304-3940%2892%2990774-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3940%252892%252990774-2%26sid%3Dliteratum%253Aachs%26aulast%3DHanger%26aufirst%3DD.%2BP.%26aulast%3DHughes%26aufirst%3DK.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26aulast%3DBrion%26aufirst%3DJ.-P.%26aulast%3DAnderton%26aufirst%3DB.%2BH.%26atitle%3DGlycogen%2520synthase%2520kinase-3%2520induces%2520Alzheimer%25E2%2580%2599s%2520disease-like%2520phosphorylation%2520of%2520tau%253A%2520generation%2520of%2520paired%2520helical%2520filament%2520epitopes%2520and%2520neuronal%2520localisation%2520of%2520the%2520kinase%26jtitle%3DNeurosci.%2520Lett.%26date%3D1992%26volume%3D147%26spage%3D58%26epage%3D62%26doi%3D10.1016%2F0304-3940%2892%2990774-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishiguro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imahori, K.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation sites on tau by tau protein kinase I, a bovine derived kinase generating an epitope of paired helical filaments</span>. <i>Neurosci. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1016/0304-3940(92)90839-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2F0304-3940%2892%2990839-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1284442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADyaK3sXhsVCqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=1992&pages=202-206&author=K.+Ishiguroauthor=A.+Omoriauthor=M.+Takamatsuauthor=K.+Satoauthor=M.+Ariokaauthor=T.+Uchidaauthor=K.+Imahori&title=Phosphorylation+sites+on+tau+by+tau+protein+kinase+I%2C+a+bovine+derived+kinase+generating+an+epitope+of+paired+helical+filaments&doi=10.1016%2F0304-3940%2892%2990839-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation sites on tau by tau protein kinase I, a bovine derived kinase generating an epitope of paired helical filaments</span></div><div class="casAuthors">Ishiguro, Koichi; Omori, Akira; Takamatsu, Masako; Sato, Kazuki; Arioka, Manabu; Uchida, Tsuneko; Imahori, Kazutomo</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">202-6</span>CODEN:
                <span class="NLM_cas:coden">NELED5</span>;
        ISSN:<span class="NLM_cas:issn">0304-3940</span>.
    </div><div class="casAbstract">Tau protein kinase I (TPKI) isolated from bovine brain has been detd. to phosphorylate tau at four distinct sites by detecting modified Ser and Thr residues with protein sequencer.  Ser199, Thr231, Ser396, and Ser413 were phosphorylated by TPKI (numbering of amino acids was done in relation to the longest human tau).  These phosphorylations generate an epitope of PHF (paired helical filaments) and eliminate the recognition of tau by the monoclonal antibody, tau-1.  Thus, TPKI might be responsible for at least some of the phosphorylation of tau to induce PHF formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfG_9kTJOSILVg90H21EOLACvtfcHk0lgIb9WAuP27Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhsVCqurw%253D&md5=0fb06cefdd89a7bce3449e61ec9f95f6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2F0304-3940%2892%2990839-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3940%252892%252990839-y%26sid%3Dliteratum%253Aachs%26aulast%3DIshiguro%26aufirst%3DK.%26aulast%3DOmori%26aufirst%3DA.%26aulast%3DTakamatsu%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DArioka%26aufirst%3DM.%26aulast%3DUchida%26aufirst%3DT.%26aulast%3DImahori%26aufirst%3DK.%26atitle%3DPhosphorylation%2520sites%2520on%2520tau%2520by%2520tau%2520protein%2520kinase%2520I%252C%2520a%2520bovine%2520derived%2520kinase%2520generating%2520an%2520epitope%2520of%2520paired%2520helical%2520filaments%26jtitle%3DNeurosci.%2520Lett.%26date%3D1992%26volume%3D148%26spage%3D202%26epage%3D206%26doi%3D10.1016%2F0304-3940%2892%2990839-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, J.</span></span> <span> </span><span class="NLM_article-title">Tau-based therapies in neurodegeneration: opportunities and challenges</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">883</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrd.2017.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=28983098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=863-883&author=C.+Liauthor=J.+G%C3%B6tz&title=Tau-based+therapies+in+neurodegeneration%3A+opportunities+and+challenges&doi=10.1038%2Fnrd.2017.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Tau-based therapies in neurodegeneration: opportunities and challenges</span></div><div class="casAuthors">Li, Chuanzhou; Gotz, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">863-883</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Aggregates of the microtubule-assocd. protein tau are a defining feature of several neurodegenerative diseases that are collectively known as tauopathies, and constitute one of the hallmark lesions of Alzheimer disease (AD).  Given the lack of efficacy to date of amyloid-β-targeted therapies for AD, interest is growing in tau as a potential alternative target.  Several drug candidates, which are now in clin. trials, aim to reduce tau levels or to prevent the aggregation or pathol. post-translation modifications of this protein.  In this Review, we discuss preclin. and clin. studies in light of an increased understanding of the physiol. and pathol. roles of tau, advances in animal models of tauopathy, the identification of novel targets and the availability of novel tracers to track tau.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSC7Md8i-637Vg90H21EOLACvtfcHk0lgIb9WAuP27Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqt7jF&md5=389b4be6419498344b5e53ca1be9c22e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.155%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DG%25C3%25B6tz%26aufirst%3DJ.%26atitle%3DTau-based%2520therapies%2520in%2520neurodegeneration%253A%2520opportunities%2520and%2520challenges%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D863%26epage%3D883%26doi%3D10.1038%2Fnrd.2017.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zehr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olm, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suleman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaynor, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFrancois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinstein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duff, K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of glycogen synthase kinase-3 by lithium correlates with reduced tauopathy and degeneration in vivo</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">6990</span>– <span class="NLM_lpage">6995</span>, <span class="refDoi"> DOI: 10.1073/pnas.0500466102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1073%2Fpnas.0500466102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=15867159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6990-6995&author=W.+Nobleauthor=E.+Planelauthor=C.+Zehrauthor=V.+Olmauthor=J.+Meyersonauthor=F.+Sulemanauthor=K.+Gaynorauthor=L.+Wangauthor=J.+LaFrancoisauthor=B.+Feinsteinauthor=M.+Burnsauthor=P.+Krishnamurthyauthor=Y.+Wenauthor=R.+Bhatauthor=J.+Lewisauthor=D.+Dicksonauthor=K.+Duff&title=Inhibition+of+glycogen+synthase+kinase-3+by+lithium+correlates+with+reduced+tauopathy+and+degeneration+in+vivo&doi=10.1073%2Fpnas.0500466102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of glycogen synthase kinase-3 by lithium correlates with reduced tauopathy and degeneration in vivo</span></div><div class="casAuthors">Noble, Wendy; Planel, Emmanuel; Zehr, Cindy; Olm, Vicki; Meyerson, Jordana; Suleman, Farhana; Gaynor, Kate; Wang, Lili; LaFrancois, John; Feinstein, Boris; Burns, Mark; Krishnamurthy, Pavan; Wen, Yi; Bhat, Ratan; Lewis, Jada; Dickson, Dennis; Duff, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6990-6995</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neurofibrillary tangles composed of hyperphosphorylated, aggregated tau are a common pathol. feature of tauopathies, including Alzheimer's disease.  Abnormal phosphorylation of tau by kinases or phosphatases has been proposed as a pathogenic mechanism in tangle formation.  To investigate whether kinase inhibition can reduce tauopathy and the degeneration assocd. with it in vivo, transgenic mice overexpressing mutant human tau were treated with the glycogen synthase kinase-3 (GSK-3) inhibitor lithium chloride.  Treatment resulted in significant inhibition of GSK-3 activity.  Lithium administration also resulted in significantly lower levels of phosphorylation at several epitopes of tau known to be hyperphosphorylated in Alzheimer's disease and significantly reduced levels of aggregated, insol. tau.  Administration of a second GSK-3 inhibitor also correlated with reduced insol. tau levels, supporting the idea that lithium exerts its effect through GSK-3 inhibition.  Levels of aggregated tau correlated strongly with degree of axonal degeneration, and lithium-chloride-treated mice showed less degeneration if administration was started during early stages of tangle development.  These results support the idea that kinases are involved in tauopathy progression and that kinase inhibitors may be effective therapeutically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE2qlkQMP4FLVg90H21EOLACvtfcHk0lihsQqhIq5w7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKns78%253D&md5=b3f258b86536144dc5d2e483abca4f36</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0500466102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0500466102%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DW.%26aulast%3DPlanel%26aufirst%3DE.%26aulast%3DZehr%26aufirst%3DC.%26aulast%3DOlm%26aufirst%3DV.%26aulast%3DMeyerson%26aufirst%3DJ.%26aulast%3DSuleman%26aufirst%3DF.%26aulast%3DGaynor%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLaFrancois%26aufirst%3DJ.%26aulast%3DFeinstein%26aufirst%3DB.%26aulast%3DBurns%26aufirst%3DM.%26aulast%3DKrishnamurthy%26aufirst%3DP.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DBhat%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DJ.%26aulast%3DDickson%26aufirst%3DD.%26aulast%3DDuff%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520glycogen%2520synthase%2520kinase-3%2520by%2520lithium%2520correlates%2520with%2520reduced%2520tauopathy%2520and%2520degeneration%2520in%2520vivo%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D6990%26epage%3D6995%26doi%3D10.1073%2Fpnas.0500466102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Ramos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morán, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, J.</span></span> <span> </span><span class="NLM_article-title">Decreased nuclear beta-catenin, tau hyperphosphorylation and neurodegeneration in GSK-3beta conditional transgenic mice</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1093/emboj/20.1.27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1093%2Femboj%2F20.1.27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=11226152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvVCmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=27-39&author=J.+J.+Lucasauthor=F.+Hern%C3%A1ndezauthor=P.+G%C3%B3mez-Ramosauthor=M.+A.+Mor%C3%A1nauthor=R.+Henauthor=J.+Avila&title=Decreased+nuclear+beta-catenin%2C+tau+hyperphosphorylation+and+neurodegeneration+in+GSK-3beta+conditional+transgenic+mice&doi=10.1093%2Femboj%2F20.1.27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Decreased nuclear β-catenin, tau hyperphosphorylation and neurodegeneration in GSK-3β conditional transgenic mice</span></div><div class="casAuthors">Lucas, Jose J.; Hernandez, Felix; Gomez-Ramos, Pilar; Moran, Maria A.; Hen, Rene; Avila, Jesus</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1 & 2</span>),
    <span class="NLM_cas:pages">27-39</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3β (GSK-3β) has been postulated to mediate Alzheimer's disease tau hyperphosphorylation, β-amyloid-induced neurotoxicity and presenilin-1 mutation pathogenic effects.  By using the tet-regulated system we have produced conditional transgenic mice overexpressing GSK-3β in the brain during adulthood while avoiding perinatal lethality due to embryonic transgene expression.  These mice show decreased levels of nuclear β-catenin and hyperphosphorylation of tau in hippocampal neurons, the latter resulting in pretangle-like somatodendritic localization of tau.  Neurons displaying somatodendritic localization of tau often show abnormal morphologies and detachment from the surrounding neuropil.  Reactive astrocytosis and microgliosis were also indicative of neuronal stress and death.  This was further confirmed by TUNEL and cleaved caspase-3 immunostaining of dentate gyrus granule cells.  Our results demonstrate that in vivo overexpression of GSK-3β results in neurodegeneration and suggest that these mice can be used as an animal model to study the relevance of GSK-3β deregulation to the pathogenesis of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9riwM95AslLVg90H21EOLACvtfcHk0lihsQqhIq5w7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvVCmt78%253D&md5=70a7fa8aa64bb07c4b2cea1802f63e10</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F20.1.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F20.1.27%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DJ.%2BJ.%26aulast%3DHern%25C3%25A1ndez%26aufirst%3DF.%26aulast%3DG%25C3%25B3mez-Ramos%26aufirst%3DP.%26aulast%3DMor%25C3%25A1n%26aufirst%3DM.%2BA.%26aulast%3DHen%26aufirst%3DR.%26aulast%3DAvila%26aufirst%3DJ.%26atitle%3DDecreased%2520nuclear%2520beta-catenin%252C%2520tau%2520hyperphosphorylation%2520and%2520neurodegeneration%2520in%2520GSK-3beta%2520conditional%2520transgenic%2520mice%26jtitle%3DEMBO%2520J.%26date%3D2001%26volume%3D20%26spage%3D27%26epage%3D39%26doi%3D10.1093%2Femboj%2F20.1.27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundke-iqbal, I.</span></span> <span> </span><span class="NLM_article-title">Distribution, levels, and activity of glycogen synthase kinase-3 in the Alzheimer disease brain</span>. <i>J. Neuropathol. Exp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1097/00005072-199701000-00007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1097%2F00005072-199701000-00007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=8990130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A280%3ADyaK2s7ksVelsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1997&pages=70-78&author=J.-J.+Peiauthor=T.+Tanakaauthor=Y.-C.+Tungauthor=E.+Braakauthor=K.+Iqbalauthor=I.+Grundke-iqbal&title=Distribution%2C+levels%2C+and+activity+of+glycogen+synthase+kinase-3+in+the+Alzheimer+disease+brain&doi=10.1097%2F00005072-199701000-00007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Distribution, levels, and activity of glycogen synthase kinase-3 in the Alzheimer disease brain</span></div><div class="casAuthors">Pei J J; Tanaka T; Tung Y C; Braak E; Iqbal K; Grundke-Iqbal I</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuropathology and experimental neurology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-8</span>
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    </div><div class="casAbstract">A number of studies have implicated a proline-directed protein kinase, glycogen synthase kinase-3 (GSK-3) in the hyperphosphorylation of tau in Alzheimer's disease (AD).  Toward understanding the role of GSK-3 in the abnormal hyperphosphorylation of tau in AD we have found that GSK-3 is prominently present in neuronal cell bodies and their processes and co-localizes with neurofibrillary changes in AD brain.  Furthermore, the levels of GSK-3 as determined by indirect ELISA are approximately 50% increased in the postsynaptosomal supernatant from AD brains as compared to the controls.  However, no increase in GSK-3 enzyme activity was detected.  In AD brain, with its reduced phosphatase activity, even normal levels of GSK-3 activity might be sufficient for the hyperphosphorylation of tau.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQmu8m1y1XLJ0EgZCMqtRBifW6udTcc2eZO7SubKp9ymrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2s7ksVelsQ%253D%253D&md5=18b0957c0a36ecb25875aaa4e8c6bc91</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1097%2F00005072-199701000-00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00005072-199701000-00007%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DJ.-J.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DTung%26aufirst%3DY.-C.%26aulast%3DBraak%26aufirst%3DE.%26aulast%3DIqbal%26aufirst%3DK.%26aulast%3DGrundke-iqbal%26aufirst%3DI.%26atitle%3DDistribution%252C%2520levels%252C%2520and%2520activity%2520of%2520glycogen%2520synthase%2520kinase-3%2520in%2520the%2520Alzheimer%2520disease%2520brain%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D1997%26volume%3D56%26spage%3D70%26epage%3D78%26doi%3D10.1097%2F00005072-199701000-00007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blalock, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geddes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markesbery, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landfield, P. W.</span></span> <span> </span><span class="NLM_article-title">Incipient Alzheimer’s disease: Microarray correlation analyses reveal major transcriptional and tumor suppressor responses</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">2173</span>– <span class="NLM_lpage">2178</span>, <span class="refDoi"> DOI: 10.1073/pnas.0308512100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1073%2Fpnas.0308512100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=14769913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1SrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=2173-2178&author=E.+M.+Blalockauthor=J.+W.+Geddesauthor=K.+C.+Chenauthor=N.+M.+Porterauthor=W.+R.+Markesberyauthor=P.+W.+Landfield&title=Incipient+Alzheimer%E2%80%99s+disease%3A+Microarray+correlation+analyses+reveal+major+transcriptional+and+tumor+suppressor+responses&doi=10.1073%2Fpnas.0308512100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Incipient Alzheimer's disease: Microarray correlation analyses reveal major transcriptional and tumor suppressor responses</span></div><div class="casAuthors">Blalock, Eric M.; Geddes, James W.; Chen, Kuey Chu; Porter, Nada M.; Markesbery, William R.; Landfield, Philip W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2173-2178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The pathogenesis of incipient Alzheimer's disease (AD) has been resistant to anal. because of the complexity of AD and the overlap of its early-stage markers with normal aging.  Gene microarrays provide new tools for addressing complexity because they allow overviews of the simultaneous activity of multiple cellular pathways.  However, microarray data interpretation is often hindered by low statistical power, high false pos. or false negatives, and by uncertain relevance to functional end-points.  Here, the authors analyzed hippocampal gene expression of nine control and 22 AD subjects of varying severity on 31 sep. microarrays.  The authors then tested the correlation of each gene's expression with MiniMental Status Examn. (MMSE) and neurofibrillary tangle (NFT) scores across all 31 subjects regardless of diagnosis.  These well powered tests revealed a major transcriptional response comprising thousands of genes significantly correlated with AD markers.  Several hundred of these genes were also correlated with AD markers across only control and incipient AD subjects (MMSE > 20).  Biol. process categories assocd. with incipient AD-correlated genes were identified statistically (EASE program) and revealed up-regulation of many transcription factor/signaling genes regulating proliferation and differentiation, including tumor suppressors, oligodendrocyte growth factors, and protein kinase A modulators.  In addn., up-regulation of adhesion, apoptosis, lipid metab., and initial inflammation processes occurred, and down-regulation of protein folding/metab./transport and some energy metab. and signaling pathways took place.  These findings suggest a new model of AD pathogenesis in which a genomically orchestrated up-regulation of tumor suppressor-mediated differentiation and involution processes induces the spread of pathol. along myelinated axons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM0MvvQu295bVg90H21EOLACvtfcHk0lhLKhihAMSBGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1SrsL4%253D&md5=59ecee7b27eb4ceeb906a2e605fac284</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0308512100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0308512100%26sid%3Dliteratum%253Aachs%26aulast%3DBlalock%26aufirst%3DE.%2BM.%26aulast%3DGeddes%26aufirst%3DJ.%2BW.%26aulast%3DChen%26aufirst%3DK.%2BC.%26aulast%3DPorter%26aufirst%3DN.%2BM.%26aulast%3DMarkesbery%26aufirst%3DW.%2BR.%26aulast%3DLandfield%26aufirst%3DP.%2BW.%26atitle%3DIncipient%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520Microarray%2520correlation%2520analyses%2520reveal%2520major%2520transcriptional%2520and%2520tumor%2520suppressor%2520responses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D2173%26epage%3D2178%26doi%3D10.1073%2Fpnas.0308512100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brion, J. P.</span></span> <span> </span><span class="NLM_article-title">Increased level of active GSK-3beta in Alzheimer’s disease and accumulation in argyrophilic grains and in neurones at different stages of neurofibrillary degeneration</span>. <i>Neuropathol. Appl. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2990.2006.00795.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1111%2Fj.1365-2990.2006.00795.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=17239007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlemuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2007&pages=43-55&author=K.+Leroyauthor=Z.+Yilmazauthor=J.+P.+Brion&title=Increased+level+of+active+GSK-3beta+in+Alzheimer%E2%80%99s+disease+and+accumulation+in+argyrophilic+grains+and+in+neurones+at+different+stages+of+neurofibrillary+degeneration&doi=10.1111%2Fj.1365-2990.2006.00795.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Increased level of active GSK-3β in Alzheimer's disease and accumulation in argyrophilic grains and in neurones at different stages of neurofibrillary degeneration</span></div><div class="casAuthors">Leroy, K.; Yilmaz, Z.; Brion, J.-P.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropathology and Applied Neurobiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-55</span>CODEN:
                <span class="NLM_cas:coden">NANEDL</span>;
        ISSN:<span class="NLM_cas:issn">0305-1846</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The somatodendritic accumulation of hyperphosphorylated tau proteins is an early event preceding the appearance of neurofibrillary tangles (NFT) in Alzheimer's disease (AD) and might be necessary for their formation.  Glycogen synthase kinase-3β (GSK-3β) is a physiol. kinase for tau that generates many tau phosphorylation sites identified in NFT and in other tau-pos. inclusions.  We have studied the cellular distribution and the expression of the active form of GSK-3β (GSK-3 pTyr216) in AD patients, in argyrophilic grain disease and in diffuse Lewy body disease.  By Western blotting anal., a significant increase in the level of GSK-3 (pTyr216) was obsd. in the frontal cortex of AD patients.  A population of neurons showed a somatodendritic accumulation of GSK-3 (pTyr216) but not of the inactive form of GSK-3β (GSK-3 pSer9).  Most of these GSK-3 (pTyr216)-pos. cells were pos. for six different phosphotau epitopes known to be generated by GSK-3β.  By using a quadruple labeling method using GSK-3 (pTyr216) and phosphotau immunolabelling combined with Gallyas and DAPI staining, we examd. neurons contg. a somatodendritic GSK-3 (pTyr216) immunoreactivity at different stages of neurodegeneration.  A majority of neurons at the pretangle stage without Gallyas-pos. inclusions were GSK-3 (pTyr216) pos. and this GSK-3 (pTyr216) immunoreactivity remained in most cells contg. Gallyas and phosphotau-pos. inclusions excepted in extracellular NFT.  A GSK-3 (pTyr216) immunoreactivity was present in argyrophilic grains but not in cortical Lewy bodies.  These results directly suggest that the activity of GSK-3β is increased in AD and that somatodendritic accumulation and activation of GSK-3β is an early event preceding and accompanying the formation of NFT and of other tau-pos. inclusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-g_D-oz0erLVg90H21EOLACvtfcHk0lhLKhihAMSBGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlemuro%253D&md5=fe58bb24f90616c61baf51d0c2a05149</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2990.2006.00795.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2990.2006.00795.x%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DK.%26aulast%3DYilmaz%26aufirst%3DZ.%26aulast%3DBrion%26aufirst%3DJ.%2BP.%26atitle%3DIncreased%2520level%2520of%2520active%2520GSK-3beta%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%2520and%2520accumulation%2520in%2520argyrophilic%2520grains%2520and%2520in%2520neurones%2520at%2520different%2520stages%2520of%2520neurofibrillary%2520degeneration%26jtitle%3DNeuropathol.%2520Appl.%2520Neurobiol.%26date%3D2007%26volume%3D33%26spage%3D43%26epage%3D55%26doi%3D10.1111%2Fj.1365-2990.2006.00795.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kremer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaworski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Leuven, F.</span></span> <span> </span><span class="NLM_article-title">GSK3 and Alzheimer’s disease: Facts and fiction...</span>. <i>Front. Mol. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.3389/fnmol.2011.00017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.3389%2Ffnmol.2011.00017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=21904524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC38XovVKr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=17&author=A.+Kremerauthor=J.+V.+Louisauthor=T.+Jaworskiauthor=F.+Van+Leuven&title=GSK3+and+Alzheimer%E2%80%99s+disease%3A+Facts+and+fiction...&doi=10.3389%2Ffnmol.2011.00017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">GSK3 and Alzheimer's disease: facts and fiction</span></div><div class="casAuthors">Kremer, Anna; Louis, Justin V.; Jaworski, Tomasz; Van Leuven, Fred</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Molecular Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Aug.</span>),
    <span class="NLM_cas:pages">17</span>CODEN:
                <span class="NLM_cas:coden">FMNRAJ</span>;
        ISSN:<span class="NLM_cas:issn">1662-5099</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  The physiol. functions and pathol. roles of the Glycogen synthase kinase-type 3 (GSK3) kinases in peripheral and central systems are diverse and complex, and therefore hard to unravel in mol. detail in vivo.  Our assignment to review and discuss available data to clarify the actual position of these kinases in the pathol. of Alzheimer's dementia (AD) was both ambitious and easy.  On the one hand, numerous studies are available in isolated, recombinant, or cell-based systems, which have resulted in very diverse data-sets that are hardly informative for the brain in vivo.  At the other extreme, reliable, and relevant models for the role of GSK3 in CNS are rare, if not lacking.  Moreover, (too) many in vivo studies used Li+ as "specific" inhibitor of GSK3, which is factually not valid because lithium ions are neither specific nor potent inhibitors of GSK3 in vivo.  More specific pharmacol. inhibitors of GSK3 have met with considerable problems, and are reviewed by others in this issue or elsewhere.  We conc. here on AD-related aspects of GSK3 in brain in vivo, mainly studied in transgenic mice and highlight some of the more important issues, among many remaining: activation of GSK3 by amyloid, phosphorylation of protein tau, effects on or interference with synaptic activity, differentiation between both GSK3 isoforms.  These relate directly to brain function, and brain dysfunction in AD, and are to be resolved if we want to understand the mol. pathol. of this dreadful disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkyOq4ZInd37Vg90H21EOLACvtfcHk0lgTS-ynqYDejQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovVKr&md5=ca9888444963da083e23df3c17a51521</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3389%2Ffnmol.2011.00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnmol.2011.00017%26sid%3Dliteratum%253Aachs%26aulast%3DKremer%26aufirst%3DA.%26aulast%3DLouis%26aufirst%3DJ.%2BV.%26aulast%3DJaworski%26aufirst%3DT.%26aulast%3DVan%2BLeuven%26aufirst%3DF.%26atitle%3DGSK3%2520and%2520Alzheimer%25E2%2580%2599s%2520disease%253A%2520Facts%2520and%2520fiction...%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2011%26volume%3D4%26spage%3D17%26doi%3D10.3389%2Ffnmol.2011.00017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, T. R.</span></span> <span> </span><span class="NLM_article-title">Glycogen synthase kinase-3 (GSK-3)-targeted therapy and imaging</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.7150/thno.14334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.7150%2Fthno.14334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26941849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC28XpvVertbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=571-593&author=M.+K.+Pandeyauthor=T.+R.+DeGrado&title=Glycogen+synthase+kinase-3+%28GSK-3%29-targeted+therapy+and+imaging&doi=10.7150%2Fthno.14334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Glycogen synthase kinase-3 (GSK-3)-targeted therapy and imaging</span></div><div class="casAuthors">Pandey, Mukesh K.; DeGrado, Timothy R.</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">571-593</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) is assocd. with various key biol. processes, including glucose regulation, apoptosis, protein synthesis, cell signaling, cellular transport, gene transcription, proliferation, and intracellular communication.  Accordingly, GSK-3 has been implicated in a wide variety of diseases and specifically targeted for both therapeutic and imaging applications by a large no. of academic labs. and pharmaceutical companies.  Here, we review the structure, function, expression levels, and ligand-binding properties of GSK-3 and its connection to various diseases.  A selected list of highly potent GSK-3 inhibitors, with IC50 <20 nM for ATP (ATP)-competitive inhibitors and IC50 <5 μM for non-ATP-competitive inhibitors, were analyzed for structure activity relationships.  Furthermore, ubiquitous expression of GSK-3 and its possible impact on therapy and imaging are also highlighted.  Finally, a rational perspective and possible route to selective and effective GSK-3 inhibitors is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg397DfBVXrrVg90H21EOLACvtfcHk0lgTS-ynqYDejQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpvVertbs%253D&md5=e66feec1b1b2f460000116b722678478</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.7150%2Fthno.14334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.14334%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DM.%2BK.%26aulast%3DDeGrado%26aufirst%3DT.%2BR.%26atitle%3DGlycogen%2520synthase%2520kinase-3%2520%2528GSK-3%2529-targeted%2520therapy%2520and%2520imaging%26jtitle%3DTheranostics%26date%3D2016%26volume%3D6%26spage%3D571%26epage%3D593%26doi%3D10.7150%2Fthno.14334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goedert, M.</span></span> <span> </span><span class="NLM_article-title">GSK3 inhibitors: development and therapeutic potential</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1038/nrd1415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrd1415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=15173837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVaqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=479-487&author=P.+Cohenauthor=M.+Goedert&title=GSK3+inhibitors%3A+development+and+therapeutic+potential&doi=10.1038%2Fnrd1415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">GSK3 inhibitors: development and therapeutic potential</span></div><div class="casAuthors">Cohen, Philip; Goedert, Michel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">479-487</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Glycogen synthase kinase-3 (GSK3) was initially identified more than two decades ago as an enzyme involved in the control of glycogen metab.  In recent years it has been shown to have key roles in regulating a diverse range of cellular functions, which have prompted efforts to develop GSK3 inhibitors as therapeutics.  Here, we describe the biol. of GSK3 relevant to its potential as a target for diabetes and neurodegenerative diseases, and discuss progress in the development of GSK3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL3ua5xi0_qrVg90H21EOLACvtfcHk0lgTS-ynqYDejQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVaqsrs%253D&md5=165d1354bf11faa6021ac4dba797fda2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd1415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1415%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DGoedert%26aufirst%3DM.%26atitle%3DGSK3%2520inhibitors%253A%2520development%2520and%2520therapeutic%2520potential%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D479%26epage%3D487%26doi%3D10.1038%2Fnrd1415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketterman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weïwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germain, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nag, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbe, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzaro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacher, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheah, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holson, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J. Q.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Glycogen Synthase Kinase 3 with Exquisite Kinome-Wide Selectivity and Their Functional Effects</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1952</span>– <span class="NLM_lpage">1963</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00306</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00306" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFemtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1952-1963&author=F.+F.+Wagnerauthor=J.+A.+Bishopauthor=J.+P.+Galeauthor=X.+Shiauthor=M.+Walkauthor=J.+Kettermanauthor=D.+Patnaikauthor=D.+Barkerauthor=D.+Walpitaauthor=A.+J.+Campbellauthor=S.+Nguyenauthor=M.+Lewisauthor=L.+Rossauthor=M.+We%C3%AFwerauthor=W.+F.+Anauthor=A.+R.+Germainauthor=P.+P.+Nagauthor=S.+Metkarauthor=T.+Kayaauthor=S.+Dandapaniauthor=D.+E.+Olsonauthor=A.-L.+Barbeauthor=F.+Lazzaroauthor=J.+R.+Sacherauthor=J.+H.+Cheahauthor=D.+Feiauthor=J.+Perezauthor=B.+Munozauthor=M.+Palmerauthor=K.+Stegmaierauthor=S.+L.+Schreiberauthor=E.+Scolnickauthor=Y.-L.+Zhangauthor=S.+J.+Haggartyauthor=E.+B.+Holsonauthor=J.+Q.+Pan&title=Inhibitors+of+Glycogen+Synthase+Kinase+3+with+Exquisite+Kinome-Wide+Selectivity+and+Their+Functional+Effects&doi=10.1021%2Facschembio.6b00306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Glycogen Synthase Kinase 3 with Exquisite Kinome-Wide Selectivity and Their Functional Effects</span></div><div class="casAuthors">Wagner, Florence F.; Bishop, Joshua A.; Gale, Jennifer P.; Shi, Xi; Walk, Michelle; Ketterman, Joshua; Patnaik, Debasis; Barker, Doug; Walpita, Deepika; Campbell, Arthur J.; Nguyen, Shannon; Lewis, Michael; Ross, Linda; Weiwer, Michel; An, W. Frank; Germain, Andrew R.; Nag, Partha P.; Metkar, Shailesh; Kaya, Taner; Dandapani, Sivaraman; Olson, David E.; Barbe, Anne-Laure; Lazzaro, Fanny; Sacher, Joshua R.; Cheah, Jaime H.; Fei, David; Perez, Jose; Munoz, Benito; Palmer, Michelle; Stegmaier, Kimberly; Schreiber, Stuart L.; Scolnick, Edward; Zhang, Yan-Ling; Haggarty, Stephen J.; Holson, Edward B.; Pan, Jen Q.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1952-1963</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mood stabilizer lithium, the first-line treatment for bipolar disorder, is hypothesized to exert its effects through direct inhibition of glycogen synthase kinase 3 (GSK3) and indirectly by increasing GSK3's inhibitory serine phosphorylation.  GSK3 comprises two highly similar paralogs, GSK3α and GSK3β, which are key regulatory kinases in the canonical Wnt pathway.  GSK3 stands as a nodal target within this pathway and is an attractive therapeutic target for multiple indications.  Despite being an active field of research for the past 20 years, many GSK3 inhibitors demonstrate either poor to moderate selectivity vs. the broader human kinome or physicochem. properties unsuitable for use in in vitro systems or in vivo models.  A nonconventional anal. of data from a GSK3β inhibitor high-throughput screening campaign, which excluded known GSK3 inhibitor chemotypes, led to the discovery of a novel pyrazolo-tetrahydroquinolinone scaffold with unparalleled kinome-wide selectivity for the GSK3 kinases.  Taking advantage of an uncommon tridentate interaction with the hinge region of GSK3, we developed highly selective and potent GSK3 inhibitors, BRD1652 and BRD0209, which demonstrated in vivo efficacy in a dopaminergic signaling paradigm modeling mood-related disorders.  These new chem. probes open the way for exclusive analyses of the function of GSK3 kinases in multiple signaling pathways involved in many prevalent disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt6ZO9St9MGLVg90H21EOLACvtfcHk0ljy2Wyf1ltzXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFemtr7I&md5=2cfb60c9c1936e5ef9986842aedf473f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00306%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DBishop%26aufirst%3DJ.%2BA.%26aulast%3DGale%26aufirst%3DJ.%2BP.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWalk%26aufirst%3DM.%26aulast%3DKetterman%26aufirst%3DJ.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DBarker%26aufirst%3DD.%26aulast%3DWalpita%26aufirst%3DD.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DNguyen%26aufirst%3DS.%26aulast%3DLewis%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DL.%26aulast%3DWe%25C3%25AFwer%26aufirst%3DM.%26aulast%3DAn%26aufirst%3DW.%2BF.%26aulast%3DGermain%26aufirst%3DA.%2BR.%26aulast%3DNag%26aufirst%3DP.%2BP.%26aulast%3DMetkar%26aufirst%3DS.%26aulast%3DKaya%26aufirst%3DT.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DOlson%26aufirst%3DD.%2BE.%26aulast%3DBarbe%26aufirst%3DA.-L.%26aulast%3DLazzaro%26aufirst%3DF.%26aulast%3DSacher%26aufirst%3DJ.%2BR.%26aulast%3DCheah%26aufirst%3DJ.%2BH.%26aulast%3DFei%26aufirst%3DD.%26aulast%3DPerez%26aufirst%3DJ.%26aulast%3DMunoz%26aufirst%3DB.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DScolnick%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DHolson%26aufirst%3DE.%2BB.%26aulast%3DPan%26aufirst%3DJ.%2BQ.%26atitle%3DInhibitors%2520of%2520Glycogen%2520Synthase%2520Kinase%25203%2520with%2520Exquisite%2520Kinome-Wide%2520Selectivity%2520and%2520Their%2520Functional%2520Effects%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1952%26epage%3D1963%26doi%3D10.1021%2Facschembio.6b00306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normandin, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Para, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopresti-Morrow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanyon, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, K. E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schachter, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enerson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guehl, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holson, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Fakhri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumbail, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasdev, N.</span></span> <span> </span><span class="NLM_article-title">Discovery of a highly selective glycogen synthase kinase-3 inhibitor (PF-04802367) that modulates tau phosphorylation in the brain: translation for PET neuroimaging</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9601</span>– <span class="NLM_lpage">9605</span>, <span class="refDoi"> DOI: 10.1002/anie.201603797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2Fanie.201603797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=27355874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A280%3ADC%252BC2s7kslKhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=9601-9605&author=S.+H.+Liangauthor=J.+M.+Chenauthor=M.+D.+Normandinauthor=J.+S.+Changauthor=G.+C.+Changauthor=C.+K.+Taylorauthor=P.+Trapaauthor=M.+S.+Plummerauthor=K.+S.+Paraauthor=E.+L.+Connauthor=L.+Lopresti-Morrowauthor=L.+F.+Lanyonauthor=J.+M.+Cookauthor=K.+E.+G.+Richterauthor=C.+E.+Nolanauthor=J.+B.+Schachterauthor=F.+Janatauthor=Y.+Cheauthor=V.+Shanmugasundaramauthor=B.+A.+Lefkerauthor=B.+E.+Enersonauthor=E.+Livniauthor=L.+Wangauthor=N.+J.+Guehlauthor=D.+Patnaikauthor=F.+F.+Wagnerauthor=R.+Perlisauthor=E.+B.+Holsonauthor=S.+J.+Haggartyauthor=G.+El+Fakhriauthor=R.+G.+Kurumbailauthor=N.+Vasdev&title=Discovery+of+a+highly+selective+glycogen+synthase+kinase-3+inhibitor+%28PF-04802367%29+that+modulates+tau+phosphorylation+in+the+brain%3A+translation+for+PET+neuroimaging&doi=10.1002%2Fanie.201603797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Highly Selective Glycogen Synthase Kinase-3 Inhibitor (PF-04802367) That Modulates Tau Phosphorylation in the Brain: Translation for PET Neuroimaging</span></div><div class="casAuthors">Liang Steven H; Normandin Marc D; Livni Elijahu; Wang Lu; Guehl Nicolas J; El Fakhri Georges; Vasdev Neil; Chen Jinshan Michael; Chang Jeanne S; Chang George C; Taylor Christine K; Plummer Mark S; Para Kimberly S; Conn Edward L; Lopresti-Morrow Lori; Lanyon Lorraine F; Cook James M; Richter Karl E G; Nolan Charlie E; Schachter Joel B; Janat Fouad; Che Ye; Shanmugasundaram Veerabahu; Enerson Bradley E; Kurumbail Ravi G; Trapa Patrick; Lefker Bruce A; Patnaik Debasis; Perlis Roy; Haggarty Stephen J; Wagner Florence F; Perlis Roy; Holson Edward B</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">9601-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Glycogen synthase kinase-3 (GSK-3) regulates multiple cellular processes in diabetes, oncology, and neurology.  N-(3-(1H-1,2,4-triazol-1-yl)propyl)-5-(3-chloro-4-methoxyphenyl)oxazole-4-carboxamide (PF-04802367 or PF-367) has been identified as a highly potent inhibitor, which is among the most selective antagonists of GSK-3 to date.  Its efficacy was demonstrated in modulation of tau phosphorylation in vitro and in vivo.  Whereas the kinetics of PF-367 binding in brain tissues are too fast for an effective therapeutic agent, the pharmacokinetic profile of PF-367 is ideal for discovery of radiopharmaceuticals for GSK-3 in the central nervous system.  A (11) C-isotopologue of PF-367 was synthesized and preliminary PET imaging studies in non-human primates confirmed that we have overcome the two major obstacles for imaging GSK-3, namely, reasonable brain permeability and displaceable binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBs_RxQsDg6XELvajU9pjPfW6udTcc2eal1ExLP8KnPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7kslKhtA%253D%253D&md5=ef5bafe40489bc7ab34c29988b38c9ec</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603797%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DS.%2BH.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DNormandin%26aufirst%3DM.%2BD.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DChang%26aufirst%3DG.%2BC.%26aulast%3DTaylor%26aufirst%3DC.%2BK.%26aulast%3DTrapa%26aufirst%3DP.%26aulast%3DPlummer%26aufirst%3DM.%2BS.%26aulast%3DPara%26aufirst%3DK.%2BS.%26aulast%3DConn%26aufirst%3DE.%2BL.%26aulast%3DLopresti-Morrow%26aufirst%3DL.%26aulast%3DLanyon%26aufirst%3DL.%2BF.%26aulast%3DCook%26aufirst%3DJ.%2BM.%26aulast%3DRichter%26aufirst%3DK.%2BE.%2BG.%26aulast%3DNolan%26aufirst%3DC.%2BE.%26aulast%3DSchachter%26aufirst%3DJ.%2BB.%26aulast%3DJanat%26aufirst%3DF.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DLefker%26aufirst%3DB.%2BA.%26aulast%3DEnerson%26aufirst%3DB.%2BE.%26aulast%3DLivni%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DGuehl%26aufirst%3DN.%2BJ.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DPerlis%26aufirst%3DR.%26aulast%3DHolson%26aufirst%3DE.%2BB.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DEl%2BFakhri%26aufirst%3DG.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DVasdev%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520a%2520highly%2520selective%2520glycogen%2520synthase%2520kinase-3%2520inhibitor%2520%2528PF-04802367%2529%2520that%2520modulates%2520tau%2520phosphorylation%2520in%2520the%2520brain%253A%2520translation%2520for%2520PET%2520neuroimaging%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D9601%26epage%3D9605%26doi%3D10.1002%2Fanie.201603797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snow, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raybon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pokross, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kish, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubowchik, G. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of isonicotinamides as highly selective, brain penetrable, and orally active glycogen synthase kinase-3 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1041</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01550</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01550" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmslertg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1041-1051&author=G.+Luoauthor=L.+Chenauthor=C.+R.+Burtonauthor=H.+Xiaoauthor=P.+Sivaprakasamauthor=C.+M.+Krauseauthor=Y.+Caoauthor=N.+Liuauthor=J.+Lippyauthor=W.+J.+Clarkeauthor=K.+Snowauthor=J.+Raybonauthor=V.+Aroraauthor=M.+Pokrossauthor=K.+Kishauthor=H.+A.+Lewisauthor=D.+R.+Langleyauthor=J.+E.+Macorauthor=G.+M.+Dubowchik&title=Discovery+of+isonicotinamides+as+highly+selective%2C+brain+penetrable%2C+and+orally+active+glycogen+synthase+kinase-3+inhibitors&doi=10.1021%2Facs.jmedchem.5b01550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Isonicotinamides as Highly Selective, Brain Penetrable, and Orally Active Glycogen Synthase Kinase-3 Inhibitors</span></div><div class="casAuthors">Luo, Guanglin; Chen, Ling; Burton, Catherine R.; Xiao, Hong; Sivaprakasam, Prasanna; Krause, Carol M.; Cao, Yang; Liu, Nengyin; Lippy, Jonathan; Clarke, Wendy J.; Snow, Kimberly; Raybon, Joseph; Arora, Vinod; Pokross, Matt; Kish, Kevin; Lewis, Hal A.; Langley, David R.; Macor, John E.; Dubowchik, Gene M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1041-1051</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GSK-3 is a serine/threonine kinase that has numerous substrates.  Many of these proteins are involved in the regulation of diverse cellular functions, including metab., differentiation, proliferation, and apoptosis.  Inhibition of GSK-3 may be useful in treating a no. of diseases including Alzheimer's disease (AD), type II diabetes, mood disorders, and some cancers, but the approach poses significant challenges.  Here, the authors present a class of isonicotinamides, e.g. I [R = F, Cl] that are potent, highly kinase-selective GSK-3 inhibitors, the members of which demonstrated oral activity in a triple-transgenic mouse model of AD.  The remarkably high kinase selectivity and straightforward synthesis of these compds. bode well for their further exploration as tool compds. and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaIEil5d6i8rVg90H21EOLACvtfcHk0lgrG4xQJ-jJqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmslertg%253D%253D&md5=d0412c3fe9b430c59f861fa10d3eff79</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01550%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DBurton%26aufirst%3DC.%2BR.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DSivaprakasam%26aufirst%3DP.%26aulast%3DKrause%26aufirst%3DC.%2BM.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DLippy%26aufirst%3DJ.%26aulast%3DClarke%26aufirst%3DW.%2BJ.%26aulast%3DSnow%26aufirst%3DK.%26aulast%3DRaybon%26aufirst%3DJ.%26aulast%3DArora%26aufirst%3DV.%26aulast%3DPokross%26aufirst%3DM.%26aulast%3DKish%26aufirst%3DK.%26aulast%3DLewis%26aufirst%3DH.%2BA.%26aulast%3DLangley%26aufirst%3DD.%2BR.%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26aulast%3DDubowchik%26aufirst%3DG.%2BM.%26atitle%3DDiscovery%2520of%2520isonicotinamides%2520as%2520highly%2520selective%252C%2520brain%2520penetrable%252C%2520and%2520orally%2520active%2520glycogen%2520synthase%2520kinase-3%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1041%26epage%3D1051%26doi%3D10.1021%2Facs.jmedchem.5b01550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastassiadis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span> <span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0lgNXnG9DPK84Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lgNXnG9DPK84Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benajiba, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weïwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacher, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puissant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Back, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galinsky, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermine, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemann, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holson, E. B.</span></span> <span> </span><span class="NLM_article-title">Exploiting an Asp-Glu “switch” in glycogen synthase kinase 3 to design paralog-selective inhibitors for use in acute myeloid leukemia</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">eaam8460</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aam8460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1126%2Fscitranslmed.aam8460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=29515000" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&author=F.+F.+Wagnerauthor=L.+Benajibaauthor=A.+J.+Campbellauthor=M.+We%C3%AFwerauthor=J.+R.+Sacherauthor=J.+P.+Galeauthor=L.+Rossauthor=A.+Puissantauthor=G.+Alexeauthor=A.+Conwayauthor=M.+Backauthor=Y.+Pikmanauthor=I.+Galinskyauthor=D.+J.+DeAngeloauthor=R.+M.+Stoneauthor=T.+Kayaauthor=X.+Shiauthor=M.+B.+Robersauthor=T.+Machleidtauthor=J.+Wilkinsonauthor=O.+Hermineauthor=A.+Kungauthor=A.+J.+Steinauthor=D.+Lakshminarasimhanauthor=M.+T.+Hemannauthor=E.+Scolnickauthor=Y.-L.+Zhangauthor=J.+Q.+Panauthor=K.+Stegmaierauthor=E.+B.+Holson&title=Exploiting+an+Asp-Glu+%E2%80%9Cswitch%E2%80%9D+in+glycogen+synthase+kinase+3+to+design+paralog-selective+inhibitors+for+use+in+acute+myeloid+leukemia&doi=10.1126%2Fscitranslmed.aam8460"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aam8460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aam8460%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DBenajiba%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DWe%25C3%25AFwer%26aufirst%3DM.%26aulast%3DSacher%26aufirst%3DJ.%2BR.%26aulast%3DGale%26aufirst%3DJ.%2BP.%26aulast%3DRoss%26aufirst%3DL.%26aulast%3DPuissant%26aufirst%3DA.%26aulast%3DAlexe%26aufirst%3DG.%26aulast%3DConway%26aufirst%3DA.%26aulast%3DBack%26aufirst%3DM.%26aulast%3DPikman%26aufirst%3DY.%26aulast%3DGalinsky%26aufirst%3DI.%26aulast%3DDeAngelo%26aufirst%3DD.%2BJ.%26aulast%3DStone%26aufirst%3DR.%2BM.%26aulast%3DKaya%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DHermine%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DA.%2BJ.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DHemann%26aufirst%3DM.%2BT.%26aulast%3DScolnick%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DPan%26aufirst%3DJ.%2BQ.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DHolson%26aufirst%3DE.%2BB.%26atitle%3DExploiting%2520an%2520Asp-Glu%2520%25E2%2580%259Cswitch%25E2%2580%259D%2520in%2520glycogen%2520synthase%2520kinase%25203%2520to%2520design%2520paralog-selective%2520inhibitors%2520for%2520use%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2018%26volume%3D10%26doi%3D10.1126%2Fscitranslmed.aam8460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergner, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyanova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannaiyan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manrique-Hoyos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kongi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantuti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanisch, U.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span> <span> </span><span class="NLM_article-title">Cell type- and brain region-resolved mouse brain proteome</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1819</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1038/nn.4160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnn.4160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26523646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1819-1831&author=K.+Sharmaauthor=S.+Schmittauthor=C.+G.+Bergnerauthor=S.+Tyanovaauthor=N.+Kannaiyanauthor=N.+Manrique-Hoyosauthor=K.+Kongiauthor=L.+Cantutiauthor=U.-K.+Hanischauthor=M.-A.+Philipsauthor=M.+J.+Rossnerauthor=M.+Mannauthor=M.+Simons&title=Cell+type-+and+brain+region-resolved+mouse+brain+proteome&doi=10.1038%2Fnn.4160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cell type- and brain region-resolved mouse brain proteome</span></div><div class="casAuthors">Sharma, Kirti; Schmitt, Sebastian; Bergner, Caroline G.; Tyanova, Stefka; Kannaiyan, Nirmal; Manrique-Hoyos, Natalia; Kongi, Karina; Cantuti, Ludovico; Hanisch, Uwe-Karsten; Philips, Mari-Anne; Rossner, Moritz J.; Mann, Matthias; Simons, Mikael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1819-1831</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Brain transcriptome and connectome maps are being generated, but an equiv. effort on the proteome is currently lacking.  We performed high-resoln. mass spectrometry-based proteomics for in-depth anal. of the mouse brain and its major brain regions and cell types.  Comparisons of the 12,934 identified proteins in oligodendrocytes, astrocytes, microglia and cortical neurons with deep sequencing data of the transcriptome indicated deep coverage of the proteome.  Cell type-specific proteins defined as tenfold more abundant than av. expression represented about a tenth of the proteome, with an overrepresentation of cell surface proteins.  To demonstrate the utility of our resource, we focused on this class of proteins and identified Lsamp, an adhesion mol. of the IgLON family, as a neg. regulator of myelination.  Our findings provide a framework for a system-level understanding of cell-type diversity in the CNS and serves as a rich resource for analyses of brain development and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHhWGHaIaSYrVg90H21EOLACvtfcHk0lgRB8rZkE3_BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7bI&md5=30caea646f82e42968f84275ac17c106</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnn.4160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4160%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DK.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DBergner%26aufirst%3DC.%2BG.%26aulast%3DTyanova%26aufirst%3DS.%26aulast%3DKannaiyan%26aufirst%3DN.%26aulast%3DManrique-Hoyos%26aufirst%3DN.%26aulast%3DKongi%26aufirst%3DK.%26aulast%3DCantuti%26aufirst%3DL.%26aulast%3DHanisch%26aufirst%3DU.-K.%26aulast%3DPhilips%26aufirst%3DM.-A.%26aulast%3DRossner%26aufirst%3DM.%2BJ.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DSimons%26aufirst%3DM.%26atitle%3DCell%2520type-%2520and%2520brain%2520region-resolved%2520mouse%2520brain%2520proteome%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D1819%26epage%3D1831%26doi%3D10.1038%2Fnn.4160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorokina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S. G. N.</span></span> <span> </span><span class="NLM_article-title">Proteomic analysis of postsynaptic proteins in regions of the human neocortex</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1038/s41593-017-0025-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fs41593-017-0025-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=29203896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVyitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2018&pages=130-138&author=M.+Royauthor=O.+Sorokinaauthor=N.+Skeneauthor=C.+Simonnetauthor=F.+Mazzoauthor=R.+Zwartauthor=E.+Sherauthor=C.+Smithauthor=J.+D.+Armstrongauthor=S.+G.+N.+Grant&title=Proteomic+analysis+of+postsynaptic+proteins+in+regions+of+the+human+neocortex&doi=10.1038%2Fs41593-017-0025-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analysis of postsynaptic proteins in regions of the human neocortex</span></div><div class="casAuthors">Roy, Marcia; Sorokina, Oksana; Skene, Nathan; Simonnet, Clemence; Mazzo, Francesca; Zwart, Ruud; Sher, Emanuele; Smith, Colin; Armstrong, J. Douglas; Grant, Seth G. N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">130-138</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The postsynaptic proteome of excitatory synapses comprises ∼1,000 highly conserved proteins that control the behavioral repertoire, and mutations disrupting their function cause >130 brain diseases.  In this document the compn. of postsynaptic proteomes in human neocortical regions and integrate it with genetic, functional and structural magnetic resonance imaging, positron emission tomog. imaging, and behavioral data.  Neocortical regions show signatures of expression of individual proteins, protein complexes, biochem. and metabolic pathways.  This characterized the compositional signatures in brain regions involved with language, emotion and memory functions.  Integrating large-scale GWAS with regional proteome data identifies the same cortical region for smoking behavior as found with fMRI data.  The neocortical postsynaptic proteome data resource can be used to link genetics to brain imaging and behavior, and to study the role of postsynaptic proteins in localization of brain functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE7Af1gt2EVLVg90H21EOLACvtfcHk0lgRB8rZkE3_BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVyitb4%253D&md5=065cb3a291a830866065e63addceddc5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41593-017-0025-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-017-0025-9%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%26aulast%3DSorokina%26aufirst%3DO.%26aulast%3DSkene%26aufirst%3DN.%26aulast%3DSimonnet%26aufirst%3DC.%26aulast%3DMazzo%26aufirst%3DF.%26aulast%3DZwart%26aufirst%3DR.%26aulast%3DSher%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DC.%26aulast%3DArmstrong%26aufirst%3DJ.%2BD.%26aulast%3DGrant%26aufirst%3DS.%2BG.%2BN.%26atitle%3DProteomic%2520analysis%2520of%2520postsynaptic%2520proteins%2520in%2520regions%2520of%2520the%2520human%2520neocortex%26jtitle%3DNat.%2520Neurosci.%26date%3D2018%26volume%3D21%26spage%3D130%26epage%3D138%26doi%3D10.1038%2Fs41593-017-0025-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeusen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swafford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursalaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deaton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Pinzon, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hering, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin Seidel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glynne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of DYRK1A and GSK3B induces human β-cell proliferation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8372</span>, <span class="refDoi"> DOI: 10.1038/ncomms9372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fncomms9372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26496802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8372&author=W.+Shenauthor=B.+Taylorauthor=Q.+Jinauthor=V.+Nguyen-Tranauthor=S.+Meeusenauthor=Y.-Q.+Zhangauthor=A.+Kamireddyauthor=A.+Swaffordauthor=A.+F.+Powersauthor=J.+Walkerauthor=J.+Lambauthor=B.+Bursalayaauthor=M.+DiDonatoauthor=G.+Harbauthor=M.+Qiuauthor=C.+M.+Filippiauthor=L.+Deatonauthor=C.+N.+Turkauthor=W.+L.+Suarez-Pinzonauthor=Y.+Liuauthor=X.+Haoauthor=T.+Moauthor=S.+Yanauthor=J.+Liauthor=A.+E.+Hermanauthor=B.+J.+Heringauthor=T.+Wuauthor=H.+Martin+Seidelauthor=P.+McNamaraauthor=R.+Glynneauthor=B.+Laffitte&title=Inhibition+of+DYRK1A+and+GSK3B+induces+human+%CE%B2-cell+proliferation&doi=10.1038%2Fncomms9372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A and GSK3B induces human β-cell proliferation</span></div><div class="casAuthors">Shen, Weijun; Taylor, Brandon; Jin, Qihui; Nguyen-Tran, Van; Meeusen, Shelly; Zhang, You-Qing; Kamireddy, Anwesh; Swafford, Austin; Powers, Andrew F.; Walker, John; Lamb, John; Bursalaya, Badry; DiDonato, Michael; Harb, George; Qiu, Minhua; Filippi, Christophe M.; Deaton, Lisa; Turk, Carolina N.; Suarez-Pinzon, Wilma L.; Liu, Yahu; Hao, Xueshi; Mo, Tingting; Yan, Shanshan; Li, Jing; Herman, Ann E.; Hering, Bernhard J.; Wu, Tom; Seidel, H. Martin; McNamara, Peter; Glynne, Richard; Laffitte, Bryan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8372</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insufficient pancreatic β-cell mass or function results in diabetes mellitus.  While significant progress has been made in regulating insulin secretion from β-cells in diabetic patients, no pharmacol. agents have been described that increase β-cell replication in humans.  Here we report aminopyrazine compds. that stimulate robust β-cell proliferation in adult primary islets, most likely as a result of combined inhibition of DYRK1A and GSK3B.  Aminopyrazine-treated human islets retain functionality in vitro and after transplantation into diabetic mice.  Oral dosing of these compds. in diabetic mice induces β-cell proliferation, increases β-cell mass and insulin content, and improves glycemic control.  Biochem., genetic and cell biol. data point to Dyrk1a as the key mol. target.  This study supports the feasibility of treating diabetes with an oral therapy to restore β-cell mass, and highlights a tractable pathway for future drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVHzP-VkH5nrVg90H21EOLACvtfcHk0ljo7IIEvBQllw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK&md5=5ef960af3bfd2795b9fdce52876e5d59</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fncomms9372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9372%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTaylor%26aufirst%3DB.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DNguyen-Tran%26aufirst%3DV.%26aulast%3DMeeusen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.-Q.%26aulast%3DKamireddy%26aufirst%3DA.%26aulast%3DSwafford%26aufirst%3DA.%26aulast%3DPowers%26aufirst%3DA.%2BF.%26aulast%3DWalker%26aufirst%3DJ.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DBursalaya%26aufirst%3DB.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DHarb%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DM.%26aulast%3DFilippi%26aufirst%3DC.%2BM.%26aulast%3DDeaton%26aufirst%3DL.%26aulast%3DTurk%26aufirst%3DC.%2BN.%26aulast%3DSuarez-Pinzon%26aufirst%3DW.%2BL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHerman%26aufirst%3DA.%2BE.%26aulast%3DHering%26aufirst%3DB.%2BJ.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DMartin%2BSeidel%26aufirst%3DH.%26aulast%3DMcNamara%26aufirst%3DP.%26aulast%3DGlynne%26aufirst%3DR.%26aulast%3DLaffitte%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520DYRK1A%2520and%2520GSK3B%2520induces%2520human%2520%25CE%25B2-cell%2520proliferation%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8372%26doi%3D10.1038%2Fncomms9372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartoschik, T.</span>; <span class="NLM_string-name">Maschberger, M.</span>; <span class="NLM_string-name">Feoli, A.</span>; <span class="NLM_string-name">André, T.</span>; <span class="NLM_string-name">Baaske, P.</span>; <span class="NLM_string-name">Duhr, S.</span>; <span class="NLM_string-name">Breitsprecher, D.</span></span> <span> </span><span class="NLM_article-title">Microscale Thermophoresis in Drug Discovery</span>.  <i>Applied Biophysics for Drug Discovery</i>; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">92</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2F9781119099512.ch5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=73-92&author=T.+Bartoschik&author=M.+Maschberger&author=A.+Feoli&author=T.+Andr%C3%A9&author=P.+Baaske&author=S.+Duhr&author=D.+Breitsprecher&title=Applied+Biophysics+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2F9781119099512.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781119099512.ch5%26sid%3Dliteratum%253Aachs%26aulast%3DBartoschik%26aufirst%3DT.%26atitle%3DMicroscale%2520Thermophoresis%2520in%2520Drug%2520Discovery%26btitle%3DApplied%2520Biophysics%2520for%2520Drug%2520Discovery%26pub%3DWiley%26date%3D2017%26spage%3D73%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stamos, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enos, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span> <span> </span><span class="NLM_article-title">Structural basis of GSK-3 inhibition by N-terminal phosphorylation and by the Wnt receptor LRP6</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>, <span class="NLM_elocation-id">e01998</span> <span class="refDoi"> DOI: 10.7554/eLife.01998</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.7554%2FeLife.01998" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&author=J.+L.+Stamosauthor=M.+L.+Chuauthor=M.+D.+Enosauthor=N.+Shahauthor=W.+I.+Weis&title=Structural+basis+of+GSK-3+inhibition+by+N-terminal+phosphorylation+and+by+the+Wnt+receptor+LRP6&doi=10.7554%2FeLife.01998"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7554%2FeLife.01998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01998%26sid%3Dliteratum%253Aachs%26aulast%3DStamos%26aufirst%3DJ.%2BL.%26aulast%3DChu%26aufirst%3DM.%2BL.%26aulast%3DEnos%26aufirst%3DM.%2BD.%26aulast%3DShah%26aufirst%3DN.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26atitle%3DStructural%2520basis%2520of%2520GSK-3%2520inhibition%2520by%2520N-terminal%2520phosphorylation%2520and%2520by%2520the%2520Wnt%2520receptor%2520LRP6%26jtitle%3DeLife%26date%3D2014%26volume%3D18%26doi%3D10.7554%2FeLife.01998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C.</span>; <span class="NLM_string-name">Fass, D.
M.</span>; <span class="NLM_string-name">Folz-Donahue, K.</span>; <span class="NLM_string-name">MacDonald, M. E.</span>; <span class="NLM_string-name">Haggarty, S. J.</span></span> <span> </span><span class="NLM_article-title">Highly Expandable Human iPS Cell-Derived Neural Progenitor Cells (NPC) and Neurons for Central Nervous System Disease Modeling and High-Throughput Screening</span>.  <i>Curr. Protoc. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume">92</span>,  <span class="NLM_lpage">21.8.21</span>.<span class="refDoi"> DOI: 10.1002/cphg.33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1002%2Fcphg.33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=28075486" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=-21.8.21&author=C.+Cheng&author=D.%0AM.+Fass&author=K.+Folz-Donahue&author=M.+E.+MacDonald&author=S.+J.+Haggarty&title=Highly+Expandable+Human+iPS+Cell-Derived+Neural+Progenitor+Cells+%28NPC%29+and+Neurons+for+Central+Nervous+System+Disease+Modeling+and+High-Throughput+Screening&doi=10.1002%2Fcphg.33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fcphg.33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcphg.33%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%26atitle%3DHighly%2520Expandable%2520Human%2520iPS%2520Cell-Derived%2520Neural%2520Progenitor%2520Cells%2520%2528NPC%2529%2520and%2520Neurons%2520for%2520Central%2520Nervous%2520System%2520Disease%2520Modeling%2520and%2520High-Throughput%2520Screening%26jtitle%3DCurr.%2520Protoc.%2520Hum.%2520Genet.%26date%3D2017%26volume%3D92%26epage%3D21.8.21%26doi%3D10.1002%2Fcphg.33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span>; <span class="NLM_string-name">Haggerty, S. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Functional Lithium Mimetics Affecting CRMP2 Phosphorylation in Human iPSC-Derived Neurons</span>. Unpublished results, <span class="NLM_year" style="font-weight: bold;">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=W.+N.+Zhao&author=S.+J.+Haggerty&title=Discovery+of+Functional+Lithium+Mimetics+Affecting+CRMP2+Phosphorylation+in+Human+iPSC-Derived+Neurons"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520of%2520Functional%2520Lithium%2520Mimetics%2520Affecting%2520CRMP2%2520Phosphorylation%2520in%2520Human%2520iPSC-Derived%2520Neurons%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikuchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaibuchi, K.</span></span> <span> </span><span class="NLM_article-title">GSK-3β Regulates Phosphorylation of CRMP-2 and Neuronal Polarity</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1016%2Fj.cell.2004.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=15652488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD2MXot1CgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2005&pages=137-149&author=T.+Yoshimuraauthor=Y.+Kawanoauthor=N.+Arimuraauthor=S.+Kawabataauthor=A.+Kikuchiauthor=K.+Kaibuchi&title=GSK-3%CE%B2+Regulates+Phosphorylation+of+CRMP-2+and+Neuronal+Polarity&doi=10.1016%2Fj.cell.2004.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">GSK-3β regulates phosphorylation of CRMP-2 and neuronal polarity</span></div><div class="casAuthors">Yoshimura, Takeshi; Kawano, Yoji; Arimura, Nariko; Kawabata, Saeko; Kikuchi, Akira; Kaibuchi, Kozo</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">137-149</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Neurons are highly polarized and comprised of two structurally and functionally distinct parts, an axon and dendrites.  We previously showed that collapsin response mediator protein-2 (CRMP-2) is crit. for specifying axon/dendrite fate, possibly by promoting neurite elongation via microtubule assembly.  Here, we showed that glycogen synthase kinase-3β (GSK-3β) phosphorylated CRMP-2 at Thr-514 and inactivated it.  The expression of the nonphosphorylated form of CRMP-2 or inhibition of GSK-3β induced the formation of multiple axon-like neurites in hippocampal neurons.  The expression of constitutively active GSK-3β impaired neuronal polarization, whereas the nonphosphorylated form of CRMP-2 counteracted the inhibitory effects of GSK-3β, indicating that GSK-3β regulates neuronal polarity through the phosphorylation of CRMP-2.  Treatment of hippocampal neurons with neurotrophin-3 (NT-3) induced inactivation of GSK-3β and dephosphorylation of CRMP-2.  Knockdown of CRMP-2 inhibited NT-3-induced axon outgrowth.  These results suggest that NT-3 decreases phosphorylated CRMP-2 and increases nonphosphorylated active CRMP-2, thereby promoting axon outgrowth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSzNEmMU885bVg90H21EOLACvtfcHk0liuAwgHQzf3Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXot1CgsQ%253D%253D&md5=8e54b169fadf81e88f3b93d26c4add95</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DT.%26aulast%3DKawano%26aufirst%3DY.%26aulast%3DArimura%26aufirst%3DN.%26aulast%3DKawabata%26aufirst%3DS.%26aulast%3DKikuchi%26aufirst%3DA.%26aulast%3DKaibuchi%26aufirst%3DK.%26atitle%3DGSK-3%25CE%25B2%2520Regulates%2520Phosphorylation%2520of%2520CRMP-2%2520and%2520Neuronal%2520Polarity%26jtitle%3DCell%26date%3D2005%26volume%3D120%26spage%3D137%26epage%3D149%26doi%3D10.1016%2Fj.cell.2004.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolattukudy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honnorat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikoshiba, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goshima, Y.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of CRMP2 (collapsin response mediator protein 2) is involved in proper dendritic field organization</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1360</span>– <span class="NLM_lpage">1365</span>, <span class="refDoi"> DOI: 10.1523/jneurosci.5563-11.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1523%2Fjneurosci.5563-11.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22279220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1Cmt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=1360-1365&author=N.+Yamashitaauthor=T.+Ohshimaauthor=F.+Nakamuraauthor=P.+Kolattukudyauthor=J.+Honnoratauthor=K.+Mikoshibaauthor=Y.+Goshima&title=Phosphorylation+of+CRMP2+%28collapsin+response+mediator+protein+2%29+is+involved+in+proper+dendritic+field+organization&doi=10.1523%2Fjneurosci.5563-11.2012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of CRMP2 (collapsin response mediator protein 2) is involved in proper dendritic field organization</span></div><div class="casAuthors">Yamashita, Naoya; Ohshima, Toshio; Nakamura, Fumio; Kolattukudy, Papachan; Honnorat, Jerome; Mikoshiba, Katsuhiko; Goshima, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1360-1365</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Collapsin response mediator proteins (CRMPs) are intracellular proteins that mediate signals for several extracellular mols., such as Semaphorin3A and neurotrophins.  The phosphorylation of CRMP1 and CRMP2 by Cdk5 at Ser522 is involved in axonal guidance and spine development.  Here, we found that the Ser522-phosphorylated CRMP1 and/or CRMP2 are enriched in the dendrites of cultured cortical neurons and P7 cortical section.  To det. the physiol. role of CRMPs in dendritic development, we generated CRMP2 knock-in mutant mice (crmp2ki/ki) in which the Ser residue at 522 was replaced with Ala.  Strikingly, the cortical basal dendrites of double mutant crmp2ki/ki and crmp1-/- mice exhibited severe abnormal dendritic patterning, which we defined as "curling phenotype.".  These findings demonstrate that the function of CRMP1 and CRMP2 synergistically control dendritic projection, and the phosphorylation of CRMP2 at Ser522 is essential for proper dendritic field organization in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr35ooJgSzUN7Vg90H21EOLACvtfcHk0lgPuLHBhcZraA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1Cmt7Y%253D&md5=8e52693649d0273a8758034f27a0017f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1523%2Fjneurosci.5563-11.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252Fjneurosci.5563-11.2012%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DN.%26aulast%3DOhshima%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DF.%26aulast%3DKolattukudy%26aufirst%3DP.%26aulast%3DHonnorat%26aufirst%3DJ.%26aulast%3DMikoshiba%26aufirst%3DK.%26aulast%3DGoshima%26aufirst%3DY.%26atitle%3DPhosphorylation%2520of%2520CRMP2%2520%2528collapsin%2520response%2520mediator%2520protein%25202%2529%2520is%2520involved%2520in%2520proper%2520dendritic%2520field%2520organization%26jtitle%3DJ.%2520Neurosci.%26date%3D2012%26volume%3D32%26spage%3D1360%26epage%3D1365%26doi%3D10.1523%2Fjneurosci.5563-11.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doble, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snider, W. D.</span></span> <span> </span><span class="NLM_article-title">GSK-3 is a master regulator of neural progenitor homeostasis</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1390</span>– <span class="NLM_lpage">1397</span>, <span class="refDoi"> DOI: 10.1038/nn.2408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnn.2408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=19801986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WgsbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=1390-1397&author=W.-Y.+Kimauthor=X.+Wangauthor=Y.+Wuauthor=B.+W.+Dobleauthor=S.+Patelauthor=J.+R.+Woodgettauthor=W.+D.+Snider&title=GSK-3+is+a+master+regulator+of+neural+progenitor+homeostasis&doi=10.1038%2Fnn.2408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">GSK-3 is a master regulator of neural progenitor homeostasis</span></div><div class="casAuthors">Kim, Woo-Yang; Wang, Xinshuo; Wu, Yaohong; Doble, Bradley W.; Patel, Satish; Woodgett, James R.; Snider, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1390-1397</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The development of the brain requires the exquisite coordination of progenitor proliferation and differentiation to achieve complex circuit assembly.  It has been suggested that glycogen synthase kinase 3 (GSK-3) acts as an integrating mol. for multiple proliferation and differentiation signals because of its essential role in the RTK, Wnt and Shh signaling pathways.  We created conditional mutations that deleted both the α and β forms of GSK-3 in mouse neural progenitors.  GSK-3 deletion resulted in massive hyperproliferation of neural progenitors along the entire neuraxis.  Generation of both intermediate neural progenitors and postmitotic neurons was markedly suppressed.  These effects were assocd. with the dysregulation of β-catenin, Sonic Hedgehog, Notch and fibroblast growth factor signaling.  Our results indicate that GSK-3 signaling is an essential mediator of homeostatic controls that regulate neural progenitors during mammalian brain development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovTk0M-V9NYrVg90H21EOLACvtfcHk0liqBUL0vrfQ3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WgsbzJ&md5=d7ff5a16e07b70e8b4338316c5d70b7e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnn.2408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2408%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.-Y.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDoble%26aufirst%3DB.%2BW.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DWoodgett%26aufirst%3DJ.%2BR.%26aulast%3DSnider%26aufirst%3DW.%2BD.%26atitle%3DGSK-3%2520is%2520a%2520master%2520regulator%2520of%2520neural%2520progenitor%2520homeostasis%26jtitle%3DNat.%2520Neurosci.%26date%3D2009%26volume%3D12%26spage%3D1390%26epage%3D1397%26doi%3D10.1038%2Fnn.2408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theriault, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span> <span> </span><span class="NLM_article-title">A High-Throughput Screen for Wnt/β-Catenin Signaling Pathway Modulators in Human iPSC-Derived Neural Progenitors</span>. <i>J. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1263</span>, <span class="refDoi"> DOI: 10.1177/1087057112456876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1177%2F1087057112456876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=22923789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A280%3ADC%252BC38bhsVegtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1252-1263&author=W.-N.+Zhaoauthor=C.+Chengauthor=K.+M.+Theriaultauthor=S.+D.+Sheridanauthor=L.-H.+Tsaiauthor=S.+J.+Haggarty&title=A+High-Throughput+Screen+for+Wnt%2F%CE%B2-Catenin+Signaling+Pathway+Modulators+in+Human+iPSC-Derived+Neural+Progenitors&doi=10.1177%2F1087057112456876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A high-throughput screen for Wnt/β-catenin signaling pathway modulators in human iPSC-derived neural progenitors</span></div><div class="casAuthors">Zhao Wen-Ning; Cheng Chialin; Theriault Kraig M; Sheridan Steven D; Tsai Li-Huei; Haggarty Stephen J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1252-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Wnt/β-catenin signaling has emerged as a central player in pathways implicated in the pathophysiology and treatment of neuropsychiatric disorders.  To identify potential novel therapeutics for these disorders, high-throughput screening (HTS) assays reporting on Wnt/β-catenin signaling in disease-relevant contexts are needed.  The use of human patient-derived induced pluripotent stem cell (iPSC) models provides ideal disease-relevant context if these stem cell cultures can be adapted for HTS-compatible formats.  Here, we describe a sensitive, HTS-compatible Wnt/β-catenin signaling reporter system generated in homogeneous, expandable neural progenitor cells (NPCs) derived from human iPSCs.  We validated this system by demonstrating dose-responsive stimulation by several known Wnt/β-catenin signaling pathway modulators, including Wnt3a, a glycogen synthase kinase-3 (GSK3) inhibitor, and the bipolar disorder therapeutic lithium.  These responses were robust and reproducible over time across many repeated assays.  We then conducted a screen of ~1500 compounds from a library of Food and Drug Administration-approved drugs and known bioactives and confirmed the HTS hits, revealing multiple chemical and biological classes of novel small-molecule probes of Wnt/β-catenin signaling.  Generating these type of pathway-selective, cell-based phenotypic assays in human iPSC-derived neural cells will advance the field of human experimental neurobiology toward the goal of identifying and validating targets for neuropsychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTNG8hBgzHx2K_SFaaT9INfW6udTcc2eYOHi07nr90Dbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38bhsVegtw%253D%253D&md5=ad3b356f5acafe845e06450a6ac39f1e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1177%2F1087057112456876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112456876%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DW.-N.%26aulast%3DCheng%26aufirst%3DC.%26aulast%3DTheriault%26aufirst%3DK.%2BM.%26aulast%3DSheridan%26aufirst%3DS.%2BD.%26aulast%3DTsai%26aufirst%3DL.-H.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26atitle%3DA%2520High-Throughput%2520Screen%2520for%2520Wnt%252F%25CE%25B2-Catenin%2520Signaling%2520Pathway%2520Modulators%2520in%2520Human%2520iPSC-Derived%2520Neural%2520Progenitors%26jtitle%3DJ.%2520Biomol.%2520Screen.%26date%3D2012%26volume%3D17%26spage%3D1252%26epage%3D1263%26doi%3D10.1177%2F1087057112456876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harigaya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawarabayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsubara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakazato, Y.</span></span> <span> </span><span class="NLM_article-title">Amyloid cored plaques in Tg2576 transgenic mice are characterized by giant plaques, slightly activated microglia, and the lack of paired helical filament-typed, dystrophic neurites</span>. <i>Virchows Arch.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>441</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1007/s00428-002-0643-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1007%2Fs00428-002-0643-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=12404061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkslSrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=441&publication_year=2002&pages=358-367&author=A.+Sasakiauthor=M.+Shojiauthor=Y.+Harigayaauthor=T.+Kawarabayashiauthor=M.+Ikedaauthor=M.+Naitoauthor=E.+Matsubaraauthor=K.+Abeauthor=Y.+Nakazato&title=Amyloid+cored+plaques+in+Tg2576+transgenic+mice+are+characterized+by+giant+plaques%2C+slightly+activated+microglia%2C+and+the+lack+of+paired+helical+filament-typed%2C+dystrophic+neurites&doi=10.1007%2Fs00428-002-0643-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Amyloid cored plaques in Tg2576 transgenic mice are characterized by giant plaques, slightly activated microglia, and the lack of paired helical filament-typed, dystrophic neurites</span></div><div class="casAuthors">Sasaki, Atsushi; Shoji, Mikio; Harigaya, Yasuo; Kawarabayashi, Takeshi; Ikeda, Masaki; Naito, Makoto; Matsubara, Etsuro; Abe, Koji; Nakazato, Yoichi</div><div class="citationInfo"><span class="NLM_cas:title">Virchows Archiv</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">441</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-367</span>CODEN:
                <span class="NLM_cas:coden">VARCEM</span>;
        ISSN:<span class="NLM_cas:issn">0945-6317</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">We examd. the brains of Tg2576 transgenic mice carrying human amyloid precursor protein with the Swedish mutation and Alzheimer's disease (AD) by means of immunohistochem. and electron microscopy to clarify the characteristics of amyloid-assocd. pathol. in the transgenic mice.  In 12- to 29-mo-old Tg2576 mice, congophilic cored plaques in the neocortex and hippocampus were labeled by all of the Aβ1-, Aβ40- and 42-specific antibodies, as seen in the classical plaques in AD.  However, large-sized (>50 μm in core diam.) plaques were seen more frequently in the older mice (18-29 mo) than in those with AD (approx. 20% vs 2% in total cored plaques), and Tg2576 mice contained giant plaques (>75 μm in core diam.), which were almost never seen in the brain of those with AD.  Neither thread-like structures nor peripheral coronas were obsd. in the cored plaques of the transgenic mice in the silver impregnations.  Immunohistochem., plaque-accompanied microglia showed a slight enlargement of the cytoplasm with consistent labeling of Mac-1 and macrosialin (murine CD68), and with partial labeling of Ia antigen and macrophage-colony stimulating factor receptor.  Ultrastructurally, the microglia surrounding the extracellular amyloid fibrils in the large, cored plaques showed some organella with phagocytic activity, such as secondary lysosomal, dense bodies, but intracellular amyloid fibrils were not evident.  Dystrophic neurites in the plaques of the transgenic mice contained many dense multilaminar bodies, but no paired helical filaments.  Our results suggest that giant cored plaques without coronas or paired helical filament-typed, dystrophic neurites are characteristic in Tg2576 mice, and that plaque-assocd. microglia in transgenic mice are activated to be in phagocytic function but not sufficient enough to digest extracellularly deposited amyloid fibrils.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbPbvLPXqSzLVg90H21EOLACvtfcHk0lgp-ESfG7mF9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkslSrtg%253D%253D&md5=9fac7e33706e4a04ee95d99f55e1942c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs00428-002-0643-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00428-002-0643-8%26sid%3Dliteratum%253Aachs%26aulast%3DSasaki%26aufirst%3DA.%26aulast%3DShoji%26aufirst%3DM.%26aulast%3DHarigaya%26aufirst%3DY.%26aulast%3DKawarabayashi%26aufirst%3DT.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DMatsubara%26aufirst%3DE.%26aulast%3DAbe%26aufirst%3DK.%26aulast%3DNakazato%26aufirst%3DY.%26atitle%3DAmyloid%2520cored%2520plaques%2520in%2520Tg2576%2520transgenic%2520mice%2520are%2520characterized%2520by%2520giant%2520plaques%252C%2520slightly%2520activated%2520microglia%252C%2520and%2520the%2520lack%2520of%2520paired%2520helical%2520filament-typed%252C%2520dystrophic%2520neurites%26jtitle%3DVirchows%2520Arch.%26date%3D2002%26volume%3D441%26spage%3D358%26epage%3D367%26doi%3D10.1007%2Fs00428-002-0643-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bero, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirrito, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raichle, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzman, D. M.</span></span> <span> </span><span class="NLM_article-title">Neuronal activity regulates the regional vulnerability to amyloid-β deposition</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nn.2801</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnn.2801" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=21532579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVGltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=750-756&author=A.+W.+Beroauthor=P.+Yanauthor=J.+H.+Rohauthor=J.+R.+Cirritoauthor=F.+R.+Stewartauthor=M.+E.+Raichleauthor=J.-M.+Leeauthor=D.+M.+Holtzman&title=Neuronal+activity+regulates+the+regional+vulnerability+to+amyloid-%CE%B2+deposition&doi=10.1038%2Fnn.2801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Neuronal activity regulates the regional vulnerability to amyloid-β deposition</span></div><div class="casAuthors">Bero, Adam W.; Yan, Ping; Roh, Jee Hoon; Cirrito, John R.; Stewart, Floy R.; Raichle, Marcus E.; Lee, Jin-Moo; Holtzman, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Amyloid-β (Aβ) plaque deposition in specific brain regions is a pathol. hallmark of Alzheimer's disease.  However, the mechanism underlying the regional vulnerability to Aβ deposition in Alzheimer's disease is unknown.  Herein, we provide evidence that endogenous neuronal activity regulates the regional concn. of interstitial fluid (ISF) Aβ, which drives local Aβ aggregation.  Using in vivo microdialysis, we show that ISF Aβ concns. in several brain regions of APP transgenic mice before plaque deposition were commensurate with the degree of subsequent plaque deposition and with the concn. of lactate, a marker of neuronal activity.  Furthermore, unilateral vibrissal stimulation increased ISF Aβ, and unilateral vibrissal deprivation decreased ISF Aβ and lactate, in contralateral barrel cortex.  Long-term unilateral vibrissal deprivation decreased amyloid plaque formation and growth.  Our results suggest a mechanism to account for the vulnerability of specific brain regions to Aβ deposition in Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvnBN2KKH8WbVg90H21EOLACvtfcHk0lgp-ESfG7mF9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVGltro%253D&md5=d1adad353ca3c996d554bb7112b9859a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnn.2801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2801%26sid%3Dliteratum%253Aachs%26aulast%3DBero%26aufirst%3DA.%2BW.%26aulast%3DYan%26aufirst%3DP.%26aulast%3DRoh%26aufirst%3DJ.%2BH.%26aulast%3DCirrito%26aufirst%3DJ.%2BR.%26aulast%3DStewart%26aufirst%3DF.%2BR.%26aulast%3DRaichle%26aufirst%3DM.%2BE.%26aulast%3DLee%26aufirst%3DJ.-M.%26aulast%3DHoltzman%26aufirst%3DD.%2BM.%26atitle%3DNeuronal%2520activity%2520regulates%2520the%2520regional%2520vulnerability%2520to%2520amyloid-%25CE%25B2%2520deposition%26jtitle%3DNat.%2520Neurosci.%26date%3D2011%26volume%3D14%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnn.2801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernard-Gauthier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirrmacher, R.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the development and application of radiolabeled kinase inhibitors for PET imaging</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">22000</span>– <span class="NLM_lpage">22027</span>, <span class="refDoi"> DOI: 10.3390/molecules201219816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.3390%2Fmolecules201219816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=26690113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVKgsL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=22000-22027&author=V.+Bernard-Gauthierauthor=J.+Baileyauthor=S.+Berkeauthor=R.+Schirrmacher&title=Recent+advances+in+the+development+and+application+of+radiolabeled+kinase+inhibitors+for+PET+imaging&doi=10.3390%2Fmolecules201219816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the development and application of radiolabeled kinase inhibitors for PET imaging</span></div><div class="casAuthors">Bernard-Gauthier, Vadim; Bailey, Justin J.; Berke, Sheldon; Schirrmacher, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">22000-22027</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Over the last 20 years, intensive investigation and multiple clin. successes targeting protein kinases, mostly for cancer treatment, have identified small mol. kinase inhibitors as a prominent therapeutic class.  In the course of those investigations, radiolabeled kinase inhibitors for positron emission tomog. (PET) imaging have been synthesized and evaluated as diagnostic imaging probes for cancer characterization.  Given that inhibitor coverage of the kinome is continuously expanding, in vivo PET imaging will likely find increasing applications for therapy monitoring and receptor d. studies both in- and outside of oncol. conditions.  Early investigated radiolabeled inhibitors, which are mostly based on clin. approved tyrosine kinase inhibitor (TKI) isotopologues, have now entered clin. trials.  Novel radioligands for cancer and PET neuroimaging originating from novel but relevant target kinases are currently being explored in preclin. studies.  This article reviews the literature involving radiotracer design, radiochem. approaches, biol. tracer evaluation and nuclear imaging results of radiolabeled kinase inhibitors for PET reported between 2010 and mid-2015.  Aspects regarding the usefulness of pursuing selective vs. promiscuous inhibitor scaffolds and the inherent challenges assocd. with intracellular enzyme imaging will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoK7e4uBNDBLVg90H21EOLACvtfcHk0lgp-ESfG7mF9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVKgsL%252FI&md5=be84e68fe296d62fdcfd15857dda05f0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3390%2Fmolecules201219816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules201219816%26sid%3Dliteratum%253Aachs%26aulast%3DBernard-Gauthier%26aufirst%3DV.%26aulast%3DBailey%26aufirst%3DJ.%26aulast%3DBerke%26aufirst%3DS.%26aulast%3DSchirrmacher%26aufirst%3DR.%26atitle%3DRecent%2520advances%2520in%2520the%2520development%2520and%2520application%2520of%2520radiolabeled%2520kinase%2520inhibitors%2520for%2520PET%2520imaging%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D22000%26epage%3D22027%26doi%3D10.3390%2Fmolecules201219816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: the road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+the+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lgnrQ_AxR4vlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520the%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D377%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimwood, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaddan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zasadny, K.</span></span> <span> </span><span class="NLM_article-title">Design and selection parameters to accelerate the discovery of novel central nervous system positron emission tomography (PET) ligands and their application in the development of a novel phosphodiesterase 2A PET ligand</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4568</span>– <span class="NLM_lpage">4579</span>, <span class="refDoi"> DOI: 10.1021/jm400312y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400312y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4568-4579&author=L.+Zhangauthor=A.+Villalobosauthor=E.+M.+Beckauthor=T.+Bocanauthor=T.+A.+Chappieauthor=L.+Chenauthor=S.+Grimwoodauthor=S.+D.+Heckauthor=C.+J.+Helalauthor=X.+Houauthor=J.+M.+Humphreyauthor=J.+Luauthor=M.+B.+Skaddanauthor=T.+J.+McCarthyauthor=P.+R.+Verhoestauthor=T.+T.+Wagerauthor=K.+Zasadny&title=Design+and+selection+parameters+to+accelerate+the+discovery+of+novel+central+nervous+system+positron+emission+tomography+%28PET%29+ligands+and+their+application+in+the+development+of+a+novel+phosphodiesterase+2A+PET+ligand&doi=10.1021%2Fjm400312y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Selection Parameters to Accelerate the Discovery of Novel Central Nervous System Positron Emission Tomography (PET) Ligands and Their Application in the Development of a Novel Phosphodiesterase 2A PET Ligand</span></div><div class="casAuthors">Zhang, Lei; Villalobos, Anabella; Beck, Elizabeth M.; Bocan, Thomas; Chappie, Thomas A.; Chen, Laigao; Grimwood, Sarah; Heck, Steven D.; Helal, Christopher J.; Hou, Xinjun; Humphrey, John M.; Lu, Jiemin; Skaddan, Marc B.; McCarthy, Timothy J.; Verhoest, Patrick R.; Wager, Travis T.; Zasadny, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4568-4579</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To accelerate the discovery of novel small mol. central nervous system (CNS) positron emission tomog. (PET) ligands, we aimed to define a property space that would facilitate ligand design and prioritization, thereby providing a higher probability of success for novel PET ligand development.  Toward this end, we built a database consisting of 62 PET ligands that have successfully reached the clinic and 15 radioligands that failed in late-stage development as neg. controls.  A systematic anal. of these ligands identified a set of preferred parameters for physicochem. properties, brain permeability, and nonspecific binding (NSB).  These preferred parameters have subsequently been applied to several programs and have led to the successful development of novel PET ligands with reduced resources and timelines.  This strategy is illustrated here by the discovery of the novel phosphodiesterase 2A (PDE2A) PET ligand 4-(3-[18F]fluoroazetidin-1-yl)-7-methyl-5-{1-methyl-5-[4-(trifluoromethyl)phenyl]-1H-pyrazol-4-yl}imidazo[5,1-f][1,2,4]triazine, [18F]PF-05270430 (5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTCG9TZE0X-bVg90H21EOLACvtfcHk0lgnrQ_AxR4vlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D&md5=ee7ce04829d7456a632c02c331c4edfa</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm400312y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400312y%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DVillalobos%26aufirst%3DA.%26aulast%3DBeck%26aufirst%3DE.%2BM.%26aulast%3DBocan%26aufirst%3DT.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DGrimwood%26aufirst%3DS.%26aulast%3DHeck%26aufirst%3DS.%2BD.%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DSkaddan%26aufirst%3DM.%2BB.%26aulast%3DMcCarthy%26aufirst%3DT.%2BJ.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DZasadny%26aufirst%3DK.%26atitle%3DDesign%2520and%2520selection%2520parameters%2520to%2520accelerate%2520the%2520discovery%2520of%2520novel%2520central%2520nervous%2520system%2520positron%2520emission%2520tomography%2520%2528PET%2529%2520ligands%2520and%2520their%2520application%2520in%2520the%2520development%2520of%2520a%2520novel%2520phosphodiesterase%25202A%2520PET%2520ligand%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4568%26epage%3D4579%26doi%3D10.1021%2Fjm400312y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Englebienne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 5. Force-field-based prediction of binding affinities of ligands to proteins</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2564</span>– <span class="NLM_lpage">2571</span>, <span class="refDoi"> DOI: 10.1021/ci900251k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900251k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtleqtLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2564-2571&author=P.+Englebienneauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+5.+Force-field-based+prediction+of+binding+affinities+of+ligands+to+proteins&doi=10.1021%2Fci900251k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. 5. Force-Field-Based Prediction of Binding Affinities of Ligands to Proteins</span></div><div class="casAuthors">Englebienne, Pablo; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2564-2571</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein our efforts in the development of three empirical scoring functions with application in protein-ligand docking.  A first scoring function was developed from 209 crystal structures of protein-ligand complexes and a second one from 946 cross-docked complexes.  Tuning of the coeffs. for the different terms making up these functions was performed by an iterative approach to optimize the correlations between obsd. activities and calcd. scores.  A third scoring function was developed from libraries of known actives and decoys docked to six different protein conformational ensembles.  In the latter case, the tuning of the coeffs. was performed so as to optimize the area under the curve of a receiver operating characteristic (ROC) for the discrimination of actives and inactives.  The newly developed scoring functions were next assessed on independent sets of protein-ligand complexes for their ability to predict binding affinities and to discriminate actives from inactives.  In the first validation the first function, which was trained on active compds. only, performed as well as other commonly used ones.  On a high-throughput virtual screening validation on five protein conformational ensembles, the third scoring function that included data from inactive compds. performed significantly better.  This validation showed that the inclusion of data from inactive compds. is crit. for performance in virtual high-throughput screening applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJkz6UVDkQ8LVg90H21EOLACvtfcHk0ljM5w6s-nx91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtleqtLrJ&md5=25916732a94e7cb8b4db2cdb5c913828</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci900251k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900251k%26sid%3Dliteratum%253Aachs%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25205.%2520Force-field-based%2520prediction%2520of%2520binding%2520affinities%2520of%2520ligands%2520to%2520proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2564%26epage%3D2571%26doi%3D10.1021%2Fci900251k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englebienne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza-Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbeil, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagna-Slater, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Integrating Medicinal Chemistry, Organic/Combinatorial Chemistry, and Computational Chemistry for the Discovery of Selective Estrogen Receptor Modulators with Forecaster, a Novel Platform for Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1021/ci2004779</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2004779" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=210-224&author=E.+Therrienauthor=P.+Englebienneauthor=A.+G.+Arrowsmithauthor=R.+Mendoza-Sanchezauthor=C.+R.+Corbeilauthor=N.+Weillauthor=V.+Campagna-Slaterauthor=N.+Moitessier&title=Integrating+Medicinal+Chemistry%2C+Organic%2FCombinatorial+Chemistry%2C+and+Computational+Chemistry+for+the+Discovery+of+Selective+Estrogen+Receptor+Modulators+with+Forecaster%2C+a+Novel+Platform+for+Drug+Discovery&doi=10.1021%2Fci2004779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating Medicinal Chemistry, Organic/Combinatorial Chemistry, and Computational Chemistry for the Discovery of Selective Estrogen Receptor Modulators with Forecaster, a Novel Platform for Drug Discovery</span></div><div class="casAuthors">Therrien, Eric; Englebienne, Pablo; Arrowsmith, Andrew G.; Mendoza-Sanchez, Rodrigo; Corbeil, Christopher R.; Weill, Nathanael; Campagna-Slater, Valerie; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-224</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of a large medicinal chem. program, we wish to develop novel selective estrogen receptor modulators (SERMs) as potential breast cancer treatments using a combination of exptl. and computational approaches.  However, one of the remaining difficulties nowadays is to fully integrate computational (i.e., virtual, theor.) and medicinal (i.e., exptl., intuitive) chem. to take advantage of the full potential of both.  For this purpose, we have developed a Web-based platform, Forecaster, and a no. of programs (e.g., Prep., React, Select) with the aim of combining computational chem. and medicinal chem. expertise to facilitate drug discovery and development and more specifically to integrate synthesis into computer-aided drug design.  In our quest for potent SERMs, this platform was used to build virtual combinatorial libraries, filter and ext. a highly diverse library from the NCI database, and dock them to the estrogen receptor (ER), with all of these steps being fully automated by computational chemists for use by medicinal chemists.  As a result, virtual screening of a diverse library seeded with active compds. followed by a search for analogs yielded an enrichment factor of 129, with 98% of the seeded active compds. recovered, while the screening of a designed virtual combinatorial library including known actives yielded an area under the receiver operating characteristic (AU-ROC) of 0.78.  The lead optimization proved less successful, further demonstrating the challenge to simulate structure activity relationship studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt6YsO45gkpLVg90H21EOLACvtfcHk0ljM5w6s-nx91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCru7vE&md5=a7b3218263c14ae60a38962eed567e87</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci2004779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2004779%26sid%3Dliteratum%253Aachs%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DArrowsmith%26aufirst%3DA.%2BG.%26aulast%3DMendoza-Sanchez%26aufirst%3DR.%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DIntegrating%2520Medicinal%2520Chemistry%252C%2520Organic%252FCombinatorial%2520Chemistry%252C%2520and%2520Computational%2520Chemistry%2520for%2520the%2520Discovery%2520of%2520Selective%2520Estrogen%2520Receptor%2520Modulators%2520with%2520Forecaster%252C%2520a%2520Novel%2520Platform%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D210%26epage%3D224%26doi%3D10.1021%2Fci2004779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corbeil, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englebienne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yannopoulos, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilimoria, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">L’Heureux, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moitessier, N.</span></span> <span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 2. Development and application of FITTED 1.5 to the virtual screening of potential HCV polymerase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">902</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1021/ci700398h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700398h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVWgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=902-909&author=C.+R.+Corbeilauthor=P.+Englebienneauthor=C.+G.+Yannopoulosauthor=L.+Chanauthor=S.+K.+Dasauthor=D.+Bilimoriaauthor=L.+L%E2%80%99Heureuxauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+2.+Development+and+application+of+FITTED+1.5+to+the+virtual+screening+of+potential+HCV+polymerase+inhibitors&doi=10.1021%2Fci700398h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. Development and Application of FITTED 1.5 to the Virtual Screening of Potential HCV Polymerase Inhibitors</span></div><div class="casAuthors">Corbeil, Christopher R.; Englebienne, Pablo; Yannopoulos, Constantin G.; Chan, Laval; Das, Sanjoy K.; Bilimoria, Darius; L'Heureux, Lucille; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">902-909</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HCV NS5B polymerase is a validated target for the treatment of hepatitis C, known to be one of the most challenging enzymes for docking programs.  In order to improve the low accuracy of existing docking methods obsd. with this challenging enzyme, the authors have significantly modified and updated Fitted 1.0, a recently reported docking program, into FITTED 1.5.  This enhanced version is now applicable to the virtual screening of compd. libraries and includes new features such as filters and pharmacophore- or interaction-site-oriented docking.  As a first validation, FITTED 1.5 was applied to the testing set previously developed for FITTED 1.0 and extended to include hepatitis C virus (HCV) polymerase inhibitors.  This first validation showed an increased accuracy as well as an increase in speed.  It also shows that the accuracy toward HCV polymerase is better than previously obsd. with other programs.  Next, application of FITTED 1.5 to the virtual screening of the Maybridge library seeded with known HCV polymerase inhibitors revealed its ability to recover most of these actives in the top 5% of the hit list.  As a third validation, further biol. assays uncovered HCV polymerase inhibition for selected Maybridge compds. ranked in the top of the hit list.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXlpXNyj_8m7Vg90H21EOLACvtfcHk0ljM5w6s-nx91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVWgtr4%253D&md5=6a661760aea54b56b878b7f64c9e3ca7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci700398h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700398h%26sid%3Dliteratum%253Aachs%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DYannopoulos%26aufirst%3DC.%2BG.%26aulast%3DChan%26aufirst%3DL.%26aulast%3DDas%26aufirst%3DS.%2BK.%26aulast%3DBilimoria%26aufirst%3DD.%26aulast%3DL%25E2%2580%2599Heureux%26aufirst%3DL.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25202.%2520Development%2520and%2520application%2520of%2520FITTED%25201.5%2520to%2520the%2520virtual%2520screening%2520of%2520potential%2520HCV%2520polymerase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D902%26epage%3D909%26doi%3D10.1021%2Fci700398h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N','PDB','5K5N'); return false;">PDB: 5K5N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR','PDB','3ANR'); return false;">PDB: 3ANR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NM3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NM3','PDB','4NM3'); return false;">PDB: 4NM3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK','PDB','5KPK'); return false;">PDB: 5KPK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i74"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01030">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_76835"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01030">10.1021/acs.jmedchem.9b01030</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures illustrating chemistry and radiochemistry methods, HEK-293T P301S hTau cellular assay, in vitro autoradiography, kinase assays, MST assays, culturing and neural differentiation of human iPSC-derived NPCs, WNT signaling reporter assays, NPC proliferation assay with CellTiter-Glo, high-content neuronal assay for p-CRMP2<sup>T514</sup>, pTau assay, western blot analysis, rodents and nonhuman primate PET imaging studied, neuropharmacokinetic analysis, and NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_002.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_001.pdf">jm9b01030_si_001.pdf (14.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01030/suppl_file/jm9b01030_si_002.xlsx">jm9b01030_si_002.xlsx (13.72 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Existing PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KPK">5KPK</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K5N">5K5N</a> were used for in silico docking studies.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01030&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01030%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-21%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01030" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997da5f8393dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
