
<html lang="en"     class="pb-page"  data-request-id="915d68ad-3e0d-4cae-863b-29e4d6abc345"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c00796;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor" /></meta><meta name="dc.Creator" content="Jonathan T.  Seal" /></meta><meta name="dc.Creator" content="Stephen J.  Atkinson" /></meta><meta name="dc.Creator" content="Helen  Aylott" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Royston C. B.  Copley" /></meta><meta name="dc.Creator" content="Laurie  Gordon" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="James R. J.  Gray" /></meta><meta name="dc.Creator" content="Lee A.  Harrison" /></meta><meta name="dc.Creator" content="Thomas G.  Hayhow" /></meta><meta name="dc.Creator" content="Matthew  Lindon" /></meta><meta name="dc.Creator" content="Cassie  Messenger" /></meta><meta name="dc.Creator" content="Anne-Marie  Michon" /></meta><meta name="dc.Creator" content="Darren  Mitchell" /></meta><meta name="dc.Creator" content="Alex  Preston" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Inmaculada  Rioja" /></meta><meta name="dc.Creator" content="Simon  Taylor" /></meta><meta name="dc.Creator" content="Ian D.  Wall" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="James M.  Woolven" /></meta><meta name="dc.Creator" content="Emmanuel H.  Demont" /></meta><meta name="dc.Description" content="The profound efficacy, yet associated toxicity of pan-BET inhibitors is well documented. The possibility of an ameliorated safety profile driven by significantly selective (&gt;100-fold) inhibition of..." /></meta><meta name="Description" content="The profound efficacy, yet associated toxicity of pan-BET inhibitors is well documented. The possibility of an ameliorated safety profile driven by significantly selective (&gt;100-fold) inhibition of..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 23, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00796" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00796" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00796" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00796" /></link>
        
    
    

<title>The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00796" /></meta><meta property="og:title" content="The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0020.jpeg" /></meta><meta property="og:description" content="The profound efficacy, yet associated toxicity of pan-BET inhibitors is well documented. The possibility of an ameliorated safety profile driven by significantly selective (&gt;100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homology. Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors. A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046), was the basis for the high selectivity observed. This culminated in two tool molecules, 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement. Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00796"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00796">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00796&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00796&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00796&amp;href=/doi/10.1021/acs.jmedchem.0c00796" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9093-9126</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00605" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01832" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jonathan T. Seal</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan T. Seal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>For J.T.S: phone, 01438764528; email, <a href="/cdn-cgi/l/email-protection#38525756164c164b5d5954785f4b53165b5755"><span class="__cf_email__" data-cfemail="5a303534742e74293f3b361a3d293174393537">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan+T.++Seal">Jonathan T. Seal</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0148-5487" title="Orcid link">http://orcid.org/0000-0003-0148-5487</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Stephen J. Atkinson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen J. Atkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>For S.J.A: email, <a href="/cdn-cgi/l/email-protection#ee9d9a8b9e868b80c08f9a8587809d8180dfae8f9d9a9c8f948b808b8d8fc08d8183"><span class="__cf_email__" data-cfemail="4d3e39283d252823632c392624233e22237c0d2c3e393f2c372823282e2c632e2220">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+J.++Atkinson">Stephen J. Atkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3636-3674" title="Orcid link">http://orcid.org/0000-0003-3636-3674</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Helen Aylott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helen Aylott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helen++Aylott">Helen Aylott</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Royston C. B. Copley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Royston C. B. Copley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Royston+C.+B.++Copley">Royston C. B. Copley</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Laurie Gordon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurie Gordon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurie++Gordon">Laurie Gordon</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Paola Grandi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Grandi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">James R. J. Gray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James R. J. Gray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Quantitative Pharmacology,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+R.+J.++Gray">James R. J. Gray</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lee A. Harrison</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lee A. Harrison</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lee+A.++Harrison">Lee A. Harrison</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Thomas G. Hayhow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas G. Hayhow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+G.++Hayhow">Thomas G. Hayhow</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Matthew Lindon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew Lindon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++Lindon">Matthew Lindon</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Cassie Messenger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cassie Messenger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cassie++Messenger">Cassie Messenger</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Anne-Marie Michon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Marie Michon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Michon">Anne-Marie Michon</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Darren Mitchell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren Mitchell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren++Mitchell">Darren Mitchell</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Alex Preston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex Preston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex++Preston">Alex Preston</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0334-0679" title="Orcid link">http://orcid.org/0000-0003-0334-0679</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Inmaculada Rioja</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Inmaculada Rioja</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Inmaculada++Rioja">Inmaculada Rioja</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Simon Taylor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon Taylor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Quantitative Pharmacology,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Taylor">Simon Taylor</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ian D. Wall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian D. Wall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian+D.++Wall">Ian D. Wall</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">James M. Woolven</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James M. Woolven</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+M.++Woolven">James M. Woolven</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Emmanuel H. Demont</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emmanuel H. Demont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit,Â  Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H.++Demont">Emmanuel H. Demont</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7307-3129" title="Orcid link">http://orcid.org/0000-0001-7307-3129</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00796&amp;href=/doi/10.1021%2Facs.jmedchem.0c00796" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9093â9126</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 23, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 May 2020</li><li><span class="item_label"><b>Published</b> online</span>23 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00796</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editorsâ Choice" /></img><span>ACS Editorsâ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00796"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">11506</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00796" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;T. Seal&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;J. Atkinson&quot;},{&quot;first_name&quot;:&quot;Helen&quot;,&quot;last_name&quot;:&quot;Aylott&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Royston&quot;,&quot;last_name&quot;:&quot;C. B. Copley&quot;},{&quot;first_name&quot;:&quot;Laurie&quot;,&quot;last_name&quot;:&quot;Gordon&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;R. J. Gray&quot;},{&quot;first_name&quot;:&quot;Lee&quot;,&quot;last_name&quot;:&quot;A. Harrison&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;G. Hayhow&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;Lindon&quot;},{&quot;first_name&quot;:&quot;Cassie&quot;,&quot;last_name&quot;:&quot;Messenger&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Michon&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;Mitchell&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;Preston&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Inmaculada&quot;,&quot;last_name&quot;:&quot;Rioja&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Taylor&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;D. Wall&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;M. Woolven&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H. Demont&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9093-9126&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00796&quot;},&quot;abstract&quot;:&quot;The profound efficacy, yet associated toxicity of pan-BET inhibitors is well documented. The possibility of an ameliorated safety profile driven by significantly selective (&gt;100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homology. Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors. A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046), was the basis for the high selectivity observed. This culminated in two tool molecules, 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement. Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00796&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00796" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00796&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00796" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00796&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00796" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00796&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00796&amp;href=/doi/10.1021/acs.jmedchem.0c00796" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00796" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00796" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26pmid%3D32702236%26genre%3Darticle%26aulast%3DSeal%26date%3D2020%26atitle%3DThe%2BOptimization%2Bof%2Ba%2BNovel%252C%2BWeak%2BBromo%2Band%2BExtra%2BTerminal%2BDomain%2B%2528BET%2529%2BBromodomain%2BFragment%2BLigand%2Bto%2Ba%2BPotent%2Band%2BSelective%2BSecond%2BBromodomain%2B%2528BD2%2529%2BInhibitor%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9093%26epage%3D9126%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The profound efficacy, yet associated toxicity of pan-BET inhibitors is well documented. The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homology. Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors. A template hopping approach, enabled by our parallel research into an orthogonal template (<b>15</b>, GSK046), was the basis for the high selectivity observed. This culminated in two tool molecules, <b>20</b> (GSK620) and <b>56</b> (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement. Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain and extra terminal (BET) family of proteins, comprising the ubiquitous BRD2, 3, and 4 and the testis restricted BRDT, are epigenetic readers whose tandem bromodomains (BDs: N terminus, BD1; and C-terminus, BD2) recognize and bind acetylated lysine histone tail peptides of nucleosomes. Once bound to chromatin these proteins recruit transcriptional complexes and therefore play a key role in gene transcription.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The therapeutic potential of a pan-bromo and extra terminal domain inhibitor (iBET), binding equipotently to the eight bromodomains of the family, has now been extensively reported both in oncology<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11">(2â11)</a> and immunoinflammation.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23">(12â23)</a> Moreover, iBETs are actively progressing in the clinic for oncology indications, illustrating the tremendous potential of this epigenetic reader family as a therapeutic target.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24â26)</a> Nevertheless, it is also well-known that a number of nonclinical toxicities and dose-limiting clinical findings have been associated with pan-inhibition of the BET family.<a onclick="showRef(event, 'ref25 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref25 ref27 ref28 ref29">(25,27â29)</a> There have been far fewer reports of selective BET bromodomain inhibitors, although these are now starting to emerge in the patent<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37">(30â37)</a> and publication literature. Both BD1 (<b>1</b>â<b>7</b>)<a onclick="showRef(event, 'ref38 ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41 ref42 ref43 ref44">(38â44)</a> and BD2 (<b>8</b>â<b>12</b>)<a onclick="showRef(event, 'ref45 ref46 ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47 ref48 ref49 ref50">(45â50)</a> selective (>5-fold) inhibitors, binding to either the four BD1 or the four BD2 domains of the BET proteins have been published and the structures are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> with the acetylated lysine mimetics highlighted in blue. Despite the emergence of such inhibitors, there is some debate as to whether 50â100-fold selective inhibitors are really capable of teasing out the relative roles of BD1 and BD2 due to the profound effects offered by iBET. Notwithstanding this, the recent disclosure that BD2-biased inhibitors such as RVX-297 (<b>9b</b>)<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> maintain a range of immune relevant efficacies in vitro and in vivo is noteworthy.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> In parallel to our own research,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> AbbVie recently published ABBV-744 (<b>12</b>),<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> a highly selective BD2 inhibitor which retains some of the antiproliferative activity of pan-BET inhibitors, albeit restricted largely to acute myeloid leukemia and certain prostate cancer cell lines. Furthermore, there have been a number of studies detailing the different roles in chromatin binding that the BD1 and BD2 domains have.<a onclick="showRef(event, 'ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56 ref57">(54â57)</a> The multivalent interaction between the BET family and chromatin or other binding partners is complex and context-dependent and remains poorly understood. It is difficult to predict what different efficacy or safety profiles might be found when domain-selective inhibitors are compared to pan-BET inhibitors. We hope that the publication of potent, highly selective BD1 and BD2 domain-selective BET inhibitors will enable advances in our biological understanding of the mechanism of these highly important and disease-relevant targets.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Published domain selective BET bromodomain inhibitors and their literature potencies/affinities to the BD1 and BD2 bromodomain of BRD4 with the acetylated lysine mimetic colored blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As part of our strategy to understand and profile more pan-BD2 inhibitors, a program of hit finding was undertaken targeting inhibitors of the second bromodomain ultimately aiming for a pIC<sub>50</sub> > 7 and selectivity over the first bromodomain of at least 100-fold. This approach was driven by the high degree of sequence homology in the acetylated lysine (KAc) binding region for the BD2 (second) domain across the BRD 2,3,4 and T proteins. This is in contrast to reduced homology between the first (BD1) and second domains for the same proteins (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf" class="ext-link">Figure S3aâc</a>); it was reasoned that pan-BD2 selectivity had a higher probability of success (versus single isoform selectivity). Accordingly, an HTS screen was initiated where the entire GSK compound collection (over 2 M compounds) was screened at single concentration versus the BRD4 dual domain truncated protein construct, considered as representative of the BET family. Here the BD1 domain was mutated (Y390A) to prevent binding of acetyl lysine mimetics at this site and allow evaluation of potency exclusively at the second domain. Following the single concentration screen (TR-FRET assay format), a triaging strategy, including an orthogonal BD2 biochemical assay (FP assay format), full curve follow up against BRD4 BD2 and BD1, and SPR biophysical characterization, led to real and robust hits. A focus on high ligand efficiency (LE<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a>/LLE<sub>AT</sub><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a>), a minimum of 3-fold selectivity over BD1 and CHROMlogD<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> < 4 led to the identification of <b>13a</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) as one of our preferred hits. Given the molecule contained only two aromatic rings and had a BRD4 BD2 derived LE of 0.35 and LLE<sub>AT</sub> of 0.34, this was an appealing start point, especially given the level of BRD4 BD2 selectivity was already >10-fold. While <b>13a</b> contained a potentially metabolically vulnerable methyl amide warhead, initial microsomal clearance in both rat and human was <1 mL/min/g tissue. The BD2 selectivity for BRD3 and BRDT was similar for <b>13a</b>, but the selectivity profile for BRD2 was somewhat lower and would therefore be monitored as we optimized this start point. The physicochemical profile was further improved by removal of the Br and F atoms to give <b>13b</b>, with no deleterious effect on potency and a large improvement in LLE<sub>AT</sub> (0.43). As part of generating an understanding of the key core functionality required for potency and selectivity, screening of the corresponding 2-pyridone <b>14a</b> led to 10-fold increase in potency for BRD4 BD2, albeit accompanied by a similar rise in the BRD4 BD1 potency. Notably, an equivalent potency and selectivity was observed against BRD2,3, T, suggestive that this series was capable of generating pan-BD2 inhibitors. With a BRD4 BD2 derived LE of 0.45 and LLE<sub>AT</sub> of 0.51, this represented a highly efficient start point to begin optimization. When the methyl group of the amide was replaced with an ethyl (<b>14b</b>), a reduction of almost 1 log in potency was observed, together with reduced BD2 selectivity, suggestive that the methyl group plays an important and specific role in the binding of this template.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure and Properties of BD2 HTS Hit <b>13a</b> and Optimized Fragments <b>13b</b>, <b>14a</b>, and <b>14b</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0010.gif" alt="" id="gr2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><b>13a</b></th><th class="colsep0 rowsep0" align="center"><b>13b</b></th><th class="colsep0 rowsep0" align="center"><b>14a</b></th><th class="colsep0 rowsep0" align="center"><b>14b</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2/BD1 pIC<sub>50</sub> (<i>n</i>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">5.1Â (6)/4.0 (1<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a>)</td><td class="colsep0 rowsep0" align="left">4.9Â (5)/<4.3Â (5)</td><td class="colsep0 rowsep0" align="left">5.9Â (9)/5.1Â (9)</td><td class="colsep0 rowsep0" align="left">5.2Â (4)/4.8Â (4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2 BD2/BD1 pIC<sub>50</sub> (<i>n</i>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">4.6 (6<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a>)/4.5 (2<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a>)</td><td class="colsep0 rowsep0" align="left">4.6 (8<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a>)/4.8 (2<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a>)</td><td class="colsep0 rowsep0" align="left">5.6Â (8)/4.8Â (8)</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3 BD2/BD1 pIC<sub>50</sub> (<i>n</i>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">5.0Â (6)/<4.3Â (6)</td><td class="colsep0 rowsep0" align="left">5.0Â (4)/4.8 (2<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a>)</td><td class="colsep0 rowsep0" align="left">6.3Â (4)/4.9Â (4)</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRDT BD2/BD1 pIC<sub>50</sub> (<i>n</i>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">4.9 (5<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a>)/4.4 (2<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a>)</td><td class="colsep0 rowsep0" align="left">4.7 (6<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a>)/4.5 (2<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a>)</td><td class="colsep0 rowsep0" align="left">5.7Â (6)/5.0Â (6)</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 selectivity (fold)</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">>4</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 LE/LLE<sub>AT</sub></td><td class="colsep0 rowsep0" align="left">0.35/0.34</td><td class="colsep0 rowsep0" align="left">0.37/0.43</td><td class="colsep0 rowsep0" align="left">0.45/0.51</td><td class="colsep0 rowsep0" align="left">0.37/0.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CHROMlogD (pH 7.4)/clogP</td><td class="colsep0 rowsep0" align="left">2.6/1.6</td><td class="colsep0 rowsep0" align="left">1.6/0.6</td><td class="colsep0 rowsep0" align="left">3.0/0.6</td><td class="colsep0 rowsep0" align="left">3.7/1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a> (Î¼g/mL)/AMP<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a> (nm/s)</td><td class="colsep0 rowsep0" align="left">â¥143/260</td><td class="colsep0 rowsep0" align="left">â¥108/58</td><td class="colsep0 rowsep0" align="left">78/595</td><td class="colsep0 rowsep0" align="left">â/790</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">microsomal Cli (rat, hu, mL/min/g tissue)</td><td class="colsep0 rowsep0" align="left">0.76/0.53</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">In all tables, the activity of the inhibitors has been determined using a TR-FRET assay, see Experimental Section for details. The average standard deviation for pIC<sub>50</sub> is 0.1.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 5).</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 2).</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 8).</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 6).</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1).</p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">CLND = chemiluminescent nitrogen detection.</p></div><div class="footnote" id="t1fn8"><sup><sup>h</sup></sup><p class="last">AMP = artificial membrane permeability.</p></div></div></div><div class="NLM_p">There is a wealth of structural information concerning the binding mode of a variety of pan-BET inhibitors to their target proteins.<a onclick="showRef(event, 'ref3 ref62 ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref3 ref62 ref63 ref64">(3,62â64)</a> Indeed, an X-ray crystal structure of <b>14a</b> in BRD2 BD2 was soon solved (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). revealing the key interactions. The pendant methyl amide serves as the KAc-mimetic, its carbonyl oxygen forming hydrogen-bonds to Asn429 and the conserved water network in an analogous manner to the carbonyl of the acetylated histone. The amide hydrogen bonds internally to the pyridone 2-carbonyl oxygen, while its methyl group fills the KAc-methyl pocket. Ligand-efficient fragments containing related aryl amides have been shown to bind relatively weakly to other bromodomains in a similar way, but so far, these have been limited to examples outside the BET family such as BAZ2B.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The benzyl group occupies the hydrophobic âWPFâ shelf, packing against Val435 and Trp370. It also contacts a histidine residue (His433) in an edge-to-face interaction. A similar interaction has been noted between His433 and the shelf group of other inhibitors.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In BD1, this histidine is replaced with an aspartic acid residue, and this âAsp/His switchâ and the differing interactions of the shelf aromatic ring as a result could account for the BD2-domain selectivity observed for <b>14a</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) X-ray structure of <b>14a</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB0">6ZB0</a>) showing key interactions with Asn429 and a network of five water molecules, the benzyl group occupies the âWPFâ shelf. (b) <b>14a</b> (green) overlaid with alternate tool <b>15</b> (magenta) in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SWP">6SWP</a><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a>), highlighting the additional vector and interaction with Asn429 utilized by <b>15</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the excellent efficiency of <b>14a</b>, the compound remained a relatively weak inhibitor, with a large increase in BD2 potency required. Among the published pan-BET inhibitors, almost all series gain potency by occupying three key regions of the BET binding site: a potent, often heteroaromatic warhead which serves as the KAc-mimetic, the WPF shelf, and/or the ZA channel. In this case, we believed we had achieved BD2 bias by starting from a relatively weak nonaromatic warhead and engaging the shelf of the protein. While the benzylic methylene offered a vector to enter the ZA channel, this did not lead to the required level of potency and selectivity (data not shown) and an alternative growth vector was sought.</div><div class="NLM_p">In parallel to our efforts on this series, a second HTS hit was being elaborated and SAR-driven optimization had led to the tool <b>15</b> (GSK046, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).<a onclick="showRef(event, 'ref53 ref66'); return false;" href="javascript:void(0);" class="ref ref53 ref66">(53,66)</a> The acetamide warhead of <b>15</b> and its âWPF shelfâ benzyl group bound in an analogous manner to <b>14a</b>, but <b>15</b> offered >1000-fold selectivity for BRD4 BD2 over BD1. The increase in potency and selectivity was believed to be due to the addition of a second amide substituent entering a cleft in the binding site formed by Leu383, Tyr428, and Asn429 (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). Additionally, the NH of the C1 amide substituent made a hydrogen bond with the carbonyl of Asn429 (which also interacts with the warhead carbonyl, vide supra). Comparing the X-ray structures of bound compounds <b>14a</b> with <b>15</b>, we considered that adding an amide group to the 5<i>-</i>position of the pyridone core may lead to a similar increase in potency and selectivity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure and BRD4 Potency and Selectivity of GSK046 (<b>15</b>)<a onclick="showRef(event, 'ref53 ref66'); return false;" href="javascript:void(0);" class="ref ref53 ref66">(53,66)</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0011.gif" alt="" id="gr4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">profile</th><th class="colsep0 rowsep0" align="center"><b>15</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 pIC<sub>50</sub> (<i>n</i>)</td><td class="colsep0 rowsep0" align="left">7.3Â (14)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1 pIC<sub>50</sub> (<i>n</i>)</td><td class="colsep0 rowsep0" align="left">4.2Â (3<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">selectivity (fold)</td><td class="colsep0 rowsep0" align="left">1400</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (10).</p></div></div></div><div class="NLM_p">The installation of an amide group at C5 of <b>14a</b> (cf. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) delivered potent and highly selective inhibitors while also improving the physicochemical properties of the series (i.e., lower CHROMlogD and higher CLND solubility vs <b>14a</b>). <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> shows that increasing the size of the amide from primary (<b>16</b>, R<sup>2</sup> = H) to a secondary methyl (<b>17</b>, R<sup>2</sup> = Me) and ethyl amide (<b>18</b>, R<sup>2</sup> = Et) offers a stepwise increase in both potency and selectivity compared to <b>13</b>. Compounds <b>16</b>â<b>18</b> maintain excellent binding efficiency, with similar LE values and an increase in LLE<sub>AT</sub> vs fragment <b>13</b>. Ethyl amide <b>18</b> in particular is close to 100 nM, with >100-fold selectivity and an LE of 0.41 and LLE<sub>AT</sub> of 0.52. These data validated both our hit finding approach of starting with a small and efficient fragment and of working on multiple templates in parallel during the hit-to-lead phase.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR for the Amide Vector (R<sup>2</sup>)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0012.gif" alt="" id="gr5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0013.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1).</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 2).</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 3).</p></div></div><div></div></div><div class="NLM_p">To assess the SAR at R<sup>2</sup>, a range of logical modifications were investigated. For example, branching of the ethyl amide to iPr (<b>19</b>) was less well tolerated, however, small cyclic alkyl groups proved optimal, with cPr and cBu (<b>20</b> and <b>21</b>) boasting selectivity levels of 200-fold. Larger rings were less well tolerated (e.g., THP <b>22</b>). This is in contrast to acetamide <b>15</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and represents divergent SAR between the series.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Searching for additional affinity, the effect of growing from the promising ethyl amide was also investigated; the installation of polarity from the Î²-carbon, such as for <b>23</b> and <b>24</b>, was poorly tolerated with a loss of >3-fold of potency relative to <b>18</b>. Indeed, branching from this carbon of the amide was also deleterious (<b>25</b>). Potency could be restored through Î³-branched examples <b>26</b> and <b>27</b>, although this appeared to offer no advantage to the smaller derivatives of similar potency already identified where both the LE and LLE<sub>AT</sub> values were significantly higher (cf. <b>18</b> and <b>20</b>). While a basic amine was not tolerated in example <b>28</b>, extending the carbon chain further did provide a BRD4 BD2 potency boost and led to the most selective compounds prepared yet (>300-fold over BD1). Piperidine <b>29</b>, expectedly, showed poor predicted permeability in an artificial membrane high throughput assay<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> by virtue of its high basicity, although this was somewhat attenuated in the less-basic morpholine <b>30</b>. Nevertheless, it was clear that the small and neutral ethyl and cyclopropyl amides <b>18</b> and <b>20</b> offered the best balance of potency, selectivity, and physicochemical properties. Indeed, cPr analogue <b>20</b> had a low clogP and CHROMlogD, was predicted to be permeable, and was soluble in our high throughput CLND kinetic solubility assay (conducted from DMSO solution).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div><div class="NLM_p"><b>20</b> was soaked into the BRD2 BD2 construct and bound as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a. In addition to the interactions of the KAc mimetic and shelf groups, already described for <b>14a</b>, the additional hydrogen bond between the amide NH and Asn429 that we had hoped to engage, was indeed observed. The cPr group shows excellent shape complementarity, with a narrow groove created by His433 and Pro430 on one side and Tyr428 and Leu383 on the other (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). This may explain why larger groups in this position are less well tolerated. The origin of BD2 selectivity arises from residue differences between BD1 and BD2 as highlighted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d. The key residue changes are His433 and Pro430 (BRD2 BD2 numbering) to Asp144 and Lys141 (BRD4 BD1 numbering). Both changes affect the shape of the groove that the cyclopropyl group occupies. As described earlier, in BD2 His433 makes a key edge-to-face interaction with the shelf benzyl group. In addition, the tight-packed arrangement of the cyclopropyl amide and His433 adds significantly to the surface area of this residue buried by <b>20</b>. The combination of all these positive interactions which are absent in BD1 are likely to drive the observed selectivity.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Key interactions of GSK620 (<b>20</b>) with BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>, cyan). (b) GSK620 shown with the protein surface representation of BRD2 BD2. (c) Key interactions of GSK620 (<b>20</b>) with BRD4 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>, white). (d) Overlay of GSK620 (<b>20</b>) in BRD2 BD2 (cyan) and BRD4 BD1 (white), providing a rationale for the domain selectivity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It is important to realize that this âAsp/His switchâ involves conformational differences as well as an amino acid exchange. In BRD2 BD2, His433 points toward the KAc site and the bound inhibitor <b>20</b>, whereas in BRD4 BD1, the equivalent Asp144 side chain points away (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,c). This is consistent with systematic differences found in deposited Protein Data Bank crystal structures, where the BET BD1 domain aspartic acids typically lie in <i>gauche</i>(+) chi1 rotameric states while the BD2 histidines are predominantly seen in the <i>trans</i> rotamer with minor <i>gauche</i>(+) components (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figure S3eâg">Figure S3eâg</a>). In this conformation, the imidazole of BRD4 His433 is able to accept a helix-capping hydrogen-bond from the Val435 backbone NH. In BRD4 BD1, the Asp144 side chain forms a conserved hydrogen bond with the free backbone NH of Lys141, perhaps explaining the prevalence of its <i>gauche</i>(+) conformation. The substitution of Lys141 for Pro430 in BRD2 BD2 makes this interaction impossible and may sterically restrict the loop, encouraging the histidine to lie in its <i>trans</i> conformation. The Asp144/Lys141 and His433/Pro430 combinations are conserved in all BET BD1 and BD2 domains (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figure S3d">Figure S3d</a>), so this rationalization of BD2 over BD1 selectivity extends across the family.</div><div class="NLM_p">The potency and physicochemical properties of amides <b>18</b> and <b>20</b> looked highly promising, and indeed the CLND solubility was toward the maximum limit of the high throughput assay for the vast majority of analogues. However, upon crystallization of cPr amide <b>20</b> [confirmed by X-ray powder diffraction (XRPD) analysis] and evaluation of its fasted-state simulated intestinal fluid (FaSSIF) solubility, a far more relevant end point for an orally targeted inhibitor, its solubility was only moderate (13â35 Î¼g/mL, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Given the attractive physicochemical space occupied by this inhibitor, it was likely this low solubility was being driven by its highly crystalline nature. To investigate this further, a small molecule crystal structure was determined for <b>20</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). The Me-amide group forms an intramolecular NâHÂ·Â·Â·O hydrogen bond with the pyridone oxygen atom, O14. The cPr-amide forms an intermolecular NâHÂ·Â·Â·O hydrogen bond with the oxygen atom of the Me-amide, O16. These intermolecular hydrogen bonds between the two pivotal amides connect molecules of <b>20</b> into columns (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b) that extend in the direction of the crystallographic <i>b</i>-axis. There are no Ï-stacking interactions in this structure. Given the amide groups of <b>20</b> make important hydrogen bonds to the BD2 domain and were therefore considered nonreplaceable, we interrogated the other proximal interactions within the small molecule crystal structure. It was noteworthy that the benzylic methylene resided in close proximity to the carbonyl oxygen of the cPr-amide in an adjacent molecule, while the 6-position of the pyridone core also sits close to the same CH of its neighbor (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c). We hypothesized that adding substituents to these positions may be sufficient to disrupt the packing and lead to increased solubility. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, this hypothesis proved correct and crystalline (by XRPD) single enantiomer <b>31</b> not only showed a markedly increased FaSSIF solubility of 815 Î¼g/mL but was well tolerated in terms of potency and selectivity. Interestingly, further additions to the cleft region offered no advantage to <b>20</b> and <b>31</b> in terms of potency or selectivity. Unfortunately, while crystalline (by XRPD) 6-methylpyridone <b>32</b> also had an excellent FaSSIF solubility, this substituent impacted binding at BD2, with a low BRD4 BD2 pIC<sub>50</sub> of 5.7 resulting, and was not progressed further.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Potency, Selectivity, and Solubilities of Methylated Derivatives of <b>20</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0014.gif" alt="" id="gr8" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><b>20</b></th><th class="colsep0 rowsep0" align="center"><b>31</b></th><th class="colsep0 rowsep0" align="center"><b>32</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2/BD1 pIC<sub>50</sub> (<i>n</i>)</td><td class="colsep0 rowsep0" align="left">7.1Â (16)/4.8Â (14)</td><td class="colsep0 rowsep0" align="left">7.1Â (4)/4.9Â (4)</td><td class="colsep0 rowsep0" align="left">5.7Â (5)/<4.3Â (4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 selectivity (fold)</td><td class="colsep0 rowsep0" align="left">220</td><td class="colsep0 rowsep0" align="left">170</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CHROMlogD (pHÂ 7.4)/clogP</td><td class="colsep0 rowsep0" align="left">3.1/0.1</td><td class="colsep0 rowsep0" align="left">3.5/0.4</td><td class="colsep0 rowsep0" align="left">2.9/0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (Î¼g/mL)/AMP<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (nm/s)</td><td class="colsep0 rowsep0" align="left">â¥154/200</td><td class="colsep0 rowsep0" align="left">â¥168/300</td><td class="colsep0 rowsep0" align="left">â¥126/110</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FaSSIF<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a> (Î¼g/mL)</td><td class="colsep0 rowsep0" align="left">13â35</td><td class="colsep0 rowsep0" align="left">815</td><td class="colsep0 rowsep0" align="left">577</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">CLND = chemiluminescent nitrogen detection solubility.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">AMP = artificial membrane permeability</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">FaSSIF = fasted state simulated intestinal fluid solubility.</p></div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) A view of a molecule of <b>20</b> from the crystal structure, showing the numbering scheme employed. Anisotropic atomic displacement ellipsoids for the non-hydrogen atoms are shown at the 50% probability level. Hydrogen atoms are displayed with an arbitrarily small radius. (b) Molecules of <b>20</b> forming part of a hydrogen bonded column within the crystal structure. Hydrogen bonds are displayed as dashed lines with the donors and acceptors labeled. (c) Packing plot of the crystal structure of <b>20</b> viewed down the crystallographic <i>b</i>-axis showing the close proximity of the benzylic methylene and 6-pyridone hydrogens to an adjacent molecule (circled). Hydrogen bonds are displayed as dashed lines with the donors and acceptors labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Further profiling of <b>20</b> and <b>31</b> will be discussed subsequently, but it was also important to evaluate the SAR of the shelf region of the template. Because of its key role in BD2 binding, further affinity may be possible in this region, while diversity could also be valuable for physicochemical properties and metabolic stability. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, the replacement of the phenyl ring of <b>20</b> with cyclohexyl <b>33</b> was tolerated. Given the edge-to-face interactions observed for <b>20</b>, this was somewhat surprising, but later rationalized by crystallography: the cyclohexyl ring adopts a different conformation to the phenyl group and packs nicely against the tryptophan. Unfortunately, capitalizing on this observation was problematic. Cyclohexyl <b>33</b> itself offered no benefit in terms of lipophilicity (CHROMlogD = 4 vs 3.1 for <b>20</b>) and attempts to insert polarity either onto or into this ring led to a significant reduction in potency (e.g., THPs <b>34</b>â<b>36</b>). It would have been similarly desirable to increase the polarity of the aromatic ring, but unfortunately pyridyls <b>37</b>â<b>39</b> showed a 10-fold reduction in potency, with no obvious means of obtaining this back elsewhere in the inhibitor. A range of electron withdrawing groups (EWG) and electron donating groups (EDG) were then prepared, but as <b>40</b>â<b>46</b> show, the potency and selectivity SARs were flat, with no compound offering an advantage in these terms over the unsubstituted lead <b>20</b>. The incorporation of a 3-CN group (<b>47</b>) proved detrimental, while in contrast, the addition of solubilizing groups such as secondary alcohol <b>48</b> or morpholine <b>49</b> were well tolerated and led to a large increase in FaSSIF solubility. Morpholine <b>49</b> had suboptimal in vitro clearance data in rat (14.7 mL/min/g tissue), however, alcohol <b>48</b> was worthy of further evaluation and will be discussed further alongside our other preferred inhibitors.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR for the âWPF Shelfâ Vector (R<sub>5</sub>)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0015.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0016.gif" alt="" id="gr12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0017.gif" alt="" id="GRAPHIC-d7e1604-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 3).</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 2).</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1).</p></div></div><div></div></div><div class="NLM_p">The addition of a <i>meta</i>-aniline (<b>50</b>) appeared to offer a small selectivity advantage over <b>20</b>. Hoping to capitalize on this observation and given that a free aniline was not desired from a genetic toxicity viewpoint,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> it was hypothesized that bicyclic motifs bearing an appropriate H-bond donor at this position were worthy of investigation. Cyclization of the <i>meta</i>- and <i>para</i>-positions to provide benzimidazole <b>51</b> showed a reduction in potency relative to aniline <b>50</b>, however cyclization of the <i>ortho</i>- and <i>meta</i>-positions looked more encouraging, with benzimidazole <b>52</b> equipotent to the aniline. Preparation of regioisomeric indazoles <b>53</b> and <b>54</b> was also tolerated, although in both cases selectivity was reduced relative to the benzimidazole. Benzofuran <b>55</b>, lacking any H-bond donor, was only moderately potent and selective. This was in contrast to indole <b>56</b>, which was not only 3-fold more potent at BRD4 BD2 than any other example tested but also exquisitely selective versus BD1 (>1000-fold). Interestingly, the same potency increase was not observed with regioisomeric indole <b>57</b>. A crystal structure in BRD2 BD2 for <b>56</b> was solved and provided a rationale for its high potency. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the indole group, as expected, still occupies the WPF shelf. However, the additional Ï-system of the pyrrole ring now appears to engage in a more optimal edge-to-face interaction with His433, which remains in the same place as observed for <b>20</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The NH of the indole also makes a through-water H-bond with Asp434. It is hypothesized that it is this combination of interactions which drives the additional BD2 potency of <b>56</b>.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray structure of <b>56</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB2">6ZB2</a>) showing: (a) the key interactions with His433, Asp434, and Trp370 on the WPF shelf. (b) The binding mode with a surface to highlight its optimal position on the WPF shelf.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the excellent potency of <b>56</b>, it was postulated that further polarity might be able to be incorporated in the Ph ring. 5-Aza-indole <b>58</b> was prepared, and while a 1 log drop-off in biochemical potency was observed (in accord with <i>o</i>-<b>37</b> vs <b>20</b>), <b>58</b> still had a reasonable potency/selectivity profile, given the highly polar nature of the compound. Finally, to understand the effect of the second Ï-system on the enhanced potency of indole <b>56</b>, the corresponding indoline <b>59</b> was prepared. In this instance, 3-fold potency was lost versus BD2, supporting the binding hypothesis in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, that the extended Ï-system, as well as the <i>meta</i>-hydrogen bond donor, has a role in the high potency of <b>56</b>. Nevertheless, given indoline <b>59</b> has one less aromatic ring, has a reduced CHROMlogD of 2.1 (vs 3.1 for <b>20</b> and 2.9 for <b>56</b>), and a BD2 selectivity of 415-fold and an enhanced solubility, this analogue was also worthy of further investigation.</div><div class="NLM_p">With the desired level of potency and selectivity in hand, and having controlled the physicochemical properties closely (lipophilicity, permeability, and solubility), a diverse set of optimized inhibitors remained. In search of an orally bioavailable molecule, a selection of lead compounds were evaluated for their pharmacokinetics in vivo. Before committing to such studies, all compounds were evaluated in vitro using isolated hepatocytes. As indicated in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>, the ethyl and cPr amides <b>18</b> and <b>20</b> have low measured in vitro Cl<sub>i</sub> in hepatocytes for rat, human, and dog (all <1 mg/mL/g tissue). This translated into low in vivo clearance for both compounds in the rat and, when dosed orally, excellent oral bioavailability. Given <b>20</b> had slightly superior selectivity, this example was also profiled in the dog, with a blood clearance (Cl<sub>b</sub>) of 2 mL/min/kg and an oral bioavailability (<i>F</i>%) of 57. It was also notable that both compounds had high free fractions in rat and dog, illustrating that unbound clearances are also low and high <i>free</i> blood concentrations in vivo are achievable.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> A number of the more selective biaryl shelf examples were also profiled. While indazole <b>53</b> had raised Cl<sub>b</sub> in vivo (in contrast to its in vitro rat profile) and poor oral bioavailability (likely driven by poor absorption), the highly selective indole <b>56</b> and indoline <b>59</b> both had low/moderate rat Cl<sub>b</sub> of 23 and 16 mL/min/kg, respectively. In the case of indoline <b>59</b>, this translated into an oral bioavailability of 53%. While the oral leg of the rat study for <b>56</b> was run in a different species (female Lewis) and so the bioavailability cannot be calculated, <b>56</b> also had good oral exposure as shown by an AUC/D of 23.6 minÂ·kg/L. Finally, two examples where increased FaSSIF solubility had been achieved were tested. Secondary alcohol <b>48</b> unfortunately showed high Cl<sub>b</sub> in rat in vivo, which correlated with raised in vitro clearance in hepatocytes in that species. Interestingly, this compound is predicted to have low clearance in dog and human, however, given the improved rat Cl<sub>b</sub> profile of other analogues, this was not tested further. Finally, the addition of the methyl group to the benzylic methylene to provide <b>31</b> unfortunately resulted in high rat Cl<sub>b</sub>. Overall, the excellent pharmacokinetics of <b>20</b>, together with the enhanced potency and selectivity of <b>56</b>, allowed us to select these as our two preferred BD2-selective and in vivo capable tools.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Pharmacokinetic Profile of Selected BD2 Inhibitors Following Intravenous Infusion and Oral Administration in the Male Wistar Han Rat and Beagle Dog</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center">hepatocyte CL<sub>int</sub> (mL/min/gÂ tissue)</th><th class="colsep0 rowsep0" align="center">dose iv (<i>n</i>)/po (<i>n</i>) (mg/kg)</th><th class="colsep0 rowsep0" align="center">CL<sub>b</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center"><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center"><i>t</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center"><i>F</i><sub>po</sub> (%)</th><th class="colsep0 rowsep0" align="center">fu<sub>b</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left">0.95</td><td class="colsep0 rowsep0" align="left">1Â (1)/3Â (3)</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">67</td><td class="colsep0 rowsep0" align="left">0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.45</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left"><1.26</td><td class="colsep0 rowsep0" align="left">0.5Â (1)/1Â (1)</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">0.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">â¯â¯â¯â¯â¯â¯â¯â¯</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left"><0.80</td><td class="colsep0 rowsep0" align="left">1Â (3)/3Â (3)</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">104</td><td class="colsep0 rowsep0" align="left">0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.87</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left"><1.26</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">â¯â¯â¯â¯â¯â¯â¯â¯</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left"><0.80</td><td class="colsep0 rowsep0" align="left">1.5Â (1)/3Â (3)</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.45</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">â¯â¯â¯â¯â¯â¯â¯â¯</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left">1.70</td><td class="colsep0 rowsep0" align="left">1Â (1)/10 (3<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a>)</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.45</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left"><1.26</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">â¯â¯â¯â¯â¯â¯â¯â¯</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left"><0.80</td><td class="colsep0 rowsep0" align="left">1Â (1)/3Â (3)</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">0.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.45</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left"><1.26</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">â¯â¯â¯â¯â¯â¯â¯â¯</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">1.4Â (1)/â</td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">0.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.45</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left"><1.26</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">â¯â¯â¯â¯â¯â¯â¯â¯</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left">1.50</td><td class="colsep0 rowsep0" align="left">1.4Â (1)/â</td><td class="colsep0 rowsep0" align="left">72</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.87</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>F</i>% cannot be calculated as po dosing carried out using alternate female Lewis rat strain, po AUC/D = 23.6 min.kg/L.</p></div></div></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Chemistry</h3><div class="NLM_p">To synthesize this series of analogues, a plan was envisaged which commenced from commercial pyridone <b>60</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). To vary the R<sup>2</sup> amide substituent, the R<sup>3</sup> = benzyl âshelfâ group can be installed at an early stage. Accordingly, selective <i>N</i>-benzylation and a subsequent two-step process converts the ester into the corresponding methyl amide <b>61</b> in 66% yield for the three steps. At this stage, two alternate routes were utilized in parallel. Bromide <b>61</b> can be carbonylated either in the presence of 2,4,6-trichlorophenyl formate using palladium acetate and Xantphos catalystâligand system.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The resultant activated ester <b>62</b> is formed in 87% yield and can be easily displaced by the desired amines in the presence of catalytic DMAP and triethylamine in THF at 45 Â°C to form analogues <b>16</b>â<b>17</b>, <b>19</b>, <b>21</b>, <b>25</b>â<b>27</b>, and <b>63</b>. Alternatively, carbonylation under similar palladium-catalyzed conditions (Pd(OAc)<sub>2</sub>, dppb), but using carbon monoxide and EtOH affords the ethyl ester.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> This transformation can be practically carried out on a small scale using a balloon of CO. However, we were able to scale this reaction using a Parr reactor vessel purged with CO to 50 psi and heated to 100 Â°C overnight. This approach delivered >20 g of this key intermediate. After sodium hydroxide-mediated hydrolysis, this provided carboxylic acid <b>64</b> in 75% yield over the two steps. While both processes were similarly effective, employing 2,4,6-trichlorophenyl formate, which itself had to be prepared in a single step, was less atom efficient than preparing the ethyl ester. Finally, carboxylic acid <b>64</b> could be converted to the desired amides <b>18</b>, <b>20</b>, <b>22</b>â<b>24</b>, and <b>65</b>â<b>66</b> using a HATU-mediated amide coupling in either DMF or DCM. For piperidine and morpholine containing analogues <b>63</b>, <b>65</b>, and <b>66</b> a subsequent Boc-deprotection step was required to afford the final free amines <b>28</b>â<b>30</b>.<named-content content-type="anchor" rid="sch2" type="simple"></named-content></div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0007.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Divergent Synthetic Routes for the Variance of R<sup>2</sup> and R<sup>3</sup><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditios: (i) NaH, DMF/THF (1:1) 0 Â°C, 30 min, then BnBr, 0 Â°C to rt, 2 h (89%); (ii) LiOH, THF/H<sub>2</sub>O/MeOH (1:1:1), rt, 2 h (78%); (iii) (COCl)<sub>2</sub>, DMF (0.5 mol %), DCM, rt, 2 h, then MeNH<sub>2</sub> (2 M in THF), Et<sub>3</sub>N, THF, 0 Â°C, 1.5 h (95%); (iv) Pd(OAc)<sub>2</sub> (10 mol %), dppb (10 mol %), Et<sub>3</sub>N, EtOH, DMSO, CO, 100 Â°C, 50 psi, 16 h (94%); (v) NaOH, MeOH/THF (1:1), rt, 2 h (80%); (vi) Pd(OAc)<sub>2</sub> (5 mol %), Xantphos (10 mol %), Et<sub>3</sub>N, toluene, 80 Â°C, 20 min, then 2,4,6-trichlorophenyl formate, 80 Â°C, 2.5 h (87%); (vii) R<sup>2</sup>NH<sub>2</sub>, HATU, DIPEA or Et<sub>3</sub>N, DMF or DCM, rt, 2 h; (viii) R<sup>2</sup>NH<sub>2</sub>, DMAP (0.2 equiv), Et<sub>3</sub>N, THF, 45 Â°C, 3 h; (ix) TFA, DCM, rt, 1 h; (x) MeNH<sub>2</sub> (2 M in THF), THF, reflux, 23 h (44%); (xi) K<sub>2</sub>CO<sub>3</sub>, R<sup>3</sup>CH<sub>2</sub>Br, MeOH or DMF, 65â90 Â°C; (xii) cPrNH<sub>2</sub>, Co<sub>2</sub>(CO)<sub>8</sub>, Pd(OAc)<sub>2</sub>, xantphos or cataXium A, DMAP, 1,4-dioxane, Î¼w 75â80 Â°C; (xiii) Cs<sub>2</sub>CO<sub>3</sub>, MeOH/THF (1:2), 70 Â°C, 1 h (41%).</p></p></figure><div class="NLM_p">For analogues where the R<sup>3</sup> shelf group needed to be varied, direct aminocarbonylation was utilized, which meant only a single step was required after the different R<sup>3</sup> diversity had been installed. As such, methyl ester <b>60</b> could be converted to the corresponding methyl amide <b>67</b> in a single step with an excess of methylamine in refluxing THF in 44% yield. A variety of R<sup>3</sup> groups (<b>68</b>â<b>81</b>) could then be incorporated via <i>N</i>-alkylation of the pyridone using potassium carbonate and heating in either DMF or MeOH. Finally, the key aminocarbonylation step directly installs the amide using dicobalt octacarbonyl as the CO source, in the presence of palladium acetate and either Xantphos or CataCXium A as the ligand.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> The reaction was normally conducted under microwave irradiation in the presence of DMAP, in 1,4-dioxane at 75â80 Â°C and delivers the final compounds (<b>33</b>â<b>36</b>, <b>40</b>â<b>44</b>, <b>46</b>â<b>47</b>, and <b>53</b>â<b>54</b>) in low to moderate yield. Indole <b>57</b> was prepared as its <i>N</i>-tosyl derivative <b>82</b>, and the final inhibitor was obtained by deprotection with cesium carbonate in MeOH/THF at 70 Â°C.</div><div class="NLM_p">Variations of this synthetic route were followed for additional analogues <b>37</b>â<b>39</b>, <b>48</b>â<b>52</b>, <b>55</b>â<b>56</b>, and <b>58</b>â<b>59</b> as detailed in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The key advantage to this alternative approach was the use of pyridone <i>O</i>-alkylation as a protecting group strategy to allow preparation of advanced amides <b>86</b>â<b>88</b>, allowing for a late stage introduction of the R<sup>3</sup> âshelfâ substituent either by Mitsunobu or alkylation chemistry. Overall, this late stage diversification was attractive for analogue synthesis, especially where the shelf substituents required bespoke preparation. Additional late stage functional group interconversions were required to afford alcohol <b>48</b> and morpholine <b>49</b> as detailed in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. While this route was excellent for R<sup>3</sup> diversification, it is less step efficient than the <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> route for scale up.</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Alternative Routes for the Preparation of Pyridone Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) BnBr, Ag<sub>2</sub>CO<sub>3</sub>, CHCl<sub>3</sub>, reflux, 16 h (67%); (ii) LiOH, THF/MeOH (1:1), rt, 2 h (94%); (iii) (COCl)<sub>2</sub>, DMF (0.5 mol %), DCM, rt, 2 h, then MeNH<sub>2</sub> (2 M in THF), THF or 2-MeTHF, rt, 6 h (54â95%); (iv) Pd(OAc)<sub>2</sub> (5 mol %), Xantphos (10 mol %), Et<sub>3</sub>N, toluene, 80 Â°C, 20 min, then 2,4,6-trichlorophenyl formate, 80 Â°C, 2â18 h (57%); (v) R<sup>2</sup>NH<sub>2</sub>, Et<sub>3</sub>N, DMAP (5 mol %), THF, 45 Â°C, 45 min â16 h (85â86%); (vi) TFA, 80â90 Â°C, 1 h (83â99%); (vii) CMTP, toluene, Î¼w 120 Â°C, or DIAD, PPh<sub>3</sub>, toluene, rt, 3â24 h; (viii) K<sub>2</sub>CO<sub>3</sub>, THF or DMF, rt â50 Â°C; (ix) MeMgBr (3 M in Et<sub>2</sub>O), THF, â78 Â°C, 45 min (43%); (x) chiral separation (Chiralpak AD-H); (xi) DMP, DCM, 0 Â°C â rt, 48 h (quant); (xii) morpholine, Et<sub>3</sub>N, DCM, rt, 45 min, then STAB, rt 24 h (25%); (xiii) TFA, DCM, rt, 30â90 min (80%) or HCl, IPA, rt, 3 days (48%); (xiv) Cs<sub>2</sub>CO<sub>3</sub>, MeOH/THF (1:2), 70 Â°C, 1 h (68%); (xv) Pd(OAc)<sub>2</sub> (5 mol %), Xantphos (6 mol %), Et<sub>3</sub>N, MeOH, DMF, 60 Â°C, 21 h (60%); (xvi) NaI, MeCN, rt, 16 h (25%); (xvii) K<sub>2</sub>CO<sub>3</sub>, DMF, 90 Â°C, 1 h (67%); (xviii) LiOH, 1,4-dioxane/water (1:1), rt, 30 min (90%); xix. R<sup>2</sup>NH<sub>2</sub>, HATU, DIPEA, DMF, rt, 7 h (quant).</p></p></figure><div class="NLM_p">Finally, separate synthetic routes were utilized to prepare 6-methylpyridone <b>32</b> and benzylic-methyl derivative <b>31</b>. Their preparation is outlined in <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a>. 6-Methylpyridone <b>32</b> was prepared from known intermediate <b>94</b><a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> in four steps using similar chemistry to that already discussed. On the other hand, pyran <b>96</b> was used as a novel start point for the preparation of pyridone <b>31</b> and allowed the commercial (<i>R</i>)-1-phenylethanamine to be used as the source of the chiral methyl group. Furthermore, selective bromination of the resultant pyridone <b>97</b> provided the functional handle to allow access to the carbonylation chemistry already discussed.</div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0008.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of 6-Methylpyridone <b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) K<sub>2</sub>CO<sub>3</sub>, BnBr, DMF, 40 Â°C, 4 h (35%); (ii) MeNH<sub>2</sub> (2 M in THF), MeOH, 50 Â°C, 30 min (82%); (iii) LiOH, 1,4-dioxane/water (1:1), rt, 1 h 45 min (99%); (iv) cPrNH<sub>2</sub>, HATU, DIPEA, DMF, rt, 1 h (74%).</p></p></figure><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0009.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Homochiral Preparation of Pyridone <b>31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (<i>R</i>)-1-phenylethanamine, DMF/THF (1:4), rt, 30 min, then EDC, DMAP (6 mol %), rt, 64 h (75%); (ii) NBS, 2-MeTHF, rt, 15 min; (iii) MeNH<sub>2</sub> (2 M in THF), MeOH, 50 Â°C, 90 min (78%, 2 steps); (iv) Pd(OAc)<sub>2</sub> (10 mol %), Xantphos (7 mol %), Et<sub>3</sub>N, MeOH, DMF, 60 Â°C, 4 h (60%); (v) NaOH, MeOH/H<sub>2</sub>O, rt, 16 h (90%); (vi) cPrNH<sub>2</sub>, HATU, DIPEA, DMF, rt, 1 h (89%).</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we have described the lead optimization of a weak fragment-like hit into a range of potent pan-BD2 selective inhibitors, possessing excellent physicochemical properties and with selectivities from 200- to >1000-fold. Analogues <b>20</b> (GSK620) and <b>56</b> (GSK549) were selected for further profiling, and as shown in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>, both compounds show agreement in their selectivity for all BET family members (BRD2, BRD3, BRD4, and BRDT BD2 vs BD1), with indole <b>56</b> maintaining its extremely high selectivity against BD1 for the other family members. This illustrates the high sequence homology between the BD2 domains of these four proteins. Both <b>20</b> and <b>56</b> were evaluated using a BET surface plasmon resonance (SPR) assay.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> Here, the same mutant protein constructs are used as for the FRET assay, but binding affinity can be assessed. Pleasingly, both compounds were similarly potent and selective as observed in the FRET assay. The selectivity of <b>56</b> was also evaluated against the wider bromodomain family using the DiscoverX BromoScan screening platform. As illustrated by <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf" class="ext-link">Table S12</a>, <b>20</b> is highly selective for the BET-BD2 family of proteins, with >200-fold selectivity over all other bromodomains. Both <b>20</b> and <b>56</b> have also been profiled against a range of known liability targets as shown in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf" class="ext-link">Table S13</a>, and both show minimal activity against all targets tested. <b>20</b> is also Ames<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> and mouse lymphoma assay (MLA)<a onclick="showRef(event, 'ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref77 ref78">(77,78)</a> negative, while both compounds showed low levels of inhibition against indicators of hepatoxicity such as the bile salt export pump (BSEP) assay (pIC<sub>50</sub> < 3.7 for <b>20</b> and 4.3 for <b>56</b>).<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> They were also negative in the glutathione (GSH) trapping assay,<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> and both showed no toxicity in the cell health evaluation of hepG2 cells against three end points. Both <b>20</b> and <b>56</b> do not have a CYP3A4 metabolism-dependent inhibition (MDI) or direct inhibition liability as tested in house, and for <b>20</b>, this was confirmed in an external time-dependent inhibition (TDI) assay at high concentration.<a onclick="showRef(event, 'ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref81 ref82">(81,82)</a> These data are particularly encouraging for the unsubstituted indole, as this is often implicated in toxicities caused by reactive metabolites,<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> although at least in our in vitro screens, this has not been observed to date.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Profile of Lead Compounds <b>20</b> (GSK620) and <b>56</b> (GSK549)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0018.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 8).</p></div><div class="footnote" id="tbl7-fn1"><sup><sup>b</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 10).</p></div><div class="footnote" id="tbl7-fn2"><sup><sup>c</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 9).</p></div><div class="footnote" id="tbl7-fn3"><sup><sup>d</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 6).</p></div><div class="footnote" id="tbl7-fn4"><sup><sup>e</sup></sup><p class="last">Also tested 5.5 (<i>n</i> = 1).</p></div><div class="footnote" id="tbl7-fn5"><sup><sup>f</sup></sup><p class="last">Also tested <3.7 (<i>n</i> = 1).</p></div></div><div></div></div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0006.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. BromoScan profile of <b>20</b> (GSK620).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Overall, both <b>20</b> and <b>56</b> are potent and selective pan-BD2 inhibitors with excellent in vivo PK properties and excellent developability properties, with the exception of moderate FaSSIF solubility driven by their highly crystalline nature. Importantly, both of these tools also show a biological phenotype in human whole blood as shown in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. Here, human blood samples are stimulated with lipopolysaccharide (LPS), which produces a strong immune response. In this assay, the monocyte chemattractant protein 1 (MCP-1/CCL2) is measured. This is a chemokine which recruits monocytes, memory T cells, and dendritic cells to sites of inflammation.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Herein, we show that highly BD2 selective compounds <b>20</b> and <b>56</b> are capable of reducing the MCP-1 response in a concentration-dependent manner with (an expected) â¼1 log drop off in potency relative to the biochemical BRD4 BD2 potencies observed, providing strong evidence that <b>20</b> and <b>56</b> are indeed engaging BD2 in cells. Indeed, these data in concert with the differing selectivity profiles of the two compounds and their excellent pharmacokinetics make them excellent tools for in vitro and in vivo evaluation of the phenotype associated with pan-BD2 inhibition. A detailed comparison of the functional contribution of the BD1 and BD2 bromodomains has recently been disclosed.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> This compared the efficacy of <b>20</b> alongside structurally differentiated BD2 inhibitor <b>15</b> (GSK046) and BD1 selective tool, <b>7</b>.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Highlighting the utility of <b>20</b> as an in vivo tool, efficacy was observed in separate models of inflammatory arthritis, psoriasis, and hepatitis. Together these data demonstrate that BD2 inhibitors are effective in models of inflammatory and autoimmune disease, opening up an exciting possible new therapeutic approach against these pathologies.</div></div><div id="sec3_1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">General Experimental</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04257" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04257" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Unless otherwise stated, all reactions were carried out under an atmosphere of nitrogen in heat- or oven-dried glassware and anhydrous solvent. Solvents and reagents were purchased from commercial suppliers and used as received. Reactions were monitored by thin layer chromatography (TLC) or liquid chromatographyâmass spectrometry (LCMS). TLC was carried out on glass or aluminum-backed 60 silica plates coated with UV<sub>254</sub> fluorescent indicator. Spots were visualized using UV light (254 or 365 nm) or alkaline KMnO<sub>4</sub> solution, followed by gentle heating. LCMS analysis was carried out on a Waters Acquity UPLC instrument equipped with a CSH C18 column (50 mm Ã 2.1 mm, 1.7 Î¼m packing diameter) and Waters micromass ZQ MS using alternate-scan positive and negative electrospray. Analytes were detected as a summed UV wavelength of 210â350 nm. Three liquid phase methods were used: <i>formic</i>, 40 Â°C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) H<sub>2</sub>O containing 0.1% volume/volume (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. <i>High pH</i>, 40 Â°C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia and (B) acetonitrile. <i>TFA</i>: 40 Â°C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) 0.1% v/v aqueous TFA solution and (B) 0.1% v/v TFA solution in acetonitrile. Flash column chromatography was carried out using Biotage SP4 or Isolera One apparatus with SNAP silica cartridges. Mass directed automatic purification (MDAP) was carried out using a Waters ZQ MS using alternate-scan positive and negative electrospray and a summed UV wavelength of 210â350 nm. Two liquid phase methods were used: <i>formic</i>, Sunfire C18 column (100 mm Ã 19 mm, 5 Î¼m packing diameter, 20 mL/min flow rate) or Sunfire C18 column (150 mm Ã 30 mm, 5 Î¼m packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases as (A) H<sub>2</sub>O containing 0.1% volume/volume (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. <i>High pH</i>: Xbridge C18 column (100 mm Ã 19 mm, 5 Î¼m packing diameter, 20 mL/min flow rate) or Xbridge C18 column (150 mm Ã 30 mm, 5 Î¼m packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases as (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia and (B) acetonitrile. NMR spectra were recorded at ambient temperature (unless otherwise stated) using standard pulse methods on any of the following spectrometers and signal frequencies: Bruker AV-400 (<sup>1</sup>H = 400 MHz, <sup>13</sup>C = 101 MHz), Bruker AV-600 (<sup>1</sup>H = 600 MHz, <sup>13</sup>C = 150 MHz), or Bruker AV4 700 MHz spectrometer (<sup>1</sup>H = 700 MHz, <sup>13</sup>C = 176 MHz). Chemical shifts are referenced to trimethylsilane (TMS) or the residual solvent peak and are reported in ppm. Coupling constants are quoted to the nearest 0.1 Hz, and multiplicities are given by the following abbreviations and combinations thereof: s (singlet), Î´ (doublet), t (triplet), q (quartet), quin (quintet), sxt (sextet), m (multiplet), br (broad). Liquid chromatography high resolution mass spectra (HRMS) were recorded on a Waters XEVO G2-XS Q-Tof mass spectrometer with positive electrospray ionization mode over a scan range 100â200 AMU, with analytes separated on an Acquity UPLC CSH C18 column (100 mm Ã 2.1 mm, 1.7 Î¼m packing diameter) at 50 Â°C. Purity of synthesized compounds was determined by LCMS analysis. All compounds for biological testing were >95% pure.</div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthetic Procedures</h3><div id="sec3_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>16</b>)</h4><div id="sec3_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Step 1: Methyl 1-Benzyl-5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylate (<b>99</b>)</h5><div class="NLM_p last">Sodium hydride (5.17 g, 60% dispersion in mineral oil, 129 mmol) was added to a solution of methyl 5-bromo-2-oxo-1,2-dihydro-3-pyridinecarboxylate (<b>60</b>, 25 g, 108 mmol) in DMF (200 mL) and THF (200 mL) at 0 Â°C, and the mixture was stirred for 30 min, giving a dense suspension. Benzyl bromide (14.10 mL, 119 mmol) was added, and the mixture stirred for a further 2 h, allowing to warm to rt, then the resulting clear brown solution was added to water (400 mL) and extracted with EtOAc (2 Ã 300 mL). The combined organics were washed with water (2 Ã 200 mL), dried, and evaporated in vacuo to give methyl 1-benzyl-5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylate (<b>99</b>, 31 g, 96 mmol, 89% yield) as a beige solid. This material was carried through to the next step without purification. LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.98 min, [MH]+ = 322.0 and 324.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.16 (d, <i>J</i> = 2.9 Hz, 1 H) 7.62 (d, <i>J</i> = 2.9 Hz, 1 H) 7.30â7.43 (m, 5 H) 5.15 (s, 2 H) 3.92 (s, 3 H).</div></div><div id="sec3_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Step 2: 1-Benzyl-5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylic Acid (<b>100</b>)</h5><div class="NLM_p last">Lithium hydroxide (6.91 g, 289 mmol) in water (200 mL) was added to a mixture of methyl 1-benzyl-5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylate (<b>99</b>, 31 g, 96 mmol), THF (200 mL), and methanol (200 mL), and the mixture was stirred at rt for 2 h, then evaporated in vacuo to about half volume, giving a dense suspension. This was diluted with water (200 mL) and acidified with acetic acid to pH 5, then extracted with EtOAc (2 Ã 300 mL). The combined organics were dried over sodium sulfate and evaporated in vacuo to give an off-white solid. The product was suspended in ether (200 mL), sonicated, diluted with cyclohexane (100 mL), and collected by filtration to give 1-benzyl-5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylic acid (<b>100</b>, 23 g, 74.6 mmol, 78% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.01 min, [MH]+ = 308.0 and 310.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 14.02 (br s, 1 H) 8.55 (d, <i>J</i> = 2.7 Hz, 1 H) 7.73 (d, <i>J</i> = 2.7 Hz, 1 H) 7.40â7.47 (m, 3 H) 7.31â7.37 (m, 2 H) 5.25 (s, 2 H).</div></div><div id="sec3_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Step 3: 1-Benzyl-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>61</b>)</h5><div class="NLM_p last">1-Benzyl-5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylic acid (<b>100</b>, 28 g, 91 mmol) was suspended in DCM (300 mL), and oxalyl chloride (23.86 mL, 273 mmol) and DMF (0.352 mL, 4.54 mmol) were added, then the mixture was stirred for 2 h at rt. The solvent was evaporated in vacuo to give a brown residue, which was then dissolved in THF (300 mL) and Et<sub>3</sub>N (12.67 mL, 91 mmol) was added. The mixture was cooled in an ice bath, then methanamine (91 mL, 2 M in THF, 182 mmol) was added dropwise over 30 min and the mixture stirred for a further 1 h at 0 Â°C. The solvent was evaporated in vacuo, and the solid residue was partitioned between water (300 mL) and DCM (300 mL), and the organic layer was washed with brine, dried, and evaporated in vacuo to give 1-benzyl-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>61</b>, 27.6 g, 86 mmol, 95% yield) as a brown solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.97 min, [MH]+ = 321.0 and 323.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.57 (br s, 1 H) 8.60 (d, <i>J</i> = 2.9 Hz, 1 H) 7.62 (d, <i>J</i> = 2.9 Hz, 1 H) 7.34â7.48 (m, 3 H) 7.29â7.33 (m, 2 H) 5.20 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Step 4: 2,4,6-Trichlorophenyl 1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>62</b>)</h5><div class="NLM_p last">1-Benzyl-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>61</b>, 2 g, 6.23 mmol), Xantphos (0.360 g, 0.623 mmol), palladium acetate (0.070 g, 0.311 mmol), and Et<sub>3</sub>N (1.302 mL, 9.34 mmol) were combined in a three-necked flask equipped with a dropping funnel and a condenser with a nitrogen bubbler on the top. Toluene (30 mL) was added, and the mixture was heated at 80 Â°C for 20 min, then a solution of 2,4,6-trichlorophenyl formate (2.106 g, 9.34 mmol) in toluene (20 mL) was added dropwise over 30 min and heating continued for 2 h. The reaction mixture was diluted with EtOAc (50 mL) and washed with water (50 mL) and brine (50 mL), dried, and evaporated in vacuo to give an orange oil. This was dissolved in DCM (10 mL) and loaded onto a 50 g silica column, then eluted with 0â50% EtOAc/cyclohexane and the product-containing fractions evaporated in vacuo to give 2,4,6-trichlorophenyl 1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>62</b>, 2.52 g, 5.41 mmol, 87% yield) as a beige solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.36 min, [MH]+ = 465, 467. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.32 (br. d, <i>J</i> = 4.4 Hz, 1 H) 9.20 (d, <i>J</i> = 2.7 Hz, 1 H) 8.58 (d, <i>J</i> = 2.7 Hz, 1 H) 7.30â7.50 (m, 7 H) 5.29 (s, 2 H) 3.01 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_2_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Step 5: 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>16</b>)</h5><div class="NLM_p last">Prepared from <b>62</b> (1 g, 2.15 mmol) and ammonia (21.47 mL, 10.74 mmol, 0.5 M in 1,4-dioxane) using the same procedure as for compound <b>21</b>, to give 1-benzyl-<i>N</i><sup><i>3</i></sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (562 mg, 1.87 mmol, 87% yield) as a white solid. LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.74 min, [MH]<sup>+</sup> = 286.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.36 (br q, <i>J</i> = 4.4 Hz, 1 H) 8.82 (d, <i>J</i> = 2.5 Hz, 1 H) 8.78 (d, <i>J</i> = 2.9 Hz, 1 H) 7.88â8.19 (m, 1 H) 7.12â7.57 (m, 6 H) 5.29 (s, 2 H) 2.83 (d, <i>J</i> = 4.4 Hz, 3 H).</div></div></div><div id="sec3_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-Benzyl-<i>N</i><sup>3</sup>,<i>N</i><sup>5</sup>-dimethyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>17</b>)</h4><div class="NLM_p last">Prepared from <b>62</b> (40 mg, 0.086 mmol) and methanamine (2 M in THF, 0.09 mL, 0.180 mmol) using the same procedure as for <b>21</b>, to give 1-benzyl-<i>N</i><sup>3</sup>,<i>N</i><sup>5</sup>-dimethyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (16 mg, 0.048 mmol, 56% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.75 min, [MH]<sup>+</sup> = 300.0. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.85 (d, <i>J</i> = 2.9 Hz, 1 H) 8.58 (d, <i>J</i> = 2.9 Hz, 1 H) 7.26â7.44 (m, 5 H) 5.34 (s, 2 H) 2.97 (s, 3 H) 2.90 (s, 3 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1-Benzyl-<i>N</i><sup>5</sup>-ethyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>18</b>)</h4><div id="sec3_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Step 1: Ethyl 1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>101</b>)</h5><div class="NLM_p last">1-Benzyl-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>61</b>, 23 g, 71.6 mmol), DMSO (60 mL), ethanol (70 g, 1519 mmol), Et<sub>3</sub>N (19.96 mL, 143 mmol), dppb (3.05 g, 7.16 mmol), and palladium acetate (1.608 g, 7.16 mmol) were placed in a steel Parr vessel, which was then purged with carbon monoxide by filling to 50 psi, then releasing the pressure, then refilled to 50 psi and heated overnight at 100 Â°C. The mixture was diluted with water (200 mL) and extracted with EtOAc (2 Ã 300 mL), the organic layer washed with water (2 Ã 300 mL), then dried and evaporated in vacuo, and the residue was triturated with ether (200 mL) and the solid collected by filtration to give ethyl 1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>101</b>, 21.2 g, 67.4 mmol, 94% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.99 min, [MH]+ = 315.2. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.37 (br s, 1 H) 9.03 (d, <i>J</i> = 2.4 Hz, 1 H) 8.38 (d, <i>J</i> = 2.7 Hz, 1 H) 7.34â7.42 (m, 3 H) 7.28â7.34 (m, 2 H) 5.25 (s, 2 H) 4.35 (q, <i>J</i> = 7.1 Hz, 2 H) 2.99 (d, <i>J</i> = 4.9 Hz, 3 H) 1.37 (t, <i>J</i> = 7.2 Hz, 3 H).</div></div><div id="sec3_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Step 2: 1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic Acid (<b>64</b>)</h5><div class="NLM_p last">Sodium hydroxide (99 mL, 199 mmol, 2 M aq) was added to a solution of ethyl 1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>101</b>, 20.8 g, 66.2 mmol) in a mixture of methanol (100 mL) and THF (100 mL), and the resulting solution was stirred for 2 h at rt, then evaporated in vacuo to approximately 100 mL volume. The mixture was diluted with water (200 mL), then filtered to remove a dark-gray solid, the filtrate was washed with MTBE (200 mL), then acidified to pH 4 with 2 M HCl and the resulting suspension stirred for 2 h, then filtered and the product washed with water, then dried in the vacuum oven to give 1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic acid (<b>64</b>, 15.2 g, 53.1 mmol, 80% yield). LCMS (2 min High pH): <i>R</i><sub><i>t</i></sub> = 0.58 min, [MH]+ = 287.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 13.19 (br s, 1 H) 9.14â9.34 (m, 1 H) 8.88 (d, <i>J</i> = 2.7 Hz, 1 H) 8.70 (d, <i>J</i> = 2.7 Hz, 1 H) 7.25â7.42 (m, 5 H) 5.33 (s, 2 H) 2.82 (d, <i>J</i> = 4.6 Hz, 3 H).</div></div><div id="sec3_2_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Step 3: 1-Benzyl-<i>N</i><sup>5</sup>-ethyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>18</b>)</h5><div class="NLM_p last">Prepared from <b>64</b> (250 mg, 0.873 mmol) and ethanamine (2 M in THF, 0.873 mL, 1.747 mmol) using the same procedure as for <b>20</b>, to give 1-benzyl-<i>N</i><sup>5</sup>-ethyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (242 mg, 0.772 mmol, 88% yield) as a cream solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.80 min, [MH]<sup>+</sup> = 314.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.31â9.43 (m, 1 H) 8.82 (d, <i>J</i> = 2.7 Hz, 1 H) 8.73 (d, <i>J</i> = 2.7 Hz, 1 H) 8.56 (br. t, <i>J</i> = 5.3, 5.3 Hz, 1 H) 7.27â7.40 (m, 5 H) 5.30 (s, 2 H) 3.22â3.28 (m, 2 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 1.11 (t, <i>J</i> = 7.2 Hz, 3 H).</div></div></div><div id="sec3_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-Benzyl-<i>N</i><sup>5</sup>-isopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>19</b>)</h4><div class="NLM_p last">Prepared from <b>62</b> (50 mg, 0.107 mmol) and propan-2-amine (0.018 mL, 0.215 mmol), using the same procedure as for compound <b>21</b>, to give 1-benzyl-<i>N</i><sup>5</sup>-isopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (32 mg, 0.083 mmol, 77% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.92 min, [MH]<sup>+</sup> = 328.0. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 9.67â9.94 (m, 1 H) 8.87 (d, <i>J</i> = 2.4 Hz, 1 H) 8.58 (d, <i>J</i> = 2.4 Hz, 1 H) 7.28â7.44 (m, 5 H) 5.33 (s, 2 H) 4.17 (quin, <i>J</i> = 6.4 Hz, 1 H) 2.92â3.01 (m, 3 H) 1.25 (d, <i>J</i> = 6.8 Hz, 6 H). <i>One exchangeable proton not observed.</i></div></div><div id="sec3_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 1-Benzyl-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>20</b>)</h4><div class="NLM_p last">1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic acid (<b>64</b>, 5.5 g, 19.21 mmol) was suspended in DCM (100 mL), and Et<sub>3</sub>N (3.21 mL, 23.05 mmol), HATU (9.50 g, 24.98 mmol), and cyclopropylamine (1.625 mL, 23.05 mmol) were added, then the mixture was stirred for 2 h at rt. The mixture was washed with water (100 mL), 0.5 M HCl (100 mL), and saturated sodium bicarbonate solution (100 mL), and the organic layer was dried and evaporated in vacuo to give a pale-yellow solid. The solid was suspended in ether (20 mL) and sonicated, then filtered and the solid dried in the vacuum oven to give 1-benzyl-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (5.25 g, 16.14 mmol, 84% yield) as a colorless solid. The product was combined with another batch, prepared by a similar method (2.4 g), the combined material was dissolved by refluxing in ethanol (200 mL) for 20 min, then Silicycle thiourea palladium scavenging resin (10 g) was added and the mixture heated for a further 30 min. The mixture was filtered into a Buchner flask and allowed to cool to rt over 1 h, then cooled in an ice bath for 1 h and the resulting solid collected by filtration and dried in the vacuum oven to give 1-benzyl-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (6.76 g, 20.78 mmol, 66% yield) as a colorless solid. LCMS (2 min high pH): <i>R</i><sub><i>t</i></sub> = 0.84 min, [MH]<sup>+</sup> = 326.3. HRMS (C<sub>18</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>): [M + H]<sup>+</sup> calculated 326.1499, found 326.1499. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.35 (d, <i>J</i> = 4.9 Hz, 1 H) 8.80 (d, <i>J</i> = 2.7 Hz, 1 H) 8.72 (d, <i>J</i> = 2.7 Hz, 1 H) 8.54 (d, <i>J</i> = 3.9 Hz, 1 H) 7.25â7.42 (m, 5 H) 5.29 (s, 2 H) 2.75â2.89 (m, 4 H) 0.65â0.72 (m, 2 H) 0.53â0.59 (m, 2 H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 163.9, 163.1, 161.1, 144.0, 140.8, 136.2, 128.6 (2C), 127.7, 127.7 (2C), 119.3, 113.5, 52.8, 25.8, 22.9, 5.5 (2C).</div></div><div id="sec3_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-Benzyl-<i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>21</b>)</h4><div class="NLM_p last">2,4,6-Trichlorophenyl 1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>62</b>, 401 mg, 0.861 mmol), cyclobutanamine (0.15 mL, 1.757 mmol), DMAP (23 mg, 0.188 mmol), triethylamine (0.48 mL, 3.44 mmol), and THF (8 mL) were stirred at 45 Â°C under N<sub>2</sub> for 3 h. The reaction mixture was concentrated to give 600 mg of an off-white solid, which was purified by chromatography on SiO<sub>2</sub> (Biotage SNAP 50 g cartridge, eluting with 0â100% ethyl acetate/cyclohexane). The desired fractions were concentrated to give 1-benzyl-<i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (295 mg, 0.782 mmol, 91% yield) as an off-white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.91 min, [MH]<sup>+</sup> = 340.0. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.88 (d, <i>J</i> = 2.9 Hz, 1 H) 8.59 (d, <i>J</i> = 2.9 Hz, 1 H) 7.29â7.43 (m, 5 H) 5.33 (s, 2 H) 4.46 (quin, <i>J</i> = 8.3 Hz, 1 H) 2.97 (s, 3 H) 2.22â2.47 (m, 2 H) 1.95â2.22 (m, 2 H) 1.62â1.91 (m, 2 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(tetrahydro-2<i>H</i>-pyran-4-yl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>22</b>)</h4><div class="NLM_p last">Prepared from <b>64</b> (49.5 mg, 0.173 mmol) and tetrahydro-2<i>H</i>-pyran-4-amine, hydrochloride (50.2 mg, 0.365 mmol) using the same procedure as for <b>20</b> to give 1-benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(tetrahydro-2<i>H</i>-pyran-4-yl)-1,2-dihydropyridine-3,5-dicarboxamide (58.8 mg, 0.159 mmol, 92% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.81 min, [MH]<sup>+</sup> = 370.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.32â9.42 (m, 1 H) 8.85 (d, <i>J</i> = 2.7 Hz, 1 H) 8.75 (d, <i>J</i> = 2.7 Hz, 1 H) 8.43 (d, <i>J</i> = 7.6 Hz, 1 H) 7.26â7.40 (m, 5 H) 5.30 (s, 2 H) 3.92â4.05 (m, 1 H) 3.87 (br. dd, <i>J</i> = 11.9, 2.1 Hz, 2 H) 3.37 (td, <i>J</i> = 11.7, 1.8 Hz, 2 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 1.75 (br. dd, <i>J</i> = 12.5, 2.2 Hz, 2 H) 1.56 (qd, <i>J</i> = 12.0, 4.4 Hz, 2 H).</div></div><div id="sec3_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1-Benzyl-<i>N</i><sup>5</sup>-(2-methoxyethyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>23</b>)</h4><div class="NLM_p last">Prepared from <b>64</b> (50 mg, 0.175 mmol) and 2-methoxyethanamine (0.015 mL, 0.175 mmol) using the same procedure as <b>20</b> to give the title compound (46 mg, 0.135 mmol, 77%). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.78 min, [MH]<sup>+</sup> = 344.3. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.86 (d, <i>J</i> = 2.7 Hz, 1 H) 8.58 (d, <i>J</i> = 2.7 Hz, 1 H) 7.29â7.40 (m, 5 H) 5.33 (s, 2 H) 3.51â3.56 (m, 4 H) 3.37 (s, 3 H) 2.96 (s, 3 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1-Benzyl-<i>N</i><sup>5</sup>-(2-hydroxyethyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>24</b>)</h4><div class="NLM_p last">Prepared from <b>64</b> (154.3 mg, 0.539 mmol) and ethanolamine (0.065 mL, 1.078 mmol) using the same procedure as <b>20</b> to give 1-benzyl-<i>N</i><sup>5</sup>-(2-hydroxyethyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (175 mg, 0.531 mmol, 99% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.69 min, [MH]<sup>+</sup> = 330.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.37 (br. q, <i>J</i> = 4.4, 4.4, 4.4 Hz, 1 H) 8.83 (d, <i>J</i> = 2.7 Hz, 1 H) 8.79 (d, <i>J</i> = 2.7 Hz, 1 H) 8.56 (br. t, <i>J</i> = 5.4, 5.4 Hz, 1 H) 7.27â7.40 (m, 5 H) 5.30 (s, 2 H) 4.71 (t, <i>J</i> = 5.6 Hz, 1 H) 3.50 (q, <i>J</i> = 5.9 Hz, 2 H) 3.26â3.34 (obs q, <i>J</i> = 5.6 Hz, 2 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 1-Benzyl-<i>N</i><sup>5</sup>-isobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>25</b>)</h4><div class="NLM_p last">Prepared from <b>62</b> (52 mg, 0.112 mmol) and 2-methylpropan-1-amine (0.02 mL, 0.201 mmol), using the same procedure as for compound <b>21</b> to give 1-benzyl-<i>N</i><sup>5</sup>-isobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (36 mg, 0.095 mmol, 85% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.98 min, [MH]<sup>+</sup> = 342.0. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.88 (d, <i>J</i> = 2.4 Hz, 1 H) 8.58 (d, <i>J</i> = 2.9 Hz, 1 H) 7.28â7.45 (m, 5 H) 5.34 (s, 2 H) 3.17 (d, <i>J</i> = 6.8 Hz, 2 H) 2.97 (s, 3 H) 1.91 (dquin, <i>J</i> = 13.4, 6.8, 6.8, 6.8, 6.8 Hz, 1 H) 0.96 (d, <i>J</i> = 6.4 Hz, 6 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1-Benzyl-<i>N</i><sup>5</sup>-isopentyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>26</b>)</h4><div class="NLM_p last">Prepared from <b>62</b> (50 mg, 0.107 mmol) and 3-methylbutan-1-amine (18.72 mg, 0.215 mmol), using the same procedure as for compound <b>21</b> to give the title compound (6 mg, 16%). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.04 min, [MH]<sup>+</sup> = 356.3. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.85 (d, <i>J</i> = 2.4 Hz, 1 H) 8.56 (d, <i>J</i> = 2.4 Hz, 1 H) 7.28â7.42 (m, 5 H) 5.33 (s, 2 H) 3.35â3.42 (m, 2 H) 2.97 (s, 3 H) 1.59â1.74 (m, 1 H) 1.45â1.55 (m, 2 H) 0.97 (d, <i>J</i> = 6.4 Hz, 6 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>27</b>)</h4><div class="NLM_p last">Prepared from <b>62</b> (61 mg, 0.131 mmol) and 2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanamine (33.8 mg, 0.262 mmol), using the same procedure as for compound <b>21</b> to give 1-benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1,2-dihydropyridine-3,5-dicarboxamide (9 mg, 0.020 mmol, 16% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.90 min, [MH]<sup>+</sup> = 398.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.86 (d, <i>J</i> = 2.4 Hz, 1 H) 8.57 (d, <i>J</i> = 2.9 Hz, 1 H) 7.28â7.44 (m, 5 H) 5.34 (s, 2 H) 3.88â3.98 (m, 2 H) 3.36â3.47 (m, 4 H) 2.97 (s, 3 H) 1.52â1.76 (m, 6 H) 1.22â1.40 (m, 2 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(2-(piperidin-3-yl)ethyl)-1,2-dihydropyridine-3,5-dicarboxamide, Hydrochloride (<b>28</b>)</h4><div id="sec3_2_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Step 1. <i>tert</i>-Butyl 3-(2-(1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)ethyl)piperidine-1-carboxylate (<b>65</b>)</h5><div class="NLM_p last">Prepared from <b>64</b> (101 mg, 0.353 mmol) and <i>tert</i>-butyl 3-(2-aminoethyl)piperidine-1-carboxylate (174 mg, 0.762 mmol), using the same procedure as for <b>20</b> to give <i>tert</i>-butyl 3-(2-(1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)ethyl)piperidine-1-carboxylate (175 mg, 0.282 mmol, 80% yield) as a colorless oil. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.18 min, [MH]<sup>+</sup> = 397.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.45 (d, <i>J</i> = 2.7 Hz, 1 H) 8.21 (d, <i>J</i> = 2.7 Hz, 1 H) 7.27â7.41 (m, 5 H) 5.23â5.37 (ABq, <i>J</i> = 14.4 Hz, 2 H) 2.97â3.11 (m, 3 H) 2.94 (s, 3 H) 2.69â2.84 (m, 2 H) 1.87â1.99 (m, 1 H) 1.66â1.80 (m, 1 H) 1.19â1.65 (m, 14 H).</div></div><div id="sec3_2_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Step 2: 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(2-(piperidin-3-yl)ethyl)-1,2-dihydropyridine-3,5-dicarboxamide, Hydrochloride (<b>28</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl 3-(2-(1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)ethyl)piperidine-1-carboxylate (<b>65</b>, 146 mg, 0.294 mmol) and TFA (1 mL, 12.98 mmol) were stirred at rt in DCM (4 mL) for 1 h. The reaction mixture was concentrated and loaded onto a 2 g SCX cartridge (preconditioned with MeOH) and eluted with MeOH (40 mL), followed by 2 M NH<sub>3</sub> in MeOH (40 mL). The methanolic ammonia fractions containing product were combined and concentrated to give 112 mg of a colorless oil. The reaction was purified by MDAP (high pH). Fractions containing the desired product were concentrated to give 1-benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(2-(piperidin-3-yl)ethyl)-1,2-dihydropyridine-3,5-dicarboxamide (93 mg, 0.211 mmol, 72% yield). The yellow oil was dissolved in 1 mL of MeOH and 1 M HCl in diethyl ether (0.23 mL, 0.230 mmol) was added and the solvent removed to afford 1-benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(2-(piperidin-3-yl)ethyl)-1,2-dihydropyridine-3,5-dicarboxamide, hydrochloride (97 mg, 0.202 mmol, 69% yield), as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.54 min, [MH]<sup>+</sup> = 397.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.39 (br q, <i>J</i> = 4.7 Hz, 1 H) 8.45 (d, <i>J</i> = 2.4 Hz, 1 H) 8.34 (d, <i>J</i> = 2.4 Hz, 1 H) 7.71â7.91 (m, 3 H) 7.29â7.42 (m, 5 H) 5.25â5.34 (ABq, <i>J</i> = 14.2 Hz, 2 H) 3.90 (br s, 2 H) 2.95â3.12 (m, 1 H) 2.74â2.90 (m, 6 H) 1.78â1.88 (m, 1 H) 1.54â1.73 (m, 2 H) 1.36â1.54 (m, 3 H) 1.15â1.32 (m, 1 H).</div></div></div><div id="sec3_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(3-(piperidin-3-yl)propyl)-1,2-dihydropyridine-3,5-dicarboxamide Hydrochloride (<b>29</b>)</h4><div id="sec3_2_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Step 1: <i>tert</i>-Butyl 3-(3-(1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)propyl)piperidine-1-carboxylate (<b>66</b>)</h5><div class="NLM_p last">Prepared from <b>64</b> (123 mg, 0.430 mmol) and <i>tert</i>-butyl 3-(3-aminopropyl)piperidine-1-carboxylate (125 mg, 0.516 mmol), using the same procedure as for <b>20</b> to give <i>tert</i>-butyl 3-(3-(1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)propyl)piperidine-1-carboxylate (188 mg, 0.331 mmol, 77% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.20 min, [MH]<sup>+</sup> = 511.2. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.54 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.75 (d, <i>J</i> = 2.9 Hz, 1 H) 8.47 (d, <i>J</i> = 2.9 Hz, 1 H) 7.32â7.41 (m, 5 H) 6.36 (br s, 1 H) 5.27 (s, 2 H) 3.87â4.03 (m, 2 H) 3.36â3.49 (m, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.74â2.86 (m, 2 H) 2.39â2.59 (m, 1 H) 1.76â1.92 (m, 1 H) 1.62â1.69 (m, 3 H) 1.41â1.53 (m, 9 H) 1.18â1.36 (m, 3 H) 1.04â1.17 (m, 1 H).</div></div><div id="sec3_2_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Step 2: 1-Benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(3-(piperidin-3-yl)propyl)-1,2-dihydropyridine-3,5-dicarboxamide Hydrochloride (<b>29</b>)</h5><div class="NLM_p last">Prepared from <b>66</b> (177 mg, 0.347 mmol), using the same deprotection conditions as for <b>28</b> to give 1-benzyl-<i>N</i><sup>3</sup>-methyl-2-oxo-<i>N</i><sup>5</sup>-(3-(piperidin-3-yl)propyl)-1,2-dihydropyridine-3,5-dicarboxamide, hydrochloride (40 mg, 0.081 mmol, 23% yield), as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.57 min, [MH]<sup>+</sup> = 411.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.38 (q, <i>J</i> = 4.9 Hz, 1 H) 8.78â8.86 (m, 2 H) 8.60 (br t, <i>J</i> = 5.1 Hz, 1 H) 7.28â7.40 (m, 5 H) 5.31 (s, 2 H) 3.19â3.25 (m, 3 H) 2.92â3.08 (m, 3 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 2.52â2.62 (m, 1 H) 2.19â2.39 (m, 1 H) 1.69â1.84 (m, 1 H) 1.58â1.70 (m, 1 H) 1.34â1.58 (m, 4 H) 1.13â1.31 (m, 2 H) 0.95â1.06 (m, 1 H).</div></div></div><div id="sec3_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (Â±)-1-Benzyl-<i>N</i><sup>3</sup>-methyl-<i>N</i><sup>5</sup>-(3-(morpholin-2-yl)propyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>30</b>)</h4><div id="sec3_2_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Step 1: (Â±)-<i>tert</i>-Butyl 2-(3-(1-Benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)propyl)morpholine-4-carboxylate (<b>63</b>)</h5><div class="NLM_p last">Prepared from <b>62</b> (25.7 mg, 0.055 mmol) and <i>tert</i>-butyl 2-(3-aminopropyl)morpholine-4-carboxylate (18 mg, 0.037 mmol), using the same procedure as for compound <b>21</b> to give <i>tert</i>-butyl 2-(3-(1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)propyl)morpholine-4-carboxylate (8 mg, 0.012 mmol, 34% yield) as a colorless oil. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.07 min, [MH]+ = 513.5.</div></div><div id="sec3_2_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> Step 2: (Â±)-1-Benzyl-<i>N</i><sup>3</sup>-methyl-<i>N</i><sup>5</sup>-(3-(morpholin-2-yl)propyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>30</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl 2-(3-(1-benzyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxamido)propyl)morpholine-4-carboxylate (<b>63</b>, 11 mg, 0.017 mmol) and TFA (0.1 mL, 1.298 mmol) were stirred at rt in dichloromethane (0.4 mL) for 30 min. The reaction mixture was then concentrated and loaded onto a 500 mg SCX cartridge (preconditioned with MeOH) and eluted with MeOH (4 CVâs) followed by 2 M NH<sub>3</sub> in MeOH (4 CVâs). The ammonia fractions containing product were combined and concentrated to give 15 mg of a colorless oil. This oil was purified by MDAP (high pH). The appropriate fractions were concentrated to give (Â±)-1-benzyl-<i>N</i><sup>3</sup>-methyl-<i>N</i><sup>5</sup>-(3-(morpholin-2-yl)propyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (2 mg, 4.36 Î¼mol, 25% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.54 min, [MH]+ = 413.5. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 9.79 (br s, 1 H) 8.85 (d, <i>J</i> = 2.9 Hz, 1 H) 8.45â8.62 (m, 2 H) 7.26â7.44 (m, 5 H) 5.34 (s, 2 H) 4.04 (dd, <i>J</i> = 12.7, 3.4 Hz, 1 H) 3.75 (td, <i>J</i> = 12.5, 2.4 Hz, 1 H) 3.62â3.71 (m, 1 H) 3.34â3.44 (m, 2 H) 3.12â3.25 (m, 2 H) 3.00â3.12 (m, 1 H) 2.97 (s, 3 H) 2.79 (dd, <i>J</i> = 12.2, 11.2 Hz, 1 H) 1.62â1.86 (m, 2 H) 1.51â1.61 (m, 2 H). <i>One exchangeable proton not observed</i>.</div></div></div><div id="sec3_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>)-<i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>31</b>)<b>:</b></h4><div id="sec3_2_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> Step 1: (<i>R</i>)-Methyl 2-Oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3-carboxylate (<b>97</b>)</h5><div class="NLM_p last">(<i>R</i>)-1-Phenylethanamine (8.93 mL, 70.2 mmol) was added to a stirred solution of methyl 2-oxo-2<i>H</i>-pyran-3-carboxylate (<b>96</b>, 10.3 g, 66.8 mmol, commercially available from, for example, Sigma-Aldrich) in a mixture of dry DMF (43 mL) and dry THF (173 mL). The resulting dark-red solution was stirred for 30 min under N<sub>2</sub>. EDC (16.66 g, 87 mmol) and DMAP (0.506 g, 4.14 mmol) were added, and the resulting suspension stirred over the weekend. The reaction mixture was evaporated in vacuo to a brown slurry. The residue was partitioned between EtOAc and water and the aqueous layer removed. The organic layer was washed (3 Ã 2 M aq HCl, 1Ã brine), dried over MgSO<sub>4</sub>, and filtered through silica eluting with EtOAc. The filtrate was evaporated in vacuo to give the product as a brown oil (12.94 g). LCMS (2 min TFA): <i>R</i><sub><i>t</i></sub> = 0.84 min, [MH]<sup>+</sup> = 258.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.97â8.01 (m, 1 H) 7.94 (dd, <i>J</i> = 6.8, 2.4 Hz, 1 H) 7.25â7.41 (m, 5 H) 6.34 (t, <i>J</i> = 6.8 Hz, 1 H) 6.20 (q, <i>J</i> = 7.2 Hz, 1 H) 3.74 (s, 3 H) 1.71 (d, <i>J</i> = 7.3 Hz, 3 H).</div></div><div id="sec3_2_16_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Step 2: (<i>R</i>)-Methyl 5-Bromo-2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3-carboxylate (<b>131</b>)</h5><div class="NLM_p last">NBS (10.74 g, 60.4 mmol) was added in one portion to a dark-brown solution of (<i>R</i>)-methyl 2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3-carboxylate (<b>97</b>, 12.94 g, 50.3 mmol) in 2-MeTHF (150 mL). The initial suspension became a light-brown solution and was stirred for 15 min whereupon it was a dark-brown solution. The reaction mixture was washed [3Ã satd aq NaHCO<sub>3</sub> (40 mL), 1Ã aq 10% sodium thiosulfate (20 mL), and 1Ã brine (10 mL)], dried over MgSO<sub>4</sub>, and evaporated in vacuo to a black oil. The residue was dissolved in toluene (40 mL), filtered through Celite, washed with toluene (80 mL), and evaporated in vacuo to give the product (19.62 g, 81% yield, 70% purity) as a black oil, which was used without further purification. LCMS (2 min TFA): <i>R</i><sub><i>t</i></sub> = 1.02 min, [MH]<sup>+</sup> = 336.0 and 337.9.</div></div><div id="sec3_2_16_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Step 3: (<i>R</i>)-5-Bromo-<i>N</i>-methyl-2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3-carboxamide (<b>98</b>)</h5><div class="NLM_p last">Methylamine solution (74 mL, 40% aq, 855 mmol) was added to a solution of (<i>R</i>)-methyl 5-bromo-2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3-carboxylate (<b>131</b>, 19.2 g, 40.0 mmol) in methanol (133 mL). The resulting solution was heated to 50 Â°C with a balloon fitted to the top of a condenser. The reaction mixture was stirred for 90 min. The reaction mixture was evaporated in vacuo to a black gum that was suspended in EtOAc. The suspension was filtered through silica eluting with EtOAc and the filtrate evaporated to give the product (13.1 g, 83% yield, 85% purity) as a brown gum. LCMS (2 min TFA): <i>R</i><sub><i>t</i></sub> = 1.01 min, [MH]<sup>+</sup> = 335.1 and 337.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.45 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.22â8.32 (m, 2 H) 7.25â7.47 (m, 5 H) 6.20 (q, <i>J</i> = 7.2 Hz, 1 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 1.79 (d, <i>J</i> = 6.8 Hz, 3 H).</div></div><div id="sec3_2_16_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> Step 4: (<i>R</i>)-Methyl 5-(Methylcarbamoyl)-6-oxo-1-(1-phenylethyl)-1,6-dihydropyridine-3-carboxylate (<b>132</b>)</h5><div class="NLM_p last">Xantphos (1.65 g, 2.85 mmol) and palladium(II) acetate (0.877 g, 3.91 mmol) were added to a solution of (<i>R</i>)-5-bromo-<i>N</i>-methyl-2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3-carboxamide (<b>98</b>, 13.1 g, 39.1 mmol), triethylamine (16.34 mL, 117 mmol) and methanol (15.81 mL, 391 mmol) in DMF (220 mL). Carbon monoxide was sparged through the mixture until a brown suspension formed. The reaction was held under a balloon of carbon monoxide and heated to 60 Â°C for 4 h. The reaction mixture was cooled to rt and sparged with N<sub>2</sub> to remove any residual carbon monoxide. The reaction mixture was filtered through Celite, rinsing with EtOAc and the filtrate evaporated in vacuo to a black slurry. The residue was partitioned between EtOAc (350 mL) and water (100 mL). The aqueous layer was removed, the organic layer washed (2Ã water [50 mL], 1Ã brine [50 mL]), dried over MgSO<sub>4</sub>, and evaporated in vacuo to a black gum. The gum was dissolved in toluene (60 mL) and loaded on to a Biotage 340 g silica column. The column was eluted with cyclohexane:EtOAc (20 â 66%). The product containing fractions were evaporated to give the product (7.43 g, 58% yield) as a brown gum. LCMS (2 min TFA): <i>R</i><sub><i>t</i></sub> = 0.94 min, [MH]<sup>+</sup> = 315.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.18â9.31 (m, 1 H) 8.68 (d, <i>J</i> = 2.4 Hz, 1 H) 8.46 (d, <i>J</i> = 2.4 Hz, 1 H) 7.28â7.46 (m, 5 H) 6.21 (q, <i>J</i> = 7.3 Hz, 1 H) 3.81 (s, 3 H) 2.84 (d, <i>J</i> = 4.9 Hz, 3 H) 1.81 (d, <i>J</i> = 6.8 Hz, 3 H).</div></div><div id="sec3_2_16_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> Step 5: (<i>R</i>)-5-(Methylcarbamoyl)-6-oxo-1-(1-phenylethyl)-1,6-dihydropyridine-3-carboxylic Acid (<b>133</b>)</h5><div class="NLM_p last">Sodium hydroxide (1.891 g, 47.3 mmol) was added to a solution of (<i>R</i>)-methyl 5-(methylcarbamoyl)-6-oxo-1-(1-phenylethyl)-1,6-dihydropyridine-3-carboxylate (<b>132</b>, 7.43 g, 23.64 mmol) in methanol (70 mL). Water was added to the stirred suspension and the resulting solution stirred overnight. The reaction mixture was evaporated in vacuo to a pale-brown solid and acidified with 2 M aq HCl (100 mL). Acetone (10 mL) was added and the suspension stirred for 15 min and filtered. The filtercake was washed [water:acetone (1:1, 20 mL), acetone (20 mL)] and dried in vacuo to give the product (6.40 g, 86% yield) as a beige solid. LCMS (2 min TFA): <i>R</i><sub><i>t</i></sub> = 0.82 min, [MH]<sup>+</sup> = 301.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 13.21 (br s, 1 H) 9.29 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.68 (d, <i>J</i> = 2.4 Hz, 1 H) 8.40 (d, <i>J</i> = 2.4 Hz, 1 H) 7.23â7.52 (m, 5 H) 6.22 (q, <i>J</i> = 7.2 Hz, 1 H) 2.84 (d, <i>J</i> = 4.9 Hz, 3 H) 1.80 (d, <i>J</i> = 7.3 Hz, 3 H).</div></div><div id="sec3_2_16_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> Step 6: (<i>R</i>)-<i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-(1-phenylethyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>31</b>)</h5><div class="NLM_p last">HATU (95 mg, 0.250 mmol) was added to a solution of (<i>R</i>)-5-(methylcarbamoyl)-6-oxo-1-(1-phenylethyl)-1,6-dihydropyridine-3-carboxylic acid (<b>133</b>, 50 mg, 0.166 mmol) and DIPEA (0.058 mL, 0.333 mmol) in DMF (0.5 mL). The solution was stirred for 5 min and cyclopropylamine (0.014 mL, 0.200 mmol) added. The resulting solution was stirred for 1 h. The reaction mixture was purified by MDAP (formic). The product containing fraction was azeotroped in vacuo to dryness with EtOH to give a white solid (50 mg, 84% yield). LCMS (2 min TFA): <i>R</i><sub><i>t</i></sub> = 0.85 min, [MH]<sup>+</sup> = 340.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.39 (q, <i>J</i> = 4.4 Hz, 1 H) 8.80 (d, <i>J</i> = 2.4 Hz, 1 H) 8.59 (d, <i>J</i> = 3.4 Hz, 1 H) 8.41 (d, <i>J</i> = 2.4 Hz, 1 H) 7.35â7.44 (m, 2 H) 7.29â7.35 (m, 3 H) 6.23 (q, <i>J</i> = 7.3 Hz, 1 H) 2.73â2.87 (m, 4 H) 1.80 (d, <i>J</i> = 7.3 Hz, 3 H) 0.65â0.73 (m, 2 H) 0.51â0.58 (m, 2 H). Chiral HPLC (25 cm Chiralcel OJ, col no. OJOOCE-IF013; eluent, 5% EtOH/heptane, flow rate = 1 mL/min, detection wavelength = 215 nm, RT, sample dissolved in EtOH/heptane): <i>R</i><sub><i>t</i></sub> = 12.8 min, <i>er</i> = 100:0 (enantiomer, <i>R</i><sub><i>t</i></sub> = 17.8 min).</div></div></div><div id="sec3_2_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 1-Benzyl-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>,6-dimethyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>32</b>)</h4><div id="sec3_2_99_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Step 1: Methyl 2-((Dimethylamino)methylene)-3-oxobutanoate (<b>128</b>)<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></h5><div class="NLM_p last">A solution of methyl 3-oxobutanoate (4640 Î¼L, 43.0 mmol) and DMF-DMA (6050 Î¼L, 45.2 mmol) was heated to 80 Â°C for 3 h, after which TLC (80% ethyl acetate/cyclohexane) indicated reaction completion. The reaction mixture was evaporated in vacuo and azeotroped with toluene three times to remove any residual DMF-DMA. The residual red oil was left to stand overnight, after which time it solidified to give methyl 2-((dimethylamino)methylene)-3-oxobutanoate (5761 mg, 30.3 mmol, 70% yield) as a single unknown geometric isomer (<i>E</i>/<i>Z</i>). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.50 min, [MH]<sup>+</sup> = 172.1.</div></div><div id="sec3_2_99_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Step 2: Dimethyl 6-Methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxylate (<b>94</b>)<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a></h5><div class="NLM_p last">A mixture of methyl 2-((dimethylamino)methylene)-3-oxobutanoate (<b>128</b>, 4600 mg, 26.9 mmol) and methyl 2-cyanoacetate (2371 Î¼L, 26.9 mmol) was stirred for 5 h at 40 Â°C, then cooled to rt and left to stand over a weekend. The solid reaction mixture was suspended in diethyl ether (50 mL) by sonication and filtered. The filter cake was washed with diethyl ether (3 Ã 15 mL) and dried in vacuo to yield the desired product dimethyl 6-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxylate (5480 mg, 21.90 mmol, 82% yield) as an orange solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.58 min, [MH]<sup>+</sup> = 226.2.</div></div><div id="sec3_2_99_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> Step 3: Dimethyl 1-benzyl-6-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxylate (<b>129</b>)</h5><div class="NLM_p last">To a stirred solution of dimethyl 6-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxylate (<b>94</b>, 2000 mg, 8.88 mmol) in DMF (40 mL) was added potassium carbonate (3682 mg, 26.6 mmol) and (bromomethyl)benzene (1.056 mL, 8.88 mmol), and the reaction was heated to 40 Â°C for 4 h. The reaction mixture was then poured onto water (400 mL) and extracted with ethyl acetate (3 Ã 100 mL). The combined organics were washed with brine (3 Ã 50 mL), dried through a hydrophobic frit, and evaporated in vacuo to yield an orange solid (2.35 g). The residue was loaded in dichloromethane onto a 50 g SNAP cartridge and purified via Biotage SP4 flash chromatography, eluting from 17 to 87% ethyl acetate/cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield the desired product, dimethyl 1-benzyl-6-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxylate (1034 mg, 3.12 mmol, 35% yield), as an orange solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.97 min, [MH]<sup>+</sup> = 316.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.78 (s, 1 H) 7.25â7.38 (obs m, 3 H) 7.13â7.20 (m, 2 H) 5.50 (br s, 2 H) 3.94 (s, 3 H) 3.87 (s, 3 H) 2.81 (s, 3 H).</div></div><div id="sec3_2_99_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> Step 4: Methyl 1-Benzyl-2-methyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>95</b>)</h5><div class="NLM_p last">Methanamine (2 M in THF, 7.9 mL, 15.80 mmol) was added to a solution of dimethyl 1-benzyl-6-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxylate (<b>129</b>, 500 mg, 1.586 mmol) in methanol (5.8 mL), and the resulting solution was heated to 50 Â°C for 30 min then left to stand at rt overnight. The reaction mixture was evaporated in vacuo to yield the crude product as a yellow solid (525 mg). The residue was loaded in dichloromethane onto a 25 g SNAP cartridge and purified via Biotage SP4 flash chromatography, eluting from 0â50% (3:1 ethyl acetate:ethanol)/cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield the desired product methyl 1-benzyl-2-methyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (430 mg, 1.300 mmol, 82% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.95 min, [MH]<sup>+</sup> = 315.1. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.25 (q, <i>J</i> = 4.3 Hz, 1 H) 8.78 (s, 1 H) 7.32â7.37 (m, 2 H) 7.26â7.31 (m, 1 H) 7.13 (d, <i>J</i> = 7.3 Hz, 2 H) 5.51 (br s, 2 H) 3.82 (s, 3 H) 2.83 (d, <i>J</i> = 4.8 Hz, 3 H) 2.73 (s, 3 H).</div></div><div id="sec3_2_99_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> Step 5: 1-Benzyl-2-methyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic Acid (<b>130</b>)</h5><div class="NLM_p last">Methyl 1-benzyl-2-methyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>95</b>, 420 mg, 1.336 mmol) was suspended in 1,4-dioxane (3.4 mL). Water (3.4 mL) was added, followed by lithium hydroxide (64.0 mg, 2.67 mmol) and the reaction mixture stirred at rt for 1 h 45 min. The reaction mixture was neutralized with 2 M HCl and evaporated in vacuo to yield 1-benzyl-2-methyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic acid (613 mg, 1.327 mmol, 99% yield) as a white solid which was used crude in the subsequent reaction. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.83 min, [MH]<sup>+</sup> = 301.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 12.19â14.36 (m, 1 H) 9.28 (q, <i>J</i> = 4.7 Hz, 1 H) 8.80 (s, 1 H) 7.31â7.38 (m, 2 H) 7.24â7.31 (m, 1 H) 7.10â7.16 (m, 2 H) 5.51 (s, 2 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 2.75 (s, 3 H).</div></div><div id="sec3_2_99_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> Step 6: 1-Benzyl-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>,6-dimethyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>32</b>)</h5><div class="NLM_p last">To a solution of 1-benzyl-2-methyl-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic acid (<b>130</b>, 120 mg, 0.240 mmol) and HATU (164 mg, 0.432 mmol) in DMF (2.4 mL) was added DIPEA (84 Î¼L, 0.480 mmol) and cyclopropanamine (19.9 Î¼L, 0.287 mmol). The reaction mixture was stirred for 1 h. The reaction mixture was then poured onto water (30 mL) and extracted with ethyl acetate (3 Ã 10 mL). The ethyl acetate portions were washed with brine (2 Ã 10 mL) and evaporated in vacuo to yield the crude product (162 mg) as a red gum. The residue was loaded in dichloromethane and purified via Biotage SP4 flash chromatography, eluting from 12 to 62% (3:1 ethyl acetate:ethanol)/cyclohexane. The relevant fractions were combined and evaporated in vacuo. The residue was taken up in ethyl acetate (30 mL) and washed with water (2 Ã 30 mL) and brine (10 mL). The organic layer was dried through a hydrophobic frit and evaporated in vacuo to yield the desired product, 1-benzyl-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>,6-dimethyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (63 mg, 0.176 mmol, 74% yield) as an off-white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.80 min, [MH]<sup>+</sup> = 340.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.35â9.49 (m, 1 H) 8.52 (d, <i>J</i> = 3.9 Hz, 1 H) 8.29 (s, 1 H) 7.32â7.40 (m, 2 H) 7.25â7.32 (m, 1 H) 7.10â7.17 (m, 2 H) 5.47 (s, 2 H) 2.75â2.87 (m, 4 H) 2.47 (s, 3 H) 0.65â0.72 (m, 2 H) 0.49â0.55 (m, 2 H).</div></div></div><div id="sec3_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1-(Cyclohexylmethyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>33</b>)</h4><div id="sec3_2_18_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> Step 1: 5-Bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>67</b>)</h5><div class="NLM_p last">Methyl 5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylate (<b>60</b>, 2 g, 8.62 mmol) and methylamine (13 mL, 26.0 mmol, 2 M in THF) were refluxed under N<sub>2</sub>. After 4 h, a white precipitate had formed. THF (15 mL) was added, and the solution was refluxed for 1 h. Further methylamine (13 mL, 26.0 mmol, 2 M in THF) was added and the reaction refluxed for 2 h. Further methylamine (22 mL, 44.0 mmol, 2 M in THF) was added and the reaction refluxed overnight. The solution was concentrated to give a yellow solid. This was transferred to 2 Ã 20 mL microwave vials with methylamine (15 mL, 30.0 mmol, 2 M in THF) and THF (15 mL), and both were heated at 80 Â°C for 1 h. The suspension from the first microwave vial was concentrated and triturated from diethyl ether to give 5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>67</b>, 880 mg, 38% yield, 85% purity). The suspension from the second microwave vial was concentrated and triturated from diethyl ether to give further 5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>67</b>, 880 mg, 40% yield, 90% purity). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.50 min, [MH]+ = 231.0, 233.0. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.38 (d, <i>J</i> = 2.9 Hz, 1 H) 7.87 (d, <i>J</i> = 2.9 Hz, 1 H) 2.95 (s, 3 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_2_18_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> Step 2: 5-Bromo-1-(cyclohexylmethyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>69</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (103.4 mg, 0.448 mmol) and (bromomethyl)cyclohexane (0.156 mL, 1.119 mmol), using the same procedure as for <b>68</b> to give 5-bromo-1-(cyclohexylmethyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (83.4 mg, 0.255 mmol, 57% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.16 min, [MH]<sup>+</sup> = 327.1, 329.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.61 (br s, 1 H) 8.59 (d, <i>J</i> = 2.9 Hz, 1 H) 7.54 (d, <i>J</i> = 2.9 Hz, 1 H) 3.83 (d, <i>J</i> = 7.3 Hz, 2 H) 2.99 (d, <i>J</i> = 4.9 Hz, 3 H) 1.66â1.91 (m, 6 H) 1.14â1.33 (m, 3 H) 0.96â1.09 (m, 2 H).</div></div><div id="sec3_2_18_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> Step 3: 1-(Cyclohexylmethyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>33</b>)</h5><div class="NLM_p last">Prepared from <b>69</b> (30.2 mg, 0.092 mmol) and cyclopropylamine (0.020 mL, 0.277 mmol), using the same procedure as for <b>35</b> to give 1-(cyclohexylmethyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (7.4 mg, 0.022 mmol, 24% yield) as a cream solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.97 min, [MH]<sup>+</sup> = 332.3. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.67 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.83 (d, <i>J</i> = 2.9 Hz, 1 H) 8.35 (d, <i>J</i> = 2.9 Hz, 1 H) 6.95 (br s, 1 H) 3.93 (d, <i>J</i> = 7.3 Hz, 2 H) 3.00 (d, <i>J</i> = 5.4 Hz, 3 H) 2.82â2.96 (m, 1 H) 1.64â1.92 (m, 6 H) 1.13â1.30 (m, 3 H) 0.98â1.13 (m, 2 H) 0.82â0.95 (m, 2 H) 0.59â0.73 (m, 2 H).</div></div></div><div id="sec3_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (Â±)-<i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-2-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>34</b>)</h4><div id="sec3_2_19_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> Step 1: (Â±)-5-Bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-2-yl)methyl)-1,2-dihydropyridine-3-carboxamide (<b>70</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (99.8 mg, 0.432 mmol) and (Â±)-2-(bromomethyl)tetrahydro-2<i>H</i>-pyran (0.138 mL, 1.080 mmol), using the same procedure as for <b>68</b> to give (Â±)-5-bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-2-yl)methyl)-1,2-dihydropyridine-3-carboxamide (45.3 mg, 0.138 mmol, 32% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.92 min, [MH]<sup>+</sup> = 329.1, 331.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.56 (br s, 1 H) 8.59 (d, <i>J</i> = 2.9 Hz, 1 H) 7.72 (d, <i>J</i> = 2.9 Hz, 1 H) 4.36â4.43 (m, 1 H) 3.94â4.00 (m, 1 H) 3.56â3.64 (m, 2 H) 3.29â3.40 (m, 1 H) 2.99 (d, <i>J</i> = 4.9 Hz, 3 H) 1.85â1.95 (m, 1 H) 1.73 (br d, <i>J</i> = 12.2 Hz, 1 H) 1.49â1.64 (m, 3 H) 1.23â1.37 (m, 1 H).</div></div><div id="sec3_2_19_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> Step 2: (Â±)-<i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-2-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>34</b>)</h5><div class="NLM_p last">Prepared from <b>70</b> (45.3 mg, 0.138 mmol) and cyclopropylamine (0.029 mL, 0.413 mmol), using the same procedure as for <b>35</b> to give (Â±)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-2-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (3.7 mg, 0.011 mmol, 8% yield) as a yellow solid. LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.78 min, [MH]<sup>+</sup> = 334.2. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.56 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.76 (d, <i>J</i> = 2.9 Hz, 1 H) 8.44 (d, <i>J</i> = 2.9 Hz, 1 H) 6.69 (br s, 1 H) 4.39 (dd, <i>J</i> = 13.2, 2.4 Hz, 1 H) 3.91â3.98 (m, 1 H) 3.76 (dd, <i>J</i> = 13.2, 8.8 Hz, 1 H) 3.63 (ddt, <i>J</i> = 11.0, 8.7, 2.4, 2.4 Hz, 1 H) 3.26â3.36 (m, 1 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.83â2.97 (m, 1 H) 1.85â1.96 (m, 1 H) 1.72 (br d, <i>J</i> = 12.7 Hz, 1 H) 1.46â1.67 (m, 3 H) 1.24â1.46 (m, 1 H) 0.81â0.94 (m, 2 H) 0.57â0.71 (m, 2 H).</div></div></div><div id="sec3_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (Â±)-<i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>35</b>)</h4><div id="sec3_2_20_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> Step 1: (Â±)-5-Bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1,2-dihydropyridine-3-carboxamide (<b>68</b>).</h5><div class="NLM_p last">5-Bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>67</b>, 102.8 mg, 0.445 mmol), (Â±)-3-(bromomethyl)tetrahydro-2<i>H</i>-pyran (197.6 mg, 1.104 mmol), and potassium carbonate (134.5 mg, 0.973 mmol) were stirred in methanol (2 mL), and the mixture was heated at 60 Â°C for 8 days. The reaction mixture was then sealed and left to stand at room temperature for 2.75 days before stirring at 60 Â°C was resumed for three days. Over this time, the solvent level was maintained at 2 mL by the addition of further methanol. After allowing to cool to rt, the mixture was partitioned between water (5 mL) and ethyl acetate (5 mL). The aqueous phase was extracted with further ethyl acetate (2 Ã 5 mL), and the combined organic phases were dried by filtering through a hydrophobic frit. The solvent was evaporated under a stream of nitrogen and the residue was dissolved in a mixture of DMSO/methanol (1:1, 2 mL) and directly purified by MDAP (formic). The required fractions were concentrated in vacuo to give (Â±)-5-bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1,2-dihydropyridine-3-carboxamide (71.2 mg, 0.216 mmol, 49% yield) as a cream solid. A small amount of the <i>O</i>-alkylated regioisomer: (Â±)-5-bromo-<i>N</i>-methyl-2-((tetrahydro-2<i>H</i>-pyran-3-yl)methoxy)nicotinamide (7.2 mg, 0.022 mmol, 5% yield) was also isolated. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.77 min, [MH]<sup>+</sup> = 329.1, 331.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.47 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.35 (d, <i>J</i> = 2.9 Hz, 1 H) 8.28 (d, <i>J</i> = 2.9 Hz, 1 H) 3.87â3.96 (m, 2 H) 3.63â3.73 (m, 2 H) 3.31â3.39 (m, 1 H) 3.17 (dd, <i>J</i> = 11.2, 9.3 Hz, 1 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 1.99â2.09 (m, 1 H) 1.58â1.69 (m, 2 H) 1.37â1.49 (m, 1 H) 1.21â1.31 (m, 1 H).</div></div><div id="sec3_2_20_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> Step 2: (Â±)-<i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>35</b>)</h5><div class="NLM_p last">A microwave vial was charged with (Â±)-5-bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1,2-dihydropyridine-3-carboxamide (<b>68</b>, 66.8 mg, 0.203 mmol), cobalt octacarbonyl (17.8 mg, 0.052 mmol), palladium(II) acetate (3.6 mg, 0.016 mmol), Xantphos (6.1 mg, 10.54 Î¼mol), and DMAP (52.8 mg, 0.432 mmol). 1,4-Dioxane (2 mL) was added, followed by cyclopropylamine (0.043 mL, 0.609 mmol), the vial was sealed, and the reaction mixture was heated in a microwave reactor at 75 Â°C for 30 min. The mixture was diluted with ethyl acetate (5 mL) and filtered using a Celite cartridge (2.5 g), which was eluted with further ethyl acetate (3 Ã 5 mL). The combined organic fractions were concentrated in vacuo to give a brown residue, which was dissolved in DMSO/methanol (1:1, 2 mL) and directly purified by MDAP (formic). The required fractions were combined and concentrated in vacuo to give the desired product as a pale-brown solid, (Â±)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (18.1 mg, 0.054 mmol, 27% yield). LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.69 min, [MH]<sup>+</sup> = 334.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.41 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.77 (d, <i>J</i> = 2.9 Hz, 1 H) 8.51 (d, <i>J</i> = 2.9 Hz, 2 H) 3.88â4.08 (m, 2 H) 3.59â3.77 (m, 2 H) 3.29â3.42 (m, 1 H) 3.19 (dd, <i>J</i> = 11.0, 9.0 Hz, 1 H) 2.84 (d, <i>J</i> = 4.9 Hz, 3 H) 1.99â2.11 (m, 1 H) 1.58â1.70 (m, 2 H) 1.36â1.50 (m, 1 H) 1.23â1.35 (m, 1 H) 0.66â0.73 (m, 2 H) 0.54â0.60 (m, 2 H).</div></div></div><div id="sec3_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>36</b>)</h4><div id="sec3_2_21_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> Step 1: 5-Bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1,2-dihydropyridine-3-carboxamide (<b>71</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (99.2 mg, 0.429 mmol) and 4-(bromomethyl)tetrahydro-2<i>H</i>-pyran (194.4 mg, 1.086 mmol), using the same procedure as for <b>68</b> to give 5-bromo-<i>N</i>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1,2-dihydropyridine-3-carboxamide (70.4 mg, 0.214 mmol, 50% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.73 min, [MH]<sup>+</sup> = 329.1 and 331.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.49 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.37 (d, <i>J</i> = 2.9 Hz, 1 H) 8.29 (d, <i>J</i> = 2.9 Hz, 1 H) 3.92 (d, <i>J</i> = 7.3 Hz, 2 H) 3.84 (dd, <i>J</i> = 11.5, 2.7 Hz, 2 H) 3.23 (td, <i>J</i> = 11.6, 2.2 Hz, 2 H) 2.83 (d, <i>J</i> = 4.9 Hz, 3 H) 1.98â2.14 (m, 1 H) 1.42 (br dd, <i>J</i> = 12.7, 2.0 Hz, 2 H) 1.27 (qd, <i>J</i> = 12.2, 4.4 Hz, 2 H).</div></div><div id="sec3_2_21_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>36</b>)</h5><div class="NLM_p last">Prepared from <b>71</b> (65.2 mg, 0.198 mmol) and cyclopropylamine (0.028 mL, 0.396 mmol), using the same procedure as for <b>35</b> to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (13.8 mg, 0.041 mmol, 21% yield) as a cream-colored solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.63 min, [MH]<sup>+</sup> = 334.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.44 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.78 (d, <i>J</i> = 2.4 Hz, 1 H) 8.46â8.56 (m, 2 H) 3.99 (d, <i>J</i> = 7.3 Hz, 2 H) 3.84 (br dd, <i>J</i> = 11.5, 2.7 Hz, 2 H) 3.23 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H) 2.80â2.87 (m, 4 H) 1.98â2.13 (m, 1 H) 1.37â1.51 (m, 2 H) 1.22â1.37 (m, 2 H) 0.63â0.73 (m, 2 H) 0.52â0.63 (m, 2 H).</div></div></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Compounds <b>37</b>â<b>39</b></h3><div id="sec3_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Step 1: Methyl 2-(Benzyloxy)-5-bromonicotinate (<b>110</b>)</h4><div class="NLM_p last">Methyl 5-bromo-2-oxo-1,2-dihydropyridine-3-carboxylate (<b>60</b>, 5 g, 21.55 mmol, commercially available from, for example, Sigma-Aldrich) was dissolved in chloroform (100 mL), then silver carbonate (11.88 g, 43.1 mmol) and benzyl bromide (3.33 mL, 28.0 mmol) were added and the mixture heated at reflux overnight. The mixture was filtered and the filtrate evaporated in vacuo to give a pale-yellow liquid. This was dissolved in DCM (5 mL) and loaded onto a 50 g silica column, then eluted with 0â50% EtOAc/cyclohexane and the product-containing fractions evaporated in vacuo to give methyl 2-(benzyloxy)-5-bromonicotinate (4.65 g, 14.43 mmol, 67% yield) as a colorless solid. LCMS (2 min High pH): <i>R</i><sub><i>t</i></sub> = 1.37 min, [MH]+ = 322.1, 324.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.35 (d, <i>J</i> = 2.7 Hz, 1 H) 8.29 (d, <i>J</i> = 2.4 Hz, 1 H) 7.51 (d, <i>J</i> = 7.6 Hz, 2 H) 7.38 (t, <i>J</i> = 7.5 Hz, 2 H) 7.28â7.34 (m, 1 H) 5.51 (s, 2 H) 3.93 (s, 3 H).</div></div><div id="sec3_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Step 2: 2-(Benzyloxy)-5-bromonicotinic Acid (<b>111</b>)</h4><div class="NLM_p last">Methyl 2-(benzyloxy)-5-bromonicotinate (<b>110</b>, 4.6 g, 14.28 mmol) was dissolved in THF (50 mL) and methanol (50 mL), then LiOH (1.368 g, 57.1 mmol) in water (50 mL) was added and the mixture stirred for 2 h at rt. The solvent was evaporated in vacuo, and the residue was suspended in water (100 mL) and acidified with 2 M HCl to pH 4, then extracted with 10% MeOH/DCM (3 Ã 100 mL, poor solubility) and the organic layer washed with water, dried, and evaporated in vacuo to give 2-(benzyloxy)-5-bromonicotinic acid (4.15 g, 13.47 mmol, 94% yield) as a colorless solid. LCMS (2 min high pH): <i>R</i><sub><i>t</i></sub> = 0.68 min, [MH]+ = 308.2, 310.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.43 (d, <i>J</i> = 2.4 Hz, 1 H) 8.18 (d, <i>J</i> = 2.7 Hz, 1 H) 7.47 (d, <i>J</i> = 7.1 Hz, 2 H) 7.37 (t, <i>J</i> = 7.3 Hz, 2 H) 7.27â7.33 (m, 1 H) 5.43 (s, 2 H).</div></div><div id="sec3_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Step 3: 2-(Benzyloxy)-5-bromo-<i>N</i>-methylnicotinamide (<b>85</b>)</h4><div class="NLM_p last">2-(Benzyloxy)-5-bromonicotinic acid (<b>111</b>, 4.2 g, 13.63 mmol) was suspended in DCM (50 mL) and oxalyl chloride (2.386 mL, 27.3 mmol) was added, followed by DMF (0.053 mL, 0.682 mmol), and the reaction mixture was stirred for 2 h at rt, then evaporated in vacuo. The residue was dissolved in THF (50 mL), then methanamine (13.63 mL, 2 M in THF, 27.3 mmol) was added and the resulting suspension stirred for 2 h at rt, then evaporated in vacuo. The residue was dissolved in THF (50 mL) and methanamine (13.63 mL, 2 M in THF, 27.3 mmol) was added, then the resulting mixture stirred for 2 h, then evaporated in vacuo. The residue was partitioned between EtOAc (100 mL) and water (100 mL), the aqueous layer was extracted with further EtOAc (100 mL) and the combined organics washed with brine, dried, and evaporated in vacuo to give a yellow gummy solid. This was dissolved in a mixture of DCM (20 mL) and methanol (5 mL) with difficulty, then loaded onto a 50 g silica column, which was then sucked dry using a vacuum line. The column was eluted with 0â100% EtOAc/cyclohexane to give 2-(benzyloxy)-5-bromo-<i>N</i>-methylnicotinamide (2.35 g, 7.32 mmol, 54% yield). LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 1.19 min, [MH]+ = 321.1, 323.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.65 (d, <i>J</i> = 2.4 Hz, 1 H) 8.31 (d, <i>J</i> = 2.4 Hz, 1 H) 7.87 (br s, 1 H) 7.33â7.48 (m, 5 H) 5.53 (s, 2 H) 2.94 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Step 4: 2,4,6-Trichlorophenyl 6-(Benzyloxy)-5-(methylcarbamoyl)nicotinate (<b>116</b>)</h4><div class="NLM_p last">Prepared from <b>85</b> (2 g, 6.23 mmol) and 2,4,6-trichlorophenyl formate (2.106 g, 9.34 mmol), using the same procedure as for <b>62</b> to give 2,4,6-trichlorophenyl 6-(benzyloxy)-5-(methylcarbamoyl)nicotinate (1.65 g, 3.54 mmol, 56.9% yield). LCMS (2 min High pH): <i>R</i><sub><i>t</i></sub> = 1.52 min, [MH]+ = 465.3. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.31 (d, <i>J</i> = 2.4 Hz, 1 H) 9.10 (d, <i>J</i> = 2.4 Hz, 1 H) 7.80 (d, <i>J</i> = 3.7 Hz, 1 H) 7.38â7.53 (m, 7 H) 5.68 (s, 2 H) 2.98 (d, <i>J</i> = 4.6 Hz, 3 H).</div></div><div id="sec3_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Step 5: 2-(Benzyloxy)-<i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methylpyridine-3,5-dicarboxamide (<b>118</b>)</h4><div class="NLM_p last">Prepared from <b>116</b> (1.65 g, 3.54 mmol) and cyclobutanamine (0.605 mL, 7.09 mmol), using the same procedure as for <b>21</b> to give 2-(benzyloxy)-<i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methylpyridine-3,5-dicarboxamide (1.025 g, 3.02 mmol, 85% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.01 min, [MH]<sup>+</sup> = 340.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.77 (d, <i>J</i> = 7.8 Hz, 1 H) 8.70 (d, <i>J</i> = 2.4 Hz, 1 H) 8.51 (d, <i>J</i> = 2.4 Hz, 1 H) 8.33 (br d, <i>J</i> = 4.4 Hz, 1 H) 7.46â7.51 (m, 2 H) 7.36â7.42 (m, 2 H) 7.29â7.35 (m, 1 H) 5.56 (s, 2 H) 4.41 (sxt, <i>J</i> = 8.1 Hz, 1 H) 2.81 (d, <i>J</i> = 4.9 Hz, 3 H) 2.15â2.28 (m, 2 H) 2.00â2.14 (m, 2 H) 1.61â1.75 (m, 2 H).</div></div><div id="sec3_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Step 6: Standard Procedure for the Alkylation of <i>N</i><sup>5</sup>-Cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide</h4><div class="NLM_p last">A stock solution of <i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>88</b>, 475 mg) in DMSO (11.4 mL) was made up. Then 0.6 mL of this solution was added separately to each of 2-(bromomethyl)pyridine, 3-(bromomethyl)pyridine and 4-(chloromethyl)pyridine, (0.301 mmol). Potassium carbonate (41.6 mg, 0.301 mmol) was added to each of the reaction vessels and the reactions left stirring overnight. The samples were then filtered before purification. The samples were dissolved in DMSO (0.8 mL) and purified by high pH MDAP. The solvent was dried under a stream of nitrogen to give the required products: <i>N</i><sup><i>5</i></sup><i>-Cyclobutyl-N</i><sup><i>3</i></sup><i>-methyl-2-oxo-1-(pyridin-2-ylmethyl)-1,2-dihydropyridine-3,5-dicarboxamide</i> (<b>37</b>): (13 mg, 34% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.71 min, [MH]<sup>+</sup> = 341.0. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.30 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.87 (d, <i>J</i> = 2.6 Hz, 1 H) 8.70â8.81 (m, 2 H) 8.47 (d, <i>J</i> = 4.5 Hz, 1 H) 7.79 (td, <i>J</i> = 7.7, 1.5 Hz, 1 H) 7.37 (d, <i>J</i> = 7.9 Hz, 1 H) 7.30 (dd, <i>J</i> = 6.8, 5.3 Hz, 1 H) 5.39 (s, 2 H) 4.40 (sxt, <i>J</i> = 7.9 Hz, 1 H) 2.80 (d, <i>J</i> = 4.9 Hz, 3 H) 2.16â2.27 (m, 2 H) 2.00â2.13 (m, 2 H) 1.61â1.74 (m, 2 H). <i>N</i><sup><i>5</i></sup><i>-Cyclobutyl-N</i><sup><i>3</i></sup><i>-methyl-2-oxo-1-(pyridin-3-ylmethyl)-1,2-dihydropyridine-3,5-dicarboxamide</i> (<b>38</b>): (12 mg, 32% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.51 min, [MH]<sup>+</sup> = 341.0. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.31 (br d, <i>J</i> = 4.5 Hz, 1 H) 8.83 (d, <i>J</i> = 1.9 Hz, 2 H) 8.76 (br d, <i>J</i> = 7.2 Hz, 1 H) 8.63 (s, 1 H) 8.51 (d, <i>J</i> = 4.2 Hz, 1 H) 7.74 (br d, <i>J</i> = 7.9 Hz, 1 H) 7.38 (dd, <i>J</i> = 7.7, 4.7 Hz, 1 H) 5.32 (s, 2 H) 4.39 (sxt, <i>J</i> = 8.2 Hz, 1 H) 2.82 (d, <i>J</i> = 4.9 Hz, 3 H) 2.13â2.28 (m, 2 H) 2.02â2.10 (m, 2 H) 1.62â1.74 (m, 2 H). <i>N</i><sup><i>5</i></sup><i>-Cyclobutyl-N</i><sup><i>3</i></sup><i>-methyl-2-oxo-1-(pyridin-4-ylmethyl)-1,2-dihydropyridine-3,5-dicarboxamide</i> (<b>39</b>): (15 mg, 39% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.46 min, [MH]<sup>+</sup> = 341.0. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.28 (br d, <i>J</i> = 4.5 Hz, 1 H) 8.87 (d, <i>J</i> = 2.6 Hz, 1 H) 8.71â8.81 (m, 2 H) 8.53 (d, <i>J</i> = 6.0 Hz, 2 H) 7.22 (d, <i>J</i> = 6.0 Hz, 2 H) 5.33 (s, 2 H) 4.39 (sxt, <i>J</i> = 8.2 Hz, 1 H) 2.81 (d, <i>J</i> = 4.9 Hz, 3 H) 2.21 (dtd, <i>J</i> = 10.8, 8.0, 8.0, 2.6 Hz, 2 H) 2.00â2.12 (m, 2 H) 1.61â1.75 (m, 2 H).</div></div><div id="sec3_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>N</i><sup>5</sup>-Cyclopropyl-1-(2-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>40</b>)</h4><div id="sec3_3_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> Step 1: 5-Bromo-1-(2-fluorobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>72</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (500 mg, 2.164 mmol) and 1-(bromomethyl)-2-fluorobenzene (0.392 mL, 3.25 mmol), using the same procedure as for <b>68</b> to give the title compound (536.3 mg, 73%) as a pale-yellow solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.98 min, [MH]<sup>+</sup> = 338.9 and 340.9. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.51 (br s, 1 H) 8.58 (d, <i>J</i> = 2.9 Hz, 1 H) 7.74 (dd, <i>J</i> = 2.9, 1.0 Hz, 1 H) 7.44 (td, <i>J</i> = 7.5, 1.7 Hz, 1 H) 7.37 (tdd, <i>J</i> = 7.8, 7.8, 5.7, 1.7 Hz, 1 H) 7.11â7.21 (m, 2 H) 5.22 (s, 2 H) 2.99 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-1-(2-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>40</b>)</h5><div class="NLM_p last">Prepared from <b>72</b> (150 mg, 0.442 mmol) and cyclopropanamine (0.061 mL, 0.885 mmol) using the same procedure as for <b>35</b>, to give <i>N</i><sup>5</sup>-cyclopropyl-1-(2-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (59.6 mg, 0.174 mmol, 39% yield) as a pale-yellow solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.82 min, [MH]<sup>+</sup> = 344.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.29 (q, <i>J</i> = 4.4 Hz, 1 H) 8.82 (d, <i>J</i> = 2.9 Hz, 1 H) 8.70 (d, <i>J</i> = 2.0 Hz, 1 H) 8.57 (br d, <i>J</i> = 4.4 Hz, 1 H) 7.32â7.43 (m, 1 H) 7.16â7.26 (m, 3 H) 5.34 (s, 2 H) 2.78â2.88 (m, 4 H) 0.67â0.73 (m, 2 H) 0.54â0.61 (m, 2 H).</div></div></div><div id="sec3_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>N<sup>5</sup></i>-Cyclopropyl-1-(3-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>41</b>)</h4><div id="sec3_3_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> Step 1: 5-Bromo-1-(3-fluorobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>73</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (150 mg, 0.649 mmol) and 1-(bromomethyl)-3-fluorobenzene (116 Î¼L, 0.946 mmol), using the same procedure as for <b>68</b> to give 5-bromo-1-(3-fluorobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (135 mg, 0.398 mmol, 61% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.96 min, [MH]<sup>+</sup> = 338.9, 340.9. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.51 (br s, 1 H) 8.61 (d, <i>J</i> = 2.9 Hz, 1 H) 7.63 (d, <i>J</i> = 2.9 Hz, 1 H) 7.38 (td, <i>J</i> = 7.8, 5.9 Hz, 1 H) 7.01â7.10 (m, 3 H) 5.18 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-1-(3-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>41</b>)</h5><div class="NLM_p last">Prepared from <b>73</b> (122 mg, 0.360 mmol) and cyclopropylamine (50.7 Î¼L, 0.719 mmol), using the same procedure as for <b>35</b> to give <i>N</i><sup>5</sup>-cyclopropyl-1-(3-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (38 mg, 0.105 mmol, 29% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.84 min, [MH]<sup>+</sup> = 344.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.50 (br s, 1 H) 8.74 (d, <i>J</i> = 2.4 Hz, 1 H) 8.47 (d, <i>J</i> = 2.9 Hz, 1 H) 7.31â7.40 (m, 1 H) 7.02â7.12 (m, 3 H) 6.63 (br s, 1 H) 5.25 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.85â2.94 (m, 1 H) 0.85â0.92 (m, 2 H) 0.61â0.67 (m, 2 H).</div></div></div><div id="sec3_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> <i>N<sup>5</sup></i>-Cyclopropyl-1-(4-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>42</b>)</h4><div id="sec3_3_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> Step 1: 5-Bromo-1-(4-fluorobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>74</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (150 mg, 0.649 mmol) and 1-(bromomethyl)-4-fluorobenzene (118 Î¼L, 0.947 mmol), using the same procedure as for <b>68</b> to give 5-bromo-1-(4-fluorobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (127 mg, 0.374 mmol, 58% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.96 min, [MH]<sup>+</sup> = 339.0, 341.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.53 (br s, 1 H) 8.60 (d, <i>J</i> = 2.9 Hz, 1 H) 7.62 (d, <i>J</i> = 2.9 Hz, 1 H) 7.29â7.34 (m, 2 H) 7.05â7.13 (m, 2 H) 5.16 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-1-(4-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>42</b>)</h5><div class="NLM_p last">Prepared from <b>74</b> (123 mg, 0.363 mmol) and cyclopropylamine (51.1 Î¼L, 0.725 mmol) using the same procedure as for <b>35</b>, to give <i>N</i><sup>5</sup>-cyclopropyl-1-(4-fluorobenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (47 mg, 0.130 mmol, 36% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.85 min, [MH]<sup>+</sup> = 344.2. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.52 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.76 (d, <i>J</i> = 2.9 Hz, 1 H) 8.49 (d, <i>J</i> = 2.9 Hz, 1 H) 7.32â7.37 (m, 2 H) 7.04â7.10 (m, 2 H) 6.74 (br s, 1 H) 5.23 (s, 2 H) 2.99 (d, <i>J</i> = 5.4 Hz, 3 H) 2.85â2.92 (m, 1 H) 0.83â0.92 (m, 2 H) 0.60â0.67 (m, 2 H).</div></div></div><div id="sec3_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(3-methylbenzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>43</b>)</h4><div id="sec3_3_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> Step 1: 5-Bromo-<i>N</i>-methyl-1-(3-methylbenzyl)-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>75</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (300 mg, 1.298 mmol) and 1-(bromomethyl)-3-methylbenzene (0.263 mL, 1.948 mmol), using the same procedure as for <b>68</b> to give 5-bromo-<i>N</i>-methyl-1-(3-methylbenzyl)-2-oxo-1,2-dihydropyridine-3-carboxamide (205.8 mg, 0.614 mmol, 47% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.05 min, [MH]<sup>+</sup> = 335.0 and 337.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.58 (br s, 1 H) 8.59 (d, <i>J</i> = 2.9 Hz, 1 H) 7.61 (d, <i>J</i> = 2.9 Hz, 1 H) 7.27â7.32 (obs m, 1 H) 7.19 (d, <i>J</i> = 7.8 Hz, 1 H) 7.08â7.11 (m, 2 H) 5.16 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.38 (s, 3 H).</div></div><div id="sec3_3_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(3-methylbenzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>43</b>)</h5><div class="NLM_p last">Prepared from <b>75</b> (100 mg, 0.298 mmol) and cyclopropanamine (0.041 mL, 0.597 mmol) using the same procedure as for <b>35</b>, to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(3-methylbenzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (52.7 mg, 0.155 mmol, 52% yield) as a yellow solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.90 min, [MH]<sup>+</sup> = 340.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.55 (br d, <i>J</i> = 1.5 Hz, 1 H) 8.69 (br t, <i>J</i> = 2.9 Hz, 1 H) 8.46 (d, <i>J</i> = 2.4 Hz, 1 H) 7.24â7.30 (obs m, 1 H) 7.10â7.19 (m, 3 H) 6.46 (br d, <i>J</i> = 10.8 Hz, 1 H) 5.22 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.85â2.93 (m, 1 H) 2.36 (s, 3 H) 0.85â0.92 (m, 2 H) 0.61â0.67 (m, 2 H).</div></div></div><div id="sec3_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(4-methylbenzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>44</b>)</h4><div id="sec3_3_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> Step 1: 5-Bromo-<i>N</i>-methyl-1-(4-methylbenzyl)-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>76</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (136 mg, 0.589 mmol) and 1-(bromomethyl)-4-methylbenzene (327 mg, 1.766 mmol), using the same procedure as for <b>68</b> to give 5-bromo-<i>N</i>-methyl-1-(4-methylbenzyl)-2-oxo-1,2-dihydropyridine-3-carboxamide (120 mg, 0.358 mmol, 61% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.04 min, [MH]<sup>+</sup> = 335.0, 337.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.58 (br d, <i>J</i> = 1.5 Hz, 1 H) 8.58 (d, <i>J</i> = 2.9 Hz, 1 H) 7.60 (d, <i>J</i> = 2.9 Hz, 1 H) 7.21 (app s, 4 H) 5.15 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.38 (s, 3 H).</div></div><div id="sec3_3_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(4-methylbenzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>44</b>)</h5><div class="NLM_p last">Prepared from <b>76</b> (120 mg, 0.358 mmol) and cyclopropylamine (50.5 Î¼L, 0.716 mmol) using the same procedure as for <b>35</b>, to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(4-methylbenzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (51 mg, 0.150 mmol, 42% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.90 min, [MH]<sup>+</sup> = 340.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.56 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.73 (d, <i>J</i> = 2.4 Hz, 1 H) 8.47 (d, <i>J</i> = 2.9 Hz, 1 H) 7.15â7.26 (m, 4 H) 6.66 (br s, 1 H) 5.22 (s, 2 H) 2.99 (d, <i>J</i> = 4.9 Hz, 3 H) 2.84â2.92 (m, 1 H) 2.35 (s, 3 H) 0.83â0.91 (m, 2 H) 0.60â0.65 (m, 2 H).</div></div></div><div id="sec3_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> <i>N</i><sup>5</sup>-Cyclopropyl-1-(3-methoxybenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>45</b>)</h4><div id="sec3_3_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> Step 1: 5-Bromo-2-methoxynicotinoyl Chloride (<b>112</b>)</h5><div class="NLM_p last">5-Bromo-2-methoxynicotinic acid (<b>83</b>, 15 g, 64.6 mmol, commercially available from, for example, Apollo Scientific) was suspended in DCM (100 mL) and then oxalyl chloride (16.98 mL, 194 mmol) was added, followed by DMF (5.01 mL, 64.6 mmol), and the mixture was stirred for 18 h at rt. The solvent was evaporated in vacuo, and the residue was redissolved in DCM (100 mL) and evaporated to dryness to give 5-bromo-2-methoxynicotinoyl chloride (16.33 g, 65.2 mmol, quant), which was used in the next step immediately. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.49 (d, <i>J</i> = 2.7 Hz, 1 H) 8.44 (d, <i>J</i> = 2.4 Hz, 1 H) 4.06 (s, 3 H).</div></div><div id="sec3_3_12_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> Step 2: 5-Bromo-2-methoxy-<i>N</i>-methylnicotinamide (<b>84</b>)</h5><div class="NLM_p last">5-Bromo-2-methoxynicotinoyl chloride (<b>112</b>, 16 g, 63.9 mmol) was dissolved in 2-methyltetrahydrofuran (100 mL) and Et<sub>3</sub>N (8.90 mL, 63.9 mmol) was added, followed by methanamine (31.9 mL, 2 M in THF, 63.9 mmol), and the mixture was stirred for 3 h at rt, then added to water (200 mL) and extracted with EtOAc (200 mL). The organic layer was washed with brine (200 mL), dried, and evaporated in vacuo to give 5-bromo-2-methoxy-<i>N</i>-methylnicotinamide (14.8 g, 60.4 mmol, 95% yield) as a pale-yellow solid. LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.83 min, [MH]+ = 245.1, 247.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.62 (d, <i>J</i> = 2.4 Hz, 1 H) 8.29 (d, <i>J</i> = 2.4 Hz, 1 H) 7.80 (br s, 1 H) 4.09 (s, 3 H) 3.02 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_12_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> Step 3: Methyl-6-methoxy-5-(methylcarbamoyl)nicotinate (<b>113</b>)</h5><div class="NLM_p last">Carbon monoxide was gently bubbled through a mixture of 5-bromo-2-methoxy-<i>N</i>-methylnicotinamide (<b>84</b>, 10.6 g, 43.3 mmol), Xantphos (1.502 g, 2.60 mmol), triethylamine (12.06 mL, 87 mmol), palladium(II) acetate (0.486 g, 2.163 mmol), and methanol (17.50 mL, 433 mmol) in DMF (150 mL) until a yellow/green suspension resulted. The suspension was held under a balloon of carbon monoxide and heated to 60 Â°C for 5 h. LCMS showed significant SM, so the reaction was left overnight (16 h). The reaction mixture was then allowed to cool to rt. The solution was diluted with water (300 mL) and extracted with EtOAc (3 Ã 300 mL) and the combined organics back extracted with brine (3 Ã 100 mL). The combined organics were then dried (Na<sub>2</sub>SO<sub>4</sub>) and evaporated in vacuo to a brown solid. The residue was dissolved in DCM, loaded on to a 340 g Biotage silica SNAP column, and eluted with 20 â 80% EtOAc/cyclohexane. The product containing fractions were evaporated in vacuo to a yellow solid, methyl 6-methoxy-5-(methylcarbamoyl)nicotinate (4 g, 17.84 mmol, 41% yield). As the yield was lower than expected, the retained aqueous layer was analyzed by LCMS and found to contain further product. This was therefore further extracted with DCM (3 Ã 100 mL), the combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo (for a prolonged period to remove DMF). The aqueous layer was reanalyzed by LCMS and found to no longer contain product. The crude product from the organic phase, a yellow solid was taken up in DCM and added to a SNAP silica cartridge (100 g) and eluted with 20 â 80% EtOAc/cyclohexane. The product containing fractions were evaporated in vacuo to a yellow solid, methyl 6-methoxy-5-(methylcarbamoyl)nicotinate (1.9 g, 8.47 mmol, 20% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.67 min, [MH]+ = 225.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.82 (d, <i>J</i> = 2.2 Hz, 1 H) 8.55 (d, <i>J</i> = 2.4 Hz, 1 H) 8.30 (br. d, <i>J</i> = 3.9 Hz, 1 H) 4.05 (s, 3 H) 3.87 (s, 3 H) 2.82 (d, <i>J</i> = 4.6 Hz, 3 H).</div></div><div id="sec3_3_12_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> Step 4: Methyl 5-(Methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>86</b>)</h5><div class="NLM_p last">Sodium iodide (4.88 g, 32.6 mmol) was added to a solution of methyl 6-methoxy-5-(methylcarbamoyl)nicotinate (<b>113</b>, 3.65 g, 16.28 mmol) in acetonitrile (100 mL), and this solution was stirred for 10 min. TMS-Cl (10.40 mL, 81 mmol) was added dropwise, and the reaction mixture was stirred at rt for 1 h. The reaction was quenched with water (100 mL), and the mixture was extracted five times with a mix of DCM/MeOH and the combined organic phase was dried over a hydrophobic frit and evaporated under vacuum. The crude material was dissolved in DCM and loaded onto a 100 g SNAP silica cartridge and eluted with 0â100% ethanol in EtOAc. The appropriate fractions were evaporated under vacuum, and the desired product was obtained. methyl 5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (1.5 g, 7.14 mmol, 44% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.47 min, [MH]+ = 211.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 10.25 (br. s, 1 H) 9.55 (br. d, <i>J</i> = 4.4 Hz, 1 H) 8.63 (d, <i>J</i> = 2.7 Hz, 1 H) 8.32 (d, <i>J</i> = 2.7 Hz, 1 H) 3.80 (s, 3 H) 2.82 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_12_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> Step 5: Methyl 1-(3-Methoxybenzyl)-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>114</b>)</h5><div class="NLM_p last">Methyl 5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>86</b>, 580 mg, 2.76 mmol), 1-(bromomethyl)-3-methoxybenzene (0.580 mL, 4.14 mmol), potassium carbonate (770 mg, 5.57 mmol), and DMF (5 mL) were stirred at 90 Â°C for 1 h. This was washed with LiCl (20 mL), partitioned between EtOAc (40 mL) and water (40 mL), the aqueous phase was extracted with EtOAc (2 Ã 40 mL), dried over a hydrophobic frit, and concentrated to give a colorless oil. This was purified by chromatography on SiO<sub>2</sub> (Biotage SNAP 100 g cartridge, eluting with 0â100% EtOAc/cyclohexane). The appropriate fractions were concentrated to give methyl 1-(3-methoxybenzyl)-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (683 mg, 1.861 mmol, 67% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.91 min, [MH]+ = 331.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.22 (br. d, <i>J</i> = 4.6 Hz, 1 H) 8.93 (d, <i>J</i> = 2.7 Hz, 1 H) 8.70 (d, <i>J</i> = 2.7 Hz, 1 H) 7.27 (t, <i>J</i> = 7.9 Hz, 1 H) 6.92 (m, <i>J</i> = 1.7 Hz, 1 H) 6.84â6.90 (m, 2 H) 5.30 (s, 2 H) 3.84 (s, 3 H) 3.73 (s, 3 H) 2.83 (s, 3 H).</div></div><div id="sec3_3_12_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> Step 6: 1-(3-Methoxybenzyl)-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic Acid (<b>115</b>)</h5><div class="NLM_p last">Methyl 1-(3-methoxybenzyl)-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylate (<b>114</b>, 670 mg, 2.028 mmol), lithium hydroxide (146 mg, 6.08 mmol), 1,4-dioxane (3 mL), and water (3 mL) were stirred at rt for 30 min. Acetic acid (1 mL, 17.47 mmol) was added, and the solution was partitioned between EtOAc (20 mL) and water (20 mL), and the aqueous phase was extracted with EtOAc (2 Ã 20 mL), dried over a hydrophobic frit, and concentrated to give 1-(3-methoxybenzyl)-5-(methylcarbamoyl)-6-oxo-1,6-dihydropyridine-3-carboxylic acid (641 mg, 1.824 mmol, 90% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.81 min, [MH]+ = 317.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 13.09 (br. s, 1 H) 9.26 (br. q, <i>J</i> = 4.4, 4.4, 4.4 Hz, 1 H) 8.84 (d, <i>J</i> = 2.7 Hz, 1 H) 8.70 (d, <i>J</i> = 2.4 Hz, 1 H) 7.27 (t, <i>J</i> = 7.9 Hz, 1 H) 6.91â6.94 (m, 1 H) 6.84â6.90 (m, 2 H) 5.29 (s, 2 H) 3.73 (s, 3 H) 2.82 (d, <i>J</i> = 4.9 Hz, 3 H).</div></div><div id="sec3_3_12_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> Step 7: <i>N</i><sup>5</sup>-Cyclopropyl-1-(3-methoxybenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>45</b>)</h5><div class="NLM_p last">Prepared from <b>115</b> (70 mg, 0.221 mmol) using the same procedure as for <b>20</b>, to give <i>N</i><sup>5</sup>-cyclopropyl-1-(3-methoxybenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (97 mg, 0.232 mmol, 100% yield) as a colorless oil. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.84 min, [MH]<sup>+</sup> = 356.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.83 (d, <i>J</i> = 2.4 Hz, 1 H) 8.55 (d, <i>J</i> = 2.9 Hz, 1 H) 7.27 (t, <i>J</i> = 7.8 Hz, 1 H) 6.87â6.94 (m, 3 H) 5.28 (s, 2 H) 3.79 (s, 3 H) 2.96 (s, 3 H) 2.78â2.86 (m, 1 H) 0.76â0.84 (m, 2 H) 0.60â0.66 (m, 2 H).</div></div></div><div id="sec3_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> <i>N</i><sup>5</sup>-Cyclopropyl-1-(4-methoxybenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>46</b>)</h4><div id="sec3_3_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> Step 1: 5-Bromo-1-(4-methoxybenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>77</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (205 mg, 0.887 mmol) and 1-(bromomethyl)-4-methoxybenzene (274 mg, 1.363 mmol), using the same procedure as for <b>68</b> to give 5-bromo-1-(4-methoxybenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (85 mg, 0.218 mmol, 25% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.97 min, [MH]<sup>+</sup> = 351, 353. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.42 (d, <i>J</i> = 2.9 Hz, 1 H) 8.19 (d, <i>J</i> = 2.9 Hz, 1 H) 7.31â7.37 (m, 2 H) 6.90â6.95 (m, 2 H) 5.18 (s, 2 H) 3.79 (s, 3 H) 2.95 (s, 3 H). <i>Exchangeable proton not observed</i>.</div></div><div id="sec3_3_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-1-(4-methoxybenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>46</b>)</h5><div class="NLM_p last">Prepared from <b>77</b> (100 mg, 0.285 mmol) and cyclopropanamine (0.04 mL, 0.577 mmol), using the same procedure as for <b>35</b> to give <i>N</i><sup>5</sup>-cyclopropyl-1-(4-methoxybenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (37 mg, 0.094 mmol, 33% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.82 min, [MH]<sup>+</sup> = 356.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 9.80 (br d, <i>J</i> = 3.9 Hz, 1 H) 8.81 (d, <i>J</i> = 2.4 Hz, 1 H) 8.54 (d, <i>J</i> = 2.4 Hz, 1 H) 7.32â7.36 (m, 2 H) 6.90â6.94 (m, 2 H) 5.24 (s, 2 H) 3.79 (s, 3 H) 2.96 (d, <i>J</i> = 4.9 Hz, 3 H) 2.82 (tt, <i>J</i> = 7.5, 3.8 Hz, 1 H) 0.76â0.84 (m, 2 H) 0.59â0.68 (m, 2 H). <i>One exchangeable proton not observed</i>.</div></div></div><div id="sec3_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 1-(3-Cyanobenzyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>47</b>)</h4><div id="sec3_3_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> Step 1: 5-Bromo-1-(3-cyanobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>78</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (155 mg, 0.671 mmol) and 3-(bromomethyl)benzonitrile (200 mg, 1.020 mmol), using the same procedure as for <b>68</b> to give 5-bromo-1-(3-cyanobenzyl)-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (182 mg, 0.473 mmol, 71% yield) as a white solid. LCMS (2 min, formic):<i>R</i><sub><i>t</i></sub> = 0.88 min, [MH]<sup>+</sup> = 346.0, 348.0. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.46 (d, <i>J</i> = 2.9 Hz, 1 H) 8.35 (d, <i>J</i> = 2.9 Hz, 1 H) 7.80 (s, 1 H) 7.68â7.72 (m, 2 H) 7.52â7.60 (m, 1 H) 5.29 (s, 2 H) 2.95 (s, 3 H). <i>Exchangeable proton not observed</i>.</div></div><div id="sec3_3_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> Step 2: 1-(3-Cyanobenzyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>47</b>)</h5><div class="NLM_p last">Prepared from <b>78</b> (92 mg, 0.266 mmol) and cyclopropanamine (0.04 mL, 0.577 mmol), using the same procedure as for <b>35</b> to give 1-(3-cyanobenzyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (13 mg, 0.033 mmol, 13% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.76 min, [MH]<sup>+</sup> = 351.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.84 (d, <i>J</i> = 2.4 Hz, 1 H) 8.64 (d, <i>J</i> = 2.9 Hz, 1 H) 7.82 (s, 1 H) 7.69â7.73 (m, 2 H) 7.54â7.58 (m, 1 H) 5.36 (s, 2 H) 2.94â2.97 (m, 3 H) 2.84 (tt, <i>J</i> = 7.5, 3.8 Hz, 1 H) 0.78â0.85 (m, 2 H) 0.62â0.68 (m, 2 H). <i>Exchangeable protons partially evident: 9.70 (br s, 0.65 H) and 8.45 (br s, 0.49 H)</i>.</div></div></div><div id="sec3_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> (<i>R</i>*)-<i>N</i><sup>5</sup>-Cyclopropyl-1-(3-(1-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>48</b>)</h4><div id="sec3_3_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> Step 1: 2-(Benzyloxy)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methylpyridine-3,5-dicarboxamide (<b>117</b>)</h5><div class="NLM_p last">Prepared from <b>116</b> (771 mg, 1.656 mmol) and cyclopropanamine (327 mg, 5.73 mmol), using the same procedure as for <b>21</b> to give 2-(benzyloxy)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methylpyridine-3,5-dicarboxamide (485.2 mg, 1.417 mmol, 86% yield) as a cream solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.90 min, [MH]+ = 326.3. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.88 (d, <i>J</i> = 2.7 Hz, 1 H) 8.72 (d, <i>J</i> = 2.4 Hz, 1 H) 7.88 (br s, 1 H) 7.32â7.50 (m, 5 H) 6.40 (br s, 1 H) 5.62 (s, 2 H) 2.87â2.99 (m, 4 H) 0.82â0.95 (m, 2 H) 0.57â0.70 (m, 2 H).</div></div><div id="sec3_3_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>87</b>)</h5><div class="NLM_p last">2-(Benzyloxy)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methylpyridine-3,5-dicarboxamide (<b>117</b>, 485 mg, 1.491 mmol) was taken up in TFA (5 mL, 64.9 mmol) and heated to 90 Â°C for 3 h and the reaction was concentrated in vacuo. The residue was stirred in Et<sub>2</sub>O (20 mL) for 30 min and then left to stand over the weekend. The resulting precipitate was collected by filtration to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (364.9 mg, 1.474 mmol, 99% yield) as a cream solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.45 min, [MH]+ = 236.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 12.79 (br s, 1 H) 9.41 (d, <i>J</i> = 4.9 Hz, 1 H) 8.76 (d, <i>J</i> = 2.7 Hz, 1 H) 8.44 (d, <i>J</i> = 3.7 Hz, 1 H) 8.19 (d, <i>J</i> = 2.7 Hz, 1 H) 2.76â2.87 (m, 4 H) 0.64â0.71 (m, 2 H) 0.51â0.57 (m, 2 H).</div></div><div id="sec3_3_15_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> Step 3: <i>N</i><sup>5</sup>-Cyclopropyl-1-(3-formylbenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>89</b>)</h5><div class="NLM_p last">Prepared from <b>87</b> (300 mg, 1.275 mmol) and 3-(bromomethyl)benzaldehyde (381 mg, 1.913 mmol) using the same procedure as for <b>93</b>, to give <i>N</i><sup>5</sup>-cyclopropyl-1-(3-formylbenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (369 mg, 0.992 mmol, 78% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.73 min, [MH]<sup>+</sup> = 354.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 10.00 (s, 1 H) 9.32 (q, <i>J</i> = 4.4 Hz, 1 H) 8.82 (s, 2 H) 8.59 (d, <i>J</i> = 3.9 Hz, 1 H) 7.82â7.88 (m, 2 H) 7.66â7.70 (m, 1 H) 7.61 (t, <i>J</i> = 7.8 Hz, 1 H) 5.39 (s, 2 H) 2.77â2.88 (m, 4 H) 0.66â0.74 (m, 2 H) 0.53â0.60 (m, 2 H).</div></div><div id="sec3_3_15_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> Step 4: (<i>R</i>*)-<i>N</i><sup>5</sup>-Cyclopropyl-1-(3-(1-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>48</b>)</h5><div class="NLM_p last">To a solution of <i>N</i><sup>5</sup>-cyclopropyl-1-(3-formylbenzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>89</b>, 183 mg, 0.518 mmol) in THF (10 mL) at â78 Â°C was added dropwise a solution of methylmagnesium bromide (3 M in diethyl ether, 0.690 mL, 2.071 mmol). The reaction was stirred at â78 Â°C for 45 min. The reaction was quenched with methanol while still at â78 Â°C. The solution was warmed to ambient temperature and concentrated in vacuo. The residue was suspended in ethyl acetate (50 mL) and washed with water (50 mL). Some insoluble solid remained in the aqueous layer. The aqueous layer was back extracted with ethyl acetate (2 Ã 20 mL), and the combined organics were washed with brine (10 mL), dried through a hydrophobic frit, and evaporated in vacuo to yield the crude product as a yellow glass (202 mg). The solid was loaded in the minimum volume of dichloromethane onto a SNAP cartridge (10 g) and purified via Biotage SP4 flash chromatography, eluting from 15 to 75% (3:1 ethyl acetate:ethanol)/cyclohexane). The relevant fractions were combined and evaporated in vacuo to yield a pale-yellow gum (145 mg). The sample was dissolved in MeOH:DMSO (2 Ã 1 mL, 1:1) and purified by MDAP (high pH). The relevant fractions were combined and evaporated in vacuo to yield the desired product, <i>N</i><sup>5</sup>-cyclopropyl-1-(3-(1-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (62 mg, 0.156 mmol, 30% yield) as a yellow solid. This product was combined with a second batch of the product prepared in an analogous manner, by sonicating together in diethyl ether and evaporating in vacuo to yield <i>N</i><sup>5</sup>-cyclopropyl-1-(3-(1-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (86.4 mg, 0.222 mmol, 43% yield) as a pale-yellow solid. The enantiomers were separated by chiral HPLC: The sample was dissolved in EtOH (2 mL). Injection: 1 mL of the solution was injected onto the column (column: 30 mm Ã 25 cm Chiralpak AD-H (5 Î¼m, lot no. ADH12143-01), which was eluted with 25% EtOH (+0.2% isopropylamine)/heptane (+0.2% isopropylamine), flow rate = 25 mL/min, detection wavelength, 215 nm, 4, ref 550, 100. Total number of injections = 4. Fractions from 22 to 25.5 min were bulked and labeled peak 1. Fractions from 25.5 to 28.5 min were bulked and labeled mix. Fractions from 28.5 to 35 min were bulked and labeled peak 2. The bulked mix fractions were concentrated in vacuo and reprocessed using the prep method above. The bulked pure fractions were concentrated in vacuo and then transferred to a weighed flask. Peak 1 was the desired product and was dried in vacuo to afford (<i>R*</i>)-<i>N</i><sup>5</sup>-cyclopropyl-1-(3-(1-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (36 mg, 0.097 mmol, 19% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.71 min, [MH]+ = 370.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.37 (q, <i>J</i> = 4.4 Hz, 1 H) 8.81 (d, <i>J</i> = 2.9 Hz, 1 H) 8.71 (d, <i>J</i> = 2.9 Hz, 1 H) 8.55 (d, <i>J</i> = 4.4 Hz, 1 H) 7.25â7.33 (m, 3 H) 7.14 (dt, <i>J</i> = 6.8, 2.0 Hz, 1 H) 5.29 (s, 2 H) 5.15 (d, <i>J</i> = 3.9 Hz, 1 H) 4.63â4.78 (m, 1 H) 2.78â2.87 (m, 3 H) 1.30 (d, <i>J</i> = 6.8 Hz, 4 H) 0.65â0.74 (m, 2 H) 0.53â0.60 (m, 2 H).</div></div></div><div id="sec3_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(3-(2-morpholinoethyl)benzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>49</b>)</h4><div id="sec3_3_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> Step 1: 2-(3-(Bromomethyl)phenyl)ethanol (<b>119</b>)</h5><div class="NLM_p last">Borane tetrahydrofuran complex (1 M in THF, 4.37 mL, 4.37 mmol) was added dropwise to a THF (20 mL) solution of 2-(3-(bromomethyl)phenyl)acetic acid (500 mg, 2.18 mmol, commercially available from, for example, Fluorochem) at 0 Â°C. The mixture was allowed to warm to rt and stirred for 2 h. Excess reagent was quenched by the slow addition of MeOH at 0 Â°C. The reaction mixture was concentrated in vacuo, loaded in DCM, and purified by Biotage Isolera flash chromatography, using a SNAP 25 g silica cartridge and eluting with a gradient of 0â100% EtOAc/cyclohexane to give after concentration in vacuo 2-(3-(bromomethyl)phenyl)ethanol (440 mg, 2.05 mmol, 94% yield) as a brown residue. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.87 min, [MH]<sup>+</sup> = 216. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.25â7.32 (m, 3 H) 7.13â7.21 (m, 1 H) 4.50â4.75 (m, 3 H) 3.61 (t, <i>J</i> = 6.8 Hz, 2 H) 2.72 (t, <i>J</i> = 7.1 Hz, 2 H).</div></div><div id="sec3_3_16_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> Step 2: <i>N</i><sup>5</sup>-Cyclopropyl-1-(3-(2-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>90</b>)</h5><div class="NLM_p last">2-(3-(Bromomethyl)phenyl)ethanol (<b>119</b>, 247 mg, 1.15 mmol) was added to a solution of <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>87</b>, 270 mg, 1.15 mmol) and potassium carbonate (317 mg, 2.30 mmol) in THF (15 mL). The reaction mixture was left to stir at rt for 2 h. The reaction mixture was heated to 50 Â°C and left to stir under N<sub>2</sub> overnight. The reaction mixture was then concentrated in vacuo and separated between DCM and water. The organic solution was concentrated in vacuo, loaded in DCM, and purified by Biotage Isolera flash chromatography using a SNAP 25 g silica cartridge and eluting with a gradient of 0â10% EtOH/EtOAc to give, after concentration in vacuo, <i>N</i><sup>5</sup>-cyclopropyl-1-(3-(2-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (310 mg, 0.08 mmol, 23% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.69 min, [MH]<sup>+</sup> = 370. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.37 (q, <i>J</i> = 4.7 Hz, 1 H) 8.80 (d, <i>J</i> = 2.9 Hz, 1 H) 8.70 (d, <i>J</i> = 2.4 Hz, 1 H) 8.54 (d, <i>J</i> = 3.9 Hz, 1 H) 7.26 (t, <i>J</i> = 7.3 Hz, 1 H) 7.14â7.20 (m, 2 H) 7.10 (br d, <i>J</i> = 7.8 Hz, 1 H) 5.26 (s, 2 H) 4.60 (t, <i>J</i> = 5.4 Hz, 1 H) 3.58 (td, <i>J</i> = 7.1, 5.4 Hz, 2 H) 2.83 (d, <i>J</i> = 4.9 Hz, 4 H) 2.70 (t, <i>J</i> = 7.1 Hz, 2 H) 0.65â0.73 (m, 2 H) 0.54â0.60 (m, 2 H).</div></div><div id="sec3_3_16_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> Step 3: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-(3-(2-oxoethyl)benzyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>120</b>)</h5><div class="NLM_p last">DessâMartin periodinane (149 mg, 0.35 mmol) was added to a solution of <i>N</i><sup>5</sup>-cyclopropyl-1-(3-(2-hydroxyethyl)benzyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>90</b>, 100 mg, 0.27 mmol) in DCM (2.5 mL) at 0 Â°C. The reaction mixture was allowed to warm to rt and stirred for 48 h. The reaction was quenched with water and extracted with DCM. The organic layer was passed through a hydrophobic frit and concentrated in vacuo to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-(3-(2-oxoethyl)benzyl)-1,2-dihydropyridine-3,5-dicarboxamide (100 mg, 0.27 mmol, quant) as a white solid, which was used crude in the next step. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.70 min, [MH]+ = 368.</div></div><div id="sec3_3_16_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> Step 4: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(3-(2-morpholinoethyl)benzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>49</b>)</h5><div class="NLM_p last">A mixture of <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-(3-(2-oxoethyl)benzyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>120</b>, 100 mg, 0.27 mmol), morpholine (0.047 mL, 0.54 mmol), and triethylamine (0.152 mL, 1.09 mmol) in DCM (3 mL) were stirred at rt for 45 min. Sodium triacetoxyborohydride (231 mg, 1.09 mmol) was added, and the reaction was stirred at rt for 24 h. The reaction mixture was then left to stand for 9 days. Saturated NaHCO<sub>3</sub> (aq, 40 mL) was added and the mixture stirred at rt for 15 min. The organic phase was separated. The aqueous phase was extracted with DCM. The combined organics were passed through a hydrophobic frit and concentrated in vacuo. The resulting compound was then purified by MDAP (high pH). The appropriate fractions were combined and evaporated to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-(3-(2-morpholinoethyl)benzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (30 mg, 0.07 mmol, 25% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.44 min, [MH]+ = 439.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.37 (q, <i>J</i> = 4.4 Hz, 1 H) 8.80 (d, <i>J</i> = 2.4 Hz, 1 H) 8.70 (d, <i>J</i> = 2.4 Hz, 1 H) 8.55 (d, <i>J</i> = 3.9 Hz, 1 H) 7.26 (t, <i>J</i> = 7.7 Hz, 1 H) 7.14â7.22 (m, 2 H) 7.11 (d, <i>J</i> = 7.3 Hz, 1 H) 5.26 (s, 2 H) 3.50â3.58 (m, 4 H) 2.77â2.87 (m, 4 H) 2.67â2.76 (m, 2 H) 2.44â2.51 (obs m, 2 H) 2.34â2.43 (m, 4 H) 0.65â0.74 (m, 2 H) 0.52â0.60 (m, 2 H).</div></div></div><div id="sec3_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> <i>N</i><sup>5</sup>-Cyclobutyl-<i>N</i><sup>3</sup>-methyl-1-(3-(methylamino)benzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>50</b>)</h4><div id="sec3_3_17_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> Step 1: <i>N</i><sup>5</sup>-Cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>88</b>)</h5><div class="NLM_p last">Prepared from <b>118</b> (1.18 g, 3.48 mmol) using the same procedure as for <b>87</b>, to give <i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (718 mg, 2.88 mmol, 83% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.55 min, [MH]<sup>+</sup> = 250.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 12.91 (br s, 1 H) 9.53 (br s, 1 H) 8.90 (br s, 1 H) 8.74 (br s, 1 H) 8.34 (br s, 1 H) 4.39â4.53 (m, 1 H) 2.95 (br d, <i>J</i> = 2.5 Hz, 4 H) 2.29 (br s, 2 H) 2.07â2.21 (m, 2 H) 1.78 (br s, 2 H).</div></div><div id="sec3_3_17_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i123" class="anchor-spacer"></div><h5 class="article-section__title" id="_i123"> Step 2: <i>N</i><sup>5</sup>-Cyclobutyl-<i>N</i><sup>3</sup>-methyl-1-(3-(methylamino)benzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>50</b>)</h5><div class="NLM_p last"><i>N</i><sup>5</sup>-Cyclobutyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>88</b>, 25 mg, 0.100 mmol), (3-(methylamino)phenyl)methanol (20.64 mg, 0.150 mmol), and 2-(tributylphosphoranylidene)acetonitrile (0.083 mL, 0.316 mmol) were combined in toluene (0.75 mL) and the reaction mixture heated in a 5 mL microwave vial at 120 Â°C for 30 min. The reaction mixture was poured onto water (10 mL) and extracted with ethyl acetate (3 Ã 8 mL). The combined ethyl acetate layers were washed with brine (10 mL), dried through a hydrophobic frit, and evaporated in vacuo to yield the crude product (170 mg). The residue was dissolved in MeOH/DMSO (1:1, 2 Ã 1 mL and purified by MDAP (high pH) The solvent was dried under a stream of nitrogen to leave a white powder. The fractions were combined in dichloromethane, evaporated in vacuo, sonicated with diethyl ether, and evaporated once more to yield the product, <i>N</i><sup>5</sup>-cyclobutyl-<i>N</i><sup>3</sup>-methyl-1-(3-(methylamino)benzyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (17 mg, 0.046 mmol, 46% yield) as an off-white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.72 min, [MH]+ = 369.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.59 (br d, <i>J</i> = 3.4 Hz, 1 H) 8.73 (d, <i>J</i> = 2.9 Hz, 1 H) 8.44 (d, <i>J</i> = 2.9 Hz, 1 H) 7.18 (t, <i>J</i> = 7.8 Hz, 1 H) 6.62 (d, <i>J</i> = 7.3 Hz, 1 H) 6.57 (ddd, <i>J</i> = 8.1, 2.5, 1.0 Hz, 1 H) 6.52 (t, <i>J</i> = 2.0 Hz, 1 H) 6.42 (br d, <i>J</i> = 7.3 Hz, 1 H) 5.17 (s, 2 H) 4.54 (sxt, <i>J</i> = 8.1 Hz, 1 H) 3.74â3.92 (m, 1 H) 3.01 (d, <i>J</i> = 4.9 Hz, 3 H) 2.83 (s, 3 H) 2.34â2.47 (m, 2 H) 1.90â2.06 (m, 2 H) 1.71â1.87 (m, 2 H).</div></div></div><div id="sec3_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> 1-((1<i>H</i>-Benzo[<i>d</i>]imidazol-6-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>51</b>)</h4><div id="sec3_3_18_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i125" class="anchor-spacer"></div><h5 class="article-section__title" id="_i125"> Step 1: 6-(Bromomethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>121</b>)</h5><div class="NLM_p last">1<i>H</i>-Benzo[<i>d</i>]imidazol-6-yl)methanol (205 mg, 1.384 mmol, commercially available from, for example, Fluorochem) and HBr (3.4 mL, 48% in water, 28.2 mmol) were heated at 80 Â°C for 30 min. The pH of the solution was adjusted to pH 9 with sodium bicarbonate solution and extracted with EtOAc (2 Ã 20 mL). The combined organic layers were dried over a hydrophobic frit and concentrated to give 6-(bromomethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole (90 mg, 0.341 mmol, 25% yield) as a colorless oil. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.26 (s, 1 H) 7.65 (s, 1 H) 7.61 (d, <i>J</i> = 8.3 Hz, 1 H) 7.32 (dd, <i>J</i> = 8.3, 1.5 Hz, 1 H) 5.66â5.66 (m, 1 H) 4.74 (s, 2 H). <i>Exchangeable proton not observed</i>.</div></div><div id="sec3_3_18_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> Step 2: 1-((1<i>H</i>-Benzo[<i>d</i>]imidazol-6-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>51</b>)</h5><div class="NLM_p last">Prepared from <b>87</b> (60 mg, 0.255 mmol) and 6-(bromomethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>121</b>, 88 mg, 0.417 mmol), using the same procedure as for <b>93</b> to give 1-((1<i>H</i>-benzo[<i>d</i>]imidazol-6-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (7 mg, 0.017 mmol, 7% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.40 min, [MH]+ = 366.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 9.81 (br d, <i>J</i> = 4.4 Hz, 1 H) 8.83 (d, <i>J</i> = 2.9 Hz, 1 H) 8.60 (d, <i>J</i> = 2.9 Hz, 1 H) 8.36â8.36 (m, 1 H) 8.22 (br s, 1 H) 7.69 (s, 1 H) 7.63 (d, <i>J</i> = 8.3 Hz, 1 H) 7.36 (d, <i>J</i> = 8.3 Hz, 1 H) 5.45 (s, 2 H) 2.97 (d, <i>J</i> = 4.9 Hz, 3 H) 2.81 (tt, <i>J</i> = 7.4, 3.9 Hz, 1 H) 0.76â0.84 (m, 2 H) 0.58â0.66 (m, 2 H). <i>One exchangeable proton not observed</i>.</div></div></div><div id="sec3_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-((2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-4-yl)methyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>52</b>)</h4><div id="sec3_3_19_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i128" class="anchor-spacer"></div><h5 class="article-section__title" id="_i128"> Step 1: Methyl 2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-7-carboxylate (<b>122</b>)</h5><div class="NLM_p last">To 2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-7-carboxylic acid (500 mg, 2.84 mmol, commercially available from, for example, Fluorochem) in methanol (30 mL), sulfuric acid (2.84 mL, 53.3 mmol) was added, and the reaction was stirred at 65 Â°C for 4 h. The reaction was then left to sit at rt for 3 days. The reaction mixture was basified with aqueous ammonia at 65 Â°C. The reaction mixture was extracted with DCM and concentrated under vacuum to give the title compound (410 mg, 2.156 mmol, 76% yield) as a yellow solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.35 min, [MH]<sup>+</sup> = 191.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 12.19 (br s, 1 H) 7.79 (d, <i>J</i> = 8.3 Hz, 1 H) 7.75 (d, <i>J</i> = 8.3 Hz, 1 H) 7.23 (t, <i>J</i> = 7.8 Hz, 1 H) 3.95 (s, 3 H) 2.56 (s, 3 H).</div></div><div id="sec3_3_19_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> Step 2: (2-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-7-yl)methanol (<b>123</b>)</h5><div class="NLM_p last">Lithium borohydride (85 mg, 3.88 mmol) and methanol (4 mL, 99 mmol) were dissolved in THF (20 mL). Then, methyl 2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-7-carboxylate (<b>122</b>, 410 mg, 2.156 mmol) in THF (5 mL) was added to the mixture. The reaction was then stirred overnight at 50 Â°C under an inert atmosphere. The reaction mixture was quenched with water and 2 M hydrochloric acid. The reaction mixture was then partitioned between water (50 mL) and ethyl acetate (50 mL). The aqueous layer was then extracted with ethyl acetate (2 Ã 50 mL). The combined organic layers were passed through a hydrophobic frit, and the solvent was removed under vacuum to give mainly unreacted SM. DIBAL-H (1.812 mL, 25% in toluene, 2.69 mmol) was added dropwise to the crude recovered sample (500 mg) in a solution of anhydrous DCM (20 mL) cooled to 0 Â°C. The reaction was allowed to stir at 0 Â°C for 1 h. Further DIBAL-H (1.812 mL, 25% in toluene, 2.69 mmol) was added to the solution, and the reaction mixture was allowed to stir overnight. Methanol (4 mL, 99 mmol) was added slowly to the solution, followed by Rochelleâs salt solution (40 mL) and the mixture allowed to stir for 40 min. The organic layer was separated and the aqueous layer extracted with DCM (2 Ã 20 mL).The combined organics were washed with water (40 mL) followed by brine (40 mL). The organic layer was passed through a hydrophobic frit and concentrated under vacuum to give the title compound (250 mg, 1.541 mmol, 72% yield) as a white solid, which was used crude in the next reaction. LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.48 min, [MH]<sup>+</sup> = 163.1.</div></div><div id="sec3_3_19_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i130" class="anchor-spacer"></div><h5 class="article-section__title" id="_i130"> Step 3: <i>tert</i>-Butyl 7-(Hydroxymethyl)-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-1-carboxylate (<b>124</b>)</h5><div class="NLM_p last">To a solution of (2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-7-yl)methanol (<b>123</b>, 250 mg, 1.541 mmol) in acetonitrile (11 mL) and water (2.75 mL) was added Boc<sub>2</sub>O (0.501 mL, 2.158 mmol) and sodium bicarbonate (259 mg, 3.08 mmol). The mixture was stirred overnight at rt. The reaction mixture was diluted with ethyl acetate (70 mL) and washed with 10% aqueous citric acid (3 Ã 25 mL). The aqueous layer was extracted with ethyl acetate (3 Ã 25 mL). The combined ethyl acetate portions were washed with water (25 mL) and brine (25 mL) before drying through a hydrophobic frit and evaporating in vacuo to yield the crude product. The product was loaded in dichloromethane onto a SNAP cartridge (25 g) and purified via Biotage SP4 flash chromatography, eluting from 0 to 100% ethyl acetate/cyclohexane. The relevant fractions were evaporated in vacuo to yield the title compound (128 mg, 0.488 mmol, 32% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.90 min, [MH]<sup>+</sup> = 263.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.76 (dd, <i>J</i> = 8.1, 1.2 Hz, 1 H) 7.34â7.40 (m, 1 H) 7.31 (t, <i>J</i> = 7.8 Hz, 1 H) 5.12â5.19 (m, 1 H) 4.88 (d, <i>J</i> = 5.4 Hz, 2 H) 2.73 (s, 3 H) 1.67 (s, 9 H).</div></div><div id="sec3_3_19_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i131" class="anchor-spacer"></div><h5 class="article-section__title" id="_i131"> Step 4: <i>tert</i>-Butyl 7-((5-(Cyclopropylcarbamoyl)-3-(methylcarbamoyl)-2-oxopyridin-1(2<i>H</i>)-yl)methyl)-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-1-carboxylate (<b>91</b>)</h5><div class="NLM_p last">DIAD (0.087 mL, 0.446 mmol) was added to a suspension of <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>87</b>, 50 mg, 0.213 mmol), <i>tert</i>-butyl 4-(hydroxymethyl)-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-1-carboxylate (<b>124</b>, 66.9 mg, 0.255 mmol), and triphenylphosphine (117 mg, 0.446 mmol) in toluene (2 mL). The reaction was stirred at rt under N<sub>2</sub> overnight. Further DIAD (0.087 mL, 0.446 mmol) and triphenylphosphine (117 mg, 0.446 mmol) were added and the reaction stirred for 3 h. The reaction was concentrated, loaded in DCM, and purified by chromatography on SiO<sub>2</sub> (Biotage SNAP 10 g cartridge, eluting with 0â100% EtOAc/cyclohexane). The appropriate fractions were concentrated to give the crude product. This was purified further by MDAP (high pH). The appropriate fractions were combined to give the title compound (14 mg, 0.029 mmol, 14% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.07 min, [MH]<sup>+</sup> = 480.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.80 (d, <i>J</i> = 2.4 Hz, 1 H) 8.73 (d, <i>J</i> = 2.4 Hz, 1 H) 7.95 (d, <i>J</i> = 7.8 Hz, 1 H) 7.34 (t, <i>J</i> = 7.8 Hz, 1 H) 7.23 (d, <i>J</i> = 7.3 Hz, 1 H) 5.66 (s, 2 H) 2.94â2.97 (m, 3 H) 2.84 (s, 3 H) 2.76â2.83 (m, 1 H) 1.73 (s, 9 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_3_19_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i132" class="anchor-spacer"></div><h5 class="article-section__title" id="_i132"> Step 5: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-1-((2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-4-yl)methyl)-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>52</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl 7-((5-(cyclopropylcarbamoyl)-3-(methylcarbamoyl)-2-oxopyridin-1(2<i>H</i>)-yl)methyl)-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-1-carboxylate (<b>91</b>, 10 mg, 0.021 mmol) was dissolved in HCl in IPA (0.634 Î¼L, 0.021 mmol) and allowed to stir at rt over 3 days. The reaction mixture was concentrated under vacuum, dissolved in methanol, and loaded onto a preconditioned SCX column (1 g). Methanol (10 mL) was then passed through the column followed by 2 M methanolic ammonia. The methanolic ammonia fractions were combined and concentrated under vacuum to give the title compound (3.7 mg, 9.75 Î¼mol, 47% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.42 min, [MH]<sup>+</sup> = 380.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.82 (d, <i>J</i> = 2.9 Hz, 1 H) 8.62 (d, <i>J</i> = 2.9 Hz, 1 H) 7.46 (br d, <i>J</i> = 8.3 Hz, 1 H) 7.19 (t, <i>J</i> = 7.6 Hz, 1 H) 7.09 (d, <i>J</i> = 7.3 Hz, 1 H) 5.61 (s, 2 H) 2.96 (s, 3 H) 2.79 (tt, <i>J</i> = 7.2, 3.8 Hz, 1 H) 2.63 (s, 3 H) 0.74â0.83 (m, 2 H) 0.58â0.65 (m, 2 H). <i>Exchangeable protons not observed</i>.</div></div></div><div id="sec3_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> 1-((1<i>H</i>-Indazol-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>53</b>)</h4><div id="sec3_3_20_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i134" class="anchor-spacer"></div><h5 class="article-section__title" id="_i134"> Step 1: 4-(Bromomethyl)-1<i>H</i>-indazole, hydrobromide (<b>105</b>)</h5><div class="NLM_p last">1<i>H</i>-Indazol-4-yl)methanol (202 mg, 1.363 mmol, commercially available from, for example, Apollo Scientific) and HBr (3.3 mL, 48% in water, 27.4 mmol) were heated at 80 Â°C for 2 h. The resulting suspension was allowed to cool to rt, filtered under vacuum, washed with cold water, and dried in a vacuum oven to give 4-(bromomethyl)-1<i>H</i>-indazole, hydrobromide (<b>105</b>, 213 mg, 0.657 mmol, 48% yield) as an off-white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.85 min, [MH]<sup>+</sup> = 211, 213. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.43 (d, <i>J</i> = 1.0 Hz, 1 H) 7.59 (d, <i>J</i> = 8.3 Hz, 1 H) 7.44 (dd, <i>J</i> = 8.3, 6.8 Hz, 1 H) 7.28 (d, <i>J</i> = 7.3 Hz, 1 H) 4.95 (s, 2 H).</div></div><div id="sec3_3_20_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i135" class="anchor-spacer"></div><h5 class="article-section__title" id="_i135"> Step 2: 1-((1<i>H</i>-Indazol-4-yl)methyl)-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>79</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (130 mg, 0.563 mmol), 4-(bromomethyl)-1<i>H</i>-indazole, hydrobromide (<b>105</b>, 214 mg, 0.731 mmol), using the same procedure as for <b>68</b> to give 1-((1<i>H</i>-indazol-4-yl)methyl)-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (234 mg, 0.453 mmol, 81% yield) as a colorless oil, which was used crude in the subsequent step. LCMS (2 min, formic): 74% purity, <i>R</i><sub><i>t</i></sub> = 0.78 min, [MH]<sup>+</sup> = 361, 363. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.42 (d, <i>J</i> = 2.9 Hz, 1 H) 8.19 (d, <i>J</i> = 1.0 Hz, 1 H) 8.17 (d, <i>J</i> = 2.9 Hz, 1 H) 7.53 (d, <i>J</i> = 8.3 Hz, 1 H) 7.36 (dd, <i>J</i> = 8.3, 6.8 Hz, 1 H) 7.01â7.06 (m, 1 H) 5.57 (s, 2 H) 2.94 (s, 3 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_3_20_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i136" class="anchor-spacer"></div><h5 class="article-section__title" id="_i136"> Step 3: 1-((1<i>H</i>-Indazol-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>53</b>)</h5><div class="NLM_p last">Prepared from <b>79</b> (140 mg, 0.388 mmol) and cyclopropanamine (0.05 mL, 0.722 mmol), using the same procedure as for <b>35</b> to give 1-((1<i>H</i>-indazol-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (10 mg, 0.025 mmol, 6% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.67 min, [MH]<sup>+</sup> = 366.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.85 (d, <i>J</i> = 2.9 Hz, 1 H) 8.57 (d, <i>J</i> = 2.4 Hz, 1 H) 8.53 (br s, 1 H) 8.19 (s, 1 H) 7.56 (d, <i>J</i> = 8.8 Hz, 1 H) 7.40 (dd, <i>J</i> = 8.1, 7.1 Hz, 1 H) 7.07 (d, <i>J</i> = 7.3 Hz, 1 H) 5.69 (s, 2 H) 2.97 (s, 3 H) 2.77â2.83 (m, 1 H) 0.75â0.81 (m, 2 H) 0.58â0.63 (m, 2 H). <i>Two exchangeable protons not observed</i>.</div></div></div><div id="sec3_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> 1-((1<i>H</i>-Indazol-7-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>54</b>)</h4><div id="sec3_3_21_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i138" class="anchor-spacer"></div><h5 class="article-section__title" id="_i138"> Step 1: 7-(Bromomethyl)-1<i>H</i>-indazole, Hydrobromide (<b>106</b>)</h5><div class="NLM_p last">(1<i>H</i>-Indazol-7-yl)methanol (250 mg, 1.687 mmol, commercially available from, for example, Fluorochem) and HBr (4 mL, 48% in water, 33.2 mmol) were heated at 80 Â°C for 1 h. The suspension was allowed to cool to rt, filtered under vacuum, washed with cold water, and dried in a vacuum oven to give 7-(bromomethyl)-1<i>H</i>-indazole, hydrobromide (<b>106</b>, 449 mg, 1.307 mmol, 77% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.84 min, [MH]<sup>+</sup> = 211, 213. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.15 (s, 1 H) 7.77 (dd, <i>J</i> = 7.8, 1.0 Hz, 1 H) 7.45 (d, <i>J</i> = 7.8 Hz, 1 H) 7.11 (dd, <i>J</i> = 8.1, 7.1 Hz, 1 H) 5.05 (s, 2 H). <i>Exchangeable proton not observed</i>.</div></div><div id="sec3_3_21_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i139" class="anchor-spacer"></div><h5 class="article-section__title" id="_i139"> Step 2: 1-((1<i>H</i>-Indazol-7-yl)methyl)-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (<b>80</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (312 mg, 1.350 mmol), 7-(bromomethyl)-1<i>H</i>-indazole, hydrobromide (<b>106</b>, 415 mg, 1.421 mmol), using the same procedure as for <b>68</b> to give 1-((1<i>H</i>-indazol-7-yl)methyl)-5-bromo-<i>N</i>-methyl-2-oxo-1,2-dihydropyridine-3-carboxamide (396 mg, 0.877 mmol, 65% yield) as an off-white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.86 min, [MH]<sup>+</sup> = 361, 363. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.48 (d, <i>J</i> = 2.9 Hz, 1 H) 8.29 (d, <i>J</i> = 2.4 Hz, 1 H) 8.12 (s, 1 H) 7.81 (d, <i>J</i> = 8.3 Hz, 1 H) 7.34 (d, <i>J</i> = 7.3 Hz, 1 H) 7.13â7.23 (m, 1 H) 5.57 (s, 2 H) 2.97 (s, 3 H). <i>Exchangeable protons not observed</i>.</div></div><div id="sec3_3_21_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i140" class="anchor-spacer"></div><h5 class="article-section__title" id="_i140"> Step 3: 1-((1<i>H</i>-Indazol-7-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>54</b>)</h5><div class="NLM_p last">Prepared from <b>80</b> (204 mg, 0.565 mmol) and cyclopropanamine (0.08 mL, 1.155 mmol), using the same procedure as for <b>35</b> to give 1-((1<i>H</i>-indazol-7-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (38 mg, 0.094 mmol, 17% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.75 min, [MH]<sup>+</sup> = 366.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 9.75 (br s, 1 H) 8.86 (d, <i>J</i> = 2.9 Hz, 1 H) 8.62 (d, <i>J</i> = 2.4 Hz, 1 H) 8.13 (s, 1 H) 7.80 (d, <i>J</i> = 8.3 Hz, 1 H) 7.33 (d, <i>J</i> = 7.3 Hz, 1 H) 7.17 (t, <i>J</i> = 7.6 Hz, 1 H) 5.63 (s, 2 H) 2.98 (d, <i>J</i> = 4.9 Hz, 3 H) 2.74â2.85 (m, 1 H) 0.71â0.83 (m, 2 H) 0.53â0.65 (m, 2 H). <i>Two exchangeable protons not observed</i>.</div></div></div><div id="sec3_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> 1-(Benzofuran-4-ylmethyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>55</b>)</h4><div id="sec3_3_22_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i142" class="anchor-spacer"></div><h5 class="article-section__title" id="_i142"> Step 1: 4-(Bromomethyl)benzofuran (<b>125</b>)</h5><div class="NLM_p last">Benzofuran-4-ylmethanol (35 mg, 0.236 mmol) was dissolved in diethyl ether (1 mL) and DCM (1 mL) at 0 Â°C under N<sub>2</sub>. PBr<sub>3</sub> (0.04 mL, 0.424 mmol) was added dropwise, and the reaction was stirred at rt under N<sub>2</sub>. After 30 min, TLC (eluting with 50:50 EtOAc:water) showed complete conversion to a nonpolar product. The solution was quenched with water (10 mL) and extracted with diethyl ether (3 Ã 20 mL), dried over a hydrophobic frit, and concentrated to give 4-(bromomethyl)benzofuran (41 mg, 0.117 mmol, 49% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.15 min, product does not ionize at correct [MH]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 7.73 (d, <i>J</i> = 2.4 Hz, 1 H) 7.47â7.54 (m, 1 H) 7.27â7.29 (obs m, 2 H) 6.98 (dd, <i>J</i> = 2.4, 1.0 Hz, 1 H) 4.77 (s, 2 H).</div></div><div id="sec3_3_22_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i143" class="anchor-spacer"></div><h5 class="article-section__title" id="_i143"> Step 2: 1-(Benzofuran-4-ylmethyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>55</b>)</h5><div class="NLM_p last">Prepared from <b>87</b> (44 mg, 0.187 mmol) and 4-(bromomethyl)benzofuran (<b>105</b>, 45 mg, 0.213 mmol), using the same procedure as for <b>93</b> to give 1-(benzofuran-4-ylmethyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (13 mg, 0.032 mmol, 17% yield) as a colorless oil. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.88 min, [MH]+ = 366.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.83 (d, <i>J</i> = 2.4 Hz, 1 H) 8.53 (d, <i>J</i> = 2.9 Hz, 1 H) 7.81 (d, <i>J</i> = 2.4 Hz, 1 H) 7.52 (d, <i>J</i> = 8.3 Hz, 1 H) 7.32 (t, <i>J</i> = 7.8 Hz, 1 H) 7.19 (d, <i>J</i> = 7.3 Hz, 1 H) 7.01 (d, <i>J</i> = 2.0 Hz, 1 H) 5.59 (s, 2 H) 2.97 (s, 3 H) 2.80 (tt, <i>J</i> = 7.2, 3.8 Hz, 1 H) 0.74â0.84 (m, 2 H) 0.56â0.65 (m, 2 H). <i>Exchangeable protons not observed</i>.</div></div></div><div id="sec3_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> 1-((1<i>H</i>-Indol-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>56</b>)</h4><div id="sec3_3_23_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i145" class="anchor-spacer"></div><h5 class="article-section__title" id="_i145"> Step 1: Methyl 1-Tosyl-1<i>H</i>-indole-4-carboxylate (<b>102</b>)</h5><div class="NLM_p last">Methyl 1<i>H</i>-indole-4-carboxylate (750 mg, 4.28 mmol, commercially available from, for example, Sigma-Aldrich) was dissolved in DMF (13.591 mL) at 0 Â°C. Sodium hydride (205 mg, 5.14 mmol, 60% dispersion in mineral oil) was added in portions. The reaction was stirred at 0 Â°C for 10 min before warming to rt and stirring for 30 min. Tosyl-Cl (979 mg, 5.14 mmol) was added, and the reaction mixture was stirred at rt for 10 min. The reaction was cooled to 0 Â°C and quenched by the dropwise addition of water (3.86 mL, 214 mmol) before pouring onto saturated aqueous lithium chloride (140 mL). The product was extracted with ethyl acetate (3 Ã 30 mL), and the combined organic portions were dried through a hydrophobic frit and evaporated in vacuo to yield the crude product (2056 mg). The residue was dry loaded onto a 50 g SNAP silica cartridge and purified via Biotage SP4 flash chromatography, eluting from 0 to 25% ethyl acetate:cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield the pure product, methyl 1-tosyl-1<i>H</i>-indole-4-carboxylate (<b>102</b>, 1039 mg, 3.15 mmol, 74% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.30 min, [MH]+ = 330.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 8.23 (dt, <i>J</i> = 8.3, 1.0 Hz, 1 H) 7.99 (dd, <i>J</i> = 7.8, 1.0 Hz, 1 H) 7.74â7.80 (m, 2 H) 7.71 (d, <i>J</i> = 3.4 Hz, 1 H) 7.33â7.42 (m, 2 H) 7.24 (d, <i>J</i> = 7.8 Hz, 2 H) 3.97 (s, 3 H) 2.36 (s, 3 H).</div></div><div id="sec3_3_23_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i146" class="anchor-spacer"></div><h5 class="article-section__title" id="_i146"> Step 2: (1-Tosyl-1<i>H</i>-indol-4-yl)methanol (<b>103</b>)</h5><div class="NLM_p last">A solution of methyl 1-tosyl-1<i>H</i>-indole-4-carboxylate (<b>102</b>, 1016 mg, 3.08 mmol) in DCM (30 mL) was cooled to â78 Â°C, and DIBAL-H (1 M in toluene, 13.57 mL, 13.57 mmol) was added dropwise over 1 h. The reaction mixture was stirred for a further 1.5 h, followed by a further 40 min. The reaction was quenched with methanol (0.125 mL, 3.08 mmol) when still at â78 Â°C and then allowed to warm to ambient temperature. The reaction was diluted with saturated Rochelleâs salt solution (60 mL) and stirred for 16 h. The layers were separated, and the aqueous phase was extracted with dichloromethane (2 Ã 50 mL). The combined organic layers were dried through a hydrophobic frit and evaporated in vacuo to yield the crude product (913 mg). The residue was loaded in dichloromethane onto a 50 g SNAP cartridge and purified via Biotage SP4, eluting from 15 to 75% ethyl acetate/cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield the pure product, (1-tosyl-1<i>H</i>-indol-4-yl)methanol (<b>103</b>, 901 mg, 2.84 mmol, 92% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.07 min, [M + Na]+ = 324.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 7.97 (d, <i>J</i> = 8.3 Hz, 1 H) 7.75â7.82 (m, 2 H) 7.62 (d, <i>J</i> = 3.4 Hz, 1 H) 7.29â7.35 (m, 1 H) 7.21â7.27 (m, 3 H) 6.81â6.87 (m, 1 H) 4.91 (d, <i>J</i> = 5.4 Hz, 2 H) 2.36 (s, 3 H) 1.69 (t, <i>J</i> = 5.9 Hz, 1 H).</div></div><div id="sec3_3_23_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i147" class="anchor-spacer"></div><h5 class="article-section__title" id="_i147"> Step 3: 4-(Bromomethyl)-1-tosyl-1<i>H</i>-indole (<b>104</b>)</h5><div class="NLM_p last">(1-Tosyl-1<i>H</i>-indol-4-yl)methanol (<b>103</b>, 500 mg, 1.659 mmol) and HBr (3995 Î¼L, 48% in water, 33.2 mmol) were heated at 80 Â°C monitoring by LCMS. Initial LCMS indicated formation of product. and the reaction was heated for a further 4 h. The reaction mixture was poured onto water (10 mL), and the product was extracted with dichloromethane (3 Ã 20 mL). The combined organic portions were dried through a hydrophobic frit and evaporated in vacuo to yield the crude product, 4-(bromomethyl)-1-tosyl-1<i>H</i>-indole (<b>104</b>, 564 mg, 1.316 mmol, 79% yield) as a purple solid, which was used without further purification. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.35 min, [M-H]- = 362.0, 364.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 7.90â8.04 (m, 1 H) 7.72â7.86 (m, 2 H) 7.68 (d, <i>J</i> = 3.9 Hz, 1 H) 7.21â7.29 (m, 4 H) 6.85 (d, <i>J</i> = 4.4 Hz, 1 H) 4.71 (s, 2 H) 2.37 (s, 3 H).</div></div><div id="sec3_3_23_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i148" class="anchor-spacer"></div><h5 class="article-section__title" id="_i148"> Step 4: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-4-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>93</b>)</h5><div class="NLM_p last">To a solution of <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>87</b>, 100 mg, 0.425 mmol) in DMF (4.2 mL) was added potassium carbonate (118 mg, 0.850 mmol) and 4-(bromomethyl)-1-tosyl-1<i>H</i>-indole (<b>104</b>, 232 mg, 0.638 mmol), and the reaction was stirred at rt for 2.5 h. The reaction was quenched with water (77 Î¼L, 4.25 mmol), poured onto water (60 mL), and the aqueous layer extracted with ethyl acetate (3 Ã 15 mL). The combined organic layers were washed with brine (2 Ã 5 mL), dried through a hydrophobic frit, and evaporated in vacuo. The residue was loaded in dichloromethane onto a 25 g SNAP cartridge and purified via Biotage SP4 flash chromatography, eluting from 12 to 62% (3:1 ethyl acetate:ethanol)/cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-4-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (226 mg, 0.414 mmol, 97% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.11 min, [MH]+ = 519.3. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.46â9.63 (m, 1 H) 8.70 (d, <i>J</i> = 2.4 Hz, 1 H) 8.40 (d, <i>J</i> = 2.9 Hz, 1 H) 8.00 (d, <i>J</i> = 8.3 Hz, 1 H) 7.75â7.88 (m, 2 H) 7.64 (d, <i>J</i> = 3.4 Hz, 1 H) 7.28â7.34 (m, 1 H) 7.26 (d, <i>J</i> = 7.8 Hz, 2 H) 7.11 (d, <i>J</i> = 6.8 Hz, 1 H) 6.78 (dd, <i>J</i> = 3.9, 1.0 Hz, 1 H) 6.56 (br s, 1 H) 5.47 (s, 2 H) 2.99 (d, <i>J</i> = 4.9 Hz, 3 H) 2.79â2.88 (m, 1 H) 2.37 (s, 3 H) 0.78â0.90 (m, 2 H) 0.53â0.66 (m, 2 H).</div></div><div id="sec3_3_23_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i149" class="anchor-spacer"></div><h5 class="article-section__title" id="_i149"> Step 5: 1-((1<i>H</i>-Indol-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>56</b>)</h5><div class="NLM_p last">To a solution of <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-4-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>93</b>, 216 mg, 0.417 mmol) in methanol (771 Î¼L) and THF (1543 Î¼L) stirred at room temperature was added solid cesium carbonate (543 mg, 1.666 mmol) in one charge. The reaction mixture was stirred at 70 Â°C for 30 min. The reaction mixture was concentrated in vacuo and taken up in ethyl acetate (15 mL) and water (30 mL). The aqueous layer was extracted with ethyl acetate (2 Ã 15 mL), and the combined organics were washed with brine (10 mL), dried through a hydrophobic frit, and evaporated in vacuo to yield the desired product 1-((1<i>H</i>-indol-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (108 mg, 0.282 mmol, 68% yield). LCMS (2 min, high pH): <i>R</i><sub><i>t</i></sub> = 0.82 min, [MH]+ = 365.2. HRMS (C<sub>20</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>): [M + H]<sup>+</sup> calculated 365.1614, found 365.1606. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.25 (br s, 1 H) 9.45 (q, <i>J</i> = 4.6 Hz, 1 H) 8.81 (d, <i>J</i> = 2.9 Hz, 1 H) 8.58 (d, <i>J</i> = 2.9 Hz, 1 H) 8.54 (d, <i>J</i> = 3.9 Hz, 1 H) 7.35â7.41 (m, 2 H) 7.04â7.10 (m, 1 H) 6.83 (d, <i>J</i> = 6.4 Hz, 1 H) 6.49â6.54 (m, 1 H) 5.55 (s, 2 H) 2.84 (d, <i>J</i> = 4.9 Hz, 3 H) 2.78 (tq, <i>J</i> = 7.4, 3.9 Hz, 1 H) 0.63â0.69 (m, 2 H) 0.51â0.57 (m, 2 H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 164.3, 163.7, 161.7, 144.4, 141.0, 136.4, 127.5, 126.5, 126.3, 121.4, 119.6, 118.7, 113.7, 112.0, 99.5, 51.0, 26.4, 23.5, 6.0 (2C).</div></div></div><div id="sec3_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> 1-((1<i>H</i>-Indol-7-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>57</b>)</h4><div id="sec3_3_24_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i151" class="anchor-spacer"></div><h5 class="article-section__title" id="_i151"> Step 1: Methyl 1-Tosyl-1<i>H</i>-indole-7-carboxylate (<b>107</b>)</h5><div class="NLM_p last">Methyl 1<i>H</i>-indole-7-carboxylate (1 g, 5.71 mmol, commercially available from, for example, Apollo Scientific) was dissolved in DMF (18.12 mL) at 0 Â°C. Sodium hydride (0.251 g, 60% dispersion in mineral oil, 6.28 mmol) was added in portions. The reaction was stirred at 0 Â°C for 10 min before warming to rt and stirring for 30 min. Tosyl-Cl (1.197 g, 6.28 mmol) was added, and the reaction mixture was stirred for 2 h. The reaction was cooled back down to 0 Â°C, and a further portion of sodium hydride (0.114 g, 60% dispersion in mineral oil, 2.85 mmol) was added portionwise. The reaction mixture was stirred for 10 min before warming to rt and stirring for 30 min. An additional portion of tosyl-Cl (0.544 g, 2.85 mmol) was added at this point. The reaction was stirred for a further 1.5 h. The reaction was quenched by the dropwise addition of water (5.14 mL, 285 mmol). The reaction was poured onto saturated aqueous lithium chloride (100 mL), and the product was extracted with ethyl acetate (3 Ã 30 mL). The combined organic portions were dried through a hydrophobic frit and evaporated in vacuo to yield the crude product (2158 mg). The residue was dry loaded onto a 50 g SNAP silica cartridge and purified via Biotage SP4 flash chromatography, eluting from 0â25% ethyl acetate/cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield the pure product, methyl 1-tosyl-1<i>H</i>-indole-7-carboxylate (<b>107</b>, 1159 mg, 3.52 mmol, 62% yield) as a yellow solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.18 min, [MH]<sup>+</sup> = 330.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 7.57â7.63 (m, 3 H) 7.53 (dd, <i>J</i> = 7.3, 1.0 Hz, 1 H) 7.50 (d, <i>J</i> = 3.4 Hz, 1 H) 7.28â7.33 (m, 1 H) 7.15â7.21 (m, 2 H) 6.69 (d, <i>J</i> = 3.9 Hz, 1 H) 3.96 (s, 3 H) 2.35 (s, 3 H).</div></div><div id="sec3_3_24_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i152" class="anchor-spacer"></div><h5 class="article-section__title" id="_i152"> Step 2: (1-Tosyl-1<i>H</i>-indol-7-yl)methanol (<b>108</b>)</h5><div class="NLM_p last">A solution of methyl 1-tosyl-1<i>H</i>-indole-7-carboxylate (<b>107</b>, 1117 mg, 3.39 mmol) in DCM (33.913 mL) was cooled to â78 Â°C, and DIBAL-H (14.92 mL, 1 M in toluene, 14.92 mmol) was added dropwise over 15 min. The reaction mixture was stirred for 1.5 h more. The reaction was quenched with methanol (6.04 mL, 149 mmol) when still at â78 Â°C and then allowed to warm to ambient temperature. The reaction was diluted with Rochelleâs salt solution (60 mL) and stirred for 16 h. The layers were separated, and the aqueous phase was extracted with dichloromethane (2 Ã 50 mL). The combined organic layers were dried through a hydrophobic frit and evaporated in vacuo to yield the crude product (1065 mg). The residue was loaded in dichloromethane and purified via Biotage SP4, eluting from 10â50% ethyl acetate/cyclohexane. The relevant fractions were combined and evaporated in vacuo to yield a clear oil. The product was air-dried to yield (1-tosyl-1<i>H</i>-indol-7-yl)methanol (<b>108</b>, 901 mg, 2.84 mmol, 84% yield). LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.07 min, [M â H]<sup>â</sup> = 300.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 7.72 (d, <i>J</i> = 3.9 Hz, 1 H) 7.53â7.58 (m, 2 H) 7.51 (dd, <i>J</i> = 7.6, 1.2 Hz, 1 H) 7.36 (dd, <i>J</i> = 7.3, 1.5 Hz, 1 H) 7.20â7.30 (obs m, 3 H) 6.75 (d, <i>J</i> = 3.4 Hz, 1 H) 4.91 (d, <i>J</i> = 7.3 Hz, 2 H) 3.23 (t, <i>J</i> = 7.3 Hz, 1 H) 2.38 (s, 3 H).</div></div><div id="sec3_3_24_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i153" class="anchor-spacer"></div><h5 class="article-section__title" id="_i153"> Step 3: 7-(Bromomethyl)-1-tosyl-1<i>H</i>-indole (<b>109</b>)</h5><div class="NLM_p last">(1-Tosyl-1<i>H</i>-indol-7-yl)methanol (<b>108</b>, 500 mg, 1.659 mmol) and HBr (3995 Î¼L, 48% in water, 33.2 mmol) were heated at 80 Â°C for 1 h. The reaction mixture was filtered through a sinter funnel and washed with water. The collected precipitate was dissolved in dichloromethane (100 mL), dried through a hydrophobic frit, and evaporated in vacuo to yield the crude product 7-(bromomethyl)-1-tosyl-1<i>H</i>-indole (<b>109</b>, 602 mg, 1.322 mmol, 80% yield) as a deep-red oil, which was used without further purification. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.34 min, [MH]<sup>+</sup> = 364.0, 366.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 7.75 (d, <i>J</i> = 3.9 Hz, 1 H) 7.58â7.64 (m, 2 H) 7.49 (dd, <i>J</i> = 7.8, 1.0 Hz, 1 H) 7.43 (dd, <i>J</i> = 7.6, 1.2 Hz, 1 H) 7.21â7.29 (obs m, 3 H) 6.72 (d, <i>J</i> = 3.4 Hz, 1 H) 5.18 (s, 2 H) 2.37 (s, 3 H).</div></div><div id="sec3_3_24_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i154" class="anchor-spacer"></div><h5 class="article-section__title" id="_i154"> Step 4: 5-Bromo-<i>N</i>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-7-yl)methyl)-1,2-dihydropyridine-3-carboxamide (<b>81</b>)</h5><div class="NLM_p last">Prepared from <b>67</b> (300 mg, 1.298 mmol) and 7-(bromomethyl)-1-tosyl-1<i>H</i>-indole (<b>109</b>, 568 mg, 1.558 mmol), using the same procedure as for <b>68</b> to give 5-bromo-<i>N</i>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-7-yl)methyl)-1,2-dihydropyridine-3-carboxamide (326 mg, 0.570 mmol, 44% yield) as an orange solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.22 min, [MH]<sup>+</sup> = 513.9, 515.9. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.59 (br s, 1 H) 8.57 (d, <i>J</i> = 2.4 Hz, 1 H) 7.81 (d, <i>J</i> = 3.9 Hz, 1 H) 7.58 (dd, <i>J</i> = 7.6, 1.2 Hz, 1 H) 7.50â7.55 (m, 2 H) 7.21â7.28 (m, 3 H) 7.19 (d, <i>J</i> = 2.9 Hz, 1 H) 6.87 (d, <i>J</i> = 6.8 Hz, 1 H) 6.79 (d, <i>J</i> = 3.4 Hz, 1 H) 5.68 (s, 2 H) 3.01 (d, <i>J</i> = 4.9 Hz, 3 H) 2.38 (s, 3 H).</div></div><div id="sec3_3_24_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i155" class="anchor-spacer"></div><h5 class="article-section__title" id="_i155"> Step 5: <i>N</i><sup>5</sup>-Cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-7-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (<b>82</b>)</h5><div class="NLM_p last">Prepared from <b>81</b> (297 mg, 0.577 mmol) and cyclopropylamine (81 Î¼L, 1.155 mmol), using the same procedure as for <b>35</b> to give <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1-((1-tosyl-1<i>H</i>-indol-7-yl)methyl)-1,2-dihydropyridine-3,5-dicarboxamide (251 mg, 0.290 mmol, 50% yield) as an orange solid, which was used without further purification in subsequent chemistry. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.09 min, [MH]<sup>+</sup> = 519.0. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.54 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.72 (d, <i>J</i> = 2.4 Hz, 1 H) 8.10 (d, <i>J</i> = 2.9 Hz, 1 H) 7.76 (d, <i>J</i> = 3.4 Hz, 1 H) 7.52â7.56 (m, 3 H) 7.17â7.26 (m, 3 H) 6.75â6.79 (m, 2 H) 6.48 (br s, 1 H) 5.74 (s, 2 H) 3.00 (d, <i>J</i> = 4.9 Hz, 3 H) 2.85 (tq, <i>J</i> = 7.1, 3.6 Hz, 1 H) 2.36 (s, 3 H) 0.82â0.89 (m, 2 H) 0.59â0.65 (m, 2 H).</div></div><div id="sec3_3_24_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i156" class="anchor-spacer"></div><h5 class="article-section__title" id="_i156"> Step 6: 1-((1<i>H</i>-Indol-7-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>57</b>)</h5><div class="NLM_p last">Prepared from <b>82</b> (124 mg, 0.120 mmol), using the same procedure as for <b>56</b> to give 1-((1<i>H</i>-indol-7-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (18.8 mg, 0.049 mmol, 41% yield) as a white solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.91 min, [MH]<sup>+</sup> = 365.1. <sup>1</sup>H NMR (400 MHz, CHCl<sub>3</sub>-<i>d</i>) Î´ ppm 9.94 (br s, 1 H) 9.56 (br d, <i>J</i> = 4.9 Hz, 1 H) 8.69 (d, <i>J</i> = 2.4 Hz, 1 H) 8.61 (d, <i>J</i> = 2.4 Hz, 1 H) 7.69 (d, <i>J</i> = 7.8 Hz, 1 H) 7.27â7.32 (obs m, 2 H) 7.13 (dd, <i>J</i> = 8.1, 7.1 Hz, 1 H) 6.58 (dd, <i>J</i> = 3.2, 2.2 Hz, 1 H) 6.46 (br s, 1 H) 5.52 (s, 2 H) 3.06 (d, <i>J</i> = 4.9 Hz, 3 H) 2.82â2.90 (m, 1 H) 0.83â0.89 (m, 2 H) 0.57â0.63 (m, 2 H).</div></div></div><div id="sec3_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i157" class="anchor-spacer"></div><h4 class="article-section__title" id="_i157"> 1-((1<i>H</i>-Pyrrolo[3,2-<i>c</i>]pyridin-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>58</b>)</h4><div id="sec3_3_25_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i158" class="anchor-spacer"></div><h5 class="article-section__title" id="_i158"> Step 1: (1<i>H</i>-Pyrrolo[3,2-<i>c</i>]pyridin-4-yl)methanol, Hydrochloride (<b>126</b>)</h5><div class="NLM_p last">To a solution of 1<i>H</i>-pyrrolo[3,2-<i>c</i>]pyridine-4-carboxylic acid (400 mg, 2.47 mmol, commercially available from, for example, Sigma-Aldrich) in THF (16 mL), was added borane tetrahydrofuran complex (1 M in THF, 4.93 mL, 4.93 mmol) at rt. The reaction was stirred at rt for 3 h. The reaction was then quenched with methanol (0.998 mL, 24.67 mmol) and hydrochloric acid (1M, 3.08 mL, 6.17 mmol) and stirred for 1 h at rt. The reaction mixture was left to stand overnight. A precipitate was noted in the reaction mixture, which was filtered off to yield (1<i>H</i>-pyrrolo[3,2-<i>c</i>]pyridin-4-yl)methanol, hydrochloride (306 mg, 1.33 mmol, 54% yield) as an off-white solid which was used without further purification. LCMS (2 min High pH): <i>R</i><sub><i>t</i></sub> = 0.43 min, [MH]<sup>+</sup> = 149.1.</div></div><div id="sec3_3_25_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i159" class="anchor-spacer"></div><h5 class="article-section__title" id="_i159"> Step 2: 1-((1<i>H</i>-Pyrrolo[3,2-<i>c</i>]pyridin-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>58</b>)</h5><div class="NLM_p last">To a suspension of <i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>87</b>, 55 mg, 0.23 mmol) in acetonitrile (2.3 mL) was added (1<i>H</i>-pyrrolo[3,2-<i>c</i>]pyridin-4-yl)methanol hydrochloride (86 mg, 0.468 mmol), triphenylphosphine (184 mg, 0.70 mmol), triethylamine (0.068 mL, 0.49 mmol), and DIAD (0.136 mL, 0.701 mmol). The reaction was stirred at rt for 5 h. Further portions of triphenylphosphine (184 mg, 0.70 mmol) and DIAD (0.136 mL, 0.70 mmol) were added. After 21 h, the reaction mixture was poured onto saturated aqueous sodium bicarbonate (10 mL) and extracted with ethyl acetate (3 Ã 15 mL). The combined organics were dried through a hydrophobic frit and evaporated in vacuo to yield the crude product (727 mg). The residue was dry loaded in methanol onto a 50 g SNAP cartridge and purified by Biotage SP4 flash chromatography, eluting from 18 to 88% (80:20 DCM:2 M methanolic ammonia)/DCM. Poor separation was achieved, and all fractions containing product were recombined and evaporated in vacuo to yield a clear glass (41 mg). The sample was dissolved in 1:1 MeOH:DMSO (1 mL) and purified by MDAP (high pH). The solvent was dried under a stream of nitrogen to give 1-((1<i>H</i>-pyrrolo[3,2-<i>c</i>]pyridin-4-yl)methyl)-<i>N</i><sup>5</sup>-cyclopropyl-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (3.2 mg, 8.32 Î¼mol, 4% yield) as a white solid. LCMS (2 min high pH): <i>R</i><sub><i>t</i></sub> = 0.66 min, [MH]<sup>+</sup> = 366.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.61 (br s, 1 H) 9.32 (q, <i>J</i> = 4.7 Hz, 1 H) 8.84 (d, <i>J</i> = 2.9 Hz, 1 H) 8.73 (d, <i>J</i> = 2.4 Hz, 1 H) 8.56 (d, <i>J</i> = 4.4 Hz, 1 H) 8.01 (d, <i>J</i> = 5.4 Hz, 1 H) 7.49 (dd, <i>J</i> = 2.9, 2.0 Hz, 1 H) 7.27â7.36 (m, 1 H) 6.67 (d, <i>J</i> = 2.9 Hz, 1 H) 5.66 (s, 2 H) 2.84 (tq, <i>J</i> = 7.4, 3.9 Hz, 1 H) 2.79 (d, <i>J</i> = 4.9 Hz, 3 H) 0.66â0.72 (m, 2 H) 0.55â0.60 (m, 2 H).</div></div></div><div id="sec3_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i160" class="anchor-spacer"></div><h4 class="article-section__title" id="_i160"> <i>N</i><sup>5</sup>-Cyclopropyl-1-(indolin-4-ylmethyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>59</b>)</h4><div id="sec3_3_26_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i161" class="anchor-spacer"></div><h5 class="article-section__title" id="_i161"> Step 1: <i>tert</i>-Butyl 4-(Hydroxymethyl)indoline-1-carboxylate (<b>127</b>)</h5><div class="NLM_p last">Indolin-4-ylmethanol (301 mg, 2.018 mmol, commercially available from, for example, Fluorochem) was dissolved in DCM (5 mL), Boc-anhydride (660 mg, 3.03 mmol) was added, and the reaction was stirred at rt under N<sub>2</sub> for 2 h. The reaction was diluted with satd aq sodium bicarbonate (10 mL), extracted with DCM (2 Ã 10 mL), dried over a hydrophobic frit, and concentrated to give an orange oil (776 mg). This was purified by chromatography on SiO<sub>2</sub> (Biotage SNAP 50 g cartridge, eluting with 0â50% EtOAc/cyclohexane). The appropriate fractions were concentrated to give <i>tert</i>-butyl 4-(hydroxymethyl)indoline-1-carboxylate (472 mg, 1.704 mmol, 84% yield) as a colorless oil. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.00 min, [MH]<sup>+</sup> = 194.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.52 (br s, 1 H) 7.12 (t, <i>J</i> = 7.7 Hz, 1 H) 6.95 (d, <i>J</i> = 7.6 Hz, 1 H) 5.05 (t, <i>J</i> = 5.5 Hz, 1 H) 4.42 (d, <i>J</i> = 5.4 Hz, 2 H) 3.91 (t, <i>J</i> = 8.7 Hz, 2 H) 3.00 (t, <i>J</i> = 8.7 Hz, 2 H) 1.50 (s, 9 H).</div></div><div id="sec3_3_26_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i162" class="anchor-spacer"></div><h5 class="article-section__title" id="_i162"> Step 2: <i>tert</i>-Butyl 4-((5-(Cyclopropylcarbamoyl)-3-(methylcarbamoyl)-2-oxopyridin-1(2<i>H</i>)-yl)methyl)indoline-1-carboxylate (<b>92</b>)</h5><div class="NLM_p last">Prepared from <b>87</b> (50 mg, 0.213 mmol) and <i>tert</i>-butyl 4-(hydroxymethyl)indoline-1-carboxylate (<b>127</b>, 79 mg, 0.319 mmol), using the same procedure as for <b>91</b> to give <i>tert</i>-butyl 4-((5-(cyclopropylcarbamoyl)-3-(methylcarbamoyl)-2-oxopyridin-1(2<i>H</i>)-yl)methyl)indoline-1-carboxylate (41 mg, 0.075 mmol, 35% yield) as a colorless oil. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 1.08 min, [MH]+ = 467.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.83 (d, <i>J</i> = 2.7 Hz, 1 H) 8.47 (d, <i>J</i> = 2.7 Hz, 1 H) 7.60â7.71 (m, 1 H) 7.14 (t, <i>J</i> = 7.9 Hz, 1 H) 6.73 (d, <i>J</i> = 7.6 Hz, 1 H) 5.24 (s, 2 H) 4.00 (t, <i>J</i> = 8.7 Hz, 2 H) 3.14 (t, <i>J</i> = 8.7 Hz, 2 H) 2.93 (s, 3 H) 2.80 (tt, <i>J</i> = 7.3, 3.7 Hz, 1 H) 1.56 (s, 9 H) 0.74â0.82 (m, 2 H) 0.58â0.65 (m, 2 H).</div></div><div id="sec3_3_26_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i163" class="anchor-spacer"></div><h5 class="article-section__title" id="_i163"> Step 3: <i>N</i><sup>5</sup>-Cyclopropyl-1-(indolin-4-ylmethyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (<b>59</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl 4-((5-(cyclopropylcarbamoyl)-3-(methylcarbamoyl)-2-oxopyridin-1(2<i>H</i>)-yl)methyl)indoline-1-carboxylate (<b>92</b>, 43 mg, 0.092 mmol) and TFA (0.5 mL, 6.49 mmol) were stirred at rt in DCM for 30 min (2 mL). The reaction mixture was concentrated and loaded onto a SCX cartridge (5 g, preconditioned with MeOH) and eluted with MeOH (20 mL), followed by 2 M NH<sub>3</sub> in MeOH (20 mL). The ammonia fractions containing product were combined and concentrated to give <i>N</i><sup>5</sup>-cyclopropyl-1-(indolin-4-ylmethyl)-<i>N</i><sup>3</sup>-methyl-2-oxo-1,2-dihydropyridine-3,5-dicarboxamide (30 mg, 0.074 mmol, 80% yield) as a yellow solid. LCMS (2 min, formic): <i>R</i><sub><i>t</i></sub> = 0.45 min, [MH]+ = 367.2. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.81 (d, <i>J</i> = 2.7 Hz, 1 H) 8.43 (d, <i>J</i> = 2.7 Hz, 1 H) 6.97 (t, <i>J</i> = 7.8 Hz, 1 H) 6.61 (d, <i>J</i> = 7.8 Hz, 1 H) 6.47 (d, <i>J</i> = 7.6 Hz, 1 H) 5.22 (s, 2 H) 3.49 (t, <i>J</i> = 8.4 Hz, 2 H) 2.98 (t, <i>J</i> = 8.4 Hz, 2 H) 2.94 (s, 3 H) 2.79 (tt, <i>J</i> = 7.3, 3.8 Hz, 1 H) 0.74â0.81 (m, 2 H) 0.58â0.63 (m, 2 H).</div></div></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i164" class="anchor-spacer"></div><h3 class="article-section__title" id="_i164"> X-ray Diffraction Study of <b>20</b></h3><div class="NLM_p">Data were collected with an Oxford Diffraction Gemini A Ultra diffractometer at 150(2)K using Cu KÎ± X-radiation (Î» = 1.54178 Ã).</div><div class="NLM_p">Crystal data and refinement summary for <b>20</b>: C<sub>18</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>; M = 325.36; colorless needle from the slow evaporation of a solution of <b>20</b> in methanol and water; 0.24 Ã 0.04 Ã 0.01 mm<sup>3</sup>; monoclinic; space group <i>P</i>2<sub>1</sub>/<i>n</i> (alt <i>P</i>2<sub>1</sub>/<i>c</i>, no. 14); <i>a</i> = 13.4672(4) Ã, <i>b</i> = 5.79643(15) Ã, <i>c</i> = 20.7574(6) Ã, Î² = 97.299(3)Â°, <i>V</i> = 1607.23(8) Ã<sup>3</sup>; <i>Z</i> = 4; <i>D</i><sub>calc</sub> = 1.345 Mg m<sup>â3</sup>; <i>Î¸</i><sub>max</sub> = 67.00Â°; reflections collected = 11968; independent reflections = 2833; <i>R</i><sub>int</sub> = 0.0438; coverage = 99.1%; restraints = 0; parameters = 227; <i>S</i> = 1.032; <i>R</i><sub>1</sub> [<i>I</i> > 2Ï(<i>I</i>)] = 0.0384; <i>wR</i><sub>2</sub> (all data) = 0.0940; largest difference peak and hole = 0.240 and â0.190 eÃ<sup>â3</sup>.</div><div class="NLM_p last">A description of the refinement and the full tables associated with the crystal structure are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf" class="ext-link">Supporting Information</a>. A crystallographic information file has been deposited with the Cambridge Crystallographic Data Centre. CCDC 2001550 contains the supplementary crystallographic data for this paper. The data can be obtained free of charge from The Cambridge Crystallographic Data Centre via <a href="http://www.ccdc.cam.ac.uk/structures" class="extLink">www.ccdc.cam.ac.uk/structures</a>.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i165" class="anchor-spacer"></div><h3 class="article-section__title" id="_i165"> BRD4 Mutant TR-FRET Assay<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></h3><div class="NLM_p">Tandem bromodomains of 6His-Thr-BRD4(1â477) were expressed, with an appropriate mutation in BD2 (Y390A) to monitor compound binding to BD1 or in BD1 (97A) to monitor compound binding to BD2. Analogous Y â A mutants were used to measure binding to the other BET bromodomains: 6His-Thr-BRD2 (1â473 Y386A or Y113A), 6His-Thr-BRD3 (1â435 Y348A or Y73A), and 6His-FLAG-Tev-BRDT (1â397 Y309A or Y66A). The AlexaFluor 647 labeled BET bromodomain ligand was prepared as follows: To a solution of AlexaFluor 647 hydroxysuccinimide ester in DMF was added a 1.8-fold excess of <i>N</i>-(5-aminopentyl)-2-((4<i>S</i>)-6-(4-chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]-diazepin-4-yl)acetamide, also in DMF, and when thoroughly mixed, the solution was basified by the addition of a 3-fold excess of diisopropylethylamine. Reaction progress was followed by electrospray LC/MS, and when judged complete, the product was isolated and purified by reversed-phase C18 HPLC. The final compound was characterized by mass spectroscopy and analytical reversed-phase HPLC.</div><div class="NLM_p last">Compounds were titrated from 10 mM in 100% DMSO and 50 nL transferred to a low volume black 384 well microtiter plate using a Labcyte Echo 555. A Thermo Scientific Multidrop Combi was used to dispense 5 Î¼L of 20 nM protein in an assay buffer of 50 mM HEPES, 150 mM NaCl, 5% glycerol, 1 mM DTT, and 1 mM CHAPS, pH 7.4, and in the presence of 100 nM fluorescent ligand (â¼<i>K</i><sub>d</sub> concentration for the interaction between BRD4 BD1 and ligand). After equilibrating for 30 min in the dark at rt, the bromodomain protein:fluorescent ligand interaction was detected using TR-FRET following a 5 Î¼L addition of 3 nM europium chelate labeled anti-6His antibody (PerkinElmer, W1024, AD0111) in assay buffer. Time resolved fluorescence (TRF) was then detected on a TRF laser equipped PerkinElmer Envision multimode plate reader (excitation = 337 nm; emission 1 = 615 nm; emission 2 = 665 nm; dual wavelength bias dichroic = 400 nm, 630 nm). TR-FRET ratio was calculated using the following equation: ratio = ((acceptor fluorescence at 665 nm)/(donor fluorescence at 615 nm)) Ã 1000. TR-FRET ratio data were normalized to high (DMSO) and low (compound control derivative of I-BET762) controls and IC<sub>50</sub> values determined for each of the compounds tested by fitting the fluorescence ratio data to a four-parameter model:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_m001.gif" alt="" id="_i166" /></img></span>where â<i>a</i>â is the minimum, â<i>b</i>â is the Hill slope, â<i>c</i>â is the IC<sub>50</sub>, and â<i>d</i>â is the maximum.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i167" class="anchor-spacer"></div><h3 class="article-section__title" id="_i167"> BRD4 SPR Assay</h3><div class="NLM_p last">BRD4 SPR was measured using published protocols.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> In brief, SPR were performed using a Biacore S200 instrument (GE Healthcare) on CM5 chips (GE Healthcare) at 25 Â°C. Two proteins, (1) 6His-Thr-Brd4 (1â477) Y390A and (2) His-tagged cleaved BRD4 (347â463), were immobilized onto the same CM5 to allow simultaneously affinity data to be obtained for BRD4-BD1 and BRD4-BD2 binding respectively. The top concentration used for triple dilution dose response titrations was 10 Î¼M.</div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i168" class="anchor-spacer"></div><h3 class="article-section__title" id="_i168"> Physicochemical Properties</h3><div class="NLM_p last">Permeability across a lipid membrane, chromatographic log <i>D</i> at pH 7.4, and CLND solubility by precipitation into saline were measured using published protocols.<a onclick="showRef(event, 'ref86 ref87 ref88 ref89'); return false;" href="javascript:void(0);" class="ref ref86 ref87 ref88 ref89">(86â89)</a></div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i169" class="anchor-spacer"></div><h3 class="article-section__title" id="_i169"> FaSSIF Solubility</h3><div class="NLM_p">Compounds were dissolved in DMSO at 2.5 mg/mL and then diluted in Fasted State Simulated Intestinal Fluid (FaSSIF pH 6.5) at 125 Î¼g/mL (final DMSO concentration is 5%). After 16 h of incubation at 25 Â°C, the suspension was filtered. The concentration of the compound was determined by a fast HPLC gradient. The ratio of the peak areas obtained from the standards and the sample filtrate was used to calculate the solubility of the compound.</div><div class="NLM_p">All animal studies were ethically reviewed and carried out in accordance with Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare, and Treatment of Animals.</div><div class="NLM_p last">The human biological samples were sourced ethically, and their research use was in accord with the terms of the informed consents under an IRB/EC approved protocol.</div></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i170" class="anchor-spacer"></div><h3 class="article-section__title" id="_i170"> Intrinsic Clearance (CL<sub>int</sub>) Measurements</h3><div class="NLM_p"><i>The human biological samples were sourced ethically, and their research use was in accord with the terms of the informed consents under an IRB/EC approved protocol.</i></div><div class="NLM_p">Microsome intrinsic clearance data were determined by Cyprotex UK. To test the metabolic stability of <b>12</b>, it was incubated in male Wistar Han rat and mixed gender pooled human liver microsomes. Microsomes (final protein concentration 0.5 mg/mL), 0.1 M phosphate buffer, pH 7.4, and test compound (final substrate concentration = 0.5 Î¼M) were preincubated at 37 Â°C prior to the addition of NADPH (final concentration = 1 mM) to initiate the reaction. The test compound was incubated for 0, 5, 15, 30, and 45 min. The control (minus NADPH) was incubated for 45 min only. The reactions were stopped by the addition of 50 Î¼L of methanol containing internal standard at the appropriate time points. Following protein precipitation, the compound remaining in the supernatants was measured using specific LC-MS/MS methods as a ratio to the internal standard in the absence of a calibration curve. Peak area ratios (compound to IS) were fitted to an unweighted logarithmic decline in substrate. Using the first-order rate constant, clearance was calculated by adjustment for protein concentration, volume of the incubation, and hepatic scaling factor (52.5 mg microsomal protein/g liver for all species).</div><div class="NLM_p last">Hepatocyte intrinsic clearance data were determined by Cyprotex UK. Test compound (0.5 Î¼M) was incubated with cryopreserved hepatocytes in suspension. Samples were removed at six time points over the course of a 60 min (rat) or 120 min (human) experiment and test compound analyzed by LC-MS/MS. Cryopreserved pooled hepatocytes were purchased from a reputable commercial supplier and stored in liquid nitrogen prior to use. Williams E media supplemented with 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine and 25 mM HEPES and test compound (final substrate concentration 0.5 Î¼M; final DMSO concentration 0.25%) was preincubated at 37 Â°C prior to the addition of a suspension of cryopreserved hepatocytes (final cell density 0.5 Ã 10<sup>6</sup> viable cells/mL in Williams E media supplemented with 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine and 25 mM HEPES) to initiate the reaction. The final incubation volume was 500 Î¼L. The reactions were stopped by transferring 50 Î¼L of incubate to 100 Î¼L of acetonitrile at the appropriate time points. The termination plates were centrifuged at 2500 rpm at 4 Â°C for 30 min to precipitate the protein. The remaining incubate (200 Î¼L) was crashed with 400 Î¼L of acetonitrile at the end of the incubation. Following protein precipitation, the sample supernatants were combined in cassettes of up to four compounds and analyzed using Cyprotex generic LC-MS/MS conditions.</div></div><div id="sec3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i171" class="anchor-spacer"></div><h3 class="article-section__title" id="_i171"> Intrinsic Clearance (CL<sub>int</sub>) Data Analysis</h3><div class="NLM_p last">From a plot of ln peak area ratio (compound peak area/internal standard peak area) against time, the gradient of the line was determined. Subsequently, half-life (<i>t</i><sub>1/2</sub>) and intrinsic clearance (CL<sub>int</sub>) were calculated using the equations below:<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_m002.gif" alt="" id="_i172" /></img></span>where <i>V</i> = incubation volume (Î¼L)/number of cells</div></div><div id="sec3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i173" class="anchor-spacer"></div><h3 class="article-section__title" id="_i173"> Fraction Unbound in Blood</h3><div class="NLM_p last">Control blood from Wistar Han rat and beagle dog were obtained on the day of experimentation from in house GSK stock animals. The fraction unbound in blood of each species was determined using rapid equilibrium dialysis technology (RED plate (Linden Bioscience, Woburn, MA) at a concentration of 200 and 1000 ng/mL. Blood was dialyzed against phosphate buffered saline solution by incubating the dialysis units at 37 Â°C for 4 h. Following incubation, aliquots of blood and buffer were matrix matched prior to analysis by LCâMS/MS. The unbound fraction was determined using the peak area ratios in buffer and in blood as a mean value of the two concentrations investigated.</div></div><div id="sec3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i174" class="anchor-spacer"></div><h3 class="article-section__title" id="_i174"> In Vivo DMPK Studies</h3><div class="NLM_p"><i>All animal studies were ethically reviewed and carried out in accordance with Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare, and Treatment of Animals.</i></div><div class="NLM_p">For all in vivo studies, the temperature and humidity were nominally maintained at 21 Â°C Â± 2 Â°C and 55% Â± 10%, respectively. The diet for rodents was 5LF2 Eurodent Diet 14% (PMI Labdiet, Richmond, IN) and for dogs was Harlan Teklad 2021C (HarlanTeklad, Madison, WI). There were no known contaminants in the diet or water at concentrations that could interfere with the outcome of the studies.</div><div id="sec3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i175" class="anchor-spacer"></div><h4 class="article-section__title" id="_i175"> Rat Surgical Preparation for iv Infusion Study</h4><div class="NLM_p last">Male Wistar Han rats (supplied by Charles River UK Ltd.) were surgically prepared at GSK with implanted cannulae in the femoral vein (for drug administration) and jugular vein (for blood sampling). The rats received cefuroxime (116 mg/kg sc) and carprofen (7.5 mg/kg sc) as a preoperative antibiotic and analgesic, respectively. The rats were allowed to recover for at least 2 days prior to dosing and had free access to food and water throughout.</div></div><div id="sec3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i176" class="anchor-spacer"></div><h4 class="article-section__title" id="_i176"> Rat iv <i>n</i> = 1 PK Study</h4><div class="NLM_p last">Surgically prepared male Wistar Han Rats received a 1 h intravenous (iv) infusion of compound of interest as a discrete dose, formulated in DMSO and 10% (w/v) Kleptose HPB in saline aq (2%:98% (v/v)) at a concentration of 0.2 mg/mL to achieve a target dose of 1 mg/kg. Serial blood samples (25 Î¼L) were collected predose and up to 7 h after the start of the iv infusion. Diluted blood samples were analyzed for parent compound using a specific LCâMS/MS assay (LLQ = 2â10 ng/mL). At the end of the study, the rats were euthanized by a Schedule 1 technique.</div></div><div id="sec3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i177" class="anchor-spacer"></div><h4 class="article-section__title" id="_i177"> Rat PO <i>n</i> = 3 PK Study</h4><div class="NLM_p last">Three naÃ¯ve male Wistar Han rats (Female Lewis rats for <b>56</b>) with no surgical preparation received an oral gavage administration of compound of interest as a discrete dose, suspended in 1% (w/v) methylcellulose aq at a concentration of 0.6 mg/mL to achieve a target dose of 3 mg/kg (a concentration of 2 mg/mL to achieve a target dose of 10 mg/kg for <b>56</b>). Serial blood samples (25 Î¼L) were collected via temporary tail vein cannulation up to 7 h after oral dosing and additional blood sampling via tail vein venepuncture up to 24 h after oral dosing. Diluted blood samples were analyzed for parent compound using a specific LCâMS/MS assay (LLQ = 2â10 ng/mL). At the end of the study, the rats were euthanized by a Schedule 1 technique.</div></div><div id="sec3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i178" class="anchor-spacer"></div><h4 class="article-section__title" id="_i178"> Rat iv PO <i>n</i> = 3 Crossover PK Study</h4><div class="NLM_p last">This study was conducted as a crossover design over two dosing occasions, with 5 days between dose administrations in three surgically prepared male Wistar Han rats. On the first dosing occasion, rats each received a 1 h intravenous (iv) infusion of compound <b>18</b> formulated in DMSO and 10% (w/v) Kleptose HPB in saline aq (2%:98% (v/v)) at a concentration of 0.2 mg/mL to achieve a target dose of 1 mg/kg. On the second dosing occasion, the same three rats were administered with compound <b>18</b> suspended in 1% (w/v) methylcellulose 400 aq at a concentration of 0.6 mg/mL orally, at a target dose of 3 mg/kg. Serial blood samples (25 Î¼L) were collected predose and up to 24 h after the start of the iv infusion and after oral dosing. Diluted blood samples were analyzed using a specific LCâMS/MS assay (LLQ = 2 ng/mL). At the end of the study, the rats were euthanized by a Schedule 1 technique.</div></div><div id="sec3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i179" class="anchor-spacer"></div><h4 class="article-section__title" id="_i179"> Dog PK Study</h4><div class="NLM_p last">One healthy, laboratory-bred, male Beagle dog (supplied by Harlan Laboratories, U.K.) was used. The dog was fasted overnight prior to each dose administration and fed approximately 4 h after the start of dosing and had free access to water throughout the study. This study was conducted as a crossover design, with 8 days between dose administrations. On the first dosing occasion, the dog received a 1 h intravenous (iv) infusion of <b>20</b> formulated in DMSO and 10% (w/v) Kleptose HPB in saline aq (2%:98% (v/v)), at a concentration of 0.1 mg/mL to achieve a target dose of 0.5 mg/kg. On a subsequent dosing occasion, the same dog was administered with <b>20</b>, suspended in 1% (w/v) methylcellulose aq at a concentration of 0.1 mg/mL to achieve a target dose of 1 mg/kg. A temporary cannula was inserted into the cephalic vein from which serial blood samples (50 Î¼L) were collected predose and up to 26 h after the start of dosing. After collection of the 2 h time point the cannula was removed and later time points were taken via direct venepuncture of the jugular vein. Diluted blood samples were analyzed for parent drug concentration using a specific LCâMS/MS assay (LLQ = 5 ng/mL). At the end of each study the dog was returned to the colony.</div></div></div><div id="sec3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i180" class="anchor-spacer"></div><h3 class="article-section__title" id="_i180"> Blood Sample Analysis</h3><div class="NLM_p last">Diluted blood samples (1:1 with water) were extracted using protein precipitation with acetonitrile containing an analytical internal standard. An aliquot of the supernatant was analyzed by reverse phase LCâMS/MS using a heat assisted electrospray interface in positive ion mode. Samples were assayed against calibration standards prepared in control blood.</div></div><div id="sec3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i181" class="anchor-spacer"></div><h3 class="article-section__title" id="_i181"> PK Data Analysis from PK Studies</h3><div class="NLM_p last">PK parameters were obtained from the blood concentrationâtime profiles using noncompartmental analysis with WinNonlin Professional 6.3 (Pharsight, Mountain View, CA).</div></div><div id="sec3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i182" class="anchor-spacer"></div><h3 class="article-section__title" id="_i182"> hWB MCP-1 Assay</h3><div class="NLM_p"><i>The human biological samples were sourced ethically and their research use was in accord with the terms of the informed consents under an IRB/EC approved protocol.</i></div><div class="NLM_p last">Compounds to be tested were diluted in 100% DMSO to give a range of appropriate concentrations at 140Ã the required final assay concentration, of which 1 Î¼L was added to a 96 well tissue culture plate. Then 130 Î¼L of human whole blood, collected into sodium heparin anticoagulant (1 unit/mL final) was added to each well and plates were incubated at 37 Â°C (5% C02) for 30 min before the addition of 10 Î¼L of 2.8 Î¼g/mL LPS (Salmonella Typhosa), diluted in complete RPMI 1640 (final concentration 200 ng/mL), to give a total volume of 140 Î¼L per well. After further incubation for 24 h at 37 Â°C, 140 Î¼L of PBS was added to each well. The plates were sealed, shaken for 10 min, and then centrifuged (2500 rpm Ã 10 min). Finally, 100 Î¼L of the supernatant was removed and MCP-1 levels assayed immediately by immunoassay (MesoScale Discovery technology).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i183"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00796" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18567" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18567" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00796?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00796</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">DiscoverX BROMOscan Bromodomain Profiling of <b>20</b>, cross screening data of liability target panel for <b>20</b> and <b>56</b>, sequence alignment and differences of BET proteins, X-ray crystallographic data and selected LCMS and NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Structure factors for datablock <b>20</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_003.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Crystallographic information for <b>20</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_004.cif" class="ext-link">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf">jm0c00796_si_001.pdf (2.47 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_002.csv">jm0c00796_si_002.csv (10.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_003.pdf">jm0c00796_si_003.pdf (96.17 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_004.cif">jm0c00796_si_004.cif (26.75 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB0">6ZB0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB2">6ZB2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>. Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00796" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43631" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43631" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan T. Seal</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0148-5487" title="Orcid link">http://orcid.org/0000-0003-0148-5487</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#117b7e7f3f653f6274707d5176627a3f727e7c"><span class="__cf_email__" data-cfemail="6d0702034319431e080c012d0a1e06430e0200">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen J. Atkinson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span>Present Address:
                        For S.J.A.: Discovery Sciences, AstraZeneca, Oncology R&D, Cambridge Science Park, Unit 310 Darwin Building, Cambridge, CB4 0WG, U.K</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3636-3674" title="Orcid link">http://orcid.org/0000-0003-3636-3674</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#394a4d5c49515c5717584d5250574a56570879584a4d4b58435c575c5a58175a5654"><span class="__cf_email__" data-cfemail="65161100150d000b4b04110e0c0b160a0b542504161117041f000b0006044b060a08">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helen Aylott</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Royston C. B. Copley</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurie Gordon</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstrasse
1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James R. J. Gray</span> - <span class="hlFld-Affiliation affiliation">Quantitative Pharmacology,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lee A. Harrison</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas G. Hayhow</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span>Present Address:
                        For T.G.H.: Discovery Sciences, AstraZeneca, Oncology R&D, Cambridge Science Park, Unit 310 Darwin Building, Cambridge, CB4 0WG, U.K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew Lindon</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span>Present Address:
                        For M.L.: Research and Early Development, RIA, Biopharmaceuticals R&D, AstraZeneca, SE-431 83, MÃ¶lndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cassie Messenger</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Michon</span> - <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstrasse
1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren Mitchell</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex Preston</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0334-0679" title="Orcid link">http://orcid.org/0000-0003-0334-0679</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Inmaculada Rioja</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Taylor</span> - <span class="hlFld-Affiliation affiliation">Quantitative Pharmacology,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstrasse
1, 69117 Heidelberg, Germany</span>;Â 
    <span>Present Address:
                        For S.T.: Drug Discovery Services Europe, Pharmaron, Hertford Road, Hoddesdon EN11 9BU, UK</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian D. Wall</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James M. Woolven</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H. Demont</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit,Â 
    Immunoinflammation
Therapy Area Unit,Â 
    , , , , GlaxoSmithKline,
Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7307-3129" title="Orcid link">http://orcid.org/0000-0001-7307-3129</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i190">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16883" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16883" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank members of Platform Technology & Science at GSK for protein reagent generation, assay, and crystallization support; Tony Cooper and Heather Barnett for support with chemistry arrays; Eric Hortense, Richard Briers, Steve Jackson, and Sean Hindley for analytical and purification support; Sean Lynn, Richard Upton and Stephen Richards for assistance with NMR analysis; and William Clegg and Michael Probert (University of Newcastle) for the GSK-funded crystal structure of <b>20</b>.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AMP</td><td class="NLM_def"><p class="first last">artificial membrane permeability</p></td></tr><tr><td class="NLM_term">BAZ2B</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain 2B</p></td></tr><tr><td class="NLM_term">BD1</td><td class="NLM_def"><p class="first last">bromodomain 1 (N-terminal bromodomain)</p></td></tr><tr><td class="NLM_term">BD2</td><td class="NLM_def"><p class="first last">bromodomain 2 (C-terminal bromodomain)</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromo and extra-terminal domain</p></td></tr><tr><td class="NLM_term">BRD2,3,4,T</td><td class="NLM_def"><p class="first last">bromodomain containing protein 2,3,4,T</p></td></tr><tr><td class="NLM_term">BSEP</td><td class="NLM_def"><p class="first last">bile salt export pump</p></td></tr><tr><td class="NLM_term">CHAPS</td><td class="NLM_def"><p class="first last">(3-((3-cholamidopropyl) dimethylammonio)-1-propanesulfonate)</p></td></tr><tr><td class="NLM_term">CL<sub>b</sub></td><td class="NLM_def"><p class="first last">blood clearance</p></td></tr><tr><td class="NLM_term">CL<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">CLND</td><td class="NLM_def"><p class="first last">chemiluminscent nitrogen detection</p></td></tr><tr><td class="NLM_term">CMTP</td><td class="NLM_def"><p class="first last">2-(tributylphosphoranylidene)acetonitrile</p></td></tr><tr><td class="NLM_term">D</td><td class="NLM_def"><p class="first last">dose</p></td></tr><tr><td class="NLM_term">DIAD</td><td class="NLM_def"><p class="first last">diisopropyl azodicarboxylate</p></td></tr><tr><td class="NLM_term">dppb</td><td class="NLM_def"><p class="first last">1,4-bis(diphenylphosphino)butane</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-(dimethylamino)propyl)-<i>N</i>â²-ethylcarbodiimide</p></td></tr><tr><td class="NLM_term">EDG</td><td class="NLM_def"><p class="first last">electron-donating group</p></td></tr><tr><td class="NLM_term">EWG</td><td class="NLM_def"><p class="first last">electron-withdrawing group</p></td></tr><tr><td class="NLM_term">FACS</td><td class="NLM_def"><p class="first last">flow cytometry staining buffer</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">fu<sub>b</sub></td><td class="NLM_def"><p class="first last">fraction unbound in blood</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HepG2</td><td class="NLM_def"><p class="first last">human liver cancer cell line</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">(4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid)</p></td></tr><tr><td class="NLM_term">hWB</td><td class="NLM_def"><p class="first last">human whole blood</p></td></tr><tr><td class="NLM_term"><i>k</i></td><td class="NLM_def"><p class="first last">elimination rate constant</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">LLE<sub>at</sub></td><td class="NLM_def"><p class="first last">Astex lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">MCP-1/CCL2</td><td class="NLM_def"><p class="first last">monocyte chemoattractant protein-1</p></td></tr><tr><td class="NLM_term">MDAP</td><td class="NLM_def"><p class="first last">mass-directed auto preparation</p></td></tr><tr><td class="NLM_term">MDI</td><td class="NLM_def"><p class="first last">metabolism dependent inhibition</p></td></tr><tr><td class="NLM_term">MLA</td><td class="NLM_def"><p class="first last">mouse lymphoma assay</p></td></tr><tr><td class="NLM_term">RPMI</td><td class="NLM_def"><p class="first last">Roswell Park Memorial Institute</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">STAB</td><td class="NLM_def"><p class="first last">sodium triacetoxyborohydride</p></td></tr><tr><td class="NLM_term">TDI</td><td class="NLM_def"><p class="first last">time dependent inhibition</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term"><i>V</i></td><td class="NLM_def"><p class="first last">incubation volume</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution at steady state</p></td></tr><tr><td class="NLM_term">WPF</td><td class="NLM_def"><p class="first last">tryptophan-proline-phenylalanine</p></td></tr><tr><td class="NLM_term">XRPD</td><td class="NLM_def"><p class="first last">X-ray powder diffraction</p></td></tr><tr><td class="NLM_term">2-MeTHF</td><td class="NLM_def"><p class="first last">2-methyltetrahydrofuran.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i192">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32183" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32183" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 89 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">246</span>â <span class="NLM_lpage">262</span>, <span class="refDoi">Â DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+bromodomain-containing+proteins+and+their+roles+in+homeostasis+and+cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0lhUekr8-IoONg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520bromodomain-containing%2520proteins%2520and%2520their%2520roles%2520in%2520homeostasis%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">146</span>â <span class="NLM_lpage">153</span>, <span class="refDoi">Â DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&issue=3&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lhUekr8-IoONg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26issue%3D3%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>, <span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhyI9DLrp3xDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">697</span>, <span class="refDoi">Â DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697&issue=5&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0lhyI9DLrp3xDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26issue%3D5%26spage%3D697%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>, <span class="refDoi">Â DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lhRdhfQ8rw7cA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1748</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-12-3066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F1078-0432.CCR-12-3066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23403638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1748&issue=7&author=Z.+Chengauthor=Y.+Gongauthor=Y.+Maauthor=K.+Luauthor=X.+Luauthor=L.+A.+Pierceauthor=R.+C.+Thompsonauthor=S.+Mullerauthor=S.+Knappauthor=J.+Wang&title=Inhibition+of+BET+Bromodomain+Targets+Genetically+Diverse+Glioblastoma&doi=10.1158%2F1078-0432.CCR-12-3066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span></div><div class="casAuthors">Cheng, Zhixiang; Gong, Yuanying; Ma, Yufang; Lu, Kaihua; Lu, Xiang; Pierce, Larry A.; Thompson, Reid C.; Muller, Susanne; Knapp, Stefan; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1748-1759</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Glioblastoma is refractory to conventional therapies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic readers that selectively bind to acetylated lysine residues on histone tails.  These proteins recently emerged as important therapeutic targets in NUT midline carcinoma and several types of hematopoietic cancers.  In this study, the therapeutic potential of a novel BET bromodomain inhibitor, JQ1, was assessed in a panel of genetically heterogeneous glioblastoma samples.  Exptl. Design: The antineoplastic effects of JQ1 were shown using ex vivo cultures derived from primary glioblastoma xenograft lines and surgical specimens of different genetic background.  The in vivo efficacy was assessed in orthotopic glioblastoma tumors.  Results: We showed that JQ1 induced marked G1 cell-cycle arrest and apoptosis, which was phenocopied by knockdown of individual BET family members.  JQ1 treatment resulted in significant changes in expression of genes that play important roles in glioblastoma such as c-Myc, p21CIP1/WAF1, hTERT, Bcl-2, and Bcl-xL.  Unlike the observations in some hematopoietic cancer cell lines, exogenous c-Myc did not significantly protect glioblastoma cells against JQ1.  In contrast, ectopically expressed Bcl-xL partially rescued cells from JQ1-induced apoptosis, and knockdown of p21CIP1/WAF1 attenuated JQ1-induced cell-cycle arrest.  Cells genetically engineered for Akt hyperactivation or p53/Rb inactivation did not compromise JQ1 efficacy, suggesting that these frequently mutated signaling pathways may not confer resistance to JQ1.  Furthermore, JQ1 significantly repressed growth of orthotopic glioblastoma tumors.  Conclusion: Our results suggest potentially broad therapeutic use of BET bromodomain inhibitors for treating genetically diverse glioblastoma tumors.  Clin Cancer Res; 19(7); 1748-59. Â©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OsFKqlwyDbVg90H21EOLACvtfcHk0lhRdhfQ8rw7cA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D&md5=1722189e5d950da233b785558f2c9f58</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3066%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPierce%26aufirst%3DL.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BC.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520BET%2520Bromodomain%2520Targets%2520Genetically%2520Diverse%2520Glioblastoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26issue%3D7%26spage%3D1748%26doi%3D10.1158%2F1078-0432.CCR-12-3066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">da
Motta, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purshouse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bastiani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morotti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigfield, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, A.</span></span> <span> </span><span class="NLM_article-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">122</span>, <span class="refDoi">Â DOI: 10.1038/onc.2016.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fonc.2016.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27292261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=122&author=L.+L.+da%0AMottaauthor=I.+Ledakiauthor=K.+Purshouseauthor=S.+Haiderauthor=M.+A.+De+Bastianiauthor=D.+Babanauthor=M.+Morottiauthor=G.+Steersauthor=S.+Wigfieldauthor=E.+Bridgesauthor=J.+L.+Liauthor=S.+Knappauthor=D.+Ebnerauthor=F.+Klamtauthor=A.+L.+Harrisauthor=A.+McIntyre&title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&doi=10.1038%2Fonc.2016.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span></div><div class="casAuthors">da Motta, L. L.; Ledaki, I.; Purshouse, K.; Haider, S.; De Bastiani, M. A.; Baban, D.; Morotti, M.; Steers, G.; Wigfield, S.; Bridges, E.; Li, J.-L.; Knapp, S.; Ebner, D.; Klamt, F.; Harris, A. L.; McIntyre, A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-132</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of bromodomain and extra-terminal inhibitors (BETi) has enabled translational epigenetic studies in cancer.  BET proteins regulate transcription by selectively recognizing acetylated lysine residues on chromatin.  BETi compete with this process leading to both downregulation and upregulation of gene expression.  Hypoxia enables progression of triple neg. breast cancer (TNBC), the most aggressive form of breast cancer, partly by driving metabolic adaptation, angiogenesis and metastasis through upregulation of hypoxia-regulated genes (for example, carbonic anhydrase 9 (CA9) and vascular endothelial growth factor A (VEGF-A).  Responses to hypoxia can be mediated epigenetically, thus we investigated whether BETi JQ1 could impair the TNBC response induced by hypoxia and exert anti-tumor effects.  JQ1 significantly modulated 44% of hypoxia-induced genes, of which two-thirds were downregulated including CA9 and VEGF-A.  JQ1 prevented HIF binding to the hypoxia response element in CA9 promoter, but did not alter HIF expression or activity, suggesting some HIF targets are BET-dependent.  JQ1 reduced TNBC growth in vitro and in vivo and inhibited xenograft vascularization.  These findings identify that BETi dually targets angiogenesis and the hypoxic response, an effective combination at reducing tumor growth in preclin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79-F0o2tp6bVg90H21EOLACvtfcHk0lhRdhfQ8rw7cA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D&md5=9f1592548db8e6306703bdab7e1e1c90</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.184%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BMotta%26aufirst%3DL.%2BL.%26aulast%3DLedaki%26aufirst%3DI.%26aulast%3DPurshouse%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DS.%26aulast%3DDe%2BBastiani%26aufirst%3DM.%2BA.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DMorotti%26aufirst%3DM.%26aulast%3DSteers%26aufirst%3DG.%26aulast%3DWigfield%26aufirst%3DS.%26aulast%3DBridges%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DMcIntyre%26aufirst%3DA.%26atitle%3DThe%2520BET%2520inhibitor%2520JQ1%2520selectively%2520impairs%2520tumour%2520response%2520to%2520hypoxia%2520and%2520downregulates%2520CA9%2520and%2520angiogenesis%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D122%26doi%3D10.1038%2Fonc.2016.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Connor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: A Novel Bivalent BET Bromodomain Inhibitor Highly Active against Hematologic Malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2563</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F1535-7163.MCT-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563&issue=11&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%27Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+A+Novel+Bivalent+BET+Bromodomain+Inhibitor+Highly+Active+against+Hematologic+Malignancies&doi=10.1158%2F1535-7163.MCT-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0lhRjePf_2ozwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%2527Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520A%2520Novel%2520Bivalent%2520BET%2520Bromodomain%2520Inhibitor%2520Highly%2520Active%2520against%2520Hematologic%2520Malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26issue%3D11%26spage%3D2563%26doi%3D10.1158%2F1535-7163.MCT-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1628</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628&issue=7&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+Bromodomain+Inhibitor+OTX015+Affects+Pathogenetic+Pathways+in+Preclinical+B-cell+Tumor+Models+and+Synergizes+with+Targeted+Drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. Â©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0lhRjePf_2ozwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520OTX015%2520Affects%2520Pathogenetic%2520Pathways%2520in%2520Preclinical%2520B-cell%2520Tumor%2520Models%2520and%2520Synergizes%2520with%2520Targeted%2520Drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26issue%3D7%26spage%3D1628%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Sullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3336</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F0008-5472.CAN-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336&issue=11&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%27Sullivanauthor=S.+M%C3%BCllerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+Highly+Selective+Protein+Interaction+Inhibitor%2C+Targeting+BET+Bromodomains&doi=10.1158%2F0008-5472.CAN-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0lhuSn8Jm0dZGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%2527Sullivan%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520Highly%2520Selective%2520Protein%2520Interaction%2520Inhibitor%252C%2520Targeting%2520BET%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26issue%3D11%26spage%3D3336%26doi%3D10.1158%2F0008-5472.CAN-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segura, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanals-Cirera, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaziel-Sovran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanniford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GonzÃ¡lez-Gomez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubierre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvishian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, E.</span></span> <span> </span><span class="NLM_article-title">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">6264</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-0122-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F0008-5472.CAN-13-0122-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23950209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6264&issue=20&author=M.+F.+Seguraauthor=B.+Fontanals-Cireraauthor=A.+Gaziel-Sovranauthor=M.+V.+Guijarroauthor=D.+Hannifordauthor=G.+Zhangauthor=P.+Gonz%C3%A1lez-Gomezauthor=M.+Moranteauthor=L.+Jubierreauthor=W.+Zhangauthor=F.+Darvishianauthor=M.+Ohlmeyerauthor=I.+Osmanauthor=M.-M.+Zhouauthor=E.+Hernando&title=BRD4+Sustains+Melanoma+Proliferation+and+Represents+a+New+Target+for+Epigenetic+Therapy&doi=10.1158%2F0008-5472.CAN-13-0122-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span></div><div class="casAuthors">Segura, Miguel F.; Fontanals-Cirera, Barbara; Gaziel-Sovran, Avital; Guijarro, Maria V.; Hanniford, Doug; Zhang, Guangtao; Gonzalez-Gomez, Pilar; Morante, Marta; Jubierre, Luz; Zhang, Weijia; Darvishian, Farbod; Ohlmeyer, Michael; Osman, Iman; Zhou, Ming-Ming; Hernando, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6264-6276</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Metastatic melanoma remains a mostly incurable disease.  Although newly approved targeted therapies are efficacious in a subset of patients, resistance and relapse rapidly ensue.  Alternative therapeutic strategies to manipulate epigenetic regulators and disrupt the transcriptional program that maintains tumor cell identity are emerging.  Bromodomain and extraterminal domain (BET) proteins are epigenome readers known to exert key roles at the interface between chromatin remodeling and transcriptional regulation.  Here, we report that BRD4, a BET family member, is significantly upregulated in primary and metastatic melanoma tissues compared with melanocytes and nevi.  Treatment with BET inhibitors impaired melanoma cell proliferation in vitro and tumor growth and metastatic behavior in vivo, effects that were mostly recapitulated by individual silencing of BRD4.  RNA sequencing of BET inhibitor-treated cells followed by Gene Ontol. anal. showed a striking impact on transcriptional programs controlling cell growth, proliferation, cell-cycle regulation, and differentiation.  In particular, we found that, rapidly after BET displacement, key cell-cycle genes (SKP2, ERK1, and c-MYC) were downregulated concomitantly with the accumulation of cyclin-dependent kinase (CDK) inhibitors (p21 and p27), followed by cell-cycle arrest.  Importantly, BET inhibitor efficacy was not influenced by BRAF or NRAS mutational status, opening the possibility of using these small-mol. compds. to treat patients for whom no effective targeted therapy exists.  Collectively, our study reveals a crit. role for BRD4 in melanoma tumor maintenance and renders it a legitimate and novel target for epigenetic therapy directed against the core transcriptional program of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80TZH2ueQFbVg90H21EOLACvtfcHk0lhuSn8Jm0dZGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N&md5=a4a77f188de01c4c84f7293a32478fc7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0122-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0122-T%26sid%3Dliteratum%253Aachs%26aulast%3DSegura%26aufirst%3DM.%2BF.%26aulast%3DFontanals-Cirera%26aufirst%3DB.%26aulast%3DGaziel-Sovran%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%2BV.%26aulast%3DHanniford%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Gomez%26aufirst%3DP.%26aulast%3DMorante%26aufirst%3DM.%26aulast%3DJubierre%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDarvishian%26aufirst%3DF.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DHernando%26aufirst%3DE.%26atitle%3DBRD4%2520Sustains%2520Melanoma%2520Proliferation%2520and%2520Represents%2520a%2520New%2520Target%2520for%2520Epigenetic%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26issue%3D20%26spage%3D6264%26doi%3D10.1158%2F0008-5472.CAN-13-0122-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: breaking through the immune barrier</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">835</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2016.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrd.2016.185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27765940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=835&author=D.+F.+Toughauthor=P.+P.+Takauthor=A.+Tarakhovskyauthor=R.+K.+Prinjha&title=Epigenetic+drug+discovery%3A+breaking+through+the+immune+barrier&doi=10.1038%2Fnrd.2016.185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: breaking through the immune barrier</span></div><div class="casAuthors">Tough, David F.; Tak, Paul P.; Tarakhovsky, Alexander; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">835-853</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immune-mediated diseases are clin. heterogeneous but they share genetic and pathogenic mechanisms.  These diseases may develop from the interplay of genetic factors and environmental or lifestyle factors.  Exposure to such factors, including infectious agents, is assocd. with coordinated changes in gene transcription owing to epigenetic alterations.  A growing understanding of how epigenetic mechanisms control gene expression patterns and cell function has been aided by the development of small-mol. inhibitors that target these processes.  These chem. tools have helped to reveal the importance of epigenetics in guiding cell fate decisions during immune responses and have also highlighted the potential for targeting epigenetic mechanisms for the treatment of inflammation and immune-mediated diseases.  In this Review, we discuss the most advanced areas of epigenetic drug development for autoimmune and inflammatory diseases and summarize the promising preclin. data in this exciting and evolving field.  These agents will inevitably begin to move into clin. trials for use in patients with immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplOyEVMzBi37Vg90H21EOLACvtfcHk0lgTEcuJrcPzyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE&md5=f939bcbbe3d74ce24718eef706aae3d1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.185%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520breaking%2520through%2520the%2520immune%2520barrier%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D835%26doi%3D10.1038%2Fnrd.2016.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">573</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27925476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=573-584&issue=4&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span></div><div class="casAuthors">Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Genome-wide assocn. studies have identified thousands of single nucleotide polymorphisms in the human genome that are statistically assocd. with particular disease traits.  In this perspective, we review emerging data suggesting that most single nucleotide polymorphisms assocd. with immune-mediated diseases are found in regulatory regions of the DNA - parts of the genome that control expression of the protein encoding genes - rather than causing mutations in proteins.  We discuss how the emerging understanding of particular gene regulatory regions, gene enhancers and the epigenetic mechanisms by which they are regulated is opening up new opportunities for the treatment of immune-mediated diseases, focusing particularly on the BET family of epigenetic reader proteins as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFU7Sh3r75ZbVg90H21EOLACvtfcHk0lgTEcuJrcPzyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D&md5=8a2d44dd000c465b904d4ef2e6f37b77</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26issue%3D4%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses T(H)17-mediated pathology</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>210</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2181</span>â <span class="NLM_lpage">2190</span>, <span class="refDoi">Â DOI: 10.1084/jem.20130376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1084%2Fjem.20130376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24101376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2013&pages=2181-2190&issue=11&author=D.+A.+Meleauthor=A.+Salmeronauthor=S.+Ghoshauthor=H.-R.+Huangauthor=B.+M.+Bryantauthor=J.+M.+Lora&title=BET+bromodomain+inhibition+suppresses+T%28H%2917-mediated+pathology&doi=10.1084%2Fjem.20130376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses TH17-mediated pathology</span></div><div class="casAuthors">Mele, Deanna A.; Salmeron, Andres; Ghosh, Srimoyee; Huang, Hon-Ren; Bryant, Barbara M.; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin (IL) 17-producing T helper (TH17) cells have been selected through evolution for their ability to control fungal and bacterial infections.  It is also firmly established that their aberrant generation and activation results in autoimmune conditions.  Using a characterized potent and selective small mol. inhibitor, we show that the bromodomain and extra-terminal domain (BET) family of chromatin adaptors plays fundamental and selective roles in human and murine TH17 differentiation from naive CD4+ T cells, as well as in the activation of previously differentiated TH17 cells.  We provide evidence that BET controls TH17 differentiation in a bromodomain-dependent manner through a mechanism that includes the direct regulation of multiple effector TH17-assocd. cytokines, including IL17, IL21, and GMCSF.  We also demonstrate that BET family members Brd2 and Brd4 assoc. with the Il17 locus in TH17 cells, and that this assocn. requires bromodomains.  We recapitulate the crit. role of BET bromodomains in TH17 differentiation in vivo and show that therapeutic dosing of the BET inhibitor is efficacious in mouse models of autoimmunity.  Our results identify the BET family of proteins as a fundamental link between chromatin signaling and TH17 biol., and support the notion of BET inhibition as a point of therapeutic intervention in autoimmune conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBqVtggp2Fa7Vg90H21EOLACvtfcHk0lgTEcuJrcPzyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF&md5=e6822f456cf42cdead9779afa92c05e0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1084%2Fjem.20130376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20130376%26sid%3Dliteratum%253Aachs%26aulast%3DMele%26aufirst%3DD.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520T%2528H%252917-mediated%2520pathology%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2013%26volume%3D210%26issue%3D11%26spage%3D2181%26epage%3D2190%26doi%3D10.1084%2Fjem.20130376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">14532</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532&issue=36&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0ljNGKpV1EfT3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D36%26spage%3D14532%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarilina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivashkiv, L. B.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">287</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1002/eji.201444862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1002%2Feji.201444862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25345375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=287-297&issue=1&author=C.+H.+Chanauthor=C.+Fangauthor=Y.+Qiaoauthor=A.+Yarilinaauthor=R.+K.+Prinjhaauthor=L.+B.+Ivashkiv&title=BET+bromodomain+inhibition+suppresses+transcriptional+responses+to+cytokine-Jak-STAT+signaling+in+a+gene-specific+manner+in+human+monocytes&doi=10.1002%2Feji.201444862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span></div><div class="casAuthors">Chan, Chun Hin; Fang, Celestia; Qiao, Yu; Yarilina, Anna; Prinjha, Rab K.; Ivashkiv, Lionel B.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">EJIMAF</span>;
        ISSN:<span class="NLM_cas:issn">0014-2980</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Disruption of the interaction of bromo and extraterminal (BET) proteins with acetylated histones using small mol. inhibitors suppresses Myc-driven cancers and TLR-induced inflammation in mouse models.  The predominant mechanism of BET inhibitor action is to suppress BET-mediated recruitment of pos. transcription elongation factor b and, thus, transcription elongation.  We investigated the effects of BET inhibitor I-BET151 on transcriptional responses to TLR4 and TNF in primary human monocytes and also on responses to cytokines IFN-Î², IFN-Î³, IL-4, and IL-10, which activate the JAK-STAT signaling pathway and are important for monocyte polarization and inflammatory diseases.  I-BET151 suppressed TLR4- and TNF-induced IFN responses by diminishing both autocrine IFN-Î² expression and transcriptional responses to IFN-Î².  I-BET151 inhibited cytokine-induced transcription of STAT targets in a gene-specific manner without affecting STAT activation or recruitment.  This inhibition was independent of Myc or other upstream activators.  IFN-stimulated gene transcription is regulated primarily at the level of transcription initiation.  Accordingly, we found that I-BET151 suppressed the recruitment of transcriptional machinery to the CXCL10 promoter and an upstream enhancer.  Our findings suggest that BET inhibition reduces inflammation partially through suppressing cytokine activity and expands the understanding of the inhibitory and potentially selective immunosuppressive effects of inhibiting BET proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbE0f71lXpp7Vg90H21EOLACvtfcHk0ljNGKpV1EfT3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP&md5=efa081bc6c79db9233ae1a99ebf49945</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Feji.201444862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201444862%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DYarilina%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DIvashkiv%26aufirst%3DL.%2BB.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520transcriptional%2520responses%2520to%2520cytokine-Jak-STAT%2520signaling%2520in%2520a%2520gene-specific%2520manner%2520in%2520human%2520monocytes%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2015%26volume%3D45%26issue%3D1%26spage%3D287%26epage%3D297%26doi%3D10.1002%2Feji.201444862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>â <span class="NLM_lpage">76</span>, <span class="refDoi">Â DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0ljj7ulOwDfmHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3670</span>, <span class="refDoi">Â DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670&issue=7&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+Protein+Function+Is+Required+for+Inflammation%3A+Brd2+Genetic+Disruption+and+BET+Inhibitor+JQ1+Impair+Mouse+Macrophage+Inflammatory+Responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-Î± while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0lgFAm2mw96zxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520Protein%2520Function%2520Is%2520Required%2520for%2520Inflammation%253A%2520Brd2%2520Genetic%2520Disruption%2520and%2520BET%2520Inhibitor%2520JQ1%2520Impair%2520Mouse%2520Macrophage%2520Inflammatory%2520Responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26issue%3D7%26spage%3D3670%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">422</span>, <span class="refDoi">Â DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422&issue=2&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.-L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- Î±, interleukin (IL)-1Î² and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- Î± and IL-1Î² .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0lgFAm2mw96zxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26issue%3D2%26spage%3D422%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lhxhp8FALN2eA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitor JQ1 Blocks Inflammation and Bone Destruction</span>. <i>J. Dent. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">657</span>â <span class="NLM_lpage">662</span>, <span class="refDoi">Â DOI: 10.1177/0022034514534261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1177%2F0022034514534261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24799421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2014&pages=657-662&issue=7&author=S.+Mengauthor=L.+Zhangauthor=Y.+Tangauthor=Q.+Tuauthor=L.+Zhengauthor=L.+Yuauthor=D.+Murrayauthor=J.+Chengauthor=S.+H.+Kimauthor=X.+Zhouauthor=J.+Chen&title=BET+Inhibitor+JQ1+Blocks+Inflammation+and+Bone+Destruction&doi=10.1177%2F0022034514534261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor JQ1 blocks inflammation and bone destruction</span></div><div class="casAuthors">Meng, S.; Zhang, L.; Tang, Y.; Tu, Q.; Zheng, L.; Yu, L.; Murray, D.; Cheng, J.; Kim, S. H.; Zhou, X.; Chen, J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dental Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">657-662</span>CODEN:
                <span class="NLM_cas:coden">JDREAF</span>;
        ISSN:<span class="NLM_cas:issn">0022-0345</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">BET proteins are a group of epigenetic regulators controlling transcription through reading acetylated histone tails and recruiting transcription complexes.  They are considered as potential therapeutic targets in many distinct diseases.  A novel synthetic bromodomain and extraterminal domain (BET) inhibitor, JQ1, was proved to suppress oncogene transcription and inflammatory responses.  The present study was aimed to investigate the effects of JQ1 on inflammatory response and bone destruction in exptl. periodontitis.  We found that JQ1 significantly suppressed lipopolysaccharide (LPS)-stimulated inflammatory cytokine transcription, including interleukin (IL)-1Î², IL-6, and tumor necrosis factor alpha (TNF-Î±), as well as receptor activator of nuclear factor kappa- B ligand (RANKL)-induced osteoclast markers, such as c-Fos, nuclear factor of activated T-cells, cytoplasmic, calcineurin- dependent 1 (NFATc1), tartrate-resistant acid phosphatase (TRAP) and cathepsin K in vitro.  JQ1 also inhibited toll-like receptors 2/4 (TLR2/4) expression and nuclear factor kappa-light-chain-enhancer of activated B cells (NF-ÎºB) phosphorylation and nuclear translocation.  Chromatin immunopptn. and quant. polymerase chain reaction (ChIP-qPCR) revealed that JQ1 neutralized BRD4 enrichment at several gene promoter regions, including NF-ÎºB, TNF-Î±, c-Fos, and NFATc1.  In a murine periodontitis model, systemic administration of JQ1 significantly inhibited inflammatory cytokine expression in diseased gingival tissues.  Alveolar bone loss was alleviated in JQ1-treated mice because of reduced osteoclasts in periodontal tissues.  These unprecedented results suggest the BET inhibitor JQ1 as a prospective new approach for treating periodontitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtMKKZI5MhYLVg90H21EOLACvtfcHk0ljtVklUG0vImw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI&md5=7e65e561c2141b78954deb6b779ed9b5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F0022034514534261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0022034514534261%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTu%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DBET%2520Inhibitor%2520JQ1%2520Blocks%2520Inflammation%2520and%2520Bone%2520Destruction%26jtitle%3DJ.%2520Dent.%2520Res.%26date%3D2014%26volume%3D93%26issue%3D7%26spage%3D657%26epage%3D662%26doi%3D10.1177%2F0022034514534261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>â <span class="NLM_lpage">257</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORÎ±/Î³ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0lj0twtpVryWoQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.-g.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.-c.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span>. <i>Immunol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>166</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1016/j.imlet.2015.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.imlet.2015.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=26093279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2015&pages=103-108&issue=2&author=Q.-g.+Zhangauthor=J.+Qianauthor=Y.-c.+Zhu&title=Targeting+bromodomain-containing+protein+4+%28BRD4%29+benefits+rheumatoid+arthritis&doi=10.1016%2Fj.imlet.2015.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span></div><div class="casAuthors">Zhang, Qing-gang; Qian, Jing; Zhu, Yu-chang</div><div class="citationInfo"><span class="NLM_cas:title">Immunology Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">IMLED6</span>;
        ISSN:<span class="NLM_cas:issn">0165-2478</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We aimed to explore the effects of bromodomain-contg. protein 4 (BRD4) inhibition on tumor necrosis factor (TNF)-Î±-stimulated human rheumatoid arthritis fibroblast-like synoviocytes (RA-FLS) behavior and the therapeutic implications using BRD4 inhibitor JQ1 were explored in vivo.  The levels of interleukin (IL)-1Î², IL-6, IL-17 and IL-18 in cultural supernatants from TNFÎ±-stimulated RA-FLS were measured by ELISA.  RA-FLS migration and invasion in vitro were investigated using wound healing and Matrigel assay.  Expression of signaling pathway proteins was measured by Western blot.  The in vivo effects of BRD4 inhibitor JQ1 were elucidated using collagen-induced arthritis (CIA) mice.  We found BRD4 silencing reduced the secretion of IL-1Î², IL-6, IL-17 and IL-18 from TNFÎ±-stimulated human RA-FLS.  Downregulation of BRD4 inhibited FBS-induced migration and invasion of human RA-FLS.  BRD4 silencing decreased the phosphorylation of c-Jun and activation of NFÎºB in TNFÎ±-stimulated RA-FLS.  In vivo, BRD4 inhibitor JQ1 reduced the inflammatory response, autoantibody prodn. and joint damage of CIA model.  Our data suggest for the first time that BRD4 inhibition has anti-inflammatory property in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-RTAy64CzFbVg90H21EOLACvtfcHk0ljYTe4Vh8nRsA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM&md5=395fce2cd419dd86aebd9fd93ce80c0f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.imlet.2015.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.imlet.2015.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.-g.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DY.-c.%26atitle%3DTargeting%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520benefits%2520rheumatoid%2520arthritis%26jtitle%3DImmunol.%2520Lett.%26date%3D2015%26volume%3D166%26issue%3D2%26spage%3D103%26epage%3D108%26doi%3D10.1016%2Fj.imlet.2015.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">4533</span>â <span class="NLM_lpage">4558</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&issue=11&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+Targeting+Bromodomain-Containing+Protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0ljYTe4Vh8nRsA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520Targeting%2520Bromodomain-Containing%2520Protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D11%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1776</span>â <span class="NLM_lpage">1787</span>, <span class="refDoi">Â DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&issue=8&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2eYpt9eDQFKiBrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2017%26volume%3D28%26issue%3D8%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alqahtani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choucair, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashraf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammouda, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alloghbi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senzer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemunaitis, J.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and Extra-Terminal Motif Inhibitors: A Review of Preclinical and Clinical Advances in Cancer Therapy</span>. <i>Future Science OA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">FSO372</span>, <span class="refDoi">Â DOI: 10.4155/fsoa-2018-0115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.4155%2Ffsoa-2018-0115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=30906568" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=FSO372&issue=3&author=A.+Alqahtaniauthor=K.+Choucairauthor=M.+Ashrafauthor=D.+M.+Hammoudaauthor=A.+Alloghbiauthor=T.+Khanauthor=N.+Senzerauthor=J.+Nemunaitis&title=Bromodomain+and+Extra-Terminal+Motif+Inhibitors%3A+A+Review+of+Preclinical+and+Clinical+Advances+in+Cancer+Therapy&doi=10.4155%2Ffsoa-2018-0115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.4155%2Ffsoa-2018-0115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffsoa-2018-0115%26sid%3Dliteratum%253Aachs%26aulast%3DAlqahtani%26aufirst%3DA.%26aulast%3DChoucair%26aufirst%3DK.%26aulast%3DAshraf%26aufirst%3DM.%26aulast%3DHammouda%26aufirst%3DD.%2BM.%26aulast%3DAlloghbi%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DSenzer%26aufirst%3DN.%26aulast%3DNemunaitis%26aufirst%3DJ.%26atitle%3DBromodomain%2520and%2520Extra-Terminal%2520Motif%2520Inhibitors%253A%2520A%2520Review%2520of%2520Preclinical%2520and%2520Clinical%2520Advances%2520in%2520Cancer%2520Therapy%26jtitle%3DFuture%2520Science%2520OA%26date%3D2019%26volume%3D5%26issue%3D3%26spage%3DFSO372%26doi%3D10.4155%2Ffsoa-2018-0115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">Clinical Trials for BET Inhibitors Run Ahead of the Science</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>â <span class="NLM_lpage">50</span>, <span class="refDoi">Â DOI: 10.1016/j.ddtec.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.ddtec.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27769357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=45-50&author=G.+Andrieuauthor=A.+C.+Belkinaauthor=G.+V.+Denis&title=Clinical+Trials+for+BET+Inhibitors+Run+Ahead+of+the+Science&doi=10.1016%2Fj.ddtec.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical trials for BET inhibitors run ahead of the science</span></div><div class="casAuthors">Andrieu Guillaume; Belkina Anna C; Denis Gerald V</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several cancer clinical trials for small molecule inhibitors of BET bromodomain proteins have been initiated.  There is enthusiasm for the anti-proliferative effect of inhibiting BRD4, one of the targets of these inhibitors, which is thought to cooperate with MYC, a long-desired target for cancer therapeutics.  However, no current inhibitor is selective for BRD4 among the three somatic BET proteins, which include BRD2 and BRD3; their respective functions are partially overlapping and none are functionally redundant with BRD4.  Each BET protein controls distinct transcriptional pathways that are important for functions beyond cancer cell proliferation, including insulin production, cytokine gene transcription, T cell differentiation, adipogenesis and most seriously, active repression of dangerous latent viruses like HIV.  BET inhibitors have been shown to reactivate HIV in human cells.  Failure to appreciate that at concentrations used, no available BET inhibitor is member-selective, or to develop a sound biological basis to understand the diverse functions of BET proteins before undertaking for these clinical trials is reckless and likely to lead to adverse events.  More mechanistic information from new basic science studies should enable proper focus on the most relevant cancers and define the expected side effect profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-Lt7lRqEFUJVfW6udTcc2eb9ETMc5ukOPbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D&md5=5fcc9e96fa590221fb07c5e98ae91d59</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DAndrieu%26aufirst%3DG.%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DClinical%2520Trials%2520for%2520BET%2520Inhibitors%2520Run%2520Ahead%2520of%2520the%2520Science%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D45%26epage%3D50%26doi%3D10.1016%2Fj.ddtec.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postel-Vinay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbschleb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewerton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, M.</span></span> <span> </span><span class="NLM_article-title">First-in-Human Phase I Study of the Bromodomain and Extraterminal Motif Inhibitor BAY 1238097: Emerging Pharmacokinetic/Pharmacodynamic Relationship and Early Termination due to Unexpected Toxicity</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">110</span>, <span class="refDoi">Â DOI: 10.1016/j.ejca.2018.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.ejca.2018.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=30711772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2019&pages=103-110&author=S.+Postel-Vinayauthor=K.+Herbschlebauthor=C.+Massardauthor=V.+Woodcockauthor=J.-C.+Soriaauthor=A.+O.+Walterauthor=F.+Ewertonauthor=M.+Poelmanauthor=N.+Bensonauthor=M.+Ockerauthor=G.+Wilkinsonauthor=M.+Middleton&title=First-in-Human+Phase+I+Study+of+the+Bromodomain+and+Extraterminal+Motif+Inhibitor+BAY+1238097%3A+Emerging+Pharmacokinetic%2FPharmacodynamic+Relationship+and+Early+Termination+due+to+Unexpected+Toxicity&doi=10.1016%2Fj.ejca.2018.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human phase I study of the bromodomain and extraterminal motif inhibitor BAY 1238097: emerging pharmacokinetic/pharmacodynamic relationship and early termination due to unexpected toxicity</span></div><div class="casAuthors">Postel-Vinay, Sophie; Herbschleb, Karin; Massard, Christophe; Woodcock, Victoria; Soria, Jean-Charles; Walter, Annette O.; Ewerton, Flavio; Poelman, Martine; Benson, Neil; Ocker, Matthias; Wilkinson, Gary; Middleton, Mark</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-110</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal motif (BET) protein inhibition is a promising cancer treatment strategy, notably for targeting MYC- or BRD4-driven diseases.  A first-in-human study investigated the safety, pharmacokinetics, max. tolerated dose and recommended phase II dose of the BET inhibitor BAY 1238097 in patients with advanced malignancies.In this phase I, open-label, non-randomised, multicentre study, patients with cytol. or histol. confirmed advanced refractory malignancies received oral BAY 1238097 twice weekly in 21-day cycles using an adaptive dose-escalation design at a starting dose of 10 mg/wk.  Model-based dose-response anal. was performed to guide dose escalation.  Safety, pharmacokinetics, pharmacodynamics and tumor response were evaluated.Eight patients were enrolled at three dose levels (10 mg/wk, n = 3; 40 mg/wk, n = 3; 80 mg/wk, n = 2).  Both patients receiving 80 mg/wk had dose-limiting toxicities (DLTs) (grade 3 vomiting, grade 3 headache and grade 2/3 back pain).  The most common adverse events were nausea, vomiting, headache, back pain and fatigue.  Pharmacokinetic anal. indicated a linear dose response with increasing dose.  Two patients displayed prolonged stable disease; no responses were obsd.  Biomarker evaluation of MYC and HEXIM1 expression demonstrated an emerging pharmacokinetic/pharmacodynamic relationship, with a trend towards decreased MYC and increased HEXIM1 expression in response to treatment.The study was prematurely terminated because of the occurrence of DLTs at a dose below targeted drug exposure.  Pharmacokinetic modeling indicated that an alternate dosing schedule whereby DLTs could be avoided while reaching efficacious exposure was not feasible.  Registration no.: NCT02369029.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9snvNdTKTtrVg90H21EOLACvtfcHk0litHyRvr54Gmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D&md5=c5969b7827810fd1146338b84c8f8621</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2018.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2018.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DPostel-Vinay%26aufirst%3DS.%26aulast%3DHerbschleb%26aufirst%3DK.%26aulast%3DMassard%26aufirst%3DC.%26aulast%3DWoodcock%26aufirst%3DV.%26aulast%3DSoria%26aufirst%3DJ.-C.%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DEwerton%26aufirst%3DF.%26aulast%3DPoelman%26aufirst%3DM.%26aulast%3DBenson%26aufirst%3DN.%26aulast%3DOcker%26aufirst%3DM.%26aulast%3DWilkinson%26aufirst%3DG.%26aulast%3DMiddleton%26aufirst%3DM.%26atitle%3DFirst-in-Human%2520Phase%2520I%2520Study%2520of%2520the%2520Bromodomain%2520and%2520Extraterminal%2520Motif%2520Inhibitor%2520BAY%25201238097%253A%2520Emerging%2520Pharmacokinetic%252FPharmacodynamic%2520Relationship%2520and%2520Early%2520Termination%2520due%2520to%2520Unexpected%2520Toxicity%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2019%26volume%3D109%26spage%3D103%26epage%3D110%26doi%3D10.1016%2Fj.ejca.2018.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odenike, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldoss, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzieri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prebet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freise, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, B. A.</span></span> <span> </span><span class="NLM_article-title">Results from the First-in-Human Study of Mivebresib (ABBV-075), a Pan-Inhibitor of Bromodomain and Extra Terminal Proteins, in Patients with Relapsed/Refractory Acute Myeloid Leukemia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">15suppl</span>),  <span class="NLM_fpage">7030</span>, <span class="refDoi">Â DOI: 10.1200/JCO.2019.37.15_suppl.7030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1200%2FJCO.2019.37.15_suppl.7030" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=7030&issue=15suppl&author=O.+Odenikeauthor=J.+E.+Wolffauthor=G.+Borthakurauthor=I.+T.+Aldossauthor=D.+Rizzieriauthor=T.+Prebetauthor=B.+Huauthor=M.+Dinhauthor=X.+Chenauthor=D.+Modiauthor=K.+J.+Freiseauthor=B.+A.+Jonas&title=Results+from+the+First-in-Human+Study+of+Mivebresib+%28ABBV-075%29%2C+a+Pan-Inhibitor+of+Bromodomain+and+Extra+Terminal+Proteins%2C+in+Patients+with+Relapsed%2FRefractory+Acute+Myeloid+Leukemia&doi=10.1200%2FJCO.2019.37.15_suppl.7030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.7030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.7030%26sid%3Dliteratum%253Aachs%26aulast%3DOdenike%26aufirst%3DO.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26aulast%3DBorthakur%26aufirst%3DG.%26aulast%3DAldoss%26aufirst%3DI.%2BT.%26aulast%3DRizzieri%26aufirst%3DD.%26aulast%3DPrebet%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DModi%26aufirst%3DD.%26aulast%3DFreise%26aufirst%3DK.%2BJ.%26aulast%3DJonas%26aufirst%3DB.%2BA.%26atitle%3DResults%2520from%2520the%2520First-in-Human%2520Study%2520of%2520Mivebresib%2520%2528ABBV-075%2529%252C%2520a%2520Pan-Inhibitor%2520of%2520Bromodomain%2520and%2520Extra%2520Terminal%2520Proteins%252C%2520in%2520Patients%2520with%2520Relapsed%252FRefractory%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26issue%3D15suppl%26spage%3D7030%26doi%3D10.1200%2FJCO.2019.37.15_suppl.7030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amans, D.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hirst, D. J.</span>; <span class="NLM_string-name">Law, R. P.</span>; <span class="NLM_string-name">Lindon, M.</span>; <span class="NLM_string-name">Preston, A.</span>; <span class="NLM_string-name">Seal, J. T.</span>; <span class="NLM_string-name">Wellaway, C. R.</span></span> <span> </span><span class="NLM_article-title">Preparation of acylaminotetrahydroquinoline derivatives for use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2014140076A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+Amans&author=S.+J.+Atkinson&author=L.+A.+Harrison&author=D.+J.+Hirst&author=R.+P.+Law&author=M.+Lindon&author=A.+Preston&author=J.+T.+Seal&author=C.+R.+Wellaway&title=Preparation+of+acylaminotetrahydroquinoline+derivatives+for+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAmans%26aufirst%3DD.%26atitle%3DPreparation%2520of%2520acylaminotetrahydroquinoline%2520derivatives%2520for%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hayhow, T. G. C.</span>; <span class="NLM_string-name">House, D.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Preston, A. G.</span>; <span class="NLM_string-name">Seal, J. T.</span>; <span class="NLM_string-name">Wall, I. D.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Woolven, J. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of pyridinone-dicarboxamide for use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2017050714A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+J.+Atkinson&author=E.+H.+Demont&author=L.+A.+Harrison&author=T.+G.+C.+Hayhow&author=D.+House&author=M.+J.+Lindon&author=A.+G.+Preston&author=J.+T.+Seal&author=I.+D.+Wall&author=R.+J.+Watson&author=J.+M.+Woolven&title=Preparation+of+pyridinone-dicarboxamide+for+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26atitle%3DPreparation%2520of%2520pyridinone-dicarboxamide%2520for%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Aylott, H. E.</span>; <span class="NLM_string-name">Cooper, A. W. J.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hayhow, T. G. C.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Preston, A. G.</span>; <span class="NLM_string-name">Seal, J. T.</span>; <span class="NLM_string-name">Wall, I. D.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Woolven, J. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of pyridinone dicarboxamide derivatives for use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2017037116A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+J.+Atkinson&author=H.+E.+Aylott&author=A.+W.+J.+Cooper&author=E.+H.+Demont&author=L.+A.+Harrison&author=T.+G.+C.+Hayhow&author=M.+J.+Lindon&author=A.+G.+Preston&author=J.+T.+Seal&author=I.+D.+Wall&author=R.+J.+Watson&author=J.+M.+Woolven&title=Preparation+of+pyridinone+dicarboxamide+derivatives+for+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26atitle%3DPreparation%2520of%2520pyridinone%2520dicarboxamide%2520derivatives%2520for%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amans, D.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Barker, M. D.</span>; <span class="NLM_string-name">Bit, R. A.</span>; <span class="NLM_string-name">Brown, J. A.</span>; <span class="NLM_string-name">Campbell, M.</span>; <span class="NLM_string-name">Garton, N. S.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Shipley, T. J.</span>; <span class="NLM_string-name">Theodoulou, N. H.</span>; <span class="NLM_string-name">Wellaway, C. R.</span>; <span class="NLM_string-name">Westaway, S. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of furopyridines as bromodomain inhibitors useful in treating cancer, inflammation, and autoimmune disorders</span>. World Patent <span class="NLM_patent">WO2014140077A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+Amans&author=P.+Bamborough&author=M.+D.+Barker&author=R.+A.+Bit&author=J.+A.+Brown&author=M.+Campbell&author=N.+S.+Garton&author=M.+J.+Lindon&author=T.+J.+Shipley&author=N.+H.+Theodoulou&author=C.+R.+Wellaway&author=S.+M.+Westaway&title=Preparation+of+furopyridines+as+bromodomain+inhibitors+useful+in+treating+cancer%2C+inflammation%2C+and+autoimmune+disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAmans%26aufirst%3DD.%26atitle%3DPreparation%2520of%2520furopyridines%2520as%2520bromodomain%2520inhibitors%2520useful%2520in%2520treating%2520cancer%252C%2520inflammation%252C%2520and%2520autoimmune%2520disorders%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Dai, Y.</span>; <span class="NLM_string-name">Holms, J.</span>; <span class="NLM_string-name">Liu, D.</span>; <span class="NLM_string-name">McClellan, W.</span>; <span class="NLM_string-name">McDaniel, K.</span>; <span class="NLM_string-name">Hasvold, L.</span>; <span class="NLM_string-name">Fidanze, S. D.</span>; <span class="NLM_string-name">Sheppard, G.</span>; <span class="NLM_string-name">Marjanovic, J.</span></span> <span> </span><span class="NLM_article-title">Preparation of pyrrolopyridinone derivatives as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2014206345A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=L.+Wang&author=Y.+Dai&author=J.+Holms&author=D.+Liu&author=W.+McClellan&author=K.+McDaniel&author=L.+Hasvold&author=S.+D.+Fidanze&author=G.+Sheppard&author=J.+Marjanovic&title=Preparation+of+pyrrolopyridinone+derivatives+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26atitle%3DPreparation%2520of%2520pyrrolopyridinone%2520derivatives%2520as%2520bromodomain%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, S.</span>; <span class="NLM_string-name">Baeurle, S.</span>; <span class="NLM_string-name">Cleve, A.</span>; <span class="NLM_string-name">Haendler, B.</span>; <span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span> <span> </span><span class="NLM_article-title">Preparation of 1-phenyl-3H-2,3-benzodiazepines and their use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2015121268A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=S.+Siegel&author=S.+Baeurle&author=A.+Cleve&author=B.+Haendler&author=A.+E.+Fernandez-Montalvan&title=Preparation+of+1-phenyl-3H-2%2C3-benzodiazepines+and+their+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DS.%26atitle%3DPreparation%2520of%25201-phenyl-3H-2%252C3-benzodiazepines%2520and%2520their%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Combs, A. P.</span>; <span class="NLM_string-name">Sparks, R. B.</span>; <span class="NLM_string-name">Maduskuie, T. P.,  Jr.</span>; <span class="NLM_string-name">Rodgers, J. D.</span></span> <span> </span><span class="NLM_article-title">Preparation of tricyclic heterocycles as bet protein inhibitors</span>. World Patent <span class="NLM_patent">WO2014143768A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=A.+P.+Combs&author=R.+B.+Sparks&author=T.+P.+Maduskuie&author=J.+D.+Rodgers&title=Preparation+of+tricyclic+heterocycles+as+bet+protein+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCombs%26aufirst%3DA.%2BP.%26atitle%3DPreparation%2520of%2520tricyclic%2520heterocycles%2520as%2520bet%2520protein%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerner Seitzberg, J.</span>; <span class="NLM_string-name">Titilola Akinleminu
Kronborg, T.</span>; <span class="NLM_string-name">Poljak, V.</span>; <span class="NLM_string-name">Friberg, G.</span>; <span class="NLM_string-name">Teuber, L.</span></span> <span> </span><span class="NLM_article-title">Heterocyclic compounds as bromodomain Inhibitors and their preparation</span>. World Patent <span class="NLM_patent">WO2016016316A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Gerner+Seitzberg&author=T.+Titilola+Akinleminu%0AKronborg&author=V.+Poljak&author=G.+Friberg&author=L.+Teuber&title=Heterocyclic+compounds+as+bromodomain+Inhibitors+and+their+preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGerner%2BSeitzberg%26aufirst%3DJ.%26atitle%3DHeterocyclic%2520compounds%2520as%2520bromodomain%2520Inhibitors%2520and%2520their%2520preparation%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gacias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matikainen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaccia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">841</span>â <span class="NLM_lpage">854</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.chembiol.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24954007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=841-854&issue=7&author=M.+Gaciasauthor=G.+Gerona-Navarroauthor=A.+N.+Plotnikovauthor=G.+Zhangauthor=L.+Zengauthor=J.+Kaurauthor=G.+Moyauthor=E.+Rusinovaauthor=Y.+Rodriguezauthor=B.+Matikainenauthor=A.+Vincekauthor=J.+Joshuaauthor=P.+Casacciaauthor=M.-M.+Zhou&title=Selective+Chemical+Modulation+of+Gene+Transcription+Favors+Oligodendrocyte+Lineage+Progression&doi=10.1016%2Fj.chembiol.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span></div><div class="casAuthors">Gacias, Mar; Gerona-Navarro, Guillermo; Plotnikov, Alexander N.; Zhang, Guangtao; Zeng, Lei; Kaur, Jasbir; Moy, Gregory; Rusinova, Elena; Rodriguez, Yoel; Matikainen, Bridget; Vincek, Adam; Joshua, Jennifer; Casaccia, Patrizia; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-854</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation regulates gene expression through modulating protein-protein interactions in chromatin.  Chem. inhibition of acetyl-lysine binding bromodomains of the major chromatin regulators BET (bromodomain and extraterminal domain) proteins has been shown to effectively block cell proliferation in cancer and inflammation.  However, whether selective inhibition of individual BET bromodomains has distinctive functional consequences remains only partially understood.  In this study, we show that selective chem. inhibition of the first bromodomain of BET proteins using our small-mol. inhibitor, Olinone, accelerated the progression of mouse primary oligodendrocyte progenitors toward differentiation, whereas inhibition of both bromodomains of BET proteins hindered differentiation.  This effect was target specific, as it was not detected in cells treated with inactive analogs and independent of any effect on proliferation.  Therefore, selective chem. modulation of individual bromodomains, rather than use of broad-based inhibitors, may enhance regenerative strategies in disorders characterized by myelin loss such as aging and neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1fCvOR18yE7Vg90H21EOLACvtfcHk0lg6kCGvBM_T-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK&md5=6b377663a8d0d07441ab4f8a5d8191b3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DGacias%26aufirst%3DM.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DKaur%26aufirst%3DJ.%26aulast%3DMoy%26aufirst%3DG.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DMatikainen%26aufirst%3DB.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DCasaccia%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DSelective%2520Chemical%2520Modulation%2520of%2520Gene%2520Transcription%2520Favors%2520Oligodendrocyte%2520Lineage%2520Progression%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26issue%3D7%26spage%3D841%26epage%3D854%26doi%3D10.1016%2Fj.chembiol.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meslamani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2952</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1615601114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.1615601114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28265070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=2952&issue=11&author=K.+Cheungauthor=G.+Luauthor=R.+Sharmaauthor=A.+Vincekauthor=R.+Zhangauthor=A.+N.+Plotnikovauthor=F.+Zhangauthor=Q.+Zhangauthor=Y.+Juauthor=Y.+Huauthor=L.+Zhaoauthor=X.+Hanauthor=J.+Meslamaniauthor=F.+Xuauthor=A.+Jaganathanauthor=T.+Shenauthor=H.+Zhuauthor=E.+Rusinovaauthor=L.+Zengauthor=J.+Zhouauthor=J.+Yangauthor=L.+Pengauthor=M.+Ohlmeyerauthor=M.+J.+Walshauthor=D.+Y.+Zhangauthor=H.+Xiongauthor=M.-M.+Zhou&title=BET+N-terminal+bromodomain+inhibition+selectively+blocks+Th17+cell+differentiation+and+ameliorates+colitis+in+mice&doi=10.1073%2Fpnas.1615601114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span></div><div class="casAuthors">Cheung, Kalung; Lu, Geming; Sharma, Rajal; Vincek, Adam; Zhang, Ruihua; Plotnikov, Alexander N.; Zhang, Fan; Zhang, Qiang; Ju, Ying; Hu, Yuan; Zhao, Li; Han, Xinye; Meslamani, Jamel; Xu, Feihong; Jaganathan, Anbalagan; Shen, Tong; Zhu, Hongfa; Rusinova, Elena; Zeng, Lei; Zhou, Jiachi; Yang, Jianjun; Peng, Liang; Ohlmeyer, Michael; Walsh, Martin J.; Zhang, David Y.; Xiong, Huabao; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2952-2957</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">T-helper 17 (Th17) cells have important functions in adaptor immunity and have also been implicated in inflammatory disorders.  The bromodomain and extraterminal domain (BET) family proteins regulate gene transcription during lineage-specific differentiation of naive CD4+ T cells to produce mature T-helper cells.  Inhibition of acetyl-lysine binding of the BET proteins by pan-BET bromodomain (BrD) inhibitors, such as JQ1, broadly affects differentiation of Th17, Th1, and Th2 cells that have distinct immune functions, thus limiting their therapeutic potential.  Whether these BET proteins represent viable new epigenetic drug targets for inflammatory disorders has remained an unanswered question.  In this study, we report that selective inhibition of the first bromodomain of BET proteins with our newly designed small mol. MS402 inhibits primarily Th17 cell differentiation with a little or almost no effect on Th1 or Th2 and Treg cells.  MS402 preferentially renders Brd4 binding to Th17 signature gene loci over those of housekeeping genes and reduces Brd4 recruitment of p-TEFb to phosphorylate and activate RNA polymerase II for transcription elongation.  We further show that MS402 prevents and ameliorates T-cell transfer-induced colitis in mice by blocking Th17 cell overdevelopment.  Thus, selective pharmacol. modulation of individual bromodomains likely represents a strategy for treatment of inflammatory bowel diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBIvlTKnR97LVg90H21EOLACvtfcHk0ljdUSZ750eK0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D&md5=367b7c0545edde6cc0ef107e03e1310f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1615601114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1615601114%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DJu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJaganathan%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DZhang%26aufirst%3DD.%2BY.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DBET%2520N-terminal%2520bromodomain%2520inhibition%2520selectively%2520blocks%2520Th17%2520cell%2520differentiation%2520and%2520ameliorates%2520colitis%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D11%26spage%3D2952%26doi%3D10.1073%2Fpnas.1615601114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">HÃ¼gle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostrovskyi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breit, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhardt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einsle, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¼nther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohlwend, D.</span></span> <span> </span><span class="NLM_article-title">4-Acyl Pyrrole Derivatives Yield Novel Vectors for Designing Inhibitors of the Acetyl-Lysine Recognition Site of BRD4(1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1518</span>â <span class="NLM_lpage">1530</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01267</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01267" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1yksA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1518-1530&issue=4&author=M.+H%C3%BCgleauthor=X.+Lucasauthor=G.+Weitzelauthor=D.+Ostrovskyiauthor=B.+Breitauthor=S.+Gerhardtauthor=O.+Einsleauthor=S.+G%C3%BCntherauthor=D.+Wohlwend&title=4-Acyl+Pyrrole+Derivatives+Yield+Novel+Vectors+for+Designing+Inhibitors+of+the+Acetyl-Lysine+Recognition+Site+of+BRD4%281%29&doi=10.1021%2Facs.jmedchem.5b01267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">4-Acyl Pyrrole Derivatives Yield Novel Vectors for Designing Inhibitors of the Acetyl-Lysine Recognition Site of BRD4(1)</span></div><div class="casAuthors">Huegle, Martin; Lucas, Xavier; Weitzel, Gerhard; Ostrovskyi, Dmytro; Breit, Bernhard; Gerhardt, Stefan; Einsle, Oliver; Guenther, Stefan; Wohlwend, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1518-1530</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several human diseases, including cancer, show altered signaling pathways resulting from changes in the activity levels of epigenetic modulators.  In the past few years, small-mol. inhibitors against specific modulators, including the bromodomain and extra-terminal (BET) bromodomain family of acetylation readers, have shown early promise in the treatment of the genetically defined midline carcinoma and hematopoietic malignancies.  We have recently developed a novel potent inhibitor of BET proteins, 1 (XD14), which exerts a strong inhibitory potential on the proliferation of specific leukemia cell lines.  In the study presented here, we designed analogs of 1 to study the potential of substitutions on the 4-acyl pyrrole backbone to occupy addnl. sites within the substrate recognition site of BRD4(1).  The compds. were profiled using ITC, DSF, and X-ray crystallog.  We could introduce several substitutions that address previously untargeted areas of the substrate recognition site.  This work may substantially contribute to the development of therapeutics with increased target specificity against BRD4-related malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwg6psp_B35bVg90H21EOLACvtfcHk0ljdUSZ750eK0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1yksA%253D%253D&md5=07296d23042a3ce2f2ecc1b6ae691e30</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01267%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCgle%26aufirst%3DM.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWeitzel%26aufirst%3DG.%26aulast%3DOstrovskyi%26aufirst%3DD.%26aulast%3DBreit%26aufirst%3DB.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DEinsle%26aufirst%3DO.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DWohlwend%26aufirst%3DD.%26atitle%3D4-Acyl%2520Pyrrole%2520Derivatives%2520Yield%2520Novel%2520Vectors%2520for%2520Designing%2520Inhibitors%2520of%2520the%2520Acetyl-Lysine%2520Recognition%2520Site%2520of%2520BRD4%25281%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1518%26epage%3D1530%26doi%3D10.1021%2Facs.jmedchem.5b01267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voitovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derviaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rebuffet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemot, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunel, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collette, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morelli, X.</span></span> <span> </span><span class="NLM_article-title">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1634</span>â <span class="NLM_lpage">1641</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1634-1641&issue=4&author=B.+Rauxauthor=Y.+Voitovichauthor=C.+Derviauxauthor=A.+Lugariauthor=E.+Rebuffetauthor=S.+Milhasauthor=S.+Prietauthor=T.+Rouxauthor=E.+Trinquetauthor=J.-C.+Guillemotauthor=S.+Knappauthor=J.-M.+Brunelauthor=A.+Y.+Fedorovauthor=Y.+Colletteauthor=P.+Rocheauthor=S.+Betziauthor=S.+Combesauthor=X.+Morelli&title=Exploring+Selective+Inhibition+of+the+First+Bromodomain+of+the+Human+Bromodomain+and+Extra-terminal+Domain+%28BET%29+Proteins&doi=10.1021%2Facs.jmedchem.5b01708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span></div><div class="casAuthors">Raux, Brigitt; Voitovich, Yuliia; Derviaux, Carine; Lugari, Adrien; Rebuffet, Etienne; Milhas, Sabine; Priet, Stephane; Roux, Thomas; Trinquet, Eric; Guillemot, Jean-Claude; Knapp, Stefan; Brunel, Jean-Michel; Fedorov, Alexey Yu.; Collette, Yves; Roche, Philippe; Betzi, Stephane; Combes, Sebastien; Morelli, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A midthroughput screening follow-up program targeting the first bromodomain of the human BRD4 protein, BRD4(BD1), identified an acetylated-mimic xanthine deriv. inhibitor.  This compd. binds with an affinity in the low micromolar range yet exerts suitable unexpected selectivity in vitro against the other members of the bromodomain and extra-terminal domain (BET) family.  A structure-based program pinpointed a role of the ZA loop, paving the way for the development of potent and selective BET-BRDi probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouQdXvF3r4zLVg90H21EOLACvtfcHk0lj_7c3P5HmEHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D&md5=fb9cfc20c76f65fcf409370b60aecc0a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01708%26sid%3Dliteratum%253Aachs%26aulast%3DRaux%26aufirst%3DB.%26aulast%3DVoitovich%26aufirst%3DY.%26aulast%3DDerviaux%26aufirst%3DC.%26aulast%3DLugari%26aufirst%3DA.%26aulast%3DRebuffet%26aufirst%3DE.%26aulast%3DMilhas%26aufirst%3DS.%26aulast%3DPriet%26aufirst%3DS.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DGuillemot%26aufirst%3DJ.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrunel%26aufirst%3DJ.-M.%26aulast%3DFedorov%26aufirst%3DA.%2BY.%26aulast%3DCollette%26aufirst%3DY.%26aulast%3DRoche%26aufirst%3DP.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26atitle%3DExploring%2520Selective%2520Inhibition%2520of%2520the%2520First%2520Bromodomain%2520of%2520the%2520Human%2520Bromodomain%2520and%2520Extra-terminal%2520Domain%2520%2528BET%2529%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Facs.jmedchem.5b01708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">9251</span>â <span class="NLM_lpage">9264</span>, <span class="refDoi">Â DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&issue=22&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.-M.+Zhou&title=Structure-Guided+Design+of+Potent+Diazobenzene+Inhibitors+for+the+BET+Bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-ÎºB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0lj_7c3P5HmEHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure-Guided%2520Design%2520of%2520Potent%2520Diazobenzene%2520Inhibitors%2520for%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D22%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual KinaseâBromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9316</span>â <span class="NLM_lpage">9334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&issue=20&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.-Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Molecular+Basis+for+the+N-Terminal+Bromodomain-and-Extra-Terminal-Family+Selectivity+of+a+Dual+Kinase%E2%80%93Bromodomain+Inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-Îµ-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-ÎºB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38Î±.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0liAamNo4MDekg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DMolecular%2520Basis%2520for%2520the%2520N-Terminal%2520Bromodomain-and-Extra-Terminal-Family%2520Selectivity%2520of%2520a%2520Dual%2520Kinase%25E2%2580%2593Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D20%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Bernard, S.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Craggs, P. D.</span>; <span class="NLM_string-name">Cutler, L.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Evans, J. P.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Karamshi, B.</span>; <span class="NLM_string-name">Lewis, A. J.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Soden, P. E.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Willis, R.</span>; <span class="NLM_string-name">Woolven, J. M.</span>; <span class="NLM_string-name">Wyspianska, B. S.</span>; <span class="NLM_string-name">Kerr, W. J.</span>; <span class="NLM_string-name">Prinjha, R. K.</span></span>, <span> </span><span class="NLM_article-title">Structure-based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-terminal Bromodomains that Retains an Anti-inflammatory and Anti-proliferative Phenotype</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=C.+R.+Wellaway&author=P.+Bamborough&author=S.+Bernard&author=C.-w.+Chung&author=P.+D.+Craggs&author=L.+Cutler&author=E.+H.+Demont&author=J.+P.+Evans&author=L.+Gordon&author=B.+Karamshi&author=A.+J.+Lewis&author=M.+J.+Lindon&author=D.+J.+Mitchell&author=I.+Rioja&author=P.+E.+Soden&author=S.+Taylor&author=R.+J.+Watson&author=R.+Willis&author=J.+M.+Woolven&author=B.+S.+Wyspianska&author=W.+J.+Kerr&author=R.+K.+Prinjha&title=Structure-based+Design+of+a+Bromodomain+and+Extraterminal+Domain+%28BET%29+Inhibitor+Selective+for+the+N-terminal+Bromodomains+that+Retains+an+Anti-inflammatory+and+Anti-proliferative+Phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26atitle%3DStructure-based%2520Design%2520of%2520a%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520Inhibitor%2520Selective%2520for%2520the%2520N-terminal%2520Bromodomains%2520that%2520Retains%2520an%2520Anti-inflammatory%2520and%2520Anti-proliferative%2520Phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1492</span>â <span class="NLM_lpage">1500</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1492-1500&issue=4&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=M.+Zengerleauthor=C.+Tallantauthor=A.+Ciulli&title=New+Synthetic+Routes+to+Triazolo-benzodiazepine+Analogues%3A+Expanding+the+Scope+of+the+Bump-and-Hole+Approach+for+Selective+Bromo+and+Extra-Terminal+%28BET%29+Bromodomain+Inhibition&doi=10.1021%2Facs.jmedchem.5b01135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Zengerle, Michael; Tallant, Cynthia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1492-1500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe new synthetic routes developed toward a range of substituted analogs of bromo and extra-terminal (BET) bromodomain inhibitors I-BET762/JQ1 based on the triazolo-benzodiazepine scaffold.  These new routes allow for the derivatization of the methoxyphenyl and chlorophenyl rings, in addn. to the diazepine ternary center and the side chain methylene moiety.  Substitution at the level of the side chain methylene afforded compds. targeting specifically and potently engineered BET bromodomains designed as part of a bump and hole approach.  We further demonstrate that marked selectivity for the second over the first bromodomain can be achieved with an indole deriv. that exploits differential interaction with an aspartate/histidine conservative substitution on the BC loop of BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpYVun8F94bVg90H21EOLACvtfcHk0liAamNo4MDekg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE&md5=65dd39e3f7e0bc52169ffb1466218c43</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520Synthetic%2520Routes%2520to%2520Triazolo-benzodiazepine%2520Analogues%253A%2520Expanding%2520the%2520Scope%2520of%2520the%2520Bump-and-Hole%2520Approach%2520for%2520Selective%2520Bromo%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Bromodomain%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1492%26epage%3D1500%26doi%3D10.1021%2Facs.jmedchem.5b01135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">49</span>),  <span class="NLM_fpage">19754</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754&issue=49&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lgbZtwdIbLrSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D49%26spage%3D19754%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4317</span>â <span class="NLM_lpage">4334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&issue=10&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+Tetrahydroquinoxalines+as+Bromodomain+and+Extra-Terminal+Domain+%28BET%29+Inhibitors+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0lgbZtwdIbLrSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520Tetrahydroquinoxalines%2520as%2520Bromodomain%2520and%2520Extra-Terminal%2520Domain%2520%2528BET%2529%2520Inhibitors%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D10%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span> (<span class="NLM_issue">7794</span>),  <span class="NLM_fpage">306</span>â <span class="NLM_lpage">310</span>, <span class="refDoi">Â DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&issue=7794&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lgbZtwdIbLrSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26issue%3D7794%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>â <span class="NLM_lpage">5623</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+L.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+Kovarauthor=M.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-Ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+Bromodomain+Inhibitor+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0liQaRu0Gv07fg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%2BL.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-Ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery, Structural Insight, and Bioactivities of BY27 as a Selective Inhibitor of the Second Bromodomains of BET Proteins</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">111633</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2019.111633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.ejmech.2019.111633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=31461688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2019&pages=111633&author=D.+Chenauthor=T.+Luauthor=Z.+Yanauthor=W.+Luauthor=F.+Zhouauthor=X.+Lyuauthor=B.+Xuauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=Y.+Zhao&title=Discovery%2C+Structural+Insight%2C+and+Bioactivities+of+BY27+as+a+Selective+Inhibitor+of+the+Second+Bromodomains+of+BET+Proteins&doi=10.1016%2Fj.ejmech.2019.111633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins</span></div><div class="casAuthors">Chen, Deheng; Lu, Tian; Yan, Ziqin; Lu, Wenchao; Zhou, Feilong; Lyu, Xilin; Xu, Biling; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zhao, Yujun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111633</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, selective inhibition of BET BD2 is emerging as a promising strategy for drug discovery.  Despite significant progress in this area, systematic studies of selective BET BD2 inhibitors are still few.  In this study, we report the discovery of a potent and selective BET BD2 inhibitor BY27 (47).  Our high resoln. co-crystal structures of 47/BRD2 BD1 and BD2 showed that the triazole group of 47, water mols., H433 and N429 in BRD2 BD2 established a water-bridged H-bonding network, which is responsible for the obsd. selectivities.  DNA microarray anal. of HepG2 cells treated with 47 or OTX015 demonstrated the transcriptome impact differences between a BET BD2 selective inhibitor and a pan BET inhibitor.  In a MV4-11 mouse xenograft model, 47 caused 67% of tumor growth inhibition and was less toxic than a pan BET inhibitor 1 at high doses.  We conclude that the improved safety profile of selective BET BD2 inhibitors warrant future studies in BET assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre_6szzmO9ZbVg90H21EOLACvtfcHk0ljZqOngZZcsvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ&md5=e3a21e4e692b25a94e0035a2e75d33f7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DLyu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DDiscovery%252C%2520Structural%2520Insight%252C%2520and%2520Bioactivities%2520of%2520BY27%2520as%2520a%2520Selective%2520Inhibitor%2520of%2520the%2520Second%2520Bromodomains%2520of%2520BET%2520Proteins%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D182%26spage%3D111633%26doi%3D10.1016%2Fj.ejmech.2019.111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span> <span> </span><span class="NLM_article-title">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>477</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">62</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">Â DOI: 10.1016/j.bbrc.2016.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.bbrc.2016.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27282480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2016&pages=62-67&issue=1&author=O.+A.+Kharenkoauthor=E.+M.+Gesnerauthor=R.+G.+Patelauthor=K.+Norekauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=H.+C.+Hansen&title=RVX-297-+a+novel+BD2+selective+inhibitor+of+BET+bromodomains&doi=10.1016%2Fj.bbrc.2016.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span></div><div class="casAuthors">Kharenko, Olesya A.; Gesner, Emily M.; Patel, Reena G.; Norek, Karen; White, Andre; Fontano, Eric; Suto, Robert K.; Young, Peter R.; McLure, Kevin G.; Hansen, Henrik C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that specifically bind to the acetyl lysine residues of histones and transcription factors.  Small mol. BET bromodomain inhibitors can disrupt this interaction which leads to potential modulation of several disease states.  Here we describe the binding properties of a novel BET inhibitor RVX-297 that is structurally related to the clin. compd. RVX-208, currently undergoing phase III clin. trials for the treatment of cardiovascular diseases, but is distinctly different in its biol. and pharmacokinetic profiles.  We report that RVX-297 preferentially binds to the BD2 domains of the BET bromodomain and Extra Terminal (BET) family of protein.  We demonstrate the differential binding modes of RVX-297 in BD1 and BD2 domains of BRD4 and BRD2 using X-ray crystallog., and describe the structural differences driving the BD2 selective binding of RVX-297.  The isothermal titrn. calorimetry (ITC) data illustrate the related differential thermodn. of binding of RVX-297 to single as well as dual BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLUg96I7HObVg90H21EOLACvtfcHk0ljZqOngZZcsvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D&md5=a3d064e2245bf14ae525eac2490808cb</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26atitle%3DRVX-297-%2520a%2520novel%2520BD2%2520selective%2520inhibitor%2520of%2520BET%2520bromodomains%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D477%26issue%3D1%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.bbrc.2016.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marusic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenoy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulikowski, E.</span></span> <span> </span><span class="NLM_article-title">RVX-297, a BET Bromodomain Inhibitor, Has Therapeutic Effects in Preclinical Models of Acute Inflammation and Autoimmune Disease</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">694</span>, <span class="refDoi">Â DOI: 10.1124/mol.117.110379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1124%2Fmol.117.110379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28974538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=694&issue=6&author=R.+Jahagirdarauthor=S.+Attwellauthor=S.+Marusicauthor=A.+Bendeleauthor=N.+Shenoyauthor=K.+G.+McLureauthor=D.+Gilhamauthor=K.+Norekauthor=H.+C.+Hansenauthor=R.+Yuauthor=J.+Tobinauthor=G.+S.+Wagnerauthor=P.+R.+Youngauthor=N.+C.+W.+Wongauthor=E.+Kulikowski&title=RVX-297%2C+a+BET+Bromodomain+Inhibitor%2C+Has+Therapeutic+Effects+in+Preclinical+Models+of+Acute+Inflammation+and+Autoimmune+Disease&doi=10.1124%2Fmol.117.110379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297, a BET bromodomain inhibitor, has therapeutic effects in preclinical models of acute inflammation and autoimmune disease</span></div><div class="casAuthors">Jahagirdar, Ravi; Attwell, Sarah; Marusic, Suzana; Bendele, Alison; Shenoy, Narmada; McLure, Kevin G.; Gilham, Dean; Norek, Karen; Hansen, Henrik C.; Yu, Raymond; Tobin, Jennifer; Wagner, Gregory S.; Young, Peter R.; Wong, Norman C. W.; Kulikowski, Ewelina</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">694-706</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Bromodomain (BD) and extra-terminal domain contg. proteins (BET) are chromatin adapters that bind acetylated histone marks via two tandem BDs, BD1 and BD2, to regulate gene transcription.  BET proteins are involved in transcriptional reprogramming in response to inflammatory stimuli.  BET BD inhibitors (BETis) that are nonselective for BD1 or BD2 have recognized anti-inflammatory properties in vitro and counter pathol. in models of inflammation or autoimmune disease.  Although both BD1 and BD2 bind acetylated histone residues, they may independently regulate the expression of BET-sensitive genes.  Here we characterized the ability of RVX-297, a novel orally active BETi with selectivity for BD2, to modulate inflammatory processes in vitro, in vivo, and ex vivo.  RVX-297 suppressed inflammatory gene expression in multiple immune cell types in culture.  Mechanistically, RVX-297 displaced BET proteins from the promoters of sensitive genes and disrupted recruitment of active RNA polymerase II, a property shared with pan-BETis that nonselectively bind BET BDs.  In the lipopolysaccharide model of inflammation, RVX-297 reduced proinflammatory mediators assessed in splenic gene expression and serum proteins.  RVX-297 also countered pathol. in three rodent models of polyarthritis: rat and mouse collagen-induced arthritis, and mouse collagen antibody-induced arthritis.  Further, RVX-297 prevented murine exptl. autoimmune encephalomyelitis (a model of human multiple sclerosis) disease development when administered prophylactically and reduced hallmarks of pathol. when administered therapeutically.  We show for the first time that a BD2-selective BETi maintains anti-inflammatory properties and is effective in preclin. models of acute inflammation and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7r1JpLemFA7Vg90H21EOLACvtfcHk0ljt6gz11U3s-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK&md5=3f1a1b86b17315c413dd1f13d3a7736a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.110379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.110379%26sid%3Dliteratum%253Aachs%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DMarusic%26aufirst%3DS.%26aulast%3DBendele%26aufirst%3DA.%26aulast%3DShenoy%26aufirst%3DN.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGilham%26aufirst%3DD.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DTobin%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DKulikowski%26aufirst%3DE.%26atitle%3DRVX-297%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%252C%2520Has%2520Therapeutic%2520Effects%2520in%2520Preclinical%2520Models%2520of%2520Acute%2520Inflammation%2520and%2520Autoimmune%2520Disease%26jtitle%3DMol.%2520Pharmacol.%26date%3D2017%26volume%3D92%26issue%3D6%26spage%3D694%26doi%3D10.1124%2Fmol.117.110379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective Targeting of BD1 and BD2 of the BET Proteins in Cancer and Immuno-Inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">eaaz8455</span>, <span class="refDoi">Â DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=eaaz8455&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.-M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.-J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+Targeting+of+BD1+and+BD2+of+the+BET+Proteins+in+Cancer+and+Immuno-Inflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520Targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520Proteins%2520in%2520Cancer%2520and%2520Immuno-Inflammation%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3Deaaz8455%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, T. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rybin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GrÃ¶tsch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlomagno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, C. W.</span></span> <span> </span><span class="NLM_article-title">A bromodomainâDNA interaction facilitates acetylation-dependent bivalent nucleosome recognition by the BET protein BRDT</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13855</span>, <span class="refDoi">Â DOI: 10.1038/ncomms13855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fncomms13855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27991587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGjtb7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13855&author=T.+C.+R.+Millerauthor=B.+Simonauthor=V.+Rybinauthor=H.+Gr%C3%B6tschauthor=S.+Curtetauthor=S.+Khochbinauthor=T.+Carlomagnoauthor=C.+W.+M%C3%BCller&title=A+bromodomain%E2%80%93DNA+interaction+facilitates+acetylation-dependent+bivalent+nucleosome+recognition+by+the+BET+protein+BRDT&doi=10.1038%2Fncomms13855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A bromodomain-DNA interaction facilitates acetylation-dependent bivalent nucleosome recognition by the BET protein BRDT</span></div><div class="casAuthors">Miller, Thomas C. R.; Simon, Bernd; Rybin, Vladimir; Groetsch, Helga; Curtet, Sandrine; Khochbin, Saadi; Carlomagno, Teresa; Mueller, Christoph W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13855</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomains are crit. components of many chromatin modifying/remodelling proteins and are emerging therapeutic targets, yet how they interact with nucleosomes, rather than acetylated peptides, remains unclear.  Using BRDT as a model, we characterized how the BET family of bromodomains interacts with site-specifically acetylated nucleosomes.  Here we report that BRDT interacts with nucleosomes through its first (BD1), but not second (BD2) bromodomain, and that acetylated histone recognition by BD1 is complemented by a bromodomain-DNA interaction.  Simultaneous DNA and histone recognition enhances BRDT's nucleosome binding affinity and specificity, and its ability to localize to acetylated chromatin in cells.  Conservation of DNA binding in bromodomains of BRD2, BRD3 and BRD4, indicates that bivalent nucleosome recognition is a key feature of these bromodomains and possibly others.  Our results elucidate the mol. mechanism of BRDT assocn. with nucleosomes and identify structural features of the BET bromodomains that may be targeted for therapeutic inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxf-nKlLEUBbVg90H21EOLACvtfcHk0liQpZcw9f4BDw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGjtb7J&md5=08e3161181401de04130148fa4c23150</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fncomms13855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13855%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DT.%2BC.%2BR.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DRybin%26aufirst%3DV.%26aulast%3DGr%25C3%25B6tsch%26aufirst%3DH.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DCarlomagno%26aufirst%3DT.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BW.%26atitle%3DA%2520bromodomain%25E2%2580%2593DNA%2520interaction%2520facilitates%2520acetylation-dependent%2520bivalent%2520nucleosome%2520recognition%2520by%2520the%2520BET%2520protein%2520BRDT%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13855%26doi%3D10.1038%2Fncomms13855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, T. T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span> (<span class="NLM_issue">6209</span>),  <span class="NLM_fpage">638</span>, <span class="refDoi">Â DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638&issue=6209&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.+L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=A+bump-and-hole+approach+to+engineer+controlled+selectivity+of+BET+bromodomain+chemical+probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0lh08cmgZeub4Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.%2BL.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DA%2520bump-and-hole%2520approach%2520to%2520engineer%2520controlled%2520selectivity%2520of%2520BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26issue%3D6209%26spage%3D638%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Runcie, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Beurden, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulthard, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of a â³bump-and-holeâ³ approach to allele-selective BET bromodomain inhibition</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2452</span>, <span class="refDoi">Â DOI: 10.1039/C7SC02536J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1039%2FC7SC02536J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=29732121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2452&author=A.+C.+Runcieauthor=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Testaauthor=L.+van+Beurdenauthor=M.+G.+J.+Baudauthor=O.+Epemoluauthor=L.+C.+J.+Ellisauthor=K.+D.+Readauthor=V.+Coulthardauthor=A.+Brienauthor=A.+Ciulli&title=Optimization+of+a+%E2%80%B3bump-and-hole%E2%80%B3+approach+to+allele-selective+BET+bromodomain+inhibition&doi=10.1039%2FC7SC02536J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a "bump-and-hole" approach to allele-selective BET bromodomain inhibition</span></div><div class="casAuthors">Runcie, A. C.; Zengerle, M.; Chan, K.-H.; Testa, A.; van Beurden, L.; Baud, M. G. J.; Epemolu, O.; Ellis, L. C. J.; Read, K. D.; Coulthard, V.; Brien, A.; Ciulli, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2452-2468</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Allele-specific chem. genetics enables selective inhibition within families of highly-conserved proteins.  The four BET (bromodomain & extra-terminal domain) proteins - BRD2, BRD3, BRD4 and BRDT bind acetylated chromatin via their bromodomains and regulate processes such as cell proliferation and inflammation.  BET bromodomains are of particular interest, as they are attractive therapeutic targets but existing inhibitors are pan-selective.  We previously established a bump-&-hole system for the BET bromodomains, pairing a leucine/alanine mutation with an ethyl-derived analog of an established benzodiazepine scaffold.  Here we optimize upon this system with the introduction of a more conservative and less disruptive leucine/valine mutation.  Extensive structure-activity-relationships of diverse benzodiazepine analogs guided the development of potent, mutant-selective inhibitors with desirable physiochem. properties.  The active enantiomer of our best compd. - 9-ME-1 - shows â¼200 nM potency, >100-fold selectivity for the L/V mutant over wild-type and excellent DMPK properties.  Through a variety of in vitro and cellular assays we validate the capabilities of our optimized system, and then utilize it to compare the relative importance of the first and second bromodomains to chromatin binding.  These expts. confirm the primacy of the first bromodomain in all BET proteins, but also significant variation in the importance of the second bromodomain.  We also show that, despite having a minor role in chromatin recognition, BRD4 BD2 is still essential for gene expression, likely through the recruitment of non-histone proteins.  The disclosed inhibitor:mutant pair provides a powerful tool for future cellular and in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq47qHG4-zFurVg90H21EOLACvtfcHk0lhIFGZ4m4K-dw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D&md5=9a44bdccabca9f1d4dfdc983250d9aa3</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2FC7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02536J%26sid%3Dliteratum%253Aachs%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3Dvan%2BBeurden%26aufirst%3DL.%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DEllis%26aufirst%3DL.%2BC.%2BJ.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCoulthard%26aufirst%3DV.%26aulast%3DBrien%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DOptimization%2520of%2520a%2520%25E2%2580%25B3bump-and-hole%25E2%2580%25B3%2520approach%2520to%2520allele-selective%2520BET%2520bromodomain%2520inhibition%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26spage%3D2452%26doi%3D10.1039%2FC7SC02536J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olp, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. C.</span></span> <span> </span><span class="NLM_article-title">Nucleosome Scaffolding by Brd4 Tandem Bromodomains in Acetylation-Dependent Chromatin Compartmentalization</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">699967</span>, <span class="refDoi">Â DOI: 10.1101/699967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1101%2F699967" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=699967&author=M.+D.+Olpauthor=V.+Jacksonauthor=B.+C.+Smith&title=Nucleosome+Scaffolding+by+Brd4+Tandem+Bromodomains+in+Acetylation-Dependent+Chromatin+Compartmentalization&doi=10.1101%2F699967"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1101%2F699967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F699967%26sid%3Dliteratum%253Aachs%26aulast%3DOlp%26aufirst%3DM.%2BD.%26aulast%3DJackson%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DB.%2BC.%26atitle%3DNucleosome%2520Scaffolding%2520by%2520Brd4%2520Tandem%2520Bromodomains%2520in%2520Acetylation-Dependent%2520Chromatin%2520Compartmentalization%26jtitle%3DbioRxiv%26date%3D2019%26spage%3D699967%26doi%3D10.1101%2F699967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">The maximal affinity of ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">9997</span>, <span class="refDoi">Â DOI: 10.1073/pnas.96.18.9997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997&issue=18&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands&doi=10.1073%2Fpnas.96.18.9997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of â-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0ljZhbdCVG6dtg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26issue%3D18%26spage%3D9997%26doi%3D10.1073%2Fpnas.96.18.9997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">430</span>â <span class="NLM_lpage">431</span>, <span class="refDoi">Â DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&issue=10&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ea76hEW-REj6Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26issue%3D10%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span> <span> </span><span class="NLM_article-title">Assessing the lipophilicity of fragments and early hits</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">663</span>â <span class="NLM_lpage">667</span>, <span class="refDoi">Â DOI: 10.1007/s10822-011-9435-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1007%2Fs10822-011-9435-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21614595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=663-667&issue=7&author=P.+N.+Mortensonauthor=C.+W.+Murray&title=Assessing+the+lipophilicity+of+fragments+and+early+hits&doi=10.1007%2Fs10822-011-9435-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the lipophilicity of fragments and early hits</span></div><div class="casAuthors">Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">663-667</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  A key challenge in many drug discovery programs is to accurately assess the potential value of screening hits.  This is particularly true in fragment-based drug design (FBDD), where the hits often bind relatively weakly, but are correspondingly small.  Ligand efficiency (LE) considers both the potency and the size of the mol., and enables us to est. whether or not an initial hit is likely to be optimizable to a potent, druglike lead.  While size is a key property that needs to be controlled in a small mol. drug, there are a no. of addnl. properties that should also be considered.  Lipophilicity is amongst the most important of these addnl. properties, and here we present a new efficiency index (LLEAT) that combines lipophilicity, size and potency.  The index is intuitively defined, and has been designed to have the same target value and dynamic range as LE, making it easily interpretable by medicinal chemists.  Monitoring both LE and LLEAT should help both in the selection of more promising fragment hits, and controlling mol. wt. and lipophilicity during optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCnbfcwp30BrVg90H21EOLACvtfcHk0ljvLwkJh5aJIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbnL&md5=e6d68046d3d20bd2ea088ce218451c25</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9435-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9435-z%26sid%3Dliteratum%253Aachs%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DAssessing%2520the%2520lipophilicity%2520of%2520fragments%2520and%2520early%2520hits%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26issue%3D7%26spage%3D663%26epage%3D667%26doi%3D10.1007%2Fs10822-011-9435-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery II: The Impact of Chromatographic Hydrophobicity Measurements and Aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">822</span>â <span class="NLM_lpage">830</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&issue=17&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+Physical+in+Drug+Discovery+II%3A+The+Impact+of+Chromatographic+Hydrophobicity+Measurements+and+Aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0ljvLwkJh5aJIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%2520II%253A%2520The%2520Impact%2520of%2520Chromatographic%2520Hydrophobicity%2520Measurements%2520and%2520Aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D17%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ClÃ©ment, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3827</span>â <span class="NLM_lpage">3838</span>, <span class="refDoi">Â DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&issue=11&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lgOocNoFTUsxQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D11%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>, <span class="refDoi">Â DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lgOocNoFTUsxQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">214</span>â <span class="NLM_lpage">231</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&issue=1&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize Îµ-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lgOocNoFTUsxQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26issue%3D1%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muniz, J. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Low-Druggability Bromodomains: Fragment Based Screening and Inhibitor Design against the BAZ2B Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">10183</span>â <span class="NLM_lpage">10187</span>, <span class="refDoi">Â DOI: 10.1021/jm401582c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401582c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltL%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=10183-10187&issue=24&author=F.+M.+Fergusonauthor=O.+Fedorovauthor=A.+Chaikuadauthor=M.+Philpottauthor=J.+R.+C.+Munizauthor=I.+Felletarauthor=F.+von+Delftauthor=T.+Heightmanauthor=S.+Knappauthor=C.+Abellauthor=A.+Ciulli&title=Targeting+Low-Druggability+Bromodomains%3A+Fragment+Based+Screening+and+Inhibitor+Design+against+the+BAZ2B+Bromodomain&doi=10.1021%2Fjm401582c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Low-Druggability Bromodomains: Fragment Based Screening and Inhibitor Design against the BAZ2B Bromodomain</span></div><div class="casAuthors">Ferguson, Fleur M.; Fedorov, Oleg; Chaikuad, Apirat; Philpott, Martin; Muniz, Joao R. C.; Felletar, Ildiko; von Delft, Frank; Heightman, Tom; Knapp, Stefan; Abell, Chris; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10183-10187</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader domains that have recently become popular targets.  In contrast to BET bromodomains, which have proven druggable, bromodomains from other regions of the phylogenetic tree have shallower pockets.  We describe successful targeting of the challenging BAZ2B bromodomain using biophys. fragment screening and structure-based optimization of high ligand-efficiency fragments into a novel series of low-micromolar inhibitors.  Our results provide attractive leads for development of BAZ2B chem. probes and indicate the whole family may be tractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcsl8yJ7AirVg90H21EOLACvtfcHk0lgvJpAayjBrQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltL%252FK&md5=e10f9392a72a2eb946cf4aabb0a27fb1</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjm401582c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401582c%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMuniz%26aufirst%3DJ.%2BR.%2BC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DHeightman%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520Low-Druggability%2520Bromodomains%253A%2520Fragment%2520Based%2520Screening%2520and%2520Inhibitor%2520Design%2520against%2520the%2520BAZ2B%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D24%26spage%3D10183%26epage%3D10187%26doi%3D10.1021%2Fjm401582c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Seal, J.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Gray, J. R. J.</span>; <span class="NLM_string-name">Woolven, J.</span>; <span class="NLM_string-name">Wall, I.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Rianjongdee, F.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Michon, A.-M.</span>; <span class="NLM_string-name">Grandi, P.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Jones, E. J.</span>; <span class="NLM_string-name">Craggs, P. D.</span>; <span class="NLM_string-name">Prinjha, R. K.</span>; <span class="NLM_string-name">Lindon, M.</span>; <span class="NLM_string-name">Demont, E. H.</span></span>, <span> </span><span class="NLM_article-title">The Design and Synthesis of a Highly Selective and In Vivo Capable Inhibitor of the Second Bromodomain (BD2) of the Bromodomain and Extra Terminal Domain (BET) Family of Proteins</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Preston&author=S.+J.+Atkinson&author=J.+Seal&author=D.+J.+Mitchell&author=R.+J.+Watson&author=J.+R.+J.+Gray&author=J.+Woolven&author=I.+Wall&author=C.-w.+Chung&author=P.+Bamborough&author=F.+Rianjongdee&author=S.+Taylor&author=A.-M.+Michon&author=P.+Grandi&author=I.+Rioja&author=L.+Gordon&author=E.+J.+Jones&author=P.+D.+Craggs&author=R.+K.+Prinjha&author=M.+Lindon&author=E.+H.+Demont&title=The+Design+and+Synthesis+of+a+Highly+Selective+and+In+Vivo+Capable+Inhibitor+of+the+Second+Bromodomain+%28BD2%29+of+the+Bromodomain+and+Extra+Terminal+Domain+%28BET%29+Family+of+Proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26atitle%3DThe%2520Design%2520and%2520Synthesis%2520of%2520a%2520Highly%2520Selective%2520and%2520In%2520Vivo%2520Capable%2520Inhibitor%2520of%2520the%2520Second%2520Bromodomain%2520%2528BD2%2529%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520Family%2520of%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2615</span>â <span class="NLM_lpage">2623</span>, <span class="refDoi">Â DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&issue=12&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+Properties+That+Influence+the+Oral+Bioavailability+of+Drug+Candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area â¤140 Ã2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lgvJpAayjBrQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520Properties%2520That%2520Influence%2520the%2520Oral%2520Bioavailability%2520of%2520Drug%2520Candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26issue%3D12%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">648</span>â <span class="NLM_lpage">655</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and â¼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lgaVjwtJgGWJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skipper, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wogan, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannenbaum, S. R.</span></span> <span> </span><span class="NLM_article-title">Monocyclic aromatic amines as potential human carcinogens: old is new again</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">50</span>â <span class="NLM_lpage">58</span>, <span class="refDoi">Â DOI: 10.1093/carcin/bgp267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1093%2Fcarcin%2Fbgp267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=19887514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=50-58&issue=1&author=P.+L.+Skipperauthor=M.+Y.+Kimauthor=H.+L.+P.+Sunauthor=G.+N.+Woganauthor=S.+R.+Tannenbaum&title=Monocyclic+aromatic+amines+as+potential+human+carcinogens%3A+old+is+new+again&doi=10.1093%2Fcarcin%2Fbgp267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyclic aromatic amines as potential human carcinogens: old is new again</span></div><div class="casAuthors">Skipper, Paul L.; Kim, Min Young; Sun, H.-L. Patty; Wogan, Gerald N.; Tannenbaum, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-58</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Alkylanilines are a group of chems. whose ubiquitous presence in the environment is a result of the multitude of sources from which they originate.  Exposure assessments indicate that most individuals experience lifelong exposure to these compds.  Many alkylanilines have biol. activity similar to that of the carcinogenic multi-ring arom. amines.  This review provides an overview of human exposure and biol. effects.  It also describes recent investigations into the biochem. mechanisms of action that lead to the assessment that they are most probably more complex than those of the more extensively investigated multi-ring arom. amines.  Not only is nitrenium ion chem. implicated in DNA damage by alkylanilines but also reactions involving quinone imines and perhaps reactive oxygen species.  Recent results described here indicate that alkylanilines can be potent genotoxins for cultured mammalian cells when activated by exogenous or endogenous phase I and phase II xenobiotic-metabolizing enzymes.  The nature of specific DNA damage products responsible for mutagenicity remains to be identified but evidence to date supports mechanisms of activation through obligatory N-hydroxylation as well as subsequent conjugation by sulfation and/or acetylation.  A fuller understanding of the mechanisms of alkylaniline genotoxicity is expected to provide important insights into the environmental and genetic origins of one or more human cancers and may reveal a substantial role for this group of compds. as potential human chem. carcinogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8jtJ_-C4s7Vg90H21EOLACvtfcHk0lgaVjwtJgGWJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D&md5=37e5a874a1ee9edf96acb3ad73ec2925</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp267%26sid%3Dliteratum%253Aachs%26aulast%3DSkipper%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DM.%2BY.%26aulast%3DSun%26aufirst%3DH.%2BL.%2BP.%26aulast%3DWogan%26aufirst%3DG.%2BN.%26aulast%3DTannenbaum%26aufirst%3DS.%2BR.%26atitle%3DMonocyclic%2520aromatic%2520amines%2520as%2520potential%2520human%2520carcinogens%253A%2520old%2520is%2520new%2520again%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26issue%3D1%26spage%3D50%26epage%3D58%26doi%3D10.1093%2Fcarcin%2Fbgp267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span> <span> </span><span class="NLM_article-title">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">929</span>, <span class="refDoi">Â DOI: 10.1038/nrd3287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrd3287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21119731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=929&author=D.+A.+Smithauthor=L.+Diauthor=E.+H.+Kerns&title=The+effect+of+plasma+protein+binding+on+in+vivo+efficacy%3A+misconceptions+in+drug+discovery&doi=10.1038%2Fnrd3287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span></div><div class="casAuthors">Smith, Dennis A.; Di, Li; Kerns, Edward H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">929-939</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Data from in vitro plasma protein binding expts. that det. the fraction of protein-bound drug are frequently used in drug discovery to guide structure design and to prioritize compds. for in vivo studies.  However, we consider that these practices are usually misleading, because in vivo efficacy is detd. by the free (unbound) drug concn. surrounding the therapeutic target, not by the free drug fraction.  These practices yield no enhancement of the in vivo free drug concn.  So, decisions based on free drug fraction could result in the wrong compds. being advanced through drug discovery programs.  This Perspective provides guidance on the application of plasma protein binding information in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOa3ippodhGbVg90H21EOLACvtfcHk0lgRWyVxPLxknQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI&md5=a45d00f68ab44580fa38a6acef49493a</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnrd3287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3287%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DThe%2520effect%2520of%2520plasma%2520protein%2520binding%2520on%2520in%2520vivo%2520efficacy%253A%2520misconceptions%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D929%26doi%3D10.1038%2Fnrd3287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manabe, K.</span></span> <span> </span><span class="NLM_article-title">Trichlorophenyl Formate: Highly Reactive and Easily Accessible Crystalline CO Surrogate for Palladium-Catalyzed Carbonylation of Aryl/Alkenyl Halides and Triflates</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">5370</span>â <span class="NLM_lpage">5373</span>, <span class="refDoi">Â DOI: 10.1021/ol302593z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol302593z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVejs7vP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=5370-5373&issue=20&author=T.+Uedaauthor=H.+Konishiauthor=K.+Manabe&title=Trichlorophenyl+Formate%3A+Highly+Reactive+and+Easily+Accessible+Crystalline+CO+Surrogate+for+Palladium-Catalyzed+Carbonylation+of+Aryl%2FAlkenyl+Halides+and+Triflates&doi=10.1021%2Fol302593z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Trichlorophenyl Formate: Highly Reactive and Easily Accessible Crystalline CO Surrogate for Palladium-Catalyzed Carbonylation of Aryl/Alkenyl Halides and Triflates</span></div><div class="casAuthors">Ueda, Tsuyoshi; Konishi, Hideyuki; Manabe, Kei</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5370-5373</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The high utility of 2,4,6-trichlorophenyl formate, a highly reactive and easily accessible cryst. CO surrogate, is demonstrated.  The decarbonylation with NEt3 to generate CO proceeded rapidly at rt, thereby allowing external-CO-free Pd-catalyzed carbonylation of aryl/alkenyl halides and triflates.  The high reactivity of the CO surrogate enabled carbonylation at rt and significantly reduced the quantities of formate to near-stoichiometric levels.  The obtained trichlorophenyl esters can be readily converted to a variety of carboxylic acid derivs. in high yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUQt4b7-33nrVg90H21EOLACvtfcHk0lgRWyVxPLxknQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVejs7vP&md5=74392d6c4847f88bd452016f5ddd08aa</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fol302593z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol302593z%26sid%3Dliteratum%253Aachs%26aulast%3DUeda%26aufirst%3DT.%26aulast%3DKonishi%26aufirst%3DH.%26aulast%3DManabe%26aufirst%3DK.%26atitle%3DTrichlorophenyl%2520Formate%253A%2520Highly%2520Reactive%2520and%2520Easily%2520Accessible%2520Crystalline%2520CO%2520Surrogate%2520for%2520Palladium-Catalyzed%2520Carbonylation%2520of%2520Aryl%252FAlkenyl%2520Halides%2520and%2520Triflates%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26issue%3D20%26spage%3D5370%26epage%3D5373%26doi%3D10.1021%2Fol302593z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoenberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartoletti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, R. F.</span></span> <span> </span><span class="NLM_article-title">Palladium-catalyzed carboalkoxylation of aryl, benzyl, and vinylic halides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">3318</span>â <span class="NLM_lpage">3326</span>, <span class="refDoi">Â DOI: 10.1021/jo00937a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00937a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaE2MXjvFah" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1974&pages=3318-3326&issue=23&author=A.+Schoenbergauthor=I.+Bartolettiauthor=R.+F.+Heck&title=Palladium-catalyzed+carboalkoxylation+of+aryl%2C+benzyl%2C+and+vinylic+halides&doi=10.1021%2Fjo00937a003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-catalyzed carboalkoxylation of aryl, benzyl, and vinylic halides</span></div><div class="casAuthors">Schoenberg, A.; Bartoletti, I.; Heck, R. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3318-26</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Aryl and vinylic bromides and iodides and benzyl chloride react with CO and an alc. at 100Â° or below and atm. pressure in the presence of a tertiary amine and a catalytic amt. of a palladium-triphenylphosphine complex to form esters.  The reaction is tolerant of a variety of functional groups and shows appreciable stereospecificity at 60-80Â° with cis and trans vinylic halides producing esters with retained configuration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcLSPgXtJ2HrVg90H21EOLACvtfcHk0lgRWyVxPLxknQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXjvFah&md5=bdcd89180b1e93776b8e6030f2ed6619</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjo00937a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00937a003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoenberg%26aufirst%3DA.%26aulast%3DBartoletti%26aufirst%3DI.%26aulast%3DHeck%26aufirst%3DR.%2BF.%26atitle%3DPalladium-catalyzed%2520carboalkoxylation%2520of%2520aryl%252C%2520benzyl%252C%2520and%2520vinylic%2520halides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1974%26volume%3D39%26issue%3D23%26spage%3D3318%26epage%3D3326%26doi%3D10.1021%2Fjo00937a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baburajan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elango, K. P.</span></span> <span> </span><span class="NLM_article-title">Co2(CO)8 as a convenient in situ CO source for the direct synthesis of benzamides from aryl halides (Br/I) via aminocarbonylation</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1006</span>â <span class="NLM_lpage">1010</span>, <span class="refDoi">Â DOI: 10.1016/j.tetlet.2013.12.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.tetlet.2013.12.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVKltA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=1006-1010&issue=5&author=P.+Baburajanauthor=K.+P.+Elango&title=Co2%28CO%298+as+a+convenient+in+situ+CO+source+for+the+direct+synthesis+of+benzamides+from+aryl+halides+%28Br%2FI%29+via+aminocarbonylation&doi=10.1016%2Fj.tetlet.2013.12.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Co2(CO)8 as a convenient in situ CO source for the direct synthesis of benzamides from aryl halides (Br/I) via aminocarbonylation</span></div><div class="casAuthors">Baburajan, Poongavanam; Elango, Kuppanagounder P.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1006-1010</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The direct synthesis of benzamides from aryl halides (Br, I) via aminocarbonylation, using solid Co2(CO)8 as a convenient CO source, was carried out.  The developed method is applicable to a wide variety of 1Â° and cyclic and acyclic 2Â° amines.  Nitro substituted (o, m and p) aryl halides were easily converted to the corresponding benzamides, without the redn. of the nitro group, in high yields using this in situ carbonylation methodol. under microwave irradn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWg2GxYmZpcrVg90H21EOLACvtfcHk0lh7-s3jxKFH1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVKltA%253D%253D&md5=85b67932aaa916be3111b3d29e3c3f7c</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2013.12.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2013.12.062%26sid%3Dliteratum%253Aachs%26aulast%3DBaburajan%26aufirst%3DP.%26aulast%3DElango%26aufirst%3DK.%2BP.%26atitle%3DCo2%2528CO%25298%2520as%2520a%2520convenient%2520in%2520situ%2520CO%2520source%2520for%2520the%2520direct%2520synthesis%2520of%2520benzamides%2520from%2520aryl%2520halides%2520%2528Br%252FI%2529%2520via%2520aminocarbonylation%26jtitle%3DTetrahedron%2520Lett.%26date%3D2014%26volume%3D55%26issue%3D5%26spage%3D1006%26epage%3D1010%26doi%3D10.1016%2Fj.tetlet.2013.12.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krasnaya, Z. A.</span></span> <span> </span><span class="NLM_article-title">Interaction of 5-Acetyl(alkoxycarbonyl)-3-alkoxycarbonyl-6-methylpyridin-2(1H)-ones with Primary Aromatic Amines and Hydrazine Hydrate</span>. <i>Chem. Heterocycl. Compd.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1155</span>â <span class="NLM_lpage">1161</span>, <span class="refDoi">Â DOI: 10.1023/B:COHC.0000048288.74767.6f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1023%2FB%3ACOHC.0000048288.74767.6f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWmu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2004&pages=1155-1161&issue=9&author=Z.+A.+Krasnaya&title=Interaction+of+5-Acetyl%28alkoxycarbonyl%29-3-alkoxycarbonyl-6-methylpyridin-2%281H%29-ones+with+Primary+Aromatic+Amines+and+Hydrazine+Hydrate&doi=10.1023%2FB%3ACOHC.0000048288.74767.6f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Reaction of 5-acetyl(alkoxycarbonyl)-3-alkoxycarbonyl-6-methylpyridin-2(1H)-ones with primary aromatic amines and hydrazine hydrate</span></div><div class="casAuthors">Krasnaya, Zh. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1155-1161</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer Science+Business Media, Inc.</span>)
        </div><div class="casAbstract">5-Acetyl(alkoxycarbonyl)-3-alkoxycarbonyl-6-methylpyridin-2(1H)-ones have been obtained from enaminocarbonyl compds.  Their reactions with benzylamine, Î±-phenylethylamine, and hydrazine hydrate have been studied, as a result of which a series of amides, hydrazides, and hydrazones of 2(1H)-pyridones has been synthesized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojTpi6AedYubVg90H21EOLACvtfcHk0lh7-s3jxKFH1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWmu7Y%253D&md5=a45bcc4f9e9446164c3757845afa346c</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1023%2FB%3ACOHC.0000048288.74767.6f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253ACOHC.0000048288.74767.6f%26sid%3Dliteratum%253Aachs%26aulast%3DKrasnaya%26aufirst%3DZ.%2BA.%26atitle%3DInteraction%2520of%25205-Acetyl%2528alkoxycarbonyl%2529-3-alkoxycarbonyl-6-methylpyridin-2%25281H%2529-ones%2520with%2520Primary%2520Aromatic%2520Amines%2520and%2520Hydrazine%2520Hydrate%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2004%26volume%3D40%26issue%3D9%26spage%3D1155%26epage%3D1161%26doi%3D10.1023%2FB%3ACOHC.0000048288.74767.6f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malmqvist, M.</span></span> <span> </span><span class="NLM_article-title">Biospecific interaction analysis using biosensor technology</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">186</span>, <span class="refDoi">Â DOI: 10.1038/361186a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2F361186a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=7678451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADyaK3s7jtlehtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=1993&pages=186&author=M.+Malmqvist&title=Biospecific+interaction+analysis+using+biosensor+technology&doi=10.1038%2F361186a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Biospecific interaction analysis using biosensor technology</span></div><div class="casAuthors">Malmqvist M</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">6408</span>),
    <span class="NLM_cas:pages">186-7</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Fundamental information that enhances our understanding of biospecific interactions can be obtained using a new analytical system based on biosensor technology.  The functional characteristics of biospecific interaction, such as kinetics, affinity and binding position, are examined by label-free analysis of proteins in free solution binding to an immobilized ligand at a hydrophilic sensor surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSqysTbpfmjRrjnyZkbE8yXfW6udTcc2eZilC4PMxCLCrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3s7jtlehtA%253D%253D&md5=23725c0430dd70b08f8c6d3751425703</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2F361186a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F361186a0%26sid%3Dliteratum%253Aachs%26aulast%3DMalmqvist%26aufirst%3DM.%26atitle%3DBiospecific%2520interaction%2520analysis%2520using%2520biosensor%2520technology%26jtitle%3DNature%26date%3D1993%26volume%3D361%26spage%3D186%26doi%3D10.1038%2F361186a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ames, B.</span></span> <span> </span><span class="NLM_article-title">Identifying environmental chemicals causing mutations and cancer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>204</i></span> (<span class="NLM_issue">4393</span>),  <span class="NLM_fpage">587</span>, <span class="refDoi">Â DOI: 10.1126/science.373122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1126%2Fscience.373122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=373122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=1979&pages=587&issue=4393&author=B.+Ames&title=Identifying+environmental+chemicals+causing+mutations+and+cancer&doi=10.1126%2Fscience.373122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying environmental chemicals causing mutations and cancer</span></div><div class="casAuthors">Ames, Bruce N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">4393</span>),
    <span class="NLM_cas:pages">587-93</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 57 refs. on environmental chem. induced carcinogenesis and mutagenesis and their relation to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2AzWl_6PJ4bVg90H21EOLACvtfcHk0li3mt2SQ5L2gQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D&md5=9ac98bad1482e9cda849723a71f32a9d</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1126%2Fscience.373122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.373122%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DB.%26atitle%3DIdentifying%2520environmental%2520chemicals%2520causing%2520mutations%2520and%2520cancer%26jtitle%3DScience%26date%3D1979%26volume%3D204%26issue%3D4393%26spage%3D587%26doi%3D10.1126%2Fscience.373122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clive, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machesko, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernheim, N. J.</span></span> <span> </span><span class="NLM_article-title">A mutational assay system using the thymidine kinase locus in mouse lymphoma cells</span>. <i>Mutat. Res., Fundam. Mol. Mech. Mutagen.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">77</span>â <span class="NLM_lpage">87</span>, <span class="refDoi">Â DOI: 10.1016/0027-5107(72)90066-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2F0027-5107%2872%2990066-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=5053587" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1972&pages=77-87&issue=1&author=D.+Cliveauthor=W.+G.+Flammauthor=M.+R.+Macheskoauthor=N.+J.+Bernheim&title=A+mutational+assay+system+using+the+thymidine+kinase+locus+in+mouse+lymphoma+cells&doi=10.1016%2F0027-5107%2872%2990066-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2F0027-5107%2872%2990066-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0027-5107%252872%252990066-8%26sid%3Dliteratum%253Aachs%26aulast%3DClive%26aufirst%3DD.%26aulast%3DFlamm%26aufirst%3DW.%2BG.%26aulast%3DMachesko%26aufirst%3DM.%2BR.%26aulast%3DBernheim%26aufirst%3DN.%2BJ.%26atitle%3DA%2520mutational%2520assay%2520system%2520using%2520the%2520thymidine%2520kinase%2520locus%2520in%2520mouse%2520lymphoma%2520cells%26jtitle%3DMutat.%2520Res.%252C%2520Fundam.%2520Mol.%2520Mech.%2520Mutagen.%26date%3D1972%26volume%3D16%26issue%3D1%26spage%3D77%26epage%3D87%26doi%3D10.1016%2F0027-5107%2872%2990066-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M.</span></span> <i>Mouse Lymphoma Assay</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_fpage">323</span>â <span class="NLM_lpage">342</span>, <span class="refDoi">Â DOI: 10.1007/978-1-4939-1068-7_19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1007%2F978-1-4939-1068-7_19" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Chen%2C+T.%3B+Guo%2C+X.%3B+Moore%2C+M.+Mouse+Lymphoma+Assay+2014%2C+323%E2%80%93342%2C+10.1007%2F978-1-4939-1068-7_19"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-1068-7_19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-1068-7_19%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DMoore%26aufirst%3DM.%26jtitle%3DMouse%2520Lymphoma%2520Assay%26date%3D2014%26spage%3D323%26epage%3D342%26doi%3D10.1007%2F978-1-4939-1068-7_19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stieger, B.</span></span> <span> </span><span class="NLM_article-title">Role of the bile salt export pump, BSEP, in acquired forms of cholestasis</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">437</span>â <span class="NLM_lpage">445</span>, <span class="refDoi">Â DOI: 10.3109/03602530903492004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.3109%2F03602530903492004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=20028269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotlSmt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2010&pages=437-445&issue=3&author=B.+Stieger&title=Role+of+the+bile+salt+export+pump%2C+BSEP%2C+in+acquired+forms+of+cholestasis&doi=10.3109%2F03602530903492004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the bile salt export pump, BSEP, in acquired forms of cholestasis</span></div><div class="casAuthors">Stieger, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-445</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Generation of bile is a key function of the liver.  Its impairment leads to accumulation of cytotoxic bile salts in hepatocytes and, consequently, to liver disease.  The bile salt export pump, BSEP, is critically involved in the secretion of bile salts into bile.  Its function can be disturbed or abolished by inherited mutations.  This will lead to progressive intrahepatic cholestasis and severe liver disease.  In addn. to mutations, BSEP can be inhibited by acquired factors, such as xenobiotics or drugs, aberrant bile salt metabolites, or pregnancy.  This inhibition will lead to acquired cholestasis.  Some drugs are now known to be competitive inhibitors of Bsep.  In addn., a polymorphism in the gene coding for BSEP has been identified as a potential susceptibility factor for acquired cholestasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYni5fpYpR8bVg90H21EOLACvtfcHk0li3mt2SQ5L2gQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotlSmt7w%253D&md5=4ea89d1fbe6a1da4933b3f72a6c212f9</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.3109%2F03602530903492004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602530903492004%26sid%3Dliteratum%253Aachs%26aulast%3DStieger%26aufirst%3DB.%26atitle%3DRole%2520of%2520the%2520bile%2520salt%2520export%2520pump%252C%2520BSEP%252C%2520in%2520acquired%2520forms%2520of%2520cholestasis%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2010%26volume%3D42%26issue%3D3%26spage%3D437%26epage%3D445%26doi%3D10.3109%2F03602530903492004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span> <span> </span><span class="NLM_article-title">Metabolism and Toxicity of Drugs. Two Decades of Progress in Industrial Drug Metabolism</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">129</span>â <span class="NLM_lpage">137</span>, <span class="refDoi">Â DOI: 10.1021/tx7002273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx7002273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=129-137&issue=1&author=T.+A.+Baillie&title=Metabolism+and+Toxicity+of+Drugs.+Two+Decades+of+Progress+in+Industrial+Drug+Metabolism&doi=10.1021%2Ftx7002273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism and Toxicity of Drugs. Two Decades of Progress in Industrial Drug Metabolism</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-137</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The science of drug metab. and pharmacokinetics (DMPK) has developed significantly over the past 20 years, and its functional role in today's pharmaceutical industry has matured to the point where DMPK has become an indispensable discipline in support of drug discovery and development.  While contributions to the lead optimization phase of discovery efforts have been particularly noteworthy in helping to select only the best drug candidates for entry into development, it should be recognized that the scope of DMPK spans the continuum of discovery through clin. evaluation and even into the post-marketing phase; as such, the breadth of DMPK's involvement is almost unique in contemporary pharmaceutical research.  This perspective summarizes notable advances in the field, many of which have been made possible by technol. developments in areas such as mol. biol., genetics, and bioanal. chem., and highlights the crit. nature of key partnerships between Drug Metab., Medicinal Chem., and Safety Assessment groups in attempting to advance drug candidates with a low potential for causing adverse events in humans.  Finally, some speculative predictions are made of the future role of DMPK in pharmaceutical research, where current advances in our mechanistic understanding of the mol. processes that control the absorption, disposition, metab., elimination, and toxicity of drugs and their biotransformation products will combine to further enhance the impact of DMPK in drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUrTeLhTDYbVg90H21EOLACvtfcHk0ljcpiHnc1iKmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF&md5=e8f659251bf82cb0b1602855e725c390</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Ftx7002273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx7002273%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DMetabolism%2520and%2520Toxicity%2520of%2520Drugs.%2520Two%2520Decades%2520of%2520Progress%2520in%2520Industrial%2520Drug%2520Metabolism%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26issue%3D1%26spage%3D129%26epage%3D137%26doi%3D10.1021%2Ftx7002273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masubuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horie, T.</span></span> <span> </span><span class="NLM_article-title">Toxicological Significance of Mechanism-Based Inactivation of Cytochrome P450 Enzymes by Drugs</span>. <i>Crit. Rev. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">389</span>â <span class="NLM_lpage">412</span>, <span class="refDoi">Â DOI: 10.1080/10408440701215233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1080%2F10408440701215233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=17612953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1altr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=389-412&issue=5&author=Y.+Masubuchiauthor=T.+Horie&title=Toxicological+Significance+of+Mechanism-Based+Inactivation+of+Cytochrome+P450+Enzymes+by+Drugs&doi=10.1080%2F10408440701215233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicological significance of mechanism-based inactivation of cytochrome P450 enzymes by drugs</span></div><div class="casAuthors">Masubuchi, Yasuhiro; Horie, Toshiharu</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">389-412</span>CODEN:
                <span class="NLM_cas:coden">CRTXB2</span>;
        ISSN:<span class="NLM_cas:issn">1040-8444</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Cytochrome P 450 (P 450) enzymes oxidize xenobiotics into chem. reactive metabolites or intermediates as well as into stable metabolites.  If the reactivity of the product is very high, it binds to a catalytic site or sites of the enzyme itself and inactivates it.  This phenomenon is referred to as mechanism-based inactivation.  Many clin. important drugs are mechanism-based inactivators that include macrolide antibiotics, calcium channel blockers, and selective serotonin uptake inhibitors, but are not always structurally and pharmacol. related.  The inactivation of P450s during drug therapy results in serious drug interactions, since irreversibility of the binding allows enzyme inhibition to be prolonged after elimination of the causal drug.  The inhibition of the metab. of drugs with narrow therapeutic indexes, such as terfenadine and astemizole, leads to toxicities.  On the other hand, the fate of P450s after the inactivation and the toxicol. consequences remains to be elucidated, while it has been suggested that P450s modified and degraded are involved in some forms of tissue toxicity.  Porphyrinogenic drugs, such as griseofulvin, cause mechanism-based heme inactivation, leading to formation of ferrochelatase-inhibitory N-alkylated protoporphyrins and resulting in porphyria.  Involvement of P 450-derived free heme in halothane-induced hepatotoxicity and catalytic iron in cisplatin-induced nephrotoxicity has also been suggested.  Autoantibodies against P450s have been found in hepatitis following administration of tienilic acid and dihydralazine.  Tienilic acid is activated by and covalently bound to CYP2C9, and the neoantigens thus formed activate immune systems, resulting in the formation of an autoantibody directed against CYP2C9, named anti-liver/kidney microsomal autoantibody type 2, whereas the pathol. role of the autoantibodies in drug-induced hepatitis remains largely unknown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmbxxZLDTkDLVg90H21EOLACvtfcHk0ljcpiHnc1iKmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1altr4%253D&md5=bd451ac2e6f723ea0a537b27187ef7ba</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1080%2F10408440701215233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408440701215233%26sid%3Dliteratum%253Aachs%26aulast%3DMasubuchi%26aufirst%3DY.%26aulast%3DHorie%26aufirst%3DT.%26atitle%3DToxicological%2520Significance%2520of%2520Mechanism-Based%2520Inactivation%2520of%2520Cytochrome%2520P450%2520Enzymes%2520by%2520Drugs%26jtitle%3DCrit.%2520Rev.%2520Toxicol.%26date%3D2007%26volume%3D37%26issue%3D5%26spage%3D389%26epage%3D412%26doi%3D10.1080%2F10408440701215233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, M.</span></span> <span> </span><span class="NLM_article-title">Drug-Mediated Inactivation of Cytochrome P450</span>. <i>Clin. Exp. Pharmacol. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">465</span>â <span class="NLM_lpage">470</span>, <span class="refDoi">Â DOI: 10.1111/j.1440-1681.1997.tb01228.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1111%2Fj.1440-1681.1997.tb01228.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=9248661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaK2sXkvVahtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1997&pages=465-470&issue=7&author=M.+Murray&title=Drug-Mediated+Inactivation+of+Cytochrome+P450&doi=10.1111%2Fj.1440-1681.1997.tb01228.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-mediated inactivation of cytochrome P450</span></div><div class="casAuthors">Murray, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Experimental Pharmacology and Physiology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-470</span>CODEN:
                <span class="NLM_cas:coden">CEXPB9</span>;
        ISSN:<span class="NLM_cas:issn">0305-1870</span>.
    
            (<span class="NLM_cas:orgname">Blackwell</span>)
        </div><div class="casAbstract">A review with 62 refs.  Multiple forms of cytochrome P 450 (CYP) catalyze the oxidn. of chems. of endogenous and exogenous origin, including drugs, carcinogens, steroids and eicosanoids.  However, this unusual low substrate specificity also makes CYP susceptible to inhibition by a wide range of drugs, leading to pharmacokinetic interactions of potential clin. significance.  Some drugs are converted by CYP to reactive metabolites that bind covalently to sites within the active center of the same CYP.  Such mechanism-based inhibition leads to CYP inactivation or complexation.  These processes give rise to longterm effects on drug pharmacokinetics, as the inactivated or complexed CYP must be replaced by newly synthesized CYP protein.  Drugs that inactivate CYP generally possess recognizable functional groups that are oxidized to reactive products.  Thus, drugs with side chains contg. unsatd. carbon-carbon bonds and furan ring systems are assocd. with CYP inactivation.  Nitrogen-contg. systems may also inactivate CYP.  Metabolites formed from drugs contg. alkylamino and methylenedioxy functionalities can trap CYP as inert complexes without eliciting inactivation.  However, the functional effects of inactivation and complexation on drug pharmacokinetics are indistinguishable.  Drugs that elicit CYP complexation include the first generation macrolide antibiotics, but newer analogs appear much safer.  Some antidepressants, antiepileptics and tuberculostatic agents have been assocd. with CYP complexation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuVcGKdWFUrVg90H21EOLACvtfcHk0ljcpiHnc1iKmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvVahtbs%253D&md5=8a991524e90fc66ec7475a2f19259804</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1111%2Fj.1440-1681.1997.tb01228.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1440-1681.1997.tb01228.x%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DM.%26atitle%3DDrug-Mediated%2520Inactivation%2520of%2520Cytochrome%2520P450%26jtitle%3DClin.%2520Exp.%2520Pharmacol.%2520Physiol.%26date%3D1997%26volume%3D24%26issue%3D7%26spage%3D465%26epage%3D470%26doi%3D10.1111%2Fj.1440-1681.1997.tb01228.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh-Bakht, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâDonnell, J. P.</span></span> <span> </span><span class="NLM_article-title">Biotransformation Reactions of Five-Membered Aromatic Heterocyclic Rings</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">269</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1021/tx015574b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx015574b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2002&pages=269-299&issue=3&author=D.+K.+Dalvieauthor=A.+S.+Kalgutkarauthor=S.+C.+Khojasteh-Bakhtauthor=R.+S.+Obachauthor=J.+P.+O%E2%80%99Donnell&title=Biotransformation+Reactions+of+Five-Membered+Aromatic+Heterocyclic+Rings&doi=10.1021%2Ftx015574b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Biotransformation Reactions of Five-Membered Aromatic Heterocyclic Rings</span></div><div class="casAuthors">Dalvie, Deepak K.; Kalgutkar, Amit S.; Khojasteh-Bakht, S. Cyrus; Obach, R. Scott; O'Donnell, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-299</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 278 refs. is given on the biotransformation pathways of most commonly used 5-membered arom. heterocyclic rings.  The effect of physicochem. properties such as the electronic effects of heteroatoms, aromaticity, and acidity/basicity (pKa) of the rings are correlated on their biotransformation.  The generation of reactive metabolites following oxidn. or redn. of these heterocycles and the subsequent toxicol. consequences are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNaEz19aehbVg90H21EOLACvtfcHk0li00NG-Qgj8hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D&md5=e8e2dfd12772f9ff2289b173bdefa590</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Ftx015574b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx015574b%26sid%3Dliteratum%253Aachs%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DKhojasteh-Bakht%26aufirst%3DS.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26atitle%3DBiotransformation%2520Reactions%2520of%2520Five-Membered%2520Aromatic%2520Heterocyclic%2520Rings%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2002%26volume%3D15%26issue%3D3%26spage%3D269%26epage%3D299%26doi%3D10.1021%2Ftx015574b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshmane, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremlev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawaya, B. E.</span></span> <span> </span><span class="NLM_article-title">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span>. <i>J. Interferon Cytokine Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">313</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1089/jir.2008.0027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1089%2Fjir.2008.0027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=19441883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=313-326&issue=6&author=S.+L.+Deshmaneauthor=S.+Kremlevauthor=S.+Aminiauthor=B.+E.+Sawaya&title=Monocyte+Chemoattractant+Protein-1+%28MCP-1%29%3A+An+Overview&doi=10.1089%2Fjir.2008.0027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span></div><div class="casAuthors">Deshmane, Satish L.; Kremlev, Sergey; Amini, Shohreh; Sawaya, Bassel E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Interferon & Cytokine Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">313-326</span>CODEN:
                <span class="NLM_cas:coden">JICRFJ</span>;
        ISSN:<span class="NLM_cas:issn">1079-9907</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Chemokines constitute a family of chemoattractant cytokines and are subdivided into four families on the basis of the no. and spacing of the conserved cysteine residues in the N-terminus of the protein.  Chemokines play a major role in selectively recruiting monocytes, neutrophils, and lymphocytes, as well as in inducing chemotaxis through the activation of G-protein-coupled receptors.  Monocyte chemoattractant protein-1 (MCP-1/CCL2) is one of the key chemokines that regulate migration and infiltration of monocytes/macrophages.  Both CCL2 and its receptor CCR2 have been demonstrated to be induced and involved in various diseases.  Migration of monocytes from the blood stream across the vascular endothelium is required for routine immunol. surveillance of tissues, as well as in response to inflammation.  This review will discuss these biol. processes and the structure and function of CCL2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSmnT35tYDZLVg90H21EOLACvtfcHk0li00NG-Qgj8hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D&md5=d58ea0c1925c2eff93631916f93edad3</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1089%2Fjir.2008.0027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fjir.2008.0027%26sid%3Dliteratum%253Aachs%26aulast%3DDeshmane%26aufirst%3DS.%2BL.%26aulast%3DKremlev%26aufirst%3DS.%26aulast%3DAmini%26aufirst%3DS.%26aulast%3DSawaya%26aufirst%3DB.%2BE.%26atitle%3DMonocyte%2520Chemoattractant%2520Protein-1%2520%2528MCP-1%2529%253A%2520An%2520Overview%26jtitle%3DJ.%2520Interferon%2520Cytokine%2520Res.%26date%3D2009%26volume%3D29%26issue%3D6%26spage%3D313%26epage%3D326%26doi%3D10.1089%2Fjir.2008.0027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. W.</span>; <span class="NLM_string-name">Fotouhi, N.</span>; <span class="NLM_string-name">Gillespie, P.</span>; <span class="NLM_string-name">Goodnow, R. A.,  Jr.</span>; <span class="NLM_string-name">Guertin, K. R.</span>; <span class="NLM_string-name">Haynes, N.-E.</span>; <span class="NLM_string-name">Myers, M. P.</span>; <span class="NLM_string-name">Pietranico-Cole, S. L.</span>; <span class="NLM_string-name">Qi, L.</span>; <span class="NLM_string-name">Rossman, P. L.</span>; <span class="NLM_string-name">Scott, N. R.</span>; <span class="NLM_string-name">Thakkar, K. C.</span>; <span class="NLM_string-name">Tilley, J. W.</span>; <span class="NLM_string-name">Zhang, Q.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrazole-4-carboxamide Derivatives as 11-Beta-Hydroxysteroid Dehydrogenase Form I (11-beta-HSD1) Inhibitors</span>. World Patent <span class="NLM_patent">WO2007107470A2</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=K.+W.+Anderson&author=N.+Fotouhi&author=P.+Gillespie&author=R.+A.+Goodnow&author=K.+R.+Guertin&author=N.-E.+Haynes&author=M.+P.+Myers&author=S.+L.+Pietranico-Cole&author=L.+Qi&author=P.+L.+Rossman&author=N.+R.+Scott&author=K.+C.+Thakkar&author=J.+W.+Tilley&author=Q.+Zhang&title=Preparation+of+Pyrazole-4-carboxamide+Derivatives+as+11-Beta-Hydroxysteroid+Dehydrogenase+Form+I+%2811-beta-HSD1%29+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DK.%2BW.%26atitle%3DPreparation%2520of%2520Pyrazole-4-carboxamide%2520Derivatives%2520as%252011-Beta-Hydroxysteroid%2520Dehydrogenase%2520Form%2520I%2520%252811-beta-HSD1%2529%2520Inhibitors%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camurri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaramella, A.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3716</span>â <span class="NLM_lpage">3722</span>, <span class="refDoi">Â DOI: 10.1021/ac001388j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac001388j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=3716-3722&issue=15&author=G.+Camurriauthor=A.+Zaramella&title=High-Throughput+Liquid+Chromatography%2FMass+Spectrometry+Method+for+the+Determination+of+the+Chromatographic+Hydrophobicity+Index&doi=10.1021%2Fac001388j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span></div><div class="casAuthors">Camurri, Giulio; Zaramella, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3716-3722</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast gradient reversed-phase liq. chromatog. (LC) method, using an acetonitrile gradient was developed to det. the chromatog. hydrophobicity index (CHI), as reported by Valco et al. (Anal. Chem. 1997, 69, 2022-2029).The anal. method provides retention times, based on UV detection at two different wavelengths, which then are converted into CHI values after calibration with a set of test compds.  The CHI of each compd. is measured at three different pH values, 2.0, 7.4, and 10.5; so using an 8-min gradient at each pH value one compd. can be analyzed in â¼24 min.  The aim of this work is to improve the throughput of the CHI screening using a LC/MS approach, so the application of the LC/MS technique is an extension of the LC/UV approach previously reported by Valco et al.  This approach allows contemporary injection of N compds. into the LC/MS system, the retention time of each compd. can be then extd. from the selected ion recording chromatograms.  The throughput of the existing screening method could be increased by N times, where N is the no. of compds. injected, so only three runs are needed to det. the CHI at three different pH values for a set of N compds.  The highest value of N depends on the total no. of channels that can be monitored simultaneously; 32 channels were used.  This LC/MS method was tested for a no. of com. products analyzed as mixts., and data obtained were compared with those coming from the classical LC/UV approach.  In the same way, the method was tested for a no. of compds. assocd. with two GlaxoWellcome projects in the antibacterial area.  The LC/MS method can be successfully applied for analyzing compds. in mixts. and for compds. with poor UV absorption, which cannot be analyzed with the std. LC/UV method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpqYuIDpy5rVg90H21EOLACvtfcHk0liiZ41Wpqn6Uw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D&md5=8773c87e08ba7ddce3c2d679360a6220</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fac001388j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac001388j%26sid%3Dliteratum%253Aachs%26aulast%3DCamurri%26aufirst%3DG.%26aulast%3DZaramella%26aufirst%3DA.%26atitle%3DHigh-Throughput%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520the%2520Determination%2520of%2520the%2520Chromatographic%2520Hydrophobicity%2520Index%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26issue%3D15%26spage%3D3716%26epage%3D3722%26doi%3D10.1021%2Fac001388j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. P.</span></span> <span> </span><span class="NLM_article-title">Fast Gradient HPLC Method to Determine Compounds Binding to Human Serum Albumin. Relationships with Octanol/Water and Immobilized Artificial Membrane Lipophilicity</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2236</span>â <span class="NLM_lpage">2248</span>, <span class="refDoi">Â DOI: 10.1002/jps.10494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1002%2Fjps.10494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=14603509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=2236-2248&issue=11&author=K.+Valkoauthor=S.+Nunhuckauthor=C.+Bevanauthor=M.+H.+Abrahamauthor=D.+P.+Reynolds&title=Fast+Gradient+HPLC+Method+to+Determine+Compounds+Binding+to+Human+Serum+Albumin.+Relationships+with+Octanol%2FWater+and+Immobilized+Artificial+Membrane+Lipophilicity&doi=10.1002%2Fjps.10494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span></div><div class="casAuthors">Valko, Klara; Nunhuck, Shenaz; Bevan, Chris; Abraham, Michael H.; Reynolds, Derek P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2236-2248</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A fast gradient HPLC method (cycle time 15 min) has been developed to det. Human Serum Albumin (HSA) binding of discovery compds. using chem. bonded protein stationary phases.  The HSA binding values were derived from the gradient retention times that were converted to the logarithm of the equil. consts. (logK HSA) using data from a calibration set of mols.  The method has been validated using literature plasma protein binding data of 68 known drug mols.  The method is fully automated, and has been used for lead optimization in more than 20 company projects.  The HSA binding data obtained for more than 4000 compds. were suitable to set up global and project specific quant. structure binding relationships that helped compd. design in early drug discovery.  The obtained HSA binding of known drug mols. were compared to the Immobilizd Artificial Membrane binding data (CHI IAM) obtained by our previously described HPLC-based method.  The solvation equation approach has been used to characterize the normal binding ability of HSA, and this relationship shows that compd. lipophilicity is a significant factor.  It was found that the selectivity of the "baseline" lipophilicity governing HSA binding, membrane interaction, and octanol/water partition are very similar.  However, the effect of the presence of pos. or neg. charges have very different effects.  It was found that neg. charged compds. bind more strongly to HSA than it would be expected from the lipophilicity of the ionized species at pH 7.4.  Several compds. showed stronger HSA binding than can be expected from their lipophilicity alone, and comparison between predicted and exptl. binding affinity allows the identification of compds. that have good complementarities with any of the known binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1LUBLdv-t7Vg90H21EOLACvtfcHk0liiZ41Wpqn6Uw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D&md5=14eb1e0f0ad616846a2b5d1428133fa9</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fjps.10494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10494%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26atitle%3DFast%2520Gradient%2520HPLC%2520Method%2520to%2520Determine%2520Compounds%2520Binding%2520to%2520Human%2520Serum%2520Albumin.%2520Relationships%2520with%2520Octanol%252FWater%2520and%2520Immobilized%2520Artificial%2520Membrane%2520Lipophilicity%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26issue%3D11%26spage%3D2236%26epage%3D2248%26doi%3D10.1002%2Fjps.10494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for the ATAD2 Bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11382</span>â <span class="NLM_lpage">11386</span>, <span class="refDoi">Â DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&issue=38&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+Chemical+Probe+for+the+ATAD2+Bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0liiZ41Wpqn6Uw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520the%2520ATAD2%2520Bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26issue%3D38%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">6151</span>â <span class="NLM_lpage">6178</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&issue=15&author=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-Based+Optimization+of+Naphthyridones+into+Potent+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0liFJn73f9AFzw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Naphthyridones%2520into%2520Potent%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D15%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Petrakis N. Chalikidi, Taimuraz T. Magkoev, Andrey V. Gutnov, Oleg P. Demidov, Maxim G. Uchuskin, Igor V. Trushkov, <span class="NLM_string-name hlFld-ContribAuthor">Vladimir T. Abaev</span>. </span><span class="cited-content_cbyCitation_article-title">One-Step Synthesis of Triphenylphosphonium Salts from (Het)arylmethyl Alcohols. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (14)
                                     , 9838-9846. <a href="https://doi.org/10.1021/acs.joc.1c00733" title="DOI URL">https://doi.org/10.1021/acs.joc.1c00733</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.1c00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.1c00733%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DOne-Step%252BSynthesis%252Bof%252BTriphenylphosphonium%252BSalts%252Bfrom%252B%252528Het%252529arylmethyl%252BAlcohols%26aulast%3DChalikidi%26aufirst%3DPetrakis%2BN.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D29032021%26date%3D07072021%26volume%3D86%26issue%3D14%26spage%3D9838%26epage%3D9846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francesco Rianjongdee, Stephen J. Atkinson, Chun-wa Chung, Paola Grandi, James R. J. Gray, Laura J. Kaushansky, Patricia Medeiros, Cassie Messenger, Alex Phillipou, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Alexander L. Satz, Simon Taylor, Ian D. Wall, Robert J. Watson, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00412%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BSelective%252BBET%252BBD2%252BInhibitor%252Bfrom%252Ba%252BDNA-Encoded%252BLibrary%252BTechnology%252BScreening%252BHit%26aulast%3DRianjongdee%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26date%3D08032021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Helen E. Aylott, Stephen J. Atkinson, Paul Bamborough, Anna Bassil, Chun-wa Chung, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Cassie Messenger, Darren Mitchell, Alexander Phillipou, Alex Preston, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan T. Seal, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Template-Hopping Approach Leads to Potent, Selective, and Highly Soluble Bromo and Extraterminal Domain (BET) Second Bromodomain (BD2) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3249-3281. <a href="https://doi.org/10.1021/acs.jmedchem.0c02156" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTemplate-Hopping%252BApproach%252BLeads%252Bto%252BPotent%25252C%252BSelective%25252C%252Band%252BHighly%252BSoluble%252BBromo%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitors%26aulast%3DAylott%26aufirst%3DHelen%2BE.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D04032021%26volume%3D64%26issue%3D6%26spage%3D3249%26epage%3D3281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Sunil Bhakta, Nicole Blaquiere, Jinhua Chen, Josefa dela Cruz-Chuh, Karen E. Gascoigne, Steven J. Hartman, Mingtao He, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liang Liu, Liling Liu, Qi Liu, Ying Lu, Fanwei Meng, Melinda M. Mulvihill, Aimee OâDonohue, Rebecca K. Rowntree, Leanna R. Staben, Steven T. Staben, John Wai, Jian Wang, BinQing Wei, Catherine Wilson, Jianfeng Xin, Zijin Xu, Hui Yao, Donglu Zhang, Hongyan Zhang, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 1: Exploration of Antibody Linker, Payload Loading, and Payload Molecular Properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2534-2575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01845" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01845</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01845%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B1%25253A%252BExploration%252Bof%252BAntibody%252BLinker%25252C%252BPayload%252BLoading%25252C%252Band%252BPayload%252BMolecular%252BProperties%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2534%26epage%3D2575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Jinhua Chen, Nicholas Corr, Josefa dela Cruz-Chuh, Geoffrey Del Rosario, Aaron Fullerton, Steven J. Hartman, Fan Jiang, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liling Liu, Ying Lu, Melinda M. Mulvihill, Jeremy M. Murray, Aimee OâDonohue, Rebecca K. Rowntree, William S. Sawyer, Leanna R. Staben, John Wai, Jian Wang, BinQing Wei, Wentao Wei, Zijin Xu, Hui Yao, Shang-Fan Yu, Donglu Zhang, Hongyan Zhang, Shenhua Zhang, Yongxin Zhao, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2576-2607. <a href="https://doi.org/10.1021/acs.jmedchem.0c01846" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01846</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01846%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B2%25253A%252BImprovement%252Bof%252BIn%252BVitro%252BAntiproliferation%252BActivity%252Band%252BIn%252BVivo%252BAntitumor%252BEfficacy%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2576%26epage%3D2607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stuart J. Conway,  (<span class="NLM_role">Associate Editor, <i>Journal of Medicinal Chemistry</i></span>)Paola Arimondo,  (<span class="NLM_role">Guest Editor, Institut Pasteur Paris</span>)Cheryl Arrowsmith,  (<span class="NLM_role">Guest Editor, University of Toronto</span>)Jian Jin,  (<span class="NLM_role">Guest Editor, Icahn School of Medicine at Mount Sinai</span>)Cheng Luo,  (<span class="NLM_role">Guest Editor, Shanghai Institute of Materia Medica</span>)Nicholas Meanwell,  (<span class="NLM_role">Perspectives Editor, <i>Journal of Medicinal Chemistry</i></span>)Wendy Young,  (<span class="NLM_role">Drug Annotations Editor, <i>Journal of Medicinal Chemistry</i></span>)Gunda Georg,  (<span class="NLM_role">Editor-in-Chief, <i>Journal of Medicinal Chemistry</i></span>)<span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span> (<span class="NLM_role">Editor-in-Chief, <i>Journal of Medicinal Chemistry</i></span>). </span><span class="cited-content_cbyCitation_article-title">Epigenetics 2.0: Special Issue on EpigeneticsâCall for Papers. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12129-12130. <a href="https://doi.org/10.1021/acs.jmedchem.0c01748" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01748</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01748%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEpigenetics%252B2.0%25253A%252BSpecial%252BIssue%252Bon%252BEpigenetics%2525E2%252580%252594Call%252Bfor%252BPapers%26aulast%3DConway%26aufirst%3DStuart%2BJ.%26date%3D2020%26date%3D2020%26date%3D22102020%26volume%3D63%26issue%3D21%26spage%3D12129%26epage%3D12130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Paul Bamborough, Sharon G. Bernard, Chun-wa Chung, Peter D. Craggs, Leanne Cutler, Emmanuel H. Demont, John P. Evans, Laurie Gordon, Bhumika Karamshi, Antonia J. Lewis, Matthew J. Lindon, Darren J. Mitchell, Inmaculada Rioja, Peter E. Soden, Simon Taylor, Robert J. Watson, Rob Willis, James M. Woolven, Beata S. WyspiaÅska, William J. Kerr, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9020-9044. <a href="https://doi.org/10.1021/acs.jmedchem.0c00566" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00566</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BInhibitor%252BSelective%252Bfor%252Bthe%252BN-Terminal%252BBromodomains%252BThat%252BRetains%252Ban%252BAnti-inflammatory%252Band%252BAntiproliferative%252BPhenotype%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06042020%26date%3D03082020%26volume%3D63%26issue%3D17%26spage%3D9020%26epage%3D9044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alex Preston, Stephen Atkinson, Paul Bamborough, Chun-wa Chung, Peter D. Craggs, Laurie Gordon, Paola Grandi, James R. J. Gray, Emma J. Jones, Matthew Lindon, Anne-Marie Michon, Darren J. Mitchell, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan Seal, Simon Taylor, Ian Wall, Robert J. Watson, James Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9070-9092. <a href="https://doi.org/10.1021/acs.jmedchem.0c00605" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00605</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252Ba%252BHighly%252BSelective%252Band%252BIn%252BVivo-Capable%252BInhibitor%252Bof%252Bthe%252BSecond%252BBromodomain%252Bof%252Bthe%252BBromodomain%252Band%252BExtra%252BTerminal%252BDomain%252BFamily%252Bof%252BProteins%26aulast%3DPreston%26aufirst%3DAlex%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16042020%26date%3D20082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9070%26epage%3D9092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Watson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Rob Davis, Laurie Gordon, Paola Grandi, Massimo Petretich, Alex Phillipou, Rab K. Prinjha, Inmaculada Rioja, Peter Soden, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9045-9069. <a href="https://doi.org/10.1021/acs.jmedchem.0c00614" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGSK789%25253A%252BA%252BSelective%252BInhibitor%252Bof%252Bthe%252BFirst%252BBromodomains%252B%252528BD1%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DWatson%26aufirst%3DRobert%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D04082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9045%26epage%3D9069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alex Preston, Stephen J. Atkinson, Paul Bamborough, Chun-wa Chung, Laurie J. Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Antonia J. Lewis, David Lugo, Cassie Messenger, Anne-Marie Michon, Darren J. Mitchell, Rab K. Prinjha, Inmaculada Rioja, Jon Seal, Simon Taylor, Pierre Thesmar, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK973 Is an Inhibitor of the Second Bromodomains (BD2s) of the Bromodomain and Extra-Terminal (BET) Family. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1581-1587. <a href="https://doi.org/10.1021/acsmedchemlett.0c00247" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00247</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00247%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DGSK973%252BIs%252Ban%252BInhibitor%252Bof%252Bthe%252BSecond%252BBromodomains%252B%252528BD2s%252529%252Bof%252Bthe%252BBromodomain%252Band%252BExtra-Terminal%252B%252528BET%252529%252BFamily%26aulast%3DPreston%26aufirst%3DAlex%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D06072020%26date%3D09072020%26date%3D06072020%26volume%3D11%26issue%3D8%26spage%3D1581%26epage%3D1587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabelle A.  Engelberg</span>, <span class="hlFld-ContribAuthor ">Caroline A.  Foley</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Improved methods for targeting epigenetic reader domains of acetylated and methylated lysine. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 132-144. <a href="https://doi.org/10.1016/j.cbpa.2021.03.002" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DImproved%252Bmethods%252Bfor%252Btargeting%252Bepigenetic%252Breader%252Bdomains%252Bof%252Bacetylated%252Band%252Bmethylated%252Blysine%26aulast%3DEngelberg%26aufirst%3DIsabelle%2BA.%26date%3D2021%26volume%3D63%26spage%3D132%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tingting  Liu</span>, <span class="hlFld-ContribAuthor ">Shubin  Song</span>, <span class="hlFld-ContribAuthor ">Xu  Wang</span>, <span class="hlFld-ContribAuthor ">Jifu  Hao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule inhibitors of breast cancer-related targets: Potential therapeutic agents for breast cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112954. <a href="https://doi.org/10.1016/j.ejmech.2020.112954" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112954%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-molecule%252Binhibitors%252Bof%252Bbreast%252Bcancer-related%252Btargets%25253A%252BPotential%252Btherapeutic%252Bagents%252Bfor%252Bbreast%252Bcancer%26aulast%3DLiu%26aufirst%3DTingting%26date%3D2021%26volume%3D210%26spage%3D112954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Massimo  Petretich</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>. </span><span class="cited-content_cbyCitation_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 184-193. <a href="https://doi.org/10.1016/j.cbpa.2020.02.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDomain-selective%252Btargeting%252Bof%252BBET%252Bproteins%252Bin%252Bcancer%252Band%252Bimmunological%252Bdiseases%26aulast%3DPetretich%26aufirst%3DMassimo%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Published domain selective BET bromodomain inhibitors and their literature potencies/affinities to the BD1 and BD2 bromodomain of BRD4 with the acetylated lysine mimetic colored blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) X-ray structure of <b>14a</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB0">6ZB0</a>) showing key interactions with Asn429 and a network of five water molecules, the benzyl group occupies the âWPFâ shelf. (b) <b>14a</b> (green) overlaid with alternate tool <b>15</b> (magenta) in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SWP">6SWP</a><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a>), highlighting the additional vector and interaction with Asn429 utilized by <b>15</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Key interactions of GSK620 (<b>20</b>) with BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>, cyan). (b) GSK620 shown with the protein surface representation of BRD2 BD2. (c) Key interactions of GSK620 (<b>20</b>) with BRD4 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>, white). (d) Overlay of GSK620 (<b>20</b>) in BRD2 BD2 (cyan) and BRD4 BD1 (white), providing a rationale for the domain selectivity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) A view of a molecule of <b>20</b> from the crystal structure, showing the numbering scheme employed. Anisotropic atomic displacement ellipsoids for the non-hydrogen atoms are shown at the 50% probability level. Hydrogen atoms are displayed with an arbitrarily small radius. (b) Molecules of <b>20</b> forming part of a hydrogen bonded column within the crystal structure. Hydrogen bonds are displayed as dashed lines with the donors and acceptors labeled. (c) Packing plot of the crystal structure of <b>20</b> viewed down the crystallographic <i>b</i>-axis showing the close proximity of the benzylic methylene and 6-pyridone hydrogens to an adjacent molecule (circled). Hydrogen bonds are displayed as dashed lines with the donors and acceptors labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray structure of <b>56</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB2">6ZB2</a>) showing: (a) the key interactions with His433, Asp434, and Trp370 on the WPF shelf. (b) The binding mode with a surface to highlight its optimal position on the WPF shelf.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0007.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Divergent Synthetic Routes for the Variance of R<sup>2</sup> and R<sup>3</sup><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditios: (i) NaH, DMF/THF (1:1) 0 Â°C, 30 min, then BnBr, 0 Â°C to rt, 2 h (89%); (ii) LiOH, THF/H<sub>2</sub>O/MeOH (1:1:1), rt, 2 h (78%); (iii) (COCl)<sub>2</sub>, DMF (0.5 mol %), DCM, rt, 2 h, then MeNH<sub>2</sub> (2 M in THF), Et<sub>3</sub>N, THF, 0 Â°C, 1.5 h (95%); (iv) Pd(OAc)<sub>2</sub> (10 mol %), dppb (10 mol %), Et<sub>3</sub>N, EtOH, DMSO, CO, 100 Â°C, 50 psi, 16 h (94%); (v) NaOH, MeOH/THF (1:1), rt, 2 h (80%); (vi) Pd(OAc)<sub>2</sub> (5 mol %), Xantphos (10 mol %), Et<sub>3</sub>N, toluene, 80 Â°C, 20 min, then 2,4,6-trichlorophenyl formate, 80 Â°C, 2.5 h (87%); (vii) R<sup>2</sup>NH<sub>2</sub>, HATU, DIPEA or Et<sub>3</sub>N, DMF or DCM, rt, 2 h; (viii) R<sup>2</sup>NH<sub>2</sub>, DMAP (0.2 equiv), Et<sub>3</sub>N, THF, 45 Â°C, 3 h; (ix) TFA, DCM, rt, 1 h; (x) MeNH<sub>2</sub> (2 M in THF), THF, reflux, 23 h (44%); (xi) K<sub>2</sub>CO<sub>3</sub>, R<sup>3</sup>CH<sub>2</sub>Br, MeOH or DMF, 65â90 Â°C; (xii) cPrNH<sub>2</sub>, Co<sub>2</sub>(CO)<sub>8</sub>, Pd(OAc)<sub>2</sub>, xantphos or cataXium A, DMAP, 1,4-dioxane, Î¼w 75â80 Â°C; (xiii) Cs<sub>2</sub>CO<sub>3</sub>, MeOH/THF (1:2), 70 Â°C, 1 h (41%).</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Alternative Routes for the Preparation of Pyridone Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) BnBr, Ag<sub>2</sub>CO<sub>3</sub>, CHCl<sub>3</sub>, reflux, 16 h (67%); (ii) LiOH, THF/MeOH (1:1), rt, 2 h (94%); (iii) (COCl)<sub>2</sub>, DMF (0.5 mol %), DCM, rt, 2 h, then MeNH<sub>2</sub> (2 M in THF), THF or 2-MeTHF, rt, 6 h (54â95%); (iv) Pd(OAc)<sub>2</sub> (5 mol %), Xantphos (10 mol %), Et<sub>3</sub>N, toluene, 80 Â°C, 20 min, then 2,4,6-trichlorophenyl formate, 80 Â°C, 2â18 h (57%); (v) R<sup>2</sup>NH<sub>2</sub>, Et<sub>3</sub>N, DMAP (5 mol %), THF, 45 Â°C, 45 min â16 h (85â86%); (vi) TFA, 80â90 Â°C, 1 h (83â99%); (vii) CMTP, toluene, Î¼w 120 Â°C, or DIAD, PPh<sub>3</sub>, toluene, rt, 3â24 h; (viii) K<sub>2</sub>CO<sub>3</sub>, THF or DMF, rt â50 Â°C; (ix) MeMgBr (3 M in Et<sub>2</sub>O), THF, â78 Â°C, 45 min (43%); (x) chiral separation (Chiralpak AD-H); (xi) DMP, DCM, 0 Â°C â rt, 48 h (quant); (xii) morpholine, Et<sub>3</sub>N, DCM, rt, 45 min, then STAB, rt 24 h (25%); (xiii) TFA, DCM, rt, 30â90 min (80%) or HCl, IPA, rt, 3 days (48%); (xiv) Cs<sub>2</sub>CO<sub>3</sub>, MeOH/THF (1:2), 70 Â°C, 1 h (68%); (xv) Pd(OAc)<sub>2</sub> (5 mol %), Xantphos (6 mol %), Et<sub>3</sub>N, MeOH, DMF, 60 Â°C, 21 h (60%); (xvi) NaI, MeCN, rt, 16 h (25%); (xvii) K<sub>2</sub>CO<sub>3</sub>, DMF, 90 Â°C, 1 h (67%); (xviii) LiOH, 1,4-dioxane/water (1:1), rt, 30 min (90%); xix. R<sup>2</sup>NH<sub>2</sub>, HATU, DIPEA, DMF, rt, 7 h (quant).</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0008.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of 6-Methylpyridone <b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) K<sub>2</sub>CO<sub>3</sub>, BnBr, DMF, 40 Â°C, 4 h (35%); (ii) MeNH<sub>2</sub> (2 M in THF), MeOH, 50 Â°C, 30 min (82%); (iii) LiOH, 1,4-dioxane/water (1:1), rt, 1 h 45 min (99%); (iv) cPrNH<sub>2</sub>, HATU, DIPEA, DMF, rt, 1 h (74%).</p></p></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0009.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Homochiral Preparation of Pyridone <b>31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (<i>R</i>)-1-phenylethanamine, DMF/THF (1:4), rt, 30 min, then EDC, DMAP (6 mol %), rt, 64 h (75%); (ii) NBS, 2-MeTHF, rt, 15 min; (iii) MeNH<sub>2</sub> (2 M in THF), MeOH, 50 Â°C, 90 min (78%, 2 steps); (iv) Pd(OAc)<sub>2</sub> (10 mol %), Xantphos (7 mol %), Et<sub>3</sub>N, MeOH, DMF, 60 Â°C, 4 h (60%); (v) NaOH, MeOH/H<sub>2</sub>O, rt, 16 h (90%); (vi) cPrNH<sub>2</sub>, HATU, DIPEA, DMF, rt, 1 h (89%).</p></p></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/medium/jm0c00796_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0006.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. BromoScan profile of <b>20</b> (GSK620).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00796/20200903/images/large/jm0c00796_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00796&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i192">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25567" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25567" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 89 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">246</span>â <span class="NLM_lpage">262</span>, <span class="refDoi">Â DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+bromodomain-containing+proteins+and+their+roles+in+homeostasis+and+cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0lhsIEQnOdLNJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520bromodomain-containing%2520proteins%2520and%2520their%2520roles%2520in%2520homeostasis%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">146</span>â <span class="NLM_lpage">153</span>, <span class="refDoi">Â DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&issue=3&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lhsIEQnOdLNJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26issue%3D3%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>, <span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0liRE6-WaFRoWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">697</span>, <span class="refDoi">Â DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697&issue=5&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0liRE6-WaFRoWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26issue%3D5%26spage%3D697%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>, <span class="refDoi">Â DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lhzTyh9GDBTQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1748</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-12-3066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F1078-0432.CCR-12-3066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23403638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1748&issue=7&author=Z.+Chengauthor=Y.+Gongauthor=Y.+Maauthor=K.+Luauthor=X.+Luauthor=L.+A.+Pierceauthor=R.+C.+Thompsonauthor=S.+Mullerauthor=S.+Knappauthor=J.+Wang&title=Inhibition+of+BET+Bromodomain+Targets+Genetically+Diverse+Glioblastoma&doi=10.1158%2F1078-0432.CCR-12-3066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span></div><div class="casAuthors">Cheng, Zhixiang; Gong, Yuanying; Ma, Yufang; Lu, Kaihua; Lu, Xiang; Pierce, Larry A.; Thompson, Reid C.; Muller, Susanne; Knapp, Stefan; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1748-1759</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Glioblastoma is refractory to conventional therapies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic readers that selectively bind to acetylated lysine residues on histone tails.  These proteins recently emerged as important therapeutic targets in NUT midline carcinoma and several types of hematopoietic cancers.  In this study, the therapeutic potential of a novel BET bromodomain inhibitor, JQ1, was assessed in a panel of genetically heterogeneous glioblastoma samples.  Exptl. Design: The antineoplastic effects of JQ1 were shown using ex vivo cultures derived from primary glioblastoma xenograft lines and surgical specimens of different genetic background.  The in vivo efficacy was assessed in orthotopic glioblastoma tumors.  Results: We showed that JQ1 induced marked G1 cell-cycle arrest and apoptosis, which was phenocopied by knockdown of individual BET family members.  JQ1 treatment resulted in significant changes in expression of genes that play important roles in glioblastoma such as c-Myc, p21CIP1/WAF1, hTERT, Bcl-2, and Bcl-xL.  Unlike the observations in some hematopoietic cancer cell lines, exogenous c-Myc did not significantly protect glioblastoma cells against JQ1.  In contrast, ectopically expressed Bcl-xL partially rescued cells from JQ1-induced apoptosis, and knockdown of p21CIP1/WAF1 attenuated JQ1-induced cell-cycle arrest.  Cells genetically engineered for Akt hyperactivation or p53/Rb inactivation did not compromise JQ1 efficacy, suggesting that these frequently mutated signaling pathways may not confer resistance to JQ1.  Furthermore, JQ1 significantly repressed growth of orthotopic glioblastoma tumors.  Conclusion: Our results suggest potentially broad therapeutic use of BET bromodomain inhibitors for treating genetically diverse glioblastoma tumors.  Clin Cancer Res; 19(7); 1748-59. Â©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OsFKqlwyDbVg90H21EOLACvtfcHk0lhzTyh9GDBTQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D&md5=1722189e5d950da233b785558f2c9f58</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3066%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPierce%26aufirst%3DL.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BC.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520BET%2520Bromodomain%2520Targets%2520Genetically%2520Diverse%2520Glioblastoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26issue%3D7%26spage%3D1748%26doi%3D10.1158%2F1078-0432.CCR-12-3066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">da
Motta, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purshouse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bastiani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morotti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigfield, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, A.</span></span> <span> </span><span class="NLM_article-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">122</span>, <span class="refDoi">Â DOI: 10.1038/onc.2016.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fonc.2016.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27292261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=122&author=L.+L.+da%0AMottaauthor=I.+Ledakiauthor=K.+Purshouseauthor=S.+Haiderauthor=M.+A.+De+Bastianiauthor=D.+Babanauthor=M.+Morottiauthor=G.+Steersauthor=S.+Wigfieldauthor=E.+Bridgesauthor=J.+L.+Liauthor=S.+Knappauthor=D.+Ebnerauthor=F.+Klamtauthor=A.+L.+Harrisauthor=A.+McIntyre&title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&doi=10.1038%2Fonc.2016.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span></div><div class="casAuthors">da Motta, L. L.; Ledaki, I.; Purshouse, K.; Haider, S.; De Bastiani, M. A.; Baban, D.; Morotti, M.; Steers, G.; Wigfield, S.; Bridges, E.; Li, J.-L.; Knapp, S.; Ebner, D.; Klamt, F.; Harris, A. L.; McIntyre, A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-132</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of bromodomain and extra-terminal inhibitors (BETi) has enabled translational epigenetic studies in cancer.  BET proteins regulate transcription by selectively recognizing acetylated lysine residues on chromatin.  BETi compete with this process leading to both downregulation and upregulation of gene expression.  Hypoxia enables progression of triple neg. breast cancer (TNBC), the most aggressive form of breast cancer, partly by driving metabolic adaptation, angiogenesis and metastasis through upregulation of hypoxia-regulated genes (for example, carbonic anhydrase 9 (CA9) and vascular endothelial growth factor A (VEGF-A).  Responses to hypoxia can be mediated epigenetically, thus we investigated whether BETi JQ1 could impair the TNBC response induced by hypoxia and exert anti-tumor effects.  JQ1 significantly modulated 44% of hypoxia-induced genes, of which two-thirds were downregulated including CA9 and VEGF-A.  JQ1 prevented HIF binding to the hypoxia response element in CA9 promoter, but did not alter HIF expression or activity, suggesting some HIF targets are BET-dependent.  JQ1 reduced TNBC growth in vitro and in vivo and inhibited xenograft vascularization.  These findings identify that BETi dually targets angiogenesis and the hypoxic response, an effective combination at reducing tumor growth in preclin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79-F0o2tp6bVg90H21EOLACvtfcHk0ljuc-l_QU5evg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D&md5=9f1592548db8e6306703bdab7e1e1c90</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.184%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BMotta%26aufirst%3DL.%2BL.%26aulast%3DLedaki%26aufirst%3DI.%26aulast%3DPurshouse%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DS.%26aulast%3DDe%2BBastiani%26aufirst%3DM.%2BA.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DMorotti%26aufirst%3DM.%26aulast%3DSteers%26aufirst%3DG.%26aulast%3DWigfield%26aufirst%3DS.%26aulast%3DBridges%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DMcIntyre%26aufirst%3DA.%26atitle%3DThe%2520BET%2520inhibitor%2520JQ1%2520selectively%2520impairs%2520tumour%2520response%2520to%2520hypoxia%2520and%2520downregulates%2520CA9%2520and%2520angiogenesis%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D122%26doi%3D10.1038%2Fonc.2016.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Connor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: A Novel Bivalent BET Bromodomain Inhibitor Highly Active against Hematologic Malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2563</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F1535-7163.MCT-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563&issue=11&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%27Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+A+Novel+Bivalent+BET+Bromodomain+Inhibitor+Highly+Active+against+Hematologic+Malignancies&doi=10.1158%2F1535-7163.MCT-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0ljuc-l_QU5evg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%2527Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520A%2520Novel%2520Bivalent%2520BET%2520Bromodomain%2520Inhibitor%2520Highly%2520Active%2520against%2520Hematologic%2520Malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26issue%3D11%26spage%3D2563%26doi%3D10.1158%2F1535-7163.MCT-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1628</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628&issue=7&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+Bromodomain+Inhibitor+OTX015+Affects+Pathogenetic+Pathways+in+Preclinical+B-cell+Tumor+Models+and+Synergizes+with+Targeted+Drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. Â©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0ljpZu1V9Ed2Nw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520OTX015%2520Affects%2520Pathogenetic%2520Pathways%2520in%2520Preclinical%2520B-cell%2520Tumor%2520Models%2520and%2520Synergizes%2520with%2520Targeted%2520Drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26issue%3D7%26spage%3D1628%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Sullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3336</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F0008-5472.CAN-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336&issue=11&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%27Sullivanauthor=S.+M%C3%BCllerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+Highly+Selective+Protein+Interaction+Inhibitor%2C+Targeting+BET+Bromodomains&doi=10.1158%2F0008-5472.CAN-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0ljpZu1V9Ed2Nw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%2527Sullivan%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520Highly%2520Selective%2520Protein%2520Interaction%2520Inhibitor%252C%2520Targeting%2520BET%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26issue%3D11%26spage%3D3336%26doi%3D10.1158%2F0008-5472.CAN-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segura, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanals-Cirera, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaziel-Sovran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanniford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GonzÃ¡lez-Gomez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubierre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvishian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, E.</span></span> <span> </span><span class="NLM_article-title">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">6264</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-0122-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1158%2F0008-5472.CAN-13-0122-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23950209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6264&issue=20&author=M.+F.+Seguraauthor=B.+Fontanals-Cireraauthor=A.+Gaziel-Sovranauthor=M.+V.+Guijarroauthor=D.+Hannifordauthor=G.+Zhangauthor=P.+Gonz%C3%A1lez-Gomezauthor=M.+Moranteauthor=L.+Jubierreauthor=W.+Zhangauthor=F.+Darvishianauthor=M.+Ohlmeyerauthor=I.+Osmanauthor=M.-M.+Zhouauthor=E.+Hernando&title=BRD4+Sustains+Melanoma+Proliferation+and+Represents+a+New+Target+for+Epigenetic+Therapy&doi=10.1158%2F0008-5472.CAN-13-0122-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span></div><div class="casAuthors">Segura, Miguel F.; Fontanals-Cirera, Barbara; Gaziel-Sovran, Avital; Guijarro, Maria V.; Hanniford, Doug; Zhang, Guangtao; Gonzalez-Gomez, Pilar; Morante, Marta; Jubierre, Luz; Zhang, Weijia; Darvishian, Farbod; Ohlmeyer, Michael; Osman, Iman; Zhou, Ming-Ming; Hernando, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6264-6276</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Metastatic melanoma remains a mostly incurable disease.  Although newly approved targeted therapies are efficacious in a subset of patients, resistance and relapse rapidly ensue.  Alternative therapeutic strategies to manipulate epigenetic regulators and disrupt the transcriptional program that maintains tumor cell identity are emerging.  Bromodomain and extraterminal domain (BET) proteins are epigenome readers known to exert key roles at the interface between chromatin remodeling and transcriptional regulation.  Here, we report that BRD4, a BET family member, is significantly upregulated in primary and metastatic melanoma tissues compared with melanocytes and nevi.  Treatment with BET inhibitors impaired melanoma cell proliferation in vitro and tumor growth and metastatic behavior in vivo, effects that were mostly recapitulated by individual silencing of BRD4.  RNA sequencing of BET inhibitor-treated cells followed by Gene Ontol. anal. showed a striking impact on transcriptional programs controlling cell growth, proliferation, cell-cycle regulation, and differentiation.  In particular, we found that, rapidly after BET displacement, key cell-cycle genes (SKP2, ERK1, and c-MYC) were downregulated concomitantly with the accumulation of cyclin-dependent kinase (CDK) inhibitors (p21 and p27), followed by cell-cycle arrest.  Importantly, BET inhibitor efficacy was not influenced by BRAF or NRAS mutational status, opening the possibility of using these small-mol. compds. to treat patients for whom no effective targeted therapy exists.  Collectively, our study reveals a crit. role for BRD4 in melanoma tumor maintenance and renders it a legitimate and novel target for epigenetic therapy directed against the core transcriptional program of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80TZH2ueQFbVg90H21EOLACvtfcHk0ljpZu1V9Ed2Nw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N&md5=a4a77f188de01c4c84f7293a32478fc7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0122-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0122-T%26sid%3Dliteratum%253Aachs%26aulast%3DSegura%26aufirst%3DM.%2BF.%26aulast%3DFontanals-Cirera%26aufirst%3DB.%26aulast%3DGaziel-Sovran%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%2BV.%26aulast%3DHanniford%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Gomez%26aufirst%3DP.%26aulast%3DMorante%26aufirst%3DM.%26aulast%3DJubierre%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDarvishian%26aufirst%3DF.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DHernando%26aufirst%3DE.%26atitle%3DBRD4%2520Sustains%2520Melanoma%2520Proliferation%2520and%2520Represents%2520a%2520New%2520Target%2520for%2520Epigenetic%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26issue%3D20%26spage%3D6264%26doi%3D10.1158%2F0008-5472.CAN-13-0122-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: breaking through the immune barrier</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">835</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2016.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrd.2016.185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27765940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=835&author=D.+F.+Toughauthor=P.+P.+Takauthor=A.+Tarakhovskyauthor=R.+K.+Prinjha&title=Epigenetic+drug+discovery%3A+breaking+through+the+immune+barrier&doi=10.1038%2Fnrd.2016.185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: breaking through the immune barrier</span></div><div class="casAuthors">Tough, David F.; Tak, Paul P.; Tarakhovsky, Alexander; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">835-853</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immune-mediated diseases are clin. heterogeneous but they share genetic and pathogenic mechanisms.  These diseases may develop from the interplay of genetic factors and environmental or lifestyle factors.  Exposure to such factors, including infectious agents, is assocd. with coordinated changes in gene transcription owing to epigenetic alterations.  A growing understanding of how epigenetic mechanisms control gene expression patterns and cell function has been aided by the development of small-mol. inhibitors that target these processes.  These chem. tools have helped to reveal the importance of epigenetics in guiding cell fate decisions during immune responses and have also highlighted the potential for targeting epigenetic mechanisms for the treatment of inflammation and immune-mediated diseases.  In this Review, we discuss the most advanced areas of epigenetic drug development for autoimmune and inflammatory diseases and summarize the promising preclin. data in this exciting and evolving field.  These agents will inevitably begin to move into clin. trials for use in patients with immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplOyEVMzBi37Vg90H21EOLACvtfcHk0lixcmAm_jJvXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE&md5=f939bcbbe3d74ce24718eef706aae3d1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.185%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520breaking%2520through%2520the%2520immune%2520barrier%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D835%26doi%3D10.1038%2Fnrd.2016.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">573</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27925476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=573-584&issue=4&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span></div><div class="casAuthors">Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Genome-wide assocn. studies have identified thousands of single nucleotide polymorphisms in the human genome that are statistically assocd. with particular disease traits.  In this perspective, we review emerging data suggesting that most single nucleotide polymorphisms assocd. with immune-mediated diseases are found in regulatory regions of the DNA - parts of the genome that control expression of the protein encoding genes - rather than causing mutations in proteins.  We discuss how the emerging understanding of particular gene regulatory regions, gene enhancers and the epigenetic mechanisms by which they are regulated is opening up new opportunities for the treatment of immune-mediated diseases, focusing particularly on the BET family of epigenetic reader proteins as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFU7Sh3r75ZbVg90H21EOLACvtfcHk0lixcmAm_jJvXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D&md5=8a2d44dd000c465b904d4ef2e6f37b77</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26issue%3D4%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses T(H)17-mediated pathology</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>210</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2181</span>â <span class="NLM_lpage">2190</span>, <span class="refDoi">Â DOI: 10.1084/jem.20130376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1084%2Fjem.20130376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24101376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2013&pages=2181-2190&issue=11&author=D.+A.+Meleauthor=A.+Salmeronauthor=S.+Ghoshauthor=H.-R.+Huangauthor=B.+M.+Bryantauthor=J.+M.+Lora&title=BET+bromodomain+inhibition+suppresses+T%28H%2917-mediated+pathology&doi=10.1084%2Fjem.20130376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses TH17-mediated pathology</span></div><div class="casAuthors">Mele, Deanna A.; Salmeron, Andres; Ghosh, Srimoyee; Huang, Hon-Ren; Bryant, Barbara M.; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin (IL) 17-producing T helper (TH17) cells have been selected through evolution for their ability to control fungal and bacterial infections.  It is also firmly established that their aberrant generation and activation results in autoimmune conditions.  Using a characterized potent and selective small mol. inhibitor, we show that the bromodomain and extra-terminal domain (BET) family of chromatin adaptors plays fundamental and selective roles in human and murine TH17 differentiation from naive CD4+ T cells, as well as in the activation of previously differentiated TH17 cells.  We provide evidence that BET controls TH17 differentiation in a bromodomain-dependent manner through a mechanism that includes the direct regulation of multiple effector TH17-assocd. cytokines, including IL17, IL21, and GMCSF.  We also demonstrate that BET family members Brd2 and Brd4 assoc. with the Il17 locus in TH17 cells, and that this assocn. requires bromodomains.  We recapitulate the crit. role of BET bromodomains in TH17 differentiation in vivo and show that therapeutic dosing of the BET inhibitor is efficacious in mouse models of autoimmunity.  Our results identify the BET family of proteins as a fundamental link between chromatin signaling and TH17 biol., and support the notion of BET inhibition as a point of therapeutic intervention in autoimmune conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBqVtggp2Fa7Vg90H21EOLACvtfcHk0lixcmAm_jJvXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF&md5=e6822f456cf42cdead9779afa92c05e0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1084%2Fjem.20130376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20130376%26sid%3Dliteratum%253Aachs%26aulast%3DMele%26aufirst%3DD.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520T%2528H%252917-mediated%2520pathology%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2013%26volume%3D210%26issue%3D11%26spage%3D2181%26epage%3D2190%26doi%3D10.1084%2Fjem.20130376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">14532</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532&issue=36&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0lgiI3XXUEgXkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D36%26spage%3D14532%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarilina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivashkiv, L. B.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">287</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1002/eji.201444862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1002%2Feji.201444862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25345375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=287-297&issue=1&author=C.+H.+Chanauthor=C.+Fangauthor=Y.+Qiaoauthor=A.+Yarilinaauthor=R.+K.+Prinjhaauthor=L.+B.+Ivashkiv&title=BET+bromodomain+inhibition+suppresses+transcriptional+responses+to+cytokine-Jak-STAT+signaling+in+a+gene-specific+manner+in+human+monocytes&doi=10.1002%2Feji.201444862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span></div><div class="casAuthors">Chan, Chun Hin; Fang, Celestia; Qiao, Yu; Yarilina, Anna; Prinjha, Rab K.; Ivashkiv, Lionel B.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">EJIMAF</span>;
        ISSN:<span class="NLM_cas:issn">0014-2980</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Disruption of the interaction of bromo and extraterminal (BET) proteins with acetylated histones using small mol. inhibitors suppresses Myc-driven cancers and TLR-induced inflammation in mouse models.  The predominant mechanism of BET inhibitor action is to suppress BET-mediated recruitment of pos. transcription elongation factor b and, thus, transcription elongation.  We investigated the effects of BET inhibitor I-BET151 on transcriptional responses to TLR4 and TNF in primary human monocytes and also on responses to cytokines IFN-Î², IFN-Î³, IL-4, and IL-10, which activate the JAK-STAT signaling pathway and are important for monocyte polarization and inflammatory diseases.  I-BET151 suppressed TLR4- and TNF-induced IFN responses by diminishing both autocrine IFN-Î² expression and transcriptional responses to IFN-Î².  I-BET151 inhibited cytokine-induced transcription of STAT targets in a gene-specific manner without affecting STAT activation or recruitment.  This inhibition was independent of Myc or other upstream activators.  IFN-stimulated gene transcription is regulated primarily at the level of transcription initiation.  Accordingly, we found that I-BET151 suppressed the recruitment of transcriptional machinery to the CXCL10 promoter and an upstream enhancer.  Our findings suggest that BET inhibition reduces inflammation partially through suppressing cytokine activity and expands the understanding of the inhibitory and potentially selective immunosuppressive effects of inhibiting BET proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbE0f71lXpp7Vg90H21EOLACvtfcHk0lgiI3XXUEgXkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP&md5=efa081bc6c79db9233ae1a99ebf49945</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Feji.201444862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201444862%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DYarilina%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DIvashkiv%26aufirst%3DL.%2BB.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520transcriptional%2520responses%2520to%2520cytokine-Jak-STAT%2520signaling%2520in%2520a%2520gene-specific%2520manner%2520in%2520human%2520monocytes%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2015%26volume%3D45%26issue%3D1%26spage%3D287%26epage%3D297%26doi%3D10.1002%2Feji.201444862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>â <span class="NLM_lpage">76</span>, <span class="refDoi">Â DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0lhmtI_TyKt_Vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3670</span>, <span class="refDoi">Â DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670&issue=7&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+Protein+Function+Is+Required+for+Inflammation%3A+Brd2+Genetic+Disruption+and+BET+Inhibitor+JQ1+Impair+Mouse+Macrophage+Inflammatory+Responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-Î± while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0lhmtI_TyKt_Vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520Protein%2520Function%2520Is%2520Required%2520for%2520Inflammation%253A%2520Brd2%2520Genetic%2520Disruption%2520and%2520BET%2520Inhibitor%2520JQ1%2520Impair%2520Mouse%2520Macrophage%2520Inflammatory%2520Responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26issue%3D7%26spage%3D3670%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">422</span>, <span class="refDoi">Â DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422&issue=2&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.-L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- Î±, interleukin (IL)-1Î² and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- Î± and IL-1Î² .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0lhmtI_TyKt_Vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26issue%3D2%26spage%3D422%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0liSNJE2IKnx-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitor JQ1 Blocks Inflammation and Bone Destruction</span>. <i>J. Dent. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">657</span>â <span class="NLM_lpage">662</span>, <span class="refDoi">Â DOI: 10.1177/0022034514534261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1177%2F0022034514534261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24799421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2014&pages=657-662&issue=7&author=S.+Mengauthor=L.+Zhangauthor=Y.+Tangauthor=Q.+Tuauthor=L.+Zhengauthor=L.+Yuauthor=D.+Murrayauthor=J.+Chengauthor=S.+H.+Kimauthor=X.+Zhouauthor=J.+Chen&title=BET+Inhibitor+JQ1+Blocks+Inflammation+and+Bone+Destruction&doi=10.1177%2F0022034514534261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor JQ1 blocks inflammation and bone destruction</span></div><div class="casAuthors">Meng, S.; Zhang, L.; Tang, Y.; Tu, Q.; Zheng, L.; Yu, L.; Murray, D.; Cheng, J.; Kim, S. H.; Zhou, X.; Chen, J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dental Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">657-662</span>CODEN:
                <span class="NLM_cas:coden">JDREAF</span>;
        ISSN:<span class="NLM_cas:issn">0022-0345</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">BET proteins are a group of epigenetic regulators controlling transcription through reading acetylated histone tails and recruiting transcription complexes.  They are considered as potential therapeutic targets in many distinct diseases.  A novel synthetic bromodomain and extraterminal domain (BET) inhibitor, JQ1, was proved to suppress oncogene transcription and inflammatory responses.  The present study was aimed to investigate the effects of JQ1 on inflammatory response and bone destruction in exptl. periodontitis.  We found that JQ1 significantly suppressed lipopolysaccharide (LPS)-stimulated inflammatory cytokine transcription, including interleukin (IL)-1Î², IL-6, and tumor necrosis factor alpha (TNF-Î±), as well as receptor activator of nuclear factor kappa- B ligand (RANKL)-induced osteoclast markers, such as c-Fos, nuclear factor of activated T-cells, cytoplasmic, calcineurin- dependent 1 (NFATc1), tartrate-resistant acid phosphatase (TRAP) and cathepsin K in vitro.  JQ1 also inhibited toll-like receptors 2/4 (TLR2/4) expression and nuclear factor kappa-light-chain-enhancer of activated B cells (NF-ÎºB) phosphorylation and nuclear translocation.  Chromatin immunopptn. and quant. polymerase chain reaction (ChIP-qPCR) revealed that JQ1 neutralized BRD4 enrichment at several gene promoter regions, including NF-ÎºB, TNF-Î±, c-Fos, and NFATc1.  In a murine periodontitis model, systemic administration of JQ1 significantly inhibited inflammatory cytokine expression in diseased gingival tissues.  Alveolar bone loss was alleviated in JQ1-treated mice because of reduced osteoclasts in periodontal tissues.  These unprecedented results suggest the BET inhibitor JQ1 as a prospective new approach for treating periodontitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtMKKZI5MhYLVg90H21EOLACvtfcHk0liSNJE2IKnx-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI&md5=7e65e561c2141b78954deb6b779ed9b5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F0022034514534261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0022034514534261%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTu%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DBET%2520Inhibitor%2520JQ1%2520Blocks%2520Inflammation%2520and%2520Bone%2520Destruction%26jtitle%3DJ.%2520Dent.%2520Res.%26date%3D2014%26volume%3D93%26issue%3D7%26spage%3D657%26epage%3D662%26doi%3D10.1177%2F0022034514534261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>â <span class="NLM_lpage">257</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORÎ±/Î³ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0ljpHLll4dWWuQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.-g.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.-c.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span>. <i>Immunol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>166</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1016/j.imlet.2015.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.imlet.2015.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=26093279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2015&pages=103-108&issue=2&author=Q.-g.+Zhangauthor=J.+Qianauthor=Y.-c.+Zhu&title=Targeting+bromodomain-containing+protein+4+%28BRD4%29+benefits+rheumatoid+arthritis&doi=10.1016%2Fj.imlet.2015.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span></div><div class="casAuthors">Zhang, Qing-gang; Qian, Jing; Zhu, Yu-chang</div><div class="citationInfo"><span class="NLM_cas:title">Immunology Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">IMLED6</span>;
        ISSN:<span class="NLM_cas:issn">0165-2478</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We aimed to explore the effects of bromodomain-contg. protein 4 (BRD4) inhibition on tumor necrosis factor (TNF)-Î±-stimulated human rheumatoid arthritis fibroblast-like synoviocytes (RA-FLS) behavior and the therapeutic implications using BRD4 inhibitor JQ1 were explored in vivo.  The levels of interleukin (IL)-1Î², IL-6, IL-17 and IL-18 in cultural supernatants from TNFÎ±-stimulated RA-FLS were measured by ELISA.  RA-FLS migration and invasion in vitro were investigated using wound healing and Matrigel assay.  Expression of signaling pathway proteins was measured by Western blot.  The in vivo effects of BRD4 inhibitor JQ1 were elucidated using collagen-induced arthritis (CIA) mice.  We found BRD4 silencing reduced the secretion of IL-1Î², IL-6, IL-17 and IL-18 from TNFÎ±-stimulated human RA-FLS.  Downregulation of BRD4 inhibited FBS-induced migration and invasion of human RA-FLS.  BRD4 silencing decreased the phosphorylation of c-Jun and activation of NFÎºB in TNFÎ±-stimulated RA-FLS.  In vivo, BRD4 inhibitor JQ1 reduced the inflammatory response, autoantibody prodn. and joint damage of CIA model.  Our data suggest for the first time that BRD4 inhibition has anti-inflammatory property in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-RTAy64CzFbVg90H21EOLACvtfcHk0ljpHLll4dWWuQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM&md5=395fce2cd419dd86aebd9fd93ce80c0f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.imlet.2015.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.imlet.2015.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.-g.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DY.-c.%26atitle%3DTargeting%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520benefits%2520rheumatoid%2520arthritis%26jtitle%3DImmunol.%2520Lett.%26date%3D2015%26volume%3D166%26issue%3D2%26spage%3D103%26epage%3D108%26doi%3D10.1016%2Fj.imlet.2015.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">4533</span>â <span class="NLM_lpage">4558</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&issue=11&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+Targeting+Bromodomain-Containing+Protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lhl80nFhvjp-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520Targeting%2520Bromodomain-Containing%2520Protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D11%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1776</span>â <span class="NLM_lpage">1787</span>, <span class="refDoi">Â DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&issue=8&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2ebnWiYxJQwmerntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2017%26volume%3D28%26issue%3D8%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alqahtani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choucair, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashraf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammouda, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alloghbi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senzer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemunaitis, J.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and Extra-Terminal Motif Inhibitors: A Review of Preclinical and Clinical Advances in Cancer Therapy</span>. <i>Future Science OA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">FSO372</span>, <span class="refDoi">Â DOI: 10.4155/fsoa-2018-0115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.4155%2Ffsoa-2018-0115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=30906568" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=FSO372&issue=3&author=A.+Alqahtaniauthor=K.+Choucairauthor=M.+Ashrafauthor=D.+M.+Hammoudaauthor=A.+Alloghbiauthor=T.+Khanauthor=N.+Senzerauthor=J.+Nemunaitis&title=Bromodomain+and+Extra-Terminal+Motif+Inhibitors%3A+A+Review+of+Preclinical+and+Clinical+Advances+in+Cancer+Therapy&doi=10.4155%2Ffsoa-2018-0115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.4155%2Ffsoa-2018-0115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffsoa-2018-0115%26sid%3Dliteratum%253Aachs%26aulast%3DAlqahtani%26aufirst%3DA.%26aulast%3DChoucair%26aufirst%3DK.%26aulast%3DAshraf%26aufirst%3DM.%26aulast%3DHammouda%26aufirst%3DD.%2BM.%26aulast%3DAlloghbi%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DSenzer%26aufirst%3DN.%26aulast%3DNemunaitis%26aufirst%3DJ.%26atitle%3DBromodomain%2520and%2520Extra-Terminal%2520Motif%2520Inhibitors%253A%2520A%2520Review%2520of%2520Preclinical%2520and%2520Clinical%2520Advances%2520in%2520Cancer%2520Therapy%26jtitle%3DFuture%2520Science%2520OA%26date%3D2019%26volume%3D5%26issue%3D3%26spage%3DFSO372%26doi%3D10.4155%2Ffsoa-2018-0115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">Clinical Trials for BET Inhibitors Run Ahead of the Science</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>â <span class="NLM_lpage">50</span>, <span class="refDoi">Â DOI: 10.1016/j.ddtec.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.ddtec.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27769357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=45-50&author=G.+Andrieuauthor=A.+C.+Belkinaauthor=G.+V.+Denis&title=Clinical+Trials+for+BET+Inhibitors+Run+Ahead+of+the+Science&doi=10.1016%2Fj.ddtec.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical trials for BET inhibitors run ahead of the science</span></div><div class="casAuthors">Andrieu Guillaume; Belkina Anna C; Denis Gerald V</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several cancer clinical trials for small molecule inhibitors of BET bromodomain proteins have been initiated.  There is enthusiasm for the anti-proliferative effect of inhibiting BRD4, one of the targets of these inhibitors, which is thought to cooperate with MYC, a long-desired target for cancer therapeutics.  However, no current inhibitor is selective for BRD4 among the three somatic BET proteins, which include BRD2 and BRD3; their respective functions are partially overlapping and none are functionally redundant with BRD4.  Each BET protein controls distinct transcriptional pathways that are important for functions beyond cancer cell proliferation, including insulin production, cytokine gene transcription, T cell differentiation, adipogenesis and most seriously, active repression of dangerous latent viruses like HIV.  BET inhibitors have been shown to reactivate HIV in human cells.  Failure to appreciate that at concentrations used, no available BET inhibitor is member-selective, or to develop a sound biological basis to understand the diverse functions of BET proteins before undertaking for these clinical trials is reckless and likely to lead to adverse events.  More mechanistic information from new basic science studies should enable proper focus on the most relevant cancers and define the expected side effect profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-Lt7lRqEFUJVfW6udTcc2eaU4l8lupEeCrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D&md5=5fcc9e96fa590221fb07c5e98ae91d59</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DAndrieu%26aufirst%3DG.%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DClinical%2520Trials%2520for%2520BET%2520Inhibitors%2520Run%2520Ahead%2520of%2520the%2520Science%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D45%26epage%3D50%26doi%3D10.1016%2Fj.ddtec.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postel-Vinay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbschleb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewerton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, M.</span></span> <span> </span><span class="NLM_article-title">First-in-Human Phase I Study of the Bromodomain and Extraterminal Motif Inhibitor BAY 1238097: Emerging Pharmacokinetic/Pharmacodynamic Relationship and Early Termination due to Unexpected Toxicity</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">110</span>, <span class="refDoi">Â DOI: 10.1016/j.ejca.2018.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.ejca.2018.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=30711772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2019&pages=103-110&author=S.+Postel-Vinayauthor=K.+Herbschlebauthor=C.+Massardauthor=V.+Woodcockauthor=J.-C.+Soriaauthor=A.+O.+Walterauthor=F.+Ewertonauthor=M.+Poelmanauthor=N.+Bensonauthor=M.+Ockerauthor=G.+Wilkinsonauthor=M.+Middleton&title=First-in-Human+Phase+I+Study+of+the+Bromodomain+and+Extraterminal+Motif+Inhibitor+BAY+1238097%3A+Emerging+Pharmacokinetic%2FPharmacodynamic+Relationship+and+Early+Termination+due+to+Unexpected+Toxicity&doi=10.1016%2Fj.ejca.2018.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human phase I study of the bromodomain and extraterminal motif inhibitor BAY 1238097: emerging pharmacokinetic/pharmacodynamic relationship and early termination due to unexpected toxicity</span></div><div class="casAuthors">Postel-Vinay, Sophie; Herbschleb, Karin; Massard, Christophe; Woodcock, Victoria; Soria, Jean-Charles; Walter, Annette O.; Ewerton, Flavio; Poelman, Martine; Benson, Neil; Ocker, Matthias; Wilkinson, Gary; Middleton, Mark</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-110</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal motif (BET) protein inhibition is a promising cancer treatment strategy, notably for targeting MYC- or BRD4-driven diseases.  A first-in-human study investigated the safety, pharmacokinetics, max. tolerated dose and recommended phase II dose of the BET inhibitor BAY 1238097 in patients with advanced malignancies.In this phase I, open-label, non-randomised, multicentre study, patients with cytol. or histol. confirmed advanced refractory malignancies received oral BAY 1238097 twice weekly in 21-day cycles using an adaptive dose-escalation design at a starting dose of 10 mg/wk.  Model-based dose-response anal. was performed to guide dose escalation.  Safety, pharmacokinetics, pharmacodynamics and tumor response were evaluated.Eight patients were enrolled at three dose levels (10 mg/wk, n = 3; 40 mg/wk, n = 3; 80 mg/wk, n = 2).  Both patients receiving 80 mg/wk had dose-limiting toxicities (DLTs) (grade 3 vomiting, grade 3 headache and grade 2/3 back pain).  The most common adverse events were nausea, vomiting, headache, back pain and fatigue.  Pharmacokinetic anal. indicated a linear dose response with increasing dose.  Two patients displayed prolonged stable disease; no responses were obsd.  Biomarker evaluation of MYC and HEXIM1 expression demonstrated an emerging pharmacokinetic/pharmacodynamic relationship, with a trend towards decreased MYC and increased HEXIM1 expression in response to treatment.The study was prematurely terminated because of the occurrence of DLTs at a dose below targeted drug exposure.  Pharmacokinetic modeling indicated that an alternate dosing schedule whereby DLTs could be avoided while reaching efficacious exposure was not feasible.  Registration no.: NCT02369029.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9snvNdTKTtrVg90H21EOLACvtfcHk0ljmojRwSDP5dw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D&md5=c5969b7827810fd1146338b84c8f8621</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2018.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2018.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DPostel-Vinay%26aufirst%3DS.%26aulast%3DHerbschleb%26aufirst%3DK.%26aulast%3DMassard%26aufirst%3DC.%26aulast%3DWoodcock%26aufirst%3DV.%26aulast%3DSoria%26aufirst%3DJ.-C.%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DEwerton%26aufirst%3DF.%26aulast%3DPoelman%26aufirst%3DM.%26aulast%3DBenson%26aufirst%3DN.%26aulast%3DOcker%26aufirst%3DM.%26aulast%3DWilkinson%26aufirst%3DG.%26aulast%3DMiddleton%26aufirst%3DM.%26atitle%3DFirst-in-Human%2520Phase%2520I%2520Study%2520of%2520the%2520Bromodomain%2520and%2520Extraterminal%2520Motif%2520Inhibitor%2520BAY%25201238097%253A%2520Emerging%2520Pharmacokinetic%252FPharmacodynamic%2520Relationship%2520and%2520Early%2520Termination%2520due%2520to%2520Unexpected%2520Toxicity%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2019%26volume%3D109%26spage%3D103%26epage%3D110%26doi%3D10.1016%2Fj.ejca.2018.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odenike, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldoss, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzieri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prebet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freise, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, B. A.</span></span> <span> </span><span class="NLM_article-title">Results from the First-in-Human Study of Mivebresib (ABBV-075), a Pan-Inhibitor of Bromodomain and Extra Terminal Proteins, in Patients with Relapsed/Refractory Acute Myeloid Leukemia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">15suppl</span>),  <span class="NLM_fpage">7030</span>, <span class="refDoi">Â DOI: 10.1200/JCO.2019.37.15_suppl.7030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1200%2FJCO.2019.37.15_suppl.7030" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=7030&issue=15suppl&author=O.+Odenikeauthor=J.+E.+Wolffauthor=G.+Borthakurauthor=I.+T.+Aldossauthor=D.+Rizzieriauthor=T.+Prebetauthor=B.+Huauthor=M.+Dinhauthor=X.+Chenauthor=D.+Modiauthor=K.+J.+Freiseauthor=B.+A.+Jonas&title=Results+from+the+First-in-Human+Study+of+Mivebresib+%28ABBV-075%29%2C+a+Pan-Inhibitor+of+Bromodomain+and+Extra+Terminal+Proteins%2C+in+Patients+with+Relapsed%2FRefractory+Acute+Myeloid+Leukemia&doi=10.1200%2FJCO.2019.37.15_suppl.7030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.7030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.7030%26sid%3Dliteratum%253Aachs%26aulast%3DOdenike%26aufirst%3DO.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26aulast%3DBorthakur%26aufirst%3DG.%26aulast%3DAldoss%26aufirst%3DI.%2BT.%26aulast%3DRizzieri%26aufirst%3DD.%26aulast%3DPrebet%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DModi%26aufirst%3DD.%26aulast%3DFreise%26aufirst%3DK.%2BJ.%26aulast%3DJonas%26aufirst%3DB.%2BA.%26atitle%3DResults%2520from%2520the%2520First-in-Human%2520Study%2520of%2520Mivebresib%2520%2528ABBV-075%2529%252C%2520a%2520Pan-Inhibitor%2520of%2520Bromodomain%2520and%2520Extra%2520Terminal%2520Proteins%252C%2520in%2520Patients%2520with%2520Relapsed%252FRefractory%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26issue%3D15suppl%26spage%3D7030%26doi%3D10.1200%2FJCO.2019.37.15_suppl.7030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amans, D.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hirst, D. J.</span>; <span class="NLM_string-name">Law, R. P.</span>; <span class="NLM_string-name">Lindon, M.</span>; <span class="NLM_string-name">Preston, A.</span>; <span class="NLM_string-name">Seal, J. T.</span>; <span class="NLM_string-name">Wellaway, C. R.</span></span> <span> </span><span class="NLM_article-title">Preparation of acylaminotetrahydroquinoline derivatives for use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2014140076A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+Amans&author=S.+J.+Atkinson&author=L.+A.+Harrison&author=D.+J.+Hirst&author=R.+P.+Law&author=M.+Lindon&author=A.+Preston&author=J.+T.+Seal&author=C.+R.+Wellaway&title=Preparation+of+acylaminotetrahydroquinoline+derivatives+for+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAmans%26aufirst%3DD.%26atitle%3DPreparation%2520of%2520acylaminotetrahydroquinoline%2520derivatives%2520for%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hayhow, T. G. C.</span>; <span class="NLM_string-name">House, D.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Preston, A. G.</span>; <span class="NLM_string-name">Seal, J. T.</span>; <span class="NLM_string-name">Wall, I. D.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Woolven, J. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of pyridinone-dicarboxamide for use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2017050714A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+J.+Atkinson&author=E.+H.+Demont&author=L.+A.+Harrison&author=T.+G.+C.+Hayhow&author=D.+House&author=M.+J.+Lindon&author=A.+G.+Preston&author=J.+T.+Seal&author=I.+D.+Wall&author=R.+J.+Watson&author=J.+M.+Woolven&title=Preparation+of+pyridinone-dicarboxamide+for+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26atitle%3DPreparation%2520of%2520pyridinone-dicarboxamide%2520for%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Aylott, H. E.</span>; <span class="NLM_string-name">Cooper, A. W. J.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hayhow, T. G. C.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Preston, A. G.</span>; <span class="NLM_string-name">Seal, J. T.</span>; <span class="NLM_string-name">Wall, I. D.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Woolven, J. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of pyridinone dicarboxamide derivatives for use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2017037116A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+J.+Atkinson&author=H.+E.+Aylott&author=A.+W.+J.+Cooper&author=E.+H.+Demont&author=L.+A.+Harrison&author=T.+G.+C.+Hayhow&author=M.+J.+Lindon&author=A.+G.+Preston&author=J.+T.+Seal&author=I.+D.+Wall&author=R.+J.+Watson&author=J.+M.+Woolven&title=Preparation+of+pyridinone+dicarboxamide+derivatives+for+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26atitle%3DPreparation%2520of%2520pyridinone%2520dicarboxamide%2520derivatives%2520for%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amans, D.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Barker, M. D.</span>; <span class="NLM_string-name">Bit, R. A.</span>; <span class="NLM_string-name">Brown, J. A.</span>; <span class="NLM_string-name">Campbell, M.</span>; <span class="NLM_string-name">Garton, N. S.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Shipley, T. J.</span>; <span class="NLM_string-name">Theodoulou, N. H.</span>; <span class="NLM_string-name">Wellaway, C. R.</span>; <span class="NLM_string-name">Westaway, S. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of furopyridines as bromodomain inhibitors useful in treating cancer, inflammation, and autoimmune disorders</span>. World Patent <span class="NLM_patent">WO2014140077A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+Amans&author=P.+Bamborough&author=M.+D.+Barker&author=R.+A.+Bit&author=J.+A.+Brown&author=M.+Campbell&author=N.+S.+Garton&author=M.+J.+Lindon&author=T.+J.+Shipley&author=N.+H.+Theodoulou&author=C.+R.+Wellaway&author=S.+M.+Westaway&title=Preparation+of+furopyridines+as+bromodomain+inhibitors+useful+in+treating+cancer%2C+inflammation%2C+and+autoimmune+disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAmans%26aufirst%3DD.%26atitle%3DPreparation%2520of%2520furopyridines%2520as%2520bromodomain%2520inhibitors%2520useful%2520in%2520treating%2520cancer%252C%2520inflammation%252C%2520and%2520autoimmune%2520disorders%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Dai, Y.</span>; <span class="NLM_string-name">Holms, J.</span>; <span class="NLM_string-name">Liu, D.</span>; <span class="NLM_string-name">McClellan, W.</span>; <span class="NLM_string-name">McDaniel, K.</span>; <span class="NLM_string-name">Hasvold, L.</span>; <span class="NLM_string-name">Fidanze, S. D.</span>; <span class="NLM_string-name">Sheppard, G.</span>; <span class="NLM_string-name">Marjanovic, J.</span></span> <span> </span><span class="NLM_article-title">Preparation of pyrrolopyridinone derivatives as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2014206345A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=L.+Wang&author=Y.+Dai&author=J.+Holms&author=D.+Liu&author=W.+McClellan&author=K.+McDaniel&author=L.+Hasvold&author=S.+D.+Fidanze&author=G.+Sheppard&author=J.+Marjanovic&title=Preparation+of+pyrrolopyridinone+derivatives+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26atitle%3DPreparation%2520of%2520pyrrolopyridinone%2520derivatives%2520as%2520bromodomain%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, S.</span>; <span class="NLM_string-name">Baeurle, S.</span>; <span class="NLM_string-name">Cleve, A.</span>; <span class="NLM_string-name">Haendler, B.</span>; <span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span> <span> </span><span class="NLM_article-title">Preparation of 1-phenyl-3H-2,3-benzodiazepines and their use as bromodomain inhibitors</span>. World Patent <span class="NLM_patent">WO2015121268A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=S.+Siegel&author=S.+Baeurle&author=A.+Cleve&author=B.+Haendler&author=A.+E.+Fernandez-Montalvan&title=Preparation+of+1-phenyl-3H-2%2C3-benzodiazepines+and+their+use+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DS.%26atitle%3DPreparation%2520of%25201-phenyl-3H-2%252C3-benzodiazepines%2520and%2520their%2520use%2520as%2520bromodomain%2520inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Combs, A. P.</span>; <span class="NLM_string-name">Sparks, R. B.</span>; <span class="NLM_string-name">Maduskuie, T. P.,  Jr.</span>; <span class="NLM_string-name">Rodgers, J. D.</span></span> <span> </span><span class="NLM_article-title">Preparation of tricyclic heterocycles as bet protein inhibitors</span>. World Patent <span class="NLM_patent">WO2014143768A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=A.+P.+Combs&author=R.+B.+Sparks&author=T.+P.+Maduskuie&author=J.+D.+Rodgers&title=Preparation+of+tricyclic+heterocycles+as+bet+protein+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCombs%26aufirst%3DA.%2BP.%26atitle%3DPreparation%2520of%2520tricyclic%2520heterocycles%2520as%2520bet%2520protein%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerner Seitzberg, J.</span>; <span class="NLM_string-name">Titilola Akinleminu
Kronborg, T.</span>; <span class="NLM_string-name">Poljak, V.</span>; <span class="NLM_string-name">Friberg, G.</span>; <span class="NLM_string-name">Teuber, L.</span></span> <span> </span><span class="NLM_article-title">Heterocyclic compounds as bromodomain Inhibitors and their preparation</span>. World Patent <span class="NLM_patent">WO2016016316A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Gerner+Seitzberg&author=T.+Titilola+Akinleminu%0AKronborg&author=V.+Poljak&author=G.+Friberg&author=L.+Teuber&title=Heterocyclic+compounds+as+bromodomain+Inhibitors+and+their+preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGerner%2BSeitzberg%26aufirst%3DJ.%26atitle%3DHeterocyclic%2520compounds%2520as%2520bromodomain%2520Inhibitors%2520and%2520their%2520preparation%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gacias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matikainen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaccia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">841</span>â <span class="NLM_lpage">854</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.chembiol.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24954007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=841-854&issue=7&author=M.+Gaciasauthor=G.+Gerona-Navarroauthor=A.+N.+Plotnikovauthor=G.+Zhangauthor=L.+Zengauthor=J.+Kaurauthor=G.+Moyauthor=E.+Rusinovaauthor=Y.+Rodriguezauthor=B.+Matikainenauthor=A.+Vincekauthor=J.+Joshuaauthor=P.+Casacciaauthor=M.-M.+Zhou&title=Selective+Chemical+Modulation+of+Gene+Transcription+Favors+Oligodendrocyte+Lineage+Progression&doi=10.1016%2Fj.chembiol.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span></div><div class="casAuthors">Gacias, Mar; Gerona-Navarro, Guillermo; Plotnikov, Alexander N.; Zhang, Guangtao; Zeng, Lei; Kaur, Jasbir; Moy, Gregory; Rusinova, Elena; Rodriguez, Yoel; Matikainen, Bridget; Vincek, Adam; Joshua, Jennifer; Casaccia, Patrizia; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-854</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation regulates gene expression through modulating protein-protein interactions in chromatin.  Chem. inhibition of acetyl-lysine binding bromodomains of the major chromatin regulators BET (bromodomain and extraterminal domain) proteins has been shown to effectively block cell proliferation in cancer and inflammation.  However, whether selective inhibition of individual BET bromodomains has distinctive functional consequences remains only partially understood.  In this study, we show that selective chem. inhibition of the first bromodomain of BET proteins using our small-mol. inhibitor, Olinone, accelerated the progression of mouse primary oligodendrocyte progenitors toward differentiation, whereas inhibition of both bromodomains of BET proteins hindered differentiation.  This effect was target specific, as it was not detected in cells treated with inactive analogs and independent of any effect on proliferation.  Therefore, selective chem. modulation of individual bromodomains, rather than use of broad-based inhibitors, may enhance regenerative strategies in disorders characterized by myelin loss such as aging and neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1fCvOR18yE7Vg90H21EOLACvtfcHk0lhMsZzJkf5exw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK&md5=6b377663a8d0d07441ab4f8a5d8191b3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DGacias%26aufirst%3DM.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DKaur%26aufirst%3DJ.%26aulast%3DMoy%26aufirst%3DG.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DMatikainen%26aufirst%3DB.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DCasaccia%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DSelective%2520Chemical%2520Modulation%2520of%2520Gene%2520Transcription%2520Favors%2520Oligodendrocyte%2520Lineage%2520Progression%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26issue%3D7%26spage%3D841%26epage%3D854%26doi%3D10.1016%2Fj.chembiol.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meslamani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2952</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1615601114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.1615601114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28265070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=2952&issue=11&author=K.+Cheungauthor=G.+Luauthor=R.+Sharmaauthor=A.+Vincekauthor=R.+Zhangauthor=A.+N.+Plotnikovauthor=F.+Zhangauthor=Q.+Zhangauthor=Y.+Juauthor=Y.+Huauthor=L.+Zhaoauthor=X.+Hanauthor=J.+Meslamaniauthor=F.+Xuauthor=A.+Jaganathanauthor=T.+Shenauthor=H.+Zhuauthor=E.+Rusinovaauthor=L.+Zengauthor=J.+Zhouauthor=J.+Yangauthor=L.+Pengauthor=M.+Ohlmeyerauthor=M.+J.+Walshauthor=D.+Y.+Zhangauthor=H.+Xiongauthor=M.-M.+Zhou&title=BET+N-terminal+bromodomain+inhibition+selectively+blocks+Th17+cell+differentiation+and+ameliorates+colitis+in+mice&doi=10.1073%2Fpnas.1615601114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">BET N-terminal bromodomain inhibition selectively blocks Th17 cell differentiation and ameliorates colitis in mice</span></div><div class="casAuthors">Cheung, Kalung; Lu, Geming; Sharma, Rajal; Vincek, Adam; Zhang, Ruihua; Plotnikov, Alexander N.; Zhang, Fan; Zhang, Qiang; Ju, Ying; Hu, Yuan; Zhao, Li; Han, Xinye; Meslamani, Jamel; Xu, Feihong; Jaganathan, Anbalagan; Shen, Tong; Zhu, Hongfa; Rusinova, Elena; Zeng, Lei; Zhou, Jiachi; Yang, Jianjun; Peng, Liang; Ohlmeyer, Michael; Walsh, Martin J.; Zhang, David Y.; Xiong, Huabao; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2952-2957</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">T-helper 17 (Th17) cells have important functions in adaptor immunity and have also been implicated in inflammatory disorders.  The bromodomain and extraterminal domain (BET) family proteins regulate gene transcription during lineage-specific differentiation of naive CD4+ T cells to produce mature T-helper cells.  Inhibition of acetyl-lysine binding of the BET proteins by pan-BET bromodomain (BrD) inhibitors, such as JQ1, broadly affects differentiation of Th17, Th1, and Th2 cells that have distinct immune functions, thus limiting their therapeutic potential.  Whether these BET proteins represent viable new epigenetic drug targets for inflammatory disorders has remained an unanswered question.  In this study, we report that selective inhibition of the first bromodomain of BET proteins with our newly designed small mol. MS402 inhibits primarily Th17 cell differentiation with a little or almost no effect on Th1 or Th2 and Treg cells.  MS402 preferentially renders Brd4 binding to Th17 signature gene loci over those of housekeeping genes and reduces Brd4 recruitment of p-TEFb to phosphorylate and activate RNA polymerase II for transcription elongation.  We further show that MS402 prevents and ameliorates T-cell transfer-induced colitis in mice by blocking Th17 cell overdevelopment.  Thus, selective pharmacol. modulation of individual bromodomains likely represents a strategy for treatment of inflammatory bowel diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBIvlTKnR97LVg90H21EOLACvtfcHk0lhMsZzJkf5exw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbo%253D&md5=367b7c0545edde6cc0ef107e03e1310f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1615601114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1615601114%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DJu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJaganathan%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DZhang%26aufirst%3DD.%2BY.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DBET%2520N-terminal%2520bromodomain%2520inhibition%2520selectively%2520blocks%2520Th17%2520cell%2520differentiation%2520and%2520ameliorates%2520colitis%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D11%26spage%3D2952%26doi%3D10.1073%2Fpnas.1615601114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">HÃ¼gle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostrovskyi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breit, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhardt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einsle, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¼nther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohlwend, D.</span></span> <span> </span><span class="NLM_article-title">4-Acyl Pyrrole Derivatives Yield Novel Vectors for Designing Inhibitors of the Acetyl-Lysine Recognition Site of BRD4(1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1518</span>â <span class="NLM_lpage">1530</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01267</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01267" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs1yksA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1518-1530&issue=4&author=M.+H%C3%BCgleauthor=X.+Lucasauthor=G.+Weitzelauthor=D.+Ostrovskyiauthor=B.+Breitauthor=S.+Gerhardtauthor=O.+Einsleauthor=S.+G%C3%BCntherauthor=D.+Wohlwend&title=4-Acyl+Pyrrole+Derivatives+Yield+Novel+Vectors+for+Designing+Inhibitors+of+the+Acetyl-Lysine+Recognition+Site+of+BRD4%281%29&doi=10.1021%2Facs.jmedchem.5b01267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">4-Acyl Pyrrole Derivatives Yield Novel Vectors for Designing Inhibitors of the Acetyl-Lysine Recognition Site of BRD4(1)</span></div><div class="casAuthors">Huegle, Martin; Lucas, Xavier; Weitzel, Gerhard; Ostrovskyi, Dmytro; Breit, Bernhard; Gerhardt, Stefan; Einsle, Oliver; Guenther, Stefan; Wohlwend, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1518-1530</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several human diseases, including cancer, show altered signaling pathways resulting from changes in the activity levels of epigenetic modulators.  In the past few years, small-mol. inhibitors against specific modulators, including the bromodomain and extra-terminal (BET) bromodomain family of acetylation readers, have shown early promise in the treatment of the genetically defined midline carcinoma and hematopoietic malignancies.  We have recently developed a novel potent inhibitor of BET proteins, 1 (XD14), which exerts a strong inhibitory potential on the proliferation of specific leukemia cell lines.  In the study presented here, we designed analogs of 1 to study the potential of substitutions on the 4-acyl pyrrole backbone to occupy addnl. sites within the substrate recognition site of BRD4(1).  The compds. were profiled using ITC, DSF, and X-ray crystallog.  We could introduce several substitutions that address previously untargeted areas of the substrate recognition site.  This work may substantially contribute to the development of therapeutics with increased target specificity against BRD4-related malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwg6psp_B35bVg90H21EOLACvtfcHk0lgZihkjhwrCmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs1yksA%253D%253D&md5=07296d23042a3ce2f2ecc1b6ae691e30</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01267%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCgle%26aufirst%3DM.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWeitzel%26aufirst%3DG.%26aulast%3DOstrovskyi%26aufirst%3DD.%26aulast%3DBreit%26aufirst%3DB.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DEinsle%26aufirst%3DO.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DWohlwend%26aufirst%3DD.%26atitle%3D4-Acyl%2520Pyrrole%2520Derivatives%2520Yield%2520Novel%2520Vectors%2520for%2520Designing%2520Inhibitors%2520of%2520the%2520Acetyl-Lysine%2520Recognition%2520Site%2520of%2520BRD4%25281%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1518%26epage%3D1530%26doi%3D10.1021%2Facs.jmedchem.5b01267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voitovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derviaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rebuffet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemot, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunel, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collette, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morelli, X.</span></span> <span> </span><span class="NLM_article-title">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1634</span>â <span class="NLM_lpage">1641</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1634-1641&issue=4&author=B.+Rauxauthor=Y.+Voitovichauthor=C.+Derviauxauthor=A.+Lugariauthor=E.+Rebuffetauthor=S.+Milhasauthor=S.+Prietauthor=T.+Rouxauthor=E.+Trinquetauthor=J.-C.+Guillemotauthor=S.+Knappauthor=J.-M.+Brunelauthor=A.+Y.+Fedorovauthor=Y.+Colletteauthor=P.+Rocheauthor=S.+Betziauthor=S.+Combesauthor=X.+Morelli&title=Exploring+Selective+Inhibition+of+the+First+Bromodomain+of+the+Human+Bromodomain+and+Extra-terminal+Domain+%28BET%29+Proteins&doi=10.1021%2Facs.jmedchem.5b01708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span></div><div class="casAuthors">Raux, Brigitt; Voitovich, Yuliia; Derviaux, Carine; Lugari, Adrien; Rebuffet, Etienne; Milhas, Sabine; Priet, Stephane; Roux, Thomas; Trinquet, Eric; Guillemot, Jean-Claude; Knapp, Stefan; Brunel, Jean-Michel; Fedorov, Alexey Yu.; Collette, Yves; Roche, Philippe; Betzi, Stephane; Combes, Sebastien; Morelli, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A midthroughput screening follow-up program targeting the first bromodomain of the human BRD4 protein, BRD4(BD1), identified an acetylated-mimic xanthine deriv. inhibitor.  This compd. binds with an affinity in the low micromolar range yet exerts suitable unexpected selectivity in vitro against the other members of the bromodomain and extra-terminal domain (BET) family.  A structure-based program pinpointed a role of the ZA loop, paving the way for the development of potent and selective BET-BRDi probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouQdXvF3r4zLVg90H21EOLACvtfcHk0lhOJRCmSNe6VQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D&md5=fb9cfc20c76f65fcf409370b60aecc0a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01708%26sid%3Dliteratum%253Aachs%26aulast%3DRaux%26aufirst%3DB.%26aulast%3DVoitovich%26aufirst%3DY.%26aulast%3DDerviaux%26aufirst%3DC.%26aulast%3DLugari%26aufirst%3DA.%26aulast%3DRebuffet%26aufirst%3DE.%26aulast%3DMilhas%26aufirst%3DS.%26aulast%3DPriet%26aufirst%3DS.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DGuillemot%26aufirst%3DJ.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrunel%26aufirst%3DJ.-M.%26aulast%3DFedorov%26aufirst%3DA.%2BY.%26aulast%3DCollette%26aufirst%3DY.%26aulast%3DRoche%26aufirst%3DP.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26atitle%3DExploring%2520Selective%2520Inhibition%2520of%2520the%2520First%2520Bromodomain%2520of%2520the%2520Human%2520Bromodomain%2520and%2520Extra-terminal%2520Domain%2520%2528BET%2529%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Facs.jmedchem.5b01708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">9251</span>â <span class="NLM_lpage">9264</span>, <span class="refDoi">Â DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&issue=22&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.-M.+Zhou&title=Structure-Guided+Design+of+Potent+Diazobenzene+Inhibitors+for+the+BET+Bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-ÎºB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0lgVVHhec_V-Kw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure-Guided%2520Design%2520of%2520Potent%2520Diazobenzene%2520Inhibitors%2520for%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D22%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual KinaseâBromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9316</span>â <span class="NLM_lpage">9334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&issue=20&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.-Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Molecular+Basis+for+the+N-Terminal+Bromodomain-and-Extra-Terminal-Family+Selectivity+of+a+Dual+Kinase%E2%80%93Bromodomain+Inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-Îµ-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-ÎºB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38Î±.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0ljnIGYj5b-CfQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DMolecular%2520Basis%2520for%2520the%2520N-Terminal%2520Bromodomain-and-Extra-Terminal-Family%2520Selectivity%2520of%2520a%2520Dual%2520Kinase%25E2%2580%2593Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D20%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Bernard, S.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Craggs, P. D.</span>; <span class="NLM_string-name">Cutler, L.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Evans, J. P.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Karamshi, B.</span>; <span class="NLM_string-name">Lewis, A. J.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Soden, P. E.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Willis, R.</span>; <span class="NLM_string-name">Woolven, J. M.</span>; <span class="NLM_string-name">Wyspianska, B. S.</span>; <span class="NLM_string-name">Kerr, W. J.</span>; <span class="NLM_string-name">Prinjha, R. K.</span></span>, <span> </span><span class="NLM_article-title">Structure-based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-terminal Bromodomains that Retains an Anti-inflammatory and Anti-proliferative Phenotype</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=C.+R.+Wellaway&author=P.+Bamborough&author=S.+Bernard&author=C.-w.+Chung&author=P.+D.+Craggs&author=L.+Cutler&author=E.+H.+Demont&author=J.+P.+Evans&author=L.+Gordon&author=B.+Karamshi&author=A.+J.+Lewis&author=M.+J.+Lindon&author=D.+J.+Mitchell&author=I.+Rioja&author=P.+E.+Soden&author=S.+Taylor&author=R.+J.+Watson&author=R.+Willis&author=J.+M.+Woolven&author=B.+S.+Wyspianska&author=W.+J.+Kerr&author=R.+K.+Prinjha&title=Structure-based+Design+of+a+Bromodomain+and+Extraterminal+Domain+%28BET%29+Inhibitor+Selective+for+the+N-terminal+Bromodomains+that+Retains+an+Anti-inflammatory+and+Anti-proliferative+Phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26atitle%3DStructure-based%2520Design%2520of%2520a%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520Inhibitor%2520Selective%2520for%2520the%2520N-terminal%2520Bromodomains%2520that%2520Retains%2520an%2520Anti-inflammatory%2520and%2520Anti-proliferative%2520Phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1492</span>â <span class="NLM_lpage">1500</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1492-1500&issue=4&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=M.+Zengerleauthor=C.+Tallantauthor=A.+Ciulli&title=New+Synthetic+Routes+to+Triazolo-benzodiazepine+Analogues%3A+Expanding+the+Scope+of+the+Bump-and-Hole+Approach+for+Selective+Bromo+and+Extra-Terminal+%28BET%29+Bromodomain+Inhibition&doi=10.1021%2Facs.jmedchem.5b01135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Zengerle, Michael; Tallant, Cynthia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1492-1500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe new synthetic routes developed toward a range of substituted analogs of bromo and extra-terminal (BET) bromodomain inhibitors I-BET762/JQ1 based on the triazolo-benzodiazepine scaffold.  These new routes allow for the derivatization of the methoxyphenyl and chlorophenyl rings, in addn. to the diazepine ternary center and the side chain methylene moiety.  Substitution at the level of the side chain methylene afforded compds. targeting specifically and potently engineered BET bromodomains designed as part of a bump and hole approach.  We further demonstrate that marked selectivity for the second over the first bromodomain can be achieved with an indole deriv. that exploits differential interaction with an aspartate/histidine conservative substitution on the BC loop of BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpYVun8F94bVg90H21EOLACvtfcHk0ljIb11xHRc5jg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE&md5=65dd39e3f7e0bc52169ffb1466218c43</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520Synthetic%2520Routes%2520to%2520Triazolo-benzodiazepine%2520Analogues%253A%2520Expanding%2520the%2520Scope%2520of%2520the%2520Bump-and-Hole%2520Approach%2520for%2520Selective%2520Bromo%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Bromodomain%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1492%26epage%3D1500%26doi%3D10.1021%2Facs.jmedchem.5b01135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">49</span>),  <span class="NLM_fpage">19754</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754&issue=49&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lhoFPkCb8Ru8g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D49%26spage%3D19754%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4317</span>â <span class="NLM_lpage">4334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&issue=10&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+Tetrahydroquinoxalines+as+Bromodomain+and+Extra-Terminal+Domain+%28BET%29+Inhibitors+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0liByosSkFVz6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520Tetrahydroquinoxalines%2520as%2520Bromodomain%2520and%2520Extra-Terminal%2520Domain%2520%2528BET%2529%2520Inhibitors%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D10%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span> (<span class="NLM_issue">7794</span>),  <span class="NLM_fpage">306</span>â <span class="NLM_lpage">310</span>, <span class="refDoi">Â DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&issue=7794&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0liByosSkFVz6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26issue%3D7794%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>â <span class="NLM_lpage">5623</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+L.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+Kovarauthor=M.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-Ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+Bromodomain+Inhibitor+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0liByosSkFVz6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%2BL.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-Ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery, Structural Insight, and Bioactivities of BY27 as a Selective Inhibitor of the Second Bromodomains of BET Proteins</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">111633</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2019.111633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.ejmech.2019.111633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=31461688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2019&pages=111633&author=D.+Chenauthor=T.+Luauthor=Z.+Yanauthor=W.+Luauthor=F.+Zhouauthor=X.+Lyuauthor=B.+Xuauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=Y.+Zhao&title=Discovery%2C+Structural+Insight%2C+and+Bioactivities+of+BY27+as+a+Selective+Inhibitor+of+the+Second+Bromodomains+of+BET+Proteins&doi=10.1016%2Fj.ejmech.2019.111633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins</span></div><div class="casAuthors">Chen, Deheng; Lu, Tian; Yan, Ziqin; Lu, Wenchao; Zhou, Feilong; Lyu, Xilin; Xu, Biling; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zhao, Yujun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111633</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, selective inhibition of BET BD2 is emerging as a promising strategy for drug discovery.  Despite significant progress in this area, systematic studies of selective BET BD2 inhibitors are still few.  In this study, we report the discovery of a potent and selective BET BD2 inhibitor BY27 (47).  Our high resoln. co-crystal structures of 47/BRD2 BD1 and BD2 showed that the triazole group of 47, water mols., H433 and N429 in BRD2 BD2 established a water-bridged H-bonding network, which is responsible for the obsd. selectivities.  DNA microarray anal. of HepG2 cells treated with 47 or OTX015 demonstrated the transcriptome impact differences between a BET BD2 selective inhibitor and a pan BET inhibitor.  In a MV4-11 mouse xenograft model, 47 caused 67% of tumor growth inhibition and was less toxic than a pan BET inhibitor 1 at high doses.  We conclude that the improved safety profile of selective BET BD2 inhibitors warrant future studies in BET assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre_6szzmO9ZbVg90H21EOLACvtfcHk0lj5CBsik_Nj_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ&md5=e3a21e4e692b25a94e0035a2e75d33f7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DLyu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DDiscovery%252C%2520Structural%2520Insight%252C%2520and%2520Bioactivities%2520of%2520BY27%2520as%2520a%2520Selective%2520Inhibitor%2520of%2520the%2520Second%2520Bromodomains%2520of%2520BET%2520Proteins%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D182%26spage%3D111633%26doi%3D10.1016%2Fj.ejmech.2019.111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span> <span> </span><span class="NLM_article-title">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>477</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">62</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">Â DOI: 10.1016/j.bbrc.2016.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.bbrc.2016.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27282480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2016&pages=62-67&issue=1&author=O.+A.+Kharenkoauthor=E.+M.+Gesnerauthor=R.+G.+Patelauthor=K.+Norekauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=H.+C.+Hansen&title=RVX-297-+a+novel+BD2+selective+inhibitor+of+BET+bromodomains&doi=10.1016%2Fj.bbrc.2016.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span></div><div class="casAuthors">Kharenko, Olesya A.; Gesner, Emily M.; Patel, Reena G.; Norek, Karen; White, Andre; Fontano, Eric; Suto, Robert K.; Young, Peter R.; McLure, Kevin G.; Hansen, Henrik C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that specifically bind to the acetyl lysine residues of histones and transcription factors.  Small mol. BET bromodomain inhibitors can disrupt this interaction which leads to potential modulation of several disease states.  Here we describe the binding properties of a novel BET inhibitor RVX-297 that is structurally related to the clin. compd. RVX-208, currently undergoing phase III clin. trials for the treatment of cardiovascular diseases, but is distinctly different in its biol. and pharmacokinetic profiles.  We report that RVX-297 preferentially binds to the BD2 domains of the BET bromodomain and Extra Terminal (BET) family of protein.  We demonstrate the differential binding modes of RVX-297 in BD1 and BD2 domains of BRD4 and BRD2 using X-ray crystallog., and describe the structural differences driving the BD2 selective binding of RVX-297.  The isothermal titrn. calorimetry (ITC) data illustrate the related differential thermodn. of binding of RVX-297 to single as well as dual BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLUg96I7HObVg90H21EOLACvtfcHk0lj5CBsik_Nj_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D&md5=a3d064e2245bf14ae525eac2490808cb</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26atitle%3DRVX-297-%2520a%2520novel%2520BD2%2520selective%2520inhibitor%2520of%2520BET%2520bromodomains%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D477%26issue%3D1%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.bbrc.2016.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marusic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenoy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulikowski, E.</span></span> <span> </span><span class="NLM_article-title">RVX-297, a BET Bromodomain Inhibitor, Has Therapeutic Effects in Preclinical Models of Acute Inflammation and Autoimmune Disease</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">694</span>, <span class="refDoi">Â DOI: 10.1124/mol.117.110379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1124%2Fmol.117.110379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=28974538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=694&issue=6&author=R.+Jahagirdarauthor=S.+Attwellauthor=S.+Marusicauthor=A.+Bendeleauthor=N.+Shenoyauthor=K.+G.+McLureauthor=D.+Gilhamauthor=K.+Norekauthor=H.+C.+Hansenauthor=R.+Yuauthor=J.+Tobinauthor=G.+S.+Wagnerauthor=P.+R.+Youngauthor=N.+C.+W.+Wongauthor=E.+Kulikowski&title=RVX-297%2C+a+BET+Bromodomain+Inhibitor%2C+Has+Therapeutic+Effects+in+Preclinical+Models+of+Acute+Inflammation+and+Autoimmune+Disease&doi=10.1124%2Fmol.117.110379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297, a BET bromodomain inhibitor, has therapeutic effects in preclinical models of acute inflammation and autoimmune disease</span></div><div class="casAuthors">Jahagirdar, Ravi; Attwell, Sarah; Marusic, Suzana; Bendele, Alison; Shenoy, Narmada; McLure, Kevin G.; Gilham, Dean; Norek, Karen; Hansen, Henrik C.; Yu, Raymond; Tobin, Jennifer; Wagner, Gregory S.; Young, Peter R.; Wong, Norman C. W.; Kulikowski, Ewelina</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">694-706</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Bromodomain (BD) and extra-terminal domain contg. proteins (BET) are chromatin adapters that bind acetylated histone marks via two tandem BDs, BD1 and BD2, to regulate gene transcription.  BET proteins are involved in transcriptional reprogramming in response to inflammatory stimuli.  BET BD inhibitors (BETis) that are nonselective for BD1 or BD2 have recognized anti-inflammatory properties in vitro and counter pathol. in models of inflammation or autoimmune disease.  Although both BD1 and BD2 bind acetylated histone residues, they may independently regulate the expression of BET-sensitive genes.  Here we characterized the ability of RVX-297, a novel orally active BETi with selectivity for BD2, to modulate inflammatory processes in vitro, in vivo, and ex vivo.  RVX-297 suppressed inflammatory gene expression in multiple immune cell types in culture.  Mechanistically, RVX-297 displaced BET proteins from the promoters of sensitive genes and disrupted recruitment of active RNA polymerase II, a property shared with pan-BETis that nonselectively bind BET BDs.  In the lipopolysaccharide model of inflammation, RVX-297 reduced proinflammatory mediators assessed in splenic gene expression and serum proteins.  RVX-297 also countered pathol. in three rodent models of polyarthritis: rat and mouse collagen-induced arthritis, and mouse collagen antibody-induced arthritis.  Further, RVX-297 prevented murine exptl. autoimmune encephalomyelitis (a model of human multiple sclerosis) disease development when administered prophylactically and reduced hallmarks of pathol. when administered therapeutically.  We show for the first time that a BD2-selective BETi maintains anti-inflammatory properties and is effective in preclin. models of acute inflammation and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7r1JpLemFA7Vg90H21EOLACvtfcHk0lj5CBsik_Nj_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK&md5=3f1a1b86b17315c413dd1f13d3a7736a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.110379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.110379%26sid%3Dliteratum%253Aachs%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DMarusic%26aufirst%3DS.%26aulast%3DBendele%26aufirst%3DA.%26aulast%3DShenoy%26aufirst%3DN.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGilham%26aufirst%3DD.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DTobin%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DKulikowski%26aufirst%3DE.%26atitle%3DRVX-297%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%252C%2520Has%2520Therapeutic%2520Effects%2520in%2520Preclinical%2520Models%2520of%2520Acute%2520Inflammation%2520and%2520Autoimmune%2520Disease%26jtitle%3DMol.%2520Pharmacol.%26date%3D2017%26volume%3D92%26issue%3D6%26spage%3D694%26doi%3D10.1124%2Fmol.117.110379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective Targeting of BD1 and BD2 of the BET Proteins in Cancer and Immuno-Inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">eaaz8455</span>, <span class="refDoi">Â DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=eaaz8455&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.-M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.-J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+Targeting+of+BD1+and+BD2+of+the+BET+Proteins+in+Cancer+and+Immuno-Inflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520Targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520Proteins%2520in%2520Cancer%2520and%2520Immuno-Inflammation%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3Deaaz8455%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, T. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rybin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GrÃ¶tsch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlomagno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, C. W.</span></span> <span> </span><span class="NLM_article-title">A bromodomainâDNA interaction facilitates acetylation-dependent bivalent nucleosome recognition by the BET protein BRDT</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13855</span>, <span class="refDoi">Â DOI: 10.1038/ncomms13855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fncomms13855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=27991587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGjtb7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13855&author=T.+C.+R.+Millerauthor=B.+Simonauthor=V.+Rybinauthor=H.+Gr%C3%B6tschauthor=S.+Curtetauthor=S.+Khochbinauthor=T.+Carlomagnoauthor=C.+W.+M%C3%BCller&title=A+bromodomain%E2%80%93DNA+interaction+facilitates+acetylation-dependent+bivalent+nucleosome+recognition+by+the+BET+protein+BRDT&doi=10.1038%2Fncomms13855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A bromodomain-DNA interaction facilitates acetylation-dependent bivalent nucleosome recognition by the BET protein BRDT</span></div><div class="casAuthors">Miller, Thomas C. R.; Simon, Bernd; Rybin, Vladimir; Groetsch, Helga; Curtet, Sandrine; Khochbin, Saadi; Carlomagno, Teresa; Mueller, Christoph W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13855</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomains are crit. components of many chromatin modifying/remodelling proteins and are emerging therapeutic targets, yet how they interact with nucleosomes, rather than acetylated peptides, remains unclear.  Using BRDT as a model, we characterized how the BET family of bromodomains interacts with site-specifically acetylated nucleosomes.  Here we report that BRDT interacts with nucleosomes through its first (BD1), but not second (BD2) bromodomain, and that acetylated histone recognition by BD1 is complemented by a bromodomain-DNA interaction.  Simultaneous DNA and histone recognition enhances BRDT's nucleosome binding affinity and specificity, and its ability to localize to acetylated chromatin in cells.  Conservation of DNA binding in bromodomains of BRD2, BRD3 and BRD4, indicates that bivalent nucleosome recognition is a key feature of these bromodomains and possibly others.  Our results elucidate the mol. mechanism of BRDT assocn. with nucleosomes and identify structural features of the BET bromodomains that may be targeted for therapeutic inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxf-nKlLEUBbVg90H21EOLACvtfcHk0li1G-ynrlJVTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGjtb7J&md5=08e3161181401de04130148fa4c23150</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fncomms13855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13855%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DT.%2BC.%2BR.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DRybin%26aufirst%3DV.%26aulast%3DGr%25C3%25B6tsch%26aufirst%3DH.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DCarlomagno%26aufirst%3DT.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BW.%26atitle%3DA%2520bromodomain%25E2%2580%2593DNA%2520interaction%2520facilitates%2520acetylation-dependent%2520bivalent%2520nucleosome%2520recognition%2520by%2520the%2520BET%2520protein%2520BRDT%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13855%26doi%3D10.1038%2Fncomms13855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, T. T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span> (<span class="NLM_issue">6209</span>),  <span class="NLM_fpage">638</span>, <span class="refDoi">Â DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638&issue=6209&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.+L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=A+bump-and-hole+approach+to+engineer+controlled+selectivity+of+BET+bromodomain+chemical+probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0li1G-ynrlJVTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.%2BL.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DA%2520bump-and-hole%2520approach%2520to%2520engineer%2520controlled%2520selectivity%2520of%2520BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26issue%3D6209%26spage%3D638%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Runcie, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Beurden, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulthard, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of a â³bump-and-holeâ³ approach to allele-selective BET bromodomain inhibition</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2452</span>, <span class="refDoi">Â DOI: 10.1039/C7SC02536J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1039%2FC7SC02536J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=29732121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2452&author=A.+C.+Runcieauthor=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Testaauthor=L.+van+Beurdenauthor=M.+G.+J.+Baudauthor=O.+Epemoluauthor=L.+C.+J.+Ellisauthor=K.+D.+Readauthor=V.+Coulthardauthor=A.+Brienauthor=A.+Ciulli&title=Optimization+of+a+%E2%80%B3bump-and-hole%E2%80%B3+approach+to+allele-selective+BET+bromodomain+inhibition&doi=10.1039%2FC7SC02536J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a "bump-and-hole" approach to allele-selective BET bromodomain inhibition</span></div><div class="casAuthors">Runcie, A. C.; Zengerle, M.; Chan, K.-H.; Testa, A.; van Beurden, L.; Baud, M. G. J.; Epemolu, O.; Ellis, L. C. J.; Read, K. D.; Coulthard, V.; Brien, A.; Ciulli, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2452-2468</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Allele-specific chem. genetics enables selective inhibition within families of highly-conserved proteins.  The four BET (bromodomain & extra-terminal domain) proteins - BRD2, BRD3, BRD4 and BRDT bind acetylated chromatin via their bromodomains and regulate processes such as cell proliferation and inflammation.  BET bromodomains are of particular interest, as they are attractive therapeutic targets but existing inhibitors are pan-selective.  We previously established a bump-&-hole system for the BET bromodomains, pairing a leucine/alanine mutation with an ethyl-derived analog of an established benzodiazepine scaffold.  Here we optimize upon this system with the introduction of a more conservative and less disruptive leucine/valine mutation.  Extensive structure-activity-relationships of diverse benzodiazepine analogs guided the development of potent, mutant-selective inhibitors with desirable physiochem. properties.  The active enantiomer of our best compd. - 9-ME-1 - shows â¼200 nM potency, >100-fold selectivity for the L/V mutant over wild-type and excellent DMPK properties.  Through a variety of in vitro and cellular assays we validate the capabilities of our optimized system, and then utilize it to compare the relative importance of the first and second bromodomains to chromatin binding.  These expts. confirm the primacy of the first bromodomain in all BET proteins, but also significant variation in the importance of the second bromodomain.  We also show that, despite having a minor role in chromatin recognition, BRD4 BD2 is still essential for gene expression, likely through the recruitment of non-histone proteins.  The disclosed inhibitor:mutant pair provides a powerful tool for future cellular and in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq47qHG4-zFurVg90H21EOLACvtfcHk0liqvA13c9a1kQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D&md5=9a44bdccabca9f1d4dfdc983250d9aa3</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2FC7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02536J%26sid%3Dliteratum%253Aachs%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3Dvan%2BBeurden%26aufirst%3DL.%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DEllis%26aufirst%3DL.%2BC.%2BJ.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCoulthard%26aufirst%3DV.%26aulast%3DBrien%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DOptimization%2520of%2520a%2520%25E2%2580%25B3bump-and-hole%25E2%2580%25B3%2520approach%2520to%2520allele-selective%2520BET%2520bromodomain%2520inhibition%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26spage%3D2452%26doi%3D10.1039%2FC7SC02536J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olp, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. C.</span></span> <span> </span><span class="NLM_article-title">Nucleosome Scaffolding by Brd4 Tandem Bromodomains in Acetylation-Dependent Chromatin Compartmentalization</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">699967</span>, <span class="refDoi">Â DOI: 10.1101/699967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1101%2F699967" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=699967&author=M.+D.+Olpauthor=V.+Jacksonauthor=B.+C.+Smith&title=Nucleosome+Scaffolding+by+Brd4+Tandem+Bromodomains+in+Acetylation-Dependent+Chromatin+Compartmentalization&doi=10.1101%2F699967"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1101%2F699967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F699967%26sid%3Dliteratum%253Aachs%26aulast%3DOlp%26aufirst%3DM.%2BD.%26aulast%3DJackson%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DB.%2BC.%26atitle%3DNucleosome%2520Scaffolding%2520by%2520Brd4%2520Tandem%2520Bromodomains%2520in%2520Acetylation-Dependent%2520Chromatin%2520Compartmentalization%26jtitle%3DbioRxiv%26date%3D2019%26spage%3D699967%26doi%3D10.1101%2F699967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">The maximal affinity of ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">9997</span>, <span class="refDoi">Â DOI: 10.1073/pnas.96.18.9997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997&issue=18&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands&doi=10.1073%2Fpnas.96.18.9997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of â-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0liqvA13c9a1kQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26issue%3D18%26spage%3D9997%26doi%3D10.1073%2Fpnas.96.18.9997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">430</span>â <span class="NLM_lpage">431</span>, <span class="refDoi">Â DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&issue=10&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eaOqD_b7crBFLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26issue%3D10%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span> <span> </span><span class="NLM_article-title">Assessing the lipophilicity of fragments and early hits</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">663</span>â <span class="NLM_lpage">667</span>, <span class="refDoi">Â DOI: 10.1007/s10822-011-9435-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1007%2Fs10822-011-9435-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21614595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=663-667&issue=7&author=P.+N.+Mortensonauthor=C.+W.+Murray&title=Assessing+the+lipophilicity+of+fragments+and+early+hits&doi=10.1007%2Fs10822-011-9435-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the lipophilicity of fragments and early hits</span></div><div class="casAuthors">Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">663-667</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  A key challenge in many drug discovery programs is to accurately assess the potential value of screening hits.  This is particularly true in fragment-based drug design (FBDD), where the hits often bind relatively weakly, but are correspondingly small.  Ligand efficiency (LE) considers both the potency and the size of the mol., and enables us to est. whether or not an initial hit is likely to be optimizable to a potent, druglike lead.  While size is a key property that needs to be controlled in a small mol. drug, there are a no. of addnl. properties that should also be considered.  Lipophilicity is amongst the most important of these addnl. properties, and here we present a new efficiency index (LLEAT) that combines lipophilicity, size and potency.  The index is intuitively defined, and has been designed to have the same target value and dynamic range as LE, making it easily interpretable by medicinal chemists.  Monitoring both LE and LLEAT should help both in the selection of more promising fragment hits, and controlling mol. wt. and lipophilicity during optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCnbfcwp30BrVg90H21EOLACvtfcHk0liE9rbpskTs_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbnL&md5=e6d68046d3d20bd2ea088ce218451c25</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9435-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9435-z%26sid%3Dliteratum%253Aachs%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DAssessing%2520the%2520lipophilicity%2520of%2520fragments%2520and%2520early%2520hits%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26issue%3D7%26spage%3D663%26epage%3D667%26doi%3D10.1007%2Fs10822-011-9435-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery II: The Impact of Chromatographic Hydrophobicity Measurements and Aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">822</span>â <span class="NLM_lpage">830</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&issue=17&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+Physical+in+Drug+Discovery+II%3A+The+Impact+of+Chromatographic+Hydrophobicity+Measurements+and+Aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0liE9rbpskTs_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%2520II%253A%2520The%2520Impact%2520of%2520Chromatographic%2520Hydrophobicity%2520Measurements%2520and%2520Aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D17%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ClÃ©ment, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3827</span>â <span class="NLM_lpage">3838</span>, <span class="refDoi">Â DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&issue=11&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lj7mY2q5bJw5g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D11%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>, <span class="refDoi">Â DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lj7mY2q5bJw5g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">214</span>â <span class="NLM_lpage">231</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&issue=1&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize Îµ-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lj7mY2q5bJw5g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26issue%3D1%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muniz, J. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Low-Druggability Bromodomains: Fragment Based Screening and Inhibitor Design against the BAZ2B Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">10183</span>â <span class="NLM_lpage">10187</span>, <span class="refDoi">Â DOI: 10.1021/jm401582c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401582c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltL%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=10183-10187&issue=24&author=F.+M.+Fergusonauthor=O.+Fedorovauthor=A.+Chaikuadauthor=M.+Philpottauthor=J.+R.+C.+Munizauthor=I.+Felletarauthor=F.+von+Delftauthor=T.+Heightmanauthor=S.+Knappauthor=C.+Abellauthor=A.+Ciulli&title=Targeting+Low-Druggability+Bromodomains%3A+Fragment+Based+Screening+and+Inhibitor+Design+against+the+BAZ2B+Bromodomain&doi=10.1021%2Fjm401582c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Low-Druggability Bromodomains: Fragment Based Screening and Inhibitor Design against the BAZ2B Bromodomain</span></div><div class="casAuthors">Ferguson, Fleur M.; Fedorov, Oleg; Chaikuad, Apirat; Philpott, Martin; Muniz, Joao R. C.; Felletar, Ildiko; von Delft, Frank; Heightman, Tom; Knapp, Stefan; Abell, Chris; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10183-10187</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader domains that have recently become popular targets.  In contrast to BET bromodomains, which have proven druggable, bromodomains from other regions of the phylogenetic tree have shallower pockets.  We describe successful targeting of the challenging BAZ2B bromodomain using biophys. fragment screening and structure-based optimization of high ligand-efficiency fragments into a novel series of low-micromolar inhibitors.  Our results provide attractive leads for development of BAZ2B chem. probes and indicate the whole family may be tractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcsl8yJ7AirVg90H21EOLACvtfcHk0lgf58Zx1PQL2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltL%252FK&md5=e10f9392a72a2eb946cf4aabb0a27fb1</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjm401582c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401582c%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMuniz%26aufirst%3DJ.%2BR.%2BC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DHeightman%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520Low-Druggability%2520Bromodomains%253A%2520Fragment%2520Based%2520Screening%2520and%2520Inhibitor%2520Design%2520against%2520the%2520BAZ2B%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D24%26spage%3D10183%26epage%3D10187%26doi%3D10.1021%2Fjm401582c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Seal, J.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Gray, J. R. J.</span>; <span class="NLM_string-name">Woolven, J.</span>; <span class="NLM_string-name">Wall, I.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Rianjongdee, F.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Michon, A.-M.</span>; <span class="NLM_string-name">Grandi, P.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Jones, E. J.</span>; <span class="NLM_string-name">Craggs, P. D.</span>; <span class="NLM_string-name">Prinjha, R. K.</span>; <span class="NLM_string-name">Lindon, M.</span>; <span class="NLM_string-name">Demont, E. H.</span></span>, <span> </span><span class="NLM_article-title">The Design and Synthesis of a Highly Selective and In Vivo Capable Inhibitor of the Second Bromodomain (BD2) of the Bromodomain and Extra Terminal Domain (BET) Family of Proteins</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Preston&author=S.+J.+Atkinson&author=J.+Seal&author=D.+J.+Mitchell&author=R.+J.+Watson&author=J.+R.+J.+Gray&author=J.+Woolven&author=I.+Wall&author=C.-w.+Chung&author=P.+Bamborough&author=F.+Rianjongdee&author=S.+Taylor&author=A.-M.+Michon&author=P.+Grandi&author=I.+Rioja&author=L.+Gordon&author=E.+J.+Jones&author=P.+D.+Craggs&author=R.+K.+Prinjha&author=M.+Lindon&author=E.+H.+Demont&title=The+Design+and+Synthesis+of+a+Highly+Selective+and+In+Vivo+Capable+Inhibitor+of+the+Second+Bromodomain+%28BD2%29+of+the+Bromodomain+and+Extra+Terminal+Domain+%28BET%29+Family+of+Proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26atitle%3DThe%2520Design%2520and%2520Synthesis%2520of%2520a%2520Highly%2520Selective%2520and%2520In%2520Vivo%2520Capable%2520Inhibitor%2520of%2520the%2520Second%2520Bromodomain%2520%2528BD2%2529%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520Family%2520of%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2615</span>â <span class="NLM_lpage">2623</span>, <span class="refDoi">Â DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&issue=12&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+Properties+That+Influence+the+Oral+Bioavailability+of+Drug+Candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area â¤140 Ã2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0ljs02ZNCdpE-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520Properties%2520That%2520Influence%2520the%2520Oral%2520Bioavailability%2520of%2520Drug%2520Candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26issue%3D12%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">648</span>â <span class="NLM_lpage">655</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and â¼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0ljs02ZNCdpE-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skipper, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wogan, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannenbaum, S. R.</span></span> <span> </span><span class="NLM_article-title">Monocyclic aromatic amines as potential human carcinogens: old is new again</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">50</span>â <span class="NLM_lpage">58</span>, <span class="refDoi">Â DOI: 10.1093/carcin/bgp267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1093%2Fcarcin%2Fbgp267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=19887514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=50-58&issue=1&author=P.+L.+Skipperauthor=M.+Y.+Kimauthor=H.+L.+P.+Sunauthor=G.+N.+Woganauthor=S.+R.+Tannenbaum&title=Monocyclic+aromatic+amines+as+potential+human+carcinogens%3A+old+is+new+again&doi=10.1093%2Fcarcin%2Fbgp267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyclic aromatic amines as potential human carcinogens: old is new again</span></div><div class="casAuthors">Skipper, Paul L.; Kim, Min Young; Sun, H.-L. Patty; Wogan, Gerald N.; Tannenbaum, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-58</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Alkylanilines are a group of chems. whose ubiquitous presence in the environment is a result of the multitude of sources from which they originate.  Exposure assessments indicate that most individuals experience lifelong exposure to these compds.  Many alkylanilines have biol. activity similar to that of the carcinogenic multi-ring arom. amines.  This review provides an overview of human exposure and biol. effects.  It also describes recent investigations into the biochem. mechanisms of action that lead to the assessment that they are most probably more complex than those of the more extensively investigated multi-ring arom. amines.  Not only is nitrenium ion chem. implicated in DNA damage by alkylanilines but also reactions involving quinone imines and perhaps reactive oxygen species.  Recent results described here indicate that alkylanilines can be potent genotoxins for cultured mammalian cells when activated by exogenous or endogenous phase I and phase II xenobiotic-metabolizing enzymes.  The nature of specific DNA damage products responsible for mutagenicity remains to be identified but evidence to date supports mechanisms of activation through obligatory N-hydroxylation as well as subsequent conjugation by sulfation and/or acetylation.  A fuller understanding of the mechanisms of alkylaniline genotoxicity is expected to provide important insights into the environmental and genetic origins of one or more human cancers and may reveal a substantial role for this group of compds. as potential human chem. carcinogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8jtJ_-C4s7Vg90H21EOLACvtfcHk0ljsosJk3YH1Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D&md5=37e5a874a1ee9edf96acb3ad73ec2925</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp267%26sid%3Dliteratum%253Aachs%26aulast%3DSkipper%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DM.%2BY.%26aulast%3DSun%26aufirst%3DH.%2BL.%2BP.%26aulast%3DWogan%26aufirst%3DG.%2BN.%26aulast%3DTannenbaum%26aufirst%3DS.%2BR.%26atitle%3DMonocyclic%2520aromatic%2520amines%2520as%2520potential%2520human%2520carcinogens%253A%2520old%2520is%2520new%2520again%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26issue%3D1%26spage%3D50%26epage%3D58%26doi%3D10.1093%2Fcarcin%2Fbgp267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span> <span> </span><span class="NLM_article-title">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">929</span>, <span class="refDoi">Â DOI: 10.1038/nrd3287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2Fnrd3287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=21119731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=929&author=D.+A.+Smithauthor=L.+Diauthor=E.+H.+Kerns&title=The+effect+of+plasma+protein+binding+on+in+vivo+efficacy%3A+misconceptions+in+drug+discovery&doi=10.1038%2Fnrd3287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span></div><div class="casAuthors">Smith, Dennis A.; Di, Li; Kerns, Edward H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">929-939</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Data from in vitro plasma protein binding expts. that det. the fraction of protein-bound drug are frequently used in drug discovery to guide structure design and to prioritize compds. for in vivo studies.  However, we consider that these practices are usually misleading, because in vivo efficacy is detd. by the free (unbound) drug concn. surrounding the therapeutic target, not by the free drug fraction.  These practices yield no enhancement of the in vivo free drug concn.  So, decisions based on free drug fraction could result in the wrong compds. being advanced through drug discovery programs.  This Perspective provides guidance on the application of plasma protein binding information in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOa3ippodhGbVg90H21EOLACvtfcHk0ljsosJk3YH1Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI&md5=a45d00f68ab44580fa38a6acef49493a</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnrd3287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3287%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DThe%2520effect%2520of%2520plasma%2520protein%2520binding%2520on%2520in%2520vivo%2520efficacy%253A%2520misconceptions%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D929%26doi%3D10.1038%2Fnrd3287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manabe, K.</span></span> <span> </span><span class="NLM_article-title">Trichlorophenyl Formate: Highly Reactive and Easily Accessible Crystalline CO Surrogate for Palladium-Catalyzed Carbonylation of Aryl/Alkenyl Halides and Triflates</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">5370</span>â <span class="NLM_lpage">5373</span>, <span class="refDoi">Â DOI: 10.1021/ol302593z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol302593z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVejs7vP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=5370-5373&issue=20&author=T.+Uedaauthor=H.+Konishiauthor=K.+Manabe&title=Trichlorophenyl+Formate%3A+Highly+Reactive+and+Easily+Accessible+Crystalline+CO+Surrogate+for+Palladium-Catalyzed+Carbonylation+of+Aryl%2FAlkenyl+Halides+and+Triflates&doi=10.1021%2Fol302593z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Trichlorophenyl Formate: Highly Reactive and Easily Accessible Crystalline CO Surrogate for Palladium-Catalyzed Carbonylation of Aryl/Alkenyl Halides and Triflates</span></div><div class="casAuthors">Ueda, Tsuyoshi; Konishi, Hideyuki; Manabe, Kei</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5370-5373</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The high utility of 2,4,6-trichlorophenyl formate, a highly reactive and easily accessible cryst. CO surrogate, is demonstrated.  The decarbonylation with NEt3 to generate CO proceeded rapidly at rt, thereby allowing external-CO-free Pd-catalyzed carbonylation of aryl/alkenyl halides and triflates.  The high reactivity of the CO surrogate enabled carbonylation at rt and significantly reduced the quantities of formate to near-stoichiometric levels.  The obtained trichlorophenyl esters can be readily converted to a variety of carboxylic acid derivs. in high yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUQt4b7-33nrVg90H21EOLACvtfcHk0ljsosJk3YH1Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVejs7vP&md5=74392d6c4847f88bd452016f5ddd08aa</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fol302593z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol302593z%26sid%3Dliteratum%253Aachs%26aulast%3DUeda%26aufirst%3DT.%26aulast%3DKonishi%26aufirst%3DH.%26aulast%3DManabe%26aufirst%3DK.%26atitle%3DTrichlorophenyl%2520Formate%253A%2520Highly%2520Reactive%2520and%2520Easily%2520Accessible%2520Crystalline%2520CO%2520Surrogate%2520for%2520Palladium-Catalyzed%2520Carbonylation%2520of%2520Aryl%252FAlkenyl%2520Halides%2520and%2520Triflates%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26issue%3D20%26spage%3D5370%26epage%3D5373%26doi%3D10.1021%2Fol302593z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoenberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartoletti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, R. F.</span></span> <span> </span><span class="NLM_article-title">Palladium-catalyzed carboalkoxylation of aryl, benzyl, and vinylic halides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">3318</span>â <span class="NLM_lpage">3326</span>, <span class="refDoi">Â DOI: 10.1021/jo00937a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00937a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaE2MXjvFah" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1974&pages=3318-3326&issue=23&author=A.+Schoenbergauthor=I.+Bartolettiauthor=R.+F.+Heck&title=Palladium-catalyzed+carboalkoxylation+of+aryl%2C+benzyl%2C+and+vinylic+halides&doi=10.1021%2Fjo00937a003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-catalyzed carboalkoxylation of aryl, benzyl, and vinylic halides</span></div><div class="casAuthors">Schoenberg, A.; Bartoletti, I.; Heck, R. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3318-26</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Aryl and vinylic bromides and iodides and benzyl chloride react with CO and an alc. at 100Â° or below and atm. pressure in the presence of a tertiary amine and a catalytic amt. of a palladium-triphenylphosphine complex to form esters.  The reaction is tolerant of a variety of functional groups and shows appreciable stereospecificity at 60-80Â° with cis and trans vinylic halides producing esters with retained configuration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcLSPgXtJ2HrVg90H21EOLACvtfcHk0ljsosJk3YH1Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXjvFah&md5=bdcd89180b1e93776b8e6030f2ed6619</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjo00937a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00937a003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoenberg%26aufirst%3DA.%26aulast%3DBartoletti%26aufirst%3DI.%26aulast%3DHeck%26aufirst%3DR.%2BF.%26atitle%3DPalladium-catalyzed%2520carboalkoxylation%2520of%2520aryl%252C%2520benzyl%252C%2520and%2520vinylic%2520halides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1974%26volume%3D39%26issue%3D23%26spage%3D3318%26epage%3D3326%26doi%3D10.1021%2Fjo00937a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baburajan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elango, K. P.</span></span> <span> </span><span class="NLM_article-title">Co2(CO)8 as a convenient in situ CO source for the direct synthesis of benzamides from aryl halides (Br/I) via aminocarbonylation</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1006</span>â <span class="NLM_lpage">1010</span>, <span class="refDoi">Â DOI: 10.1016/j.tetlet.2013.12.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2Fj.tetlet.2013.12.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVKltA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=1006-1010&issue=5&author=P.+Baburajanauthor=K.+P.+Elango&title=Co2%28CO%298+as+a+convenient+in+situ+CO+source+for+the+direct+synthesis+of+benzamides+from+aryl+halides+%28Br%2FI%29+via+aminocarbonylation&doi=10.1016%2Fj.tetlet.2013.12.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Co2(CO)8 as a convenient in situ CO source for the direct synthesis of benzamides from aryl halides (Br/I) via aminocarbonylation</span></div><div class="casAuthors">Baburajan, Poongavanam; Elango, Kuppanagounder P.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1006-1010</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The direct synthesis of benzamides from aryl halides (Br, I) via aminocarbonylation, using solid Co2(CO)8 as a convenient CO source, was carried out.  The developed method is applicable to a wide variety of 1Â° and cyclic and acyclic 2Â° amines.  Nitro substituted (o, m and p) aryl halides were easily converted to the corresponding benzamides, without the redn. of the nitro group, in high yields using this in situ carbonylation methodol. under microwave irradn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWg2GxYmZpcrVg90H21EOLACvtfcHk0ljdOqI60rGX6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVKltA%253D%253D&md5=85b67932aaa916be3111b3d29e3c3f7c</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2013.12.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2013.12.062%26sid%3Dliteratum%253Aachs%26aulast%3DBaburajan%26aufirst%3DP.%26aulast%3DElango%26aufirst%3DK.%2BP.%26atitle%3DCo2%2528CO%25298%2520as%2520a%2520convenient%2520in%2520situ%2520CO%2520source%2520for%2520the%2520direct%2520synthesis%2520of%2520benzamides%2520from%2520aryl%2520halides%2520%2528Br%252FI%2529%2520via%2520aminocarbonylation%26jtitle%3DTetrahedron%2520Lett.%26date%3D2014%26volume%3D55%26issue%3D5%26spage%3D1006%26epage%3D1010%26doi%3D10.1016%2Fj.tetlet.2013.12.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krasnaya, Z. A.</span></span> <span> </span><span class="NLM_article-title">Interaction of 5-Acetyl(alkoxycarbonyl)-3-alkoxycarbonyl-6-methylpyridin-2(1H)-ones with Primary Aromatic Amines and Hydrazine Hydrate</span>. <i>Chem. Heterocycl. Compd.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1155</span>â <span class="NLM_lpage">1161</span>, <span class="refDoi">Â DOI: 10.1023/B:COHC.0000048288.74767.6f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1023%2FB%3ACOHC.0000048288.74767.6f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWmu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2004&pages=1155-1161&issue=9&author=Z.+A.+Krasnaya&title=Interaction+of+5-Acetyl%28alkoxycarbonyl%29-3-alkoxycarbonyl-6-methylpyridin-2%281H%29-ones+with+Primary+Aromatic+Amines+and+Hydrazine+Hydrate&doi=10.1023%2FB%3ACOHC.0000048288.74767.6f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Reaction of 5-acetyl(alkoxycarbonyl)-3-alkoxycarbonyl-6-methylpyridin-2(1H)-ones with primary aromatic amines and hydrazine hydrate</span></div><div class="casAuthors">Krasnaya, Zh. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1155-1161</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer Science+Business Media, Inc.</span>)
        </div><div class="casAbstract">5-Acetyl(alkoxycarbonyl)-3-alkoxycarbonyl-6-methylpyridin-2(1H)-ones have been obtained from enaminocarbonyl compds.  Their reactions with benzylamine, Î±-phenylethylamine, and hydrazine hydrate have been studied, as a result of which a series of amides, hydrazides, and hydrazones of 2(1H)-pyridones has been synthesized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojTpi6AedYubVg90H21EOLACvtfcHk0ljdOqI60rGX6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWmu7Y%253D&md5=a45bcc4f9e9446164c3757845afa346c</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1023%2FB%3ACOHC.0000048288.74767.6f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253ACOHC.0000048288.74767.6f%26sid%3Dliteratum%253Aachs%26aulast%3DKrasnaya%26aufirst%3DZ.%2BA.%26atitle%3DInteraction%2520of%25205-Acetyl%2528alkoxycarbonyl%2529-3-alkoxycarbonyl-6-methylpyridin-2%25281H%2529-ones%2520with%2520Primary%2520Aromatic%2520Amines%2520and%2520Hydrazine%2520Hydrate%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2004%26volume%3D40%26issue%3D9%26spage%3D1155%26epage%3D1161%26doi%3D10.1023%2FB%3ACOHC.0000048288.74767.6f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malmqvist, M.</span></span> <span> </span><span class="NLM_article-title">Biospecific interaction analysis using biosensor technology</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">186</span>, <span class="refDoi">Â DOI: 10.1038/361186a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1038%2F361186a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=7678451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A280%3ADyaK3s7jtlehtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=1993&pages=186&author=M.+Malmqvist&title=Biospecific+interaction+analysis+using+biosensor+technology&doi=10.1038%2F361186a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Biospecific interaction analysis using biosensor technology</span></div><div class="casAuthors">Malmqvist M</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">6408</span>),
    <span class="NLM_cas:pages">186-7</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Fundamental information that enhances our understanding of biospecific interactions can be obtained using a new analytical system based on biosensor technology.  The functional characteristics of biospecific interaction, such as kinetics, affinity and binding position, are examined by label-free analysis of proteins in free solution binding to an immobilized ligand at a hydrophilic sensor surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSqysTbpfmjRrjnyZkbE8yXfW6udTcc2ebJ6DDB5qj9Bbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3s7jtlehtA%253D%253D&md5=23725c0430dd70b08f8c6d3751425703</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2F361186a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F361186a0%26sid%3Dliteratum%253Aachs%26aulast%3DMalmqvist%26aufirst%3DM.%26atitle%3DBiospecific%2520interaction%2520analysis%2520using%2520biosensor%2520technology%26jtitle%3DNature%26date%3D1993%26volume%3D361%26spage%3D186%26doi%3D10.1038%2F361186a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ames, B.</span></span> <span> </span><span class="NLM_article-title">Identifying environmental chemicals causing mutations and cancer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>204</i></span> (<span class="NLM_issue">4393</span>),  <span class="NLM_fpage">587</span>, <span class="refDoi">Â DOI: 10.1126/science.373122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1126%2Fscience.373122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=373122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=1979&pages=587&issue=4393&author=B.+Ames&title=Identifying+environmental+chemicals+causing+mutations+and+cancer&doi=10.1126%2Fscience.373122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying environmental chemicals causing mutations and cancer</span></div><div class="casAuthors">Ames, Bruce N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">4393</span>),
    <span class="NLM_cas:pages">587-93</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 57 refs. on environmental chem. induced carcinogenesis and mutagenesis and their relation to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2AzWl_6PJ4bVg90H21EOLACvtfcHk0ljdOqI60rGX6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D&md5=9ac98bad1482e9cda849723a71f32a9d</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1126%2Fscience.373122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.373122%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DB.%26atitle%3DIdentifying%2520environmental%2520chemicals%2520causing%2520mutations%2520and%2520cancer%26jtitle%3DScience%26date%3D1979%26volume%3D204%26issue%3D4393%26spage%3D587%26doi%3D10.1126%2Fscience.373122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clive, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machesko, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernheim, N. J.</span></span> <span> </span><span class="NLM_article-title">A mutational assay system using the thymidine kinase locus in mouse lymphoma cells</span>. <i>Mutat. Res., Fundam. Mol. Mech. Mutagen.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">77</span>â <span class="NLM_lpage">87</span>, <span class="refDoi">Â DOI: 10.1016/0027-5107(72)90066-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1016%2F0027-5107%2872%2990066-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=5053587" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1972&pages=77-87&issue=1&author=D.+Cliveauthor=W.+G.+Flammauthor=M.+R.+Macheskoauthor=N.+J.+Bernheim&title=A+mutational+assay+system+using+the+thymidine+kinase+locus+in+mouse+lymphoma+cells&doi=10.1016%2F0027-5107%2872%2990066-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2F0027-5107%2872%2990066-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0027-5107%252872%252990066-8%26sid%3Dliteratum%253Aachs%26aulast%3DClive%26aufirst%3DD.%26aulast%3DFlamm%26aufirst%3DW.%2BG.%26aulast%3DMachesko%26aufirst%3DM.%2BR.%26aulast%3DBernheim%26aufirst%3DN.%2BJ.%26atitle%3DA%2520mutational%2520assay%2520system%2520using%2520the%2520thymidine%2520kinase%2520locus%2520in%2520mouse%2520lymphoma%2520cells%26jtitle%3DMutat.%2520Res.%252C%2520Fundam.%2520Mol.%2520Mech.%2520Mutagen.%26date%3D1972%26volume%3D16%26issue%3D1%26spage%3D77%26epage%3D87%26doi%3D10.1016%2F0027-5107%2872%2990066-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M.</span></span> <i>Mouse Lymphoma Assay</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_fpage">323</span>â <span class="NLM_lpage">342</span>, <span class="refDoi">Â DOI: 10.1007/978-1-4939-1068-7_19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1007%2F978-1-4939-1068-7_19" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Chen%2C+T.%3B+Guo%2C+X.%3B+Moore%2C+M.+Mouse+Lymphoma+Assay+2014%2C+323%E2%80%93342%2C+10.1007%2F978-1-4939-1068-7_19"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-1068-7_19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-1068-7_19%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DMoore%26aufirst%3DM.%26jtitle%3DMouse%2520Lymphoma%2520Assay%26date%3D2014%26spage%3D323%26epage%3D342%26doi%3D10.1007%2F978-1-4939-1068-7_19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stieger, B.</span></span> <span> </span><span class="NLM_article-title">Role of the bile salt export pump, BSEP, in acquired forms of cholestasis</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">437</span>â <span class="NLM_lpage">445</span>, <span class="refDoi">Â DOI: 10.3109/03602530903492004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.3109%2F03602530903492004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=20028269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotlSmt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2010&pages=437-445&issue=3&author=B.+Stieger&title=Role+of+the+bile+salt+export+pump%2C+BSEP%2C+in+acquired+forms+of+cholestasis&doi=10.3109%2F03602530903492004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the bile salt export pump, BSEP, in acquired forms of cholestasis</span></div><div class="casAuthors">Stieger, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-445</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Generation of bile is a key function of the liver.  Its impairment leads to accumulation of cytotoxic bile salts in hepatocytes and, consequently, to liver disease.  The bile salt export pump, BSEP, is critically involved in the secretion of bile salts into bile.  Its function can be disturbed or abolished by inherited mutations.  This will lead to progressive intrahepatic cholestasis and severe liver disease.  In addn. to mutations, BSEP can be inhibited by acquired factors, such as xenobiotics or drugs, aberrant bile salt metabolites, or pregnancy.  This inhibition will lead to acquired cholestasis.  Some drugs are now known to be competitive inhibitors of Bsep.  In addn., a polymorphism in the gene coding for BSEP has been identified as a potential susceptibility factor for acquired cholestasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYni5fpYpR8bVg90H21EOLACvtfcHk0liayDCzNEUxlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotlSmt7w%253D&md5=4ea89d1fbe6a1da4933b3f72a6c212f9</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.3109%2F03602530903492004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602530903492004%26sid%3Dliteratum%253Aachs%26aulast%3DStieger%26aufirst%3DB.%26atitle%3DRole%2520of%2520the%2520bile%2520salt%2520export%2520pump%252C%2520BSEP%252C%2520in%2520acquired%2520forms%2520of%2520cholestasis%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2010%26volume%3D42%26issue%3D3%26spage%3D437%26epage%3D445%26doi%3D10.3109%2F03602530903492004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span> <span> </span><span class="NLM_article-title">Metabolism and Toxicity of Drugs. Two Decades of Progress in Industrial Drug Metabolism</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">129</span>â <span class="NLM_lpage">137</span>, <span class="refDoi">Â DOI: 10.1021/tx7002273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx7002273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=129-137&issue=1&author=T.+A.+Baillie&title=Metabolism+and+Toxicity+of+Drugs.+Two+Decades+of+Progress+in+Industrial+Drug+Metabolism&doi=10.1021%2Ftx7002273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism and Toxicity of Drugs. Two Decades of Progress in Industrial Drug Metabolism</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-137</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The science of drug metab. and pharmacokinetics (DMPK) has developed significantly over the past 20 years, and its functional role in today's pharmaceutical industry has matured to the point where DMPK has become an indispensable discipline in support of drug discovery and development.  While contributions to the lead optimization phase of discovery efforts have been particularly noteworthy in helping to select only the best drug candidates for entry into development, it should be recognized that the scope of DMPK spans the continuum of discovery through clin. evaluation and even into the post-marketing phase; as such, the breadth of DMPK's involvement is almost unique in contemporary pharmaceutical research.  This perspective summarizes notable advances in the field, many of which have been made possible by technol. developments in areas such as mol. biol., genetics, and bioanal. chem., and highlights the crit. nature of key partnerships between Drug Metab., Medicinal Chem., and Safety Assessment groups in attempting to advance drug candidates with a low potential for causing adverse events in humans.  Finally, some speculative predictions are made of the future role of DMPK in pharmaceutical research, where current advances in our mechanistic understanding of the mol. processes that control the absorption, disposition, metab., elimination, and toxicity of drugs and their biotransformation products will combine to further enhance the impact of DMPK in drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUrTeLhTDYbVg90H21EOLACvtfcHk0liayDCzNEUxlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF&md5=e8f659251bf82cb0b1602855e725c390</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Ftx7002273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx7002273%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DMetabolism%2520and%2520Toxicity%2520of%2520Drugs.%2520Two%2520Decades%2520of%2520Progress%2520in%2520Industrial%2520Drug%2520Metabolism%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26issue%3D1%26spage%3D129%26epage%3D137%26doi%3D10.1021%2Ftx7002273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masubuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horie, T.</span></span> <span> </span><span class="NLM_article-title">Toxicological Significance of Mechanism-Based Inactivation of Cytochrome P450 Enzymes by Drugs</span>. <i>Crit. Rev. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">389</span>â <span class="NLM_lpage">412</span>, <span class="refDoi">Â DOI: 10.1080/10408440701215233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1080%2F10408440701215233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=17612953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1altr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=389-412&issue=5&author=Y.+Masubuchiauthor=T.+Horie&title=Toxicological+Significance+of+Mechanism-Based+Inactivation+of+Cytochrome+P450+Enzymes+by+Drugs&doi=10.1080%2F10408440701215233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicological significance of mechanism-based inactivation of cytochrome P450 enzymes by drugs</span></div><div class="casAuthors">Masubuchi, Yasuhiro; Horie, Toshiharu</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">389-412</span>CODEN:
                <span class="NLM_cas:coden">CRTXB2</span>;
        ISSN:<span class="NLM_cas:issn">1040-8444</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Cytochrome P 450 (P 450) enzymes oxidize xenobiotics into chem. reactive metabolites or intermediates as well as into stable metabolites.  If the reactivity of the product is very high, it binds to a catalytic site or sites of the enzyme itself and inactivates it.  This phenomenon is referred to as mechanism-based inactivation.  Many clin. important drugs are mechanism-based inactivators that include macrolide antibiotics, calcium channel blockers, and selective serotonin uptake inhibitors, but are not always structurally and pharmacol. related.  The inactivation of P450s during drug therapy results in serious drug interactions, since irreversibility of the binding allows enzyme inhibition to be prolonged after elimination of the causal drug.  The inhibition of the metab. of drugs with narrow therapeutic indexes, such as terfenadine and astemizole, leads to toxicities.  On the other hand, the fate of P450s after the inactivation and the toxicol. consequences remains to be elucidated, while it has been suggested that P450s modified and degraded are involved in some forms of tissue toxicity.  Porphyrinogenic drugs, such as griseofulvin, cause mechanism-based heme inactivation, leading to formation of ferrochelatase-inhibitory N-alkylated protoporphyrins and resulting in porphyria.  Involvement of P 450-derived free heme in halothane-induced hepatotoxicity and catalytic iron in cisplatin-induced nephrotoxicity has also been suggested.  Autoantibodies against P450s have been found in hepatitis following administration of tienilic acid and dihydralazine.  Tienilic acid is activated by and covalently bound to CYP2C9, and the neoantigens thus formed activate immune systems, resulting in the formation of an autoantibody directed against CYP2C9, named anti-liver/kidney microsomal autoantibody type 2, whereas the pathol. role of the autoantibodies in drug-induced hepatitis remains largely unknown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmbxxZLDTkDLVg90H21EOLACvtfcHk0lgt9M-xJ6KCLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1altr4%253D&md5=bd451ac2e6f723ea0a537b27187ef7ba</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1080%2F10408440701215233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408440701215233%26sid%3Dliteratum%253Aachs%26aulast%3DMasubuchi%26aufirst%3DY.%26aulast%3DHorie%26aufirst%3DT.%26atitle%3DToxicological%2520Significance%2520of%2520Mechanism-Based%2520Inactivation%2520of%2520Cytochrome%2520P450%2520Enzymes%2520by%2520Drugs%26jtitle%3DCrit.%2520Rev.%2520Toxicol.%26date%3D2007%26volume%3D37%26issue%3D5%26spage%3D389%26epage%3D412%26doi%3D10.1080%2F10408440701215233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, M.</span></span> <span> </span><span class="NLM_article-title">Drug-Mediated Inactivation of Cytochrome P450</span>. <i>Clin. Exp. Pharmacol. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">465</span>â <span class="NLM_lpage">470</span>, <span class="refDoi">Â DOI: 10.1111/j.1440-1681.1997.tb01228.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1111%2Fj.1440-1681.1997.tb01228.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=9248661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADyaK2sXkvVahtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1997&pages=465-470&issue=7&author=M.+Murray&title=Drug-Mediated+Inactivation+of+Cytochrome+P450&doi=10.1111%2Fj.1440-1681.1997.tb01228.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-mediated inactivation of cytochrome P450</span></div><div class="casAuthors">Murray, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Experimental Pharmacology and Physiology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-470</span>CODEN:
                <span class="NLM_cas:coden">CEXPB9</span>;
        ISSN:<span class="NLM_cas:issn">0305-1870</span>.
    
            (<span class="NLM_cas:orgname">Blackwell</span>)
        </div><div class="casAbstract">A review with 62 refs.  Multiple forms of cytochrome P 450 (CYP) catalyze the oxidn. of chems. of endogenous and exogenous origin, including drugs, carcinogens, steroids and eicosanoids.  However, this unusual low substrate specificity also makes CYP susceptible to inhibition by a wide range of drugs, leading to pharmacokinetic interactions of potential clin. significance.  Some drugs are converted by CYP to reactive metabolites that bind covalently to sites within the active center of the same CYP.  Such mechanism-based inhibition leads to CYP inactivation or complexation.  These processes give rise to longterm effects on drug pharmacokinetics, as the inactivated or complexed CYP must be replaced by newly synthesized CYP protein.  Drugs that inactivate CYP generally possess recognizable functional groups that are oxidized to reactive products.  Thus, drugs with side chains contg. unsatd. carbon-carbon bonds and furan ring systems are assocd. with CYP inactivation.  Nitrogen-contg. systems may also inactivate CYP.  Metabolites formed from drugs contg. alkylamino and methylenedioxy functionalities can trap CYP as inert complexes without eliciting inactivation.  However, the functional effects of inactivation and complexation on drug pharmacokinetics are indistinguishable.  Drugs that elicit CYP complexation include the first generation macrolide antibiotics, but newer analogs appear much safer.  Some antidepressants, antiepileptics and tuberculostatic agents have been assocd. with CYP complexation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuVcGKdWFUrVg90H21EOLACvtfcHk0lgt9M-xJ6KCLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvVahtbs%253D&md5=8a991524e90fc66ec7475a2f19259804</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1111%2Fj.1440-1681.1997.tb01228.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1440-1681.1997.tb01228.x%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DM.%26atitle%3DDrug-Mediated%2520Inactivation%2520of%2520Cytochrome%2520P450%26jtitle%3DClin.%2520Exp.%2520Pharmacol.%2520Physiol.%26date%3D1997%26volume%3D24%26issue%3D7%26spage%3D465%26epage%3D470%26doi%3D10.1111%2Fj.1440-1681.1997.tb01228.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh-Bakht, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâDonnell, J. P.</span></span> <span> </span><span class="NLM_article-title">Biotransformation Reactions of Five-Membered Aromatic Heterocyclic Rings</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">269</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1021/tx015574b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx015574b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2002&pages=269-299&issue=3&author=D.+K.+Dalvieauthor=A.+S.+Kalgutkarauthor=S.+C.+Khojasteh-Bakhtauthor=R.+S.+Obachauthor=J.+P.+O%E2%80%99Donnell&title=Biotransformation+Reactions+of+Five-Membered+Aromatic+Heterocyclic+Rings&doi=10.1021%2Ftx015574b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Biotransformation Reactions of Five-Membered Aromatic Heterocyclic Rings</span></div><div class="casAuthors">Dalvie, Deepak K.; Kalgutkar, Amit S.; Khojasteh-Bakht, S. Cyrus; Obach, R. Scott; O'Donnell, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-299</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 278 refs. is given on the biotransformation pathways of most commonly used 5-membered arom. heterocyclic rings.  The effect of physicochem. properties such as the electronic effects of heteroatoms, aromaticity, and acidity/basicity (pKa) of the rings are correlated on their biotransformation.  The generation of reactive metabolites following oxidn. or redn. of these heterocycles and the subsequent toxicol. consequences are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNaEz19aehbVg90H21EOLACvtfcHk0lgt9M-xJ6KCLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D&md5=e8e2dfd12772f9ff2289b173bdefa590</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Ftx015574b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx015574b%26sid%3Dliteratum%253Aachs%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DKhojasteh-Bakht%26aufirst%3DS.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26atitle%3DBiotransformation%2520Reactions%2520of%2520Five-Membered%2520Aromatic%2520Heterocyclic%2520Rings%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2002%26volume%3D15%26issue%3D3%26spage%3D269%26epage%3D299%26doi%3D10.1021%2Ftx015574b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshmane, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremlev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawaya, B. E.</span></span> <span> </span><span class="NLM_article-title">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span>. <i>J. Interferon Cytokine Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">313</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1089/jir.2008.0027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1089%2Fjir.2008.0027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=19441883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=313-326&issue=6&author=S.+L.+Deshmaneauthor=S.+Kremlevauthor=S.+Aminiauthor=B.+E.+Sawaya&title=Monocyte+Chemoattractant+Protein-1+%28MCP-1%29%3A+An+Overview&doi=10.1089%2Fjir.2008.0027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span></div><div class="casAuthors">Deshmane, Satish L.; Kremlev, Sergey; Amini, Shohreh; Sawaya, Bassel E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Interferon & Cytokine Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">313-326</span>CODEN:
                <span class="NLM_cas:coden">JICRFJ</span>;
        ISSN:<span class="NLM_cas:issn">1079-9907</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Chemokines constitute a family of chemoattractant cytokines and are subdivided into four families on the basis of the no. and spacing of the conserved cysteine residues in the N-terminus of the protein.  Chemokines play a major role in selectively recruiting monocytes, neutrophils, and lymphocytes, as well as in inducing chemotaxis through the activation of G-protein-coupled receptors.  Monocyte chemoattractant protein-1 (MCP-1/CCL2) is one of the key chemokines that regulate migration and infiltration of monocytes/macrophages.  Both CCL2 and its receptor CCR2 have been demonstrated to be induced and involved in various diseases.  Migration of monocytes from the blood stream across the vascular endothelium is required for routine immunol. surveillance of tissues, as well as in response to inflammation.  This review will discuss these biol. processes and the structure and function of CCL2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSmnT35tYDZLVg90H21EOLACvtfcHk0lgt9M-xJ6KCLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D&md5=d58ea0c1925c2eff93631916f93edad3</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1089%2Fjir.2008.0027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fjir.2008.0027%26sid%3Dliteratum%253Aachs%26aulast%3DDeshmane%26aufirst%3DS.%2BL.%26aulast%3DKremlev%26aufirst%3DS.%26aulast%3DAmini%26aufirst%3DS.%26aulast%3DSawaya%26aufirst%3DB.%2BE.%26atitle%3DMonocyte%2520Chemoattractant%2520Protein-1%2520%2528MCP-1%2529%253A%2520An%2520Overview%26jtitle%3DJ.%2520Interferon%2520Cytokine%2520Res.%26date%3D2009%26volume%3D29%26issue%3D6%26spage%3D313%26epage%3D326%26doi%3D10.1089%2Fjir.2008.0027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. W.</span>; <span class="NLM_string-name">Fotouhi, N.</span>; <span class="NLM_string-name">Gillespie, P.</span>; <span class="NLM_string-name">Goodnow, R. A.,  Jr.</span>; <span class="NLM_string-name">Guertin, K. R.</span>; <span class="NLM_string-name">Haynes, N.-E.</span>; <span class="NLM_string-name">Myers, M. P.</span>; <span class="NLM_string-name">Pietranico-Cole, S. L.</span>; <span class="NLM_string-name">Qi, L.</span>; <span class="NLM_string-name">Rossman, P. L.</span>; <span class="NLM_string-name">Scott, N. R.</span>; <span class="NLM_string-name">Thakkar, K. C.</span>; <span class="NLM_string-name">Tilley, J. W.</span>; <span class="NLM_string-name">Zhang, Q.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrazole-4-carboxamide Derivatives as 11-Beta-Hydroxysteroid Dehydrogenase Form I (11-beta-HSD1) Inhibitors</span>. World Patent <span class="NLM_patent">WO2007107470A2</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=K.+W.+Anderson&author=N.+Fotouhi&author=P.+Gillespie&author=R.+A.+Goodnow&author=K.+R.+Guertin&author=N.-E.+Haynes&author=M.+P.+Myers&author=S.+L.+Pietranico-Cole&author=L.+Qi&author=P.+L.+Rossman&author=N.+R.+Scott&author=K.+C.+Thakkar&author=J.+W.+Tilley&author=Q.+Zhang&title=Preparation+of+Pyrazole-4-carboxamide+Derivatives+as+11-Beta-Hydroxysteroid+Dehydrogenase+Form+I+%2811-beta-HSD1%29+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DK.%2BW.%26atitle%3DPreparation%2520of%2520Pyrazole-4-carboxamide%2520Derivatives%2520as%252011-Beta-Hydroxysteroid%2520Dehydrogenase%2520Form%2520I%2520%252811-beta-HSD1%2529%2520Inhibitors%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camurri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaramella, A.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3716</span>â <span class="NLM_lpage">3722</span>, <span class="refDoi">Â DOI: 10.1021/ac001388j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac001388j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=3716-3722&issue=15&author=G.+Camurriauthor=A.+Zaramella&title=High-Throughput+Liquid+Chromatography%2FMass+Spectrometry+Method+for+the+Determination+of+the+Chromatographic+Hydrophobicity+Index&doi=10.1021%2Fac001388j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span></div><div class="casAuthors">Camurri, Giulio; Zaramella, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3716-3722</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast gradient reversed-phase liq. chromatog. (LC) method, using an acetonitrile gradient was developed to det. the chromatog. hydrophobicity index (CHI), as reported by Valco et al. (Anal. Chem. 1997, 69, 2022-2029).The anal. method provides retention times, based on UV detection at two different wavelengths, which then are converted into CHI values after calibration with a set of test compds.  The CHI of each compd. is measured at three different pH values, 2.0, 7.4, and 10.5; so using an 8-min gradient at each pH value one compd. can be analyzed in â¼24 min.  The aim of this work is to improve the throughput of the CHI screening using a LC/MS approach, so the application of the LC/MS technique is an extension of the LC/UV approach previously reported by Valco et al.  This approach allows contemporary injection of N compds. into the LC/MS system, the retention time of each compd. can be then extd. from the selected ion recording chromatograms.  The throughput of the existing screening method could be increased by N times, where N is the no. of compds. injected, so only three runs are needed to det. the CHI at three different pH values for a set of N compds.  The highest value of N depends on the total no. of channels that can be monitored simultaneously; 32 channels were used.  This LC/MS method was tested for a no. of com. products analyzed as mixts., and data obtained were compared with those coming from the classical LC/UV approach.  In the same way, the method was tested for a no. of compds. assocd. with two GlaxoWellcome projects in the antibacterial area.  The LC/MS method can be successfully applied for analyzing compds. in mixts. and for compds. with poor UV absorption, which cannot be analyzed with the std. LC/UV method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpqYuIDpy5rVg90H21EOLACvtfcHk0lgzf5tVw14jNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D&md5=8773c87e08ba7ddce3c2d679360a6220</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fac001388j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac001388j%26sid%3Dliteratum%253Aachs%26aulast%3DCamurri%26aufirst%3DG.%26aulast%3DZaramella%26aufirst%3DA.%26atitle%3DHigh-Throughput%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520the%2520Determination%2520of%2520the%2520Chromatographic%2520Hydrophobicity%2520Index%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26issue%3D15%26spage%3D3716%26epage%3D3722%26doi%3D10.1021%2Fac001388j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. P.</span></span> <span> </span><span class="NLM_article-title">Fast Gradient HPLC Method to Determine Compounds Binding to Human Serum Albumin. Relationships with Octanol/Water and Immobilized Artificial Membrane Lipophilicity</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2236</span>â <span class="NLM_lpage">2248</span>, <span class="refDoi">Â DOI: 10.1002/jps.10494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1002%2Fjps.10494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=14603509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=2236-2248&issue=11&author=K.+Valkoauthor=S.+Nunhuckauthor=C.+Bevanauthor=M.+H.+Abrahamauthor=D.+P.+Reynolds&title=Fast+Gradient+HPLC+Method+to+Determine+Compounds+Binding+to+Human+Serum+Albumin.+Relationships+with+Octanol%2FWater+and+Immobilized+Artificial+Membrane+Lipophilicity&doi=10.1002%2Fjps.10494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span></div><div class="casAuthors">Valko, Klara; Nunhuck, Shenaz; Bevan, Chris; Abraham, Michael H.; Reynolds, Derek P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2236-2248</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A fast gradient HPLC method (cycle time 15 min) has been developed to det. Human Serum Albumin (HSA) binding of discovery compds. using chem. bonded protein stationary phases.  The HSA binding values were derived from the gradient retention times that were converted to the logarithm of the equil. consts. (logK HSA) using data from a calibration set of mols.  The method has been validated using literature plasma protein binding data of 68 known drug mols.  The method is fully automated, and has been used for lead optimization in more than 20 company projects.  The HSA binding data obtained for more than 4000 compds. were suitable to set up global and project specific quant. structure binding relationships that helped compd. design in early drug discovery.  The obtained HSA binding of known drug mols. were compared to the Immobilizd Artificial Membrane binding data (CHI IAM) obtained by our previously described HPLC-based method.  The solvation equation approach has been used to characterize the normal binding ability of HSA, and this relationship shows that compd. lipophilicity is a significant factor.  It was found that the selectivity of the "baseline" lipophilicity governing HSA binding, membrane interaction, and octanol/water partition are very similar.  However, the effect of the presence of pos. or neg. charges have very different effects.  It was found that neg. charged compds. bind more strongly to HSA than it would be expected from the lipophilicity of the ionized species at pH 7.4.  Several compds. showed stronger HSA binding than can be expected from their lipophilicity alone, and comparison between predicted and exptl. binding affinity allows the identification of compds. that have good complementarities with any of the known binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1LUBLdv-t7Vg90H21EOLACvtfcHk0lgzf5tVw14jNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D&md5=14eb1e0f0ad616846a2b5d1428133fa9</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fjps.10494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10494%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26atitle%3DFast%2520Gradient%2520HPLC%2520Method%2520to%2520Determine%2520Compounds%2520Binding%2520to%2520Human%2520Serum%2520Albumin.%2520Relationships%2520with%2520Octanol%252FWater%2520and%2520Immobilized%2520Artificial%2520Membrane%2520Lipophilicity%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26issue%3D11%26spage%3D2236%26epage%3D2248%26doi%3D10.1002%2Fjps.10494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for the ATAD2 Bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11382</span>â <span class="NLM_lpage">11386</span>, <span class="refDoi">Â DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&issue=38&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+Chemical+Probe+for+the+ATAD2+Bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lgzf5tVw14jNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520the%2520ATAD2%2520Bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26issue%3D38%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">6151</span>â <span class="NLM_lpage">6178</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&issue=15&author=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-Based+Optimization+of+Naphthyridones+into+Potent+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lhyixvBSJ71XQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Naphthyridones%2520into%2520Potent%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D15%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB0','PDB','6ZB0'); return false;">PDB: 6ZB0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SWP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SWP','PDB','6SWP'); return false;">PDB: 6SWP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1','PDB','6ZB1'); return false;">PDB: 6ZB1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3','PDB','6ZB3'); return false;">PDB: 6ZB3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB2','PDB','6ZB2'); return false;">PDB: 6ZB2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i183"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00796">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_53403"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00796?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00796</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">DiscoverX BROMOscan Bromodomain Profiling of <b>20</b>, cross screening data of liability target panel for <b>20</b> and <b>56</b>, sequence alignment and differences of BET proteins, X-ray crystallographic data and selected LCMS and NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_002.csv">CSV</a>)</p></li><li><p class="inline">Structure factors for datablock <b>20</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_003.pdf">PDF</a>)</p></li><li><p class="inline">Crystallographic information for <b>20</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_004.cif">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_001.pdf">jm0c00796_si_001.pdf (2.47 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_002.csv">jm0c00796_si_002.csv (10.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_003.pdf">jm0c00796_si_003.pdf (96.17 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00796/suppl_file/jm0c00796_si_004.cif">jm0c00796_si_004.cif (26.75 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB0">6ZB0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB2">6ZB2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>. Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00796%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00796" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799dfac49013d06","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
