
<html lang="en"     class="pb-page"  data-request-id="ed53165b-fb55-4fa9-990b-e8abfb7eb530"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c02207;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Targeting the FtsZ Allosteric Binding Site with a Novel Fluorescence Polarization Screen, Cytological and Structural Approaches for Antibacterial Discovery" /></meta><meta name="dc.Creator" content="Sonia  Huecas" /></meta><meta name="dc.Creator" content="Lidia  Araújo-Bazán" /></meta><meta name="dc.Creator" content="Federico M.  Ruiz" /></meta><meta name="dc.Creator" content="Laura B.  Ruiz-Ávila" /></meta><meta name="dc.Creator" content="R. Fernando  Martínez" /></meta><meta name="dc.Creator" content="Andrea  Escobar-Peña" /></meta><meta name="dc.Creator" content="Marta  Artola" /></meta><meta name="dc.Creator" content="Henar  Vázquez-Villa" /></meta><meta name="dc.Creator" content="Mar  Martín-Fontecha" /></meta><meta name="dc.Creator" content="Carlos  Fernández-Tornero" /></meta><meta name="dc.Creator" content="María L.  López-Rodríguez" /></meta><meta name="dc.Creator" content="José M.  Andreu" /></meta><meta name="dc.Description" content="Bacterial resistance to antibiotics makes previously manageable infections again disabling and lethal, highlighting the need for new antibacterial strategies. In this regard, inhibition of the bact..." /></meta><meta name="Description" content="Bacterial resistance to antibiotics makes previously manageable infections again disabling and lethal, highlighting the need for new antibacterial strategies. In this regard, inhibition of the bact..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 28, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02207" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02207" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02207" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02207" /></link>
        
    
    

<title>Targeting the FtsZ Allosteric Binding Site with a Novel Fluorescence Polarization Screen, Cytological and Structural Approaches for Antibacterial Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02207" /></meta><meta property="og:title" content="Targeting the FtsZ Allosteric Binding Site with a Novel Fluorescence Polarization Screen, Cytological and Structural Approaches for Antibacterial Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0012.jpeg" /></meta><meta property="og:description" content="Bacterial resistance to antibiotics makes previously manageable infections again disabling and lethal, highlighting the need for new antibacterial strategies. In this regard, inhibition of the bacterial division process by targeting key protein FtsZ has been recognized as an attractive approach for discovering new antibiotics. Binding of small molecules to the cleft between the N-terminal guanosine triphosphate (GTP)-binding and the C-terminal subdomains allosterically impairs the FtsZ function, eventually inhibiting bacterial division. Nonetheless, the lack of appropriate chemical tools to develop a binding screen against this site has hampered the discovery of FtsZ antibacterial inhibitors. Herein, we describe the first competitive binding assay to identify FtsZ allosteric ligands interacting with the interdomain cleft, based on the use of specific high-affinity fluorescent probes. This novel assay, together with phenotypic profiling and X-ray crystallographic insights, enables the identification and characterization of FtsZ inhibitors of bacterial division aiming at the discovery of more effective antibacterials." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02207"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02207">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02207&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02207&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02207&amp;href=/doi/10.1021/acs.jmedchem.0c02207" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 5730-5745</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02184" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02217" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting the FtsZ Allosteric Binding Site with a Novel Fluorescence Polarization Screen, Cytological and Structural Approaches for Antibacterial Discovery</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sonia Huecas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sonia Huecas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sonia++Huecas">Sonia Huecas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lidia Araújo-Bazán</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lidia Araújo-Bazán</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lidia++Ara%C3%BAjo-Baz%C3%A1n">Lidia Araújo-Bazán</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Federico M. Ruiz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Federico M. Ruiz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Federico+M.++Ruiz">Federico M. Ruiz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura B. Ruiz-Ávila</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura B. Ruiz-Ávila</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura+B.++Ruiz-%C3%81vila">Laura B. Ruiz-Ávila</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">R. Fernando Martínez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">R. Fernando Martínez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=R.+Fernando++Mart%C3%ADnez">R. Fernando Martínez</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3278-6074" title="Orcid link">http://orcid.org/0000-0002-3278-6074</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrea Escobar-Peña</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Escobar-Peña</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Escobar-Pe%C3%B1a">Andrea Escobar-Peña</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marta Artola</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marta Artola</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marta++Artola">Marta Artola</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3051-3902" title="Orcid link">http://orcid.org/0000-0002-3051-3902</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Henar Vázquez-Villa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Henar Vázquez-Villa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Henar++V%C3%A1zquez-Villa">Henar Vázquez-Villa</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7911-3160" title="Orcid link">http://orcid.org/0000-0001-7911-3160</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mar Martín-Fontecha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mar Martín-Fontecha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mar++Mart%C3%ADn-Fontecha">Mar Martín-Fontecha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos Fernández-Tornero</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos Fernández-Tornero</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d4b7b2a0bba6bab1a6bb94b7bdb6fab7a7bdb7fab1a7"><span class="__cf_email__" data-cfemail="bad9dcced5c8d4dfc8d5fad9d3d894d9c9d3d994dfc9">[email protected]</span></a>. Tel: 34-91-8373112.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos++Fern%C3%A1ndez-Tornero">Carlos Fernández-Tornero</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">María L. López-Rodríguez</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">María L. López-Rodríguez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2a47465f5046586a5f4947044f59"><span class="__cf_email__" data-cfemail="c5a8a9b0bfa9b785b0a6a8eba0b6">[email protected]</span></a>. Tel: 34-91-3944239.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mar%C3%ADa+L.++L%C3%B3pez-Rodr%C3%ADguez">María L. López-Rodríguez</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8607-1085" title="Orcid link">http://orcid.org/0000-0001-8607-1085</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">José M. Andreu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">José M. Andreu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centro de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#107a3e7d3e717e74627565507379723e736379733e7563"><span class="__cf_email__" data-cfemail="244e0a490a454a4056415164474d460a47574d470a4157">[email protected]</span></a>. Tel: 34-91-8373112.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jos%C3%A9+M.++Andreu">José M. Andreu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8064-6933" title="Orcid link">http://orcid.org/0000-0001-8064-6933</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02207&amp;href=/doi/10.1021%2Facs.jmedchem.0c02207" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 5730–5745</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 28, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 December 2020</li><li><span class="item_label"><b>Published</b> online</span>28 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02207" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02207</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5730%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSonia%2BHuecas%252C%2BLidia%2BAra%25C3%25BAjo-Baz%25C3%25A1n%252C%2BFederico%2BM.%2BRuiz%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c02207%26title%3DTargeting%2Bthe%2BFtsZ%2BAllosteric%2BBinding%2BSite%2Bwith%2Ba%2BNovel%2BFluorescence%2BPolarization%2BScreen%252C%2BCytological%2Band%2BStructural%2BApproaches%2Bfor%2BAntibacterial%2BDiscovery%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5745%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02207"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">639</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02207" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting the FtsZ Allosteric Binding Site with a Novel Fluorescence Polarization Screen, Cytological and Structural Approaches for Antibacterial Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sonia&quot;,&quot;last_name&quot;:&quot;Huecas&quot;},{&quot;first_name&quot;:&quot;Lidia&quot;,&quot;last_name&quot;:&quot;Araújo-Bazán&quot;},{&quot;first_name&quot;:&quot;Federico&quot;,&quot;last_name&quot;:&quot;M. Ruiz&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;B. Ruiz-Ávila&quot;},{&quot;first_name&quot;:&quot;R.&quot;,&quot;last_name&quot;:&quot;Fernando Martínez&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Escobar-Peña&quot;},{&quot;first_name&quot;:&quot;Marta&quot;,&quot;last_name&quot;:&quot;Artola&quot;},{&quot;first_name&quot;:&quot;Henar&quot;,&quot;last_name&quot;:&quot;Vázquez-Villa&quot;},{&quot;first_name&quot;:&quot;Mar&quot;,&quot;last_name&quot;:&quot;Martín-Fontecha&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;Fernández-Tornero&quot;},{&quot;first_name&quot;:&quot;María&quot;,&quot;last_name&quot;:&quot;L. López-Rodríguez&quot;},{&quot;first_name&quot;:&quot;José&quot;,&quot;last_name&quot;:&quot;M. Andreu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5730-5745&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02207&quot;},&quot;abstract&quot;:&quot;Bacterial resistance to antibiotics makes previously manageable infections again disabling and lethal, highlighting the need for new antibacterial strategies. In this regard, inhibition of the bacterial division process by targeting key protein FtsZ has been recognized as an attractive approach for discovering new antibiotics. Binding of small molecules to the cleft between the N-terminal guanosine triphosphate (GTP)-binding and the C-terminal subdomains allosterically impairs the FtsZ function, eventually inhibiting bacterial division. Nonetheless, the lack of appropriate chemical tools to develop a binding screen against this site has hampered the discovery of FtsZ antibacterial inhibitors. Herein, we describe the first competitive binding assay to identify FtsZ allosteric ligands interacting with the interdomain cleft, based on the use of specific high-affinity fluorescent probes. This novel assay, together with phenotypic profiling and X-ray crystallographic insights, enables the identification and charac&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02207&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02207" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02207&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02207" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02207&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02207" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02207&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02207&amp;href=/doi/10.1021/acs.jmedchem.0c02207" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02207" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02207" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02207%26sid%3Dliteratum%253Aachs%26pmid%3D33908781%26genre%3Darticle%26aulast%3DHuecas%26date%3D2021%26atitle%3DTargeting%2Bthe%2BFtsZ%2BAllosteric%2BBinding%2BSite%2Bwith%2Ba%2BNovel%2BFluorescence%2BPolarization%2BScreen%252C%2BCytological%2Band%2BStructural%2BApproaches%2Bfor%2BAntibacterial%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D5730%26epage%3D5745%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=290998" title="Magnetic properties">Magnetic properties</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Bacterial resistance to antibiotics makes previously manageable infections again disabling and lethal, highlighting the need for new antibacterial strategies. In this regard, inhibition of the bacterial division process by targeting key protein FtsZ has been recognized as an attractive approach for discovering new antibiotics. Binding of small molecules to the cleft between the N-terminal guanosine triphosphate (GTP)-binding and the C-terminal subdomains allosterically impairs the FtsZ function, eventually inhibiting bacterial division. Nonetheless, the lack of appropriate chemical tools to develop a binding screen against this site has hampered the discovery of FtsZ antibacterial inhibitors. Herein, we describe the first competitive binding assay to identify FtsZ allosteric ligands interacting with the interdomain cleft, based on the use of specific high-affinity fluorescent probes. This novel assay, together with phenotypic profiling and X-ray crystallographic insights, enables the identification and characterization of FtsZ inhibitors of bacterial division aiming at the discovery of more effective antibacterials.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48136" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48136" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">New antibiotics are urgently needed to cope with the global rise of bacterial pathogens resistant to antibiotics in use, which renders lethal infections that once were treatable and controllable.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> To discover new antibiotics, processes essential for bacterial reproduction and spreading must be targeted, such as bacterial cell division, a clinically unexplored target. While eukaryotic mitosis has been vastly exploited for cancer treatments, targeting the bacterial cell division remains a clinical challenge. In this context, considerable knowledge has been gathered on the function, structure, dynamics, and interacting partners of essential cell division protein, FtsZ, since its localization forming a ring at the division site was discovered.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The FtsZ ring orchestrates the assembly of the divisomal machinery in most bacteria.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> It is formed by FtsZ clusters that undergo treadmilling and guide cell envelope invagination at the division site.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> Bacterial division and FtsZ have been recognized as attractive targets for discovering new antibiotics.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a> However, for many inhibitors reported in the literature, FtsZ targeting has not been really evidenced, slowing the progress in FtsZ-inhibitor development.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> FtsZ was validated as an antibacterial target of the potent antistaphylococcal experimental inhibitor PC190723 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>),<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> which is synergistic with β-lactams.<a onclick="showRef(event, 'ref2 ref15'); return false;" href="javascript:void(0);" class="ref ref2 ref15">(2,15)</a> However, in spite of considerable synthetic efforts, the benzamide derivative PC190723, many analogues, and other inhibitors,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> have not made it into the clinic, possibly due to unsuitable pharmacological properties and a relatively high frequency of resistance mutations. Only one prodrug analogue of PC190723, TXA709<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Chart S1</a>), was designated as a qualified infectious disease product for <i>Staphylococcus aureus</i> infections and has recently completed a phase I clinical trial.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fluorescent probes designed for the allosteric binding site of FtsZ. (A) Representation of the core structure of <i>S. aureus</i> FtsZ (SaFtsZ) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD">4DXD</a>), showing the nucleotide (red-green spheres) in its binding pocket and PC190723 (green-blue-red spheres) bound into a cleft between the N-terminal and C-terminal domains. The yellow circle marks a cavity above PC190723 potentially available for ligand binding and the positions of helices H6 and H7 within the structure are highlighted. (B) Zoom of the encircled surface. (C) Chemical structures of PC190723, compound <b>1</b>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and the fluorescent probes <b>2</b>–<b>5</b> employed in this work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">FtsZ filaments form by head to tail self-association of FtsZ monomers; this is followed by hydrolysis of FtsZ-bound guanosine triphosphate (GTP) that triggers monomer dissociation. Cooperative assembly of single-stranded FtsZ filaments is made possible by FtsZ monomers switching between a low and a high association affinity state during the FtsZ filament assembly.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> In relaxed unassembled FtsZ monomers from several species, a side cleft between the N-terminal GTP-binding and the C-terminal subdomains is closed, whereas this cleft opens in tense FtsZ monomers from <i>S. aureus</i> (SaFtsZ) forming crystal filaments,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> providing a binding site for allosteric modulators of FtsZ assembly. Indeed, PC190723 acts as an allosteric inhibitor that stabilizes FtsZ filaments against disassembly upon GTP hydrolysis and selectively inhibits cell division in <i>Bacillus</i> and <i>Staphylococcus</i> species.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> PC190723 fits snugly in a flat conformation into its binding site at the bottom part of the open cleft in SaFtsZ crystals,<a onclick="showRef(event, 'ref15 ref25'); return false;" href="javascript:void(0);" class="ref ref15 ref25">(15,25)</a> leaving apparently little possibility for chemical modification. However, there is an unfilled pocket above connected by a tunnel to PC190723 (encircled in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B) and also a broader cavity around this pocket, formed by helices H6 and H7 and the C-terminal subdomain. These cavities are known since the first FtsZ structure was described<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> but lack cognate ligands. Thus, there appears to be room for other ligands binding into the interdomain cleft in addition to PC190723.</div><div class="NLM_p">We developed fluorescent probes for the interdomain cleft of FtsZ, in which a small fluorophore replaces the chloro-thiazolopyridine tail of inhibitor PC190723. Among them, 4-chloro-7-nitro-2,1,3-benzoxadiazole (NBD) probe <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C) was employed to demonstrate the FtsZ assembly switch in solution, in which the side cleft between the N- and C-terminal domains, containing the benzamide binding site, opens in assembled FtsZ subunits and closes in unassembled monomers.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The missing crystal structures of closed, relaxed SaFtsZ monomers were then determined,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> supporting an equilibrium between the open and closed-cleft conformers, relaxing to closed by the dissociation of tight longitudinal contacts in SaFtsZ filaments.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The FtsZ polymerization-induced assembly switch enables the filament treadmilling displacement<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> that is required for the FtsZ function in cell division, underscoring the importance of targeting the interdomain cleft to inhibit FtsZ filament dynamics. Mechanistic and structural studies together with complementary cytological profiling methods could allow the characterization of effective FtsZ inhibitors.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> Nonetheless, the lack of appropriate chemical tools to develop a binding screen against this site has hampered the discovery of new FtsZ allosteric inhibitors.</div><div class="NLM_p">In this work, we describe the first competitive binding assay for screening FtsZ ligands based on the use of new high-affinity fluorescent probes <b>2</b> and <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Complementary phenotypic assays and X-ray crystallography have allowed the validation of this screen for the identification of FtsZ allosteric inhibitors. Hence, a series of structurally simplified benzamide analogues <b>6</b>–<b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) have been synthesized and evaluated in this new assay, identifying FtsZ-targeting inhibitors of bacterial division. Altogether, the biological and structural studies reported in this work provide relevant information about the molecular recognition of FtsZ allosteric ligands, which may contribute to the discovery of more effective antibacterial agents.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of compounds <b>6</b>–<b>26</b> evaluated as FtsZ inhibitors in this work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33430" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33430" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of fluorescent probes <b>2</b>–<b>5</b> is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Racemic probes <b>2</b>–<b>4</b> were prepared starting from 2,6-difluoro-3-hydroxybenzamide (<b>27</b>), which was O-alkylated with the appropriate <i>N</i>-Boc-2-bromoalkylamine <b>28</b>–<b>30</b>, followed by the removal of the Boc group by treatment with trifluoroacetic acid and coupling of the resulting intermediates <b>35</b>–<b>37</b> with NBD chloride. For the synthesis of (<i>R</i>)- and (<i>S</i>)-<b>2</b>, the Mitsunobu reaction under microwave (MW) irradiation between benzamide <b>27</b> and the corresponding enantiomeric form of <i>N</i>-Boc-aminoalcohol <b>31</b>, subsequent deprotection of the amino group, and reaction with NBD chloride afforded the desired enantioenriched probes. Racemic probe <b>5</b> and its enantiomers were obtained by applying the same sequence of Mitsunobu reaction, deprotection, and NBD-coupling but using the appropriate form of primary alcohol <b>38</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>2</b>−<b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>28</b>–<b>30</b>, K<sub>2</sub>CO<sub>3</sub>, NaI, dimethylformamide (DMF), rt, 24 h, 14–23%; (b) (<i>R</i>)- or (<i>S</i>)-<b>31</b>, diisopropyl azodicarboxylate (DIAD), PBu<sub>3</sub>, DMF, MW, 150 °C, 2 h, 28–31%; (c) trifluoroacetic acid (TFA), dichloromethane (DCM), rt, 1 h, 70–90%; (d) NBD-Cl, Cs<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 1 h, 7–50%; and (e) <i>rac</i>-, (<i>R</i>)-, or (<i>S</i>)-<b>38</b>, DIAD, PBu<sub>3</sub>, DMF, 80 °C, 48 h, 18–19%.</p></p></figure><div class="NLM_p">Regarding the synthesis of compounds <b>6</b>–<b>14</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), derivatives <b>6</b> and <b>9</b> were prepared by reductive amination of 3-chlorobenzaldehyde with 3- and 4-aminobenzamide, respectively. <i>N</i>-Methyl analogues <b>7</b> and <b>10</b> were synthesized by alkylation of regioisomers <b>6</b> and <b>9</b> with iodomethane, whereas reductive amination between <b>6</b> and propionaldehyde provided <i>N</i>-propyl derivative <b>8</b>. For the synthesis of <i>O</i>-(alkylphenyl)benzamides <b>11</b>–<b>14</b>, 3- or 4-hydroxybenzamide was alkylated with the proper bromoalkylaryl derivative under classical Williamson conditions (<b>11</b>, <b>13</b>, and <b>14</b>) or with 2-(3-chlorophenyl)ethanol in a Mitsunobu reaction to obtain compound <b>12</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>6</b>−<b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) 3-chlorobenzaldehyde, MW, 100 °C, 10 min; (ii) H<sub>2</sub>, Pd/C, MeOH, rt, 18 h, 32–74%; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 60 °C, 18 h, 31–35%; (c) (i) CH<sub>3</sub>CH<sub>2</sub>CHO, MeOH, rt, 18 h; (ii) NaBH<sub>3</sub>CN, MeOH, rt, 72 h, 52%; (d) K<sub>2</sub>CO<sub>3</sub>, NaI, DMF, rt, 24 h, 47–80%; and (e) PPh<sub>3</sub>, diethyl azodicarboxylate (DEAD), Et<sub>3</sub>N, tetrahydrofuran (THF), rt, 24 h, 46%.</p></p></figure><div class="NLM_p">Benzamide bioisosteres <b>15</b>–<b>17</b> were synthesized as depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The reaction of 3-[(3-chlorobenzyl)oxy]benzaldehyde with trimethyl(trifluoromethyl)silane afforded trifluoromethyl alcohol <b>41</b>, which was oxidized to ketone <b>42</b> with the Dess–Martin reagent, followed by reductive amination using <i>tert</i>-butanesulfinamide as an ammonia equivalent to provide target compound <b>15</b>. Imidazole derivatives <b>16</b> and <b>17</b> were prepared by the Pd- and Cu-mediated cross-coupling reaction of imidazole and 3-iodoanisole, subsequent demethylation with boron tribromide of <b>43</b>, and O-benzylation of phenol <b>44</b> with the corresponding chloro- or trifluoromethyl-substituted benzyl bromide.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>15</b>−<b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) TMSCF<sub>3</sub>, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 48 h; (ii) 2 M HCl, rt, 4 h, 87%; (b) Dess–Martin periodinane, DCM, rt, 18 h, 99%; (c) (i) (CH<sub>3</sub>)<sub>3</sub>CSONH<sub>2</sub>, Ti(OiPr)<sub>4</sub>, Et<sub>2</sub>O, reflux, 24 h; (ii) NaBH<sub>4</sub>, Et<sub>2</sub>O, rt, 24 h; (iii) 4 M HCl, 1,4-dioxane, rt, 1 h, 24%; (d) Pd(OAc)<sub>2</sub>, CuI, DMF, MW, 200 °C, 40 min, 53%; (e) BBr<sub>3</sub>, DCM, 0 °C to rt, 24 h, 95%; and (f) 3-chlorobenzyl bromide or 3-(trifluoromethyl)benzyl bromide, K<sub>2</sub>CO<sub>3</sub>, NaI, DMF, rt, 24 h, 72–75%.</p></p></figure><div class="NLM_p">2,6-Difluorobenzamides <b>18</b>–<b>26</b> were synthesized starting from benzamide <b>27</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Direct alkylation of <b>27</b> with the adequately substituted benzyl bromide under basic conditions afforded compounds <b>18</b>–<b>21</b> and Mitsunobu reaction of <b>27</b> with commercial or synthesized alcohols <b>45</b>–<b>47</b> gave analogues <b>22</b>–<b>26</b>. Primary alcohols <b>45</b> and <b>46</b> were prepared by the Ullman-type reaction between 1-iodo-3-(trifluoromethyl)benzene and 2-aminoethanol or 2-aminopentanol, respectively. Secondary alcohol <b>47</b> was obtained by the reduction of ketone <b>48</b> with lithium aluminum hydride.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0011.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>18</b>−<b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, NaI, DMF, rt, 24 h, 81–98%; (b) PBu<sub>3</sub>, DIAD, DMF, MW, 150 °C, 1.5 h, 45–50%; (c) K<sub>3</sub>PO<sub>4</sub>, CuI, (CH<sub>2</sub>OH)<sub>2</sub>, <i>i</i>-PrOH, MW, 150 °C, 105 min, 46–56%; and (d) LiAlH<sub>4</sub>, THF, rt, 2 h, 76%.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Enhanced Affinity Fluorescent Probes for the Interdomain Cleft of FtsZ</h3><div class="NLM_p">A key step in the development of a fluorescence-based binding assay targeting the FtsZ interdomain cleft is the generation of specific probes for this site. These tools should undergo a sensitive modification of their fluorescence properties when bound to the protein, which returns to those of the free probe when displaced by a reference ligand such as PC190723. Competition assays of this type require that the probe and the competitor have commensurate affinities, resulting in a limited measurement range of competitor affinities with a single probe.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Thus, specific medium affinity probes are required for the effective detection of weak but specifically competing molecules during inhibitor screening, whereas high-affinity probes are needed for the measurement of the dissociation constant (<i>K</i><sub>D</sub>) of high-affinity compounds. Our previous studies demonstrated that probe <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C) and its analogues with longer spacers between the difluorobenzamide ring and the fluorophore specifically bind in a narrow affinity range (<i>K</i><sub>D</sub> = 11–26 μM) with moderate fluorescence anisotropy changes<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> that were deemed insufficient for the development of a competitive assay. However, their biochemical effects, together with molecular dynamics simulations of model complexes, supported the notion that an extension of the PC190723 binding site is available for inhibitors binding into the FtsZ interdomain cleft.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Thus, taking probe <b>1</b> as the starting point, we resorted to adding aliphatic side chains to the spacer to gain additional hydrophobic interactions by increasing the length of the aliphatic chain according to the space available in the allosteric interdomain cleft. These new NBD-based compounds <b>2</b>–<b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) were characterized by their fluorescence spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S1</a>), the anisotropy changes upon specific reversible binding to FtsZ polymers (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), fluorescence microscopy of <i>Bacillus subtilis</i> cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), and binding affinity titrations (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S2 and Table S1</a>). FtsZ polymers were formed in the presence of the slowly hydrolyzable GTP analogue GMPCPP and magnesium. The four probes showed large anisotropy values in the presence of FtsZ polymers compared to the free probe (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). However, the recovery of the initial anisotropy values after the addition of excess competing PC190723, supporting binding specificity, was only achieved in the case of probes <b>1</b>, <b>2</b>, and <b>5</b>. Compounds <b>3</b> and <b>4</b>, with the longer side chains, exhibited an increase in background anisotropy with unassembled FtsZ in the absence of magnesium that were deemed nonspecific. In addition, the anisotropy of <b>4</b> with FtsZ polymers in the presence of magnesium was not reduced by PC190723 competition (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). We thus focused on specific propyl-substituted probes <b>2</b> and <b>5</b>, and both enantiomers of each compound were synthesized and evaluated. The <i>S</i> enantiomer of <b>2</b> [(<i>S</i>)-<b>2</b>] displayed enhanced affinity (<i>K</i><sub>D</sub> = 8 ± 1 μM; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a>) and a large anisotropy change compared to <b>1</b>, whereas (<i>R</i>)-<b>2</b> had a very small fluorescence anisotropy increase (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Better results were obtained when the propyl side chain was incorporated at the carbon next to the amino group, and (<i>R</i>)-<b>5</b> resulted in the highest affinity probe (<i>K</i><sub>D</sub> = 1.9 ± 0.6 μM) and broadest anisotropy changes, whereas (<i>S</i>)-<b>5</b> presented lower affinity (<i>K</i><sub>D</sub> = 15 ± 2 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The replacement of the NBD by boron dipyrromethene (BODIPY) or acrylodan fluorophores, as well as the substitution of the oxygen of the alkoxybenzamide by nitrogen, resulted in smaller or nonspecific anisotropy changes (probes <b>SP1</b>–<b>SP5</b><a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S1</a>, synthesis in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Scheme S1</a>).</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Specific binding of fluorescent probes to FtsZ′s allosteric site, activity on bacterial cells, and competitive binding assay for ligands binding to the site. (A) Fluorescence anisotropy values of the free probes (10 μM) in buffer, to which FtsZ (10 μM, unassembled) was added, followed by the slowly hydrolyzable GTP analogue GMPCPP (0.1 mM). BsFtsZ polymerization was then induced by the addition of MgCl<sub>2</sub> (10 mM) and anisotropy values were recorded 5–10 min later. Nonfluorescent PC190723 (10 μM) was subsequently added to displace the fluorescent probe and anisotropy was measured 5 min later. (B) Fluorescence microscopy images of <i>B. subtilis</i> cells treated for 1.5 h with the minimal division inhibitory concentrations (MDC) of (<i>S</i>)-<b>2</b> and each enantiomer of <b>5</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a>), acquired employing the probe fluorescence. (C) Displacement curves of (<i>S</i>)-<b>2</b> (3 μM) from stabilized FtsZ-GMPCPP polymers (8 μM binding sites) by PC190723 (black circles, <i>K</i><sub>D</sub> < 0.1 μM), and its moieties 2,6-difluoro-3-methoxybenzamide (DFMBA, <i>K</i><sub>D</sub> = 0.67 ± 0.05 mM) (red), and 6-(chloro[1,3]thiazolo[5,4-<i>b</i>]pyridin-2-yl)methanol (CTPM, <i>K</i><sub>D</sub> = 1.0 ± 0.1 mM) (blue). Solid lines are best fits to the data of a single site competition model (<a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>). The addition of guanosine diphosphate (GDP) (void circles) does not displace (<i>S</i>)-<b>2</b> from its binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Treating <i>B. subtilis</i> cells with probes <b>2</b> and <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) revealed that (<i>S</i>)-<b>2</b> and both enantiomers of <b>5</b> (at 25–100 μM) induced the characteristic filamentous phenotype due to the inhibition of cell division, supporting FtsZ targeting.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> These probes faintly stained intracellular structures, similarly to <b>1</b>,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> whereas (<i>R</i>)-<b>2</b> was inactive in these assays. Interestingly, the higher affinity probe (<i>R</i>)-<b>5</b> showed the best efficacy in the impairment of cell division and displayed antibacterial activity in <i>B. subtilis</i> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a>). Altogether, cytological profiling and in vitro binding measurements were in qualitative agreement and support the suitability of fluorescent probes (<i>S</i>)-<b>2</b> and (<i>R</i>)-<b>5</b> for the development of a competitive binding assay.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Competitive Fluorescent Method for Ligand Binding into the FtsZ Interdomain Cleft</h3><div class="NLM_p last">Capitalizing on the anisotropy decrease of the fluorescent benzamide probes (<i>S</i>)-<b>2</b> and (<i>R</i>)-<b>5</b> upon dissociation, we designed homogeneous assays to specifically determine the binding of any molecule to the PC190723 binding site in FtsZ polymers. It should be noted that this assay is more challenging than the ones described for ligands binding into the nucleotide orthosteric binding site of FtsZ monomers.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> In this new method, stabilized FtsZ polymers are required that do not disassemble upon nucleotide hydrolysis or by the action of other FtsZ polymerization inhibitors binding elsewhere in the protein molecule, producing misleading anisotropy changes. Gentle cross-linking of the FtsZ-GMPCPP polymers with 0.15% (v/v) glutaraldehyde, adapted from microtubule-fluorescent taxoid studies,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> rendered stabilized FtsZ from <i>B. subtilis</i> (BsFtsZ) polymer preparations that were resistant to depolymerization by GDP addition but still capable of probe binding with the same affinity values as noncross-linked polymers (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S2 and Table S1</a>). We then employed mixtures of stabilized FtsZ polymers and probe (<i>S</i>)-<b>2</b> in competition assays, measuring the decrease in fluorescence anisotropy upon displacement of the probe to determine binding at increasing concentrations of competing test ligands. As a proof of concept, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C shows the competition binding isotherms of the reference high-affinity ligand PC190723 and its weakly binding moieties DFMBA and CTPM. Importantly, when the potent polymerization inhibitor GDP was used as a negative control, no change in probe anisotropy was observed.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Binding Screen for Allosteric FtsZ Inhibitors Combined with Cytological Methods</h3><div class="NLM_p">Once the fluorescence method for ligand binding into the interdomain cleft was set up, we evaluated its potential to screen for FtsZ allosteric inhibitors. Complementary to this screen, cytological profiling tests of the identified binders allow ascertaining their ability to target FtsZ in bacterial cells. Fluorescence anisotropy measurements of probe (<i>S</i>)-<b>2</b> at two concentrations of each tested compound (20 and 200 μM) were acquired; the solubility range of positively testing compounds was determined spectrophotometrically. Of note, the screen results can be displayed as raw anisotropy data, permitting to distinguish at a glance weak inhibitors from higher affinity candidates that markedly decrease the anisotropy values at both concentrations (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Among several previously described FtsZ inhibitors, two selective disruptors with an unknown binding site, PC170942<a onclick="showRef(event, 'ref35 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref37">(35,37)</a> and zantrin Z3,<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> gave no evidence of binding in our assay, and the specific GTP-replacing FtsZ-inhibitor UCM44<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> showed a marginal inhibitory effect (chemical structures in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Chart S2</a>). The promiscuous inhibitors resveratrol,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> plumbagin,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and tiplaxtinin<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Chart S2</a>) weakly modified anisotropy and total fluorescence intensity, although none of them impaired <i>B. subtilis</i> cell division or GTP-FtsZ ring localization in our hands. It should be noticed that aggregating compounds may sequester the protein or the probe or may scatter highly polarized light, potentially interfering with the fluorescence anisotropy measurements. Colored and fluorescent compounds can also interfere. Compounds initially testing positive in the anisotropy screen should thus be checked for any unexpected changes of the total fluorescence intensity in the same samples, as well as for any artefactual effects on replicates omitting FtsZ.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fluorescence anisotropy screen for FtsZ allosteric inhibitors combined with cell-based methods. (A) Screen for FtsZ inhibitors employing the fluorescence anisotropy of probe (<i>S</i>)-<b>2</b> (3 μM) and stabilized FtsZ-GMPCPP polymers (8 μM binding sites). The anisotropy values of the partially bound probe (void black circles), the free probe (void gray circles), and the values with two concentrations of each compound, 20 μM (gray circles) and 200 μM (black circles) are shown. Control values with the corresponding concentrations of dimethyl sulfoxide (DMSO) vehicle (2% void and gray circles; 4%, black circle) are also shown. (B) Chemical structures of synthetic ligands <b>6</b>–<b>26</b>. (C) Effects of low-affinity inhibitors <b>6</b>, <b>7</b>, and <b>9</b> on <i>B. subtilis</i> 168 cell division. Raw data plots (<i>n</i> ≥ 30) are shown with the mean values marked (statistics in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S2</a>). The cells were incubated for 3 h with each compound, and cell lengths were measured. (D) Representative phase-contrast images of cells after 3 h with <b>7</b> or <b>9</b> (0.75 mM each). (E) Cellular localization of FtsZ-GFP in <i>B. subtilis</i> SU570 following growth for 1.5 h in the absence (inset) and presence of <b>7</b> (0.2 mM). Bars, 10 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, <i>N</i>- or <i>O</i>-(alkylphenyl)benzamides <b>6</b>–<b>14</b>, <b>18</b>–<b>26</b>, and amide bioisosteres <b>15</b>–<b>17</b> were evaluated as FtsZ allosteric inhibitors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A,B). Different types of structural modifications around the three main structural features of benzamide inhibitors<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> were explored: (i) linear or branched alkylenamino or alkylenoxy linkers of different lengths as spacers, similar to the recent structure–activity relationship studies;<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (ii) difluorination of the benzamide core; and (iii) the position and effects of substituents around the phenyl ring selected as the aromatic moiety. Starting with lower affinity compounds, 3- and 4-aminobenzamides <b>6</b>, <b>7</b>, <b>9</b>, and <b>10</b> were identified as weak binders (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Complementary competition measurements indicated <i>K</i><sub>D</sub> values around 0.2 mM in an affinity order of <b>7</b> > <b>9</b> ≥ <b>6</b> ≥ <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S3</a>). Interestingly, compound <b>7</b> with a methylamino linker, markedly inhibited <i>B. subtilis</i> cell division at 0.75 mM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), inducing a characteristic filamentous phenotype (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), whereas its propyl-amino analogue <b>8</b> was toxic to the bacterial cells. In addition, compound <b>7</b> (0.2 mM) effectively induced irregularly spaced Z-rings and delocalization of GFP-FtsZ into punctate <i>foci</i> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E), characteristic of alkoxybenzamide inhibitors.<a onclick="showRef(event, 'ref14 ref44'); return false;" href="javascript:void(0);" class="ref ref14 ref44">(14,44)</a></div><div class="NLM_p last">These results indicate that our fluorescence binding screen method combined with phenotypic tests can effectively detect relatively weak FtsZ-targeting compounds, which may lead to more effective cell division inhibitors. Note that <b>7</b> inhibits <i>B. subtilis</i> division at a ∼20-fold lower concentration than the weak inhibitor 3-methoxybenzamide,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> which was the starting point in the PC190723 development.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The other way around, compounds inhibiting bacterial division may now be tested in fluorescence anisotropy assays for potentially targeting the allosteric interdomain cleft (this work) or the nucleotide-binding site of FtsZ.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Enhancing the Affinity of Simplified Benzamide Inhibitors</h3><div class="NLM_p">Evaluation of 3-alkoxybenzamide <b>11</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> showed that this oxygen derivative was more effective than nitrogen analogues <b>6</b>–<b>10</b> in our anisotropy screen (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Compound <b>11</b> had more than 10-fold lower <i>K</i><sub>D</sub> (10 μM, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) and a minimal cell division inhibitory concentration (MDC) value (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) than its 3-aminobenzamide analogue <b>6</b>. The <i>K</i><sub>D</sub> values, determined with the high-affinity probe (<i>R</i>)-<b>5</b>, are useful to compare the affinities of the different inhibitors among them and correlate with the corresponding MDC values. Derivatives of <b>11</b>, bearing an ethoxy or a propoxy spacer (<b>12</b> or <b>13</b>, respectively), had similar screen results (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A) and cell division inhibitory activities (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), whereas the 4-substituted analogue <b>14</b> was toxic. Replacement of the amide group of <b>11</b> by isosteres gave compounds <b>15</b>–<b>17</b> with reduced inhibition (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A) that were also toxic to the cells. However, the addition of fluorine atoms at 2- and 6-positions of the benzamide ring of <b>11</b> (as in PC190723) rendered the 3-chloro derivative <b>18</b><a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> and the 3-trifluoromethyl analogue <b>20</b> that were both stronger competitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A) with 1.3 and 0.7 μM <i>K</i><sub>D</sub> values, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>20</b> induced a marked divisomal inhibition phenotype on <i>B. subtilis</i>, consisting of cell filamentation and GFP-FtsZ delocalization (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; raw data plots in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S4</a> and statistics in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S2</a>), as well as characteristic enlarged spherical cells in <i>S. aureus</i> and FtsZ-mCherry delocalization (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Analogues of <b>20</b> with ethoxy and propoxy linkers <b>22</b> and <b>23</b>, respectively, were weaker binders, as were the 2- and 4-trifluoromethyl analogues <b>19</b> and <b>21</b>. Adding a positively charged dimethylaminoethyl side chain at the linker rendered analogue <b>24</b> water-soluble, although canceled binding and cellular activity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). At this point, we sought to further enhance inhibitor affinity with an analogous strategy to that employed with the fluorescent probes above. The starting point was compound <b>25</b> with an aminoethoxy linker between the two aromatic subunits, which is a higher affinity 3-trifluoromethylphenyl equivalent of the NBD probe <b>1</b> with comparable <i>K</i><sub>D</sub> and cellular activity values to <b>20</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The incorporation of a propyl side chain to the linker in (<i>R</i>)-<b>26</b>, equivalent to probe (<i>R</i>)-<b>5</b>, resulted in maximal affinity in the series (<i>K</i><sub>D</sub> = 0.2 μM) with a 1 μM division inhibitory activity on <i>B. subtilis</i> cells, comparable to PC190723 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; raw data plots in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S4</a> and statistics in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S2</a>). The (<i>S</i>)-<b>26</b> enantiomer has 10–20-fold weaker affinity and MDC than the (<i>R</i>)-enantiomer, which can be related to the 8-fold reduced affinity of probe (<i>S</i>)-<b>5</b> with respect to (<i>R</i>)-<b>5</b>.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Simplified benzamide FtsZ inhibitors with enhanced affinity and phenotype. (A) Displacement curves of probe (<i>R</i>)-<b>5</b> (3 μM) from stabilized BsFtsZ-GMPCPP polymers (2.7 μM binding sites) by synthetic compounds <b>11</b> (black), <b>20</b> (blue), <b>25</b> (green), (<i>R</i>)-<b>26</b> (red), (<i>S</i>)-<b>26</b> (cyan), and by PC190723 (gray). Lines correspond to the best-fitted competition curve in each case. See <i>K</i><sub>D</sub> values in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. (B) Left column, <i>B. subtilis</i> 168 cells after 3 h of culture with <b>20</b>, <b>25</b>, and (<i>R</i>)- and (<i>S</i>)-<b>26</b>, observed by phase-contrast microscopy; right column, cellular localization of FtsZ-GFP (<i>B. subtilis</i> SU570) following growth for 1.5 h in the presence of the same inhibitors. (C) <i>S. aureus</i> cells expressing FtsZ-mCherry (RN4220 + pCN-ftsZmch) grown for 1.5 h in the absence (control) or presence of <b>20</b>, <b>25</b>, (<i>R</i>)-, (<i>S</i>)-<b>26</b>, and PC190723, observed by fluorescence microscopy. (D) Top, structures of inhibitors <b>20</b>, <b>25</b>, and (<i>R</i>)-<b>26</b>. Bottom, mapping on the SaFtsZ structure (gray cartoon diagram of PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VOA">3VOA</a>) of the amino acid residues modified in mutants resistant to <b>20</b> (blue), <b>25</b> (green), and (<i>R</i>)-<b>26</b> (red spheres).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibition of Bacterial Cell Division and Growth by FtsZ Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center">MDC<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> (μM) <i>B. subtilis</i></th><th class="colsep0 rowsep0" align="center">MDC (μM) <i>S. aureus</i><i>Mu50</i></th><th class="colsep0 rowsep0" align="center">MIC<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> (μM) <i>B. subtilis</i></th><th class="colsep0 rowsep0" align="center" char=".">MIC (μM) <i>S. aureus</i><i>Mu50</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">9.3 ± 0.5</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">7.1 ± 0.5</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">9.5 ± 0.5</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">1.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">71 ± 6</td><td class="colsep0 rowsep0" align="left">>100<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">>100<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="char" char=".">>100<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">0.7 ± 0.2 (0.7 ± 0.2)<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="char" char=".">50<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">6.7 ± 0.4</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">5.9 ± 0.4</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="char" char=".">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">1.5 ± 0.1</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char=".">50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">145 ± 10</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">1.9 ± 0.2 (0.3 ± 0.1)<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>rac</i>-<b>26</b></td><td class="colsep0 rowsep0" align="left">1.1 ± 0.1</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char=".">50<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>26</b></td><td class="colsep0 rowsep0" align="left">0.2 ± 0.1 (<0.1)<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="char" char=".">25<a class="ref internalNav" href="#t1fn9" aria-label="i">i</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>26</b></td><td class="colsep0 rowsep0" align="left">3.8 ± 0.2 (7.0 ± 0.5)<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">100<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a></td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PC190723</td><td class="colsep0 rowsep0" align="left"><0.1 (<0.1)<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Binding affinities (<i>K</i><sub>D</sub>) to stabilized polymers of FtsZ from <i>B. subtilis</i> are the mean ± standard error of the mean (SEM).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">MDC, minimal division inhibitory concentration.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">MIC, minimal growth inhibitory concentration.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">ND, not determined.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Above compound solubility in the culture medium.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">In parenthesis, binding affinity values to stabilized SaFtsZ polymers.</p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">MIC = 25 μM on <i>S. aureus</i> subsp. <i>aureus</i> (ATCC 25923), <i>Staphylococcus haemolyticus</i>, and <i>Staphylococcus epidermidis</i>.</p></div><div class="footnote" id="t1fn8"><sup><sup>h</sup></sup><p class="last">MIC = 100 μM on <i>S. aureus</i> subsp. <i>aureus</i>, 50 μM on <i>S. haemolyticus</i>, and <i>S. epidermidis</i>.</p></div><div class="footnote" id="t1fn9"><sup><sup>i</sup></sup><p class="last">MIC = 25 μM on <i>S. aureus</i> subsp. <i>aureus</i>, <i>S. haemolyticus</i>, and <i>S. epidermidis</i>.</p></div><div class="footnote" id="t1fn10"><sup><sup>j</sup></sup><p class="last">Toxic at this concentration. Approximate MDC values of toxic compounds, omitted from the table, were: <b>15</b> 250 μM, <b>16</b> 50 μM, <b>17</b> 50 μM, and <b>14</b>, MIC > 500 μM.</p></div></div></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Antibacterial Activity of Inhibitors and FtsZ Cleft Targeting in <i>S. aureus</i></h3><div class="NLM_p">Inspection of MDC and MIC values (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) shows that the inhibitor activity on <i>B. subtilis</i> follows the affinity improvement along the compound series, reaching values similar to PC190723. The affinities of binding of <b>20</b>, <b>25</b>, and (<i>S</i>)-<b>26</b> to FtsZ from <i>B. subtilis</i> and <i>S. aureus</i> are reasonably similar (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), as may be expected from the conservation of the PC109723 binding site in these two species.</div><div class="NLM_p">Compound <b>20</b> is active against methicillin-resistant <i>S. aureus</i> Mu50, <i>S. haemolyticus</i>, and <i>S. epidermidis</i> (MIC 25–50 μM). However, in contrast to PC190723, the MDC value of <b>20</b> on <i>S. aureus</i> Mu50 is 5-fold higher than on <i>B. subtilis</i>, suggesting that the 3-trifluoromethylphenyl tail confers somewhat less activity against this strain than the chloro-thiazolopyridine tail. Then, the <i>S. aureus</i> MIC marginally decreases from <b>20</b> to (<i>R</i>)-<b>26</b>, compared to a 10-fold decrease in <i>B. subtilis</i>, indicating that the linker modifications that enhance affinity improve the antistaphylococcal activity little. We explored the cytotoxic effect of the best FtsZ inhibitors <b>18</b>, <b>20</b>, <b>25</b>, and (<i>R</i>)-<b>26</b> on human lung fibroblasts IMR90 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S3</a>) and none of them significantly affected cell viability at their MIC value on <i>S. aureus</i> Mu50. Determination of the minimal bactericidal concentration (MBC) values showed that inhibitors <b>18</b> (MBC 100 μM), <b>20</b> (MBC 100 μM), <b>25</b> (MBC 100 μM), and (<i>R</i>)-<b>26</b> (MBC 25 μM) are bactericidal for <i>S. aureus</i>, as PC190723 (MBC 5 μM).</div><div class="NLM_p last">Puzzled by the comparatively high MDC and MIC values on <i>S. aureus</i>, we analyzed spontaneous <i>S. aureus</i> Mu50 (ATCC700699) mutants resistant to <b>20</b>, <b>25</b>, or (<i>R</i>)-<b>26</b>, which easily generated resistant strains (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S4</a>). Most mutations mapped to the <i>ftsZ</i> gene, at positions corresponding to amino acids (P115), (V151), A182, <b>G196</b>, <b>V214</b>, A237, L249, L261, M262, <b>N263</b>, (A285), V297, T309, and (T358). These residues predominantly cluster around the interdomain cleft (exceptions in parenthesis) and concentrate at the PC190723 binding site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D); several of them are typically modified in PC190723-resistant mutants<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (in the bold type above). Therefore, the resistance mutation results strongly support that the inhibitors tested target <i>S. aureus</i> FtsZ at the interdomain cleft. The reduced activity in comparison to <i>B. subtilis</i> might be caused by permeability differences or by the action of staphylococcal multidrug resistance efflux pumps.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Structural Insights into Inhibitor Binding to <i>S. aureus</i> FtsZ</h3><div class="NLM_p">To confirm the proposed interaction of the inhibitors identified in our fluorescence screening, as a validation of the ligand-binding assay, we were able to determine the crystal structures of the SaFtsZ folded core (residues 12–316)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> in complex with inhibitors <b>18</b>, <b>20</b>, and DFMBA to 1.55–1.70 Å resolution (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S5</a>). Omit electron density maps around the compounds clearly prove that the three molecules bind into the interdomain cleft (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S5</a>), as PC190723,<a onclick="showRef(event, 'ref15 ref25'); return false;" href="javascript:void(0);" class="ref ref15 ref25">(15,25)</a> TXA6101, and TXA707<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> (chemical structures in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Chart S1</a>). The protein structure in the three complexes is essentially the same as in inhibitor-free SaFtsZ (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VOA">3VOA</a>, mean root-mean-square deviation (rmsd) = 0.39 Å), the SaFtsZ-PC190723 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD">4DXD</a>, mean rmsd = 0.41 Å), and the SaFtsZ-TXA6101 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XDU">5XDU</a>, mean rmsd = 0.43 Å). Compounds <b>18</b> and <b>20</b> slightly increase the distance between helix H7 and the C-terminal domain by 0.5 Å as compared to DFMBA, while a larger displacement of 1 Å is observed in the other cases.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structures of FtsZ-inhibitor complexes. (A) Ligand–protein interactions in the binding site, indicated by dashed magenta lines (see the main text and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S6</a>; the green sphere represents a coordinated metal ion). Complex structures have been deposited at the PDB with accession numbers: SaFtsZ-DFMBA, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD1">6YD1</a>; SaFtsZ-<b>18</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD5">6YD5</a>; SaFtsZ-<b>20</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD6">6YD6</a>. (B) Binding site surface, shown in light gray around the ligands. Magenta surface: Thr309 and Gly196. Cyan surface: Met226 and Gly193. (C) Structural comparison between the SaFtsZ-<b>20</b> (gray) complex with SaFtsZ-<b>18</b>, PC190723 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD">4DXD</a>), and TXA6101 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XDU">5XDU</a>) complexes (pink carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The protein interactions with the superimposable benzamide rings of <b>18</b>, <b>20</b>, DFMBA, PC190723, and TXA6101 are conserved (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">S6</a>). Nevertheless, these interactions appear insufficient for strong binding, as suggested by the 66% occupancy of DFMBA, compatible with its low affinity, while that of <b>18</b> and <b>20</b> is 100%. On the other hand, the halogenated phenyl moieties of <b>18</b> and <b>20</b> bind deep into the pocket with their benzene rings connecting to T309 and I311. This last residue adopts the same conformation in all examined complexes, while T309 acts as a gate presenting two different conformations, closed and open. The first is observed with DFMBA, the smaller bound molecule not reaching T309, and with TXA6101. In this case, the hydroxyl group of the T309 side chain points to the solvent, while the methyl group lies next to G196, thus closing the central region of the binding site (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, purple). The open conformation is observed in the presence of <b>18</b>, <b>20</b>, or PC190723. In this arrangement, the T309 side chain points its hydroxyl group toward N263 and allows access to the central region of the binding site (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, purple). Interestingly, compounds <b>18</b> and <b>20</b> adopt the flattest geometry, with their aromatic rings located in the same plane, as compared to PC190723 and TXA6101 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). This geometry places the halogenated benzene ring close to the position occupied by the oxazole ring of the flexible compound TXA6101 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Chart S1</a>), where the 5-bromo group interacting with a deep hydrophobic pocket was linked to the enhanced compound activity.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Unlike TXA6101, compounds <b>18</b>, <b>20</b>, and PC190723 do not displace the M226 side chain (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C), which constitutes another gate (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, cyan) giving access to a ligand-induced pocket. Compounds <b>18</b> and <b>20</b> differ in their terminal halogenated group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). The chlorine atom of <b>18</b> is placed at 4 Å from both the O atom of G193 and the α-carbon of G196, while the trifluoromethyl group of <b>20</b> interacts with the backbone carbonyl oxygens of Q192, G193, G227, and V310.</div><div class="NLM_p">Following crystal soaking with PC190723, we could reproduce a complex structure superimposable with <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD">4DXD</a> (not shown). In contrast, systematic soaking and cocrystallization experiments under similar conditions, employing the inhibitors with modified spacers between the benzamide and phenyl rings [<b>22</b>–<b>25</b>, (<i>R</i>)<i>-</i>, and (<i>S</i>)-<b>26</b>], all of the probes [<b>1</b>, (<i>R</i>)<i>-</i>, (<i>S</i>)-<b>2</b>, <b>3</b>, <b>4</b>, (<i>R</i>)-, and (<i>S</i>)-<b>5</b>], as well as probes with longer spacers,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> gave FtsZ electron density maps clearly showing an empty interdomain cleft. Note, however, that each of these ligands binds specifically to FtsZ polymers in solution, several of them with low micromolar affinities (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and higher solubility than PC190723 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S6</a>). Recently, the crystal structure of SaFtsZ in complex with a fluorescent BODIPY derivative of TXA6101 bound into the interdomain cleft has been reported. Nonetheless, this probe (BOFP, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Chart S1</a>) binds to unassembled SaFtsZ in solution without nucleotide and magnesium, as well as to Gram-negative FtsZs, in puzzling contrast with its complex filament crystal structure,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> the specificity of our probes, and the characteristic selectivity of PC190723 analogues.</div><div class="NLM_p">SaFtsZ mutations conferring resistance to the inhibitors can be analyzed based on the SaFtsZ-<b>18</b> and -<b>20</b> complex structures. In particular, mutations G196S, M262I, and T309I confer resistance to <b>20</b>. The trifluoromethyl group of <b>20</b> is placed close to G196 and the introduction of a side chain in this position would produce steric clashes similarly to PC190723. Residue M262 locates at the center of the β-sheet that forms one side of the ligand-binding cleft. While its side chain points away from the cleft, the M262I mutation can modify the hydrophobic packing of the β-sheet and, thus, potentially affect inhibitor binding. An important modification is introduced in the binding cleft with the T309I mutation. The introduction of a bulkier residue will likely impede its movement, keeping closed the central region of the binding site, thus clogging the binding of compounds.</div><div class="NLM_p last">In summary, our crystallographic analysis confirms the allosteric site for compounds <b>18</b> and <b>20</b> and provides insights into the binding to FtsZ of simplified benzamide-based bacterial cell division inhibitors. Their halogenated phenyl rings, replacing the larger chloro-thiazolopyridine moiety of PC190723, structurally superpose to the middle bromo-oxazole ring of TXA6101 upon binding to FtsZ. Potential ligand extensions might reach the ligand-induced cavity occupied by the TXA6101 trifluoromethyl group or the upper unfilled cavity in FtsZ.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72646" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72646" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The allosteric benzamide binding site at the cleft between the nucleotide-binding and GTPase-activating domains of FtsZ has been explored in this work. This cleft opens when FtsZ switches into its active conformation for assembly into dynamic polar filament clusters that guide bacterial division. We have developed a novel competitive assay employing specific fluorescent probes to screen for allosteric inhibitors and evaluate their binding affinity. Fluorescence polarization screening, together with cytological profiling and X-ray crystallography, has permitted us to identify effective, structurally simplified benzamide FtsZ-targeting inhibitors of bacterial division in <i>B. subtilis</i> and methicillin-resistant <i>S. aureus</i>. Our in vitro competitive binding assay and in situ cell-based methodology, combined with structural insights into ligand extension, should facilitate the discovery and validation of new structurally diverse FtsZ allosteric inhibitors leading to more effective antibacterial agents.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56056" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56056" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis</h3><div class="NLM_p">Unless stated otherwise, starting materials, reagents, and solvents were purchased as high-grade commercial products from Abcr, Acros, Scharlab, Sigma-Aldrich, or Thermo Fisher Scientific and were used without further purification. All nonaqueous reactions were performed under an argon atmosphere in oven-dried glassware. Tetrahydrofuran (THF), diethyl ether, and dichloromethane (DCM) were dried using a Pure Solv Micro 100 Liter solvent purification system. Triethylamine was dried over KOH and distilled before use. Reactions under MW irradiation were performed in a Biotage Initiator 2.5 reactor. Reactions were monitored by analytical thin-layer chromatography (TLC) on silica gel plates supplied by Merck (Kieselgel 60 F-254) with detection by UV light (254 nm) or 10% phosphomolybdic acid solution in EtOH. Flash chromatography was performed on a Varian 971-FP flash purification system using silica gel cartridges (Varian, particle size 50 μm). Melting points (mp, uncorrected) were determined on a Stuart Scientific electrothermal apparatus. Infrared (IR) spectra were measured on a Bruker Tensor 27 instrument equipped with a Specac attenuated total reflection (ATR) accessory of 5200–650 cm<sup>–1</sup> transmission range; frequencies (ν) are expressed in cm<sup>–1</sup>. Nuclear magnetic resonance (NMR) spectra were recorded on a Bruker Avance III 700 MHz (<sup>1</sup>H, 700 MHz; <sup>13</sup>C, 175 MHz), Bruker Avance 500 MHz (<sup>1</sup>H, 500 MHz; <sup>13</sup>C, 125 MHz), or Bruker DPX 300 MHz (<sup>1</sup>H, 300 MHz; <sup>13</sup>C, 75 MHz) instruments at the Universidad Complutense de Madrid (UCM) NMR core facilities. Chemical shifts (δ) are expressed in parts per million relative to internal tetramethylsilane; coupling constants (<i>J</i>) are in hertz (Hz). The following abbreviations are used to describe peak patterns when appropriate: singlet (s), doublet (d), triplet (t), quartet (q), quintet (qt), sextet (sext), multiplet (m), broad (br), and apparent (app). Two-dimensional (2D) NMR experiments (heteronuclear multiple quantum coherence (HMQC) and heteronuclear multiple bond correlation (HMBC)) of representative compounds were carried out to assign protons and carbons of the new structures. High-resolution mass spectrometry (HRMS) was carried out on a Bruker FTMS APEX-Q-IV spectrometer in an electrospray ionization (ESI) mode or on a Bruker matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF)/TOF ULTRAFLEX spectrometer at the UCM’s mass spectrometry core facility. High-performance liquid chromatography coupled to mass spectrometry (HPLC-MS) analysis was performed using an Agilent 1200LC-MSD VL instrument. LC separation was achieved with a Zorbax Eclipse XDB-C18 column (5 μm, 4.6 mm × 150 mm) or a Zorbax SB-C3 column (5 μm, 2.1 mm × 50 mm), both together with a guard column (5 μm, 4.6 mm × 12.5 mm). The gradient mobile phases consisted of A (95:5 water/acetonitrile) and B (5:95 water/acetonitrile) with 0.1% ammonium hydroxide and 0.1% formic acid as the solvent modifiers. MS analysis was performed with an ESI source. The capillary voltage was set to 3.0 kV and the fragmentor voltage was set at 72 eV. The drying gas temperature was 350 °C, the drying gas flow was 10 L/min, and the nebulizer pressure was 20 psi. Spectra were acquired in a positive or negative ionization mode from 100 to 1200 <i>m</i>/<i>z</i> and in a UV mode at four different wavelengths (210, 230, 254, and 280 nm). Optical rotation [α] was measured on an Anton Paar MCP 100 modular circular polarimeter using a sodium lamp (λ = 589 nm) with a 1 dm path length; concentrations (c) are given as g/100 mL. The enantiomeric excess (ee) was determined by HPLC using a chiral column (Chiralpak IA, 5 μm, 4.6 mm × 150 mm) and hexane/isopropanol/triethylamine/trifluoroacetic acid (70/30/0.3/0.1) as mobile phase (1 mL/min). HPLC traces were compared to racemic samples obtained by mixing equal amounts of the enantiopure compounds obtained independently.</div><div class="NLM_p last">Spectroscopic data of all described compounds were consistent with the proposed structures. Satisfactory HPLC-MS chromatograms were obtained for all tested compounds, which confirmed a purity of at least 95%.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure for the Synthesis of Final Compounds <b>2</b>–<b>5</b></h3><div class="NLM_p">To a solution of amine <b>35</b>–<b>37</b> or <b>40</b> (1 equiv) and Cs<sub>2</sub>CO<sub>3</sub> (3 equiv) in dry acetonitrile (50 mL/mmol), a solution of NBD-Cl (1–1.2 equiv) in dry acetonitrile (1 mL/mmol) was added and the reaction was stirred at 80 °C for 1 h. Then, the solvent was evaporated under reduced pressure and the crude was purified by chromatography (from hexane to hexane/EtOAc, 1:2) to afford final compounds <b>2</b>–<b>5</b>.</div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 2,6-Difluoro-3-[(1-{[(7-nitro-2,1,3-benzoxadiazol-4-yl)amino]methyl}butyl)oxy]benzamide (<b>2</b>)</h4><div class="NLM_p">Obtained from amine <b>35</b> (28 mg, 0.11 mmol), NBD-Cl (22 mg, 0.11 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (108 mg, 0.33 mmol) as an oil in 50% yield (23 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 1:2) 0.52; IR (ATR) ν 3351 (NH), 1683 (CO), 1623, 1507, 1489 (Ar); <sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>) δ 0.99 (t, <i>J</i> = 7.5, 3H, CH<sub>3</sub>), 1.46–1.54 (m, 2H, CH<sub>2</sub>), 1.68–1.77 (m, 1H, 1/2CH<sub>2</sub>), 1.81–1.89 (m, 1H, 1/2CH<sub>2</sub>), 3.67–3.82 (m, 2H, CH<sub>2</sub>N), 4.44–4.52 (m, 1H, CHO), 6.00 (br s, 2H, NH<sub>2</sub>), 6.25 (d, <i>J</i> = 8.5, 1H, CH<sub>NBD</sub>), 6.59 (br s, 1H, NH), 6.88 (td, <i>J</i> = 9.0, 1.5, 1H, H<sub>5</sub>), 7.07 (td, <i>J</i> = 9.0, 5.0, 1H, H<sub>4</sub>), 8.49 (d, <i>J</i> = 8.5, 1H, CH<sub>NBD</sub>); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 175 MHz) δ 14.2 (CH<sub>3</sub>), 18.6, 34.1 (2CH<sub>2</sub>), 46.9 (CH<sub>2</sub>N), 80.3 (CHO), 99.3 (br s, CH<sub>NBD</sub>), 111.9 (dd, <i>J</i><sub>C–F</sub> = 23.5, 4.0, C<sub>5</sub>), 114.6 (dd, <i>J</i><sub>C–F</sub> = 20.9, 16.8, C<sub>1</sub>), 122.0 (dd, <i>J</i> = 9.8, 2.3, C<sub>4</sub>), 124.6 (C<sub>NBD</sub>), 136.4 (CH<sub>NBD</sub>), 142.1 (dd, <i>J</i><sub>C–F</sub> = 11.0, 3.0, C<sub>3</sub>), 144.0 (2C<sub>NBD</sub>), 144.4, (C<sub>NBD</sub>), 151.6 (dd, <i>J</i><sub>C–F</sub> = 254.0, 7.0, CF), 154.7 (dd, <i>J</i><sub>C–F</sub> = 249.0, 5.5, CF), 161.8 (CONH<sub>2</sub>); ESI-HRMS (calcd, found for C<sub>18</sub>H<sub>21</sub>F<sub>2</sub>N<sub>6</sub>O<sub>5</sub> [M + NH<sub>4</sub>]<sup>+</sup>): 439.1536, 439.1548.</div><div id="sec5_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> (<i>R</i>)-<b>2</b></h5><div class="NLM_p last">Obtained from amine (<i>R</i>)-<b>35</b> (60 mg, 0.23 mmol), NBD-Cl (56 mg, 0.28 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (151 mg, 0.46 mmol) as an oil in 31% yield (30 mg). [α]<sub>D</sub><sup>24</sup> = +35.0 (<i>c</i> = 0.16, CHCl<sub>3</sub>). Chiral HPLC (<i>t</i><sub>R</sub>, min): 7.99, ee = 98%.</div></div><div id="sec5_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> (<i>S</i>)-<b>2</b></h5><div class="NLM_p last">Obtained from amine (<i>S</i>)-<b>35</b> (65 mg, 0.25 mmol), NBD-Cl (60 mg, 0.30 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (164 mg, 0.50 mmol) as an oil in 33% yield (35 mg). [α]<sub>D</sub><sup>24</sup> = −41.0 (<i>c</i> = 0.16, CHCl<sub>3</sub>). Chiral HPLC (<i>t</i><sub>R</sub>, min): 6.82, ee = 99%.</div></div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2,6-Difluoro-3-[(1-{[(7-nitro-2,1,3-benzoxadiazol-4-yl)amino]methyl}hexyl)oxy]benzamide (<b>3</b>)</h4><div class="NLM_p last">Obtained from amine <b>36</b> (17 mg, 0.06 mmol), NBD-Cl (12 mg, 0.06 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (59 mg, 0.18 mmol) as an oil in 30% yield (8 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 1:2) 0.42; IR (ATR) ν 3314 (NH), 1675 (CO), 1626, 1582, 1487 (Ar); <sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>) δ 0.90 (t, <i>J</i> = 7.0, 3H, CH<sub>3</sub>), 1.30–1.34 (m, 4H, 2CH<sub>2</sub>), 1.44–1.49 (m, 2H, CH<sub>2</sub>), 1.70–1.76 (m, 1H, 1/2CH<sub>2</sub>), 1.83–1.88 (m, 1H, 1/2CH<sub>2</sub>), 3.69–3.76 (m, 1H, 1/2CH<sub>2</sub>N), 3.79–3.85 (m, 1H, 1/2CH<sub>2</sub>N), 4.43–4.50 (m, 1H, CHO), 5.96–5.99 (m, 2H, NH<sub>2</sub>), 6.25 (d, <i>J</i> = 8.6, 1H, CH<sub>NBD</sub>), 6.58 (t, <i>J</i> = 5.7, 1H, NH), 6.89 (td, <i>J</i> = 9.0, 1.9, 1H, H<sub>5</sub>), 7.07 (td, <i>J</i> = 9.0, 5.2, 1H, H<sub>4</sub>), 8.49 (d, <i>J</i> = 8.6, 1H, CH<sub>NBD</sub>); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 175 MHz) δ 14.1 (CH<sub>3</sub>), 22.6, 24.9, 31.8, 32.0 (4CH<sub>2</sub>), 46.8 (CH<sub>2</sub>N), 80.6 (CHO), 99.2 (br s, CH<sub>NBD</sub>), 111.9 (dd, <i>J</i><sub>C–F</sub> = 23.5, 3.5, C<sub>5</sub>), 114.6 (dd, <i>J</i><sub>C–F</sub> = 20.9, 17.3, C<sub>1</sub>), 122.3 (dd, <i>J</i><sub>C–F</sub> = 9.8, 2.2, C<sub>4</sub>), 124.9 (C<sub>NBD</sub>), 136.3 (CH<sub>NBD</sub>), 142.0 (dd, <i>J</i><sub>C–F</sub> = 11.5, 2.5, C<sub>3</sub>), 143.8, 144.0, 144.5 (3C<sub>NBD</sub>), 151.8 (dd, <i>J</i><sub>C–F</sub> = 254.0, 7.0, CF), 154.9 (dd, <i>J</i><sub>C–F</sub> = 249.0, 5.0, CF), 161.4 (CONH<sub>2</sub>); ESI-HRMS (calcd, found for C<sub>20</sub>H<sub>20</sub>F<sub>2</sub>N<sub>5</sub>O<sub>5</sub> [M – H]<sup>−</sup>): 448.1438, 448.1419.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 2,6-Difluoro-3-[(1-{[(7-nitro-2,1,3-benzoxadiazol-4-yl)amino]methyl}nonyl)-oxy]benzamide (<b>4</b>)</h4><div class="NLM_p last">Obtained from amine <b>37</b> (110 mg, 0.26 mmol), NBD-Cl (62 mg, 0.26 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (254 mg, 0.78 mmol) as an oil in 11% yield (14 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 1:1) 0.42; IR (ATR) ν 3426 (NH), 1700 (CO), 1378, 1364 (Ar); <sup>1</sup>H NMR (700 MHz, acetone-<i>d</i><sub>6</sub>) δ 0.86 (t, <i>J</i> = 7.1, 3H, CH<sub>3</sub>), 1.25–1.36 (m, 10H, 5CH<sub>2</sub>), 1.50–1.59 (m, 2H, CH<sub>2</sub>), 1.85–1.88 (m, 2H, CH<sub>2</sub>), 4.01 (m, 2H, CH<sub>2</sub>N), 4.82 (qt, <i>J</i> = 5.7, 2H, CHO), 6.61 (d, <i>J</i> = 8.8, 1H, CH<sub>NBD</sub>), 6.90 (td, <i>J</i> = 9.0, 1.8, 1H, H<sub>5</sub>), 7.15 (br s, 1H, 1/2NH<sub>2</sub>), 7.26 (td, <i>J</i> = 9.2, 5.2, 1H, H<sub>4</sub>), 7.37 (br s, 1H, 1/2NH<sub>2</sub>), 8.31 (br s, 1H, NH), 8.53 (d, <i>J</i> = 8.7, 1H, CH<sub>NBD</sub>); <sup>13</sup>C NMR (175 MHz, acetone-<i>d</i><sub>6</sub>) δ 14.3 (CH<sub>3</sub>), 23.3, 25.6, 30.2 (3CH<sub>2</sub>), 30.3 (2CH<sub>2</sub>), 32.6, 32.8 (2CH<sub>2</sub>), 47.8 (br s, CH<sub>2</sub>N), 80.1 (CHO), 100.2 (br s, CH<sub>NBD</sub>), 111.7 (dd, <i>J</i><sub>C–F</sub> = 23.4, 3.9, C<sub>5</sub>), 117.6 (dd, <i>J</i><sub>C–F</sub> = 24.3, 20.4, C<sub>1</sub>), 119.3 (dd, <i>J</i><sub>C–F</sub> = 9.0, 1.7, C<sub>4</sub>), 123.9 (C<sub>NBD</sub>), 137.7 (br s, CH<sub>NBD</sub>), 143.4 (dd, <i>J</i><sub>C–F</sub> = 11.1, 3.2, C<sub>3</sub>), 145.1 (C<sub>NBD</sub>), 145.6 (2C<sub>NBD</sub>), 150.7 (dd, <i>J</i><sub>C–F</sub> = 249.7, 8.6, CF), 153.9 (dd, <i>J</i><sub>C–F</sub> = 243.2, 6.5, CF), 162.0 (CONH<sub>2</sub>); ESI-HRMS (calcd, found for C<sub>23</sub>H<sub>26</sub>F<sub>2</sub>N<sub>5</sub>O<sub>5</sub> [M – H]<sup>−</sup>): 490.1902, 490.1891.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2,6-Difluoro-3-{2-[(4-nitro-2,1,3-benzoxadiazol-7-yl)amino]pentoxy}benzamide (<b>5</b>)</h4><div class="NLM_p">Obtained from amine <b>40</b> (15 mg, 0.06 mmol), NBD-Cl (11 mg, 0.06 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (55 mg, 0.18 mmol) as an oil in 13% yield (3 mg). Chromatography: DCM/EtOAc, 9:1–7:3. <i>R</i><sub>f</sub> (hexane/EtOAc, 1:2) 0.32; IR (ATR) ν 3390 (NH), 1726 (CO), 1634, 1485 (Ar); <sup>1</sup>H NMR (700 MHz, methanol-<i>d</i><sub>4</sub>) δ 1.01 (t, <i>J</i> = 7.5, 3H, CH<sub>3</sub>), 1.46–1.60 (m, 2H, CH<sub>2</sub>), 1.81–1.91 (m, 2H, CH<sub>2</sub>), 4.19–4.21 (m, 1H, 1/2CH<sub>2</sub>O), 4.30 (dd, <i>J</i> = 10.0, 4.0, 1H, 1/2CH<sub>2</sub>O), 4.37 (br s, 1H, CHN), 6.54 (d, <i>J</i> = 9.0, 1H, CH<sub>NBD</sub>), 6.93 (td, <i>J</i> = 9.0, 2.0, 1H, H<sub>5</sub>), 7.18 (td, <i>J</i> = 9.0, 5.0, 1H, H<sub>4</sub>), 8.52 (d, <i>J</i> = 8.5, 1H, CH<sub>NBD</sub>); <sup>13</sup>C NMR (175 MHz, methanol-<i>d</i><sub>4</sub>) δ 14.2 (CH<sub>3</sub>), 20.3, 34.0 (2CH<sub>2</sub>), 54.8 (CHN), 73.3 (CH<sub>2</sub>O), 100.4 (CH<sub>NBD</sub>), 111.9 (dd, <i>J</i><sub>C–F</sub> = 23.0, 4.0, C<sub>5</sub>), 116.8 (dd, <i>J</i><sub>C–F</sub> = 24.0, 20.0, C<sub>1</sub>), 117.9 (d, <i>J</i><sub>C–F</sub> = 10.0, C<sub>4</sub>), 123.3 (C<sub>NBD</sub>), 138.5 (CH<sub>NBD</sub>), 144.7 (dd, <i>J</i><sub>C–F</sub> = 11.0, 3.0, C<sub>3</sub>), 145.6 (2C<sub>NBD</sub>), 145.9 (C<sub>NBD</sub>), 150.4 (dd, <i>J</i><sub>C–F</sub> = 251.5, 8.0, CF), 154.3 (dd, <i>J</i><sub>C–F</sub> = 244.0, 6.0, CF), 165.2 (CONH<sub>2</sub>); ESI-HRMS (calcd., found for C<sub>18</sub>H<sub>16</sub>F<sub>2</sub>N<sub>5</sub>O<sub>5</sub> [M – H]<sup>−</sup>): 420.1125, 420.1124.</div><div id="sec5_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> (<i>R</i>)-<b>5</b></h5><div class="NLM_p last">Obtained from amine (<i>R</i>)-<b>40</b> (40 mg, 0.15 mmol), NBD-Cl (31 mg, 0.16 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (100 mg, 0.30 mmol) as an oil in 30% yield (19 mg).</div></div><div id="sec5_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> (<i>S</i>)-<b>5</b></h5><div class="NLM_p last">Obtained from amine (<i>S</i>)-<b>40</b> (141 mg, 0.55 mmol), NBD-Cl (110 mg, 0.55 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (538 mg, 1.65 mmol) as an oil in 7% yield (19 mg).</div></div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for the Synthesis of Compounds <b>18</b> and <b>20</b></h3><div class="NLM_p">To a suspension of compound <b>27</b> (1 equiv) in anhydrous DMF (6 mL/mmol), potassium carbonate (1.5 equiv), sodium iodide (0.2 equiv), and the corresponding bromo derivative (1.5 equiv) were added. The reaction mixture was stirred at rt for 24 h. Then, the mixture was diluted in EtOAc and washed with brine (3×). The organic phase was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by chromatography (hexane to hexane/EtOAc, 2:8) to yield the desired compounds.</div><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 3-[(3-Chlorobenzyl)oxy]-2,6-difluorobenzamide (<b>18</b>)</h4><div class="NLM_p last">Obtained from <b>27</b> (237 mg, 1.4 mmol) and 3-chlorobenzyl bromide (0.25 mL, 2.0 mmol) as a white solid in 89% yield (293 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 3:7) 0.62; mp: 125–127 °C (lit.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> 121–122 °C); IR (ATR) ν 3378, 3192 (NH), 1655 (CO), 1594, 1492, 1455 (Ar); <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 5.23 (s, 2H, CH<sub>2</sub>), 6.94 (td, <i>J</i> = 9.0, 2.0, 1H, H<sub>5</sub>), 7.19 (br s, 1H, 1/2NH<sub>2</sub>), 7.26 (td, <i>J</i> = 9.0, 5.0, 1H, H<sub>4</sub>), 7.36–7.47 (m, 4H, H<sub>3′</sub>-H<sub>5′</sub>, 1/2NH<sub>2</sub>), 7.54 (br s, 1H, H<sub>2′</sub>); <sup>13</sup>C NMR (75 MHz, acetone-<i>d</i><sub>6</sub>) δ 71.4 (CH<sub>2</sub>), 111.6 (dd, <i>J</i> = 23.0, 4.5, C<sub>5</sub>), 117.2 (dd, <i>J</i> = 9.4, 2.7, C<sub>4</sub>), 117.6 (t, <i>J</i> = 19.9, C<sub>1</sub>), 126.9 (CH<sub>Ar</sub>), 128.3 (C<sub>2′</sub>), 129.0, 131.1 (2CH<sub>Ar</sub>), 134.8 (C<sub>3′</sub>), 140.1 (C<sub>1′</sub>), 144.0 (dd, <i>J</i><sub>C–F</sub> = 11.0, 3.0, C<sub>3</sub>), 150.0 (dd, <i>J</i><sub>C–F</sub> = 249.9, 8.0, CF), 153.8 (dd, <i>J</i><sub>C–F</sub> = 243.6, 6.6, CF), 162.1 (CONH<sub>2</sub>); ESI-HRMS (calcd found for C<sub>14</sub>H<sub>11</sub>ClF<sub>2</sub>NO<sub>2</sub> [M(<sup>35</sup>Cl) + H]<sup>+</sup>): 298.0441, 298.0453; (calcd found for C<sub>14</sub>H<sub>11</sub>ClF<sub>2</sub>NO<sub>2</sub> [M(<sup>37</sup>Cl) + H]<sup>+</sup>): 300.0411, 300.0424.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2,6-Difluoro-3-{[3-(trifluoromethyl)benzyl]oxy}benzamide (<b>20</b>)</h4><div class="NLM_p last">Obtained from <b>27</b> (217 mg, 1.3 mmol) and 3-(trifluoromethyl)benzyl bromide (0.28 mL, 1.9 mmol) as a white solid in 98% yield (272 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 3:7) 0.60; mp: 140–143 °C; IR (ATR) ν 3371, 3194 (NH), 1655 (CO), 1592, 1495, 1456 (Ar); <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 5.33 (s, 2H, CH<sub>2</sub>), 6.98 (td, <i>J</i> = 9.0, 2.0, 1H, H<sub>5</sub>), 7.19 (br s, 1H, 1/2NH<sub>2</sub>), 7.30 (td, <i>J</i> = 9.0, 5.0, 1H, H<sub>4</sub>), 7.46 (br s, 1H, 1/2NH<sub>2</sub>), 7.65–7.72 (m, 2H, H<sub>4′</sub>, H<sub>5′</sub>), 7.81 (d, <i>J</i> = 7.5 1H, H<sub>6′</sub>), 7.85 (s, 1H, H<sub>2′</sub>); <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>) δ 71.6 (CH<sub>2</sub>), 111.6 (dd, <i>J</i><sub>C–F</sub> = 23.3, 3.1, C<sub>5</sub>), 117.3 (dd, <i>J</i><sub>C–F</sub> = 9.5, 2.0, C<sub>4</sub>), 117.6 (t, <i>J</i><sub>C–F</sub> = 23.9, C<sub>1</sub>), 125.1 (d, <i>J</i><sub>C–F</sub> = 2.8, C<sub>2′</sub>) 125.3 (q, <i>J</i><sub>C–F</sub> = 271.5, CF<sub>3</sub>), 125.7 (d, <i>J</i><sub>C–F</sub> = 2.7, C<sub>4′</sub>) 130.4 (C<sub>5′</sub>), 131.2 (q, <i>J</i><sub>C–F</sub> = 32.0, C<sub>3′</sub>), 132.3 (C<sub>6′</sub>), 139.1 (C<sub>1′</sub>), 144.1 (dd, <i>J</i> = 11.0, 2.0, C<sub>3</sub>), 150.1 (dd, <i>J</i><sub>C–F</sub> = 250.0, 8.0, CF), 153.9 (dd, <i>J</i><sub>C–F</sub> = 243.0, 6.0, CF), 162.1 (CONH<sub>2</sub>); ESI-HRMS (calcd found for C<sub>15</sub>H<sub>11</sub>F<sub>5</sub>NO<sub>2</sub> [M + H]<sup>+</sup>): 332.0710, 332.0720.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> General Procedure for the Synthesis of Compounds <b>25</b> and <b>26</b></h3><div class="NLM_p">A solution of compound <b>27</b>, alcohol <b>45</b> or <b>46</b> (1 equiv), tributylphosphine (1 equiv), and diisopropyl azodicarboxylate (1 equiv) in anhydrous DMF (8 mL/mmol) was stirred under MW irradiation at 150 °C for 90 min. Then, the reaction was diluted with EtOAc (30 mL) and washed with brine (3 × 20 mL). The organic phase was dried, filtered, and concentrated under reduced pressure. The residue was purified by chromatography (from hexane to hexane/EtOAc, 1:1) to yield the desired final compounds <b>25</b> and <b>26</b>.</div><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2,6-Difluoro-3-(2-{[3-(trifluoromethyl)phenyl]amino}ethoxy)benzamide (<b>25</b>)</h4><div class="NLM_p last">Obtained from <b>27</b> (80 mg, 0.46 mmol) and alcohol <b>45</b> (95 mg, 0.46 mmol) as an oil in 49% yield (81 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 1:1) 0.25; IR (ATR) ν 3354, 3189 (NH), 1671 (CO), 1614, 1490 (Ar); <sup>1</sup>H NMR (300 MHz, methanol-<i>d</i><sub>4</sub>) 3.57 (t, <i>J</i> = 5.4, 2H, CH<sub>2</sub>N), 4.22 (t, <i>J</i> = 5.4, 2H, CH<sub>2</sub>O), 6.79–7.03 (m, 3H, H<sub>2′</sub>, H<sub>4′</sub>, H<sub>6′</sub>, H<sub>5</sub>), 7.18 (td, <i>J</i> = 9.2, 5.1, 1H, H<sub>4</sub>), 7.28 (t, <i>J</i> = 7.8, 1H, H<sub>5′</sub>); <sup>13</sup>C NMR (75 MHz, methanol-<i>d</i><sub>4</sub>) δ mixture of rotamers: 44.1 (CH<sub>2</sub>N), 70.1 (CH<sub>2</sub>O), 110.1 (q, <i>J</i><sub>C–F</sub> = 4.0, C<sub>2′</sub>), 111.9 and 112.1 (dd, <i>J</i><sub>C–F</sub> = 23.2, 4.2, C<sub>5</sub>), 114.4 (q, <i>J</i><sub>C–F</sub> = 3.9, C<sub>4′</sub>), 117.2 (br s, C<sub>1</sub>, C<sub>6′</sub>), 117.9 (dd, <i>J</i><sub>C–F</sub> = 9.3, 2.8) and 119.6 (dd, <i>J</i><sub>C–F</sub> = 9.2, 4.0, C<sub>4</sub>), 125.9 (q, <i>J</i><sub>C–F</sub> = 271.5, CF<sub>3</sub>), 130.8 (C<sub>5′</sub>), 132.5 (q, <i>J</i><sub>C–F</sub> = 31.4, C<sub>3′</sub>), 144.9 (dd, <i>J</i><sub>C–F</sub> = 10.9, 3.4, C<sub>3</sub>), 149.9 (C<sub>1′</sub>), 150.5 (dd, <i>J</i><sub>C–F</sub> = 251.5, 7.6, CF), 154.2 (dd, <i>J</i><sub>C–F</sub> = 243.7, 6.0, CF), 165.4 (CONH<sub>2</sub>); MALDI-HRMS (calcd found for C<sub>16</sub>H<sub>13</sub>F<sub>5</sub>N<sub>2</sub>O<sub>2</sub> [M]<sup>+</sup>): 360.0897, 360.0903.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2,6-Difluoro-3-[(2-{[3-(trifluoromethyl)phenyl]amino}pentyl)oxy]benzamide (<b>26</b>)</h4><div class="NLM_p">Obtained from <b>27</b> (52 mg, 0.30 mmol) and alcohol <b>46</b> (75 mg, 0.30 mmol) as a colorless oil in 45% yield (30 mg). <i>R</i><sub>f</sub> (hexane/EtOAc, 1:1) 0.48; IR (ATR) ν 3363 (NH), 1685 (CO), 1491, 1445 (Ar); <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 0.98 (t, <i>J</i> = 7.0, 3H, CH<sub>3</sub>), 1.41–1.69 (m, 3H, 31/2CH<sub>2</sub>), 1.77–1.84 (m, 1H, 1/2CH<sub>2</sub>), 3.75–3.79 (m, 1H, CHN), 4.04 (AB system, <i>J</i> = 9.3, 3.9, 2H, CH<sub>2</sub>O), 5.95 (br s, 2H, NH<sub>2</sub>), 6.77 (d, <i>J</i> = 8.1, 1H, H<sub>6′</sub>), 6.82 (s, 1H, H<sub>2′</sub>), 6.87 (dd, <i>J</i> = 9.1, 1.8, 1H, H<sub>5</sub>), 6.93 (d, <i>J</i> = 8.0, 1H, H<sub>4′</sub>), 6.96 (td, <i>J</i> = 9.1, 5.1, 1H, H<sub>4</sub>), 7.25 (t, <i>J</i> = 7.8, 1H, H<sub>5′</sub>); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 14.2 (CH<sub>3</sub>), 19.5 (<i>C</i>H<sub>2</sub>CH<sub>3</sub>), 34.4 (<i>C</i>H<sub>2</sub>CH), 52.5 (CHN), 72.0 (CH<sub>2</sub>O), 109.4 (q, <i>J</i><sub>C–F</sub> = 3.8, C<sub>2′</sub>), 111.4 (dd, <i>J</i><sub>C–F</sub> = 23.7, 4.3, C<sub>5</sub>), 114.1 (q, <i>J</i><sub>C–F</sub> = 4.0, C<sub>4′</sub>), 114.2 (q, <i>J</i><sub>C–F</sub> = 36.3, C<sub>1</sub>), 116.4 (C<sub>6′</sub>), 117.7 (dd, <i>J</i><sub>C–F</sub> = 9.8, 3.0, C<sub>4</sub>), 125.2 (q, <i>J</i><sub>C–F</sub> = 272.4, CF<sub>3</sub>), 130.0 (C<sub>5′</sub>), 131.9 (q, <i>J</i><sub>C–F</sub> = 31.7, C<sub>3′</sub>), 144.0 (dd, <i>J</i><sub>C–F</sub> = 11.4, 3.4, C<sub>3</sub>), 147.6 (C<sub>1′</sub>), 150.6 (dd, <i>J</i><sub>C–F</sub> = 254.9, 7.2, CF), 153.9 (dd, <i>J</i><sub>C–F</sub> = 247.2, 5.4, CF), 162.0 (CONH<sub>2</sub>); ESI-HRMS (calcd found for C<sub>19</sub>H<sub>19</sub>F<sub>5</sub>N<sub>2</sub>O<sub>2</sub>Na [M + Na]<sup>+</sup>): 425.1264, 425.1700.</div><div id="sec5_4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> (<i>R</i>)-<b>26</b></h5><div class="NLM_p last">Obtained from <b>27</b> (50 mg, 0.29 mmol) and alcohol (<i>R</i>)-<b>46</b> (73 mg, 0.29 mmol) as an oil in 49% yield (57 mg). [α]<sub>D</sub><sup>24</sup> = +29.0 (<i>c</i> = 0.15, CHCl<sub>3</sub>).</div></div><div id="sec5_4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> (<i>S</i>)-<b>26</b></h5><div class="NLM_p last">Obtained from <b>27</b> (116 mg, 0.67 mmol) and alcohol (<i>S</i>)-<b>46</b> (150 mg, 0.61 mmol) as an oil in 36% yield (88 mg). [α]<sub>D</sub><sup>24</sup> = −28.5 (<i>c</i> = 0.15, CHCl<sub>3</sub>).</div></div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Fluorescent Probes and Inhibitors</h3><div class="NLM_p last">All tested compounds were dissolved in DMSO (spectroscopic grade, Merck) before use and kept frozen and dry. Previously reported compounds PC190723, DFMBA, CTPM, UCM44, and PC170942 were synthesized as described.<a onclick="showRef(event, 'ref23 ref35'); return false;" href="javascript:void(0);" class="ref ref23 ref35">(23,35)</a> Zantrin Z3 was obtained from Mcule. Plumbagin, resveratrol, and tiplaxtinin were purchased from Sigma-Aldrich.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Protein Purification and Assembly Conditions</h3><div class="NLM_p">FtsZ from <i>B. subtilis</i> (BsFtsZ) was prepared as described previously.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> FtsZ was assembled in 50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES)-KOH, 50 mM KCl, 1 mM ethylenediaminetetraacetic acid (EDTA), pH 6.8 (HEPES buffer) plus 10 mM MgCl<sub>2</sub>, and 1 mM GTP or 0.1 mM GMPCPP at 25 °C. FtsZ assembly was monitored by light scattering.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><div class="NLM_p last">Full-length <i>S. aureus</i> FtsZ (SaFtsZ) and the truncated protein SaFtsZ(12–315) were purified as described.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> SaFtsZ was assembled in 50 mM MES, 50 mM KCl, 1 mM EDTA, pH 6.5 plus 10 mM MgCl<sub>2</sub>, and 0.1 mM GMPCPP.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Fluorescence and Anisotropy Measurements</h3><div class="NLM_p last">The fluorescence spectra and anisotropy values of the different probes were acquired with a Fluoromax-4 (Horiba Jobin Yvon) photon-counting L-format spectrofluorometer using 2 mm × 10 mm cells at 25 °C, employing the maximum excitation and emission wavelength for each probe with 2 and 5 nm bandwidths, respectively. Anisotropy values were automatically measured as <i>r</i> = (Ivv – <i>G</i> Ivh)/(Ivv + 2<i>G</i> Ivh), where the subscripts refer to the vertical (v) or horizontal (h) positions of the excitation (first subscript) and emission (second subscript) polarizers, and <i>G</i> = Ihv/Ihh. Total intensity values were simultaneously determined as <i>I</i><sub>T</sub> = Ivv +2<i>G</i> Ivh. Note that measurement correction procedures with T-format (double channel) fluorometers and with plate readers are slightly different.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Preparation of Cross-Linked FtsZ Polymers</h3><div class="NLM_p last">FtsZ (10–15 μM) was assembled in HEPES buffer, 10 mM MgCl<sub>2</sub>, 50 μM GMPCPP at 25 °C for 10 min, and then 0.15% (v/v) glutaraldehyde<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (distilled grade for microscopy, TAAB Laboratories, U.K.) was added to the solution that was incubated at 25 °C for 10 min more. (This was the minimal glutaraldehyde concentration determined to stabilize FtsZ polymers specifically binding probe 1.) The remains of the cross-linking agent were quenched by adding 60 mM NaBH<sub>4</sub>, the sample was incubated on ice for 10 min and degassed.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Cross-linked polymers were centrifuged for 10 min at 8200<i>g</i> (5000 rpm) and 4 °C in 15 mL Falcon tubes employing a Rotina 380R (Hettich) centrifuge, the supernatant was removed and the pellet was resuspended in the same volume of HEPES buffer, 10 mM MgCl<sub>2</sub>, containing 5 μM GMPCPP. Fixed FtsZ polymers were active in binding assays after more than 2 days at 4 °C; they could also be frozen in liquid nitrogen with a small loss of binding capacity. However, they were observed to precipitate above 20 μM FtsZ concentration. Cross-linked polymers of SaFtsZ were similarly prepared.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Stoichiometry and Affinity of Binding of the Fluorescent Probes to FtsZ Polymers</h3><div class="NLM_p">The stoichiometry of binding of (<i>R</i>)-<b>5</b> to stabilized FtsZ polymers was measured using a centrifugation assay adapted from ref <a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a>. Then, FtsZ polymers (10 μM) were incubated at 25 °C in HEPES buffer, 10 mM MgCl<sub>2</sub>, 2% DMSO, and 0.1 mM GMPCPP at 25 °C in the presence of different probe concentrations, in a final volume of 0.1 mL. The samples were then centrifuged for 20 min at 100 000 rpm and 25 °C in a TLA-100 rotor. After centrifugation, the supernatant was carefully withdrawn and the pellets were resuspended in the same volume of buffer. The concentration of the free probe was determined spectrophotometrically in the supernatant, employing an extinction coefficient ε<sub>483</sub> = 25 400 M<sup>–1</sup> cm<sup>–1</sup> for (<i>R</i>)-<b>5</b> and the concentration of probe bound to FtsZ was calculated as the difference of the known total concentration of probe minus the free concentration. To calculate the FtsZ polymer concentration, 5 μL of the resuspended pellet was applied to sodium dodecyl sulfate (SDS)-12% polyacrylamide gels, stained with Coomassie blue, scanned with a CS-800 calibrated densitometer (BioRad), and the protein bands quantified using Quantity One software (BioRad).</div><div class="NLM_p last">Binding affinity of the probes to FtsZ was measured by an increase in anisotropy of the probe (adapted from ref <a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a>). Fixed concentrations of the probes (3 μM) were titrated with different FtsZ polymer concentrations (0–40 μM) in HEPES buffer with Mg<sup>2+</sup>, to obtain the anisotropy increment, Δ<i>r</i><sub>max</sub>, corresponding to the bound probe. The increase in anisotropy was plotted against the FtsZ polymer concentration (calculated by subtracting from the total protein concentration the critical concentration for polymerization under the same conditions) and iteratively least-squares fitted with an isotherm of binding to one site. The estimated values of Δ<i>r</i><sub>max</sub> were used to approximate the free FtsZ concentrations, and these new values were employed again until an unchanging Δ<i>r</i><sub>max</sub> value was obtained. The convergent data were used to calculate the binding constant of the fluorescent probe to FtsZ polymers (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a>).</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Affinity of Ligands Competing with the Fluorescent Probes</h3><div class="NLM_p last">Competition assays were performed by measuring, through the decrease in fluorescence anisotropy, the displacement of the fluorescent probes from stabilized FtsZ polymers. Increasing concentrations of competing ligands were added to the mixtures of cross-linked FtsZ polymers (10–15 μM) with probe <b>1</b> (3 μM), 7–8 μM FtsZ polymers with probe (<i>S</i>)-<b>2</b> (3 μM), or 2–3 μM FtsZ polymers with (<i>R</i>)-<b>5</b> (3 μM). HEPES buffer with 10 mM MgCl<sub>2</sub> and 0.1 mM GMPCPP was employed, with a final volume of 0.5 mL, and the anisotropy measured at 25 °C. Under these conditions, the initial fraction of the probe bound was between 0.4 and 0.5. The fraction of the probe bound (Vb) was calculated<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_m001.gif" alt="" id="_i42" /></img></span>where <i>r</i><sub>free</sub> and <i>r</i><sub>max</sub> are the anisotropy values of the free and bound probes, respectively, and <i>R</i> is the ratio between the fluorescence intensities of the bound and free probe (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a>). The binding data were plotted against the competing ligand concentration and fitted by competition equilibrium of the ligand and the probe for the same binding site. The resulting system of equations<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> was numerically solved using the Equigra v. 5.0 program,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> which provided the best-fitted value of the competitor binding constant.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Cellular Methods</h3><div class="NLM_p"><i>B. subtilis</i> 168 and <i>Staphylococcus</i> spp. were grown in cation adjusted Mueller–Hinton broth (CAMHB; Becton, Dickinson and Company) at 37 °C. <i>B. subtilis</i> SU570 were grown in the same medium at 30 °C. In all cases, the cells were grown to an absorbance of 0.1–0.2 at 600 nm and then incubated either with the compounds or DMSO vehicle. Microscopy assays and cell measurement were performed as previously described.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> MIC values were determined by a broth macrodilution method.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> MBC values were determined as the lowest concentration of inhibitor at which there was a 99.9% reduction in colony-forming units/mL compared to the inoculum. To construct a <i>S. aureus</i> strain expressing FtsZ-mCherry, the plasmid pCN-ftsZmch<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> was electroporated into RN4220 and 0.1 μM cadmium chloride (Sigma-Aldrich) was used to induce the expression of the construct under the control of the P<i>cad</i> promoter.</div><div class="NLM_p">Isolation of FtsZ-inhibitor-resistant mutants was performed by plating 100 μL of <i>S. aureus</i> Mu50 culture (∼10<sup>8</sup> cells/mL) into CAMHB supplemented with the corresponding FtsZ inhibitor at a concentration higher than the MIC and 2% DMSO. The plates were incubated for 24 h at 37 °C, and several colonies were picked for further analysis. Genomic DNA was extracted from each colony using the Wizard Genomic DNA Purification Kit (Promega) and was used in polymerase chain reactions (PCR) for amplification of <i>ftsZ</i> using the primers SaFtsZ-F (5′ TGGCCAATAAAACTAGGAG 3′) and SaFtsZ-R (5′ TGTTATCTGATGATTTGTGTTG 3′). The resulting PCR products were purified using the DNA gel extraction kit (Cultek) and sequenced (Macrogen Inc., Spain). Sequence analysis was performed with BLAST (NCBI).</div><div class="NLM_p last">The sensitivity of the IMR90 cell line to compounds was tested through a standard 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The results were reported as the viability percentage of the tested compound relative to vehicle (DMSO), obtained from two or three independent experiments performed in triplicate.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Crystallization and Structure Determination</h3><div class="NLM_p last">Crystallization assays were carried out using purified truncated SaFtsZ(12–316) at 14 mg/mL. For cocrystallization experiments, compounds (at 1–4 mM concentrations, depending on solubility) and 4% (v/v) 1-methyl-2-pyrrolidone (compound solvent; Sigma, analytical standard) were included in the protein solution. Crystals were grown at 295 K by vapor diffusion (sitting drop) under previously employed conditions:<a onclick="showRef(event, 'ref25 ref34'); return false;" href="javascript:void(0);" class="ref ref25 ref34">(25,34)</a> 0.2 M lithium sulfate, 10% ethylene glycol, 0.1 M Tris/HCl, pH 8.4–9.0, and 24–28% PEG5000 MME. The crystals employed to solve the structure of the SaFtsZ-DFMBA complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S5</a>) were obtained with 4 mM DFMBA at 27% PEG5000 MME, pH 8.7. Crystal soaking experiments were typically made with 5–20 mM compound and 10–20% (v/v) 1-methyl-2-pyrrolidone for 16–20 h. For the SaFtsZ-<b>18</b> and SaFtsZ-<b>20</b> complex structures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S5</a>), crystals grown in 28% PEG5000 MME, pH 8.6, and 25% PEG5000 MME, pH 8.4, respectively, were soaked in a 10 mM compound and 20% (v/v) 1-methyl-2-pyrrolidone. All crystals were flash-cooled by immersion in liquid nitrogen. Diffraction data were collected at the ALBA synchrotron (Spain) and the ESRF synchrotron (France). All data were processed using XDS<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and Aimless from the CCP4 Suite.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Data collection and refinement statistics are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S5</a>. The structures were determined through molecular replacement using Molrep<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and the PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VOA">3VOA</a> as a searching model. Model building and refinement were done using Coot<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and PHENIX,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> respectively. Refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S5</a>. Structural figures were prepared using PyMOL (Schrödinger Inc.).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02207" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05244" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05244" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02207?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02207</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Fluorescent benzamide probes synthesized in this work (Figure S1); isotherms of binding of the fluorescent probes <b>1</b>, (<i>S</i>)-<b>2</b>, and (<i>R</i>)-<b>5</b> to FtsZ polymers measured by fluorescence anisotropy (Figure S2); competition isotherms of weak FtsZ inhibitors from the binding screen (Figure S3); effect of high affinity FtsZ inhibitors on bacterial cell division (Figure S4); expanded view of the SaFtsZ binding cleft, with omit electron density maps (green), contoured at 3σ around compounds DFMBA, <b>18</b>, and <b>20</b> (Figure S5); ligand interaction diagram of residues located in the binding site (Figure S6); binding parameters of the fluorescent probes to FtsZ-GMPCPP polymers and activity on bacterial cells (Table S1); cell length of <i>B. subtilis</i> 168 and diameter of <i>S. aureus</i> cells incubated with FtsZ inhibitors (Table S2); cytotoxic effect of FtsZ inhibitors in IMR90 cell line (Table S3); spontaneous mutations of inhibitor-resistant <i>S. aureus</i> strains (Table S4); data collection and refinement statistics (Table S5); structure of FtsZ inhibitors TXA707, TXA709 and TXA6101, and fluorescent probe BOFP (Chart S1); structure of previously identified FtsZ inhibitors screened for binding against the interdomain cleft of the protein in this work (Chart S2); synthetic scheme S1; detailed synthetic procedures and characterization data of intermediates <b>31</b>–<b>37</b>, <b>39</b>–<b>47</b>, and <b>50</b>; final compounds <b>6–17</b>, <b>19</b>, <b>21</b>, <b>22</b>–<b>24</b> and probes <b>SP1</b>–<b>SP5</b>; solubility of all tested compounds (Table S6); and NMR spectra of final compounds and HPLC traces analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf">jm0c02207_si_001.pdf (4.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_002.csv">jm0c02207_si_002.csv (2.85 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD1">6YD1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD5">6YD5</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD6">6YD6</a> correspond to SaFtsZ in complex with DFMBA, compound <b>18</b>, and compound <b>20</b>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02207" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos Fernández-Tornero</span> - <span class="hlFld-Affiliation affiliation">Centro
de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#086b6e7c677a666d7a67486b616a266b7b616b266d7b"><span class="__cf_email__" data-cfemail="ccafaab8a3bea2a9bea38cafa5aee2afbfa5afe2a9bf">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">María L. López-Rodríguez</span> - <span class="hlFld-Affiliation affiliation">Dept.
Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8607-1085" title="Orcid link">http://orcid.org/0000-0001-8607-1085</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a1cccdd4dbcdd3e1d4c2cc8fc4d2"><span class="__cf_email__" data-cfemail="ed80819897819fad988e80c3889e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">José M. Andreu</span> - <span class="hlFld-Affiliation affiliation">Centro
de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8064-6933" title="Orcid link">http://orcid.org/0000-0001-8064-6933</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a5cf8bc88bc4cbc1d7c0d0e5c6ccc78bc6d6ccc68bc0d6"><span class="__cf_email__" data-cfemail="8be1a5e6a5eae5eff9eefecbe8e2e9a5e8f8e2e8a5eef8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sonia Huecas</span> - <span class="hlFld-Affiliation affiliation">Centro
de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lidia Araújo-Bazán</span> - <span class="hlFld-Affiliation affiliation">Centro
de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federico M. Ruiz</span> - <span class="hlFld-Affiliation affiliation">Centro
de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura B. Ruiz-Ávila</span> - <span class="hlFld-Affiliation affiliation">Centro
de Investigaciones Biológicas Margarita Salas, CSIC, Ramiro de Maeztu 9, 28040 Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">R. Fernando Martínez</span> - <span class="hlFld-Affiliation affiliation">Dept.
Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</span>; 
    <span>Present Address:
                        Dept. Química Orgánica e Inorgánica, Facultad de Ciencias, Univ. de Extremadura, E-06071 Badajoz, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3278-6074" title="Orcid link">http://orcid.org/0000-0002-3278-6074</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Escobar-Peña</span> - <span class="hlFld-Affiliation affiliation">Dept.
Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marta Artola</span> - <span class="hlFld-Affiliation affiliation">Dept.
Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</span>; 
    <span>Present Address:
                        Leiden Institute of Chemistry, Leiden University, 2333 CD Leiden, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3051-3902" title="Orcid link">http://orcid.org/0000-0002-3051-3902</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Henar Vázquez-Villa</span> - <span class="hlFld-Affiliation affiliation">Dept.
Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7911-3160" title="Orcid link">http://orcid.org/0000-0001-7911-3160</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mar Martín-Fontecha</span> - <span class="hlFld-Affiliation affiliation">Dept.
Química Orgánica, Facultad de Ciencias Químicas, UCM, Avda. Complutense s/n, 28040 Madrid, Spain</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.H. and L.A.-B. contributed equally to this work. This manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by grants from the Spanish MECD BFU 2014-51823-R, SAF2016-78792-R, PID2019-106279RB-I00, BFU 2017-87387-P.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4950-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71255" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71255" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank David Juan (CIB, CSIC) for protein purification, the staff at ALBA and ESRF synchrotrons for helpful assistance, Srdja Drakulic for support in data collection, and Mariana G. Pinho (ITQB, Lisboa) for the plasmid pCN-ftsZmch. This work was supported by grants BFU 2014-51823-R (J.M.A.), SAF2016-78792-R, PID2019-106279RB-I00 (M.L.L.-R.), BFU 2017-87387-P (C.F.-T.), and predoctoral FPU fellowships from MECD (A.E.-P. and M.A.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BsFtsZ</td><td class="NLM_def"><p class="first last">FtsZ from <i>Bacillus subtilis</i></p></td></tr><tr><td class="NLM_term">CTPM</td><td class="NLM_def"><p class="first last">6-(chloro[1,3]thiazolo[5,4-<i>b</i>]pyridin-2-yl)methanol</p></td></tr><tr><td class="NLM_term">DFMBA</td><td class="NLM_def"><p class="first last">2,6-difluoro-3-methoxybenzamide</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">MDC</td><td class="NLM_def"><p class="first last">minimal cell division inhibitory concentration</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">NBD</td><td class="NLM_def"><p class="first last">7-nitrobenzoxadiazole</p></td></tr><tr><td class="NLM_term">SaFtsZ</td><td class="NLM_def"><p class="first last">FtsZ from <i>Staphylococcus aureus</i></p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogberg, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plachouras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoxha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonsen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colomb-Cotinat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzschmar, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devleesschauwer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouakrim, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struelens, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suetens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group">the Burden of AMR Collaborative Group</span> <span> </span><span class="NLM_article-title">Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(18)30605-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2FS1473-3099%2818%2930605-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30409683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=56-66&author=A.+Cassiniauthor=L.+D.+Hogbergauthor=D.+Plachourasauthor=A.+Quattrocchiauthor=A.+Hoxhaauthor=G.+S.+Simonsenauthor=M.+Colomb-Cotinatauthor=M.+E.+Kretzschmarauthor=B.+Devleesschauwerauthor=M.+Cecchiniauthor=D.+A.+Ouakrimauthor=T.+C.+Oliveiraauthor=M.+J.+Struelensauthor=C.+Suetensauthor=D.+L.+Monnetauthor=the+Burden+of+AMR+Collaborative+Group&title=Attributable+deaths+and+disability-adjusted+life-years+caused+by+infections+with+antibiotic-resistant+bacteria+in+the+EU+and+the+European+Economic+Area+in+2015%3A+a+population-level+modelling+analysis&doi=10.1016%2FS1473-3099%2818%2930605-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis</span></div><div class="casAuthors">Cassini Alessandro; Hogberg Liselotte Diaz; Plachouras Diamantis; Quattrocchi Annalisa; Hoxha Ana; Struelens Marc J; Suetens Carl; Monnet Dominique L; Simonsen Gunnar Skov; Colomb-Cotinat Melanie; Kretzschmar Mirjam E; Devleesschauwer Brecht; Cecchini Michele; Ouakrim Driss Ait; Oliveira Tiago Cravo</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-66</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Infections due to antibiotic-resistant bacteria are threatening modern health care.  However, estimating their incidence, complications, and attributable mortality is challenging.  We aimed to estimate the burden of infections caused by antibiotic-resistant bacteria of public health concern in countries of the EU and European Economic Area (EEA) in 2015, measured in number of cases, attributable deaths, and disability-adjusted life-years (DALYs).  METHODS:  We estimated the incidence of infections with 16 antibiotic resistance-bacterium combinations from European Antimicrobial Resistance Surveillance Network (EARS-Net) 2015 data that was country-corrected for population coverage.  We multiplied the number of bloodstream infections (BSIs) by a conversion factor derived from the European Centre for Disease Prevention and Control point prevalence survey of health-care-associated infections in European acute care hospitals in 2011-12 to estimate the number of non-BSIs.  We developed disease outcome models for five types of infection on the basis of systematic reviews of the literature.  FINDINGS:  From EARS-Net data collected between Jan 1, 2015, and Dec 31, 2015, we estimated 671 689 (95% uncertainty interval [UI] 583 148-763 966) infections with antibiotic-resistant bacteria, of which 63·5% (426 277 of 671 689) were associated with health care.  These infections accounted for an estimated 33 110 (28 480-38 430) attributable deaths and 874 541 (768 837-989 068) DALYs.  The burden for the EU and EEA was highest in infants (aged <1 year) and people aged 65 years or older, had increased since 2007, and was highest in Italy and Greece.  INTERPRETATION:  Our results present the health burden of five types of infection with antibiotic-resistant bacteria expressed, for the first time, in DALYs.  The estimated burden of infections with antibiotic-resistant bacteria in the EU and EEA is substantial compared with that of other infectious diseases, and has increased since 2007.  Our burden estimates provide useful information for public health decision-makers prioritising interventions for infectious diseases.  FUNDING:  European Centre for Disease Prevention and Control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR76qXWYwPPXq69lAbTj9rGfW6udTcc2eZ4zCAkC9_DmLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D&md5=151619810730b07e3d84692f1d966b9a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2818%2930605-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252818%252930605-4%26sid%3Dliteratum%253Aachs%26aulast%3DCassini%26aufirst%3DA.%26aulast%3DHogberg%26aufirst%3DL.%2BD.%26aulast%3DPlachouras%26aufirst%3DD.%26aulast%3DQuattrocchi%26aufirst%3DA.%26aulast%3DHoxha%26aufirst%3DA.%26aulast%3DSimonsen%26aufirst%3DG.%2BS.%26aulast%3DColomb-Cotinat%26aufirst%3DM.%26aulast%3DKretzschmar%26aufirst%3DM.%2BE.%26aulast%3DDevleesschauwer%26aufirst%3DB.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DOuakrim%26aufirst%3DD.%2BA.%26aulast%3DOliveira%26aufirst%3DT.%2BC.%26aulast%3DStruelens%26aufirst%3DM.%2BJ.%26aulast%3DSuetens%26aufirst%3DC.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3D%26atitle%3DAttributable%2520deaths%2520and%2520disability-adjusted%2520life-years%2520caused%2520by%2520infections%2520with%2520antibiotic-resistant%2520bacteria%2520in%2520the%2520EU%2520and%2520the%2520European%2520Economic%2520Area%2520in%25202015%253A%2520a%2520population-level%2520modelling%2520analysis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2019%26volume%3D19%26spage%3D56%26epage%3D66%26doi%3D10.1016%2FS1473-3099%2818%2930605-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foster, T. J.</span></span> <span> </span><span class="NLM_article-title">Can β-lactam antibiotics be resurrected to combat MRSA?</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2018.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.tim.2018.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30031590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yitrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=26-38&author=T.+J.+Foster&title=Can+%CE%B2-lactam+antibiotics+be+resurrected+to+combat+MRSA%3F&doi=10.1016%2Fj.tim.2018.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Can β-Lactam Antibiotics Be Resurrected to Combat MRSA?</span></div><div class="casAuthors">Foster, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26-38</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of β -lactam antibiotics to treat infections caused by Staphylococcus aureus has been severely compromised by the acquisition by horizontal gene transfer of a gene that encodes the β -lactam-insensitive penicillin-binding protein PBP2a.  This allows methicillin-resistant S. aureus (MRSA) to proliferate in the presence of β -lactam antibiotics.  Paradoxically the dependence on PBP2a for the essential transpeptidase activity in cell wall peptidoglycan biosynthesis is the 'Achilles heel' of MRSA.  Compds. that disrupt the divisome, wall teichoic acid, and functional membrane microdomains act synergistically with β -lactams against MRSA.  These include drugs such as statins that are widely used in human medicine.  The antibiotics vancomycin and daptomycin are also synergistic with β -lactams, and combinations have been employed to treat persistent MRSA infections.  An addnl. benefit of exposing MRSA to β -lactams could be a redn. in virulence mediated by interfering with the global regulator Agr.  The mechanistic basis of synergy is discussed, and the possibility that β -lactams can be resurrected to combat MRSA infections is explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOd6SVTdIQqbVg90H21EOLACvtfcHk0liOz-z3hSQ1Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yitrbN&md5=72df9b9e79c6e0de8b757d0cf50bcebc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2018.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2018.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DFoster%26aufirst%3DT.%2BJ.%26atitle%3DCan%2520%25CE%25B2-lactam%2520antibiotics%2520be%2520resurrected%2520to%2520combat%2520MRSA%253F%26jtitle%3DTrends%2520Microbiol.%26date%3D2019%26volume%3D27%26spage%3D26%26epage%3D38%26doi%3D10.1016%2Fj.tim.2018.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>Antibiotic Resistance 2020</i>; <span class="NLM_publisher-name">WHO</span>, <span class="NLM_year">2020</span>. Can be found under: <a href="https://www.who.int/news-room/fact-sheets/detail/antibiotic-resistance" class="extLink">https://www.who.int/news-room/fact-sheets/detail/antibiotic-resistance</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=World+Health+Organization&title=Antibiotic+Resistance+2020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DAntibiotic%2520Resistance%25202020%26pub%3DWHO%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Centers for Disease Control and Prevention</span>.  <i>2019
Antibiotic Resistance Threats Report</i>; <span class="NLM_publisher-name">WHO</span>, <span class="NLM_year">2019</span>. Can be found under: <a href="https://www.cdc.gov/drugresistance/biggest-threats.html" class="extLink">https://www.cdc.gov/drugresistance/biggest-threats.html</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Centers+for+Disease+Control+and+Prevention&title=2019%0AAntibiotic+Resistance+Threats+Report"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3D2019%250AAntibiotic%2520Resistance%2520Threats%2520Report%26pub%3DWHO%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutkenhaus, J.</span></span> <span> </span><span class="NLM_article-title">FtsZ ring structure associated with division in <i>Escherichia coli</i></span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>354</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1038/354161a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2F354161a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1944597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADyaK38XlsFeltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=354&publication_year=1991&pages=161-164&author=E.+Biauthor=J.+Lutkenhaus&title=FtsZ+ring+structure+associated+with+division+in+Escherichia+coli&doi=10.1038%2F354161a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">FtsZ ring structure associated with division in Escherichia coli</span></div><div class="casAuthors">Bi, Erfei; Lutkenhaus, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">354</span>
        (<span class="NLM_cas:issue">6349</span>),
    <span class="NLM_cas:pages">161-4</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Genes for cell division have been identified in E. coli by the isolation of conditional lethal mutations that block cell division, but do not affect DNA replication or segregation.  Of these genes, ftsZ is of great interest as it acts earliest in the division pathway, is essential, its level dictates the frequency of division, and it is thought to be the target of two cell-division inhibitors, SulA, produced in response to DNA damage, and MinCD, which prevents division at old sites.  Immunoelectromicroscopy was used to localize the FtsZ protein to the division site.  The results suggest that FtsZ self-assembles into a ring structure at the future division site and may function as a cytoskeletal element.  The formation of this ring may be the point at which division is regulated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGys1rwzKlK7Vg90H21EOLACvtfcHk0liOz-z3hSQ1Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFeltg%253D%253D&md5=b7ea7db9fd481d35d2eea461f291d223</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F354161a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F354161a0%26sid%3Dliteratum%253Aachs%26aulast%3DBi%26aufirst%3DE.%26aulast%3DLutkenhaus%26aufirst%3DJ.%26atitle%3DFtsZ%2520ring%2520structure%2520associated%2520with%2520division%2520in%2520Escherichia%2520coli%26jtitle%3DNature%26date%3D1991%26volume%3D354%26spage%3D161%26epage%3D164%26doi%3D10.1038%2F354161a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">den
Blaauwen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamoen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, P. A.</span></span> <span> </span><span class="NLM_article-title">The divisome at 25: the road ahead</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2017.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.mib.2017.01.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28254403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2rs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=85-94&author=T.+den%0ABlaauwenauthor=L.+Hamoenauthor=P.+A.+Levin&title=The+divisome+at+25%3A+the+road+ahead&doi=10.1016%2Fj.mib.2017.01.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The divisome at 25: the road ahead</span></div><div class="casAuthors">den Blaauwen, Tanneke; Hamoen, Leendert W.; Levin, Petra Anne</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  At the same time, significant questions about the mechanisms coordinating division with cell growth, DNA replication, and chromosome segregation remain.  Not only did the presence of the FtsZ ring provide fodder for the next 25 years of research, the application of a then cutting-edge approach-immunogold labeling of bacterial cells-inspired other investigators to apply similarly state-of-the-art technologies in their own work.  The identification of the FtsZ ring by Bi and Lutkenhaus in 1991 was a defining moment for the field of bacterial cell division.  These efforts have led to important advances in our understanding of the factors underlying assembly and maintenance of the division machinery.  This review addresses the most prominent of these questions, setting the stage for the next 25 years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCo3OTjAkkpbVg90H21EOLACvtfcHk0liOz-z3hSQ1Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2rs70%253D&md5=be73c3e3d85f3c51882065d82764202e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2017.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2017.01.007%26sid%3Dliteratum%253Aachs%26aulast%3Dden%2BBlaauwen%26aufirst%3DT.%26aulast%3DHamoen%26aufirst%3DL.%26aulast%3DLevin%26aufirst%3DP.%2BA.%26atitle%3DThe%2520divisome%2520at%252025%253A%2520the%2520road%2520ahead%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2017%26volume%3D36%26spage%3D85%26epage%3D94%26doi%3D10.1016%2Fj.mib.2017.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisson-Filho, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squyres, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuru, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jukes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanNieuwenhze, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, E. C.</span></span> <span> </span><span class="NLM_article-title">Treadmilling by FtsZ filaments drives peptidoglycan synthesis and bacterial cell division</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">743</span>, <span class="refDoi"> DOI: 10.1126/science.aak9973</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscience.aak9973" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28209898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=739-743&author=A.+W.+Bisson-Filhoauthor=Y.+P.+Hsuauthor=G.+R.+Squyresauthor=E.+Kuruauthor=F.+Wuauthor=C.+Jukesauthor=Y.+Sunauthor=C.+Dekkerauthor=S.+Holdenauthor=M.+S.+VanNieuwenhzeauthor=Y.+V.+Brunauthor=E.+C.+Garner&title=Treadmilling+by+FtsZ+filaments+drives+peptidoglycan+synthesis+and+bacterial+cell+division&doi=10.1126%2Fscience.aak9973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Treadmilling by FtsZ filaments drives peptidoglycan synthesis and bacterial cell division</span></div><div class="casAuthors">Bisson-Filho, Alexandre W.; Hsu, Yen-Pang; Squyres, Georgia R.; Kuru, Erkin; Wu, Fabai; Jukes, Calum; Sun, Yingjie; Dekker, Cees; Holden, Seamus; Van Nieuwenhze, Michael S.; Brun, Yves V.; Garner, Ethan C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6326</span>),
    <span class="NLM_cas:pages">739-743</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The mechanism by which bacteria divide is not well understood.  Cell division is mediated by filaments of FtsZ and FtsA (FtsAZ) that recruit septal peptidoglycan-synthesizing enzymes to the division site.  To understand how these components coordinate to divide cells, we visualized their movements relative to the dynamics of cell wall synthesis during cytokinesis.  The division septum was built at discrete sites that moved around the division plane.  FtsAZ filaments treadmilled circumferentially around the division ring and drove the motions of the peptidoglycan-synthesizing enzymes.  The FtsZ treadmilling rate controlled both the rate of peptidoglycan synthesis and cell division.  Thus, FtsZ treadmilling guides the progressive insertion of new cell wall by building increasingly smaller concentric rings of peptidoglycan to divide the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVjGpjX3axorVg90H21EOLACvtfcHk0liI2Vz_g9gHjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yjtbc%253D&md5=3e37ca023a21ac1d08bbb87e2aeb496e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.aak9973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aak9973%26sid%3Dliteratum%253Aachs%26aulast%3DBisson-Filho%26aufirst%3DA.%2BW.%26aulast%3DHsu%26aufirst%3DY.%2BP.%26aulast%3DSquyres%26aufirst%3DG.%2BR.%26aulast%3DKuru%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DJukes%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DDekker%26aufirst%3DC.%26aulast%3DHolden%26aufirst%3DS.%26aulast%3DVanNieuwenhze%26aufirst%3DM.%2BS.%26aulast%3DBrun%26aufirst%3DY.%2BV.%26aulast%3DGarner%26aufirst%3DE.%2BC.%26atitle%3DTreadmilling%2520by%2520FtsZ%2520filaments%2520drives%2520peptidoglycan%2520synthesis%2520and%2520bacterial%2520cell%2520division%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D739%26epage%3D743%26doi%3D10.1126%2Fscience.aak9973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miguel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuillen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span> <span> </span><span class="NLM_article-title">GTPase activity–coupled treadmilling of the bacterial tubulin FtsZ organizes septal cell wall synthesis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">744</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1126/science.aak9995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscience.aak9995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28209899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yjur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=744-747&author=X.+Yangauthor=Z.+Lyuauthor=A.+Miguelauthor=R.+McQuillenauthor=K.+C.+Huangauthor=J.+Xiao&title=GTPase+activity%E2%80%93coupled+treadmilling+of+the+bacterial+tubulin+FtsZ+organizes+septal+cell+wall+synthesis&doi=10.1126%2Fscience.aak9995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">GTPase activity-coupled treadmilling of the bacterial tubulin FtsZ organizes septal cell wall synthesis</span></div><div class="casAuthors">Yang, Xinxing; Lyu, Zhixin; Miguel, Amanda; McQuillen, Ryan; Huang, Kerwyn Casey; Xiao, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6326</span>),
    <span class="NLM_cas:pages">744-747</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The bacterial tubulin FtsZ is the central component of the cell division machinery, coordinating an ensemble of proteins involved in septal cell wall synthesis to ensure successful constriction.  How cells achieve this coordination is unknown.  In Escherichia coli cells, FtsZ exhibits dynamic treadmilling predominantly detd. by its GTPase activity.  The treadmilling dynamics direct the processive movement of the septal cell wall synthesis machinery but do not limit the rate of septal synthesis.  In FtsZ mutants with severely reduced treadmilling, the spatial distribution of septal synthesis and the mol. compn. and ultrastructure of the septal cell wall were substantially altered.  Thus, FtsZ treadmilling provides a mechanism for achieving uniform septal cell wall synthesis to enable correct polar morphol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfemmDnK_v5bVg90H21EOLACvtfcHk0liI2Vz_g9gHjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yjur4%253D&md5=d332376a876204c2adec4c7a7b070781</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.aak9995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aak9995%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLyu%26aufirst%3DZ.%26aulast%3DMiguel%26aufirst%3DA.%26aulast%3DMcQuillen%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DK.%2BC.%26aulast%3DXiao%26aufirst%3DJ.%26atitle%3DGTPase%2520activity%25E2%2580%2593coupled%2520treadmilling%2520of%2520the%2520bacterial%2520tubulin%2520FtsZ%2520organizes%2520septal%2520cell%2520wall%2520synthesis%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D744%26epage%3D747%26doi%3D10.1126%2Fscience.aak9995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichmann, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saraiva, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veiga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macario, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanNieuwenhze, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipe, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinho, M. G.</span></span> <span> </span><span class="NLM_article-title">Peptidoglycan synthesis drives an FtsZ-treadmilling-independent step of cytokinesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>554</i></span>,  <span class="NLM_fpage">528</span>– <span class="NLM_lpage">532</span>, <span class="refDoi"> DOI: 10.1038/nature25506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fnature25506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=29443967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisl2ls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=554&publication_year=2018&pages=528-532&author=J.+M.+Monteiroauthor=A.+R.+Pereiraauthor=N.+T.+Reichmannauthor=B.+M.+Saraivaauthor=P.+B.+Fernandesauthor=H.+Veigaauthor=A.+C.+Tavaresauthor=M.+Santosauthor=M.+T.+Ferreiraauthor=V.+Macarioauthor=M.+S.+VanNieuwenhzeauthor=S.+R.+Filipeauthor=M.+G.+Pinho&title=Peptidoglycan+synthesis+drives+an+FtsZ-treadmilling-independent+step+of+cytokinesis&doi=10.1038%2Fnature25506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidoglycan synthesis drives an FtsZ-treadmilling-independent step of cytokinesis</span></div><div class="casAuthors">Monteiro, Joao M.; Pereira, Ana R.; Reichmann, Nathalie T.; Saraiva, Bruno M.; Fernandes, Pedro B.; Veiga, Helena; Tavares, Andreia C.; Santos, Margarida; Ferreira, Maria T.; Macario, Vania; Van Nieuwenhze, Michael S.; Filipe, Sergio R.; Pinho, Mariana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">554</span>
        (<span class="NLM_cas:issue">7693</span>),
    <span class="NLM_cas:pages">528-532</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Peptidoglycan is the main component of the bacterial wall and protects cells from the mech. stress that results from high intracellular turgor.  Peptidoglycan biosynthesis is very similar in all bacteria; bacterial shapes are therefore mainly detd. by the spatial and temporal regulation of peptidoglycan synthesis rather than by the chem. compn. of peptidoglycan.  The form of rod-shaped bacteria, such as Bacillus subtilis or Escherichia coli, is generated by the action of two peptidoglycan synthesis machineries that act at the septum and at the lateral wall in processes coordinated by the cytoskeletal proteins FtsZ and MreB, resp.  The tubulin homolog FtsZ is the first protein recruited to the division site, where it assembles in filaments-forming the Z ring-that undergo treadmilling and recruit later divisome proteins.  The rate of treadmilling in B. subtilis controls the rates of both peptidoglycan synthesis and cell division.  The actin homolog MreB forms discrete patches that move circumferentially around the cell in tracks perpendicular to the long axis of the cell, and organize the insertion of new cell wall during elongation.  Cocci such as Staphylococcus aureus possess only one type of peptidoglycan synthesis machinery, which is diverted from the cell periphery to the septum in prepn. for division.  The mol. cue that coordinates this transition has remained elusive.  Here we investigate the localization of S. aureus peptidoglycan biosynthesis proteins and show that the recruitment of the putative lipid II flippase MurJ to the septum, by the DivIB-DivIC-FtsL complex, drives peptidoglycan incorporation to the midcell.  MurJ recruitment corresponds to a turning point in cytokinesis, which is slow and dependent on FtsZ treadmilling before MurJ arrival but becomes faster and independent of FtsZ treadmilling after peptidoglycan synthesis activity is directed to the septum, where it provides addnl. force for cell envelope constriction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPOmsMFMKeZrVg90H21EOLACvtfcHk0liI2Vz_g9gHjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisl2ls7s%253D&md5=0c30964816aac226e31fd0561b5140c2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature25506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25506%26sid%3Dliteratum%253Aachs%26aulast%3DMonteiro%26aufirst%3DJ.%2BM.%26aulast%3DPereira%26aufirst%3DA.%2BR.%26aulast%3DReichmann%26aufirst%3DN.%2BT.%26aulast%3DSaraiva%26aufirst%3DB.%2BM.%26aulast%3DFernandes%26aufirst%3DP.%2BB.%26aulast%3DVeiga%26aufirst%3DH.%26aulast%3DTavares%26aufirst%3DA.%2BC.%26aulast%3DSantos%26aufirst%3DM.%26aulast%3DFerreira%26aufirst%3DM.%2BT.%26aulast%3DMacario%26aufirst%3DV.%26aulast%3DVanNieuwenhze%26aufirst%3DM.%2BS.%26aulast%3DFilipe%26aufirst%3DS.%2BR.%26aulast%3DPinho%26aufirst%3DM.%2BG.%26atitle%3DPeptidoglycan%2520synthesis%2520drives%2520an%2520FtsZ-treadmilling-independent%2520step%2520of%2520cytokinesis%26jtitle%3DNature%26date%3D2018%26volume%3D554%26spage%3D528%26epage%3D532%26doi%3D10.1038%2Fnature25506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lock, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harry, E. J.</span></span> <span> </span><span class="NLM_article-title">Cell-division inhibitors: new insights for future antibiotics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">324</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1038/nrd2510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fnrd2510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18323848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVKmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=324-338&author=R.+L.+Lockauthor=E.+J.+Harry&title=Cell-division+inhibitors%3A+new+insights+for+future+antibiotics&doi=10.1038%2Fnrd2510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-division inhibitors: new insights for future antibiotics</span></div><div class="casAuthors">Lock, Rowena L.; Harry, Elizabeth J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">324-338</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The growing problem of antibiotic resistance has been exacerbated by the use of new drugs that are merely variants of older overused antibiotics.  While it is naive to expect to restrain the spread of resistance without controlling antibacterial usage, the desperate need for drugs with novel targets has been recognized by health organizations, industry and academia alike.  The wealth of knowledge available about the bacterial cell-division pathway has aided target-driven approaches to identify novel inhibitors.  Here, we discuss the therapeutic potential of inhibiting bacterial cell division, and review the progress made in this exciting new area of antibacterial discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplHXog8Fa6zLVg90H21EOLACvtfcHk0lhvQ0rAxrw_QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVKmsrw%253D&md5=da2514ca45300b30d46db98a3b4be004</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd2510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2510%26sid%3Dliteratum%253Aachs%26aulast%3DLock%26aufirst%3DR.%2BL.%26aulast%3DHarry%26aufirst%3DE.%2BJ.%26atitle%3DCell-division%2520inhibitors%253A%2520new%2520insights%2520for%2520future%2520antibiotics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D324%26epage%3D338%26doi%3D10.1038%2Fnrd2510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the assembly of bacterial cell division protein FtsZ with small molecules</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1021/cb2003626</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb2003626" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSit7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=269-277&author=C.+Schaffner-Barberoauthor=M.+Martin-Fontechaauthor=P.+Chaconauthor=J.+M.+Andreu&title=Targeting+the+assembly+of+bacterial+cell+division+protein+FtsZ+with+small+molecules&doi=10.1021%2Fcb2003626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the assembly of bacterial cell division protein FtsZ with small molecules</span></div><div class="casAuthors">Schaffner-Barbero, Claudia; Martin-Fontecha, Mar; Chacon, Pablo; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-277</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  FtsZ is the key protein of bacterial cell division and an emergent target for new antibiotics.  It is a filament-forming GTPase and a structural homolog of eukaryotic tubulin.  A no. of FtsZ-interacting compds. have been reported, some of which have powerful antibacterial activity.  Here, the authors review recent advances and new approaches in modulating FtsZ assembly with small mols.  This includes analyzing their chem. features, binding sites, mechanisms of action, the methods employed, and computational insights, aimed at a better understanding of their mol. recognition by FtsZ and at rational antibiotic design.  A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAWWLj_dUcQ7Vg90H21EOLACvtfcHk0lhvQ0rAxrw_QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSit7vP&md5=81413a2f6c1ed3283644c85cc7ac9440</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb2003626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb2003626%26sid%3Dliteratum%253Aachs%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DTargeting%2520the%2520assembly%2520of%2520bacterial%2520cell%2520division%2520protein%2520FtsZ%2520with%2520small%2520molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D269%26epage%3D277%26doi%3D10.1021%2Fcb2003626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">den
Blaauwen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monasterio, O.</span></span> <span> </span><span class="NLM_article-title">Bacterial cell division proteins as antibiotic targets</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2014.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.bioorg.2014.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=24755375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCmtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=27-38&author=T.+den%0ABlaauwenauthor=J.+M.+Andreuauthor=O.+Monasterio&title=Bacterial+cell+division+proteins+as+antibiotic+targets&doi=10.1016%2Fj.bioorg.2014.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial cell division proteins as antibiotic targets</span></div><div class="casAuthors">den Blaauwen, Tanneke; Andreu, Jose M.; Monasterio, Octavio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteins involved in bacterial cell division often do not have a counterpart in eukaryotic cells and they are essential for the survival of the bacteria.  The genetic accessibility of many bacterial species in combination with the Green Fluorescence Protein revolution to study localization of proteins and the availability of crystal structures has increased our knowledge on bacterial cell division considerably in this century.  Consequently, bacterial cell division proteins are more and more recognized as potential new antibiotic targets.  An international effort to find small mols. that inhibit the cell division initiating protein FtsZ has yielded many compds. of which some are promising as leads for preclin. use.  The essential transglycosylase activity of peptidoglycan synthases has recently become accessible to inhibitor screening.  Enzymic assays for and structural information on essential integral membrane proteins such as MraY and FtsW involved in lipid II (the peptidoglycan building block precursor) biosynthesis have put these proteins on the list of potential new targets.  This review summarises and discusses the results and approaches to the development of lead compds. that inhibit bacterial cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqngOkmUR6AP7Vg90H21EOLACvtfcHk0lhvQ0rAxrw_QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCmtr4%253D&md5=44a078b9b351b7b2c4d81c26de0da512</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2014.03.007%26sid%3Dliteratum%253Aachs%26aulast%3Dden%2BBlaauwen%26aufirst%3DT.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DMonasterio%26aufirst%3DO.%26atitle%3DBacterial%2520cell%2520division%2520proteins%2520as%2520antibiotic%2520targets%26jtitle%3DBioorg.%2520Chem.%26date%3D2014%26volume%3D55%26spage%3D27%26epage%3D38%26doi%3D10.1016%2Fj.bioorg.2014.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kusuma, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ung, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottomley, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harry, E. J.</span></span> <span> </span><span class="NLM_article-title">FtsZ as an antibacterial target: status and guidelines for progressing this avenue</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1279</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.9b00055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.9b00055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=1279-1294&author=K.+D.+Kusumaauthor=M.+Payneauthor=A.+T.+Ungauthor=A.+L.+Bottomleyauthor=E.+J.+Harry&title=FtsZ+as+an+antibacterial+target%3A+status+and+guidelines+for+progressing+this+avenue&doi=10.1021%2Facsinfecdis.9b00055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FtsZ as an Antibacterial Target: Status and Guidelines for Progressing This Avenue</span></div><div class="casAuthors">Kusuma, Kennardy D.; Payne, Matthew; Ung, Alison T.; Bottomley, Amy L.; Harry, Elizabeth J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1279-1294</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The disturbing increase in the no. of bacterial pathogens that are resistant to multiple, or sometimes all, current antibiotics highlights the desperate need to pursue the discovery and development of novel classes of antibacterials.  The wealth of knowledge available about the bacterial cell division machinery has aided target-driven approaches to identify new inhibitor compds.  The main division target being pursued is the highly conserved and essential protein, FtsZ.  Despite very active research on FtsZ inhibitors for several years, this protein is not yet targeted by any com. antibiotic.  Here we discuss the suitability of FtsZ as an antibacterial target for drug development, and review progress achieved in this area.  We use hindsight to highlight the gaps that have slowed progress in FtsZ inhibitor development, and to suggest guidelines for concluding that FtsZ is actually the target of these mols., a key missing link in several studies.  In moving forward, a multidisciplinary and a communicative and collaborative process, with sharing of research expertise, is crit. if we are to succeed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NungbAnpqrVg90H21EOLACvtfcHk0ljpddFtvZG72w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSju7zJ&md5=5a867b4471c875047397d393100f287f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.9b00055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.9b00055%26sid%3Dliteratum%253Aachs%26aulast%3DKusuma%26aufirst%3DK.%2BD.%26aulast%3DPayne%26aufirst%3DM.%26aulast%3DUng%26aufirst%3DA.%2BT.%26aulast%3DBottomley%26aufirst%3DA.%2BL.%26aulast%3DHarry%26aufirst%3DE.%2BJ.%26atitle%3DFtsZ%2520as%2520an%2520antibacterial%2520target%253A%2520status%2520and%2520guidelines%2520for%2520progressing%2520this%2520avenue%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D1279%26epage%3D1294%26doi%3D10.1021%2Facsinfecdis.9b00055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haydon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ure, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barynin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedelnikova, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heal, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiwale, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taneja, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czaplewski, L. G.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of FtsZ with potent and selective anti-staphylococcal activity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">1673</span>– <span class="NLM_lpage">1675</span>, <span class="refDoi"> DOI: 10.1126/science.1159961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscience.1159961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18801997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=1673-1675&author=D.+J.+Haydonauthor=N.+R.+Stokesauthor=R.+Ureauthor=G.+Galbraithauthor=J.+M.+Bennettauthor=D.+R.+Brownauthor=P.+J.+Bakerauthor=V.+V.+Baryninauthor=D.+W.+Riceauthor=S.+E.+Sedelnikovaauthor=J.+R.+Healauthor=J.+M.+Sheridanauthor=S.+T.+Aiwaleauthor=P.+K.+Chauhanauthor=A.+Srivastavaauthor=A.+Tanejaauthor=I.+Collinsauthor=J.+Erringtonauthor=L.+G.+Czaplewski&title=An+inhibitor+of+FtsZ+with+potent+and+selective+anti-staphylococcal+activity&doi=10.1126%2Fscience.1159961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Inhibitor of FtsZ with Potent and Selective Anti-Staphylococcal Activity</span></div><div class="casAuthors">Haydon, David J.; Stokes, Neil R.; Ure, Rebecca; Galbraith, Greta; Bennett, James M.; Brown, David R.; Baker, Patrick J.; Barynin, Vladimir V.; Rice, David W.; Sedelnikova, Sveta E.; Heal, Jonathan R.; Sheridan, Joseph M.; Aiwale, Sachin T.; Chauhan, Pramod K.; Srivastava, Anil; Taneja, Amit; Collins, Ian; Errington, Jeff; Czaplewski, Lloyd G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5896</span>),
    <span class="NLM_cas:pages">1673-1675</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">FtsZ is an essential bacterial guanosine triphosphatase and homolog of mammalian β-tubulin that polymerizes and assembles into a ring to initiate cell division.  We have created a class of small synthetic antibacterials, exemplified by PC190723, which inhibits FtsZ and prevents cell division.  PC190723 has potent and selective in vitro bactericidal activity against staphylococci, including methicillin- and multi-drug-resistant Staphylococcus aureus.  The putative inhibitor-binding site of PC190723 was mapped to a region of FtsZ that is analogous to the Taxol-binding site of tubulin.  PC190723 was efficacious in an in vivo model of infection, curing mice infected with a LD of S. aureus.  The data validate FtsZ as a target for antibacterial intervention and identify PC190723 as suitable for optimization into a new anti-staphylococcal therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmbbLDvOUp8LVg90H21EOLACvtfcHk0ljpddFtvZG72w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtrrJ&md5=6573e2e19954f44e7632c99c5d8e396c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.1159961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1159961%26sid%3Dliteratum%253Aachs%26aulast%3DHaydon%26aufirst%3DD.%2BJ.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DUre%26aufirst%3DR.%26aulast%3DGalbraith%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBrown%26aufirst%3DD.%2BR.%26aulast%3DBaker%26aufirst%3DP.%2BJ.%26aulast%3DBarynin%26aufirst%3DV.%2BV.%26aulast%3DRice%26aufirst%3DD.%2BW.%26aulast%3DSedelnikova%26aufirst%3DS.%2BE.%26aulast%3DHeal%26aufirst%3DJ.%2BR.%26aulast%3DSheridan%26aufirst%3DJ.%2BM.%26aulast%3DAiwale%26aufirst%3DS.%2BT.%26aulast%3DChauhan%26aufirst%3DP.%2BK.%26aulast%3DSrivastava%26aufirst%3DA.%26aulast%3DTaneja%26aufirst%3DA.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DErrington%26aufirst%3DJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26atitle%3DAn%2520inhibitor%2520of%2520FtsZ%2520with%2520potent%2520and%2520selective%2520anti-staphylococcal%2520activity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D1673%26epage%3D1675%26doi%3D10.1126%2Fscience.1159961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therien, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeau-Jacob, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benton-Perdomo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petcu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigneault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claveau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaillancourt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skorey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meredith, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sillaots, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang-Jarantow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramtohul, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baryshnikova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinho, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Restoring methicillin-resistant Staphylococcus aureus susceptibility to β-lactam antibiotics</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">126ra135</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.3003592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscitranslmed.3003592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&author=C.+M.+Tanauthor=A.+G.+Therienauthor=J.+Luauthor=S.+H.+Leeauthor=A.+Caronauthor=C.+J.+Gillauthor=C.+Lebeau-Jacobauthor=L.+Benton-Perdomoauthor=J.+M.+Monteiroauthor=P.+M.+Pereiraauthor=N.+L.+Elsenauthor=J.+Wuauthor=K.+Deschampsauthor=M.+Petcuauthor=S.+Wongauthor=E.+Daigneaultauthor=S.+Kramerauthor=L.+Liangauthor=E.+Maxwellauthor=D.+Claveauauthor=J.+Vaillancourtauthor=K.+Skoreyauthor=J.+Tamauthor=H.+Wangauthor=T.+C.+Meredithauthor=S.+Sillaotsauthor=L.+Wang-Jarantowauthor=Y.+Ramtohulauthor=E.+Langloisauthor=F.+Landryauthor=J.+C.+Reidauthor=G.+Parthasarathyauthor=S.+Sharmaauthor=A.+Baryshnikovaauthor=K.+J.+Lumbauthor=M.+G.+Pinhoauthor=S.+M.+Soissonauthor=T.+Roemer&title=Restoring+methicillin-resistant+Staphylococcus+aureus+susceptibility+to+%CE%B2-lactam+antibiotics&doi=10.1126%2Fscitranslmed.3003592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3003592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3003592%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DC.%2BM.%26aulast%3DTherien%26aufirst%3DA.%2BG.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DCaron%26aufirst%3DA.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DLebeau-Jacob%26aufirst%3DC.%26aulast%3DBenton-Perdomo%26aufirst%3DL.%26aulast%3DMonteiro%26aufirst%3DJ.%2BM.%26aulast%3DPereira%26aufirst%3DP.%2BM.%26aulast%3DElsen%26aufirst%3DN.%2BL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDeschamps%26aufirst%3DK.%26aulast%3DPetcu%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DDaigneault%26aufirst%3DE.%26aulast%3DKramer%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DMaxwell%26aufirst%3DE.%26aulast%3DClaveau%26aufirst%3DD.%26aulast%3DVaillancourt%26aufirst%3DJ.%26aulast%3DSkorey%26aufirst%3DK.%26aulast%3DTam%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DMeredith%26aufirst%3DT.%2BC.%26aulast%3DSillaots%26aufirst%3DS.%26aulast%3DWang-Jarantow%26aufirst%3DL.%26aulast%3DRamtohul%26aufirst%3DY.%26aulast%3DLanglois%26aufirst%3DE.%26aulast%3DLandry%26aufirst%3DF.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DBaryshnikova%26aufirst%3DA.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DPinho%26aufirst%3DM.%2BG.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DRestoring%2520methicillin-resistant%2520Staphylococcus%2520aureus%2520susceptibility%2520to%2520%25CE%25B2-lactam%2520antibiotics%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2012%26volume%3D4%26doi%3D10.1126%2Fscitranslmed.3003592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casiraghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suigo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valoti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straniero, V.</span></span> <span> </span><span class="NLM_article-title">Targeting bacterial cell division: a binding-site centered approach to the most promising inhibitors of the essential protein FtsZ</span>. <i>Antibiotics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">69</span> <span class="refDoi"> DOI: 10.3390/antibiotics9020069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3390%2Fantibiotics9020069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1OitLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=A.+Casiraghiauthor=L.+Suigoauthor=E.+Valotiauthor=V.+Straniero&title=Targeting+bacterial+cell+division%3A+a+binding-site+centered+approach+to+the+most+promising+inhibitors+of+the+essential+protein+FtsZ&doi=10.3390%2Fantibiotics9020069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bacterial cell division: a binding site-centered approach to the most promising inhibitors of the essential protein FtsZ</span></div><div class="casAuthors">Casiraghi, Andrea; Suigo, Lorenzo; Valoti, Ermanno; Straniero, Valentina</div><div class="citationInfo"><span class="NLM_cas:title">Antibiotics (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">69</span>CODEN:
                <span class="NLM_cas:coden">ABSNC4</span>;
        ISSN:<span class="NLM_cas:issn">2079-6382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Binary fission is the most common mode of bacterial cell division and is mediated by a multiprotein complex denominated the divisome.  The constriction of the Z-ring splits the mother bacterial cell into two daughter cells of the same size.  The Z-ring is formed by the polymn. of FtsZ, a bacterial protein homolog of eukaryotic tubulin, and it represents the first step of bacterial cytokinesis.  The high grade of conservation of FtsZ in most prokaryotic organisms and its relevance in orchestrating the whole division system make this protein a fascinating target in antibiotic research.  Indeed, FtsZ inhibition results in the complete blockage of the division system and, consequently, in a bacteriostatic or a bactericidal effect.  Since many papers and reviews already discussed the physiol. of FtsZ and its auxiliary proteins, as well as the mol. mechanisms in which they are involved, here, we focus on the discussion of the most compelling FtsZ inhibitors, classified by their main protein binding sites and following a medicinal chem. approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmzHUPYa5suLVg90H21EOLACvtfcHk0lhbm2X-7Zncrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1OitLzP&md5=6d5eb79696b35059f1e6266902c54f4b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Fantibiotics9020069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantibiotics9020069%26sid%3Dliteratum%253Aachs%26aulast%3DCasiraghi%26aufirst%3DA.%26aulast%3DSuigo%26aufirst%3DL.%26aulast%3DValoti%26aufirst%3DE.%26aulast%3DStraniero%26aufirst%3DV.%26atitle%3DTargeting%2520bacterial%2520cell%2520division%253A%2520a%2520binding-site%2520centered%2520approach%2520to%2520the%2520most%2520promising%2520inhibitors%2520of%2520the%2520essential%2520protein%2520FtsZ%26jtitle%3DAntibiotics%26date%3D2020%26volume%3D9%26doi%3D10.3390%2Fantibiotics9020069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoie, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, D. S.</span></span> <span> </span><span class="NLM_article-title">Combining the FtsZ-targeting prodrug TXA709 and the cephalosporin cefdinir confers synergy and reduces the frequency of resistance in methicillin-resistant Staphylococcus aureus</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4290</span>– <span class="NLM_lpage">4296</span>, <span class="refDoi"> DOI: 10.1128/AAC.00613-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1128%2FAAC.00613-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=27161635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=4290-4296&author=M.+Kaulauthor=L.+Markauthor=A.+K.+Parhiauthor=E.+J.+LaVoieauthor=D.+S.+Pilch&title=Combining+the+FtsZ-targeting+prodrug+TXA709+and+the+cephalosporin+cefdinir+confers+synergy+and+reduces+the+frequency+of+resistance+in+methicillin-resistant+Staphylococcus+aureus&doi=10.1128%2FAAC.00613-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Combining the FtsZ-targeting prodrug TXA709 and the cephalosporin cefdinir confers synergy and reduces the frequency of resistance in methicillin-resistant Staphylococcus aureus</span></div><div class="casAuthors">Kaul, Malvika; Mark, Lilly; Parhi, Ajit K.; LaVoie, Edmond J.; Pilch, Daniel S.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4296</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Combination therapy of bacterial infections with synergistic drug partners offers distinct advantages over monotherapy.  Among these advantages are (1) redn. of the drug dose required for efficacy, (2) reduced potential for drug-induced toxicity, and (3) reduced potential for the emergence of resistance.  Here, we describe the synergistic actions of the 3rd-generation oral cephalosporin cefdinir and TXA709, a new, FtsZ-targeting prodrug that we have developed with improved pharmacokinetics and enhanced in vivo efficacy against methicillin-resistant Staphylococcus aureus (MRSA) relative to earlier agents.  The active product of TXA709 (TXA707) acts synergistically with cefdinir in vitro against clin. isolates of MRSA, vancomycin-intermediate S. aureus (VISA), vancomycin-resistant S. aureus (VRSA), and linezolid-resistant S. aureus (LRSA).  In addn., relative to TXA707 alone, the combination of TXA707 and cefdinir significantly reduces or eliminates the detectable emergence of resistance.  We also demonstrate synergy in vivo with oral administration of the prodrug TXA709 and cefdinir in mouse models of both systemic and tissue (thigh) infections with MRSA.  This synergy reduces the dose of TXA709 required for efficacy 3-fold.  Viewed as a whole, our results highlight the potential of TXA709 and cefdinir as a promising combination for the treatment of drug-resistant staphylococcal infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzb-r4hn823bVg90H21EOLACvtfcHk0lhbm2X-7Zncrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyksL4%253D&md5=a124444e0323bc13e8f5e664a0bdd130</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FAAC.00613-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00613-16%26sid%3Dliteratum%253Aachs%26aulast%3DKaul%26aufirst%3DM.%26aulast%3DMark%26aufirst%3DL.%26aulast%3DParhi%26aufirst%3DA.%2BK.%26aulast%3DLaVoie%26aufirst%3DE.%2BJ.%26aulast%3DPilch%26aufirst%3DD.%2BS.%26atitle%3DCombining%2520the%2520FtsZ-targeting%2520prodrug%2520TXA709%2520and%2520the%2520cephalosporin%2520cefdinir%2520confers%2520synergy%2520and%2520reduces%2520the%2520frequency%2520of%2520resistance%2520in%2520methicillin-resistant%2520Staphylococcus%2520aureus%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D4290%26epage%3D4296%26doi%3D10.1128%2FAAC.00613-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span>Taxis
Pharmaceuticals Pipeline, <span class="NLM_year">2020</span>. Can be found under: <a href="https://www.taxispharma.com/research-development/our-pipeline/" class="extLink">https://www.taxispharma.com/research-development/our-pipeline/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Taxis%0APharmaceuticals+Pipeline%2C+2020.+Can+be+found+under%3A+https%3A%2F%2Fwww.taxispharma.com%2Fresearch-development%2Four-pipeline%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llorca, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boskovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Benito, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valpuesta, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Energetics and geometry of FtsZ polymers: nucleated self-assembly of single protofilaments</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1796</span>– <span class="NLM_lpage">1806</span>, <span class="refDoi"> DOI: 10.1529/biophysj.107.115493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1529%2Fbiophysj.107.115493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18024502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXit1yqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2008&pages=1796-1806&author=S.+Huecasauthor=O.+Llorcaauthor=J.+Boskovicauthor=J.+Martin-Benitoauthor=J.+M.+Valpuestaauthor=J.+M.+Andreu&title=Energetics+and+geometry+of+FtsZ+polymers%3A+nucleated+self-assembly+of+single+protofilaments&doi=10.1529%2Fbiophysj.107.115493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Energetics and geometry of FtsZ polymers: nucleated self-assembly of single protofilaments</span></div><div class="casAuthors">Huecas, Sonia; Llorca, Oscar; Boskovic, Jasminka; Martin-Benito, Jaime; Valpuesta, Jose Maria; Andreu, Jose Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1796-1806</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">Essential cell division protein FtsZ is an assembling GTPase which directs the cytokinetic ring formation in dividing bacterial cells.  FtsZ shares the structural fold of eukaryotic tubulin and assembles forming tubulin-like protofilaments, but does not form microtubules.  Two puzzling problems in FtsZ assembly are the nature of protofilament assocn. and a possible mechanism for nucleated self-assembly of single-stranded protofilaments above a crit. FtsZ concn.  We assembled two-dimensional arrays of FtsZ on carbon supports, studied linear polymers of FtsZ with cryo-electron microscopy of vitrified unsupported solns., and formulated possible polymn. models.  Nucleated self-assembly of FtsZ from Escherichia coli with GTP and magnesium produces flexible filaments 4-6 nm-wide, only compatible with a single protofilament.  This agrees with previous scanning TEM results and is supported by recent cryo-electron tomog. studies of two bacterial cells.  Observations of double-stranded FtsZ filaments in neg. stain may come from protofilament accretion on the carbon support.  Preferential protofilament cyclization does not apply to FtsZ assembly.  The apparently cooperative polymn. of a single protofilament with identical intermonomer contacts is explained by the switching of one inactive monomer into the active structure preceding assocn. of the next, creating a dimer nucleus.  FtsZ behaves as a cooperative linear assembly machine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZhX8LMlXWbVg90H21EOLACvtfcHk0lhb_TpOnEZ2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXit1yqurw%253D&md5=4d778a930ba385b1843d64231d54cf99</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.107.115493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.107.115493%26sid%3Dliteratum%253Aachs%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DLlorca%26aufirst%3DO.%26aulast%3DBoskovic%26aufirst%3DJ.%26aulast%3DMartin-Benito%26aufirst%3DJ.%26aulast%3DValpuesta%26aufirst%3DJ.%2BM.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DEnergetics%2520and%2520geometry%2520of%2520FtsZ%2520polymers%253A%2520nucleated%2520self-assembly%2520of%2520single%2520protofilaments%26jtitle%3DBiophys.%2520J.%26date%3D2008%26volume%3D94%26spage%3D1796%26epage%3D1806%26doi%3D10.1529%2Fbiophysj.107.115493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dajkovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutkenhaus, J.</span></span> <span> </span><span class="NLM_article-title">Investigation of regulation of FtsZ assembly by SulA and development of a model for FtsZ polymerization</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">2513</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1128/JB.01612-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1128%2FJB.01612-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18245292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVylsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2008&pages=2513-2526&author=A.+Dajkovicauthor=A.+Mukherjeeauthor=J.+Lutkenhaus&title=Investigation+of+regulation+of+FtsZ+assembly+by+SulA+and+development+of+a+model+for+FtsZ+polymerization&doi=10.1128%2FJB.01612-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of regulation of FtsZ assembly by SulA and development of a model for FtsZ polymerization</span></div><div class="casAuthors">Dajkovic, Alex; Mukherjee, Amit; Lutkenhaus, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2513-2526</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In Escherichia coli FtsZ organizes into a cytoskeletal ring structure, the Z ring, which effects cell division.  FtsZ is a GTPase, but the free energy of GTP hydrolysis does not appear to be used for generation of the constriction force, leaving open the question of the function of the GTPase activity of FtsZ.  Here we study the mechanism by which SulA, an inhibitor of FtsZ induced during the SOS response, inhibits FtsZ function.  We studied the effects of SulA on the in vitro activities of FtsZ, on Z rings in vivo, and on a kinetic model for FtsZ polymn. in silico.  We found that the binding of SulA to FtsZ is necessary but not sufficient for inhibition of polymn., since the assembly of FtsZ polymers in the absence of the GTPase activity was not inhibited by SulA.  We developed a new model for FtsZ polymn. that accounts for the cooperativity of FtsZ and could account for cooperativity obsd. in other linear polymers.  When SulA was included in the kinetic scheme, simulations revealed that SulA with strong affinity for FtsZ delayed, but did not prevent, the assembly of polymers when they were not hydrolyzing GTP.  Furthermore, the simulations indicated that SulA controls the assembly of FtsZ by binding to a polymn.-competent form of the FtsZ mol. and preventing it from participating in assembly.  In vivo stoichiometry of the disruption of Z rings by SulA suggests that FtsZ may undergo two cooperative transitions in forming the Z ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQXpzN21oQVLVg90H21EOLACvtfcHk0lhb_TpOnEZ2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVylsr4%253D&md5=1c2f4f19f1cc1a482226ebc2e47fccd7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FJB.01612-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.01612-07%26sid%3Dliteratum%253Aachs%26aulast%3DDajkovic%26aufirst%3DA.%26aulast%3DMukherjee%26aufirst%3DA.%26aulast%3DLutkenhaus%26aufirst%3DJ.%26atitle%3DInvestigation%2520of%2520regulation%2520of%2520FtsZ%2520assembly%2520by%2520SulA%2520and%2520development%2520of%2520a%2520model%2520for%2520FtsZ%2520polymerization%26jtitle%3DJ.%2520Bacteriol.%26date%3D2008%26volume%3D190%26spage%3D2513%26epage%3D2526%26doi%3D10.1128%2FJB.01612-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miraldi, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romberg, L.</span></span> <span> </span><span class="NLM_article-title">Allosteric models for cooperative polymerization of linear polymers</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">2470</span>– <span class="NLM_lpage">2486</span>, <span class="refDoi"> DOI: 10.1529/biophysj.107.126219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1529%2Fbiophysj.107.126219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18502809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGqsr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2008&pages=2470-2486&author=E.+R.+Miraldiauthor=P.+J.+Thomasauthor=L.+Romberg&title=Allosteric+models+for+cooperative+polymerization+of+linear+polymers&doi=10.1529%2Fbiophysj.107.126219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric models for cooperative polymerization of linear polymers</span></div><div class="casAuthors">Miraldi, Emily R.; Thomas, Peter J.; Romberg, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2470-2486</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">In the cytoskeleton, unfavorable nucleation steps allow cells to regulate where, when, and how many polymers assemble.  Nucleated polymn. is traditionally explained by a model in which multistranded polymers assemble cooperatively, whereas linear, single-stranded polymers do not.  Recent data on the assembly of FtsZ, the bacterial homolog of tubulin, do not fit either category.  FtsZ can polymerize into single-stranded protofilaments that are stable in the absence of lateral interactions, but that assemble cooperatively.  We developed a model for cooperative polymn. that does not require polymers to be multistranded.  Instead, a conformational change allows subunits in oligomers to assoc. with high affinity, whereas a lower-affinity conformation is favored in monomers.  We derive equations for calcg. polymer concns., subunit conformations, and the apparent affinity of subunits for polymer ends.  Certain combinations of equil. consts. produce the sharp crit. concns. characteristic of cooperative polymn.  In these cases, the low-affinity conformation predominates in monomers, whereas virtually all polymers are composed of high-affinity subunits.  Our model predicts that the three routes to forming HH dimers all involve unstable intermediates, limiting nucleation.  The math. framework developed here can represent allosteric assembly systems with a variety of biochem. interpretations, some of which can show cooperativity, and others of which cannot.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI1xJvmvFr5rVg90H21EOLACvtfcHk0lhb_TpOnEZ2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGqsr3O&md5=04dae4360e5767ded052b9dc081e59a6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.107.126219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.107.126219%26sid%3Dliteratum%253Aachs%26aulast%3DMiraldi%26aufirst%3DE.%2BR.%26aulast%3DThomas%26aufirst%3DP.%2BJ.%26aulast%3DRomberg%26aufirst%3DL.%26atitle%3DAllosteric%2520models%2520for%2520cooperative%2520polymerization%2520of%2520linear%2520polymers%26jtitle%3DBiophys.%2520J.%26date%3D2008%26volume%3D95%26spage%3D2470%26epage%3D2486%26doi%3D10.1529%2Fbiophysj.107.126219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, I.</span></span> <span> </span><span class="NLM_article-title">Structural change in FtsZ induced by intermolecular interactions between bound GTP and the T7 loop</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">3501</span>– <span class="NLM_lpage">3509</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.514901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M113.514901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=24347164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2ltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=3501-3509&author=T.+Matsuiauthor=X.+Hanauthor=R.+J.+Yuauthor=M.+Yaoauthor=I.+Tanaka&title=Structural+change+in+FtsZ+induced+by+intermolecular+interactions+between+bound+GTP+and+the+T7+loop&doi=10.1074%2Fjbc.M113.514901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural change in FtsZ induced by intermolecular interactions between bound GTP and the T7 loop</span></div><div class="casAuthors">Matsui, Takashi; Han, Xuerong; Yu, Jian; Yao, Min; Tanaka, Isao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3501-3509</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">FtsZ is a prokaryotic homolog of tubulin and is a key mol. in bacterial cell division.  FtsZ with bound GTP polymerizes into tubulin-like protofilaments.  Upon polymn., the T7 loop of one subunit is inserted into the nucleotide-binding pocket of the 2nd subunit, which results in GTP hydrolysis.  Thus, the T7 loop is important for both polymn. and hydrolysis in the tubulin/FtsZ family.  Although x-ray crystallog. revealed both straight and curved conformations of tubulin, only a curved structure was known for FtsZ.  Recently, however, FtsZ from Staphylococcus aureus has been shown to have a very different conformation from the canonical FtsZ structure.  The present study was performed to investigate the structure of FtsZ from Staphylococcus aureus by mutagenesis expts.; the effects of amino acid changes in the T7 loop on the structure as well as on GTPase activity were studied.  These analyses indicated that FtsZ changes its conformation suitable for polymn. and GTP hydrolysis by movement between N- and C-subdomains via intermol. interactions between bound nucleotide and residues in the T7 loop.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWwvoI8Jo8srVg90H21EOLACvtfcHk0lhx23Ga1Vm96w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2ltro%253D&md5=e91683c358b4cada51a79642586153de</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.514901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.514901%26sid%3Dliteratum%253Aachs%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DR.%2BJ.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DI.%26atitle%3DStructural%2520change%2520in%2520FtsZ%2520induced%2520by%2520intermolecular%2520interactions%2520between%2520bound%2520GTP%2520and%2520the%2520T7%2520loop%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D3501%26epage%3D3509%26doi%3D10.1074%2Fjbc.M113.514901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez-Ramirez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llorca, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Galiano, A. J.</span></span> <span> </span><span class="NLM_article-title">The antibacterial cell division inhibitor PC190723 is an FtsZ polymer-stabilizing agent that induces filament assembly and condensation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">14239</span>– <span class="NLM_lpage">14246</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.094722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M109.094722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20212044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVOiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=14239-14246&author=J.+M.+Andreuauthor=C.+Schaffner-Barberoauthor=S.+Huecasauthor=D.+Alonsoauthor=M.+L.+Lopez-Rodriguezauthor=L.+B.+Ruiz-Avilaauthor=R.+Nu%C3%B1ez-Ramirezauthor=O.+Llorcaauthor=A.+J.+Martin-Galiano&title=The+antibacterial+cell+division+inhibitor+PC190723+is+an+FtsZ+polymer-stabilizing+agent+that+induces+filament+assembly+and+condensation&doi=10.1074%2Fjbc.M109.094722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Antibacterial Cell Division Inhibitor PC190723 Is an FtsZ Polymer-stabilizing Agent That Induces Filament Assembly and Condensation</span></div><div class="casAuthors">Andreu, Jose M.; Schaffner-Barbero, Claudia; Huecas, Sonia; Alonso, Dulce; Lopez-Rodriguez, Maria L.; Ruiz-Avila, Laura B.; Nunez-Ramirez, Rafael; Llorca, Oscar; Martin-Galiano, Antonio J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">14239-14246</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cell division protein FtsZ can form single-stranded filaments with a cooperative behavior by self-switching assembly.  Subsequent condensation and bending of FtsZ filaments are important for the formation and constriction of the cytokinetic ring.  PC190723 is an effective bactericidal cell division inhibitor that targets FtsZ in the pathogen Staphylococcus aureus and Bacillus subtilis and does not affect Escherichia coli cells, which apparently binds to a zone equiv. to the binding site of the antitumor drug taxol in tubulin.  We have found that the benzamide deriv. PC190723 is an FtsZ polymer-stabilizing agent.  PC190723 induced nucleated assembly of Bs-FtsZ into single-stranded coiled protofilaments and polymorphic condensates, including bundles, coils, and toroids, whose formation could be modulated with different soln. conditions.  Under conditions for reversible assembly of Bs-FtsZ, PC190723 binding reduced the GTPase activity and induced the formation of straight bundles and ribbons, which was also obsd. with Sa-FtsZ but not with nonsusceptible Ec-FtsZ.  The fragment 2,6-difluoro-3-methoxybenzamide also induced Bs-FtsZ bundling.  We propose that polymer stabilization by PC190723 suppresses in vivo FtsZ polymer dynamics and bacterial division.  The biochem. action of PC190723 on FtsZ parallels that of the microtubule-stabilizing agent taxol on the eukaryotic structural homolog tubulin.  Both taxol and PC190723 stabilize polymers against disassembly by preferential binding to each assembled protein.  It is yet to be investigated whether both ligands target structurally related assembly switches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdWKjWSpN8rVg90H21EOLACvtfcHk0lhx23Ga1Vm96w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVOiurs%253D&md5=d6fc5ba584a2d89c44dcf6809a4abf90</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.094722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.094722%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAlonso%26aufirst%3DD.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DNu%25C3%25B1ez-Ramirez%26aufirst%3DR.%26aulast%3DLlorca%26aufirst%3DO.%26aulast%3DMartin-Galiano%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520antibacterial%2520cell%2520division%2520inhibitor%2520PC190723%2520is%2520an%2520FtsZ%2520polymer-stabilizing%2520agent%2520that%2520induces%2520filament%2520assembly%2520and%2520condensation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D14239%26epage%3D14246%26doi%3D10.1074%2Fjbc.M109.094722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span> <span> </span><span class="NLM_article-title">Mechanism of action of the cell-division inhibitor PC190723: modulation of FtsZ assembly cooperativity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">12342</span>– <span class="NLM_lpage">12345</span>, <span class="refDoi"> DOI: 10.1021/ja303564a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303564a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVejtbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=12342-12345&author=N.+L.+Elsenauthor=J.+Luauthor=G.+Parthasarathyauthor=J.+C.+Reidauthor=S.+Sharmaauthor=S.+M.+Soissonauthor=K.+J.+Lumb&title=Mechanism+of+action+of+the+cell-division+inhibitor+PC190723%3A+modulation+of+FtsZ+assembly+cooperativity&doi=10.1021%2Fja303564a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Action of the Cell-Division Inhibitor PC190723: Modulation of FtsZ Assembly Cooperativity</span></div><div class="casAuthors">Elsen, Nathaniel L.; Lu, Jun; Parthasarathy, Gopal; Reid, John C.; Sharma, Sujata; Soisson, Stephen M.; Lumb, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">12342-12345</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cooperative assembly of FtsZ, the prokaryotic homolog of tubulin, plays an essential role in cell division.  FtsZ is a potential drug target, as illustrated by the small-mol. cell-cycle inhibitor and antibacterial agent PC190723 that targets FtsZ.  We demonstrate that PC190723 neg. modulates Staphylococcus aureus FtsZ polymn. cooperativity as reflected in polymn. at lower concns. without a defined crit. concn.  The crystal structure of the S. aureus FtsZ-PC190723 complex shows a domain movement that would stabilize the FtsZ protofilament over the monomeric state, with the conformational change mediated from the GTP-binding site to the C-terminal domain via helix 7.  Together, the results reveal the mol. mechanism of FtsZ modulation by PC190723 and a conformational switch to the high-affinity state that enables polymer assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWzgrLJ8OPQLVg90H21EOLACvtfcHk0lgltLrnUcjE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVejtbzJ&md5=052b9be970a494eb3524448ab967cc6a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fja303564a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303564a%26sid%3Dliteratum%253Aachs%26aulast%3DElsen%26aufirst%3DN.%2BL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26atitle%3DMechanism%2520of%2520action%2520of%2520the%2520cell-division%2520inhibitor%2520PC190723%253A%2520modulation%2520of%2520FtsZ%2520assembly%2520cooperativity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D12342%26epage%3D12345%26doi%3D10.1021%2Fja303564a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, I.</span></span> <span> </span><span class="NLM_article-title">Structural reorganization of the bacterial cell-division protein FtsZ from <i>Staphylococcus aureus</i></span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">1175</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1107/S0907444912022640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444912022640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=22948918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1yqtbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=1175-1188&author=T.+Matsuiauthor=J.+Yamaneauthor=N.+Mogiauthor=H.+Yamaguchiauthor=H.+Takemotoauthor=M.+Yaoauthor=I.+Tanaka&title=Structural+reorganization+of+the+bacterial+cell-division+protein+FtsZ+from+Staphylococcus+aureus&doi=10.1107%2FS0907444912022640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural reorganization of the bacterial cell-division protein FtsZ from Staphylococcus aureus</span></div><div class="casAuthors">Matsui, Takashi; Yamane, Junji; Mogi, Nobuyuki; Yamaguchi, Hiroto; Takemoto, Hiroshi; Yao, Min; Tanaka, Isao</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1175-1188</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">FtsZ is a key mol. in bacterial cell division.  In the presence of GTP, it polymerizes into tubulin-like protofilaments by head-to-tail assocn.  Protofilaments of FtsZ seem to adopt a straight or a curved conformation in relation to the bound nucleotide.  However, although several bacterial and archaeal FtsZ structures have been detd., all of the structures reported previously are considered to have a curved conformation.  In this study, structures of FtsZ from Staphylococcus aureus (SaFtsZ) were detd. in apo, GDP-bound and inhibitor-complex forms and it was found that SaFtsZ undergoes marked conformational changes.  The accumulated evidence suggests that the GDP-bound structure has the features of the straight form.  The structural change between the curved and straight forms shows intriguing similarity to the eukaryotic cytoskeletal protein tubulin.  Furthermore, the structure of the apo form showed an unexpectedly large conformational change in the core region.  FtsZ has also been recognized as a novel target for antibacterial drugs.  The structure of the complex with the inhibitor PC190723, which has potent and selective anti-staphylococcal activity, indicated that the inhibitor binds at the cleft between the two subdomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr36sS4BLhTmbVg90H21EOLACvtfcHk0lgltLrnUcjE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1yqtbjJ&md5=c06bbdec17d75b9173deaf2f8a262b91</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444912022640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912022640%26sid%3Dliteratum%253Aachs%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DYamane%26aufirst%3DJ.%26aulast%3DMogi%26aufirst%3DN.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DI.%26atitle%3DStructural%2520reorganization%2520of%2520the%2520bacterial%2520cell-division%2520protein%2520FtsZ%2520from%2520Staphylococcus%2520aureus%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D1175%26epage%3D1188%26doi%3D10.1107%2FS0907444912022640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Löwe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amos, L. A.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the bacterial cell-division protein FtsZ</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1038/34472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2F34472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=9428770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADyaK1cXmt1KisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1998&pages=203-206&author=J.+L%C3%B6weauthor=L.+A.+Amos&title=Crystal+structure+of+the+bacterial+cell-division+protein+FtsZ&doi=10.1038%2F34472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the bacterial cell-division protein FtsZ</span></div><div class="casAuthors">Lowe, Jan; Amos, Linda A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">6663</span>),
    <span class="NLM_cas:pages">203-206</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Bacterial cell division ends with septation, the constriction of the cell wall and cell membranes that leads to the formation of two daughter cells.  During septation, FtsZ, a protein of relative mol. mass 40,000 which is ubiquitous in eubacteria and is also found in archaea and chloroplasts, localizes early at the division site to form a ring-shaped septum.  This septum is required for the mechanochem. process of membrane constriction.  FtsZ is a GTPase with weak sequence homol. to tubulins.  The nature of FtsZ polymers in vivo is unknown, but FtsZ can form tubules, sheets, and minirings in vitro.  Here, the authors report the crystal structure at 2.8 Å resoln. of recombinant FtsZ from the hyperthermophilic methanogen Methanococcus jannaschii.  FtsZ has two domains, one of which is a GTPase domain with a fold related to one found in the proteins p21ras and elongation factor EF-Tu.  The carboxy-terminal domain, whose function is unknown, is a four-stranded β-sheet tilted by 90° against the β-sheet of the GTPase domain.  The two domains are arranged around a central helix.  GDP binding is different from that typically found in GTPases and involves four phosphate-binding loops and a sugar-binding loop in the first domain, with guanine being recognized by residues in the central connecting helix.  The three-dimensional structure of FtsZ is similar to the structure of α- and β-tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIoEQ_Zfl-LVg90H21EOLACvtfcHk0lgltLrnUcjE2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmt1KisA%253D%253D&md5=21ed6e85c5ea92524bdfecdc6028f448</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2F34472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F34472%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6we%26aufirst%3DJ.%26aulast%3DAmos%26aufirst%3DL.%2BA.%26atitle%3DCrystal%2520structure%2520of%2520the%2520bacterial%2520cell-division%2520protein%2520FtsZ%26jtitle%3DNature%26date%3D1998%26volume%3D391%26spage%3D203%26epage%3D206%26doi%3D10.1038%2F34472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araujo-Bazan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Villa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Galiano, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span> <span> </span><span class="NLM_article-title">The structural assembly switch of cell division protein FtsZ probed with fluorescent allosteric inhibitors</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1534</span>, <span class="refDoi"> DOI: 10.1039/C6SC03792E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1039%2FC6SC03792E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28616148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslalsbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1525-1534&author=M.+Artolaauthor=L.+B.+Ruiz-Avilaauthor=E.+Ramirez-Aportelaauthor=R.+F.+Martinezauthor=L.+Araujo-Bazanauthor=H.+Vazquez-Villaauthor=M.+Martin-Fontechaauthor=M.+A.+Olivaauthor=A.+J.+Martin-Galianoauthor=P.+Chaconauthor=M.+L.+Lopez-Rodriguezauthor=J.+M.+Andreuauthor=S.+Huecas&title=The+structural+assembly+switch+of+cell+division+protein+FtsZ+probed+with+fluorescent+allosteric+inhibitors&doi=10.1039%2FC6SC03792E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The structural assembly switch of cell division protein FtsZ probed with fluorescent allosteric inhibitors</span></div><div class="casAuthors">Artola, Marta; Ruiz-Avila, Laura B.; Ramirez-Aportela, Erney; Martinez, R. Fernando; Araujo-Bazan, Lidia; Vazquez-Villa, Henar; Martin-Fontecha, Mar; Oliva, Maria A.; Martin-Galiano, A. Javier; Chacon, Pablo; Lopez-Rodriguez, Maria L.; Andreu, Jose M.; Huecas, Sonia</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1525-1534</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">FtsZ is a widely conserved tubulin-like GTPase that directs bacterial cell division and a new target for antibiotic discovery.  This protein assembly machine cooperatively polymerizes forming single-stranded filaments, by means of self-switching between inactive and actively assocg. monomer conformations.  The structural switch mechanism was proposed to involve a movement of the C-terminal and N-terminal FtsZ domains, opening a cleft between them, allosterically coupled to the formation of a tight assocn. interface between consecutive subunits along the filament.  The effective antibacterial benzamide PC190723 binds into the open interdomain cleft and stabilizes FtsZ filaments, thus impairing correct formation of the FtsZ ring for cell division.  We have designed fluorescent analogs of PC190723 to probe the FtsZ structural assembly switch.  Among them, nitrobenzoxadiazole probes specifically bind to assembled FtsZ rather than to monomers.  Probes with several spacer lengths between the fluorophore and benzamide moieties suggest a binding site extension along the interdomain cleft.  These probes label FtsZ rings of live Bacillus subtilis and Staphylococcus aureus, without apparently modifying normal cell morphol. and growth, but at high concns. they induce impaired bacterial division phenotypes typical of benzamide antibacterials.  During the FtsZ assembly-disassembly process, the fluorescence anisotropy of the probes changes upon binding and dissocg. from FtsZ, thus reporting open and closed FtsZ interdomain clefts.  Our results demonstrate the structural mechanism of the FtsZ assembly switch, and suggest that the probes bind into the open clefts in cellular FtsZ polymers preferably to unassembled FtsZ in the bacterial cytosol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCMy2u0m10FrVg90H21EOLACvtfcHk0lgaI_9UudP_Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslalsbbL&md5=54bba97e4d822c4e96d70e4b82be65bd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC6SC03792E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC03792E%26sid%3Dliteratum%253Aachs%26aulast%3DArtola%26aufirst%3DM.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DRamirez-Aportela%26aufirst%3DE.%26aulast%3DMartinez%26aufirst%3DR.%2BF.%26aulast%3DAraujo-Bazan%26aufirst%3DL.%26aulast%3DVazquez-Villa%26aufirst%3DH.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DOliva%26aufirst%3DM.%2BA.%26aulast%3DMartin-Galiano%26aufirst%3DA.%2BJ.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DHuecas%26aufirst%3DS.%26atitle%3DThe%2520structural%2520assembly%2520switch%2520of%2520cell%2520division%2520protein%2520FtsZ%2520probed%2520with%2520fluorescent%2520allosteric%2520inhibitors%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D1525%26epage%3D1534%26doi%3D10.1039%2FC6SC03792E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizohata, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shigeta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, H.</span></span> <span> </span><span class="NLM_article-title">Identification of the key interactions in structural transition pathway of FtsZ from Staphylococcus aureus</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2017.04.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.jsb.2017.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28456664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A280%3ADC%252BC1crisFejuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=65-73&author=J.+Fujitaauthor=R.+Haradaauthor=Y.+Maedaauthor=Y.+Saitoauthor=E.+Mizohataauthor=T.+Inoueauthor=Y.+Shigetaauthor=H.+Matsumura&title=Identification+of+the+key+interactions+in+structural+transition+pathway+of+FtsZ+from+Staphylococcus+aureus&doi=10.1016%2Fj.jsb.2017.04.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of the key interactions in structural transition pathway of FtsZ from Staphylococcus aureus</span></div><div class="casAuthors">Fujita Junso; Maeda Yoko; Saito Yuki; Mizohata Eiichi; Inoue Tsuyoshi; Harada Ryuhei; Shigeta Yasuteru; Matsumura Hiroyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of structural biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">65-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The tubulin-homolog protein FtsZ is essential for bacterial cell division.  FtsZ polymerizes to form protofilaments that assemble into a contractile ring-shaped structure in the presence of GTP.  Recent studies showed that FtsZ treadmilling coupled with the GTPase activity drives cell wall synthesis and bacterial cell division.  The treadmilling caused by assembly and disassembly of FtsZ links to a conformational change of the monomer from a tense (T) to a relaxed (R) state, but considerable controversy still remains concerning the mechanism.  In this study, we report crystal structures of FtsZ from Staphylococcus aureus corresponding to the T and R state conformations in the same crystal, indicating the structural equilibrium of the two state.  The two structures identified a key residue Arg29, whose importance was also confirmed by our modified MD simulations.  Crystal structures of the R29A mutant showed T and R state-like conformations with slight but important structural changes compared to those of wild-type.  Collectively, these data provide new insights for understanding how intramolecular interactions are related to the structural transition of FtsZ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6iihP5eyfJWgRQsDqrjW_fW6udTcc2eZxauSIWiT5Zrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crisFejuw%253D%253D&md5=b5132354e5b4720ce49c6c03032331da</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2017.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2017.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DJ.%26aulast%3DHarada%26aufirst%3DR.%26aulast%3DMaeda%26aufirst%3DY.%26aulast%3DSaito%26aufirst%3DY.%26aulast%3DMizohata%26aufirst%3DE.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DShigeta%26aufirst%3DY.%26aulast%3DMatsumura%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520the%2520key%2520interactions%2520in%2520structural%2520transition%2520pathway%2520of%2520FtsZ%2520from%2520Staphylococcus%2520aureus%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2017%26volume%3D198%26spage%3D65%26epage%3D73%26doi%3D10.1016%2Fj.jsb.2017.04.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagstaff, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Sanchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kureisaite-Ciziene, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, J.</span></span> <span> </span><span class="NLM_article-title">A polymerization-associated structural switch in FtsZ that enables treadmilling of model filaments</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e00254-17</span> <span class="refDoi"> DOI: 10.1128/mBio.00254-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1128%2FmBio.00254-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28465423" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=J.+M.+Wagstaffauthor=M.+Tsimauthor=M.+A.+Olivaauthor=A.+Garcia-Sanchezauthor=D.+Kureisaite-Cizieneauthor=J.+M.+Andreuauthor=J.+Lowe&title=A+polymerization-associated+structural+switch+in+FtsZ+that+enables+treadmilling+of+model+filaments&doi=10.1128%2FmBio.00254-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FmBio.00254-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.00254-17%26sid%3Dliteratum%253Aachs%26aulast%3DWagstaff%26aufirst%3DJ.%2BM.%26aulast%3DTsim%26aufirst%3DM.%26aulast%3DOliva%26aufirst%3DM.%2BA.%26aulast%3DGarcia-Sanchez%26aufirst%3DA.%26aulast%3DKureisaite-Ciziene%26aufirst%3DD.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DLowe%26aufirst%3DJ.%26atitle%3DA%2520polymerization-associated%2520structural%2520switch%2520in%2520FtsZ%2520that%2520enables%2520treadmilling%2520of%2520model%2520filaments%26jtitle%3DmBio%26date%3D2017%26volume%3D8%26doi%3D10.1128%2FmBio.00254-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Blanco, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span> <span> </span><span class="NLM_article-title">Understanding nucleotide-regulated FtsZ filament dynamics and the monomer assembly switch with large-scale atomistic simulations</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">2164</span>– <span class="NLM_lpage">2176</span>, <span class="refDoi"> DOI: 10.1016/j.bpj.2014.09.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.bpj.2014.09.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=25418101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGrsL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2014&pages=2164-2176&author=E.+Ram%C3%ADrez-Aportelaauthor=J.+R.+Lopez-Blancoauthor=J.+M.+Andreuauthor=P.+Chacon&title=Understanding+nucleotide-regulated+FtsZ+filament+dynamics+and+the+monomer+assembly+switch+with+large-scale+atomistic+simulations&doi=10.1016%2Fj.bpj.2014.09.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Nucleotide-Regulated FtsZ Filament Dynamics and the Monomer Assembly Switch with Large-Scale Atomistic Simulations</span></div><div class="casAuthors">Ramirez-Aportela, Erney; Lopez-Blanco, Jose Ramon; Andreu, Jose Manuel; Chacon, Pablo</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2164-2176</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bacterial cytoskeletal protein FtsZ assembles in a head-to-tail manner, forming dynamic filaments that are essential for cell division.  Here, we study their dynamics using unbiased atomistic mol. simulations from representative filament crystal structures.  In agreement with exptl. data, we find different filament curvatures that are supported by a nucleotide-regulated hinge motion between consecutive FtsZ monomers.  Whereas GTP-FtsZ filaments bend and twist in a preferred orientation, thereby burying the nucleotide, the differently curved GDP-FtsZ filaments exhibit a heterogeneous distribution of open and closed interfaces between monomers.  We identify a coordinated Mg2+ ion as the key structural element in closing the nucleotide site and stabilizing GTP filaments, whereas the loss of the contacts with loop T7 from the next monomer in GDP filaments leads to open interfaces that are more prone to depolymn.  We monitored the FtsZ monomer assembly switch, which involves opening/closing of the cleft between the C-terminal domain and the H7 helix, and obsd. the relaxation of isolated and filament minus-end monomers into the closed-cleft inactive conformation.  This result validates the proposed switch between the low-affinity monomeric closed-cleft conformation and the active open-cleft FtsZ conformation within filaments.  Finally, we obsd. how the antibiotic PC190723 suppresses the disassembly switch and allosterically induces closure of the intermonomer interfaces, thus stabilizing the filament.  Our studies provide detailed structural and dynamic insights into modulation of both the intrinsic curvature of the FtsZ filaments and the mol. switch coupled to the high-affinity end-wise assocn. of FtsZ monomers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7kR2AP1MkN7Vg90H21EOLACvtfcHk0lgts-AazUhGFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGrsL3M&md5=59c71ab7e2dfb53693a2336981e316cd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2014.09.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2014.09.033%26sid%3Dliteratum%253Aachs%26aulast%3DRam%25C3%25ADrez-Aportela%26aufirst%3DE.%26aulast%3DLopez-Blanco%26aufirst%3DJ.%2BR.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DChacon%26aufirst%3DP.%26atitle%3DUnderstanding%2520nucleotide-regulated%2520FtsZ%2520filament%2520dynamics%2520and%2520the%2520monomer%2520assembly%2520switch%2520with%2520large-scale%2520atomistic%2520simulations%26jtitle%3DBiophys.%2520J.%26date%3D2014%26volume%3D107%26spage%3D2164%26epage%3D2176%26doi%3D10.1016%2Fj.bpj.2014.09.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Araújo-Bazán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Cytological profile of antibacterial FtsZ inhibitors and synthetic peptide MciZ</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">1558</span> <span class="refDoi"> DOI: 10.3389/fmicb.2016.01558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3389%2Ffmicb.2016.01558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=27752253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A280%3ADC%252BC2srhsFOnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=L.+Ara%C3%BAjo-Baz%C3%A1nauthor=L.+B.+Ruiz-Avilaauthor=D.+Andreuauthor=S.+Huecasauthor=J.+M.+Andreu&title=Cytological+profile+of+antibacterial+FtsZ+inhibitors+and+synthetic+peptide+MciZ&doi=10.3389%2Ffmicb.2016.01558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cytological Profile of Antibacterial FtsZ Inhibitors and Synthetic Peptide MciZ</span></div><div class="casAuthors">Araujo-Bazan Lidia; Ruiz-Avila Laura B; Huecas Sonia; Andreu Jose M; Andreu David</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1558</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">Cell division protein FtsZ is the organizer of the cytokinetic ring in almost all bacteria and a target for the discovery of new antibacterial agents that are needed to counter widespread antibiotic resistance.  Bacterial cytological profiling, using quantitative microscopy, is a powerful approach for identifying the mechanism of action of antibacterial molecules affecting different cellular pathways.  We have determined the cytological profile on Bacillus subtilis cells of a selection of small molecule inhibitors targeting FtsZ on different binding sites.  FtsZ inhibitors lead to long undivided cells, impair the normal assembly of FtsZ into the midcell Z-rings, induce aberrant ring distributions, punctate FtsZ foci, membrane spots and also modify nucleoid length.  Quantitative analysis of cell and nucleoid length combined, or the Z-ring distribution, allows categorizing FtsZ inhibitors and to distinguish them from antibiotics with other mechanisms of action, which should be useful for identifying new antibacterial FtsZ inhibitors.  Biochemical assays of FtsZ polymerization and GTPase activity combined explain the cellular effects of the FtsZ polymer stabilizing agent PC190723 and its fragments.  MciZ is a 40-aminoacid endogenous inhibitor of cell division normally expressed during sporulation in B. subtilis.  Using FtsZ cytological profiling we have determined that exogenous synthetic MciZ is an effective inhibitor of B. subtilis cell division, Z-ring formation and localization.  This finding supports our cell-based approach to screen for FtsZ inhibitors and opens new possibilities for peptide inhibitors of bacterial cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQwn9gu9848Z7FU0A1HOshhfW6udTcc2eYxV9yhJi7zh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srhsFOnsg%253D%253D&md5=4fc9fa0327e0f3b86662f7fb7b0fe742</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2016.01558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2016.01558%26sid%3Dliteratum%253Aachs%26aulast%3DAra%25C3%25BAjo-Baz%25C3%25A1n%26aufirst%3DL.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DAndreu%26aufirst%3DD.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DCytological%2520profile%2520of%2520antibacterial%2520FtsZ%2520inhibitors%2520and%2520synthetic%2520peptide%2520MciZ%26jtitle%3DFront.%2520Microbiol.%26date%3D2016%26volume%3D7%26doi%3D10.3389%2Ffmicb.2016.01558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Araújo-Bazán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Synthetic developmental regulator MciZ targets FtsZ across Bacillus species and inhibits bacterial division</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">965</span>– <span class="NLM_lpage">980</span>, <span class="refDoi"> DOI: 10.1111/mmi.14198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Fmmi.14198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30636070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVyhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2019&pages=965-980&author=L.+Ara%C3%BAjo-Baz%C3%A1nauthor=S.+Huecasauthor=J.+Valleauthor=D.+Andreuauthor=J.+M.+Andreu&title=Synthetic+developmental+regulator+MciZ+targets+FtsZ+across+Bacillus+species+and+inhibits+bacterial+division&doi=10.1111%2Fmmi.14198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic developmental regulator MciZ targets FtsZ across Bacillus species and inhibits bacterial division</span></div><div class="casAuthors">Araujo-Bazan, Lidia; Huecas, Sonia; Valle, Javier; Andreu, David; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">965-980</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Cell division in most bacteria is directed by FtsZ, a conserved tubulin-like GTPase that assembles forming the cytokinetic Z-ring and constitutes a target for the discovery of new antibiotics.  The developmental regulator MciZ, a 40-amino acid peptide endogenously produced during Bacillus subtilis sporulation, halts cytokinesis in the mother cell by inhibiting FtsZ.  The crystal structure of a FtsZ:MciZ complex revealed that bound MciZ extends the C-terminal β-sheet of FtsZ blocking its assembly interface.  Here we demonstrate that exogenously added MciZ specifically inhibits B. subtilis cell division, sporulation and germination, and provide insight into MciZ mol. recognition by FtsZ from different bacteria.  MciZ and FtsZ form a complex with sub-micromolar affinity, analyzed by anal. ultracentrifugation, laser biolayer interferometry and isothermal titrn. calorimetry.  Synthetic MciZ analogs, carrying single amino acid substitutions impairing MciZ β-strand formation or hydrogen bonding to FtsZ, show a gradual redn. in affinity that resembles their impaired activity in bacteria.  Gene sequences encoding MciZ spread across genus Bacillus and synthetic MciZ slows down cell division in Bacillus species, including pathogenic Bacillus cereus and Bacillus anthracis.  Moreover, B. subtilis MciZ is recognized by the homologous FtsZ from Staphylococcus aureus and inhibits division when it is expressed into S. aureus cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXrHpsPiqwnrVg90H21EOLACvtfcHk0lgts-AazUhGFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVyhsLY%253D&md5=bb61f1f52a179f0c524a573240bf6b9b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fmmi.14198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fmmi.14198%26sid%3Dliteratum%253Aachs%26aulast%3DAra%25C3%25BAjo-Baz%25C3%25A1n%26aufirst%3DL.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DValle%26aufirst%3DJ.%26aulast%3DAndreu%26aufirst%3DD.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DSynthetic%2520developmental%2520regulator%2520MciZ%2520targets%2520FtsZ%2520across%2520Bacillus%2520species%2520and%2520inhibits%2520bacterial%2520division%26jtitle%3DMol.%2520Microbiol.%26date%3D2019%26volume%3D111%26spage%3D965%26epage%3D980%26doi%3D10.1111%2Fmmi.14198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Díaz, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buey, R. M.</span></span> <span> </span><span class="NLM_article-title">Characterizing ligand-microtubule binding by competition methods</span>. <i>Methods Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1007/978-1-59745-442-1_17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1007%2F978-1-59745-442-1_17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18085234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wns7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2007&pages=245-260&author=J.+F.+D%C3%ADazauthor=R.+M.+Buey&title=Characterizing+ligand-microtubule+binding+by+competition+methods&doi=10.1007%2F978-1-59745-442-1_17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Characterizing ligand-microtubule binding by competition methods</span></div><div class="casAuthors">Diaz, Jose Fernando; Buey, Ruben Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">Microtubule Protocols</span>),
    <span class="NLM_cas:pages">245-260</span>CODEN:
                <span class="NLM_cas:coden">MMMEFN</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">The knowledge of the thermodn. and kinetics of drug-microtubule interaction is essential to understand the structure/affinity relationship of a given ligand family.  When a ligand does not show an appropriate signal change (absorbance or fluorescence) upon binding, the extensive direct characterization of its binding affinities and kinetic rate consts. of assocn. and dissocn. becomes a complex task.  In those cases it is possible to obtain these parameters by competition of the ligand with a ref. one of the same binding site that shows such change.  Nevertheless, although the exptl. setup of the competition measurements is easier, the treatment of the data is complex because simultaneous equil./kinetic equations have to be solved.  In this chapter, the taxoid-binding site of the microtubules will be used as an example to describe exptl. competition and data anal. methods to det. the binding consts. and kinetic rates of assocn. and dissocn. of ligands for microtubules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jHl9QBWSmbVg90H21EOLACvtfcHk0lhecMnN7A72fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wns7jK&md5=324bb229b1639db49245f2b85ccc19ec</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2F978-1-59745-442-1_17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-59745-442-1_17%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25ADaz%26aufirst%3DJ.%2BF.%26aulast%3DBuey%26aufirst%3DR.%2BM.%26atitle%3DCharacterizing%2520ligand-microtubule%2520binding%2520by%2520competition%2520methods%26jtitle%3DMethods%2520Mol.%2520Med.%26date%3D2007%26volume%3D137%26spage%3D245%26epage%3D260%26doi%3D10.1007%2F978-1-59745-442-1_17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canosa-Valls, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araújo-Bazán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurents, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Tornero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Nucleotide-induced folding of cell division protein FtsZ from Staphylococcus aureus</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">4048</span>– <span class="NLM_lpage">4067</span>, <span class="refDoi"> DOI: 10.1111/febs.15235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Ffebs.15235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=31997533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjs1ansbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2020&pages=4048-4067&author=S.+Huecasauthor=A.+J.+Canosa-Vallsauthor=L.+Ara%C3%BAjo-Baz%C3%A1nauthor=F.+M.+Ruizauthor=D.+V.+Laurentsauthor=C.+Fern%C3%A1ndez-Torneroauthor=J.+M.+Andreu&title=Nucleotide-induced+folding+of+cell+division+protein+FtsZ+from+Staphylococcus+aureus&doi=10.1111%2Ffebs.15235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide-induced folding of cell division protein FtsZ from Staphylococcus aureus</span></div><div class="casAuthors">Huecas, Sonia; Canosa-Valls, Alejandro J.; Araujo-Bazan, Lidia; Ruiz, Federico M.; Laurents, Douglas V.; Fernandez-Tornero, Carlos; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4048-4067</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The essential bacterial division protein FtsZ uses GTP binding and hydrolysis to assemble into dynamic filaments that treadmill around the Z-ring, guiding septal wall synthesis and cell division.  FtsZ is a structural homolog of tubulin and a target for discovering new antibiotics.  Here, using FtsZ from the pathogen S. aureus (SaFtsZ), we reveal that, prior to assembly, FtsZ monomers require nucleotide binding for folding; this is possibly relevant to other mesophilic FtsZs.  Apo-SaFtsZ is essentially unfolded, as assessed by NMR and CD.  Binding of GTP (≥ 1 mM) dramatically shifts the equil. toward the active folded protein.  Supportingly, SaFtsZ refolded with GDP crystallizes in a native structure.  Apo-SaFtsZ also folds with 3.4 M glycerol, enabling high-affinity GTP binding (KD 20 nM detd. by isothermal titrn. calorimetry) similar to thermophilic stable FtsZ.  Other stabilizing agents that enhance nucleotide binding include ethylene glycol, trimethylamine N-oxide, and several bacterial osmolytes.  High salt stabilizes SaFtsZ without bound nucleotide in an inactive twisted conformation.  We identified a cavity behind the SaFtsZ-GDP nucleotide-binding pocket that harbors different small compds., which is available for extended nucleotide-replacing inhibitors.  Furthermore, we devised a competition assay to detect any inhibitors that overlap the nucleotide site of SaFtsZ, or Escherichia coli FtsZ, employing osmolyte-stabilized apo-FtsZs and the specific fluorescence anisotropy change in mant-GTP upon dissocn. from the protein.  This robust assay provides a basis to screening for high-affinity GTP-replacing ligands, which combined with structural studies and phenotypic profiling should facilitate development of a next generation of FtsZ-targeting antibacterial inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV19nVjkdRurVg90H21EOLACvtfcHk0lhecMnN7A72fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjs1ansbs%253D&md5=4f40b31dde49f0a4b480f15fc5b433ef</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Ffebs.15235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.15235%26sid%3Dliteratum%253Aachs%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DCanosa-Valls%26aufirst%3DA.%2BJ.%26aulast%3DAra%25C3%25BAjo-Baz%25C3%25A1n%26aufirst%3DL.%26aulast%3DRuiz%26aufirst%3DF.%2BM.%26aulast%3DLaurents%26aufirst%3DD.%2BV.%26aulast%3DFern%25C3%25A1ndez-Tornero%26aufirst%3DC.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DNucleotide-induced%2520folding%2520of%2520cell%2520division%2520protein%2520FtsZ%2520from%2520Staphylococcus%2520aureus%26jtitle%3DFEBS%2520J.%26date%3D2020%26volume%3D287%26spage%3D4048%26epage%3D4067%26doi%3D10.1111%2Ffebs.15235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergonos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercenado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barasoain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Villa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Synthetic inhibitors of bacterial cell division targeting the GTP-binding site of FtsZ</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2072</span>– <span class="NLM_lpage">2083</span>, <span class="refDoi"> DOI: 10.1021/cb400208z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400208z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSnsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2072-2083&author=L.+B.+Ruiz-Avilaauthor=S.+Huecasauthor=M.+Artolaauthor=A.+Vergonosauthor=E.+Ramirez-Aportelaauthor=E.+Cercenadoauthor=I.+Barasoainauthor=H.+Vazquez-Villaauthor=M.+Martin-Fontechaauthor=P.+Chaconauthor=M.+L.+Lopez-Rodriguezauthor=J.+M.+Andreu&title=Synthetic+inhibitors+of+bacterial+cell+division+targeting+the+GTP-binding+site+of+FtsZ&doi=10.1021%2Fcb400208z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Inhibitors of Bacterial Cell Division Targeting the GTP-Binding Site of FtsZ</span></div><div class="casAuthors">Ruiz-Avila, Laura B.; Huecas, Sonia; Artola, Marta; Vergonos, Albert; Ramirez-Aportela, Erney; Cercenado, Emilia; Barasoain, Isabel; Vazquez-Villa, Henar; Martin-Fontecha, Mar; Chacon, Pablo; Lopez-Rodriguez, Maria L.; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2072-2083</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell division protein FtsZ is the organizer of the cytokinetic Z-ring in most bacteria and a target for new antibiotics.  FtsZ assembles with GTP into filaments that hydrolyze the nucleotide at the assocn. interface between monomers and then disassemble.  FtsZ's GTP was replaced with non-nucleotide synthetic inhibitors of bacterial division.  The authors searched for these small mols. among compds. from the literature, from virtual screening (VS), and from their inhouse synthetic library (UCM), employing a fluorescence anisotropy primary assay.  From these screens they identified the polyhydroxy arom. compd. UCM05 and its simplified analog UCM44 that specifically bind to Bacillus subtilis FtsZ monomers with micromolar affinities and perturb normal assembly, as examd. with light scattering, polymer sedimentation, and neg. stain electron microscopy.  On the other hand, these ligands induce the cooperative assembly of nucleotide-devoid archaeal FtsZ into distinct well-ordered polymers, different from GTP-induced filaments.  These FtsZ inhibitors impair localization of FtsZ into the Z-ring and inhibit bacterial cell division.  The chlorinated analog UCM53 inhibits the growth of clin. isolates of antibiotic-resistant Staphylococcus aureus and Enterococcus faecalis.  It is suggested that these interfacial inhibitors recapitulate binding and some assembly-inducing effects of GTP but impair the correct structural dynamics of FtsZ filaments and thus inhibit bacterial division, possibly by binding to a small fraction of the FtsZ mols. in a bacterial cell, which opens a new approach to FtsZ-based antibacterial drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIArKPncswxrVg90H21EOLACvtfcHk0lhecMnN7A72fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSnsbnO&md5=3a08fa169f279cc7c7790b23a25eae6f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcb400208z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400208z%26sid%3Dliteratum%253Aachs%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DArtola%26aufirst%3DM.%26aulast%3DVergonos%26aufirst%3DA.%26aulast%3DRamirez-Aportela%26aufirst%3DE.%26aulast%3DCercenado%26aufirst%3DE.%26aulast%3DBarasoain%26aufirst%3DI.%26aulast%3DVazquez-Villa%26aufirst%3DH.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DSynthetic%2520inhibitors%2520of%2520bacterial%2520cell%2520division%2520targeting%2520the%2520GTP-binding%2520site%2520of%2520FtsZ%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2072%26epage%3D2083%26doi%3D10.1021%2Fcb400208z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barasoain, I.</span></span> <span> </span><span class="NLM_article-title">The interaction of baccatin III with the taxol binding site of microtubules determined by a homogeneous assay with fluorescent taxoid</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">11975</span>– <span class="NLM_lpage">11984</span>, <span class="refDoi"> DOI: 10.1021/bi010869+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi010869%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1OgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=11975-11984&author=J.+M.+Andreuauthor=I.+Barasoain&title=The+interaction+of+baccatin+III+with+the+taxol+binding+site+of+microtubules+determined+by+a+homogeneous+assay+with+fluorescent+taxoid&doi=10.1021%2Fbi010869%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The Interaction of Baccatin III with the Taxol Binding Site of Microtubules Determined by a Homogeneous Assay with Fluorescent Taxoid</span></div><div class="casAuthors">Andreu, Jose M.; Barasoain, Isabel</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">11975-11984</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ubiquitous Taxol binding site of microtubules also binds newly discovered ligands.  We have designed a homogeneous assay for the high throughput detection of Taxol biomimetics, based on the displacement of 7-O-[N-(2,7-difluoro-4'-fluoresceincarbonyl)-L-alanyl]Taxol from its binding site in dild. solns. of preserved microtubules.  The state of this ref. ligand is measured by fluorescence anisotropy in a microplate reader, with varying concns. of nonfluorescent competitors.  The binding equil. const. of Taxol has a value Kb = 3.7×107 M-1.  We have found that baccatin III, an analog of Taxol without the C-13 side chain, binds with Kb = 1.5×105 M-1, whereas the side chain Me ester is inactive.  This was unexpected from the structure-activity relation of taxoids but compatible with models of Taxol docked at the microtubule site.  Baccatin III binding has been confirmed by displacement of [3H]Taxol and by direct HPLC measurements of its cosedimentation with microtubules, among other methods.  Consequently, baccatin III induces microtubule bundles and multipolar spindles in PtK2 and U937 cells, and mitotic arrest and apoptotic death of the U937 cells, at concns. 200-500-fold larger than Taxol.  The simplest anal. of these results strongly suggests that the interaction of the C-2 C-4 substituted taxane ring system with the microtubule binding site provides most (75%) of the free energy change of Taxol binding and is sufficient to activate microtubule stabilization and transmit the antitumor effects of Taxol, whereas the C-13 side chain provides a weak specific anchor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4qoUCYS9qXLVg90H21EOLACvtfcHk0lguuauMAAbnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1OgsLs%253D&md5=06ff2874f0298f9358699bc3335d50e4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fbi010869%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi010869%252B%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DBarasoain%26aufirst%3DI.%26atitle%3DThe%2520interaction%2520of%2520baccatin%2520III%2520with%2520the%2520taxol%2520binding%2520site%2520of%2520microtubules%2520determined%2520by%2520a%2520homogeneous%2520assay%2520with%2520fluorescent%2520taxoid%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D11975%26epage%3D11984%26doi%3D10.1021%2Fbi010869%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomaides, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haydon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czaplewski, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, J.</span></span> <span> </span><span class="NLM_article-title">Novel inhibitors of bacterial cytokinesis identified by a cell-based antibiotic screening assay</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">39709</span>– <span class="NLM_lpage">39715</span>, <span class="refDoi"> DOI: 10.1074/jbc.M506741200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M506741200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=16174771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1GjsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=39709-39715&author=N.+R.+Stokesauthor=J.+Sieversauthor=S.+Barkerauthor=J.+M.+Bennettauthor=D.+R.+Brownauthor=I.+Collinsauthor=V.+M.+Erringtonauthor=D.+Foulgerauthor=M.+Hallauthor=R.+Halseyauthor=H.+Johnsonauthor=V.+Roseauthor=H.+B.+Thomaidesauthor=D.+J.+Haydonauthor=L.+G.+Czaplewskiauthor=J.+Errington&title=Novel+inhibitors+of+bacterial+cytokinesis+identified+by+a+cell-based+antibiotic+screening+assay&doi=10.1074%2Fjbc.M506741200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitors of Bacterial Cytokinesis Identified by a Cell-based Antibiotic Screening Assay</span></div><div class="casAuthors">Stokes, Neil R.; Sievers, Joerg; Barker, Stephanie; Bennett, James M.; Brown, David R.; Collins, Ian; Errington, Veronica M.; Foulger, David; Hall, Michelle; Halsey, Rowena; Johnson, Hazel; Rose, Valerie; Thomaides, Helena B.; Haydon, David J.; Czaplewski, Lloyd G.; Errington, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">39709-39715</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The continuous emergence of antibiotic resistance demands that novel classes of antibiotics continue to be developed.  The division machinery of bacteria is an attractive target because it comprises seven or more essential proteins that are conserved almost throughout the bacteria but are absent from humans.  We describe the development of a cell-based assay for inhibitors of cell division and its use to isolate a new inhibitor of FtsZ protein, a key player in the division machinery.  Biochem., cytol., and genetic data are presented that demonstrate that FtsZ is the specific target for the compd.  We also describe the effects of more potent analogs of the original hit compd. that act on important pathogens, again at the level of cell division.  The assay and the compds. have the potential to provide novel antibiotics with no pool of pre-existing resistance.  They have provided new insight into cytokinesis in bacteria and offer important reagents for further studies of the cell division machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_AR5gKzeaVbVg90H21EOLACvtfcHk0lguuauMAAbnuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1GjsrbK&md5=7f0a57ef32a558366517e826e25b5347</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M506741200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M506741200%26sid%3Dliteratum%253Aachs%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DSievers%26aufirst%3DJ.%26aulast%3DBarker%26aufirst%3DS.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBrown%26aufirst%3DD.%2BR.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DErrington%26aufirst%3DV.%2BM.%26aulast%3DFoulger%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DM.%26aulast%3DHalsey%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DH.%26aulast%3DRose%26aufirst%3DV.%26aulast%3DThomaides%26aufirst%3DH.%2BB.%26aulast%3DHaydon%26aufirst%3DD.%2BJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26aulast%3DErrington%26aufirst%3DJ.%26atitle%3DNovel%2520inhibitors%2520of%2520bacterial%2520cytokinesis%2520identified%2520by%2520a%2520cell-based%2520antibiotic%2520screening%2520assay%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D39709%26epage%3D39715%26doi%3D10.1074%2Fjbc.M506741200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margalit, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romberg, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mets, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RayChaudhuri, D.</span></span> <span> </span><span class="NLM_article-title">Targeting cell division: Small-molecule inhibitors of FtsZ GTPase perturb cytokinetic ring assembly and induce bacterial lethality</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">11821</span>– <span class="NLM_lpage">11826</span>, <span class="refDoi"> DOI: 10.1073/pnas.0404439101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1073%2Fpnas.0404439101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=15289600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntVers7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=11821-11826&author=D.+N.+Margalitauthor=L.+Rombergauthor=R.+B.+Metsauthor=A.+M.+Hebertauthor=T.+J.+Mitchisonauthor=M.+W.+Kirschnerauthor=D.+RayChaudhuri&title=Targeting+cell+division%3A+Small-molecule+inhibitors+of+FtsZ+GTPase+perturb+cytokinetic+ring+assembly+and+induce+bacterial+lethality&doi=10.1073%2Fpnas.0404439101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cell division: Small-molecule inhibitors of FtsZ GTPase perturb cytokinetic ring assembly and induce bacterial lethality</span></div><div class="casAuthors">Margalit, Danielle N.; Romberg, Laura; Mets, Rebecca B.; Hebert, Alan M.; Mitchison, Timothy J.; Kirschner, Marc W.; RayChaudhuri, Debabrata</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">11821-11826</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">FtsZ, the ancestral homolog of eukaryotic tubulins, is a GTPase that assembles into a cytokinetic ring structure essential for cell division in prokaryotic cells.  Similar to tubulin, purified FtsZ polymerizes into dynamic protofilaments in the presence of GTP; polymer assembly is accompanied by GTP hydrolysis.  The authors used a high-throughput protein-based chem. screen to identify small mols. that target assembly-dependent GTPase activity of FtsZ.  Here, the authors report the identification of five structurally diverse compds., named Zantrins, which inhibit FtsZ GTPase either by destabilizing the FtsZ protofilaments or by inducing filament hyperstability through increased lateral assocn.  These two classes of FtsZ inhibitors are reminiscent of the antitubulin drugs colchicine and Taxol, resp.  The authors also show that Zantrins perturb FtsZ ring assembly in Escherichia coli cells and cause lethality to a variety of bacteria in broth cultures, indicating that FtsZ antagonists may serve as chem. leads for the development of new broad-spectrum antibacterial agents.  Our results illustrate the utility of small-mol. chem. probes to study FtsZ polymn. dynamics and the feasibility of FtsZ as a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOPgYgvnXeKLVg90H21EOLACvtfcHk0li0mHjYkPmAwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntVers7s%253D&md5=ed037cc4b71bb294aff750bea98351e8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0404439101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0404439101%26sid%3Dliteratum%253Aachs%26aulast%3DMargalit%26aufirst%3DD.%2BN.%26aulast%3DRomberg%26aufirst%3DL.%26aulast%3DMets%26aufirst%3DR.%2BB.%26aulast%3DHebert%26aufirst%3DA.%2BM.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DRayChaudhuri%26aufirst%3DD.%26atitle%3DTargeting%2520cell%2520division%253A%2520Small-molecule%2520inhibitors%2520of%2520FtsZ%2520GTPase%2520perturb%2520cytokinetic%2520ring%2520assembly%2520and%2520induce%2520bacterial%2520lethality%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D11821%26epage%3D11826%26doi%3D10.1073%2Fpnas.0404439101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorto, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackner, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ames, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. T.</span></span> <span> </span><span class="NLM_article-title">Comparison of small molecule inhibitors of the bacterial cell division protein FtsZ and identification of a reliable cross-species inhibitor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1918</span>– <span class="NLM_lpage">1928</span>, <span class="refDoi"> DOI: 10.1021/cb300340j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300340j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1918-1928&author=D.+E.+Andersonauthor=M.+B.+Kimauthor=J.+T.+Mooreauthor=T.+E.+O%E2%80%99Brienauthor=N.+A.+Sortoauthor=C.+I.+Groveauthor=L.+L.+Lacknerauthor=J.+B.+Amesauthor=J.+T.+Shaw&title=Comparison+of+small+molecule+inhibitors+of+the+bacterial+cell+division+protein+FtsZ+and+identification+of+a+reliable+cross-species+inhibitor&doi=10.1021%2Fcb300340j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Small Molecule Inhibitors of the Bacterial Cell Division Protein FtsZ and Identification of a Reliable Cross-Species Inhibitor</span></div><div class="casAuthors">Anderson, David E.; Kim, Michelle B.; Moore, Jared T.; O'Brien, Terrence E.; Sorto, Nohemy A.; Grove, Charles I.; Lackner, Laura L.; Ames, James B.; Shaw, Jared T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1918-1928</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FtsZ is a guanosine triphosphatase (GTPase) that mediates cytokinesis in bacteria.  FtsZ is homologous in structure to eukaryotic tubulin and polymerizes in a similar head-to-tail fashion.  The study of tubulin's function in eukaryotic cells has benefited greatly from specific and potent small mol. inhibitors, including colchicine and taxol.  Although many small mol. inhibitors of FtsZ have been reported, none has emerged as a generally useful probe for modulating bacterial cell division.  With the goal of establishing a useful and reliable small mol. inhibitor of FtsZ, a broad biochem. cross-comparison of reported FtsZ inhibitors was undertaken.  Several of these mols., including phenolic natural products, are unselective inhibitors that seem to derive their activity from the formation of microscopic colloids or aggregates.  Other compds., including the natural product viriditoxin and the drug development candidate PC190723, exhibit no inhibition of GTPase activity using protocols or under published conditions.  Of the compds. studied, only zantrin Z3 exhibits good levels of inhibition, maintains activity under conditions that disrupt small mol. aggregates, and provides a platform for exploration of structure-activity relationships (SAR).  Preliminary SAR studies have identified slight modifications to the two side chains of this structure that modulate the inhibitory activity of zantrin Z3.  Collectively, these studies will help focus future investigations toward the establishment of probes for FtsZ that fill the roles of colchicine and taxol in studies of tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvE8_fy6gEwLVg90H21EOLACvtfcHk0li0mHjYkPmAwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL&md5=ab69ea62af74ad72f6177db5f556c990</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fcb300340j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300340j%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BE.%26aulast%3DKim%26aufirst%3DM.%2BB.%26aulast%3DMoore%26aufirst%3DJ.%2BT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%2BE.%26aulast%3DSorto%26aufirst%3DN.%2BA.%26aulast%3DGrove%26aufirst%3DC.%2BI.%26aulast%3DLackner%26aufirst%3DL.%2BL.%26aulast%3DAmes%26aufirst%3DJ.%2BB.%26aulast%3DShaw%26aufirst%3DJ.%2BT.%26atitle%3DComparison%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520bacterial%2520cell%2520division%2520protein%2520FtsZ%2520and%2520identification%2520of%2520a%2520reliable%2520cross-species%2520inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1918%26epage%3D1928%26doi%3D10.1021%2Fcb300340j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, Y. H.</span></span> <span> </span><span class="NLM_article-title">Resveratrol antibacterial activity against Escherichia coli is mediated by Z-ring formation inhibition via suppression of FtsZ expression</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">10029</span> <span class="refDoi"> DOI: 10.1038/srep10029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fsrep10029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=25942564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOrs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&author=D.+Hwangauthor=Y.+H.+Lim&title=Resveratrol+antibacterial+activity+against+Escherichia+coli+is+mediated+by+Z-ring+formation+inhibition+via+suppression+of+FtsZ+expression&doi=10.1038%2Fsrep10029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol antibacterial activity against Escherichia coli is mediated by Z-ring formation inhibition via suppression of FtsZ expression</span></div><div class="casAuthors">Hwang, Dahyun; Lim, Young-Hee</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10029</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Resveratrol exhibits a potent antimicrobial activity.  However, the mechanism underlying its antibacterial activity has not been shown.  In this study, the antibacterial mechanism of resveratrol was investigated.  To investigate induction of the SOS response, a strain contg. the lacZ+gene under the control of an SOS-inducible sulA promoter was constructed.  DNA damage was measured by pulse-field gel electrophoresis (PFGE).  After resveratrol treatment, the cells were obsd. by confocal microscopy.  For the RNA silencing assay, ftsZ-specific antisense peptide nucleic acid (PNA) was used.  Reactive oxygen species (ROS) prodn. increased in Escherichia coli after resveratrol treatment; however, cell growth was not recovered by ROS quenching, indicating that, in this expt., ROS formation and cell death following resveratrol treatment were not directly correlated.  Resveratrol treatment increased DNA fragmentation in cells, while SOS response-related gene expression levels increased in a dose-dependent manner.  Cell elongation was obsd. after resveratrol treatment.  Elongation was induced by inhibiting FtsZ, an essential cell-division protein in prokaryotes, and resulted in significant inhibition of Z-ring the formation in E. coli.  The expression of ftsZ mRNA was suppressed by resveratrol.  Thus, the results indicate that resveratrol inhibits bacterial cell growth by suppressing FtsZ expression and Z-ring formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovOV74pcApNrVg90H21EOLACvtfcHk0li0mHjYkPmAwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOrs7bJ&md5=9e72a33b5085fe517ba981d21b2e6c93</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fsrep10029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep10029%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DY.%2BH.%26atitle%3DResveratrol%2520antibacterial%2520activity%2520against%2520Escherichia%2520coli%2520is%2520mediated%2520by%2520Z-ring%2520formation%2520inhibition%2520via%2520suppression%2520of%2520FtsZ%2520expression%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26doi%3D10.1038%2Fsrep10029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jindal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panda, D.</span></span> <span> </span><span class="NLM_article-title">Plumbagin inhibits cytokinesis in <i>Bacillus subtilis</i> by inhibiting FtsZ assembly--a mechanistic study of its antibacterial activity</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">4585</span>– <span class="NLM_lpage">4599</span>, <span class="refDoi"> DOI: 10.1111/febs.12429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Ffebs.12429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=23841620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGmtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=4585-4599&author=A.+Bhattacharyaauthor=B.+Jindalauthor=P.+Singhauthor=A.+Dattaauthor=D.+Panda&title=Plumbagin+inhibits+cytokinesis+in+Bacillus+subtilis+by+inhibiting+FtsZ+assembly%2D%2Da+mechanistic+study+of+its+antibacterial+activity&doi=10.1111%2Ffebs.12429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Plumbagin inhibits cytokinesis in Bacillus subtilis by inhibiting FtsZ assembly - a mechanistic study of its antibacterial activity</span></div><div class="casAuthors">Bhattacharya, Anusri; Jindal, Bhavya; Singh, Parminder; Datta, Anindya; Panda, Dulal</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4585-4599</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The assembly of FtsZ plays a central role in construction of the cytokinetic Z-ring that orchestrates bacterial cell division.  A naturally occurring naphthoquinone, plumbagin, is known to exhibit antibacterial properties against several types of bacteria.  In this study, plumbagin was found to perturb formation of the Z-ring in Bacillus subtilis 168 cells and to cause elongation of these cells without an apparent effect on nucleoid segregation, indicating that it may inhibit FtsZ assembly.  Furthermore, it bound to purified B. subtilis FtsZ (BsFtsZ) with a dissocn. const. of 20.7 ± 5.6 μm, and inhibited the assembly and GTPase activity of BsFtsZin vitro.  Interestingly, plumbagin did not inhibit either the assembly or GTPase activity of Escherichia coliFtsZ (EcFtsZ) in vitro.  Using docking anal., a putative plumbagin-binding site on BsFtsZ was identified, and the anal. indicated that hydrophobic interactions and hydrogen bonds predominate.  Based on the in silico anal., two variants of BsFtsZ, namely D199A and V307R, were constructed to explore the binding interaction of plumbagin and BsFtsZ.  The effects of plumbagin on the assembly and GTPase activity of the variant BsFtsZ proteins in vitro indicated that the residues D199 and V307 may be involved in the binding of plumbagin to BsFtsZ.  The results suggest that plumbagin inhibits bacterial proliferation by inhibiting the assembly of FtsZ, and provide insight into the binding site of plumbagin on BsFtsZ, which may help in the design of potent FtsZ-targeted antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKE9tC5NyLNLVg90H21EOLACvtfcHk0lgL79emln9Y1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGmtL7F&md5=634b9981160910c671c8e8a637036371</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12429%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharya%26aufirst%3DA.%26aulast%3DJindal%26aufirst%3DB.%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DDatta%26aufirst%3DA.%26aulast%3DPanda%26aufirst%3DD.%26atitle%3DPlumbagin%2520inhibits%2520cytokinesis%2520in%2520Bacillus%2520subtilis%2520by%2520inhibiting%2520FtsZ%2520assembly--a%2520mechanistic%2520study%2520of%2520its%2520antibacterial%2520activity%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D4585%26epage%3D4599%26doi%3D10.1111%2Ffebs.12429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. Y.</span></span> <span> </span><span class="NLM_article-title">New application of tiplaxtinin as an effective FtsZ-targeting chemotype for an antimicrobial study</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1909</span>– <span class="NLM_lpage">1913</span>, <span class="refDoi"> DOI: 10.1039/C7MD00387K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1039%2FC7MD00387K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30108711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeku7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1909-1913&author=N.+Sunauthor=Y.+Y.+Zhengauthor=R.+L.+Duauthor=S.+Y.+Caiauthor=K.+Zhangauthor=L.+Y.+Soauthor=K.+C.+Cheungauthor=C.+Zhuoauthor=Y.+J.+Luauthor=K.+Y.+Wong&title=New+application+of+tiplaxtinin+as+an+effective+FtsZ-targeting+chemotype+for+an+antimicrobial+study&doi=10.1039%2FC7MD00387K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">New application of tiplaxtinin as an effective FtsZ-targeting chemotype for an antimicrobial study</span></div><div class="casAuthors">Sun, Ning; Zheng, Yuan-Yuan; Du, Ruo-Lan; Cai, Sen-Yuan; Zhang, Kun; So, Lok-Yan; Cheung, Kwan-Choi; Zhuo, Chao; Lu, Yu-Jing; Wong, Kwok-Yin</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1909-1913</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The filamenting temp.-sensitive mutant Z (FtsZ) protein is generally recognized as a promising antimicrobial drug target.  In the present study, a small org. mol. (tiplaxtinin) was identified for the first time as an excellent cell division inhibitor by using a cell-based screening approach from a library with 250 compds.  Tiplaxtinin possesses potent antibacterial activity against Gram-pos. pathogens.  Both in vitro and in vivo results reveal that the compd. is able to disrupt dynamic assembly of FtsZ and Z-ring formation effectively through the mechanism of stimulating FtsZ polymn. and impairing GTPase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvq-t42EDbMrVg90H21EOLACvtfcHk0lgL79emln9Y1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeku7fI&md5=6114b7a52e448e75f5acf7caba5a1313</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC7MD00387K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00387K%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DY.%2BY.%26aulast%3DDu%26aufirst%3DR.%2BL.%26aulast%3DCai%26aufirst%3DS.%2BY.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DSo%26aufirst%3DL.%2BY.%26aulast%3DCheung%26aufirst%3DK.%2BC.%26aulast%3DZhuo%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DY.%2BJ.%26aulast%3DWong%26aufirst%3DK.%2BY.%26atitle%3DNew%2520application%2520of%2520tiplaxtinin%2520as%2520an%2520effective%2520FtsZ-targeting%2520chemotype%2520for%2520an%2520antimicrobial%2520study%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1909%26epage%3D1913%26doi%3D10.1039%2FC7MD00387K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Straniero, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suigo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casiraghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián-Pérez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrast, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanotto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zdovc, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Giuli
Morghen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radaelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valoti, E.</span></span> <span> </span><span class="NLM_article-title">Benzamide derivatives targeting the cell division protein FtsZ: Modifications of the linker and the benzodioxane scaffold and their effects on antimicrobial activity</span>. <i>Antibiotics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">160</span> <span class="refDoi"> DOI: 10.3390/antibiotics9040160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3390%2Fantibiotics9040160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitF2htrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=V.+Stranieroauthor=L.+Suigoauthor=A.+Casiraghiauthor=V.+Sebasti%C3%A1n-P%C3%A9rezauthor=M.+Hrastauthor=C.+Zanottoauthor=I.+Zdovcauthor=C.+De+Giuli%0AMorghenauthor=A.+Radaelliauthor=E.+Valoti&title=Benzamide+derivatives+targeting+the+cell+division+protein+FtsZ%3A+Modifications+of+the+linker+and+the+benzodioxane+scaffold+and+their+effects+on+antimicrobial+activity&doi=10.3390%2Fantibiotics9040160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Benzamide derivatives targeting the cell division protein FtsZ: modifications of the linker and the benzodioxane scaffold and their effects on antimicrobial activity</span></div><div class="casAuthors">Straniero, Valentina; Suigo, Lorenzo; Casiraghi, Andrea; Sebastian-Perez, Victor; Hrast, Martina; Zanotto, Carlo; Zdovc, Irena; Morghen, Carlo De Giuli; Radaelli, Antonia; Valoti, Ermanno</div><div class="citationInfo"><span class="NLM_cas:title">Antibiotics (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">160</span>CODEN:
                <span class="NLM_cas:coden">ABSNC4</span>;
        ISSN:<span class="NLM_cas:issn">2079-6382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Filamentous temp.-sensitive Z (FtsZ) is a prokaryotic protein with an essential role in the bacterial cell division process.  It is widely conserved and expressed in both Gram-pos. and Gram-neg. strains.  In the last decade, several research groups have pointed out mols. able to target FtsZ in Staphylococcus aureus, Bacillus subtilis and other Gram-pos. strains, with sub-micromolar Min. Inhibitory Concns. (MICs).  Conversely, no promising derivs. active on Gram-negatives have been found up to now.  Here, we report our results on a class of benzamide compds., which showed comparable inhibitory activities on both S. aureus and Escherichia coli FtsZ, even though they proved to be substrates of E. coli efflux pump AcrAB, thus affecting the antimicrobial activity.  These surprising results confirmed how a single mol. can target both species while maintaining potent antimicrobial activity.  A further computational study helped us decipher the structural features necessary for broad spectrum activity and assess the drug-like profile and the on-target activity of this family of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjNbAHfAFWqrVg90H21EOLACvtfcHk0lgmj9qki-19yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitF2htrfE&md5=bb4172456078311ffc4b0e29789408e0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3390%2Fantibiotics9040160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantibiotics9040160%26sid%3Dliteratum%253Aachs%26aulast%3DStraniero%26aufirst%3DV.%26aulast%3DSuigo%26aufirst%3DL.%26aulast%3DCasiraghi%26aufirst%3DA.%26aulast%3DSebasti%25C3%25A1n-P%25C3%25A9rez%26aufirst%3DV.%26aulast%3DHrast%26aufirst%3DM.%26aulast%3DZanotto%26aufirst%3DC.%26aulast%3DZdovc%26aufirst%3DI.%26aulast%3DDe%2BGiuli%2BMorghen%26aufirst%3DC.%26aulast%3DRadaelli%26aufirst%3DA.%26aulast%3DValoti%26aufirst%3DE.%26atitle%3DBenzamide%2520derivatives%2520targeting%2520the%2520cell%2520division%2520protein%2520FtsZ%253A%2520Modifications%2520of%2520the%2520linker%2520and%2520the%2520benzodioxane%2520scaffold%2520and%2520their%2520effects%2520on%2520antimicrobial%2520activity%26jtitle%3DAntibiotics%26date%3D2020%26volume%3D9%26doi%3D10.3390%2Fantibiotics9040160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czaplewski, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, J.</span></span> <span> </span><span class="NLM_article-title">Multiple effects of benzamide antibiotics on FtsZ function</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2011.07559.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Fj.1365-2958.2011.07559.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=21276094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlt1yitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=68-84&author=D.+W.+Adamsauthor=L.+J.+Wuauthor=L.+G.+Czaplewskiauthor=J.+Errington&title=Multiple+effects+of+benzamide+antibiotics+on+FtsZ+function&doi=10.1111%2Fj.1365-2958.2011.07559.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple effects of benzamide antibiotics on FtsZ function</span></div><div class="casAuthors">Adams, David W.; Wu, Ling Juan; Czaplewski, Lloyd G.; Errington, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-84</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cell division in almost all bacteria is orchestrated by the essential tubulin homolog FtsZ, which assembles into a ring-like structure and acts as a scaffold for the division machinery.  Division was recently validated as an important target for antibiotics by the demonstration that low-mol.-wt. inhibitors of FtsZ, called benzamides, can cure mice infected with Staphylococcus aureus.  In treated cells of Bacillus subtilis, the authors show that FtsZ assembles into foci throughout the cell, including abnormal locations at the cell poles and over the nucleoid.  These foci are not inactive aggregates because they remain dynamic, turning over almost as rapidly as untreated polymers.  Remarkably, although division is completely blocked, the foci efficiently recruit division proteins that normally co-assemble with FtsZ.  However, they show no affinity for components of the Min or Nucleoid occlusion systems.  In vitro, the benzamides strongly promote the polymn. of FtsZ, into hyperstable polymers, which are highly curved.  Importantly, even at low concns., benzamides transform the structure of the Z ring, resulting in abnormal helical cell division events.  It is proposed that benzamides act principally by promoting an FtsZ protomer conformation that is incompatible with a higher-order level of assembly needed to make a division ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor5Ah875KKmrVg90H21EOLACvtfcHk0lgmj9qki-19yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlt1yitr8%253D&md5=300166da23db8d3762446411e7954e24</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2011.07559.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2011.07559.x%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DD.%2BW.%26aulast%3DWu%26aufirst%3DL.%2BJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26aulast%3DErrington%26aufirst%3DJ.%26atitle%3DMultiple%2520effects%2520of%2520benzamide%2520antibiotics%2520on%2520FtsZ%2520function%26jtitle%3DMol.%2520Microbiol.%26date%3D2011%26volume%3D80%26spage%3D68%26epage%3D84%26doi%3D10.1111%2Fj.1365-2958.2011.07559.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel FtsZ-targeted 3-arylalkoxy-2,6-difluorobenzamides as potential antimicrobial agents</span>. <i>Chem. Biol. Drug. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Fcbdd.12658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=26348110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2016&pages=257-264&author=S.+Qiangauthor=C.+Wangauthor=H.+Venterauthor=X.+Liauthor=Y.+Wangauthor=L.+Guoauthor=R.+Maauthor=S.+Ma&title=Synthesis+and+biological+evaluation+of+novel+FtsZ-targeted+3-arylalkoxy-2%2C6-difluorobenzamides+as+potential+antimicrobial+agents&doi=10.1111%2Fcbdd.12658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Evaluation of Novel FtsZ-targeted 3-arylalkoxy-2,6-difluorobenzamides as Potential Antimicrobial Agents</span></div><div class="casAuthors">Qiang, Shengsheng; Wang, Changde; Venter, Henrietta; Li, Xin; Wang, Yi; Guo, Liwei; Ma, Ruixin; Ma, Shutao</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-264</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Novel series of 3-O-arylalkylbenzamides, e.g., I and 3-O-arylalkyl-2,6-difluorobenzamide derivs. II [R1 = H, 3-Cl, 3-Me, etc.] were synthesized from 3-hydroxybenzoic acid and 2,4-difluorophenol resp.  The synthesized derivs. were evaluated for their cell division inhibitory activity and antibacterial activity, results showed that the 3-O-arylalkyl-2,6-difluorobenzamide derivs. possessed much better cell division inhibitory activity and antibacterial activity than the 3-O-arylalkylbenzamide derivs.  Among them, 3-O-chlorobenzyl deriv. II [R3 = 3-Cl] was the most effective in antibacterial activity (0.5, 4, and 8 μg/mL) against Bacillus subtilis ATCC9372, methicillin-resistant Staphylococcus aureus ATCC29213 and penicillin-resistant Staphylococcus aureus PR, while 3-O-methylbenzyl deriv. II [R3 = 3-Me] only exhibited the most potent activity (2 μg/mL) against Staphylococcus aureus ATCC25923.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZDjHjJapcGLVg90H21EOLACvtfcHk0lgS-Q_znRHW8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsLvK&md5=7bc3d5b669f862794b014c38ebce2dee</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12658%26sid%3Dliteratum%253Aachs%26aulast%3DQiang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DVenter%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520FtsZ-targeted%25203-arylalkoxy-2%252C6-difluorobenzamides%2520as%2520potential%2520antimicrobial%2520agents%26jtitle%3DChem.%2520Biol.%2520Drug.%2520Des.%26date%3D2016%26volume%3D87%26spage%3D257%26epage%3D264%26doi%3D10.1111%2Fcbdd.12658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chai, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittall, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogunniyi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semple, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venter, H.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial action and reversal of resistance in MRSA by difluorobenzamide derivatives targeted FtsZ</span>. <i>Antibiotics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">873</span> <span class="refDoi"> DOI: 10.3390/antibiotics9120873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3390%2Fantibiotics9120873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3MXksF2msrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=W.+C.+Chaiauthor=J.+J.+Whittallauthor=D.+Songauthor=S.+W.+Polyakauthor=A.+D.+Ogunniyiauthor=Y.+Wangauthor=F.+Biauthor=S.+Maauthor=S.+J.+Sempleauthor=H.+Venter&title=Antimicrobial+action+and+reversal+of+resistance+in+MRSA+by+difluorobenzamide+derivatives+targeted+FtsZ&doi=10.3390%2Fantibiotics9120873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial action and reversal of resistance in MRSA by difluorobenzamide derivatives targeted at FtsZ</span></div><div class="casAuthors">Chai, Wern Chern; Whittall, Jonathan J.; Song, Di; Polyak, Steven W.; Ogunniyi, Abiodun D.; Wang, Yinhu; Bi, Fangchao; Ma, Shutao; Semple, Susan J.; Venter, Henrietta</div><div class="citationInfo"><span class="NLM_cas:title">Antibiotics (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">873</span>CODEN:
                <span class="NLM_cas:coden">ABSNC4</span>;
        ISSN:<span class="NLM_cas:issn">2079-6382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The bacterial cell division protein, FtsZ, has been identified as a target for antimicrobial development.  Derivs. of 3-methoxybenzamide have shown promising activities as FtsZ inhibitors in Gram-pos. bacteria.  We sought to characterize the activity of five difluorobenzamide derivs. with non-heterocyclic substituents attached through the 3-oxygen.  These compds. exhibited antimicrobial activity against methicillin resistant Staphylococcus aureus (MRSA), with an isopentyloxy-substituted compd. showing modest activity against vancomycin resistant Enterococcus faecium (VRE).  The compds. were able to reverse resistance to oxacillin in highly resistant clin. MRSA strains at concns. far below their MICs.  Three of the compds. inhibited an Escherichia coli strain lacking the AcrAB components of a drug efflux pump, which suggests the lack of Gram-neg. activity can partly be attributed to efflux.  The compds. inhibited cell division by targeting S. aureus FtsZ, producing a dose-dependent increase in GTPase rate which increased the rate of FtsZ polymn. and stabilized the FtsZ polymers.  These compds. did not affect the polymn. of mammalian tubulin and did not display haemolytic activity or cytotoxicity.  These derivs. are therefore promising compds. for further development as antimicrobial agents or as resistance breakers to re-sensitive MRSA to beta-lactam antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtyDYtpkKW1rVg90H21EOLACvtfcHk0lgS-Q_znRHW8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXksF2msrs%253D&md5=23f78c3ab1957509c587879f4fda85cf</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3390%2Fantibiotics9120873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantibiotics9120873%26sid%3Dliteratum%253Aachs%26aulast%3DChai%26aufirst%3DW.%2BC.%26aulast%3DWhittall%26aufirst%3DJ.%2BJ.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DPolyak%26aufirst%3DS.%2BW.%26aulast%3DOgunniyi%26aufirst%3DA.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DSemple%26aufirst%3DS.%2BJ.%26aulast%3DVenter%26aufirst%3DH.%26atitle%3DAntimicrobial%2520action%2520and%2520reversal%2520of%2520resistance%2520in%2520MRSA%2520by%2520difluorobenzamide%2520derivatives%2520targeted%2520FtsZ%26jtitle%3DAntibiotics%26date%3D2020%26volume%3D9%26doi%3D10.3390%2Fantibiotics9120873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizohata, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoie, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, H.</span></span> <span> </span><span class="NLM_article-title">Structural flexibility of an inhibitor overcomes drug resistance mutations in Staphylococcus aureus FtsZ</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1947</span>– <span class="NLM_lpage">1955</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00323</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00323" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1947-1955&author=J.+Fujitaauthor=Y.+Maedaauthor=E.+Mizohataauthor=T.+Inoueauthor=M.+Kaulauthor=A.+K.+Parhiauthor=E.+J.+LaVoieauthor=D.+S.+Pilchauthor=H.+Matsumura&title=Structural+flexibility+of+an+inhibitor+overcomes+drug+resistance+mutations+in+Staphylococcus+aureus+FtsZ&doi=10.1021%2Facschembio.7b00323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Flexibility of an Inhibitor Overcomes Drug Resistance Mutations in Staphylococcus aureus FtsZ</span></div><div class="casAuthors">Fujita, Junso; Maeda, Yoko; Mizohata, Eiichi; Inoue, Tsuyoshi; Kaul, Malvika; Parhi, Ajit K.; LaVoie, Edmond J.; Pilch, Daniel S.; Matsumura, Hiroyoshi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1947-1955</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the effort to combat antibiotic resistance, inhibitors of the essential bacterial protein FtsZ have emerged as a promising new class of compds. with clin. potential.  One such FtsZ inhibitor (TXA707) is assocd. with potent activity against clin. isolates of methicillin-resistant Staphylococcus aureus (MRSA) that are resistant to current std.-of-care antibiotics.  However, mutations in S. aureus FtsZ (SaFtsZ) that confer resistance to TXA707 have been obsd., with mutations in the Gly196 and Gly193 residues being among the most prevalent.  Here, we describe structural studies of an FtsZ inhibitor, TXA6101, which retains activity against MRSA that express either G196S or G193D mutant FtsZ.  We present the crystal structures of TXA6101 in complex with both wildtype SaFtsZ and G196S mutant SaFtsZ, as well the crystal structure of TXA707 in complex with wildtype SaFtsZ.  Comparison of the three structures reveals a mol. basis for the differential targeting abilities of TXA6101 and TXA707.  The greater structural flexibility of TXA6101 relative to TXA707 enables TXA6101 to avoid steric clash with Ser196 and Asp193.  Our structures also demonstrate that the binding of TXA6101 induces previously unobserved conformational rearrangements of SaFtsZ residues in the binding pocket.  In the aggregate, the structures reported in this work reveal key factors for overcoming drug resistance mutations in SaFtsZ, and offer a structural basis for the design of FtsZ inhibitors with enhanced antibacterial potency and a reduced susceptibility to mutational resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCauV7BfLWLbVg90H21EOLACvtfcHk0lgFCYh4wRNVgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyks70%253D&md5=f1b5edb58bc3dbb3cc98e019075dafe3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00323%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DJ.%26aulast%3DMaeda%26aufirst%3DY.%26aulast%3DMizohata%26aufirst%3DE.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DKaul%26aufirst%3DM.%26aulast%3DParhi%26aufirst%3DA.%2BK.%26aulast%3DLaVoie%26aufirst%3DE.%2BJ.%26aulast%3DPilch%26aufirst%3DD.%2BS.%26aulast%3DMatsumura%26aufirst%3DH.%26atitle%3DStructural%2520flexibility%2520of%2520an%2520inhibitor%2520overcomes%2520drug%2520resistance%2520mutations%2520in%2520Staphylococcus%2520aureus%2520FtsZ%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1947%26epage%3D1955%26doi%3D10.1021%2Facschembio.7b00323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer-González, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Tameemi, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoie, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, D. S.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of a fluorescent probe for the visualization of FtsZ in clinically important gram-positive and gram-negative bacterial pathogens</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">20092</span> <span class="refDoi"> DOI: 10.1038/s41598-019-56557-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fs41598-019-56557-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=31882782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtFehtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&author=E.+Ferrer-Gonz%C3%A1lezauthor=J.+Fujitaauthor=T.+Yoshizawaauthor=J.+M.+Nelsonauthor=A.+J.+Pilchauthor=E.+Hillmanauthor=M.+Ozawaauthor=N.+Kurodaauthor=H.+M.+Al-Tameemiauthor=J.+M.+Boydauthor=E.+J.+LaVoieauthor=H.+Matsumuraauthor=D.+S.+Pilch&title=Structure-guided+design+of+a+fluorescent+probe+for+the+visualization+of+FtsZ+in+clinically+important+gram-positive+and+gram-negative+bacterial+pathogens&doi=10.1038%2Fs41598-019-56557-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of a Fluorescent Probe for the Visualization of FtsZ in Clinically Important Gram-Positive and Gram-Negative Bacterial Pathogens</span></div><div class="casAuthors">Ferrer-Gonzalez, Edgar; Fujita, Junso; Yoshizawa, Takuya; Nelson, Julia M.; Pilch, Alyssa J.; Hillman, Elani; Ozawa, Mayuki; Kuroda, Natsuko; Al-Tameemi, Hassan M.; Boyd, Jeffrey M.; La Voie, Edmond J.; Matsumura, Hiroyoshi; Pilch, Daniel S.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20092</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Addressing the growing problem of antibiotic resistance requires the development of new drugs with novel antibacterial targets.  FtsZ has been identified as an appealing new target for antibacterial agents.  Here, we describe the structure-guided design of a new fluorescent probe (BOFP) in which a BODIPY fluorophore has been conjugated to an oxazole-benzamide FtsZ inhibitor.  Crystallog. studies have enabled us to identify the optimal position for tethering the fluorophore that facilitates the high-affinity FtsZ binding of BOFP.  Fluorescence anisotropy studies demonstrate that BOFP binds the FtsZ proteins from the Gram-pos. pathogens Staphylococcus aureus, Enterococcus fecalis, Enterococcus faecium, Streptococcus pyogenes, Streptococcus agalactiae, and Streptococcus pneumoniae with Kd values of 0.6-4.6μM.  Significantly, BOFP binds the FtsZ proteins from the Gram-neg. pathogens Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa, and Acinetobacter baumannii with an even higher affinity (Kd = 0.2-0.8μM).  Fluorescence microscopy studies reveal that BOFP can effectively label FtsZ in all the above Gram-pos. and Gram-neg. pathogens.  In addn., BOFP is effective at monitoring the impact of non-fluorescent inhibitors on FtsZ localization in these target pathogens.  Viewed as a whole, our results highlight the utility of BOFP as a powerful tool for identifying new broad-spectrum FtsZ inhibitors and understanding their mechanisms of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbcIjEm4NzxLVg90H21EOLACvtfcHk0lgFCYh4wRNVgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtFehtw%253D%253D&md5=30d5745a77228c389567e02ce7fbc5d7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-56557-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-56557-x%26sid%3Dliteratum%253Aachs%26aulast%3DFerrer-Gonz%25C3%25A1lez%26aufirst%3DE.%26aulast%3DFujita%26aufirst%3DJ.%26aulast%3DYoshizawa%26aufirst%3DT.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DPilch%26aufirst%3DA.%2BJ.%26aulast%3DHillman%26aufirst%3DE.%26aulast%3DOzawa%26aufirst%3DM.%26aulast%3DKuroda%26aufirst%3DN.%26aulast%3DAl-Tameemi%26aufirst%3DH.%2BM.%26aulast%3DBoyd%26aufirst%3DJ.%2BM.%26aulast%3DLaVoie%26aufirst%3DE.%2BJ.%26aulast%3DMatsumura%26aufirst%3DH.%26aulast%3DPilch%26aufirst%3DD.%2BS.%26atitle%3DStructure-guided%2520design%2520of%2520a%2520fluorescent%2520probe%2520for%2520the%2520visualization%2520of%2520FtsZ%2520in%2520clinically%2520important%2520gram-positive%2520and%2520gram-negative%2520bacterial%2520pathogens%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26doi%3D10.1038%2Fs41598-019-56557-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yebenes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacios, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menendez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">The interactions of cell division protein FtsZ with guanine nucleotides</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">37515</span>– <span class="NLM_lpage">37528</span>, <span class="refDoi"> DOI: 10.1074/jbc.M706399200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M706399200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=17977836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGjtrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=37515-37528&author=S.+Huecasauthor=C.+Schaffner-Barberoauthor=W.+Garciaauthor=H.+Yebenesauthor=J.+M.+Palaciosauthor=J.+F.+Diazauthor=M.+Menendezauthor=J.+M.+Andreu&title=The+interactions+of+cell+division+protein+FtsZ+with+guanine+nucleotides&doi=10.1074%2Fjbc.M706399200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Interactions of Cell Division Protein FtsZ with Guanine Nucleotides</span></div><div class="casAuthors">Huecas, Sonia; Schaffner-Barbero, Claudia; Garcia, Wanius; Yebenes, Hugo; Palacios, Juan Manuel; Diaz, Jose Fernando; Menendez, Margarita; Andreu, Jose Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">37515-37528</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Prokaryotic cell division protein FtsZ, an assembling GTPase, directs the formation of the septosome between daughter cells.  FtsZ is an attractive target for the development of new antibiotics.  Assembly dynamics of FtsZ is regulated by the binding, hydrolysis, and exchange of GTP.  We have detd. the energetics of nucleotide binding to model apoFtsZ from Methanococcus jannaschii and studied the kinetics of 2'/3'-O-(N-methylanthraniloyl) (mant)-nucleotide binding and dissocn. from FtsZ polymers, employing calorimetric, fluorescence, and stopped-flow methods.  FtsZ binds GTP and GDP with Kb values ranging from 20 to 300 μM-1 under various conditions.  GTP·Mg2+ and GDP·Mg2+ bind with slightly reduced affinity.  Bound GTP and the coordinated Mg2+ ion play a minor structural role in FtsZ monomers, but Mg2+-assisted GTP hydrolysis triggers polymer disassembly.  Mant-GTP binds and dissocs. quickly from FtsZ monomers, with ∼10-fold lower affinity than GTP.  Mant-GTP displacement measured by fluorescence anisotropy provides a method to test the binding of any competing mols. to the FtsZ nucleotide site.  Mant-GTP is very slowly hydrolyzed and remains exchangeable in FtsZ polymers, but it becomes kinetically stabilized, with a 30-fold slower k+ and ∼500-fold slower k- than in monomers.  The mant-GTP dissocn. rate from FtsZ polymers is comparable with the GTP hydrolysis turnover and with the reported subunit turnover in Escherichia coli FtsZ polymers.  Although FtsZ polymers can exchange nucleotide, unlike its eukaryotic structural homolog tubulin, GDP dissocn. may be slow enough for polymer disassembly to take place first, resulting in FtsZ polymers cycling with GTP hydrolysis similarly to microtubules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUDpFp5TCovLVg90H21EOLACvtfcHk0lg4OjCUajWAow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGjtrjI&md5=05ab248863dbfc30d2a39c9c21180c27</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M706399200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M706399200%26sid%3Dliteratum%253Aachs%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DGarcia%26aufirst%3DW.%26aulast%3DYebenes%26aufirst%3DH.%26aulast%3DPalacios%26aufirst%3DJ.%2BM.%26aulast%3DDiaz%26aufirst%3DJ.%2BF.%26aulast%3DMenendez%26aufirst%3DM.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520interactions%2520of%2520cell%2520division%2520protein%2520FtsZ%2520with%2520guanine%2520nucleotides%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D37515%26epage%3D37528%26doi%3D10.1074%2Fjbc.M706399200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Redondo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lappchen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Blaauwen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morreale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Insights into nucleotide recognition by cell division protein FtsZ from a mant-GTP competition assay and molecular dynamics</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">10458</span>– <span class="NLM_lpage">10472</span>, <span class="refDoi"> DOI: 10.1021/bi101577p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi101577p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVaqsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=10458-10472&author=C.+Schaffner-Barberoauthor=R.+Gil-Redondoauthor=L.+B.+Ruiz-Avilaauthor=S.+Huecasauthor=T.+Lappchenauthor=T.+den+Blaauwenauthor=J.+F.+Diazauthor=A.+Morrealeauthor=J.+M.+Andreu&title=Insights+into+nucleotide+recognition+by+cell+division+protein+FtsZ+from+a+mant-GTP+competition+assay+and+molecular+dynamics&doi=10.1021%2Fbi101577p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into Nucleotide Recognition by Cell Division Protein FtsZ from a mant-GTP: Competition Assay and Molecular Dynamics</span></div><div class="casAuthors">Schaffner-Barbero, Claudia; Gil-Redondo, Ruben; Ruiz-Avila, Laura B.; Huecas, Sonia; Lappchen, Tilman; den Blaauwen, Tanneke; Diaz, J. Fernando; Morreale, Antonio; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">10458-10472</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Essential cell division protein FtsZ forms the bacterial cytokinetic ring and is a target for new antibiotics.  FtsZ monomers bind GTP and assemble into filaments.  Hydrolysis to GDP at the assocn. interface between monomers leads to filament disassembly.  We have developed a homogeneous competition assay, employing the fluorescence anisotropy change of mant-GTP upon binding to nucleotide-free FtsZ, which detects compds. binding to the nucleotide site in FtsZ monomers and measures their affinities within the millimolar to 10 nM range.  We have employed this method to det. the apparent contributions of the guanine, ribose, and the α-, β-, and γ-phosphates to the free energy change of nucleotide binding.  Similar relative contributions have also been estd. through mol. dynamics and binding free energy calcns., employing the crystal structures of FtsZ-nucleotide complexes.  We find an energetically dominant contribution of the β-phosphate, comparable to the whole guanosine moiety.  GTP and GDP bind with similar obsd. affinity to FtsZ monomers.  Loss of the regulatory γ-phosphate results in a predicted accommodation of GDP which has not been obsd. in the crystal structures.  The binding affinities of a series of C8-substituted GTP analogs, known to inhibit FtsZ but not eukaryotic tubulin assembly, correlate with their inhibitory capacity on FtsZ polymn.  Our methods permit testing of FtsZ inhibitors targeting its nucleotide site, as well as compds. from virtual screening of large synthetic libraries.  Our results give insight into the FtsZ-nucleotide interactions, which could be useful in the rational design of new inhibitors, esp. GTP phosphate mimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0eGYV6QbGTrVg90H21EOLACvtfcHk0lg4OjCUajWAow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVaqsL7M&md5=109e4d4a0a2365c5dc3db9d5642963d6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fbi101577p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi101577p%26sid%3Dliteratum%253Aachs%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DGil-Redondo%26aufirst%3DR.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DLappchen%26aufirst%3DT.%26aulast%3Dden%2BBlaauwen%26aufirst%3DT.%26aulast%3DDiaz%26aufirst%3DJ.%2BF.%26aulast%3DMorreale%26aufirst%3DA.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DInsights%2520into%2520nucleotide%2520recognition%2520by%2520cell%2520division%2520protein%2520FtsZ%2520from%2520a%2520mant-GTP%2520competition%2520assay%2520and%2520molecular%2520dynamics%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D10458%26epage%3D10472%26doi%3D10.1021%2Fbi101577p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergonos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araujo-Bazan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Villa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turrado, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nodwell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barasoain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span> <span> </span><span class="NLM_article-title">Effective GTP-replacing FtsZ inhibitors and antibacterial mechanism of action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">843</span>, <span class="refDoi"> DOI: 10.1021/cb500974d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500974d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVClsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=834-843&author=M.+Artolaauthor=L.+B.+Ruiz-Avilaauthor=A.+Vergonosauthor=S.+Huecasauthor=L.+Araujo-Bazanauthor=M.+Martin-Fontechaauthor=H.+Vazquez-Villaauthor=C.+Turradoauthor=E.+Ramirez-Aportelaauthor=A.+Hoeglauthor=M.+Nodwellauthor=I.+Barasoainauthor=P.+Chaconauthor=S.+A.+Sieberauthor=J.+M.+Andreuauthor=M.+L.+Lopez-Rodriguez&title=Effective+GTP-replacing+FtsZ+inhibitors+and+antibacterial+mechanism+of+action&doi=10.1021%2Fcb500974d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Effective GTP-replacing FtsZ inhibitors and antibacterial mechanism of action</span></div><div class="casAuthors">Artola, Marta; Ruiz-Avila, Laura B.; Vergonos, Albert; Huecas, Sonia; Araujo-Bazan, Lidia; Martin-Fontecha, Mar; Vazquez-Villa, Henar; Turrado, Carlos; Ramirez-Aportela, Erney; Hoegl, Annabelle; Nodwell, Matthew; Barasoain, Isabel; Chacon, Pablo; Sieber, Stephan A.; Andreu, Jose M.; Lopez-Rodriguez, Maria L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">834-843</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The essential cell division protein FtsZ is considered an attractive target in the search for antibacterials with novel mechanisms of action to overcome the resistance problem.  FtsZ undergoes GTP-dependent assembly at midcell to form the Z-ring, a dynamic structure that evolves until final constriction of the cell.  Therefore, mols. able to inhibit its activity will eventually disrupt bacterial viability.  The authors report a new series of small mols. able to replace GTP and to specifically inhibit FtsZ, blocking the bacterial division process.  These new synthesized inhibitors interact with the GTP-binding site of FtsZ (Kd = 0.4-0.8 μM), display antibacterial activity against Gram-pos. pathogenic bacteria, and show selectivity against tubulin.  Biphenyl deriv. 28 stands out as a potent FtsZ inhibitor (Kd = 0.5 μM) with high antibacterial activity [MIC (MRSA) = 7 μM].  In depth anal. of the mechanism of action of compds. 22, 28, 33, and 36 has revealed that they act as effective inhibitors of correct FtsZ assembly, blocking bacterial division and thus leading to filamentous undivided cells.  These findings provide a compelling rationale for the development of compds. targeting the GTP-binding site as antibacterial agents and open the door to antibiotics with novel mechanisms of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPmkh6dxtlrbVg90H21EOLACvtfcHk0lg4OjCUajWAow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVClsLrP&md5=95534237838bac7f7591d50521c9afe9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fcb500974d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500974d%26sid%3Dliteratum%253Aachs%26aulast%3DArtola%26aufirst%3DM.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DVergonos%26aufirst%3DA.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAraujo-Bazan%26aufirst%3DL.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DVazquez-Villa%26aufirst%3DH.%26aulast%3DTurrado%26aufirst%3DC.%26aulast%3DRamirez-Aportela%26aufirst%3DE.%26aulast%3DHoegl%26aufirst%3DA.%26aulast%3DNodwell%26aufirst%3DM.%26aulast%3DBarasoain%26aufirst%3DI.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26atitle%3DEffective%2520GTP-replacing%2520FtsZ%2520inhibitors%2520and%2520antibacterial%2520mechanism%2520of%2520action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D834%26epage%3D843%26doi%3D10.1021%2Fcb500974d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marín-Ramos, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balabasquer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega-Nogales, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrecillas, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Ordóñez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcos-Ramiro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar-Garrido, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medrano, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajate, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollinedo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallardo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Rodríguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega-Gutiérrez, S.</span></span> <span> </span><span class="NLM_article-title">A potent isoprenylcysteine carboxylmethyltransferase (ICMT) inhibitor improves survival in Ras-driven acute myeloid leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6035</span>– <span class="NLM_lpage">6046</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00145</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00145" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFaqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6035-6046&author=N.+I.+Mar%C3%ADn-Ramosauthor=M.+Balabasquerauthor=F.+J.+Ortega-Nogalesauthor=I.+R.+Torrecillasauthor=A.+Gil-Ord%C3%B3%C3%B1ezauthor=B.+Marcos-Ramiroauthor=P.+Aguilar-Garridoauthor=I.+Cushmanauthor=A.+Romeroauthor=F.+J.+Medranoauthor=C.+Gajateauthor=F.+Mollinedoauthor=M.+R.+Philipsauthor=M.+Campilloauthor=M.+Gallardoauthor=M.+Mart%C3%ADn-Fontechaauthor=M.+L.+L%C3%B3pez-Rodr%C3%ADguezauthor=S.+Ortega-Guti%C3%A9rrez&title=A+potent+isoprenylcysteine+carboxylmethyltransferase+%28ICMT%29+inhibitor+improves+survival+in+Ras-driven+acute+myeloid+leukemia&doi=10.1021%2Facs.jmedchem.9b00145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent Isoprenylcysteine Carboxylmethyltransferase (ICMT) Inhibitor Improves Survival in Ras-Driven Acute Myeloid Leukemia</span></div><div class="casAuthors">Marin-Ramos, Nagore I.; Balabasquer, Moises; Ortega-Nogales, Francisco J.; Torrecillas, Ivan R.; Gil-Ordonez, Ana; Marcos-Ramiro, Beatriz; Aguilar-Garrido, Pedro; Cushman, Ian; Romero, Antonio; Medrano, Francisco J.; Gajate, Consuelo; Mollinedo, Faustino; Philips, Mark R.; Campillo, Mercedes; Gallardo, Miguel; Martin-Fontecha, Mar; Lopez-Rodriguez, Maria L.; Ortega-Gutierrez, Silvia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6035-6046</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Blockade of Ras activity by inhibiting its post-translational methylation catalyzed by isoprenylcysteine carboxylmethyltransferase (ICMT) has been suggested as a promising antitumor strategy.  However, the paucity of inhibitors has precluded the clin. validation of this approach.  In this work we report a potent ICMT inhibitor, compd. 3 [UCM-1336, IC50 = 2 μM], which is selective against the other enzymes involved in the post-translational modifications of Ras.  Compd. 3 significantly impairs the membrane assocn. of the four Ras isoforms, leading to a decrease of Ras activity and to inhibition of Ras downstream signaling pathways.  In addn., it induces cell death in a variety of Ras-mutated tumor cell lines and increases survival in an in vivo model of acute myeloid leukemia.  Because ICMT inhibition impairs the activity of the four Ras isoforms regardless of its activating mutation, compd. 3 surmounts many of the common limitations of available Ras inhibitors described so far.  In addn., these results validate ICMT as a valuable target for the treatment of Ras-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQdAP3zRIM7Vg90H21EOLACvtfcHk0lgE51h10mQCmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFaqsbzP&md5=99eacad9ac479cc0604a46b66d039e8e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00145%26sid%3Dliteratum%253Aachs%26aulast%3DMar%25C3%25ADn-Ramos%26aufirst%3DN.%2BI.%26aulast%3DBalabasquer%26aufirst%3DM.%26aulast%3DOrtega-Nogales%26aufirst%3DF.%2BJ.%26aulast%3DTorrecillas%26aufirst%3DI.%2BR.%26aulast%3DGil-Ord%25C3%25B3%25C3%25B1ez%26aufirst%3DA.%26aulast%3DMarcos-Ramiro%26aufirst%3DB.%26aulast%3DAguilar-Garrido%26aufirst%3DP.%26aulast%3DCushman%26aufirst%3DI.%26aulast%3DRomero%26aufirst%3DA.%26aulast%3DMedrano%26aufirst%3DF.%2BJ.%26aulast%3DGajate%26aufirst%3DC.%26aulast%3DMollinedo%26aufirst%3DF.%26aulast%3DPhilips%26aufirst%3DM.%2BR.%26aulast%3DCampillo%26aufirst%3DM.%26aulast%3DGallardo%26aufirst%3DM.%26aulast%3DMart%25C3%25ADn-Fontecha%26aufirst%3DM.%26aulast%3DL%25C3%25B3pez-Rodr%25C3%25ADguez%26aufirst%3DM.%2BL.%26aulast%3DOrtega-Guti%25C3%25A9rrez%26aufirst%3DS.%26atitle%3DA%2520potent%2520isoprenylcysteine%2520carboxylmethyltransferase%2520%2528ICMT%2529%2520inhibitor%2520improves%2520survival%2520in%2520Ras-driven%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6035%26epage%3D6046%26doi%3D10.1021%2Facs.jmedchem.9b00145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgE51h10mQCmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh9VOfoh1mJOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh9VOfoh1mJOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BunkoczNi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+BunkoczNiauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lh9VOfoh1mJOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczNi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fluorescent probes designed for the allosteric binding site of FtsZ. (A) Representation of the core structure of <i>S. aureus</i> FtsZ (SaFtsZ) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD">4DXD</a>), showing the nucleotide (red-green spheres) in its binding pocket and PC190723 (green-blue-red spheres) bound into a cleft between the N-terminal and C-terminal domains. The yellow circle marks a cavity above PC190723 potentially available for ligand binding and the positions of helices H6 and H7 within the structure are highlighted. (B) Zoom of the encircled surface. (C) Chemical structures of PC190723, compound <b>1</b>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and the fluorescent probes <b>2</b>–<b>5</b> employed in this work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of compounds <b>6</b>–<b>26</b> evaluated as FtsZ inhibitors in this work.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>2</b>−<b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>28</b>–<b>30</b>, K<sub>2</sub>CO<sub>3</sub>, NaI, dimethylformamide (DMF), rt, 24 h, 14–23%; (b) (<i>R</i>)- or (<i>S</i>)-<b>31</b>, diisopropyl azodicarboxylate (DIAD), PBu<sub>3</sub>, DMF, MW, 150 °C, 2 h, 28–31%; (c) trifluoroacetic acid (TFA), dichloromethane (DCM), rt, 1 h, 70–90%; (d) NBD-Cl, Cs<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 1 h, 7–50%; and (e) <i>rac</i>-, (<i>R</i>)-, or (<i>S</i>)-<b>38</b>, DIAD, PBu<sub>3</sub>, DMF, 80 °C, 48 h, 18–19%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>6</b>−<b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) 3-chlorobenzaldehyde, MW, 100 °C, 10 min; (ii) H<sub>2</sub>, Pd/C, MeOH, rt, 18 h, 32–74%; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 60 °C, 18 h, 31–35%; (c) (i) CH<sub>3</sub>CH<sub>2</sub>CHO, MeOH, rt, 18 h; (ii) NaBH<sub>3</sub>CN, MeOH, rt, 72 h, 52%; (d) K<sub>2</sub>CO<sub>3</sub>, NaI, DMF, rt, 24 h, 47–80%; and (e) PPh<sub>3</sub>, diethyl azodicarboxylate (DEAD), Et<sub>3</sub>N, tetrahydrofuran (THF), rt, 24 h, 46%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>15</b>−<b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) TMSCF<sub>3</sub>, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 48 h; (ii) 2 M HCl, rt, 4 h, 87%; (b) Dess–Martin periodinane, DCM, rt, 18 h, 99%; (c) (i) (CH<sub>3</sub>)<sub>3</sub>CSONH<sub>2</sub>, Ti(OiPr)<sub>4</sub>, Et<sub>2</sub>O, reflux, 24 h; (ii) NaBH<sub>4</sub>, Et<sub>2</sub>O, rt, 24 h; (iii) 4 M HCl, 1,4-dioxane, rt, 1 h, 24%; (d) Pd(OAc)<sub>2</sub>, CuI, DMF, MW, 200 °C, 40 min, 53%; (e) BBr<sub>3</sub>, DCM, 0 °C to rt, 24 h, 95%; and (f) 3-chlorobenzyl bromide or 3-(trifluoromethyl)benzyl bromide, K<sub>2</sub>CO<sub>3</sub>, NaI, DMF, rt, 24 h, 72–75%.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0011.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>18</b>−<b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, NaI, DMF, rt, 24 h, 81–98%; (b) PBu<sub>3</sub>, DIAD, DMF, MW, 150 °C, 1.5 h, 45–50%; (c) K<sub>3</sub>PO<sub>4</sub>, CuI, (CH<sub>2</sub>OH)<sub>2</sub>, <i>i</i>-PrOH, MW, 150 °C, 105 min, 46–56%; and (d) LiAlH<sub>4</sub>, THF, rt, 2 h, 76%.</p></p></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Specific binding of fluorescent probes to FtsZ′s allosteric site, activity on bacterial cells, and competitive binding assay for ligands binding to the site. (A) Fluorescence anisotropy values of the free probes (10 μM) in buffer, to which FtsZ (10 μM, unassembled) was added, followed by the slowly hydrolyzable GTP analogue GMPCPP (0.1 mM). BsFtsZ polymerization was then induced by the addition of MgCl<sub>2</sub> (10 mM) and anisotropy values were recorded 5–10 min later. Nonfluorescent PC190723 (10 μM) was subsequently added to displace the fluorescent probe and anisotropy was measured 5 min later. (B) Fluorescence microscopy images of <i>B. subtilis</i> cells treated for 1.5 h with the minimal division inhibitory concentrations (MDC) of (<i>S</i>)-<b>2</b> and each enantiomer of <b>5</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S1</a>), acquired employing the probe fluorescence. (C) Displacement curves of (<i>S</i>)-<b>2</b> (3 μM) from stabilized FtsZ-GMPCPP polymers (8 μM binding sites) by PC190723 (black circles, <i>K</i><sub>D</sub> < 0.1 μM), and its moieties 2,6-difluoro-3-methoxybenzamide (DFMBA, <i>K</i><sub>D</sub> = 0.67 ± 0.05 mM) (red), and 6-(chloro[1,3]thiazolo[5,4-<i>b</i>]pyridin-2-yl)methanol (CTPM, <i>K</i><sub>D</sub> = 1.0 ± 0.1 mM) (blue). Solid lines are best fits to the data of a single site competition model (<a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>). The addition of guanosine diphosphate (GDP) (void circles) does not displace (<i>S</i>)-<b>2</b> from its binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fluorescence anisotropy screen for FtsZ allosteric inhibitors combined with cell-based methods. (A) Screen for FtsZ inhibitors employing the fluorescence anisotropy of probe (<i>S</i>)-<b>2</b> (3 μM) and stabilized FtsZ-GMPCPP polymers (8 μM binding sites). The anisotropy values of the partially bound probe (void black circles), the free probe (void gray circles), and the values with two concentrations of each compound, 20 μM (gray circles) and 200 μM (black circles) are shown. Control values with the corresponding concentrations of dimethyl sulfoxide (DMSO) vehicle (2% void and gray circles; 4%, black circle) are also shown. (B) Chemical structures of synthetic ligands <b>6</b>–<b>26</b>. (C) Effects of low-affinity inhibitors <b>6</b>, <b>7</b>, and <b>9</b> on <i>B. subtilis</i> 168 cell division. Raw data plots (<i>n</i> ≥ 30) are shown with the mean values marked (statistics in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Table S2</a>). The cells were incubated for 3 h with each compound, and cell lengths were measured. (D) Representative phase-contrast images of cells after 3 h with <b>7</b> or <b>9</b> (0.75 mM each). (E) Cellular localization of FtsZ-GFP in <i>B. subtilis</i> SU570 following growth for 1.5 h in the absence (inset) and presence of <b>7</b> (0.2 mM). Bars, 10 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Simplified benzamide FtsZ inhibitors with enhanced affinity and phenotype. (A) Displacement curves of probe (<i>R</i>)-<b>5</b> (3 μM) from stabilized BsFtsZ-GMPCPP polymers (2.7 μM binding sites) by synthetic compounds <b>11</b> (black), <b>20</b> (blue), <b>25</b> (green), (<i>R</i>)-<b>26</b> (red), (<i>S</i>)-<b>26</b> (cyan), and by PC190723 (gray). Lines correspond to the best-fitted competition curve in each case. See <i>K</i><sub>D</sub> values in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. (B) Left column, <i>B. subtilis</i> 168 cells after 3 h of culture with <b>20</b>, <b>25</b>, and (<i>R</i>)- and (<i>S</i>)-<b>26</b>, observed by phase-contrast microscopy; right column, cellular localization of FtsZ-GFP (<i>B. subtilis</i> SU570) following growth for 1.5 h in the presence of the same inhibitors. (C) <i>S. aureus</i> cells expressing FtsZ-mCherry (RN4220 + pCN-ftsZmch) grown for 1.5 h in the absence (control) or presence of <b>20</b>, <b>25</b>, (<i>R</i>)-, (<i>S</i>)-<b>26</b>, and PC190723, observed by fluorescence microscopy. (D) Top, structures of inhibitors <b>20</b>, <b>25</b>, and (<i>R</i>)-<b>26</b>. Bottom, mapping on the SaFtsZ structure (gray cartoon diagram of PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VOA">3VOA</a>) of the amino acid residues modified in mutants resistant to <b>20</b> (blue), <b>25</b> (green), and (<i>R</i>)-<b>26</b> (red spheres).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/medium/jm0c02207_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structures of FtsZ-inhibitor complexes. (A) Ligand–protein interactions in the binding site, indicated by dashed magenta lines (see the main text and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf" class="ext-link">Figure S6</a>; the green sphere represents a coordinated metal ion). Complex structures have been deposited at the PDB with accession numbers: SaFtsZ-DFMBA, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD1">6YD1</a>; SaFtsZ-<b>18</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD5">6YD5</a>; SaFtsZ-<b>20</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD6">6YD6</a>. (B) Binding site surface, shown in light gray around the ligands. Magenta surface: Thr309 and Gly196. Cyan surface: Met226 and Gly193. (C) Structural comparison between the SaFtsZ-<b>20</b> (gray) complex with SaFtsZ-<b>18</b>, PC190723 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD">4DXD</a>), and TXA6101 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XDU">5XDU</a>) complexes (pink carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02207/20210506/images/large/jm0c02207_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02207&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21544" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21544" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogberg, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plachouras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoxha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonsen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colomb-Cotinat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzschmar, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devleesschauwer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouakrim, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struelens, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suetens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group">the Burden of AMR Collaborative Group</span> <span> </span><span class="NLM_article-title">Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(18)30605-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2FS1473-3099%2818%2930605-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30409683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=56-66&author=A.+Cassiniauthor=L.+D.+Hogbergauthor=D.+Plachourasauthor=A.+Quattrocchiauthor=A.+Hoxhaauthor=G.+S.+Simonsenauthor=M.+Colomb-Cotinatauthor=M.+E.+Kretzschmarauthor=B.+Devleesschauwerauthor=M.+Cecchiniauthor=D.+A.+Ouakrimauthor=T.+C.+Oliveiraauthor=M.+J.+Struelensauthor=C.+Suetensauthor=D.+L.+Monnetauthor=the+Burden+of+AMR+Collaborative+Group&title=Attributable+deaths+and+disability-adjusted+life-years+caused+by+infections+with+antibiotic-resistant+bacteria+in+the+EU+and+the+European+Economic+Area+in+2015%3A+a+population-level+modelling+analysis&doi=10.1016%2FS1473-3099%2818%2930605-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis</span></div><div class="casAuthors">Cassini Alessandro; Hogberg Liselotte Diaz; Plachouras Diamantis; Quattrocchi Annalisa; Hoxha Ana; Struelens Marc J; Suetens Carl; Monnet Dominique L; Simonsen Gunnar Skov; Colomb-Cotinat Melanie; Kretzschmar Mirjam E; Devleesschauwer Brecht; Cecchini Michele; Ouakrim Driss Ait; Oliveira Tiago Cravo</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-66</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Infections due to antibiotic-resistant bacteria are threatening modern health care.  However, estimating their incidence, complications, and attributable mortality is challenging.  We aimed to estimate the burden of infections caused by antibiotic-resistant bacteria of public health concern in countries of the EU and European Economic Area (EEA) in 2015, measured in number of cases, attributable deaths, and disability-adjusted life-years (DALYs).  METHODS:  We estimated the incidence of infections with 16 antibiotic resistance-bacterium combinations from European Antimicrobial Resistance Surveillance Network (EARS-Net) 2015 data that was country-corrected for population coverage.  We multiplied the number of bloodstream infections (BSIs) by a conversion factor derived from the European Centre for Disease Prevention and Control point prevalence survey of health-care-associated infections in European acute care hospitals in 2011-12 to estimate the number of non-BSIs.  We developed disease outcome models for five types of infection on the basis of systematic reviews of the literature.  FINDINGS:  From EARS-Net data collected between Jan 1, 2015, and Dec 31, 2015, we estimated 671 689 (95% uncertainty interval [UI] 583 148-763 966) infections with antibiotic-resistant bacteria, of which 63·5% (426 277 of 671 689) were associated with health care.  These infections accounted for an estimated 33 110 (28 480-38 430) attributable deaths and 874 541 (768 837-989 068) DALYs.  The burden for the EU and EEA was highest in infants (aged <1 year) and people aged 65 years or older, had increased since 2007, and was highest in Italy and Greece.  INTERPRETATION:  Our results present the health burden of five types of infection with antibiotic-resistant bacteria expressed, for the first time, in DALYs.  The estimated burden of infections with antibiotic-resistant bacteria in the EU and EEA is substantial compared with that of other infectious diseases, and has increased since 2007.  Our burden estimates provide useful information for public health decision-makers prioritising interventions for infectious diseases.  FUNDING:  European Centre for Disease Prevention and Control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR76qXWYwPPXq69lAbTj9rGfW6udTcc2eZBi1jQb_tE87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D&md5=151619810730b07e3d84692f1d966b9a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2818%2930605-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252818%252930605-4%26sid%3Dliteratum%253Aachs%26aulast%3DCassini%26aufirst%3DA.%26aulast%3DHogberg%26aufirst%3DL.%2BD.%26aulast%3DPlachouras%26aufirst%3DD.%26aulast%3DQuattrocchi%26aufirst%3DA.%26aulast%3DHoxha%26aufirst%3DA.%26aulast%3DSimonsen%26aufirst%3DG.%2BS.%26aulast%3DColomb-Cotinat%26aufirst%3DM.%26aulast%3DKretzschmar%26aufirst%3DM.%2BE.%26aulast%3DDevleesschauwer%26aufirst%3DB.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DOuakrim%26aufirst%3DD.%2BA.%26aulast%3DOliveira%26aufirst%3DT.%2BC.%26aulast%3DStruelens%26aufirst%3DM.%2BJ.%26aulast%3DSuetens%26aufirst%3DC.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3D%26atitle%3DAttributable%2520deaths%2520and%2520disability-adjusted%2520life-years%2520caused%2520by%2520infections%2520with%2520antibiotic-resistant%2520bacteria%2520in%2520the%2520EU%2520and%2520the%2520European%2520Economic%2520Area%2520in%25202015%253A%2520a%2520population-level%2520modelling%2520analysis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2019%26volume%3D19%26spage%3D56%26epage%3D66%26doi%3D10.1016%2FS1473-3099%2818%2930605-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foster, T. J.</span></span> <span> </span><span class="NLM_article-title">Can β-lactam antibiotics be resurrected to combat MRSA?</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2018.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.tim.2018.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30031590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yitrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=26-38&author=T.+J.+Foster&title=Can+%CE%B2-lactam+antibiotics+be+resurrected+to+combat+MRSA%3F&doi=10.1016%2Fj.tim.2018.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Can β-Lactam Antibiotics Be Resurrected to Combat MRSA?</span></div><div class="casAuthors">Foster, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26-38</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of β -lactam antibiotics to treat infections caused by Staphylococcus aureus has been severely compromised by the acquisition by horizontal gene transfer of a gene that encodes the β -lactam-insensitive penicillin-binding protein PBP2a.  This allows methicillin-resistant S. aureus (MRSA) to proliferate in the presence of β -lactam antibiotics.  Paradoxically the dependence on PBP2a for the essential transpeptidase activity in cell wall peptidoglycan biosynthesis is the 'Achilles heel' of MRSA.  Compds. that disrupt the divisome, wall teichoic acid, and functional membrane microdomains act synergistically with β -lactams against MRSA.  These include drugs such as statins that are widely used in human medicine.  The antibiotics vancomycin and daptomycin are also synergistic with β -lactams, and combinations have been employed to treat persistent MRSA infections.  An addnl. benefit of exposing MRSA to β -lactams could be a redn. in virulence mediated by interfering with the global regulator Agr.  The mechanistic basis of synergy is discussed, and the possibility that β -lactams can be resurrected to combat MRSA infections is explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOd6SVTdIQqbVg90H21EOLACvtfcHk0lhVgoTSVbXnWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yitrbN&md5=72df9b9e79c6e0de8b757d0cf50bcebc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2018.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2018.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DFoster%26aufirst%3DT.%2BJ.%26atitle%3DCan%2520%25CE%25B2-lactam%2520antibiotics%2520be%2520resurrected%2520to%2520combat%2520MRSA%253F%26jtitle%3DTrends%2520Microbiol.%26date%3D2019%26volume%3D27%26spage%3D26%26epage%3D38%26doi%3D10.1016%2Fj.tim.2018.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>Antibiotic Resistance 2020</i>; <span class="NLM_publisher-name">WHO</span>, <span class="NLM_year">2020</span>. Can be found under: <a href="https://www.who.int/news-room/fact-sheets/detail/antibiotic-resistance" class="extLink">https://www.who.int/news-room/fact-sheets/detail/antibiotic-resistance</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=World+Health+Organization&title=Antibiotic+Resistance+2020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DAntibiotic%2520Resistance%25202020%26pub%3DWHO%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Centers for Disease Control and Prevention</span>.  <i>2019
Antibiotic Resistance Threats Report</i>; <span class="NLM_publisher-name">WHO</span>, <span class="NLM_year">2019</span>. Can be found under: <a href="https://www.cdc.gov/drugresistance/biggest-threats.html" class="extLink">https://www.cdc.gov/drugresistance/biggest-threats.html</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Centers+for+Disease+Control+and+Prevention&title=2019%0AAntibiotic+Resistance+Threats+Report"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3D2019%250AAntibiotic%2520Resistance%2520Threats%2520Report%26pub%3DWHO%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutkenhaus, J.</span></span> <span> </span><span class="NLM_article-title">FtsZ ring structure associated with division in <i>Escherichia coli</i></span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>354</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1038/354161a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2F354161a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1944597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADyaK38XlsFeltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=354&publication_year=1991&pages=161-164&author=E.+Biauthor=J.+Lutkenhaus&title=FtsZ+ring+structure+associated+with+division+in+Escherichia+coli&doi=10.1038%2F354161a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">FtsZ ring structure associated with division in Escherichia coli</span></div><div class="casAuthors">Bi, Erfei; Lutkenhaus, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">354</span>
        (<span class="NLM_cas:issue">6349</span>),
    <span class="NLM_cas:pages">161-4</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Genes for cell division have been identified in E. coli by the isolation of conditional lethal mutations that block cell division, but do not affect DNA replication or segregation.  Of these genes, ftsZ is of great interest as it acts earliest in the division pathway, is essential, its level dictates the frequency of division, and it is thought to be the target of two cell-division inhibitors, SulA, produced in response to DNA damage, and MinCD, which prevents division at old sites.  Immunoelectromicroscopy was used to localize the FtsZ protein to the division site.  The results suggest that FtsZ self-assembles into a ring structure at the future division site and may function as a cytoskeletal element.  The formation of this ring may be the point at which division is regulated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGys1rwzKlK7Vg90H21EOLACvtfcHk0lhVgoTSVbXnWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFeltg%253D%253D&md5=b7ea7db9fd481d35d2eea461f291d223</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F354161a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F354161a0%26sid%3Dliteratum%253Aachs%26aulast%3DBi%26aufirst%3DE.%26aulast%3DLutkenhaus%26aufirst%3DJ.%26atitle%3DFtsZ%2520ring%2520structure%2520associated%2520with%2520division%2520in%2520Escherichia%2520coli%26jtitle%3DNature%26date%3D1991%26volume%3D354%26spage%3D161%26epage%3D164%26doi%3D10.1038%2F354161a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">den
Blaauwen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamoen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, P. A.</span></span> <span> </span><span class="NLM_article-title">The divisome at 25: the road ahead</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2017.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.mib.2017.01.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28254403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2rs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=85-94&author=T.+den%0ABlaauwenauthor=L.+Hamoenauthor=P.+A.+Levin&title=The+divisome+at+25%3A+the+road+ahead&doi=10.1016%2Fj.mib.2017.01.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The divisome at 25: the road ahead</span></div><div class="casAuthors">den Blaauwen, Tanneke; Hamoen, Leendert W.; Levin, Petra Anne</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  At the same time, significant questions about the mechanisms coordinating division with cell growth, DNA replication, and chromosome segregation remain.  Not only did the presence of the FtsZ ring provide fodder for the next 25 years of research, the application of a then cutting-edge approach-immunogold labeling of bacterial cells-inspired other investigators to apply similarly state-of-the-art technologies in their own work.  The identification of the FtsZ ring by Bi and Lutkenhaus in 1991 was a defining moment for the field of bacterial cell division.  These efforts have led to important advances in our understanding of the factors underlying assembly and maintenance of the division machinery.  This review addresses the most prominent of these questions, setting the stage for the next 25 years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCo3OTjAkkpbVg90H21EOLACvtfcHk0lhVgoTSVbXnWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2rs70%253D&md5=be73c3e3d85f3c51882065d82764202e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2017.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2017.01.007%26sid%3Dliteratum%253Aachs%26aulast%3Dden%2BBlaauwen%26aufirst%3DT.%26aulast%3DHamoen%26aufirst%3DL.%26aulast%3DLevin%26aufirst%3DP.%2BA.%26atitle%3DThe%2520divisome%2520at%252025%253A%2520the%2520road%2520ahead%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2017%26volume%3D36%26spage%3D85%26epage%3D94%26doi%3D10.1016%2Fj.mib.2017.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisson-Filho, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squyres, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuru, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jukes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanNieuwenhze, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, E. C.</span></span> <span> </span><span class="NLM_article-title">Treadmilling by FtsZ filaments drives peptidoglycan synthesis and bacterial cell division</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">743</span>, <span class="refDoi"> DOI: 10.1126/science.aak9973</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscience.aak9973" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28209898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=739-743&author=A.+W.+Bisson-Filhoauthor=Y.+P.+Hsuauthor=G.+R.+Squyresauthor=E.+Kuruauthor=F.+Wuauthor=C.+Jukesauthor=Y.+Sunauthor=C.+Dekkerauthor=S.+Holdenauthor=M.+S.+VanNieuwenhzeauthor=Y.+V.+Brunauthor=E.+C.+Garner&title=Treadmilling+by+FtsZ+filaments+drives+peptidoglycan+synthesis+and+bacterial+cell+division&doi=10.1126%2Fscience.aak9973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Treadmilling by FtsZ filaments drives peptidoglycan synthesis and bacterial cell division</span></div><div class="casAuthors">Bisson-Filho, Alexandre W.; Hsu, Yen-Pang; Squyres, Georgia R.; Kuru, Erkin; Wu, Fabai; Jukes, Calum; Sun, Yingjie; Dekker, Cees; Holden, Seamus; Van Nieuwenhze, Michael S.; Brun, Yves V.; Garner, Ethan C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6326</span>),
    <span class="NLM_cas:pages">739-743</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The mechanism by which bacteria divide is not well understood.  Cell division is mediated by filaments of FtsZ and FtsA (FtsAZ) that recruit septal peptidoglycan-synthesizing enzymes to the division site.  To understand how these components coordinate to divide cells, we visualized their movements relative to the dynamics of cell wall synthesis during cytokinesis.  The division septum was built at discrete sites that moved around the division plane.  FtsAZ filaments treadmilled circumferentially around the division ring and drove the motions of the peptidoglycan-synthesizing enzymes.  The FtsZ treadmilling rate controlled both the rate of peptidoglycan synthesis and cell division.  Thus, FtsZ treadmilling guides the progressive insertion of new cell wall by building increasingly smaller concentric rings of peptidoglycan to divide the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVjGpjX3axorVg90H21EOLACvtfcHk0lidRUdOwB3miw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yjtbc%253D&md5=3e37ca023a21ac1d08bbb87e2aeb496e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.aak9973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aak9973%26sid%3Dliteratum%253Aachs%26aulast%3DBisson-Filho%26aufirst%3DA.%2BW.%26aulast%3DHsu%26aufirst%3DY.%2BP.%26aulast%3DSquyres%26aufirst%3DG.%2BR.%26aulast%3DKuru%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DJukes%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DDekker%26aufirst%3DC.%26aulast%3DHolden%26aufirst%3DS.%26aulast%3DVanNieuwenhze%26aufirst%3DM.%2BS.%26aulast%3DBrun%26aufirst%3DY.%2BV.%26aulast%3DGarner%26aufirst%3DE.%2BC.%26atitle%3DTreadmilling%2520by%2520FtsZ%2520filaments%2520drives%2520peptidoglycan%2520synthesis%2520and%2520bacterial%2520cell%2520division%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D739%26epage%3D743%26doi%3D10.1126%2Fscience.aak9973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miguel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuillen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span> <span> </span><span class="NLM_article-title">GTPase activity–coupled treadmilling of the bacterial tubulin FtsZ organizes septal cell wall synthesis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">744</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1126/science.aak9995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscience.aak9995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28209899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yjur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=744-747&author=X.+Yangauthor=Z.+Lyuauthor=A.+Miguelauthor=R.+McQuillenauthor=K.+C.+Huangauthor=J.+Xiao&title=GTPase+activity%E2%80%93coupled+treadmilling+of+the+bacterial+tubulin+FtsZ+organizes+septal+cell+wall+synthesis&doi=10.1126%2Fscience.aak9995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">GTPase activity-coupled treadmilling of the bacterial tubulin FtsZ organizes septal cell wall synthesis</span></div><div class="casAuthors">Yang, Xinxing; Lyu, Zhixin; Miguel, Amanda; McQuillen, Ryan; Huang, Kerwyn Casey; Xiao, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6326</span>),
    <span class="NLM_cas:pages">744-747</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The bacterial tubulin FtsZ is the central component of the cell division machinery, coordinating an ensemble of proteins involved in septal cell wall synthesis to ensure successful constriction.  How cells achieve this coordination is unknown.  In Escherichia coli cells, FtsZ exhibits dynamic treadmilling predominantly detd. by its GTPase activity.  The treadmilling dynamics direct the processive movement of the septal cell wall synthesis machinery but do not limit the rate of septal synthesis.  In FtsZ mutants with severely reduced treadmilling, the spatial distribution of septal synthesis and the mol. compn. and ultrastructure of the septal cell wall were substantially altered.  Thus, FtsZ treadmilling provides a mechanism for achieving uniform septal cell wall synthesis to enable correct polar morphol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfemmDnK_v5bVg90H21EOLACvtfcHk0lidRUdOwB3miw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yjur4%253D&md5=d332376a876204c2adec4c7a7b070781</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.aak9995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aak9995%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLyu%26aufirst%3DZ.%26aulast%3DMiguel%26aufirst%3DA.%26aulast%3DMcQuillen%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DK.%2BC.%26aulast%3DXiao%26aufirst%3DJ.%26atitle%3DGTPase%2520activity%25E2%2580%2593coupled%2520treadmilling%2520of%2520the%2520bacterial%2520tubulin%2520FtsZ%2520organizes%2520septal%2520cell%2520wall%2520synthesis%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D744%26epage%3D747%26doi%3D10.1126%2Fscience.aak9995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichmann, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saraiva, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veiga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macario, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanNieuwenhze, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipe, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinho, M. G.</span></span> <span> </span><span class="NLM_article-title">Peptidoglycan synthesis drives an FtsZ-treadmilling-independent step of cytokinesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>554</i></span>,  <span class="NLM_fpage">528</span>– <span class="NLM_lpage">532</span>, <span class="refDoi"> DOI: 10.1038/nature25506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fnature25506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=29443967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisl2ls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=554&publication_year=2018&pages=528-532&author=J.+M.+Monteiroauthor=A.+R.+Pereiraauthor=N.+T.+Reichmannauthor=B.+M.+Saraivaauthor=P.+B.+Fernandesauthor=H.+Veigaauthor=A.+C.+Tavaresauthor=M.+Santosauthor=M.+T.+Ferreiraauthor=V.+Macarioauthor=M.+S.+VanNieuwenhzeauthor=S.+R.+Filipeauthor=M.+G.+Pinho&title=Peptidoglycan+synthesis+drives+an+FtsZ-treadmilling-independent+step+of+cytokinesis&doi=10.1038%2Fnature25506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidoglycan synthesis drives an FtsZ-treadmilling-independent step of cytokinesis</span></div><div class="casAuthors">Monteiro, Joao M.; Pereira, Ana R.; Reichmann, Nathalie T.; Saraiva, Bruno M.; Fernandes, Pedro B.; Veiga, Helena; Tavares, Andreia C.; Santos, Margarida; Ferreira, Maria T.; Macario, Vania; Van Nieuwenhze, Michael S.; Filipe, Sergio R.; Pinho, Mariana G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">554</span>
        (<span class="NLM_cas:issue">7693</span>),
    <span class="NLM_cas:pages">528-532</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Peptidoglycan is the main component of the bacterial wall and protects cells from the mech. stress that results from high intracellular turgor.  Peptidoglycan biosynthesis is very similar in all bacteria; bacterial shapes are therefore mainly detd. by the spatial and temporal regulation of peptidoglycan synthesis rather than by the chem. compn. of peptidoglycan.  The form of rod-shaped bacteria, such as Bacillus subtilis or Escherichia coli, is generated by the action of two peptidoglycan synthesis machineries that act at the septum and at the lateral wall in processes coordinated by the cytoskeletal proteins FtsZ and MreB, resp.  The tubulin homolog FtsZ is the first protein recruited to the division site, where it assembles in filaments-forming the Z ring-that undergo treadmilling and recruit later divisome proteins.  The rate of treadmilling in B. subtilis controls the rates of both peptidoglycan synthesis and cell division.  The actin homolog MreB forms discrete patches that move circumferentially around the cell in tracks perpendicular to the long axis of the cell, and organize the insertion of new cell wall during elongation.  Cocci such as Staphylococcus aureus possess only one type of peptidoglycan synthesis machinery, which is diverted from the cell periphery to the septum in prepn. for division.  The mol. cue that coordinates this transition has remained elusive.  Here we investigate the localization of S. aureus peptidoglycan biosynthesis proteins and show that the recruitment of the putative lipid II flippase MurJ to the septum, by the DivIB-DivIC-FtsL complex, drives peptidoglycan incorporation to the midcell.  MurJ recruitment corresponds to a turning point in cytokinesis, which is slow and dependent on FtsZ treadmilling before MurJ arrival but becomes faster and independent of FtsZ treadmilling after peptidoglycan synthesis activity is directed to the septum, where it provides addnl. force for cell envelope constriction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPOmsMFMKeZrVg90H21EOLACvtfcHk0lidRUdOwB3miw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisl2ls7s%253D&md5=0c30964816aac226e31fd0561b5140c2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature25506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25506%26sid%3Dliteratum%253Aachs%26aulast%3DMonteiro%26aufirst%3DJ.%2BM.%26aulast%3DPereira%26aufirst%3DA.%2BR.%26aulast%3DReichmann%26aufirst%3DN.%2BT.%26aulast%3DSaraiva%26aufirst%3DB.%2BM.%26aulast%3DFernandes%26aufirst%3DP.%2BB.%26aulast%3DVeiga%26aufirst%3DH.%26aulast%3DTavares%26aufirst%3DA.%2BC.%26aulast%3DSantos%26aufirst%3DM.%26aulast%3DFerreira%26aufirst%3DM.%2BT.%26aulast%3DMacario%26aufirst%3DV.%26aulast%3DVanNieuwenhze%26aufirst%3DM.%2BS.%26aulast%3DFilipe%26aufirst%3DS.%2BR.%26aulast%3DPinho%26aufirst%3DM.%2BG.%26atitle%3DPeptidoglycan%2520synthesis%2520drives%2520an%2520FtsZ-treadmilling-independent%2520step%2520of%2520cytokinesis%26jtitle%3DNature%26date%3D2018%26volume%3D554%26spage%3D528%26epage%3D532%26doi%3D10.1038%2Fnature25506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lock, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harry, E. J.</span></span> <span> </span><span class="NLM_article-title">Cell-division inhibitors: new insights for future antibiotics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">324</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1038/nrd2510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fnrd2510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18323848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVKmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=324-338&author=R.+L.+Lockauthor=E.+J.+Harry&title=Cell-division+inhibitors%3A+new+insights+for+future+antibiotics&doi=10.1038%2Fnrd2510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-division inhibitors: new insights for future antibiotics</span></div><div class="casAuthors">Lock, Rowena L.; Harry, Elizabeth J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">324-338</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The growing problem of antibiotic resistance has been exacerbated by the use of new drugs that are merely variants of older overused antibiotics.  While it is naive to expect to restrain the spread of resistance without controlling antibacterial usage, the desperate need for drugs with novel targets has been recognized by health organizations, industry and academia alike.  The wealth of knowledge available about the bacterial cell-division pathway has aided target-driven approaches to identify novel inhibitors.  Here, we discuss the therapeutic potential of inhibiting bacterial cell division, and review the progress made in this exciting new area of antibacterial discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplHXog8Fa6zLVg90H21EOLACvtfcHk0lhTYGiMziMz8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVKmsrw%253D&md5=da2514ca45300b30d46db98a3b4be004</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd2510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2510%26sid%3Dliteratum%253Aachs%26aulast%3DLock%26aufirst%3DR.%2BL.%26aulast%3DHarry%26aufirst%3DE.%2BJ.%26atitle%3DCell-division%2520inhibitors%253A%2520new%2520insights%2520for%2520future%2520antibiotics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D324%26epage%3D338%26doi%3D10.1038%2Fnrd2510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the assembly of bacterial cell division protein FtsZ with small molecules</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1021/cb2003626</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb2003626" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSit7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=269-277&author=C.+Schaffner-Barberoauthor=M.+Martin-Fontechaauthor=P.+Chaconauthor=J.+M.+Andreu&title=Targeting+the+assembly+of+bacterial+cell+division+protein+FtsZ+with+small+molecules&doi=10.1021%2Fcb2003626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the assembly of bacterial cell division protein FtsZ with small molecules</span></div><div class="casAuthors">Schaffner-Barbero, Claudia; Martin-Fontecha, Mar; Chacon, Pablo; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-277</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  FtsZ is the key protein of bacterial cell division and an emergent target for new antibiotics.  It is a filament-forming GTPase and a structural homolog of eukaryotic tubulin.  A no. of FtsZ-interacting compds. have been reported, some of which have powerful antibacterial activity.  Here, the authors review recent advances and new approaches in modulating FtsZ assembly with small mols.  This includes analyzing their chem. features, binding sites, mechanisms of action, the methods employed, and computational insights, aimed at a better understanding of their mol. recognition by FtsZ and at rational antibiotic design.  A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAWWLj_dUcQ7Vg90H21EOLACvtfcHk0liTn486wYBhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSit7vP&md5=81413a2f6c1ed3283644c85cc7ac9440</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb2003626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb2003626%26sid%3Dliteratum%253Aachs%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DTargeting%2520the%2520assembly%2520of%2520bacterial%2520cell%2520division%2520protein%2520FtsZ%2520with%2520small%2520molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D269%26epage%3D277%26doi%3D10.1021%2Fcb2003626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">den
Blaauwen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monasterio, O.</span></span> <span> </span><span class="NLM_article-title">Bacterial cell division proteins as antibiotic targets</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2014.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.bioorg.2014.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=24755375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCmtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=27-38&author=T.+den%0ABlaauwenauthor=J.+M.+Andreuauthor=O.+Monasterio&title=Bacterial+cell+division+proteins+as+antibiotic+targets&doi=10.1016%2Fj.bioorg.2014.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial cell division proteins as antibiotic targets</span></div><div class="casAuthors">den Blaauwen, Tanneke; Andreu, Jose M.; Monasterio, Octavio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteins involved in bacterial cell division often do not have a counterpart in eukaryotic cells and they are essential for the survival of the bacteria.  The genetic accessibility of many bacterial species in combination with the Green Fluorescence Protein revolution to study localization of proteins and the availability of crystal structures has increased our knowledge on bacterial cell division considerably in this century.  Consequently, bacterial cell division proteins are more and more recognized as potential new antibiotic targets.  An international effort to find small mols. that inhibit the cell division initiating protein FtsZ has yielded many compds. of which some are promising as leads for preclin. use.  The essential transglycosylase activity of peptidoglycan synthases has recently become accessible to inhibitor screening.  Enzymic assays for and structural information on essential integral membrane proteins such as MraY and FtsW involved in lipid II (the peptidoglycan building block precursor) biosynthesis have put these proteins on the list of potential new targets.  This review summarises and discusses the results and approaches to the development of lead compds. that inhibit bacterial cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqngOkmUR6AP7Vg90H21EOLACvtfcHk0liTn486wYBhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCmtr4%253D&md5=44a078b9b351b7b2c4d81c26de0da512</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2014.03.007%26sid%3Dliteratum%253Aachs%26aulast%3Dden%2BBlaauwen%26aufirst%3DT.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DMonasterio%26aufirst%3DO.%26atitle%3DBacterial%2520cell%2520division%2520proteins%2520as%2520antibiotic%2520targets%26jtitle%3DBioorg.%2520Chem.%26date%3D2014%26volume%3D55%26spage%3D27%26epage%3D38%26doi%3D10.1016%2Fj.bioorg.2014.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kusuma, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ung, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottomley, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harry, E. J.</span></span> <span> </span><span class="NLM_article-title">FtsZ as an antibacterial target: status and guidelines for progressing this avenue</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1279</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.9b00055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.9b00055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=1279-1294&author=K.+D.+Kusumaauthor=M.+Payneauthor=A.+T.+Ungauthor=A.+L.+Bottomleyauthor=E.+J.+Harry&title=FtsZ+as+an+antibacterial+target%3A+status+and+guidelines+for+progressing+this+avenue&doi=10.1021%2Facsinfecdis.9b00055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FtsZ as an Antibacterial Target: Status and Guidelines for Progressing This Avenue</span></div><div class="casAuthors">Kusuma, Kennardy D.; Payne, Matthew; Ung, Alison T.; Bottomley, Amy L.; Harry, Elizabeth J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1279-1294</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The disturbing increase in the no. of bacterial pathogens that are resistant to multiple, or sometimes all, current antibiotics highlights the desperate need to pursue the discovery and development of novel classes of antibacterials.  The wealth of knowledge available about the bacterial cell division machinery has aided target-driven approaches to identify new inhibitor compds.  The main division target being pursued is the highly conserved and essential protein, FtsZ.  Despite very active research on FtsZ inhibitors for several years, this protein is not yet targeted by any com. antibiotic.  Here we discuss the suitability of FtsZ as an antibacterial target for drug development, and review progress achieved in this area.  We use hindsight to highlight the gaps that have slowed progress in FtsZ inhibitor development, and to suggest guidelines for concluding that FtsZ is actually the target of these mols., a key missing link in several studies.  In moving forward, a multidisciplinary and a communicative and collaborative process, with sharing of research expertise, is crit. if we are to succeed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NungbAnpqrVg90H21EOLACvtfcHk0liTn486wYBhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSju7zJ&md5=5a867b4471c875047397d393100f287f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.9b00055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.9b00055%26sid%3Dliteratum%253Aachs%26aulast%3DKusuma%26aufirst%3DK.%2BD.%26aulast%3DPayne%26aufirst%3DM.%26aulast%3DUng%26aufirst%3DA.%2BT.%26aulast%3DBottomley%26aufirst%3DA.%2BL.%26aulast%3DHarry%26aufirst%3DE.%2BJ.%26atitle%3DFtsZ%2520as%2520an%2520antibacterial%2520target%253A%2520status%2520and%2520guidelines%2520for%2520progressing%2520this%2520avenue%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D1279%26epage%3D1294%26doi%3D10.1021%2Facsinfecdis.9b00055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haydon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ure, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barynin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedelnikova, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heal, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiwale, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taneja, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czaplewski, L. G.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of FtsZ with potent and selective anti-staphylococcal activity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">1673</span>– <span class="NLM_lpage">1675</span>, <span class="refDoi"> DOI: 10.1126/science.1159961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscience.1159961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18801997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=1673-1675&author=D.+J.+Haydonauthor=N.+R.+Stokesauthor=R.+Ureauthor=G.+Galbraithauthor=J.+M.+Bennettauthor=D.+R.+Brownauthor=P.+J.+Bakerauthor=V.+V.+Baryninauthor=D.+W.+Riceauthor=S.+E.+Sedelnikovaauthor=J.+R.+Healauthor=J.+M.+Sheridanauthor=S.+T.+Aiwaleauthor=P.+K.+Chauhanauthor=A.+Srivastavaauthor=A.+Tanejaauthor=I.+Collinsauthor=J.+Erringtonauthor=L.+G.+Czaplewski&title=An+inhibitor+of+FtsZ+with+potent+and+selective+anti-staphylococcal+activity&doi=10.1126%2Fscience.1159961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Inhibitor of FtsZ with Potent and Selective Anti-Staphylococcal Activity</span></div><div class="casAuthors">Haydon, David J.; Stokes, Neil R.; Ure, Rebecca; Galbraith, Greta; Bennett, James M.; Brown, David R.; Baker, Patrick J.; Barynin, Vladimir V.; Rice, David W.; Sedelnikova, Sveta E.; Heal, Jonathan R.; Sheridan, Joseph M.; Aiwale, Sachin T.; Chauhan, Pramod K.; Srivastava, Anil; Taneja, Amit; Collins, Ian; Errington, Jeff; Czaplewski, Lloyd G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5896</span>),
    <span class="NLM_cas:pages">1673-1675</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">FtsZ is an essential bacterial guanosine triphosphatase and homolog of mammalian β-tubulin that polymerizes and assembles into a ring to initiate cell division.  We have created a class of small synthetic antibacterials, exemplified by PC190723, which inhibits FtsZ and prevents cell division.  PC190723 has potent and selective in vitro bactericidal activity against staphylococci, including methicillin- and multi-drug-resistant Staphylococcus aureus.  The putative inhibitor-binding site of PC190723 was mapped to a region of FtsZ that is analogous to the Taxol-binding site of tubulin.  PC190723 was efficacious in an in vivo model of infection, curing mice infected with a LD of S. aureus.  The data validate FtsZ as a target for antibacterial intervention and identify PC190723 as suitable for optimization into a new anti-staphylococcal therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmbbLDvOUp8LVg90H21EOLACvtfcHk0ljK5RyP4p5n5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtrrJ&md5=6573e2e19954f44e7632c99c5d8e396c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.1159961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1159961%26sid%3Dliteratum%253Aachs%26aulast%3DHaydon%26aufirst%3DD.%2BJ.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DUre%26aufirst%3DR.%26aulast%3DGalbraith%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBrown%26aufirst%3DD.%2BR.%26aulast%3DBaker%26aufirst%3DP.%2BJ.%26aulast%3DBarynin%26aufirst%3DV.%2BV.%26aulast%3DRice%26aufirst%3DD.%2BW.%26aulast%3DSedelnikova%26aufirst%3DS.%2BE.%26aulast%3DHeal%26aufirst%3DJ.%2BR.%26aulast%3DSheridan%26aufirst%3DJ.%2BM.%26aulast%3DAiwale%26aufirst%3DS.%2BT.%26aulast%3DChauhan%26aufirst%3DP.%2BK.%26aulast%3DSrivastava%26aufirst%3DA.%26aulast%3DTaneja%26aufirst%3DA.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DErrington%26aufirst%3DJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26atitle%3DAn%2520inhibitor%2520of%2520FtsZ%2520with%2520potent%2520and%2520selective%2520anti-staphylococcal%2520activity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D1673%26epage%3D1675%26doi%3D10.1126%2Fscience.1159961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therien, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeau-Jacob, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benton-Perdomo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petcu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigneault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claveau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaillancourt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skorey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meredith, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sillaots, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang-Jarantow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramtohul, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baryshnikova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinho, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Restoring methicillin-resistant Staphylococcus aureus susceptibility to β-lactam antibiotics</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">126ra135</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.3003592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1126%2Fscitranslmed.3003592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&author=C.+M.+Tanauthor=A.+G.+Therienauthor=J.+Luauthor=S.+H.+Leeauthor=A.+Caronauthor=C.+J.+Gillauthor=C.+Lebeau-Jacobauthor=L.+Benton-Perdomoauthor=J.+M.+Monteiroauthor=P.+M.+Pereiraauthor=N.+L.+Elsenauthor=J.+Wuauthor=K.+Deschampsauthor=M.+Petcuauthor=S.+Wongauthor=E.+Daigneaultauthor=S.+Kramerauthor=L.+Liangauthor=E.+Maxwellauthor=D.+Claveauauthor=J.+Vaillancourtauthor=K.+Skoreyauthor=J.+Tamauthor=H.+Wangauthor=T.+C.+Meredithauthor=S.+Sillaotsauthor=L.+Wang-Jarantowauthor=Y.+Ramtohulauthor=E.+Langloisauthor=F.+Landryauthor=J.+C.+Reidauthor=G.+Parthasarathyauthor=S.+Sharmaauthor=A.+Baryshnikovaauthor=K.+J.+Lumbauthor=M.+G.+Pinhoauthor=S.+M.+Soissonauthor=T.+Roemer&title=Restoring+methicillin-resistant+Staphylococcus+aureus+susceptibility+to+%CE%B2-lactam+antibiotics&doi=10.1126%2Fscitranslmed.3003592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3003592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3003592%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DC.%2BM.%26aulast%3DTherien%26aufirst%3DA.%2BG.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DCaron%26aufirst%3DA.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DLebeau-Jacob%26aufirst%3DC.%26aulast%3DBenton-Perdomo%26aufirst%3DL.%26aulast%3DMonteiro%26aufirst%3DJ.%2BM.%26aulast%3DPereira%26aufirst%3DP.%2BM.%26aulast%3DElsen%26aufirst%3DN.%2BL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDeschamps%26aufirst%3DK.%26aulast%3DPetcu%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DDaigneault%26aufirst%3DE.%26aulast%3DKramer%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DMaxwell%26aufirst%3DE.%26aulast%3DClaveau%26aufirst%3DD.%26aulast%3DVaillancourt%26aufirst%3DJ.%26aulast%3DSkorey%26aufirst%3DK.%26aulast%3DTam%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DMeredith%26aufirst%3DT.%2BC.%26aulast%3DSillaots%26aufirst%3DS.%26aulast%3DWang-Jarantow%26aufirst%3DL.%26aulast%3DRamtohul%26aufirst%3DY.%26aulast%3DLanglois%26aufirst%3DE.%26aulast%3DLandry%26aufirst%3DF.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DBaryshnikova%26aufirst%3DA.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DPinho%26aufirst%3DM.%2BG.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DRestoring%2520methicillin-resistant%2520Staphylococcus%2520aureus%2520susceptibility%2520to%2520%25CE%25B2-lactam%2520antibiotics%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2012%26volume%3D4%26doi%3D10.1126%2Fscitranslmed.3003592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casiraghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suigo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valoti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straniero, V.</span></span> <span> </span><span class="NLM_article-title">Targeting bacterial cell division: a binding-site centered approach to the most promising inhibitors of the essential protein FtsZ</span>. <i>Antibiotics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">69</span> <span class="refDoi"> DOI: 10.3390/antibiotics9020069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3390%2Fantibiotics9020069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1OitLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=A.+Casiraghiauthor=L.+Suigoauthor=E.+Valotiauthor=V.+Straniero&title=Targeting+bacterial+cell+division%3A+a+binding-site+centered+approach+to+the+most+promising+inhibitors+of+the+essential+protein+FtsZ&doi=10.3390%2Fantibiotics9020069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bacterial cell division: a binding site-centered approach to the most promising inhibitors of the essential protein FtsZ</span></div><div class="casAuthors">Casiraghi, Andrea; Suigo, Lorenzo; Valoti, Ermanno; Straniero, Valentina</div><div class="citationInfo"><span class="NLM_cas:title">Antibiotics (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">69</span>CODEN:
                <span class="NLM_cas:coden">ABSNC4</span>;
        ISSN:<span class="NLM_cas:issn">2079-6382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Binary fission is the most common mode of bacterial cell division and is mediated by a multiprotein complex denominated the divisome.  The constriction of the Z-ring splits the mother bacterial cell into two daughter cells of the same size.  The Z-ring is formed by the polymn. of FtsZ, a bacterial protein homolog of eukaryotic tubulin, and it represents the first step of bacterial cytokinesis.  The high grade of conservation of FtsZ in most prokaryotic organisms and its relevance in orchestrating the whole division system make this protein a fascinating target in antibiotic research.  Indeed, FtsZ inhibition results in the complete blockage of the division system and, consequently, in a bacteriostatic or a bactericidal effect.  Since many papers and reviews already discussed the physiol. of FtsZ and its auxiliary proteins, as well as the mol. mechanisms in which they are involved, here, we focus on the discussion of the most compelling FtsZ inhibitors, classified by their main protein binding sites and following a medicinal chem. approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmzHUPYa5suLVg90H21EOLACvtfcHk0ljK5RyP4p5n5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1OitLzP&md5=6d5eb79696b35059f1e6266902c54f4b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Fantibiotics9020069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantibiotics9020069%26sid%3Dliteratum%253Aachs%26aulast%3DCasiraghi%26aufirst%3DA.%26aulast%3DSuigo%26aufirst%3DL.%26aulast%3DValoti%26aufirst%3DE.%26aulast%3DStraniero%26aufirst%3DV.%26atitle%3DTargeting%2520bacterial%2520cell%2520division%253A%2520a%2520binding-site%2520centered%2520approach%2520to%2520the%2520most%2520promising%2520inhibitors%2520of%2520the%2520essential%2520protein%2520FtsZ%26jtitle%3DAntibiotics%26date%3D2020%26volume%3D9%26doi%3D10.3390%2Fantibiotics9020069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoie, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, D. S.</span></span> <span> </span><span class="NLM_article-title">Combining the FtsZ-targeting prodrug TXA709 and the cephalosporin cefdinir confers synergy and reduces the frequency of resistance in methicillin-resistant Staphylococcus aureus</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4290</span>– <span class="NLM_lpage">4296</span>, <span class="refDoi"> DOI: 10.1128/AAC.00613-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1128%2FAAC.00613-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=27161635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=4290-4296&author=M.+Kaulauthor=L.+Markauthor=A.+K.+Parhiauthor=E.+J.+LaVoieauthor=D.+S.+Pilch&title=Combining+the+FtsZ-targeting+prodrug+TXA709+and+the+cephalosporin+cefdinir+confers+synergy+and+reduces+the+frequency+of+resistance+in+methicillin-resistant+Staphylococcus+aureus&doi=10.1128%2FAAC.00613-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Combining the FtsZ-targeting prodrug TXA709 and the cephalosporin cefdinir confers synergy and reduces the frequency of resistance in methicillin-resistant Staphylococcus aureus</span></div><div class="casAuthors">Kaul, Malvika; Mark, Lilly; Parhi, Ajit K.; LaVoie, Edmond J.; Pilch, Daniel S.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4296</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Combination therapy of bacterial infections with synergistic drug partners offers distinct advantages over monotherapy.  Among these advantages are (1) redn. of the drug dose required for efficacy, (2) reduced potential for drug-induced toxicity, and (3) reduced potential for the emergence of resistance.  Here, we describe the synergistic actions of the 3rd-generation oral cephalosporin cefdinir and TXA709, a new, FtsZ-targeting prodrug that we have developed with improved pharmacokinetics and enhanced in vivo efficacy against methicillin-resistant Staphylococcus aureus (MRSA) relative to earlier agents.  The active product of TXA709 (TXA707) acts synergistically with cefdinir in vitro against clin. isolates of MRSA, vancomycin-intermediate S. aureus (VISA), vancomycin-resistant S. aureus (VRSA), and linezolid-resistant S. aureus (LRSA).  In addn., relative to TXA707 alone, the combination of TXA707 and cefdinir significantly reduces or eliminates the detectable emergence of resistance.  We also demonstrate synergy in vivo with oral administration of the prodrug TXA709 and cefdinir in mouse models of both systemic and tissue (thigh) infections with MRSA.  This synergy reduces the dose of TXA709 required for efficacy 3-fold.  Viewed as a whole, our results highlight the potential of TXA709 and cefdinir as a promising combination for the treatment of drug-resistant staphylococcal infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzb-r4hn823bVg90H21EOLACvtfcHk0lgDhGKDjlmt9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyksL4%253D&md5=a124444e0323bc13e8f5e664a0bdd130</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FAAC.00613-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00613-16%26sid%3Dliteratum%253Aachs%26aulast%3DKaul%26aufirst%3DM.%26aulast%3DMark%26aufirst%3DL.%26aulast%3DParhi%26aufirst%3DA.%2BK.%26aulast%3DLaVoie%26aufirst%3DE.%2BJ.%26aulast%3DPilch%26aufirst%3DD.%2BS.%26atitle%3DCombining%2520the%2520FtsZ-targeting%2520prodrug%2520TXA709%2520and%2520the%2520cephalosporin%2520cefdinir%2520confers%2520synergy%2520and%2520reduces%2520the%2520frequency%2520of%2520resistance%2520in%2520methicillin-resistant%2520Staphylococcus%2520aureus%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D4290%26epage%3D4296%26doi%3D10.1128%2FAAC.00613-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span>Taxis
Pharmaceuticals Pipeline, <span class="NLM_year">2020</span>. Can be found under: <a href="https://www.taxispharma.com/research-development/our-pipeline/" class="extLink">https://www.taxispharma.com/research-development/our-pipeline/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Taxis%0APharmaceuticals+Pipeline%2C+2020.+Can+be+found+under%3A+https%3A%2F%2Fwww.taxispharma.com%2Fresearch-development%2Four-pipeline%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llorca, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boskovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Benito, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valpuesta, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Energetics and geometry of FtsZ polymers: nucleated self-assembly of single protofilaments</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1796</span>– <span class="NLM_lpage">1806</span>, <span class="refDoi"> DOI: 10.1529/biophysj.107.115493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1529%2Fbiophysj.107.115493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18024502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXit1yqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2008&pages=1796-1806&author=S.+Huecasauthor=O.+Llorcaauthor=J.+Boskovicauthor=J.+Martin-Benitoauthor=J.+M.+Valpuestaauthor=J.+M.+Andreu&title=Energetics+and+geometry+of+FtsZ+polymers%3A+nucleated+self-assembly+of+single+protofilaments&doi=10.1529%2Fbiophysj.107.115493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Energetics and geometry of FtsZ polymers: nucleated self-assembly of single protofilaments</span></div><div class="casAuthors">Huecas, Sonia; Llorca, Oscar; Boskovic, Jasminka; Martin-Benito, Jaime; Valpuesta, Jose Maria; Andreu, Jose Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1796-1806</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">Essential cell division protein FtsZ is an assembling GTPase which directs the cytokinetic ring formation in dividing bacterial cells.  FtsZ shares the structural fold of eukaryotic tubulin and assembles forming tubulin-like protofilaments, but does not form microtubules.  Two puzzling problems in FtsZ assembly are the nature of protofilament assocn. and a possible mechanism for nucleated self-assembly of single-stranded protofilaments above a crit. FtsZ concn.  We assembled two-dimensional arrays of FtsZ on carbon supports, studied linear polymers of FtsZ with cryo-electron microscopy of vitrified unsupported solns., and formulated possible polymn. models.  Nucleated self-assembly of FtsZ from Escherichia coli with GTP and magnesium produces flexible filaments 4-6 nm-wide, only compatible with a single protofilament.  This agrees with previous scanning TEM results and is supported by recent cryo-electron tomog. studies of two bacterial cells.  Observations of double-stranded FtsZ filaments in neg. stain may come from protofilament accretion on the carbon support.  Preferential protofilament cyclization does not apply to FtsZ assembly.  The apparently cooperative polymn. of a single protofilament with identical intermonomer contacts is explained by the switching of one inactive monomer into the active structure preceding assocn. of the next, creating a dimer nucleus.  FtsZ behaves as a cooperative linear assembly machine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZhX8LMlXWbVg90H21EOLACvtfcHk0lgDhGKDjlmt9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXit1yqurw%253D&md5=4d778a930ba385b1843d64231d54cf99</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.107.115493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.107.115493%26sid%3Dliteratum%253Aachs%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DLlorca%26aufirst%3DO.%26aulast%3DBoskovic%26aufirst%3DJ.%26aulast%3DMartin-Benito%26aufirst%3DJ.%26aulast%3DValpuesta%26aufirst%3DJ.%2BM.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DEnergetics%2520and%2520geometry%2520of%2520FtsZ%2520polymers%253A%2520nucleated%2520self-assembly%2520of%2520single%2520protofilaments%26jtitle%3DBiophys.%2520J.%26date%3D2008%26volume%3D94%26spage%3D1796%26epage%3D1806%26doi%3D10.1529%2Fbiophysj.107.115493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dajkovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutkenhaus, J.</span></span> <span> </span><span class="NLM_article-title">Investigation of regulation of FtsZ assembly by SulA and development of a model for FtsZ polymerization</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">2513</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1128/JB.01612-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1128%2FJB.01612-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18245292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVylsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2008&pages=2513-2526&author=A.+Dajkovicauthor=A.+Mukherjeeauthor=J.+Lutkenhaus&title=Investigation+of+regulation+of+FtsZ+assembly+by+SulA+and+development+of+a+model+for+FtsZ+polymerization&doi=10.1128%2FJB.01612-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of regulation of FtsZ assembly by SulA and development of a model for FtsZ polymerization</span></div><div class="casAuthors">Dajkovic, Alex; Mukherjee, Amit; Lutkenhaus, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2513-2526</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In Escherichia coli FtsZ organizes into a cytoskeletal ring structure, the Z ring, which effects cell division.  FtsZ is a GTPase, but the free energy of GTP hydrolysis does not appear to be used for generation of the constriction force, leaving open the question of the function of the GTPase activity of FtsZ.  Here we study the mechanism by which SulA, an inhibitor of FtsZ induced during the SOS response, inhibits FtsZ function.  We studied the effects of SulA on the in vitro activities of FtsZ, on Z rings in vivo, and on a kinetic model for FtsZ polymn. in silico.  We found that the binding of SulA to FtsZ is necessary but not sufficient for inhibition of polymn., since the assembly of FtsZ polymers in the absence of the GTPase activity was not inhibited by SulA.  We developed a new model for FtsZ polymn. that accounts for the cooperativity of FtsZ and could account for cooperativity obsd. in other linear polymers.  When SulA was included in the kinetic scheme, simulations revealed that SulA with strong affinity for FtsZ delayed, but did not prevent, the assembly of polymers when they were not hydrolyzing GTP.  Furthermore, the simulations indicated that SulA controls the assembly of FtsZ by binding to a polymn.-competent form of the FtsZ mol. and preventing it from participating in assembly.  In vivo stoichiometry of the disruption of Z rings by SulA suggests that FtsZ may undergo two cooperative transitions in forming the Z ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQXpzN21oQVLVg90H21EOLACvtfcHk0lhDnXOZ_Je4JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVylsr4%253D&md5=1c2f4f19f1cc1a482226ebc2e47fccd7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FJB.01612-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.01612-07%26sid%3Dliteratum%253Aachs%26aulast%3DDajkovic%26aufirst%3DA.%26aulast%3DMukherjee%26aufirst%3DA.%26aulast%3DLutkenhaus%26aufirst%3DJ.%26atitle%3DInvestigation%2520of%2520regulation%2520of%2520FtsZ%2520assembly%2520by%2520SulA%2520and%2520development%2520of%2520a%2520model%2520for%2520FtsZ%2520polymerization%26jtitle%3DJ.%2520Bacteriol.%26date%3D2008%26volume%3D190%26spage%3D2513%26epage%3D2526%26doi%3D10.1128%2FJB.01612-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miraldi, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romberg, L.</span></span> <span> </span><span class="NLM_article-title">Allosteric models for cooperative polymerization of linear polymers</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">2470</span>– <span class="NLM_lpage">2486</span>, <span class="refDoi"> DOI: 10.1529/biophysj.107.126219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1529%2Fbiophysj.107.126219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18502809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGqsr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2008&pages=2470-2486&author=E.+R.+Miraldiauthor=P.+J.+Thomasauthor=L.+Romberg&title=Allosteric+models+for+cooperative+polymerization+of+linear+polymers&doi=10.1529%2Fbiophysj.107.126219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric models for cooperative polymerization of linear polymers</span></div><div class="casAuthors">Miraldi, Emily R.; Thomas, Peter J.; Romberg, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2470-2486</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">In the cytoskeleton, unfavorable nucleation steps allow cells to regulate where, when, and how many polymers assemble.  Nucleated polymn. is traditionally explained by a model in which multistranded polymers assemble cooperatively, whereas linear, single-stranded polymers do not.  Recent data on the assembly of FtsZ, the bacterial homolog of tubulin, do not fit either category.  FtsZ can polymerize into single-stranded protofilaments that are stable in the absence of lateral interactions, but that assemble cooperatively.  We developed a model for cooperative polymn. that does not require polymers to be multistranded.  Instead, a conformational change allows subunits in oligomers to assoc. with high affinity, whereas a lower-affinity conformation is favored in monomers.  We derive equations for calcg. polymer concns., subunit conformations, and the apparent affinity of subunits for polymer ends.  Certain combinations of equil. consts. produce the sharp crit. concns. characteristic of cooperative polymn.  In these cases, the low-affinity conformation predominates in monomers, whereas virtually all polymers are composed of high-affinity subunits.  Our model predicts that the three routes to forming HH dimers all involve unstable intermediates, limiting nucleation.  The math. framework developed here can represent allosteric assembly systems with a variety of biochem. interpretations, some of which can show cooperativity, and others of which cannot.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI1xJvmvFr5rVg90H21EOLACvtfcHk0lhDnXOZ_Je4JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGqsr3O&md5=04dae4360e5767ded052b9dc081e59a6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.107.126219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.107.126219%26sid%3Dliteratum%253Aachs%26aulast%3DMiraldi%26aufirst%3DE.%2BR.%26aulast%3DThomas%26aufirst%3DP.%2BJ.%26aulast%3DRomberg%26aufirst%3DL.%26atitle%3DAllosteric%2520models%2520for%2520cooperative%2520polymerization%2520of%2520linear%2520polymers%26jtitle%3DBiophys.%2520J.%26date%3D2008%26volume%3D95%26spage%3D2470%26epage%3D2486%26doi%3D10.1529%2Fbiophysj.107.126219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, I.</span></span> <span> </span><span class="NLM_article-title">Structural change in FtsZ induced by intermolecular interactions between bound GTP and the T7 loop</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">3501</span>– <span class="NLM_lpage">3509</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.514901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M113.514901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=24347164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2ltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=3501-3509&author=T.+Matsuiauthor=X.+Hanauthor=R.+J.+Yuauthor=M.+Yaoauthor=I.+Tanaka&title=Structural+change+in+FtsZ+induced+by+intermolecular+interactions+between+bound+GTP+and+the+T7+loop&doi=10.1074%2Fjbc.M113.514901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural change in FtsZ induced by intermolecular interactions between bound GTP and the T7 loop</span></div><div class="casAuthors">Matsui, Takashi; Han, Xuerong; Yu, Jian; Yao, Min; Tanaka, Isao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3501-3509</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">FtsZ is a prokaryotic homolog of tubulin and is a key mol. in bacterial cell division.  FtsZ with bound GTP polymerizes into tubulin-like protofilaments.  Upon polymn., the T7 loop of one subunit is inserted into the nucleotide-binding pocket of the 2nd subunit, which results in GTP hydrolysis.  Thus, the T7 loop is important for both polymn. and hydrolysis in the tubulin/FtsZ family.  Although x-ray crystallog. revealed both straight and curved conformations of tubulin, only a curved structure was known for FtsZ.  Recently, however, FtsZ from Staphylococcus aureus has been shown to have a very different conformation from the canonical FtsZ structure.  The present study was performed to investigate the structure of FtsZ from Staphylococcus aureus by mutagenesis expts.; the effects of amino acid changes in the T7 loop on the structure as well as on GTPase activity were studied.  These analyses indicated that FtsZ changes its conformation suitable for polymn. and GTP hydrolysis by movement between N- and C-subdomains via intermol. interactions between bound nucleotide and residues in the T7 loop.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWwvoI8Jo8srVg90H21EOLACvtfcHk0lhDnXOZ_Je4JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2ltro%253D&md5=e91683c358b4cada51a79642586153de</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.514901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.514901%26sid%3Dliteratum%253Aachs%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DR.%2BJ.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DI.%26atitle%3DStructural%2520change%2520in%2520FtsZ%2520induced%2520by%2520intermolecular%2520interactions%2520between%2520bound%2520GTP%2520and%2520the%2520T7%2520loop%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D3501%26epage%3D3509%26doi%3D10.1074%2Fjbc.M113.514901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez-Ramirez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llorca, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Galiano, A. J.</span></span> <span> </span><span class="NLM_article-title">The antibacterial cell division inhibitor PC190723 is an FtsZ polymer-stabilizing agent that induces filament assembly and condensation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">14239</span>– <span class="NLM_lpage">14246</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.094722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M109.094722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20212044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVOiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=14239-14246&author=J.+M.+Andreuauthor=C.+Schaffner-Barberoauthor=S.+Huecasauthor=D.+Alonsoauthor=M.+L.+Lopez-Rodriguezauthor=L.+B.+Ruiz-Avilaauthor=R.+Nu%C3%B1ez-Ramirezauthor=O.+Llorcaauthor=A.+J.+Martin-Galiano&title=The+antibacterial+cell+division+inhibitor+PC190723+is+an+FtsZ+polymer-stabilizing+agent+that+induces+filament+assembly+and+condensation&doi=10.1074%2Fjbc.M109.094722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Antibacterial Cell Division Inhibitor PC190723 Is an FtsZ Polymer-stabilizing Agent That Induces Filament Assembly and Condensation</span></div><div class="casAuthors">Andreu, Jose M.; Schaffner-Barbero, Claudia; Huecas, Sonia; Alonso, Dulce; Lopez-Rodriguez, Maria L.; Ruiz-Avila, Laura B.; Nunez-Ramirez, Rafael; Llorca, Oscar; Martin-Galiano, Antonio J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">14239-14246</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cell division protein FtsZ can form single-stranded filaments with a cooperative behavior by self-switching assembly.  Subsequent condensation and bending of FtsZ filaments are important for the formation and constriction of the cytokinetic ring.  PC190723 is an effective bactericidal cell division inhibitor that targets FtsZ in the pathogen Staphylococcus aureus and Bacillus subtilis and does not affect Escherichia coli cells, which apparently binds to a zone equiv. to the binding site of the antitumor drug taxol in tubulin.  We have found that the benzamide deriv. PC190723 is an FtsZ polymer-stabilizing agent.  PC190723 induced nucleated assembly of Bs-FtsZ into single-stranded coiled protofilaments and polymorphic condensates, including bundles, coils, and toroids, whose formation could be modulated with different soln. conditions.  Under conditions for reversible assembly of Bs-FtsZ, PC190723 binding reduced the GTPase activity and induced the formation of straight bundles and ribbons, which was also obsd. with Sa-FtsZ but not with nonsusceptible Ec-FtsZ.  The fragment 2,6-difluoro-3-methoxybenzamide also induced Bs-FtsZ bundling.  We propose that polymer stabilization by PC190723 suppresses in vivo FtsZ polymer dynamics and bacterial division.  The biochem. action of PC190723 on FtsZ parallels that of the microtubule-stabilizing agent taxol on the eukaryotic structural homolog tubulin.  Both taxol and PC190723 stabilize polymers against disassembly by preferential binding to each assembled protein.  It is yet to be investigated whether both ligands target structurally related assembly switches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdWKjWSpN8rVg90H21EOLACvtfcHk0lhDnXOZ_Je4JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVOiurs%253D&md5=d6fc5ba584a2d89c44dcf6809a4abf90</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.094722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.094722%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAlonso%26aufirst%3DD.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DNu%25C3%25B1ez-Ramirez%26aufirst%3DR.%26aulast%3DLlorca%26aufirst%3DO.%26aulast%3DMartin-Galiano%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520antibacterial%2520cell%2520division%2520inhibitor%2520PC190723%2520is%2520an%2520FtsZ%2520polymer-stabilizing%2520agent%2520that%2520induces%2520filament%2520assembly%2520and%2520condensation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D14239%26epage%3D14246%26doi%3D10.1074%2Fjbc.M109.094722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elsen, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span> <span> </span><span class="NLM_article-title">Mechanism of action of the cell-division inhibitor PC190723: modulation of FtsZ assembly cooperativity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">12342</span>– <span class="NLM_lpage">12345</span>, <span class="refDoi"> DOI: 10.1021/ja303564a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303564a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVejtbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=12342-12345&author=N.+L.+Elsenauthor=J.+Luauthor=G.+Parthasarathyauthor=J.+C.+Reidauthor=S.+Sharmaauthor=S.+M.+Soissonauthor=K.+J.+Lumb&title=Mechanism+of+action+of+the+cell-division+inhibitor+PC190723%3A+modulation+of+FtsZ+assembly+cooperativity&doi=10.1021%2Fja303564a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Action of the Cell-Division Inhibitor PC190723: Modulation of FtsZ Assembly Cooperativity</span></div><div class="casAuthors">Elsen, Nathaniel L.; Lu, Jun; Parthasarathy, Gopal; Reid, John C.; Sharma, Sujata; Soisson, Stephen M.; Lumb, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">12342-12345</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cooperative assembly of FtsZ, the prokaryotic homolog of tubulin, plays an essential role in cell division.  FtsZ is a potential drug target, as illustrated by the small-mol. cell-cycle inhibitor and antibacterial agent PC190723 that targets FtsZ.  We demonstrate that PC190723 neg. modulates Staphylococcus aureus FtsZ polymn. cooperativity as reflected in polymn. at lower concns. without a defined crit. concn.  The crystal structure of the S. aureus FtsZ-PC190723 complex shows a domain movement that would stabilize the FtsZ protofilament over the monomeric state, with the conformational change mediated from the GTP-binding site to the C-terminal domain via helix 7.  Together, the results reveal the mol. mechanism of FtsZ modulation by PC190723 and a conformational switch to the high-affinity state that enables polymer assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWzgrLJ8OPQLVg90H21EOLACvtfcHk0ljCFeKxOIe_Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVejtbzJ&md5=052b9be970a494eb3524448ab967cc6a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fja303564a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303564a%26sid%3Dliteratum%253Aachs%26aulast%3DElsen%26aufirst%3DN.%2BL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26atitle%3DMechanism%2520of%2520action%2520of%2520the%2520cell-division%2520inhibitor%2520PC190723%253A%2520modulation%2520of%2520FtsZ%2520assembly%2520cooperativity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D12342%26epage%3D12345%26doi%3D10.1021%2Fja303564a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, I.</span></span> <span> </span><span class="NLM_article-title">Structural reorganization of the bacterial cell-division protein FtsZ from <i>Staphylococcus aureus</i></span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">1175</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1107/S0907444912022640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444912022640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=22948918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1yqtbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=1175-1188&author=T.+Matsuiauthor=J.+Yamaneauthor=N.+Mogiauthor=H.+Yamaguchiauthor=H.+Takemotoauthor=M.+Yaoauthor=I.+Tanaka&title=Structural+reorganization+of+the+bacterial+cell-division+protein+FtsZ+from+Staphylococcus+aureus&doi=10.1107%2FS0907444912022640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural reorganization of the bacterial cell-division protein FtsZ from Staphylococcus aureus</span></div><div class="casAuthors">Matsui, Takashi; Yamane, Junji; Mogi, Nobuyuki; Yamaguchi, Hiroto; Takemoto, Hiroshi; Yao, Min; Tanaka, Isao</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1175-1188</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">FtsZ is a key mol. in bacterial cell division.  In the presence of GTP, it polymerizes into tubulin-like protofilaments by head-to-tail assocn.  Protofilaments of FtsZ seem to adopt a straight or a curved conformation in relation to the bound nucleotide.  However, although several bacterial and archaeal FtsZ structures have been detd., all of the structures reported previously are considered to have a curved conformation.  In this study, structures of FtsZ from Staphylococcus aureus (SaFtsZ) were detd. in apo, GDP-bound and inhibitor-complex forms and it was found that SaFtsZ undergoes marked conformational changes.  The accumulated evidence suggests that the GDP-bound structure has the features of the straight form.  The structural change between the curved and straight forms shows intriguing similarity to the eukaryotic cytoskeletal protein tubulin.  Furthermore, the structure of the apo form showed an unexpectedly large conformational change in the core region.  FtsZ has also been recognized as a novel target for antibacterial drugs.  The structure of the complex with the inhibitor PC190723, which has potent and selective anti-staphylococcal activity, indicated that the inhibitor binds at the cleft between the two subdomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr36sS4BLhTmbVg90H21EOLACvtfcHk0ljCFeKxOIe_Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1yqtbjJ&md5=c06bbdec17d75b9173deaf2f8a262b91</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444912022640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912022640%26sid%3Dliteratum%253Aachs%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DYamane%26aufirst%3DJ.%26aulast%3DMogi%26aufirst%3DN.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DI.%26atitle%3DStructural%2520reorganization%2520of%2520the%2520bacterial%2520cell-division%2520protein%2520FtsZ%2520from%2520Staphylococcus%2520aureus%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D1175%26epage%3D1188%26doi%3D10.1107%2FS0907444912022640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Löwe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amos, L. A.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the bacterial cell-division protein FtsZ</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1038/34472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2F34472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=9428770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADyaK1cXmt1KisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1998&pages=203-206&author=J.+L%C3%B6weauthor=L.+A.+Amos&title=Crystal+structure+of+the+bacterial+cell-division+protein+FtsZ&doi=10.1038%2F34472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the bacterial cell-division protein FtsZ</span></div><div class="casAuthors">Lowe, Jan; Amos, Linda A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">6663</span>),
    <span class="NLM_cas:pages">203-206</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Bacterial cell division ends with septation, the constriction of the cell wall and cell membranes that leads to the formation of two daughter cells.  During septation, FtsZ, a protein of relative mol. mass 40,000 which is ubiquitous in eubacteria and is also found in archaea and chloroplasts, localizes early at the division site to form a ring-shaped septum.  This septum is required for the mechanochem. process of membrane constriction.  FtsZ is a GTPase with weak sequence homol. to tubulins.  The nature of FtsZ polymers in vivo is unknown, but FtsZ can form tubules, sheets, and minirings in vitro.  Here, the authors report the crystal structure at 2.8 Å resoln. of recombinant FtsZ from the hyperthermophilic methanogen Methanococcus jannaschii.  FtsZ has two domains, one of which is a GTPase domain with a fold related to one found in the proteins p21ras and elongation factor EF-Tu.  The carboxy-terminal domain, whose function is unknown, is a four-stranded β-sheet tilted by 90° against the β-sheet of the GTPase domain.  The two domains are arranged around a central helix.  GDP binding is different from that typically found in GTPases and involves four phosphate-binding loops and a sugar-binding loop in the first domain, with guanine being recognized by residues in the central connecting helix.  The three-dimensional structure of FtsZ is similar to the structure of α- and β-tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIoEQ_Zfl-LVg90H21EOLACvtfcHk0lgy3pce6wTLwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmt1KisA%253D%253D&md5=21ed6e85c5ea92524bdfecdc6028f448</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2F34472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F34472%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6we%26aufirst%3DJ.%26aulast%3DAmos%26aufirst%3DL.%2BA.%26atitle%3DCrystal%2520structure%2520of%2520the%2520bacterial%2520cell-division%2520protein%2520FtsZ%26jtitle%3DNature%26date%3D1998%26volume%3D391%26spage%3D203%26epage%3D206%26doi%3D10.1038%2F34472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araujo-Bazan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Villa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Galiano, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span> <span> </span><span class="NLM_article-title">The structural assembly switch of cell division protein FtsZ probed with fluorescent allosteric inhibitors</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1534</span>, <span class="refDoi"> DOI: 10.1039/C6SC03792E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1039%2FC6SC03792E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28616148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslalsbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1525-1534&author=M.+Artolaauthor=L.+B.+Ruiz-Avilaauthor=E.+Ramirez-Aportelaauthor=R.+F.+Martinezauthor=L.+Araujo-Bazanauthor=H.+Vazquez-Villaauthor=M.+Martin-Fontechaauthor=M.+A.+Olivaauthor=A.+J.+Martin-Galianoauthor=P.+Chaconauthor=M.+L.+Lopez-Rodriguezauthor=J.+M.+Andreuauthor=S.+Huecas&title=The+structural+assembly+switch+of+cell+division+protein+FtsZ+probed+with+fluorescent+allosteric+inhibitors&doi=10.1039%2FC6SC03792E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The structural assembly switch of cell division protein FtsZ probed with fluorescent allosteric inhibitors</span></div><div class="casAuthors">Artola, Marta; Ruiz-Avila, Laura B.; Ramirez-Aportela, Erney; Martinez, R. Fernando; Araujo-Bazan, Lidia; Vazquez-Villa, Henar; Martin-Fontecha, Mar; Oliva, Maria A.; Martin-Galiano, A. Javier; Chacon, Pablo; Lopez-Rodriguez, Maria L.; Andreu, Jose M.; Huecas, Sonia</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1525-1534</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">FtsZ is a widely conserved tubulin-like GTPase that directs bacterial cell division and a new target for antibiotic discovery.  This protein assembly machine cooperatively polymerizes forming single-stranded filaments, by means of self-switching between inactive and actively assocg. monomer conformations.  The structural switch mechanism was proposed to involve a movement of the C-terminal and N-terminal FtsZ domains, opening a cleft between them, allosterically coupled to the formation of a tight assocn. interface between consecutive subunits along the filament.  The effective antibacterial benzamide PC190723 binds into the open interdomain cleft and stabilizes FtsZ filaments, thus impairing correct formation of the FtsZ ring for cell division.  We have designed fluorescent analogs of PC190723 to probe the FtsZ structural assembly switch.  Among them, nitrobenzoxadiazole probes specifically bind to assembled FtsZ rather than to monomers.  Probes with several spacer lengths between the fluorophore and benzamide moieties suggest a binding site extension along the interdomain cleft.  These probes label FtsZ rings of live Bacillus subtilis and Staphylococcus aureus, without apparently modifying normal cell morphol. and growth, but at high concns. they induce impaired bacterial division phenotypes typical of benzamide antibacterials.  During the FtsZ assembly-disassembly process, the fluorescence anisotropy of the probes changes upon binding and dissocg. from FtsZ, thus reporting open and closed FtsZ interdomain clefts.  Our results demonstrate the structural mechanism of the FtsZ assembly switch, and suggest that the probes bind into the open clefts in cellular FtsZ polymers preferably to unassembled FtsZ in the bacterial cytosol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCMy2u0m10FrVg90H21EOLACvtfcHk0lgy3pce6wTLwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslalsbbL&md5=54bba97e4d822c4e96d70e4b82be65bd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC6SC03792E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC03792E%26sid%3Dliteratum%253Aachs%26aulast%3DArtola%26aufirst%3DM.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DRamirez-Aportela%26aufirst%3DE.%26aulast%3DMartinez%26aufirst%3DR.%2BF.%26aulast%3DAraujo-Bazan%26aufirst%3DL.%26aulast%3DVazquez-Villa%26aufirst%3DH.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DOliva%26aufirst%3DM.%2BA.%26aulast%3DMartin-Galiano%26aufirst%3DA.%2BJ.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DHuecas%26aufirst%3DS.%26atitle%3DThe%2520structural%2520assembly%2520switch%2520of%2520cell%2520division%2520protein%2520FtsZ%2520probed%2520with%2520fluorescent%2520allosteric%2520inhibitors%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D1525%26epage%3D1534%26doi%3D10.1039%2FC6SC03792E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizohata, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shigeta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, H.</span></span> <span> </span><span class="NLM_article-title">Identification of the key interactions in structural transition pathway of FtsZ from Staphylococcus aureus</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2017.04.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.jsb.2017.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28456664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A280%3ADC%252BC1crisFejuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=65-73&author=J.+Fujitaauthor=R.+Haradaauthor=Y.+Maedaauthor=Y.+Saitoauthor=E.+Mizohataauthor=T.+Inoueauthor=Y.+Shigetaauthor=H.+Matsumura&title=Identification+of+the+key+interactions+in+structural+transition+pathway+of+FtsZ+from+Staphylococcus+aureus&doi=10.1016%2Fj.jsb.2017.04.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of the key interactions in structural transition pathway of FtsZ from Staphylococcus aureus</span></div><div class="casAuthors">Fujita Junso; Maeda Yoko; Saito Yuki; Mizohata Eiichi; Inoue Tsuyoshi; Harada Ryuhei; Shigeta Yasuteru; Matsumura Hiroyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of structural biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">65-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The tubulin-homolog protein FtsZ is essential for bacterial cell division.  FtsZ polymerizes to form protofilaments that assemble into a contractile ring-shaped structure in the presence of GTP.  Recent studies showed that FtsZ treadmilling coupled with the GTPase activity drives cell wall synthesis and bacterial cell division.  The treadmilling caused by assembly and disassembly of FtsZ links to a conformational change of the monomer from a tense (T) to a relaxed (R) state, but considerable controversy still remains concerning the mechanism.  In this study, we report crystal structures of FtsZ from Staphylococcus aureus corresponding to the T and R state conformations in the same crystal, indicating the structural equilibrium of the two state.  The two structures identified a key residue Arg29, whose importance was also confirmed by our modified MD simulations.  Crystal structures of the R29A mutant showed T and R state-like conformations with slight but important structural changes compared to those of wild-type.  Collectively, these data provide new insights for understanding how intramolecular interactions are related to the structural transition of FtsZ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6iihP5eyfJWgRQsDqrjW_fW6udTcc2eacfgCWoXXBIbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crisFejuw%253D%253D&md5=b5132354e5b4720ce49c6c03032331da</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2017.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2017.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DJ.%26aulast%3DHarada%26aufirst%3DR.%26aulast%3DMaeda%26aufirst%3DY.%26aulast%3DSaito%26aufirst%3DY.%26aulast%3DMizohata%26aufirst%3DE.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DShigeta%26aufirst%3DY.%26aulast%3DMatsumura%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520the%2520key%2520interactions%2520in%2520structural%2520transition%2520pathway%2520of%2520FtsZ%2520from%2520Staphylococcus%2520aureus%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2017%26volume%3D198%26spage%3D65%26epage%3D73%26doi%3D10.1016%2Fj.jsb.2017.04.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagstaff, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Sanchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kureisaite-Ciziene, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, J.</span></span> <span> </span><span class="NLM_article-title">A polymerization-associated structural switch in FtsZ that enables treadmilling of model filaments</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e00254-17</span> <span class="refDoi"> DOI: 10.1128/mBio.00254-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1128%2FmBio.00254-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=28465423" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=J.+M.+Wagstaffauthor=M.+Tsimauthor=M.+A.+Olivaauthor=A.+Garcia-Sanchezauthor=D.+Kureisaite-Cizieneauthor=J.+M.+Andreuauthor=J.+Lowe&title=A+polymerization-associated+structural+switch+in+FtsZ+that+enables+treadmilling+of+model+filaments&doi=10.1128%2FmBio.00254-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1128%2FmBio.00254-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.00254-17%26sid%3Dliteratum%253Aachs%26aulast%3DWagstaff%26aufirst%3DJ.%2BM.%26aulast%3DTsim%26aufirst%3DM.%26aulast%3DOliva%26aufirst%3DM.%2BA.%26aulast%3DGarcia-Sanchez%26aufirst%3DA.%26aulast%3DKureisaite-Ciziene%26aufirst%3DD.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DLowe%26aufirst%3DJ.%26atitle%3DA%2520polymerization-associated%2520structural%2520switch%2520in%2520FtsZ%2520that%2520enables%2520treadmilling%2520of%2520model%2520filaments%26jtitle%3DmBio%26date%3D2017%26volume%3D8%26doi%3D10.1128%2FmBio.00254-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Blanco, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span> <span> </span><span class="NLM_article-title">Understanding nucleotide-regulated FtsZ filament dynamics and the monomer assembly switch with large-scale atomistic simulations</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">2164</span>– <span class="NLM_lpage">2176</span>, <span class="refDoi"> DOI: 10.1016/j.bpj.2014.09.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1016%2Fj.bpj.2014.09.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=25418101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGrsL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2014&pages=2164-2176&author=E.+Ram%C3%ADrez-Aportelaauthor=J.+R.+Lopez-Blancoauthor=J.+M.+Andreuauthor=P.+Chacon&title=Understanding+nucleotide-regulated+FtsZ+filament+dynamics+and+the+monomer+assembly+switch+with+large-scale+atomistic+simulations&doi=10.1016%2Fj.bpj.2014.09.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Nucleotide-Regulated FtsZ Filament Dynamics and the Monomer Assembly Switch with Large-Scale Atomistic Simulations</span></div><div class="casAuthors">Ramirez-Aportela, Erney; Lopez-Blanco, Jose Ramon; Andreu, Jose Manuel; Chacon, Pablo</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2164-2176</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bacterial cytoskeletal protein FtsZ assembles in a head-to-tail manner, forming dynamic filaments that are essential for cell division.  Here, we study their dynamics using unbiased atomistic mol. simulations from representative filament crystal structures.  In agreement with exptl. data, we find different filament curvatures that are supported by a nucleotide-regulated hinge motion between consecutive FtsZ monomers.  Whereas GTP-FtsZ filaments bend and twist in a preferred orientation, thereby burying the nucleotide, the differently curved GDP-FtsZ filaments exhibit a heterogeneous distribution of open and closed interfaces between monomers.  We identify a coordinated Mg2+ ion as the key structural element in closing the nucleotide site and stabilizing GTP filaments, whereas the loss of the contacts with loop T7 from the next monomer in GDP filaments leads to open interfaces that are more prone to depolymn.  We monitored the FtsZ monomer assembly switch, which involves opening/closing of the cleft between the C-terminal domain and the H7 helix, and obsd. the relaxation of isolated and filament minus-end monomers into the closed-cleft inactive conformation.  This result validates the proposed switch between the low-affinity monomeric closed-cleft conformation and the active open-cleft FtsZ conformation within filaments.  Finally, we obsd. how the antibiotic PC190723 suppresses the disassembly switch and allosterically induces closure of the intermonomer interfaces, thus stabilizing the filament.  Our studies provide detailed structural and dynamic insights into modulation of both the intrinsic curvature of the FtsZ filaments and the mol. switch coupled to the high-affinity end-wise assocn. of FtsZ monomers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7kR2AP1MkN7Vg90H21EOLACvtfcHk0lg3iot_XCRO9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGrsL3M&md5=59c71ab7e2dfb53693a2336981e316cd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2014.09.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2014.09.033%26sid%3Dliteratum%253Aachs%26aulast%3DRam%25C3%25ADrez-Aportela%26aufirst%3DE.%26aulast%3DLopez-Blanco%26aufirst%3DJ.%2BR.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DChacon%26aufirst%3DP.%26atitle%3DUnderstanding%2520nucleotide-regulated%2520FtsZ%2520filament%2520dynamics%2520and%2520the%2520monomer%2520assembly%2520switch%2520with%2520large-scale%2520atomistic%2520simulations%26jtitle%3DBiophys.%2520J.%26date%3D2014%26volume%3D107%26spage%3D2164%26epage%3D2176%26doi%3D10.1016%2Fj.bpj.2014.09.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Araújo-Bazán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Cytological profile of antibacterial FtsZ inhibitors and synthetic peptide MciZ</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">1558</span> <span class="refDoi"> DOI: 10.3389/fmicb.2016.01558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3389%2Ffmicb.2016.01558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=27752253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A280%3ADC%252BC2srhsFOnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=L.+Ara%C3%BAjo-Baz%C3%A1nauthor=L.+B.+Ruiz-Avilaauthor=D.+Andreuauthor=S.+Huecasauthor=J.+M.+Andreu&title=Cytological+profile+of+antibacterial+FtsZ+inhibitors+and+synthetic+peptide+MciZ&doi=10.3389%2Ffmicb.2016.01558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cytological Profile of Antibacterial FtsZ Inhibitors and Synthetic Peptide MciZ</span></div><div class="casAuthors">Araujo-Bazan Lidia; Ruiz-Avila Laura B; Huecas Sonia; Andreu Jose M; Andreu David</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1558</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">Cell division protein FtsZ is the organizer of the cytokinetic ring in almost all bacteria and a target for the discovery of new antibacterial agents that are needed to counter widespread antibiotic resistance.  Bacterial cytological profiling, using quantitative microscopy, is a powerful approach for identifying the mechanism of action of antibacterial molecules affecting different cellular pathways.  We have determined the cytological profile on Bacillus subtilis cells of a selection of small molecule inhibitors targeting FtsZ on different binding sites.  FtsZ inhibitors lead to long undivided cells, impair the normal assembly of FtsZ into the midcell Z-rings, induce aberrant ring distributions, punctate FtsZ foci, membrane spots and also modify nucleoid length.  Quantitative analysis of cell and nucleoid length combined, or the Z-ring distribution, allows categorizing FtsZ inhibitors and to distinguish them from antibiotics with other mechanisms of action, which should be useful for identifying new antibacterial FtsZ inhibitors.  Biochemical assays of FtsZ polymerization and GTPase activity combined explain the cellular effects of the FtsZ polymer stabilizing agent PC190723 and its fragments.  MciZ is a 40-aminoacid endogenous inhibitor of cell division normally expressed during sporulation in B. subtilis.  Using FtsZ cytological profiling we have determined that exogenous synthetic MciZ is an effective inhibitor of B. subtilis cell division, Z-ring formation and localization.  This finding supports our cell-based approach to screen for FtsZ inhibitors and opens new possibilities for peptide inhibitors of bacterial cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQwn9gu9848Z7FU0A1HOshhfW6udTcc2eZWwVB8eHWAHrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srhsFOnsg%253D%253D&md5=4fc9fa0327e0f3b86662f7fb7b0fe742</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2016.01558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2016.01558%26sid%3Dliteratum%253Aachs%26aulast%3DAra%25C3%25BAjo-Baz%25C3%25A1n%26aufirst%3DL.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DAndreu%26aufirst%3DD.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DCytological%2520profile%2520of%2520antibacterial%2520FtsZ%2520inhibitors%2520and%2520synthetic%2520peptide%2520MciZ%26jtitle%3DFront.%2520Microbiol.%26date%3D2016%26volume%3D7%26doi%3D10.3389%2Ffmicb.2016.01558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Araújo-Bazán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Synthetic developmental regulator MciZ targets FtsZ across Bacillus species and inhibits bacterial division</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">965</span>– <span class="NLM_lpage">980</span>, <span class="refDoi"> DOI: 10.1111/mmi.14198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Fmmi.14198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30636070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVyhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2019&pages=965-980&author=L.+Ara%C3%BAjo-Baz%C3%A1nauthor=S.+Huecasauthor=J.+Valleauthor=D.+Andreuauthor=J.+M.+Andreu&title=Synthetic+developmental+regulator+MciZ+targets+FtsZ+across+Bacillus+species+and+inhibits+bacterial+division&doi=10.1111%2Fmmi.14198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic developmental regulator MciZ targets FtsZ across Bacillus species and inhibits bacterial division</span></div><div class="casAuthors">Araujo-Bazan, Lidia; Huecas, Sonia; Valle, Javier; Andreu, David; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">965-980</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Cell division in most bacteria is directed by FtsZ, a conserved tubulin-like GTPase that assembles forming the cytokinetic Z-ring and constitutes a target for the discovery of new antibiotics.  The developmental regulator MciZ, a 40-amino acid peptide endogenously produced during Bacillus subtilis sporulation, halts cytokinesis in the mother cell by inhibiting FtsZ.  The crystal structure of a FtsZ:MciZ complex revealed that bound MciZ extends the C-terminal β-sheet of FtsZ blocking its assembly interface.  Here we demonstrate that exogenously added MciZ specifically inhibits B. subtilis cell division, sporulation and germination, and provide insight into MciZ mol. recognition by FtsZ from different bacteria.  MciZ and FtsZ form a complex with sub-micromolar affinity, analyzed by anal. ultracentrifugation, laser biolayer interferometry and isothermal titrn. calorimetry.  Synthetic MciZ analogs, carrying single amino acid substitutions impairing MciZ β-strand formation or hydrogen bonding to FtsZ, show a gradual redn. in affinity that resembles their impaired activity in bacteria.  Gene sequences encoding MciZ spread across genus Bacillus and synthetic MciZ slows down cell division in Bacillus species, including pathogenic Bacillus cereus and Bacillus anthracis.  Moreover, B. subtilis MciZ is recognized by the homologous FtsZ from Staphylococcus aureus and inhibits division when it is expressed into S. aureus cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXrHpsPiqwnrVg90H21EOLACvtfcHk0liFysviB2CgqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVyhsLY%253D&md5=bb61f1f52a179f0c524a573240bf6b9b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fmmi.14198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fmmi.14198%26sid%3Dliteratum%253Aachs%26aulast%3DAra%25C3%25BAjo-Baz%25C3%25A1n%26aufirst%3DL.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DValle%26aufirst%3DJ.%26aulast%3DAndreu%26aufirst%3DD.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DSynthetic%2520developmental%2520regulator%2520MciZ%2520targets%2520FtsZ%2520across%2520Bacillus%2520species%2520and%2520inhibits%2520bacterial%2520division%26jtitle%3DMol.%2520Microbiol.%26date%3D2019%26volume%3D111%26spage%3D965%26epage%3D980%26doi%3D10.1111%2Fmmi.14198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Díaz, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buey, R. M.</span></span> <span> </span><span class="NLM_article-title">Characterizing ligand-microtubule binding by competition methods</span>. <i>Methods Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1007/978-1-59745-442-1_17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1007%2F978-1-59745-442-1_17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=18085234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wns7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2007&pages=245-260&author=J.+F.+D%C3%ADazauthor=R.+M.+Buey&title=Characterizing+ligand-microtubule+binding+by+competition+methods&doi=10.1007%2F978-1-59745-442-1_17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Characterizing ligand-microtubule binding by competition methods</span></div><div class="casAuthors">Diaz, Jose Fernando; Buey, Ruben Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">Microtubule Protocols</span>),
    <span class="NLM_cas:pages">245-260</span>CODEN:
                <span class="NLM_cas:coden">MMMEFN</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">The knowledge of the thermodn. and kinetics of drug-microtubule interaction is essential to understand the structure/affinity relationship of a given ligand family.  When a ligand does not show an appropriate signal change (absorbance or fluorescence) upon binding, the extensive direct characterization of its binding affinities and kinetic rate consts. of assocn. and dissocn. becomes a complex task.  In those cases it is possible to obtain these parameters by competition of the ligand with a ref. one of the same binding site that shows such change.  Nevertheless, although the exptl. setup of the competition measurements is easier, the treatment of the data is complex because simultaneous equil./kinetic equations have to be solved.  In this chapter, the taxoid-binding site of the microtubules will be used as an example to describe exptl. competition and data anal. methods to det. the binding consts. and kinetic rates of assocn. and dissocn. of ligands for microtubules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jHl9QBWSmbVg90H21EOLACvtfcHk0liFysviB2CgqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wns7jK&md5=324bb229b1639db49245f2b85ccc19ec</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2F978-1-59745-442-1_17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-59745-442-1_17%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25ADaz%26aufirst%3DJ.%2BF.%26aulast%3DBuey%26aufirst%3DR.%2BM.%26atitle%3DCharacterizing%2520ligand-microtubule%2520binding%2520by%2520competition%2520methods%26jtitle%3DMethods%2520Mol.%2520Med.%26date%3D2007%26volume%3D137%26spage%3D245%26epage%3D260%26doi%3D10.1007%2F978-1-59745-442-1_17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canosa-Valls, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araújo-Bazán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurents, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Tornero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Nucleotide-induced folding of cell division protein FtsZ from Staphylococcus aureus</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">4048</span>– <span class="NLM_lpage">4067</span>, <span class="refDoi"> DOI: 10.1111/febs.15235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Ffebs.15235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=31997533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjs1ansbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2020&pages=4048-4067&author=S.+Huecasauthor=A.+J.+Canosa-Vallsauthor=L.+Ara%C3%BAjo-Baz%C3%A1nauthor=F.+M.+Ruizauthor=D.+V.+Laurentsauthor=C.+Fern%C3%A1ndez-Torneroauthor=J.+M.+Andreu&title=Nucleotide-induced+folding+of+cell+division+protein+FtsZ+from+Staphylococcus+aureus&doi=10.1111%2Ffebs.15235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleotide-induced folding of cell division protein FtsZ from Staphylococcus aureus</span></div><div class="casAuthors">Huecas, Sonia; Canosa-Valls, Alejandro J.; Araujo-Bazan, Lidia; Ruiz, Federico M.; Laurents, Douglas V.; Fernandez-Tornero, Carlos; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4048-4067</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The essential bacterial division protein FtsZ uses GTP binding and hydrolysis to assemble into dynamic filaments that treadmill around the Z-ring, guiding septal wall synthesis and cell division.  FtsZ is a structural homolog of tubulin and a target for discovering new antibiotics.  Here, using FtsZ from the pathogen S. aureus (SaFtsZ), we reveal that, prior to assembly, FtsZ monomers require nucleotide binding for folding; this is possibly relevant to other mesophilic FtsZs.  Apo-SaFtsZ is essentially unfolded, as assessed by NMR and CD.  Binding of GTP (≥ 1 mM) dramatically shifts the equil. toward the active folded protein.  Supportingly, SaFtsZ refolded with GDP crystallizes in a native structure.  Apo-SaFtsZ also folds with 3.4 M glycerol, enabling high-affinity GTP binding (KD 20 nM detd. by isothermal titrn. calorimetry) similar to thermophilic stable FtsZ.  Other stabilizing agents that enhance nucleotide binding include ethylene glycol, trimethylamine N-oxide, and several bacterial osmolytes.  High salt stabilizes SaFtsZ without bound nucleotide in an inactive twisted conformation.  We identified a cavity behind the SaFtsZ-GDP nucleotide-binding pocket that harbors different small compds., which is available for extended nucleotide-replacing inhibitors.  Furthermore, we devised a competition assay to detect any inhibitors that overlap the nucleotide site of SaFtsZ, or Escherichia coli FtsZ, employing osmolyte-stabilized apo-FtsZs and the specific fluorescence anisotropy change in mant-GTP upon dissocn. from the protein.  This robust assay provides a basis to screening for high-affinity GTP-replacing ligands, which combined with structural studies and phenotypic profiling should facilitate development of a next generation of FtsZ-targeting antibacterial inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV19nVjkdRurVg90H21EOLACvtfcHk0liFysviB2CgqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjs1ansbs%253D&md5=4f40b31dde49f0a4b480f15fc5b433ef</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Ffebs.15235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.15235%26sid%3Dliteratum%253Aachs%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DCanosa-Valls%26aufirst%3DA.%2BJ.%26aulast%3DAra%25C3%25BAjo-Baz%25C3%25A1n%26aufirst%3DL.%26aulast%3DRuiz%26aufirst%3DF.%2BM.%26aulast%3DLaurents%26aufirst%3DD.%2BV.%26aulast%3DFern%25C3%25A1ndez-Tornero%26aufirst%3DC.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DNucleotide-induced%2520folding%2520of%2520cell%2520division%2520protein%2520FtsZ%2520from%2520Staphylococcus%2520aureus%26jtitle%3DFEBS%2520J.%26date%3D2020%26volume%3D287%26spage%3D4048%26epage%3D4067%26doi%3D10.1111%2Ffebs.15235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergonos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cercenado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barasoain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Villa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Synthetic inhibitors of bacterial cell division targeting the GTP-binding site of FtsZ</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2072</span>– <span class="NLM_lpage">2083</span>, <span class="refDoi"> DOI: 10.1021/cb400208z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400208z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSnsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2072-2083&author=L.+B.+Ruiz-Avilaauthor=S.+Huecasauthor=M.+Artolaauthor=A.+Vergonosauthor=E.+Ramirez-Aportelaauthor=E.+Cercenadoauthor=I.+Barasoainauthor=H.+Vazquez-Villaauthor=M.+Martin-Fontechaauthor=P.+Chaconauthor=M.+L.+Lopez-Rodriguezauthor=J.+M.+Andreu&title=Synthetic+inhibitors+of+bacterial+cell+division+targeting+the+GTP-binding+site+of+FtsZ&doi=10.1021%2Fcb400208z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Inhibitors of Bacterial Cell Division Targeting the GTP-Binding Site of FtsZ</span></div><div class="casAuthors">Ruiz-Avila, Laura B.; Huecas, Sonia; Artola, Marta; Vergonos, Albert; Ramirez-Aportela, Erney; Cercenado, Emilia; Barasoain, Isabel; Vazquez-Villa, Henar; Martin-Fontecha, Mar; Chacon, Pablo; Lopez-Rodriguez, Maria L.; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2072-2083</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell division protein FtsZ is the organizer of the cytokinetic Z-ring in most bacteria and a target for new antibiotics.  FtsZ assembles with GTP into filaments that hydrolyze the nucleotide at the assocn. interface between monomers and then disassemble.  FtsZ's GTP was replaced with non-nucleotide synthetic inhibitors of bacterial division.  The authors searched for these small mols. among compds. from the literature, from virtual screening (VS), and from their inhouse synthetic library (UCM), employing a fluorescence anisotropy primary assay.  From these screens they identified the polyhydroxy arom. compd. UCM05 and its simplified analog UCM44 that specifically bind to Bacillus subtilis FtsZ monomers with micromolar affinities and perturb normal assembly, as examd. with light scattering, polymer sedimentation, and neg. stain electron microscopy.  On the other hand, these ligands induce the cooperative assembly of nucleotide-devoid archaeal FtsZ into distinct well-ordered polymers, different from GTP-induced filaments.  These FtsZ inhibitors impair localization of FtsZ into the Z-ring and inhibit bacterial cell division.  The chlorinated analog UCM53 inhibits the growth of clin. isolates of antibiotic-resistant Staphylococcus aureus and Enterococcus faecalis.  It is suggested that these interfacial inhibitors recapitulate binding and some assembly-inducing effects of GTP but impair the correct structural dynamics of FtsZ filaments and thus inhibit bacterial division, possibly by binding to a small fraction of the FtsZ mols. in a bacterial cell, which opens a new approach to FtsZ-based antibacterial drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIArKPncswxrVg90H21EOLACvtfcHk0lg56fI06zuJrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSnsbnO&md5=3a08fa169f279cc7c7790b23a25eae6f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcb400208z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400208z%26sid%3Dliteratum%253Aachs%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DArtola%26aufirst%3DM.%26aulast%3DVergonos%26aufirst%3DA.%26aulast%3DRamirez-Aportela%26aufirst%3DE.%26aulast%3DCercenado%26aufirst%3DE.%26aulast%3DBarasoain%26aufirst%3DI.%26aulast%3DVazquez-Villa%26aufirst%3DH.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DSynthetic%2520inhibitors%2520of%2520bacterial%2520cell%2520division%2520targeting%2520the%2520GTP-binding%2520site%2520of%2520FtsZ%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2072%26epage%3D2083%26doi%3D10.1021%2Fcb400208z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barasoain, I.</span></span> <span> </span><span class="NLM_article-title">The interaction of baccatin III with the taxol binding site of microtubules determined by a homogeneous assay with fluorescent taxoid</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">11975</span>– <span class="NLM_lpage">11984</span>, <span class="refDoi"> DOI: 10.1021/bi010869+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi010869%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1OgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=11975-11984&author=J.+M.+Andreuauthor=I.+Barasoain&title=The+interaction+of+baccatin+III+with+the+taxol+binding+site+of+microtubules+determined+by+a+homogeneous+assay+with+fluorescent+taxoid&doi=10.1021%2Fbi010869%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The Interaction of Baccatin III with the Taxol Binding Site of Microtubules Determined by a Homogeneous Assay with Fluorescent Taxoid</span></div><div class="casAuthors">Andreu, Jose M.; Barasoain, Isabel</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">11975-11984</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ubiquitous Taxol binding site of microtubules also binds newly discovered ligands.  We have designed a homogeneous assay for the high throughput detection of Taxol biomimetics, based on the displacement of 7-O-[N-(2,7-difluoro-4'-fluoresceincarbonyl)-L-alanyl]Taxol from its binding site in dild. solns. of preserved microtubules.  The state of this ref. ligand is measured by fluorescence anisotropy in a microplate reader, with varying concns. of nonfluorescent competitors.  The binding equil. const. of Taxol has a value Kb = 3.7×107 M-1.  We have found that baccatin III, an analog of Taxol without the C-13 side chain, binds with Kb = 1.5×105 M-1, whereas the side chain Me ester is inactive.  This was unexpected from the structure-activity relation of taxoids but compatible with models of Taxol docked at the microtubule site.  Baccatin III binding has been confirmed by displacement of [3H]Taxol and by direct HPLC measurements of its cosedimentation with microtubules, among other methods.  Consequently, baccatin III induces microtubule bundles and multipolar spindles in PtK2 and U937 cells, and mitotic arrest and apoptotic death of the U937 cells, at concns. 200-500-fold larger than Taxol.  The simplest anal. of these results strongly suggests that the interaction of the C-2 C-4 substituted taxane ring system with the microtubule binding site provides most (75%) of the free energy change of Taxol binding and is sufficient to activate microtubule stabilization and transmit the antitumor effects of Taxol, whereas the C-13 side chain provides a weak specific anchor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4qoUCYS9qXLVg90H21EOLACvtfcHk0lg56fI06zuJrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1OgsLs%253D&md5=06ff2874f0298f9358699bc3335d50e4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fbi010869%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi010869%252B%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DBarasoain%26aufirst%3DI.%26atitle%3DThe%2520interaction%2520of%2520baccatin%2520III%2520with%2520the%2520taxol%2520binding%2520site%2520of%2520microtubules%2520determined%2520by%2520a%2520homogeneous%2520assay%2520with%2520fluorescent%2520taxoid%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D11975%26epage%3D11984%26doi%3D10.1021%2Fbi010869%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomaides, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haydon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czaplewski, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, J.</span></span> <span> </span><span class="NLM_article-title">Novel inhibitors of bacterial cytokinesis identified by a cell-based antibiotic screening assay</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">39709</span>– <span class="NLM_lpage">39715</span>, <span class="refDoi"> DOI: 10.1074/jbc.M506741200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M506741200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=16174771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1GjsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=39709-39715&author=N.+R.+Stokesauthor=J.+Sieversauthor=S.+Barkerauthor=J.+M.+Bennettauthor=D.+R.+Brownauthor=I.+Collinsauthor=V.+M.+Erringtonauthor=D.+Foulgerauthor=M.+Hallauthor=R.+Halseyauthor=H.+Johnsonauthor=V.+Roseauthor=H.+B.+Thomaidesauthor=D.+J.+Haydonauthor=L.+G.+Czaplewskiauthor=J.+Errington&title=Novel+inhibitors+of+bacterial+cytokinesis+identified+by+a+cell-based+antibiotic+screening+assay&doi=10.1074%2Fjbc.M506741200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitors of Bacterial Cytokinesis Identified by a Cell-based Antibiotic Screening Assay</span></div><div class="casAuthors">Stokes, Neil R.; Sievers, Joerg; Barker, Stephanie; Bennett, James M.; Brown, David R.; Collins, Ian; Errington, Veronica M.; Foulger, David; Hall, Michelle; Halsey, Rowena; Johnson, Hazel; Rose, Valerie; Thomaides, Helena B.; Haydon, David J.; Czaplewski, Lloyd G.; Errington, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">39709-39715</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The continuous emergence of antibiotic resistance demands that novel classes of antibiotics continue to be developed.  The division machinery of bacteria is an attractive target because it comprises seven or more essential proteins that are conserved almost throughout the bacteria but are absent from humans.  We describe the development of a cell-based assay for inhibitors of cell division and its use to isolate a new inhibitor of FtsZ protein, a key player in the division machinery.  Biochem., cytol., and genetic data are presented that demonstrate that FtsZ is the specific target for the compd.  We also describe the effects of more potent analogs of the original hit compd. that act on important pathogens, again at the level of cell division.  The assay and the compds. have the potential to provide novel antibiotics with no pool of pre-existing resistance.  They have provided new insight into cytokinesis in bacteria and offer important reagents for further studies of the cell division machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_AR5gKzeaVbVg90H21EOLACvtfcHk0ljufe0IaTEHZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1GjsrbK&md5=7f0a57ef32a558366517e826e25b5347</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M506741200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M506741200%26sid%3Dliteratum%253Aachs%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DSievers%26aufirst%3DJ.%26aulast%3DBarker%26aufirst%3DS.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBrown%26aufirst%3DD.%2BR.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DErrington%26aufirst%3DV.%2BM.%26aulast%3DFoulger%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DM.%26aulast%3DHalsey%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DH.%26aulast%3DRose%26aufirst%3DV.%26aulast%3DThomaides%26aufirst%3DH.%2BB.%26aulast%3DHaydon%26aufirst%3DD.%2BJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26aulast%3DErrington%26aufirst%3DJ.%26atitle%3DNovel%2520inhibitors%2520of%2520bacterial%2520cytokinesis%2520identified%2520by%2520a%2520cell-based%2520antibiotic%2520screening%2520assay%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D39709%26epage%3D39715%26doi%3D10.1074%2Fjbc.M506741200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margalit, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romberg, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mets, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RayChaudhuri, D.</span></span> <span> </span><span class="NLM_article-title">Targeting cell division: Small-molecule inhibitors of FtsZ GTPase perturb cytokinetic ring assembly and induce bacterial lethality</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">11821</span>– <span class="NLM_lpage">11826</span>, <span class="refDoi"> DOI: 10.1073/pnas.0404439101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1073%2Fpnas.0404439101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=15289600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntVers7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=11821-11826&author=D.+N.+Margalitauthor=L.+Rombergauthor=R.+B.+Metsauthor=A.+M.+Hebertauthor=T.+J.+Mitchisonauthor=M.+W.+Kirschnerauthor=D.+RayChaudhuri&title=Targeting+cell+division%3A+Small-molecule+inhibitors+of+FtsZ+GTPase+perturb+cytokinetic+ring+assembly+and+induce+bacterial+lethality&doi=10.1073%2Fpnas.0404439101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cell division: Small-molecule inhibitors of FtsZ GTPase perturb cytokinetic ring assembly and induce bacterial lethality</span></div><div class="casAuthors">Margalit, Danielle N.; Romberg, Laura; Mets, Rebecca B.; Hebert, Alan M.; Mitchison, Timothy J.; Kirschner, Marc W.; RayChaudhuri, Debabrata</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">11821-11826</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">FtsZ, the ancestral homolog of eukaryotic tubulins, is a GTPase that assembles into a cytokinetic ring structure essential for cell division in prokaryotic cells.  Similar to tubulin, purified FtsZ polymerizes into dynamic protofilaments in the presence of GTP; polymer assembly is accompanied by GTP hydrolysis.  The authors used a high-throughput protein-based chem. screen to identify small mols. that target assembly-dependent GTPase activity of FtsZ.  Here, the authors report the identification of five structurally diverse compds., named Zantrins, which inhibit FtsZ GTPase either by destabilizing the FtsZ protofilaments or by inducing filament hyperstability through increased lateral assocn.  These two classes of FtsZ inhibitors are reminiscent of the antitubulin drugs colchicine and Taxol, resp.  The authors also show that Zantrins perturb FtsZ ring assembly in Escherichia coli cells and cause lethality to a variety of bacteria in broth cultures, indicating that FtsZ antagonists may serve as chem. leads for the development of new broad-spectrum antibacterial agents.  Our results illustrate the utility of small-mol. chem. probes to study FtsZ polymn. dynamics and the feasibility of FtsZ as a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOPgYgvnXeKLVg90H21EOLACvtfcHk0ljufe0IaTEHZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntVers7s%253D&md5=ed037cc4b71bb294aff750bea98351e8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0404439101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0404439101%26sid%3Dliteratum%253Aachs%26aulast%3DMargalit%26aufirst%3DD.%2BN.%26aulast%3DRomberg%26aufirst%3DL.%26aulast%3DMets%26aufirst%3DR.%2BB.%26aulast%3DHebert%26aufirst%3DA.%2BM.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DRayChaudhuri%26aufirst%3DD.%26atitle%3DTargeting%2520cell%2520division%253A%2520Small-molecule%2520inhibitors%2520of%2520FtsZ%2520GTPase%2520perturb%2520cytokinetic%2520ring%2520assembly%2520and%2520induce%2520bacterial%2520lethality%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D11821%26epage%3D11826%26doi%3D10.1073%2Fpnas.0404439101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorto, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackner, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ames, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. T.</span></span> <span> </span><span class="NLM_article-title">Comparison of small molecule inhibitors of the bacterial cell division protein FtsZ and identification of a reliable cross-species inhibitor</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1918</span>– <span class="NLM_lpage">1928</span>, <span class="refDoi"> DOI: 10.1021/cb300340j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300340j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1918-1928&author=D.+E.+Andersonauthor=M.+B.+Kimauthor=J.+T.+Mooreauthor=T.+E.+O%E2%80%99Brienauthor=N.+A.+Sortoauthor=C.+I.+Groveauthor=L.+L.+Lacknerauthor=J.+B.+Amesauthor=J.+T.+Shaw&title=Comparison+of+small+molecule+inhibitors+of+the+bacterial+cell+division+protein+FtsZ+and+identification+of+a+reliable+cross-species+inhibitor&doi=10.1021%2Fcb300340j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Small Molecule Inhibitors of the Bacterial Cell Division Protein FtsZ and Identification of a Reliable Cross-Species Inhibitor</span></div><div class="casAuthors">Anderson, David E.; Kim, Michelle B.; Moore, Jared T.; O'Brien, Terrence E.; Sorto, Nohemy A.; Grove, Charles I.; Lackner, Laura L.; Ames, James B.; Shaw, Jared T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1918-1928</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FtsZ is a guanosine triphosphatase (GTPase) that mediates cytokinesis in bacteria.  FtsZ is homologous in structure to eukaryotic tubulin and polymerizes in a similar head-to-tail fashion.  The study of tubulin's function in eukaryotic cells has benefited greatly from specific and potent small mol. inhibitors, including colchicine and taxol.  Although many small mol. inhibitors of FtsZ have been reported, none has emerged as a generally useful probe for modulating bacterial cell division.  With the goal of establishing a useful and reliable small mol. inhibitor of FtsZ, a broad biochem. cross-comparison of reported FtsZ inhibitors was undertaken.  Several of these mols., including phenolic natural products, are unselective inhibitors that seem to derive their activity from the formation of microscopic colloids or aggregates.  Other compds., including the natural product viriditoxin and the drug development candidate PC190723, exhibit no inhibition of GTPase activity using protocols or under published conditions.  Of the compds. studied, only zantrin Z3 exhibits good levels of inhibition, maintains activity under conditions that disrupt small mol. aggregates, and provides a platform for exploration of structure-activity relationships (SAR).  Preliminary SAR studies have identified slight modifications to the two side chains of this structure that modulate the inhibitory activity of zantrin Z3.  Collectively, these studies will help focus future investigations toward the establishment of probes for FtsZ that fill the roles of colchicine and taxol in studies of tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvE8_fy6gEwLVg90H21EOLACvtfcHk0ljufe0IaTEHZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL&md5=ab69ea62af74ad72f6177db5f556c990</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fcb300340j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300340j%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BE.%26aulast%3DKim%26aufirst%3DM.%2BB.%26aulast%3DMoore%26aufirst%3DJ.%2BT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%2BE.%26aulast%3DSorto%26aufirst%3DN.%2BA.%26aulast%3DGrove%26aufirst%3DC.%2BI.%26aulast%3DLackner%26aufirst%3DL.%2BL.%26aulast%3DAmes%26aufirst%3DJ.%2BB.%26aulast%3DShaw%26aufirst%3DJ.%2BT.%26atitle%3DComparison%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520bacterial%2520cell%2520division%2520protein%2520FtsZ%2520and%2520identification%2520of%2520a%2520reliable%2520cross-species%2520inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1918%26epage%3D1928%26doi%3D10.1021%2Fcb300340j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, Y. H.</span></span> <span> </span><span class="NLM_article-title">Resveratrol antibacterial activity against Escherichia coli is mediated by Z-ring formation inhibition via suppression of FtsZ expression</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">10029</span> <span class="refDoi"> DOI: 10.1038/srep10029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fsrep10029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=25942564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOrs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&author=D.+Hwangauthor=Y.+H.+Lim&title=Resveratrol+antibacterial+activity+against+Escherichia+coli+is+mediated+by+Z-ring+formation+inhibition+via+suppression+of+FtsZ+expression&doi=10.1038%2Fsrep10029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol antibacterial activity against Escherichia coli is mediated by Z-ring formation inhibition via suppression of FtsZ expression</span></div><div class="casAuthors">Hwang, Dahyun; Lim, Young-Hee</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10029</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Resveratrol exhibits a potent antimicrobial activity.  However, the mechanism underlying its antibacterial activity has not been shown.  In this study, the antibacterial mechanism of resveratrol was investigated.  To investigate induction of the SOS response, a strain contg. the lacZ+gene under the control of an SOS-inducible sulA promoter was constructed.  DNA damage was measured by pulse-field gel electrophoresis (PFGE).  After resveratrol treatment, the cells were obsd. by confocal microscopy.  For the RNA silencing assay, ftsZ-specific antisense peptide nucleic acid (PNA) was used.  Reactive oxygen species (ROS) prodn. increased in Escherichia coli after resveratrol treatment; however, cell growth was not recovered by ROS quenching, indicating that, in this expt., ROS formation and cell death following resveratrol treatment were not directly correlated.  Resveratrol treatment increased DNA fragmentation in cells, while SOS response-related gene expression levels increased in a dose-dependent manner.  Cell elongation was obsd. after resveratrol treatment.  Elongation was induced by inhibiting FtsZ, an essential cell-division protein in prokaryotes, and resulted in significant inhibition of Z-ring the formation in E. coli.  The expression of ftsZ mRNA was suppressed by resveratrol.  Thus, the results indicate that resveratrol inhibits bacterial cell growth by suppressing FtsZ expression and Z-ring formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovOV74pcApNrVg90H21EOLACvtfcHk0ljhJ5Q81cD85Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOrs7bJ&md5=9e72a33b5085fe517ba981d21b2e6c93</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fsrep10029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep10029%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DY.%2BH.%26atitle%3DResveratrol%2520antibacterial%2520activity%2520against%2520Escherichia%2520coli%2520is%2520mediated%2520by%2520Z-ring%2520formation%2520inhibition%2520via%2520suppression%2520of%2520FtsZ%2520expression%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26doi%3D10.1038%2Fsrep10029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jindal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panda, D.</span></span> <span> </span><span class="NLM_article-title">Plumbagin inhibits cytokinesis in <i>Bacillus subtilis</i> by inhibiting FtsZ assembly--a mechanistic study of its antibacterial activity</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">4585</span>– <span class="NLM_lpage">4599</span>, <span class="refDoi"> DOI: 10.1111/febs.12429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Ffebs.12429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=23841620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGmtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=4585-4599&author=A.+Bhattacharyaauthor=B.+Jindalauthor=P.+Singhauthor=A.+Dattaauthor=D.+Panda&title=Plumbagin+inhibits+cytokinesis+in+Bacillus+subtilis+by+inhibiting+FtsZ+assembly%2D%2Da+mechanistic+study+of+its+antibacterial+activity&doi=10.1111%2Ffebs.12429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Plumbagin inhibits cytokinesis in Bacillus subtilis by inhibiting FtsZ assembly - a mechanistic study of its antibacterial activity</span></div><div class="casAuthors">Bhattacharya, Anusri; Jindal, Bhavya; Singh, Parminder; Datta, Anindya; Panda, Dulal</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4585-4599</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The assembly of FtsZ plays a central role in construction of the cytokinetic Z-ring that orchestrates bacterial cell division.  A naturally occurring naphthoquinone, plumbagin, is known to exhibit antibacterial properties against several types of bacteria.  In this study, plumbagin was found to perturb formation of the Z-ring in Bacillus subtilis 168 cells and to cause elongation of these cells without an apparent effect on nucleoid segregation, indicating that it may inhibit FtsZ assembly.  Furthermore, it bound to purified B. subtilis FtsZ (BsFtsZ) with a dissocn. const. of 20.7 ± 5.6 μm, and inhibited the assembly and GTPase activity of BsFtsZin vitro.  Interestingly, plumbagin did not inhibit either the assembly or GTPase activity of Escherichia coliFtsZ (EcFtsZ) in vitro.  Using docking anal., a putative plumbagin-binding site on BsFtsZ was identified, and the anal. indicated that hydrophobic interactions and hydrogen bonds predominate.  Based on the in silico anal., two variants of BsFtsZ, namely D199A and V307R, were constructed to explore the binding interaction of plumbagin and BsFtsZ.  The effects of plumbagin on the assembly and GTPase activity of the variant BsFtsZ proteins in vitro indicated that the residues D199 and V307 may be involved in the binding of plumbagin to BsFtsZ.  The results suggest that plumbagin inhibits bacterial proliferation by inhibiting the assembly of FtsZ, and provide insight into the binding site of plumbagin on BsFtsZ, which may help in the design of potent FtsZ-targeted antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKE9tC5NyLNLVg90H21EOLACvtfcHk0ljhJ5Q81cD85Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGmtL7F&md5=634b9981160910c671c8e8a637036371</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12429%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharya%26aufirst%3DA.%26aulast%3DJindal%26aufirst%3DB.%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DDatta%26aufirst%3DA.%26aulast%3DPanda%26aufirst%3DD.%26atitle%3DPlumbagin%2520inhibits%2520cytokinesis%2520in%2520Bacillus%2520subtilis%2520by%2520inhibiting%2520FtsZ%2520assembly--a%2520mechanistic%2520study%2520of%2520its%2520antibacterial%2520activity%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D4585%26epage%3D4599%26doi%3D10.1111%2Ffebs.12429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. Y.</span></span> <span> </span><span class="NLM_article-title">New application of tiplaxtinin as an effective FtsZ-targeting chemotype for an antimicrobial study</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1909</span>– <span class="NLM_lpage">1913</span>, <span class="refDoi"> DOI: 10.1039/C7MD00387K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1039%2FC7MD00387K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=30108711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeku7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1909-1913&author=N.+Sunauthor=Y.+Y.+Zhengauthor=R.+L.+Duauthor=S.+Y.+Caiauthor=K.+Zhangauthor=L.+Y.+Soauthor=K.+C.+Cheungauthor=C.+Zhuoauthor=Y.+J.+Luauthor=K.+Y.+Wong&title=New+application+of+tiplaxtinin+as+an+effective+FtsZ-targeting+chemotype+for+an+antimicrobial+study&doi=10.1039%2FC7MD00387K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">New application of tiplaxtinin as an effective FtsZ-targeting chemotype for an antimicrobial study</span></div><div class="casAuthors">Sun, Ning; Zheng, Yuan-Yuan; Du, Ruo-Lan; Cai, Sen-Yuan; Zhang, Kun; So, Lok-Yan; Cheung, Kwan-Choi; Zhuo, Chao; Lu, Yu-Jing; Wong, Kwok-Yin</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1909-1913</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The filamenting temp.-sensitive mutant Z (FtsZ) protein is generally recognized as a promising antimicrobial drug target.  In the present study, a small org. mol. (tiplaxtinin) was identified for the first time as an excellent cell division inhibitor by using a cell-based screening approach from a library with 250 compds.  Tiplaxtinin possesses potent antibacterial activity against Gram-pos. pathogens.  Both in vitro and in vivo results reveal that the compd. is able to disrupt dynamic assembly of FtsZ and Z-ring formation effectively through the mechanism of stimulating FtsZ polymn. and impairing GTPase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvq-t42EDbMrVg90H21EOLACvtfcHk0ljhJ5Q81cD85Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeku7fI&md5=6114b7a52e448e75f5acf7caba5a1313</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC7MD00387K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00387K%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DY.%2BY.%26aulast%3DDu%26aufirst%3DR.%2BL.%26aulast%3DCai%26aufirst%3DS.%2BY.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DSo%26aufirst%3DL.%2BY.%26aulast%3DCheung%26aufirst%3DK.%2BC.%26aulast%3DZhuo%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DY.%2BJ.%26aulast%3DWong%26aufirst%3DK.%2BY.%26atitle%3DNew%2520application%2520of%2520tiplaxtinin%2520as%2520an%2520effective%2520FtsZ-targeting%2520chemotype%2520for%2520an%2520antimicrobial%2520study%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1909%26epage%3D1913%26doi%3D10.1039%2FC7MD00387K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Straniero, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suigo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casiraghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián-Pérez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrast, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanotto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zdovc, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Giuli
Morghen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radaelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valoti, E.</span></span> <span> </span><span class="NLM_article-title">Benzamide derivatives targeting the cell division protein FtsZ: Modifications of the linker and the benzodioxane scaffold and their effects on antimicrobial activity</span>. <i>Antibiotics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">160</span> <span class="refDoi"> DOI: 10.3390/antibiotics9040160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3390%2Fantibiotics9040160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitF2htrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=V.+Stranieroauthor=L.+Suigoauthor=A.+Casiraghiauthor=V.+Sebasti%C3%A1n-P%C3%A9rezauthor=M.+Hrastauthor=C.+Zanottoauthor=I.+Zdovcauthor=C.+De+Giuli%0AMorghenauthor=A.+Radaelliauthor=E.+Valoti&title=Benzamide+derivatives+targeting+the+cell+division+protein+FtsZ%3A+Modifications+of+the+linker+and+the+benzodioxane+scaffold+and+their+effects+on+antimicrobial+activity&doi=10.3390%2Fantibiotics9040160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Benzamide derivatives targeting the cell division protein FtsZ: modifications of the linker and the benzodioxane scaffold and their effects on antimicrobial activity</span></div><div class="casAuthors">Straniero, Valentina; Suigo, Lorenzo; Casiraghi, Andrea; Sebastian-Perez, Victor; Hrast, Martina; Zanotto, Carlo; Zdovc, Irena; Morghen, Carlo De Giuli; Radaelli, Antonia; Valoti, Ermanno</div><div class="citationInfo"><span class="NLM_cas:title">Antibiotics (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">160</span>CODEN:
                <span class="NLM_cas:coden">ABSNC4</span>;
        ISSN:<span class="NLM_cas:issn">2079-6382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Filamentous temp.-sensitive Z (FtsZ) is a prokaryotic protein with an essential role in the bacterial cell division process.  It is widely conserved and expressed in both Gram-pos. and Gram-neg. strains.  In the last decade, several research groups have pointed out mols. able to target FtsZ in Staphylococcus aureus, Bacillus subtilis and other Gram-pos. strains, with sub-micromolar Min. Inhibitory Concns. (MICs).  Conversely, no promising derivs. active on Gram-negatives have been found up to now.  Here, we report our results on a class of benzamide compds., which showed comparable inhibitory activities on both S. aureus and Escherichia coli FtsZ, even though they proved to be substrates of E. coli efflux pump AcrAB, thus affecting the antimicrobial activity.  These surprising results confirmed how a single mol. can target both species while maintaining potent antimicrobial activity.  A further computational study helped us decipher the structural features necessary for broad spectrum activity and assess the drug-like profile and the on-target activity of this family of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjNbAHfAFWqrVg90H21EOLACvtfcHk0lgxs9G4A8qHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitF2htrfE&md5=bb4172456078311ffc4b0e29789408e0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3390%2Fantibiotics9040160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantibiotics9040160%26sid%3Dliteratum%253Aachs%26aulast%3DStraniero%26aufirst%3DV.%26aulast%3DSuigo%26aufirst%3DL.%26aulast%3DCasiraghi%26aufirst%3DA.%26aulast%3DSebasti%25C3%25A1n-P%25C3%25A9rez%26aufirst%3DV.%26aulast%3DHrast%26aufirst%3DM.%26aulast%3DZanotto%26aufirst%3DC.%26aulast%3DZdovc%26aufirst%3DI.%26aulast%3DDe%2BGiuli%2BMorghen%26aufirst%3DC.%26aulast%3DRadaelli%26aufirst%3DA.%26aulast%3DValoti%26aufirst%3DE.%26atitle%3DBenzamide%2520derivatives%2520targeting%2520the%2520cell%2520division%2520protein%2520FtsZ%253A%2520Modifications%2520of%2520the%2520linker%2520and%2520the%2520benzodioxane%2520scaffold%2520and%2520their%2520effects%2520on%2520antimicrobial%2520activity%26jtitle%3DAntibiotics%26date%3D2020%26volume%3D9%26doi%3D10.3390%2Fantibiotics9040160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czaplewski, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errington, J.</span></span> <span> </span><span class="NLM_article-title">Multiple effects of benzamide antibiotics on FtsZ function</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2011.07559.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Fj.1365-2958.2011.07559.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=21276094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlt1yitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=68-84&author=D.+W.+Adamsauthor=L.+J.+Wuauthor=L.+G.+Czaplewskiauthor=J.+Errington&title=Multiple+effects+of+benzamide+antibiotics+on+FtsZ+function&doi=10.1111%2Fj.1365-2958.2011.07559.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple effects of benzamide antibiotics on FtsZ function</span></div><div class="casAuthors">Adams, David W.; Wu, Ling Juan; Czaplewski, Lloyd G.; Errington, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-84</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cell division in almost all bacteria is orchestrated by the essential tubulin homolog FtsZ, which assembles into a ring-like structure and acts as a scaffold for the division machinery.  Division was recently validated as an important target for antibiotics by the demonstration that low-mol.-wt. inhibitors of FtsZ, called benzamides, can cure mice infected with Staphylococcus aureus.  In treated cells of Bacillus subtilis, the authors show that FtsZ assembles into foci throughout the cell, including abnormal locations at the cell poles and over the nucleoid.  These foci are not inactive aggregates because they remain dynamic, turning over almost as rapidly as untreated polymers.  Remarkably, although division is completely blocked, the foci efficiently recruit division proteins that normally co-assemble with FtsZ.  However, they show no affinity for components of the Min or Nucleoid occlusion systems.  In vitro, the benzamides strongly promote the polymn. of FtsZ, into hyperstable polymers, which are highly curved.  Importantly, even at low concns., benzamides transform the structure of the Z ring, resulting in abnormal helical cell division events.  It is proposed that benzamides act principally by promoting an FtsZ protomer conformation that is incompatible with a higher-order level of assembly needed to make a division ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor5Ah875KKmrVg90H21EOLACvtfcHk0lgxs9G4A8qHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlt1yitr8%253D&md5=300166da23db8d3762446411e7954e24</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2011.07559.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2011.07559.x%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DD.%2BW.%26aulast%3DWu%26aufirst%3DL.%2BJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26aulast%3DErrington%26aufirst%3DJ.%26atitle%3DMultiple%2520effects%2520of%2520benzamide%2520antibiotics%2520on%2520FtsZ%2520function%26jtitle%3DMol.%2520Microbiol.%26date%3D2011%26volume%3D80%26spage%3D68%26epage%3D84%26doi%3D10.1111%2Fj.1365-2958.2011.07559.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel FtsZ-targeted 3-arylalkoxy-2,6-difluorobenzamides as potential antimicrobial agents</span>. <i>Chem. Biol. Drug. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1111%2Fcbdd.12658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=26348110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2016&pages=257-264&author=S.+Qiangauthor=C.+Wangauthor=H.+Venterauthor=X.+Liauthor=Y.+Wangauthor=L.+Guoauthor=R.+Maauthor=S.+Ma&title=Synthesis+and+biological+evaluation+of+novel+FtsZ-targeted+3-arylalkoxy-2%2C6-difluorobenzamides+as+potential+antimicrobial+agents&doi=10.1111%2Fcbdd.12658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Evaluation of Novel FtsZ-targeted 3-arylalkoxy-2,6-difluorobenzamides as Potential Antimicrobial Agents</span></div><div class="casAuthors">Qiang, Shengsheng; Wang, Changde; Venter, Henrietta; Li, Xin; Wang, Yi; Guo, Liwei; Ma, Ruixin; Ma, Shutao</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-264</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Novel series of 3-O-arylalkylbenzamides, e.g., I and 3-O-arylalkyl-2,6-difluorobenzamide derivs. II [R1 = H, 3-Cl, 3-Me, etc.] were synthesized from 3-hydroxybenzoic acid and 2,4-difluorophenol resp.  The synthesized derivs. were evaluated for their cell division inhibitory activity and antibacterial activity, results showed that the 3-O-arylalkyl-2,6-difluorobenzamide derivs. possessed much better cell division inhibitory activity and antibacterial activity than the 3-O-arylalkylbenzamide derivs.  Among them, 3-O-chlorobenzyl deriv. II [R3 = 3-Cl] was the most effective in antibacterial activity (0.5, 4, and 8 μg/mL) against Bacillus subtilis ATCC9372, methicillin-resistant Staphylococcus aureus ATCC29213 and penicillin-resistant Staphylococcus aureus PR, while 3-O-methylbenzyl deriv. II [R3 = 3-Me] only exhibited the most potent activity (2 μg/mL) against Staphylococcus aureus ATCC25923.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZDjHjJapcGLVg90H21EOLACvtfcHk0lgxs9G4A8qHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsLvK&md5=7bc3d5b669f862794b014c38ebce2dee</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12658%26sid%3Dliteratum%253Aachs%26aulast%3DQiang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DVenter%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520FtsZ-targeted%25203-arylalkoxy-2%252C6-difluorobenzamides%2520as%2520potential%2520antimicrobial%2520agents%26jtitle%3DChem.%2520Biol.%2520Drug.%2520Des.%26date%3D2016%26volume%3D87%26spage%3D257%26epage%3D264%26doi%3D10.1111%2Fcbdd.12658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chai, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittall, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogunniyi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semple, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venter, H.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial action and reversal of resistance in MRSA by difluorobenzamide derivatives targeted FtsZ</span>. <i>Antibiotics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">873</span> <span class="refDoi"> DOI: 10.3390/antibiotics9120873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.3390%2Fantibiotics9120873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3MXksF2msrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=W.+C.+Chaiauthor=J.+J.+Whittallauthor=D.+Songauthor=S.+W.+Polyakauthor=A.+D.+Ogunniyiauthor=Y.+Wangauthor=F.+Biauthor=S.+Maauthor=S.+J.+Sempleauthor=H.+Venter&title=Antimicrobial+action+and+reversal+of+resistance+in+MRSA+by+difluorobenzamide+derivatives+targeted+FtsZ&doi=10.3390%2Fantibiotics9120873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial action and reversal of resistance in MRSA by difluorobenzamide derivatives targeted at FtsZ</span></div><div class="casAuthors">Chai, Wern Chern; Whittall, Jonathan J.; Song, Di; Polyak, Steven W.; Ogunniyi, Abiodun D.; Wang, Yinhu; Bi, Fangchao; Ma, Shutao; Semple, Susan J.; Venter, Henrietta</div><div class="citationInfo"><span class="NLM_cas:title">Antibiotics (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">873</span>CODEN:
                <span class="NLM_cas:coden">ABSNC4</span>;
        ISSN:<span class="NLM_cas:issn">2079-6382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The bacterial cell division protein, FtsZ, has been identified as a target for antimicrobial development.  Derivs. of 3-methoxybenzamide have shown promising activities as FtsZ inhibitors in Gram-pos. bacteria.  We sought to characterize the activity of five difluorobenzamide derivs. with non-heterocyclic substituents attached through the 3-oxygen.  These compds. exhibited antimicrobial activity against methicillin resistant Staphylococcus aureus (MRSA), with an isopentyloxy-substituted compd. showing modest activity against vancomycin resistant Enterococcus faecium (VRE).  The compds. were able to reverse resistance to oxacillin in highly resistant clin. MRSA strains at concns. far below their MICs.  Three of the compds. inhibited an Escherichia coli strain lacking the AcrAB components of a drug efflux pump, which suggests the lack of Gram-neg. activity can partly be attributed to efflux.  The compds. inhibited cell division by targeting S. aureus FtsZ, producing a dose-dependent increase in GTPase rate which increased the rate of FtsZ polymn. and stabilized the FtsZ polymers.  These compds. did not affect the polymn. of mammalian tubulin and did not display haemolytic activity or cytotoxicity.  These derivs. are therefore promising compds. for further development as antimicrobial agents or as resistance breakers to re-sensitive MRSA to beta-lactam antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtyDYtpkKW1rVg90H21EOLACvtfcHk0lgW4-s2SMmybg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXksF2msrs%253D&md5=23f78c3ab1957509c587879f4fda85cf</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.3390%2Fantibiotics9120873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantibiotics9120873%26sid%3Dliteratum%253Aachs%26aulast%3DChai%26aufirst%3DW.%2BC.%26aulast%3DWhittall%26aufirst%3DJ.%2BJ.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DPolyak%26aufirst%3DS.%2BW.%26aulast%3DOgunniyi%26aufirst%3DA.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DSemple%26aufirst%3DS.%2BJ.%26aulast%3DVenter%26aufirst%3DH.%26atitle%3DAntimicrobial%2520action%2520and%2520reversal%2520of%2520resistance%2520in%2520MRSA%2520by%2520difluorobenzamide%2520derivatives%2520targeted%2520FtsZ%26jtitle%3DAntibiotics%26date%3D2020%26volume%3D9%26doi%3D10.3390%2Fantibiotics9120873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizohata, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoie, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, H.</span></span> <span> </span><span class="NLM_article-title">Structural flexibility of an inhibitor overcomes drug resistance mutations in Staphylococcus aureus FtsZ</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1947</span>– <span class="NLM_lpage">1955</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00323</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00323" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1947-1955&author=J.+Fujitaauthor=Y.+Maedaauthor=E.+Mizohataauthor=T.+Inoueauthor=M.+Kaulauthor=A.+K.+Parhiauthor=E.+J.+LaVoieauthor=D.+S.+Pilchauthor=H.+Matsumura&title=Structural+flexibility+of+an+inhibitor+overcomes+drug+resistance+mutations+in+Staphylococcus+aureus+FtsZ&doi=10.1021%2Facschembio.7b00323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Flexibility of an Inhibitor Overcomes Drug Resistance Mutations in Staphylococcus aureus FtsZ</span></div><div class="casAuthors">Fujita, Junso; Maeda, Yoko; Mizohata, Eiichi; Inoue, Tsuyoshi; Kaul, Malvika; Parhi, Ajit K.; LaVoie, Edmond J.; Pilch, Daniel S.; Matsumura, Hiroyoshi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1947-1955</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the effort to combat antibiotic resistance, inhibitors of the essential bacterial protein FtsZ have emerged as a promising new class of compds. with clin. potential.  One such FtsZ inhibitor (TXA707) is assocd. with potent activity against clin. isolates of methicillin-resistant Staphylococcus aureus (MRSA) that are resistant to current std.-of-care antibiotics.  However, mutations in S. aureus FtsZ (SaFtsZ) that confer resistance to TXA707 have been obsd., with mutations in the Gly196 and Gly193 residues being among the most prevalent.  Here, we describe structural studies of an FtsZ inhibitor, TXA6101, which retains activity against MRSA that express either G196S or G193D mutant FtsZ.  We present the crystal structures of TXA6101 in complex with both wildtype SaFtsZ and G196S mutant SaFtsZ, as well the crystal structure of TXA707 in complex with wildtype SaFtsZ.  Comparison of the three structures reveals a mol. basis for the differential targeting abilities of TXA6101 and TXA707.  The greater structural flexibility of TXA6101 relative to TXA707 enables TXA6101 to avoid steric clash with Ser196 and Asp193.  Our structures also demonstrate that the binding of TXA6101 induces previously unobserved conformational rearrangements of SaFtsZ residues in the binding pocket.  In the aggregate, the structures reported in this work reveal key factors for overcoming drug resistance mutations in SaFtsZ, and offer a structural basis for the design of FtsZ inhibitors with enhanced antibacterial potency and a reduced susceptibility to mutational resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCauV7BfLWLbVg90H21EOLACvtfcHk0lgW4-s2SMmybg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyks70%253D&md5=f1b5edb58bc3dbb3cc98e019075dafe3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00323%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DJ.%26aulast%3DMaeda%26aufirst%3DY.%26aulast%3DMizohata%26aufirst%3DE.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DKaul%26aufirst%3DM.%26aulast%3DParhi%26aufirst%3DA.%2BK.%26aulast%3DLaVoie%26aufirst%3DE.%2BJ.%26aulast%3DPilch%26aufirst%3DD.%2BS.%26aulast%3DMatsumura%26aufirst%3DH.%26atitle%3DStructural%2520flexibility%2520of%2520an%2520inhibitor%2520overcomes%2520drug%2520resistance%2520mutations%2520in%2520Staphylococcus%2520aureus%2520FtsZ%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1947%26epage%3D1955%26doi%3D10.1021%2Facschembio.7b00323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer-González, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Tameemi, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoie, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilch, D. S.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of a fluorescent probe for the visualization of FtsZ in clinically important gram-positive and gram-negative bacterial pathogens</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">20092</span> <span class="refDoi"> DOI: 10.1038/s41598-019-56557-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1038%2Fs41598-019-56557-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=31882782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtFehtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&author=E.+Ferrer-Gonz%C3%A1lezauthor=J.+Fujitaauthor=T.+Yoshizawaauthor=J.+M.+Nelsonauthor=A.+J.+Pilchauthor=E.+Hillmanauthor=M.+Ozawaauthor=N.+Kurodaauthor=H.+M.+Al-Tameemiauthor=J.+M.+Boydauthor=E.+J.+LaVoieauthor=H.+Matsumuraauthor=D.+S.+Pilch&title=Structure-guided+design+of+a+fluorescent+probe+for+the+visualization+of+FtsZ+in+clinically+important+gram-positive+and+gram-negative+bacterial+pathogens&doi=10.1038%2Fs41598-019-56557-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of a Fluorescent Probe for the Visualization of FtsZ in Clinically Important Gram-Positive and Gram-Negative Bacterial Pathogens</span></div><div class="casAuthors">Ferrer-Gonzalez, Edgar; Fujita, Junso; Yoshizawa, Takuya; Nelson, Julia M.; Pilch, Alyssa J.; Hillman, Elani; Ozawa, Mayuki; Kuroda, Natsuko; Al-Tameemi, Hassan M.; Boyd, Jeffrey M.; La Voie, Edmond J.; Matsumura, Hiroyoshi; Pilch, Daniel S.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20092</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Addressing the growing problem of antibiotic resistance requires the development of new drugs with novel antibacterial targets.  FtsZ has been identified as an appealing new target for antibacterial agents.  Here, we describe the structure-guided design of a new fluorescent probe (BOFP) in which a BODIPY fluorophore has been conjugated to an oxazole-benzamide FtsZ inhibitor.  Crystallog. studies have enabled us to identify the optimal position for tethering the fluorophore that facilitates the high-affinity FtsZ binding of BOFP.  Fluorescence anisotropy studies demonstrate that BOFP binds the FtsZ proteins from the Gram-pos. pathogens Staphylococcus aureus, Enterococcus fecalis, Enterococcus faecium, Streptococcus pyogenes, Streptococcus agalactiae, and Streptococcus pneumoniae with Kd values of 0.6-4.6μM.  Significantly, BOFP binds the FtsZ proteins from the Gram-neg. pathogens Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa, and Acinetobacter baumannii with an even higher affinity (Kd = 0.2-0.8μM).  Fluorescence microscopy studies reveal that BOFP can effectively label FtsZ in all the above Gram-pos. and Gram-neg. pathogens.  In addn., BOFP is effective at monitoring the impact of non-fluorescent inhibitors on FtsZ localization in these target pathogens.  Viewed as a whole, our results highlight the utility of BOFP as a powerful tool for identifying new broad-spectrum FtsZ inhibitors and understanding their mechanisms of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbcIjEm4NzxLVg90H21EOLACvtfcHk0liwgZBSE4gIwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtFehtw%253D%253D&md5=30d5745a77228c389567e02ce7fbc5d7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-56557-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-56557-x%26sid%3Dliteratum%253Aachs%26aulast%3DFerrer-Gonz%25C3%25A1lez%26aufirst%3DE.%26aulast%3DFujita%26aufirst%3DJ.%26aulast%3DYoshizawa%26aufirst%3DT.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DPilch%26aufirst%3DA.%2BJ.%26aulast%3DHillman%26aufirst%3DE.%26aulast%3DOzawa%26aufirst%3DM.%26aulast%3DKuroda%26aufirst%3DN.%26aulast%3DAl-Tameemi%26aufirst%3DH.%2BM.%26aulast%3DBoyd%26aufirst%3DJ.%2BM.%26aulast%3DLaVoie%26aufirst%3DE.%2BJ.%26aulast%3DMatsumura%26aufirst%3DH.%26aulast%3DPilch%26aufirst%3DD.%2BS.%26atitle%3DStructure-guided%2520design%2520of%2520a%2520fluorescent%2520probe%2520for%2520the%2520visualization%2520of%2520FtsZ%2520in%2520clinically%2520important%2520gram-positive%2520and%2520gram-negative%2520bacterial%2520pathogens%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26doi%3D10.1038%2Fs41598-019-56557-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yebenes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacios, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menendez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">The interactions of cell division protein FtsZ with guanine nucleotides</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">37515</span>– <span class="NLM_lpage">37528</span>, <span class="refDoi"> DOI: 10.1074/jbc.M706399200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1074%2Fjbc.M706399200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=17977836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGjtrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=37515-37528&author=S.+Huecasauthor=C.+Schaffner-Barberoauthor=W.+Garciaauthor=H.+Yebenesauthor=J.+M.+Palaciosauthor=J.+F.+Diazauthor=M.+Menendezauthor=J.+M.+Andreu&title=The+interactions+of+cell+division+protein+FtsZ+with+guanine+nucleotides&doi=10.1074%2Fjbc.M706399200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Interactions of Cell Division Protein FtsZ with Guanine Nucleotides</span></div><div class="casAuthors">Huecas, Sonia; Schaffner-Barbero, Claudia; Garcia, Wanius; Yebenes, Hugo; Palacios, Juan Manuel; Diaz, Jose Fernando; Menendez, Margarita; Andreu, Jose Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">37515-37528</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Prokaryotic cell division protein FtsZ, an assembling GTPase, directs the formation of the septosome between daughter cells.  FtsZ is an attractive target for the development of new antibiotics.  Assembly dynamics of FtsZ is regulated by the binding, hydrolysis, and exchange of GTP.  We have detd. the energetics of nucleotide binding to model apoFtsZ from Methanococcus jannaschii and studied the kinetics of 2'/3'-O-(N-methylanthraniloyl) (mant)-nucleotide binding and dissocn. from FtsZ polymers, employing calorimetric, fluorescence, and stopped-flow methods.  FtsZ binds GTP and GDP with Kb values ranging from 20 to 300 μM-1 under various conditions.  GTP·Mg2+ and GDP·Mg2+ bind with slightly reduced affinity.  Bound GTP and the coordinated Mg2+ ion play a minor structural role in FtsZ monomers, but Mg2+-assisted GTP hydrolysis triggers polymer disassembly.  Mant-GTP binds and dissocs. quickly from FtsZ monomers, with ∼10-fold lower affinity than GTP.  Mant-GTP displacement measured by fluorescence anisotropy provides a method to test the binding of any competing mols. to the FtsZ nucleotide site.  Mant-GTP is very slowly hydrolyzed and remains exchangeable in FtsZ polymers, but it becomes kinetically stabilized, with a 30-fold slower k+ and ∼500-fold slower k- than in monomers.  The mant-GTP dissocn. rate from FtsZ polymers is comparable with the GTP hydrolysis turnover and with the reported subunit turnover in Escherichia coli FtsZ polymers.  Although FtsZ polymers can exchange nucleotide, unlike its eukaryotic structural homolog tubulin, GDP dissocn. may be slow enough for polymer disassembly to take place first, resulting in FtsZ polymers cycling with GTP hydrolysis similarly to microtubules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUDpFp5TCovLVg90H21EOLACvtfcHk0liwgZBSE4gIwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGjtrjI&md5=05ab248863dbfc30d2a39c9c21180c27</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M706399200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M706399200%26sid%3Dliteratum%253Aachs%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DGarcia%26aufirst%3DW.%26aulast%3DYebenes%26aufirst%3DH.%26aulast%3DPalacios%26aufirst%3DJ.%2BM.%26aulast%3DDiaz%26aufirst%3DJ.%2BF.%26aulast%3DMenendez%26aufirst%3DM.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520interactions%2520of%2520cell%2520division%2520protein%2520FtsZ%2520with%2520guanine%2520nucleotides%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D37515%26epage%3D37528%26doi%3D10.1074%2Fjbc.M706399200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaffner-Barbero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Redondo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lappchen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Blaauwen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morreale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span> <span> </span><span class="NLM_article-title">Insights into nucleotide recognition by cell division protein FtsZ from a mant-GTP competition assay and molecular dynamics</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">10458</span>– <span class="NLM_lpage">10472</span>, <span class="refDoi"> DOI: 10.1021/bi101577p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi101577p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVaqsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=10458-10472&author=C.+Schaffner-Barberoauthor=R.+Gil-Redondoauthor=L.+B.+Ruiz-Avilaauthor=S.+Huecasauthor=T.+Lappchenauthor=T.+den+Blaauwenauthor=J.+F.+Diazauthor=A.+Morrealeauthor=J.+M.+Andreu&title=Insights+into+nucleotide+recognition+by+cell+division+protein+FtsZ+from+a+mant-GTP+competition+assay+and+molecular+dynamics&doi=10.1021%2Fbi101577p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into Nucleotide Recognition by Cell Division Protein FtsZ from a mant-GTP: Competition Assay and Molecular Dynamics</span></div><div class="casAuthors">Schaffner-Barbero, Claudia; Gil-Redondo, Ruben; Ruiz-Avila, Laura B.; Huecas, Sonia; Lappchen, Tilman; den Blaauwen, Tanneke; Diaz, J. Fernando; Morreale, Antonio; Andreu, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">10458-10472</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Essential cell division protein FtsZ forms the bacterial cytokinetic ring and is a target for new antibiotics.  FtsZ monomers bind GTP and assemble into filaments.  Hydrolysis to GDP at the assocn. interface between monomers leads to filament disassembly.  We have developed a homogeneous competition assay, employing the fluorescence anisotropy change of mant-GTP upon binding to nucleotide-free FtsZ, which detects compds. binding to the nucleotide site in FtsZ monomers and measures their affinities within the millimolar to 10 nM range.  We have employed this method to det. the apparent contributions of the guanine, ribose, and the α-, β-, and γ-phosphates to the free energy change of nucleotide binding.  Similar relative contributions have also been estd. through mol. dynamics and binding free energy calcns., employing the crystal structures of FtsZ-nucleotide complexes.  We find an energetically dominant contribution of the β-phosphate, comparable to the whole guanosine moiety.  GTP and GDP bind with similar obsd. affinity to FtsZ monomers.  Loss of the regulatory γ-phosphate results in a predicted accommodation of GDP which has not been obsd. in the crystal structures.  The binding affinities of a series of C8-substituted GTP analogs, known to inhibit FtsZ but not eukaryotic tubulin assembly, correlate with their inhibitory capacity on FtsZ polymn.  Our methods permit testing of FtsZ inhibitors targeting its nucleotide site, as well as compds. from virtual screening of large synthetic libraries.  Our results give insight into the FtsZ-nucleotide interactions, which could be useful in the rational design of new inhibitors, esp. GTP phosphate mimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0eGYV6QbGTrVg90H21EOLACvtfcHk0liwgZBSE4gIwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVaqsL7M&md5=109e4d4a0a2365c5dc3db9d5642963d6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fbi101577p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi101577p%26sid%3Dliteratum%253Aachs%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DGil-Redondo%26aufirst%3DR.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DLappchen%26aufirst%3DT.%26aulast%3Dden%2BBlaauwen%26aufirst%3DT.%26aulast%3DDiaz%26aufirst%3DJ.%2BF.%26aulast%3DMorreale%26aufirst%3DA.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26atitle%3DInsights%2520into%2520nucleotide%2520recognition%2520by%2520cell%2520division%2520protein%2520FtsZ%2520from%2520a%2520mant-GTP%2520competition%2520assay%2520and%2520molecular%2520dynamics%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D10458%26epage%3D10472%26doi%3D10.1021%2Fbi101577p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Avila, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergonos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huecas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araujo-Bazan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Villa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turrado, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Aportela, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nodwell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barasoain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Rodriguez, M. L.</span></span> <span> </span><span class="NLM_article-title">Effective GTP-replacing FtsZ inhibitors and antibacterial mechanism of action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">843</span>, <span class="refDoi"> DOI: 10.1021/cb500974d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500974d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVClsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=834-843&author=M.+Artolaauthor=L.+B.+Ruiz-Avilaauthor=A.+Vergonosauthor=S.+Huecasauthor=L.+Araujo-Bazanauthor=M.+Martin-Fontechaauthor=H.+Vazquez-Villaauthor=C.+Turradoauthor=E.+Ramirez-Aportelaauthor=A.+Hoeglauthor=M.+Nodwellauthor=I.+Barasoainauthor=P.+Chaconauthor=S.+A.+Sieberauthor=J.+M.+Andreuauthor=M.+L.+Lopez-Rodriguez&title=Effective+GTP-replacing+FtsZ+inhibitors+and+antibacterial+mechanism+of+action&doi=10.1021%2Fcb500974d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Effective GTP-replacing FtsZ inhibitors and antibacterial mechanism of action</span></div><div class="casAuthors">Artola, Marta; Ruiz-Avila, Laura B.; Vergonos, Albert; Huecas, Sonia; Araujo-Bazan, Lidia; Martin-Fontecha, Mar; Vazquez-Villa, Henar; Turrado, Carlos; Ramirez-Aportela, Erney; Hoegl, Annabelle; Nodwell, Matthew; Barasoain, Isabel; Chacon, Pablo; Sieber, Stephan A.; Andreu, Jose M.; Lopez-Rodriguez, Maria L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">834-843</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The essential cell division protein FtsZ is considered an attractive target in the search for antibacterials with novel mechanisms of action to overcome the resistance problem.  FtsZ undergoes GTP-dependent assembly at midcell to form the Z-ring, a dynamic structure that evolves until final constriction of the cell.  Therefore, mols. able to inhibit its activity will eventually disrupt bacterial viability.  The authors report a new series of small mols. able to replace GTP and to specifically inhibit FtsZ, blocking the bacterial division process.  These new synthesized inhibitors interact with the GTP-binding site of FtsZ (Kd = 0.4-0.8 μM), display antibacterial activity against Gram-pos. pathogenic bacteria, and show selectivity against tubulin.  Biphenyl deriv. 28 stands out as a potent FtsZ inhibitor (Kd = 0.5 μM) with high antibacterial activity [MIC (MRSA) = 7 μM].  In depth anal. of the mechanism of action of compds. 22, 28, 33, and 36 has revealed that they act as effective inhibitors of correct FtsZ assembly, blocking bacterial division and thus leading to filamentous undivided cells.  These findings provide a compelling rationale for the development of compds. targeting the GTP-binding site as antibacterial agents and open the door to antibiotics with novel mechanisms of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPmkh6dxtlrbVg90H21EOLACvtfcHk0liyp_VP9oCoXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVClsLrP&md5=95534237838bac7f7591d50521c9afe9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fcb500974d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500974d%26sid%3Dliteratum%253Aachs%26aulast%3DArtola%26aufirst%3DM.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DVergonos%26aufirst%3DA.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAraujo-Bazan%26aufirst%3DL.%26aulast%3DMartin-Fontecha%26aufirst%3DM.%26aulast%3DVazquez-Villa%26aufirst%3DH.%26aulast%3DTurrado%26aufirst%3DC.%26aulast%3DRamirez-Aportela%26aufirst%3DE.%26aulast%3DHoegl%26aufirst%3DA.%26aulast%3DNodwell%26aufirst%3DM.%26aulast%3DBarasoain%26aufirst%3DI.%26aulast%3DChacon%26aufirst%3DP.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26atitle%3DEffective%2520GTP-replacing%2520FtsZ%2520inhibitors%2520and%2520antibacterial%2520mechanism%2520of%2520action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D834%26epage%3D843%26doi%3D10.1021%2Fcb500974d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marín-Ramos, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balabasquer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega-Nogales, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrecillas, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Ordóñez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcos-Ramiro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar-Garrido, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medrano, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajate, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollinedo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallardo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Fontecha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Rodríguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega-Gutiérrez, S.</span></span> <span> </span><span class="NLM_article-title">A potent isoprenylcysteine carboxylmethyltransferase (ICMT) inhibitor improves survival in Ras-driven acute myeloid leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6035</span>– <span class="NLM_lpage">6046</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00145</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00145" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFaqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6035-6046&author=N.+I.+Mar%C3%ADn-Ramosauthor=M.+Balabasquerauthor=F.+J.+Ortega-Nogalesauthor=I.+R.+Torrecillasauthor=A.+Gil-Ord%C3%B3%C3%B1ezauthor=B.+Marcos-Ramiroauthor=P.+Aguilar-Garridoauthor=I.+Cushmanauthor=A.+Romeroauthor=F.+J.+Medranoauthor=C.+Gajateauthor=F.+Mollinedoauthor=M.+R.+Philipsauthor=M.+Campilloauthor=M.+Gallardoauthor=M.+Mart%C3%ADn-Fontechaauthor=M.+L.+L%C3%B3pez-Rodr%C3%ADguezauthor=S.+Ortega-Guti%C3%A9rrez&title=A+potent+isoprenylcysteine+carboxylmethyltransferase+%28ICMT%29+inhibitor+improves+survival+in+Ras-driven+acute+myeloid+leukemia&doi=10.1021%2Facs.jmedchem.9b00145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent Isoprenylcysteine Carboxylmethyltransferase (ICMT) Inhibitor Improves Survival in Ras-Driven Acute Myeloid Leukemia</span></div><div class="casAuthors">Marin-Ramos, Nagore I.; Balabasquer, Moises; Ortega-Nogales, Francisco J.; Torrecillas, Ivan R.; Gil-Ordonez, Ana; Marcos-Ramiro, Beatriz; Aguilar-Garrido, Pedro; Cushman, Ian; Romero, Antonio; Medrano, Francisco J.; Gajate, Consuelo; Mollinedo, Faustino; Philips, Mark R.; Campillo, Mercedes; Gallardo, Miguel; Martin-Fontecha, Mar; Lopez-Rodriguez, Maria L.; Ortega-Gutierrez, Silvia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6035-6046</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Blockade of Ras activity by inhibiting its post-translational methylation catalyzed by isoprenylcysteine carboxylmethyltransferase (ICMT) has been suggested as a promising antitumor strategy.  However, the paucity of inhibitors has precluded the clin. validation of this approach.  In this work we report a potent ICMT inhibitor, compd. 3 [UCM-1336, IC50 = 2 μM], which is selective against the other enzymes involved in the post-translational modifications of Ras.  Compd. 3 significantly impairs the membrane assocn. of the four Ras isoforms, leading to a decrease of Ras activity and to inhibition of Ras downstream signaling pathways.  In addn., it induces cell death in a variety of Ras-mutated tumor cell lines and increases survival in an in vivo model of acute myeloid leukemia.  Because ICMT inhibition impairs the activity of the four Ras isoforms regardless of its activating mutation, compd. 3 surmounts many of the common limitations of available Ras inhibitors described so far.  In addn., these results validate ICMT as a valuable target for the treatment of Ras-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQdAP3zRIM7Vg90H21EOLACvtfcHk0liyp_VP9oCoXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFaqsbzP&md5=99eacad9ac479cc0604a46b66d039e8e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00145%26sid%3Dliteratum%253Aachs%26aulast%3DMar%25C3%25ADn-Ramos%26aufirst%3DN.%2BI.%26aulast%3DBalabasquer%26aufirst%3DM.%26aulast%3DOrtega-Nogales%26aufirst%3DF.%2BJ.%26aulast%3DTorrecillas%26aufirst%3DI.%2BR.%26aulast%3DGil-Ord%25C3%25B3%25C3%25B1ez%26aufirst%3DA.%26aulast%3DMarcos-Ramiro%26aufirst%3DB.%26aulast%3DAguilar-Garrido%26aufirst%3DP.%26aulast%3DCushman%26aufirst%3DI.%26aulast%3DRomero%26aufirst%3DA.%26aulast%3DMedrano%26aufirst%3DF.%2BJ.%26aulast%3DGajate%26aufirst%3DC.%26aulast%3DMollinedo%26aufirst%3DF.%26aulast%3DPhilips%26aufirst%3DM.%2BR.%26aulast%3DCampillo%26aufirst%3DM.%26aulast%3DGallardo%26aufirst%3DM.%26aulast%3DMart%25C3%25ADn-Fontecha%26aufirst%3DM.%26aulast%3DL%25C3%25B3pez-Rodr%25C3%25ADguez%26aufirst%3DM.%2BL.%26aulast%3DOrtega-Guti%25C3%25A9rrez%26aufirst%3DS.%26atitle%3DA%2520potent%2520isoprenylcysteine%2520carboxylmethyltransferase%2520%2528ICMT%2529%2520inhibitor%2520improves%2520survival%2520in%2520Ras-driven%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6035%26epage%3D6046%26doi%3D10.1021%2Facs.jmedchem.9b00145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lh_7aMDhRAz0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh_7aMDhRAz0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh8G-zSQB8xlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BunkoczNi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+BunkoczNiauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lh8G-zSQB8xlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczNi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DXD','PDB','4DXD'); return false;">PDB: 4DXD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VOA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VOA','PDB','3VOA'); return false;">PDB: 3VOA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XDU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XDU','PDB','5XDU'); return false;">PDB: 5XDU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD1','PDB','6YD1'); return false;">PDB: 6YD1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD5','PDB','6YD5'); return false;">PDB: 6YD5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD6','PDB','6YD6'); return false;">PDB: 6YD6</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02207">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81331"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02207?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02207</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Fluorescent benzamide probes synthesized in this work (Figure S1); isotherms of binding of the fluorescent probes <b>1</b>, (<i>S</i>)-<b>2</b>, and (<i>R</i>)-<b>5</b> to FtsZ polymers measured by fluorescence anisotropy (Figure S2); competition isotherms of weak FtsZ inhibitors from the binding screen (Figure S3); effect of high affinity FtsZ inhibitors on bacterial cell division (Figure S4); expanded view of the SaFtsZ binding cleft, with omit electron density maps (green), contoured at 3σ around compounds DFMBA, <b>18</b>, and <b>20</b> (Figure S5); ligand interaction diagram of residues located in the binding site (Figure S6); binding parameters of the fluorescent probes to FtsZ-GMPCPP polymers and activity on bacterial cells (Table S1); cell length of <i>B. subtilis</i> 168 and diameter of <i>S. aureus</i> cells incubated with FtsZ inhibitors (Table S2); cytotoxic effect of FtsZ inhibitors in IMR90 cell line (Table S3); spontaneous mutations of inhibitor-resistant <i>S. aureus</i> strains (Table S4); data collection and refinement statistics (Table S5); structure of FtsZ inhibitors TXA707, TXA709 and TXA6101, and fluorescent probe BOFP (Chart S1); structure of previously identified FtsZ inhibitors screened for binding against the interdomain cleft of the protein in this work (Chart S2); synthetic scheme S1; detailed synthetic procedures and characterization data of intermediates <b>31</b>–<b>37</b>, <b>39</b>–<b>47</b>, and <b>50</b>; final compounds <b>6–17</b>, <b>19</b>, <b>21</b>, <b>22</b>–<b>24</b> and probes <b>SP1</b>–<b>SP5</b>; solubility of all tested compounds (Table S6); and NMR spectra of final compounds and HPLC traces analysis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_001.pdf">jm0c02207_si_001.pdf (4.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02207/suppl_file/jm0c02207_si_002.csv">jm0c02207_si_002.csv (2.85 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD1">6YD1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD5">6YD5</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YD6">6YD6</a> correspond to SaFtsZ in complex with DFMBA, compound <b>18</b>, and compound <b>20</b>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02207&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02207%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02207" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b55e9b983ce2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
