
<html lang="en"     class="pb-page"  data-request-id="6fd77089-f770-47f3-8520-30e5a0bb5be8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00191;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2021.64.issue-14"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Identification of Compounds Targeting HuD. Another Brick in the Wall of Neurodegenerative Disease Treatment" /></meta><meta name="dc.Creator" content="Francesca Alessandra  Ambrosio" /></meta><meta name="dc.Creator" content="Adriana  Coricello" /></meta><meta name="dc.Creator" content="Giosuè  Costa" /></meta><meta name="dc.Creator" content="Antonio  Lupia" /></meta><meta name="dc.Creator" content="Mariachiara  Micaelli" /></meta><meta name="dc.Creator" content="Nicoletta  Marchesi" /></meta><meta name="dc.Creator" content="Federico  Sala" /></meta><meta name="dc.Creator" content="Alessia  Pascale" /></meta><meta name="dc.Creator" content="Daniela  Rossi" /></meta><meta name="dc.Creator" content="Francesca  Vasile" /></meta><meta name="dc.Creator" content="Stefano  Alcaro" /></meta><meta name="dc.Creator" content="Simona  Collina" /></meta><meta name="dc.Description" content="ELAV-like (ELAVL) RNA-binding proteins play a pivotal role in post-transcriptional processes, and their dysregulation is involved in several pathologies. This work was focused on HuD (ELAVL4), whic..." /></meta><meta name="Description" content="ELAV-like (ELAVL) RNA-binding proteins play a pivotal role in post-transcriptional processes, and their dysregulation is involved in several pathologies. This work was focused on HuD (ELAVL4), whic..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 5, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00191" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00191" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00191" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00191" /></link>
        
    
    

<title>Identification of Compounds Targeting HuD. Another Brick in the Wall of Neurodegenerative Disease Treatment | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00191" /></meta><meta property="og:title" content="Identification of Compounds Targeting HuD. Another Brick in the Wall of Neurodegenerative Disease Treatment" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0008.jpeg" /></meta><meta property="og:description" content="ELAV-like (ELAVL) RNA-binding proteins play a pivotal role in post-transcriptional processes, and their dysregulation is involved in several pathologies. This work was focused on HuD (ELAVL4), which is specifically expressed in nervous tissues, and involved in differentiation and synaptic plasticity mechanisms. HuD represents a new, albeit unexplored, candidate target for the treatment of several relevant neurodegenerative diseases. The aim of this pioneering work was the identification of new molecules able to recognize and bind HuD, thus interfering with its activity. We combined virtual screening, molecular dynamics (MD), and STD-NMR techniques. Starting from around 51 000 compounds, four promising hits eventually provided experimental evidence of their ability to bind HuD. Among the selected best hits, folic acid was found to be the most interesting one, being able to well recognize the HuD binding site. Our results provide a basis for the identification of new HuD interfering compounds which may be useful against neurodegenerative syndromes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00191"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00191">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00191&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00191&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00191&amp;href=/doi/10.1021/acs.jmedchem.1c00191" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 9989-10000</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00184" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01249" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of Compounds Targeting HuD. Another Brick in the Wall of Neurodegenerative Disease Treatment</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Francesca Alessandra Ambrosio</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesca Alessandra Ambrosio</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento di Scienze della Salute, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesca+Alessandra++Ambrosio">Francesca Alessandra Ambrosio</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4874-2946" title="Orcid link">https://orcid.org/0000-0003-4874-2946</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adriana Coricello</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adriana Coricello</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento di Scienze della Salute, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adriana++Coricello">Adriana Coricello</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Giosuè Costa</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giosuè Costa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento di Scienze della Salute, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div><div class="loa-info-affiliations-info">Net4Science Academic Spin-Off, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div><div class="loa-info-affiliations-info">Associazione CRISEA—Centro di Ricerca e Servizi Avanzati per l’Innovazione Rurale, Località Condoleo, Belcastro, Catanzaro, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8deaeee2fef9eccdf8e3e4eef7a3e4f9"><span class="__cf_email__" data-cfemail="88efebe7fbfce9c8fde6e1ebf2a6e1fc">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giosu%C3%A8++Costa">Giosuè Costa</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0947-9479" title="Orcid link">https://orcid.org/0000-0003-0947-9479</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antonio Lupia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antonio Lupia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Net4Science Academic Spin-Off, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div><div class="loa-info-affiliations-info">Associazione CRISEA—Centro di Ricerca e Servizi Avanzati per l’Innovazione Rurale, Località Condoleo, Belcastro, Catanzaro, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antonio++Lupia">Antonio Lupia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mariachiara Micaelli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mariachiara Micaelli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CIBIO—Department of Cellular, Computational and Integrative Biology, University of Trento, Via Sommarive 9, Povo, 38123 Trento, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mariachiara++Micaelli">Mariachiara Micaelli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicoletta Marchesi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicoletta Marchesi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicoletta++Marchesi">Nicoletta Marchesi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Federico Sala</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Federico Sala</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Milan, Via Golgi 19, 20133 Milano, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Federico++Sala">Federico Sala</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alessia Pascale</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alessia Pascale</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alessia++Pascale">Alessia Pascale</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniela Rossi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniela Rossi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniela++Rossi">Daniela Rossi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Francesca Vasile</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesca Vasile</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Milan, Via Golgi 19, 20133 Milano, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#680e1a09060b0d1b0b09461e091b01040d281d0601050146011c"><span class="__cf_email__" data-cfemail="315743505f52544252501f475042585d5471445f585c581f5845">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesca++Vasile">Francesca Vasile</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefano Alcaro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefano Alcaro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento di Scienze della Salute, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div><div class="loa-info-affiliations-info">Net4Science Academic Spin-Off, Università “Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</div><div class="loa-info-affiliations-info">Associazione CRISEA—Centro di Ricerca e Servizi Avanzati per l’Innovazione Rurale, Località Condoleo, Belcastro, Catanzaro, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefano++Alcaro">Stefano Alcaro</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Simona Collina</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simona Collina</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f4879d999b9a95da979b98989d9a95b4819a9d8482da9d80"><span class="__cf_email__" data-cfemail="6013090d0f0e014e030f0c0c090e0120150e0910164e0914">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simona++Collina">Simona Collina</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2954-7558" title="Orcid link">https://orcid.org/0000-0002-2954-7558</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00191&amp;href=/doi/10.1021%2Facs.jmedchem.1c00191" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 9989–10000</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 4, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 February 2021</li><li><span class="item_label"><b>Published</b> online</span>5 July 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00191" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00191</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9989%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFrancesca%2BAlessandra%2BAmbrosio%252C%2BAdriana%2BCoricello%252C%2BGiosu%25C3%25A8%2BCosta%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D14%26contentID%3Dacs.jmedchem.1c00191%26title%3DIdentification%2Bof%2BCompounds%2BTargeting%2BHuD.%2BAnother%2BBrick%2Bin%2Bthe%2BWall%2Bof%2BNeurodegenerative%2BDisease%2BTreatment%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10000%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00191"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">698</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00191" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of Compounds Targeting HuD. Another Brick in the Wall of Neurodegenerative Disease Treatment&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Francesca&quot;,&quot;last_name&quot;:&quot;Alessandra Ambrosio&quot;},{&quot;first_name&quot;:&quot;Adriana&quot;,&quot;last_name&quot;:&quot;Coricello&quot;},{&quot;first_name&quot;:&quot;Giosuè&quot;,&quot;last_name&quot;:&quot;Costa&quot;},{&quot;first_name&quot;:&quot;Antonio&quot;,&quot;last_name&quot;:&quot;Lupia&quot;},{&quot;first_name&quot;:&quot;Mariachiara&quot;,&quot;last_name&quot;:&quot;Micaelli&quot;},{&quot;first_name&quot;:&quot;Nicoletta&quot;,&quot;last_name&quot;:&quot;Marchesi&quot;},{&quot;first_name&quot;:&quot;Federico&quot;,&quot;last_name&quot;:&quot;Sala&quot;},{&quot;first_name&quot;:&quot;Alessia&quot;,&quot;last_name&quot;:&quot;Pascale&quot;},{&quot;first_name&quot;:&quot;Daniela&quot;,&quot;last_name&quot;:&quot;Rossi&quot;},{&quot;first_name&quot;:&quot;Francesca&quot;,&quot;last_name&quot;:&quot;Vasile&quot;},{&quot;first_name&quot;:&quot;Stefano&quot;,&quot;last_name&quot;:&quot;Alcaro&quot;},{&quot;first_name&quot;:&quot;Simona&quot;,&quot;last_name&quot;:&quot;Collina&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;9989-10000&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00191&quot;},&quot;abstract&quot;:&quot;ELAV-like (ELAVL) RNA-binding proteins play a pivotal role in post-transcriptional processes, and their dysregulation is involved in several pathologies. This work was focused on HuD (ELAVL4), which is specifically expressed in nervous tissues, and involved in differentiation and synaptic plasticity mechanisms. HuD represents a new, albeit unexplored, candidate target for the treatment of several relevant neurodegenerative diseases. The aim of this pioneering work was the identification of new molecules able to recognize and bind HuD, thus interfering with its activity. We combined virtual screening, molecular dynamics (MD), and STD-NMR techniques. Starting from around 51 000 compounds, four promising hits eventually provided experimental evidence of their ability to bind HuD. Among the selected best hits, folic acid was found to be the most interesting one, being able to well recognize the HuD binding site. Our results provide a basis for the identification of new HuD interfering compounds which may be usefu&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00191&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00191" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00191&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00191" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00191&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00191" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00191&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00191&amp;href=/doi/10.1021/acs.jmedchem.1c00191" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00191" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00191" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00191%26sid%3Dliteratum%253Aachs%26pmid%3D34219450%26genre%3Darticle%26aulast%3DAmbrosio%26date%3D2021%26atitle%3DIdentification%2Bof%2BCompounds%2BTargeting%2BHuD.%2BAnother%2BBrick%2Bin%2Bthe%2BWall%2Bof%2BNeurodegenerative%2BDisease%2BTreatment%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D14%26spage%3D9989%26epage%3D10000%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (7)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292101" title="Nutrition">Nutrition</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">ELAV-like (ELAVL) RNA-binding proteins play a pivotal role in post-transcriptional processes, and their dysregulation is involved in several pathologies. This work was focused on HuD (ELAVL4), which is specifically expressed in nervous tissues, and involved in differentiation and synaptic plasticity mechanisms. HuD represents a new, albeit unexplored, candidate target for the treatment of several relevant neurodegenerative diseases. The aim of this pioneering work was the identification of new molecules able to recognize and bind HuD, thus interfering with its activity. We combined virtual screening, molecular dynamics (MD), and STD-NMR techniques. Starting from around 51 000 compounds, four promising hits eventually provided experimental evidence of their ability to bind HuD. Among the selected best hits, folic acid was found to be the most interesting one, being able to well recognize the HuD binding site. Our results provide a basis for the identification of new HuD interfering compounds which may be useful against neurodegenerative syndromes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05784" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05784" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">RNA-binding proteins (RBPs) play a prominent role in the fate of target messenger RNAs (mRNAs) coding for pivotal proteins in key cellular functions. Among RBPs, the family of mammalian ELAV/Hu RBPs is involved in controlling the functional activities of diverse RNA populations. The ubiquitously expressed HuR (ELAVL1) and the neuron-specific members HuB (ELAVL2), HuC (ELAVL3), and HuD (ELAVL4) are the four known members of the ELAV family.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The ELAV/Hu proteins bind to the (A/U)-rich elements (AREs), which are preferentially localized within the 3′-untranslated regions (UTRs) of selected mRNAs and finely regulate their metabolic pathways. These proteins can affect the post-transcriptional fate of the targeted mRNAs at multiple levels, such as polyadenylation, alternative splicing, and nuclear export to cytoplasmic localization, stability, and translation.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> Among the neuronal RNA-binding proteins, HuD has been intensively studied. It is expressed in nervous tissues where it is mainly involved in neuronal differentiation, in synaptic plasticity mechanisms, both in the central and in the peripheral nervous system, in tissue reparation after nerve injury and in memory processes.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> Moreover, HuD is specifically implicated in the motor neuron function, indeed an intimate relationship between HuD and motor neuron diseases has been demonstrated.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> Altered expression and malfunctioning of the protein have been associated with the pathogenesis of nervous system disorders and childhood brain tumors (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). For example, overexpression of HuD has been detected in patients suffering from schizophrenia, while polymorphisms of the HuD gene have been related to the pathogenesis of Parkinson’s disease, affecting, in particular, the age at onset.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> Conversely, the role played by HuD in the pathogenesis of Alzheimer’s disease (AD) seems to be ambiguous. On one hand, some studies suggested that HuD expression decreases as a function of clinical dementia progression, an event that favors the accumulation of amyloid-β (Aβ) peptides in AD brains.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> In line with this concept, in human SH-SY5Y neuroblastoma cells, it has been demonstrated that HuD gene silencing decreases the expression of ADAM10 protein, the α-secretase that prevents Aβ formation.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> On the other hand, other studies associated overexpression of HuD to AD incidence according to at least two different mechanisms: (1) HuD may contribute to the stabilization of amyloid precursor protein (APP) and β-site APP-cleaving enzyme 1, the secretase that favors Aβ formation (BACE1) and (2) HuD is involved in the upregulation of neuroserpin, an inhibitor of tissue plasminogen activator, hence its upregulation reduces the brain levels of plasmin, one of the main enzymes involved in Aβ clearance.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HuD protein and its involvement in neuronal functions.<a onclick="showRef(event, 'ref11 ref19'); return false;" href="javascript:void(0);" class="ref ref11 ref19">(11,19)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To sum up, targeting HuD could thus turn out to be effective in counteracting neurodegenerative diseases. Nevertheless, this protein still remains poorly investigated from a medicinal chemistry standpoint.</div><div class="NLM_p">From a structural perspective, three RNA recognition motifs (RRMs) are present in all ELAVL proteins. Specifically, RRM1 and RRM2 are essential for mRNA binding and constitute a sort of “clamp” that holds the mRNA strand, while the third RRM interacts with the poly(A) tail and is involved in the protein–protein interactions required for multimerization.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> Sequence-alignment studies of different ELAV proteins showed that each of the first two RRM domains contains two highly conserved sequences of eight and six amino acids, named ribonucleoprotein 1 and 2 (RNP1 and RNP2) motifs, respectively. From a three-dimensional point of view, the binding occurs at the interface of the first two RRM domains, where four of the β-sheets (two for each RRM domain) form the upper and lower “surfaces” of the clamp. Specifically, the four β-sheets contain the highly conserved sequences RNP1 and RNP2.<a onclick="showRef(event, 'ref20 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref22">(20,22)</a></div><div class="NLM_p">In our previous experiences with ELAV proteins, we had evaluated and confirmed the importance of the ELAVL RNP sequences in binding and stabilizing specific target mRNAs using real-time quantitative PCR, saturation transfer difference (STD) NMR, and diffusion-ordered spectroscopy (DOSY).<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> Moreover, we had assessed the potential of computer-aided drug design (CADD) as a useful tool in future drug-discovery programs involving ELAVL proteins.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">The aim of the current work is the identification of new ligands able to bind the HuD binding site and alter the formation of the HuD-mRNA complex, thus taking the first step toward the discovery of novel compounds with potential application in the treatment of neurological disorders.</div><div class="NLM_p">Initially, we carried out virtual screening (VS) studies, using a database of natural compounds and Food and Drug Administration (FDA)-approved drugs. Currently, VS techniques represent a powerful and useful tool for the drug-discovery process and even if the VS approach is not immune to traps and failings, a large number of successful applications have been reported.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30">(27−30)</a> In addition, the drug repurposing of already approved drugs is a good alternative for drug development as it offers various advantages. First of all, repositioned drugs have already obtained approval from the FDA, which means they have successfully passed safety trials and therefore the risk of failure is lower,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and they have thoroughly been investigated for absorption, distribution, metabolism, excretion, and toxicity (ADMET) mechanisms. Moreover, established formulations and manufacturing methods and phase IV (post-marketing surveillance) safety data are generally already available for these drugs. In summary, usually less money investments are needed and consequently, this strategy is economically attractive. Accordingly, in this study, we performed a VS of natural compounds and FDA-approved drugs. Starting from about 51 000 compounds, 10 promising hits were selected.</div><div class="NLM_p last">Afterward, considering the commercial availability and the suitability for NMR analysis, 4 compounds were purchased. To analyze the ligand–HuD interaction and the capability of ligands to bind HuD, saturation transfer difference (STD) NMR investigations were performed.<a onclick="showRef(event, 'ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35">(32−35)</a> For each of the 4 compounds the best docking pose was submitted for molecular dynamics simulations (MDs) to deeply investigate the behavior of the ligands in the HuD binding site. Interestingly, the experimental data showed that the 4 selected hits can bind HuD. Finally, we observed a clear correlation between the molecular modeling outcomes and the observed STD-NMR results.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Target Optimization and Virtual Screening</h3><div class="NLM_p">Molecular docking simulations were performed to elucidate the binding mode and the interactions between the compounds and the HuD binding site. Due to the similarity between the 3D structures of HuD and HuR, in agreement with our previous works, we used the HuD “<i>closed”</i> conformation structure for the molecular recognition studies (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39">(36−39)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. 3D representation of the (A) HuD-mRNA complex, (B) HuD <i>closed</i> conformation structure, and (C) HuD-RNA complex superimposed on the HuD <i>closed</i> conformation structure. The two protein conformations and the mRNA are shown as slate, magenta, and gray cartoon, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The docking results were filtered considering the Lipinski rule of 5<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and all poses of compounds were ranked and analyzed based on the Glide score (G-score) value. All of the compounds with a G-score value within 3 kcal/mol from the best hit were considered, thus10 hits for the further thermodynamic studies were selected (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. 2D structures of the 10 hits identified by means of the virtual screening approach.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. G-Score Values of the 10 Hits Identified by Means of the Virtual Screening Approach</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center" char=".">glide G-Score<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">alloxazine</td><td class="colsep0 rowsep0" align="char" char=".">–7.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aurintricarboxylic acid</td><td class="colsep0 rowsep0" align="char" char=".">–9.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cefazolin<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">–8.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">enalapril<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">–6.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">folic acid<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">–9.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">methoxatin</td><td class="colsep0 rowsep0" align="char" char=".">–9.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">phosphoglycolic acid</td><td class="colsep0 rowsep0" align="char" char=".">–7.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">portulacaxanthin II</td><td class="colsep0 rowsep0" align="char" char=".">–9.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rosmarinic acid<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">–8.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tilundronic acid</td><td class="colsep0 rowsep0" align="char" char=".">–7.75</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">G-Score values are expressed in kcal/mol.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Best selected hits.</p></div></div></div><div class="NLM_p last">Each selected <i>hit</i> was searched within several vendor databases. Finally, based on their commercial availability, 4 best hits were purchased and subjected to the STD-NMR experimental evaluation.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> NMR Analysis</h3><div class="NLM_p">The solubility and stability over time of the compounds reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> dissolved in D<sub>2</sub>O and phosphate buffer were monitored by performing <sup>1</sup>H NMR experiments for 24 h. Four molecules folic acid, rosmarinic acid, cefazolin, and enalapril maleate proved to be sufficiently soluble and stable to perform the interaction studies with the protein in phosphate buffer. Mono-dimensional and bi-dimensional spectra of all compounds were analyzed and used for their characterization. COSY and HSQC were used for the chemical shift assignment, while NOESY spectra were useful for the identification of conformational preferences.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p">The interaction studies between the ligands and the isolated protein were performed using STD-NMR experiments and it was proved that all of the ligands were able to bind the protein. To describe the ligand–protein interaction, we reported the absolute STD values that are strongly indicative for mapping the ligand epitope. Although in the literature the STD technique is also used for the determination of the dissociation constant, the determination of KD was not possible for these molecules due to their relatively low solubility (<20 mM). We performed a detailed analysis of the binding epitope and the STD absolute percentages obtained for each ligand are reported in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. 2D Representation and Absolute STD % of Folic Acid, Rosmarinic Acid, Cefazolin, and Enalapril Maleate in the Presence of HuD</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0007.gif" alt="" id="fx1" /></img><div></div></div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Folic Acid</h4><div class="NLM_p last">The STD spectrum of folic acid highlighted a strong interaction for both the aromatic portions. Interestingly, the absolute STD value observed for the proton of the pteridine ring (9.4%, a surprisingly high value compared with the STD data obtained for the protons of the compounds derived from this screening or from other screened natural molecules<a onclick="showRef(event, 'ref37 ref42'); return false;" href="javascript:void(0);" class="ref ref37 ref42">(37,42)</a>) suggests that this moiety is strongly involved in the binding event. An additional interaction was observed for the second aromatic moiety, as reported in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, indicating that the whole molecule is well positioned within the binding site.</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Rosmarinic Acid</h4><div class="NLM_p last">Rosmarinic acid showed several interactions, as evidenced by the STD signals, corresponding to all of the protons of the molecule. The most intense interaction (0.5 STD %) is attributed to proton 8′ of the α,β-unsaturated moiety, while all of the other protons produce interaction signals with the range of intensity between 0.26 and 0.19 STD % (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The STD data suggest the involvement of both aromatic moieties in the binding event.</div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Cefazolin</h4><div class="NLM_p last">Regarding cefazolin, three interactions are detected in the STD spectrum and they are related to the methyl group of the thiadiazolic ring, to proton 5 belonging to the β-lactam ring, and to proton 8 of the tetrazole (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In particular, proton 8 shows the most intense STD signal (9.5%), thus suggesting that the aromatic tetrazole ring is tightly bound to the protein. On the other hand, proton 5 and proton 1 also show relevant STD signals (1.7 and 0.8%, respectively), hinting that the thiadiazolic and the β-lactam rings may also play a key role in the binding event.</div></div><div id="sec2_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Enalapril Maleate</h4><div class="NLM_p last">From the proton spectrum analysis, carried out in D<sub>2</sub>O and in buffer, we noticed that enalapril is sensitive to the pH of the solution, due to the presence of an acid and an amino group in the structure. The interaction studies allowed the identification of the epitope that is essentially represented by the aromatic, the ethoxy and pyrrolidine moieties (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Ligand–Protein Interaction Analysis</h3><div class="NLM_p">By analyzing the binding mode of the 4 best candidates, we observed that all of the compounds are involved in productive interactions within the HuD binding site.</div><div class="NLM_p">Folic acid is well accommodated in the HuD binding pocket. The dihydropteridine nucleus forms an H-bond with the side chain of Arg155 and two H-bonds with Asn126. The amino group is engaged in an H-bond interaction with the side chain of Thr153.</div><div class="NLM_p">Through its carboxyl groups, folic acid establishes different H-bond and salt-bridge interactions with the guanidine groups of Arg116, Arg123, and Arg172 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. 3D representations of the best pose of (A) folic acid (green sticks), (B) rosmarinic acid (orange sticks), (C) cefazolin (cyan sticks), and (D) enalapril (light-blue sticks) docked into the HuD binding pocket. The ligands are shown in sticks, HuD is shown as a magenta cartoon, and the enzyme residues involved in crucial contacts with the compounds are reported as gray carbon sticks. Hydrogen bonds, salt bridges, aromatic hydrogen bonds, π–π stacking, and π-cation interactions are represented by yellow, green, cyan, violet, and red dashed lines, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Rosmarinic acid interacts through its carboxyl moiety with the guanidine groups of Arg116, Arg123, and Arg172 establishing many H-bonding and different salt-bridge interactions. The 3-(3,4-dihydroxyphenyl) ring forms a π–π stacking interaction with Phe84 and three H-bonds with Arg116. The ester oxygen forms an H-bond with Arg123. The 2-(3,4-dihydroxyphenyl) ring is engaged in an H-bond interaction with Thr153 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div><div class="NLM_p">Cefazolin interacts with Arg123 and Arg172 forming salt bridges and an H-bond interaction with its carboxyl group. The thiadiazole ring forms two H-bonding interactions with Arg116, while the amide oxygen atom forms an H-bonding and an aromatic H-bonding interaction with Tyr45. Moreover, Arg155 forms a hydrogen bond with one of the tetrazole nitrogen atoms (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).</div><div class="NLM_p">Enalapril establishes H-bonding interactions with Arg123 and Arg172 through the carboxyl and carbonyl groups. The aromatic ring is involved in a π-cation interaction with Arg155 and in an aromatic hydrogen-bonding interaction with Leu80. Arg116 and Arg172 are both involved in forming salt-bridge interactions with the oxygen atom of the carboxyl group (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D).</div><div class="NLM_p last">The virtual screening results showed that all of the compounds are involved in salt-bridge interactions with Arg172 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) (for the 3D representation of other compounds see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Figure S9</a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Molecular Dynamics</h3><div class="NLM_p">To further investigate the ligand–protein interaction profile, we performed MDs. Starting from the best pose of folic acid, rosmarinic acid, cefazolin, and enalapril, 100 ns of MDs were carried out. During the whole trajectory the root mean square deviation (RMSD) for each complex was calculated and the analysis revealed a higher stability of the protein in the presence of folic acid (RMSD = 4.13 ± 0.42 Å), cefazolin (RMSD = 6.08 ± 0.77 Å), and enalapril (RMSD = 6.33 ± 1.00 Å) compared to rosmarinic acid (RMSD = 7.98 ± 3.94 Å) as shown in the protein RMSD plot (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Figure S10</a>).</div><div class="NLM_p">A similar trend was also observed in the behavior of the ligands. As the ligand RMSD plot shows (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Figure S11</a>), folic acid (RMSD = 3.75 ± 0.44 Å) and cefazolin (RMSD = 4.99 ± 0.49 Å) maintain a more stable behavior throughout the whole simulation compared to rosmarinc acid (RMSD = 8.88 ± 4.12 Å) and enalapril (RMSD = 9.72 ± 2.14 Å).</div><div class="NLM_p">For each ligand, the root mean square fluctuation (RMSF), which gives information about the ligand atom position changes during the whole simulation, was also calculated to characterize the fluctuations in the ligand position within the complex (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Clearly, the general pattern of fluctuation is significantly lower for folic acid and cefazolin compared to rosmarinic acid and enalapril.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. RMSF plots of (A) folic acid, (B) rosmarinic acid, (C) cefazolin, and (D) enalapril.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, folic acid shows several long-lasting interactions with residues of both the RRM domains (Tyr82, Tyr114, Arg116, Asn126, Thr153, Arg155, Phe170, Arg172), hence keeping the protein in a closed conformation. Although cefazolin and enalapril are involved in a minor number of interactions with the protein, they are still able to stabilize it because these interactions involve residues from both the RRM domains and are persistent throughout the simulation. On the other hand, rosmarinic acid is not able to maintain all of the interactions with the two RRM domains during the whole simulation (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Figure S12</a>).</div><div class="NLM_p">Furthermore, the presence of folic acid seems to best stabilize the protein as is shown by the RMSF plots of the protein backbone calculated during the simulations in the presence of folic acid and cefazolin (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Figure S13</a>).</div><div class="NLM_p">Noteworthily, these results are in agreement with the experimental data, as the minimum rate of fluctuations can indeed explain the strong signals observed during the STD experiments.</div><div class="NLM_p">Folic acid and cefazolin, which are characterized by strong STD signals, are stable and well accommodated in the HuD binding site and associated with a lower level of RMSF. STD-NMR analysis revealed that all of the selected hits can bind HuD and the epitope mapping evidenced that the selected molecules bind HuD with different features. In particular, folic acid and cefazolin exhibited the highest STD signals, thus suggesting that both molecules are well stabilized inside the HuD binding site, as confirmed by the MDs results. The MDs outcomes highlighted a lower level of fluctuations and more stable behavior of the two compounds compared to enalapril and rosmarinic acid.</div><div class="NLM_p">In particular, folic acid is the most attractive compound, in fact, it is well accommodated in the HuD binding site, establishing strong and lasting interactions within the HuD binding pocket.</div><div class="NLM_p last">The obtained data hence provided a deeper insight into the binding mode of these compounds and highlighted a clear correlation between the molecular modeling outcomes and the observed STD-NMR results.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cell Data</h3><div class="NLM_p last">In preliminary experiments, the effect of folic acid on the expression of HuD in SH-SY5Y cells using alpha-tubulin as the loading control was assessed. SH-SY5Y cells were exposed to 100 nM folic acid for 4 h and HuD protein levels were measured in the total homogenates. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Figure S14</a>, an increased expression of the HuD protein was observed in cells treated with folic acid with respect to controls.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this work we applied a structure-based virtual screening (SBVS) approach to a database of both FDA-approved drugs and natural compounds, thus offering the possibility of drug repositioning and shortening the drug development process.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The initial <i>in silico</i> studies indicated some promising compounds which were in-depth investigated. STD-NMR analysis revealed that all of the selected hits can bind HuD. The STD-NMR results are in agreement with the MDs data. Among all compounds, the most interesting one is folic acid. In fact, by combining STD-NMR and MDs results we found that folic acid is able to bind HuD and well stabilize the protein, forming different stable interactions and mimicking the mRNA binding mode.</div><div class="NLM_p">HuD is implicated in the pathogenesis of AD by contributing to the accumulation of Aβ peptides in AD brains, to the stabilization of APP and BACE1, and to the upregulation of neuroserpin.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a></div><div class="NLM_p">Hence, given the role of HuD in the genesis and accumulation of Aβ, our findings support the notion that folic acid supplementation may represent, together with other micronutrients, a valid nutritional intervention in AD.<a onclick="showRef(event, 'ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45 ref46">(43−46)</a></div><div class="NLM_p">Folic acid is a key cofactor in the one-carbon metabolism promoting the re-methylation of homocysteine to methionine. Methionine is further converted to S-adenosylmethionine, which is the primary methyl donor in most mammalian reactions. Concerning the underlying biological mechanism discussed here, it is tempting to speculate that folic acid may bind the HuD protein and regulate its activity/function via methylation, modulating, for example, its direct interaction with ARE-containing mRNAs. Accordingly, in PC12 cells, it has been demonstrated that the methylation of HuD by arginine methyltransferase CARM1 accelerates the decay of the target cyclin inhibitor mRNA p21cip1/waf1 (whose protein is involved in cell cycle arrest), thus maintaining the cells in a proliferative state.</div><div class="NLM_p">Conversely, the inhibition of HuD methylation increases p21cip1/waf1 mRNA half-life, resulting in a slow growth rate and robust neurogenesis in response to nerve growth factors. These findings thus suggest that unmethylated HuD may lower the decay of this specific target transcript by forming a tighter complex with it.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> With reference to AD, it would be interesting, for instance, to explore whether folic acid administration could be associated with HuD methylation and, in turn, with an accelerated decay of its specific target transcripts APP and BACE1.</div><div class="NLM_p">Thus, the ability of folic acid to affect HuD protein expression was evaluated in a preliminary biological setting. Within this context, the human neuroblastoma cells represent a valuable tool for drawing the effect of compounds influencing HuD activity/function.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Cell-based assay results support the molecular modeling and STD-NMR data, highlighting the ability of folic acid to recognize and affect HuD expression. In fact, in human SH-SY5Y cells treated with folic acid, an increase in HuD protein levels was observed.</div><div class="NLM_p">In this regard, our hypothesis is that folic acid interaction with HuD triggers a series of intracellular signals that finally result in an increase of HuD protein expression.</div><div class="NLM_p last">These findings could be an important starting point for the future treatment of some neurological diseases and in particular for AD, due to the crucial role of HuD protein in the pathogenesis of AD.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60662" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60662" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The RNA-binding protein HuD plays a crucial role in the post-transcriptional control of gene expression during neuronal development and its dysfunction/dysregulation may contribute to the pathogenesis of several disorders involving the nervous system, at both the central and peripheral level (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). In this work, we combined SBVS and STD-NMR studies to identify compounds that can bind HuD. Three of the four identified hits (folic acid, cefazolin, and enalapril) are drugs already on the market for the treatment of other pathological conditions. They could therefore be repurposed as novel candidates for the treatment of neurodegenerative diseases. Their development process would be considerably faster since they have already been tested in terms of their safety in preclinical models and human trials. Of particular interest is the evidence that folic acid can act as a HuD-mRNA interfering compound, since it could clarify a still unknown mechanism of action of this widely used vitamin and could justify its use in the prevention of some neurodegenerative diseases.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Additional experiments are ongoing to deeply understand the biological outcomes resulting from HuD and folic acid interaction. In conclusion, our results pave the way toward the validation of HuD as a drug target and may open a novel route for the discovery of innovative agents to counteract neurodegenerative diseases.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25313" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25313" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Molecular Modeling</h3><div class="NLM_p">Starting from the crystal structure of HuD and the (A/U)-rich element of <i>c-fos</i> mRNA, deposited in the Protein Data Bank (PDB) with the PDB code 1FXL,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> our molecular modeling studies were carried out. The receptor structure was properly processed using the Protein Preparation Wizard.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Hydrogens were added, disulfide bonds were created, and seleno-methionines were converted into methionines. The hydrogen-bonding network was optimized and the pKa of the residues along with their protonation states were calculated at pH 7.4. The complex was then submitted to 10000 iterations of energy minimization using the Polak-Ribiere conjugate gradient (PRCG) algorithm in the MacroModel <i>tool</i> and OPLS_2005 as the force field.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> Molecular dynamics (MD) simulation was performed to explore all of the conformational states of HuD and to examine the potential HuD rearrangement.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The MD was run using the Desmond package v. 3.8.<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> The system was immersed in an orthorhombic box of TIP4P water molecules, extending at least 10 Å from the protein and counter ions were added to neutralize the system charge. The system temperature was set at 300 K and the NPT ensemble was selected. The simulation was carried out for 500 ns and trajectories and energies were recorded at 500 ps intervals. The resulting trajectory was clustered with respect to the root mean square deviation (RMSD) to explore the collection of all of the obtained structures, giving ten representative structures, which were submitted to 10000 iterations of energy minimization using the Polak-Ribiere conjugated gradient (PRCG) algorithm, using the MacroModel <i>tool</i> and OPLS-2005 as the force field.<a onclick="showRef(event, 'ref52 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref52 ref54 ref55">(52,54,55)</a></div><div class="NLM_p last">The two conformations with the highest and the lowest potential energy values, corresponding to an “<i>open”</i> HuD conformation and a <i>closed</i> HuD conformation, were selected. According to our previous works,<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> we noticed that more interactions were established between the ligands and the receptor in the presence of the closed conformation, thus, for further docking studies, we chose the HuD closed conformation. Molecular docking recognition studies were carried out using Glide v. 6.7 software, using the standard precision (SP) algorithm and the binding pocket was identified by placing a cube centered on the <i>c-fos</i> mRNA. Ten poses per ligand were generated.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> For the virtual screening studies, we used a library of 50914 compounds obtained from different databases available online: FooDB, the world’s largest and most comprehensive resource on food constituents, chemistry, and biology;<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> DrugBank, a comprehensive, freely accessible, online database containing information on drugs and drug targets;<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> ZINC15 (a free database of commercially available compounds) biogenic subset, which contains derivatives of natural products, endogenous human metabolites, metabolites, and natural products.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The library was prepared using the LigPrep tool.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Hydrogens were added, salts were removed, and ionization states were calculated using Epik at pH 7.4 ± 0.0. Each structure was submitted to 10 000 iterations of energy minimization using the Polak-Ribiere conjugated gradient (PRCG) algorithm, using the MacroModel <i>tool</i> and OPLS_2005 force field.<a onclick="showRef(event, 'ref52 ref55'); return false;" href="javascript:void(0);" class="ref ref52 ref55">(52,55)</a> After the virtual screening, the results were filtered considering the Lipinski rule of 5<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and the compounds obtained were ranked by the G-score value. Considering the theoretical binding affinity score value of the best hit we retained, for further studies, all of the structures with a Glide score value within 3 kcal/mol from the best hit. Eventually, 10 compounds were selected. Each selected hit was searched within several vendors databases and finally, based on their commercial availability, 4 best hits were selected and submitted for STD-NMR experimental evaluation. The best docking pose of the 4 selected compounds was subjected to 100 ns MDs. The MDs were carried out according to the protocol described above. The simulation interaction diagram tool implemented in Desmond v. 3.8<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> was used to analyze the behavior of the ligands and the protein. The analysis included the RMSD and RMSF calculations evaluated during the whole simulation.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Evaluation of Pan Assay Interference Compounds (PAINS)</h3><div class="NLM_p last">The PAINS properties of the chemical structures of the investigated compounds were theoretically investigated using the zinc PAINS Pattern Identifier web server.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> The method highlighted that among the 10 compounds rosmarinic acid and methoxatin contain PAINS: the catechol moiety in rosmarinic acid and the imine and quinone moieties in methoxantine, respectively. The method did not highlight any PAINS related to the other investigated molecules. Interestingly, folic acid, the most promising hit, does not contain any PAINS.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> NMR Experiments</h3><div class="NLM_p last">Folic acid (≥97%, LC-MS [M + H]<sup>+</sup> = 442.147), rosmarinic acid (96%, LC-MS [M + H]<sup>+</sup> = 361.093), cefazolin (≥97%, LC-MS [M + H]<sup>+</sup> = 455.037), and enalapril maleate (≥98%, LC-MS [M + H]<sup>+</sup> = 377.208) were purchased from Merck-Sigma-Aldrich.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Stability Evaluation and Characterization</h3><div class="NLM_p last">The stability test and the characterization of all of the compounds were performed at 298 K on a Bruker Avance 400 MHz spectrometer in a 5 mm NMR tube. The experiments were carried out in D<sub>2</sub>O and aqueous phosphate buffer at pH 7.4 (20 mM). To obtain clear spectra, the water signal suppression was achieved using an excitation sculpting sequence from the Bruker library. Two-dimensional experiments (COSY, NOESY, and HSQC) were carried out on the samples at a concentration range of 5–10 mM. NOESY experiments were performed with a mixing time of 200 and 700 ms. All proton and carbon chemical shifts were assigned unambiguously, and the assignment is reported in the SM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">Tables S1–S4</a>).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> HuD Expression and Purification</h3><div class="NLM_p last">HuD gene was cloned in a pET-42a vector, provided by Prof A. Quattrone, (University of Trento) and expressed in <i>Escherichia coli</i> BL21 (DE3) cultured in Luria Broth (LB) medium. After induction with 1 mM IPTG, cells were grown at 310 K for 6 h and collected. Pellets were dissolved in equilibration buffer (20 mM Tris-Cl pH 8, 100 mM NaCl, 3 mM MgCl<sub>2</sub>, 0.5% Triton X-100) and disrupted by sonication. After centrifugation, the supernatant was filtered and used for purification on a Ni-NTA column using a 20–500 mM imidazole gradient. The protein was stored in phosphate buffer (20 mM) and imidazole (300 mM). The final protein concentration was 3 μM. For the NMR studies, the protein was transferred in a Sartorius Vivaspin 500 concentrator (molecular weight cut-off of 5 K) and subjected to four cycles with a Hermle Z 233 M-2 laboratory centrifuge, at 10 000 rpm for 10 min, to remove the imidazole that could interfere with the interaction study. The final protein concentration was about 12 μM in 20 mM phosphate buffer.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> STD-NMR Interaction Studies</h3><div class="NLM_p last">The interaction between the ligands and the isolated protein was detected using STD-NMR experiments. The spectra were acquired with a Bruker Avance 600 MHz instrument at 283 K, in a 3 mm NMR tube and in the phosphate buffer previously described (200 μl). All protein–ligand samples were prepared in a 1000:1 ligand/protein ratio. The interaction between ligands and isolated proteins was detected using STD experiments. In the STD experiments, water suppression was achieved using the WATERGATE 3-9-19 pulse sequence. The on-resonance irradiation of the protein was applied at −0.05 ppm. Off-resonance irradiation was applied at 200 ppm, where no protein signals were visible. Selective presaturation of the protein was achieved by a train of Gauss-shaped pulses of 49 ms length each. The total length of the saturation train depends on the L7 parameter (the loop counter). STD experiments were performed with L7 = 60 leading to a total saturation time of 2.94 s. The intensities of all STD effects (absolute STD) were calculated by division through integrals over the respective signals in the STD-NMR reference spectra. The different signal intensities of the individual protons are best analyzed from the integral values in the reference and STD spectra, respectively. (<i>I</i><sub>0</sub> – <i>I</i><sub>sat</sub>)/<i>I</i><sub>0</sub> is the fractional STD effect, expressing the signal intensity in the STD spectrum as a fraction of the intensity of an unsaturated reference spectrum. In this equation, <i>I</i><sub>0</sub> is the intensity of one signal in the off-resonance or reference NMR spectrum, <i>I</i><sub>sat</sub> is the intensity of a signal in the on-resonance NMR spectrum, and <i>I</i><sub>0</sub>– <i>I</i><sub>sat</sub> represents the intensity of the STD-NMR spectrum.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Cell Culture</h3><div class="NLM_p last">Human neuroblastoma SH-SY5Y cells were obtained from ATCC (Manassas, VA) and cultured in a humidified incubator at 37 7̅C with 5% CO<sub>2</sub>. The SH-SY5Y cells were grown in Eagle’s minimum essential medium (EMEM) supplemented with 10% fetal bovine serum, 1% penicillin–streptomycin, <span class="smallcaps smallerCapital">l</span>-glutamine (2 mM), nonessential amino acids (1 mM), and sodium pyruvate (1 mM). The cells were treated with 100 nM folic acid for 4 h and then homogenized in a specific cell lysis buffer (Cell Signaling Technology).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Western Blotting</h3><div class="NLM_p last">Proteins were measured according to Bradford’s method using bovine albumin as the internal standard. Proteins were diluted in 2xSDS protein gel loading solution, boiled for 5 min, and separated on 12% SDS-PAGE gel. The anti-HuD mouse monoclonal antibody (Sigma-Aldrich) and the rat monoclonal antibody anti-α-tubulin (Thermo Fisher Scientific) were diluted (1:500 and 1:1000, respectively) in TBST buffer [10 mM Tris-HCl, 100 mM NaCl, 0.1% (v/v) Tween 20, pH 7.5] containing 6% milk. The nitrocellulose membrane signals were detected by chemiluminescence. The same membranes were re-probed with the α-tubulin antibody and used to normalize the data. Statistical analysis of Western blot data was performed on the densitometric values obtained using the ImageJ image-processing program (<a href="https://imagej.nih.gov/ij" class="extLink">https://imagej.nih.gov/ij</a>).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Statistical Analysis</h3><div class="NLM_p last">For statistical analysis, the GraphPad Instat statistical package (version 3.05 GraphPad Software, San Diego, CA) was used. The data were analyzed by an unpaired <i>t</i>-test. Differences were considered statistically significant when <i>p</i> values <0.05.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00191" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67473" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67473" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00191?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00191</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Glide score value (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB complex of cefazolin docked into HuD (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB complex of enalapril docked into HuD (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB complex of folic acid docked into HuD (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB complex of rosmarinic acid docked into HuD (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Structural animation of folic acid in complex with HuD (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_006.avi" class="ext-link">AVI</a>)</p></li><li><p class="inline">NMR analysis. Figures S1-S3-S5-S7: Chemical structures of the best hits. Tables S1-S2-S3-S4: <sup>1</sup>H and <sup>13</sup>C assignment of the best hits. Figures S2-S4-S6-S8: <sup>1</sup>H-NMR and STD-NMR of the best hits in the presence of HuD. Molecular modeling analysis. Table S5: Molecular formula strings of selected compounds. Figure S9: 3D representations of the best poses of methoxatin, aurintricarboxylic acid, portulacaxanthin II, alloxazine, tilundronic acid, and phosphoglycolic acid within the HuD binding pocket. Figure S10: HuD protein backbone RMSD trend calculated in the presence of the best hits. Figure S11: Ligands RMSD trends of the best hits. Figure S12: Timeline representation of the interactions and contacts between the best hits and the protein. Figure S13: HuD protein backbone RMSF calculated in the presence of folic acid and cefazolin. Figure S14: Effect of folic acid on HuD protein (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_001.csv">jm1c00191_si_001.csv (0.88 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_002.pdb">jm1c00191_si_002.pdb (214.73 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_003.pdb">jm1c00191_si_003.pdb (215.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_004.pdb">jm1c00191_si_004.pdb (215.52 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_005.pdb">jm1c00191_si_005.pdb (214.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_006.avi">jm1c00191_si_006.avi (50.92 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf">jm1c00191_si_007.pdf (1.01 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00191" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giosuè Costa</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze della Salute, Università
“Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Net4Science
Academic Spin-Off, Università “Magna
Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Associazione
CRISEA—Centro di Ricerca e Servizi Avanzati per l’Innovazione
Rurale, Località
Condoleo, Belcastro, Catanzaro, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0947-9479" title="Orcid link">https://orcid.org/0000-0003-0947-9479</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6f080c001c1b0e2f1a01060c1541061b"><span class="__cf_email__" data-cfemail="96f1f5f9e5e2f7d6e3f8fff5ecb8ffe2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesca Vasile</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Milan, Via Golgi 19, 20133 Milano, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b5d3c7d4dbd6d0c6d6d49bc3d4c6dcd9d0f5c0dbdcd8dc9bdcc1"><span class="__cf_email__" data-cfemail="583e2a39363b3d2b3b39762e392b31343d182d3631353176312c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simona Collina</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2954-7558" title="Orcid link">https://orcid.org/0000-0002-2954-7558</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ec9f858183828dc28f83808085828dac9982859c9ac28598"><span class="__cf_email__" data-cfemail="3e4d575351505f105d51525257505f7e4b50574e4810574a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesca Alessandra Ambrosio</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze della Salute, Università
“Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4874-2946" title="Orcid link">https://orcid.org/0000-0003-4874-2946</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adriana Coricello</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze della Salute, Università
“Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonio Lupia</span> - <span class="hlFld-Affiliation affiliation">Net4Science
Academic Spin-Off, Università “Magna
Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Associazione
CRISEA—Centro di Ricerca e Servizi Avanzati per l’Innovazione
Rurale, Località
Condoleo, Belcastro, Catanzaro, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mariachiara Micaelli</span> - <span class="hlFld-Affiliation affiliation">CIBIO—Department
of Cellular, Computational and Integrative Biology, University of Trento, Via Sommarive 9, Povo, 38123 Trento, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicoletta Marchesi</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federico Sala</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Milan, Via Golgi 19, 20133 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessia Pascale</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniela Rossi</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Sciences, University of Pavia, Via Taramelli 12, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefano Alcaro</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze della Salute, Università
“Magna Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Net4Science
Academic Spin-Off, Università “Magna
Græcia” di Catanzaro, Campus “S. Venuta”, 88100 Catanzaro, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Associazione
CRISEA—Centro di Ricerca e Servizi Avanzati per l’Innovazione
Rurale, Località
Condoleo, Belcastro, Catanzaro, Italy</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.A.A. and A.C. contributed equally to this work. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This research was funded by the PRIN 2017 research project “Novel anticancer agents endowed with multi-targeting mechanism of action” (201744BN5T). The present work was co-founded by the European Committee, FESR FSE 2014-2020, and by Regione Calabria. The author A.C. is responsible for this publication and the European Committee and Regione Calabria decline any responsibility concerning the use of the information disclosed in the paper.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e1110-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01215" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01215" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">F.A.A., G.C., and S.A. thank the European Cooperation in Science and Technology (COST Action - CA15135. Multi-target paradigm for innovative ligand identification in the drug-discovery process). This work has been partially supported by the PON VQA project funded by the Italian Minister of Economy (MISE). The authors aknowledge Dr. Serena Della Volpe for the compounds characterization.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AD</td><td class="NLM_def"><p class="first last">Alzheimer’s disease</p></td></tr><tr><td class="NLM_term">APP</td><td class="NLM_def"><p class="first last">Amyloid Precursor Protein</p></td></tr><tr><td class="NLM_term">AREs</td><td class="NLM_def"><p class="first last">(A/U)-rich elements</p></td></tr><tr><td class="NLM_term">Aβ</td><td class="NLM_def"><p class="first last">Amyloid-β</p></td></tr><tr><td class="NLM_term">BACE1</td><td class="NLM_def"><p class="first last">β-site APP-cleaving enzyme 1</p></td></tr><tr><td class="NLM_term">CADD</td><td class="NLM_def"><p class="first last">computer-aided drug design</p></td></tr><tr><td class="NLM_term">DOSY</td><td class="NLM_def"><p class="first last">diffusion-ordered spectroscopy</p></td></tr><tr><td class="NLM_term">ELAV</td><td class="NLM_def"><p class="first last">embryonic lethal abnormal vision</p></td></tr><tr><td class="NLM_term">ELAVL</td><td class="NLM_def"><p class="first last">embryonic lethal abnormal vision like</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">food and drug administration</p></td></tr><tr><td class="NLM_term">MDs</td><td class="NLM_def"><p class="first last">molecular dynamics simulations</p></td></tr><tr><td class="NLM_term">MM-GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics/generalized Born surface area</p></td></tr><tr><td class="NLM_term">mRNA</td><td class="NLM_def"><p class="first last">messenger RNA</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">Pan assay interference compounds</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PS1</td><td class="NLM_def"><p class="first last">presenilin 1</p></td></tr><tr><td class="NLM_term">RBPs</td><td class="NLM_def"><p class="first last">RNA-binding proteins</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">Root mean square deviation</p></td></tr><tr><td class="NLM_term">RNP2</td><td class="NLM_def"><p class="first last">Ribonucleoprotein</p></td></tr><tr><td class="NLM_term">RRMs</td><td class="NLM_def"><p class="first last">RNA recognition motifs</p></td></tr><tr><td class="NLM_term">SBVS</td><td class="NLM_def"><p class="first last">structure-based virtual screening</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">saturation transfer difference</p></td></tr><tr><td class="NLM_term">UTRs</td><td class="NLM_def"><p class="first last">untranslated regions</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screening</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talman, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jäntti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuominen, R. K.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase C Activation as a Potential Therapeutic Strategy in Alzheimer’s Disease: Is there a Role for Embryonic Lethal Abnormal Vision-like Proteins?</span>. <i>Basic Clin. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1111/bcpt.12581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1111%2Fbcpt.12581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=27001133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVOnu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2016&pages=149-160&author=V.+Talmanauthor=A.+Pascaleauthor=M.+J%C3%A4nttiauthor=M.+Amadioauthor=R.+K.+Tuominen&title=Protein+Kinase+C+Activation+as+a+Potential+Therapeutic+Strategy+in+Alzheimer%E2%80%99s+Disease%3A+Is+there+a+Role+for+Embryonic+Lethal+Abnormal+Vision-like+Proteins%3F&doi=10.1111%2Fbcpt.12581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C activation as a potential therapeutic strategy in Alzheimer's disease: Is there a role for embryonic lethal abnormal vision-like proteins?</span></div><div class="casAuthors">Talman, Virpi; Pascale, Alessia; Jaentti, Maria; Amadio, Marialaura; Tuominen, Raimo K.</div><div class="citationInfo"><span class="NLM_cas:title">Basic & Clinical Pharmacology & Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">149-160</span>CODEN:
                <span class="NLM_cas:coden">BCPTBO</span>;
        ISSN:<span class="NLM_cas:issn">1742-7835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease (AD), the most common cause of dementia, is an irreversible and progressive neurodegenerative disorder.  It affects predominantly brain areas that are crit. for memory and learning and is characterized by two main pathol. hallmarks: extracellular amyloid plaques and intracellular neurofibrillary tangles.  Protein kinase C (PKC) has been classified as one of the cognitive kinases controlling memory and learning.  By regulating several signalling pathways involved in amyloid and tau pathologies, it also plays an inhibitory role in AD pathophysiol.  Among downstream targets of PKC are the embryonic lethal abnormal vision (ELAV)-like RNA-binding proteins that modulate the stability and the translation of specific target mRNAs involved in synaptic remodelling linked to cognitive processes.  This MiniReview summarizes the current evidence on the role of PKC and ELAV-like proteins in learning and memory, highlighting how their derangement can contribute to AD pathophysiol.  This last aspect emphasizes the potential of pharmacol. activation of PKC as a promising therapeutic strategy for the treatment of AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoAKKMkypK_bVg90H21EOLACvtfcHk0ljyHXzXBP2ITA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVOnu70%253D&md5=8850cf732159192dc719f96089fab34d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Fbcpt.12581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcpt.12581%26sid%3Dliteratum%253Aachs%26aulast%3DTalman%26aufirst%3DV.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DJ%25C3%25A4ntti%26aufirst%3DM.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DTuominen%26aufirst%3DR.%2BK.%26atitle%3DProtein%2520Kinase%2520C%2520Activation%2520as%2520a%2520Potential%2520Therapeutic%2520Strategy%2520in%2520Alzheimer%25E2%2580%2599s%2520Disease%253A%2520Is%2520there%2520a%2520Role%2520for%2520Embryonic%2520Lethal%2520Abnormal%2520Vision-like%2520Proteins%253F%26jtitle%3DBasic%2520Clin.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D119%26spage%3D149%26epage%3D160%26doi%3D10.1111%2Fbcpt.12581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bellavia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mecarozzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campese, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazioli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Screpanti, I.</span></span> <span> </span><span class="NLM_article-title">Notch3 and the Notch3-upregulated RNA-binding protein HuD regulate Ikaros alternative splicing</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1038%2Fsj.emboj.7601626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=17332745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1Wkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=1670-1680&author=D.+Bellaviaauthor=M.+Mecarozziauthor=A.+F.+Campeseauthor=P.+Grazioliauthor=C.+Taloraauthor=L.+Fratiauthor=A.+Gulinoauthor=I.+Screpanti&title=Notch3+and+the+Notch3-upregulated+RNA-binding+protein+HuD+regulate+Ikaros+alternative+splicing&doi=10.1038%2Fsj.emboj.7601626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Notch3 and the Notch3-upregulated RNA-binding protein HuD regulate Ikaros alternative splicing</span></div><div class="casAuthors">Bellavia, Diana; Mecarozzi, Marco; Campese, Antonio F.; Grazioli, Paola; Talora, Claudio; Frati, Luigi; Gulino, Alberto; Screpanti, Isabella</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1670-1680</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Constitutive activation of the transmembrane receptor, Notch3, and loss of function of the hematopoietic transcription repressor, Ikaros (IK), play direct roles in T-cell differentiation and leukemogenesis that are dependent on pre-T-cell receptor (pre-TCR) signaling.  We demonstrate the occurrence of crosstalk between Notch3 and IK that results in transcriptional regulation of the gene encoding the pTα chain of the pre-TCR.  We also show that, in the presence of the pre-TCR, constitutive activation of Notch3 in thymocytes causes increased expression of dominant-neg. non-DNA-binding IK isoforms, which are able to restrain the IK inhibition of Notch3's transcriptional activation of pTα.  This effect appears to be mediated by Notch3's pre-TCR-dependent upregulation of the RNA-binding protein, HuD.  Notch3 signaling thus appears to play a crit. role in the diminished IK activity described in several lymphoid leukemias.  By exerting transcription-activating and transcription-repressing effects on the pTα promoter, Notch3 and IK cooperate in the fine-tuning of pre-TCR expression and function, which has important implications for the regulation of thymocyte differentiation and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn8EHpXXmf9rVg90H21EOLACvtfcHk0lgKVKDuz0pdVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1Wkt7w%253D&md5=405fe0573c16421b8b2d3d4516ea50a8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601626%26sid%3Dliteratum%253Aachs%26aulast%3DBellavia%26aufirst%3DD.%26aulast%3DMecarozzi%26aufirst%3DM.%26aulast%3DCampese%26aufirst%3DA.%2BF.%26aulast%3DGrazioli%26aufirst%3DP.%26aulast%3DTalora%26aufirst%3DC.%26aulast%3DFrati%26aufirst%3DL.%26aulast%3DGulino%26aufirst%3DA.%26aulast%3DScrepanti%26aufirst%3DI.%26atitle%3DNotch3%2520and%2520the%2520Notch3-upregulated%2520RNA-binding%2520protein%2520HuD%2520regulate%2520Ikaros%2520alternative%2520splicing%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D1670%26epage%3D1680%26doi%3D10.1038%2Fsj.emboj.7601626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mansfield, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keene, J. D.</span></span> <span> </span><span class="NLM_article-title">Neuron-specific ELAV/Hu proteins suppress HuR mRNA during neuronal differentiation by alternative polyadenylation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2734</span>– <span class="NLM_lpage">2746</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr1114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkr1114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=22139917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFOrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=2734-2746&author=K.+D.+Mansfieldauthor=J.+D.+Keene&title=Neuron-specific+ELAV%2FHu+proteins+suppress+HuR+mRNA+during+neuronal+differentiation+by+alternative+polyadenylation&doi=10.1093%2Fnar%2Fgkr1114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Neuron-specific ELAV/Hu proteins suppress HuR mRNA during neuronal differentiation by alternative polyadenylation</span></div><div class="casAuthors">Mansfield, Kyle D.; Keene, Jack D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2734-2746</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The ubiquitously expressed RNA-binding protein HuR increases the stability and translation of mRNAs encoding growth regulatory proteins that promote proliferation in a variety of cell types.  However, the three neuron-specific ELAV/Hu proteins, HuB, HuC and HuD, while binding to the same types of mRNAs, are required instead for neuronal differentiation, and it becomes difficult to reconcile these contrary functions when all four Hu proteins are expressed in the same neuron.  HuR mRNA exists as three alternatively polyadenylated variants, a 1.5-kb testes-specific mRNA isoform, a ubiquitous 2.4-kb isoform and a 6.0-kb isoform that we now show is induced during neuronal differentiation and appears to be neuron-specific.  This 6.0-kb neuron-specific mRNA isoform is inherently less stable and produces less HuR protein than the ubiquitous 2.4-kb mRNA.  Furthermore, we show that neuronal HuB, HuC and HuD, as well as HuR itself, can bind at the 2.4-kb mRNA polyadenylation site, and when overexpressed can affect alternative polyadenylation to generate an extended HuR 3'-UTR that is translationally suppressed.  We propose that the regulation of HuR protein expression by alternative polyadenylation allows neurons to post-transcriptionally regulate mRNAs-encoding factors required for proliferation vs. differentiation to facilitate neuronal differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4X4ncI3_Hf7Vg90H21EOLACvtfcHk0lgKVKDuz0pdVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFOrsr0%253D&md5=3383427fc30286d573d8fe46c3f0da54</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1114%26sid%3Dliteratum%253Aachs%26aulast%3DMansfield%26aufirst%3DK.%2BD.%26aulast%3DKeene%26aufirst%3DJ.%2BD.%26atitle%3DNeuron-specific%2520ELAV%252FHu%2520proteins%2520suppress%2520HuR%2520mRNA%2520during%2520neuronal%2520differentiation%2520by%2520alternative%2520polyadenylation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D2734%26epage%3D2746%26doi%3D10.1093%2Fnar%2Fgkr1114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span> <span> </span><span class="NLM_article-title">HuD promotes BDNF expression in brain neurons via selective stabilization of the BDNF long 3’UTR mRNA</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e55718</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0055718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1371%2Fjournal.pone.0055718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=23383270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1yltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=M.+Allenauthor=C.+Birdauthor=W.+Fengauthor=G.+Liuauthor=W.+Liauthor=N.+I.+Perrone-Bizzozeroauthor=Y.+Feng&title=HuD+promotes+BDNF+expression+in+brain+neurons+via+selective+stabilization+of+the+BDNF+long+3%E2%80%99UTR+mRNA&doi=10.1371%2Fjournal.pone.0055718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">HuD promotes BDNF expression in brain neurons via selective stabilization of the BDNF long 3'UTR mRNA</span></div><div class="casAuthors">Allen, Megan; Bird, Clark; Feng, Wei; Liu, Guanglu; Li, Wenqi; Perrone-Bizzozero, Nora I.; Feng, Yue</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e55718</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Complex regulation of brain-derived neurotrophic factor (BDNF) governs its intricate functions in brain development and neuronal plasticity.  Besides tight transcriptional control from multiple distinct promoters, alternative 3'end processing of the BDNF transcripts generates either a long or a short 3'untranslated region (3'UTR).  Previous reports indicate that distinct RNA sequence in the BDNF 3'UTRs differentially regulates BDNF prodn. in the brain to accommodate neuronal activity changes, conceivably through differential interactions with undefined trans-acting factors that regulate stability and translation of these BDNF mRNA isoforms.  In this study, we report that the neuronal RNA-binding protein (RBP) HuD interacts with a highly conserved AU-rich element (ARE) specifically located in the BDNF long 3'UTR.  Such interaction is necessary and sufficient for selective stabilization of mRNAs that contain the BDNF long 3'UTR in vitro and in vivo.  Moreover, in a HuD transgenic mouse model, the BDNF long 3'UTR mRNA is increased in the hippocampal dentate granule cells (DGCs), leading to elevated expression of BDNF protein that is transported and stored in the mossy fiber (MF) terminals.  Our results identify HuD as the first trans-acting factor that enhances BDNF expression specifically through the long 3'UTR and a novel mechanism that regulates BDNF protein prodn. in selected neuronal populations by HuD abundance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo0ryguUd-zbVg90H21EOLACvtfcHk0lh_gpbVpc7b2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1yltrs%253D&md5=d8207a900a52204059249350c6f23c2d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0055718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0055718%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DBird%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%2BI.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DHuD%2520promotes%2520BDNF%2520expression%2520in%2520brain%2520neurons%2520via%2520selective%2520stabilization%2520of%2520the%2520BDNF%2520long%25203%25E2%2580%2599UTR%2520mRNA%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0055718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, S.</span></span> <span> </span><span class="NLM_article-title">Different motif requirements for the localization zipcode element of β-actin mRNA binding by HuD and ZBP1</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">7432</span>– <span class="NLM_lpage">7446</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkv699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=26152301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGrur3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=7432-7446&author=H.+H.+Kimauthor=S.+J.+Leeauthor=A.+S.+Gardinerauthor=N.+I.+Perrone-Bizzozeroauthor=S.+Yoo&title=Different+motif+requirements+for+the+localization+zipcode+element+of+%CE%B2-actin+mRNA+binding+by+HuD+and+ZBP1&doi=10.1093%2Fnar%2Fgkv699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Different motif requirements for the localization zipcode element of β-actin mRNA binding by HuD and ZBP1</span></div><div class="casAuthors">Kim, Hak Hee; Lee, Seung Joon; Gardiner, Amy S.; Perrone-Bizzozero, Nora I.; Yoo, Soonmoon</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7432-7446</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The interactions of RNA-binding proteins (RBPs) with their target transcripts are essential for regulating gene expression at the post-transcriptional level including mRNA export/localization, stability, and translation.  ZBP1 and HuD are RBPs that play pivotal roles in mRNA transport and local translational control in neuronal processes.  While HuD possesses 3 RNA recognition motifs (RRMs), ZBP1 contains 2 RRMs and 4 K homol. (KH) domains that either increase target specificity or provide a multi-target binding capability.  Here, the authors used isolated cis-element sequences of the target mRNA to examine directly protein-RNA interactions in cell-free systems.  The authors found that both ZBP1 and HuD bound the zipcode element in rat β-actin mRNA's 3' UTR.  Differences between HuD and ZBP1 were obsd. in their binding preference to the element.  HuD showed a binding preference for U-rich sequences.  In contrast, ZBP1 binding to the zipcode RNA depended more on the structural level, as it required the proper spatial organization of a stem-loop that was mainly detd. by the U-rich element juxtaposed to the 3' end of a 5'-ACACCC-3' motif.  On the basis of this work, the authors propose that ZBP1 and HuD bind to overlapping sites in the β-actin zipcode, but they recognize different features of this target sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd4lheAQddorVg90H21EOLACvtfcHk0lh_gpbVpc7b2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGrur3P&md5=6282fed52d7bc1d83eab2f02cbda285e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv699%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.%2BH.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DGardiner%26aufirst%3DA.%2BS.%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%2BI.%26aulast%3DYoo%26aufirst%3DS.%26atitle%3DDifferent%2520motif%2520requirements%2520for%2520the%2520localization%2520zipcode%2520element%2520of%2520%25CE%25B2-actin%2520mRNA%2520binding%2520by%2520HuD%2520and%2520ZBP1%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D7432%26epage%3D7446%26doi%3D10.1093%2Fnar%2Fgkv699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Good, P. J.</span></span> <span> </span><span class="NLM_article-title">A conserved family of elav-like genes in vertebrates</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">4557</span>– <span class="NLM_lpage">4561</span>, <span class="refDoi"> DOI: 10.1073/pnas.92.10.4557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.92.10.4557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=7753842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADyaK2MXls1aksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=4557-4561&author=P.+J.+Good&title=A+conserved+family+of+elav-like+genes+in+vertebrates&doi=10.1073%2Fpnas.92.10.4557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved family of elav-like genes in vertebrates</span></div><div class="casAuthors">Good, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4557-61</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A large family of genes encodes proteins with RNA recognition motifs that are presumed to bind RNA and to function in posttranscription regulation.  Neural-specific members of this family include elav, a gene required for correct differentiation and maintenance of neurons in Drosophila melanogaster, and a related gene, HuD, which is expressed in human neuronal cells.  I have identified genes related to elav and HuD in Xenopus laevis, zebrafish, and mouse that define a family of four closely related vertebrate elav-like genes (elrA, elrB, elrC, and elrD) in fish, frogs, and mammals.  In addn. to protein sequence conservation, a segment of the 3'-untranslated sequence of elrD is also conserved, implying a functional role in elrD expression.  In adult frogs, elrC and elrD are exclusively expressed in the brain, whereas elrB is expressed in brain, testis, and ovary.  During Xenopus development, elrC and elrD RNAs are detected by late gastrula and late neurula stages, resp., whereas a nervous system-specific elrB RNA species is expressed by early tadpole stage.  Addnl. elrB transcripts are detected in the ovary and early embryo, demonstrating a maternal supply of mRNA and possibility of protein.  These expression patterns suggest a role for different elav-like genes in early development and neuronal differentiation.  Surprisingly, elrA is expressed in all adult tissues tested and at all times during development.  Thus, the widely expressed elrA is expected to have a related function in all cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKH_4_XJAyJrVg90H21EOLACvtfcHk0lh_gpbVpc7b2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXls1aksbs%253D&md5=d001ce3c18f2723e37a09011464cc878</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.10.4557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.10.4557%26sid%3Dliteratum%253Aachs%26aulast%3DGood%26aufirst%3DP.%2BJ.%26atitle%3DA%2520conserved%2520family%2520of%2520elav-like%2520genes%2520in%2520vertebrates%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D4557%26epage%3D4561%26doi%3D10.1073%2Fpnas.92.10.4557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanna, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghelardini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galeotti, N.</span></span> <span> </span><span class="NLM_article-title">HuD-mediated distinct BDNF regulatory pathways promote regeneration after nerve injury</span>. <i>Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1659</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.brainres.2017.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.brainres.2017.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=28111162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1659&publication_year=2017&pages=55-63&author=M.+D.+Sannaauthor=C.+Ghelardiniauthor=N.+Galeotti&title=HuD-mediated+distinct+BDNF+regulatory+pathways+promote+regeneration+after+nerve+injury&doi=10.1016%2Fj.brainres.2017.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">HuD-mediated distinct BDNF regulatory pathways promote regeneration after nerve injury</span></div><div class="casAuthors">Sanna, Maria Domenica; Ghelardini, Carla; Galeotti, Nicoletta</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1659</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">55-63</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Up-regulation of brain-derived neurotrophic factor (BDNF) synthesis is an important mechanism of peripheral nerve regeneration after injury.  However, the cellular and mol. mechanisms underlying this process are not fully understood.  This study examines the role of BDNF in the spared nerve injury (SNI) mice model.  Protein expression and cellular localization were investigated in the dorsal root ganglia (DRG) and spinal cord by western blotting and immunofluorescence expts. resp.  BDNF protein was markedly increased 3 and 7 days post-injury in the spinal cord and DRG.  Following nerve injury sensory neurons produce mols. to promote regeneration, such as growth-assocd. protein 43 (GAP-43) and cytoskeletal proteins.  Our results show that the expression of GAP-43 was increased in the DRG and spinal cord while, an increased of p-NFH content was detected in the spinal cord, with no modification in the DRG.  Both events were counteracted by the administration of an anti-BDNF antibody.  In DRG of SNI mice we also detected an increase of HuD expression, a RNA-binding protein known to stabilize BDNF and GAP-43 mRNA.  Silencing of HuD prevented the nerve injury-induced BDNF and GAP-43 enhanced expression in the DRG.  HuD-mediated BDNF synthesis in the primary sensory neurons, is followed by an anterograde transport of the neurotrophin to the central terminals of the primary afferents in the spinal dorsal horn, to modulate GAP-43 and NFH activation.  Our data suggest that BDNF, GAP-43 and p-NFH proteins increase are linked events required for the enhanced regeneration after nerve injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZPkXrRniCfLVg90H21EOLACvtfcHk0ljhHz2cDMAzjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrtL4%253D&md5=f7b70b966265acd8ebbc337a08d5c3ae</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.brainres.2017.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainres.2017.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DSanna%26aufirst%3DM.%2BD.%26aulast%3DGhelardini%26aufirst%3DC.%26aulast%3DGaleotti%26aufirst%3DN.%26atitle%3DHuD-mediated%2520distinct%2520BDNF%2520regulatory%2520pathways%2520promote%2520regeneration%2520after%2520nerve%2520injury%26jtitle%3DBrain%2520Res.%26date%3D2017%26volume%3D1659%26spage%3D55%26epage%3D63%26doi%3D10.1016%2Fj.brainres.2017.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gusev, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dottorini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkon, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrone, A.</span></span> <span> </span><span class="NLM_article-title">Increase of the RNA-binding protein HuD and posttranscriptional up-regulation of the GAP-43 gene during spatial memory</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">1217</span>– <span class="NLM_lpage">1222</span>, <span class="refDoi"> DOI: 10.1073/pnas.0307674100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.0307674100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=14745023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlWjt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=1217-1222&author=A.+Pascaleauthor=P.+A.+Gusevauthor=M.+Amadioauthor=T.+Dottoriniauthor=S.+Govoniauthor=D.+L.+Alkonauthor=A.+Quattrone&title=Increase+of+the+RNA-binding+protein+HuD+and+posttranscriptional+up-regulation+of+the+GAP-43+gene+during+spatial+memory&doi=10.1073%2Fpnas.0307674100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Increase of the RNA-binding protein HuD and posttranscriptional up-regulation of the GAP-43 gene during spatial memory</span></div><div class="casAuthors">Pascale, Alessia; Gusev, Pavel A.; Amadio, Marialaura; Dottorini, Tania; Govoni, Stefano; Alkon, Daniel L.; Quattrone, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1217-1222</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neuronal ELAV-like proteins (HuB, HuC, and HuD) are highly conserved RNA-binding proteins able to selectively assoc. with the 3' UTR of a subset of target mRNAs and increase their cytoplasmic stability and rate of translation.  We previously demonstrated the involvement of these proteins in learning, reporting that they undergo a sustained up-regulation in the hippocampus of mice trained in a spatial discrimination task.  Here, we extend this finding, showing that a similar up-regulation occurs in the hippocampus of rats trained in another spatial learning paradigm, the Morris water maze.  HuD, a strictly neuron-specific ELAV-like protein, is shown to increase after learning, with a preferential binding to the cytoskeletal fraction.  HuD up-regulation is assocd. with an enhancement of GAP-43 mRNA and protein levels, with an apparently increased HuD colocalization with the GAP-43 mRNA and an increased assocn. of neuronal ELAV-like proteins with the GAP-43 mRNA.  These learning-dependent biochem. events appear to be spatiotemporally controlled, because they do not occur in another brain region involved in learning, the retrosplenial cortex, and at the level of protein expression they show extinction 1 mo after training despite memory retention.  By contrast, HuD mRNA levels still remain increased after 1 mo in the CA1 region.  This persistence may have implications for long-term memory recall.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQJs6DY-0pg7Vg90H21EOLACvtfcHk0ljhHz2cDMAzjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlWjt70%253D&md5=3c4db9a1d503b91e1dd06267b55b1248</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0307674100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0307674100%26sid%3Dliteratum%253Aachs%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DGusev%26aufirst%3DP.%2BA.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DDottorini%26aufirst%3DT.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DAlkon%26aufirst%3DD.%2BL.%26aulast%3DQuattrone%26aufirst%3DA.%26atitle%3DIncrease%2520of%2520the%2520RNA-binding%2520protein%2520HuD%2520and%2520posttranscriptional%2520up-regulation%2520of%2520the%2520GAP-43%2520gene%2520during%2520spatial%2520memory%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D1217%26epage%3D1222%26doi%3D10.1073%2Fpnas.0307674100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akamatsu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuhashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakakibara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, H.</span></span> <span> </span><span class="NLM_article-title">The RNA-binding protein HuD regulates neuronal cell identity and maturation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4625</span>– <span class="NLM_lpage">4630</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407523102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.0407523102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=15764704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFCqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=4625-4630&author=W.+Akamatsuauthor=H.+Fujiharaauthor=T.+Mitsuhashiauthor=M.+Yanoauthor=S.+Shibataauthor=Y.+Hayakawaauthor=H.+J.+Okanoauthor=S.+Sakakibaraauthor=H.+Takanoauthor=T.+Takanoauthor=T.+Takahashiauthor=T.+Nodaauthor=H.+Okano&title=The+RNA-binding+protein+HuD+regulates+neuronal+cell+identity+and+maturation&doi=10.1073%2Fpnas.0407523102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The RNA-binding protein HuD regulates neuronal cell identity and maturation</span></div><div class="casAuthors">Akamatsu, Wado; Fujihara, Hiroaki; Mitsuhashi, Takayuki; Yano, Masato; Shibata, Shinsuke; Hayakawa, Yoshika; Okano, Hirotaka James; Sakakibara, Shin-ichi; Takano, Hiroshi; Takano, Toshiya; Takahashi, Takao; Noda, Tetsuo; Okano, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4625-4630</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neural Hu proteins (HuB/C/D) are RNA-binding proteins that have been shown to induce neuronal differentiation activity when overexpressed in immature neural progenitor cells or undifferentiated neuronal tumors.  Newly generated HuD-deficient mice exhibited a transient impaired-cranial-nerve-development phenotype at an early embryonic stage.  Adult HuD-deficient mice exhibited an abnormal hind-limb reflex and poor rotarod performance.  Anal. of neurosphere formation revealed that the no. and self-renewal capacity of the neural stem/progenitor cells were increased in HuD-deficient mice.  HuD-deficient primary neurospheres also generated a smaller no. of neurons.  Cohort anal. of the cellular proliferative activity by using BrdUrd and iododeoxuridine labeling revealed that the no. of differentiating quiescent cells in the embryonic cerebral wall was decreased.  Long-term administration of BrdUrd revealed that the no. of slowly dividing stem cells in the adult subventricular zone was increased in the HuD-deficient mice.  Taken together, the results suggest that HuD is required at multiple points during neuronal development, including neg. regulation of proliferative activity and neuronal cell-fate acquisition of neural stem/progenitor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKvkth9WQFKLVg90H21EOLACvtfcHk0liqRu80wTJa0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivFCqs7k%253D&md5=465cd4b39906116d684c343569464de1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407523102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407523102%26sid%3Dliteratum%253Aachs%26aulast%3DAkamatsu%26aufirst%3DW.%26aulast%3DFujihara%26aufirst%3DH.%26aulast%3DMitsuhashi%26aufirst%3DT.%26aulast%3DYano%26aufirst%3DM.%26aulast%3DShibata%26aufirst%3DS.%26aulast%3DHayakawa%26aufirst%3DY.%26aulast%3DOkano%26aufirst%3DH.%2BJ.%26aulast%3DSakakibara%26aufirst%3DS.%26aulast%3DTakano%26aufirst%3DH.%26aulast%3DTakano%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DNoda%26aufirst%3DT.%26aulast%3DOkano%26aufirst%3DH.%26atitle%3DThe%2520RNA-binding%2520protein%2520HuD%2520regulates%2520neuronal%2520cell%2520identity%2520and%2520maturation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D4625%26epage%3D4630%26doi%3D10.1073%2Fpnas.0407523102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassell, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossoll, W.</span></span> <span> </span><span class="NLM_article-title">The ALS disease protein TDP-43 is actively transported in motor neuron axons and regulates axon outgrowth</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3703</span>– <span class="NLM_lpage">3718</span>, <span class="refDoi"> DOI: 10.1093/hmg/dds205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fhmg%2Fdds205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=22641816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSrtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=3703-3718&author=C.+Falliniauthor=G.+J.+Bassellauthor=W.+Rossoll&title=The+ALS+disease+protein+TDP-43+is+actively+transported+in+motor+neuron+axons+and+regulates+axon+outgrowth&doi=10.1093%2Fhmg%2Fdds205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The ALS disease protein TDP-43 is actively transported in motor neuron axons and regulates axon outgrowth</span></div><div class="casAuthors">Fallini, Claudia; Bassell, Gary J.; Rossoll, Wilfried</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3703-3718</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disease specifically affecting cortical and spinal motor neurons.  Cytoplasmic inclusions contg. hyperphosphorylated and ubiquitinated TDP-43 are a pathol. hallmark of ALS, and mutations in the gene encoding TDP-43 have been directly linked to the development of the disease.  TDP-43 is a ubiquitous DNA/RNA-binding protein with a nuclear role in pre-mRNA splicing.  However, the selective vulnerability and axonal degeneration of motor neurons in ALS pose the question of whether TDP-43 may have an addnl. role in the regulation of the cytoplasmic and axonal fate of mRNAs, processes important for neuron function.  To investigate this possibility, we have characterized TDP-43 localization and dynamics in primary cultured motor neurons.  Using a combination of cell imaging and biochem. techniques, we demonstrate that TDP-43 is localized and actively transported in live motor neuron axons, and that it co-localizes with well-studied axonal mRNA-binding proteins.  Expression of the TDP-43 C-terminal fragment led to the formation of hyperphosphorylated and ubiquitinated inclusions in motor neuron cell bodies and neurites, and these inclusions specifically sequestered the mRNA-binding protein HuD.  Addnl., we showed that overexpression of full-length or mutant TDP-43 in motor neurons caused a severe impairment in axon outgrowth, which was dependent on the C-terminal protein-interacting domain of TDP-43.  Taken together, our results suggest a role of TDP-43 in the regulation of axonal growth, and suggest that impairment in the post-transcriptional regulation of mRNAs in the cytoplasm of motor neurons may be a major factor in the development of ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS_5CS-Bp0y7Vg90H21EOLACvtfcHk0liqRu80wTJa0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSrtLfO&md5=1e019bc6ecc209d527056b1f1d9cbe87</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fdds205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fdds205%26sid%3Dliteratum%253Aachs%26aulast%3DFallini%26aufirst%3DC.%26aulast%3DBassell%26aufirst%3DG.%2BJ.%26aulast%3DRossoll%26aufirst%3DW.%26atitle%3DThe%2520ALS%2520disease%2520protein%2520TDP-43%2520is%2520actively%2520transported%2520in%2520motor%2520neuron%2520axons%2520and%2520regulates%2520axon%2520outgrowth%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2012%26volume%3D21%26spage%3D3703%26epage%3D3718%26doi%3D10.1093%2Fhmg%2Fdds205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. W.</span></span> <span> </span><span class="NLM_article-title">Role of HuD in nervous system function and pathology</span>. <i>Front. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>S5</i></span>,  <span class="NLM_fpage">554</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.2741/S389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.2741%2FS389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtV2nt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=S5&publication_year=2013&pages=554-563&author=N.+Perrone-Bizzozeroauthor=C.+W.+Bird&title=Role+of+HuD+in+nervous+system+function+and+pathology&doi=10.2741%2FS389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Role of HuD in nervous system function and pathology</span></div><div class="casAuthors">Perrone-Bizzozero, Nora; Bird, Clark W.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Scholar Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">S5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">554-563</span>CODEN:
                <span class="NLM_cas:coden">FBSEAU</span>;
        ISSN:<span class="NLM_cas:issn">1945-0524</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Hu proteins are a family of RNA-binding proteins (RBPs) that are homologs of Drosophila ELAV, a protein required for nervous system development.  Three of these proteins (HuB, HuC, and HuD) are primarily expressed in neurons.  The fourth member, HuR is ubiquitously expressed in all tissues.  At the mol. level, Hu proteins are known to interact with AU-rich instability conferring sequences in the 3' UTR of specific target mRNAs, stabilizing the mRNAs.  These proteins are not only the best known mRNA stabilizers but also the earliest markers of the neuronal cell lineage.  Among the neuronal Hu proteins, HuD has been shown to accelerate neuronal differentiation and axonal outgrowth in neurons both in culture and in vivo.  In addn., HuD and other Hu proteins participate in synaptic plasticity mechanisms in the mature central nervous system and promote regeneration of peripheral nerves.  Furthermore, HuD has been implicated in pathol. conditions from neurodegenerative disorders such as Parkinson's and Alzheimer's disease to childhood brain tumors.  This review will focus on the involvement of HuD in nervous system function and pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYUHgaqYZNx7Vg90H21EOLACvtfcHk0lgxDZ9HdtwGEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2nt77L&md5=5ba942d739a9c9ae42b1cb3b051675ba</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2741%2FS389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252FS389%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%26aulast%3DBird%26aufirst%3DC.%2BW.%26atitle%3DRole%2520of%2520HuD%2520in%2520nervous%2520system%2520function%2520and%2520pathology%26jtitle%3DFront.%2520Biosci.%26date%3D2013%26volume%3DS5%26spage%3D554%26epage%3D563%26doi%3D10.2741%2FS389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hakak, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxbaum, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haroutunian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fienberg, A. A.</span></span> <span> </span><span class="NLM_article-title">Genome-wide expression analysis reveals dysregulation of myelination-related genes in chronic schizophrenia</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">4746</span>– <span class="NLM_lpage">4751</span>, <span class="refDoi"> DOI: 10.1073/pnas.081071198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.081071198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=11296301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtVant7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=4746-4751&author=Y.+Hakakauthor=J.+R.+Walkerauthor=C.+Liauthor=W.+H.+Wongauthor=K.+L.+Davisauthor=J.+D.+Buxbaumauthor=V.+Haroutunianauthor=A.+A.+Fienberg&title=Genome-wide+expression+analysis+reveals+dysregulation+of+myelination-related+genes+in+chronic+schizophrenia&doi=10.1073%2Fpnas.081071198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide expression analysis reveals dysregulation of myelination-related genes in chronic schizophrenia</span></div><div class="casAuthors">Hakak, Yaron; Walker, John R.; Li, Cheng; Wong, Wing Hung; Davis, Kenneth L.; Buxbaum, Joseph D.; Haroutunian, Vahram; Fienberg, Allen A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4746-4751</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neuropathol. and brain imaging studies suggest that schizophrenia may result from neurodevelopmental defects.  Cytoarchitectural studies indicate cellular abnormalities suggestive of a disruption in neuronal connectivity in schizophrenia, particularly in the dorsolateral prefrontal cortex.  Yet, the mol. mechanisms underlying these findings remain unclear.  To identify mol. substrates assocd. with schizophrenia, DNA microarray anal. was used to assay gene expression levels in postmortem dorsolateral prefrontal cortex of schizophrenic and control patients.  Genes detd. to have altered expression levels in schizophrenics relative to controls are involved in a no. of biol. processes, including synaptic plasticity, neuronal development, neurotransmission, and signal transduction.  Most notable was the differential expression of myelination-related genes suggesting a disruption in oligodendrocyte function in schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3XChhEHjfyrVg90H21EOLACvtfcHk0lgxDZ9HdtwGEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtVant7w%253D&md5=45717c744f07b3906b1221b14c0ab6ed</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.081071198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.081071198%26sid%3Dliteratum%253Aachs%26aulast%3DHakak%26aufirst%3DY.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DW.%2BH.%26aulast%3DDavis%26aufirst%3DK.%2BL.%26aulast%3DBuxbaum%26aufirst%3DJ.%2BD.%26aulast%3DHaroutunian%26aufirst%3DV.%26aulast%3DFienberg%26aufirst%3DA.%2BA.%26atitle%3DGenome-wide%2520expression%2520analysis%2520reveals%2520dysregulation%2520of%2520myelination-related%2520genes%2520in%2520chronic%2520schizophrenia%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D4746%26epage%3D4751%26doi%3D10.1073%2Fpnas.081071198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noureddine, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelly, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Walt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pericak-Vance, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. J.</span></span> <span> </span><span class="NLM_article-title">Association between the neuron-specific RNA-binding protein ELAVL4 and Parkinson disease</span>. <i>Hum Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1007/s00439-005-1259-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1007%2Fs00439-005-1259-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=15827745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlCnt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2005&pages=27-33&author=M.+A.+Noureddineauthor=X.+J.+Qinauthor=S.+A.+Oliveiraauthor=T.+J.+Skellyauthor=J.+van+der%0AWaltauthor=M.+A.+Hauserauthor=M.+A.+Pericak-Vanceauthor=J.+M.+Vanceauthor=Y.+J.+Li&title=Association+between+the+neuron-specific+RNA-binding+protein+ELAVL4+and+Parkinson+disease&doi=10.1007%2Fs00439-005-1259-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Association between the neuron-specific RNA-binding protein ELAVL4 and Parkinson disease</span></div><div class="casAuthors">Noureddine, Maher A.; Qin, Xue-Jun; Oliveira, Sofia A.; Skelly, Tara J.; van der Walt, Joelle; Hauser, Michael A.; Pericak-Vance, Margaret A.; Vance, Jeffery M.; Li, Yi-Ju</div><div class="citationInfo"><span class="NLM_cas:title">Human Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-33</span>CODEN:
                <span class="NLM_cas:coden">HUGEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-6717</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Inflammatory processes have been implicated in the cascade of events that lead to nerve cell death.  In the nervous system, a no. of genes involved in inflammation pathways are regulated post-transcriptionally via the interaction of their mRNAs with specific RNA-binding Hu proteins, the vertebrate homologs of the Drosophila ELAV (for embryonic lethal abnormal vision).  The gene encoding ELAVL4, a member of the Hu family of proteins, is located 2 Mb from the chromosome 1p linkage region peak for age-at-onset (AAO) of Parkinson disease (PD) (LOD = 3.41).  Nine single-nucleotide polymorphisms (SNPs) in ELAVL4 were genotyped for 266 multiplex families (1,223 samples).  Addnl. genotyping in 377 singleton families was performed for a subset of five SNPs (SNPs 1-5) that were not in linkage disequil.  SNP 2 (located in the first intron of ELAVL4) showed a strong significant assocn. with AAO of PD (P = 0.006), and SNP 5 (a coding SNP in ELAVL4) showed a moderately significant assocn. (P = 0.035).  Haplotype anal. revealed that the A-C haplotype at SNPs 2 and 3 has the strongest significant assocn. with AAO (P = 0.0001) among all combinations of two or three loci.  The A-C haplotype remained significant for AAO after the inclusion of the C allele at SNP 5 to this haplotype (A-C-C haplotype, P = 0.00018).  Although SNP 5 was found to assoc. with PD risk in the early-onset subset of PD families (at least one affected with AAO < 40 years, 60 families), we believe that it is a byproduct of its assocn. with AAO.  Taken together, these results suggest a potential role for ELAVL4 as a modifier gene for AAO of PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodgOlJw6z41bVg90H21EOLACvtfcHk0liV8SF0m2bQgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktlCnt7o%253D&md5=587d249c80c2a5e2d4a7c935365b92a8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00439-005-1259-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00439-005-1259-2%26sid%3Dliteratum%253Aachs%26aulast%3DNoureddine%26aufirst%3DM.%2BA.%26aulast%3DQin%26aufirst%3DX.%2BJ.%26aulast%3DOliveira%26aufirst%3DS.%2BA.%26aulast%3DSkelly%26aufirst%3DT.%2BJ.%26aulast%3Dvan%2Bder%2BWalt%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DM.%2BA.%26aulast%3DPericak-Vance%26aufirst%3DM.%2BA.%26aulast%3DVance%26aufirst%3DJ.%2BM.%26aulast%3DLi%26aufirst%3DY.%2BJ.%26atitle%3DAssociation%2520between%2520the%2520neuron-specific%2520RNA-binding%2520protein%2520ELAVL4%2520and%2520Parkinson%2520disease%26jtitle%3DHum%2520Genet.%26date%3D2005%26volume%3D117%26spage%3D27%26epage%3D33%26doi%3D10.1007%2Fs00439-005-1259-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkon, D. L.</span></span> <span> </span><span class="NLM_article-title">PKCε promotes HuD-Mediated Neprilysin mRNA Stability and Enhances Neprilysin-Induced Aβ Degradation in Brain Neurons</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e97756</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0097756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1371%2Fjournal.pone.0097756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24848988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGqt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=C.+S.+Limauthor=D.+L.+Alkon&title=PKC%CE%B5+promotes+HuD-Mediated+Neprilysin+mRNA+Stability+and+Enhances+Neprilysin-Induced+A%CE%B2+Degradation+in+Brain+Neurons&doi=10.1371%2Fjournal.pone.0097756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">PKCε promotes HuD-mediated neprilysin mRNA stability and enhances neprilysin-induced Aβ degradation in brain neurons</span></div><div class="casAuthors">Lim, Chol Seung; Alkon, Daniel L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e97756/1-e97756/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Amyloid-beta (Aβ) peptide accumulation in the brain is a pathol. hallmark of all forms of Alzheimer's disease.  An imbalance between Aβ prodn. and clearance from the brain may contribute to accumulation of neurotoxic Aβ and subsequent synaptic loss, which is the strongest correlate of the extent of memory loss in AD.  The activity of neprilysin (NEP), a potent Aβ-degrading enzyme, is decreased in the AD brain.  Expression of HuD, an mRNA-binding protein important for synaptogenesis and neuronal plasticity, is also decreased in the AD brain.  HuD is regulated by protein kinase Cε (PKCε), and we previously demonstrated that PKCε activation decreases Aβ levels.  We hypothesized that PKCε acts through HuD to stabilize NEP mRNA, modulate its localization, and support NEP activity.  Conversely, loss of PKCε-activated HuD in AD leads to decreased NEP activity and accumulation of Aβ.  Here we show that HuD is assocd. with NEP mRNA in cultures of human SK-N-SH cells.  Treatment with bryostatin, a PKCε-selective activator, enhanced NEP assocn. with HuD and increased NEP mRNA stability.  Activation of PKCε also increased NEP protein levels, increased NEP phosphorylation, and induced cell surface expression.  In addn., specific PKCε activation directly stimulated NEP activity, leading to degrdn. of a monomeric form of Aβ peptide and decreased Aβ neuronal toxicity, as measured by cell viability.  Bryostatin treatment also rescued Aβ-mediated inhibition of HuD-NEP mRNA binding, NEP protein expression, and NEP cell membrane translocation.  These results suggest that PKCe activation reduces Aβ by up-regulating, via the mRNA-binding protein HuD, Aβ-degrading enzymes such as NEP.  Thus, PKCε activation may have therapeutic efficacy for AD by reducing neurotoxic Aβ accumulation as well as having direct anti-apoptotic and synaptogenic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ0k9zM7N4yLVg90H21EOLACvtfcHk0liV8SF0m2bQgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGqt73M&md5=b4f8f7f89fdcddd04bbea84f68a7fc13</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0097756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0097756%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DAlkon%26aufirst%3DD.%2BL.%26atitle%3DPKC%25CE%25B5%2520promotes%2520HuD-Mediated%2520Neprilysin%2520mRNA%2520Stability%2520and%2520Enhances%2520Neprilysin-Induced%2520A%25CE%25B2%2520Degradation%2520in%2520Brain%2520Neurons%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0097756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haroutunian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasinetti, G. M.</span></span> <span> </span><span class="NLM_article-title">nELAV proteins alteration in Alzheimer’s disease brain: a novel putative target for amyloid-beta reverberating on AbetaPP processing</span>. <i>J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.3233/JAD-2009-0967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.3233%2FJAD-2009-0967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=19221430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BD1M3mtlWnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=409-419&author=M.+Amadioauthor=A.+Pascaleauthor=J.+Wangauthor=L.+Hoauthor=A.+Quattroneauthor=S.+Gandyauthor=V.+Haroutunianauthor=M.+Racchiauthor=G.+M.+Pasinetti&title=nELAV+proteins+alteration+in+Alzheimer%E2%80%99s+disease+brain%3A+a+novel+putative+target+for+amyloid-beta+reverberating+on+AbetaPP+processing&doi=10.3233%2FJAD-2009-0967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">nELAV proteins alteration in Alzheimer's disease brain: a novel putative target for amyloid-beta reverberating on AbetaPP processing</span></div><div class="casAuthors">Amadio Marialaura; Pascale Alessia; Wang Jun; Ho Lap; Quattrone Alessandro; Gandy Sam; Haroutunian Vahram; Racchi Marco; Pasinetti Giulio Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Alzheimer's disease : JAD</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">409-19</span>
        ISSN:<span class="NLM_cas:issn">1387-2877</span>.
    </div><div class="casAbstract">Neuronal ELAV (nELAV) proteins are RNA-binding proteins which play a physiological role in controlling gene expression in memory formation, and their alteration may contribute to cognitive impairment associated with neurodegenerative pathologies such as Alzheimer's disease (AD).  Indeed, we found that the content of nELAV proteins is significantly decreased along with clinical dementia progression in the hippocampi of AD brains, where it inversely correlates with the amount of amyloid-beta (Abeta).  To check the direct influence of Abeta on nELAV, we performed in vitro experiments using human SH-SY5Y cells, finding that Abeta(1-42) specifically determines nELAV proteins reduction.  Since ADAM10 mRNA has the predicted sequences targeted by nELAV, we investigated whether Abeta, through nELAV proteins, could originate a vicious circle affecting amyloid-beta protein precursor (AbetaPP) processing.  Immunoprecipitation experiments showed that indeed nELAV proteins bind to ADAM10 mRNA and that this binding is disrupted by Abeta(1-42) exposure, resulting in a decreased ADAM10 protein expression.  ADAM10 protein diminution was also found in AD hippocampi.  These data show for the first time the involvement of nELAV in AD pathology and suggest that their alteration may affect genes implicated in AbetaPP processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCmsmmtP0GCWBAiYWy6n3SfW6udTcc2eYYGkhg5Naf07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3mtlWnug%253D%253D&md5=581fc30d7c4fa95ae8d4b80fdc6a0dde</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3233%2FJAD-2009-0967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJAD-2009-0967%26sid%3Dliteratum%253Aachs%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DQuattrone%26aufirst%3DA.%26aulast%3DGandy%26aufirst%3DS.%26aulast%3DHaroutunian%26aufirst%3DV.%26aulast%3DRacchi%26aufirst%3DM.%26aulast%3DPasinetti%26aufirst%3DG.%2BM.%26atitle%3DnELAV%2520proteins%2520alteration%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%2520brain%253A%2520a%2520novel%2520putative%2520target%2520for%2520amyloid-beta%2520reverberating%2520on%2520AbetaPP%2520processing%26jtitle%3DJ.%2520Alzheimers%2520Dis.%26date%3D2009%26volume%3D16%26spage%3D409%26epage%3D419%26doi%3D10.3233%2FJAD-2009-0967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marchesi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombrita, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span> <span> </span><span class="NLM_article-title">PKC Activation Counteracts ADAM10 Deficit in HuD-Silenced Neuroblastoma Cells</span>. <i>J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.3233/JAD-160299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.3233%2FJAD-160299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=27472877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BC2s3ls1CisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2016&pages=535-547&author=N.+Marchesiauthor=M.+Amadioauthor=C.+Colombritaauthor=S.+Govoniauthor=A.+Rattiauthor=A.+Pascale&title=PKC+Activation+Counteracts+ADAM10+Deficit+in+HuD-Silenced+Neuroblastoma+Cells&doi=10.3233%2FJAD-160299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PKC Activation Counteracts ADAM10 Deficit in HuD-Silenced Neuroblastoma Cells</span></div><div class="casAuthors">Marchesi Nicoletta; Amadio Marialaura; Govoni Stefano; Pascale Alessia; Colombrita Claudia; Ratti Antonia; Colombrita Claudia; Ratti Antonia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Alzheimer's disease : JAD</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">535-47</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neuronal ELAV/Hu (nELAV) are RNA-binding proteins that mainly regulate gene expression by increasing the stability and/or translation rate of target mRNAs bearing ARE (adenine and uracil-rich elements) sequences.  Among nELAV target transcripts there is ADAM10, an α-secretase involved in the non-amyloidogenic processing of the amyloid-β protein precursor (AβPP) which leads to the production of the neuroprotective sAβPPα peptide.  The aim of this study was to evaluate if nELAV depletion affects ADAM10 expression in human SH-SY5Y neuroblastoma cells.  We also studied the effects of Bryostatin-1, a molecule able to activate nELAV protein cascade.  The specific HuD/nELAV gene silencing decreased both nELAV and ADAM10 protein contents; similar results were obtained by Aβ40 treatment in wild-type SH-SY5Y cells.  In HuD-silenced cells, the exposure to Bryostatin-1 counteracted both nELAV and ADAM10 proteins downregulation, by restoring nELAV/ADAM10 basal levels.  We also found that sAβPPα release, which seemed not to be compromised by Aβ40 challenge or HuD-silencing, was favored by Bryostatin-1.  Overall, these findings strongly suggest that a deficiency in nELAV content negatively affects ADAM10 expression and may play a role in neurodegenerative diseases, which may benefit by molecules activating ELAV cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSihkLH2ne6q7FU0A1HOshhfW6udTcc2eYYGkhg5Naf07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3ls1CisA%253D%253D&md5=21faa73ec391f8f0e69091c695df90ca</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3233%2FJAD-160299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJAD-160299%26sid%3Dliteratum%253Aachs%26aulast%3DMarchesi%26aufirst%3DN.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DColombrita%26aufirst%3DC.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DRatti%26aufirst%3DA.%26aulast%3DPascale%26aufirst%3DA.%26atitle%3DPKC%2520Activation%2520Counteracts%2520ADAM10%2520Deficit%2520in%2520HuD-Silenced%2520Neuroblastoma%2520Cells%26jtitle%3DJ.%2520Alzheimers%2520Dis.%26date%3D2016%26volume%3D54%26spage%3D535%26epage%3D547%26doi%3D10.3233%2FJAD-160299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelmohsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grammatikakis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noh, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martindale, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faghihi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlestedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troncoso, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Cabo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorospe, M.</span></span> <span> </span><span class="NLM_article-title">HuD regulates coding and noncoding RNA to induce APP→Aβ processing</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1401</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.04.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.celrep.2014.04.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24857657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=1401-1409&author=M.+J.+Kangauthor=K.+Abdelmohsenauthor=E.+R.+Hutchisonauthor=S.+J.+Mitchellauthor=I.+Grammatikakisauthor=R.+Guoauthor=J.+H.+Nohauthor=J.+L.+Martindaleauthor=X.+Yangauthor=E.+K.+Leeauthor=M.+A.+Faghihiauthor=C.+Wahlestedtauthor=J.+C.+Troncosoauthor=O.+Pletnikovaauthor=N.+Perrone-Bizzozeroauthor=S.+M.+Resnickauthor=R.+de+Caboauthor=M.+P.+Mattsonauthor=M.+Gorospe&title=HuD+regulates+coding+and+noncoding+RNA+to+induce+APP%E2%86%92A%CE%B2+processing&doi=10.1016%2Fj.celrep.2014.04.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">HuD Regulates Coding and Noncoding RNA to Induce APP→Aβ Processing</span></div><div class="casAuthors">Kang, Min-Ju; Abdelmohsen, Kotb; Hutchison, Emmette R.; Mitchell, Sarah J.; Grammatikakis, Ioannis; Guo, Rong; Noh, Ji Heon; Martindale, Jennifer L.; Yang, Xiaoling; Lee, Eun Kyung; Faghihi, Mohammad A.; Wahlestedt, Claes; Troncoso, Juan C.; Pletnikova, Olga; Perrone-Bizzozero, Nora; Resnick, Susan M.; de Cabo, Rafael; Mattson, Mark P.; Gorospe, Myriam</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1401-1409</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The primarily neuronal RNA-binding protein HuD is implicated in learning and memory.  Here, we report the identification of several HuD target transcripts linked to Alzheimer's disease (AD) pathogenesis.  HuD interacted with the 3' UTRs of APP mRNA (encoding amyloid precursor protein) and BACE1 mRNA (encoding β-site APP-cleaving enzyme 1) and increased the half-lives of these mRNAs.  HuD also assocd. with and stabilized the long noncoding (lnc)RNA BACE1AS, which partly complements BACE1 mRNA and enhances BACE1 expression.  Consistent with HuD promoting prodn. of APP and APP-cleaving enzyme, the levels of APP, BACE1, BACE1AS, and Aβ were higher in the brain of HuD-overexpressing mice.  Importantly, cortex (superior temporal gyrus) from patients with AD displayed significantly higher levels of HuD and, accordingly, elevated APP, BACE1, BACE1AS, and Aβ than did cortical tissue from healthy age-matched individuals.  We propose that HuD jointly promotes the prodn. of APP and the cleavage of its amyloidogenic fragment, Aβ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_4Ooj2xbBJLVg90H21EOLACvtfcHk0lgyWcn8eVAmGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslyit78%253D&md5=7bc526b55d6e59d3de164e44605ab3f4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.04.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.04.050%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DM.%2BJ.%26aulast%3DAbdelmohsen%26aufirst%3DK.%26aulast%3DHutchison%26aufirst%3DE.%2BR.%26aulast%3DMitchell%26aufirst%3DS.%2BJ.%26aulast%3DGrammatikakis%26aufirst%3DI.%26aulast%3DGuo%26aufirst%3DR.%26aulast%3DNoh%26aufirst%3DJ.%2BH.%26aulast%3DMartindale%26aufirst%3DJ.%2BL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DE.%2BK.%26aulast%3DFaghihi%26aufirst%3DM.%2BA.%26aulast%3DWahlestedt%26aufirst%3DC.%26aulast%3DTroncoso%26aufirst%3DJ.%2BC.%26aulast%3DPletnikova%26aufirst%3DO.%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%26aulast%3DResnick%26aufirst%3DS.%2BM.%26aulast%3Dde%2BCabo%26aufirst%3DR.%26aulast%3DMattson%26aufirst%3DM.%2BP.%26aulast%3DGorospe%26aufirst%3DM.%26atitle%3DHuD%2520regulates%2520coding%2520and%2520noncoding%2520RNA%2520to%2520induce%2520APP%25E2%2586%2592A%25CE%25B2%2520processing%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D7%26spage%3D1401%26epage%3D1409%26doi%3D10.1016%2Fj.celrep.2014.04.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subhadra, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeds, N. W.</span></span> <span> </span><span class="NLM_article-title">Neuroserpin up-regulation in the Alzheimer’s disease brain is associated with elevated thyroid hormone receptor-β1 and HuD expression</span>. <i>Neurochem. Int.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1016/j.neuint.2013.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.neuint.2013.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24036060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1KjtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=476-481&author=B.+Subhadraauthor=K.+Schallerauthor=N.+W.+Seeds&title=Neuroserpin+up-regulation+in+the+Alzheimer%E2%80%99s+disease+brain+is+associated+with+elevated+thyroid+hormone+receptor-%CE%B21+and+HuD+expression&doi=10.1016%2Fj.neuint.2013.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Neuroserpin up-regulation in the Alzheimer's disease brain is associated with elevated thyroid hormone receptor-β1 and HuD expression</span></div><div class="casAuthors">Subhadra, Bobban; Schaller, Kristin; Seeds, Nicholas W.</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry International</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">476-481</span>CODEN:
                <span class="NLM_cas:coden">NEUIDS</span>;
        ISSN:<span class="NLM_cas:issn">0197-0186</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Neuroserpin, the major inhibitor of tissue plasminogen activator (tPA) in brain, has been shown to be up-regulated in Alzheimer's disease (AD).  Inhibition of tPA activity leads to reduced brain levels of plasmin, one of the main enzymes responsible for the degrdn. and clearance of amyloid-beta and its plaques from the brain.  Thyroid hormone is one of the few factors known to enhance expression of neuroserpin in neurons.  Thyroid hormone acts on neurons by binding to its receptors THR1α and THR1β, which then function in the nucleus to up-regulate the expression of numerous genes including the RNA-binding protein HuD.  HuD acts post-transcriptionally to enhance expression of numerous proteins including neuroserpin by stabilizing their mRNAs.  A series of Alzheimer's disease brain tissues were compared to age-matched control brains for their expression of neuroserpin, THRβ1 and HuD by western blotting.  Alzheimer's disease brain tissues with elevated neuroserpin protein also showed increased expression of THRβ1 and HuD.  Pair-wise analyses showed significant correlation p-values between neuroserpin, THRβ1 and HuD levels; suggesting that the up-regulation of neuroserpin in Alzheimer's disease brain may result from an activation of the thyroid hormone response system in these individuals.  These findings provide evidence for a potential relationship between thyroid hormone disorders and Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqegCfn9IjJ2bVg90H21EOLACvtfcHk0lgyWcn8eVAmGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1KjtLfP&md5=ddb630161929307387da2686aff92da2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2013.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2013.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DSubhadra%26aufirst%3DB.%26aulast%3DSchaller%26aufirst%3DK.%26aulast%3DSeeds%26aufirst%3DN.%2BW.%26atitle%3DNeuroserpin%2520up-regulation%2520in%2520the%2520Alzheimer%25E2%2580%2599s%2520disease%2520brain%2520is%2520associated%2520with%2520elevated%2520thyroid%2520hormone%2520receptor-%25CE%25B21%2520and%2520HuD%2520expression%26jtitle%3DNeurochem.%2520Int.%26date%3D2013%26volume%3D63%26spage%3D476%26epage%3D481%26doi%3D10.1016%2Fj.neuint.2013.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N.</span></span> <span> </span><span class="NLM_article-title">Neuropsychiatric Implications Of RNA-Binding Proteins HuD And KSRP Revealed By Genome-Wide Identification Of Their Targets</span>. <i>Eur. Neuropsychopharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>, <span class="NLM_elocation-id">S721</span> <span class="refDoi"> DOI: 10.1016/j.euroneuro.2017.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.euroneuro.2017.06.036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&author=N.+Perrone-Bizzozero&title=Neuropsychiatric+Implications+Of+RNA-Binding+Proteins+HuD+And+KSRP+Revealed+By+Genome-Wide+Identification+Of+Their+Targets&doi=10.1016%2Fj.euroneuro.2017.06.036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.euroneuro.2017.06.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.euroneuro.2017.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%26atitle%3DNeuropsychiatric%2520Implications%2520Of%2520RNA-Binding%2520Proteins%2520HuD%2520And%2520KSRP%2520Revealed%2520By%2520Genome-Wide%2520Identification%2520Of%2520Their%2520Targets%26jtitle%3DEur.%2520Neuropsychopharmacol.%26date%3D2019%26volume%3D29%26doi%3D10.1016%2Fj.euroneuro.2017.06.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka Hall, T. M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for recognition of AU-rich element RNA by the HuD protein</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1038/84131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1038%2F84131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=11175903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtVShtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=141-145&author=X.+Wangauthor=T.+M.+Tanaka+Hall&title=Structural+basis+for+recognition+of+AU-rich+element+RNA+by+the+HuD+protein&doi=10.1038%2F84131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for recognition of AU-rich element RNA by the HuD protein</span></div><div class="casAuthors">Wang, Xiaoqiang; Hall, Traci M. Tanaka</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-145</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Hu proteins bind to adenosine-uridine (AU)-rich elements (AREs) in the 3' untranslated regions of many short-lived mRNAs, thereby stabilizing them.  Here the authors report the crystal structures of the first two RNA recognition motif (RRM) domains of the HuD protein in complex with an 11-nucleotide fragment of a class I ARE (the c-fos ARE; to 1.8 Å), and with an 11-nucleotide fragment of a class II ARE (the tumor necrosis factor α ARE; to 2.3 Å).  These structures reveal a consensus RNA recognition sequence that suggests a preference for pyrimidine-rich sequences and a requirement for a central uracil residue in the clustered AUUUA repeats found in class II AREs.  Comparison to structures of other RRM domain-nucleic acid complexes reveals two base recognition pockets in all the structures that interact with bases using residues in conserved ribonucleoprotein motifs and at the C-terminal ends of RRM domains.  Different conformations of nucleic acid can be bound by RRM domains by using different combinations of base recognition pockets and multiple RRM domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon8csOdxeA7rVg90H21EOLACvtfcHk0lgDk9D0pl8HgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtVShtbs%253D&md5=8176233d7440eb9dc0c6be318419841d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2F84131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F84131%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DTanaka%2BHall%26aufirst%3DT.%2BM.%26atitle%3DStructural%2520basis%2520for%2520recognition%2520of%2520AU-rich%2520element%2520RNA%2520by%2520the%2520HuD%2520protein%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D141%26epage%3D145%26doi%3D10.1038%2F84131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toba, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K.</span></span> <span> </span><span class="NLM_article-title">The third RNA recognition motif of Drosophila ELAV protein has a role in multimerization</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1390</span>– <span class="NLM_lpage">1399</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm1168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkm1168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=18203745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislKiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=1390-1399&author=G.+Tobaauthor=K.+White&title=The+third+RNA+recognition+motif+of+Drosophila+ELAV+protein+has+a+role+in+multimerization&doi=10.1093%2Fnar%2Fgkm1168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The third RNA recognition motif of Drosophila ELAV protein has a role in multimerization</span></div><div class="casAuthors">Toba, Gakuta; White, Kalpana</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1390-1399</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ELAV is a neuron-specific RNA-binding protein in Drosophila that is required for development and maintenance of neurons.  ELAV regulates alternative splicing of Neuroglian and erect wing (ewg) transcripts, and has been shown to form a multimeric complex on the last ewg intron.  The protein has three RNA recognition motifs (RRM1, 2 and 3) with a hinge region between RRM2 and 3.  In this study, we used the yeast two-hybrid system to det. the multimerization domain of ELAV.  Using deletion constructs, we mapped an interaction activity to a region contg. most of RRM3.  We found three conserved short sequences in RRM3 that were essential for the interaction, and also sufficient to give the interaction activity to RRM2 when introduced into it.  In our in vivo functional assay, a mutation in one of the three sequences showed reduced activity in splicing regulation, underlining the functional importance of multimerization.  However, RRM2 with the three RRM3 interaction sequences did not function as RRM3 in vivo, which suggested that multimerization is not the only function of RRM3.  Our results are consistent with a model in which RRM3 serves as a bi-functional domain that interacts with both RNA and protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPofMdm8XepbVg90H21EOLACvtfcHk0lgDk9D0pl8HgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislKiu7s%253D&md5=874693a3d7e6e26eca3c0d8bb87f42c6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm1168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm1168%26sid%3Dliteratum%253Aachs%26aulast%3DToba%26aufirst%3DG.%26aulast%3DWhite%26aufirst%3DK.%26atitle%3DThe%2520third%2520RNA%2520recognition%2520motif%2520of%2520Drosophila%2520ELAV%2520protein%2520has%2520a%2520role%2520in%2520multimerization%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D1390%26epage%3D1399%26doi%3D10.1093%2Fnar%2Fgkm1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baraglia, A. C.</span></span> <span> </span><span class="NLM_article-title">Multitarget drug design strategy applied to the system PKC/ELAV proteins/mRNA</span>. <i>Sci. Acta</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">80</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=76-80&author=A.+C.+Baraglia&title=Multitarget+drug+design+strategy+applied+to+the+system+PKC%2FELAV+proteins%2FmRNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaraglia%26aufirst%3DA.%2BC.%26atitle%3DMultitarget%2520drug%2520design%2520strategy%2520applied%2520to%2520the%2520system%2520PKC%252FELAV%2520proteins%252FmRNA%26jtitle%3DSci.%2520Acta%26date%3D2009%26volume%3D3%26spage%3D76%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pricl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaggeri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Identification of peptides with ELAV-like mRNA-stabilizing effect: an integrated in vitro/in silico approach</span>. <i>Chem. Biol. Drug. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">707</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1111%2Fcbdd.12117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=23398665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlOhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=707-714&author=M.+Amadioauthor=A.+Pascaleauthor=S.+Govoniauthor=E.+Lauriniauthor=S.+Priclauthor=R.+Gaggeriauthor=D.+Rossiauthor=S.+Collina&title=Identification+of+peptides+with+ELAV-like+mRNA-stabilizing+effect%3A+an+integrated+in+vitro%2Fin+silico+approach&doi=10.1111%2Fcbdd.12117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of peptides with ELAV-like mRNA-stabilizing effect: an integrated in vitro/in silico approach</span></div><div class="casAuthors">Amadio, Marialaura; Pascale, Alessia; Govoni, Stefano; Laurini, Erik; Pricl, Sabrina; Gaggeri, Raffaella; Rossi, Daniela; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">707-714</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Embryonic lethal abnormal vision (ELAV) proteins are RNA-binding proteins that bind specific adenine and uridine-rich elements mainly located in the 3'-untranslated region of target mRNAs, preventing their otherwise rapid degrdn. and thus increasing gene expression.  Starting from their previous discovery and applying an integrated in vitro/in silico approach, the authors report, here, a deeper understanding of the mRNA-stabilizing activity of four peptides derived from the ELAV proteins structure.  The stabilizing effect on the VEGF transcript (mRNAVEGF) exerted by each peptide, tested individually, was initially evaluated, and no effects were evidenced.  Hence, the biol. effects of all peptides couples were investigated.  Interestingly, in accordance with preliminary mol. dynamics results, only one of all possible peptide couples resulted highly effective in stabilizing mRNAVEGF.  These two peptides were thus identified as valuable starting point for designing small mols. with ELAV-mimicking properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopEudduyPj3LVg90H21EOLACvtfcHk0lizTBcaaFpGbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlOhs7g%253D&md5=d8c72a22107f0c7897ce7866967fa284</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12117%26sid%3Dliteratum%253Aachs%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DLaurini%26aufirst%3DE.%26aulast%3DPricl%26aufirst%3DS.%26aulast%3DGaggeri%26aufirst%3DR.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520peptides%2520with%2520ELAV-like%2520mRNA-stabilizing%2520effect%253A%2520an%2520integrated%2520in%2520vitro%252Fin%2520silico%2520approach%26jtitle%3DChem.%2520Biol.%2520Drug.%2520Des.%26date%3D2013%26volume%3D81%26spage%3D707%26epage%3D714%26doi%3D10.1111%2Fcbdd.12117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carnevale Baraglia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzolina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Small Peptides Derived from Embryonic Lethal Abnormal Vision Proteins Structure Showing RNA-Stabilizing Properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5017</span>– <span class="NLM_lpage">5019</span>, <span class="refDoi"> DOI: 10.1021/jm900741e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900741e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlSgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5017-5019&author=D.+Rossiauthor=M.+Amadioauthor=A.+Carnevale+Baragliaauthor=O.+Azzolinaauthor=A.+Rattiauthor=S.+Govoniauthor=A.+Pascaleauthor=S.+Collina&title=Discovery+of+Small+Peptides+Derived+from+Embryonic+Lethal+Abnormal+Vision+Proteins+Structure+Showing+RNA-Stabilizing+Properties&doi=10.1021%2Fjm900741e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Peptides Derived from Embryonic Lethal Abnormal Vision Proteins Structure Showing RNA-Stabilizing Properties</span></div><div class="casAuthors">Rossi, Daniela; Amadio, Marialaura; Carnevale Baraglia, Anna; Azzolina, Ornella; Ratti, Antonia; Govoni, Stefano; Pascale, Alessia; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5017-5019</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four peptides corresponding to highly conserved motifs within the first two RNA recognition motif-type domains of ELAV proteins were prepd., and their effect on the stability of NOVA-1 and VEGF ELAV-target mRNAs was evaluated.  Biol. results show that in the presence of phorbol 12-myristate 13-acetate (a PKC activator triggering the ELAV pathway), an equimolar mixt. of peptides induces a statistically significant stabilization of the selected transcripts, suggesting a synergic effect of the two stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIP-aXg_U6ebVg90H21EOLACvtfcHk0ljOHldajBQzlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlSgt7Y%253D&md5=9392e8393a0d6e1437cebc88a9bc0956</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm900741e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900741e%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DCarnevale%2BBaraglia%26aufirst%3DA.%26aulast%3DAzzolina%26aufirst%3DO.%26aulast%3DRatti%26aufirst%3DA.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Small%2520Peptides%2520Derived%2520from%2520Embryonic%2520Lethal%2520Abnormal%2520Vision%2520Proteins%2520Structure%2520Showing%2520RNA-Stabilizing%2520Properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5017%26epage%3D5019%26doi%3D10.1021%2Fjm900741e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span> <span> </span><span class="NLM_article-title">Diffusion-Ordered Spectroscopy and Saturation Transfer Difference NMR Spectroscopy Studies of Selective Interactions between ELAV Protein Fragments and an mRNA Target</span>. <i>Euro J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">6399</span>– <span class="NLM_lpage">6404</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201403014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fejoc.201403014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=6399-6404&author=F.+Vasileauthor=D.+Rossiauthor=S.+Collinaauthor=D.+Potenza&title=Diffusion-Ordered+Spectroscopy+and+Saturation+Transfer+Difference+NMR+Spectroscopy+Studies+of+Selective+Interactions+between+ELAV+Protein+Fragments+and+an+mRNA+Target&doi=10.1002%2Fejoc.201403014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201403014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201403014%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DCollina%26aufirst%3DS.%26aulast%3DPotenza%26aufirst%3DD.%26atitle%3DDiffusion-Ordered%2520Spectroscopy%2520and%2520Saturation%2520Transfer%2520Difference%2520NMR%2520Spectroscopy%2520Studies%2520of%2520Selective%2520Interactions%2520between%2520ELAV%2520Protein%2520Fragments%2520and%2520an%2520mRNA%2520Target%26jtitle%3DEuro%2520J.%2520Org.%2520Chem.%26date%3D2014%26volume%3D29%26spage%3D6399%26epage%3D6404%26doi%3D10.1002%2Fejoc.201403014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nasti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unver, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Compounds Interfering with Embryonic Lethal Abnormal Vision (ELAV) Protein-RNA Complexes: An Avenue for Discovering New Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8257</span>– <span class="NLM_lpage">8267</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01871</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01871" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Knt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8257-8267&author=R.+Nastiauthor=D.+Rossiauthor=M.+Amadioauthor=A.+Pascaleauthor=M.+Y.+Unverauthor=A.+K.+H.+Hirschauthor=S.+Collina&title=Compounds+Interfering+with+Embryonic+Lethal+Abnormal+Vision+%28ELAV%29+Protein-RNA+Complexes%3A+An+Avenue+for+Discovering+New+Drugs&doi=10.1021%2Facs.jmedchem.6b01871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Compounds Interfering with Embryonic Lethal Abnormal Vision (ELAV) Protein-RNA Complexes: An Avenue for Discovering New Drugs</span></div><div class="casAuthors">Nasti, Rita; Rossi, Daniela; Amadio, Marialaura; Pascale, Alessia; Unver, M. Yagiz; Hirsch, Anna K. H.; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8257-8267</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  RNA-binding proteins play a key role in post-transcriptional processes.  Among these proteins, embryonic lethal abnormal vision (ELAV) proteins are among the best described.  ELAV proteins predominantly act as pos. regulators of gene expression, and their dysregulation is involved in several pathologies, such as cancer, inflammation, and neurodegenerative diseases.  Only a few structurally unrelated compds. interfering with ELAV protein-mRNA complexes have been identified by applying high-throughput screening approaches.  Considering the structural diversity of the compds. discovered so far and the different techniques employed for screening their ability to interfere with ELAV protein-mRNA complexes, drawing conclusions from structure-activity relationships remains a challenge.  The authors performed docking studies to understand the interactions of compds. reported over the past decade to be inhibitors of ELAV proteins and to evaluate the potential of computer-aided drug design to target this family of proteins for further drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzpb1q7pqzprVg90H21EOLACvtfcHk0lgCJmSvLQXImA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Knt7g%253D&md5=d320ddeb667d511959e882e9ab6681d6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01871%26sid%3Dliteratum%253Aachs%26aulast%3DNasti%26aufirst%3DR.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DUnver%26aufirst%3DM.%2BY.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DCompounds%2520Interfering%2520with%2520Embryonic%2520Lethal%2520Abnormal%2520Vision%2520%2528ELAV%2529%2520Protein-RNA%2520Complexes%253A%2520An%2520Avenue%2520for%2520Discovering%2520New%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8257%26epage%3D8267%26doi%3D10.1021%2Facs.jmedchem.6b01871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moraca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talarico, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagliardi, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distinto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maccioni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tramontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artese, A.</span></span> <span> </span><span class="NLM_article-title">Novel natural non-nucleoside inhibitors of HIV-1 reverse transcriptase identified by shape- and structure-based virtual screening techniques</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.10.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.ejmech.2018.10.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=30342421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFOns7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2019&pages=1-10&author=G.+Costaauthor=R.+Roccaauthor=A.+Coronaauthor=N.+Grandiauthor=F.+Moracaauthor=I.+Romeoauthor=C.+Talaricoauthor=M.+G.+Gagliardiauthor=F.+A.+Ambrosioauthor=F.+Ortusoauthor=S.+Alcaroauthor=S.+Distintoauthor=E.+Maccioniauthor=E.+Tramontanoauthor=A.+Artese&title=Novel+natural+non-nucleoside+inhibitors+of+HIV-1+reverse+transcriptase+identified+by+shape-+and+structure-based+virtual+screening+techniques&doi=10.1016%2Fj.ejmech.2018.10.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel natural non-nucleoside inhibitors of HIV-1 reverse transcriptase identified by shape- and structure-based virtual screening techniques</span></div><div class="casAuthors">Costa, Giosue; Rocca, Roberta; Corona, Angela; Grandi, Nicole; Moraca, Federica; Romeo, Isabella; Talarico, Carmine; Gagliardi, Maria Giovanna; Ambrosio, Francesca Alessandra; Ortuso, Francesco; Alcaro, Stefano; Distinto, Simona; Maccioni, Elias; Tramontano, Enzo; Artese, Anna</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In this work the authors report a parallel application of both docking- and shape-based virtual screening (VS) methods, followed by Mol. Dynamics simulations (MDs), for discovering new compds. able to inhibit the human immunodeficiency virus type 1 (HIV-1) reverse transcriptase (RT) RNA-dependent DNA polymerase activity.  Specifically, the authors screened more than 143000 natural compds. com. available in the ZINC database against the best five RT crystallog. models, taking into account the five approved NNRTIs as query compds.  As a result, 20 hit mols. were selected and tested on biochem. assays for the inhibition of the RNA dependent DNA polymerase RT function and, among them, an indoline pyrrolidine (hit 1), an indonyl piperazine (hit 2) and an indolyl indolinone (hit 3) derivs. were identified as novel nonnucleoside RT inhibitors in the low micromolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxkhVzeZgmwrVg90H21EOLACvtfcHk0lj46TgNJgF9AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFOns7zM&md5=c9de900cbade80d59e06b4843e2db647</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.029%26sid%3Dliteratum%253Aachs%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DRocca%26aufirst%3DR.%26aulast%3DCorona%26aufirst%3DA.%26aulast%3DGrandi%26aufirst%3DN.%26aulast%3DMoraca%26aufirst%3DF.%26aulast%3DRomeo%26aufirst%3DI.%26aulast%3DTalarico%26aufirst%3DC.%26aulast%3DGagliardi%26aufirst%3DM.%2BG.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DOrtuso%26aufirst%3DF.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DDistinto%26aufirst%3DS.%26aulast%3DMaccioni%26aufirst%3DE.%26aulast%3DTramontano%26aufirst%3DE.%26aulast%3DArtese%26aufirst%3DA.%26atitle%3DNovel%2520natural%2520non-nucleoside%2520inhibitors%2520of%2520HIV-1%2520reverse%2520transcriptase%2520identified%2520by%2520shape-%2520and%2520structure-based%2520virtual%2520screening%2520techniques%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D161%26spage%3D1%26epage%3D10%26doi%3D10.1016%2Fj.ejmech.2018.10.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ripphausen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nisius, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltason, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Quo vadis, virtual screening? A comprehensive survey of prospective applications</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8461</span>– <span class="NLM_lpage">8467</span>, <span class="refDoi"> DOI: 10.1021/jm101020z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101020z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GqtrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8461-8467&author=P.+Ripphausenauthor=B.+Nisiusauthor=L.+Peltasonauthor=J.+Bajorath&title=Quo+vadis%2C+virtual+screening%3F+A+comprehensive+survey+of+prospective+applications&doi=10.1021%2Fjm101020z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Quo Vadis, Virtual Screening? A Comprehensive Survey of Prospective Applications</span></div><div class="casAuthors">Ripphausen, Peter; Nisius, Britta; Peltason, Lisa; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8461-8467</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Our anal. has revealed that the majority of prospective virtual screening (VS) studies have thus far been reported in only a small no. of journals.  In addn., we have found that docking investigations clearly dominate the prospective VS field at present.  We see an increasing no. of successful VS studies that are based on docking into homol. models, and the potency of such identified hits is on av. higher than for hits identified by docking into X-ray structures, although model-based docking calcns. are more approx. in nature.  In this context, it is interesting to note that NMR structures have thus far only rarely been used in practical VS applications.  Currently there are three times more structure-based virtual screening (SBVS) than ligand-based virtual screening (LBVS) applications available, but LBVS methods identify hits that are on av. more potent.  In particular, nearly all of the still limited no. of LBVS investigations that combine 2D and 3D components yield potent hits.  Furthermore, kinases, G-protein-coupled receptors (GPCRs), and proteases are the most popular VS targets.  For kinases and GPCRs, potent submicromolar VS hits are frequently obtained, in contrast to proteases, where the majority of reported hits are only weakly potent.  However, protein kinases and proteases only represent ∼36 % of all enzymes targeted by VS, and more than 50 % of these enzymes do not belong to the five major families.  Taken together, the results of our anal. have provided us with a rather detailed view of the prospective VS landscape and there have certainly been a few surprises.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHMK13XT0tD7Vg90H21EOLACvtfcHk0lj46TgNJgF9AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GqtrnK&md5=c20f82a1869c0ea176b177bc74ad92a4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm101020z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101020z%26sid%3Dliteratum%253Aachs%26aulast%3DRipphausen%26aufirst%3DP.%26aulast%3DNisius%26aufirst%3DB.%26aulast%3DPeltason%26aufirst%3DL.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DQuo%2520vadis%252C%2520virtual%2520screening%253F%2520A%2520comprehensive%2520survey%2520of%2520prospective%2520applications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8461%26epage%3D8467%26doi%3D10.1021%2Fjm101020z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Romero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Sastre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Computer-Aided Discovery and Characterization of Novel Ebola Virus Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3582</span>– <span class="NLM_lpage">3594</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntF2nt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3582-3594&author=S.+J.+Capuzziauthor=W.+Sunauthor=E.+N.+Muratovauthor=C.+Mart%C3%ADnez-Romeroauthor=S.+Heauthor=W.+Zhuauthor=H.+Liauthor=G.+Tawaauthor=E.+G.+Fisherauthor=M.+Xuauthor=P.+Shinnauthor=X.+Qiuauthor=A.+Garc%C3%ADa-Sastreauthor=W.+Zhengauthor=A.+Tropsha&title=Computer-Aided+Discovery+and+Characterization+of+Novel+Ebola+Virus+Inhibitors&doi=10.1021%2Facs.jmedchem.8b00035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-Aided Discovery and Characterization of Novel Ebola Virus Inhibitors</span></div><div class="casAuthors">Capuzzi, Stephen J.; Sun, Wei; Muratov, Eugene N.; Martinez-Romero, Carles; He, Shihua; Zhu, Wenjun; Li, Hao; Tawa, Gregory; Fisher, Ethan G.; Xu, Miao; Shinn, Paul; Qiu, Xiangguo; Garcia-Sastre, Adolfo; Zheng, Wei; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3582-3594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Ebola virus (EBOV) causes severe human infection that lacks effective treatment.  A recent screen identified a series of compds. that block EBOV-like particle entry into human cells.  Using data from this screen, quant. structure-activity relationship models were built and employed for virtual screening of a ∼17 million compd. library.  Exptl. testing of 102 hits yielded 14 compds. with IC50 values under 10 μM, including several sub-micromolar inhibitors, and more than 10-fold selectivity against host cytotoxicity.  These confirmed hits include FDA-approved drugs and clin. candidates with non-antiviral indications, as well as compds. with novel scaffolds and no previously known bioactivity.  Five selected hits inhibited BSL-4 live-EBOV infection in a dose-dependent manner, including vindesine (0.34 μM).  Addnl. studies of these novel anti-EBOV compds. revealed their mechanisms of action, including the inhibition of NPC1 protein, cathepsin B/L, and lysosomal function.  Compds. identified in this study are among the most potent and well-characterized anti-EBOV inhibitors reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPJn7FlyPoBbVg90H21EOLACvtfcHk0lj1_Oo4szKzVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntF2nt78%253D&md5=9091a9598c7d8c40539bcd5b73c683df</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00035%26sid%3Dliteratum%253Aachs%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DMart%25C3%25ADnez-Romero%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DTawa%26aufirst%3DG.%26aulast%3DFisher%26aufirst%3DE.%2BG.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DGarc%25C3%25ADa-Sastre%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DComputer-Aided%2520Discovery%2520and%2520Characterization%2520of%2520Novel%2520Ebola%2520Virus%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3582%26epage%3D3594%26doi%3D10.1021%2Facs.jmedchem.8b00035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identifying novel selective non-nucleoside DNA methyltransferase 1 inhibitors through docking-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9028</span>– <span class="NLM_lpage">9041</span>, <span class="refDoi"> DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+Chenauthor=Y.+Wangauthor=W.+Zhouauthor=S.+Liauthor=J.+Pengauthor=Z.+Shiauthor=J.+Huauthor=Y.+C.+Liuauthor=H.+Dingauthor=Y.+Linauthor=L.+Liauthor=S.+Chengauthor=J.+Liuauthor=T.+Luauthor=H.+Jiangauthor=B.+Liuauthor=M.+Zhengauthor=C.+Luo&title=Identifying+novel+selective+non-nucleoside+DNA+methyltransferase+1+inhibitors+through+docking-based+virtual+screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0lj1_Oo4szKzVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520novel%2520selective%2520non-nucleoside%2520DNA%2520methyltransferase%25201%2520inhibitors%2520through%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peters, J.-U.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology - foe or friend?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8955</span>– <span class="NLM_lpage">8971</span>, <span class="refDoi"> DOI: 10.1021/jm400856t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400856t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8955-8971&author=J.-U.+Peters&title=Polypharmacology+-+foe+or+friend%3F&doi=10.1021%2Fjm400856t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology - Foe or Friend?</span></div><div class="casAuthors">Peters, Jens-Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8955-8971</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Polypharmacol. describes the activity of compds. at multiple targets.  Current research focuses on two aspects of polypharmacol.: (1) unintended polypharmacol. can lead to adverse effects; (2) polypharmacol. across several disease-relevant targets can improve therapeutic efficacy, prevent drug resistance, or reduce therapeutic-target-related adverse effects.  This perspective reviews these interconnected aspects of polypharmacol.  The first part discusses the relevance of polypharmacol. for the safety of drugs, the mitigation of safety risks, and methods to identify polypharmacol. compds. early in the drug discovery process.  The second part discusses the advantages of polypharmacol. in the treatment of multigenic diseases and infections, and opportunities for drug discovery and drug repurposing.  This perspective aims to provide a balanced view on polypharmacol., which can compromise the safety of drugs, but can also confer superior efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5vnFGPfTnGLVg90H21EOLACvtfcHk0lj1_Oo4szKzVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO&md5=f4aeb6efddd4bfdf4e94656303323cba</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm400856t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400856t%26sid%3Dliteratum%253Aachs%26aulast%3DPeters%26aufirst%3DJ.-U.%26atitle%3DPolypharmacology%2520-%2520foe%2520or%2520friend%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8955%26epage%3D8971%26doi%3D10.1021%2Fjm400856t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, T.</span></span> <span> </span><span class="NLM_article-title">NMR spectroscopy techniques for screening and identifying ligand binding to protein receptors</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">864</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1002/anie.200390233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fanie.200390233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitFWltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=864-890&author=B.+Meyerauthor=T.+Peters&title=NMR+spectroscopy+techniques+for+screening+and+identifying+ligand+binding+to+protein+receptors&doi=10.1002%2Fanie.200390233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">NMR spectroscopy techniques for screening and identifying ligand binding to protein receptors</span></div><div class="casAuthors">Meyer, Bernd; Peters, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">864-890</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Picking the locks of protein binding: NMR spectroscopic techniques for the characterization of binding processes have recently gained much attention.  Binding epitopes on the ligands and on the protein can be identified easily without the aid of crystal structures (see scheme).  Hereby, rational design of new drugs has become more accessible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNviqXqZtItbVg90H21EOLACvtfcHk0ljrhYlb1HMmsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitFWltL4%253D&md5=e0b38bf974639d9414fb57e8d0def3e1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.200390233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200390233%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DB.%26aulast%3DPeters%26aufirst%3DT.%26atitle%3DNMR%2520spectroscopy%2520techniques%2520for%2520screening%2520and%2520identifying%2520ligand%2520binding%2520to%2520protein%2520receptors%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26spage%3D864%26epage%3D890%26doi%3D10.1002%2Fanie.200390233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sattin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Functionalized 2-(4-Hydroxyphenyl)-3-methylbenzofuran Allosteric Modulators of Hsp90 Activity</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3349</span>– <span class="NLM_lpage">3364</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201600420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fejoc.201600420" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2016&pages=3349-3364&author=S.+Sattinauthor=M.+Panzaauthor=F.+Vasileauthor=F.+Berniauthor=G.+Gotiauthor=J.+Taoauthor=E.+Moroniauthor=D.+Agardauthor=G.+Colomboauthor=A.+Bernardi&title=Synthesis+of+Functionalized+2-%284-Hydroxyphenyl%29-3-methylbenzofuran+Allosteric+Modulators+of+Hsp90+Activity&doi=10.1002%2Fejoc.201600420"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201600420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201600420%26sid%3Dliteratum%253Aachs%26aulast%3DSattin%26aufirst%3DS.%26aulast%3DPanza%26aufirst%3DM.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DBerni%26aufirst%3DF.%26aulast%3DGoti%26aufirst%3DG.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DMoroni%26aufirst%3DE.%26aulast%3DAgard%26aufirst%3DD.%26aulast%3DColombo%26aufirst%3DG.%26aulast%3DBernardi%26aufirst%3DA.%26atitle%3DSynthesis%2520of%2520Functionalized%25202-%25284-Hydroxyphenyl%2529-3-methylbenzofuran%2520Allosteric%2520Modulators%2520of%2520Hsp90%2520Activity%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2016%26volume%3D20%26spage%3D3349%26epage%3D3364%26doi%3D10.1002%2Fejoc.201600420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guzzetti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arosio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tringali, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piarulli, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gennari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pignataro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvisi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span> <span> </span><span class="NLM_article-title">Insights into the Binding of Cyclic RGD Peptidomimetics to α 5 β 1 Integrin by using Live-Cell NMR And Computational Studies</span>. <i>ChemistryOpen</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1002/open.201600112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fopen.201600112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=28168158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVGhtL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=128-136&author=I.+Guzzettiauthor=M.+Civeraauthor=F.+Vasileauthor=D.+Arosioauthor=C.+Tringaliauthor=U.+Piarulliauthor=C.+Gennariauthor=L.+Pignataroauthor=L.+Belvisiauthor=D.+Potenza&title=Insights+into+the+Binding+of+Cyclic+RGD+Peptidomimetics+to+%CE%B1+5+%CE%B2+1+Integrin+by+using+Live-Cell+NMR+And+Computational+Studies&doi=10.1002%2Fopen.201600112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Binding of Cyclic RGD Peptidomimetics to α5β1 Integrin by using Live-Cell NMR And Computational Studies</span></div><div class="casAuthors">Guzzetti, Ileana; Civera, Monica; Vasile, Francesca; Arosio, Daniela; Tringali, Cristina; Piarulli, Umberto; Gennari, Cesare; Pignataro, Luca; Belvisi, Laura; Potenza, Donatella</div><div class="citationInfo"><span class="NLM_cas:title">ChemistryOpen</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-136</span>CODEN:
                <span class="NLM_cas:coden">CHOPCK</span>;
        ISSN:<span class="NLM_cas:issn">2191-1363</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The interaction of a small library of cyclic DKP-RGD peptidomimetics with α5β1 integrin has been investigated by means of an integrated exptl. and computational approach.  Bioaffinity NMR techniques, including satn. transfer difference (STD) and transferred NOESY, were applied to the ligands in a suspension of intact MDA-MB-231 breast cancer cells, in which integrin α5β1 is highly expressed.  The NMR data were compared with the docking calcns. of the RGD ligands in the crystal structure of the α5β1 binding site, and were integrated with competitive binding assays to the purified α5β1 integrin.  Ligand binding epitopes involve protons of both the RGD moiety and the DKP scaffold, although the stereochem. and the functionalization of the DKP scaffold as well as the macrocycle conformation det. a great variability in the interaction.  The ligand showing the highest no. of STD signals is also the most potent α5β1 ligand of the series, displaying a nanomolar IC50 value.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc6FZSkfXJnLVg90H21EOLACvtfcHk0ljrhYlb1HMmsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVGhtL%252FE&md5=4ca0c3658b32164a78a6cbe33faa6859</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fopen.201600112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fopen.201600112%26sid%3Dliteratum%253Aachs%26aulast%3DGuzzetti%26aufirst%3DI.%26aulast%3DCivera%26aufirst%3DM.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DArosio%26aufirst%3DD.%26aulast%3DTringali%26aufirst%3DC.%26aulast%3DPiarulli%26aufirst%3DU.%26aulast%3DGennari%26aufirst%3DC.%26aulast%3DPignataro%26aufirst%3DL.%26aulast%3DBelvisi%26aufirst%3DL.%26aulast%3DPotenza%26aufirst%3DD.%26atitle%3DInsights%2520into%2520the%2520Binding%2520of%2520Cyclic%2520RGD%2520Peptidomimetics%2520to%2520%25CE%25B1%25205%2520%25CE%25B2%25201%2520Integrin%2520by%2520using%2520Live-Cell%2520NMR%2520And%2520Computational%2520Studies%26jtitle%3DChemistryOpen%26date%3D2017%26volume%3D6%26spage%3D128%26epage%3D136%26doi%3D10.1002%2Fopen.201600112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Civera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vettraino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosdocimi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvisi, L.</span></span> <span> </span><span class="NLM_article-title">Exploring E-cadherin-peptidomimetics interaction using NMR and computational studies</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e1007041</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1007041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1371%2Fjournal.pcbi.1007041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=31158220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFSgs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=M.+Civeraauthor=F.+Vasileauthor=D.+Potenzaauthor=C.+Colomboauthor=S.+Parenteauthor=C.+Vettrainoauthor=T.+Prosdocimiauthor=E.+Parisiniauthor=L.+Belvisi&title=Exploring+E-cadherin-peptidomimetics+interaction+using+NMR+and+computational+studies&doi=10.1371%2Fjournal.pcbi.1007041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring E-cadherin-peptidomimetics interaction using NMR and computational studies</span></div><div class="casAuthors">Civera, Monica; Vasile, Francesca; Potenza, Donatella; Colombo, Cinzia; Parente, Sara; Vettraino, Chiara; Prosdocimi, Tommaso; Parisini, Emilio; Elvisi, Laura</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1007041</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Cadherins are homophilic cell-cell adhesion mols. whose aberrant expression has often been shown to correlate with different stages of tumor progression.  In this work, we investigate the interaction of two peptidomimetic ligands with the extracellular portion of human E-cadherin using a combination of NMR and computational techniques.  Both ligands have been previously developed as mimics of the tetrapeptide sequence Asp1-Trp2-Val3- Ile4 of the cadherin adhesion arm, and have been shown to inhibit E-cadherin-mediated adhesion in epithelial ovarian cancer cells with millimolar potency.  To sample a set of possible interactions of these ligands with the E-cadherin extracellular portion, STD-NMR expts. in the presence of two slightly different constructs, the wild type E-cadherin-EC1-EC2 fragment and the truncated E-cadherin-(Val3)-EC1-EC2 fragment, were carried out at three temps.  Depending on the protein construct, a different binding epitope of the ligand and also a different temp. effect on STD signals were obsd., both suggesting an involvement of the Asp1-Trp2 protein sequence among all the possible binding events.  To interpret the exptl. results at the at. level and to probe the role of the cadherin adhesion arm in the dynamic interaction with the peptidomimetic ligand, a computational protocol based on docking calcns. and mol. dynamics simulations was applied.  In agreement with NMR data, the simulations at different temps. unveil high variability/ dynamism in ligand-cadherin binding, thus explaining the differences in ligand binding epi- topes.  In particular, the modulation of the signals seems to be dependent on the protein flex- ibility, esp. at the level of the adhesive arm, which appears to participate in the interaction with the ligand.  Overall, these results will help the design of novel cadherin inhibi- tors that might prevent the swap dimer formation by targeting both the Trp2 binding pocket and the adhesive arm residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsFnpAtN6EHrVg90H21EOLACvtfcHk0lgPAiaD7MkPaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSgs73F&md5=b6d9019e697e508f49b044190048d42f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007041%26sid%3Dliteratum%253Aachs%26aulast%3DCivera%26aufirst%3DM.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DColombo%26aufirst%3DC.%26aulast%3DParente%26aufirst%3DS.%26aulast%3DVettraino%26aufirst%3DC.%26aulast%3DProsdocimi%26aufirst%3DT.%26aulast%3DParisini%26aufirst%3DE.%26aulast%3DBelvisi%26aufirst%3DL.%26atitle%3DExploring%2520E-cadherin-peptidomimetics%2520interaction%2520using%2520NMR%2520and%2520computational%2520studies%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2019%26volume%3D15%26doi%3D10.1371%2Fjournal.pcbi.1007041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Della
Volpe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queirolo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unver, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jumde, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giacomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Novel Compounds Targeting the RNA-Binding Protein HuR. Structure-Based Design, Synthesis, and Interaction Studies</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00600</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00600" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVKmu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=615-620&author=S.+Della%0AVolpeauthor=R.+Nastiauthor=M.+Queiroloauthor=M.+Y.+Unverauthor=V.+K.+Jumdeauthor=A.+D%C3%B6mlingauthor=F.+Vasileauthor=D.+Potenzaauthor=F.+A.+Ambrosioauthor=G.+Costaauthor=S.+Alcaroauthor=C.+Zucalauthor=A.+Provenzaniauthor=M.+Di+Giacomoauthor=D.+Rossiauthor=A.+K.+H.+Hirschauthor=S.+Collina&title=Novel+Compounds+Targeting+the+RNA-Binding+Protein+HuR.+Structure-Based+Design%2C+Synthesis%2C+and+Interaction+Studies&doi=10.1021%2Facsmedchemlett.8b00600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Compounds Targeting the RNA-Binding Protein HuR. Structure-Based Design, Synthesis, and Interaction Studies</span></div><div class="casAuthors">Della Volpe, Serena; Nasti, Rita; Queirolo, Michele; Unver, M. Yagiz; Jumde, Varsha K.; Domling, Alexander; Vasile, Francesca; Potenza, Donatella; Ambrosio, Francesca Alessandra; Costa, Giosue; Alcaro, Stefano; Zucal, Chiara; Provenzani, Alessandro; Di Giacomo, Marcello; Rossi, Daniela; Hirsch, Anna K. H.; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">615-620</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The key role of RNA-binding proteins (RBPs) in regulating post-transcriptional processes and their involvement in several pathologies (i.e., cancer and neurodegeneration) have highlighted their potential as therapeutic targets.  In this scenario, Embryonic Lethal Abnormal Vision (ELAV) or Hu proteins and their complexes with target mRNAs have been gaining growing attention.  Compds. able to modulate the complex stability could constitute an innovative pharmacol. strategy for the treatment of numerous diseases.  Nevertheless, medicinal-chem. efforts aimed at developing such compds. are still at an early stage.  As part of our ongoing research in this field, we hereby present the rational design and synthesis of structurally novel HuR ligands, potentially acting as HuR-RNA interferers.  The following assessment of the structural features of their interaction with HuR, combining satn.-transfer difference NMR and in silico studies, provides a guide for further research on the development of new effective interfering compds. of the HuR-RNA complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokPXf_kMvpfLVg90H21EOLACvtfcHk0lgPAiaD7MkPaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVKmu78%253D&md5=abdb2ca797160f7e2c122ea791741f7c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00600%26sid%3Dliteratum%253Aachs%26aulast%3DDella%2BVolpe%26aufirst%3DS.%26aulast%3DNasti%26aufirst%3DR.%26aulast%3DQueirolo%26aufirst%3DM.%26aulast%3DUnver%26aufirst%3DM.%2BY.%26aulast%3DJumde%26aufirst%3DV.%2BK.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DZucal%26aufirst%3DC.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DDi%2BGiacomo%26aufirst%3DM.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DNovel%2520Compounds%2520Targeting%2520the%2520RNA-Binding%2520Protein%2520HuR.%2520Structure-Based%2520Design%252C%2520Synthesis%252C%2520and%2520Interaction%2520Studies%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D615%26epage%3D620%26doi%3D10.1021%2Facsmedchemlett.8b00600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Della Volpe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unver, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Exploration of ligand binding modes towards the identification of compounds targeting HuR: a combined STD-NMR and Molecular Modelling approach</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">13780</span> <span class="refDoi"> DOI: 10.1038/s41598-018-32084-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1038%2Fs41598-018-32084-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=30214075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BB3c3os12itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=F.+Vasileauthor=S.+Della+Volpeauthor=F.+A.+Ambrosioauthor=G.+Costaauthor=M.+Y.+Unverauthor=C.+Zucalauthor=D.+Rossiauthor=E.+Martinoauthor=A.+Provenzaniauthor=A.+K.+H.+Hirschauthor=S.+Alcaroauthor=D.+Potenzaauthor=S.+Collina&title=Exploration+of+ligand+binding+modes+towards+the+identification+of+compounds+targeting+HuR%3A+a+combined+STD-NMR+and+Molecular+Modelling+approach&doi=10.1038%2Fs41598-018-32084-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of ligand binding modes towards the identification of compounds targeting HuR: a combined STD-NMR and Molecular Modelling approach</span></div><div class="casAuthors">Vasile Francesca; Della Volpe Serena; Potenza Donatella; Della Volpe Serena; Rossi Daniela; Collina Simona; Ambrosio Francesca Alessandra; Costa Giosue; Alcaro Stefano; Unver M Yagiz; Hirsch Anna K H; Zucal Chiara; Provenzani Alessandro; Martino Emanuela; Hirsch Anna K H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13780</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Post-transcriptional processes have been recognised as pivotal in the control of gene expression, and impairments in RNA processing are reported in several pathologies (i.e., cancer and neurodegeneration).  Focusing on RNA-binding proteins (RBPs), the involvement of Embryonic Lethal Abnormal Vision (ELAV) or Hu proteins and their complexes with target mRNAs in the aetiology of various dysfunctions, has suggested the great potential of compounds able to interfere with the complex stability as an innovative pharmacological strategy for the treatment of numerous diseases.  Here, we present a rational follow-up investigation of the interaction between ELAV isoform HuR and structurally-related compounds (i.e., flavonoids and coumarins), naturally decorated with different functional groups, by means of STD-NMR and Molecular Modelling.  Our results represent the foundation for the development of potent and selective ligands able to interfere with ELAV-RNA complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRWjR92U2RfrXy-c-E1v-fmfW6udTcc2eauu8EOKQ7jv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3os12itA%253D%253D&md5=9503a9f87bb6bbd695974a32063edd79</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-32084-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-32084-z%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DDella%2BVolpe%26aufirst%3DS.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DUnver%26aufirst%3DM.%2BY.%26aulast%3DZucal%26aufirst%3DC.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DMartino%26aufirst%3DE.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DExploration%2520of%2520ligand%2520binding%2520modes%2520towards%2520the%2520identification%2520of%2520compounds%2520targeting%2520HuR%253A%2520a%2520combined%2520STD-NMR%2520and%2520Molecular%2520Modelling%2520approach%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26doi%3D10.1038%2Fs41598-018-32084-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volpe, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Listro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parafioriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giacomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">BOPC1 Enantiomers Preparation and HuR Interaction Study. From Molecular Modeling to a Curious DEEP-STD NMR Application</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">888</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00659</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00659" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=883-888&author=S.+D.+Volpeauthor=R.+Listroauthor=M.+Parafioritiauthor=M.+Di+Giacomoauthor=D.+Rossiauthor=F.+A.+Ambrosioauthor=G.+Costaauthor=S.+Alcaroauthor=F.+Ortusoauthor=A.+K.+H.+Hirschauthor=F.+Vasileauthor=S.+Collina&title=BOPC1+Enantiomers+Preparation+and+HuR+Interaction+Study.+From+Molecular+Modeling+to+a+Curious+DEEP-STD+NMR+Application&doi=10.1021%2Facsmedchemlett.9b00659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">BOPC1 Enantiomers Preparation and HuR Interaction Study. From Molecular Modeling to a Curious DEEP-STD NMR Application</span></div><div class="casAuthors">Volpe, Serena Della; Listro, Roberta; Parafioriti, Michela; Di Giacomo, Marcello; Rossi, Daniela; Ambrosio, Francesca Alessandra; Costa, Giosue; Alcaro, Stefano; Ortuso, Francesco; Hirsch, Anna K. H.; Vasile, Francesca; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">883-888</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Hu family of RNA-binding proteins plays a crucial role in post-transcriptional processes; indeed, Hu-RNA complexes are involved in various dysfunctions (i.e., inflammation, neurodegeneration, and cancer) and have been recently proposed as promising therapeutic targets.  Intrigued by this concept, our research efforts aim at identifying small mols. able to modulate HuR-RNA interactions, with a focus on subtype HuR, upregulated and dysregulated in several cancers.  By applying structure-based design, we had already identified racemic trans-BOPC1 as promising HuR binder.  In this Letter, we accomplished the enantio-resoln., the assignment of the abs. configuration, and the recognition study with HuR of enantiomerically pure trans-BOPC1.  For the first time, we apply DEEP (differential epitope mapping)-STD NMR to study the interaction of BOPC1 with HuR and compare its enantiomers, gaining information on ligand orientation and amino acids involved in the interaction, and thus increasing focus on the in silico binding site model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz5a0llcCGGLVg90H21EOLACvtfcHk0lgD2fIo_JuUUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSjsb8%253D&md5=26c2fca64cba054da303bb5d5958e8d3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00659%26sid%3Dliteratum%253Aachs%26aulast%3DVolpe%26aufirst%3DS.%2BD.%26aulast%3DListro%26aufirst%3DR.%26aulast%3DParafioriti%26aufirst%3DM.%26aulast%3DDi%2BGiacomo%26aufirst%3DM.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DOrtuso%26aufirst%3DF.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DBOPC1%2520Enantiomers%2520Preparation%2520and%2520HuR%2520Interaction%2520Study.%2520From%2520Molecular%2520Modeling%2520to%2520a%2520Curious%2520DEEP-STD%2520NMR%2520Application%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D883%26epage%3D888%26doi%3D10.1021%2Facsmedchemlett.9b00659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manzoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maio, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Agostino, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miceli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baj, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerofolini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elezgarai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biasini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchinat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fragai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seneci, P.</span></span> <span> </span><span class="NLM_article-title">Interfering with HuR-RNA Interaction: Design, Synthesis and Biological Characterization of Tanshinone Mimics as Novel, Effective HuR Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1483</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVensA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1483-1498&author=L.+Manzoniauthor=C.+Zucalauthor=D.+D.+Maioauthor=V.+G.+D%E2%80%99Agostinoauthor=N.+Thongonauthor=I.+Bonomoauthor=P.+Lalauthor=M.+Miceliauthor=V.+Bajauthor=M.+Brambillaauthor=L.+Cerofoliniauthor=S.+Elezgaraiauthor=E.+Biasiniauthor=C.+Luchinatauthor=E.+Novellinoauthor=M.+Fragaiauthor=L.+Marinelliauthor=A.+Provenzaniauthor=P.+Seneci&title=Interfering+with+HuR-RNA+Interaction%3A+Design%2C+Synthesis+and+Biological+Characterization+of+Tanshinone+Mimics+as+Novel%2C+Effective+HuR+Inhibitors&doi=10.1021%2Facs.jmedchem.7b01176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Interfering with HuR-RNA Interaction: Design, Synthesis and Biological Characterization of Tanshinone Mimics as Novel, Effective HuR Inhibitors</span></div><div class="casAuthors">Manzoni, Leonardo; Zucal, Chiara; Maio, Danilo Di; D'Agostino, Vito G.; Thongon, Natthakan; Bonomo, Isabelle; Lal, Preet; Miceli, Marco; Baj, Vanessa; Brambilla, Marta; Cerofolini, Linda; Elezgarai, Saioa; Biasini, Emiliano; Luchinat, Claudio; Novellino, Ettore; Fragai, Marco; Marinelli, Luciana; Provenzani, Alessandro; Seneci, Pierfausto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1483-1498</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The human antigen R (HuR) is an RNA-binding protein known to modulate the expression of target mRNA coding for proteins involved in inflammation, tumorigenesis, and stress responses and is a valuable drug target.  We previously found that dihydrotanshinone-I (DHTS, 1) prevents the assocn. of HuR with its RNA substrate, thus imparing its function.  Herein, inspired by DHTS structure, we designed and synthesized an array of ortho-quinones (tanshinone mimics) using a function-oriented synthetic approach.  Among others, compd. 6a and 6n turned out to be more effective than 1, showing a nanomolar Ki and disrupting HuR binding to RNA in cells.  A combined approach of NMR titrn. and mol. dynamics (MD) simulations suggests that 6a stabilizes HuR in a peculiar closed conformation, which is incompatible with RNA binding.  Alpha screen and RNA-electrophoretic mobility shift assays (REMSA) data on newly synthesized compds. allowed, for the first time, the generation of structure activity relationships (SARs), thus providing a solid background for the generation of highly effective HuR disruptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS0iVE3OcEC7Vg90H21EOLACvtfcHk0lgD2fIo_JuUUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVensA%253D%253D&md5=e9ad9fbce3f181afc02a716c566be2fc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01176%26sid%3Dliteratum%253Aachs%26aulast%3DManzoni%26aufirst%3DL.%26aulast%3DZucal%26aufirst%3DC.%26aulast%3DMaio%26aufirst%3DD.%2BD.%26aulast%3DD%25E2%2580%2599Agostino%26aufirst%3DV.%2BG.%26aulast%3DThongon%26aufirst%3DN.%26aulast%3DBonomo%26aufirst%3DI.%26aulast%3DLal%26aufirst%3DP.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DBaj%26aufirst%3DV.%26aulast%3DBrambilla%26aufirst%3DM.%26aulast%3DCerofolini%26aufirst%3DL.%26aulast%3DElezgarai%26aufirst%3DS.%26aulast%3DBiasini%26aufirst%3DE.%26aulast%3DLuchinat%26aufirst%3DC.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DFragai%26aufirst%3DM.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DSeneci%26aufirst%3DP.%26atitle%3DInterfering%2520with%2520HuR-RNA%2520Interaction%253A%2520Design%252C%2520Synthesis%2520and%2520Biological%2520Characterization%2520of%2520Tanshinone%2520Mimics%2520as%2520Novel%252C%2520Effective%2520HuR%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1483%26epage%3D1498%26doi%3D10.1021%2Facs.jmedchem.7b01176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lgD2fIo_JuUUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvisi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiana, G.</span></span> <span> </span><span class="NLM_article-title">Thermodynamically-Weighted Conformational Ensemble of Cyclic RGD Peptidomimetics from NOE Data</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">7098</span>– <span class="NLM_lpage">7107</span>, <span class="refDoi"> DOI: 10.1021/acs.jpcb.6b04941</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.6b04941" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFersrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2016&pages=7098-7107&author=F.+Vasileauthor=M.+Civeraauthor=L.+Belvisiauthor=D.+Potenzaauthor=G.+Tiana&title=Thermodynamically-Weighted+Conformational+Ensemble+of+Cyclic+RGD+Peptidomimetics+from+NOE+Data&doi=10.1021%2Facs.jpcb.6b04941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamically-Weighted Conformational Ensemble of Cyclic RGD Peptidomimetics from NOE Data</span></div><div class="casAuthors">Vasile, F.; Civera, M.; Belvisi, L.; Potenza, D.; Tiana, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">7098-7107</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the case of flexible mols., the std. approach of transforming NOE intensities into spatial restraints and of building conformational models minimizing these restraints greatly neglects the richness of mol. conformations.  Making use of NOE intensities measured in triplicate and of an iterative mol.-dynamics scheme, we optimized a force field to generate a set of conformations whose ensemble is compatible with the exptl. data, and is weighted according to the Boltzmann distribution.  This scheme is applied to two cyclic peptidomimetic ligands of integrins.  Their difference in binding affinity is recapitulated in terms of their difference in conformational fluctuations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7hONyJihVz7Vg90H21EOLACvtfcHk0ljvcBqlWqgfmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFersrbO&md5=c49c258ca2ec935388b06a777bab1456</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.6b04941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.6b04941%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DCivera%26aufirst%3DM.%26aulast%3DBelvisi%26aufirst%3DL.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DTiana%26aufirst%3DG.%26atitle%3DThermodynamically-Weighted%2520Conformational%2520Ensemble%2520of%2520Cyclic%2520RGD%2520Peptidomimetics%2520from%2520NOE%2520Data%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2016%26volume%3D120%26spage%3D7098%26epage%3D7107%26doi%3D10.1021%2Facs.jpcb.6b04941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubinelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panigada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soprana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siccardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span> <span> </span><span class="NLM_article-title">NMR interaction studies of Neu5Ac-α-(2,6)-Gal-β-(1-4)-GlcNAc with influenza-virus hemagglutinin expressed in transfected human cells</span>. <i>Glycobiology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1093/glycob/cwx092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fglycob%2Fcwx092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=29087468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWht73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=42-49&author=F.+Vasileauthor=F.+Gubinelliauthor=M.+Panigadaauthor=E.+Sopranaauthor=A.+Siccardiauthor=D.+Potenza&title=NMR+interaction+studies+of+Neu5Ac-%CE%B1-%282%2C6%29-Gal-%CE%B2-%281-4%29-GlcNAc+with+influenza-virus+hemagglutinin+expressed+in+transfected+human+cells&doi=10.1093%2Fglycob%2Fcwx092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">NMR interaction studies of Neu5Ac-α-(2,6)-Gal-β-(1-4)-GlcNAc with influenza-virus hemagglutinin expressed in transfected human cells</span></div><div class="casAuthors">Vasile, Francesca; Gubinelli, Francesco; Panigada, Maddalena; Soprana, Elisa; Siccardi, Antonio; Potenza, Donatella</div><div class="citationInfo"><span class="NLM_cas:title">Glycobiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-49</span>CODEN:
                <span class="NLM_cas:coden">GLYCE3</span>;
        ISSN:<span class="NLM_cas:issn">1460-2423</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The emergence of escape-mutants of influenza hemagglutinin (HA) following vaccination compels the yearly re-formulation of flu vaccines.  Since binding the sialic acid receptor remains in all cases essential for infection, small-mol. inhibitors of HA binding to sialic acid could be interesting therapeutic complements or alternatives to immuno-prophylaxis in the control of flu epidemics.  In this work, we made use of NMR spectroscopy to study the interaction between a deriv. of sialic acid (the Neu5Ac-α-(2,6)-Gal-β-(1-4)-GlcNAc trisaccharide) and HAs (H1 and H5) from human and avian strains of influenza virus, directly expressed on the surface of stable transfected 293 T human cells.  The HAs were shown to retain their native trimeric conformation and binding proper- ties.  Exploiting the magnetization transfer between the proteins and the ligand, we obtained evidence of the binding event and mapped the (non-identical) sugar epitopes recognized by the two HA species.  The rapid and reliable method for screening sialic acid-related HA ligands we have developed could yield useful information for an efficient drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9m74VJJzKPLVg90H21EOLACvtfcHk0ljvcBqlWqgfmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWht73N&md5=cdb70a5358c8354e72d39d501f6b83fa</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fglycob%2Fcwx092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fglycob%252Fcwx092%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DGubinelli%26aufirst%3DF.%26aulast%3DPanigada%26aufirst%3DM.%26aulast%3DSoprana%26aufirst%3DE.%26aulast%3DSiccardi%26aufirst%3DA.%26aulast%3DPotenza%26aufirst%3DD.%26atitle%3DNMR%2520interaction%2520studies%2520of%2520Neu5Ac-%25CE%25B1-%25282%252C6%2529-Gal-%25CE%25B2-%25281-4%2529-GlcNAc%2520with%2520influenza-virus%2520hemagglutinin%2520expressed%2520in%2520transfected%2520human%2520cells%26jtitle%3DGlycobiology%26date%3D2018%26volume%3D28%26spage%3D42%26epage%3D49%26doi%3D10.1093%2Fglycob%2Fcwx092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engelborghs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanoiu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandewoude, M.</span></span> <span> </span><span class="NLM_article-title">Rationale and clinical data supporting nutritional intervention in Alzheimer’s disease</span>. <i>Acta Clin. Belg.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1179/0001551213Z.0000000006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1179%2F0001551213Z.0000000006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24635394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BC2crksFCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=17-24&author=S.+Engelborghsauthor=C.+Gillesauthor=A.+Ivanoiuauthor=M.+Vandewoude&title=Rationale+and+clinical+data+supporting+nutritional+intervention+in+Alzheimer%E2%80%99s+disease&doi=10.1179%2F0001551213Z.0000000006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Rationale and clinical data supporting nutritional intervention in Alzheimer's disease</span></div><div class="casAuthors">Engelborghs S; Gilles C; Ivanoiu A; Vandewoude M</div><div class="citationInfo"><span class="NLM_cas:title">Acta clinica Belgica</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-24</span>
        ISSN:<span class="NLM_cas:issn">1784-3286</span>.
    </div><div class="casAbstract">Adequate nutrition plays an important role in the maintenance of cognitive function, particularly during aging.  Malnutrition is amongst the risk factors for developing mild cognitive impairment (MCI) and Alzheimer's disease (AD).  Epidemiological studies have associated deficiencies in some nutrients with a higher risk of cognitive dysfunction and/or AD.  Cognitive decline in AD is correlated with synaptic loss and many of the components required to maintain optimal synaptic function are derived from dietary sources.  As synapses are part of the neuronal membrane and are continuously being remodelled, the availability of sufficient levels of nutritional precursors (mainly uridine monophosphate, choline and omega-3 fatty acids) to make the phospholipids required to build neuronal membranes may have beneficial effects on synaptic degeneration in AD.  In addition, B-vitamins, phospholipids and other micronutrients act as cofactors to enhance the supply of precursors required to make neuronal membranes and synapses.  Despite this, no randomized controlled trial has hitherto provided evidence that any single nutrient has a beneficial effect on cognition or lowers the risk for AD.  However, a multi-target approach using combinations of (micro)nutrients might have beneficial effects on cognitive function in neurodegenerative brain disorders like AD leading to synaptic degeneration.  Here we review the clinical evidence for supplementation, based on a multi-target approach with a focus on key nutrients with a proposed role in synaptic dysfunction.  Based on preclinical evidence, a nutrient mixture, Souvenaid(®) (Nutricia N.V., Zoetermeer, The Netherlands) was developed.  Clinical trials with Souvenaid(®) have shown improved memory performance in patients with mild AD.  Further clinical trials to evaluate the effects of nutritional intervention in MCI and early dementia due to AD are on-going.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfYnw-ONC_vVQKX3ooHxWjfW6udTcc2eZVtFg5uRFj3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crksFCgsQ%253D%253D&md5=0c6d69afdaf9ae1f3bea1679aa685c49</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1179%2F0001551213Z.0000000006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1179%252F0001551213Z.0000000006%26sid%3Dliteratum%253Aachs%26aulast%3DEngelborghs%26aufirst%3DS.%26aulast%3DGilles%26aufirst%3DC.%26aulast%3DIvanoiu%26aufirst%3DA.%26aulast%3DVandewoude%26aufirst%3DM.%26atitle%3DRationale%2520and%2520clinical%2520data%2520supporting%2520nutritional%2520intervention%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DActa%2520Clin.%2520Belg.%26date%3D2014%26volume%3D69%26spage%3D17%26epage%3D24%26doi%3D10.1179%2F0001551213Z.0000000006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhobale, M.</span></span> <span> </span><span class="NLM_article-title">Neurotrophic Factors and Maternal Nutrition During Pregnancy</span>. <i>Vitam. Horm.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1016/bs.vh.2016.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fbs.vh.2016.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=28215301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVOgsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2017&pages=343-366&author=M.+Dhobale&title=Neurotrophic+Factors+and+Maternal+Nutrition+During+Pregnancy&doi=10.1016%2Fbs.vh.2016.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Neurotrophic factors and maternal nutrition during pregnancy</span></div><div class="casAuthors">Dhobale, M.</div><div class="citationInfo"><span class="NLM_cas:title">Vitamins and Hormones (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">Neurotrophins</span>),
    <span class="NLM_cas:pages">343-366</span>CODEN:
                <span class="NLM_cas:coden">VIHOAQ</span>;
        ISSN:<span class="NLM_cas:issn">0083-6729</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Maternal nutrition is one of the major determinants of pregnancy outcome.  It has been suggested that reduced intakes or lack of specific nutrients during pregnancy influences the length of gestation, proper placental and fetal growth during pregnancy.  Maternal nutrition, particularly micronutrients such as folate and vitamin B12, and long-chain polyunsatd. fatty acids (LCPUFA) are the major determinants of the one carbon cycle and are suggested to be at the heart of intrauterine programming of diseases in adult life.  LCPUFA play a key role in the normal feto-placental development, as well as in the development and functional maturation of the brain and central nervous system and also regulate the levels of neurotrophic factors.  These neurotrophic factors are known to regulate the development of the placenta at the materno-fetal interface and act in a paracrine and endocrine manner.  Neurotrophic factors like brain-derived neurotrophic factor and nerve growth factor are proteins involved in angiogenesis and potentiate the placental development.  This chapter mainly focuses on micronutrients since they play a main physiol. role during pregnancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUoyM5ZkiyZLVg90H21EOLACvtfcHk0lgOpBvfoxi5ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVOgsL4%253D&md5=ed7beda14dd881ef1c20f970e9f386a9</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fbs.vh.2016.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.vh.2016.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DDhobale%26aufirst%3DM.%26atitle%3DNeurotrophic%2520Factors%2520and%2520Maternal%2520Nutrition%2520During%2520Pregnancy%26jtitle%3DVitam.%2520Horm.%26date%3D2017%26volume%3D104%26spage%3D343%26epage%3D366%26doi%3D10.1016%2Fbs.vh.2016.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coşar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ipçioğlu, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gültepe, M.</span></span> <span> </span><span class="NLM_article-title">Folate and homocysteine metabolisms and their roles in the biochemical basis of neuropsychiatry</span>. <i>Turk. J. Med. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.3906/sag-1211-39</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.3906%2Fsag-1211-39" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=25558551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVKjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2014&pages=1-9&author=A.+Co%C5%9Farauthor=O.+M.+Ip%C3%A7io%C4%9Fluauthor=O.+Ozcanauthor=M.+G%C3%BCltepe&title=Folate+and+homocysteine+metabolisms+and+their+roles+in+the+biochemical+basis+of+neuropsychiatry&doi=10.3906%2Fsag-1211-39"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Folate and homocysteine metabolisms and their roles in the biochemical basis of neuropsychiatry</span></div><div class="casAuthors">Cosar, Alpaslan; Ipcioglu, Osman Metin; Ozcan, Omer; Gultepe, Mustafa</div><div class="citationInfo"><span class="NLM_cas:title">Turkish Journal of Medical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">TJMEEA</span>;
        ISSN:<span class="NLM_cas:issn">1300-0144</span>.
    
            (<span class="NLM_cas:orgname">Scientific and Technological Research Council of Turkey</span>)
        </div><div class="casAbstract">A review.  The term 'one-carbon metab.' is commonly used to describe 3 sep. metabolic processes: folate metab., the homocysteine remethylation cycle, and the transsulfuration pathway.  Folate metab. concerns the biochem. reactions in which endogenous and exogenous one-carbon units are transferred to tetrahydrofolates.  The remethylation cycle is used for the synthesis of methionine from homocysteine with one-carbon units that come from folate.  This methionine is then used for the synthesis of S-adenosyl methionine, which is a general donor of Me groups for many biochem. reactions in the human body.  In the transsulfuration pathway, some amino acids and polypeptides, such as cystathionine, cysteine, and glutathione, are synthesized from homocysteine.  The kinetics of the enzymes in this pathway are regulated by the substrates of the remethylation cycle.  The methylation process has been thought to have an important role in the biochem. basis of neuropsychiatry.  An elevated homocysteine level is the most important marker of folate and vitamin B12 deficiencies, and also the most reliable biochem. sign of functional insufficiency.  Some neurol. and neuropsychiatric diseases, such as psychosis, Alzheimer's disease, and autism, have been found to be related to disorders of one-carbon metab.  This review aims to summarize both one-carbon metab. and its relationships with neuropsychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNIEIl1dodQrVg90H21EOLACvtfcHk0lgOpBvfoxi5ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVKjsrs%253D&md5=6852620d34834d85163941877f7aa40a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.3906%2Fsag-1211-39&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3906%252Fsag-1211-39%26sid%3Dliteratum%253Aachs%26aulast%3DCo%25C5%259Far%26aufirst%3DA.%26aulast%3DIp%25C3%25A7io%25C4%259Flu%26aufirst%3DO.%2BM.%26aulast%3DOzcan%26aufirst%3DO.%26aulast%3DG%25C3%25BCltepe%26aufirst%3DM.%26atitle%3DFolate%2520and%2520homocysteine%2520metabolisms%2520and%2520their%2520roles%2520in%2520the%2520biochemical%2520basis%2520of%2520neuropsychiatry%26jtitle%3DTurk.%2520J.%2520Med.%2520Sci.%26date%3D2014%26volume%3D44%26spage%3D1%26epage%3D9%26doi%3D10.3906%2Fsag-1211-39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, I.</span></span> <span> </span><span class="NLM_article-title">Alzheimer’s disease pathogenesis: Is there a role for folate?</span>. <i>Mech Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2017.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.mad.2017.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=29037490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslChs77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=86-94&author=N.+Robinsonauthor=P.+Grabowskiauthor=I.+Rehman&title=Alzheimer%E2%80%99s+disease+pathogenesis%3A+Is+there+a+role+for+folate%3F&doi=10.1016%2Fj.mad.2017.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease pathogenesis: Is there a role for folate?</span></div><div class="casAuthors">Robinson, Natassia; Grabowski, Peter; Rehman, Ishtiaq</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">86-94</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetic modifications, including changes in DNA methylation, have been implicated in a wide range of diseases including neurol. diseases such as Alzheimer's.  The role of dietary folate in providing Me groups required for maintenance and modulation of DNA methylation makes it a nutrient of interest in Alzheimer's.  Late onset Alzheimer's disease is the most common form of dementia and at present its etiol. is largely undetd.  From epidemiol. studies, the interactions between folate, B-vitamins and homocysteine as well as the long latency period has led to difficulties in interpretation of the data, thus current evidence exploring the role of dietary folate in Alzheimer's is contradictory and unresolved.  Therefore, examg. the effects at a mol. level and exploring potential epigenetic mechanisms could increase our understanding of the disease and etiol.  The aim of this review is to examine the role that folate could play in Alzheimer's disease neuropathol. and will focus on the effects of folate on DNA methylation which link to disease pathol., initiation and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBwPf1M7HdLVg90H21EOLACvtfcHk0lgOpBvfoxi5ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslChs77L&md5=efa72dacaa5eedfd8d9bf44d8b90b72b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2017.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2017.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DN.%26aulast%3DGrabowski%26aufirst%3DP.%26aulast%3DRehman%26aufirst%3DI.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%2520pathogenesis%253A%2520Is%2520there%2520a%2520role%2520for%2520folate%253F%26jtitle%3DMech%2520Ageing%2520Dev.%26date%3D2018%26volume%3D174%26spage%3D86%26epage%3D94%26doi%3D10.1016%2Fj.mad.2017.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yachi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tohyama, M.</span></span> <span> </span><span class="NLM_article-title">CARM1 Regulates Proliferation of PC12 Cells by Methylating HuD</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2273</span>– <span class="NLM_lpage">2285</span>, <span class="refDoi"> DOI: 10.1128/MCB.26.6.2273-2285.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1128%2FMCB.26.6.2273-2285.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=16508003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2273-2285&author=T.+Fujiwaraauthor=Y.+Moriauthor=D.+L.+Chuauthor=Y.+Koyamaauthor=S.+Miyataauthor=H.+Tanakaauthor=K.+Yachiauthor=T.+Kuboauthor=H.+Yoshikawaauthor=M.+Tohyama&title=CARM1+Regulates+Proliferation+of+PC12+Cells+by+Methylating+HuD&doi=10.1128%2FMCB.26.6.2273-2285.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">CARM1 regulates proliferation of PC12 cells by methylating HuD</span></div><div class="casAuthors">Fujiwara, Tatsuji; Mori, Yasutake; Chu, Dong Ling; Koyama, Yoshihisa; Miyata, Shingo; Tanaka, Hiroyuki; Yachi, Kohji; Kubo, Tateki; Yoshikawa, Hideki; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2273-2285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">HuD is an RNA-binding protein that has been shown to induce neuronal differentiation by stabilizing labile mRNAs carrying AU-rich instability elements.  Here, we show a novel mechanism of arginine methylation of HuD by coactivator-assocd. arginine methyltransferase 1 (CARM1) that affected mRNA turnover of p21cip1/waf1 mRNA in PC12 cells.  CARM1 specifically methylated HuD in vitro and in vivo and colocalized with HuD in the cytoplasm.  Inhibition of HuD methylation by CARM1 knockdown elongated the p21cip1/waf1 mRNA half-life and resulted in a slow growth rate and robust neuritogenesis in response to nerve growth factor (NGF).  Methylation-resistant HuD bound more p21cip1/waf1 mRNA than did the wild type, and its overexpression upregulated p21cip1/waf1 protein expression.  These results suggested that CARM1-methylated HuD maintains PC12 cells in the proliferative state by committing p21cip1/waf1 mRNA to its decay system.  Since the methylated population of HuD was reduced in NGF-treated PC12 cells, downregulation of HuD methylation is a possible pathway through which NGF induces differentiation of PC12 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71TYVPwUyRLVg90H21EOLACvtfcHk0lgOpBvfoxi5ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D&md5=a557599ff6460f5ab365362323b87a2b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.6.2273-2285.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.6.2273-2285.2006%26sid%3Dliteratum%253Aachs%26aulast%3DFujiwara%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DChu%26aufirst%3DD.%2BL.%26aulast%3DKoyama%26aufirst%3DY.%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DYachi%26aufirst%3DK.%26aulast%3DKubo%26aufirst%3DT.%26aulast%3DYoshikawa%26aufirst%3DH.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DCARM1%2520Regulates%2520Proliferation%2520of%2520PC12%2520Cells%2520by%2520Methylating%2520HuD%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2273%26epage%3D2285%26doi%3D10.1128%2FMCB.26.6.2273-2285.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craenen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verslegers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baatout, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benotmane, M. A.</span></span> <span> </span><span class="NLM_article-title">An appraisal of folates as key factors in cognition and ageing-related diseases</span>. <i>Crit. Rev. Food Sci. Nutr.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1080/10408398.2018.1549017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1080%2F10408398.2018.1549017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=30729795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFymsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=722-739&author=K.+Craenenauthor=M.+Verslegersauthor=S.+Baatoutauthor=M.+A.+Benotmane&title=An+appraisal+of+folates+as+key+factors+in+cognition+and+ageing-related+diseases&doi=10.1080%2F10408398.2018.1549017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An appraisal of folates as key factors in cognition and ageing-related diseases</span></div><div class="casAuthors">Craenen, Kai; Verslegers, Mieke; Baatout, Sarah; Abderrafi Benotmane, Mohammed</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Food Science and Nutrition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">722-739</span>CODEN:
                <span class="NLM_cas:coden">CRFND6</span>;
        ISSN:<span class="NLM_cas:issn">1040-8398</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Folic acid (FA) is often consumed as a food supplement and can be found in fortified staple foods in various western countries.  Even though FA supplementation during pregnancy is known to prevent severe congenital anomalies in the developing child (e.g., neural tube defects), much less is known about its influence on cognition and neurol. functioning.  In this review, we address the advances in this field and situate how folate intake during pregnancy, postnatal life, adulthood and in the elderly affects cognition.  In addn., an assocn. between folate status and ageing, dementia and other neurodegenerative disorders such as Alzheimer's disease, Parkinson's disease and amyotrophic lateral sclerosis is discussed.  While its role in the incidence and severity of these diseases is becoming apparent, the underlying action of folates and related metabolites remains elusive.  Finally, the potential of FA as a nutraceutical has been proposed, although the efficacy will highly depend on the interplay with other micronutrients, the disease stage and the duration of supplementation.  Hence, the lack of consistent data urges for more animal studies and (pre)clin. trials in humans to ascertain a potential beneficial role for folates in the treatment or amelioration of cognitive decline and ageing-related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf6W9bg0cB-7Vg90H21EOLACvtfcHk0liwzxKBvPaotQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFymsb8%253D&md5=30489f9ea4310a877ce4d7bfef58f948</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1080%2F10408398.2018.1549017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408398.2018.1549017%26sid%3Dliteratum%253Aachs%26aulast%3DCraenen%26aufirst%3DK.%26aulast%3DVerslegers%26aufirst%3DM.%26aulast%3DBaatout%26aufirst%3DS.%26aulast%3DBenotmane%26aufirst%3DM.%2BA.%26atitle%3DAn%2520appraisal%2520of%2520folates%2520as%2520key%2520factors%2520in%2520cognition%2520and%2520ageing-related%2520diseases%26jtitle%3DCrit.%2520Rev.%2520Food%2520Sci.%2520Nutr.%26date%3D2020%26volume%3D60%26spage%3D722%26epage%3D739%26doi%3D10.1080%2F10408398.2018.1549017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scapagnini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkon, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrone, A.</span></span> <span> </span><span class="NLM_article-title">Neuronal ELAV proteins enhance mRNA stability by a PKCα-dependent pathway</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">12065</span>– <span class="NLM_lpage">12070</span>, <span class="refDoi"> DOI: 10.1073/pnas.0504702102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.0504702102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=16099831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2MXps1yru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=12065-12070&author=A.+Pascaleauthor=M.+Amadioauthor=G.+Scapagniniauthor=C.+Lanniauthor=M.+Racchiauthor=A.+Provenzaniauthor=S.+Govoniauthor=D.+L.+Alkonauthor=A.+Quattrone&title=Neuronal+ELAV+proteins+enhance+mRNA+stability+by+a+PKC%CE%B1-dependent+pathway&doi=10.1073%2Fpnas.0504702102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Neuronal ELAV proteins enhance mRNA stability by a PKCα-dependent pathway</span></div><div class="casAuthors">Pascale, Alessia; Amadio, Marialaura; Scapagnini, Giovanni; Lanni, Cristina; Racchi, Marco; Provenzani, Alessandro; Govoni, Stefano; Alkon, Daniel L.; Quattrone, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12065-12070</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">More than 1 in 20 human genes bear in the mRNA 3' UTR a specific motif called the adenine- and uridine-rich element (ARE), which posttranscriptionally dets. its expression in response to cell environmental signals.  ELAV (embryonic lethal abnormal vision) proteins are the only known ARE-binding factors that are able to stabilize the bound mRNAs, thereby pos. controlling gene expression.  Here, we show that in human neuroblastoma SH-SY5Y cells, neuron-specific ELAV (nELAV) proteins (HuB, HuC, and HuD) are up-regulated and redistributed by 15 min of treatment with the activators of PKC phorbol esters and bryostatin-1.  PKC stimulation also induces nELAV proteins to colocalize with the translocated PKCα isoenzyme preferentially on the cytoskeleton, with a concomitant increase in nELAV threonine phosphorylation.  The same treatment promotes stabilization of growth-assocd. protein 43 (GAP-43) mRNA, a well known nELAV target, and induces an early increase in GAP-43 protein concn., again only in the cytoskeletal cell fraction.  Genetic or pharmacol. inactivation of PKCα abolishes nELAV protein cytoskeletal up-regulation, GAP-43 mRNA stabilization, and GAP-43 protein increase, demonstrating the primary role of this specific PKC isoenzyme in the cascade of nELAV recruitment.  Finally, in vivo PKC activation is assocd. with an up-regulation of nELAV proteins in the hippocampal rat brain.  These findings suggest a model for gene expression regulation by nELAV proteins through a PKCα-dependent pathway that is relevant for the cellular programs in which ARE-mediated control plays a pivotal role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowTnFfQNrvp7Vg90H21EOLACvtfcHk0liwzxKBvPaotQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXps1yru7s%253D&md5=ac18f404ee001a080e2f64ee17d08a59</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0504702102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0504702102%26sid%3Dliteratum%253Aachs%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DScapagnini%26aufirst%3DG.%26aulast%3DLanni%26aufirst%3DC.%26aulast%3DRacchi%26aufirst%3DM.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DAlkon%26aufirst%3DD.%2BL.%26aulast%3DQuattrone%26aufirst%3DA.%26atitle%3DNeuronal%2520ELAV%2520proteins%2520enhance%2520mRNA%2520stability%2520by%2520a%2520PKC%25CE%25B1-dependent%2520pathway%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D12065%26epage%3D12070%26doi%3D10.1073%2Fpnas.0504702102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span> <i>Protein Preparation Wizard, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Protein+Preparation+Wizard%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DProtein%2520Preparation%2520Wizard%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span> <i>MacroModel, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+MacroModel%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C%0ANY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMacroModel%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribiere, E. P. eG.</span></span> <span> </span><span class="NLM_article-title">Note sur la convergence de méthodes de directions conjuguées</span>. <i>Revue française d’informatique et de recherche opérationnelle. Série rouge</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1051/m2an/196903r100351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1051%2Fm2an%2F196903r100351" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1969&pages=35-43&author=E.+P.+eG.+Ribiere&title=Note+sur+la+convergence+de+m%C3%A9thodes+de+directions+conjugu%C3%A9es&doi=10.1051%2Fm2an%2F196903r100351"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1051%2Fm2an%2F196903r100351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1051%252Fm2an%252F196903r100351%26sid%3Dliteratum%253Aachs%26aulast%3DRibiere%26aufirst%3DE.%2BP.%2BeG.%26atitle%3DNote%2520sur%2520la%2520convergence%2520de%2520m%25C3%25A9thodes%2520de%2520directions%2520conjugu%25C3%25A9es%26jtitle%3DRevue%2520fran%25C3%25A7aise%2520d%25E2%2580%2599informatique%2520et%2520de%2520recherche%2520op%25C3%25A9rationnelle.%2520S%25C3%25A9rie%2520rouge%26date%3D1969%26volume%3D3%26spage%3D35%26epage%3D43%26doi%3D10.1051%2Fm2an%2F196903r100351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span>Desmond Molecular
Dynamics
System, D.E. Shaw Research, New York, NY, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Desmond+Molecular%0ADynamics%0ASystem%2C+D.E.+Shaw+Research%2C+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span>Maestro-Desmond
Interoperability
Tools, Schrödinger, New York, NY, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro-Desmond%0AInteroperability%0ATools%2C+Schr%C3%B6dinger%2C+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">11225</span>– <span class="NLM_lpage">11236</span>, <span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+Jorgensenauthor=D.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+Testing+of+the+OPLS+All-Atom+Force+Field+on+Conformational+Energetics+and+Properties+of+Organic+Liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0lhyfhN433qJiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%26aulast%3DMaxwell%26aufirst%3DD.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520Testing%2520of%2520the%2520OPLS%2520All-Atom%2520Force%2520Field%2520on%2520Conformational%2520Energetics%2520and%2520Properties%2520of%2520Organic%2520Liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span> <i>Glide, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="note"><p class="first last">foodb.ca.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: a major update to the DrugBank database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&author=D.+S.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+a+major+update+to+the+DrugBank+database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lhyfhN433qJiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520a%2520major%2520update%2520to%2520the%2520DrugBank%2520database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolstad, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span> <span> </span><span class="NLM_article-title">ZINC: A Free Tool to Discover Chemistry for Biology</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1757</span>– <span class="NLM_lpage">1768</span>, <span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+Free+Tool+to+Discover+Chemistry+for+Biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lhyfhN433qJiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520Free%2520Tool%2520to%2520Discover%2520Chemistry%2520for%2520Biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span> <i>LigPrep, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C%0ANY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigPrep%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">ZINC 15 – Ligand Discovery for Everyone</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2324</span>– <span class="NLM_lpage">2337</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&author=T.+Sterlingauthor=J.+J.+Irwin&title=ZINC+15+%E2%80%93+Ligand+Discovery+for+Everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0lgqD_hDp0FM4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZINC%252015%2520%25E2%2580%2593%2520Ligand%2520Discovery%2520for%2520Everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="note"><p class="first last">zinc15.docking.org/patterns/home.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':[],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':[]}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HuD protein and its involvement in neuronal functions.<a onclick="showRef(event, 'ref11 ref19'); return false;" href="javascript:void(0);" class="ref ref11 ref19">(11,19)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. 3D representation of the (A) HuD-mRNA complex, (B) HuD <i>closed</i> conformation structure, and (C) HuD-RNA complex superimposed on the HuD <i>closed</i> conformation structure. The two protein conformations and the mRNA are shown as slate, magenta, and gray cartoon, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. 2D structures of the 10 hits identified by means of the virtual screening approach.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. 3D representations of the best pose of (A) folic acid (green sticks), (B) rosmarinic acid (orange sticks), (C) cefazolin (cyan sticks), and (D) enalapril (light-blue sticks) docked into the HuD binding pocket. The ligands are shown in sticks, HuD is shown as a magenta cartoon, and the enzyme residues involved in crucial contacts with the compounds are reported as gray carbon sticks. Hydrogen bonds, salt bridges, aromatic hydrogen bonds, π–π stacking, and π-cation interactions are represented by yellow, green, cyan, violet, and red dashed lines, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/medium/jm1c00191_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. RMSF plots of (A) folic acid, (B) rosmarinic acid, (C) cefazolin, and (D) enalapril.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00191/20210716/images/large/jm1c00191_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00191&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35736" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35736" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talman, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jäntti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuominen, R. K.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase C Activation as a Potential Therapeutic Strategy in Alzheimer’s Disease: Is there a Role for Embryonic Lethal Abnormal Vision-like Proteins?</span>. <i>Basic Clin. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1111/bcpt.12581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1111%2Fbcpt.12581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=27001133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVOnu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2016&pages=149-160&author=V.+Talmanauthor=A.+Pascaleauthor=M.+J%C3%A4nttiauthor=M.+Amadioauthor=R.+K.+Tuominen&title=Protein+Kinase+C+Activation+as+a+Potential+Therapeutic+Strategy+in+Alzheimer%E2%80%99s+Disease%3A+Is+there+a+Role+for+Embryonic+Lethal+Abnormal+Vision-like+Proteins%3F&doi=10.1111%2Fbcpt.12581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C activation as a potential therapeutic strategy in Alzheimer's disease: Is there a role for embryonic lethal abnormal vision-like proteins?</span></div><div class="casAuthors">Talman, Virpi; Pascale, Alessia; Jaentti, Maria; Amadio, Marialaura; Tuominen, Raimo K.</div><div class="citationInfo"><span class="NLM_cas:title">Basic & Clinical Pharmacology & Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">149-160</span>CODEN:
                <span class="NLM_cas:coden">BCPTBO</span>;
        ISSN:<span class="NLM_cas:issn">1742-7835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease (AD), the most common cause of dementia, is an irreversible and progressive neurodegenerative disorder.  It affects predominantly brain areas that are crit. for memory and learning and is characterized by two main pathol. hallmarks: extracellular amyloid plaques and intracellular neurofibrillary tangles.  Protein kinase C (PKC) has been classified as one of the cognitive kinases controlling memory and learning.  By regulating several signalling pathways involved in amyloid and tau pathologies, it also plays an inhibitory role in AD pathophysiol.  Among downstream targets of PKC are the embryonic lethal abnormal vision (ELAV)-like RNA-binding proteins that modulate the stability and the translation of specific target mRNAs involved in synaptic remodelling linked to cognitive processes.  This MiniReview summarizes the current evidence on the role of PKC and ELAV-like proteins in learning and memory, highlighting how their derangement can contribute to AD pathophysiol.  This last aspect emphasizes the potential of pharmacol. activation of PKC as a promising therapeutic strategy for the treatment of AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoAKKMkypK_bVg90H21EOLACvtfcHk0lhtvSpDLHP10Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVOnu70%253D&md5=8850cf732159192dc719f96089fab34d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Fbcpt.12581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcpt.12581%26sid%3Dliteratum%253Aachs%26aulast%3DTalman%26aufirst%3DV.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DJ%25C3%25A4ntti%26aufirst%3DM.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DTuominen%26aufirst%3DR.%2BK.%26atitle%3DProtein%2520Kinase%2520C%2520Activation%2520as%2520a%2520Potential%2520Therapeutic%2520Strategy%2520in%2520Alzheimer%25E2%2580%2599s%2520Disease%253A%2520Is%2520there%2520a%2520Role%2520for%2520Embryonic%2520Lethal%2520Abnormal%2520Vision-like%2520Proteins%253F%26jtitle%3DBasic%2520Clin.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D119%26spage%3D149%26epage%3D160%26doi%3D10.1111%2Fbcpt.12581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bellavia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mecarozzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campese, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazioli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Screpanti, I.</span></span> <span> </span><span class="NLM_article-title">Notch3 and the Notch3-upregulated RNA-binding protein HuD regulate Ikaros alternative splicing</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1038%2Fsj.emboj.7601626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=17332745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1Wkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=1670-1680&author=D.+Bellaviaauthor=M.+Mecarozziauthor=A.+F.+Campeseauthor=P.+Grazioliauthor=C.+Taloraauthor=L.+Fratiauthor=A.+Gulinoauthor=I.+Screpanti&title=Notch3+and+the+Notch3-upregulated+RNA-binding+protein+HuD+regulate+Ikaros+alternative+splicing&doi=10.1038%2Fsj.emboj.7601626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Notch3 and the Notch3-upregulated RNA-binding protein HuD regulate Ikaros alternative splicing</span></div><div class="casAuthors">Bellavia, Diana; Mecarozzi, Marco; Campese, Antonio F.; Grazioli, Paola; Talora, Claudio; Frati, Luigi; Gulino, Alberto; Screpanti, Isabella</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1670-1680</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Constitutive activation of the transmembrane receptor, Notch3, and loss of function of the hematopoietic transcription repressor, Ikaros (IK), play direct roles in T-cell differentiation and leukemogenesis that are dependent on pre-T-cell receptor (pre-TCR) signaling.  We demonstrate the occurrence of crosstalk between Notch3 and IK that results in transcriptional regulation of the gene encoding the pTα chain of the pre-TCR.  We also show that, in the presence of the pre-TCR, constitutive activation of Notch3 in thymocytes causes increased expression of dominant-neg. non-DNA-binding IK isoforms, which are able to restrain the IK inhibition of Notch3's transcriptional activation of pTα.  This effect appears to be mediated by Notch3's pre-TCR-dependent upregulation of the RNA-binding protein, HuD.  Notch3 signaling thus appears to play a crit. role in the diminished IK activity described in several lymphoid leukemias.  By exerting transcription-activating and transcription-repressing effects on the pTα promoter, Notch3 and IK cooperate in the fine-tuning of pre-TCR expression and function, which has important implications for the regulation of thymocyte differentiation and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn8EHpXXmf9rVg90H21EOLACvtfcHk0liMYkVg-ru2Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1Wkt7w%253D&md5=405fe0573c16421b8b2d3d4516ea50a8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601626%26sid%3Dliteratum%253Aachs%26aulast%3DBellavia%26aufirst%3DD.%26aulast%3DMecarozzi%26aufirst%3DM.%26aulast%3DCampese%26aufirst%3DA.%2BF.%26aulast%3DGrazioli%26aufirst%3DP.%26aulast%3DTalora%26aufirst%3DC.%26aulast%3DFrati%26aufirst%3DL.%26aulast%3DGulino%26aufirst%3DA.%26aulast%3DScrepanti%26aufirst%3DI.%26atitle%3DNotch3%2520and%2520the%2520Notch3-upregulated%2520RNA-binding%2520protein%2520HuD%2520regulate%2520Ikaros%2520alternative%2520splicing%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D1670%26epage%3D1680%26doi%3D10.1038%2Fsj.emboj.7601626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mansfield, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keene, J. D.</span></span> <span> </span><span class="NLM_article-title">Neuron-specific ELAV/Hu proteins suppress HuR mRNA during neuronal differentiation by alternative polyadenylation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2734</span>– <span class="NLM_lpage">2746</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr1114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkr1114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=22139917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFOrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=2734-2746&author=K.+D.+Mansfieldauthor=J.+D.+Keene&title=Neuron-specific+ELAV%2FHu+proteins+suppress+HuR+mRNA+during+neuronal+differentiation+by+alternative+polyadenylation&doi=10.1093%2Fnar%2Fgkr1114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Neuron-specific ELAV/Hu proteins suppress HuR mRNA during neuronal differentiation by alternative polyadenylation</span></div><div class="casAuthors">Mansfield, Kyle D.; Keene, Jack D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2734-2746</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The ubiquitously expressed RNA-binding protein HuR increases the stability and translation of mRNAs encoding growth regulatory proteins that promote proliferation in a variety of cell types.  However, the three neuron-specific ELAV/Hu proteins, HuB, HuC and HuD, while binding to the same types of mRNAs, are required instead for neuronal differentiation, and it becomes difficult to reconcile these contrary functions when all four Hu proteins are expressed in the same neuron.  HuR mRNA exists as three alternatively polyadenylated variants, a 1.5-kb testes-specific mRNA isoform, a ubiquitous 2.4-kb isoform and a 6.0-kb isoform that we now show is induced during neuronal differentiation and appears to be neuron-specific.  This 6.0-kb neuron-specific mRNA isoform is inherently less stable and produces less HuR protein than the ubiquitous 2.4-kb mRNA.  Furthermore, we show that neuronal HuB, HuC and HuD, as well as HuR itself, can bind at the 2.4-kb mRNA polyadenylation site, and when overexpressed can affect alternative polyadenylation to generate an extended HuR 3'-UTR that is translationally suppressed.  We propose that the regulation of HuR protein expression by alternative polyadenylation allows neurons to post-transcriptionally regulate mRNAs-encoding factors required for proliferation vs. differentiation to facilitate neuronal differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4X4ncI3_Hf7Vg90H21EOLACvtfcHk0liMYkVg-ru2Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFOrsr0%253D&md5=3383427fc30286d573d8fe46c3f0da54</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1114%26sid%3Dliteratum%253Aachs%26aulast%3DMansfield%26aufirst%3DK.%2BD.%26aulast%3DKeene%26aufirst%3DJ.%2BD.%26atitle%3DNeuron-specific%2520ELAV%252FHu%2520proteins%2520suppress%2520HuR%2520mRNA%2520during%2520neuronal%2520differentiation%2520by%2520alternative%2520polyadenylation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D2734%26epage%3D2746%26doi%3D10.1093%2Fnar%2Fgkr1114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span> <span> </span><span class="NLM_article-title">HuD promotes BDNF expression in brain neurons via selective stabilization of the BDNF long 3’UTR mRNA</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e55718</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0055718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1371%2Fjournal.pone.0055718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=23383270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1yltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=M.+Allenauthor=C.+Birdauthor=W.+Fengauthor=G.+Liuauthor=W.+Liauthor=N.+I.+Perrone-Bizzozeroauthor=Y.+Feng&title=HuD+promotes+BDNF+expression+in+brain+neurons+via+selective+stabilization+of+the+BDNF+long+3%E2%80%99UTR+mRNA&doi=10.1371%2Fjournal.pone.0055718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">HuD promotes BDNF expression in brain neurons via selective stabilization of the BDNF long 3'UTR mRNA</span></div><div class="casAuthors">Allen, Megan; Bird, Clark; Feng, Wei; Liu, Guanglu; Li, Wenqi; Perrone-Bizzozero, Nora I.; Feng, Yue</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e55718</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Complex regulation of brain-derived neurotrophic factor (BDNF) governs its intricate functions in brain development and neuronal plasticity.  Besides tight transcriptional control from multiple distinct promoters, alternative 3'end processing of the BDNF transcripts generates either a long or a short 3'untranslated region (3'UTR).  Previous reports indicate that distinct RNA sequence in the BDNF 3'UTRs differentially regulates BDNF prodn. in the brain to accommodate neuronal activity changes, conceivably through differential interactions with undefined trans-acting factors that regulate stability and translation of these BDNF mRNA isoforms.  In this study, we report that the neuronal RNA-binding protein (RBP) HuD interacts with a highly conserved AU-rich element (ARE) specifically located in the BDNF long 3'UTR.  Such interaction is necessary and sufficient for selective stabilization of mRNAs that contain the BDNF long 3'UTR in vitro and in vivo.  Moreover, in a HuD transgenic mouse model, the BDNF long 3'UTR mRNA is increased in the hippocampal dentate granule cells (DGCs), leading to elevated expression of BDNF protein that is transported and stored in the mossy fiber (MF) terminals.  Our results identify HuD as the first trans-acting factor that enhances BDNF expression specifically through the long 3'UTR and a novel mechanism that regulates BDNF protein prodn. in selected neuronal populations by HuD abundance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo0ryguUd-zbVg90H21EOLACvtfcHk0liMYkVg-ru2Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1yltrs%253D&md5=d8207a900a52204059249350c6f23c2d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0055718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0055718%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DBird%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%2BI.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DHuD%2520promotes%2520BDNF%2520expression%2520in%2520brain%2520neurons%2520via%2520selective%2520stabilization%2520of%2520the%2520BDNF%2520long%25203%25E2%2580%2599UTR%2520mRNA%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0055718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, S.</span></span> <span> </span><span class="NLM_article-title">Different motif requirements for the localization zipcode element of β-actin mRNA binding by HuD and ZBP1</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">7432</span>– <span class="NLM_lpage">7446</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkv699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=26152301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGrur3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=7432-7446&author=H.+H.+Kimauthor=S.+J.+Leeauthor=A.+S.+Gardinerauthor=N.+I.+Perrone-Bizzozeroauthor=S.+Yoo&title=Different+motif+requirements+for+the+localization+zipcode+element+of+%CE%B2-actin+mRNA+binding+by+HuD+and+ZBP1&doi=10.1093%2Fnar%2Fgkv699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Different motif requirements for the localization zipcode element of β-actin mRNA binding by HuD and ZBP1</span></div><div class="casAuthors">Kim, Hak Hee; Lee, Seung Joon; Gardiner, Amy S.; Perrone-Bizzozero, Nora I.; Yoo, Soonmoon</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7432-7446</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The interactions of RNA-binding proteins (RBPs) with their target transcripts are essential for regulating gene expression at the post-transcriptional level including mRNA export/localization, stability, and translation.  ZBP1 and HuD are RBPs that play pivotal roles in mRNA transport and local translational control in neuronal processes.  While HuD possesses 3 RNA recognition motifs (RRMs), ZBP1 contains 2 RRMs and 4 K homol. (KH) domains that either increase target specificity or provide a multi-target binding capability.  Here, the authors used isolated cis-element sequences of the target mRNA to examine directly protein-RNA interactions in cell-free systems.  The authors found that both ZBP1 and HuD bound the zipcode element in rat β-actin mRNA's 3' UTR.  Differences between HuD and ZBP1 were obsd. in their binding preference to the element.  HuD showed a binding preference for U-rich sequences.  In contrast, ZBP1 binding to the zipcode RNA depended more on the structural level, as it required the proper spatial organization of a stem-loop that was mainly detd. by the U-rich element juxtaposed to the 3' end of a 5'-ACACCC-3' motif.  On the basis of this work, the authors propose that ZBP1 and HuD bind to overlapping sites in the β-actin zipcode, but they recognize different features of this target sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd4lheAQddorVg90H21EOLACvtfcHk0lhrmXJzx_6O0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGrur3P&md5=6282fed52d7bc1d83eab2f02cbda285e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv699%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.%2BH.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DGardiner%26aufirst%3DA.%2BS.%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%2BI.%26aulast%3DYoo%26aufirst%3DS.%26atitle%3DDifferent%2520motif%2520requirements%2520for%2520the%2520localization%2520zipcode%2520element%2520of%2520%25CE%25B2-actin%2520mRNA%2520binding%2520by%2520HuD%2520and%2520ZBP1%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D7432%26epage%3D7446%26doi%3D10.1093%2Fnar%2Fgkv699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Good, P. J.</span></span> <span> </span><span class="NLM_article-title">A conserved family of elav-like genes in vertebrates</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">4557</span>– <span class="NLM_lpage">4561</span>, <span class="refDoi"> DOI: 10.1073/pnas.92.10.4557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.92.10.4557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=7753842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADyaK2MXls1aksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=4557-4561&author=P.+J.+Good&title=A+conserved+family+of+elav-like+genes+in+vertebrates&doi=10.1073%2Fpnas.92.10.4557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved family of elav-like genes in vertebrates</span></div><div class="casAuthors">Good, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4557-61</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A large family of genes encodes proteins with RNA recognition motifs that are presumed to bind RNA and to function in posttranscription regulation.  Neural-specific members of this family include elav, a gene required for correct differentiation and maintenance of neurons in Drosophila melanogaster, and a related gene, HuD, which is expressed in human neuronal cells.  I have identified genes related to elav and HuD in Xenopus laevis, zebrafish, and mouse that define a family of four closely related vertebrate elav-like genes (elrA, elrB, elrC, and elrD) in fish, frogs, and mammals.  In addn. to protein sequence conservation, a segment of the 3'-untranslated sequence of elrD is also conserved, implying a functional role in elrD expression.  In adult frogs, elrC and elrD are exclusively expressed in the brain, whereas elrB is expressed in brain, testis, and ovary.  During Xenopus development, elrC and elrD RNAs are detected by late gastrula and late neurula stages, resp., whereas a nervous system-specific elrB RNA species is expressed by early tadpole stage.  Addnl. elrB transcripts are detected in the ovary and early embryo, demonstrating a maternal supply of mRNA and possibility of protein.  These expression patterns suggest a role for different elav-like genes in early development and neuronal differentiation.  Surprisingly, elrA is expressed in all adult tissues tested and at all times during development.  Thus, the widely expressed elrA is expected to have a related function in all cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKH_4_XJAyJrVg90H21EOLACvtfcHk0lhrmXJzx_6O0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXls1aksbs%253D&md5=d001ce3c18f2723e37a09011464cc878</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.10.4557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.10.4557%26sid%3Dliteratum%253Aachs%26aulast%3DGood%26aufirst%3DP.%2BJ.%26atitle%3DA%2520conserved%2520family%2520of%2520elav-like%2520genes%2520in%2520vertebrates%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D4557%26epage%3D4561%26doi%3D10.1073%2Fpnas.92.10.4557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanna, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghelardini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galeotti, N.</span></span> <span> </span><span class="NLM_article-title">HuD-mediated distinct BDNF regulatory pathways promote regeneration after nerve injury</span>. <i>Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1659</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.brainres.2017.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.brainres.2017.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=28111162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1659&publication_year=2017&pages=55-63&author=M.+D.+Sannaauthor=C.+Ghelardiniauthor=N.+Galeotti&title=HuD-mediated+distinct+BDNF+regulatory+pathways+promote+regeneration+after+nerve+injury&doi=10.1016%2Fj.brainres.2017.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">HuD-mediated distinct BDNF regulatory pathways promote regeneration after nerve injury</span></div><div class="casAuthors">Sanna, Maria Domenica; Ghelardini, Carla; Galeotti, Nicoletta</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1659</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">55-63</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Up-regulation of brain-derived neurotrophic factor (BDNF) synthesis is an important mechanism of peripheral nerve regeneration after injury.  However, the cellular and mol. mechanisms underlying this process are not fully understood.  This study examines the role of BDNF in the spared nerve injury (SNI) mice model.  Protein expression and cellular localization were investigated in the dorsal root ganglia (DRG) and spinal cord by western blotting and immunofluorescence expts. resp.  BDNF protein was markedly increased 3 and 7 days post-injury in the spinal cord and DRG.  Following nerve injury sensory neurons produce mols. to promote regeneration, such as growth-assocd. protein 43 (GAP-43) and cytoskeletal proteins.  Our results show that the expression of GAP-43 was increased in the DRG and spinal cord while, an increased of p-NFH content was detected in the spinal cord, with no modification in the DRG.  Both events were counteracted by the administration of an anti-BDNF antibody.  In DRG of SNI mice we also detected an increase of HuD expression, a RNA-binding protein known to stabilize BDNF and GAP-43 mRNA.  Silencing of HuD prevented the nerve injury-induced BDNF and GAP-43 enhanced expression in the DRG.  HuD-mediated BDNF synthesis in the primary sensory neurons, is followed by an anterograde transport of the neurotrophin to the central terminals of the primary afferents in the spinal dorsal horn, to modulate GAP-43 and NFH activation.  Our data suggest that BDNF, GAP-43 and p-NFH proteins increase are linked events required for the enhanced regeneration after nerve injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZPkXrRniCfLVg90H21EOLACvtfcHk0lhrmXJzx_6O0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrtL4%253D&md5=f7b70b966265acd8ebbc337a08d5c3ae</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.brainres.2017.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainres.2017.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DSanna%26aufirst%3DM.%2BD.%26aulast%3DGhelardini%26aufirst%3DC.%26aulast%3DGaleotti%26aufirst%3DN.%26atitle%3DHuD-mediated%2520distinct%2520BDNF%2520regulatory%2520pathways%2520promote%2520regeneration%2520after%2520nerve%2520injury%26jtitle%3DBrain%2520Res.%26date%3D2017%26volume%3D1659%26spage%3D55%26epage%3D63%26doi%3D10.1016%2Fj.brainres.2017.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gusev, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dottorini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkon, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrone, A.</span></span> <span> </span><span class="NLM_article-title">Increase of the RNA-binding protein HuD and posttranscriptional up-regulation of the GAP-43 gene during spatial memory</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">1217</span>– <span class="NLM_lpage">1222</span>, <span class="refDoi"> DOI: 10.1073/pnas.0307674100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.0307674100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=14745023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlWjt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=1217-1222&author=A.+Pascaleauthor=P.+A.+Gusevauthor=M.+Amadioauthor=T.+Dottoriniauthor=S.+Govoniauthor=D.+L.+Alkonauthor=A.+Quattrone&title=Increase+of+the+RNA-binding+protein+HuD+and+posttranscriptional+up-regulation+of+the+GAP-43+gene+during+spatial+memory&doi=10.1073%2Fpnas.0307674100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Increase of the RNA-binding protein HuD and posttranscriptional up-regulation of the GAP-43 gene during spatial memory</span></div><div class="casAuthors">Pascale, Alessia; Gusev, Pavel A.; Amadio, Marialaura; Dottorini, Tania; Govoni, Stefano; Alkon, Daniel L.; Quattrone, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1217-1222</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neuronal ELAV-like proteins (HuB, HuC, and HuD) are highly conserved RNA-binding proteins able to selectively assoc. with the 3' UTR of a subset of target mRNAs and increase their cytoplasmic stability and rate of translation.  We previously demonstrated the involvement of these proteins in learning, reporting that they undergo a sustained up-regulation in the hippocampus of mice trained in a spatial discrimination task.  Here, we extend this finding, showing that a similar up-regulation occurs in the hippocampus of rats trained in another spatial learning paradigm, the Morris water maze.  HuD, a strictly neuron-specific ELAV-like protein, is shown to increase after learning, with a preferential binding to the cytoskeletal fraction.  HuD up-regulation is assocd. with an enhancement of GAP-43 mRNA and protein levels, with an apparently increased HuD colocalization with the GAP-43 mRNA and an increased assocn. of neuronal ELAV-like proteins with the GAP-43 mRNA.  These learning-dependent biochem. events appear to be spatiotemporally controlled, because they do not occur in another brain region involved in learning, the retrosplenial cortex, and at the level of protein expression they show extinction 1 mo after training despite memory retention.  By contrast, HuD mRNA levels still remain increased after 1 mo in the CA1 region.  This persistence may have implications for long-term memory recall.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQJs6DY-0pg7Vg90H21EOLACvtfcHk0lhrmXJzx_6O0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlWjt70%253D&md5=3c4db9a1d503b91e1dd06267b55b1248</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0307674100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0307674100%26sid%3Dliteratum%253Aachs%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DGusev%26aufirst%3DP.%2BA.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DDottorini%26aufirst%3DT.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DAlkon%26aufirst%3DD.%2BL.%26aulast%3DQuattrone%26aufirst%3DA.%26atitle%3DIncrease%2520of%2520the%2520RNA-binding%2520protein%2520HuD%2520and%2520posttranscriptional%2520up-regulation%2520of%2520the%2520GAP-43%2520gene%2520during%2520spatial%2520memory%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D1217%26epage%3D1222%26doi%3D10.1073%2Fpnas.0307674100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akamatsu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuhashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakakibara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, H.</span></span> <span> </span><span class="NLM_article-title">The RNA-binding protein HuD regulates neuronal cell identity and maturation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4625</span>– <span class="NLM_lpage">4630</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407523102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.0407523102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=15764704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFCqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=4625-4630&author=W.+Akamatsuauthor=H.+Fujiharaauthor=T.+Mitsuhashiauthor=M.+Yanoauthor=S.+Shibataauthor=Y.+Hayakawaauthor=H.+J.+Okanoauthor=S.+Sakakibaraauthor=H.+Takanoauthor=T.+Takanoauthor=T.+Takahashiauthor=T.+Nodaauthor=H.+Okano&title=The+RNA-binding+protein+HuD+regulates+neuronal+cell+identity+and+maturation&doi=10.1073%2Fpnas.0407523102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The RNA-binding protein HuD regulates neuronal cell identity and maturation</span></div><div class="casAuthors">Akamatsu, Wado; Fujihara, Hiroaki; Mitsuhashi, Takayuki; Yano, Masato; Shibata, Shinsuke; Hayakawa, Yoshika; Okano, Hirotaka James; Sakakibara, Shin-ichi; Takano, Hiroshi; Takano, Toshiya; Takahashi, Takao; Noda, Tetsuo; Okano, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4625-4630</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neural Hu proteins (HuB/C/D) are RNA-binding proteins that have been shown to induce neuronal differentiation activity when overexpressed in immature neural progenitor cells or undifferentiated neuronal tumors.  Newly generated HuD-deficient mice exhibited a transient impaired-cranial-nerve-development phenotype at an early embryonic stage.  Adult HuD-deficient mice exhibited an abnormal hind-limb reflex and poor rotarod performance.  Anal. of neurosphere formation revealed that the no. and self-renewal capacity of the neural stem/progenitor cells were increased in HuD-deficient mice.  HuD-deficient primary neurospheres also generated a smaller no. of neurons.  Cohort anal. of the cellular proliferative activity by using BrdUrd and iododeoxuridine labeling revealed that the no. of differentiating quiescent cells in the embryonic cerebral wall was decreased.  Long-term administration of BrdUrd revealed that the no. of slowly dividing stem cells in the adult subventricular zone was increased in the HuD-deficient mice.  Taken together, the results suggest that HuD is required at multiple points during neuronal development, including neg. regulation of proliferative activity and neuronal cell-fate acquisition of neural stem/progenitor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKvkth9WQFKLVg90H21EOLACvtfcHk0lgqIzFba31jig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivFCqs7k%253D&md5=465cd4b39906116d684c343569464de1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407523102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407523102%26sid%3Dliteratum%253Aachs%26aulast%3DAkamatsu%26aufirst%3DW.%26aulast%3DFujihara%26aufirst%3DH.%26aulast%3DMitsuhashi%26aufirst%3DT.%26aulast%3DYano%26aufirst%3DM.%26aulast%3DShibata%26aufirst%3DS.%26aulast%3DHayakawa%26aufirst%3DY.%26aulast%3DOkano%26aufirst%3DH.%2BJ.%26aulast%3DSakakibara%26aufirst%3DS.%26aulast%3DTakano%26aufirst%3DH.%26aulast%3DTakano%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DNoda%26aufirst%3DT.%26aulast%3DOkano%26aufirst%3DH.%26atitle%3DThe%2520RNA-binding%2520protein%2520HuD%2520regulates%2520neuronal%2520cell%2520identity%2520and%2520maturation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D4625%26epage%3D4630%26doi%3D10.1073%2Fpnas.0407523102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassell, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossoll, W.</span></span> <span> </span><span class="NLM_article-title">The ALS disease protein TDP-43 is actively transported in motor neuron axons and regulates axon outgrowth</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3703</span>– <span class="NLM_lpage">3718</span>, <span class="refDoi"> DOI: 10.1093/hmg/dds205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fhmg%2Fdds205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=22641816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSrtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=3703-3718&author=C.+Falliniauthor=G.+J.+Bassellauthor=W.+Rossoll&title=The+ALS+disease+protein+TDP-43+is+actively+transported+in+motor+neuron+axons+and+regulates+axon+outgrowth&doi=10.1093%2Fhmg%2Fdds205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The ALS disease protein TDP-43 is actively transported in motor neuron axons and regulates axon outgrowth</span></div><div class="casAuthors">Fallini, Claudia; Bassell, Gary J.; Rossoll, Wilfried</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3703-3718</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disease specifically affecting cortical and spinal motor neurons.  Cytoplasmic inclusions contg. hyperphosphorylated and ubiquitinated TDP-43 are a pathol. hallmark of ALS, and mutations in the gene encoding TDP-43 have been directly linked to the development of the disease.  TDP-43 is a ubiquitous DNA/RNA-binding protein with a nuclear role in pre-mRNA splicing.  However, the selective vulnerability and axonal degeneration of motor neurons in ALS pose the question of whether TDP-43 may have an addnl. role in the regulation of the cytoplasmic and axonal fate of mRNAs, processes important for neuron function.  To investigate this possibility, we have characterized TDP-43 localization and dynamics in primary cultured motor neurons.  Using a combination of cell imaging and biochem. techniques, we demonstrate that TDP-43 is localized and actively transported in live motor neuron axons, and that it co-localizes with well-studied axonal mRNA-binding proteins.  Expression of the TDP-43 C-terminal fragment led to the formation of hyperphosphorylated and ubiquitinated inclusions in motor neuron cell bodies and neurites, and these inclusions specifically sequestered the mRNA-binding protein HuD.  Addnl., we showed that overexpression of full-length or mutant TDP-43 in motor neurons caused a severe impairment in axon outgrowth, which was dependent on the C-terminal protein-interacting domain of TDP-43.  Taken together, our results suggest a role of TDP-43 in the regulation of axonal growth, and suggest that impairment in the post-transcriptional regulation of mRNAs in the cytoplasm of motor neurons may be a major factor in the development of ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS_5CS-Bp0y7Vg90H21EOLACvtfcHk0lgqIzFba31jig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSrtLfO&md5=1e019bc6ecc209d527056b1f1d9cbe87</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fdds205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fdds205%26sid%3Dliteratum%253Aachs%26aulast%3DFallini%26aufirst%3DC.%26aulast%3DBassell%26aufirst%3DG.%2BJ.%26aulast%3DRossoll%26aufirst%3DW.%26atitle%3DThe%2520ALS%2520disease%2520protein%2520TDP-43%2520is%2520actively%2520transported%2520in%2520motor%2520neuron%2520axons%2520and%2520regulates%2520axon%2520outgrowth%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2012%26volume%3D21%26spage%3D3703%26epage%3D3718%26doi%3D10.1093%2Fhmg%2Fdds205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. W.</span></span> <span> </span><span class="NLM_article-title">Role of HuD in nervous system function and pathology</span>. <i>Front. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>S5</i></span>,  <span class="NLM_fpage">554</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.2741/S389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.2741%2FS389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtV2nt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=S5&publication_year=2013&pages=554-563&author=N.+Perrone-Bizzozeroauthor=C.+W.+Bird&title=Role+of+HuD+in+nervous+system+function+and+pathology&doi=10.2741%2FS389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Role of HuD in nervous system function and pathology</span></div><div class="casAuthors">Perrone-Bizzozero, Nora; Bird, Clark W.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Scholar Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">S5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">554-563</span>CODEN:
                <span class="NLM_cas:coden">FBSEAU</span>;
        ISSN:<span class="NLM_cas:issn">1945-0524</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Hu proteins are a family of RNA-binding proteins (RBPs) that are homologs of Drosophila ELAV, a protein required for nervous system development.  Three of these proteins (HuB, HuC, and HuD) are primarily expressed in neurons.  The fourth member, HuR is ubiquitously expressed in all tissues.  At the mol. level, Hu proteins are known to interact with AU-rich instability conferring sequences in the 3' UTR of specific target mRNAs, stabilizing the mRNAs.  These proteins are not only the best known mRNA stabilizers but also the earliest markers of the neuronal cell lineage.  Among the neuronal Hu proteins, HuD has been shown to accelerate neuronal differentiation and axonal outgrowth in neurons both in culture and in vivo.  In addn., HuD and other Hu proteins participate in synaptic plasticity mechanisms in the mature central nervous system and promote regeneration of peripheral nerves.  Furthermore, HuD has been implicated in pathol. conditions from neurodegenerative disorders such as Parkinson's and Alzheimer's disease to childhood brain tumors.  This review will focus on the involvement of HuD in nervous system function and pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYUHgaqYZNx7Vg90H21EOLACvtfcHk0lgqIzFba31jig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2nt77L&md5=5ba942d739a9c9ae42b1cb3b051675ba</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2741%2FS389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252FS389%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%26aulast%3DBird%26aufirst%3DC.%2BW.%26atitle%3DRole%2520of%2520HuD%2520in%2520nervous%2520system%2520function%2520and%2520pathology%26jtitle%3DFront.%2520Biosci.%26date%3D2013%26volume%3DS5%26spage%3D554%26epage%3D563%26doi%3D10.2741%2FS389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hakak, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxbaum, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haroutunian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fienberg, A. A.</span></span> <span> </span><span class="NLM_article-title">Genome-wide expression analysis reveals dysregulation of myelination-related genes in chronic schizophrenia</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">4746</span>– <span class="NLM_lpage">4751</span>, <span class="refDoi"> DOI: 10.1073/pnas.081071198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.081071198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=11296301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtVant7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=4746-4751&author=Y.+Hakakauthor=J.+R.+Walkerauthor=C.+Liauthor=W.+H.+Wongauthor=K.+L.+Davisauthor=J.+D.+Buxbaumauthor=V.+Haroutunianauthor=A.+A.+Fienberg&title=Genome-wide+expression+analysis+reveals+dysregulation+of+myelination-related+genes+in+chronic+schizophrenia&doi=10.1073%2Fpnas.081071198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide expression analysis reveals dysregulation of myelination-related genes in chronic schizophrenia</span></div><div class="casAuthors">Hakak, Yaron; Walker, John R.; Li, Cheng; Wong, Wing Hung; Davis, Kenneth L.; Buxbaum, Joseph D.; Haroutunian, Vahram; Fienberg, Allen A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4746-4751</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Neuropathol. and brain imaging studies suggest that schizophrenia may result from neurodevelopmental defects.  Cytoarchitectural studies indicate cellular abnormalities suggestive of a disruption in neuronal connectivity in schizophrenia, particularly in the dorsolateral prefrontal cortex.  Yet, the mol. mechanisms underlying these findings remain unclear.  To identify mol. substrates assocd. with schizophrenia, DNA microarray anal. was used to assay gene expression levels in postmortem dorsolateral prefrontal cortex of schizophrenic and control patients.  Genes detd. to have altered expression levels in schizophrenics relative to controls are involved in a no. of biol. processes, including synaptic plasticity, neuronal development, neurotransmission, and signal transduction.  Most notable was the differential expression of myelination-related genes suggesting a disruption in oligodendrocyte function in schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3XChhEHjfyrVg90H21EOLACvtfcHk0lhXVWWnKSRvYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtVant7w%253D&md5=45717c744f07b3906b1221b14c0ab6ed</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.081071198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.081071198%26sid%3Dliteratum%253Aachs%26aulast%3DHakak%26aufirst%3DY.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DW.%2BH.%26aulast%3DDavis%26aufirst%3DK.%2BL.%26aulast%3DBuxbaum%26aufirst%3DJ.%2BD.%26aulast%3DHaroutunian%26aufirst%3DV.%26aulast%3DFienberg%26aufirst%3DA.%2BA.%26atitle%3DGenome-wide%2520expression%2520analysis%2520reveals%2520dysregulation%2520of%2520myelination-related%2520genes%2520in%2520chronic%2520schizophrenia%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D4746%26epage%3D4751%26doi%3D10.1073%2Fpnas.081071198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noureddine, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelly, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Walt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pericak-Vance, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. J.</span></span> <span> </span><span class="NLM_article-title">Association between the neuron-specific RNA-binding protein ELAVL4 and Parkinson disease</span>. <i>Hum Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1007/s00439-005-1259-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1007%2Fs00439-005-1259-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=15827745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlCnt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2005&pages=27-33&author=M.+A.+Noureddineauthor=X.+J.+Qinauthor=S.+A.+Oliveiraauthor=T.+J.+Skellyauthor=J.+van+der%0AWaltauthor=M.+A.+Hauserauthor=M.+A.+Pericak-Vanceauthor=J.+M.+Vanceauthor=Y.+J.+Li&title=Association+between+the+neuron-specific+RNA-binding+protein+ELAVL4+and+Parkinson+disease&doi=10.1007%2Fs00439-005-1259-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Association between the neuron-specific RNA-binding protein ELAVL4 and Parkinson disease</span></div><div class="casAuthors">Noureddine, Maher A.; Qin, Xue-Jun; Oliveira, Sofia A.; Skelly, Tara J.; van der Walt, Joelle; Hauser, Michael A.; Pericak-Vance, Margaret A.; Vance, Jeffery M.; Li, Yi-Ju</div><div class="citationInfo"><span class="NLM_cas:title">Human Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-33</span>CODEN:
                <span class="NLM_cas:coden">HUGEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-6717</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Inflammatory processes have been implicated in the cascade of events that lead to nerve cell death.  In the nervous system, a no. of genes involved in inflammation pathways are regulated post-transcriptionally via the interaction of their mRNAs with specific RNA-binding Hu proteins, the vertebrate homologs of the Drosophila ELAV (for embryonic lethal abnormal vision).  The gene encoding ELAVL4, a member of the Hu family of proteins, is located 2 Mb from the chromosome 1p linkage region peak for age-at-onset (AAO) of Parkinson disease (PD) (LOD = 3.41).  Nine single-nucleotide polymorphisms (SNPs) in ELAVL4 were genotyped for 266 multiplex families (1,223 samples).  Addnl. genotyping in 377 singleton families was performed for a subset of five SNPs (SNPs 1-5) that were not in linkage disequil.  SNP 2 (located in the first intron of ELAVL4) showed a strong significant assocn. with AAO of PD (P = 0.006), and SNP 5 (a coding SNP in ELAVL4) showed a moderately significant assocn. (P = 0.035).  Haplotype anal. revealed that the A-C haplotype at SNPs 2 and 3 has the strongest significant assocn. with AAO (P = 0.0001) among all combinations of two or three loci.  The A-C haplotype remained significant for AAO after the inclusion of the C allele at SNP 5 to this haplotype (A-C-C haplotype, P = 0.00018).  Although SNP 5 was found to assoc. with PD risk in the early-onset subset of PD families (at least one affected with AAO < 40 years, 60 families), we believe that it is a byproduct of its assocn. with AAO.  Taken together, these results suggest a potential role for ELAVL4 as a modifier gene for AAO of PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodgOlJw6z41bVg90H21EOLACvtfcHk0lhXVWWnKSRvYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktlCnt7o%253D&md5=587d249c80c2a5e2d4a7c935365b92a8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00439-005-1259-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00439-005-1259-2%26sid%3Dliteratum%253Aachs%26aulast%3DNoureddine%26aufirst%3DM.%2BA.%26aulast%3DQin%26aufirst%3DX.%2BJ.%26aulast%3DOliveira%26aufirst%3DS.%2BA.%26aulast%3DSkelly%26aufirst%3DT.%2BJ.%26aulast%3Dvan%2Bder%2BWalt%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DM.%2BA.%26aulast%3DPericak-Vance%26aufirst%3DM.%2BA.%26aulast%3DVance%26aufirst%3DJ.%2BM.%26aulast%3DLi%26aufirst%3DY.%2BJ.%26atitle%3DAssociation%2520between%2520the%2520neuron-specific%2520RNA-binding%2520protein%2520ELAVL4%2520and%2520Parkinson%2520disease%26jtitle%3DHum%2520Genet.%26date%3D2005%26volume%3D117%26spage%3D27%26epage%3D33%26doi%3D10.1007%2Fs00439-005-1259-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkon, D. L.</span></span> <span> </span><span class="NLM_article-title">PKCε promotes HuD-Mediated Neprilysin mRNA Stability and Enhances Neprilysin-Induced Aβ Degradation in Brain Neurons</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e97756</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0097756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1371%2Fjournal.pone.0097756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24848988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGqt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=C.+S.+Limauthor=D.+L.+Alkon&title=PKC%CE%B5+promotes+HuD-Mediated+Neprilysin+mRNA+Stability+and+Enhances+Neprilysin-Induced+A%CE%B2+Degradation+in+Brain+Neurons&doi=10.1371%2Fjournal.pone.0097756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">PKCε promotes HuD-mediated neprilysin mRNA stability and enhances neprilysin-induced Aβ degradation in brain neurons</span></div><div class="casAuthors">Lim, Chol Seung; Alkon, Daniel L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e97756/1-e97756/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Amyloid-beta (Aβ) peptide accumulation in the brain is a pathol. hallmark of all forms of Alzheimer's disease.  An imbalance between Aβ prodn. and clearance from the brain may contribute to accumulation of neurotoxic Aβ and subsequent synaptic loss, which is the strongest correlate of the extent of memory loss in AD.  The activity of neprilysin (NEP), a potent Aβ-degrading enzyme, is decreased in the AD brain.  Expression of HuD, an mRNA-binding protein important for synaptogenesis and neuronal plasticity, is also decreased in the AD brain.  HuD is regulated by protein kinase Cε (PKCε), and we previously demonstrated that PKCε activation decreases Aβ levels.  We hypothesized that PKCε acts through HuD to stabilize NEP mRNA, modulate its localization, and support NEP activity.  Conversely, loss of PKCε-activated HuD in AD leads to decreased NEP activity and accumulation of Aβ.  Here we show that HuD is assocd. with NEP mRNA in cultures of human SK-N-SH cells.  Treatment with bryostatin, a PKCε-selective activator, enhanced NEP assocn. with HuD and increased NEP mRNA stability.  Activation of PKCε also increased NEP protein levels, increased NEP phosphorylation, and induced cell surface expression.  In addn., specific PKCε activation directly stimulated NEP activity, leading to degrdn. of a monomeric form of Aβ peptide and decreased Aβ neuronal toxicity, as measured by cell viability.  Bryostatin treatment also rescued Aβ-mediated inhibition of HuD-NEP mRNA binding, NEP protein expression, and NEP cell membrane translocation.  These results suggest that PKCe activation reduces Aβ by up-regulating, via the mRNA-binding protein HuD, Aβ-degrading enzymes such as NEP.  Thus, PKCε activation may have therapeutic efficacy for AD by reducing neurotoxic Aβ accumulation as well as having direct anti-apoptotic and synaptogenic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ0k9zM7N4yLVg90H21EOLACvtfcHk0lhXVWWnKSRvYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGqt73M&md5=b4f8f7f89fdcddd04bbea84f68a7fc13</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0097756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0097756%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DAlkon%26aufirst%3DD.%2BL.%26atitle%3DPKC%25CE%25B5%2520promotes%2520HuD-Mediated%2520Neprilysin%2520mRNA%2520Stability%2520and%2520Enhances%2520Neprilysin-Induced%2520A%25CE%25B2%2520Degradation%2520in%2520Brain%2520Neurons%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0097756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haroutunian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasinetti, G. M.</span></span> <span> </span><span class="NLM_article-title">nELAV proteins alteration in Alzheimer’s disease brain: a novel putative target for amyloid-beta reverberating on AbetaPP processing</span>. <i>J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.3233/JAD-2009-0967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.3233%2FJAD-2009-0967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=19221430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BD1M3mtlWnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=409-419&author=M.+Amadioauthor=A.+Pascaleauthor=J.+Wangauthor=L.+Hoauthor=A.+Quattroneauthor=S.+Gandyauthor=V.+Haroutunianauthor=M.+Racchiauthor=G.+M.+Pasinetti&title=nELAV+proteins+alteration+in+Alzheimer%E2%80%99s+disease+brain%3A+a+novel+putative+target+for+amyloid-beta+reverberating+on+AbetaPP+processing&doi=10.3233%2FJAD-2009-0967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">nELAV proteins alteration in Alzheimer's disease brain: a novel putative target for amyloid-beta reverberating on AbetaPP processing</span></div><div class="casAuthors">Amadio Marialaura; Pascale Alessia; Wang Jun; Ho Lap; Quattrone Alessandro; Gandy Sam; Haroutunian Vahram; Racchi Marco; Pasinetti Giulio Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Alzheimer's disease : JAD</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">409-19</span>
        ISSN:<span class="NLM_cas:issn">1387-2877</span>.
    </div><div class="casAbstract">Neuronal ELAV (nELAV) proteins are RNA-binding proteins which play a physiological role in controlling gene expression in memory formation, and their alteration may contribute to cognitive impairment associated with neurodegenerative pathologies such as Alzheimer's disease (AD).  Indeed, we found that the content of nELAV proteins is significantly decreased along with clinical dementia progression in the hippocampi of AD brains, where it inversely correlates with the amount of amyloid-beta (Abeta).  To check the direct influence of Abeta on nELAV, we performed in vitro experiments using human SH-SY5Y cells, finding that Abeta(1-42) specifically determines nELAV proteins reduction.  Since ADAM10 mRNA has the predicted sequences targeted by nELAV, we investigated whether Abeta, through nELAV proteins, could originate a vicious circle affecting amyloid-beta protein precursor (AbetaPP) processing.  Immunoprecipitation experiments showed that indeed nELAV proteins bind to ADAM10 mRNA and that this binding is disrupted by Abeta(1-42) exposure, resulting in a decreased ADAM10 protein expression.  ADAM10 protein diminution was also found in AD hippocampi.  These data show for the first time the involvement of nELAV in AD pathology and suggest that their alteration may affect genes implicated in AbetaPP processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCmsmmtP0GCWBAiYWy6n3SfW6udTcc2ebNCnmcQnFpOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3mtlWnug%253D%253D&md5=581fc30d7c4fa95ae8d4b80fdc6a0dde</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3233%2FJAD-2009-0967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJAD-2009-0967%26sid%3Dliteratum%253Aachs%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DQuattrone%26aufirst%3DA.%26aulast%3DGandy%26aufirst%3DS.%26aulast%3DHaroutunian%26aufirst%3DV.%26aulast%3DRacchi%26aufirst%3DM.%26aulast%3DPasinetti%26aufirst%3DG.%2BM.%26atitle%3DnELAV%2520proteins%2520alteration%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%2520brain%253A%2520a%2520novel%2520putative%2520target%2520for%2520amyloid-beta%2520reverberating%2520on%2520AbetaPP%2520processing%26jtitle%3DJ.%2520Alzheimers%2520Dis.%26date%3D2009%26volume%3D16%26spage%3D409%26epage%3D419%26doi%3D10.3233%2FJAD-2009-0967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marchesi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombrita, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span> <span> </span><span class="NLM_article-title">PKC Activation Counteracts ADAM10 Deficit in HuD-Silenced Neuroblastoma Cells</span>. <i>J. Alzheimers Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.3233/JAD-160299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.3233%2FJAD-160299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=27472877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BC2s3ls1CisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2016&pages=535-547&author=N.+Marchesiauthor=M.+Amadioauthor=C.+Colombritaauthor=S.+Govoniauthor=A.+Rattiauthor=A.+Pascale&title=PKC+Activation+Counteracts+ADAM10+Deficit+in+HuD-Silenced+Neuroblastoma+Cells&doi=10.3233%2FJAD-160299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PKC Activation Counteracts ADAM10 Deficit in HuD-Silenced Neuroblastoma Cells</span></div><div class="casAuthors">Marchesi Nicoletta; Amadio Marialaura; Govoni Stefano; Pascale Alessia; Colombrita Claudia; Ratti Antonia; Colombrita Claudia; Ratti Antonia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Alzheimer's disease : JAD</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">535-47</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neuronal ELAV/Hu (nELAV) are RNA-binding proteins that mainly regulate gene expression by increasing the stability and/or translation rate of target mRNAs bearing ARE (adenine and uracil-rich elements) sequences.  Among nELAV target transcripts there is ADAM10, an α-secretase involved in the non-amyloidogenic processing of the amyloid-β protein precursor (AβPP) which leads to the production of the neuroprotective sAβPPα peptide.  The aim of this study was to evaluate if nELAV depletion affects ADAM10 expression in human SH-SY5Y neuroblastoma cells.  We also studied the effects of Bryostatin-1, a molecule able to activate nELAV protein cascade.  The specific HuD/nELAV gene silencing decreased both nELAV and ADAM10 protein contents; similar results were obtained by Aβ40 treatment in wild-type SH-SY5Y cells.  In HuD-silenced cells, the exposure to Bryostatin-1 counteracted both nELAV and ADAM10 proteins downregulation, by restoring nELAV/ADAM10 basal levels.  We also found that sAβPPα release, which seemed not to be compromised by Aβ40 challenge or HuD-silencing, was favored by Bryostatin-1.  Overall, these findings strongly suggest that a deficiency in nELAV content negatively affects ADAM10 expression and may play a role in neurodegenerative diseases, which may benefit by molecules activating ELAV cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSihkLH2ne6q7FU0A1HOshhfW6udTcc2ebhXRJP-e0VJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3ls1CisA%253D%253D&md5=21faa73ec391f8f0e69091c695df90ca</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3233%2FJAD-160299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJAD-160299%26sid%3Dliteratum%253Aachs%26aulast%3DMarchesi%26aufirst%3DN.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DColombrita%26aufirst%3DC.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DRatti%26aufirst%3DA.%26aulast%3DPascale%26aufirst%3DA.%26atitle%3DPKC%2520Activation%2520Counteracts%2520ADAM10%2520Deficit%2520in%2520HuD-Silenced%2520Neuroblastoma%2520Cells%26jtitle%3DJ.%2520Alzheimers%2520Dis.%26date%3D2016%26volume%3D54%26spage%3D535%26epage%3D547%26doi%3D10.3233%2FJAD-160299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelmohsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grammatikakis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noh, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martindale, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faghihi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlestedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troncoso, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Cabo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorospe, M.</span></span> <span> </span><span class="NLM_article-title">HuD regulates coding and noncoding RNA to induce APP→Aβ processing</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1401</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.04.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.celrep.2014.04.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24857657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=1401-1409&author=M.+J.+Kangauthor=K.+Abdelmohsenauthor=E.+R.+Hutchisonauthor=S.+J.+Mitchellauthor=I.+Grammatikakisauthor=R.+Guoauthor=J.+H.+Nohauthor=J.+L.+Martindaleauthor=X.+Yangauthor=E.+K.+Leeauthor=M.+A.+Faghihiauthor=C.+Wahlestedtauthor=J.+C.+Troncosoauthor=O.+Pletnikovaauthor=N.+Perrone-Bizzozeroauthor=S.+M.+Resnickauthor=R.+de+Caboauthor=M.+P.+Mattsonauthor=M.+Gorospe&title=HuD+regulates+coding+and+noncoding+RNA+to+induce+APP%E2%86%92A%CE%B2+processing&doi=10.1016%2Fj.celrep.2014.04.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">HuD Regulates Coding and Noncoding RNA to Induce APP→Aβ Processing</span></div><div class="casAuthors">Kang, Min-Ju; Abdelmohsen, Kotb; Hutchison, Emmette R.; Mitchell, Sarah J.; Grammatikakis, Ioannis; Guo, Rong; Noh, Ji Heon; Martindale, Jennifer L.; Yang, Xiaoling; Lee, Eun Kyung; Faghihi, Mohammad A.; Wahlestedt, Claes; Troncoso, Juan C.; Pletnikova, Olga; Perrone-Bizzozero, Nora; Resnick, Susan M.; de Cabo, Rafael; Mattson, Mark P.; Gorospe, Myriam</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1401-1409</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The primarily neuronal RNA-binding protein HuD is implicated in learning and memory.  Here, we report the identification of several HuD target transcripts linked to Alzheimer's disease (AD) pathogenesis.  HuD interacted with the 3' UTRs of APP mRNA (encoding amyloid precursor protein) and BACE1 mRNA (encoding β-site APP-cleaving enzyme 1) and increased the half-lives of these mRNAs.  HuD also assocd. with and stabilized the long noncoding (lnc)RNA BACE1AS, which partly complements BACE1 mRNA and enhances BACE1 expression.  Consistent with HuD promoting prodn. of APP and APP-cleaving enzyme, the levels of APP, BACE1, BACE1AS, and Aβ were higher in the brain of HuD-overexpressing mice.  Importantly, cortex (superior temporal gyrus) from patients with AD displayed significantly higher levels of HuD and, accordingly, elevated APP, BACE1, BACE1AS, and Aβ than did cortical tissue from healthy age-matched individuals.  We propose that HuD jointly promotes the prodn. of APP and the cleavage of its amyloidogenic fragment, Aβ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_4Ooj2xbBJLVg90H21EOLACvtfcHk0lhleOGk3f4OlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslyit78%253D&md5=7bc526b55d6e59d3de164e44605ab3f4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.04.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.04.050%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DM.%2BJ.%26aulast%3DAbdelmohsen%26aufirst%3DK.%26aulast%3DHutchison%26aufirst%3DE.%2BR.%26aulast%3DMitchell%26aufirst%3DS.%2BJ.%26aulast%3DGrammatikakis%26aufirst%3DI.%26aulast%3DGuo%26aufirst%3DR.%26aulast%3DNoh%26aufirst%3DJ.%2BH.%26aulast%3DMartindale%26aufirst%3DJ.%2BL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DE.%2BK.%26aulast%3DFaghihi%26aufirst%3DM.%2BA.%26aulast%3DWahlestedt%26aufirst%3DC.%26aulast%3DTroncoso%26aufirst%3DJ.%2BC.%26aulast%3DPletnikova%26aufirst%3DO.%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%26aulast%3DResnick%26aufirst%3DS.%2BM.%26aulast%3Dde%2BCabo%26aufirst%3DR.%26aulast%3DMattson%26aufirst%3DM.%2BP.%26aulast%3DGorospe%26aufirst%3DM.%26atitle%3DHuD%2520regulates%2520coding%2520and%2520noncoding%2520RNA%2520to%2520induce%2520APP%25E2%2586%2592A%25CE%25B2%2520processing%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D7%26spage%3D1401%26epage%3D1409%26doi%3D10.1016%2Fj.celrep.2014.04.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subhadra, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeds, N. W.</span></span> <span> </span><span class="NLM_article-title">Neuroserpin up-regulation in the Alzheimer’s disease brain is associated with elevated thyroid hormone receptor-β1 and HuD expression</span>. <i>Neurochem. Int.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1016/j.neuint.2013.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.neuint.2013.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24036060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1KjtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=476-481&author=B.+Subhadraauthor=K.+Schallerauthor=N.+W.+Seeds&title=Neuroserpin+up-regulation+in+the+Alzheimer%E2%80%99s+disease+brain+is+associated+with+elevated+thyroid+hormone+receptor-%CE%B21+and+HuD+expression&doi=10.1016%2Fj.neuint.2013.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Neuroserpin up-regulation in the Alzheimer's disease brain is associated with elevated thyroid hormone receptor-β1 and HuD expression</span></div><div class="casAuthors">Subhadra, Bobban; Schaller, Kristin; Seeds, Nicholas W.</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry International</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">476-481</span>CODEN:
                <span class="NLM_cas:coden">NEUIDS</span>;
        ISSN:<span class="NLM_cas:issn">0197-0186</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Neuroserpin, the major inhibitor of tissue plasminogen activator (tPA) in brain, has been shown to be up-regulated in Alzheimer's disease (AD).  Inhibition of tPA activity leads to reduced brain levels of plasmin, one of the main enzymes responsible for the degrdn. and clearance of amyloid-beta and its plaques from the brain.  Thyroid hormone is one of the few factors known to enhance expression of neuroserpin in neurons.  Thyroid hormone acts on neurons by binding to its receptors THR1α and THR1β, which then function in the nucleus to up-regulate the expression of numerous genes including the RNA-binding protein HuD.  HuD acts post-transcriptionally to enhance expression of numerous proteins including neuroserpin by stabilizing their mRNAs.  A series of Alzheimer's disease brain tissues were compared to age-matched control brains for their expression of neuroserpin, THRβ1 and HuD by western blotting.  Alzheimer's disease brain tissues with elevated neuroserpin protein also showed increased expression of THRβ1 and HuD.  Pair-wise analyses showed significant correlation p-values between neuroserpin, THRβ1 and HuD levels; suggesting that the up-regulation of neuroserpin in Alzheimer's disease brain may result from an activation of the thyroid hormone response system in these individuals.  These findings provide evidence for a potential relationship between thyroid hormone disorders and Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqegCfn9IjJ2bVg90H21EOLACvtfcHk0lhleOGk3f4OlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1KjtLfP&md5=ddb630161929307387da2686aff92da2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2013.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2013.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DSubhadra%26aufirst%3DB.%26aulast%3DSchaller%26aufirst%3DK.%26aulast%3DSeeds%26aufirst%3DN.%2BW.%26atitle%3DNeuroserpin%2520up-regulation%2520in%2520the%2520Alzheimer%25E2%2580%2599s%2520disease%2520brain%2520is%2520associated%2520with%2520elevated%2520thyroid%2520hormone%2520receptor-%25CE%25B21%2520and%2520HuD%2520expression%26jtitle%3DNeurochem.%2520Int.%26date%3D2013%26volume%3D63%26spage%3D476%26epage%3D481%26doi%3D10.1016%2Fj.neuint.2013.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perrone-Bizzozero, N.</span></span> <span> </span><span class="NLM_article-title">Neuropsychiatric Implications Of RNA-Binding Proteins HuD And KSRP Revealed By Genome-Wide Identification Of Their Targets</span>. <i>Eur. Neuropsychopharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>, <span class="NLM_elocation-id">S721</span> <span class="refDoi"> DOI: 10.1016/j.euroneuro.2017.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.euroneuro.2017.06.036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&author=N.+Perrone-Bizzozero&title=Neuropsychiatric+Implications+Of+RNA-Binding+Proteins+HuD+And+KSRP+Revealed+By+Genome-Wide+Identification+Of+Their+Targets&doi=10.1016%2Fj.euroneuro.2017.06.036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.euroneuro.2017.06.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.euroneuro.2017.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone-Bizzozero%26aufirst%3DN.%26atitle%3DNeuropsychiatric%2520Implications%2520Of%2520RNA-Binding%2520Proteins%2520HuD%2520And%2520KSRP%2520Revealed%2520By%2520Genome-Wide%2520Identification%2520Of%2520Their%2520Targets%26jtitle%3DEur.%2520Neuropsychopharmacol.%26date%3D2019%26volume%3D29%26doi%3D10.1016%2Fj.euroneuro.2017.06.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka Hall, T. M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for recognition of AU-rich element RNA by the HuD protein</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1038/84131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1038%2F84131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=11175903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtVShtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=141-145&author=X.+Wangauthor=T.+M.+Tanaka+Hall&title=Structural+basis+for+recognition+of+AU-rich+element+RNA+by+the+HuD+protein&doi=10.1038%2F84131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for recognition of AU-rich element RNA by the HuD protein</span></div><div class="casAuthors">Wang, Xiaoqiang; Hall, Traci M. Tanaka</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-145</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Hu proteins bind to adenosine-uridine (AU)-rich elements (AREs) in the 3' untranslated regions of many short-lived mRNAs, thereby stabilizing them.  Here the authors report the crystal structures of the first two RNA recognition motif (RRM) domains of the HuD protein in complex with an 11-nucleotide fragment of a class I ARE (the c-fos ARE; to 1.8 Å), and with an 11-nucleotide fragment of a class II ARE (the tumor necrosis factor α ARE; to 2.3 Å).  These structures reveal a consensus RNA recognition sequence that suggests a preference for pyrimidine-rich sequences and a requirement for a central uracil residue in the clustered AUUUA repeats found in class II AREs.  Comparison to structures of other RRM domain-nucleic acid complexes reveals two base recognition pockets in all the structures that interact with bases using residues in conserved ribonucleoprotein motifs and at the C-terminal ends of RRM domains.  Different conformations of nucleic acid can be bound by RRM domains by using different combinations of base recognition pockets and multiple RRM domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon8csOdxeA7rVg90H21EOLACvtfcHk0lg41uCY3TQ4mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtVShtbs%253D&md5=8176233d7440eb9dc0c6be318419841d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2F84131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F84131%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DTanaka%2BHall%26aufirst%3DT.%2BM.%26atitle%3DStructural%2520basis%2520for%2520recognition%2520of%2520AU-rich%2520element%2520RNA%2520by%2520the%2520HuD%2520protein%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D141%26epage%3D145%26doi%3D10.1038%2F84131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toba, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K.</span></span> <span> </span><span class="NLM_article-title">The third RNA recognition motif of Drosophila ELAV protein has a role in multimerization</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1390</span>– <span class="NLM_lpage">1399</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm1168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkm1168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=18203745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislKiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=1390-1399&author=G.+Tobaauthor=K.+White&title=The+third+RNA+recognition+motif+of+Drosophila+ELAV+protein+has+a+role+in+multimerization&doi=10.1093%2Fnar%2Fgkm1168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The third RNA recognition motif of Drosophila ELAV protein has a role in multimerization</span></div><div class="casAuthors">Toba, Gakuta; White, Kalpana</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1390-1399</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ELAV is a neuron-specific RNA-binding protein in Drosophila that is required for development and maintenance of neurons.  ELAV regulates alternative splicing of Neuroglian and erect wing (ewg) transcripts, and has been shown to form a multimeric complex on the last ewg intron.  The protein has three RNA recognition motifs (RRM1, 2 and 3) with a hinge region between RRM2 and 3.  In this study, we used the yeast two-hybrid system to det. the multimerization domain of ELAV.  Using deletion constructs, we mapped an interaction activity to a region contg. most of RRM3.  We found three conserved short sequences in RRM3 that were essential for the interaction, and also sufficient to give the interaction activity to RRM2 when introduced into it.  In our in vivo functional assay, a mutation in one of the three sequences showed reduced activity in splicing regulation, underlining the functional importance of multimerization.  However, RRM2 with the three RRM3 interaction sequences did not function as RRM3 in vivo, which suggested that multimerization is not the only function of RRM3.  Our results are consistent with a model in which RRM3 serves as a bi-functional domain that interacts with both RNA and protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPofMdm8XepbVg90H21EOLACvtfcHk0lg41uCY3TQ4mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislKiu7s%253D&md5=874693a3d7e6e26eca3c0d8bb87f42c6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm1168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm1168%26sid%3Dliteratum%253Aachs%26aulast%3DToba%26aufirst%3DG.%26aulast%3DWhite%26aufirst%3DK.%26atitle%3DThe%2520third%2520RNA%2520recognition%2520motif%2520of%2520Drosophila%2520ELAV%2520protein%2520has%2520a%2520role%2520in%2520multimerization%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D1390%26epage%3D1399%26doi%3D10.1093%2Fnar%2Fgkm1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baraglia, A. C.</span></span> <span> </span><span class="NLM_article-title">Multitarget drug design strategy applied to the system PKC/ELAV proteins/mRNA</span>. <i>Sci. Acta</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">80</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=76-80&author=A.+C.+Baraglia&title=Multitarget+drug+design+strategy+applied+to+the+system+PKC%2FELAV+proteins%2FmRNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaraglia%26aufirst%3DA.%2BC.%26atitle%3DMultitarget%2520drug%2520design%2520strategy%2520applied%2520to%2520the%2520system%2520PKC%252FELAV%2520proteins%252FmRNA%26jtitle%3DSci.%2520Acta%26date%3D2009%26volume%3D3%26spage%3D76%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pricl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaggeri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Identification of peptides with ELAV-like mRNA-stabilizing effect: an integrated in vitro/in silico approach</span>. <i>Chem. Biol. Drug. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">707</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1111%2Fcbdd.12117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=23398665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlOhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=707-714&author=M.+Amadioauthor=A.+Pascaleauthor=S.+Govoniauthor=E.+Lauriniauthor=S.+Priclauthor=R.+Gaggeriauthor=D.+Rossiauthor=S.+Collina&title=Identification+of+peptides+with+ELAV-like+mRNA-stabilizing+effect%3A+an+integrated+in+vitro%2Fin+silico+approach&doi=10.1111%2Fcbdd.12117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of peptides with ELAV-like mRNA-stabilizing effect: an integrated in vitro/in silico approach</span></div><div class="casAuthors">Amadio, Marialaura; Pascale, Alessia; Govoni, Stefano; Laurini, Erik; Pricl, Sabrina; Gaggeri, Raffaella; Rossi, Daniela; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">707-714</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Embryonic lethal abnormal vision (ELAV) proteins are RNA-binding proteins that bind specific adenine and uridine-rich elements mainly located in the 3'-untranslated region of target mRNAs, preventing their otherwise rapid degrdn. and thus increasing gene expression.  Starting from their previous discovery and applying an integrated in vitro/in silico approach, the authors report, here, a deeper understanding of the mRNA-stabilizing activity of four peptides derived from the ELAV proteins structure.  The stabilizing effect on the VEGF transcript (mRNAVEGF) exerted by each peptide, tested individually, was initially evaluated, and no effects were evidenced.  Hence, the biol. effects of all peptides couples were investigated.  Interestingly, in accordance with preliminary mol. dynamics results, only one of all possible peptide couples resulted highly effective in stabilizing mRNAVEGF.  These two peptides were thus identified as valuable starting point for designing small mols. with ELAV-mimicking properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopEudduyPj3LVg90H21EOLACvtfcHk0lg41uCY3TQ4mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlOhs7g%253D&md5=d8c72a22107f0c7897ce7866967fa284</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12117%26sid%3Dliteratum%253Aachs%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DLaurini%26aufirst%3DE.%26aulast%3DPricl%26aufirst%3DS.%26aulast%3DGaggeri%26aufirst%3DR.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520peptides%2520with%2520ELAV-like%2520mRNA-stabilizing%2520effect%253A%2520an%2520integrated%2520in%2520vitro%252Fin%2520silico%2520approach%26jtitle%3DChem.%2520Biol.%2520Drug.%2520Des.%26date%3D2013%26volume%3D81%26spage%3D707%26epage%3D714%26doi%3D10.1111%2Fcbdd.12117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carnevale Baraglia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzolina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Small Peptides Derived from Embryonic Lethal Abnormal Vision Proteins Structure Showing RNA-Stabilizing Properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5017</span>– <span class="NLM_lpage">5019</span>, <span class="refDoi"> DOI: 10.1021/jm900741e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900741e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlSgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5017-5019&author=D.+Rossiauthor=M.+Amadioauthor=A.+Carnevale+Baragliaauthor=O.+Azzolinaauthor=A.+Rattiauthor=S.+Govoniauthor=A.+Pascaleauthor=S.+Collina&title=Discovery+of+Small+Peptides+Derived+from+Embryonic+Lethal+Abnormal+Vision+Proteins+Structure+Showing+RNA-Stabilizing+Properties&doi=10.1021%2Fjm900741e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Peptides Derived from Embryonic Lethal Abnormal Vision Proteins Structure Showing RNA-Stabilizing Properties</span></div><div class="casAuthors">Rossi, Daniela; Amadio, Marialaura; Carnevale Baraglia, Anna; Azzolina, Ornella; Ratti, Antonia; Govoni, Stefano; Pascale, Alessia; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5017-5019</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four peptides corresponding to highly conserved motifs within the first two RNA recognition motif-type domains of ELAV proteins were prepd., and their effect on the stability of NOVA-1 and VEGF ELAV-target mRNAs was evaluated.  Biol. results show that in the presence of phorbol 12-myristate 13-acetate (a PKC activator triggering the ELAV pathway), an equimolar mixt. of peptides induces a statistically significant stabilization of the selected transcripts, suggesting a synergic effect of the two stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIP-aXg_U6ebVg90H21EOLACvtfcHk0ljwxYMBWaJrPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlSgt7Y%253D&md5=9392e8393a0d6e1437cebc88a9bc0956</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm900741e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900741e%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DCarnevale%2BBaraglia%26aufirst%3DA.%26aulast%3DAzzolina%26aufirst%3DO.%26aulast%3DRatti%26aufirst%3DA.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Small%2520Peptides%2520Derived%2520from%2520Embryonic%2520Lethal%2520Abnormal%2520Vision%2520Proteins%2520Structure%2520Showing%2520RNA-Stabilizing%2520Properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5017%26epage%3D5019%26doi%3D10.1021%2Fjm900741e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span> <span> </span><span class="NLM_article-title">Diffusion-Ordered Spectroscopy and Saturation Transfer Difference NMR Spectroscopy Studies of Selective Interactions between ELAV Protein Fragments and an mRNA Target</span>. <i>Euro J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">6399</span>– <span class="NLM_lpage">6404</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201403014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fejoc.201403014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=6399-6404&author=F.+Vasileauthor=D.+Rossiauthor=S.+Collinaauthor=D.+Potenza&title=Diffusion-Ordered+Spectroscopy+and+Saturation+Transfer+Difference+NMR+Spectroscopy+Studies+of+Selective+Interactions+between+ELAV+Protein+Fragments+and+an+mRNA+Target&doi=10.1002%2Fejoc.201403014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201403014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201403014%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DCollina%26aufirst%3DS.%26aulast%3DPotenza%26aufirst%3DD.%26atitle%3DDiffusion-Ordered%2520Spectroscopy%2520and%2520Saturation%2520Transfer%2520Difference%2520NMR%2520Spectroscopy%2520Studies%2520of%2520Selective%2520Interactions%2520between%2520ELAV%2520Protein%2520Fragments%2520and%2520an%2520mRNA%2520Target%26jtitle%3DEuro%2520J.%2520Org.%2520Chem.%26date%3D2014%26volume%3D29%26spage%3D6399%26epage%3D6404%26doi%3D10.1002%2Fejoc.201403014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nasti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unver, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Compounds Interfering with Embryonic Lethal Abnormal Vision (ELAV) Protein-RNA Complexes: An Avenue for Discovering New Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8257</span>– <span class="NLM_lpage">8267</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01871</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01871" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Knt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8257-8267&author=R.+Nastiauthor=D.+Rossiauthor=M.+Amadioauthor=A.+Pascaleauthor=M.+Y.+Unverauthor=A.+K.+H.+Hirschauthor=S.+Collina&title=Compounds+Interfering+with+Embryonic+Lethal+Abnormal+Vision+%28ELAV%29+Protein-RNA+Complexes%3A+An+Avenue+for+Discovering+New+Drugs&doi=10.1021%2Facs.jmedchem.6b01871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Compounds Interfering with Embryonic Lethal Abnormal Vision (ELAV) Protein-RNA Complexes: An Avenue for Discovering New Drugs</span></div><div class="casAuthors">Nasti, Rita; Rossi, Daniela; Amadio, Marialaura; Pascale, Alessia; Unver, M. Yagiz; Hirsch, Anna K. H.; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8257-8267</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  RNA-binding proteins play a key role in post-transcriptional processes.  Among these proteins, embryonic lethal abnormal vision (ELAV) proteins are among the best described.  ELAV proteins predominantly act as pos. regulators of gene expression, and their dysregulation is involved in several pathologies, such as cancer, inflammation, and neurodegenerative diseases.  Only a few structurally unrelated compds. interfering with ELAV protein-mRNA complexes have been identified by applying high-throughput screening approaches.  Considering the structural diversity of the compds. discovered so far and the different techniques employed for screening their ability to interfere with ELAV protein-mRNA complexes, drawing conclusions from structure-activity relationships remains a challenge.  The authors performed docking studies to understand the interactions of compds. reported over the past decade to be inhibitors of ELAV proteins and to evaluate the potential of computer-aided drug design to target this family of proteins for further drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzpb1q7pqzprVg90H21EOLACvtfcHk0lgcjrG2vXYKsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Knt7g%253D&md5=d320ddeb667d511959e882e9ab6681d6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01871%26sid%3Dliteratum%253Aachs%26aulast%3DNasti%26aufirst%3DR.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DUnver%26aufirst%3DM.%2BY.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DCompounds%2520Interfering%2520with%2520Embryonic%2520Lethal%2520Abnormal%2520Vision%2520%2528ELAV%2529%2520Protein-RNA%2520Complexes%253A%2520An%2520Avenue%2520for%2520Discovering%2520New%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8257%26epage%3D8267%26doi%3D10.1021%2Facs.jmedchem.6b01871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moraca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talarico, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagliardi, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Distinto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maccioni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tramontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artese, A.</span></span> <span> </span><span class="NLM_article-title">Novel natural non-nucleoside inhibitors of HIV-1 reverse transcriptase identified by shape- and structure-based virtual screening techniques</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.10.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.ejmech.2018.10.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=30342421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFOns7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2019&pages=1-10&author=G.+Costaauthor=R.+Roccaauthor=A.+Coronaauthor=N.+Grandiauthor=F.+Moracaauthor=I.+Romeoauthor=C.+Talaricoauthor=M.+G.+Gagliardiauthor=F.+A.+Ambrosioauthor=F.+Ortusoauthor=S.+Alcaroauthor=S.+Distintoauthor=E.+Maccioniauthor=E.+Tramontanoauthor=A.+Artese&title=Novel+natural+non-nucleoside+inhibitors+of+HIV-1+reverse+transcriptase+identified+by+shape-+and+structure-based+virtual+screening+techniques&doi=10.1016%2Fj.ejmech.2018.10.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel natural non-nucleoside inhibitors of HIV-1 reverse transcriptase identified by shape- and structure-based virtual screening techniques</span></div><div class="casAuthors">Costa, Giosue; Rocca, Roberta; Corona, Angela; Grandi, Nicole; Moraca, Federica; Romeo, Isabella; Talarico, Carmine; Gagliardi, Maria Giovanna; Ambrosio, Francesca Alessandra; Ortuso, Francesco; Alcaro, Stefano; Distinto, Simona; Maccioni, Elias; Tramontano, Enzo; Artese, Anna</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In this work the authors report a parallel application of both docking- and shape-based virtual screening (VS) methods, followed by Mol. Dynamics simulations (MDs), for discovering new compds. able to inhibit the human immunodeficiency virus type 1 (HIV-1) reverse transcriptase (RT) RNA-dependent DNA polymerase activity.  Specifically, the authors screened more than 143000 natural compds. com. available in the ZINC database against the best five RT crystallog. models, taking into account the five approved NNRTIs as query compds.  As a result, 20 hit mols. were selected and tested on biochem. assays for the inhibition of the RNA dependent DNA polymerase RT function and, among them, an indoline pyrrolidine (hit 1), an indonyl piperazine (hit 2) and an indolyl indolinone (hit 3) derivs. were identified as novel nonnucleoside RT inhibitors in the low micromolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxkhVzeZgmwrVg90H21EOLACvtfcHk0lgcjrG2vXYKsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFOns7zM&md5=c9de900cbade80d59e06b4843e2db647</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.029%26sid%3Dliteratum%253Aachs%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DRocca%26aufirst%3DR.%26aulast%3DCorona%26aufirst%3DA.%26aulast%3DGrandi%26aufirst%3DN.%26aulast%3DMoraca%26aufirst%3DF.%26aulast%3DRomeo%26aufirst%3DI.%26aulast%3DTalarico%26aufirst%3DC.%26aulast%3DGagliardi%26aufirst%3DM.%2BG.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DOrtuso%26aufirst%3DF.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DDistinto%26aufirst%3DS.%26aulast%3DMaccioni%26aufirst%3DE.%26aulast%3DTramontano%26aufirst%3DE.%26aulast%3DArtese%26aufirst%3DA.%26atitle%3DNovel%2520natural%2520non-nucleoside%2520inhibitors%2520of%2520HIV-1%2520reverse%2520transcriptase%2520identified%2520by%2520shape-%2520and%2520structure-based%2520virtual%2520screening%2520techniques%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D161%26spage%3D1%26epage%3D10%26doi%3D10.1016%2Fj.ejmech.2018.10.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ripphausen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nisius, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltason, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Quo vadis, virtual screening? A comprehensive survey of prospective applications</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8461</span>– <span class="NLM_lpage">8467</span>, <span class="refDoi"> DOI: 10.1021/jm101020z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101020z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GqtrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8461-8467&author=P.+Ripphausenauthor=B.+Nisiusauthor=L.+Peltasonauthor=J.+Bajorath&title=Quo+vadis%2C+virtual+screening%3F+A+comprehensive+survey+of+prospective+applications&doi=10.1021%2Fjm101020z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Quo Vadis, Virtual Screening? A Comprehensive Survey of Prospective Applications</span></div><div class="casAuthors">Ripphausen, Peter; Nisius, Britta; Peltason, Lisa; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8461-8467</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Our anal. has revealed that the majority of prospective virtual screening (VS) studies have thus far been reported in only a small no. of journals.  In addn., we have found that docking investigations clearly dominate the prospective VS field at present.  We see an increasing no. of successful VS studies that are based on docking into homol. models, and the potency of such identified hits is on av. higher than for hits identified by docking into X-ray structures, although model-based docking calcns. are more approx. in nature.  In this context, it is interesting to note that NMR structures have thus far only rarely been used in practical VS applications.  Currently there are three times more structure-based virtual screening (SBVS) than ligand-based virtual screening (LBVS) applications available, but LBVS methods identify hits that are on av. more potent.  In particular, nearly all of the still limited no. of LBVS investigations that combine 2D and 3D components yield potent hits.  Furthermore, kinases, G-protein-coupled receptors (GPCRs), and proteases are the most popular VS targets.  For kinases and GPCRs, potent submicromolar VS hits are frequently obtained, in contrast to proteases, where the majority of reported hits are only weakly potent.  However, protein kinases and proteases only represent ∼36 % of all enzymes targeted by VS, and more than 50 % of these enzymes do not belong to the five major families.  Taken together, the results of our anal. have provided us with a rather detailed view of the prospective VS landscape and there have certainly been a few surprises.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHMK13XT0tD7Vg90H21EOLACvtfcHk0lgcjrG2vXYKsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GqtrnK&md5=c20f82a1869c0ea176b177bc74ad92a4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm101020z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101020z%26sid%3Dliteratum%253Aachs%26aulast%3DRipphausen%26aufirst%3DP.%26aulast%3DNisius%26aufirst%3DB.%26aulast%3DPeltason%26aufirst%3DL.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DQuo%2520vadis%252C%2520virtual%2520screening%253F%2520A%2520comprehensive%2520survey%2520of%2520prospective%2520applications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8461%26epage%3D8467%26doi%3D10.1021%2Fjm101020z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Romero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Sastre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Computer-Aided Discovery and Characterization of Novel Ebola Virus Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3582</span>– <span class="NLM_lpage">3594</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntF2nt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3582-3594&author=S.+J.+Capuzziauthor=W.+Sunauthor=E.+N.+Muratovauthor=C.+Mart%C3%ADnez-Romeroauthor=S.+Heauthor=W.+Zhuauthor=H.+Liauthor=G.+Tawaauthor=E.+G.+Fisherauthor=M.+Xuauthor=P.+Shinnauthor=X.+Qiuauthor=A.+Garc%C3%ADa-Sastreauthor=W.+Zhengauthor=A.+Tropsha&title=Computer-Aided+Discovery+and+Characterization+of+Novel+Ebola+Virus+Inhibitors&doi=10.1021%2Facs.jmedchem.8b00035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-Aided Discovery and Characterization of Novel Ebola Virus Inhibitors</span></div><div class="casAuthors">Capuzzi, Stephen J.; Sun, Wei; Muratov, Eugene N.; Martinez-Romero, Carles; He, Shihua; Zhu, Wenjun; Li, Hao; Tawa, Gregory; Fisher, Ethan G.; Xu, Miao; Shinn, Paul; Qiu, Xiangguo; Garcia-Sastre, Adolfo; Zheng, Wei; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3582-3594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Ebola virus (EBOV) causes severe human infection that lacks effective treatment.  A recent screen identified a series of compds. that block EBOV-like particle entry into human cells.  Using data from this screen, quant. structure-activity relationship models were built and employed for virtual screening of a ∼17 million compd. library.  Exptl. testing of 102 hits yielded 14 compds. with IC50 values under 10 μM, including several sub-micromolar inhibitors, and more than 10-fold selectivity against host cytotoxicity.  These confirmed hits include FDA-approved drugs and clin. candidates with non-antiviral indications, as well as compds. with novel scaffolds and no previously known bioactivity.  Five selected hits inhibited BSL-4 live-EBOV infection in a dose-dependent manner, including vindesine (0.34 μM).  Addnl. studies of these novel anti-EBOV compds. revealed their mechanisms of action, including the inhibition of NPC1 protein, cathepsin B/L, and lysosomal function.  Compds. identified in this study are among the most potent and well-characterized anti-EBOV inhibitors reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPJn7FlyPoBbVg90H21EOLACvtfcHk0lh4FoyjruTppg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntF2nt78%253D&md5=9091a9598c7d8c40539bcd5b73c683df</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00035%26sid%3Dliteratum%253Aachs%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DMart%25C3%25ADnez-Romero%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DTawa%26aufirst%3DG.%26aulast%3DFisher%26aufirst%3DE.%2BG.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DGarc%25C3%25ADa-Sastre%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DComputer-Aided%2520Discovery%2520and%2520Characterization%2520of%2520Novel%2520Ebola%2520Virus%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3582%26epage%3D3594%26doi%3D10.1021%2Facs.jmedchem.8b00035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identifying novel selective non-nucleoside DNA methyltransferase 1 inhibitors through docking-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9028</span>– <span class="NLM_lpage">9041</span>, <span class="refDoi"> DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+Chenauthor=Y.+Wangauthor=W.+Zhouauthor=S.+Liauthor=J.+Pengauthor=Z.+Shiauthor=J.+Huauthor=Y.+C.+Liuauthor=H.+Dingauthor=Y.+Linauthor=L.+Liauthor=S.+Chengauthor=J.+Liuauthor=T.+Luauthor=H.+Jiangauthor=B.+Liuauthor=M.+Zhengauthor=C.+Luo&title=Identifying+novel+selective+non-nucleoside+DNA+methyltransferase+1+inhibitors+through+docking-based+virtual+screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0lh4FoyjruTppg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520novel%2520selective%2520non-nucleoside%2520DNA%2520methyltransferase%25201%2520inhibitors%2520through%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peters, J.-U.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology - foe or friend?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8955</span>– <span class="NLM_lpage">8971</span>, <span class="refDoi"> DOI: 10.1021/jm400856t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400856t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8955-8971&author=J.-U.+Peters&title=Polypharmacology+-+foe+or+friend%3F&doi=10.1021%2Fjm400856t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology - Foe or Friend?</span></div><div class="casAuthors">Peters, Jens-Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8955-8971</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Polypharmacol. describes the activity of compds. at multiple targets.  Current research focuses on two aspects of polypharmacol.: (1) unintended polypharmacol. can lead to adverse effects; (2) polypharmacol. across several disease-relevant targets can improve therapeutic efficacy, prevent drug resistance, or reduce therapeutic-target-related adverse effects.  This perspective reviews these interconnected aspects of polypharmacol.  The first part discusses the relevance of polypharmacol. for the safety of drugs, the mitigation of safety risks, and methods to identify polypharmacol. compds. early in the drug discovery process.  The second part discusses the advantages of polypharmacol. in the treatment of multigenic diseases and infections, and opportunities for drug discovery and drug repurposing.  This perspective aims to provide a balanced view on polypharmacol., which can compromise the safety of drugs, but can also confer superior efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5vnFGPfTnGLVg90H21EOLACvtfcHk0lh4FoyjruTppg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO&md5=f4aeb6efddd4bfdf4e94656303323cba</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm400856t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400856t%26sid%3Dliteratum%253Aachs%26aulast%3DPeters%26aufirst%3DJ.-U.%26atitle%3DPolypharmacology%2520-%2520foe%2520or%2520friend%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8955%26epage%3D8971%26doi%3D10.1021%2Fjm400856t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, T.</span></span> <span> </span><span class="NLM_article-title">NMR spectroscopy techniques for screening and identifying ligand binding to protein receptors</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">864</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1002/anie.200390233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fanie.200390233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitFWltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=864-890&author=B.+Meyerauthor=T.+Peters&title=NMR+spectroscopy+techniques+for+screening+and+identifying+ligand+binding+to+protein+receptors&doi=10.1002%2Fanie.200390233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">NMR spectroscopy techniques for screening and identifying ligand binding to protein receptors</span></div><div class="casAuthors">Meyer, Bernd; Peters, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">864-890</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Picking the locks of protein binding: NMR spectroscopic techniques for the characterization of binding processes have recently gained much attention.  Binding epitopes on the ligands and on the protein can be identified easily without the aid of crystal structures (see scheme).  Hereby, rational design of new drugs has become more accessible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNviqXqZtItbVg90H21EOLACvtfcHk0ljPxH41stiDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitFWltL4%253D&md5=e0b38bf974639d9414fb57e8d0def3e1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.200390233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200390233%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DB.%26aulast%3DPeters%26aufirst%3DT.%26atitle%3DNMR%2520spectroscopy%2520techniques%2520for%2520screening%2520and%2520identifying%2520ligand%2520binding%2520to%2520protein%2520receptors%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26spage%3D864%26epage%3D890%26doi%3D10.1002%2Fanie.200390233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sattin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Functionalized 2-(4-Hydroxyphenyl)-3-methylbenzofuran Allosteric Modulators of Hsp90 Activity</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3349</span>– <span class="NLM_lpage">3364</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201600420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fejoc.201600420" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2016&pages=3349-3364&author=S.+Sattinauthor=M.+Panzaauthor=F.+Vasileauthor=F.+Berniauthor=G.+Gotiauthor=J.+Taoauthor=E.+Moroniauthor=D.+Agardauthor=G.+Colomboauthor=A.+Bernardi&title=Synthesis+of+Functionalized+2-%284-Hydroxyphenyl%29-3-methylbenzofuran+Allosteric+Modulators+of+Hsp90+Activity&doi=10.1002%2Fejoc.201600420"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201600420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201600420%26sid%3Dliteratum%253Aachs%26aulast%3DSattin%26aufirst%3DS.%26aulast%3DPanza%26aufirst%3DM.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DBerni%26aufirst%3DF.%26aulast%3DGoti%26aufirst%3DG.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DMoroni%26aufirst%3DE.%26aulast%3DAgard%26aufirst%3DD.%26aulast%3DColombo%26aufirst%3DG.%26aulast%3DBernardi%26aufirst%3DA.%26atitle%3DSynthesis%2520of%2520Functionalized%25202-%25284-Hydroxyphenyl%2529-3-methylbenzofuran%2520Allosteric%2520Modulators%2520of%2520Hsp90%2520Activity%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2016%26volume%3D20%26spage%3D3349%26epage%3D3364%26doi%3D10.1002%2Fejoc.201600420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guzzetti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arosio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tringali, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piarulli, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gennari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pignataro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvisi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span> <span> </span><span class="NLM_article-title">Insights into the Binding of Cyclic RGD Peptidomimetics to α 5 β 1 Integrin by using Live-Cell NMR And Computational Studies</span>. <i>ChemistryOpen</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1002/open.201600112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1002%2Fopen.201600112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=28168158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVGhtL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=128-136&author=I.+Guzzettiauthor=M.+Civeraauthor=F.+Vasileauthor=D.+Arosioauthor=C.+Tringaliauthor=U.+Piarulliauthor=C.+Gennariauthor=L.+Pignataroauthor=L.+Belvisiauthor=D.+Potenza&title=Insights+into+the+Binding+of+Cyclic+RGD+Peptidomimetics+to+%CE%B1+5+%CE%B2+1+Integrin+by+using+Live-Cell+NMR+And+Computational+Studies&doi=10.1002%2Fopen.201600112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Binding of Cyclic RGD Peptidomimetics to α5β1 Integrin by using Live-Cell NMR And Computational Studies</span></div><div class="casAuthors">Guzzetti, Ileana; Civera, Monica; Vasile, Francesca; Arosio, Daniela; Tringali, Cristina; Piarulli, Umberto; Gennari, Cesare; Pignataro, Luca; Belvisi, Laura; Potenza, Donatella</div><div class="citationInfo"><span class="NLM_cas:title">ChemistryOpen</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-136</span>CODEN:
                <span class="NLM_cas:coden">CHOPCK</span>;
        ISSN:<span class="NLM_cas:issn">2191-1363</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The interaction of a small library of cyclic DKP-RGD peptidomimetics with α5β1 integrin has been investigated by means of an integrated exptl. and computational approach.  Bioaffinity NMR techniques, including satn. transfer difference (STD) and transferred NOESY, were applied to the ligands in a suspension of intact MDA-MB-231 breast cancer cells, in which integrin α5β1 is highly expressed.  The NMR data were compared with the docking calcns. of the RGD ligands in the crystal structure of the α5β1 binding site, and were integrated with competitive binding assays to the purified α5β1 integrin.  Ligand binding epitopes involve protons of both the RGD moiety and the DKP scaffold, although the stereochem. and the functionalization of the DKP scaffold as well as the macrocycle conformation det. a great variability in the interaction.  The ligand showing the highest no. of STD signals is also the most potent α5β1 ligand of the series, displaying a nanomolar IC50 value.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc6FZSkfXJnLVg90H21EOLACvtfcHk0ljPxH41stiDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVGhtL%252FE&md5=4ca0c3658b32164a78a6cbe33faa6859</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fopen.201600112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fopen.201600112%26sid%3Dliteratum%253Aachs%26aulast%3DGuzzetti%26aufirst%3DI.%26aulast%3DCivera%26aufirst%3DM.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DArosio%26aufirst%3DD.%26aulast%3DTringali%26aufirst%3DC.%26aulast%3DPiarulli%26aufirst%3DU.%26aulast%3DGennari%26aufirst%3DC.%26aulast%3DPignataro%26aufirst%3DL.%26aulast%3DBelvisi%26aufirst%3DL.%26aulast%3DPotenza%26aufirst%3DD.%26atitle%3DInsights%2520into%2520the%2520Binding%2520of%2520Cyclic%2520RGD%2520Peptidomimetics%2520to%2520%25CE%25B1%25205%2520%25CE%25B2%25201%2520Integrin%2520by%2520using%2520Live-Cell%2520NMR%2520And%2520Computational%2520Studies%26jtitle%3DChemistryOpen%26date%3D2017%26volume%3D6%26spage%3D128%26epage%3D136%26doi%3D10.1002%2Fopen.201600112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Civera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vettraino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosdocimi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvisi, L.</span></span> <span> </span><span class="NLM_article-title">Exploring E-cadherin-peptidomimetics interaction using NMR and computational studies</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e1007041</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1007041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1371%2Fjournal.pcbi.1007041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=31158220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFSgs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=M.+Civeraauthor=F.+Vasileauthor=D.+Potenzaauthor=C.+Colomboauthor=S.+Parenteauthor=C.+Vettrainoauthor=T.+Prosdocimiauthor=E.+Parisiniauthor=L.+Belvisi&title=Exploring+E-cadherin-peptidomimetics+interaction+using+NMR+and+computational+studies&doi=10.1371%2Fjournal.pcbi.1007041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring E-cadherin-peptidomimetics interaction using NMR and computational studies</span></div><div class="casAuthors">Civera, Monica; Vasile, Francesca; Potenza, Donatella; Colombo, Cinzia; Parente, Sara; Vettraino, Chiara; Prosdocimi, Tommaso; Parisini, Emilio; Elvisi, Laura</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1007041</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Cadherins are homophilic cell-cell adhesion mols. whose aberrant expression has often been shown to correlate with different stages of tumor progression.  In this work, we investigate the interaction of two peptidomimetic ligands with the extracellular portion of human E-cadherin using a combination of NMR and computational techniques.  Both ligands have been previously developed as mimics of the tetrapeptide sequence Asp1-Trp2-Val3- Ile4 of the cadherin adhesion arm, and have been shown to inhibit E-cadherin-mediated adhesion in epithelial ovarian cancer cells with millimolar potency.  To sample a set of possible interactions of these ligands with the E-cadherin extracellular portion, STD-NMR expts. in the presence of two slightly different constructs, the wild type E-cadherin-EC1-EC2 fragment and the truncated E-cadherin-(Val3)-EC1-EC2 fragment, were carried out at three temps.  Depending on the protein construct, a different binding epitope of the ligand and also a different temp. effect on STD signals were obsd., both suggesting an involvement of the Asp1-Trp2 protein sequence among all the possible binding events.  To interpret the exptl. results at the at. level and to probe the role of the cadherin adhesion arm in the dynamic interaction with the peptidomimetic ligand, a computational protocol based on docking calcns. and mol. dynamics simulations was applied.  In agreement with NMR data, the simulations at different temps. unveil high variability/ dynamism in ligand-cadherin binding, thus explaining the differences in ligand binding epi- topes.  In particular, the modulation of the signals seems to be dependent on the protein flex- ibility, esp. at the level of the adhesive arm, which appears to participate in the interaction with the ligand.  Overall, these results will help the design of novel cadherin inhibi- tors that might prevent the swap dimer formation by targeting both the Trp2 binding pocket and the adhesive arm residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsFnpAtN6EHrVg90H21EOLACvtfcHk0ljPxH41stiDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSgs73F&md5=b6d9019e697e508f49b044190048d42f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007041%26sid%3Dliteratum%253Aachs%26aulast%3DCivera%26aufirst%3DM.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DColombo%26aufirst%3DC.%26aulast%3DParente%26aufirst%3DS.%26aulast%3DVettraino%26aufirst%3DC.%26aulast%3DProsdocimi%26aufirst%3DT.%26aulast%3DParisini%26aufirst%3DE.%26aulast%3DBelvisi%26aufirst%3DL.%26atitle%3DExploring%2520E-cadherin-peptidomimetics%2520interaction%2520using%2520NMR%2520and%2520computational%2520studies%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2019%26volume%3D15%26doi%3D10.1371%2Fjournal.pcbi.1007041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Della
Volpe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queirolo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unver, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jumde, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giacomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Novel Compounds Targeting the RNA-Binding Protein HuR. Structure-Based Design, Synthesis, and Interaction Studies</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00600</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00600" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVKmu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=615-620&author=S.+Della%0AVolpeauthor=R.+Nastiauthor=M.+Queiroloauthor=M.+Y.+Unverauthor=V.+K.+Jumdeauthor=A.+D%C3%B6mlingauthor=F.+Vasileauthor=D.+Potenzaauthor=F.+A.+Ambrosioauthor=G.+Costaauthor=S.+Alcaroauthor=C.+Zucalauthor=A.+Provenzaniauthor=M.+Di+Giacomoauthor=D.+Rossiauthor=A.+K.+H.+Hirschauthor=S.+Collina&title=Novel+Compounds+Targeting+the+RNA-Binding+Protein+HuR.+Structure-Based+Design%2C+Synthesis%2C+and+Interaction+Studies&doi=10.1021%2Facsmedchemlett.8b00600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Compounds Targeting the RNA-Binding Protein HuR. Structure-Based Design, Synthesis, and Interaction Studies</span></div><div class="casAuthors">Della Volpe, Serena; Nasti, Rita; Queirolo, Michele; Unver, M. Yagiz; Jumde, Varsha K.; Domling, Alexander; Vasile, Francesca; Potenza, Donatella; Ambrosio, Francesca Alessandra; Costa, Giosue; Alcaro, Stefano; Zucal, Chiara; Provenzani, Alessandro; Di Giacomo, Marcello; Rossi, Daniela; Hirsch, Anna K. H.; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">615-620</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The key role of RNA-binding proteins (RBPs) in regulating post-transcriptional processes and their involvement in several pathologies (i.e., cancer and neurodegeneration) have highlighted their potential as therapeutic targets.  In this scenario, Embryonic Lethal Abnormal Vision (ELAV) or Hu proteins and their complexes with target mRNAs have been gaining growing attention.  Compds. able to modulate the complex stability could constitute an innovative pharmacol. strategy for the treatment of numerous diseases.  Nevertheless, medicinal-chem. efforts aimed at developing such compds. are still at an early stage.  As part of our ongoing research in this field, we hereby present the rational design and synthesis of structurally novel HuR ligands, potentially acting as HuR-RNA interferers.  The following assessment of the structural features of their interaction with HuR, combining satn.-transfer difference NMR and in silico studies, provides a guide for further research on the development of new effective interfering compds. of the HuR-RNA complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokPXf_kMvpfLVg90H21EOLACvtfcHk0lh-eBiUPA6yzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVKmu78%253D&md5=abdb2ca797160f7e2c122ea791741f7c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00600%26sid%3Dliteratum%253Aachs%26aulast%3DDella%2BVolpe%26aufirst%3DS.%26aulast%3DNasti%26aufirst%3DR.%26aulast%3DQueirolo%26aufirst%3DM.%26aulast%3DUnver%26aufirst%3DM.%2BY.%26aulast%3DJumde%26aufirst%3DV.%2BK.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DZucal%26aufirst%3DC.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DDi%2BGiacomo%26aufirst%3DM.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DNovel%2520Compounds%2520Targeting%2520the%2520RNA-Binding%2520Protein%2520HuR.%2520Structure-Based%2520Design%252C%2520Synthesis%252C%2520and%2520Interaction%2520Studies%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D615%26epage%3D620%26doi%3D10.1021%2Facsmedchemlett.8b00600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Della Volpe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unver, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">Exploration of ligand binding modes towards the identification of compounds targeting HuR: a combined STD-NMR and Molecular Modelling approach</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">13780</span> <span class="refDoi"> DOI: 10.1038/s41598-018-32084-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1038%2Fs41598-018-32084-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=30214075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BB3c3os12itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=F.+Vasileauthor=S.+Della+Volpeauthor=F.+A.+Ambrosioauthor=G.+Costaauthor=M.+Y.+Unverauthor=C.+Zucalauthor=D.+Rossiauthor=E.+Martinoauthor=A.+Provenzaniauthor=A.+K.+H.+Hirschauthor=S.+Alcaroauthor=D.+Potenzaauthor=S.+Collina&title=Exploration+of+ligand+binding+modes+towards+the+identification+of+compounds+targeting+HuR%3A+a+combined+STD-NMR+and+Molecular+Modelling+approach&doi=10.1038%2Fs41598-018-32084-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of ligand binding modes towards the identification of compounds targeting HuR: a combined STD-NMR and Molecular Modelling approach</span></div><div class="casAuthors">Vasile Francesca; Della Volpe Serena; Potenza Donatella; Della Volpe Serena; Rossi Daniela; Collina Simona; Ambrosio Francesca Alessandra; Costa Giosue; Alcaro Stefano; Unver M Yagiz; Hirsch Anna K H; Zucal Chiara; Provenzani Alessandro; Martino Emanuela; Hirsch Anna K H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13780</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Post-transcriptional processes have been recognised as pivotal in the control of gene expression, and impairments in RNA processing are reported in several pathologies (i.e., cancer and neurodegeneration).  Focusing on RNA-binding proteins (RBPs), the involvement of Embryonic Lethal Abnormal Vision (ELAV) or Hu proteins and their complexes with target mRNAs in the aetiology of various dysfunctions, has suggested the great potential of compounds able to interfere with the complex stability as an innovative pharmacological strategy for the treatment of numerous diseases.  Here, we present a rational follow-up investigation of the interaction between ELAV isoform HuR and structurally-related compounds (i.e., flavonoids and coumarins), naturally decorated with different functional groups, by means of STD-NMR and Molecular Modelling.  Our results represent the foundation for the development of potent and selective ligands able to interfere with ELAV-RNA complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRWjR92U2RfrXy-c-E1v-fmfW6udTcc2eZRCtMg1OMnTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3os12itA%253D%253D&md5=9503a9f87bb6bbd695974a32063edd79</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-32084-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-32084-z%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DDella%2BVolpe%26aufirst%3DS.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DUnver%26aufirst%3DM.%2BY.%26aulast%3DZucal%26aufirst%3DC.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DMartino%26aufirst%3DE.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DExploration%2520of%2520ligand%2520binding%2520modes%2520towards%2520the%2520identification%2520of%2520compounds%2520targeting%2520HuR%253A%2520a%2520combined%2520STD-NMR%2520and%2520Molecular%2520Modelling%2520approach%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26doi%3D10.1038%2Fs41598-018-32084-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volpe, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Listro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parafioriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giacomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collina, S.</span></span> <span> </span><span class="NLM_article-title">BOPC1 Enantiomers Preparation and HuR Interaction Study. From Molecular Modeling to a Curious DEEP-STD NMR Application</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">888</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00659</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00659" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=883-888&author=S.+D.+Volpeauthor=R.+Listroauthor=M.+Parafioritiauthor=M.+Di+Giacomoauthor=D.+Rossiauthor=F.+A.+Ambrosioauthor=G.+Costaauthor=S.+Alcaroauthor=F.+Ortusoauthor=A.+K.+H.+Hirschauthor=F.+Vasileauthor=S.+Collina&title=BOPC1+Enantiomers+Preparation+and+HuR+Interaction+Study.+From+Molecular+Modeling+to+a+Curious+DEEP-STD+NMR+Application&doi=10.1021%2Facsmedchemlett.9b00659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">BOPC1 Enantiomers Preparation and HuR Interaction Study. From Molecular Modeling to a Curious DEEP-STD NMR Application</span></div><div class="casAuthors">Volpe, Serena Della; Listro, Roberta; Parafioriti, Michela; Di Giacomo, Marcello; Rossi, Daniela; Ambrosio, Francesca Alessandra; Costa, Giosue; Alcaro, Stefano; Ortuso, Francesco; Hirsch, Anna K. H.; Vasile, Francesca; Collina, Simona</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">883-888</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Hu family of RNA-binding proteins plays a crucial role in post-transcriptional processes; indeed, Hu-RNA complexes are involved in various dysfunctions (i.e., inflammation, neurodegeneration, and cancer) and have been recently proposed as promising therapeutic targets.  Intrigued by this concept, our research efforts aim at identifying small mols. able to modulate HuR-RNA interactions, with a focus on subtype HuR, upregulated and dysregulated in several cancers.  By applying structure-based design, we had already identified racemic trans-BOPC1 as promising HuR binder.  In this Letter, we accomplished the enantio-resoln., the assignment of the abs. configuration, and the recognition study with HuR of enantiomerically pure trans-BOPC1.  For the first time, we apply DEEP (differential epitope mapping)-STD NMR to study the interaction of BOPC1 with HuR and compare its enantiomers, gaining information on ligand orientation and amino acids involved in the interaction, and thus increasing focus on the in silico binding site model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz5a0llcCGGLVg90H21EOLACvtfcHk0liQwPPvZA2jfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSjsb8%253D&md5=26c2fca64cba054da303bb5d5958e8d3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00659%26sid%3Dliteratum%253Aachs%26aulast%3DVolpe%26aufirst%3DS.%2BD.%26aulast%3DListro%26aufirst%3DR.%26aulast%3DParafioriti%26aufirst%3DM.%26aulast%3DDi%2BGiacomo%26aufirst%3DM.%26aulast%3DRossi%26aufirst%3DD.%26aulast%3DAmbrosio%26aufirst%3DF.%2BA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DOrtuso%26aufirst%3DF.%26aulast%3DHirsch%26aufirst%3DA.%2BK.%2BH.%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DCollina%26aufirst%3DS.%26atitle%3DBOPC1%2520Enantiomers%2520Preparation%2520and%2520HuR%2520Interaction%2520Study.%2520From%2520Molecular%2520Modeling%2520to%2520a%2520Curious%2520DEEP-STD%2520NMR%2520Application%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D883%26epage%3D888%26doi%3D10.1021%2Facsmedchemlett.9b00659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manzoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maio, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Agostino, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miceli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baj, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerofolini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elezgarai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biasini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchinat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fragai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seneci, P.</span></span> <span> </span><span class="NLM_article-title">Interfering with HuR-RNA Interaction: Design, Synthesis and Biological Characterization of Tanshinone Mimics as Novel, Effective HuR Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1483</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVensA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1483-1498&author=L.+Manzoniauthor=C.+Zucalauthor=D.+D.+Maioauthor=V.+G.+D%E2%80%99Agostinoauthor=N.+Thongonauthor=I.+Bonomoauthor=P.+Lalauthor=M.+Miceliauthor=V.+Bajauthor=M.+Brambillaauthor=L.+Cerofoliniauthor=S.+Elezgaraiauthor=E.+Biasiniauthor=C.+Luchinatauthor=E.+Novellinoauthor=M.+Fragaiauthor=L.+Marinelliauthor=A.+Provenzaniauthor=P.+Seneci&title=Interfering+with+HuR-RNA+Interaction%3A+Design%2C+Synthesis+and+Biological+Characterization+of+Tanshinone+Mimics+as+Novel%2C+Effective+HuR+Inhibitors&doi=10.1021%2Facs.jmedchem.7b01176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Interfering with HuR-RNA Interaction: Design, Synthesis and Biological Characterization of Tanshinone Mimics as Novel, Effective HuR Inhibitors</span></div><div class="casAuthors">Manzoni, Leonardo; Zucal, Chiara; Maio, Danilo Di; D'Agostino, Vito G.; Thongon, Natthakan; Bonomo, Isabelle; Lal, Preet; Miceli, Marco; Baj, Vanessa; Brambilla, Marta; Cerofolini, Linda; Elezgarai, Saioa; Biasini, Emiliano; Luchinat, Claudio; Novellino, Ettore; Fragai, Marco; Marinelli, Luciana; Provenzani, Alessandro; Seneci, Pierfausto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1483-1498</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The human antigen R (HuR) is an RNA-binding protein known to modulate the expression of target mRNA coding for proteins involved in inflammation, tumorigenesis, and stress responses and is a valuable drug target.  We previously found that dihydrotanshinone-I (DHTS, 1) prevents the assocn. of HuR with its RNA substrate, thus imparing its function.  Herein, inspired by DHTS structure, we designed and synthesized an array of ortho-quinones (tanshinone mimics) using a function-oriented synthetic approach.  Among others, compd. 6a and 6n turned out to be more effective than 1, showing a nanomolar Ki and disrupting HuR binding to RNA in cells.  A combined approach of NMR titrn. and mol. dynamics (MD) simulations suggests that 6a stabilizes HuR in a peculiar closed conformation, which is incompatible with RNA binding.  Alpha screen and RNA-electrophoretic mobility shift assays (REMSA) data on newly synthesized compds. allowed, for the first time, the generation of structure activity relationships (SARs), thus providing a solid background for the generation of highly effective HuR disruptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS0iVE3OcEC7Vg90H21EOLACvtfcHk0liQwPPvZA2jfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVensA%253D%253D&md5=e9ad9fbce3f181afc02a716c566be2fc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01176%26sid%3Dliteratum%253Aachs%26aulast%3DManzoni%26aufirst%3DL.%26aulast%3DZucal%26aufirst%3DC.%26aulast%3DMaio%26aufirst%3DD.%2BD.%26aulast%3DD%25E2%2580%2599Agostino%26aufirst%3DV.%2BG.%26aulast%3DThongon%26aufirst%3DN.%26aulast%3DBonomo%26aufirst%3DI.%26aulast%3DLal%26aufirst%3DP.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DBaj%26aufirst%3DV.%26aulast%3DBrambilla%26aufirst%3DM.%26aulast%3DCerofolini%26aufirst%3DL.%26aulast%3DElezgarai%26aufirst%3DS.%26aulast%3DBiasini%26aufirst%3DE.%26aulast%3DLuchinat%26aufirst%3DC.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DFragai%26aufirst%3DM.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DSeneci%26aufirst%3DP.%26atitle%3DInterfering%2520with%2520HuR-RNA%2520Interaction%253A%2520Design%252C%2520Synthesis%2520and%2520Biological%2520Characterization%2520of%2520Tanshinone%2520Mimics%2520as%2520Novel%252C%2520Effective%2520HuR%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1483%26epage%3D1498%26doi%3D10.1021%2Facs.jmedchem.7b01176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0liQwPPvZA2jfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvisi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiana, G.</span></span> <span> </span><span class="NLM_article-title">Thermodynamically-Weighted Conformational Ensemble of Cyclic RGD Peptidomimetics from NOE Data</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">7098</span>– <span class="NLM_lpage">7107</span>, <span class="refDoi"> DOI: 10.1021/acs.jpcb.6b04941</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.6b04941" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFersrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2016&pages=7098-7107&author=F.+Vasileauthor=M.+Civeraauthor=L.+Belvisiauthor=D.+Potenzaauthor=G.+Tiana&title=Thermodynamically-Weighted+Conformational+Ensemble+of+Cyclic+RGD+Peptidomimetics+from+NOE+Data&doi=10.1021%2Facs.jpcb.6b04941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamically-Weighted Conformational Ensemble of Cyclic RGD Peptidomimetics from NOE Data</span></div><div class="casAuthors">Vasile, F.; Civera, M.; Belvisi, L.; Potenza, D.; Tiana, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">7098-7107</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the case of flexible mols., the std. approach of transforming NOE intensities into spatial restraints and of building conformational models minimizing these restraints greatly neglects the richness of mol. conformations.  Making use of NOE intensities measured in triplicate and of an iterative mol.-dynamics scheme, we optimized a force field to generate a set of conformations whose ensemble is compatible with the exptl. data, and is weighted according to the Boltzmann distribution.  This scheme is applied to two cyclic peptidomimetic ligands of integrins.  Their difference in binding affinity is recapitulated in terms of their difference in conformational fluctuations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7hONyJihVz7Vg90H21EOLACvtfcHk0lgMBbqm6tlgfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFersrbO&md5=c49c258ca2ec935388b06a777bab1456</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.6b04941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.6b04941%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DCivera%26aufirst%3DM.%26aulast%3DBelvisi%26aufirst%3DL.%26aulast%3DPotenza%26aufirst%3DD.%26aulast%3DTiana%26aufirst%3DG.%26atitle%3DThermodynamically-Weighted%2520Conformational%2520Ensemble%2520of%2520Cyclic%2520RGD%2520Peptidomimetics%2520from%2520NOE%2520Data%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2016%26volume%3D120%26spage%3D7098%26epage%3D7107%26doi%3D10.1021%2Facs.jpcb.6b04941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubinelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panigada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soprana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siccardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, D.</span></span> <span> </span><span class="NLM_article-title">NMR interaction studies of Neu5Ac-α-(2,6)-Gal-β-(1-4)-GlcNAc with influenza-virus hemagglutinin expressed in transfected human cells</span>. <i>Glycobiology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1093/glycob/cwx092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fglycob%2Fcwx092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=29087468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWht73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=42-49&author=F.+Vasileauthor=F.+Gubinelliauthor=M.+Panigadaauthor=E.+Sopranaauthor=A.+Siccardiauthor=D.+Potenza&title=NMR+interaction+studies+of+Neu5Ac-%CE%B1-%282%2C6%29-Gal-%CE%B2-%281-4%29-GlcNAc+with+influenza-virus+hemagglutinin+expressed+in+transfected+human+cells&doi=10.1093%2Fglycob%2Fcwx092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">NMR interaction studies of Neu5Ac-α-(2,6)-Gal-β-(1-4)-GlcNAc with influenza-virus hemagglutinin expressed in transfected human cells</span></div><div class="casAuthors">Vasile, Francesca; Gubinelli, Francesco; Panigada, Maddalena; Soprana, Elisa; Siccardi, Antonio; Potenza, Donatella</div><div class="citationInfo"><span class="NLM_cas:title">Glycobiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-49</span>CODEN:
                <span class="NLM_cas:coden">GLYCE3</span>;
        ISSN:<span class="NLM_cas:issn">1460-2423</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The emergence of escape-mutants of influenza hemagglutinin (HA) following vaccination compels the yearly re-formulation of flu vaccines.  Since binding the sialic acid receptor remains in all cases essential for infection, small-mol. inhibitors of HA binding to sialic acid could be interesting therapeutic complements or alternatives to immuno-prophylaxis in the control of flu epidemics.  In this work, we made use of NMR spectroscopy to study the interaction between a deriv. of sialic acid (the Neu5Ac-α-(2,6)-Gal-β-(1-4)-GlcNAc trisaccharide) and HAs (H1 and H5) from human and avian strains of influenza virus, directly expressed on the surface of stable transfected 293 T human cells.  The HAs were shown to retain their native trimeric conformation and binding proper- ties.  Exploiting the magnetization transfer between the proteins and the ligand, we obtained evidence of the binding event and mapped the (non-identical) sugar epitopes recognized by the two HA species.  The rapid and reliable method for screening sialic acid-related HA ligands we have developed could yield useful information for an efficient drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9m74VJJzKPLVg90H21EOLACvtfcHk0lgMBbqm6tlgfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWht73N&md5=cdb70a5358c8354e72d39d501f6b83fa</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fglycob%2Fcwx092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fglycob%252Fcwx092%26sid%3Dliteratum%253Aachs%26aulast%3DVasile%26aufirst%3DF.%26aulast%3DGubinelli%26aufirst%3DF.%26aulast%3DPanigada%26aufirst%3DM.%26aulast%3DSoprana%26aufirst%3DE.%26aulast%3DSiccardi%26aufirst%3DA.%26aulast%3DPotenza%26aufirst%3DD.%26atitle%3DNMR%2520interaction%2520studies%2520of%2520Neu5Ac-%25CE%25B1-%25282%252C6%2529-Gal-%25CE%25B2-%25281-4%2529-GlcNAc%2520with%2520influenza-virus%2520hemagglutinin%2520expressed%2520in%2520transfected%2520human%2520cells%26jtitle%3DGlycobiology%26date%3D2018%26volume%3D28%26spage%3D42%26epage%3D49%26doi%3D10.1093%2Fglycob%2Fcwx092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engelborghs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanoiu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandewoude, M.</span></span> <span> </span><span class="NLM_article-title">Rationale and clinical data supporting nutritional intervention in Alzheimer’s disease</span>. <i>Acta Clin. Belg.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1179/0001551213Z.0000000006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1179%2F0001551213Z.0000000006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=24635394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A280%3ADC%252BC2crksFCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=17-24&author=S.+Engelborghsauthor=C.+Gillesauthor=A.+Ivanoiuauthor=M.+Vandewoude&title=Rationale+and+clinical+data+supporting+nutritional+intervention+in+Alzheimer%E2%80%99s+disease&doi=10.1179%2F0001551213Z.0000000006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Rationale and clinical data supporting nutritional intervention in Alzheimer's disease</span></div><div class="casAuthors">Engelborghs S; Gilles C; Ivanoiu A; Vandewoude M</div><div class="citationInfo"><span class="NLM_cas:title">Acta clinica Belgica</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-24</span>
        ISSN:<span class="NLM_cas:issn">1784-3286</span>.
    </div><div class="casAbstract">Adequate nutrition plays an important role in the maintenance of cognitive function, particularly during aging.  Malnutrition is amongst the risk factors for developing mild cognitive impairment (MCI) and Alzheimer's disease (AD).  Epidemiological studies have associated deficiencies in some nutrients with a higher risk of cognitive dysfunction and/or AD.  Cognitive decline in AD is correlated with synaptic loss and many of the components required to maintain optimal synaptic function are derived from dietary sources.  As synapses are part of the neuronal membrane and are continuously being remodelled, the availability of sufficient levels of nutritional precursors (mainly uridine monophosphate, choline and omega-3 fatty acids) to make the phospholipids required to build neuronal membranes may have beneficial effects on synaptic degeneration in AD.  In addition, B-vitamins, phospholipids and other micronutrients act as cofactors to enhance the supply of precursors required to make neuronal membranes and synapses.  Despite this, no randomized controlled trial has hitherto provided evidence that any single nutrient has a beneficial effect on cognition or lowers the risk for AD.  However, a multi-target approach using combinations of (micro)nutrients might have beneficial effects on cognitive function in neurodegenerative brain disorders like AD leading to synaptic degeneration.  Here we review the clinical evidence for supplementation, based on a multi-target approach with a focus on key nutrients with a proposed role in synaptic dysfunction.  Based on preclinical evidence, a nutrient mixture, Souvenaid(®) (Nutricia N.V., Zoetermeer, The Netherlands) was developed.  Clinical trials with Souvenaid(®) have shown improved memory performance in patients with mild AD.  Further clinical trials to evaluate the effects of nutritional intervention in MCI and early dementia due to AD are on-going.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfYnw-ONC_vVQKX3ooHxWjfW6udTcc2eZogu-8RBslVLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crksFCgsQ%253D%253D&md5=0c6d69afdaf9ae1f3bea1679aa685c49</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1179%2F0001551213Z.0000000006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1179%252F0001551213Z.0000000006%26sid%3Dliteratum%253Aachs%26aulast%3DEngelborghs%26aufirst%3DS.%26aulast%3DGilles%26aufirst%3DC.%26aulast%3DIvanoiu%26aufirst%3DA.%26aulast%3DVandewoude%26aufirst%3DM.%26atitle%3DRationale%2520and%2520clinical%2520data%2520supporting%2520nutritional%2520intervention%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DActa%2520Clin.%2520Belg.%26date%3D2014%26volume%3D69%26spage%3D17%26epage%3D24%26doi%3D10.1179%2F0001551213Z.0000000006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhobale, M.</span></span> <span> </span><span class="NLM_article-title">Neurotrophic Factors and Maternal Nutrition During Pregnancy</span>. <i>Vitam. Horm.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1016/bs.vh.2016.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fbs.vh.2016.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=28215301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVOgsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2017&pages=343-366&author=M.+Dhobale&title=Neurotrophic+Factors+and+Maternal+Nutrition+During+Pregnancy&doi=10.1016%2Fbs.vh.2016.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Neurotrophic factors and maternal nutrition during pregnancy</span></div><div class="casAuthors">Dhobale, M.</div><div class="citationInfo"><span class="NLM_cas:title">Vitamins and Hormones (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">Neurotrophins</span>),
    <span class="NLM_cas:pages">343-366</span>CODEN:
                <span class="NLM_cas:coden">VIHOAQ</span>;
        ISSN:<span class="NLM_cas:issn">0083-6729</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Maternal nutrition is one of the major determinants of pregnancy outcome.  It has been suggested that reduced intakes or lack of specific nutrients during pregnancy influences the length of gestation, proper placental and fetal growth during pregnancy.  Maternal nutrition, particularly micronutrients such as folate and vitamin B12, and long-chain polyunsatd. fatty acids (LCPUFA) are the major determinants of the one carbon cycle and are suggested to be at the heart of intrauterine programming of diseases in adult life.  LCPUFA play a key role in the normal feto-placental development, as well as in the development and functional maturation of the brain and central nervous system and also regulate the levels of neurotrophic factors.  These neurotrophic factors are known to regulate the development of the placenta at the materno-fetal interface and act in a paracrine and endocrine manner.  Neurotrophic factors like brain-derived neurotrophic factor and nerve growth factor are proteins involved in angiogenesis and potentiate the placental development.  This chapter mainly focuses on micronutrients since they play a main physiol. role during pregnancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUoyM5ZkiyZLVg90H21EOLACvtfcHk0lgMBbqm6tlgfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVOgsL4%253D&md5=ed7beda14dd881ef1c20f970e9f386a9</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fbs.vh.2016.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.vh.2016.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DDhobale%26aufirst%3DM.%26atitle%3DNeurotrophic%2520Factors%2520and%2520Maternal%2520Nutrition%2520During%2520Pregnancy%26jtitle%3DVitam.%2520Horm.%26date%3D2017%26volume%3D104%26spage%3D343%26epage%3D366%26doi%3D10.1016%2Fbs.vh.2016.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coşar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ipçioğlu, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gültepe, M.</span></span> <span> </span><span class="NLM_article-title">Folate and homocysteine metabolisms and their roles in the biochemical basis of neuropsychiatry</span>. <i>Turk. J. Med. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.3906/sag-1211-39</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.3906%2Fsag-1211-39" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=25558551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVKjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2014&pages=1-9&author=A.+Co%C5%9Farauthor=O.+M.+Ip%C3%A7io%C4%9Fluauthor=O.+Ozcanauthor=M.+G%C3%BCltepe&title=Folate+and+homocysteine+metabolisms+and+their+roles+in+the+biochemical+basis+of+neuropsychiatry&doi=10.3906%2Fsag-1211-39"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Folate and homocysteine metabolisms and their roles in the biochemical basis of neuropsychiatry</span></div><div class="casAuthors">Cosar, Alpaslan; Ipcioglu, Osman Metin; Ozcan, Omer; Gultepe, Mustafa</div><div class="citationInfo"><span class="NLM_cas:title">Turkish Journal of Medical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">TJMEEA</span>;
        ISSN:<span class="NLM_cas:issn">1300-0144</span>.
    
            (<span class="NLM_cas:orgname">Scientific and Technological Research Council of Turkey</span>)
        </div><div class="casAbstract">A review.  The term 'one-carbon metab.' is commonly used to describe 3 sep. metabolic processes: folate metab., the homocysteine remethylation cycle, and the transsulfuration pathway.  Folate metab. concerns the biochem. reactions in which endogenous and exogenous one-carbon units are transferred to tetrahydrofolates.  The remethylation cycle is used for the synthesis of methionine from homocysteine with one-carbon units that come from folate.  This methionine is then used for the synthesis of S-adenosyl methionine, which is a general donor of Me groups for many biochem. reactions in the human body.  In the transsulfuration pathway, some amino acids and polypeptides, such as cystathionine, cysteine, and glutathione, are synthesized from homocysteine.  The kinetics of the enzymes in this pathway are regulated by the substrates of the remethylation cycle.  The methylation process has been thought to have an important role in the biochem. basis of neuropsychiatry.  An elevated homocysteine level is the most important marker of folate and vitamin B12 deficiencies, and also the most reliable biochem. sign of functional insufficiency.  Some neurol. and neuropsychiatric diseases, such as psychosis, Alzheimer's disease, and autism, have been found to be related to disorders of one-carbon metab.  This review aims to summarize both one-carbon metab. and its relationships with neuropsychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNIEIl1dodQrVg90H21EOLACvtfcHk0lijrIDCfp9LUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVKjsrs%253D&md5=6852620d34834d85163941877f7aa40a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.3906%2Fsag-1211-39&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3906%252Fsag-1211-39%26sid%3Dliteratum%253Aachs%26aulast%3DCo%25C5%259Far%26aufirst%3DA.%26aulast%3DIp%25C3%25A7io%25C4%259Flu%26aufirst%3DO.%2BM.%26aulast%3DOzcan%26aufirst%3DO.%26aulast%3DG%25C3%25BCltepe%26aufirst%3DM.%26atitle%3DFolate%2520and%2520homocysteine%2520metabolisms%2520and%2520their%2520roles%2520in%2520the%2520biochemical%2520basis%2520of%2520neuropsychiatry%26jtitle%3DTurk.%2520J.%2520Med.%2520Sci.%26date%3D2014%26volume%3D44%26spage%3D1%26epage%3D9%26doi%3D10.3906%2Fsag-1211-39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, I.</span></span> <span> </span><span class="NLM_article-title">Alzheimer’s disease pathogenesis: Is there a role for folate?</span>. <i>Mech Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2017.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1016%2Fj.mad.2017.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=29037490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslChs77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=86-94&author=N.+Robinsonauthor=P.+Grabowskiauthor=I.+Rehman&title=Alzheimer%E2%80%99s+disease+pathogenesis%3A+Is+there+a+role+for+folate%3F&doi=10.1016%2Fj.mad.2017.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease pathogenesis: Is there a role for folate?</span></div><div class="casAuthors">Robinson, Natassia; Grabowski, Peter; Rehman, Ishtiaq</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">86-94</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetic modifications, including changes in DNA methylation, have been implicated in a wide range of diseases including neurol. diseases such as Alzheimer's.  The role of dietary folate in providing Me groups required for maintenance and modulation of DNA methylation makes it a nutrient of interest in Alzheimer's.  Late onset Alzheimer's disease is the most common form of dementia and at present its etiol. is largely undetd.  From epidemiol. studies, the interactions between folate, B-vitamins and homocysteine as well as the long latency period has led to difficulties in interpretation of the data, thus current evidence exploring the role of dietary folate in Alzheimer's is contradictory and unresolved.  Therefore, examg. the effects at a mol. level and exploring potential epigenetic mechanisms could increase our understanding of the disease and etiol.  The aim of this review is to examine the role that folate could play in Alzheimer's disease neuropathol. and will focus on the effects of folate on DNA methylation which link to disease pathol., initiation and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBwPf1M7HdLVg90H21EOLACvtfcHk0lijrIDCfp9LUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslChs77L&md5=efa72dacaa5eedfd8d9bf44d8b90b72b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2017.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2017.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DN.%26aulast%3DGrabowski%26aufirst%3DP.%26aulast%3DRehman%26aufirst%3DI.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%2520pathogenesis%253A%2520Is%2520there%2520a%2520role%2520for%2520folate%253F%26jtitle%3DMech%2520Ageing%2520Dev.%26date%3D2018%26volume%3D174%26spage%3D86%26epage%3D94%26doi%3D10.1016%2Fj.mad.2017.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yachi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tohyama, M.</span></span> <span> </span><span class="NLM_article-title">CARM1 Regulates Proliferation of PC12 Cells by Methylating HuD</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2273</span>– <span class="NLM_lpage">2285</span>, <span class="refDoi"> DOI: 10.1128/MCB.26.6.2273-2285.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1128%2FMCB.26.6.2273-2285.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=16508003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2273-2285&author=T.+Fujiwaraauthor=Y.+Moriauthor=D.+L.+Chuauthor=Y.+Koyamaauthor=S.+Miyataauthor=H.+Tanakaauthor=K.+Yachiauthor=T.+Kuboauthor=H.+Yoshikawaauthor=M.+Tohyama&title=CARM1+Regulates+Proliferation+of+PC12+Cells+by+Methylating+HuD&doi=10.1128%2FMCB.26.6.2273-2285.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">CARM1 regulates proliferation of PC12 cells by methylating HuD</span></div><div class="casAuthors">Fujiwara, Tatsuji; Mori, Yasutake; Chu, Dong Ling; Koyama, Yoshihisa; Miyata, Shingo; Tanaka, Hiroyuki; Yachi, Kohji; Kubo, Tateki; Yoshikawa, Hideki; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2273-2285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">HuD is an RNA-binding protein that has been shown to induce neuronal differentiation by stabilizing labile mRNAs carrying AU-rich instability elements.  Here, we show a novel mechanism of arginine methylation of HuD by coactivator-assocd. arginine methyltransferase 1 (CARM1) that affected mRNA turnover of p21cip1/waf1 mRNA in PC12 cells.  CARM1 specifically methylated HuD in vitro and in vivo and colocalized with HuD in the cytoplasm.  Inhibition of HuD methylation by CARM1 knockdown elongated the p21cip1/waf1 mRNA half-life and resulted in a slow growth rate and robust neuritogenesis in response to nerve growth factor (NGF).  Methylation-resistant HuD bound more p21cip1/waf1 mRNA than did the wild type, and its overexpression upregulated p21cip1/waf1 protein expression.  These results suggested that CARM1-methylated HuD maintains PC12 cells in the proliferative state by committing p21cip1/waf1 mRNA to its decay system.  Since the methylated population of HuD was reduced in NGF-treated PC12 cells, downregulation of HuD methylation is a possible pathway through which NGF induces differentiation of PC12 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71TYVPwUyRLVg90H21EOLACvtfcHk0lijrIDCfp9LUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D&md5=a557599ff6460f5ab365362323b87a2b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.6.2273-2285.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.6.2273-2285.2006%26sid%3Dliteratum%253Aachs%26aulast%3DFujiwara%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DChu%26aufirst%3DD.%2BL.%26aulast%3DKoyama%26aufirst%3DY.%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DYachi%26aufirst%3DK.%26aulast%3DKubo%26aufirst%3DT.%26aulast%3DYoshikawa%26aufirst%3DH.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DCARM1%2520Regulates%2520Proliferation%2520of%2520PC12%2520Cells%2520by%2520Methylating%2520HuD%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2273%26epage%3D2285%26doi%3D10.1128%2FMCB.26.6.2273-2285.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craenen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verslegers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baatout, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benotmane, M. A.</span></span> <span> </span><span class="NLM_article-title">An appraisal of folates as key factors in cognition and ageing-related diseases</span>. <i>Crit. Rev. Food Sci. Nutr.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1080/10408398.2018.1549017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1080%2F10408398.2018.1549017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=30729795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFymsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=722-739&author=K.+Craenenauthor=M.+Verslegersauthor=S.+Baatoutauthor=M.+A.+Benotmane&title=An+appraisal+of+folates+as+key+factors+in+cognition+and+ageing-related+diseases&doi=10.1080%2F10408398.2018.1549017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An appraisal of folates as key factors in cognition and ageing-related diseases</span></div><div class="casAuthors">Craenen, Kai; Verslegers, Mieke; Baatout, Sarah; Abderrafi Benotmane, Mohammed</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Food Science and Nutrition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">722-739</span>CODEN:
                <span class="NLM_cas:coden">CRFND6</span>;
        ISSN:<span class="NLM_cas:issn">1040-8398</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Folic acid (FA) is often consumed as a food supplement and can be found in fortified staple foods in various western countries.  Even though FA supplementation during pregnancy is known to prevent severe congenital anomalies in the developing child (e.g., neural tube defects), much less is known about its influence on cognition and neurol. functioning.  In this review, we address the advances in this field and situate how folate intake during pregnancy, postnatal life, adulthood and in the elderly affects cognition.  In addn., an assocn. between folate status and ageing, dementia and other neurodegenerative disorders such as Alzheimer's disease, Parkinson's disease and amyotrophic lateral sclerosis is discussed.  While its role in the incidence and severity of these diseases is becoming apparent, the underlying action of folates and related metabolites remains elusive.  Finally, the potential of FA as a nutraceutical has been proposed, although the efficacy will highly depend on the interplay with other micronutrients, the disease stage and the duration of supplementation.  Hence, the lack of consistent data urges for more animal studies and (pre)clin. trials in humans to ascertain a potential beneficial role for folates in the treatment or amelioration of cognitive decline and ageing-related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf6W9bg0cB-7Vg90H21EOLACvtfcHk0ljqiEBXDymuPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFymsb8%253D&md5=30489f9ea4310a877ce4d7bfef58f948</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1080%2F10408398.2018.1549017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408398.2018.1549017%26sid%3Dliteratum%253Aachs%26aulast%3DCraenen%26aufirst%3DK.%26aulast%3DVerslegers%26aufirst%3DM.%26aulast%3DBaatout%26aufirst%3DS.%26aulast%3DBenotmane%26aufirst%3DM.%2BA.%26atitle%3DAn%2520appraisal%2520of%2520folates%2520as%2520key%2520factors%2520in%2520cognition%2520and%2520ageing-related%2520diseases%26jtitle%3DCrit.%2520Rev.%2520Food%2520Sci.%2520Nutr.%26date%3D2020%26volume%3D60%26spage%3D722%26epage%3D739%26doi%3D10.1080%2F10408398.2018.1549017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pascale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scapagnini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govoni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkon, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrone, A.</span></span> <span> </span><span class="NLM_article-title">Neuronal ELAV proteins enhance mRNA stability by a PKCα-dependent pathway</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">12065</span>– <span class="NLM_lpage">12070</span>, <span class="refDoi"> DOI: 10.1073/pnas.0504702102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1073%2Fpnas.0504702102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=16099831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BD2MXps1yru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=12065-12070&author=A.+Pascaleauthor=M.+Amadioauthor=G.+Scapagniniauthor=C.+Lanniauthor=M.+Racchiauthor=A.+Provenzaniauthor=S.+Govoniauthor=D.+L.+Alkonauthor=A.+Quattrone&title=Neuronal+ELAV+proteins+enhance+mRNA+stability+by+a+PKC%CE%B1-dependent+pathway&doi=10.1073%2Fpnas.0504702102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Neuronal ELAV proteins enhance mRNA stability by a PKCα-dependent pathway</span></div><div class="casAuthors">Pascale, Alessia; Amadio, Marialaura; Scapagnini, Giovanni; Lanni, Cristina; Racchi, Marco; Provenzani, Alessandro; Govoni, Stefano; Alkon, Daniel L.; Quattrone, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12065-12070</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">More than 1 in 20 human genes bear in the mRNA 3' UTR a specific motif called the adenine- and uridine-rich element (ARE), which posttranscriptionally dets. its expression in response to cell environmental signals.  ELAV (embryonic lethal abnormal vision) proteins are the only known ARE-binding factors that are able to stabilize the bound mRNAs, thereby pos. controlling gene expression.  Here, we show that in human neuroblastoma SH-SY5Y cells, neuron-specific ELAV (nELAV) proteins (HuB, HuC, and HuD) are up-regulated and redistributed by 15 min of treatment with the activators of PKC phorbol esters and bryostatin-1.  PKC stimulation also induces nELAV proteins to colocalize with the translocated PKCα isoenzyme preferentially on the cytoskeleton, with a concomitant increase in nELAV threonine phosphorylation.  The same treatment promotes stabilization of growth-assocd. protein 43 (GAP-43) mRNA, a well known nELAV target, and induces an early increase in GAP-43 protein concn., again only in the cytoskeletal cell fraction.  Genetic or pharmacol. inactivation of PKCα abolishes nELAV protein cytoskeletal up-regulation, GAP-43 mRNA stabilization, and GAP-43 protein increase, demonstrating the primary role of this specific PKC isoenzyme in the cascade of nELAV recruitment.  Finally, in vivo PKC activation is assocd. with an up-regulation of nELAV proteins in the hippocampal rat brain.  These findings suggest a model for gene expression regulation by nELAV proteins through a PKCα-dependent pathway that is relevant for the cellular programs in which ARE-mediated control plays a pivotal role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowTnFfQNrvp7Vg90H21EOLACvtfcHk0ljqiEBXDymuPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXps1yru7s%253D&md5=ac18f404ee001a080e2f64ee17d08a59</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0504702102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0504702102%26sid%3Dliteratum%253Aachs%26aulast%3DPascale%26aufirst%3DA.%26aulast%3DAmadio%26aufirst%3DM.%26aulast%3DScapagnini%26aufirst%3DG.%26aulast%3DLanni%26aufirst%3DC.%26aulast%3DRacchi%26aufirst%3DM.%26aulast%3DProvenzani%26aufirst%3DA.%26aulast%3DGovoni%26aufirst%3DS.%26aulast%3DAlkon%26aufirst%3DD.%2BL.%26aulast%3DQuattrone%26aufirst%3DA.%26atitle%3DNeuronal%2520ELAV%2520proteins%2520enhance%2520mRNA%2520stability%2520by%2520a%2520PKC%25CE%25B1-dependent%2520pathway%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D12065%26epage%3D12070%26doi%3D10.1073%2Fpnas.0504702102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span> <i>Protein Preparation Wizard, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Protein+Preparation+Wizard%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DProtein%2520Preparation%2520Wizard%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span> <i>MacroModel, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+MacroModel%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C%0ANY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMacroModel%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribiere, E. P. eG.</span></span> <span> </span><span class="NLM_article-title">Note sur la convergence de méthodes de directions conjuguées</span>. <i>Revue française d’informatique et de recherche opérationnelle. Série rouge</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1051/m2an/196903r100351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1051%2Fm2an%2F196903r100351" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1969&pages=35-43&author=E.+P.+eG.+Ribiere&title=Note+sur+la+convergence+de+m%C3%A9thodes+de+directions+conjugu%C3%A9es&doi=10.1051%2Fm2an%2F196903r100351"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1051%2Fm2an%2F196903r100351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1051%252Fm2an%252F196903r100351%26sid%3Dliteratum%253Aachs%26aulast%3DRibiere%26aufirst%3DE.%2BP.%2BeG.%26atitle%3DNote%2520sur%2520la%2520convergence%2520de%2520m%25C3%25A9thodes%2520de%2520directions%2520conjugu%25C3%25A9es%26jtitle%3DRevue%2520fran%25C3%25A7aise%2520d%25E2%2580%2599informatique%2520et%2520de%2520recherche%2520op%25C3%25A9rationnelle.%2520S%25C3%25A9rie%2520rouge%26date%3D1969%26volume%3D3%26spage%3D35%26epage%3D43%26doi%3D10.1051%2Fm2an%2F196903r100351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span>Desmond Molecular
Dynamics
System, D.E. Shaw Research, New York, NY, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Desmond+Molecular%0ADynamics%0ASystem%2C+D.E.+Shaw+Research%2C+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span>Maestro-Desmond
Interoperability
Tools, Schrödinger, New York, NY, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro-Desmond%0AInteroperability%0ATools%2C+Schr%C3%B6dinger%2C+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">11225</span>– <span class="NLM_lpage">11236</span>, <span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+Jorgensenauthor=D.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+Testing+of+the+OPLS+All-Atom+Force+Field+on+Conformational+Energetics+and+Properties+of+Organic+Liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0ljqiEBXDymuPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%26aulast%3DMaxwell%26aufirst%3DD.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520Testing%2520of%2520the%2520OPLS%2520All-Atom%2520Force%2520Field%2520on%2520Conformational%2520Energetics%2520and%2520Properties%2520of%2520Organic%2520Liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span> <i>Glide, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="note"><p class="first last">foodb.ca.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: a major update to the DrugBank database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&author=D.+S.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+a+major+update+to+the+DrugBank+database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lgZcVBNqNnXJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520a%2520major%2520update%2520to%2520the%2520DrugBank%2520database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolstad, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span> <span> </span><span class="NLM_article-title">ZINC: A Free Tool to Discover Chemistry for Biology</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1757</span>– <span class="NLM_lpage">1768</span>, <span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+Free+Tool+to+Discover+Chemistry+for+Biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lgZcVBNqNnXJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520Free%2520Tool%2520to%2520Discover%2520Chemistry%2520for%2520Biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span> <i>LigPrep, Schrödinger</i>; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+Schr%C3%B6dinger%3B+LLC%3A+New+York%2C%0ANY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DLigPrep%252C%2520Schr%25C3%25B6dinger%26pub%3DLLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">ZINC 15 – Ligand Discovery for Everyone</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2324</span>– <span class="NLM_lpage">2337</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&author=T.+Sterlingauthor=J.+J.+Irwin&title=ZINC+15+%E2%80%93+Ligand+Discovery+for+Everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0lgZcVBNqNnXJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZINC%252015%2520%25E2%2580%2593%2520Ligand%2520Discovery%2520for%2520Everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="note"><p class="first last">zinc15.docking.org/patterns/home.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_02802"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00191?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00191</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Glide score value (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_001.csv">CSV</a>)</p></li><li><p class="inline">PDB complex of cefazolin docked into HuD (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_002.pdb">PDB</a>)</p></li><li><p class="inline">PDB complex of enalapril docked into HuD (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_003.pdb">PDB</a>)</p></li><li><p class="inline">PDB complex of folic acid docked into HuD (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_004.pdb">PDB</a>)</p></li><li><p class="inline">PDB complex of rosmarinic acid docked into HuD (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Structural animation of folic acid in complex with HuD (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_006.avi">AVI</a>)</p></li><li><p class="inline">NMR analysis. Figures S1-S3-S5-S7: Chemical structures of the best hits. Tables S1-S2-S3-S4: <sup>1</sup>H and <sup>13</sup>C assignment of the best hits. Figures S2-S4-S6-S8: <sup>1</sup>H-NMR and STD-NMR of the best hits in the presence of HuD. Molecular modeling analysis. Table S5: Molecular formula strings of selected compounds. Figure S9: 3D representations of the best poses of methoxatin, aurintricarboxylic acid, portulacaxanthin II, alloxazine, tilundronic acid, and phosphoglycolic acid within the HuD binding pocket. Figure S10: HuD protein backbone RMSD trend calculated in the presence of the best hits. Figure S11: Ligands RMSD trends of the best hits. Figure S12: Timeline representation of the interactions and contacts between the best hits and the protein. Figure S13: HuD protein backbone RMSF calculated in the presence of folic acid and cefazolin. Figure S14: Effect of folic acid on HuD protein (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_001.csv">jm1c00191_si_001.csv (0.88 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_002.pdb">jm1c00191_si_002.pdb (214.73 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_003.pdb">jm1c00191_si_003.pdb (215.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_004.pdb">jm1c00191_si_004.pdb (215.52 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_005.pdb">jm1c00191_si_005.pdb (214.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_006.avi">jm1c00191_si_006.avi (50.92 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00191/suppl_file/jm1c00191_si_007.pdf">jm1c00191_si_007.pdf (1.01 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00191&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00191%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-14" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00191" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679916531e3a22a8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
