
<html lang="en"     class="pb-page"  data-request-id="dd471d55-872d-4a87-a60e-92571f40b14b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.6b01317;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2017.60.issue-7"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Competitive and Noncompetitive Ligands of the Organic Cation Transporter 1 (OCT1; SLC22A1)" /></meta><meta name="dc.Creator" content="Eugene C.  Chen" /></meta><meta name="dc.Creator" content="Natalia  Khuri" /></meta><meta name="dc.Creator" content="Xiaomin  Liang" /></meta><meta name="dc.Creator" content="Adrian  Stecula" /></meta><meta name="dc.Creator" content="Huan-Chieh  Chien" /></meta><meta name="dc.Creator" content="Sook Wah  Yee" /></meta><meta name="dc.Creator" content="Yong  Huang" /></meta><meta name="dc.Creator" content="Andrej  Sali" /></meta><meta name="dc.Creator" content="Kathleen M.  Giacomini" /></meta><meta name="dc.Description" content="Organic cation transporter 1 (OCT1) plays a critical role in the hepatocellular uptake of structurally diverse endogenous compounds and xenobiotics. Here we identified competitive and noncompetitiv..." /></meta><meta name="Description" content="Organic cation transporter 1 (OCT1) plays a critical role in the hepatocellular uptake of structurally diverse endogenous compounds and xenobiotics. Here we identified competitive and noncompetitiv..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 15, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01317" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01317" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01317" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01317" /></link>
        
    
    

<title>Discovery of Competitive and Noncompetitive Ligands of the Organic Cation Transporter 1 (OCT1; SLC22A1) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01317" /></meta><meta property="og:title" content="Discovery of Competitive and Noncompetitive Ligands of the Organic Cation Transporter 1 (OCT1; SLC22A1)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0009.jpeg" /></meta><meta property="og:description" content="Organic cation transporter 1 (OCT1) plays a critical role in the hepatocellular uptake of structurally diverse endogenous compounds and xenobiotics. Here we identified competitive and noncompetitive OCT1-interacting ligands in a library of 1780 prescription drugs by combining in silico and in vitro methods. Ligands were predicted by docking against a comparative model based on a eukaryotic homologue. In parallel, high-throughput screening (HTS) was conducted using the fluorescent probe substrate ASP+ in cells overexpressing human OCT1. Thirty competitive OCT1 ligands, defined as ligands predicted in silico as well as found by HTS, were identified. Of the 167 ligands identified by HTS, five were predicted to potentially cause clinical drug interactions. Finally, virtual screening of 29 332 metabolites predicted 146 competitive OCT1 ligands, of which an endogenous neurotoxin, 1-benzyl-1,2,3,4-tetrahydroisoquinoline, was experimentally validated. In conclusion, by combining docking and in vitro HTS, competitive and noncompetitive ligands of OCT1 can be predicted." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01317"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01317">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01317&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01317&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01317&amp;href=/doi/10.1021/acs.jmedchem.6b01317" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 2685-2696</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01261" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01357" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Competitive and Noncompetitive Ligands of the Organic Cation Transporter 1 (OCT1; SLC22A1)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eugene+C.++Chen">Eugene C. Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>#</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Natalia++Khuri">Natalia Khuri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span><span class="author-xref-symbol "><sup>○</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9031-8124" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaomin++Liang">Xiaomin Liang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Adrian++Stecula">Adrian Stecula</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Huan-Chieh++Chien">Huan-Chieh Chien</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sook+Wah++Yee">Sook Wah Yee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Huang">Yong Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrej++Sali">Andrej Sali</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-0435-6197" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kathleen+M.++Giacomini">Kathleen M. Giacomini</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-8041-5430" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, California 94143, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Optivia Biotechnology, Menlo Park, California 94025, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Pharmaceutical Chemistry, University of California, San Francisco, California 94158, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">California Institute of Quantitative Biosciences, University of California, San Francisco, California 94158, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Institute of Human Genetics, University of California, San Francisco, California 94143, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#eaa18b9e8293c48d838b898587838483aa9f89998cc48f8e9f"><span class="__cf_email__" data-cfemail="034862776b7a2d646a62606c6e6a6d6a43766070652d666776">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01317&amp;href=/doi/10.1021%2Facs.jmedchem.6b01317" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 2685–2696</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 23, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 September 2016</li><li><span class="item_label"><b>Published</b> online</span>15 March 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 April 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01317" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01317</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2685%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DEugene%2BC.%2BChen%252C%2BNatalia%2BKhuri%252C%2BXiaomin%2BLiang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D7%26contentID%3Dacs.jmedchem.6b01317%26title%3DDiscovery%2Bof%2BCompetitive%2Band%2BNoncompetitive%2BLigands%2Bof%2Bthe%2BOrganic%2BCation%2BTransporter%2B1%2B%2528OCT1%253B%2BSLC22A1%2529%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2696%26publicationDate%3DApril%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01317"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1891</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">28</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01317" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Competitive and Noncompetitive Ligands of the Organic Cation Transporter 1 (OCT1; SLC22A1)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Eugene&quot;,&quot;last_name&quot;:&quot;C. Chen&quot;},{&quot;first_name&quot;:&quot;Natalia&quot;,&quot;last_name&quot;:&quot;Khuri&quot;},{&quot;first_name&quot;:&quot;Xiaomin&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Adrian&quot;,&quot;last_name&quot;:&quot;Stecula&quot;},{&quot;first_name&quot;:&quot;Huan-Chieh&quot;,&quot;last_name&quot;:&quot;Chien&quot;},{&quot;first_name&quot;:&quot;Sook&quot;,&quot;last_name&quot;:&quot;Wah Yee&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Andrej&quot;,&quot;last_name&quot;:&quot;Sali&quot;},{&quot;first_name&quot;:&quot;Kathleen&quot;,&quot;last_name&quot;:&quot;M. Giacomini&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;2685-2696&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01317&quot;},&quot;abstract&quot;:&quot;Organic cation transporter 1 (OCT1) plays a critical role in the hepatocellular uptake of structurally diverse endogenous compounds and xenobiotics. Here we identified competitive and noncompetitive OCT1-interacting ligands in a library of 1780 prescription drugs by combining in silico and in vitro methods. Ligands were predicted by docking against a comparative model based on a eukaryotic homologue. In parallel, high-throughput screening (HTS) was conducted using the fluorescent probe substrate ASP+ in cells overexpressing human OCT1. Thirty competitive OCT1 ligands, defined as ligands predicted in silico as well as found by HTS, were identified. Of the 167 ligands identified by HTS, five were predicted to potentially cause clinical drug interactions. Finally, virtual screening of 29 332 metabolites predicted 146 competitive OCT1 ligands, of which an endogenous neurotoxin, 1-benzyl-1,2,3,4-tetrahydroisoquinoline, was experimentally validated. In conclusion, by combining docking and in vitro HTS, competitive &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01317&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01317" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01317&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01317" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01317&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01317" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01317&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01317&amp;href=/doi/10.1021/acs.jmedchem.6b01317" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01317" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01317" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01317%26sid%3Dliteratum%253Aachs%26pmid%3D28230985%26genre%3Darticle%26aulast%3DChen%26date%3D2017%26atitle%3DDiscovery%2Bof%2BCompetitive%2Band%2BNoncompetitive%2BLigands%2Bof%2Bthe%2BOrganic%2BCation%2BTransporter%2B1%2B%2528OCT1%253B%2BSLC22A1%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D7%26spage%3D2685%26epage%3D2696%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/jmcmar.2017.60.issue-7/20170413/jmcmar.2017.60.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Organic cation transporter 1 (OCT1) plays a critical role in the hepatocellular uptake of structurally diverse endogenous compounds and xenobiotics. Here we identified competitive and noncompetitive OCT1-interacting ligands in a library of 1780 prescription drugs by combining in silico and in vitro methods. Ligands were predicted by docking against a comparative model based on a eukaryotic homologue. In parallel, high-throughput screening (HTS) was conducted using the fluorescent probe substrate ASP<sup>+</sup> in cells overexpressing human OCT1. Thirty competitive OCT1 ligands, defined as ligands predicted in silico as well as found by HTS, were identified. Of the 167 ligands identified by HTS, five were predicted to potentially cause clinical drug interactions. Finally, virtual screening of 29 332 metabolites predicted 146 competitive OCT1 ligands, of which an endogenous neurotoxin, 1-benzyl-1,2,3,4-tetrahydroisoquinoline, was experimentally validated. In conclusion, by combining docking and in vitro HTS, competitive and noncompetitive ligands of OCT1 can be predicted.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26726" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26726" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Organic cation transporter 1 (OCT1; SLC22A1), a polyspecific membrane transporter, is among the most abundantly expressed transporters in human liver. Localized to the sinusoidal membrane of hepatocytes, OCT1 mediates the hepatic uptake of a diverse array of small positively charged hydrophilic compounds, including many endogenous bioactive amines<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (e.g., dopamine, histamine, and serotonin). We recently identified OCT1 as a high-capacity transporter of thiamine in the liver and showed that the transporter plays a key role in modulating hepatic energy status and lipid content.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Although the transporter clearly has important endogenous functions,<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> OCT1 has been characterized primarily as a drug transporter, capable of transporting a wide variety of prescription drugs, including the antidiabetic drug metformin and the opioid analgesic morphine. Genetic variants of OCT1 with reduced function have been associated with decreased response to metformin<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> as well as high systemic plasma levels of morphine and the active metabolite of the opioidergic drug tramadol.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Furthermore, administration of the calcium channel blocker verapamil (a potent inhibitor of OCT1) has been shown to reduce response to metformin, presumably through reducing hepatic drug levels.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In recognition of its critical role in drug disposition and response, OCT1 was included in a group of transporters of clinical importance by the International Transporter Consortium.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In 2012, the European Medicines Agency (EMA) recommended in vitro inhibition studies against OCT1 for investigational drugs in its Guidance on the Investigation of Drug Interactions.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">To date, more than 50 inhibitors of OCT1 transport have been identified by in vitro inhibition studies using radioactive or fluorescent probe substrates.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> However, these studies have not identified the mechanisms by which inhibitors modulate OCT1 transport. Growing evidence suggests that OCT1-mediated transport can be inhibited in a “substrate-dependent” manner due to the presence of multiple, possibly overlapping binding sites on the protein.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Identification and characterization of OCT1 ligands could be facilitated by the availability of an atomic structure. However, the three-dimensional (3D) structures of human OCT1 and its mammalian orthologues have not yet been determined. Although several residues important for substrate binding have been reported and rationalized with OCT1 comparative models built using atomic structures of bacterial homologues,<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17">(12-17)</a> accurate prediction of the binding site(s) remains challenging because of the low sequence identity between bacterial proteins and human OCT1. Recently, a structure of a high-affinity phosphate transporter from the fungus <i>Piriformospora indica</i> (PiPT) was determined by X-ray crystallography.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The transporter shares approximately 20% sequence identity with human solute carrier (SLC) transporters, especially within the SLC22 family, thus providing a new opportunity for comparative modeling of OCT1 and virtual screening.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p last">We used a combination of in silico and high-throughput screening (HTS) methods to identify prescription drugs and endogenous metabolites that are ligands of OCT1, with the goals of predicting clinical drug interactions and understanding their interactions with OCT1 protein. To this end, we screened a prescription drug library in silico for compounds that interact with a predicted binding site on OCT1 using comparative structure modeling and virtual docking. In parallel, we conducted HTS for inhibitors of OCT1-mediated transport of the fluorescent ligand ASP<sup>+</sup> against the same prescription drug library. We identified 167 ligands in the screened library and predicted 30 competitive ligands. Moreover, we showed that combining structure-guided ligand discovery with structure–activity relationship models from HTS data enables the prediction of competitive ligands of OCT1 from endogenous and exogenous metabolites.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Comparative Model of OCT1 and Its Validation by Docking of Known Substrates</h3><div class="NLM_p">We modeled human OCT1 on the basis of the 2.9 Å structure of PiPT, a high-affinity phosphate transporter from <i>Piriformospora indica</i>, crystallized in an inward-facing occluded state with a bound substrate<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (Protein Data Bank ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4J05">4J05</a>). The best-scoring 3D model was selected using the normalized discrete optimized protein energy (zDOPE) potential.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The zDOPE score of −0.22 suggests that 60% of its Cα atoms are within 3.5 Å of their correct positions.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The model includes all 12 transmembrane helices (TMHs), organized into the N- and C-terminal domains, and putative binding sites were identified in the translocation cavity between the two domains.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The following regions were not modeled: a large extracellular loop between TMHs 1 and 2, an intracellular loop between TMHs 6 and 7, and the intracellular N- and C-termini. While these regions may play a role in OCT1 homo-oligomerization, studies in rat showed that homo-oligomerization does not affect substrate affinity and transport function.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Because ASP<sup>+</sup> was used as a probe substrate, we first docked ASP<sup>+</sup> into two predicted binding sites and selected the site with the best score (−6.08) for all subsequent docking. Next, we validated the accuracy of the comparative model of the OCT1 transporter by confirming that (1) known OCT1 substrates docked favorably against the predicted binding site and (2) residues implicated in OCT1 transport<a onclick="showRef(event, 'ref12 ref14 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref14 ref16 ref17">(12, 14, 16, 17)</a> were localized in the predicted binding site, as follows. First, we docked 15 known OCT1 endogenous and drug substrates against the predicted binding site (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). Twelve out of 15 substrates (80%) had favorable (negative) docking scores, ranging from −24.44 for acyclovir to −2.88 for oxaliplatin (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Table S1</a>). The positive scores for three compounds (e.g., prostaglandins and pentamidine) resulted from steric clashes between ligand and transporter atoms, indicating that either the predicted binding site is too small to accommodate larger OCT1 ligands or that these compounds bind at a different site in the translocation pore. We also performed an enrichment analysis of docked substrates and decoys and computed a logAUC metric of 22.94, which suggests docking accuracy comparable to that in previously reported virtual screening experiments for human SLC transporters.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> Second, we analyzed favorable docking poses to determine the frequency of predicted hydrogen bonds between binding site residues and substrate molecules. Thirteen residues in TMHs 4, 10, and 11 formed hydrogen bonds with substrate molecules (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). Previous mutagenesis studies and homology modeling efforts suggested that these residues are important for substrate binding and OCT1-mediated transport in other species.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17">(12-17)</a> In particular, negatively charged D474 is important for interactions with positively charged OCT1 substrates. Additionally, in rat, mutations of Y221 and D474 resulted in reduced uptake of tetraethylammonium (TEA).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Finally, docked substrates formed non-covalent interactions with W217, T225, I449, and Q447 in TMHs 4, 10, and 11. The equivalent residues in rat have also been implicated in ligand–transporter interactions.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Predicted binding site of OCT1 and representative substrates. (A) Thirteen residues (S29, F32, C36, N156, Y221, F273, W354, Y361, I446, S358, C450, C473, and D474) formed hydrogen bonds with docked substrate molecules. Six residues in the predicted binding site (W217, F244, I449, D357, Q447, and G476) participated in non-covalent and/or polar substrate–transporter interactions. OCT1 residues are shown as cornflower blue sticks. Structures of 12 favorably docked known OCT1 substrates are shown as orange sticks. Oxygen, nitrogen, sulfur, and hydrogen atoms are depicted in red, dark blue, yellow, and white, respectively. (B) 2D structures of representative OCT1 substrates drawn using MarvinView 14.7.7.0 (ChemAxon).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Prediction of New Ligands by Virtual Screening and Validation by HTS</h3><div class="NLM_p">We predicted new ligands of OCT1 by docking each one of the 1780 compounds in the Pharmakon drug library (MicroSource Discovery Systems; <a href="http://www.msdiscovery.com/" class="extLink">http://www.msdiscovery.com/</a>) against the predicted binding site on the OCT1 model. From the 1780 compounds, 471 putative OCT1 ligands were predicted (normalized docking scores less than −1). These predictions were then tested by HTS of the entire Pharmakon library. The HTS assay relied on the uptake of the fluorescent substrate ASP<sup>+</sup> by OCT1-overexpressing HEK cells to assess the activity of the transporter.<a onclick="showRef(event, 'ref10 ref24'); return false;" href="javascript:void(0);" class="ref ref10 ref24">(10, 24)</a> The uptake of ASP<sup>+</sup> was linear for the first 5 min, and the <i>K</i><sub>m</sub> was determined to be 21.2 μM (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A,B). Thus, a substrate concentration of 2 μM was used to minimize the effect of substrate concentration on the IC<sub>50</sub> values, and an incubation time of 2 min was used to measure the initial rate of uptake. The average Z-prime score of the HTS was 0.80, indicating a reproducible assay.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Of the 1780 Pharmakon compounds, 167 compounds (9%) were determined to be OCT1 inhibitors (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C), defined as compounds that inhibited 50% or more of ASP<sup>+</sup> uptake at 20 μM. Drugs known to inhibit OCT1 activity at 20 μM were generally confirmed by the screen. Of the 167 compounds, 30 were also predicted as ligands by virtual screening. The overall accuracy of virtual screening was 70%. The sensitivity and specificity of predictions were 77% and 12%, respectively. The low specificity is not surprising because the docking pipeline was executed in a fully automated fashion; in contrast, typical structure-based virtual screening involves manual postdocking selection of ligand poses.<a onclick="showRef(event, 'ref22 ref23 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref26">(22, 23, 26)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Uptake of ASP<sup>+</sup> and HTS screening data. (A) Time-dependent ASP<sup>+</sup> uptake in HEK cells overexpressing OCT1 (●) or empty vector (■). (B) Overexpression of OCT1 increases ASP<sup>+</sup> uptake in HEK cells. ASP<sup>+</sup> uptake studies were conducted in HEK cells overexpressing OCT1 or empty vector. Cells were incubated with increasing concentrations of ASP<sup>+</sup> for 2 min. The uptake kinetic parameters were calculated using the difference in ASP<sup>+</sup> accumulation between cells overexpressing OCT1 and empty vector cells. Each data point represents mean ± SD, <i>n</i> = 6. (C) Distribution of inhibition values from HTS of 1780 compounds. A total of 167 inhibitors were identified among the 1780 Pharmakon compounds. (D) Distribution of the 167 inhibitors in various pharmacological classes. Brown color represents mixed class.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We examined compounds predicted to be OCT1 ligands by the comparative model but not identified as inhibitors by HTS. In view of the published literature, we identified 13 previously reported substrates and five inhibitors of OCT1. Among these predicted ligands, cimetidine (rank #21), metformin (rank #84), and thiamine (rank #200) inhibited OCT1-mediated uptake of ASP<sup>+</sup> by 4.13%, 11.1%, and −1.5%, respectively. The docking poses of all three compounds predicted favorable interactions with the comparative model; in particular, metformin and cimetidine formed hydrogen bonds with Asp474 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A,B). The inability of HTS to identify some of the previously reported ligands can be explained by their OCT1 affinity, which is much weaker than that of ASP<sup>+</sup>. For example, the reported IC<sub>50</sub> values of cimetidine and metformin for inhibition of OCT1-mediated transport of YM155 and MPP<sup>+</sup> were 149 and 1230 μM, respectively,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> and the IC<sub>50</sub> of thiamine was determined to be 4.1 mM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Figure S1</a>). Because our HTS assay measured inhibition of OCT1 activity by test compounds at 20 μM, it was unable to identify ligands such as cimetidine, metformin, and thiamine.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking results and physiochemical properties of OCT1 ligands in the Pharmakon library. Compounds are shown in orange sticks and hydrogen bonds as black dotted lines. (A) Predicted pose of metformin and its hydrogen bond with aspartic acid residue 474. (B) Predicted pose of cimetidine and its hydrogen bonds with D474 and S29. (C) Differences in distribution of physicochemical properties for predicted competitive (<i>n</i> = 30) and noncompetitive (<i>n</i> = 137) inhibitors of OCT1. Only significantly different distributions are shown (Student’s <i>t</i> test, <i>p</i> < 0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">HTS identified tricyclic antidepressants, antihistamines, and α-adrenergic receptor agonists, agreeing with previously published results.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> HTS also identified a high proportion of ligands from other drug classes, including calcium channel blockers, β-adrenergic receptor agonists/antagonists, and muscarinic acetylcholine receptor agonists/antagonists (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D). Selected hits from different classes of drugs were validated in vitro (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Figure S2</a>). Finally, HTS identified drugs that were less known to interact with OCT1, including carvedilol<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (an antihypertensive medication) and ethopropazine (an anti-Parkinsonian agent). Selected hits not known previously to interact with OCT1 were validated by determining their IC<sub>50</sub> values in vitro (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). Analysis of the physicochemical properties showed that OCT1 ligands tend to have fewer hydrogen-bond donors and acceptors and are less polar but more lipophilic than nonligands (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). As expected, ligands were more likely to be positively charged (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Selected inhibition studies of previously unknown OCT1 ligands and their estimated half-maximal inhibitory concentrations (IC<sub>50</sub>), with 95% confidence limits in parentheses. (A) Alfuzosin, an α<sub>1</sub>-adrenergic receptor antagonist. IC<sub>50</sub> = 14.9 (11.9; 18.5) μM. (B) Bithionol, an anthelmintic. IC<sub>50</sub> = 2.2 (2.0; 2.5) μM. (C) Camylofine, an antimuscarinic. IC<sub>50</sub> = 9.1 (8.0; 10.4) μM. (D) Carbetapentane, an antitussive. IC<sub>50</sub> = 1.6 (1.2; 2.0) μM. (E) Carvedilol, a nonselective β/α<sub>1</sub>-adrenergic receptor antagonist. IC<sub>50</sub> = 3.4 (3.0; 3.9) μM. (F) Dobutamine, a sympathomimetic. IC<sub>50</sub> = 4.17 (3.73; 4.67) μM. Each data point represents mean ± SD, <i>n</i> = 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Differences in physicochemical properties of 167 inhibitors and 1613 noninhibitors. Shown are box plots of molecular weight (MW), number of heavy atoms (HA), molecular volume (MV), number of hydrogen-bond donors (HBD), number of hydrogen-bond acceptors (HBA), number of rotatable bonds (RB), SLogP, total polar surface area (TPSA), and charge at pH 7.4. Statistically significant differences were estimated using Student’s <i>t</i> test. Distributions of inhibitors and noninhibitors were significantly different (<i>p</i> < 0.05) for HBD, HBA, SLogP, TPSA, and charge.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Summary of IC<sub>50</sub> Values for Selected Inhibition Studies</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> 95% confidence interval</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ketoconazole</td><td class="colsep0 rowsep0" align="char" char=".">2.60</td><td class="colsep0 rowsep0" align="left">2.49 to 2.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">closantel</td><td class="colsep0 rowsep0" align="char" char=".">3.00</td><td class="colsep0 rowsep0" align="left">2.83 to 3.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dobutamine</td><td class="colsep0 rowsep0" align="char" char=".">4.17</td><td class="colsep0 rowsep0" align="left">3.73 to 4.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">alfuzosin</td><td class="colsep0 rowsep0" align="char" char=".">14.87</td><td class="colsep0 rowsep0" align="left">11.92 to 18.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">erlotinib</td><td class="colsep0 rowsep0" align="char" char=".">16.24</td><td class="colsep0 rowsep0" align="left">11.34 to 23.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carbetapentane</td><td class="colsep0 rowsep0" align="char" char=".">1.55</td><td class="colsep0 rowsep0" align="left">1.19 to 2.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clotrimazole</td><td class="colsep0 rowsep0" align="char" char=".">11.97</td><td class="colsep0 rowsep0" align="left">9.97 to 14.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bithionol</td><td class="colsep0 rowsep0" align="char" char=".">2.23</td><td class="colsep0 rowsep0" align="left">1.98 to 2.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carvediolol</td><td class="colsep0 rowsep0" align="char" char=".">3.43</td><td class="colsep0 rowsep0" align="left">3.02 to 3.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clonidine</td><td class="colsep0 rowsep0" align="char" char=".">18.98</td><td class="colsep0 rowsep0" align="left">15.82 to 22.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">trimethoprim</td><td class="colsep0 rowsep0" align="char" char=".">50.68</td><td class="colsep0 rowsep0" align="left">41.47 to 61.93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">guanabenz</td><td class="colsep0 rowsep0" align="char" char=".">4.85</td><td class="colsep0 rowsep0" align="left">4.17 to 5.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pyrimethamine</td><td class="colsep0 rowsep0" align="char" char=".">13.57</td><td class="colsep0 rowsep0" align="left">10.74 to 17.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dichlorophene</td><td class="colsep0 rowsep0" align="char" char=".">8.41</td><td class="colsep0 rowsep0" align="left">6.61 to 10.69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">imipramine</td><td class="colsep0 rowsep0" align="char" char=".">7.95</td><td class="colsep0 rowsep0" align="left">5.91 to 10.68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cloperastine</td><td class="colsep0 rowsep0" align="char" char=".">14.89</td><td class="colsep0 rowsep0" align="left">14.00 to 15.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dextromethorphan</td><td class="colsep0 rowsep0" align="char" char=".">10.45</td><td class="colsep0 rowsep0" align="left">8.66 to 12.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">propafenone</td><td class="colsep0 rowsep0" align="char" char=".">15.54</td><td class="colsep0 rowsep0" align="left">14.04 to 17.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tacrine</td><td class="colsep0 rowsep0" align="char" char=".">21.72</td><td class="colsep0 rowsep0" align="left">18.77 to 25.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ethopropazine</td><td class="colsep0 rowsep0" align="char" char=".">20.46</td><td class="colsep0 rowsep0" align="left">18.31 to 22.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nitroprusside</td><td class="colsep0 rowsep0" align="char" char=".">43.84</td><td class="colsep0 rowsep0" align="left">38.08 to 50.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sunitinib</td><td class="colsep0 rowsep0" align="char" char=".">6.10</td><td class="colsep0 rowsep0" align="left">5.47 to 6.79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">desipramine</td><td class="colsep0 rowsep0" align="char" char=".">9.18</td><td class="colsep0 rowsep0" align="left">8.08 to 10.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">doxepin</td><td class="colsep0 rowsep0" align="char" char=".">11.19</td><td class="colsep0 rowsep0" align="left">9.89 to 12.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">camylofine</td><td class="colsep0 rowsep0" align="char" char=".">9.12</td><td class="colsep0 rowsep0" align="left">8.03 to 10.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thiamine</td><td class="colsep0 rowsep0" align="char" char=".">4354</td><td class="colsep0 rowsep0" align="left">3704 to 5119</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Prediction of Competitive Ligands</h3><div class="NLM_p">Thirty of the 1780 compounds in the library were identified as ligands by both virtual screening and HTS. Because docking was performed against the predicted binding site, we hypothesize that it can only identify competitive inhibitors or substrates. In contrast, HTS can identify both competitive and noncompetitive inhibitors or substrates. Thus, the 30 compounds that were identified with both methods are likely to be competitive inhibitors or even substrates. The remaining 137 compounds are hypothesized to be noncompetitive and/or mixed inhibitors of OCT1. It is also expected that this group may contain misclassified compounds. For example, inhibitors could be missed in HTS and thus be left out of our downstream analysis, resulting in false negatives. Alternatively, since docking was performed against a single conformation of a comparative transporter model, competitive inhibitors could have been misclassified by our protocol. We evaluated some of the compounds from this set of 137 inhibitors. Lineweaver–Burk plots were constructed for selected compounds in each class (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). Tacrine and ethopropazine, neither previously known to inhibit OCT1, were confirmed to be competitive inhibitors of ASP<sup>+</sup> and metformin uptake by OCT1 (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">S3</a>). Furthermore, we assessed the relative competitiveness of these inhibitors by calculating <i>K</i><sub>is</sub> (the dissociation constant for the transporter–inhibitor complex) and <i>K</i><sub>ii</sub> (the dissociation constant for the transporter–substrate–inhibitor complex) using previously published methods.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The ratios of <i>K</i><sub>ii</sub> and <i>K</i><sub>is</sub> for 14 compounds are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Table S2</a>. A larger <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub> value indicates a mode of inhibition that is relatively more competitive, whereas a lower <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub> value reflects less competitiveness. Our <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub> values ranged from 1.26 to 95.62 (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Table S2</a>). Thiamine had the highest value of <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub>, which indicated that it competitively inhibits ASP<sup>+</sup> uptake. In addition, thiamine has been validated as an OCT1 substrate in our previous publication.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Among the 14 compounds, compounds that were predicted to be competitive inhibitors resulted in relatively higher <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub>, whereas those that were predicted to be noncompetitive inhibitors had lower <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub> values.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Lineweaver–Burk plots for discriminating between competitive and noncompetitive inhibitors of OCT1. The inhibitory effects on ASP<sup>+</sup> uptake by OCT1 with increasing concentrations of ASP<sup>+</sup> were measured at various concentrations (● < ■ < ▲ < ▼) of the following inhibitors: (A) tacrine; (B) ethopropazine; (C) clonidine; (D) pyrimethamine; (E) trimethoprim; (F) dichlorophene; (G) alfuzosin; (H) clotrimazole; (I) imipramine; (J) dextromethorphan; (K) propafenone; (L) carvedilol. Each data point represents mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Physicochemical Properties of Putative Competitive and Noncompetitive Inhibitors</h3><div class="NLM_p">We analyzed the physicochemical properties of 30 putative competitive and 137 noncompetitive ligands. Our analysis revealed that noncompetitive ligands were significantly larger and more hydrophobic than competitive ligands (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C), leading to the prediction that the two types of ligands might bind to different sites on the transporter.</div><div class="NLM_p">In liver, several SLC transporters participate in uptake of drugs across the sinusoidal membrane into hepatocytes. A comparison of our HTS results with those for two other liver uptake transporters,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> OATP1B1 (SLCO1B1) and OATP1B3 (SLCO1B3), allowed us to identify OCT1-selective versus pan inhibitors (i.e., compounds that inhibited transport of all three liver transporters). Fifty compounds inhibited the three liver transporters, whereas 112 inhibited OCT1 only. As expected, differences in charge were significant (Student’s <i>t</i> test, <i>p</i> < 0.05; <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Figure S4</a>). Additionally, we found that OCT1 inhibitors were significantly smaller and less hydrophobic (Student’s <i>t</i> test, <i>p</i> < 0.05). This result showed that multiway comparison of HTS data for several transporters can help in identifying compounds that are selective for a specific transporter and underscores the need for HTS of additional SLC transporters.</div><div class="NLM_p last">We next determined the fractions of inhibitors with different predicted inhibitory mechanisms among the OCT1-selective inhibitors and pan inhibitors (see the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx" class="ext-link">Supporting Information</a>). Eight compounds inhibited OCT1 and OATP1B1 but not OATP1B3 (see the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx" class="ext-link">Supporting Information</a>). Of these eight compounds, only ethacridine lactate was predicted to competitively inhibit OCT1. Likewise, only two out of 10 OCT1/OATP1B3 inhibitors were predicted to inhibit OCT1 competitively, and only two predicted competitive inhibitors were identified among 30 pan inhibitors. Interestingly, 22 out of 30 predicted competitive ligands of OCT1 (73%) were found among the OCT1-selective inhibitors. That is, these 22 were not inhibitors of OATP1B1 or OATP1B3. In contrast, only five out 30 predicted competitive ligands (16%) were also inhibitors of OATP1B1 and OATP1B3 (the remaining three compounds were not screened in OATP1B1/B3 HTS). This finding not only supports the accuracy of our comparative model but also highlights the importance of combining structure-guided ligand discovery with HTS. By combining experimental HTS with docking against a comparative model of OCT1, we can efficiently predict competitive OCT1-selective inhibitors with 73% accuracy. Therefore, a structure-guided approach greatly accelerates the identification of selective inhibitors and provides valuable information for drug–drug interaction studies compared with the conventional trial-and-error approach. Pan, OCT1/OATP1B1, and OCT1/OATP1B3 inhibitors were enriched for noncompetitive inhibitors compared with OCT1-selective inhibitors.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Modeling and Validation</h3><div class="NLM_p">Data from our OCT1 HTS experiment were used as a training set to construct a binary structure–activity relationship (SAR) model correlating molecular features of 1780 compounds from the Pharmacon library with their inhibitory activities, discretized into two classes: inhibitors and noninhibitors. The Random Forest (RF) algorithm<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> was employed to build an ensemble classifier (SAR-I). We evaluated the accuracy of the SAR-I model (i.e., the area under the receiver operating characteristic curve (auROC)) by 100 repeated cross-validation runs (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). The average auROC of RF classifiers in this retrospective validation was 0.89 ± 0.3. This accuracy is comparable to the accuracies of retrospective validation of SAR models for other transporters.<a onclick="showRef(event, 'ref24 ref34'); return false;" href="javascript:void(0);" class="ref ref24 ref34">(24, 34)</a> In addition, we estimated the accuracy of RF-based SAR models in prospective validation as follows. First, we used molecular features and inhibitory outcomes of 183 compounds from a small previously published OCT1 inhibition screen<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> to develop a new SAR model (SAR-II). Next, SAR-II was utilized to predict the classes of the 1780 Pharmakon compounds. The auROC for this prospective validation was 0.84 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). Thus, the decrease in accuracy measures between retrospective and prospective validation was only 5%, strongly suggesting that our OCT1 SAR-I model is highly accurate (approximately 84%). We also used the SAR-II model to predict the sensitivity and specificity of the classifier at different cutoff values (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>C). The sensitivity and specificity of the SAR-II model at a cutoff value of 0.6 were 92% and 43%, respectively; at a cutoff value of 0.4, they were 82% and 65%, respectively. Finally, the observed and SAR-II-predicted classification scores of the 1780 Pharmakon compounds were moderately correlated (Pearson correlation coefficient of 0.50; <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>D). These results suggest that SAR models can accurately predict OCT1 ligands by virtual screening.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Results of structure–activity relationship modeling. (A) ROC curves for 100 retrospective cross-validation runs. The average ROC curve is shown in black. The ROC curve of a random classifier is shown as a red dotted line. (B) ROC curve for the SAR model tested in prospective validation of 1780 predicted inhibition values. The performances at different cutoff values (shown on the right <i>y</i>-axis) are indicated by rainbow colors. (C) Accuracy of classification of 1780 compounds using the SAR-II model as a function of cutoff. A cutoff of 0.38 is indicated by the red dotted line. (D) Observed vs predicted inhibition values for 1780 compounds. Two classification cutoffs are drawn vertically, and the HTS classification cutoff is drawn horizontally.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Virtual Screening of Endogenous and Drug Metabolites</h3><div class="NLM_p">In addition to searching for OCT1 ligands among the 1780 prescription drugs, we applied our structure-based and SAR model methods to predict OCT1 ligands among a larger set of 29 332 endogenous and drug metabolites in the Human Metabolome Database (HMDB).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> We found that 864 out of the 29 332 compounds (3%) docked favorably. We then computed ligand–OCT1 interaction values for the 864 compounds using the SAR-I model, allowing us to predict 146 competitive ligands. Among these 146 compounds, 1-benzyl-1,2,3,4-tetrahydroisoquinoline (1BnTIQ), an endogenous amine present at high levels in the cerebrospinal fluid of Parkinson’s disease patients,<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a> docked favorably against OCT1 (rank #14) and had SAR-I inhibition score of 0.48. 1BnTIQ inhibited OCT1 at 82.1 μM, and the Lineweaver–Burk plot confirmed that 1BnTIQ inhibited OCT1 competitively (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). This validation suggests that combined docking and SAR-I virtual screening can predict OCT1 metabolite ligands in addition to prescription drug ligands.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibitory effects of 1BnTIQ on OCT1 transport. (A) 1BnTIQ inhibited ASP<sup>+</sup> uptake by OCT1, and the IC<sub>50</sub> was determined to be 82.1 μM. (B) The inhibitory effect of 1BnTIQ on ASP<sup>+</sup> uptake by OCT1 indicated competitive inhibition. Each data point represents mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">OCT1, a protein of great pharmacological interest, transports a wide array of drugs into and out of the liver and thus serves as a major determinant of drug metabolism and action. Because of its clinical importance, OCT1 has become a focus of many pharmacogenomics and drug interaction studies, which have prompted the EMA to recommend that all new drugs undergo in vitro testing to assess their liability to interact with OCT1.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The goals of the current study were (i) to develop robust computational models to predict the interaction of new molecular entities with OCT1 and (ii) to use a combination of experimental and computational approaches to identify prescription drug and metabolite ligands of OCT1. By combining in silico and in vitro approaches, we sought to gain information about whether a ligand binds competitively or noncompetitively on OCT1.</div><div class="NLM_p">Three major findings emerged from the current studies. First, a comparative structural model of OCT1 successfully discriminated ligands from nonligands of the transporter. Second, by combining molecular docking and HTS approaches, we were indeed successful in determining whether a ligand interacts competitively or noncompetitively with the transporter. Third, we identified 30 and 137 prescription drugs as competitive and noncompetitive ligands of OCT1, respectively, including drugs not known to interact with the transporter. We now discuss each one of these findings in turn.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> OCT1 Ligands Can Be Identified Accurately by Virtual Screening against an OCT1 Comparative Model</h3><div class="NLM_p">Accurate prediction of inhibitors and substrates of OCT1 is challenging. First, although several atomic structures have been resolved for bacterial members of the major facilitator superfamily of transporters, low sequence identity between bacterial homologues and human OCT1 casts doubt on the accuracy of comparative models built using these structures as templates.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Second, OCT1 is considered a polyspecific transporter that transports compounds of different sizes and molecular features. For example, OCT1 mediates the uptake of compounds ranging from small cations such as tetraethylammonium to monoamines (metformin), chemotherapy drugs (oxaliplatin), and hormone-like lipid compounds (prostaglandin E1).<a onclick="showRef(event, 'ref4 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref4 ref38 ref39 ref40">(4, 38-40)</a> This broad specificity may result in inaccurate docking and SAR models. For example, ligand-based models may underpredict ligands, especially in chemical spaces unsampled by the training set. Similarly, docking generally does not consider multiple binding sites, which are characteristic of polyspecific transporters. Finally, docking also depends on the accuracy of the target structure, and therefore, a combination of several computational and experimental validation experiments should be performed to ascertain the applicability of a homology model for docking.</div><div class="NLM_p last">Here we built a comparative model of human OCT1 in an inward-facing occluded conformation using a recently determined structure of its eukaryotic homologue, a phosphate transporter from <i>Piriformospora indica</i> (PiPT).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> We confirmed that 80% of the known structurally diverse substrates can be docked favorably against the predicted binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Table S1</a>). Three compounds that had unfavorable docking scores against this binding site had steric clashes with side chains of the binding cavity, underscoring the need to model transporters in alternative conformations. Unfortunately, there are no template structures for alternative conformations of OCT1, and molecular dynamics simulations still lack computing power to model conformational changes of transporters. The average pairwise Tanimoto coefficient of substrates used to validate the predicted binding site was 0.33, indicating that the compounds were structurally unrelated (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Figure S5</a>). In an unbiased validation of the OCT1 comparative model, we docked 1780 compounds from the Pharmakon library and validated 70% of the predicted binders and nonbinders in vitro. Previous SAR and pharmacophore models of OCT1–ligand interactions identified hydrophobicity and charge as the main physicochemical properties required for inhibition.<a onclick="showRef(event, 'ref10 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref10 ref41 ref42">(10, 41, 42)</a> Our screening results confirmed that charge and hydrophobicity are positively correlated with the inhibitory activity of compounds (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). In addition, ligands had fewer hydrogen-bond donors and acceptors and were less polar than nonligands.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> A Combination of Virtual Screening and HTS Can Determine Whether a Ligand Binds Competitively or Noncompetitively</h3><div class="NLM_p">It has been frequently assumed that inhibition of SLC-mediated transport is primarily competitive, although some recent studies have challenged this assumption by showing that competitive, noncompetitive, and mixed-type inhibition can also occur.<a onclick="showRef(event, 'ref11 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref11 ref43 ref44 ref45">(11, 43-45)</a> The International Transporter Consortium recently pointed out that the lack of understanding of inhibition mechanisms remains a limiting factor in transporter studies in drug development.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> We showed that inhibitors of OCT1-mediated ASP<sup>+</sup> transport can be divided into two groups on the basis of their docking scores against the OCT1 model. Predicted competitive inhibitors of ASP<sup>+</sup> transport are compounds that are identified by experimental screening as well as predicted by virtual screening against the predicted substrate binding site; our assay and calculation do not distinguish between competitive inhibitors and substrates. In contrast, predicted noncompetitive inhibitors are compounds that are identified by experimental screening as well as predicted not to bind by virtual screening. Competitive ligands were significantly smaller and less hydrophobic than the noncompetitive ligands (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). Because we modeled OCT1 in an inward-facing occluded conformation, we predict competitive ligands from only those compounds that fit into the compact translocation cavity. In addition, a broad range of <i>K</i><sub>ii</sub>/<i>K</i><sub>is</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Table S2</a>) suggested various degrees of competitiveness among these inhibitors. Some of the inhibitors may bind to another binding cavity or to alternative OCT1 conformations, and these will not be predicted as competitive ligands by our docking approach. Furthermore, interactions with OCT1 have been shown to be ligand-dependent.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> We used ASP<sup>+</sup> as our primary probe substrate in this study, but the mechanism of ligand-dependent interaction will need to be further investigated.</div><div class="NLM_p">Additionally, we showed that 87% of predicted competitive ligands were selective for OCT1. In contrast, only 13% of predicted competitive ligands were identified among inhibitors of the three hepatic uptake transporters OCT1, OATP1B1, and OATP1B3. This result provides further confidence in the accuracy of the comparative model and its ability to predict competitive ligands.</div><div class="NLM_p last">To predict competitive ligands in a large virtual library of potential ligands, we combined the SAR model (built using our HTS data) and docking against a comparative OCT1 model. Indeed, 146 putative competitive ligands among endogenous and exogenous metabolites in the HMDB library were predicted; one of them, 1BnTIQ, was experimentally tested and validated (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). 1BnTIQ is an endogenous amine detected in human cerebrospinal fluid that accumulates in patients with Parkinson’s disease<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and is able to induce parkinsonism in both mice and monkeys.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a> 1BnTIQ inhibits complex I in the mitochondria and induces dopaminergic death in the same manner as MPP<sup>+</sup>, a neurotoxin also known to induce parkinsonism. Structurally similar to MPP<sup>+</sup>, 1BnTIQ is hydrophilic and requires an uptake mechanism to enter cells. A likely uptake mechanism is suggested by our identification of 1BnTIQ as an OCT1 ligand. While promising, our modeling method has low specificity, resulting in a large number of false positives. The modeling could be improved by docking against alternative conformations of the transporter once additional template structures become available.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Novel OCT1 Inhibitors Can Be Identified by HTS</h3><div class="NLM_p">Because of the critical role OCT1 plays in drug disposition and response, efforts have been made to identify and characterize OCT1 inhibitors. For example, 20 pharmacologically diverse antidepressants and 14 antipsychotics were screened using an OCT1-mediated radiolabeled MPP<sup>+</sup> uptake assay, identifying drugs that could potentially inhibit 50% or more OCT1 activity in the brain.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> In another study, 191 drugs from various sources were compiled, followed by a medium-throughput identification of 62 inhibitors.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">In this study, we conducted an extensive HTS of 1780 drugs that have reached at least clinical trials in the United States or are marketed in Europe and/or Asia. We were able to confirm most of the previously known inhibitors. We also estimate that we identified at least 100 compounds previously unknown to interact with the transporter. Moreover, we grouped the identified OCT1 ligands into therapeutic classes, including tricyclic antidepressants, antihistamines, steroids, and α-adrenergic receptor agonists, all of which were previously reported to be more likely to interact with OCT1.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Our HTS also identified additional drug classes that were enriched in OCT1 ligands, including β-adrenergic receptor agonists/antagonists, calcium channel blockers, and muscarinic acetylcholine receptor agonists/antagonists. Among the inhibitors identified by HTS, selected compounds were validated by determining their IC<sub>50</sub> values (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> and <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Five of them (carbetapentane, carvedilol, erlotinib, griseofulvin, and ketoconazole) had IC<sub>50</sub> values that were at least 10% of their maximum plasma concentrations achieved after therapeutic doses of the drug (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). These estimates suggest the possibility of clinical drug–drug interactions with OCT1 substrates. Furthermore, OCT1 inhibitors may potentially have beneficial effects on hepatic steatosis.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> With the exception of erlotinib and ketoconazole, the drugs noted here were newly identified OCT1 ligands. We also estimated their IC<sub>50</sub> values for mouse OCT1 and two other human SLC uptake transporters (OCT2 and MATE1) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Table S3</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Identified OCT1 Inhibitors That Could Cause Drug–Drug Interactions</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="right" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>MAX</sub> (μM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>C</i><sub>Portal Vein</sub> (μM)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>MAX</sub>/IC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carbetapentane</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="right">N.A.</td><td class="colsep0 rowsep0" align="char" char=".">0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carvedilol</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="right">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">erlotinib</td><td class="colsep0 rowsep0" align="char" char=".">16.2</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="right">7.0</td><td class="colsep0 rowsep0" align="char" char=".">0.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">griseofulvin</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="right">4.6</td><td class="colsep0 rowsep0" align="char" char=".">0.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ketoconazole</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="right">10.0</td><td class="colsep0 rowsep0" align="char" char=".">2.54</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">IC<sub>50</sub> is the estimated half-maximal inhibitory concentration (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>).</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last"><i>C</i><sub>MAX</sub> values were obtained from <a href="http://www.micromedexsolutions.com/" class="extLink">http://www.micromedexsolutions.com/</a>.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last"><i>C</i><sub>Portal Vein</sub> values were calculated using the equation described previously.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></p></div></div></div><div class="NLM_p last">In conclusion, we have developed a comparative structural model of OCT1 that discriminates ligands from nonligands and used it together with an in vitro HTS assay. By combining the two approaches, we were able to predict whether a ligand binds competitively or noncompetitively. The structure-guided approach also accurately predicted inhibitors specific to OCT1 rather than two other hepatic drug transporters, OATP1B1 and OATP1B3. Finally, we conducted a virtual screen against a metabolite library using both comparative and SAR models built from HTS data and accurately identified and validated the parkinsonism-producing neurotoxin 1BnTIQ as a competitive inhibitor of OCT1.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86365" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86365" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemicals</h3><div class="NLM_p last">The MicroSource Pharmakon compound library (Gaylordsville, CT) was obtained through the Small Molecule Discovery Center at the University of California, San Francisco (San Francisco, CA). 4-(4-(Dimethylamino)styryl)-<i>N</i>-methylpyridinium iodide was purchased from Molecular Probes (Grand Island, NY). All other chemicals were purchased from Sigma-Aldrich (St. Louis, MO). All chemicals used in the studies were purchased from commercial vendors and had purities of 98% or higher. All cell culture media and supplements were purchased from Life Technologies (Carlsbad, CA), except for fetal bovine serum, which was purchased from GE Healthcare Life Sciences (South Logan, UT).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Cell Culture</h3><div class="NLM_p last">Human embryonic kidney (HEK-293) cell line stably overexpressing OCT1 was established previously in our laboratory.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The cells were maintained in Dulbecco’s Modified Eagle’s Medium (DMEM H-21) supplemented with hygromycin B (75 μg/mL), penicillin (100 units/mL), streptomycin (100 mg/mL), and 10% fetal bovine serum in a humidified atmosphere with 5% CO<sub>2</sub> at 37 °C.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> In Vitro Uptake Studies</h3><div class="NLM_p last">HEK-293 cells overexpressing OCT1 were seeded in black, clear-bottom, poly(<span class="smallcaps smallerCapital">d</span>-lysine)-coated 96-well plates (Greiner Bio-One, Monroe, NC) and allowed to grow for 48 h until approximately 90% confluency. For uptake kinetics studies, cells were incubated with Hank’s balanced salt solution (HBSS) containing serial dilution of ASP<sup>+</sup> for 2 min at 37 °C. At the end of the experiments, the media were aspirated, and the cells were washed twice with ice-cold HBSS containing 50 μM spironolactone. The <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> were calculated by fitting the data to the Michaelis–Menten equation. For time course studies, cells were incubated with HBSS containing 2 μM ASP<sup>+</sup> at 37 °C. At various time points, the experiment was stopped as previously described. For IC<sub>50</sub> determination, cells were incubated with HBSS containing 2 μM ASP<sup>+</sup> and serial dilution of inhibitors for 2 min at 37 °C. IC<sub>50</sub> was determined using appropriate curve fitting. For Lineweaver–Burk plots, cells were incubated with HBSS containing serial dilution of ASP<sup>+</sup> and the inhibitor of interest at four different fixed concentrations for 2 min at 37 °C. The reciprocal value of ASP<sup>+</sup> uptake at each inhibitor concentration was fitted with linear regression. The signal of ASP<sup>+</sup> was measured using an Analyst AD plate reader (Molecular Devices, Sunnyvale, CA) with excitation and emission filters tuned at wavelengths of 485 and 585 nm, respectively. All statistical analysis and curve fitting were done using GraphPad Prism software (La Jolla, CA).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> High-Throughput Screening</h3><div class="NLM_p last">The high-throughput screen was performed at the Small Molecule Discovery Center at the University of California, San Francisco. HEK-293 cells overexpressing OCT1 were seeded in black, clear bottom, poly(<span class="smallcaps smallerCapital">d</span>-lysine)-coated 96-well plates (Greiner Bio-One) and allowed to grow for 48 h until approximately 90% confluency using methods established previously.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Cells were incubated with HBSS containing 2 μM ASP<sup>+</sup> and 20 μM test compound at ambient temperature for approximately 2 min. At the end of the experiment, media were aspirated, and cells were washed twice with HBSS containing 50 μM spironolactone. Nonspecific transport was determined in wells on each assay plate using 100 μM spironolactone as the OCT1 inhibitor. The screen was carried out with a Biomek FXp liquid handler (Beckman Coulter, Brea, CA). Fluorescence was measured as previously described.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> OCT1 Structure Modeling and Docking</h3><div class="NLM_p last">Human OCT1 was modeled using the 2.9 Å structure of a high-affinity phosphate transporter from <i>Piriformospora indica</i> (PiPT) crystallized in an inward-facing occluded state with bound phosphate.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The template was selected on the basis of the shared MFS fold assignment,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> structure quality, sequence similarity to OCT1, and ligand-bound conformation. The sequence alignment was obtained by a manual refinement of gaps in the output from the PROMALS3D<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and MUSCLE<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> servers (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">Figure S6</a>). One hundred models were generated using the “automodel” class of MODELLER 9.14 (<a href="https://salilab.org/modeller/contact.html" class="extLink">https://salilab.org/modeller/contact.html</a>) and the normalized discrete optimized protein energy (zDOPE) potential.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The top-scoring model was used to predict putative binding sites with the FTMap web server.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Two of the predicted binding sites were identified in the translocation cavity between the two domains.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> ASP<sup>+</sup> probe substrate was docked against the two binding sites with UCSF DOCK 3.6.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52, 53)</a> The size of the docking box was 38 Å × 40 Å × 38 Å. The pose with the best docking score was used as the template for another round of comparative modeling by MODELER 9.14, generating 100 new models of OCT1. Each of the models was then evaluated for ligand enrichment from a set of challenging decoys based on enrichment curves and corresponding logAUC values.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Sixty decoys were generated using the Database of Useful Decoys (DUD)<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(55)</a> for each of the selected experimentally validated substrates of OCT1. The best-scoring model was used for subsequent virtual screening. Compounds in the Pharmakon library were downloaded from the ZINC database<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(56)</a> and docked against the predicted binding site on the comparative model using UCSF DOCK 3.6. A negative DOCK score predicts a favorable interaction with the transporter, while a positive score predicts an unfavorable (or unlikely) intermolecular interaction. Normalized docking scores were computed by subtracting the average docking score of all compounds (including the nonbinders) from the docking score of an individual compound and dividing by the standard deviation of all docking scores. The PAINS-Remover online web server was used to check Pharmkon compounds for the likelihood of interference in screening.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> None of the 1780 compounds was reported as a pan-assay interference compound.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Structure–Activity Relationship Modeling of OCT1 Inhibition</h3><div class="NLM_p last">Two-dimensional (2D) structure files of the 1780 compounds from the Pharmakon library were also downloaded from the ZINC database,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(56)</a> and 2900 molecular descriptors and the charge at pH 7.4 for each compound were computed using PaDEL software<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and cxcalc (ChemAxon; <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>), respectively. Noninformative descriptors (i.e., molecular descriptors with near-zero variance and redundant descriptors defined by a correlation higher than 0.95 to an accepted descriptor) were removed. Next, the information content and correlation between descriptor values and percent inhibition of 1780 compounds were computed with the cfs filtering algorithm in the FSelector package<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> in R, and the 21 most informative descriptors were retained for further modeling. These descriptors are Moreau–Broto autocorrelation descriptors based on atomic polarizability (ATSp4), number of doubly bound carbons bound to two other carbons (C2SP2), count of atom-type E-State >CH– (nsssCH), sum of E-States for weak hydrogen-bond acceptors (SwHBa), sum of E-State descriptors of strength for potential hydrogen bonds of path length 7 (SHBint7), sum of atom-type E-State ═C< (SdssC), minimum E-State descriptors of strength for potential hydrogen bonds of path length 8 (minHBint8), minimum atom-type H E-State −OH (minHsOH), minimum atom-type E-State −CH<sub>3</sub> (minsCH3), minimum atom-type E-State −CH<sub>2</sub>– (minssCH2), minimum atom-type E-State >C< (minssssC), minimum atom-type E-State >NH<sup>+</sup>– (minsssN), maximum E-States for (strong) hydrogen-bond acceptors (maxHBa), maximum E-State descriptors of strength for potential hydrogen bonds of path length 2 (maxHBint2), maximum atom-type E-State ═C< (maxdssC), a measure of electronegative atom count (ETA_Epsilon_1), a measure of hydrogen-bonding propensity of the molecules and/or polar surface area (ETA_Psi_1), a measure of electronegative atom count of the molecule relative to molecular size (ETA_BetaP_s), Mannhold LogP (MLogP), logarithm of the calculated octanol/water partition coefficient (XLog), and charge (Charge). Percent inhibition values determined by HTS were discretized into two outcomes: values of at least 50% were mapped to “1” (i.e., ligand), and values of less than 50%, including negative values, were mapped to “0” (i.e., nonligand). Binary SAR models were built with the RF algorithm.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Their accuracy was estimated using a double-loop fivefold cross-validation<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> protocol in the caret package in R. To evaluate the accuracy of the models, the average area under the receiver operating characteristic curve (auROC) was computed from 100 repeated double-loop cross-validation runs. For prospective validation, 21 molecular descriptors for 183 compounds<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> were computed (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx" class="ext-link">Supporting Information</a>), and SAR models were optimized by repeated fivefold cross-validation. The model was employed to predict inhibition values for the 1780 compounds in the Pharmakon library.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Virtual Screening against HMDB Library</h3><div class="NLM_p last">For virtual screening, molecular structure files of the HMDB compounds were downloaded from the ZINC database,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(56)</a> and 21 molecular features (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx" class="ext-link">Supporting Information</a>) were computed for each compound. The ligand score ranging between 0 and 1 was computed for each compound by SAR-I. Compounds with ligand scores higher than 0.4 were labeled as ligands. In parallel, each compound was docked against predicted binding sites on the OCT1 transporter using UCSF DOCK 3.6. Normalized docking scores were computed by subtracting the average docking score of all docked compounds from the docking score of a compound and dividing by the standard deviation of all docking scores. Compounds with normalized docking scores less than −1 and ligand scores greater than 0.4 were labeled as competitive inhibitors.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Pairwise Compound Similarity Computation and Clustering</h3><div class="NLM_p last">The MayaChemTools (<a href="http://www.mayachemtools.org" class="extLink">http://www.mayachemtools.org</a>) package was used to compute 2D extended connectivity fingerprints (ECFPs) and pairwise Tanimoto coefficients. The <i>hclust</i> function from the <i>stats</i> library<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> was used to perform hierarchical compound clustering.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01317">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82384" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82384" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01317" class="ext-link">10.1021/acs.jmedchem.6b01317</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Summary of 15 substrates used to validate OCT1 docking site, the ratios of <i>K</i><sub>ii</sub> and <i>K</i><sub>is</sub> for selective inhibitors, comparison of IC<sub>50</sub> values for OCT1 inhibitors in four different cell lines, effect of thiamine on the uptake of ASP<sup>+</sup> by OCT1, selected inhibition studies of OCT1 ligands and their estimated IC<sub>50</sub> values, Lineweaver–Burk plots for discriminating between competitive and noncompetitive inhibitors of OCT1-mediated metformin uptake, differences in physicochemical properties of OCT1-selective and pan inhibitors, 2D dissimilarity clustering of 15 known substrates of OCT1, and pairwise sequence alignment of PiPT and OCT1 (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Full list of Pharmakon library compounds with SMILES strings, ASP<sup>+</sup> inhibition data, and docking scores; list of HMDB compounds with docking ranks; OCT1, OATP1B1, and OATP1B3 inhibition results; names and descriptions of the 21 descriptors used in SAR modeling; and slope and intercept data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">OCT1 comparative model (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf">jm6b01317_si_001.pdf (718.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx">jm6b01317_si_002.xlsx (367.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_003.pdb">jm6b01317_si_003.pdb (219.8 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01317" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16403" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16403" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathleen M. Giacomini</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, University
of California, San Francisco, California 94158, United States</span>; 
    <span class="hlFld-Affiliation affiliation">California
Institute of Quantitative Biosciences, University
of California, San Francisco, California 94158, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Human Genetics, University of California, San Francisco, California 94143, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8041-5430" title="Orcid link">http://orcid.org/0000-0001-8041-5430</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e5ae84918d9ccb828c84868a888c8b8ca590869683cb808190"><span class="__cf_email__" data-cfemail="0942687d6170276e60686a6664606760497c6a7a6f276c6d7c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eugene C. Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalia Khuri</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9031-8124" title="Orcid link">http://orcid.org/0000-0001-9031-8124</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaomin Liang</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adrian Stecula</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huan-Chieh Chien</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sook Wah Yee</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Huang</span> - <span class="hlFld-Affiliation affiliation">Optivia
Biotechnology, Menlo Park, California 94025, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrej Sali</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Sciences, University of California, San
Francisco, California 94143, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, University
of California, San Francisco, California 94158, United States</span>; 
    <span class="hlFld-Affiliation affiliation">California
Institute of Quantitative Biosciences, University
of California, San Francisco, California 94158, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0435-6197" title="Orcid link">http://orcid.org/0000-0003-0435-6197</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>E.C.C., N.K., and X.L. contributed equally to this work. E.C.C., N.K., X.L., Y.H., A. Sali, and K.M.G. designed the research; E.C.C., N.K., X.L., A. Stecula, H.-C.C., and S.W.Y. conducted the experiments; E.C.C., N.K., X.L., A. Stecula, H.-C.C., A. Sali, and K.M.G. analyzed the data; E.C.C., N.K., X.L., A. Sali, and K.M.G. wrote the paper.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d47e1792-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Robert M. Stroud, Dr. Ethan Geier, and Dr. Avner Schlessinger for helpful discussions and Steven Chen from the UCSF Small Molecule Discovery Center for help with the assay automation. The project was supported by grants from the National Institutes of Health (R01 GM54762, U54 GM62529, P01 GM71790, and P01 A135707 to A. Sali; U19 GM61390 and R01 GM117163 to K.M.G.; T32 GM007175 to E.C.C., X.L., and A. Stecula; T32 GM008284 to N.K.; 2R44GM086970-03A1 to Y.H.). We also acknowledge hardware gifts from Intel and IBM.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i29" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i29"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i30" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i30"> Abbreviations Used</h2><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">ASP<sup>+</sup></td><td class="NLM_def"><p class="first last">4-Di-1-ASP</p></td></tr><tr><td class="NLM_term">OCT1</td><td class="NLM_def"><p class="first last">organic cation transporter 1</p></td></tr><tr><td class="NLM_term">SLC</td><td class="NLM_def"><p class="first last">solute carrier</p></td></tr><tr><td class="NLM_term">TMH</td><td class="NLM_def"><p class="first last">transmembrane helix</p></td></tr><tr><td class="NLM_term">HMDB</td><td class="NLM_def"><p class="first last">Human Metabolome Database</p></td></tr><tr><td class="NLM_term">1BnTIQ</td><td class="NLM_def"><p class="first last">1-benzyl-1,2,3,4-tetrahydroisoquinoline</p></td></tr><tr><td class="NLM_term">HEK-293</td><td class="NLM_def"><p class="first last">human embryonic kidney cell line</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Boxberger, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagenbuch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampe, J. N.</span><span> </span><span class="NLM_article-title">Common Drugs Inhibit Human Organic Cation Transporter 1 (OCT1)-Mediated Neurotransmitter Uptake</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">990</span><span class="NLM_x">–</span> <span class="NLM_lpage">995</span><span class="refDoi"> DOI: 10.1124/dmd.113.055095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fdmd.113.055095" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=990-995&issue=6&author=K.+H.+Boxbergerauthor=B.+Hagenbuchauthor=J.+N.+Lampe&title=Common+Drugs+Inhibit+Human+Organic+Cation+Transporter+1+%28OCT1%29-Mediated+Neurotransmitter+Uptake&doi=10.1124%2Fdmd.113.055095"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fdmd.113.055095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.113.055095%26sid%3Dliteratum%253Aachs%26aulast%3DBoxberger%26aufirst%3DK.%2BH.%26aulast%3DHagenbuch%26aufirst%3DB.%26aulast%3DLampe%26aufirst%3DJ.%2BN.%26atitle%3DCommon%2520Drugs%2520Inhibit%2520Human%2520Organic%2520Cation%2520Transporter%25201%2520%2528OCT1%2529-Mediated%2520Neurotransmitter%2520Uptake%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2014%26volume%3D42%26issue%3D6%26spage%3D990%26epage%3D995%26doi%3D10.1124%2Fdmd.113.055095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zur, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keshari, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostrem, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Younger, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurhanewicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashrafi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">OCT1 Is a High-Capacity Thiamine Transporter That Regulates Hepatic Steatosis and Is a Target of Metformin</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x"> (</span><span class="NLM_issue">27</span><span class="NLM_x">) </span> <span class="NLM_fpage">9983</span><span class="NLM_x">–</span> <span class="NLM_lpage">9988</span><span class="refDoi"> DOI: 10.1073/pnas.1314939111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1073%2Fpnas.1314939111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=24961373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVamu7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=9983-9988&issue=27&author=L.+Chenauthor=Y.+Shuauthor=X.+Liangauthor=E.+C.+Chenauthor=S.+W.+Yeeauthor=A.+A.+Zurauthor=S.+Liauthor=L.+Xuauthor=K.+R.+Keshariauthor=M.+J.+Linauthor=H.-C.+Chienauthor=Y.+Zhangauthor=K.+M.+Morrisseyauthor=J.+Liuauthor=J.+Ostremauthor=N.+S.+Youngerauthor=J.+Kurhanewiczauthor=K.+M.+Shokatauthor=K.+Ashrafiauthor=K.+M.+Giacomini&title=OCT1+Is+a+High-Capacity+Thiamine+Transporter+That+Regulates+Hepatic+Steatosis+and+Is+a+Target+of+Metformin&doi=10.1073%2Fpnas.1314939111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">OCT1 is a high-capacity thiamine transporter that regulates hepatic steatosis and is a target of metformin</span></div><div class="casAuthors">Chen, Ligong; Shu, Yan; Liang, Xiaomin; Chen, Eugene C.; Yee, Sook Wah; Zur, Arik A.; Li, Shuanglian; Xu, Lu; Keshari, Kayvan R.; Lin, Michael J.; Chien, Huan-Chieh; Zhang, Youcai; Morrissey, Kari M.; Liu, Jason; Ostrem, Jonathan; Younger, Noah S.; Kurhanewicz, John; Shokat, Kevan M.; Ashrafi, Kaveh; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">9983-9988</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Org. cation transporter 1, OCT1 (SLC22A1), is the major hepatic uptake transporter for metformin, the most prescribed antidiabetic drug.  However, its endogenous role is poorly understood.  Here we show that similar to metformin treatment, loss of Oct1 caused an increase in the ratio of AMP to ATP, activated the energy sensor AMP-activated kinase (AMPK), and substantially reduced triglyceride (TG) levels in livers from healthy and leptin-deficient mice.  Conversely, livers of human OCT1 transgenic mice fed high-fat diets were enlarged with high TG levels.  Metabolomic and isotopic uptake methods identified thiamine as a principal endogenous substrate of OCT1.  Thiamine deficiency enhanced the phosphorylation of AMPK and its downstream target, acetyl-CoA carboxylase.  Metformin and the biguanide analog, phenformin, competitively inhibited OCT1-mediated thiamine uptake.  Acute administration of metformin to wild-type mice reduced intestinal accumulation of thiamine.  These findings suggest that OCT1 plays a role in hepatic steatosis through modulation of energy status.  The studies implicate OCT1 as well as metformin in thiamine disposition, suggesting an intriguing and parallel mechanism for metformin and its major hepatic transporter in metabolic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsCRGBQOEPerVg90H21EOLACvtfcHk0liftqzS01mPuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVamu7nI&md5=660cd2ac8d797fd445f5997d48aa71f2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1314939111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1314939111%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DE.%2BC.%26aulast%3DYee%26aufirst%3DS.%2BW.%26aulast%3DZur%26aufirst%3DA.%2BA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DKeshari%26aufirst%3DK.%2BR.%26aulast%3DLin%26aufirst%3DM.%2BJ.%26aulast%3DChien%26aufirst%3DH.-C.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMorrissey%26aufirst%3DK.%2BM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DOstrem%26aufirst%3DJ.%26aulast%3DYounger%26aufirst%3DN.%2BS.%26aulast%3DKurhanewicz%26aufirst%3DJ.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DAshrafi%26aufirst%3DK.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DOCT1%2520Is%2520a%2520High-Capacity%2520Thiamine%2520Transporter%2520That%2520Regulates%2520Hepatic%2520Steatosis%2520and%2520Is%2520a%2520Target%2520of%2520Metformin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26issue%3D27%26spage%3D9983%26epage%3D9988%26doi%3D10.1073%2Fpnas.1314939111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kell, D. B.</span><span> </span><span class="NLM_article-title">Implications of Endogenous Roles of Transporters for Drug Discovery: Hitchhiking and Metabolite-Likeness</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">143</span><span class="refDoi"> DOI: 10.1038/nrd.2015.44</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnrd.2015.44" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=143&issue=2&author=D.+B.+Kell&title=Implications+of+Endogenous+Roles+of+Transporters+for+Drug+Discovery%3A+Hitchhiking+and+Metabolite-Likeness&doi=10.1038%2Fnrd.2015.44"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.44&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.44%26sid%3Dliteratum%253Aachs%26aulast%3DKell%26aufirst%3DD.%2BB.%26atitle%3DImplications%2520of%2520Endogenous%2520Roles%2520of%2520Transporters%2520for%2520Drug%2520Discovery%253A%2520Hitchhiking%2520and%2520Metabolite-Likeness%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26issue%3D2%26spage%3D143%26doi%3D10.1038%2Fnrd.2015.44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Shu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheardown, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ianculescu, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burchard, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brett, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Effect of Genetic Variation in the Organic Cation Transporter 1 (OCT1) on Metformin Action</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1422</span><span class="NLM_x">–</span> <span class="NLM_lpage">1431</span><span class="refDoi"> DOI: 10.1172/JCI30558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1172%2FJCI30558" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2007&pages=1422-1431&issue=5&author=Y.+Shuauthor=S.+A.+Sheardownauthor=C.+Brownauthor=R.+P.+Owenauthor=S.+Zhangauthor=R.+A.+Castroauthor=A.+G.+Ianculescuauthor=L.+Yueauthor=J.+C.+Loauthor=E.+G.+Burchardauthor=C.+M.+Brettauthor=K.+M.+Giacomini&title=Effect+of+Genetic+Variation+in+the+Organic+Cation+Transporter+1+%28OCT1%29+on+Metformin+Action&doi=10.1172%2FJCI30558"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1172%2FJCI30558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI30558%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DY.%26aulast%3DSheardown%26aufirst%3DS.%2BA.%26aulast%3DBrown%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DR.%2BP.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DCastro%26aufirst%3DR.%2BA.%26aulast%3DIanculescu%26aufirst%3DA.%2BG.%26aulast%3DYue%26aufirst%3DL.%26aulast%3DLo%26aufirst%3DJ.%2BC.%26aulast%3DBurchard%26aufirst%3DE.%2BG.%26aulast%3DBrett%26aufirst%3DC.%2BM.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DEffect%2520of%2520Genetic%2520Variation%2520in%2520the%2520Organic%2520Cation%2520Transporter%25201%2520%2528OCT1%2529%2520on%2520Metformin%2520Action%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2007%26volume%3D117%26issue%3D5%26spage%3D1422%26epage%3D1431%26doi%3D10.1172%2FJCI30558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Tzvetkov, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saadatmand, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lötsch, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegeder, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stingl, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brockmöller, J.</span><span> </span><span class="NLM_article-title">Genetically Polymorphic OCT1: Another Piece in the Puzzle of the Variable Pharmacokinetics and Pharmacodynamics of the Opioidergic Drug Tramadol</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span><span class="refDoi"> DOI: 10.1038/clpt.2011.56</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fclpt.2011.56" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21562485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVaktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2011&pages=143-150&issue=1&author=M.+V.+Tzvetkovauthor=A.+R.+Saadatmandauthor=J.+L%C3%B6tschauthor=I.+Tegederauthor=J.+C.+Stinglauthor=J.+Brockm%C3%B6ller&title=Genetically+Polymorphic+OCT1%3A+Another+Piece+in+the+Puzzle+of+the+Variable+Pharmacokinetics+and+Pharmacodynamics+of+the+Opioidergic+Drug+Tramadol&doi=10.1038%2Fclpt.2011.56"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Genetically Polymorphic OCT1: Another Piece in the Puzzle of the Variable Pharmacokinetics and Pharmacodynamics of the Opioidergic Drug Tramadol</span></div><div class="casAuthors">Tzvetkov, M. V.; Saadatmand, A. R.; Loetsch, J.; Tegeder, I.; Stingl, J. C.; Brockmoeller, J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">143-150</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We investigated whether tramadol or its active metabolite, O-desmethyltramadol, are substrates of the org. cation transporter OCT1 and whether polymorphisms in OCT1 affect tramadol and O-desmethyltramadol pharmacokinetics.  Tramadol showed high permeability through parallel artificial membrane permeability assays (PAMPAs).  Tramadol uptake in HEK293 cells did not change after OCT1 overexpression, and the concns. of tramadol in the plasma of healthy volunteers were independent of their OCT1 genotypes.  In contrast, O-desmethyltramadol showed low membrane permeability, and OCT1 overexpression increased O-desmethyltramadol uptake 2.4-fold.  This increase in uptake was reversed by OCT1 inhibitors and absent when loss-of-function OCT1 variants were overexpressed.  Volunteers carrying loss-of-function OCT1 polymorphisms had significantly higher plasma concns. of O-desmethyltramadol (P = 0.002, n = 41) and significantly prolonged miosis, a surrogate marker of opioidergic effects (P = 0.005, n = 24).  In conclusion, polymorphisms in OCT1 influence the pharmacokinetics of O-desmethyltramadol, presumably by affecting its uptake into liver cells, and thus may modulate the efficacy of tramadol treatment.  Clin. Pharmacol. & Therapeutics (2011) 90 1, 143-150. doi:10.1038/clpt.2011.56.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUQAyYl097DrVg90H21EOLACvtfcHk0liftqzS01mPuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVaktrk%253D&md5=55027b4c5ff59392d666d033baadf89b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2011.56&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2011.56%26sid%3Dliteratum%253Aachs%26aulast%3DTzvetkov%26aufirst%3DM.%2BV.%26aulast%3DSaadatmand%26aufirst%3DA.%2BR.%26aulast%3DL%25C3%25B6tsch%26aufirst%3DJ.%26aulast%3DTegeder%26aufirst%3DI.%26aulast%3DStingl%26aufirst%3DJ.%2BC.%26aulast%3DBrockm%25C3%25B6ller%26aufirst%3DJ.%26atitle%3DGenetically%2520Polymorphic%2520OCT1%253A%2520Another%2520Piece%2520in%2520the%2520Puzzle%2520of%2520the%2520Variable%2520Pharmacokinetics%2520and%2520Pharmacodynamics%2520of%2520the%2520Opioidergic%2520Drug%2520Tramadol%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2011%26volume%3D90%26issue%3D1%26spage%3D143%26epage%3D150%26doi%3D10.1038%2Fclpt.2011.56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Cho, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J.-Y.</span><span> </span><span class="NLM_article-title">Verapamil Decreases the Glucose-Lowering Effect of Metformin in Healthy Volunteers</span> <span class="citation_source-journal">Br. J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1426</span><span class="NLM_x">–</span> <span class="NLM_lpage">1432</span><span class="refDoi"> DOI: 10.1111/bcp.12476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1111%2Fbcp.12476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=25060604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCju7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2014&pages=1426-1432&issue=6&author=S.+K.+Choauthor=C.+O.+Kimauthor=E.+S.+Parkauthor=J.-Y.+Chung&title=Verapamil+Decreases+the+Glucose-Lowering+Effect+of+Metformin+in+Healthy+Volunteers&doi=10.1111%2Fbcp.12476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Verapamil decreases the glucose-lowering effect of metformin in healthy volunteers</span></div><div class="casAuthors">Cho, Sung Kweon; Kim, Choon Ok; Park, Eun Seok; Chung, Jae-Yong</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1426-1432</span>CODEN:
                <span class="NLM_cas:coden">BCPHBM</span>;
        ISSN:<span class="NLM_cas:issn">1365-2125</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Aim : The org. cation transporter 1 (OCT1) plays a key role in the cellular transport of metformin and its subsequent glucose-lowering effect.  A recent non-clin. study reported that metformin uptake into hepatocytes is regulated via OCT1, and that uptake was strongly inhibited by verapamil.  Therefore, we investigated the effects of verapamil co-administration on the pharmacokinetics and pharmacodynamics of metformin in humans.  Methods : We evaluated the pharmacokinetics and the anti-hyperglycemic effects of metformin using an oral glucose tolerance test (OGTT) in 12 healthy participants, before (day 1) and after metformin treatment (day 2), and again on days 15 and 16 after co-administration with verapamil.  Results : Verapamil inhibited the ability of metformin to reduce max. blood glucose concns. (ΔGmax) by 62.5% (P = 0.008) and decreased the area under the glucose concn.-time curve (ΔAUCgluc) by 238% (P = 0.015).  However, verapamil did not significantly alter the Cmax and the AUC of metformin, nor its renal clearance.  Conclusions : Our results suggest that verapamil remarkably decreases the glucose-lowering effect of metformin, possibly by acting as a competitive inhibitor of OCT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCHQT4gxaQG7Vg90H21EOLACvtfcHk0liSsaFYxgJVgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCju7%252FO&md5=20cde8d523cb19a37f0113b570534024</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fbcp.12476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcp.12476%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DS.%2BK.%26aulast%3DKim%26aufirst%3DC.%2BO.%26aulast%3DPark%26aufirst%3DE.%2BS.%26aulast%3DChung%26aufirst%3DJ.-Y.%26atitle%3DVerapamil%2520Decreases%2520the%2520Glucose-Lowering%2520Effect%2520of%2520Metformin%2520in%2520Healthy%2520Volunteers%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2014%26volume%3D78%26issue%3D6%26spage%3D1426%26epage%3D1432%26doi%3D10.1111%2Fbcp.12476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tweedie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benet, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brouwer, K. L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillgren, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmaster, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keppler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niemi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ware, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamek-Gliszczynski, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span> </span><span class="NLM_article-title">Membrane Transporters in Drug Development</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1038/nrd3028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnrd3028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=20190787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFelu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=215-236&issue=3&author=K.+M.+Giacominiauthor=S.-M.+Huangauthor=D.+J.+Tweedieauthor=L.+Z.+Benetauthor=K.+L.+R.+Brouwerauthor=X.+Chuauthor=A.+Dahlinauthor=R.+Eversauthor=V.+Fischerauthor=K.+M.+Hillgrenauthor=K.+A.+Hoffmasterauthor=T.+Ishikawaauthor=D.+Kepplerauthor=R.+B.+Kimauthor=C.+A.+Leeauthor=M.+Niemiauthor=J.+W.+Polliauthor=Y.+Sugiyamaauthor=P.+W.+Swaanauthor=J.+A.+Wareauthor=S.+H.+Wrightauthor=S.+W.+Yeeauthor=M.+J.+Zamek-Gliszczynskiauthor=L.+Zhang&title=Membrane+Transporters+in+Drug+Development&doi=10.1038%2Fnrd3028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane transporters in drug development</span></div><div class="casAuthors">Giacomini, Kathleen M.; Huang, Shiew-Mei; Tweedie, Donald J.; Benet, Leslie Z.; Brouwer, Kim L. R.; Chu, Xiaoyan; Dahlin, Amber; Evers, Raymond; Fischer, Volker; Hillgren, Kathleen M.; Hoffmaster, Keith A.; Ishikawa, Toshihisa; Keppler, Dietrich; Kim, Richard B.; Lee, Caroline A.; Niemi, Mikko; Polli, Joseph W.; Sugiyama, Yuicchi; Swaan, Peter W.; Ware, Joseph A.; Wright, Stephen H.; Wah Yee, Sook; Zamek-Gliszczynski, Maciej J.; Zhang, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-236</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Membrane transporters can be major determinants of the pharmacokinetic, safety and efficacy profiles of drugs.  This presents several key questions for drug development, including which transporters are clin. important in drug absorption and disposition, and which in vitro methods are suitable for studying drug interactions with these transporters.  In addn., what criteria should trigger follow-up clin. studies, and which clin. studies should be conducted if needed.  In this article, we provide the recommendations of the International Transporter Consortium on these issues, and present decision trees that are intended to help guide clin. studies on the currently recognized most important drug transporter interactions.  The recommendations are generally intended to support clin. development and filing of a new drug application.  Overall, it is advised that the timing of transporter investigations should be driven by efficacy, safety and clin. trial enrollment questions (for example, exclusion and inclusion criteria), as well as a need for further understanding of the absorption, distribution, metab. and excretion properties of the drug mol., and information required for drug labeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiBIf0gk93W7Vg90H21EOLACvtfcHk0liSsaFYxgJVgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFelu74%253D&md5=89d9ce746284c1814fb01cd9b9a72027</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd3028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3028%26sid%3Dliteratum%253Aachs%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DHuang%26aufirst%3DS.-M.%26aulast%3DTweedie%26aufirst%3DD.%2BJ.%26aulast%3DBenet%26aufirst%3DL.%2BZ.%26aulast%3DBrouwer%26aufirst%3DK.%2BL.%2BR.%26aulast%3DChu%26aufirst%3DX.%26aulast%3DDahlin%26aufirst%3DA.%26aulast%3DEvers%26aufirst%3DR.%26aulast%3DFischer%26aufirst%3DV.%26aulast%3DHillgren%26aufirst%3DK.%2BM.%26aulast%3DHoffmaster%26aufirst%3DK.%2BA.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DKeppler%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DR.%2BB.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DNiemi%26aufirst%3DM.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DSugiyama%26aufirst%3DY.%26aulast%3DSwaan%26aufirst%3DP.%2BW.%26aulast%3DWare%26aufirst%3DJ.%2BA.%26aulast%3DWright%26aufirst%3DS.%2BH.%26aulast%3DYee%26aufirst%3DS.%2BW.%26aulast%3DZamek-Gliszczynski%26aufirst%3DM.%2BJ.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DMembrane%2520Transporters%2520in%2520Drug%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26issue%3D3%26spage%3D215%26epage%3D236%26doi%3D10.1038%2Fnrd3028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span> <span class="citation_source-book">Guideline on the Investigation of Drug Interactions</span>; <span class="NLM_publisher-name">European Medicines Agency</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Guideline+on+the+Investigation+of+Drug+Interactions%3B+European+Medicines+Agency%3A+London%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGuideline%2520on%2520the%2520Investigation%2520of%2520Drug%2520Interactions%26pub%3DEuropean%2520Medicines%2520Agency%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Haenisch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drescher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiemer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giros, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gautron, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bönisch, H.</span><span> </span><span class="NLM_article-title">Interaction of Antidepressant and Antipsychotic Drugs with the Human Organic Cation Transporters hOCT1, hOCT2 and hOCT3</span> <span class="citation_source-journal">Naunyn-Schmiedeberg's Arch. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">385</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1007/s00210-012-0781-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1007%2Fs00210-012-0781-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=22806583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSgsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=385&publication_year=2012&pages=1017-1023&issue=10&author=B.+Haenischauthor=E.+Drescherauthor=L.+Thiemerauthor=H.+Xinauthor=B.+Girosauthor=S.+Gautronauthor=H.+B%C3%B6nisch&title=Interaction+of+Antidepressant+and+Antipsychotic+Drugs+with+the+Human+Organic+Cation+Transporters+hOCT1%2C+hOCT2+and+hOCT3&doi=10.1007%2Fs00210-012-0781-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of antidepressant and antipsychotic drugs with the human organic cation transporters hOCT1, hOCT2 and hOCT3</span></div><div class="casAuthors">Haenisch, Britta; Drescher, Eva; Thiemer, Lidia; Xin, Hu; Giros, Bruno; Gautron, Sophie; Boenisch, Heinz</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">385</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1017-1023</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Besides the three antidepressant-sensitive, Na+- and Cl--dependent monoamine transporters, Na+-independent org. cation transporters (OCTs) are known to transport monoamines.  However, little is known about the interactions of psychotropic drugs with human (h) OCTs.  In the present study, a series of diverse antidepressant and antipsychotic drugs were examd. for their inhibitory potency at hOCT1, hOCT2 and hOCT3 by measuring inhibition of [3H]-MPP+ uptake into HEK293 cells stably expressing one of the three hOCTs.  The inhibitory potencies (IC50s) ranged from 1 to 900 μM.  Most of the examd. drugs showed highest inhibitory potency at hOCT1 which is very sparsely expressed in the brain and mainly involved in renal and hepatic clearance of cationic drugs.  At their upper therapeutic plasma concns., several drugs are expected to inhibit by more than 20 % hOCT1 and could thus interfere with the pharmacokinetics of hOCT1-transported drugs in the kidney and liver, namely trimipramine, desipramine and fluoxetine (by about 37 %), levomepromazine and nefazodone (by about 32 %), and clozapine and amitriptyline (by about 22 %).  At hOCT2 and hOCT3, which are involved in monoamine homeostasis in the brain, IC50s of most psychoactive drugs were in the high micromolar range.  At their upper plasma concns., only three compds., bupropion, nefazodone and clozapine, showed potential for inhibition, of about 18 % at hOCT2 (bupropion), about 22 % at hOCT3 (nefazodone) and of approx. 10 % at hOCT2 and hOCT3 (clozapine).  Thus, under the assumption of a tenfold accumulation in the brain, bupropion, nefazodone and clozapine may notably inhibit the corresponding hOCTs.  It remains to be shown whether such a direct inhibition plays a role in the clin. effects of these three drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQuUHCihnvR7Vg90H21EOLACvtfcHk0liXGEyVco6evQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSgsLzL&md5=21d138b3871c65e49d781ac606504187</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs00210-012-0781-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-012-0781-8%26sid%3Dliteratum%253Aachs%26aulast%3DHaenisch%26aufirst%3DB.%26aulast%3DDrescher%26aufirst%3DE.%26aulast%3DThiemer%26aufirst%3DL.%26aulast%3DXin%26aufirst%3DH.%26aulast%3DGiros%26aufirst%3DB.%26aulast%3DGautron%26aufirst%3DS.%26aulast%3DB%25C3%25B6nisch%26aufirst%3DH.%26atitle%3DInteraction%2520of%2520Antidepressant%2520and%2520Antipsychotic%2520Drugs%2520with%2520the%2520Human%2520Organic%2520Cation%2520Transporters%2520hOCT1%252C%2520hOCT2%2520and%2520hOCT3%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2012%26volume%3D385%26issue%3D10%26spage%3D1017%26epage%3D1023%26doi%3D10.1007%2Fs00210-012-0781-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Ahlin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustavsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norinder, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergström, C. A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Structural Requirements for Drug Inhibition of the Liver Specific Human Organic Cation Transport Protein 1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">5932</span><span class="NLM_x">–</span> <span class="NLM_lpage">5942</span><span class="refDoi"> DOI: 10.1021/jm8003152</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8003152" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5932-5942&issue=19&author=G.+Ahlinauthor=J.+Karlssonauthor=J.+M.+Pedersenauthor=L.+Gustavssonauthor=R.+Larssonauthor=P.+Matssonauthor=U.+Norinderauthor=C.+A.+S.+Bergstr%C3%B6mauthor=P.+Artursson&title=Structural+Requirements+for+Drug+Inhibition+of+the+Liver+Specific+Human+Organic+Cation+Transport+Protein+1&doi=10.1021%2Fjm8003152"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm8003152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8003152%26sid%3Dliteratum%253Aachs%26aulast%3DAhlin%26aufirst%3DG.%26aulast%3DKarlsson%26aufirst%3DJ.%26aulast%3DPedersen%26aufirst%3DJ.%2BM.%26aulast%3DGustavsson%26aufirst%3DL.%26aulast%3DLarsson%26aufirst%3DR.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DNorinder%26aufirst%3DU.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DC.%2BA.%2BS.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DStructural%2520Requirements%2520for%2520Drug%2520Inhibition%2520of%2520the%2520Liver%2520Specific%2520Human%2520Organic%2520Cation%2520Transport%2520Protein%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D19%26spage%3D5932%26epage%3D5942%26doi%3D10.1021%2Fjm8003152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Belzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morales, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagadish, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mash, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Substrate-Dependent Ligand Inhibition of the Human Organic Cation Transporter OCT2</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">300</span><span class="NLM_x">–</span> <span class="NLM_lpage">310</span><span class="refDoi"> DOI: 10.1124/jpet.113.203257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.113.203257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23709117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOjsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2013&pages=300-310&issue=2&author=M.+Belzerauthor=M.+Moralesauthor=B.+Jagadishauthor=E.+A.+Mashauthor=S.+H.+Wright&title=Substrate-Dependent+Ligand+Inhibition+of+the+Human+Organic+Cation+Transporter+OCT2&doi=10.1124%2Fjpet.113.203257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-dependent ligand inhibition of the human organic cation transporter OCT2</span></div><div class="casAuthors">Belzer, Mathew; Morales, Mark; Jagadish, Bhumasamudram; Mash, Eugene A.; Wright, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-310</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Org. cation transporter 2 (OCT2) mediates the initial step in renal secretion of org. cations: uptake from the blood, across the basolateral membrane, and into the renal proximal tubule cells.  Because of its potential as a target for unwanted drug-drug interactions (DDIs), considerable attention has been directed toward understanding the basis of OCT2 selectivity.  These studies typically assess selectivity based on ligand inhibition profiles for OCT2-mediated transport of a probe substrate.  However, little attention has been given to the potential influence of the substrate on the profile of ligand inhibition.  Here we compared the IC50 values obtained for a set of structurally distinct inhibitors against OCT2-mediated transport of three structurally distinct substrates: 1-methyl-4-phenylpyridinium (MPP); metformin; and a novel fluorescent substrate, N,N,N-trimethyl-2-[methyl(7-nitrobenzo[c][l,2,5]oxadiazol-4-yl)amino]ethanaminium iodide (NBD-MTMA).  The median IC50 value for inhibition of MPP transport was 9-fold higher than that for inhibition of metformin transport.  Similarly, the median IC50 value for inhibition of MPP transport was 5-fold higher than that for NBD-MTMA transport.  However, this was not a systematic difference in inhibitory efficacy; the ratio of IC50 values, MPP vs. NBD-MTMA, ranged from 88-fold (ipratropium) to 0.3-fold (metformin).  These data show that 1) the choice of OCT2 substrate significantly influences both quant. and qual. inhibitory interactions with cationic drugs; and 2) ligand interactions with OCT2 are not restricted to competition for a common ligand binding site, consistent with a binding surface characterized by multiple, possibly overlapping interaction sites.  Development of predictive models of DDIs with OCT2 must take into account the substrate dependence of ligand interaction with this protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfkgfWyk5fVLVg90H21EOLACvtfcHk0liXGEyVco6evQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOjsbfK&md5=d640680933397fa257f3896395fd7fc1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1124%2Fjpet.113.203257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.113.203257%26sid%3Dliteratum%253Aachs%26aulast%3DBelzer%26aufirst%3DM.%26aulast%3DMorales%26aufirst%3DM.%26aulast%3DJagadish%26aufirst%3DB.%26aulast%3DMash%26aufirst%3DE.%2BA.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DSubstrate-Dependent%2520Ligand%2520Inhibition%2520of%2520the%2520Human%2520Organic%2520Cation%2520Transporter%2520OCT2%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2013%26volume%3D346%26issue%3D2%26spage%3D300%26epage%3D310%26doi%3D10.1124%2Fjpet.113.203257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Polyspecific Organic Cation Transporters: Their Functions and Interactions with Drugs</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">381</span><span class="refDoi"> DOI: 10.1016/j.tips.2004.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1016%2Fj.tips.2004.05.005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=375-381&issue=7&author=H.+Koepsell&title=Polyspecific+Organic+Cation+Transporters%3A+Their+Functions+and+Interactions+with+Drugs&doi=10.1016%2Fj.tips.2004.05.005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2004.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2004.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DPolyspecific%2520Organic%2520Cation%2520Transporters%253A%2520Their%2520Functions%2520and%2520Interactions%2520with%2520Drugs%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2004%26volume%3D25%26issue%3D7%26spage%3D375%26epage%3D381%26doi%3D10.1016%2Fj.tips.2004.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirahatti, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahadevan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">A Conserved Glutamate Residue in Transmembrane Helix 10 Influences Substrate Specificity of Rabbit OCT2 (SLC22A2)</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x"> (</span><span class="NLM_issue">41</span><span class="NLM_x">) </span> <span class="NLM_fpage">34813</span><span class="NLM_x">–</span> <span class="NLM_lpage">34822</span><span class="refDoi"> DOI: 10.1074/jbc.M506342200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1074%2Fjbc.M506342200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=34813-34822&issue=41&author=X.+Zhangauthor=N.+V.+Shirahattiauthor=D.+Mahadevanauthor=S.+H.+Wright&title=A+Conserved+Glutamate+Residue+in+Transmembrane+Helix+10+Influences+Substrate+Specificity+of+Rabbit+OCT2+%28SLC22A2%29&doi=10.1074%2Fjbc.M506342200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M506342200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M506342200%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DShirahatti%26aufirst%3DN.%2BV.%26aulast%3DMahadevan%26aufirst%3DD.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DA%2520Conserved%2520Glutamate%2520Residue%2520in%2520Transmembrane%2520Helix%252010%2520Influences%2520Substrate%2520Specificity%2520of%2520Rabbit%2520OCT2%2520%2528SLC22A2%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26issue%3D41%26spage%3D34813%26epage%3D34822%26doi%3D10.1074%2Fjbc.M506342200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Substrate Recognition and Translocation by Polyspecific Organic Cation Transporters</span> <span class="citation_source-journal">Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">392</span><span class="NLM_x"> (</span><span class="NLM_issue">1–2</span><span class="NLM_x">) </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">101</span><span class="refDoi"> DOI: 10.1515/bc.2011.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1515%2Fbc.2011.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2011&pages=95-101&issue=1%E2%80%932&author=H.+Koepsell&title=Substrate+Recognition+and+Translocation+by+Polyspecific+Organic+Cation+Transporters&doi=10.1515%2Fbc.2011.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1515%2Fbc.2011.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fbc.2011.009%26sid%3Dliteratum%253Aachs%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DSubstrate%2520Recognition%2520and%2520Translocation%2520by%2520Polyspecific%2520Organic%2520Cation%2520Transporters%26jtitle%3DBiol.%2520Chem.%26date%3D2011%26volume%3D392%26issue%3D1%25E2%2580%25932%26spage%3D95%26epage%3D101%26doi%3D10.1515%2Fbc.2011.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatskaya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Subtype-Specific Affinity for Corticosterone of Rat Organic Cation Transporters rOCT1 and rOCT2 Depends on Three Amino Acids Within the Substrate Binding Region</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1612</span><span class="NLM_x">–</span> <span class="NLM_lpage">1619</span><span class="refDoi"> DOI: 10.1124/mol.104.008821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.104.008821" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2005&pages=1612-1619&issue=5&author=V.+Gorboulevauthor=N.+Shatskayaauthor=C.+Volkauthor=H.+Koepsell&title=Subtype-Specific+Affinity+for+Corticosterone+of+Rat+Organic+Cation+Transporters+rOCT1+and+rOCT2+Depends+on+Three+Amino+Acids+Within+the+Substrate+Binding+Region&doi=10.1124%2Fmol.104.008821"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1124%2Fmol.104.008821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.104.008821%26sid%3Dliteratum%253Aachs%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DShatskaya%26aufirst%3DN.%26aulast%3DVolk%26aufirst%3DC.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DSubtype-Specific%2520Affinity%2520for%2520Corticosterone%2520of%2520Rat%2520Organic%2520Cation%2520Transporters%2520rOCT1%2520and%2520rOCT2%2520Depends%2520on%2520Three%2520Amino%2520Acids%2520Within%2520the%2520Substrate%2520Binding%2520Region%26jtitle%3DMol.%2520Pharmacol.%26date%3D2005%26volume%3D67%26issue%3D5%26spage%3D1612%26epage%3D1619%26doi%3D10.1124%2Fmol.104.008821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Popp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorbunov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatskaya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Amino Acids Critical for Substrate Affinity of Rat Organic Cation Transporter 1 Line the Substrate Binding Region in a Model Derived From the Tertiary Structure of Lactose Permease</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1600</span><span class="NLM_x">–</span> <span class="NLM_lpage">1611</span><span class="refDoi"> DOI: 10.1124/mol.104.008839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.104.008839" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2005&pages=1600-1611&issue=5&author=C.+Poppauthor=V.+Gorboulevauthor=T.+D.+M%C3%BCllerauthor=D.+Gorbunovauthor=N.+Shatskayaauthor=H.+Koepsell&title=Amino+Acids+Critical+for+Substrate+Affinity+of+Rat+Organic+Cation+Transporter+1+Line+the+Substrate+Binding+Region+in+a+Model+Derived+From+the+Tertiary+Structure+of+Lactose+Permease&doi=10.1124%2Fmol.104.008839"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1124%2Fmol.104.008839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.104.008839%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DC.%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DM%25C3%25BCller%26aufirst%3DT.%2BD.%26aulast%3DGorbunov%26aufirst%3DD.%26aulast%3DShatskaya%26aufirst%3DN.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DAmino%2520Acids%2520Critical%2520for%2520Substrate%2520Affinity%2520of%2520Rat%2520Organic%2520Cation%2520Transporter%25201%2520Line%2520the%2520Substrate%2520Binding%2520Region%2520in%2520a%2520Model%2520Derived%2520From%2520the%2520Tertiary%2520Structure%2520of%2520Lactose%2520Permease%26jtitle%3DMol.%2520Pharmacol.%26date%3D2005%26volume%3D67%26issue%3D5%26spage%3D1600%26epage%3D1611%26doi%3D10.1124%2Fmol.104.008839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Volk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotzsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Five Amino Acids in the Innermost Cavity of the Substrate Binding Cleft of Organic Cation Transporter 1 Interact with Extracellular and Intracellular Corticosterone</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">289</span><span class="refDoi"> DOI: 10.1124/mol.109.054783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.109.054783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=275-289&issue=2&author=C.+Volkauthor=V.+Gorboulevauthor=A.+Kotzschauthor=T.+D.+M%C3%BCllerauthor=H.+Koepsell&title=Five+Amino+Acids+in+the+Innermost+Cavity+of+the+Substrate+Binding+Cleft+of+Organic+Cation+Transporter+1+Interact+with+Extracellular+and+Intracellular+Corticosterone&doi=10.1124%2Fmol.109.054783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.054783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.054783%26sid%3Dliteratum%253Aachs%26aulast%3DVolk%26aufirst%3DC.%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DKotzsch%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DT.%2BD.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DFive%2520Amino%2520Acids%2520in%2520the%2520Innermost%2520Cavity%2520of%2520the%2520Substrate%2520Binding%2520Cleft%2520of%2520Organic%2520Cation%2520Transporter%25201%2520Interact%2520with%2520Extracellular%2520and%2520Intracellular%2520Corticosterone%26jtitle%3DMol.%2520Pharmacol.%26date%3D2009%26volume%3D76%26issue%3D2%26spage%3D275%26epage%3D289%26doi%3D10.1124%2Fmol.109.054783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Pedersen, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waight, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risenmay, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roe-Zurz, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harries, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johri, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">Crystal Structure of a Eukaryotic Phosphate Transporter</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">496</span><span class="NLM_x"> (</span><span class="NLM_issue">7446</span><span class="NLM_x">) </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">536</span><span class="refDoi"> DOI: 10.1038/nature12042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnature12042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=533-536&issue=7446&author=B.+P.+Pedersenauthor=H.+Kumarauthor=A.+B.+Waightauthor=A.+J.+Risenmayauthor=Z.+Roe-Zurzauthor=B.+H.+Chauauthor=A.+Schlessingerauthor=M.+Bonomiauthor=W.+Harriesauthor=A.+Saliauthor=A.+K.+Johriauthor=R.+M.+Stroud&title=Crystal+Structure+of+a+Eukaryotic+Phosphate+Transporter&doi=10.1038%2Fnature12042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature12042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12042%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DB.%2BP.%26aulast%3DKumar%26aufirst%3DH.%26aulast%3DWaight%26aufirst%3DA.%2BB.%26aulast%3DRisenmay%26aufirst%3DA.%2BJ.%26aulast%3DRoe-Zurz%26aufirst%3DZ.%26aulast%3DChau%26aufirst%3DB.%2BH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DHarries%26aufirst%3DW.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DJohri%26aufirst%3DA.%2BK.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Eukaryotic%2520Phosphate%2520Transporter%26jtitle%3DNature%26date%3D2013%26volume%3D496%26issue%3D7446%26spage%3D533%26epage%3D536%26doi%3D10.1038%2Fnature12042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Shen, M.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Statistical Potential for Assessment and Prediction of Protein Structures</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2507</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span><span class="refDoi"> DOI: 10.1110/ps.062416606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1110%2Fps.062416606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=17075131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=2507-2524&issue=11&author=M.-Y.+Shenauthor=A.+Sali&title=Statistical+Potential+for+Assessment+and+Prediction+of+Protein+Structures&doi=10.1110%2Fps.062416606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical potential for assessment and prediction of protein structures</span></div><div class="casAuthors">Shen, Min-Yi; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2507-2524</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Protein structures in the Protein Data Bank provide a wealth of data about the interactions that det. the native states of proteins.  Using the probability theory, the authors derive an at. distance-dependent statistical potential from a sample of native structures that does not depend on any adjustable parameters (Discrete Optimized Protein Energy, or DOPE).  DOPE is based on an improved ref. state that corresponds to noninteracting atoms in a homogeneous sphere with the radius dependent on a sample native structure; it thus accounts for the finite and spherical shape of the native structures.  The DOPE potential was extd. from a nonredundant set of 1472 crystallog. structures.  The authors tested DOPE and five other scoring functions by the detection of the native state among six multiple target decoy sets, the correlation between the score and model error, and the identification of the most accurate nonnative structure in the decoy set.  For all decoy sets, DOPE is the best performing function in terms of all criteria, except for a tie in one criterion for one decoy set.  To facilitate its use in various applications, such as model assessment, loop modeling, and fitting into cryo-electron microscopy mass d. maps combined with comparative protein structure modeling, DOPE was incorporated into the modeling package MODELLER-8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz7bz4vq-p_7Vg90H21EOLACvtfcHk0lj0FVdJ-H0n5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO&md5=f4e7ef56cd100164e347103a6888ee8e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1110%2Fps.062416606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.062416606%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DM.-Y.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStatistical%2520Potential%2520for%2520Assessment%2520and%2520Prediction%2520of%2520Protein%2520Structures%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26issue%3D11%26spage%3D2507%26epage%3D2524%26doi%3D10.1110%2Fps.062416606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Eramian, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eswar, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, M.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">How Well Can the Accuracy of Comparative Protein Structure Models Be Predicted?</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1881</span><span class="NLM_x">–</span> <span class="NLM_lpage">1893</span><span class="refDoi"> DOI: 10.1110/ps.036061.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1110%2Fps.036061.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=18832340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGlsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2008&pages=1881-1893&issue=11&author=D.+Eramianauthor=N.+Eswarauthor=M.-Y.+Shenauthor=A.+Sali&title=How+Well+Can+the+Accuracy+of+Comparative+Protein+Structure+Models+Be+Predicted%3F&doi=10.1110%2Fps.036061.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">How well can the accuracy of comparative protein structure models be predicted?</span></div><div class="casAuthors">Eramian, David; Eswar, Narayanan; Shen, Min-Yi; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1881-1893</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Comparative structure models are available for two orders of magnitude more protein sequences than are exptl. detd. structures.  These models, however, suffer from two limitations that exptl. detd. structures do not: they frequently contain significant errors, and their accuracy cannot be readily assessed.  We have addressed the latter limitation by developing a protocol optimized specifically for predicting the Cα root-mean-squared deviation (RMSD) and native overlap (NO3.5Å) errors of a model in the absence of its native structure.  In contrast to most traditional assessment scores that merely predict one model is more accurate than others, this approach quantifies the error in an abs. sense, thus helping to det. whether or not the model is suitable for intended applications.  The assessment relies on a model-specific scoring function constructed by a support vector machine.  This regression optimizes the wts. of up to nine features, including various sequence similarity measures and statistical potentials, extd. from a tailored training set of models unique to the model being assessed: if possible, we use similarly sized models with the same fold; otherwise, we use similarly sized models with the same secondary structure compn.  This protocol predicts the RMSD and NO3.5Å errors for a diverse set of 580,317 comparative models of 6174 sequences with correlation coeffs. (r) of 0.84 and 0.86, resp., to the actual errors.  This scoring function achieves the best correlation compared to 13 other tested assessment criteria that achieved correlations ranging from 0.35 to 0.71.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcmHLOHj4_vrVg90H21EOLACvtfcHk0lj0FVdJ-H0n5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlGlsb%252FM&md5=fbd8fbe28b7846a7dc8b6e9e0991343f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1110%2Fps.036061.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.036061.108%26sid%3Dliteratum%253Aachs%26aulast%3DEramian%26aufirst%3DD.%26aulast%3DEswar%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DM.-Y.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DHow%2520Well%2520Can%2520the%2520Accuracy%2520of%2520Comparative%2520Protein%2520Structure%2520Models%2520Be%2520Predicted%253F%26jtitle%3DProtein%2520Sci.%26date%3D2008%26volume%3D17%26issue%3D11%26spage%3D1881%26epage%3D1893%26doi%3D10.1110%2Fps.036061.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Keller, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egenberger, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernhard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uzelac, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorbunov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wirth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koppatz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dötsch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunte, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">The Large Extracellular Loop of Organic Cation Transporter 1 Influences Substrate Affinity and Is Pivotal for Oligomerization</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x"> (</span><span class="NLM_issue">43</span><span class="NLM_x">) </span> <span class="NLM_fpage">37874</span><span class="NLM_x">–</span> <span class="NLM_lpage">37886</span><span class="refDoi"> DOI: 10.1074/jbc.M111.289330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1074%2Fjbc.M111.289330" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=37874-37886&issue=43&author=T.+Kellerauthor=B.+Egenbergerauthor=V.+Gorboulevauthor=F.+Bernhardauthor=Z.+Uzelacauthor=D.+Gorbunovauthor=C.+Wirthauthor=S.+Koppatzauthor=V.+D%C3%B6tschauthor=C.+Hunteauthor=H.+H.+Sitteauthor=H.+Koepsell&title=The+Large+Extracellular+Loop+of+Organic+Cation+Transporter+1+Influences+Substrate+Affinity+and+Is+Pivotal+for+Oligomerization&doi=10.1074%2Fjbc.M111.289330"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.289330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.289330%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DT.%26aulast%3DEgenberger%26aufirst%3DB.%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DBernhard%26aufirst%3DF.%26aulast%3DUzelac%26aufirst%3DZ.%26aulast%3DGorbunov%26aufirst%3DD.%26aulast%3DWirth%26aufirst%3DC.%26aulast%3DKoppatz%26aufirst%3DS.%26aulast%3DD%25C3%25B6tsch%26aufirst%3DV.%26aulast%3DHunte%26aufirst%3DC.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DThe%2520Large%2520Extracellular%2520Loop%2520of%2520Organic%2520Cation%2520Transporter%25201%2520Influences%2520Substrate%2520Affinity%2520and%2520Is%2520Pivotal%2520for%2520Oligomerization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26issue%3D43%26spage%3D37874%26epage%3D37886%26doi%3D10.1074%2Fjbc.M111.289330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittwer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">High Selectivity of the Γ-Aminobutyric Acid Transporter 2 (GAT-2, SLC6A13) Revealed by Structure-Based Approach</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x"> (</span><span class="NLM_issue">45</span><span class="NLM_x">) </span> <span class="NLM_fpage">37745</span><span class="NLM_x">–</span> <span class="NLM_lpage">37756</span><span class="refDoi"> DOI: 10.1074/jbc.M112.388157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1074%2Fjbc.M112.388157" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=37745-37756&issue=45&author=A.+Schlessingerauthor=M.+B.+Wittwerauthor=A.+Dahlinauthor=N.+Khuriauthor=M.+Bonomiauthor=H.+Fanauthor=K.+M.+Giacominiauthor=A.+Sali&title=High+Selectivity+of+the+%CE%93-Aminobutyric+Acid+Transporter+2+%28GAT-2%2C+SLC6A13%29+Revealed+by+Structure-Based+Approach&doi=10.1074%2Fjbc.M112.388157"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.388157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.388157%26sid%3Dliteratum%253Aachs%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26aulast%3DDahlin%26aufirst%3DA.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DHigh%2520Selectivity%2520of%2520the%2520%25CE%2593-Aminobutyric%2520Acid%2520Transporter%25202%2520%2528GAT-2%252C%2520SLC6A13%2529%2520Revealed%2520by%2520Structure-Based%2520Approach%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26issue%3D45%26spage%3D37745%26epage%3D37756%26doi%3D10.1074%2Fjbc.M112.388157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Structure-Based Discovery of Prescription Drugs That Interact with the Norepinephrine Transporter, NET</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x"> (</span><span class="NLM_issue">38</span><span class="NLM_x">) </span> <span class="NLM_fpage">15810</span><span class="NLM_x">–</span> <span class="NLM_lpage">15815</span><span class="refDoi"> DOI: 10.1073/pnas.1106030108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1073%2Fpnas.1106030108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=15810-15815&issue=38&author=A.+Schlessingerauthor=E.+Geierauthor=H.+Fanauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+M.+Giacominiauthor=A.+Sali&title=Structure-Based+Discovery+of+Prescription+Drugs+That+Interact+with+the+Norepinephrine+Transporter%2C+NET&doi=10.1073%2Fpnas.1106030108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1106030108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1106030108%26sid%3Dliteratum%253Aachs%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DGeier%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Prescription%2520Drugs%2520That%2520Interact%2520with%2520the%2520Norepinephrine%2520Transporter%252C%2520NET%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D38%26spage%3D15810%26epage%3D15815%26doi%3D10.1073%2Fpnas.1106030108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Kido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Profiling of a Prescription Drug Library for Potential Renal Drug-Drug Interactions Mediated by the Organic Cation Transporter 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x"> (</span><span class="NLM_issue">13</span><span class="NLM_x">) </span> <span class="NLM_fpage">4548</span><span class="NLM_x">–</span> <span class="NLM_lpage">4558</span><span class="refDoi"> DOI: 10.1021/jm2001629</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2001629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1agsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4548-4558&issue=13&author=Y.+Kidoauthor=P.+Matssonauthor=K.+M.+Giacomini&title=Profiling+of+a+Prescription+Drug+Library+for+Potential+Renal+Drug-Drug+Interactions+Mediated+by+the+Organic+Cation+Transporter+2&doi=10.1021%2Fjm2001629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling of a Prescription Drug Library for Potential Renal Drug-Drug Interactions Mediated by the Organic Cation Transporter 2</span></div><div class="casAuthors">Kido, Yasuto; Matsson, Par; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4548-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-drug interactions (DDIs) are major causes of serious adverse drug reactions.  Most DDIs have a pharmacokinetic basis in which one drug reduces the elimination of a second drug, leading to potentially toxic drug levels.  As a major organ of drug elimination, the kidney represents an important site for DDIs.  Here, we screened a prescription drug library against the renal org. cation transporter OCT2/SLC22A2, which mediates the first step in the renal secretion of many cationic drugs.  Of the 910 compds. screened, 244 inhibited OCT2.  Computational analyses revealed key properties of inhibitors vs. noninhibitors, which included overall mol. charge.  Four of six potential clin. inhibitors were transporter-selective in follow-up screens against addnl. transporters: OCT1/SLC22A1, MATE1/SLC47A1, and MATE2-K/SLC47A2.  Two compds. showed different kinetics of interaction with the common polymorphism OCT2-A270S, suggesting a role of genetics in modulating renal DDIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRPsHJQI_skbVg90H21EOLACvtfcHk0liFDm7ze8O7EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnt1agsL0%253D&md5=907473f7116011468f807466c99f5b97</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm2001629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2001629%26sid%3Dliteratum%253Aachs%26aulast%3DKido%26aufirst%3DY.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DProfiling%2520of%2520a%2520Prescription%2520Drug%2520Library%2520for%2520Potential%2520Renal%2520Drug-Drug%2520Interactions%2520Mediated%2520by%2520the%2520Organic%2520Cation%2520Transporter%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D13%26spage%3D4548%26epage%3D4558%26doi%3D10.1021%2Fjm2001629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Zhang, J.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, T. D. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oldenburg, K. R.</span><span> </span><span class="NLM_article-title">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1177/108705719900400206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=67-73&issue=2&author=J.-H.+Zhangauthor=T.+D.+Y.+Chungauthor=K.+R.+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays&doi=10.1177%2F108705719900400206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2eaQVNGIfewm3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.-H.%26aulast%3DChung%26aufirst%3DT.%2BD.%2BY.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D1999%26volume%3D4%26issue%3D2%26spage%3D67%26epage%3D73%26doi%3D10.1177%2F108705719900400206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand Discovery From a Dopamine D3 Receptor Homology Model and Crystal Structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&issue=11&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+Discovery+From+a+Dopamine+D3+Receptor+Homology+Model+and+Crystal+Structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0liFDm7ze8O7EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520Discovery%2520From%2520a%2520Dopamine%2520D3%2520Receptor%2520Homology%2520Model%2520and%2520Crystal%2520Structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D11%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Minematsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, K.-I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Usui, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamimura, H.</span><span> </span><span class="NLM_article-title">Characterization of Human Organic Cation Transporter 1 (OCT1/SLC22A1)- and OCT2 (SLC22A2)-Mediated Transport of 1-(2-Methoxyethyl)-2-Methyl-4,9-Dioxo-3-(Pyrazin-2-Ylmethyl)- 4,9-Dihydro-1H-Naphtho[2,3-D]Imidazolium Bromide (YM155 Monobromide), a Novel Small Molecule Survivin Suppressant</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1124/dmd.109.028142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fdmd.109.028142" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=1-4&issue=1&author=T.+Minematsuauthor=M.+Iwaiauthor=K.-I.+Umeharaauthor=T.+Usuiauthor=H.+Kamimura&title=Characterization+of+Human+Organic+Cation+Transporter+1+%28OCT1%2FSLC22A1%29-+and+OCT2+%28SLC22A2%29-Mediated+Transport+of+1-%282-Methoxyethyl%29-2-Methyl-4%2C9-Dioxo-3-%28Pyrazin-2-Ylmethyl%29-+4%2C9-Dihydro-1H-Naphtho%5B2%2C3-D%5DImidazolium+Bromide+%28YM155+Monobromide%29%2C+a+Novel+Small+Molecule+Survivin+Suppressant&doi=10.1124%2Fdmd.109.028142"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fdmd.109.028142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.109.028142%26sid%3Dliteratum%253Aachs%26aulast%3DMinematsu%26aufirst%3DT.%26aulast%3DIwai%26aufirst%3DM.%26aulast%3DUmehara%26aufirst%3DK.-I.%26aulast%3DUsui%26aufirst%3DT.%26aulast%3DKamimura%26aufirst%3DH.%26atitle%3DCharacterization%2520of%2520Human%2520Organic%2520Cation%2520Transporter%25201%2520%2528OCT1%252FSLC22A1%2529-%2520and%2520OCT2%2520%2528SLC22A2%2529-Mediated%2520Transport%2520of%25201-%25282-Methoxyethyl%2529-2-Methyl-4%252C9-Dioxo-3-%2528Pyrazin-2-Ylmethyl%2529-%25204%252C9-Dihydro-1H-Naphtho%255B2%252C3-D%255DImidazolium%2520Bromide%2520%2528YM155%2520Monobromide%2529%252C%2520a%2520Novel%2520Small%2520Molecule%2520Survivin%2520Suppressant%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2010%26volume%3D38%26issue%3D1%26spage%3D1%26epage%3D4%26doi%3D10.1124%2Fdmd.109.028142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Bourdet, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakker, D. R.</span><span> </span><span class="NLM_article-title">Differential Substrate and Inhibitory Activities of Ranitidine and Famotidine Toward Human Organic Cation Transporter 1 (hOCT1; SLC22A1), hOCT2 (SLC22A2), and hOCT3 (SLC22A3)</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">315</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">1288</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1124/jpet.105.091223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.105.091223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=16141367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlalurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=315&publication_year=2005&pages=1288-1297&issue=3&author=D.+L.+Bourdetauthor=J.+B.+Pritchardauthor=D.+R.+Thakker&title=Differential+Substrate+and+Inhibitory+Activities+of+Ranitidine+and+Famotidine+Toward+Human+Organic+Cation+Transporter+1+%28hOCT1%3B+SLC22A1%29%2C+hOCT2+%28SLC22A2%29%2C+and+hOCT3+%28SLC22A3%29&doi=10.1124%2Fjpet.105.091223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Differential substrate and inhibitory activities of ranitidine and famotidine toward human organic cation transporter 1 (hOCT1; SLC22A1), hOCT2 (SLC22A2), and hOCT3 (SLC22A3)</span></div><div class="casAuthors">Bourdet, David L.; Pritchard, John B.; Thakker, Dhiren R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">315</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1288-1297</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Human org. cation transporters (hOCTs) are expressed in organs of drug absorption and elimination and play an important role in the uptake and elimination of xenobiotics.  The purpose of this study was to evaluate the substrate and inhibitory activity of the H2-receptor antagonists ranitidine and famotidine toward hOCTs and to det. the hOCT isoforms involved in the absorption and elimination of these compds. in humans.  Inhibition and substrate specificity of hOCT1, hOCT2, and hOCT3 for ranitidine and famotidine were elucidated in cRNA-injected Xenopus laevis oocytes.  Ranitidine and famotidine exhibited similarly potent inhibition of [3H]1-methyl-4-Ph pyridinium uptake into hOCT1-expressing (IC50 = 33 and 28 μM, resp.) and hOCT2-expressing oocytes (IC50 = 76 and 114 μM, resp.).  Famotidine exhibited potent inhibition of hOCT3; in contrast, ranitidine was a moderately weak inhibitor (IC50 = 6.7 and 290 μM, resp.).  [3H]Ranitidine uptake was stimulated by hOCT1 (Km = 70±9 μM) and to a much smaller extent by hOCT2.  No stimulation of [3H]ranitidine uptake was obsd. in hOCT3-expressing oocytes.  Trans-Stimulation and electrophysiol. studies suggested that famotidine also is an hOCT1 substrate and exhibits poor or no substrate activity toward hOCT2 and hOCT3.  Thus, hOCT1, which is expressed in the intestine and liver, is likely to play a major role in the intestinal absorption and hepatic disposition of ranitidine and famotidine in humans, whereas hOCT2, the major isoform present in the kidney, may play only a minor role in their renal elimination.  Famotidine seems to be one of the most potent inhibitors of hOCT3 yet identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM1LN-rRwlxrVg90H21EOLACvtfcHk0lipehLW0ED86A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlalurnE&md5=f7028e4e563179d4fbaa080ccedbdf25</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1124%2Fjpet.105.091223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.105.091223%26sid%3Dliteratum%253Aachs%26aulast%3DBourdet%26aufirst%3DD.%2BL.%26aulast%3DPritchard%26aufirst%3DJ.%2BB.%26aulast%3DThakker%26aufirst%3DD.%2BR.%26atitle%3DDifferential%2520Substrate%2520and%2520Inhibitory%2520Activities%2520of%2520Ranitidine%2520and%2520Famotidine%2520Toward%2520Human%2520Organic%2520Cation%2520Transporter%25201%2520%2528hOCT1%253B%2520SLC22A1%2529%252C%2520hOCT2%2520%2528SLC22A2%2529%252C%2520and%2520hOCT3%2520%2528SLC22A3%2529%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2005%26volume%3D315%26issue%3D3%26spage%3D1288%26epage%3D1297%26doi%3D10.1124%2Fjpet.105.091223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Bachmakov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaeser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">König, J.</span><span> </span><span class="NLM_article-title">Einfluss Von B-Blockern Auf Den Transport Von Metformin Durch Den Hepatischen Aufnahmetransporter Für Organische Kationen OCT1</span> <span class="citation_source-journal">Diabetologie und Stoffwechsel</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">274</span><span class="refDoi"> DOI: 10.1055/s-0029-1222078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1055%2Fs-0029-1222078" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=274&author=I.+Bachmakovauthor=H.+Glaeserauthor=J.+K%C3%B6nig&title=Einfluss+Von+B-Blockern+Auf+Den+Transport+Von+Metformin+Durch+Den+Hepatischen+Aufnahmetransporter+F%C3%BCr+Organische+Kationen+OCT1&doi=10.1055%2Fs-0029-1222078"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1055%2Fs-0029-1222078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0029-1222078%26sid%3Dliteratum%253Aachs%26aulast%3DBachmakov%26aufirst%3DI.%26aulast%3DGlaeser%26aufirst%3DH.%26aulast%3DK%25C3%25B6nig%26aufirst%3DJ.%26atitle%3DEinfluss%2520Von%2520B-Blockern%2520Auf%2520Den%2520Transport%2520Von%2520Metformin%2520Durch%2520Den%2520Hepatischen%2520Aufnahmetransporter%2520F%25C3%25BCr%2520Organische%2520Kationen%2520OCT1%26jtitle%3DDiabetologie%2520und%2520Stoffwechsel%26date%3D2009%26volume%3D4%26spage%3D274%26doi%3D10.1055%2Fs-0029-1222078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Barr, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, J. P.</span><span> </span><span class="NLM_article-title">Inhibition of Human Liver Aldehyde Oxidase: Implications for Potential Drug-Drug Interactions</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2381</span><span class="NLM_x">–</span> <span class="NLM_lpage">2386</span><span class="refDoi"> DOI: 10.1124/dmd.111.041806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fdmd.111.041806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21940905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCru77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=2381-2386&issue=12&author=J.+T.+Barrauthor=J.+P.+Jones&title=Inhibition+of+Human+Liver+Aldehyde+Oxidase%3A+Implications+for+Potential+Drug-Drug+Interactions&doi=10.1124%2Fdmd.111.041806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human liver aldehyde oxidase: implications for potential drug-drug interactions</span></div><div class="casAuthors">Barr, John T.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2381-2386</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">During the course of our research efforts to understand the kinetics of human aldehyde oxidase as a xenobiotic-clearing enzyme, we investigated the effect of eight different inhibitors on the oxidn. of the probe substrate phthalazine.  Satn. kinetic parameters for phthalazine oxidn. in human liver cytosol were found to be the following: Km = 8.0 ± 0.4 μM and Vmax = 4.3 ± 0.1 nmol · min-1 · mg protein-1.  Inhibitory potency of the inhibitors tested ranged from 0.1 to 5 μM.  Of the eight different inhibitor compds. tested, seven were obsd. to inhibit through a mixed mode and one through a strictly competitive mode.  A ratio of the Kii and Kis values was used to assess the relative competitiveness of each inhibitor.  For the mixed inhibitors, the mode of inhibition varied from mostly uncompetitive to predominantly competitive (Kii/Kis values ranging from 0.1 to 15).  The implications for potential drug-drug interactions and inhibition mechanism are discussed.  We found two inhibitors, clozapine and chlorpromazine, that have a moderate predicted risk of drug-drug interactions based on the Ki value relative to the inhibitor concn. in human plasma, having a calcd. [I]/Ki value of 0.4 and 0.8, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS6gGC3rD4U7Vg90H21EOLACvtfcHk0lipehLW0ED86A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCru77F&md5=1113e63fdb48c3f8ed07c5ca4ae8cf24</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.041806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.041806%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%26aufirst%3DJ.%2BT.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DInhibition%2520of%2520Human%2520Liver%2520Aldehyde%2520Oxidase%253A%2520Implications%2520for%2520Potential%2520Drug-Drug%2520Interactions%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26issue%3D12%26spage%3D2381%26epage%3D2386%26doi%3D10.1124%2Fdmd.111.041806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Gui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obaidat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaguturu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagenbuch, B.</span><span> </span><span class="NLM_article-title">Development of a Cell-Based High-Throughput Assay to Screen for Inhibitors of Organic Anion Transporting Polypeptides 1B1 and 1B3</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.2174/1875397301004010001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.2174%2F1875397301004010001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=20448812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFShtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2010&pages=1-8&author=C.+Guiauthor=A.+Obaidatauthor=R.+Chaguturuauthor=B.+Hagenbuch&title=Development+of+a+Cell-Based+High-Throughput+Assay+to+Screen+for+Inhibitors+of+Organic+Anion+Transporting+Polypeptides+1B1+and+1B3&doi=10.2174%2F1875397301004010001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a cell-based high-throughput assay to screen for inhibitors of organic anion transporting polypeptides 1B1 and 1B3</span></div><div class="casAuthors">Gui, Chunshan; Obaidat, Amanda; Chaguturu, Rathnam; Hagenbuch, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The two org. anion transporting polypeptides (OATPs) 1B1 and 1B3 are expressed at the sinusoidal membrane of hepatocytes.  They have a broad and overlapping substrate specificity and transport many endobiotics and drugs.  Specific inhibitors are required to det. the contribution of each OATP to the hepatocellular uptake of common substrates.  We have developed a cell-based high-throughput assay to screen chem. libraries in order to identify such inhibitors for OATP1B1 and OATP1B3.  We have used OATP1B1- or OATP1B3-expressing Chinese Hamster Ovary cells on 96-well plates and detd. uptake of fluorescein-methotrexate (FMTX).  We validated the assay with known inhibitors and screened the well characterized Prestwick library of 1120 drugs.  Along with several known OATP inhibitors including rifampicin, cyclosporine A and mifepristone we identified some new inhibitors.  For inhibitors that seemed to be able to distinguish between OATP1B1- and OATP1B3-mediated FMTX uptake IC50 values were detd.  Estropipate (estrone-3-sulfate stabilized with piperazine) was the most selective OATP1B1 inhibitor (IC50 = 0.06 μM vs. 19.3 μM for OATP1B3).  Ursolic acid was the most selective OATP1B3 inhibitor (IC50 = 2.3 μM vs. 12.5 μM for OATP1B1).  In conclusion, this cell-based assay should allow us to identify even more specific inhibitors by screening larger chem. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9MovUxE4hD7Vg90H21EOLACvtfcHk0ljGlyiwzIkPgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFShtbo%253D&md5=707ec92dbc83bfe2f9f228378ddcc55e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.2174%2F1875397301004010001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301004010001%26sid%3Dliteratum%253Aachs%26aulast%3DGui%26aufirst%3DC.%26aulast%3DObaidat%26aufirst%3DA.%26aulast%3DChaguturu%26aufirst%3DR.%26aulast%3DHagenbuch%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Cell-Based%2520High-Throughput%2520Assay%2520to%2520Screen%2520for%2520Inhibitors%2520of%2520Organic%2520Anion%2520Transporting%2520Polypeptides%25201B1%2520and%25201B3%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2010%26volume%3D4%26spage%3D1%26epage%3D8%26doi%3D10.2174%2F1875397301004010001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">De Bruyn, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Westen, G. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stieger, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Witte, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustijns, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annaert, P. P.</span><span> </span><span class="NLM_article-title">Structure-Based Identification of OATP1B1/3 Inhibitors</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1257</span><span class="NLM_x">–</span> <span class="NLM_lpage">1267</span><span class="refDoi"> DOI: 10.1124/mol.112.084152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.112.084152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23571415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=1257-1267&issue=6&author=T.+De+Bruynauthor=G.+J.+P.+van+Westenauthor=A.+P.+Ijzermanauthor=B.+Stiegerauthor=P.+de+Witteauthor=P.+F.+Augustijnsauthor=P.+P.+Annaert&title=Structure-Based+Identification+of+OATP1B1%2F3+Inhibitors&doi=10.1124%2Fmol.112.084152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based identification of OATP1B1/3 inhibitors</span></div><div class="casAuthors">De Bruyn, Tom; van Western, Gerard J. P.; IJzerman, Adriaan P.; Stieger, Bruno; de Witte, Peter; Augustijns, Patrick F.; Annaert, Pieter P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1257-1267</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Several recent studies show that inhibition of the hepatic transport proteins org. anion-transporting polypeptide 1B1 (OATP1B1) and 1B3 (OATP1B3) can result in clin. relevant drug-drug interactions (DDI).  To avoid late-stage development drug failures due to OATP1B-mediated DDI, predictive in vitro and in silico methods should be implemented at an early stage of the drug candidate evaluation process.  In the present study, we first developed a high-throughput in vitro transporter inhibition assay for the OATP1B subfamily.  A total of 2000 compds. were tested as potential modulators of the uptake of the OATP1B substrate sodium fluorescein, in OATP1B1- or 1B3-transfected Chinese hamster ovary cells.  At an equimolar substrate-inhibitor concn. of 10 μM, 212 and 139 mols. were identified as OATP1B1 and OATP1B3 inhibitors, resp. (min. 50% inhibition).  For 69 compds., previously not identified as OATP1B inhibitors, concn.-dependent inhibition was also detd., yielding Ki values ranging from 0.06 to 6.5 μM.  Based on these in vitro data, we subsequently developed a proteochemometrics-based in silico model, which predicted OATP1B inhibitors in the test group (20% of the dataset) with high specificity (86%) and sensitivity (78%).  Moreover, several physicochem. compd. properties and substructures related to OATP1B1/1B3 inhibition or inactivity were identified.  Finally, model performance was prospectively verified with a set of 54 compds. not included in the original dataset.  This validation indicated that 80 and 74% of the compds. were correctly classified for OATP1B1 and OATP1B3 inhibition, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktLtqKIqWh7Vg90H21EOLACvtfcHk0ljGlyiwzIkPgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D&md5=bbf1617e8efba061f65b8f643baa5b98</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.084152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.084152%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BBruyn%26aufirst%3DT.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DStieger%26aufirst%3DB.%26aulast%3Dde%2BWitte%26aufirst%3DP.%26aulast%3DAugustijns%26aufirst%3DP.%2BF.%26aulast%3DAnnaert%26aufirst%3DP.%2BP.%26atitle%3DStructure-Based%2520Identification%2520of%2520OATP1B1%252F3%2520Inhibitors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26issue%3D6%26spage%3D1257%26epage%3D1267%26doi%3D10.1124%2Fmol.112.084152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Svetnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feuston, B. P.</span><span> </span><span class="NLM_article-title">Random Forest: a Classification and Regression Tool for Compound Classification and QSAR Modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1947</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1021/ci034160g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci034160g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=1947-1958&issue=6&author=V.+Svetnikauthor=A.+Liawauthor=C.+Tongauthor=J.+C.+Culbersonauthor=R.+P.+Sheridanauthor=B.+P.+Feuston&title=Random+Forest%3A+a+Classification+and+Regression+Tool+for+Compound+Classification+and+QSAR+Modeling&doi=10.1021%2Fci034160g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Random Forest: A Classification and Regression Tool for Compound Classification and QSAR Modeling</span></div><div class="casAuthors">Svetnik, Vladimir; Liaw, Andy; Tong, Christopher; Culberson, J. Christopher; Sheridan, Robert P.; Feuston, Bradley P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1947-1958</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new classification and regression tool, Random Forest, is introduced and investigated for predicting a compd.'s quant. or categorical biol. activity based on a quant. description of the compd.'s mol. structure.  Random Forest is an ensemble of unpruned classification or regression trees created by using bootstrap samples of the training data and random feature selection in tree induction.  Prediction is made by aggregating (majority vote or averaging) the predictions of the ensemble.  The authors built predictive models for six cheminformatics data sets.  The authors anal. demonstrates that Random Forest is a powerful tool capable of delivering performance that is among the most accurate methods to date.  The authors also present three addnl. features of Random Forest: built-in performance assessment, a measure of relative importance of descriptors, and a measure of compd. similarity that is weighted by the relative importance of descriptors.  It is the combination of relatively high prediction accuracy and its collection of desired features that makes Random Forest uniquely suited for modeling in cheminformatics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjoRt5v7rYwLVg90H21EOLACvtfcHk0ljGlyiwzIkPgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D&md5=dea7867551ec30260b0091b90593a660</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci034160g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034160g%26sid%3Dliteratum%253Aachs%26aulast%3DSvetnik%26aufirst%3DV.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DC.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DFeuston%26aufirst%3DB.%2BP.%26atitle%3DRandom%2520Forest%253A%2520a%2520Classification%2520and%2520Regression%2520Tool%2520for%2520Compound%2520Classification%2520and%2520QSAR%2520Modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2003%26volume%3D43%26issue%3D6%26spage%3D1947%26epage%3D1958%26doi%3D10.1021%2Fci034160g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Wittwer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zur, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosaka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Discovery of Potent, Selective Multidrug and Toxin Extrusion Transporter 1 (MATE1, SLC47A1) Inhibitors Through Prescription Drug Profiling and Computational Modeling</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">781</span><span class="NLM_x">–</span> <span class="NLM_lpage">795</span><span class="refDoi"> DOI: 10.1021/jm301302s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301302s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjurjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=781-795&issue=3&author=M.+B.+Wittwerauthor=A.+A.+Zurauthor=N.+Khuriauthor=Y.+Kidoauthor=A.+Kosakaauthor=X.+Zhangauthor=K.+M.+Morrisseyauthor=A.+Saliauthor=Y.+Huangauthor=K.+M.+Giacomini&title=Discovery+of+Potent%2C+Selective+Multidrug+and+Toxin+Extrusion+Transporter+1+%28MATE1%2C+SLC47A1%29+Inhibitors+Through+Prescription+Drug+Profiling+and+Computational+Modeling&doi=10.1021%2Fjm301302s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent, Selective Multidrug and Toxin Extrusion Transporter 1 (MATE1, SLC47A1) Inhibitors Through Prescription Drug Profiling and Computational Modeling</span></div><div class="casAuthors">Wittwer, Matthias B.; Zur, Arik A.; Khuri, Natalia; Kido, Yasuto; Kosaka, Alan; Zhang, Xuexiang; Morrissey, Kari M.; Sali, Andrej; Huang, Yong; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">781-795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The human multidrug and toxin extrusion (MATE) transporter 1 contributes to the tissue distribution and excretion of many drugs.  Inhibition of MATE1 may result in potential drug-drug interactions (DDIs) and alterations in drug exposure and accumulation in various tissues.  The primary goals of this project were to identify MATE1 inhibitors with clin. importance or in vitro utility and to elucidate the physicochem. properties that differ between MATE1 and OCT2 inhibitors.  Using a fluorescence assay of ASP+ uptake in cells stably expressing MATE1, over 900 prescription drugs were screened and 84 potential MATE1 inhibitors were found.  We identified several MATE1 selective inhibitors including four FDA-approved medications that may be clin. relevant MATE1 inhibitors and could cause a clin. DDI.  In parallel, a QSAR model identified distinct mol. properties of MATE1 vs. OCT2 inhibitors and was used to screen the DrugBank in silico library for new hits in a larger chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonSgabmRJ4orVg90H21EOLACvtfcHk0lh0Nyv787VGzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjurjE&md5=9c26a96ca5db7601155b54f8b8818af3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm301302s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301302s%26sid%3Dliteratum%253Aachs%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26aulast%3DZur%26aufirst%3DA.%2BA.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DKido%26aufirst%3DY.%26aulast%3DKosaka%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMorrissey%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DDiscovery%2520of%2520Potent%252C%2520Selective%2520Multidrug%2520and%2520Toxin%2520Extrusion%2520Transporter%25201%2520%2528MATE1%252C%2520SLC47A1%2529%2520Inhibitors%2520Through%2520Prescription%2520Drug%2520Profiling%2520and%2520Computational%2520Modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D3%26spage%3D781%26epage%3D795%26doi%3D10.1021%2Fjm301302s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Wishart, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djoumbou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandal, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aziat, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouatra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinelnikov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arndt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yallou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjorndahl, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Pineiro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neveu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greiner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scalbert, A.</span><span> </span><span class="NLM_article-title">HMDB 3.0--the Human Metabolome Database in 2013</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D801</span><span class="NLM_x">–</span> <span class="NLM_lpage">D807</span><span class="refDoi"> DOI: 10.1093/nar/gks1065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fnar%2Fgks1065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D801-D807&issue=D1&author=D.+S.+Wishartauthor=T.+Jewisonauthor=A.+C.+Guoauthor=M.+Wilsonauthor=C.+Knoxauthor=Y.+Liuauthor=Y.+Djoumbouauthor=R.+Mandalauthor=F.+Aziatauthor=E.+Dongauthor=S.+Bouatraauthor=I.+Sinelnikovauthor=D.+Arndtauthor=J.+Xiaauthor=P.+Liuauthor=F.+Yallouauthor=T.+Bjorndahlauthor=R.+Perez-Pineiroauthor=R.+Eisnerauthor=F.+Allenauthor=V.+Neveuauthor=R.+Greinerauthor=A.+Scalbert&title=HMDB+3.0%2D%2Dthe+Human+Metabolome+Database+in+2013&doi=10.1093%2Fnar%2Fgks1065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks1065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks1065%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DJewison%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DWilson%26aufirst%3DM.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDjoumbou%26aufirst%3DY.%26aulast%3DMandal%26aufirst%3DR.%26aulast%3DAziat%26aufirst%3DF.%26aulast%3DDong%26aufirst%3DE.%26aulast%3DBouatra%26aufirst%3DS.%26aulast%3DSinelnikov%26aufirst%3DI.%26aulast%3DArndt%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DYallou%26aufirst%3DF.%26aulast%3DBjorndahl%26aufirst%3DT.%26aulast%3DPerez-Pineiro%26aufirst%3DR.%26aulast%3DEisner%26aufirst%3DR.%26aulast%3DAllen%26aufirst%3DF.%26aulast%3DNeveu%26aufirst%3DV.%26aulast%3DGreiner%26aufirst%3DR.%26aulast%3DScalbert%26aufirst%3DA.%26atitle%3DHMDB%25203.0--the%2520Human%2520Metabolome%2520Database%2520in%25202013%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26issue%3DD1%26spage%3DD801%26epage%3DD807%26doi%3D10.1093%2Fnar%2Fgks1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Kotake, Y.</span><span> </span><span class="NLM_article-title">[Tetrahydroisoquinoline Derivatives as Possible Parkinson’s Disease-Inducing Substances]</span> <span class="citation_source-journal">Yakugaku Zasshi</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">975</span><span class="NLM_x">–</span> <span class="NLM_lpage">982</span><span class="refDoi"> DOI: 10.1248/yakushi.122.975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1248%2Fyakushi.122.975" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2002&pages=975-982&issue=11&author=Y.+Kotake&title=%5BTetrahydroisoquinoline+Derivatives+as+Possible+Parkinson%E2%80%99s+Disease-Inducing+Substances%5D&doi=10.1248%2Fyakushi.122.975"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1248%2Fyakushi.122.975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fyakushi.122.975%26sid%3Dliteratum%253Aachs%26aulast%3DKotake%26aufirst%3DY.%26atitle%3D%255BTetrahydroisoquinoline%2520Derivatives%2520as%2520Possible%2520Parkinson%25E2%2580%2599s%2520Disease-Inducing%2520Substances%255D%26jtitle%3DYakugaku%2520Zasshi%26date%3D2002%26volume%3D122%26issue%3D11%26spage%3D975%26epage%3D982%26doi%3D10.1248%2Fyakushi.122.975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Kotake, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makino, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirobe, M.</span><span> </span><span class="NLM_article-title">1-Benzyl-1,2,3,4-Tetrahydroisoquinoline as a Parkinsonism-Inducing Agent: a Novel Endogenous Amine in Mouse Brain and Parkinsonian CSF</span> <span class="citation_source-journal">J. Neurochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">2633</span><span class="NLM_x">–</span> <span class="NLM_lpage">2638</span><span class="refDoi"> DOI: 10.1046/j.1471-4159.1995.65062633.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1046%2Fj.1471-4159.1995.65062633.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=7595560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADyaK2MXpsFersbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1995&pages=2633-2638&issue=6&author=Y.+Kotakeauthor=Y.+Tasakiauthor=Y.+Makinoauthor=S.+Ohtaauthor=M.+Hirobe&title=1-Benzyl-1%2C2%2C3%2C4-Tetrahydroisoquinoline+as+a+Parkinsonism-Inducing+Agent%3A+a+Novel+Endogenous+Amine+in+Mouse+Brain+and+Parkinsonian+CSF&doi=10.1046%2Fj.1471-4159.1995.65062633.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">1-Benzyl-1,2,3,4-tetrahydroisoquinoline as a parkinsonism-inducing agent: a novel endogenous amine in mouse brain and parkinsonian CSF</span></div><div class="casAuthors">Kotake, Yaichiro; Tasaki, Yoshikazu; Makino, Yukiko; Ohta, Shigeru; Hirobe, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2633-8</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Lippincott-Raven</span>)
        </div><div class="casAbstract">1-Benzyl-1,2,3,4-tetrahydroisoquinoline (1BnTIQ) was detected as a novel endogenous amine in mouse brain and parkinsonian CSF by using the gas chromatog.-selected ion-monitoring method.  The level of 1BnTIQ was very high in CSF of some parkinsonian patients compared with that of controls with other neurol. diseases, the mean value being three times higher (parkinsonians: 1.17 ng/mL of CSF, vs. controls: 0.40 ng/mL of CSF; not significantly different).  The pole test, a toxicol. examn. to evaluate behavior abnormalities related to Parkinson's disease, was used to examine the pharmacol. effect of 1BnTIQ in mice.  Repeated administration of 1BnTIQ induced behavior abnormalities, which pretreatment with 1-methyl-1,2,3,4-tetrahydroisoquinoline could prevent.  The authors suggest that 1BnTIQ may be related to the idiopathic Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1xDVD1jNQbVg90H21EOLACvtfcHk0lhcgctmXTDywg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsFersbw%253D&md5=890490f56fd9ef3e7ef26a03befb3519</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.1995.65062633.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.1995.65062633.x%26sid%3Dliteratum%253Aachs%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DTasaki%26aufirst%3DY.%26aulast%3DMakino%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DS.%26aulast%3DHirobe%26aufirst%3DM.%26atitle%3D1-Benzyl-1%252C2%252C3%252C4-Tetrahydroisoquinoline%2520as%2520a%2520Parkinsonism-Inducing%2520Agent%253A%2520a%2520Novel%2520Endogenous%2520Amine%2520in%2520Mouse%2520Brain%2520and%2520Parkinsonian%2520CSF%26jtitle%3DJ.%2520Neurochem.%26date%3D1995%26volume%3D65%26issue%3D6%26spage%3D2633%26epage%3D2638%26doi%3D10.1046%2Fj.1471-4159.1995.65062633.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaner, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Functional Characterization of an Organic Cation Transporter (hOCT1) in a Transiently Transfected Human Cell Line (HeLa)</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">354</span><span class="NLM_x">–</span> <span class="NLM_lpage">361</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=9655880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADyaK1cXksFCgs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=1998&pages=354-361&issue=1&author=L.+Zhangauthor=M.+E.+Schanerauthor=K.+M.+Giacomini&title=Functional+Characterization+of+an+Organic+Cation+Transporter+%28hOCT1%29+in+a+Transiently+Transfected+Human+Cell+Line+%28HeLa%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of an organic cation transporter (hOCT1) in a transiently transfected human cell line (HeLa)</span></div><div class="casAuthors">Zhang, Lei; Schaner, Marci E.; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">354-361</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">Recently, a polyspecific org. cation transporter, hOCT1, was cloned from human liver.  To date, limited studies examg. the functional characteristics of the transporter have been performed.  The purpose of the present study was to develop a mammalian expression system for hOCT1 and to characterize the interactions of various compds. with the cloned transporter.  Lipofection was used to transiently transfect the hOCT1 plasmid DNA in a human cell line, HeLa.  We tested the interaction of an array of org. cations and other compds. with hOCT1 by detg. Ki values in inhibiting 14C-tetraethylammonium (TEA) transport in the transfected cells.  Transient expression of hOCT1 activity was obsd. between 24 and 72 h post-transfection, with maximal expression at approx. 40 h.  TEA transport was temp. dependent and saturable with Vmax and Km values of 2.89 ± 0.448 nmol/mg protein/30 min and 229 ± 78.4 μM, resp.  14C-TEA uptake in hOCT1 plasmid DNA-transfected HeLa cells was trans-stimulated by unlabeled TEA and 1-methyl-4-phenyl-pyridinium.  Org. cations, including clonidine, quinine, quinidine and verapamil (0.1 mM), significantly inhibited 14C-TEA uptake, whereas the org. anion, p-aminohippuric acid (5 mM), did not.  The neutral compds., corticosterone (Ki, 7.0 μM) and midazolam (Ki, 3.7 μM) potently inhibited 14C-TEA uptake.  The Ki values of several compds. in interacting with hOCT1 differed substantially from the corresponding values for the rat org. cation transporter, rOCT1, and the human kidney-specific org. cation transporter, hOCT2, detd. in previous studies.  Transiently transfected HeLa cells represent a useful tool in studying the interactions and kinetics of org. cations and other xenobiotics with hOCT1 and in understanding the mol. events involved in org. cation transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQuMS6U2vXBLVg90H21EOLACvtfcHk0lhcgctmXTDywg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksFCgs70%253D&md5=81ab14d6ccec01adf0aff127a3b2f911</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DSchaner%26aufirst%3DM.%2BE.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DFunctional%2520Characterization%2520of%2520an%2520Organic%2520Cation%2520Transporter%2520%2528hOCT1%2529%2520in%2520a%2520Transiently%2520Transfected%2520Human%2520Cell%2520Line%2520%2528HeLa%2529%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1998%26volume%3D286%26issue%3D1%26spage%3D354%26epage%3D361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovejoy, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shima, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagpacan, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapuk, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippard, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Organic Cation Transporters Are Determinants of Oxaliplatin Cytotoxicity</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">8847</span><span class="NLM_x">–</span> <span class="NLM_lpage">8857</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-0769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1158%2F0008-5472.CAN-06-0769" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=8847-8857&issue=17&author=S.+Zhangauthor=K.+S.+Lovejoyauthor=J.+E.+Shimaauthor=L.+L.+Lagpacanauthor=Y.+Shuauthor=A.+Lapukauthor=Y.+Chenauthor=T.+Komoriauthor=J.+W.+Grayauthor=X.+Chenauthor=S.+J.+Lippardauthor=K.+M.+Giacomini&title=Organic+Cation+Transporters+Are+Determinants+of+Oxaliplatin+Cytotoxicity&doi=10.1158%2F0008-5472.CAN-06-0769"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-0769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-0769%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLovejoy%26aufirst%3DK.%2BS.%26aulast%3DShima%26aufirst%3DJ.%2BE.%26aulast%3DLagpacan%26aufirst%3DL.%2BL.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DLapuk%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKomori%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLippard%26aufirst%3DS.%2BJ.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DOrganic%2520Cation%2520Transporters%2520Are%2520Determinants%2520of%2520Oxaliplatin%2520Cytotoxicity%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26issue%3D17%26spage%3D8847%26epage%3D8857%26doi%3D10.1158%2F0008-5472.CAN-06-0769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Kimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narikawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enomoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endou, H.</span><span> </span><span class="NLM_article-title">Human Organic Anion Transporters and Human Organic Cation Transporters Mediate Renal Transport of Prostaglandins</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">301</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">298</span><span class="refDoi"> DOI: 10.1124/jpet.301.1.293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.301.1.293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=11907186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1Kqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=301&publication_year=2002&pages=293-298&issue=1&author=H.+Kimuraauthor=M.+Takedaauthor=S.+Narikawaauthor=A.+Enomotoauthor=K.+Ichidaauthor=H.+Endou&title=Human+Organic+Anion+Transporters+and+Human+Organic+Cation+Transporters+Mediate+Renal+Transport+of+Prostaglandins&doi=10.1124%2Fjpet.301.1.293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Human organic anion transporters and human organic cation transporters mediate renal transport of prostaglandins</span></div><div class="casAuthors">Kimura, Hiroaki; Takeda, Michio; Narikawa, Shinichi; Enomoto, Atsushi; Ichida, Kimiyoshi; Endou, Hitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">301</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">293-298</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Prostaglandin E2 (PGE2) and prostaglandin F2α (PGF2α) have been used for the induction of labor and the termination of pregnancy.  Renal excretion is shown to be an important pathway for the elimination of PGE2 and PGF2α.  The purpose of this study was to elucidate the mol. mechanism of renal PGE2 and PGF2α transport using cells stably expressing human org. anion transporter (hOAT) 1, hOAT2, hOAT3, and hOAT4; and human org. cation transporter (hOCT) 1 and hOCT2.  A time- and dose-dependent increase in PGE2 and PGF2α uptake was obsd. in cells expressing hOAT1, hOAT2, hOAT3, hOAT4, hOCT1, and hOCT2.  The Km values of PGE2 uptake by hOAT1, hOAT2, hOAT3, hOAT4, hOCT1, and hOCT2 were 970, 713, 345, 154, 657, and 28.9 nM, resp., whereas those of PGF2α uptake by hOAT1, hOAT3, hOAT4, hOCT1, and hOCT2 were 575, 1092, 692, 477, and 334 nM, resp.  PGE2 and PGF2α significantly inhibited org. anion uptake by hOATs and org. cation uptake by hOCTs.  In conclusion, considering the localization of these transporters, the results suggest that PGE2 and PGF2α transport in the basolateral membrane of the proximal tubule is mediated by multiple pathways including hOAT1, hOAT2, hOAT3, and hOCT2, whereas that in the apical side is mediated by hOAT4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2SONHwtVyO7Vg90H21EOLACvtfcHk0lhcgctmXTDywg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1Kqs7o%253D&md5=dab78409f108dec560468cfb9969bdb5</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fjpet.301.1.293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.301.1.293%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DTakeda%26aufirst%3DM.%26aulast%3DNarikawa%26aufirst%3DS.%26aulast%3DEnomoto%26aufirst%3DA.%26aulast%3DIchida%26aufirst%3DK.%26aulast%3DEndou%26aufirst%3DH.%26atitle%3DHuman%2520Organic%2520Anion%2520Transporters%2520and%2520Human%2520Organic%2520Cation%2520Transporters%2520Mediate%2520Renal%2520Transport%2520of%2520Prostaglandins%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D301%26issue%3D1%26spage%3D293%26epage%3D298%26doi%3D10.1124%2Fjpet.301.1.293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Bednarczyk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wikel, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Influence of Molecular Structure on Substrate Binding to the Human Organic Cation Transporter, hOCT1</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1124/mol.63.3.489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.63.3.489" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=489-498&issue=3&author=D.+Bednarczykauthor=S.+Ekinsauthor=J.+H.+Wikelauthor=S.+H.+Wright&title=Influence+of+Molecular+Structure+on+Substrate+Binding+to+the+Human+Organic+Cation+Transporter%2C+hOCT1&doi=10.1124%2Fmol.63.3.489"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1124%2Fmol.63.3.489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.63.3.489%26sid%3Dliteratum%253Aachs%26aulast%3DBednarczyk%26aufirst%3DD.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DWikel%26aufirst%3DJ.%2BH.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DInfluence%2520of%2520Molecular%2520Structure%2520on%2520Substrate%2520Binding%2520to%2520the%2520Human%2520Organic%2520Cation%2520Transporter%252C%2520hOCT1%26jtitle%3DMol.%2520Pharmacol.%26date%3D2003%26volume%3D63%26issue%3D3%26spage%3D489%26epage%3D498%26doi%3D10.1124%2Fmol.63.3.489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Moaddel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravichandran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bighi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaguchi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainer, I. W.</span><span> </span><span class="NLM_article-title">Pharmacophore Modelling of Stereoselective Binding to the Human Organic Cation Transporter (hOCT1)</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">151</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1305</span><span class="NLM_x">–</span> <span class="NLM_lpage">1314</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fsj.bjp.0707341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=17592512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFehsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2007&pages=1305-1314&issue=8&author=R.+Moaddelauthor=S.+Ravichandranauthor=F.+Bighiauthor=R.+Yamaguchiauthor=I.+W.+Wainer&title=Pharmacophore+Modelling+of+Stereoselective+Binding+to+the+Human+Organic+Cation+Transporter+%28hOCT1%29&doi=10.1038%2Fsj.bjp.0707341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore modeling of stereoselective binding to the human organic cation transporter (hOCT1)</span></div><div class="casAuthors">Moaddel, R.; Ravichandran, S.; Bighi, F.; Yamaguchi, R.; Wainer, I. W.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1305-1314</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The human org. cation transporter-1 (hOCT1) is a polyspecific transporter that plays a role in drug distribution, metab. and excretion.  Previous studies have demonstrated that hOCT1 binding can be stereoselective, but the mechanism for stereochem. recognition has not been described.  The purpose of this study was to develop a pharmacophore model to describe stereoselective binding to hOCT1.  Exptl. approach: A set of 22 compds. including 8 pairs of enantiomers and five pairs of diastereomers was used to develop a pharmacophore model.  The pharmacophore modeling was carried out using Catalyst version 4.11 and HypoGen and was based upon the correlation of the structures and activities (Ki values) of the compds. used in the study.  The resulting model contained a pos. ion, hydrophobic and two hydrogen-bond acceptor interaction sites.  The relative enantioselectivity of 8/8 enantiomeric pairs and diastereoselectivity of 5/5 diastereomers was described by mapping to a combination of at least 3 of the 4 functional feature sites of the model.  The pharmacophore model describes stereoselective interactions with hOCT1 at one of the binding sites on the mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXDIit5N8HbVg90H21EOLACvtfcHk0lhhev_2QwUyOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFehsL4%253D&md5=8a18c5b919d50b37f4497777841fc864</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707341%26sid%3Dliteratum%253Aachs%26aulast%3DMoaddel%26aufirst%3DR.%26aulast%3DRavichandran%26aufirst%3DS.%26aulast%3DBighi%26aufirst%3DF.%26aulast%3DYamaguchi%26aufirst%3DR.%26aulast%3DWainer%26aufirst%3DI.%2BW.%26atitle%3DPharmacophore%2520Modelling%2520of%2520Stereoselective%2520Binding%2520to%2520the%2520Human%2520Organic%2520Cation%2520Transporter%2520%2528hOCT1%2529%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D151%26issue%3D8%26spage%3D1305%26epage%3D1314%26doi%3D10.1038%2Fsj.bjp.0707341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Harper, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Multiple Mechanisms of Ligand Interaction with the Human Organic Cation Transporter, OCT2</span> <span class="citation_source-journal">Am. J. Physiol. Renal Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">304</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">F56</span><span class="NLM_x">–</span> <span class="NLM_lpage">F67</span><span class="refDoi"> DOI: 10.1152/ajprenal.00486.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1152%2Fajprenal.00486.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23034939" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2013&pages=F56-F67&issue=1&author=J.+N.+Harperauthor=S.+H.+Wright&title=Multiple+Mechanisms+of+Ligand+Interaction+with+the+Human+Organic+Cation+Transporter%2C+OCT2&doi=10.1152%2Fajprenal.00486.2012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1152%2Fajprenal.00486.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajprenal.00486.2012%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DJ.%2BN.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DMultiple%2520Mechanisms%2520of%2520Ligand%2520Interaction%2520with%2520the%2520Human%2520Organic%2520Cation%2520Transporter%252C%2520OCT2%26jtitle%3DAm.%2520J.%2520Physiol.%2520Renal%2520Physiol.%26date%3D2013%26volume%3D304%26issue%3D1%26spage%3DF56%26epage%3DF67%26doi%3D10.1152%2Fajprenal.00486.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Computational Modeling to Accelerate the Identification of Substrates and Inhibitors for Transporters That Affect Drug Disposition</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">661</span><span class="NLM_x">–</span> <span class="NLM_lpage">665</span><span class="refDoi"> DOI: 10.1038/clpt.2012.164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fclpt.2012.164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23010651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=661-665&issue=5&author=S.+Ekinsauthor=J.+E.+Polliauthor=P.+W.+Swaanauthor=S.+H.+Wright&title=Computational+Modeling+to+Accelerate+the+Identification+of+Substrates+and+Inhibitors+for+Transporters+That+Affect+Drug+Disposition&doi=10.1038%2Fclpt.2012.164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Modeling to Accelerate the Identification of Substrates and Inhibitors for Transporters That Affect Drug Disposition</span></div><div class="casAuthors">Ekins, S.; Polli, J. E.; Swaan, P. W.; Wright, S. H.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">661-665</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review describes the increased use of computational approaches to predicting drug interactions with human transporters that affect drug disposition and may lead to toxicity.  Ligand-based methods predominantly use limited exptl. data but provide new insights into structure-activity relationships.  Development of models capable of identifying new transport substrates and unwanted drug-drug interactions requires novel applications of current computational methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2kRgkvrcdObVg90H21EOLACvtfcHk0lhhev_2QwUyOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChsbjF&md5=47b55b7738690be24d97a57e121bf342</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2012.164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2012.164%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DPolli%26aufirst%3DJ.%2BE.%26aulast%3DSwaan%26aufirst%3DP.%2BW.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DComputational%2520Modeling%2520to%2520Accelerate%2520the%2520Identification%2520of%2520Substrates%2520and%2520Inhibitors%2520for%2520Transporters%2520That%2520Affect%2520Drug%2520Disposition%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2012%26volume%3D92%26issue%3D5%26spage%3D661%26epage%3D665%26doi%3D10.1038%2Fclpt.2012.164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Martínez-Guerrero, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Substrate-Dependent Inhibition of Human MATE1 by Cationic Ionic Liquids</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">495</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1124/jpet.113.204206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.113.204206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23785176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCmtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2013&pages=495-503&issue=3&author=L.+J.+Mart%C3%ADnez-Guerreroauthor=S.+H.+Wright&title=Substrate-Dependent+Inhibition+of+Human+MATE1+by+Cationic+Ionic+Liquids&doi=10.1124%2Fjpet.113.204206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-dependent inhibition of human MATE1 by cationic ionic liquids</span></div><div class="casAuthors">Martinez-Guerrero, Lucy J.; Wright, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-503</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The multidrug and toxin extruders 1- and 2-K (MATE1 and MATE2-K) are expressed in the luminal membrane of renal proximal tubule cells and provide the active step in the secretion of mols. that carry a net pos. charge at physiol. pH, so-called org. cations.  The present study tested whether structurally distinct MATE substrates can display different quant. profiles of inhibition when interacting with structurally distinct ligands.  The tested ligands were three structurally similar cationic ionic liqs. (ILs, salts in the liq. state: N-butylpyridinium, NBuPy; 1-methyl-3-butylimidazolium, Bmim; and N-butyl-N-methylpyrrolidinium, BmPy).  Uptake was measured using Chinese hamster ovary cells that stably expressed MATE1 or MATE2-K.  By trans-stimulation, all three ILs were transported by both MATE transporters.  The three ILs also inhibited uptake of three structurally distinct MATE substrates: 1-methyl-4-phenylpyridinium (MPP), triethylmethylammonium (TEMA), and N,N,N-trimethyl-2-[methyl(7-nitrobenzo[c][1,2,5]oxadiazol-4-yl)amino]ethanaminium (NBD-MTMA).  MATE1 displayed a higher affinity for the pyridinium-based NBuPy (IC50 values, 2-4 μM) than for either the pyrrolidinium- (BmPy; 20-70 μM) or imidazolium-based ILs (Bmim; 15-60 μM).  Inhibition of MPP, TEMA, and NBD-MTMA transport by NBuPy was competitive, with comparable Ki values against all substrates.  Bmim also competitively blocked the three substrates but with Ki values that differed significantly (20 μM against MPP and 30 μM against NBD-MTMA vs. 60 μM against TEMA).  Together, these data indicate that renal secretion of ILs by the human kidney involves MATE transporters and suggest that the mechanism of transport inhibition is ligand-dependent, supporting the hypothesis that the binding of substrates to MATE transporters involves interaction with a binding surface with multiple binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiPXNU1hIVbbVg90H21EOLACvtfcHk0lhhev_2QwUyOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCmtbbP&md5=50bcb1341615d29158f74b5b15627d73</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1124%2Fjpet.113.204206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.113.204206%26sid%3Dliteratum%253Aachs%26aulast%3DMart%25C3%25ADnez-Guerrero%26aufirst%3DL.%2BJ.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DSubstrate-Dependent%2520Inhibition%2520of%2520Human%2520MATE1%2520by%2520Cationic%2520Ionic%2520Liquids%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2013%26volume%3D346%26issue%3D3%26spage%3D495%26epage%3D503%26doi%3D10.1124%2Fjpet.113.204206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Tweedie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poirier, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span> </span><span class="NLM_article-title">International Transporter Consortium. Transporter Studies in Drug Development: Experience to Date and Follow-Up on Decision Trees From the International Transporter Consortium</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">125</span><span class="refDoi"> DOI: 10.1038/clpt.2013.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fclpt.2013.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23588318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A280%3ADC%252BC3srltl2ruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2013&pages=113-125&issue=1&author=D.+Tweedieauthor=J.+W.+Polliauthor=E.+G.+Berglundauthor=S.+M.+Huangauthor=L.+Zhangauthor=A.+Poirierauthor=X.+Chuauthor=B.+Feng&title=International+Transporter+Consortium.+Transporter+Studies+in+Drug+Development%3A+Experience+to+Date+and+Follow-Up+on+Decision+Trees+From+the+International+Transporter+Consortium&doi=10.1038%2Fclpt.2013.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Transporter studies in drug development: experience to date and follow-up on decision trees from the International Transporter Consortium</span></div><div class="casAuthors">Tweedie D; Polli J W; Berglund E Gil; Huang S M; Zhang L; Poirier A; Chu X; Feng B</div><div class="citationInfo"><span class="NLM_cas:title">Clinical pharmacology and therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The International Transporter Consortium (ITC) organized a second workshop in March 2012 to expand on the themes developed during the inaugural ITC workshop held in 2008.  The final session of the workshop provided perspectives from regulatory and industry-based scientists, with input from academic scientists, and focused primarily on the decision trees published from the first workshop.  These decision trees have become a central part of subsequent regulatory drug-drug interaction (DDI) guidances issued over the past few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTujrz08uS8s1WTCLT8lX9SfW6udTcc2ebyfa78MYDeZbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srltl2ruw%253D%253D&md5=bc460caa642b9132681aa4184f8c9f58</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2013.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2013.77%26sid%3Dliteratum%253Aachs%26aulast%3DTweedie%26aufirst%3DD.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DBerglund%26aufirst%3DE.%2BG.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPoirier%26aufirst%3DA.%26aulast%3DChu%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DB.%26atitle%3DInternational%2520Transporter%2520Consortium.%2520Transporter%2520Studies%2520in%2520Drug%2520Development%253A%2520Experience%2520to%2520Date%2520and%2520Follow-Up%2520on%2520Decision%2520Trees%2520From%2520the%2520International%2520Transporter%2520Consortium%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2013%26volume%3D94%26issue%3D1%26spage%3D113%26epage%3D125%26doi%3D10.1038%2Fclpt.2013.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Abe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Utsunomiya, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinohara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyatake, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sano, T.</span><span> </span><span class="NLM_article-title">Biochemical and Pathological Study of Endogenous 1-Benzyl-1,2,3,4-Tetrahydroisoquinoline-Induced Parkinsonism in the Mouse</span> <span class="citation_source-journal">Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">907</span><span class="NLM_x"> (</span><span class="NLM_issue">1–2</span><span class="NLM_x">) </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span><span class="refDoi"> DOI: 10.1016/S0006-8993(01)02573-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1016%2FS0006-8993%2801%2902573-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=11430895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Onurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=907&publication_year=2001&pages=134-138&issue=1%E2%80%932&author=K.+Abeauthor=K.+Taguchiauthor=T.+Wasaiauthor=J.+Renauthor=I.+Utsunomiyaauthor=T.+Shinoharaauthor=T.+Miyatakeauthor=T.+Sano&title=Biochemical+and+Pathological+Study+of+Endogenous+1-Benzyl-1%2C2%2C3%2C4-Tetrahydroisoquinoline-Induced+Parkinsonism+in+the+Mouse&doi=10.1016%2FS0006-8993%2801%2902573-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical and pathological study of endogenous 1-benzyl-1,2,3,4-tetrahydroisoquinoline-induced parkinsonism in the mouse</span></div><div class="casAuthors">Abe, K.; Taguchi, K.; Wasai, T.; Ren, J.; Utsunomiya, I.; Shinohara, T.; Miyatake, T.; Sano, T.</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">907</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">134-138</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The authors administered 1-benzyl-1,2,3,4-tetrahydroisoquinoline (1-BnTIQ; 80 mg/kg, i.p.), an endogenous neurotoxin known to cause bradykinesia, the Parkinson's disease-like symptom, to obtain biochem. and pathol. evidence of behavioral abnormalities.  Immunohistochem. anal. demonstrated that 1-BnTIQ did not decrease the no. of tyrosine hydroxylase-pos. cells in the substantia nigra.  Biochem. anal. demonstrated significantly increased striatal dopamine (DA) content while DA metabolites in the striatum remained at control levels.  Thus, the 1-BnTIQ-induced bradykinesia has a different mechanism of action than that underlying the MPTP-induced depletion of striatal DA neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqYuZ0JkjIJLVg90H21EOLACvtfcHk0lguYCsIZzbT4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Onurg%253D&md5=ef52a6650394ecda1ba0f93a5700cb5b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0006-8993%2801%2902573-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-8993%252801%252902573-2%26sid%3Dliteratum%253Aachs%26aulast%3DAbe%26aufirst%3DK.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DWasai%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DUtsunomiya%26aufirst%3DI.%26aulast%3DShinohara%26aufirst%3DT.%26aulast%3DMiyatake%26aufirst%3DT.%26aulast%3DSano%26aufirst%3DT.%26atitle%3DBiochemical%2520and%2520Pathological%2520Study%2520of%2520Endogenous%25201-Benzyl-1%252C2%252C3%252C4-Tetrahydroisoquinoline-Induced%2520Parkinsonism%2520in%2520the%2520Mouse%26jtitle%3DBrain%2520Res.%26date%3D2001%26volume%3D907%26issue%3D1%25E2%2580%25932%26spage%3D134%26epage%3D138%26doi%3D10.1016%2FS0006-8993%2801%2902573-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Kotake, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirobe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, S.</span><span> </span><span class="NLM_article-title">Chronic Administration of 1-Benzyl-1,2,3,4-Tetrahydroisoquinoline, an Endogenous Amine in the Brain, Induces Parkinsonism in a Primate</span> <span class="citation_source-journal">Neurosci. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">217</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1016/0304-3940(96)13065-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1016%2F0304-3940%2896%2913065-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=1996&pages=69-71&issue=1&author=Y.+Kotakeauthor=M.+Yoshidaauthor=M.+Ogawaauthor=Y.+Tasakiauthor=M.+Hirobeauthor=S.+Ohta&title=Chronic+Administration+of+1-Benzyl-1%2C2%2C3%2C4-Tetrahydroisoquinoline%2C+an+Endogenous+Amine+in+the+Brain%2C+Induces+Parkinsonism+in+a+Primate&doi=10.1016%2F0304-3940%2896%2913065-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0304-3940%2896%2913065-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3940%252896%252913065-2%26sid%3Dliteratum%253Aachs%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DOgawa%26aufirst%3DM.%26aulast%3DTasaki%26aufirst%3DY.%26aulast%3DHirobe%26aufirst%3DM.%26aulast%3DOhta%26aufirst%3DS.%26atitle%3DChronic%2520Administration%2520of%25201-Benzyl-1%252C2%252C3%252C4-Tetrahydroisoquinoline%252C%2520an%2520Endogenous%2520Amine%2520in%2520the%2520Brain%252C%2520Induces%2520Parkinsonism%2520in%2520a%2520Primate%26jtitle%3DNeurosci.%2520Lett.%26date%3D1996%26volume%3D217%26issue%3D1%26spage%3D69%26epage%3D71%26doi%3D10.1016%2F0304-3940%2896%2913065-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Pei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grishin, N. V.</span><span> </span><span class="NLM_article-title">PROMALS3D: a Tool for Multiple Protein Sequence and Structure Alignments</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2295</span><span class="NLM_x">–</span> <span class="NLM_lpage">2300</span><span class="refDoi"> DOI: 10.1093/nar/gkn072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fnar%2Fgkn072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=18287115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFGgtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=2295-2300&issue=7&author=J.+Peiauthor=B.-H.+Kimauthor=N.+V.+Grishin&title=PROMALS3D%3A+a+Tool+for+Multiple+Protein+Sequence+and+Structure+Alignments&doi=10.1093%2Fnar%2Fgkn072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">PROMALS3D: a tool for multiple protein sequence and structure alignments</span></div><div class="casAuthors">Pei, Jimin; Kim, Bong-Hyun; Grishin, Nick V.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2295-2300</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although multiple sequence alignments (MSAs) are essential for a wide range of applications from structure modeling to prediction of functional sites, construction of accurate MSAs for distantly related proteins remains a largely unsolved problem.  The rapidly increasing database of spatial structures is a valuable source to improve alignment quality.  The authors explore the use of 3D structural information to guide sequence alignments constructed by their MSA program PROMALS.  The resulting tool, PROMALS3D, automatically identifies homologs with known 3D structures for the input sequences, derives structural constraints through structure-based alignments and combines them with sequence constraints to construct consistency-based multiple sequence alignments.  The output is a consensus alignment that brings together sequence and structural information about input proteins and their homologs.  PROMALS3D can also align sequences of multiple input structures, with the output representing a multiple structure-based alignment refined in combination with sequence constraints.  The advantage of PROMALS3D is that it gives researchers an easy way to produce high-quality alignments consistent with both sequences and structures of proteins.  PROMALS3D outperforms a no. of existing methods for constructing multiple sequence or structural alignments using both ref.-dependent and ref.-independent evaluation methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoid8W72Zq7JLVg90H21EOLACvtfcHk0lguYCsIZzbT4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFGgtrs%253D&md5=8487ac1df3b9d1dc24976dcf412dd7ef</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn072%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DB.-H.%26aulast%3DGrishin%26aufirst%3DN.%2BV.%26atitle%3DPROMALS3D%253A%2520a%2520Tool%2520for%2520Multiple%2520Protein%2520Sequence%2520and%2520Structure%2520Alignments%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26issue%3D7%26spage%3D2295%26epage%3D2300%26doi%3D10.1093%2Fnar%2Fgkn072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Edgar, R. C.</span><span> </span><span class="NLM_article-title">MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1792</span><span class="NLM_x">–</span> <span class="NLM_lpage">1797</span><span class="refDoi"> DOI: 10.1093/nar/gkh340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fnar%2Fgkh340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=15034147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisF2ks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=1792-1797&issue=5&author=R.+C.+Edgar&title=MUSCLE%3A+Multiple+Sequence+Alignment+with+High+Accuracy+and+High+Throughput&doi=10.1093%2Fnar%2Fgkh340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span></div><div class="casAuthors">Edgar, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1792-1797</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences.  Elements of the algorithm include fast distance estn. using kmer counting, progressive alignment using a new profile function we call the log-expectation score, and refinement using tree-dependent restricted partitioning.  The speed and accuracy of MUSCLE are compared with T-Coffee, MAFFT and CLUSTALW on four test sets of ref. alignments: BAliBASE, SABmark, SMART and a new benchmark, PREFAB.  MUSCLE achieves the highest, or joint highest, rank in accuracy on each of these sets.  Without refinement, MUSCLE achieves av. accuracy statistically indistinguishable from T-Coffee and MAFFT, and is the fastest of the tested methods for large nos. of sequences, aligning 5000 sequences of av. length 350 in 7 min on a current desktop computer.  The MUSCLE program, source code and PREFAB test data are freely available at http://www.drive5.com/muscle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUXqjEriOqy7Vg90H21EOLACvtfcHk0lgDL3QgNM-ZCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisF2ks7w%253D&md5=1414065843fa881a1b66b98f4c9cba2e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh340%26sid%3Dliteratum%253Aachs%26aulast%3DEdgar%26aufirst%3DR.%2BC.%26atitle%3DMUSCLE%253A%2520Multiple%2520Sequence%2520Alignment%2520with%2520High%2520Accuracy%2520and%2520High%2520Throughput%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26issue%3D5%26spage%3D1792%26epage%3D1797%26doi%3D10.1093%2Fnar%2Fgkh340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">The FTMap Family of Web Servers for Determining and Characterizing Ligand-Binding Hot Spots of Proteins</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&issue=5&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+Family+of+Web+Servers+for+Determining+and+Characterizing+Ligand-Binding+Hot+Spots+of+Proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lgDL3QgNM-ZCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520Family%2520of%2520Web%2520Servers%2520for%2520Determining%2520and%2520Characterizing%2520Ligand-Binding%2520Hot%2520Spots%2520of%2520Proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26issue%3D5%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking Sets for Molecular Docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">23</span><span class="NLM_x">) </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span><span class="refDoi"> DOI: 10.1021/jm0608356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&issue=23&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+Sets+for+Molecular+Docking&doi=10.1021%2Fjm0608356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0lgDL3QgNM-ZCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520Sets%2520for%2520Molecular%2520Docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D23%26spage%3D6789%26epage%3D6801%26doi%3D10.1021%2Fjm0608356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Yap, C. W.</span><span> </span><span class="NLM_article-title">PaDEL-Descriptor: an Open Source Software to Calculate Molecular Descriptors and Fingerprints</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">1474</span><span class="refDoi"> DOI: 10.1002/jcc.21707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1002%2Fjcc.21707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21425294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsF2isLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=1466-1474&issue=7&author=C.+W.+Yap&title=PaDEL-Descriptor%3A+an+Open+Source+Software+to+Calculate+Molecular+Descriptors+and+Fingerprints&doi=10.1002%2Fjcc.21707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">PaDEL-descriptor: An open source software to calculate molecular descriptors and fingerprints</span></div><div class="casAuthors">Yap, Chun Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1466-1474</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Introduction : PaDEL-Descriptor is a software for calcg. mol. descriptors and fingerprints.  The software currently calcs. 797 descriptors (663 1D, 2D descriptors, and 134 3D descriptors) and 10 types of fingerprints.  These descriptors and fingerprints are calcd. mainly using The Chem. Development Kit.  Some addnl. descriptors and fingerprints were added, which include atom type electrotopol. state descriptors, McGowan vol., mol. linear free energy relation descriptors, ring counts, count of chem. substructures identified by Laggner, and binary fingerprints and count of chem. substructures identified by Klekota and Roth.  Methods : PaDEL-Descriptor was developed using the Java language and consists of a library component and an interface component.  The library component allows it to be easily integrated into quant. structure activity relationship software to provide the descriptor calcn. feature while the interface component allows it to be used as a standalone software.  The software uses a Master/Worker pattern to take advantage of the multiple CPU cores that are present in most modern computers to speed up calcns. of mol. descriptors.  Results : The software has several advantages over existing standalone mol. descriptor calcn. software.  It is free and open source, has both graphical user interface and command line interfaces, can work on all major platforms (Windows, Linux, MacOS), supports more than 90 different mol. file formats, and is multithreaded.  Conclusion : PaDEL-Descriptor is a useful addn. to the currently available mol. descriptor calcn. software.  The software can be downloaded at . © 2010 Wiley Periodicals, Inc. J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_2-BJmBJEJrVg90H21EOLACvtfcHk0lj9p4vKqaIJ2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsF2isLc%253D&md5=af74b9b0bc5621a6371e93caf8b2bfc7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21707%26sid%3Dliteratum%253Aachs%26aulast%3DYap%26aufirst%3DC.%2BW.%26atitle%3DPaDEL-Descriptor%253A%2520an%2520Open%2520Source%2520Software%2520to%2520Calculate%2520Molecular%2520Descriptors%2520and%2520Fingerprints%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26issue%3D7%26spage%3D1466%26epage%3D1474%26doi%3D10.1002%2Fjcc.21707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Molecular Docking Screens Using Comparative Models of Proteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2512</span><span class="NLM_x">–</span> <span class="NLM_lpage">2527</span><span class="refDoi"> DOI: 10.1021/ci9003706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2512-2527&issue=11&author=H.+Fanauthor=J.+J.+Irwinauthor=B.+M.+Webbauthor=G.+Klebeauthor=B.+K.+Shoichetauthor=A.+Sali&title=Molecular+Docking+Screens+Using+Comparative+Models+of+Proteins&doi=10.1021%2Fci9003706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screens Using Comparative Models of Proteins</span></div><div class="casAuthors">Fan, Hao; Irwin, John J.; Webb, Benjamin M.; Klebe, Gerhard; Shoichet, Brian K.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2512-2527</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two orders of magnitude more protein sequences can be modeled by comparative modeling than have been detd. by X-ray crystallog. and NMR spectroscopy.  Investigators have nevertheless been cautious about using comparative models for ligand discovery because of concerns about model errors.  We suggest how to exploit comparative models for mol. screens, based on docking against a wide range of crystallog. structures and comparative models with known ligands.  To account for the variation in the ligand-binding pocket as it binds different ligands, we calc. "consensus" enrichment by ranking each library compd. by its best docking score against all available comparative models and/or modeling templates.  For the majority of the targets, the consensus enrichment for multiple models was better than or comparable to that of the holo and apo X-ray structures.  Even for single models, the models are significantly more enriching than the template structure if the template is paralogous and shares more than 25% sequence identity with the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrgXhylX-BJrVg90H21EOLACvtfcHk0lj9p4vKqaIJ2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M&md5=11bd5284c6b43592906e45b6aea3019d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci9003706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003706%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DMolecular%2520Docking%2520Screens%2520Using%2520Comparative%2520Models%2520of%2520Proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26issue%3D11%26spage%3D2512%26epage%3D2527%26doi%3D10.1021%2Fci9003706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carchia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">6582</span><span class="NLM_x">–</span> <span class="NLM_lpage">6594</span><span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&issue=14&author=M.+M.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+Useful+Decoys%2C+Enhanced+%28DUD-E%29%3A+Better+Ligands+and+Decoys+for+Better+Benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0lj9p4vKqaIJ2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520Useful%2520Decoys%252C%2520Enhanced%2520%2528DUD-E%2529%253A%2520Better%2520Ligands%2520and%2520Decoys%2520for%2520Better%2520Benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D14%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—A Free Database of Commercially Available Compounds for Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&issue=1&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94A+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0lhhcwYxFn7jBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594A%2520Free%2520Database%2520of%2520Commercially%2520Available%2520Compounds%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26issue%3D1%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) From Screening Libraries and for Their Exclusion in Bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+From+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhhcwYxFn7jBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520From%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Cheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">FSelector: a Ruby Gem for Feature Selection</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">21</span><span class="NLM_x">) </span> <span class="NLM_fpage">2851</span><span class="NLM_x">–</span> <span class="NLM_lpage">2852</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bts528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fbioinformatics%2Fbts528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=22942017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGjsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=2851-2852&issue=21&author=T.+Chengauthor=Y.+Wangauthor=S.+H.+Bryant&title=FSelector%3A+a+Ruby+Gem+for+Feature+Selection&doi=10.1093%2Fbioinformatics%2Fbts528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">FSelector: a Ruby gem for feature selection</span></div><div class="casAuthors">Cheng, Tiejun; Wang, Yanli; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2851-2852</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: The FSelector package contains a comprehensive list of feature selection algorithms for supporting bioinformatics and machine learning research.  FSelector primarily collects and implements the filter type of feature selection techniques, which are computationally efficient for mining large datasets.  In particular, FSelector allows ensemble feature selection that takes advantage of multiple feature selection algorithms to yield more robust results.  FSelector also provides many useful auxiliary tools, including normalization, discretization and missing data imputation.  Availability: FSelector, written in the Ruby programming language, is free and open-source software that runs on all Ruby supporting platforms, including Windows, Linux and Mac OS X. FSelector is available from https://rubygems.org/gems/fselector and can be installed like a breeze via the command gem install f selector.  The source code is available (https://github.com/need47/fselector) and is fully documented (http://rubydoc.info/gems/fselector/frames).  Contact: ywang cbi.nlm.nih.gov or bryant cbi.nlm.nih.gov Supplementary Information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhXzhX-g7Q7LVg90H21EOLACvtfcHk0lhhcwYxFn7jBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGjsr3L&md5=72fd6933b2f959b534ca1f634d1500e5</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts528%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DFSelector%253A%2520a%2520Ruby%2520Gem%2520for%2520Feature%2520Selection%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26issue%3D21%26spage%3D2851%26epage%3D2852%26doi%3D10.1093%2Fbioinformatics%2Fbts528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">R Development Core Team.</span><span class="NLM_x"> </span> <span class="citation_source-book">R: A Language and Environment for Statistical Computing</span>; <span class="NLM_publisher-name">The R Foundation for Statistical Computing</span>: <span class="NLM_publisher-loc">Vienna, Austria</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>available online at <a href="http://www.R-project.org/" class="extLink">http://www.R-project.org/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=R+Development+Core+Team.&title=R%3A+A+Language+and+Environment+for+Statistical+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DR%253A%2520A%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DThe%2520R%2520Foundation%2520for%2520Statistical%2520Computing%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Ito, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiba, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horikawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishigami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gotoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatsubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanamitsu, S.-I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahara, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niinuma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukamoto, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, Y.</span><span> </span><span class="NLM_article-title">Which Concentration of the Inhibitor Should Be Used to Predict in Vivo Drug Interactions From in Vitro Data?</span> <span class="citation_source-journal">AAPS PharmSci</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1208/ps040425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1208%2Fps040425" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=53-60&issue=4&author=K.+Itoauthor=K.+Chibaauthor=M.+Horikawaauthor=M.+Ishigamiauthor=N.+Mizunoauthor=J.+Aokiauthor=Y.+Gotohauthor=T.+Iwatsuboauthor=S.-I.+Kanamitsuauthor=M.+Katoauthor=I.+Kawaharaauthor=K.+Niinumaauthor=A.+Nishinoauthor=N.+Satoauthor=Y.+Tsukamotoauthor=K.+Uedaauthor=T.+Itohauthor=Y.+Sugiyama&title=Which+Concentration+of+the+Inhibitor+Should+Be+Used+to+Predict+in+Vivo+Drug+Interactions+From+in+Vitro+Data%3F&doi=10.1208%2Fps040425"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1208%2Fps040425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fps040425%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DK.%26aulast%3DHorikawa%26aufirst%3DM.%26aulast%3DIshigami%26aufirst%3DM.%26aulast%3DMizuno%26aufirst%3DN.%26aulast%3DAoki%26aufirst%3DJ.%26aulast%3DGotoh%26aufirst%3DY.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DKanamitsu%26aufirst%3DS.-I.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKawahara%26aufirst%3DI.%26aulast%3DNiinuma%26aufirst%3DK.%26aulast%3DNishino%26aufirst%3DA.%26aulast%3DSato%26aufirst%3DN.%26aulast%3DTsukamoto%26aufirst%3DY.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DY.%26atitle%3DWhich%2520Concentration%2520of%2520the%2520Inhibitor%2520Should%2520Be%2520Used%2520to%2520Predict%2520in%2520Vivo%2520Drug%2520Interactions%2520From%2520in%2520Vitro%2520Data%253F%26jtitle%3DAAPS%2520PharmSci%26date%3D2002%26volume%3D4%26issue%3D4%26spage%3D53%26epage%3D60%26doi%3D10.1208%2Fps040425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref56':['cit56'],'ref55':['cit55'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 28 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ole Jensen, Jürgen Brockmöller, <span class="NLM_string-name hlFld-ContribAuthor">Christof Dücker</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Novel High-Affinity Substrates of OCT1 Using Machine Learning-Guided Virtual Screening and Experimental Validation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2762-2776. <a href="https://doi.org/10.1021/acs.jmedchem.0c02047" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02047%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BNovel%252BHigh-Affinity%252BSubstrates%252Bof%252BOCT1%252BUsing%252BMachine%252BLearning-Guided%252BVirtual%252BScreening%252Band%252BExperimental%252BValidation%26aulast%3DJensen%26aufirst%3DOle%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D25112020%26date%3D19022021%26volume%3D64%26issue%3D5%26spage%3D2762%26epage%3D2776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marleen
J. Meyer, Viktoria E. Neumann, Hannah Rosa Friesacher, Barbara Zdrazil, Jürgen Brockmöller, <span class="NLM_string-name hlFld-ContribAuthor">Mladen V. Tzvetkov</span>. </span><span class="cited-content_cbyCitation_article-title">Opioids as Substrates and Inhibitors of the Genetically Highly Variable Organic Cation Transporter OCT1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9890-9905. <a href="https://doi.org/10.1021/acs.jmedchem.9b01301" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01301%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOpioids%252Bas%252BSubstrates%252Band%252BInhibitors%252Bof%252Bthe%252BGenetically%252BHighly%252BVariable%252BOrganic%252BCation%252BTransporter%252BOCT1%26aulast%3DMeyer%26aufirst%3DMarleen%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09082019%26date%3D04112019%26date%3D09102019%26volume%3D62%26issue%3D21%26spage%3D9890%26epage%3D9905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia  Khuri</span>, <span class="hlFld-ContribAuthor ">Arik A.  Zur</span>, <span class="hlFld-ContribAuthor ">Matthias B.  Wittwer</span>, <span class="hlFld-ContribAuthor ">Lawrence  Lin</span>, <span class="hlFld-ContribAuthor ">Sook Wah  Yee</span>, <span class="hlFld-ContribAuthor ">Andrej  Sali</span>, and <span class="hlFld-ContribAuthor ">Kathleen M.  Giacomini</span>  . </span><span class="cited-content_cbyCitation_article-title">Computational Discovery and Experimental Validation of Inhibitors of the Human Intestinal Transporter OATP2B1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (6)
                                     , 1402-1413. <a href="https://doi.org/10.1021/acs.jcim.6b00720" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00720</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00720%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComputational%252BDiscovery%252Band%252BExperimental%252BValidation%252Bof%252BInhibitors%252Bof%252Bthe%252BHuman%252BIntestinal%252BTransporter%252BOATP2B1%26aulast%3DKhuri%26aufirst%3DNatalia%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D28112016%26date%3D15062017%26date%3D26062017%26date%3D31052017%26volume%3D57%26issue%3D6%26spage%3D1402%26epage%3D1413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sook Wah  Yee</span>, <span class="hlFld-ContribAuthor ">Bianca  Vora</span>, <span class="hlFld-ContribAuthor ">Tomiko  Oskotsky</span>, <span class="hlFld-ContribAuthor ">Ling  Zou</span>, <span class="hlFld-ContribAuthor ">Sebastian  Jakobsen</span>, <span class="hlFld-ContribAuthor ">Osatohanmwen J.  Enogieru</span>, <span class="hlFld-ContribAuthor ">Megan L.  Koleske</span>, <span class="hlFld-ContribAuthor ">Idit  Kosti</span>, <span class="hlFld-ContribAuthor ">Mattias  Rödin</span>, <span class="hlFld-ContribAuthor ">Marina  Sirota</span>, <span class="hlFld-ContribAuthor ">Kathleen M.  Giacomini</span>. </span><span class="cited-content_cbyCitation_article-title">Drugs in COVID‐19 Clinical Trials: Predicting Transporter‐Mediated Drug‐Drug Interactions Using In Vitro Assays and Real‐World Data. </span><span class="cited-content_cbyCitation_journal-name">Clinical Pharmacology & Therapeutics</span><span> <strong>2021,</strong> <em>110 </em>
                                    (1)
                                     , 108-122. <a href="https://doi.org/10.1002/cpt.2236" title="DOI URL">https://doi.org/10.1002/cpt.2236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpt.2236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpt.2236%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Pharmacology%2520%2526%2520Therapeutics%26atitle%3DDrugs%252Bin%252BCOVID%2525E2%252580%25259019%252BClinical%252BTrials%25253A%252BPredicting%252BTransporter%2525E2%252580%252590Mediated%252BDrug%2525E2%252580%252590Drug%252BInteractions%252BUsing%252BIn%252BVitro%252BAssays%252Band%252BReal%2525E2%252580%252590World%252BData%26aulast%3DYee%26aufirst%3DSook%2BWah%26date%3D2021%26date%3D2021%26volume%3D110%26issue%3D1%26spage%3D108%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hermann  Koepsell</span>. </span><span class="cited-content_cbyCitation_article-title">Update on drug-drug interaction at organic cation transporters: mechanisms, clinical impact, and proposal for advanced
              in vitro
              testing. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Metabolism & Toxicology</span><span> <strong>2021,</strong> <em>17 </em>
                                    (6)
                                     , 635-653. <a href="https://doi.org/10.1080/17425255.2021.1915284" title="DOI URL">https://doi.org/10.1080/17425255.2021.1915284</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17425255.2021.1915284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17425255.2021.1915284%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Metabolism%2520%2526%2520Toxicology%26atitle%3DUpdate%252Bon%252Bdrug-drug%252Binteraction%252Bat%252Borganic%252Bcation%252Btransporters%25253A%252Bmechanisms%25252C%252Bclinical%252Bimpact%25252C%252Band%252Bproposal%252Bfor%252Badvanced%252Bin%252Bvitro%252Btesting%26aulast%3DKoepsell%26aufirst%3DHermann%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D6%26spage%3D635%26epage%3D653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marleen J.  Meyer</span>, <span class="hlFld-ContribAuthor ">Mladen V.  Tzvetkov</span>. </span><span class="cited-content_cbyCitation_article-title">OCT1 Polyspecificity—Friend or Foe?. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.698153" title="DOI URL">https://doi.org/10.3389/fphar.2021.698153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.698153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.698153%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DOCT1%252BPolyspecificity%2525E2%252580%252594Friend%252Bor%252BFoe%25253F%26aulast%3DMeyer%26aufirst%3DMarleen%2BJ.%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bridget L.  Morse</span>, <span class="hlFld-ContribAuthor ">John K.  Fallon</span>, <span class="hlFld-ContribAuthor ">Anil  Kolur</span>, <span class="hlFld-ContribAuthor ">Andrew T.  Hogan</span>, <span class="hlFld-ContribAuthor ">Philip C.  Smith</span>, <span class="hlFld-ContribAuthor ">Kathleen M.  Hillgren</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of Hepatic Transporter Tissue Expression in Rodents and Interspecies Hepatic OCT1 Activity. </span><span class="cited-content_cbyCitation_journal-name">The AAPS Journal</span><span> <strong>2021,</strong> <em>23 </em>
                                    (3)
                                     <a href="https://doi.org/10.1208/s12248-021-00583-z" title="DOI URL">https://doi.org/10.1208/s12248-021-00583-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1208/s12248-021-00583-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1208%2Fs12248-021-00583-z%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520AAPS%2520Journal%26atitle%3DComparison%252Bof%252BHepatic%252BTransporter%252BTissue%252BExpression%252Bin%252BRodents%252Band%252BInterspecies%252BHepatic%252BOCT1%252BActivity%26aulast%3DMorse%26aufirst%3DBridget%2BL.%26date%3D2021%26date%3D2021%26volume%3D23%26issue%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bastian  Haberkorn</span>, <span class="hlFld-ContribAuthor ">Martin F.  Fromm</span>, <span class="hlFld-ContribAuthor ">Jörg  König</span>. </span><span class="cited-content_cbyCitation_article-title">Transport of Drugs and Endogenous Compounds Mediated by Human OCT1: Studies in Single- and Double-Transfected Cell Models. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.662535" title="DOI URL">https://doi.org/10.3389/fphar.2021.662535</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.662535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.662535%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DTransport%252Bof%252BDrugs%252Band%252BEndogenous%252BCompounds%252BMediated%252Bby%252BHuman%252BOCT1%25253A%252BStudies%252Bin%252BSingle-%252Band%252BDouble-Transfected%252BCell%252BModels%26aulast%3DHaberkorn%26aufirst%3DBastian%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shiwei  Zhou</span>, <span class="hlFld-ContribAuthor ">Sujuan  Zeng</span>, <span class="hlFld-ContribAuthor ">Yan  Shu</span>. </span><span class="cited-content_cbyCitation_article-title">Drug-Drug Interactions at Organic Cation Transporter 1. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.628705" title="DOI URL">https://doi.org/10.3389/fphar.2021.628705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.628705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.628705%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DDrug-Drug%252BInteractions%252Bat%252BOrganic%252BCation%252BTransporter%252B1%26aulast%3DZhou%26aufirst%3DShiwei%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leydy  Cano</span>, <span class="hlFld-ContribAuthor ">Alejandro  Soto-Ospina</span>, <span class="hlFld-ContribAuthor ">Pedronel  Araque</span>, <span class="hlFld-ContribAuthor ">Maria Antonieta  Caro-Gomez</span>, <span class="hlFld-ContribAuthor ">Maria Victoria  Parra-Marin</span>, <span class="hlFld-ContribAuthor ">Gabriel  Bedoya</span>, <span class="hlFld-ContribAuthor ">Constanza  Duque</span>. </span><span class="cited-content_cbyCitation_article-title">Diffusion Mechanism Modeling of Metformin in Human Organic Cationic Amino Acid Transporter one and Functional Impact of S189L, R206C, and G401S Mutation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.587590" title="DOI URL">https://doi.org/10.3389/fphar.2020.587590</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.587590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.587590%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DDiffusion%252BMechanism%252BModeling%252Bof%252BMetformin%252Bin%252BHuman%252BOrganic%252BCationic%252BAmino%252BAcid%252BTransporter%252Bone%252Band%252BFunctional%252BImpact%252Bof%252BS189L%25252C%252BR206C%25252C%252Band%252BG401S%252BMutation%26aulast%3DCano%26aufirst%3DLeydy%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marleen J.  Meyer</span>, <span class="hlFld-ContribAuthor ">Alzbeta  Tuerkova</span>, <span class="hlFld-ContribAuthor ">Sarah  Römer</span>, <span class="hlFld-ContribAuthor ">Christoph  Wenzel</span>, <span class="hlFld-ContribAuthor ">Tina  Seitz</span>, <span class="hlFld-ContribAuthor ">Jochen  Gaedcke</span>, <span class="hlFld-ContribAuthor ">Stefan  Oswald</span>, <span class="hlFld-ContribAuthor ">Jürgen  Brockmöller</span>, <span class="hlFld-ContribAuthor ">Barbara  Zdrazil</span>, <span class="hlFld-ContribAuthor ">Mladen V.  Tzvetkov</span>. </span><span class="cited-content_cbyCitation_article-title">Differences in Metformin and Thiamine Uptake between Human and Mouse Organic Cation Transporter 1: Structural Determinants and Potential Consequences for Intrahepatic Concentrations. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2020,</strong> <em>48 </em>
                                    (12)
                                     , 1380-1392. <a href="https://doi.org/10.1124/dmd.120.000170" title="DOI URL">https://doi.org/10.1124/dmd.120.000170</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.120.000170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.120.000170%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DDifferences%252Bin%252BMetformin%252Band%252BThiamine%252BUptake%252Bbetween%252BHuman%252Band%252BMouse%252BOrganic%252BCation%252BTransporter%252B1%25253A%252BStructural%252BDeterminants%252Band%252BPotential%252BConsequences%252Bfor%252BIntrahepatic%252BConcentrations%26aulast%3DMeyer%26aufirst%3DMarleen%2BJ.%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D12%26spage%3D1380%26epage%3D1392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Metee  Jinakote</span>, <span class="hlFld-ContribAuthor ">Atcharaporn  Ontawong</span>, <span class="hlFld-ContribAuthor ">Sunhapas  Soodvilai</span>, <span class="hlFld-ContribAuthor ">Jeerawat  Pimta</span>, <span class="hlFld-ContribAuthor ">Tipthida  Pasachan</span>, <span class="hlFld-ContribAuthor ">Varanuj  Chatsudthipong</span>, <span class="hlFld-ContribAuthor ">Chutima  Srimaroeng</span>. </span><span class="cited-content_cbyCitation_article-title">High affinity of 4‐(4‐(dimethylamino)styryl)‐
              N
              ‐methylpyridinium transport for assessing organic cation drugs in hepatocellular carcinoma cells. </span><span class="cited-content_cbyCitation_journal-name">Fundamental & Clinical Pharmacology</span><span> <strong>2020,</strong> <em>34 </em>
                                    (3)
                                     , 365-379. <a href="https://doi.org/10.1111/fcp.12531" title="DOI URL">https://doi.org/10.1111/fcp.12531</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/fcp.12531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffcp.12531%26sid%3Dliteratum%253Aachs%26jtitle%3DFundamental%2520%2526%2520Clinical%2520Pharmacology%26atitle%3DHigh%252Baffinity%252Bof%252B4%2525E2%252580%252590%2525284%2525E2%252580%252590%252528dimethylamino%252529styryl%252529%2525E2%252580%252590%252BN%252B%2525E2%252580%252590methylpyridinium%252Btransport%252Bfor%252Bassessing%252Borganic%252Bcation%252Bdrugs%252Bin%252Bhepatocellular%252Bcarcinoma%252Bcells%26aulast%3DJinakote%26aufirst%3DMetee%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D3%26spage%3D365%26epage%3D379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia  Khuri</span>. </span><span class="cited-content_cbyCitation_article-title">Mining environmental chemicals with boosted trees. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 1082-1089. <a href="https://doi.org/10.1145/3341105.3373897" title="DOI URL">https://doi.org/10.1145/3341105.3373897</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1145/3341105.3373897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1145%2F3341105.3373897%26sid%3Dliteratum%253Aachs%26atitle%3DMining%252Benvironmental%252Bchemicals%252Bwith%252Bboosted%252Btrees%26aulast%3DKhuri%26aufirst%3DNatalia%26date%3D2020%26date%3D2020%26spage%3D1082%26epage%3D1089%26pub%3DACM%26atitle%3DProceedings%252Bof%252Bthe%252B35th%252BAnnual%252BACM%252BSymposium%252Bon%252BApplied%252BComputing%26aulast%3DHung%26aufirst%3DChih-Cheng%26date%3D2020%26volume%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anurag T. K.  Baidya</span>, <span class="hlFld-ContribAuthor ">Kalyan  Ghosh</span>, <span class="hlFld-ContribAuthor ">Sk. Abdul  Amin</span>, <span class="hlFld-ContribAuthor ">Nilanjan  Adhikari</span>, <span class="hlFld-ContribAuthor ">Jayabalan  Nirmal</span>, <span class="hlFld-ContribAuthor ">Tarun  Jha</span>, <span class="hlFld-ContribAuthor ">Shovanlal  Gayen</span>. </span><span class="cited-content_cbyCitation_article-title">In silico
              modelling, identification of crucial molecular fingerprints, and prediction of new possible substrates of human organic cationic transporters 1 and 2. </span><span class="cited-content_cbyCitation_journal-name">New Journal of Chemistry</span><span> <strong>2020,</strong> <em>44 </em>
                                    (10)
                                     , 4129-4143. <a href="https://doi.org/10.1039/C9NJ05825G" title="DOI URL">https://doi.org/10.1039/C9NJ05825G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9NJ05825G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9NJ05825G%26sid%3Dliteratum%253Aachs%26jtitle%3DNew%2520Journal%2520of%2520Chemistry%26atitle%3DIn%252Bsilico%252Bmodelling%25252C%252Bidentification%252Bof%252Bcrucial%252Bmolecular%252Bfingerprints%25252C%252Band%252Bprediction%252Bof%252Bnew%252Bpossible%252Bsubstrates%252Bof%252Bhuman%252Borganic%252Bcationic%252Btransporters%252B1%252Band%252B2%26aulast%3DBaidya%26aufirst%3DAnurag%2BT.%2BK.%26date%3D2020%26date%3D2020%26volume%3D44%26issue%3D10%26spage%3D4129%26epage%3D4143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ole  Jensen</span>, <span class="hlFld-ContribAuthor ">Johannes  Matthaei</span>, <span class="hlFld-ContribAuthor ">Felix  Blome</span>, <span class="hlFld-ContribAuthor ">Matthias  Schwab</span>, <span class="hlFld-ContribAuthor ">Mladen V.  Tzvetkov</span>, <span class="hlFld-ContribAuthor ">Jürgen  Brockmöller</span>. </span><span class="cited-content_cbyCitation_article-title">Variability and Heritability of Thiamine Pharmacokinetics With Focus on OCT1 Effects on Membrane Transport and Pharmacokinetics in Humans. </span><span class="cited-content_cbyCitation_journal-name">Clinical Pharmacology & Therapeutics</span><span> <strong>2020,</strong> <em>107 </em>
                                    (3)
                                     , 628-638. <a href="https://doi.org/10.1002/cpt.1666" title="DOI URL">https://doi.org/10.1002/cpt.1666</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpt.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpt.1666%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Pharmacology%2520%2526%2520Therapeutics%26atitle%3DVariability%252Band%252BHeritability%252Bof%252BThiamine%252BPharmacokinetics%252BWith%252BFocus%252Bon%252BOCT1%252BEffects%252Bon%252BMembrane%252BTransport%252Band%252BPharmacokinetics%252Bin%252BHumans%26aulast%3DJensen%26aufirst%3DOle%26date%3D2020%26date%3D2019%26volume%3D107%26issue%3D3%26spage%3D628%26epage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hermann  Koepsell</span>, . </span><span class="cited-content_cbyCitation_article-title">Organic Cation Transporters in Health and Disease. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Reviews</span><span> <strong>2020,</strong> <em>72 </em>
                                    (1)
                                     , 253-319. <a href="https://doi.org/10.1124/pr.118.015578" title="DOI URL">https://doi.org/10.1124/pr.118.015578</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/pr.118.015578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fpr.118.015578%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Reviews%26atitle%3DOrganic%252BCation%252BTransporters%252Bin%252BHealth%252Band%252BDisease%26aulast%3DKoepsell%26aufirst%3DHermann%26date%3D2020%26date%3D2019%26volume%3D72%26issue%3D1%26spage%3D253%26epage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel-Ann A.  Garibsingh</span>, <span class="hlFld-ContribAuthor ">Avner  Schlessinger</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Challenges in Rational Drug Design for SLCs. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2019,</strong> <em>40 </em>
                                    (10)
                                     , 790-800. <a href="https://doi.org/10.1016/j.tips.2019.08.006" title="DOI URL">https://doi.org/10.1016/j.tips.2019.08.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2019.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2019.08.006%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DAdvances%252Band%252BChallenges%252Bin%252BRational%252BDrug%252BDesign%252Bfor%252BSLCs%26aulast%3DGaribsingh%26aufirst%3DRachel-Ann%2BA.%26date%3D2019%26volume%3D40%26issue%3D10%26spage%3D790%26epage%3D800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongshuo  Jiang</span>, <span class="hlFld-ContribAuthor ">Guijie  Mao</span>, <span class="hlFld-ContribAuthor ">Hongfeng  Wu</span>, <span class="hlFld-ContribAuthor ">Qi  An</span>, <span class="hlFld-ContribAuthor ">Minghui  Zuo</span>, <span class="hlFld-ContribAuthor ">Weihao  Guo</span>, <span class="hlFld-ContribAuthor ">Chunzhao  Xu</span>, <span class="hlFld-ContribAuthor ">Zhizhong  Sun</span>, <span class="hlFld-ContribAuthor ">Wenyi  Chu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of dibenzocycloketones by acyl radical cyclization from aromatic carboxylic acids using methylene blue as a photocatalyst. </span><span class="cited-content_cbyCitation_journal-name">Green Chemistry</span><span> <strong>2019,</strong> <em>21 </em>
                                    (19)
                                     , 5368-5373. <a href="https://doi.org/10.1039/C9GC02380A" title="DOI URL">https://doi.org/10.1039/C9GC02380A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9GC02380A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9GC02380A%26sid%3Dliteratum%253Aachs%26jtitle%3DGreen%2520Chemistry%26atitle%3DSynthesis%252Bof%252Bdibenzocycloketones%252Bby%252Bacyl%252Bradical%252Bcyclization%252Bfrom%252Baromatic%252Bcarboxylic%252Bacids%252Busing%252Bmethylene%252Bblue%252Bas%252Ba%252Bphotocatalyst%26aulast%3DJiang%26aufirst%3DHongshuo%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D19%26spage%3D5368%26epage%3D5373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sook Wah  Yee</span>, <span class="hlFld-ContribAuthor ">Adrian  Stecula</span>, <span class="hlFld-ContribAuthor ">Huan-Chieh  Chien</span>, <span class="hlFld-ContribAuthor ">Ling  Zou</span>, <span class="hlFld-ContribAuthor ">Elena V.  Feofanova</span>, <span class="hlFld-ContribAuthor ">Marjolein  van Borselen</span>, <span class="hlFld-ContribAuthor ">Kit Wun Kathy  Cheung</span>, <span class="hlFld-ContribAuthor ">Noha A.  Yousri</span>, <span class="hlFld-ContribAuthor ">Karsten  Suhre</span>, <span class="hlFld-ContribAuthor ">Jason M.  Kinchen</span>, <span class="hlFld-ContribAuthor ">Eric  Boerwinkle</span>, <span class="hlFld-ContribAuthor ">Roshanak  Irannejad</span>, <span class="hlFld-ContribAuthor ">Bing  Yu</span>, <span class="hlFld-ContribAuthor ">Kathleen M.  Giacomini</span>, . </span><span class="cited-content_cbyCitation_article-title">Unraveling the functional role of the orphan solute carrier, SLC22A24 in the transport of steroid conjugates through metabolomic and genome-wide association studies. </span><span class="cited-content_cbyCitation_journal-name">PLOS Genetics</span><span> <strong>2019,</strong> <em>15 </em>
                                    (9)
                                     , e1008208. <a href="https://doi.org/10.1371/journal.pgen.1008208" title="DOI URL">https://doi.org/10.1371/journal.pgen.1008208</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pgen.1008208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pgen.1008208%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Genetics%26atitle%3DUnraveling%252Bthe%252Bfunctional%252Brole%252Bof%252Bthe%252Borphan%252Bsolute%252Bcarrier%25252C%252BSLC22A24%252Bin%252Bthe%252Btransport%252Bof%252Bsteroid%252Bconjugates%252Bthrough%252Bmetabolomic%252Band%252Bgenome-wide%252Bassociation%252Bstudies%26aulast%3DYee%26aufirst%3DSook%2BWah%26date%3D2019%26date%3D2019%26volume%3D15%26issue%3D9%26spage%3De1008208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thorsten  Keller</span>, <span class="hlFld-ContribAuthor ">Valentin  Gorboulev</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Mueller</span>, <span class="hlFld-ContribAuthor ">Volker  Dötsch</span>, <span class="hlFld-ContribAuthor ">Frank  Bernhard</span>, <span class="hlFld-ContribAuthor ">Hermann  Koepsell</span>. </span><span class="cited-content_cbyCitation_article-title">Rat Organic Cation Transporter 1 Contains Three Binding Sites for Substrate 1-Methyl-4-phenylpyridinium per Monomer. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2019,</strong> <em>95 </em>
                                    (2)
                                     , 169-182. <a href="https://doi.org/10.1124/mol.118.113498" title="DOI URL">https://doi.org/10.1124/mol.118.113498</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.118.113498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.118.113498%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DRat%252BOrganic%252BCation%252BTransporter%252B1%252BContains%252BThree%252BBinding%252BSites%252Bfor%252BSubstrate%252B1-Methyl-4-phenylpyridinium%252Bper%252BMonomer%26aulast%3DKeller%26aufirst%3DThorsten%26date%3D2019%26date%3D2018%26volume%3D95%26issue%3D2%26spage%3D169%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hermann  Koepsell</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple binding sites in organic cation transporters require sophisticated procedures to identify interactions of novel drugs. </span><span class="cited-content_cbyCitation_journal-name">Biological Chemistry</span><span> <strong>2019,</strong> <em>400 </em>
                                    (2)
                                     , 195-207. <a href="https://doi.org/10.1515/hsz-2018-0191" title="DOI URL">https://doi.org/10.1515/hsz-2018-0191</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1515/hsz-2018-0191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1515%2Fhsz-2018-0191%26sid%3Dliteratum%253Aachs%26jtitle%3DBiological%2520Chemistry%26atitle%3DMultiple%252Bbinding%252Bsites%252Bin%252Borganic%252Bcation%252Btransporters%252Brequire%252Bsophisticated%252Bprocedures%252Bto%252Bidentify%252Binteractions%252Bof%252Bnovel%252Bdrugs%26aulast%3DKoepsell%26aufirst%3DHermann%26date%3D2019%26date%3D2018%26volume%3D400%26issue%3D2%26spage%3D195%26epage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alžběta  Türková</span>, <span class="hlFld-ContribAuthor ">Barbara  Zdrazil</span>. </span><span class="cited-content_cbyCitation_article-title">Current Advances in Studying Clinically Relevant Transporters of the Solute Carrier (SLC) Family by Connecting Computational Modeling and Data Science. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2019,</strong> <em>17 </em>, 390-405. <a href="https://doi.org/10.1016/j.csbj.2019.03.002" title="DOI URL">https://doi.org/10.1016/j.csbj.2019.03.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2019.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2019.03.002%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DCurrent%252BAdvances%252Bin%252BStudying%252BClinically%252BRelevant%252BTransporters%252Bof%252Bthe%252BSolute%252BCarrier%252B%252528SLC%252529%252BFamily%252Bby%252BConnecting%252BComputational%252BModeling%252Band%252BData%252BScience%26aulast%3DT%25C3%25BCrkov%25C3%25A1%26aufirst%3DAl%25C5%25BEb%25C4%259Bta%26date%3D2019%26volume%3D17%26spage%3D390%26epage%3D405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Avner  Schlessinger</span>, <span class="hlFld-ContribAuthor ">Matthew A.  Welch</span>, <span class="hlFld-ContribAuthor ">Herman  van Vlijmen</span>, <span class="hlFld-ContribAuthor ">Ken  Korzekwa</span>, <span class="hlFld-ContribAuthor ">Peter W.  Swaan</span>, <span class="hlFld-ContribAuthor ">Pär  Matsson</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Modeling of Drug-Transporter Interactions-An International Transporter Consortium Perspective. </span><span class="cited-content_cbyCitation_journal-name">Clinical Pharmacology & Therapeutics</span><span> <strong>2018,</strong> <em>104 </em>
                                    (5)
                                     , 818-835. <a href="https://doi.org/10.1002/cpt.1174" title="DOI URL">https://doi.org/10.1002/cpt.1174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpt.1174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpt.1174%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Pharmacology%2520%2526%2520Therapeutics%26atitle%3DMolecular%252BModeling%252Bof%252BDrug-Transporter%252BInteractions-An%252BInternational%252BTransporter%252BConsortium%252BPerspective%26aulast%3DSchlessinger%26aufirst%3DAvner%26date%3D2018%26date%3D2018%26volume%3D104%26issue%3D5%26spage%3D818%26epage%3D835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kelli H.  Boxberger</span>, <span class="hlFld-ContribAuthor ">Bruno  Hagenbuch</span>, <span class="hlFld-ContribAuthor ">Jed N.  Lampe</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-dependent modulation of hOCT1 transport reveals discrete ligand binding sites within the substrate translocation channel. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2018,</strong> <em>156 </em>, 371-384. <a href="https://doi.org/10.1016/j.bcp.2018.08.028" title="DOI URL">https://doi.org/10.1016/j.bcp.2018.08.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2018.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2018.08.028%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DLigand-dependent%252Bmodulation%252Bof%252BhOCT1%252Btransport%252Breveals%252Bdiscrete%252Bligand%252Bbinding%252Bsites%252Bwithin%252Bthe%252Bsubstrate%252Btranslocation%252Bchannel%26aulast%3DBoxberger%26aufirst%3DKelli%2BH.%26date%3D2018%26volume%3D156%26spage%3D371%26epage%3D384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lisa  Chedik</span>, <span class="hlFld-ContribAuthor ">Arnaud  Bruyere</span>, <span class="hlFld-ContribAuthor ">Astrid  Bacle</span>, <span class="hlFld-ContribAuthor ">Sophie  Potin</span>, <span class="hlFld-ContribAuthor ">Marc  Le Vée</span>, <span class="hlFld-ContribAuthor ">Olivier  Fardel</span>. </span><span class="cited-content_cbyCitation_article-title">Interactions of pesticides with membrane drug transporters: implications for toxicokinetics and toxicity. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Metabolism & Toxicology</span><span> <strong>2018,</strong> <em>14 </em>
                                    (7)
                                     , 739-752. <a href="https://doi.org/10.1080/17425255.2018.1487398" title="DOI URL">https://doi.org/10.1080/17425255.2018.1487398</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17425255.2018.1487398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17425255.2018.1487398%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Metabolism%2520%2526%2520Toxicology%26atitle%3DInteractions%252Bof%252Bpesticides%252Bwith%252Bmembrane%252Bdrug%252Btransporters%25253A%252Bimplications%252Bfor%252Btoxicokinetics%252Band%252Btoxicity%26aulast%3DChedik%26aufirst%3DLisa%26date%3D2018%26date%3D2018%26volume%3D14%26issue%3D7%26spage%3D739%26epage%3D752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentin  Gorboulev</span>, <span class="hlFld-ContribAuthor ">Saba  Rehman</span>, <span class="hlFld-ContribAuthor ">Christoph M.  Albert</span>, <span class="hlFld-ContribAuthor ">Ursula  Roth</span>, <span class="hlFld-ContribAuthor ">Marleen J.  Meyer</span>, <span class="hlFld-ContribAuthor ">Mladen V.  Tzvetkov</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Mueller</span>, <span class="hlFld-ContribAuthor ">Hermann  Koepsell</span>. </span><span class="cited-content_cbyCitation_article-title">Assay Conditions Influence Affinities of Rat Organic Cation Transporter 1: Analysis of Mutagenesis in the Modeled Outward-Facing Cleft by Measuring Effects of Substrates and Inhibitors on Initial Uptake. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2018,</strong> <em>93 </em>
                                    (4)
                                     , 402-415. <a href="https://doi.org/10.1124/mol.117.110767" title="DOI URL">https://doi.org/10.1124/mol.117.110767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.117.110767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.117.110767%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DAssay%252BConditions%252BInfluence%252BAffinities%252Bof%252BRat%252BOrganic%252BCation%252BTransporter%252B1%25253A%252BAnalysis%252Bof%252BMutagenesis%252Bin%252Bthe%252BModeled%252BOutward-Facing%252BCleft%252Bby%252BMeasuring%252BEffects%252Bof%252BSubstrates%252Band%252BInhibitors%252Bon%252BInitial%252BUptake%26aulast%3DGorboulev%26aufirst%3DValentin%26date%3D2018%26date%3D2018%26volume%3D93%26issue%3D4%26spage%3D402%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Balakumar  Chandrasekaran</span>, <span class="hlFld-ContribAuthor ">Sara Nidal  Abed</span>, <span class="hlFld-ContribAuthor ">Omar  Al-Attraqchi</span>, <span class="hlFld-ContribAuthor ">Kaushik  Kuche</span>, <span class="hlFld-ContribAuthor ">Rakesh K.  Tekade</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Prediction of Pharmacokinetic (ADMET) Properties. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 731-755. <a href="https://doi.org/10.1016/B978-0-12-814421-3.00021-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-814421-3.00021-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-814421-3.00021-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-814421-3.00021-X%26sid%3Dliteratum%253Aachs%26atitle%3DComputer-Aided%252BPrediction%252Bof%252BPharmacokinetic%252B%252528ADMET%252529%252BProperties%26aulast%3DChandrasekaran%26aufirst%3DBalakumar%26date%3D2018%26spage%3D731%26epage%3D755%26pub%3DElsevier%26atitle%3DDosage%252BForm%252BDesign%252BParameters%26date%3D2018%26volume%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marleen Julia  Meyer</span>, <span class="hlFld-ContribAuthor ">Tina  Seitz</span>, <span class="hlFld-ContribAuthor ">Jürgen  Brockmöller</span>, <span class="hlFld-ContribAuthor ">Mladen Vassilev  Tzvetkov</span>, . </span><span class="cited-content_cbyCitation_article-title">Effects of genetic polymorphisms on the OCT1 and OCT2-mediated uptake of ranitidine. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (12)
                                     , e0189521. <a href="https://doi.org/10.1371/journal.pone.0189521" title="DOI URL">https://doi.org/10.1371/journal.pone.0189521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0189521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0189521%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DEffects%252Bof%252Bgenetic%252Bpolymorphisms%252Bon%252Bthe%252BOCT1%252Band%252BOCT2-mediated%252Buptake%252Bof%252Branitidine%26aulast%3DMeyer%26aufirst%3DMarleen%2BJulia%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D12%26spage%3De0189521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Predicted binding site of OCT1 and representative substrates. (A) Thirteen residues (S29, F32, C36, N156, Y221, F273, W354, Y361, I446, S358, C450, C473, and D474) formed hydrogen bonds with docked substrate molecules. Six residues in the predicted binding site (W217, F244, I449, D357, Q447, and G476) participated in non-covalent and/or polar substrate–transporter interactions. OCT1 residues are shown as cornflower blue sticks. Structures of 12 favorably docked known OCT1 substrates are shown as orange sticks. Oxygen, nitrogen, sulfur, and hydrogen atoms are depicted in red, dark blue, yellow, and white, respectively. (B) 2D structures of representative OCT1 substrates drawn using MarvinView 14.7.7.0 (ChemAxon).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Uptake of ASP<sup>+</sup> and HTS screening data. (A) Time-dependent ASP<sup>+</sup> uptake in HEK cells overexpressing OCT1 (●) or empty vector (■). (B) Overexpression of OCT1 increases ASP<sup>+</sup> uptake in HEK cells. ASP<sup>+</sup> uptake studies were conducted in HEK cells overexpressing OCT1 or empty vector. Cells were incubated with increasing concentrations of ASP<sup>+</sup> for 2 min. The uptake kinetic parameters were calculated using the difference in ASP<sup>+</sup> accumulation between cells overexpressing OCT1 and empty vector cells. Each data point represents mean ± SD, <i>n</i> = 6. (C) Distribution of inhibition values from HTS of 1780 compounds. A total of 167 inhibitors were identified among the 1780 Pharmakon compounds. (D) Distribution of the 167 inhibitors in various pharmacological classes. Brown color represents mixed class.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking results and physiochemical properties of OCT1 ligands in the Pharmakon library. Compounds are shown in orange sticks and hydrogen bonds as black dotted lines. (A) Predicted pose of metformin and its hydrogen bond with aspartic acid residue 474. (B) Predicted pose of cimetidine and its hydrogen bonds with D474 and S29. (C) Differences in distribution of physicochemical properties for predicted competitive (<i>n</i> = 30) and noncompetitive (<i>n</i> = 137) inhibitors of OCT1. Only significantly different distributions are shown (Student’s <i>t</i> test, <i>p</i> < 0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Selected inhibition studies of previously unknown OCT1 ligands and their estimated half-maximal inhibitory concentrations (IC<sub>50</sub>), with 95% confidence limits in parentheses. (A) Alfuzosin, an α<sub>1</sub>-adrenergic receptor antagonist. IC<sub>50</sub> = 14.9 (11.9; 18.5) μM. (B) Bithionol, an anthelmintic. IC<sub>50</sub> = 2.2 (2.0; 2.5) μM. (C) Camylofine, an antimuscarinic. IC<sub>50</sub> = 9.1 (8.0; 10.4) μM. (D) Carbetapentane, an antitussive. IC<sub>50</sub> = 1.6 (1.2; 2.0) μM. (E) Carvedilol, a nonselective β/α<sub>1</sub>-adrenergic receptor antagonist. IC<sub>50</sub> = 3.4 (3.0; 3.9) μM. (F) Dobutamine, a sympathomimetic. IC<sub>50</sub> = 4.17 (3.73; 4.67) μM. Each data point represents mean ± SD, <i>n</i> = 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Differences in physicochemical properties of 167 inhibitors and 1613 noninhibitors. Shown are box plots of molecular weight (MW), number of heavy atoms (HA), molecular volume (MV), number of hydrogen-bond donors (HBD), number of hydrogen-bond acceptors (HBA), number of rotatable bonds (RB), SLogP, total polar surface area (TPSA), and charge at pH 7.4. Statistically significant differences were estimated using Student’s <i>t</i> test. Distributions of inhibitors and noninhibitors were significantly different (<i>p</i> < 0.05) for HBD, HBA, SLogP, TPSA, and charge.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Lineweaver–Burk plots for discriminating between competitive and noncompetitive inhibitors of OCT1. The inhibitory effects on ASP<sup>+</sup> uptake by OCT1 with increasing concentrations of ASP<sup>+</sup> were measured at various concentrations (● < ■ < ▲ < ▼) of the following inhibitors: (A) tacrine; (B) ethopropazine; (C) clonidine; (D) pyrimethamine; (E) trimethoprim; (F) dichlorophene; (G) alfuzosin; (H) clotrimazole; (I) imipramine; (J) dextromethorphan; (K) propafenone; (L) carvedilol. Each data point represents mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Results of structure–activity relationship modeling. (A) ROC curves for 100 retrospective cross-validation runs. The average ROC curve is shown in black. The ROC curve of a random classifier is shown as a red dotted line. (B) ROC curve for the SAR model tested in prospective validation of 1780 predicted inhibition values. The performances at different cutoff values (shown on the right <i>y</i>-axis) are indicated by rainbow colors. (C) Accuracy of classification of 1780 compounds using the SAR-II model as a function of cutoff. A cutoff of 0.38 is indicated by the red dotted line. (D) Observed vs predicted inhibition values for 1780 compounds. Two classification cutoffs are drawn vertically, and the HTS classification cutoff is drawn horizontally.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/medium/jm-2016-01317c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibitory effects of 1BnTIQ on OCT1 transport. (A) 1BnTIQ inhibited ASP<sup>+</sup> uptake by OCT1, and the IC<sub>50</sub> was determined to be 82.1 μM. (B) The inhibitory effect of 1BnTIQ on ASP<sup>+</sup> uptake by OCT1 indicated competitive inhibition. Each data point represents mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01317/20170407/images/large/jm-2016-01317c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01317&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47770" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47770" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Boxberger, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagenbuch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampe, J. N.</span><span> </span><span class="NLM_article-title">Common Drugs Inhibit Human Organic Cation Transporter 1 (OCT1)-Mediated Neurotransmitter Uptake</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">990</span><span class="NLM_x">–</span> <span class="NLM_lpage">995</span><span class="refDoi"> DOI: 10.1124/dmd.113.055095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fdmd.113.055095" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=990-995&issue=6&author=K.+H.+Boxbergerauthor=B.+Hagenbuchauthor=J.+N.+Lampe&title=Common+Drugs+Inhibit+Human+Organic+Cation+Transporter+1+%28OCT1%29-Mediated+Neurotransmitter+Uptake&doi=10.1124%2Fdmd.113.055095"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fdmd.113.055095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.113.055095%26sid%3Dliteratum%253Aachs%26aulast%3DBoxberger%26aufirst%3DK.%2BH.%26aulast%3DHagenbuch%26aufirst%3DB.%26aulast%3DLampe%26aufirst%3DJ.%2BN.%26atitle%3DCommon%2520Drugs%2520Inhibit%2520Human%2520Organic%2520Cation%2520Transporter%25201%2520%2528OCT1%2529-Mediated%2520Neurotransmitter%2520Uptake%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2014%26volume%3D42%26issue%3D6%26spage%3D990%26epage%3D995%26doi%3D10.1124%2Fdmd.113.055095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zur, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keshari, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostrem, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Younger, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurhanewicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashrafi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">OCT1 Is a High-Capacity Thiamine Transporter That Regulates Hepatic Steatosis and Is a Target of Metformin</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x"> (</span><span class="NLM_issue">27</span><span class="NLM_x">) </span> <span class="NLM_fpage">9983</span><span class="NLM_x">–</span> <span class="NLM_lpage">9988</span><span class="refDoi"> DOI: 10.1073/pnas.1314939111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1073%2Fpnas.1314939111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=24961373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVamu7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=9983-9988&issue=27&author=L.+Chenauthor=Y.+Shuauthor=X.+Liangauthor=E.+C.+Chenauthor=S.+W.+Yeeauthor=A.+A.+Zurauthor=S.+Liauthor=L.+Xuauthor=K.+R.+Keshariauthor=M.+J.+Linauthor=H.-C.+Chienauthor=Y.+Zhangauthor=K.+M.+Morrisseyauthor=J.+Liuauthor=J.+Ostremauthor=N.+S.+Youngerauthor=J.+Kurhanewiczauthor=K.+M.+Shokatauthor=K.+Ashrafiauthor=K.+M.+Giacomini&title=OCT1+Is+a+High-Capacity+Thiamine+Transporter+That+Regulates+Hepatic+Steatosis+and+Is+a+Target+of+Metformin&doi=10.1073%2Fpnas.1314939111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">OCT1 is a high-capacity thiamine transporter that regulates hepatic steatosis and is a target of metformin</span></div><div class="casAuthors">Chen, Ligong; Shu, Yan; Liang, Xiaomin; Chen, Eugene C.; Yee, Sook Wah; Zur, Arik A.; Li, Shuanglian; Xu, Lu; Keshari, Kayvan R.; Lin, Michael J.; Chien, Huan-Chieh; Zhang, Youcai; Morrissey, Kari M.; Liu, Jason; Ostrem, Jonathan; Younger, Noah S.; Kurhanewicz, John; Shokat, Kevan M.; Ashrafi, Kaveh; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">9983-9988</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Org. cation transporter 1, OCT1 (SLC22A1), is the major hepatic uptake transporter for metformin, the most prescribed antidiabetic drug.  However, its endogenous role is poorly understood.  Here we show that similar to metformin treatment, loss of Oct1 caused an increase in the ratio of AMP to ATP, activated the energy sensor AMP-activated kinase (AMPK), and substantially reduced triglyceride (TG) levels in livers from healthy and leptin-deficient mice.  Conversely, livers of human OCT1 transgenic mice fed high-fat diets were enlarged with high TG levels.  Metabolomic and isotopic uptake methods identified thiamine as a principal endogenous substrate of OCT1.  Thiamine deficiency enhanced the phosphorylation of AMPK and its downstream target, acetyl-CoA carboxylase.  Metformin and the biguanide analog, phenformin, competitively inhibited OCT1-mediated thiamine uptake.  Acute administration of metformin to wild-type mice reduced intestinal accumulation of thiamine.  These findings suggest that OCT1 plays a role in hepatic steatosis through modulation of energy status.  The studies implicate OCT1 as well as metformin in thiamine disposition, suggesting an intriguing and parallel mechanism for metformin and its major hepatic transporter in metabolic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsCRGBQOEPerVg90H21EOLACvtfcHk0lgDi-XpIg0-ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVamu7nI&md5=660cd2ac8d797fd445f5997d48aa71f2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1314939111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1314939111%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DE.%2BC.%26aulast%3DYee%26aufirst%3DS.%2BW.%26aulast%3DZur%26aufirst%3DA.%2BA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DKeshari%26aufirst%3DK.%2BR.%26aulast%3DLin%26aufirst%3DM.%2BJ.%26aulast%3DChien%26aufirst%3DH.-C.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMorrissey%26aufirst%3DK.%2BM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DOstrem%26aufirst%3DJ.%26aulast%3DYounger%26aufirst%3DN.%2BS.%26aulast%3DKurhanewicz%26aufirst%3DJ.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DAshrafi%26aufirst%3DK.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DOCT1%2520Is%2520a%2520High-Capacity%2520Thiamine%2520Transporter%2520That%2520Regulates%2520Hepatic%2520Steatosis%2520and%2520Is%2520a%2520Target%2520of%2520Metformin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26issue%3D27%26spage%3D9983%26epage%3D9988%26doi%3D10.1073%2Fpnas.1314939111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kell, D. B.</span><span> </span><span class="NLM_article-title">Implications of Endogenous Roles of Transporters for Drug Discovery: Hitchhiking and Metabolite-Likeness</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">143</span><span class="refDoi"> DOI: 10.1038/nrd.2015.44</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnrd.2015.44" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=143&issue=2&author=D.+B.+Kell&title=Implications+of+Endogenous+Roles+of+Transporters+for+Drug+Discovery%3A+Hitchhiking+and+Metabolite-Likeness&doi=10.1038%2Fnrd.2015.44"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.44&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.44%26sid%3Dliteratum%253Aachs%26aulast%3DKell%26aufirst%3DD.%2BB.%26atitle%3DImplications%2520of%2520Endogenous%2520Roles%2520of%2520Transporters%2520for%2520Drug%2520Discovery%253A%2520Hitchhiking%2520and%2520Metabolite-Likeness%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26issue%3D2%26spage%3D143%26doi%3D10.1038%2Fnrd.2015.44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Shu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheardown, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ianculescu, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burchard, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brett, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Effect of Genetic Variation in the Organic Cation Transporter 1 (OCT1) on Metformin Action</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1422</span><span class="NLM_x">–</span> <span class="NLM_lpage">1431</span><span class="refDoi"> DOI: 10.1172/JCI30558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1172%2FJCI30558" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2007&pages=1422-1431&issue=5&author=Y.+Shuauthor=S.+A.+Sheardownauthor=C.+Brownauthor=R.+P.+Owenauthor=S.+Zhangauthor=R.+A.+Castroauthor=A.+G.+Ianculescuauthor=L.+Yueauthor=J.+C.+Loauthor=E.+G.+Burchardauthor=C.+M.+Brettauthor=K.+M.+Giacomini&title=Effect+of+Genetic+Variation+in+the+Organic+Cation+Transporter+1+%28OCT1%29+on+Metformin+Action&doi=10.1172%2FJCI30558"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1172%2FJCI30558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI30558%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DY.%26aulast%3DSheardown%26aufirst%3DS.%2BA.%26aulast%3DBrown%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DR.%2BP.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DCastro%26aufirst%3DR.%2BA.%26aulast%3DIanculescu%26aufirst%3DA.%2BG.%26aulast%3DYue%26aufirst%3DL.%26aulast%3DLo%26aufirst%3DJ.%2BC.%26aulast%3DBurchard%26aufirst%3DE.%2BG.%26aulast%3DBrett%26aufirst%3DC.%2BM.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DEffect%2520of%2520Genetic%2520Variation%2520in%2520the%2520Organic%2520Cation%2520Transporter%25201%2520%2528OCT1%2529%2520on%2520Metformin%2520Action%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2007%26volume%3D117%26issue%3D5%26spage%3D1422%26epage%3D1431%26doi%3D10.1172%2FJCI30558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Tzvetkov, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saadatmand, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lötsch, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tegeder, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stingl, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brockmöller, J.</span><span> </span><span class="NLM_article-title">Genetically Polymorphic OCT1: Another Piece in the Puzzle of the Variable Pharmacokinetics and Pharmacodynamics of the Opioidergic Drug Tramadol</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span><span class="refDoi"> DOI: 10.1038/clpt.2011.56</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fclpt.2011.56" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21562485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVaktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2011&pages=143-150&issue=1&author=M.+V.+Tzvetkovauthor=A.+R.+Saadatmandauthor=J.+L%C3%B6tschauthor=I.+Tegederauthor=J.+C.+Stinglauthor=J.+Brockm%C3%B6ller&title=Genetically+Polymorphic+OCT1%3A+Another+Piece+in+the+Puzzle+of+the+Variable+Pharmacokinetics+and+Pharmacodynamics+of+the+Opioidergic+Drug+Tramadol&doi=10.1038%2Fclpt.2011.56"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Genetically Polymorphic OCT1: Another Piece in the Puzzle of the Variable Pharmacokinetics and Pharmacodynamics of the Opioidergic Drug Tramadol</span></div><div class="casAuthors">Tzvetkov, M. V.; Saadatmand, A. R.; Loetsch, J.; Tegeder, I.; Stingl, J. C.; Brockmoeller, J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">143-150</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We investigated whether tramadol or its active metabolite, O-desmethyltramadol, are substrates of the org. cation transporter OCT1 and whether polymorphisms in OCT1 affect tramadol and O-desmethyltramadol pharmacokinetics.  Tramadol showed high permeability through parallel artificial membrane permeability assays (PAMPAs).  Tramadol uptake in HEK293 cells did not change after OCT1 overexpression, and the concns. of tramadol in the plasma of healthy volunteers were independent of their OCT1 genotypes.  In contrast, O-desmethyltramadol showed low membrane permeability, and OCT1 overexpression increased O-desmethyltramadol uptake 2.4-fold.  This increase in uptake was reversed by OCT1 inhibitors and absent when loss-of-function OCT1 variants were overexpressed.  Volunteers carrying loss-of-function OCT1 polymorphisms had significantly higher plasma concns. of O-desmethyltramadol (P = 0.002, n = 41) and significantly prolonged miosis, a surrogate marker of opioidergic effects (P = 0.005, n = 24).  In conclusion, polymorphisms in OCT1 influence the pharmacokinetics of O-desmethyltramadol, presumably by affecting its uptake into liver cells, and thus may modulate the efficacy of tramadol treatment.  Clin. Pharmacol. & Therapeutics (2011) 90 1, 143-150. doi:10.1038/clpt.2011.56.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUQAyYl097DrVg90H21EOLACvtfcHk0lhq2y2PKxTUOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVaktrk%253D&md5=55027b4c5ff59392d666d033baadf89b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2011.56&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2011.56%26sid%3Dliteratum%253Aachs%26aulast%3DTzvetkov%26aufirst%3DM.%2BV.%26aulast%3DSaadatmand%26aufirst%3DA.%2BR.%26aulast%3DL%25C3%25B6tsch%26aufirst%3DJ.%26aulast%3DTegeder%26aufirst%3DI.%26aulast%3DStingl%26aufirst%3DJ.%2BC.%26aulast%3DBrockm%25C3%25B6ller%26aufirst%3DJ.%26atitle%3DGenetically%2520Polymorphic%2520OCT1%253A%2520Another%2520Piece%2520in%2520the%2520Puzzle%2520of%2520the%2520Variable%2520Pharmacokinetics%2520and%2520Pharmacodynamics%2520of%2520the%2520Opioidergic%2520Drug%2520Tramadol%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2011%26volume%3D90%26issue%3D1%26spage%3D143%26epage%3D150%26doi%3D10.1038%2Fclpt.2011.56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Cho, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J.-Y.</span><span> </span><span class="NLM_article-title">Verapamil Decreases the Glucose-Lowering Effect of Metformin in Healthy Volunteers</span> <span class="citation_source-journal">Br. J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1426</span><span class="NLM_x">–</span> <span class="NLM_lpage">1432</span><span class="refDoi"> DOI: 10.1111/bcp.12476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1111%2Fbcp.12476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=25060604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCju7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2014&pages=1426-1432&issue=6&author=S.+K.+Choauthor=C.+O.+Kimauthor=E.+S.+Parkauthor=J.-Y.+Chung&title=Verapamil+Decreases+the+Glucose-Lowering+Effect+of+Metformin+in+Healthy+Volunteers&doi=10.1111%2Fbcp.12476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Verapamil decreases the glucose-lowering effect of metformin in healthy volunteers</span></div><div class="casAuthors">Cho, Sung Kweon; Kim, Choon Ok; Park, Eun Seok; Chung, Jae-Yong</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1426-1432</span>CODEN:
                <span class="NLM_cas:coden">BCPHBM</span>;
        ISSN:<span class="NLM_cas:issn">1365-2125</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Aim : The org. cation transporter 1 (OCT1) plays a key role in the cellular transport of metformin and its subsequent glucose-lowering effect.  A recent non-clin. study reported that metformin uptake into hepatocytes is regulated via OCT1, and that uptake was strongly inhibited by verapamil.  Therefore, we investigated the effects of verapamil co-administration on the pharmacokinetics and pharmacodynamics of metformin in humans.  Methods : We evaluated the pharmacokinetics and the anti-hyperglycemic effects of metformin using an oral glucose tolerance test (OGTT) in 12 healthy participants, before (day 1) and after metformin treatment (day 2), and again on days 15 and 16 after co-administration with verapamil.  Results : Verapamil inhibited the ability of metformin to reduce max. blood glucose concns. (ΔGmax) by 62.5% (P = 0.008) and decreased the area under the glucose concn.-time curve (ΔAUCgluc) by 238% (P = 0.015).  However, verapamil did not significantly alter the Cmax and the AUC of metformin, nor its renal clearance.  Conclusions : Our results suggest that verapamil remarkably decreases the glucose-lowering effect of metformin, possibly by acting as a competitive inhibitor of OCT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCHQT4gxaQG7Vg90H21EOLACvtfcHk0lhq2y2PKxTUOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCju7%252FO&md5=20cde8d523cb19a37f0113b570534024</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fbcp.12476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbcp.12476%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DS.%2BK.%26aulast%3DKim%26aufirst%3DC.%2BO.%26aulast%3DPark%26aufirst%3DE.%2BS.%26aulast%3DChung%26aufirst%3DJ.-Y.%26atitle%3DVerapamil%2520Decreases%2520the%2520Glucose-Lowering%2520Effect%2520of%2520Metformin%2520in%2520Healthy%2520Volunteers%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2014%26volume%3D78%26issue%3D6%26spage%3D1426%26epage%3D1432%26doi%3D10.1111%2Fbcp.12476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tweedie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benet, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brouwer, K. L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillgren, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmaster, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keppler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niemi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ware, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamek-Gliszczynski, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span> </span><span class="NLM_article-title">Membrane Transporters in Drug Development</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1038/nrd3028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnrd3028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=20190787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFelu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=215-236&issue=3&author=K.+M.+Giacominiauthor=S.-M.+Huangauthor=D.+J.+Tweedieauthor=L.+Z.+Benetauthor=K.+L.+R.+Brouwerauthor=X.+Chuauthor=A.+Dahlinauthor=R.+Eversauthor=V.+Fischerauthor=K.+M.+Hillgrenauthor=K.+A.+Hoffmasterauthor=T.+Ishikawaauthor=D.+Kepplerauthor=R.+B.+Kimauthor=C.+A.+Leeauthor=M.+Niemiauthor=J.+W.+Polliauthor=Y.+Sugiyamaauthor=P.+W.+Swaanauthor=J.+A.+Wareauthor=S.+H.+Wrightauthor=S.+W.+Yeeauthor=M.+J.+Zamek-Gliszczynskiauthor=L.+Zhang&title=Membrane+Transporters+in+Drug+Development&doi=10.1038%2Fnrd3028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane transporters in drug development</span></div><div class="casAuthors">Giacomini, Kathleen M.; Huang, Shiew-Mei; Tweedie, Donald J.; Benet, Leslie Z.; Brouwer, Kim L. R.; Chu, Xiaoyan; Dahlin, Amber; Evers, Raymond; Fischer, Volker; Hillgren, Kathleen M.; Hoffmaster, Keith A.; Ishikawa, Toshihisa; Keppler, Dietrich; Kim, Richard B.; Lee, Caroline A.; Niemi, Mikko; Polli, Joseph W.; Sugiyama, Yuicchi; Swaan, Peter W.; Ware, Joseph A.; Wright, Stephen H.; Wah Yee, Sook; Zamek-Gliszczynski, Maciej J.; Zhang, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-236</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Membrane transporters can be major determinants of the pharmacokinetic, safety and efficacy profiles of drugs.  This presents several key questions for drug development, including which transporters are clin. important in drug absorption and disposition, and which in vitro methods are suitable for studying drug interactions with these transporters.  In addn., what criteria should trigger follow-up clin. studies, and which clin. studies should be conducted if needed.  In this article, we provide the recommendations of the International Transporter Consortium on these issues, and present decision trees that are intended to help guide clin. studies on the currently recognized most important drug transporter interactions.  The recommendations are generally intended to support clin. development and filing of a new drug application.  Overall, it is advised that the timing of transporter investigations should be driven by efficacy, safety and clin. trial enrollment questions (for example, exclusion and inclusion criteria), as well as a need for further understanding of the absorption, distribution, metab. and excretion properties of the drug mol., and information required for drug labeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiBIf0gk93W7Vg90H21EOLACvtfcHk0lhq2y2PKxTUOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFelu74%253D&md5=89d9ce746284c1814fb01cd9b9a72027</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd3028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3028%26sid%3Dliteratum%253Aachs%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DHuang%26aufirst%3DS.-M.%26aulast%3DTweedie%26aufirst%3DD.%2BJ.%26aulast%3DBenet%26aufirst%3DL.%2BZ.%26aulast%3DBrouwer%26aufirst%3DK.%2BL.%2BR.%26aulast%3DChu%26aufirst%3DX.%26aulast%3DDahlin%26aufirst%3DA.%26aulast%3DEvers%26aufirst%3DR.%26aulast%3DFischer%26aufirst%3DV.%26aulast%3DHillgren%26aufirst%3DK.%2BM.%26aulast%3DHoffmaster%26aufirst%3DK.%2BA.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DKeppler%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DR.%2BB.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DNiemi%26aufirst%3DM.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DSugiyama%26aufirst%3DY.%26aulast%3DSwaan%26aufirst%3DP.%2BW.%26aulast%3DWare%26aufirst%3DJ.%2BA.%26aulast%3DWright%26aufirst%3DS.%2BH.%26aulast%3DYee%26aufirst%3DS.%2BW.%26aulast%3DZamek-Gliszczynski%26aufirst%3DM.%2BJ.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DMembrane%2520Transporters%2520in%2520Drug%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26issue%3D3%26spage%3D215%26epage%3D236%26doi%3D10.1038%2Fnrd3028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span> <span class="citation_source-book">Guideline on the Investigation of Drug Interactions</span>; <span class="NLM_publisher-name">European Medicines Agency</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Guideline+on+the+Investigation+of+Drug+Interactions%3B+European+Medicines+Agency%3A+London%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGuideline%2520on%2520the%2520Investigation%2520of%2520Drug%2520Interactions%26pub%3DEuropean%2520Medicines%2520Agency%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Haenisch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drescher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiemer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giros, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gautron, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bönisch, H.</span><span> </span><span class="NLM_article-title">Interaction of Antidepressant and Antipsychotic Drugs with the Human Organic Cation Transporters hOCT1, hOCT2 and hOCT3</span> <span class="citation_source-journal">Naunyn-Schmiedeberg's Arch. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">385</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1007/s00210-012-0781-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1007%2Fs00210-012-0781-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=22806583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSgsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=385&publication_year=2012&pages=1017-1023&issue=10&author=B.+Haenischauthor=E.+Drescherauthor=L.+Thiemerauthor=H.+Xinauthor=B.+Girosauthor=S.+Gautronauthor=H.+B%C3%B6nisch&title=Interaction+of+Antidepressant+and+Antipsychotic+Drugs+with+the+Human+Organic+Cation+Transporters+hOCT1%2C+hOCT2+and+hOCT3&doi=10.1007%2Fs00210-012-0781-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of antidepressant and antipsychotic drugs with the human organic cation transporters hOCT1, hOCT2 and hOCT3</span></div><div class="casAuthors">Haenisch, Britta; Drescher, Eva; Thiemer, Lidia; Xin, Hu; Giros, Bruno; Gautron, Sophie; Boenisch, Heinz</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">385</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1017-1023</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Besides the three antidepressant-sensitive, Na+- and Cl--dependent monoamine transporters, Na+-independent org. cation transporters (OCTs) are known to transport monoamines.  However, little is known about the interactions of psychotropic drugs with human (h) OCTs.  In the present study, a series of diverse antidepressant and antipsychotic drugs were examd. for their inhibitory potency at hOCT1, hOCT2 and hOCT3 by measuring inhibition of [3H]-MPP+ uptake into HEK293 cells stably expressing one of the three hOCTs.  The inhibitory potencies (IC50s) ranged from 1 to 900 μM.  Most of the examd. drugs showed highest inhibitory potency at hOCT1 which is very sparsely expressed in the brain and mainly involved in renal and hepatic clearance of cationic drugs.  At their upper therapeutic plasma concns., several drugs are expected to inhibit by more than 20 % hOCT1 and could thus interfere with the pharmacokinetics of hOCT1-transported drugs in the kidney and liver, namely trimipramine, desipramine and fluoxetine (by about 37 %), levomepromazine and nefazodone (by about 32 %), and clozapine and amitriptyline (by about 22 %).  At hOCT2 and hOCT3, which are involved in monoamine homeostasis in the brain, IC50s of most psychoactive drugs were in the high micromolar range.  At their upper plasma concns., only three compds., bupropion, nefazodone and clozapine, showed potential for inhibition, of about 18 % at hOCT2 (bupropion), about 22 % at hOCT3 (nefazodone) and of approx. 10 % at hOCT2 and hOCT3 (clozapine).  Thus, under the assumption of a tenfold accumulation in the brain, bupropion, nefazodone and clozapine may notably inhibit the corresponding hOCTs.  It remains to be shown whether such a direct inhibition plays a role in the clin. effects of these three drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQuUHCihnvR7Vg90H21EOLACvtfcHk0lh3qh_1NPGwUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSgsLzL&md5=21d138b3871c65e49d781ac606504187</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs00210-012-0781-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-012-0781-8%26sid%3Dliteratum%253Aachs%26aulast%3DHaenisch%26aufirst%3DB.%26aulast%3DDrescher%26aufirst%3DE.%26aulast%3DThiemer%26aufirst%3DL.%26aulast%3DXin%26aufirst%3DH.%26aulast%3DGiros%26aufirst%3DB.%26aulast%3DGautron%26aufirst%3DS.%26aulast%3DB%25C3%25B6nisch%26aufirst%3DH.%26atitle%3DInteraction%2520of%2520Antidepressant%2520and%2520Antipsychotic%2520Drugs%2520with%2520the%2520Human%2520Organic%2520Cation%2520Transporters%2520hOCT1%252C%2520hOCT2%2520and%2520hOCT3%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2012%26volume%3D385%26issue%3D10%26spage%3D1017%26epage%3D1023%26doi%3D10.1007%2Fs00210-012-0781-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Ahlin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustavsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norinder, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergström, C. A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span> </span><span class="NLM_article-title">Structural Requirements for Drug Inhibition of the Liver Specific Human Organic Cation Transport Protein 1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">5932</span><span class="NLM_x">–</span> <span class="NLM_lpage">5942</span><span class="refDoi"> DOI: 10.1021/jm8003152</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8003152" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5932-5942&issue=19&author=G.+Ahlinauthor=J.+Karlssonauthor=J.+M.+Pedersenauthor=L.+Gustavssonauthor=R.+Larssonauthor=P.+Matssonauthor=U.+Norinderauthor=C.+A.+S.+Bergstr%C3%B6mauthor=P.+Artursson&title=Structural+Requirements+for+Drug+Inhibition+of+the+Liver+Specific+Human+Organic+Cation+Transport+Protein+1&doi=10.1021%2Fjm8003152"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm8003152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8003152%26sid%3Dliteratum%253Aachs%26aulast%3DAhlin%26aufirst%3DG.%26aulast%3DKarlsson%26aufirst%3DJ.%26aulast%3DPedersen%26aufirst%3DJ.%2BM.%26aulast%3DGustavsson%26aufirst%3DL.%26aulast%3DLarsson%26aufirst%3DR.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DNorinder%26aufirst%3DU.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DC.%2BA.%2BS.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DStructural%2520Requirements%2520for%2520Drug%2520Inhibition%2520of%2520the%2520Liver%2520Specific%2520Human%2520Organic%2520Cation%2520Transport%2520Protein%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D19%26spage%3D5932%26epage%3D5942%26doi%3D10.1021%2Fjm8003152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Belzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morales, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagadish, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mash, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Substrate-Dependent Ligand Inhibition of the Human Organic Cation Transporter OCT2</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">300</span><span class="NLM_x">–</span> <span class="NLM_lpage">310</span><span class="refDoi"> DOI: 10.1124/jpet.113.203257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.113.203257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23709117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOjsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2013&pages=300-310&issue=2&author=M.+Belzerauthor=M.+Moralesauthor=B.+Jagadishauthor=E.+A.+Mashauthor=S.+H.+Wright&title=Substrate-Dependent+Ligand+Inhibition+of+the+Human+Organic+Cation+Transporter+OCT2&doi=10.1124%2Fjpet.113.203257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-dependent ligand inhibition of the human organic cation transporter OCT2</span></div><div class="casAuthors">Belzer, Mathew; Morales, Mark; Jagadish, Bhumasamudram; Mash, Eugene A.; Wright, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-310</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Org. cation transporter 2 (OCT2) mediates the initial step in renal secretion of org. cations: uptake from the blood, across the basolateral membrane, and into the renal proximal tubule cells.  Because of its potential as a target for unwanted drug-drug interactions (DDIs), considerable attention has been directed toward understanding the basis of OCT2 selectivity.  These studies typically assess selectivity based on ligand inhibition profiles for OCT2-mediated transport of a probe substrate.  However, little attention has been given to the potential influence of the substrate on the profile of ligand inhibition.  Here we compared the IC50 values obtained for a set of structurally distinct inhibitors against OCT2-mediated transport of three structurally distinct substrates: 1-methyl-4-phenylpyridinium (MPP); metformin; and a novel fluorescent substrate, N,N,N-trimethyl-2-[methyl(7-nitrobenzo[c][l,2,5]oxadiazol-4-yl)amino]ethanaminium iodide (NBD-MTMA).  The median IC50 value for inhibition of MPP transport was 9-fold higher than that for inhibition of metformin transport.  Similarly, the median IC50 value for inhibition of MPP transport was 5-fold higher than that for NBD-MTMA transport.  However, this was not a systematic difference in inhibitory efficacy; the ratio of IC50 values, MPP vs. NBD-MTMA, ranged from 88-fold (ipratropium) to 0.3-fold (metformin).  These data show that 1) the choice of OCT2 substrate significantly influences both quant. and qual. inhibitory interactions with cationic drugs; and 2) ligand interactions with OCT2 are not restricted to competition for a common ligand binding site, consistent with a binding surface characterized by multiple, possibly overlapping interaction sites.  Development of predictive models of DDIs with OCT2 must take into account the substrate dependence of ligand interaction with this protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfkgfWyk5fVLVg90H21EOLACvtfcHk0lh3qh_1NPGwUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOjsbfK&md5=d640680933397fa257f3896395fd7fc1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1124%2Fjpet.113.203257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.113.203257%26sid%3Dliteratum%253Aachs%26aulast%3DBelzer%26aufirst%3DM.%26aulast%3DMorales%26aufirst%3DM.%26aulast%3DJagadish%26aufirst%3DB.%26aulast%3DMash%26aufirst%3DE.%2BA.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DSubstrate-Dependent%2520Ligand%2520Inhibition%2520of%2520the%2520Human%2520Organic%2520Cation%2520Transporter%2520OCT2%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2013%26volume%3D346%26issue%3D2%26spage%3D300%26epage%3D310%26doi%3D10.1124%2Fjpet.113.203257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Polyspecific Organic Cation Transporters: Their Functions and Interactions with Drugs</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">381</span><span class="refDoi"> DOI: 10.1016/j.tips.2004.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1016%2Fj.tips.2004.05.005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=375-381&issue=7&author=H.+Koepsell&title=Polyspecific+Organic+Cation+Transporters%3A+Their+Functions+and+Interactions+with+Drugs&doi=10.1016%2Fj.tips.2004.05.005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2004.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2004.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DPolyspecific%2520Organic%2520Cation%2520Transporters%253A%2520Their%2520Functions%2520and%2520Interactions%2520with%2520Drugs%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2004%26volume%3D25%26issue%3D7%26spage%3D375%26epage%3D381%26doi%3D10.1016%2Fj.tips.2004.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirahatti, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahadevan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">A Conserved Glutamate Residue in Transmembrane Helix 10 Influences Substrate Specificity of Rabbit OCT2 (SLC22A2)</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x"> (</span><span class="NLM_issue">41</span><span class="NLM_x">) </span> <span class="NLM_fpage">34813</span><span class="NLM_x">–</span> <span class="NLM_lpage">34822</span><span class="refDoi"> DOI: 10.1074/jbc.M506342200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1074%2Fjbc.M506342200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=34813-34822&issue=41&author=X.+Zhangauthor=N.+V.+Shirahattiauthor=D.+Mahadevanauthor=S.+H.+Wright&title=A+Conserved+Glutamate+Residue+in+Transmembrane+Helix+10+Influences+Substrate+Specificity+of+Rabbit+OCT2+%28SLC22A2%29&doi=10.1074%2Fjbc.M506342200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M506342200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M506342200%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DShirahatti%26aufirst%3DN.%2BV.%26aulast%3DMahadevan%26aufirst%3DD.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DA%2520Conserved%2520Glutamate%2520Residue%2520in%2520Transmembrane%2520Helix%252010%2520Influences%2520Substrate%2520Specificity%2520of%2520Rabbit%2520OCT2%2520%2528SLC22A2%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26issue%3D41%26spage%3D34813%26epage%3D34822%26doi%3D10.1074%2Fjbc.M506342200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Substrate Recognition and Translocation by Polyspecific Organic Cation Transporters</span> <span class="citation_source-journal">Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">392</span><span class="NLM_x"> (</span><span class="NLM_issue">1–2</span><span class="NLM_x">) </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">101</span><span class="refDoi"> DOI: 10.1515/bc.2011.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1515%2Fbc.2011.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2011&pages=95-101&issue=1%E2%80%932&author=H.+Koepsell&title=Substrate+Recognition+and+Translocation+by+Polyspecific+Organic+Cation+Transporters&doi=10.1515%2Fbc.2011.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1515%2Fbc.2011.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fbc.2011.009%26sid%3Dliteratum%253Aachs%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DSubstrate%2520Recognition%2520and%2520Translocation%2520by%2520Polyspecific%2520Organic%2520Cation%2520Transporters%26jtitle%3DBiol.%2520Chem.%26date%3D2011%26volume%3D392%26issue%3D1%25E2%2580%25932%26spage%3D95%26epage%3D101%26doi%3D10.1515%2Fbc.2011.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatskaya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Subtype-Specific Affinity for Corticosterone of Rat Organic Cation Transporters rOCT1 and rOCT2 Depends on Three Amino Acids Within the Substrate Binding Region</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1612</span><span class="NLM_x">–</span> <span class="NLM_lpage">1619</span><span class="refDoi"> DOI: 10.1124/mol.104.008821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.104.008821" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2005&pages=1612-1619&issue=5&author=V.+Gorboulevauthor=N.+Shatskayaauthor=C.+Volkauthor=H.+Koepsell&title=Subtype-Specific+Affinity+for+Corticosterone+of+Rat+Organic+Cation+Transporters+rOCT1+and+rOCT2+Depends+on+Three+Amino+Acids+Within+the+Substrate+Binding+Region&doi=10.1124%2Fmol.104.008821"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1124%2Fmol.104.008821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.104.008821%26sid%3Dliteratum%253Aachs%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DShatskaya%26aufirst%3DN.%26aulast%3DVolk%26aufirst%3DC.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DSubtype-Specific%2520Affinity%2520for%2520Corticosterone%2520of%2520Rat%2520Organic%2520Cation%2520Transporters%2520rOCT1%2520and%2520rOCT2%2520Depends%2520on%2520Three%2520Amino%2520Acids%2520Within%2520the%2520Substrate%2520Binding%2520Region%26jtitle%3DMol.%2520Pharmacol.%26date%3D2005%26volume%3D67%26issue%3D5%26spage%3D1612%26epage%3D1619%26doi%3D10.1124%2Fmol.104.008821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Popp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorbunov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatskaya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Amino Acids Critical for Substrate Affinity of Rat Organic Cation Transporter 1 Line the Substrate Binding Region in a Model Derived From the Tertiary Structure of Lactose Permease</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1600</span><span class="NLM_x">–</span> <span class="NLM_lpage">1611</span><span class="refDoi"> DOI: 10.1124/mol.104.008839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.104.008839" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2005&pages=1600-1611&issue=5&author=C.+Poppauthor=V.+Gorboulevauthor=T.+D.+M%C3%BCllerauthor=D.+Gorbunovauthor=N.+Shatskayaauthor=H.+Koepsell&title=Amino+Acids+Critical+for+Substrate+Affinity+of+Rat+Organic+Cation+Transporter+1+Line+the+Substrate+Binding+Region+in+a+Model+Derived+From+the+Tertiary+Structure+of+Lactose+Permease&doi=10.1124%2Fmol.104.008839"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1124%2Fmol.104.008839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.104.008839%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DC.%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DM%25C3%25BCller%26aufirst%3DT.%2BD.%26aulast%3DGorbunov%26aufirst%3DD.%26aulast%3DShatskaya%26aufirst%3DN.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DAmino%2520Acids%2520Critical%2520for%2520Substrate%2520Affinity%2520of%2520Rat%2520Organic%2520Cation%2520Transporter%25201%2520Line%2520the%2520Substrate%2520Binding%2520Region%2520in%2520a%2520Model%2520Derived%2520From%2520the%2520Tertiary%2520Structure%2520of%2520Lactose%2520Permease%26jtitle%3DMol.%2520Pharmacol.%26date%3D2005%26volume%3D67%26issue%3D5%26spage%3D1600%26epage%3D1611%26doi%3D10.1124%2Fmol.104.008839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Volk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotzsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">Five Amino Acids in the Innermost Cavity of the Substrate Binding Cleft of Organic Cation Transporter 1 Interact with Extracellular and Intracellular Corticosterone</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">289</span><span class="refDoi"> DOI: 10.1124/mol.109.054783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.109.054783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=275-289&issue=2&author=C.+Volkauthor=V.+Gorboulevauthor=A.+Kotzschauthor=T.+D.+M%C3%BCllerauthor=H.+Koepsell&title=Five+Amino+Acids+in+the+Innermost+Cavity+of+the+Substrate+Binding+Cleft+of+Organic+Cation+Transporter+1+Interact+with+Extracellular+and+Intracellular+Corticosterone&doi=10.1124%2Fmol.109.054783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.054783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.054783%26sid%3Dliteratum%253Aachs%26aulast%3DVolk%26aufirst%3DC.%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DKotzsch%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DT.%2BD.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DFive%2520Amino%2520Acids%2520in%2520the%2520Innermost%2520Cavity%2520of%2520the%2520Substrate%2520Binding%2520Cleft%2520of%2520Organic%2520Cation%2520Transporter%25201%2520Interact%2520with%2520Extracellular%2520and%2520Intracellular%2520Corticosterone%26jtitle%3DMol.%2520Pharmacol.%26date%3D2009%26volume%3D76%26issue%3D2%26spage%3D275%26epage%3D289%26doi%3D10.1124%2Fmol.109.054783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Pedersen, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waight, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risenmay, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roe-Zurz, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harries, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johri, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">Crystal Structure of a Eukaryotic Phosphate Transporter</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">496</span><span class="NLM_x"> (</span><span class="NLM_issue">7446</span><span class="NLM_x">) </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">536</span><span class="refDoi"> DOI: 10.1038/nature12042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnature12042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=533-536&issue=7446&author=B.+P.+Pedersenauthor=H.+Kumarauthor=A.+B.+Waightauthor=A.+J.+Risenmayauthor=Z.+Roe-Zurzauthor=B.+H.+Chauauthor=A.+Schlessingerauthor=M.+Bonomiauthor=W.+Harriesauthor=A.+Saliauthor=A.+K.+Johriauthor=R.+M.+Stroud&title=Crystal+Structure+of+a+Eukaryotic+Phosphate+Transporter&doi=10.1038%2Fnature12042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature12042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12042%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DB.%2BP.%26aulast%3DKumar%26aufirst%3DH.%26aulast%3DWaight%26aufirst%3DA.%2BB.%26aulast%3DRisenmay%26aufirst%3DA.%2BJ.%26aulast%3DRoe-Zurz%26aufirst%3DZ.%26aulast%3DChau%26aufirst%3DB.%2BH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DHarries%26aufirst%3DW.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DJohri%26aufirst%3DA.%2BK.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Eukaryotic%2520Phosphate%2520Transporter%26jtitle%3DNature%26date%3D2013%26volume%3D496%26issue%3D7446%26spage%3D533%26epage%3D536%26doi%3D10.1038%2Fnature12042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Shen, M.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Statistical Potential for Assessment and Prediction of Protein Structures</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2507</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span><span class="refDoi"> DOI: 10.1110/ps.062416606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1110%2Fps.062416606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=17075131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=2507-2524&issue=11&author=M.-Y.+Shenauthor=A.+Sali&title=Statistical+Potential+for+Assessment+and+Prediction+of+Protein+Structures&doi=10.1110%2Fps.062416606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical potential for assessment and prediction of protein structures</span></div><div class="casAuthors">Shen, Min-Yi; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2507-2524</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Protein structures in the Protein Data Bank provide a wealth of data about the interactions that det. the native states of proteins.  Using the probability theory, the authors derive an at. distance-dependent statistical potential from a sample of native structures that does not depend on any adjustable parameters (Discrete Optimized Protein Energy, or DOPE).  DOPE is based on an improved ref. state that corresponds to noninteracting atoms in a homogeneous sphere with the radius dependent on a sample native structure; it thus accounts for the finite and spherical shape of the native structures.  The DOPE potential was extd. from a nonredundant set of 1472 crystallog. structures.  The authors tested DOPE and five other scoring functions by the detection of the native state among six multiple target decoy sets, the correlation between the score and model error, and the identification of the most accurate nonnative structure in the decoy set.  For all decoy sets, DOPE is the best performing function in terms of all criteria, except for a tie in one criterion for one decoy set.  To facilitate its use in various applications, such as model assessment, loop modeling, and fitting into cryo-electron microscopy mass d. maps combined with comparative protein structure modeling, DOPE was incorporated into the modeling package MODELLER-8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz7bz4vq-p_7Vg90H21EOLACvtfcHk0lhF3FDmD330Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO&md5=f4e7ef56cd100164e347103a6888ee8e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1110%2Fps.062416606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.062416606%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DM.-Y.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStatistical%2520Potential%2520for%2520Assessment%2520and%2520Prediction%2520of%2520Protein%2520Structures%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26issue%3D11%26spage%3D2507%26epage%3D2524%26doi%3D10.1110%2Fps.062416606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Eramian, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eswar, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, M.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">How Well Can the Accuracy of Comparative Protein Structure Models Be Predicted?</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1881</span><span class="NLM_x">–</span> <span class="NLM_lpage">1893</span><span class="refDoi"> DOI: 10.1110/ps.036061.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1110%2Fps.036061.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=18832340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGlsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2008&pages=1881-1893&issue=11&author=D.+Eramianauthor=N.+Eswarauthor=M.-Y.+Shenauthor=A.+Sali&title=How+Well+Can+the+Accuracy+of+Comparative+Protein+Structure+Models+Be+Predicted%3F&doi=10.1110%2Fps.036061.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">How well can the accuracy of comparative protein structure models be predicted?</span></div><div class="casAuthors">Eramian, David; Eswar, Narayanan; Shen, Min-Yi; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1881-1893</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Comparative structure models are available for two orders of magnitude more protein sequences than are exptl. detd. structures.  These models, however, suffer from two limitations that exptl. detd. structures do not: they frequently contain significant errors, and their accuracy cannot be readily assessed.  We have addressed the latter limitation by developing a protocol optimized specifically for predicting the Cα root-mean-squared deviation (RMSD) and native overlap (NO3.5Å) errors of a model in the absence of its native structure.  In contrast to most traditional assessment scores that merely predict one model is more accurate than others, this approach quantifies the error in an abs. sense, thus helping to det. whether or not the model is suitable for intended applications.  The assessment relies on a model-specific scoring function constructed by a support vector machine.  This regression optimizes the wts. of up to nine features, including various sequence similarity measures and statistical potentials, extd. from a tailored training set of models unique to the model being assessed: if possible, we use similarly sized models with the same fold; otherwise, we use similarly sized models with the same secondary structure compn.  This protocol predicts the RMSD and NO3.5Å errors for a diverse set of 580,317 comparative models of 6174 sequences with correlation coeffs. (r) of 0.84 and 0.86, resp., to the actual errors.  This scoring function achieves the best correlation compared to 13 other tested assessment criteria that achieved correlations ranging from 0.35 to 0.71.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcmHLOHj4_vrVg90H21EOLACvtfcHk0lhF3FDmD330Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlGlsb%252FM&md5=fbd8fbe28b7846a7dc8b6e9e0991343f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1110%2Fps.036061.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.036061.108%26sid%3Dliteratum%253Aachs%26aulast%3DEramian%26aufirst%3DD.%26aulast%3DEswar%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DM.-Y.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DHow%2520Well%2520Can%2520the%2520Accuracy%2520of%2520Comparative%2520Protein%2520Structure%2520Models%2520Be%2520Predicted%253F%26jtitle%3DProtein%2520Sci.%26date%3D2008%26volume%3D17%26issue%3D11%26spage%3D1881%26epage%3D1893%26doi%3D10.1110%2Fps.036061.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Keller, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egenberger, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorboulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernhard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uzelac, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorbunov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wirth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koppatz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dötsch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunte, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koepsell, H.</span><span> </span><span class="NLM_article-title">The Large Extracellular Loop of Organic Cation Transporter 1 Influences Substrate Affinity and Is Pivotal for Oligomerization</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x"> (</span><span class="NLM_issue">43</span><span class="NLM_x">) </span> <span class="NLM_fpage">37874</span><span class="NLM_x">–</span> <span class="NLM_lpage">37886</span><span class="refDoi"> DOI: 10.1074/jbc.M111.289330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1074%2Fjbc.M111.289330" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=37874-37886&issue=43&author=T.+Kellerauthor=B.+Egenbergerauthor=V.+Gorboulevauthor=F.+Bernhardauthor=Z.+Uzelacauthor=D.+Gorbunovauthor=C.+Wirthauthor=S.+Koppatzauthor=V.+D%C3%B6tschauthor=C.+Hunteauthor=H.+H.+Sitteauthor=H.+Koepsell&title=The+Large+Extracellular+Loop+of+Organic+Cation+Transporter+1+Influences+Substrate+Affinity+and+Is+Pivotal+for+Oligomerization&doi=10.1074%2Fjbc.M111.289330"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.289330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.289330%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DT.%26aulast%3DEgenberger%26aufirst%3DB.%26aulast%3DGorboulev%26aufirst%3DV.%26aulast%3DBernhard%26aufirst%3DF.%26aulast%3DUzelac%26aufirst%3DZ.%26aulast%3DGorbunov%26aufirst%3DD.%26aulast%3DWirth%26aufirst%3DC.%26aulast%3DKoppatz%26aufirst%3DS.%26aulast%3DD%25C3%25B6tsch%26aufirst%3DV.%26aulast%3DHunte%26aufirst%3DC.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DKoepsell%26aufirst%3DH.%26atitle%3DThe%2520Large%2520Extracellular%2520Loop%2520of%2520Organic%2520Cation%2520Transporter%25201%2520Influences%2520Substrate%2520Affinity%2520and%2520Is%2520Pivotal%2520for%2520Oligomerization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26issue%3D43%26spage%3D37874%26epage%3D37886%26doi%3D10.1074%2Fjbc.M111.289330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittwer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">High Selectivity of the Γ-Aminobutyric Acid Transporter 2 (GAT-2, SLC6A13) Revealed by Structure-Based Approach</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x"> (</span><span class="NLM_issue">45</span><span class="NLM_x">) </span> <span class="NLM_fpage">37745</span><span class="NLM_x">–</span> <span class="NLM_lpage">37756</span><span class="refDoi"> DOI: 10.1074/jbc.M112.388157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1074%2Fjbc.M112.388157" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=37745-37756&issue=45&author=A.+Schlessingerauthor=M.+B.+Wittwerauthor=A.+Dahlinauthor=N.+Khuriauthor=M.+Bonomiauthor=H.+Fanauthor=K.+M.+Giacominiauthor=A.+Sali&title=High+Selectivity+of+the+%CE%93-Aminobutyric+Acid+Transporter+2+%28GAT-2%2C+SLC6A13%29+Revealed+by+Structure-Based+Approach&doi=10.1074%2Fjbc.M112.388157"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.388157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.388157%26sid%3Dliteratum%253Aachs%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26aulast%3DDahlin%26aufirst%3DA.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DHigh%2520Selectivity%2520of%2520the%2520%25CE%2593-Aminobutyric%2520Acid%2520Transporter%25202%2520%2528GAT-2%252C%2520SLC6A13%2529%2520Revealed%2520by%2520Structure-Based%2520Approach%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26issue%3D45%26spage%3D37745%26epage%3D37756%26doi%3D10.1074%2Fjbc.M112.388157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Structure-Based Discovery of Prescription Drugs That Interact with the Norepinephrine Transporter, NET</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x"> (</span><span class="NLM_issue">38</span><span class="NLM_x">) </span> <span class="NLM_fpage">15810</span><span class="NLM_x">–</span> <span class="NLM_lpage">15815</span><span class="refDoi"> DOI: 10.1073/pnas.1106030108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1073%2Fpnas.1106030108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=15810-15815&issue=38&author=A.+Schlessingerauthor=E.+Geierauthor=H.+Fanauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+M.+Giacominiauthor=A.+Sali&title=Structure-Based+Discovery+of+Prescription+Drugs+That+Interact+with+the+Norepinephrine+Transporter%2C+NET&doi=10.1073%2Fpnas.1106030108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1106030108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1106030108%26sid%3Dliteratum%253Aachs%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DGeier%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Prescription%2520Drugs%2520That%2520Interact%2520with%2520the%2520Norepinephrine%2520Transporter%252C%2520NET%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D38%26spage%3D15810%26epage%3D15815%26doi%3D10.1073%2Fpnas.1106030108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Kido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Profiling of a Prescription Drug Library for Potential Renal Drug-Drug Interactions Mediated by the Organic Cation Transporter 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x"> (</span><span class="NLM_issue">13</span><span class="NLM_x">) </span> <span class="NLM_fpage">4548</span><span class="NLM_x">–</span> <span class="NLM_lpage">4558</span><span class="refDoi"> DOI: 10.1021/jm2001629</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2001629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1agsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4548-4558&issue=13&author=Y.+Kidoauthor=P.+Matssonauthor=K.+M.+Giacomini&title=Profiling+of+a+Prescription+Drug+Library+for+Potential+Renal+Drug-Drug+Interactions+Mediated+by+the+Organic+Cation+Transporter+2&doi=10.1021%2Fjm2001629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling of a Prescription Drug Library for Potential Renal Drug-Drug Interactions Mediated by the Organic Cation Transporter 2</span></div><div class="casAuthors">Kido, Yasuto; Matsson, Par; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4548-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-drug interactions (DDIs) are major causes of serious adverse drug reactions.  Most DDIs have a pharmacokinetic basis in which one drug reduces the elimination of a second drug, leading to potentially toxic drug levels.  As a major organ of drug elimination, the kidney represents an important site for DDIs.  Here, we screened a prescription drug library against the renal org. cation transporter OCT2/SLC22A2, which mediates the first step in the renal secretion of many cationic drugs.  Of the 910 compds. screened, 244 inhibited OCT2.  Computational analyses revealed key properties of inhibitors vs. noninhibitors, which included overall mol. charge.  Four of six potential clin. inhibitors were transporter-selective in follow-up screens against addnl. transporters: OCT1/SLC22A1, MATE1/SLC47A1, and MATE2-K/SLC47A2.  Two compds. showed different kinetics of interaction with the common polymorphism OCT2-A270S, suggesting a role of genetics in modulating renal DDIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRPsHJQI_skbVg90H21EOLACvtfcHk0liu0PxAEeOIuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnt1agsL0%253D&md5=907473f7116011468f807466c99f5b97</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm2001629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2001629%26sid%3Dliteratum%253Aachs%26aulast%3DKido%26aufirst%3DY.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DProfiling%2520of%2520a%2520Prescription%2520Drug%2520Library%2520for%2520Potential%2520Renal%2520Drug-Drug%2520Interactions%2520Mediated%2520by%2520the%2520Organic%2520Cation%2520Transporter%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D13%26spage%3D4548%26epage%3D4558%26doi%3D10.1021%2Fjm2001629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Zhang, J.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, T. D. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oldenburg, K. R.</span><span> </span><span class="NLM_article-title">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1177/108705719900400206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=67-73&issue=2&author=J.-H.+Zhangauthor=T.+D.+Y.+Chungauthor=K.+R.+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays&doi=10.1177%2F108705719900400206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2eajRNhco5siTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.-H.%26aulast%3DChung%26aufirst%3DT.%2BD.%2BY.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D1999%26volume%3D4%26issue%3D2%26spage%3D67%26epage%3D73%26doi%3D10.1177%2F108705719900400206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand Discovery From a Dopamine D3 Receptor Homology Model and Crystal Structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&issue=11&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+Discovery+From+a+Dopamine+D3+Receptor+Homology+Model+and+Crystal+Structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0li7EEUZ45b2SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520Discovery%2520From%2520a%2520Dopamine%2520D3%2520Receptor%2520Homology%2520Model%2520and%2520Crystal%2520Structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D11%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Minematsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, K.-I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Usui, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamimura, H.</span><span> </span><span class="NLM_article-title">Characterization of Human Organic Cation Transporter 1 (OCT1/SLC22A1)- and OCT2 (SLC22A2)-Mediated Transport of 1-(2-Methoxyethyl)-2-Methyl-4,9-Dioxo-3-(Pyrazin-2-Ylmethyl)- 4,9-Dihydro-1H-Naphtho[2,3-D]Imidazolium Bromide (YM155 Monobromide), a Novel Small Molecule Survivin Suppressant</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1124/dmd.109.028142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fdmd.109.028142" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=1-4&issue=1&author=T.+Minematsuauthor=M.+Iwaiauthor=K.-I.+Umeharaauthor=T.+Usuiauthor=H.+Kamimura&title=Characterization+of+Human+Organic+Cation+Transporter+1+%28OCT1%2FSLC22A1%29-+and+OCT2+%28SLC22A2%29-Mediated+Transport+of+1-%282-Methoxyethyl%29-2-Methyl-4%2C9-Dioxo-3-%28Pyrazin-2-Ylmethyl%29-+4%2C9-Dihydro-1H-Naphtho%5B2%2C3-D%5DImidazolium+Bromide+%28YM155+Monobromide%29%2C+a+Novel+Small+Molecule+Survivin+Suppressant&doi=10.1124%2Fdmd.109.028142"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fdmd.109.028142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.109.028142%26sid%3Dliteratum%253Aachs%26aulast%3DMinematsu%26aufirst%3DT.%26aulast%3DIwai%26aufirst%3DM.%26aulast%3DUmehara%26aufirst%3DK.-I.%26aulast%3DUsui%26aufirst%3DT.%26aulast%3DKamimura%26aufirst%3DH.%26atitle%3DCharacterization%2520of%2520Human%2520Organic%2520Cation%2520Transporter%25201%2520%2528OCT1%252FSLC22A1%2529-%2520and%2520OCT2%2520%2528SLC22A2%2529-Mediated%2520Transport%2520of%25201-%25282-Methoxyethyl%2529-2-Methyl-4%252C9-Dioxo-3-%2528Pyrazin-2-Ylmethyl%2529-%25204%252C9-Dihydro-1H-Naphtho%255B2%252C3-D%255DImidazolium%2520Bromide%2520%2528YM155%2520Monobromide%2529%252C%2520a%2520Novel%2520Small%2520Molecule%2520Survivin%2520Suppressant%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2010%26volume%3D38%26issue%3D1%26spage%3D1%26epage%3D4%26doi%3D10.1124%2Fdmd.109.028142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Bourdet, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakker, D. R.</span><span> </span><span class="NLM_article-title">Differential Substrate and Inhibitory Activities of Ranitidine and Famotidine Toward Human Organic Cation Transporter 1 (hOCT1; SLC22A1), hOCT2 (SLC22A2), and hOCT3 (SLC22A3)</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">315</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">1288</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1124/jpet.105.091223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.105.091223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=16141367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlalurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=315&publication_year=2005&pages=1288-1297&issue=3&author=D.+L.+Bourdetauthor=J.+B.+Pritchardauthor=D.+R.+Thakker&title=Differential+Substrate+and+Inhibitory+Activities+of+Ranitidine+and+Famotidine+Toward+Human+Organic+Cation+Transporter+1+%28hOCT1%3B+SLC22A1%29%2C+hOCT2+%28SLC22A2%29%2C+and+hOCT3+%28SLC22A3%29&doi=10.1124%2Fjpet.105.091223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Differential substrate and inhibitory activities of ranitidine and famotidine toward human organic cation transporter 1 (hOCT1; SLC22A1), hOCT2 (SLC22A2), and hOCT3 (SLC22A3)</span></div><div class="casAuthors">Bourdet, David L.; Pritchard, John B.; Thakker, Dhiren R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">315</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1288-1297</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Human org. cation transporters (hOCTs) are expressed in organs of drug absorption and elimination and play an important role in the uptake and elimination of xenobiotics.  The purpose of this study was to evaluate the substrate and inhibitory activity of the H2-receptor antagonists ranitidine and famotidine toward hOCTs and to det. the hOCT isoforms involved in the absorption and elimination of these compds. in humans.  Inhibition and substrate specificity of hOCT1, hOCT2, and hOCT3 for ranitidine and famotidine were elucidated in cRNA-injected Xenopus laevis oocytes.  Ranitidine and famotidine exhibited similarly potent inhibition of [3H]1-methyl-4-Ph pyridinium uptake into hOCT1-expressing (IC50 = 33 and 28 μM, resp.) and hOCT2-expressing oocytes (IC50 = 76 and 114 μM, resp.).  Famotidine exhibited potent inhibition of hOCT3; in contrast, ranitidine was a moderately weak inhibitor (IC50 = 6.7 and 290 μM, resp.).  [3H]Ranitidine uptake was stimulated by hOCT1 (Km = 70±9 μM) and to a much smaller extent by hOCT2.  No stimulation of [3H]ranitidine uptake was obsd. in hOCT3-expressing oocytes.  Trans-Stimulation and electrophysiol. studies suggested that famotidine also is an hOCT1 substrate and exhibits poor or no substrate activity toward hOCT2 and hOCT3.  Thus, hOCT1, which is expressed in the intestine and liver, is likely to play a major role in the intestinal absorption and hepatic disposition of ranitidine and famotidine in humans, whereas hOCT2, the major isoform present in the kidney, may play only a minor role in their renal elimination.  Famotidine seems to be one of the most potent inhibitors of hOCT3 yet identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM1LN-rRwlxrVg90H21EOLACvtfcHk0ljjLI6rJRl5_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlalurnE&md5=f7028e4e563179d4fbaa080ccedbdf25</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1124%2Fjpet.105.091223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.105.091223%26sid%3Dliteratum%253Aachs%26aulast%3DBourdet%26aufirst%3DD.%2BL.%26aulast%3DPritchard%26aufirst%3DJ.%2BB.%26aulast%3DThakker%26aufirst%3DD.%2BR.%26atitle%3DDifferential%2520Substrate%2520and%2520Inhibitory%2520Activities%2520of%2520Ranitidine%2520and%2520Famotidine%2520Toward%2520Human%2520Organic%2520Cation%2520Transporter%25201%2520%2528hOCT1%253B%2520SLC22A1%2529%252C%2520hOCT2%2520%2528SLC22A2%2529%252C%2520and%2520hOCT3%2520%2528SLC22A3%2529%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2005%26volume%3D315%26issue%3D3%26spage%3D1288%26epage%3D1297%26doi%3D10.1124%2Fjpet.105.091223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Bachmakov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaeser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">König, J.</span><span> </span><span class="NLM_article-title">Einfluss Von B-Blockern Auf Den Transport Von Metformin Durch Den Hepatischen Aufnahmetransporter Für Organische Kationen OCT1</span> <span class="citation_source-journal">Diabetologie und Stoffwechsel</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">274</span><span class="refDoi"> DOI: 10.1055/s-0029-1222078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1055%2Fs-0029-1222078" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=274&author=I.+Bachmakovauthor=H.+Glaeserauthor=J.+K%C3%B6nig&title=Einfluss+Von+B-Blockern+Auf+Den+Transport+Von+Metformin+Durch+Den+Hepatischen+Aufnahmetransporter+F%C3%BCr+Organische+Kationen+OCT1&doi=10.1055%2Fs-0029-1222078"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1055%2Fs-0029-1222078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0029-1222078%26sid%3Dliteratum%253Aachs%26aulast%3DBachmakov%26aufirst%3DI.%26aulast%3DGlaeser%26aufirst%3DH.%26aulast%3DK%25C3%25B6nig%26aufirst%3DJ.%26atitle%3DEinfluss%2520Von%2520B-Blockern%2520Auf%2520Den%2520Transport%2520Von%2520Metformin%2520Durch%2520Den%2520Hepatischen%2520Aufnahmetransporter%2520F%25C3%25BCr%2520Organische%2520Kationen%2520OCT1%26jtitle%3DDiabetologie%2520und%2520Stoffwechsel%26date%3D2009%26volume%3D4%26spage%3D274%26doi%3D10.1055%2Fs-0029-1222078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Barr, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, J. P.</span><span> </span><span class="NLM_article-title">Inhibition of Human Liver Aldehyde Oxidase: Implications for Potential Drug-Drug Interactions</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2381</span><span class="NLM_x">–</span> <span class="NLM_lpage">2386</span><span class="refDoi"> DOI: 10.1124/dmd.111.041806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fdmd.111.041806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21940905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCru77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=2381-2386&issue=12&author=J.+T.+Barrauthor=J.+P.+Jones&title=Inhibition+of+Human+Liver+Aldehyde+Oxidase%3A+Implications+for+Potential+Drug-Drug+Interactions&doi=10.1124%2Fdmd.111.041806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human liver aldehyde oxidase: implications for potential drug-drug interactions</span></div><div class="casAuthors">Barr, John T.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2381-2386</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">During the course of our research efforts to understand the kinetics of human aldehyde oxidase as a xenobiotic-clearing enzyme, we investigated the effect of eight different inhibitors on the oxidn. of the probe substrate phthalazine.  Satn. kinetic parameters for phthalazine oxidn. in human liver cytosol were found to be the following: Km = 8.0 ± 0.4 μM and Vmax = 4.3 ± 0.1 nmol · min-1 · mg protein-1.  Inhibitory potency of the inhibitors tested ranged from 0.1 to 5 μM.  Of the eight different inhibitor compds. tested, seven were obsd. to inhibit through a mixed mode and one through a strictly competitive mode.  A ratio of the Kii and Kis values was used to assess the relative competitiveness of each inhibitor.  For the mixed inhibitors, the mode of inhibition varied from mostly uncompetitive to predominantly competitive (Kii/Kis values ranging from 0.1 to 15).  The implications for potential drug-drug interactions and inhibition mechanism are discussed.  We found two inhibitors, clozapine and chlorpromazine, that have a moderate predicted risk of drug-drug interactions based on the Ki value relative to the inhibitor concn. in human plasma, having a calcd. [I]/Ki value of 0.4 and 0.8, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS6gGC3rD4U7Vg90H21EOLACvtfcHk0ljjLI6rJRl5_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCru77F&md5=1113e63fdb48c3f8ed07c5ca4ae8cf24</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.041806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.041806%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%26aufirst%3DJ.%2BT.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DInhibition%2520of%2520Human%2520Liver%2520Aldehyde%2520Oxidase%253A%2520Implications%2520for%2520Potential%2520Drug-Drug%2520Interactions%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26issue%3D12%26spage%3D2381%26epage%3D2386%26doi%3D10.1124%2Fdmd.111.041806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Gui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obaidat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaguturu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagenbuch, B.</span><span> </span><span class="NLM_article-title">Development of a Cell-Based High-Throughput Assay to Screen for Inhibitors of Organic Anion Transporting Polypeptides 1B1 and 1B3</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.2174/1875397301004010001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.2174%2F1875397301004010001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=20448812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFShtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2010&pages=1-8&author=C.+Guiauthor=A.+Obaidatauthor=R.+Chaguturuauthor=B.+Hagenbuch&title=Development+of+a+Cell-Based+High-Throughput+Assay+to+Screen+for+Inhibitors+of+Organic+Anion+Transporting+Polypeptides+1B1+and+1B3&doi=10.2174%2F1875397301004010001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a cell-based high-throughput assay to screen for inhibitors of organic anion transporting polypeptides 1B1 and 1B3</span></div><div class="casAuthors">Gui, Chunshan; Obaidat, Amanda; Chaguturu, Rathnam; Hagenbuch, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The two org. anion transporting polypeptides (OATPs) 1B1 and 1B3 are expressed at the sinusoidal membrane of hepatocytes.  They have a broad and overlapping substrate specificity and transport many endobiotics and drugs.  Specific inhibitors are required to det. the contribution of each OATP to the hepatocellular uptake of common substrates.  We have developed a cell-based high-throughput assay to screen chem. libraries in order to identify such inhibitors for OATP1B1 and OATP1B3.  We have used OATP1B1- or OATP1B3-expressing Chinese Hamster Ovary cells on 96-well plates and detd. uptake of fluorescein-methotrexate (FMTX).  We validated the assay with known inhibitors and screened the well characterized Prestwick library of 1120 drugs.  Along with several known OATP inhibitors including rifampicin, cyclosporine A and mifepristone we identified some new inhibitors.  For inhibitors that seemed to be able to distinguish between OATP1B1- and OATP1B3-mediated FMTX uptake IC50 values were detd.  Estropipate (estrone-3-sulfate stabilized with piperazine) was the most selective OATP1B1 inhibitor (IC50 = 0.06 μM vs. 19.3 μM for OATP1B3).  Ursolic acid was the most selective OATP1B3 inhibitor (IC50 = 2.3 μM vs. 12.5 μM for OATP1B1).  In conclusion, this cell-based assay should allow us to identify even more specific inhibitors by screening larger chem. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9MovUxE4hD7Vg90H21EOLACvtfcHk0ljjLI6rJRl5_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFShtbo%253D&md5=707ec92dbc83bfe2f9f228378ddcc55e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.2174%2F1875397301004010001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301004010001%26sid%3Dliteratum%253Aachs%26aulast%3DGui%26aufirst%3DC.%26aulast%3DObaidat%26aufirst%3DA.%26aulast%3DChaguturu%26aufirst%3DR.%26aulast%3DHagenbuch%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Cell-Based%2520High-Throughput%2520Assay%2520to%2520Screen%2520for%2520Inhibitors%2520of%2520Organic%2520Anion%2520Transporting%2520Polypeptides%25201B1%2520and%25201B3%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2010%26volume%3D4%26spage%3D1%26epage%3D8%26doi%3D10.2174%2F1875397301004010001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">De Bruyn, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Westen, G. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stieger, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Witte, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustijns, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annaert, P. P.</span><span> </span><span class="NLM_article-title">Structure-Based Identification of OATP1B1/3 Inhibitors</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1257</span><span class="NLM_x">–</span> <span class="NLM_lpage">1267</span><span class="refDoi"> DOI: 10.1124/mol.112.084152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.112.084152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23571415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=1257-1267&issue=6&author=T.+De+Bruynauthor=G.+J.+P.+van+Westenauthor=A.+P.+Ijzermanauthor=B.+Stiegerauthor=P.+de+Witteauthor=P.+F.+Augustijnsauthor=P.+P.+Annaert&title=Structure-Based+Identification+of+OATP1B1%2F3+Inhibitors&doi=10.1124%2Fmol.112.084152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based identification of OATP1B1/3 inhibitors</span></div><div class="casAuthors">De Bruyn, Tom; van Western, Gerard J. P.; IJzerman, Adriaan P.; Stieger, Bruno; de Witte, Peter; Augustijns, Patrick F.; Annaert, Pieter P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1257-1267</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Several recent studies show that inhibition of the hepatic transport proteins org. anion-transporting polypeptide 1B1 (OATP1B1) and 1B3 (OATP1B3) can result in clin. relevant drug-drug interactions (DDI).  To avoid late-stage development drug failures due to OATP1B-mediated DDI, predictive in vitro and in silico methods should be implemented at an early stage of the drug candidate evaluation process.  In the present study, we first developed a high-throughput in vitro transporter inhibition assay for the OATP1B subfamily.  A total of 2000 compds. were tested as potential modulators of the uptake of the OATP1B substrate sodium fluorescein, in OATP1B1- or 1B3-transfected Chinese hamster ovary cells.  At an equimolar substrate-inhibitor concn. of 10 μM, 212 and 139 mols. were identified as OATP1B1 and OATP1B3 inhibitors, resp. (min. 50% inhibition).  For 69 compds., previously not identified as OATP1B inhibitors, concn.-dependent inhibition was also detd., yielding Ki values ranging from 0.06 to 6.5 μM.  Based on these in vitro data, we subsequently developed a proteochemometrics-based in silico model, which predicted OATP1B inhibitors in the test group (20% of the dataset) with high specificity (86%) and sensitivity (78%).  Moreover, several physicochem. compd. properties and substructures related to OATP1B1/1B3 inhibition or inactivity were identified.  Finally, model performance was prospectively verified with a set of 54 compds. not included in the original dataset.  This validation indicated that 80 and 74% of the compds. were correctly classified for OATP1B1 and OATP1B3 inhibition, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktLtqKIqWh7Vg90H21EOLACvtfcHk0lh4l6VENRIXAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D&md5=bbf1617e8efba061f65b8f643baa5b98</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.084152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.084152%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BBruyn%26aufirst%3DT.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DStieger%26aufirst%3DB.%26aulast%3Dde%2BWitte%26aufirst%3DP.%26aulast%3DAugustijns%26aufirst%3DP.%2BF.%26aulast%3DAnnaert%26aufirst%3DP.%2BP.%26atitle%3DStructure-Based%2520Identification%2520of%2520OATP1B1%252F3%2520Inhibitors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26issue%3D6%26spage%3D1257%26epage%3D1267%26doi%3D10.1124%2Fmol.112.084152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Svetnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feuston, B. P.</span><span> </span><span class="NLM_article-title">Random Forest: a Classification and Regression Tool for Compound Classification and QSAR Modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1947</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1021/ci034160g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci034160g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=1947-1958&issue=6&author=V.+Svetnikauthor=A.+Liawauthor=C.+Tongauthor=J.+C.+Culbersonauthor=R.+P.+Sheridanauthor=B.+P.+Feuston&title=Random+Forest%3A+a+Classification+and+Regression+Tool+for+Compound+Classification+and+QSAR+Modeling&doi=10.1021%2Fci034160g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Random Forest: A Classification and Regression Tool for Compound Classification and QSAR Modeling</span></div><div class="casAuthors">Svetnik, Vladimir; Liaw, Andy; Tong, Christopher; Culberson, J. Christopher; Sheridan, Robert P.; Feuston, Bradley P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1947-1958</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new classification and regression tool, Random Forest, is introduced and investigated for predicting a compd.'s quant. or categorical biol. activity based on a quant. description of the compd.'s mol. structure.  Random Forest is an ensemble of unpruned classification or regression trees created by using bootstrap samples of the training data and random feature selection in tree induction.  Prediction is made by aggregating (majority vote or averaging) the predictions of the ensemble.  The authors built predictive models for six cheminformatics data sets.  The authors anal. demonstrates that Random Forest is a powerful tool capable of delivering performance that is among the most accurate methods to date.  The authors also present three addnl. features of Random Forest: built-in performance assessment, a measure of relative importance of descriptors, and a measure of compd. similarity that is weighted by the relative importance of descriptors.  It is the combination of relatively high prediction accuracy and its collection of desired features that makes Random Forest uniquely suited for modeling in cheminformatics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjoRt5v7rYwLVg90H21EOLACvtfcHk0lh4l6VENRIXAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D&md5=dea7867551ec30260b0091b90593a660</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci034160g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034160g%26sid%3Dliteratum%253Aachs%26aulast%3DSvetnik%26aufirst%3DV.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DC.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DFeuston%26aufirst%3DB.%2BP.%26atitle%3DRandom%2520Forest%253A%2520a%2520Classification%2520and%2520Regression%2520Tool%2520for%2520Compound%2520Classification%2520and%2520QSAR%2520Modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2003%26volume%3D43%26issue%3D6%26spage%3D1947%26epage%3D1958%26doi%3D10.1021%2Fci034160g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Wittwer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zur, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosaka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Discovery of Potent, Selective Multidrug and Toxin Extrusion Transporter 1 (MATE1, SLC47A1) Inhibitors Through Prescription Drug Profiling and Computational Modeling</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">781</span><span class="NLM_x">–</span> <span class="NLM_lpage">795</span><span class="refDoi"> DOI: 10.1021/jm301302s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301302s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjurjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=781-795&issue=3&author=M.+B.+Wittwerauthor=A.+A.+Zurauthor=N.+Khuriauthor=Y.+Kidoauthor=A.+Kosakaauthor=X.+Zhangauthor=K.+M.+Morrisseyauthor=A.+Saliauthor=Y.+Huangauthor=K.+M.+Giacomini&title=Discovery+of+Potent%2C+Selective+Multidrug+and+Toxin+Extrusion+Transporter+1+%28MATE1%2C+SLC47A1%29+Inhibitors+Through+Prescription+Drug+Profiling+and+Computational+Modeling&doi=10.1021%2Fjm301302s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent, Selective Multidrug and Toxin Extrusion Transporter 1 (MATE1, SLC47A1) Inhibitors Through Prescription Drug Profiling and Computational Modeling</span></div><div class="casAuthors">Wittwer, Matthias B.; Zur, Arik A.; Khuri, Natalia; Kido, Yasuto; Kosaka, Alan; Zhang, Xuexiang; Morrissey, Kari M.; Sali, Andrej; Huang, Yong; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">781-795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The human multidrug and toxin extrusion (MATE) transporter 1 contributes to the tissue distribution and excretion of many drugs.  Inhibition of MATE1 may result in potential drug-drug interactions (DDIs) and alterations in drug exposure and accumulation in various tissues.  The primary goals of this project were to identify MATE1 inhibitors with clin. importance or in vitro utility and to elucidate the physicochem. properties that differ between MATE1 and OCT2 inhibitors.  Using a fluorescence assay of ASP+ uptake in cells stably expressing MATE1, over 900 prescription drugs were screened and 84 potential MATE1 inhibitors were found.  We identified several MATE1 selective inhibitors including four FDA-approved medications that may be clin. relevant MATE1 inhibitors and could cause a clin. DDI.  In parallel, a QSAR model identified distinct mol. properties of MATE1 vs. OCT2 inhibitors and was used to screen the DrugBank in silico library for new hits in a larger chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonSgabmRJ4orVg90H21EOLACvtfcHk0lh4l6VENRIXAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjurjE&md5=9c26a96ca5db7601155b54f8b8818af3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm301302s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301302s%26sid%3Dliteratum%253Aachs%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26aulast%3DZur%26aufirst%3DA.%2BA.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DKido%26aufirst%3DY.%26aulast%3DKosaka%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMorrissey%26aufirst%3DK.%2BM.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DDiscovery%2520of%2520Potent%252C%2520Selective%2520Multidrug%2520and%2520Toxin%2520Extrusion%2520Transporter%25201%2520%2528MATE1%252C%2520SLC47A1%2529%2520Inhibitors%2520Through%2520Prescription%2520Drug%2520Profiling%2520and%2520Computational%2520Modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D3%26spage%3D781%26epage%3D795%26doi%3D10.1021%2Fjm301302s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Wishart, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djoumbou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandal, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aziat, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouatra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinelnikov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arndt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yallou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjorndahl, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Pineiro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neveu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greiner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scalbert, A.</span><span> </span><span class="NLM_article-title">HMDB 3.0--the Human Metabolome Database in 2013</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D801</span><span class="NLM_x">–</span> <span class="NLM_lpage">D807</span><span class="refDoi"> DOI: 10.1093/nar/gks1065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fnar%2Fgks1065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=D801-D807&issue=D1&author=D.+S.+Wishartauthor=T.+Jewisonauthor=A.+C.+Guoauthor=M.+Wilsonauthor=C.+Knoxauthor=Y.+Liuauthor=Y.+Djoumbouauthor=R.+Mandalauthor=F.+Aziatauthor=E.+Dongauthor=S.+Bouatraauthor=I.+Sinelnikovauthor=D.+Arndtauthor=J.+Xiaauthor=P.+Liuauthor=F.+Yallouauthor=T.+Bjorndahlauthor=R.+Perez-Pineiroauthor=R.+Eisnerauthor=F.+Allenauthor=V.+Neveuauthor=R.+Greinerauthor=A.+Scalbert&title=HMDB+3.0%2D%2Dthe+Human+Metabolome+Database+in+2013&doi=10.1093%2Fnar%2Fgks1065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks1065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks1065%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DJewison%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DWilson%26aufirst%3DM.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDjoumbou%26aufirst%3DY.%26aulast%3DMandal%26aufirst%3DR.%26aulast%3DAziat%26aufirst%3DF.%26aulast%3DDong%26aufirst%3DE.%26aulast%3DBouatra%26aufirst%3DS.%26aulast%3DSinelnikov%26aufirst%3DI.%26aulast%3DArndt%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DYallou%26aufirst%3DF.%26aulast%3DBjorndahl%26aufirst%3DT.%26aulast%3DPerez-Pineiro%26aufirst%3DR.%26aulast%3DEisner%26aufirst%3DR.%26aulast%3DAllen%26aufirst%3DF.%26aulast%3DNeveu%26aufirst%3DV.%26aulast%3DGreiner%26aufirst%3DR.%26aulast%3DScalbert%26aufirst%3DA.%26atitle%3DHMDB%25203.0--the%2520Human%2520Metabolome%2520Database%2520in%25202013%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26issue%3DD1%26spage%3DD801%26epage%3DD807%26doi%3D10.1093%2Fnar%2Fgks1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Kotake, Y.</span><span> </span><span class="NLM_article-title">[Tetrahydroisoquinoline Derivatives as Possible Parkinson’s Disease-Inducing Substances]</span> <span class="citation_source-journal">Yakugaku Zasshi</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">975</span><span class="NLM_x">–</span> <span class="NLM_lpage">982</span><span class="refDoi"> DOI: 10.1248/yakushi.122.975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1248%2Fyakushi.122.975" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2002&pages=975-982&issue=11&author=Y.+Kotake&title=%5BTetrahydroisoquinoline+Derivatives+as+Possible+Parkinson%E2%80%99s+Disease-Inducing+Substances%5D&doi=10.1248%2Fyakushi.122.975"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1248%2Fyakushi.122.975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fyakushi.122.975%26sid%3Dliteratum%253Aachs%26aulast%3DKotake%26aufirst%3DY.%26atitle%3D%255BTetrahydroisoquinoline%2520Derivatives%2520as%2520Possible%2520Parkinson%25E2%2580%2599s%2520Disease-Inducing%2520Substances%255D%26jtitle%3DYakugaku%2520Zasshi%26date%3D2002%26volume%3D122%26issue%3D11%26spage%3D975%26epage%3D982%26doi%3D10.1248%2Fyakushi.122.975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Kotake, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makino, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirobe, M.</span><span> </span><span class="NLM_article-title">1-Benzyl-1,2,3,4-Tetrahydroisoquinoline as a Parkinsonism-Inducing Agent: a Novel Endogenous Amine in Mouse Brain and Parkinsonian CSF</span> <span class="citation_source-journal">J. Neurochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">2633</span><span class="NLM_x">–</span> <span class="NLM_lpage">2638</span><span class="refDoi"> DOI: 10.1046/j.1471-4159.1995.65062633.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1046%2Fj.1471-4159.1995.65062633.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=7595560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADyaK2MXpsFersbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1995&pages=2633-2638&issue=6&author=Y.+Kotakeauthor=Y.+Tasakiauthor=Y.+Makinoauthor=S.+Ohtaauthor=M.+Hirobe&title=1-Benzyl-1%2C2%2C3%2C4-Tetrahydroisoquinoline+as+a+Parkinsonism-Inducing+Agent%3A+a+Novel+Endogenous+Amine+in+Mouse+Brain+and+Parkinsonian+CSF&doi=10.1046%2Fj.1471-4159.1995.65062633.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">1-Benzyl-1,2,3,4-tetrahydroisoquinoline as a parkinsonism-inducing agent: a novel endogenous amine in mouse brain and parkinsonian CSF</span></div><div class="casAuthors">Kotake, Yaichiro; Tasaki, Yoshikazu; Makino, Yukiko; Ohta, Shigeru; Hirobe, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2633-8</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Lippincott-Raven</span>)
        </div><div class="casAbstract">1-Benzyl-1,2,3,4-tetrahydroisoquinoline (1BnTIQ) was detected as a novel endogenous amine in mouse brain and parkinsonian CSF by using the gas chromatog.-selected ion-monitoring method.  The level of 1BnTIQ was very high in CSF of some parkinsonian patients compared with that of controls with other neurol. diseases, the mean value being three times higher (parkinsonians: 1.17 ng/mL of CSF, vs. controls: 0.40 ng/mL of CSF; not significantly different).  The pole test, a toxicol. examn. to evaluate behavior abnormalities related to Parkinson's disease, was used to examine the pharmacol. effect of 1BnTIQ in mice.  Repeated administration of 1BnTIQ induced behavior abnormalities, which pretreatment with 1-methyl-1,2,3,4-tetrahydroisoquinoline could prevent.  The authors suggest that 1BnTIQ may be related to the idiopathic Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1xDVD1jNQbVg90H21EOLACvtfcHk0lisVi_QjhXRhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsFersbw%253D&md5=890490f56fd9ef3e7ef26a03befb3519</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.1995.65062633.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.1995.65062633.x%26sid%3Dliteratum%253Aachs%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DTasaki%26aufirst%3DY.%26aulast%3DMakino%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DS.%26aulast%3DHirobe%26aufirst%3DM.%26atitle%3D1-Benzyl-1%252C2%252C3%252C4-Tetrahydroisoquinoline%2520as%2520a%2520Parkinsonism-Inducing%2520Agent%253A%2520a%2520Novel%2520Endogenous%2520Amine%2520in%2520Mouse%2520Brain%2520and%2520Parkinsonian%2520CSF%26jtitle%3DJ.%2520Neurochem.%26date%3D1995%26volume%3D65%26issue%3D6%26spage%3D2633%26epage%3D2638%26doi%3D10.1046%2Fj.1471-4159.1995.65062633.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaner, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Functional Characterization of an Organic Cation Transporter (hOCT1) in a Transiently Transfected Human Cell Line (HeLa)</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">354</span><span class="NLM_x">–</span> <span class="NLM_lpage">361</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=9655880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADyaK1cXksFCgs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=1998&pages=354-361&issue=1&author=L.+Zhangauthor=M.+E.+Schanerauthor=K.+M.+Giacomini&title=Functional+Characterization+of+an+Organic+Cation+Transporter+%28hOCT1%29+in+a+Transiently+Transfected+Human+Cell+Line+%28HeLa%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of an organic cation transporter (hOCT1) in a transiently transfected human cell line (HeLa)</span></div><div class="casAuthors">Zhang, Lei; Schaner, Marci E.; Giacomini, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">354-361</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">Recently, a polyspecific org. cation transporter, hOCT1, was cloned from human liver.  To date, limited studies examg. the functional characteristics of the transporter have been performed.  The purpose of the present study was to develop a mammalian expression system for hOCT1 and to characterize the interactions of various compds. with the cloned transporter.  Lipofection was used to transiently transfect the hOCT1 plasmid DNA in a human cell line, HeLa.  We tested the interaction of an array of org. cations and other compds. with hOCT1 by detg. Ki values in inhibiting 14C-tetraethylammonium (TEA) transport in the transfected cells.  Transient expression of hOCT1 activity was obsd. between 24 and 72 h post-transfection, with maximal expression at approx. 40 h.  TEA transport was temp. dependent and saturable with Vmax and Km values of 2.89 ± 0.448 nmol/mg protein/30 min and 229 ± 78.4 μM, resp.  14C-TEA uptake in hOCT1 plasmid DNA-transfected HeLa cells was trans-stimulated by unlabeled TEA and 1-methyl-4-phenyl-pyridinium.  Org. cations, including clonidine, quinine, quinidine and verapamil (0.1 mM), significantly inhibited 14C-TEA uptake, whereas the org. anion, p-aminohippuric acid (5 mM), did not.  The neutral compds., corticosterone (Ki, 7.0 μM) and midazolam (Ki, 3.7 μM) potently inhibited 14C-TEA uptake.  The Ki values of several compds. in interacting with hOCT1 differed substantially from the corresponding values for the rat org. cation transporter, rOCT1, and the human kidney-specific org. cation transporter, hOCT2, detd. in previous studies.  Transiently transfected HeLa cells represent a useful tool in studying the interactions and kinetics of org. cations and other xenobiotics with hOCT1 and in understanding the mol. events involved in org. cation transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQuMS6U2vXBLVg90H21EOLACvtfcHk0lisVi_QjhXRhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksFCgs70%253D&md5=81ab14d6ccec01adf0aff127a3b2f911</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DSchaner%26aufirst%3DM.%2BE.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DFunctional%2520Characterization%2520of%2520an%2520Organic%2520Cation%2520Transporter%2520%2528hOCT1%2529%2520in%2520a%2520Transiently%2520Transfected%2520Human%2520Cell%2520Line%2520%2528HeLa%2529%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1998%26volume%3D286%26issue%3D1%26spage%3D354%26epage%3D361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovejoy, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shima, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagpacan, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapuk, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippard, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacomini, K. M.</span><span> </span><span class="NLM_article-title">Organic Cation Transporters Are Determinants of Oxaliplatin Cytotoxicity</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">8847</span><span class="NLM_x">–</span> <span class="NLM_lpage">8857</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-0769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1158%2F0008-5472.CAN-06-0769" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=8847-8857&issue=17&author=S.+Zhangauthor=K.+S.+Lovejoyauthor=J.+E.+Shimaauthor=L.+L.+Lagpacanauthor=Y.+Shuauthor=A.+Lapukauthor=Y.+Chenauthor=T.+Komoriauthor=J.+W.+Grayauthor=X.+Chenauthor=S.+J.+Lippardauthor=K.+M.+Giacomini&title=Organic+Cation+Transporters+Are+Determinants+of+Oxaliplatin+Cytotoxicity&doi=10.1158%2F0008-5472.CAN-06-0769"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-0769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-0769%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLovejoy%26aufirst%3DK.%2BS.%26aulast%3DShima%26aufirst%3DJ.%2BE.%26aulast%3DLagpacan%26aufirst%3DL.%2BL.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DLapuk%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKomori%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLippard%26aufirst%3DS.%2BJ.%26aulast%3DGiacomini%26aufirst%3DK.%2BM.%26atitle%3DOrganic%2520Cation%2520Transporters%2520Are%2520Determinants%2520of%2520Oxaliplatin%2520Cytotoxicity%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26issue%3D17%26spage%3D8847%26epage%3D8857%26doi%3D10.1158%2F0008-5472.CAN-06-0769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Kimura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narikawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enomoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endou, H.</span><span> </span><span class="NLM_article-title">Human Organic Anion Transporters and Human Organic Cation Transporters Mediate Renal Transport of Prostaglandins</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">301</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">298</span><span class="refDoi"> DOI: 10.1124/jpet.301.1.293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.301.1.293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=11907186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1Kqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=301&publication_year=2002&pages=293-298&issue=1&author=H.+Kimuraauthor=M.+Takedaauthor=S.+Narikawaauthor=A.+Enomotoauthor=K.+Ichidaauthor=H.+Endou&title=Human+Organic+Anion+Transporters+and+Human+Organic+Cation+Transporters+Mediate+Renal+Transport+of+Prostaglandins&doi=10.1124%2Fjpet.301.1.293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Human organic anion transporters and human organic cation transporters mediate renal transport of prostaglandins</span></div><div class="casAuthors">Kimura, Hiroaki; Takeda, Michio; Narikawa, Shinichi; Enomoto, Atsushi; Ichida, Kimiyoshi; Endou, Hitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">301</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">293-298</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Prostaglandin E2 (PGE2) and prostaglandin F2α (PGF2α) have been used for the induction of labor and the termination of pregnancy.  Renal excretion is shown to be an important pathway for the elimination of PGE2 and PGF2α.  The purpose of this study was to elucidate the mol. mechanism of renal PGE2 and PGF2α transport using cells stably expressing human org. anion transporter (hOAT) 1, hOAT2, hOAT3, and hOAT4; and human org. cation transporter (hOCT) 1 and hOCT2.  A time- and dose-dependent increase in PGE2 and PGF2α uptake was obsd. in cells expressing hOAT1, hOAT2, hOAT3, hOAT4, hOCT1, and hOCT2.  The Km values of PGE2 uptake by hOAT1, hOAT2, hOAT3, hOAT4, hOCT1, and hOCT2 were 970, 713, 345, 154, 657, and 28.9 nM, resp., whereas those of PGF2α uptake by hOAT1, hOAT3, hOAT4, hOCT1, and hOCT2 were 575, 1092, 692, 477, and 334 nM, resp.  PGE2 and PGF2α significantly inhibited org. anion uptake by hOATs and org. cation uptake by hOCTs.  In conclusion, considering the localization of these transporters, the results suggest that PGE2 and PGF2α transport in the basolateral membrane of the proximal tubule is mediated by multiple pathways including hOAT1, hOAT2, hOAT3, and hOCT2, whereas that in the apical side is mediated by hOAT4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2SONHwtVyO7Vg90H21EOLACvtfcHk0lhIc4yz1WKN9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1Kqs7o%253D&md5=dab78409f108dec560468cfb9969bdb5</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fjpet.301.1.293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.301.1.293%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DTakeda%26aufirst%3DM.%26aulast%3DNarikawa%26aufirst%3DS.%26aulast%3DEnomoto%26aufirst%3DA.%26aulast%3DIchida%26aufirst%3DK.%26aulast%3DEndou%26aufirst%3DH.%26atitle%3DHuman%2520Organic%2520Anion%2520Transporters%2520and%2520Human%2520Organic%2520Cation%2520Transporters%2520Mediate%2520Renal%2520Transport%2520of%2520Prostaglandins%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D301%26issue%3D1%26spage%3D293%26epage%3D298%26doi%3D10.1124%2Fjpet.301.1.293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Bednarczyk, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wikel, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Influence of Molecular Structure on Substrate Binding to the Human Organic Cation Transporter, hOCT1</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1124/mol.63.3.489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fmol.63.3.489" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=489-498&issue=3&author=D.+Bednarczykauthor=S.+Ekinsauthor=J.+H.+Wikelauthor=S.+H.+Wright&title=Influence+of+Molecular+Structure+on+Substrate+Binding+to+the+Human+Organic+Cation+Transporter%2C+hOCT1&doi=10.1124%2Fmol.63.3.489"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1124%2Fmol.63.3.489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.63.3.489%26sid%3Dliteratum%253Aachs%26aulast%3DBednarczyk%26aufirst%3DD.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DWikel%26aufirst%3DJ.%2BH.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DInfluence%2520of%2520Molecular%2520Structure%2520on%2520Substrate%2520Binding%2520to%2520the%2520Human%2520Organic%2520Cation%2520Transporter%252C%2520hOCT1%26jtitle%3DMol.%2520Pharmacol.%26date%3D2003%26volume%3D63%26issue%3D3%26spage%3D489%26epage%3D498%26doi%3D10.1124%2Fmol.63.3.489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Moaddel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravichandran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bighi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaguchi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainer, I. W.</span><span> </span><span class="NLM_article-title">Pharmacophore Modelling of Stereoselective Binding to the Human Organic Cation Transporter (hOCT1)</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">151</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1305</span><span class="NLM_x">–</span> <span class="NLM_lpage">1314</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fsj.bjp.0707341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=17592512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFehsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2007&pages=1305-1314&issue=8&author=R.+Moaddelauthor=S.+Ravichandranauthor=F.+Bighiauthor=R.+Yamaguchiauthor=I.+W.+Wainer&title=Pharmacophore+Modelling+of+Stereoselective+Binding+to+the+Human+Organic+Cation+Transporter+%28hOCT1%29&doi=10.1038%2Fsj.bjp.0707341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore modeling of stereoselective binding to the human organic cation transporter (hOCT1)</span></div><div class="casAuthors">Moaddel, R.; Ravichandran, S.; Bighi, F.; Yamaguchi, R.; Wainer, I. W.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1305-1314</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The human org. cation transporter-1 (hOCT1) is a polyspecific transporter that plays a role in drug distribution, metab. and excretion.  Previous studies have demonstrated that hOCT1 binding can be stereoselective, but the mechanism for stereochem. recognition has not been described.  The purpose of this study was to develop a pharmacophore model to describe stereoselective binding to hOCT1.  Exptl. approach: A set of 22 compds. including 8 pairs of enantiomers and five pairs of diastereomers was used to develop a pharmacophore model.  The pharmacophore modeling was carried out using Catalyst version 4.11 and HypoGen and was based upon the correlation of the structures and activities (Ki values) of the compds. used in the study.  The resulting model contained a pos. ion, hydrophobic and two hydrogen-bond acceptor interaction sites.  The relative enantioselectivity of 8/8 enantiomeric pairs and diastereoselectivity of 5/5 diastereomers was described by mapping to a combination of at least 3 of the 4 functional feature sites of the model.  The pharmacophore model describes stereoselective interactions with hOCT1 at one of the binding sites on the mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXDIit5N8HbVg90H21EOLACvtfcHk0lhIc4yz1WKN9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFehsL4%253D&md5=8a18c5b919d50b37f4497777841fc864</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707341%26sid%3Dliteratum%253Aachs%26aulast%3DMoaddel%26aufirst%3DR.%26aulast%3DRavichandran%26aufirst%3DS.%26aulast%3DBighi%26aufirst%3DF.%26aulast%3DYamaguchi%26aufirst%3DR.%26aulast%3DWainer%26aufirst%3DI.%2BW.%26atitle%3DPharmacophore%2520Modelling%2520of%2520Stereoselective%2520Binding%2520to%2520the%2520Human%2520Organic%2520Cation%2520Transporter%2520%2528hOCT1%2529%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D151%26issue%3D8%26spage%3D1305%26epage%3D1314%26doi%3D10.1038%2Fsj.bjp.0707341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Harper, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Multiple Mechanisms of Ligand Interaction with the Human Organic Cation Transporter, OCT2</span> <span class="citation_source-journal">Am. J. Physiol. Renal Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">304</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">F56</span><span class="NLM_x">–</span> <span class="NLM_lpage">F67</span><span class="refDoi"> DOI: 10.1152/ajprenal.00486.2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1152%2Fajprenal.00486.2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23034939" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2013&pages=F56-F67&issue=1&author=J.+N.+Harperauthor=S.+H.+Wright&title=Multiple+Mechanisms+of+Ligand+Interaction+with+the+Human+Organic+Cation+Transporter%2C+OCT2&doi=10.1152%2Fajprenal.00486.2012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1152%2Fajprenal.00486.2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajprenal.00486.2012%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DJ.%2BN.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DMultiple%2520Mechanisms%2520of%2520Ligand%2520Interaction%2520with%2520the%2520Human%2520Organic%2520Cation%2520Transporter%252C%2520OCT2%26jtitle%3DAm.%2520J.%2520Physiol.%2520Renal%2520Physiol.%26date%3D2013%26volume%3D304%26issue%3D1%26spage%3DF56%26epage%3DF67%26doi%3D10.1152%2Fajprenal.00486.2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Computational Modeling to Accelerate the Identification of Substrates and Inhibitors for Transporters That Affect Drug Disposition</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">661</span><span class="NLM_x">–</span> <span class="NLM_lpage">665</span><span class="refDoi"> DOI: 10.1038/clpt.2012.164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fclpt.2012.164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23010651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=661-665&issue=5&author=S.+Ekinsauthor=J.+E.+Polliauthor=P.+W.+Swaanauthor=S.+H.+Wright&title=Computational+Modeling+to+Accelerate+the+Identification+of+Substrates+and+Inhibitors+for+Transporters+That+Affect+Drug+Disposition&doi=10.1038%2Fclpt.2012.164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Modeling to Accelerate the Identification of Substrates and Inhibitors for Transporters That Affect Drug Disposition</span></div><div class="casAuthors">Ekins, S.; Polli, J. E.; Swaan, P. W.; Wright, S. H.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">661-665</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review describes the increased use of computational approaches to predicting drug interactions with human transporters that affect drug disposition and may lead to toxicity.  Ligand-based methods predominantly use limited exptl. data but provide new insights into structure-activity relationships.  Development of models capable of identifying new transport substrates and unwanted drug-drug interactions requires novel applications of current computational methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2kRgkvrcdObVg90H21EOLACvtfcHk0lhTBcX3bZhoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChsbjF&md5=47b55b7738690be24d97a57e121bf342</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2012.164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2012.164%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DPolli%26aufirst%3DJ.%2BE.%26aulast%3DSwaan%26aufirst%3DP.%2BW.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DComputational%2520Modeling%2520to%2520Accelerate%2520the%2520Identification%2520of%2520Substrates%2520and%2520Inhibitors%2520for%2520Transporters%2520That%2520Affect%2520Drug%2520Disposition%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2012%26volume%3D92%26issue%3D5%26spage%3D661%26epage%3D665%26doi%3D10.1038%2Fclpt.2012.164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Martínez-Guerrero, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. H.</span><span> </span><span class="NLM_article-title">Substrate-Dependent Inhibition of Human MATE1 by Cationic Ionic Liquids</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">495</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1124/jpet.113.204206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1124%2Fjpet.113.204206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23785176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCmtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2013&pages=495-503&issue=3&author=L.+J.+Mart%C3%ADnez-Guerreroauthor=S.+H.+Wright&title=Substrate-Dependent+Inhibition+of+Human+MATE1+by+Cationic+Ionic+Liquids&doi=10.1124%2Fjpet.113.204206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-dependent inhibition of human MATE1 by cationic ionic liquids</span></div><div class="casAuthors">Martinez-Guerrero, Lucy J.; Wright, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-503</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The multidrug and toxin extruders 1- and 2-K (MATE1 and MATE2-K) are expressed in the luminal membrane of renal proximal tubule cells and provide the active step in the secretion of mols. that carry a net pos. charge at physiol. pH, so-called org. cations.  The present study tested whether structurally distinct MATE substrates can display different quant. profiles of inhibition when interacting with structurally distinct ligands.  The tested ligands were three structurally similar cationic ionic liqs. (ILs, salts in the liq. state: N-butylpyridinium, NBuPy; 1-methyl-3-butylimidazolium, Bmim; and N-butyl-N-methylpyrrolidinium, BmPy).  Uptake was measured using Chinese hamster ovary cells that stably expressed MATE1 or MATE2-K.  By trans-stimulation, all three ILs were transported by both MATE transporters.  The three ILs also inhibited uptake of three structurally distinct MATE substrates: 1-methyl-4-phenylpyridinium (MPP), triethylmethylammonium (TEMA), and N,N,N-trimethyl-2-[methyl(7-nitrobenzo[c][1,2,5]oxadiazol-4-yl)amino]ethanaminium (NBD-MTMA).  MATE1 displayed a higher affinity for the pyridinium-based NBuPy (IC50 values, 2-4 μM) than for either the pyrrolidinium- (BmPy; 20-70 μM) or imidazolium-based ILs (Bmim; 15-60 μM).  Inhibition of MPP, TEMA, and NBD-MTMA transport by NBuPy was competitive, with comparable Ki values against all substrates.  Bmim also competitively blocked the three substrates but with Ki values that differed significantly (20 μM against MPP and 30 μM against NBD-MTMA vs. 60 μM against TEMA).  Together, these data indicate that renal secretion of ILs by the human kidney involves MATE transporters and suggest that the mechanism of transport inhibition is ligand-dependent, supporting the hypothesis that the binding of substrates to MATE transporters involves interaction with a binding surface with multiple binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiPXNU1hIVbbVg90H21EOLACvtfcHk0lhTBcX3bZhoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCmtbbP&md5=50bcb1341615d29158f74b5b15627d73</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1124%2Fjpet.113.204206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.113.204206%26sid%3Dliteratum%253Aachs%26aulast%3DMart%25C3%25ADnez-Guerrero%26aufirst%3DL.%2BJ.%26aulast%3DWright%26aufirst%3DS.%2BH.%26atitle%3DSubstrate-Dependent%2520Inhibition%2520of%2520Human%2520MATE1%2520by%2520Cationic%2520Ionic%2520Liquids%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2013%26volume%3D346%26issue%3D3%26spage%3D495%26epage%3D503%26doi%3D10.1124%2Fjpet.113.204206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Tweedie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poirier, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span> </span><span class="NLM_article-title">International Transporter Consortium. Transporter Studies in Drug Development: Experience to Date and Follow-Up on Decision Trees From the International Transporter Consortium</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">125</span><span class="refDoi"> DOI: 10.1038/clpt.2013.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fclpt.2013.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=23588318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A280%3ADC%252BC3srltl2ruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2013&pages=113-125&issue=1&author=D.+Tweedieauthor=J.+W.+Polliauthor=E.+G.+Berglundauthor=S.+M.+Huangauthor=L.+Zhangauthor=A.+Poirierauthor=X.+Chuauthor=B.+Feng&title=International+Transporter+Consortium.+Transporter+Studies+in+Drug+Development%3A+Experience+to+Date+and+Follow-Up+on+Decision+Trees+From+the+International+Transporter+Consortium&doi=10.1038%2Fclpt.2013.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Transporter studies in drug development: experience to date and follow-up on decision trees from the International Transporter Consortium</span></div><div class="casAuthors">Tweedie D; Polli J W; Berglund E Gil; Huang S M; Zhang L; Poirier A; Chu X; Feng B</div><div class="citationInfo"><span class="NLM_cas:title">Clinical pharmacology and therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The International Transporter Consortium (ITC) organized a second workshop in March 2012 to expand on the themes developed during the inaugural ITC workshop held in 2008.  The final session of the workshop provided perspectives from regulatory and industry-based scientists, with input from academic scientists, and focused primarily on the decision trees published from the first workshop.  These decision trees have become a central part of subsequent regulatory drug-drug interaction (DDI) guidances issued over the past few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTujrz08uS8s1WTCLT8lX9SfW6udTcc2eZxkR0coJ3YZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srltl2ruw%253D%253D&md5=bc460caa642b9132681aa4184f8c9f58</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2013.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2013.77%26sid%3Dliteratum%253Aachs%26aulast%3DTweedie%26aufirst%3DD.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DBerglund%26aufirst%3DE.%2BG.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPoirier%26aufirst%3DA.%26aulast%3DChu%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DB.%26atitle%3DInternational%2520Transporter%2520Consortium.%2520Transporter%2520Studies%2520in%2520Drug%2520Development%253A%2520Experience%2520to%2520Date%2520and%2520Follow-Up%2520on%2520Decision%2520Trees%2520From%2520the%2520International%2520Transporter%2520Consortium%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2013%26volume%3D94%26issue%3D1%26spage%3D113%26epage%3D125%26doi%3D10.1038%2Fclpt.2013.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Abe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Utsunomiya, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinohara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyatake, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sano, T.</span><span> </span><span class="NLM_article-title">Biochemical and Pathological Study of Endogenous 1-Benzyl-1,2,3,4-Tetrahydroisoquinoline-Induced Parkinsonism in the Mouse</span> <span class="citation_source-journal">Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">907</span><span class="NLM_x"> (</span><span class="NLM_issue">1–2</span><span class="NLM_x">) </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span><span class="refDoi"> DOI: 10.1016/S0006-8993(01)02573-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1016%2FS0006-8993%2801%2902573-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=11430895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Onurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=907&publication_year=2001&pages=134-138&issue=1%E2%80%932&author=K.+Abeauthor=K.+Taguchiauthor=T.+Wasaiauthor=J.+Renauthor=I.+Utsunomiyaauthor=T.+Shinoharaauthor=T.+Miyatakeauthor=T.+Sano&title=Biochemical+and+Pathological+Study+of+Endogenous+1-Benzyl-1%2C2%2C3%2C4-Tetrahydroisoquinoline-Induced+Parkinsonism+in+the+Mouse&doi=10.1016%2FS0006-8993%2801%2902573-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical and pathological study of endogenous 1-benzyl-1,2,3,4-tetrahydroisoquinoline-induced parkinsonism in the mouse</span></div><div class="casAuthors">Abe, K.; Taguchi, K.; Wasai, T.; Ren, J.; Utsunomiya, I.; Shinohara, T.; Miyatake, T.; Sano, T.</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">907</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">134-138</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The authors administered 1-benzyl-1,2,3,4-tetrahydroisoquinoline (1-BnTIQ; 80 mg/kg, i.p.), an endogenous neurotoxin known to cause bradykinesia, the Parkinson's disease-like symptom, to obtain biochem. and pathol. evidence of behavioral abnormalities.  Immunohistochem. anal. demonstrated that 1-BnTIQ did not decrease the no. of tyrosine hydroxylase-pos. cells in the substantia nigra.  Biochem. anal. demonstrated significantly increased striatal dopamine (DA) content while DA metabolites in the striatum remained at control levels.  Thus, the 1-BnTIQ-induced bradykinesia has a different mechanism of action than that underlying the MPTP-induced depletion of striatal DA neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqYuZ0JkjIJLVg90H21EOLACvtfcHk0li3kJcSRgyauQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Onurg%253D&md5=ef52a6650394ecda1ba0f93a5700cb5b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0006-8993%2801%2902573-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-8993%252801%252902573-2%26sid%3Dliteratum%253Aachs%26aulast%3DAbe%26aufirst%3DK.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DWasai%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DUtsunomiya%26aufirst%3DI.%26aulast%3DShinohara%26aufirst%3DT.%26aulast%3DMiyatake%26aufirst%3DT.%26aulast%3DSano%26aufirst%3DT.%26atitle%3DBiochemical%2520and%2520Pathological%2520Study%2520of%2520Endogenous%25201-Benzyl-1%252C2%252C3%252C4-Tetrahydroisoquinoline-Induced%2520Parkinsonism%2520in%2520the%2520Mouse%26jtitle%3DBrain%2520Res.%26date%3D2001%26volume%3D907%26issue%3D1%25E2%2580%25932%26spage%3D134%26epage%3D138%26doi%3D10.1016%2FS0006-8993%2801%2902573-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Kotake, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirobe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, S.</span><span> </span><span class="NLM_article-title">Chronic Administration of 1-Benzyl-1,2,3,4-Tetrahydroisoquinoline, an Endogenous Amine in the Brain, Induces Parkinsonism in a Primate</span> <span class="citation_source-journal">Neurosci. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">217</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1016/0304-3940(96)13065-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1016%2F0304-3940%2896%2913065-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=1996&pages=69-71&issue=1&author=Y.+Kotakeauthor=M.+Yoshidaauthor=M.+Ogawaauthor=Y.+Tasakiauthor=M.+Hirobeauthor=S.+Ohta&title=Chronic+Administration+of+1-Benzyl-1%2C2%2C3%2C4-Tetrahydroisoquinoline%2C+an+Endogenous+Amine+in+the+Brain%2C+Induces+Parkinsonism+in+a+Primate&doi=10.1016%2F0304-3940%2896%2913065-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0304-3940%2896%2913065-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3940%252896%252913065-2%26sid%3Dliteratum%253Aachs%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DOgawa%26aufirst%3DM.%26aulast%3DTasaki%26aufirst%3DY.%26aulast%3DHirobe%26aufirst%3DM.%26aulast%3DOhta%26aufirst%3DS.%26atitle%3DChronic%2520Administration%2520of%25201-Benzyl-1%252C2%252C3%252C4-Tetrahydroisoquinoline%252C%2520an%2520Endogenous%2520Amine%2520in%2520the%2520Brain%252C%2520Induces%2520Parkinsonism%2520in%2520a%2520Primate%26jtitle%3DNeurosci.%2520Lett.%26date%3D1996%26volume%3D217%26issue%3D1%26spage%3D69%26epage%3D71%26doi%3D10.1016%2F0304-3940%2896%2913065-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Pei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grishin, N. V.</span><span> </span><span class="NLM_article-title">PROMALS3D: a Tool for Multiple Protein Sequence and Structure Alignments</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2295</span><span class="NLM_x">–</span> <span class="NLM_lpage">2300</span><span class="refDoi"> DOI: 10.1093/nar/gkn072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fnar%2Fgkn072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=18287115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFGgtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=2295-2300&issue=7&author=J.+Peiauthor=B.-H.+Kimauthor=N.+V.+Grishin&title=PROMALS3D%3A+a+Tool+for+Multiple+Protein+Sequence+and+Structure+Alignments&doi=10.1093%2Fnar%2Fgkn072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">PROMALS3D: a tool for multiple protein sequence and structure alignments</span></div><div class="casAuthors">Pei, Jimin; Kim, Bong-Hyun; Grishin, Nick V.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2295-2300</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although multiple sequence alignments (MSAs) are essential for a wide range of applications from structure modeling to prediction of functional sites, construction of accurate MSAs for distantly related proteins remains a largely unsolved problem.  The rapidly increasing database of spatial structures is a valuable source to improve alignment quality.  The authors explore the use of 3D structural information to guide sequence alignments constructed by their MSA program PROMALS.  The resulting tool, PROMALS3D, automatically identifies homologs with known 3D structures for the input sequences, derives structural constraints through structure-based alignments and combines them with sequence constraints to construct consistency-based multiple sequence alignments.  The output is a consensus alignment that brings together sequence and structural information about input proteins and their homologs.  PROMALS3D can also align sequences of multiple input structures, with the output representing a multiple structure-based alignment refined in combination with sequence constraints.  The advantage of PROMALS3D is that it gives researchers an easy way to produce high-quality alignments consistent with both sequences and structures of proteins.  PROMALS3D outperforms a no. of existing methods for constructing multiple sequence or structural alignments using both ref.-dependent and ref.-independent evaluation methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoid8W72Zq7JLVg90H21EOLACvtfcHk0li3kJcSRgyauQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFGgtrs%253D&md5=8487ac1df3b9d1dc24976dcf412dd7ef</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn072%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DB.-H.%26aulast%3DGrishin%26aufirst%3DN.%2BV.%26atitle%3DPROMALS3D%253A%2520a%2520Tool%2520for%2520Multiple%2520Protein%2520Sequence%2520and%2520Structure%2520Alignments%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26issue%3D7%26spage%3D2295%26epage%3D2300%26doi%3D10.1093%2Fnar%2Fgkn072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Edgar, R. C.</span><span> </span><span class="NLM_article-title">MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1792</span><span class="NLM_x">–</span> <span class="NLM_lpage">1797</span><span class="refDoi"> DOI: 10.1093/nar/gkh340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fnar%2Fgkh340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=15034147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisF2ks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=1792-1797&issue=5&author=R.+C.+Edgar&title=MUSCLE%3A+Multiple+Sequence+Alignment+with+High+Accuracy+and+High+Throughput&doi=10.1093%2Fnar%2Fgkh340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span></div><div class="casAuthors">Edgar, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1792-1797</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences.  Elements of the algorithm include fast distance estn. using kmer counting, progressive alignment using a new profile function we call the log-expectation score, and refinement using tree-dependent restricted partitioning.  The speed and accuracy of MUSCLE are compared with T-Coffee, MAFFT and CLUSTALW on four test sets of ref. alignments: BAliBASE, SABmark, SMART and a new benchmark, PREFAB.  MUSCLE achieves the highest, or joint highest, rank in accuracy on each of these sets.  Without refinement, MUSCLE achieves av. accuracy statistically indistinguishable from T-Coffee and MAFFT, and is the fastest of the tested methods for large nos. of sequences, aligning 5000 sequences of av. length 350 in 7 min on a current desktop computer.  The MUSCLE program, source code and PREFAB test data are freely available at http://www.drive5.com/muscle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUXqjEriOqy7Vg90H21EOLACvtfcHk0li3kJcSRgyauQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisF2ks7w%253D&md5=1414065843fa881a1b66b98f4c9cba2e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh340%26sid%3Dliteratum%253Aachs%26aulast%3DEdgar%26aufirst%3DR.%2BC.%26atitle%3DMUSCLE%253A%2520Multiple%2520Sequence%2520Alignment%2520with%2520High%2520Accuracy%2520and%2520High%2520Throughput%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26issue%3D5%26spage%3D1792%26epage%3D1797%26doi%3D10.1093%2Fnar%2Fgkh340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">The FTMap Family of Web Servers for Determining and Characterizing Ligand-Binding Hot Spots of Proteins</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&issue=5&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+Family+of+Web+Servers+for+Determining+and+Characterizing+Ligand-Binding+Hot+Spots+of+Proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0ljJg4aXhfKdVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520Family%2520of%2520Web%2520Servers%2520for%2520Determining%2520and%2520Characterizing%2520Ligand-Binding%2520Hot%2520Spots%2520of%2520Proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26issue%3D5%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking Sets for Molecular Docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">23</span><span class="NLM_x">) </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span><span class="refDoi"> DOI: 10.1021/jm0608356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&issue=23&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+Sets+for+Molecular+Docking&doi=10.1021%2Fjm0608356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0ljJg4aXhfKdVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520Sets%2520for%2520Molecular%2520Docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D23%26spage%3D6789%26epage%3D6801%26doi%3D10.1021%2Fjm0608356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Yap, C. W.</span><span> </span><span class="NLM_article-title">PaDEL-Descriptor: an Open Source Software to Calculate Molecular Descriptors and Fingerprints</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">1474</span><span class="refDoi"> DOI: 10.1002/jcc.21707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1002%2Fjcc.21707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=21425294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsF2isLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=1466-1474&issue=7&author=C.+W.+Yap&title=PaDEL-Descriptor%3A+an+Open+Source+Software+to+Calculate+Molecular+Descriptors+and+Fingerprints&doi=10.1002%2Fjcc.21707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">PaDEL-descriptor: An open source software to calculate molecular descriptors and fingerprints</span></div><div class="casAuthors">Yap, Chun Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1466-1474</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Introduction : PaDEL-Descriptor is a software for calcg. mol. descriptors and fingerprints.  The software currently calcs. 797 descriptors (663 1D, 2D descriptors, and 134 3D descriptors) and 10 types of fingerprints.  These descriptors and fingerprints are calcd. mainly using The Chem. Development Kit.  Some addnl. descriptors and fingerprints were added, which include atom type electrotopol. state descriptors, McGowan vol., mol. linear free energy relation descriptors, ring counts, count of chem. substructures identified by Laggner, and binary fingerprints and count of chem. substructures identified by Klekota and Roth.  Methods : PaDEL-Descriptor was developed using the Java language and consists of a library component and an interface component.  The library component allows it to be easily integrated into quant. structure activity relationship software to provide the descriptor calcn. feature while the interface component allows it to be used as a standalone software.  The software uses a Master/Worker pattern to take advantage of the multiple CPU cores that are present in most modern computers to speed up calcns. of mol. descriptors.  Results : The software has several advantages over existing standalone mol. descriptor calcn. software.  It is free and open source, has both graphical user interface and command line interfaces, can work on all major platforms (Windows, Linux, MacOS), supports more than 90 different mol. file formats, and is multithreaded.  Conclusion : PaDEL-Descriptor is a useful addn. to the currently available mol. descriptor calcn. software.  The software can be downloaded at . © 2010 Wiley Periodicals, Inc. J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_2-BJmBJEJrVg90H21EOLACvtfcHk0ljJg4aXhfKdVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsF2isLc%253D&md5=af74b9b0bc5621a6371e93caf8b2bfc7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21707%26sid%3Dliteratum%253Aachs%26aulast%3DYap%26aufirst%3DC.%2BW.%26atitle%3DPaDEL-Descriptor%253A%2520an%2520Open%2520Source%2520Software%2520to%2520Calculate%2520Molecular%2520Descriptors%2520and%2520Fingerprints%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26issue%3D7%26spage%3D1466%26epage%3D1474%26doi%3D10.1002%2Fjcc.21707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Molecular Docking Screens Using Comparative Models of Proteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2512</span><span class="NLM_x">–</span> <span class="NLM_lpage">2527</span><span class="refDoi"> DOI: 10.1021/ci9003706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2512-2527&issue=11&author=H.+Fanauthor=J.+J.+Irwinauthor=B.+M.+Webbauthor=G.+Klebeauthor=B.+K.+Shoichetauthor=A.+Sali&title=Molecular+Docking+Screens+Using+Comparative+Models+of+Proteins&doi=10.1021%2Fci9003706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screens Using Comparative Models of Proteins</span></div><div class="casAuthors">Fan, Hao; Irwin, John J.; Webb, Benjamin M.; Klebe, Gerhard; Shoichet, Brian K.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2512-2527</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two orders of magnitude more protein sequences can be modeled by comparative modeling than have been detd. by X-ray crystallog. and NMR spectroscopy.  Investigators have nevertheless been cautious about using comparative models for ligand discovery because of concerns about model errors.  We suggest how to exploit comparative models for mol. screens, based on docking against a wide range of crystallog. structures and comparative models with known ligands.  To account for the variation in the ligand-binding pocket as it binds different ligands, we calc. "consensus" enrichment by ranking each library compd. by its best docking score against all available comparative models and/or modeling templates.  For the majority of the targets, the consensus enrichment for multiple models was better than or comparable to that of the holo and apo X-ray structures.  Even for single models, the models are significantly more enriching than the template structure if the template is paralogous and shares more than 25% sequence identity with the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrgXhylX-BJrVg90H21EOLACvtfcHk0ljJg4aXhfKdVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M&md5=11bd5284c6b43592906e45b6aea3019d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci9003706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003706%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DMolecular%2520Docking%2520Screens%2520Using%2520Comparative%2520Models%2520of%2520Proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26issue%3D11%26spage%3D2512%26epage%3D2527%26doi%3D10.1021%2Fci9003706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carchia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">6582</span><span class="NLM_x">–</span> <span class="NLM_lpage">6594</span><span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&issue=14&author=M.+M.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+Useful+Decoys%2C+Enhanced+%28DUD-E%29%3A+Better+Ligands+and+Decoys+for+Better+Benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0ljKhZdp9G0IwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520Useful%2520Decoys%252C%2520Enhanced%2520%2528DUD-E%2529%253A%2520Better%2520Ligands%2520and%2520Decoys%2520for%2520Better%2520Benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D14%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—A Free Database of Commercially Available Compounds for Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&issue=1&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94A+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0ljKhZdp9G0IwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594A%2520Free%2520Database%2520of%2520Commercially%2520Available%2520Compounds%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26issue%3D1%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) From Screening Libraries and for Their Exclusion in Bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+From+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljKhZdp9G0IwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520From%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Cheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">FSelector: a Ruby Gem for Feature Selection</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">21</span><span class="NLM_x">) </span> <span class="NLM_fpage">2851</span><span class="NLM_x">–</span> <span class="NLM_lpage">2852</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bts528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1093%2Fbioinformatics%2Fbts528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=22942017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGjsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=2851-2852&issue=21&author=T.+Chengauthor=Y.+Wangauthor=S.+H.+Bryant&title=FSelector%3A+a+Ruby+Gem+for+Feature+Selection&doi=10.1093%2Fbioinformatics%2Fbts528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">FSelector: a Ruby gem for feature selection</span></div><div class="casAuthors">Cheng, Tiejun; Wang, Yanli; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2851-2852</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: The FSelector package contains a comprehensive list of feature selection algorithms for supporting bioinformatics and machine learning research.  FSelector primarily collects and implements the filter type of feature selection techniques, which are computationally efficient for mining large datasets.  In particular, FSelector allows ensemble feature selection that takes advantage of multiple feature selection algorithms to yield more robust results.  FSelector also provides many useful auxiliary tools, including normalization, discretization and missing data imputation.  Availability: FSelector, written in the Ruby programming language, is free and open-source software that runs on all Ruby supporting platforms, including Windows, Linux and Mac OS X. FSelector is available from https://rubygems.org/gems/fselector and can be installed like a breeze via the command gem install f selector.  The source code is available (https://github.com/need47/fselector) and is fully documented (http://rubydoc.info/gems/fselector/frames).  Contact: ywang cbi.nlm.nih.gov or bryant cbi.nlm.nih.gov Supplementary Information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhXzhX-g7Q7LVg90H21EOLACvtfcHk0lhvUpzFG9f_fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGjsr3L&md5=72fd6933b2f959b534ca1f634d1500e5</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts528%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DFSelector%253A%2520a%2520Ruby%2520Gem%2520for%2520Feature%2520Selection%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26issue%3D21%26spage%3D2851%26epage%3D2852%26doi%3D10.1093%2Fbioinformatics%2Fbts528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">R Development Core Team.</span><span class="NLM_x"> </span> <span class="citation_source-book">R: A Language and Environment for Statistical Computing</span>; <span class="NLM_publisher-name">The R Foundation for Statistical Computing</span>: <span class="NLM_publisher-loc">Vienna, Austria</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>available online at <a href="http://www.R-project.org/" class="extLink">http://www.R-project.org/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=R+Development+Core+Team.&title=R%3A+A+Language+and+Environment+for+Statistical+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DR%253A%2520A%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DThe%2520R%2520Foundation%2520for%2520Statistical%2520Computing%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Ito, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiba, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horikawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishigami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gotoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatsubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanamitsu, S.-I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahara, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niinuma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukamoto, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, Y.</span><span> </span><span class="NLM_article-title">Which Concentration of the Inhibitor Should Be Used to Predict in Vivo Drug Interactions From in Vitro Data?</span> <span class="citation_source-journal">AAPS PharmSci</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1208/ps040425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;key=10.1208%2Fps040425" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=53-60&issue=4&author=K.+Itoauthor=K.+Chibaauthor=M.+Horikawaauthor=M.+Ishigamiauthor=N.+Mizunoauthor=J.+Aokiauthor=Y.+Gotohauthor=T.+Iwatsuboauthor=S.-I.+Kanamitsuauthor=M.+Katoauthor=I.+Kawaharaauthor=K.+Niinumaauthor=A.+Nishinoauthor=N.+Satoauthor=Y.+Tsukamotoauthor=K.+Uedaauthor=T.+Itohauthor=Y.+Sugiyama&title=Which+Concentration+of+the+Inhibitor+Should+Be+Used+to+Predict+in+Vivo+Drug+Interactions+From+in+Vitro+Data%3F&doi=10.1208%2Fps040425"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1208%2Fps040425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fps040425%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DK.%26aulast%3DHorikawa%26aufirst%3DM.%26aulast%3DIshigami%26aufirst%3DM.%26aulast%3DMizuno%26aufirst%3DN.%26aulast%3DAoki%26aufirst%3DJ.%26aulast%3DGotoh%26aufirst%3DY.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DKanamitsu%26aufirst%3DS.-I.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DKawahara%26aufirst%3DI.%26aulast%3DNiinuma%26aufirst%3DK.%26aulast%3DNishino%26aufirst%3DA.%26aulast%3DSato%26aufirst%3DN.%26aulast%3DTsukamoto%26aufirst%3DY.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DY.%26atitle%3DWhich%2520Concentration%2520of%2520the%2520Inhibitor%2520Should%2520Be%2520Used%2520to%2520Predict%2520in%2520Vivo%2520Drug%2520Interactions%2520From%2520in%2520Vitro%2520Data%253F%26jtitle%3DAAPS%2520PharmSci%26date%3D2002%26volume%3D4%26issue%3D4%26spage%3D53%26epage%3D60%26doi%3D10.1208%2Fps040425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4J05" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4J05','PDB','4J05'); return false;">PDB: 4J05</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01317">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_43313"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01317">10.1021/acs.jmedchem.6b01317</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Summary of 15 substrates used to validate OCT1 docking site, the ratios of <i>K</i><sub>ii</sub> and <i>K</i><sub>is</sub> for selective inhibitors, comparison of IC<sub>50</sub> values for OCT1 inhibitors in four different cell lines, effect of thiamine on the uptake of ASP<sup>+</sup> by OCT1, selected inhibition studies of OCT1 ligands and their estimated IC<sub>50</sub> values, Lineweaver–Burk plots for discriminating between competitive and noncompetitive inhibitors of OCT1-mediated metformin uptake, differences in physicochemical properties of OCT1-selective and pan inhibitors, 2D dissimilarity clustering of 15 known substrates of OCT1, and pairwise sequence alignment of PiPT and OCT1 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Full list of Pharmakon library compounds with SMILES strings, ASP<sup>+</sup> inhibition data, and docking scores; list of HMDB compounds with docking ranks; OCT1, OATP1B1, and OATP1B3 inhibition results; names and descriptions of the 21 descriptors used in SAR modeling; and slope and intercept data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">OCT1 comparative model (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_001.pdf">jm6b01317_si_001.pdf (718.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_002.xlsx">jm6b01317_si_002.xlsx (367.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01317/suppl_file/jm6b01317_si_003.pdb">jm6b01317_si_003.pdb (219.8 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01317&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01317%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-7" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01317" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799ef3859693d1e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
